{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Library design for SI-14\n",
    "\n",
    "by Pu Zheng and Jun-Han Su\n",
    "\n",
    "This library may be ordered from TWIST"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#minimum imports:\n",
    "import time,os,sys,glob\n",
    "import cPickle as pickle\n",
    "import numpy as np\n",
    "import khmer\n",
    "sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "\n",
    "from LibraryConstruction import fastaread,fastawrite,fastacombine\n",
    "import LibraryDesigner as ld\n",
    "import LibraryConstruction as lc"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 0.1 Indexing for human genome hg38\n",
    "(skip step 0 if indeces have been created)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of fasta files: 455\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_JH159147v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_GL383565v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270539v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270889v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270887v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270876v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270510v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270859v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270755v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15_GL383555v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270511v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270933v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr13_KI270839v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270364v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_GL000251v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr13_KI270843v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_GL383583v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270932v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15_KI270727v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270312v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15_KI270848v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15_KI270906v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270396v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrX_KI270913v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_KI270799v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270923v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL949749v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_KI270827v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270391v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270784v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270418v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_KI270722v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_GL000218v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270411v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_GL000255v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270386v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_GL383552v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270716v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270866v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270921v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270760v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr7_KI270809v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270750v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270767v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_KI270795v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270811v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270857v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270303v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270781v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270783v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_KI270786v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270815v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270920v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18_KI270912v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270886v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270335v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KB663609v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270732v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270388v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270374v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_GL383519v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270744v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270389v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr16.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270752v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270395v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr7.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270910v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr13_KI270838v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_KI270832v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_KI270794v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_GL383547v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270521v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270900v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_GL000219v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_KI270830v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_GL000213v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18_GL383567v1_alt.fa\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr16_KI270854v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr21_KI270874v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270772v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr16_KI270728v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr9_GL383542v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270766v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270333v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_JH159136v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270310v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270922v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270916v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr16_KI270856v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270382v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270387v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270885v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrX.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_GL000214v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_KI270801v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270733v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_GL949742v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_JH159148v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270706v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_GL000195v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270384v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270915v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_KI270802v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr7_KI270804v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270861v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_KI270798v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270868v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_KI270846v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_KI270726v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_KI270721v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL949748v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270538v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270765v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270589v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270423v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270420v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270593v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_KI270835v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr7_KI270805v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_KI270725v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15_GL383554v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr20.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270442v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270709v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270363v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270820v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270914v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270909v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270895v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270931v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr13_KI270840v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_GL000009v2_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr7_KI270808v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270518v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_KI270897v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr13.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270379v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_KI270903v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_GL383550v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270756v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270934v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_GL383528v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270512v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270516v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270762v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_KI270785v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_GL000254v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270591v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrX_KI270881v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_KI270787v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_KI270829v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_GL000208v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr9.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270739v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270736v1_random.fa\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15_KI270849v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270584v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_KB021644v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr21_KI270872v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270749v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_GL877876v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_KI270796v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270734v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270467v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270711v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270745v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_KI270902v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270338v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_GL383563v3_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270810v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270334v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270757v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270865v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270816v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr21_GL383578v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270867v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270879v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270515v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr7_KI270807v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270746v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270882v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270509v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_KI270758v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr9_KI270720v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270936v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_GL000008v2_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270340v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_GL000226v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL949753v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_GL000252v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270381v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr10_GL383545v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_GL000250v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270771v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL383574v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270924v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_KI270844v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270778v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270708v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270928v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270754v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr20_GL383577v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr9_GL383539v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_GL383553v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL383573v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270320v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270710v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_KI270847v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr21_GL383580v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_KI270797v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_GL000194v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270878v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270883v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270336v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270779v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270530v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr9_KI270717v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr9_KI270718v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270888v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_GL383522v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270315v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_KI270834v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270588v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270858v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL000209v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270821v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270893v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270764v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270907v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18_KI270911v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270918v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270875v1_alt.fa\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270337v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270822v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270468v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270894v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_KI270896v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270317v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr21.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18_KI270864v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15_KI270851v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_GL000253v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270371v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_GL000221v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270383v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270761v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_KI270791v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270926v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270777v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270507v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270737v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_GL383526v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270316v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270390v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270330v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270743v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_GL383549v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_KI270790v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270891v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270583v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270417v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270580v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270769v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr20_KI270871v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270528v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_GL000225v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_GL383551v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15_KI270852v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270738v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270419v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270753v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270818v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270322v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr20_KI270869v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18_GL383571v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270362v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18_GL383568v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270908v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270890v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrX_KI270880v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270302v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15_KI270850v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270929v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr10.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_GL000258v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_GL383582v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270770v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270465v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_GL383533v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270773v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr16_GL383556v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr9_KI270823v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL383575v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270731v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_JH636055v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270817v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270735v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270581v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270935v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_KI270792v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270747v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_GL383520v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270714v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270751v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270394v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr16_GL383557v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270814v1_alt.fa\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr21_GL383579v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270305v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_GL383566v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_GL000224v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270937v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr15_KI270905v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_GL000205v2_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_GL383527v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270862v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270466v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_GL000216v2.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr9_GL383541v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr13_KI270841v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18_GL383570v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270819v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_KI270925v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270763v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270529v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270715v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270548v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270438v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270366v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr10_KI270825v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270780v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18_KI270863v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270422v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270376v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270582v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18_GL383572v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270812v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270392v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_GL000220v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270742v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270768v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_GL000257v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_KI270833v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270813v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr9_KI270719v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270412v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270414v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_KI270831v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr10_KI270824v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr7_GL383534v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL949751v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270938v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_GL383521v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270860v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_KI270898v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_GL383532v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL949752v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_KI270800v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270748v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270919v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_KI270724v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr7_KI270803v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr22_KI270877v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr20_KI270870v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270892v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrM.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270544v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr9_GL383540v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270712v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270930v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr8_KI270901v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr7_KI270806v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270707v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270579v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270329v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270587v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL383576v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270884v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_GL339449v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL949747v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr21_GL383581v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270304v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_KI270788v1_alt.fa\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270776v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270393v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_GL383530v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270424v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr10_GL383546v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrY.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr7_KI270899v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_GL582966v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270425v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr21_KI270873v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_KI270836v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr6_GL000256v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_KI270826v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270373v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18_GL383569v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270375v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_GL383564v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270508v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270435v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL949746v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr3_KI270782v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrY_KI270740v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_KI270837v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270385v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr16_KI270853v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270429v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270448v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr13_KI270842v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270713v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_GL877875v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270741v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_JH159146v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_KI270793v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270311v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr16_KI270855v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr18.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270729v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_KI270917v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_GL383518v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr17_KI270730v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_KI270845v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr19_GL949750v2_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_JH159137v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270517v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr4_KI270789v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr12_KI270904v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270378v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr11_KI270927v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270774v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr1_KI270759v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr5_GL383531v1_alt.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270372v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270519v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr14_KI270723v1_random.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270590v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chrUn_KI270522v1.fa\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/chr2_KI270775v1_alt.fa\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "12002721064"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#Only do it once!\n",
    "#This example is for the human genome hg38\n",
    "\n",
    "#Construct whole genome hash table / similarly for transcriptome file\n",
    "#Note: This is NOT degenerate for reverse-complement and it only maps the + strand.\n",
    "\n",
    "#minimum imports:\n",
    "#import khmer,sys,glob\n",
    "#sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "#from LibraryConstruction import fastaread,fastawrite\n",
    "\n",
    "#khmer's hash tables are very easy to understand at least with single threads (no parallel computation)\n",
    "#Simply, each hash(sequence) = number in base 4. Z(num_table) prime numbers < a big number are specified \n",
    "#and the remainded of the hash is used to adress the Z tables. This allows for an easy bloom filter for finding\n",
    "#missing kmers. Considering its simplicity for single thread there should be a Windows version!\n",
    "#http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0101271\n",
    "\n",
    "ksize = 17 #word size\n",
    "kmer = khmer.Countgraph(ksize, 3e9, 4) #hash total table size 2e9*4. 4 is the number of tables (prime numbers) to use\n",
    "kmer.set_use_bigcount(True) #This allows 2bytes for each count. Thus the maximum count is 2**16-1\n",
    "#The total RAM required for this is thus 2e9*4*2*1.2 bytes = 19.2 gigabytes. \n",
    "#~20% is used for hash specific elements as documented by khmer.\n",
    "#The size of the hash table is allocated at the beginning. \n",
    "#Note: if the size is too small there will be many collisions in the hash resulting in 'overinflated' counts.\n",
    "#Note: unknown characters are mapped to A. (N->A)\n",
    "#Note: the sequences need to be capitalized. (c->A, but C->C)\n",
    "\n",
    "fa_fls = glob.glob(r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms/*.fa') #permanant dir\n",
    "print 'Number of fasta files: '+str(len(fa_fls));\n",
    "save_file = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Indeces/human/hg38/full_word'+str(ksize)+'_.kmer' #permanant dir\n",
    "\n",
    "for fa_fl in fa_fls:\n",
    "    print \"Dealing with file:\"+fa_fl\n",
    "    nms,seqs = fastaread(fa_fl)\n",
    "    for seq in seqs:\n",
    "        kmer.consume(seq.upper())\n",
    "kmer.save(save_file)\n",
    "os.path.getsize(save_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 0.2 Indexing for human mRNA"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "hg38-mRNA\t\t\t Homo_sapiens.GRCh38.ncrna.fa  rtRNA_hg38.fasta\r\n",
      "Homo_sapiens.GRCh38.cdna.all.fa  Info.txt\r\n"
     ]
    }
   ],
   "source": [
    "! ls /n/boslfs/LABS/zhuang_lab/User/pzheng/Transcriptome/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of fasta files: 2\n",
      "Dealing with mRNA file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Transcriptome/Homo_sapiens.GRCh38.cdna.all.fa\n",
      "Dealing with mRNA file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Transcriptome/Homo_sapiens.GRCh38.ncrna.fa\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "8000055296"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Indexing for human mRNA\n",
    "ksize = 17 #word size\n",
    "mRNA_kmer = khmer.Countgraph(ksize, 2e9, 4) #hash total table size 2e9*4. 4 is the number of tables (prime numbers) to use\n",
    "mRNA_kmer.set_use_bigcount(True) #This allows 2bytes for each count. Thus the maximum count is 2**16-1\n",
    "#The total RAM required for this is thus 2e9*4*2*1.2 bytes = 19.2 gigabytes. \n",
    "#~20% is used for hash specific elements as documented by khmer.\n",
    "#The size of the hash table is allocated at the beginning. \n",
    "#Note: if the size is too small there will be many collisions in the hash resulting in 'overinflated' counts.\n",
    "#Note: unknown characters are mapped to A. (N->A)\n",
    "#Note: the sequences need to be capitalized. (c->A, but C->C)\n",
    "\n",
    "mRNA_fa_fls = glob.glob(r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Transcriptome/*.fa') #permanant dir\n",
    "print 'Number of fasta files: '+str(len(mRNA_fa_fls));\n",
    "mRNA_save_file = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Indeces/human/hg38/transcriptome_word'+str(ksize)+'_.kmer' #permanant dir\n",
    "\n",
    "for fa_fl in mRNA_fa_fls:\n",
    "    print \"Dealing with mRNA file:\"+fa_fl\n",
    "    nms,seqs = fastaread(fa_fl)\n",
    "    for seq in seqs:\n",
    "        if len(seq) <= ksize:\n",
    "            continue;\n",
    "        mRNA_kmer.consume(seq.upper())\n",
    "mRNA_kmer.save(mRNA_save_file)\n",
    "os.path.getsize(mRNA_save_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 0.3 Indexing for human repeat sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of fasta files: 1\n",
      "Dealing with file:/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/HumanRepeats.fasta\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "7999999826"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ksize = 17 #word size\n",
    "kmer = khmer.Countgraph(ksize, 2e9, 4) #hash total table size 2e9*4. 4 is the number of tables (prime numbers) to use\n",
    "kmer.set_use_bigcount(True) #This allows 2bytes for each count. Thus the maximum count is 2**16-1\n",
    "#The total RAM required for this is thus 2e9*4*2*1.2 bytes = 19.2 gigabytes. \n",
    "#~20% is used for hash specific elements as documented by khmer.\n",
    "#The size of the hash table is allocated at the beginning. \n",
    "#Note: if the size is too small there will be many collisions in the hash resulting in 'overinflated' counts.\n",
    "#Note: unknown characters are mapped to A. (N->A)\n",
    "#Note: the sequences need to be capitalized. (c->A, but C->C)\n",
    "\n",
    "fa_fls = glob.glob(r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/HumanRepeats.fasta') #permanant dir\n",
    "print 'Number of fasta files: '+str(len(fa_fls));\n",
    "save_file = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Indeces/human/hg38/HumanRepeats_word'+str(ksize)+'_.kmer' #permanant dir\n",
    "\n",
    "for fa_fl in fa_fls:\n",
    "    print \"Dealing with file:\"+fa_fl\n",
    "    nms,seqs = fastaread(fa_fl)\n",
    "    for seq in seqs:\n",
    "        kmer.consume(seq.upper())\n",
    "kmer.save(save_file)\n",
    "os.path.getsize(save_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "full_word17_.kmer\t   mRNA17_.kmer\r\n",
      "HumanRepeats_word17_.kmer  transcriptome_word17_.kmer\r\n"
     ]
    }
   ],
   "source": [
    "!ls /n/boslfs/LABS/zhuang_lab/User/pzheng/Indeces/human/hg38/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 1.1 EMT"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Batch_Extract_Sequences(region_folder, reg_filename =r'Regions.txt', \\\n",
    "                            genome_folder=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms',\\\n",
    "                            save_dir=r'region_seqs',\\\n",
    "                            merge=True, save=True, verbose=True, \n",
    "                            resolution=10000, flanking=100000):\n",
    "    '''Function to extract sequences for all regions written in a file\n",
    "    Inputs:\n",
    "        region_folder: directory for this library and also directory for the region file, string\n",
    "        \n",
    "        merge: if merge all regions together and arrange region id, bool'''\n",
    "    # save dir full path\n",
    "    save_folder = region_folder + os.sep + save_dir;\n",
    "    # check input type\n",
    "    if not isinstance(region_folder, str) and not isinstance(reg_filename, str) \\\n",
    "        and not isinstance(genome_folder, str):\n",
    "        raise ValueError('wrong input format!');\n",
    "\n",
    "    def read_region_file(region_folder=region_folder, reg_filename=reg_filename, _verbose=verbose):\n",
    "        '''Sub-function to read region file'''\n",
    "        # region filename\n",
    "        _reg_filename = region_folder + os.sep + reg_filename;\n",
    "        _reg_file = open(_reg_filename, 'r');    \n",
    "        if _verbose:\n",
    "            print '-- Input region file is: '+ _reg_filename;\n",
    "        # start reading\n",
    "        _lines = _reg_file.read().split('\\n')\n",
    "        _titles = _lines[0].split('\\t');\n",
    "        # save a list of dictionaries\n",
    "        _reg_list = [];\n",
    "        for _line in _lines[1:]:\n",
    "            _reg_dic = {}; # dinctionary to save all informations\n",
    "            _info = _line.split('\\t'); # split informations\n",
    "            if len(_info) != len(_titles): # sanity check to make sure they are of the same size\n",
    "                continue;\n",
    "            for _i in range(len(_info)): # save info to dic\n",
    "                _reg_dic[_titles[_i]] = _info[_i];\n",
    "            _reg_list.append(_reg_dic); # save dic to list\n",
    "        _reg_file.close();\n",
    "        return _reg_list\n",
    "\n",
    "    def parse_region(reg_dic):\n",
    "        '''given a dictionary of one region, \n",
    "        report:\n",
    "            _chrom: str\n",
    "            _start: int\n",
    "            _stop: int'''\n",
    "        region_str = reg_dic['Region'];\n",
    "        # grab chromosome\n",
    "        _chrom = region_str.split(':')[0];\n",
    "        _locus = region_str.split(':')[1];\n",
    "        # grab start and stop positions\n",
    "        _start, _stop = _locus.split('-')\n",
    "        _start = int(_start.replace(',', ''));\n",
    "        _stop = int(_stop.replace(',', ''));\n",
    "        # return in this order:\n",
    "        return _chrom, _start, _stop\n",
    "    \n",
    "    def extract_sequence(reg_dic, genome_folder=genome_folder, \\\n",
    "                         resolution=resolution, flanking=flanking, \\\n",
    "                         save_folder=save_folder, save=save, merge=merge,_verbose=verbose):\n",
    "        from math import ceil\n",
    "        '''sub-function to extract sequences of one locus\n",
    "        Given:\n",
    "        reg_dic: dic for region info, dictionary\n",
    "        genome_folder: dir for genome files, str\n",
    "        resolution: resolution of each region in bp, int\n",
    "        flanking: upstream and downstream included in bp, int\n",
    "        save: if save as fasta files, bool\n",
    "        Return:\n",
    "        dic of sequences of designed regions\n",
    "        Dependencies:\n",
    "        ld.fastaread, ld.fastawrite, ./parse_region'''\n",
    "        if _verbose:\n",
    "            print \"-- Dealing with:\", reg_dic;\n",
    "        # get chromosome, start and stop information\n",
    "        _reg_dic=reg_dic.copy()\n",
    "        _chrom, _start, _stop = parse_region(_reg_dic);\n",
    "        # dir to store sequences, this will be returned\n",
    "        _seq_dic = {};\n",
    "        _seqs = []\n",
    "        _names = []\n",
    "        # read chromosome seq\n",
    "        if _verbose:\n",
    "            print \"--- Reading genome file: \", str(_chrom)+'.fa';\n",
    "        _, _wholechr = ld.fastaread(genome_folder+os.sep+_chrom+'.fa')\n",
    "        _wholechr = _wholechr[0];\n",
    "        # number of regions\n",
    "        _n_reg = int(ceil( float(_stop+flanking - (_start-flanking)) / resolution))\n",
    "        _reg_dic['regions'] = _n_reg;\n",
    "        if _verbose:\n",
    "            print \"--- Number of regions: \", _n_reg;\n",
    "        # extract all required seq\n",
    "        _whole_seq = _wholechr[_start-flanking: min(_start-flanking+_n_reg*resolution, len(_wholechr))];\n",
    "        for _i in range(_n_reg):\n",
    "            # extract sequence for this region\n",
    "            if len(_whole_seq) < (_i+1)*resolution:\n",
    "                break;\n",
    "            _seq = _whole_seq[_i*resolution:(_i+1)*resolution];\n",
    "            _name = _chrom+':'+str(_start-flanking+_i*resolution)+'-'+\\\n",
    "                     str(_start-flanking+(_i+1)*resolution)+'_reg_'+str(_i+1);\n",
    "            _seq_dic[_name] = _seq;\n",
    "            _seqs.append(_seq);\n",
    "            _names.append(_name);\n",
    "        # if Save\n",
    "        if save:                \n",
    "            # mkdir if not exist for save folder\n",
    "            if not os.path.exists(save_folder):\n",
    "                os.makedirs(save_folder)\n",
    "                \n",
    "            if merge: # NOTICE! this will never overwrite!\n",
    "                save_sub_folder = save_folder+os.sep+'merged';\n",
    "                if not os.path.exists(save_sub_folder):\n",
    "                    os.makedirs(save_sub_folder)\n",
    "                ex_file_num = len(glob.glob(save_sub_folder+os.sep+r'*'));\n",
    "                # writing files\n",
    "                for _i in range(_n_reg):\n",
    "                    _filename = save_sub_folder + os.sep + 'reg_' + str(ex_file_num+_i+1) + '.fasta';\n",
    "                    # save as fasta\n",
    "                    if 'Gene' in reg_dic.keys():\n",
    "                        ld.fastawrite(_filename, [_names[_i]+'_gene_'+reg_dic['Gene']], [_seqs[_i].upper()])\n",
    "                    else:\n",
    "                        ld.fastawrite(_filename, [_names[_i]], [_seqs[_i].upper()])\n",
    "                print '-- Number of region: '+str(len(glob.glob(save_sub_folder+os.sep+r'*')));\n",
    "            else:\n",
    "                # assign correct name of the sub folder\n",
    "                if 'Gene' in reg_dic.keys():\n",
    "                    save_sub_folder = save_folder+os.sep+reg_dic['Gene'];\n",
    "                else:\n",
    "                    save_sub_folder = save_folder+os.sep+_chrom+str(_start);\n",
    "                # mkdir if not exist for this region\n",
    "                if not os.path.exists(save_sub_folder):\n",
    "                    os.makedirs(save_sub_folder)\n",
    "                # writing files\n",
    "                for _i in range(_n_reg):\n",
    "                    _filename = save_sub_folder + os.sep + 'reg_' + str(_i+1) + '.fasta';\n",
    "                    # save as fasta\n",
    "                    ld.fastawrite(_filename, [_names[_i]], [_seqs[_i].upper()])\n",
    "                \n",
    "        return _seq_dic, _reg_dic\n",
    "        \n",
    "    ## read region file\n",
    "    reg_list = read_region_file();\n",
    "    # extract sequences and save!\n",
    "    seq_dic_list, reg_update_list= [],[];\n",
    "    for reg_dic in reg_list:\n",
    "        _seqs, _dic= extract_sequence(reg_dic, save=True)\n",
    "        seq_dic_list.append(_seqs);\n",
    "        reg_update_list.append(_dic)\n",
    "        \n",
    "    return seq_dic_list, reg_update_list\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Input region file is: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/Regions.txt\n",
      "-- Dealing with: {'Region': 'chr16:68,636,189-68,835,548', 'Gene': 'CDH1-3', 'Type': 'Epi-Mes', 'Strand': '+'}\n",
      "--- Reading genome file:  chr16.fa\n",
      "--- Number of regions:  22\n",
      "-- Dealing with: {'Region': 'chr18:27,932,879-28,177,481', 'Gene': 'CDH2', 'Type': 'Mes', 'Strand': '-'}\n",
      "--- Reading genome file:  chr18.fa\n",
      "--- Number of regions:  25\n",
      "-- Dealing with: {'Region': 'chr3:190,305,701-190,322,475', 'Gene': 'CLDN1', 'Type': 'Epi', 'Strand': '-'}\n",
      "--- Reading genome file:  chr3.fa\n",
      "--- Number of regions:  13\n",
      "-- Dealing with: {'Region': 'chr10:17,227,935-17,237,593', 'Gene': 'VIM', 'Type': 'Mes', 'Strand': '+'}\n",
      "--- Reading genome file:  chr10.fa\n",
      "--- Number of regions:  13\n",
      "-- Dealing with: {'Region': 'chr12:52,948,871-52,952,901', 'Gene': 'KRT18', 'Type': 'Epi', 'Strand': '+'}\n",
      "--- Reading genome file:  chr12.fa\n",
      "--- Number of regions:  13\n",
      "-- Dealing with: {'Region': 'chr17:41,582,279-41,586,921', 'Gene': 'KRT14', 'Type': 'Epi', 'Strand': '-'}\n",
      "--- Reading genome file:  chr17.fa\n",
      "--- Number of regions:  13\n",
      "-- Dealing with: {'Region': 'chr16:55,389,700-55,506,691', 'Gene': 'MMP2', 'Type': 'Mes', 'Strand': '+'}\n",
      "--- Reading genome file:  chr16.fa\n",
      "--- Number of regions:  18\n",
      "-- Dealing with: {'Region': 'chr2:215,360,440-215,436,172', 'Gene': 'FN1', 'Type': 'Mes', 'Strand': '-'}\n",
      "--- Reading genome file:  chr2.fa\n",
      "--- Number of regions:  16\n",
      "-- Dealing with: {'Region': 'chr12:54,395,261-54,419,460', 'Gene': 'ITGA5', 'Type': 'Epi', 'Strand': '-'}\n",
      "--- Reading genome file:  chr12.fa\n",
      "--- Number of regions:  14\n",
      "-- Dealing with: {'Region': 'chr20:46,008,908-46,016,561', 'Gene': 'MMP9', 'Type': 'Mes', 'Strand': '+'}\n",
      "--- Reading genome file:  chr20.fa\n",
      "--- Number of regions:  13\n",
      "-- Dealing with: {'Region': 'chr10:31,318,495-31,529,814', 'Gene': 'ZEB1', 'Type': 'Mes', 'Strand': '+'}\n",
      "--- Reading genome file:  chr10.fa\n",
      "--- Number of regions:  23\n",
      "-- Dealing with: {'Region': 'chr15:29,699,367-29,968,865', 'Gene': 'ZO-1', 'Type': 'Epi', 'Strand': '-'}\n",
      "--- Reading genome file:  chr15.fa\n",
      "--- Number of regions:  26\n",
      "-- Dealing with: {'Region': 'chr1:162,631,265-162,787,400', 'Gene': 'DDR2', 'Type': 'Mes', 'Strand': '+'}\n",
      "--- Reading genome file:  chr1.fa\n",
      "--- Number of regions:  20\n",
      "-- Dealing with: {'Region': 'chr17:50,183,289-50,201,648', 'Gene': 'COL1A1', 'Type': 'Epi', 'Strand': '-'}\n",
      "--- Reading genome file:  chr17.fa\n",
      "--- Number of regions:  13\n",
      "-- Dealing with: {'Region': 'chr17:41,500,981-41,505,705', 'Gene': 'KRT13', 'Type': 'Epi', 'Strand': '-'}\n",
      "--- Reading genome file:  chr17.fa\n",
      "--- Number of regions:  13\n",
      "-- Dealing with: {'Region': 'chrX:15,494,402-15,602,148', 'Gene': 'ACE2', 'Type': 'Mes', 'Strand': '-'}\n",
      "--- Reading genome file:  chrX.fa\n",
      "--- Number of regions:  18\n",
      "-- Dealing with: {'Region': 'chr11:65,422,774-65,506,516', 'Gene': 'MALAT1-NEAT1', 'Type': 'Mes', 'Strand': '+'}\n",
      "--- Reading genome file:  chr11.fa\n",
      "--- Number of regions:  17\n",
      "-- Dealing with: {'Region': 'chr18:6,941,744-7,117,814', 'Gene': 'LAMA1', 'Type': 'Epi', 'Strand': '-'}\n",
      "--- Reading genome file:  chr18.fa\n",
      "--- Number of regions:  21\n"
     ]
    }
   ],
   "source": [
    "## Some Inputs\n",
    "# human genome\n",
    "genome_folder = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Genomes/human/hg38/chroms';\n",
    "resolution = 20000;\n",
    "EMT_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT';\n",
    "save_dir = r'region_seqs-'+str(resolution)\n",
    "merged=False\n",
    "seq_dic_list, reg_list = Batch_Extract_Sequences(region_folder=EMT_dir,\n",
    "                                                 genome_folder=genome_folder,\n",
    "                                                 save_dir=save_dir,\n",
    "                                                 resolution=20000,\n",
    "                                                 flanking=120000,\n",
    "                                                 merge=merged, save=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Batch_Probe_Design(master_dir, resolution,\n",
    "                       index_dir=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Indeces/human/hg38',\n",
    "                       seq_dir=r'region_seqs',\n",
    "                       save_dir=r'reports',\n",
    "                       merge=True, save=True, verbose=True):\n",
    "    '''Function to batch design probes.\n",
    "    Input\n",
    "        master: directory for this library, string\n",
    "        resolution: resolution for parsed directory, if given will change seq_dir and save_dir\n",
    "        index_dir: directory for stored index files, string\n",
    "        seq_dir: sub directory for all parsed sequences, string\n",
    "        save_dir: sub directory for saving probe reports, string\n",
    "        merge: whether sequences are merged, bool\n",
    "        save: whether save reports, bool\n",
    "        verbose: print a lot of things!, bool\n",
    "    Output:\n",
    "        _n_reg: Number of designed regions, int\n",
    "        '''\n",
    "    # imports\n",
    "    if verbose:\n",
    "        print'-1.Importing packages';\n",
    "    import time,os,sys\n",
    "    sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "    import LibraryDesigner as ld\n",
    "    import matplotlib.pyplot as plt\n",
    "    plt.switch_backend('agg')\n",
    "    # names for indeces\n",
    "    if verbose:\n",
    "        print '-2.Checking index file';\n",
    "    _genome_index = index_dir + os.sep + r'full_word17_.kmer';\n",
    "    _repeat_index = index_dir + os.sep + r'HumanRepeats_word17_.kmer';\n",
    "    _transcriptome_index = index_dir + os.sep + r'transcriptome_word17_.kmer';\n",
    "    if not os.path.isfile(_genome_index) or not os.path.isfile(_repeat_index) or not os.path.isfile(_transcriptome_index):\n",
    "        print 'test'\n",
    "        raise EOFError('could not find index files!');\n",
    "\n",
    "    # naming related to resolution\n",
    "    if verbose:\n",
    "        print '-3.Patching Resolution:',resolution;\n",
    "    if resolution: # not \"None\" or 0\n",
    "        _seq_dir = seq_dir + '-' +str(resolution);\n",
    "        _save_dir = save_dir + '-' +str(resolution);\n",
    "    else:\n",
    "        _seq_dir = seq_dir;\n",
    "        _save_dir = save_dir;\n",
    "        \n",
    "    # naming related\n",
    "    if verbose:\n",
    "        print '-4.Getting folder list if merged:', merge;\n",
    "    if merge:\n",
    "        _seq_dir += os.sep + 'merged';\n",
    "        _save_dir += os.sep + 'merged';\n",
    "        # convert to list for consistency\n",
    "        _seq_dirs = [_seq_dir]\n",
    "        _save_dirs = [_save_dir]\n",
    "    else:\n",
    "        _seq_dirs, _save_dirs = [], []\n",
    "        # call all subfolders\n",
    "        _subfolders = next(os.walk(master_dir+os.sep+_seq_dir))[1];\n",
    "        for _subfolder in _subfolders:\n",
    "            if not 'merge' in _subfolder:\n",
    "                _seq_dirs.append(_seq_dir + os.sep + _subfolder);\n",
    "                _save_dirs.append(_save_dir + os.sep + _subfolder);\n",
    "    if verbose:\n",
    "        print \"--4.1.Input folders:\\n\",_seq_dirs\n",
    "        print \"--4.2.Output folders:\\n\",_save_dirs\n",
    "    \n",
    "    def _probe_design(_input_folder, _output_folder, \n",
    "                      _genome_index=_genome_index, _repeat_index=_repeat_index, _transcriptome_index=_transcriptome_index,\n",
    "                      _save=save, _verbose=verbose):\n",
    "        '''Design probes for sequence files in the same folder'''\n",
    "        # input files\n",
    "        _input_files = glob.glob(_input_folder + os.sep+r'*.fasta');\n",
    "        # output folder\n",
    "        if not os.path.exists(_output_folder):\n",
    "            os.makedirs(_output_folder);\n",
    "        # probe_number\n",
    "        _pb_numbers = []\n",
    "        for _in_file in _input_files:\n",
    "            # save file name\n",
    "            _save_file = _output_folder+os.sep+os.path.basename(_in_file).replace('.fasta','.pbr')\n",
    "            # start iteration if no report exists\n",
    "            if not os.path.exists(_save_file):\n",
    "                # get local genome, doesn't count in probe designer filter\n",
    "                _local_genome_fl = _in_file\n",
    "                if _verbose:\n",
    "                    print '--- ', _in_file;\n",
    "                \n",
    "                _pb_designer = ld.pb_reports_class(\n",
    "                    sequence_dic={'file':_in_file,'use_revc':False,'use_kmer':True},\n",
    "                    map_dic={'genome':{'file':_genome_index,'use_revc':True,'use_kmer':True},\n",
    "                          'rep_genome':{'file':_repeat_index,'use_revc':True,'use_kmer':True},\n",
    "                          'local_genome':{'file':_local_genome_fl,'force_list':True,'use_revc':True,'use_kmer':True}},\n",
    "                    save_file=_save_file,\n",
    "                    params_dic={'word_size':17,'pb_len':42,'buffer_len':2,'max_count':2**16-1,'check_on_go':False,'auto':False},\n",
    "                    dic_check={('genome','local_genome'):75,'rep_genome':0,'gc':[0.25,0.85],'tm':70,'masks':['AAAAA','TTTTT','GGGGG','CCCCC']})\n",
    "                if _save:\n",
    "                    _pb_designer.computeOTmaps()\n",
    "                    _pb_designer.compute_pb_report()\n",
    "                    _pb_designer.perform_check_end()\n",
    "                    _pb_designer.plots()\n",
    "                _pb_numbers.append(len(_pb_designer.pb_reports_keep))\n",
    "                if _verbose:\n",
    "                    print '--- ', len(_pb_designer.pb_reports_keep)\n",
    "                \n",
    "        return len(_input_files), _pb_numbers;\n",
    "    \n",
    "    # Design Probes\n",
    "    if verbose:\n",
    "        print '-5.Design probes in each folder';\n",
    "    _reg_nums = []\n",
    "    _pb_num_list = []\n",
    "    for _seq_dir, _save_dir in zip(_seq_dirs, _save_dirs):\n",
    "        if verbose:\n",
    "            print '--  sequences from:', _seq_dir\n",
    "        _reg_num, _pb_nums = _probe_design(_input_folder=master_dir+os.sep+_seq_dir,\n",
    "                                  _output_folder=master_dir+os.sep+_save_dir);\n",
    "        _reg_nums.append(_reg_num);\n",
    "        _pb_num_list += _pb_nums;\n",
    "        if verbose:\n",
    "            print '--  regions designed:', _reg_num    \n",
    "            print '--  probes for this region:', sum(_pb_nums)\n",
    "    return _reg_nums, _pb_num_list\n",
    "        \n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-1.Importing packages\n",
      "-2.Checking index file\n",
      "-3.Patching Resolution: 20000\n",
      "-4.Getting folder list if merged: False\n",
      "--4.1.Input folders:\n",
      "['region_seqs-20000/MMP9', 'region_seqs-20000/KRT18', 'region_seqs-20000/KRT13', 'region_seqs-20000/CDH1-3', 'region_seqs-20000/CDH2', 'region_seqs-20000/ZEB1', 'region_seqs-20000/FN1', 'region_seqs-20000/KRT14', 'region_seqs-20000/MALAT1-NEAT1', 'region_seqs-20000/ZO-1', 'region_seqs-20000/VIM', 'region_seqs-20000/ITGA5', 'region_seqs-20000/ACE2', 'region_seqs-20000/LAMA1', 'region_seqs-20000/DDR2', 'region_seqs-20000/COL1A1', 'region_seqs-20000/CLDN1', 'region_seqs-20000/MMP2']\n",
      "--4.2.Output folders:\n",
      "['reports-20000/MMP9', 'reports-20000/KRT18', 'reports-20000/KRT13', 'reports-20000/CDH1-3', 'reports-20000/CDH2', 'reports-20000/ZEB1', 'reports-20000/FN1', 'reports-20000/KRT14', 'reports-20000/MALAT1-NEAT1', 'reports-20000/ZO-1', 'reports-20000/VIM', 'reports-20000/ITGA5', 'reports-20000/ACE2', 'reports-20000/LAMA1', 'reports-20000/DDR2', 'reports-20000/COL1A1', 'reports-20000/CLDN1', 'reports-20000/MMP2']\n",
      "-5.Design probes in each folder\n",
      "--  sequences from: region_seqs-20000/MMP9\n",
      "--  regions designed: 13\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/KRT18\n",
      "--  regions designed: 13\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/KRT13\n",
      "--  regions designed: 13\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/CDH1-3\n",
      "--  regions designed: 22\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/CDH2\n",
      "--  regions designed: 25\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/ZEB1\n",
      "--  regions designed: 23\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/FN1\n",
      "--  regions designed: 16\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/KRT14\n",
      "--  regions designed: 13\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/MALAT1-NEAT1\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_8.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.431905031\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.98751282692\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.29407310486\n",
      "---  193\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_13.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.288393974\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 8.81635904312\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.07924294472\n",
      "---  304\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_16.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 140.681197166\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.58805203438\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.11730098724\n",
      "---  339\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_14.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 137.749115944\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 14.9287350178\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 2.04588413239\n",
      "---  293\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_17.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 142.243323088\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 14.7224440575\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.92905592918\n",
      "---  349\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_12.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 140.891603947\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.77995491028\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.16790604591\n",
      "---  296\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_4.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 140.378731966\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 14.9760251045\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 2.49337720871\n",
      "---  202\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_7.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 142.721307993\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 15.11358881\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 2.48233008385\n",
      "---  281\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_6.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 137.32546401\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.71808815002\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.20090508461\n",
      "---  239\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_5.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.340546131\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.71096801758\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.1494410038\n",
      "---  283\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_11.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.34404397\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.94843912125\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.37269592285\n",
      "---  137\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_3.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.38625598\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.76546907425\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.1494178772\n",
      "---  261\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_9.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.285442114\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.91957616806\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.3024559021\n",
      "---  155\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_15.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.286478043\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.86199688911\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.18042993546\n",
      "---  290\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_1.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.274150848\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.71530818939\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.26859593391\n",
      "---  249\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_2.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 137.43313694\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.82784700394\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.26471209526\n",
      "---  181\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/MALAT1-NEAT1/reg_10.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.300534964\n",
      "Dealing with sequence: 1 out of 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Time(s): 7.65946507454\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.11282205582\n",
      "---  268\n",
      "--  regions designed: 17\n",
      "--  probes for this region: 4320\n",
      "--  sequences from: region_seqs-20000/ZO-1\n",
      "--  regions designed: 26\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/VIM\n",
      "--  regions designed: 13\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/ITGA5\n",
      "--  regions designed: 14\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/ACE2\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/ACE2/reg_3.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.266581059\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.59713101387\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.18821001053\n",
      "---  194\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/ACE2/reg_9.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.275927067\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.60486292839\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.06075191498\n",
      "---  277\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/ACE2/reg_15.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.272759199\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.66044807434\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.08740901947\n",
      "---  273\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/ACE2/reg_1.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.264651775\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.48145985603\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.13830399513\n",
      "---  287\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/ACE2/reg_2.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.270209074\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.47125911713\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.00021004677\n",
      "---  278\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/ACE2/reg_10.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.256906986\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.48414897919\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.09256792068\n",
      "---  286\n",
      "--  regions designed: 18\n",
      "--  probes for this region: 1595\n",
      "--  sequences from: region_seqs-20000/LAMA1\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_8.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.273468971\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.60522603989\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.02498984337\n",
      "---  305\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_13.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.264647007\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.59084606171\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.02065896988\n",
      "---  241\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_16.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.411429882\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.66424798965\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.18976593018\n",
      "---  211\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_14.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.267772913\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.43633413315\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.23700904846\n",
      "---  206\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_17.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.365311861\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.64738583565\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.27356505394\n",
      "---  187\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_12.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.303732872\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.52720689774\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.1813390255\n",
      "---  213\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_4.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.331946135\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.62135910988\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.0570230484\n",
      "---  274\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_7.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.286235809\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.21283388138\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.888473033905\n",
      "---  312\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_6.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.654304981\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.30809402466\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.976435184479\n",
      "---  257\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_5.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.260226965\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.28643798828\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.986461877823\n",
      "---  240\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_11.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.432688951\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.21950697899\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.973086118698\n",
      "---  305\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_18.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.26662302\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.30210494995\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.978449106216\n",
      "---  248\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_21.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.256609917\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.19001293182\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.896743059158\n",
      "---  309\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_3.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.330291033\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.44510197639\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.959674835205\n",
      "---  287\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_19.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.26939106\n",
      "Dealing with sequence: 1 out of 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Time(s): 7.51612186432\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.13717699051\n",
      "---  173\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_9.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.264953852\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.44173884392\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.946707963943\n",
      "---  298\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_20.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.265758991\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.35339593887\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.17881011963\n",
      "---  172\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_15.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.26198101\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.35423803329\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.08079004288\n",
      "---  242\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_1.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.261363983\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.39199495316\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.03247594833\n",
      "---  221\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_2.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.315752029\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.3599960804\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.993446111679\n",
      "---  252\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/LAMA1/reg_10.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.31986618\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.39848589897\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.03392791748\n",
      "---  258\n",
      "--  regions designed: 21\n",
      "--  probes for this region: 5211\n",
      "--  sequences from: region_seqs-20000/DDR2\n",
      "--  regions designed: 20\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/COL1A1\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_8.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.318860054\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.36945390701\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.964533090591\n",
      "---  293\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_13.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 138.969966888\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.50393104553\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.11330509186\n",
      "---  245\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_12.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.415214062\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.40253806114\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.17910599709\n",
      "---  211\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_4.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.415815115\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.36830997467\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.920157909393\n",
      "---  307\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_7.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.267269135\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.33997797966\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.868182182312\n",
      "---  333\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_6.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.277475834\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.33090114594\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.940440893173\n",
      "---  307\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_5.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.266975164\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.38971495628\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.96160697937\n",
      "---  290\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_11.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.258625984\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.33893299103\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.00907087326\n",
      "---  320\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_3.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.268972874\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.33864593506\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.910153150558\n",
      "---  327\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_9.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.254251003\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.44023394585\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.20247006416\n",
      "---  129\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_1.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.256438971\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.34067392349\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.957273006439\n",
      "---  290\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_2.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.259706974\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.35478997231\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 0.940680027008\n",
      "---  293\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/region_seqs-20000/COL1A1/reg_10.fasta\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_genome\n",
      "Setting attribute: map_rep_genome\n",
      "Time(s): 136.256362915\n",
      "Dealing with sequence: 1 out of 1\n",
      "Time(s): 7.37229800224\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 1.17576003075\n",
      "---  184\n",
      "--  regions designed: 13\n",
      "--  probes for this region: 3529\n",
      "--  sequences from: region_seqs-20000/CLDN1\n",
      "--  regions designed: 13\n",
      "--  probes for this region: 0\n",
      "--  sequences from: region_seqs-20000/MMP2\n",
      "--  regions designed: 18\n",
      "--  probes for this region: 0\n"
     ]
    }
   ],
   "source": [
    "resolution = 20000;\n",
    "EMT_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT';\n",
    "merged=False\n",
    "reg_nums = Batch_Probe_Design(master_dir=EMT_dir,\n",
    "                         resolution=resolution,\n",
    "                         merge=merged)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "## 3. Design barcoding scheme"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Design_Scheme(master_dir, resolution,\n",
    "                  reg_filename =r'Regions.txt', \n",
    "                  report_dir=r'reports',\n",
    "                  n_color=2,\n",
    "                  continue_num=['bc_gene','bc_common'],\n",
    "                  merge=True, save=True, verbose=True):\n",
    "    '''Function to design barcoding scheme for EMT\n",
    "    Input\n",
    "        master: directory for this library, string\n",
    "        resolution: resolution for parsed directory, if given will change seq_dir and save_dir\n",
    "        report_dir: sub directory for saved probe reports, string\n",
    "        continue_num: the order for numbering barcodes, list\n",
    "        merge: whether sequences are merged, bool\n",
    "        save: whether save reports, bool\n",
    "        verbose: print a lot of things!, bool\n",
    "    Output:\n",
    "        scheme: dictionary to map filename to barcode ids\n",
    "        '''\n",
    "    # imports\n",
    "    if verbose:\n",
    "        print'- 1.Importing packages';\n",
    "    import time,os,sys,glob\n",
    "    sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "    import LibraryDesigner as ld\n",
    "    import cPickle as pickle\n",
    "    # naming related to resolution\n",
    "    if verbose:\n",
    "        print '- 2.Patching Resolution:',resolution;\n",
    "    if resolution: # not \"None\" or 0\n",
    "        _report_dir = report_dir + '-' +str(resolution);\n",
    "    else:\n",
    "        _report_dir = report_dir\n",
    "        \n",
    "    # naming related\n",
    "    if verbose:\n",
    "        print '- 3.Getting folder list if merged:', merge;\n",
    "    if merge:\n",
    "        _report_dir += os.sep + 'merged';\n",
    "        # convert to list for consistency\n",
    "        _report_dirs = [_report_dir]\n",
    "    else:\n",
    "        _report_dirs = []; # initialize\n",
    "        # call all subfolders\n",
    "        _subfolders = next(os.walk(master_dir+os.sep+_report_dir))[1];\n",
    "        for _subfolder in _subfolders:\n",
    "            if not 'merge' in _subfolder:\n",
    "                _report_dirs.append(_report_dir + os.sep + _subfolder);\n",
    "    if verbose:\n",
    "        print \"-- 3.1 Input folders:\\n\", _report_dirs\n",
    " \n",
    "    \n",
    "    def _region_file_to_dic(region_folder=master_dir, reg_filename=reg_filename, _verbose=verbose):\n",
    "        '''Sub-function to read region file'''\n",
    "        # region filename\n",
    "        _reg_filename = region_folder + os.sep + reg_filename;\n",
    "        _reg_file = open(_reg_filename, 'r');    \n",
    "        if _verbose:\n",
    "            print '-- Input region file is: '+ _reg_filename;\n",
    "        # start reading\n",
    "        _lines = _reg_file.read().split('\\n')\n",
    "        _titles = _lines[0].split('\\t');\n",
    "        # save a list of dictionaries\n",
    "        _reg_list = [];\n",
    "        for _line in _lines[1:]:\n",
    "            _reg_dic = {}; # dinctionary to save all informations\n",
    "            _info = _line.split('\\t'); # split informations\n",
    "            if len(_info) != len(_titles): # sanity check to make sure they are of the same size\n",
    "                continue;\n",
    "            for _i in range(len(_info)): # save info to dic\n",
    "                _reg_dic[_titles[_i]] = _info[_i];\n",
    "            _reg_list.append(_reg_dic); # save dic to list\n",
    "        _reg_file.close();\n",
    "        \n",
    "        _gene_dic = {}\n",
    "        for _reg_dic in _reg_list:\n",
    "            _region_str = _reg_dic['Region'];\n",
    "            _chrom = _region_str.split(':')[0];\n",
    "            _locus = _region_str.split(':')[1];\n",
    "            _start, _stop = _locus.split('-')             # grab start and stop positions\n",
    "            _start = int(_start.replace(',', ''));\n",
    "            _stop = int(_stop.replace(',', ''));\n",
    "            \n",
    "            # save info\n",
    "            _tmp_dic={'chrom':_chrom, 'start':_start, 'stop':_stop}\n",
    "            if 'Strand' in _reg_dic:\n",
    "                _tmp_dic['strand'] = _reg_dic['Strand'];\n",
    "            if 'Type' in _reg_dic:\n",
    "                _tmp_dic['type'] = _reg_dic['Type'];\n",
    "            _gene_dic[_reg_dic['Gene']] = _tmp_dic;\n",
    "            \n",
    "        return _gene_dic\n",
    "        \n",
    "    ## Read region file and get a dictionary\n",
    "    gene_dic = _region_file_to_dic()\n",
    "    if verbose:\n",
    "        print '- 4.Reading region file. Genes are:', gene_dic.keys()\n",
    "    \n",
    "    ## design barcode scheme\n",
    "    # initialize\n",
    "    _scheme = {}\n",
    "    _base = {'color':None,'bc_common':None,'bc_gene':None,'bc_unique':None} \n",
    "    if verbose:\n",
    "        print '- 5.Designing barcode scheme.'\n",
    "    # count region number for each gene and assign colors\n",
    "    _num_gene = 0;\n",
    "    _num_unique = 0;\n",
    "    for _gene, _info in sorted(gene_dic.items(), key=lambda (k,v):[v['chrom'],v['start']]):\n",
    "        # search all folders to get a match for this gene\n",
    "        _gene_dir = [_dir for _dir in _report_dirs if (_gene in _dir)]\n",
    "        # if there is a match for this gene\n",
    "        if _gene_dir: \n",
    "            print _gene, _gene_dir\n",
    "            \n",
    "            _pb_reports = glob.glob(master_dir+os.sep+_gene_dir[0]+os.sep+\"*.pbr\")\n",
    "            _num_common = 0;\n",
    "            for _report in sorted(_pb_reports, key=lambda l:int(l.split('.pbr')[0].split('_')[-1])):\n",
    "                print \"--- \", _report\n",
    "                _scheme[_report] = {'color':_num_gene%n_color,\n",
    "                                    'id':_num_unique+1,\n",
    "                                    'gene':_gene,\n",
    "                                    'bc_common':n_color*(int(_report.split('.')[-2].split('_')[-1])-1) + 1 +_num_gene%n_color,\n",
    "                                    'bc_gene':_num_gene,\n",
    "                                    'bc_unique':[_num_unique]*2,\n",
    "                                    'chrom': gene_dic[_gene]['chrom'],\n",
    "                                    'start': gene_dic[_gene]['start'],\n",
    "                                    'stop': gene_dic[_gene]['stop'],}\n",
    "                _num_unique += 1;\n",
    "                _num_common += 1;\n",
    "            \n",
    "            # finishing calculate this gene\n",
    "            _num_gene += 1; # also used later\n",
    "\n",
    "    # re-assign barcodes by continued-numbering:\n",
    "    if continue_num:\n",
    "        if verbose:\n",
    "            print \"- 6.Re-number barcodes by continued numbering\"\n",
    "        for i in range(len(continue_num)-1):\n",
    "            prev_max = -1;\n",
    "            for _k,_v in sorted(_scheme.items()):\n",
    "                if continue_num[i] in _scheme[_k].keys():\n",
    "                    if _scheme[_k][continue_num[i]] > prev_max:\n",
    "                        prev_max = _scheme[_k][continue_num[i]]\n",
    "            print '-- For '+continue_num[i]+', max barcode id is:', prev_max\n",
    "            for _k,_v in sorted(_scheme.items()):\n",
    "                _scheme[_k][continue_num[i+1]] += prev_max \n",
    "        \n",
    "    if save:\n",
    "        pickle.dump(_scheme, open(master_dir+os.sep+'scheme.pkl','w'));\n",
    "    return _scheme    \n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- 1.Importing packages\n",
      "- 2.Patching Resolution: 20000\n",
      "- 3.Getting folder list if merged: False\n",
      "-- 3.1 Input folders:\n",
      "['reports-20000/MMP9', 'reports-20000/KRT18', 'reports-20000/KRT13', 'reports-20000/CDH1-3', 'reports-20000/CDH2', 'reports-20000/ZEB1', 'reports-20000/FN1', 'reports-20000/KRT14', 'reports-20000/MALAT1-NEAT1', 'reports-20000/ZO-1', 'reports-20000/VIM', 'reports-20000/ITGA5', 'reports-20000/ACE2', 'reports-20000/LAMA1', 'reports-20000/DDR2', 'reports-20000/COL1A1', 'reports-20000/CLDN1', 'reports-20000/MMP2']\n",
      "-- Input region file is: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/Regions.txt\n",
      "- 4.Reading region file. Genes are: ['LAMA1', 'MALAT1-NEAT1', 'FN1', 'DDR2', 'ZEB1', 'COL1A1', 'ITGA5', 'VIM', 'KRT14', 'ACE2', 'ZO-1', 'MMP9', 'CLDN1', 'CDH2', 'MMP2', 'CDH1-3', 'KRT18']\n",
      "- 5.Designing barcode scheme.\n",
      "DDR2 ['reports-20000/DDR2']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_14.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_15.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_16.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_17.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_18.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_19.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_20.pbr\n",
      "VIM ['reports-20000/VIM']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_13.pbr\n",
      "ZEB1 ['reports-20000/ZEB1']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_14.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_15.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_16.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_17.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_18.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_19.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_20.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_21.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_22.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_23.pbr\n",
      "MALAT1-NEAT1 ['reports-20000/MALAT1-NEAT1']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_14.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_15.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_16.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_17.pbr\n",
      "KRT18 ['reports-20000/KRT18']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_13.pbr\n",
      "ITGA5 ['reports-20000/ITGA5']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_14.pbr\n",
      "ZO-1 ['reports-20000/ZO-1']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_14.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_15.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_16.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_17.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_18.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_19.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_20.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_21.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_22.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_23.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_24.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_25.pbr\n",
      "MMP2 ['reports-20000/MMP2']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_14.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_15.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_16.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_17.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_18.pbr\n",
      "CDH1-3 ['reports-20000/CDH1-3']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_14.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_15.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_16.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_17.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_18.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_19.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_20.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_21.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_22.pbr\n",
      "KRT14 ['reports-20000/KRT14']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_13.pbr\n",
      "COL1A1 ['reports-20000/COL1A1']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_13.pbr\n",
      "LAMA1 ['reports-20000/LAMA1']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_14.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_15.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_16.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_17.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_18.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_19.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_20.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_21.pbr\n",
      "CDH2 ['reports-20000/CDH2']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_14.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_15.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_16.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_17.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_18.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_19.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_20.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_21.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_22.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_23.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_24.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_25.pbr\n",
      "FN1 ['reports-20000/FN1']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_14.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_15.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_16.pbr\n",
      "MMP9 ['reports-20000/MMP9']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_13.pbr\n",
      "CLDN1 ['reports-20000/CLDN1']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_13.pbr\n",
      "ACE2 ['reports-20000/ACE2']\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_1.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_2.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_3.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_4.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_5.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_6.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_7.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_8.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_9.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_10.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_11.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_12.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_13.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_14.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_15.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_16.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_17.pbr\n",
      "---  /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_18.pbr\n",
      "- 6.Re-number barcodes by continued numbering\n",
      "-- For bc_gene, max barcode id is: 16\n"
     ]
    }
   ],
   "source": [
    "EMT_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT';\n",
    "resolution = 20000;\n",
    "merge=False;\n",
    "barcode_scheme = Design_Scheme(EMT_dir, resolution,\n",
    "                               merge=False,\n",
    "                               n_color=2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_1.pbr': {'bc_common': 17,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [279, 279],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 280,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_10.pbr': {'bc_common': 35,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [288, 288],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 289,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_11.pbr': {'bc_common': 37,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [289, 289],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 290,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_12.pbr': {'bc_common': 39,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [290, 290],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 291,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_13.pbr': {'bc_common': 41,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [291, 291],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 292,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_14.pbr': {'bc_common': 43,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [292, 292],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 293,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_15.pbr': {'bc_common': 45,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [293, 293],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 294,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_16.pbr': {'bc_common': 47,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [294, 294],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 295,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_17.pbr': {'bc_common': 49,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [295, 295],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 296,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_18.pbr': {'bc_common': 51,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [296, 296],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 297,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_2.pbr': {'bc_common': 19,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [280, 280],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 281,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_3.pbr': {'bc_common': 21,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [281, 281],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 282,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_4.pbr': {'bc_common': 23,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [282, 282],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 283,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_5.pbr': {'bc_common': 25,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [283, 283],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 284,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_6.pbr': {'bc_common': 27,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [284, 284],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 285,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_7.pbr': {'bc_common': 29,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [285, 285],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 286,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_8.pbr': {'bc_common': 31,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [286, 286],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 287,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_9.pbr': {'bc_common': 33,\n",
       "  'bc_gene': 16,\n",
       "  'bc_unique': [287, 287],\n",
       "  'chrom': 'chrX',\n",
       "  'color': 0,\n",
       "  'gene': 'ACE2',\n",
       "  'id': 288,\n",
       "  'start': 15494402,\n",
       "  'stop': 15602148},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_1.pbr': {'bc_common': 17,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [143, 143],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 144,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_10.pbr': {'bc_common': 35,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [152, 152],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 153,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_11.pbr': {'bc_common': 37,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [153, 153],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 154,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_12.pbr': {'bc_common': 39,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [154, 154],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 155,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_13.pbr': {'bc_common': 41,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [155, 155],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 156,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_14.pbr': {'bc_common': 43,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [156, 156],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 157,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_15.pbr': {'bc_common': 45,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [157, 157],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 158,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_16.pbr': {'bc_common': 47,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [158, 158],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 159,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_17.pbr': {'bc_common': 49,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [159, 159],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 160,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_18.pbr': {'bc_common': 51,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [160, 160],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 161,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_19.pbr': {'bc_common': 53,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [161, 161],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 162,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_2.pbr': {'bc_common': 19,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [144, 144],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 145,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_20.pbr': {'bc_common': 55,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [162, 162],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 163,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_21.pbr': {'bc_common': 57,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [163, 163],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 164,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_22.pbr': {'bc_common': 59,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [164, 164],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 165,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_3.pbr': {'bc_common': 21,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [145, 145],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 146,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_4.pbr': {'bc_common': 23,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [146, 146],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 147,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_5.pbr': {'bc_common': 25,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [147, 147],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 148,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_6.pbr': {'bc_common': 27,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [148, 148],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 149,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_7.pbr': {'bc_common': 29,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [149, 149],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 150,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_8.pbr': {'bc_common': 31,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [150, 150],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 151,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_9.pbr': {'bc_common': 33,\n",
       "  'bc_gene': 8,\n",
       "  'bc_unique': [151, 151],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH1-3',\n",
       "  'id': 152,\n",
       "  'start': 68636189,\n",
       "  'stop': 68835548},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_1.pbr': {'bc_common': 17,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [212, 212],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 213,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_10.pbr': {'bc_common': 35,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [221, 221],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 222,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_11.pbr': {'bc_common': 37,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [222, 222],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 223,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_12.pbr': {'bc_common': 39,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [223, 223],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 224,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_13.pbr': {'bc_common': 41,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [224, 224],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 225,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_14.pbr': {'bc_common': 43,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [225, 225],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 226,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_15.pbr': {'bc_common': 45,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [226, 226],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 227,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_16.pbr': {'bc_common': 47,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [227, 227],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 228,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_17.pbr': {'bc_common': 49,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [228, 228],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 229,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_18.pbr': {'bc_common': 51,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [229, 229],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 230,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_19.pbr': {'bc_common': 53,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [230, 230],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 231,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_2.pbr': {'bc_common': 19,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [213, 213],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 214,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_20.pbr': {'bc_common': 55,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [231, 231],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 232,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_21.pbr': {'bc_common': 57,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [232, 232],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 233,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_22.pbr': {'bc_common': 59,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [233, 233],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 234,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_23.pbr': {'bc_common': 61,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [234, 234],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 235,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_24.pbr': {'bc_common': 63,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [235, 235],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 236,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_25.pbr': {'bc_common': 65,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [236, 236],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 237,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_3.pbr': {'bc_common': 21,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [214, 214],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 215,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_4.pbr': {'bc_common': 23,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [215, 215],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 216,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_5.pbr': {'bc_common': 25,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [216, 216],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 217,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_6.pbr': {'bc_common': 27,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [217, 217],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 218,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_7.pbr': {'bc_common': 29,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [218, 218],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 219,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_8.pbr': {'bc_common': 31,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [219, 219],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 220,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_9.pbr': {'bc_common': 33,\n",
       "  'bc_gene': 12,\n",
       "  'bc_unique': [220, 220],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 0,\n",
       "  'gene': 'CDH2',\n",
       "  'id': 221,\n",
       "  'start': 27932879,\n",
       "  'stop': 28177481},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_1.pbr': {'bc_common': 18,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [266, 266],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 267,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_10.pbr': {'bc_common': 36,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [275, 275],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 276,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_11.pbr': {'bc_common': 38,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [276, 276],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 277,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_12.pbr': {'bc_common': 40,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [277, 277],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 278,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_13.pbr': {'bc_common': 42,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [278, 278],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 279,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_2.pbr': {'bc_common': 20,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [267, 267],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 268,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_3.pbr': {'bc_common': 22,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [268, 268],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 269,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_4.pbr': {'bc_common': 24,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [269, 269],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 270,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_5.pbr': {'bc_common': 26,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [270, 270],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 271,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_6.pbr': {'bc_common': 28,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [271, 271],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 272,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_7.pbr': {'bc_common': 30,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [272, 272],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 273,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_8.pbr': {'bc_common': 32,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [273, 273],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 274,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_9.pbr': {'bc_common': 34,\n",
       "  'bc_gene': 15,\n",
       "  'bc_unique': [274, 274],\n",
       "  'chrom': 'chr3',\n",
       "  'color': 1,\n",
       "  'gene': 'CLDN1',\n",
       "  'id': 275,\n",
       "  'start': 190305701,\n",
       "  'stop': 190322475},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_1.pbr': {'bc_common': 17,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [178, 178],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 179,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_10.pbr': {'bc_common': 35,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [187, 187],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 188,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_11.pbr': {'bc_common': 37,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [188, 188],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 189,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_12.pbr': {'bc_common': 39,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [189, 189],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 190,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_13.pbr': {'bc_common': 41,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [190, 190],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 191,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_2.pbr': {'bc_common': 19,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [179, 179],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 180,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_3.pbr': {'bc_common': 21,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [180, 180],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 181,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_4.pbr': {'bc_common': 23,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [181, 181],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 182,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_5.pbr': {'bc_common': 25,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [182, 182],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 183,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_6.pbr': {'bc_common': 27,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [183, 183],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 184,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_7.pbr': {'bc_common': 29,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [184, 184],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 185,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_8.pbr': {'bc_common': 31,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [185, 185],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 186,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_9.pbr': {'bc_common': 33,\n",
       "  'bc_gene': 10,\n",
       "  'bc_unique': [186, 186],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 0,\n",
       "  'gene': 'COL1A1',\n",
       "  'id': 187,\n",
       "  'start': 50183289,\n",
       "  'stop': 50201648},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_1.pbr': {'bc_common': 17,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [0, 0],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 1,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_10.pbr': {'bc_common': 35,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [9, 9],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 10,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_11.pbr': {'bc_common': 37,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [10, 10],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 11,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_12.pbr': {'bc_common': 39,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [11, 11],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 12,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_13.pbr': {'bc_common': 41,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [12, 12],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 13,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_14.pbr': {'bc_common': 43,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [13, 13],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 14,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_15.pbr': {'bc_common': 45,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [14, 14],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 15,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_16.pbr': {'bc_common': 47,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [15, 15],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 16,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_17.pbr': {'bc_common': 49,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [16, 16],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 17,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_18.pbr': {'bc_common': 51,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [17, 17],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 18,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_19.pbr': {'bc_common': 53,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [18, 18],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 19,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_2.pbr': {'bc_common': 19,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [1, 1],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 2,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_20.pbr': {'bc_common': 55,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [19, 19],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 20,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_3.pbr': {'bc_common': 21,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [2, 2],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 3,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_4.pbr': {'bc_common': 23,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [3, 3],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 4,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_5.pbr': {'bc_common': 25,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [4, 4],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 5,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_6.pbr': {'bc_common': 27,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [5, 5],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 6,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_7.pbr': {'bc_common': 29,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [6, 6],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 7,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_8.pbr': {'bc_common': 31,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [7, 7],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 8,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_9.pbr': {'bc_common': 33,\n",
       "  'bc_gene': 0,\n",
       "  'bc_unique': [8, 8],\n",
       "  'chrom': 'chr1',\n",
       "  'color': 0,\n",
       "  'gene': 'DDR2',\n",
       "  'id': 9,\n",
       "  'start': 162631265,\n",
       "  'stop': 162787400},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_1.pbr': {'bc_common': 18,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [237, 237],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 238,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_10.pbr': {'bc_common': 36,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [246, 246],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 247,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_11.pbr': {'bc_common': 38,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [247, 247],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 248,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_12.pbr': {'bc_common': 40,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [248, 248],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 249,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_13.pbr': {'bc_common': 42,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [249, 249],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 250,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_14.pbr': {'bc_common': 44,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [250, 250],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 251,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_15.pbr': {'bc_common': 46,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [251, 251],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 252,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_16.pbr': {'bc_common': 48,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [252, 252],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 253,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_2.pbr': {'bc_common': 20,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [238, 238],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 239,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_3.pbr': {'bc_common': 22,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [239, 239],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 240,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_4.pbr': {'bc_common': 24,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [240, 240],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 241,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_5.pbr': {'bc_common': 26,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [241, 241],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 242,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_6.pbr': {'bc_common': 28,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [242, 242],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 243,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_7.pbr': {'bc_common': 30,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [243, 243],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 244,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_8.pbr': {'bc_common': 32,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [244, 244],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 245,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_9.pbr': {'bc_common': 34,\n",
       "  'bc_gene': 13,\n",
       "  'bc_unique': [245, 245],\n",
       "  'chrom': 'chr2',\n",
       "  'color': 1,\n",
       "  'gene': 'FN1',\n",
       "  'id': 246,\n",
       "  'start': 215360440,\n",
       "  'stop': 215436172},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_1.pbr': {'bc_common': 18,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [86, 86],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 87,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_10.pbr': {'bc_common': 36,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [95, 95],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 96,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_11.pbr': {'bc_common': 38,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [96, 96],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 97,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_12.pbr': {'bc_common': 40,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [97, 97],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 98,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_13.pbr': {'bc_common': 42,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [98, 98],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 99,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_14.pbr': {'bc_common': 44,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [99, 99],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 100,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_2.pbr': {'bc_common': 20,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [87, 87],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 88,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_3.pbr': {'bc_common': 22,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [88, 88],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 89,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_4.pbr': {'bc_common': 24,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [89, 89],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 90,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_5.pbr': {'bc_common': 26,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [90, 90],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 91,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_6.pbr': {'bc_common': 28,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [91, 91],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 92,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_7.pbr': {'bc_common': 30,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [92, 92],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 93,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_8.pbr': {'bc_common': 32,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [93, 93],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 94,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_9.pbr': {'bc_common': 34,\n",
       "  'bc_gene': 5,\n",
       "  'bc_unique': [94, 94],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 1,\n",
       "  'gene': 'ITGA5',\n",
       "  'id': 95,\n",
       "  'start': 54395261,\n",
       "  'stop': 54419460},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_1.pbr': {'bc_common': 18,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [165, 165],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 166,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_10.pbr': {'bc_common': 36,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [174, 174],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 175,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_11.pbr': {'bc_common': 38,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [175, 175],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 176,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_12.pbr': {'bc_common': 40,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [176, 176],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 177,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_13.pbr': {'bc_common': 42,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [177, 177],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 178,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_2.pbr': {'bc_common': 20,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [166, 166],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 167,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_3.pbr': {'bc_common': 22,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [167, 167],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 168,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_4.pbr': {'bc_common': 24,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [168, 168],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 169,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_5.pbr': {'bc_common': 26,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [169, 169],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 170,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_6.pbr': {'bc_common': 28,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [170, 170],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 171,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_7.pbr': {'bc_common': 30,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [171, 171],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 172,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_8.pbr': {'bc_common': 32,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [172, 172],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 173,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_9.pbr': {'bc_common': 34,\n",
       "  'bc_gene': 9,\n",
       "  'bc_unique': [173, 173],\n",
       "  'chrom': 'chr17',\n",
       "  'color': 1,\n",
       "  'gene': 'KRT14',\n",
       "  'id': 174,\n",
       "  'start': 41582279,\n",
       "  'stop': 41586921},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_1.pbr': {'bc_common': 17,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [73, 73],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 74,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_10.pbr': {'bc_common': 35,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [82, 82],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 83,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_11.pbr': {'bc_common': 37,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [83, 83],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 84,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_12.pbr': {'bc_common': 39,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [84, 84],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 85,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_13.pbr': {'bc_common': 41,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [85, 85],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 86,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_2.pbr': {'bc_common': 19,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [74, 74],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 75,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_3.pbr': {'bc_common': 21,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [75, 75],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 76,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_4.pbr': {'bc_common': 23,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [76, 76],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 77,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_5.pbr': {'bc_common': 25,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [77, 77],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 78,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_6.pbr': {'bc_common': 27,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [78, 78],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 79,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_7.pbr': {'bc_common': 29,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [79, 79],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 80,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_8.pbr': {'bc_common': 31,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [80, 80],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 81,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_9.pbr': {'bc_common': 33,\n",
       "  'bc_gene': 4,\n",
       "  'bc_unique': [81, 81],\n",
       "  'chrom': 'chr12',\n",
       "  'color': 0,\n",
       "  'gene': 'KRT18',\n",
       "  'id': 82,\n",
       "  'start': 52948871,\n",
       "  'stop': 52952901},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_1.pbr': {'bc_common': 18,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [191, 191],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 192,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_10.pbr': {'bc_common': 36,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [200, 200],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 201,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_11.pbr': {'bc_common': 38,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [201, 201],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 202,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_12.pbr': {'bc_common': 40,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [202, 202],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 203,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_13.pbr': {'bc_common': 42,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [203, 203],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 204,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_14.pbr': {'bc_common': 44,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [204, 204],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 205,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_15.pbr': {'bc_common': 46,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [205, 205],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 206,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_16.pbr': {'bc_common': 48,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [206, 206],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 207,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_17.pbr': {'bc_common': 50,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [207, 207],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 208,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_18.pbr': {'bc_common': 52,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [208, 208],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 209,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_19.pbr': {'bc_common': 54,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [209, 209],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 210,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_2.pbr': {'bc_common': 20,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [192, 192],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 193,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_20.pbr': {'bc_common': 56,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [210, 210],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 211,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_21.pbr': {'bc_common': 58,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [211, 211],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 212,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_3.pbr': {'bc_common': 22,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [193, 193],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 194,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_4.pbr': {'bc_common': 24,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [194, 194],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 195,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_5.pbr': {'bc_common': 26,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [195, 195],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 196,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_6.pbr': {'bc_common': 28,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [196, 196],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 197,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_7.pbr': {'bc_common': 30,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [197, 197],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 198,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_8.pbr': {'bc_common': 32,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [198, 198],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 199,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_9.pbr': {'bc_common': 34,\n",
       "  'bc_gene': 11,\n",
       "  'bc_unique': [199, 199],\n",
       "  'chrom': 'chr18',\n",
       "  'color': 1,\n",
       "  'gene': 'LAMA1',\n",
       "  'id': 200,\n",
       "  'start': 6941744,\n",
       "  'stop': 7117814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_1.pbr': {'bc_common': 18,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [56, 56],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 57,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_10.pbr': {'bc_common': 36,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [65, 65],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 66,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_11.pbr': {'bc_common': 38,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [66, 66],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 67,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_12.pbr': {'bc_common': 40,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [67, 67],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 68,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_13.pbr': {'bc_common': 42,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [68, 68],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 69,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_14.pbr': {'bc_common': 44,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [69, 69],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 70,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_15.pbr': {'bc_common': 46,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [70, 70],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 71,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_16.pbr': {'bc_common': 48,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [71, 71],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 72,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_17.pbr': {'bc_common': 50,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [72, 72],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 73,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_2.pbr': {'bc_common': 20,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [57, 57],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 58,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_3.pbr': {'bc_common': 22,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [58, 58],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 59,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_4.pbr': {'bc_common': 24,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [59, 59],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 60,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_5.pbr': {'bc_common': 26,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [60, 60],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 61,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_6.pbr': {'bc_common': 28,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [61, 61],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 62,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_7.pbr': {'bc_common': 30,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [62, 62],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 63,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_8.pbr': {'bc_common': 32,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [63, 63],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 64,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_9.pbr': {'bc_common': 34,\n",
       "  'bc_gene': 3,\n",
       "  'bc_unique': [64, 64],\n",
       "  'chrom': 'chr11',\n",
       "  'color': 1,\n",
       "  'gene': 'MALAT1-NEAT1',\n",
       "  'id': 65,\n",
       "  'start': 65422774,\n",
       "  'stop': 65506516},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_1.pbr': {'bc_common': 18,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [125, 125],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 126,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_10.pbr': {'bc_common': 36,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [134, 134],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 135,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_11.pbr': {'bc_common': 38,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [135, 135],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 136,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_12.pbr': {'bc_common': 40,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [136, 136],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 137,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_13.pbr': {'bc_common': 42,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [137, 137],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 138,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_14.pbr': {'bc_common': 44,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [138, 138],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 139,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_15.pbr': {'bc_common': 46,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [139, 139],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 140,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_16.pbr': {'bc_common': 48,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [140, 140],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 141,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_17.pbr': {'bc_common': 50,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [141, 141],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 142,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_18.pbr': {'bc_common': 52,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [142, 142],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 143,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_2.pbr': {'bc_common': 20,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [126, 126],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 127,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_3.pbr': {'bc_common': 22,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [127, 127],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 128,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_4.pbr': {'bc_common': 24,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [128, 128],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 129,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_5.pbr': {'bc_common': 26,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [129, 129],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 130,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_6.pbr': {'bc_common': 28,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [130, 130],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 131,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_7.pbr': {'bc_common': 30,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [131, 131],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 132,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_8.pbr': {'bc_common': 32,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [132, 132],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 133,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_9.pbr': {'bc_common': 34,\n",
       "  'bc_gene': 7,\n",
       "  'bc_unique': [133, 133],\n",
       "  'chrom': 'chr16',\n",
       "  'color': 1,\n",
       "  'gene': 'MMP2',\n",
       "  'id': 134,\n",
       "  'start': 55389700,\n",
       "  'stop': 55506691},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_1.pbr': {'bc_common': 17,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [253, 253],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 254,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_10.pbr': {'bc_common': 35,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [262, 262],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 263,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_11.pbr': {'bc_common': 37,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [263, 263],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 264,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_12.pbr': {'bc_common': 39,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [264, 264],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 265,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_13.pbr': {'bc_common': 41,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [265, 265],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 266,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_2.pbr': {'bc_common': 19,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [254, 254],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 255,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_3.pbr': {'bc_common': 21,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [255, 255],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 256,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_4.pbr': {'bc_common': 23,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [256, 256],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 257,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_5.pbr': {'bc_common': 25,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [257, 257],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 258,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_6.pbr': {'bc_common': 27,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [258, 258],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 259,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_7.pbr': {'bc_common': 29,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [259, 259],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 260,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_8.pbr': {'bc_common': 31,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [260, 260],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 261,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_9.pbr': {'bc_common': 33,\n",
       "  'bc_gene': 14,\n",
       "  'bc_unique': [261, 261],\n",
       "  'chrom': 'chr20',\n",
       "  'color': 0,\n",
       "  'gene': 'MMP9',\n",
       "  'id': 262,\n",
       "  'start': 46008908,\n",
       "  'stop': 46016561},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_1.pbr': {'bc_common': 18,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [20, 20],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 21,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_10.pbr': {'bc_common': 36,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [29, 29],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 30,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_11.pbr': {'bc_common': 38,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [30, 30],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 31,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_12.pbr': {'bc_common': 40,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [31, 31],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 32,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_13.pbr': {'bc_common': 42,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [32, 32],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 33,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_2.pbr': {'bc_common': 20,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [21, 21],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 22,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_3.pbr': {'bc_common': 22,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [22, 22],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 23,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_4.pbr': {'bc_common': 24,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [23, 23],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 24,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_5.pbr': {'bc_common': 26,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [24, 24],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 25,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_6.pbr': {'bc_common': 28,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [25, 25],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 26,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_7.pbr': {'bc_common': 30,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [26, 26],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 27,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_8.pbr': {'bc_common': 32,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [27, 27],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 28,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_9.pbr': {'bc_common': 34,\n",
       "  'bc_gene': 1,\n",
       "  'bc_unique': [28, 28],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 1,\n",
       "  'gene': 'VIM',\n",
       "  'id': 29,\n",
       "  'start': 17227935,\n",
       "  'stop': 17237593},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_1.pbr': {'bc_common': 17,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [33, 33],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 34,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_10.pbr': {'bc_common': 35,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [42, 42],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 43,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_11.pbr': {'bc_common': 37,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [43, 43],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 44,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_12.pbr': {'bc_common': 39,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [44, 44],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 45,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_13.pbr': {'bc_common': 41,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [45, 45],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 46,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_14.pbr': {'bc_common': 43,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [46, 46],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 47,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_15.pbr': {'bc_common': 45,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [47, 47],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 48,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_16.pbr': {'bc_common': 47,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [48, 48],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 49,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_17.pbr': {'bc_common': 49,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [49, 49],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 50,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_18.pbr': {'bc_common': 51,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [50, 50],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 51,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_19.pbr': {'bc_common': 53,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [51, 51],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 52,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_2.pbr': {'bc_common': 19,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [34, 34],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 35,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_20.pbr': {'bc_common': 55,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [52, 52],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 53,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_21.pbr': {'bc_common': 57,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [53, 53],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 54,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_22.pbr': {'bc_common': 59,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [54, 54],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 55,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_23.pbr': {'bc_common': 61,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [55, 55],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 56,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_3.pbr': {'bc_common': 21,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [35, 35],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 36,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_4.pbr': {'bc_common': 23,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [36, 36],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 37,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_5.pbr': {'bc_common': 25,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [37, 37],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 38,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_6.pbr': {'bc_common': 27,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [38, 38],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 39,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_7.pbr': {'bc_common': 29,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [39, 39],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 40,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_8.pbr': {'bc_common': 31,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [40, 40],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 41,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_9.pbr': {'bc_common': 33,\n",
       "  'bc_gene': 2,\n",
       "  'bc_unique': [41, 41],\n",
       "  'chrom': 'chr10',\n",
       "  'color': 0,\n",
       "  'gene': 'ZEB1',\n",
       "  'id': 42,\n",
       "  'start': 31318495,\n",
       "  'stop': 31529814},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_1.pbr': {'bc_common': 17,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [100, 100],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 101,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_10.pbr': {'bc_common': 35,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [109, 109],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 110,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_11.pbr': {'bc_common': 37,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [110, 110],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 111,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_12.pbr': {'bc_common': 39,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [111, 111],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 112,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_13.pbr': {'bc_common': 41,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [112, 112],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 113,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_14.pbr': {'bc_common': 43,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [113, 113],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 114,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_15.pbr': {'bc_common': 45,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [114, 114],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 115,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_16.pbr': {'bc_common': 47,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [115, 115],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 116,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_17.pbr': {'bc_common': 49,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [116, 116],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 117,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_18.pbr': {'bc_common': 51,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [117, 117],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 118,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_19.pbr': {'bc_common': 53,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [118, 118],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 119,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_2.pbr': {'bc_common': 19,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [101, 101],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 102,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_20.pbr': {'bc_common': 55,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [119, 119],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 120,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_21.pbr': {'bc_common': 57,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [120, 120],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 121,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_22.pbr': {'bc_common': 59,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [121, 121],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 122,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_23.pbr': {'bc_common': 61,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [122, 122],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 123,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_24.pbr': {'bc_common': 63,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [123, 123],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 124,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_25.pbr': {'bc_common': 65,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [124, 124],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 125,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_3.pbr': {'bc_common': 21,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [102, 102],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 103,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_4.pbr': {'bc_common': 23,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [103, 103],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 104,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_5.pbr': {'bc_common': 25,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [104, 104],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 105,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_6.pbr': {'bc_common': 27,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [105, 105],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 106,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_7.pbr': {'bc_common': 29,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [106, 106],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 107,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_8.pbr': {'bc_common': 31,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [107, 107],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 108,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865},\n",
       " '/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_9.pbr': {'bc_common': 33,\n",
       "  'bc_gene': 6,\n",
       "  'bc_unique': [108, 108],\n",
       "  'chrom': 'chr15',\n",
       "  'color': 0,\n",
       "  'gene': 'ZO-1',\n",
       "  'id': 109,\n",
       "  'start': 29699367,\n",
       "  'stop': 29968865}}"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "barcode_scheme"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "## 4. patch barcodes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.1 imports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# minimal imports for biopython\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.Alphabet import IUPAC\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "import os,glob,time\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.2 import barcodes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Barcodes loaded: Stv: 75, NDB: 1052\n"
     ]
    }
   ],
   "source": [
    "## Read all barcodes\n",
    "barcode_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Barcodes';\n",
    "\n",
    "# read all Stv barcodes\n",
    "#stv_adaptor = [1,2,17,77,78,79,80,81,82,83,84] # barcodes saved for adaptors\n",
    "#stv_bad = [34,38,41] # barcodes performed badly\n",
    "#stv_mask = stv_adaptor + stv_bad \n",
    "stv_mask =[]\n",
    "with open(barcode_dir+os.sep+'top_Stvs.fasta', \"rU\") as handle:\n",
    "    stv_barcodes = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        if int(record.id.split('_')[1]) not in stv_mask:\n",
    "            stv_barcodes.append(record);\n",
    "            \n",
    "# read all NDB barcodes\n",
    "ndb_mask = [];\n",
    "\n",
    "with open(barcode_dir+os.sep+'NDBs.fasta', \"rU\") as handle:\n",
    "    ndb_barcodes = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        if int(record.id.split('_')[1]) not in ndb_mask:\n",
    "            ndb_barcodes.append(record);\n",
    "print \"Barcodes loaded: Stv: \"+str(len(stv_barcodes))+\", NDB: \"+str(len(ndb_barcodes));"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.3 import primers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Primers loaded: forward: 11, reverse: 6\n"
     ]
    }
   ],
   "source": [
    "## Read all primers\n",
    "primer_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Primers';\n",
    "fwd_primer_filename = 'forward_primers_keep.fasta';\n",
    "rev_primer_filename = 'reverse_primers_keep.fasta';\n",
    "\n",
    "# read all forward primers\n",
    "with open(primer_dir+os.sep+fwd_primer_filename, \"rU\") as handle:\n",
    "    fwd_primers = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        fwd_primers.append(record);\n",
    "# read all forward primers\n",
    "with open(primer_dir+os.sep+rev_primer_filename, \"rU\") as handle:\n",
    "    rev_primers = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        rev_primers.append(record);\n",
    "print \"Primers loaded: forward: \"+str(len(fwd_primers))+\", reverse: \"+str(len(rev_primers));\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.4 define used parameters"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- forward primer: ID: W1A07_primer_6\n",
      "Name: W1A07_primer_6\n",
      "Description: W1A07_primer_6\n",
      "Number of features: 0\n",
      "Seq('CGCAAACTGGTGCGGAAGGC', SingleLetterAlphabet())\n",
      "- reverse primer: ID: W1A12_primer_11\n",
      "Name: W1A12_primer_11\n",
      "Description: W1A12_primer_11\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCCATTGCCCGCGAGGTCGAG', SingleLetterAlphabet())\n"
     ]
    }
   ],
   "source": [
    "## Parameters used for patch barcodes & primers\n",
    "# dir\n",
    "master_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT';\n",
    "# barcodes\n",
    "barcode_source = {'bc_gene':'stv',\n",
    "                  'bc_common': 'stv',\n",
    "                  'bc_unique':'ndb'};\n",
    "# primers\n",
    "fprimer = fwd_primers[3];\n",
    "print '- forward primer:', fprimer\n",
    "rprimer = rev_primers[3];\n",
    "print '- reverse primer:', rprimer\n",
    "# dic for region -> tad\n",
    "if not 'barcode_scheme' in vars():\n",
    "    import cPickle as pickle\n",
    "    print '- loading barcode scheme'\n",
    "    barcode_scheme = pickle.load(open(master_dir+os.sep+'scheme.pkl','r'))\n",
    "if not 'pb_records' in vars():\n",
    "    pb_dir = r'final_probes'\n",
    "    print '- loading all probes'\n",
    "    with open(master_dir+os.sep+pb_dir+os.sep+'candidate_probes.fasta', \"rU\") as handle:\n",
    "        pb_records = [];\n",
    "        for record in SeqIO.parse(handle, \"fasta\"):\n",
    "            pb_records.append(record);\n",
    "if not 'pb_lists' in vars():\n",
    "    print '- loading pb_lists'\n",
    "    pb_lists = pickle.load(open(master_dir+os.sep+pb_dir+os.sep+'list.pkl', \"rU\"))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.5 Patch barcodes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Patch_Barcodes(barcode_scheme, master_dir,\n",
    "                   fwd_primer,rev_primer,\n",
    "                   barcode_source, stv_barcodes, ndb_barcodes, barcode_starts={'stv':1,'ndb':1},\n",
    "                   save_folder=r'final_probes',\n",
    "                   add_rand_gap=0,\n",
    "                   save=True, verbose=True):\n",
    "    '''Function to patch barcodes to designed targeting probes\n",
    "    Inputs:\n",
    "        barcode_scheme: encoding scheme for the barcode, dictionary(filepath->barcode)\n",
    "        master_dir: master directory for this library, string\n",
    "        fwd_primer: forward primer,20mer, biopython SeqRecord\n",
    "        rev_primer: reverse primer,40mer(rc), last 20mer-rc should be used\n",
    "        barcode_source: dictionary to determine the source of barcodes, dictionary\n",
    "        stv_barcodes: old barcodes,30mer, biopython SeqRecord list\n",
    "        ndb_barcodes: new barcodes,30mer, biopython SeqRecord list\n",
    "        barcode_starts: id of the first unused barcode, dictionary\n",
    "        save_folder: sub-directory for save files, string\n",
    "        add_rand_gap: whether adding (or length) of random gaps between barcodes, int\n",
    "        save: whether save, bool\n",
    "        verbose: whether say something, bool\n",
    "    Outputs:\n",
    "        total library SeqRecord\n",
    "        '''\n",
    "    ## minimal imports\n",
    "    if verbose:\n",
    "        print \"- Importing packages.\"\n",
    "    from Bio import SeqIO\n",
    "    from Bio.Seq import Seq\n",
    "    from Bio.Alphabet import IUPAC\n",
    "    from Bio.SeqRecord import SeqRecord \n",
    "    import numpy as np;\n",
    "    import glob, os, sys, time\n",
    "    import LibraryDesigner as ld\n",
    "    \n",
    "    ## check inputs:\n",
    "    if verbose:\n",
    "        print \"- Check inputs\"\n",
    "    # check barcode_source\n",
    "    barcode_types = barcode_scheme.values()[0].keys();\n",
    "    for k, v in barcode_source.iteritems():\n",
    "        if k not in barcode_types:\n",
    "            raise ValueError('wrong barcode_source input!');\n",
    "    \n",
    "    ## prepare barcodes\n",
    "    # filter stv_barcodes and ndb_barcodes\n",
    "    if verbose:\n",
    "        print \"- check barcode starts: \", barcode_starts\n",
    "    _stv_barcodes, _ndb_barcodes = [],[];\n",
    "    for record in stv_barcodes:\n",
    "        if not int(record.id.split('_')[1]) < barcode_starts['stv']:\n",
    "            _stv_barcodes.append(record)\n",
    "    for record in ndb_barcodes:\n",
    "        if not int(record.id.split('_')[1]) < barcode_starts['ndb']:\n",
    "            _ndb_barcodes.append(record)\n",
    "\n",
    "            \n",
    "    def _patch_barcode_per_file(_file, _file_encodings, \n",
    "                                _fwd_primer=fwd_primer, _rev_primer=rev_primer,\n",
    "                                _barcode_source=barcode_source, _stv_barcodes=_stv_barcodes, _ndb_barcodes=_ndb_barcodes,\n",
    "                                _add_rand_gap=add_rand_gap, _verbose=verbose):\n",
    "        from random import choice\n",
    "        import os\n",
    "        if _verbose:\n",
    "            print \"-- patch barcodes for:\", _file\n",
    "        # load probe report\n",
    "        _pb = ld.pb_reports_class()\n",
    "        _pb.load_pbr(_file)\n",
    "        \n",
    "        # extract encoding info:\n",
    "        _encoding = _file_encodings[_file];\n",
    "        \n",
    "        # initialize, save all infos here\n",
    "        _plist = [];\n",
    "        _precords = [];\n",
    "        for _info in _pb.pb_reports_keep.values():\n",
    "            _tmp_info = _info.copy();\n",
    "\n",
    "            # extract all encoding info from reg_encodings\n",
    "            _tmp_info['reg_index'] = _encoding['id']\n",
    "            _tmp_info['color'] = _encoding['color']\n",
    "            if 'gene' in _encoding.keys():\n",
    "                _tmp_info['gene'] = _encoding['gene']\n",
    "\n",
    "            # extract barcode info\n",
    "            _islist = False; # variable used for later design\n",
    "            for _k,_v in _barcode_source.iteritems():\n",
    "                if isinstance(_encoding[_k], list):\n",
    "                    _islist = _k; # variable used for later design\n",
    "                    _bcs = [];\n",
    "                    for _bid in _encoding[_k]:\n",
    "                        if _v == 'stv':\n",
    "                            _bcs.append(_stv_barcodes[_bid]);\n",
    "                        elif _v == 'ndb':\n",
    "                            _bcs.append(_ndb_barcodes[_bid]);\n",
    "                    _tmp_info[_k] = _bcs;\n",
    "                else:\n",
    "                    if _v == 'stv':\n",
    "                        _tmp_info[_k] =_stv_barcodes[_encoding[_k]];\n",
    "                    elif _v == 'ndb':\n",
    "                        _tmp_info[_k] =_ndb_barcodes[_encoding[_k]];\n",
    "            # extract primer info:\n",
    "            _tmp_info['fwd_primer'] = _fwd_primer;\n",
    "            _tmp_info['rev_primer'] = _rev_primer;\n",
    "\n",
    "            ## generate_whole sequence\n",
    "            # fwd_primer(20)\n",
    "            # barcode 1 [from list, 1], (reverse-complement of last 20)\n",
    "            # barcode 2, (reverse-complement of last 20)\n",
    "            # target sequence\n",
    "            # barcode 3, (reverse-complement of last 20)\n",
    "            # barcode 4 [from list, 1], (reverse-complement of last 20)\n",
    "            # rev_primer, (reverse-complement of last 20)\n",
    "            _seq_list = []; # start\n",
    "            _seq_list.append(_tmp_info['fwd_primer'].seq) # fwd primer\n",
    "            if _islist:\n",
    "                _seq_list += [_bc.seq[-20:].reverse_complement() for _bc in _tmp_info[_islist]]; # list barcodes, usually for decoding\n",
    "                for _k,_v in _barcode_source.iteritems():\n",
    "                    if _k != _islist:\n",
    "                        _seq_list.insert(-1, _tmp_info[_k].seq[-20:].reverse_complement()) # other barcodes\n",
    "                _seq_list.insert(-2, Seq(_tmp_info['seq']) ) # target sequence in the middle\n",
    "            else:\n",
    "                for _k,_v in _barcode_source.iteritems():\n",
    "                    _seq_list.append(_tmp_info[_k].seq[-20:].reverse_complement()) # other barcodes\n",
    "                _seq_list.insert(-2, Seq(_tmp_info['seq']) ) # target sequence in the middle\n",
    "\n",
    "            _seq_list.append(_tmp_info['rev_primer'].seq[-20:].reverse_complement()) # reverse primer\n",
    "            # result\n",
    "            dna_alphabet = ['A','A','C','G','T','T']; # used for adding random gap, if needed\n",
    "            _total_seq = Seq('');\n",
    "            for j in range(len(_seq_list)):\n",
    "                _seq = _seq_list[j]\n",
    "                _total_seq += _seq;\n",
    "                if j > 0 and j < len(_seq_list)-2:\n",
    "                    _total_seq += ''.join([choice(dna_alphabet) for i in range(_add_rand_gap)]);\n",
    "            _tmp_info['total_seq'] = _total_seq;\n",
    "\n",
    "            ## Generate total_name:\n",
    "            # chr21:10350001-10400001_reg_208_gene_chr21_pb_41577 (from base name)\n",
    "            # primer_[4,11]\n",
    "            # barcodes_75,109,[]\n",
    "            # color_0\n",
    "            \n",
    "            # base name\n",
    "            _total_name = _tmp_info['name'].split('reg_')[0] + 'reg_'+str(_tmp_info['reg_index']);\n",
    "            if 'gene' in _tmp_info['name']:\n",
    "                _total_name += '_gene' + _tmp_info['name'].split('gene')[1]\n",
    "            elif 'gene' in _tmp_info.keys():\n",
    "                _total_name += '_gene_'+_tmp_info['gene'] + '_pb'+_tmp_info['name'].split('pb')[1];\n",
    "            # primer name\n",
    "            _primer_sets = [int(_tmp_info['fwd_primer'].id.split('_')[-1]), int(_tmp_info['rev_primer'].id.split('_')[-1])]\n",
    "            _total_name += '_primer_'+str(_primer_sets).replace(' ','')\n",
    "            # barcode name\n",
    "            _barcode_sets = [];\n",
    "            if _islist:\n",
    "                _barcode_sets.append([rec.id for rec in _tmp_info[_islist]]);\n",
    "                for _k,_v in _barcode_source.iteritems():\n",
    "                    if _k != _islist:\n",
    "                        _barcode_sets.append(_tmp_info[_k].id);\n",
    "            else:\n",
    "                for _k,_v in _barcode_source.iteritems():\n",
    "                    _barcode_sets.append(_tmp_info[_k].id);        \n",
    "            _total_name += '_barcodes_'+str(_barcode_sets).replace(' ','')\n",
    "            # color\n",
    "            _total_name += '_color_'+str(_tmp_info['color'])\n",
    "            \n",
    "            ## save\n",
    "            _tmp_info['total_name'] = _total_name;\n",
    "            ## Append\n",
    "            _plist.append(_tmp_info) # to plist\n",
    "            _precords.append(SeqRecord(_total_seq, id=_total_name, description='', name=_total_name)); # to seq record\n",
    "\n",
    "        return _plist, _precords            \n",
    "    \n",
    "    # initialize\n",
    "    _pb_lists, _pb_records = [],[];\n",
    "    # loop through all files\n",
    "    for _fl,_info in sorted(barcode_scheme.items()):\n",
    "        _list, _records = _patch_barcode_per_file(_fl, barcode_scheme);\n",
    "        # store info here \n",
    "        _pb_lists.append(_list);\n",
    "        _pb_records += _records\n",
    "    \n",
    "    ## save:\n",
    "    if save:\n",
    "        import cPickle as pickle\n",
    "        if not os.path.exists(master_dir + os.sep + save_folder):\n",
    "            os.makedirs(master_dir + os.sep + save_folder)\n",
    "        list_savefile = master_dir + os.sep + save_folder + os.sep + 'list.pkl';\n",
    "        pb_savefile = master_dir + os.sep + save_folder + os.sep + 'candidate_probes.fasta';\n",
    "        if verbose:\n",
    "            print \"- Saving list to:\", list_savefile\n",
    "        pickle.dump(_pb_lists, open(list_savefile,'w'));\n",
    "        if verbose:\n",
    "            print \"- Saving probes to:\", pb_savefile\n",
    "        with open(pb_savefile, 'w') as output_handle:\n",
    "            SeqIO.write(_pb_records, output_handle, 'fasta');\n",
    "    \n",
    "    return _pb_lists, _pb_records;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Importing packages.\n",
      "- Check inputs\n",
      "- check barcode starts:  {'stv': 1, 'ndb': 1}\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_15.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_16.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_17.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_18.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ACE2/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_15.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_16.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_17.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_18.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_19.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_20.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_21.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_22.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH1-3/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_15.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_16.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_17.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_18.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_19.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_20.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_21.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_22.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_23.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_24.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_25.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CDH2/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_4.pbr\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/CLDN1/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/COL1A1/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_15.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_16.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_17.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_18.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_19.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_20.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/DDR2/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_15.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_16.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/FN1/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ITGA5/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_2.pbr\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT14/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/KRT18/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_15.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_16.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_17.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_18.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_19.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_20.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_21.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/LAMA1/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_15.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_16.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_17.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MALAT1-NEAT1/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_15.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_16.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_17.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_18.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_6.pbr\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP2/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/MMP9/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/VIM/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_15.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_16.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_17.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_18.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_19.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_20.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_21.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_22.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_23.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_7.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZEB1/reg_9.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_1.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_10.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_11.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_12.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_13.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_14.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_15.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_16.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_17.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_18.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_19.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_2.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_20.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_21.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_22.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_23.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_24.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_25.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_3.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_4.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_5.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_6.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_7.pbr\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_8.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_9.pbr\n",
      "- Saving list to: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/final_probes/list.pkl\n",
      "- Saving probes to: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/final_probes/candidate_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "pb_lists, pb_records = Patch_Barcodes(barcode_scheme=barcode_scheme, \n",
    "                                     master_dir=master_dir, \n",
    "                                     fwd_primer=fprimer,\n",
    "                                     rev_primer=rprimer,\n",
    "                                     barcode_source=barcode_source,\n",
    "                                     stv_barcodes=stv_barcodes,\n",
    "                                     ndb_barcodes=ndb_barcodes,\n",
    "                                     add_rand_gap=0)    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 5. Check Probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- forward primer: ID: W1A07_primer_6\n",
      "Name: W1A07_primer_6\n",
      "Description: W1A07_primer_6\n",
      "Number of features: 0\n",
      "Seq('CGCAAACTGGTGCGGAAGGC', SingleLetterAlphabet())\n",
      "- reverse primer: ID: W1A12_primer_11\n",
      "Name: W1A12_primer_11\n",
      "Description: W1A12_primer_11\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCCATTGCCCGCGAGGTCGAG', SingleLetterAlphabet())\n"
     ]
    }
   ],
   "source": [
    "## Parameters used for patch barcodes & primers\n",
    "# dir\n",
    "master_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT';\n",
    "# barcodes\n",
    "barcode_source = {'bc_gene':'stv',\n",
    "                  'bc_common': 'stv',\n",
    "                  'bc_unique':'ndb'};\n",
    "# primers\n",
    "fprimer = fwd_primers[3];\n",
    "print '- forward primer:', fprimer\n",
    "rprimer = rev_primers[3];\n",
    "print '- reverse primer:', rprimer\n",
    "# dic for region -> tad\n",
    "if not 'barcode_scheme' in vars():\n",
    "    import cPickle as pickle\n",
    "    print 'loading barcode scheme'\n",
    "    barcode_scheme = pickle.load(open(master_dir+os.sep+'scheme.pkl','r'))\n",
    "if not 'pb_records' in vars():\n",
    "    pb_dir = r'final_probes'\n",
    "    print 'loading all probes'\n",
    "    with open(master_dir+os.sep+pb_dir+os.sep+'candidate_probes.fasta', \"rU\") as handle:\n",
    "        pb_records = [];\n",
    "        for record in SeqIO.parse(handle, \"fasta\"):\n",
    "            pb_records.append(record);\n",
    "if not 'pb_lists' in vars():\n",
    "    print '- loading pb_lists'\n",
    "    pb_lists = pickle.load(open(master_dir+os.sep+pb_dir+os.sep+'list.pkl', \"rU\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Check_Probes(pb_records, pb_lists, reg_encodings, master_dir, \n",
    "                 fwd_primer,rev_primer,\n",
    "                 barcode_source, stv_barcodes, ndb_barcodes, barcode_starts={'stv':1,'ndb':1},\n",
    "                 report_folder=r'reports/centered_merged',save_folder=r'final_probes',\n",
    "                 add_rand_gap=0, total_bc=4, barcode_len=20, target_len=42,  \n",
    "                 word_size=17, max_internal_hits=5, max_genome_hits=200,\n",
    "                 index_dir=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Indeces/human/hg38',\n",
    "                 save=True, verbose=True):\n",
    "    # imports\n",
    "    import os,glob,sys\n",
    "    sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "    from LibraryConstruction import fastaread,fastawrite,fastacombine\n",
    "    import LibraryDesigner as ld\n",
    "    import numpy as np\n",
    "    \n",
    "    def _check_primer_usage(pb_records=pb_records, fwd_primer=fwd_primer, rev_primer=rev_primer,\n",
    "                            _verbose=verbose):\n",
    "        '''Check whether forward or reverse primer are used in all probes'''\n",
    "        if _verbose:\n",
    "            print \"-- Checking primer usage, total probes:\", len(pb_records)\n",
    "        fwd_len = len(fwd_primer.seq);\n",
    "        rev_len = len(rev_primer.seq[-20:].reverse_complement());\n",
    "        \n",
    "        for record in pb_records:\n",
    "            if record.seq[:fwd_len] != fwd_primer.seq:\n",
    "                if _verbose:\n",
    "                    print \"--- Forward primer incorrect!\"\n",
    "                return False\n",
    "            if record.seq[-rev_len:] != rev_primer.seq[-20:].reverse_complement():\n",
    "                if _verbose:\n",
    "                    print \"--- Forward primer incorrect!\"\n",
    "                return False\n",
    "        return True # if no error applies\n",
    "    \n",
    "    def _check_region_size(pb_records=pb_records, pb_lists=pb_lists):\n",
    "        '''Generate a dirctionary '''\n",
    "        # get original region size\n",
    "        _reg_size_dic = {}\n",
    "        for lst in pb_lists:\n",
    "            _reg_size_dic[lst[0]['reg_index']] = len(lst);\n",
    "        # get region size from probe names\n",
    "        _size_from_rec = {}\n",
    "        for record in pb_records:\n",
    "            reg_id = int(record.id.split('_reg_')[1].split('_')[0]);\n",
    "            if reg_id not in _size_from_rec.keys():\n",
    "                _size_from_rec[reg_id] = 1; # if not in key, create\n",
    "            else:\n",
    "                _size_from_rec[reg_id] += 1; # otherwise, add count\n",
    "        # compare\n",
    "        _match = True;\n",
    "        for k,v in sorted(_size_from_rec.items()):\n",
    "            if k not in _reg_size_dic.keys():\n",
    "                print \"region list and region id in probes not match for\", k\n",
    "                _match = False\n",
    "                break\n",
    "            else:\n",
    "                if v != _reg_size_dic[k]:\n",
    "                    print \"region size doesn't match for:\", k\n",
    "                    _match = False\n",
    "                    break\n",
    "        \n",
    "        return _reg_size_dic, _match;\n",
    "    \n",
    "    \n",
    "    def _check_region_to_barcode(pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes,\n",
    "                                 total_bc=total_bc):\n",
    "        '''Generate map from region id to barcodes used in this region'''\n",
    "        import re\n",
    "        _reg_to_barcode = {}\n",
    "        for record in pb_records:\n",
    "            # region id\n",
    "            reg_id = int(record.id.split('_reg_')[1].split('_')[0]);\n",
    "            if reg_id not in _reg_to_barcode.keys():\n",
    "                # barcode ids\n",
    "                stv_matches = re.findall('\\'Stv_(.+?)\\'', record.id, re.DOTALL)\n",
    "                ndb_matches = re.findall('\\'NDB_(.+?)\\'', record.id, re.DOTALL)\n",
    "                stv_names = ['Stv_'+str(stv_id) for stv_id in stv_matches]\n",
    "                ndb_names = ['NDB_'+str(ndb_id) for ndb_id in ndb_matches]\n",
    "                _reg_to_barcode[reg_id] = stv_names+ndb_names\n",
    "        \n",
    "        ## barcode check\n",
    "        _barcode_check = True;\n",
    "        # barcode names\n",
    "        bc_names = [stv.id for stv in stv_barcodes] + [ndb.id for ndb in ndb_barcodes]\n",
    "        # search through previous dictionary\n",
    "        for reg,bcs in sorted(_reg_to_barcode.items()):\n",
    "            for bc in bcs:\n",
    "                if len(bcs) != total_bc:\n",
    "                    print \"-- Error in barcode number for region:\", reg\n",
    "                    _barcode_check = False\n",
    "                    break\n",
    "                if bc not in bc_names:\n",
    "                    print \"-- Wrong barcode name for barcode: \"+str(bc)+\", region: \"+str(reg)\n",
    "                    _barcode_check = False\n",
    "                    break\n",
    "        \n",
    "        return _reg_to_barcode, _barcode_check;\n",
    "        \n",
    "    def _parsing_probe_sequence(record, fwd_primer=fwd_primer, rev_primer=rev_primer,\n",
    "                                add_rand_gap=add_rand_gap, barcode_len=barcode_len, target_len=target_len):\n",
    "        '''parse a probe sequence to acquire all barcode binding sites'''\n",
    "        # take in a seq record, parse the sequence and return a list of all included barcodes (20mer,RC)\n",
    "        barcode_list = [];\n",
    "        _main_seq = record.seq[len(fwd_primer.seq):-20];\n",
    "        \n",
    "        \n",
    "        # trim last 2 barcodes\n",
    "        for i in range(2):\n",
    "            barcode_list.append(_main_seq[-barcode_len:]);\n",
    "            _main_seq = _main_seq[:-(barcode_len+add_rand_gap)];\n",
    "        # trim all barcodes from the beginning\n",
    "        while len(_main_seq) > target_len:\n",
    "            barcode_list.append(_main_seq[:barcode_len]);\n",
    "            _main_seq = _main_seq[(barcode_len+add_rand_gap):];\n",
    "        \n",
    "        return barcode_list;\n",
    "    \n",
    "    def _finding_barcode_name(barcode_list, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes, \n",
    "                              barcode_len=barcode_len, total_bc=total_bc):\n",
    "        '''Given barcode list generated by parsing probe, return a list of barcode names'''\n",
    "        _name_list = [];\n",
    "        for bc_site in barcode_list:\n",
    "            for bc in stv_barcodes+ndb_barcodes:\n",
    "                if bc.seq[-barcode_len:] == bc_site.reverse_complement():\n",
    "                    _name_list.append(bc.id);\n",
    "                    break;\n",
    "        \n",
    "        if len(_name_list) < total_bc:\n",
    "            print \"-- Failed in finding some barcodes.\"\n",
    "            return False\n",
    "        return _name_list;\n",
    "    \n",
    "    def _check_barcode_to_region(reg_to_barcode, \n",
    "                                 pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes):\n",
    "        '''Generate map from barcode id to region id'''\n",
    "        _barcode_to_reg = {}\n",
    "        _reg_id_exists = []\n",
    "        for record in pb_records:\n",
    "            reg_id = int(record.id.split('_reg_')[1].split('_')[0]);\n",
    "            if reg_id in _reg_id_exists:\n",
    "                continue;\n",
    "            else:\n",
    "                _barcode_list = _parsing_probe_sequence(record)\n",
    "                _name_list = _finding_barcode_name(_barcode_list)\n",
    "                for _n in _name_list:\n",
    "                    if _n not in _barcode_to_reg.keys(): # create if not in dic\n",
    "                        _barcode_to_reg[_n] = [reg_id]\n",
    "                    else: # otherwise, append\n",
    "                        _barcode_to_reg[_n].append(reg_id)\n",
    "            _reg_id_exists.append(reg_id)\n",
    "        ## check region distribution\n",
    "        # invert dic from reg_to_barcode\n",
    "        _inv_dic = {}\n",
    "        for reg,bcs in sorted(reg_to_barcode.items()):\n",
    "            for bc in bcs:\n",
    "                if bc not in _inv_dic.keys():\n",
    "                    _inv_dic[bc] = [reg];\n",
    "                else:\n",
    "                    _inv_dic[bc].append(reg);\n",
    "        # compare\n",
    "        _region_check=True\n",
    "        for bc, regs in sorted(_inv_dic.items()):\n",
    "            if bc not in _barcode_to_reg.keys():\n",
    "                print \"-- \"+str(bc)+\" not in barcode_to_region dic!\"\n",
    "                _region_check = False\n",
    "                break\n",
    "            else:\n",
    "                if sorted(regs) != sorted(_barcode_to_reg[bc]):\n",
    "                    print \"-- \"+str(bc)+\" and region\"+str(regs)+\" not compatible with barcode_to_region dic!\"\n",
    "                    _region_check = False\n",
    "                    break\n",
    "                    \n",
    "        return _barcode_to_reg, _region_check\n",
    "    \n",
    "    def _check_barcode_to_color(pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes, \n",
    "                                stv_color=True, ndb_color=False,\n",
    "                                _save=save, master_dir=master_dir, save_folder=save_folder):\n",
    "        '''If multi_color is applied, generate a barcode_to_color dic for adaptor design'''\n",
    "        if 'color' not in str(pb_records[0].id):\n",
    "            print \"-- color check not applied\";\n",
    "            return False\n",
    "        elif not stv_color and not ndb_color:\n",
    "            print \"-- color check turned off in both stv and ndb\";\n",
    "            return False\n",
    "        else:\n",
    "            # get barcodes\n",
    "            _barcode_names = []\n",
    "            if stv_color: # if stv has multi-color\n",
    "                _barcode_names += [bc.id for bc in stv_barcodes];\n",
    "            if ndb_color: # if ndb has multi-color\n",
    "                _barcode_names += [bc.id for bc in ndb_barcodes];\n",
    "            # initialize color dic\n",
    "            _barcode_to_color = {};\n",
    "            _exist_regs = [];\n",
    "            # search through all probes\n",
    "            for record in pb_records:\n",
    "                _reg_id = int(record.id.split('_reg_')[1].split('_')[0]); \n",
    "                if _reg_id in _exist_regs:\n",
    "                    continue\n",
    "                else: \n",
    "                    _exist_regs.append(_reg_id);\n",
    "                _color = int(str(record.id).split('color_')[1])\n",
    "                _barcode_list = _parsing_probe_sequence(record)\n",
    "                _name_list = _finding_barcode_name(_barcode_list)\n",
    "                \n",
    "                for _name in _name_list:\n",
    "                    if _name in _barcode_names:\n",
    "                        if _name not in _barcode_to_color.keys():\n",
    "                            _barcode_to_color[_name] = [_color]\n",
    "                        else:\n",
    "                            _barcode_to_color[_name].append(_color);\n",
    "            # keep the unique colors\n",
    "            _barcode_to_unique_color = {}\n",
    "            for k,v in sorted(_barcode_to_color.items()):\n",
    "                _barcode_to_unique_color[k] = np.unique(v)\n",
    "            if _save:\n",
    "                import csv\n",
    "                with open(master_dir+os.sep+save_folder+os.sep+'color-usage.csv','w') as output_handle:\n",
    "                    fieldnames = ['barcode', 'color']\n",
    "                    writer = csv.DictWriter(output_handle, fieldnames=fieldnames)\n",
    "                    writer.writeheader()\n",
    "                    for _barcode, _color in sorted(_barcode_to_unique_color.items(), key=lambda (k,v):int(k.split('_')[1])):\n",
    "                        writer.writerow({'barcode': _barcode, 'color': _color})\n",
    "                \n",
    "        return _barcode_to_unique_color\n",
    "                            \n",
    "    \n",
    "    def _construct_internal_map(master_dir=master_dir, word_size=word_size):\n",
    "        '''Using functions in LibraryDesign, compute an internal khmer map'''\n",
    "        _int_map = khmer.Countgraph(word_size, 1e9, 2) \n",
    "        _int_map.set_use_bigcount(True)\n",
    "        _nms,_seqs = fastaread(master_dir+os.sep+'final_probes'+os.sep+'candidate_probes.fasta')\n",
    "        for _seq in _seqs:\n",
    "            _int_map.consume(_seq.upper())\n",
    "        return _int_map\n",
    "    \n",
    "    def _check_barcode_in_probes(barcode_to_reg, reg_size_dic, int_map, \n",
    "                                 stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes,\n",
    "                                 barcode_len=barcode_len, max_internal_hits=max_internal_hits):\n",
    "        '''Check barcode appearance in probes, whether that match barcode_to_region scheme'''\n",
    "        _barcode_in_probes = {}\n",
    "        for bc_name, regs in sorted(barcode_to_reg.items()):\n",
    "            bc = None\n",
    "            for _bc in stv_barcodes+ndb_barcodes:\n",
    "                if bc_name == _bc.id:\n",
    "                    bc = _bc\n",
    "                    break\n",
    "            bc_hits = int_map.get_kmer_counts( str(bc.seq[-barcode_len:].reverse_complement()).upper());\n",
    "            if max(bc_hits) - min(bc_hits) > max_internal_hits:\n",
    "                print \"-- Barcode: \"+str(bc)+\" has more off-target in different part of itself!\"\n",
    "                return False\n",
    "            else:\n",
    "                regs,reg_cts = np.unique(regs, return_counts=True);\n",
    "                bc_in_probe = 0;\n",
    "                for reg,ct in zip(regs,reg_cts):\n",
    "                    bc_in_probe += reg_size_dic[reg] * ct;\n",
    "                if max(bc_hits) - bc_in_probe > max_internal_hits:\n",
    "                    print \"-- Barcode: \"+str(bc)+\" has more off-target than threshold!\"\n",
    "                    return False\n",
    "            _barcode_in_probes[bc_name] = bc_in_probe;\n",
    "        return _barcode_in_probes, True\n",
    "    \n",
    "    def _check_between_probes(int_map, pb_lists=pb_lists, pb_records=pb_records):\n",
    "        pass \n",
    "    \n",
    "    def _check_against_genome(pb_records=pb_records, max_genome_hits=max_genome_hits, index_dir=index_dir):\n",
    "        '''Use Khmer to compare probe against genome'''\n",
    "        hg38 = khmer.load_countgraph(index_dir+os.sep+'full_word17_.kmer')\n",
    "        _failed_num = 0;\n",
    "        _keep_pb_records = [];\n",
    "        for record in pb_records:\n",
    "            _kmer_hits = hg38.get_kmer_counts(str(record.seq).upper());\n",
    "            if sum(_kmer_hits) > max_genome_hits:\n",
    "                print '-- Max_genome_hits is: '+str(max_genome_hits)+\", this seq got hits: \"+ str(sum(_kmer_hits))\n",
    "                _failed_num += 1;\n",
    "            else:\n",
    "                _keep_pb_records.append(record);\n",
    "                \n",
    "        return _keep_pb_records, _failed_num # if nothing goes wrong\n",
    "    \n",
    "    def _plot_info():\n",
    "        pass\n",
    "            \n",
    "    ## check primers\n",
    "    primer_usage = _check_primer_usage()\n",
    "    if verbose:\n",
    "        print \"\\n- 1.Passing primer usage check? -\", primer_usage\n",
    "    \n",
    "    ## check region size\n",
    "    reg_size_dic, size_match = _check_region_size()\n",
    "    if verbose:\n",
    "        print \"\\n- 2.Passing region size check? -\", size_match    \n",
    "        for k,v in sorted(reg_size_dic.items()):\n",
    "            print k,':',v\n",
    "        \n",
    "    ## check region to barcode\n",
    "    reg_to_barcode, reg2bc = _check_region_to_barcode()\n",
    "    if verbose:\n",
    "        print \"\\n- 3.Passing region to barcode mapping check? -\", reg2bc    \n",
    "        for k,v in sorted(reg_to_barcode.items(), key=lambda (k,v):k):\n",
    "            print k,':',v\n",
    "        \n",
    "    ## check barcode to region (this step must be run after step 3) \n",
    "    barcode_to_reg, bc2reg = _check_barcode_to_region(reg_to_barcode)\n",
    "    if verbose:\n",
    "        print \"\\n- 4.Passing barcode to region mapping check? -\", bc2reg    \n",
    "        for k,v in sorted(barcode_to_reg.items(), key=lambda (k,v):[k[0],int(k.split('_')[1])]):\n",
    "            print k,':',v\n",
    "    \n",
    "    ## check barcode to region (this step must be run after step 3) \n",
    "    barcode_to_color = _check_barcode_to_color()\n",
    "    if verbose:\n",
    "        print \"\\n- 5.Calculating barcode to color dictionary.\"\n",
    "        for k,v in sorted(barcode_to_color.items(), key=lambda (k,v):[k[0],int(k.split('_')[1])]):\n",
    "            print k,':',v    \n",
    "    \n",
    "    \n",
    "    ## Construct an internal map\n",
    "    int_map = _construct_internal_map();\n",
    "    if verbose:\n",
    "        print \"\\n- 6.Constructing internal khmer map\";\n",
    "    \n",
    "    ## Check barcodes total counts in probes\n",
    "    barcode_in_probes, _bc_counting = _check_barcode_in_probes(barcode_to_reg, reg_size_dic, int_map)\n",
    "    if verbose:\n",
    "        print \"\\n- 7.Passing if counting barcode appearance times in probes\", _bc_counting;    \n",
    "\n",
    "    ## Check against each other    \n",
    "    \n",
    "    ## Check against genome\n",
    "    kept_records, failed_num = _check_against_genome();\n",
    "    if verbose:\n",
    "        print \"\\n- 8.Probes not passing through genome filter:\", failed_num;  \n",
    "    \n",
    "    # check region size for kept probes\n",
    "    _size_from_rec = {}\n",
    "    for record in pb_records:\n",
    "        reg_id = int(record.id.split('_reg_')[1].split('_')[0]);\n",
    "        if reg_id not in _size_from_rec.keys():\n",
    "            _size_from_rec[reg_id] = 1; # if not in key, create\n",
    "        else:\n",
    "            _size_from_rec[reg_id] += 1; # otherwise, add count\n",
    "    if verbose:\n",
    "        print \"--  re-check region size:\"\n",
    "        for k,v in sorted(_size_from_rec.items()):\n",
    "            print k,':',v\n",
    "    \n",
    "    if save:\n",
    "        pb_savefile = master_dir + os.sep + save_folder + os.sep + 'filtered_probes.fasta';\n",
    "        if verbose:\n",
    "            print \"\\n- 9.Saving probes to:\", pb_savefile\n",
    "        with open(pb_savefile, 'w') as output_handle:\n",
    "            SeqIO.write(kept_records, output_handle, 'fasta');  \n",
    "        \n",
    "        \n",
    "    return kept_records, _size_from_rec"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 69075\n",
      "\n",
      "- 1.Passing primer usage check? - True\n",
      "\n",
      "- 2.Passing region size check? - True\n",
      "1 : 237\n",
      "2 : 180\n",
      "3 : 233\n",
      "4 : 222\n",
      "5 : 203\n",
      "6 : 282\n",
      "7 : 268\n",
      "8 : 278\n",
      "9 : 264\n",
      "10 : 345\n",
      "11 : 288\n",
      "12 : 170\n",
      "13 : 288\n",
      "14 : 278\n",
      "15 : 166\n",
      "16 : 179\n",
      "17 : 259\n",
      "18 : 248\n",
      "19 : 284\n",
      "20 : 279\n",
      "21 : 269\n",
      "22 : 269\n",
      "23 : 201\n",
      "24 : 220\n",
      "25 : 247\n",
      "26 : 251\n",
      "27 : 270\n",
      "28 : 221\n",
      "29 : 214\n",
      "30 : 193\n",
      "31 : 211\n",
      "32 : 234\n",
      "33 : 274\n",
      "34 : 280\n",
      "35 : 164\n",
      "36 : 166\n",
      "37 : 270\n",
      "38 : 199\n",
      "39 : 213\n",
      "40 : 274\n",
      "41 : 269\n",
      "42 : 209\n",
      "43 : 260\n",
      "44 : 177\n",
      "45 : 247\n",
      "46 : 119\n",
      "47 : 138\n",
      "48 : 235\n",
      "49 : 238\n",
      "50 : 270\n",
      "51 : 171\n",
      "52 : 126\n",
      "53 : 275\n",
      "54 : 301\n",
      "55 : 238\n",
      "56 : 225\n",
      "57 : 249\n",
      "58 : 181\n",
      "59 : 261\n",
      "60 : 202\n",
      "61 : 283\n",
      "62 : 239\n",
      "63 : 281\n",
      "64 : 193\n",
      "65 : 155\n",
      "66 : 268\n",
      "67 : 137\n",
      "68 : 296\n",
      "69 : 304\n",
      "70 : 293\n",
      "71 : 290\n",
      "72 : 339\n",
      "73 : 349\n",
      "74 : 298\n",
      "75 : 213\n",
      "76 : 262\n",
      "77 : 223\n",
      "78 : 238\n",
      "79 : 159\n",
      "80 : 175\n",
      "81 : 197\n",
      "82 : 170\n",
      "83 : 188\n",
      "84 : 200\n",
      "85 : 334\n",
      "86 : 241\n",
      "87 : 237\n",
      "88 : 209\n",
      "89 : 141\n",
      "90 : 263\n",
      "91 : 298\n",
      "92 : 301\n",
      "93 : 303\n",
      "94 : 289\n",
      "95 : 230\n",
      "96 : 237\n",
      "97 : 193\n",
      "98 : 216\n",
      "99 : 321\n",
      "100 : 327\n",
      "101 : 83\n",
      "102 : 118\n",
      "103 : 104\n",
      "104 : 87\n",
      "105 : 116\n",
      "106 : 136\n",
      "107 : 144\n",
      "108 : 113\n",
      "109 : 95\n",
      "110 : 114\n",
      "111 : 70\n",
      "112 : 113\n",
      "113 : 120\n",
      "114 : 74\n",
      "115 : 123\n",
      "116 : 95\n",
      "117 : 122\n",
      "118 : 123\n",
      "119 : 103\n",
      "120 : 118\n",
      "121 : 97\n",
      "122 : 81\n",
      "123 : 83\n",
      "124 : 87\n",
      "125 : 98\n",
      "126 : 290\n",
      "127 : 316\n",
      "128 : 291\n",
      "129 : 313\n",
      "130 : 270\n",
      "131 : 337\n",
      "132 : 311\n",
      "133 : 314\n",
      "134 : 297\n",
      "135 : 275\n",
      "136 : 287\n",
      "137 : 319\n",
      "138 : 212\n",
      "139 : 259\n",
      "140 : 229\n",
      "141 : 269\n",
      "142 : 94\n",
      "143 : 168\n",
      "144 : 201\n",
      "145 : 209\n",
      "146 : 271\n",
      "147 : 119\n",
      "148 : 155\n",
      "149 : 145\n",
      "150 : 236\n",
      "151 : 222\n",
      "152 : 246\n",
      "153 : 216\n",
      "154 : 154\n",
      "155 : 197\n",
      "156 : 217\n",
      "157 : 204\n",
      "158 : 206\n",
      "159 : 234\n",
      "160 : 138\n",
      "161 : 171\n",
      "162 : 168\n",
      "163 : 199\n",
      "164 : 193\n",
      "165 : 172\n",
      "166 : 298\n",
      "167 : 336\n",
      "168 : 346\n",
      "169 : 309\n",
      "170 : 275\n",
      "171 : 305\n",
      "172 : 231\n",
      "173 : 180\n",
      "174 : 212\n",
      "175 : 271\n",
      "176 : 203\n",
      "177 : 202\n",
      "178 : 189\n",
      "179 : 290\n",
      "180 : 293\n",
      "181 : 327\n",
      "182 : 307\n",
      "183 : 290\n",
      "184 : 307\n",
      "185 : 333\n",
      "186 : 293\n",
      "187 : 129\n",
      "188 : 184\n",
      "189 : 320\n",
      "190 : 211\n",
      "191 : 245\n",
      "192 : 221\n",
      "193 : 252\n",
      "194 : 287\n",
      "195 : 274\n",
      "196 : 240\n",
      "197 : 257\n",
      "198 : 312\n",
      "199 : 305\n",
      "200 : 298\n",
      "201 : 258\n",
      "202 : 305\n",
      "203 : 213\n",
      "204 : 241\n",
      "205 : 206\n",
      "206 : 242\n",
      "207 : 211\n",
      "208 : 187\n",
      "209 : 248\n",
      "210 : 173\n",
      "211 : 172\n",
      "212 : 309\n",
      "213 : 236\n",
      "214 : 243\n",
      "215 : 205\n",
      "216 : 259\n",
      "217 : 215\n",
      "218 : 250\n",
      "219 : 257\n",
      "220 : 281\n",
      "221 : 318\n",
      "222 : 265\n",
      "223 : 239\n",
      "224 : 284\n",
      "225 : 274\n",
      "226 : 298\n",
      "227 : 260\n",
      "228 : 295\n",
      "229 : 271\n",
      "230 : 308\n",
      "231 : 275\n",
      "232 : 221\n",
      "233 : 168\n",
      "234 : 151\n",
      "235 : 206\n",
      "236 : 201\n",
      "237 : 87\n",
      "238 : 140\n",
      "239 : 236\n",
      "240 : 203\n",
      "241 : 191\n",
      "242 : 193\n",
      "243 : 243\n",
      "244 : 292\n",
      "245 : 279\n",
      "246 : 263\n",
      "247 : 283\n",
      "248 : 248\n",
      "249 : 233\n",
      "250 : 263\n",
      "251 : 252\n",
      "252 : 236\n",
      "253 : 265\n",
      "254 : 261\n",
      "255 : 198\n",
      "256 : 265\n",
      "257 : 300\n",
      "258 : 226\n",
      "259 : 196\n",
      "260 : 315\n",
      "261 : 325\n",
      "262 : 346\n",
      "263 : 308\n",
      "264 : 237\n",
      "265 : 277\n",
      "266 : 277\n",
      "267 : 225\n",
      "268 : 193\n",
      "269 : 225\n",
      "270 : 215\n",
      "271 : 218\n",
      "272 : 211\n",
      "273 : 307\n",
      "274 : 259\n",
      "275 : 269\n",
      "276 : 307\n",
      "277 : 250\n",
      "278 : 206\n",
      "279 : 265\n",
      "280 : 287\n",
      "281 : 278\n",
      "282 : 194\n",
      "283 : 275\n",
      "284 : 267\n",
      "285 : 277\n",
      "286 : 309\n",
      "287 : 284\n",
      "288 : 277\n",
      "289 : 286\n",
      "290 : 259\n",
      "291 : 281\n",
      "292 : 274\n",
      "293 : 249\n",
      "294 : 273\n",
      "295 : 275\n",
      "296 : 246\n",
      "297 : 206\n",
      "\n",
      "- 3.Passing region to barcode mapping check? - True\n",
      "1 : ['Stv_3', 'Stv_23', 'NDB_1', 'NDB_1']\n",
      "2 : ['Stv_3', 'Stv_26', 'NDB_2', 'NDB_2']\n",
      "3 : ['Stv_3', 'Stv_28', 'NDB_3', 'NDB_3']\n",
      "4 : ['Stv_3', 'Stv_30', 'NDB_4', 'NDB_4']\n",
      "5 : ['Stv_3', 'Stv_32', 'NDB_5', 'NDB_5']\n",
      "6 : ['Stv_3', 'Stv_35', 'NDB_6', 'NDB_6']\n",
      "7 : ['Stv_3', 'Stv_37', 'NDB_7', 'NDB_7']\n",
      "8 : ['Stv_3', 'Stv_40', 'NDB_8', 'NDB_8']\n",
      "9 : ['Stv_3', 'Stv_44', 'NDB_9', 'NDB_9']\n",
      "10 : ['Stv_3', 'Stv_46', 'NDB_10', 'NDB_10']\n",
      "11 : ['Stv_3', 'Stv_50', 'NDB_11', 'NDB_11']\n",
      "12 : ['Stv_3', 'Stv_54', 'NDB_12', 'NDB_12']\n",
      "13 : ['Stv_3', 'Stv_60', 'NDB_13', 'NDB_13']\n",
      "14 : ['Stv_3', 'Stv_63', 'NDB_14', 'NDB_14']\n",
      "15 : ['Stv_3', 'Stv_65', 'NDB_15', 'NDB_15']\n",
      "16 : ['Stv_3', 'Stv_87', 'NDB_16', 'NDB_16']\n",
      "17 : ['Stv_3', 'Stv_90', 'NDB_17', 'NDB_17']\n",
      "18 : ['Stv_3', 'Stv_92', 'NDB_18', 'NDB_18']\n",
      "19 : ['Stv_3', 'Stv_95', 'NDB_19', 'NDB_19']\n",
      "20 : ['Stv_3', 'Stv_100', 'NDB_20', 'NDB_20']\n",
      "21 : ['Stv_4', 'Stv_25', 'NDB_21', 'NDB_21']\n",
      "22 : ['Stv_4', 'Stv_27', 'NDB_22', 'NDB_22']\n",
      "23 : ['Stv_4', 'Stv_29', 'NDB_23', 'NDB_23']\n",
      "24 : ['Stv_4', 'Stv_31', 'NDB_24', 'NDB_24']\n",
      "25 : ['Stv_4', 'Stv_33', 'NDB_25', 'NDB_25']\n",
      "26 : ['Stv_4', 'Stv_36', 'NDB_26', 'NDB_26']\n",
      "27 : ['Stv_4', 'Stv_39', 'NDB_27', 'NDB_27']\n",
      "28 : ['Stv_4', 'Stv_42', 'NDB_28', 'NDB_28']\n",
      "29 : ['Stv_4', 'Stv_45', 'NDB_29', 'NDB_29']\n",
      "30 : ['Stv_4', 'Stv_48', 'NDB_30', 'NDB_30']\n",
      "31 : ['Stv_4', 'Stv_53', 'NDB_31', 'NDB_31']\n",
      "32 : ['Stv_4', 'Stv_59', 'NDB_32', 'NDB_32']\n",
      "33 : ['Stv_4', 'Stv_61', 'NDB_33', 'NDB_33']\n",
      "34 : ['Stv_5', 'Stv_23', 'NDB_34', 'NDB_34']\n",
      "35 : ['Stv_5', 'Stv_26', 'NDB_35', 'NDB_35']\n",
      "36 : ['Stv_5', 'Stv_28', 'NDB_36', 'NDB_36']\n",
      "37 : ['Stv_5', 'Stv_30', 'NDB_37', 'NDB_37']\n",
      "38 : ['Stv_5', 'Stv_32', 'NDB_38', 'NDB_38']\n",
      "39 : ['Stv_5', 'Stv_35', 'NDB_39', 'NDB_39']\n",
      "40 : ['Stv_5', 'Stv_37', 'NDB_40', 'NDB_40']\n",
      "41 : ['Stv_5', 'Stv_40', 'NDB_41', 'NDB_41']\n",
      "42 : ['Stv_5', 'Stv_44', 'NDB_42', 'NDB_42']\n",
      "43 : ['Stv_5', 'Stv_46', 'NDB_43', 'NDB_43']\n",
      "44 : ['Stv_5', 'Stv_50', 'NDB_44', 'NDB_44']\n",
      "45 : ['Stv_5', 'Stv_54', 'NDB_45', 'NDB_45']\n",
      "46 : ['Stv_5', 'Stv_60', 'NDB_46', 'NDB_46']\n",
      "47 : ['Stv_5', 'Stv_63', 'NDB_47', 'NDB_47']\n",
      "48 : ['Stv_5', 'Stv_65', 'NDB_48', 'NDB_48']\n",
      "49 : ['Stv_5', 'Stv_87', 'NDB_49', 'NDB_49']\n",
      "50 : ['Stv_5', 'Stv_90', 'NDB_50', 'NDB_50']\n",
      "51 : ['Stv_5', 'Stv_92', 'NDB_51', 'NDB_51']\n",
      "52 : ['Stv_5', 'Stv_95', 'NDB_52', 'NDB_52']\n",
      "53 : ['Stv_5', 'Stv_100', 'NDB_53', 'NDB_53']\n",
      "54 : ['Stv_5', 'Stv_104', 'NDB_54', 'NDB_54']\n",
      "55 : ['Stv_5', 'Stv_106', 'NDB_55', 'NDB_55']\n",
      "56 : ['Stv_5', 'Stv_109', 'NDB_56', 'NDB_56']\n",
      "57 : ['Stv_6', 'Stv_25', 'NDB_57', 'NDB_57']\n",
      "58 : ['Stv_6', 'Stv_27', 'NDB_58', 'NDB_58']\n",
      "59 : ['Stv_6', 'Stv_29', 'NDB_59', 'NDB_59']\n",
      "60 : ['Stv_6', 'Stv_31', 'NDB_60', 'NDB_60']\n",
      "61 : ['Stv_6', 'Stv_33', 'NDB_61', 'NDB_61']\n",
      "62 : ['Stv_6', 'Stv_36', 'NDB_62', 'NDB_62']\n",
      "63 : ['Stv_6', 'Stv_39', 'NDB_63', 'NDB_63']\n",
      "64 : ['Stv_6', 'Stv_42', 'NDB_64', 'NDB_64']\n",
      "65 : ['Stv_6', 'Stv_45', 'NDB_65', 'NDB_65']\n",
      "66 : ['Stv_6', 'Stv_48', 'NDB_66', 'NDB_66']\n",
      "67 : ['Stv_6', 'Stv_53', 'NDB_67', 'NDB_67']\n",
      "68 : ['Stv_6', 'Stv_59', 'NDB_68', 'NDB_68']\n",
      "69 : ['Stv_6', 'Stv_61', 'NDB_69', 'NDB_69']\n",
      "70 : ['Stv_6', 'Stv_64', 'NDB_70', 'NDB_70']\n",
      "71 : ['Stv_6', 'Stv_86', 'NDB_71', 'NDB_71']\n",
      "72 : ['Stv_6', 'Stv_88', 'NDB_72', 'NDB_72']\n",
      "73 : ['Stv_6', 'Stv_91', 'NDB_73', 'NDB_73']\n",
      "74 : ['Stv_7', 'Stv_23', 'NDB_74', 'NDB_74']\n",
      "75 : ['Stv_7', 'Stv_26', 'NDB_75', 'NDB_75']\n",
      "76 : ['Stv_7', 'Stv_28', 'NDB_76', 'NDB_76']\n",
      "77 : ['Stv_7', 'Stv_30', 'NDB_77', 'NDB_77']\n",
      "78 : ['Stv_7', 'Stv_32', 'NDB_78', 'NDB_78']\n",
      "79 : ['Stv_7', 'Stv_35', 'NDB_79', 'NDB_79']\n",
      "80 : ['Stv_7', 'Stv_37', 'NDB_80', 'NDB_80']\n",
      "81 : ['Stv_7', 'Stv_40', 'NDB_81', 'NDB_81']\n",
      "82 : ['Stv_7', 'Stv_44', 'NDB_82', 'NDB_82']\n",
      "83 : ['Stv_7', 'Stv_46', 'NDB_83', 'NDB_83']\n",
      "84 : ['Stv_7', 'Stv_50', 'NDB_84', 'NDB_84']\n",
      "85 : ['Stv_7', 'Stv_54', 'NDB_85', 'NDB_85']\n",
      "86 : ['Stv_7', 'Stv_60', 'NDB_86', 'NDB_86']\n",
      "87 : ['Stv_8', 'Stv_25', 'NDB_87', 'NDB_87']\n",
      "88 : ['Stv_8', 'Stv_27', 'NDB_88', 'NDB_88']\n",
      "89 : ['Stv_8', 'Stv_29', 'NDB_89', 'NDB_89']\n",
      "90 : ['Stv_8', 'Stv_31', 'NDB_90', 'NDB_90']\n",
      "91 : ['Stv_8', 'Stv_33', 'NDB_91', 'NDB_91']\n",
      "92 : ['Stv_8', 'Stv_36', 'NDB_92', 'NDB_92']\n",
      "93 : ['Stv_8', 'Stv_39', 'NDB_93', 'NDB_93']\n",
      "94 : ['Stv_8', 'Stv_42', 'NDB_94', 'NDB_94']\n",
      "95 : ['Stv_8', 'Stv_45', 'NDB_95', 'NDB_95']\n",
      "96 : ['Stv_8', 'Stv_48', 'NDB_96', 'NDB_96']\n",
      "97 : ['Stv_8', 'Stv_53', 'NDB_97', 'NDB_97']\n",
      "98 : ['Stv_8', 'Stv_59', 'NDB_98', 'NDB_98']\n",
      "99 : ['Stv_8', 'Stv_61', 'NDB_99', 'NDB_99']\n",
      "100 : ['Stv_8', 'Stv_64', 'NDB_100', 'NDB_100']\n",
      "101 : ['Stv_9', 'Stv_23', 'NDB_101', 'NDB_101']\n",
      "102 : ['Stv_9', 'Stv_26', 'NDB_102', 'NDB_102']\n",
      "103 : ['Stv_9', 'Stv_28', 'NDB_103', 'NDB_103']\n",
      "104 : ['Stv_9', 'Stv_30', 'NDB_104', 'NDB_104']\n",
      "105 : ['Stv_9', 'Stv_32', 'NDB_105', 'NDB_105']\n",
      "106 : ['Stv_9', 'Stv_35', 'NDB_106', 'NDB_106']\n",
      "107 : ['Stv_9', 'Stv_37', 'NDB_107', 'NDB_107']\n",
      "108 : ['Stv_9', 'Stv_40', 'NDB_108', 'NDB_108']\n",
      "109 : ['Stv_9', 'Stv_44', 'NDB_109', 'NDB_109']\n",
      "110 : ['Stv_9', 'Stv_46', 'NDB_110', 'NDB_110']\n",
      "111 : ['Stv_9', 'Stv_50', 'NDB_111', 'NDB_111']\n",
      "112 : ['Stv_9', 'Stv_54', 'NDB_112', 'NDB_112']\n",
      "113 : ['Stv_9', 'Stv_60', 'NDB_113', 'NDB_113']\n",
      "114 : ['Stv_9', 'Stv_63', 'NDB_114', 'NDB_114']\n",
      "115 : ['Stv_9', 'Stv_65', 'NDB_115', 'NDB_115']\n",
      "116 : ['Stv_9', 'Stv_87', 'NDB_116', 'NDB_116']\n",
      "117 : ['Stv_9', 'Stv_90', 'NDB_117', 'NDB_117']\n",
      "118 : ['Stv_9', 'Stv_92', 'NDB_118', 'NDB_118']\n",
      "119 : ['Stv_9', 'Stv_95', 'NDB_119', 'NDB_119']\n",
      "120 : ['Stv_9', 'Stv_100', 'NDB_120', 'NDB_120']\n",
      "121 : ['Stv_9', 'Stv_104', 'NDB_121', 'NDB_121']\n",
      "122 : ['Stv_9', 'Stv_106', 'NDB_122', 'NDB_122']\n",
      "123 : ['Stv_9', 'Stv_109', 'NDB_123', 'NDB_123']\n",
      "124 : ['Stv_9', 'Stv_119', 'NDB_124', 'NDB_124']\n",
      "125 : ['Stv_9', 'Stv_121', 'NDB_125', 'NDB_125']\n",
      "126 : ['Stv_10', 'Stv_25', 'NDB_126', 'NDB_126']\n",
      "127 : ['Stv_10', 'Stv_27', 'NDB_127', 'NDB_127']\n",
      "128 : ['Stv_10', 'Stv_29', 'NDB_128', 'NDB_128']\n",
      "129 : ['Stv_10', 'Stv_31', 'NDB_129', 'NDB_129']\n",
      "130 : ['Stv_10', 'Stv_33', 'NDB_130', 'NDB_130']\n",
      "131 : ['Stv_10', 'Stv_36', 'NDB_131', 'NDB_131']\n",
      "132 : ['Stv_10', 'Stv_39', 'NDB_132', 'NDB_132']\n",
      "133 : ['Stv_10', 'Stv_42', 'NDB_133', 'NDB_133']\n",
      "134 : ['Stv_10', 'Stv_45', 'NDB_134', 'NDB_134']\n",
      "135 : ['Stv_10', 'Stv_48', 'NDB_135', 'NDB_135']\n",
      "136 : ['Stv_10', 'Stv_53', 'NDB_136', 'NDB_136']\n",
      "137 : ['Stv_10', 'Stv_59', 'NDB_137', 'NDB_137']\n",
      "138 : ['Stv_10', 'Stv_61', 'NDB_138', 'NDB_138']\n",
      "139 : ['Stv_10', 'Stv_64', 'NDB_139', 'NDB_139']\n",
      "140 : ['Stv_10', 'Stv_86', 'NDB_140', 'NDB_140']\n",
      "141 : ['Stv_10', 'Stv_88', 'NDB_141', 'NDB_141']\n",
      "142 : ['Stv_10', 'Stv_91', 'NDB_142', 'NDB_142']\n",
      "143 : ['Stv_10', 'Stv_94', 'NDB_143', 'NDB_143']\n",
      "144 : ['Stv_11', 'Stv_23', 'NDB_144', 'NDB_144']\n",
      "145 : ['Stv_11', 'Stv_26', 'NDB_145', 'NDB_145']\n",
      "146 : ['Stv_11', 'Stv_28', 'NDB_146', 'NDB_146']\n",
      "147 : ['Stv_11', 'Stv_30', 'NDB_147', 'NDB_147']\n",
      "148 : ['Stv_11', 'Stv_32', 'NDB_148', 'NDB_148']\n",
      "149 : ['Stv_11', 'Stv_35', 'NDB_149', 'NDB_149']\n",
      "150 : ['Stv_11', 'Stv_37', 'NDB_150', 'NDB_150']\n",
      "151 : ['Stv_11', 'Stv_40', 'NDB_151', 'NDB_151']\n",
      "152 : ['Stv_11', 'Stv_44', 'NDB_152', 'NDB_152']\n",
      "153 : ['Stv_11', 'Stv_46', 'NDB_153', 'NDB_153']\n",
      "154 : ['Stv_11', 'Stv_50', 'NDB_154', 'NDB_154']\n",
      "155 : ['Stv_11', 'Stv_54', 'NDB_155', 'NDB_155']\n",
      "156 : ['Stv_11', 'Stv_60', 'NDB_156', 'NDB_156']\n",
      "157 : ['Stv_11', 'Stv_63', 'NDB_157', 'NDB_157']\n",
      "158 : ['Stv_11', 'Stv_65', 'NDB_158', 'NDB_158']\n",
      "159 : ['Stv_11', 'Stv_87', 'NDB_159', 'NDB_159']\n",
      "160 : ['Stv_11', 'Stv_90', 'NDB_160', 'NDB_160']\n",
      "161 : ['Stv_11', 'Stv_92', 'NDB_161', 'NDB_161']\n",
      "162 : ['Stv_11', 'Stv_95', 'NDB_162', 'NDB_162']\n",
      "163 : ['Stv_11', 'Stv_100', 'NDB_163', 'NDB_163']\n",
      "164 : ['Stv_11', 'Stv_104', 'NDB_164', 'NDB_164']\n",
      "165 : ['Stv_11', 'Stv_106', 'NDB_165', 'NDB_165']\n",
      "166 : ['Stv_12', 'Stv_25', 'NDB_166', 'NDB_166']\n",
      "167 : ['Stv_12', 'Stv_27', 'NDB_167', 'NDB_167']\n",
      "168 : ['Stv_12', 'Stv_29', 'NDB_168', 'NDB_168']\n",
      "169 : ['Stv_12', 'Stv_31', 'NDB_169', 'NDB_169']\n",
      "170 : ['Stv_12', 'Stv_33', 'NDB_170', 'NDB_170']\n",
      "171 : ['Stv_12', 'Stv_36', 'NDB_171', 'NDB_171']\n",
      "172 : ['Stv_12', 'Stv_39', 'NDB_172', 'NDB_172']\n",
      "173 : ['Stv_12', 'Stv_42', 'NDB_173', 'NDB_173']\n",
      "174 : ['Stv_12', 'Stv_45', 'NDB_174', 'NDB_174']\n",
      "175 : ['Stv_12', 'Stv_48', 'NDB_175', 'NDB_175']\n",
      "176 : ['Stv_12', 'Stv_53', 'NDB_176', 'NDB_176']\n",
      "177 : ['Stv_12', 'Stv_59', 'NDB_177', 'NDB_177']\n",
      "178 : ['Stv_12', 'Stv_61', 'NDB_178', 'NDB_178']\n",
      "179 : ['Stv_13', 'Stv_23', 'NDB_179', 'NDB_179']\n",
      "180 : ['Stv_13', 'Stv_26', 'NDB_180', 'NDB_180']\n",
      "181 : ['Stv_13', 'Stv_28', 'NDB_181', 'NDB_181']\n",
      "182 : ['Stv_13', 'Stv_30', 'NDB_182', 'NDB_182']\n",
      "183 : ['Stv_13', 'Stv_32', 'NDB_183', 'NDB_183']\n",
      "184 : ['Stv_13', 'Stv_35', 'NDB_184', 'NDB_184']\n",
      "185 : ['Stv_13', 'Stv_37', 'NDB_185', 'NDB_185']\n",
      "186 : ['Stv_13', 'Stv_40', 'NDB_186', 'NDB_186']\n",
      "187 : ['Stv_13', 'Stv_44', 'NDB_187', 'NDB_187']\n",
      "188 : ['Stv_13', 'Stv_46', 'NDB_188', 'NDB_188']\n",
      "189 : ['Stv_13', 'Stv_50', 'NDB_189', 'NDB_189']\n",
      "190 : ['Stv_13', 'Stv_54', 'NDB_190', 'NDB_190']\n",
      "191 : ['Stv_13', 'Stv_60', 'NDB_191', 'NDB_191']\n",
      "192 : ['Stv_14', 'Stv_25', 'NDB_192', 'NDB_192']\n",
      "193 : ['Stv_14', 'Stv_27', 'NDB_193', 'NDB_193']\n",
      "194 : ['Stv_14', 'Stv_29', 'NDB_194', 'NDB_194']\n",
      "195 : ['Stv_14', 'Stv_31', 'NDB_195', 'NDB_195']\n",
      "196 : ['Stv_14', 'Stv_33', 'NDB_196', 'NDB_196']\n",
      "197 : ['Stv_14', 'Stv_36', 'NDB_197', 'NDB_197']\n",
      "198 : ['Stv_14', 'Stv_39', 'NDB_198', 'NDB_198']\n",
      "199 : ['Stv_14', 'Stv_42', 'NDB_199', 'NDB_199']\n",
      "200 : ['Stv_14', 'Stv_45', 'NDB_200', 'NDB_200']\n",
      "201 : ['Stv_14', 'Stv_48', 'NDB_201', 'NDB_201']\n",
      "202 : ['Stv_14', 'Stv_53', 'NDB_202', 'NDB_202']\n",
      "203 : ['Stv_14', 'Stv_59', 'NDB_203', 'NDB_203']\n",
      "204 : ['Stv_14', 'Stv_61', 'NDB_204', 'NDB_204']\n",
      "205 : ['Stv_14', 'Stv_64', 'NDB_205', 'NDB_205']\n",
      "206 : ['Stv_14', 'Stv_86', 'NDB_206', 'NDB_206']\n",
      "207 : ['Stv_14', 'Stv_88', 'NDB_207', 'NDB_207']\n",
      "208 : ['Stv_14', 'Stv_91', 'NDB_208', 'NDB_208']\n",
      "209 : ['Stv_14', 'Stv_94', 'NDB_209', 'NDB_209']\n",
      "210 : ['Stv_14', 'Stv_99', 'NDB_210', 'NDB_210']\n",
      "211 : ['Stv_14', 'Stv_101', 'NDB_211', 'NDB_211']\n",
      "212 : ['Stv_14', 'Stv_105', 'NDB_212', 'NDB_212']\n",
      "213 : ['Stv_16', 'Stv_23', 'NDB_213', 'NDB_213']\n",
      "214 : ['Stv_16', 'Stv_26', 'NDB_214', 'NDB_214']\n",
      "215 : ['Stv_16', 'Stv_28', 'NDB_215', 'NDB_215']\n",
      "216 : ['Stv_16', 'Stv_30', 'NDB_216', 'NDB_216']\n",
      "217 : ['Stv_16', 'Stv_32', 'NDB_217', 'NDB_217']\n",
      "218 : ['Stv_16', 'Stv_35', 'NDB_218', 'NDB_218']\n",
      "219 : ['Stv_16', 'Stv_37', 'NDB_219', 'NDB_219']\n",
      "220 : ['Stv_16', 'Stv_40', 'NDB_220', 'NDB_220']\n",
      "221 : ['Stv_16', 'Stv_44', 'NDB_221', 'NDB_221']\n",
      "222 : ['Stv_16', 'Stv_46', 'NDB_222', 'NDB_222']\n",
      "223 : ['Stv_16', 'Stv_50', 'NDB_223', 'NDB_223']\n",
      "224 : ['Stv_16', 'Stv_54', 'NDB_224', 'NDB_224']\n",
      "225 : ['Stv_16', 'Stv_60', 'NDB_225', 'NDB_225']\n",
      "226 : ['Stv_16', 'Stv_63', 'NDB_226', 'NDB_226']\n",
      "227 : ['Stv_16', 'Stv_65', 'NDB_227', 'NDB_227']\n",
      "228 : ['Stv_16', 'Stv_87', 'NDB_228', 'NDB_228']\n",
      "229 : ['Stv_16', 'Stv_90', 'NDB_229', 'NDB_229']\n",
      "230 : ['Stv_16', 'Stv_92', 'NDB_230', 'NDB_230']\n",
      "231 : ['Stv_16', 'Stv_95', 'NDB_231', 'NDB_231']\n",
      "232 : ['Stv_16', 'Stv_100', 'NDB_232', 'NDB_232']\n",
      "233 : ['Stv_16', 'Stv_104', 'NDB_233', 'NDB_233']\n",
      "234 : ['Stv_16', 'Stv_106', 'NDB_234', 'NDB_234']\n",
      "235 : ['Stv_16', 'Stv_109', 'NDB_235', 'NDB_235']\n",
      "236 : ['Stv_16', 'Stv_119', 'NDB_236', 'NDB_236']\n",
      "237 : ['Stv_16', 'Stv_121', 'NDB_237', 'NDB_237']\n",
      "238 : ['Stv_19', 'Stv_25', 'NDB_238', 'NDB_238']\n",
      "239 : ['Stv_19', 'Stv_27', 'NDB_239', 'NDB_239']\n",
      "240 : ['Stv_19', 'Stv_29', 'NDB_240', 'NDB_240']\n",
      "241 : ['Stv_19', 'Stv_31', 'NDB_241', 'NDB_241']\n",
      "242 : ['Stv_19', 'Stv_33', 'NDB_242', 'NDB_242']\n",
      "243 : ['Stv_19', 'Stv_36', 'NDB_243', 'NDB_243']\n",
      "244 : ['Stv_19', 'Stv_39', 'NDB_244', 'NDB_244']\n",
      "245 : ['Stv_19', 'Stv_42', 'NDB_245', 'NDB_245']\n",
      "246 : ['Stv_19', 'Stv_45', 'NDB_246', 'NDB_246']\n",
      "247 : ['Stv_19', 'Stv_48', 'NDB_247', 'NDB_247']\n",
      "248 : ['Stv_19', 'Stv_53', 'NDB_248', 'NDB_248']\n",
      "249 : ['Stv_19', 'Stv_59', 'NDB_249', 'NDB_249']\n",
      "250 : ['Stv_19', 'Stv_61', 'NDB_250', 'NDB_250']\n",
      "251 : ['Stv_19', 'Stv_64', 'NDB_251', 'NDB_251']\n",
      "252 : ['Stv_19', 'Stv_86', 'NDB_252', 'NDB_252']\n",
      "253 : ['Stv_19', 'Stv_88', 'NDB_253', 'NDB_253']\n",
      "254 : ['Stv_20', 'Stv_23', 'NDB_254', 'NDB_254']\n",
      "255 : ['Stv_20', 'Stv_26', 'NDB_255', 'NDB_255']\n",
      "256 : ['Stv_20', 'Stv_28', 'NDB_256', 'NDB_256']\n",
      "257 : ['Stv_20', 'Stv_30', 'NDB_257', 'NDB_257']\n",
      "258 : ['Stv_20', 'Stv_32', 'NDB_258', 'NDB_258']\n",
      "259 : ['Stv_20', 'Stv_35', 'NDB_259', 'NDB_259']\n",
      "260 : ['Stv_20', 'Stv_37', 'NDB_260', 'NDB_260']\n",
      "261 : ['Stv_20', 'Stv_40', 'NDB_261', 'NDB_261']\n",
      "262 : ['Stv_20', 'Stv_44', 'NDB_262', 'NDB_262']\n",
      "263 : ['Stv_20', 'Stv_46', 'NDB_263', 'NDB_263']\n",
      "264 : ['Stv_20', 'Stv_50', 'NDB_264', 'NDB_264']\n",
      "265 : ['Stv_20', 'Stv_54', 'NDB_265', 'NDB_265']\n",
      "266 : ['Stv_20', 'Stv_60', 'NDB_266', 'NDB_266']\n",
      "267 : ['Stv_21', 'Stv_25', 'NDB_267', 'NDB_267']\n",
      "268 : ['Stv_21', 'Stv_27', 'NDB_268', 'NDB_268']\n",
      "269 : ['Stv_21', 'Stv_29', 'NDB_269', 'NDB_269']\n",
      "270 : ['Stv_21', 'Stv_31', 'NDB_270', 'NDB_270']\n",
      "271 : ['Stv_21', 'Stv_33', 'NDB_271', 'NDB_271']\n",
      "272 : ['Stv_21', 'Stv_36', 'NDB_272', 'NDB_272']\n",
      "273 : ['Stv_21', 'Stv_39', 'NDB_273', 'NDB_273']\n",
      "274 : ['Stv_21', 'Stv_42', 'NDB_274', 'NDB_274']\n",
      "275 : ['Stv_21', 'Stv_45', 'NDB_275', 'NDB_275']\n",
      "276 : ['Stv_21', 'Stv_48', 'NDB_276', 'NDB_276']\n",
      "277 : ['Stv_21', 'Stv_53', 'NDB_277', 'NDB_277']\n",
      "278 : ['Stv_21', 'Stv_59', 'NDB_278', 'NDB_278']\n",
      "279 : ['Stv_21', 'Stv_61', 'NDB_279', 'NDB_279']\n",
      "280 : ['Stv_22', 'Stv_23', 'NDB_280', 'NDB_280']\n",
      "281 : ['Stv_22', 'Stv_26', 'NDB_281', 'NDB_281']\n",
      "282 : ['Stv_22', 'Stv_28', 'NDB_282', 'NDB_282']\n",
      "283 : ['Stv_22', 'Stv_30', 'NDB_283', 'NDB_283']\n",
      "284 : ['Stv_22', 'Stv_32', 'NDB_284', 'NDB_284']\n",
      "285 : ['Stv_22', 'Stv_35', 'NDB_285', 'NDB_285']\n",
      "286 : ['Stv_22', 'Stv_37', 'NDB_286', 'NDB_286']\n",
      "287 : ['Stv_22', 'Stv_40', 'NDB_287', 'NDB_287']\n",
      "288 : ['Stv_22', 'Stv_44', 'NDB_288', 'NDB_288']\n",
      "289 : ['Stv_22', 'Stv_46', 'NDB_289', 'NDB_289']\n",
      "290 : ['Stv_22', 'Stv_50', 'NDB_290', 'NDB_290']\n",
      "291 : ['Stv_22', 'Stv_54', 'NDB_291', 'NDB_291']\n",
      "292 : ['Stv_22', 'Stv_60', 'NDB_292', 'NDB_292']\n",
      "293 : ['Stv_22', 'Stv_63', 'NDB_293', 'NDB_293']\n",
      "294 : ['Stv_22', 'Stv_65', 'NDB_294', 'NDB_294']\n",
      "295 : ['Stv_22', 'Stv_87', 'NDB_295', 'NDB_295']\n",
      "296 : ['Stv_22', 'Stv_90', 'NDB_296', 'NDB_296']\n",
      "297 : ['Stv_22', 'Stv_92', 'NDB_297', 'NDB_297']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "- 4.Passing barcode to region mapping check? - True\n",
      "NDB_1 : [1, 1]\n",
      "NDB_2 : [2, 2]\n",
      "NDB_3 : [3, 3]\n",
      "NDB_4 : [4, 4]\n",
      "NDB_5 : [5, 5]\n",
      "NDB_6 : [6, 6]\n",
      "NDB_7 : [7, 7]\n",
      "NDB_8 : [8, 8]\n",
      "NDB_9 : [9, 9]\n",
      "NDB_10 : [10, 10]\n",
      "NDB_11 : [11, 11]\n",
      "NDB_12 : [12, 12]\n",
      "NDB_13 : [13, 13]\n",
      "NDB_14 : [14, 14]\n",
      "NDB_15 : [15, 15]\n",
      "NDB_16 : [16, 16]\n",
      "NDB_17 : [17, 17]\n",
      "NDB_18 : [18, 18]\n",
      "NDB_19 : [19, 19]\n",
      "NDB_20 : [20, 20]\n",
      "NDB_21 : [21, 21]\n",
      "NDB_22 : [22, 22]\n",
      "NDB_23 : [23, 23]\n",
      "NDB_24 : [24, 24]\n",
      "NDB_25 : [25, 25]\n",
      "NDB_26 : [26, 26]\n",
      "NDB_27 : [27, 27]\n",
      "NDB_28 : [28, 28]\n",
      "NDB_29 : [29, 29]\n",
      "NDB_30 : [30, 30]\n",
      "NDB_31 : [31, 31]\n",
      "NDB_32 : [32, 32]\n",
      "NDB_33 : [33, 33]\n",
      "NDB_34 : [34, 34]\n",
      "NDB_35 : [35, 35]\n",
      "NDB_36 : [36, 36]\n",
      "NDB_37 : [37, 37]\n",
      "NDB_38 : [38, 38]\n",
      "NDB_39 : [39, 39]\n",
      "NDB_40 : [40, 40]\n",
      "NDB_41 : [41, 41]\n",
      "NDB_42 : [42, 42]\n",
      "NDB_43 : [43, 43]\n",
      "NDB_44 : [44, 44]\n",
      "NDB_45 : [45, 45]\n",
      "NDB_46 : [46, 46]\n",
      "NDB_47 : [47, 47]\n",
      "NDB_48 : [48, 48]\n",
      "NDB_49 : [49, 49]\n",
      "NDB_50 : [50, 50]\n",
      "NDB_51 : [51, 51]\n",
      "NDB_52 : [52, 52]\n",
      "NDB_53 : [53, 53]\n",
      "NDB_54 : [54, 54]\n",
      "NDB_55 : [55, 55]\n",
      "NDB_56 : [56, 56]\n",
      "NDB_57 : [57, 57]\n",
      "NDB_58 : [58, 58]\n",
      "NDB_59 : [59, 59]\n",
      "NDB_60 : [60, 60]\n",
      "NDB_61 : [61, 61]\n",
      "NDB_62 : [62, 62]\n",
      "NDB_63 : [63, 63]\n",
      "NDB_64 : [64, 64]\n",
      "NDB_65 : [65, 65]\n",
      "NDB_66 : [66, 66]\n",
      "NDB_67 : [67, 67]\n",
      "NDB_68 : [68, 68]\n",
      "NDB_69 : [69, 69]\n",
      "NDB_70 : [70, 70]\n",
      "NDB_71 : [71, 71]\n",
      "NDB_72 : [72, 72]\n",
      "NDB_73 : [73, 73]\n",
      "NDB_74 : [74, 74]\n",
      "NDB_75 : [75, 75]\n",
      "NDB_76 : [76, 76]\n",
      "NDB_77 : [77, 77]\n",
      "NDB_78 : [78, 78]\n",
      "NDB_79 : [79, 79]\n",
      "NDB_80 : [80, 80]\n",
      "NDB_81 : [81, 81]\n",
      "NDB_82 : [82, 82]\n",
      "NDB_83 : [83, 83]\n",
      "NDB_84 : [84, 84]\n",
      "NDB_85 : [85, 85]\n",
      "NDB_86 : [86, 86]\n",
      "NDB_87 : [87, 87]\n",
      "NDB_88 : [88, 88]\n",
      "NDB_89 : [89, 89]\n",
      "NDB_90 : [90, 90]\n",
      "NDB_91 : [91, 91]\n",
      "NDB_92 : [92, 92]\n",
      "NDB_93 : [93, 93]\n",
      "NDB_94 : [94, 94]\n",
      "NDB_95 : [95, 95]\n",
      "NDB_96 : [96, 96]\n",
      "NDB_97 : [97, 97]\n",
      "NDB_98 : [98, 98]\n",
      "NDB_99 : [99, 99]\n",
      "NDB_100 : [100, 100]\n",
      "NDB_101 : [101, 101]\n",
      "NDB_102 : [102, 102]\n",
      "NDB_103 : [103, 103]\n",
      "NDB_104 : [104, 104]\n",
      "NDB_105 : [105, 105]\n",
      "NDB_106 : [106, 106]\n",
      "NDB_107 : [107, 107]\n",
      "NDB_108 : [108, 108]\n",
      "NDB_109 : [109, 109]\n",
      "NDB_110 : [110, 110]\n",
      "NDB_111 : [111, 111]\n",
      "NDB_112 : [112, 112]\n",
      "NDB_113 : [113, 113]\n",
      "NDB_114 : [114, 114]\n",
      "NDB_115 : [115, 115]\n",
      "NDB_116 : [116, 116]\n",
      "NDB_117 : [117, 117]\n",
      "NDB_118 : [118, 118]\n",
      "NDB_119 : [119, 119]\n",
      "NDB_120 : [120, 120]\n",
      "NDB_121 : [121, 121]\n",
      "NDB_122 : [122, 122]\n",
      "NDB_123 : [123, 123]\n",
      "NDB_124 : [124, 124]\n",
      "NDB_125 : [125, 125]\n",
      "NDB_126 : [126, 126]\n",
      "NDB_127 : [127, 127]\n",
      "NDB_128 : [128, 128]\n",
      "NDB_129 : [129, 129]\n",
      "NDB_130 : [130, 130]\n",
      "NDB_131 : [131, 131]\n",
      "NDB_132 : [132, 132]\n",
      "NDB_133 : [133, 133]\n",
      "NDB_134 : [134, 134]\n",
      "NDB_135 : [135, 135]\n",
      "NDB_136 : [136, 136]\n",
      "NDB_137 : [137, 137]\n",
      "NDB_138 : [138, 138]\n",
      "NDB_139 : [139, 139]\n",
      "NDB_140 : [140, 140]\n",
      "NDB_141 : [141, 141]\n",
      "NDB_142 : [142, 142]\n",
      "NDB_143 : [143, 143]\n",
      "NDB_144 : [144, 144]\n",
      "NDB_145 : [145, 145]\n",
      "NDB_146 : [146, 146]\n",
      "NDB_147 : [147, 147]\n",
      "NDB_148 : [148, 148]\n",
      "NDB_149 : [149, 149]\n",
      "NDB_150 : [150, 150]\n",
      "NDB_151 : [151, 151]\n",
      "NDB_152 : [152, 152]\n",
      "NDB_153 : [153, 153]\n",
      "NDB_154 : [154, 154]\n",
      "NDB_155 : [155, 155]\n",
      "NDB_156 : [156, 156]\n",
      "NDB_157 : [157, 157]\n",
      "NDB_158 : [158, 158]\n",
      "NDB_159 : [159, 159]\n",
      "NDB_160 : [160, 160]\n",
      "NDB_161 : [161, 161]\n",
      "NDB_162 : [162, 162]\n",
      "NDB_163 : [163, 163]\n",
      "NDB_164 : [164, 164]\n",
      "NDB_165 : [165, 165]\n",
      "NDB_166 : [166, 166]\n",
      "NDB_167 : [167, 167]\n",
      "NDB_168 : [168, 168]\n",
      "NDB_169 : [169, 169]\n",
      "NDB_170 : [170, 170]\n",
      "NDB_171 : [171, 171]\n",
      "NDB_172 : [172, 172]\n",
      "NDB_173 : [173, 173]\n",
      "NDB_174 : [174, 174]\n",
      "NDB_175 : [175, 175]\n",
      "NDB_176 : [176, 176]\n",
      "NDB_177 : [177, 177]\n",
      "NDB_178 : [178, 178]\n",
      "NDB_179 : [179, 179]\n",
      "NDB_180 : [180, 180]\n",
      "NDB_181 : [181, 181]\n",
      "NDB_182 : [182, 182]\n",
      "NDB_183 : [183, 183]\n",
      "NDB_184 : [184, 184]\n",
      "NDB_185 : [185, 185]\n",
      "NDB_186 : [186, 186]\n",
      "NDB_187 : [187, 187]\n",
      "NDB_188 : [188, 188]\n",
      "NDB_189 : [189, 189]\n",
      "NDB_190 : [190, 190]\n",
      "NDB_191 : [191, 191]\n",
      "NDB_192 : [192, 192]\n",
      "NDB_193 : [193, 193]\n",
      "NDB_194 : [194, 194]\n",
      "NDB_195 : [195, 195]\n",
      "NDB_196 : [196, 196]\n",
      "NDB_197 : [197, 197]\n",
      "NDB_198 : [198, 198]\n",
      "NDB_199 : [199, 199]\n",
      "NDB_200 : [200, 200]\n",
      "NDB_201 : [201, 201]\n",
      "NDB_202 : [202, 202]\n",
      "NDB_203 : [203, 203]\n",
      "NDB_204 : [204, 204]\n",
      "NDB_205 : [205, 205]\n",
      "NDB_206 : [206, 206]\n",
      "NDB_207 : [207, 207]\n",
      "NDB_208 : [208, 208]\n",
      "NDB_209 : [209, 209]\n",
      "NDB_210 : [210, 210]\n",
      "NDB_211 : [211, 211]\n",
      "NDB_212 : [212, 212]\n",
      "NDB_213 : [213, 213]\n",
      "NDB_214 : [214, 214]\n",
      "NDB_215 : [215, 215]\n",
      "NDB_216 : [216, 216]\n",
      "NDB_217 : [217, 217]\n",
      "NDB_218 : [218, 218]\n",
      "NDB_219 : [219, 219]\n",
      "NDB_220 : [220, 220]\n",
      "NDB_221 : [221, 221]\n",
      "NDB_222 : [222, 222]\n",
      "NDB_223 : [223, 223]\n",
      "NDB_224 : [224, 224]\n",
      "NDB_225 : [225, 225]\n",
      "NDB_226 : [226, 226]\n",
      "NDB_227 : [227, 227]\n",
      "NDB_228 : [228, 228]\n",
      "NDB_229 : [229, 229]\n",
      "NDB_230 : [230, 230]\n",
      "NDB_231 : [231, 231]\n",
      "NDB_232 : [232, 232]\n",
      "NDB_233 : [233, 233]\n",
      "NDB_234 : [234, 234]\n",
      "NDB_235 : [235, 235]\n",
      "NDB_236 : [236, 236]\n",
      "NDB_237 : [237, 237]\n",
      "NDB_238 : [238, 238]\n",
      "NDB_239 : [239, 239]\n",
      "NDB_240 : [240, 240]\n",
      "NDB_241 : [241, 241]\n",
      "NDB_242 : [242, 242]\n",
      "NDB_243 : [243, 243]\n",
      "NDB_244 : [244, 244]\n",
      "NDB_245 : [245, 245]\n",
      "NDB_246 : [246, 246]\n",
      "NDB_247 : [247, 247]\n",
      "NDB_248 : [248, 248]\n",
      "NDB_249 : [249, 249]\n",
      "NDB_250 : [250, 250]\n",
      "NDB_251 : [251, 251]\n",
      "NDB_252 : [252, 252]\n",
      "NDB_253 : [253, 253]\n",
      "NDB_254 : [254, 254]\n",
      "NDB_255 : [255, 255]\n",
      "NDB_256 : [256, 256]\n",
      "NDB_257 : [257, 257]\n",
      "NDB_258 : [258, 258]\n",
      "NDB_259 : [259, 259]\n",
      "NDB_260 : [260, 260]\n",
      "NDB_261 : [261, 261]\n",
      "NDB_262 : [262, 262]\n",
      "NDB_263 : [263, 263]\n",
      "NDB_264 : [264, 264]\n",
      "NDB_265 : [265, 265]\n",
      "NDB_266 : [266, 266]\n",
      "NDB_267 : [267, 267]\n",
      "NDB_268 : [268, 268]\n",
      "NDB_269 : [269, 269]\n",
      "NDB_270 : [270, 270]\n",
      "NDB_271 : [271, 271]\n",
      "NDB_272 : [272, 272]\n",
      "NDB_273 : [273, 273]\n",
      "NDB_274 : [274, 274]\n",
      "NDB_275 : [275, 275]\n",
      "NDB_276 : [276, 276]\n",
      "NDB_277 : [277, 277]\n",
      "NDB_278 : [278, 278]\n",
      "NDB_279 : [279, 279]\n",
      "NDB_280 : [280, 280]\n",
      "NDB_281 : [281, 281]\n",
      "NDB_282 : [282, 282]\n",
      "NDB_283 : [283, 283]\n",
      "NDB_284 : [284, 284]\n",
      "NDB_285 : [285, 285]\n",
      "NDB_286 : [286, 286]\n",
      "NDB_287 : [287, 287]\n",
      "NDB_288 : [288, 288]\n",
      "NDB_289 : [289, 289]\n",
      "NDB_290 : [290, 290]\n",
      "NDB_291 : [291, 291]\n",
      "NDB_292 : [292, 292]\n",
      "NDB_293 : [293, 293]\n",
      "NDB_294 : [294, 294]\n",
      "NDB_295 : [295, 295]\n",
      "NDB_296 : [296, 296]\n",
      "NDB_297 : [297, 297]\n",
      "Stv_3 : [1, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 2, 20, 3, 4, 5, 6, 7, 8, 9]\n",
      "Stv_4 : [21, 30, 31, 32, 33, 22, 23, 24, 25, 26, 27, 28, 29]\n",
      "Stv_5 : [34, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 35, 53, 54, 55, 56, 36, 37, 38, 39, 40, 41, 42]\n",
      "Stv_6 : [57, 66, 67, 68, 69, 70, 71, 72, 73, 58, 59, 60, 61, 62, 63, 64, 65]\n",
      "Stv_7 : [74, 83, 84, 85, 86, 75, 76, 77, 78, 79, 80, 81, 82]\n",
      "Stv_8 : [87, 96, 97, 98, 99, 100, 88, 89, 90, 91, 92, 93, 94, 95]\n",
      "Stv_9 : [101, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 102, 120, 121, 122, 123, 124, 125, 103, 104, 105, 106, 107, 108, 109]\n",
      "Stv_10 : [126, 135, 136, 137, 138, 139, 140, 141, 142, 143, 127, 128, 129, 130, 131, 132, 133, 134]\n",
      "Stv_11 : [144, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 145, 163, 164, 165, 146, 147, 148, 149, 150, 151, 152]\n",
      "Stv_12 : [166, 175, 176, 177, 178, 167, 168, 169, 170, 171, 172, 173, 174]\n",
      "Stv_13 : [179, 188, 189, 190, 191, 180, 181, 182, 183, 184, 185, 186, 187]\n",
      "Stv_14 : [192, 201, 202, 203, 204, 205, 206, 207, 208, 209, 210, 193, 211, 212, 194, 195, 196, 197, 198, 199, 200]\n",
      "Stv_16 : [213, 222, 223, 224, 225, 226, 227, 228, 229, 230, 231, 214, 232, 233, 234, 235, 236, 237, 215, 216, 217, 218, 219, 220, 221]\n",
      "Stv_19 : [238, 247, 248, 249, 250, 251, 252, 253, 239, 240, 241, 242, 243, 244, 245, 246]\n",
      "Stv_20 : [254, 263, 264, 265, 266, 255, 256, 257, 258, 259, 260, 261, 262]\n",
      "Stv_21 : [267, 276, 277, 278, 279, 268, 269, 270, 271, 272, 273, 274, 275]\n",
      "Stv_22 : [280, 289, 290, 291, 292, 293, 294, 295, 296, 297, 281, 282, 283, 284, 285, 286, 287, 288]\n",
      "Stv_23 : [280, 144, 213, 179, 1, 74, 254, 34, 101]\n",
      "Stv_25 : [267, 238, 87, 166, 192, 57, 126, 21]\n",
      "Stv_26 : [281, 145, 214, 180, 2, 75, 255, 35, 102]\n",
      "Stv_27 : [268, 239, 88, 167, 193, 58, 127, 22]\n",
      "Stv_28 : [282, 146, 215, 181, 3, 76, 256, 36, 103]\n",
      "Stv_29 : [269, 240, 89, 168, 194, 59, 128, 23]\n",
      "Stv_30 : [283, 147, 216, 182, 4, 77, 257, 37, 104]\n",
      "Stv_31 : [270, 241, 90, 169, 195, 60, 129, 24]\n",
      "Stv_32 : [284, 148, 217, 183, 5, 78, 258, 38, 105]\n",
      "Stv_33 : [271, 242, 91, 170, 196, 61, 130, 25]\n",
      "Stv_35 : [285, 149, 218, 184, 6, 79, 259, 39, 106]\n",
      "Stv_36 : [272, 243, 92, 171, 197, 62, 131, 26]\n",
      "Stv_37 : [286, 150, 219, 185, 7, 80, 260, 40, 107]\n",
      "Stv_39 : [273, 244, 93, 172, 198, 63, 132, 27]\n",
      "Stv_40 : [287, 151, 220, 186, 8, 81, 261, 41, 108]\n",
      "Stv_42 : [274, 245, 94, 173, 199, 64, 133, 28]\n",
      "Stv_44 : [288, 152, 221, 187, 9, 82, 262, 42, 109]\n",
      "Stv_45 : [275, 246, 95, 174, 200, 65, 134, 29]\n",
      "Stv_46 : [289, 153, 222, 188, 10, 83, 263, 43, 110]\n",
      "Stv_48 : [276, 247, 96, 175, 201, 66, 135, 30]\n",
      "Stv_50 : [290, 154, 223, 189, 11, 84, 264, 44, 111]\n",
      "Stv_53 : [277, 248, 97, 176, 202, 67, 136, 31]\n",
      "Stv_54 : [291, 155, 224, 190, 12, 85, 265, 45, 112]\n",
      "Stv_59 : [278, 249, 98, 177, 203, 68, 137, 32]\n",
      "Stv_60 : [292, 156, 225, 191, 13, 86, 266, 46, 113]\n",
      "Stv_61 : [279, 250, 99, 178, 204, 69, 138, 33]\n",
      "Stv_63 : [293, 157, 226, 14, 47, 114]\n",
      "Stv_64 : [251, 100, 205, 70, 139]\n",
      "Stv_65 : [294, 158, 227, 15, 48, 115]\n",
      "Stv_86 : [252, 206, 71, 140]\n",
      "Stv_87 : [295, 159, 228, 16, 49, 116]\n",
      "Stv_88 : [253, 207, 72, 141]\n",
      "Stv_90 : [296, 160, 229, 17, 50, 117]\n",
      "Stv_91 : [208, 73, 142]\n",
      "Stv_92 : [297, 161, 230, 18, 51, 118]\n",
      "Stv_94 : [209, 143]\n",
      "Stv_95 : [162, 231, 19, 52, 119]\n",
      "Stv_99 : [210]\n",
      "Stv_100 : [163, 232, 20, 53, 120]\n",
      "Stv_101 : [211]\n",
      "Stv_104 : [164, 233, 54, 121]\n",
      "Stv_105 : [212]\n",
      "Stv_106 : [165, 234, 55, 122]\n",
      "Stv_109 : [235, 56, 123]\n",
      "Stv_119 : [236, 124]\n",
      "Stv_121 : [237, 125]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "- 5.Calculating barcode to color dictionary.\n",
      "Stv_3 : [0]\n",
      "Stv_4 : [1]\n",
      "Stv_5 : [0]\n",
      "Stv_6 : [1]\n",
      "Stv_7 : [0]\n",
      "Stv_8 : [1]\n",
      "Stv_9 : [0]\n",
      "Stv_10 : [1]\n",
      "Stv_11 : [0]\n",
      "Stv_12 : [1]\n",
      "Stv_13 : [0]\n",
      "Stv_14 : [1]\n",
      "Stv_16 : [0]\n",
      "Stv_19 : [1]\n",
      "Stv_20 : [0]\n",
      "Stv_21 : [1]\n",
      "Stv_22 : [0]\n",
      "Stv_23 : [0]\n",
      "Stv_25 : [1]\n",
      "Stv_26 : [0]\n",
      "Stv_27 : [1]\n",
      "Stv_28 : [0]\n",
      "Stv_29 : [1]\n",
      "Stv_30 : [0]\n",
      "Stv_31 : [1]\n",
      "Stv_32 : [0]\n",
      "Stv_33 : [1]\n",
      "Stv_35 : [0]\n",
      "Stv_36 : [1]\n",
      "Stv_37 : [0]\n",
      "Stv_39 : [1]\n",
      "Stv_40 : [0]\n",
      "Stv_42 : [1]\n",
      "Stv_44 : [0]\n",
      "Stv_45 : [1]\n",
      "Stv_46 : [0]\n",
      "Stv_48 : [1]\n",
      "Stv_50 : [0]\n",
      "Stv_53 : [1]\n",
      "Stv_54 : [0]\n",
      "Stv_59 : [1]\n",
      "Stv_60 : [0]\n",
      "Stv_61 : [1]\n",
      "Stv_63 : [0]\n",
      "Stv_64 : [1]\n",
      "Stv_65 : [0]\n",
      "Stv_86 : [1]\n",
      "Stv_87 : [0]\n",
      "Stv_88 : [1]\n",
      "Stv_90 : [0]\n",
      "Stv_91 : [1]\n",
      "Stv_92 : [0]\n",
      "Stv_94 : [1]\n",
      "Stv_95 : [0]\n",
      "Stv_99 : [1]\n",
      "Stv_100 : [0]\n",
      "Stv_101 : [1]\n",
      "Stv_104 : [0]\n",
      "Stv_105 : [1]\n",
      "Stv_106 : [0]\n",
      "Stv_109 : [0]\n",
      "Stv_119 : [0]\n",
      "Stv_121 : [0]\n",
      "\n",
      "- 6.Constructing internal khmer map\n",
      "\n",
      "- 7.Passing if counting barcode appearance times in probes True\n",
      "-- Max_genome_hits is: 200, this seq got hits: 268\n",
      "-- Max_genome_hits is: 200, this seq got hits: 288\n",
      "-- Max_genome_hits is: 200, this seq got hits: 279\n",
      "-- Max_genome_hits is: 200, this seq got hits: 216\n",
      "-- Max_genome_hits is: 200, this seq got hits: 286\n",
      "-- Max_genome_hits is: 200, this seq got hits: 202\n",
      "-- Max_genome_hits is: 200, this seq got hits: 334\n",
      "-- Max_genome_hits is: 200, this seq got hits: 353\n",
      "-- Max_genome_hits is: 200, this seq got hits: 501\n",
      "-- Max_genome_hits is: 200, this seq got hits: 263\n",
      "-- Max_genome_hits is: 200, this seq got hits: 8810\n",
      "-- Max_genome_hits is: 200, this seq got hits: 916\n",
      "-- Max_genome_hits is: 200, this seq got hits: 349\n",
      "-- Max_genome_hits is: 200, this seq got hits: 429\n",
      "-- Max_genome_hits is: 200, this seq got hits: 424\n",
      "-- Max_genome_hits is: 200, this seq got hits: 808\n",
      "-- Max_genome_hits is: 200, this seq got hits: 222\n",
      "-- Max_genome_hits is: 200, this seq got hits: 216\n",
      "-- Max_genome_hits is: 200, this seq got hits: 217\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1096\n",
      "-- Max_genome_hits is: 200, this seq got hits: 525\n",
      "-- Max_genome_hits is: 200, this seq got hits: 982\n",
      "-- Max_genome_hits is: 200, this seq got hits: 317\n",
      "-- Max_genome_hits is: 200, this seq got hits: 251\n",
      "-- Max_genome_hits is: 200, this seq got hits: 273\n",
      "-- Max_genome_hits is: 200, this seq got hits: 315\n",
      "-- Max_genome_hits is: 200, this seq got hits: 316\n",
      "-- Max_genome_hits is: 200, this seq got hits: 969\n",
      "-- Max_genome_hits is: 200, this seq got hits: 285\n",
      "-- Max_genome_hits is: 200, this seq got hits: 216\n",
      "-- Max_genome_hits is: 200, this seq got hits: 215\n",
      "-- Max_genome_hits is: 200, this seq got hits: 277\n",
      "-- Max_genome_hits is: 200, this seq got hits: 359\n",
      "-- Max_genome_hits is: 200, this seq got hits: 888\n",
      "-- Max_genome_hits is: 200, this seq got hits: 838\n",
      "-- Max_genome_hits is: 200, this seq got hits: 275\n",
      "-- Max_genome_hits is: 200, this seq got hits: 298\n",
      "-- Max_genome_hits is: 200, this seq got hits: 350\n",
      "-- Max_genome_hits is: 200, this seq got hits: 224\n",
      "-- Max_genome_hits is: 200, this seq got hits: 225\n",
      "-- Max_genome_hits is: 200, this seq got hits: 242\n",
      "-- Max_genome_hits is: 200, this seq got hits: 225\n",
      "-- Max_genome_hits is: 200, this seq got hits: 222\n",
      "-- Max_genome_hits is: 200, this seq got hits: 922\n",
      "-- Max_genome_hits is: 200, this seq got hits: 253\n",
      "-- Max_genome_hits is: 200, this seq got hits: 532\n",
      "-- Max_genome_hits is: 200, this seq got hits: 224\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 253\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 286\n",
      "-- Max_genome_hits is: 200, this seq got hits: 651\n",
      "-- Max_genome_hits is: 200, this seq got hits: 248\n",
      "-- Max_genome_hits is: 200, this seq got hits: 293\n",
      "-- Max_genome_hits is: 200, this seq got hits: 335\n",
      "-- Max_genome_hits is: 200, this seq got hits: 407\n",
      "-- Max_genome_hits is: 200, this seq got hits: 814\n",
      "-- Max_genome_hits is: 200, this seq got hits: 884\n",
      "-- Max_genome_hits is: 200, this seq got hits: 218\n",
      "-- Max_genome_hits is: 200, this seq got hits: 289\n",
      "-- Max_genome_hits is: 200, this seq got hits: 271\n",
      "-- Max_genome_hits is: 200, this seq got hits: 211\n",
      "-- Max_genome_hits is: 200, this seq got hits: 331\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 242\n",
      "-- Max_genome_hits is: 200, this seq got hits: 471\n",
      "-- Max_genome_hits is: 200, this seq got hits: 286\n",
      "-- Max_genome_hits is: 200, this seq got hits: 564\n",
      "-- Max_genome_hits is: 200, this seq got hits: 399\n",
      "-- Max_genome_hits is: 200, this seq got hits: 232\n",
      "-- Max_genome_hits is: 200, this seq got hits: 253\n",
      "-- Max_genome_hits is: 200, this seq got hits: 463\n",
      "-- Max_genome_hits is: 200, this seq got hits: 229\n",
      "-- Max_genome_hits is: 200, this seq got hits: 496\n",
      "-- Max_genome_hits is: 200, this seq got hits: 474\n",
      "-- Max_genome_hits is: 200, this seq got hits: 241\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1200\n",
      "-- Max_genome_hits is: 200, this seq got hits: 609\n",
      "-- Max_genome_hits is: 200, this seq got hits: 202\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 288\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 294\n",
      "-- Max_genome_hits is: 200, this seq got hits: 315\n",
      "-- Max_genome_hits is: 200, this seq got hits: 8634\n",
      "-- Max_genome_hits is: 200, this seq got hits: 642\n",
      "-- Max_genome_hits is: 200, this seq got hits: 401\n",
      "-- Max_genome_hits is: 200, this seq got hits: 308\n",
      "-- Max_genome_hits is: 200, this seq got hits: 288\n",
      "-- Max_genome_hits is: 200, this seq got hits: 3945\n",
      "-- Max_genome_hits is: 200, this seq got hits: 344\n",
      "-- Max_genome_hits is: 200, this seq got hits: 274\n",
      "-- Max_genome_hits is: 200, this seq got hits: 237\n",
      "-- Max_genome_hits is: 200, this seq got hits: 263\n",
      "-- Max_genome_hits is: 200, this seq got hits: 558\n",
      "-- Max_genome_hits is: 200, this seq got hits: 240\n",
      "-- Max_genome_hits is: 200, this seq got hits: 236\n",
      "-- Max_genome_hits is: 200, this seq got hits: 444\n",
      "-- Max_genome_hits is: 200, this seq got hits: 359\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1181\n",
      "-- Max_genome_hits is: 200, this seq got hits: 484\n",
      "-- Max_genome_hits is: 200, this seq got hits: 234\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1107\n",
      "-- Max_genome_hits is: 200, this seq got hits: 301\n",
      "-- Max_genome_hits is: 200, this seq got hits: 279\n",
      "-- Max_genome_hits is: 200, this seq got hits: 15588\n",
      "-- Max_genome_hits is: 200, this seq got hits: 218\n",
      "-- Max_genome_hits is: 200, this seq got hits: 223\n",
      "-- Max_genome_hits is: 200, this seq got hits: 790\n",
      "-- Max_genome_hits is: 200, this seq got hits: 482\n",
      "-- Max_genome_hits is: 200, this seq got hits: 347\n",
      "-- Max_genome_hits is: 200, this seq got hits: 479\n",
      "-- Max_genome_hits is: 200, this seq got hits: 232\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 307\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 205\n",
      "-- Max_genome_hits is: 200, this seq got hits: 305\n",
      "-- Max_genome_hits is: 200, this seq got hits: 424\n",
      "-- Max_genome_hits is: 200, this seq got hits: 201\n",
      "-- Max_genome_hits is: 200, this seq got hits: 203\n",
      "-- Max_genome_hits is: 200, this seq got hits: 212\n",
      "-- Max_genome_hits is: 200, this seq got hits: 209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 264\n",
      "-- Max_genome_hits is: 200, this seq got hits: 443\n",
      "-- Max_genome_hits is: 200, this seq got hits: 241\n",
      "-- Max_genome_hits is: 200, this seq got hits: 686\n",
      "-- Max_genome_hits is: 200, this seq got hits: 210\n",
      "-- Max_genome_hits is: 200, this seq got hits: 375\n",
      "-- Max_genome_hits is: 200, this seq got hits: 226\n",
      "-- Max_genome_hits is: 200, this seq got hits: 218\n",
      "-- Max_genome_hits is: 200, this seq got hits: 955\n",
      "-- Max_genome_hits is: 200, this seq got hits: 341\n",
      "-- Max_genome_hits is: 200, this seq got hits: 302\n",
      "-- Max_genome_hits is: 200, this seq got hits: 244\n",
      "-- Max_genome_hits is: 200, this seq got hits: 247\n",
      "-- Max_genome_hits is: 200, this seq got hits: 278\n",
      "-- Max_genome_hits is: 200, this seq got hits: 226\n",
      "-- Max_genome_hits is: 200, this seq got hits: 323\n",
      "-- Max_genome_hits is: 200, this seq got hits: 332\n",
      "-- Max_genome_hits is: 200, this seq got hits: 243\n",
      "-- Max_genome_hits is: 200, this seq got hits: 302\n",
      "-- Max_genome_hits is: 200, this seq got hits: 216\n",
      "-- Max_genome_hits is: 200, this seq got hits: 477\n",
      "-- Max_genome_hits is: 200, this seq got hits: 348\n",
      "-- Max_genome_hits is: 200, this seq got hits: 257\n",
      "-- Max_genome_hits is: 200, this seq got hits: 491\n",
      "-- Max_genome_hits is: 200, this seq got hits: 216\n",
      "-- Max_genome_hits is: 200, this seq got hits: 309\n",
      "-- Max_genome_hits is: 200, this seq got hits: 223\n",
      "-- Max_genome_hits is: 200, this seq got hits: 264\n",
      "-- Max_genome_hits is: 200, this seq got hits: 826\n",
      "-- Max_genome_hits is: 200, this seq got hits: 711\n",
      "-- Max_genome_hits is: 200, this seq got hits: 201\n",
      "-- Max_genome_hits is: 200, this seq got hits: 485\n",
      "-- Max_genome_hits is: 200, this seq got hits: 665\n",
      "-- Max_genome_hits is: 200, this seq got hits: 345\n",
      "-- Max_genome_hits is: 200, this seq got hits: 321\n",
      "-- Max_genome_hits is: 200, this seq got hits: 432\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 241\n",
      "-- Max_genome_hits is: 200, this seq got hits: 269\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2565\n",
      "-- Max_genome_hits is: 200, this seq got hits: 668\n",
      "-- Max_genome_hits is: 200, this seq got hits: 260\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Max_genome_hits is: 200, this seq got hits: 1325\n",
      "-- Max_genome_hits is: 200, this seq got hits: 297\n",
      "-- Max_genome_hits is: 200, this seq got hits: 332\n",
      "-- Max_genome_hits is: 200, this seq got hits: 374\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n",
      "-- Max_genome_hits is: 200, this seq got hits: 331\n",
      "-- Max_genome_hits is: 200, this seq got hits: 225\n",
      "-- Max_genome_hits is: 200, this seq got hits: 834\n",
      "-- Max_genome_hits is: 200, this seq got hits: 499\n",
      "-- Max_genome_hits is: 200, this seq got hits: 210\n",
      "-- Max_genome_hits is: 200, this seq got hits: 407\n",
      "-- Max_genome_hits is: 200, this seq got hits: 885\n",
      "-- Max_genome_hits is: 200, this seq got hits: 533\n",
      "-- Max_genome_hits is: 200, this seq got hits: 402\n",
      "-- Max_genome_hits is: 200, this seq got hits: 513\n",
      "-- Max_genome_hits is: 200, this seq got hits: 215\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 469\n",
      "-- Max_genome_hits is: 200, this seq got hits: 280\n",
      "-- Max_genome_hits is: 200, this seq got hits: 245\n",
      "-- Max_genome_hits is: 200, this seq got hits: 223\n",
      "-- Max_genome_hits is: 200, this seq got hits: 463\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 630\n",
      "-- Max_genome_hits is: 200, this seq got hits: 3216\n",
      "-- Max_genome_hits is: 200, this seq got hits: 245\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n",
      "-- Max_genome_hits is: 200, this seq got hits: 221\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 399\n",
      "-- Max_genome_hits is: 200, this seq got hits: 212\n",
      "-- Max_genome_hits is: 200, this seq got hits: 727\n",
      "-- Max_genome_hits is: 200, this seq got hits: 333\n",
      "-- Max_genome_hits is: 200, this seq got hits: 264\n",
      "-- Max_genome_hits is: 200, this seq got hits: 223\n",
      "-- Max_genome_hits is: 200, this seq got hits: 248\n",
      "-- Max_genome_hits is: 200, this seq got hits: 339\n",
      "-- Max_genome_hits is: 200, this seq got hits: 217\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1422\n",
      "-- Max_genome_hits is: 200, this seq got hits: 248\n",
      "-- Max_genome_hits is: 200, this seq got hits: 353\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1251\n",
      "-- Max_genome_hits is: 200, this seq got hits: 362\n",
      "-- Max_genome_hits is: 200, this seq got hits: 230\n",
      "-- Max_genome_hits is: 200, this seq got hits: 280\n",
      "-- Max_genome_hits is: 200, this seq got hits: 353\n",
      "-- Max_genome_hits is: 200, this seq got hits: 508\n",
      "-- Max_genome_hits is: 200, this seq got hits: 202\n",
      "-- Max_genome_hits is: 200, this seq got hits: 611\n",
      "-- Max_genome_hits is: 200, this seq got hits: 412\n",
      "-- Max_genome_hits is: 200, this seq got hits: 4825\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1003\n",
      "-- Max_genome_hits is: 200, this seq got hits: 288\n",
      "-- Max_genome_hits is: 200, this seq got hits: 225\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1220\n",
      "-- Max_genome_hits is: 200, this seq got hits: 272\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1177\n",
      "-- Max_genome_hits is: 200, this seq got hits: 337\n",
      "-- Max_genome_hits is: 200, this seq got hits: 549\n",
      "-- Max_genome_hits is: 200, this seq got hits: 451\n",
      "-- Max_genome_hits is: 200, this seq got hits: 234\n",
      "-- Max_genome_hits is: 200, this seq got hits: 217\n",
      "-- Max_genome_hits is: 200, this seq got hits: 748\n",
      "-- Max_genome_hits is: 200, this seq got hits: 353\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2625\n",
      "-- Max_genome_hits is: 200, this seq got hits: 514\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 469\n",
      "-- Max_genome_hits is: 200, this seq got hits: 348\n",
      "-- Max_genome_hits is: 200, this seq got hits: 229\n",
      "-- Max_genome_hits is: 200, this seq got hits: 359\n",
      "-- Max_genome_hits is: 200, this seq got hits: 315\n",
      "-- Max_genome_hits is: 200, this seq got hits: 672\n",
      "-- Max_genome_hits is: 200, this seq got hits: 256\n",
      "-- Max_genome_hits is: 200, this seq got hits: 217\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2101\n",
      "-- Max_genome_hits is: 200, this seq got hits: 652\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1023\n",
      "-- Max_genome_hits is: 200, this seq got hits: 275\n",
      "-- Max_genome_hits is: 200, this seq got hits: 245\n",
      "-- Max_genome_hits is: 200, this seq got hits: 254\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 339\n",
      "-- Max_genome_hits is: 200, this seq got hits: 673\n",
      "-- Max_genome_hits is: 200, this seq got hits: 249\n",
      "-- Max_genome_hits is: 200, this seq got hits: 839\n",
      "-- Max_genome_hits is: 200, this seq got hits: 376\n",
      "-- Max_genome_hits is: 200, this seq got hits: 209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 275\n",
      "-- Max_genome_hits is: 200, this seq got hits: 265\n",
      "-- Max_genome_hits is: 200, this seq got hits: 229\n",
      "-- Max_genome_hits is: 200, this seq got hits: 770\n",
      "-- Max_genome_hits is: 200, this seq got hits: 230\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1230\n",
      "-- Max_genome_hits is: 200, this seq got hits: 230\n",
      "-- Max_genome_hits is: 200, this seq got hits: 576\n",
      "-- Max_genome_hits is: 200, this seq got hits: 315\n",
      "-- Max_genome_hits is: 200, this seq got hits: 380\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1718\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1069\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2616\n",
      "-- Max_genome_hits is: 200, this seq got hits: 455\n",
      "-- Max_genome_hits is: 200, this seq got hits: 220\n",
      "-- Max_genome_hits is: 200, this seq got hits: 328\n",
      "-- Max_genome_hits is: 200, this seq got hits: 205\n",
      "-- Max_genome_hits is: 200, this seq got hits: 262\n",
      "-- Max_genome_hits is: 200, this seq got hits: 249\n",
      "-- Max_genome_hits is: 200, this seq got hits: 489\n",
      "-- Max_genome_hits is: 200, this seq got hits: 205\n",
      "-- Max_genome_hits is: 200, this seq got hits: 378\n",
      "-- Max_genome_hits is: 200, this seq got hits: 237\n",
      "-- Max_genome_hits is: 200, this seq got hits: 268\n",
      "-- Max_genome_hits is: 200, this seq got hits: 294\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 245\n",
      "-- Max_genome_hits is: 200, this seq got hits: 6640\n",
      "-- Max_genome_hits is: 200, this seq got hits: 475\n",
      "-- Max_genome_hits is: 200, this seq got hits: 261\n",
      "-- Max_genome_hits is: 200, this seq got hits: 314\n",
      "-- Max_genome_hits is: 200, this seq got hits: 230\n",
      "-- Max_genome_hits is: 200, this seq got hits: 322\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 391\n",
      "-- Max_genome_hits is: 200, this seq got hits: 663\n",
      "-- Max_genome_hits is: 200, this seq got hits: 246\n",
      "-- Max_genome_hits is: 200, this seq got hits: 294\n",
      "-- Max_genome_hits is: 200, this seq got hits: 688\n",
      "-- Max_genome_hits is: 200, this seq got hits: 429\n",
      "-- Max_genome_hits is: 200, this seq got hits: 207\n",
      "-- Max_genome_hits is: 200, this seq got hits: 714\n",
      "-- Max_genome_hits is: 200, this seq got hits: 212\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1070\n",
      "-- Max_genome_hits is: 200, this seq got hits: 258\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2202\n",
      "-- Max_genome_hits is: 200, this seq got hits: 230\n",
      "-- Max_genome_hits is: 200, this seq got hits: 314\n",
      "-- Max_genome_hits is: 200, this seq got hits: 221\n",
      "-- Max_genome_hits is: 200, this seq got hits: 3221\n",
      "-- Max_genome_hits is: 200, this seq got hits: 342\n",
      "-- Max_genome_hits is: 200, this seq got hits: 221\n",
      "-- Max_genome_hits is: 200, this seq got hits: 266\n",
      "-- Max_genome_hits is: 200, this seq got hits: 250\n",
      "-- Max_genome_hits is: 200, this seq got hits: 240\n",
      "-- Max_genome_hits is: 200, this seq got hits: 224\n",
      "-- Max_genome_hits is: 200, this seq got hits: 252\n",
      "-- Max_genome_hits is: 200, this seq got hits: 227\n",
      "-- Max_genome_hits is: 200, this seq got hits: 490\n",
      "-- Max_genome_hits is: 200, this seq got hits: 305\n",
      "-- Max_genome_hits is: 200, this seq got hits: 232\n",
      "-- Max_genome_hits is: 200, this seq got hits: 312\n",
      "-- Max_genome_hits is: 200, this seq got hits: 256\n",
      "-- Max_genome_hits is: 200, this seq got hits: 216\n",
      "-- Max_genome_hits is: 200, this seq got hits: 330\n",
      "-- Max_genome_hits is: 200, this seq got hits: 633\n",
      "-- Max_genome_hits is: 200, this seq got hits: 202\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1155\n",
      "-- Max_genome_hits is: 200, this seq got hits: 240\n",
      "-- Max_genome_hits is: 200, this seq got hits: 339\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 626\n",
      "-- Max_genome_hits is: 200, this seq got hits: 282\n",
      "-- Max_genome_hits is: 200, this seq got hits: 201\n",
      "-- Max_genome_hits is: 200, this seq got hits: 292\n",
      "-- Max_genome_hits is: 200, this seq got hits: 983\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n",
      "-- Max_genome_hits is: 200, this seq got hits: 511\n",
      "-- Max_genome_hits is: 200, this seq got hits: 348\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2261\n",
      "-- Max_genome_hits is: 200, this seq got hits: 3007\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Max_genome_hits is: 200, this seq got hits: 632\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2733\n",
      "-- Max_genome_hits is: 200, this seq got hits: 225\n",
      "-- Max_genome_hits is: 200, this seq got hits: 232\n",
      "-- Max_genome_hits is: 200, this seq got hits: 244\n",
      "-- Max_genome_hits is: 200, this seq got hits: 231\n",
      "-- Max_genome_hits is: 200, this seq got hits: 276\n",
      "-- Max_genome_hits is: 200, this seq got hits: 630\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n",
      "-- Max_genome_hits is: 200, this seq got hits: 229\n",
      "-- Max_genome_hits is: 200, this seq got hits: 238\n",
      "-- Max_genome_hits is: 200, this seq got hits: 209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 358\n",
      "-- Max_genome_hits is: 200, this seq got hits: 276\n",
      "-- Max_genome_hits is: 200, this seq got hits: 647\n",
      "-- Max_genome_hits is: 200, this seq got hits: 282\n",
      "-- Max_genome_hits is: 200, this seq got hits: 275\n",
      "-- Max_genome_hits is: 200, this seq got hits: 363\n",
      "-- Max_genome_hits is: 200, this seq got hits: 351\n",
      "-- Max_genome_hits is: 200, this seq got hits: 216\n",
      "-- Max_genome_hits is: 200, this seq got hits: 222\n",
      "-- Max_genome_hits is: 200, this seq got hits: 219\n",
      "-- Max_genome_hits is: 200, this seq got hits: 509\n",
      "-- Max_genome_hits is: 200, this seq got hits: 471\n",
      "-- Max_genome_hits is: 200, this seq got hits: 210\n",
      "-- Max_genome_hits is: 200, this seq got hits: 263\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1407\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 411\n",
      "-- Max_genome_hits is: 200, this seq got hits: 261\n",
      "-- Max_genome_hits is: 200, this seq got hits: 209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 381\n",
      "-- Max_genome_hits is: 200, this seq got hits: 3205\n",
      "-- Max_genome_hits is: 200, this seq got hits: 276\n",
      "-- Max_genome_hits is: 200, this seq got hits: 232\n",
      "-- Max_genome_hits is: 200, this seq got hits: 398\n",
      "-- Max_genome_hits is: 200, this seq got hits: 473\n",
      "-- Max_genome_hits is: 200, this seq got hits: 334\n",
      "-- Max_genome_hits is: 200, this seq got hits: 266\n",
      "-- Max_genome_hits is: 200, this seq got hits: 306\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1603\n",
      "-- Max_genome_hits is: 200, this seq got hits: 288\n",
      "-- Max_genome_hits is: 200, this seq got hits: 207\n",
      "-- Max_genome_hits is: 200, this seq got hits: 238\n",
      "-- Max_genome_hits is: 200, this seq got hits: 376\n",
      "-- Max_genome_hits is: 200, this seq got hits: 391\n",
      "-- Max_genome_hits is: 200, this seq got hits: 448\n",
      "-- Max_genome_hits is: 200, this seq got hits: 628\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 246\n",
      "-- Max_genome_hits is: 200, this seq got hits: 655\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2833\n",
      "-- Max_genome_hits is: 200, this seq got hits: 9817\n",
      "-- Max_genome_hits is: 200, this seq got hits: 279\n",
      "-- Max_genome_hits is: 200, this seq got hits: 258\n",
      "-- Max_genome_hits is: 200, this seq got hits: 324\n",
      "-- Max_genome_hits is: 200, this seq got hits: 913\n",
      "-- Max_genome_hits is: 200, this seq got hits: 548\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1560\n",
      "-- Max_genome_hits is: 200, this seq got hits: 238\n",
      "-- Max_genome_hits is: 200, this seq got hits: 386\n",
      "-- Max_genome_hits is: 200, this seq got hits: 326\n",
      "-- Max_genome_hits is: 200, this seq got hits: 263\n",
      "-- Max_genome_hits is: 200, this seq got hits: 470\n",
      "-- Max_genome_hits is: 200, this seq got hits: 238\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 305\n",
      "-- Max_genome_hits is: 200, this seq got hits: 219\n",
      "-- Max_genome_hits is: 200, this seq got hits: 492\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1223\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 280\n",
      "-- Max_genome_hits is: 200, this seq got hits: 527\n",
      "-- Max_genome_hits is: 200, this seq got hits: 282\n",
      "-- Max_genome_hits is: 200, this seq got hits: 235\n",
      "-- Max_genome_hits is: 200, this seq got hits: 590\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 249\n",
      "-- Max_genome_hits is: 200, this seq got hits: 226\n",
      "-- Max_genome_hits is: 200, this seq got hits: 317\n",
      "-- Max_genome_hits is: 200, this seq got hits: 295\n",
      "-- Max_genome_hits is: 200, this seq got hits: 312\n",
      "-- Max_genome_hits is: 200, this seq got hits: 205\n",
      "-- Max_genome_hits is: 200, this seq got hits: 391\n",
      "-- Max_genome_hits is: 200, this seq got hits: 284\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n",
      "-- Max_genome_hits is: 200, this seq got hits: 266\n",
      "-- Max_genome_hits is: 200, this seq got hits: 251\n",
      "-- Max_genome_hits is: 200, this seq got hits: 201\n",
      "-- Max_genome_hits is: 200, this seq got hits: 294\n",
      "-- Max_genome_hits is: 200, this seq got hits: 219\n",
      "-- Max_genome_hits is: 200, this seq got hits: 203\n",
      "-- Max_genome_hits is: 200, this seq got hits: 614\n",
      "-- Max_genome_hits is: 200, this seq got hits: 507\n",
      "\n",
      "- 8.Probes not passing through genome filter: 431\n",
      "--  re-check region size:\n",
      "1 : 237\n",
      "2 : 180\n",
      "3 : 233\n",
      "4 : 222\n",
      "5 : 203\n",
      "6 : 282\n",
      "7 : 268\n",
      "8 : 278\n",
      "9 : 264\n",
      "10 : 345\n",
      "11 : 288\n",
      "12 : 170\n",
      "13 : 288\n",
      "14 : 278\n",
      "15 : 166\n",
      "16 : 179\n",
      "17 : 259\n",
      "18 : 248\n",
      "19 : 284\n",
      "20 : 279\n",
      "21 : 269\n",
      "22 : 269\n",
      "23 : 201\n",
      "24 : 220\n",
      "25 : 247\n",
      "26 : 251\n",
      "27 : 270\n",
      "28 : 221\n",
      "29 : 214\n",
      "30 : 193\n",
      "31 : 211\n",
      "32 : 234\n",
      "33 : 274\n",
      "34 : 280\n",
      "35 : 164\n",
      "36 : 166\n",
      "37 : 270\n",
      "38 : 199\n",
      "39 : 213\n",
      "40 : 274\n",
      "41 : 269\n",
      "42 : 209\n",
      "43 : 260\n",
      "44 : 177\n",
      "45 : 247\n",
      "46 : 119\n",
      "47 : 138\n",
      "48 : 235\n",
      "49 : 238\n",
      "50 : 270\n",
      "51 : 171\n",
      "52 : 126\n",
      "53 : 275\n",
      "54 : 301\n",
      "55 : 238\n",
      "56 : 225\n",
      "57 : 249\n",
      "58 : 181\n",
      "59 : 261\n",
      "60 : 202\n",
      "61 : 283\n",
      "62 : 239\n",
      "63 : 281\n",
      "64 : 193\n",
      "65 : 155\n",
      "66 : 268\n",
      "67 : 137\n",
      "68 : 296\n",
      "69 : 304\n",
      "70 : 293\n",
      "71 : 290\n",
      "72 : 339\n",
      "73 : 349\n",
      "74 : 298\n",
      "75 : 213\n",
      "76 : 262\n",
      "77 : 223\n",
      "78 : 238\n",
      "79 : 159\n",
      "80 : 175\n",
      "81 : 197\n",
      "82 : 170\n",
      "83 : 188\n",
      "84 : 200\n",
      "85 : 334\n",
      "86 : 241\n",
      "87 : 237\n",
      "88 : 209\n",
      "89 : 141\n",
      "90 : 263\n",
      "91 : 298\n",
      "92 : 301\n",
      "93 : 303\n",
      "94 : 289\n",
      "95 : 230\n",
      "96 : 237\n",
      "97 : 193\n",
      "98 : 216\n",
      "99 : 321\n",
      "100 : 327\n",
      "101 : 83\n",
      "102 : 118\n",
      "103 : 104\n",
      "104 : 87\n",
      "105 : 116\n",
      "106 : 136\n",
      "107 : 144\n",
      "108 : 113\n",
      "109 : 95\n",
      "110 : 114\n",
      "111 : 70\n",
      "112 : 113\n",
      "113 : 120\n",
      "114 : 74\n",
      "115 : 123\n",
      "116 : 95\n",
      "117 : 122\n",
      "118 : 123\n",
      "119 : 103\n",
      "120 : 118\n",
      "121 : 97\n",
      "122 : 81\n",
      "123 : 83\n",
      "124 : 87\n",
      "125 : 98\n",
      "126 : 290\n",
      "127 : 316\n",
      "128 : 291\n",
      "129 : 313\n",
      "130 : 270\n",
      "131 : 337\n",
      "132 : 311\n",
      "133 : 314\n",
      "134 : 297\n",
      "135 : 275\n",
      "136 : 287\n",
      "137 : 319\n",
      "138 : 212\n",
      "139 : 259\n",
      "140 : 229\n",
      "141 : 269\n",
      "142 : 94\n",
      "143 : 168\n",
      "144 : 201\n",
      "145 : 209\n",
      "146 : 271\n",
      "147 : 119\n",
      "148 : 155\n",
      "149 : 145\n",
      "150 : 236\n",
      "151 : 222\n",
      "152 : 246\n",
      "153 : 216\n",
      "154 : 154\n",
      "155 : 197\n",
      "156 : 217\n",
      "157 : 204\n",
      "158 : 206\n",
      "159 : 234\n",
      "160 : 138\n",
      "161 : 171\n",
      "162 : 168\n",
      "163 : 199\n",
      "164 : 193\n",
      "165 : 172\n",
      "166 : 298\n",
      "167 : 336\n",
      "168 : 346\n",
      "169 : 309\n",
      "170 : 275\n",
      "171 : 305\n",
      "172 : 231\n",
      "173 : 180\n",
      "174 : 212\n",
      "175 : 271\n",
      "176 : 203\n",
      "177 : 202\n",
      "178 : 189\n",
      "179 : 290\n",
      "180 : 293\n",
      "181 : 327\n",
      "182 : 307\n",
      "183 : 290\n",
      "184 : 307\n",
      "185 : 333\n",
      "186 : 293\n",
      "187 : 129\n",
      "188 : 184\n",
      "189 : 320\n",
      "190 : 211\n",
      "191 : 245\n",
      "192 : 221\n",
      "193 : 252\n",
      "194 : 287\n",
      "195 : 274\n",
      "196 : 240\n",
      "197 : 257\n",
      "198 : 312\n",
      "199 : 305\n",
      "200 : 298\n",
      "201 : 258\n",
      "202 : 305\n",
      "203 : 213\n",
      "204 : 241\n",
      "205 : 206\n",
      "206 : 242\n",
      "207 : 211\n",
      "208 : 187\n",
      "209 : 248\n",
      "210 : 173\n",
      "211 : 172\n",
      "212 : 309\n",
      "213 : 236\n",
      "214 : 243\n",
      "215 : 205\n",
      "216 : 259\n",
      "217 : 215\n",
      "218 : 250\n",
      "219 : 257\n",
      "220 : 281\n",
      "221 : 318\n",
      "222 : 265\n",
      "223 : 239\n",
      "224 : 284\n",
      "225 : 274\n",
      "226 : 298\n",
      "227 : 260\n",
      "228 : 295\n",
      "229 : 271\n",
      "230 : 308\n",
      "231 : 275\n",
      "232 : 221\n",
      "233 : 168\n",
      "234 : 151\n",
      "235 : 206\n",
      "236 : 201\n",
      "237 : 87\n",
      "238 : 140\n",
      "239 : 236\n",
      "240 : 203\n",
      "241 : 191\n",
      "242 : 193\n",
      "243 : 243\n",
      "244 : 292\n",
      "245 : 279\n",
      "246 : 263\n",
      "247 : 283\n",
      "248 : 248\n",
      "249 : 233\n",
      "250 : 263\n",
      "251 : 252\n",
      "252 : 236\n",
      "253 : 265\n",
      "254 : 261\n",
      "255 : 198\n",
      "256 : 265\n",
      "257 : 300\n",
      "258 : 226\n",
      "259 : 196\n",
      "260 : 315\n",
      "261 : 325\n",
      "262 : 346\n",
      "263 : 308\n",
      "264 : 237\n",
      "265 : 277\n",
      "266 : 277\n",
      "267 : 225\n",
      "268 : 193\n",
      "269 : 225\n",
      "270 : 215\n",
      "271 : 218\n",
      "272 : 211\n",
      "273 : 307\n",
      "274 : 259\n",
      "275 : 269\n",
      "276 : 307\n",
      "277 : 250\n",
      "278 : 206\n",
      "279 : 265\n",
      "280 : 287\n",
      "281 : 278\n",
      "282 : 194\n",
      "283 : 275\n",
      "284 : 267\n",
      "285 : 277\n",
      "286 : 309\n",
      "287 : 284\n",
      "288 : 277\n",
      "289 : 286\n",
      "290 : 259\n",
      "291 : 281\n",
      "292 : 274\n",
      "293 : 249\n",
      "294 : 273\n",
      "295 : 275\n",
      "296 : 246\n",
      "297 : 206\n",
      "\n",
      "- 9.Saving probes to: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/final_probes/filtered_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "kept_records, kept_size_dic = Check_Probes(pb_records, pb_lists, barcode_scheme, master_dir, \n",
    "                                        fwd_primer=fprimer, rev_primer=rprimer, barcode_source=barcode_source,\n",
    "                                        stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "68644"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(kept_records)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Other Examples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pbde = ld.pb_reports_class()\n",
    "pbde.load_pbr(r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/EMT/reports-20000/ZO-1/reg_26.pbr')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SeqRecord(seq=Seq('CGCAAACTGGTGCGGAAGGCGGGTCACTTACTAAGGCGCGAACCGGTACATGAC...TGG', SingleLetterAlphabet()), id=\"chrX:15574402-15594402_reg_282_gene_ACE2_pb_11948_primer_[6,11]_barcodes_[['NDB_282','NDB_282'],'Stv_22','Stv_50']_color_0\", name=\"chrX:15574402-15594402_reg_282_gene_ACE2_pb_11948_primer_[6,11]_barcodes_[['NDB_282','NDB_282'],'Stv_22','Stv_50']_color_0\", description=\"chrX:15574402-15594402_reg_282_gene_ACE2_pb_11948_primer_[6,11]_barcodes_[['NDB_282','NDB_282'],'Stv_22','Stv_50']_color_0\", dbxrefs=[])"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "kept_records[600]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
