{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "0691c309",
   "metadata": {},
   "source": [
    "# Design CTP-14 human genome DNA-MERFISH library \n",
    "\n",
    "by Pu Zheng\n",
    "\n",
    "2022.10.14\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "137597b3",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3b459614",
   "metadata": {},
   "source": [
    "<a id='0'></a>\n",
    "# 0 Minimum required packages and settings"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4846ea43",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "id": "99a843b8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11860\n"
     ]
    }
   ],
   "source": [
    "%run \"..\\..\\Startup_py3.py\"\n",
    "sys.path.append(r\"..\\..\\..\\..\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "print(os.getpid())\n",
    "\n",
    "# library design specific tools\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bd4cf4d8",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "id": "d1426dad",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Use pool_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\n"
     ]
    }
   ],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain'\n",
    "if not os.path.exists(pool_folder):\n",
    "    print(f\"Create pool_folder: {pool_folder}\")\n",
    "    os.makedirs(pool_folder)\n",
    "else:\n",
    "    print(f\"Use pool_folder: {pool_folder}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "id": "714f9640",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\n"
     ]
    }
   ],
   "source": [
    "resolution = 0\n",
    "flanking = 10000\n",
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'human_genome_library')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "217df8b4",
   "metadata": {},
   "source": [
    "<a id='1.1'></a>\n",
    "## 1.1 select regions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "980fa9c6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- load sequence: 1, size=248956422\n",
      "-- load sequence: 10, size=133797422\n",
      "-- load sequence: 11, size=135086622\n",
      "-- load sequence: 12, size=133275309\n",
      "-- load sequence: 13, size=114364328\n",
      "-- load sequence: 14, size=107043718\n",
      "-- load sequence: 15, size=101991189\n",
      "-- load sequence: 16, size=90338345\n",
      "-- load sequence: 17, size=83257441\n",
      "-- load sequence: 18, size=80373285\n",
      "-- load sequence: 19, size=58617616\n",
      "-- load sequence: 2, size=242193529\n",
      "-- load sequence: 20, size=64444167\n",
      "-- load sequence: 21, size=46709983\n",
      "-- load sequence: 22, size=50818468\n",
      "-- load sequence: 3, size=198295559\n",
      "-- load sequence: 4, size=190214555\n",
      "-- load sequence: 5, size=181538259\n",
      "-- load sequence: 6, size=170805979\n",
      "-- load sequence: 7, size=159345973\n",
      "-- load sequence: 8, size=145138636\n",
      "-- load sequence: 9, size=138394717\n",
      "-- load sequence: MT, size=16569\n",
      "-- load sequence: X, size=156040895\n",
      "-- load sequence: Y, size=57227415\n",
      "-- load sequence: KI270728.1, size=1872759\n",
      "-- load sequence: KI270727.1, size=448248\n",
      "-- load sequence: KI270442.1, size=392061\n",
      "-- load sequence: KI270729.1, size=280839\n",
      "-- load sequence: GL000225.1, size=211173\n",
      "-- load sequence: KI270743.1, size=210658\n",
      "-- load sequence: GL000008.2, size=209709\n",
      "-- load sequence: GL000009.2, size=201709\n",
      "-- load sequence: KI270747.1, size=198735\n",
      "-- load sequence: KI270722.1, size=194050\n",
      "-- load sequence: GL000194.1, size=191469\n",
      "-- load sequence: KI270742.1, size=186739\n",
      "-- load sequence: GL000205.2, size=185591\n",
      "-- load sequence: GL000195.1, size=182896\n",
      "-- load sequence: KI270736.1, size=181920\n",
      "-- load sequence: KI270733.1, size=179772\n",
      "-- load sequence: GL000224.1, size=179693\n",
      "-- load sequence: GL000219.1, size=179198\n",
      "-- load sequence: KI270719.1, size=176845\n",
      "-- load sequence: GL000216.2, size=176608\n",
      "-- load sequence: KI270712.1, size=176043\n",
      "-- load sequence: KI270706.1, size=175055\n",
      "-- load sequence: KI270725.1, size=172810\n",
      "-- load sequence: KI270744.1, size=168472\n",
      "-- load sequence: KI270734.1, size=165050\n",
      "-- load sequence: GL000213.1, size=164239\n",
      "-- load sequence: GL000220.1, size=161802\n",
      "-- load sequence: KI270715.1, size=161471\n",
      "-- load sequence: GL000218.1, size=161147\n",
      "-- load sequence: KI270749.1, size=158759\n",
      "-- load sequence: KI270741.1, size=157432\n",
      "-- load sequence: GL000221.1, size=155397\n",
      "-- load sequence: KI270716.1, size=153799\n",
      "-- load sequence: KI270731.1, size=150754\n",
      "-- load sequence: KI270751.1, size=150742\n",
      "-- load sequence: KI270750.1, size=148850\n",
      "-- load sequence: KI270519.1, size=138126\n",
      "-- load sequence: GL000214.1, size=137718\n",
      "-- load sequence: KI270708.1, size=127682\n",
      "-- load sequence: KI270730.1, size=112551\n",
      "-- load sequence: KI270438.1, size=112505\n",
      "-- load sequence: KI270737.1, size=103838\n",
      "-- load sequence: KI270721.1, size=100316\n",
      "-- load sequence: KI270738.1, size=99375\n",
      "-- load sequence: KI270748.1, size=93321\n",
      "-- load sequence: KI270435.1, size=92983\n",
      "-- load sequence: GL000208.1, size=92689\n",
      "-- load sequence: KI270538.1, size=91309\n",
      "-- load sequence: KI270756.1, size=79590\n",
      "-- load sequence: KI270739.1, size=73985\n",
      "-- load sequence: KI270757.1, size=71251\n",
      "-- load sequence: KI270709.1, size=66860\n",
      "-- load sequence: KI270746.1, size=66486\n",
      "-- load sequence: KI270753.1, size=62944\n",
      "-- load sequence: KI270589.1, size=44474\n",
      "-- load sequence: KI270726.1, size=43739\n",
      "-- load sequence: KI270735.1, size=42811\n",
      "-- load sequence: KI270711.1, size=42210\n",
      "-- load sequence: KI270745.1, size=41891\n",
      "-- load sequence: KI270714.1, size=41717\n",
      "-- load sequence: KI270732.1, size=41543\n",
      "-- load sequence: KI270713.1, size=40745\n",
      "-- load sequence: KI270754.1, size=40191\n",
      "-- load sequence: KI270710.1, size=40176\n",
      "-- load sequence: KI270717.1, size=40062\n",
      "-- load sequence: KI270724.1, size=39555\n",
      "-- load sequence: KI270720.1, size=39050\n",
      "-- load sequence: KI270723.1, size=38115\n",
      "-- load sequence: KI270718.1, size=38054\n",
      "-- load sequence: KI270317.1, size=37690\n",
      "-- load sequence: KI270740.1, size=37240\n",
      "-- load sequence: KI270755.1, size=36723\n",
      "-- load sequence: KI270707.1, size=32032\n",
      "-- load sequence: KI270579.1, size=31033\n",
      "-- load sequence: KI270752.1, size=27745\n",
      "-- load sequence: KI270512.1, size=22689\n",
      "-- load sequence: KI270322.1, size=21476\n",
      "-- load sequence: GL000226.1, size=15008\n",
      "-- load sequence: KI270311.1, size=12399\n",
      "-- load sequence: KI270366.1, size=8320\n",
      "-- load sequence: KI270511.1, size=8127\n",
      "-- load sequence: KI270448.1, size=7992\n",
      "-- load sequence: KI270521.1, size=7642\n",
      "-- load sequence: KI270581.1, size=7046\n",
      "-- load sequence: KI270582.1, size=6504\n",
      "-- load sequence: KI270515.1, size=6361\n",
      "-- load sequence: KI270588.1, size=6158\n",
      "-- load sequence: KI270591.1, size=5796\n",
      "-- load sequence: KI270522.1, size=5674\n",
      "-- load sequence: KI270507.1, size=5353\n",
      "-- load sequence: KI270590.1, size=4685\n",
      "-- load sequence: KI270584.1, size=4513\n",
      "-- load sequence: KI270320.1, size=4416\n",
      "-- load sequence: KI270382.1, size=4215\n",
      "-- load sequence: KI270468.1, size=4055\n",
      "-- load sequence: KI270467.1, size=3920\n",
      "-- load sequence: KI270362.1, size=3530\n",
      "-- load sequence: KI270517.1, size=3253\n",
      "-- load sequence: KI270593.1, size=3041\n",
      "-- load sequence: KI270528.1, size=2983\n",
      "-- load sequence: KI270587.1, size=2969\n",
      "-- load sequence: KI270364.1, size=2855\n",
      "-- load sequence: KI270371.1, size=2805\n",
      "-- load sequence: KI270333.1, size=2699\n",
      "-- load sequence: KI270374.1, size=2656\n",
      "-- load sequence: KI270411.1, size=2646\n",
      "-- load sequence: KI270414.1, size=2489\n",
      "-- load sequence: KI270510.1, size=2415\n",
      "-- load sequence: KI270390.1, size=2387\n",
      "-- load sequence: KI270375.1, size=2378\n",
      "-- load sequence: KI270420.1, size=2321\n",
      "-- load sequence: KI270509.1, size=2318\n",
      "-- load sequence: KI270315.1, size=2276\n",
      "-- load sequence: KI270302.1, size=2274\n",
      "-- load sequence: KI270518.1, size=2186\n",
      "-- load sequence: KI270530.1, size=2168\n",
      "-- load sequence: KI270304.1, size=2165\n",
      "-- load sequence: KI270418.1, size=2145\n",
      "-- load sequence: KI270424.1, size=2140\n",
      "-- load sequence: KI270417.1, size=2043\n",
      "-- load sequence: KI270508.1, size=1951\n",
      "-- load sequence: KI270303.1, size=1942\n",
      "-- load sequence: KI270381.1, size=1930\n",
      "-- load sequence: KI270529.1, size=1899\n",
      "-- load sequence: KI270425.1, size=1884\n",
      "-- load sequence: KI270396.1, size=1880\n",
      "-- load sequence: KI270363.1, size=1803\n",
      "-- load sequence: KI270386.1, size=1788\n",
      "-- load sequence: KI270465.1, size=1774\n",
      "-- load sequence: KI270383.1, size=1750\n",
      "-- load sequence: KI270384.1, size=1658\n",
      "-- load sequence: KI270330.1, size=1652\n",
      "-- load sequence: KI270372.1, size=1650\n",
      "-- load sequence: KI270548.1, size=1599\n",
      "-- load sequence: KI270580.1, size=1553\n",
      "-- load sequence: KI270387.1, size=1537\n",
      "-- load sequence: KI270391.1, size=1484\n",
      "-- load sequence: KI270305.1, size=1472\n",
      "-- load sequence: KI270373.1, size=1451\n",
      "-- load sequence: KI270422.1, size=1445\n",
      "-- load sequence: KI270316.1, size=1444\n",
      "-- load sequence: KI270340.1, size=1428\n",
      "-- load sequence: KI270338.1, size=1428\n",
      "-- load sequence: KI270583.1, size=1400\n",
      "-- load sequence: KI270334.1, size=1368\n",
      "-- load sequence: KI270429.1, size=1361\n",
      "-- load sequence: KI270393.1, size=1308\n",
      "-- load sequence: KI270516.1, size=1300\n",
      "-- load sequence: KI270389.1, size=1298\n",
      "-- load sequence: KI270466.1, size=1233\n",
      "-- load sequence: KI270388.1, size=1216\n",
      "-- load sequence: KI270544.1, size=1202\n",
      "-- load sequence: KI270310.1, size=1201\n",
      "-- load sequence: KI270412.1, size=1179\n",
      "-- load sequence: KI270395.1, size=1143\n",
      "-- load sequence: KI270376.1, size=1136\n",
      "-- load sequence: KI270337.1, size=1121\n",
      "-- load sequence: KI270335.1, size=1048\n",
      "-- load sequence: KI270378.1, size=1048\n",
      "-- load sequence: KI270379.1, size=1045\n",
      "-- load sequence: KI270329.1, size=1040\n",
      "-- load sequence: KI270419.1, size=1029\n",
      "-- load sequence: KI270336.1, size=1026\n",
      "-- load sequence: KI270312.1, size=998\n",
      "-- load sequence: KI270539.1, size=993\n",
      "-- load sequence: KI270385.1, size=990\n",
      "-- load sequence: KI270423.1, size=981\n",
      "-- load sequence: KI270392.1, size=971\n",
      "-- load sequence: KI270394.1, size=970\n",
      "- 194 sequences loaded.\n"
     ]
    }
   ],
   "source": [
    "# prepare sequence reader\n",
    "reload(ia.library_tools.sequences)\n",
    "seq_rd = ia.library_tools.sequences.sequence_reader(genome_folder)\n",
    "seq_rd.load_ref_sequences() # load genomic sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "5a5d3385",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3088269832\n"
     ]
    }
   ],
   "source": [
    "# calculate total length\n",
    "min_chr_len = 2000000 # ONLY taking care of chromosomes longer than 1Mb\n",
    "chr_len_dict = {_name:len(_record) for _name, _record in seq_rd.ref_seq_dict.items() if len(_record) > min_chr_len}\n",
    "total_len = sum(list(chr_len_dict.values()))\n",
    "print(total_len)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1904a504",
   "metadata": {},
   "source": [
    "## select resolution for each chromosome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "5859125b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading from file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\chr_2_design_region_num.pkl\n",
      "Loading from file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\chr_2_design_resolution.pkl\n"
     ]
    }
   ],
   "source": [
    "overwrite = False\n",
    "region_num_filename = os.path.join(library_folder, 'chr_2_design_region_num.pkl')\n",
    "resolution_filename = os.path.join(library_folder, 'chr_2_design_resolution.pkl')\n",
    "\n",
    "# if both files exists, directly load\n",
    "if os.path.exists(region_num_filename) and os.path.exists(resolution_filename) and not overwrite:\n",
    "    print(f\"Loading from file: {region_num_filename}\")\n",
    "    chr_2_region_num = pickle.load(open(region_num_filename, 'rb'))\n",
    "    print(f\"Loading from file: {resolution_filename}\")\n",
    "    chr_2_resolution = pickle.load(open(resolution_filename, 'rb'))\n",
    "else:\n",
    "    # Selecte regions in an adaptive manner to gaurentee \n",
    "    allowed_resolutions = (np.arange(25,0,-5) * 1e5).astype(np.int32)\n",
    "    min_chr_region_num = 40\n",
    "    chr_2_region_num = {}\n",
    "    chr_2_resolution = {}\n",
    "    for _name, _len in sorted(chr_len_dict.items(), key=lambda v:v[1]): \n",
    "        # iterative\n",
    "        for _resolution in allowed_resolutions:\n",
    "            # select the shortest chromosome first\n",
    "            _curr_num = np.floor(_len/_resolution).astype(np.int32)\n",
    "            if _curr_num >= min_chr_region_num:\n",
    "                chr_2_region_num[_name] = _curr_num\n",
    "                chr_2_resolution[_name] = _resolution\n",
    "                print(_name, _resolution, _curr_num)\n",
    "                min_chr_region_num = max(_curr_num * 0.8, min_chr_region_num)\n",
    "                break    \n",
    "    print(np.sum(list(chr_2_region_num.values())))\n",
    "    # save\n",
    "    print(f\"Saving to file: {region_num_filename}\")\n",
    "    pickle.dump(chr_2_region_num, open(region_num_filename, 'wb'))\n",
    "    print(f\"Saving to file: {resolution_filename}\")\n",
    "    pickle.dump(chr_2_resolution, open(resolution_filename, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b8d42f5b",
   "metadata": {},
   "source": [
    "## For each chromosome, calculate region center:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "16b3b917",
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3.structure_tools.distance import sort_chr\n",
    "region_size = 30000 # extract 30kb around the targets"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "ea7a5258",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading from file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\region_dicts.pkl\n"
     ]
    }
   ],
   "source": [
    "region_dict_savefile = os.path.join(sequence_folder, 'region_dicts.pkl')\n",
    "if os.path.exists(region_dict_savefile) and not overwrite:\n",
    "    print(f\"Loading from file: {region_dict_savefile}\")\n",
    "    region_dicts = pickle.load(open(region_dict_savefile, 'rb'))\n",
    "else:\n",
    "    region_seqs = []\n",
    "    region_dicts = []\n",
    "    region_id = 0\n",
    "    for _chr in sorted(chr_2_region_num.keys(), key=lambda _v:sort_chr(_v)):\n",
    "        print(_chr)\n",
    "        _region_num = chr_2_region_num[_chr]\n",
    "        _resolution = chr_2_resolution[_chr]\n",
    "        # get region center\n",
    "        _reg_centers = (np.arange(0.5,_region_num+0.5, 1) * _resolution ).astype(np.int32)\n",
    "        _reg_starts = _reg_centers - int(region_size/2) + 1\n",
    "        _reg_ends = _reg_centers + int(region_size/2)\n",
    "        _seqs = [seq_rd.ref_seq_dict[_chr][_s:_e] for _s,_e in zip(_reg_starts, _reg_ends)]\n",
    "        region_seqs.extend(_seqs)\n",
    "\n",
    "        for _s,_e in zip(_reg_starts, _reg_ends):\n",
    "            _reg_dict = {\n",
    "                'Chr':_chr,\n",
    "                'Start':_s,\n",
    "                'End':_e,\n",
    "                'Region':f\"{_chr}:{_s}-{_e}\",\n",
    "                'Strand':'+',\n",
    "                'Gene': region_id,\n",
    "            }\n",
    "            region_dicts.append(_reg_dict)\n",
    "            region_id += 1\n",
    "    # save\n",
    "    print(f\"Saving to file: {region_dict_savefile}\")\n",
    "    pickle.dump(region_dicts, open(region_dict_savefile, 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "0ab776c8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- searching among 24 references\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n"
     ]
    }
   ],
   "source": [
    "seqs = seq_rd.find_sequence_for_region(region_dicts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "2970dec8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\0_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\2_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\3_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\4_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\5_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\6_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\7_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\8_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\9_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\10_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\11_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\12_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\13_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\14_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\15_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\16_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\17_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\18_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\19_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\20_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\21_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\22_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\23_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\24_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\25_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\26_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\27_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\28_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\29_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\30_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\31_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\32_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\33_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\34_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\35_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\36_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\37_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\38_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\39_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\40_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\41_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\42_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\43_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\44_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\45_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\46_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\47_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\48_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\49_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\50_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\51_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\52_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\53_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\54_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\55_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\56_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\57_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\58_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\59_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\60_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\61_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\62_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\63_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\64_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\65_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\66_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\67_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\68_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\69_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\70_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\71_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\72_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\73_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\74_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\75_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\76_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\77_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\78_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\79_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\80_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\81_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\82_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\83_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\84_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\85_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\86_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\87_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\88_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\89_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\90_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\91_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\92_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\93_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\94_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\95_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\96_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\97_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\98_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\99_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\100_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\101_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\102_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\103_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\104_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\105_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\106_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\107_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\108_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\109_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\110_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\111_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\112_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\113_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\114_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\115_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\116_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\117_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\118_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\119_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\120_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\121_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\122_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\123_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\124_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\125_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\126_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\127_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\128_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\129_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\130_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\131_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\132_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\133_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\134_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\135_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\136_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\137_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\138_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\139_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\140_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\141_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\142_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\143_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\144_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\145_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\146_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\147_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\148_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\149_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\150_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\151_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\152_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\153_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\154_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\155_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\156_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\157_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\158_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\159_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\160_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\161_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\162_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\163_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\164_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\165_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\166_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\167_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\168_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\169_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\170_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\171_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\172_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\173_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\174_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\175_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\176_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\177_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\178_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\179_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\180_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\181_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\182_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\183_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\184_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\185_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\186_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\187_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\188_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\189_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\190_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\191_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\192_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\193_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\194_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\195_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\196_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\197_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\198_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\199_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\200_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\201_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\202_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\203_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\204_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\205_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\206_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\207_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\208_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\209_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\210_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\211_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\212_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\213_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\214_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\215_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\216_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\217_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\218_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\219_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\220_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\221_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\222_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\223_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\224_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\225_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\226_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\227_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\228_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\229_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\230_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\231_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\232_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\233_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\234_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\235_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\236_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\237_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\238_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\239_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\240_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\241_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\242_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\243_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\244_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\245_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\246_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\247_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\248_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\249_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\250_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\251_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\252_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\253_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\254_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\255_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\256_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\257_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\258_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\259_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\260_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\261_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\262_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\263_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\264_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\265_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\266_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\267_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\268_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\269_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\270_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\271_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\272_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\273_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\274_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\275_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\276_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\277_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\278_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\279_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\280_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\281_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\282_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\283_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\284_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\285_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\286_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\287_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\288_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\289_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\290_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\291_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\292_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\293_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\294_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\295_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\296_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\297_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\298_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\299_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\300_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\301_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\302_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\303_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\304_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\305_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\306_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\307_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\308_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\309_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\310_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\311_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\312_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\313_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\314_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\315_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\316_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\317_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\318_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\319_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\320_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\321_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\322_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\323_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\324_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\325_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\326_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\327_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\328_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\329_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\330_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\331_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\332_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\333_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\334_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\335_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\336_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\337_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\338_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\339_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\340_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\341_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\342_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\343_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\344_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\345_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\346_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\347_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\348_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\349_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\350_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\351_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\352_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\353_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\354_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\355_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\356_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\357_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\358_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\359_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\360_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\361_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\362_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\363_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\364_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\365_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\366_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\367_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\368_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\369_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\370_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\371_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\372_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\373_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\374_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\375_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\376_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\377_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\378_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\379_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\380_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\381_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\382_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\383_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\384_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\385_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\386_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\387_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\388_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\389_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\390_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\391_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\392_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\393_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\394_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\395_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\396_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\397_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\398_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\399_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\400_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\401_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\402_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\403_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\404_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\405_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\406_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\407_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\408_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\409_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\410_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\411_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\412_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\413_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\414_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\415_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\416_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\417_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\418_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\419_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\420_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\421_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\422_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\423_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\424_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\425_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\426_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\427_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\428_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\429_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\430_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\431_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\432_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\433_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\434_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\435_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\436_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\437_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\438_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\439_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\440_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\441_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\442_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\443_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\444_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\445_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\446_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\447_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\448_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\449_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\450_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\451_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\452_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\453_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\454_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\455_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\456_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\457_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\458_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\459_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\460_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\461_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\462_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\463_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\464_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\465_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\466_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\467_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\468_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\469_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\470_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\471_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\472_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\473_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\474_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\475_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\476_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\477_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\478_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\479_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\480_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\481_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\482_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\483_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\484_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\485_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\486_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\487_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\488_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\489_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\490_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\491_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\492_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\493_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\494_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\495_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\496_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\497_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\498_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\499_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\500_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\501_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\502_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\503_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\504_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\505_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\506_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\507_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\508_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\509_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\510_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\511_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\512_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\513_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\514_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\515_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\516_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\517_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\518_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\519_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\520_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\521_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\522_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\523_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\524_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\525_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\526_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\527_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\528_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\529_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\530_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\531_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\532_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\533_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\534_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\535_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\536_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\537_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\538_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\539_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\540_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\541_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\542_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\543_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\544_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\545_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\546_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\547_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\548_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\549_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\550_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\551_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\552_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\553_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\554_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\555_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\556_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\557_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\558_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\559_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\560_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\561_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\562_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\563_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\564_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\565_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\566_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\567_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\568_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\569_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\570_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\571_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\572_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\573_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\574_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\575_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\576_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\577_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\578_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\579_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\580_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\581_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\582_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\583_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\584_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\585_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\586_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\587_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\588_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\589_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\590_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\591_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\592_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\593_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\594_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\595_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\596_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\597_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\598_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\599_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\600_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\601_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\602_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\603_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\604_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\605_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\606_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\607_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\608_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\609_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\610_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\611_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\612_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\613_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\614_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\615_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\616_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\617_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\618_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\619_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\620_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\621_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\622_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\623_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\624_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\625_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\626_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\627_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\628_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\629_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\630_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\631_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\632_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\633_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\634_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\635_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\636_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\637_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\638_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\639_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\640_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\641_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\642_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\643_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\644_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\645_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\646_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\647_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\648_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\649_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\650_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\651_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\652_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\653_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\654_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\655_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\656_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\657_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\658_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\659_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\660_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\661_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\662_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\663_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\664_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\665_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\666_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\667_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\668_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\669_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\670_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\671_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\672_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\673_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\674_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\675_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\676_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\677_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\678_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\679_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\680_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\681_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\682_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\683_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\684_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\685_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\686_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\687_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\688_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\689_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\690_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\691_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\692_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\693_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\694_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\695_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\696_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\697_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\698_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\699_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\700_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\701_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\702_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\703_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\704_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\705_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\706_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\707_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\708_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\709_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\710_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\711_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\712_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\713_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\714_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\715_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\716_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\717_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\718_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\719_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\720_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\721_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\722_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\723_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\724_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\725_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\726_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\727_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\728_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\729_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\730_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\731_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\732_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\733_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\734_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\735_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\736_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\737_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\738_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\739_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\740_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\741_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\742_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\743_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\744_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\745_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\746_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\747_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\748_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\749_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\750_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\751_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\752_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\753_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\754_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\755_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\756_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\757_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\758_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\759_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\760_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\761_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\762_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\763_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\764_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\765_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\766_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\767_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\768_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\769_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\770_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\771_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\772_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\773_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\774_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\775_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\776_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\777_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\778_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\779_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\780_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\781_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\782_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\783_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\784_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\785_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\786_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\787_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\788_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\789_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\790_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\791_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\792_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\793_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\794_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\795_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\796_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\797_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\798_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\799_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\800_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\801_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\802_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\803_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\804_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\805_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\806_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\807_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\808_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\809_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\810_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\811_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\812_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\813_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\814_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\815_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\816_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\817_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\818_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\819_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\820_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\821_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\822_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\823_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\824_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\825_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\826_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\827_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\828_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\829_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\830_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\831_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\832_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\833_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\834_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\835_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\836_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\837_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\838_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\839_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\840_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\841_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\842_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\843_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\844_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\845_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\846_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\847_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\848_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\849_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\850_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\851_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\852_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\853_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\854_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\855_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\856_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\857_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\858_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\859_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\860_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\861_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\862_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\863_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\864_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\865_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\866_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\867_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\868_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\869_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\870_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\871_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\872_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\873_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\874_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\875_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\876_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\877_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\878_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\879_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\880_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\881_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\882_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\883_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\884_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\885_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\886_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\887_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\888_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\889_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\890_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\891_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\892_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\893_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\894_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\895_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\896_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\897_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\898_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\899_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\900_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\901_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\902_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\903_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\904_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\905_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\906_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\907_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\908_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\909_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\910_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\911_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\912_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\913_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\914_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\915_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\916_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\917_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\918_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\919_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\920_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\921_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\922_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\923_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\924_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\925_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\926_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\927_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\928_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\929_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\930_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\931_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\932_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\933_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\934_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\935_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\936_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\937_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\938_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\939_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\940_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\941_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\942_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\943_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\944_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\945_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\946_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\947_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\948_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\949_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\950_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\951_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\952_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\953_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\954_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\955_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\956_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\957_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\958_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\959_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\960_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\961_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\962_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\963_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\964_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\965_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\966_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\967_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\968_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\969_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\970_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\971_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\972_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\973_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\974_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\975_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\976_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\977_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\978_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\979_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\980_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\981_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\982_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\983_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\984_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\985_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\986_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\987_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\988_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\989_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\990_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\991_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\992_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\993_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\994_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\995_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\996_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\997_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\998_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\999_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1000_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1001_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1002_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1003_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1004_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1005_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1006_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1007_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1008_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1009_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1010_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1011_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1012_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1013_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1014_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1015_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1016_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1017_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1018_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1019_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1020_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1021_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1022_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1023_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1024_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1025_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1026_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1027_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1028_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1029_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1030_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1031_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1032_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1033_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1034_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1035_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1036_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1037_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1038_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1039_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1040_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1041_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1042_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1043_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1044_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1045_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1046_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1047_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1048_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1049_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1050_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1051_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1052_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1053_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1054_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1055_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1056_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1057_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1058_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1059_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1060_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1061_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1062_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1063_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1064_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1065_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1066_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1067_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1068_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1069_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1070_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1071_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1072_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1073_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1074_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1075_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1076_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1077_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1078_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1079_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1080_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1081_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1082_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1083_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1084_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1085_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1086_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1087_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1088_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1089_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1090_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1091_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1092_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1093_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1094_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1095_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1096_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1097_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1098_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1099_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1100_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1101_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1102_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1103_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1104_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1105_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1106_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1107_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1108_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1109_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1110_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1111_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1112_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1113_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1114_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1115_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1116_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1117_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1118_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1119_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1120_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1121_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1122_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1123_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1124_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1125_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1126_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1127_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1128_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1129_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1130_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1131_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1132_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1133_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1134_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1135_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1136_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1137_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1138_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1139_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1140_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1141_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1142_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1143_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1144_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1145_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1146_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1147_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1148_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1149_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1150_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1151_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1152_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1153_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1154_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1155_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1156_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1157_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1158_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1159_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1160_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1161_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1162_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1163_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1164_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1165_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1166_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1167_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1168_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1169_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1170_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1171_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1172_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1173_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1174_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1175_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1176_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1177_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1178_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1179_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1180_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1181_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1182_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1183_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1184_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1185_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1186_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1187_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1188_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1189_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1190_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1191_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1192_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1193_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1194_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1195_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1196_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1197_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1198_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1199_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1200_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1201_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1202_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1203_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1204_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1205_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1206_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1207_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1208_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1209_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1210_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1211_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1212_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1213_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1214_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1215_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1216_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1217_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1218_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1219_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1220_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1221_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1222_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1223_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1224_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1225_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1226_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1227_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1228_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1229_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1230_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1231_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1232_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1233_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1234_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1235_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1236_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1237_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1238_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1239_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1240_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1241_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1242_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1243_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1244_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1245_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1246_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1247_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1248_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1249_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1250_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1251_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1252_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1253_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1254_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1255_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1256_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1257_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1258_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1259_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1260_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1261_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1262_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1263_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1264_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1265_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1266_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1267_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1268_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1269_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1270_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1271_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1272_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1273_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1274_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1275_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1276_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1277_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1278_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1279_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1280_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1281_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1282_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1283_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1284_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1285_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1286_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1287_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1288_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1289_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1290_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1291_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1292_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1293_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1294_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1295_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1296_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1297_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1298_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1299_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1300_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1301_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1302_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1303_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1304_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1305_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1306_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1307_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1308_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1309_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1310_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1311_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1312_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1313_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1314_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1315_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1316_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1317_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1318_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1319_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1320_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1321_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1322_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1323_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1324_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1325_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1326_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1327_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1328_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1329_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1330_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1331_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1332_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1333_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1334_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1335_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1336_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1337_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1338_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1339_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1340_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1341_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1342_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1343_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1344_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1345_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1346_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1347_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1348_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1349_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1350_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1351_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1352_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1353_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1354_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1355_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1356_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1357_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1358_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1359_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1360_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1361_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1362_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1363_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1364_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1365_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1366_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1367_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1368_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1369_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1370_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1371_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1372_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1373_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1374_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1375_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1376_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1377_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1378_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1379_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1380_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1381_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1382_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1383_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1384_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1385_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1386_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1387_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1388_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1389_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1390_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1391_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1392_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1393_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1394_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1395_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1396_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1397_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1398_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1399_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1400_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1401_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1402_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1403_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1404_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1405_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1406_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1407_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1408_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1409_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1410_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1411_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1412_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1413_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1414_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1415_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1416_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1417_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1418_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1419_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1420_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1421_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1422_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1423_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1424_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1425_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1426_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1427_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1428_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1429_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1430_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1431_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1432_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1433_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1434_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1435_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1436_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1437_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1438_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1439_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1440_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1441_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1442_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1443_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1444_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1445_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1446_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1447_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1448_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1449_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1450_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1451_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1452_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1453_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1454_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1455_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1456_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1457_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1458_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1459_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1460_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1461_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1462_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1463_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1464_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1465_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1466_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1467_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1468_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1469_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1470_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1471_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1472_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1473_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1474_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1475_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1476_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1477_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1478_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1479_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1480_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1481_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1482_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1483_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1484_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1485_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1486_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1487_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1488_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1489_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1490_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1491_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1492_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1493_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1494_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1495_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1496_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1497_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1498_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1499_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1500_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1501_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1502_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1503_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1504_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1505_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1506_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1507_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1508_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1509_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1510_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1511_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1512_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1513_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1514_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1515_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1516_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1517_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1518_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1519_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1520_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1521_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1522_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1523_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1524_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1525_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1526_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1527_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1528_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1529_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1530_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1531_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1532_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1533_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1534_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1535_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1536_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1537_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1538_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1539_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1540_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1541_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1542_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1543_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1544_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1545_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1546_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1547_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1548_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1549_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1550_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1551_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1552_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1553_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1554_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1555_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1556_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1557_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1558_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1559_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1560_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1561_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1562_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1563_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1564_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1565_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1566_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1567_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1568_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1569_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1570_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1571_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1572_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1573_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1574_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1575_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1576_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1577_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1578_reg_0.fasta\n"
     ]
    }
   ],
   "source": [
    "seq_rd.save_sequences(save_folder=sequence_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fb106a4a",
   "metadata": {},
   "source": [
    "<a id='2'></a>\n",
    "# 2. Design probe targeting sequences by probe_designer\n",
    "\n",
    "run probe_designer, remeber to clear the memory usage because each table should take ~32GB"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cfad443e",
   "metadata": {},
   "source": [
    "<a id='2.1'></a>\n",
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires hg38 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "414635fc",
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "85dda5fb",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "35c7e4f5",
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3 import library_tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "dc089b78",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\hg38_genome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'hg38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {genome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "29731c46",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "88940c0a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\hg38_transcriptome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'hg38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {transcriptome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0b477fab",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "dac1a780",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\Repbase_v2603_repeat_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {repeat_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "688a066f",
   "metadata": {},
   "source": [
    "<a id='2.2'></a>\n",
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "e7c4541c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1579 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'hg38_genome_17w.npy')\n",
    "transcriptome_index = os.path.join(reference_folder, 'hg38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "#ref_merfish_index = os.path.join(reference_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "# get input files \n",
    "input_files = glob.glob(os.path.join(sequence_folder, '*.fasta'))\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "probe_savefile = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "print(probe_savefile)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "722e6b4b",
   "metadata": {},
   "source": [
    "### create pb_designer class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "id": "f3c242de",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_complement (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 1579\n",
      "    - save_file location: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr\n",
      "\n",
      "- Fail to load from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr, file doesn't exist.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             #'ref_merfish':{'file':ref_merfish_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=probe_savefile,\n",
    "    params_dic={'word_size':17,'pb_len':42,'buffer_len':2,'max_count':2**16-1,\n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): 25,\n",
    "               'rep_genome': 0,\n",
    "               'transcriptome': 25, # 14\n",
    "               #'ref_merfish': 14,\n",
    "               'gc':[0.25,0.75],'tm': 37+0.62*50+5,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b84dae63",
   "metadata": {},
   "source": [
    "### calculate probe reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "686689a5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 117.838s.\n",
      "-- setting attribute: map_transcriptome\n",
      "--- finish map_transcriptome in 121.144s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 111.222s.\n",
      "Time(s): 350.2070834636688\n",
      "- Designing targeting sequence for 1579 regions\n",
      "-- designing region: 1:1235001-1265000_strand_+_gene_0 -- region: 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\0_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "- Designed 59918 candidate probes in 16.210s.\n",
      "-- designing region: 1:3735001-3765000_strand_+_gene_1 -- region: 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.343s.\n",
      "- Designed 59918 candidate probes in 23.067s.\n",
      "-- designing region: 1:6235001-6265000_strand_+_gene_2 -- region: 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\2_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.335s.\n",
      "- Designed 59918 candidate probes in 19.631s.\n",
      "-- designing region: 1:8735001-8765000_strand_+_gene_3 -- region: 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\3_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.425s.\n",
      "-- designing region: 1:11235001-11265000_strand_+_gene_4 -- region: 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\4_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.494s.\n",
      "-- designing region: 1:13735001-13765000_strand_+_gene_5 -- region: 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\5_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.456s.\n",
      "-- designing region: 1:16235001-16265000_strand_+_gene_6 -- region: 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\6_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.448s.\n",
      "-- designing region: 1:18735001-18765000_strand_+_gene_7 -- region: 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\7_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 19.049s.\n",
      "-- designing region: 1:21235001-21265000_strand_+_gene_8 -- region: 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\8_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.419s.\n",
      "-- designing region: 1:23735001-23765000_strand_+_gene_9 -- region: 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\9_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.416s.\n",
      "-- designing region: 1:26235001-26265000_strand_+_gene_10 -- region: 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\10_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.424s.\n",
      "-- designing region: 1:28735001-28765000_strand_+_gene_11 -- region: 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\11_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.458s.\n",
      "-- designing region: 1:31235001-31265000_strand_+_gene_12 -- region: 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\12_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.379s.\n",
      "-- designing region: 1:33735001-33765000_strand_+_gene_13 -- region: 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\13_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.405s.\n",
      "-- designing region: 1:36235001-36265000_strand_+_gene_14 -- region: 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\14_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.388s.\n",
      "-- designing region: 1:38735001-38765000_strand_+_gene_15 -- region: 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\15_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.395s.\n",
      "-- designing region: 1:41235001-41265000_strand_+_gene_16 -- region: 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\16_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.388s.\n",
      "-- designing region: 1:43735001-43765000_strand_+_gene_17 -- region: 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\17_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.381s.\n",
      "-- designing region: 1:46235001-46265000_strand_+_gene_18 -- region: 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\18_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.379s.\n",
      "-- designing region: 1:48735001-48765000_strand_+_gene_19 -- region: 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\19_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.443s.\n",
      "-- designing region: 1:51235001-51265000_strand_+_gene_20 -- region: 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\20_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.409s.\n",
      "-- designing region: 1:53735001-53765000_strand_+_gene_21 -- region: 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\21_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.431s.\n",
      "-- designing region: 1:56235001-56265000_strand_+_gene_22 -- region: 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\22_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.444s.\n",
      "-- designing region: 1:58735001-58765000_strand_+_gene_23 -- region: 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\23_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 1:61235001-61265000_strand_+_gene_24 -- region: 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\24_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.401s.\n",
      "-- designing region: 1:63735001-63765000_strand_+_gene_25 -- region: 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\25_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.409s.\n",
      "-- designing region: 1:66235001-66265000_strand_+_gene_26 -- region: 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\26_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 1:68735001-68765000_strand_+_gene_27 -- region: 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\27_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.440s.\n",
      "-- designing region: 1:71235001-71265000_strand_+_gene_28 -- region: 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\28_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 16.000s.\n",
      "-- designing region: 1:73735001-73765000_strand_+_gene_29 -- region: 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\29_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 19.696s.\n",
      "-- designing region: 1:76235001-76265000_strand_+_gene_30 -- region: 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\30_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.345s.\n",
      "- Designed 59918 candidate probes in 29.179s.\n",
      "-- designing region: 1:78735001-78765000_strand_+_gene_31 -- region: 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\31_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.258s.\n",
      "- Designed 59918 candidate probes in 18.508s.\n",
      "-- designing region: 1:81235001-81265000_strand_+_gene_32 -- region: 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\32_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 20.955s.\n",
      "-- designing region: 1:83735001-83765000_strand_+_gene_33 -- region: 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\33_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.654s.\n",
      "- Designed 59918 candidate probes in 16.666s.\n",
      "-- designing region: 1:86235001-86265000_strand_+_gene_34 -- region: 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\34_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.516s.\n",
      "-- designing region: 1:88735001-88765000_strand_+_gene_35 -- region: 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\35_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 1:91235001-91265000_strand_+_gene_36 -- region: 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\36_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 1:93735001-93765000_strand_+_gene_37 -- region: 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\37_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.733s.\n",
      "-- designing region: 1:96235001-96265000_strand_+_gene_38 -- region: 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\38_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.311s.\n",
      "- Designed 59918 candidate probes in 15.626s.\n",
      "-- designing region: 1:98735001-98765000_strand_+_gene_39 -- region: 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\39_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 1:101235001-101265000_strand_+_gene_40 -- region: 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\40_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.305s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: 1:103735001-103765000_strand_+_gene_41 -- region: 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\41_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.734s.\n",
      "-- designing region: 1:106235001-106265000_strand_+_gene_42 -- region: 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\42_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 1:108735001-108765000_strand_+_gene_43 -- region: 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\43_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 1:111235001-111265000_strand_+_gene_44 -- region: 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\44_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.674s.\n",
      "-- designing region: 1:113735001-113765000_strand_+_gene_45 -- region: 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\45_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 59918 candidate probes in 15.656s.\n",
      "-- designing region: 1:116235001-116265000_strand_+_gene_46 -- region: 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\46_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.691s.\n",
      "-- designing region: 1:118735001-118765000_strand_+_gene_47 -- region: 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\47_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.892s.\n",
      "-- designing region: 1:121235001-121265000_strand_+_gene_48 -- region: 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\48_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 1:123735001-123765000_strand_+_gene_49 -- region: 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\49_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.222s.\n",
      "- Designed 59918 candidate probes in 15.390s.\n",
      "-- designing region: 1:126235001-126265000_strand_+_gene_50 -- region: 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\50_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.219s.\n",
      "- Designed 0 candidate probes in 0.513s.\n",
      "-- designing region: 1:128735001-128765000_strand_+_gene_51 -- region: 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\51_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.584s.\n",
      "- Designed 0 candidate probes in 0.879s.\n",
      "-- designing region: 1:131235001-131265000_strand_+_gene_52 -- region: 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\52_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.586s.\n",
      "- Designed 0 candidate probes in 0.880s.\n",
      "-- designing region: 1:133735001-133765000_strand_+_gene_53 -- region: 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\53_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.580s.\n",
      "- Designed 0 candidate probes in 0.877s.\n",
      "-- designing region: 1:136235001-136265000_strand_+_gene_54 -- region: 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\54_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.592s.\n",
      "- Designed 0 candidate probes in 0.885s.\n",
      "-- designing region: 1:138735001-138765000_strand_+_gene_55 -- region: 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\55_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.575s.\n",
      "- Designed 0 candidate probes in 0.870s.\n",
      "-- designing region: 1:141235001-141265000_strand_+_gene_56 -- region: 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\56_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.590s.\n",
      "- Designed 0 candidate probes in 0.886s.\n",
      "-- designing region: 1:143735001-143765000_strand_+_gene_57 -- region: 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\57_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.653s.\n",
      "- Designed 59918 candidate probes in 15.938s.\n",
      "-- designing region: 1:146235001-146265000_strand_+_gene_58 -- region: 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\58_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 1:148735001-148765000_strand_+_gene_59 -- region: 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\59_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.584s.\n",
      "-- designing region: 1:151235001-151265000_strand_+_gene_60 -- region: 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\60_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 1:153735001-153765000_strand_+_gene_61 -- region: 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\61_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: 1:156235001-156265000_strand_+_gene_62 -- region: 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\62_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.659s.\n",
      "-- designing region: 1:158735001-158765000_strand_+_gene_63 -- region: 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\63_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 21.260s.\n",
      "-- designing region: 1:161235001-161265000_strand_+_gene_64 -- region: 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\64_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 1:163735001-163765000_strand_+_gene_65 -- region: 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\65_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 1:166235001-166265000_strand_+_gene_66 -- region: 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\66_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.620s.\n",
      "-- designing region: 1:168735001-168765000_strand_+_gene_67 -- region: 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\67_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 1:171235001-171265000_strand_+_gene_68 -- region: 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\68_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 1:173735001-173765000_strand_+_gene_69 -- region: 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\69_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.318s.\n",
      "-- designing region: 1:176235001-176265000_strand_+_gene_70 -- region: 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\70_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 1:178735001-178765000_strand_+_gene_71 -- region: 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\71_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 1:181235001-181265000_strand_+_gene_72 -- region: 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\72_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.658s.\n",
      "-- designing region: 1:183735001-183765000_strand_+_gene_73 -- region: 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\73_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 1:186235001-186265000_strand_+_gene_74 -- region: 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\74_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 1:188735001-188765000_strand_+_gene_75 -- region: 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\75_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 1:191235001-191265000_strand_+_gene_76 -- region: 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\76_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.548s.\n",
      "-- designing region: 1:193735001-193765000_strand_+_gene_77 -- region: 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\77_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 1:196235001-196265000_strand_+_gene_78 -- region: 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\78_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 1:198735001-198765000_strand_+_gene_79 -- region: 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\79_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 1:201235001-201265000_strand_+_gene_80 -- region: 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\80_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 1:203735001-203765000_strand_+_gene_81 -- region: 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\81_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 1:206235001-206265000_strand_+_gene_82 -- region: 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\82_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 1:208735001-208765000_strand_+_gene_83 -- region: 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\83_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 1:211235001-211265000_strand_+_gene_84 -- region: 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\84_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 1:213735001-213765000_strand_+_gene_85 -- region: 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\85_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 1:216235001-216265000_strand_+_gene_86 -- region: 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\86_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 1:218735001-218765000_strand_+_gene_87 -- region: 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\87_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 1:221235001-221265000_strand_+_gene_88 -- region: 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\88_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.550s.\n",
      "-- designing region: 1:223735001-223765000_strand_+_gene_89 -- region: 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\89_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 1:226235001-226265000_strand_+_gene_90 -- region: 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\90_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: 1:228735001-228765000_strand_+_gene_91 -- region: 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\91_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 1:231235001-231265000_strand_+_gene_92 -- region: 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\92_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 1:233735001-233765000_strand_+_gene_93 -- region: 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\93_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.612s.\n",
      "-- designing region: 1:236235001-236265000_strand_+_gene_94 -- region: 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\94_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 1:238735001-238765000_strand_+_gene_95 -- region: 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\95_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 1:241235001-241265000_strand_+_gene_96 -- region: 96, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\96_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 23.354s.\n",
      "-- designing region: 1:243735001-243765000_strand_+_gene_97 -- region: 97, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\97_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 18.353s.\n",
      "-- designing region: 1:246235001-246265000_strand_+_gene_98 -- region: 98, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\98_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.497s.\n",
      "-- designing region: 2:1235001-1265000_strand_+_gene_99 -- region: 99, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\99_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.439s.\n",
      "-- designing region: 2:3735001-3765000_strand_+_gene_100 -- region: 100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\100_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.522s.\n",
      "-- designing region: 2:6235001-6265000_strand_+_gene_101 -- region: 101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\101_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.490s.\n",
      "-- designing region: 2:8735001-8765000_strand_+_gene_102 -- region: 102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\102_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 16.071s.\n",
      "-- designing region: 2:11235001-11265000_strand_+_gene_103 -- region: 103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\103_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.448s.\n",
      "-- designing region: 2:13735001-13765000_strand_+_gene_104 -- region: 104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\104_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.465s.\n",
      "-- designing region: 2:16235001-16265000_strand_+_gene_105 -- region: 105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\105_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.482s.\n",
      "-- designing region: 2:18735001-18765000_strand_+_gene_106 -- region: 106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\106_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.445s.\n",
      "-- designing region: 2:21235001-21265000_strand_+_gene_107 -- region: 107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\107_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.474s.\n",
      "-- designing region: 2:23735001-23765000_strand_+_gene_108 -- region: 108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\108_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.472s.\n",
      "-- designing region: 2:26235001-26265000_strand_+_gene_109 -- region: 109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\109_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.439s.\n",
      "-- designing region: 2:28735001-28765000_strand_+_gene_110 -- region: 110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\110_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.448s.\n",
      "-- designing region: 2:31235001-31265000_strand_+_gene_111 -- region: 111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\111_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.491s.\n",
      "-- designing region: 2:33735001-33765000_strand_+_gene_112 -- region: 112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\112_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.486s.\n",
      "-- designing region: 2:36235001-36265000_strand_+_gene_113 -- region: 113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\113_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.456s.\n",
      "-- designing region: 2:38735001-38765000_strand_+_gene_114 -- region: 114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\114_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.466s.\n",
      "-- designing region: 2:41235001-41265000_strand_+_gene_115 -- region: 115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\115_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.440s.\n",
      "-- designing region: 2:43735001-43765000_strand_+_gene_116 -- region: 116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\116_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.455s.\n",
      "-- designing region: 2:46235001-46265000_strand_+_gene_117 -- region: 117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\117_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.493s.\n",
      "-- designing region: 2:48735001-48765000_strand_+_gene_118 -- region: 118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\118_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.488s.\n",
      "-- designing region: 2:51235001-51265000_strand_+_gene_119 -- region: 119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\119_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.446s.\n",
      "-- designing region: 2:53735001-53765000_strand_+_gene_120 -- region: 120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\120_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.483s.\n",
      "-- designing region: 2:56235001-56265000_strand_+_gene_121 -- region: 121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\121_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.497s.\n",
      "-- designing region: 2:58735001-58765000_strand_+_gene_122 -- region: 122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\122_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 2:61235001-61265000_strand_+_gene_123 -- region: 123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\123_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.498s.\n",
      "-- designing region: 2:63735001-63765000_strand_+_gene_124 -- region: 124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\124_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.561s.\n",
      "-- designing region: 2:66235001-66265000_strand_+_gene_125 -- region: 125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\125_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.532s.\n",
      "-- designing region: 2:68735001-68765000_strand_+_gene_126 -- region: 126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\126_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 2:71235001-71265000_strand_+_gene_127 -- region: 127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\127_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 2:73735001-73765000_strand_+_gene_128 -- region: 128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\128_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.672s.\n",
      "-- designing region: 2:76235001-76265000_strand_+_gene_129 -- region: 129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\129_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 2:78735001-78765000_strand_+_gene_130 -- region: 130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\130_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.526s.\n",
      "-- designing region: 2:81235001-81265000_strand_+_gene_131 -- region: 131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\131_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 2:83735001-83765000_strand_+_gene_132 -- region: 132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\132_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 2:86235001-86265000_strand_+_gene_133 -- region: 133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\133_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 2:88735001-88765000_strand_+_gene_134 -- region: 134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\134_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 2:91235001-91265000_strand_+_gene_135 -- region: 135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\135_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "- Designed 0 candidate probes in 0.516s.\n",
      "-- designing region: 2:93735001-93765000_strand_+_gene_136 -- region: 136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\136_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.597s.\n",
      "- Designed 59918 candidate probes in 15.678s.\n",
      "-- designing region: 2:96235001-96265000_strand_+_gene_137 -- region: 137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\137_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 2:98735001-98765000_strand_+_gene_138 -- region: 138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\138_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.558s.\n",
      "-- designing region: 2:101235001-101265000_strand_+_gene_139 -- region: 139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\139_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 25.058s.\n",
      "-- designing region: 2:103735001-103765000_strand_+_gene_140 -- region: 140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\140_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.551s.\n",
      "-- designing region: 2:106235001-106265000_strand_+_gene_141 -- region: 141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\141_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.516s.\n",
      "-- designing region: 2:108735001-108765000_strand_+_gene_142 -- region: 142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\142_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.333s.\n",
      "-- designing region: 2:111235001-111265000_strand_+_gene_143 -- region: 143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\143_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 2:113735001-113765000_strand_+_gene_144 -- region: 144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\144_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 2:116235001-116265000_strand_+_gene_145 -- region: 145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\145_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.561s.\n",
      "-- designing region: 2:118735001-118765000_strand_+_gene_146 -- region: 146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\146_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 2:121235001-121265000_strand_+_gene_147 -- region: 147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\147_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 2:123735001-123765000_strand_+_gene_148 -- region: 148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\148_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 2:126235001-126265000_strand_+_gene_149 -- region: 149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\149_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.569s.\n",
      "-- designing region: 2:128735001-128765000_strand_+_gene_150 -- region: 150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\150_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 2:131235001-131265000_strand_+_gene_151 -- region: 151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\151_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.590s.\n",
      "-- designing region: 2:133735001-133765000_strand_+_gene_152 -- region: 152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\152_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 2:136235001-136265000_strand_+_gene_153 -- region: 153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\153_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 2:138735001-138765000_strand_+_gene_154 -- region: 154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\154_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 2:141235001-141265000_strand_+_gene_155 -- region: 155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\155_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.620s.\n",
      "-- designing region: 2:143735001-143765000_strand_+_gene_156 -- region: 156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\156_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 2:146235001-146265000_strand_+_gene_157 -- region: 157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\157_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.930s.\n",
      "-- designing region: 2:148735001-148765000_strand_+_gene_158 -- region: 158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\158_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 17.022s.\n",
      "-- designing region: 2:151235001-151265000_strand_+_gene_159 -- region: 159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\159_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.802s.\n",
      "- Designed 59918 candidate probes in 18.827s.\n",
      "-- designing region: 2:153735001-153765000_strand_+_gene_160 -- region: 160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\160_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.515s.\n",
      "-- designing region: 2:156235001-156265000_strand_+_gene_161 -- region: 161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\161_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 2:158735001-158765000_strand_+_gene_162 -- region: 162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\162_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 2:161235001-161265000_strand_+_gene_163 -- region: 163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\163_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.503s.\n",
      "-- designing region: 2:163735001-163765000_strand_+_gene_164 -- region: 164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\164_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 2:166235001-166265000_strand_+_gene_165 -- region: 165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\165_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.466s.\n",
      "-- designing region: 2:168735001-168765000_strand_+_gene_166 -- region: 166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\166_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.506s.\n",
      "-- designing region: 2:171235001-171265000_strand_+_gene_167 -- region: 167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\167_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.491s.\n",
      "-- designing region: 2:173735001-173765000_strand_+_gene_168 -- region: 168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\168_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.354s.\n",
      "-- designing region: 2:176235001-176265000_strand_+_gene_169 -- region: 169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\169_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.498s.\n",
      "-- designing region: 2:178735001-178765000_strand_+_gene_170 -- region: 170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\170_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 2:181235001-181265000_strand_+_gene_171 -- region: 171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\171_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.513s.\n",
      "-- designing region: 2:183735001-183765000_strand_+_gene_172 -- region: 172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\172_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.479s.\n",
      "-- designing region: 2:186235001-186265000_strand_+_gene_173 -- region: 173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\173_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 2:188735001-188765000_strand_+_gene_174 -- region: 174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\174_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.442s.\n",
      "-- designing region: 2:191235001-191265000_strand_+_gene_175 -- region: 175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\175_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 2:193735001-193765000_strand_+_gene_176 -- region: 176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\176_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 2:196235001-196265000_strand_+_gene_177 -- region: 177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\177_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.470s.\n",
      "-- designing region: 2:198735001-198765000_strand_+_gene_178 -- region: 178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\178_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 2:201235001-201265000_strand_+_gene_179 -- region: 179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\179_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 2:203735001-203765000_strand_+_gene_180 -- region: 180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\180_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 2:206235001-206265000_strand_+_gene_181 -- region: 181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\181_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.747s.\n",
      "-- designing region: 2:208735001-208765000_strand_+_gene_182 -- region: 182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\182_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.494s.\n",
      "-- designing region: 2:211235001-211265000_strand_+_gene_183 -- region: 183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\183_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.496s.\n",
      "-- designing region: 2:213735001-213765000_strand_+_gene_184 -- region: 184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\184_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.301s.\n",
      "- Designed 59918 candidate probes in 15.456s.\n",
      "-- designing region: 2:216235001-216265000_strand_+_gene_185 -- region: 185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\185_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.542s.\n",
      "-- designing region: 2:218735001-218765000_strand_+_gene_186 -- region: 186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\186_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.507s.\n",
      "-- designing region: 2:221235001-221265000_strand_+_gene_187 -- region: 187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\187_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.488s.\n",
      "-- designing region: 2:223735001-223765000_strand_+_gene_188 -- region: 188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\188_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.502s.\n",
      "-- designing region: 2:226235001-226265000_strand_+_gene_189 -- region: 189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\189_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 2:228735001-228765000_strand_+_gene_190 -- region: 190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\190_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 27.791s.\n",
      "-- designing region: 2:231235001-231265000_strand_+_gene_191 -- region: 191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\191_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 2:233735001-233765000_strand_+_gene_192 -- region: 192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\192_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 2:236235001-236265000_strand_+_gene_193 -- region: 193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\193_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.655s.\n",
      "-- designing region: 2:238735001-238765000_strand_+_gene_194 -- region: 194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\194_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.624s.\n",
      "-- designing region: 3:1235001-1265000_strand_+_gene_195 -- region: 195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\195_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.589s.\n",
      "-- designing region: 3:3735001-3765000_strand_+_gene_196 -- region: 196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\196_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 3:6235001-6265000_strand_+_gene_197 -- region: 197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\197_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.584s.\n",
      "-- designing region: 3:8735001-8765000_strand_+_gene_198 -- region: 198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\198_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: 3:11235001-11265000_strand_+_gene_199 -- region: 199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\199_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 16.750s.\n",
      "-- designing region: 3:13735001-13765000_strand_+_gene_200 -- region: 200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\200_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 3:16235001-16265000_strand_+_gene_201 -- region: 201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\201_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 3:18735001-18765000_strand_+_gene_202 -- region: 202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\202_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.523s.\n",
      "-- designing region: 3:21235001-21265000_strand_+_gene_203 -- region: 203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\203_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 3:23735001-23765000_strand_+_gene_204 -- region: 204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\204_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 3:26235001-26265000_strand_+_gene_205 -- region: 205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\205_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 3:28735001-28765000_strand_+_gene_206 -- region: 206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\206_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 3:31235001-31265000_strand_+_gene_207 -- region: 207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\207_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 3:33735001-33765000_strand_+_gene_208 -- region: 208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\208_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.509s.\n",
      "-- designing region: 3:36235001-36265000_strand_+_gene_209 -- region: 209, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\209_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 3:38735001-38765000_strand_+_gene_210 -- region: 210, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\210_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.546s.\n",
      "-- designing region: 3:41235001-41265000_strand_+_gene_211 -- region: 211, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\211_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.584s.\n",
      "-- designing region: 3:43735001-43765000_strand_+_gene_212 -- region: 212, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\212_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.549s.\n",
      "-- designing region: 3:46235001-46265000_strand_+_gene_213 -- region: 213, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\213_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.558s.\n",
      "-- designing region: 3:48735001-48765000_strand_+_gene_214 -- region: 214, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\214_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.537s.\n",
      "-- designing region: 3:51235001-51265000_strand_+_gene_215 -- region: 215, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\215_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 3:53735001-53765000_strand_+_gene_216 -- region: 216, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\216_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 3:56235001-56265000_strand_+_gene_217 -- region: 217, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\217_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 3:58735001-58765000_strand_+_gene_218 -- region: 218, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\218_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.859s.\n",
      "-- designing region: 3:61235001-61265000_strand_+_gene_219 -- region: 219, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\219_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 18.882s.\n",
      "-- designing region: 3:63735001-63765000_strand_+_gene_220 -- region: 220, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\220_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 25.131s.\n",
      "-- designing region: 3:66235001-66265000_strand_+_gene_221 -- region: 221, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\221_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.338s.\n",
      "-- designing region: 3:68735001-68765000_strand_+_gene_222 -- region: 222, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\222_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 3:71235001-71265000_strand_+_gene_223 -- region: 223, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\223_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.642s.\n",
      "-- designing region: 3:73735001-73765000_strand_+_gene_224 -- region: 224, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\224_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.303s.\n",
      "- Designed 59918 candidate probes in 15.684s.\n",
      "-- designing region: 3:76235001-76265000_strand_+_gene_225 -- region: 225, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\225_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.580s.\n",
      "-- designing region: 3:78735001-78765000_strand_+_gene_226 -- region: 226, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\226_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.312s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 3:81235001-81265000_strand_+_gene_227 -- region: 227, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\227_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 3:83735001-83765000_strand_+_gene_228 -- region: 228, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\228_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.312s.\n",
      "- Designed 59918 candidate probes in 15.601s.\n",
      "-- designing region: 3:86235001-86265000_strand_+_gene_229 -- region: 229, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\229_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 3:88735001-88765000_strand_+_gene_230 -- region: 230, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\230_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 3:91235001-91265000_strand_+_gene_231 -- region: 231, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\231_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.236s.\n",
      "- Designed 41860 candidate probes in 11.001s.\n",
      "-- designing region: 3:93735001-93765000_strand_+_gene_232 -- region: 232, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\232_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.481s.\n",
      "-- designing region: 3:96235001-96265000_strand_+_gene_233 -- region: 233, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\233_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.475s.\n",
      "-- designing region: 3:98735001-98765000_strand_+_gene_234 -- region: 234, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\234_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.551s.\n",
      "-- designing region: 3:101235001-101265000_strand_+_gene_235 -- region: 235, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\235_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 3:103735001-103765000_strand_+_gene_236 -- region: 236, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\236_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.546s.\n",
      "-- designing region: 3:106235001-106265000_strand_+_gene_237 -- region: 237, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\237_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 3:108735001-108765000_strand_+_gene_238 -- region: 238, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\238_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 3:111235001-111265000_strand_+_gene_239 -- region: 239, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\239_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 3:113735001-113765000_strand_+_gene_240 -- region: 240, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\240_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 3:116235001-116265000_strand_+_gene_241 -- region: 241, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\241_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 3:118735001-118765000_strand_+_gene_242 -- region: 242, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\242_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.680s.\n",
      "-- designing region: 3:121235001-121265000_strand_+_gene_243 -- region: 243, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\243_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 3:123735001-123765000_strand_+_gene_244 -- region: 244, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\244_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.607s.\n",
      "-- designing region: 3:126235001-126265000_strand_+_gene_245 -- region: 245, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\245_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 3:128735001-128765000_strand_+_gene_246 -- region: 246, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\246_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.647s.\n",
      "-- designing region: 3:131235001-131265000_strand_+_gene_247 -- region: 247, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\247_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.638s.\n",
      "-- designing region: 3:133735001-133765000_strand_+_gene_248 -- region: 248, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\248_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 3:136235001-136265000_strand_+_gene_249 -- region: 249, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\249_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 3:138735001-138765000_strand_+_gene_250 -- region: 250, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\250_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.626s.\n",
      "-- designing region: 3:141235001-141265000_strand_+_gene_251 -- region: 251, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\251_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.624s.\n",
      "-- designing region: 3:143735001-143765000_strand_+_gene_252 -- region: 252, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\252_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.676s.\n",
      "-- designing region: 3:146235001-146265000_strand_+_gene_253 -- region: 253, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\253_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 3:148735001-148765000_strand_+_gene_254 -- region: 254, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\254_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 3:151235001-151265000_strand_+_gene_255 -- region: 255, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\255_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 31.099s.\n",
      "-- designing region: 3:153735001-153765000_strand_+_gene_256 -- region: 256, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\256_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.519s.\n",
      "-- designing region: 3:156235001-156265000_strand_+_gene_257 -- region: 257, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\257_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.616s.\n",
      "-- designing region: 3:158735001-158765000_strand_+_gene_258 -- region: 258, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\258_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 3:161235001-161265000_strand_+_gene_259 -- region: 259, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\259_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.528s.\n",
      "-- designing region: 3:163735001-163765000_strand_+_gene_260 -- region: 260, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\260_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 3:166235001-166265000_strand_+_gene_261 -- region: 261, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\261_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.530s.\n",
      "-- designing region: 3:168735001-168765000_strand_+_gene_262 -- region: 262, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\262_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.460s.\n",
      "-- designing region: 3:171235001-171265000_strand_+_gene_263 -- region: 263, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\263_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 3:173735001-173765000_strand_+_gene_264 -- region: 264, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\264_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.523s.\n",
      "-- designing region: 3:176235001-176265000_strand_+_gene_265 -- region: 265, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\265_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 3:178735001-178765000_strand_+_gene_266 -- region: 266, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\266_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.485s.\n",
      "-- designing region: 3:181235001-181265000_strand_+_gene_267 -- region: 267, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\267_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.545s.\n",
      "-- designing region: 3:183735001-183765000_strand_+_gene_268 -- region: 268, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\268_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.550s.\n",
      "-- designing region: 3:186235001-186265000_strand_+_gene_269 -- region: 269, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\269_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 3:188735001-188765000_strand_+_gene_270 -- region: 270, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\270_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 3:191235001-191265000_strand_+_gene_271 -- region: 271, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\271_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.537s.\n",
      "-- designing region: 3:193735001-193765000_strand_+_gene_272 -- region: 272, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\272_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 3:196235001-196265000_strand_+_gene_273 -- region: 273, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\273_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.540s.\n",
      "-- designing region: 4:1235001-1265000_strand_+_gene_274 -- region: 274, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\274_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 4:3735001-3765000_strand_+_gene_275 -- region: 275, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\275_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 4:6235001-6265000_strand_+_gene_276 -- region: 276, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\276_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.686s.\n",
      "-- designing region: 4:8735001-8765000_strand_+_gene_277 -- region: 277, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\277_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.580s.\n",
      "-- designing region: 4:11235001-11265000_strand_+_gene_278 -- region: 278, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\278_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.828s.\n",
      "-- designing region: 4:13735001-13765000_strand_+_gene_279 -- region: 279, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\279_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 18.755s.\n",
      "-- designing region: 4:16235001-16265000_strand_+_gene_280 -- region: 280, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\280_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 23.827s.\n",
      "-- designing region: 4:18735001-18765000_strand_+_gene_281 -- region: 281, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\281_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.507s.\n",
      "-- designing region: 4:21235001-21265000_strand_+_gene_282 -- region: 282, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\282_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 4:23735001-23765000_strand_+_gene_283 -- region: 283, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\283_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 4:26235001-26265000_strand_+_gene_284 -- region: 284, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\284_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 4:28735001-28765000_strand_+_gene_285 -- region: 285, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\285_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.562s.\n",
      "-- designing region: 4:31235001-31265000_strand_+_gene_286 -- region: 286, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\286_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.729s.\n",
      "-- designing region: 4:33735001-33765000_strand_+_gene_287 -- region: 287, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\287_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 4:36235001-36265000_strand_+_gene_288 -- region: 288, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\288_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 4:38735001-38765000_strand_+_gene_289 -- region: 289, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\289_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 4:41235001-41265000_strand_+_gene_290 -- region: 290, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\290_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.300s.\n",
      "- Designed 59918 candidate probes in 15.670s.\n",
      "-- designing region: 4:43735001-43765000_strand_+_gene_291 -- region: 291, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\291_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 4:46235001-46265000_strand_+_gene_292 -- region: 292, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\292_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 4:48735001-48765000_strand_+_gene_293 -- region: 293, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\293_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.562s.\n",
      "-- designing region: 4:51235001-51265000_strand_+_gene_294 -- region: 294, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\294_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "- Designed 59918 candidate probes in 15.361s.\n",
      "-- designing region: 4:53735001-53765000_strand_+_gene_295 -- region: 295, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\295_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 4:56235001-56265000_strand_+_gene_296 -- region: 296, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\296_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 4:58735001-58765000_strand_+_gene_297 -- region: 297, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\297_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 4:61235001-61265000_strand_+_gene_298 -- region: 298, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\298_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 4:63735001-63765000_strand_+_gene_299 -- region: 299, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\299_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.300s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 4:66235001-66265000_strand_+_gene_300 -- region: 300, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\300_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 4:68735001-68765000_strand_+_gene_301 -- region: 301, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\301_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 4:71235001-71265000_strand_+_gene_302 -- region: 302, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\302_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 4:73735001-73765000_strand_+_gene_303 -- region: 303, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\303_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.641s.\n",
      "-- designing region: 4:76235001-76265000_strand_+_gene_304 -- region: 304, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\304_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 4:78735001-78765000_strand_+_gene_305 -- region: 305, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\305_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 4:81235001-81265000_strand_+_gene_306 -- region: 306, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\306_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 4:83735001-83765000_strand_+_gene_307 -- region: 307, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\307_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.701s.\n",
      "-- designing region: 4:86235001-86265000_strand_+_gene_308 -- region: 308, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\308_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 4:88735001-88765000_strand_+_gene_309 -- region: 309, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\309_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 4:91235001-91265000_strand_+_gene_310 -- region: 310, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\310_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 4:93735001-93765000_strand_+_gene_311 -- region: 311, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\311_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 4:96235001-96265000_strand_+_gene_312 -- region: 312, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\312_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 4:98735001-98765000_strand_+_gene_313 -- region: 313, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\313_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 4:101235001-101265000_strand_+_gene_314 -- region: 314, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\314_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 4:103735001-103765000_strand_+_gene_315 -- region: 315, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\315_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 4:106235001-106265000_strand_+_gene_316 -- region: 316, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\316_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.503s.\n",
      "-- designing region: 4:108735001-108765000_strand_+_gene_317 -- region: 317, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\317_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.626s.\n",
      "-- designing region: 4:111235001-111265000_strand_+_gene_318 -- region: 318, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\318_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.555s.\n",
      "-- designing region: 4:113735001-113765000_strand_+_gene_319 -- region: 319, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\319_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 4:116235001-116265000_strand_+_gene_320 -- region: 320, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\320_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.550s.\n",
      "-- designing region: 4:118735001-118765000_strand_+_gene_321 -- region: 321, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\321_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 4:121235001-121265000_strand_+_gene_322 -- region: 322, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\322_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 4:123735001-123765000_strand_+_gene_323 -- region: 323, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\323_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 4:126235001-126265000_strand_+_gene_324 -- region: 324, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\324_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.529s.\n",
      "-- designing region: 4:128735001-128765000_strand_+_gene_325 -- region: 325, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\325_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 4:131235001-131265000_strand_+_gene_326 -- region: 326, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\326_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 4:133735001-133765000_strand_+_gene_327 -- region: 327, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\327_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: 4:136235001-136265000_strand_+_gene_328 -- region: 328, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\328_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 4:138735001-138765000_strand_+_gene_329 -- region: 329, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\329_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: 4:141235001-141265000_strand_+_gene_330 -- region: 330, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\330_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.530s.\n",
      "-- designing region: 4:143735001-143765000_strand_+_gene_331 -- region: 331, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\331_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.496s.\n",
      "-- designing region: 4:146235001-146265000_strand_+_gene_332 -- region: 332, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\332_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.665s.\n",
      "-- designing region: 4:148735001-148765000_strand_+_gene_333 -- region: 333, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\333_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.525s.\n",
      "-- designing region: 4:151235001-151265000_strand_+_gene_334 -- region: 334, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\334_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.551s.\n",
      "-- designing region: 4:153735001-153765000_strand_+_gene_335 -- region: 335, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\335_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 35.304s.\n",
      "-- designing region: 4:156235001-156265000_strand_+_gene_336 -- region: 336, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\336_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 4:158735001-158765000_strand_+_gene_337 -- region: 337, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\337_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.716s.\n",
      "-- designing region: 4:161235001-161265000_strand_+_gene_338 -- region: 338, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\338_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 4:163735001-163765000_strand_+_gene_339 -- region: 339, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\339_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 25.287s.\n",
      "-- designing region: 4:166235001-166265000_strand_+_gene_340 -- region: 340, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\340_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.700s.\n",
      "- Designed 59918 candidate probes in 18.302s.\n",
      "-- designing region: 4:168735001-168765000_strand_+_gene_341 -- region: 341, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\341_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 4:171235001-171265000_strand_+_gene_342 -- region: 342, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\342_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 4:173735001-173765000_strand_+_gene_343 -- region: 343, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\343_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: 4:176235001-176265000_strand_+_gene_344 -- region: 344, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\344_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.622s.\n",
      "-- designing region: 4:178735001-178765000_strand_+_gene_345 -- region: 345, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\345_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 4:181235001-181265000_strand_+_gene_346 -- region: 346, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\346_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 4:183735001-183765000_strand_+_gene_347 -- region: 347, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\347_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.671s.\n",
      "-- designing region: 4:186235001-186265000_strand_+_gene_348 -- region: 348, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\348_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 4:188735001-188765000_strand_+_gene_349 -- region: 349, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\349_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 5:1235001-1265000_strand_+_gene_350 -- region: 350, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\350_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.677s.\n",
      "-- designing region: 5:3735001-3765000_strand_+_gene_351 -- region: 351, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\351_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 5:6235001-6265000_strand_+_gene_352 -- region: 352, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\352_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 5:8735001-8765000_strand_+_gene_353 -- region: 353, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\353_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 5:11235001-11265000_strand_+_gene_354 -- region: 354, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\354_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 5:13735001-13765000_strand_+_gene_355 -- region: 355, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\355_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 5:16235001-16265000_strand_+_gene_356 -- region: 356, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\356_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 5:18735001-18765000_strand_+_gene_357 -- region: 357, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\357_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 5:21235001-21265000_strand_+_gene_358 -- region: 358, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\358_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 5:23735001-23765000_strand_+_gene_359 -- region: 359, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\359_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.534s.\n",
      "-- designing region: 5:26235001-26265000_strand_+_gene_360 -- region: 360, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\360_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 5:28735001-28765000_strand_+_gene_361 -- region: 361, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\361_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.612s.\n",
      "-- designing region: 5:31235001-31265000_strand_+_gene_362 -- region: 362, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\362_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 5:33735001-33765000_strand_+_gene_363 -- region: 363, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\363_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 5:36235001-36265000_strand_+_gene_364 -- region: 364, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\364_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 5:38735001-38765000_strand_+_gene_365 -- region: 365, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\365_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.654s.\n",
      "-- designing region: 5:41235001-41265000_strand_+_gene_366 -- region: 366, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\366_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 5:43735001-43765000_strand_+_gene_367 -- region: 367, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\367_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 5:46235001-46265000_strand_+_gene_368 -- region: 368, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\368_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.259s.\n",
      "- Designed 59918 candidate probes in 15.511s.\n",
      "-- designing region: 5:48735001-48765000_strand_+_gene_369 -- region: 369, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\369_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.233s.\n",
      "- Designed 59918 candidate probes in 15.364s.\n",
      "-- designing region: 5:51235001-51265000_strand_+_gene_370 -- region: 370, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\370_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.486s.\n",
      "-- designing region: 5:53735001-53765000_strand_+_gene_371 -- region: 371, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\371_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 5:56235001-56265000_strand_+_gene_372 -- region: 372, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\372_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 5:58735001-58765000_strand_+_gene_373 -- region: 373, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\373_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 5:61235001-61265000_strand_+_gene_374 -- region: 374, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\374_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 5:63735001-63765000_strand_+_gene_375 -- region: 375, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\375_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.528s.\n",
      "-- designing region: 5:66235001-66265000_strand_+_gene_376 -- region: 376, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\376_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 5:68735001-68765000_strand_+_gene_377 -- region: 377, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\377_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 5:71235001-71265000_strand_+_gene_378 -- region: 378, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\378_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.525s.\n",
      "-- designing region: 5:73735001-73765000_strand_+_gene_379 -- region: 379, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\379_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.539s.\n",
      "-- designing region: 5:76235001-76265000_strand_+_gene_380 -- region: 380, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\380_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.548s.\n",
      "-- designing region: 5:78735001-78765000_strand_+_gene_381 -- region: 381, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\381_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 5:81235001-81265000_strand_+_gene_382 -- region: 382, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\382_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 5:83735001-83765000_strand_+_gene_383 -- region: 383, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\383_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.503s.\n",
      "-- designing region: 5:86235001-86265000_strand_+_gene_384 -- region: 384, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\384_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: 5:88735001-88765000_strand_+_gene_385 -- region: 385, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\385_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.519s.\n",
      "-- designing region: 5:91235001-91265000_strand_+_gene_386 -- region: 386, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\386_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.529s.\n",
      "-- designing region: 5:93735001-93765000_strand_+_gene_387 -- region: 387, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\387_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.508s.\n",
      "-- designing region: 5:96235001-96265000_strand_+_gene_388 -- region: 388, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\388_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.548s.\n",
      "-- designing region: 5:98735001-98765000_strand_+_gene_389 -- region: 389, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\389_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.522s.\n",
      "-- designing region: 5:101235001-101265000_strand_+_gene_390 -- region: 390, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\390_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.502s.\n",
      "-- designing region: 5:103735001-103765000_strand_+_gene_391 -- region: 391, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\391_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 5:106235001-106265000_strand_+_gene_392 -- region: 392, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\392_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 5:108735001-108765000_strand_+_gene_393 -- region: 393, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\393_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 18.066s.\n",
      "-- designing region: 5:111235001-111265000_strand_+_gene_394 -- region: 394, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\394_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.469s.\n",
      "-- designing region: 5:113735001-113765000_strand_+_gene_395 -- region: 395, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\395_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.446s.\n",
      "-- designing region: 5:116235001-116265000_strand_+_gene_396 -- region: 396, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\396_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.544s.\n",
      "-- designing region: 5:118735001-118765000_strand_+_gene_397 -- region: 397, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\397_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 5:121235001-121265000_strand_+_gene_398 -- region: 398, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\398_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.724s.\n",
      "-- designing region: 5:123735001-123765000_strand_+_gene_399 -- region: 399, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\399_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 16.902s.\n",
      "-- designing region: 5:126235001-126265000_strand_+_gene_400 -- region: 400, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\400_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.634s.\n",
      "- Designed 59918 candidate probes in 26.355s.\n",
      "-- designing region: 5:128735001-128765000_strand_+_gene_401 -- region: 401, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\401_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 5:131235001-131265000_strand_+_gene_402 -- region: 402, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\402_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 5:133735001-133765000_strand_+_gene_403 -- region: 403, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\403_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 5:136235001-136265000_strand_+_gene_404 -- region: 404, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\404_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 5:138735001-138765000_strand_+_gene_405 -- region: 405, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\405_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 5:141235001-141265000_strand_+_gene_406 -- region: 406, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\406_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 5:143735001-143765000_strand_+_gene_407 -- region: 407, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\407_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 5:146235001-146265000_strand_+_gene_408 -- region: 408, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\408_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.566s.\n",
      "-- designing region: 5:148735001-148765000_strand_+_gene_409 -- region: 409, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\409_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 5:151235001-151265000_strand_+_gene_410 -- region: 410, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\410_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 5:153735001-153765000_strand_+_gene_411 -- region: 411, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\411_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.645s.\n",
      "-- designing region: 5:156235001-156265000_strand_+_gene_412 -- region: 412, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\412_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.627s.\n",
      "-- designing region: 5:158735001-158765000_strand_+_gene_413 -- region: 413, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\413_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 5:161235001-161265000_strand_+_gene_414 -- region: 414, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\414_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 5:163735001-163765000_strand_+_gene_415 -- region: 415, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\415_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 5:166235001-166265000_strand_+_gene_416 -- region: 416, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\416_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 5:168735001-168765000_strand_+_gene_417 -- region: 417, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\417_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 5:171235001-171265000_strand_+_gene_418 -- region: 418, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\418_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.577s.\n",
      "-- designing region: 5:173735001-173765000_strand_+_gene_419 -- region: 419, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\419_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 5:176235001-176265000_strand_+_gene_420 -- region: 420, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\420_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 5:178735001-178765000_strand_+_gene_421 -- region: 421, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\421_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 6:1235001-1265000_strand_+_gene_422 -- region: 422, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\422_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 6:3735001-3765000_strand_+_gene_423 -- region: 423, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\423_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 6:6235001-6265000_strand_+_gene_424 -- region: 424, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\424_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 6:8735001-8765000_strand_+_gene_425 -- region: 425, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\425_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 6:11235001-11265000_strand_+_gene_426 -- region: 426, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\426_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.658s.\n",
      "-- designing region: 6:13735001-13765000_strand_+_gene_427 -- region: 427, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\427_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 6:16235001-16265000_strand_+_gene_428 -- region: 428, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\428_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 6:18735001-18765000_strand_+_gene_429 -- region: 429, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\429_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.607s.\n",
      "-- designing region: 6:21235001-21265000_strand_+_gene_430 -- region: 430, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\430_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 6:23735001-23765000_strand_+_gene_431 -- region: 431, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\431_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.617s.\n",
      "-- designing region: 6:26235001-26265000_strand_+_gene_432 -- region: 432, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\432_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.715s.\n",
      "-- designing region: 6:28735001-28765000_strand_+_gene_433 -- region: 433, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\433_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 6:31235001-31265000_strand_+_gene_434 -- region: 434, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\434_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.576s.\n",
      "-- designing region: 6:33735001-33765000_strand_+_gene_435 -- region: 435, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\435_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 40.413s.\n",
      "-- designing region: 6:36235001-36265000_strand_+_gene_436 -- region: 436, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\436_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 6:38735001-38765000_strand_+_gene_437 -- region: 437, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\437_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 6:41235001-41265000_strand_+_gene_438 -- region: 438, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\438_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.562s.\n",
      "-- designing region: 6:43735001-43765000_strand_+_gene_439 -- region: 439, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\439_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 6:46235001-46265000_strand_+_gene_440 -- region: 440, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\440_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.544s.\n",
      "-- designing region: 6:48735001-48765000_strand_+_gene_441 -- region: 441, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\441_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.576s.\n",
      "-- designing region: 6:51235001-51265000_strand_+_gene_442 -- region: 442, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\442_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 6:53735001-53765000_strand_+_gene_443 -- region: 443, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\443_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.558s.\n",
      "-- designing region: 6:56235001-56265000_strand_+_gene_444 -- region: 444, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\444_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.544s.\n",
      "-- designing region: 6:58735001-58765000_strand_+_gene_445 -- region: 445, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\445_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.226s.\n",
      "- Designed 59918 candidate probes in 15.332s.\n",
      "-- designing region: 6:61235001-61265000_strand_+_gene_446 -- region: 446, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\446_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.526s.\n",
      "-- designing region: 6:63735001-63765000_strand_+_gene_447 -- region: 447, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\447_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.558s.\n",
      "-- designing region: 6:66235001-66265000_strand_+_gene_448 -- region: 448, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\448_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.532s.\n",
      "-- designing region: 6:68735001-68765000_strand_+_gene_449 -- region: 449, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\449_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.534s.\n",
      "-- designing region: 6:71235001-71265000_strand_+_gene_450 -- region: 450, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\450_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 6:73735001-73765000_strand_+_gene_451 -- region: 451, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\451_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 6:76235001-76265000_strand_+_gene_452 -- region: 452, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\452_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 6:78735001-78765000_strand_+_gene_453 -- region: 453, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\453_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.540s.\n",
      "-- designing region: 6:81235001-81265000_strand_+_gene_454 -- region: 454, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\454_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.520s.\n",
      "-- designing region: 6:83735001-83765000_strand_+_gene_455 -- region: 455, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\455_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 6:86235001-86265000_strand_+_gene_456 -- region: 456, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\456_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 6:88735001-88765000_strand_+_gene_457 -- region: 457, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\457_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 6:91235001-91265000_strand_+_gene_458 -- region: 458, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\458_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.767s.\n",
      "-- designing region: 6:93735001-93765000_strand_+_gene_459 -- region: 459, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\459_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 22.793s.\n",
      "-- designing region: 6:96235001-96265000_strand_+_gene_460 -- region: 460, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\460_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.649s.\n",
      "- Designed 59918 candidate probes in 19.907s.\n",
      "-- designing region: 6:98735001-98765000_strand_+_gene_461 -- region: 461, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\461_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 6:101235001-101265000_strand_+_gene_462 -- region: 462, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\462_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 6:103735001-103765000_strand_+_gene_463 -- region: 463, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\463_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.661s.\n",
      "-- designing region: 6:106235001-106265000_strand_+_gene_464 -- region: 464, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\464_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 6:108735001-108765000_strand_+_gene_465 -- region: 465, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\465_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.667s.\n",
      "-- designing region: 6:111235001-111265000_strand_+_gene_466 -- region: 466, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\466_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.695s.\n",
      "-- designing region: 6:113735001-113765000_strand_+_gene_467 -- region: 467, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\467_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.648s.\n",
      "-- designing region: 6:116235001-116265000_strand_+_gene_468 -- region: 468, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\468_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.662s.\n",
      "-- designing region: 6:118735001-118765000_strand_+_gene_469 -- region: 469, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\469_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 6:121235001-121265000_strand_+_gene_470 -- region: 470, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\470_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.670s.\n",
      "-- designing region: 6:123735001-123765000_strand_+_gene_471 -- region: 471, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\471_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.613s.\n",
      "-- designing region: 6:126235001-126265000_strand_+_gene_472 -- region: 472, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\472_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 6:128735001-128765000_strand_+_gene_473 -- region: 473, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\473_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 6:131235001-131265000_strand_+_gene_474 -- region: 474, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\474_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.638s.\n",
      "-- designing region: 6:133735001-133765000_strand_+_gene_475 -- region: 475, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\475_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 6:136235001-136265000_strand_+_gene_476 -- region: 476, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\476_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 6:138735001-138765000_strand_+_gene_477 -- region: 477, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\477_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 6:141235001-141265000_strand_+_gene_478 -- region: 478, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\478_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 6:143735001-143765000_strand_+_gene_479 -- region: 479, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\479_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.669s.\n",
      "-- designing region: 6:146235001-146265000_strand_+_gene_480 -- region: 480, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\480_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 6:148735001-148765000_strand_+_gene_481 -- region: 481, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\481_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 6:151235001-151265000_strand_+_gene_482 -- region: 482, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\482_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.607s.\n",
      "-- designing region: 6:153735001-153765000_strand_+_gene_483 -- region: 483, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\483_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.689s.\n",
      "-- designing region: 6:156235001-156265000_strand_+_gene_484 -- region: 484, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\484_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 6:158735001-158765000_strand_+_gene_485 -- region: 485, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\485_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 6:161235001-161265000_strand_+_gene_486 -- region: 486, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\486_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 6:163735001-163765000_strand_+_gene_487 -- region: 487, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\487_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.672s.\n",
      "-- designing region: 6:166235001-166265000_strand_+_gene_488 -- region: 488, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\488_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 6:168735001-168765000_strand_+_gene_489 -- region: 489, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\489_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.627s.\n",
      "-- designing region: 7:1235001-1265000_strand_+_gene_490 -- region: 490, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\490_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.666s.\n",
      "-- designing region: 7:3735001-3765000_strand_+_gene_491 -- region: 491, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\491_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 7:6235001-6265000_strand_+_gene_492 -- region: 492, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\492_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 7:8735001-8765000_strand_+_gene_493 -- region: 493, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\493_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.569s.\n",
      "-- designing region: 7:11235001-11265000_strand_+_gene_494 -- region: 494, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\494_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 7:13735001-13765000_strand_+_gene_495 -- region: 495, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\495_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 7:16235001-16265000_strand_+_gene_496 -- region: 496, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\496_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.303s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 7:18735001-18765000_strand_+_gene_497 -- region: 497, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\497_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 7:21235001-21265000_strand_+_gene_498 -- region: 498, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\498_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 7:23735001-23765000_strand_+_gene_499 -- region: 499, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\499_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 7:26235001-26265000_strand_+_gene_500 -- region: 500, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\500_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.599s.\n",
      "-- designing region: 7:28735001-28765000_strand_+_gene_501 -- region: 501, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\501_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 7:31235001-31265000_strand_+_gene_502 -- region: 502, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\502_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 7:33735001-33765000_strand_+_gene_503 -- region: 503, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\503_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 7:36235001-36265000_strand_+_gene_504 -- region: 504, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\504_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 7:38735001-38765000_strand_+_gene_505 -- region: 505, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\505_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.569s.\n",
      "-- designing region: 7:41235001-41265000_strand_+_gene_506 -- region: 506, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\506_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.304s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 7:43735001-43765000_strand_+_gene_507 -- region: 507, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\507_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.493s.\n",
      "-- designing region: 7:46235001-46265000_strand_+_gene_508 -- region: 508, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\508_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: 7:48735001-48765000_strand_+_gene_509 -- region: 509, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\509_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.538s.\n",
      "-- designing region: 7:51235001-51265000_strand_+_gene_510 -- region: 510, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\510_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.529s.\n",
      "-- designing region: 7:53735001-53765000_strand_+_gene_511 -- region: 511, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\511_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.536s.\n",
      "-- designing region: 7:56235001-56265000_strand_+_gene_512 -- region: 512, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\512_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.545s.\n",
      "-- designing region: 7:58735001-58765000_strand_+_gene_513 -- region: 513, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\513_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.228s.\n",
      "- Designed 59636 candidate probes in 15.237s.\n",
      "-- designing region: 7:61235001-61265000_strand_+_gene_514 -- region: 514, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\514_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "- Designed 59918 candidate probes in 15.422s.\n",
      "-- designing region: 7:63735001-63765000_strand_+_gene_515 -- region: 515, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\515_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 7:66235001-66265000_strand_+_gene_516 -- region: 516, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\516_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 7:68735001-68765000_strand_+_gene_517 -- region: 517, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\517_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.665s.\n",
      "-- designing region: 7:71235001-71265000_strand_+_gene_518 -- region: 518, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\518_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.648s.\n",
      "-- designing region: 7:73735001-73765000_strand_+_gene_519 -- region: 519, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\519_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.720s.\n",
      "-- designing region: 7:76235001-76265000_strand_+_gene_520 -- region: 520, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\520_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 21.128s.\n",
      "-- designing region: 7:78735001-78765000_strand_+_gene_521 -- region: 521, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\521_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.693s.\n",
      "- Designed 59918 candidate probes in 18.523s.\n",
      "-- designing region: 7:81235001-81265000_strand_+_gene_522 -- region: 522, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\522_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 7:83735001-83765000_strand_+_gene_523 -- region: 523, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\523_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 7:86235001-86265000_strand_+_gene_524 -- region: 524, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\524_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 7:88735001-88765000_strand_+_gene_525 -- region: 525, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\525_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 7:91235001-91265000_strand_+_gene_526 -- region: 526, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\526_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.616s.\n",
      "-- designing region: 7:93735001-93765000_strand_+_gene_527 -- region: 527, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\527_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.690s.\n",
      "-- designing region: 7:96235001-96265000_strand_+_gene_528 -- region: 528, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\528_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 7:98735001-98765000_strand_+_gene_529 -- region: 529, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\529_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.651s.\n",
      "-- designing region: 7:101235001-101265000_strand_+_gene_530 -- region: 530, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\530_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 7:103735001-103765000_strand_+_gene_531 -- region: 531, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\531_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: 7:106235001-106265000_strand_+_gene_532 -- region: 532, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\532_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 7:108735001-108765000_strand_+_gene_533 -- region: 533, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\533_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 7:111235001-111265000_strand_+_gene_534 -- region: 534, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\534_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 7:113735001-113765000_strand_+_gene_535 -- region: 535, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\535_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 7:116235001-116265000_strand_+_gene_536 -- region: 536, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\536_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 7:118735001-118765000_strand_+_gene_537 -- region: 537, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\537_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 7:121235001-121265000_strand_+_gene_538 -- region: 538, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\538_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 7:123735001-123765000_strand_+_gene_539 -- region: 539, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\539_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 7:126235001-126265000_strand_+_gene_540 -- region: 540, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\540_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 7:128735001-128765000_strand_+_gene_541 -- region: 541, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\541_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.647s.\n",
      "-- designing region: 7:131235001-131265000_strand_+_gene_542 -- region: 542, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\542_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.662s.\n",
      "-- designing region: 7:133735001-133765000_strand_+_gene_543 -- region: 543, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\543_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 7:136235001-136265000_strand_+_gene_544 -- region: 544, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\544_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 7:138735001-138765000_strand_+_gene_545 -- region: 545, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\545_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.661s.\n",
      "-- designing region: 7:141235001-141265000_strand_+_gene_546 -- region: 546, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\546_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.671s.\n",
      "-- designing region: 7:143735001-143765000_strand_+_gene_547 -- region: 547, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\547_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 7:146235001-146265000_strand_+_gene_548 -- region: 548, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\548_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 7:148735001-148765000_strand_+_gene_549 -- region: 549, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\549_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 7:151235001-151265000_strand_+_gene_550 -- region: 550, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\550_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.742s.\n",
      "-- designing region: 7:153735001-153765000_strand_+_gene_551 -- region: 551, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\551_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 7:156235001-156265000_strand_+_gene_552 -- region: 552, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\552_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 8:1235001-1265000_strand_+_gene_553 -- region: 553, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\553_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.251s.\n",
      "- Designed 59918 candidate probes in 15.478s.\n",
      "-- designing region: 8:3735001-3765000_strand_+_gene_554 -- region: 554, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\554_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.537s.\n",
      "-- designing region: 8:6235001-6265000_strand_+_gene_555 -- region: 555, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\555_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 8:8735001-8765000_strand_+_gene_556 -- region: 556, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\556_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 8:11235001-11265000_strand_+_gene_557 -- region: 557, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\557_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.574s.\n",
      "-- designing region: 8:13735001-13765000_strand_+_gene_558 -- region: 558, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\558_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.525s.\n",
      "-- designing region: 8:16235001-16265000_strand_+_gene_559 -- region: 559, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\559_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.522s.\n",
      "-- designing region: 8:18735001-18765000_strand_+_gene_560 -- region: 560, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\560_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 46.748s.\n",
      "-- designing region: 8:21235001-21265000_strand_+_gene_561 -- region: 561, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\561_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 8:23735001-23765000_strand_+_gene_562 -- region: 562, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\562_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 8:26235001-26265000_strand_+_gene_563 -- region: 563, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\563_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.538s.\n",
      "-- designing region: 8:28735001-28765000_strand_+_gene_564 -- region: 564, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\564_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.638s.\n",
      "-- designing region: 8:31235001-31265000_strand_+_gene_565 -- region: 565, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\565_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 8:33735001-33765000_strand_+_gene_566 -- region: 566, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\566_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.509s.\n",
      "-- designing region: 8:36235001-36265000_strand_+_gene_567 -- region: 567, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\567_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.540s.\n",
      "-- designing region: 8:38735001-38765000_strand_+_gene_568 -- region: 568, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\568_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 8:41235001-41265000_strand_+_gene_569 -- region: 569, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\569_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 8:43735001-43765000_strand_+_gene_570 -- region: 570, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\570_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "- Designed 59918 candidate probes in 15.495s.\n",
      "-- designing region: 8:46235001-46265000_strand_+_gene_571 -- region: 571, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\571_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 8:48735001-48765000_strand_+_gene_572 -- region: 572, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\572_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.550s.\n",
      "-- designing region: 8:51235001-51265000_strand_+_gene_573 -- region: 573, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\573_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.507s.\n",
      "-- designing region: 8:53735001-53765000_strand_+_gene_574 -- region: 574, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\574_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 8:56235001-56265000_strand_+_gene_575 -- region: 575, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\575_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 8:58735001-58765000_strand_+_gene_576 -- region: 576, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\576_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 8:61235001-61265000_strand_+_gene_577 -- region: 577, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\577_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.574s.\n",
      "-- designing region: 8:63735001-63765000_strand_+_gene_578 -- region: 578, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\578_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.710s.\n",
      "-- designing region: 8:66235001-66265000_strand_+_gene_579 -- region: 579, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\579_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 18.588s.\n",
      "-- designing region: 8:68735001-68765000_strand_+_gene_580 -- region: 580, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\580_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.527s.\n",
      "- Designed 59918 candidate probes in 22.086s.\n",
      "-- designing region: 8:71235001-71265000_strand_+_gene_581 -- region: 581, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\581_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: 8:73735001-73765000_strand_+_gene_582 -- region: 582, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\582_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 8:76235001-76265000_strand_+_gene_583 -- region: 583, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\583_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.306s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 8:78735001-78765000_strand_+_gene_584 -- region: 584, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\584_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 8:81235001-81265000_strand_+_gene_585 -- region: 585, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\585_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.696s.\n",
      "-- designing region: 8:83735001-83765000_strand_+_gene_586 -- region: 586, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\586_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: 8:86235001-86265000_strand_+_gene_587 -- region: 587, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\587_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 8:88735001-88765000_strand_+_gene_588 -- region: 588, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\588_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.704s.\n",
      "-- designing region: 8:91235001-91265000_strand_+_gene_589 -- region: 589, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\589_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 8:93735001-93765000_strand_+_gene_590 -- region: 590, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\590_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.701s.\n",
      "-- designing region: 8:96235001-96265000_strand_+_gene_591 -- region: 591, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\591_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.662s.\n",
      "-- designing region: 8:98735001-98765000_strand_+_gene_592 -- region: 592, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\592_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: 8:101235001-101265000_strand_+_gene_593 -- region: 593, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\593_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 8:103735001-103765000_strand_+_gene_594 -- region: 594, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\594_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.640s.\n",
      "-- designing region: 8:106235001-106265000_strand_+_gene_595 -- region: 595, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\595_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.551s.\n",
      "-- designing region: 8:108735001-108765000_strand_+_gene_596 -- region: 596, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\596_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 8:111235001-111265000_strand_+_gene_597 -- region: 597, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\597_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.300s.\n",
      "- Designed 59918 candidate probes in 15.647s.\n",
      "-- designing region: 8:113735001-113765000_strand_+_gene_598 -- region: 598, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\598_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 8:116235001-116265000_strand_+_gene_599 -- region: 599, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\599_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.688s.\n",
      "-- designing region: 8:118735001-118765000_strand_+_gene_600 -- region: 600, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\600_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 8:121235001-121265000_strand_+_gene_601 -- region: 601, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\601_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 8:123735001-123765000_strand_+_gene_602 -- region: 602, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\602_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.589s.\n",
      "-- designing region: 8:126235001-126265000_strand_+_gene_603 -- region: 603, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\603_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 8:128735001-128765000_strand_+_gene_604 -- region: 604, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\604_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 8:131235001-131265000_strand_+_gene_605 -- region: 605, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\605_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.617s.\n",
      "-- designing region: 8:133735001-133765000_strand_+_gene_606 -- region: 606, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\606_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.680s.\n",
      "-- designing region: 8:136235001-136265000_strand_+_gene_607 -- region: 607, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\607_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 8:138735001-138765000_strand_+_gene_608 -- region: 608, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\608_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.640s.\n",
      "-- designing region: 8:141235001-141265000_strand_+_gene_609 -- region: 609, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\609_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 8:143735001-143765000_strand_+_gene_610 -- region: 610, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\610_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.654s.\n",
      "-- designing region: 9:985001-1015000_strand_+_gene_611 -- region: 611, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\611_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 9:2985001-3015000_strand_+_gene_612 -- region: 612, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\612_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 9:4985001-5015000_strand_+_gene_613 -- region: 613, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\613_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 9:6985001-7015000_strand_+_gene_614 -- region: 614, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\614_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 9:8985001-9015000_strand_+_gene_615 -- region: 615, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\615_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 9:10985001-11015000_strand_+_gene_616 -- region: 616, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\616_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 9:12985001-13015000_strand_+_gene_617 -- region: 617, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\617_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.555s.\n",
      "-- designing region: 9:14985001-15015000_strand_+_gene_618 -- region: 618, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\618_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.536s.\n",
      "-- designing region: 9:16985001-17015000_strand_+_gene_619 -- region: 619, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\619_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.513s.\n",
      "-- designing region: 9:18985001-19015000_strand_+_gene_620 -- region: 620, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\620_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 9:20985001-21015000_strand_+_gene_621 -- region: 621, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\621_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 9:22985001-23015000_strand_+_gene_622 -- region: 622, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\622_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.580s.\n",
      "-- designing region: 9:24985001-25015000_strand_+_gene_623 -- region: 623, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\623_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 9:26985001-27015000_strand_+_gene_624 -- region: 624, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\624_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.517s.\n",
      "-- designing region: 9:28985001-29015000_strand_+_gene_625 -- region: 625, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\625_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 9:30985001-31015000_strand_+_gene_626 -- region: 626, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\626_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 9:32985001-33015000_strand_+_gene_627 -- region: 627, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\627_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 9:34985001-35015000_strand_+_gene_628 -- region: 628, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\628_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 9:36985001-37015000_strand_+_gene_629 -- region: 629, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\629_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 9:38985001-39015000_strand_+_gene_630 -- region: 630, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\630_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.576s.\n",
      "-- designing region: 9:40985001-41015000_strand_+_gene_631 -- region: 631, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\631_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 9:42985001-43015000_strand_+_gene_632 -- region: 632, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\632_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 9:44985001-45015000_strand_+_gene_633 -- region: 633, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\633_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.225s.\n",
      "- Designed 59918 candidate probes in 15.290s.\n",
      "-- designing region: 9:46985001-47015000_strand_+_gene_634 -- region: 634, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\634_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.213s.\n",
      "- Designed 0 candidate probes in 0.507s.\n",
      "-- designing region: 9:48985001-49015000_strand_+_gene_635 -- region: 635, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\635_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.558s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 9:50985001-51015000_strand_+_gene_636 -- region: 636, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\636_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.585s.\n",
      "- Designed 0 candidate probes in 0.877s.\n",
      "-- designing region: 9:52985001-53015000_strand_+_gene_637 -- region: 637, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\637_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.579s.\n",
      "- Designed 0 candidate probes in 0.872s.\n",
      "-- designing region: 9:54985001-55015000_strand_+_gene_638 -- region: 638, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\638_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.574s.\n",
      "- Designed 0 candidate probes in 0.868s.\n",
      "-- designing region: 9:56985001-57015000_strand_+_gene_639 -- region: 639, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\639_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.558s.\n",
      "- Designed 0 candidate probes in 0.852s.\n",
      "-- designing region: 9:58985001-59015000_strand_+_gene_640 -- region: 640, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\640_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.581s.\n",
      "- Designed 0 candidate probes in 0.875s.\n",
      "-- designing region: 9:60985001-61015000_strand_+_gene_641 -- region: 641, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\641_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.604s.\n",
      "- Designed 37692 candidate probes in 10.403s.\n",
      "-- designing region: 9:62985001-63015000_strand_+_gene_642 -- region: 642, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\642_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.233s.\n",
      "- Designed 13176 candidate probes in 4.141s.\n",
      "-- designing region: 9:64985001-65015000_strand_+_gene_643 -- region: 643, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\643_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.223s.\n",
      "- Designed 26166 candidate probes in 7.211s.\n",
      "-- designing region: 9:66985001-67015000_strand_+_gene_644 -- region: 644, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\644_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.545s.\n",
      "-- designing region: 9:68985001-69015000_strand_+_gene_645 -- region: 645, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\645_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.264s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 9:70985001-71015000_strand_+_gene_646 -- region: 646, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\646_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 9:72985001-73015000_strand_+_gene_647 -- region: 647, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\647_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 9:74985001-75015000_strand_+_gene_648 -- region: 648, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\648_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 9:76985001-77015000_strand_+_gene_649 -- region: 649, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\649_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 20.911s.\n",
      "-- designing region: 9:78985001-79015000_strand_+_gene_650 -- region: 650, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\650_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 18.399s.\n",
      "-- designing region: 9:80985001-81015000_strand_+_gene_651 -- region: 651, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\651_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 9:82985001-83015000_strand_+_gene_652 -- region: 652, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\652_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.690s.\n",
      "-- designing region: 9:84985001-85015000_strand_+_gene_653 -- region: 653, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\653_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 9:86985001-87015000_strand_+_gene_654 -- region: 654, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\654_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.669s.\n",
      "-- designing region: 9:88985001-89015000_strand_+_gene_655 -- region: 655, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\655_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.695s.\n",
      "-- designing region: 9:90985001-91015000_strand_+_gene_656 -- region: 656, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\656_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.738s.\n",
      "-- designing region: 9:92985001-93015000_strand_+_gene_657 -- region: 657, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\657_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.709s.\n",
      "-- designing region: 9:94985001-95015000_strand_+_gene_658 -- region: 658, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\658_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 9:96985001-97015000_strand_+_gene_659 -- region: 659, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\659_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 9:98985001-99015000_strand_+_gene_660 -- region: 660, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\660_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.747s.\n",
      "-- designing region: 9:100985001-101015000_strand_+_gene_661 -- region: 661, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\661_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.690s.\n",
      "-- designing region: 9:102985001-103015000_strand_+_gene_662 -- region: 662, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\662_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.538s.\n",
      "-- designing region: 9:104985001-105015000_strand_+_gene_663 -- region: 663, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\663_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 9:106985001-107015000_strand_+_gene_664 -- region: 664, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\664_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.654s.\n",
      "-- designing region: 9:108985001-109015000_strand_+_gene_665 -- region: 665, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\665_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 9:110985001-111015000_strand_+_gene_666 -- region: 666, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\666_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 9:112985001-113015000_strand_+_gene_667 -- region: 667, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\667_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 9:114985001-115015000_strand_+_gene_668 -- region: 668, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\668_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 9:116985001-117015000_strand_+_gene_669 -- region: 669, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\669_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.643s.\n",
      "-- designing region: 9:118985001-119015000_strand_+_gene_670 -- region: 670, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\670_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.601s.\n",
      "-- designing region: 9:120985001-121015000_strand_+_gene_671 -- region: 671, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\671_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 9:122985001-123015000_strand_+_gene_672 -- region: 672, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\672_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 9:124985001-125015000_strand_+_gene_673 -- region: 673, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\673_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.577s.\n",
      "-- designing region: 9:126985001-127015000_strand_+_gene_674 -- region: 674, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\674_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.738s.\n",
      "-- designing region: 9:128985001-129015000_strand_+_gene_675 -- region: 675, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\675_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.684s.\n",
      "-- designing region: 9:130985001-131015000_strand_+_gene_676 -- region: 676, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\676_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.696s.\n",
      "-- designing region: 9:132985001-133015000_strand_+_gene_677 -- region: 677, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\677_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 9:134985001-135015000_strand_+_gene_678 -- region: 678, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\678_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.698s.\n",
      "-- designing region: 9:136985001-137015000_strand_+_gene_679 -- region: 679, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\679_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.665s.\n",
      "-- designing region: 10:985001-1015000_strand_+_gene_680 -- region: 680, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\680_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.638s.\n",
      "-- designing region: 10:2985001-3015000_strand_+_gene_681 -- region: 681, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\681_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.690s.\n",
      "-- designing region: 10:4985001-5015000_strand_+_gene_682 -- region: 682, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\682_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 10:6985001-7015000_strand_+_gene_683 -- region: 683, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\683_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.624s.\n",
      "-- designing region: 10:8985001-9015000_strand_+_gene_684 -- region: 684, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\684_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 10:10985001-11015000_strand_+_gene_685 -- region: 685, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\685_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 10:12985001-13015000_strand_+_gene_686 -- region: 686, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\686_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 10:14985001-15015000_strand_+_gene_687 -- region: 687, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\687_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 10:16985001-17015000_strand_+_gene_688 -- region: 688, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\688_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 10:18985001-19015000_strand_+_gene_689 -- region: 689, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\689_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.539s.\n",
      "-- designing region: 10:20985001-21015000_strand_+_gene_690 -- region: 690, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\690_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 10:22985001-23015000_strand_+_gene_691 -- region: 691, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\691_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 10:24985001-25015000_strand_+_gene_692 -- region: 692, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\692_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 10:26985001-27015000_strand_+_gene_693 -- region: 693, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\693_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 16.019s.\n",
      "-- designing region: 10:28985001-29015000_strand_+_gene_694 -- region: 694, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\694_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 10:30985001-31015000_strand_+_gene_695 -- region: 695, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\695_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 10:32985001-33015000_strand_+_gene_696 -- region: 696, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\696_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 10:34985001-35015000_strand_+_gene_697 -- region: 697, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\697_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 10:36985001-37015000_strand_+_gene_698 -- region: 698, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\698_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.544s.\n",
      "-- designing region: 10:38985001-39015000_strand_+_gene_699 -- region: 699, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\699_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 10:40985001-41015000_strand_+_gene_700 -- region: 700, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\700_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.227s.\n",
      "- Designed 59918 candidate probes in 15.326s.\n",
      "-- designing region: 10:42985001-43015000_strand_+_gene_701 -- region: 701, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\701_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 10:44985001-45015000_strand_+_gene_702 -- region: 702, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\702_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 10:46985001-47015000_strand_+_gene_703 -- region: 703, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\703_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 10:48985001-49015000_strand_+_gene_704 -- region: 704, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\704_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 10:50985001-51015000_strand_+_gene_705 -- region: 705, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\705_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 10:52985001-53015000_strand_+_gene_706 -- region: 706, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\706_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.580s.\n",
      "-- designing region: 10:54985001-55015000_strand_+_gene_707 -- region: 707, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\707_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 10:56985001-57015000_strand_+_gene_708 -- region: 708, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\708_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.779s.\n",
      "-- designing region: 10:58985001-59015000_strand_+_gene_709 -- region: 709, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\709_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 10:60985001-61015000_strand_+_gene_710 -- region: 710, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\710_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 22.786s.\n",
      "-- designing region: 10:62985001-63015000_strand_+_gene_711 -- region: 711, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\711_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.632s.\n",
      "- Designed 59918 candidate probes in 17.746s.\n",
      "-- designing region: 10:64985001-65015000_strand_+_gene_712 -- region: 712, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\712_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 10:66985001-67015000_strand_+_gene_713 -- region: 713, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\713_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 10:68985001-69015000_strand_+_gene_714 -- region: 714, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\714_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.643s.\n",
      "-- designing region: 10:70985001-71015000_strand_+_gene_715 -- region: 715, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\715_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.727s.\n",
      "-- designing region: 10:72985001-73015000_strand_+_gene_716 -- region: 716, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\716_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.669s.\n",
      "-- designing region: 10:74985001-75015000_strand_+_gene_717 -- region: 717, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\717_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 10:76985001-77015000_strand_+_gene_718 -- region: 718, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\718_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.730s.\n",
      "-- designing region: 10:78985001-79015000_strand_+_gene_719 -- region: 719, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\719_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.689s.\n",
      "-- designing region: 10:80985001-81015000_strand_+_gene_720 -- region: 720, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\720_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 10:82985001-83015000_strand_+_gene_721 -- region: 721, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\721_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.642s.\n",
      "-- designing region: 10:84985001-85015000_strand_+_gene_722 -- region: 722, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\722_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.665s.\n",
      "-- designing region: 10:86985001-87015000_strand_+_gene_723 -- region: 723, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\723_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 54.988s.\n",
      "-- designing region: 10:88985001-89015000_strand_+_gene_724 -- region: 724, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\724_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 10:90985001-91015000_strand_+_gene_725 -- region: 725, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\725_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 10:92985001-93015000_strand_+_gene_726 -- region: 726, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\726_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 10:94985001-95015000_strand_+_gene_727 -- region: 727, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\727_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 10:96985001-97015000_strand_+_gene_728 -- region: 728, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\728_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.656s.\n",
      "-- designing region: 10:98985001-99015000_strand_+_gene_729 -- region: 729, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\729_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 10:100985001-101015000_strand_+_gene_730 -- region: 730, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\730_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.681s.\n",
      "-- designing region: 10:102985001-103015000_strand_+_gene_731 -- region: 731, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\731_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.613s.\n",
      "-- designing region: 10:104985001-105015000_strand_+_gene_732 -- region: 732, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\732_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 10:106985001-107015000_strand_+_gene_733 -- region: 733, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\733_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.651s.\n",
      "-- designing region: 10:108985001-109015000_strand_+_gene_734 -- region: 734, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\734_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.599s.\n",
      "-- designing region: 10:110985001-111015000_strand_+_gene_735 -- region: 735, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\735_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 10:112985001-113015000_strand_+_gene_736 -- region: 736, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\736_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.651s.\n",
      "-- designing region: 10:114985001-115015000_strand_+_gene_737 -- region: 737, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\737_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 10:116985001-117015000_strand_+_gene_738 -- region: 738, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\738_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 10:118985001-119015000_strand_+_gene_739 -- region: 739, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\739_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.610s.\n",
      "-- designing region: 10:120985001-121015000_strand_+_gene_740 -- region: 740, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\740_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 10:122985001-123015000_strand_+_gene_741 -- region: 741, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\741_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 10:124985001-125015000_strand_+_gene_742 -- region: 742, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\742_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.640s.\n",
      "-- designing region: 10:126985001-127015000_strand_+_gene_743 -- region: 743, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\743_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 10:128985001-129015000_strand_+_gene_744 -- region: 744, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\744_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 10:130985001-131015000_strand_+_gene_745 -- region: 745, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\745_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.640s.\n",
      "-- designing region: 11:985001-1015000_strand_+_gene_746 -- region: 746, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\746_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 11:2985001-3015000_strand_+_gene_747 -- region: 747, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\747_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.622s.\n",
      "-- designing region: 11:4985001-5015000_strand_+_gene_748 -- region: 748, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\748_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 11:6985001-7015000_strand_+_gene_749 -- region: 749, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\749_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 11:8985001-9015000_strand_+_gene_750 -- region: 750, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\750_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 11:10985001-11015000_strand_+_gene_751 -- region: 751, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\751_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 11:12985001-13015000_strand_+_gene_752 -- region: 752, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\752_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 11:14985001-15015000_strand_+_gene_753 -- region: 753, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\753_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.599s.\n",
      "-- designing region: 11:16985001-17015000_strand_+_gene_754 -- region: 754, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\754_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 11:18985001-19015000_strand_+_gene_755 -- region: 755, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\755_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.562s.\n",
      "-- designing region: 11:20985001-21015000_strand_+_gene_756 -- region: 756, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\756_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 11:22985001-23015000_strand_+_gene_757 -- region: 757, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\757_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.586s.\n",
      "-- designing region: 11:24985001-25015000_strand_+_gene_758 -- region: 758, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\758_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.627s.\n",
      "-- designing region: 11:26985001-27015000_strand_+_gene_759 -- region: 759, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\759_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 11:28985001-29015000_strand_+_gene_760 -- region: 760, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\760_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.518s.\n",
      "-- designing region: 11:30985001-31015000_strand_+_gene_761 -- region: 761, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\761_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.505s.\n",
      "-- designing region: 11:32985001-33015000_strand_+_gene_762 -- region: 762, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\762_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.507s.\n",
      "-- designing region: 11:34985001-35015000_strand_+_gene_763 -- region: 763, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\763_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 11:36985001-37015000_strand_+_gene_764 -- region: 764, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\764_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 11:38985001-39015000_strand_+_gene_765 -- region: 765, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\765_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 11:40985001-41015000_strand_+_gene_766 -- region: 766, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\766_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.517s.\n",
      "-- designing region: 11:42985001-43015000_strand_+_gene_767 -- region: 767, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\767_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.761s.\n",
      "-- designing region: 11:44985001-45015000_strand_+_gene_768 -- region: 768, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\768_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.520s.\n",
      "-- designing region: 11:46985001-47015000_strand_+_gene_769 -- region: 769, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\769_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 22.997s.\n",
      "-- designing region: 11:48985001-49015000_strand_+_gene_770 -- region: 770, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\770_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.306s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 11:50985001-51015000_strand_+_gene_771 -- region: 771, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\771_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.210s.\n",
      "- Designed 0 candidate probes in 0.509s.\n",
      "-- designing region: 11:52985001-53015000_strand_+_gene_772 -- region: 772, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\772_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.595s.\n",
      "- Designed 59918 candidate probes in 15.892s.\n",
      "-- designing region: 11:54985001-55015000_strand_+_gene_773 -- region: 773, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\773_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.261s.\n",
      "- Designed 59918 candidate probes in 15.514s.\n",
      "-- designing region: 11:56985001-57015000_strand_+_gene_774 -- region: 774, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\774_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 11:58985001-59015000_strand_+_gene_775 -- region: 775, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\775_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 11:60985001-61015000_strand_+_gene_776 -- region: 776, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\776_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.682s.\n",
      "-- designing region: 11:62985001-63015000_strand_+_gene_777 -- region: 777, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\777_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.751s.\n",
      "-- designing region: 11:64985001-65015000_strand_+_gene_778 -- region: 778, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\778_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.655s.\n",
      "-- designing region: 11:66985001-67015000_strand_+_gene_779 -- region: 779, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\779_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.711s.\n",
      "-- designing region: 11:68985001-69015000_strand_+_gene_780 -- region: 780, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\780_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.707s.\n",
      "-- designing region: 11:70985001-71015000_strand_+_gene_781 -- region: 781, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\781_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.204s.\n",
      "- Designed 0 candidate probes in 0.511s.\n",
      "-- designing region: 11:72985001-73015000_strand_+_gene_782 -- region: 782, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\782_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.640s.\n",
      "- Designed 59918 candidate probes in 16.006s.\n",
      "-- designing region: 11:74985001-75015000_strand_+_gene_783 -- region: 783, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\783_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 11:76985001-77015000_strand_+_gene_784 -- region: 784, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\784_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.678s.\n",
      "-- designing region: 11:78985001-79015000_strand_+_gene_785 -- region: 785, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\785_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.650s.\n",
      "-- designing region: 11:80985001-81015000_strand_+_gene_786 -- region: 786, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\786_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 11:82985001-83015000_strand_+_gene_787 -- region: 787, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\787_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 11:84985001-85015000_strand_+_gene_788 -- region: 788, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\788_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 11:86985001-87015000_strand_+_gene_789 -- region: 789, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\789_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.589s.\n",
      "-- designing region: 11:88985001-89015000_strand_+_gene_790 -- region: 790, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\790_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.634s.\n",
      "-- designing region: 11:90985001-91015000_strand_+_gene_791 -- region: 791, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\791_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 11:92985001-93015000_strand_+_gene_792 -- region: 792, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\792_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: 11:94985001-95015000_strand_+_gene_793 -- region: 793, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\793_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 11:96985001-97015000_strand_+_gene_794 -- region: 794, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\794_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 20.724s.\n",
      "-- designing region: 11:98985001-99015000_strand_+_gene_795 -- region: 795, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\795_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: 11:100985001-101015000_strand_+_gene_796 -- region: 796, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\796_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 11:102985001-103015000_strand_+_gene_797 -- region: 797, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\797_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.410s.\n",
      "-- designing region: 11:104985001-105015000_strand_+_gene_798 -- region: 798, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\798_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.447s.\n",
      "-- designing region: 11:106985001-107015000_strand_+_gene_799 -- region: 799, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\799_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 11:108985001-109015000_strand_+_gene_800 -- region: 800, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\800_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 11:110985001-111015000_strand_+_gene_801 -- region: 801, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\801_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.555s.\n",
      "-- designing region: 11:112985001-113015000_strand_+_gene_802 -- region: 802, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\802_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 11:114985001-115015000_strand_+_gene_803 -- region: 803, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\803_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.536s.\n",
      "-- designing region: 11:116985001-117015000_strand_+_gene_804 -- region: 804, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\804_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 11:118985001-119015000_strand_+_gene_805 -- region: 805, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\805_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 11:120985001-121015000_strand_+_gene_806 -- region: 806, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\806_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.566s.\n",
      "-- designing region: 11:122985001-123015000_strand_+_gene_807 -- region: 807, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\807_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.518s.\n",
      "-- designing region: 11:124985001-125015000_strand_+_gene_808 -- region: 808, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\808_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.540s.\n",
      "-- designing region: 11:126985001-127015000_strand_+_gene_809 -- region: 809, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\809_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 11:128985001-129015000_strand_+_gene_810 -- region: 810, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\810_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.514s.\n",
      "-- designing region: 11:130985001-131015000_strand_+_gene_811 -- region: 811, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\811_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.545s.\n",
      "-- designing region: 11:132985001-133015000_strand_+_gene_812 -- region: 812, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\812_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.569s.\n",
      "-- designing region: 12:985001-1015000_strand_+_gene_813 -- region: 813, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\813_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.530s.\n",
      "-- designing region: 12:2985001-3015000_strand_+_gene_814 -- region: 814, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\814_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 12:4985001-5015000_strand_+_gene_815 -- region: 815, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\815_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.610s.\n",
      "-- designing region: 12:6985001-7015000_strand_+_gene_816 -- region: 816, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\816_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 12:8985001-9015000_strand_+_gene_817 -- region: 817, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\817_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.539s.\n",
      "-- designing region: 12:10985001-11015000_strand_+_gene_818 -- region: 818, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\818_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.484s.\n",
      "-- designing region: 12:12985001-13015000_strand_+_gene_819 -- region: 819, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\819_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 12:14985001-15015000_strand_+_gene_820 -- region: 820, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\820_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 12:16985001-17015000_strand_+_gene_821 -- region: 821, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\821_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.524s.\n",
      "-- designing region: 12:18985001-19015000_strand_+_gene_822 -- region: 822, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\822_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.460s.\n",
      "-- designing region: 12:20985001-21015000_strand_+_gene_823 -- region: 823, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\823_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.479s.\n",
      "-- designing region: 12:22985001-23015000_strand_+_gene_824 -- region: 824, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\824_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 12:24985001-25015000_strand_+_gene_825 -- region: 825, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\825_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.483s.\n",
      "-- designing region: 12:26985001-27015000_strand_+_gene_826 -- region: 826, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\826_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 12:28985001-29015000_strand_+_gene_827 -- region: 827, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\827_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 12:30985001-31015000_strand_+_gene_828 -- region: 828, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\828_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 12:32985001-33015000_strand_+_gene_829 -- region: 829, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\829_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.526s.\n",
      "-- designing region: 12:34985001-35015000_strand_+_gene_830 -- region: 830, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\830_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.217s.\n",
      "- Designed 59918 candidate probes in 15.718s.\n",
      "-- designing region: 12:36985001-37015000_strand_+_gene_831 -- region: 831, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\831_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.228s.\n",
      "- Designed 59918 candidate probes in 15.279s.\n",
      "-- designing region: 12:38985001-39015000_strand_+_gene_832 -- region: 832, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\832_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 20.195s.\n",
      "-- designing region: 12:40985001-41015000_strand_+_gene_833 -- region: 833, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\833_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 18.488s.\n",
      "-- designing region: 12:42985001-43015000_strand_+_gene_834 -- region: 834, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\834_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 12:44985001-45015000_strand_+_gene_835 -- region: 835, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\835_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 12:46985001-47015000_strand_+_gene_836 -- region: 836, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\836_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 12:48985001-49015000_strand_+_gene_837 -- region: 837, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\837_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.647s.\n",
      "-- designing region: 12:50985001-51015000_strand_+_gene_838 -- region: 838, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\838_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 12:52985001-53015000_strand_+_gene_839 -- region: 839, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\839_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.231s.\n",
      "- Designed 59918 candidate probes in 15.544s.\n",
      "-- designing region: 12:54985001-55015000_strand_+_gene_840 -- region: 840, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\840_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.641s.\n",
      "-- designing region: 12:56985001-57015000_strand_+_gene_841 -- region: 841, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\841_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.651s.\n",
      "-- designing region: 12:58985001-59015000_strand_+_gene_842 -- region: 842, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\842_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 12:60985001-61015000_strand_+_gene_843 -- region: 843, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\843_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 12:62985001-63015000_strand_+_gene_844 -- region: 844, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\844_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.605s.\n",
      "-- designing region: 12:64985001-65015000_strand_+_gene_845 -- region: 845, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\845_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 12:66985001-67015000_strand_+_gene_846 -- region: 846, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\846_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 12:68985001-69015000_strand_+_gene_847 -- region: 847, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\847_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 12:70985001-71015000_strand_+_gene_848 -- region: 848, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\848_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 12:72985001-73015000_strand_+_gene_849 -- region: 849, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\849_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 12:74985001-75015000_strand_+_gene_850 -- region: 850, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\850_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.577s.\n",
      "-- designing region: 12:76985001-77015000_strand_+_gene_851 -- region: 851, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\851_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 12:78985001-79015000_strand_+_gene_852 -- region: 852, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\852_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 12:80985001-81015000_strand_+_gene_853 -- region: 853, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\853_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.586s.\n",
      "-- designing region: 12:82985001-83015000_strand_+_gene_854 -- region: 854, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\854_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 12:84985001-85015000_strand_+_gene_855 -- region: 855, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\855_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: 12:86985001-87015000_strand_+_gene_856 -- region: 856, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\856_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 12:88985001-89015000_strand_+_gene_857 -- region: 857, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\857_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 12:90985001-91015000_strand_+_gene_858 -- region: 858, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\858_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 12:92985001-93015000_strand_+_gene_859 -- region: 859, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\859_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 12:94985001-95015000_strand_+_gene_860 -- region: 860, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\860_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 12:96985001-97015000_strand_+_gene_861 -- region: 861, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\861_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.586s.\n",
      "-- designing region: 12:98985001-99015000_strand_+_gene_862 -- region: 862, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\862_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 12:100985001-101015000_strand_+_gene_863 -- region: 863, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\863_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 12:102985001-103015000_strand_+_gene_864 -- region: 864, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\864_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 12:104985001-105015000_strand_+_gene_865 -- region: 865, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\865_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.577s.\n",
      "-- designing region: 12:106985001-107015000_strand_+_gene_866 -- region: 866, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\866_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 12:108985001-109015000_strand_+_gene_867 -- region: 867, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\867_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.599s.\n",
      "-- designing region: 12:110985001-111015000_strand_+_gene_868 -- region: 868, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\868_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 12:112985001-113015000_strand_+_gene_869 -- region: 869, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\869_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 12:114985001-115015000_strand_+_gene_870 -- region: 870, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\870_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 12:116985001-117015000_strand_+_gene_871 -- region: 871, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\871_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.516s.\n",
      "-- designing region: 12:118985001-119015000_strand_+_gene_872 -- region: 872, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\872_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.311s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 12:120985001-121015000_strand_+_gene_873 -- region: 873, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\873_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.537s.\n",
      "-- designing region: 12:122985001-123015000_strand_+_gene_874 -- region: 874, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\874_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.601s.\n",
      "-- designing region: 12:124985001-125015000_strand_+_gene_875 -- region: 875, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\875_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 12:126985001-127015000_strand_+_gene_876 -- region: 876, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\876_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 12:128985001-129015000_strand_+_gene_877 -- region: 877, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\877_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.574s.\n",
      "-- designing region: 12:130985001-131015000_strand_+_gene_878 -- region: 878, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\878_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: 13:985001-1015000_strand_+_gene_879 -- region: 879, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\879_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.209s.\n",
      "- Designed 0 candidate probes in 0.500s.\n",
      "-- designing region: 13:2985001-3015000_strand_+_gene_880 -- region: 880, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\880_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.565s.\n",
      "- Designed 0 candidate probes in 0.859s.\n",
      "-- designing region: 13:4985001-5015000_strand_+_gene_881 -- region: 881, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\881_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.844s.\n",
      "-- designing region: 13:6985001-7015000_strand_+_gene_882 -- region: 882, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\882_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: 13:8985001-9015000_strand_+_gene_883 -- region: 883, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\883_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: 13:10985001-11015000_strand_+_gene_884 -- region: 884, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\884_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.559s.\n",
      "- Designed 0 candidate probes in 0.850s.\n",
      "-- designing region: 13:12985001-13015000_strand_+_gene_885 -- region: 885, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\885_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: 13:14985001-15015000_strand_+_gene_886 -- region: 886, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\886_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.561s.\n",
      "- Designed 0 candidate probes in 0.853s.\n",
      "-- designing region: 13:16985001-17015000_strand_+_gene_887 -- region: 887, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\887_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.570s.\n",
      "- Designed 59918 candidate probes in 15.641s.\n",
      "-- designing region: 13:18985001-19015000_strand_+_gene_888 -- region: 888, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\888_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 13:20985001-21015000_strand_+_gene_889 -- region: 889, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\889_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 13:22985001-23015000_strand_+_gene_890 -- region: 890, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\890_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.530s.\n",
      "-- designing region: 13:24985001-25015000_strand_+_gene_891 -- region: 891, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\891_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.457s.\n",
      "-- designing region: 13:26985001-27015000_strand_+_gene_892 -- region: 892, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\892_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.496s.\n",
      "-- designing region: 13:28985001-29015000_strand_+_gene_893 -- region: 893, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\893_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.476s.\n",
      "-- designing region: 13:30985001-31015000_strand_+_gene_894 -- region: 894, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\894_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.460s.\n",
      "-- designing region: 13:32985001-33015000_strand_+_gene_895 -- region: 895, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\895_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.476s.\n",
      "-- designing region: 13:34985001-35015000_strand_+_gene_896 -- region: 896, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\896_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.459s.\n",
      "-- designing region: 13:36985001-37015000_strand_+_gene_897 -- region: 897, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\897_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.440s.\n",
      "-- designing region: 13:38985001-39015000_strand_+_gene_898 -- region: 898, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\898_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.462s.\n",
      "-- designing region: 13:40985001-41015000_strand_+_gene_899 -- region: 899, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\899_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.437s.\n",
      "-- designing region: 13:42985001-43015000_strand_+_gene_900 -- region: 900, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\900_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 13:44985001-45015000_strand_+_gene_901 -- region: 901, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\901_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 17.715s.\n",
      "-- designing region: 13:46985001-47015000_strand_+_gene_902 -- region: 902, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\902_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.364s.\n",
      "- Designed 59918 candidate probes in 22.833s.\n",
      "-- designing region: 13:48985001-49015000_strand_+_gene_903 -- region: 903, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\903_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.496s.\n",
      "- Designed 59918 candidate probes in 15.739s.\n",
      "-- designing region: 13:50985001-51015000_strand_+_gene_904 -- region: 904, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\904_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 13:52985001-53015000_strand_+_gene_905 -- region: 905, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\905_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.663s.\n",
      "-- designing region: 13:54985001-55015000_strand_+_gene_906 -- region: 906, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\906_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.659s.\n",
      "-- designing region: 13:56985001-57015000_strand_+_gene_907 -- region: 907, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\907_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.616s.\n",
      "-- designing region: 13:58985001-59015000_strand_+_gene_908 -- region: 908, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\908_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 13:60985001-61015000_strand_+_gene_909 -- region: 909, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\909_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 13:62985001-63015000_strand_+_gene_910 -- region: 910, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\910_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.616s.\n",
      "-- designing region: 13:64985001-65015000_strand_+_gene_911 -- region: 911, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\911_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 13:66985001-67015000_strand_+_gene_912 -- region: 912, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\912_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.648s.\n",
      "-- designing region: 13:68985001-69015000_strand_+_gene_913 -- region: 913, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\913_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.584s.\n",
      "-- designing region: 13:70985001-71015000_strand_+_gene_914 -- region: 914, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\914_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 13:72985001-73015000_strand_+_gene_915 -- region: 915, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\915_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 13:74985001-75015000_strand_+_gene_916 -- region: 916, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\916_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.210s.\n",
      "- Designed 59918 candidate probes in 15.522s.\n",
      "-- designing region: 13:76985001-77015000_strand_+_gene_917 -- region: 917, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\917_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 13:78985001-79015000_strand_+_gene_918 -- region: 918, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\918_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.589s.\n",
      "-- designing region: 13:80985001-81015000_strand_+_gene_919 -- region: 919, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\919_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.669s.\n",
      "-- designing region: 13:82985001-83015000_strand_+_gene_920 -- region: 920, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\920_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.545s.\n",
      "-- designing region: 13:84985001-85015000_strand_+_gene_921 -- region: 921, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\921_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.518s.\n",
      "-- designing region: 13:86985001-87015000_strand_+_gene_922 -- region: 922, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\922_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.643s.\n",
      "-- designing region: 13:88985001-89015000_strand_+_gene_923 -- region: 923, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\923_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.487s.\n",
      "-- designing region: 13:90985001-91015000_strand_+_gene_924 -- region: 924, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\924_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 13:92985001-93015000_strand_+_gene_925 -- region: 925, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\925_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 64.615s.\n",
      "-- designing region: 13:94985001-95015000_strand_+_gene_926 -- region: 926, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\926_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 13:96985001-97015000_strand_+_gene_927 -- region: 927, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\927_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 13:98985001-99015000_strand_+_gene_928 -- region: 928, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\928_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 13:100985001-101015000_strand_+_gene_929 -- region: 929, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\929_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.610s.\n",
      "-- designing region: 13:102985001-103015000_strand_+_gene_930 -- region: 930, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\930_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 13:104985001-105015000_strand_+_gene_931 -- region: 931, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\931_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 13:106985001-107015000_strand_+_gene_932 -- region: 932, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\932_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 13:108985001-109015000_strand_+_gene_933 -- region: 933, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\933_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 13:110985001-111015000_strand_+_gene_934 -- region: 934, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\934_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 13:112985001-113015000_strand_+_gene_935 -- region: 935, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\935_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 14:735001-765000_strand_+_gene_936 -- region: 936, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\936_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.210s.\n",
      "- Designed 0 candidate probes in 0.502s.\n",
      "-- designing region: 14:2235001-2265000_strand_+_gene_937 -- region: 937, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\937_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.550s.\n",
      "- Designed 0 candidate probes in 0.844s.\n",
      "-- designing region: 14:3735001-3765000_strand_+_gene_938 -- region: 938, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\938_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 14:5235001-5265000_strand_+_gene_939 -- region: 939, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\939_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: 14:6735001-6765000_strand_+_gene_940 -- region: 940, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\940_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 14:8235001-8265000_strand_+_gene_941 -- region: 941, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\941_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.552s.\n",
      "- Designed 0 candidate probes in 0.845s.\n",
      "-- designing region: 14:9735001-9765000_strand_+_gene_942 -- region: 942, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\942_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.564s.\n",
      "- Designed 0 candidate probes in 0.858s.\n",
      "-- designing region: 14:11235001-11265000_strand_+_gene_943 -- region: 943, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\943_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.549s.\n",
      "- Designed 0 candidate probes in 0.842s.\n",
      "-- designing region: 14:12735001-12765000_strand_+_gene_944 -- region: 944, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\944_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.574s.\n",
      "- Designed 0 candidate probes in 0.868s.\n",
      "-- designing region: 14:14235001-14265000_strand_+_gene_945 -- region: 945, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\945_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.557s.\n",
      "- Designed 0 candidate probes in 0.850s.\n",
      "-- designing region: 14:15735001-15765000_strand_+_gene_946 -- region: 946, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\946_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.557s.\n",
      "- Designed 0 candidate probes in 0.852s.\n",
      "-- designing region: 14:17235001-17265000_strand_+_gene_947 -- region: 947, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\947_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.574s.\n",
      "- Designed 59918 candidate probes in 15.722s.\n",
      "-- designing region: 14:18735001-18765000_strand_+_gene_948 -- region: 948, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\948_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "- Designed 0 candidate probes in 0.505s.\n",
      "-- designing region: 14:20235001-20265000_strand_+_gene_949 -- region: 949, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\949_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.626s.\n",
      "- Designed 59918 candidate probes in 15.687s.\n",
      "-- designing region: 14:21735001-21765000_strand_+_gene_950 -- region: 950, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\950_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.540s.\n",
      "-- designing region: 14:23235001-23265000_strand_+_gene_951 -- region: 951, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\951_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: 14:24735001-24765000_strand_+_gene_952 -- region: 952, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\952_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.529s.\n",
      "-- designing region: 14:26235001-26265000_strand_+_gene_953 -- region: 953, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\953_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.447s.\n",
      "-- designing region: 14:27735001-27765000_strand_+_gene_954 -- region: 954, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\954_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 14:29235001-29265000_strand_+_gene_955 -- region: 955, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\955_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "- Designed 59918 candidate probes in 15.406s.\n",
      "-- designing region: 14:30735001-30765000_strand_+_gene_956 -- region: 956, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\956_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.441s.\n",
      "-- designing region: 14:32235001-32265000_strand_+_gene_957 -- region: 957, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\957_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.529s.\n",
      "-- designing region: 14:33735001-33765000_strand_+_gene_958 -- region: 958, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\958_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.477s.\n",
      "-- designing region: 14:35235001-35265000_strand_+_gene_959 -- region: 959, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\959_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "- Designed 59918 candidate probes in 15.423s.\n",
      "-- designing region: 14:36735001-36765000_strand_+_gene_960 -- region: 960, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\960_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.445s.\n",
      "-- designing region: 14:38235001-38265000_strand_+_gene_961 -- region: 961, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\961_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.461s.\n",
      "-- designing region: 14:39735001-39765000_strand_+_gene_962 -- region: 962, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\962_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.434s.\n",
      "-- designing region: 14:41235001-41265000_strand_+_gene_963 -- region: 963, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\963_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.421s.\n",
      "-- designing region: 14:42735001-42765000_strand_+_gene_964 -- region: 964, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\964_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.387s.\n",
      "-- designing region: 14:44235001-44265000_strand_+_gene_965 -- region: 965, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\965_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.413s.\n",
      "-- designing region: 14:45735001-45765000_strand_+_gene_966 -- region: 966, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\966_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.417s.\n",
      "-- designing region: 14:47235001-47265000_strand_+_gene_967 -- region: 967, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\967_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.412s.\n",
      "-- designing region: 14:48735001-48765000_strand_+_gene_968 -- region: 968, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\968_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.461s.\n",
      "-- designing region: 14:50235001-50265000_strand_+_gene_969 -- region: 969, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\969_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 14:51735001-51765000_strand_+_gene_970 -- region: 970, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\970_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.430s.\n",
      "-- designing region: 14:53235001-53265000_strand_+_gene_971 -- region: 971, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\971_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 19.509s.\n",
      "-- designing region: 14:54735001-54765000_strand_+_gene_972 -- region: 972, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\972_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.614s.\n",
      "- Designed 59918 candidate probes in 19.927s.\n",
      "-- designing region: 14:56235001-56265000_strand_+_gene_973 -- region: 973, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\973_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 14:57735001-57765000_strand_+_gene_974 -- region: 974, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\974_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 14:59235001-59265000_strand_+_gene_975 -- region: 975, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\975_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.666s.\n",
      "-- designing region: 14:60735001-60765000_strand_+_gene_976 -- region: 976, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\976_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.661s.\n",
      "-- designing region: 14:62235001-62265000_strand_+_gene_977 -- region: 977, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\977_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.670s.\n",
      "-- designing region: 14:63735001-63765000_strand_+_gene_978 -- region: 978, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\978_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 14:65235001-65265000_strand_+_gene_979 -- region: 979, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\979_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.689s.\n",
      "-- designing region: 14:66735001-66765000_strand_+_gene_980 -- region: 980, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\980_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.613s.\n",
      "-- designing region: 14:68235001-68265000_strand_+_gene_981 -- region: 981, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\981_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.634s.\n",
      "-- designing region: 14:69735001-69765000_strand_+_gene_982 -- region: 982, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\982_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 14:71235001-71265000_strand_+_gene_983 -- region: 983, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\983_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 14:72735001-72765000_strand_+_gene_984 -- region: 984, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\984_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.674s.\n",
      "-- designing region: 14:74235001-74265000_strand_+_gene_985 -- region: 985, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\985_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.671s.\n",
      "-- designing region: 14:75735001-75765000_strand_+_gene_986 -- region: 986, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\986_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.642s.\n",
      "-- designing region: 14:77235001-77265000_strand_+_gene_987 -- region: 987, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\987_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.718s.\n",
      "-- designing region: 14:78735001-78765000_strand_+_gene_988 -- region: 988, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\988_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.634s.\n",
      "-- designing region: 14:80235001-80265000_strand_+_gene_989 -- region: 989, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\989_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 14:81735001-81765000_strand_+_gene_990 -- region: 990, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\990_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 14:83235001-83265000_strand_+_gene_991 -- region: 991, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\991_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 14:84735001-84765000_strand_+_gene_992 -- region: 992, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\992_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 14:86235001-86265000_strand_+_gene_993 -- region: 993, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\993_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.584s.\n",
      "-- designing region: 14:87735001-87765000_strand_+_gene_994 -- region: 994, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\994_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.612s.\n",
      "-- designing region: 14:89235001-89265000_strand_+_gene_995 -- region: 995, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\995_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.642s.\n",
      "-- designing region: 14:90735001-90765000_strand_+_gene_996 -- region: 996, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\996_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 14:92235001-92265000_strand_+_gene_997 -- region: 997, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\997_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 14:93735001-93765000_strand_+_gene_998 -- region: 998, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\998_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.681s.\n",
      "-- designing region: 14:95235001-95265000_strand_+_gene_999 -- region: 999, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\999_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 14:96735001-96765000_strand_+_gene_1000 -- region: 1000, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1000_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.687s.\n",
      "-- designing region: 14:98235001-98265000_strand_+_gene_1001 -- region: 1001, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1001_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.569s.\n",
      "-- designing region: 14:99735001-99765000_strand_+_gene_1002 -- region: 1002, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1002_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 14:101235001-101265000_strand_+_gene_1003 -- region: 1003, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1003_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 14:102735001-102765000_strand_+_gene_1004 -- region: 1004, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1004_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.504s.\n",
      "-- designing region: 14:104235001-104265000_strand_+_gene_1005 -- region: 1005, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1005_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.488s.\n",
      "-- designing region: 14:105735001-105765000_strand_+_gene_1006 -- region: 1006, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1006_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.477s.\n",
      "-- designing region: 15:735001-765000_strand_+_gene_1007 -- region: 1007, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1007_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "- Designed 0 candidate probes in 0.496s.\n",
      "-- designing region: 15:2235001-2265000_strand_+_gene_1008 -- region: 1008, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1008_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.562s.\n",
      "- Designed 0 candidate probes in 0.854s.\n",
      "-- designing region: 15:3735001-3765000_strand_+_gene_1009 -- region: 1009, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1009_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.558s.\n",
      "- Designed 0 candidate probes in 0.851s.\n",
      "-- designing region: 15:5235001-5265000_strand_+_gene_1010 -- region: 1010, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1010_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.557s.\n",
      "- Designed 0 candidate probes in 0.851s.\n",
      "-- designing region: 15:6735001-6765000_strand_+_gene_1011 -- region: 1011, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1011_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 15:8235001-8265000_strand_+_gene_1012 -- region: 1012, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1012_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.560s.\n",
      "- Designed 0 candidate probes in 0.854s.\n",
      "-- designing region: 15:9735001-9765000_strand_+_gene_1013 -- region: 1013, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1013_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: 15:11235001-11265000_strand_+_gene_1014 -- region: 1014, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1014_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.559s.\n",
      "- Designed 0 candidate probes in 0.853s.\n",
      "-- designing region: 15:12735001-12765000_strand_+_gene_1015 -- region: 1015, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1015_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.513s.\n",
      "- Designed 0 candidate probes in 0.808s.\n",
      "-- designing region: 15:14235001-14265000_strand_+_gene_1016 -- region: 1016, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1016_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.557s.\n",
      "- Designed 0 candidate probes in 0.851s.\n",
      "-- designing region: 15:15735001-15765000_strand_+_gene_1017 -- region: 1017, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1017_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.556s.\n",
      "- Designed 0 candidate probes in 0.850s.\n",
      "-- designing region: 15:17235001-17265000_strand_+_gene_1018 -- region: 1018, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1018_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.593s.\n",
      "- Designed 59918 candidate probes in 15.650s.\n",
      "-- designing region: 15:18735001-18765000_strand_+_gene_1019 -- region: 1019, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1019_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.221s.\n",
      "- Designed 59918 candidate probes in 15.232s.\n",
      "-- designing region: 15:20235001-20265000_strand_+_gene_1020 -- region: 1020, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1020_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "- Designed 59918 candidate probes in 15.316s.\n",
      "-- designing region: 15:21735001-21765000_strand_+_gene_1021 -- region: 1021, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1021_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 15:23235001-23265000_strand_+_gene_1022 -- region: 1022, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1022_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.202s.\n",
      "- Designed 0 candidate probes in 0.495s.\n",
      "-- designing region: 15:24735001-24765000_strand_+_gene_1023 -- region: 1023, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1023_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.627s.\n",
      "- Designed 59918 candidate probes in 15.783s.\n",
      "-- designing region: 15:26235001-26265000_strand_+_gene_1024 -- region: 1024, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1024_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.442s.\n",
      "-- designing region: 15:27735001-27765000_strand_+_gene_1025 -- region: 1025, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1025_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.476s.\n",
      "-- designing region: 15:29235001-29265000_strand_+_gene_1026 -- region: 1026, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1026_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.424s.\n",
      "-- designing region: 15:30735001-30765000_strand_+_gene_1027 -- region: 1027, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1027_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.442s.\n",
      "-- designing region: 15:32235001-32265000_strand_+_gene_1028 -- region: 1028, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1028_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.454s.\n",
      "-- designing region: 15:33735001-33765000_strand_+_gene_1029 -- region: 1029, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1029_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "- Designed 59918 candidate probes in 15.437s.\n",
      "-- designing region: 15:35235001-35265000_strand_+_gene_1030 -- region: 1030, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1030_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.446s.\n",
      "-- designing region: 15:36735001-36765000_strand_+_gene_1031 -- region: 1031, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1031_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.414s.\n",
      "-- designing region: 15:38235001-38265000_strand_+_gene_1032 -- region: 1032, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1032_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.449s.\n",
      "-- designing region: 15:39735001-39765000_strand_+_gene_1033 -- region: 1033, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1033_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.415s.\n",
      "-- designing region: 15:41235001-41265000_strand_+_gene_1034 -- region: 1034, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1034_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.424s.\n",
      "-- designing region: 15:42735001-42765000_strand_+_gene_1035 -- region: 1035, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1035_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.428s.\n",
      "-- designing region: 15:44235001-44265000_strand_+_gene_1036 -- region: 1036, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1036_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.444s.\n",
      "-- designing region: 15:45735001-45765000_strand_+_gene_1037 -- region: 1037, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1037_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.448s.\n",
      "-- designing region: 15:47235001-47265000_strand_+_gene_1038 -- region: 1038, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1038_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.414s.\n",
      "-- designing region: 15:48735001-48765000_strand_+_gene_1039 -- region: 1039, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1039_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.430s.\n",
      "-- designing region: 15:50235001-50265000_strand_+_gene_1040 -- region: 1040, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1040_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.458s.\n",
      "-- designing region: 15:51735001-51765000_strand_+_gene_1041 -- region: 1041, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1041_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.478s.\n",
      "-- designing region: 15:53235001-53265000_strand_+_gene_1042 -- region: 1042, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1042_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 15:54735001-54765000_strand_+_gene_1043 -- region: 1043, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1043_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.502s.\n",
      "-- designing region: 15:56235001-56265000_strand_+_gene_1044 -- region: 1044, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1044_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 19.604s.\n",
      "-- designing region: 15:57735001-57765000_strand_+_gene_1045 -- region: 1045, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1045_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.598s.\n",
      "- Designed 59918 candidate probes in 21.340s.\n",
      "-- designing region: 15:59235001-59265000_strand_+_gene_1046 -- region: 1046, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1046_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.532s.\n",
      "-- designing region: 15:60735001-60765000_strand_+_gene_1047 -- region: 1047, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1047_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.309s.\n",
      "- Designed 59918 candidate probes in 15.689s.\n",
      "-- designing region: 15:62235001-62265000_strand_+_gene_1048 -- region: 1048, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1048_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 15:63735001-63765000_strand_+_gene_1049 -- region: 1049, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1049_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.624s.\n",
      "-- designing region: 15:65235001-65265000_strand_+_gene_1050 -- region: 1050, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1050_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 15:66735001-66765000_strand_+_gene_1051 -- region: 1051, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1051_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.672s.\n",
      "-- designing region: 15:68235001-68265000_strand_+_gene_1052 -- region: 1052, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1052_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.577s.\n",
      "-- designing region: 15:69735001-69765000_strand_+_gene_1053 -- region: 1053, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1053_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.693s.\n",
      "-- designing region: 15:71235001-71265000_strand_+_gene_1054 -- region: 1054, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1054_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 15:72735001-72765000_strand_+_gene_1055 -- region: 1055, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1055_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.613s.\n",
      "-- designing region: 15:74235001-74265000_strand_+_gene_1056 -- region: 1056, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1056_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: 15:75735001-75765000_strand_+_gene_1057 -- region: 1057, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1057_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.688s.\n",
      "-- designing region: 15:77235001-77265000_strand_+_gene_1058 -- region: 1058, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1058_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 15:78735001-78765000_strand_+_gene_1059 -- region: 1059, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1059_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 15:80235001-80265000_strand_+_gene_1060 -- region: 1060, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1060_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.620s.\n",
      "-- designing region: 15:81735001-81765000_strand_+_gene_1061 -- region: 1061, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1061_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.676s.\n",
      "-- designing region: 15:83235001-83265000_strand_+_gene_1062 -- region: 1062, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1062_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 15:84735001-84765000_strand_+_gene_1063 -- region: 1063, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1063_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 15:86235001-86265000_strand_+_gene_1064 -- region: 1064, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1064_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 15:87735001-87765000_strand_+_gene_1065 -- region: 1065, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1065_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 15:89235001-89265000_strand_+_gene_1066 -- region: 1066, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1066_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 15:90735001-90765000_strand_+_gene_1067 -- region: 1067, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1067_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.496s.\n",
      "-- designing region: 15:92235001-92265000_strand_+_gene_1068 -- region: 1068, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1068_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.561s.\n",
      "-- designing region: 15:93735001-93765000_strand_+_gene_1069 -- region: 1069, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1069_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 15:95235001-95265000_strand_+_gene_1070 -- region: 1070, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1070_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 15:96735001-96765000_strand_+_gene_1071 -- region: 1071, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1071_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.555s.\n",
      "-- designing region: 15:98235001-98265000_strand_+_gene_1072 -- region: 1072, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1072_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.574s.\n",
      "-- designing region: 15:99735001-99765000_strand_+_gene_1073 -- region: 1073, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1073_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 16:735001-765000_strand_+_gene_1074 -- region: 1074, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1074_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 16:2235001-2265000_strand_+_gene_1075 -- region: 1075, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1075_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 16:3735001-3765000_strand_+_gene_1076 -- region: 1076, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1076_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.536s.\n",
      "-- designing region: 16:5235001-5265000_strand_+_gene_1077 -- region: 1077, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1077_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 16:6735001-6765000_strand_+_gene_1078 -- region: 1078, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1078_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 16:8235001-8265000_strand_+_gene_1079 -- region: 1079, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1079_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.405s.\n",
      "-- designing region: 16:9735001-9765000_strand_+_gene_1080 -- region: 1080, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1080_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.526s.\n",
      "-- designing region: 16:11235001-11265000_strand_+_gene_1081 -- region: 1081, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1081_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.346s.\n",
      "-- designing region: 16:12735001-12765000_strand_+_gene_1082 -- region: 1082, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1082_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.394s.\n",
      "-- designing region: 16:14235001-14265000_strand_+_gene_1083 -- region: 1083, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1083_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.407s.\n",
      "-- designing region: 16:15735001-15765000_strand_+_gene_1084 -- region: 1084, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1084_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.433s.\n",
      "-- designing region: 16:17235001-17265000_strand_+_gene_1085 -- region: 1085, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1085_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.405s.\n",
      "-- designing region: 16:18735001-18765000_strand_+_gene_1086 -- region: 1086, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1086_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.418s.\n",
      "-- designing region: 16:20235001-20265000_strand_+_gene_1087 -- region: 1087, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1087_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.408s.\n",
      "-- designing region: 16:21735001-21765000_strand_+_gene_1088 -- region: 1088, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1088_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.423s.\n",
      "-- designing region: 16:23235001-23265000_strand_+_gene_1089 -- region: 1089, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1089_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.370s.\n",
      "-- designing region: 16:24735001-24765000_strand_+_gene_1090 -- region: 1090, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1090_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.429s.\n",
      "-- designing region: 16:26235001-26265000_strand_+_gene_1091 -- region: 1091, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1091_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.424s.\n",
      "-- designing region: 16:27735001-27765000_strand_+_gene_1092 -- region: 1092, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1092_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.462s.\n",
      "-- designing region: 16:29235001-29265000_strand_+_gene_1093 -- region: 1093, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1093_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 16:30735001-30765000_strand_+_gene_1094 -- region: 1094, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1094_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.470s.\n",
      "-- designing region: 16:32235001-32265000_strand_+_gene_1095 -- region: 1095, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1095_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.455s.\n",
      "-- designing region: 16:33735001-33765000_strand_+_gene_1096 -- region: 1096, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1096_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.374s.\n",
      "-- designing region: 16:35235001-35265000_strand_+_gene_1097 -- region: 1097, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1097_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.419s.\n",
      "-- designing region: 16:36735001-36765000_strand_+_gene_1098 -- region: 1098, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1098_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.218s.\n",
      "- Designed 59918 candidate probes in 15.283s.\n",
      "-- designing region: 16:38235001-38265000_strand_+_gene_1099 -- region: 1099, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1099_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.212s.\n",
      "- Designed 59918 candidate probes in 15.224s.\n",
      "-- designing region: 16:39735001-39765000_strand_+_gene_1100 -- region: 1100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1100_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.202s.\n",
      "- Designed 0 candidate probes in 0.496s.\n",
      "-- designing region: 16:41235001-41265000_strand_+_gene_1101 -- region: 1101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1101_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.556s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: 16:42735001-42765000_strand_+_gene_1102 -- region: 1102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1102_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.844s.\n",
      "-- designing region: 16:44235001-44265000_strand_+_gene_1103 -- region: 1103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1103_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.844s.\n",
      "-- designing region: 16:45735001-45765000_strand_+_gene_1104 -- region: 1104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1104_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.564s.\n",
      "- Designed 0 candidate probes in 0.859s.\n",
      "-- designing region: 16:47235001-47265000_strand_+_gene_1105 -- region: 1105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1105_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.620s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 16:48735001-48765000_strand_+_gene_1106 -- region: 1106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1106_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.455s.\n",
      "-- designing region: 16:50235001-50265000_strand_+_gene_1107 -- region: 1107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1107_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.477s.\n",
      "-- designing region: 16:51735001-51765000_strand_+_gene_1108 -- region: 1108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1108_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.465s.\n",
      "-- designing region: 16:53235001-53265000_strand_+_gene_1109 -- region: 1109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1109_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.539s.\n",
      "-- designing region: 16:54735001-54765000_strand_+_gene_1110 -- region: 1110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1110_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.664s.\n",
      "-- designing region: 16:56235001-56265000_strand_+_gene_1111 -- region: 1111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1111_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.378s.\n",
      "- Designed 59918 candidate probes in 28.245s.\n",
      "-- designing region: 16:57735001-57765000_strand_+_gene_1112 -- region: 1112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1112_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 16:59235001-59265000_strand_+_gene_1113 -- region: 1113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1113_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.654s.\n",
      "-- designing region: 16:60735001-60765000_strand_+_gene_1114 -- region: 1114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1114_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 16:62235001-62265000_strand_+_gene_1115 -- region: 1115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1115_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 16:63735001-63765000_strand_+_gene_1116 -- region: 1116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1116_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.607s.\n",
      "-- designing region: 16:65235001-65265000_strand_+_gene_1117 -- region: 1117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1117_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 16:66735001-66765000_strand_+_gene_1118 -- region: 1118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1118_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 16:68235001-68265000_strand_+_gene_1119 -- region: 1119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1119_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.659s.\n",
      "-- designing region: 16:69735001-69765000_strand_+_gene_1120 -- region: 1120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1120_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 16:71235001-71265000_strand_+_gene_1121 -- region: 1121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1121_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 16:72735001-72765000_strand_+_gene_1122 -- region: 1122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1122_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 16:74235001-74265000_strand_+_gene_1123 -- region: 1123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1123_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 16:75735001-75765000_strand_+_gene_1124 -- region: 1124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1124_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.718s.\n",
      "-- designing region: 16:77235001-77265000_strand_+_gene_1125 -- region: 1125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1125_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 16:78735001-78765000_strand_+_gene_1126 -- region: 1126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1126_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 16:80235001-80265000_strand_+_gene_1127 -- region: 1127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1127_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 16:81735001-81765000_strand_+_gene_1128 -- region: 1128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1128_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 16:83235001-83265000_strand_+_gene_1129 -- region: 1129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1129_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 16:84735001-84765000_strand_+_gene_1130 -- region: 1130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1130_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.624s.\n",
      "-- designing region: 16:86235001-86265000_strand_+_gene_1131 -- region: 1131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1131_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.620s.\n",
      "-- designing region: 16:87735001-87765000_strand_+_gene_1132 -- region: 1132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1132_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 16:89235001-89265000_strand_+_gene_1133 -- region: 1133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1133_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 17:735001-765000_strand_+_gene_1134 -- region: 1134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1134_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 17:2235001-2265000_strand_+_gene_1135 -- region: 1135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1135_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 17:3735001-3765000_strand_+_gene_1136 -- region: 1136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1136_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 17:5235001-5265000_strand_+_gene_1137 -- region: 1137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1137_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 17:6735001-6765000_strand_+_gene_1138 -- region: 1138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1138_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 17:8235001-8265000_strand_+_gene_1139 -- region: 1139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1139_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 17:9735001-9765000_strand_+_gene_1140 -- region: 1140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1140_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 17:11235001-11265000_strand_+_gene_1141 -- region: 1141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1141_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 17:12735001-12765000_strand_+_gene_1142 -- region: 1142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1142_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 17:14235001-14265000_strand_+_gene_1143 -- region: 1143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1143_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.555s.\n",
      "-- designing region: 17:15735001-15765000_strand_+_gene_1144 -- region: 1144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1144_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 17:17235001-17265000_strand_+_gene_1145 -- region: 1145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1145_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 17:18735001-18765000_strand_+_gene_1146 -- region: 1146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1146_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.401s.\n",
      "-- designing region: 17:20235001-20265000_strand_+_gene_1147 -- region: 1147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1147_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: 17:21735001-21765000_strand_+_gene_1148 -- region: 1148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1148_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 17:23235001-23265000_strand_+_gene_1149 -- region: 1149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1149_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.215s.\n",
      "- Designed 59918 candidate probes in 15.257s.\n",
      "-- designing region: 17:24735001-24765000_strand_+_gene_1150 -- region: 1150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1150_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.220s.\n",
      "- Designed 59918 candidate probes in 15.215s.\n",
      "-- designing region: 17:26235001-26265000_strand_+_gene_1151 -- region: 1151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1151_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.223s.\n",
      "- Designed 59918 candidate probes in 15.257s.\n",
      "-- designing region: 17:27735001-27765000_strand_+_gene_1152 -- region: 1152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1152_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.364s.\n",
      "-- designing region: 17:29235001-29265000_strand_+_gene_1153 -- region: 1153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1153_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.458s.\n",
      "-- designing region: 17:30735001-30765000_strand_+_gene_1154 -- region: 1154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1154_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.391s.\n",
      "-- designing region: 17:32235001-32265000_strand_+_gene_1155 -- region: 1155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1155_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "- Designed 59918 candidate probes in 15.392s.\n",
      "-- designing region: 17:33735001-33765000_strand_+_gene_1156 -- region: 1156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1156_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.433s.\n",
      "-- designing region: 17:35235001-35265000_strand_+_gene_1157 -- region: 1157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1157_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.418s.\n",
      "-- designing region: 17:36735001-36765000_strand_+_gene_1158 -- region: 1158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1158_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.480s.\n",
      "-- designing region: 17:38235001-38265000_strand_+_gene_1159 -- region: 1159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1159_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "- Designed 59918 candidate probes in 15.354s.\n",
      "-- designing region: 17:39735001-39765000_strand_+_gene_1160 -- region: 1160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1160_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.455s.\n",
      "-- designing region: 17:41235001-41265000_strand_+_gene_1161 -- region: 1161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1161_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "- Designed 59918 candidate probes in 15.425s.\n",
      "-- designing region: 17:42735001-42765000_strand_+_gene_1162 -- region: 1162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1162_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.470s.\n",
      "-- designing region: 17:44235001-44265000_strand_+_gene_1163 -- region: 1163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1163_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.262s.\n",
      "- Designed 59918 candidate probes in 15.439s.\n",
      "-- designing region: 17:45735001-45765000_strand_+_gene_1164 -- region: 1164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1164_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.475s.\n",
      "-- designing region: 17:47235001-47265000_strand_+_gene_1165 -- region: 1165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1165_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: 17:48735001-48765000_strand_+_gene_1166 -- region: 1166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1166_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.483s.\n",
      "-- designing region: 17:50235001-50265000_strand_+_gene_1167 -- region: 1167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1167_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.457s.\n",
      "-- designing region: 17:51735001-51765000_strand_+_gene_1168 -- region: 1168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1168_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.470s.\n",
      "-- designing region: 17:53235001-53265000_strand_+_gene_1169 -- region: 1169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1169_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.474s.\n",
      "-- designing region: 17:54735001-54765000_strand_+_gene_1170 -- region: 1170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1170_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 17:56235001-56265000_strand_+_gene_1171 -- region: 1171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1171_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.467s.\n",
      "-- designing region: 17:57735001-57765000_strand_+_gene_1172 -- region: 1172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1172_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 19.991s.\n",
      "-- designing region: 17:59235001-59265000_strand_+_gene_1173 -- region: 1173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1173_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.583s.\n",
      "- Designed 59918 candidate probes in 21.880s.\n",
      "-- designing region: 17:60735001-60765000_strand_+_gene_1174 -- region: 1174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1174_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.522s.\n",
      "-- designing region: 17:62235001-62265000_strand_+_gene_1175 -- region: 1175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1175_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.643s.\n",
      "-- designing region: 17:63735001-63765000_strand_+_gene_1176 -- region: 1176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1176_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.576s.\n",
      "-- designing region: 17:65235001-65265000_strand_+_gene_1177 -- region: 1177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1177_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.650s.\n",
      "-- designing region: 17:66735001-66765000_strand_+_gene_1178 -- region: 1178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1178_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.697s.\n",
      "-- designing region: 17:68235001-68265000_strand_+_gene_1179 -- region: 1179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1179_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.715s.\n",
      "-- designing region: 17:69735001-69765000_strand_+_gene_1180 -- region: 1180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1180_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.671s.\n",
      "-- designing region: 17:71235001-71265000_strand_+_gene_1181 -- region: 1181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1181_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 17:72735001-72765000_strand_+_gene_1182 -- region: 1182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1182_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.702s.\n",
      "-- designing region: 17:74235001-74265000_strand_+_gene_1183 -- region: 1183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1183_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 17:75735001-75765000_strand_+_gene_1184 -- region: 1184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1184_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.676s.\n",
      "-- designing region: 17:77235001-77265000_strand_+_gene_1185 -- region: 1185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1185_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.634s.\n",
      "-- designing region: 17:78735001-78765000_strand_+_gene_1186 -- region: 1186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1186_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.641s.\n",
      "-- designing region: 17:80235001-80265000_strand_+_gene_1187 -- region: 1187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1187_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.650s.\n",
      "-- designing region: 17:81735001-81765000_strand_+_gene_1188 -- region: 1188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1188_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.224s.\n",
      "- Designed 15002 candidate probes in 4.639s.\n",
      "-- designing region: 18:735001-765000_strand_+_gene_1189 -- region: 1189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1189_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 18:2235001-2265000_strand_+_gene_1190 -- region: 1190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1190_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 18:3735001-3765000_strand_+_gene_1191 -- region: 1191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1191_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 18:5235001-5265000_strand_+_gene_1192 -- region: 1192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1192_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 18:6735001-6765000_strand_+_gene_1193 -- region: 1193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1193_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 18:8235001-8265000_strand_+_gene_1194 -- region: 1194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1194_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 18:9735001-9765000_strand_+_gene_1195 -- region: 1195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1195_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 77.578s.\n",
      "-- designing region: 18:11235001-11265000_strand_+_gene_1196 -- region: 1196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1196_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.727s.\n",
      "-- designing region: 18:12735001-12765000_strand_+_gene_1197 -- region: 1197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1197_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 18:14235001-14265000_strand_+_gene_1198 -- region: 1198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1198_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 18:15735001-15765000_strand_+_gene_1199 -- region: 1199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1199_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.221s.\n",
      "- Designed 59918 candidate probes in 15.394s.\n",
      "-- designing region: 18:17235001-17265000_strand_+_gene_1200 -- region: 1200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1200_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "- Designed 59918 candidate probes in 15.532s.\n",
      "-- designing region: 18:18735001-18765000_strand_+_gene_1201 -- region: 1201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1201_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.224s.\n",
      "- Designed 59918 candidate probes in 15.335s.\n",
      "-- designing region: 18:20235001-20265000_strand_+_gene_1202 -- region: 1202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1202_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.220s.\n",
      "- Designed 59918 candidate probes in 15.303s.\n",
      "-- designing region: 18:21735001-21765000_strand_+_gene_1203 -- region: 1203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1203_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 18:23235001-23265000_strand_+_gene_1204 -- region: 1204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1204_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.664s.\n",
      "-- designing region: 18:24735001-24765000_strand_+_gene_1205 -- region: 1205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1205_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 18:26235001-26265000_strand_+_gene_1206 -- region: 1206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1206_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.660s.\n",
      "-- designing region: 18:27735001-27765000_strand_+_gene_1207 -- region: 1207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1207_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 18:29235001-29265000_strand_+_gene_1208 -- region: 1208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1208_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 18:30735001-30765000_strand_+_gene_1209 -- region: 1209, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1209_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.558s.\n",
      "-- designing region: 18:32235001-32265000_strand_+_gene_1210 -- region: 1210, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1210_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 18:33735001-33765000_strand_+_gene_1211 -- region: 1211, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1211_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.458s.\n",
      "-- designing region: 18:35235001-35265000_strand_+_gene_1212 -- region: 1212, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1212_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.482s.\n",
      "-- designing region: 18:36735001-36765000_strand_+_gene_1213 -- region: 1213, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1213_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.490s.\n",
      "-- designing region: 18:38235001-38265000_strand_+_gene_1214 -- region: 1214, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1214_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.478s.\n",
      "-- designing region: 18:39735001-39765000_strand_+_gene_1215 -- region: 1215, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1215_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.317s.\n",
      "- Designed 59918 candidate probes in 15.507s.\n",
      "-- designing region: 18:41235001-41265000_strand_+_gene_1216 -- region: 1216, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1216_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.402s.\n",
      "-- designing region: 18:42735001-42765000_strand_+_gene_1217 -- region: 1217, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1217_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.437s.\n",
      "-- designing region: 18:44235001-44265000_strand_+_gene_1218 -- region: 1218, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1218_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.405s.\n",
      "-- designing region: 18:45735001-45765000_strand_+_gene_1219 -- region: 1219, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1219_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.459s.\n",
      "-- designing region: 18:47235001-47265000_strand_+_gene_1220 -- region: 1220, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1220_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.451s.\n",
      "-- designing region: 18:48735001-48765000_strand_+_gene_1221 -- region: 1221, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1221_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: 18:50235001-50265000_strand_+_gene_1222 -- region: 1222, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1222_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.408s.\n",
      "-- designing region: 18:51735001-51765000_strand_+_gene_1223 -- region: 1223, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1223_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.466s.\n",
      "-- designing region: 18:53235001-53265000_strand_+_gene_1224 -- region: 1224, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1224_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.429s.\n",
      "-- designing region: 18:54735001-54765000_strand_+_gene_1225 -- region: 1225, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1225_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.400s.\n",
      "-- designing region: 18:56235001-56265000_strand_+_gene_1226 -- region: 1226, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1226_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.454s.\n",
      "-- designing region: 18:57735001-57765000_strand_+_gene_1227 -- region: 1227, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1227_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.431s.\n",
      "-- designing region: 18:59235001-59265000_strand_+_gene_1228 -- region: 1228, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1228_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.487s.\n",
      "-- designing region: 18:60735001-60765000_strand_+_gene_1229 -- region: 1229, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1229_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 18:62235001-62265000_strand_+_gene_1230 -- region: 1230, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1230_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "- Designed 59918 candidate probes in 15.430s.\n",
      "-- designing region: 18:63735001-63765000_strand_+_gene_1231 -- region: 1231, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1231_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.262s.\n",
      "- Designed 59918 candidate probes in 28.285s.\n",
      "-- designing region: 18:65235001-65265000_strand_+_gene_1232 -- region: 1232, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1232_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: 18:66735001-66765000_strand_+_gene_1233 -- region: 1233, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1233_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.622s.\n",
      "-- designing region: 18:68235001-68265000_strand_+_gene_1234 -- region: 1234, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1234_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 18:69735001-69765000_strand_+_gene_1235 -- region: 1235, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1235_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 18:71235001-71265000_strand_+_gene_1236 -- region: 1236, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1236_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 18:72735001-72765000_strand_+_gene_1237 -- region: 1237, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1237_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 18:74235001-74265000_strand_+_gene_1238 -- region: 1238, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1238_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.697s.\n",
      "-- designing region: 18:75735001-75765000_strand_+_gene_1239 -- region: 1239, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1239_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 18:77235001-77265000_strand_+_gene_1240 -- region: 1240, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1240_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 18:78735001-78765000_strand_+_gene_1241 -- region: 1241, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1241_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.306s.\n",
      "- Designed 59918 candidate probes in 15.667s.\n",
      "-- designing region: 19:485001-515000_strand_+_gene_1242 -- region: 1242, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1242_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 19:1485001-1515000_strand_+_gene_1243 -- region: 1243, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1243_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.613s.\n",
      "-- designing region: 19:2485001-2515000_strand_+_gene_1244 -- region: 1244, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1244_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 19:3485001-3515000_strand_+_gene_1245 -- region: 1245, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1245_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: 19:4485001-4515000_strand_+_gene_1246 -- region: 1246, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1246_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 19:5485001-5515000_strand_+_gene_1247 -- region: 1247, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1247_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 19:6485001-6515000_strand_+_gene_1248 -- region: 1248, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1248_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 19:7485001-7515000_strand_+_gene_1249 -- region: 1249, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1249_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 19:8485001-8515000_strand_+_gene_1250 -- region: 1250, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1250_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.640s.\n",
      "-- designing region: 19:9485001-9515000_strand_+_gene_1251 -- region: 1251, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1251_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 19:10485001-10515000_strand_+_gene_1252 -- region: 1252, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1252_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 19:11485001-11515000_strand_+_gene_1253 -- region: 1253, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1253_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 19:12485001-12515000_strand_+_gene_1254 -- region: 1254, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1254_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 19:13485001-13515000_strand_+_gene_1255 -- region: 1255, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1255_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 19:14485001-14515000_strand_+_gene_1256 -- region: 1256, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1256_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 19:15485001-15515000_strand_+_gene_1257 -- region: 1257, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1257_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 19:16485001-16515000_strand_+_gene_1258 -- region: 1258, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1258_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 19:17485001-17515000_strand_+_gene_1259 -- region: 1259, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1259_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 19:18485001-18515000_strand_+_gene_1260 -- region: 1260, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1260_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.656s.\n",
      "-- designing region: 19:19485001-19515000_strand_+_gene_1261 -- region: 1261, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1261_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.444s.\n",
      "-- designing region: 19:20485001-20515000_strand_+_gene_1262 -- region: 1262, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1262_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 19:21485001-21515000_strand_+_gene_1263 -- region: 1263, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1263_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.561s.\n",
      "-- designing region: 19:22485001-22515000_strand_+_gene_1264 -- region: 1264, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1264_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 19:23485001-23515000_strand_+_gene_1265 -- region: 1265, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1265_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.504s.\n",
      "-- designing region: 19:24485001-24515000_strand_+_gene_1266 -- region: 1266, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1266_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.224s.\n",
      "- Designed 31958 candidate probes in 8.605s.\n",
      "-- designing region: 19:25485001-25515000_strand_+_gene_1267 -- region: 1267, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1267_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.221s.\n",
      "- Designed 59918 candidate probes in 15.242s.\n",
      "-- designing region: 19:26485001-26515000_strand_+_gene_1268 -- region: 1268, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1268_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.217s.\n",
      "- Designed 59918 candidate probes in 15.274s.\n",
      "-- designing region: 19:27485001-27515000_strand_+_gene_1269 -- region: 1269, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1269_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "- Designed 59918 candidate probes in 15.277s.\n",
      "-- designing region: 19:28485001-28515000_strand_+_gene_1270 -- region: 1270, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1270_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 19:29485001-29515000_strand_+_gene_1271 -- region: 1271, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1271_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.470s.\n",
      "-- designing region: 19:30485001-30515000_strand_+_gene_1272 -- region: 1272, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1272_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.429s.\n",
      "-- designing region: 19:31485001-31515000_strand_+_gene_1273 -- region: 1273, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1273_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "- Designed 59918 candidate probes in 15.440s.\n",
      "-- designing region: 19:32485001-32515000_strand_+_gene_1274 -- region: 1274, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1274_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.403s.\n",
      "-- designing region: 19:33485001-33515000_strand_+_gene_1275 -- region: 1275, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1275_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.474s.\n",
      "-- designing region: 19:34485001-34515000_strand_+_gene_1276 -- region: 1276, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1276_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.441s.\n",
      "-- designing region: 19:35485001-35515000_strand_+_gene_1277 -- region: 1277, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1277_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.466s.\n",
      "-- designing region: 19:36485001-36515000_strand_+_gene_1278 -- region: 1278, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1278_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.412s.\n",
      "-- designing region: 19:37485001-37515000_strand_+_gene_1279 -- region: 1279, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1279_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.414s.\n",
      "-- designing region: 19:38485001-38515000_strand_+_gene_1280 -- region: 1280, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1280_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.456s.\n",
      "-- designing region: 19:39485001-39515000_strand_+_gene_1281 -- region: 1281, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1281_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.431s.\n",
      "-- designing region: 19:40485001-40515000_strand_+_gene_1282 -- region: 1282, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1282_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.433s.\n",
      "-- designing region: 19:41485001-41515000_strand_+_gene_1283 -- region: 1283, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1283_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.449s.\n",
      "-- designing region: 19:42485001-42515000_strand_+_gene_1284 -- region: 1284, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1284_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.422s.\n",
      "-- designing region: 19:43485001-43515000_strand_+_gene_1285 -- region: 1285, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1285_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.414s.\n",
      "-- designing region: 19:44485001-44515000_strand_+_gene_1286 -- region: 1286, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1286_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.413s.\n",
      "-- designing region: 19:45485001-45515000_strand_+_gene_1287 -- region: 1287, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1287_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.464s.\n",
      "-- designing region: 19:46485001-46515000_strand_+_gene_1288 -- region: 1288, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1288_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.450s.\n",
      "-- designing region: 19:47485001-47515000_strand_+_gene_1289 -- region: 1289, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1289_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.539s.\n",
      "-- designing region: 19:48485001-48515000_strand_+_gene_1290 -- region: 1290, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1290_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.534s.\n",
      "-- designing region: 19:49485001-49515000_strand_+_gene_1291 -- region: 1291, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1291_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 19:50485001-50515000_strand_+_gene_1292 -- region: 1292, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1292_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.465s.\n",
      "-- designing region: 19:51485001-51515000_strand_+_gene_1293 -- region: 1293, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1293_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.511s.\n",
      "- Designed 59918 candidate probes in 26.966s.\n",
      "-- designing region: 19:52485001-52515000_strand_+_gene_1294 -- region: 1294, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1294_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.663s.\n",
      "- Designed 59918 candidate probes in 17.707s.\n",
      "-- designing region: 19:53485001-53515000_strand_+_gene_1295 -- region: 1295, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1295_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 19:54485001-54515000_strand_+_gene_1296 -- region: 1296, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1296_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.648s.\n",
      "-- designing region: 19:55485001-55515000_strand_+_gene_1297 -- region: 1297, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1297_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 19:56485001-56515000_strand_+_gene_1298 -- region: 1298, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1298_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.658s.\n",
      "-- designing region: 19:57485001-57515000_strand_+_gene_1299 -- region: 1299, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1299_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 20:485001-515000_strand_+_gene_1300 -- region: 1300, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1300_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 20:1485001-1515000_strand_+_gene_1301 -- region: 1301, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1301_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 20:2485001-2515000_strand_+_gene_1302 -- region: 1302, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1302_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.690s.\n",
      "-- designing region: 20:3485001-3515000_strand_+_gene_1303 -- region: 1303, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1303_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.660s.\n",
      "-- designing region: 20:4485001-4515000_strand_+_gene_1304 -- region: 1304, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1304_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.665s.\n",
      "-- designing region: 20:5485001-5515000_strand_+_gene_1305 -- region: 1305, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1305_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.704s.\n",
      "-- designing region: 20:6485001-6515000_strand_+_gene_1306 -- region: 1306, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1306_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.658s.\n",
      "-- designing region: 20:7485001-7515000_strand_+_gene_1307 -- region: 1307, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1307_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.549s.\n",
      "-- designing region: 20:8485001-8515000_strand_+_gene_1308 -- region: 1308, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1308_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.616s.\n",
      "-- designing region: 20:9485001-9515000_strand_+_gene_1309 -- region: 1309, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1309_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.676s.\n",
      "-- designing region: 20:10485001-10515000_strand_+_gene_1310 -- region: 1310, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1310_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.687s.\n",
      "-- designing region: 20:11485001-11515000_strand_+_gene_1311 -- region: 1311, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1311_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 20:12485001-12515000_strand_+_gene_1312 -- region: 1312, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1312_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 20:13485001-13515000_strand_+_gene_1313 -- region: 1313, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1313_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.672s.\n",
      "-- designing region: 20:14485001-14515000_strand_+_gene_1314 -- region: 1314, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1314_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 20:15485001-15515000_strand_+_gene_1315 -- region: 1315, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1315_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.659s.\n",
      "-- designing region: 20:16485001-16515000_strand_+_gene_1316 -- region: 1316, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1316_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 20:17485001-17515000_strand_+_gene_1317 -- region: 1317, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1317_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.754s.\n",
      "-- designing region: 20:18485001-18515000_strand_+_gene_1318 -- region: 1318, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1318_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 20:19485001-19515000_strand_+_gene_1319 -- region: 1319, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1319_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.645s.\n",
      "-- designing region: 20:20485001-20515000_strand_+_gene_1320 -- region: 1320, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1320_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.697s.\n",
      "-- designing region: 20:21485001-21515000_strand_+_gene_1321 -- region: 1321, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1321_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 20:22485001-22515000_strand_+_gene_1322 -- region: 1322, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1322_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.664s.\n",
      "-- designing region: 20:23485001-23515000_strand_+_gene_1323 -- region: 1323, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1323_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.468s.\n",
      "-- designing region: 20:24485001-24515000_strand_+_gene_1324 -- region: 1324, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1324_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.627s.\n",
      "-- designing region: 20:25485001-25515000_strand_+_gene_1325 -- region: 1325, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1325_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.627s.\n",
      "-- designing region: 20:26485001-26515000_strand_+_gene_1326 -- region: 1326, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1326_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.218s.\n",
      "- Designed 59918 candidate probes in 15.355s.\n",
      "-- designing region: 20:27485001-27515000_strand_+_gene_1327 -- region: 1327, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1327_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.215s.\n",
      "- Designed 59636 candidate probes in 15.368s.\n",
      "-- designing region: 20:28485001-28515000_strand_+_gene_1328 -- region: 1328, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1328_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "- Designed 48460 candidate probes in 12.533s.\n",
      "-- designing region: 20:29485001-29515000_strand_+_gene_1329 -- region: 1329, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1329_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.475s.\n",
      "-- designing region: 20:30485001-30515000_strand_+_gene_1330 -- region: 1330, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1330_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.509s.\n",
      "-- designing region: 20:31485001-31515000_strand_+_gene_1331 -- region: 1331, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1331_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.261s.\n",
      "- Designed 59918 candidate probes in 15.451s.\n",
      "-- designing region: 20:32485001-32515000_strand_+_gene_1332 -- region: 1332, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1332_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 20:33485001-33515000_strand_+_gene_1333 -- region: 1333, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1333_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.464s.\n",
      "-- designing region: 20:34485001-34515000_strand_+_gene_1334 -- region: 1334, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1334_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.429s.\n",
      "-- designing region: 20:35485001-35515000_strand_+_gene_1335 -- region: 1335, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1335_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "- Designed 59918 candidate probes in 15.481s.\n",
      "-- designing region: 20:36485001-36515000_strand_+_gene_1336 -- region: 1336, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1336_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 20:37485001-37515000_strand_+_gene_1337 -- region: 1337, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1337_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.502s.\n",
      "-- designing region: 20:38485001-38515000_strand_+_gene_1338 -- region: 1338, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1338_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: 20:39485001-39515000_strand_+_gene_1339 -- region: 1339, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1339_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.492s.\n",
      "-- designing region: 20:40485001-40515000_strand_+_gene_1340 -- region: 1340, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1340_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 20:41485001-41515000_strand_+_gene_1341 -- region: 1341, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1341_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.504s.\n",
      "-- designing region: 20:42485001-42515000_strand_+_gene_1342 -- region: 1342, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1342_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 20:43485001-43515000_strand_+_gene_1343 -- region: 1343, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1343_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.448s.\n",
      "-- designing region: 20:44485001-44515000_strand_+_gene_1344 -- region: 1344, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1344_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: 20:45485001-45515000_strand_+_gene_1345 -- region: 1345, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1345_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: 20:46485001-46515000_strand_+_gene_1346 -- region: 1346, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1346_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.454s.\n",
      "-- designing region: 20:47485001-47515000_strand_+_gene_1347 -- region: 1347, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1347_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.436s.\n",
      "-- designing region: 20:48485001-48515000_strand_+_gene_1348 -- region: 1348, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1348_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "- Designed 59918 candidate probes in 15.430s.\n",
      "-- designing region: 20:49485001-49515000_strand_+_gene_1349 -- region: 1349, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1349_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.476s.\n",
      "-- designing region: 20:50485001-50515000_strand_+_gene_1350 -- region: 1350, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1350_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.495s.\n",
      "-- designing region: 20:51485001-51515000_strand_+_gene_1351 -- region: 1351, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1351_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.551s.\n",
      "-- designing region: 20:52485001-52515000_strand_+_gene_1352 -- region: 1352, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1352_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 20:53485001-53515000_strand_+_gene_1353 -- region: 1353, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1353_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: 20:54485001-54515000_strand_+_gene_1354 -- region: 1354, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1354_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 18.235s.\n",
      "-- designing region: 20:55485001-55515000_strand_+_gene_1355 -- region: 1355, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1355_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.422s.\n",
      "- Designed 59918 candidate probes in 22.270s.\n",
      "-- designing region: 20:56485001-56515000_strand_+_gene_1356 -- region: 1356, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1356_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 20:57485001-57515000_strand_+_gene_1357 -- region: 1357, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1357_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.721s.\n",
      "-- designing region: 20:58485001-58515000_strand_+_gene_1358 -- region: 1358, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1358_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.737s.\n",
      "-- designing region: 20:59485001-59515000_strand_+_gene_1359 -- region: 1359, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1359_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.686s.\n",
      "-- designing region: 20:60485001-60515000_strand_+_gene_1360 -- region: 1360, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1360_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 20:61485001-61515000_strand_+_gene_1361 -- region: 1361, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1361_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.727s.\n",
      "-- designing region: 20:62485001-62515000_strand_+_gene_1362 -- region: 1362, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1362_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.725s.\n",
      "-- designing region: 20:63485001-63515000_strand_+_gene_1363 -- region: 1363, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1363_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.717s.\n",
      "-- designing region: 21:485001-515000_strand_+_gene_1364 -- region: 1364, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1364_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "- Designed 0 candidate probes in 0.503s.\n",
      "-- designing region: 21:1485001-1515000_strand_+_gene_1365 -- region: 1365, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1365_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.576s.\n",
      "- Designed 0 candidate probes in 0.872s.\n",
      "-- designing region: 21:2485001-2515000_strand_+_gene_1366 -- region: 1366, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1366_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.588s.\n",
      "- Designed 0 candidate probes in 0.883s.\n",
      "-- designing region: 21:3485001-3515000_strand_+_gene_1367 -- region: 1367, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1367_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.581s.\n",
      "- Designed 0 candidate probes in 0.876s.\n",
      "-- designing region: 21:4485001-4515000_strand_+_gene_1368 -- region: 1368, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1368_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.577s.\n",
      "- Designed 0 candidate probes in 0.877s.\n",
      "-- designing region: 21:5485001-5515000_strand_+_gene_1369 -- region: 1369, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1369_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.617s.\n",
      "- Designed 31894 candidate probes in 9.145s.\n",
      "-- designing region: 21:6485001-6515000_strand_+_gene_1370 -- region: 1370, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1370_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.679s.\n",
      "-- designing region: 21:7485001-7515000_strand_+_gene_1371 -- region: 1371, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1371_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.249s.\n",
      "- Designed 31698 candidate probes in 8.763s.\n",
      "-- designing region: 21:8485001-8515000_strand_+_gene_1372 -- region: 1372, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1372_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.213s.\n",
      "- Designed 0 candidate probes in 0.509s.\n",
      "-- designing region: 21:9485001-9515000_strand_+_gene_1373 -- region: 1373, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1373_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.578s.\n",
      "- Designed 0 candidate probes in 0.876s.\n",
      "-- designing region: 21:10485001-10515000_strand_+_gene_1374 -- region: 1374, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1374_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.659s.\n",
      "- Designed 59918 candidate probes in 16.001s.\n",
      "-- designing region: 21:11485001-11515000_strand_+_gene_1375 -- region: 1375, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1375_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.225s.\n",
      "- Designed 59918 candidate probes in 15.499s.\n",
      "-- designing region: 21:12485001-12515000_strand_+_gene_1376 -- region: 1376, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1376_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.218s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 21:13485001-13515000_strand_+_gene_1377 -- region: 1377, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1377_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.513s.\n",
      "-- designing region: 21:14485001-14515000_strand_+_gene_1378 -- region: 1378, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1378_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 21:15485001-15515000_strand_+_gene_1379 -- region: 1379, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1379_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 21:16485001-16515000_strand_+_gene_1380 -- region: 1380, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1380_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.620s.\n",
      "-- designing region: 21:17485001-17515000_strand_+_gene_1381 -- region: 1381, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1381_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.679s.\n",
      "-- designing region: 21:18485001-18515000_strand_+_gene_1382 -- region: 1382, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1382_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.654s.\n",
      "-- designing region: 21:19485001-19515000_strand_+_gene_1383 -- region: 1383, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1383_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.692s.\n",
      "-- designing region: 21:20485001-20515000_strand_+_gene_1384 -- region: 1384, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1384_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 21:21485001-21515000_strand_+_gene_1385 -- region: 1385, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1385_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 21:22485001-22515000_strand_+_gene_1386 -- region: 1386, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1386_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.607s.\n",
      "-- designing region: 21:23485001-23515000_strand_+_gene_1387 -- region: 1387, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1387_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 21:24485001-24515000_strand_+_gene_1388 -- region: 1388, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1388_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.499s.\n",
      "-- designing region: 21:25485001-25515000_strand_+_gene_1389 -- region: 1389, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1389_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.658s.\n",
      "-- designing region: 21:26485001-26515000_strand_+_gene_1390 -- region: 1390, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1390_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 21:27485001-27515000_strand_+_gene_1391 -- region: 1391, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1391_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 21:28485001-28515000_strand_+_gene_1392 -- region: 1392, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1392_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.610s.\n",
      "-- designing region: 21:29485001-29515000_strand_+_gene_1393 -- region: 1393, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1393_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 21:30485001-30515000_strand_+_gene_1394 -- region: 1394, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1394_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 21:31485001-31515000_strand_+_gene_1395 -- region: 1395, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1395_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 21:32485001-32515000_strand_+_gene_1396 -- region: 1396, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1396_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 21:33485001-33515000_strand_+_gene_1397 -- region: 1397, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1397_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.612s.\n",
      "-- designing region: 21:34485001-34515000_strand_+_gene_1398 -- region: 1398, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1398_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.509s.\n",
      "-- designing region: 21:35485001-35515000_strand_+_gene_1399 -- region: 1399, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1399_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 21:36485001-36515000_strand_+_gene_1400 -- region: 1400, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1400_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 21:37485001-37515000_strand_+_gene_1401 -- region: 1401, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1401_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.466s.\n",
      "-- designing region: 21:38485001-38515000_strand_+_gene_1402 -- region: 1402, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1402_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.496s.\n",
      "-- designing region: 21:39485001-39515000_strand_+_gene_1403 -- region: 1403, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1403_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.506s.\n",
      "-- designing region: 21:40485001-40515000_strand_+_gene_1404 -- region: 1404, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1404_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.479s.\n",
      "-- designing region: 21:41485001-41515000_strand_+_gene_1405 -- region: 1405, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1405_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.511s.\n",
      "-- designing region: 21:42485001-42515000_strand_+_gene_1406 -- region: 1406, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1406_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.506s.\n",
      "-- designing region: 21:43485001-43515000_strand_+_gene_1407 -- region: 1407, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1407_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.528s.\n",
      "-- designing region: 21:44485001-44515000_strand_+_gene_1408 -- region: 1408, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1408_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: 21:45485001-45515000_strand_+_gene_1409 -- region: 1409, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1409_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.501s.\n",
      "-- designing region: 22:485001-515000_strand_+_gene_1410 -- region: 1410, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1410_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.209s.\n",
      "- Designed 0 candidate probes in 0.504s.\n",
      "-- designing region: 22:1485001-1515000_strand_+_gene_1411 -- region: 1411, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1411_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 22:2485001-2515000_strand_+_gene_1412 -- region: 1412, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1412_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.560s.\n",
      "- Designed 0 candidate probes in 0.854s.\n",
      "-- designing region: 22:3485001-3515000_strand_+_gene_1413 -- region: 1413, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1413_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.550s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: 22:4485001-4515000_strand_+_gene_1414 -- region: 1414, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1414_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 22:5485001-5515000_strand_+_gene_1415 -- region: 1415, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1415_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.551s.\n",
      "- Designed 0 candidate probes in 0.844s.\n",
      "-- designing region: 22:6485001-6515000_strand_+_gene_1416 -- region: 1416, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1416_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.556s.\n",
      "- Designed 0 candidate probes in 0.852s.\n",
      "-- designing region: 22:7485001-7515000_strand_+_gene_1417 -- region: 1417, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1417_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.848s.\n",
      "-- designing region: 22:8485001-8515000_strand_+_gene_1418 -- region: 1418, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1418_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: 22:9485001-9515000_strand_+_gene_1419 -- region: 1419, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1419_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 22:10485001-10515000_strand_+_gene_1420 -- region: 1420, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1420_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.565s.\n",
      "- Designed 9918 candidate probes in 3.683s.\n",
      "-- designing region: 22:11485001-11515000_strand_+_gene_1421 -- region: 1421, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1421_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.230s.\n",
      "- Designed 24592 candidate probes in 6.814s.\n",
      "-- designing region: 22:12485001-12515000_strand_+_gene_1422 -- region: 1422, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1422_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "- Designed 52538 candidate probes in 13.543s.\n",
      "-- designing region: 22:13485001-13515000_strand_+_gene_1423 -- region: 1423, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1423_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.228s.\n",
      "- Designed 59918 candidate probes in 15.335s.\n",
      "-- designing region: 22:14485001-14515000_strand_+_gene_1424 -- region: 1424, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1424_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.220s.\n",
      "- Designed 59918 candidate probes in 15.280s.\n",
      "-- designing region: 22:15485001-15515000_strand_+_gene_1425 -- region: 1425, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1425_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.397s.\n",
      "-- designing region: 22:16485001-16515000_strand_+_gene_1426 -- region: 1426, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1426_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.476s.\n",
      "-- designing region: 22:17485001-17515000_strand_+_gene_1427 -- region: 1427, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1427_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.481s.\n",
      "-- designing region: 22:18485001-18515000_strand_+_gene_1428 -- region: 1428, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1428_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.483s.\n",
      "-- designing region: 22:19485001-19515000_strand_+_gene_1429 -- region: 1429, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1429_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.478s.\n",
      "-- designing region: 22:20485001-20515000_strand_+_gene_1430 -- region: 1430, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1430_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.464s.\n",
      "-- designing region: 22:21485001-21515000_strand_+_gene_1431 -- region: 1431, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1431_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 22:22485001-22515000_strand_+_gene_1432 -- region: 1432, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1432_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.548s.\n",
      "-- designing region: 22:23485001-23515000_strand_+_gene_1433 -- region: 1433, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1433_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.363s.\n",
      "-- designing region: 22:24485001-24515000_strand_+_gene_1434 -- region: 1434, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1434_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 17.608s.\n",
      "-- designing region: 22:25485001-25515000_strand_+_gene_1435 -- region: 1435, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1435_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.418s.\n",
      "- Designed 59918 candidate probes in 23.789s.\n",
      "-- designing region: 22:26485001-26515000_strand_+_gene_1436 -- region: 1436, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1436_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 22:27485001-27515000_strand_+_gene_1437 -- region: 1437, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1437_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.716s.\n",
      "-- designing region: 22:28485001-28515000_strand_+_gene_1438 -- region: 1438, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1438_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 22:29485001-29515000_strand_+_gene_1439 -- region: 1439, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1439_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: 22:30485001-30515000_strand_+_gene_1440 -- region: 1440, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1440_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.694s.\n",
      "-- designing region: 22:31485001-31515000_strand_+_gene_1441 -- region: 1441, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1441_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 22:32485001-32515000_strand_+_gene_1442 -- region: 1442, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1442_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.695s.\n",
      "-- designing region: 22:33485001-33515000_strand_+_gene_1443 -- region: 1443, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1443_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.679s.\n",
      "-- designing region: 22:34485001-34515000_strand_+_gene_1444 -- region: 1444, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1444_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.700s.\n",
      "-- designing region: 22:35485001-35515000_strand_+_gene_1445 -- region: 1445, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1445_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 22:36485001-36515000_strand_+_gene_1446 -- region: 1446, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1446_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.634s.\n",
      "-- designing region: 22:37485001-37515000_strand_+_gene_1447 -- region: 1447, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1447_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.694s.\n",
      "-- designing region: 22:38485001-38515000_strand_+_gene_1448 -- region: 1448, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1448_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.701s.\n",
      "-- designing region: 22:39485001-39515000_strand_+_gene_1449 -- region: 1449, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1449_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.674s.\n",
      "-- designing region: 22:40485001-40515000_strand_+_gene_1450 -- region: 1450, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1450_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 16.330s.\n",
      "-- designing region: 22:41485001-41515000_strand_+_gene_1451 -- region: 1451, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1451_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 22:42485001-42515000_strand_+_gene_1452 -- region: 1452, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1452_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 22:43485001-43515000_strand_+_gene_1453 -- region: 1453, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1453_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 22:44485001-44515000_strand_+_gene_1454 -- region: 1454, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1454_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.672s.\n",
      "-- designing region: 22:45485001-45515000_strand_+_gene_1455 -- region: 1455, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1455_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.728s.\n",
      "-- designing region: 22:46485001-46515000_strand_+_gene_1456 -- region: 1456, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1456_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.656s.\n",
      "-- designing region: 22:47485001-47515000_strand_+_gene_1457 -- region: 1457, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1457_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 22:48485001-48515000_strand_+_gene_1458 -- region: 1458, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1458_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.699s.\n",
      "-- designing region: 22:49485001-49515000_strand_+_gene_1459 -- region: 1459, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1459_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.730s.\n",
      "-- designing region: X:1235001-1265000_strand_+_gene_1460 -- region: 1460, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1460_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: X:3735001-3765000_strand_+_gene_1461 -- region: 1461, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1461_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.599s.\n",
      "-- designing region: X:6235001-6265000_strand_+_gene_1462 -- region: 1462, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1462_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.698s.\n",
      "-- designing region: X:8735001-8765000_strand_+_gene_1463 -- region: 1463, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1463_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.612s.\n",
      "-- designing region: X:11235001-11265000_strand_+_gene_1464 -- region: 1464, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1464_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.647s.\n",
      "-- designing region: X:13735001-13765000_strand_+_gene_1465 -- region: 1465, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1465_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.664s.\n",
      "-- designing region: X:16235001-16265000_strand_+_gene_1466 -- region: 1466, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1466_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.656s.\n",
      "-- designing region: X:18735001-18765000_strand_+_gene_1467 -- region: 1467, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1467_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: X:21235001-21265000_strand_+_gene_1468 -- region: 1468, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1468_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: X:23735001-23765000_strand_+_gene_1469 -- region: 1469, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1469_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.566s.\n",
      "-- designing region: X:26235001-26265000_strand_+_gene_1470 -- region: 1470, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1470_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: X:28735001-28765000_strand_+_gene_1471 -- region: 1471, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1471_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.524s.\n",
      "-- designing region: X:31235001-31265000_strand_+_gene_1472 -- region: 1472, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1472_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.497s.\n",
      "-- designing region: X:33735001-33765000_strand_+_gene_1473 -- region: 1473, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1473_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.499s.\n",
      "-- designing region: X:36235001-36265000_strand_+_gene_1474 -- region: 1474, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1474_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.509s.\n",
      "-- designing region: X:38735001-38765000_strand_+_gene_1475 -- region: 1475, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1475_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: X:41235001-41265000_strand_+_gene_1476 -- region: 1476, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1476_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.482s.\n",
      "-- designing region: X:43735001-43765000_strand_+_gene_1477 -- region: 1477, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1477_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.503s.\n",
      "-- designing region: X:46235001-46265000_strand_+_gene_1478 -- region: 1478, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1478_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.488s.\n",
      "-- designing region: X:48735001-48765000_strand_+_gene_1479 -- region: 1479, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1479_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.259s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: X:51235001-51265000_strand_+_gene_1480 -- region: 1480, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1480_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.482s.\n",
      "-- designing region: X:53735001-53765000_strand_+_gene_1481 -- region: 1481, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1481_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.450s.\n",
      "-- designing region: X:56235001-56265000_strand_+_gene_1482 -- region: 1482, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1482_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.472s.\n",
      "-- designing region: X:58735001-58765000_strand_+_gene_1483 -- region: 1483, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1483_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "- Designed 59636 candidate probes in 15.285s.\n",
      "-- designing region: X:61235001-61265000_strand_+_gene_1484 -- region: 1484, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1484_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.225s.\n",
      "- Designed 59918 candidate probes in 15.341s.\n",
      "-- designing region: X:63735001-63765000_strand_+_gene_1485 -- region: 1485, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1485_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.436s.\n",
      "-- designing region: X:66235001-66265000_strand_+_gene_1486 -- region: 1486, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1486_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.525s.\n",
      "-- designing region: X:68735001-68765000_strand_+_gene_1487 -- region: 1487, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1487_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.501s.\n",
      "-- designing region: X:71235001-71265000_strand_+_gene_1488 -- region: 1488, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1488_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.506s.\n",
      "-- designing region: X:73735001-73765000_strand_+_gene_1489 -- region: 1489, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1489_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.462s.\n",
      "-- designing region: X:76235001-76265000_strand_+_gene_1490 -- region: 1490, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1490_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.457s.\n",
      "-- designing region: X:78735001-78765000_strand_+_gene_1491 -- region: 1491, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1491_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.518s.\n",
      "-- designing region: X:81235001-81265000_strand_+_gene_1492 -- region: 1492, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1492_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.490s.\n",
      "-- designing region: X:83735001-83765000_strand_+_gene_1493 -- region: 1493, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1493_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.502s.\n",
      "-- designing region: X:86235001-86265000_strand_+_gene_1494 -- region: 1494, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1494_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.532s.\n",
      "-- designing region: X:88735001-88765000_strand_+_gene_1495 -- region: 1495, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1495_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.488s.\n",
      "-- designing region: X:91235001-91265000_strand_+_gene_1496 -- region: 1496, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1496_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 20.020s.\n",
      "-- designing region: X:93735001-93765000_strand_+_gene_1497 -- region: 1497, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1497_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.622s.\n",
      "- Designed 59918 candidate probes in 22.269s.\n",
      "-- designing region: X:96235001-96265000_strand_+_gene_1498 -- region: 1498, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1498_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: X:98735001-98765000_strand_+_gene_1499 -- region: 1499, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1499_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: X:101235001-101265000_strand_+_gene_1500 -- region: 1500, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1500_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: X:103735001-103765000_strand_+_gene_1501 -- region: 1501, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1501_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.687s.\n",
      "-- designing region: X:106235001-106265000_strand_+_gene_1502 -- region: 1502, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1502_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.670s.\n",
      "-- designing region: X:108735001-108765000_strand_+_gene_1503 -- region: 1503, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1503_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: X:111235001-111265000_strand_+_gene_1504 -- region: 1504, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1504_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.678s.\n",
      "-- designing region: X:113735001-113765000_strand_+_gene_1505 -- region: 1505, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1505_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: X:116235001-116265000_strand_+_gene_1506 -- region: 1506, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1506_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.695s.\n",
      "-- designing region: X:118735001-118765000_strand_+_gene_1507 -- region: 1507, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1507_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.723s.\n",
      "-- designing region: X:121235001-121265000_strand_+_gene_1508 -- region: 1508, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1508_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.671s.\n",
      "-- designing region: X:123735001-123765000_strand_+_gene_1509 -- region: 1509, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1509_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.969s.\n",
      "-- designing region: X:126235001-126265000_strand_+_gene_1510 -- region: 1510, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1510_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: X:128735001-128765000_strand_+_gene_1511 -- region: 1511, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1511_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: X:131235001-131265000_strand_+_gene_1512 -- region: 1512, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1512_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.678s.\n",
      "-- designing region: X:133735001-133765000_strand_+_gene_1513 -- region: 1513, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1513_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: X:136235001-136265000_strand_+_gene_1514 -- region: 1514, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1514_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.669s.\n",
      "-- designing region: X:138735001-138765000_strand_+_gene_1515 -- region: 1515, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1515_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 92.578s.\n",
      "-- designing region: X:141235001-141265000_strand_+_gene_1516 -- region: 1516, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1516_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.938s.\n",
      "-- designing region: X:143735001-143765000_strand_+_gene_1517 -- region: 1517, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1517_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.217s.\n",
      "- Designed 59918 candidate probes in 15.807s.\n",
      "-- designing region: X:146235001-146265000_strand_+_gene_1518 -- region: 1518, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1518_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.862s.\n",
      "-- designing region: X:148735001-148765000_strand_+_gene_1519 -- region: 1519, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1519_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.243s.\n",
      "- Designed 59918 candidate probes in 16.043s.\n",
      "-- designing region: X:151235001-151265000_strand_+_gene_1520 -- region: 1520, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1520_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.940s.\n",
      "-- designing region: X:153735001-153765000_strand_+_gene_1521 -- region: 1521, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1521_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.891s.\n",
      "-- designing region: Y:485001-515000_strand_+_gene_1522 -- region: 1522, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1522_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.145s.\n",
      "- Designed 0 candidate probes in 0.440s.\n",
      "-- designing region: Y:1485001-1515000_strand_+_gene_1523 -- region: 1523, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1523_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.459s.\n",
      "- Designed 0 candidate probes in 0.762s.\n",
      "-- designing region: Y:2485001-2515000_strand_+_gene_1524 -- region: 1524, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1524_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.450s.\n",
      "- Designed 0 candidate probes in 0.746s.\n",
      "-- designing region: Y:3485001-3515000_strand_+_gene_1525 -- region: 1525, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1525_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.581s.\n",
      "- Designed 59918 candidate probes in 16.273s.\n",
      "-- designing region: Y:4485001-4515000_strand_+_gene_1526 -- region: 1526, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1526_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.882s.\n",
      "-- designing region: Y:5485001-5515000_strand_+_gene_1527 -- region: 1527, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1527_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.870s.\n",
      "-- designing region: Y:6485001-6515000_strand_+_gene_1528 -- region: 1528, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1528_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.919s.\n",
      "-- designing region: Y:7485001-7515000_strand_+_gene_1529 -- region: 1529, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1529_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.213s.\n",
      "- Designed 59918 candidate probes in 15.770s.\n",
      "-- designing region: Y:8485001-8515000_strand_+_gene_1530 -- region: 1530, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1530_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.798s.\n",
      "-- designing region: Y:9485001-9515000_strand_+_gene_1531 -- region: 1531, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1531_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.256s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: Y:10485001-10515000_strand_+_gene_1532 -- region: 1532, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1532_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "- Designed 59918 candidate probes in 15.355s.\n",
      "-- designing region: Y:11485001-11515000_strand_+_gene_1533 -- region: 1533, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1533_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "- Designed 59918 candidate probes in 15.424s.\n",
      "-- designing region: Y:12485001-12515000_strand_+_gene_1534 -- region: 1534, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1534_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.354s.\n",
      "- Designed 59918 candidate probes in 15.528s.\n",
      "-- designing region: Y:13485001-13515000_strand_+_gene_1535 -- region: 1535, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1535_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: Y:14485001-14515000_strand_+_gene_1536 -- region: 1536, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1536_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.537s.\n",
      "-- designing region: Y:15485001-15515000_strand_+_gene_1537 -- region: 1537, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1537_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.534s.\n",
      "-- designing region: Y:16485001-16515000_strand_+_gene_1538 -- region: 1538, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1538_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.484s.\n",
      "-- designing region: Y:17485001-17515000_strand_+_gene_1539 -- region: 1539, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1539_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: Y:18485001-18515000_strand_+_gene_1540 -- region: 1540, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1540_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.457s.\n",
      "-- designing region: Y:19485001-19515000_strand_+_gene_1541 -- region: 1541, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1541_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.506s.\n",
      "-- designing region: Y:20485001-20515000_strand_+_gene_1542 -- region: 1542, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1542_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: Y:21485001-21515000_strand_+_gene_1543 -- region: 1543, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1543_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: Y:22485001-22515000_strand_+_gene_1544 -- region: 1544, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1544_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: Y:23485001-23515000_strand_+_gene_1545 -- region: 1545, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1545_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.491s.\n",
      "-- designing region: Y:24485001-24515000_strand_+_gene_1546 -- region: 1546, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1546_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.514s.\n",
      "-- designing region: Y:25485001-25515000_strand_+_gene_1547 -- region: 1547, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1547_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: Y:26485001-26515000_strand_+_gene_1548 -- region: 1548, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1548_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.495s.\n",
      "-- designing region: Y:27485001-27515000_strand_+_gene_1549 -- region: 1549, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1549_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "- Designed 0 candidate probes in 0.495s.\n",
      "-- designing region: Y:28485001-28515000_strand_+_gene_1550 -- region: 1550, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1550_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.563s.\n",
      "- Designed 0 candidate probes in 0.857s.\n",
      "-- designing region: Y:29485001-29515000_strand_+_gene_1551 -- region: 1551, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1551_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: Y:30485001-30515000_strand_+_gene_1552 -- region: 1552, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1552_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.561s.\n",
      "- Designed 0 candidate probes in 0.856s.\n",
      "-- designing region: Y:31485001-31515000_strand_+_gene_1553 -- region: 1553, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1553_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.550s.\n",
      "- Designed 0 candidate probes in 0.841s.\n",
      "-- designing region: Y:32485001-32515000_strand_+_gene_1554 -- region: 1554, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1554_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.560s.\n",
      "- Designed 0 candidate probes in 0.854s.\n",
      "-- designing region: Y:33485001-33515000_strand_+_gene_1555 -- region: 1555, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1555_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: Y:34485001-34515000_strand_+_gene_1556 -- region: 1556, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1556_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.851s.\n",
      "-- designing region: Y:35485001-35515000_strand_+_gene_1557 -- region: 1557, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1557_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: Y:36485001-36515000_strand_+_gene_1558 -- region: 1558, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1558_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.552s.\n",
      "- Designed 0 candidate probes in 0.842s.\n",
      "-- designing region: Y:37485001-37515000_strand_+_gene_1559 -- region: 1559, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1559_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.850s.\n",
      "-- designing region: Y:38485001-38515000_strand_+_gene_1560 -- region: 1560, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1560_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.551s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: Y:39485001-39515000_strand_+_gene_1561 -- region: 1561, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1561_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: Y:40485001-40515000_strand_+_gene_1562 -- region: 1562, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1562_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.561s.\n",
      "- Designed 0 candidate probes in 0.855s.\n",
      "-- designing region: Y:41485001-41515000_strand_+_gene_1563 -- region: 1563, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1563_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: Y:42485001-42515000_strand_+_gene_1564 -- region: 1564, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1564_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.559s.\n",
      "- Designed 0 candidate probes in 0.854s.\n",
      "-- designing region: Y:43485001-43515000_strand_+_gene_1565 -- region: 1565, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1565_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.551s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: Y:44485001-44515000_strand_+_gene_1566 -- region: 1566, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1566_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.558s.\n",
      "- Designed 0 candidate probes in 0.851s.\n",
      "-- designing region: Y:45485001-45515000_strand_+_gene_1567 -- region: 1567, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1567_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: Y:46485001-46515000_strand_+_gene_1568 -- region: 1568, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1568_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.551s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: Y:47485001-47515000_strand_+_gene_1569 -- region: 1569, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1569_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: Y:48485001-48515000_strand_+_gene_1570 -- region: 1570, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1570_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: Y:49485001-49515000_strand_+_gene_1571 -- region: 1571, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1571_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.552s.\n",
      "- Designed 0 candidate probes in 0.848s.\n",
      "-- designing region: Y:50485001-50515000_strand_+_gene_1572 -- region: 1572, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1572_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.848s.\n",
      "-- designing region: Y:51485001-51515000_strand_+_gene_1573 -- region: 1573, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1573_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.850s.\n",
      "-- designing region: Y:52485001-52515000_strand_+_gene_1574 -- region: 1574, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1574_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.564s.\n",
      "- Designed 0 candidate probes in 0.860s.\n",
      "-- designing region: Y:53485001-53515000_strand_+_gene_1575 -- region: 1575, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1575_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: Y:54485001-54515000_strand_+_gene_1576 -- region: 1576, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1576_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.561s.\n",
      "- Designed 0 candidate probes in 0.855s.\n",
      "-- designing region: Y:55485001-55515000_strand_+_gene_1577 -- region: 1577, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1577_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: Y:56485001-56515000_strand_+_gene_1578 -- region: 1578, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1578_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.848s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr\n",
      "CPU times: total: 7h 7min 3s\n",
      "Wall time: 7h 14min 10s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "pb_designer.computeOTmaps() # load the tables \n",
    "pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e23bacce",
   "metadata": {},
   "source": [
    "### check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "8541e32f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 1:1235001-1265000_strand_+_gene_0, 57068 candidate probes\n",
      "--- 19930 probes passed check_dic selection.\n",
      "finish in 1.719s, 344 probes kept.\n",
      "-- check region:1 1:3735001-3765000_strand_+_gene_1, 57548 candidate probes\n",
      "--- 17833 probes passed check_dic selection.\n",
      "finish in 1.605s, 319 probes kept.\n",
      "-- check region:2 1:6235001-6265000_strand_+_gene_2, 58584 candidate probes\n",
      "--- 24326 probes passed check_dic selection.\n",
      "finish in 1.832s, 377 probes kept.\n",
      "-- check region:3 1:8735001-8765000_strand_+_gene_3, 58690 candidate probes\n",
      "--- 12490 probes passed check_dic selection.\n",
      "finish in 1.391s, 242 probes kept.\n",
      "-- check region:4 1:11235001-11265000_strand_+_gene_4, 55772 candidate probes\n",
      "--- 15903 probes passed check_dic selection.\n",
      "finish in 1.482s, 256 probes kept.\n",
      "-- check region:5 1:13735001-13765000_strand_+_gene_5, 58958 candidate probes\n",
      "--- 16480 probes passed check_dic selection.\n",
      "finish in 1.511s, 303 probes kept.\n",
      "-- check region:6 1:16235001-16265000_strand_+_gene_6, 57896 candidate probes\n",
      "--- 16713 probes passed check_dic selection.\n",
      "finish in 1.550s, 300 probes kept.\n",
      "-- check region:7 1:18735001-18765000_strand_+_gene_7, 58950 candidate probes\n",
      "--- 19815 probes passed check_dic selection.\n",
      "finish in 1.670s, 341 probes kept.\n",
      "-- check region:8 1:21235001-21265000_strand_+_gene_8, 58040 candidate probes\n",
      "--- 20685 probes passed check_dic selection.\n",
      "finish in 1.693s, 323 probes kept.\n",
      "-- check region:9 1:23735001-23765000_strand_+_gene_9, 57028 candidate probes\n",
      "--- 14310 probes passed check_dic selection.\n",
      "finish in 1.445s, 264 probes kept.\n",
      "-- check region:10 1:26235001-26265000_strand_+_gene_10, 56510 candidate probes\n",
      "--- 11115 probes passed check_dic selection.\n",
      "finish in 1.315s, 216 probes kept.\n",
      "-- check region:11 1:28735001-28765000_strand_+_gene_11, 57208 candidate probes\n",
      "--- 10131 probes passed check_dic selection.\n",
      "finish in 1.286s, 212 probes kept.\n",
      "-- check region:12 1:31235001-31265000_strand_+_gene_12, 57094 candidate probes\n",
      "--- 13737 probes passed check_dic selection.\n",
      "finish in 1.412s, 240 probes kept.\n",
      "-- check region:13 1:33735001-33765000_strand_+_gene_13, 58082 candidate probes\n",
      "--- 21173 probes passed check_dic selection.\n",
      "finish in 1.715s, 341 probes kept.\n",
      "-- check region:14 1:36235001-36265000_strand_+_gene_14, 54616 candidate probes\n",
      "--- 8851 probes passed check_dic selection.\n",
      "finish in 1.180s, 176 probes kept.\n",
      "-- check region:15 1:38735001-38765000_strand_+_gene_15, 58356 candidate probes\n",
      "--- 23078 probes passed check_dic selection.\n",
      "finish in 1.780s, 343 probes kept.\n",
      "-- check region:16 1:41235001-41265000_strand_+_gene_16, 58510 candidate probes\n",
      "--- 12680 probes passed check_dic selection.\n",
      "finish in 1.388s, 245 probes kept.\n",
      "-- check region:17 1:43735001-43765000_strand_+_gene_17, 57914 candidate probes\n",
      "--- 10224 probes passed check_dic selection.\n",
      "finish in 1.285s, 203 probes kept.\n",
      "-- check region:18 1:46235001-46265000_strand_+_gene_18, 53492 candidate probes\n",
      "--- 11883 probes passed check_dic selection.\n",
      "finish in 1.282s, 204 probes kept.\n",
      "-- check region:19 1:48735001-48765000_strand_+_gene_19, 59554 candidate probes\n",
      "--- 24548 probes passed check_dic selection.\n",
      "finish in 1.879s, 391 probes kept.\n",
      "-- check region:20 1:51235001-51265000_strand_+_gene_20, 58438 candidate probes\n",
      "--- 9236 probes passed check_dic selection.\n",
      "finish in 1.275s, 206 probes kept.\n",
      "-- check region:21 1:53735001-53765000_strand_+_gene_21, 58628 candidate probes\n",
      "--- 14481 probes passed check_dic selection.\n",
      "finish in 1.458s, 255 probes kept.\n",
      "-- check region:22 1:56235001-56265000_strand_+_gene_22, 56386 candidate probes\n",
      "--- 17122 probes passed check_dic selection.\n",
      "finish in 1.519s, 303 probes kept.\n",
      "-- check region:23 1:58735001-58765000_strand_+_gene_23, 58276 candidate probes\n",
      "--- 19752 probes passed check_dic selection.\n",
      "finish in 1.649s, 333 probes kept.\n",
      "-- check region:24 1:61235001-61265000_strand_+_gene_24, 58850 candidate probes\n",
      "--- 14774 probes passed check_dic selection.\n",
      "finish in 1.454s, 279 probes kept.\n",
      "-- check region:25 1:63735001-63765000_strand_+_gene_25, 58436 candidate probes\n",
      "--- 13647 probes passed check_dic selection.\n",
      "finish in 1.424s, 238 probes kept.\n",
      "-- check region:26 1:66235001-66265000_strand_+_gene_26, 59758 candidate probes\n",
      "--- 17731 probes passed check_dic selection.\n",
      "finish in 1.589s, 328 probes kept.\n",
      "-- check region:27 1:68735001-68765000_strand_+_gene_27, 50588 candidate probes\n",
      "--- 6119 probes passed check_dic selection.\n",
      "finish in 1.018s, 149 probes kept.\n",
      "-- check region:28 1:71235001-71265000_strand_+_gene_28, 59792 candidate probes\n",
      "--- 12680 probes passed check_dic selection.\n",
      "finish in 1.387s, 262 probes kept.\n",
      "-- check region:29 1:73735001-73765000_strand_+_gene_29, 59876 candidate probes\n",
      "--- 9387 probes passed check_dic selection.\n",
      "finish in 1.246s, 199 probes kept.\n",
      "-- check region:30 1:76235001-76265000_strand_+_gene_30, 58912 candidate probes\n",
      "--- 14145 probes passed check_dic selection.\n",
      "finish in 1.438s, 263 probes kept.\n",
      "-- check region:31 1:78735001-78765000_strand_+_gene_31, 59318 candidate probes\n",
      "--- 9584 probes passed check_dic selection.\n",
      "finish in 1.263s, 215 probes kept.\n",
      "-- check region:32 1:81235001-81265000_strand_+_gene_32, 56240 candidate probes\n",
      "--- 9269 probes passed check_dic selection.\n",
      "finish in 1.211s, 190 probes kept.\n",
      "-- check region:33 1:83735001-83765000_strand_+_gene_33, 59506 candidate probes\n",
      "--- 19081 probes passed check_dic selection.\n",
      "finish in 1.633s, 338 probes kept.\n",
      "-- check region:34 1:86235001-86265000_strand_+_gene_34, 54984 candidate probes\n",
      "--- 12953 probes passed check_dic selection.\n",
      "finish in 1.324s, 230 probes kept.\n",
      "-- check region:35 1:88735001-88765000_strand_+_gene_35, 59162 candidate probes\n",
      "--- 9335 probes passed check_dic selection.\n",
      "finish in 1.247s, 203 probes kept.\n",
      "-- check region:36 1:91235001-91265000_strand_+_gene_36, 53656 candidate probes\n",
      "--- 6720 probes passed check_dic selection.\n",
      "finish in 1.088s, 154 probes kept.\n",
      "-- check region:37 1:93735001-93765000_strand_+_gene_37, 59440 candidate probes\n",
      "--- 23915 probes passed check_dic selection.\n",
      "finish in 1.839s, 358 probes kept.\n",
      "-- check region:38 1:96235001-96265000_strand_+_gene_38, 51066 candidate probes\n",
      "--- 10362 probes passed check_dic selection.\n",
      "finish in 1.177s, 211 probes kept.\n",
      "-- check region:39 1:98735001-98765000_strand_+_gene_39, 59716 candidate probes\n",
      "--- 11984 probes passed check_dic selection.\n",
      "finish in 1.354s, 237 probes kept.\n",
      "-- check region:40 1:101235001-101265000_strand_+_gene_40, 58420 candidate probes\n",
      "--- 15801 probes passed check_dic selection.\n",
      "finish in 1.513s, 293 probes kept.\n",
      "-- check region:41 1:103735001-103765000_strand_+_gene_41, 59728 candidate probes\n",
      "--- 676 probes passed check_dic selection.\n",
      "finish in 0.963s, 22 probes kept.\n",
      "-- check region:42 1:106235001-106265000_strand_+_gene_42, 55970 candidate probes\n",
      "--- 9129 probes passed check_dic selection.\n",
      "finish in 1.193s, 201 probes kept.\n",
      "-- check region:43 1:108735001-108765000_strand_+_gene_43, 59108 candidate probes\n",
      "--- 12044 probes passed check_dic selection.\n",
      "finish in 1.361s, 235 probes kept.\n",
      "-- check region:44 1:111235001-111265000_strand_+_gene_44, 59668 candidate probes\n",
      "--- 19819 probes passed check_dic selection.\n",
      "finish in 1.701s, 317 probes kept.\n",
      "-- check region:45 1:113735001-113765000_strand_+_gene_45, 58218 candidate probes\n",
      "--- 7562 probes passed check_dic selection.\n",
      "finish in 1.180s, 187 probes kept.\n",
      "-- check region:46 1:116235001-116265000_strand_+_gene_46, 59238 candidate probes\n",
      "--- 18078 probes passed check_dic selection.\n",
      "finish in 1.601s, 309 probes kept.\n",
      "-- check region:47 1:118735001-118765000_strand_+_gene_47, 57846 candidate probes\n",
      "--- 10969 probes passed check_dic selection.\n",
      "finish in 1.301s, 213 probes kept.\n",
      "-- check region:48 1:121235001-121265000_strand_+_gene_48, 41546 candidate probes\n",
      "--- 82 probes passed check_dic selection.\n",
      "finish in 0.670s, 3 probes kept.\n",
      "-- check region:49 1:123735001-123765000_strand_+_gene_49, 20180 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.324s, 0 probes kept.\n",
      "-- check region:50 1:126235001-126265000_strand_+_gene_50, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:51 1:128735001-128765000_strand_+_gene_51, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:52 1:131235001-131265000_strand_+_gene_52, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:53 1:133735001-133765000_strand_+_gene_53, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:54 1:136235001-136265000_strand_+_gene_54, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:55 1:138735001-138765000_strand_+_gene_55, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:56 1:141235001-141265000_strand_+_gene_56, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:57 1:143735001-143765000_strand_+_gene_57, 57134 candidate probes\n",
      "--- 1466 probes passed check_dic selection.\n",
      "finish in 0.973s, 60 probes kept.\n",
      "-- check region:58 1:146235001-146265000_strand_+_gene_58, 57912 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.925s, 0 probes kept.\n",
      "-- check region:59 1:148735001-148765000_strand_+_gene_59, 58022 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.911s, 0 probes kept.\n",
      "-- check region:60 1:151235001-151265000_strand_+_gene_60, 56816 candidate probes\n",
      "--- 8909 probes passed check_dic selection.\n",
      "finish in 1.247s, 187 probes kept.\n",
      "-- check region:61 1:153735001-153765000_strand_+_gene_61, 57570 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.949s, 0 probes kept.\n",
      "-- check region:62 1:156235001-156265000_strand_+_gene_62, 57474 candidate probes\n",
      "--- 18009 probes passed check_dic selection.\n",
      "finish in 1.608s, 330 probes kept.\n",
      "-- check region:63 1:158735001-158765000_strand_+_gene_63, 55158 candidate probes\n",
      "--- 11401 probes passed check_dic selection.\n",
      "finish in 1.281s, 241 probes kept.\n",
      "-- check region:64 1:161235001-161265000_strand_+_gene_64, 55970 candidate probes\n",
      "--- 11378 probes passed check_dic selection.\n",
      "finish in 1.320s, 199 probes kept.\n",
      "-- check region:65 1:163735001-163765000_strand_+_gene_65, 51598 candidate probes\n",
      "--- 5205 probes passed check_dic selection.\n",
      "finish in 0.996s, 114 probes kept.\n",
      "-- check region:66 1:166235001-166265000_strand_+_gene_66, 43668 candidate probes\n",
      "--- 6591 probes passed check_dic selection.\n",
      "finish in 0.932s, 127 probes kept.\n",
      "-- check region:67 1:168735001-168765000_strand_+_gene_67, 55396 candidate probes\n",
      "--- 16568 probes passed check_dic selection.\n",
      "finish in 1.483s, 271 probes kept.\n",
      "-- check region:68 1:171235001-171265000_strand_+_gene_68, 58478 candidate probes\n",
      "--- 13146 probes passed check_dic selection.\n",
      "finish in 1.408s, 226 probes kept.\n",
      "-- check region:69 1:173735001-173765000_strand_+_gene_69, 57446 candidate probes\n",
      "--- 5044 probes passed check_dic selection.\n",
      "finish in 1.082s, 128 probes kept.\n",
      "-- check region:70 1:176235001-176265000_strand_+_gene_70, 48472 candidate probes\n",
      "--- 5579 probes passed check_dic selection.\n",
      "finish in 0.957s, 120 probes kept.\n",
      "-- check region:71 1:178735001-178765000_strand_+_gene_71, 57946 candidate probes\n",
      "--- 5882 probes passed check_dic selection.\n",
      "finish in 1.124s, 137 probes kept.\n",
      "-- check region:72 1:181235001-181265000_strand_+_gene_72, 58242 candidate probes\n",
      "--- 16338 probes passed check_dic selection.\n",
      "finish in 1.533s, 267 probes kept.\n",
      "-- check region:73 1:183735001-183765000_strand_+_gene_73, 57770 candidate probes\n",
      "--- 9739 probes passed check_dic selection.\n",
      "finish in 1.311s, 198 probes kept.\n",
      "-- check region:74 1:186235001-186265000_strand_+_gene_74, 53270 candidate probes\n",
      "--- 9123 probes passed check_dic selection.\n",
      "finish in 1.177s, 173 probes kept.\n",
      "-- check region:75 1:188735001-188765000_strand_+_gene_75, 51052 candidate probes\n",
      "--- 6244 probes passed check_dic selection.\n",
      "finish in 1.018s, 149 probes kept.\n",
      "-- check region:76 1:191235001-191265000_strand_+_gene_76, 58588 candidate probes\n",
      "--- 3622 probes passed check_dic selection.\n",
      "finish in 1.039s, 121 probes kept.\n",
      "-- check region:77 1:193735001-193765000_strand_+_gene_77, 59194 candidate probes\n",
      "--- 10009 probes passed check_dic selection.\n",
      "finish in 1.270s, 218 probes kept.\n",
      "-- check region:78 1:196235001-196265000_strand_+_gene_78, 59454 candidate probes\n",
      "--- 4442 probes passed check_dic selection.\n",
      "finish in 1.067s, 142 probes kept.\n",
      "-- check region:79 1:198735001-198765000_strand_+_gene_79, 59752 candidate probes\n",
      "--- 8681 probes passed check_dic selection.\n",
      "finish in 1.236s, 198 probes kept.\n",
      "-- check region:80 1:201235001-201265000_strand_+_gene_80, 51892 candidate probes\n",
      "--- 14788 probes passed check_dic selection.\n",
      "finish in 1.357s, 247 probes kept.\n",
      "-- check region:81 1:203735001-203765000_strand_+_gene_81, 56642 candidate probes\n",
      "--- 11938 probes passed check_dic selection.\n",
      "finish in 1.363s, 223 probes kept.\n",
      "-- check region:82 1:206235001-206265000_strand_+_gene_82, 59142 candidate probes\n",
      "--- 124 probes passed check_dic selection.\n",
      "finish in 0.958s, 8 probes kept.\n",
      "-- check region:83 1:208735001-208765000_strand_+_gene_83, 57958 candidate probes\n",
      "--- 14684 probes passed check_dic selection.\n",
      "finish in 1.442s, 259 probes kept.\n",
      "-- check region:84 1:211235001-211265000_strand_+_gene_84, 55774 candidate probes\n",
      "--- 9538 probes passed check_dic selection.\n",
      "finish in 1.217s, 191 probes kept.\n",
      "-- check region:85 1:213735001-213765000_strand_+_gene_85, 59034 candidate probes\n",
      "--- 19858 probes passed check_dic selection.\n",
      "finish in 1.657s, 313 probes kept.\n",
      "-- check region:86 1:216235001-216265000_strand_+_gene_86, 59444 candidate probes\n",
      "--- 10988 probes passed check_dic selection.\n",
      "finish in 1.316s, 237 probes kept.\n",
      "-- check region:87 1:218735001-218765000_strand_+_gene_87, 59202 candidate probes\n",
      "--- 11848 probes passed check_dic selection.\n",
      "finish in 1.368s, 236 probes kept.\n",
      "-- check region:88 1:221235001-221265000_strand_+_gene_88, 55744 candidate probes\n",
      "--- 9701 probes passed check_dic selection.\n",
      "finish in 1.227s, 204 probes kept.\n",
      "-- check region:89 1:223735001-223765000_strand_+_gene_89, 58376 candidate probes\n",
      "--- 21410 probes passed check_dic selection.\n",
      "finish in 1.720s, 335 probes kept.\n",
      "-- check region:90 1:226235001-226265000_strand_+_gene_90, 55972 candidate probes\n",
      "--- 4457 probes passed check_dic selection.\n",
      "finish in 1.037s, 118 probes kept.\n",
      "-- check region:91 1:228735001-228765000_strand_+_gene_91, 59636 candidate probes\n",
      "--- 22761 probes passed check_dic selection.\n",
      "finish in 1.803s, 363 probes kept.\n",
      "-- check region:92 1:231235001-231265000_strand_+_gene_92, 58162 candidate probes\n",
      "--- 15167 probes passed check_dic selection.\n",
      "finish in 1.481s, 271 probes kept.\n",
      "-- check region:93 1:233735001-233765000_strand_+_gene_93, 58296 candidate probes\n",
      "--- 11461 probes passed check_dic selection.\n",
      "finish in 1.318s, 212 probes kept.\n",
      "-- check region:94 1:236235001-236265000_strand_+_gene_94, 57698 candidate probes\n",
      "--- 8821 probes passed check_dic selection.\n",
      "finish in 1.217s, 178 probes kept.\n",
      "-- check region:95 1:238735001-238765000_strand_+_gene_95, 58698 candidate probes\n",
      "--- 9685 probes passed check_dic selection.\n",
      "finish in 1.260s, 214 probes kept.\n",
      "-- check region:96 1:241235001-241265000_strand_+_gene_96, 57548 candidate probes\n",
      "--- 14944 probes passed check_dic selection.\n",
      "finish in 1.448s, 258 probes kept.\n",
      "-- check region:97 1:243735001-243765000_strand_+_gene_97, 59166 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.920s, 0 probes kept.\n",
      "-- check region:98 1:246235001-246265000_strand_+_gene_98, 54048 candidate probes\n",
      "--- 7653 probes passed check_dic selection.\n",
      "finish in 1.120s, 168 probes kept.\n",
      "-- check region:99 2:1235001-1265000_strand_+_gene_99, 58542 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.926s, 0 probes kept.\n",
      "-- check region:100 2:3735001-3765000_strand_+_gene_100, 59230 candidate probes\n",
      "--- 22508 probes passed check_dic selection.\n",
      "finish in 1.763s, 344 probes kept.\n",
      "-- check region:101 2:6235001-6265000_strand_+_gene_101, 58343 candidate probes\n",
      "--- 15962 probes passed check_dic selection.\n",
      "finish in 1.491s, 286 probes kept.\n",
      "-- check region:102 2:8735001-8765000_strand_+_gene_102, 59048 candidate probes\n",
      "--- 11359 probes passed check_dic selection.\n",
      "finish in 1.335s, 240 probes kept.\n",
      "-- check region:103 2:11235001-11265000_strand_+_gene_103, 57902 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 8477 probes passed check_dic selection.\n",
      "finish in 1.215s, 205 probes kept.\n",
      "-- check region:104 2:13735001-13765000_strand_+_gene_104, 52880 candidate probes\n",
      "--- 3896 probes passed check_dic selection.\n",
      "finish in 0.969s, 119 probes kept.\n",
      "-- check region:105 2:16235001-16265000_strand_+_gene_105, 59376 candidate probes\n",
      "--- 23556 probes passed check_dic selection.\n",
      "finish in 1.818s, 352 probes kept.\n",
      "-- check region:106 2:18735001-18765000_strand_+_gene_106, 57758 candidate probes\n",
      "--- 13112 probes passed check_dic selection.\n",
      "finish in 1.377s, 244 probes kept.\n",
      "-- check region:107 2:21235001-21265000_strand_+_gene_107, 58946 candidate probes\n",
      "--- 19363 probes passed check_dic selection.\n",
      "finish in 1.624s, 305 probes kept.\n",
      "-- check region:108 2:23735001-23765000_strand_+_gene_108, 53196 candidate probes\n",
      "--- 9170 probes passed check_dic selection.\n",
      "finish in 1.173s, 213 probes kept.\n",
      "-- check region:109 2:26235001-26265000_strand_+_gene_109, 57444 candidate probes\n",
      "--- 10774 probes passed check_dic selection.\n",
      "finish in 1.303s, 225 probes kept.\n",
      "-- check region:110 2:28735001-28765000_strand_+_gene_110, 58032 candidate probes\n",
      "--- 12274 probes passed check_dic selection.\n",
      "finish in 1.354s, 230 probes kept.\n",
      "-- check region:111 2:31235001-31265000_strand_+_gene_111, 58774 candidate probes\n",
      "--- 25061 probes passed check_dic selection.\n",
      "finish in 1.865s, 358 probes kept.\n",
      "-- check region:112 2:33735001-33765000_strand_+_gene_112, 56124 candidate probes\n",
      "--- 7636 probes passed check_dic selection.\n",
      "finish in 1.158s, 167 probes kept.\n",
      "-- check region:113 2:36235001-36265000_strand_+_gene_113, 59548 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.937s, 0 probes kept.\n",
      "-- check region:114 2:38735001-38765000_strand_+_gene_114, 55546 candidate probes\n",
      "--- 11253 probes passed check_dic selection.\n",
      "finish in 1.294s, 215 probes kept.\n",
      "-- check region:115 2:41235001-41265000_strand_+_gene_115, 59450 candidate probes\n",
      "--- 11107 probes passed check_dic selection.\n",
      "finish in 1.319s, 224 probes kept.\n",
      "-- check region:116 2:43735001-43765000_strand_+_gene_116, 58272 candidate probes\n",
      "--- 11170 probes passed check_dic selection.\n",
      "finish in 1.319s, 228 probes kept.\n",
      "-- check region:117 2:46235001-46265000_strand_+_gene_117, 58614 candidate probes\n",
      "--- 16184 probes passed check_dic selection.\n",
      "finish in 1.524s, 276 probes kept.\n",
      "-- check region:118 2:48735001-48765000_strand_+_gene_118, 49048 candidate probes\n",
      "--- 12350 probes passed check_dic selection.\n",
      "finish in 1.227s, 216 probes kept.\n",
      "-- check region:119 2:51235001-51265000_strand_+_gene_119, 59480 candidate probes\n",
      "--- 10455 probes passed check_dic selection.\n",
      "finish in 1.291s, 221 probes kept.\n",
      "-- check region:120 2:53735001-53765000_strand_+_gene_120, 59252 candidate probes\n",
      "--- 8651 probes passed check_dic selection.\n",
      "finish in 1.245s, 200 probes kept.\n",
      "-- check region:121 2:56235001-56265000_strand_+_gene_121, 59064 candidate probes\n",
      "--- 16258 probes passed check_dic selection.\n",
      "finish in 1.524s, 295 probes kept.\n",
      "-- check region:122 2:58735001-58765000_strand_+_gene_122, 59346 candidate probes\n",
      "--- 15901 probes passed check_dic selection.\n",
      "finish in 1.515s, 270 probes kept.\n",
      "-- check region:123 2:61235001-61265000_strand_+_gene_123, 58406 candidate probes\n",
      "--- 7260 probes passed check_dic selection.\n",
      "finish in 1.178s, 182 probes kept.\n",
      "-- check region:124 2:63735001-63765000_strand_+_gene_124, 59256 candidate probes\n",
      "--- 13150 probes passed check_dic selection.\n",
      "finish in 1.411s, 235 probes kept.\n",
      "-- check region:125 2:66235001-66265000_strand_+_gene_125, 55576 candidate probes\n",
      "--- 14592 probes passed check_dic selection.\n",
      "finish in 1.411s, 251 probes kept.\n",
      "-- check region:126 2:68735001-68765000_strand_+_gene_126, 59500 candidate probes\n",
      "--- 16537 probes passed check_dic selection.\n",
      "finish in 1.556s, 291 probes kept.\n",
      "-- check region:127 2:71235001-71265000_strand_+_gene_127, 55764 candidate probes\n",
      "--- 12597 probes passed check_dic selection.\n",
      "finish in 1.355s, 222 probes kept.\n",
      "-- check region:128 2:73735001-73765000_strand_+_gene_128, 57684 candidate probes\n",
      "--- 18209 probes passed check_dic selection.\n",
      "finish in 1.582s, 275 probes kept.\n",
      "-- check region:129 2:76235001-76265000_strand_+_gene_129, 59578 candidate probes\n",
      "--- 10650 probes passed check_dic selection.\n",
      "finish in 1.310s, 216 probes kept.\n",
      "-- check region:130 2:78735001-78765000_strand_+_gene_130, 57898 candidate probes\n",
      "--- 6208 probes passed check_dic selection.\n",
      "finish in 1.130s, 162 probes kept.\n",
      "-- check region:131 2:81235001-81265000_strand_+_gene_131, 59160 candidate probes\n",
      "--- 8527 probes passed check_dic selection.\n",
      "finish in 1.214s, 190 probes kept.\n",
      "-- check region:132 2:83735001-83765000_strand_+_gene_132, 59078 candidate probes\n",
      "--- 9011 probes passed check_dic selection.\n",
      "finish in 1.243s, 192 probes kept.\n",
      "-- check region:133 2:86235001-86265000_strand_+_gene_133, 57158 candidate probes\n",
      "--- 19043 probes passed check_dic selection.\n",
      "finish in 1.612s, 312 probes kept.\n",
      "-- check region:134 2:88735001-88765000_strand_+_gene_134, 58082 candidate probes\n",
      "--- 13769 probes passed check_dic selection.\n",
      "finish in 1.431s, 237 probes kept.\n",
      "-- check region:135 2:91235001-91265000_strand_+_gene_135, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:136 2:93735001-93765000_strand_+_gene_136, 19752 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.319s, 0 probes kept.\n",
      "-- check region:137 2:96235001-96265000_strand_+_gene_137, 57980 candidate probes\n",
      "--- 19278 probes passed check_dic selection.\n",
      "finish in 1.651s, 315 probes kept.\n",
      "-- check region:138 2:98735001-98765000_strand_+_gene_138, 58622 candidate probes\n",
      "--- 16089 probes passed check_dic selection.\n",
      "finish in 1.525s, 285 probes kept.\n",
      "-- check region:139 2:101235001-101265000_strand_+_gene_139, 56614 candidate probes\n",
      "--- 8754 probes passed check_dic selection.\n",
      "finish in 1.217s, 174 probes kept.\n",
      "-- check region:140 2:103735001-103765000_strand_+_gene_140, 59398 candidate probes\n",
      "--- 14226 probes passed check_dic selection.\n",
      "finish in 1.443s, 261 probes kept.\n",
      "-- check region:141 2:106235001-106265000_strand_+_gene_141, 57460 candidate probes\n",
      "--- 3231 probes passed check_dic selection.\n",
      "finish in 1.043s, 74 probes kept.\n",
      "-- check region:142 2:108735001-108765000_strand_+_gene_142, 57586 candidate probes\n",
      "--- 1138 probes passed check_dic selection.\n",
      "finish in 0.942s, 39 probes kept.\n",
      "-- check region:143 2:111235001-111265000_strand_+_gene_143, 58894 candidate probes\n",
      "--- 10479 probes passed check_dic selection.\n",
      "finish in 1.317s, 186 probes kept.\n",
      "-- check region:144 2:113735001-113765000_strand_+_gene_144, 57568 candidate probes\n",
      "--- 7818 probes passed check_dic selection.\n",
      "finish in 1.182s, 187 probes kept.\n",
      "-- check region:145 2:116235001-116265000_strand_+_gene_145, 58094 candidate probes\n",
      "--- 7994 probes passed check_dic selection.\n",
      "finish in 1.196s, 178 probes kept.\n",
      "-- check region:146 2:118735001-118765000_strand_+_gene_146, 59292 candidate probes\n",
      "--- 18064 probes passed check_dic selection.\n",
      "finish in 1.597s, 294 probes kept.\n",
      "-- check region:147 2:121235001-121265000_strand_+_gene_147, 58632 candidate probes\n",
      "--- 24527 probes passed check_dic selection.\n",
      "finish in 1.859s, 355 probes kept.\n",
      "-- check region:148 2:123735001-123765000_strand_+_gene_148, 57596 candidate probes\n",
      "--- 8024 probes passed check_dic selection.\n",
      "finish in 1.168s, 193 probes kept.\n",
      "-- check region:149 2:126235001-126265000_strand_+_gene_149, 59054 candidate probes\n",
      "--- 18165 probes passed check_dic selection.\n",
      "finish in 1.598s, 309 probes kept.\n",
      "-- check region:150 2:128735001-128765000_strand_+_gene_150, 58502 candidate probes\n",
      "--- 15266 probes passed check_dic selection.\n",
      "finish in 1.489s, 240 probes kept.\n",
      "-- check region:151 2:131235001-131265000_strand_+_gene_151, 58856 candidate probes\n",
      "--- 34 probes passed check_dic selection.\n",
      "finish in 0.925s, 3 probes kept.\n",
      "-- check region:152 2:133735001-133765000_strand_+_gene_152, 58456 candidate probes\n",
      "--- 14344 probes passed check_dic selection.\n",
      "finish in 1.443s, 248 probes kept.\n",
      "-- check region:153 2:136235001-136265000_strand_+_gene_153, 57306 candidate probes\n",
      "--- 19389 probes passed check_dic selection.\n",
      "finish in 1.639s, 307 probes kept.\n",
      "-- check region:154 2:138735001-138765000_strand_+_gene_154, 59390 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 16035 probes passed check_dic selection.\n",
      "finish in 1.518s, 283 probes kept.\n",
      "-- check region:155 2:141235001-141265000_strand_+_gene_155, 59734 candidate probes\n",
      "--- 11963 probes passed check_dic selection.\n",
      "finish in 1.351s, 252 probes kept.\n",
      "-- check region:156 2:143735001-143765000_strand_+_gene_156, 59532 candidate probes\n",
      "--- 17151 probes passed check_dic selection.\n",
      "finish in 1.552s, 292 probes kept.\n",
      "-- check region:157 2:146235001-146265000_strand_+_gene_157, 59104 candidate probes\n",
      "--- 11906 probes passed check_dic selection.\n",
      "finish in 1.345s, 255 probes kept.\n",
      "-- check region:158 2:148735001-148765000_strand_+_gene_158, 59332 candidate probes\n",
      "--- 10875 probes passed check_dic selection.\n",
      "finish in 1.304s, 227 probes kept.\n",
      "-- check region:159 2:151235001-151265000_strand_+_gene_159, 57610 candidate probes\n",
      "--- 12441 probes passed check_dic selection.\n",
      "finish in 1.362s, 231 probes kept.\n",
      "-- check region:160 2:153735001-153765000_strand_+_gene_160, 59614 candidate probes\n",
      "--- 5641 probes passed check_dic selection.\n",
      "finish in 1.124s, 136 probes kept.\n",
      "-- check region:161 2:156235001-156265000_strand_+_gene_161, 50744 candidate probes\n",
      "--- 5130 probes passed check_dic selection.\n",
      "finish in 0.973s, 134 probes kept.\n",
      "-- check region:162 2:158735001-158765000_strand_+_gene_162, 59256 candidate probes\n",
      "--- 22055 probes passed check_dic selection.\n",
      "finish in 1.742s, 338 probes kept.\n",
      "-- check region:163 2:161235001-161265000_strand_+_gene_163, 59112 candidate probes\n",
      "--- 11385 probes passed check_dic selection.\n",
      "finish in 1.331s, 234 probes kept.\n",
      "-- check region:164 2:163735001-163765000_strand_+_gene_164, 59834 candidate probes\n",
      "--- 15486 probes passed check_dic selection.\n",
      "finish in 1.487s, 303 probes kept.\n",
      "-- check region:165 2:166235001-166265000_strand_+_gene_165, 59524 candidate probes\n",
      "--- 10991 probes passed check_dic selection.\n",
      "finish in 1.314s, 244 probes kept.\n",
      "-- check region:166 2:168735001-168765000_strand_+_gene_166, 58398 candidate probes\n",
      "--- 17450 probes passed check_dic selection.\n",
      "finish in 1.552s, 304 probes kept.\n",
      "-- check region:167 2:171235001-171265000_strand_+_gene_167, 57682 candidate probes\n",
      "--- 11079 probes passed check_dic selection.\n",
      "finish in 1.318s, 217 probes kept.\n",
      "-- check region:168 2:173735001-173765000_strand_+_gene_168, 58986 candidate probes\n",
      "--- 18991 probes passed check_dic selection.\n",
      "finish in 1.624s, 301 probes kept.\n",
      "-- check region:169 2:176235001-176265000_strand_+_gene_169, 58332 candidate probes\n",
      "--- 12767 probes passed check_dic selection.\n",
      "finish in 1.375s, 238 probes kept.\n",
      "-- check region:170 2:178735001-178765000_strand_+_gene_170, 59638 candidate probes\n",
      "--- 11977 probes passed check_dic selection.\n",
      "finish in 1.383s, 303 probes kept.\n",
      "-- check region:171 2:181235001-181265000_strand_+_gene_171, 59462 candidate probes\n",
      "--- 14568 probes passed check_dic selection.\n",
      "finish in 1.445s, 272 probes kept.\n",
      "-- check region:172 2:183735001-183765000_strand_+_gene_172, 56990 candidate probes\n",
      "--- 10101 probes passed check_dic selection.\n",
      "finish in 1.240s, 200 probes kept.\n",
      "-- check region:173 2:186235001-186265000_strand_+_gene_173, 55870 candidate probes\n",
      "--- 7584 probes passed check_dic selection.\n",
      "finish in 1.140s, 173 probes kept.\n",
      "-- check region:174 2:188735001-188765000_strand_+_gene_174, 59046 candidate probes\n",
      "--- 8218 probes passed check_dic selection.\n",
      "finish in 1.203s, 198 probes kept.\n",
      "-- check region:175 2:191235001-191265000_strand_+_gene_175, 59416 candidate probes\n",
      "--- 18199 probes passed check_dic selection.\n",
      "finish in 1.595s, 317 probes kept.\n",
      "-- check region:176 2:193735001-193765000_strand_+_gene_176, 59382 candidate probes\n",
      "--- 5916 probes passed check_dic selection.\n",
      "finish in 1.124s, 152 probes kept.\n",
      "-- check region:177 2:196235001-196265000_strand_+_gene_177, 58500 candidate probes\n",
      "--- 14625 probes passed check_dic selection.\n",
      "finish in 1.453s, 279 probes kept.\n",
      "-- check region:178 2:198735001-198765000_strand_+_gene_178, 59618 candidate probes\n",
      "--- 15309 probes passed check_dic selection.\n",
      "finish in 1.479s, 280 probes kept.\n",
      "-- check region:179 2:201235001-201265000_strand_+_gene_179, 56916 candidate probes\n",
      "--- 12584 probes passed check_dic selection.\n",
      "finish in 1.350s, 230 probes kept.\n",
      "-- check region:180 2:203735001-203765000_strand_+_gene_180, 57976 candidate probes\n",
      "--- 11272 probes passed check_dic selection.\n",
      "finish in 1.335s, 226 probes kept.\n",
      "-- check region:181 2:206235001-206265000_strand_+_gene_181, 56772 candidate probes\n",
      "--- 14550 probes passed check_dic selection.\n",
      "finish in 1.427s, 248 probes kept.\n",
      "-- check region:182 2:208735001-208765000_strand_+_gene_182, 47466 candidate probes\n",
      "--- 4136 probes passed check_dic selection.\n",
      "finish in 0.889s, 93 probes kept.\n",
      "-- check region:183 2:211235001-211265000_strand_+_gene_183, 55966 candidate probes\n",
      "--- 10636 probes passed check_dic selection.\n",
      "finish in 1.262s, 204 probes kept.\n",
      "-- check region:184 2:213735001-213765000_strand_+_gene_184, 57980 candidate probes\n",
      "--- 8299 probes passed check_dic selection.\n",
      "finish in 1.191s, 173 probes kept.\n",
      "-- check region:185 2:216235001-216265000_strand_+_gene_185, 58664 candidate probes\n",
      "--- 19004 probes passed check_dic selection.\n",
      "finish in 1.634s, 313 probes kept.\n",
      "-- check region:186 2:218735001-218765000_strand_+_gene_186, 58428 candidate probes\n",
      "--- 16735 probes passed check_dic selection.\n",
      "finish in 1.558s, 301 probes kept.\n",
      "-- check region:187 2:221235001-221265000_strand_+_gene_187, 58854 candidate probes\n",
      "--- 12326 probes passed check_dic selection.\n",
      "finish in 1.361s, 239 probes kept.\n",
      "-- check region:188 2:223735001-223765000_strand_+_gene_188, 57874 candidate probes\n",
      "--- 13246 probes passed check_dic selection.\n",
      "finish in 1.400s, 233 probes kept.\n",
      "-- check region:189 2:226235001-226265000_strand_+_gene_189, 57792 candidate probes\n",
      "--- 8688 probes passed check_dic selection.\n",
      "finish in 1.204s, 191 probes kept.\n",
      "-- check region:190 2:228735001-228765000_strand_+_gene_190, 47512 candidate probes\n",
      "--- 7527 probes passed check_dic selection.\n",
      "finish in 1.018s, 150 probes kept.\n",
      "-- check region:191 2:231235001-231265000_strand_+_gene_191, 56606 candidate probes\n",
      "--- 17496 probes passed check_dic selection.\n",
      "finish in 1.536s, 274 probes kept.\n",
      "-- check region:192 2:233735001-233765000_strand_+_gene_192, 56054 candidate probes\n",
      "--- 14438 probes passed check_dic selection.\n",
      "finish in 1.428s, 270 probes kept.\n",
      "-- check region:193 2:236235001-236265000_strand_+_gene_193, 59022 candidate probes\n",
      "--- 22615 probes passed check_dic selection.\n",
      "finish in 1.775s, 357 probes kept.\n",
      "-- check region:194 2:238735001-238765000_strand_+_gene_194, 58294 candidate probes\n",
      "--- 18206 probes passed check_dic selection.\n",
      "finish in 1.592s, 297 probes kept.\n",
      "-- check region:195 3:1235001-1265000_strand_+_gene_195, 57398 candidate probes\n",
      "--- 10304 probes passed check_dic selection.\n",
      "finish in 1.268s, 199 probes kept.\n",
      "-- check region:196 3:3735001-3765000_strand_+_gene_196, 59590 candidate probes\n",
      "--- 18284 probes passed check_dic selection.\n",
      "finish in 1.616s, 317 probes kept.\n",
      "-- check region:197 3:6235001-6265000_strand_+_gene_197, 59196 candidate probes\n",
      "--- 12831 probes passed check_dic selection.\n",
      "finish in 1.394s, 251 probes kept.\n",
      "-- check region:198 3:8735001-8765000_strand_+_gene_198, 59624 candidate probes\n",
      "--- 24162 probes passed check_dic selection.\n",
      "finish in 1.846s, 379 probes kept.\n",
      "-- check region:199 3:11235001-11265000_strand_+_gene_199, 58214 candidate probes\n",
      "--- 12341 probes passed check_dic selection.\n",
      "finish in 1.388s, 275 probes kept.\n",
      "-- check region:200 3:13735001-13765000_strand_+_gene_200, 58840 candidate probes\n",
      "--- 14436 probes passed check_dic selection.\n",
      "finish in 1.458s, 229 probes kept.\n",
      "-- check region:201 3:16235001-16265000_strand_+_gene_201, 59164 candidate probes\n",
      "--- 22549 probes passed check_dic selection.\n",
      "finish in 1.782s, 361 probes kept.\n",
      "-- check region:202 3:18735001-18765000_strand_+_gene_202, 59750 candidate probes\n",
      "--- 16468 probes passed check_dic selection.\n",
      "finish in 1.543s, 331 probes kept.\n",
      "-- check region:203 3:21235001-21265000_strand_+_gene_203, 50848 candidate probes\n",
      "--- 9906 probes passed check_dic selection.\n",
      "finish in 1.145s, 190 probes kept.\n",
      "-- check region:204 3:23735001-23765000_strand_+_gene_204, 58004 candidate probes\n",
      "--- 17337 probes passed check_dic selection.\n",
      "finish in 1.553s, 265 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:205 3:26235001-26265000_strand_+_gene_205, 53482 candidate probes\n",
      "--- 12872 probes passed check_dic selection.\n",
      "finish in 1.297s, 241 probes kept.\n",
      "-- check region:206 3:28735001-28765000_strand_+_gene_206, 59384 candidate probes\n",
      "--- 14205 probes passed check_dic selection.\n",
      "finish in 1.451s, 262 probes kept.\n",
      "-- check region:207 3:31235001-31265000_strand_+_gene_207, 59732 candidate probes\n",
      "--- 18090 probes passed check_dic selection.\n",
      "finish in 1.596s, 324 probes kept.\n",
      "-- check region:208 3:33735001-33765000_strand_+_gene_208, 56604 candidate probes\n",
      "--- 8194 probes passed check_dic selection.\n",
      "finish in 1.188s, 174 probes kept.\n",
      "-- check region:209 3:36235001-36265000_strand_+_gene_209, 55888 candidate probes\n",
      "--- 8315 probes passed check_dic selection.\n",
      "finish in 1.174s, 180 probes kept.\n",
      "-- check region:210 3:38735001-38765000_strand_+_gene_210, 59248 candidate probes\n",
      "--- 23150 probes passed check_dic selection.\n",
      "finish in 1.814s, 365 probes kept.\n",
      "-- check region:211 3:41235001-41265000_strand_+_gene_211, 59596 candidate probes\n",
      "--- 21119 probes passed check_dic selection.\n",
      "finish in 1.730s, 349 probes kept.\n",
      "-- check region:212 3:43735001-43765000_strand_+_gene_212, 58990 candidate probes\n",
      "--- 21172 probes passed check_dic selection.\n",
      "finish in 1.707s, 330 probes kept.\n",
      "-- check region:213 3:46235001-46265000_strand_+_gene_213, 56570 candidate probes\n",
      "--- 10030 probes passed check_dic selection.\n",
      "finish in 1.265s, 200 probes kept.\n",
      "-- check region:214 3:48735001-48765000_strand_+_gene_214, 56314 candidate probes\n",
      "--- 11716 probes passed check_dic selection.\n",
      "finish in 1.336s, 219 probes kept.\n",
      "-- check region:215 3:51235001-51265000_strand_+_gene_215, 52006 candidate probes\n",
      "--- 7476 probes passed check_dic selection.\n",
      "finish in 1.088s, 142 probes kept.\n",
      "-- check region:216 3:53735001-53765000_strand_+_gene_216, 59534 candidate probes\n",
      "--- 29186 probes passed check_dic selection.\n",
      "finish in 2.036s, 408 probes kept.\n",
      "-- check region:217 3:56235001-56265000_strand_+_gene_217, 58812 candidate probes\n",
      "--- 15237 probes passed check_dic selection.\n",
      "finish in 1.480s, 278 probes kept.\n",
      "-- check region:218 3:58735001-58765000_strand_+_gene_218, 53942 candidate probes\n",
      "--- 14116 probes passed check_dic selection.\n",
      "finish in 1.369s, 252 probes kept.\n",
      "-- check region:219 3:61235001-61265000_strand_+_gene_219, 57882 candidate probes\n",
      "--- 14729 probes passed check_dic selection.\n",
      "finish in 1.445s, 258 probes kept.\n",
      "-- check region:220 3:63735001-63765000_strand_+_gene_220, 52330 candidate probes\n",
      "--- 9463 probes passed check_dic selection.\n",
      "finish in 1.160s, 214 probes kept.\n",
      "-- check region:221 3:66235001-66265000_strand_+_gene_221, 58814 candidate probes\n",
      "--- 14483 probes passed check_dic selection.\n",
      "finish in 1.443s, 256 probes kept.\n",
      "-- check region:222 3:68735001-68765000_strand_+_gene_222, 59486 candidate probes\n",
      "--- 16427 probes passed check_dic selection.\n",
      "finish in 1.535s, 292 probes kept.\n",
      "-- check region:223 3:71235001-71265000_strand_+_gene_223, 58364 candidate probes\n",
      "--- 20247 probes passed check_dic selection.\n",
      "finish in 1.671s, 318 probes kept.\n",
      "-- check region:224 3:73735001-73765000_strand_+_gene_224, 59628 candidate probes\n",
      "--- 20134 probes passed check_dic selection.\n",
      "finish in 1.681s, 322 probes kept.\n",
      "-- check region:225 3:76235001-76265000_strand_+_gene_225, 57928 candidate probes\n",
      "--- 11379 probes passed check_dic selection.\n",
      "finish in 1.304s, 234 probes kept.\n",
      "-- check region:226 3:78735001-78765000_strand_+_gene_226, 59290 candidate probes\n",
      "--- 10209 probes passed check_dic selection.\n",
      "finish in 1.274s, 220 probes kept.\n",
      "-- check region:227 3:81235001-81265000_strand_+_gene_227, 59290 candidate probes\n",
      "--- 12039 probes passed check_dic selection.\n",
      "finish in 1.348s, 242 probes kept.\n",
      "-- check region:228 3:83735001-83765000_strand_+_gene_228, 57548 candidate probes\n",
      "--- 6074 probes passed check_dic selection.\n",
      "finish in 1.104s, 148 probes kept.\n",
      "-- check region:229 3:86235001-86265000_strand_+_gene_229, 58872 candidate probes\n",
      "--- 8771 probes passed check_dic selection.\n",
      "finish in 1.214s, 197 probes kept.\n",
      "-- check region:230 3:88735001-88765000_strand_+_gene_230, 56192 candidate probes\n",
      "--- 8239 probes passed check_dic selection.\n",
      "finish in 1.183s, 166 probes kept.\n",
      "-- check region:231 3:91235001-91265000_strand_+_gene_231, 18710 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.296s, 0 probes kept.\n",
      "-- check region:232 3:93735001-93765000_strand_+_gene_232, 53752 candidate probes\n",
      "--- 294 probes passed check_dic selection.\n",
      "finish in 0.856s, 23 probes kept.\n",
      "-- check region:233 3:96235001-96265000_strand_+_gene_233, 59240 candidate probes\n",
      "--- 9866 probes passed check_dic selection.\n",
      "finish in 1.300s, 196 probes kept.\n",
      "-- check region:234 3:98735001-98765000_strand_+_gene_234, 58976 candidate probes\n",
      "--- 12682 probes passed check_dic selection.\n",
      "finish in 1.389s, 255 probes kept.\n",
      "-- check region:235 3:101235001-101265000_strand_+_gene_235, 58610 candidate probes\n",
      "--- 11582 probes passed check_dic selection.\n",
      "finish in 1.348s, 233 probes kept.\n",
      "-- check region:236 3:103735001-103765000_strand_+_gene_236, 57596 candidate probes\n",
      "--- 6626 probes passed check_dic selection.\n",
      "finish in 1.130s, 182 probes kept.\n",
      "-- check region:237 3:106235001-106265000_strand_+_gene_237, 58878 candidate probes\n",
      "--- 7412 probes passed check_dic selection.\n",
      "finish in 1.183s, 179 probes kept.\n",
      "-- check region:238 3:108735001-108765000_strand_+_gene_238, 48358 candidate probes\n",
      "--- 15053 probes passed check_dic selection.\n",
      "finish in 1.328s, 251 probes kept.\n",
      "-- check region:239 3:111235001-111265000_strand_+_gene_239, 57338 candidate probes\n",
      "--- 11428 probes passed check_dic selection.\n",
      "finish in 1.311s, 231 probes kept.\n",
      "-- check region:240 3:113735001-113765000_strand_+_gene_240, 58244 candidate probes\n",
      "--- 12043 probes passed check_dic selection.\n",
      "finish in 1.338s, 234 probes kept.\n",
      "-- check region:241 3:116235001-116265000_strand_+_gene_241, 59358 candidate probes\n",
      "--- 14479 probes passed check_dic selection.\n",
      "finish in 1.443s, 272 probes kept.\n",
      "-- check region:242 3:118735001-118765000_strand_+_gene_242, 58754 candidate probes\n",
      "--- 16899 probes passed check_dic selection.\n",
      "finish in 1.535s, 288 probes kept.\n",
      "-- check region:243 3:121235001-121265000_strand_+_gene_243, 58780 candidate probes\n",
      "--- 6093 probes passed check_dic selection.\n",
      "finish in 1.132s, 165 probes kept.\n",
      "-- check region:244 3:123735001-123765000_strand_+_gene_244, 59342 candidate probes\n",
      "--- 19997 probes passed check_dic selection.\n",
      "finish in 1.677s, 333 probes kept.\n",
      "-- check region:245 3:126235001-126265000_strand_+_gene_245, 59608 candidate probes\n",
      "--- 13194 probes passed check_dic selection.\n",
      "finish in 1.429s, 238 probes kept.\n",
      "-- check region:246 3:128735001-128765000_strand_+_gene_246, 56974 candidate probes\n",
      "--- 10794 probes passed check_dic selection.\n",
      "finish in 1.304s, 205 probes kept.\n",
      "-- check region:247 3:131235001-131265000_strand_+_gene_247, 57664 candidate probes\n",
      "--- 13218 probes passed check_dic selection.\n",
      "finish in 1.385s, 234 probes kept.\n",
      "-- check region:248 3:133735001-133765000_strand_+_gene_248, 56368 candidate probes\n",
      "--- 16119 probes passed check_dic selection.\n",
      "finish in 1.502s, 277 probes kept.\n",
      "-- check region:249 3:136235001-136265000_strand_+_gene_249, 57468 candidate probes\n",
      "--- 13977 probes passed check_dic selection.\n",
      "finish in 1.426s, 242 probes kept.\n",
      "-- check region:250 3:138735001-138765000_strand_+_gene_250, 57170 candidate probes\n",
      "--- 6813 probes passed check_dic selection.\n",
      "finish in 1.146s, 172 probes kept.\n",
      "-- check region:251 3:141235001-141265000_strand_+_gene_251, 59574 candidate probes\n",
      "--- 24158 probes passed check_dic selection.\n",
      "finish in 1.838s, 357 probes kept.\n",
      "-- check region:252 3:143735001-143765000_strand_+_gene_252, 57258 candidate probes\n",
      "--- 16249 probes passed check_dic selection.\n",
      "finish in 1.492s, 287 probes kept.\n",
      "-- check region:253 3:146235001-146265000_strand_+_gene_253, 59396 candidate probes\n",
      "--- 7039 probes passed check_dic selection.\n",
      "finish in 1.174s, 188 probes kept.\n",
      "-- check region:254 3:148735001-148765000_strand_+_gene_254, 58592 candidate probes\n",
      "--- 15835 probes passed check_dic selection.\n",
      "finish in 1.503s, 282 probes kept.\n",
      "-- check region:255 3:151235001-151265000_strand_+_gene_255, 51454 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 12065 probes passed check_dic selection.\n",
      "finish in 1.245s, 225 probes kept.\n",
      "-- check region:256 3:153735001-153765000_strand_+_gene_256, 58366 candidate probes\n",
      "--- 11123 probes passed check_dic selection.\n",
      "finish in 1.316s, 218 probes kept.\n",
      "-- check region:257 3:156235001-156265000_strand_+_gene_257, 59370 candidate probes\n",
      "--- 17780 probes passed check_dic selection.\n",
      "finish in 1.580s, 288 probes kept.\n",
      "-- check region:258 3:158735001-158765000_strand_+_gene_258, 57960 candidate probes\n",
      "--- 17467 probes passed check_dic selection.\n",
      "finish in 1.574s, 314 probes kept.\n",
      "-- check region:259 3:161235001-161265000_strand_+_gene_259, 59342 candidate probes\n",
      "--- 6225 probes passed check_dic selection.\n",
      "finish in 1.142s, 160 probes kept.\n",
      "-- check region:260 3:163735001-163765000_strand_+_gene_260, 56448 candidate probes\n",
      "--- 7752 probes passed check_dic selection.\n",
      "finish in 1.147s, 162 probes kept.\n",
      "-- check region:261 3:166235001-166265000_strand_+_gene_261, 59188 candidate probes\n",
      "--- 6472 probes passed check_dic selection.\n",
      "finish in 1.140s, 159 probes kept.\n",
      "-- check region:262 3:168735001-168765000_strand_+_gene_262, 55842 candidate probes\n",
      "--- 8709 probes passed check_dic selection.\n",
      "finish in 1.179s, 179 probes kept.\n",
      "-- check region:263 3:171235001-171265000_strand_+_gene_263, 58874 candidate probes\n",
      "--- 18708 probes passed check_dic selection.\n",
      "finish in 1.604s, 324 probes kept.\n",
      "-- check region:264 3:173735001-173765000_strand_+_gene_264, 59812 candidate probes\n",
      "--- 12823 probes passed check_dic selection.\n",
      "finish in 1.391s, 259 probes kept.\n",
      "-- check region:265 3:176235001-176265000_strand_+_gene_265, 59200 candidate probes\n",
      "--- 9727 probes passed check_dic selection.\n",
      "finish in 1.268s, 220 probes kept.\n",
      "-- check region:266 3:178735001-178765000_strand_+_gene_266, 57116 candidate probes\n",
      "--- 13646 probes passed check_dic selection.\n",
      "finish in 1.392s, 251 probes kept.\n",
      "-- check region:267 3:181235001-181265000_strand_+_gene_267, 58802 candidate probes\n",
      "--- 13988 probes passed check_dic selection.\n",
      "finish in 1.424s, 263 probes kept.\n",
      "-- check region:268 3:183735001-183765000_strand_+_gene_268, 54334 candidate probes\n",
      "--- 9242 probes passed check_dic selection.\n",
      "finish in 1.193s, 175 probes kept.\n",
      "-- check region:269 3:186235001-186265000_strand_+_gene_269, 58764 candidate probes\n",
      "--- 24235 probes passed check_dic selection.\n",
      "finish in 1.828s, 351 probes kept.\n",
      "-- check region:270 3:188735001-188765000_strand_+_gene_270, 59176 candidate probes\n",
      "--- 17886 probes passed check_dic selection.\n",
      "finish in 1.572s, 313 probes kept.\n",
      "-- check region:271 3:191235001-191265000_strand_+_gene_271, 59214 candidate probes\n",
      "--- 9298 probes passed check_dic selection.\n",
      "finish in 1.256s, 220 probes kept.\n",
      "-- check region:272 3:193735001-193765000_strand_+_gene_272, 58370 candidate probes\n",
      "--- 12730 probes passed check_dic selection.\n",
      "finish in 1.382s, 232 probes kept.\n",
      "-- check region:273 3:196235001-196265000_strand_+_gene_273, 55152 candidate probes\n",
      "--- 11823 probes passed check_dic selection.\n",
      "finish in 1.327s, 231 probes kept.\n",
      "-- check region:274 4:1235001-1265000_strand_+_gene_274, 58124 candidate probes\n",
      "--- 20310 probes passed check_dic selection.\n",
      "finish in 1.694s, 330 probes kept.\n",
      "-- check region:275 4:3735001-3765000_strand_+_gene_275, 59168 candidate probes\n",
      "--- 25816 probes passed check_dic selection.\n",
      "finish in 1.918s, 390 probes kept.\n",
      "-- check region:276 4:6235001-6265000_strand_+_gene_276, 58618 candidate probes\n",
      "--- 21027 probes passed check_dic selection.\n",
      "finish in 1.721s, 325 probes kept.\n",
      "-- check region:277 4:8735001-8765000_strand_+_gene_277, 59046 candidate probes\n",
      "--- 20298 probes passed check_dic selection.\n",
      "finish in 1.696s, 313 probes kept.\n",
      "-- check region:278 4:11235001-11265000_strand_+_gene_278, 51360 candidate probes\n",
      "--- 11205 probes passed check_dic selection.\n",
      "finish in 1.215s, 224 probes kept.\n",
      "-- check region:279 4:13735001-13765000_strand_+_gene_279, 57268 candidate probes\n",
      "--- 11460 probes passed check_dic selection.\n",
      "finish in 1.314s, 210 probes kept.\n",
      "-- check region:280 4:16235001-16265000_strand_+_gene_280, 59292 candidate probes\n",
      "--- 16471 probes passed check_dic selection.\n",
      "finish in 1.539s, 309 probes kept.\n",
      "-- check region:281 4:18735001-18765000_strand_+_gene_281, 59206 candidate probes\n",
      "--- 12124 probes passed check_dic selection.\n",
      "finish in 1.359s, 239 probes kept.\n",
      "-- check region:282 4:21235001-21265000_strand_+_gene_282, 55818 candidate probes\n",
      "--- 14508 probes passed check_dic selection.\n",
      "finish in 1.400s, 262 probes kept.\n",
      "-- check region:283 4:23735001-23765000_strand_+_gene_283, 59586 candidate probes\n",
      "--- 16401 probes passed check_dic selection.\n",
      "finish in 1.537s, 293 probes kept.\n",
      "-- check region:284 4:26235001-26265000_strand_+_gene_284, 57248 candidate probes\n",
      "--- 10912 probes passed check_dic selection.\n",
      "finish in 1.300s, 215 probes kept.\n",
      "-- check region:285 4:28735001-28765000_strand_+_gene_285, 58894 candidate probes\n",
      "--- 10900 probes passed check_dic selection.\n",
      "finish in 1.297s, 234 probes kept.\n",
      "-- check region:286 4:31235001-31265000_strand_+_gene_286, 59308 candidate probes\n",
      "--- 8805 probes passed check_dic selection.\n",
      "finish in 1.247s, 187 probes kept.\n",
      "-- check region:287 4:33735001-33765000_strand_+_gene_287, 58596 candidate probes\n",
      "--- 8397 probes passed check_dic selection.\n",
      "finish in 1.212s, 194 probes kept.\n",
      "-- check region:288 4:36235001-36265000_strand_+_gene_288, 59066 candidate probes\n",
      "--- 12385 probes passed check_dic selection.\n",
      "finish in 1.365s, 249 probes kept.\n",
      "-- check region:289 4:38735001-38765000_strand_+_gene_289, 58170 candidate probes\n",
      "--- 15084 probes passed check_dic selection.\n",
      "finish in 1.529s, 250 probes kept.\n",
      "-- check region:290 4:41235001-41265000_strand_+_gene_290, 57280 candidate probes\n",
      "--- 12929 probes passed check_dic selection.\n",
      "finish in 1.370s, 253 probes kept.\n",
      "-- check region:291 4:43735001-43765000_strand_+_gene_291, 56116 candidate probes\n",
      "--- 8611 probes passed check_dic selection.\n",
      "finish in 1.212s, 195 probes kept.\n",
      "-- check region:292 4:46235001-46265000_strand_+_gene_292, 58820 candidate probes\n",
      "--- 8920 probes passed check_dic selection.\n",
      "finish in 1.226s, 226 probes kept.\n",
      "-- check region:293 4:48735001-48765000_strand_+_gene_293, 54528 candidate probes\n",
      "--- 7654 probes passed check_dic selection.\n",
      "finish in 1.136s, 157 probes kept.\n",
      "-- check region:294 4:51235001-51265000_strand_+_gene_294, 19210 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.304s, 0 probes kept.\n",
      "-- check region:295 4:53735001-53765000_strand_+_gene_295, 55208 candidate probes\n",
      "--- 11082 probes passed check_dic selection.\n",
      "finish in 1.276s, 209 probes kept.\n",
      "-- check region:296 4:56235001-56265000_strand_+_gene_296, 57570 candidate probes\n",
      "--- 16329 probes passed check_dic selection.\n",
      "finish in 1.511s, 274 probes kept.\n",
      "-- check region:297 4:58735001-58765000_strand_+_gene_297, 58324 candidate probes\n",
      "--- 8317 probes passed check_dic selection.\n",
      "finish in 1.196s, 194 probes kept.\n",
      "-- check region:298 4:61235001-61265000_strand_+_gene_298, 59286 candidate probes\n",
      "--- 13121 probes passed check_dic selection.\n",
      "finish in 1.388s, 253 probes kept.\n",
      "-- check region:299 4:63735001-63765000_strand_+_gene_299, 57076 candidate probes\n",
      "--- 4696 probes passed check_dic selection.\n",
      "finish in 1.038s, 141 probes kept.\n",
      "-- check region:300 4:66235001-66265000_strand_+_gene_300, 52848 candidate probes\n",
      "--- 6661 probes passed check_dic selection.\n",
      "finish in 1.069s, 151 probes kept.\n",
      "-- check region:301 4:68735001-68765000_strand_+_gene_301, 59446 candidate probes\n",
      "--- 550 probes passed check_dic selection.\n",
      "finish in 0.961s, 25 probes kept.\n",
      "-- check region:302 4:71235001-71265000_strand_+_gene_302, 59776 candidate probes\n",
      "--- 17706 probes passed check_dic selection.\n",
      "finish in 1.575s, 328 probes kept.\n",
      "-- check region:303 4:73735001-73765000_strand_+_gene_303, 57074 candidate probes\n",
      "--- 10599 probes passed check_dic selection.\n",
      "finish in 1.267s, 213 probes kept.\n",
      "-- check region:304 4:76235001-76265000_strand_+_gene_304, 58618 candidate probes\n",
      "--- 12953 probes passed check_dic selection.\n",
      "finish in 1.394s, 232 probes kept.\n",
      "-- check region:305 4:78735001-78765000_strand_+_gene_305, 57434 candidate probes\n",
      "--- 11524 probes passed check_dic selection.\n",
      "finish in 1.310s, 222 probes kept.\n",
      "-- check region:306 4:81235001-81265000_strand_+_gene_306, 58416 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 12242 probes passed check_dic selection.\n",
      "finish in 1.355s, 242 probes kept.\n",
      "-- check region:307 4:83735001-83765000_strand_+_gene_307, 59474 candidate probes\n",
      "--- 11535 probes passed check_dic selection.\n",
      "finish in 1.342s, 231 probes kept.\n",
      "-- check region:308 4:86235001-86265000_strand_+_gene_308, 52754 candidate probes\n",
      "--- 10618 probes passed check_dic selection.\n",
      "finish in 1.207s, 203 probes kept.\n",
      "-- check region:309 4:88735001-88765000_strand_+_gene_309, 58880 candidate probes\n",
      "--- 18640 probes passed check_dic selection.\n",
      "finish in 1.613s, 343 probes kept.\n",
      "-- check region:310 4:91235001-91265000_strand_+_gene_310, 55942 candidate probes\n",
      "--- 6910 probes passed check_dic selection.\n",
      "finish in 1.112s, 166 probes kept.\n",
      "-- check region:311 4:93735001-93765000_strand_+_gene_311, 59373 candidate probes\n",
      "--- 13572 probes passed check_dic selection.\n",
      "finish in 1.411s, 267 probes kept.\n",
      "-- check region:312 4:96235001-96265000_strand_+_gene_312, 59822 candidate probes\n",
      "--- 11560 probes passed check_dic selection.\n",
      "finish in 1.344s, 241 probes kept.\n",
      "-- check region:313 4:98735001-98765000_strand_+_gene_313, 54682 candidate probes\n",
      "--- 9726 probes passed check_dic selection.\n",
      "finish in 1.212s, 182 probes kept.\n",
      "-- check region:314 4:101235001-101265000_strand_+_gene_314, 59568 candidate probes\n",
      "--- 12113 probes passed check_dic selection.\n",
      "finish in 1.351s, 253 probes kept.\n",
      "-- check region:315 4:103735001-103765000_strand_+_gene_315, 59362 candidate probes\n",
      "--- 9957 probes passed check_dic selection.\n",
      "finish in 1.278s, 228 probes kept.\n",
      "-- check region:316 4:106235001-106265000_strand_+_gene_316, 59386 candidate probes\n",
      "--- 7965 probes passed check_dic selection.\n",
      "finish in 1.200s, 183 probes kept.\n",
      "-- check region:317 4:108735001-108765000_strand_+_gene_317, 58312 candidate probes\n",
      "--- 13293 probes passed check_dic selection.\n",
      "finish in 1.406s, 251 probes kept.\n",
      "-- check region:318 4:111235001-111265000_strand_+_gene_318, 54424 candidate probes\n",
      "--- 8236 probes passed check_dic selection.\n",
      "finish in 1.141s, 178 probes kept.\n",
      "-- check region:319 4:113735001-113765000_strand_+_gene_319, 59692 candidate probes\n",
      "--- 11581 probes passed check_dic selection.\n",
      "finish in 1.335s, 249 probes kept.\n",
      "-- check region:320 4:116235001-116265000_strand_+_gene_320, 55300 candidate probes\n",
      "--- 6259 probes passed check_dic selection.\n",
      "finish in 1.099s, 155 probes kept.\n",
      "-- check region:321 4:118735001-118765000_strand_+_gene_321, 59562 candidate probes\n",
      "--- 13916 probes passed check_dic selection.\n",
      "finish in 1.436s, 279 probes kept.\n",
      "-- check region:322 4:121235001-121265000_strand_+_gene_322, 57988 candidate probes\n",
      "--- 14672 probes passed check_dic selection.\n",
      "finish in 1.442s, 258 probes kept.\n",
      "-- check region:323 4:123735001-123765000_strand_+_gene_323, 59171 candidate probes\n",
      "--- 13605 probes passed check_dic selection.\n",
      "finish in 1.416s, 248 probes kept.\n",
      "-- check region:324 4:126235001-126265000_strand_+_gene_324, 59134 candidate probes\n",
      "--- 7381 probes passed check_dic selection.\n",
      "finish in 1.174s, 171 probes kept.\n",
      "-- check region:325 4:128735001-128765000_strand_+_gene_325, 57776 candidate probes\n",
      "--- 8482 probes passed check_dic selection.\n",
      "finish in 1.218s, 176 probes kept.\n",
      "-- check region:326 4:131235001-131265000_strand_+_gene_326, 59414 candidate probes\n",
      "--- 8441 probes passed check_dic selection.\n",
      "finish in 1.210s, 191 probes kept.\n",
      "-- check region:327 4:133735001-133765000_strand_+_gene_327, 53616 candidate probes\n",
      "--- 4203 probes passed check_dic selection.\n",
      "finish in 0.991s, 106 probes kept.\n",
      "-- check region:328 4:136235001-136265000_strand_+_gene_328, 58890 candidate probes\n",
      "--- 6531 probes passed check_dic selection.\n",
      "finish in 1.153s, 170 probes kept.\n",
      "-- check region:329 4:138735001-138765000_strand_+_gene_329, 58016 candidate probes\n",
      "--- 17038 probes passed check_dic selection.\n",
      "finish in 1.538s, 295 probes kept.\n",
      "-- check region:330 4:141235001-141265000_strand_+_gene_330, 59480 candidate probes\n",
      "--- 10940 probes passed check_dic selection.\n",
      "finish in 1.332s, 218 probes kept.\n",
      "-- check region:331 4:143735001-143765000_strand_+_gene_331, 58548 candidate probes\n",
      "--- 5834 probes passed check_dic selection.\n",
      "finish in 1.119s, 154 probes kept.\n",
      "-- check region:332 4:146235001-146265000_strand_+_gene_332, 59396 candidate probes\n",
      "--- 22539 probes passed check_dic selection.\n",
      "finish in 1.831s, 377 probes kept.\n",
      "-- check region:333 4:148735001-148765000_strand_+_gene_333, 57418 candidate probes\n",
      "--- 11058 probes passed check_dic selection.\n",
      "finish in 1.299s, 214 probes kept.\n",
      "-- check region:334 4:151235001-151265000_strand_+_gene_334, 57116 candidate probes\n",
      "--- 11642 probes passed check_dic selection.\n",
      "finish in 1.314s, 234 probes kept.\n",
      "-- check region:335 4:153735001-153765000_strand_+_gene_335, 58926 candidate probes\n",
      "--- 15490 probes passed check_dic selection.\n",
      "finish in 1.493s, 271 probes kept.\n",
      "-- check region:336 4:156235001-156265000_strand_+_gene_336, 59654 candidate probes\n",
      "--- 11363 probes passed check_dic selection.\n",
      "finish in 1.336s, 244 probes kept.\n",
      "-- check region:337 4:158735001-158765000_strand_+_gene_337, 56642 candidate probes\n",
      "--- 8361 probes passed check_dic selection.\n",
      "finish in 1.203s, 181 probes kept.\n",
      "-- check region:338 4:161235001-161265000_strand_+_gene_338, 59884 candidate probes\n",
      "--- 7976 probes passed check_dic selection.\n",
      "finish in 1.217s, 222 probes kept.\n",
      "-- check region:339 4:163735001-163765000_strand_+_gene_339, 57884 candidate probes\n",
      "--- 12569 probes passed check_dic selection.\n",
      "finish in 1.352s, 257 probes kept.\n",
      "-- check region:340 4:166235001-166265000_strand_+_gene_340, 55378 candidate probes\n",
      "--- 10964 probes passed check_dic selection.\n",
      "finish in 1.245s, 201 probes kept.\n",
      "-- check region:341 4:168735001-168765000_strand_+_gene_341, 58206 candidate probes\n",
      "--- 15769 probes passed check_dic selection.\n",
      "finish in 1.483s, 284 probes kept.\n",
      "-- check region:342 4:171235001-171265000_strand_+_gene_342, 58712 candidate probes\n",
      "--- 5718 probes passed check_dic selection.\n",
      "finish in 1.112s, 157 probes kept.\n",
      "-- check region:343 4:173735001-173765000_strand_+_gene_343, 57288 candidate probes\n",
      "--- 14086 probes passed check_dic selection.\n",
      "finish in 1.392s, 260 probes kept.\n",
      "-- check region:344 4:176235001-176265000_strand_+_gene_344, 58226 candidate probes\n",
      "--- 11960 probes passed check_dic selection.\n",
      "finish in 1.338s, 234 probes kept.\n",
      "-- check region:345 4:178735001-178765000_strand_+_gene_345, 59584 candidate probes\n",
      "--- 10358 probes passed check_dic selection.\n",
      "finish in 1.292s, 225 probes kept.\n",
      "-- check region:346 4:181235001-181265000_strand_+_gene_346, 54286 candidate probes\n",
      "--- 16090 probes passed check_dic selection.\n",
      "finish in 1.439s, 279 probes kept.\n",
      "-- check region:347 4:183735001-183765000_strand_+_gene_347, 56876 candidate probes\n",
      "--- 11453 probes passed check_dic selection.\n",
      "finish in 1.300s, 202 probes kept.\n",
      "-- check region:348 4:186235001-186265000_strand_+_gene_348, 56416 candidate probes\n",
      "--- 11895 probes passed check_dic selection.\n",
      "finish in 1.334s, 223 probes kept.\n",
      "-- check region:349 4:188735001-188765000_strand_+_gene_349, 58986 candidate probes\n",
      "--- 7998 probes passed check_dic selection.\n",
      "finish in 1.208s, 161 probes kept.\n",
      "-- check region:350 5:1235001-1265000_strand_+_gene_350, 58844 candidate probes\n",
      "--- 28944 probes passed check_dic selection.\n",
      "finish in 2.026s, 415 probes kept.\n",
      "-- check region:351 5:3735001-3765000_strand_+_gene_351, 58624 candidate probes\n",
      "--- 21753 probes passed check_dic selection.\n",
      "finish in 1.730s, 319 probes kept.\n",
      "-- check region:352 5:6235001-6265000_strand_+_gene_352, 59232 candidate probes\n",
      "--- 19360 probes passed check_dic selection.\n",
      "finish in 1.632s, 325 probes kept.\n",
      "-- check region:353 5:8735001-8765000_strand_+_gene_353, 54702 candidate probes\n",
      "--- 7310 probes passed check_dic selection.\n",
      "finish in 1.129s, 158 probes kept.\n",
      "-- check region:354 5:11235001-11265000_strand_+_gene_354, 58940 candidate probes\n",
      "--- 17777 probes passed check_dic selection.\n",
      "finish in 1.575s, 293 probes kept.\n",
      "-- check region:355 5:13735001-13765000_strand_+_gene_355, 56876 candidate probes\n",
      "--- 12344 probes passed check_dic selection.\n",
      "finish in 1.340s, 234 probes kept.\n",
      "-- check region:356 5:16235001-16265000_strand_+_gene_356, 59652 candidate probes\n",
      "--- 13984 probes passed check_dic selection.\n",
      "finish in 1.437s, 257 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:357 5:18735001-18765000_strand_+_gene_357, 59204 candidate probes\n",
      "--- 7013 probes passed check_dic selection.\n",
      "finish in 1.180s, 170 probes kept.\n",
      "-- check region:358 5:21235001-21265000_strand_+_gene_358, 58456 candidate probes\n",
      "--- 9329 probes passed check_dic selection.\n",
      "finish in 1.248s, 216 probes kept.\n",
      "-- check region:359 5:23735001-23765000_strand_+_gene_359, 58770 candidate probes\n",
      "--- 8580 probes passed check_dic selection.\n",
      "finish in 1.214s, 179 probes kept.\n",
      "-- check region:360 5:26235001-26265000_strand_+_gene_360, 59062 candidate probes\n",
      "--- 8788 probes passed check_dic selection.\n",
      "finish in 1.227s, 190 probes kept.\n",
      "-- check region:361 5:28735001-28765000_strand_+_gene_361, 53544 candidate probes\n",
      "--- 8271 probes passed check_dic selection.\n",
      "finish in 1.122s, 175 probes kept.\n",
      "-- check region:362 5:31235001-31265000_strand_+_gene_362, 59162 candidate probes\n",
      "--- 17331 probes passed check_dic selection.\n",
      "finish in 1.577s, 315 probes kept.\n",
      "-- check region:363 5:33735001-33765000_strand_+_gene_363, 59068 candidate probes\n",
      "--- 21668 probes passed check_dic selection.\n",
      "finish in 1.746s, 350 probes kept.\n",
      "-- check region:364 5:36235001-36265000_strand_+_gene_364, 59388 candidate probes\n",
      "--- 13534 probes passed check_dic selection.\n",
      "finish in 1.414s, 253 probes kept.\n",
      "-- check region:365 5:38735001-38765000_strand_+_gene_365, 59046 candidate probes\n",
      "--- 16349 probes passed check_dic selection.\n",
      "finish in 1.529s, 284 probes kept.\n",
      "-- check region:366 5:41235001-41265000_strand_+_gene_366, 53282 candidate probes\n",
      "--- 11233 probes passed check_dic selection.\n",
      "finish in 1.238s, 214 probes kept.\n",
      "-- check region:367 5:43735001-43765000_strand_+_gene_367, 54478 candidate probes\n",
      "--- 4208 probes passed check_dic selection.\n",
      "finish in 1.015s, 102 probes kept.\n",
      "-- check region:368 5:46235001-46265000_strand_+_gene_368, 59530 candidate probes\n",
      "--- 1032 probes passed check_dic selection.\n",
      "finish in 0.966s, 64 probes kept.\n",
      "-- check region:369 5:48735001-48765000_strand_+_gene_369, 24304 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.391s, 0 probes kept.\n",
      "-- check region:370 5:51235001-51265000_strand_+_gene_370, 50960 candidate probes\n",
      "--- 6825 probes passed check_dic selection.\n",
      "finish in 1.035s, 158 probes kept.\n",
      "-- check region:371 5:53735001-53765000_strand_+_gene_371, 54598 candidate probes\n",
      "--- 3175 probes passed check_dic selection.\n",
      "finish in 0.980s, 84 probes kept.\n",
      "-- check region:372 5:56235001-56265000_strand_+_gene_372, 58392 candidate probes\n",
      "--- 13554 probes passed check_dic selection.\n",
      "finish in 1.409s, 236 probes kept.\n",
      "-- check region:373 5:58735001-58765000_strand_+_gene_373, 58844 candidate probes\n",
      "--- 14895 probes passed check_dic selection.\n",
      "finish in 1.461s, 270 probes kept.\n",
      "-- check region:374 5:61235001-61265000_strand_+_gene_374, 52802 candidate probes\n",
      "--- 13482 probes passed check_dic selection.\n",
      "finish in 1.320s, 224 probes kept.\n",
      "-- check region:375 5:63735001-63765000_strand_+_gene_375, 58246 candidate probes\n",
      "--- 11151 probes passed check_dic selection.\n",
      "finish in 1.314s, 226 probes kept.\n",
      "-- check region:376 5:66235001-66265000_strand_+_gene_376, 52526 candidate probes\n",
      "--- 13116 probes passed check_dic selection.\n",
      "finish in 1.316s, 222 probes kept.\n",
      "-- check region:377 5:68735001-68765000_strand_+_gene_377, 59138 candidate probes\n",
      "--- 21259 probes passed check_dic selection.\n",
      "finish in 1.718s, 340 probes kept.\n",
      "-- check region:378 5:71235001-71265000_strand_+_gene_378, 55834 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.868s, 0 probes kept.\n",
      "-- check region:379 5:73735001-73765000_strand_+_gene_379, 58704 candidate probes\n",
      "--- 16691 probes passed check_dic selection.\n",
      "finish in 1.519s, 287 probes kept.\n",
      "-- check region:380 5:76235001-76265000_strand_+_gene_380, 57776 candidate probes\n",
      "--- 12588 probes passed check_dic selection.\n",
      "finish in 1.376s, 224 probes kept.\n",
      "-- check region:381 5:78735001-78765000_strand_+_gene_381, 58622 candidate probes\n",
      "--- 17248 probes passed check_dic selection.\n",
      "finish in 1.561s, 278 probes kept.\n",
      "-- check region:382 5:81235001-81265000_strand_+_gene_382, 57584 candidate probes\n",
      "--- 18961 probes passed check_dic selection.\n",
      "finish in 1.625s, 318 probes kept.\n",
      "-- check region:383 5:83735001-83765000_strand_+_gene_383, 57522 candidate probes\n",
      "--- 7500 probes passed check_dic selection.\n",
      "finish in 1.176s, 176 probes kept.\n",
      "-- check region:384 5:86235001-86265000_strand_+_gene_384, 58670 candidate probes\n",
      "--- 7344 probes passed check_dic selection.\n",
      "finish in 1.164s, 166 probes kept.\n",
      "-- check region:385 5:88735001-88765000_strand_+_gene_385, 58144 candidate probes\n",
      "--- 13547 probes passed check_dic selection.\n",
      "finish in 1.398s, 272 probes kept.\n",
      "-- check region:386 5:91235001-91265000_strand_+_gene_386, 58938 candidate probes\n",
      "--- 10074 probes passed check_dic selection.\n",
      "finish in 1.282s, 215 probes kept.\n",
      "-- check region:387 5:93735001-93765000_strand_+_gene_387, 56008 candidate probes\n",
      "--- 8698 probes passed check_dic selection.\n",
      "finish in 1.204s, 192 probes kept.\n",
      "-- check region:388 5:96235001-96265000_strand_+_gene_388, 57526 candidate probes\n",
      "--- 15719 probes passed check_dic selection.\n",
      "finish in 1.461s, 277 probes kept.\n",
      "-- check region:389 5:98735001-98765000_strand_+_gene_389, 57504 candidate probes\n",
      "--- 7234 probes passed check_dic selection.\n",
      "finish in 1.157s, 170 probes kept.\n",
      "-- check region:390 5:101235001-101265000_strand_+_gene_390, 59332 candidate probes\n",
      "--- 6865 probes passed check_dic selection.\n",
      "finish in 1.173s, 164 probes kept.\n",
      "-- check region:391 5:103735001-103765000_strand_+_gene_391, 58602 candidate probes\n",
      "--- 12027 probes passed check_dic selection.\n",
      "finish in 1.351s, 257 probes kept.\n",
      "-- check region:392 5:106235001-106265000_strand_+_gene_392, 58906 candidate probes\n",
      "--- 8898 probes passed check_dic selection.\n",
      "finish in 1.259s, 202 probes kept.\n",
      "-- check region:393 5:108735001-108765000_strand_+_gene_393, 58444 candidate probes\n",
      "--- 11195 probes passed check_dic selection.\n",
      "finish in 1.310s, 229 probes kept.\n",
      "-- check region:394 5:111235001-111265000_strand_+_gene_394, 59592 candidate probes\n",
      "--- 18167 probes passed check_dic selection.\n",
      "finish in 1.596s, 309 probes kept.\n",
      "-- check region:395 5:113735001-113765000_strand_+_gene_395, 58296 candidate probes\n",
      "--- 13054 probes passed check_dic selection.\n",
      "finish in 1.385s, 256 probes kept.\n",
      "-- check region:396 5:116235001-116265000_strand_+_gene_396, 53912 candidate probes\n",
      "--- 7453 probes passed check_dic selection.\n",
      "finish in 1.107s, 182 probes kept.\n",
      "-- check region:397 5:118735001-118765000_strand_+_gene_397, 58838 candidate probes\n",
      "--- 14321 probes passed check_dic selection.\n",
      "finish in 1.442s, 245 probes kept.\n",
      "-- check region:398 5:121235001-121265000_strand_+_gene_398, 56712 candidate probes\n",
      "--- 6667 probes passed check_dic selection.\n",
      "finish in 1.116s, 162 probes kept.\n",
      "-- check region:399 5:123735001-123765000_strand_+_gene_399, 59612 candidate probes\n",
      "--- 11138 probes passed check_dic selection.\n",
      "finish in 1.316s, 196 probes kept.\n",
      "-- check region:400 5:126235001-126265000_strand_+_gene_400, 58076 candidate probes\n",
      "--- 12039 probes passed check_dic selection.\n",
      "finish in 1.342s, 222 probes kept.\n",
      "-- check region:401 5:128735001-128765000_strand_+_gene_401, 58658 candidate probes\n",
      "--- 12589 probes passed check_dic selection.\n",
      "finish in 1.368s, 237 probes kept.\n",
      "-- check region:402 5:131235001-131265000_strand_+_gene_402, 57384 candidate probes\n",
      "--- 10030 probes passed check_dic selection.\n",
      "finish in 1.253s, 191 probes kept.\n",
      "-- check region:403 5:133735001-133765000_strand_+_gene_403, 59326 candidate probes\n",
      "--- 26130 probes passed check_dic selection.\n",
      "finish in 1.907s, 377 probes kept.\n",
      "-- check region:404 5:136235001-136265000_strand_+_gene_404, 59770 candidate probes\n",
      "--- 20940 probes passed check_dic selection.\n",
      "finish in 1.713s, 331 probes kept.\n",
      "-- check region:405 5:138735001-138765000_strand_+_gene_405, 56450 candidate probes\n",
      "--- 10844 probes passed check_dic selection.\n",
      "finish in 1.289s, 205 probes kept.\n",
      "-- check region:406 5:141235001-141265000_strand_+_gene_406, 58602 candidate probes\n",
      "--- 10206 probes passed check_dic selection.\n",
      "finish in 1.301s, 261 probes kept.\n",
      "-- check region:407 5:143735001-143765000_strand_+_gene_407, 59248 candidate probes\n",
      "--- 17059 probes passed check_dic selection.\n",
      "finish in 1.556s, 306 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:408 5:146235001-146265000_strand_+_gene_408, 56880 candidate probes\n",
      "--- 6809 probes passed check_dic selection.\n",
      "finish in 1.143s, 163 probes kept.\n",
      "-- check region:409 5:148735001-148765000_strand_+_gene_409, 58354 candidate probes\n",
      "--- 12846 probes passed check_dic selection.\n",
      "finish in 1.386s, 221 probes kept.\n",
      "-- check region:410 5:151235001-151265000_strand_+_gene_410, 58452 candidate probes\n",
      "--- 16114 probes passed check_dic selection.\n",
      "finish in 1.519s, 271 probes kept.\n",
      "-- check region:411 5:153735001-153765000_strand_+_gene_411, 59156 candidate probes\n",
      "--- 16064 probes passed check_dic selection.\n",
      "finish in 1.512s, 282 probes kept.\n",
      "-- check region:412 5:156235001-156265000_strand_+_gene_412, 58598 candidate probes\n",
      "--- 10484 probes passed check_dic selection.\n",
      "finish in 1.281s, 212 probes kept.\n",
      "-- check region:413 5:158735001-158765000_strand_+_gene_413, 59412 candidate probes\n",
      "--- 18989 probes passed check_dic selection.\n",
      "finish in 1.622s, 339 probes kept.\n",
      "-- check region:414 5:161235001-161265000_strand_+_gene_414, 55674 candidate probes\n",
      "--- 9346 probes passed check_dic selection.\n",
      "finish in 1.224s, 188 probes kept.\n",
      "-- check region:415 5:163735001-163765000_strand_+_gene_415, 59108 candidate probes\n",
      "--- 13183 probes passed check_dic selection.\n",
      "finish in 1.403s, 252 probes kept.\n",
      "-- check region:416 5:166235001-166265000_strand_+_gene_416, 59350 candidate probes\n",
      "--- 13622 probes passed check_dic selection.\n",
      "finish in 1.407s, 252 probes kept.\n",
      "-- check region:417 5:168735001-168765000_strand_+_gene_417, 58344 candidate probes\n",
      "--- 20345 probes passed check_dic selection.\n",
      "finish in 1.686s, 315 probes kept.\n",
      "-- check region:418 5:171235001-171265000_strand_+_gene_418, 58362 candidate probes\n",
      "--- 9558 probes passed check_dic selection.\n",
      "finish in 1.287s, 200 probes kept.\n",
      "-- check region:419 5:173735001-173765000_strand_+_gene_419, 58678 candidate probes\n",
      "--- 23552 probes passed check_dic selection.\n",
      "finish in 1.809s, 355 probes kept.\n",
      "-- check region:420 5:176235001-176265000_strand_+_gene_420, 54728 candidate probes\n",
      "--- 1000 probes passed check_dic selection.\n",
      "finish in 0.900s, 37 probes kept.\n",
      "-- check region:421 5:178735001-178765000_strand_+_gene_421, 57852 candidate probes\n",
      "--- 10228 probes passed check_dic selection.\n",
      "finish in 1.286s, 212 probes kept.\n",
      "-- check region:422 6:1235001-1265000_strand_+_gene_422, 58392 candidate probes\n",
      "--- 19603 probes passed check_dic selection.\n",
      "finish in 1.647s, 298 probes kept.\n",
      "-- check region:423 6:3735001-3765000_strand_+_gene_423, 59398 candidate probes\n",
      "--- 22974 probes passed check_dic selection.\n",
      "finish in 1.787s, 345 probes kept.\n",
      "-- check region:424 6:6235001-6265000_strand_+_gene_424, 58358 candidate probes\n",
      "--- 18492 probes passed check_dic selection.\n",
      "finish in 1.593s, 323 probes kept.\n",
      "-- check region:425 6:8735001-8765000_strand_+_gene_425, 59282 candidate probes\n",
      "--- 11841 probes passed check_dic selection.\n",
      "finish in 1.354s, 221 probes kept.\n",
      "-- check region:426 6:11235001-11265000_strand_+_gene_426, 59502 candidate probes\n",
      "--- 22652 probes passed check_dic selection.\n",
      "finish in 1.773s, 350 probes kept.\n",
      "-- check region:427 6:13735001-13765000_strand_+_gene_427, 56734 candidate probes\n",
      "--- 12281 probes passed check_dic selection.\n",
      "finish in 1.342s, 217 probes kept.\n",
      "-- check region:428 6:16235001-16265000_strand_+_gene_428, 55176 candidate probes\n",
      "--- 15424 probes passed check_dic selection.\n",
      "finish in 1.442s, 243 probes kept.\n",
      "-- check region:429 6:18735001-18765000_strand_+_gene_429, 55708 candidate probes\n",
      "--- 9154 probes passed check_dic selection.\n",
      "finish in 1.204s, 193 probes kept.\n",
      "-- check region:430 6:21235001-21265000_strand_+_gene_430, 58278 candidate probes\n",
      "--- 13443 probes passed check_dic selection.\n",
      "finish in 1.400s, 236 probes kept.\n",
      "-- check region:431 6:23735001-23765000_strand_+_gene_431, 58448 candidate probes\n",
      "--- 11542 probes passed check_dic selection.\n",
      "finish in 1.330s, 237 probes kept.\n",
      "-- check region:432 6:26235001-26265000_strand_+_gene_432, 58348 candidate probes\n",
      "--- 14722 probes passed check_dic selection.\n",
      "finish in 1.456s, 270 probes kept.\n",
      "-- check region:433 6:28735001-28765000_strand_+_gene_433, 57786 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.921s, 0 probes kept.\n",
      "-- check region:434 6:31235001-31265000_strand_+_gene_434, 53592 candidate probes\n",
      "--- 108 probes passed check_dic selection.\n",
      "finish in 0.856s, 5 probes kept.\n",
      "-- check region:435 6:33735001-33765000_strand_+_gene_435, 58186 candidate probes\n",
      "--- 22674 probes passed check_dic selection.\n",
      "finish in 1.770s, 357 probes kept.\n",
      "-- check region:436 6:36235001-36265000_strand_+_gene_436, 58796 candidate probes\n",
      "--- 18289 probes passed check_dic selection.\n",
      "finish in 1.612s, 294 probes kept.\n",
      "-- check region:437 6:38735001-38765000_strand_+_gene_437, 55554 candidate probes\n",
      "--- 7860 probes passed check_dic selection.\n",
      "finish in 1.156s, 188 probes kept.\n",
      "-- check region:438 6:41235001-41265000_strand_+_gene_438, 56446 candidate probes\n",
      "--- 10300 probes passed check_dic selection.\n",
      "finish in 1.273s, 206 probes kept.\n",
      "-- check region:439 6:43735001-43765000_strand_+_gene_439, 58982 candidate probes\n",
      "--- 19427 probes passed check_dic selection.\n",
      "finish in 1.664s, 310 probes kept.\n",
      "-- check region:440 6:46235001-46265000_strand_+_gene_440, 59438 candidate probes\n",
      "--- 17864 probes passed check_dic selection.\n",
      "finish in 1.585s, 321 probes kept.\n",
      "-- check region:441 6:48735001-48765000_strand_+_gene_441, 51024 candidate probes\n",
      "--- 7988 probes passed check_dic selection.\n",
      "finish in 1.076s, 183 probes kept.\n",
      "-- check region:442 6:51235001-51265000_strand_+_gene_442, 59470 candidate probes\n",
      "--- 14199 probes passed check_dic selection.\n",
      "finish in 1.437s, 276 probes kept.\n",
      "-- check region:443 6:53735001-53765000_strand_+_gene_443, 58526 candidate probes\n",
      "--- 12697 probes passed check_dic selection.\n",
      "finish in 1.380s, 241 probes kept.\n",
      "-- check region:444 6:56235001-56265000_strand_+_gene_444, 59790 candidate probes\n",
      "--- 16212 probes passed check_dic selection.\n",
      "finish in 1.511s, 307 probes kept.\n",
      "-- check region:445 6:58735001-58765000_strand_+_gene_445, 14622 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.238s, 0 probes kept.\n",
      "-- check region:446 6:61235001-61265000_strand_+_gene_446, 55292 candidate probes\n",
      "--- 4728 probes passed check_dic selection.\n",
      "finish in 1.032s, 108 probes kept.\n",
      "-- check region:447 6:63735001-63765000_strand_+_gene_447, 54288 candidate probes\n",
      "--- 8054 probes passed check_dic selection.\n",
      "finish in 1.145s, 181 probes kept.\n",
      "-- check region:448 6:66235001-66265000_strand_+_gene_448, 58510 candidate probes\n",
      "--- 4580 probes passed check_dic selection.\n",
      "finish in 1.079s, 127 probes kept.\n",
      "-- check region:449 6:68735001-68765000_strand_+_gene_449, 59456 candidate probes\n",
      "--- 11111 probes passed check_dic selection.\n",
      "finish in 1.317s, 228 probes kept.\n",
      "-- check region:450 6:71235001-71265000_strand_+_gene_450, 59491 candidate probes\n",
      "--- 13566 probes passed check_dic selection.\n",
      "finish in 1.417s, 268 probes kept.\n",
      "-- check region:451 6:73735001-73765000_strand_+_gene_451, 58302 candidate probes\n",
      "--- 13529 probes passed check_dic selection.\n",
      "finish in 1.407s, 258 probes kept.\n",
      "-- check region:452 6:76235001-76265000_strand_+_gene_452, 59190 candidate probes\n",
      "--- 8912 probes passed check_dic selection.\n",
      "finish in 1.244s, 197 probes kept.\n",
      "-- check region:453 6:78735001-78765000_strand_+_gene_453, 52894 candidate probes\n",
      "--- 2144 probes passed check_dic selection.\n",
      "finish in 0.910s, 67 probes kept.\n",
      "-- check region:454 6:81235001-81265000_strand_+_gene_454, 57654 candidate probes\n",
      "--- 6470 probes passed check_dic selection.\n",
      "finish in 1.121s, 145 probes kept.\n",
      "-- check region:455 6:83735001-83765000_strand_+_gene_455, 58740 candidate probes\n",
      "--- 5414 probes passed check_dic selection.\n",
      "finish in 1.105s, 142 probes kept.\n",
      "-- check region:456 6:86235001-86265000_strand_+_gene_456, 57828 candidate probes\n",
      "--- 11139 probes passed check_dic selection.\n",
      "finish in 1.313s, 217 probes kept.\n",
      "-- check region:457 6:88735001-88765000_strand_+_gene_457, 59404 candidate probes\n",
      "--- 9824 probes passed check_dic selection.\n",
      "finish in 1.277s, 207 probes kept.\n",
      "-- check region:458 6:91235001-91265000_strand_+_gene_458, 59664 candidate probes\n",
      "--- 12553 probes passed check_dic selection.\n",
      "finish in 1.385s, 236 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:459 6:93735001-93765000_strand_+_gene_459, 59612 candidate probes\n",
      "--- 8077 probes passed check_dic selection.\n",
      "finish in 1.198s, 180 probes kept.\n",
      "-- check region:460 6:96235001-96265000_strand_+_gene_460, 58726 candidate probes\n",
      "--- 8874 probes passed check_dic selection.\n",
      "finish in 1.232s, 202 probes kept.\n",
      "-- check region:461 6:98735001-98765000_strand_+_gene_461, 59402 candidate probes\n",
      "--- 10817 probes passed check_dic selection.\n",
      "finish in 1.317s, 209 probes kept.\n",
      "-- check region:462 6:101235001-101265000_strand_+_gene_462, 56344 candidate probes\n",
      "--- 7306 probes passed check_dic selection.\n",
      "finish in 1.138s, 170 probes kept.\n",
      "-- check region:463 6:103735001-103765000_strand_+_gene_463, 57636 candidate probes\n",
      "--- 6226 probes passed check_dic selection.\n",
      "finish in 1.119s, 143 probes kept.\n",
      "-- check region:464 6:106235001-106265000_strand_+_gene_464, 59076 candidate probes\n",
      "--- 9690 probes passed check_dic selection.\n",
      "finish in 1.264s, 211 probes kept.\n",
      "-- check region:465 6:108735001-108765000_strand_+_gene_465, 58654 candidate probes\n",
      "--- 21972 probes passed check_dic selection.\n",
      "finish in 1.739s, 352 probes kept.\n",
      "-- check region:466 6:111235001-111265000_strand_+_gene_466, 57222 candidate probes\n",
      "--- 10460 probes passed check_dic selection.\n",
      "finish in 1.277s, 189 probes kept.\n",
      "-- check region:467 6:113735001-113765000_strand_+_gene_467, 57176 candidate probes\n",
      "--- 10480 probes passed check_dic selection.\n",
      "finish in 1.274s, 200 probes kept.\n",
      "-- check region:468 6:116235001-116265000_strand_+_gene_468, 59232 candidate probes\n",
      "--- 7985 probes passed check_dic selection.\n",
      "finish in 1.235s, 222 probes kept.\n",
      "-- check region:469 6:118735001-118765000_strand_+_gene_469, 58286 candidate probes\n",
      "--- 15356 probes passed check_dic selection.\n",
      "finish in 1.487s, 260 probes kept.\n",
      "-- check region:470 6:121235001-121265000_strand_+_gene_470, 58672 candidate probes\n",
      "--- 5042 probes passed check_dic selection.\n",
      "finish in 1.095s, 131 probes kept.\n",
      "-- check region:471 6:123735001-123765000_strand_+_gene_471, 59022 candidate probes\n",
      "--- 8233 probes passed check_dic selection.\n",
      "finish in 1.207s, 199 probes kept.\n",
      "-- check region:472 6:126235001-126265000_strand_+_gene_472, 54670 candidate probes\n",
      "--- 10094 probes passed check_dic selection.\n",
      "finish in 1.216s, 203 probes kept.\n",
      "-- check region:473 6:128735001-128765000_strand_+_gene_473, 59134 candidate probes\n",
      "--- 10493 probes passed check_dic selection.\n",
      "finish in 1.301s, 203 probes kept.\n",
      "-- check region:474 6:131235001-131265000_strand_+_gene_474, 53924 candidate probes\n",
      "--- 12981 probes passed check_dic selection.\n",
      "finish in 1.319s, 235 probes kept.\n",
      "-- check region:475 6:133735001-133765000_strand_+_gene_475, 59592 candidate probes\n",
      "--- 13540 probes passed check_dic selection.\n",
      "finish in 1.414s, 263 probes kept.\n",
      "-- check region:476 6:136235001-136265000_strand_+_gene_476, 58416 candidate probes\n",
      "--- 11193 probes passed check_dic selection.\n",
      "finish in 1.321s, 236 probes kept.\n",
      "-- check region:477 6:138735001-138765000_strand_+_gene_477, 57216 candidate probes\n",
      "--- 11848 probes passed check_dic selection.\n",
      "finish in 1.334s, 215 probes kept.\n",
      "-- check region:478 6:141235001-141265000_strand_+_gene_478, 57824 candidate probes\n",
      "--- 7652 probes passed check_dic selection.\n",
      "finish in 1.157s, 189 probes kept.\n",
      "-- check region:479 6:143735001-143765000_strand_+_gene_479, 58634 candidate probes\n",
      "--- 17328 probes passed check_dic selection.\n",
      "finish in 1.560s, 287 probes kept.\n",
      "-- check region:480 6:146235001-146265000_strand_+_gene_480, 58952 candidate probes\n",
      "--- 14006 probes passed check_dic selection.\n",
      "finish in 1.429s, 257 probes kept.\n",
      "-- check region:481 6:148735001-148765000_strand_+_gene_481, 57956 candidate probes\n",
      "--- 16188 probes passed check_dic selection.\n",
      "finish in 1.491s, 269 probes kept.\n",
      "-- check region:482 6:151235001-151265000_strand_+_gene_482, 57186 candidate probes\n",
      "--- 13863 probes passed check_dic selection.\n",
      "finish in 1.416s, 230 probes kept.\n",
      "-- check region:483 6:153735001-153765000_strand_+_gene_483, 59198 candidate probes\n",
      "--- 10037 probes passed check_dic selection.\n",
      "finish in 1.293s, 220 probes kept.\n",
      "-- check region:484 6:156235001-156265000_strand_+_gene_484, 58910 candidate probes\n",
      "--- 14100 probes passed check_dic selection.\n",
      "finish in 1.446s, 265 probes kept.\n",
      "-- check region:485 6:158735001-158765000_strand_+_gene_485, 56788 candidate probes\n",
      "--- 15744 probes passed check_dic selection.\n",
      "finish in 1.475s, 255 probes kept.\n",
      "-- check region:486 6:161235001-161265000_strand_+_gene_486, 58420 candidate probes\n",
      "--- 20027 probes passed check_dic selection.\n",
      "finish in 1.658s, 294 probes kept.\n",
      "-- check region:487 6:163735001-163765000_strand_+_gene_487, 59034 candidate probes\n",
      "--- 21320 probes passed check_dic selection.\n",
      "finish in 1.740s, 342 probes kept.\n",
      "-- check region:488 6:166235001-166265000_strand_+_gene_488, 58184 candidate probes\n",
      "--- 52 probes passed check_dic selection.\n",
      "finish in 0.935s, 3 probes kept.\n",
      "-- check region:489 6:168735001-168765000_strand_+_gene_489, 58548 candidate probes\n",
      "--- 11867 probes passed check_dic selection.\n",
      "finish in 1.353s, 216 probes kept.\n",
      "-- check region:490 7:1235001-1265000_strand_+_gene_490, 59058 candidate probes\n",
      "--- 25941 probes passed check_dic selection.\n",
      "finish in 1.874s, 384 probes kept.\n",
      "-- check region:491 7:3735001-3765000_strand_+_gene_491, 58846 candidate probes\n",
      "--- 15596 probes passed check_dic selection.\n",
      "finish in 1.495s, 276 probes kept.\n",
      "-- check region:492 7:6235001-6265000_strand_+_gene_492, 57965 candidate probes\n",
      "--- 13241 probes passed check_dic selection.\n",
      "finish in 1.400s, 229 probes kept.\n",
      "-- check region:493 7:8735001-8765000_strand_+_gene_493, 58774 candidate probes\n",
      "--- 12135 probes passed check_dic selection.\n",
      "finish in 1.354s, 239 probes kept.\n",
      "-- check region:494 7:11235001-11265000_strand_+_gene_494, 59314 candidate probes\n",
      "--- 14673 probes passed check_dic selection.\n",
      "finish in 1.466s, 293 probes kept.\n",
      "-- check region:495 7:13735001-13765000_strand_+_gene_495, 59226 candidate probes\n",
      "--- 13267 probes passed check_dic selection.\n",
      "finish in 1.389s, 259 probes kept.\n",
      "-- check region:496 7:16235001-16265000_strand_+_gene_496, 56808 candidate probes\n",
      "--- 6990 probes passed check_dic selection.\n",
      "finish in 1.137s, 167 probes kept.\n",
      "-- check region:497 7:18735001-18765000_strand_+_gene_497, 57740 candidate probes\n",
      "--- 10322 probes passed check_dic selection.\n",
      "finish in 1.265s, 228 probes kept.\n",
      "-- check region:498 7:21235001-21265000_strand_+_gene_498, 58528 candidate probes\n",
      "--- 13638 probes passed check_dic selection.\n",
      "finish in 1.401s, 262 probes kept.\n",
      "-- check region:499 7:23735001-23765000_strand_+_gene_499, 57398 candidate probes\n",
      "--- 7698 probes passed check_dic selection.\n",
      "finish in 1.179s, 162 probes kept.\n",
      "-- check region:500 7:26235001-26265000_strand_+_gene_500, 55900 candidate probes\n",
      "--- 5262 probes passed check_dic selection.\n",
      "finish in 1.084s, 142 probes kept.\n",
      "-- check region:501 7:28735001-28765000_strand_+_gene_501, 57528 candidate probes\n",
      "--- 18329 probes passed check_dic selection.\n",
      "finish in 1.564s, 304 probes kept.\n",
      "-- check region:502 7:31235001-31265000_strand_+_gene_502, 59096 candidate probes\n",
      "--- 16295 probes passed check_dic selection.\n",
      "finish in 1.520s, 269 probes kept.\n",
      "-- check region:503 7:33735001-33765000_strand_+_gene_503, 59038 candidate probes\n",
      "--- 19694 probes passed check_dic selection.\n",
      "finish in 1.663s, 318 probes kept.\n",
      "-- check region:504 7:36235001-36265000_strand_+_gene_504, 56492 candidate probes\n",
      "--- 12168 probes passed check_dic selection.\n",
      "finish in 1.340s, 220 probes kept.\n",
      "-- check region:505 7:38735001-38765000_strand_+_gene_505, 59808 candidate probes\n",
      "--- 17556 probes passed check_dic selection.\n",
      "finish in 1.572s, 308 probes kept.\n",
      "-- check region:506 7:41235001-41265000_strand_+_gene_506, 59594 candidate probes\n",
      "--- 17450 probes passed check_dic selection.\n",
      "finish in 1.569s, 306 probes kept.\n",
      "-- check region:507 7:43735001-43765000_strand_+_gene_507, 57900 candidate probes\n",
      "--- 10945 probes passed check_dic selection.\n",
      "finish in 1.307s, 202 probes kept.\n",
      "-- check region:508 7:46235001-46265000_strand_+_gene_508, 59208 candidate probes\n",
      "--- 10168 probes passed check_dic selection.\n",
      "finish in 1.284s, 221 probes kept.\n",
      "-- check region:509 7:48735001-48765000_strand_+_gene_509, 59608 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 15186 probes passed check_dic selection.\n",
      "finish in 1.479s, 272 probes kept.\n",
      "-- check region:510 7:51235001-51265000_strand_+_gene_510, 59338 candidate probes\n",
      "--- 21452 probes passed check_dic selection.\n",
      "finish in 1.729s, 336 probes kept.\n",
      "-- check region:511 7:53735001-53765000_strand_+_gene_511, 59534 candidate probes\n",
      "--- 8992 probes passed check_dic selection.\n",
      "finish in 1.248s, 184 probes kept.\n",
      "-- check region:512 7:56235001-56265000_strand_+_gene_512, 56492 candidate probes\n",
      "--- 9995 probes passed check_dic selection.\n",
      "finish in 1.270s, 197 probes kept.\n",
      "-- check region:513 7:58735001-58765000_strand_+_gene_513, 11780 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.191s, 0 probes kept.\n",
      "-- check region:514 7:61235001-61265000_strand_+_gene_514, 22414 candidate probes\n",
      "--- 18 probes passed check_dic selection.\n",
      "finish in 0.359s, 2 probes kept.\n",
      "-- check region:515 7:63735001-63765000_strand_+_gene_515, 33010 candidate probes\n",
      "--- 322 probes passed check_dic selection.\n",
      "finish in 0.547s, 20 probes kept.\n",
      "-- check region:516 7:66235001-66265000_strand_+_gene_516, 57870 candidate probes\n",
      "--- 10584 probes passed check_dic selection.\n",
      "finish in 1.297s, 227 probes kept.\n",
      "-- check region:517 7:68735001-68765000_strand_+_gene_517, 57906 candidate probes\n",
      "--- 14651 probes passed check_dic selection.\n",
      "finish in 1.459s, 262 probes kept.\n",
      "-- check region:518 7:71235001-71265000_strand_+_gene_518, 58108 candidate probes\n",
      "--- 15277 probes passed check_dic selection.\n",
      "finish in 1.485s, 264 probes kept.\n",
      "-- check region:519 7:73735001-73765000_strand_+_gene_519, 55418 candidate probes\n",
      "--- 10501 probes passed check_dic selection.\n",
      "finish in 1.282s, 188 probes kept.\n",
      "-- check region:520 7:76235001-76265000_strand_+_gene_520, 55754 candidate probes\n",
      "--- 13254 probes passed check_dic selection.\n",
      "finish in 1.380s, 212 probes kept.\n",
      "-- check region:521 7:78735001-78765000_strand_+_gene_521, 58934 candidate probes\n",
      "--- 12509 probes passed check_dic selection.\n",
      "finish in 1.367s, 265 probes kept.\n",
      "-- check region:522 7:81235001-81265000_strand_+_gene_522, 59296 candidate probes\n",
      "--- 7588 probes passed check_dic selection.\n",
      "finish in 1.196s, 187 probes kept.\n",
      "-- check region:523 7:83735001-83765000_strand_+_gene_523, 52294 candidate probes\n",
      "--- 7339 probes passed check_dic selection.\n",
      "finish in 1.080s, 152 probes kept.\n",
      "-- check region:524 7:86235001-86265000_strand_+_gene_524, 56958 candidate probes\n",
      "--- 8650 probes passed check_dic selection.\n",
      "finish in 1.188s, 196 probes kept.\n",
      "-- check region:525 7:88735001-88765000_strand_+_gene_525, 59626 candidate probes\n",
      "--- 10108 probes passed check_dic selection.\n",
      "finish in 1.278s, 216 probes kept.\n",
      "-- check region:526 7:91235001-91265000_strand_+_gene_526, 58114 candidate probes\n",
      "--- 12610 probes passed check_dic selection.\n",
      "finish in 1.366s, 223 probes kept.\n",
      "-- check region:527 7:93735001-93765000_strand_+_gene_527, 58410 candidate probes\n",
      "--- 12178 probes passed check_dic selection.\n",
      "finish in 1.343s, 255 probes kept.\n",
      "-- check region:528 7:96235001-96265000_strand_+_gene_528, 58342 candidate probes\n",
      "--- 14230 probes passed check_dic selection.\n",
      "finish in 1.434s, 249 probes kept.\n",
      "-- check region:529 7:98735001-98765000_strand_+_gene_529, 56898 candidate probes\n",
      "--- 17912 probes passed check_dic selection.\n",
      "finish in 1.562s, 269 probes kept.\n",
      "-- check region:530 7:101235001-101265000_strand_+_gene_530, 56806 candidate probes\n",
      "--- 12896 probes passed check_dic selection.\n",
      "finish in 1.408s, 223 probes kept.\n",
      "-- check region:531 7:103735001-103765000_strand_+_gene_531, 54226 candidate probes\n",
      "--- 10681 probes passed check_dic selection.\n",
      "finish in 1.236s, 201 probes kept.\n",
      "-- check region:532 7:106235001-106265000_strand_+_gene_532, 55740 candidate probes\n",
      "--- 12049 probes passed check_dic selection.\n",
      "finish in 1.312s, 232 probes kept.\n",
      "-- check region:533 7:108735001-108765000_strand_+_gene_533, 58946 candidate probes\n",
      "--- 10228 probes passed check_dic selection.\n",
      "finish in 1.292s, 201 probes kept.\n",
      "-- check region:534 7:111235001-111265000_strand_+_gene_534, 48940 candidate probes\n",
      "--- 8227 probes passed check_dic selection.\n",
      "finish in 1.053s, 181 probes kept.\n",
      "-- check region:535 7:113735001-113765000_strand_+_gene_535, 45718 candidate probes\n",
      "--- 5035 probes passed check_dic selection.\n",
      "finish in 0.894s, 116 probes kept.\n",
      "-- check region:536 7:116235001-116265000_strand_+_gene_536, 58996 candidate probes\n",
      "--- 12561 probes passed check_dic selection.\n",
      "finish in 1.383s, 259 probes kept.\n",
      "-- check region:537 7:118735001-118765000_strand_+_gene_537, 59498 candidate probes\n",
      "--- 11385 probes passed check_dic selection.\n",
      "finish in 1.329s, 231 probes kept.\n",
      "-- check region:538 7:121235001-121265000_strand_+_gene_538, 55112 candidate probes\n",
      "--- 11315 probes passed check_dic selection.\n",
      "finish in 1.273s, 214 probes kept.\n",
      "-- check region:539 7:123735001-123765000_strand_+_gene_539, 59296 candidate probes\n",
      "--- 12614 probes passed check_dic selection.\n",
      "finish in 1.366s, 242 probes kept.\n",
      "-- check region:540 7:126235001-126265000_strand_+_gene_540, 58816 candidate probes\n",
      "--- 8154 probes passed check_dic selection.\n",
      "finish in 1.195s, 179 probes kept.\n",
      "-- check region:541 7:128735001-128765000_strand_+_gene_541, 58330 candidate probes\n",
      "--- 13976 probes passed check_dic selection.\n",
      "finish in 1.431s, 248 probes kept.\n",
      "-- check region:542 7:131235001-131265000_strand_+_gene_542, 57724 candidate probes\n",
      "--- 13859 probes passed check_dic selection.\n",
      "finish in 1.413s, 227 probes kept.\n",
      "-- check region:543 7:133735001-133765000_strand_+_gene_543, 59574 candidate probes\n",
      "--- 16651 probes passed check_dic selection.\n",
      "finish in 1.542s, 302 probes kept.\n",
      "-- check region:544 7:136235001-136265000_strand_+_gene_544, 59234 candidate probes\n",
      "--- 14305 probes passed check_dic selection.\n",
      "finish in 1.454s, 276 probes kept.\n",
      "-- check region:545 7:138735001-138765000_strand_+_gene_545, 57266 candidate probes\n",
      "--- 16229 probes passed check_dic selection.\n",
      "finish in 1.510s, 267 probes kept.\n",
      "-- check region:546 7:141235001-141265000_strand_+_gene_546, 59082 candidate probes\n",
      "--- 19321 probes passed check_dic selection.\n",
      "finish in 1.630s, 305 probes kept.\n",
      "-- check region:547 7:143735001-143765000_strand_+_gene_547, 55556 candidate probes\n",
      "--- 1807 probes passed check_dic selection.\n",
      "finish in 0.947s, 44 probes kept.\n",
      "-- check region:548 7:146235001-146265000_strand_+_gene_548, 57240 candidate probes\n",
      "--- 9504 probes passed check_dic selection.\n",
      "finish in 1.237s, 191 probes kept.\n",
      "-- check region:549 7:148735001-148765000_strand_+_gene_549, 58218 candidate probes\n",
      "--- 10746 probes passed check_dic selection.\n",
      "finish in 1.299s, 206 probes kept.\n",
      "-- check region:550 7:151235001-151265000_strand_+_gene_550, 58426 candidate probes\n",
      "--- 23572 probes passed check_dic selection.\n",
      "finish in 1.831s, 378 probes kept.\n",
      "-- check region:551 7:153735001-153765000_strand_+_gene_551, 59062 candidate probes\n",
      "--- 3248 probes passed check_dic selection.\n",
      "finish in 1.039s, 90 probes kept.\n",
      "-- check region:552 7:156235001-156265000_strand_+_gene_552, 59514 candidate probes\n",
      "--- 17668 probes passed check_dic selection.\n",
      "finish in 1.580s, 264 probes kept.\n",
      "-- check region:553 8:1235001-1265000_strand_+_gene_553, 53214 candidate probes\n",
      "--- 1480 probes passed check_dic selection.\n",
      "finish in 0.920s, 49 probes kept.\n",
      "-- check region:554 8:3735001-3765000_strand_+_gene_554, 59280 candidate probes\n",
      "--- 18404 probes passed check_dic selection.\n",
      "finish in 1.594s, 306 probes kept.\n",
      "-- check region:555 8:6235001-6265000_strand_+_gene_555, 57736 candidate probes\n",
      "--- 13714 probes passed check_dic selection.\n",
      "finish in 1.405s, 241 probes kept.\n",
      "-- check region:556 8:8735001-8765000_strand_+_gene_556, 58102 candidate probes\n",
      "--- 13515 probes passed check_dic selection.\n",
      "finish in 1.402s, 234 probes kept.\n",
      "-- check region:557 8:11235001-11265000_strand_+_gene_557, 59186 candidate probes\n",
      "--- 15948 probes passed check_dic selection.\n",
      "finish in 1.514s, 273 probes kept.\n",
      "-- check region:558 8:13735001-13765000_strand_+_gene_558, 59458 candidate probes\n",
      "--- 12585 probes passed check_dic selection.\n",
      "finish in 1.387s, 251 probes kept.\n",
      "-- check region:559 8:16235001-16265000_strand_+_gene_559, 59112 candidate probes\n",
      "--- 11281 probes passed check_dic selection.\n",
      "finish in 1.326s, 221 probes kept.\n",
      "-- check region:560 8:18735001-18765000_strand_+_gene_560, 58018 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 13812 probes passed check_dic selection.\n",
      "finish in 1.410s, 261 probes kept.\n",
      "-- check region:561 8:21235001-21265000_strand_+_gene_561, 58978 candidate probes\n",
      "--- 13366 probes passed check_dic selection.\n",
      "finish in 1.396s, 240 probes kept.\n",
      "-- check region:562 8:23735001-23765000_strand_+_gene_562, 58728 candidate probes\n",
      "--- 15209 probes passed check_dic selection.\n",
      "finish in 1.483s, 249 probes kept.\n",
      "-- check region:563 8:26235001-26265000_strand_+_gene_563, 57378 candidate probes\n",
      "--- 10265 probes passed check_dic selection.\n",
      "finish in 1.287s, 193 probes kept.\n",
      "-- check region:564 8:28735001-28765000_strand_+_gene_564, 56384 candidate probes\n",
      "--- 14050 probes passed check_dic selection.\n",
      "finish in 1.415s, 270 probes kept.\n",
      "-- check region:565 8:31235001-31265000_strand_+_gene_565, 57680 candidate probes\n",
      "--- 11456 probes passed check_dic selection.\n",
      "finish in 1.328s, 218 probes kept.\n",
      "-- check region:566 8:33735001-33765000_strand_+_gene_566, 58946 candidate probes\n",
      "--- 8690 probes passed check_dic selection.\n",
      "finish in 1.251s, 169 probes kept.\n",
      "-- check region:567 8:36235001-36265000_strand_+_gene_567, 58530 candidate probes\n",
      "--- 13148 probes passed check_dic selection.\n",
      "finish in 1.403s, 229 probes kept.\n",
      "-- check region:568 8:38735001-38765000_strand_+_gene_568, 58086 candidate probes\n",
      "--- 12693 probes passed check_dic selection.\n",
      "finish in 1.376s, 222 probes kept.\n",
      "-- check region:569 8:41235001-41265000_strand_+_gene_569, 59066 candidate probes\n",
      "--- 17171 probes passed check_dic selection.\n",
      "finish in 1.566s, 292 probes kept.\n",
      "-- check region:570 8:43735001-43765000_strand_+_gene_570, 58986 candidate probes\n",
      "--- 2317 probes passed check_dic selection.\n",
      "finish in 1.017s, 104 probes kept.\n",
      "-- check region:571 8:46235001-46265000_strand_+_gene_571, 59692 candidate probes\n",
      "--- 18551 probes passed check_dic selection.\n",
      "finish in 1.656s, 356 probes kept.\n",
      "-- check region:572 8:48735001-48765000_strand_+_gene_572, 58682 candidate probes\n",
      "--- 17341 probes passed check_dic selection.\n",
      "finish in 1.548s, 280 probes kept.\n",
      "-- check region:573 8:51235001-51265000_strand_+_gene_573, 59564 candidate probes\n",
      "--- 9165 probes passed check_dic selection.\n",
      "finish in 1.260s, 206 probes kept.\n",
      "-- check region:574 8:53735001-53765000_strand_+_gene_574, 58094 candidate probes\n",
      "--- 13846 probes passed check_dic selection.\n",
      "finish in 1.411s, 258 probes kept.\n",
      "-- check region:575 8:56235001-56265000_strand_+_gene_575, 55372 candidate probes\n",
      "--- 10284 probes passed check_dic selection.\n",
      "finish in 1.235s, 204 probes kept.\n",
      "-- check region:576 8:58735001-58765000_strand_+_gene_576, 59668 candidate probes\n",
      "--- 17149 probes passed check_dic selection.\n",
      "finish in 1.552s, 291 probes kept.\n",
      "-- check region:577 8:61235001-61265000_strand_+_gene_577, 56988 candidate probes\n",
      "--- 10344 probes passed check_dic selection.\n",
      "finish in 1.276s, 207 probes kept.\n",
      "-- check region:578 8:63735001-63765000_strand_+_gene_578, 59666 candidate probes\n",
      "--- 12430 probes passed check_dic selection.\n",
      "finish in 1.381s, 246 probes kept.\n",
      "-- check region:579 8:66235001-66265000_strand_+_gene_579, 58238 candidate probes\n",
      "--- 6592 probes passed check_dic selection.\n",
      "finish in 1.149s, 151 probes kept.\n",
      "-- check region:580 8:68735001-68765000_strand_+_gene_580, 52198 candidate probes\n",
      "--- 10313 probes passed check_dic selection.\n",
      "finish in 1.185s, 202 probes kept.\n",
      "-- check region:581 8:71235001-71265000_strand_+_gene_581, 59522 candidate probes\n",
      "--- 13847 probes passed check_dic selection.\n",
      "finish in 1.421s, 250 probes kept.\n",
      "-- check region:582 8:73735001-73765000_strand_+_gene_582, 57024 candidate probes\n",
      "--- 10981 probes passed check_dic selection.\n",
      "finish in 1.289s, 199 probes kept.\n",
      "-- check region:583 8:76235001-76265000_strand_+_gene_583, 59650 candidate probes\n",
      "--- 7832 probes passed check_dic selection.\n",
      "finish in 1.200s, 181 probes kept.\n",
      "-- check region:584 8:78735001-78765000_strand_+_gene_584, 58456 candidate probes\n",
      "--- 5299 probes passed check_dic selection.\n",
      "finish in 1.092s, 139 probes kept.\n",
      "-- check region:585 8:81235001-81265000_strand_+_gene_585, 59522 candidate probes\n",
      "--- 12095 probes passed check_dic selection.\n",
      "finish in 1.371s, 241 probes kept.\n",
      "-- check region:586 8:83735001-83765000_strand_+_gene_586, 59586 candidate probes\n",
      "--- 9264 probes passed check_dic selection.\n",
      "finish in 1.248s, 218 probes kept.\n",
      "-- check region:587 8:86235001-86265000_strand_+_gene_587, 59368 candidate probes\n",
      "--- 5999 probes passed check_dic selection.\n",
      "finish in 1.140s, 156 probes kept.\n",
      "-- check region:588 8:88735001-88765000_strand_+_gene_588, 57440 candidate probes\n",
      "--- 7672 probes passed check_dic selection.\n",
      "finish in 1.168s, 177 probes kept.\n",
      "-- check region:589 8:91235001-91265000_strand_+_gene_589, 59736 candidate probes\n",
      "--- 13972 probes passed check_dic selection.\n",
      "finish in 1.426s, 275 probes kept.\n",
      "-- check region:590 8:93735001-93765000_strand_+_gene_590, 58602 candidate probes\n",
      "--- 11231 probes passed check_dic selection.\n",
      "finish in 1.342s, 225 probes kept.\n",
      "-- check region:591 8:96235001-96265000_strand_+_gene_591, 59180 candidate probes\n",
      "--- 16137 probes passed check_dic selection.\n",
      "finish in 1.513s, 283 probes kept.\n",
      "-- check region:592 8:98735001-98765000_strand_+_gene_592, 52732 candidate probes\n",
      "--- 6568 probes passed check_dic selection.\n",
      "finish in 1.062s, 144 probes kept.\n",
      "-- check region:593 8:101235001-101265000_strand_+_gene_593, 59166 candidate probes\n",
      "--- 17489 probes passed check_dic selection.\n",
      "finish in 1.574s, 306 probes kept.\n",
      "-- check region:594 8:103735001-103765000_strand_+_gene_594, 54296 candidate probes\n",
      "--- 7094 probes passed check_dic selection.\n",
      "finish in 1.100s, 155 probes kept.\n",
      "-- check region:595 8:106235001-106265000_strand_+_gene_595, 59424 candidate probes\n",
      "--- 7965 probes passed check_dic selection.\n",
      "finish in 1.209s, 180 probes kept.\n",
      "-- check region:596 8:108735001-108765000_strand_+_gene_596, 59488 candidate probes\n",
      "--- 12145 probes passed check_dic selection.\n",
      "finish in 1.363s, 254 probes kept.\n",
      "-- check region:597 8:111235001-111265000_strand_+_gene_597, 59800 candidate probes\n",
      "--- 14840 probes passed check_dic selection.\n",
      "finish in 1.468s, 277 probes kept.\n",
      "-- check region:598 8:113735001-113765000_strand_+_gene_598, 59640 candidate probes\n",
      "--- 11169 probes passed check_dic selection.\n",
      "finish in 1.318s, 226 probes kept.\n",
      "-- check region:599 8:116235001-116265000_strand_+_gene_599, 59356 candidate probes\n",
      "--- 17227 probes passed check_dic selection.\n",
      "finish in 1.558s, 304 probes kept.\n",
      "-- check region:600 8:118735001-118765000_strand_+_gene_600, 59846 candidate probes\n",
      "--- 16373 probes passed check_dic selection.\n",
      "finish in 1.546s, 294 probes kept.\n",
      "-- check region:601 8:121235001-121265000_strand_+_gene_601, 59528 candidate probes\n",
      "--- 16894 probes passed check_dic selection.\n",
      "finish in 1.541s, 299 probes kept.\n",
      "-- check region:602 8:123735001-123765000_strand_+_gene_602, 58336 candidate probes\n",
      "--- 14617 probes passed check_dic selection.\n",
      "finish in 1.456s, 261 probes kept.\n",
      "-- check region:603 8:126235001-126265000_strand_+_gene_603, 58372 candidate probes\n",
      "--- 12673 probes passed check_dic selection.\n",
      "finish in 1.370s, 251 probes kept.\n",
      "-- check region:604 8:128735001-128765000_strand_+_gene_604, 48616 candidate probes\n",
      "--- 102 probes passed check_dic selection.\n",
      "finish in 0.766s, 6 probes kept.\n",
      "-- check region:605 8:131235001-131265000_strand_+_gene_605, 51326 candidate probes\n",
      "--- 11282 probes passed check_dic selection.\n",
      "finish in 1.222s, 187 probes kept.\n",
      "-- check region:606 8:133735001-133765000_strand_+_gene_606, 56408 candidate probes\n",
      "--- 18163 probes passed check_dic selection.\n",
      "finish in 1.550s, 293 probes kept.\n",
      "-- check region:607 8:136235001-136265000_strand_+_gene_607, 59599 candidate probes\n",
      "--- 16356 probes passed check_dic selection.\n",
      "finish in 1.534s, 281 probes kept.\n",
      "-- check region:608 8:138735001-138765000_strand_+_gene_608, 58988 candidate probes\n",
      "--- 23497 probes passed check_dic selection.\n",
      "finish in 1.801s, 355 probes kept.\n",
      "-- check region:609 8:141235001-141265000_strand_+_gene_609, 55248 candidate probes\n",
      "--- 23012 probes passed check_dic selection.\n",
      "finish in 1.744s, 323 probes kept.\n",
      "-- check region:610 8:143735001-143765000_strand_+_gene_610, 57796 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.923s, 0 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:611 9:985001-1015000_strand_+_gene_611, 58302 candidate probes\n",
      "--- 17781 probes passed check_dic selection.\n",
      "finish in 1.571s, 302 probes kept.\n",
      "-- check region:612 9:2985001-3015000_strand_+_gene_612, 59036 candidate probes\n",
      "--- 19552 probes passed check_dic selection.\n",
      "finish in 1.650s, 311 probes kept.\n",
      "-- check region:613 9:4985001-5015000_strand_+_gene_613, 58348 candidate probes\n",
      "--- 8937 probes passed check_dic selection.\n",
      "finish in 1.233s, 200 probes kept.\n",
      "-- check region:614 9:6985001-7015000_strand_+_gene_614, 58838 candidate probes\n",
      "--- 15060 probes passed check_dic selection.\n",
      "finish in 1.467s, 277 probes kept.\n",
      "-- check region:615 9:8985001-9015000_strand_+_gene_615, 59752 candidate probes\n",
      "--- 13016 probes passed check_dic selection.\n",
      "finish in 1.392s, 258 probes kept.\n",
      "-- check region:616 9:10985001-11015000_strand_+_gene_616, 58558 candidate probes\n",
      "--- 7764 probes passed check_dic selection.\n",
      "finish in 1.193s, 173 probes kept.\n",
      "-- check region:617 9:12985001-13015000_strand_+_gene_617, 59440 candidate probes\n",
      "--- 12918 probes passed check_dic selection.\n",
      "finish in 1.387s, 254 probes kept.\n",
      "-- check region:618 9:14985001-15015000_strand_+_gene_618, 58940 candidate probes\n",
      "--- 9758 probes passed check_dic selection.\n",
      "finish in 1.284s, 192 probes kept.\n",
      "-- check region:619 9:16985001-17015000_strand_+_gene_619, 59236 candidate probes\n",
      "--- 15747 probes passed check_dic selection.\n",
      "finish in 1.498s, 282 probes kept.\n",
      "-- check region:620 9:18985001-19015000_strand_+_gene_620, 56436 candidate probes\n",
      "--- 10313 probes passed check_dic selection.\n",
      "finish in 1.267s, 196 probes kept.\n",
      "-- check region:621 9:20985001-21015000_strand_+_gene_621, 59524 candidate probes\n",
      "--- 11207 probes passed check_dic selection.\n",
      "finish in 1.337s, 248 probes kept.\n",
      "-- check region:622 9:22985001-23015000_strand_+_gene_622, 58370 candidate probes\n",
      "--- 11766 probes passed check_dic selection.\n",
      "finish in 1.328s, 247 probes kept.\n",
      "-- check region:623 9:24985001-25015000_strand_+_gene_623, 59076 candidate probes\n",
      "--- 8287 probes passed check_dic selection.\n",
      "finish in 1.202s, 185 probes kept.\n",
      "-- check region:624 9:26985001-27015000_strand_+_gene_624, 57982 candidate probes\n",
      "--- 4332 probes passed check_dic selection.\n",
      "finish in 1.054s, 132 probes kept.\n",
      "-- check region:625 9:28985001-29015000_strand_+_gene_625, 52910 candidate probes\n",
      "--- 7284 probes passed check_dic selection.\n",
      "finish in 1.088s, 168 probes kept.\n",
      "-- check region:626 9:30985001-31015000_strand_+_gene_626, 56480 candidate probes\n",
      "--- 4242 probes passed check_dic selection.\n",
      "finish in 1.040s, 118 probes kept.\n",
      "-- check region:627 9:32985001-33015000_strand_+_gene_627, 56252 candidate probes\n",
      "--- 14177 probes passed check_dic selection.\n",
      "finish in 1.411s, 253 probes kept.\n",
      "-- check region:628 9:34985001-35015000_strand_+_gene_628, 57820 candidate probes\n",
      "--- 15020 probes passed check_dic selection.\n",
      "finish in 1.512s, 273 probes kept.\n",
      "-- check region:629 9:36985001-37015000_strand_+_gene_629, 59692 candidate probes\n",
      "--- 23904 probes passed check_dic selection.\n",
      "finish in 1.830s, 360 probes kept.\n",
      "-- check region:630 9:38985001-39015000_strand_+_gene_630, 58402 candidate probes\n",
      "--- 778 probes passed check_dic selection.\n",
      "finish in 0.955s, 37 probes kept.\n",
      "-- check region:631 9:40985001-41015000_strand_+_gene_631, 48250 candidate probes\n",
      "--- 314 probes passed check_dic selection.\n",
      "finish in 0.763s, 21 probes kept.\n",
      "-- check region:632 9:42985001-43015000_strand_+_gene_632, 51826 candidate probes\n",
      "--- 95 probes passed check_dic selection.\n",
      "finish in 0.807s, 9 probes kept.\n",
      "-- check region:633 9:44985001-45015000_strand_+_gene_633, 14458 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.232s, 0 probes kept.\n",
      "-- check region:634 9:46985001-47015000_strand_+_gene_634, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:635 9:48985001-49015000_strand_+_gene_635, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:636 9:50985001-51015000_strand_+_gene_636, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:637 9:52985001-53015000_strand_+_gene_637, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:638 9:54985001-55015000_strand_+_gene_638, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:639 9:56985001-57015000_strand_+_gene_639, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:640 9:58985001-59015000_strand_+_gene_640, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:641 9:60985001-61015000_strand_+_gene_641, 33884 candidate probes\n",
      "--- 156 probes passed check_dic selection.\n",
      "finish in 0.543s, 5 probes kept.\n",
      "-- check region:642 9:62985001-63015000_strand_+_gene_642, 13162 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.203s, 0 probes kept.\n",
      "-- check region:643 9:64985001-65015000_strand_+_gene_643, 25074 candidate probes\n",
      "--- 8 probes passed check_dic selection.\n",
      "finish in 0.393s, 1 probes kept.\n",
      "-- check region:644 9:66985001-67015000_strand_+_gene_644, 58532 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.934s, 0 probes kept.\n",
      "-- check region:645 9:68985001-69015000_strand_+_gene_645, 57344 candidate probes\n",
      "--- 15338 probes passed check_dic selection.\n",
      "finish in 1.477s, 257 probes kept.\n",
      "-- check region:646 9:70985001-71015000_strand_+_gene_646, 59258 candidate probes\n",
      "--- 12671 probes passed check_dic selection.\n",
      "finish in 1.389s, 241 probes kept.\n",
      "-- check region:647 9:72985001-73015000_strand_+_gene_647, 59194 candidate probes\n",
      "--- 10255 probes passed check_dic selection.\n",
      "finish in 1.279s, 219 probes kept.\n",
      "-- check region:648 9:74985001-75015000_strand_+_gene_648, 58784 candidate probes\n",
      "--- 10421 probes passed check_dic selection.\n",
      "finish in 1.295s, 215 probes kept.\n",
      "-- check region:649 9:76985001-77015000_strand_+_gene_649, 58082 candidate probes\n",
      "--- 15798 probes passed check_dic selection.\n",
      "finish in 1.489s, 262 probes kept.\n",
      "-- check region:650 9:78985001-79015000_strand_+_gene_650, 58878 candidate probes\n",
      "--- 11398 probes passed check_dic selection.\n",
      "finish in 1.320s, 241 probes kept.\n",
      "-- check region:651 9:80985001-81015000_strand_+_gene_651, 57854 candidate probes\n",
      "--- 6058 probes passed check_dic selection.\n",
      "finish in 1.164s, 146 probes kept.\n",
      "-- check region:652 9:82985001-83015000_strand_+_gene_652, 59538 candidate probes\n",
      "--- 17144 probes passed check_dic selection.\n",
      "finish in 1.561s, 306 probes kept.\n",
      "-- check region:653 9:84985001-85015000_strand_+_gene_653, 59666 candidate probes\n",
      "--- 24656 probes passed check_dic selection.\n",
      "finish in 1.864s, 373 probes kept.\n",
      "-- check region:654 9:86985001-87015000_strand_+_gene_654, 59664 candidate probes\n",
      "--- 13367 probes passed check_dic selection.\n",
      "finish in 1.423s, 271 probes kept.\n",
      "-- check region:655 9:88985001-89015000_strand_+_gene_655, 59446 candidate probes\n",
      "--- 18315 probes passed check_dic selection.\n",
      "finish in 1.664s, 401 probes kept.\n",
      "-- check region:656 9:90985001-91015000_strand_+_gene_656, 58472 candidate probes\n",
      "--- 17236 probes passed check_dic selection.\n",
      "finish in 1.544s, 292 probes kept.\n",
      "-- check region:657 9:92985001-93015000_strand_+_gene_657, 54310 candidate probes\n",
      "--- 10507 probes passed check_dic selection.\n",
      "finish in 1.258s, 186 probes kept.\n",
      "-- check region:658 9:94985001-95015000_strand_+_gene_658, 58864 candidate probes\n",
      "--- 18646 probes passed check_dic selection.\n",
      "finish in 1.597s, 311 probes kept.\n",
      "-- check region:659 9:96985001-97015000_strand_+_gene_659, 57732 candidate probes\n",
      "--- 1321 probes passed check_dic selection.\n",
      "finish in 0.962s, 54 probes kept.\n",
      "-- check region:660 9:98985001-99015000_strand_+_gene_660, 59066 candidate probes\n",
      "--- 17208 probes passed check_dic selection.\n",
      "finish in 1.577s, 278 probes kept.\n",
      "-- check region:661 9:100985001-101015000_strand_+_gene_661, 56326 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.878s, 0 probes kept.\n",
      "-- check region:662 9:102985001-103015000_strand_+_gene_662, 58904 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 6849 probes passed check_dic selection.\n",
      "finish in 1.153s, 184 probes kept.\n",
      "-- check region:663 9:104985001-105015000_strand_+_gene_663, 57592 candidate probes\n",
      "--- 11358 probes passed check_dic selection.\n",
      "finish in 1.331s, 205 probes kept.\n",
      "-- check region:664 9:106985001-107015000_strand_+_gene_664, 58432 candidate probes\n",
      "--- 13332 probes passed check_dic selection.\n",
      "finish in 1.401s, 255 probes kept.\n",
      "-- check region:665 9:108985001-109015000_strand_+_gene_665, 55922 candidate probes\n",
      "--- 14603 probes passed check_dic selection.\n",
      "finish in 1.408s, 266 probes kept.\n",
      "-- check region:666 9:110985001-111015000_strand_+_gene_666, 59410 candidate probes\n",
      "--- 13341 probes passed check_dic selection.\n",
      "finish in 1.408s, 273 probes kept.\n",
      "-- check region:667 9:112985001-113015000_strand_+_gene_667, 58482 candidate probes\n",
      "--- 6954 probes passed check_dic selection.\n",
      "finish in 1.171s, 172 probes kept.\n",
      "-- check region:668 9:114985001-115015000_strand_+_gene_668, 59146 candidate probes\n",
      "--- 20907 probes passed check_dic selection.\n",
      "finish in 1.717s, 328 probes kept.\n",
      "-- check region:669 9:116985001-117015000_strand_+_gene_669, 59744 candidate probes\n",
      "--- 17506 probes passed check_dic selection.\n",
      "finish in 1.586s, 312 probes kept.\n",
      "-- check region:670 9:118985001-119015000_strand_+_gene_670, 59286 candidate probes\n",
      "--- 17504 probes passed check_dic selection.\n",
      "finish in 1.574s, 311 probes kept.\n",
      "-- check region:671 9:120985001-121015000_strand_+_gene_671, 58408 candidate probes\n",
      "--- 9760 probes passed check_dic selection.\n",
      "finish in 1.264s, 199 probes kept.\n",
      "-- check region:672 9:122985001-123015000_strand_+_gene_672, 58572 candidate probes\n",
      "--- 6881 probes passed check_dic selection.\n",
      "finish in 1.164s, 171 probes kept.\n",
      "-- check region:673 9:124985001-125015000_strand_+_gene_673, 51506 candidate probes\n",
      "--- 5614 probes passed check_dic selection.\n",
      "finish in 1.028s, 120 probes kept.\n",
      "-- check region:674 9:126985001-127015000_strand_+_gene_674, 52094 candidate probes\n",
      "--- 16587 probes passed check_dic selection.\n",
      "finish in 1.436s, 258 probes kept.\n",
      "-- check region:675 9:128985001-129015000_strand_+_gene_675, 57862 candidate probes\n",
      "--- 17152 probes passed check_dic selection.\n",
      "finish in 1.577s, 327 probes kept.\n",
      "-- check region:676 9:130985001-131015000_strand_+_gene_676, 58506 candidate probes\n",
      "--- 22148 probes passed check_dic selection.\n",
      "finish in 1.755s, 330 probes kept.\n",
      "-- check region:677 9:132985001-133015000_strand_+_gene_677, 58040 candidate probes\n",
      "--- 19348 probes passed check_dic selection.\n",
      "finish in 1.663s, 308 probes kept.\n",
      "-- check region:678 9:134985001-135015000_strand_+_gene_678, 58590 candidate probes\n",
      "--- 24332 probes passed check_dic selection.\n",
      "finish in 1.833s, 347 probes kept.\n",
      "-- check region:679 9:136985001-137015000_strand_+_gene_679, 57460 candidate probes\n",
      "--- 16796 probes passed check_dic selection.\n",
      "finish in 1.532s, 320 probes kept.\n",
      "-- check region:680 10:985001-1015000_strand_+_gene_680, 57276 candidate probes\n",
      "--- 16217 probes passed check_dic selection.\n",
      "finish in 1.517s, 294 probes kept.\n",
      "-- check region:681 10:2985001-3015000_strand_+_gene_681, 58898 candidate probes\n",
      "--- 19585 probes passed check_dic selection.\n",
      "finish in 1.656s, 308 probes kept.\n",
      "-- check region:682 10:4985001-5015000_strand_+_gene_682, 59714 candidate probes\n",
      "--- 5166 probes passed check_dic selection.\n",
      "finish in 1.133s, 147 probes kept.\n",
      "-- check region:683 10:6985001-7015000_strand_+_gene_683, 58992 candidate probes\n",
      "--- 17639 probes passed check_dic selection.\n",
      "finish in 1.570s, 285 probes kept.\n",
      "-- check region:684 10:8985001-9015000_strand_+_gene_684, 58622 candidate probes\n",
      "--- 13481 probes passed check_dic selection.\n",
      "finish in 1.393s, 253 probes kept.\n",
      "-- check region:685 10:10985001-11015000_strand_+_gene_685, 59622 candidate probes\n",
      "--- 19918 probes passed check_dic selection.\n",
      "finish in 1.659s, 328 probes kept.\n",
      "-- check region:686 10:12985001-13015000_strand_+_gene_686, 58560 candidate probes\n",
      "--- 15810 probes passed check_dic selection.\n",
      "finish in 1.500s, 266 probes kept.\n",
      "-- check region:687 10:14985001-15015000_strand_+_gene_687, 56296 candidate probes\n",
      "--- 1778 probes passed check_dic selection.\n",
      "finish in 0.976s, 78 probes kept.\n",
      "-- check region:688 10:16985001-17015000_strand_+_gene_688, 58772 candidate probes\n",
      "--- 16248 probes passed check_dic selection.\n",
      "finish in 1.535s, 278 probes kept.\n",
      "-- check region:689 10:18985001-19015000_strand_+_gene_689, 52136 candidate probes\n",
      "--- 7588 probes passed check_dic selection.\n",
      "finish in 1.102s, 160 probes kept.\n",
      "-- check region:690 10:20985001-21015000_strand_+_gene_690, 59368 candidate probes\n",
      "--- 16381 probes passed check_dic selection.\n",
      "finish in 1.523s, 289 probes kept.\n",
      "-- check region:691 10:22985001-23015000_strand_+_gene_691, 59236 candidate probes\n",
      "--- 15827 probes passed check_dic selection.\n",
      "finish in 1.511s, 270 probes kept.\n",
      "-- check region:692 10:24985001-25015000_strand_+_gene_692, 56912 candidate probes\n",
      "--- 8981 probes passed check_dic selection.\n",
      "finish in 1.218s, 189 probes kept.\n",
      "-- check region:693 10:26985001-27015000_strand_+_gene_693, 57314 candidate probes\n",
      "--- 12806 probes passed check_dic selection.\n",
      "finish in 1.366s, 246 probes kept.\n",
      "-- check region:694 10:28985001-29015000_strand_+_gene_694, 57602 candidate probes\n",
      "--- 14624 probes passed check_dic selection.\n",
      "finish in 1.444s, 236 probes kept.\n",
      "-- check region:695 10:30985001-31015000_strand_+_gene_695, 58752 candidate probes\n",
      "--- 12652 probes passed check_dic selection.\n",
      "finish in 1.381s, 246 probes kept.\n",
      "-- check region:696 10:32985001-33015000_strand_+_gene_696, 57946 candidate probes\n",
      "--- 9196 probes passed check_dic selection.\n",
      "finish in 1.240s, 170 probes kept.\n",
      "-- check region:697 10:34985001-35015000_strand_+_gene_697, 57302 candidate probes\n",
      "--- 12472 probes passed check_dic selection.\n",
      "finish in 1.361s, 220 probes kept.\n",
      "-- check region:698 10:36985001-37015000_strand_+_gene_698, 58858 candidate probes\n",
      "--- 9130 probes passed check_dic selection.\n",
      "finish in 1.259s, 227 probes kept.\n",
      "-- check region:699 10:38985001-39015000_strand_+_gene_699, 54966 candidate probes\n",
      "--- 1946 probes passed check_dic selection.\n",
      "finish in 0.942s, 66 probes kept.\n",
      "-- check region:700 10:40985001-41015000_strand_+_gene_700, 17262 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.277s, 0 probes kept.\n",
      "-- check region:701 10:42985001-43015000_strand_+_gene_701, 59490 candidate probes\n",
      "--- 17204 probes passed check_dic selection.\n",
      "finish in 1.585s, 289 probes kept.\n",
      "-- check region:702 10:44985001-45015000_strand_+_gene_702, 52884 candidate probes\n",
      "--- 16162 probes passed check_dic selection.\n",
      "finish in 1.464s, 291 probes kept.\n",
      "-- check region:703 10:46985001-47015000_strand_+_gene_703, 56504 candidate probes\n",
      "--- 9847 probes passed check_dic selection.\n",
      "finish in 1.232s, 181 probes kept.\n",
      "-- check region:704 10:48985001-49015000_strand_+_gene_704, 57466 candidate probes\n",
      "--- 16334 probes passed check_dic selection.\n",
      "finish in 1.520s, 284 probes kept.\n",
      "-- check region:705 10:50985001-51015000_strand_+_gene_705, 59426 candidate probes\n",
      "--- 16551 probes passed check_dic selection.\n",
      "finish in 1.526s, 292 probes kept.\n",
      "-- check region:706 10:52985001-53015000_strand_+_gene_706, 59542 candidate probes\n",
      "--- 10349 probes passed check_dic selection.\n",
      "finish in 1.291s, 246 probes kept.\n",
      "-- check region:707 10:54985001-55015000_strand_+_gene_707, 58968 candidate probes\n",
      "--- 7118 probes passed check_dic selection.\n",
      "finish in 1.164s, 182 probes kept.\n",
      "-- check region:708 10:56985001-57015000_strand_+_gene_708, 57900 candidate probes\n",
      "--- 11011 probes passed check_dic selection.\n",
      "finish in 1.295s, 213 probes kept.\n",
      "-- check region:709 10:58985001-59015000_strand_+_gene_709, 59226 candidate probes\n",
      "--- 16056 probes passed check_dic selection.\n",
      "finish in 1.513s, 289 probes kept.\n",
      "-- check region:710 10:60985001-61015000_strand_+_gene_710, 57470 candidate probes\n",
      "--- 12368 probes passed check_dic selection.\n",
      "finish in 1.345s, 234 probes kept.\n",
      "-- check region:711 10:62985001-63015000_strand_+_gene_711, 59222 candidate probes\n",
      "--- 15436 probes passed check_dic selection.\n",
      "finish in 1.522s, 290 probes kept.\n",
      "-- check region:712 10:64985001-65015000_strand_+_gene_712, 59138 candidate probes\n",
      "--- 9931 probes passed check_dic selection.\n",
      "finish in 1.260s, 201 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:713 10:66985001-67015000_strand_+_gene_713, 59136 candidate probes\n",
      "--- 11383 probes passed check_dic selection.\n",
      "finish in 1.322s, 236 probes kept.\n",
      "-- check region:714 10:68985001-69015000_strand_+_gene_714, 58172 candidate probes\n",
      "--- 9358 probes passed check_dic selection.\n",
      "finish in 1.261s, 196 probes kept.\n",
      "-- check region:715 10:70985001-71015000_strand_+_gene_715, 59488 candidate probes\n",
      "--- 24131 probes passed check_dic selection.\n",
      "finish in 1.849s, 353 probes kept.\n",
      "-- check region:716 10:72985001-73015000_strand_+_gene_716, 55342 candidate probes\n",
      "--- 5170 probes passed check_dic selection.\n",
      "finish in 1.056s, 136 probes kept.\n",
      "-- check region:717 10:74985001-75015000_strand_+_gene_717, 58428 candidate probes\n",
      "--- 14826 probes passed check_dic selection.\n",
      "finish in 1.460s, 282 probes kept.\n",
      "-- check region:718 10:76985001-77015000_strand_+_gene_718, 59802 candidate probes\n",
      "--- 24600 probes passed check_dic selection.\n",
      "finish in 1.872s, 391 probes kept.\n",
      "-- check region:719 10:78985001-79015000_strand_+_gene_719, 59394 candidate probes\n",
      "--- 24352 probes passed check_dic selection.\n",
      "finish in 1.856s, 376 probes kept.\n",
      "-- check region:720 10:80985001-81015000_strand_+_gene_720, 58174 candidate probes\n",
      "--- 11806 probes passed check_dic selection.\n",
      "finish in 1.340s, 218 probes kept.\n",
      "-- check region:721 10:82985001-83015000_strand_+_gene_721, 59694 candidate probes\n",
      "--- 13924 probes passed check_dic selection.\n",
      "finish in 1.431s, 270 probes kept.\n",
      "-- check region:722 10:84985001-85015000_strand_+_gene_722, 59558 candidate probes\n",
      "--- 18092 probes passed check_dic selection.\n",
      "finish in 1.602s, 309 probes kept.\n",
      "-- check region:723 10:86985001-87015000_strand_+_gene_723, 57284 candidate probes\n",
      "--- 1851 probes passed check_dic selection.\n",
      "finish in 0.986s, 73 probes kept.\n",
      "-- check region:724 10:88985001-89015000_strand_+_gene_724, 59616 candidate probes\n",
      "--- 14502 probes passed check_dic selection.\n",
      "finish in 1.465s, 288 probes kept.\n",
      "-- check region:725 10:90985001-91015000_strand_+_gene_725, 59298 candidate probes\n",
      "--- 15469 probes passed check_dic selection.\n",
      "finish in 1.496s, 271 probes kept.\n",
      "-- check region:726 10:92985001-93015000_strand_+_gene_726, 59100 candidate probes\n",
      "--- 9080 probes passed check_dic selection.\n",
      "finish in 1.238s, 209 probes kept.\n",
      "-- check region:727 10:94985001-95015000_strand_+_gene_727, 58790 candidate probes\n",
      "--- 4922 probes passed check_dic selection.\n",
      "finish in 1.109s, 138 probes kept.\n",
      "-- check region:728 10:96985001-97015000_strand_+_gene_728, 59228 candidate probes\n",
      "--- 16797 probes passed check_dic selection.\n",
      "finish in 1.601s, 365 probes kept.\n",
      "-- check region:729 10:98985001-99015000_strand_+_gene_729, 53948 candidate probes\n",
      "--- 10135 probes passed check_dic selection.\n",
      "finish in 1.211s, 200 probes kept.\n",
      "-- check region:730 10:100985001-101015000_strand_+_gene_730, 58554 candidate probes\n",
      "--- 20086 probes passed check_dic selection.\n",
      "finish in 1.698s, 379 probes kept.\n",
      "-- check region:731 10:102985001-103015000_strand_+_gene_731, 57494 candidate probes\n",
      "--- 9173 probes passed check_dic selection.\n",
      "finish in 1.238s, 174 probes kept.\n",
      "-- check region:732 10:104985001-105015000_strand_+_gene_732, 59442 candidate probes\n",
      "--- 15676 probes passed check_dic selection.\n",
      "finish in 1.504s, 296 probes kept.\n",
      "-- check region:733 10:106985001-107015000_strand_+_gene_733, 57964 candidate probes\n",
      "--- 15393 probes passed check_dic selection.\n",
      "finish in 1.466s, 274 probes kept.\n",
      "-- check region:734 10:108985001-109015000_strand_+_gene_734, 55870 candidate probes\n",
      "--- 11633 probes passed check_dic selection.\n",
      "finish in 1.297s, 225 probes kept.\n",
      "-- check region:735 10:110985001-111015000_strand_+_gene_735, 58520 candidate probes\n",
      "--- 8803 probes passed check_dic selection.\n",
      "finish in 1.224s, 206 probes kept.\n",
      "-- check region:736 10:112985001-113015000_strand_+_gene_736, 58910 candidate probes\n",
      "--- 24659 probes passed check_dic selection.\n",
      "finish in 1.847s, 362 probes kept.\n",
      "-- check region:737 10:114985001-115015000_strand_+_gene_737, 58240 candidate probes\n",
      "--- 14856 probes passed check_dic selection.\n",
      "finish in 1.456s, 252 probes kept.\n",
      "-- check region:738 10:116985001-117015000_strand_+_gene_738, 57842 candidate probes\n",
      "--- 11235 probes passed check_dic selection.\n",
      "finish in 1.308s, 222 probes kept.\n",
      "-- check region:739 10:118985001-119015000_strand_+_gene_739, 58221 candidate probes\n",
      "--- 13436 probes passed check_dic selection.\n",
      "finish in 1.413s, 229 probes kept.\n",
      "-- check region:740 10:120985001-121015000_strand_+_gene_740, 59016 candidate probes\n",
      "--- 13317 probes passed check_dic selection.\n",
      "finish in 1.421s, 249 probes kept.\n",
      "-- check region:741 10:122985001-123015000_strand_+_gene_741, 59000 candidate probes\n",
      "--- 13316 probes passed check_dic selection.\n",
      "finish in 1.415s, 266 probes kept.\n",
      "-- check region:742 10:124985001-125015000_strand_+_gene_742, 59540 candidate probes\n",
      "--- 23122 probes passed check_dic selection.\n",
      "finish in 1.844s, 353 probes kept.\n",
      "-- check region:743 10:126985001-127015000_strand_+_gene_743, 58574 candidate probes\n",
      "--- 19516 probes passed check_dic selection.\n",
      "finish in 1.644s, 309 probes kept.\n",
      "-- check region:744 10:128985001-129015000_strand_+_gene_744, 57656 candidate probes\n",
      "--- 20176 probes passed check_dic selection.\n",
      "finish in 1.648s, 308 probes kept.\n",
      "-- check region:745 10:130985001-131015000_strand_+_gene_745, 59644 candidate probes\n",
      "--- 25898 probes passed check_dic selection.\n",
      "finish in 1.897s, 367 probes kept.\n",
      "-- check region:746 11:985001-1015000_strand_+_gene_746, 58442 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.944s, 0 probes kept.\n",
      "-- check region:747 11:2985001-3015000_strand_+_gene_747, 58292 candidate probes\n",
      "--- 655 probes passed check_dic selection.\n",
      "finish in 0.961s, 23 probes kept.\n",
      "-- check region:748 11:4985001-5015000_strand_+_gene_748, 59264 candidate probes\n",
      "--- 13069 probes passed check_dic selection.\n",
      "finish in 1.406s, 254 probes kept.\n",
      "-- check region:749 11:6985001-7015000_strand_+_gene_749, 59584 candidate probes\n",
      "--- 8405 probes passed check_dic selection.\n",
      "finish in 1.235s, 199 probes kept.\n",
      "-- check region:750 11:8985001-9015000_strand_+_gene_750, 57640 candidate probes\n",
      "--- 17277 probes passed check_dic selection.\n",
      "finish in 1.534s, 287 probes kept.\n",
      "-- check region:751 11:10985001-11015000_strand_+_gene_751, 59254 candidate probes\n",
      "--- 17511 probes passed check_dic selection.\n",
      "finish in 1.575s, 293 probes kept.\n",
      "-- check region:752 11:12985001-13015000_strand_+_gene_752, 59354 candidate probes\n",
      "--- 17723 probes passed check_dic selection.\n",
      "finish in 1.586s, 307 probes kept.\n",
      "-- check region:753 11:14985001-15015000_strand_+_gene_753, 58204 candidate probes\n",
      "--- 5896 probes passed check_dic selection.\n",
      "finish in 1.120s, 145 probes kept.\n",
      "-- check region:754 11:16985001-17015000_strand_+_gene_754, 58510 candidate probes\n",
      "--- 21750 probes passed check_dic selection.\n",
      "finish in 1.727s, 344 probes kept.\n",
      "-- check region:755 11:18985001-19015000_strand_+_gene_755, 56760 candidate probes\n",
      "--- 1584 probes passed check_dic selection.\n",
      "finish in 0.959s, 59 probes kept.\n",
      "-- check region:756 11:20985001-21015000_strand_+_gene_756, 59242 candidate probes\n",
      "--- 19074 probes passed check_dic selection.\n",
      "finish in 1.616s, 319 probes kept.\n",
      "-- check region:757 11:22985001-23015000_strand_+_gene_757, 55970 candidate probes\n",
      "--- 7886 probes passed check_dic selection.\n",
      "finish in 1.170s, 192 probes kept.\n",
      "-- check region:758 11:24985001-25015000_strand_+_gene_758, 55938 candidate probes\n",
      "--- 6178 probes passed check_dic selection.\n",
      "finish in 1.099s, 141 probes kept.\n",
      "-- check region:759 11:26985001-27015000_strand_+_gene_759, 59694 candidate probes\n",
      "--- 16300 probes passed check_dic selection.\n",
      "finish in 1.531s, 306 probes kept.\n",
      "-- check region:760 11:28985001-29015000_strand_+_gene_760, 59466 candidate probes\n",
      "--- 9687 probes passed check_dic selection.\n",
      "finish in 1.262s, 220 probes kept.\n",
      "-- check region:761 11:30985001-31015000_strand_+_gene_761, 59162 candidate probes\n",
      "--- 11361 probes passed check_dic selection.\n",
      "finish in 1.330s, 222 probes kept.\n",
      "-- check region:762 11:32985001-33015000_strand_+_gene_762, 57980 candidate probes\n",
      "--- 5871 probes passed check_dic selection.\n",
      "finish in 1.139s, 141 probes kept.\n",
      "-- check region:763 11:34985001-35015000_strand_+_gene_763, 59402 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 14029 probes passed check_dic selection.\n",
      "finish in 1.431s, 270 probes kept.\n",
      "-- check region:764 11:36985001-37015000_strand_+_gene_764, 59224 candidate probes\n",
      "--- 14174 probes passed check_dic selection.\n",
      "finish in 1.430s, 261 probes kept.\n",
      "-- check region:765 11:38985001-39015000_strand_+_gene_765, 56146 candidate probes\n",
      "--- 8819 probes passed check_dic selection.\n",
      "finish in 1.192s, 191 probes kept.\n",
      "-- check region:766 11:40985001-41015000_strand_+_gene_766, 59026 candidate probes\n",
      "--- 12936 probes passed check_dic selection.\n",
      "finish in 1.385s, 254 probes kept.\n",
      "-- check region:767 11:42985001-43015000_strand_+_gene_767, 52574 candidate probes\n",
      "--- 9651 probes passed check_dic selection.\n",
      "finish in 1.182s, 199 probes kept.\n",
      "-- check region:768 11:44985001-45015000_strand_+_gene_768, 55880 candidate probes\n",
      "--- 18220 probes passed check_dic selection.\n",
      "finish in 1.570s, 278 probes kept.\n",
      "-- check region:769 11:46985001-47015000_strand_+_gene_769, 54456 candidate probes\n",
      "--- 7673 probes passed check_dic selection.\n",
      "finish in 1.136s, 160 probes kept.\n",
      "-- check region:770 11:48985001-49015000_strand_+_gene_770, 58544 candidate probes\n",
      "--- 2150 probes passed check_dic selection.\n",
      "finish in 0.988s, 81 probes kept.\n",
      "-- check region:771 11:50985001-51015000_strand_+_gene_771, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:772 11:52985001-53015000_strand_+_gene_772, 17150 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.279s, 0 probes kept.\n",
      "-- check region:773 11:54985001-55015000_strand_+_gene_773, 58334 candidate probes\n",
      "--- 796 probes passed check_dic selection.\n",
      "finish in 0.953s, 46 probes kept.\n",
      "-- check region:774 11:56985001-57015000_strand_+_gene_774, 58692 candidate probes\n",
      "--- 7743 probes passed check_dic selection.\n",
      "finish in 1.207s, 202 probes kept.\n",
      "-- check region:775 11:58985001-59015000_strand_+_gene_775, 59228 candidate probes\n",
      "--- 7492 probes passed check_dic selection.\n",
      "finish in 1.212s, 202 probes kept.\n",
      "-- check region:776 11:60985001-61015000_strand_+_gene_776, 58014 candidate probes\n",
      "--- 20452 probes passed check_dic selection.\n",
      "finish in 1.665s, 317 probes kept.\n",
      "-- check region:777 11:62985001-63015000_strand_+_gene_777, 59040 candidate probes\n",
      "--- 20294 probes passed check_dic selection.\n",
      "finish in 1.711s, 373 probes kept.\n",
      "-- check region:778 11:64985001-65015000_strand_+_gene_778, 56550 candidate probes\n",
      "--- 15225 probes passed check_dic selection.\n",
      "finish in 1.466s, 252 probes kept.\n",
      "-- check region:779 11:66985001-67015000_strand_+_gene_779, 56500 candidate probes\n",
      "--- 11773 probes passed check_dic selection.\n",
      "finish in 1.336s, 208 probes kept.\n",
      "-- check region:780 11:68985001-69015000_strand_+_gene_780, 58790 candidate probes\n",
      "--- 21252 probes passed check_dic selection.\n",
      "finish in 1.742s, 349 probes kept.\n",
      "-- check region:781 11:70985001-71015000_strand_+_gene_781, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:782 11:72985001-73015000_strand_+_gene_782, 55200 candidate probes\n",
      "--- 10501 probes passed check_dic selection.\n",
      "finish in 1.241s, 206 probes kept.\n",
      "-- check region:783 11:74985001-75015000_strand_+_gene_783, 58832 candidate probes\n",
      "--- 15098 probes passed check_dic selection.\n",
      "finish in 1.489s, 286 probes kept.\n",
      "-- check region:784 11:76985001-77015000_strand_+_gene_784, 58832 candidate probes\n",
      "--- 8764 probes passed check_dic selection.\n",
      "finish in 1.237s, 196 probes kept.\n",
      "-- check region:785 11:78985001-79015000_strand_+_gene_785, 59296 candidate probes\n",
      "--- 17557 probes passed check_dic selection.\n",
      "finish in 1.573s, 295 probes kept.\n",
      "-- check region:786 11:80985001-81015000_strand_+_gene_786, 59696 candidate probes\n",
      "--- 11432 probes passed check_dic selection.\n",
      "finish in 1.335s, 234 probes kept.\n",
      "-- check region:787 11:82985001-83015000_strand_+_gene_787, 59090 candidate probes\n",
      "--- 16890 probes passed check_dic selection.\n",
      "finish in 1.546s, 293 probes kept.\n",
      "-- check region:788 11:84985001-85015000_strand_+_gene_788, 58364 candidate probes\n",
      "--- 8371 probes passed check_dic selection.\n",
      "finish in 1.204s, 182 probes kept.\n",
      "-- check region:789 11:86985001-87015000_strand_+_gene_789, 57600 candidate probes\n",
      "--- 10658 probes passed check_dic selection.\n",
      "finish in 1.302s, 209 probes kept.\n",
      "-- check region:790 11:88985001-89015000_strand_+_gene_790, 50500 candidate probes\n",
      "--- 2720 probes passed check_dic selection.\n",
      "finish in 0.884s, 91 probes kept.\n",
      "-- check region:791 11:90985001-91015000_strand_+_gene_791, 58256 candidate probes\n",
      "--- 6472 probes passed check_dic selection.\n",
      "finish in 1.125s, 164 probes kept.\n",
      "-- check region:792 11:92985001-93015000_strand_+_gene_792, 56370 candidate probes\n",
      "--- 8040 probes passed check_dic selection.\n",
      "finish in 1.175s, 168 probes kept.\n",
      "-- check region:793 11:94985001-95015000_strand_+_gene_793, 59377 candidate probes\n",
      "--- 11026 probes passed check_dic selection.\n",
      "finish in 1.337s, 231 probes kept.\n",
      "-- check region:794 11:96985001-97015000_strand_+_gene_794, 59492 candidate probes\n",
      "--- 15335 probes passed check_dic selection.\n",
      "finish in 1.488s, 298 probes kept.\n",
      "-- check region:795 11:98985001-99015000_strand_+_gene_795, 55680 candidate probes\n",
      "--- 5489 probes passed check_dic selection.\n",
      "finish in 1.060s, 126 probes kept.\n",
      "-- check region:796 11:100985001-101015000_strand_+_gene_796, 59050 candidate probes\n",
      "--- 15093 probes passed check_dic selection.\n",
      "finish in 1.477s, 289 probes kept.\n",
      "-- check region:797 11:102985001-103015000_strand_+_gene_797, 59512 candidate probes\n",
      "--- 12901 probes passed check_dic selection.\n",
      "finish in 1.409s, 249 probes kept.\n",
      "-- check region:798 11:104985001-105015000_strand_+_gene_798, 59742 candidate probes\n",
      "--- 10260 probes passed check_dic selection.\n",
      "finish in 1.297s, 244 probes kept.\n",
      "-- check region:799 11:106985001-107015000_strand_+_gene_799, 59180 candidate probes\n",
      "--- 16220 probes passed check_dic selection.\n",
      "finish in 1.506s, 289 probes kept.\n",
      "-- check region:800 11:108985001-109015000_strand_+_gene_800, 58707 candidate probes\n",
      "--- 10232 probes passed check_dic selection.\n",
      "finish in 1.300s, 196 probes kept.\n",
      "-- check region:801 11:110985001-111015000_strand_+_gene_801, 59620 candidate probes\n",
      "--- 17803 probes passed check_dic selection.\n",
      "finish in 1.589s, 320 probes kept.\n",
      "-- check region:802 11:112985001-113015000_strand_+_gene_802, 58526 candidate probes\n",
      "--- 13569 probes passed check_dic selection.\n",
      "finish in 1.399s, 257 probes kept.\n",
      "-- check region:803 11:114985001-115015000_strand_+_gene_803, 59880 candidate probes\n",
      "--- 17121 probes passed check_dic selection.\n",
      "finish in 1.576s, 289 probes kept.\n",
      "-- check region:804 11:116985001-117015000_strand_+_gene_804, 58776 candidate probes\n",
      "--- 11908 probes passed check_dic selection.\n",
      "finish in 1.359s, 220 probes kept.\n",
      "-- check region:805 11:118985001-119015000_strand_+_gene_805, 55404 candidate probes\n",
      "--- 7656 probes passed check_dic selection.\n",
      "finish in 1.183s, 168 probes kept.\n",
      "-- check region:806 11:120985001-121015000_strand_+_gene_806, 58954 candidate probes\n",
      "--- 15925 probes passed check_dic selection.\n",
      "finish in 1.513s, 291 probes kept.\n",
      "-- check region:807 11:122985001-123015000_strand_+_gene_807, 57380 candidate probes\n",
      "--- 12288 probes passed check_dic selection.\n",
      "finish in 1.348s, 233 probes kept.\n",
      "-- check region:808 11:124985001-125015000_strand_+_gene_808, 59206 candidate probes\n",
      "--- 8711 probes passed check_dic selection.\n",
      "finish in 1.240s, 197 probes kept.\n",
      "-- check region:809 11:126985001-127015000_strand_+_gene_809, 59906 candidate probes\n",
      "--- 25728 probes passed check_dic selection.\n",
      "finish in 1.927s, 405 probes kept.\n",
      "-- check region:810 11:128985001-129015000_strand_+_gene_810, 58772 candidate probes\n",
      "--- 6511 probes passed check_dic selection.\n",
      "finish in 1.145s, 160 probes kept.\n",
      "-- check region:811 11:130985001-131015000_strand_+_gene_811, 53680 candidate probes\n",
      "--- 13312 probes passed check_dic selection.\n",
      "finish in 1.327s, 234 probes kept.\n",
      "-- check region:812 11:132985001-133015000_strand_+_gene_812, 59558 candidate probes\n",
      "--- 23353 probes passed check_dic selection.\n",
      "finish in 1.813s, 358 probes kept.\n",
      "-- check region:813 12:985001-1015000_strand_+_gene_813, 57412 candidate probes\n",
      "--- 9561 probes passed check_dic selection.\n",
      "finish in 1.252s, 187 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:814 12:2985001-3015000_strand_+_gene_814, 57616 candidate probes\n",
      "--- 18014 probes passed check_dic selection.\n",
      "finish in 1.584s, 275 probes kept.\n",
      "-- check region:815 12:4985001-5015000_strand_+_gene_815, 58606 candidate probes\n",
      "--- 21198 probes passed check_dic selection.\n",
      "finish in 1.711s, 332 probes kept.\n",
      "-- check region:816 12:6985001-7015000_strand_+_gene_816, 55888 candidate probes\n",
      "--- 11805 probes passed check_dic selection.\n",
      "finish in 1.324s, 209 probes kept.\n",
      "-- check region:817 12:8985001-9015000_strand_+_gene_817, 58298 candidate probes\n",
      "--- 9355 probes passed check_dic selection.\n",
      "finish in 1.258s, 204 probes kept.\n",
      "-- check region:818 12:10985001-11015000_strand_+_gene_818, 59662 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.925s, 0 probes kept.\n",
      "-- check region:819 12:12985001-13015000_strand_+_gene_819, 59104 candidate probes\n",
      "--- 14508 probes passed check_dic selection.\n",
      "finish in 1.461s, 268 probes kept.\n",
      "-- check region:820 12:14985001-15015000_strand_+_gene_820, 59774 candidate probes\n",
      "--- 13800 probes passed check_dic selection.\n",
      "finish in 1.434s, 255 probes kept.\n",
      "-- check region:821 12:16985001-17015000_strand_+_gene_821, 53870 candidate probes\n",
      "--- 5701 probes passed check_dic selection.\n",
      "finish in 1.059s, 142 probes kept.\n",
      "-- check region:822 12:18985001-19015000_strand_+_gene_822, 58762 candidate probes\n",
      "--- 14919 probes passed check_dic selection.\n",
      "finish in 1.460s, 257 probes kept.\n",
      "-- check region:823 12:20985001-21015000_strand_+_gene_823, 58088 candidate probes\n",
      "--- 5722 probes passed check_dic selection.\n",
      "finish in 1.110s, 129 probes kept.\n",
      "-- check region:824 12:22985001-23015000_strand_+_gene_824, 59410 candidate probes\n",
      "--- 15742 probes passed check_dic selection.\n",
      "finish in 1.514s, 288 probes kept.\n",
      "-- check region:825 12:24985001-25015000_strand_+_gene_825, 58612 candidate probes\n",
      "--- 9910 probes passed check_dic selection.\n",
      "finish in 1.285s, 225 probes kept.\n",
      "-- check region:826 12:26985001-27015000_strand_+_gene_826, 57806 candidate probes\n",
      "--- 13023 probes passed check_dic selection.\n",
      "finish in 1.383s, 263 probes kept.\n",
      "-- check region:827 12:28985001-29015000_strand_+_gene_827, 59638 candidate probes\n",
      "--- 15388 probes passed check_dic selection.\n",
      "finish in 1.502s, 283 probes kept.\n",
      "-- check region:828 12:30985001-31015000_strand_+_gene_828, 59740 candidate probes\n",
      "--- 22934 probes passed check_dic selection.\n",
      "finish in 1.829s, 377 probes kept.\n",
      "-- check region:829 12:32985001-33015000_strand_+_gene_829, 57316 candidate probes\n",
      "--- 12425 probes passed check_dic selection.\n",
      "finish in 1.361s, 237 probes kept.\n",
      "-- check region:830 12:34985001-35015000_strand_+_gene_830, 10682 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.172s, 0 probes kept.\n",
      "-- check region:831 12:36985001-37015000_strand_+_gene_831, 19554 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.314s, 0 probes kept.\n",
      "-- check region:832 12:38985001-39015000_strand_+_gene_832, 54924 candidate probes\n",
      "--- 7497 probes passed check_dic selection.\n",
      "finish in 1.122s, 168 probes kept.\n",
      "-- check region:833 12:40985001-41015000_strand_+_gene_833, 58984 candidate probes\n",
      "--- 9233 probes passed check_dic selection.\n",
      "finish in 1.254s, 204 probes kept.\n",
      "-- check region:834 12:42985001-43015000_strand_+_gene_834, 58858 candidate probes\n",
      "--- 17164 probes passed check_dic selection.\n",
      "finish in 1.549s, 310 probes kept.\n",
      "-- check region:835 12:44985001-45015000_strand_+_gene_835, 58648 candidate probes\n",
      "--- 11650 probes passed check_dic selection.\n",
      "finish in 1.335s, 236 probes kept.\n",
      "-- check region:836 12:46985001-47015000_strand_+_gene_836, 52818 candidate probes\n",
      "--- 7349 probes passed check_dic selection.\n",
      "finish in 1.087s, 164 probes kept.\n",
      "-- check region:837 12:48985001-49015000_strand_+_gene_837, 58254 candidate probes\n",
      "--- 17667 probes passed check_dic selection.\n",
      "finish in 1.594s, 302 probes kept.\n",
      "-- check region:838 12:50985001-51015000_strand_+_gene_838, 57504 candidate probes\n",
      "--- 11374 probes passed check_dic selection.\n",
      "finish in 1.342s, 226 probes kept.\n",
      "-- check region:839 12:52985001-53015000_strand_+_gene_839, 55362 candidate probes\n",
      "--- 10306 probes passed check_dic selection.\n",
      "finish in 1.263s, 194 probes kept.\n",
      "-- check region:840 12:54985001-55015000_strand_+_gene_840, 59194 candidate probes\n",
      "--- 14043 probes passed check_dic selection.\n",
      "finish in 1.449s, 264 probes kept.\n",
      "-- check region:841 12:56985001-57015000_strand_+_gene_841, 59618 candidate probes\n",
      "--- 20008 probes passed check_dic selection.\n",
      "finish in 1.706s, 373 probes kept.\n",
      "-- check region:842 12:58985001-59015000_strand_+_gene_842, 56162 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.872s, 0 probes kept.\n",
      "-- check region:843 12:60985001-61015000_strand_+_gene_843, 53988 candidate probes\n",
      "--- 6448 probes passed check_dic selection.\n",
      "finish in 1.064s, 148 probes kept.\n",
      "-- check region:844 12:62985001-63015000_strand_+_gene_844, 59420 candidate probes\n",
      "--- 12253 probes passed check_dic selection.\n",
      "finish in 1.368s, 256 probes kept.\n",
      "-- check region:845 12:64985001-65015000_strand_+_gene_845, 59110 candidate probes\n",
      "--- 16355 probes passed check_dic selection.\n",
      "finish in 1.524s, 300 probes kept.\n",
      "-- check region:846 12:66985001-67015000_strand_+_gene_846, 58866 candidate probes\n",
      "--- 13819 probes passed check_dic selection.\n",
      "finish in 1.417s, 261 probes kept.\n",
      "-- check region:847 12:68985001-69015000_strand_+_gene_847, 52944 candidate probes\n",
      "--- 8607 probes passed check_dic selection.\n",
      "finish in 1.157s, 174 probes kept.\n",
      "-- check region:848 12:70985001-71015000_strand_+_gene_848, 58702 candidate probes\n",
      "--- 16329 probes passed check_dic selection.\n",
      "finish in 1.502s, 294 probes kept.\n",
      "-- check region:849 12:72985001-73015000_strand_+_gene_849, 58162 candidate probes\n",
      "--- 5079 probes passed check_dic selection.\n",
      "finish in 1.096s, 123 probes kept.\n",
      "-- check region:850 12:74985001-75015000_strand_+_gene_850, 55610 candidate probes\n",
      "--- 7859 probes passed check_dic selection.\n",
      "finish in 1.147s, 181 probes kept.\n",
      "-- check region:851 12:76985001-77015000_strand_+_gene_851, 59040 candidate probes\n",
      "--- 12968 probes passed check_dic selection.\n",
      "finish in 1.392s, 241 probes kept.\n",
      "-- check region:852 12:78985001-79015000_strand_+_gene_852, 58806 candidate probes\n",
      "--- 14154 probes passed check_dic selection.\n",
      "finish in 1.421s, 279 probes kept.\n",
      "-- check region:853 12:80985001-81015000_strand_+_gene_853, 56986 candidate probes\n",
      "--- 9669 probes passed check_dic selection.\n",
      "finish in 1.232s, 194 probes kept.\n",
      "-- check region:854 12:82985001-83015000_strand_+_gene_854, 59092 candidate probes\n",
      "--- 9228 probes passed check_dic selection.\n",
      "finish in 1.258s, 215 probes kept.\n",
      "-- check region:855 12:84985001-85015000_strand_+_gene_855, 59566 candidate probes\n",
      "--- 8968 probes passed check_dic selection.\n",
      "finish in 1.245s, 199 probes kept.\n",
      "-- check region:856 12:86985001-87015000_strand_+_gene_856, 59418 candidate probes\n",
      "--- 6328 probes passed check_dic selection.\n",
      "finish in 1.142s, 163 probes kept.\n",
      "-- check region:857 12:88985001-89015000_strand_+_gene_857, 59386 candidate probes\n",
      "--- 15567 probes passed check_dic selection.\n",
      "finish in 1.494s, 300 probes kept.\n",
      "-- check region:858 12:90985001-91015000_strand_+_gene_858, 59488 candidate probes\n",
      "--- 11415 probes passed check_dic selection.\n",
      "finish in 1.332s, 252 probes kept.\n",
      "-- check region:859 12:92985001-93015000_strand_+_gene_859, 58844 candidate probes\n",
      "--- 12656 probes passed check_dic selection.\n",
      "finish in 1.392s, 240 probes kept.\n",
      "-- check region:860 12:94985001-95015000_strand_+_gene_860, 58144 candidate probes\n",
      "--- 6965 probes passed check_dic selection.\n",
      "finish in 1.173s, 144 probes kept.\n",
      "-- check region:861 12:96985001-97015000_strand_+_gene_861, 59646 candidate probes\n",
      "--- 19339 probes passed check_dic selection.\n",
      "finish in 1.637s, 333 probes kept.\n",
      "-- check region:862 12:98985001-99015000_strand_+_gene_862, 59154 candidate probes\n",
      "--- 12494 probes passed check_dic selection.\n",
      "finish in 1.376s, 233 probes kept.\n",
      "-- check region:863 12:100985001-101015000_strand_+_gene_863, 59114 candidate probes\n",
      "--- 16091 probes passed check_dic selection.\n",
      "finish in 1.520s, 282 probes kept.\n",
      "-- check region:864 12:102985001-103015000_strand_+_gene_864, 59516 candidate probes\n",
      "--- 13166 probes passed check_dic selection.\n",
      "finish in 1.410s, 250 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:865 12:104985001-105015000_strand_+_gene_865, 58218 candidate probes\n",
      "--- 8145 probes passed check_dic selection.\n",
      "finish in 1.231s, 223 probes kept.\n",
      "-- check region:866 12:106985001-107015000_strand_+_gene_866, 58960 candidate probes\n",
      "--- 9151 probes passed check_dic selection.\n",
      "finish in 1.271s, 216 probes kept.\n",
      "-- check region:867 12:108985001-109015000_strand_+_gene_867, 49866 candidate probes\n",
      "--- 9620 probes passed check_dic selection.\n",
      "finish in 1.151s, 164 probes kept.\n",
      "-- check region:868 12:110985001-111015000_strand_+_gene_868, 58634 candidate probes\n",
      "--- 17830 probes passed check_dic selection.\n",
      "finish in 1.591s, 289 probes kept.\n",
      "-- check region:869 12:112985001-113015000_strand_+_gene_869, 59554 candidate probes\n",
      "--- 15892 probes passed check_dic selection.\n",
      "finish in 1.538s, 301 probes kept.\n",
      "-- check region:870 12:114985001-115015000_strand_+_gene_870, 59480 candidate probes\n",
      "--- 20547 probes passed check_dic selection.\n",
      "finish in 1.683s, 324 probes kept.\n",
      "-- check region:871 12:116985001-117015000_strand_+_gene_871, 59214 candidate probes\n",
      "--- 21254 probes passed check_dic selection.\n",
      "finish in 1.710s, 329 probes kept.\n",
      "-- check region:872 12:118985001-119015000_strand_+_gene_872, 59702 candidate probes\n",
      "--- 21898 probes passed check_dic selection.\n",
      "finish in 1.766s, 360 probes kept.\n",
      "-- check region:873 12:120985001-121015000_strand_+_gene_873, 57502 candidate probes\n",
      "--- 13716 probes passed check_dic selection.\n",
      "finish in 1.449s, 265 probes kept.\n",
      "-- check region:874 12:122985001-123015000_strand_+_gene_874, 59552 candidate probes\n",
      "--- 23039 probes passed check_dic selection.\n",
      "finish in 1.828s, 370 probes kept.\n",
      "-- check region:875 12:124985001-125015000_strand_+_gene_875, 57088 candidate probes\n",
      "--- 13122 probes passed check_dic selection.\n",
      "finish in 1.377s, 212 probes kept.\n",
      "-- check region:876 12:126985001-127015000_strand_+_gene_876, 58578 candidate probes\n",
      "--- 14792 probes passed check_dic selection.\n",
      "finish in 1.456s, 262 probes kept.\n",
      "-- check region:877 12:128985001-129015000_strand_+_gene_877, 57628 candidate probes\n",
      "--- 16059 probes passed check_dic selection.\n",
      "finish in 1.501s, 268 probes kept.\n",
      "-- check region:878 12:130985001-131015000_strand_+_gene_878, 57340 candidate probes\n",
      "--- 24179 probes passed check_dic selection.\n",
      "finish in 1.797s, 347 probes kept.\n",
      "-- check region:879 13:985001-1015000_strand_+_gene_879, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.002s, 0 probes kept.\n",
      "-- check region:880 13:2985001-3015000_strand_+_gene_880, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:881 13:4985001-5015000_strand_+_gene_881, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:882 13:6985001-7015000_strand_+_gene_882, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:883 13:8985001-9015000_strand_+_gene_883, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:884 13:10985001-11015000_strand_+_gene_884, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:885 13:12985001-13015000_strand_+_gene_885, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:886 13:14985001-15015000_strand_+_gene_886, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:887 13:16985001-17015000_strand_+_gene_887, 18844 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.303s, 0 probes kept.\n",
      "-- check region:888 13:18985001-19015000_strand_+_gene_888, 57388 candidate probes\n",
      "--- 10784 probes passed check_dic selection.\n",
      "finish in 1.315s, 215 probes kept.\n",
      "-- check region:889 13:20985001-21015000_strand_+_gene_889, 59070 candidate probes\n",
      "--- 17426 probes passed check_dic selection.\n",
      "finish in 1.570s, 279 probes kept.\n",
      "-- check region:890 13:22985001-23015000_strand_+_gene_890, 57672 candidate probes\n",
      "--- 10204 probes passed check_dic selection.\n",
      "finish in 1.276s, 205 probes kept.\n",
      "-- check region:891 13:24985001-25015000_strand_+_gene_891, 56834 candidate probes\n",
      "--- 7417 probes passed check_dic selection.\n",
      "finish in 1.180s, 169 probes kept.\n",
      "-- check region:892 13:26985001-27015000_strand_+_gene_892, 59168 candidate probes\n",
      "--- 20167 probes passed check_dic selection.\n",
      "finish in 1.680s, 314 probes kept.\n",
      "-- check region:893 13:28985001-29015000_strand_+_gene_893, 55352 candidate probes\n",
      "--- 15305 probes passed check_dic selection.\n",
      "finish in 1.438s, 252 probes kept.\n",
      "-- check region:894 13:30985001-31015000_strand_+_gene_894, 58262 candidate probes\n",
      "--- 19787 probes passed check_dic selection.\n",
      "finish in 1.630s, 308 probes kept.\n",
      "-- check region:895 13:32985001-33015000_strand_+_gene_895, 59404 candidate probes\n",
      "--- 16427 probes passed check_dic selection.\n",
      "finish in 1.533s, 294 probes kept.\n",
      "-- check region:896 13:34985001-35015000_strand_+_gene_896, 58196 candidate probes\n",
      "--- 7400 probes passed check_dic selection.\n",
      "finish in 1.184s, 158 probes kept.\n",
      "-- check region:897 13:36985001-37015000_strand_+_gene_897, 59138 candidate probes\n",
      "--- 14687 probes passed check_dic selection.\n",
      "finish in 1.464s, 286 probes kept.\n",
      "-- check region:898 13:38985001-39015000_strand_+_gene_898, 49564 candidate probes\n",
      "--- 7616 probes passed check_dic selection.\n",
      "finish in 1.057s, 182 probes kept.\n",
      "-- check region:899 13:40985001-41015000_strand_+_gene_899, 58998 candidate probes\n",
      "--- 9082 probes passed check_dic selection.\n",
      "finish in 1.243s, 211 probes kept.\n",
      "-- check region:900 13:42985001-43015000_strand_+_gene_900, 58270 candidate probes\n",
      "--- 13735 probes passed check_dic selection.\n",
      "finish in 1.423s, 251 probes kept.\n",
      "-- check region:901 13:44985001-45015000_strand_+_gene_901, 58374 candidate probes\n",
      "--- 11458 probes passed check_dic selection.\n",
      "finish in 1.338s, 230 probes kept.\n",
      "-- check region:902 13:46985001-47015000_strand_+_gene_902, 59684 candidate probes\n",
      "--- 13844 probes passed check_dic selection.\n",
      "finish in 1.439s, 271 probes kept.\n",
      "-- check region:903 13:48985001-49015000_strand_+_gene_903, 56732 candidate probes\n",
      "--- 5572 probes passed check_dic selection.\n",
      "finish in 1.073s, 144 probes kept.\n",
      "-- check region:904 13:50985001-51015000_strand_+_gene_904, 55228 candidate probes\n",
      "--- 21081 probes passed check_dic selection.\n",
      "finish in 1.645s, 320 probes kept.\n",
      "-- check region:905 13:52985001-53015000_strand_+_gene_905, 59084 candidate probes\n",
      "--- 18729 probes passed check_dic selection.\n",
      "finish in 1.621s, 311 probes kept.\n",
      "-- check region:906 13:54985001-55015000_strand_+_gene_906, 59224 candidate probes\n",
      "--- 7730 probes passed check_dic selection.\n",
      "finish in 1.198s, 194 probes kept.\n",
      "-- check region:907 13:56985001-57015000_strand_+_gene_907, 55990 candidate probes\n",
      "--- 5812 probes passed check_dic selection.\n",
      "finish in 1.061s, 143 probes kept.\n",
      "-- check region:908 13:58985001-59015000_strand_+_gene_908, 58968 candidate probes\n",
      "--- 11075 probes passed check_dic selection.\n",
      "finish in 1.317s, 216 probes kept.\n",
      "-- check region:909 13:60985001-61015000_strand_+_gene_909, 59142 candidate probes\n",
      "--- 8744 probes passed check_dic selection.\n",
      "finish in 1.243s, 181 probes kept.\n",
      "-- check region:910 13:62985001-63015000_strand_+_gene_910, 59572 candidate probes\n",
      "--- 6135 probes passed check_dic selection.\n",
      "finish in 1.148s, 155 probes kept.\n",
      "-- check region:911 13:64985001-65015000_strand_+_gene_911, 59538 candidate probes\n",
      "--- 8840 probes passed check_dic selection.\n",
      "finish in 1.225s, 187 probes kept.\n",
      "-- check region:912 13:66985001-67015000_strand_+_gene_912, 58460 candidate probes\n",
      "--- 11459 probes passed check_dic selection.\n",
      "finish in 1.312s, 230 probes kept.\n",
      "-- check region:913 13:68985001-69015000_strand_+_gene_913, 59516 candidate probes\n",
      "--- 6367 probes passed check_dic selection.\n",
      "finish in 1.144s, 169 probes kept.\n",
      "-- check region:914 13:70985001-71015000_strand_+_gene_914, 59698 candidate probes\n",
      "--- 8288 probes passed check_dic selection.\n",
      "finish in 1.214s, 176 probes kept.\n",
      "-- check region:915 13:72985001-73015000_strand_+_gene_915, 56930 candidate probes\n",
      "--- 7842 probes passed check_dic selection.\n",
      "finish in 1.177s, 166 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:916 13:74985001-75015000_strand_+_gene_916, 58368 candidate probes\n",
      "--- 12510 probes passed check_dic selection.\n",
      "finish in 1.363s, 250 probes kept.\n",
      "-- check region:917 13:76985001-77015000_strand_+_gene_917, 58920 candidate probes\n",
      "--- 8109 probes passed check_dic selection.\n",
      "finish in 1.261s, 250 probes kept.\n",
      "-- check region:918 13:78985001-79015000_strand_+_gene_918, 59502 candidate probes\n",
      "--- 13807 probes passed check_dic selection.\n",
      "finish in 1.423s, 259 probes kept.\n",
      "-- check region:919 13:80985001-81015000_strand_+_gene_919, 58502 candidate probes\n",
      "--- 5788 probes passed check_dic selection.\n",
      "finish in 1.127s, 157 probes kept.\n",
      "-- check region:920 13:82985001-83015000_strand_+_gene_920, 58926 candidate probes\n",
      "--- 8356 probes passed check_dic selection.\n",
      "finish in 1.198s, 187 probes kept.\n",
      "-- check region:921 13:84985001-85015000_strand_+_gene_921, 59250 candidate probes\n",
      "--- 7460 probes passed check_dic selection.\n",
      "finish in 1.172s, 173 probes kept.\n",
      "-- check region:922 13:86985001-87015000_strand_+_gene_922, 59534 candidate probes\n",
      "--- 7916 probes passed check_dic selection.\n",
      "finish in 1.199s, 174 probes kept.\n",
      "-- check region:923 13:88985001-89015000_strand_+_gene_923, 59682 candidate probes\n",
      "--- 5132 probes passed check_dic selection.\n",
      "finish in 1.090s, 141 probes kept.\n",
      "-- check region:924 13:90985001-91015000_strand_+_gene_924, 59060 candidate probes\n",
      "--- 11831 probes passed check_dic selection.\n",
      "finish in 1.342s, 236 probes kept.\n",
      "-- check region:925 13:92985001-93015000_strand_+_gene_925, 59556 candidate probes\n",
      "--- 10512 probes passed check_dic selection.\n",
      "finish in 1.302s, 214 probes kept.\n",
      "-- check region:926 13:94985001-95015000_strand_+_gene_926, 59078 candidate probes\n",
      "--- 16807 probes passed check_dic selection.\n",
      "finish in 1.537s, 293 probes kept.\n",
      "-- check region:927 13:96985001-97015000_strand_+_gene_927, 59754 candidate probes\n",
      "--- 17233 probes passed check_dic selection.\n",
      "finish in 1.569s, 314 probes kept.\n",
      "-- check region:928 13:98985001-99015000_strand_+_gene_928, 59354 candidate probes\n",
      "--- 20029 probes passed check_dic selection.\n",
      "finish in 1.667s, 334 probes kept.\n",
      "-- check region:929 13:100985001-101015000_strand_+_gene_929, 58822 candidate probes\n",
      "--- 15102 probes passed check_dic selection.\n",
      "finish in 1.468s, 261 probes kept.\n",
      "-- check region:930 13:102985001-103015000_strand_+_gene_930, 59748 candidate probes\n",
      "--- 14654 probes passed check_dic selection.\n",
      "finish in 1.469s, 266 probes kept.\n",
      "-- check region:931 13:104985001-105015000_strand_+_gene_931, 59566 candidate probes\n",
      "--- 12651 probes passed check_dic selection.\n",
      "finish in 1.373s, 249 probes kept.\n",
      "-- check region:932 13:106985001-107015000_strand_+_gene_932, 59598 candidate probes\n",
      "--- 17996 probes passed check_dic selection.\n",
      "finish in 1.583s, 297 probes kept.\n",
      "-- check region:933 13:108985001-109015000_strand_+_gene_933, 56530 candidate probes\n",
      "--- 12977 probes passed check_dic selection.\n",
      "finish in 1.350s, 224 probes kept.\n",
      "-- check region:934 13:110985001-111015000_strand_+_gene_934, 59304 candidate probes\n",
      "--- 11902 probes passed check_dic selection.\n",
      "finish in 1.363s, 209 probes kept.\n",
      "-- check region:935 13:112985001-113015000_strand_+_gene_935, 57404 candidate probes\n",
      "--- 32292 probes passed check_dic selection.\n",
      "finish in 2.148s, 426 probes kept.\n",
      "-- check region:936 14:735001-765000_strand_+_gene_936, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.002s, 0 probes kept.\n",
      "-- check region:937 14:2235001-2265000_strand_+_gene_937, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:938 14:3735001-3765000_strand_+_gene_938, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:939 14:5235001-5265000_strand_+_gene_939, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:940 14:6735001-6765000_strand_+_gene_940, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:941 14:8235001-8265000_strand_+_gene_941, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:942 14:9735001-9765000_strand_+_gene_942, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:943 14:11235001-11265000_strand_+_gene_943, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:944 14:12735001-12765000_strand_+_gene_944, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:945 14:14235001-14265000_strand_+_gene_945, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:946 14:15735001-15765000_strand_+_gene_946, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:947 14:17235001-17265000_strand_+_gene_947, 18830 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.300s, 0 probes kept.\n",
      "-- check region:948 14:18735001-18765000_strand_+_gene_948, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:949 14:20235001-20265000_strand_+_gene_949, 55628 candidate probes\n",
      "--- 6152 probes passed check_dic selection.\n",
      "finish in 1.102s, 160 probes kept.\n",
      "-- check region:950 14:21735001-21765000_strand_+_gene_950, 54778 candidate probes\n",
      "--- 9054 probes passed check_dic selection.\n",
      "finish in 1.193s, 178 probes kept.\n",
      "-- check region:951 14:23235001-23265000_strand_+_gene_951, 58102 candidate probes\n",
      "--- 8736 probes passed check_dic selection.\n",
      "finish in 1.235s, 188 probes kept.\n",
      "-- check region:952 14:24735001-24765000_strand_+_gene_952, 54004 candidate probes\n",
      "--- 10023 probes passed check_dic selection.\n",
      "finish in 1.218s, 190 probes kept.\n",
      "-- check region:953 14:26235001-26265000_strand_+_gene_953, 59552 candidate probes\n",
      "--- 12971 probes passed check_dic selection.\n",
      "finish in 1.395s, 245 probes kept.\n",
      "-- check region:954 14:27735001-27765000_strand_+_gene_954, 59542 candidate probes\n",
      "--- 2627 probes passed check_dic selection.\n",
      "finish in 1.019s, 90 probes kept.\n",
      "-- check region:955 14:29235001-29265000_strand_+_gene_955, 56294 candidate probes\n",
      "--- 6829 probes passed check_dic selection.\n",
      "finish in 1.117s, 174 probes kept.\n",
      "-- check region:956 14:30735001-30765000_strand_+_gene_956, 58516 candidate probes\n",
      "--- 16637 probes passed check_dic selection.\n",
      "finish in 1.518s, 265 probes kept.\n",
      "-- check region:957 14:32235001-32265000_strand_+_gene_957, 54616 candidate probes\n",
      "--- 13878 probes passed check_dic selection.\n",
      "finish in 1.389s, 258 probes kept.\n",
      "-- check region:958 14:33735001-33765000_strand_+_gene_958, 58938 candidate probes\n",
      "--- 19137 probes passed check_dic selection.\n",
      "finish in 1.620s, 328 probes kept.\n",
      "-- check region:959 14:35235001-35265000_strand_+_gene_959, 58264 candidate probes\n",
      "--- 10905 probes passed check_dic selection.\n",
      "finish in 1.314s, 213 probes kept.\n",
      "-- check region:960 14:36735001-36765000_strand_+_gene_960, 58292 candidate probes\n",
      "--- 17287 probes passed check_dic selection.\n",
      "finish in 1.546s, 279 probes kept.\n",
      "-- check region:961 14:38235001-38265000_strand_+_gene_961, 59362 candidate probes\n",
      "--- 13119 probes passed check_dic selection.\n",
      "finish in 1.411s, 237 probes kept.\n",
      "-- check region:962 14:39735001-39765000_strand_+_gene_962, 59388 candidate probes\n",
      "--- 10430 probes passed check_dic selection.\n",
      "finish in 1.297s, 226 probes kept.\n",
      "-- check region:963 14:41235001-41265000_strand_+_gene_963, 59582 candidate probes\n",
      "--- 9588 probes passed check_dic selection.\n",
      "finish in 1.275s, 195 probes kept.\n",
      "-- check region:964 14:42735001-42765000_strand_+_gene_964, 59552 candidate probes\n",
      "--- 5992 probes passed check_dic selection.\n",
      "finish in 1.130s, 149 probes kept.\n",
      "-- check region:965 14:44235001-44265000_strand_+_gene_965, 59194 candidate probes\n",
      "--- 8340 probes passed check_dic selection.\n",
      "finish in 1.210s, 191 probes kept.\n",
      "-- check region:966 14:45735001-45765000_strand_+_gene_966, 58030 candidate probes\n",
      "--- 6482 probes passed check_dic selection.\n",
      "finish in 1.140s, 161 probes kept.\n",
      "-- check region:967 14:47235001-47265000_strand_+_gene_967, 59204 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 10562 probes passed check_dic selection.\n",
      "finish in 1.316s, 229 probes kept.\n",
      "-- check region:968 14:48735001-48765000_strand_+_gene_968, 59742 candidate probes\n",
      "--- 11408 probes passed check_dic selection.\n",
      "finish in 1.343s, 228 probes kept.\n",
      "-- check region:969 14:50235001-50265000_strand_+_gene_969, 56888 candidate probes\n",
      "--- 6524 probes passed check_dic selection.\n",
      "finish in 1.136s, 156 probes kept.\n",
      "-- check region:970 14:51735001-51765000_strand_+_gene_970, 59320 candidate probes\n",
      "--- 3847 probes passed check_dic selection.\n",
      "finish in 1.096s, 110 probes kept.\n",
      "-- check region:971 14:53235001-53265000_strand_+_gene_971, 59384 candidate probes\n",
      "--- 9337 probes passed check_dic selection.\n",
      "finish in 1.271s, 173 probes kept.\n",
      "-- check region:972 14:54735001-54765000_strand_+_gene_972, 59426 candidate probes\n",
      "--- 26923 probes passed check_dic selection.\n",
      "finish in 1.954s, 392 probes kept.\n",
      "-- check region:973 14:56235001-56265000_strand_+_gene_973, 59256 candidate probes\n",
      "--- 15202 probes passed check_dic selection.\n",
      "finish in 1.473s, 270 probes kept.\n",
      "-- check region:974 14:57735001-57765000_strand_+_gene_974, 56600 candidate probes\n",
      "--- 11856 probes passed check_dic selection.\n",
      "finish in 1.318s, 209 probes kept.\n",
      "-- check region:975 14:59235001-59265000_strand_+_gene_975, 58950 candidate probes\n",
      "--- 17523 probes passed check_dic selection.\n",
      "finish in 1.563s, 315 probes kept.\n",
      "-- check region:976 14:60735001-60765000_strand_+_gene_976, 58238 candidate probes\n",
      "--- 8622 probes passed check_dic selection.\n",
      "finish in 1.215s, 194 probes kept.\n",
      "-- check region:977 14:62235001-62265000_strand_+_gene_977, 59486 candidate probes\n",
      "--- 16162 probes passed check_dic selection.\n",
      "finish in 1.525s, 297 probes kept.\n",
      "-- check region:978 14:63735001-63765000_strand_+_gene_978, 55934 candidate probes\n",
      "--- 5197 probes passed check_dic selection.\n",
      "finish in 1.072s, 130 probes kept.\n",
      "-- check region:979 14:65235001-65265000_strand_+_gene_979, 57636 candidate probes\n",
      "--- 14862 probes passed check_dic selection.\n",
      "finish in 1.463s, 255 probes kept.\n",
      "-- check region:980 14:66735001-66765000_strand_+_gene_980, 58796 candidate probes\n",
      "--- 6549 probes passed check_dic selection.\n",
      "finish in 1.144s, 173 probes kept.\n",
      "-- check region:981 14:68235001-68265000_strand_+_gene_981, 59020 candidate probes\n",
      "--- 17812 probes passed check_dic selection.\n",
      "finish in 1.576s, 302 probes kept.\n",
      "-- check region:982 14:69735001-69765000_strand_+_gene_982, 57262 candidate probes\n",
      "--- 9839 probes passed check_dic selection.\n",
      "finish in 1.265s, 171 probes kept.\n",
      "-- check region:983 14:71235001-71265000_strand_+_gene_983, 57784 candidate probes\n",
      "--- 14720 probes passed check_dic selection.\n",
      "finish in 1.454s, 236 probes kept.\n",
      "-- check region:984 14:72735001-72765000_strand_+_gene_984, 59518 candidate probes\n",
      "--- 17910 probes passed check_dic selection.\n",
      "finish in 1.606s, 269 probes kept.\n",
      "-- check region:985 14:74235001-74265000_strand_+_gene_985, 58210 candidate probes\n",
      "--- 23978 probes passed check_dic selection.\n",
      "finish in 1.821s, 342 probes kept.\n",
      "-- check region:986 14:75735001-75765000_strand_+_gene_986, 59016 candidate probes\n",
      "--- 11477 probes passed check_dic selection.\n",
      "finish in 1.337s, 233 probes kept.\n",
      "-- check region:987 14:77235001-77265000_strand_+_gene_987, 59074 candidate probes\n",
      "--- 24179 probes passed check_dic selection.\n",
      "finish in 1.855s, 374 probes kept.\n",
      "-- check region:988 14:78735001-78765000_strand_+_gene_988, 59596 candidate probes\n",
      "--- 20464 probes passed check_dic selection.\n",
      "finish in 1.683s, 348 probes kept.\n",
      "-- check region:989 14:80235001-80265000_strand_+_gene_989, 55414 candidate probes\n",
      "--- 7318 probes passed check_dic selection.\n",
      "finish in 1.131s, 154 probes kept.\n",
      "-- check region:990 14:81735001-81765000_strand_+_gene_990, 59612 candidate probes\n",
      "--- 13638 probes passed check_dic selection.\n",
      "finish in 1.420s, 281 probes kept.\n",
      "-- check region:991 14:83235001-83265000_strand_+_gene_991, 59444 candidate probes\n",
      "--- 11090 probes passed check_dic selection.\n",
      "finish in 1.320s, 247 probes kept.\n",
      "-- check region:992 14:84735001-84765000_strand_+_gene_992, 59438 candidate probes\n",
      "--- 11365 probes passed check_dic selection.\n",
      "finish in 1.333s, 237 probes kept.\n",
      "-- check region:993 14:86235001-86265000_strand_+_gene_993, 56026 candidate probes\n",
      "--- 7062 probes passed check_dic selection.\n",
      "finish in 1.112s, 153 probes kept.\n",
      "-- check region:994 14:87735001-87765000_strand_+_gene_994, 59232 candidate probes\n",
      "--- 15416 probes passed check_dic selection.\n",
      "finish in 1.511s, 286 probes kept.\n",
      "-- check region:995 14:89235001-89265000_strand_+_gene_995, 58874 candidate probes\n",
      "--- 22293 probes passed check_dic selection.\n",
      "finish in 1.769s, 337 probes kept.\n",
      "-- check region:996 14:90735001-90765000_strand_+_gene_996, 58914 candidate probes\n",
      "--- 15697 probes passed check_dic selection.\n",
      "finish in 1.496s, 255 probes kept.\n",
      "-- check region:997 14:92235001-92265000_strand_+_gene_997, 59290 candidate probes\n",
      "--- 19992 probes passed check_dic selection.\n",
      "finish in 1.677s, 304 probes kept.\n",
      "-- check region:998 14:93735001-93765000_strand_+_gene_998, 59220 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.973s, 0 probes kept.\n",
      "-- check region:999 14:95235001-95265000_strand_+_gene_999, 58842 candidate probes\n",
      "--- 22857 probes passed check_dic selection.\n",
      "finish in 1.783s, 341 probes kept.\n",
      "-- check region:1000 14:96735001-96765000_strand_+_gene_1000, 59126 candidate probes\n",
      "--- 21089 probes passed check_dic selection.\n",
      "finish in 1.710s, 326 probes kept.\n",
      "-- check region:1001 14:98235001-98265000_strand_+_gene_1001, 58978 candidate probes\n",
      "--- 21920 probes passed check_dic selection.\n",
      "finish in 1.745s, 341 probes kept.\n",
      "-- check region:1002 14:99735001-99765000_strand_+_gene_1002, 59442 candidate probes\n",
      "--- 26380 probes passed check_dic selection.\n",
      "finish in 1.939s, 381 probes kept.\n",
      "-- check region:1003 14:101235001-101265000_strand_+_gene_1003, 58358 candidate probes\n",
      "--- 24596 probes passed check_dic selection.\n",
      "finish in 1.834s, 356 probes kept.\n",
      "-- check region:1004 14:102735001-102765000_strand_+_gene_1004, 56204 candidate probes\n",
      "--- 20535 probes passed check_dic selection.\n",
      "finish in 1.650s, 283 probes kept.\n",
      "-- check region:1005 14:104235001-104265000_strand_+_gene_1005, 57338 candidate probes\n",
      "--- 19117 probes passed check_dic selection.\n",
      "finish in 1.613s, 321 probes kept.\n",
      "-- check region:1006 14:105735001-105765000_strand_+_gene_1006, 56982 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.930s, 0 probes kept.\n",
      "-- check region:1007 15:735001-765000_strand_+_gene_1007, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1008 15:2235001-2265000_strand_+_gene_1008, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1009 15:3735001-3765000_strand_+_gene_1009, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1010 15:5235001-5265000_strand_+_gene_1010, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1011 15:6735001-6765000_strand_+_gene_1011, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1012 15:8235001-8265000_strand_+_gene_1012, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1013 15:9735001-9765000_strand_+_gene_1013, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1014 15:11235001-11265000_strand_+_gene_1014, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1015 15:12735001-12765000_strand_+_gene_1015, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1016 15:14235001-14265000_strand_+_gene_1016, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1017 15:15735001-15765000_strand_+_gene_1017, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1018 15:17235001-17265000_strand_+_gene_1018, 20232 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.319s, 0 probes kept.\n",
      "-- check region:1019 15:18735001-18765000_strand_+_gene_1019, 9492 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.153s, 0 probes kept.\n",
      "-- check region:1020 15:20235001-20265000_strand_+_gene_1020, 57112 candidate probes\n",
      "--- 1290 probes passed check_dic selection.\n",
      "finish in 0.985s, 52 probes kept.\n",
      "-- check region:1021 15:21735001-21765000_strand_+_gene_1021, 58998 candidate probes\n",
      "--- 176 probes passed check_dic selection.\n",
      "finish in 0.948s, 11 probes kept.\n",
      "-- check region:1022 15:23235001-23265000_strand_+_gene_1022, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1023 15:24735001-24765000_strand_+_gene_1023, 59194 candidate probes\n",
      "--- 10244 probes passed check_dic selection.\n",
      "finish in 1.302s, 217 probes kept.\n",
      "-- check region:1024 15:26235001-26265000_strand_+_gene_1024, 58096 candidate probes\n",
      "--- 14110 probes passed check_dic selection.\n",
      "finish in 1.429s, 242 probes kept.\n",
      "-- check region:1025 15:27735001-27765000_strand_+_gene_1025, 59374 candidate probes\n",
      "--- 18759 probes passed check_dic selection.\n",
      "finish in 1.630s, 306 probes kept.\n",
      "-- check region:1026 15:29235001-29265000_strand_+_gene_1026, 58654 candidate probes\n",
      "--- 354 probes passed check_dic selection.\n",
      "finish in 0.940s, 17 probes kept.\n",
      "-- check region:1027 15:30735001-30765000_strand_+_gene_1027, 59116 candidate probes\n",
      "--- 64 probes passed check_dic selection.\n",
      "finish in 0.945s, 3 probes kept.\n",
      "-- check region:1028 15:32235001-32265000_strand_+_gene_1028, 58334 candidate probes\n",
      "--- 20 probes passed check_dic selection.\n",
      "finish in 0.915s, 1 probes kept.\n",
      "-- check region:1029 15:33735001-33765000_strand_+_gene_1029, 57146 candidate probes\n",
      "--- 16079 probes passed check_dic selection.\n",
      "finish in 1.518s, 235 probes kept.\n",
      "-- check region:1030 15:35235001-35265000_strand_+_gene_1030, 56208 candidate probes\n",
      "--- 10526 probes passed check_dic selection.\n",
      "finish in 1.262s, 209 probes kept.\n",
      "-- check region:1031 15:36735001-36765000_strand_+_gene_1031, 58514 candidate probes\n",
      "--- 16434 probes passed check_dic selection.\n",
      "finish in 1.508s, 290 probes kept.\n",
      "-- check region:1032 15:38235001-38265000_strand_+_gene_1032, 59318 candidate probes\n",
      "--- 13933 probes passed check_dic selection.\n",
      "finish in 1.415s, 275 probes kept.\n",
      "-- check region:1033 15:39735001-39765000_strand_+_gene_1033, 59910 candidate probes\n",
      "--- 10895 probes passed check_dic selection.\n",
      "finish in 1.324s, 234 probes kept.\n",
      "-- check region:1034 15:41235001-41265000_strand_+_gene_1034, 56752 candidate probes\n",
      "--- 10290 probes passed check_dic selection.\n",
      "finish in 1.276s, 183 probes kept.\n",
      "-- check region:1035 15:42735001-42765000_strand_+_gene_1035, 58436 candidate probes\n",
      "--- 9185 probes passed check_dic selection.\n",
      "finish in 1.270s, 201 probes kept.\n",
      "-- check region:1036 15:44235001-44265000_strand_+_gene_1036, 48536 candidate probes\n",
      "--- 6845 probes passed check_dic selection.\n",
      "finish in 1.018s, 130 probes kept.\n",
      "-- check region:1037 15:45735001-45765000_strand_+_gene_1037, 56914 candidate probes\n",
      "--- 12719 probes passed check_dic selection.\n",
      "finish in 1.363s, 237 probes kept.\n",
      "-- check region:1038 15:47235001-47265000_strand_+_gene_1038, 59030 candidate probes\n",
      "--- 11325 probes passed check_dic selection.\n",
      "finish in 1.327s, 227 probes kept.\n",
      "-- check region:1039 15:48735001-48765000_strand_+_gene_1039, 59338 candidate probes\n",
      "--- 11135 probes passed check_dic selection.\n",
      "finish in 1.329s, 246 probes kept.\n",
      "-- check region:1040 15:50235001-50265000_strand_+_gene_1040, 59072 candidate probes\n",
      "--- 20458 probes passed check_dic selection.\n",
      "finish in 1.690s, 350 probes kept.\n",
      "-- check region:1041 15:51735001-51765000_strand_+_gene_1041, 59178 candidate probes\n",
      "--- 13355 probes passed check_dic selection.\n",
      "finish in 1.395s, 254 probes kept.\n",
      "-- check region:1042 15:53235001-53265000_strand_+_gene_1042, 59444 candidate probes\n",
      "--- 19562 probes passed check_dic selection.\n",
      "finish in 1.644s, 325 probes kept.\n",
      "-- check region:1043 15:54735001-54765000_strand_+_gene_1043, 56686 candidate probes\n",
      "--- 5321 probes passed check_dic selection.\n",
      "finish in 1.081s, 122 probes kept.\n",
      "-- check region:1044 15:56235001-56265000_strand_+_gene_1044, 59420 candidate probes\n",
      "--- 13851 probes passed check_dic selection.\n",
      "finish in 1.413s, 253 probes kept.\n",
      "-- check region:1045 15:57735001-57765000_strand_+_gene_1045, 59656 candidate probes\n",
      "--- 22466 probes passed check_dic selection.\n",
      "finish in 1.776s, 347 probes kept.\n",
      "-- check region:1046 15:59235001-59265000_strand_+_gene_1046, 58744 candidate probes\n",
      "--- 14904 probes passed check_dic selection.\n",
      "finish in 1.467s, 263 probes kept.\n",
      "-- check region:1047 15:60735001-60765000_strand_+_gene_1047, 59146 candidate probes\n",
      "--- 22824 probes passed check_dic selection.\n",
      "finish in 1.774s, 350 probes kept.\n",
      "-- check region:1048 15:62235001-62265000_strand_+_gene_1048, 57960 candidate probes\n",
      "--- 12330 probes passed check_dic selection.\n",
      "finish in 1.391s, 296 probes kept.\n",
      "-- check region:1049 15:63735001-63765000_strand_+_gene_1049, 58796 candidate probes\n",
      "--- 8083 probes passed check_dic selection.\n",
      "finish in 1.207s, 176 probes kept.\n",
      "-- check region:1050 15:65235001-65265000_strand_+_gene_1050, 56824 candidate probes\n",
      "--- 17031 probes passed check_dic selection.\n",
      "finish in 1.526s, 267 probes kept.\n",
      "-- check region:1051 15:66735001-66765000_strand_+_gene_1051, 59416 candidate probes\n",
      "--- 26041 probes passed check_dic selection.\n",
      "finish in 1.912s, 384 probes kept.\n",
      "-- check region:1052 15:68235001-68265000_strand_+_gene_1052, 57294 candidate probes\n",
      "--- 5566 probes passed check_dic selection.\n",
      "finish in 1.092s, 135 probes kept.\n",
      "-- check region:1053 15:69735001-69765000_strand_+_gene_1053, 59552 candidate probes\n",
      "--- 23263 probes passed check_dic selection.\n",
      "finish in 1.812s, 351 probes kept.\n",
      "-- check region:1054 15:71235001-71265000_strand_+_gene_1054, 58938 candidate probes\n",
      "--- 20713 probes passed check_dic selection.\n",
      "finish in 1.696s, 320 probes kept.\n",
      "-- check region:1055 15:72735001-72765000_strand_+_gene_1055, 58468 candidate probes\n",
      "--- 12035 probes passed check_dic selection.\n",
      "finish in 1.381s, 268 probes kept.\n",
      "-- check region:1056 15:74235001-74265000_strand_+_gene_1056, 59028 candidate probes\n",
      "--- 22794 probes passed check_dic selection.\n",
      "finish in 1.788s, 361 probes kept.\n",
      "-- check region:1057 15:75735001-75765000_strand_+_gene_1057, 57102 candidate probes\n",
      "--- 5366 probes passed check_dic selection.\n",
      "finish in 1.116s, 155 probes kept.\n",
      "-- check region:1058 15:77235001-77265000_strand_+_gene_1058, 53196 candidate probes\n",
      "--- 5149 probes passed check_dic selection.\n",
      "finish in 1.014s, 131 probes kept.\n",
      "-- check region:1059 15:78735001-78765000_strand_+_gene_1059, 59200 candidate probes\n",
      "--- 5507 probes passed check_dic selection.\n",
      "finish in 1.169s, 164 probes kept.\n",
      "-- check region:1060 15:80235001-80265000_strand_+_gene_1060, 59040 candidate probes\n",
      "--- 20713 probes passed check_dic selection.\n",
      "finish in 1.698s, 315 probes kept.\n",
      "-- check region:1061 15:81735001-81765000_strand_+_gene_1061, 56942 candidate probes\n",
      "--- 20454 probes passed check_dic selection.\n",
      "finish in 1.638s, 309 probes kept.\n",
      "-- check region:1062 15:83235001-83265000_strand_+_gene_1062, 58602 candidate probes\n",
      "--- 12134 probes passed check_dic selection.\n",
      "finish in 1.376s, 240 probes kept.\n",
      "-- check region:1063 15:84735001-84765000_strand_+_gene_1063, 58352 candidate probes\n",
      "--- 15700 probes passed check_dic selection.\n",
      "finish in 1.481s, 266 probes kept.\n",
      "-- check region:1064 15:86235001-86265000_strand_+_gene_1064, 59552 candidate probes\n",
      "--- 18547 probes passed check_dic selection.\n",
      "finish in 1.620s, 327 probes kept.\n",
      "-- check region:1065 15:87735001-87765000_strand_+_gene_1065, 59450 candidate probes\n",
      "--- 19490 probes passed check_dic selection.\n",
      "finish in 1.644s, 318 probes kept.\n",
      "-- check region:1066 15:89235001-89265000_strand_+_gene_1066, 58344 candidate probes\n",
      "--- 10108 probes passed check_dic selection.\n",
      "finish in 1.287s, 207 probes kept.\n",
      "-- check region:1067 15:90735001-90765000_strand_+_gene_1067, 58724 candidate probes\n",
      "--- 9631 probes passed check_dic selection.\n",
      "finish in 1.270s, 204 probes kept.\n",
      "-- check region:1068 15:92235001-92265000_strand_+_gene_1068, 55828 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 6624 probes passed check_dic selection.\n",
      "finish in 1.112s, 147 probes kept.\n",
      "-- check region:1069 15:93735001-93765000_strand_+_gene_1069, 57714 candidate probes\n",
      "--- 7185 probes passed check_dic selection.\n",
      "finish in 1.170s, 160 probes kept.\n",
      "-- check region:1070 15:95235001-95265000_strand_+_gene_1070, 59286 candidate probes\n",
      "--- 12443 probes passed check_dic selection.\n",
      "finish in 1.378s, 250 probes kept.\n",
      "-- check region:1071 15:96735001-96765000_strand_+_gene_1071, 58072 candidate probes\n",
      "--- 16991 probes passed check_dic selection.\n",
      "finish in 1.534s, 291 probes kept.\n",
      "-- check region:1072 15:98235001-98265000_strand_+_gene_1072, 59102 candidate probes\n",
      "--- 13463 probes passed check_dic selection.\n",
      "finish in 1.412s, 259 probes kept.\n",
      "-- check region:1073 15:99735001-99765000_strand_+_gene_1073, 58492 candidate probes\n",
      "--- 16026 probes passed check_dic selection.\n",
      "finish in 1.503s, 263 probes kept.\n",
      "-- check region:1074 16:735001-765000_strand_+_gene_1074, 55932 candidate probes\n",
      "--- 15767 probes passed check_dic selection.\n",
      "finish in 1.497s, 273 probes kept.\n",
      "-- check region:1075 16:2235001-2265000_strand_+_gene_1075, 57606 candidate probes\n",
      "--- 13675 probes passed check_dic selection.\n",
      "finish in 1.428s, 256 probes kept.\n",
      "-- check region:1076 16:3735001-3765000_strand_+_gene_1076, 57836 candidate probes\n",
      "--- 17336 probes passed check_dic selection.\n",
      "finish in 1.569s, 293 probes kept.\n",
      "-- check region:1077 16:5235001-5265000_strand_+_gene_1077, 59308 candidate probes\n",
      "--- 3292 probes passed check_dic selection.\n",
      "finish in 1.075s, 88 probes kept.\n",
      "-- check region:1078 16:6735001-6765000_strand_+_gene_1078, 58632 candidate probes\n",
      "--- 13331 probes passed check_dic selection.\n",
      "finish in 1.395s, 241 probes kept.\n",
      "-- check region:1079 16:8235001-8265000_strand_+_gene_1079, 59702 candidate probes\n",
      "--- 17077 probes passed check_dic selection.\n",
      "finish in 1.558s, 302 probes kept.\n",
      "-- check region:1080 16:9735001-9765000_strand_+_gene_1080, 59256 candidate probes\n",
      "--- 17743 probes passed check_dic selection.\n",
      "finish in 1.621s, 362 probes kept.\n",
      "-- check region:1081 16:11235001-11265000_strand_+_gene_1081, 58996 candidate probes\n",
      "--- 18797 probes passed check_dic selection.\n",
      "finish in 1.617s, 298 probes kept.\n",
      "-- check region:1082 16:12735001-12765000_strand_+_gene_1082, 58850 candidate probes\n",
      "--- 13954 probes passed check_dic selection.\n",
      "finish in 1.448s, 263 probes kept.\n",
      "-- check region:1083 16:14235001-14265000_strand_+_gene_1083, 59282 candidate probes\n",
      "--- 17516 probes passed check_dic selection.\n",
      "finish in 1.590s, 325 probes kept.\n",
      "-- check region:1084 16:15735001-15765000_strand_+_gene_1084, 58414 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.959s, 0 probes kept.\n",
      "-- check region:1085 16:17235001-17265000_strand_+_gene_1085, 59198 candidate probes\n",
      "--- 23024 probes passed check_dic selection.\n",
      "finish in 1.785s, 341 probes kept.\n",
      "-- check region:1086 16:18735001-18765000_strand_+_gene_1086, 57596 candidate probes\n",
      "--- 668 probes passed check_dic selection.\n",
      "finish in 0.965s, 28 probes kept.\n",
      "-- check region:1087 16:20235001-20265000_strand_+_gene_1087, 58992 candidate probes\n",
      "--- 20531 probes passed check_dic selection.\n",
      "finish in 1.701s, 336 probes kept.\n",
      "-- check region:1088 16:21735001-21765000_strand_+_gene_1088, 58806 candidate probes\n",
      "--- 688 probes passed check_dic selection.\n",
      "finish in 0.978s, 25 probes kept.\n",
      "-- check region:1089 16:23235001-23265000_strand_+_gene_1089, 58860 candidate probes\n",
      "--- 12590 probes passed check_dic selection.\n",
      "finish in 1.393s, 207 probes kept.\n",
      "-- check region:1090 16:24735001-24765000_strand_+_gene_1090, 59486 candidate probes\n",
      "--- 14238 probes passed check_dic selection.\n",
      "finish in 1.450s, 261 probes kept.\n",
      "-- check region:1091 16:26235001-26265000_strand_+_gene_1091, 59670 candidate probes\n",
      "--- 11069 probes passed check_dic selection.\n",
      "finish in 1.339s, 226 probes kept.\n",
      "-- check region:1092 16:27735001-27765000_strand_+_gene_1092, 59368 candidate probes\n",
      "--- 22538 probes passed check_dic selection.\n",
      "finish in 1.821s, 355 probes kept.\n",
      "-- check region:1093 16:29235001-29265000_strand_+_gene_1093, 58836 candidate probes\n",
      "--- 8098 probes passed check_dic selection.\n",
      "finish in 1.244s, 203 probes kept.\n",
      "-- check region:1094 16:30735001-30765000_strand_+_gene_1094, 57978 candidate probes\n",
      "--- 15674 probes passed check_dic selection.\n",
      "finish in 1.534s, 317 probes kept.\n",
      "-- check region:1095 16:32235001-32265000_strand_+_gene_1095, 59838 candidate probes\n",
      "--- 14 probes passed check_dic selection.\n",
      "finish in 0.955s, 2 probes kept.\n",
      "-- check region:1096 16:33735001-33765000_strand_+_gene_1096, 57806 candidate probes\n",
      "--- 248 probes passed check_dic selection.\n",
      "finish in 0.911s, 14 probes kept.\n",
      "-- check region:1097 16:35235001-35265000_strand_+_gene_1097, 59852 candidate probes\n",
      "--- 9304 probes passed check_dic selection.\n",
      "finish in 1.286s, 230 probes kept.\n",
      "-- check region:1098 16:36735001-36765000_strand_+_gene_1098, 8216 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.133s, 0 probes kept.\n",
      "-- check region:1099 16:38235001-38265000_strand_+_gene_1099, 9408 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.153s, 0 probes kept.\n",
      "-- check region:1100 16:39735001-39765000_strand_+_gene_1100, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1101 16:41235001-41265000_strand_+_gene_1101, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1102 16:42735001-42765000_strand_+_gene_1102, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1103 16:44235001-44265000_strand_+_gene_1103, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1104 16:45735001-45765000_strand_+_gene_1104, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1105 16:47235001-47265000_strand_+_gene_1105, 58574 candidate probes\n",
      "--- 11166 probes passed check_dic selection.\n",
      "finish in 1.319s, 223 probes kept.\n",
      "-- check region:1106 16:48735001-48765000_strand_+_gene_1106, 57616 candidate probes\n",
      "--- 10715 probes passed check_dic selection.\n",
      "finish in 1.305s, 189 probes kept.\n",
      "-- check region:1107 16:50235001-50265000_strand_+_gene_1107, 58822 candidate probes\n",
      "--- 17599 probes passed check_dic selection.\n",
      "finish in 1.579s, 267 probes kept.\n",
      "-- check region:1108 16:51735001-51765000_strand_+_gene_1108, 59908 candidate probes\n",
      "--- 17933 probes passed check_dic selection.\n",
      "finish in 1.581s, 295 probes kept.\n",
      "-- check region:1109 16:53235001-53265000_strand_+_gene_1109, 59266 candidate probes\n",
      "--- 7204 probes passed check_dic selection.\n",
      "finish in 1.183s, 184 probes kept.\n",
      "-- check region:1110 16:54735001-54765000_strand_+_gene_1110, 59154 candidate probes\n",
      "--- 20774 probes passed check_dic selection.\n",
      "finish in 1.691s, 336 probes kept.\n",
      "-- check region:1111 16:56235001-56265000_strand_+_gene_1111, 59834 candidate probes\n",
      "--- 26866 probes passed check_dic selection.\n",
      "finish in 1.934s, 381 probes kept.\n",
      "-- check region:1112 16:57735001-57765000_strand_+_gene_1112, 58976 candidate probes\n",
      "--- 24770 probes passed check_dic selection.\n",
      "finish in 1.876s, 398 probes kept.\n",
      "-- check region:1113 16:59235001-59265000_strand_+_gene_1113, 59218 candidate probes\n",
      "--- 13972 probes passed check_dic selection.\n",
      "finish in 1.430s, 266 probes kept.\n",
      "-- check region:1114 16:60735001-60765000_strand_+_gene_1114, 59284 candidate probes\n",
      "--- 10830 probes passed check_dic selection.\n",
      "finish in 1.314s, 217 probes kept.\n",
      "-- check region:1115 16:62235001-62265000_strand_+_gene_1115, 59560 candidate probes\n",
      "--- 12400 probes passed check_dic selection.\n",
      "finish in 1.376s, 253 probes kept.\n",
      "-- check region:1116 16:63735001-63765000_strand_+_gene_1116, 59230 candidate probes\n",
      "--- 7748 probes passed check_dic selection.\n",
      "finish in 1.201s, 184 probes kept.\n",
      "-- check region:1117 16:65235001-65265000_strand_+_gene_1117, 55438 candidate probes\n",
      "--- 16117 probes passed check_dic selection.\n",
      "finish in 1.454s, 288 probes kept.\n",
      "-- check region:1118 16:66735001-66765000_strand_+_gene_1118, 57134 candidate probes\n",
      "--- 11571 probes passed check_dic selection.\n",
      "finish in 1.329s, 231 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1119 16:68235001-68265000_strand_+_gene_1119, 58084 candidate probes\n",
      "--- 21599 probes passed check_dic selection.\n",
      "finish in 1.730s, 328 probes kept.\n",
      "-- check region:1120 16:69735001-69765000_strand_+_gene_1120, 57914 candidate probes\n",
      "--- 12618 probes passed check_dic selection.\n",
      "finish in 1.393s, 228 probes kept.\n",
      "-- check region:1121 16:71235001-71265000_strand_+_gene_1121, 59502 candidate probes\n",
      "--- 17116 probes passed check_dic selection.\n",
      "finish in 1.564s, 298 probes kept.\n",
      "-- check region:1122 16:72735001-72765000_strand_+_gene_1122, 59080 candidate probes\n",
      "--- 11320 probes passed check_dic selection.\n",
      "finish in 1.336s, 227 probes kept.\n",
      "-- check region:1123 16:74235001-74265000_strand_+_gene_1123, 58092 candidate probes\n",
      "--- 15410 probes passed check_dic selection.\n",
      "finish in 1.479s, 267 probes kept.\n",
      "-- check region:1124 16:75735001-75765000_strand_+_gene_1124, 55488 candidate probes\n",
      "--- 10625 probes passed check_dic selection.\n",
      "finish in 1.267s, 201 probes kept.\n",
      "-- check region:1125 16:77235001-77265000_strand_+_gene_1125, 57718 candidate probes\n",
      "--- 11950 probes passed check_dic selection.\n",
      "finish in 1.345s, 208 probes kept.\n",
      "-- check region:1126 16:78735001-78765000_strand_+_gene_1126, 59246 candidate probes\n",
      "--- 21069 probes passed check_dic selection.\n",
      "finish in 1.713s, 320 probes kept.\n",
      "-- check region:1127 16:80235001-80265000_strand_+_gene_1127, 57418 candidate probes\n",
      "--- 14243 probes passed check_dic selection.\n",
      "finish in 1.406s, 241 probes kept.\n",
      "-- check region:1128 16:81735001-81765000_strand_+_gene_1128, 58144 candidate probes\n",
      "--- 13753 probes passed check_dic selection.\n",
      "finish in 1.436s, 235 probes kept.\n",
      "-- check region:1129 16:83235001-83265000_strand_+_gene_1129, 59594 candidate probes\n",
      "--- 20187 probes passed check_dic selection.\n",
      "finish in 1.670s, 335 probes kept.\n",
      "-- check region:1130 16:84735001-84765000_strand_+_gene_1130, 58968 candidate probes\n",
      "--- 19618 probes passed check_dic selection.\n",
      "finish in 1.640s, 313 probes kept.\n",
      "-- check region:1131 16:86235001-86265000_strand_+_gene_1131, 49850 candidate probes\n",
      "--- 18095 probes passed check_dic selection.\n",
      "finish in 1.457s, 272 probes kept.\n",
      "-- check region:1132 16:87735001-87765000_strand_+_gene_1132, 56974 candidate probes\n",
      "--- 19986 probes passed check_dic selection.\n",
      "finish in 1.645s, 290 probes kept.\n",
      "-- check region:1133 16:89235001-89265000_strand_+_gene_1133, 55660 candidate probes\n",
      "--- 11862 probes passed check_dic selection.\n",
      "finish in 1.313s, 205 probes kept.\n",
      "-- check region:1134 17:735001-765000_strand_+_gene_1134, 56602 candidate probes\n",
      "--- 17190 probes passed check_dic selection.\n",
      "finish in 1.574s, 331 probes kept.\n",
      "-- check region:1135 17:2235001-2265000_strand_+_gene_1135, 57046 candidate probes\n",
      "--- 17374 probes passed check_dic selection.\n",
      "finish in 1.534s, 268 probes kept.\n",
      "-- check region:1136 17:3735001-3765000_strand_+_gene_1136, 56836 candidate probes\n",
      "--- 20544 probes passed check_dic selection.\n",
      "finish in 1.668s, 294 probes kept.\n",
      "-- check region:1137 17:5235001-5265000_strand_+_gene_1137, 54046 candidate probes\n",
      "--- 10212 probes passed check_dic selection.\n",
      "finish in 1.246s, 196 probes kept.\n",
      "-- check region:1138 17:6735001-6765000_strand_+_gene_1138, 58168 candidate probes\n",
      "--- 12176 probes passed check_dic selection.\n",
      "finish in 1.374s, 222 probes kept.\n",
      "-- check region:1139 17:8235001-8265000_strand_+_gene_1139, 57506 candidate probes\n",
      "--- 14953 probes passed check_dic selection.\n",
      "finish in 1.480s, 256 probes kept.\n",
      "-- check region:1140 17:9735001-9765000_strand_+_gene_1140, 57798 candidate probes\n",
      "--- 14699 probes passed check_dic selection.\n",
      "finish in 1.440s, 242 probes kept.\n",
      "-- check region:1141 17:11235001-11265000_strand_+_gene_1141, 59408 candidate probes\n",
      "--- 25557 probes passed check_dic selection.\n",
      "finish in 1.887s, 375 probes kept.\n",
      "-- check region:1142 17:12735001-12765000_strand_+_gene_1142, 58088 candidate probes\n",
      "--- 14630 probes passed check_dic selection.\n",
      "finish in 1.475s, 263 probes kept.\n",
      "-- check region:1143 17:14235001-14265000_strand_+_gene_1143, 58124 candidate probes\n",
      "--- 10590 probes passed check_dic selection.\n",
      "finish in 1.302s, 206 probes kept.\n",
      "-- check region:1144 17:15735001-15765000_strand_+_gene_1144, 59342 candidate probes\n",
      "--- 4051 probes passed check_dic selection.\n",
      "finish in 1.104s, 122 probes kept.\n",
      "-- check region:1145 17:17235001-17265000_strand_+_gene_1145, 57004 candidate probes\n",
      "--- 12463 probes passed check_dic selection.\n",
      "finish in 1.369s, 215 probes kept.\n",
      "-- check region:1146 17:18735001-18765000_strand_+_gene_1146, 56140 candidate probes\n",
      "--- 2810 probes passed check_dic selection.\n",
      "finish in 1.008s, 94 probes kept.\n",
      "-- check region:1147 17:20235001-20265000_strand_+_gene_1147, 58944 candidate probes\n",
      "--- 11917 probes passed check_dic selection.\n",
      "finish in 1.367s, 257 probes kept.\n",
      "-- check region:1148 17:21735001-21765000_strand_+_gene_1148, 58912 candidate probes\n",
      "--- 3517 probes passed check_dic selection.\n",
      "finish in 1.082s, 101 probes kept.\n",
      "-- check region:1149 17:23235001-23265000_strand_+_gene_1149, 7348 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.120s, 0 probes kept.\n",
      "-- check region:1150 17:24735001-24765000_strand_+_gene_1150, 7626 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.124s, 0 probes kept.\n",
      "-- check region:1151 17:26235001-26265000_strand_+_gene_1151, 19024 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.313s, 0 probes kept.\n",
      "-- check region:1152 17:27735001-27765000_strand_+_gene_1152, 59528 candidate probes\n",
      "--- 8242 probes passed check_dic selection.\n",
      "finish in 1.271s, 194 probes kept.\n",
      "-- check region:1153 17:29235001-29265000_strand_+_gene_1153, 57844 candidate probes\n",
      "--- 13514 probes passed check_dic selection.\n",
      "finish in 1.415s, 269 probes kept.\n",
      "-- check region:1154 17:30735001-30765000_strand_+_gene_1154, 57030 candidate probes\n",
      "--- 1418 probes passed check_dic selection.\n",
      "finish in 0.960s, 54 probes kept.\n",
      "-- check region:1155 17:32235001-32265000_strand_+_gene_1155, 57146 candidate probes\n",
      "--- 15463 probes passed check_dic selection.\n",
      "finish in 1.491s, 254 probes kept.\n",
      "-- check region:1156 17:33735001-33765000_strand_+_gene_1156, 59339 candidate probes\n",
      "--- 19916 probes passed check_dic selection.\n",
      "finish in 1.677s, 327 probes kept.\n",
      "-- check region:1157 17:35235001-35265000_strand_+_gene_1157, 58518 candidate probes\n",
      "--- 16298 probes passed check_dic selection.\n",
      "finish in 1.529s, 276 probes kept.\n",
      "-- check region:1158 17:36735001-36765000_strand_+_gene_1158, 59342 candidate probes\n",
      "--- 690 probes passed check_dic selection.\n",
      "finish in 0.997s, 20 probes kept.\n",
      "-- check region:1159 17:38235001-38265000_strand_+_gene_1159, 54094 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.857s, 0 probes kept.\n",
      "-- check region:1160 17:39735001-39765000_strand_+_gene_1160, 58672 candidate probes\n",
      "--- 17391 probes passed check_dic selection.\n",
      "finish in 1.590s, 317 probes kept.\n",
      "-- check region:1161 17:41235001-41265000_strand_+_gene_1161, 53046 candidate probes\n",
      "--- 24 probes passed check_dic selection.\n",
      "finish in 0.845s, 3 probes kept.\n",
      "-- check region:1162 17:42735001-42765000_strand_+_gene_1162, 57652 candidate probes\n",
      "--- 11949 probes passed check_dic selection.\n",
      "finish in 1.358s, 222 probes kept.\n",
      "-- check region:1163 17:44235001-44265000_strand_+_gene_1163, 56470 candidate probes\n",
      "--- 15429 probes passed check_dic selection.\n",
      "finish in 1.476s, 272 probes kept.\n",
      "-- check region:1164 17:45735001-45765000_strand_+_gene_1164, 59000 candidate probes\n",
      "--- 52 probes passed check_dic selection.\n",
      "finish in 0.967s, 3 probes kept.\n",
      "-- check region:1165 17:47235001-47265000_strand_+_gene_1165, 58640 candidate probes\n",
      "--- 16446 probes passed check_dic selection.\n",
      "finish in 1.520s, 270 probes kept.\n",
      "-- check region:1166 17:48735001-48765000_strand_+_gene_1166, 59218 candidate probes\n",
      "--- 22559 probes passed check_dic selection.\n",
      "finish in 1.772s, 342 probes kept.\n",
      "-- check region:1167 17:50235001-50265000_strand_+_gene_1167, 58532 candidate probes\n",
      "--- 9860 probes passed check_dic selection.\n",
      "finish in 1.285s, 192 probes kept.\n",
      "-- check region:1168 17:51735001-51765000_strand_+_gene_1168, 59490 candidate probes\n",
      "--- 22720 probes passed check_dic selection.\n",
      "finish in 1.771s, 346 probes kept.\n",
      "-- check region:1169 17:53235001-53265000_strand_+_gene_1169, 57240 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 8520 probes passed check_dic selection.\n",
      "finish in 1.192s, 185 probes kept.\n",
      "-- check region:1170 17:54735001-54765000_strand_+_gene_1170, 59522 candidate probes\n",
      "--- 12547 probes passed check_dic selection.\n",
      "finish in 1.388s, 232 probes kept.\n",
      "-- check region:1171 17:56235001-56265000_strand_+_gene_1171, 59358 candidate probes\n",
      "--- 15239 probes passed check_dic selection.\n",
      "finish in 1.480s, 268 probes kept.\n",
      "-- check region:1172 17:57735001-57765000_strand_+_gene_1172, 58822 candidate probes\n",
      "--- 22615 probes passed check_dic selection.\n",
      "finish in 1.765s, 329 probes kept.\n",
      "-- check region:1173 17:59235001-59265000_strand_+_gene_1173, 56443 candidate probes\n",
      "--- 3837 probes passed check_dic selection.\n",
      "finish in 1.030s, 96 probes kept.\n",
      "-- check region:1174 17:60735001-60765000_strand_+_gene_1174, 56646 candidate probes\n",
      "--- 4662 probes passed check_dic selection.\n",
      "finish in 1.054s, 128 probes kept.\n",
      "-- check region:1175 17:62235001-62265000_strand_+_gene_1175, 57584 candidate probes\n",
      "--- 1192 probes passed check_dic selection.\n",
      "finish in 0.962s, 44 probes kept.\n",
      "-- check region:1176 17:63735001-63765000_strand_+_gene_1176, 57806 candidate probes\n",
      "--- 9858 probes passed check_dic selection.\n",
      "finish in 1.278s, 212 probes kept.\n",
      "-- check region:1177 17:65235001-65265000_strand_+_gene_1177, 58900 candidate probes\n",
      "--- 14497 probes passed check_dic selection.\n",
      "finish in 1.467s, 241 probes kept.\n",
      "-- check region:1178 17:66735001-66765000_strand_+_gene_1178, 58416 candidate probes\n",
      "--- 20744 probes passed check_dic selection.\n",
      "finish in 1.691s, 321 probes kept.\n",
      "-- check region:1179 17:68235001-68265000_strand_+_gene_1179, 58958 candidate probes\n",
      "--- 7607 probes passed check_dic selection.\n",
      "finish in 1.213s, 197 probes kept.\n",
      "-- check region:1180 17:69735001-69765000_strand_+_gene_1180, 56278 candidate probes\n",
      "--- 9900 probes passed check_dic selection.\n",
      "finish in 1.241s, 213 probes kept.\n",
      "-- check region:1181 17:71235001-71265000_strand_+_gene_1181, 59236 candidate probes\n",
      "--- 14571 probes passed check_dic selection.\n",
      "finish in 1.444s, 271 probes kept.\n",
      "-- check region:1182 17:72735001-72765000_strand_+_gene_1182, 58224 candidate probes\n",
      "--- 20385 probes passed check_dic selection.\n",
      "finish in 1.668s, 291 probes kept.\n",
      "-- check region:1183 17:74235001-74265000_strand_+_gene_1183, 58840 candidate probes\n",
      "--- 20318 probes passed check_dic selection.\n",
      "finish in 1.708s, 337 probes kept.\n",
      "-- check region:1184 17:75735001-75765000_strand_+_gene_1184, 58250 candidate probes\n",
      "--- 18079 probes passed check_dic selection.\n",
      "finish in 1.602s, 310 probes kept.\n",
      "-- check region:1185 17:77235001-77265000_strand_+_gene_1185, 57304 candidate probes\n",
      "--- 202 probes passed check_dic selection.\n",
      "finish in 0.939s, 4 probes kept.\n",
      "-- check region:1186 17:78735001-78765000_strand_+_gene_1186, 58864 candidate probes\n",
      "--- 15108 probes passed check_dic selection.\n",
      "finish in 1.484s, 265 probes kept.\n",
      "-- check region:1187 17:80235001-80265000_strand_+_gene_1187, 57108 candidate probes\n",
      "--- 18795 probes passed check_dic selection.\n",
      "finish in 1.593s, 285 probes kept.\n",
      "-- check region:1188 17:81735001-81765000_strand_+_gene_1188, 14538 candidate probes\n",
      "--- 6139 probes passed check_dic selection.\n",
      "finish in 0.461s, 77 probes kept.\n",
      "-- check region:1189 18:735001-765000_strand_+_gene_1189, 57644 candidate probes\n",
      "--- 7096 probes passed check_dic selection.\n",
      "finish in 1.164s, 171 probes kept.\n",
      "-- check region:1190 18:2235001-2265000_strand_+_gene_1190, 52140 candidate probes\n",
      "--- 9189 probes passed check_dic selection.\n",
      "finish in 1.155s, 187 probes kept.\n",
      "-- check region:1191 18:3735001-3765000_strand_+_gene_1191, 54522 candidate probes\n",
      "--- 10060 probes passed check_dic selection.\n",
      "finish in 1.221s, 194 probes kept.\n",
      "-- check region:1192 18:5235001-5265000_strand_+_gene_1192, 59758 candidate probes\n",
      "--- 9373 probes passed check_dic selection.\n",
      "finish in 1.291s, 242 probes kept.\n",
      "-- check region:1193 18:6735001-6765000_strand_+_gene_1193, 59512 candidate probes\n",
      "--- 15014 probes passed check_dic selection.\n",
      "finish in 1.470s, 279 probes kept.\n",
      "-- check region:1194 18:8235001-8265000_strand_+_gene_1194, 59074 candidate probes\n",
      "--- 16333 probes passed check_dic selection.\n",
      "finish in 1.521s, 291 probes kept.\n",
      "-- check region:1195 18:9735001-9765000_strand_+_gene_1195, 59004 candidate probes\n",
      "--- 18999 probes passed check_dic selection.\n",
      "finish in 1.621s, 288 probes kept.\n",
      "-- check region:1196 18:11235001-11265000_strand_+_gene_1196, 58964 candidate probes\n",
      "--- 15547 probes passed check_dic selection.\n",
      "finish in 1.481s, 277 probes kept.\n",
      "-- check region:1197 18:12735001-12765000_strand_+_gene_1197, 56960 candidate probes\n",
      "--- 10378 probes passed check_dic selection.\n",
      "finish in 1.291s, 184 probes kept.\n",
      "-- check region:1198 18:14235001-14265000_strand_+_gene_1198, 59672 candidate probes\n",
      "--- 367 probes passed check_dic selection.\n",
      "finish in 0.969s, 30 probes kept.\n",
      "-- check region:1199 18:15735001-15765000_strand_+_gene_1199, 23734 candidate probes\n",
      "--- 10 probes passed check_dic selection.\n",
      "finish in 0.387s, 2 probes kept.\n",
      "-- check region:1200 18:17235001-17265000_strand_+_gene_1200, 8108 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.132s, 0 probes kept.\n",
      "-- check region:1201 18:18735001-18765000_strand_+_gene_1201, 8552 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.138s, 0 probes kept.\n",
      "-- check region:1202 18:20235001-20265000_strand_+_gene_1202, 13110 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.216s, 0 probes kept.\n",
      "-- check region:1203 18:21735001-21765000_strand_+_gene_1203, 58394 candidate probes\n",
      "--- 9079 probes passed check_dic selection.\n",
      "finish in 1.244s, 191 probes kept.\n",
      "-- check region:1204 18:23235001-23265000_strand_+_gene_1204, 58866 candidate probes\n",
      "--- 22842 probes passed check_dic selection.\n",
      "finish in 1.799s, 346 probes kept.\n",
      "-- check region:1205 18:24735001-24765000_strand_+_gene_1205, 59700 candidate probes\n",
      "--- 20588 probes passed check_dic selection.\n",
      "finish in 1.694s, 321 probes kept.\n",
      "-- check region:1206 18:26235001-26265000_strand_+_gene_1206, 52008 candidate probes\n",
      "--- 3965 probes passed check_dic selection.\n",
      "finish in 0.979s, 108 probes kept.\n",
      "-- check region:1207 18:27735001-27765000_strand_+_gene_1207, 51782 candidate probes\n",
      "--- 10618 probes passed check_dic selection.\n",
      "finish in 1.196s, 213 probes kept.\n",
      "-- check region:1208 18:29235001-29265000_strand_+_gene_1208, 59308 candidate probes\n",
      "--- 8807 probes passed check_dic selection.\n",
      "finish in 1.235s, 208 probes kept.\n",
      "-- check region:1209 18:30735001-30765000_strand_+_gene_1209, 59246 candidate probes\n",
      "--- 7695 probes passed check_dic selection.\n",
      "finish in 1.188s, 169 probes kept.\n",
      "-- check region:1210 18:32235001-32265000_strand_+_gene_1210, 59058 candidate probes\n",
      "--- 15145 probes passed check_dic selection.\n",
      "finish in 1.489s, 270 probes kept.\n",
      "-- check region:1211 18:33735001-33765000_strand_+_gene_1211, 59874 candidate probes\n",
      "--- 13724 probes passed check_dic selection.\n",
      "finish in 1.454s, 302 probes kept.\n",
      "-- check region:1212 18:35235001-35265000_strand_+_gene_1212, 59144 candidate probes\n",
      "--- 10851 probes passed check_dic selection.\n",
      "finish in 1.344s, 278 probes kept.\n",
      "-- check region:1213 18:36735001-36765000_strand_+_gene_1213, 58980 candidate probes\n",
      "--- 17961 probes passed check_dic selection.\n",
      "finish in 1.572s, 293 probes kept.\n",
      "-- check region:1214 18:38235001-38265000_strand_+_gene_1214, 59830 candidate probes\n",
      "--- 17620 probes passed check_dic selection.\n",
      "finish in 1.574s, 317 probes kept.\n",
      "-- check region:1215 18:39735001-39765000_strand_+_gene_1215, 59682 candidate probes\n",
      "--- 11122 probes passed check_dic selection.\n",
      "finish in 1.334s, 221 probes kept.\n",
      "-- check region:1216 18:41235001-41265000_strand_+_gene_1216, 59352 candidate probes\n",
      "--- 11677 probes passed check_dic selection.\n",
      "finish in 1.325s, 216 probes kept.\n",
      "-- check region:1217 18:42735001-42765000_strand_+_gene_1217, 59056 candidate probes\n",
      "--- 9338 probes passed check_dic selection.\n",
      "finish in 1.261s, 190 probes kept.\n",
      "-- check region:1218 18:44235001-44265000_strand_+_gene_1218, 59088 candidate probes\n",
      "--- 12128 probes passed check_dic selection.\n",
      "finish in 1.360s, 235 probes kept.\n",
      "-- check region:1219 18:45735001-45765000_strand_+_gene_1219, 58932 candidate probes\n",
      "--- 20142 probes passed check_dic selection.\n",
      "finish in 1.675s, 315 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1220 18:47235001-47265000_strand_+_gene_1220, 59756 candidate probes\n",
      "--- 22586 probes passed check_dic selection.\n",
      "finish in 1.740s, 358 probes kept.\n",
      "-- check region:1221 18:48735001-48765000_strand_+_gene_1221, 59670 candidate probes\n",
      "--- 27239 probes passed check_dic selection.\n",
      "finish in 1.971s, 411 probes kept.\n",
      "-- check region:1222 18:50235001-50265000_strand_+_gene_1222, 58282 candidate probes\n",
      "--- 8971 probes passed check_dic selection.\n",
      "finish in 1.247s, 179 probes kept.\n",
      "-- check region:1223 18:51735001-51765000_strand_+_gene_1223, 52010 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.818s, 0 probes kept.\n",
      "-- check region:1224 18:53235001-53265000_strand_+_gene_1224, 56100 candidate probes\n",
      "--- 12868 probes passed check_dic selection.\n",
      "finish in 1.345s, 231 probes kept.\n",
      "-- check region:1225 18:54735001-54765000_strand_+_gene_1225, 59210 candidate probes\n",
      "--- 14691 probes passed check_dic selection.\n",
      "finish in 1.461s, 282 probes kept.\n",
      "-- check region:1226 18:56235001-56265000_strand_+_gene_1226, 56232 candidate probes\n",
      "--- 13033 probes passed check_dic selection.\n",
      "finish in 1.361s, 238 probes kept.\n",
      "-- check region:1227 18:57735001-57765000_strand_+_gene_1227, 58110 candidate probes\n",
      "--- 12225 probes passed check_dic selection.\n",
      "finish in 1.362s, 215 probes kept.\n",
      "-- check region:1228 18:59235001-59265000_strand_+_gene_1228, 53920 candidate probes\n",
      "--- 9937 probes passed check_dic selection.\n",
      "finish in 1.215s, 167 probes kept.\n",
      "-- check region:1229 18:60735001-60765000_strand_+_gene_1229, 59842 candidate probes\n",
      "--- 10356 probes passed check_dic selection.\n",
      "finish in 1.296s, 235 probes kept.\n",
      "-- check region:1230 18:62235001-62265000_strand_+_gene_1230, 59788 candidate probes\n",
      "--- 8868 probes passed check_dic selection.\n",
      "finish in 1.241s, 223 probes kept.\n",
      "-- check region:1231 18:63735001-63765000_strand_+_gene_1231, 58527 candidate probes\n",
      "--- 13794 probes passed check_dic selection.\n",
      "finish in 1.414s, 241 probes kept.\n",
      "-- check region:1232 18:65235001-65265000_strand_+_gene_1232, 59726 candidate probes\n",
      "--- 12814 probes passed check_dic selection.\n",
      "finish in 1.393s, 237 probes kept.\n",
      "-- check region:1233 18:66735001-66765000_strand_+_gene_1233, 58182 candidate probes\n",
      "--- 10850 probes passed check_dic selection.\n",
      "finish in 1.296s, 223 probes kept.\n",
      "-- check region:1234 18:68235001-68265000_strand_+_gene_1234, 57708 candidate probes\n",
      "--- 8539 probes passed check_dic selection.\n",
      "finish in 1.202s, 169 probes kept.\n",
      "-- check region:1235 18:69735001-69765000_strand_+_gene_1235, 58716 candidate probes\n",
      "--- 12895 probes passed check_dic selection.\n",
      "finish in 1.391s, 241 probes kept.\n",
      "-- check region:1236 18:71235001-71265000_strand_+_gene_1236, 59658 candidate probes\n",
      "--- 15045 probes passed check_dic selection.\n",
      "finish in 1.463s, 271 probes kept.\n",
      "-- check region:1237 18:72735001-72765000_strand_+_gene_1237, 59676 candidate probes\n",
      "--- 10914 probes passed check_dic selection.\n",
      "finish in 1.310s, 228 probes kept.\n",
      "-- check region:1238 18:74235001-74265000_strand_+_gene_1238, 58844 candidate probes\n",
      "--- 15841 probes passed check_dic selection.\n",
      "finish in 1.507s, 287 probes kept.\n",
      "-- check region:1239 18:75735001-75765000_strand_+_gene_1239, 59660 candidate probes\n",
      "--- 13905 probes passed check_dic selection.\n",
      "finish in 1.436s, 249 probes kept.\n",
      "-- check region:1240 18:77235001-77265000_strand_+_gene_1240, 59276 candidate probes\n",
      "--- 18596 probes passed check_dic selection.\n",
      "finish in 1.613s, 297 probes kept.\n",
      "-- check region:1241 18:78735001-78765000_strand_+_gene_1241, 58946 candidate probes\n",
      "--- 25275 probes passed check_dic selection.\n",
      "finish in 1.858s, 344 probes kept.\n",
      "-- check region:1242 19:485001-515000_strand_+_gene_1242, 56288 candidate probes\n",
      "--- 15473 probes passed check_dic selection.\n",
      "finish in 1.447s, 279 probes kept.\n",
      "-- check region:1243 19:1485001-1515000_strand_+_gene_1243, 56996 candidate probes\n",
      "--- 16468 probes passed check_dic selection.\n",
      "finish in 1.514s, 297 probes kept.\n",
      "-- check region:1244 19:2485001-2515000_strand_+_gene_1244, 56828 candidate probes\n",
      "--- 16670 probes passed check_dic selection.\n",
      "finish in 1.525s, 260 probes kept.\n",
      "-- check region:1245 19:3485001-3515000_strand_+_gene_1245, 57806 candidate probes\n",
      "--- 18034 probes passed check_dic selection.\n",
      "finish in 1.587s, 283 probes kept.\n",
      "-- check region:1246 19:4485001-4515000_strand_+_gene_1246, 57486 candidate probes\n",
      "--- 17480 probes passed check_dic selection.\n",
      "finish in 1.582s, 305 probes kept.\n",
      "-- check region:1247 19:5485001-5515000_strand_+_gene_1247, 56364 candidate probes\n",
      "--- 15411 probes passed check_dic selection.\n",
      "finish in 1.476s, 234 probes kept.\n",
      "-- check region:1248 19:6485001-6515000_strand_+_gene_1248, 56274 candidate probes\n",
      "--- 8231 probes passed check_dic selection.\n",
      "finish in 1.219s, 160 probes kept.\n",
      "-- check region:1249 19:7485001-7515000_strand_+_gene_1249, 57666 candidate probes\n",
      "--- 13787 probes passed check_dic selection.\n",
      "finish in 1.431s, 237 probes kept.\n",
      "-- check region:1250 19:8485001-8515000_strand_+_gene_1250, 58086 candidate probes\n",
      "--- 18194 probes passed check_dic selection.\n",
      "finish in 1.609s, 310 probes kept.\n",
      "-- check region:1251 19:9485001-9515000_strand_+_gene_1251, 58304 candidate probes\n",
      "--- 8894 probes passed check_dic selection.\n",
      "finish in 1.255s, 181 probes kept.\n",
      "-- check region:1252 19:10485001-10515000_strand_+_gene_1252, 57182 candidate probes\n",
      "--- 9158 probes passed check_dic selection.\n",
      "finish in 1.258s, 181 probes kept.\n",
      "-- check region:1253 19:11485001-11515000_strand_+_gene_1253, 57644 candidate probes\n",
      "--- 13446 probes passed check_dic selection.\n",
      "finish in 1.431s, 246 probes kept.\n",
      "-- check region:1254 19:12485001-12515000_strand_+_gene_1254, 55432 candidate probes\n",
      "--- 9454 probes passed check_dic selection.\n",
      "finish in 1.235s, 178 probes kept.\n",
      "-- check region:1255 19:13485001-13515000_strand_+_gene_1255, 58666 candidate probes\n",
      "--- 18750 probes passed check_dic selection.\n",
      "finish in 1.647s, 303 probes kept.\n",
      "-- check region:1256 19:14485001-14515000_strand_+_gene_1256, 56834 candidate probes\n",
      "--- 11434 probes passed check_dic selection.\n",
      "finish in 1.337s, 186 probes kept.\n",
      "-- check region:1257 19:15485001-15515000_strand_+_gene_1257, 52788 candidate probes\n",
      "--- 9011 probes passed check_dic selection.\n",
      "finish in 1.174s, 173 probes kept.\n",
      "-- check region:1258 19:16485001-16515000_strand_+_gene_1258, 56572 candidate probes\n",
      "--- 9106 probes passed check_dic selection.\n",
      "finish in 1.252s, 168 probes kept.\n",
      "-- check region:1259 19:17485001-17515000_strand_+_gene_1259, 57042 candidate probes\n",
      "--- 10537 probes passed check_dic selection.\n",
      "finish in 1.321s, 216 probes kept.\n",
      "-- check region:1260 19:18485001-18515000_strand_+_gene_1260, 58710 candidate probes\n",
      "--- 25864 probes passed check_dic selection.\n",
      "finish in 1.888s, 364 probes kept.\n",
      "-- check region:1261 19:19485001-19515000_strand_+_gene_1261, 59330 candidate probes\n",
      "--- 25052 probes passed check_dic selection.\n",
      "finish in 1.903s, 377 probes kept.\n",
      "-- check region:1262 19:20485001-20515000_strand_+_gene_1262, 56392 candidate probes\n",
      "--- 2198 probes passed check_dic selection.\n",
      "finish in 0.984s, 85 probes kept.\n",
      "-- check region:1263 19:21485001-21515000_strand_+_gene_1263, 56332 candidate probes\n",
      "--- 6287 probes passed check_dic selection.\n",
      "finish in 1.144s, 189 probes kept.\n",
      "-- check region:1264 19:22485001-22515000_strand_+_gene_1264, 59238 candidate probes\n",
      "--- 14505 probes passed check_dic selection.\n",
      "finish in 1.479s, 279 probes kept.\n",
      "-- check region:1265 19:23485001-23515000_strand_+_gene_1265, 59282 candidate probes\n",
      "--- 2747 probes passed check_dic selection.\n",
      "finish in 1.027s, 99 probes kept.\n",
      "-- check region:1266 19:24485001-24515000_strand_+_gene_1266, 20878 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.332s, 0 probes kept.\n",
      "-- check region:1267 19:25485001-25515000_strand_+_gene_1267, 22550 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.364s, 0 probes kept.\n",
      "-- check region:1268 19:26485001-26515000_strand_+_gene_1268, 31012 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.504s, 0 probes kept.\n",
      "-- check region:1269 19:27485001-27515000_strand_+_gene_1269, 57738 candidate probes\n",
      "--- 822 probes passed check_dic selection.\n",
      "finish in 0.936s, 51 probes kept.\n",
      "-- check region:1270 19:28485001-28515000_strand_+_gene_1270, 58418 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 19578 probes passed check_dic selection.\n",
      "finish in 1.645s, 305 probes kept.\n",
      "-- check region:1271 19:29485001-29515000_strand_+_gene_1271, 59160 candidate probes\n",
      "--- 21005 probes passed check_dic selection.\n",
      "finish in 1.716s, 312 probes kept.\n",
      "-- check region:1272 19:30485001-30515000_strand_+_gene_1272, 59460 candidate probes\n",
      "--- 21842 probes passed check_dic selection.\n",
      "finish in 1.730s, 317 probes kept.\n",
      "-- check region:1273 19:31485001-31515000_strand_+_gene_1273, 58640 candidate probes\n",
      "--- 19115 probes passed check_dic selection.\n",
      "finish in 1.628s, 297 probes kept.\n",
      "-- check region:1274 19:32485001-32515000_strand_+_gene_1274, 58402 candidate probes\n",
      "--- 9122 probes passed check_dic selection.\n",
      "finish in 1.250s, 192 probes kept.\n",
      "-- check region:1275 19:33485001-33515000_strand_+_gene_1275, 59294 candidate probes\n",
      "--- 24844 probes passed check_dic selection.\n",
      "finish in 1.858s, 356 probes kept.\n",
      "-- check region:1276 19:34485001-34515000_strand_+_gene_1276, 58174 candidate probes\n",
      "--- 11915 probes passed check_dic selection.\n",
      "finish in 1.394s, 244 probes kept.\n",
      "-- check region:1277 19:35485001-35515000_strand_+_gene_1277, 59040 candidate probes\n",
      "--- 21971 probes passed check_dic selection.\n",
      "finish in 1.768s, 362 probes kept.\n",
      "-- check region:1278 19:36485001-36515000_strand_+_gene_1278, 55924 candidate probes\n",
      "--- 6573 probes passed check_dic selection.\n",
      "finish in 1.138s, 150 probes kept.\n",
      "-- check region:1279 19:37485001-37515000_strand_+_gene_1279, 57780 candidate probes\n",
      "--- 9485 probes passed check_dic selection.\n",
      "finish in 1.264s, 193 probes kept.\n",
      "-- check region:1280 19:38485001-38515000_strand_+_gene_1280, 58442 candidate probes\n",
      "--- 17256 probes passed check_dic selection.\n",
      "finish in 1.581s, 296 probes kept.\n",
      "-- check region:1281 19:39485001-39515000_strand_+_gene_1281, 57530 candidate probes\n",
      "--- 15422 probes passed check_dic selection.\n",
      "finish in 1.488s, 275 probes kept.\n",
      "-- check region:1282 19:40485001-40515000_strand_+_gene_1282, 58330 candidate probes\n",
      "--- 13423 probes passed check_dic selection.\n",
      "finish in 1.438s, 234 probes kept.\n",
      "-- check region:1283 19:41485001-41515000_strand_+_gene_1283, 54888 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.898s, 0 probes kept.\n",
      "-- check region:1284 19:42485001-42515000_strand_+_gene_1284, 58452 candidate probes\n",
      "--- 8743 probes passed check_dic selection.\n",
      "finish in 1.251s, 188 probes kept.\n",
      "-- check region:1285 19:43485001-43515000_strand_+_gene_1285, 57426 candidate probes\n",
      "--- 11009 probes passed check_dic selection.\n",
      "finish in 1.331s, 206 probes kept.\n",
      "-- check region:1286 19:44485001-44515000_strand_+_gene_1286, 58624 candidate probes\n",
      "--- 15915 probes passed check_dic selection.\n",
      "finish in 1.505s, 263 probes kept.\n",
      "-- check region:1287 19:45485001-45515000_strand_+_gene_1287, 57798 candidate probes\n",
      "--- 16734 probes passed check_dic selection.\n",
      "finish in 1.525s, 287 probes kept.\n",
      "-- check region:1288 19:46485001-46515000_strand_+_gene_1288, 57602 candidate probes\n",
      "--- 14123 probes passed check_dic selection.\n",
      "finish in 1.452s, 239 probes kept.\n",
      "-- check region:1289 19:47485001-47515000_strand_+_gene_1289, 59248 candidate probes\n",
      "--- 23648 probes passed check_dic selection.\n",
      "finish in 1.835s, 368 probes kept.\n",
      "-- check region:1290 19:48485001-48515000_strand_+_gene_1290, 57934 candidate probes\n",
      "--- 18310 probes passed check_dic selection.\n",
      "finish in 1.575s, 315 probes kept.\n",
      "-- check region:1291 19:49485001-49515000_strand_+_gene_1291, 57822 candidate probes\n",
      "--- 16539 probes passed check_dic selection.\n",
      "finish in 1.544s, 273 probes kept.\n",
      "-- check region:1292 19:50485001-50515000_strand_+_gene_1292, 57788 candidate probes\n",
      "--- 12440 probes passed check_dic selection.\n",
      "finish in 1.394s, 248 probes kept.\n",
      "-- check region:1293 19:51485001-51515000_strand_+_gene_1293, 59632 candidate probes\n",
      "--- 16600 probes passed check_dic selection.\n",
      "finish in 1.569s, 312 probes kept.\n",
      "-- check region:1294 19:52485001-52515000_strand_+_gene_1294, 58292 candidate probes\n",
      "--- 2392 probes passed check_dic selection.\n",
      "finish in 1.023s, 90 probes kept.\n",
      "-- check region:1295 19:53485001-53515000_strand_+_gene_1295, 58898 candidate probes\n",
      "--- 7360 probes passed check_dic selection.\n",
      "finish in 1.211s, 167 probes kept.\n",
      "-- check region:1296 19:54485001-54515000_strand_+_gene_1296, 57672 candidate probes\n",
      "--- 16 probes passed check_dic selection.\n",
      "finish in 0.937s, 1 probes kept.\n",
      "-- check region:1297 19:55485001-55515000_strand_+_gene_1297, 57892 candidate probes\n",
      "--- 16913 probes passed check_dic selection.\n",
      "finish in 1.559s, 312 probes kept.\n",
      "-- check region:1298 19:56485001-56515000_strand_+_gene_1298, 59204 candidate probes\n",
      "--- 15083 probes passed check_dic selection.\n",
      "finish in 1.502s, 282 probes kept.\n",
      "-- check region:1299 19:57485001-57515000_strand_+_gene_1299, 55476 candidate probes\n",
      "--- 12190 probes passed check_dic selection.\n",
      "finish in 1.336s, 267 probes kept.\n",
      "-- check region:1300 20:485001-515000_strand_+_gene_1300, 58700 candidate probes\n",
      "--- 14106 probes passed check_dic selection.\n",
      "finish in 1.439s, 266 probes kept.\n",
      "-- check region:1301 20:1485001-1515000_strand_+_gene_1301, 59640 candidate probes\n",
      "--- 14457 probes passed check_dic selection.\n",
      "finish in 1.463s, 252 probes kept.\n",
      "-- check region:1302 20:2485001-2515000_strand_+_gene_1302, 55052 candidate probes\n",
      "--- 12281 probes passed check_dic selection.\n",
      "finish in 1.319s, 220 probes kept.\n",
      "-- check region:1303 20:3485001-3515000_strand_+_gene_1303, 51866 candidate probes\n",
      "--- 7947 probes passed check_dic selection.\n",
      "finish in 1.111s, 174 probes kept.\n",
      "-- check region:1304 20:4485001-4515000_strand_+_gene_1304, 58106 candidate probes\n",
      "--- 16252 probes passed check_dic selection.\n",
      "finish in 1.514s, 268 probes kept.\n",
      "-- check region:1305 20:5485001-5515000_strand_+_gene_1305, 59160 candidate probes\n",
      "--- 16302 probes passed check_dic selection.\n",
      "finish in 1.569s, 297 probes kept.\n",
      "-- check region:1306 20:6485001-6515000_strand_+_gene_1306, 59660 candidate probes\n",
      "--- 17777 probes passed check_dic selection.\n",
      "finish in 1.590s, 285 probes kept.\n",
      "-- check region:1307 20:7485001-7515000_strand_+_gene_1307, 59674 candidate probes\n",
      "--- 14377 probes passed check_dic selection.\n",
      "finish in 1.453s, 271 probes kept.\n",
      "-- check region:1308 20:8485001-8515000_strand_+_gene_1308, 58590 candidate probes\n",
      "--- 13312 probes passed check_dic selection.\n",
      "finish in 1.387s, 243 probes kept.\n",
      "-- check region:1309 20:9485001-9515000_strand_+_gene_1309, 59788 candidate probes\n",
      "--- 18555 probes passed check_dic selection.\n",
      "finish in 1.624s, 325 probes kept.\n",
      "-- check region:1310 20:10485001-10515000_strand_+_gene_1310, 59820 candidate probes\n",
      "--- 16428 probes passed check_dic selection.\n",
      "finish in 1.530s, 285 probes kept.\n",
      "-- check region:1311 20:11485001-11515000_strand_+_gene_1311, 59304 candidate probes\n",
      "--- 12063 probes passed check_dic selection.\n",
      "finish in 1.360s, 211 probes kept.\n",
      "-- check region:1312 20:12485001-12515000_strand_+_gene_1312, 55692 candidate probes\n",
      "--- 14455 probes passed check_dic selection.\n",
      "finish in 1.397s, 257 probes kept.\n",
      "-- check region:1313 20:13485001-13515000_strand_+_gene_1313, 58770 candidate probes\n",
      "--- 9311 probes passed check_dic selection.\n",
      "finish in 1.245s, 201 probes kept.\n",
      "-- check region:1314 20:14485001-14515000_strand_+_gene_1314, 59578 candidate probes\n",
      "--- 14057 probes passed check_dic selection.\n",
      "finish in 1.426s, 272 probes kept.\n",
      "-- check region:1315 20:15485001-15515000_strand_+_gene_1315, 59908 candidate probes\n",
      "--- 21051 probes passed check_dic selection.\n",
      "finish in 1.706s, 347 probes kept.\n",
      "-- check region:1316 20:16485001-16515000_strand_+_gene_1316, 59380 candidate probes\n",
      "--- 16276 probes passed check_dic selection.\n",
      "finish in 1.526s, 301 probes kept.\n",
      "-- check region:1317 20:17485001-17515000_strand_+_gene_1317, 59660 candidate probes\n",
      "--- 21960 probes passed check_dic selection.\n",
      "finish in 1.763s, 344 probes kept.\n",
      "-- check region:1318 20:18485001-18515000_strand_+_gene_1318, 57968 candidate probes\n",
      "--- 13162 probes passed check_dic selection.\n",
      "finish in 1.406s, 253 probes kept.\n",
      "-- check region:1319 20:19485001-19515000_strand_+_gene_1319, 59682 candidate probes\n",
      "--- 24951 probes passed check_dic selection.\n",
      "finish in 1.880s, 385 probes kept.\n",
      "-- check region:1320 20:20485001-20515000_strand_+_gene_1320, 59884 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 16153 probes passed check_dic selection.\n",
      "finish in 1.529s, 286 probes kept.\n",
      "-- check region:1321 20:21485001-21515000_strand_+_gene_1321, 59504 candidate probes\n",
      "--- 20139 probes passed check_dic selection.\n",
      "finish in 1.667s, 321 probes kept.\n",
      "-- check region:1322 20:22485001-22515000_strand_+_gene_1322, 59278 candidate probes\n",
      "--- 20471 probes passed check_dic selection.\n",
      "finish in 1.676s, 310 probes kept.\n",
      "-- check region:1323 20:23485001-23515000_strand_+_gene_1323, 59402 candidate probes\n",
      "--- 18457 probes passed check_dic selection.\n",
      "finish in 1.615s, 306 probes kept.\n",
      "-- check region:1324 20:24485001-24515000_strand_+_gene_1324, 58566 candidate probes\n",
      "--- 21023 probes passed check_dic selection.\n",
      "finish in 1.705s, 321 probes kept.\n",
      "-- check region:1325 20:25485001-25515000_strand_+_gene_1325, 59006 candidate probes\n",
      "--- 24951 probes passed check_dic selection.\n",
      "finish in 1.873s, 367 probes kept.\n",
      "-- check region:1326 20:26485001-26515000_strand_+_gene_1326, 14778 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.241s, 0 probes kept.\n",
      "-- check region:1327 20:27485001-27515000_strand_+_gene_1327, 6236 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.102s, 0 probes kept.\n",
      "-- check region:1328 20:28485001-28515000_strand_+_gene_1328, 28964 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.468s, 0 probes kept.\n",
      "-- check region:1329 20:29485001-29515000_strand_+_gene_1329, 59880 candidate probes\n",
      "--- 747 probes passed check_dic selection.\n",
      "finish in 0.955s, 35 probes kept.\n",
      "-- check region:1330 20:30485001-30515000_strand_+_gene_1330, 58496 candidate probes\n",
      "--- 4033 probes passed check_dic selection.\n",
      "finish in 1.095s, 129 probes kept.\n",
      "-- check region:1331 20:31485001-31515000_strand_+_gene_1331, 57724 candidate probes\n",
      "--- 13858 probes passed check_dic selection.\n",
      "finish in 1.432s, 242 probes kept.\n",
      "-- check region:1332 20:32485001-32515000_strand_+_gene_1332, 58594 candidate probes\n",
      "--- 17267 probes passed check_dic selection.\n",
      "finish in 1.566s, 278 probes kept.\n",
      "-- check region:1333 20:33485001-33515000_strand_+_gene_1333, 56884 candidate probes\n",
      "--- 7589 probes passed check_dic selection.\n",
      "finish in 1.189s, 136 probes kept.\n",
      "-- check region:1334 20:34485001-34515000_strand_+_gene_1334, 58802 candidate probes\n",
      "--- 6479 probes passed check_dic selection.\n",
      "finish in 1.160s, 161 probes kept.\n",
      "-- check region:1335 20:35485001-35515000_strand_+_gene_1335, 59078 candidate probes\n",
      "--- 17792 probes passed check_dic selection.\n",
      "finish in 1.612s, 326 probes kept.\n",
      "-- check region:1336 20:36485001-36515000_strand_+_gene_1336, 57734 candidate probes\n",
      "--- 15527 probes passed check_dic selection.\n",
      "finish in 1.507s, 266 probes kept.\n",
      "-- check region:1337 20:37485001-37515000_strand_+_gene_1337, 58956 candidate probes\n",
      "--- 18895 probes passed check_dic selection.\n",
      "finish in 1.635s, 339 probes kept.\n",
      "-- check region:1338 20:38485001-38515000_strand_+_gene_1338, 58212 candidate probes\n",
      "--- 9892 probes passed check_dic selection.\n",
      "finish in 1.272s, 188 probes kept.\n",
      "-- check region:1339 20:39485001-39515000_strand_+_gene_1339, 57442 candidate probes\n",
      "--- 13598 probes passed check_dic selection.\n",
      "finish in 1.394s, 238 probes kept.\n",
      "-- check region:1340 20:40485001-40515000_strand_+_gene_1340, 59726 candidate probes\n",
      "--- 23193 probes passed check_dic selection.\n",
      "finish in 1.796s, 344 probes kept.\n",
      "-- check region:1341 20:41485001-41515000_strand_+_gene_1341, 59860 candidate probes\n",
      "--- 18311 probes passed check_dic selection.\n",
      "finish in 1.610s, 308 probes kept.\n",
      "-- check region:1342 20:42485001-42515000_strand_+_gene_1342, 58744 candidate probes\n",
      "--- 12570 probes passed check_dic selection.\n",
      "finish in 1.371s, 261 probes kept.\n",
      "-- check region:1343 20:43485001-43515000_strand_+_gene_1343, 58452 candidate probes\n",
      "--- 11013 probes passed check_dic selection.\n",
      "finish in 1.338s, 201 probes kept.\n",
      "-- check region:1344 20:44485001-44515000_strand_+_gene_1344, 58736 candidate probes\n",
      "--- 12500 probes passed check_dic selection.\n",
      "finish in 1.408s, 274 probes kept.\n",
      "-- check region:1345 20:45485001-45515000_strand_+_gene_1345, 58952 candidate probes\n",
      "--- 17536 probes passed check_dic selection.\n",
      "finish in 1.583s, 312 probes kept.\n",
      "-- check region:1346 20:46485001-46515000_strand_+_gene_1346, 59012 candidate probes\n",
      "--- 14043 probes passed check_dic selection.\n",
      "finish in 1.445s, 298 probes kept.\n",
      "-- check region:1347 20:47485001-47515000_strand_+_gene_1347, 58054 candidate probes\n",
      "--- 9822 probes passed check_dic selection.\n",
      "finish in 1.278s, 179 probes kept.\n",
      "-- check region:1348 20:48485001-48515000_strand_+_gene_1348, 47686 candidate probes\n",
      "--- 4560 probes passed check_dic selection.\n",
      "finish in 0.944s, 130 probes kept.\n",
      "-- check region:1349 20:49485001-49515000_strand_+_gene_1349, 59304 candidate probes\n",
      "--- 16231 probes passed check_dic selection.\n",
      "finish in 1.546s, 302 probes kept.\n",
      "-- check region:1350 20:50485001-50515000_strand_+_gene_1350, 58630 candidate probes\n",
      "--- 16406 probes passed check_dic selection.\n",
      "finish in 1.526s, 265 probes kept.\n",
      "-- check region:1351 20:51485001-51515000_strand_+_gene_1351, 59144 candidate probes\n",
      "--- 23657 probes passed check_dic selection.\n",
      "finish in 1.822s, 353 probes kept.\n",
      "-- check region:1352 20:52485001-52515000_strand_+_gene_1352, 59268 candidate probes\n",
      "--- 14122 probes passed check_dic selection.\n",
      "finish in 1.436s, 243 probes kept.\n",
      "-- check region:1353 20:53485001-53515000_strand_+_gene_1353, 51788 candidate probes\n",
      "--- 9094 probes passed check_dic selection.\n",
      "finish in 1.162s, 192 probes kept.\n",
      "-- check region:1354 20:54485001-54515000_strand_+_gene_1354, 59166 candidate probes\n",
      "--- 14557 probes passed check_dic selection.\n",
      "finish in 1.452s, 278 probes kept.\n",
      "-- check region:1355 20:55485001-55515000_strand_+_gene_1355, 57148 candidate probes\n",
      "--- 9763 probes passed check_dic selection.\n",
      "finish in 1.253s, 187 probes kept.\n",
      "-- check region:1356 20:56485001-56515000_strand_+_gene_1356, 59004 candidate probes\n",
      "--- 16289 probes passed check_dic selection.\n",
      "finish in 1.530s, 291 probes kept.\n",
      "-- check region:1357 20:57485001-57515000_strand_+_gene_1357, 58458 candidate probes\n",
      "--- 12802 probes passed check_dic selection.\n",
      "finish in 1.403s, 248 probes kept.\n",
      "-- check region:1358 20:58485001-58515000_strand_+_gene_1358, 59608 candidate probes\n",
      "--- 20391 probes passed check_dic selection.\n",
      "finish in 1.681s, 327 probes kept.\n",
      "-- check region:1359 20:59485001-59515000_strand_+_gene_1359, 57672 candidate probes\n",
      "--- 16841 probes passed check_dic selection.\n",
      "finish in 1.538s, 267 probes kept.\n",
      "-- check region:1360 20:60485001-60515000_strand_+_gene_1360, 58954 candidate probes\n",
      "--- 21254 probes passed check_dic selection.\n",
      "finish in 1.711s, 324 probes kept.\n",
      "-- check region:1361 20:61485001-61515000_strand_+_gene_1361, 59334 candidate probes\n",
      "--- 27403 probes passed check_dic selection.\n",
      "finish in 1.945s, 397 probes kept.\n",
      "-- check region:1362 20:62485001-62515000_strand_+_gene_1362, 59254 candidate probes\n",
      "--- 25924 probes passed check_dic selection.\n",
      "finish in 1.928s, 385 probes kept.\n",
      "-- check region:1363 20:63485001-63515000_strand_+_gene_1363, 58414 candidate probes\n",
      "--- 19305 probes passed check_dic selection.\n",
      "finish in 1.614s, 314 probes kept.\n",
      "-- check region:1364 21:485001-515000_strand_+_gene_1364, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.002s, 0 probes kept.\n",
      "-- check region:1365 21:1485001-1515000_strand_+_gene_1365, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1366 21:2485001-2515000_strand_+_gene_1366, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1367 21:3485001-3515000_strand_+_gene_1367, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1368 21:4485001-4515000_strand_+_gene_1368, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1369 21:5485001-5515000_strand_+_gene_1369, 31680 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.510s, 0 probes kept.\n",
      "-- check region:1370 21:6485001-6515000_strand_+_gene_1370, 56744 candidate probes\n",
      "--- 285 probes passed check_dic selection.\n",
      "finish in 0.919s, 10 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1371 21:7485001-7515000_strand_+_gene_1371, 31554 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.523s, 0 probes kept.\n",
      "-- check region:1372 21:8485001-8515000_strand_+_gene_1372, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1373 21:9485001-9515000_strand_+_gene_1373, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1374 21:10485001-10515000_strand_+_gene_1374, 58988 candidate probes\n",
      "--- 609 probes passed check_dic selection.\n",
      "finish in 0.941s, 20 probes kept.\n",
      "-- check region:1375 21:11485001-11515000_strand_+_gene_1375, 22244 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.358s, 0 probes kept.\n",
      "-- check region:1376 21:12485001-12515000_strand_+_gene_1376, 14540 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.236s, 0 probes kept.\n",
      "-- check region:1377 21:13485001-13515000_strand_+_gene_1377, 58784 candidate probes\n",
      "--- 511 probes passed check_dic selection.\n",
      "finish in 0.931s, 28 probes kept.\n",
      "-- check region:1378 21:14485001-14515000_strand_+_gene_1378, 59228 candidate probes\n",
      "--- 10625 probes passed check_dic selection.\n",
      "finish in 1.307s, 217 probes kept.\n",
      "-- check region:1379 21:15485001-15515000_strand_+_gene_1379, 59880 candidate probes\n",
      "--- 14963 probes passed check_dic selection.\n",
      "finish in 1.483s, 292 probes kept.\n",
      "-- check region:1380 21:16485001-16515000_strand_+_gene_1380, 59458 candidate probes\n",
      "--- 8539 probes passed check_dic selection.\n",
      "finish in 1.228s, 192 probes kept.\n",
      "-- check region:1381 21:17485001-17515000_strand_+_gene_1381, 57600 candidate probes\n",
      "--- 11588 probes passed check_dic selection.\n",
      "finish in 1.327s, 214 probes kept.\n",
      "-- check region:1382 21:18485001-18515000_strand_+_gene_1382, 59346 candidate probes\n",
      "--- 10356 probes passed check_dic selection.\n",
      "finish in 1.295s, 210 probes kept.\n",
      "-- check region:1383 21:19485001-19515000_strand_+_gene_1383, 59682 candidate probes\n",
      "--- 12771 probes passed check_dic selection.\n",
      "finish in 1.394s, 230 probes kept.\n",
      "-- check region:1384 21:20485001-20515000_strand_+_gene_1384, 59630 candidate probes\n",
      "--- 6867 probes passed check_dic selection.\n",
      "finish in 1.180s, 148 probes kept.\n",
      "-- check region:1385 21:21485001-21515000_strand_+_gene_1385, 59420 candidate probes\n",
      "--- 5879 probes passed check_dic selection.\n",
      "finish in 1.113s, 163 probes kept.\n",
      "-- check region:1386 21:22485001-22515000_strand_+_gene_1386, 59050 candidate probes\n",
      "--- 4613 probes passed check_dic selection.\n",
      "finish in 1.075s, 128 probes kept.\n",
      "-- check region:1387 21:23485001-23515000_strand_+_gene_1387, 59658 candidate probes\n",
      "--- 8588 probes passed check_dic selection.\n",
      "finish in 1.228s, 190 probes kept.\n",
      "-- check region:1388 21:24485001-24515000_strand_+_gene_1388, 59020 candidate probes\n",
      "--- 7306 probes passed check_dic selection.\n",
      "finish in 1.177s, 176 probes kept.\n",
      "-- check region:1389 21:25485001-25515000_strand_+_gene_1389, 59446 candidate probes\n",
      "--- 13679 probes passed check_dic selection.\n",
      "finish in 1.423s, 251 probes kept.\n",
      "-- check region:1390 21:26485001-26515000_strand_+_gene_1390, 59440 candidate probes\n",
      "--- 16291 probes passed check_dic selection.\n",
      "finish in 1.514s, 288 probes kept.\n",
      "-- check region:1391 21:27485001-27515000_strand_+_gene_1391, 59702 candidate probes\n",
      "--- 10896 probes passed check_dic selection.\n",
      "finish in 1.316s, 233 probes kept.\n",
      "-- check region:1392 21:28485001-28515000_strand_+_gene_1392, 59326 candidate probes\n",
      "--- 12032 probes passed check_dic selection.\n",
      "finish in 1.366s, 244 probes kept.\n",
      "-- check region:1393 21:29485001-29515000_strand_+_gene_1393, 59066 candidate probes\n",
      "--- 20103 probes passed check_dic selection.\n",
      "finish in 1.662s, 323 probes kept.\n",
      "-- check region:1394 21:30485001-30515000_strand_+_gene_1394, 59386 candidate probes\n",
      "--- 8207 probes passed check_dic selection.\n",
      "finish in 1.221s, 174 probes kept.\n",
      "-- check region:1395 21:31485001-31515000_strand_+_gene_1395, 58500 candidate probes\n",
      "--- 17069 probes passed check_dic selection.\n",
      "finish in 1.553s, 282 probes kept.\n",
      "-- check region:1396 21:32485001-32515000_strand_+_gene_1396, 59026 candidate probes\n",
      "--- 9040 probes passed check_dic selection.\n",
      "finish in 1.283s, 192 probes kept.\n",
      "-- check region:1397 21:33485001-33515000_strand_+_gene_1397, 57250 candidate probes\n",
      "--- 518 probes passed check_dic selection.\n",
      "finish in 0.940s, 12 probes kept.\n",
      "-- check region:1398 21:34485001-34515000_strand_+_gene_1398, 59502 candidate probes\n",
      "--- 10557 probes passed check_dic selection.\n",
      "finish in 1.337s, 184 probes kept.\n",
      "-- check region:1399 21:35485001-35515000_strand_+_gene_1399, 52718 candidate probes\n",
      "--- 16742 probes passed check_dic selection.\n",
      "finish in 1.438s, 268 probes kept.\n",
      "-- check region:1400 21:36485001-36515000_strand_+_gene_1400, 58570 candidate probes\n",
      "--- 19307 probes passed check_dic selection.\n",
      "finish in 1.646s, 308 probes kept.\n",
      "-- check region:1401 21:37485001-37515000_strand_+_gene_1401, 59804 candidate probes\n",
      "--- 12135 probes passed check_dic selection.\n",
      "finish in 1.378s, 268 probes kept.\n",
      "-- check region:1402 21:38485001-38515000_strand_+_gene_1402, 59248 candidate probes\n",
      "--- 19212 probes passed check_dic selection.\n",
      "finish in 1.627s, 308 probes kept.\n",
      "-- check region:1403 21:39485001-39515000_strand_+_gene_1403, 58465 candidate probes\n",
      "--- 12226 probes passed check_dic selection.\n",
      "finish in 1.371s, 223 probes kept.\n",
      "-- check region:1404 21:40485001-40515000_strand_+_gene_1404, 58622 candidate probes\n",
      "--- 13851 probes passed check_dic selection.\n",
      "finish in 1.417s, 235 probes kept.\n",
      "-- check region:1405 21:41485001-41515000_strand_+_gene_1405, 58990 candidate probes\n",
      "--- 21181 probes passed check_dic selection.\n",
      "finish in 1.707s, 336 probes kept.\n",
      "-- check region:1406 21:42485001-42515000_strand_+_gene_1406, 59476 candidate probes\n",
      "--- 18463 probes passed check_dic selection.\n",
      "finish in 1.625s, 328 probes kept.\n",
      "-- check region:1407 21:43485001-43515000_strand_+_gene_1407, 59266 candidate probes\n",
      "--- 594 probes passed check_dic selection.\n",
      "finish in 0.997s, 16 probes kept.\n",
      "-- check region:1408 21:44485001-44515000_strand_+_gene_1408, 57570 candidate probes\n",
      "--- 21544 probes passed check_dic selection.\n",
      "finish in 1.713s, 354 probes kept.\n",
      "-- check region:1409 21:45485001-45515000_strand_+_gene_1409, 57600 candidate probes\n",
      "--- 21632 probes passed check_dic selection.\n",
      "finish in 1.729s, 374 probes kept.\n",
      "-- check region:1410 22:485001-515000_strand_+_gene_1410, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1411 22:1485001-1515000_strand_+_gene_1411, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1412 22:2485001-2515000_strand_+_gene_1412, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1413 22:3485001-3515000_strand_+_gene_1413, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1414 22:4485001-4515000_strand_+_gene_1414, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1415 22:5485001-5515000_strand_+_gene_1415, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1416 22:6485001-6515000_strand_+_gene_1416, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1417 22:7485001-7515000_strand_+_gene_1417, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1418 22:8485001-8515000_strand_+_gene_1418, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1419 22:9485001-9515000_strand_+_gene_1419, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1420 22:10485001-10515000_strand_+_gene_1420, 9918 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.154s, 0 probes kept.\n",
      "-- check region:1421 22:11485001-11515000_strand_+_gene_1421, 24432 candidate probes\n",
      "--- 34 probes passed check_dic selection.\n",
      "finish in 0.390s, 2 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1422 22:12485001-12515000_strand_+_gene_1422, 49006 candidate probes\n",
      "--- 6 probes passed check_dic selection.\n",
      "finish in 0.767s, 2 probes kept.\n",
      "-- check region:1423 22:13485001-13515000_strand_+_gene_1423, 30824 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.494s, 0 probes kept.\n",
      "-- check region:1424 22:14485001-14515000_strand_+_gene_1424, 24076 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.387s, 0 probes kept.\n",
      "-- check region:1425 22:15485001-15515000_strand_+_gene_1425, 57756 candidate probes\n",
      "--- 70 probes passed check_dic selection.\n",
      "finish in 0.915s, 4 probes kept.\n",
      "-- check region:1426 22:16485001-16515000_strand_+_gene_1426, 57836 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.924s, 0 probes kept.\n",
      "-- check region:1427 22:17485001-17515000_strand_+_gene_1427, 58050 candidate probes\n",
      "--- 13549 probes passed check_dic selection.\n",
      "finish in 1.415s, 253 probes kept.\n",
      "-- check region:1428 22:18485001-18515000_strand_+_gene_1428, 59258 candidate probes\n",
      "--- 12 probes passed check_dic selection.\n",
      "finish in 0.961s, 1 probes kept.\n",
      "-- check region:1429 22:19485001-19515000_strand_+_gene_1429, 58994 candidate probes\n",
      "--- 15548 probes passed check_dic selection.\n",
      "finish in 1.507s, 266 probes kept.\n",
      "-- check region:1430 22:20485001-20515000_strand_+_gene_1430, 57474 candidate probes\n",
      "--- 14857 probes passed check_dic selection.\n",
      "finish in 1.462s, 240 probes kept.\n",
      "-- check region:1431 22:21485001-21515000_strand_+_gene_1431, 58600 candidate probes\n",
      "--- 226 probes passed check_dic selection.\n",
      "finish in 0.975s, 10 probes kept.\n",
      "-- check region:1432 22:22485001-22515000_strand_+_gene_1432, 58804 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.941s, 0 probes kept.\n",
      "-- check region:1433 22:23485001-23515000_strand_+_gene_1433, 58844 candidate probes\n",
      "--- 652 probes passed check_dic selection.\n",
      "finish in 0.980s, 28 probes kept.\n",
      "-- check region:1434 22:24485001-24515000_strand_+_gene_1434, 58880 candidate probes\n",
      "--- 18319 probes passed check_dic selection.\n",
      "finish in 1.604s, 295 probes kept.\n",
      "-- check region:1435 22:25485001-25515000_strand_+_gene_1435, 59130 candidate probes\n",
      "--- 15385 probes passed check_dic selection.\n",
      "finish in 1.502s, 255 probes kept.\n",
      "-- check region:1436 22:26485001-26515000_strand_+_gene_1436, 59232 candidate probes\n",
      "--- 15898 probes passed check_dic selection.\n",
      "finish in 1.549s, 298 probes kept.\n",
      "-- check region:1437 22:27485001-27515000_strand_+_gene_1437, 59372 candidate probes\n",
      "--- 23777 probes passed check_dic selection.\n",
      "finish in 1.830s, 360 probes kept.\n",
      "-- check region:1438 22:28485001-28515000_strand_+_gene_1438, 58654 candidate probes\n",
      "--- 8365 probes passed check_dic selection.\n",
      "finish in 1.218s, 174 probes kept.\n",
      "-- check region:1439 22:29485001-29515000_strand_+_gene_1439, 57212 candidate probes\n",
      "--- 10096 probes passed check_dic selection.\n",
      "finish in 1.290s, 198 probes kept.\n",
      "-- check region:1440 22:30485001-30515000_strand_+_gene_1440, 57876 candidate probes\n",
      "--- 14068 probes passed check_dic selection.\n",
      "finish in 1.456s, 236 probes kept.\n",
      "-- check region:1441 22:31485001-31515000_strand_+_gene_1441, 58342 candidate probes\n",
      "--- 10172 probes passed check_dic selection.\n",
      "finish in 1.302s, 184 probes kept.\n",
      "-- check region:1442 22:32485001-32515000_strand_+_gene_1442, 58900 candidate probes\n",
      "--- 12718 probes passed check_dic selection.\n",
      "finish in 1.406s, 248 probes kept.\n",
      "-- check region:1443 22:33485001-33515000_strand_+_gene_1443, 59184 candidate probes\n",
      "--- 20151 probes passed check_dic selection.\n",
      "finish in 1.684s, 316 probes kept.\n",
      "-- check region:1444 22:34485001-34515000_strand_+_gene_1444, 59824 candidate probes\n",
      "--- 18930 probes passed check_dic selection.\n",
      "finish in 1.630s, 314 probes kept.\n",
      "-- check region:1445 22:35485001-35515000_strand_+_gene_1445, 58838 candidate probes\n",
      "--- 18043 probes passed check_dic selection.\n",
      "finish in 1.636s, 316 probes kept.\n",
      "-- check region:1446 22:36485001-36515000_strand_+_gene_1446, 58398 candidate probes\n",
      "--- 16684 probes passed check_dic selection.\n",
      "finish in 1.561s, 274 probes kept.\n",
      "-- check region:1447 22:37485001-37515000_strand_+_gene_1447, 58966 candidate probes\n",
      "--- 23550 probes passed check_dic selection.\n",
      "finish in 1.851s, 380 probes kept.\n",
      "-- check region:1448 22:38485001-38515000_strand_+_gene_1448, 59078 candidate probes\n",
      "--- 12851 probes passed check_dic selection.\n",
      "finish in 1.421s, 267 probes kept.\n",
      "-- check region:1449 22:39485001-39515000_strand_+_gene_1449, 58714 candidate probes\n",
      "--- 19186 probes passed check_dic selection.\n",
      "finish in 1.662s, 343 probes kept.\n",
      "-- check region:1450 22:40485001-40515000_strand_+_gene_1450, 57068 candidate probes\n",
      "--- 11704 probes passed check_dic selection.\n",
      "finish in 1.314s, 211 probes kept.\n",
      "-- check region:1451 22:41485001-41515000_strand_+_gene_1451, 58680 candidate probes\n",
      "--- 16779 probes passed check_dic selection.\n",
      "finish in 1.557s, 255 probes kept.\n",
      "-- check region:1452 22:42485001-42515000_strand_+_gene_1452, 58410 candidate probes\n",
      "--- 5656 probes passed check_dic selection.\n",
      "finish in 1.162s, 113 probes kept.\n",
      "-- check region:1453 22:43485001-43515000_strand_+_gene_1453, 58166 candidate probes\n",
      "--- 19071 probes passed check_dic selection.\n",
      "finish in 1.640s, 304 probes kept.\n",
      "-- check region:1454 22:44485001-44515000_strand_+_gene_1454, 59152 candidate probes\n",
      "--- 13931 probes passed check_dic selection.\n",
      "finish in 1.465s, 277 probes kept.\n",
      "-- check region:1455 22:45485001-45515000_strand_+_gene_1455, 59142 candidate probes\n",
      "--- 17915 probes passed check_dic selection.\n",
      "finish in 1.587s, 290 probes kept.\n",
      "-- check region:1456 22:46485001-46515000_strand_+_gene_1456, 58534 candidate probes\n",
      "--- 18561 probes passed check_dic selection.\n",
      "finish in 1.615s, 270 probes kept.\n",
      "-- check region:1457 22:47485001-47515000_strand_+_gene_1457, 59836 candidate probes\n",
      "--- 16255 probes passed check_dic selection.\n",
      "finish in 1.532s, 281 probes kept.\n",
      "-- check region:1458 22:48485001-48515000_strand_+_gene_1458, 58512 candidate probes\n",
      "--- 27765 probes passed check_dic selection.\n",
      "finish in 1.949s, 396 probes kept.\n",
      "-- check region:1459 22:49485001-49515000_strand_+_gene_1459, 58628 candidate probes\n",
      "--- 21007 probes passed check_dic selection.\n",
      "finish in 1.699s, 326 probes kept.\n",
      "-- check region:1460 X:1235001-1265000_strand_+_gene_1460, 55362 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.899s, 0 probes kept.\n",
      "-- check region:1461 X:3735001-3765000_strand_+_gene_1461, 58728 candidate probes\n",
      "--- 6190 probes passed check_dic selection.\n",
      "finish in 1.162s, 143 probes kept.\n",
      "-- check region:1462 X:6235001-6265000_strand_+_gene_1462, 59254 candidate probes\n",
      "--- 12499 probes passed check_dic selection.\n",
      "finish in 1.378s, 238 probes kept.\n",
      "-- check region:1463 X:8735001-8765000_strand_+_gene_1463, 59380 candidate probes\n",
      "--- 14874 probes passed check_dic selection.\n",
      "finish in 1.470s, 268 probes kept.\n",
      "-- check region:1464 X:11235001-11265000_strand_+_gene_1464, 59812 candidate probes\n",
      "--- 15527 probes passed check_dic selection.\n",
      "finish in 1.503s, 281 probes kept.\n",
      "-- check region:1465 X:13735001-13765000_strand_+_gene_1465, 59464 candidate probes\n",
      "--- 10659 probes passed check_dic selection.\n",
      "finish in 1.319s, 231 probes kept.\n",
      "-- check region:1466 X:16235001-16265000_strand_+_gene_1466, 58333 candidate probes\n",
      "--- 15096 probes passed check_dic selection.\n",
      "finish in 1.460s, 286 probes kept.\n",
      "-- check region:1467 X:18735001-18765000_strand_+_gene_1467, 57376 candidate probes\n",
      "--- 9946 probes passed check_dic selection.\n",
      "finish in 1.265s, 193 probes kept.\n",
      "-- check region:1468 X:21235001-21265000_strand_+_gene_1468, 58220 candidate probes\n",
      "--- 7266 probes passed check_dic selection.\n",
      "finish in 1.172s, 180 probes kept.\n",
      "-- check region:1469 X:23735001-23765000_strand_+_gene_1469, 58684 candidate probes\n",
      "--- 12063 probes passed check_dic selection.\n",
      "finish in 1.370s, 202 probes kept.\n",
      "-- check region:1470 X:26235001-26265000_strand_+_gene_1470, 59730 candidate probes\n",
      "--- 7810 probes passed check_dic selection.\n",
      "finish in 1.205s, 194 probes kept.\n",
      "-- check region:1471 X:28735001-28765000_strand_+_gene_1471, 59248 candidate probes\n",
      "--- 10801 probes passed check_dic selection.\n",
      "finish in 1.326s, 221 probes kept.\n",
      "-- check region:1472 X:31235001-31265000_strand_+_gene_1472, 59684 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 14614 probes passed check_dic selection.\n",
      "finish in 1.463s, 278 probes kept.\n",
      "-- check region:1473 X:33735001-33765000_strand_+_gene_1473, 59578 candidate probes\n",
      "--- 5273 probes passed check_dic selection.\n",
      "finish in 1.121s, 117 probes kept.\n",
      "-- check region:1474 X:36235001-36265000_strand_+_gene_1474, 59624 candidate probes\n",
      "--- 8020 probes passed check_dic selection.\n",
      "finish in 1.218s, 173 probes kept.\n",
      "-- check region:1475 X:38735001-38765000_strand_+_gene_1475, 59298 candidate probes\n",
      "--- 13004 probes passed check_dic selection.\n",
      "finish in 1.402s, 234 probes kept.\n",
      "-- check region:1476 X:41235001-41265000_strand_+_gene_1476, 58072 candidate probes\n",
      "--- 10764 probes passed check_dic selection.\n",
      "finish in 1.325s, 197 probes kept.\n",
      "-- check region:1477 X:43735001-43765000_strand_+_gene_1477, 59508 candidate probes\n",
      "--- 20954 probes passed check_dic selection.\n",
      "finish in 1.703s, 341 probes kept.\n",
      "-- check region:1478 X:46235001-46265000_strand_+_gene_1478, 59218 candidate probes\n",
      "--- 12451 probes passed check_dic selection.\n",
      "finish in 1.391s, 230 probes kept.\n",
      "-- check region:1479 X:48735001-48765000_strand_+_gene_1479, 59316 candidate probes\n",
      "--- 13858 probes passed check_dic selection.\n",
      "finish in 1.462s, 216 probes kept.\n",
      "-- check region:1480 X:51235001-51265000_strand_+_gene_1480, 52854 candidate probes\n",
      "--- 5648 probes passed check_dic selection.\n",
      "finish in 1.033s, 117 probes kept.\n",
      "-- check region:1481 X:53735001-53765000_strand_+_gene_1481, 58697 candidate probes\n",
      "--- 9625 probes passed check_dic selection.\n",
      "finish in 1.273s, 198 probes kept.\n",
      "-- check region:1482 X:56235001-56265000_strand_+_gene_1482, 59576 candidate probes\n",
      "--- 9227 probes passed check_dic selection.\n",
      "finish in 1.272s, 212 probes kept.\n",
      "-- check region:1483 X:58735001-58765000_strand_+_gene_1483, 9078 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.149s, 0 probes kept.\n",
      "-- check region:1484 X:61235001-61265000_strand_+_gene_1484, 14938 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.245s, 0 probes kept.\n",
      "-- check region:1485 X:63735001-63765000_strand_+_gene_1485, 59766 candidate probes\n",
      "--- 21627 probes passed check_dic selection.\n",
      "finish in 1.762s, 351 probes kept.\n",
      "-- check region:1486 X:66235001-66265000_strand_+_gene_1486, 59378 candidate probes\n",
      "--- 11101 probes passed check_dic selection.\n",
      "finish in 1.331s, 227 probes kept.\n",
      "-- check region:1487 X:68735001-68765000_strand_+_gene_1487, 54946 candidate probes\n",
      "--- 16748 probes passed check_dic selection.\n",
      "finish in 1.496s, 250 probes kept.\n",
      "-- check region:1488 X:71235001-71265000_strand_+_gene_1488, 59148 candidate probes\n",
      "--- 17961 probes passed check_dic selection.\n",
      "finish in 1.637s, 311 probes kept.\n",
      "-- check region:1489 X:73735001-73765000_strand_+_gene_1489, 59226 candidate probes\n",
      "--- 3364 probes passed check_dic selection.\n",
      "finish in 1.043s, 94 probes kept.\n",
      "-- check region:1490 X:76235001-76265000_strand_+_gene_1490, 58148 candidate probes\n",
      "--- 6258 probes passed check_dic selection.\n",
      "finish in 1.142s, 128 probes kept.\n",
      "-- check region:1491 X:78735001-78765000_strand_+_gene_1491, 58668 candidate probes\n",
      "--- 11796 probes passed check_dic selection.\n",
      "finish in 1.349s, 239 probes kept.\n",
      "-- check region:1492 X:81235001-81265000_strand_+_gene_1492, 59818 candidate probes\n",
      "--- 11494 probes passed check_dic selection.\n",
      "finish in 1.349s, 238 probes kept.\n",
      "-- check region:1493 X:83735001-83765000_strand_+_gene_1493, 55734 candidate probes\n",
      "--- 4698 probes passed check_dic selection.\n",
      "finish in 1.040s, 129 probes kept.\n",
      "-- check region:1494 X:86235001-86265000_strand_+_gene_1494, 59918 candidate probes\n",
      "--- 8284 probes passed check_dic selection.\n",
      "finish in 1.227s, 189 probes kept.\n",
      "-- check region:1495 X:88735001-88765000_strand_+_gene_1495, 58958 candidate probes\n",
      "--- 9101 probes passed check_dic selection.\n",
      "finish in 1.247s, 199 probes kept.\n",
      "-- check region:1496 X:91235001-91265000_strand_+_gene_1496, 59392 candidate probes\n",
      "--- 2340 probes passed check_dic selection.\n",
      "finish in 1.004s, 72 probes kept.\n",
      "-- check region:1497 X:93735001-93765000_strand_+_gene_1497, 59808 candidate probes\n",
      "--- 11783 probes passed check_dic selection.\n",
      "finish in 1.357s, 230 probes kept.\n",
      "-- check region:1498 X:96235001-96265000_strand_+_gene_1498, 56348 candidate probes\n",
      "--- 6540 probes passed check_dic selection.\n",
      "finish in 1.124s, 146 probes kept.\n",
      "-- check region:1499 X:98735001-98765000_strand_+_gene_1499, 59510 candidate probes\n",
      "--- 7158 probes passed check_dic selection.\n",
      "finish in 1.181s, 160 probes kept.\n",
      "-- check region:1500 X:101235001-101265000_strand_+_gene_1500, 59685 candidate probes\n",
      "--- 21757 probes passed check_dic selection.\n",
      "finish in 1.762s, 369 probes kept.\n",
      "-- check region:1501 X:103735001-103765000_strand_+_gene_1501, 59456 candidate probes\n",
      "--- 14928 probes passed check_dic selection.\n",
      "finish in 1.497s, 253 probes kept.\n",
      "-- check region:1502 X:106235001-106265000_strand_+_gene_1502, 59546 candidate probes\n",
      "--- 6128 probes passed check_dic selection.\n",
      "finish in 1.151s, 139 probes kept.\n",
      "-- check region:1503 X:108735001-108765000_strand_+_gene_1503, 59272 candidate probes\n",
      "--- 9066 probes passed check_dic selection.\n",
      "finish in 1.253s, 186 probes kept.\n",
      "-- check region:1504 X:111235001-111265000_strand_+_gene_1504, 59732 candidate probes\n",
      "--- 16068 probes passed check_dic selection.\n",
      "finish in 1.532s, 285 probes kept.\n",
      "-- check region:1505 X:113735001-113765000_strand_+_gene_1505, 58880 candidate probes\n",
      "--- 12208 probes passed check_dic selection.\n",
      "finish in 1.372s, 239 probes kept.\n",
      "-- check region:1506 X:116235001-116265000_strand_+_gene_1506, 57626 candidate probes\n",
      "--- 7983 probes passed check_dic selection.\n",
      "finish in 1.210s, 163 probes kept.\n",
      "-- check region:1507 X:118735001-118765000_strand_+_gene_1507, 59792 candidate probes\n",
      "--- 13805 probes passed check_dic selection.\n",
      "finish in 1.448s, 242 probes kept.\n",
      "-- check region:1508 X:121235001-121265000_strand_+_gene_1508, 58342 candidate probes\n",
      "--- 9602 probes passed check_dic selection.\n",
      "finish in 1.258s, 189 probes kept.\n",
      "-- check region:1509 X:123735001-123765000_strand_+_gene_1509, 58824 candidate probes\n",
      "--- 10320 probes passed check_dic selection.\n",
      "finish in 1.327s, 188 probes kept.\n",
      "-- check region:1510 X:126235001-126265000_strand_+_gene_1510, 59190 candidate probes\n",
      "--- 7934 probes passed check_dic selection.\n",
      "finish in 1.206s, 176 probes kept.\n",
      "-- check region:1511 X:128735001-128765000_strand_+_gene_1511, 55998 candidate probes\n",
      "--- 10713 probes passed check_dic selection.\n",
      "finish in 1.265s, 218 probes kept.\n",
      "-- check region:1512 X:131235001-131265000_strand_+_gene_1512, 55748 candidate probes\n",
      "--- 14099 probes passed check_dic selection.\n",
      "finish in 1.391s, 248 probes kept.\n",
      "-- check region:1513 X:133735001-133765000_strand_+_gene_1513, 59614 candidate probes\n",
      "--- 13108 probes passed check_dic selection.\n",
      "finish in 1.412s, 245 probes kept.\n",
      "-- check region:1514 X:136235001-136265000_strand_+_gene_1514, 59634 candidate probes\n",
      "--- 13366 probes passed check_dic selection.\n",
      "finish in 1.425s, 262 probes kept.\n",
      "-- check region:1515 X:138735001-138765000_strand_+_gene_1515, 59892 candidate probes\n",
      "--- 22348 probes passed check_dic selection.\n",
      "finish in 1.756s, 357 probes kept.\n",
      "-- check region:1516 X:141235001-141265000_strand_+_gene_1516, 59238 candidate probes\n",
      "--- 5375 probes passed check_dic selection.\n",
      "finish in 1.136s, 103 probes kept.\n",
      "-- check region:1517 X:143735001-143765000_strand_+_gene_1517, 58711 candidate probes\n",
      "--- 11908 probes passed check_dic selection.\n",
      "finish in 1.340s, 232 probes kept.\n",
      "-- check region:1518 X:146235001-146265000_strand_+_gene_1518, 59350 candidate probes\n",
      "--- 10178 probes passed check_dic selection.\n",
      "finish in 1.302s, 217 probes kept.\n",
      "-- check region:1519 X:148735001-148765000_strand_+_gene_1519, 59806 candidate probes\n",
      "--- 13113 probes passed check_dic selection.\n",
      "finish in 1.401s, 239 probes kept.\n",
      "-- check region:1520 X:151235001-151265000_strand_+_gene_1520, 59636 candidate probes\n",
      "--- 12627 probes passed check_dic selection.\n",
      "finish in 1.402s, 221 probes kept.\n",
      "-- check region:1521 X:153735001-153765000_strand_+_gene_1521, 59770 candidate probes\n",
      "--- 9160 probes passed check_dic selection.\n",
      "finish in 1.300s, 176 probes kept.\n",
      "-- check region:1522 Y:485001-515000_strand_+_gene_1522, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1523 Y:1485001-1515000_strand_+_gene_1523, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1524 Y:2485001-2515000_strand_+_gene_1524, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1525 Y:3485001-3515000_strand_+_gene_1525, 59838 candidate probes\n",
      "--- 1152 probes passed check_dic selection.\n",
      "finish in 0.976s, 51 probes kept.\n",
      "-- check region:1526 Y:4485001-4515000_strand_+_gene_1526, 59752 candidate probes\n",
      "--- 1756 probes passed check_dic selection.\n",
      "finish in 0.998s, 60 probes kept.\n",
      "-- check region:1527 Y:5485001-5515000_strand_+_gene_1527, 59722 candidate probes\n",
      "--- 1845 probes passed check_dic selection.\n",
      "finish in 0.989s, 72 probes kept.\n",
      "-- check region:1528 Y:6485001-6515000_strand_+_gene_1528, 58818 candidate probes\n",
      "--- 1213 probes passed check_dic selection.\n",
      "finish in 0.984s, 44 probes kept.\n",
      "-- check region:1529 Y:7485001-7515000_strand_+_gene_1529, 59780 candidate probes\n",
      "--- 6251 probes passed check_dic selection.\n",
      "finish in 1.179s, 166 probes kept.\n",
      "-- check region:1530 Y:8485001-8515000_strand_+_gene_1530, 59662 candidate probes\n",
      "--- 8003 probes passed check_dic selection.\n",
      "finish in 1.245s, 218 probes kept.\n",
      "-- check region:1531 Y:9485001-9515000_strand_+_gene_1531, 43183 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.692s, 0 probes kept.\n",
      "-- check region:1532 Y:10485001-10515000_strand_+_gene_1532, 14916 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.235s, 0 probes kept.\n",
      "-- check region:1533 Y:11485001-11515000_strand_+_gene_1533, 59602 candidate probes\n",
      "--- 1038 probes passed check_dic selection.\n",
      "finish in 0.980s, 69 probes kept.\n",
      "-- check region:1534 Y:12485001-12515000_strand_+_gene_1534, 59602 candidate probes\n",
      "--- 5891 probes passed check_dic selection.\n",
      "finish in 1.178s, 149 probes kept.\n",
      "-- check region:1535 Y:13485001-13515000_strand_+_gene_1535, 59790 candidate probes\n",
      "--- 8766 probes passed check_dic selection.\n",
      "finish in 1.261s, 187 probes kept.\n",
      "-- check region:1536 Y:14485001-14515000_strand_+_gene_1536, 59418 candidate probes\n",
      "--- 8322 probes passed check_dic selection.\n",
      "finish in 1.251s, 187 probes kept.\n",
      "-- check region:1537 Y:15485001-15515000_strand_+_gene_1537, 59910 candidate probes\n",
      "--- 7808 probes passed check_dic selection.\n",
      "finish in 1.229s, 170 probes kept.\n",
      "-- check region:1538 Y:16485001-16515000_strand_+_gene_1538, 59362 candidate probes\n",
      "--- 6829 probes passed check_dic selection.\n",
      "finish in 1.165s, 164 probes kept.\n",
      "-- check region:1539 Y:17485001-17515000_strand_+_gene_1539, 59880 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.940s, 0 probes kept.\n",
      "-- check region:1540 Y:18485001-18515000_strand_+_gene_1540, 59529 candidate probes\n",
      "--- 3 probes passed check_dic selection.\n",
      "finish in 0.920s, 1 probes kept.\n",
      "-- check region:1541 Y:19485001-19515000_strand_+_gene_1541, 59676 candidate probes\n",
      "--- 13487 probes passed check_dic selection.\n",
      "finish in 1.427s, 251 probes kept.\n",
      "-- check region:1542 Y:20485001-20515000_strand_+_gene_1542, 58894 candidate probes\n",
      "--- 10008 probes passed check_dic selection.\n",
      "finish in 1.274s, 204 probes kept.\n",
      "-- check region:1543 Y:21485001-21515000_strand_+_gene_1543, 59474 candidate probes\n",
      "--- 670 probes passed check_dic selection.\n",
      "finish in 0.958s, 30 probes kept.\n",
      "-- check region:1544 Y:22485001-22515000_strand_+_gene_1544, 59894 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.966s, 0 probes kept.\n",
      "-- check region:1545 Y:23485001-23515000_strand_+_gene_1545, 59870 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.938s, 0 probes kept.\n",
      "-- check region:1546 Y:24485001-24515000_strand_+_gene_1546, 59200 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.951s, 0 probes kept.\n",
      "-- check region:1547 Y:25485001-25515000_strand_+_gene_1547, 59862 candidate probes\n",
      "--- 45 probes passed check_dic selection.\n",
      "finish in 0.959s, 2 probes kept.\n",
      "-- check region:1548 Y:26485001-26515000_strand_+_gene_1548, 59088 candidate probes\n",
      "--- 3246 probes passed check_dic selection.\n",
      "finish in 1.051s, 106 probes kept.\n",
      "-- check region:1549 Y:27485001-27515000_strand_+_gene_1549, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1550 Y:28485001-28515000_strand_+_gene_1550, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1551 Y:29485001-29515000_strand_+_gene_1551, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1552 Y:30485001-30515000_strand_+_gene_1552, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1553 Y:31485001-31515000_strand_+_gene_1553, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1554 Y:32485001-32515000_strand_+_gene_1554, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1555 Y:33485001-33515000_strand_+_gene_1555, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1556 Y:34485001-34515000_strand_+_gene_1556, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1557 Y:35485001-35515000_strand_+_gene_1557, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1558 Y:36485001-36515000_strand_+_gene_1558, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1559 Y:37485001-37515000_strand_+_gene_1559, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1560 Y:38485001-38515000_strand_+_gene_1560, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1561 Y:39485001-39515000_strand_+_gene_1561, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1562 Y:40485001-40515000_strand_+_gene_1562, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1563 Y:41485001-41515000_strand_+_gene_1563, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1564 Y:42485001-42515000_strand_+_gene_1564, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1565 Y:43485001-43515000_strand_+_gene_1565, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1566 Y:44485001-44515000_strand_+_gene_1566, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1567 Y:45485001-45515000_strand_+_gene_1567, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1568 Y:46485001-46515000_strand_+_gene_1568, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1569 Y:47485001-47515000_strand_+_gene_1569, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1570 Y:48485001-48515000_strand_+_gene_1570, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1571 Y:49485001-49515000_strand_+_gene_1571, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1572 Y:50485001-50515000_strand_+_gene_1572, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1573 Y:51485001-51515000_strand_+_gene_1573, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1574 Y:52485001-52515000_strand_+_gene_1574, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1575 Y:53485001-53515000_strand_+_gene_1575, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1576 Y:54485001-54515000_strand_+_gene_1576, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1577 Y:55485001-55515000_strand_+_gene_1577, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1578 Y:56485001-56515000_strand_+_gene_1578, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n"
     ]
    }
   ],
   "source": [
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ae777bea",
   "metadata": {},
   "source": [
    "### save probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "4b217c7c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr\n",
      "-- number of probes kept: 328466\n"
     ]
    }
   ],
   "source": [
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "id": "f342e259",
   "metadata": {},
   "outputs": [],
   "source": [
    "del(pb_designer)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e728e379",
   "metadata": {},
   "source": [
    "## Reload from here:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "id": "842303ee",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr\n",
      "- Loading from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr.\n",
      "- loading cand_probes from file\n",
      "- loading kept_probes from file\n",
      "- 328466 filtered probes loaded\n",
      "- loading sequence_dic from file\n",
      "- loading map_dic from file\n",
      "- loading params_dic from file\n",
      "- loading check_dic from file\n",
      "- loading save_file from file\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 98,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "if 'probe_savefile' not in locals():\n",
    "    probe_savefile = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "    print(probe_savefile)\n",
    "\n",
    "# Parse segemented list\n",
    "_pbde = library_tools.design.pb_reports_class()\n",
    "_pbde.load_from_file(probe_savefile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "id": "7a63afbb",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sort genes\n",
      "- 1406 genes\n",
      "-- appending region:0 with 344 probes\n",
      "-- appending region:1 with 319 probes\n",
      "-- appending region:10 with 216 probes\n",
      "-- appending region:100 with 344 probes\n",
      "-- appending region:1000 with 326 probes\n",
      "-- appending region:1001 with 341 probes\n",
      "-- appending region:1002 with 381 probes\n",
      "-- appending region:1003 with 356 probes\n",
      "-- appending region:1004 with 283 probes\n",
      "-- appending region:1005 with 321 probes\n",
      "-- appending region:101 with 286 probes\n",
      "-- appending region:102 with 240 probes\n",
      "-- appending region:1020 with 52 probes\n",
      "-- appending region:1021 with 11 probes\n",
      "-- appending region:1023 with 217 probes\n",
      "-- appending region:1024 with 242 probes\n",
      "-- appending region:1025 with 306 probes\n",
      "-- appending region:1026 with 17 probes\n",
      "-- appending region:1027 with 3 probes\n",
      "-- appending region:1028 with 1 probes\n",
      "-- appending region:1029 with 235 probes\n",
      "-- appending region:103 with 205 probes\n",
      "-- appending region:1030 with 209 probes\n",
      "-- appending region:1031 with 290 probes\n",
      "-- appending region:1032 with 275 probes\n",
      "-- appending region:1033 with 234 probes\n",
      "-- appending region:1034 with 183 probes\n",
      "-- appending region:1035 with 201 probes\n",
      "-- appending region:1036 with 130 probes\n",
      "-- appending region:1037 with 237 probes\n",
      "-- appending region:1038 with 227 probes\n",
      "-- appending region:1039 with 246 probes\n",
      "-- appending region:104 with 119 probes\n",
      "-- appending region:1040 with 350 probes\n",
      "-- appending region:1041 with 254 probes\n",
      "-- appending region:1042 with 325 probes\n",
      "-- appending region:1043 with 122 probes\n",
      "-- appending region:1044 with 253 probes\n",
      "-- appending region:1045 with 347 probes\n",
      "-- appending region:1046 with 263 probes\n",
      "-- appending region:1047 with 350 probes\n",
      "-- appending region:1048 with 296 probes\n",
      "-- appending region:1049 with 176 probes\n",
      "-- appending region:105 with 352 probes\n",
      "-- appending region:1050 with 267 probes\n",
      "-- appending region:1051 with 384 probes\n",
      "-- appending region:1052 with 135 probes\n",
      "-- appending region:1053 with 351 probes\n",
      "-- appending region:1054 with 320 probes\n",
      "-- appending region:1055 with 268 probes\n",
      "-- appending region:1056 with 361 probes\n",
      "-- appending region:1057 with 155 probes\n",
      "-- appending region:1058 with 131 probes\n",
      "-- appending region:1059 with 164 probes\n",
      "-- appending region:106 with 244 probes\n",
      "-- appending region:1060 with 315 probes\n",
      "-- appending region:1061 with 309 probes\n",
      "-- appending region:1062 with 240 probes\n",
      "-- appending region:1063 with 266 probes\n",
      "-- appending region:1064 with 327 probes\n",
      "-- appending region:1065 with 318 probes\n",
      "-- appending region:1066 with 207 probes\n",
      "-- appending region:1067 with 204 probes\n",
      "-- appending region:1068 with 147 probes\n",
      "-- appending region:1069 with 160 probes\n",
      "-- appending region:107 with 305 probes\n",
      "-- appending region:1070 with 250 probes\n",
      "-- appending region:1071 with 291 probes\n",
      "-- appending region:1072 with 259 probes\n",
      "-- appending region:1073 with 263 probes\n",
      "-- appending region:1074 with 273 probes\n",
      "-- appending region:1075 with 256 probes\n",
      "-- appending region:1076 with 293 probes\n",
      "-- appending region:1077 with 88 probes\n",
      "-- appending region:1078 with 241 probes\n",
      "-- appending region:1079 with 302 probes\n",
      "-- appending region:108 with 213 probes\n",
      "-- appending region:1080 with 362 probes\n",
      "-- appending region:1081 with 298 probes\n",
      "-- appending region:1082 with 263 probes\n",
      "-- appending region:1083 with 325 probes\n",
      "-- appending region:1085 with 341 probes\n",
      "-- appending region:1086 with 28 probes\n",
      "-- appending region:1087 with 336 probes\n",
      "-- appending region:1088 with 25 probes\n",
      "-- appending region:1089 with 207 probes\n",
      "-- appending region:109 with 225 probes\n",
      "-- appending region:1090 with 261 probes\n",
      "-- appending region:1091 with 226 probes\n",
      "-- appending region:1092 with 355 probes\n",
      "-- appending region:1093 with 203 probes\n",
      "-- appending region:1094 with 317 probes\n",
      "-- appending region:1095 with 2 probes\n",
      "-- appending region:1096 with 14 probes\n",
      "-- appending region:1097 with 230 probes\n",
      "-- appending region:11 with 212 probes\n",
      "-- appending region:110 with 230 probes\n",
      "-- appending region:1105 with 223 probes\n",
      "-- appending region:1106 with 189 probes\n",
      "-- appending region:1107 with 267 probes\n",
      "-- appending region:1108 with 295 probes\n",
      "-- appending region:1109 with 184 probes\n",
      "-- appending region:111 with 358 probes\n",
      "-- appending region:1110 with 336 probes\n",
      "-- appending region:1111 with 381 probes\n",
      "-- appending region:1112 with 398 probes\n",
      "-- appending region:1113 with 266 probes\n",
      "-- appending region:1114 with 217 probes\n",
      "-- appending region:1115 with 253 probes\n",
      "-- appending region:1116 with 184 probes\n",
      "-- appending region:1117 with 288 probes\n",
      "-- appending region:1118 with 231 probes\n",
      "-- appending region:1119 with 328 probes\n",
      "-- appending region:112 with 167 probes\n",
      "-- appending region:1120 with 228 probes\n",
      "-- appending region:1121 with 298 probes\n",
      "-- appending region:1122 with 227 probes\n",
      "-- appending region:1123 with 267 probes\n",
      "-- appending region:1124 with 201 probes\n",
      "-- appending region:1125 with 208 probes\n",
      "-- appending region:1126 with 320 probes\n",
      "-- appending region:1127 with 241 probes\n",
      "-- appending region:1128 with 235 probes\n",
      "-- appending region:1129 with 335 probes\n",
      "-- appending region:1130 with 313 probes\n",
      "-- appending region:1131 with 272 probes\n",
      "-- appending region:1132 with 290 probes\n",
      "-- appending region:1133 with 205 probes\n",
      "-- appending region:1134 with 331 probes\n",
      "-- appending region:1135 with 268 probes\n",
      "-- appending region:1136 with 294 probes\n",
      "-- appending region:1137 with 196 probes\n",
      "-- appending region:1138 with 222 probes\n",
      "-- appending region:1139 with 256 probes\n",
      "-- appending region:114 with 215 probes\n",
      "-- appending region:1140 with 242 probes\n",
      "-- appending region:1141 with 375 probes\n",
      "-- appending region:1142 with 263 probes\n",
      "-- appending region:1143 with 206 probes\n",
      "-- appending region:1144 with 122 probes\n",
      "-- appending region:1145 with 215 probes\n",
      "-- appending region:1146 with 94 probes\n",
      "-- appending region:1147 with 257 probes\n",
      "-- appending region:1148 with 101 probes\n",
      "-- appending region:115 with 224 probes\n",
      "-- appending region:1152 with 194 probes\n",
      "-- appending region:1153 with 269 probes\n",
      "-- appending region:1154 with 54 probes\n",
      "-- appending region:1155 with 254 probes\n",
      "-- appending region:1156 with 327 probes\n",
      "-- appending region:1157 with 276 probes\n",
      "-- appending region:1158 with 20 probes\n",
      "-- appending region:116 with 228 probes\n",
      "-- appending region:1160 with 317 probes\n",
      "-- appending region:1161 with 3 probes\n",
      "-- appending region:1162 with 222 probes\n",
      "-- appending region:1163 with 272 probes\n",
      "-- appending region:1164 with 3 probes\n",
      "-- appending region:1165 with 270 probes\n",
      "-- appending region:1166 with 342 probes\n",
      "-- appending region:1167 with 192 probes\n",
      "-- appending region:1168 with 346 probes\n",
      "-- appending region:1169 with 185 probes\n",
      "-- appending region:117 with 276 probes\n",
      "-- appending region:1170 with 232 probes\n",
      "-- appending region:1171 with 268 probes\n",
      "-- appending region:1172 with 329 probes\n",
      "-- appending region:1173 with 96 probes\n",
      "-- appending region:1174 with 128 probes\n",
      "-- appending region:1175 with 44 probes\n",
      "-- appending region:1176 with 212 probes\n",
      "-- appending region:1177 with 241 probes\n",
      "-- appending region:1178 with 321 probes\n",
      "-- appending region:1179 with 197 probes\n",
      "-- appending region:118 with 216 probes\n",
      "-- appending region:1180 with 213 probes\n",
      "-- appending region:1181 with 271 probes\n",
      "-- appending region:1182 with 291 probes\n",
      "-- appending region:1183 with 337 probes\n",
      "-- appending region:1184 with 310 probes\n",
      "-- appending region:1185 with 4 probes\n",
      "-- appending region:1186 with 265 probes\n",
      "-- appending region:1187 with 285 probes\n",
      "-- appending region:1188 with 77 probes\n",
      "-- appending region:1189 with 171 probes\n",
      "-- appending region:119 with 221 probes\n",
      "-- appending region:1190 with 187 probes\n",
      "-- appending region:1191 with 194 probes\n",
      "-- appending region:1192 with 242 probes\n",
      "-- appending region:1193 with 279 probes\n",
      "-- appending region:1194 with 291 probes\n",
      "-- appending region:1195 with 288 probes\n",
      "-- appending region:1196 with 277 probes\n",
      "-- appending region:1197 with 184 probes\n",
      "-- appending region:1198 with 30 probes\n",
      "-- appending region:1199 with 2 probes\n",
      "-- appending region:12 with 240 probes\n",
      "-- appending region:120 with 200 probes\n",
      "-- appending region:1203 with 191 probes\n",
      "-- appending region:1204 with 346 probes\n",
      "-- appending region:1205 with 321 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:1206 with 108 probes\n",
      "-- appending region:1207 with 213 probes\n",
      "-- appending region:1208 with 208 probes\n",
      "-- appending region:1209 with 169 probes\n",
      "-- appending region:121 with 295 probes\n",
      "-- appending region:1210 with 270 probes\n",
      "-- appending region:1211 with 302 probes\n",
      "-- appending region:1212 with 278 probes\n",
      "-- appending region:1213 with 293 probes\n",
      "-- appending region:1214 with 317 probes\n",
      "-- appending region:1215 with 221 probes\n",
      "-- appending region:1216 with 216 probes\n",
      "-- appending region:1217 with 190 probes\n",
      "-- appending region:1218 with 235 probes\n",
      "-- appending region:1219 with 315 probes\n",
      "-- appending region:122 with 270 probes\n",
      "-- appending region:1220 with 358 probes\n",
      "-- appending region:1221 with 411 probes\n",
      "-- appending region:1222 with 179 probes\n",
      "-- appending region:1224 with 231 probes\n",
      "-- appending region:1225 with 282 probes\n",
      "-- appending region:1226 with 238 probes\n",
      "-- appending region:1227 with 215 probes\n",
      "-- appending region:1228 with 167 probes\n",
      "-- appending region:1229 with 235 probes\n",
      "-- appending region:123 with 182 probes\n",
      "-- appending region:1230 with 223 probes\n",
      "-- appending region:1231 with 241 probes\n",
      "-- appending region:1232 with 237 probes\n",
      "-- appending region:1233 with 223 probes\n",
      "-- appending region:1234 with 169 probes\n",
      "-- appending region:1235 with 241 probes\n",
      "-- appending region:1236 with 271 probes\n",
      "-- appending region:1237 with 228 probes\n",
      "-- appending region:1238 with 287 probes\n",
      "-- appending region:1239 with 249 probes\n",
      "-- appending region:124 with 235 probes\n",
      "-- appending region:1240 with 297 probes\n",
      "-- appending region:1241 with 344 probes\n",
      "-- appending region:1242 with 279 probes\n",
      "-- appending region:1243 with 297 probes\n",
      "-- appending region:1244 with 260 probes\n",
      "-- appending region:1245 with 283 probes\n",
      "-- appending region:1246 with 305 probes\n",
      "-- appending region:1247 with 234 probes\n",
      "-- appending region:1248 with 160 probes\n",
      "-- appending region:1249 with 237 probes\n",
      "-- appending region:125 with 251 probes\n",
      "-- appending region:1250 with 310 probes\n",
      "-- appending region:1251 with 181 probes\n",
      "-- appending region:1252 with 181 probes\n",
      "-- appending region:1253 with 246 probes\n",
      "-- appending region:1254 with 178 probes\n",
      "-- appending region:1255 with 303 probes\n",
      "-- appending region:1256 with 186 probes\n",
      "-- appending region:1257 with 173 probes\n",
      "-- appending region:1258 with 168 probes\n",
      "-- appending region:1259 with 216 probes\n",
      "-- appending region:126 with 291 probes\n",
      "-- appending region:1260 with 364 probes\n",
      "-- appending region:1261 with 377 probes\n",
      "-- appending region:1262 with 85 probes\n",
      "-- appending region:1263 with 189 probes\n",
      "-- appending region:1264 with 279 probes\n",
      "-- appending region:1265 with 99 probes\n",
      "-- appending region:1269 with 51 probes\n",
      "-- appending region:127 with 222 probes\n",
      "-- appending region:1270 with 305 probes\n",
      "-- appending region:1271 with 312 probes\n",
      "-- appending region:1272 with 317 probes\n",
      "-- appending region:1273 with 297 probes\n",
      "-- appending region:1274 with 192 probes\n",
      "-- appending region:1275 with 356 probes\n",
      "-- appending region:1276 with 244 probes\n",
      "-- appending region:1277 with 362 probes\n",
      "-- appending region:1278 with 150 probes\n",
      "-- appending region:1279 with 193 probes\n",
      "-- appending region:128 with 275 probes\n",
      "-- appending region:1280 with 296 probes\n",
      "-- appending region:1281 with 275 probes\n",
      "-- appending region:1282 with 234 probes\n",
      "-- appending region:1284 with 188 probes\n",
      "-- appending region:1285 with 206 probes\n",
      "-- appending region:1286 with 263 probes\n",
      "-- appending region:1287 with 287 probes\n",
      "-- appending region:1288 with 239 probes\n",
      "-- appending region:1289 with 368 probes\n",
      "-- appending region:129 with 216 probes\n",
      "-- appending region:1290 with 315 probes\n",
      "-- appending region:1291 with 273 probes\n",
      "-- appending region:1292 with 248 probes\n",
      "-- appending region:1293 with 312 probes\n",
      "-- appending region:1294 with 90 probes\n",
      "-- appending region:1295 with 167 probes\n",
      "-- appending region:1296 with 1 probes\n",
      "-- appending region:1297 with 312 probes\n",
      "-- appending region:1298 with 282 probes\n",
      "-- appending region:1299 with 267 probes\n",
      "-- appending region:13 with 341 probes\n",
      "-- appending region:130 with 162 probes\n",
      "-- appending region:1300 with 266 probes\n",
      "-- appending region:1301 with 252 probes\n",
      "-- appending region:1302 with 220 probes\n",
      "-- appending region:1303 with 174 probes\n",
      "-- appending region:1304 with 268 probes\n",
      "-- appending region:1305 with 297 probes\n",
      "-- appending region:1306 with 285 probes\n",
      "-- appending region:1307 with 271 probes\n",
      "-- appending region:1308 with 243 probes\n",
      "-- appending region:1309 with 325 probes\n",
      "-- appending region:131 with 190 probes\n",
      "-- appending region:1310 with 285 probes\n",
      "-- appending region:1311 with 211 probes\n",
      "-- appending region:1312 with 257 probes\n",
      "-- appending region:1313 with 201 probes\n",
      "-- appending region:1314 with 272 probes\n",
      "-- appending region:1315 with 347 probes\n",
      "-- appending region:1316 with 301 probes\n",
      "-- appending region:1317 with 344 probes\n",
      "-- appending region:1318 with 253 probes\n",
      "-- appending region:1319 with 385 probes\n",
      "-- appending region:132 with 192 probes\n",
      "-- appending region:1320 with 286 probes\n",
      "-- appending region:1321 with 321 probes\n",
      "-- appending region:1322 with 310 probes\n",
      "-- appending region:1323 with 306 probes\n",
      "-- appending region:1324 with 321 probes\n",
      "-- appending region:1325 with 367 probes\n",
      "-- appending region:1329 with 35 probes\n",
      "-- appending region:133 with 312 probes\n",
      "-- appending region:1330 with 129 probes\n",
      "-- appending region:1331 with 242 probes\n",
      "-- appending region:1332 with 278 probes\n",
      "-- appending region:1333 with 136 probes\n",
      "-- appending region:1334 with 161 probes\n",
      "-- appending region:1335 with 326 probes\n",
      "-- appending region:1336 with 266 probes\n",
      "-- appending region:1337 with 339 probes\n",
      "-- appending region:1338 with 188 probes\n",
      "-- appending region:1339 with 238 probes\n",
      "-- appending region:134 with 237 probes\n",
      "-- appending region:1340 with 344 probes\n",
      "-- appending region:1341 with 308 probes\n",
      "-- appending region:1342 with 261 probes\n",
      "-- appending region:1343 with 201 probes\n",
      "-- appending region:1344 with 274 probes\n",
      "-- appending region:1345 with 312 probes\n",
      "-- appending region:1346 with 298 probes\n",
      "-- appending region:1347 with 179 probes\n",
      "-- appending region:1348 with 130 probes\n",
      "-- appending region:1349 with 302 probes\n",
      "-- appending region:1350 with 265 probes\n",
      "-- appending region:1351 with 353 probes\n",
      "-- appending region:1352 with 243 probes\n",
      "-- appending region:1353 with 192 probes\n",
      "-- appending region:1354 with 278 probes\n",
      "-- appending region:1355 with 187 probes\n",
      "-- appending region:1356 with 291 probes\n",
      "-- appending region:1357 with 248 probes\n",
      "-- appending region:1358 with 327 probes\n",
      "-- appending region:1359 with 267 probes\n",
      "-- appending region:1360 with 324 probes\n",
      "-- appending region:1361 with 397 probes\n",
      "-- appending region:1362 with 385 probes\n",
      "-- appending region:1363 with 314 probes\n",
      "-- appending region:137 with 315 probes\n",
      "-- appending region:1370 with 10 probes\n",
      "-- appending region:1374 with 20 probes\n",
      "-- appending region:1377 with 28 probes\n",
      "-- appending region:1378 with 217 probes\n",
      "-- appending region:1379 with 292 probes\n",
      "-- appending region:138 with 285 probes\n",
      "-- appending region:1380 with 192 probes\n",
      "-- appending region:1381 with 214 probes\n",
      "-- appending region:1382 with 210 probes\n",
      "-- appending region:1383 with 230 probes\n",
      "-- appending region:1384 with 148 probes\n",
      "-- appending region:1385 with 163 probes\n",
      "-- appending region:1386 with 128 probes\n",
      "-- appending region:1387 with 190 probes\n",
      "-- appending region:1388 with 176 probes\n",
      "-- appending region:1389 with 251 probes\n",
      "-- appending region:139 with 174 probes\n",
      "-- appending region:1390 with 288 probes\n",
      "-- appending region:1391 with 233 probes\n",
      "-- appending region:1392 with 244 probes\n",
      "-- appending region:1393 with 323 probes\n",
      "-- appending region:1394 with 174 probes\n",
      "-- appending region:1395 with 282 probes\n",
      "-- appending region:1396 with 192 probes\n",
      "-- appending region:1397 with 12 probes\n",
      "-- appending region:1398 with 184 probes\n",
      "-- appending region:1399 with 268 probes\n",
      "-- appending region:14 with 176 probes\n",
      "-- appending region:140 with 261 probes\n",
      "-- appending region:1400 with 308 probes\n",
      "-- appending region:1401 with 268 probes\n",
      "-- appending region:1402 with 308 probes\n",
      "-- appending region:1403 with 223 probes\n",
      "-- appending region:1404 with 235 probes\n",
      "-- appending region:1405 with 336 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:1406 with 328 probes\n",
      "-- appending region:1407 with 16 probes\n",
      "-- appending region:1408 with 354 probes\n",
      "-- appending region:1409 with 374 probes\n",
      "-- appending region:141 with 74 probes\n",
      "-- appending region:142 with 39 probes\n",
      "-- appending region:1421 with 2 probes\n",
      "-- appending region:1422 with 2 probes\n",
      "-- appending region:1425 with 4 probes\n",
      "-- appending region:1427 with 253 probes\n",
      "-- appending region:1428 with 1 probes\n",
      "-- appending region:1429 with 266 probes\n",
      "-- appending region:143 with 186 probes\n",
      "-- appending region:1430 with 240 probes\n",
      "-- appending region:1431 with 10 probes\n",
      "-- appending region:1433 with 28 probes\n",
      "-- appending region:1434 with 295 probes\n",
      "-- appending region:1435 with 255 probes\n",
      "-- appending region:1436 with 298 probes\n",
      "-- appending region:1437 with 360 probes\n",
      "-- appending region:1438 with 174 probes\n",
      "-- appending region:1439 with 198 probes\n",
      "-- appending region:144 with 187 probes\n",
      "-- appending region:1440 with 236 probes\n",
      "-- appending region:1441 with 184 probes\n",
      "-- appending region:1442 with 248 probes\n",
      "-- appending region:1443 with 316 probes\n",
      "-- appending region:1444 with 314 probes\n",
      "-- appending region:1445 with 316 probes\n",
      "-- appending region:1446 with 274 probes\n",
      "-- appending region:1447 with 380 probes\n",
      "-- appending region:1448 with 267 probes\n",
      "-- appending region:1449 with 343 probes\n",
      "-- appending region:145 with 178 probes\n",
      "-- appending region:1450 with 211 probes\n",
      "-- appending region:1451 with 255 probes\n",
      "-- appending region:1452 with 113 probes\n",
      "-- appending region:1453 with 304 probes\n",
      "-- appending region:1454 with 277 probes\n",
      "-- appending region:1455 with 290 probes\n",
      "-- appending region:1456 with 270 probes\n",
      "-- appending region:1457 with 281 probes\n",
      "-- appending region:1458 with 396 probes\n",
      "-- appending region:1459 with 326 probes\n",
      "-- appending region:146 with 294 probes\n",
      "-- appending region:1461 with 143 probes\n",
      "-- appending region:1462 with 238 probes\n",
      "-- appending region:1463 with 268 probes\n",
      "-- appending region:1464 with 281 probes\n",
      "-- appending region:1465 with 231 probes\n",
      "-- appending region:1466 with 286 probes\n",
      "-- appending region:1467 with 193 probes\n",
      "-- appending region:1468 with 180 probes\n",
      "-- appending region:1469 with 202 probes\n",
      "-- appending region:147 with 355 probes\n",
      "-- appending region:1470 with 194 probes\n",
      "-- appending region:1471 with 221 probes\n",
      "-- appending region:1472 with 278 probes\n",
      "-- appending region:1473 with 117 probes\n",
      "-- appending region:1474 with 173 probes\n",
      "-- appending region:1475 with 234 probes\n",
      "-- appending region:1476 with 197 probes\n",
      "-- appending region:1477 with 341 probes\n",
      "-- appending region:1478 with 230 probes\n",
      "-- appending region:1479 with 216 probes\n",
      "-- appending region:148 with 193 probes\n",
      "-- appending region:1480 with 117 probes\n",
      "-- appending region:1481 with 198 probes\n",
      "-- appending region:1482 with 212 probes\n",
      "-- appending region:1485 with 351 probes\n",
      "-- appending region:1486 with 227 probes\n",
      "-- appending region:1487 with 250 probes\n",
      "-- appending region:1488 with 311 probes\n",
      "-- appending region:1489 with 94 probes\n",
      "-- appending region:149 with 309 probes\n",
      "-- appending region:1490 with 128 probes\n",
      "-- appending region:1491 with 239 probes\n",
      "-- appending region:1492 with 238 probes\n",
      "-- appending region:1493 with 129 probes\n",
      "-- appending region:1494 with 189 probes\n",
      "-- appending region:1495 with 199 probes\n",
      "-- appending region:1496 with 72 probes\n",
      "-- appending region:1497 with 230 probes\n",
      "-- appending region:1498 with 146 probes\n",
      "-- appending region:1499 with 160 probes\n",
      "-- appending region:15 with 343 probes\n",
      "-- appending region:150 with 240 probes\n",
      "-- appending region:1500 with 369 probes\n",
      "-- appending region:1501 with 253 probes\n",
      "-- appending region:1502 with 139 probes\n",
      "-- appending region:1503 with 186 probes\n",
      "-- appending region:1504 with 285 probes\n",
      "-- appending region:1505 with 239 probes\n",
      "-- appending region:1506 with 163 probes\n",
      "-- appending region:1507 with 242 probes\n",
      "-- appending region:1508 with 189 probes\n",
      "-- appending region:1509 with 188 probes\n",
      "-- appending region:151 with 3 probes\n",
      "-- appending region:1510 with 176 probes\n",
      "-- appending region:1511 with 218 probes\n",
      "-- appending region:1512 with 248 probes\n",
      "-- appending region:1513 with 245 probes\n",
      "-- appending region:1514 with 262 probes\n",
      "-- appending region:1515 with 357 probes\n",
      "-- appending region:1516 with 103 probes\n",
      "-- appending region:1517 with 232 probes\n",
      "-- appending region:1518 with 217 probes\n",
      "-- appending region:1519 with 239 probes\n",
      "-- appending region:152 with 248 probes\n",
      "-- appending region:1520 with 221 probes\n",
      "-- appending region:1521 with 176 probes\n",
      "-- appending region:1525 with 51 probes\n",
      "-- appending region:1526 with 60 probes\n",
      "-- appending region:1527 with 72 probes\n",
      "-- appending region:1528 with 44 probes\n",
      "-- appending region:1529 with 166 probes\n",
      "-- appending region:153 with 307 probes\n",
      "-- appending region:1530 with 218 probes\n",
      "-- appending region:1533 with 69 probes\n",
      "-- appending region:1534 with 149 probes\n",
      "-- appending region:1535 with 187 probes\n",
      "-- appending region:1536 with 187 probes\n",
      "-- appending region:1537 with 170 probes\n",
      "-- appending region:1538 with 164 probes\n",
      "-- appending region:154 with 283 probes\n",
      "-- appending region:1540 with 1 probes\n",
      "-- appending region:1541 with 251 probes\n",
      "-- appending region:1542 with 204 probes\n",
      "-- appending region:1543 with 30 probes\n",
      "-- appending region:1547 with 2 probes\n",
      "-- appending region:1548 with 106 probes\n",
      "-- appending region:155 with 252 probes\n",
      "-- appending region:156 with 292 probes\n",
      "-- appending region:157 with 255 probes\n",
      "-- appending region:158 with 227 probes\n",
      "-- appending region:159 with 231 probes\n",
      "-- appending region:16 with 245 probes\n",
      "-- appending region:160 with 136 probes\n",
      "-- appending region:161 with 134 probes\n",
      "-- appending region:162 with 338 probes\n",
      "-- appending region:163 with 234 probes\n",
      "-- appending region:164 with 303 probes\n",
      "-- appending region:165 with 244 probes\n",
      "-- appending region:166 with 304 probes\n",
      "-- appending region:167 with 217 probes\n",
      "-- appending region:168 with 301 probes\n",
      "-- appending region:169 with 238 probes\n",
      "-- appending region:17 with 203 probes\n",
      "-- appending region:170 with 303 probes\n",
      "-- appending region:171 with 272 probes\n",
      "-- appending region:172 with 200 probes\n",
      "-- appending region:173 with 173 probes\n",
      "-- appending region:174 with 198 probes\n",
      "-- appending region:175 with 317 probes\n",
      "-- appending region:176 with 152 probes\n",
      "-- appending region:177 with 279 probes\n",
      "-- appending region:178 with 280 probes\n",
      "-- appending region:179 with 230 probes\n",
      "-- appending region:18 with 204 probes\n",
      "-- appending region:180 with 226 probes\n",
      "-- appending region:181 with 248 probes\n",
      "-- appending region:182 with 93 probes\n",
      "-- appending region:183 with 204 probes\n",
      "-- appending region:184 with 173 probes\n",
      "-- appending region:185 with 313 probes\n",
      "-- appending region:186 with 301 probes\n",
      "-- appending region:187 with 239 probes\n",
      "-- appending region:188 with 233 probes\n",
      "-- appending region:189 with 191 probes\n",
      "-- appending region:19 with 391 probes\n",
      "-- appending region:190 with 150 probes\n",
      "-- appending region:191 with 274 probes\n",
      "-- appending region:192 with 270 probes\n",
      "-- appending region:193 with 357 probes\n",
      "-- appending region:194 with 297 probes\n",
      "-- appending region:195 with 199 probes\n",
      "-- appending region:196 with 317 probes\n",
      "-- appending region:197 with 251 probes\n",
      "-- appending region:198 with 379 probes\n",
      "-- appending region:199 with 275 probes\n",
      "-- appending region:2 with 377 probes\n",
      "-- appending region:20 with 206 probes\n",
      "-- appending region:200 with 229 probes\n",
      "-- appending region:201 with 361 probes\n",
      "-- appending region:202 with 331 probes\n",
      "-- appending region:203 with 190 probes\n",
      "-- appending region:204 with 265 probes\n",
      "-- appending region:205 with 241 probes\n",
      "-- appending region:206 with 262 probes\n",
      "-- appending region:207 with 324 probes\n",
      "-- appending region:208 with 174 probes\n",
      "-- appending region:209 with 180 probes\n",
      "-- appending region:21 with 255 probes\n",
      "-- appending region:210 with 365 probes\n",
      "-- appending region:211 with 349 probes\n",
      "-- appending region:212 with 330 probes\n",
      "-- appending region:213 with 200 probes\n",
      "-- appending region:214 with 219 probes\n",
      "-- appending region:215 with 142 probes\n",
      "-- appending region:216 with 408 probes\n",
      "-- appending region:217 with 278 probes\n",
      "-- appending region:218 with 252 probes\n",
      "-- appending region:219 with 258 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:22 with 303 probes\n",
      "-- appending region:220 with 214 probes\n",
      "-- appending region:221 with 256 probes\n",
      "-- appending region:222 with 292 probes\n",
      "-- appending region:223 with 318 probes\n",
      "-- appending region:224 with 322 probes\n",
      "-- appending region:225 with 234 probes\n",
      "-- appending region:226 with 220 probes\n",
      "-- appending region:227 with 242 probes\n",
      "-- appending region:228 with 148 probes\n",
      "-- appending region:229 with 197 probes\n",
      "-- appending region:23 with 333 probes\n",
      "-- appending region:230 with 166 probes\n",
      "-- appending region:232 with 23 probes\n",
      "-- appending region:233 with 196 probes\n",
      "-- appending region:234 with 255 probes\n",
      "-- appending region:235 with 233 probes\n",
      "-- appending region:236 with 182 probes\n",
      "-- appending region:237 with 179 probes\n",
      "-- appending region:238 with 251 probes\n",
      "-- appending region:239 with 231 probes\n",
      "-- appending region:24 with 279 probes\n",
      "-- appending region:240 with 234 probes\n",
      "-- appending region:241 with 272 probes\n",
      "-- appending region:242 with 288 probes\n",
      "-- appending region:243 with 165 probes\n",
      "-- appending region:244 with 333 probes\n",
      "-- appending region:245 with 238 probes\n",
      "-- appending region:246 with 205 probes\n",
      "-- appending region:247 with 234 probes\n",
      "-- appending region:248 with 277 probes\n",
      "-- appending region:249 with 242 probes\n",
      "-- appending region:25 with 238 probes\n",
      "-- appending region:250 with 172 probes\n",
      "-- appending region:251 with 357 probes\n",
      "-- appending region:252 with 287 probes\n",
      "-- appending region:253 with 188 probes\n",
      "-- appending region:254 with 282 probes\n",
      "-- appending region:255 with 225 probes\n",
      "-- appending region:256 with 218 probes\n",
      "-- appending region:257 with 288 probes\n",
      "-- appending region:258 with 314 probes\n",
      "-- appending region:259 with 160 probes\n",
      "-- appending region:26 with 328 probes\n",
      "-- appending region:260 with 162 probes\n",
      "-- appending region:261 with 159 probes\n",
      "-- appending region:262 with 179 probes\n",
      "-- appending region:263 with 324 probes\n",
      "-- appending region:264 with 259 probes\n",
      "-- appending region:265 with 220 probes\n",
      "-- appending region:266 with 251 probes\n",
      "-- appending region:267 with 263 probes\n",
      "-- appending region:268 with 175 probes\n",
      "-- appending region:269 with 351 probes\n",
      "-- appending region:27 with 149 probes\n",
      "-- appending region:270 with 313 probes\n",
      "-- appending region:271 with 220 probes\n",
      "-- appending region:272 with 232 probes\n",
      "-- appending region:273 with 231 probes\n",
      "-- appending region:274 with 330 probes\n",
      "-- appending region:275 with 390 probes\n",
      "-- appending region:276 with 325 probes\n",
      "-- appending region:277 with 313 probes\n",
      "-- appending region:278 with 224 probes\n",
      "-- appending region:279 with 210 probes\n",
      "-- appending region:28 with 262 probes\n",
      "-- appending region:280 with 309 probes\n",
      "-- appending region:281 with 239 probes\n",
      "-- appending region:282 with 262 probes\n",
      "-- appending region:283 with 293 probes\n",
      "-- appending region:284 with 215 probes\n",
      "-- appending region:285 with 234 probes\n",
      "-- appending region:286 with 187 probes\n",
      "-- appending region:287 with 194 probes\n",
      "-- appending region:288 with 249 probes\n",
      "-- appending region:289 with 250 probes\n",
      "-- appending region:29 with 199 probes\n",
      "-- appending region:290 with 253 probes\n",
      "-- appending region:291 with 195 probes\n",
      "-- appending region:292 with 226 probes\n",
      "-- appending region:293 with 157 probes\n",
      "-- appending region:295 with 209 probes\n",
      "-- appending region:296 with 274 probes\n",
      "-- appending region:297 with 194 probes\n",
      "-- appending region:298 with 253 probes\n",
      "-- appending region:299 with 141 probes\n",
      "-- appending region:3 with 242 probes\n",
      "-- appending region:30 with 263 probes\n",
      "-- appending region:300 with 151 probes\n",
      "-- appending region:301 with 25 probes\n",
      "-- appending region:302 with 328 probes\n",
      "-- appending region:303 with 213 probes\n",
      "-- appending region:304 with 232 probes\n",
      "-- appending region:305 with 222 probes\n",
      "-- appending region:306 with 242 probes\n",
      "-- appending region:307 with 231 probes\n",
      "-- appending region:308 with 203 probes\n",
      "-- appending region:309 with 343 probes\n",
      "-- appending region:31 with 215 probes\n",
      "-- appending region:310 with 166 probes\n",
      "-- appending region:311 with 267 probes\n",
      "-- appending region:312 with 241 probes\n",
      "-- appending region:313 with 182 probes\n",
      "-- appending region:314 with 253 probes\n",
      "-- appending region:315 with 228 probes\n",
      "-- appending region:316 with 183 probes\n",
      "-- appending region:317 with 251 probes\n",
      "-- appending region:318 with 178 probes\n",
      "-- appending region:319 with 249 probes\n",
      "-- appending region:32 with 190 probes\n",
      "-- appending region:320 with 155 probes\n",
      "-- appending region:321 with 279 probes\n",
      "-- appending region:322 with 258 probes\n",
      "-- appending region:323 with 248 probes\n",
      "-- appending region:324 with 171 probes\n",
      "-- appending region:325 with 176 probes\n",
      "-- appending region:326 with 191 probes\n",
      "-- appending region:327 with 106 probes\n",
      "-- appending region:328 with 170 probes\n",
      "-- appending region:329 with 295 probes\n",
      "-- appending region:33 with 338 probes\n",
      "-- appending region:330 with 218 probes\n",
      "-- appending region:331 with 154 probes\n",
      "-- appending region:332 with 377 probes\n",
      "-- appending region:333 with 214 probes\n",
      "-- appending region:334 with 234 probes\n",
      "-- appending region:335 with 271 probes\n",
      "-- appending region:336 with 244 probes\n",
      "-- appending region:337 with 181 probes\n",
      "-- appending region:338 with 222 probes\n",
      "-- appending region:339 with 257 probes\n",
      "-- appending region:34 with 230 probes\n",
      "-- appending region:340 with 201 probes\n",
      "-- appending region:341 with 284 probes\n",
      "-- appending region:342 with 157 probes\n",
      "-- appending region:343 with 260 probes\n",
      "-- appending region:344 with 234 probes\n",
      "-- appending region:345 with 225 probes\n",
      "-- appending region:346 with 279 probes\n",
      "-- appending region:347 with 202 probes\n",
      "-- appending region:348 with 223 probes\n",
      "-- appending region:349 with 161 probes\n",
      "-- appending region:35 with 203 probes\n",
      "-- appending region:350 with 415 probes\n",
      "-- appending region:351 with 319 probes\n",
      "-- appending region:352 with 325 probes\n",
      "-- appending region:353 with 158 probes\n",
      "-- appending region:354 with 293 probes\n",
      "-- appending region:355 with 234 probes\n",
      "-- appending region:356 with 257 probes\n",
      "-- appending region:357 with 170 probes\n",
      "-- appending region:358 with 216 probes\n",
      "-- appending region:359 with 179 probes\n",
      "-- appending region:36 with 154 probes\n",
      "-- appending region:360 with 190 probes\n",
      "-- appending region:361 with 175 probes\n",
      "-- appending region:362 with 315 probes\n",
      "-- appending region:363 with 350 probes\n",
      "-- appending region:364 with 253 probes\n",
      "-- appending region:365 with 284 probes\n",
      "-- appending region:366 with 214 probes\n",
      "-- appending region:367 with 102 probes\n",
      "-- appending region:368 with 64 probes\n",
      "-- appending region:37 with 358 probes\n",
      "-- appending region:370 with 158 probes\n",
      "-- appending region:371 with 84 probes\n",
      "-- appending region:372 with 236 probes\n",
      "-- appending region:373 with 270 probes\n",
      "-- appending region:374 with 224 probes\n",
      "-- appending region:375 with 226 probes\n",
      "-- appending region:376 with 222 probes\n",
      "-- appending region:377 with 340 probes\n",
      "-- appending region:379 with 287 probes\n",
      "-- appending region:38 with 211 probes\n",
      "-- appending region:380 with 224 probes\n",
      "-- appending region:381 with 278 probes\n",
      "-- appending region:382 with 318 probes\n",
      "-- appending region:383 with 176 probes\n",
      "-- appending region:384 with 166 probes\n",
      "-- appending region:385 with 272 probes\n",
      "-- appending region:386 with 215 probes\n",
      "-- appending region:387 with 192 probes\n",
      "-- appending region:388 with 277 probes\n",
      "-- appending region:389 with 170 probes\n",
      "-- appending region:39 with 237 probes\n",
      "-- appending region:390 with 164 probes\n",
      "-- appending region:391 with 257 probes\n",
      "-- appending region:392 with 202 probes\n",
      "-- appending region:393 with 229 probes\n",
      "-- appending region:394 with 309 probes\n",
      "-- appending region:395 with 256 probes\n",
      "-- appending region:396 with 182 probes\n",
      "-- appending region:397 with 245 probes\n",
      "-- appending region:398 with 162 probes\n",
      "-- appending region:399 with 196 probes\n",
      "-- appending region:4 with 256 probes\n",
      "-- appending region:40 with 293 probes\n",
      "-- appending region:400 with 222 probes\n",
      "-- appending region:401 with 237 probes\n",
      "-- appending region:402 with 191 probes\n",
      "-- appending region:403 with 377 probes\n",
      "-- appending region:404 with 331 probes\n",
      "-- appending region:405 with 205 probes\n",
      "-- appending region:406 with 261 probes\n",
      "-- appending region:407 with 306 probes\n",
      "-- appending region:408 with 163 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:409 with 221 probes\n",
      "-- appending region:41 with 22 probes\n",
      "-- appending region:410 with 271 probes\n",
      "-- appending region:411 with 282 probes\n",
      "-- appending region:412 with 212 probes\n",
      "-- appending region:413 with 339 probes\n",
      "-- appending region:414 with 188 probes\n",
      "-- appending region:415 with 252 probes\n",
      "-- appending region:416 with 252 probes\n",
      "-- appending region:417 with 315 probes\n",
      "-- appending region:418 with 200 probes\n",
      "-- appending region:419 with 355 probes\n",
      "-- appending region:42 with 201 probes\n",
      "-- appending region:420 with 37 probes\n",
      "-- appending region:421 with 212 probes\n",
      "-- appending region:422 with 298 probes\n",
      "-- appending region:423 with 345 probes\n",
      "-- appending region:424 with 323 probes\n",
      "-- appending region:425 with 221 probes\n",
      "-- appending region:426 with 350 probes\n",
      "-- appending region:427 with 217 probes\n",
      "-- appending region:428 with 243 probes\n",
      "-- appending region:429 with 193 probes\n",
      "-- appending region:43 with 235 probes\n",
      "-- appending region:430 with 236 probes\n",
      "-- appending region:431 with 237 probes\n",
      "-- appending region:432 with 270 probes\n",
      "-- appending region:434 with 5 probes\n",
      "-- appending region:435 with 357 probes\n",
      "-- appending region:436 with 294 probes\n",
      "-- appending region:437 with 188 probes\n",
      "-- appending region:438 with 206 probes\n",
      "-- appending region:439 with 310 probes\n",
      "-- appending region:44 with 317 probes\n",
      "-- appending region:440 with 321 probes\n",
      "-- appending region:441 with 183 probes\n",
      "-- appending region:442 with 276 probes\n",
      "-- appending region:443 with 241 probes\n",
      "-- appending region:444 with 307 probes\n",
      "-- appending region:446 with 108 probes\n",
      "-- appending region:447 with 181 probes\n",
      "-- appending region:448 with 127 probes\n",
      "-- appending region:449 with 228 probes\n",
      "-- appending region:45 with 187 probes\n",
      "-- appending region:450 with 268 probes\n",
      "-- appending region:451 with 258 probes\n",
      "-- appending region:452 with 197 probes\n",
      "-- appending region:453 with 67 probes\n",
      "-- appending region:454 with 145 probes\n",
      "-- appending region:455 with 142 probes\n",
      "-- appending region:456 with 217 probes\n",
      "-- appending region:457 with 207 probes\n",
      "-- appending region:458 with 236 probes\n",
      "-- appending region:459 with 180 probes\n",
      "-- appending region:46 with 309 probes\n",
      "-- appending region:460 with 202 probes\n",
      "-- appending region:461 with 209 probes\n",
      "-- appending region:462 with 170 probes\n",
      "-- appending region:463 with 143 probes\n",
      "-- appending region:464 with 211 probes\n",
      "-- appending region:465 with 352 probes\n",
      "-- appending region:466 with 189 probes\n",
      "-- appending region:467 with 200 probes\n",
      "-- appending region:468 with 222 probes\n",
      "-- appending region:469 with 260 probes\n",
      "-- appending region:47 with 213 probes\n",
      "-- appending region:470 with 131 probes\n",
      "-- appending region:471 with 199 probes\n",
      "-- appending region:472 with 203 probes\n",
      "-- appending region:473 with 203 probes\n",
      "-- appending region:474 with 235 probes\n",
      "-- appending region:475 with 263 probes\n",
      "-- appending region:476 with 236 probes\n",
      "-- appending region:477 with 215 probes\n",
      "-- appending region:478 with 189 probes\n",
      "-- appending region:479 with 287 probes\n",
      "-- appending region:48 with 3 probes\n",
      "-- appending region:480 with 257 probes\n",
      "-- appending region:481 with 269 probes\n",
      "-- appending region:482 with 230 probes\n",
      "-- appending region:483 with 220 probes\n",
      "-- appending region:484 with 265 probes\n",
      "-- appending region:485 with 255 probes\n",
      "-- appending region:486 with 294 probes\n",
      "-- appending region:487 with 342 probes\n",
      "-- appending region:488 with 3 probes\n",
      "-- appending region:489 with 216 probes\n",
      "-- appending region:490 with 384 probes\n",
      "-- appending region:491 with 276 probes\n",
      "-- appending region:492 with 229 probes\n",
      "-- appending region:493 with 239 probes\n",
      "-- appending region:494 with 293 probes\n",
      "-- appending region:495 with 259 probes\n",
      "-- appending region:496 with 167 probes\n",
      "-- appending region:497 with 228 probes\n",
      "-- appending region:498 with 262 probes\n",
      "-- appending region:499 with 162 probes\n",
      "-- appending region:5 with 303 probes\n",
      "-- appending region:500 with 142 probes\n",
      "-- appending region:501 with 304 probes\n",
      "-- appending region:502 with 269 probes\n",
      "-- appending region:503 with 318 probes\n",
      "-- appending region:504 with 220 probes\n",
      "-- appending region:505 with 308 probes\n",
      "-- appending region:506 with 306 probes\n",
      "-- appending region:507 with 202 probes\n",
      "-- appending region:508 with 221 probes\n",
      "-- appending region:509 with 272 probes\n",
      "-- appending region:510 with 336 probes\n",
      "-- appending region:511 with 184 probes\n",
      "-- appending region:512 with 197 probes\n",
      "-- appending region:514 with 2 probes\n",
      "-- appending region:515 with 20 probes\n",
      "-- appending region:516 with 227 probes\n",
      "-- appending region:517 with 262 probes\n",
      "-- appending region:518 with 264 probes\n",
      "-- appending region:519 with 188 probes\n",
      "-- appending region:520 with 212 probes\n",
      "-- appending region:521 with 265 probes\n",
      "-- appending region:522 with 187 probes\n",
      "-- appending region:523 with 152 probes\n",
      "-- appending region:524 with 196 probes\n",
      "-- appending region:525 with 216 probes\n",
      "-- appending region:526 with 223 probes\n",
      "-- appending region:527 with 255 probes\n",
      "-- appending region:528 with 249 probes\n",
      "-- appending region:529 with 269 probes\n",
      "-- appending region:530 with 223 probes\n",
      "-- appending region:531 with 201 probes\n",
      "-- appending region:532 with 232 probes\n",
      "-- appending region:533 with 201 probes\n",
      "-- appending region:534 with 181 probes\n",
      "-- appending region:535 with 116 probes\n",
      "-- appending region:536 with 259 probes\n",
      "-- appending region:537 with 231 probes\n",
      "-- appending region:538 with 214 probes\n",
      "-- appending region:539 with 242 probes\n",
      "-- appending region:540 with 179 probes\n",
      "-- appending region:541 with 248 probes\n",
      "-- appending region:542 with 227 probes\n",
      "-- appending region:543 with 302 probes\n",
      "-- appending region:544 with 276 probes\n",
      "-- appending region:545 with 267 probes\n",
      "-- appending region:546 with 305 probes\n",
      "-- appending region:547 with 44 probes\n",
      "-- appending region:548 with 191 probes\n",
      "-- appending region:549 with 206 probes\n",
      "-- appending region:550 with 378 probes\n",
      "-- appending region:551 with 90 probes\n",
      "-- appending region:552 with 264 probes\n",
      "-- appending region:553 with 49 probes\n",
      "-- appending region:554 with 306 probes\n",
      "-- appending region:555 with 241 probes\n",
      "-- appending region:556 with 234 probes\n",
      "-- appending region:557 with 273 probes\n",
      "-- appending region:558 with 251 probes\n",
      "-- appending region:559 with 221 probes\n",
      "-- appending region:560 with 261 probes\n",
      "-- appending region:561 with 240 probes\n",
      "-- appending region:562 with 249 probes\n",
      "-- appending region:563 with 193 probes\n",
      "-- appending region:564 with 270 probes\n",
      "-- appending region:565 with 218 probes\n",
      "-- appending region:566 with 169 probes\n",
      "-- appending region:567 with 229 probes\n",
      "-- appending region:568 with 222 probes\n",
      "-- appending region:569 with 292 probes\n",
      "-- appending region:57 with 60 probes\n",
      "-- appending region:570 with 104 probes\n",
      "-- appending region:571 with 356 probes\n",
      "-- appending region:572 with 280 probes\n",
      "-- appending region:573 with 206 probes\n",
      "-- appending region:574 with 258 probes\n",
      "-- appending region:575 with 204 probes\n",
      "-- appending region:576 with 291 probes\n",
      "-- appending region:577 with 207 probes\n",
      "-- appending region:578 with 246 probes\n",
      "-- appending region:579 with 151 probes\n",
      "-- appending region:580 with 202 probes\n",
      "-- appending region:581 with 250 probes\n",
      "-- appending region:582 with 199 probes\n",
      "-- appending region:583 with 181 probes\n",
      "-- appending region:584 with 139 probes\n",
      "-- appending region:585 with 241 probes\n",
      "-- appending region:586 with 218 probes\n",
      "-- appending region:587 with 156 probes\n",
      "-- appending region:588 with 177 probes\n",
      "-- appending region:589 with 275 probes\n",
      "-- appending region:590 with 225 probes\n",
      "-- appending region:591 with 283 probes\n",
      "-- appending region:592 with 144 probes\n",
      "-- appending region:593 with 306 probes\n",
      "-- appending region:594 with 155 probes\n",
      "-- appending region:595 with 180 probes\n",
      "-- appending region:596 with 254 probes\n",
      "-- appending region:597 with 277 probes\n",
      "-- appending region:598 with 226 probes\n",
      "-- appending region:599 with 304 probes\n",
      "-- appending region:6 with 300 probes\n",
      "-- appending region:60 with 187 probes\n",
      "-- appending region:600 with 294 probes\n",
      "-- appending region:601 with 299 probes\n",
      "-- appending region:602 with 261 probes\n",
      "-- appending region:603 with 251 probes\n",
      "-- appending region:604 with 6 probes\n",
      "-- appending region:605 with 187 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:606 with 293 probes\n",
      "-- appending region:607 with 281 probes\n",
      "-- appending region:608 with 355 probes\n",
      "-- appending region:609 with 323 probes\n",
      "-- appending region:611 with 302 probes\n",
      "-- appending region:612 with 311 probes\n",
      "-- appending region:613 with 200 probes\n",
      "-- appending region:614 with 277 probes\n",
      "-- appending region:615 with 258 probes\n",
      "-- appending region:616 with 173 probes\n",
      "-- appending region:617 with 254 probes\n",
      "-- appending region:618 with 192 probes\n",
      "-- appending region:619 with 282 probes\n",
      "-- appending region:62 with 330 probes\n",
      "-- appending region:620 with 196 probes\n",
      "-- appending region:621 with 248 probes\n",
      "-- appending region:622 with 247 probes\n",
      "-- appending region:623 with 185 probes\n",
      "-- appending region:624 with 132 probes\n",
      "-- appending region:625 with 168 probes\n",
      "-- appending region:626 with 118 probes\n",
      "-- appending region:627 with 253 probes\n",
      "-- appending region:628 with 273 probes\n",
      "-- appending region:629 with 360 probes\n",
      "-- appending region:63 with 241 probes\n",
      "-- appending region:630 with 37 probes\n",
      "-- appending region:631 with 21 probes\n",
      "-- appending region:632 with 9 probes\n",
      "-- appending region:64 with 199 probes\n",
      "-- appending region:641 with 5 probes\n",
      "-- appending region:643 with 1 probes\n",
      "-- appending region:645 with 257 probes\n",
      "-- appending region:646 with 241 probes\n",
      "-- appending region:647 with 219 probes\n",
      "-- appending region:648 with 215 probes\n",
      "-- appending region:649 with 262 probes\n",
      "-- appending region:65 with 114 probes\n",
      "-- appending region:650 with 241 probes\n",
      "-- appending region:651 with 146 probes\n",
      "-- appending region:652 with 306 probes\n",
      "-- appending region:653 with 373 probes\n",
      "-- appending region:654 with 271 probes\n",
      "-- appending region:655 with 401 probes\n",
      "-- appending region:656 with 292 probes\n",
      "-- appending region:657 with 186 probes\n",
      "-- appending region:658 with 311 probes\n",
      "-- appending region:659 with 54 probes\n",
      "-- appending region:66 with 127 probes\n",
      "-- appending region:660 with 278 probes\n",
      "-- appending region:662 with 184 probes\n",
      "-- appending region:663 with 205 probes\n",
      "-- appending region:664 with 255 probes\n",
      "-- appending region:665 with 266 probes\n",
      "-- appending region:666 with 273 probes\n",
      "-- appending region:667 with 172 probes\n",
      "-- appending region:668 with 328 probes\n",
      "-- appending region:669 with 312 probes\n",
      "-- appending region:67 with 271 probes\n",
      "-- appending region:670 with 311 probes\n",
      "-- appending region:671 with 199 probes\n",
      "-- appending region:672 with 171 probes\n",
      "-- appending region:673 with 120 probes\n",
      "-- appending region:674 with 258 probes\n",
      "-- appending region:675 with 327 probes\n",
      "-- appending region:676 with 330 probes\n",
      "-- appending region:677 with 308 probes\n",
      "-- appending region:678 with 347 probes\n",
      "-- appending region:679 with 320 probes\n",
      "-- appending region:68 with 226 probes\n",
      "-- appending region:680 with 294 probes\n",
      "-- appending region:681 with 308 probes\n",
      "-- appending region:682 with 147 probes\n",
      "-- appending region:683 with 285 probes\n",
      "-- appending region:684 with 253 probes\n",
      "-- appending region:685 with 328 probes\n",
      "-- appending region:686 with 266 probes\n",
      "-- appending region:687 with 78 probes\n",
      "-- appending region:688 with 278 probes\n",
      "-- appending region:689 with 160 probes\n",
      "-- appending region:69 with 128 probes\n",
      "-- appending region:690 with 289 probes\n",
      "-- appending region:691 with 270 probes\n",
      "-- appending region:692 with 189 probes\n",
      "-- appending region:693 with 246 probes\n",
      "-- appending region:694 with 236 probes\n",
      "-- appending region:695 with 246 probes\n",
      "-- appending region:696 with 170 probes\n",
      "-- appending region:697 with 220 probes\n",
      "-- appending region:698 with 227 probes\n",
      "-- appending region:699 with 66 probes\n",
      "-- appending region:7 with 341 probes\n",
      "-- appending region:70 with 120 probes\n",
      "-- appending region:701 with 289 probes\n",
      "-- appending region:702 with 291 probes\n",
      "-- appending region:703 with 181 probes\n",
      "-- appending region:704 with 284 probes\n",
      "-- appending region:705 with 292 probes\n",
      "-- appending region:706 with 246 probes\n",
      "-- appending region:707 with 182 probes\n",
      "-- appending region:708 with 213 probes\n",
      "-- appending region:709 with 289 probes\n",
      "-- appending region:71 with 137 probes\n",
      "-- appending region:710 with 234 probes\n",
      "-- appending region:711 with 290 probes\n",
      "-- appending region:712 with 201 probes\n",
      "-- appending region:713 with 236 probes\n",
      "-- appending region:714 with 196 probes\n",
      "-- appending region:715 with 353 probes\n",
      "-- appending region:716 with 136 probes\n",
      "-- appending region:717 with 282 probes\n",
      "-- appending region:718 with 391 probes\n",
      "-- appending region:719 with 376 probes\n",
      "-- appending region:72 with 267 probes\n",
      "-- appending region:720 with 218 probes\n",
      "-- appending region:721 with 270 probes\n",
      "-- appending region:722 with 309 probes\n",
      "-- appending region:723 with 73 probes\n",
      "-- appending region:724 with 288 probes\n",
      "-- appending region:725 with 271 probes\n",
      "-- appending region:726 with 209 probes\n",
      "-- appending region:727 with 138 probes\n",
      "-- appending region:728 with 365 probes\n",
      "-- appending region:729 with 200 probes\n",
      "-- appending region:73 with 198 probes\n",
      "-- appending region:730 with 379 probes\n",
      "-- appending region:731 with 174 probes\n",
      "-- appending region:732 with 296 probes\n",
      "-- appending region:733 with 274 probes\n",
      "-- appending region:734 with 225 probes\n",
      "-- appending region:735 with 206 probes\n",
      "-- appending region:736 with 362 probes\n",
      "-- appending region:737 with 252 probes\n",
      "-- appending region:738 with 222 probes\n",
      "-- appending region:739 with 229 probes\n",
      "-- appending region:74 with 173 probes\n",
      "-- appending region:740 with 249 probes\n",
      "-- appending region:741 with 266 probes\n",
      "-- appending region:742 with 353 probes\n",
      "-- appending region:743 with 309 probes\n",
      "-- appending region:744 with 308 probes\n",
      "-- appending region:745 with 367 probes\n",
      "-- appending region:747 with 23 probes\n",
      "-- appending region:748 with 254 probes\n",
      "-- appending region:749 with 199 probes\n",
      "-- appending region:75 with 149 probes\n",
      "-- appending region:750 with 287 probes\n",
      "-- appending region:751 with 293 probes\n",
      "-- appending region:752 with 307 probes\n",
      "-- appending region:753 with 145 probes\n",
      "-- appending region:754 with 344 probes\n",
      "-- appending region:755 with 59 probes\n",
      "-- appending region:756 with 319 probes\n",
      "-- appending region:757 with 192 probes\n",
      "-- appending region:758 with 141 probes\n",
      "-- appending region:759 with 306 probes\n",
      "-- appending region:76 with 121 probes\n",
      "-- appending region:760 with 220 probes\n",
      "-- appending region:761 with 222 probes\n",
      "-- appending region:762 with 141 probes\n",
      "-- appending region:763 with 270 probes\n",
      "-- appending region:764 with 261 probes\n",
      "-- appending region:765 with 191 probes\n",
      "-- appending region:766 with 254 probes\n",
      "-- appending region:767 with 199 probes\n",
      "-- appending region:768 with 278 probes\n",
      "-- appending region:769 with 160 probes\n",
      "-- appending region:77 with 218 probes\n",
      "-- appending region:770 with 81 probes\n",
      "-- appending region:773 with 46 probes\n",
      "-- appending region:774 with 202 probes\n",
      "-- appending region:775 with 202 probes\n",
      "-- appending region:776 with 317 probes\n",
      "-- appending region:777 with 373 probes\n",
      "-- appending region:778 with 252 probes\n",
      "-- appending region:779 with 208 probes\n",
      "-- appending region:78 with 142 probes\n",
      "-- appending region:780 with 349 probes\n",
      "-- appending region:782 with 206 probes\n",
      "-- appending region:783 with 286 probes\n",
      "-- appending region:784 with 196 probes\n",
      "-- appending region:785 with 295 probes\n",
      "-- appending region:786 with 234 probes\n",
      "-- appending region:787 with 293 probes\n",
      "-- appending region:788 with 182 probes\n",
      "-- appending region:789 with 209 probes\n",
      "-- appending region:79 with 198 probes\n",
      "-- appending region:790 with 91 probes\n",
      "-- appending region:791 with 164 probes\n",
      "-- appending region:792 with 168 probes\n",
      "-- appending region:793 with 231 probes\n",
      "-- appending region:794 with 298 probes\n",
      "-- appending region:795 with 126 probes\n",
      "-- appending region:796 with 289 probes\n",
      "-- appending region:797 with 249 probes\n",
      "-- appending region:798 with 244 probes\n",
      "-- appending region:799 with 289 probes\n",
      "-- appending region:8 with 323 probes\n",
      "-- appending region:80 with 247 probes\n",
      "-- appending region:800 with 196 probes\n",
      "-- appending region:801 with 320 probes\n",
      "-- appending region:802 with 257 probes\n",
      "-- appending region:803 with 289 probes\n",
      "-- appending region:804 with 220 probes\n",
      "-- appending region:805 with 168 probes\n",
      "-- appending region:806 with 291 probes\n",
      "-- appending region:807 with 233 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:808 with 197 probes\n",
      "-- appending region:809 with 405 probes\n",
      "-- appending region:81 with 223 probes\n",
      "-- appending region:810 with 160 probes\n",
      "-- appending region:811 with 234 probes\n",
      "-- appending region:812 with 358 probes\n",
      "-- appending region:813 with 187 probes\n",
      "-- appending region:814 with 275 probes\n",
      "-- appending region:815 with 332 probes\n",
      "-- appending region:816 with 209 probes\n",
      "-- appending region:817 with 204 probes\n",
      "-- appending region:819 with 268 probes\n",
      "-- appending region:82 with 8 probes\n",
      "-- appending region:820 with 255 probes\n",
      "-- appending region:821 with 142 probes\n",
      "-- appending region:822 with 257 probes\n",
      "-- appending region:823 with 129 probes\n",
      "-- appending region:824 with 288 probes\n",
      "-- appending region:825 with 225 probes\n",
      "-- appending region:826 with 263 probes\n",
      "-- appending region:827 with 283 probes\n",
      "-- appending region:828 with 377 probes\n",
      "-- appending region:829 with 237 probes\n",
      "-- appending region:83 with 259 probes\n",
      "-- appending region:832 with 168 probes\n",
      "-- appending region:833 with 204 probes\n",
      "-- appending region:834 with 310 probes\n",
      "-- appending region:835 with 236 probes\n",
      "-- appending region:836 with 164 probes\n",
      "-- appending region:837 with 302 probes\n",
      "-- appending region:838 with 226 probes\n",
      "-- appending region:839 with 194 probes\n",
      "-- appending region:84 with 191 probes\n",
      "-- appending region:840 with 264 probes\n",
      "-- appending region:841 with 373 probes\n",
      "-- appending region:843 with 148 probes\n",
      "-- appending region:844 with 256 probes\n",
      "-- appending region:845 with 300 probes\n",
      "-- appending region:846 with 261 probes\n",
      "-- appending region:847 with 174 probes\n",
      "-- appending region:848 with 294 probes\n",
      "-- appending region:849 with 123 probes\n",
      "-- appending region:85 with 313 probes\n",
      "-- appending region:850 with 181 probes\n",
      "-- appending region:851 with 241 probes\n",
      "-- appending region:852 with 279 probes\n",
      "-- appending region:853 with 194 probes\n",
      "-- appending region:854 with 215 probes\n",
      "-- appending region:855 with 199 probes\n",
      "-- appending region:856 with 163 probes\n",
      "-- appending region:857 with 300 probes\n",
      "-- appending region:858 with 252 probes\n",
      "-- appending region:859 with 240 probes\n",
      "-- appending region:86 with 237 probes\n",
      "-- appending region:860 with 144 probes\n",
      "-- appending region:861 with 333 probes\n",
      "-- appending region:862 with 233 probes\n",
      "-- appending region:863 with 282 probes\n",
      "-- appending region:864 with 250 probes\n",
      "-- appending region:865 with 223 probes\n",
      "-- appending region:866 with 216 probes\n",
      "-- appending region:867 with 164 probes\n",
      "-- appending region:868 with 289 probes\n",
      "-- appending region:869 with 301 probes\n",
      "-- appending region:87 with 236 probes\n",
      "-- appending region:870 with 324 probes\n",
      "-- appending region:871 with 329 probes\n",
      "-- appending region:872 with 360 probes\n",
      "-- appending region:873 with 265 probes\n",
      "-- appending region:874 with 370 probes\n",
      "-- appending region:875 with 212 probes\n",
      "-- appending region:876 with 262 probes\n",
      "-- appending region:877 with 268 probes\n",
      "-- appending region:878 with 347 probes\n",
      "-- appending region:88 with 204 probes\n",
      "-- appending region:888 with 215 probes\n",
      "-- appending region:889 with 279 probes\n",
      "-- appending region:89 with 335 probes\n",
      "-- appending region:890 with 205 probes\n",
      "-- appending region:891 with 169 probes\n",
      "-- appending region:892 with 314 probes\n",
      "-- appending region:893 with 252 probes\n",
      "-- appending region:894 with 308 probes\n",
      "-- appending region:895 with 294 probes\n",
      "-- appending region:896 with 158 probes\n",
      "-- appending region:897 with 286 probes\n",
      "-- appending region:898 with 182 probes\n",
      "-- appending region:899 with 211 probes\n",
      "-- appending region:9 with 264 probes\n",
      "-- appending region:90 with 118 probes\n",
      "-- appending region:900 with 251 probes\n",
      "-- appending region:901 with 230 probes\n",
      "-- appending region:902 with 271 probes\n",
      "-- appending region:903 with 144 probes\n",
      "-- appending region:904 with 320 probes\n",
      "-- appending region:905 with 311 probes\n",
      "-- appending region:906 with 194 probes\n",
      "-- appending region:907 with 143 probes\n",
      "-- appending region:908 with 216 probes\n",
      "-- appending region:909 with 181 probes\n",
      "-- appending region:91 with 363 probes\n",
      "-- appending region:910 with 155 probes\n",
      "-- appending region:911 with 187 probes\n",
      "-- appending region:912 with 230 probes\n",
      "-- appending region:913 with 169 probes\n",
      "-- appending region:914 with 176 probes\n",
      "-- appending region:915 with 166 probes\n",
      "-- appending region:916 with 250 probes\n",
      "-- appending region:917 with 250 probes\n",
      "-- appending region:918 with 259 probes\n",
      "-- appending region:919 with 157 probes\n",
      "-- appending region:92 with 271 probes\n",
      "-- appending region:920 with 187 probes\n",
      "-- appending region:921 with 173 probes\n",
      "-- appending region:922 with 174 probes\n",
      "-- appending region:923 with 141 probes\n",
      "-- appending region:924 with 236 probes\n",
      "-- appending region:925 with 214 probes\n",
      "-- appending region:926 with 293 probes\n",
      "-- appending region:927 with 314 probes\n",
      "-- appending region:928 with 334 probes\n",
      "-- appending region:929 with 261 probes\n",
      "-- appending region:93 with 212 probes\n",
      "-- appending region:930 with 266 probes\n",
      "-- appending region:931 with 249 probes\n",
      "-- appending region:932 with 297 probes\n",
      "-- appending region:933 with 224 probes\n",
      "-- appending region:934 with 209 probes\n",
      "-- appending region:935 with 426 probes\n",
      "-- appending region:94 with 178 probes\n",
      "-- appending region:949 with 160 probes\n",
      "-- appending region:95 with 214 probes\n",
      "-- appending region:950 with 178 probes\n",
      "-- appending region:951 with 188 probes\n",
      "-- appending region:952 with 190 probes\n",
      "-- appending region:953 with 245 probes\n",
      "-- appending region:954 with 90 probes\n",
      "-- appending region:955 with 174 probes\n",
      "-- appending region:956 with 265 probes\n",
      "-- appending region:957 with 258 probes\n",
      "-- appending region:958 with 328 probes\n",
      "-- appending region:959 with 213 probes\n",
      "-- appending region:96 with 258 probes\n",
      "-- appending region:960 with 279 probes\n",
      "-- appending region:961 with 237 probes\n",
      "-- appending region:962 with 226 probes\n",
      "-- appending region:963 with 195 probes\n",
      "-- appending region:964 with 149 probes\n",
      "-- appending region:965 with 191 probes\n",
      "-- appending region:966 with 161 probes\n",
      "-- appending region:967 with 229 probes\n",
      "-- appending region:968 with 228 probes\n",
      "-- appending region:969 with 156 probes\n",
      "-- appending region:970 with 110 probes\n",
      "-- appending region:971 with 173 probes\n",
      "-- appending region:972 with 392 probes\n",
      "-- appending region:973 with 270 probes\n",
      "-- appending region:974 with 209 probes\n",
      "-- appending region:975 with 315 probes\n",
      "-- appending region:976 with 194 probes\n",
      "-- appending region:977 with 297 probes\n",
      "-- appending region:978 with 130 probes\n",
      "-- appending region:979 with 255 probes\n",
      "-- appending region:98 with 168 probes\n",
      "-- appending region:980 with 173 probes\n",
      "-- appending region:981 with 302 probes\n",
      "-- appending region:982 with 171 probes\n",
      "-- appending region:983 with 236 probes\n",
      "-- appending region:984 with 269 probes\n",
      "-- appending region:985 with 342 probes\n",
      "-- appending region:986 with 233 probes\n",
      "-- appending region:987 with 374 probes\n",
      "-- appending region:988 with 348 probes\n",
      "-- appending region:989 with 154 probes\n",
      "-- appending region:990 with 281 probes\n",
      "-- appending region:991 with 247 probes\n",
      "-- appending region:992 with 237 probes\n",
      "-- appending region:993 with 153 probes\n",
      "-- appending region:994 with 286 probes\n",
      "-- appending region:995 with 337 probes\n",
      "-- appending region:996 with 255 probes\n",
      "-- appending region:997 with 304 probes\n",
      "-- appending region:999 with 341 probes\n",
      "Saving pb_dict into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\pb_dict.pkl\n"
     ]
    }
   ],
   "source": [
    "overwrite_pb_dict = False\n",
    "verbose = True\n",
    "pb_dict_filename = os.path.join(report_folder, 'pb_dict.pkl')\n",
    "\n",
    "cand_prob_nums = []\n",
    "pb_dict = {}\n",
    "\n",
    "if not os.path.exists(pb_dict_filename) or overwrite_pb_dict:\n",
    "    # get genes\n",
    "    print(\"Sort genes\")\n",
    "    gene_names = np.unique([_v['reg_name'].split('gene_')[1] for _s, _v in _pbde.kept_probes.items()])\n",
    "    print(f\"- {len(gene_names)} genes\")\n",
    "    for _gene_name in gene_names:\n",
    "        _sel_pbs = {_s:_v for _s, _v in _pbde.kept_probes.items() if _v['reg_name'].split('gene_')[1]==_gene_name}\n",
    "\n",
    "        if len(_sel_pbs) > 0:\n",
    "            cand_prob_nums.append(len(_sel_pbs))\n",
    "            if verbose:\n",
    "                print(f\"-- appending region:{_gene_name} with {len(_sel_pbs)} probes\")\n",
    "            _splitted_pbde = ld.pb_reports_class()\n",
    "            _splitted_pbde.kept_probes = _sel_pbs\n",
    "            pb_dict[_gene_name] = _splitted_pbde\n",
    "    # sazve\n",
    "    print(f\"Saving pb_dict into file: {pb_dict_filename}\")\n",
    "    pickle.dump(pb_dict, open(pb_dict_filename, 'wb'))\n",
    "else:\n",
    "    print(f\"Loading pb_dict from file: {pb_dict_filename}\")\n",
    "    pb_dict = pickle.dump(open(pb_dict_filename, 'rb'))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "id": "a78ce7a4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7snQW4FdUWxxfdJd1d0t1Il4GiYoA+VETEBJOURkGwMBBUVEQwAJXu7u7u7u5435pz9tx19p3Yc86Zuefes+b73vvknpk9e//3npnfrFmR6N69e/eAN1aAFWAFWAFWgBVgBVgBViBKFEjEABwlM83DZAVYAVaAFWAFWAFWgBXQFGAA5oXACrACrAArwAqwAqwAKxBVCjAAR9V082BZAVaAFWAFWAFWgBVgBRiAeQ2wAqwAK8AKsAKsACvACkSVAgzAUTXdPFhWgBVgBVgBVoAVYAVYAQZgXgOsACvACrACrAArwAqwAlGlAANwVE03D5YVYAVYAVaAFWAFWAFWgAGY1wArwAqwAqwAK8AKsAKsQFQpwAAcVdPNg2UFWAFWgBVgBVgBVoAVYADmNcAKsAKsACvACrACrAArEFUKMABH1XTzYFkBVoAVYAVYAVaAFWAFGIB5DbACrAArwAqwAqwAK8AKRJUCDMBRNd08WFaAFWAFWAFWgBVgBVgBBmBeA6wAK8AKsAKsACvACrACUaUAA3BUTTcPlhVgBVgBVoAVYAVYAVaAAZjXACvACrACrAArwAqwAqxAVCnAABxV082DZQVYAVaAFWAFWAFWgBVgAOY1wAqwAqwAK8AKsAKsACsQVQowAEfVdPNgWQFWgBVgBVgBVoAVYAUYgHkNsAKsACvACrACrAArwApElQIMwFE13TxYVoAVYAVYAVaAFWAFWAEGYF4DrAArwAqwAqwAK8AKsAJRpQADcFRNNw+WFWAFWAFWgBVgBVgBVoABmNcAK8AKsAKsACvACrACrEBUKcAAHFXTzYNlBVgBVoAVYAVYAVaAFWAA5jXACrACrAArwAqwAqwAKxBVCjAAR9V082BZAVaAFWAFWAFWgBVgBRiAeQ2wAqwAK8AKsAKsACvACkSVAgzAUTXdPFhWgBVgBVgBVoAVYAVYAQZgXgOsACvACrACrAArwAqwAlGlAANwVE03D5YVYAVYAVaAFWAFWAFWgAGY1wArwAqwAqwAK8AKsAKsQFQpwAAcVdPNg2UFWAFWgBVgBVgBVoAVYADmNcAKsAKsACvACrACrAArEFUKMABH1XTzYFkBVoAVYAVYAVaAFWAFGIB5DbACrAArwAqwAqwAK8AKRJUCDMBRNd08WFaAFWAFWAFWgBVgBVgBBmBeA6wAK8AKsAKsACvACrACUaUAA3BUTTcPlhVgBVgBVoAVYAVYAVaAAZjXACvACrACrAArwAqwAqxAVCnAABxV082DZQVYAVaAFWAFWAFWgBVgAOY1wAqwAqwAK8AKsAKsACsQVQowAEfVdPNgWQFWgBVgBVgBVoAVYAUYgHkNsAKsACvACrACrAArwApElQIMwFE13TxYVoAVYAVYAVaAFWAFWAEGYF4DrAArwAqwAqwAK8AKsAJRpQADcFRNNw+WFWAFWAFWgBVgBVgBVoABmNcAK8AKsAKsACvACrACrEBUKcAAHFXTzYNlBVgBVoAVYAVYAVaAFWAA5jXACrACrAArwAqwAqwAKxBVCjAAR9V082BZAVaAFWAFWAFWgBVgBRiAeQ2wAqwAK8AKsAKsACvACkSVAgzAUTXdPFhWgBVgBVgBVoAVYAVYAQZgXgOsACvACrACrAArwAqwAlGlAANwVE03D5YVYAVYAVaAFWAFWAFWgAGY1wArwAqwAqwAK8AKsAKsQFQpwAAcVdPNg2UFWAFWgBVgBVgBVoAVYADmNcAKsAKsACvACrACrAArEFUKMABH1XTzYFkBVoAVYAVYAVaAFWAFGIB5DbACrAArwAqwAqwAK8AKRJUCDMBRNd08WFaAFWAFWAFWgBVgBVgBBmAX18D169dh06ZNkDVrVkiaNKmLZ+KmWQFWgBVgBVgBVoAVCFTg9u3bcOrUKShTpgykTJmS5SEKMAC7uBxWrVoFVatWdfEM3DQrwAqwAqwAK8AKsALWCqxcuRKqVKnCMjEAe7MG9u/fDwULFgRceDlz5vTmpHwWVoAVYAVYAVaAFWAFAODYsWOaIW7fvn1QoEAB1oQB2Js1cPjwYcibNy8cOnQI8uTJ481J+SysACvACrACrAArwAoAAHOI+TJgFwgXLxFeeC6Ky02zAqwAK8AKsAKsgKUCzCEMwHFyifDCixPZ+aSsACvACrACrAArwBZgyzXAFmAXLxEGYBfF5aZZAVaAFWAFWAFWgC3AQa4BBuAghVM5jAFYRSXehxVgBVgBVoAVYAXcUIA5xFxVBmA3Vpy/TV54LorLTbMCrAArwAqwAqwAW4CDXAMMwEEKp3IYA7CKSrwPK8AKsAKsACvACrihAHMIW4DdWFe2bfLCs5WId2AFWAFWgBVgBVgBlxRgDmEAdmlpWTfLCy9OZOeTsgKsACvACrACrABngbBcA+wC4eIlwgDsorjcNCvACrACrAArwApYKsAcwhbgOLlEeOHFiex8UlaAFWAFWAFWgBVgCzBbgOPqKmAAjivl+bysACvACrACrAArwBzCFuA4uQp44cWJ7HxSVoAVYAVYAVaAFWALMFuA4+oqYACOK+X5vKwAK8AKsAKsACvAHMIW4Di5CnjhxYnsfFJWgBVgBVgBVoAVYAswW4Dj6ipgAI4r5fm8rAArwAqwAqxAeBW4d+8ebDt2CfJlTg1pUyQNb+MutcYcwhZgl5aWdbO88OJEdj4pK8AKsAKsACsQdgWmbDwGr41dC7kzpoIlHzYIe/tuNMgcwgDsxrqybZMXnq1EvAMrwAqwAqwAKxAvFGj62ULYceKS1tf9Hz8YL/rMHMIAHCcLlRdenMjOJ2UFWAFWgBVgBcKuQJPPFsDOE5cZgMOubNw0yJXgXNSdAdhFcblpVoAVYAVYAVbAQwUYgD0U24NTMQC7KDIDsIvictOsACvACrACrICHCrALhIdie3AqBmAXRWYAdlFcbpoVYAVYAVaAFfBQAQZgD8X24FQMwC6KzADsorjcNCvACrACrAAr4KECDMAeiu3BqRiAXRSZAdhFcblpVoAVYAVYAVbAQwUYgD0U24NTMQC7KDIDsIvictOsACvACrACrICHCjAAeyi2B6diAHZRZAZgF8XlplkBVsBSgVt37kLf/7ZCiZzpoE21/KwWK8AKhKgAZ4EIUcAIO5wB2MUJYQB2UVxumhVgBSwV+GXZfuj1zxZtnz0DW0CSxIlYMVaAFQhBAQbgEMSLwEMZgF2cFAZgF8XlplkBVsBSgd7/boHRS/dr++we0BySJknMirECrEAICjAAhyBeBB7KAOzipDAAuyhuCE0fOHMFsqdPCSmTJQmhFT6UFYhsBfr8twV+WsIAHNmzxL2LTwowAMen2bLvKwOwvUZB78EAHLR0rh24aNcpeO6HlVA4axqY8049187DDbMCca0ABeBdA5pDMrYAx/WU8PnjuQIMwPF8AqXuMwC7OJ8MwC6KG2TTj369BNYfOq8dvf/jB4NshQ9jBSJfAQyA+3HJPq2jO/s3h+RJ2QUi8meNexjJCjAAR/LsOO8bA7BzzZSPYABWlsqzHVsOXwwbDl9gAPZM8fh1oh8W74PUyZPAM1Xzxa+OG/S23+StgOPBbXu/ZuzyE+9nlAcQ1wowAMf1DIT3/AzA4dUzoDUGYBfFDbLph79aDJuOMAAHKV+CPmzNgXPw+LdLtTEufK8+5MucOl6Pt//krTCKAThezyF3PrIUYACOrPkItTcJEoAvX74MJUqUgCNHjsCqVaugcuXKuk5Tp06F7t27w7Zt2yBPnjzQpUsX6NSpUywdP/30Uxg+fDgcP34cypQpA0OGDIF69Zz5jDIAh7o8w3/8Q18tgs1HLrIFOPzSxvsWJ6w9DF3+2KCN4+9Xa0Cl/PfF6zFRAN7WtxmkSs5Bn/F6QhNg5+/evQeJEgEkwv+LBxsDcDyYJAddTJAA/MEHH8DPP/8MJ06cCADgZcuWQd26deH555+Htm3bwpIlS+Cjjz6CESNGQPv27XXZEH67desGAwcOhIoVK8LIkSNh0qRJsHLlSg2GVTcGYFWlvNuvxReLYOsxBmDvFI8/Z5q47jB0Hu8D4L861oDKBeI3AA+YshVGLvK5QGzt2xRSJ08afyaDe5rgFbh84zY8+OUiKJA5DYx+oUq8gGAG4IS1LBMcAG/fvl2z+A4dOhQ6duwYAMDNmzeHs2fPwooVK/RZ7NChA0yePBkQVhMnTgw3btyA7NmzA/598ODB2n537tzRwLds2bIwbtw45RXAAKwslWc7Nv9iEWxjAPZM7/h0ooQMwFv6NIU0KRiA49N6TOh9/W7BHvh42nZtmKt7NIIsaVNE/JAZgCN+ihx1MMEBcJMmTTRQfeihh6B+/fo6ACPYpk+fHj7++GPo3LmzLtKCBQs014bVq1dDpUqVYN68edCgQQNYu3YtVKhQQd+vT58+GlRfuHBB+U2VAdjRWvRk52afL4Ttxy9p5+IsEJ5IHm9OktAAeODUbfD9wr2a/pv7NIW0DMDxZi1GQ0e/nLMLhs3aqQ11ZfeGkC1dyogfNgNwxE+Row4mKAD+66+/4LXXXoNdu3ZpAEsBeOvWrVCqVCmYNm0aNGvWTBfp1KlTkC1bNvj11181t4hvvvlGa+Pq1auQKlUqfb8///wTWrduDYcOHdJ8h1U2BmAVlbzdhwHYW73j09kmrTsCb49fr3U5IbhAUADe1LsJpEuZLD5NR9T29dL1WzB2xUFoUCIbFM2eLsHqEB8BuPGwBbDr5OV4ZUBhDjG/hBIMACOwYuBb79694cUXX4T58+cHADD6+9auXRvQD7h69eq6Irdv34ZkyZLBF198AW+++SYMGDAA+vXrB9evXw9Qbfbs2dC4cWPYsGGDZmE22i5evAj4P7EdO3YMqlat6giaE+zdLkIGFh/f4CNEugTfDQrAf3asAVXiuQ/woKnbYITfAryxdxNIzwAcL9Zwlz/Ww4S1R+IVZAUjLANwMKo5P4YBOAoAGIPWZs2apfn3oi+vGQAvX74cqlWrFguAv/zyS3jjjTc0AO7fvz9cu3YtQDVsG90rNm7caBoIh/CNrhLy5sRq7Hx58xFOFIiPb/BOxsf7Bq/AP+uPwFvjfBbgP16pAVULxu8gOArAGz5qAhlSsQU4+NXh3ZGFuk6Bu/d850vIbloMwN6sKQbgBA7ABw4cgGLFisHEiROhZs2a2mgXL14MDz/8sObTi0FxBw8edOQCgQCcMmWMT5KKCwRbgAEwsveZ75dD6dwZYFAr9YwZ3twKABoOnQ97Tl1JMA+X23fuQuJEiSBx4viRRsireQ7mPAkOgKdtgxELfD7AG3o1gQypGYCDWRdeH8MA7LXi6ueLjwYUBuAEDsDC2ms2TLT4YrAbB8GpX+jB7jls5g74cu5u7fDdA5pD0iSRVX61wafzYe/phAHA127egcafLYBMqZPDP6/V8gyCj56/BjkzpFQOBg12LXl9XEIG4PW9GkPG1Mm9lpTPF4QChbtNhTt+EzBbgIMQ0MVDGIBdFDcOmk4QPsDnz5+H9et9ny7Fhv/GbA/fffcdVKlSRcvni2nQcF/0AxYbpkr7999/Y6VBw79jxgjcMA0a+v1iKjROg2a9SrtP3AS/rTgYsQBc/9P5sC+BAPDfaw7DO3/68tZOfqO2ZnV3exu5cC8MmLoNOtQtBN1alHT7dJ62n9AAGFNMYaop3Nb2bAz3pWEA9nRBBXkyBuAghfPgMAZgD0T28BQJAoCN9JJ9gHEfUQijXbt20KZNG60QRq9evUwLYQwaNEgD51GjRsGECRO4EIbCwvzgr40wfvUhbc9dA5pDsgizAD8wZB4cOHNV6198t67EReWyAh9O0VdBfNdPXs4UgMd3qA7VCmVWWPGRuwsF4DU9GkHmeJBnNXLV9K5n0QLAX83ZBUM5DZrrC4tdIMwljioARhmwFDIGzNFSyJj2jG737t0DUQoZq8mh5ReLYmBaNSdbNC48GsG8s39zSJ40slwg6g6eBwfPJgwAnrLxGLw2dq22JMe+XA1qFs7iZHkGtW9CBuB/NxyFN39fp+kyrkN1qB7PAfiT6dvh2/k+C3B8KTQQ1KJMYAcxAEfuhLIFOHLnJpieJVgADkaMcB8TjQD8xu/r4L8NRzUpIxGAa38yFw6f82X4iO8WzFlbT8DLv6zWxoKlROsVzxbuJRyrPQZg1yUO2wkGT98O3/gBeFX3RpA1XeRX2grb4ONxQwzAkTt5qgA8ffNxmLPtBHR/sGSc+95HI4eoriAGYFWlgtgvGhfeq2PWwLTNxzW1dvRvBimSJglCOfcOqfXxXDhyPmEA8IKdp+B/P67UxPr+uUrQpFQO94TztxwfARgt5aOX7oMBj5WBYhaFBRKaBXjIjO3w9TyfBTi+VNpyfQHHgxMU6TYVbrsYBLf31GX48O9N8Gy1fPBohdxxpkh8dIFQBWBxn3y8Yh4Y2rpcnGmMJ45GDlEVnAFYVakg9ovGhdf+59Uwe9sJTa3t/ZpBymSRBcA1B82Boxd8RU7iuwV42Z4z8MzI5dpYhj9bAR4qmyuIVerskPgIwKLPuTKkhKVdG5oOGL9c4BcM3MLtArFw5yl4988N8EGzEvB4JbVKks5mJvbeAQDcrSFkSx/5pWZDHXNCON5tAG7xxSLYesxXsCku74HRAMDFsqeFmZ0fiNNlGY0coio4A7CqUkHsF40LDy2SaJmMVACuMWgOHEsgALxi7xl46nsfAA9rXQ5aVXQfrOIzANs98CkA//5ydahROHxBcHGh26czdsDweb6UhCu6NYTsDMBB3MW9P8RtAL6/13S4evMOA3AQU+vUAswAHITIHh7CAOyi2NEIwG1GLYclu89ELABXGzgbTly8Eec3/3AsO2oB/rhVGXi6ar5wNGvZRlyAXKiDUu3z5I1H4fWxPgtwuIMKVfsQ6ljp8RSAl3dtCDkysAU4nPq61ZbbAFyuz0y4cO1WnN8Do8ECXDRbWpjVhS3Abl0robbLAByqghbHRyMAt/5uGazcf1ZTZVvfZpAqeWS5QFQZMBtOXUoYALx0z2l4duQKTeu+LUvB8zUKuLiafU3HBciFOijVPic0AB46cwd85S9Ks6xrA8iZIVWoUvLxHijgNgBX7j8bTl+O+3tgAADHExcdpxZgBmAPLpgQTsEAHIJ4dodGIwA/9s0SWHfwvCbN1r5NIXXypHYyefp7pNz86aDx0/v+01fgtfpFHFVzW7L7NLQZ5QPgHg+WhPZ1CrmupSpMut4RBydQ7XNCBuClHzaAXBkZgB0smzjb1W0AjpQ4CAZgb5ZYNHKIqrIMwKpKBbFfpC+8ExevQ9//tsLD5XJBs9LhySDw8FeLYdORC5paW/o0hTQpIguAK/WbBWeu3NT6F5cBIGI53bh9B4r3mK7984uny0PL8upR2Yt3nYa2P/gA+P1mxaFTvSJBrFJnh6jCpLNW3d1btc8BeZXbV4OaRcKXV1m1D+FUgpYlX/JhA8idAAAYy393m7gJKhfIBG2q5Q+nXBHTltsAXG/IPNgfAcWAGIC9WXKRziHeqGB8FgZgF9WP9IXXdtQKWLz7dFhhsNnnC2H78Utam5v7NIW0EQbAFfvNgrMRBMCXrt+CMr1nanq9Wq+wliVAdcPMAs/706AxAJurpgqfUzcdg06/+QuLJDAAXvxBfciTKbXq0orY/ahbRyS8wLohlNsArPoZ342x0TajAYCLZEsLs9kH2O2lFHT7DMBBS2d/oJcAfPnGbUieJLGjymtuBEM0HDof9py6ErEAXL7vTDh/Ne4DQMTqCQWA5+84Ce1+WsUWYJtLMRgA/q19NagV3y3As3bCl3N2aeoser8+5L0v/gMwVj5ESz1uDMD2zyCjPWgatH2DWkCiRImCayjEoxiAQxRQ8XAvOUSxSxGzGwOwi1Ph1cI7d+Um1PpkrpbmCN82kyRWu6G5AcC01PCm3k0gXcpkLirsvGk3xuy8FzFHhALA83achBdcBmD85Lz39GW4P2d67UGpCpOhaBLuY1X7TC3ACQ2AF75XH/Jljv8A/PrYtTCZATikS6Tl10tgwyFfnEZcFiuKBgAunDUNzHmnXkjzFerBXnFIqP2Mi+MZgF1U3auF9/3CPTBw6nbHlp6yvWfAxeu3w2pNoZXWNvZuAukjDIDdGHMoS4gCcKd6heF9By4Qc7efgBdH+0ohu+UCIVxahjxRFp6snDfeAzAGZs7bfgpqF8kCGVIHvpwlNAD+bNZO+MJvAV7wXj3InzlNKEs1Io6NBgAu2n0q3LpzL6z3ZTp5T363FFbtP6f9SRgprt+6A3O3n4TqhTLDfWmSezLX8RGAGw1bALtPXradG/HSzQDsyVIK+iQMwEFLZ3+gVwA8cuFeGDB1m9YhJ5aeMr1nwCUXAXjDR00gQ6rIsgC7MWb7lWC+x8Xrt6Cs3wfYKQBjrfmXfvYB8HtNi2tZJMK9iRt51nQpYFX3RvEegDHgE7NulM2TAf59vXaAXNM2HYNX/T7AY16qBrWLxu8guM9n74TPZ/tcIOa/Ww8KZIn/AIyV+nD+cAunC4Rwq3iwbM5wX0KO23MbgGmu9jU9GkHmtCmg64RN8PvKg1Agc2qY/159x30O5gAG4GBUc36MVxzivGdxfwQDsItz4NXCG7VoL/Sf4gNgJ5YeN2CQptjZ0KtJLCubi3IrNV36oxmA/tLBPEDv3bunWdQyp0kOz4Up5y4F4NfqF4b3mqoHwc3aegJe/sUHwO82KQavNyiqpIGTnRIaANOxyy460zcfg45jfEFw8QWA0XI3ZMYOqJw/EzQvEwhvX8zeBZ/N3qmNZ9679aBgGAD42/l7IFmSRJ6k3DNap24A8J5Tl6Hh0AXa6ea+8wAUyprWySUS9n3dBuB2P62E+Tt81TpFfmhVN6FwDtYOgL+etxtSJ08CL9QqGM7ThtSWUwtwoaxpYC67QISkuZsHMwC7qK5XAPzD4n3Qb/JWxw86NwC4+sA5cPzida0vkQjAoZQBnbf9JLww2hd0tjJMidtDsQDP3HIcOvy6RuvPO42LwRsN3QPgbOlSwMoEYAGml3vb6vmg/6Nl9D9RAP71papQp2jWsN0d3AIMCrmyRZT+Fg6wQ79R9B/FbVbnulA0e7qw6aPakBsAPHvrCWjvf5H86YUqUL94NtXuuLKf2wCML8348oyb+GLo1vq0EogCsFyqe/neM/C0v8y7E6OOKxNCGmUAdlthb9tnAHZRb68A+MfF+6CvH4DnvPMAFFa0YLgBwFUHzIaT/kpr63s1hoypzf3Jzly+AW1/WAm1i2SG7g/e7+JMxDRdsud0uHbrjvYHp59QqatJuG7KoQDw9M3HoeMYHwB3blQM3mrkDIBv3bkLyZIkttRdPBgTIgBjYN/Ut+rEawBu//NqmL3NBzPyesYMEMNm+SzATu4LZguCrrdwW8hVL/43f18H/4bZBYJ+SYkGAH51zBqYtvm4JjkGTWOqrkgD4L/WHIZ3/9yg9XFip5pQIV8m1SXi6n6OAThLGpj7bkwQ3O07d7UgdS8zb3jFIa4K71LjDMAuCYvNerXwRi/ZB73/81mAZ3epC0WyqVlmynw0Ay4F6Q5gJhstNbyuZ2PIZBFQ0XXCRvh95aGgYDTYaSveYxrcuH03qHMOn7sLPp3pA4rlXRtCjgwpg+2GflwoLhDUZ/XtRkXh7UbFlPuz7/QVeGT4Ymh8f3YY1rq86XEJGYBL5UoPU96MDgAWoKO8QAx2nLHlOLzi/+Lwy4tVoW6x8FnIVfvlNgCPfqEK1ItjC7DbMNrptzUwdZMAYN/zIlznPHr+GuTMkFIJ8KgFWL6f/rn6ELz318ZYAOykfdU15WS/UAAYA56bfLZQq8j4V8caSho56ZvZvl5xSDj66nUbDMAuKu7Vwvtl2X7o9c8WbSQzO9eFYoqfJt0AYFpqeG3PxpYRxR1/XQPTt/huxE6tscFOW7Ee0+BmkABMk/BjQBgGhoW6Xbh2CzA1G25OfYBp1oI3GxaFLo3VAVi1CEp0AXCMRT3cgBcuwJDXm5UFmAKGkxdjszVNLaVxBYpvjVsH/6wPbxBcJIxLaI5FerBYj9jcuC++9ttamLLJl0tZuLKEY32KL2Qd6haCbi1K2t4a6foUvsjiIArAk16rBeXzZgQR69K+dkHo8ZA3XwzlQTgFYPS7R/973Gisjpd5ub3iENsJj8AdGIBdnBSvFt6vy/ZDTz8AT32zDtyfK73SqEIJCDM7AS01LCKMzfaNCwAOxb9u0NRtMGLhXm04bgDw6/WLwLtNiyvNHe40eeNReH3sOm3/NxsUgS5N1I9t9c0SWHvQlwvU6iEbTQDspoUzHIBhtDBUATgcPrs068hP7apA/RLe+8q6AcDUlz6uwF7M7W8rDkD3iZs9A2BhMAnH+nTahhUA/7H6ELzvtwALAHbavvKN1MGOoQAwdaHzMiuLVxziQMaI2ZUB2MWp8GrhjVl+AHpM8t00/3u9NpTJk0FpVKV6TYcrN4PzhzU7QYW+M+Gcv9JaJAJwKGVGe/+7BUYv3R8xAIzpoDAoCLc3GhSBdxwAMLo/bDx8IaoBuHTu9DD5jRgXCArAP79YFR4I4yd+tx7eVgBMXXacfBkyu7Zp3ukf21WGBiWyK91nwrnT2+PWwSS/BThcVcwiCYDpvdzu5TRYXakFeMbbdaF4jvC4QDhd4xSAl37YQHMNEBsF4H9eqwXl8mZ07KaBWXt+Xrpfc1VrVjo86e1CAWAarB4On3zV+feKQ1T7E0n7MQC7OBteLTxqNZjQqSZUVAwYcAOAaanh1T0aQZa05m4CNBjDjU99RlNbqOsUuOvLMe/Y7YL6LK/s3hCypQvdB/jC1VtQrq/PBcKpBfif9UfgrXHrtWOduk80/2IRbDt20VaHcFuA1x48B2OWHdAyVoQjLZfK5UsfzHT/hADANKJfvoYoAE9/uw6UyKH2ZchMU1p58If/VYaGJeMWgPcObAGJFavbL4DBAAAgAElEQVReWq2TaAZgsS6cwquRnk7bsATgVYfg/b99PsDBAjDN2hOuqqShADDC+Ef/+lwVw/FFRuXeh/t4xSGq/Ymk/RiAXZwNrxYeJjDHROa4/dmxBlQpcJ/SqEJJCWZ2Alppzc5NIC4AuGDXKXAvSADu8sd6mLD2iDZ0KwD+ZPp22Hn8EnzTtiKkSJrEci7OX70J5fv6fP6cAvCkdUfg7fE+AHZaRKPxsAWwy0FFo3BlgRAPydwZU8GSDxsordNQdzID4DK5M8B/b8QUw4iPFmArAMY8qpgjGLdpb9WBkjlDA+D5O05CO3/p7ZHPV9YCKL3eOo9fDxPX+a7BPQNbKJd9VwXgcFv+nerz6/ID0NP/NQ+PdcMwQC3AYl04hddwADB9QcN7Ad4TxPYHAeB/X68FZfM4twBbBdk5nRexv1MApoVF6NyG43pUHYNXHKLan0jajwHYxdnwauGNW3kQPvQD8O8vV4cahTMrjcoNAKap1eyspHEBwKHc6F8buxZExSizPMBXb96G+3vN0PTv8WBJ24IB567chAr+oJcqBTLBwbNXtZRmT1fNZzuHE9cdhs7jfamCOj5QGD5srl5Eo8Gn82Hv6SvasV76AIeiv60gJjuoArCblkC3xq0KwE5iA8x0XrjzFDz/40rt5xHPVYKmpXIEOyVBH0cBePeA5pDUJo2fyonovEcbAIt1YbY+8WUeXa3Gtq8O+TKntpTT6Rr3EoDlPMMq68Jon1AAeOyKg9Btos9QNfmN2lA6t5qrYrB9Fcd5xSGh9jMujmcAdlF1rxbe+FUH4YO/fReWk/ycrgAwSa1mVywivgEwhQ2zsdG0Zq/WKwwfNLOGUjnqWyxHFcvP32sOwzv+XJmvPFAIuja3j7wW7dcZPBcOnb2mDMDhKoXs9CEZjstTBYCPnL8Gq/ad1S3q4Q6GcmvcHX5ZDTP9RQ3kNUMtwFPerA2lcoX2wF286zS0/WGFNiXfta0YNr9KJ3NMAXjXgOa2eaxV2nbT8q9yfroPDWi2ezl12rbYn77I47rAfNgFu07Vm6PrSKzbOkWzwK8vVXMNgBd/UB/yZIoB7HBbgOMKgPNnTg0L/KWl6XNaWLWDnUMnx3nFIU76FCn7MgC7OBNeLTwaMODkwR1KUQgz2ahfcUIDYLR+oRUMN7Ox0bRmbgMwTRavmnpIzFuNQXPg2AVfxT4VC3BCBOCyeTLAv6/Xhp0nLmn5OekW7oIIcQ3A4bA4Ld19Gp4d5QPgb9pUhBZS6WUXb6V60xSAd/ZvDsmTWhdyUelTNAMwrgtMA4cl3sVmBMAV8mWEiZ1quQbAclowI1h0eg3RQjCRAMBxVdzDKw5RudYibR8GYBdnxKuFRwHYSXCKGwBM27S76dCE7CoWz3BMldObKD3n098vg+V7z2p/MhsbBWAVv1yshlep/+xYQ1PRg877y3UKOqqmR/M1RzsAd5+4CX5bcTAQgMOc5iuUdWe17q0swN/M3w2Dp/t8gMMCwHtOw7MjfQA8/NkK8FDZXOG4JB21QQF4R/9mtj72Ko1HEgDTnO52L6cqYzPah1qAcV089NXigN2MALhivowwIY4AWGQ2cnoNUQC2M8Zgxgj0Lc+fOQ1Uym9edc6pCwS1AFOXNSyEUVkxVifYeRbHecUhofYzLo5nAHZRda8WHn1b/q5tJWhWWs03r0TPaXD9lnpVNIxiTZokEbSplt9UNdqmXbW0+AbArb9bBiv3BwLw3bv3AD81F8iSBh4ulwucukCcvnwDEEblTQWA6by/VLsg9HSQHJ5m61AB4HAHwbn1cDdamGYuEOXyZIB/Xq8NNL2dOD7ceW6dPrxVb0tWAPzt/D2APpy4OUmPaHbu5XvPwNPfL9d+/uqZCtp693rrMn49TPAHwW3v1wxSJrMOMlXpHy3xHO4CKCrnp/vQTAFuXSMUgHFdPDw8EIDxyxWmAKxeKLOeegyh8O9Xa1oOx+kapz7AVhZgLwCYVtW0uh+GAsA0a8/4DtWhWiG1WB2na0je3ysOCbWfcXE8A7CLqnu18Khz/dfPVoQHy6rlPKSw2qxUDujcuJiWE9Jo23T4gn6jtMphSCutydV95HbjGwA/+d1SWLX/nDYMYQGmNzV8IGOZZVHZTcUF4tSlG4Dlo4MBYBr8+GKtgtDrYfXqSKpFUMKdBs3pQzIcl6cZAIt5xApNIxftCzhVQgDg7xbsgY+n+QA4HD6HK/edhdYjlmntfflMBXiEATgcyzOgDa8BGNfFI8OXGI4DQVBcOxig+2fH+AXAX8zeBZ/N9pWut7MA069A4QTgfPelhoXv19f6QAsXjX25GtQsnCXs68eoQa84xJPBhPkkDMBhFpQ259XCo+lVvni6PLQsn1tpVMV7TNOATWxpUySFzX2aGh6L2Q/QcoDb2PbVoGYR44uXVlqTk5vHdwB+4tulsPqAD4CFdZtWh0Pt7ty9pwOwigvEyUvXoeqAOUEBMH3xaVezAPR+pJTSvONOZtZ/tIR8NXc3DHisNFTIl0l/ACZECzDqUKtIZi3FElpL6ZbQAFjkUlVeIAY7rtp/Fp78zgfATu4zoZxTPpamIozPFuDbd+4CBtVmTpsCPn2ynD7M0Uv2Qe//tur/VvkS5FRfagHGddHya3sArlrwPvjjlRqWp3L6ckstwAvfqx+QZYK+3Av3Haft05SXdhmJvADg6ZuPQccxvmeok2B1p/Mr7+8Vh4Taz7g4ngHYRdW9WnjUb2xY63LQqmIepVHJAIwHmd1w6Sei39pXg1omAExvUk4AOFxVnewG7vQmStt7/NulsMYCgLf0aQq3PQRgWgDFKQCbVcQT+mB9gb2DYixACRWA06VMCi/ULABfzt0dsHTCXekslHVntaatXCBGLNgDg/wWYFFOlrZ15cZtLYuCaiDZmgNn4fFvfQD82VPl4LEKavcZu2vSye8UgLf1bQapkofXBeLXl6pCnaJZnXQpqH3/3XAU3vRXcZz7zgNQKGtarR2vARjXxaMKAFyt4H0w3kUAlr9Q0Nz2wQIwvebiCoDz3pcKFr3vy3eOwYb40qPN8wtVoF5xb0qJe8UhQV0IcXwQA7CLE+DVwvtpyT7o47caDH6iLLSunFdpVNRdQRxgBsAqb697T12GBkMX6OeWk5vLnaIuEOGq6mQ38FBApNU3S2DtwfPaKYQFeODUbfD9wr3a34IC4IvXoerA4CzA1PL/fI380Ldlabvh67+b6SD/3SsXCHQFuXbzjm2uUeUBkh2tXCDSJE8CrzxQGIbN8n0qFZuTYNIbt+/ArhOXoVSu9JAoUSLDLoay7qzGbAXA3y/cAwOn+lwgJnaqqVn0xYb5p2t+PBcwu8e8d+spFZTAKn6tvlmqNTH0yXLweCV7AMb0cvgylTNDTJGDYOZQHPPOHxvg77WHtX9u7dsUUidPGkpz2rHUB9grAKYlj2mVPnovx765bQHGdfGYf05lIakLRPVC98G4Du5ZgPHc1CoalwBsZYxpOHQ+7Dmlnj89T6ZUsPgDHwDHVSlxrzgk5AsxDhpgAHZRdK8WHvow9p+yTRvJwMfKwLPV7Iso4L7BArDZQ4ICLbYv53aUpab7h6uqk910hgIij32zBNb5AVj4N7f/eTXM3nZCO20wAHzi4nWoFiQAU8v/c9XzQ79H4ycAI0AW7zFd01AOhrGbT5XfrQA4ZbLE8GbDonq2BNGeEwBuO2oFLN59WitEggVJjLZQ1p3VGF/5dTXM2OJbfzIsUQDGACYa3T5y4V4YMNV3z1jwXj0t+t1uW3/ovG4txM/2T9gAMM1xveGjJpAhVTK7U9j+TgEYr7c0KcIBwDGfpo3ubRjoOnf7SS0+Iu991sUgbAfg34EC8Iy36+qxF14D8IRONfWXGrnvFIBrFs4MY1+ubjk8p2ucukBgw/elSQ5rezbWzhFuALarSooFKtClDDe3ADiuKil6xSGqaz+S9mMAdnE2vFp49GHWr2UpeK5GAaVROQPg49BxzBqtXREpjRahDYfOQ9vq+bVPqW/8vk6rGiQ2O5ihAByuqk52A3d6k6bt4adChADcBADT9oIB4OMXrkP1QcFZgGnATJtq+WDAY2Xshq//Hi4L8K07dwEf5uXyZoSKxMJo1hGj8+4/fQXqfTpfO0Slep7yIP07WgFw8iSJ4b2mxXUYFG07AWCVNaWyj9Nx4f5WAEzvC3+/WgMq5Y8pkU5/m/9uPS2Lid2G17rwF5W/NOF1nzRxImhOcgNTy2q4fB7f/XMDYD5V3NDnHuMWQt3svm7RbCtOLLLo34+l0zFYMBcp84v9NQPgHxfvg76TvfMBVgXg2kWywJj27hXCQE0oANP4BlHExek1RPd3AsBWXyNDsQAv2HkK/uevpOhlIRmvOCTU6zAujmcAdlF1rxYejfbu80gp+F9NRQDuPg1u3okJgkMpzG7wRrkyxQ1GQAvN0YltOQHgcFV1sptOpzdR2h4+/BECzAAYP8neun0PyvWdqe2jEgR37MI1qDFobqxuqzxoqbUIrf5o/VfdwgXA1Mqo0mej8+47fQXq+wEYU7lhSrdwblYAjNDW/cGSugtRQgVgOe9oMAC88fB5PWPAJ4+Xgaeq+L400UIiNJiJlhgOl2sBBeBNvZtAupShW5XtAJj6/quscbGGHhm+GDYevgA50qeE5d0aBizpuATg18euhckbj2n9wS8DOD6jjVqA3a4EF24Axry+tLpdJAAwtQA7ydYU6r3QKw4JtZ9xcTwDsIuqe7XwaML73g/fD+1qqQFEMQcATB9mwoFfgIW4OdKHkwoAv/bbWpiyyXcjDldVJ7vpDAmAhy+GDYcvaKcQAX60PQTgm7fvQvm+s7R9XqtfGN5ral0K2SkAYyW6UYv3QfcWJWHJ7tO6teiZqnlhUKuydsPXfw8XAD87cjks3XNGa1cFDuwAuNdD98OLHgIwuuyi73TPSZsDtBv1fGVodH92JT1V1pTKPkonk3aysgBT16g/O9aAKiTxfjAATC3Ag1qVgWeq+gDYLEB29tYT0N4f9PPzi1W13LKhbvQes7F3E0jvAQDT9Icqa1yM0WrOzQD4h8X7oJ+LFuBdJy5BY1L1EL8MiMBGeW7wWhTWaC8AOHOa5LDG7wIRqgUYs2wU6T5NH5ITALZyx3NqAc6dMRVgLAxu83achBd+WqX9t5dpBL3ikFCv7bg4ngHYRdW9WnjUl0oVIDBdV+FuMfXfhQxmN3gawSrKxMoA/OHfG2HcqkO6onJqG1lqCsDhqupkN52hgMjDXy2GTUesAfjGrbtQoZ86AB89f00LRpI3s3kQ/cfPhWhhFr7fT1fJCx8/HjkAjBYYXGNJkwSWqjXSnwZPqq5fu3mmv1tZgHG/j1uVgQ8nbApocuTzlaFxGABY6EAfxk4gym6cqgCMKawwlZXYKByrukBQH2Aaa2AGwHO2nYCXflaPescvWX+vOQyovZlLxnt/boA//S4Q4fIrtrMA0wI4TubO6l5DA1hndq4LxbL78q+7DcDVBs6GExdv6OsAvww84U9tZ7XW8OUFX2KsNqf3ViwgNGSGr1IhbuEE4Ou37kCJnr64AtzsskBQH2AvAPjzp8rDoxXU0pXa3QPsfveKQ+z6EYm/MwC7OCteLTya8NvqEzI+jI9fvA4YkT1h7WHo8seGWKM3u8FTa47IkSoDsFxS1i64hgJwuHJ62k2n05s0bc8pAKu4QGCUfK0gABj7hVZgEcjUunIeGPxETD7RYHWQ9RH/NkuDZmQBxnWGn1XxQYsPdxqoZAfAHz18P7yg+AXDbozidzsAHvJEWXjvr40BzYUDgFGHp0Ysh0PnrsKxC9f19p1AlN0YKQDLwTsUcmnlKcy48eeaQ3rgXzAA3P/R0prvP24yACPMYcDb4t2n4MXRPgAWL81W4xHzZFV1LACAezWBDKlDd4Gg/TfyVXYFgJfth57/bNHkmNW5LhT1AzCdM/wtnGsF25OvBfwyIHI7W81NveJZYfQL3gEwTfE49c06cH+u9AF9t9Pl6s3bcH+vGUEBsFU8SkgW4O0n4YXRPguwShCp3bWv+rtXHKLan0jajwHYxdnwauF9NmsnfDFnlzYSqyCiAVO2ahWvMFIdrS1Gm9mNhVpzzAD4o382w8/LDujN2j1Y4xsAP/TVIth85KI2PpHijT5QMAgOC4tU9FuAVQD48LmrUPuTebGmws4CjAd0a1FCT3P1ZKU8MIQk1Ldb1m66QNCgNtkn3ei8e05dhob+9HlxAcBojXl7/PqwA7BT9xa7OTP6veOva2D6luPaTzIAU2viuA7VtdK2W45egAe/DCx9a3edivOuO3hOT5lFg20pQOL9B79KFM+eDj5sUUL/5KuSV1msjQKZU8P893zVs+Tt/b82wB+rfUFw63s1hoypkwcjW8AxdjnOsfodVsHDzQ68aMOWFuAIAWD8MiCq+1kJ2aBENvixXRVLrel47b7+YUNWFmAKwNPeqgMlczoDYFqSHs9lVwmOWoDdAmCaBm3w42WhdRW1dKWhLnCvOCTUfsbF8QzALqru1cIbOnOHVr0LN7QKvly3kOGo7CxhVjd4oxyGsgW4739b4cclMSVl7R6stCJRuJLa201nKBbgFl8sgq3HrAEYP71V6u8rbaziAxwKAGPaLVHqFlNS0YpSwepgZgHOnj4FrOjWKJYVxsgCTIPaZJcGI/13n7wMjYb58kc78WG3G6P43W7dY1Wzt8aFH4CdWvdVx0P3owAsR6/TjAK/v1wdahTOrCXiR3cmumEe4IIKWSAoAPdtWQqe92eboQCJhUUuXb+tNY+xAu38Po8qWTWcAvC6no0hUxr3AfipEctgRQIFYPwy8NT3y22XXsMS2eAHBwCMDdoFKVoBMPWRDgaAz1+9qcdiYF9E6XqzgVIAtgrIDsUCTI1I1IfeVvwQd/CKQ0LsZpwczgDsouxeLbwhM7bD1/N8Fl20Cnaoa5+L1GzYZhaOeeTzjQgQEg+susWyaqnRaFlgbN/uwUoBOFxJ7e2mMxQAbv7FItjmB2CR45i2h2mZEIArKwIwVuLCl4bxq2P8pkX/VSzAHzQrAZ9M9xU6aFUxNwxrXd5u+PrvTi3AXgGwkywmqoO1A2AjC/D3z1WCJqVyKJ3CTEuvAVj2XaQAPPblalCzcBZo//MqmL3tZFAATAth0BcVMwBGn1GR9kklqFAFgD/4a6N+vWDOWPSFD3WzswA/8/1yWLZXPdDT6MVLvp5/NbEA0+BEbMfqixy+aGLGFLPiK0a6yNcCfhl4WgGAG5XMBqP+p24BxnPbld+WAThL2uSwuocvD3CoAHzm8g3dEIHtYZacrs1LmGYN8RqAnaQrDXV9e8UhofYzLo5nAHZRda8WHloBhUuDajJ+xwBMIlhlABYRwoOnb4dv5se4VtASn0bnUwFg/Iz86Yyd8Hil3NoDXHVDPc5fvQUfNCse8ICgDwCn5Zebfb4Qth+/pHXBDICxmlmVAWoW4F7/bIZfiMsIHZsKAGP+WhFE0qpCbhj2lPcAbAQHVmnNjC3Al6DRsIXa8OMCgLGsb+fxgf7wI56rBE1DBGCnAY6qa5vuRy3AMgDTNHlj21eDmkWywIujV2lFHehm96Iq9qUATC37ZgCMqc+e+2GldrjKC4VTAF7ToxFkTpsiGNkCjpm66Rh0+m2t9jehE93BaaYTFQCmRWyoD7AKAGPu7aL+DAfDn60AD5XNpayBDMD4ZeCZkfYWYAwIRb94q01uOxQApkGCwfgAnzSosGnlJkYB2CojkVMLcK4MKWFpV18KPBpHwwCsvGRd3ZEB2EV5vQJg4duLQ0Gr4Kv1wm8BNqpiI7tADJu5A770u2JgX+a88wAU9te4twNgs6pOtPywqv8ddSuQg2/cBGD85Hf91l1lAMbgN7QSGm1OAfjR8rng86crKK/mcFmAjQCY+gDLQZlG56WpmeindeXB2OxoZwEe1rpcrIDQ+AjAsu8iBeDf2leDWkWyQLufVsL8HaeCAuA1B87pOWNprAEFYNowBpS1/WGF9qfv2laCZqWtLeoqAEwzzazu0QiyJDAApnm1UTej+wC+ZJfs5ctw8HKdgtD9wfuVLxX5WsAvA8+O9M2R1da0VHYY8ZwzAJ70Wi0onzejabNWFmAKwMEUwjDyv8+YOhms79XEsD9eADDNpOTGfc5MaK84xG4NReLvDMAuzorbC++f9Udi+S6iVfC1+kUMR2UHAmY3XPw7rWIjrDmyCwTNRoHHzO7yABTJltZUYWoBNqvqFIzLAropoLsCblgeGMsEi80OgLtO2AQr9p4BjI6WrUtNP1sIO074LMCiyAdtDwH42q07UHWAr7KbnQ+wCgCj68mHEzZqDzmsKEXP907jYjB01k7tXC3L54IvXARgUaVJng87H2AVAKaFFNywjNit+6FPloN3/oxMCzDCTqrkSUyvoVfHrIFpm31BcLLv4ugl+6D3f76qYqIQxfM/rgTMJU03VQvwmgNn9ZyxKgCM0N1mlADgitCsdE7Lu61TALbL7ap6a7ezALcZtRyW7A6vCwSt4ji7S10oks2XBk0FgGmGA3SBwGtMdYsFwO2rwbP+ObJqo1mpHNCwZDb4dOYOzdUKX6bkTW7bOQCnAHyp0dbr8gN6bu7Jb9SG0rkzOMoCYRRbkSl1MlinAMBWKTmdWoBzZkgJy/wWYJpLnwFYdcW6ux8DsIv6ug3AWH4Uyw/TzS0AxocmPjxxE9YxGYDlN3onAGwWMBEMAFOLolUWAqOSl+J8bavng/6PBlZWa/LZAth54rIpAGNi/us370DVgWoAXPuTuXD4nLUFWB4//XfnRsXgs9k+AH64XC746hn3LMB4DnwwFO8Rk1sTrVNOXCA2Hb4ADw+PyUAgrFs7jl+Cpp/7XCDkF5ZwXJ7BALCKxVL0zS0fYHz5waA1/KLzTpPihlKoArAoX/7cDytg0a7TIQMwjTUwswCjO4GAq2/bVAwok2w0GBUAbjtqBSze7eu/XW5X1bUzZeMxwJdx3IxcIOg5Vb9CYVtW9y4zAB6xYA8Mmubz68fN6HyXrt+CMr191SZfrFUQej0cPADTlxQrvZqXzqG/aJn1K5IA+MCZK/DAEF95dbHRUsvyWKkF2C0A/nfDUXjT/7x2w9XLbP7c5hDV6ywS92MAdnFW3F54Rg8etAq+0bCo4ajsQMDsxoZ/pwCMFaDwAi7Ww1dpRwTB0ZLM+Hdq2TDqELUAG1V1On7hOlQf5INJq77JbdO0WlYWSCsAfqxCbvhM8qltPGwB7DppDcBosavmB+DX6xeBd5sWB4xIxgpytQpnDigMUWfwXDh0NngAfrtRUfh8ti/93UNlc8LwZysqr2anLhDYML74vPLrmoD5oACMvnNL95zWApMeGb5E24/qj5keMOODPJ9xDcCYPQMrjNEtEgBY5eWPArDsu0ghS1Rio9ZMMV47X32x3+r9Z/WiCTTWwAyAqX+pSulXOwBevOu07lKBfbJLbaV6MQQAsD9YkB5LXxrcAeCYL2UqAHzh2i0o18cHwC/UKgAfPVwq1lCxIiVeixXzZwqolic/A6ibipVeD5bJqVftNLsXhxWASZBgMBZgWlxHjEsVgK1y0odiAab64PMTcytjFc3kSRLD45XyqC5Xx/u5zSGOOxRBBzAAuzgZbi88+klFDKNL42LwpgsALD98XqlbCEYs3KudVgCwnMSdBncYyUwB2KiqU7BVkagPqpwWjt6EjCr+iN+NgsoowKELRJ5MqQLqzeMYEIAFtAsArjt4Hhw8exXealgUOjcupksh/m6kjXjQWlmAcZ6/9Od/xgfU1228B2DqAoGfY3HOUidPAldv3okFwA0+nQ97T1+JBcDbj1+EZp/7XFZogYVwXZp2L35GhTC+a2v/yV70zy0LsFMAli1XFIBF+XL6wiL6j8VK1h44B+XzZYQSOdKbyk4BmMYamAEwzTCgEqwlxosp2dAtQ94e/XoJYDU6sdmltlJdP3EBwNQ9hX4pk40IRsB97spNvdpku5oFoPcjsQG4z39b4Kcl+6FUrvQw5c06uhTytUADFa30whfsyRt9ZetxM+qX3PbETjWhQr5Mps3G9gEmLhAEgP97vTaUyePMBWL3yZjAWtEBWmlO7hS6vv2+8qD2Zy8AGLOoYAo/TL+YJnkS2NK3mepydbyf2xziuEMRdAADsIuT4fbCo7l5xTDws/hbjcJvAV6y+7TuzydLJgCYBt3gPrTEp5HMAQBsUNUp2KpIh85ehTqDfcUl5KBAKwDGyl0Fu/rKQxsBMH37RwDGOu+FSDlpMwA2A5kHhsyDA2euGq5AfMDQB7N46NC23mxQRA86bFEmB3zTppLyag7GAoy+3x0sLMBGJ6cWYBUAHvBYaWhTLcZnW3lAFjvaAfDgJ8rC+1IluIQAwDTTgAgGpTlthWRYRVAUl7CycK7af1avGoauVq8+UBiGz9utfRVBdyx5ozlm0T0H3XSsNjsAfmT4Yth42FeKHLflXRtCjgwpQ14idgBM/abDZQEOBYBPX76hp1o0A2CV6xuFQ9cY4dpmJSTOHZ1jLwH439drQdk8GR35ANOXajEummZNHmswAGyVRUjonyN9SljezZcFgs4JAjAWccECPJg7e1PvpiGvY7MG3OYQ1zruQcMMwC6K7PbCo4FpYhiylZEOzw4EBGQZSaICwDS3JbYx4+26UDyHL7jDaHt97FrdqmBU1SlYAKbpp95tUgxebxDzQkA1kKPm7969pwOtEQA3GDof9p7yWTANAbhXE7h66zbUGDRX20dYgMU5EyXCal0P6lJYAbDsbyvmhvYf20cAwQ2DVL57zl0AxjRI6JMqNtkH2GiOAwCY6EfXGg1aVAVg9IPsMWmzFozTurJ1RSW7dY9Vmd7/O7AUcnwAYGoJRD1ly1UAALerAvVLZANa1lfMV9oUSeHyDV/xClUAxusKS6rLwYN0DdAyu1hspGX53JZ3WzsApo3m4xwAACAASURBVJUYsaFlXRtofQh1swNgzGWM91o7feR+WFnwKQDTbDnfzt+j5/bG9owg6+Sl63qg7f9q5Ic+LUvHkkAVgGmuZisdMQAXfVjptW81XvzNqQU4a7oUgIGNuNFnSTAAbFTxMJIAGCtepk+ZTLt+sGw4Gk/c2tzmELf67UW7DMAuquz2wjOCUvwsjm4QRpsdCFjd4NGfzCxdzgPFsgLeSMeuOAgYTCA2JwBsVNXJCoDxDf+9PzcCWkBk/ylagEC2iFsB8O07d6GIP7+mUWEJasHEUp+5M6WCwtQC3KsJXLl5G2p+bAzAiRMB7CUAXG/IPNhvYgHGoEAR6EIfOrT/WGpZ5F1WSVNE14TKAxJhiO5nBMDUBcJozdFsAdSCTtfa1qMXocWXPheIgY+V0ZLW22206qCdVc5u3X/yeBn44O+YdYvnVgnaEn2MKxeILuPXw4R1R3SpZACmECFKET/x7VJYfeBcgLzpUiSFSwoAjOWARdlcvMdgnm1a+VGes7861tB9hsMBwLQSI55r6YcNIFfG2ACMX3KcFIeYvPEovD7WF0wsCobQsdDUcXZrTeUaw33MLMAyABvFKdDYCMxwg4Gj8qZyfWv9INX6rK45TLM4ab13AExf3kQ+YRWXIDEGOeAW/64KwFZVSek9zGhu5HuCKCCEf6f971C3EHzvdyG08k22uw+q/O42h6j0IVL3YQB2cWbcXnjL956JVcXnjQZFTCPG7UCAQoksy7I9Z0wTpgsA/mPVoQBL2vS36wT4FKKFFR+0+MaLG7UAG1V1skoKT1OIyQ8lCsDyCwHVQE4bhYEjIrDPCIDrfzofsMgDbgjAuTKm1IEZ/7ahVxO4fPM2YN+08fmD4MQ5kyROBOh3LDbanqw3BgWW9Ud6i99kIMXsAPjAxE0lUb1ox8rPUH7I0H9jAZT2Di3AFIDNguCotUYVgJ0EJtmt+49blYEPJ8Q/AKaFWXBurQBYlCKmebXFeqDli60AD9MDirK5+GJ5/tpNzc/UbPv71Rp62jSstvdohdAswPJ4l3zYQHNDoluPSZtg+uYTMOHVmpAvc2qluzsFYFEymh74wk8rYZ4/d7I7AByTBu2b+bth8PQd+umN4hTo/c0oW40MW7TP8rWArjEv+MtVW4mFQcETycuWigvEhE41oaIDH+Bs6VLASr8FmPqvi3RqTgCYlu0W48JgM7xGEqMlQtqoC4QVANOvgCoATMdkdh+yAnOlBWyzk9scEo4+xlUbDMAuKu/2wqN5OcUwBHQZDcsOBPAYsxu8CgCPX3UwwJImariLvmAVKiyogUEZJXOmDwBgo6pOVgBsdTOkOSDlXLxWAIxljEv09KX5MgJgarE1AmB048BPybU/8fkfi5cRMwCWfWLpnKkA8CsPFIIRC3yBiI1KZodR/7NOVC/al9eB2QNSBm6EqJd+Dt4FwgyANx+5AA995UuPNqhVGcAsI3abE79Mu3WP58QHIN2+aVMRWpSxzltrpCfV0qwUsmoFQrsHvgyE8oObWtFE9UY5kAzHkD5lUrh43d4FggIwulpdvH7LEoARgFp9s1STCYuNtKpoHelu5wJB83Bjm6IaI5030UaDEtngx3bWpXvFcXYATKvnqc4dtm01fzReggYLy4FhspsWtktjHNpUywcDHgtM12h17lgA3K4KvDB6ld3lpsVE0K8NbgAwdYGgACxcKeyuBzoIo2cj/o7p3L5tG9tVjALw1r5NIXXypAGaHDxzFVKnSKJ9ARFucEYvJ/I9QQWA6T62ExHEDm5zSBBdiphDGIBdnAq3Fx5GROMDjW5WxRfsQADbMQNgI2uzOK+wAMsALEpYyjeFmoUzw9iXqwcAsFFVJ6uk8FY3Q/qA6PhAYcCUTUawIluAaYUlIwCmPrsL3qunfX4VJUmxfTsATpo4EewmFmBqTZCXIfqEiVRH4jcZSGkmjoYlssEPig/8uAJgmkaOrjUKwGiNfdpjAEarM3Xdwb5RAEaQRTcNhCq04uOGa2XejpNQq3AWKNfXl5JKvn7MANjKckTXgd0Dn+alxuOsABjdV/ArQcvhi7WUfHTDLzKYWsvq+sff6D0AAzAv37hj6QKBljtxf8JiI3apnuwAWF4/ohiNkWY1CmWG3ztUV7q703zqRhbgl0avgjn+8tFW0COfzGr+fly8D/pO9hUpocHCMgDL9yjcn+a4RXchXL+q55avfXSNeXF0zEutmWCPV8wDf689bLjOje6t+DenFuBQARizlCRPmlgLmKPuOvKYjJ5xVgBMswrluy+1ltEHt3ABMA2UU1qwDndym0McdieidmcAdnE63F54FBzEMPCzOGY+MNpCAWBq/ZHbFgA8buXBgE/JooSlfIOsVvA+GP9KDa2Ih4gsNqrqZJUTUxWA5VKh9Dg5byqtsGQEwDRtmREAox8zWoBFBgrZApwsSSLYNSDGBUL2iaW6ojsFBSsBJ7T/OLaRi/ZphzmxeAULwPLDEq1h6Be+bK+vSpbRRl0gVAAY/XGfqmJvAXYSmGS37jHwrvvEzQHdpwAsjqfVm94atw7+WX8UyuTOAJuOxACligVYFaLsAFi2qMuWK2pFE9Ub5UwKOGgsEYv+vGKNmc0lBWBc27jWrVwg0Hezpf8FHVPNPakYrGiWBk2+XqwAuFaRzPBb+/AAcPufV8HsbSc1WYwssmZ6hQOA5XsUnovmuMWvJfgFQ97Mzi1fC/JXHbOxPFEpD/y1xl0AxnOLfPDUR1qAtJWe6JqGLmW44ddErNhpFrNiB8Bb+jSFNCliLMDD5+6CT2f6Cg5hyrIr/hSP4QLgXBlSwlJ/tTjTG2kIP7jNISF0Lc4PZQB2cQrcXng0el4MQ7Z40uHZgYDVA9DqjdopAON5EIzQEmUFwFa+qgW7ToF793yjk29o1AIsJ4q3AmB8oJf+aIbWJlo8hrYuF7A6aOGK+e/6LMDCZxh3tANg9EHbOaC53qYMMPRkaE0u33dWwPllC7DIu4s7YVL10S9UVVrNqgCMgCvSwmHDP0mfS/EBgFWywgnAmJGhdRXrrA7Yl3ACMOYexowSdKOFG4ReJXKkg+lv19V2M7uW6Fo0KseKxxpZ9Ywmzg6A5S8I8oObArAo7CFnUsDzYonYcwoATN2g8EsT5npWBWBMNaearaNQljQw1yAPsOwyhG5Isp+v0Kx2kSwwpn01pevBzgLc/ufVMHvbCa0tIyA1O4kqANNgYdkCbFSVDIvJ4L0Dt2eq5oVBrcrG6oIqAMt+/WZjoanyzJ4T8jXh1AKM7T5YNifgtUcB+O9Xa0Kl/JksXUomrD0MXf7wFbNB3/NrN+8GFE2h43IKwJ/P3qkXHKLuQlYvQ0ILatU2u2dgPvnFHzRQWqvB7OQ2hwTTp0g5hgHYxZlwe+HRkr9iGPhZvGuLkoajCgWAVSzAmEic+lKKCj6iM/L5ESi2H7+k/WxU1lSOiKY3Lsy+cOeuj4DlGxr6a9Ud4vPDldME0T7IDxf0aRSBZ0YATEsXIwDnzJgyoDQwBvJdvn5bP7dsAcbPc/gAFVuoAIxlUEUUvngJUVnOqgCMn+ppnmM5YAYfAM/9sFIZgOVP9mLeaMS2KgA7icy3W/cYRd/TQwBWhShbAJYKi8gATCFCpHWTMyngesEo9LNXbhpeS3Q9UQDGL03oM28FwJi+SlQEVJlXMV4zC7CcNQW/wuTPnCZgyYs26hTNAr++5ByAsXhH9UKZA9rE1H+ztvoA2KpMrnztyfOH1TRHLd4HWJwHM/gIFwgaLCwDsFFRBpod5+kqeeHjx4MHYDmzi9n946nKeWH86kP6z274AIvGsW2nADxx3WHoPD4GgNE9B1+SjTY7AN7cpylgakCxDZ25A76a60s3SV8WwwXA6Fax8P36KrfuoPZxm0OC6lSEHMQA7OJEuL3wjMo9YnqVbnEEwHIaNARgzAOcLEliTWUZRLAyzxn/g9eorKkMwIWypgEsW4uRxUW6TYXbCgAsR0lbAfCFq7d0twP85IfnohvNPGEGwJifVtSglwE4RdLEsIMAsOwSQM+F1uQK/awtwJgCbvRSXxS+kwe+KgCjhZemeZNTJiHI4UNG1QJsBsAbD5+PASUFSyGON6wA3LIU9PxnS8Bc08plTi3At+7c1da8mQXYqtIUdgLTeOHapv7lRg9tOYuI/OCmECHSusmBc04AmKZCdArAKr7ddgAsV07Ea7BAFvcBuMMvq2GmH4Dt5o4uIhmAxb/xvtepfhHo5/cBpsHCVgAs1hVtFy2z6AOcOFGigOwGqhZgubiN2eMQLc2/r3QGwMJya9amPFZzAK4BlfLfZ2kBpgCM6ffQmGHm2+wUgD+Zvl3PtkNfFsWXHDTEYGQAzS4h9M+SNqa6ndmLuNkLX7jQxG0OCVc/46IdBmAXVXd74dFP/WIY7WsXhB4P3W84KjtLGB6EN4djF65B9nQpAy5olSA4GYDxszy6TmCO4CoFAm9g8oNXlDXFh/+Jize0Ck9ySiA8RrgRFO0+FW7dMbYA0yAR+RMh1UB+mNHCAnYAjKVac2ZIqWeNwL6h7xm6UQgAxkChLk2K6zfulMkwDU+MBVgGQjppaE2uaAPANDm9GwCMFg6RFxn7JgMwWsPa/bhKGYDlKH7xIKIArOIrin1xkprKbt2jb2+vMAEwuhqgfzC+hDYsmU3PCELn1irNEu6HQVcY4CpeDsV1KV/UciEVzB2dLqUvxSBuNNOA8GmW5wD3oy+iVmm+KACjq9WN2+oWYJXsHmKezFwg6BcY7DdegwgPRtApqlOq3N6xwMObv/vyABtZgF/5dTXM2OKzANvNnVFfxPzRdYgFYgQA02Bh6m9Kz/fBXxth6uZjMLFTLd39AX/HCm3rD52DTKmTw6ROtfR7tioAj3iuErxCqjua6YW+xqJUsNl6lK+zUACYrl0E2srS80Nep5PWHdGqquGG+5+8dAM6/bbWcDh2ACxfRwOnbtNz9lJ/eQTgu/fuaWXcUyZLAmjwEUGyMgDTKqNyp9CwM/ed2KW/Vdauyj5uc4hKHyJ1HwZgF2fG7YVHK56JYaBfKN5cjTY7EMBj0PKFSeFlFwCVNGi/rTgQK5gI20S/qY29m8ayANPPSaKsKUbjI0hjMBQCAM2JKcaENzD0vcW8vUY3Yxq1ixaSwU/EWHKtAPjM5RtQqf9srU0jAK45aA4cvXBd+90MgC9dvw31/MEYsgVYBmAjGBFjxBynIp8wHbfZHDrxeVS1AOMNnloh5apR+AKBOURVLcBmALzh0HlHwVKoRzgBuM8jpeCjf+0twJi6D611uKlcS5imS6TEo9ejUZol+rtR20YPbdkiagXAwqfZ6KsDWqmwvK4Z2Ii+Ld19Gp4dtUL7p1MAVsnvbAfA9AsM9mHuOw9AoaxpA251og0nLkEUgLF8czXJBaLjr2tg+pbj2nns5s5sHmX/fQrANFhYBmBxPjGusnkyBJSDpuej7YQbgDHd2m8rDuqnU3GBCBcAY0VB2YBiBcC4P7rWyW5N9D4qPx+7TtioW7jl6whfVH5Y7As2pj7AeH+cseW4XkSFjjcGgJPD6h6NgVYZlc9dJFtamN3lAfnPYfu32xwSto7GQUMMwC6K7vbCO3nxOlQdOCdgBOgX2uvh4AGYNkZvMlYALAKwzABYfPqXH+z0bVqUNaX7vNe0OAyZEZMUnt7ASvScBtdv2QOwDPJWAHzq0g2oMsAHwE9WygNDJBeI6gPnwPGL5gCMqdwQgEU0sgzAqZIlgW39msGeU5dhy9GL8NWcXbDr5GXDFVg8ezotklmeDzPwEqnlVJazKgCjiwMN8vvlxaqA+XfFhg/nl0avVgZg+fO7WF8UgNHtBF8+7Daam9XKaont2MFq74fvh97/+VJSie2rZypo1jV6vEoQHG3DDIBlVwV5rKoALFtERQS9aI+m2kLI/1/NAmCUeQQDdXDt42alJa08iS4QmApOuOAYzRfNAoGBhm2r57ecVjsArjFoDhzzv4BiQ7SEsGhYtOEkKNQOgF8dswambfYBsN3c0QHKEEr/jUHA/ads03ansRJ4Txg6y5dxADcZgPGrE9WAno+2owrA+MWi45g1dpcboCvZmOXeATBduyoA/M/6I/DWOJ8FGPdfuvsMfDY7Rkc6QGMLcAwAy9dR73+36OucFo3B++O0zcf082LwHbpq0HuGKHKBbhLUnYz2B+/1Mzr7gmvd2NzmEDf67FWbDMAuKu32wqMWSzEM9Avt/Ugpw1HZgQAehKm6jFwLrEohi4fNmOUHYkXTY5vC8imfn75Ni7KmqgBcsud0uHbrjjZO+YZGU+JgCc/Pn66g62EFwPSFwgiAqw2crbln4IbWp5wZUkHJXr7CGbghAF+8dgsaDPVFaMsAnDp5Etjat5ktkJktSdmKRPdzkvdUFYDRxaF4j5jxyQCMMPDyz6EDMM1nHRcATF/EhKbhAGCzFzj5ASvPtyoAyxZRKwAW14lR8ZVgABgtwBgEZwXAWMDgMX8hDAw0xLK9VpsdAFcdMFv7tC02tJqh9Yxuoo36xbPCT4pZUewAuNNva2DqJh8Ay9ZBlfHgPnJAqSoAi8BGMS7qgyqfOzgArggdxxi7CtD2ce5+XX5A/5OaBTgGCI10svIBpgD8xys1oGpBax9gGYCnbDxmujadAjBaksXYMTgO3dxwQwCeuumY7nphBcDov02/plE96Iu15QUS5I9uc0iQ3YqIwxiAXZwGtxceDdoSwwgVgNOlSKqVKxYPTNEu/fwpSyYAGG8SRp+dhOXTCoBFWVO6z7tNiun5F+k58QZWqtd0PR+jfEOjwYEPlc0Jw5+tqB9uBcDHL1yH6oN8FnXZdQL/Rh/ACMDop3x/L1/aNNwwlzEGXzT0A7DsA4w5JLe4BMAit7I8N2sOnAO0zGPlLhExHywA//pSVS3rg9iwWAd+HlZ1gTCzAK89eE6vGKZSMAHPT4sThGoBNroFfPlMBUAfa9yEXk5dIMxuLUZFX+i+qgBMXXLweJwPUWYc/42fbYWfqbie5UwK+HesRCXA0qrS2eJdp/XUUo4BuGUpeK5GAcu7rR0AV+4/W3fVwIZmd4kpISwaFm04yYttB8Cv/bYWpmw6pp3C7uXFbB5ldyIzAP5yzi4YRizAwuIsxkUBTBYzGADG4MhXTXxlafuYTefnZaEDMELvjVt3tNgIKwCma1e4pZhZtbGfFIARmMeuOACT1h81XG92ACxfR7RIBs0DjAaCaZuOEwD2pWuj9wxhAUZ3Pfo1jXasVK70WnVUtza3OcStfnvRLgOwiyq7vfBo3loxjOdr5Ie+LUsbjkrFAmxmDXIbgEVZU9rHdxoXC/gcKAaFNzDM1yvexK0AuEWZHPBNm5jSl7R9OaCF+lQbWYBDBWB8eOEDTWUejCbQygKMFhK88cubOBdNtaMKwOjjK0pDY7tjXqoWkFsTA/UQDhIqAO8+cUkrcjFvxylN1nABsFEJ32AAmLrkaADcqwlkSB0TBGcEwHLgHB6XPX0K/cuGVZU6CsBYhvvGrbuWFmBaClm4YFjdbu0AuFK/WQGBgVYA7KQyIoUnIx/g18auBbQoGmmsMh7cR76WKAD/93ptKJMng9bUF7N3BXy6FxZnoQ0GAd+843P9kjfajqoLBAZHmgWL0fZpxhn8ezAW4B3HL0HTzxdqzaK/LAZWm7m40bUrAhOdADCWAZ/snzNZJ6cAjAGIIgUcfsXD/Ne4IQCjBTgm/VpsAMYA0zU9G2tfS+i9lPYJi+n890Ztq6UU0m9uc0hInYvjgxmAXZwAtxee2UVl9hBTAS9a6pFag6j/nyyZbgFetj9WOincFy3A+CmP5pTFv1N/KlHVifaxS+NiAdYQcV4sHSl8cY1uxuhjK6ywTe7PDt8/X1nvshUA07RVRgCM/sHCVxL9D7EfpfyFM/AEmMv44rXbepS2bAFG6/omlwC4SoFM8GfHmrFWs9FDQwWAEyXyRbzTm/Zv7atBG38QFJ4ILd4YPR8qAKOV+vFvl2p9H9a6HLSqaO8DTKtzuWEBxsIN7/+1MUDPcAGw0ad7cSKzaHGjMVKXHDxeBmCaL1ZcJ7SYi9H1ZJXbdNGuU/oXABUARsgR84rnQl9HhGJaZYsKbAfAFfrO1At24HGzOteFotnT6U1Q7RqVzA6j/hdz3Vvd5mXrIb5M0u31sWt1mMICNRlTJ1d6atDrDF98RZEdPJgCMOZLxvK9uMkALCzOKvfuYAAYgyMR8O02LChEcz4HA8D0GfLF0+Xh8LlrrgAwvsT8svyA/tIij03uO/qy9528RQ+Ck6+jd/7YoJeBRnc+EXuCLzXoAywAmBb+EPOlAsDl8mYE9Jd3a3ObQ9zqtxftMgC7qLLbC+/2nbsBaarEUMwq8KjcRGnwFU36Tq0/smR2AIxvzfhpaNGu0wGHUncLJwAsn1/+bEsrJckPQqqBHNFN08oZuUBU7j8LTl/2FQwwB+Bb0GiYz8oRC4BTJoVNBtkwVJeglQW4cv5M8Ner4QPgxIkAtvRpFuDjPLZ9NT0LAPYZU9e9PW69MgA3/2IRYPVCsYkHEQXgz54qB49ViHsANvr6EC4AlkuE0/l3AsD0hQzbkOHMCIBlv2E8jr5QWlWpcwrAmI7qie+WBSxvLDvdppqxL7AdAJfrMxMuXPOVbMZtZue6UIwAMA00anx/dsAiDyqbHQDTku341QP9cFU2eq+RKztSAEb4QQjCjVYdw3+Lz/Eq9+5gAFhk/bEbDy26g/uqALBIXybalp8hGEiJud7lDdumFuDfX64ONQpntswDLLux/LLsgO62YtS++NvmIxe0l7Qb/oxCmubSl5S3x63T3SkwoFvsiwCMFmBRgc4IgHGt4JpByKbxIrRPFfJl1NLbubW5zSFu9duLdhmAXVTZ7YVn9rAUQQPy0FRuouXyZNBKFMsPVPrwk9sVASf42UnOp4r70s9G9FgKwKKsKe1j50bFTCN5aTtyTXYKwPKD0AqAaQU5IwCmn2ARgLOnTxlg1cFiHviAbvyZ9wBcMV9GmGBwEw3WApw0cSLN35H6OMsAjIGLaB1RtQCbA/BZePxbHyh9/lR5eLRCbturkpandcMCLKd9wg6FC4Ct0kM5AWDZJxaLp2QicGYEwLLfMI6LZhbAwgh1imaFVMmTxJoDrGQmsoCoWIAxGv9JCYCtghx1ADbJi1qm9wwty4rYRAlh1AzXYIHMaaDmx3O1n+UvP1YLyg6A8SsHAhZumOs7c9oUtusTdyjUdQr4a/VoX0tEhhn8DavBDZjqywIx6bVaUN4MgP0wpnLvDgaAMdgTAd9uw/zyWMVObFYBiGIfGYDlZwj6xwqDAj0/Xs907Y59uRrULJxFGYDRZQLLgIvMHfLY6P3isW+WwLqD5wN2kV8k6QsQdUGRARiDPivkC/QBFgB85cbtgK+F9IRoHMJ7glub2xziVr+9aJcB2EWVvVh4RjfGUAAYg6lW7DurqSIC0/C/rQBYZHnAm46cTxWPVQFgUdaUjqdRyWwwe9tJ2xmSy5PuPnlJt8IKAJ626ZgWICTymGKjculYmj8YS39+8kRgiVEsTCFKxuIDAC3GL4xepfcPAfj8tVvQxASAzfIh2w7Qv4OVBdjMihAsAGM2kPW9mgTctPFB9OxIXx5YbU28X19zE1AFYLkMr3gQrd5/VrcU4qfRluWDA2AMNPl77WHtawO1DKrAgzwHuG5E+VvxW7gAGF8kahbJYjjtZvlCjSBf9om1AmBRbUr2G8ZO5MqQUs9vjf82cx9YsPOUXl4WS66jJcwqCwTeh1qPCLQAYzGVesWzGY7dzgJM/f6xAVFCmFYBEw03LZUdRjzn3AIsUm7RDmJhk3/8AVV2AYz0OFqsB9M81hjkg3PcurUoAQOnbtf+m4LTZ7N2whdzdun7CRhTWcPBADAGe4oiIFb3oZfrFISRi2IA2KjgktxHGYDp+sFzGWW0QNerfYMkAPZfL1Y+wP9tOKqDPALw6CX79dzN8rjotfTEt0th9YFzAbvIAEyzgNAsSQjA6Bv+zp++EsxWAGwUryNOWrXAffBHx9jxG1bz4eQ3LzjESX8iaV8GYBdnw4uFF24AxnyyS/ec0VQRVlntv4n1x0gytBZOXHvEEIBp5Cw9lkY0GwGw6tTIrgy7TlzSrbAIMp3qFdbTMdE2ZQCm2SOMAJj6IOJn7Ae/XBzQRXQJQAuwDsANiwL6MYs5QgDHT5oqDzOjsVsBsJkfWbAAjBZgtMT+teaw3hX8FPnMyOX6v7EULUZIxwUAv/zLah1QhQsM9Z+kD7lg9MbCIot3B7rshAuAf2xXGRqUyG64vJ0AMH0hw8bkz/MjF+7VrYyiUIqcSgyPy50xFRw5fy2gP0bA7RSA0Rfzqe9j1guewAr+7QD4/l7T9QAkbEuUEH706yVa5Ty6NSuVA757Lib41epeIqfQwqILdKOfwNHPP1u6lEq3JpqrHF8W6wyepx9HAZh+OpcBWLzUqKxh6ktsBotyO/jCKfLnWg2qQ91CejU03M+o4JIdAM/fcRLa/RRjMDACYHS92hsiAON9avTSfXr1PnlcdG0/98OKWK558oskrQSIld7Q1QY3BGAMtHvXD8DUki+0wGJP63o10bIDle0901Bisww+SotMYScvOEShGxG5CwOwi9PixcIzujEaWTFwmCo3USqHgFL8m/z2LsuG+87bfjJWQQHczwyA6d8RpgpkSeO4j9i+nJpIBuDqhTIHpIMSfZeT2lPXiaer5IWPHw+0AJfvOxPOX/X5IMoBYfg3BGD8XUQ6ixQ4QnfMN4tWVafzIPprBcC4j1EKq2AB2OiywAfLs6OWwz3f/V/zg+4xcXPIALxq/1n9U7mqBZgCsAj6bDB0Puw9dUXrW6gAXL3QfbB8r+9LiNjCBcCYeqp5mZyGdx4nAEzXIzZmBcC1imSG39pX1z7Di0BO0QFVAKYAo2IBRkvc0xIA43VTy8T6becCPxcolwAAIABJREFUQYES+y5KCLf8eglgMRW6NS+dA75t6wIAd2sI2dKrATBN1YhpE0V+cOwnBWDqEoMp0DAVmtjEnKrcM9wEYHR5GbFgr94vFQCWn0PzdpzUKkeKjVYCFX9DwESXNuoCIdaMlQV48sajekU2vE9hKeWZW33lq+WN3hso3Ir9ZACmAbcI6MKtBYOE8bzv+YNlhS+37MaEVU7RpQh92I02J0WMDBuw+aMXHBJMvyLhGAZgF2fBi4UXbgswlUNAKf7NDoDRWjx3+wlHAExdI7C0MH6mVbnRy1Mmf7LaeeKSboVFCzB+YhL+dvRYGYApOBsBMA3C+a5t7ATyeKM7f+2mVhtebBRahTUgmDFie3YATF9YxPnDCcAINJgFQlhAMAoffb5VLcAPfrlIq4BHtcH/pgBM8+9aXZodflmtP+CED7gRAJv51Npd9hhUKH8aDRcAWwX6mVWMMrLIlu09Ay4Sn1jZP/X7hXv0z+wCgGkgp9AgT6ZUWkQ+3YzO5xSA5S8G2D6m0qtd1Nj9ww6AaflzbEsEE7YcvliPWxBjkNMfWs33pHVH9Fyu8md7PK7L+PUwYd0RrQl8yUXff5WNzg8G7IkvQ3isKQDP3AFfzt2tNy/mVOWeEQwAo8/92+N9FdSsNjlgzajiqNxHGYDx+fDi6NX6aYwK0OCXp90SAIs1I9/L8NpOhD4TWE1PAuAfl+yL5cIkToxf5d5sWFT7Z+fx62Gif27F7/KLJC27TjWSAVjoL99zEHAx3Vz5vrMMJXZSxt5unox+94JDgulXJBzDAOziLHix8LwCYPnzlSwbfuKbve0E9JFKyuJ+Zj7A9O9oISmUNW1QACz75dF8kxgMg4EJn0z3+dvRTQZgepwRANMH2setysCHEzYFtIcAfO7qTcBgL7HRcsKhArAcSS6Px6o0LO4roEZeM6rWUvyk3faHFXqlQPwE3fe/rSED8Mp9Z3Vf0WAAWKTukgEYqxe+OmZtQOYA1csdXUpkq+L9OdPD1Ld8CetVgMTsXINalYFnquYz/NkJAMtBYfJ1QAFYWJmMADjvfang0Fl7AKYWPBULsOwzjgPGYioYZGe02QFwkW5T4bYwv5ESwo8MXwwb/YG7ot0Hy+SEr9vEFMCxmndbAP5jPUxY6wNgUbJdZR1RFxXZZapr8xIwaJrvnkQriA2duQO+IgAs5lRlvQUDwPgyJtJ4WY0J3ci+IRkbggHgOdtOwEs/xwAwuoTRrB54fvSx3TWgBVD3HbFmqAYYRPj1/N0wom0lqFYos+aLK9K54brDSnJW8SPintdt4iYYuyKmxDP2QQZgIzcJ3A8B+L+NR/V0iUJ/+SsOBnsvfL8+VOhnDMB1i2UFrLLp1uYFh7jVd7fbZQB2UWEvFp6bLhDCKosSyZ+vZNkQgDFoqO/krbEUNQNgzA8syhkLeFO50csnkK0yMgAjzBglXJfLmmKKLgGvz1TNC4NaBbpAUOCgDzDRH3w4nrtyC1p8GQPAtK8iIjiYMWI79POb0bJ1G4AxqAkfBiINEFaeGjBlmyUAYzaFAY+V0br70FeLYPOR2BZgTIgvPpXTEsRWlyb9dClSd8kALAOTk0sdqzNRazUeGy4A7tuyFDxvUhXNCQDLQWEyAI9YsEeHLFEq2wiAae5voZGRBZjeA9AnFIMOrYLg5Kwh2HYoAEyzKmBbovLZw18t1gqW0O3BsjkBc9yqbHYATPPAipLtKu3SLB0IuSLTCR5L7x/U6jxkxnb4el5MajCj9JBm56bp1FR9gDHvtkjjZTWm1+oXDugX5gX+6OFSAYfYWYBnbz0B7X+JAWAMCqZfMLAxzLKwc0BzWwAWJxYuEzIA/7BoH8zZbh5ALdZ3/8lbA7JbYLvyl5Q2o5bDkt2+uBi6YewJ+gCLfOEiCNEIgOe/Vw8q9Z9tKLFII6qypoLZxwsOCaZfkXAMA7CLs+DFwnMTgIVVFiUKBYAp6FK56d9FWp1g4JBmq8D2tx+/qLshoAW4dO4MhgU1ZADecvSCHtiGANyvZWmYu/2klqQeyx6X+WiGXibaKIelHQCLpOjBjFFlmdqlJrKzANu5C+Anzed/WKm/tKDFY9DU7ZYAjP0WwUpuATBa2ZMnTQwyAIeiM82HLbQPFwBbVUUzy+1tBKTUxxT7iKm2sJKj2IwBOLCcMO6bP3NqOHDmasASMwTg7Sf1rCcqAGzkJ4+WLrR4GW12FmB5PgVwyK412LZcAt3q+qFZJIxcIDDISQSD2lXxo+ehGTdotT3chwIwdRUYPH17gKXVSeBsMACMpcdFFgMrjd5oUCTAMq0CwHI2IjSQoO++2GghJPE3vI7xeqYWYKxuiqW3RYo7uZ+4VgMAuH01DWrx3m22ifU9THI5wf1lAH5qxDI9M1IsAN5wDN7/21cwR6xH+SUWLcDz3qsH+EJktDkp263yHJD38YJDgulXJBzDAOziLHix8NwEYGpRxAA3mvJLlg0tFRh00M/AAkyr59Dj6N9FWdNgoIVmq8D2qSUXAbhEzvQBgSWiD3LwHCZFf+grX2YHBOD8mdPAx9O265/lKHDgJ+zfVwZ+OkPrELpAyNkhxPnkoLhwL71QAdgsAEv0Ex/U7X5cCVf8pUAx7c/g6TtsAbh07vQw+Y06phbgZXvO6NklMDH/Q2Vz2UrT8dc1epojkQYvnABcOGsa2OMPqBOdCRcAo8vBqOerQPEcMVXMxDmcALCcFUHOUPDdgj3a+sUNg/rGdaihPYRPX74RoK8VAGP2l2MXrkHrynlh/o5T+j0A02LdunPP0gIsl87Gk/78YlV4QALgFXvPwM6Tl6HnpM1av1D7Oe/Ui7UG5HuD+OQsp9fDAx8plwvQnUZlowBM3RHEsRSAhUVWpV2joiPiOArAFBRxvnDe6Gbn+y/2DQaAjV70jMaGRX2ob3IwADxzy3Ho8OsavXkjAMZCEzv6NwfqvmOnNeqDBSlESWf88jBy0V69hLnR8QKA5cp7uK/8JcUoVRruh1mEELwFAIsvEkYAPPfdegF5oGmfnFQttNPC6HcvOCSYfkXCMQzALs6CFwvPTQCmQCUHMMiyhQrAoqxpMAAsB3/JAIygQf3qRN9lAN54+Dw8MnyJ9jMC8Lztp/SSy3jDpACMPoZTNh0LkAEBGPMEC4iWNcqSNoV2cw1mjCrL1A6AsX/owiCDnXgYmMGXODcC8Is/rdKt4AgLn87YaQvAeDyeQ/5ULc5LARg/W+Pna7uNAjCmI0qZLAk0+HQ+7D0dkwVCDpqya5P+bgSF4QJgcR4jK6sTAJazIgQLwAUyp4b9BhZgWr0KwfXO3bt6EJMKAKO1VxTOEGM2ygMsXw+FTAphyPsJ4JMLrOC5WpbPBV88HR4Afv+vDfDHal86QCcAXHfwPDh4NtCyLnT4sHkJ/eUEfevRjxW3QVO3wYiFMdkWxLWjcs8wSsMljhfnVWnH6Dp5q2HRgPzE7WoWgN6PWLtAyBbg6ZuPQ8cx1gAscso7BWDM8/7qb76SzvjlAQEYX9jMNnGvVAFgo2IZ2C4CMAbfffC3LxZEALB8DaMFGI1JVQfOMeyOk6qFTu5hYl8vOCSYfkXCMQzALs6CFwvPXQCuC0Wy+axUdgCMnwbxBtd/iq+6Ed3MLMC0rKQoaxrMDVoG4K1HL+p+uGgBLpItbcBnRdE3UWZU//eh84AplXBDAMZPaCcu+qxlCCvU4oZR9bJfGLpinItTAI6ZL6MHXpncGWL5StIH5K07d6Fo92mmVwQCMFZgE4Er+OD+bPbOWOnCjBqwAmAMVhMFNgQAY5ERLKryRKU80MIgZZgKAMtBYk4udaPUYF4AsNkcGMGyDPhYiIWm6MIysyL4U+QaNbIAF8qSRn9xEBrh+TBdmqhehlUZ0ZIvgpgQgNEH+OdlB0xlNQJg3Fl2MwgWgAXwNft8IWw/fimgH4+WzwWfBwHARi4QH/y1EcavPqS1b5RpxUwA+kIm7/NBsxL63GB2FUzViNvAqdsC8u2K61PlvugmACPwUn/v4AD4GHQc44NU3GglUPE3dIvb1q+ZYwvw9M0xbeOXh+8X7dVy15ttGJeytW8z7csgpp6jm+xKZBRkiftjEPUUFQBOmRQQuKuZALCTqoVO7mFiXy84JJh+RcIxCQKAZ8yYAQMHDoStW7fCxYsXIXfu3PDoo4/CRx99BBkyZNB1njp1KnTv3h22bdsGefLkgS5dukCnTp1izcOnn34Kw4cPh+PHj0OZMmVgyJAhUK9e7E9ydhPoxcIzujH+r0Z+Lbl3/8dKayUkxaZyE6VjEr6b+Dc5glceO8IfvoUbATAFXXocLSspypo67SO2R9O14b+pLy9WhMISqbJVBfeTAXjdwXN6wQwshDF3x0k9ZyoCQcme03X/V4QBGtCF7aEGZy/fhIeHBxbIEGNG/0y8uQYzRru1hr8LNxK6r8q5BFzduH0HiveYbnoqtOhg8Nk5fy5kjLbGB4icL9eoATyH/CAR5126+7ReoQ/TBSHw0uIGRvD36pg1eqlTjMbGPJv1P50P+4gF2Aj2VHTEfbKlSwEnLwW6CkQaANNKY9hnORiUAnDVgvcBzp9RHmC0uIr8yUIf1BxdJYTfogzAuB8tm26kK1qN//fjSkPJrTKPqLpAiMpbRgD8WIXc8NlT5ZWm+8/Vh/RcrniAXKr6w783wrhVPgCW7zX4t8W7TkOvfzbDB81LQNNSOfRzUpccuSMUgDFdXI3CPgA2CspSdYEwqkSGbapmeVESi+wkZ0uQ7zWyBZhaabEZWghJNCvAlPqv2/ULx0cBGAMtv1+4N1aBC7kdPE4FgI1cbLAtBODJG47q2YCEBVh+iUVXDzTw0EqAtC9OirbYaWH0uxccEky/IuGYBAHAv//+O2zcuBGqVq0KmTJlgs2bN0Pv3r2hYsWKMHOmL/n0smXLoG7duvD8889D27ZtYcmSJRogjxgxAtq3b6/PBcJvt27dNKDG40eOHAmTJk2ClStXajDsZPNi4dkBTqg3v2er5YOBj5UBOYJX1iEYAKZlJUVZU7vxGOlvB8B5M6WOFemL7Wzo1QQypE6mN7nmwDl4/Nulpg9s+snZKHcqWsExC4QZACNUrYxgAL5+6w6U6GkOwAinWPlNWIDR0vL1vN3KLhBmALxk92ktvzBuokhE8R7T9GwTdgAsPkvLAGzlg2l3HRtVqYo0AJazXMgA/M383ZqPNm5WAGzk7ywD8NuNikLpXBkCovjtNPzphSoBhQ/o/lb3JVUXCFFBrc7gubHSuLWqkBuGKQCw0Yu9KMcr+tt1wkb4faUPgGlgsPid3rMwFSJe54kTJwp4IZO1ogCML5LCUIExFD8sjik5jMepAjAFd9UsEHZzaPW7nCrSDoBpoBq2a1QgCf+2pW8zcA7AMe4VoQKw7ErU9LOFsONE4BcGAcBYghnvibhZATAaeMyC+JwUbQlmvrzgkGD6FQnHJAgANhISwbVDhw5w5MgRyJUrFzRv3hzOnj0LK1b4HrS44e+TJ08GXCCJEyeGGzduQPbs2bW/Dx48WNvnzp07GviWLVsWxo0b52jOvFh4dsAYKgCLG7AcwSsLgf6leIMzKjhhZgGmACyszXbjMZoAGYBpMBtagHNlTAU/Ldkf61AZgFfvPwtPfLfMcI7RmoFlgEURCAxoO335ZsC+CMDoAyz8iOWGMBJ8RTf3LMDCj5qeV0VPUUGO+nyqLHS08H07f7eyBVguWCDWJlrQML8wblhgpFnpnGAHwJ1+WwNTNx3Xu4np0zCpvcgTi21bWeBUxifv4wUAo1sBujbIm9FLQOFuU/X1iPsjfGG2ErEFAHCB++CPjsYWYHQRwiqIdMPznbl8Q0/dhD6g6EJD01jZafhTuyqmgbPhAGAEPoR3owIDrSrmhmGt7S3AZtcH7R8FYLtUg6iJ8D+uN2ReLN9qoZkZAGNebSziIM+FynVM3Te8AGDZcin3kfo243hosQr8t1F6TLQKo2U1FABG1xu0AMulzI2uqa/m7IKhkguEDMCNhi2IdX1gW5hF6L8NxwBzCWvje6P2/9k7C+gsjq6PX0IgwYK7B3f34MGhFGipQJ0K2tJSWtyhrrxQ/aqUFgotFCcUKO7uUtzdvd/5bzLLPPuszGqehJ1z3vPSPDOzM3dmd39754oUcUh5D0MDjPdbzLv3U2HzY/EB2OhJ4t7vyRaAp06dSh06dKADBw5Qrly5KCoqit555x3q3bu3LM3FixdLpg1r166lKlWq0MKFC6lRo0a0fv16qlTpvgPFsGHD6MMPP6SLFy/KmWdEliQ5ATCfmlJt7ggBNmOTOgAjtA0eCsrCAzBLayryoFf2w8crxm9KAN5y5CIdu3gj6PpKAOYTMhitr5qGEJpIADCzI1b2kSsqUgqkX7jfLKPuLf1uF4Cv3rxDZYbMFb72/z1bVXrRiJpAaAHwkj2n6alv44/KGQDz2nY1+FMCsNrLTc05SnhyKhURG3hmL/uJMFjXavMyA8CF+82U01KjTyUAQzvP4l8jG6IWABfLkZ72qAAw9jKSOaAgcxYAmA9jZSRL7A8+8xdf3wkTCGVsXb5/2I5/8GgF3SHiJEMrPW0gAG+RI74YOZrya6vnBBcAwF1qUO2E9NB2AJgPpxaKAAxtac+JG+Q1UQuPCbvgLcOaSZEwWAQTo32GtZq77QS9nBBhAgD85T/7VGP3KvegKgArbOn5kyW+vSgAA+oRRo4lPlHOx0zSFiNZqP3uBYdYGVcotElWAAxt7e3btyVb4Oeff54KFChA06dPl/67TJkyNHv2bGrevLks99OnT1OOHDnop59+kswixo0bR927d6dr165RmjRp5HqTJ0+mjh070uHDhyXbYdHixcYzAkanNMBG1wHY4QE3elZwxjUtAEbaS6axcwuA1ULtsPVTplDmEzIYrbFaHnsA8NmrtyT7VbUCAIamPLp/aALw5Ru3qdxQ9Xz1avP5+umqUsYlvVTIPBBoATCcVVi0gC86V6HmZXMF2FurgWL3CeuDonAoX268HbHReor8HmoaYOU9qcxSxgNwtUKZafIrtan6qLgg2+biOdPT7pPBGmC7APztM1UDMn+pAbBa7GlRG2BoPLVObB6tko/eNwBg2Ccjxbta4fccny1M9CMTpyr13l8YZJqBa8HE4s1mJWTzFEQtqJMAwMP+2hZ0WiVqAsFrXL0GYLV1VGqAp286Rr2MADgynLYMbUb86YXRvakEYL0PL+UeHPv3HvpgXqATnNKZVM3EBv0gihDeeQP+iA/fxzTARr4UQQBsImmLkSx8ADYnoWQFwIBTmDygAHR///13SpcunWTvGxMTI9kB16xZU5bQnTt3KFWqVPTpp59Sr169aNSoUTRixAi6cSNQWxgXF0dNmjShTZs2SaYQWgUOePgfK8ePH5fsks2Cs5klNAJTrwAY2qdpG4+qfuXyzm783JDFh5kUIFVomTwZLTmIKTXA0Phq2eHy11cCMB+NwGgN1NJ4AoDhONRunLodce6MkbT0rUaEo2s3CoukwfdttD9Qd//olpLN4qUbt6m8CQAGrH6//F9xDfD/lgWkF2Z7k0+z/eVTVSRHIt7h0CoAd/xyBUGr71TxAoC1Xp5qMlCurTJLGQ/AVQtmpt+7mgNg3gQCcWBxvMvHcTWS6zdPV9U0mWDzsQPA0Hg+qmGy1LFqPnrvEX0NsN69wct7wB9baEJCulzRewxwBOepI+fjU0zzdtZ47r3RtLgMwLCljykW76w8dPq2oNjKogDMO9OFIgDj/fDqrxvlbaMWHQgKCwAwv3eN9tmnj1eUnK8RtQQFab+X7wvO3KbsB3JVA2ClLb2WL4ESgFkiDLMAbCZpi5EsfAA2J6FkBcBwhLty5Qpt27ZNAtmiRYvS/PnzaeXKlRIA4/9r1KgRBMCfffYZ9ezZUwLgkSNH0vXr8Q8tVtBH06ZNJUc7PUc4ON7BXEJZHnQA5k0deNnwqX3Z17MIsCnlq3RM4eP56t0OGwY1oczpUstVeGcso9tITbNsBMB5MkbS4r4NdUONGV1X73fRl7OyDwbAF6/dpgrDxTXAcIr7ccUB2wDMZxkUBuBf1ks251oFL7fHv1ohNDZRmXthApEYAKyWDEHpBIcjXAAwO2YWkRlOCLRMJhhgqiVfEdUAwy4fHzlqBVFc3n1EW1mBNqIAPPDPLfTzyvikN8xZl7+mWj/4WIIG/cSleGVK5QKZaP2hC9K/8Tx8Lba4bJ7Cp4e2A8C8M53XAKy2jnx4N8ybTzmN/1bzDWGZ78wAsMheVKsjCsA1RsfJ4TD5fgDA0zceo4EJCVxYYhYjZ2LlWNpUyEPwYXCreHES7dbY3e43WQEwL6x169ZR1apVCeYLpUuXNmUCAQCOjLzvTCJqAvGga4D/3HhU2G4LaxVqAMzbohrdeGoxLJGR7szVm9ReQwOM2LKzX6trSstqNA7+dxZKzujlrAXAiGFcKcHmU+S6yLQ1cdUhIRMIHAlDM77xcDwEoDAI4rMMfvVUFWoqogEWAOAnvlopNDaRuaJOqAOwMiU4r92qUjCzFN5L7WVeMleGoDi6WJtTl29Q9VHxwfutADDWUktjzNZemTUL1xIFYByxP/bVStXlU0YoUKskCsDIUPfTyvh4x3xoSNanyEd7pQKZaEMCAONE7NXYYjIA8+mh7QAwb0rhNQCrrSMD4Gu37kjPRGWsZjXTuExpU9HGwU1VNbOi96loPS0AVtrSa4VTRBhNmEAwAGaJWcwCsJmshaJz4+v5AKwttWQLwLAHjoiIkDS6cHzzneD0NR56N5bIERyOjf7YYB2A2fGRyMtEOValBngTl9BCb17KOJawB9SKW6rsRy2EDzQ5aVOHa4ZSQx9a5iBWHmzKNlYBeN/oloRjWf7IW2Q8nzxWkX5dc0hIywotc/vx6gDMJ1lhAMwnHVE7/u/xy3qaYaABfvLrlUJHoSJz9QqAtV6eIiYQiEKSL3NaeTo8AEMDObVbHao5eoGslWQVNQH40g05e1WPhvEaYD6Tl5HcoM3X0hiz+ahlvhMFYADW4xoAjEQ2Y9o7owFGjN8fExJ+MF8Ffu4iz6yK+TPJH38AP0TVYA6KfHroIdO2BiUXEXn+Yjy8JtlrAFZbRwbAWlnd1J6F8K3YMLgpqTmnGe03s7+LAnCl4fPk2Of8NQDAsGtmKbxZIhKzAGwmaYvZOaK+D8APIAAvXbqU6tatS7/99pvkwIYwaBcuXJDsgFl55ZVXJCc5ZRg0/B0RI1AA0rD7henDgxwGzeghD8cBZNRhAeNFblQ4g/z3X3xNdnxkdB21fpWhiaBl1HJE49srAZg/ijcav5oHM9q8XD+avlwcmMrUqC+nfrcLwAiFhZA/ouXDRyvQ5HWHhQG4wxfLZS0YrsEgiI/FimNzpAbl0077AByYzICtj/JeUQIwDxFWAPjkpRty9qruDYtQubyZTAEwbMS1gJmtqVrmO1EAhs0rQhOqlSeqF6Ax7fXjtotqgHkoZaZaZgG4Qv5Msv07bF97NroPwHx6aDsAzGuSvQZgtXXs1qAI9W5SnMb+vTcgjTKTnRoAI7oOnstqCSpEn0ui9bAH1UwtlM6k5YfOpUs37gR1iyhC0zffB2CWiMQsAJtJ2iI6N76eD8DJHIDbt28vmTsAVBG9Ac5qiOOLmL5r1qyh1KlTy4kwnn32WerUqZPkGDd48GDNRBhjxoyREmF88803hJBqD2oiDAYqRmCKl69WnEORm5YdHxldx0kAXjcwlrKmj5C75I/ijcasFdvYqJ2bv4vaJyrHwDTAZlMHv/dIefpj/VEhMwNc4xENAOaTrDANsBEAI5wSjh+1Cl5unb9ZZRgL1Mx6eGECYUcDzBKCsDnxAIwj+D9MaoBPXLxBNcfEm0DEAzA0wPdT2RrJDiHttOo7AcCweWUptJVjYQl89MYoCsC8WYJlAM6XkTYduSgNBx/PPRoVlTXAPADz2mY2dlENMN+PFwDMx6/VCt+HkwM42AJolUXNNyRrutS0blAT+jRuj5Rm3c0iCsD8s4gfjwTAm47SoGnbpD+zxCxmAVg0aYtVWfgAnMwBGNpaaHr37dtH9+7do0KFChGguE+fPpLpAytIhYwsb3wqZIQ94wu8klkq5JMnT0qaX8B0w4YNTe8/LzaeETA6EQVCL6UpEwpeDK0/V08BLCI4dnxkNB+1vpQaYD6lsd61lQBslO6Z70srtJvIXN2qYxWA945qQeEpw0xH4ICGDZ7dInGA9QCYT7LiJAA/9e0qw3SoZtbCaQDmnZbYOJwEYKUWDQkGYj9cLGwCcezCdTl7FRJLrD1wng6duyYsMjhJdpugDszsuaQGTkjMgXi7yqJ8NsCuGdkb1UqnGgVoVDtnNMA8ADNTLf6aIs+s8vky0uYEAIb5VLeG9wEYGfMalsghdcnbG5sFYCQeaVgyvh8vAJhPhKEX+QD245//vTdomfQA+JO43fRJXDA0C28+gYpaAKyMpsIn5eG7RRQhmEAMTgBglonPbEIh0aQtAlNSreIFh1gdW2K3S7Y2wIktWFzfi41n9PB1AoBFZAkTBq0MaCLt2fGR0XzU+lIGp19/6LymIxrfXgnARtnuAgA4ZRjduhuc3ENkrm7VgS1nl7rRhCQArIjI0wsAxjXgsc884TE+tjfnbTshO0uxKBBaGmAEvMdLB/bKeuCNvp0G4LJ5o2hGT+cSYfAyYOvlJgA/XasgYY8fVySFKZU7inYcvx++kY3r6IXrhBBQVosIAKuBkygA642rc80CNPJh+wCM0FrVRsXJl2InVfy1Re4x7J2tR+NljMQIXRsUkTXAPLjaAWDEv21UMmcQACPMVouyualV+dymP3L1ZMwDsJ7Wk4d/vj8+Djz7e7b0EbR2YCx9PH+3qtmE1b2o1k4UgIsNmEW37ybY6nEdAYCnbTxGQ6bHa4CRmKVKwSxkFoBFkrbYmbcXHGJnfInZ1gdgF6XvxcbSDx8AAAAgAElEQVQzeviyNLeYplFdO6LA8Y9W9AORftnxkZUxAoDzZIqk579fQyVzRdFDFfMIjQUPWjxwWeGzCRmNWe3hbdTGq9/NfvRYBeDR7cpJZggiiTD0AJiXOwPgskPm0pWb8XZ3ZufD2iC5BpJsOFWc1gAr54b/1np5ijjBIQpJgaz3neCUx8jR2dNJ/YsC8JHz12yZNf3vycrU/Rd9DbAaODkBwID94W3L6i69iAlElx/WUtyOkwH98Bpb0ecqwqJtT/jIQAQZ+AqwBAw8uPIh19hFRU0gEHc5tnQwAJvtR/R+4U0g9ABYy19C7TrZM0TQmgGxkj+JmtmE6NhE6mkBsDKaiigAs1TUiHpRerB4Rk2RpC0i89Gq4wWH2BlfYrb1AdhF6Xux8YyAMTpbOsqbOQ3BQcKtFLwQoV5WJhERs+Mjo/mo9QUABoQxb1zRTEBKAJ6z9biwjSMfwk1kfl7WMfvRs2dUC0plwQRiVLuyEgCLmEDgGvDY54+sGdTN2XpCdpZiAFxuyFy6bBOA9TJ9WVmPJA/A2dLR9dviAHz43DWq+95CK6KS2ox9shL1+OV+6lu+I7b2bgHwM7UK0jAHAFgtaoaVjzI+0kZUZDi9WDeaPpwfb+OKjHmNS8WDK590wyy44oOjYcnsUiQatecogHX21hOW11PZkAdgPa2nGQDOkSGCVg+IpQ/n7VI1m3Bs8ESE5+S4RftkTTzrWxSAEUceJmBD/9ouNfUB2MnV8aYvH4BdlHMoADCbHrRDSM3pVtGLySlyTXZ8ZA2A60m2nsMSHkRD25SWH0p614amARoHVmZtOa5ps6jsh49gITI/L+tsH95MegmKZiTyAoB3j2wheeyrAfDsLcepa4KtKEuFHIoA7IUJhB0N8OI3G1DBrOnkraa0oyyUNS3B5vaYwgSC107y0HXo7DVbzwzEieZT33oJwM/WLkRDHyqje9uJaIC1kiCYPZUoliM97TkVn24ayR6er1NYdvLiNbd2ABh9I0HP0r6NTCW0sfpscgOAc0ZF0Iq3G9MLP6yhhbucO71RmyNCM45fHAzAymgqWhpgJQAjM2G1Qlno6s07VGaIuAZYJGmL1TVCOy84xM74ErOtD8AuSt+LjScKjHGv16PYj/5xbba/dKlBT36zynL/7OtZdD78hTC3LUcvUu/fNkl/xsvl/5b9azgWHoCRyajP5E00dUN8Ku2kXFgqz8e+XEGrBFIBWwXgkQ+XpRmbxTTAAGDE5V3LOS0xiOA/PJwE4Ge/W02LHHyJMgDeevSiLYdPNRBkf9M6PhUxgVjUpwEVymYegKHZ3nYs2Ab44NmrVP/9RZZvBaSo5VPfqs1bDfh5EwhoiBGdpVaRrFRx+HzhsTgFwNVHxdGphBS7Wusm8syC+cn+01elLpDsAeNjTl4s9B9+6//HFvolIe0yu56oCQSrP6h1aRoxI14r6WbhAVjv2F8t5bHWuADA7Srloy8W73Nz6FLfcMzFdVg8ZnZBJQAXHzBb1d8D4dqgAWaKF2QmrF7YPACLJG2xIwwvOMTO+BKzrQ/ALkrfi40n8vDFFBHAveVnS1ybLfLZd/428QAYmsW3pmyR5tehcj6asv6I4VxXD2hMOTLEZ/z7bc0hub1hwxCvgA+CojkyCNt8A04R1UJ0L7Hpi35ooP6ukc2p09erVAEYKY2ZrShCZzUvm5uc0AA/991qR7VIDIDfnrLZVLxrve2iBFsnAVjpSFQwa1q6raIB5h20eOj698xVaviBuwCsNl8egN+cvIkmrztCask69OT6XJ1CNKSNfQ0wHODgCKcsZjXAkP3Bs/ERNBDr9qmaBWUnLxb5BL85AcD9WpSkMbN3uv6U4gFYT+tpBoBzRUUGRSlxayI4FUUYM2aLza6jDCeopQFWAjBOQWtEZ5V8F+DDIFpEkraI9qVWzwsOsTO+xGzrA7CL0vdi44lCC7yX2/5vmWuzFQmVpndxdnwkOh++r7YV81DlApllb9xW5XLTzC3HDefKA3C7ccsCkjQYNg7hCmYjalgFYDMiUANgaAjbVswraZGZragMwEPn0uWE4PNmYQPjQhs4Rf6985SZYerWZQD8+qSNNHW9MycFTgLwwj4NqDCnAVYCcIEsaQkZu5QmEFoAvP/0FWr0oXhiFKXwkCnwtd82qsqUzVsNnHgAtvI8wAXxcTa4TWnd9RQxgdBKg2t2T+bLnIaOnL8ujSdb+tT0ZI2CspMXD8D9pm6hiasPBYzbrAb4zWYlgrSajt0EXEc8AOtBnxkAzp0xMshJ042xo0+YorxYt7AtAP5zw1EanqBtZ5nvzAOwcdIWOzLwgkPsjC8x2/oA7KL0vdh4oi8I2Nh2GH8/C57T0xZ1PNO6Ljs+Ep2Psp9GJXPIsMP/W2+eyF6XIypeA9x+3LKAEF1Oy8fL/qCNjymWTVij6wUA7xzRXApLtuZAYNxWvNzhSIfEFijjO1WmFuVyE5+UwyxsMAB+4fs1tMAFADZKwmFmrZUArKVJEzGBQErwnScu0+/rjtDwtmVo0tojAZ70+bOkoXv3iBDejC9aALzv9BVqbAOAP36sgmyWpJQJm48aLDgBwF1iCtPA1vYBuMqI+XT26q2gJTW7J/NkjJQ/PBB55snq+emzhNi4zPETF+k3dTNNXH044Hpw1jLjwPxabDHXY+higDwAX75xm8oNnae69c04weXNlCZof5q5n8zWfaNJcdkZkbUV1QAjjCbCoDEARmZCmOroyUJtfCIxq83Oi6/vBYfYGV9itvUB2EXpe7HxRIERX6fwwnerwJGjy49rLXfPjo9E56O8EBx8DiQcMdaKzioUmovZyqKvDuOXawbVtzypRGrIUqKKyhLa2YjwlMLAbGVaegCMuL7MWYoBMJ9+FF7yo2btoMGtS9Oz360RujwApcsPayhuh/Ma4K4/r3PMm95pAGYaW6Q+jimWPQCAoYVE6nFRAN576rItv4GPOlag1yfF2+V7DcDQ7A1oZR+AK4+YT+d0APjCtVtCtsmwbT15Kd6UAo63cHwauzA+OQSze9cCYDhrRfefJbTvUUkr8YRwB4IVeQC+dOM2lU+CAPx6k+JSyDW+KAG4aP9ZdOdecBxgAPCfG4/J9tYssY1ZABaJWS24JKrVvOAQO+NLzLY+ALsofS82nijk/PRCdXrq29WuzRZajJd/Wme5f3Z8JDof5YX4I8YK+TPRpsMXDMdStWBm+r1rbaneI+OXB9inGjYO4QowR6lfPLsw0HoFwE9/u5pWHzgXIDlotwDAzFlKDYCtiDoegINjuFrpi7VBKuC/esY4CtZOAjAyIjKNLbKNvRBTWNYyYg7QrqEoAVjLCW7PycvU5GPrjrMfPlqB3pisD8BqsOCEBviletHUv2Up3eUWMYGoNHwenb92O6gftm5Dpm2lH1YcNNxW0PqeuRIPwAj19WjVfPS/hfGOXszsB/9Wsy83C8Cv1C/iiRMZD8AXr9+mCsPUNcBmTCC81gCrAbAynnZ0v5mkwr9Swg6YQIycuUNaRziC1y6ajfQ+BtQ2ig/AhrePaxV8AHZNtN6EHxEFRrsmCkZi0sv6ZNQWv7PjI9H5KPvkjxhL5MxAu05eFrmsnGTh0S+WBx3PC3UQgpW+f64aNSiRI6QAeMfw5oS4vEoAhic2bIAZAGMftSyXm3gNsBUR4+UEe0pkPnOqMBtgJ+MLKwFYy35QxAQC8bBjP4q32UW2sefrFLIFwLtPXqamNgD4g0crSJFV1AqbjxosOAHAL9eLpn4OAHDF4fPogg4Ai5rDwPGNaZKhDYajLmLQorCPPvz7rd83029rA00gcI8UMaEBhvnHN0uNo+DYvS8CAPjabc3Qa2YAmFdk2B2fSPvescXlcHSsvhKAC/ebKZ2cKIsSgCd0qUF1LACwSNIWkblo1fFCEWdnfInZ1gdgF6XvxcbjgVHPU9quhtZITJ8/UUm24zSqq/Y7Oz6yCsA4VmTe2nD2OXQu3uPaqLAXcccvVgTBmVHbUP2dpVYVlSXMEyJTuWsCIQHwd6tptSIsG+yPEQaNOUsxAIY2CVolqwX3AvbBPIcBeHr3GFPH0UbjNwvAAOS5W09Q/RLZCQ5afOEBGH9X2vbiIzEsLIXsjMXaatkA7zpxmZp9Yl0D/P4j5enN3zerioDNW01ziJi581+vL7UT3cPKiyDTWr8W9jXAWvuQjb/HL+tpxmZjh1uEPmMgDUevhyvlpfEJAMz2POagBsDIolh0wGyjrST/jhBr3y8/IFzfakUegPVMQcwAMOzUD58LtFG3Oj6RdmoArIynrbUHEUYTYdCYBhjprd9qXlJ6bmlpw9XGJJK0RWQuWnW84BA740vMtj4Auyh9LzYef3OqHWWy6ellZXJCBHoe3yL9s+Mjqy88XsPCw7DRtdmLTDRmrlF/ofA7S60qKksvABjJOWC/qwRgXHv21uOysxSyWbUqn1t6gdgBYKxD09I5HQVgmEA8U7uQplbTytqbBeDuE9ZLEU74uLLsukaxvgFeKU0A8I7jl6jFp9ZDJ773SHnqawTAKppDJwAYZgBvtyipuyQiJhBaJxFs3RC+D2H8jAqyv11KiGqCdUD0Exbrlu159NH3902S8yJfoBx48mvxEJM4Uv95ZWAkCaPxWfm9ZblcBC3/d8sOUOk8UfSchn2+GQA2o7ywMmZlGzWHQR6A//vvP00HRAAwTCDgn8AKzJBg7mIGgEViVtuZqxccYmd8idnWB2AXpe/FxuMf4nhBIyGEWtHzyHZCBHrHnSL9s+MjUWhT9pk1XWrZWztDRLicRtfo2uxF9vhXK4RS+hr1Fwq/s9SqorJMTACGZhgaYGYrymBA6+jZjHybl8lFc7Y5l/oV9xdCWDmZoUoJwFoONKye3poaATBirIanFNMAP1IlH+GDmgX5NyN3VvfdDuU0Y2vLGmCXAJhp4/TGLQLAfDQSvi8ZgBM+SMzIBwD8UIU89OU/+6VmUE60Lp9H+jeLe2ymP2VdJFb4dU2gGYWd/rTaAoBzZ0xD3xqYW0SEh9HNO/eEhsA7Mws1sFlJDYD5hDJIkKTlgKgGwMh+WL9YdlOZ+HwAtrmINpr7AGxDeEZNvQZgPecvPW2M0TxEftd72Ym0Nxu6S9kn72SSKmUKun1XxWhLZSBwwkqRIgUlJwBmqVVDCYC3DWsmxeVVZqbD32dvPSFrVZ0EYBzRom+nSqgD8Pze9XSd1mB7miplmJAJhBMyEwFgtaNzJzTA3RoUob7N7WuA+YQsagDcbcI6mrXF3B4DALepkIe+cgmARRMB2V1jAPDO45dp/5n4DHdaxQwAI441ErB4VV5tXExOSMKuyQPw3Xv/adpfI448NMCjZ91POoKPmbpFzQGwSMxqO/LwgkPsjC8x2/oA7KL0vdh4PORUKpBJM5nDqHZlacAfW12brd3+EaWibjHxyAXKifBhhnDMiweXSIGDCeo/8dVKodBpIn0mdh2WWjWUAHjrsGaEuLxKAN48tKlk08psRZk2TMv73oxskxoAIy3wnXv3VOOpimiA5/Wup+u0hugDEanCgmws9UynzMhbWfed9uXo7anx2RmVhc3n/NVbVGlEYIpjJwC4e8Mi9GYz+wCMjF6wu9Yav+g9xrcHALcun5u+XhLvqAb/CQAxCpwGEcfZTkFiIMSndbsAgHccv2wIrKEMwL0aFwsIFQiZ8Qllbt+9R8U07K/VABgOjWbTdiNaC9JXu1W84BC3xu52vz4AuyhhLzYe/wBGWK+1BwMTDbDpDXuojJwpzY0pI/D+4GnbLHeN2LV1i2UzFfBd+VI5fvGG9KcUKUjVa1dtcCwJRKdvVtKyvWctjz+UGrLMUqIvZy9MIADAiMu7cn9gGLRNg5vS3O0nZFtRJwEYL2iz2jm9dXRTAzz8r+3088qDNL5zZXrhh+B42k4AMGzjYY+pdDIqlTuKYO/rdBndrpyU2letsPkgMgJi7fLFKwAuMXC25tE8OxkqM3gOXb11N2gKGL/ZjF+sEzgjItIJi9SAY3OYRKC8MWmTUBp3vbUSzYRpd70fBAC+deceFR+o7oCIREp/bDgakHYaMZ1rRmcRig3N5C+StMXOWnnBIXbGl5htfQB2UfpebDwecqoXyqIZyWBIm9K27PmMxIQkBSwjjlFdtd8Ru7Zu0WyWPeytptBk8Nf5m1W0dO8ZK0MPuTYss5QoAMMON01qd6NAbBnalF78cW0QAG8Y1EQKVdZ3Sny0AKYNc0ID7DQIuAnAbK3CUpBqzFERAJ77Wj3dqA0wE0qTOhiA9aLH2NnceqdCbD5nr9ykKopoFl4BsN4ew9F47ybFqfTgOXRNA4DNJjzgARjZDpntLEsJ7hQANyuTk+Zucy78n9YeAABvP3ZJTkCkVS91eBgBJEVKdLZ0hiYVIv2I1unVqGhAqEC0Q0bF6OzppS5u3rlLJQbOUe1ODYBx+latUGZTACyStEV0Pmr1vOAQO+NLzLY+ALsofS82Hg85+PJUatjY9HDEMiIhZ7kbUx7QslSAN6zZawCAY4pmMxXvkr8GjndPXY4PNG+mIDpB2tThpBXs3ExfoVKXZZYKNQB+6cd1QWYmyKYUt+Ok7CzlKACXzy3koS+6bgBgaFH/djC9sgjYYnwi9ca0LyfFPtYqcODDh45SA2wmbraorFBv5MNlaeCf6mZXbD5IDqEM5+YEAIvYANces0BOT6w2L4yx1KA5dP22sxpgJHtoUjqnHKqMB+DXJ22kqeuPmhFzUN3GJXM4mgJcazAA4G3HLtHBhAycWvVMAXD2dLT/tHc2wBXzZ6KNiqRJAOB8mdPSHxuOUOncGanN2KWaADx1w1F6Z/Z9G2A4IFcukDnIrEdvQUWSttjZEF5wiJ3xJWZbH4BdlL4XG4+HnDpFs2oe4w9sVUqOV+jGlBH/8N059x8EZq+B5A0IIq5lb2XUH17uZ67cMqoW9DuO5qGdUB7Dmu4ohBqwwPqiAMw+AkTrW5kqbH1fVgFg2NH9veOUbCvKAFgrBa2Za8POUiRGq2ifoQ7ARvNApJR0EeHCMbKN+jP6Hcf6yPKnBZf4u1sALBIFotGHi3RhS++ZaccEAgDcoER2mrAqPlQZQkgiLjDK679tJECVnVKveHb6Z/dpO1042jaUAVhtoghlNmfrCXp/7i5dOazq31j6WOHfe4jBDqhW2rXrdSQSs9rOgnjBIXbGl5htfQB2UfpebDweWmBDu2SP+jF+vxYlA2yVnJ52n6bF6YN5gTnVzVzju+eqSSBqNZ1y5rSpVFOWGo0BYHb43DVq9Zn6V75R+1D8nQXWFwXaxARgvESgUWWaS2YPWWXEfDmsnVUZ+wAcKDnEykaGONEkMVblLtKOaYCRvKbaqMCEHk5ogJUAvO/0Ffpw3i56ulYhqhmdVRpiq8+WSBpMK8UuAOO0i2V8Q4jKdpXyOQbAeooQK3O128YMABfJno72eagB1gLgbj+vN8wmimfXlPVH6L0590EZipzy+TKZUqiIxKy2swZecIid8SVmWx+AXZS+FxuPh5z6xbPTYo0vf7saWiMxqcVTNGrD/44v5+e+X2OmSUBdPtC8mU42Dm4ihYVq/XnyAWAWSiyUAHjTkKbU9ed1tHxfoKPhin6NaOHO07KzlJMA7LQNcPl8iAMcuiYQRvseH4kZIlOFBABDm/5cnUIUUywbVR+1IGDoTgCwEipqjl5AJy7FO8ky+G43bplm1BwjWaIPqzbA0ADXKJxF1vR+1LECta8cD8C9f9soOVbZKbWLZA26z+z0Z7dt6pRhdOuumA2w3Ws50R4ZFZFwZtfJy7rdrewXD8C8phjO3GXzZvQB2ImF8KAPH4BdFLLXANywRHbNIP1vNitheKRjRxQ9GxWlz//ea7kLZC97/vtg73fRDqHZUgtXZNR+/aAmdOxC8gJgFklBFIARi/fO3f+EgrcjUD3CxpnV0iDaQ9cJwQC87G0A8CnZVpTZQzqhAXY6DFpiAjA+MF+LLW45NTDuA6TjjQoRAGb3JRyJqo8OBODiOdPTvN7OpkLm7wUGwA//b1mQ/afR84L9jj4u3bhN5YfOE20i1wMAVy6Ymf5KMA/58NEK1KFKPAC/9usG+tNmCLNa0VlDKqSjGQ2waWG60MAMAP++7nDAyScAGGEFlY6desMUsVe3M00vOMTO+BKzrQ/ALkrfi43HP9hjS+WguB2nVGdk10TBSEy4iccl5LY3qqv2O5wH1MI/ifaVJlVKVWcVo/ZrB8bS8Qs3NB0djNqH4u/MjlYUgN/rUF6OwmA0HwSqBwDvPXXFqGrA7wDgbr+sC7JRX9K3IS3afZoGJThLOQnATqdCdgOAWbgtkbUCdNlx1syYJpUEwUZOS6YW1mZlHCPXUACwExpgZle55+RlypMpDZUZMlceKQPgtmOX0qYj6pkzjaaFPpCq20zKW9ZnvsxpCBpwlqTFB2AjaXv7OzIq9vhlA+08oa8BxunVD8sPyimtMUqYQGBtzQCwSMxqOxLwgkPsjC8x2/oA7KL0vdh4/IsTnsUIKaVWescWp4/jrNvoGokJR44st71RXbXfkb2sy4/WNcBWtQzQQCF+cNv/LbMy7JBsw8wIRKDKaAJVCmamdVxsaasADFOT7r+sDwLgxW82kBx2BiXEkGYAXHXkfEtOjfx8YkvllCJMOFUAwNnTRzjqYb9/dEsKC0shpNkFdBUfMNvycTLMhDKnS/1AADBCS126fkeys0Xc3WMJMcKxFxgAPzR2KW22A8AqaZxF9lr+LGmoRM4oeW8ijTxST6O8+usG20ks9KIBiYzP6TpWn81Oj0O0PzMAXGvM3wHd4iQTNsDKyCZ61+7RsCj1aVZCdHim63nBIaYHFSINfAB2cSG82Hg85DQvk4vmbFNPy6mW8tHJqb9cL1rObW+lX8RPRJxYqyU8LAXdEcz+xl8DdlywDcRxaHIpDCKdAGDEtISWjMXxRJxOaID3mNQAA4C7TVgfZJuIrEtL9pyWk6jcB+A4KUKAndKoZA5HQ5a5oQFmmQhF1kovLJeInDJEhhMiQRwwCFsl0pdTdXD/1RyjbgKx+t9z1PHLFU5dSu6HAXDrz5fQ1qPWneDU0jiLDBYAXCR7elq0Kz5Sw/uPlKdHq+aX/t1r4gbNyBkifaMO7IuVGRdF27pRL6kBMFKK95wopgFWAjAUORULmANgmA++0dQHYDf2nlGfPgAbScjG714DsJ7Tj1rAbxtTC2qKWIYst72VfpG97KWf1llpKrXRSiBg1OHytxvRyUs3qN245UZVk8zvTgIwkqtsOnJBzpgFAA5PmYJ2nzRnAoGEFwDgFfsDneAQcmjpnjNylkInAVjPKdTKYroBwHtGtaBUKcOENcDlhsylyyqpeUXmkyEinLJliDBMXSvSl1N1cIyshAhmAyzyUWBlHAyA7UaBUEvjLDIeAHDBLOnkxDvvPVKeOlbNT7O2HJfuEbuleuEshI+HUClJzQlOFIDx7qj9TqAGGEmIEAdYGdlEby2Qjvn1JsVdWy4vOMS1wbvcsQ/ALgrYi43HvyT0wj7hmGXsQutOakZiQjpHltrTqK7a73hwWA2BZuV6rA1sUE9fuUntkxEAs7BKTgAEtEkIFH8zIZNTdPZ0BG27UwCMlw0y8A37a7u0JCwmKo4Q7WqA9cICWtkzbgAwS8UtslYAt4rD59GFa7etDF8KgYZEHv+e8S7RgNFA4QRZRwERXgFwy0+X0HaLKaCxFmppnI3mi98BwLkzppEhFTb4HavlF/oIEunfB2ARKWnXmde7Hr3660bD9OBqAPxF58pUpWAWUwDMsg7aG7V2ay84xK2xu92vD8AuStiLjce/ONtWzKNpP4a4mONtOKkZien5OoXp/5b9a1RN83c8OF752b72w+wA/nkzHoA7jE8+GmAWVkkEqozkBY/yDYfP043b8WGMEKczPCzMMESQsl9E20BoIaUGGOl7YQM8atYOxwHY6XiobtgA7xrZnCLCxdJQA7rsfBiEIgDjBKDxh4uDtqHdzJJ6+xqOh7CWggbYyNFJqx+shVoaZ6P7iQEwbMnXH7ogVX+3Qzl6rFoBH4BFhOdBHQAwTFGM9obaxxtCUFYrnDkotJ/esH0A9mBRNS7hA7CLsvcagNtVyqsZQxJe0V8u3u/abBHT87tlByz3j+QNThz/mR3Aoj4NJE3jI184b2todixO1Wde5U4AMGKKbjh0QY6wAQDGkb3Ry0E5F6Q8hme1EoBnv1pXstNlsTSZ9toO6LFrO+0M5AYA7xzRnCJTiQMwH8/W7H5Jlzol5YiKDCkN8IyeMZ7H4K6QPxPBfCFFCrLsEAgAVstiJ7ImBbKklaJxMAe8d9qXo8erOwjAhbLQ6gOhYwKRKmUKun33PxHRhESdWb3qUsvPlhiORQ2AEYGnRnQWUwAMB/VXY4sZXs9qBS84xOrYErudD8AuroAXG4+HnA6V80mBudWKXRtdIzE9W7uQnNveqK7a73YTaVi5JtpAA3Xm8k167KuVVrsIuXbMq9wJAEbGKkSBuH77rjTPojnSSyYQVgAYUSBW7g98Mc/sFSNFLvkkbo/UPwNg2NAhS5idAvtlJ0GgQkIijAU71UMNWhnrjuHNKU1qcQCGucDRC9etXEpqAxvu/SFkAjHp5VquOLpZFpBgQwCwWhY7keYAYKSk3pFgfjGmfTkqnTvKsUg0cFxdc+C8yFA8qZPUbICHPVRG9knQE5AaAMOHAadmytjWev3A/hd2wG4VLzjErbG73a8PwC5K2IuNx0POo1Xy0eR16gBs10bXSExP1ypIP644aFQt5H5HyJtTl2/Sk1+vkscGb/kbt+8mKa0FL1jmVe4EAMOOFgB87ZY9AEa85R4qAPxXjxias+04/W/hvgAArj4qTloXO0UZws1OX2jrhgbYTBpqQFfDDxbZ0uAiCsTZq7fsisKx9hO61KBO39y/9xzr2OWOsBanLt8wpeljQyqYNQfKlJYAACAASURBVC0BClkklcYlczgaWi/UADipaYBFk0Ytfashxby7MGCn4QO+TpFspgD4jSbFqacPwC7fserd+wDsoti9BuDHq+WnX9ccVp2RXRMFIzE9VbMg/bQy6QEw7L0QBeKpb1ffB+CIcMnpKyml7+TXh3mVOwHA9Ypnp3UHztHVBABGkoLwlGGGDiLK/aIFwNO616GZW47LEUSY/bITAFypQCbLqW619nuuqEg5pa7RPSHy+9ZhzSTnNJG1AnQ1/+Qf09p3kXEkVh1kznr6/+7fe4k1DrPXlQD40g1ToMMDcArEI3YpHF3VgplpLRe72+zcnK6f1ABY1GEcAPzQ2GWSMyQrOH2D0kCZ3EVPpkhS1aORrwF2et+J9OcDsIiULNbxGoCfrFGAfll1SHW0dk0UjETQqUYBmqBxbaO2ifk7bFABwM9+t0YeBoDk9t17cuSDxByflWs76VWuFkoMx7VmvefXDIjXACvjk/7RrbbkuPn98nj7cScBGCYLVjN9WZG7lTZbhjalDJGphAG43bhljkO9lXE71ea756rRc9y951S/bvcDAMZzwwzosDEhnThsYu2YsujNz+mTD7uytBqj3e51rbYXdehGBCEkUznPRWWB8gHPTDP7Av4vLcvltjpcw3ZecIjhIEK0gg/ALi6MFxuP1xx1rlmAfl6pDsBumyg8Ub0ATVytfm0XRWy7a9ignrh4IyANMwD4zr17cuQD2xfxuAMnvcoblshOCxMC9rNplModZVoDrAXAU7rWluzW2Ycbc+CrMTqOTl6yZwJRNm+U5UQHXi3ZL11q0CcL9gjFbR3RtoyUPnf5vsBYyl6N1Y3rIHPW899bT4DjxphE+gQA47mhTOIh0hYADJt6u/tb61qhBsBWY7SLyNKNOk9Uz08TV6ufpPLXAwC3Gbs0ICwhHBoblswhDMCRqcIIfgAp4JHpUvGCQ1wauuvd+gDsooi92Hg8AD9TqyD9oGGHCzj+Z/cZOnTumiszFn1ouHJxG53CBvX4xesBSTjgLY8wSczxy0b3idIUTjX4IBE5VjcaoJp9YomcGUyHQVs9oLEUBUIZoH/yK7Vo0prDsu26kwBcMleGZGUugLVyOrud0fq7/bvdFOhuj0+rfwAwnhvKJB4i4wEAX7l5x3aqb61rVS6QSQ6xJjIev06gBPTCiSoBGKH0Lt24I/95dLty0j0q+mE0sFUp6lI32tUl8IJDXJ2Ai537AOyicL3YeDzk6Jk5AIhW7T/rmgf4Y1Xz029rjb+aXRS3pa5hg3rswnXqymVgSps6JeF7nNm9Wuo4ERvhIYyX7JMOOBfFlspBcTsCox4gEsRek6mQV/dvTD0mBgPwby/VlE4O/tx4TJIYA2A74b6Y6K2AeiIu2wN56cRKgGNX2ND+ISW4MhOYSL+4Ny/fuOOaM6Ibtu8i80oudUQ/MrEHEC4Na8nKyIfLUmypnD4AJ5HN4AOwiwvlNQC/EFOYvl2qnowCDnKwv3QrC5ReBAoXRWy7a9igwhYP2klWAMBhKVJIWpqkWEa1K0vDpm93xImvSemcUpgyvlgJpSUBMDTAivikE1+sST+vPCg5wqGwEG5OALAVUE+K652Uxzy+U+WAj8+kNJchbUrLGQzNjLtwtnR08frtAOcpM+2N6voAbCQhZ36XAPjTJQGpyYe3LUN4ZoqeDPgaYGfWwmovPgBblZxAO68B+MW6henrJeoA3LFqPik2pFsA/EiVfPS7Rgg2AVElWpUpXWvRkfPXpdSXrMAuC8ke+C/7RBughQtDCzF42lbJjMNuaVo6J81TADA0WGY92Ff1b0w9VQAYNrBwQGQRNxB3uE+zEvTKT+scjbZgVw5+e3ckgMxZiA/9oJUs6VK7BsAV82eS0pf7xV0JIIsoNMC8omRom9LUrGwuYQAe1Lo0QXHlZvGCQ9wcv5t9+wDsonS92Hi8CYRetjcA6toD50yDi2gygfaV89LU9UddlKY7XcMG9fC5a/T6pE3yBRCjExDM23a5c3V3eh3xcFka4hAANy+Ti+ZsOxEw0PxZ0tDhc+aSMWgBMKI+8LJnF8qdMZKOX7zhjoD8XkNGAsic1XPi/dOXkBlYEh4IMt1t8gHY9RUEALf49J8AUzkAbYuyuYRNYwa3Lk3P+wDs+lppXcAHYBdF7zUAd2tQhMYtik8ooCyidk3KdjUKZwkKXaXWv14aZhdFbLtr2KAePHeN+v6+OaAvpCq9wIW3sX0hlzp4u0VJemf2zoDeES1gyPRtjmiA8TBH5AG+5M2UxnQIp5X9GlPPieuDMlQNaFmKRs3aESQdp+PtuiR+v1ubEkDmLP70xWZ3SbJ5UsuUliSF7MKgF7/ZgJp/siTAWRomDQhpJmobDjOa5+r4GmAXlkeoSx+AhcRkrZLXACwawNvMbGpGZwlKX6vWXtRz1sy1vaj7y4s16NDZa/T21C0Bl8ucNlVAfEcvxmLlGmqpZGGHNtQhAG5VLrdsn8vGZ0U7u6JfI+o1cUMQAENjMmLG9qCpZ88QYTsVshV5+m28lQAyZ/X+7f7pi7dXD42rweeAZVoMjRH5oxCRgJrZX78WJal1hTyElOUiBSYTz/oALCIqV+r4AOyKWOM79RqAkU/8swV7HJ0R8pqv2G8cd/ShCnlo+qZ4T/6kVJCKFXbRA//cGjBsN230nJTP76/Uoke+WBHQJXLZD/vLGQ1wq/K5aebmeAc1VnJkiDCdphgADBtgZYYqvDDGKDTYuE6opex1cs38vu5LAE6PfSY/2AAcFRmeZM2t/L0cKIG3mpekhyr6AJxU9oUPwC6ulNcA/FpsMfokLnEAuHX53DRDAUouitaxrn96oboEwIOnbQvoM1v61K7F6XRs8EQEJ74O4wMBGMdq0Ko64QTXpkIe+kvxYWNFNsvfjtcAKwFYK3JJUjFBcXItH8S+kLWw75RA86MHTQ7+x17yWfE3m5UgnIbGvLtQaFJQVjxTu5BQXauVvOAQq2NL7HY+ALu4Al5sPN4J7o0mxenD+bsdnVHtIlmFMk+pHZU7OhCXOkPWtD83HAvScicVEwhkUuswfnmAdOBYMWLmdvrPgSgQaqYtVmSjBcBay5ohMjzJRuFwaasmy26ROUtpfpQsJ6ozqZxREa5lhXvQZJnY88U7uF3lvMIADHO1p2v5AJxY6+YDsIuS9xqA8fX5/txdjs6oTtGstGyvsQlEy3K5aNaWQGcpRwficWdIh5wU4gAjjnG7cYEADLvakQ4B8MMV88hJKtgSZEyTSopjaqYse7sRvaqiAdbqw7eLNCPdpFsXSVv6/xFof590Z2Nt5FacSq1dyW/ltgRwCtuhcj6q+56YBhgOy0/5AOz2smj27wOwi6L3GoBhf/TunMCIAHanJwrAauGy7F47MdtHhIfRzTv3EnMIQtf+s3sdevh/ywLqwhN59KwdjphAtK+Ul6ZuCAxvlyEiPCD4u8hAkfL2p5UHafHu0yLVKXV4GN1KAvIXmoxfSVMCCNk3SGF//6CJq2DWtHTwrDsp6h80WSb2fOGHg6RQwgD8cFl6qmZBV4ftBYe4OgEXO/cB2EXherHxeBMItZBYdqeHxARL954x7EYtYYJhoxCuEJaCHAFIt6eIVM5tVQAYocWcMIFQi+9sVTtrJlZ0eFgKuuOEEbPbC+D3b0sCOAJW2t/b6jAJNkZmOLcSFCVBcSTpISMS02PV8vsAnERW0QdgFxcqqQIw75RRt1g2WrLHGIDVUua6KFq/6wQJ/NUjhtqMXRogD8TWHT3bGQBWC/WDJCE3bpvXjpuxE0+RghwBeH+jhLYErKYTDu1ZmRudlcyK5q7g1/ZKAojF/0T1AsIAjKydnX0NsFfLE3QdH4BdFL3XAKwVUsrsFHkvf1EAji2Vg+J2nDJ7Kb++TQnM6BlDrT8PBOD+LeOTYzihQMVx3mRFimur5iHJzUzG5tL5zYmoSPZ0tO/01QdaFr4JRPJZ/pfqRVPnGgWp3vtiNsCj2pWlTjV8E4jE2gE+ALso+aQIwNC8ZU0XQWeu3JQkIwrAVjPNuSj+B6JrNQDGh9A7c3Y6okF9rGp++m3t4QBZWgXg5GYm80BsMH+SrkvAB2DXRezZBZ6uVZC6xEQLAzCcQJ+sUcDV8XnBIa5OwMXOfQB2UbhebDzeBhiav9Gz7DnBpQxLQUgCcfpyPADXK56d/hFwXGpYIjst3CXm4OSiyB3p+rMnKkkxa5NCmdWrLrX8bEnAUOEM+d5cZwD4ier5aeLqQAC26qDmm8kkhR3lj9FrCRTIkpYOnfOd4JyWuxuZUY3GiBOzHo2KUv33FxlVlX73AVhITK5V8gHYNdF6nwnOCQCG8xEA+FQCANcvnl3Ic1+0novidqzrA++0Iv7DwrGOXeho9qt1qcWngQDct3l8ODwnnOBgzzZx9aGAkadOGUa37pq3AfbNZFzYAH6XSV4C+bOkocPnrif5eYTaBJBmeOhfwWnW3RwnEgf1aVpcGIDHtC8n2Qy7WbxQxLk5fjf79gHYRel6sfF4UIPzE7z/7RTATeZ0qeTA7KJgK2oqYWdsXrVNSgA857W61PyTQABGPOgP5jkDwDie+2WVMwDcuGQOWrDTtxP3ah/710kaEsiXOQ0dOe8DsNOrhSxrQ6YHZvh0+hrK/nDKhTCUohpgJIJ53Adgt5dFs38fgF0UvdcAjBtv5EybABweRlnSpqYTl25IkmlQIjstSiamDaJLnZQAeF7vetT0439Ep2a6npMaYNMX9xv4EngAJOAnwnBnkRMjxB4UQSPalqUGH4iZQCAT6WPVfA2wOzvAuFcfgI1lZLlGUgRgODjBBOL4xXgATk62vaILmZQAeH7vetTEB2DRpfXr+RIIOQn4AOzOkiDL2qBp3mqAzc7kvQ7lqWO1/GabmarvBYeYGlAIVfYB2MXF8GLj8SYQTmiA06RKSZnTpqJjCQD8IEZ3SEoAHPd6PYr9yE0NcLATXKqUKej23f9cvHP8rn0JPDgSyJMxUn7ePjizdn+mSSHLoA/A7u8DvSv4AOyi/JMiACPLV6Y0Dy4Af/p4RWpbMW+ScYJb8EZ9avzhYtd2sVoUCB+AXRO33/EDKIHcGSPlE7cHcPquTTlJAPAj5aljVV8D7NomMOjYB2AXJe81AA9qXZpGzLDn9ZoOAJw2NR29EO+U8SA5LmVLH0FrB8ZK804qUSD+fqM+NfIB2MW72O/al4C7EsgVFSn7XLh7pQerd2RZG/jn1pCe9PuPlKdHfQBOtDXyAdhF0SdFAM4QEU5RaVLJAJxcQleJpNYtnC0dLezTIEkBMMbbUNDhwspWf7xafvp1TWAcYITKu+NEmjkrA/Lb+BJIZhLIGRUhR90RnVqxHOlpz6krotUfyHrIsjbgj9AG4A8erUBIN+9m8YJD3By/m337AOyidL3YeLymcnDr0jTcpgY4Q2Q4ZUyTSg7Lk1wAGAk+7hpAG5xRlr3dKEkB8OI3GwiH3LGy1dUywfkAbEWSfhtfAuoSsALAlQtkogYlctBH83f7YtWQAJJM9P9jS0jL58NHK1AHH4ATbY18AHZR9EkRgAG/UWnC5cDssaVyUtyOky5KyZuuRaAtR4YIWj0gaZlA/PNmQ+G0m1Yk3bFqPpq09khAUxFZWrmW38aXwIMoATx3WOIh0flXKZiZEKM9OQOw1YQ7TIZIMtFvqg/AXnCI6L4NtXo+ALu4Il5sPF4DPKRNaRpmM/NNprSpKCoylZyaM7mkrxV5mGbPEEFrkhgAL+nbkOq+t9DULoaj47Vbd4XaqAGwiDZdqHO/ki8BXwKE5w5LPS8qjqoJAPxhMtYAW025zmSIJBNvhzgAf9SxArWv7JtAiO57p+v5AOy0RLn+vAZgJ1I/IgQabIAPno3PTd+0dE6atz3pa4Bh2nH5xh3d1U6KALz0rYYU8645AH6lfhH6YvE+oZ2P3PaT1wVqgMNSEPkmwELi8yv5EjCUAJxvz1y5aViPr1CtUGaqWyyZa4DDw+jWHfMp15MSAH/8WAVqV8kHYFOb38HKPgA7KExlV0kRgLOmS02AxQPJDIALZU0rz0lryZNiFAjYLNd5529Tu7hrgyI0fpEPwKaE5kBlOLv8rviYcKBbvwuHJdCnaXH6YJ53trXZ0qemM1dumZrFgwDAkanC6MZt6wCMLGtvTQltE4hPHqtID1fKa2rtzVb2gkPMjilU6vsA7OJKeLHxeBMIJ3Kf42GcITIV/XvmqiSZZmVy0txtSV8DDJu5dQfP6652UgTgFf0aUa0x5gC4W4MiNM4GAItE1HDxtkqyXb8WW4w+iduTZMef2AP36jTKCWdiM7KC0uHsVXMAXL1QFoopli1Z2wAjKdP122KmWmryRpKJvlM2m1kKz+v6AOy5yAMu6AOwi/JPigAMMwCEQtufAMDNy+SiOdtOuCglb7oWeXkmRQBe2a8x1RyzwJQQezQsSmMX7hVqo6a19AFYSHRBld5sVoLen7vLWmO/FbWvlJembjjquiS8BmCknj9nFoALZ6GYoj4A622G9x4pT31/D20AZomX3NzUXnCIm+N3s28fgF2UrhcbD8ffSFqBUDrdGxalwTZzn8MjOX1kOO0/Ha8BTi4ArBbOS7n0SRGAV/VvTDVGmwPgXo2K0md/Jw8ARug6lrTFxVvZka77tShJY2bvdKSvB7ETrwDYCWdiM+sDv4vz126baUI1CmehOskcgJGU6aqgs66a8JBk4k0fgMkLDjG1eUOosg/ALi6GFxvvyPlrNGnNYSmbzMJdp2wDMEA6fUQ47UsA4BZlc9HsrUlfA9y5ZgH6eeUh3dWG+cfagU2kOnYzwXll77l6QGOqPsokADcuRp8tEDuK71A5H01ZH+gEF0oa4HrFs9M/u0+7eBc717UTmRqdG417PUEzuXTvGccvkFwBGJF3LlgA4NpFstHHcd7ZKju+oAYd4j105aa+47JeF0kBgD97ohI9VCGPq6L1gkNcnYCLnfsA7KJwvd54P604QINsaoCRlz5dRDjtTcgy1LJcLpq1JekD8LO1C9H3yw/orjZs8dYNcgaArdjmWtmKCNtWbVScqaZmbFHVANjUxVyujFioi10CYCtgojfd4W3LyB+oUZHhdMkgKonLonOte9cAuHJemrrefRMIJ6LpmBEuYq9fvG5OA1wzOgvVik7eAAxTvMs2ABhZ1vpM3mRmKTyv+/kTlaiND8Cey51d0AdgF0WfFAE4TwIAszSbrcrlpplbjrsoJW+6frFuYfp6yb+eAbAV21wrklg7MJaqjjQHwL1jiwtrjtp7BB1W5o42bgHwlK61qEj29FRx+HyrQwtqxwfmdwsSHRusjY7qFstGS/Y4rwH26mPMawC28jFUKzor1SqSNVk7wYmErtTbpsiy9oYPwL4JhM4m8QHYxoPeqKnnALzyIA36017uc9hUIlGCFgCXzRtFW49eMpp6yP0uEvoLzijrHdIAewXAGG/lEeYg7Y0mxUk0gH6oA3CDEtlp0S7nTSAOvNNK2sN2TWH4G4E/kgXArNh/NuTuEycG5BYAe2VW5DUAWwG92kWyUo3CWYU/ZJ1YV6/7sPJhwI8RSSbWHDhPE1frm76JzKtEzgx08NxVzbBsIgoWteuMfbIStS7vm0CIrIEbdXwAdkOqCX0mRQDOlzkegHefvCLNolX53DRz830NcLm8GWnL0YsuSs2drkUcv5wEYCvOaVZmvmFQE6pkEoDNRCPwAXimlWVRbfP9c9Xo2e/WSL81LpmDFuw85Vjfdjsa3a4c9f/DmZipbtllewXAToSTNLMePgCrS8suALMkE23HLqVNR+y9s7rEFKa+zUtS8YGzVQfLmzeZWfv/PVlZese6WbzmEDfn4nTfPgA7LVGuP6833s8rD9JAmxrg/FnSUNpU4bTr5GVpJq3L56YZyQCARexe4Y29YXBTad52NX9eAfDGwU1MH9ObAmCPQk9ZvQ0blshOC5OIBvjfMS3p7SlbKE3qlJKjqWgoOquyMdNu0+CmFLfjpCNHxm4BsFpWQjNzFK3rOQBbsHWtUzQrVS+UvDXAVmyj+TVmMXZbf77E9qnlS/WiqX/LUprvhVHtytKAP8yfvo7rVJlalvMBWPTedLqeD8BOSzQRAXjCqoOWbkJeBAWzpiUEIN95InkBsMixf1IE4E1DmlKFYfNM7eK+zUvQe3PE4tF65XlvagJc5aQEwMysAsP/aN4u4VB0VmVjph320YIdJ+n1SfadhtwC4I5V89GktYERSczMUbSuVW2eaP/KelaiHcCGvFqhLMnaBMJKeDg1AI79aLHs1G11jV6uF039Wpai2VuOU9cJ64O6sZp1bnynytTCB2Cry2K7XbIA4MmTJ9OECRNo3bp1dO7cOSpSpAh17dqVXn75ZQoLC5OFNGvWLBowYADt2LGD8uXLR6+//jp169YtSIgffPABjR07lk6cOEHlypWj999/nxo0aGBa2F5rgJ0AYKQMzpg2NW06fEGaLzxU/9p0TJ57UjWBENF6wut/o0Ma4NX9G1N1k/F5TW8wIto8tCmVH2oOgN9uUZLeEYxH265SXvrDg+QDVuaONo1K5qC/XTAlcMMGmAfgT+J2h1RWOOwjAHDv30IXgEVieVvdR3w7rwHYSrxb2FlXLZi8AdhKghB+HVmSiZh3/6Yj56/b2hov14+mfi1K0alLN1Sf61Yd7nwAtrUsthsnCwCuWbMmFSxYkNq1a0c5c+akhQsX0pgxY+i1116T4BVlxYoVVK9ePXr66aepc+fOtGzZMhoyZAh9+eWX1KVLF1mQgN/+/fvT6NGjqXLlyvT111/Tn3/+SatXr5Zg2EzxGoB/WXXIth1f4WzpqHSeKNnuFzEKpycDAH6reUl6d45+EgIcuUEThmLXBMIrAN4ytCmVMwnA/VuWpNGzxBIy+ADsnA0wD8CIw/zR/NCJ4ZoUAPjxavnp1zWHzTyCLdX1GoDhc3HNZMKHBwGAraSIVgNghIk8ffmmpb3AGr1SvwhBcaAFwIDtV3/daPoaX3SuTM3L+iYQpgXnUINkAcCnT5+m7NmzB4gE2t3x48fThQsXKCIiglq0aCFph1etWiXXe+mll2jGjBlSmBBoim/evCkBNP7+3nvvSfXu3r0rgW/58uXp119/NSX2pAjA0dnTUbMyuWj8on3SXNtWzEPTNjqnAf6icxV65ed1puToRGUR6HMUgC0kqLAyz23DmlGZIXNNNR3QshSNmrVDqM3DFfPQn9z6CzXysJJVDfCItmV0Y2a7rQEe+/ce+mBe6AAwPqQW7DhFr/1m/iWuXG63QtN5BcBGe8Pp7Q2Ts+u375rq1k0AVio9TA3Mwcp2AZglmXBCA4woQlCinLp8QzXxEJzZuv8SbBphJA68D5uXzWVUzdbvXnOIrcF63DhZALCazH766SdJ23vs2DHKkiULRUVF0TvvvEO9e/eWqy9evFgybVi7di1VqVJF0hw3atSI1q9fT5UqVZLrDRs2jD788EO6ePEipUAaLMHi9cZDuJd+U+15chfJno46Vs0vp2x1GoABFusPnaf245YLStGZagNblaKRM/WhD17Hm4c2ky5oWwPsEQBvH96MSg82B8AismBSTwoAvObfc6YD5k/tVlt3D7oNwP9buJfenytmh+3MHaDfy9ZhzSQTCCtaLK8A+Inq+Wniavc1wF4DcGSqMM3wWlqrBjvrKgUyu2IDbOb54ObeRGbOM1duWb4ESzKx8fAF6vjFCrp1957lvro1KCJFgdAC4C+fqkIv/2ResYN2UDi5WbzmEDfn4nTfyRaAocWdMmUKnTp1inbt2kVlypSh2bNnU/PmzWUZQnOcI0cOAizDLGLcuHHUvXt3unbtGqVJk0auBxvjjh070uHDhyXbYdHi9cZzAoCL5khPsLVjGkIlANm1AQZY7Dt9hRp/uFhUjI7UE0lD6yQAW8nQZmWiO4Y3p1KD55hqKiKLpALAcIJ7o2kJav35UlMySGwAxgmLkUmOl2nIfQC+v31GPFxWNZ56bKkctHzfWdPmCkYbMyI8jG7eMQdn0LJXdgmAB7cuTcNnbDcatuu/Z0sfQWeuWDdd4GPsXr911/Rzkp9g94ZF6M1m2gD87TNV6YUf1pqWiQ/ApkXmaINkCcDQ6NaqVUuy8R04cKBk7xsTEyPZAcNemJU7d+5QqlSp6NNPP6VevXrRqFGjaMSIEXTjxo0AIcfFxVGTJk1o06ZNkimEVrl06RLhf6wcP36cqlevbhqcra7wr6sP0ds2NcDFc6aXNMBMW6q0AbUDwAixtqRvI9p76jLFfvSP1WlaaicS3B7xOLc4pAH2CoB3jmhOJQeZA+AhbUrTsL/EXnDKEwBLwnexEQD4u+eq0+cL9ggn98BwpnStTR3Ga59CuK0B/uqffYZ22EaQ7qRYYUqDMGihrQEuoJvUwCnbXW0Azkkr9p2hqybtdY3WKXV4GN2yAMCVCmRyxZFS5FlpNCcnfs+eIcKW7S4PwJCvVgxfkbH2aFiU+jQrIY1HLfU8H+NbpD9W56unqlBTXwNsRmSO1k12AIzIDTVq1JA0tYsWLZIAlwHwypUrpd+UAPzZZ59Rz549JQAeOXIkXb8e6DE6f/58atq0KW3evFnXEW7o0KEEcwllMas5trrCTgAwMt48Vi2/rAFwCoCfrV2IXogpTPmzpKVdJy5Ts0+8BWCRY00nAVg0RbFozNHqhbLQ6gPngrbGrpHNqcRAcwAsek1cLNQBGJngvn+uOpk1KUhsAP5myX5Dk5w/u9ehh/+3zOrjwFQ7mNLE7ThFvSZuMNVOrbJbNsBP1ihAcPTVKvho6Td1s6aZhBbYKvvTA+Dl+86Y0gDjo//wOXsRCNTmi31fMb87AOzUh4TdjZQzKoJOXrKuAeaTTNy99x8V6T/L8pB6NioqnTRpAfCELjWo0zf3/YtEL/T101WpSemcotUt1fP6JNrSIBOpUbICYNjowqYXGtyllqRviwAAIABJREFUS5dS1qxZJbFu377dlAkEADgyMlJeElETiMTWAP+25hC9NcWeDXDJXBkIziZDEzSEyjiwVjXA+0e3pLCwePvpHccvUYtPl3i65Uc+XNYwSUhiADBe2iL2xtN71KGHxgbD0O6RLUxrNsy84ELFIUZrszAA/mLxPuHQbk1L5ySENeowfoUuTOFHkbUR3ch8FIhvl/5LIwyOmf/qEUNtxpoz7RAdi7IeABhOcD1DGIA71ShAE2wA8NK3GlLMuwsNRaT1rIgtlZOW7T1jymENWb7Spw6n39Y6a7uMk4+K+e3ZAMOd5b//gsUh8qw0FKIDFXJFRdKJS4GnsWa65ZNM/Pfff1S4n3UARibR15uWkEwyqo6MCxrGry/VpMe/WmlmeFLdb56uSrE+AJuWm1MNkg0AA3qbNWtGe/bskUwdEBaNFUR3eBCc4CatOUx9p2y2tTdK5Y6SAHjI9G1SP8pUuFYBGFmwmAPhtmMXqdVn8S92rYewrUmoNB7Tvpyhg6CTALxuYCxVUXlQKocmCsBaMLR3VAsqOkA9PaeWDEW04axtqAMwtI0/PF+dRDSq7IVTp2g2wh585IvEA+Dvlv1raIYyo2eMadtmq/cNbMlhAhHKANy5ZgH6eaW+Bhie+Hzqdl4eqwUdU7UAEJq6JXtOm3JYQyZNpHBmKbCtro+yHaKflM+X0ZYJhNaz18m02HbmaxeAlTF27XzM9mpcjF5vUlwTgKd0raX7Qa0lB9gONy7la4Dt7BM7bZMFAMOWt3379vTPP/9I/1Oz00UYNIREAxyz8sorr9D06dODwqDh74gYgYIwaOgPodBCPQyaEwBcOncUwdt60LR4AO5QOR9NWX8/+5JVAOa1XztPXKLmn8RrgK1kQUK7snmjTKW3FMnUkyEinLYMcyYKhNMArAVD+0a3NH20Z0bDo0yEYudh40ZbBsD/t/RfIccddhKx9sC5RAXgH5YfkD8yteQy+9W6np2UwJYcANzjl9A1gRABYD3IEU0brpXWFgD8z+7TphzWcP90qJw3JAEYB3L3VDTA77QvZ9uXxIl7PXfGSDp+0RkNMMZjB4Bfiy1Gr8UWp7NXbqoqNqZ1r0NtLZgr+QDsxE6x3keyAGBkfPvqq6+k2L1169YNkEbp0qUl7S9LhPHss89Sp06dJLvgwYMHaybCQCINJML45ptvaOrUqUkiEcaktYep7+/2NMAAy8erFZDNBdwAYBxHPf1/q+nCtdsE4DZ7PIiwQcjYZsb5671HyhvKBjAOb3i7D0u0Xz+oCVUeMd/wzhTVAGsBMIAu2qRtmxkNT6gDMMJB/fh8dfpxxQEanPDRpid0dhKx5sA5etQBDbCZEwz+I/CnlQdVIw3wY5/Xux41/dgbW3kAMEwgrMQyVcrbLRvgp2oWJMhNqxjdS7i3ywrEzNYCYJjOLNp12lQ4LZyg4BTNaQ1w45I5qJxNDXDKsBQE21hlea9DedsniVprhJjHyPB29IKxXXSejJF0zAYA8zbAdp/pvWOL06uxxTQB2Oppzf89W5UalfQ1wIYvSpcqJAsALlSoEB08qP5gRGxflsYYqZCR5Y1PhYywZ3wBnLFUyCdPnpQ0vwDrhg0bml4Cr43PnQBgaHifqF5AziiH47vf1zmrAWaChKwBLXovNTWhI0f8qv6xpmxfRVJVhjIAz+wVI5uN8DIB0Jm1bRMxB2HXwBHujM3HTe99rxowABZNA84gdPW/56jjl/ZNINwE4LjX63kWLQUAjJTS3SaYD+avXGvYZQMUnS5P1ypIP66wDsCiEVP0AHjhrlN0+66K2lRjsnAihSOx0wCMkGxl89ozgQgPS0F3VAD4/UfK05s2FSlaa18mTxQBFkWeWXkzpRECZa1rsTjA7Hc7GmCYP8AMQksDbPW0xgdgp58S5vpLFgBsbsre1fYagCevPWz7wQW7sierF5CPwB6tko8muwTAWIlBf271BIA/eayiYZardKlT0rbh8XGi7Tws0X7DoCZUyUENsBYAG2m91Ha7iDlIUgFgZMT66YUaJOoAygB41f6z9JiO04poGDStY2Q1ufMa4K//2W+YjW/BG/U9i5eNaCLQADsBwK82LkafLtjj+IPWLgCLmgvhhOSPDUdozYHzAXNoVianJCM1aNSaLOKoP+wKAOeUzMA+ibMuZy0NsIiywOriAoBn9qor9Hy1C8AsE5wTAPxGk+LUs3ExOnf1lurJ3vze9aiJhdMaH4Ct7iRn2vkA7IwcVXtJigBcIX8m6lS9gHwEFooAjCO0lf0am9IAi+RqTywAbvnpEtp+/H78aLXNNKtXXWr5WXDkDCsAbOaIM9Q1wAyAcUrRZ/Imw7vZaQDWgggjAP54/m5DSFzUpwE1+GCR4ZycqIBoIsgE19UBDTDAHX2NnrXTiaHJfTxTqyD9YEMDLHqvAIBh76uM9woTiAU7T6maDWhNFNpfaIGd1wDnJMCknQ8Nrb0roiywurBmADhf5jR05LyxqYTWWPDMb1sxr/yzHaVGn6bFqUcjbQD++4361MhCciffBtjqTnKmnQ/AzsgxJABYFAL0pozYkoi3yWyJO1bNR5PWumMCYVUDjBzxK/s3pmImoh8gKLqRg0/a1Clpu0kN8JvNSqimtDWjAYZWAQ5IV27c0XTk0jpiE32p82tu5ogTYZy0vOpdvHWEu2YA/OeGo4YafnTKAHjl/rO6YYtENcBWAXjUzO309ZJ/dee5+M0GVP997wD4750n6ZWf7ZtAAAais6cX0vIJLzQRIZb498sPaDYxuheMfmcdw0QIZmBKYAIA4z5VcxzTGhTCSLapmIee+26NVKVqwcy09mCgZtmMDFhdADr8J+wAsJYJhIiywMqY0QZa6xk9xTTAdmMoOwnAeM53b1iUzl+9pXqyZ/Ve9cOgWd1JzrTzAdgZOSYbAK5cAABcUNamIS0y76TmRBQIXlhWTCCsADCAtOqoOF3tjRUA1nrw6Xmcs6M9hJt7p8P9zIJ6HzBzXqsrR87g5ad8qUOLbZSpyswRZ1IB4L82HRMK4cXAFqlRyw+bq2nPKQrAWhCh9kDgTSD6/7FFN6kD2lt9qVp5pO0ZBQ3wKXrl53W6zX97qSZ9HLebVu4PTsrCGi7s04AKZ0uX7AAYJhDzt5sE4Mp5CY6kDICVJ2pW1gptAOOl89gzgUiVMoXq/oftrBPh8NTmZkYDXDBrWjp49ppVERE02TA/YcWOBtgIgJf0bUh13zOOMa2cjA/AlpfXkYY+ADsiRvVOvDaBmLLuCL0hcAysN+UqBTMTAs6/Pin+OBmQ9uua+0HcQwGAs6WPN4EwE/8WL/hrt+7S1Zt3qPY7f6uKAB7KO0aYswH+582GVO/9wAcfUnjCJqzicPUoEHiY4sEML2cWGxkD0gPgua/VU82epwRghHK7fPOO7q7+qGMFeX2Ntn+rcrlp5pbQdYJjGuDZW44LHd/zEHr5xm0JQsfMDj6qFwVgLYgwAuBXf91A0zYe0xW/2t4yWi+rvyOeNDLBGQEw5PL892skhzmtkpwBeN72k6rJI7RkgSg6rSvklgFYqVCwul6A8ZK57GmAtfYuoic4EQ3ELgAXypqWDtgA4I8fq0DtKuVzBID7Ni9B3Rpoa4BX9GtEtcaov1f01tgHYKt3gDPtfAB2Ro6qvSRFAMYRXeeaBeXj5NAE4Aha2a+RKQDGCz48ZRhduXlHMxQSwqvtHNFCWktRbYHyyx+aL2hQ8mdOSxWGz1PdF8yeTPmjFgAj0QNiYqqFxFICcFRkOF26oQ/AeDH0/s3YXhbjC3UAjimajX7uUoPmbjtBL/+kr73EfHgAxn9rpQ8XBeDUKcOEw2Lx1x4zawd9+c/+kAJg2LcayRBzeO671bSQi/KA9Om7Tl6W5wLb5UIOaYAHtiolp4x+rk4h+m6Z+yYQiIP7uIoJBKBz7raTpt4YAOA2FXLLNsDK56mpzrjKTgCw1t79onNlR0xh1OZmxgTCLgDjpKtDFWcA+K3mJalrgyJ04dotVcXG6v6NqfroBaaX00+FbFpkjjbwAdhRcQZ25jUAT11/RFizpzXt6oWyUKeaBejVXzdKVZAUY+LqUNMAqwPw4NalNW1o4QEOe01ogMtoxAK1C8AIifXvmFaS3C5eu60JwMyjWLkGalE8kGMemcv2nLwsexlXyJeRckZFUtVCmemlekWocL+ZslYqU9pUUnxlvWLGxq9luVw0a8sJF+8Se10zAIbT1Qs/rDXsTAnAWtEjhAE4PIxu3bknXRfanC4/ao+Bv/bF67fpjUkbqXaRbJp71ksNMO4PyPAlg48IzOGZ/1tNi3ffD3OGvyG+Lj4uUWC6UTCrMyYQPAA/X6cw/d8ybbtpIxtfo9/Z5tECYJgdQAOsV7Afl+49I1dRaoBhWzxxtXY2O8MNnFCheZlcVCJXBls2wFoA/OVTVQw/hETHqaxnBoChTPj3zFWrl6IPHq0gZeFjRVSpoXbBt1uUpFfq6wCwYJZBZd9fPVWFmpbJZXmOIg295hCRMYVKHR+AXVwJrzceQveIava0pl29cBZJA9xrYnxGKOUD22kTiMHTturG9lQbJ0wMYAJRRJEAYmq32tR+3HLVqbEMYHoAHBEeRrtGWtcAA7ABEigAnArD1DXAVgB476nLckzYGoWz0G8v15LnyQMwYiSff4AAuE7RrDShS01atOuUkKe9EoC1sieKAjD2zM0EAIY250VBAOY3qdaL2apdoZVHGu4POHhZBeByQ+bKpjdJHYARJvCxaupOcEYAjCyLeTOnkU0elAAM87IJq+wDcIuyuah4TncA2GgfW9lfrI0eACsTqERnS0f7bQAwnH0frZrfEQDu16IkvVy/iKZiY82A2KCoISJy8gFYREru1fEB2D3ZSimW8+fPT4cPH6Z8+e5/ibp1SScAGHD1VK2CcsQEMwAcnT0d7T99laCdmLMtUGuoBA8mA1EA5h0itAD4j261qZ0GALMMYHomEHYBGDZ1e0YZAzALqq7cB2omEL90qUG1i2YjUQCGg+DZq7d0txjiY7IPHKO9OK5TZUdiwxpdx+rvDICX7jlDnb9dZdiN0wCMU4Mbt+M1wHiZ6QGk1j0QKgAMEwg9gMccMQdkcURKYFbwNx6AobkukDWtsBmR3qLxGuAXYgrTt0v1NcCtP1+imSJdVANsB4ARQi13pkgZgKGBhCMpc4IzymbHZGFkWoOTmWI5bAIwd3rBrwFCc4mcphjebCoVTAFwwvvEynXQBtk/OzoEwANalqIX60VrAvDagbFUdWSc6aH6AGxaZI428AHYUXEGduY1AIuGgtKbcq3orBIAs4D4CIkGRyFW9DTAy95uROsPnqdGJXPQheu3qQ7nbGYHgDGmSgUy0bhF+6Rh5MgQQStUNMB/dq9DD2vkY2cADMenckPVNbOpw8MI8VBRRI/Llr7VkGLejXeC49tfunGbymtch+WVV66DmgnEfQC+QrEfLZaaKDXA0f1myqGZsqWPoDNXburualEAhqkEUrmKZG1y8TbS7bp2kaz0y4s1acW+s/TE1ysNhxEEwBrpw0U1wDwAGx0dhzIA4/5AFAg9Ew4GwE99u4qW7Ll/zC8CwFbjSQ9qXZpGzNgurWuXmML0jQEArzt4jjqMV8/wJwrAiJPdsVp+1TBoRhpghFDLlTEQgDF3FgfYKJaxKAA7ogHWAODvnqsmA7vhDWWyghkALpI9He07bd0Egq0jG6LoM11tSvgQ61I3WvNkb93AWKriA7DJ3ZD41X0AdnENQgGARWxCeREAKJBxicUDVR7Z6QHw1mHNCOmEWan33kI6dC4+jI2TAJwzKoJWvN2YohUmENO616G2GgDMrq9nmmAFgAH9DPR5G2I9AEamrN5NigftPFUAfrGGZCe67/QVOSuYHgDj4+DUZWcAGKljI1OlFP4YcPFW0uyaAbBRZjfWgXIfamVPtALAX3SuohtFIZQBGGMTsaNGvc7frAqwc1UCMEw38mcJ1AAjCsuAP7YExBQX2S9mAXjbsYuqKcPZM0gEguwAMOyHAcAMeJUaYKNYxqIADHvkUjbjAON5x+zX+bWA0y3svO2WHg2L0tiFewO6MQPARXOkp72nrlgeBtPkewHA6wc1Uc0QZzR4fDQ3822AjcTk2u8+ALsmWvLcBGLaxqOy8xqblohNKC8COHFAA8y8wTvXLEA/rxTTAG8b1ozScQBc972/6fC5+Ew+dgAYkAMN8P8WxmuAbQGwjnOaXQDm4wjraZqRUx5mEMqiZgIx8cWaVKtIVtp/+oqcaQh22pM4G2DYQt9NiM4P2Zy85AwAM8dBEWhw8TbS7ZoB8Op/z1HHL9U1f3wHTmuAETrv+u270iWMvOftALBIvGGREHhawjQDwJ2+WUnL9p6VuxIBYNT5eeVBGvjnVlNbhQfgF+sW1k0egmtsP3ZJNWOiVwAM6IKDqhYAGznyMeEYhdeLLWU/E5wWAP/0QnV66lv7AIxTBeXpkZcAzBKaOAHA2IcwwdFSoOjFfdfb8Phobl7Wd4Iz9VBwsLIPwA4KU9mV1xpgJwAYcVWfrlVItgW0A8DQjB69YB+AYQJRueB9AM4VFUnL324UpAGe3qMOPTR2meqKMvjQc4KzAsAYB4srDO03tOAougDcqCi93rRE0DjVtJEMgOEN3TAhLS4idUx65b4THA/ACJd2/OIN3V0tGuiemY3oATAC+28+cjEgDJaLt1RQ13YB+Ptl/9LQv+KP2NVA2Qj+8dGD+NIo4ztV1o1FbBaAefMa2L2fNtDsw5Gt+ug4OnNF3wZcbX0wNmSCe/57/UgaqGcVgCesOkgD/jAHwHxkl5fqRdNXOqHjMLYdxy9Ri0+DU4abAuAE21G1THBGJhCIPIC1YhpUpQbYyI5ZFIBhZobTOJFMcFO61lI1C9ECYESe6fSNsT290X2O9Sg9eI58f6C+GQC26wQHe2yY8DkBwNiHz8cUJq2TvU2Dm2pG/fEB2GinJN7vPgC7KPukCsDP1Cok2wIqnTb0TCC2D29GaVPfN4FwEoCRoIMdpwHyAJ5K7cJfPWKozdilugAsgcqiffTunODkB3A82T3KnA0wD8DQwG1JAGA9ZzscDfZpJgbAv75Uk2pGZ6WDZ6/KaXH1AJhlmdPb1iJpoRkw4P/1IBAvOYBZtVHmHUCcuPXwcTTxpZpk1QTi07g9UmYzMwDMx1HmM+8ZOQzaAWAR2270X3XkfMsAvHDnKXru+/iUvVoF13jy65W0fJ+2Bhjgni9zoAkE2mkBsNL7n782D8Av14vWjZ1sFoCRThjPkh9XHAyYLnOesgLAsJvPlDa1JgAbabFFAbhe8eyEtPWfLdhjeBtN6VqbOowPjo7DRzDhO/nlxRr05NfOAHCZwXMCMlOaAWC7cYBHtStLnWoUdASAh7QpTc/VKayp2Ng0pKlm1B8fgA23aKJV8AHYRdGHAgBnSZeazhlEBeBFgJfRM7ULypqg5AjAmO/Q6dvo++WBQfX5KA78y29om9KqWkL0w2cAQhKKzUPjNcB6ANy9YRF6s1nJoJ2npgFG6tka0Vnp8LlrcqpNJCv5vWttuX3R/rPoToIJRL7MaejI+Xitu1YxC8CbDl/Qta0OBQBeuf8sPf6VeSc4OFipRRbQsgFmL0K2P6D1Z/FvjTJo2QPg1NIL+P25u3Th1BYA7zpl6PyEOTz+1YqAVMhKEwgtAIYzLVJAK0uDEtlpEZdYg/8d8h6WoKEXAeCdJy6ppgxHn0onOPy3WiIUFj7LCgDj3oqKTCVFykBRaoBfrh9NXy7WT4CCdkYmEDBVq1wwsysAjI9ukXvJ6NUJ+fLxoVHfDADbTYWMkHQI6cmK0WmO3nzwDni2TmG6cfsulRw0J6jq5qFNNZ2e9QG4MjUvm9tIlLZ+95pDbA3W48Y+ALsocK833vRNx4LCW5kFYLyMnqldSH4RmjGB2DG8OaVJnVKWaO0xC+hYwnG81st/yLSt9INCA6NcEmj5eA0wUgjD+UypAZ7RM4Zaf26sAUb/w/7aFpRVSguAYduslTwD8YhrjonPAASHw42Dm0r/1jO16NagCPVtbg6AYUrCnO0gC2h1WOEBuECWtLLjodbW/v65aqZj5mq9PLCuiDphJQSQE7dezegs9OtLtcgqAPeZvElKQa0sWgDMXoRMHrzdrdGHhVkA5h0sEd4OHz3MDEZNdnY1wAhtxsBNa21wjce+XEGr/j0nV1ECMMaNkwh+z6COFgDjOF8rtXIAABvAI65hFoDVEqHYAWDYgeMUjMkRJkItuTBoyCaGEyijYgTAMP2pViiLoAmEugaYj2DCjwcf3Y8JfEwazUG5L8wCcP4saWQfEqNrqf0+4uGyBAUOK3YAeNhDZaT3InwtlPHn0f+WoU01owvpjR1mUy3K+QBsZX2daOMDsBNS1OjDawD+a9Mx6pmQwIINSS8uLMJxfRIXeISGlxFudGbDZgeAa41ZINuj2gVgZD37/O94j2IA8PJ+jYOO5u0CMByN9iYksuAflkijXHTAbNVV5gEYHxvwBnYSgOHsBqe34xevy7nm4RD4R7c6qgBsdGyIgO4wY3lSwMaPXzM9AD575aalEEBO3Hp2AbjHL+tpxubjjgCwkW21HQCGM+u07jFU7/34kHtaAFxlxHzDONBabdccOEePfqHvSKgJwEPn0uWEFNxaAIwMaP2mBmuAG5fMQYhBrFZ4AEYmri8Wa8MjxrbrxGVq9sk/mvJRQrnTAIwkEjAt0AJgfPyycI56+98IgBEJBidDIiYQWgmCtAAYzxwRh1Kj+1cCYG5fmAVgEXMuvTEMb1tG8mdxAoD5vtSehXpKEh+AjXZK4v3uA7CLsg91AFYLG4aX0bN1CslewGbCoLGwWUykNUcvoBOX4h2ynARgPBjxklU+iMwA8PC/tgelVdUCYOXRKb9lVvVvTDUScsBnS5+a1g7UB2CYSfzdpwHBplNZ1EwgGACfvHRDvk6F/JkIa8dKsQGz6Pbd/6T/NHIcQeB1HNsbAXCzMjnpy6eqGr48IJvEBGAWEm7dwfOqdo5KGSv3IcIstfl8KSGhRtyO+xCmpQFmmiBZAxwZLoOfMr6yEuzsADBOF7C/WcxpLYitPGK+KZMn1g/GtvXoRc0TFL4e4AhRN/i/8YkwYBefR0UDrAXAsaVyBMienxtvfuQGAKtlAmQpdK2YQHz3bDUKT5lCfn5CA/xwpbyyU5ma0kFtLY0AGGZQSJDjNAAj9u5fPWOo9OC5tt+M2FPlh86lSwkfRl4DMLtXnQDgEW3L0FMJMK0GwPB/sSIzXwNse5vZ6sAHYFvi02/sNQDP2HxMzuDGRgYo0/IKVwNgvIyerV1YzqplJhGGEoBrjI6TQ3LZBeBqhTLTZwkaYC0AntkrRjcGKL9aaraffCpjpaZISwPKAzC8v5ESE+XarTuqD0SlmQg/JjUAnvxKLemo89TlG1R9VLypRYV8GWlajxh1AFZkT9o1sjm9OnGjnJlPBIARy/SxavkpRYoUQgAMG3OAV2IUBsD37v1HDT5YpGr+wdvpqu1DmKsgmgNvUqMFwEwTxPYDH3VDmUAAHxFzt50MAEU1GWntLd7BMmOaVDTntbryKYAbAMzHmtZaS8jl0S+W05oD5zUBGHbxuTMGm0Co2duiEzijzd9+X05aAGxkPoCx7T55mZp+LK4BdhqAf3y+OuG2YWHEAMDvdCgv2U3/9x9Rv5YlNRN18PM2AuDKBTJRTLHsjgIwYhQPaFWKUqUMk0zp8EFkJxUx1gPp4BE6jBUzNsAiEW3CUpCcBEi5Z5m5kiMAzJlTqN2veK6XGhxsG2z0TPQB2EhC7v7uA7CL8g0VAEZqXDx8lUUNgPEyeq52IVlD+ET1/DRx9WG5qV4UCCUAVx8VJydlsAPAsHeDxsMIgGf1qqsbA5Sf/8gZ24OySlkB4NX9G1P1BA0wYvCu6q8PwFpywNjUAPj3V2pR1UJZAuxssQbQ0rDCa4D54PFsPq9P2khT1x+VquOINl1ESl0vb2gykQGOL3omEOev3qJKiQTAfExk3k6aH3uZPFG07dgl6U968ld+9KC+ct5KAMYHY9Z0EXT11h367PFKAaH5kAJ3JmdeYVYDzAMwTg7iXq8v7zU3APjYhetySD89AH5k/HJae9A5AEZSB63wYrwG2Mh8IBQAGJkb4Y/K0nIDgN9/tAL9l/AARshArWQ9WgAMHwj4VuCEiskJp0BwWBbRAGuliFeaQCDMXP+WpeRhYMx2skBiPSoOn0cXrlkDYJGY5njGsRjoyj3LHFadAGA+ooTasxDvPjXnOCO8QOSYlr4NsJGYXPvdB2DXROt9Igy8bLv/sj5gRjhqP3v1pjAA42UEb3OWVvbxavnp1zXWABihsVjsUjsAjBdAtcJZ5Ic9Ih0sfSvYBMIuAEObsH9MK0l+ohrg1QMay5pZaCyQohnl+q27qhoBqwDMa1mZFoUtdPEBs+nW3XvSfxbPmZ52n4zPnsRMOngA/ubpqpTWB2DVu14EgNlRKKsLAP7mmWpyf3wfbSvmoWkbj8m/2QHgDJHhtLBPA11nQ/RvxwTiwrVbVHG4viYf10BILZicsKJ0doJdPLKhidjboo/mZXLJJxTKhcEx9pDp26Q/iwDwnpOXqYkZDbBKKmw7JhBwIIM5EgPgjlXz0XuPVJCnteXIRc1QjVoADNBFdrZDZ6/JNuA4BapfIoctAOaTuODaCNE2oFXpgCWw4ziGfVFp+Dw6bxGARWJfI3Qle/Yp9w6L3esEAPMxhZ0EYESOwYeym8VrRZybc3G6bx+AnZYo15/XG08LgM9dval6TPRn9zr0sCJ1MI5tka2IeQE/VjU//bZWDIBx3B4Rfj8KBCIDIEIAipMADO/gJX2DAXj2q3V1g+DzSz1q5vagrFI4Ahz6UBmpmhUAZqYZTgIwgthXKZiFeDiBRnNmr7rydHgALpkrA+08cVn6jcU1fmPSJpqBMvduAAAgAElEQVSyPj7SgRsALAJObt1mfExkLzTAZgC4feW8suZd7x7Qggw+xB6iTSzu21DX1MQuAGuFeOLXDtdoP24ZrT90IRCAOWcnswDcomwumr31hOoW4QEY4QNZNki1yhibkwA8eNrWgBjBeppqNh5EZ8HHrxYAi9hZoy/eBAKReb5/rnpAKEScAjUskV0+FQt4trUrG5BwREsDrATgLjGFaWBrZwFY+UFmxgRCz4GbzVcrljF+Z9nbWN367y+kg2evWXoU8Vnl1O7X3SNbUPGB6o7S7IJqNuw+AFtaDsca+QDsmCiDO/IagGdtOU7dJgRrgM0AMF5GyHjDvMGhwZi09n6YKD0TiGAAvh+U3y4A46ibZT3SAmDYSDb/JDgLFGwzG5bIEbBAPADHvV6PNhy6QG0q5KHIVPEArwTgvr9vCpAD6ww2vywJBNNM4zctmNDTAE9S0UbhhYqwZ3wKzlK5owiwzwoPwPgN2bBQ8HLYNbIF8aG+vn2mqhSmiWn41bY/ohlAFnzRM4G4qJNe2sXbS+qaB+Aj56+pOomVzh1F2xNkYtcEgoVWYvKAo9u3z6prgJUfj2Y1wEoAxqlHheHzNEWqpnETlT/aihx5o167ccuk+4UVpQYYdvFIB6y8h9TsbdFHy3K5aNYWdQCGycngafEaYCSQYclw1OaFcew9dZliPxK3AVYzO2IaYNzDyFzHPh5FABhmZYgLzTKpKTXA245d1PRT4OekBsD8Bx5AshE0wAl+EXzbJX0byjHD8XdRAEaWOkCjyH0vsq/U9iMz3+o+YT3N3BIYfUWZEAWmb3zCFbVrKiGerzOwVSnqUjda/hNOI5Hspe+UzSLDD6hjBMB7RrWgYhqRglhHau8nH4BNL4WjDXwAdlScgZ2FAgDjGAle+gl5EgIGqKYBxssID8IO4+PDIcGGbTIXJ1UPgPEVjPSarMCx5cUf1xLTYKiJWi0hhbIeTCB4AEas2/9n7zygrKa2MLzpTXrvHYY2tGGGOvQOIkVQAQVFREUEFJ/0DgqCgqAiCCr2gpXeexGk995779W3du4ks5M5SU5yc5I7Q7LWW0/mppycc5J8+fPvvVe8UzuOP1MPgOUHMt2v/DkRLSLo402M/geyaB/edx88hE1HLsfJnkABWG4X7sZpAKaFNVDlndczWmktKg/3HgQsENTvih6/PcMTNgBjcOTP3QI5kfUAmL4UBAvAcnJ9HgDWZlAJBoAxxd7yPrUMc40GC8DYh2afvPEY+NVoy3F9AMbrKZsFANZ6pel1SAH4jTpFlFSIWLJ95f4LqmvWaQDGna85eEHxy/MAMGbqwJdVPQA2KtVMTwbtS3JxG/n+SVMh4ktdvZLZmRYIuwDcuVoBGNw88AVMXszmg9HjE8dDu72sAOP98d9jl1WxCFoAblomp1Qx0yj3Na3EqG1L/yYl4OXoWAAO5pxoUQ1WnxilypSPy+oPzB3eLFwtNhj1qZ3f3OYQO230ahsfgAX2vNsTb+720/CqRgFGAEb/KCtQgAXA+DBCC4RcOhMrGdFCAVYAGLsWb9rZ0qYEDFZgLTwArE36rgfA83tGM3OAyg9k7fHRnpEmeVJV8Q7WTZKCi/bmt3FAPcWXiTl4l/WpLe3CDgCz1CjM4VkhX0ZVVgn0+S7oVZMJwHR8MLPBrmGNVArw9E4RkCqZwwrw7fu2yoA6cel5DcBocxjfthwTGNBSQ6sNWgVgtBLMWHMYvl5zFGa+FAklc6UzTLXkBABj9P/SveeU1G7aMcJjYBAXVgekD3aa71UXgBlfOHAfzcJzMnMx428UgHvUKaIonk+VywVVCmeG//0am1c4AMA3oN745cyppQUQ/LeRAow7WXvwovK1hAeAMQ4BLUFymkGtAmyUp5g2mgIwWh1mdI4EmgoRX+rql2ArwFiJj6bL01OA8f5w695D5bDUAhYMLBoBnzZ+gd5PWQA8uX0Fw5cyWohGO+j9moRB1+jCceaCHag3S4N2cFQTZoEMenAWALO+tjlxX6T7cJtDnG6/yP35ACywd92eePN2nIZu36gtENlQAbYAwPgwQgW45SeB2vGtK+RRPgHiv40AGD8DYQodKwsPAKMCHFUok1K0A0tkLu8TVwFe0CuamQIJA9UQwq0srIAo3F5789w0oJ5SBAJz8GKOX7sAzLJAyABMgbpotidgYW82AIfnSQ8YaY6L7AGmFgjMU4oR5UalTlmqhKEFwkMApmWh9RRg6osOVgGWo8FHz9ktle+d2SVSNbdoP+F1RMss2wFgDCZDdR+/rJh5dJ0AYJw3eDz0sNJcvxRqnpy0Splj+HfJAkE8wPL1FscCoQPAmHEEq1iyFgSPgTEWiB51iyqKJwIwVtB6ZeYmZTNsh1EqNxYA48s9Xh90kSvB4d+sAjDegy5cv6sLwEYeZdoGaoGQAZimQsQ5jUDOskDQCoK4z4QMwJgdheYZpn047umy0LpiHssAjJC+42TARiYvNKcw6154aFQTVQYY1lz2AdjKE9iddX0AFtjPoQLAqADLn9Po6bIUYIxcRwVYTtXTqnxumLU5kEILl1AAYFlp1d6I9AAYbQqohFtZ7AAwJpFf/FYAgNEyUXxA3LyQVj3A+PAqny+jan+Y6gxTYslL8QFz4W6MBQLTI2nVOdEAfO3OfQgfou9NtdLvVtd1G4BpNDirrXTevKIp3WsXgOXj3H/4yNBn6BQA4/HQb9pv1nZoWT439Pxxi3KqeAxt+XJuAP7nONN/qc2WQfvVCIAxfVRXwQBMS2zzKMB4XSKoPjd1vXQaWgXYyKNMz5sFwLTkePHsaQEDloMBYK194IUq+WFoi9KqaW1HLZV3YGSBkNfhUYBX7j8PU5YfArQBaV+UMA84LcuN+8VsIScu34bxbctCUoYgY3ZOWGmTetxxnzSlGmv7w6ObmKaM8wHY6t1d/Po+AAvsY/cB+Ax0+yZWEcFTw1yKF2/wAzACL6ZBaz5pldQz+AD8jROA0QfFuuEYdTGvAly5UGb4cNE+aVd6ALywVzQzBRLaFFiV14zaxQvAWPpYLgJBwdQOALM+x+JLSrm8GYDCT6GsaWBJDGjjOVAA1t688aarAuDOlQADR4wUYFZghpECfP3OfUNvqsBLTAoQxEBBXI5fuqUK/pGP66QCbAWAMa3U1JWHVfBoBs30d613HYt9FOo3R7c7cay1eVd5+97oxaz++OWw/9wNkIM8teOtBWD5hVN7DbHKDmP7UM39naSL4wXg7nWKqALesB2Hzt+AOuP4LRC/bjoBb2kUYDkIDtthFYCXvV0LTl29rQAwBkK+3yZcOSWeYiO4MguAaSpEtEFh+jgWANP80bgvPQVYC8DPV8kPwwQDsDaHuREAYzzKJ+0rKn336bKD8P68PaopjTnLtx2/osrpbjSXpefHu7MNLwt8qaZ5rnFlmlGCtT0LbrUHYa3DyrnOe83yruc2h/C2KxTW8wFY4Ci4PfHm7WADMN445VK59HRZCjB6ftEL1uzjAABrH05GCrBbAFwwSxopJ6r2RoTZHFgR4GhTyMwoPew0AFNvLn5GpmlxMGjnf43CoHTu9LqHZVkgMKocVV30cBeOgR9qtdACMAKhNkdrn5+3KoGMmBEjdbIkSpo7VmN8AI5N26edYzQa3AxmMZPGX+TTvlUFmBW8afTwFgXA6D1FqwD6nbHCmxYitAAsv3BqAVivEpz2K5MKgJ8qDQN/3yH96c26RZVMMPhi/mG7cnEyTTgNwOsPXVSuFaOKdXKbMQDt+OVbugB8+MJNw6AueT8UgOuEZYPpnSoBzbaCNijM2MMCYJo9xAoAd6ycHzDLCV3MYNHoHhqsAswDwHIlNT3BwuwaZf2OAdda+w/NKOEDsEBocnnXPgAL7HC3AXj+zjMqTxyeWo50KaVCGLwAjIrF81XzK6l6rCjAGAigF+ym1808CjAGwaECPH5hQAGWAZAXgFGlxc9nVpZpKw/BiNm7AUuOznqtmrKp9pibB9ZXqqDR7AxaAMYgphpFsxo2gQXAGFWO0ExTVMkvAPLOwgbOhTv3A1kgtOqFVgH+MkYBlvM8OwHANEOFlT52Yl06Pm4owGYAXHLQPCWwSJupwG0Axi8iWETCLJUUjoOZaqYdKy1wUA+wHgB/v+EY9J0VG7Qm71ObL1kPgHvWK6rEAcgArC23bgSYLA+wmQJsFYBRfcVcs3KaQa0CfPTiTag5dpnp1GcBMLUaod0KsyTwADBL6MAG0BLh+O8OlfPBiKfKuArAHy/eD+MW7oNqRTJD0sSJYfm+88rxtQD8ybIDMGbeXlX75EpqTgIwxpysPXRRdRwaUPfqN5vi5K22qwBPeKYctCiX23Q+BLOC2xwSTFvd3tYHYIE97vbE0wNgVIBZ1XJYN8ZnI/MBfgprPCGQT9dKIQxRAIw3JIz4NgfgmswIcITUjBYBGIFz+8mrUCx7WiU3MPaHFoC3DKqvVM+i+Xm1fs1vXoqC6kWzWAZgzGxRPEdaaTv52HIQIAuAWQowtUBgRamUSRMbKsCs8pxGFoibdx9AqcHzBV5J+rt2G4Dfa1UGnonMp9sgTIHVYtIqOH75Nvz0ShUlm4oRZOr1LSt7iRUFGB/KX64+DEP+2mU6Nk4CsPzFRQsl360/Bv1+iwvA2kBb2lhMPzUgRgHuVa+YYoNyCoBn/XsCev+kHwSHSmDbKYGUkDwKMKr2h87f1AVgvZc07QCxAJi+aKINqpkOAGP2kMqjFyu75AVgTNs3sqV1AEYQ1+b0lee7dr5qLRBo68H7bFjOtND1600hAcAI46sPqAH43cZhgIUscEFxA/OK0yJSPgCb3mJCcgUfgAUOi9sAvGDnGVVQCJ4aKsBWABhVgI6VCyjpxPDf36w7pvSSkQUCI2G1+XTNupdHAUYARhUYlQJcZA+s9ua6+K2aUJfh/0NIzZDamgKs124jAKYV2pwCYAyqQW8xBWCabxj/ThVgsyA4BGD0AMsPddZ5JjQAxoChvWcD1fGMQI+lImnH+/3WZaBdJX0AxmPgQ/3K7fvSVwceZUqvfLEdAG4yYaWq6Metew8A/4bWBa2qRcfeDQD+dv1RVYUy+fjaXOPBAPCRCzeh1gdshZWlADsNwKh+Yyo22WOvVYD1MpUYAbBcbAXHsuSgwIsmfgXDjD0sBRghPGqUOQBrU4g9F5UP0ONOFx4LhDbdn7w9Cwq1AEyP9cL0DYYAPHnpARg7P3gFmAoCrPsfK8f0O42Kw2u1iuj2jQ/AZk/60PzdB2CB4xIKAJwzPVog7imFEujpspQBjATuUDm/EkyGydFnrD7iKQAj/CIE2wXgrYMaQPrUyRwZae0DAfctV+eiN3enABiDagpkSaMCYFpxDn8oMXAe3L4fyOdJC2HIwEdv+F+jAmwCwLKvjudBiDd++mB2pJMt7IRHAXYbgPX6TQ8ysTxun1+2KRX85O3tADAGWWEu32cq5YWOVQpIu0IgT5QIDKPUnQRg2XKkhf9v1h1V1FzaR9pqk3oA3Lt+MeUrkJ4CbBWAf9t8Anr9qK8A/3PkklIVkycLBH5twpctPQDWK9fNA8A0DR7aoJo7DMDPRuaF0a1iA/awTTwArH1GOAXA6HH+tENsEJxTAIzt+33zSVV2E9r/WHgEM5O888s2xTrYp2FxeL22D8AWbs3xYlUfgAUOk9sALFdeo6dkFYAxBRoqAXIy+VeiC8GUFYc8BWCtAiynG9PenJe8VZMZAb51cANIn0o8AJfNkx7+6F5d6qsHDx9BEVIa89suUVCtiHULBAbV5M2UWgXAuTOkAsz1KS8UgDEQb9/ZG8pvWg+wCAC+fe8hlBgUN+WbwEtL2TVmvfgtxqOt93nZSQAe0zoc2lbKy31qPAqwvDPtfGblrzazQBg1LJhttfs18gDLliPtOjPXHVUC2uj+ENZ/+Oc4s+nUAvFW/WLKS7AMwJEjF8G563elbXGuG3lsWQqwFQDmsUDsGd5ISkMoe+y1CjCt5mY0VrQQhqwA08wyaIPC/MkfM0oh44tTJFGAMZBWTmtJj6lVgHEc3mttHYDxmTF9dWy2k/gAwH9sOQlv/hCb3o/2C86zN+oWVeWUlv9G19POb7OXBZZK7HuAuW+lQlb0AVhItwZ2GgoAnCtGAZbzxNLTZSnAmLoJfcByKqHutYvApKUHuAAYcyEmQqnJwsJrgUBf1gcLAhYIOd2Y9oaDmSFYZTO3DWkA6VIKAuDBDZQqaJiuDPsUF6cAGCO6tZH3OKZr+tZVepkCMH4aPXThpvKbFoAxGA8V4Kc/C/gaWYtVBTjUAZi+FARrgfAagJfsOQsbj1yGWf+ehDPX7ijDh19Jvnu5suGVJ1tlIgtkgg1HLqnWdVIB1gXgtUeUohb04Kg8fr/BHIDfblBMuQdg5ojx7cqB1wCcMXUyuHzrvnQ6WOzy0OimQAPntFBJq7kZDRYF4HolssG0FyqpUiGiDQpVSiYA968LkSNjLRC8AKyFdWyfGdThOtqCL/J5BWuB0CrAk5bsV8ZfPoadIDjcVg+AMYfwm/WKQoqkSYB+TUD/Of6dLnYAGL/O0FzGH7UrB0+V94PgLCCDo6v6AOxod6p35jYAL9p1Frp8vVHVCKsAjMn7n6mUTwFJ+tkRd2zkAbYDwEP/2qmyWLCGAxXgYAB4+5AGkFYQACNcy0Ug6Od4mrYMz8muAkyLeMg3XPR1r+sXC8A080DeTKng+KXbugCMwXgpkiU2BODPOlSARqVz6t7s6Q/4kDOrUOb0JYag8ei/wF7pS4eeAuwoALcJh7YR3inAcl/WGLNEGec36hSRiteYBXpi/yzefRaii2WN86XESQCWPfdaQPh67REYFFPVTQ3A+QAzRLAWqgDjZ2jZAyoDsDYLxLGLtyB67FLmvlgKMOtTOK0Et/HIJWgT87LIUoDxyxIGPuKCKSM/eqa8IQDTam5WAZjeU/A6b1kuN9MDjF8OuAA4ZVJVyWtt0Y7HEYDpdUDvJ5iCr1f9YkEDMObQnr7qiBLMuaJPbciXOfCFT9TiNoeIOg8R+/UBWESvxuzT7YmHD7eXvlIDMH4uxwpCvAowvgG3q5RXSdWDuWtp8nEjALb6EMVu4gVgzKAgP/zkUsBadQL9sqwAmB1DG0opf5xYtMekAEyrkmkB+LsuUVDVhgWCZrCQj43FTdb3q6ecDgVgtLycvhqrDGoV4IQAwJhqD/tXC8Da1HNyB+F8wUIOuASrAFM44plPwVggjCoYUgC2mubv/PW7UGnkIlXzrV672vMKHzJfKUmrB8BfrTkipWXTLph94Nv1FgG4Qm4Y3zauAqwFYNle8XrtwtCnYVicoMRgAZiW4pUBmGaO0CrAtJqbVQCmqRAxDgBtICwFGOcNHV9dBVgDwBiMOPbpsrqQp9feLtULqgpRyOs5rQDLKdNoO5xWgOl1QP3aPeoUgd4NigcNwPIO9py5JsXlhOfJwHMbCWodtzkkqMa6vLEPwAI73O2JpwfAmAdYzhNLT5dlgXi1VmFJ4ZKtBJgAHPPhygurTjq94VntTh4LBH7eRf+sDMBywQktjC7vU4uZY3Pn0IaQRhAAo7pcJqYMsJywHvtAW7XLLgBT+4Z8vljWGR9y8kIBOMsTyeHCjXvKb1oARiU6RdLEiqrFGq/POlSERqVzcD0IvVCAaYoozHqBD3h5uXLrnhQQWGX0EuVvNDVcQgRgq2n+WBDmJADLQadaSNZLyabNNEMnnq4CrAPA2q8AWJwHLUH4EoT2rIJ9Z8N/MV8P8JxZn8LpS86mo5eg9aexadBQeX79u3+VLxDUR4uWhAnPlJeKKMhZVrQAfPHGXag4Qv3ywboGWRYIXE/uUxQ2MH+ykwCMRZCwCh5deCwQ2oqHCQWAz1y9o6STQyvg2w2dA2Crz8lg1nebQ4Jpq9vb+gAssMfdnnjoD3zxy7gKsB4As0pkYq5DvLE2+HCF1DPDWpRSfbb0AoDRAkEVYKsAvGtYQ0idXIwCjAA8YdF+KTH6D10rKwFrtgD4n+Pwzq/bVDOStl1+GGFZZ0y3JC+lBs2Dm/cCWSDoJ1n8txsArFf2WdSllTxpYiWriRaA5WPSBzcGR3aYth5aVsgtqYB6C0ut1QIALZPLc35uKMBWs5ywIMxLAMYKZBggx1ooAGMqKrkQAt6jUAHWWiC0AKxNzVio72wFXu0A8NTnI4Dm46XFJGQAppkjtAB8+eY9pXCO0fxRA3B2mPZChLS6PJ/Q2ta6Yh4mAOO9IYJANq8CjPmYx7W1DsBdowvB5yRQ2ikAxlLPn3WMzQIxcfF+JQuIfAyRCjC1q6AwhF9D9V4OeNOg8dwznF7HbQ5xuv0i9+cDsMDedXvi6QEw5gGW02TR02UBMHqAm4fnUkohY15Imrw+FCwQGNU/v1d0nAAN9FOx/H+7hzWCVMmTODLSWiCS7RX4eZIGANLPlXjg716OgqqFTbJAMAB474hGUkAGffihyrtxQH0mAJulQUMlGgFS9jWyOsWqAqxnPXCkwxk7SZkssfJFg2beMHo4aceH1TYWrEaMWKhS1K0C8NI95+DjJfthYLOSUD5fRsMu0c4tuaIaa6Pq7y+BE5cDXm+rWU7wfoC5h+niKADHZF3R9ueM1YdhKKMoB6Ze/GqtGADWxiVgOXHZPsMHwJeVYiboAUYAptdi6uRJlMp/sgXCCIBpOeNgABitTqjYshRgLERCVWZeAJZ91XrXkV57tZmCRAEwK2OHSACm1wk+F/s2LmEIwPj8xfHYfOwKs6usXmNO3T/d5hCn2u3GfnwAFtjLbk88fNh2/vIf1RmhV+ziDX4Axrd5zOsof0L+9VWsZhWbMcBpBZjXA1yjWBZF/dEDYEwZVmNM3AAYTE2EmQ+cWLSQomevsAXAG49LuSfpQqvrycfGAgvo+5QXqgBrLStaBRhBHC0QdEy1/TKlY0VoWIrfAuE2AGMhD/mFjheAecaeBcBY1EBOCYj7GPd0WUl5E7FYAeBq7y0B9CjiYjXIk6VCWn04G3mAZduOdp3pqw7DsL/jVqXTK6SA50YVYBqPoKcAawtNaAG4SL858CDGP64HwGNIoOOmo8YATOcijwJMyxkHA8AYCDvkyVLQ7ZtNcXbDC8DUv4w7kVPLhQoAa7NA4IvL8L93SfMeU37iIhKA6csKPhf7NTEGYLnf9GwjVq8xp+4xbnOIU+12Yz8+AAvsZbcnnh4A45vsrZhP5PR0WQow+rn6Ny0JWCEJt8GiGPSCDgUADsuRFub1jKsAhxIAYz/TfrOrANObpry/DKmTwZZBDZShLD14vvRZFpdBzUqqIMMNANYW/RB4SUm7TpM8iWL5CM+THv6Myb2s9+DmffDo2RU+W34Q3pu7R9r9+LZloVWF0AJgqx539EmXGyZOAdYD4C9WHZYARrvweoCxHK08DvLneq0FQgvA2rEv2n+OUtyAB4D/PXYZWn2yRmoySwHGl0k5wFgGYJo5QltcgtonjK4ThNHfNp/UPW62tBgIWxewT2mMBm6AL8dU4ddTgLUALCvYlgG4ZiGYsjw2V7y8fbBBcFoLhLzftQcvKqWmRQIwZmyQ4zsw0G9As5KqIdO7X/gALPoJ4Nz+fQB2ri/j7Ml1AN57DjrPiKsAWwFgswvdCwsEBsGhB1j2/1kFYGojCHa4tTc3I38xXff7lytDlcKZDQ//E8MCwQJgfHBtG9KQCcCDm5dUfWbWAjC2I3nSRIYK8OcdK0IDCwqwNudxsH1stj0NPHIDgKcsPwijPQBgVPIyP5GC2R1UAbbqcWd9hud9SZAbY6QAy4q0dp1pKw/FgTXtC432ZIc/VVopntG3cZgyDnoArK20pj2vYv3nwr2Hj6TD4G+YkxVzs9KFKsBmAEz96DwAfPPuAyg1OFDO2Gj59dWqhtYLGghLfc24TwyKLE8sLn92rwZPTlod53DaeAG5/VYBGONG8CVRu4gC4DUHL8BzU9dLhxMJwLTCJaYZHNTcB2CzeRvffvcBWOCIhQIA582UCi7duKcoZvR0WQowJjVHv6LeTdBpBXjYX7uYVYTo8TEIDnOXyunY9AB41f9qQ/X341og9o1oLPlenVi0AGzkL7YMwAwLBAuA06ZMCtt1AHhI85IwhPgs2QCMFoiAqsVaRAIwKldy5S6744Hnf/1OQPF2A4A/X3EQRs0JKMAftisLLcu7owDzArBViw/mrS07dIGq+70EYKN54DUAbz52GVoaKMDJkySGnBlSwtGLt2DumzWgRM50QDNHaBVg3qIxVLVlKc80DsApAMbKchOfLa977y+QOTUcuXgrznBhgNiny7wF4B82HIN3Z22XArgxONJs0SuEQa8Dmt8cbYGDm5fS7RvWfZr1QmDWLhG/u80hIs5B1D59ABbVsx5Uglu29xx00ijAVgHY7E03FAAYHzL4sNHCqB4A7x/ZGJIliQcAzKkAowK6fWisAlxm8Hy4HmOBMANgzFSBfREMAM/oVAle+WYT9KxXFF6rVUQKKsLgIp7FCQCmyhV+kfjrjUD5abpYyb4gb1d8wFzlczZ9oIU6AFv9wsHyoToJwHJgKK8CjP2PvvM3vt8MfRoUh5FzYtMuUg9wvyZhyouIrAA3nbgSdp66Jg0hnsOpK7eh6nuxKfDiKMAD5ioZRHgUYDMAxmwNeL7Xbt+HbOlSSu0wAmDeojEUgBuUzA6fa4LvMqdJDpti4gC0AIx5mKnFhVcBbo6llQ0AuHqRLLDqwAXXALhhqewwpWMg6JAuVAGe/FwFaBoeKNqD9pdc6VNBYqyUY7JQAMYyx5+vPAQ1i2WFSc9VULak1i70qaPnmuce41sgzHo/dH73AVjgWLj95rV833l4YfoG1RlhyUy0QMgeUfojSwE2u9CdtkDwKMBogUAFWPb/lcyZDuYwAHj1u3UAPw1rF8wFmlQQABupb/RGiOBZuZAzFgj0wO4c1kg5TQrAQ58spSo2oFWAf+xaWeoLIwDGSHB/PEcAACAASURBVHdUnYxu9hj4JqvqVgAYi3icvXY3qKuOlp91EoARdnr+uAUCxWDyKW2cuuKQAmUiS5dqH5xGBS6qjl4Mp2IKnlj9wkG9jfJJOgnAsidZC8C0H+kEQMjFWAN5TtHtqALcv0kJZRxkAMYgxa5fb5SK97xSs7ApAGtfcv7aekoCb7pQC4QZAGNRFgxUpQsNnNMqwLwBoxRaWQCM18DmmDgALQBjWryyw2IVfl4AbhaeUwWAeE50LPQAGK+XT2IUYLxPH714E16vXUTKJ6+d03rXKx4Ln134DJMXqwBs5aZCAfiXblWkghSYX5xm8qGpLDEFH77s+wBspZdDf10fgAWOUUIEYK8U4JrFzQF4zbt1VOqPPLQ0k0Kww629oXsBwJh6aRcF4CHzFUsADwAnS5pYCexh9QcPANPttDmPjfrYCQUY1a+LNwPFPnA+/v1GjTiHtKMA67WbelcnPFMOWpTLHew0Ym6vLWXOC8BWX/C8AmCqpNMOmN8zGornSKv8yQoAazvy9NXbqiIoWrAPGzhXSaGHv5kB8JbjV+CpyQH/LMuKkCgRwOHRTQ0AOB+MblVG+Z3XL28GwDQQ1ikARiUVFVU9yKtRNAus3K9WgL94IQLQJz15acAC0ateMXizXlHmWOIfjQD4+ekbYAUBYCtBcFYvSC0ARxTIxNzF5KUHYNuJK1KBE20mIT8Izmqvh976PgALHBO3ARhvHngToUv+zKklD7D8iZz+xlKAMSfn0BaldW+CXgFwreJZlQAYzHU7u0dcC8TavnVUDz/5JLTJ8IMZ8lAAYMyDu2d4Y+U0yhAA1hYu0SrAP71SBZImSeQoAGtTvhn1Ly3VXC5vBkDAsLrQancJCYCxH7CELZYqxoUXgK3Ob1YmAicVYDkoTwsINJiQjvmCXtFQLLs5ANMUf6yqZbhPMwAuMXCekkIPz/nvbaeg+3f6CrAZALMyJ6gVYDUA874s/tW9OjSftErqJpYCjDYgzP+MSxwAHtxA5fHWU4ARoq/cuq8MRdMyOWFye30AZinA2IcfzN8Lk5YekPbTu34x6FFXH4D1PPu4rRaA9RRgVhYIq/cQCsCY6rNifjYAG+3XB2CrvR566/sALHBMQgaAb95TFEIzAH6+Sn4YZgDAXlggMAguGADW5gINZsi1AGzkv6TrovUgyiELBKZe2jtCHABPez4C6plYIGgfWgFgLOH6XusycOTCTUmJo35P3nHBCHgZEuWXIe228VEBxnOIHLlICRI0KnFcZfRiOB1jgbA6v1mZCNwAYJpOjo7Xot7RUCSbNQB+umIeGKsp24v7pOVr8d/a86Jlw3kAeOvxK9CCoQDvO3sdFu8+B+0r54N0KZOpph/NHPFspBqAea8VCsAUBOV5TTPBaAEY09CFx5Rnx4bRfdGGagGY+mnl9cwsEFYB2AkLhNMAjBYIPQXYB2Deu3L8XM8HYIHj5jYAr9x/Hjp+oVaAMXIXPxfLUfNmAIxlSdF3Rxd6E/RKAa4dllUJgJFVPy2MrutbV6ndTttv9eHOe9PD9bwAYPTeou9TXsKHzIdrMVkRhrcoBQP/2Kn8plWAf+5WBTBwR45sZ52rVQDGfegFfmj3jwCMXm1c9NJimV2S1EYh+8FFAjDNXyvSAmEXgK3Ob5reSe43q/swSoMmZ0bRroOZAuRMLmoArglFsj2h/IluR4PgqAJsF4Bp0ZhgANhojhoBMO+1YgbANBCWB4CxCt+szScln+v9h/9JzadeekwnhoUnqAdW21Y9BXjcgr1KRToMKHvDQAF2GoBZ0G52/8DfeS0QvM8CPwsET6+H3jo+AAsck1ABYAyCkwGJni7r0xgmpR/xVKxnTXsT9AKAMQgOFWA5FZV8E9VCFyaGjxq1OM6IWn248970cD2jAKRgFWBtJSR5f/gQ2z8yNvDGCgCj2oGBO0YAjL6+uiWMg+CMgNOo/5wA4EJZ0sChCzeVh7gcDESP66QCTAEY00RhuihRCy3sgNH8GVInZx6KKsBW5zcrFZfVfRgBsOyL167zybIDSi5velKL36oJhbOyAZgGwWF6RrmQhh4An712R3UP0J6XFoBnbzsNr3/3r6qPx7QOh7aV8kp/Q/+nnEOXeoCNxp8GzmkVYF4A/vuN6ko5epYC/ESKpFL2CVy0AIx5mOUCDvg7wnRYzrRStgwURCqPXgxPpEgGhbOmgfWHL0n70LuP8SjA4xfshYlLAhaItxsUg+517FkgeIPgqALsBAD7FghRd7PQ368PwALHyG0AXrX/AnT4IpAgXF7whqcHwOgdbP3ZGjh37a5SVvW5qHwwqqU+ADttgcAHGgKG0YIWCKoAx0cARu9tZEFjn5m2EIbW2yk/jLSR5yoAJoUDsE+1CnBCAGAshb337HVpyqDHfXmf2nGmj5MATEv4YpooTBclanEDgFmpuNwAYAwoGjt/b5yuW/p2LSiYJU3sPevd2cp/UwCmVQ7tAjDNmILnHF8BmGaC0QIwgjFWh5QXBOAyedIr/0YPOH4F6vLVRiWtGQ8As4LgsA/HL9wHExfvDxqA7XiAfQA2vxO5zSHmLQqdNXwAFjgWbk88FgDjgwUBGJPfaxe8eaEnDUt5hg2cJ/1splg4rQDzAnCdsGyKX1QOpNAqwBv61YVIlxVgoxzDtH12AFgLJfL+MM3lIRJ5jkUN5PGln4xZAIxqR5LEiZXIdtb0d0sB1iuNa3ZJYiEUTLM0ackBGNGyNFRiRHA7CcD4+XhoTHGRUAHgyqMWw5lrd6SusgqvoQbAy96uBQU4AJhWOdQD4HPX7qjuAdq+oQGj+Nuc7afhtW/1FeDtJ64qwWi8CjANnDO7n+rNdTMFmGaCsQrA8jE7TFuvALDefYxLASYA3KdhcenaVAQY8jKDf7MSBEeD/2g/OaEA/775pJTyEBdfATa74ybc330AFji2bgPw6gMXoP00tQJsBsB4+jQy+ZlKeeG91uGqXhHpAbYDwGXzpIc/uleP4zvd0L8uRI501wJhlIJKFABrocdpAJ7eKQLqhNmzQGBaqP8CFkPmQi0QwQDwvJ7RhleukwD85erDSnW9Sc+Vh2bh4hRgCraYzzV9anWAlXzSVCm2CsB3HzyE4gMCL7zyYnUf2v6lc1DPAjFpyX74YMG+OOO2vE8tyJ/ZXAHmAuDrd1T3AK8B2OyLGg8A03Rgcr/TTDBaAMY8zLTcslYBdhqAP1y4DybEKMBmAGwlDZoPwM7Aidsc4kyr3dmLD8AC+9nticcCYPRLXrp1T5XuhvXQk2+sLIWLPuy8skDULZENRswOVIiKjwCMwWcspZJOvx//OQb/+3W7LpToQR2Fj5EtS0P/33ao9vH2z1vhl00npL/9+mpVyQMs5zZlTf9gABjV6UecAEytBVYuQ7kSoNE2PgDr9w6rGEOwAExtOHJgqHYMPl68H8YtjAvAK9+pDXkzpVYaLPt00ftavWhWGPh7YD7TKodtI/LAmDZl45zkORMApu3UVYDbhEtFHHAJVgFmATB9yeEBYBoLIPcpzQSjBWBMQ1dykL4FQj5m+2nrYPWBi9I/9V7keRTgjxbtg48WBSwQ8QWA/TRoVu64CXddH4AFjq3bALzmwAV4TqMAIwA/HZGXGX1NH3pYvvPguZvwdEQew0hgLywQGASHFggFgPNmACwVqrVA/NO/npRHVbtYfbjzghWuZ1Rkg7bPCQCu/v4SOHH5NnSvXQTeblhcaWa5YQuUFxwzAJ71WlVIkiiRktqJda5Y6rh2WDbVT2ZASf3JWBlOb6EKMLUWWLkM3Qbgr9YcUarr2fUc8p4fDW4TpQDTEq9yu6xeI0ZBcLKf1C4AH790CxbvPgstK+SBP7eeUgCYFnnRA2BMj0fvAdrzoi+LPBaIHSevKsFovBYImjqNBcCD/9gBX609ajglZveoDk0nBvIAswCYZoIxA2C0U5TOHesBdhqAJyzaDx8uCrzYaCumae/RRhYIbRCcSAXYLQDG7CZYrbBL9YIwoFlJ3tuAo+u5zSGONl7wznwAFtjBbk88WiNdPq1CWdPAgp7R8MM/x2FAjJJi9aEXChYIqgDnzZQKVr5TJw4AbxxQDyJGxF8ApkFwNM2RPF6Xb96TCkdUL5oFkpHSzhSAMYCx329qFZkqwKIBGINrHggGYL3UZ/RSNgN2K5f912uPwKCY1HKYLqpJmZxWNre0Li1xjIUOsOABa6H5gq3CK6samdV9GFkg9AAYA6UwYEq7rPpfbciTMVYBpr/PXHfUUQCm14pXADzkz53w5ZojwQFwksSwb2QgFaIWgDENXYlBsRYXPQCmHmC9F3keBZiO6/8ahcGrtQor5xYMAOvl541PHmB8id107BJUL5JVKR1v6YbgwMpuc4gDTXZtFz4AC+xqtyeeHgAveauWlOWh2ntLVGfL+9ALBQtEuXwZAPOIUnjX3ly9AGCjKly0fTzJ1qkFwkpghhaAM6VJJgX2vFGnKPSqXwwoAGP1v8QCFWCaZ5R1aeVKnxLW9K0r/US9tVYuQ73iFz4A8/UiKvSF+82xdS+QNzICYDmgSrsOVQrpwbGEea4MqZiNpwBMqxzqKcAXbtxVvQRr73Hlhy2AyzHVz3iyQFAFuF6J7DDthQjTTqap01gKMA8Az+lRA5pMXCkdi6UA44vmgVGBVIgUgDE4EAsZxXcANvpiFp8AmPcZazqpgljBbQ4Joqmub+oDsMAud3vi0RuDfFqY63HxW7Xg1JXbUNUBAPbKAhGeJwNgJSkjAN40oB5UdFkBNqrCFQwAo8WjbN4MXLOTPtRHtyojZfLAVEeYKxQXLQCjB1jObco6wIzOlaB2cXsWiORJEsO9h4902+0EAOuVPxYFwDPXHlGKi3yKBQNCQAGmJZOtPmRZ5Xit7sMOAFOvKB0rLGGeM705ANMiL+3Q1tVGHayL+zQD4ArDF0pZcXDxCoCH/bULpq82Tv1IAbhJmRzwSfuKUpup1QhVWy0A4/2IZvXB33kUYL0XeR4FmHq7320cBt1q6ivANKBPe5OgadAwo0+2dCmZ9xEfgLkeC8pKbnOItdZ5u7YPwAL73+2Jl5ABGIPvpqw4ZAjA/w6sD/iA0y5WH+5GU0KrOjsJwNQCgUVKEPp5FhYA0+0oAP/+ejXAQDVRAIzBOfgA1luoB5h6a3nOU17HdQAmn+FFAzB+pcGvNbhgSVttmV25D0IZgOWAKi0k02wBtJofVnDMkZ4NO1QBpjmB9QD44o27qpdg7bVfcfhCqTImLwDvPHVV8eLyeoBp4BxLAR7x9y6YZpL7nAJw0zI5YXL7CioApqkQqQKM56tNcycagGl2j76Nw+AVHQDGPOiTni2vC7a8ALzu0EV45vN1Un/Y9eS7lQbNyWePlXskXddtDrHbTi+28wFYYK+7PfHojUE+LacVYKezQPA8DDAIDj97T10Zq5rgjUULo14AsNENjraPx9JALRB2Afi9VmXgmch8qllNARiVZUxVZgTAX3auBLVsKsCpkiWB2/cf6l5VVAG2C8BGqZTkAzvpAaYQ9lmHCtCotDgPsBsAjLm/C/YVZ4HQA2BaMCF7uhRw9tpdabiM1D6nAThixEK4cCMWgP/edgq6f7dZNV9pJbhgAbh9VD4YqSksNHL2LtW9jHWxzH2zBjSeELBANA3PKYEeLvK8xmv4cEwucCcAWO9FnkcBpgVOjADYDAZpEBymtMyWlv1S5DwAV4WK+TNaJgG9e4yT9x7LjWJs4DaHONFmt/bhA7DAnnZ74rEAGKNQF/WuGbIWCB4AxkpwCMBUNWEB8OaB9aG8ywpwKAAw/az7fusy0K6SMQCjB7j5pECEOWtxC4BpcJmVy9AoklwEAH+z7qgSQPpZh4rQqHQOK821tC4vAGOwJ37ux8UMLFgN0L48Wt2HkQVCDqjSrkNL5uZIl1Ip5GEEOxSAaZEXVr5yPE+0N9CvQNrzsgrAu05dU7y4dhRgFgCPmrMbPidfs8wAuFl4TpikAWA67loA1uZ55lGAnQLgfk3CoGs02wJhNsd4AfjM1TtSOWdclrxVEwqRMtq8F5taAfYBmLffEtp6PgALHFG3AXj9oYvQLubTkHxaMgCfvnobqoyODYKjn9XMukBkFgiaMkivHQjAmPqK+uZYALxlUH0oN8xdC4QoANZLXs/qIysAjMpyIhAHwFih6tY9PgU4vgDwt+uPKrmVRQOwnOoOx3n7kAaQNiU7C0QoA7DsJzUC4JzpU8Lpq4FKdpi+MGvaFMzLnwIwTfFnH4DVLw5mCjAFYN4gOBo4xwLg0XN3w5TlsXYu1onP61kDGn0UUICx9DbmZ8eFpS5qAVib55kHgPXuY1YV4P5NSsDL0YWUU7KihvICMO4cA77vP/wPahbLavb4Yv7uWyBsdVuC28gHYIFDGgoAXDTbE7Cwd02gAFytSGb44oVKkDJZEq6zF5kFQntTZzUILRDFc6SFGatjUwfFPwA2VxmoBcIuANPPt3JfUgsEDwB/9WJknAeL2YNM/h0D7zAAT29B8FkbkwWCBpdxTcSYleRCKEbbmLXXyvEoAE/pWBEalhKnACdkAB63YC98vOSA1PVohTkVA8AYvJr5CWcAGFMF0q9AWrDTeqf/2noK3vhe3wKx+/Q1xYrgFAC/N3ePKqDXDICfLJsLJmoAGANZWUFweL68AEwLYdgB4BerFYRBzUvCJ8sOwJh5e6XTCAaAqQfY6KXIyrWrt64PwE70Yvzfhw/AAsfQbQDecPgStJ2yVnVGMgDTBPGDmpWEF6sX5D5zkQowDwCjAowATHNnsgAYcy6WHbYgznmZfXrj7giNAoPb8SvA1gBYT7VhtZUG9pgBMII1+gebfaxvgQgGgDF/8XVeACbBZVbGALNjoJfZLQD+bv0xJbfy5x0rQgOBAFxjzBI4fikQBGesAKu9rFb6j3XNWb1GjCwQ8ud07TofzN8Lk5YGABiDIeVgP/TuZ0qTnHkKVAGmOa71FGAzANbmT/YCgN+ft0eV0tEMgFuUywUTngkowGgjwWBgLFZTtUgW6W9aBVhb6EQvw0MwAIzV+/JkTCUVTcL0lHhOuAxoWgK61LCnAHsHwOb3ZtYY+R5gq3ed0FvfB2CBY+I2AP9z5BI8/ZkagItlfwIW9KopnWXPHzbD4Qs34cdXqnCrv9qHpdNp0HgBuGj2J+BrUj2JCcCDGwBWetIuVh/uvGAVkgBMyrjK50EVYARrXIwA+OsXIyFa82nRTFGVf0+bMilcv8OpANsE4HJ5MwBms+Adp2DH//sNx6DvrEBxETcBeMfQhkoqO+25ar2sVm9jTnuAaS5qPQAeO38PTF4aSGVIARi9+xk5ABhT/MnjoAfAV27dU9mgtGMfNWqREnyHv7EAmProg1WAO1TOByOeKqMaHp7gz/k9o6HhRyuk7Z4qlws+igFg/DcqvFgJTl60AEwLnaB48H3XyszpEQwAU8+wCADGnO5ZdL4KWJ3rrPWX7T0HnWb8I/2EfY0Ci9XFB2CrPRZ66/sALHBMQg2A7Z6q1xYIvIkXzpYGvll3TDmF+AbAWIGtQj7jSGNqgbCtAHsMwOhXRKjQW6gFggaXWZmbXgLw1OcjAIOhRC1UATYCYG06L6vtcRqAtSWGsT1aQBgzbw98ElPMBtVDLOuNi1HJZ6oA8wDw1Vv3VV+BtABcedRiJfiOB4D3nLmmeHHtWCBYAIwA2+vHLTB7+2ndYaMA3LJ8bviwXTnddY0AGO1u33YRC8CYnx1tHbg4pQCLBmDMhNLnl23SC+aQJ0tZvXyk9X0AttVtIbWRD8ACh8NtAN545BK00SjAxbOnhfm9ooM6y1CwQBTIkgZQiZMXFgBj3tTwIaGpAIsEYKoGjm0TDk9H5FWNtxsKMELvpqOXARPho8KFn2E/WBC37K0TAFw+Xwb47TX3FOAfNhyDd2MUYNEAHD1mKRy7dEsav4QGwPTTf95MqRSrh1HJZwrAmOJPHodnI/PC6FZxC2GYATCtwob3kD+3noIeGg8wVYDtADBNncYCYPni7P3jFpi1+STz3rygVzQ0+DCgALcqnxvGWwBgWulPFADTF4spyw/C6BgAHtisJLxE7HVmX47oydMgOCNfeFAPMwc39gHYwc70aFc+AAvseB+A+TpXq0Zpt8IguLwZU8OPG48bAjB6Jsv4ABw0AM98KRJqFFVHV1t5kMmDxBpXTH+1rl+gFDINLuObKYG1KuTLALNcBGCqzE97PgLqCVSAa45dCkcvBgB459CGkCammp+2f0JNAaYWCBmOtHOGBn9RADYq+EEBGMH0f78GrCi6AHz7vsoGpVWAb917ADPXHoVqRbJA6dzp4Y8tJ+HNH7aoupcC8N4z1xUrAq8CzA3AP22BWf9yAHCF3DC+Lb8CTCv9uQHAn684CKPmBBTgYACYeoB9ALZyRzRe120Oca7l4vfkA7DAPnZ74m06eglaf6r2ADutADtdCAO73wyA0QKRO2Mq+GXTiXgLwL+9VhXKC7JAUAX4g6fLQpuKeVSz+q2ftsKv/wb6bnaP6vDff8YeYB+A1TcFCsBfvBABdUuIs0DwArC2pK/V25jTFggeAKbpvygAGyndNFMIBni+8+u2oABY208iAJimTjNUgHUAGG1Eb9QpoijArSvkgXFty+oOceF+cwBVX1wQ+EUBMML06gMXlePIDaIArA2wtvLi7AOw1auYb323OYSvVaGxlg/AAsfB7YnHAuCwHGlhXs/4b4HAT+f0cyHLAoEP0tKD58cZ0WCDoOgOrYADXZcHgOmndiseYJoTdtzTZaG1AQBjidX/4D+lvCtr+n/zUhRULxqIMJcXGV5YWSb0LiHWiw21QNDsClYuQ6za9OurVQ03sfLgNTs2LVEtGoBrjV0KRzgU4HgJwHN2K+XM82VKrVg9jJRufQDOB+gJ1i7X7txX2aDMrn0zAN539roCorwKMC8A0xdTPA/80tWnYXFJmT5y4SbUj7FA4AstvtjqLVoAppX+cJ/fveyMB1gPgKeuOAQj5+yWmje4eUnoXC02w5CV65ACsFFmELPr1a3ffQuEWz0t7jg+AIvrW3AfgC9D60/XqM4oIQAw3sSzpU0Bv2+JDa5iATA+SEv5ABwHgKkH2C4A40MVq2zp5WtlXUZmFgiaXcHKZRiRPyP84iYAbzwO7/wSUB6nd4qAOmHiFODaHyyTMrXgsmtYQ0idPCmza8oPWwCXb92XfjODPJ6xsboP7cOfRwGmFdAoABudJ7VAjGkTrozDs5GhC8A0c4SRAqwF4BpFs8DMl6Kk4dp/9roCwE9XzANjLQAwbi+Pj5MAXL1IFlh14EKcOTdt5SEYMdsHYPk6swL9Vu57dtd1m0PsttOL7XwAFtjrbk88DEISDcBeWCDwJo7gRbMLxDcAxrRdmL3AaLGbBYIqwOPbloVWFdQWCArAc9+sAY/+s64A27lMRAFwpQIZ4eduLirAPgDHGX47ADxy9i6YuvKwtK/8mVMrXuc9wxvppmWkCjAGeGLkPi56AHz9zn1VHIAZ2JspwBREeRVgCsAdK+eH4U+VZl4+9LrEFSgAHzh3HeqNDwTBtY3IA2Pa8CvAXgLwkOYloZNNBZgGwRmlxrNzLxKxjR7ohg+ZD9diUkGazT8R7dLu020OceOcnDqGD8BO9SRjP25PvH+PXYZWnyRMBThj6uSqtEEsAEYlqeSg0LRA8ACwXQsErW7lBAB/2yVKChIKdjEDYHq+Vo7lNgD/vPG4Al5YgKB2WDYrzbW0bp0PlsGhBKoAj/h7F0xbZR+A0QaA0Pi4AXC7iLzwfpu4GS/kiaW1QFAAFpUHmILdF6sOw/C/d0nNGfpkKXihagFlzltRQ6kFIj4DMAZOjp2/F7rUKAiVC2W2dP2LWNltDhFxDqL26QOwqJ4FcN0CwQLgEjnTAap+wSyhkAYtfapkMG/nGeU0WAC8e1gjKDFoXpxTdfIt3K4H2C0A/rBdWWhZXj8Ibl7PGvDoEUCTiSt1p0SoA3BkgUzwU7cqhlPayoPX7Nr4iSjAMzpXgtrF3QFgnM+pkrPLlbMsB2bnQX+3Mo9Z+7WjAC/cdRZe/nqjtLtCWdPAofMBq8feEY0gRVL2eVIFGP3tb5kAMJbhpnEAZte+CAWYpk4zUoD7/LwVfiaBvWgx+KZLwAJBFWC9oh9uA7CeB9gHYHs2JCvXazDr+gCs33sJBoAPHDgAH3zwAaxbtw527NgBYWFh0v9rlzlz5kD//v1h9+7dkCdPHujduze89tprcdbDfU2aNAnOnDkDZcqUgbFjx0KtWrUszUO3J97mY5ehpUYBdhqAvbBAoIqBFcYW7DprCYBHPFUaMGdsqVzpLY2b0cpWwIGui6V7sYSv0WLXAkEVYDMAxgT7GDEeCgBMz9fKAEUWzAQ/veID8JV45gFGH/mv/56EvBlTwYDfd8D+czekYd8/sjEkSxJb2YzOBS8AGPMNPxOZLw6I8lognAHgG1Bv/HKpDXop30IFgKevOgzDYhTgYS1KwfNVgleAtwyqDxlSs8tjW7lXiFzXyZdske10m0NEnovT+04wAPzHH39A9+7dISoqCvbt2wePHj2KA8Br166F6OhoeP7556FDhw6wevVqGDx4MEyZMgW6dOmi9C3Cb79+/WDUqFFQoUIFmDp1Kvz++++wYcMGCYZ5F7cn3pbjV+CpyatVzSuZMx3MiecKMHqAUydPAot2nzMEYPQShg2MVYDN1B/ecaTriQRguxaIyJGL4Nz1u1IzP2pXDp4qn1t1ajTYhgeAv+sSBVVdsEDEFwCmFogvO1eCWiIV4HHLFGXUSAH+bfMJ6PXjVuDJiMGa51bmsdn2eJ3xBMHR/SDcHYgB4AMjG0NSDgBGe0/vnwIWiOei8sGolnHvxTfvPlAFwprdA37ffBJ6/qjOAxwsANPcwUYK8Du/bIWfNsamdlQrwBSA2QF/bgOwXhDcjNWHYehfAQvE8BaloKMPD9iSpAAAIABJREFUwHYeLcK2cZtDhJ2IgB0nGABG4E2cOKAidOrUCTZu3BgHgBs3bgyXLl2C9evXK13ZtWtX+PvvvyW7Am5/9+5dyJ49O+Dfx4wZI6338OFDCXzDw8Phhx9+4B4GtydeQgbgFEkTw9K9530AZsw+KwCMFaZQAW48Qd8CIRKAs6dLAev71ZPOgqYX476oACCqYCb40UUF2E0ArjtuGRyMsQYYBYdhfyFAYj5dPfuAUZ+GEgAfGtUEEidOxGwuVYC9AeBYEOVVgJ0A4IPnb0DdcQEFWA/23QZgPQvEl6sPwxCHAdioPLaVe4XIdX0FWGTvurPvBAPAtLtYAIxgmy5dOnjvvfegV69eyurLly+XrA0IzBUrVoSlS5dCnTp14N9//4Xy5csr6w0dOhTGjRsHV69ehUSJ2Ddr7ZC5DcBbj1+BFhoFuFSudDC7h3MeYC8sEKgAo0K0Yl/8BeA/u1eD8DzGFggnFOAJz5SDFuXUCjCNNucC4JejoGphMUFwTgBw5UKZ4Ieu7lkgKAB/9WIk1CymrpLn5K3aCgAHc9xQAuDDo5vo3lMpAKO9B1VvIyjESm80ENZMAZaVdNqXagU4FoDrl8wOWArbbKG5g40U4P/9sk1V3ZIqwBSA20flg5EMtZsHgI2ulfbT1jELW9Dzo/OEC4CfKg14zvJiBRJpEJwPwGazjP93tzmEv2Xer/nYAPCuXbugVKlSMHfuXGjUqJHS8+fPn4ds2bLBzJkzJVvEJ598Aq+//jrcunULUqVKpaz3888/Q9u2beH48eOSd5hncXviuQHApXOngx0nrzFP3+xho9dnrGwBdF30ACdJnEjJQYm/sYLgMJim+IDQtECIBOCoUYvg7LWABYIFwNQCsbBXNDwwU4DdAmASXIaBPnvPXgf0rGOBDKPFKLLdzoPX7FqmAPz1i5EQLRCAqTXATAE2a7fR714DMAV9o/sGBWC098h2BT1V9Pa9h6pAWLN7khkAUxBtWCo7TOloDYCfr5IfhrVgp0HjBWCjXMI4xkZZIIwA+Lmp62DNwbiV3awC8FdrjsDgP3dKm2HcRQcnAHhwA8DA51BerMC9l+fhNod4ea5Wj/3YADD6fatXrw7oA65cObYyzoMHDyBZsmQwYcIE6NGjB4wcORKGDx8Od+7cUfXlokWLoH79+rB161bJCsFarl27Bvg/eTl9+jRERkZagmarA0jX33biCjw5Se0BdloB9gqAUXSXb9Z6ALxvRGMoNmCu0iVmDz87fW0FHOi6oQLAi3oHALjRRwYWCJcAmIIlKmuosPH4go2S+/sAzDerrcxj1h7tZIGg+6lDvM68AIwvd2/+EPDr6qmiIgG4cekc8GmHiqYdTBVgIwB+99dt8MM/x5X9UQX40PkbUCfGAmGkIocCAH+99ggM+sMHYNOJ4dEKPgDrd/xjB8CYJQID5eRFBuCJEyfCG2+8IQHwiBEj4Pbt26peW7hwITRo0AC2bdumGwg3ZMgQQKuEdrGiGgdzjbAAGIH17zfivwUCfavrD19Swa32IR7KAPxX9+pQJo9xNgq7FojKoxbDmWuBF7aJz5aHJ8vmUk0jqgDzAPD3L1eGKoWDz1/JUvapBYIC8LTnI6BeyexcvmD8FPttF3Z5V9EAPPOlSKhRVJwF4nFRgEUB8J37Dy0Fws7694QSWCfPnfdbl4F2lQJZICiINi2TEya3r2B6i6bFM5wAYKN9mAGwkV/eKQWYqvQjW5aG9lEOWCB8Bdh0nvGu4AOwD8Bg1QKBAJwyZUql53gsEF4rwNtPXIXmk1apRttpAPZKAX7w6BH8c+SyIQBjOqWi/UNTAQ4dAK4J2JdGCrBbAPzLphNKYQMrAEyVMr1bm5OfJymoiwbg+uOXK+nB4pMFglWa2WgMaMEPJxVgJwCYeoApADcLzwmTnnMOgPvO2gbfb4hVgOmLHZbDxrLYuLxQJT8M1bFRhBoAY2YOtKfYeRGlHuBtQxpAupS+BYIXco3W8wHYB2Apu0NCD4JjATAGrf31RvWgrqNQKIRx98FD+PfYFUMAxnRKRR5zAP742fLQXKMA0yC4Rb1rSlkgGn4UKLPKWn7oWtmRCkYsBThb2hSwoX8gCwQF4C9eiIC6JbIDLTqh1z5aMtZtAP7mpSioXjT4AEG9dlMANioQEdQFDQCeWyBIxTteAMavGz2+3yydup4v1mkApiCKX1awDWYLLWJhpN7yAnCnqgVgyJOldA9r5AE2UoCdCoKbue4oDPw9kHPfB2Cz2eH+7z4A+wAs9QCmQbty5YrkA5aXbt26wZ9//hknDRr+HTNG4IJp0ND3i6nQQjkN2o6TV6HZx2oF2GkA9ioLRKpkSWDxHuM8wKEMwH+/UR1K5xZjgagyejGcvhqwQLAAmFogFr9VEx48DA0A/nXTCaWy1/ROEVAnLDtQtTUUAdipKnl659bgw+Ww72ygQERCBmBUNxEuceEFYJzbb3gIwC3K5YIJzzgJwNvh+w2xAZ96CnDnagVgcPPQBeBv1h2VCpvgMrpVGXg2pogI/tvKlxiqAG8f0gDS+gqwI7TsA/BjAMCYtQGrvOEyefJkOHjwIIwfP176d82aNSFr1qwS+GIhDEyT1r59e6kQxqBBg3QLYYwePVoqhDFt2jSYNWtWyBfCYAFweJ708Gf3+K0AY9DTmDbhUP39pcpMZmWBODiqiRQRLS+hFARnFYBn96jOXcGOAvCk58pDs3B9D/CSt2rC/RAGYKoK6922MAsDZmMwWqw8eM2eMhTKRQNwww9XSNkw4hsAW7VA1Bq7FI5cvOU4AOOXIiuZYFgeYGqBoApwy/K54cN25cymi5SfWa7iZqwA6wPwkQs3oVaMBSLUAfjb9Ueh/28BAKZ95wOw6VRxZQUfgB8DAD5y5AgULFiQeaaY21cuY4yQjFXeaClkTHtGFyzZKZdCPnv2rKT8YlGM2rVrW5qwbk+8+ArAVA1idbAc9a+FGu1nXEyoXyiBADAPMMt9ZRWAMQtEgw/1LRA/dq0MUYWcC4JLkzwJ3Lz3UGoutUBQBXhGp0pQOywb0L/pXWyYhxfz8XoBwE4VCdFrOwVgDOpMnpRdItjSjYixstMWCDcAGF/uun9nbIGwCsCs+UYhjoJoqwq5YXxbawBs5N/t99t2Vco/qgDT475YrSAMal5Sd8iNLBBGZcOdskCIAOAdQxvCEymSBjvNhW7v5Eu2yIa6zSEiz8XpfSfILBBOd5Ld/bk98XaeugpNJ6otEGXzpIc/HFSARVggjl+6BZ8sO6AKCKF9Lud99QGYPROrjl4Mp2IsEJOfqwBNw3OqVqQWiKVv14L7Dx+5AsBYEQvLpHapURDqjQ8AtxkAsxQ57VnXKp4VvuxsDMBYEAbzYqdLmRS2DWlo9xKWtqMKsGgAbvTRCthzJqAA+wAMQDMM8ADwvQePLKVCtALAbSrmgQ+eLms6l6gCbAWAaXq/oxdvQs2xgSC4l6oXhIHNQheAMW83wjwuNIMG/tsKJFILhA/AptOMewW3OYS7YSGwog/AAgfB7YnnBgCLyAIhDwEt6WsHgLGiVMG+j58FggLwJ+0rQJMywQHwT69UAVSOnFzkB6EuAHeuBLWLZwMeAK5dPCvMMAHg89fvSgF1GLiUN1PqoE6F2jK+cyhHsl6DHhcArjl2KRy1aIHAl7vXv/tX6jq9IDirAMyy3FAFmIJou4i88H4bdg54Op60eIYRAPf/bTt8S4q+6AHwyzUKQv+mzgOwU2nQ0Mfcd1YAgMe0Doe2lfIq3eEDcFC3Hkc2dptDHGm0SzvxAVhgR7s98XadugZNJqoLHDitAHsBwLwWCK0vOL55gOmDxIoFwgoAL3u7FtwzUYBFAnDWtCngn5gsEFR9+7JzJahVPBuwKnNpL9E6YdlgeqdKAq9c9a6pAuxUijgeAMa0fsmSJEwLRPSYpXDskjUPML7cvfZtAID1ikPg1w0rqRCtADBWK3yvtfsA3DW6EPRrUkJ3vtu1QDgFwDR/uRaA5a9Pz0bmhdGtjPuOKsA7hzaENL4FwpF7nNsc4kijXdqJD8ACO9rticcE4LwZ4I/XqwV1lqLToJkpwAkBgHmC2uwCcLX3lsDJK4HCLZ+2rwCNDRTg5X1qwd0HxhYILwH4980nlXK3epO2blg2+MIHYMeuadyR1ZdFrbJn1QNsB4Bxbr/qMgAfu3gLoscGgm8xuwFmOTBbaO5gIwV4wO/b4Zt1sVkgqAJMjxsUABfIBD91q8JsshAAbhMObSNiFWB8IcHYFLTOJTV5mYtvACxbP16rVRjeaRRmNi08+91tDvHsRG0c2AdgG53Gu4nbE2/36WvQeIJGAfYBmHe4uNazEjw0fsFemLjkgLRfHgCmSgrP+nKDnQbgn7tVgUoFxFgg9BRgDGrD4LY/tpxUyt3qDUi9Etlg2gvuKcBUJXQqR7LeueH1i9cxLpjWzwwauCYtYyUr85h1jGABuMaYJXD8UuCljTcN2mcdKkC3b4wV4AcPH1nKBc5Ku0ctEBRE9cova/uHArBRDl9eAH4luhD0tasAuwDAtHz52Dbh8DQBYCvzkwLwrmENIXXy0A6Cw3O7cOMuZE6THBIlSmTlVF1d120OcfXkgjyYD8BBdqDR5m5PPBYAl8ubAX4PUgHedPSSlOamR92iUrDajpOBB7R2saoiabcP1gMcahaIjxbtg48W7ecGYKoA2wVghIRGpfU9wCv61AaMlK9vkAUi9AE4O0x7IULglavetQ/Acbs6IQMwzWWLAbo1xgQUYD3bhV0AxuIRWERCXqgCTI/7Ss1C0LexTQuECwD80z/H4Z1ft0mn8bgBsGs3oSAO5DaHBNFU1zf1AVhgl7s98facuRanxK0TAEy7qNnHK30AJh1iBP0UgOf0qAElc6UznG1uAfCdBw8Ns0D80q0KRLisAGNeX8zv++fWU0q1L73Oql8yO0x93hsAdipFnN65NZmwEnb5CrDSPTQLxGcdKkK3bzZJv+nl18Uqh1ZygbMUYD0ANitJLDea5g42UoB5AfjVWoXhfwaf2I08wJUKZISfu1VlTjenLBC0eiNmycBsGXYWqgDvHtYIUiVPYmc3/jaaHnCbQ+LTAPgALHC03J54LAAuny8D/PZacB5grwE4vnqAJyzaDx8u2id1Hw8AO2GBQEhoVDqHalbTNGgr36kNt+97B8BZnkgBGwcESiHTILiZL0VCjaJ8ANygZHb43CMAFuGPpoPVdOJK2Hkq8IUFC7skSSzm06rXFojq7y+BE5fNLRBfrz0Cg/7YKfXHlI4V4ZWZxgD86NF/lnKBWwFgs5LEVgF40B874Ou15gqwmcfUawCmfTju6bLQ2gdggVRhfdduc4j1Fnq3hQ/AAvve7YmHeVcbfqQucJAQAJg3D7AbFoh9Z6+r1FMjBdgqADujAJsD8J37xhYIkQqwGQD/tfWUUu5W79JsWCo7TOnojQKcEAG4QObUsKyPtSI/wVogqG/d6BoSDcBUvZTnm54CbFaQQt6eFrEwgmZeAH69dmHo01A/yMoHYIEP8QSwa7c5JD51mQ/AAkfL7YmnhTM8tQr5MsCseK4AhxIAY5/y5rakADz3zRpQIqcYCwRV01Ala1gqOAX411erQMX8YoLg9AD4m5eioHrRLPD3tlNKtS+9S7NRqRzwWceKAq9c9a6pB1iEP1rvC4tbCvDmgfUhY5rklvrTCwD+vGNF6BqjAOvZEbCKp5Vc4FYAuEv1gjDAoCCFVQAe/McO+IpDAe5euwi83bC47vgU6TcHsLojLvLLhDw+Efkzwi+virVA0OtjfNuy0KpC8BaIPcMbQcpkvgXC0kWps7LbHOJEm93ahw/AAnva7YmXUAE4lCwQVgB44uL9MH5hwAIRKgC86n+14dY9YwuElwA8e9tppdiB3qXZuHQO+LRDwgdgLO2d2AULhJ3gVS0AVxi+EC7dvMeEMApm8pjaUYDR9/3y1xulXbgFwCcu34Lq7weC4MzSkcnnRotnGCnAWgCWX/RxP/S4oQ7A1Mr0Ybuy0LK8D8ACscLyrt3mEMsN9HADH4AFdr7bE2//2etxovsTggLsA7DxJKUKMKpkDQwUYATg2/eMLRBeAPC3XaKgWpEsMGf7aaXYgd5ZNymTAz5p7w0Ai7CH6CnA8QmAreYB5gVgGgQ37fkI6GICwFZeUHFdmsFAHgdqgaAgapaNwSoAD/lzJ3y55ogy/HoA/EadIvBWg9BVgGnxGh+ABQKFzV27zSE2m+nJZj4AC+x2tyceC4Ar5s8Iv+p8ArNz6l5kgUgIADyvZw0IyyHGAkFzqqJKhlkS6EKD4Fa/Wwdu3n1gmAUC5wvOGycXWTHUs0B81yUKqhbJAnO3n1aKHegdv2mZnDC5fQUnm2e4L/qJV8TLAT14849XwfaTV6U/JWQAptULeT3AXgAwFphBWMfFLBuDKADuUacI9A4BAMb78JqDF6XTpGNGc3d/1K4cPFU+t61rk2aB8C0QtrqQuZHbHOJcy8XvyQdggX3s9sQ7cO461BuvDoJLCAAcXz3A1AIRKgC85t06cMNTAE4OGwfUl646+unUEgCH54TJzyVMAH5y0irYdiIAwIdHNxGWYJ/Xx653ewzWAlFl9GI4ffVOHJjSHo8GwX3xQgS89JWxBQK3t3JuZgowBWCzbAxy22nxDCMLhJECTI/7Zt2i0Kt+Md0n1WvfboI528+o+lKEB1gPgKlvf8Iz5aBFueABeO+IRpAiqe8BdgJP3OYQJ9rs1j58ABbY025PPBYAGwVB2Dl1LxRgHgBuXjYXfPxseUsPPzvnb+UB+/Hi/TAuxgNsFYB50qbJ7acKMKpk9QwUYARgVICNCmHMeq0qVMgnSgHWAeCXo6Bq4Swwb8dppdqX3vg0C88JkzwDYOfVcXqePgCrR50C8PROEfDil84CMK1iJh+ZWiBOXbkNVWMUYDMvrigA7lmvKPSspw/AV2/dhy9WHZKsT6Vzp5eaIQOwkQDiVB5g+tXGB2C7TxVx27nNIeLOxPk9+wDsfJ8qe3R74h04dwPqjV+uOqOEAMA8FogdQxvCEymShiwAz+8ZDcVzpDWcbXbToEWPWQrHLt2S9m0GwGv71oHrd4wtEGIBODYPMLUWfKcA8Bml2IFeZ8kvOwIvXdWu1RYIsQDcYtIq2OorwEr/6wGwkbJqRQFmAfColmXguah8UhsoAJtZEeRG0ypuRu0c+tdOmLE61gNcuVAm+KFrlTjHNQNg1nUgAoD1FOD5O88o+Zl9AHbrrsR/HLc5hL9l3q/pA7DAMXB74rEA2KgSkJ1TF6kAVxq5CM5fvxunWTwArE3/gzuxE93O0ye8D1iqALsFwPiZuG4JfQ+w9wAcqwBTsPz+5cpQpXBmoA9TvbF4smwumPhseZ6hcmQdatUQ8XJAG0kBWNT8xePxzmG9DvTCAjGjUyXo/OU/UpPcAuDTV29DldEBD7CZFUEUAPeqVwzerFfU0lx2E4AX7TqrBCfidYnXp52FeoD3jWgMyZMmtrMbfxtND7jNIfFpAHwAFjhabk+8g+dvQN1xagXYB2DnB5gXHiYt2Q8fLAikQbMKwFYsEFQBNgPgdX3rwvU79w0tEL+9VhXKu2yB+KFrZahcKDMs2HlGyfWqN3ItyuWCCc+4B8AU1EX0zeMIwJVHLYYz16x5gGd0rgSdZ3gHwLwgyqsAD/trF0xffVgZfqoAU/DuXb8Y9KhrD4CNsgCxvMPaa47e6/QU4KV7zikvJk4pwD4AO/fccptDnGu5+D35ACywj92eeIfO34A6GgCOLJAJfuoW+KzmxOKFAmzmAU6UCAOGmkqnxwunwfQF7zEoAC/oFQ3FshtbIL5bfwz6/bZdatrsHtWhVK6An89sqTl2KRy9GLBAoE+yTpi+Ary+X124evu+YRYIEZAn91lkwUyA1dRwocqqDMALd51Vcr3qnfdT5XLBRwkVgCevhq3Hr0innpAV4KhRi+DstcDXHt4sECIAmJYfl+cbtUBQEH2rfjF4gwNEKQB3rlYABjcvxZzKw//eBV+sMgfgtxsUg+51nAfgM1fvQPtp66B+yRzwbmN2pTkeAF6+7zy8MH2DdI5OAfD+kY0hWRJfATa79/P87jaH8LQpVNbxAVjgSLg98XwAdgeAC/adDf8FCi8ZPrzjKwD//no1KJc3g6NXBuYKnfXvScAAozwZU8cB4B+7VoaoQpmBB4Bbls8NH7Yr52j7jHZGQV1E39BjPzV5NWzxAVjpEuoB/rJzJejksAJsBYD7NCwOr9cuYjrvnAZg3uPShsngGmweeArA1YpkhtUH4qZBW7X/AnT4Yr10eKcqwfkAbDrNuFdwm0O4GxYCK/oALHAQ3J54hy/chNofLFOdUUJQgM08wG4rwIX6zoaYyqOOAjANgrNigaAKMPoka4dlU80Bmgd4Q4wCbJQFQjTkyY2j1gIZgKmfUO/SbFU+N4z3CIBFqOOPIwBHjlwE52L8/rwK8FcvRipKo1MeYDMApkFw7zQqDq/VMgdgWjzDSAEe8fcumKajANPjBgPA5fNlgN9eq2b7KcejAK85cAGemxYA4A+eLgttKgZfCc4HYNtDFmdDtznEuZaL35MPwAL72O2JxwRg8snZiVP1wgIRagBcpN8ceBBDwEYPb6oAL+wVDUUtWCBsA3DnSlC7ePwDYLRFoD1i8e6zSq5XXQCukBvGt/VGARYNwC0/WQ2bjyV8C4QdAP76xUjAQClcnAJg+tIpzzdqgaAwizaBbjULm95GnQBgmgeYF7xZCrAbALzu0EV45vN10uHHtAmHthF5TfuItULHL9bDyv0XpJ8OjGwMSX0LhK1+1G7kNoc40miXduIDsMCOdnviHblwE2ppFWAfgB0f4aL958D9hwEPhBEAT156AMbO3yutJxKAaRAc+iQNAbh/XcC8oUYK8B+vV4OyDlsgWINAFeCfu1WBSgUywZI9Z5Vcr7gNpp46ceW2ZJ+Ql9YV8sC4tmUdH1e9HVILhA/AgV4KNgsEzfjCqwBTADZSVnk9+ngeZgBM7Qz9moRB12hzAKbwatTOkbN3wdSVbA8whehQB+B/jlyCpz9bK82L91qVgWciAynkrC4+AFvtMb713eYQvlaFxlo+AAscB7cnHguAowpmgh9jgo6cOFVfAQYoNmAu3HvwyHEAtmuBoACMPslaBgrwP/3rwZVb90IWgGlEufziMH31EQlU5AU/seKnVrcWNwG41Ser4d/HQAFeuf88dPxiA5iVtaYe4JkvRUrb4OIWANOqbgOaloAuNQqZTjsKwC9WKwiDmpdkbmMEwBS8eZVnehD5JYBmljBtOGMFHgvEpqOXofWna6StqXpu9Xg0DdrBUU0gSeJEVnfhr8/oAbc5JD4Ngg/AAkfL7Yl39OJNqDlW7QH2Adj5AQ4bOBfu3LcGwIt6R0ORbPxZIKxYIJwG4D+7V4PwPM4GwbFGgSrAv3SrAhEFMsHSveeUVFd6APx0xTww1gfgoCa2FZWUdaBgFWDc58UbdyFj6uSQ2AB0KAB/81KUEmzlFADTzCvyeVKIo6LCwGYl4aXqBU37nReAR83ZDZ+vOKTsj8IqBeC+jcPgFQ7rBW3YwN93AJYo/vXVqlAo6xOmbdZbgQeAMWgTgzdxGfFUaehQOb+t4/kAbKvbTDdym0NMGxRCK/gALHAw3J54LAAOVgHQdo+vAAOUGDgPbt9/KHUNrwXCLQDGQKGaxbKqhq33T1sUC8HGAfXg8k1jBTiUABj77YtVagW4bUQeGNPGV4CDuXWVHbpASodnNod5wAivgQrDF8Klm/dU+wsWsnFnXgMwzawzuHlJ6FzNHIBpAJuRAqwFYCpWUOWZ13qhHauHj/4LWkXlAeDtJ65C80mrpMMPa1EKnq9SwNbUpAB8aFQTwxcjWwd4TDdym0PiUzf7ACxwtNyeePSmKZ9WQgDgUMsDXGrQPLh5zwfgYC8ddYnhKlAxfyZYtveckuoK97+od00pVyq1QLSLyAvvtwkP9vDc27tpgcBPyfhJ2S6Y8p7UvrPXYcy8vZKiiRX4rC5OKMA8x6QA/G2XKGgfk23ALQWYVtcc0rwkdHIQgEfP2Q1TiAJMAZiKGf2blICXo82tFzz9aXUdHgDeeeoqNJ0YAGDePmK1g3qAfQC2OlL667vNIc61XPyefAAW2MduTzwWAMvw6NRpeqEAhxoAlx48H27cfWAKKTQIDkGuSDbjT5H0c6wVC0SNMUvg+KXbUnswUChaowDTNGibBtSTlDqjILi/uleHMnn4inAEM6/UAFwVKubPCDSpPu578Vs1YdpKNQA/UykvvNc6YQJwm0/XwEYXADiYccNttQBccfhCuChYAfYCgPefva5cK7zqJi2eYaQAGwEwtV6EOgDvOXMNGn20UppSg5qVhBc5bCI+AAd7BfJv7zaH8LfM+zV9ABY4Bm5PPOobk08rIQBwqKVBKzN4PlznAOBPlh2QVDZcQgWA/x1YX/JehioAr9h3Xkl1hf225K2aUqQ8VYCfjcwLo1v5ACzw1mW6ay8U4O+6RCn5Zo3A0or14tv1R6H/bztU50s9wHvPXIeGH62Qfuf1t3ID8NzdMGV5rAeYKsA0pSVv8J3poNlYgUcBpi8JwbSVWiAOj24CiTDBu78E3QNuc0jQDXZxBz4AC+xstyeeD8DuVIILHzIfrt0xV4C9AGCMlK9RVO0BpgowDwD//UZ1KJ3bXQV41mtVoUK+jCBnB5AvSzYA55Mqyrm1uGmBePqzNfDPEfEWiGD77nEB4Dv3H0LYwHlSd63rWxdypE9p2nVYYrjy6MXSeoYKsAEAU+8xb/CdacNsrMADwAfP34C645ZLe7frV8ZtfQC2MUAcm7jNIRxNCplVfAAWOBRuTzwWAMvqqVOn6YUFwkwBxiDyQ6ObSqdoRf2x2yflhi2AK7fMA4goAOOn/MIm0djUAjH3zRpQImc6riZSCwTbmkYiAAAgAElEQVRGylcvmkW1HQ2C2zywPlwwUYC9BGBaVhVPYunbtaRIeaoAPxeVT0q35NbiJgC3/WwtbDhySTo1owBLt85d7zheWCCoAozeZQRD1mLlHsBSgEe2LA3to2IzGaBl6O6Dh5AzfSqubqcAbNTO9+bugc+WH1T2SRVgCpXB2Aq4GmywEu1LzGXe7ONVksVq6vMRylbUrmEnZZu8IwrAoTz3g+1Tt7d3m0PcPr9gjucDcDC9Z7Kt2xOPJk+Xm5YQADjUPMDlhy2AyxwA/Omyg/D+vD3SUFgFYLseYDMA3jKoPpy/bmyB8AKA5QITWgV42du1pEAhCsDto/LBSBcBeNa/J6D3T1ulcRRdCMMHYPVNlQbBffdyFDw3NVBy1ykA/mbdURjwu9oCoQVgq48IJwCYBt/xZp+w2k6e9bUvExj7kCZ5EpU9gQovdso2y+2gQXA+APOMDt86bnMIX6tCYy0fgAWOg9sTjwXA1Ypkhm+7VHbsLL1QgEMNgFkBP6wO9gKAMVCoWhF9BTjUAXjBzjPQdeYmpTuX96kFny1XA3CHyvlgxFPeKMCyVcOxC0qzo7ZT1sKGw74CLHdLfATgs9fuQNSogAXCCNTx5RjvEfKCpcCxJDguB85dh3rjA97jYDIrBDtPedR0mvf47QbFoHudorYO6wOwrW4z3chtDjFtUAit4AOwwMFwe+LRG5F8WgkBgM0sEBgrcdhFC0TEiIVw4YY656kZAKOX1SwhvV0LRPX3l8CJy4EsEGYAvHVQAzh3/Y5hEJyXCvAfW07Cmz9sUbpzRZ/a8OnygyoFuGPl/DD8qdICr1z1rqkFwgfgQN94YYH4/uXK8OzUdaZgyQNt8giLUICdAGBMU9fgwwAAD32yFLxQ1V5u3WAvEp6+pIp37/rFoEddH4CD7Xcnt3ebQ5xsu+h9+QAssIfdnngsAK5eJAt80yXKsbMUqQCPmbcHPiGKiNzoUAPgSiMXSTYCXIw+1aG/D31+uPAAMPUjWvEAUwBGn2RVjQJMg+BCHYB/2HAM3p213RCAn6+SH4a1cA+AXbVA+Aqw6l711ZojMPjPndLfRFggZq47Clg1jS7BWiDOXbsDkRwKsPZ+RxXgq7fuQ9lhC6RmuVWYhvWQ4AFgfKGOHBlQvN+sWxR61S9m63njK8C2us10I7c5xLRBIbSCD8ACB8PtiUcrEMmnFZ8AGANNpq86ovhmeQHY7SC4yJGL4Fw8AmAaBLd1cAPAB7RRGrTZPapDqVzuZoGQvbUzVh+GoX/tUq7Kle/Ull6KqAf4hSr5YahHACxaAW43ZS2s9y0QyvjrAXCX6gVhgANBcKEKwNgB6w9dlLLN1C+ZXeBTynjXPACMaRUrjlgk7ahHnSLQu0FxW+31g+BsdZvpRm5ziGmDQmgFH4AFDobbE4/mn5RPq0bRLDDzpfihAGObMUNBRMzNlBeA3bZA0KwLvAowZjMomCWN4Wyza4Go9t4SQPUfF1TJqhbW9wBvG9IAzl4NXQCeuuIQjJyzW+mngAXiAHy/4bjyt4QMwM98vhbWHfI9wPJgUwCmFoiQBmCiiBp5gMfO3wOTl7I9wAIfS5Z2zQPAV27dg3LDFkr77V67CLzd0B4A+wqwpaHhXtltDuFuWAis6AOwwEFwe+LRT2/xFYCpmhCqALzl+BVAUMGSvEZKJLVA8ACwXQuE0wBsJQNFMJcPrQQnK8BXb9+HskMDn35xCaRBQwXYB+Bg+trpbd3yAFMA/qFrZXjm84AH2DEAXnsEBv4RsFjIS9AWCALARu2MA8AFMsFP3QJBcKGy8AAwvWZfrVUY/tcozFbzfQC21W2mG7nNIaYNCqEVfAAWOBhuTzz0paI/lS7xTQG2A8BuWyCwf9GukSJpEsPZM2X5QRgd4wF2C4BRJatSOLOqXdQCsX1IA8CgFSMLhJcAjA2ftvIQjJgdUIGxgt4Xq9SV4DpVLQBDniwl8MpV75p6gEVbIJ79fB2sPXRRakAop4JyC4BpFgi3APjT9hWgcZmctucX9cQaAfAH8/fCpKUHlONExlMAxtRoWB4el1dqFoK+jUvY6jsfgG11m+lGbnOIaYNCaAUfgAUOhtsTDxO2Vxge+BQlLwkBgEMtDRrvlKEAjPlsC7hggUBIqFxIDcA0CC4+APBMosot6BUN6AumCrAPwLwzUNx6bgEwVYB/7FoZ2glUgPFemSlNcviwbTlIjG/VNhcqRDwOAHzr3gMoOSgAwF2jC0G/Jj4A25w6QjZzm0OEnISgnfoALKhjcbduTzzqxYqvAMyC+McFgJ2wQLAAmCrAO4Y2hNNXbhsqwFYyUARz+fy88Tj0+WWbtAtaYELbD6gC+gAcTE87v21CBOD5PaOheI60QXcWLwCPW7AXPl4S/xVgWi7ayPNs1rG+AmzWQ/Z+d5tD7LXSm618ABbY725PvGt37kP4kFj/JJ4alq38+sVIx85SZBo0bCQLgM3SoHlhgeDpUPSujpoTSIPGowA7AcCokkVpFGAKwDuHNgTMFmJkgfACgH9/vRqUy5tB6iuaCg1zEmO/eAnAv20+Ab1+DFSCE22BeG7qOlhz0LdAyNeXngL8co2C0L9p8KWQqcXCKQCmgbxGCnBCAeCHj/6Dwv3mSEPGqkTJc6/EdXwA5u0pa+u5zSHWWuft2j4AC+x/tyce9WLJpxXfAPjyzXtQXmPjSAgAjBXN8mc2zgIRKgA8r2cNCMuRTuCVEdg1KwgO//7TxuPwTowyjDlQMQWalwDspge4/bR1sPqAD8AsAMYqaVgpDxejT+08gVvy/ikAo92mWPbgFWBeAB6/YC9MTAAKMPblnjPXpBfrOmH2U7b5ACzmlus2h4g5CzF79QFYTL9Ke3V74t2+9xBKDJqnOqOaxbLCVwlEAZYfGHJlJPlBFx8UYKsAbAVCaRYIhARMqE8XqgDvGtYQTl42VoCtHDuYy0fPAoFfMjDX8hMpksKad+tKhRBoHuDO1QrA4OYJMwjOB2D1jKIKsAgARtsYpl3MkDo5rO1bB5IlSRzMlJa2pYG8RgpwQgLgoDvNV4Cd6ELmPtzmEGEnImDHPgAL6FR5l25PPOrFktuQEABY9gDjOaG/LmvaFNLpJWQAtmJDSGgAjGN7/c59SJo4MaRKngT6ztqmUoDdBmA3LRAdpq2HVQcuSPM7PmWBYJUHt6LE6t2GRQMwHhdfuJInSQwpkxlndeF9VFAANrJqjF+4DyYu3q/sNr5mgeDtF7P1fAXYrIfs/e42h9hrpTdb+QAssN/dnnj3Hz6Cov3nqs6oVvGs8GXn+O0BpgBMT84H4EBvVB29GE5dvSP998/dqkClAvoK8O5hjeDE5VuGHmCnvJBml5aeAqzdTgvAL1YrCIOas/2fZse087ubABxfIMCLIDic209/5qwFws58MNvGB2CzHmL/Hl/mvr2z824rtznEuzO1fmQfgK33GfcWbk+8R4/+g0IxwQhyIxMCAMseYG3HhzoA06pmIi0QVgB4z/BGcPxS6AEwDYLzATj+KcAVhy+EizfvSUMnK9dOK8DxBYBpIK+RAvzhwn0wwVeAlcvdB2ButLC0otscYqlxHq/sA7DAAXB74v33339QsG8gGldeahfPCjPiuQL8OAKwFQsEBeBfulWBCAMFGAEYFeB641foznyngoHMLi2qAMcXAP711apQMX9Gs1Oz/Xt8gQAt3LphgUjoAFypQEb4uVtV23NHxIZOvMTwtiu+zH3e8wmV9dzmkFA5b552+ADM00s21/Fi4tEbFjbbB2Cbg+fAZlQBFpkGjQLwr69WgYr59S0QPApwqANwMLlG7QwrtUD4ABzoQa8B+JXoQtBXp+CCm9DGmk80k42RAvzRon3w0aJYD7APwOth5f7Q//ph5x7i5TZecIiX52vl2D4AW+kti+t6MfF8AJ6tjJLXQUS0pK9VALaSicEHYIsXpsXVfQ9w3A7TQubvm09Czx+3QNk86eGP7tWZkGyx26XVaRAcft1oE+MBji8AbJSu7ePF+2Hcwn1Kt/gA7AOwnWvEbBsvOMSsTaHyuw/AAkfCi4mnBeA6YdlgeqdKjp2ll4UwtCcR6h5gqwBMLQELe0VDUc6cpGoAjvuJnqZB2zsi4AE2skBYOXYwE8uuBcIotVQw7dHb1k0F+PnpG2DFvvNSU7x+gTPqS5bKeuDcdcibKTWkSBrIpuCEEksBGL9utP40EASXEACYlkrHc/IB2AdgEfcvLzhExHmI2KcPwCJ6NWafXkw8H4BDUwFe+nYtKJjFuBDG3QcPocWk1ZDliRQw86VISJQoEdfsrDJ6MZyOyQLB+kRPAXjfiMZw9OJNwywQPgCru90HYHMFmDVRnQbg+KIA05L0Rgrwl6sPw5C/dvkKcEwP+B5grtu95ZW84BDLjfRoAx+ABXa8FxNPC8B1w7LBF74CLHCU9XdNFWAeAMY9YSAjL/jKR6YAzCrVaxWAF/WOhiLZgq+IZdbpvgIct4demL4BlsdTBVh7Nk4DsEoBrlkI+jYuwZxiThzXbO4a/c4LwN+tPwb9ftuu7Coif0b45VU/CA47JJS/fgQzN7zY1gsO8eI87RzTB2A7vca5jRcTL74DMM2hKXezWRaIJIkTwcFRTaTVvX740alhB4A5p5ZqNSsAvH9kQAE2skCEOgAbBRbZ6T+zbdxUgDvN2ADL9sZPC4R4AK4KrT9dIx3mlRAG4Ku37kPZYQukdhopwLQUOK7rA7BvgTC7F9n53QsOsdNOL7bxAVhgr3sx8bQAXK9ENpj2QvzxANsB4FAthewFAP/2WlUon0+dposqwAjARy4YWyB8AFbfFHwAjnuT5HnR5FnH7Par9gAnLAD+a+speOP7zb4CHNMDvgXC7Gqw97sXHGKvpe5v5QOwwD73YuL5ABw6HuAvVh2G4X8HPH5L3qoJhbI+IWS2UQXYDIAPjGwMh00BuCYUySamrbQDfAtE3OnQecYGWOorwErH6AFwt5qF4d3GYczryQnwDuZCvXr7PpQdaq4Az995Bl6ZuckHYB+Ag5luptt6wSGmjQqRFXwAFjgQXky8uACcHaa9EOHYWbaYvBq2Hr/C3J8Tvi07CnCoWiDcAuDKoxbDmWuBUsisghJUAUaryOELN0wsEKENwG6XQpZTfGH/is4D7AOw+tZCARj97a0+CVgg4gsAG2WrWLr3HHSe8Y8PwD4AO/Z8Zu3ICw4RekIO7twHYAc7U7srLyaeaADec+YaNPpopasAXKVQZvi+a+U4x5TP9XEHYFqGdnaP6lAqV3pVX2kB+ND5G4ZZIHgD9oK9dOwqwJ2qFoAhT5YK9vDc27tpgfAB+PEB4DUHL8BzU9f7AOwDMPe9yM6KXnCInXZ6sY0PwAJ73YuJJxqAsbu0x5C7UJQCHF8BePqqwzAsxgKx+K2aUFiQBaLEwHlw+/5DaRhYBTcoAB8a1QQOmgDwqv/VhjwZUwu8MgK7/mnjcXjnl23Sf1sphdyxcn4Y/lRp4e2TD+AmAL/45T+wZM856dBOXE+iOonHZsCzjln74qMCfO3OfQgfErBAGCnAm45eUvIa47p+EJwfBGd2Pdj53QsOsdNOL7bxAVhgr3sx8bRwWr9kdpj6vHMWCArAWdOmgPPX7yo96MQDOyFZINwCYDrmG/rVhWzpUqpmtRaAD5lYIFj7EHGZ2AXg9lH5YGTLMiKaxNynmxaI8Qv2wsQlB3wAjhmJhAzAO05ehWYfr1LmnA/APgCLuKl5wSEizkPEPn0AFtGrMfv0YuK5AcB445659ih0q1UYan+wzAdgnTnkBQDvGNoQnkiRVBeAD49uAgfOGVsgNg+sDxnTJBd4ZQR2bdcC8WxkXhjdKlx4++QDuAnAd+4/hNFzdkNEgUzQvGwu187R6oF41F2edcyOSwEYAzxbxgMP8PU796EMhwK87+x1aPDhCh+AY3rAzwJhdjXY+90LDrHXUve38gFYYJ97MfG0ANygZHb43GEFmHaZEw85ur+EqgAv6i0usIyOAQa5oSeaLlQBRpV+/9nrhh7g7UMaQNqUyQReGcEBcNuIPDCmTVnh7ZMPgJ7pOuOWS//cOKCeVKnvcV94rnuedcz6kVZLowD8aq3C8L9GoZkFgheAMR1hLSIghKIC/M4vW+GnjScgd4ZUsPrdOmbDFdTvPgAH1X26G3vBIWLOxPm9+gDsfJ8qe/Ri4vkAHDpp0GasPgxDY0qdugXALBuKFoAPnLtumAViz/BGkDJZEoFXRnAA3KZiHvjgafcAGFu79uBFSJEsMVTQ5FgW3kkheoBXv9kEc3ecAaOAxMcVgG/cfQClB8+XRs7IA3z66m2oMnqJMsIV82eUsoyE0nL73kNYsOsMVC2cBdDyJnLxAVhM73rBIWLOxPm9+gDsfJ+GFAA3LJUdpnR01gNMu8yJhxzd36Wb96DC8IWqUYmvleDiKwCzVGQRlwmvB/jdX7fBD/8cV5rQqnxuGN+unIgm+fvk7IF7Dx7B9pNXoVzeDHG+OMi7cOLeEO8VYIOKddqvXaEIwJzTwZHVfAB2pBvj7MQHYP1+9QFYzJyT9urFxNMqwPENgP/77z94fvoGWLn/gjIyCQOAo6FItrRCZpsZaFhVgJ0IZuQ5UbsA3KJcLpjwTHmeQ/jreNgDZvOSp2kUgGke4NdqFYZ34oMFwgCAqVKMfeEDsB8Ex3NNWF3HCw6x2kav1vcBWGDPezHxtADcqFQO+KxjRWFn6cRDTts4hOCCfef4AMw5amZjkNAAGIPDPn7WB2DO6eHZambzkqdhCRmAUUUvNmCu0g0+APsAzHNNWF3HCw6x2kav1vcBWGDPezHxEgIA45DQ8/AVYONJagYaCQ2Am5bJCZPbVxB45fq7dqIHzOYlzzESMgBrX/R9APYBmOeasLqOFxxitY1ere8DsMCe92Li+QAcOkFw9OG9sFc0FM0eGhaIK7fuQblhap81vQzcskDYTYMm+quGwFvCY7VrH4ABXjGwQGhf9CvkywCzXqv2WM0RerK+B1jM0HvBIWLOxPm9+gDsfJ8qe/Ri4sV3D7DceVYU4KSJE8GBUU2kTZ146Do1JUIVgPH8lu87D1h0YeuJq3FO1y0AtusBFp3az6nxf9z348S1mJAVYB+A1VeID8Bi7hhecIiYM3F+rz4AO9+nIQXAdcKywfROlYSdpRMPOVbjrAAw5r3FzAU+ALPL52otEHJ/0ywV/2/vTKCrKLY1vIUAIeESJBAT4UEAmUFAIYDMigxeh6tXfQ9RUfQpiiCCvHCBKKOAE8sBccCRQQYZIk4oPlQQUGZQFBxIQImEKxeQJ0EG7/oLOq8Tzsk5p9N9uuv0X2tlueR0V1d/tbvqr127qtzwAFsVwN0bp8iMfs7ZtGMfi88ytqNtoAD2j9FQADtT1xTAwblSADtjcypXNwyvuAe44wXVZNadbR17Szs6uVgVwOZTrD58oLM0cCkEYvSSbTJr7W6F2ezdfWNNjjyU/bV2HuBuDavLq7dnOGbTzNgeAna0DWYBjD1y/z59tSrcwG71ZHhPbx6EcbjguFx45iS4AV3qyYjegctJDzA9wPZ8aSXn4oYOicZ72fEMCmA7KAbJww3DKy6AM+pUlfl3t3fsLe3o5CiAS1c9oeoA+41eM+1zaVc3ucgBErPW5sroJV9pJ4A7N6gub/SnAC6d1Th/dyi7DKcEFMDhUIqNa+gBdqYe3dAhzryJ/blSANvPtDBHNwzvna17ZfiCrXL0+ElVDmxUv2Sgcwsr7OjkKIBLZ4RGHTSvkSRLB3UMmBlWnJ9zTtEjkt/8crf8Y9E27QRwp/rVZOYdzs1qlK42eLdBwI62gQLYP/ZEAexMXbuhQ5x5E/tzpQC2n6mrAhgP3/XP/5NuZ86Yb5JWWd67v5Njb2lHJ1daAezVRXDmEAMnQyBWfrdf5q//SU211qhSMey6nrdut2QuLCqAX729jXRrmBJ2HqW50BwDnD2wg7T4jyoBsyt+ElywbfFKUxbeaz8BO9qG0ghgjPd2Tfqr/S8WIkeGQFhDTgFsjVuouyiAgxOiAA5lPaX43S3D+/ngUekw+fQZ8xekVJLlQ7uU4i1KvtWOTq60Atiri+CiJYCtVq55GzLk0apWFVkcxW2YrArgtnWqyjwHw3qs8uR9RQnY0TYEE8D3dbtAHuzZMCBy47nl48rIzgm9o14tkQjgFTvy5fZX16ky+n0btNlf5MqoxadDsqK1E03UjcOFB7qlQ1x41YgfSQEcBNnOnTtl8ODBsnLlSklMTJQ+ffrI5MmTpWLF8D1sbhne/t+OSZuJy9Wb1aqaIJ/9T7eIDSPcG4zOxuyFDffekq6LhV0gzAJ42ZDO0jDVmX2ArfJetPEnGTp/S+HtugjgjPSqMn+Ac3HtVnnyPqcFcHv5+/Q16iHhCODE8mXl63G9ol4tZgF8T9d6khnkyGYUzNxW+10Anzz1p2Rv/lkap1VWf0z2EHBLh9hTemdzoQAOwPfgwYPSrFkzqV27tmRlZUl+fr4MHTpUevXqJbNmzQq7RtwyPHMDnFo5XtaOvCzsMkd6oSFUK1WIk6/G9oz09qDXRyKAvRoCMXNNjmSd2WXBiwIYnc39czd7QgAjTh3x6oFS8RAIv5+YZdtH5nBG9nuAIxPASRXLyZaHezj8lmdnf+rUn9Jy3IdyuOCEfDq8q9ROTgxahn8eOSatJ5x2VkR7ABp1MHygKwTc0iGuvGyED6UADgBsypQpMm7cOMnNzZVq1aqpK+bMmSN9+/aV7du3S+PGjcPC7JbhFRw/KY2yPlBlPDehnGx6yLlOwOjkkhPLy4asy8PiEs5FkQjg8ypXkC9GdlfZ2tHphlO+cK7xugBeumWvDHpzk3YC2OmFneHULa8JTQCx6Xe8vl4GdK4rQ3sEDlcIlUvREIj/F8CDLr1AhgXJ02gDqlUqL+tH29cmhSqr+fff/zghR46dkJS/xJd4m7mtdjpcLZLy89rYIeCWDtGBIAVwgFrq0qWLVKlSRbKzswt/PXbsmCQlJcnEiRNl2LBhYdWtW4ZnPmO+QlwZ2eFgHJzR2ZyfFC+r/2Gfp9ksZG9sXVMevb7FWcxxkMOrn+fI9JsvkqbnJ3laAH8wpJM0SvXWtN572/Lk3tkbXRPAH23fJ//9xnr1/OVDO8sFKYFDRIp7gFvUTJLs+wLvdhHWh8mLokbg2ImTUiGurOXnvb1lrww+M0jDQtIeUz9TeY29uqn0uyQ9YL5G25GWFC9rbGyTLL9EiBuN8laOj5OtY+ybRXOqvMxXLwJu6RAdKFEAB6illJQU6d+/v4r5NaemTZtK+/btZcaMGWHVrZuGFy1PqPGca1vVkKn/2TIsLuFcNG3F9/LYsh2CeM8XbrlYzk0sH85tcsPzq2Vdzr/UtW4vpJi5Nleyzuyz60UB/MmOfLntzAIc8Ir2FCymioct2CKYqh5zddOg9VtcAN/crpZM+FvzsOyBF+lNAHGhQ+ZtltTKFWTUX5vI48t2yPf5R+TpPq0Ei9wCJaNNuqbl+fLUf7XyPIBotdWeB8ECOkLATR3iyAvZmCkFcACY5cqVk/Hjx8uIESOK/NqxY0eBOF60aFHAKjh8+LDgz0h5eXmSkZEhe/bskZo1a9pYbaGzaj/pY8k7VCDt6ybLm3e1C32DxSv+99t9svybfMns2UiSEspZzMW+2/YdLpCpH+2Uv7WqoQ5+cDOtzzkg1z9/etHOulHdpfpfKrhZnLOe/ceJUzLx3e3y+ppc9duQ7vVlSPcGniojCvPWhp/kwQWnF+v1bVtLMns3ksrx7tua50CxQIrAim/z5cPt+ySzV0OpkhDewNlNdIYA7tHkPHnx1tZuFoXPjkECFMDBK5UCOIgAnjBhgmRmZhb5tUOHDpKamioLFy4MSHTMmDEyduzYs35zQwDnHToq89f9JNe3rhnR3rAx+P27+krYaxcezl7N0lwtR0kP37j7X7Ju1wG5rUN6qaarnXpBeIpfW52jjpLuWP90TD4TCcQKgR/3H5F3t+bJTW1rSXIlbw2SY4Wxn9+DApgCOCL7txoC4SUPcEQvzItJgARIgARIgARijgAFMAVwREat+yK4iF6WF5MACZAACZAACcQkAQpgCuCIDBvboCEGGNugJSefjiOdO3euOgxDh23QInpZXkwCJEACJEACJBCTBCiAKYAjMmzjIIz09PQiB2H07NlTi4MwInpZXkwCJEACJEACJBCTBCiAKYAjNmwchTxo0CBZtWqVJCQkKO8vPMM6HIUc8cvyBhIgARIgARIggZgjQAFMAeyKUdPwXMHOh5IACZAACZAACYgIdQgFsCsfAg3PFex8KAmQAAmQAAmQAAVwiTbAfYAd/EQogB2Ey6xJgARIgARIgARKJEAdQg+wK58IDc8V7HwoCZAACZAACZAAPcD0ALv1FVAAu0WezyUBEiABEiABEqAOoQfYla+AhucKdj6UBEiABEiABEiAHmB6gN36CiiA3SLP55IACZAACZAACVCH0APsyldAw3MFOx9KAiRAAiRAAiRADzA9wG59BRTAbpHnc0mABEiABEiABKhD6AF25Sug4bmCnQ8lARIgARIgARKgB5geYLe+Agpgt8jzuSRAAiRAAiRAAtQh9AC78hXk5ORInTp15Msvv5S0tDRXysCHkgAJkAAJkAAJ+JNAXl6eZGRkyK5duyQ9Pd2fEIK8NU+Cc9Ac1q1bpwyPiQRIgARIgARIgATcIgBHXJs2bdx6vCefSwHsYLUUFBTItm3bpHr16hIXF+fIk4zRHb3MRfGSS3BzIxuyibQxos3QZmgzkRLwhs2cOHFC9u/fL82bN5f4+Hj7XiIGcqIA1rwSGd8TuALJJbhhkw3ZRNrs0WZoM7SZSAnQZuwj5kxOFMDOcI1aruyYKIAjNTbaDDsm2kykBGgzkRJjO0ObidRmon09BXC0idv8PDYyFMCRmhRthh0TbSZSArSZSImxnfj9vlMAAA3WSURBVKHNRGoz0b6eAjjaxG1+3uHDh+XJJ5+UoUOHSuXKlW3OXd/syCV43ZEN2UT6ZdNmaDO0mUgJ0GbsI+ZMThTAznBlriRAAiRAAiRAAiRAAh4lQAHs0YphsUiABEiABEiABEiABJwhQAHsDFfmSgIkQAIkQAIkQAIk4FECFMAerRgWiwRIgARIgARIgARIwBkCFMDOcGWuJEACJEACJEACJEACHiVAAezRiglVrJ07d8rgwYNl5cqVkpiYKH369JHJkydLxYoVQ92q5e8LFiyQ2bNny4YNG+TAgQNSr149ueeee+Tuu++WMmXKFL7Te++9J6NGjZJvvvlGatasqXbHuPfee89658cff1yeffZZ+eWXX9QJOY899ph07dpVSzbmQh85ckQaNWokP//8s+Ao7tatW/uezcsvvyxPP/207NixQ+2U0q5dO3n77bd9z2XJkiUyadIk9a2g3ejQoYP6/4YNGxb5DmL5m/r+++8FbcHatWvlq6++Ut8O/ls82cngt99+kwcffFDeeustOXbsmFx66aXyzDPPSO3atT3V/oRic/LkSXniiSfk3Xffle3btwtOHENb+vDDD8tll11mqc3VgU0oLsVfHH1WRkaG+sbQPhdP4fRFOnDxlPGGWRgK4DBBeemygwcPSrNmzVSDmZWVJfn5+Uro9erVS2bNmuWlotpWFogWvO+1114r5513nqxYsUJ11kOGDFHiFWnNmjXSuXNnufXWW+Xmm2+Wzz//XDXGL7zwgtx5552FZUGDM3LkSHnkkUfkoosukpdeekkgBnCcNBpwnVNmZqa8/vrrsm/fviIC2K9sxowZI1OnTlWDorZt26rB0wcffKBsws82s3z5cunRo4f6Tm655RZBmwJW2O7s66+/LtxSMdbtJjs7W+677z5lG3AqnDp16iwBbDeDK6+8UjZu3KjEIwZkDz30kOK+detWTzkwQrGBmIOToV+/fnL55ZdLuXLl5LXXXpN58+apASbe00jhtrk6sAnFxdx//Pnnn3LJJZfIrl27lPgtLoBjiYuO/SYFsIa1NmXKFBk3bpzk5uZKtWrV1BvMmTNH+vbtq0bijRs31vCtSi4yzjKvXr16kYsg+qdPn6467woVKkjv3r2VwPniiy8Kr7vrrrvknXfeEWzKDk8xPC4Q0Pj3Rx99VF0HTwaE74UXXihz587Vlt23336rPL7oWAcMGFBEAPuRDTybqFd47yD2AiU/cgEHDAg//vhj+fHHH+Wcc85RaDAAhBAEL3BBinU+ELzGDNJtt90m69evP0sA28kAbRMG8/CaXnHFFYrx7t271YwWvMD4br2SQrFBuwnhfu655xYWGYIPbRCEPZwUSOG2ubqwCcXFXH+vvPKKctTccMMNahbKLIBjjYtX7DaSclAAR0LLI9d26dJFqlSpIhiJGgkfU1JSkkycOFGGDRvmkZI6W4yZM2cqb+/evXulatWqqtFFGMgDDzxQ+OBPP/1UhTagY7v44otVo4wpR3hgWrVqVXjd2LFjlXA8dOhQoSBwtvT25w6RBxEPL0q3bt0KBTBsw49s4A1fvHix8uwFSn7lAhbw2m3atEl5HY0ETgh/MMSZ3/gEEsB2M8CMFIQQBurGwAP88b1WqlRJli5dan/DYEOOwQYHgbK+4447ZNWqVSrkCCncNldHNiVxgWOmQYMGAhGM/gfeXrMAjmUuNphcVLKgAI4KZnsfkpKSIv3791diz5yaNm0q7du3lxkzZtj7QI/mBi/uwoULVQgIGlu8//vvv69CQYwEzzF4QSxjuve5556TgQMHyu+//15kuhExxjfeeKPs2bNHTevplhBPiPf67rvvlLg3C2DMCviRDQaKmCFp2bKl8q6hQ8L38dRTT6l/8ysX2PZnn32m4jRxiqQRAoE1BYhvhDDGjIrf+AQSM3YzQBsDjy9ijs0J3+6yZcsUfy+mcAUwvKNoazCQQlgZUrhtro5sSuKC0JoffvhB9UkILyougGOZixdtOFCZKIB1qSlTORFrNX78eBkxYkSR0nfs2FGJvUWLFmn4VpEVGSNqiBl4DUaPHq3iffH+iNfDFKORsDADvCB60MHDQw52BQUFRR6ImEjEsW3ZskV5UXVKEPNYvINGFgOjTz75pIgA9isbdMKYHahRo4aK9y5fvrzA05+Tk6MGCoh19avNwL4RGnTTTTcJFtggNWnSRIkwYwDoN7sJJGbsZoA2pmzZsioO3ZzQhkEQwTPsxRSuAEY7ixk4tEFYj4EUbpurI5tgXDZv3qz6Jwwmjba5uACOZS5etGEKYF1qJUQ5IegmTJggmOI1J6ziTk1NVV7RWE7YuQGxiuio0dCCh9FRwbOC34oLYEw7Dho0SDXGYHf06NEiiD766CMVJ4opYd0WwmFBH8qPGDrEMwYTwH5jU79+feVRw6p+eKWQ8vLypE6dOiqGHt8LBLDfuIDD6tWrVQwqOvCrr75ahf5gkICBIb4lhMz47ZsqSQDbZSMQeXFxccoraE5YpPn888/Lr7/+6smmOxwBjHAztKH3339/4foKQwCH0+bqyCYQF8RBd+rUSfVDCKtDCuQBDrcv0pGLJ404QKHoAdalpkzl9HMIBDpqxPSio0acWXJysiIT6VQlBHB8fHwhVV1DILAQEnFmiHXFamMkcLnqqqtU7B0WpGDKNZIQiFhhgw4IfDBgMieEP7Ro0UINIP3IBSxgF7Vq1SoyW4RwIQwqsWgHC0z99k3ZEQIR6tvRcZof9hJKAMNxAI9vz5491UJic3yzMdUfi2wCccH7Y+tNeH+xLgcJ4YrYdhOLsdHv4C+WuegiqyiAdakpUzn9uggOohcNLKavEepg3jfT7sUqupiF4e0NVl6IQHhm/LgIDp0TppqLC2CIX2x/B4+bH7nAVhISElQIFbbgMid4x+EZnjZtmlq97yc+XAQXvNUrSQAjzhUzKdh9CN8bQo3MKZYXewXiAm8vQq2CJQy8IYhjmYsu/ScFsC41ZSontkFDHCu8W4YHFKNOHIYRq9ugIZb3uuuuU4t38BcoThfbFWGhE8SxkbCtEPakLL4NGv7dWESI7XyQH0IfdNsGDe+LeDNzwv8jDg8Cr02bNkrs+ZENFgZi+6Ft27apfbORcEBI3bp1Bd8Q9pD2IxdwgFjBzIF5JxkMFOAVxtTs8OHDFS8/8SlpGzS72hVjqy/zYl0svIVNem0bNHObEowNbAahRBgoGQPt4l2qsd1XqDZXRzaBuGCNAf7MydgfGfWObwz1HctcdJFVFMC61JSpnMZBGOnp6UUOwoB3NFYPwsCJby+++KKKLUN8lTlh8Q4aYGPDejRK2BMZMYzwcAU7CANTvRCH2DUDCwdj4SAMcCkeA4x/8yMbDGxwAhMWeSEGEZ4pxP4au4bgBEU/coE9YDoWMfHYfeCaa65RA0fEAGNQjcWBaWlp6hOLdT5YQIp9j5Hg9YY3EztjIGGmDXuP280A2xRietx8EAZCu7x2EEYoNti2DQu9wAz9DvZXNyfzYmTjwIdQba4ObEJxKb5fPZgEigHGv8cSFw2llFAA61hrImpvU3RgiPfEdCa8v/BqxepRyBD76JwDJUwlGccYozPDojDzUcjo5M0JixSM4ydxYho8vxDW2DosFlIgAYz38iMbiF14w7G37fHjx5Wowclw5uN+/cgF3wBOQEQcIhYKQsxgsADvb/FFoLHMB546hH1Es13B4RHGUch//PGHZ49CDsUGbXIwduAJGzNSuG2uDmxCcTH6IrNNBRPAscRFx76TAljHWmOZSYAESIAESIAESIAELBOgALaMjjeSAAmQAAmQAAmQAAnoSIACWMdaY5lJgARIgARIgARIgAQsE6AAtoyON5IACZAACZAACZAACehIgAJYx1pjmUmABEiABEiABEiABCwToAC2jI43kgAJkAAJkAAJkAAJ6EiAAljHWmOZSYAESIAESIAESIAELBOgALaMjjeSAAmQAAmQAAmQAAnoSIACWMdaY5lJgARIgARIgARIgAQsE6AAtoyON5IACZAACZAACZAACehIgAJYx1pjmUmABEiABEiABEiABCwToAC2jI43kgAJkAAJkAAJkAAJ6EiAAljHWmOZSYAESIAESIAESIAELBOgALaMjjeSAAmQAAmQAAmQAAnoSIACWMdaY5lJgARIgARIgARIgAQsE6AAtoyON5IACZAACZAACZAACehIgAJYx1pjmUmABEiABEiABEiABCwToAC2jI43kgAJkAAJkAAJkAAJ6EiAAljHWmOZSYAESIAESIAESIAELBOgALaMjjeSAAmQAAmQAAmQAAnoSIACWMdaY5lJgARIgARIgARIgAQsE6AAtoyON5IACZAACZAACZAACehIgAJYx1pjmUmABEiABEiABEiABCwToAC2jI43kgAJkAAJkAAJkAAJ6EiAAljHWmOZSYAESIAESIAESIAELBOgALaMjjeSAAmQAAmQAAmQAAnoSIACWMdaY5lJgARIgARIgARIgAQsE6AAtoyON5IACZAACZAACZAACehIgAJYx1pjmUmABEiABEiABEiABCwToAC2jI43kgAJkAAJkAAJkAAJ6EiAAljHWmOZSYAESIAESIAESIAELBOgALaMjjeSAAmQAAmQAAmQAAnoSIACWMdaY5lJgARIgARIgARIgAQsE6AAtoyON5IACZAACZAACZAACehIgAJYx1pjmUmABEiABEiABEiABCwT+DdBFhL9zn7WPQAAAABJRU5ErkJggg==\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# number of probes histogram\n",
    "fig, ax = plt.subplots()\n",
    "ax.plot(cand_prob_nums)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "id": "7a4d0b43",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:0, directly append 344 probes\n",
      "-- filtering region:1, directly append 319 probes\n",
      "-- filtering region:10, directly append 216 probes\n",
      "-- filtering region:100, directly append 344 probes\n",
      "-- filtering region:1000, directly append 326 probes\n",
      "-- filtering region:1001, directly append 341 probes\n",
      "-- filtering region:1002, directly append 381 probes\n",
      "-- filtering region:1003, directly append 356 probes\n",
      "-- filtering region:1004, directly append 283 probes\n",
      "-- filtering region:1005, directly append 321 probes\n",
      "-- filtering region:101, directly append 286 probes\n",
      "-- filtering region:102, directly append 240 probes\n",
      "-- filtering region:1020, directly append 52 probes\n",
      "-- filtering region:1021, directly append 11 probes\n",
      "-- filtering region:1023, directly append 217 probes\n",
      "-- filtering region:1024, directly append 242 probes\n",
      "-- filtering region:1025, directly append 306 probes\n",
      "-- filtering region:1026, directly append 17 probes\n",
      "-- filtering region:1027, directly append 3 probes\n",
      "-- filtering region:1028, directly append 1 probes\n",
      "-- filtering region:1029, directly append 235 probes\n",
      "-- filtering region:103, directly append 205 probes\n",
      "-- filtering region:1030, directly append 209 probes\n",
      "-- filtering region:1031, directly append 290 probes\n",
      "-- filtering region:1032, directly append 275 probes\n",
      "-- filtering region:1033, directly append 234 probes\n",
      "-- filtering region:1034, directly append 183 probes\n",
      "-- filtering region:1035, directly append 201 probes\n",
      "-- filtering region:1036, directly append 130 probes\n",
      "-- filtering region:1037, directly append 237 probes\n",
      "-- filtering region:1038, directly append 227 probes\n",
      "-- filtering region:1039, directly append 246 probes\n",
      "-- filtering region:104, directly append 119 probes\n",
      "-- filtering region:1040, directly append 350 probes\n",
      "-- filtering region:1041, directly append 254 probes\n",
      "-- filtering region:1042, directly append 325 probes\n",
      "-- filtering region:1043, directly append 122 probes\n",
      "-- filtering region:1044, directly append 253 probes\n",
      "-- filtering region:1045, directly append 347 probes\n",
      "-- filtering region:1046, directly append 263 probes\n",
      "-- filtering region:1047, directly append 350 probes\n",
      "-- filtering region:1048, directly append 296 probes\n",
      "-- filtering region:1049, directly append 176 probes\n",
      "-- filtering region:105, directly append 352 probes\n",
      "-- filtering region:1050, directly append 267 probes\n",
      "-- filtering region:1051, directly append 384 probes\n",
      "-- filtering region:1052, directly append 135 probes\n",
      "-- filtering region:1053, directly append 351 probes\n",
      "-- filtering region:1054, directly append 320 probes\n",
      "-- filtering region:1055, directly append 268 probes\n",
      "-- filtering region:1056, directly append 361 probes\n",
      "-- filtering region:1057, directly append 155 probes\n",
      "-- filtering region:1058, directly append 131 probes\n",
      "-- filtering region:1059, directly append 164 probes\n",
      "-- filtering region:106, directly append 244 probes\n",
      "-- filtering region:1060, directly append 315 probes\n",
      "-- filtering region:1061, directly append 309 probes\n",
      "-- filtering region:1062, directly append 240 probes\n",
      "-- filtering region:1063, directly append 266 probes\n",
      "-- filtering region:1064, directly append 327 probes\n",
      "-- filtering region:1065, directly append 318 probes\n",
      "-- filtering region:1066, directly append 207 probes\n",
      "-- filtering region:1067, directly append 204 probes\n",
      "-- filtering region:1068, directly append 147 probes\n",
      "-- filtering region:1069, directly append 160 probes\n",
      "-- filtering region:107, directly append 305 probes\n",
      "-- filtering region:1070, directly append 250 probes\n",
      "-- filtering region:1071, directly append 291 probes\n",
      "-- filtering region:1072, directly append 259 probes\n",
      "-- filtering region:1073, directly append 263 probes\n",
      "-- filtering region:1074, directly append 273 probes\n",
      "-- filtering region:1075, directly append 256 probes\n",
      "-- filtering region:1076, directly append 293 probes\n",
      "-- filtering region:1077, directly append 88 probes\n",
      "-- filtering region:1078, directly append 241 probes\n",
      "-- filtering region:1079, directly append 302 probes\n",
      "-- filtering region:108, directly append 213 probes\n",
      "-- filtering region:1080, directly append 362 probes\n",
      "-- filtering region:1081, directly append 298 probes\n",
      "-- filtering region:1082, directly append 263 probes\n",
      "-- filtering region:1083, directly append 325 probes\n",
      "-- filtering region:1085, directly append 341 probes\n",
      "-- filtering region:1086, directly append 28 probes\n",
      "-- filtering region:1087, directly append 336 probes\n",
      "-- filtering region:1088, directly append 25 probes\n",
      "-- filtering region:1089, directly append 207 probes\n",
      "-- filtering region:109, directly append 225 probes\n",
      "-- filtering region:1090, directly append 261 probes\n",
      "-- filtering region:1091, directly append 226 probes\n",
      "-- filtering region:1092, directly append 355 probes\n",
      "-- filtering region:1093, directly append 203 probes\n",
      "-- filtering region:1094, directly append 317 probes\n",
      "-- filtering region:1095, directly append 2 probes\n",
      "-- filtering region:1096, directly append 14 probes\n",
      "-- filtering region:1097, directly append 230 probes\n",
      "-- filtering region:11, directly append 212 probes\n",
      "-- filtering region:110, directly append 230 probes\n",
      "-- filtering region:1105, directly append 223 probes\n",
      "-- filtering region:1106, directly append 189 probes\n",
      "-- filtering region:1107, directly append 267 probes\n",
      "-- filtering region:1108, directly append 295 probes\n",
      "-- filtering region:1109, directly append 184 probes\n",
      "-- filtering region:111, directly append 358 probes\n",
      "-- filtering region:1110, directly append 336 probes\n",
      "-- filtering region:1111, directly append 381 probes\n",
      "-- filtering region:1112, directly append 398 probes\n",
      "-- filtering region:1113, directly append 266 probes\n",
      "-- filtering region:1114, directly append 217 probes\n",
      "-- filtering region:1115, directly append 253 probes\n",
      "-- filtering region:1116, directly append 184 probes\n",
      "-- filtering region:1117, directly append 288 probes\n",
      "-- filtering region:1118, directly append 231 probes\n",
      "-- filtering region:1119, directly append 328 probes\n",
      "-- filtering region:112, directly append 167 probes\n",
      "-- filtering region:1120, directly append 228 probes\n",
      "-- filtering region:1121, directly append 298 probes\n",
      "-- filtering region:1122, directly append 227 probes\n",
      "-- filtering region:1123, directly append 267 probes\n",
      "-- filtering region:1124, directly append 201 probes\n",
      "-- filtering region:1125, directly append 208 probes\n",
      "-- filtering region:1126, directly append 320 probes\n",
      "-- filtering region:1127, directly append 241 probes\n",
      "-- filtering region:1128, directly append 235 probes\n",
      "-- filtering region:1129, directly append 335 probes\n",
      "-- filtering region:1130, directly append 313 probes\n",
      "-- filtering region:1131, directly append 272 probes\n",
      "-- filtering region:1132, directly append 290 probes\n",
      "-- filtering region:1133, directly append 205 probes\n",
      "-- filtering region:1134, directly append 331 probes\n",
      "-- filtering region:1135, directly append 268 probes\n",
      "-- filtering region:1136, directly append 294 probes\n",
      "-- filtering region:1137, directly append 196 probes\n",
      "-- filtering region:1138, directly append 222 probes\n",
      "-- filtering region:1139, directly append 256 probes\n",
      "-- filtering region:114, directly append 215 probes\n",
      "-- filtering region:1140, directly append 242 probes\n",
      "-- filtering region:1141, directly append 375 probes\n",
      "-- filtering region:1142, directly append 263 probes\n",
      "-- filtering region:1143, directly append 206 probes\n",
      "-- filtering region:1144, directly append 122 probes\n",
      "-- filtering region:1145, directly append 215 probes\n",
      "-- filtering region:1146, directly append 94 probes\n",
      "-- filtering region:1147, directly append 257 probes\n",
      "-- filtering region:1148, directly append 101 probes\n",
      "-- filtering region:115, directly append 224 probes\n",
      "-- filtering region:1152, directly append 194 probes\n",
      "-- filtering region:1153, directly append 269 probes\n",
      "-- filtering region:1154, directly append 54 probes\n",
      "-- filtering region:1155, directly append 254 probes\n",
      "-- filtering region:1156, directly append 327 probes\n",
      "-- filtering region:1157, directly append 276 probes\n",
      "-- filtering region:1158, directly append 20 probes\n",
      "-- filtering region:116, directly append 228 probes\n",
      "-- filtering region:1160, directly append 317 probes\n",
      "-- filtering region:1161, directly append 3 probes\n",
      "-- filtering region:1162, directly append 222 probes\n",
      "-- filtering region:1163, directly append 272 probes\n",
      "-- filtering region:1164, directly append 3 probes\n",
      "-- filtering region:1165, directly append 270 probes\n",
      "-- filtering region:1166, directly append 342 probes\n",
      "-- filtering region:1167, directly append 192 probes\n",
      "-- filtering region:1168, directly append 346 probes\n",
      "-- filtering region:1169, directly append 185 probes\n",
      "-- filtering region:117, directly append 276 probes\n",
      "-- filtering region:1170, directly append 232 probes\n",
      "-- filtering region:1171, directly append 268 probes\n",
      "-- filtering region:1172, directly append 329 probes\n",
      "-- filtering region:1173, directly append 96 probes\n",
      "-- filtering region:1174, directly append 128 probes\n",
      "-- filtering region:1175, directly append 44 probes\n",
      "-- filtering region:1176, directly append 212 probes\n",
      "-- filtering region:1177, directly append 241 probes\n",
      "-- filtering region:1178, directly append 321 probes\n",
      "-- filtering region:1179, directly append 197 probes\n",
      "-- filtering region:118, directly append 216 probes\n",
      "-- filtering region:1180, directly append 213 probes\n",
      "-- filtering region:1181, directly append 271 probes\n",
      "-- filtering region:1182, directly append 291 probes\n",
      "-- filtering region:1183, directly append 337 probes\n",
      "-- filtering region:1184, directly append 310 probes\n",
      "-- filtering region:1185, directly append 4 probes\n",
      "-- filtering region:1186, directly append 265 probes\n",
      "-- filtering region:1187, directly append 285 probes\n",
      "-- filtering region:1188, directly append 77 probes\n",
      "-- filtering region:1189, directly append 171 probes\n",
      "-- filtering region:119, directly append 221 probes\n",
      "-- filtering region:1190, directly append 187 probes\n",
      "-- filtering region:1191, directly append 194 probes\n",
      "-- filtering region:1192, directly append 242 probes\n",
      "-- filtering region:1193, directly append 279 probes\n",
      "-- filtering region:1194, directly append 291 probes\n",
      "-- filtering region:1195, directly append 288 probes\n",
      "-- filtering region:1196, directly append 277 probes\n",
      "-- filtering region:1197, directly append 184 probes\n",
      "-- filtering region:1198, directly append 30 probes\n",
      "-- filtering region:1199, directly append 2 probes\n",
      "-- filtering region:12, directly append 240 probes\n",
      "-- filtering region:120, directly append 200 probes\n",
      "-- filtering region:1203, directly append 191 probes\n",
      "-- filtering region:1204, directly append 346 probes\n",
      "-- filtering region:1205, directly append 321 probes\n",
      "-- filtering region:1206, directly append 108 probes\n",
      "-- filtering region:1207, directly append 213 probes\n",
      "-- filtering region:1208, directly append 208 probes\n",
      "-- filtering region:1209, directly append 169 probes\n",
      "-- filtering region:121, directly append 295 probes\n",
      "-- filtering region:1210, directly append 270 probes\n",
      "-- filtering region:1211, directly append 302 probes\n",
      "-- filtering region:1212, directly append 278 probes\n",
      "-- filtering region:1213, directly append 293 probes\n",
      "-- filtering region:1214, directly append 317 probes\n",
      "-- filtering region:1215, directly append 221 probes\n",
      "-- filtering region:1216, directly append 216 probes\n",
      "-- filtering region:1217, directly append 190 probes\n",
      "-- filtering region:1218, directly append 235 probes\n",
      "-- filtering region:1219, directly append 315 probes\n",
      "-- filtering region:122, directly append 270 probes\n",
      "-- filtering region:1220, directly append 358 probes\n",
      "-- filtering region:1221, screen genome to keep 400 probes\n",
      "-- filtering region:1222, directly append 179 probes\n",
      "-- filtering region:1224, directly append 231 probes\n",
      "-- filtering region:1225, directly append 282 probes\n",
      "-- filtering region:1226, directly append 238 probes\n",
      "-- filtering region:1227, directly append 215 probes\n",
      "-- filtering region:1228, directly append 167 probes\n",
      "-- filtering region:1229, directly append 235 probes\n",
      "-- filtering region:123, directly append 182 probes\n",
      "-- filtering region:1230, directly append 223 probes\n",
      "-- filtering region:1231, directly append 241 probes\n",
      "-- filtering region:1232, directly append 237 probes\n",
      "-- filtering region:1233, directly append 223 probes\n",
      "-- filtering region:1234, directly append 169 probes\n",
      "-- filtering region:1235, directly append 241 probes\n",
      "-- filtering region:1236, directly append 271 probes\n",
      "-- filtering region:1237, directly append 228 probes\n",
      "-- filtering region:1238, directly append 287 probes\n",
      "-- filtering region:1239, directly append 249 probes\n",
      "-- filtering region:124, directly append 235 probes\n",
      "-- filtering region:1240, directly append 297 probes\n",
      "-- filtering region:1241, directly append 344 probes\n",
      "-- filtering region:1242, directly append 279 probes\n",
      "-- filtering region:1243, directly append 297 probes\n",
      "-- filtering region:1244, directly append 260 probes\n",
      "-- filtering region:1245, directly append 283 probes\n",
      "-- filtering region:1246, directly append 305 probes\n",
      "-- filtering region:1247, directly append 234 probes\n",
      "-- filtering region:1248, directly append 160 probes\n",
      "-- filtering region:1249, directly append 237 probes\n",
      "-- filtering region:125, directly append 251 probes\n",
      "-- filtering region:1250, directly append 310 probes\n",
      "-- filtering region:1251, directly append 181 probes\n",
      "-- filtering region:1252, directly append 181 probes\n",
      "-- filtering region:1253, directly append 246 probes\n",
      "-- filtering region:1254, directly append 178 probes\n",
      "-- filtering region:1255, directly append 303 probes\n",
      "-- filtering region:1256, directly append 186 probes\n",
      "-- filtering region:1257, directly append 173 probes\n",
      "-- filtering region:1258, directly append 168 probes\n",
      "-- filtering region:1259, directly append 216 probes\n",
      "-- filtering region:126, directly append 291 probes\n",
      "-- filtering region:1260, directly append 364 probes\n",
      "-- filtering region:1261, directly append 377 probes\n",
      "-- filtering region:1262, directly append 85 probes\n",
      "-- filtering region:1263, directly append 189 probes\n",
      "-- filtering region:1264, directly append 279 probes\n",
      "-- filtering region:1265, directly append 99 probes\n",
      "-- filtering region:1269, directly append 51 probes\n",
      "-- filtering region:127, directly append 222 probes\n",
      "-- filtering region:1270, directly append 305 probes\n",
      "-- filtering region:1271, directly append 312 probes\n",
      "-- filtering region:1272, directly append 317 probes\n",
      "-- filtering region:1273, directly append 297 probes\n",
      "-- filtering region:1274, directly append 192 probes\n",
      "-- filtering region:1275, directly append 356 probes\n",
      "-- filtering region:1276, directly append 244 probes\n",
      "-- filtering region:1277, directly append 362 probes\n",
      "-- filtering region:1278, directly append 150 probes\n",
      "-- filtering region:1279, directly append 193 probes\n",
      "-- filtering region:128, directly append 275 probes\n",
      "-- filtering region:1280, directly append 296 probes\n",
      "-- filtering region:1281, directly append 275 probes\n",
      "-- filtering region:1282, directly append 234 probes\n",
      "-- filtering region:1284, directly append 188 probes\n",
      "-- filtering region:1285, directly append 206 probes\n",
      "-- filtering region:1286, directly append 263 probes\n",
      "-- filtering region:1287, directly append 287 probes\n",
      "-- filtering region:1288, directly append 239 probes\n",
      "-- filtering region:1289, directly append 368 probes\n",
      "-- filtering region:129, directly append 216 probes\n",
      "-- filtering region:1290, directly append 315 probes\n",
      "-- filtering region:1291, directly append 273 probes\n",
      "-- filtering region:1292, directly append 248 probes\n",
      "-- filtering region:1293, directly append 312 probes\n",
      "-- filtering region:1294, directly append 90 probes\n",
      "-- filtering region:1295, directly append 167 probes\n",
      "-- filtering region:1296, directly append 1 probes\n",
      "-- filtering region:1297, directly append 312 probes\n",
      "-- filtering region:1298, directly append 282 probes\n",
      "-- filtering region:1299, directly append 267 probes\n",
      "-- filtering region:13, directly append 341 probes\n",
      "-- filtering region:130, directly append 162 probes\n",
      "-- filtering region:1300, directly append 266 probes\n",
      "-- filtering region:1301, directly append 252 probes\n",
      "-- filtering region:1302, directly append 220 probes\n",
      "-- filtering region:1303, directly append 174 probes\n",
      "-- filtering region:1304, directly append 268 probes\n",
      "-- filtering region:1305, directly append 297 probes\n",
      "-- filtering region:1306, directly append 285 probes\n",
      "-- filtering region:1307, directly append 271 probes\n",
      "-- filtering region:1308, directly append 243 probes\n",
      "-- filtering region:1309, directly append 325 probes\n",
      "-- filtering region:131, directly append 190 probes\n",
      "-- filtering region:1310, directly append 285 probes\n",
      "-- filtering region:1311, directly append 211 probes\n",
      "-- filtering region:1312, directly append 257 probes\n",
      "-- filtering region:1313, directly append 201 probes\n",
      "-- filtering region:1314, directly append 272 probes\n",
      "-- filtering region:1315, directly append 347 probes\n",
      "-- filtering region:1316, directly append 301 probes\n",
      "-- filtering region:1317, directly append 344 probes\n",
      "-- filtering region:1318, directly append 253 probes\n",
      "-- filtering region:1319, directly append 385 probes\n",
      "-- filtering region:132, directly append 192 probes\n",
      "-- filtering region:1320, directly append 286 probes\n",
      "-- filtering region:1321, directly append 321 probes\n",
      "-- filtering region:1322, directly append 310 probes\n",
      "-- filtering region:1323, directly append 306 probes\n",
      "-- filtering region:1324, directly append 321 probes\n",
      "-- filtering region:1325, directly append 367 probes\n",
      "-- filtering region:1329, directly append 35 probes\n",
      "-- filtering region:133, directly append 312 probes\n",
      "-- filtering region:1330, directly append 129 probes\n",
      "-- filtering region:1331, directly append 242 probes\n",
      "-- filtering region:1332, directly append 278 probes\n",
      "-- filtering region:1333, directly append 136 probes\n",
      "-- filtering region:1334, directly append 161 probes\n",
      "-- filtering region:1335, directly append 326 probes\n",
      "-- filtering region:1336, directly append 266 probes\n",
      "-- filtering region:1337, directly append 339 probes\n",
      "-- filtering region:1338, directly append 188 probes\n",
      "-- filtering region:1339, directly append 238 probes\n",
      "-- filtering region:134, directly append 237 probes\n",
      "-- filtering region:1340, directly append 344 probes\n",
      "-- filtering region:1341, directly append 308 probes\n",
      "-- filtering region:1342, directly append 261 probes\n",
      "-- filtering region:1343, directly append 201 probes\n",
      "-- filtering region:1344, directly append 274 probes\n",
      "-- filtering region:1345, directly append 312 probes\n",
      "-- filtering region:1346, directly append 298 probes\n",
      "-- filtering region:1347, directly append 179 probes\n",
      "-- filtering region:1348, directly append 130 probes\n",
      "-- filtering region:1349, directly append 302 probes\n",
      "-- filtering region:1350, directly append 265 probes\n",
      "-- filtering region:1351, directly append 353 probes\n",
      "-- filtering region:1352, directly append 243 probes\n",
      "-- filtering region:1353, directly append 192 probes\n",
      "-- filtering region:1354, directly append 278 probes\n",
      "-- filtering region:1355, directly append 187 probes\n",
      "-- filtering region:1356, directly append 291 probes\n",
      "-- filtering region:1357, directly append 248 probes\n",
      "-- filtering region:1358, directly append 327 probes\n",
      "-- filtering region:1359, directly append 267 probes\n",
      "-- filtering region:1360, directly append 324 probes\n",
      "-- filtering region:1361, directly append 397 probes\n",
      "-- filtering region:1362, directly append 385 probes\n",
      "-- filtering region:1363, directly append 314 probes\n",
      "-- filtering region:137, directly append 315 probes\n",
      "-- filtering region:1370, directly append 10 probes\n",
      "-- filtering region:1374, directly append 20 probes\n",
      "-- filtering region:1377, directly append 28 probes\n",
      "-- filtering region:1378, directly append 217 probes\n",
      "-- filtering region:1379, directly append 292 probes\n",
      "-- filtering region:138, directly append 285 probes\n",
      "-- filtering region:1380, directly append 192 probes\n",
      "-- filtering region:1381, directly append 214 probes\n",
      "-- filtering region:1382, directly append 210 probes\n",
      "-- filtering region:1383, directly append 230 probes\n",
      "-- filtering region:1384, directly append 148 probes\n",
      "-- filtering region:1385, directly append 163 probes\n",
      "-- filtering region:1386, directly append 128 probes\n",
      "-- filtering region:1387, directly append 190 probes\n",
      "-- filtering region:1388, directly append 176 probes\n",
      "-- filtering region:1389, directly append 251 probes\n",
      "-- filtering region:139, directly append 174 probes\n",
      "-- filtering region:1390, directly append 288 probes\n",
      "-- filtering region:1391, directly append 233 probes\n",
      "-- filtering region:1392, directly append 244 probes\n",
      "-- filtering region:1393, directly append 323 probes\n",
      "-- filtering region:1394, directly append 174 probes\n",
      "-- filtering region:1395, directly append 282 probes\n",
      "-- filtering region:1396, directly append 192 probes\n",
      "-- filtering region:1397, directly append 12 probes\n",
      "-- filtering region:1398, directly append 184 probes\n",
      "-- filtering region:1399, directly append 268 probes\n",
      "-- filtering region:14, directly append 176 probes\n",
      "-- filtering region:140, directly append 261 probes\n",
      "-- filtering region:1400, directly append 308 probes\n",
      "-- filtering region:1401, directly append 268 probes\n",
      "-- filtering region:1402, directly append 308 probes\n",
      "-- filtering region:1403, directly append 223 probes\n",
      "-- filtering region:1404, directly append 235 probes\n",
      "-- filtering region:1405, directly append 336 probes\n",
      "-- filtering region:1406, directly append 328 probes\n",
      "-- filtering region:1407, directly append 16 probes\n",
      "-- filtering region:1408, directly append 354 probes\n",
      "-- filtering region:1409, directly append 374 probes\n",
      "-- filtering region:141, directly append 74 probes\n",
      "-- filtering region:142, directly append 39 probes\n",
      "-- filtering region:1421, directly append 2 probes\n",
      "-- filtering region:1422, directly append 2 probes\n",
      "-- filtering region:1425, directly append 4 probes\n",
      "-- filtering region:1427, directly append 253 probes\n",
      "-- filtering region:1428, directly append 1 probes\n",
      "-- filtering region:1429, directly append 266 probes\n",
      "-- filtering region:143, directly append 186 probes\n",
      "-- filtering region:1430, directly append 240 probes\n",
      "-- filtering region:1431, directly append 10 probes\n",
      "-- filtering region:1433, directly append 28 probes\n",
      "-- filtering region:1434, directly append 295 probes\n",
      "-- filtering region:1435, directly append 255 probes\n",
      "-- filtering region:1436, directly append 298 probes\n",
      "-- filtering region:1437, directly append 360 probes\n",
      "-- filtering region:1438, directly append 174 probes\n",
      "-- filtering region:1439, directly append 198 probes\n",
      "-- filtering region:144, directly append 187 probes\n",
      "-- filtering region:1440, directly append 236 probes\n",
      "-- filtering region:1441, directly append 184 probes\n",
      "-- filtering region:1442, directly append 248 probes\n",
      "-- filtering region:1443, directly append 316 probes\n",
      "-- filtering region:1444, directly append 314 probes\n",
      "-- filtering region:1445, directly append 316 probes\n",
      "-- filtering region:1446, directly append 274 probes\n",
      "-- filtering region:1447, directly append 380 probes\n",
      "-- filtering region:1448, directly append 267 probes\n",
      "-- filtering region:1449, directly append 343 probes\n",
      "-- filtering region:145, directly append 178 probes\n",
      "-- filtering region:1450, directly append 211 probes\n",
      "-- filtering region:1451, directly append 255 probes\n",
      "-- filtering region:1452, directly append 113 probes\n",
      "-- filtering region:1453, directly append 304 probes\n",
      "-- filtering region:1454, directly append 277 probes\n",
      "-- filtering region:1455, directly append 290 probes\n",
      "-- filtering region:1456, directly append 270 probes\n",
      "-- filtering region:1457, directly append 281 probes\n",
      "-- filtering region:1458, directly append 396 probes\n",
      "-- filtering region:1459, directly append 326 probes\n",
      "-- filtering region:146, directly append 294 probes\n",
      "-- filtering region:1461, directly append 143 probes\n",
      "-- filtering region:1462, directly append 238 probes\n",
      "-- filtering region:1463, directly append 268 probes\n",
      "-- filtering region:1464, directly append 281 probes\n",
      "-- filtering region:1465, directly append 231 probes\n",
      "-- filtering region:1466, directly append 286 probes\n",
      "-- filtering region:1467, directly append 193 probes\n",
      "-- filtering region:1468, directly append 180 probes\n",
      "-- filtering region:1469, directly append 202 probes\n",
      "-- filtering region:147, directly append 355 probes\n",
      "-- filtering region:1470, directly append 194 probes\n",
      "-- filtering region:1471, directly append 221 probes\n",
      "-- filtering region:1472, directly append 278 probes\n",
      "-- filtering region:1473, directly append 117 probes\n",
      "-- filtering region:1474, directly append 173 probes\n",
      "-- filtering region:1475, directly append 234 probes\n",
      "-- filtering region:1476, directly append 197 probes\n",
      "-- filtering region:1477, directly append 341 probes\n",
      "-- filtering region:1478, directly append 230 probes\n",
      "-- filtering region:1479, directly append 216 probes\n",
      "-- filtering region:148, directly append 193 probes\n",
      "-- filtering region:1480, directly append 117 probes\n",
      "-- filtering region:1481, directly append 198 probes\n",
      "-- filtering region:1482, directly append 212 probes\n",
      "-- filtering region:1485, directly append 351 probes\n",
      "-- filtering region:1486, directly append 227 probes\n",
      "-- filtering region:1487, directly append 250 probes\n",
      "-- filtering region:1488, directly append 311 probes\n",
      "-- filtering region:1489, directly append 94 probes\n",
      "-- filtering region:149, directly append 309 probes\n",
      "-- filtering region:1490, directly append 128 probes\n",
      "-- filtering region:1491, directly append 239 probes\n",
      "-- filtering region:1492, directly append 238 probes\n",
      "-- filtering region:1493, directly append 129 probes\n",
      "-- filtering region:1494, directly append 189 probes\n",
      "-- filtering region:1495, directly append 199 probes\n",
      "-- filtering region:1496, directly append 72 probes\n",
      "-- filtering region:1497, directly append 230 probes\n",
      "-- filtering region:1498, directly append 146 probes\n",
      "-- filtering region:1499, directly append 160 probes\n",
      "-- filtering region:15, directly append 343 probes\n",
      "-- filtering region:150, directly append 240 probes\n",
      "-- filtering region:1500, directly append 369 probes\n",
      "-- filtering region:1501, directly append 253 probes\n",
      "-- filtering region:1502, directly append 139 probes\n",
      "-- filtering region:1503, directly append 186 probes\n",
      "-- filtering region:1504, directly append 285 probes\n",
      "-- filtering region:1505, directly append 239 probes\n",
      "-- filtering region:1506, directly append 163 probes\n",
      "-- filtering region:1507, directly append 242 probes\n",
      "-- filtering region:1508, directly append 189 probes\n",
      "-- filtering region:1509, directly append 188 probes\n",
      "-- filtering region:151, directly append 3 probes\n",
      "-- filtering region:1510, directly append 176 probes\n",
      "-- filtering region:1511, directly append 218 probes\n",
      "-- filtering region:1512, directly append 248 probes\n",
      "-- filtering region:1513, directly append 245 probes\n",
      "-- filtering region:1514, directly append 262 probes\n",
      "-- filtering region:1515, directly append 357 probes\n",
      "-- filtering region:1516, directly append 103 probes\n",
      "-- filtering region:1517, directly append 232 probes\n",
      "-- filtering region:1518, directly append 217 probes\n",
      "-- filtering region:1519, directly append 239 probes\n",
      "-- filtering region:152, directly append 248 probes\n",
      "-- filtering region:1520, directly append 221 probes\n",
      "-- filtering region:1521, directly append 176 probes\n",
      "-- filtering region:1525, directly append 51 probes\n",
      "-- filtering region:1526, directly append 60 probes\n",
      "-- filtering region:1527, directly append 72 probes\n",
      "-- filtering region:1528, directly append 44 probes\n",
      "-- filtering region:1529, directly append 166 probes\n",
      "-- filtering region:153, directly append 307 probes\n",
      "-- filtering region:1530, directly append 218 probes\n",
      "-- filtering region:1533, directly append 69 probes\n",
      "-- filtering region:1534, directly append 149 probes\n",
      "-- filtering region:1535, directly append 187 probes\n",
      "-- filtering region:1536, directly append 187 probes\n",
      "-- filtering region:1537, directly append 170 probes\n",
      "-- filtering region:1538, directly append 164 probes\n",
      "-- filtering region:154, directly append 283 probes\n",
      "-- filtering region:1540, directly append 1 probes\n",
      "-- filtering region:1541, directly append 251 probes\n",
      "-- filtering region:1542, directly append 204 probes\n",
      "-- filtering region:1543, directly append 30 probes\n",
      "-- filtering region:1547, directly append 2 probes\n",
      "-- filtering region:1548, directly append 106 probes\n",
      "-- filtering region:155, directly append 252 probes\n",
      "-- filtering region:156, directly append 292 probes\n",
      "-- filtering region:157, directly append 255 probes\n",
      "-- filtering region:158, directly append 227 probes\n",
      "-- filtering region:159, directly append 231 probes\n",
      "-- filtering region:16, directly append 245 probes\n",
      "-- filtering region:160, directly append 136 probes\n",
      "-- filtering region:161, directly append 134 probes\n",
      "-- filtering region:162, directly append 338 probes\n",
      "-- filtering region:163, directly append 234 probes\n",
      "-- filtering region:164, directly append 303 probes\n",
      "-- filtering region:165, directly append 244 probes\n",
      "-- filtering region:166, directly append 304 probes\n",
      "-- filtering region:167, directly append 217 probes\n",
      "-- filtering region:168, directly append 301 probes\n",
      "-- filtering region:169, directly append 238 probes\n",
      "-- filtering region:17, directly append 203 probes\n",
      "-- filtering region:170, directly append 303 probes\n",
      "-- filtering region:171, directly append 272 probes\n",
      "-- filtering region:172, directly append 200 probes\n",
      "-- filtering region:173, directly append 173 probes\n",
      "-- filtering region:174, directly append 198 probes\n",
      "-- filtering region:175, directly append 317 probes\n",
      "-- filtering region:176, directly append 152 probes\n",
      "-- filtering region:177, directly append 279 probes\n",
      "-- filtering region:178, directly append 280 probes\n",
      "-- filtering region:179, directly append 230 probes\n",
      "-- filtering region:18, directly append 204 probes\n",
      "-- filtering region:180, directly append 226 probes\n",
      "-- filtering region:181, directly append 248 probes\n",
      "-- filtering region:182, directly append 93 probes\n",
      "-- filtering region:183, directly append 204 probes\n",
      "-- filtering region:184, directly append 173 probes\n",
      "-- filtering region:185, directly append 313 probes\n",
      "-- filtering region:186, directly append 301 probes\n",
      "-- filtering region:187, directly append 239 probes\n",
      "-- filtering region:188, directly append 233 probes\n",
      "-- filtering region:189, directly append 191 probes\n",
      "-- filtering region:19, directly append 391 probes\n",
      "-- filtering region:190, directly append 150 probes\n",
      "-- filtering region:191, directly append 274 probes\n",
      "-- filtering region:192, directly append 270 probes\n",
      "-- filtering region:193, directly append 357 probes\n",
      "-- filtering region:194, directly append 297 probes\n",
      "-- filtering region:195, directly append 199 probes\n",
      "-- filtering region:196, directly append 317 probes\n",
      "-- filtering region:197, directly append 251 probes\n",
      "-- filtering region:198, directly append 379 probes\n",
      "-- filtering region:199, directly append 275 probes\n",
      "-- filtering region:2, directly append 377 probes\n",
      "-- filtering region:20, directly append 206 probes\n",
      "-- filtering region:200, directly append 229 probes\n",
      "-- filtering region:201, directly append 361 probes\n",
      "-- filtering region:202, directly append 331 probes\n",
      "-- filtering region:203, directly append 190 probes\n",
      "-- filtering region:204, directly append 265 probes\n",
      "-- filtering region:205, directly append 241 probes\n",
      "-- filtering region:206, directly append 262 probes\n",
      "-- filtering region:207, directly append 324 probes\n",
      "-- filtering region:208, directly append 174 probes\n",
      "-- filtering region:209, directly append 180 probes\n",
      "-- filtering region:21, directly append 255 probes\n",
      "-- filtering region:210, directly append 365 probes\n",
      "-- filtering region:211, directly append 349 probes\n",
      "-- filtering region:212, directly append 330 probes\n",
      "-- filtering region:213, directly append 200 probes\n",
      "-- filtering region:214, directly append 219 probes\n",
      "-- filtering region:215, directly append 142 probes\n",
      "-- filtering region:216, screen genome to keep 400 probes\n",
      "-- filtering region:217, directly append 278 probes\n",
      "-- filtering region:218, directly append 252 probes\n",
      "-- filtering region:219, directly append 258 probes\n",
      "-- filtering region:22, directly append 303 probes\n",
      "-- filtering region:220, directly append 214 probes\n",
      "-- filtering region:221, directly append 256 probes\n",
      "-- filtering region:222, directly append 292 probes\n",
      "-- filtering region:223, directly append 318 probes\n",
      "-- filtering region:224, directly append 322 probes\n",
      "-- filtering region:225, directly append 234 probes\n",
      "-- filtering region:226, directly append 220 probes\n",
      "-- filtering region:227, directly append 242 probes\n",
      "-- filtering region:228, directly append 148 probes\n",
      "-- filtering region:229, directly append 197 probes\n",
      "-- filtering region:23, directly append 333 probes\n",
      "-- filtering region:230, directly append 166 probes\n",
      "-- filtering region:232, directly append 23 probes\n",
      "-- filtering region:233, directly append 196 probes\n",
      "-- filtering region:234, directly append 255 probes\n",
      "-- filtering region:235, directly append 233 probes\n",
      "-- filtering region:236, directly append 182 probes\n",
      "-- filtering region:237, directly append 179 probes\n",
      "-- filtering region:238, directly append 251 probes\n",
      "-- filtering region:239, directly append 231 probes\n",
      "-- filtering region:24, directly append 279 probes\n",
      "-- filtering region:240, directly append 234 probes\n",
      "-- filtering region:241, directly append 272 probes\n",
      "-- filtering region:242, directly append 288 probes\n",
      "-- filtering region:243, directly append 165 probes\n",
      "-- filtering region:244, directly append 333 probes\n",
      "-- filtering region:245, directly append 238 probes\n",
      "-- filtering region:246, directly append 205 probes\n",
      "-- filtering region:247, directly append 234 probes\n",
      "-- filtering region:248, directly append 277 probes\n",
      "-- filtering region:249, directly append 242 probes\n",
      "-- filtering region:25, directly append 238 probes\n",
      "-- filtering region:250, directly append 172 probes\n",
      "-- filtering region:251, directly append 357 probes\n",
      "-- filtering region:252, directly append 287 probes\n",
      "-- filtering region:253, directly append 188 probes\n",
      "-- filtering region:254, directly append 282 probes\n",
      "-- filtering region:255, directly append 225 probes\n",
      "-- filtering region:256, directly append 218 probes\n",
      "-- filtering region:257, directly append 288 probes\n",
      "-- filtering region:258, directly append 314 probes\n",
      "-- filtering region:259, directly append 160 probes\n",
      "-- filtering region:26, directly append 328 probes\n",
      "-- filtering region:260, directly append 162 probes\n",
      "-- filtering region:261, directly append 159 probes\n",
      "-- filtering region:262, directly append 179 probes\n",
      "-- filtering region:263, directly append 324 probes\n",
      "-- filtering region:264, directly append 259 probes\n",
      "-- filtering region:265, directly append 220 probes\n",
      "-- filtering region:266, directly append 251 probes\n",
      "-- filtering region:267, directly append 263 probes\n",
      "-- filtering region:268, directly append 175 probes\n",
      "-- filtering region:269, directly append 351 probes\n",
      "-- filtering region:27, directly append 149 probes\n",
      "-- filtering region:270, directly append 313 probes\n",
      "-- filtering region:271, directly append 220 probes\n",
      "-- filtering region:272, directly append 232 probes\n",
      "-- filtering region:273, directly append 231 probes\n",
      "-- filtering region:274, directly append 330 probes\n",
      "-- filtering region:275, directly append 390 probes\n",
      "-- filtering region:276, directly append 325 probes\n",
      "-- filtering region:277, directly append 313 probes\n",
      "-- filtering region:278, directly append 224 probes\n",
      "-- filtering region:279, directly append 210 probes\n",
      "-- filtering region:28, directly append 262 probes\n",
      "-- filtering region:280, directly append 309 probes\n",
      "-- filtering region:281, directly append 239 probes\n",
      "-- filtering region:282, directly append 262 probes\n",
      "-- filtering region:283, directly append 293 probes\n",
      "-- filtering region:284, directly append 215 probes\n",
      "-- filtering region:285, directly append 234 probes\n",
      "-- filtering region:286, directly append 187 probes\n",
      "-- filtering region:287, directly append 194 probes\n",
      "-- filtering region:288, directly append 249 probes\n",
      "-- filtering region:289, directly append 250 probes\n",
      "-- filtering region:29, directly append 199 probes\n",
      "-- filtering region:290, directly append 253 probes\n",
      "-- filtering region:291, directly append 195 probes\n",
      "-- filtering region:292, directly append 226 probes\n",
      "-- filtering region:293, directly append 157 probes\n",
      "-- filtering region:295, directly append 209 probes\n",
      "-- filtering region:296, directly append 274 probes\n",
      "-- filtering region:297, directly append 194 probes\n",
      "-- filtering region:298, directly append 253 probes\n",
      "-- filtering region:299, directly append 141 probes\n",
      "-- filtering region:3, directly append 242 probes\n",
      "-- filtering region:30, directly append 263 probes\n",
      "-- filtering region:300, directly append 151 probes\n",
      "-- filtering region:301, directly append 25 probes\n",
      "-- filtering region:302, directly append 328 probes\n",
      "-- filtering region:303, directly append 213 probes\n",
      "-- filtering region:304, directly append 232 probes\n",
      "-- filtering region:305, directly append 222 probes\n",
      "-- filtering region:306, directly append 242 probes\n",
      "-- filtering region:307, directly append 231 probes\n",
      "-- filtering region:308, directly append 203 probes\n",
      "-- filtering region:309, directly append 343 probes\n",
      "-- filtering region:31, directly append 215 probes\n",
      "-- filtering region:310, directly append 166 probes\n",
      "-- filtering region:311, directly append 267 probes\n",
      "-- filtering region:312, directly append 241 probes\n",
      "-- filtering region:313, directly append 182 probes\n",
      "-- filtering region:314, directly append 253 probes\n",
      "-- filtering region:315, directly append 228 probes\n",
      "-- filtering region:316, directly append 183 probes\n",
      "-- filtering region:317, directly append 251 probes\n",
      "-- filtering region:318, directly append 178 probes\n",
      "-- filtering region:319, directly append 249 probes\n",
      "-- filtering region:32, directly append 190 probes\n",
      "-- filtering region:320, directly append 155 probes\n",
      "-- filtering region:321, directly append 279 probes\n",
      "-- filtering region:322, directly append 258 probes\n",
      "-- filtering region:323, directly append 248 probes\n",
      "-- filtering region:324, directly append 171 probes\n",
      "-- filtering region:325, directly append 176 probes\n",
      "-- filtering region:326, directly append 191 probes\n",
      "-- filtering region:327, directly append 106 probes\n",
      "-- filtering region:328, directly append 170 probes\n",
      "-- filtering region:329, directly append 295 probes\n",
      "-- filtering region:33, directly append 338 probes\n",
      "-- filtering region:330, directly append 218 probes\n",
      "-- filtering region:331, directly append 154 probes\n",
      "-- filtering region:332, directly append 377 probes\n",
      "-- filtering region:333, directly append 214 probes\n",
      "-- filtering region:334, directly append 234 probes\n",
      "-- filtering region:335, directly append 271 probes\n",
      "-- filtering region:336, directly append 244 probes\n",
      "-- filtering region:337, directly append 181 probes\n",
      "-- filtering region:338, directly append 222 probes\n",
      "-- filtering region:339, directly append 257 probes\n",
      "-- filtering region:34, directly append 230 probes\n",
      "-- filtering region:340, directly append 201 probes\n",
      "-- filtering region:341, directly append 284 probes\n",
      "-- filtering region:342, directly append 157 probes\n",
      "-- filtering region:343, directly append 260 probes\n",
      "-- filtering region:344, directly append 234 probes\n",
      "-- filtering region:345, directly append 225 probes\n",
      "-- filtering region:346, directly append 279 probes\n",
      "-- filtering region:347, directly append 202 probes\n",
      "-- filtering region:348, directly append 223 probes\n",
      "-- filtering region:349, directly append 161 probes\n",
      "-- filtering region:35, directly append 203 probes\n",
      "-- filtering region:350, screen genome to keep 400 probes\n",
      "-- filtering region:351, directly append 319 probes\n",
      "-- filtering region:352, directly append 325 probes\n",
      "-- filtering region:353, directly append 158 probes\n",
      "-- filtering region:354, directly append 293 probes\n",
      "-- filtering region:355, directly append 234 probes\n",
      "-- filtering region:356, directly append 257 probes\n",
      "-- filtering region:357, directly append 170 probes\n",
      "-- filtering region:358, directly append 216 probes\n",
      "-- filtering region:359, directly append 179 probes\n",
      "-- filtering region:36, directly append 154 probes\n",
      "-- filtering region:360, directly append 190 probes\n",
      "-- filtering region:361, directly append 175 probes\n",
      "-- filtering region:362, directly append 315 probes\n",
      "-- filtering region:363, directly append 350 probes\n",
      "-- filtering region:364, directly append 253 probes\n",
      "-- filtering region:365, directly append 284 probes\n",
      "-- filtering region:366, directly append 214 probes\n",
      "-- filtering region:367, directly append 102 probes\n",
      "-- filtering region:368, directly append 64 probes\n",
      "-- filtering region:37, directly append 358 probes\n",
      "-- filtering region:370, directly append 158 probes\n",
      "-- filtering region:371, directly append 84 probes\n",
      "-- filtering region:372, directly append 236 probes\n",
      "-- filtering region:373, directly append 270 probes\n",
      "-- filtering region:374, directly append 224 probes\n",
      "-- filtering region:375, directly append 226 probes\n",
      "-- filtering region:376, directly append 222 probes\n",
      "-- filtering region:377, directly append 340 probes\n",
      "-- filtering region:379, directly append 287 probes\n",
      "-- filtering region:38, directly append 211 probes\n",
      "-- filtering region:380, directly append 224 probes\n",
      "-- filtering region:381, directly append 278 probes\n",
      "-- filtering region:382, directly append 318 probes\n",
      "-- filtering region:383, directly append 176 probes\n",
      "-- filtering region:384, directly append 166 probes\n",
      "-- filtering region:385, directly append 272 probes\n",
      "-- filtering region:386, directly append 215 probes\n",
      "-- filtering region:387, directly append 192 probes\n",
      "-- filtering region:388, directly append 277 probes\n",
      "-- filtering region:389, directly append 170 probes\n",
      "-- filtering region:39, directly append 237 probes\n",
      "-- filtering region:390, directly append 164 probes\n",
      "-- filtering region:391, directly append 257 probes\n",
      "-- filtering region:392, directly append 202 probes\n",
      "-- filtering region:393, directly append 229 probes\n",
      "-- filtering region:394, directly append 309 probes\n",
      "-- filtering region:395, directly append 256 probes\n",
      "-- filtering region:396, directly append 182 probes\n",
      "-- filtering region:397, directly append 245 probes\n",
      "-- filtering region:398, directly append 162 probes\n",
      "-- filtering region:399, directly append 196 probes\n",
      "-- filtering region:4, directly append 256 probes\n",
      "-- filtering region:40, directly append 293 probes\n",
      "-- filtering region:400, directly append 222 probes\n",
      "-- filtering region:401, directly append 237 probes\n",
      "-- filtering region:402, directly append 191 probes\n",
      "-- filtering region:403, directly append 377 probes\n",
      "-- filtering region:404, directly append 331 probes\n",
      "-- filtering region:405, directly append 205 probes\n",
      "-- filtering region:406, directly append 261 probes\n",
      "-- filtering region:407, directly append 306 probes\n",
      "-- filtering region:408, directly append 163 probes\n",
      "-- filtering region:409, directly append 221 probes\n",
      "-- filtering region:41, directly append 22 probes\n",
      "-- filtering region:410, directly append 271 probes\n",
      "-- filtering region:411, directly append 282 probes\n",
      "-- filtering region:412, directly append 212 probes\n",
      "-- filtering region:413, directly append 339 probes\n",
      "-- filtering region:414, directly append 188 probes\n",
      "-- filtering region:415, directly append 252 probes\n",
      "-- filtering region:416, directly append 252 probes\n",
      "-- filtering region:417, directly append 315 probes\n",
      "-- filtering region:418, directly append 200 probes\n",
      "-- filtering region:419, directly append 355 probes\n",
      "-- filtering region:42, directly append 201 probes\n",
      "-- filtering region:420, directly append 37 probes\n",
      "-- filtering region:421, directly append 212 probes\n",
      "-- filtering region:422, directly append 298 probes\n",
      "-- filtering region:423, directly append 345 probes\n",
      "-- filtering region:424, directly append 323 probes\n",
      "-- filtering region:425, directly append 221 probes\n",
      "-- filtering region:426, directly append 350 probes\n",
      "-- filtering region:427, directly append 217 probes\n",
      "-- filtering region:428, directly append 243 probes\n",
      "-- filtering region:429, directly append 193 probes\n",
      "-- filtering region:43, directly append 235 probes\n",
      "-- filtering region:430, directly append 236 probes\n",
      "-- filtering region:431, directly append 237 probes\n",
      "-- filtering region:432, directly append 270 probes\n",
      "-- filtering region:434, directly append 5 probes\n",
      "-- filtering region:435, directly append 357 probes\n",
      "-- filtering region:436, directly append 294 probes\n",
      "-- filtering region:437, directly append 188 probes\n",
      "-- filtering region:438, directly append 206 probes\n",
      "-- filtering region:439, directly append 310 probes\n",
      "-- filtering region:44, directly append 317 probes\n",
      "-- filtering region:440, directly append 321 probes\n",
      "-- filtering region:441, directly append 183 probes\n",
      "-- filtering region:442, directly append 276 probes\n",
      "-- filtering region:443, directly append 241 probes\n",
      "-- filtering region:444, directly append 307 probes\n",
      "-- filtering region:446, directly append 108 probes\n",
      "-- filtering region:447, directly append 181 probes\n",
      "-- filtering region:448, directly append 127 probes\n",
      "-- filtering region:449, directly append 228 probes\n",
      "-- filtering region:45, directly append 187 probes\n",
      "-- filtering region:450, directly append 268 probes\n",
      "-- filtering region:451, directly append 258 probes\n",
      "-- filtering region:452, directly append 197 probes\n",
      "-- filtering region:453, directly append 67 probes\n",
      "-- filtering region:454, directly append 145 probes\n",
      "-- filtering region:455, directly append 142 probes\n",
      "-- filtering region:456, directly append 217 probes\n",
      "-- filtering region:457, directly append 207 probes\n",
      "-- filtering region:458, directly append 236 probes\n",
      "-- filtering region:459, directly append 180 probes\n",
      "-- filtering region:46, directly append 309 probes\n",
      "-- filtering region:460, directly append 202 probes\n",
      "-- filtering region:461, directly append 209 probes\n",
      "-- filtering region:462, directly append 170 probes\n",
      "-- filtering region:463, directly append 143 probes\n",
      "-- filtering region:464, directly append 211 probes\n",
      "-- filtering region:465, directly append 352 probes\n",
      "-- filtering region:466, directly append 189 probes\n",
      "-- filtering region:467, directly append 200 probes\n",
      "-- filtering region:468, directly append 222 probes\n",
      "-- filtering region:469, directly append 260 probes\n",
      "-- filtering region:47, directly append 213 probes\n",
      "-- filtering region:470, directly append 131 probes\n",
      "-- filtering region:471, directly append 199 probes\n",
      "-- filtering region:472, directly append 203 probes\n",
      "-- filtering region:473, directly append 203 probes\n",
      "-- filtering region:474, directly append 235 probes\n",
      "-- filtering region:475, directly append 263 probes\n",
      "-- filtering region:476, directly append 236 probes\n",
      "-- filtering region:477, directly append 215 probes\n",
      "-- filtering region:478, directly append 189 probes\n",
      "-- filtering region:479, directly append 287 probes\n",
      "-- filtering region:48, directly append 3 probes\n",
      "-- filtering region:480, directly append 257 probes\n",
      "-- filtering region:481, directly append 269 probes\n",
      "-- filtering region:482, directly append 230 probes\n",
      "-- filtering region:483, directly append 220 probes\n",
      "-- filtering region:484, directly append 265 probes\n",
      "-- filtering region:485, directly append 255 probes\n",
      "-- filtering region:486, directly append 294 probes\n",
      "-- filtering region:487, directly append 342 probes\n",
      "-- filtering region:488, directly append 3 probes\n",
      "-- filtering region:489, directly append 216 probes\n",
      "-- filtering region:490, directly append 384 probes\n",
      "-- filtering region:491, directly append 276 probes\n",
      "-- filtering region:492, directly append 229 probes\n",
      "-- filtering region:493, directly append 239 probes\n",
      "-- filtering region:494, directly append 293 probes\n",
      "-- filtering region:495, directly append 259 probes\n",
      "-- filtering region:496, directly append 167 probes\n",
      "-- filtering region:497, directly append 228 probes\n",
      "-- filtering region:498, directly append 262 probes\n",
      "-- filtering region:499, directly append 162 probes\n",
      "-- filtering region:5, directly append 303 probes\n",
      "-- filtering region:500, directly append 142 probes\n",
      "-- filtering region:501, directly append 304 probes\n",
      "-- filtering region:502, directly append 269 probes\n",
      "-- filtering region:503, directly append 318 probes\n",
      "-- filtering region:504, directly append 220 probes\n",
      "-- filtering region:505, directly append 308 probes\n",
      "-- filtering region:506, directly append 306 probes\n",
      "-- filtering region:507, directly append 202 probes\n",
      "-- filtering region:508, directly append 221 probes\n",
      "-- filtering region:509, directly append 272 probes\n",
      "-- filtering region:510, directly append 336 probes\n",
      "-- filtering region:511, directly append 184 probes\n",
      "-- filtering region:512, directly append 197 probes\n",
      "-- filtering region:514, directly append 2 probes\n",
      "-- filtering region:515, directly append 20 probes\n",
      "-- filtering region:516, directly append 227 probes\n",
      "-- filtering region:517, directly append 262 probes\n",
      "-- filtering region:518, directly append 264 probes\n",
      "-- filtering region:519, directly append 188 probes\n",
      "-- filtering region:520, directly append 212 probes\n",
      "-- filtering region:521, directly append 265 probes\n",
      "-- filtering region:522, directly append 187 probes\n",
      "-- filtering region:523, directly append 152 probes\n",
      "-- filtering region:524, directly append 196 probes\n",
      "-- filtering region:525, directly append 216 probes\n",
      "-- filtering region:526, directly append 223 probes\n",
      "-- filtering region:527, directly append 255 probes\n",
      "-- filtering region:528, directly append 249 probes\n",
      "-- filtering region:529, directly append 269 probes\n",
      "-- filtering region:530, directly append 223 probes\n",
      "-- filtering region:531, directly append 201 probes\n",
      "-- filtering region:532, directly append 232 probes\n",
      "-- filtering region:533, directly append 201 probes\n",
      "-- filtering region:534, directly append 181 probes\n",
      "-- filtering region:535, directly append 116 probes\n",
      "-- filtering region:536, directly append 259 probes\n",
      "-- filtering region:537, directly append 231 probes\n",
      "-- filtering region:538, directly append 214 probes\n",
      "-- filtering region:539, directly append 242 probes\n",
      "-- filtering region:540, directly append 179 probes\n",
      "-- filtering region:541, directly append 248 probes\n",
      "-- filtering region:542, directly append 227 probes\n",
      "-- filtering region:543, directly append 302 probes\n",
      "-- filtering region:544, directly append 276 probes\n",
      "-- filtering region:545, directly append 267 probes\n",
      "-- filtering region:546, directly append 305 probes\n",
      "-- filtering region:547, directly append 44 probes\n",
      "-- filtering region:548, directly append 191 probes\n",
      "-- filtering region:549, directly append 206 probes\n",
      "-- filtering region:550, directly append 378 probes\n",
      "-- filtering region:551, directly append 90 probes\n",
      "-- filtering region:552, directly append 264 probes\n",
      "-- filtering region:553, directly append 49 probes\n",
      "-- filtering region:554, directly append 306 probes\n",
      "-- filtering region:555, directly append 241 probes\n",
      "-- filtering region:556, directly append 234 probes\n",
      "-- filtering region:557, directly append 273 probes\n",
      "-- filtering region:558, directly append 251 probes\n",
      "-- filtering region:559, directly append 221 probes\n",
      "-- filtering region:560, directly append 261 probes\n",
      "-- filtering region:561, directly append 240 probes\n",
      "-- filtering region:562, directly append 249 probes\n",
      "-- filtering region:563, directly append 193 probes\n",
      "-- filtering region:564, directly append 270 probes\n",
      "-- filtering region:565, directly append 218 probes\n",
      "-- filtering region:566, directly append 169 probes\n",
      "-- filtering region:567, directly append 229 probes\n",
      "-- filtering region:568, directly append 222 probes\n",
      "-- filtering region:569, directly append 292 probes\n",
      "-- filtering region:57, directly append 60 probes\n",
      "-- filtering region:570, directly append 104 probes\n",
      "-- filtering region:571, directly append 356 probes\n",
      "-- filtering region:572, directly append 280 probes\n",
      "-- filtering region:573, directly append 206 probes\n",
      "-- filtering region:574, directly append 258 probes\n",
      "-- filtering region:575, directly append 204 probes\n",
      "-- filtering region:576, directly append 291 probes\n",
      "-- filtering region:577, directly append 207 probes\n",
      "-- filtering region:578, directly append 246 probes\n",
      "-- filtering region:579, directly append 151 probes\n",
      "-- filtering region:580, directly append 202 probes\n",
      "-- filtering region:581, directly append 250 probes\n",
      "-- filtering region:582, directly append 199 probes\n",
      "-- filtering region:583, directly append 181 probes\n",
      "-- filtering region:584, directly append 139 probes\n",
      "-- filtering region:585, directly append 241 probes\n",
      "-- filtering region:586, directly append 218 probes\n",
      "-- filtering region:587, directly append 156 probes\n",
      "-- filtering region:588, directly append 177 probes\n",
      "-- filtering region:589, directly append 275 probes\n",
      "-- filtering region:590, directly append 225 probes\n",
      "-- filtering region:591, directly append 283 probes\n",
      "-- filtering region:592, directly append 144 probes\n",
      "-- filtering region:593, directly append 306 probes\n",
      "-- filtering region:594, directly append 155 probes\n",
      "-- filtering region:595, directly append 180 probes\n",
      "-- filtering region:596, directly append 254 probes\n",
      "-- filtering region:597, directly append 277 probes\n",
      "-- filtering region:598, directly append 226 probes\n",
      "-- filtering region:599, directly append 304 probes\n",
      "-- filtering region:6, directly append 300 probes\n",
      "-- filtering region:60, directly append 187 probes\n",
      "-- filtering region:600, directly append 294 probes\n",
      "-- filtering region:601, directly append 299 probes\n",
      "-- filtering region:602, directly append 261 probes\n",
      "-- filtering region:603, directly append 251 probes\n",
      "-- filtering region:604, directly append 6 probes\n",
      "-- filtering region:605, directly append 187 probes\n",
      "-- filtering region:606, directly append 293 probes\n",
      "-- filtering region:607, directly append 281 probes\n",
      "-- filtering region:608, directly append 355 probes\n",
      "-- filtering region:609, directly append 323 probes\n",
      "-- filtering region:611, directly append 302 probes\n",
      "-- filtering region:612, directly append 311 probes\n",
      "-- filtering region:613, directly append 200 probes\n",
      "-- filtering region:614, directly append 277 probes\n",
      "-- filtering region:615, directly append 258 probes\n",
      "-- filtering region:616, directly append 173 probes\n",
      "-- filtering region:617, directly append 254 probes\n",
      "-- filtering region:618, directly append 192 probes\n",
      "-- filtering region:619, directly append 282 probes\n",
      "-- filtering region:62, directly append 330 probes\n",
      "-- filtering region:620, directly append 196 probes\n",
      "-- filtering region:621, directly append 248 probes\n",
      "-- filtering region:622, directly append 247 probes\n",
      "-- filtering region:623, directly append 185 probes\n",
      "-- filtering region:624, directly append 132 probes\n",
      "-- filtering region:625, directly append 168 probes\n",
      "-- filtering region:626, directly append 118 probes\n",
      "-- filtering region:627, directly append 253 probes\n",
      "-- filtering region:628, directly append 273 probes\n",
      "-- filtering region:629, directly append 360 probes\n",
      "-- filtering region:63, directly append 241 probes\n",
      "-- filtering region:630, directly append 37 probes\n",
      "-- filtering region:631, directly append 21 probes\n",
      "-- filtering region:632, directly append 9 probes\n",
      "-- filtering region:64, directly append 199 probes\n",
      "-- filtering region:641, directly append 5 probes\n",
      "-- filtering region:643, directly append 1 probes\n",
      "-- filtering region:645, directly append 257 probes\n",
      "-- filtering region:646, directly append 241 probes\n",
      "-- filtering region:647, directly append 219 probes\n",
      "-- filtering region:648, directly append 215 probes\n",
      "-- filtering region:649, directly append 262 probes\n",
      "-- filtering region:65, directly append 114 probes\n",
      "-- filtering region:650, directly append 241 probes\n",
      "-- filtering region:651, directly append 146 probes\n",
      "-- filtering region:652, directly append 306 probes\n",
      "-- filtering region:653, directly append 373 probes\n",
      "-- filtering region:654, directly append 271 probes\n",
      "-- filtering region:655, screen genome to keep 400 probes\n",
      "-- filtering region:656, directly append 292 probes\n",
      "-- filtering region:657, directly append 186 probes\n",
      "-- filtering region:658, directly append 311 probes\n",
      "-- filtering region:659, directly append 54 probes\n",
      "-- filtering region:66, directly append 127 probes\n",
      "-- filtering region:660, directly append 278 probes\n",
      "-- filtering region:662, directly append 184 probes\n",
      "-- filtering region:663, directly append 205 probes\n",
      "-- filtering region:664, directly append 255 probes\n",
      "-- filtering region:665, directly append 266 probes\n",
      "-- filtering region:666, directly append 273 probes\n",
      "-- filtering region:667, directly append 172 probes\n",
      "-- filtering region:668, directly append 328 probes\n",
      "-- filtering region:669, directly append 312 probes\n",
      "-- filtering region:67, directly append 271 probes\n",
      "-- filtering region:670, directly append 311 probes\n",
      "-- filtering region:671, directly append 199 probes\n",
      "-- filtering region:672, directly append 171 probes\n",
      "-- filtering region:673, directly append 120 probes\n",
      "-- filtering region:674, directly append 258 probes\n",
      "-- filtering region:675, directly append 327 probes\n",
      "-- filtering region:676, directly append 330 probes\n",
      "-- filtering region:677, directly append 308 probes\n",
      "-- filtering region:678, directly append 347 probes\n",
      "-- filtering region:679, directly append 320 probes\n",
      "-- filtering region:68, directly append 226 probes\n",
      "-- filtering region:680, directly append 294 probes\n",
      "-- filtering region:681, directly append 308 probes\n",
      "-- filtering region:682, directly append 147 probes\n",
      "-- filtering region:683, directly append 285 probes\n",
      "-- filtering region:684, directly append 253 probes\n",
      "-- filtering region:685, directly append 328 probes\n",
      "-- filtering region:686, directly append 266 probes\n",
      "-- filtering region:687, directly append 78 probes\n",
      "-- filtering region:688, directly append 278 probes\n",
      "-- filtering region:689, directly append 160 probes\n",
      "-- filtering region:69, directly append 128 probes\n",
      "-- filtering region:690, directly append 289 probes\n",
      "-- filtering region:691, directly append 270 probes\n",
      "-- filtering region:692, directly append 189 probes\n",
      "-- filtering region:693, directly append 246 probes\n",
      "-- filtering region:694, directly append 236 probes\n",
      "-- filtering region:695, directly append 246 probes\n",
      "-- filtering region:696, directly append 170 probes\n",
      "-- filtering region:697, directly append 220 probes\n",
      "-- filtering region:698, directly append 227 probes\n",
      "-- filtering region:699, directly append 66 probes\n",
      "-- filtering region:7, directly append 341 probes\n",
      "-- filtering region:70, directly append 120 probes\n",
      "-- filtering region:701, directly append 289 probes\n",
      "-- filtering region:702, directly append 291 probes\n",
      "-- filtering region:703, directly append 181 probes\n",
      "-- filtering region:704, directly append 284 probes\n",
      "-- filtering region:705, directly append 292 probes\n",
      "-- filtering region:706, directly append 246 probes\n",
      "-- filtering region:707, directly append 182 probes\n",
      "-- filtering region:708, directly append 213 probes\n",
      "-- filtering region:709, directly append 289 probes\n",
      "-- filtering region:71, directly append 137 probes\n",
      "-- filtering region:710, directly append 234 probes\n",
      "-- filtering region:711, directly append 290 probes\n",
      "-- filtering region:712, directly append 201 probes\n",
      "-- filtering region:713, directly append 236 probes\n",
      "-- filtering region:714, directly append 196 probes\n",
      "-- filtering region:715, directly append 353 probes\n",
      "-- filtering region:716, directly append 136 probes\n",
      "-- filtering region:717, directly append 282 probes\n",
      "-- filtering region:718, directly append 391 probes\n",
      "-- filtering region:719, directly append 376 probes\n",
      "-- filtering region:72, directly append 267 probes\n",
      "-- filtering region:720, directly append 218 probes\n",
      "-- filtering region:721, directly append 270 probes\n",
      "-- filtering region:722, directly append 309 probes\n",
      "-- filtering region:723, directly append 73 probes\n",
      "-- filtering region:724, directly append 288 probes\n",
      "-- filtering region:725, directly append 271 probes\n",
      "-- filtering region:726, directly append 209 probes\n",
      "-- filtering region:727, directly append 138 probes\n",
      "-- filtering region:728, directly append 365 probes\n",
      "-- filtering region:729, directly append 200 probes\n",
      "-- filtering region:73, directly append 198 probes\n",
      "-- filtering region:730, directly append 379 probes\n",
      "-- filtering region:731, directly append 174 probes\n",
      "-- filtering region:732, directly append 296 probes\n",
      "-- filtering region:733, directly append 274 probes\n",
      "-- filtering region:734, directly append 225 probes\n",
      "-- filtering region:735, directly append 206 probes\n",
      "-- filtering region:736, directly append 362 probes\n",
      "-- filtering region:737, directly append 252 probes\n",
      "-- filtering region:738, directly append 222 probes\n",
      "-- filtering region:739, directly append 229 probes\n",
      "-- filtering region:74, directly append 173 probes\n",
      "-- filtering region:740, directly append 249 probes\n",
      "-- filtering region:741, directly append 266 probes\n",
      "-- filtering region:742, directly append 353 probes\n",
      "-- filtering region:743, directly append 309 probes\n",
      "-- filtering region:744, directly append 308 probes\n",
      "-- filtering region:745, directly append 367 probes\n",
      "-- filtering region:747, directly append 23 probes\n",
      "-- filtering region:748, directly append 254 probes\n",
      "-- filtering region:749, directly append 199 probes\n",
      "-- filtering region:75, directly append 149 probes\n",
      "-- filtering region:750, directly append 287 probes\n",
      "-- filtering region:751, directly append 293 probes\n",
      "-- filtering region:752, directly append 307 probes\n",
      "-- filtering region:753, directly append 145 probes\n",
      "-- filtering region:754, directly append 344 probes\n",
      "-- filtering region:755, directly append 59 probes\n",
      "-- filtering region:756, directly append 319 probes\n",
      "-- filtering region:757, directly append 192 probes\n",
      "-- filtering region:758, directly append 141 probes\n",
      "-- filtering region:759, directly append 306 probes\n",
      "-- filtering region:76, directly append 121 probes\n",
      "-- filtering region:760, directly append 220 probes\n",
      "-- filtering region:761, directly append 222 probes\n",
      "-- filtering region:762, directly append 141 probes\n",
      "-- filtering region:763, directly append 270 probes\n",
      "-- filtering region:764, directly append 261 probes\n",
      "-- filtering region:765, directly append 191 probes\n",
      "-- filtering region:766, directly append 254 probes\n",
      "-- filtering region:767, directly append 199 probes\n",
      "-- filtering region:768, directly append 278 probes\n",
      "-- filtering region:769, directly append 160 probes\n",
      "-- filtering region:77, directly append 218 probes\n",
      "-- filtering region:770, directly append 81 probes\n",
      "-- filtering region:773, directly append 46 probes\n",
      "-- filtering region:774, directly append 202 probes\n",
      "-- filtering region:775, directly append 202 probes\n",
      "-- filtering region:776, directly append 317 probes\n",
      "-- filtering region:777, directly append 373 probes\n",
      "-- filtering region:778, directly append 252 probes\n",
      "-- filtering region:779, directly append 208 probes\n",
      "-- filtering region:78, directly append 142 probes\n",
      "-- filtering region:780, directly append 349 probes\n",
      "-- filtering region:782, directly append 206 probes\n",
      "-- filtering region:783, directly append 286 probes\n",
      "-- filtering region:784, directly append 196 probes\n",
      "-- filtering region:785, directly append 295 probes\n",
      "-- filtering region:786, directly append 234 probes\n",
      "-- filtering region:787, directly append 293 probes\n",
      "-- filtering region:788, directly append 182 probes\n",
      "-- filtering region:789, directly append 209 probes\n",
      "-- filtering region:79, directly append 198 probes\n",
      "-- filtering region:790, directly append 91 probes\n",
      "-- filtering region:791, directly append 164 probes\n",
      "-- filtering region:792, directly append 168 probes\n",
      "-- filtering region:793, directly append 231 probes\n",
      "-- filtering region:794, directly append 298 probes\n",
      "-- filtering region:795, directly append 126 probes\n",
      "-- filtering region:796, directly append 289 probes\n",
      "-- filtering region:797, directly append 249 probes\n",
      "-- filtering region:798, directly append 244 probes\n",
      "-- filtering region:799, directly append 289 probes\n",
      "-- filtering region:8, directly append 323 probes\n",
      "-- filtering region:80, directly append 247 probes\n",
      "-- filtering region:800, directly append 196 probes\n",
      "-- filtering region:801, directly append 320 probes\n",
      "-- filtering region:802, directly append 257 probes\n",
      "-- filtering region:803, directly append 289 probes\n",
      "-- filtering region:804, directly append 220 probes\n",
      "-- filtering region:805, directly append 168 probes\n",
      "-- filtering region:806, directly append 291 probes\n",
      "-- filtering region:807, directly append 233 probes\n",
      "-- filtering region:808, directly append 197 probes\n",
      "-- filtering region:809, screen genome to keep 400 probes\n",
      "-- filtering region:81, directly append 223 probes\n",
      "-- filtering region:810, directly append 160 probes\n",
      "-- filtering region:811, directly append 234 probes\n",
      "-- filtering region:812, directly append 358 probes\n",
      "-- filtering region:813, directly append 187 probes\n",
      "-- filtering region:814, directly append 275 probes\n",
      "-- filtering region:815, directly append 332 probes\n",
      "-- filtering region:816, directly append 209 probes\n",
      "-- filtering region:817, directly append 204 probes\n",
      "-- filtering region:819, directly append 268 probes\n",
      "-- filtering region:82, directly append 8 probes\n",
      "-- filtering region:820, directly append 255 probes\n",
      "-- filtering region:821, directly append 142 probes\n",
      "-- filtering region:822, directly append 257 probes\n",
      "-- filtering region:823, directly append 129 probes\n",
      "-- filtering region:824, directly append 288 probes\n",
      "-- filtering region:825, directly append 225 probes\n",
      "-- filtering region:826, directly append 263 probes\n",
      "-- filtering region:827, directly append 283 probes\n",
      "-- filtering region:828, directly append 377 probes\n",
      "-- filtering region:829, directly append 237 probes\n",
      "-- filtering region:83, directly append 259 probes\n",
      "-- filtering region:832, directly append 168 probes\n",
      "-- filtering region:833, directly append 204 probes\n",
      "-- filtering region:834, directly append 310 probes\n",
      "-- filtering region:835, directly append 236 probes\n",
      "-- filtering region:836, directly append 164 probes\n",
      "-- filtering region:837, directly append 302 probes\n",
      "-- filtering region:838, directly append 226 probes\n",
      "-- filtering region:839, directly append 194 probes\n",
      "-- filtering region:84, directly append 191 probes\n",
      "-- filtering region:840, directly append 264 probes\n",
      "-- filtering region:841, directly append 373 probes\n",
      "-- filtering region:843, directly append 148 probes\n",
      "-- filtering region:844, directly append 256 probes\n",
      "-- filtering region:845, directly append 300 probes\n",
      "-- filtering region:846, directly append 261 probes\n",
      "-- filtering region:847, directly append 174 probes\n",
      "-- filtering region:848, directly append 294 probes\n",
      "-- filtering region:849, directly append 123 probes\n",
      "-- filtering region:85, directly append 313 probes\n",
      "-- filtering region:850, directly append 181 probes\n",
      "-- filtering region:851, directly append 241 probes\n",
      "-- filtering region:852, directly append 279 probes\n",
      "-- filtering region:853, directly append 194 probes\n",
      "-- filtering region:854, directly append 215 probes\n",
      "-- filtering region:855, directly append 199 probes\n",
      "-- filtering region:856, directly append 163 probes\n",
      "-- filtering region:857, directly append 300 probes\n",
      "-- filtering region:858, directly append 252 probes\n",
      "-- filtering region:859, directly append 240 probes\n",
      "-- filtering region:86, directly append 237 probes\n",
      "-- filtering region:860, directly append 144 probes\n",
      "-- filtering region:861, directly append 333 probes\n",
      "-- filtering region:862, directly append 233 probes\n",
      "-- filtering region:863, directly append 282 probes\n",
      "-- filtering region:864, directly append 250 probes\n",
      "-- filtering region:865, directly append 223 probes\n",
      "-- filtering region:866, directly append 216 probes\n",
      "-- filtering region:867, directly append 164 probes\n",
      "-- filtering region:868, directly append 289 probes\n",
      "-- filtering region:869, directly append 301 probes\n",
      "-- filtering region:87, directly append 236 probes\n",
      "-- filtering region:870, directly append 324 probes\n",
      "-- filtering region:871, directly append 329 probes\n",
      "-- filtering region:872, directly append 360 probes\n",
      "-- filtering region:873, directly append 265 probes\n",
      "-- filtering region:874, directly append 370 probes\n",
      "-- filtering region:875, directly append 212 probes\n",
      "-- filtering region:876, directly append 262 probes\n",
      "-- filtering region:877, directly append 268 probes\n",
      "-- filtering region:878, directly append 347 probes\n",
      "-- filtering region:88, directly append 204 probes\n",
      "-- filtering region:888, directly append 215 probes\n",
      "-- filtering region:889, directly append 279 probes\n",
      "-- filtering region:89, directly append 335 probes\n",
      "-- filtering region:890, directly append 205 probes\n",
      "-- filtering region:891, directly append 169 probes\n",
      "-- filtering region:892, directly append 314 probes\n",
      "-- filtering region:893, directly append 252 probes\n",
      "-- filtering region:894, directly append 308 probes\n",
      "-- filtering region:895, directly append 294 probes\n",
      "-- filtering region:896, directly append 158 probes\n",
      "-- filtering region:897, directly append 286 probes\n",
      "-- filtering region:898, directly append 182 probes\n",
      "-- filtering region:899, directly append 211 probes\n",
      "-- filtering region:9, directly append 264 probes\n",
      "-- filtering region:90, directly append 118 probes\n",
      "-- filtering region:900, directly append 251 probes\n",
      "-- filtering region:901, directly append 230 probes\n",
      "-- filtering region:902, directly append 271 probes\n",
      "-- filtering region:903, directly append 144 probes\n",
      "-- filtering region:904, directly append 320 probes\n",
      "-- filtering region:905, directly append 311 probes\n",
      "-- filtering region:906, directly append 194 probes\n",
      "-- filtering region:907, directly append 143 probes\n",
      "-- filtering region:908, directly append 216 probes\n",
      "-- filtering region:909, directly append 181 probes\n",
      "-- filtering region:91, directly append 363 probes\n",
      "-- filtering region:910, directly append 155 probes\n",
      "-- filtering region:911, directly append 187 probes\n",
      "-- filtering region:912, directly append 230 probes\n",
      "-- filtering region:913, directly append 169 probes\n",
      "-- filtering region:914, directly append 176 probes\n",
      "-- filtering region:915, directly append 166 probes\n",
      "-- filtering region:916, directly append 250 probes\n",
      "-- filtering region:917, directly append 250 probes\n",
      "-- filtering region:918, directly append 259 probes\n",
      "-- filtering region:919, directly append 157 probes\n",
      "-- filtering region:92, directly append 271 probes\n",
      "-- filtering region:920, directly append 187 probes\n",
      "-- filtering region:921, directly append 173 probes\n",
      "-- filtering region:922, directly append 174 probes\n",
      "-- filtering region:923, directly append 141 probes\n",
      "-- filtering region:924, directly append 236 probes\n",
      "-- filtering region:925, directly append 214 probes\n",
      "-- filtering region:926, directly append 293 probes\n",
      "-- filtering region:927, directly append 314 probes\n",
      "-- filtering region:928, directly append 334 probes\n",
      "-- filtering region:929, directly append 261 probes\n",
      "-- filtering region:93, directly append 212 probes\n",
      "-- filtering region:930, directly append 266 probes\n",
      "-- filtering region:931, directly append 249 probes\n",
      "-- filtering region:932, directly append 297 probes\n",
      "-- filtering region:933, directly append 224 probes\n",
      "-- filtering region:934, directly append 209 probes\n",
      "-- filtering region:935, screen genome to keep 400 probes\n",
      "-- filtering region:94, directly append 178 probes\n",
      "-- filtering region:949, directly append 160 probes\n",
      "-- filtering region:95, directly append 214 probes\n",
      "-- filtering region:950, directly append 178 probes\n",
      "-- filtering region:951, directly append 188 probes\n",
      "-- filtering region:952, directly append 190 probes\n",
      "-- filtering region:953, directly append 245 probes\n",
      "-- filtering region:954, directly append 90 probes\n",
      "-- filtering region:955, directly append 174 probes\n",
      "-- filtering region:956, directly append 265 probes\n",
      "-- filtering region:957, directly append 258 probes\n",
      "-- filtering region:958, directly append 328 probes\n",
      "-- filtering region:959, directly append 213 probes\n",
      "-- filtering region:96, directly append 258 probes\n",
      "-- filtering region:960, directly append 279 probes\n",
      "-- filtering region:961, directly append 237 probes\n",
      "-- filtering region:962, directly append 226 probes\n",
      "-- filtering region:963, directly append 195 probes\n",
      "-- filtering region:964, directly append 149 probes\n",
      "-- filtering region:965, directly append 191 probes\n",
      "-- filtering region:966, directly append 161 probes\n",
      "-- filtering region:967, directly append 229 probes\n",
      "-- filtering region:968, directly append 228 probes\n",
      "-- filtering region:969, directly append 156 probes\n",
      "-- filtering region:970, directly append 110 probes\n",
      "-- filtering region:971, directly append 173 probes\n",
      "-- filtering region:972, directly append 392 probes\n",
      "-- filtering region:973, directly append 270 probes\n",
      "-- filtering region:974, directly append 209 probes\n",
      "-- filtering region:975, directly append 315 probes\n",
      "-- filtering region:976, directly append 194 probes\n",
      "-- filtering region:977, directly append 297 probes\n",
      "-- filtering region:978, directly append 130 probes\n",
      "-- filtering region:979, directly append 255 probes\n",
      "-- filtering region:98, directly append 168 probes\n",
      "-- filtering region:980, directly append 173 probes\n",
      "-- filtering region:981, directly append 302 probes\n",
      "-- filtering region:982, directly append 171 probes\n",
      "-- filtering region:983, directly append 236 probes\n",
      "-- filtering region:984, directly append 269 probes\n",
      "-- filtering region:985, directly append 342 probes\n",
      "-- filtering region:986, directly append 233 probes\n",
      "-- filtering region:987, directly append 374 probes\n",
      "-- filtering region:988, directly append 348 probes\n",
      "-- filtering region:989, directly append 154 probes\n",
      "-- filtering region:990, directly append 281 probes\n",
      "-- filtering region:991, directly append 247 probes\n",
      "-- filtering region:992, directly append 237 probes\n",
      "-- filtering region:993, directly append 153 probes\n",
      "-- filtering region:994, directly append 286 probes\n",
      "-- filtering region:995, directly append 337 probes\n",
      "-- filtering region:996, directly append 255 probes\n",
      "-- filtering region:997, directly append 304 probes\n",
      "-- filtering region:999, directly append 341 probes\n",
      "region: 1021 removed with 11 probes\n",
      "region: 1026 removed with 17 probes\n",
      "region: 1027 removed with 3 probes\n",
      "region: 1028 removed with 1 probes\n",
      "region: 1086 removed with 28 probes\n",
      "region: 1088 removed with 25 probes\n",
      "region: 1095 removed with 2 probes\n",
      "region: 1096 removed with 14 probes\n",
      "region: 1158 removed with 20 probes\n",
      "region: 1161 removed with 3 probes\n",
      "region: 1164 removed with 3 probes\n",
      "region: 1175 removed with 44 probes\n",
      "region: 1185 removed with 4 probes\n",
      "region: 1198 removed with 30 probes\n",
      "region: 1199 removed with 2 probes\n",
      "region: 1296 removed with 1 probes\n",
      "region: 1329 removed with 35 probes\n",
      "region: 1370 removed with 10 probes\n",
      "region: 1374 removed with 20 probes\n",
      "region: 1377 removed with 28 probes\n",
      "region: 1397 removed with 12 probes\n",
      "region: 1407 removed with 16 probes\n",
      "region: 142 removed with 39 probes\n",
      "region: 1421 removed with 2 probes\n",
      "region: 1422 removed with 2 probes\n",
      "region: 1425 removed with 4 probes\n",
      "region: 1428 removed with 1 probes\n",
      "region: 1431 removed with 10 probes\n",
      "region: 1433 removed with 28 probes\n",
      "region: 151 removed with 3 probes\n",
      "region: 1528 removed with 44 probes\n",
      "region: 1540 removed with 1 probes\n",
      "region: 1543 removed with 30 probes\n",
      "region: 1547 removed with 2 probes\n",
      "region: 232 removed with 23 probes\n",
      "region: 301 removed with 25 probes\n",
      "region: 41 removed with 22 probes\n",
      "region: 420 removed with 37 probes\n",
      "region: 434 removed with 5 probes\n",
      "region: 48 removed with 3 probes\n",
      "region: 488 removed with 3 probes\n",
      "region: 514 removed with 2 probes\n",
      "region: 515 removed with 20 probes\n",
      "region: 547 removed with 44 probes\n",
      "region: 553 removed with 49 probes\n",
      "region: 604 removed with 6 probes\n",
      "region: 630 removed with 37 probes\n",
      "region: 631 removed with 21 probes\n",
      "region: 632 removed with 9 probes\n",
      "region: 641 removed with 5 probes\n",
      "region: 643 removed with 1 probes\n",
      "region: 747 removed with 23 probes\n",
      "region: 773 removed with 46 probes\n",
      "region: 82 removed with 8 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts\n",
    "reload(library_tools.quality_check)\n",
    "\n",
    "num_probes_kept = 400 # max num of probes kept for each region\n",
    "filtered_pb_dict = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dict, num_probes_kept,\n",
    "                                                                  hit_type='genome')\n",
    "\n",
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 50\n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dict.items():\n",
    "    if len(_pb_class.kept_probes) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.kept_probes)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dict[_rid])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "id": "5571da09",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1352"
      ]
     },
     "execution_count": 114,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(filtered_pb_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "id": "4b910471",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAApQAAAFKCAYAAACjE8/+AAAAAXNSR0IArs4c6QAAIABJREFUeF7sXQW4VUXXXnSDdHd3p3SDoiiK7aeIiigGJiEgqSAmBoqiYIAddHd3dyMp3eX/vPvcOXeduTtmn7P3uefyzzzP93xyz+zZM++ePfPuNWu9K9l///33H+miEdAIaAQ0AhoBjYBGQCOgEQgTgWSaUIaJnL5MI6AR0AhoBDQCGgGNgEbAQEATSj0RNAIaAY2ARkAjoBHQCGgEIkJAE8qI4NMXawQ0AhoBjYBGQCOgEdAIaEKp54BGQCOgEdAIaAQ0AhoBjUBECGhCGRF8+mKNgEZAI6AR0AhoBDQCGgFNKPUc0AhoBDQCGgGNgEZAI6ARiAgBTSgjgk9frBHQCGgENAIaAY2ARkAjoAmlngMaAY2ARkAjoBHQCGgENAIRIaAJZUTw6Ys1AhoBjYBGQCOgEdAIaAQ0odRzQCOgEdAIaAQ0AhoBjYBGICIENKGMCD59sUZAI6AR0AhoBDQCGgGNgCaUeg5oBDQCGgGNgEZAI6AR0AhEhIAmlBHBpy/WCGgENAIaAY2ARkAjoBHQhFLPAY2ARkAjoBHQCGgENAIagYgQ0IQyIvj0xRoBjYBGQCOgEdAIaAQ0AppQ6jmgEdAIaAQ0AhoBjYBGQCMQEQKaUEYEn75YI6AR0AhoBDQCGgGNgEZAE0o9BzQCGgGNgEZAI6AR0AhoBCJCQBPKiODTF2sENAIaAY2ARkAjoBHQCGhCqeeARkAjoBHQCGgENAIaAY1ARAhoQhkRfPpijYBGQCOgEdAIaAQ0AhoBTSj1HNAIaAQ0AhoBjYBGQCOgEYgIAU0oI4JPX6wR0AhoBDQCGgGNgEZAI6AJpZ4DGgGNgEZAI6AR0AhoBDQCESGgCWVE8OmLNQIaAY2ARkAjoBHQCGgENKHUc0AjoBHQCGgENAIaAY2ARiAiBDShjAg+fbFGQCOgEdAIaAQ0AhoBjYAmlHoOaAQ0AhoBjYBGQCOgEdAIRISAJpQRwacv1ghoBDQCGgGNgEZAI6AR0IRSzwGNgEZAI6AR0AhoBDQCGoGIENCEMiL49MUaAY2ARkAjoBHQCGgENAKaUOo5oBHQCGgENAIaAY2ARkAjEBECmlBGBJ++WCOgEdAIaAQ0AhoBjYBGQBNKPQc0AhoBjYBGQCOgEdAIaAQiQkATyojg0xdrBDQCGgGNgEZAI6AR0AhoQqnngEZAI6AR0AhoBDQCGgGNQEQIaEIZEXz6Yo2ARkAjoBHQCGgENAIaAU0o9RzQCGgENAIaAY2ARkAjoBGICAFNKCOCL/7iS5cu0fr16ylnzpyUMmVKj1rVzWgENAIaAY2ARkAjEKsIXLt2jY4dO0YVK1aktGnTxmo3o9IvTSg9gnn58uVUq1Ytj1rTzWgENAIaAY2ARkAjkFQQWLZsGdWsWTOpdNeXfmpC6RGse/bsoaJFixImVd68eT1qVTejEdAIaAQ0AhoBjUCsInDo0CHDmLR7924qUqRIrHYzKv3ShNIjmA8cOEAFCxak/fv3U4ECBTxqVTejEdAIaAQ0AhoBjUCsIqD3/vgnowmlR7NUTyqPgNTNaAQ0AhoBjYBGIIkgoPd+TSg9n6p6UnkOqW5QI6AR0AhoBDQCMY2A3vs1ofR8gupJ5TmkukGNgEZAI6AR0AjENAJ679eE0vMJqieV55DqBjUCGgGNgEZAIxDTCOi9XxNKzyeonlSeQ6ob1AhoBDQCGgGNQEwjoPd+TSg9n6B6UnkOqW5QI6AR0AhoBDQCMY2A3vs1ofR8gupJ5TmkukGNgEZAI6AR0AjENAJ679eE0vMJqieV55DqBjUCGgGNgEZAIxDTCOi9XxNKzyeonlSeQ6ob1AhoBDQCGgGNQEwjoPd+TSg9n6B6UnkOqW5QI6AR0AhoBDQCMY2A3vs1ofR8gupJ5TmkukGNgEZAI6AR0AjENAJ679eE0vMJqieV55DqBjUCGgGNgEZAIxDTCOi9XxNKzyeonlSeQ6ob1AhoBDQCGgGNQEwjoPd+TSg9n6B6UnkOqW5QI6AR0AhoBDQCMY2A3vtvEkJ57tw5KlOmDB08eJCWL19ONWrUCI5s0qRJ1KtXL9q8eTMVKFCAunfvTl27dk0wMd99910aMWIEHT58mCpWrEjDhg2jxo0bu57AelK5hkxfoBHQCGgENAIagSSNgN77bxJC+frrr9O3335LR44cCSGUixcvpoYNG9Kjjz5KDz/8MC1cuJD69u1LI0eOpM6dOwdHDzLZs2dPGjx4MFWrVo2+/PJL+uOPP2jZsmUGuXRT9KRyg5auqxHQCGgENAIagaSPgN77bwJCuWXLFsMiOXz4cOrSpUsIoWzTpg2dOHGCli5dGhzpU089RRMmTCA8/OTJk9Ply5cpd+7chL8PHTrUqHf9+nWDSFaqVInGjRvnaqbrSeUKLl1ZI6AR0AhoBDQCSR4BvfffBISyZcuWBvG7/fbbqUmTJkFCCaKYOXNmevvtt+mll14KjnTu3LnGUfaKFSuoevXqNHv2bGratCmtWrWKqlatGqz31ltvGST19OnTlCxZMuXJrieVMlS6okZAI6AR0AhoBG4KBPTen8QJ5S+//ELPPvssbd++3SCEnFBu2rSJypcvT5MnT6bWrVsHR3rs2DHKlSsXjR071jgG//TTT402Lly4QOnSpQvW+/nnn6ljx460f/9+w/dStehJpYqUN/UuXb1O0zcdoTrFslPOTGm8aVS3EvMIzN9+jLKmT00V8meJ+b5adfC///6jGZuPUrGcGah4zoxJdhw3W8evXr9B0zYeoaqFbqF8t8TvCTfLOG/cwLw7QiVyZaRiSWDebT9ylvaduEBNy+RyZdyJ9vPSe38SJpQggAjE6devH3Xq1InmzJkTQijhL1m/fn2CH2WdOnWCI7127RqlSpWKPvzwQ3r++edp0KBBNGDAALp06VLI/JsxYwa1aNGC1q5da1hArcqZM2cI/xPl0KFDVKtWLddENNqTP9z7nbpwhU5duEpFcmQItwlPr3vzjw00dsleypslLS3u0czTtnVjsYnAqn0n6e5PFxmd2zKgNaVNlSI2O+rQq4nrDtGzP6wyau15+7YkOYabsdMfzdxO703fRimSJ6Odg9vedEP8ffUBemn82iQz74q8MdHo66hHa1Dzcrlj9nloQpmECSWCaKZPn274R8IX0opQLlmyhGrXrp2AUH700UfUrVs3g1AOHDiQLl68GDJR0TaO09etW2cbmANCi+Nxubi1bMbsW8I6BotKsZ6T6L//iKa91JBK5c6U6N0Wi01S3pSvXLtBgydtpor5s1CH6urWcDfgn798jTKkSenmkpitKzZ8dHBpz2aUO3PamO2rXce6fr+SJq0/nGQ29iQJchidrjFwOh0/d+WmfS6Pj15Gs7ceSzLjE2t8xxoFaOg9lcN4otG5RBPKJEoo9+7dS6VKlaLff/+d6tWrZ4xiwYIF1K5dO8MnEkE6+/btc3XkDUKZNm38xqR65P3/yUJ58cp1KttnioH3Uw2LUc+2ZaPzptrc5WYglN8u2kN9/9pojHL3kLaeH+tMXh+whL3YvBQ936xkoj+zSDvw8cztNHz6NqOZpEwo8UxgpUzKH0ORPstYvF4Tyth6KppQxtbzUOlNsv9gfkoiRVgjrboLiySCb3RQjrcPFMfdVfpPNxrt1rQEvdyytLc3CKO1m4FQ9v1zA327eK8x+s39W1O61N4e4d4MGPGpMWLWdnp3WoBQLunRjPJkSZoWyud+WEUTNKEM463395IaA2fQ8XOXb1qiry2U/swfbaGMxzVJEcpTp07RmjVrQmYF/o1o7s8//5xq1qxp6ElCNgh14UcpCqSF/vrrrwSyQfg7IsJRIBsEv0lIB2nZoHiYD5++RHWGzDT+8ErLUvRc08S3dt0MZOntyVvo87k7DVyX92rueXBRUsEI37S7j5+nItkzUPLk1soKn8zeQcOmbjXwWtyjKeXNkjQDJ7r9uJr+XvvPTUtc/Nm2/W/Vb0J55tJVwmlPYrlq3IyEcvHOf+nTOTvo9dZlEi1QTxPKJEoozZYU2YfS2GzihM0fe+wxeuihhwxh8z59+lgKmw8ZMsQgoqNGjaLffvtNC5tLQO85fp4avzvH+GuvtmXpyYbF/F/dHe6QVMiS3TA+nLGd3p8RsLjNfbUxFc7ubcBTUsHoi3k7afCkLdShWgEa3tHaV8ovQnno9EUaOXcX3VO9QFQ2ped/XE1/aUKZ6GuI3AE/CeX1G/9Rlbem0dnL12j+a02oYLb0UR9/UiWU99UoSO/cYx4gK9Y4fIfuGpI4AW6aUN7khBLDQ+pFBPDw1IuQCeIFlhGRehHZdmCZhMg5ZIjclpt5Um05fIZafzDfgKT/neXp0bpF3MLjef2kQpbsBi6IFOpMfL4+lc/nrRROUsFItZ+cUC56o6ln0i7tPl5A6w+ejprF8IVxq+nPNQELpR++s56/bP9PGvSTUP577jJVHzjDQBL+zN1blIo6qp2+WU6zthyN2jyPZIDYm4v2mGQ0oUIoUS+xFBNu5r3f7TNMUkfebgcXzfo386Ras/8Utf9koQHn23dXpPtrFYomtKb3UiUhid5Rmw5A9gjyRyg/PV2XahXN5ml3kwpGqv3khHLhG00pv0dagar39+rhvDhuNf0RRyh3DW5re8zv1T11O84I+Ekocdxdqd80oxNPNyxGPRIhsDGpEkq7KO9ov7tms+hm3vud35rQGppQukXMon4sTqoT569Q+tQpItbrW7LrX7r/iyXGyN+/rzLdVdUfiRs3jyIWFhLRX1gf+vy5kVpVyEN3VM6nPIyfV+ynV39ZZ9Qf/XhNalI6l/K1KhVjCSO7/qr2E75SQ6cEfCgXvN6ECmT15thQ9f4qmKvUeWn8Gvp99UGjKvQOoXuYVMvRs5coe4Y0SXoMAvuag2bQsbP+BOWcvniVKr8VIJT/q1uY3rqzQtQfeVIilBBhh1QdiiaUUZ8qYd9QE8qwoQu9MNYI5b5/L1DDYbOpYLZ0NO/VJhFJ0szZepQeG73cGPAnD1aj2yrl9Qi18JuJNgmw62m4MjC/rDxAr/wcEBr+9KFq1Lait7jGEkZeEMrP5uykd6ZsSfKEsvv4NfRbHKHcPqgNpUqRPPwXIRGvXLjjOD00aik1KpWTvu1UKxF74s2tfSWUF65S5f4BQml3hOvNSMxbSUqEEj6nxeMIpT7y9nNWeNu2JpQe4ek3oYRPCcSQi+fKQGXyZHbsdZ8/N9CYOEmaSK0gUzYcpi7frTTu+fnD1ah1BW+Jj+NgTCrEEllqNnwO7Tx23uilGz+en5bvp9d+DVgoRzxYlW6vpG7dVMEsljDShDIegZd/Wku/rjpg/GHrwNaUJqW3clEqc8OLOo2HzaY9/15wPe+9uLcfbfhJKLn02p1V8tGH91f1Ywi2bSYlQnnt+g0q0WuyIwGPhTXO770/6hMlghtqQhkBePxSvycVokIRHapKWrwklH+uOUgvjAvINX32UDVq47ElLZxHEAsLieh30+FzaFcYhHLcsn30xm/rjWY+eqCqq+NyM8xmbzlqRI33ub0c1SiSjWIJI1VCiSAVBMgg33D61KEZfiCxBKklFC8jZaONE6zSsE6jJOUUkk3enWPIPamuSeG859G8xk9CCfejagMCWr6tyuemkY/UMP5717FzRiaraEgJJSVCibzqJTWhjOb09+RemlB6AiMZ+pYFCxb0LZf3s9+voonr1bNriFzXGF6kFspfVx6gl308mg3nEURKAk5fuEpZ0qcK59YJrmn67hzaFcbG+sPSfdTz9wCh/PD+KnRnlfwR9UfGJFKMIuqMi4t5Pwe2r0C9/9hgmqN95NydNCQJEUqkvcRxduqUoUfar/68ln6OI5Sb+rdKQJxdQBes6uV8Vr2/14Ry3rZj9Meag4amYDQIljzOWoNm0FGffCh5lLdwEQAZB4Yo0bBU2xFKnICduXjNszVRdQ5Z1dOEMlIEE+d6TSg9wt1vQsnFkFUiQ3v/sZ6+W7LPGN2OQW0oZQR+Wj+t2E+vxQWPqPhQYnFKlszfQINIyNLwaVvp41k7PCFxwDdcQsmjvN/rWJnurhZZsNPNQCgzpUlpaPWhfPloDWpRLnfwDeWEEn7BhbLHblAOCN6t78yiHBlT08yXG4cErbz2y1r6aUXAQrnhrVaUMcJc60LPdPi9lX3LCW+2TIY7762WXDF/6xXPTj88WcejlVm9GT8JJTLwIIocpXbRbDT+6brkl0+w1YjtCCU+bPGBO6ZTLWpYKqc6aD7VvHLtBpXqrY+8fYLXt2Y1ofQIWr8JJZcaUfma5YQyUsf/H5ftox5xR7MfP1CV2tlEMh85c4k6fLbIcNQfdFdFj9BN2EwkhDKSa73cWMcs3mNEh6O8e29lQ1jbqSDTxuytRwmb7i3pU4dUv9kIJQbHfVK5bmesE8qvF+ym/hM2Gc9n9iuNqWiOeNH6139ZR+NX7Dd+W9evJWVOG5ml3Ov57DQHxe9+Eco0KZPT1oFtVLvhWT0/CSWix3GkjlKl4C30x7O3Rp1QPvHNcpppoUMp5lC6VClo84DWnmEabkOXr12n0r2nGJeHE5SzfM8JwjyqVOCWcLugfJ3fe79yR2KgoiaUHj0EvycVjwxVsWr0+n09fb80YKGMlFB+t2SvcQyJ4uTr1/2nNfTbqoAkipsAFbePIZJNNJJrzfoZ7sY6euFueuvvAOkYek8l6lijoCMMAt8K+TPThG4Nbj5CmTYlnb0UsFDKc+jLebto0KTNxt+9zCzk9XxA/+wI5Ru/rqNxywOEcm2flhEfM/rRf8eJCMt8mL7DVm2LcaRNlZy2DIguoeRC2n6sXZBXqjUokL62csFb6E8PCCVIKtaDluVy0yMKySbcEEroZsJnv06x7NSlUXGV6eBpnUtXr1OZNwOE8v6aBentDvaZcvgz464EK3s3p+wZ03jaN7kxv/d+XzvvceOaUHoEqN+TikeGrn6zBWXNEGqdkochjjDw920D2yTw43IzbG5Jc/L1e2rMCpq26UgCMuDmfip1I9lEI7nWrG/hEsqvFuymAXFWrHc6VKT7ajoLxtv1/aawUCoSyjmvNKYizOqnMmeciIyXRMKOUPb4bR39uCxAKFXeZaexeT2fne4nfr+ZCOXKvSeow2eLg0P3+mMYJze1B3tLKLt+v9JQ/lCdtyqEErrFm/q3pv5/b6KvF+5WbhsVQQJTp0juiVB/JITyj9UH6cXxgSDSX7rUNQIU/Sx+7/1+9t3rtjWh9AhRvycVd+Rf1rMZ5cqc1rbnOKLGUbUXhJJvjjcToUyZPBntGNw24hkQbnDCqPm7aODEgMVtyN0V6QGFDETRJJQIkkB0avuqkQULOQHMx5TJhlByvJISoZz1ciMqljNjEAb+bnphQbnZCGViHLtCIeHxbwJau6oEzWle898Pn75EdYaEEspIhfrbfjifNh06o9xfN4Ty6bEraOpGdcPA0TOXqPl7c6lC/iye+L9GQignrPuHnvshoIjy45N1qG7x7G4eleu6fu/9rjuUiBdoQukR+H5PKu53pZIlhG9aKj6XdjDwjfyD+6rYEoykZKHMnDYlrevXynTo3y/dS1/M20Uf3V/VOKKyK1yPD759ON5S+SrmQSaD7qpAD9Uu7Dgbo0UoEVQihJj9SAvJBxoOoZT9Eh2Bs6ngByHj7gwyoeSnByt6N6ccER7J+dF/FTzD1V+1ajsx/fhmbTlCnb5Z4RuhPHT6ItUdMstov3KBLPTnc/WJE0qeSvTspau0Yu9Jw0/aTqO0zYfzabPHhDJD6hS0sX9rcruO9/1zA30bp3usEjTqNL/gK162T3hH3lw3eewTtahBSX+DjPze+52wiqXfNaH06Gn4Pak4QVTZTPmxWqRad1z/zyn1otsv23DhD3cT5b5SeTKnpSU9m5l2QbSfJV0qWtu3pTKhFBVVjlp4lOeA9hXokTqxQyi5H1Lv28pS5wbFwn1UjtfxZwmSf8bEhxJHhtBuHDY1kHpRJmmON4khQskD5pb1aka5MtmfNjiNLdx3waldp99vJkI5c/MReuJb/wjlP6cuUr23QwnlW39vpNEL9xgwc0LZ/pOFtGb/KXq4TiEa2N46sLH1B/Noy+GzxvUqR/TcQgm9V67EIeYQFAfgo//kmBU03YXrEp/TXhDKC1euUbk+U42xufWh5M9y9GM1qUkZb1Payu+F33u/03sYS79rQunR0/B7UvEgm2kvNaRSuTPZ9pw7/kdKKD+ZvSO4kTvJ28Q6oeTRg8VyZKBZrzS2JZQqizW3UIrGOt1alPq0K2f7jDiub91Rnv5Xr4jjbIyWhTKWCOWOo2ep+XvzQrCZ+XIjKs6OkR2BS0RCKfeVa8Qu7dksYs3FxCKUOOLccfScMqFxekaJaaH0m1AePHWRbmWEcsSD1ajB0NlBSPipk+rzbPn+XNp2RB1/Tihl0hcpoeT7k4ruMTLh2EnZQcO1fN/wCCVPFfzFI9WpZfk8TlMvot/93vsj6lyUL9aE0iPA/Z5UfBOa0K2+4atiV9wQSljtIJqe75Z0VK1Q1gTNfjRzO703fZvxdyd5m1gnlDwFWrm8mWnSC4FI6a2HzxoZWtpXyWcsdKqLOq5tNGw27Y1LQSfAe6J+UXrzdntC+fHM7TQ8Dte+7crR47cWdZyN/x8JJbfOC4BinVB+s3A39YuL4Jf7yrNYLe7RlPJmSef43O0quJmrEd1IurjFe3Np+01CKGdsOkKdx/hnodx/4kKQQOLIu16JHIZskCgQ9C+cPb1xPKv6PN3izwmlTPrEPaEDu/6tVtT52xU0Y7O6DyUnlE66x8j6Bi3Wfu3K0/0WfuPnLl+jCmESSpFjHthGI7Ob33u/l++s321pQukRwn5PKu6j8lvXeqbEjw+F+1wWyZ6eXmlV2jJXNP+iM5MYen/6Nvpw5naj+WH3VKJ7beRtYp1Q8owVFfNnob+71TfGJRZUkECQQdVFPRJCKQSp0Ya4r9N05P2ClWHlvpNUOk8mQ8vQyyjvWLFQQl7l99UH6ZtFgaNBUWZ0b2SkZ/SiuHnWqvezI5T8XeZHnapty/X86L9KX9xayJzaFOMQkcZO9SP9feM/pylbhtQEQo/jXRzziqJyhOzm/k6EUrS16I2mwaNx/M2uH24txJ2/XU4zNh81biWTvoSEMr6uChb8iNxJpk5lvkZCKJfu+pfu+2KJMU4nmTs3z9Cqrt97vxd9jFYbmlB6hLTfk6rfXxuDm+r4p+pQ7WL2kWs8G4fTIjlk0mYaOW+XUW3jW62M3LK8DJ602QhQQXHSS4x1QskFhoVzPMYlFrnqhbPSr8/Uiwqh5ERd1U+RL8YgoZAdKpYzA816ufFNSSihSfho3SLB+Sfm5YzuDalELnu3D9VXW2WDU21L1OOEUia/nFCa5SRHVpVs6VMry6/40X+V8SZlQrn+wGlqN2KBMUxY63Dk/dTYlb4Ryn3/XqCGwwJH3JUKZKFbJQuluPEPT9amB79cqtQPt7JNnFAu79WccmaK12cMEsq0KWl9P1go3RFKPgedZOpU5isCkyr2m2bg4NaHcuXek0ZyDRQnFy2Vee5Ux++93+n+sfS7JpQePQ2/JxWIA3QLUb57ojbVL5nDtueuCOXkzTRyboAwru/XkjKxzB03bvxHxXpOCt5raIdK1LGmtQB3l7EracrGgDaa7PjtEdRGMyqLktn9IG9RK04PTmSs4O3VKJyVfnFJKBsOnU37TlwIuV3n+kWpt8ORt0gBiQt7ti1DTzV0FhDm44Yl58KV68Z9YUXw0kK55/h5ahyXZ5iT3SW7/qXlu0/Qkw2LUdpUKTx5pLzfZg12a1rCSJXJi4ofMerDxeHrhXuodfk8VC5fZtP+hjuX7Ab/7aI91PevQBYkmVDyj0M54w+kmh79ehm1qZCHPnu4uhK+bvo/ZcMhOnjqEnW6tUjE6VH9IpQi0lgevJcpXQdN3ERfzg+sp/Axn7vtGD3tI6Hc++95ajQskLfbllB2rk0PjlIklO/OoV3Hzwfff6fJwkki6nJXDDGHhPKFncSQ2XMp2iN+j3BSFVGZrxBWrxRHKB+oVZCG3K0ubM4JpZMBxAkzld/93vtV+hArdTSh9OhJ+D2p+AKoErnGdSvFEK2OLt6evIUQyY2CVHDIVIKXEpswjh6qDZgeRMlJgJsTSi+i/awej8qiZHYt14OrWugW+r3rrcT9p2oWyUo/d3FnoTQjlE82KEq9brP3oRw2dQt9MjuA+xttyihlpODjhvXu0tUbwQ3FiVBevX6Dpm48TJUL3EIFs9nnwbYilOIezzctQd1blvbk7XEilC+3KBX0NRU3nPpiQ+Oo36moWMzDnUvhEkoe3SvraSI9H6zoKCpHjahn1f/5248ZecKrxvlF88hZOU+6E45mv7d6fx5tPaIeZex0DzEOM0KJdKMvjltjvCcqeq24F7QMMd9rF81OebKERtLLhHLO1mPU5Tv/LJT8fQKhrF8iB33KfCgFNj+4IJRu9W9lQnlX1fz0/n1VjFvLhNIu77f8HK/f+I+KM6ODUxCoyvt2+uJVqvxWwELpllCu2HOC7vk8IFI/+K6K9GBt54QRTnPT7ne/9/5I+hbtazWh9AhxvycVt1CqRK698vNaQ2aFF6sNauiULcHFDangqg6YRjf+I4Jl6KmGxYJHD2jr7bsrWjpS43dOKFWi/cKFX2VRMmuby3eYHW+HQygbDJ1F+09cDLmdFaFElPm2w+cIqROHTt0adMx/rXVp6tq4hCMcfNypUyanK9fUCSWXKXIiK3wD7NW2rGGR5BuPsOQ6dlihghOh7NGmDA2ZvCWkpSkvNqAyecwtjryiyjxRqaMwjJAqoRbK0ON5noVElj/yilBu+ucMtf1ovtEnyF5B/gpH6TUGBvJJv9S8FL3QvKTbYYXU94tQCukat89RHoxYMyGWj2NcXhISyqPU5btVwSpO74db4Lhfnx2hHP3tHVW9AAAgAElEQVR4TXp8tJrAOg8GVOmvTCjvrpqf3vOAUHIRcuDiCaFkOrhuj7yRx/veOELZ/87yhsuMn8Xvvd/PvnvdtiaUHiHq96Tix2QjH6lOrRykEIr1mGiQQl6sFh1uKVvTpwVV6R+wSN6SPhXBx0v4suBvTl98z3y3kiZvCBx5OzlnRwJ9uCTgwMkLVP+dgC+T2fG234QSx5k41nylZSnDEix8V/Hv55o6b/AhhDJFcrpyXZ1QcnkjPhdguULKNC7j4UQoBU6RPENxrROh7NeuXDBiWlwz+YUGVDZvZIQS7hwXr14PypOgbZWNWWXMPF2p7O/JPw7l43CvCCW/v3APcEsov1uyl3YdO0+9bitLKZInSzBstzqITriJeeAVobRbIwZO2ESj4lyINvfHkbd/hFImXAgGbFDS3EKJ7F3X2MJtNx/5h6zKvFUhlEJ39/HRy2j21mPGI3NqmwfQoD7wTJfa2h1GZe3mahxuCeWy3Seo48iAhVI12NFpbtr97vfeH0nfon2tJpQeIe73pOLCsWZSCFi0xi7eS7WLZaMzF6/Rw1/F++GIIVotDNyXb9WbLYJH3CCU815rEvRlQTtOGV04oXRyzo4EepVFyax9Hm1Zq0g2+qlL3ZAjQ7O/OS2oZhZKWHZ7ti2boAu837BiCj+u7i1K0fPN3BFKvvmo+FCaEUocryIXOQJ7/nj21qBfndORdzQJJT5ikF2Gl4nP16fy+eyls1Dfbp488tVSWrHnpEEqnd4Rt3PVjlBy65ggmzji+2HpPnpnSrwl1mnemRFycQ2//4gHq9KBkxcNFxbhF/ti85L0YvNSlsPimUqslB1inVDiGBbHsWakiJP6Tf1b0dytx+iZ7/2xUEKSrNUH8TqqIJQNS+UIurvYzS27OVD/nVnGc1UhfajjhlA+NnoZwQ1ApW1O/lAfeKZPHRrYyceosnbzNt0eeXNrsKpvutv3m9f3e++PpG/RvlYTSo8Q93tS8cw3nzxYjW6rlDek5/zY2mpIVovTe9O20kdxQQ9IBSeOxUAo577aJOjLgnahl/awTUYXTiidnLMjgV5lUTJrnzvH1yqajZBWkLdl9jenjd2MUD7dsBj1cCCUED//emEgMMBpgzcjD7Aa8Q3TyYfSjFAOsQjIcrJQCpwieYZmYzJrD9qncOHgRUWLFfWL9phI/8VZ6uXnaGYZdXrWquO1O/LmqgnCetj1+5U0aX3Asi+Kal/M3gVOKEV7jUvnDJKEF5qVpJdaWBNKnnrz2SbF6dVWZRIMPdYJZclek+jqdWdCCWULBOV0TWKEEkLpEExHKZU7o6FOwQMq5QeWgFBWy0/vdQz1ocSav6ZPS/rf18sMTFCc5iFXzkB9M6UQ3heVtfvk+StUNc533y2hRODg/XGyQa+2Kk3PNnF2JVJ9r83q+b33R9K3aF+rCaVHiPs9qXjU9scPVKV2lfOF9JxHXFoNyZJQTt9GEC9HQSq4WoNmGv9tEMpXmgRzOuNvTikCOaE086VBhoS9Jy4QstTw1F9WfcbGBguS7FSvsiiZtcn1FesUy0bjnoqMUE5cd4ie/SHesiHuqUIoH6tXJCgFBeskrJROhY87WTIKIUuREkqkXMNxIwon3mY+lLWLZqPxT9d16q7S705H3kj3+dL48Agld/2IFULJZbqEL2gk5FaVUPKH4TTfEpNQCnFttyREnmyle0+my8zHmP/OLZSY97BQ8vfYTKECLhL4sLklfWrHLFj8XrKFEv7TjUrljNhCWW/ITPrn9KXgrZw+ErhYOS6624ZQCtccFULJ85SjPk55Pn+kuqHxaVZU1u4T568ET8rcEsrFO/+lB74M6FB64SvstIj5vfc73T+WfteE0qOn4dekwtETJBSe/X4Vrdh70ujth/dXoTur5A/pOXeQtxqSFaH8YMY2+mBGgFAu6dGM6gwJEMqs6VMR8oYLn0r8bcCd5ekRGydnTijNfGmeGrOCpm064tgO7oWo5LJvTjF8imQBaKdFCcEv8AuUSevOY+eo2fC5xvjqFstOPz5VJ8RCKYiSU/sCYysy9HSjYtSjTeDIG31JkzLgU8Tr/69uYfp28V7j7wiAelkhatrqfuEeeXMLpRtCKci4F6+PE6GEllz3n0IJ5d/P1aeKBZyPvEv0nBT0SYsmobQ78uaEctLzDQw5I00o498NrwhluT5TQmS1+FzlgVFQtpi/7XgIoTQLKOTi524yNW05fIZafxAIkEIBoWxcKheNmB0qhWX2LoHYwk9arB+8Tp3BM+nwmXhCaeVmI65RIZRY81f3aWlIV8HXG8XJQsndiMS97q1egIbdW9l0eVBZW3kCCreEctHO40EtT9WTn0jWMb/2/kj6lFjXakLpEfJ+TaqxS/YS0i7y8sF9Vah9Ve8IJc/YwjM1YHGBYLY4ekAfnKLmOKE086VRWUzEWPlCJcvq2LVz5Mwlavn+PEPvbewTtUOw43mhby2Rnb7vHEoozayWdguqE6GcveWooW/3XNMSho8kr/9wnUL03ZJ9Rv+sjhV552V5Dv6bTCixEXFtOPxuFhVqRSgXbD8e9MPlfkii/4KMe/H6OBFKsyNvZNCpXPAWx9vbHXtGQuIgR5MqRTJqWia3aR84oZz+UkMqmTte4ojLdImj+0j6EpaF0kH2ScVC2ebD+bT50Bkl0uH4oNjHlllUtpt1I0jc+k41ZM/MSBGXbkIUPCSWnvthdbCbZukDf1qxn177ZZ1R56/nbqVKBZznH+pGQiiblclFiFqe8mJDIzUuL7UHz6AjZwISUyhOUmUqhBJWRfjRw794/vbjSs9217Fz1DTuI130xe4EQ+VZ8gAy14Ryx/GglqcmlCpvnnd1NKH0CEu/COX45fvo9V9DAxLM1P+5P5PVkKyIEc8pDSFYsWiaEcq37ihP/6tnLcPAfcG4xQt94mTObJGX+82PUuQoaLtFiec9l4+uth05a5BNFGjBfde5dgjJsyKU8FWChiWkhnixIkNdGhU3NPPsjqGhj4ZADJRnGhen11sn9FPj95KjRflvMqFE4JYcZMAJJazH2Kig7zd6YSCtIRe15/1ObEKJoJBX4zZyMWYEEEGY3qmU6j05RFrJ6dk5WWPkeWyVOtGOUCLwRuRxFpbWaBNKJ4s4D4ow+9iByLj8weL0LJx+Fxh4RSgr9ptqKCmYrTUhhLJPS5q3/Rh1+zGeUJoFFEKGTfjyWn3Q4GMW/sfwMRanI5EQSoEZl/gRf6s1aAYdjdMsxd+ckin4RSj5mir6ZneC4Teh5GQYR963V85r6CpnSJ0yQeyB05xU+d2vvV/l3rFWRxNKj56IX5Pqt1UHEhz3wWJzT/UCIT3n1gKrIVltliNmbad3p21LcBkI5cyXG4cIm/dtV44ev7WoJWqcUMpZd2TS67R5c2dv2efLblHiAUyyuDpf3CHdAQsmb8vsGJyTtXFP1aE6LO2lFaEEQYRPZMlek4NYyaQPX94/Lttv/G7lc8mB5tkjoOLCZaHkth+pU5hg3RYFxBoRvnv/DWT0aVomF83aEsjrK4oVoTTzoRTWXS9eHycLJbIzvfZrwDIkyu9d6wUFu+36YOdHFy6J436z8nwQfRm7eA+9+WcgU45soeQBdIKYmPUFAWNfzNtpuELYSSTJ7wKsO1g3Bk8K1e7kOLkhlM81KUGvtAoVsV934BTdMWJhyPxS8Ym2e1ZiHCJbC6+rQkLktqv0n0anLlw1/iyvNZxQrn6zBc3fcZyeZ4TSLKDw15UH6OW44DCr+Sf6+fnD1ah1hUDgpJkPZZPSuRJkf7LD5s4q+ejD+6uGVEHwJJ61KG4JZYdqBWh4x8CxtOh39gypaeWbLejhUUtpwY6AhdIp4xlyot/2USCNpSh2Jxgqz5Kv/RCyH3J3RVN4zNrifwOhzJU5DfX4bT3lypSGlvVqbgdzWL/5tfeH1ZlEvkgTSo8egF+T6u+1/4R8OaO7ZumkIiGUn8zeQcOmbk2ABAgldPKqx4kho0Kf28tRp/rWhBK+nhPXHzLaEoLKomG+wMuLPI5zQZL4psSPPbo2Lk6vMQue3aL0xq/raNzyAFGTj6644HPDUjlpTKdaIYSyXvHs9MOTocfgnKzJi5sdoYRz+Jr9p4K4yqSvY40C9NOKgPi803EV6nA8eJYcgSXvy6N1C9OYOP9M/A5i3WR4PKE0m/bwJcscl3bTyUIprLt2rw98mRbt+Je6NimuLCNi1h6yM8lW+t+61qNqcRlg7PpQ5s3JIdmEeF2zZ+e0eeJ6J0IJ6x0icEXAhEwoue6rICZ2pBrPesuANpbD5NdCN7bB0IDOql0xI4m8PrdQmtXl0ixifiU30ap06ofZ8/CKUFYfMJ3+PX/FuIVMKLmuL454QZ44oTQLKPx99YFgcBgiquXTCtxHPAsE3XzbqZZxb9lCWT5fZgKhVPGhFPiYE8rpdPxcYHwobgkljBIwTvB+58iYmlb0DiWUThnP1u4/RXd+Ev9xgfbEOmr2/FUI5dGzl4LBoZESyqwZUlGfPzdSvixpaVGPZm6mpFJdv/Z+pZvHWCVNKD16IH5NqikbDoekBEN3zdIftv1wPm2K82eyGpKVRfDTOTto6BRzQjm9e6OgjBDadRKK5RZKZN3Jkj5VsDtIpQW9PVFEf+Cv1frDeYaf2beP1wySSv6VKjucqxJK2dKw4eBpuv3jwNe0WPR5W2aEkvsjPlS7EA26K/5r2Y5QimNNPl5eHwu6yGb0RP2iBrZ2hW9MOBYUx3liw+Rt84Af/A6R+ebvzQ1aKM3uwz8AnAilsO7a9Ve04RQs4GShRHamN34Ldfv49Zm6VL1wNse3F0FdQmdSJSjHafPEDTmh/PHJOlS3ePaQfkxefyjE3UDOO/7u1K1BMiGIiRMGqn68IBVCsNsOnEgJJY+kxX3MgljgP4x887WZRV9lvnhFKO1E4jmhXNm7uUEoXxi3Jtg9s4DCP9ccDNaxmn9+Ecr2VfLRB5KFkhNmPwmlU8azlXtPUIfPAkLiokRMKM9colqDA8GhkRDKNhXyBBNtFMiajha83tRxzXBbwa+9320/YqG+JpQePQW/JtWsLUeo0zcrQnoJ87+cz/a2j+bTxn8CDvJWxWpT4in5+LVw0EbOZCzMovS+rSx1bhBIw4fy47J9hh/gB/dXoeI5MxInlMi6A4kNUSr1m0pn4nya8DfRHx5lzkko/0p9/NYi1Ldd+WBbqoRStjTwo7ompXPS6MedLZRYUEWuWgTSDGzvTChhUZVz9coWSsh2/LbqoDEmeXxmz5C7DOBoSlhfBJYcEy5JhN9BrOE7Ko68zdrn2HtJKCHk/He3+pbz0olMmQmb/9KlLtUo4kwoy/eZQuevBITLVQilWTCG3PFJ6w8FNQvNCCUnjLhWJpRc91WMwwkDrwmlUxAY1wE0I58Ldxynh0bFJ0+Qs2LxY1AcKWe1kJDh2Hp95M2DVuwslNDexXg4oTQLKOSEEu4I8JOUixiDOP3A72YWSricfByn/WuzZAd/MiOUVftPo5NxR/qo6PRRKvtQ2lkoHxq1hBbu+Ne4vxOh5LqPfCxmaiT43W7thp84PvrzZ01HdYfMMpqLhFCGzK/s6WnOq01U4HZVx6+931UnYqSyJpQePQi/JhWPthVdNctWc/vH82nDwfAI5edzdxIiT+UCQgmdPKFLid9lQikWh5K5MhKsmfzIG0dJXIusYt+pdDYu6pJv8HwD5pvPUfaVCp9AaGCKYrcovf7LOhq/InDkLW8MOIJuH3c8gwjKrx6rGbLAmUV+Y7MUvpB2/eD4qRBKbBJ/rPnHuAwEsN8d8YTZbFryMefJnDZEMkQmqyCoItjG2NQGtDayddgRSo69l4QSMikTujWwfNOcyBTme6/fQ5UOfu5Sl2rGEUrMXwRDwB0DLhNwn+j71wYqliMjvTd9m2Wkr9l9VcT4nQilnGQgAaFkuq9iHE4YeE0oZRcS+eFwHUAzf0tERT/y1bLgZXIQC9I29o5Tp3CbJtMrCyXXaZRJEbdQQnsXrhkvjo+3UMoBhRgodz/6oXNtqlciR4I5bUYoEQkPlyRRcOTtBaGUT3zcEsqW5XLTF4/WMLol+i2OvB/8cgkt2hkglGYfWdAtPn/5mhF4COJplpmNr/EhxO6NicF/yvNa+G7eV6NgcA33ilAiGxhUS7wufu39XvczGu1pQukRyn5NKtlXCd0104Js9/ECWn/wtO1orDYlOP6bOfAbhPKFBsGjBzTOAzT4QoT/huUUBFj4UOIoKXvGNME+VWAyHnyxkTP15Ii7hhNKOZ+rKqGUA4NW7TtJd3+6yOhT87K5adT/aoQQSrPIb5CM0r2nGNfAN7H/nebEloMPC9Ans3eGPA+Z9N1ROR/9tTZAKOV2zR4kH3PBbOlo/4lAlgyBJf+dZ+HB7yDW2NTsCCX/AHAilNxHzGrSiTawgU58PnxCiQ8JWTpLWIgQeS90U0WOe66MAGkfs2wpcpSyGIOZ75w8Pqcjby4LhGth5S+dJ1426P3p2+jDuEQC45+qYxwJR5tQOqkKcB1AM0KJLCrIpmKFGw9KksfvNF9EPmlez+59t2qPpyYUzxXPHR8dff/cENSAXdazmUGeOKGU1w3cY8K6f4LSQt88XpMal86V4NbRJJTyiY8ToXxyzAqCliYv4mMnnlCmIVhsH/hiCS3eFSCUsvWZa/l+90RtunbjBj02ernpYzDbc+yepdl74BWhRDahaS81spouYf/dr70/7A4l4oWaUHoEvl+TihMg0VUz6Z47RiygdQfCI5RfzttFgyZtToAECCWsC7XjfFlQoUebMvR0o+LBuvICwC1nWJgEOcQF/PgR/x77RC1qUDIn8Q0WwuoiKw6sTuLePCIR19otSjyrkBwYxP19WpTLTV8+6kwosRmVeTNAKOGb+JZHhPL2SnlpwrpAAJN8lG42LfmY8bW969j5YDWZrGJz+WpBIK0jCiwucIuwI5T8A8CMUHIShjR+3zweCDqwKp4RyjvLByOmxb1EdDWXourXrhw9dmtR4kEvcjYhcT2ynhTrOSlB153SxuECJ0LJhctR345QiiNzrwil/Nytno0ToeQBYGaEEnJTjzMSIRNxnnpSttA6zRevCCWXycL8n7D2HyP48L37qtCszUeChHKpQSiPh2Rj4gFqor9cSHzUozWoebmEGqQqhLJc3szUvGyuYLpb25co7se7quan9+8LpEkUhcsi4W/hEEr5lCZnpjS0vFdzuv+LxbRk1wnjVjKhhARPh88CH+XD761MeF6dx4S6ZYk+ekEoIa8GtxezYrYPWL1LUErAfuZ18Wvv97qf0WhPE0qPUPZrUvEgEtFVM+meO0csoLUOhBLBJVuPnKUi2TNQ2lSBzC0oo+bvooET1Qhl6/J5jGhjyPjAT8huI8RRUq5MaQnZfvafvGAcNV+I82cT90a0LrJUvD8jIFuEKNWC2dIb/82tT3KUox2hfPXntfTzykD0tOy/Be3Fez8POJC3Kp+bRj4SSijNpIRAMsr3nWpcIx9NW41fxUJZJk8m2nL4rNGu3Ve4wIrfi1+L32VCKQdnYIOEFduOUPIPAH4v8RHBSZjwP7V7fUQb2EAn2SzkTmQKYvqI0uRFEDEzQsldKDihhCUKfnJtK+ahB2sXDvrF8nbNLFPi9/UHThPkZgpnz0C/rgrMLzPZoIETNoUExoj0iqIdnkhAHJ06YaB65K1KKIVOqtXz4wFxZoRy5uYj9MS38SRCDmL5ZuFu6vf3JqP5Gd0bUolc8RZaq3sKDEQ+aV4vHAtl03fn0K7jgY8u+HPzjF9cBWFxj6bGkbeQBEJ94U+Mo12sGSMerEY4YhYFrj+w9oHo3V+rUPDv0SSU8okPTiX6tLMO7JN9KNFp+ZTGjFDK7gzcyAFCmS51ipA86Py5eUEoxdo4btk++n31QeODIH+cyLsbQunkemO3ltn95tfeH25/EvM6TSg9Qt+vSSVrmKG7ZpHWkG2AfINdgZM0NlQIQkMYWhRYspDbVi4I/MBRpThSlH+XSYz8OzbwXJnTkl2e8ZdblCJs+kIHc9bLjahYzoxGU1zY/LaKeemTh6olWLjxB3nR4oRSPnbnLgQgx8g5yxclM0IJklGxX2AzEYQSzuMpkyejEkxnko+fRxeKv9vhJR/pmz3HBkNnBY+5KxfIEvIBIbcNGSJE+/73X6AlbKiYI3aEEpYJbCgoZoQSedjFeOED9vVjNW3nm2jDyTLgRKZgeRTkRNzwhydrU73iOWj7kbPUIk6oXlgoh0/bGgx44Hqd8BlD2k8U+IWZPTvZos0HWKrXZCMNHi9mQTk8rR/qyj6EICnw7UTBkWH9kjk8O/JWJZQ8NajZQ+QBcWaEkqchxPWyZffrBbupf9yaopqm0GtC2Wz4HNoZZ8XHxxJ0G0XhOq3IDoagHC6eLwIKRZ/MdFv5ey3+24xQcqky1PPKQimf+IRDKIUfpei30Gq8b+RiWro7YKGU/YpX7ztJd8W5DUF26NylqwneTzNsZIzM1m67I2/xG48gd0MokVkLuq9eF7/2fq/7GY32NKH0CGW/JpVZWis5MAZDgPWPax6aDQtkAZYH+UXmiz+/DoRywvP1g9F2cptOhBJHSbkzp7XdLF9tVdoglEK2iB8P/nPqItV7OxDpxx3I8W9VCyUnSbiOy53AUvXpQ86EEiRDWCewaHdvWYoaD5tD2TKkom1HzinPIDu8oEk59B7z3LfiBi+MW01/xgXxVCt0C63aZ61xCakeWJ6F+DmI9d2fLbIllOIDQMZXWCivXLtByDyDEk1CCf04oekosBCWPU4ohSsIJ5QpkgeCdFC4Vdcs0Ad17CKSzTY7M0LJAz7QpsjXLfrOM1OZWcnNJpTXFkpHQskC4swIpSxnJgex8FMP/pFo97IIfEU+aV43HAsl/5CFKw3/MOaEEtmOFmw/FqJ1KvyJOdHiWWl43/izUSaU5XITPixUi9mRN89VjnacCKWZDyXm3+cPVw9mPcqdOQ0t7dmcOo5cTMssCCUPbIQmMt7BL+fHu9dYYSP+bvcsVQglrJN4Zlb7gNUHKtbM37pqQqk658KppwllOKiZXOMXoeT5rMVteSo88TcVQsn97vgiyI+n+NBAKCH3IkidPGwnQomjpLxZ0tkSytdal6bkyZIFo8xFbmPcC+kOIRAtExintG/cQilIrej7IpbnVVg9nSyU/LgMR8lFcmQIRrC6mT52eHEJD7SJMXb5biUdPnOZfnq6DqVJmcIQXhZBPFUL3UKrbQglMu8gyltY1EAW7x252JZQcqzMLJSXr10PBifBB2zU/9QslCByyEVsVZwslGbXwbKHo1Gkfev+01qjiiCUPMjLilBa9UWeL1akRvzdjFD2+XNDiKj8xOfrU/l8WYJNyZmpnN4jXOg5oWxYjHq0LWv5TLj/shmhlLU2ZZ9DTijnvNLYeGeciteEkstsyYLvnFAueL2Jkbca2VREEScbok+Z0qQMUajgYzEjlFynVbZQwmIP/203hNIs9SIX7Ud/ZEIJF5VHv15mBM1837mOsZ7IQTk4pfn0oWpBf+Igofx8MS3bE7BQyv6xXMgchBLYIQLerHhx5C18KMWzCJdQ1iqSjX7qUtdpGrr+3a+933VHYuACTSg9egh+TSq+sIuumn3B3/3pwhCLldmw+DEpl9HgDvT8OhDKv7rVD5I6uU0up2N2Pxwl5bvFnlDiOh7Iw1OaHTh5geq/E8j6wRdoWJyELqTZZot8u0IwXJBa0T8uw4SgGPhGcUJjlj0H1opqA6YbTeAouVC29AmCRFSmkR1xgCblex3jne53Hz9PTd6dYzQrhOyf+2FVMIjHkVA2KkZjFu0Ninrjix5SIHZH3jwgyoxQ8lziwvfKbtyiDT8IpVn2nCChZLI8bgml3fGsqoWy9x/r6bslgRztKDKhlDNTeUko5eh+u+cD6xTmXIY0KRNU4/7LZoSSBybhYtlVgCtHzHu1CRXKHvCLVpkvXlkouTYvrKRNh88N3h5BcOIZgWwilzeXphInG+KZw73lGs91ygbiRCjl1IReEUqepx7dkbVs+WnM+/dVpknrDycglDil+fiBasH1FGvxkp7NqKMqoexQiRCgNXnDYdNHy7HBRzLWNf4cZMJp9o7JhLJgtnQ0/zX3Fkq7/OJOc9Pud7/2/kj6lFjXakLpEfJ+TSquB8e7Kr+IiLpD9J1dgSSOyM/KRcfHLN6TIOgB7YBQ/vncrUFSJ7ddu2i2oJ+N2X1BYvA16cYCxQWDuXWW54a9ev1GghzZ/P4v/7Q2GDQh+iB+n7ftmPHVjtKucj76+IGqIf0zy57D/a9wlIyMC3KQiMo0siMO8pEWLG8QIkcRROnZH1YZEcYo8IO1S+uIoAtoAZ6L0/2c+2pjY9x2hJKTbzNCieCqsn0C0e4iQl6FIPhBKLk/pOiDwImrBnAiIAcymfX9r+dupUoFbjEdliqh7Pn7ekPsXxRudcffwiGUcqQt7yDvlxtCiTbMAvzwd+5uYkYo5ZSwXBQfrhFfL9wdPHXggXYq88UrQslPbRAY1fqDeC1IZLz6Pu4ZgfDO3X4sRJpKBBSqrF3hEErMXyEdpbJ2QP5q+6C2IVVln16ZUCJT0ePfxMv51CiclVZIewROaZCUQujsCkJ57+eLaPmewH5ilxxiaIdKNHPLEZq6MVSOSHSUYzNk8mYaOXdXyBjCIZT4oJ/3WkCg3I0PpUq6WJVnIdfxa+8Ppy+JfY0mlB49Ab8m1dlLV4MBIbyrcoo4FUKJ440pGwNfkiAYiFZFGbtkbwKdP3Ev1dzAZjCKjURlURbXf9+5Nt0aJxjMCSUcsfEbjnwROf7Al0uCt5QXJU4o5c1sztajQc00RI4j+46wPqJBM0KJzUWIu+MoGVbXvn+FRh2rTCM7QilHsfNgLEQ5P1q3SEgWIidCCVmY75fsDWYmguWt0zfLbQklJ99mhBJCxtCUaKQAACAASURBVCLaXfZpNRu/aKN07kw09aXAkTfkWZBqs03FvMFL3MwPcRGE9LcfDfVfFTjxzEtuCaUI9rEbD//NrH6P39bRj8sCwvooXhBKHO9PeaFhUFKL9yESQvl66zKEuSIX7m7yfNMS1L1l6ZAqPGsMfhC+p3g/cUKQM2Maw2UFxa2FEnJlOBWwGqPd8T+/hpMifCjcMSI+3zQnlFgL52w9FvJOC39ilbkZDUKJccni7CV7TQpqrOJ3WYFCjsQ3m9M4pRnesXLQlSVvlrS0uEczuuezRUHyKUfwQ+2g3YhA+lqcFMzYfDSB5VPcywwb3g/Zglm0R0IpL9lCGS6h5NmLVNZr1Tp+7f2q94+leppQevQ0/JpUfBPnXZWlHPgCYDUkLBaHTl8yfuZSJjyrhXwttCJ5Rgw3cIVDKLlg8L5/L1DDYYEjbxBKWF/M9M7sCCUnzmiHf7UjU83+kxdDLLtCX5FvJPCrE3qYCGbImzmtZVSjHT52hBLWg1+eqRe8nKdrE0L2z3y3Mni0hIhFHtVfs0jWoEUBjSATyrjl+wkWbhQEOz01doUtoYQvWYGsgaNJPn5kw4DFk3/ciGASu/GKNoSgMPK4i+AmyEVVK5Q1wb1U55ecehLXmRFKLmyuYqG00heUMRH9NCOUXAcV9f5+rj5VLGDuQwk1gM8eDg0Ms8LASgkgEkIJCz0s9XLh7iZmhPL31QdCdBvlIBbenvwOWo1PjMMrQsnFuZHi8p44uTDcnxNK+Hji2PatOJkj/A73D/gTmhEcuf/RIpTymg+3HxFwhj7JhHLGpiOW+pBiDHj2w+6pFNTZNSOUcrYxLmX39t0BQjljs7OF0oycc+zkkyfRR5lQFs6enubGpVDkbYpodKuPAJVAQtX1h9fza+8Ppy+JfY0mlB49Ab8m1YUr16hcn4AGIi/yVyP/GlcZEpcywfEcjunMCtLZCfkPlXbNNhKVr3xxHYTGcZyKsvff89RoWMCPECkRS+TMGBQj5veRCWX3n9YEc2TPfqUxFWUBAfyrHcfM0DXjxSy/N7QG7/8iYBGFNQfSGnzzUcXFyVeOW515ujZkikEQwdNjVwSPlmRCKfcBOpjjlx8wNENRYCXDkbndkTe35poRyjOXrlKlOPkkIblkN3aZUPIMG5hXneoXNQgvtxCrYon0fDwvPK4ThJLrPNr5vpndyyr/MOqazWMzQsl9eHGdfIzOo7zNlAasMLi3egEadm9CJYBICOVnD1ULsRaLe/PTATNC+evKAyG6jULD1Awjt0E5+FhY6YGF8pGvlhoBIyg43eC5x0FShFsC/Ctnbz0WIp0G94/cmdKait/Lz8eJUMpawviwaVU+j6sjb9xTlu9xIpTTNh6mp8autH2lkK3rnQ6Vgq4sglDyEy8QyrOXrhnrHrIM8fEgOxqI68wtR03vM/qxmtSkTCCjkBOhtDKeyISyCMvJzduEgQDi71YfASp+36rrD6/n194fTl8S+5qYIJQXLlygf/75h4oXL25M2KRY/JpUPBCC4yLLdHAnahX8uJTJj8v2hUQ48uuheWmmUalyD7GRuCGUiDhsG3ccuuf4eWocF5gCQokjeu6bJvpgRyhndG9EJXIFdC1RuH4ez6UtfjcjlHyssPwh+084JNssvzdvm1sgeGTowPYV6OE6hempMSuCOoqyDqX8PJ5rUsIITDp8JmCRhu4oZIfsCCU/mjQ78sZRdeX+AT1OYVmzmweiDZHnfcfRc9T8vUBgBAhly/K5qcHQ2UGtTJU5JeqkT50igUi+IJRc59EtoYSc0EO1C5t2RZVQ8g8aNATtO3wAiMIJpRsLJfIbv3NPpQR94/2Sc7g7YfrJg9Xotkrx7geivhOh/HnF/hDdRjmIhd9X/qiz6pMYh1eE8vHRywyiiAJiw/0JIZaNdQ8F7iA4ueDJHeD+AX9CHvxn1e9YJZSytJNZ/7EGDrqrYtCVBRJdi3o0Ix7kiRMKpBOFwkXv28slIJQgrgJns3sIfJwIpdXHJazJ6KO4HgYCzCkUuU18lJtlwEJdlY9gp/fF7He/9v5w+pLY10SdUL777rt0/vx56tu3rzH2+fPn0x133EFnzpyhokWL0tSpUw1imdSKX5PKilByJ3hgxXXDVLDjfl12hBKal2ZZdFTuITYSN4SSW4h4pDMIJQJ8floRyFDCSwJCOX4N/RZneZz+UkNDcgfZHUBUcbT1dNxXuxmhFMciVn2G5S97hvAIpRNm3ALBI0MFyeGZLpwIJaxKv646GPRj+/WZusYR5b4TFyy7UTBbOsPfDR91ZhbKUxeuBLONCMua3ZhEGyD0IPacUCIY5MDJiyHpIZ3w4b/zo2zxdy8IpRBHN+uLKqF8afyaEMs3yDx8XkXhhNdMXN8KhwdqFaQhd3tLKEc8WJVur5TwyJu7m5hZKHm+dPTXLojFrQ5ljoypaUXvyH0o4TM8K85yBj9eZAkThRNKzE2cXAyZvCX4O4g+3AGsEhdYrT9ijnBVCq8slHJwjJOFUpZ2MptXkCPq374CIeuOKHA7whzlOrfiN6y1fDxIiTh142FCbnerokooubIAV2eQLZTFcmSgWRaE0iphAfqGDyd8QHld/Nr7ve5nNNqLOqEsU6YMvfrqq/TEE08Y46tRowalTZuW3njjDRo4cCAVLlyYxo8fbzl2EM7BgwfTpk2bDBKaP39+at++vUFQs2SJ91OaNGkS9erVizZv3kwFChSg7t27U9euXRO0C4I7YsQIOnz4MFWsWJGGDRtGjRsHvn7cFL8mFReT5v2RM8DwzAYq/Y4GoRQSLG4IJTIvQJMRhYu6g1DiCEoQRasFHX/vzgglfAdbfRCIlkZWnpK5M1KX71YZ/wYpgpQGL06EEpY/+HiFY6F0ei58w+CEEot25YJZjKAC4TNVqUAW29ztSI35x+qDQQKJY/vXfllnSyjRP5GFyYxQcguCnLnIbGyijeI5M9DMlxsnIJQnz19xlc/YCT/ha8otgG4tlLgHn4P8nqqEkuuF4nouhYV/8yN5VWFzXCc2VghJv/LLOiOv/N3VCoSQf7cWSjMfytd/WWcoCAgCZkYo5Y9QuyQGbjPleEUoHxu9zAi2MSucUOKjE36A70yJJ5S4xkkWTbTr1kLpNI+tfndLKGVpJ7N2IVfW747yQVcWUSdtquR06WpoVij8hrHKH7uI8IZ6hlVByl/5I9UMO+7ilDplcsLex+e9eP+gpzzrZXMLJU55RPIFuT843v/ogarhwm95nV97v+cdjUKDUSeUGTJkIJC9Ro0a0cGDB6lgwYI0d+5catCgAf3xxx/0zDPP0KFDAWkUs/Ljjz/SunXrqFatWpQ1a1basGED9evXj6pVq0bTpgWO4xYvXkwNGzakRx99lB5++GFauHChQThHjhxJnTt3DjYLMtmzZ0+DoOL6L7/80ujDsmXLDHLppvg1qawclWH9g3xNqhTJjW7e/8ViWrIrIESrUniggJ0PJUTUB08KXWhV2kcdcdzshlAKzUVcz33uIPmQFULrJgK6dhZKHO23/SggFwILXI82ZYN5Z+VAFtRpViYXffVYTUupI0io3JI+ddhuAHbYced3bgWAFUEm0k6E8oVmJY2IeFh5USAEDj9ZOwul6Jvs6ymCcuCPKdLXia99RG3jaP2VlqWN6HdeEhLKs9T8vQC5h4Xy8rUbQWkZ1TllV8+MUHJLh5t7mEUSqxJKrheKe/IAJPybR6HLqe/s+gjtxIHtK1KjYbODrgvyswLJ/HbxXuWhgnCjf0Iqib9zohF8nHRvUSqkTTmQT2iYmmEku51YdU5cmzpFcto2qI3pXMIfzZ4N0oJeunaDMjJNTchkWREdWHtFJP60lxoarjDDpm4NuacdOeEVo0Uo5TzzsoUSz/6tOysEuyZLO5nh3qFaASP/N89Tbjd5MFbZHQcWSuGranateC+djrx51qsMqVPQ+SvXjebkI287QgnSXebNgLSZXMzE4ZVfFJuKfu39XvQt2m1EnVDmyJGDRo8eTe3ataMxY8bQc889RydPnqQUKVLQnDlzqG3btgSfSjcFRPCpp54yCGq+fPmoTZs2dOLECVq6dGmwGfw+YcIEwsNPnjw5Xb58mXLnzm1cN3ToUKPe9evXDSJZqVIlGjdunJsuGO2CHO/fv9+wiHpVZBFv3i6iZyElkjx5MuIRjSr3BiFA3l9E2n6/dG+IqC+/Hmn3+FGQStuizozuDalErkyudChhjYMlBoUfkYJQZk6XMoFFEfXkDYYfOfKc2ogOBPHp9uNqo33IT8gEy4lQwlqTOV2qsN0A7PDj+ZDlYzL5uor5s9D6g6ctm3uxeUlDBB0Yoox+vCb1/XNjWIRSSMsgbWfNQYF8yLIovFme3KBFIe6ISs67jXkbjp6n1aDFxsUz0SQGoXz2+1U0cX38R/Gvz9Sj6oUDEe0oXCdT6HmqfHQJwsDrOgV6qb6v4h0ym3dmhHLs4j0h4v52SQzEOmDXF/m+crS9PGbeFgSzoTm55fBZmvNqY8P3ERYxHpQj3xsR81BBQMEpBvwAh8flVxd14YJSurc5OeHtRYtQcpcU3N+JUMrSTmb4I9Cr121lg64sTvMFY+UBg/DvnrzhEC3c8a/lpeIDwYlQ8jmQJV0qgioEikwoxYkHfpPbRLCq0MqVOyRnI3Maq+rvfu39qvePpXpRJ5QgjKdPnzYsgzjmLleuXPCI+6uvvqK3336btm9Xz3EKMH/77Tfq0KED7dmzh/LkyUOZM2c22nnppZeCWMMKiqPsFStWUPXq1Wn27NnUtGlTWrVqFVWtGm8Gf+utt2j48OFGH90ECPk1qZA+y8rJGIMTCzmyoCzaaf1SW006LBB2skEgE/JRkOoExpd/qdzuCKWIaMY9dhyNt2iBUKZLncJU70wmlC+OW01/xOW8lvsKaY1vFu0x/mwW2CFSClpt8NhcEWEcrl+pHXbcAhEpoZTvgw0ax/ThWCgFoTzK8jubicLLz4FjCN/TtClTBDdtiJAjQwsior0qglBy4fDEIJRdxq4M6r1ibPBfrV44W3CY4RJKIQsTC4RSzq6FIBb4wCE6WC44Ui6ZO5PtY3YiG3aEUj7Fga/1L8/UNeaWFdHhhBISatM2HqH3JEJpZ+1yQyi5bmOkc527OjkRSri8vDh+je0tO9YoQG+0KaustCATSqzXk9YdosW77Pceqw8fvmYgMYeYP3B7OH4uIHkmE0rhk21GKPFRLrRy5YFbyW5F+kz82vsj7VdiXB91QgnfR1gnd+/ebfhL4pi6ZMmSxthbtmxJefPmpW+//dYRC1gTr169avhSdurUiQoVKkR//fWX8e/y5cvT5MmTqXXr1sF2jh07Rrly5aKxY8cax+CffvopPfvss4Y1NF26+KO6n3/+mTp27Oja0ujXpJLzVsvAeEEoGwydRftPBESI5YJc20OnhB4FOT6cuAr48i+dxx2h5EER3KJVrdAtlqkl3RBK3nezwA4hLWFFKGH5y5A6JQ2atFkVBuV6PB+y0yZUIX9m2nDwjHLbnz9czXBdCIdQiiNvngZU+CPZbfR2VjcQypyZ0gTdD5QHYlMxVgglj8ZHd6GBWKNIPKEEcRF5nJ3mGx+uyNUcC4Ry9MLdIdJZ0DAVaVLlRyQ+LK0enVVQIH+vi/aYGFQDkN93HHfLwTNQa4APoBXR4YQSbjE48n5/xraQLsY6oSzWYyLxbJCP1i1M/dmRt6wVaoY/lAOwxlcfGDh5cCrAPkQjt30F+mTWjqCahNX1KoSSp4qEBuiRMwHJM5lQCtUI/CavMTwbnNwX+M1C5sjr4tfe73U/o9Fe1AmlGNS///5L2bNnDxnj+vXrDQtjzpw5HceOY2UccaOAOP7yyy8E/0z4S9avX9/wo6xTp06wnWvXrlGqVKnoww8/pOeff54GDRpEAwYMoEuXArIqosyYMYNatGhBa9euNY6+rQoCgvA/UeD3Cb9Or4+8zV4a3idBKB8etTSYVtERPFbB6cjs1ValE/gWqbYvxNNVjvNEmyIoBP/m6Qft7hkuoTSzXjlt8C81L0XpUicP26/Ubhw8cn/dgVMhmT3k69wSSgRfwEcsHEIpLJSHTl+kukNmGV0RmX0iIZS5M6elLt/Z6+SpzjXUE4Ty0zk7gh9BvlsoO9emenGZnURfeTQ+/vZzl7pUkxPKaVuDwUhOFnE+fiHbIpMrN++XFZ7iHZLzTqO+2ZH3qPm7Qqz0dhm1xIel1b2t+s/fa06e5PfdzM88U5qUVC5fZsvUsCBS41cEjryRax2E8oMZoSdjdsenfCxmR948zZ/Tx6GbOc4tlE6EUtYKNbsPfEm7tygddGVx6ssPnWvTBzO307LdAX99nFSY+bXL7agQSp7FDFZmkWlJJpQiUYLT3ij3QbTjNEa3v2tCGY9YohFKdAHWNxAxWA5Tpkzp6jkiMOfcuXO0ceNGgxiWKFGCpk+fTkuWLDEIJf6/du3awTYFofzoo4+oW7duBqFEVPnFi6GWObQBSynatwvMQSAQjsflEm1CKfIv2/kLOZExuw0pEkIptC7dbHgIAnqqYUA2iqcfdBoD/x16i39aHHk7TTInnzYEJyACMtxAJbv78/zqyIJz5yfxqeLk68rny0wb/1G3UL5/X2V6f/r2iAglT8cHyaUP7g/Ng2535C33HxZKWCFExL3Tc1H5XbhLcEKZPBmFWHFU2kEd5aAcE0L5xDfLQ4SeeX56tD182lb6eNYOoytOPru8v0j72aNt2RDfOacPQrfjVSWUX87bFWKlh9yUyGol35Nn5TLrjwqhLNFzEl2LM8fJz8ZKCYPfC/qK/8RlCcPfOaGE4gUIpZxbG0FyZkkl5DHwhARiLJxQOn0cqj4j1LM78uYWSpA8pNlFpLddgdXupeYlqdbgmW664bquCqHkQuz8ecmEkqdydbO3IDkE1giviyaU8YgmCqGE9A+irlevXm0EwiCqGlHWCJBB9PdDDz3k6pmvXLnSkB/CcTV8Mt0ceYNQQrZIFNUj71ixUPpNKF9pWYrenRZ6FKT6cPDlXy5vZqX0ZaJNnluYH63Y3VPeYCIhlJCrgEizVdQjUkCG46uqgplIX4e6kG1BoIFVcUsokV4NJCYSCyVPx4csQ8hKEa6FEpqRyPf8zPcBCScviiCUn83ZGfT7TQxCKcvVjH+qDtUuFn8awwmlk0wVxwXpL+F+IJMrN5uqFc52FkooBrwkRXl/PndnSIQ+khiIJATyPXhWrnAJJc9bLb/vl69ddwyekQPw4DsoNG2heDF985GgG4Loo50/Hh8HtA9TxqltmBFKp49DN3NfxULp5n4ga7BAi9Sybvripq4KobSKSkcfEfwjMuCESyjlKHg3/berqwllIhJKyP7Ah/Gee+4xgmIgE4RAGRDKd955x7Ay4tjZTQEpTZMmjWFxRCDOzRSUAxzsNgy/CWWtotmCxxtungnq4su/bN7MStkmRNuwiD7bpITxTx5N6IZQyjqAbvvtlqy5bd+qPt8sVu87SXd9mjDAQVyLI72zl68p3xq+QyBakRBKnj1FSHBEQiiRcairh4RSZBTiZCcxCKUsVyMTynenbqURswMWSqfMTPwBI9PSa63LkEyuEoNQ8sAn9BHi5U2HB7IgyYVn5TL7XcVCWarXZLpyPaBLKBNKq+QP/F6IkOZ+4pxQIjUm0gd+FGc1FtchGxkX/LZ62XhCAjNC6fRxqPwSSxZK+cgb7UDzFCcXvf/YoNQspKiw3gpXFqWLwqikQiitjujRx/53VAgGpyJ15ZQXGxq9cDP35VznYQzD9BJNKONhibqFEhZEyPogkhpEEH6NglBC1gc6kRAZd1MWLFhg6FhCEB0BNWj/1KlThh+lKF26dDGCdmTZIPwdEeEo6A/8JnHUHSuyQU4vjfChtNNcs8MSuXy9tBLxe+HLv0zeTFSy12TlxwkfxReaB4K0uN6ZXQPyBgNZIBW/HuVORamiSF+H2yGzz902hNJtl+Bf+OX83RERSp49BYLI73U0t1AiSAK+i1Y5ddF3WCghEP/cDwEJJy+KGaFEJtf//nPfuuz3aPUewqdM9qGU3U8gKl+HWSiHTd1Cn8zeGeyUqt4hRPVfaVWaOLkaek8lQ7A+0iLeIbN3zsxCybP94N5mWqmiTzipKJ8vPumE3FcVQml35K1CKLlPHj4yoL/488pA1i2kxpyx+UjQDUH0D6oLFeNy19vhywXHOaFExhlYLp0+Dt08O36KYUYo0RbSmqomXsAxcJfGxenWtwO+0X4VFUJppYcMCyVcZETgFYwUsHpbvZNWY4Dof9925T0foiaU8ZBGnVDieHnixInUrFmzBIQSUj6QFZL9GvkMuPvuu43jbRA/RGcjeAY6ktCUXL58OaVOnToobP7YY48Zx+cI1OnTp4+lsPmQIUMMC+moUaMMCaJYEjZ3emkQXVkga3r639fLbNNfef4WKTSIL39EeatouYnmcPwCS8zvqw8StMhULFg3C6EUFihgwSU0FKB2rILo+a8X7omIUPLc6tiQh3esnODI++jZS9T2w/lUpWBWY5O2KiC4EIgXmqCOA1CoIAjlyLk7g9qp4RJK3E7FJ1SFUP74ZB2qWzz+yHvolC306Zx4Qqmq9Soy1iATiMgiogCLUhW3hJJHqjvdgGflMqtrRijx3HYPuc2oPmHdPyEfHkM7VKL2VfMT3FNQLl65bqk9KO7HCSXE3OGyIQglUmPCQimsxuIaqC5UUiCUPCEBHwt8raFBCgF/rz4OVQglEl4gralKwTHwkw2LWUboq7ShUkeFUMpSVKJdEEqIr4t9JFxCKVQSVPrrpo4mlIlIKIsUKWKkXoRkj2yhRAT2Z599Rlu2WGdmgTURlsidO3fSjRs3CO2BZL7yyivGUbcoyMYDrUueehH35AVBQSL14pEjRwzLJMhpkyZN3Mwno66fk8rOrA9NxAndGlCfvzZYphlzPRiPLsBCjeMJq8wFZreBXiH0Cd1IFd0shJKTmJV7T1CHz+It7JE+EmSmGR0moYSsSNfGJUJSYQqRYPnIu8+fG2iMQqYWEMos6VMT3BO8KolBKL/vXJtulaK8ZQvlD0/WpnrFcwSHKRNK5PNGAIVTEZbCWCCU/Njeqd/hEEpYEXfFEUqz9Q/yXS82D2TvuXDlmmPwDA/ygFzYnVXyGxmeUJApCLm8udUYf1/bt6VSBhlkoSqbNxNlz5jG1RGsE25mv6sQSjft4hgYCS4aDJ3t5jLXdVUI5dcLdptaVnHk3fu2csF9BH75k8KwUAqVBNedd7jAz73f67763V7ULZQgechsgyNlCI3jyBtBNfj/Vq1aGT6VvXv39nvcnrfv56Ry8hNBFg4Qy9kWeWs9H6xig/DngYVSJVJSNInAA6TyEikDVW4lE0q3WYNU7hGtOmIsnJSE6wfI+4xjMAi6h+NDKQglT8uHDBvD7k1ooezx27pgSjs7zEAokXHohXH2wstucBeEsvl7c4MZgtxcL9dVsVCOfaIWNSgZKnOWgFBKx+JIFAB/VlGQzxv5kJ2KcAcp3XuyYfXysqhaKJFo4eSFK4b7BHxVVQpP82pW32x9gxVxx+C2RnWz3xFkM++1wIf/+cvXLMWsxf24riEsm+0q5aNfVwUIJayIIJTcaoy/Q8arcv9AOl+nki5VCto8oHWSJJSwZtqdJjiNXeV3FUL51YLdpiltQSh7ti0b3Efg4z7xefdH3k81LGa043Xxc+/3uq9+txd1QnnlyhW699576e+//zZ0KKFHCdkgCI/ffvvt9Ouvv7qWEPIbJJX2/ZxUToQSwQ3I7Txry1GVrkatDr78kSlHxbFddOrJBkVp65Fzljl4zTrPN/6PZ25PkEItagP24EYYi5xPGZurkEwJ9xbwefxtVUC31anIi78IlOKZixDUMPSe2CSUTu+L0/jF7yqEckynWtSwlD2hlK2Yb0/eEkLGIFUF2Rqn8nKLUtStWUkq8+ZkQ7Tby6JKKJ8eu8Igv5ULZKG1B6xTf/K+wfVF5Ao367PZ84IVcfsgNUJ57vI1xzUme4bU9O/5QOYVHEXfVpETyro0Y/PREJKPepDxqtJ/ujLMu4e0daVoodwwq+i1hRIi4dvj0rOG0x/Va6yw5O+YrG0q2gahRDYfsY9Agxencihu3vWnGxWjHm00oVR9ZuHUizqhFJ2EvyQiuo8fP07ZsmWj5s2bG/9LqiUxCWWmtCmpVpFsIdp3sYAjUs4h5ZqKHxLvL8SescCrFr4ouVlgVNuPZr2dg9vS2gOnQnyukAtXRLhGoy9m1gSkbkQu9BbvzzO6AB2/d+6plMCHssdv6wmZT5wKJH4Qqe6UGs6pHf47LJTwt7ILBnLTngqh/LZTLWrkQChxJAoxZmRtaVU+j5GRZeTcXcGuQExfxUIkiH3ZN6fQxavX3QzFsa4Yq5myAg/KCef9QtALcr1bFbM2Rf5nK9KAqO35rzU1mjx76apj8AxOcM5cCqgiIOVqmwp5gxZKBGDN3Hw0gcV19ZstqOqAm5tQ3pI+FZ26EMiZ7WdpWS43TTP5aOLv2Bfzdprq+4JQQt1A7CMV82ehv7vVN7rrZj4+07g4QZbO6+Ln3u91X/1uL9EIpd8Di3b7fk4qp5cGi2/DUjlckbBo4IOUc8i76uYrH/2CtXWdovUD9c2iLKMxPj/ugcjSncfO22pQ+nFf3qbV8RQynrT6IEAoRV5cPjchvD1yXjxRsusnyF+GNCnopfHe5fJGm/DtdOOza9dHFUKJSN7GpXOFNCMfeeNY/MVxawwLGYIg0qZKEYKT6geU0Ggt12cKXbiSdAglfKmruCSUaVImp60D21iSBk4o/1r7j6MvbobUKeh8HGb4kGlRPnfQYg/h+ZlbjoSQfNwY1sBqLgglPgaRX9vPYqdDGc59ueU2nOsjvYa/Y7K2qWgbhPKVlqWD+wj2h7+ec08oedBjpP3m1/u593vZz2i0lWiEJb7IGQAAIABJREFUElqTyGaDTDnI3400idpCaf7InQglrlLdlKIxqcQ98OVfPGdGV4syrnWbVrBG4az0yzP1LDefaI450nshsnT3sfO2WXIivYfT9VaEElIdbT6cb1yOlG1D7g61UDq1y39H1Dl8KLv/5B2hHHRXBbq9Yj5lvzen/qoQytGP16QmDoQSx+KQ9RJFJt4QN1dxV0E0+NONilP5PlOC5MhpDKq/+2mhhC911UJZLbtitr7hWHrLAGtCyX0oVdZH+E2KyHic6LQom5t+Wx1wAYFOKPCXP4ZA3lRzXKMd1VSNqs/ErN6K3s0JLk4oVrJBbtrPmSkNHTsbyJmdGIW/YzzDFe8LCCXSQwpyHy6hFLJbXo9TE8p4RKNOKKEx2aFDB0PaB1HZ8J88evSokRcbpBKyPcjnndSKn5NKZcFU9cOKJq5YqIvnykg1BroTqi+eM4NhpXNTxMKkgpWbdqNdF4EAe0+ct83j7XefrAglBKrbfiQIZSGCWHokeCNv+4AJmzwbDghlhtTeHaOrEMqvH6tBTcvkJmhvjl2ylyrkz2JkXJm//XhwXDgWh6yXKJDDOn0x/pgR4uYqAXW92pY1JF4q9p3qStReBWA7QskjqsN53vClruaSUIogF/TdKShHpU/cDxn4I+WlIJTQCQWh/EKyroO8uVm73B6RqzwXuQ7XqvWCUObJnJYOn7kUTlc8uYa/Y7JYvrgBtDKhTcyfBbIYQQ1E5dmLdro1LUEvtyztSb95I37u/Z531ucGo04o27dvb+g8fvfdd0amHFFmzpxJjzzyCNWqVYv++OMPn4ftffN+Tir+0mRMk5LghC4XKx8V70eq3iIW6mI5M1CtQe7yxMp5d1XueLMQSjivI+ry9o8XqAzblzpWhBLyL6JfD9YuRIPvioxQQgsQeqNeFfSn5+/rvWpOSYfyq//VoGZlc9OYxXuoz58bjXs3KJkjhFDiWPyx0cst+wUfzLnbjjn2G/Iu9UvmoG4/rDZdAxwbsKngJ6FEFDWUKKyKGSmAn+Om/q2NS8x+50febkgF2pMJJWSdZm85akSu8wL3CSEtpIItspb5nXFmWa9m9NeafwwJsIOn1LQm7frO9TlVxuh1HZWAShDKbs1KhOwjcn5vlX5B47i7lEJU5TqnOn7u/U73jrXfo04oM2TIQJ9//rlBHuUyZswYQzbo/Hl31qlYANXPScUXTNm6IcauKj0STaywUOPI222eWBzpHD/n7hjmZiGU8Ns6dPoi3fZRbBPKcBZ0ee7dWSUf/bnmH8+mZGIQSgQrNS+Xm0T0sxmhxLH44zaEUiagngHioiG7KG+evcoteUMXEJxXvXA2V4QSPo8bfSKUCESBm4L4mIE4PSyUoxaEEkoX8BlV7fKZu23Lqv6yns2o1mB3H+h293Yjgu7VGHg7nFB+OGO7EbAmFxDK55qWCNlHsA9CJ9TNfOSWdi/H4ufe72U/o9FW1AllgQIFDEIJiSC5QEoIqRAPHvTOahENEHEPPycVf2msnKhVxZGjhQfug4W6aM4Mrr/as6ZPRSddRh7eLIQSx2xHz1wOHi1H83mJe1lZKCH/cseIhUY1+DUNbB+ZhfK2Snlp4rpDng0RQTmqOYxVbirmFNJ4Ij+yme7il4/WILib8HSfMkEc/VhNevwbawtlLBHKjf+cTvAxEymhRHBejSLuCCV8HpGSEsXOQrlwx3F6aNRSlccZrANC2bhUTvoj7mMGsk4glNBBjKRMebEBtf4g4BLiV/GaUEK5Ye+/F/zqrmO7nFC+P30bfThzuymh7NqkeMg+gn1w5ZstXBFKWCdhpfS6+Ln3e91Xv9uLOqFEZhocaU+ePJkyZcoUHN/Zs2eNHNx33HEHvfbaa36P2/P2/ZxUfEHNlSkNHTVxoo5FQomFukiODK7zxCIK86zJsb7dQ7tZCCV8pGCdFcEvnk9UhQatCCWiddt/EiCUsBpA+seNhUC+tddzFvnBxbGzwjAdq6jMqS8eqU4ty+eh7uPXBH3yahfNRkt3nwi2Dz/LTt+ssLzfrSWy08Id/zr2x88KYqwbDp5O4G4RKaE0y4suxoJsZVYyT8L6a+dDec9ni2jF3pOuoMEHK9wMBKFEFP7sLcfo64WREUrII90Z93646pCLykt7NnN94mPXfLEcGWjX8cQ7EeSE8r1pW+mjWTtMCSU0JOu/E5/NB8FEWCvdrD9Cx9UF3EpV/dz7lToQQ5WiQiiff/75kCEj8ObcuXNGikMRlANdShDMu+66iz766KMYgkitK35OKv7S5M2Slg6dTuhE3bZiHpq03jl9m9povKmFhbpI9gyu03rBId+tzh4WpqNnLnl6HOQNCu5agQUC1lkhz+Puam9qWxFKROve9eki4yaP1i1M/e+MjFCq6i+qjgrpJd/627sgH+BgR3jQr88frk6tK+Sh139ZR+NX7De6ComcNftPBbvtpCNar3h2WrTz5iWUkOWpVdTcQonMO8VspHZg3bzn84QpSIUP5b2fL6Lle9wTyvolcxIszyiIwp+99ajhlxhJQRDifV8siaQJx2vfvbcyvfKzd8oIsUAoYRmGdi1E1idvSLiH4eMVWW54ekgEEy3p2cwVoRQ6ro4gu6zg597vsiuJXj0qhLJo0aLKA02WLBnt2qWmZafcaBQq+jmpOKG08nm5rWJemrjeu+NDLyDDQr1y70nTYwy79p02YLNrsfkjLzQ06ZJyWdKjGZ25dJVaxgmIJ8ZYrAglgis6fBYglNBTfCtCQqka3ayKQe/bytLAiZtVqzvWAw5OhOfzh6tR6wp56c0/NhhR3ihIDYcjctVSt1h2Q/Q8McuiN5oSNmn0u92IUP/dSC2UIFq1i2U3hrd8zwkaOXenIQNTLl9munr9BpXsNdn10AWh7DhyMS1j1mCVhrJlSE01i2QNprtEFD6CcpCaNJIiR/NH0la0rg3nNMjLvlmtNfweIJSdGxSlRsPmBP+MYKKFbzR1RShFClkv+4+2/Nz7ve6r3+1FhVD6PYhYaN/PScUJZZHs6WmPic9LLBLKcBfYZMmI/vvP3VPFwpSUc3iL0SJS9MzFazFpoURwRYfPAtaix+oVoX53lHe1oMtP1GvfwZ5ty5hm2nA3k+JrY045EZ76JXLQd51rU/+/NwWPTN2ms6tTLBst2RV/RB5ufyO9DtmPEL0vH9tGSiih9lAnjlCKtQxqFRveakWXr12n0r2nuO66IJT3jVwc4l6g0hD875C5R2h/Igp/ztZjERNKJ9cGlb79f6ujSig71S9KTd6NJ5Tw/Zz7ahNX649IDOA1xn7u/V731e/2NKH0CGE/JxUnlFYajbF45O0U3eoR9EYzWJju/2JxTGzMkYwLX91IJeeFcz82bQhAbzqkbi0TWJr5JsEX7t6440cvCKXXR73YMN6ZsiUS+EOuxZxSITyQU/p73T/BTCvAfN8J9UAHHAe7tbJ5NkipIe4nK36KlFDipALi2TWKZA2xMgHfS1evh5XZSBDKcD4iQShhHRVaoQiagmxTpBZK+Hx2HmPtK+vXM0vK7aoSyo41CoZYznFUP+uVxq4IpUgM4DVefu79XvfV7/YShVBu27aN3n77bZo/fz6dOHHCyOXdsGFDev3116lUqVJ+j9mX9v2cVHxzR07gbUfOJRhDmwp5TP1PfBmsYqNO0a2KzShVw8IUzvGXUuNRrLTg9SZ09tK1sIJykicjusEsuzjOKpQ9vavjVztCyX3EHr+1CPVtF5mF0msiBR+pYVO3eva0MKcuXrlOZfvYW9BGPFiVth4+Sx/HBRTkzpyGjpxRl73C8atbP0DPBik1xP1kZUK5/cjZYC53L+4PfC9cuUbl+kx13ZwglA9+ucS1/2mOjKmpWM6MQRIPy+Lcrcfo28UBl4Vwy8hHqtPTY1eGe/n/y+tUCeW2I2dDLNE4BZjevZErQikSA3gNtJ97v9d99bu9qBPKlStXUuPGjSlNmjTUrl07IysOsudMmDCBLl26RHPnzqVq1ar5PW7P2/dzUnFCWTZvZtpsYnGKRR1KiD4/8W10vtgNQvn5Ylq2J/GPDiOZXPNfa2L4UIajQwkxaARtiSxDSDGHo6ENB72xUOLo8v64oINOtxalPu3KuVrQZVwgdg0fW68KZEHem55Qxy7c9jGnkESgQl97wvPh/VUM6RVxb/jonTh/Rfm2SB3qNlJZuXGXFZHV5u64wCtOKG+rlIeavxfI4+5VUcXX7H6CUD40aonrCHkQygJZ0wcDp3DULwKqIhkb/Gm7fLcqkib+312rQigRADhGIvtl8mSiKS82dLX+wMe6c4NinmPs597veWd9bjDqhLJZs2Z0+fJlmjp1KkHkXBSImbdq1YrSpk1LyPOd1Iqfk4oTyor5s9D6g6cTwBOLmXKg0fdklI6AsDCFIyESa/Ns7quNDQtlOJlyYJHMnSUt7TgasGBnNghlBtP5Yjduq0UeuqIPxmn+da5flHrfHhmhrFwgC609kHAuh/tMXmhW0nUAmBMOIPeV+k2z7dLweysbUl7iuN0qm5VVI9UK3UKr9sVHhYc7fi+u44FXoj1Yvu+rWZB+XBaIYveq7B7S1pAHc8LXjlA+8tXSkKxEKn0DocyZKa3ph7nK9VZ1Pn2oGnX9XhNKNxiGSygr5M9ME7o1cEUokeoV2aa8Ln7u/V731e/2ok4oQSLHjRtnWCfl8tdff9GDDz5oSAolteLnpOKEUpYkETjFYi7vaB4BYWFCBLKXFq/EmIPItgGrWDiEEgQyd+a0tJ0RSuiArnNJ2qwWeR500LVxcXqtdRlXC7qMp9toaKfngVy94tjZqa7K78Dh1IUrVKX/dNvq73SoaHwEiAjzVCmS0dXr6lFlVQvdQqtjhFDyjwYVjCKpA+s5/E153nPV9oSFMjxCmYYyp0tJu455q7/48QNVDYF7XdQRgKQUXJXsipmFEh+jfz5X39X6A1mxx2/VhFL96bivGXVCmTVrVvrss8/o/vvvT9DbH3/8kbp27UonT3p3DOYekvCuiBahtDom9FrTLzwUEu8qbP53f7owZiw94SIx6+VGBqEUGWnctIMMIBC+Fz62SE8GVQC3VkArQsktMM82KU6vtoqMUOLYasvhs26GaFsXJPfTOTs9aw844Oi62gB7Qjnorgp0/cZ/noqqezYIlw1BV7PLd7HvBwj5tAWvN6X/fb1MKQ86hwGi2GlSJqcDJyPPhc3bhevDC+PWuERcV3dCAIRy0vpDdPxcvBsJPsJ+73qrK0LZr105ekwTSie4I/o96oTy3nvvJfhRTpo0icqUKRPs/JYtW4x0jNWrV6fx48dHNKjEuDhahNIqkKF52Vw0Y/PRxBh6TNwTx2cQ3eaC0l4fqUZjoDO6N6Lzl6+FlXEDvns5M6ahrUcCJA35kEvkzkRrmci2yhisCOV7HStT958CospeWChL5MoYPJ5X6Ve06wAHRCfXHGTvgvPWHeUJx8Jv/rkx2l30/H5JxcomCOVjo5cZkj9uCgglCp6tl+X9+yrTS+O9Ex33sm9JuS0Qyhv//UffLdkXHAbcRH5zSSiRSevRukU8h8LPvd/zzvrcYNQJ5f79+42Ibvx/+fLljaCcI0eO0IYNG6hQoUI0b948Qr7vpFb8nFT8yNtKaqVZmVw0c8v/X0K5c3BbuvuzRSHkyeugj2jMyRndG9K5y9eDKQ7d3DNHxjQE/zBu9bNykbBr14pQDr6rIvX8fb1nhLJojgy0OxHTvjlhi4j7bxbuoVEOOZ7x4dKheoGbwkL5wX1V6MXxsW9lE0fenb5ZHtSTdHqe4ndY8SFXdObSNdVLlOrBl/bl/2PvKqCkOLboC+7u7izu7r5YcCL4D8FJIEKCOyQQCCEEQiCBQCCEBIIFd2dZ3N3d3ck/t2drtqanpXqmu2eWdJ3zzw871SWvqrtvv3rvXhNVbIQ6/Q9UAqAELzETDsCUfQGUWhKg/pjRyne/P+MKxLW2A0pMEjGSv/zyC23evFk63gZtUPny5aldu3aUIEGCQNjB7z6t3FQ8oFQjgwYvoL88an4bIYANHB9Wm5r+uNUjXlCuqRzA4Ql3vbJnRbp457Gm9rNaY3hRwkvJA0oQOJvloRxQLy8NWeKSNuxcOTuB99GIlq583AAFF26be+wobGiBitB8hgymSMmWMr7pMXki/Zpdx2xpP7PHx9pjgPKD6TsNf0iD1unekxf09MVrU4c3qmlB6vXXflPbdBpzqXK9fP0vzdrhu4eSZYVbYU8r3/1WjNfKNm0FlKAFat68OX366adUqVIlK+dle9tWbir+pa0mVwdeQH+1aG03mokdHhkSSs0mb/WgyCmXI7lhShETh+RTU8t7VPCZ1BzSeUnjx/LIXkU2pFm0QZAuG7XcxfNoBqBMlzgOXVbQpffJcM5FplgACUZfzHN5oYO9tCydia7ee2o41AeAEh8Kz1+aCyijku2CfW358cFDCUA52w9AaVVCDsZp5bs/Kq0TxmoroESHiRMnpgULFlCVKlWimq00x2vlpuIBpVqs5H/dQwkZN/BQ8qowlXKlNBywH+hNuezjCj6RmmPc0LcF9yTvofQl8UXtyDtv2kRu+0aP9hYhdrDfgoOBNpnTv4kW4MMaTGzWsqaMeJH5Qfgi76o3iahmO735BMvvAJSQQOVpq4wm5ViVkOMASs9dYjugRFJOzpw5acSIEcGyX00Zh12AUo3APNgTHEwxskYj+wbUpHenbPfwzkXFRKV/PirvE6k5TINEBZCb80pKRnWl0Y4IN5zV6+m0HxgLDG2QL0olF8nVoQJjNadXKy2AI2+EJ/Dk80YBJT5+25Q1PyHHAZQBBpTI7u7UqRM1bNiQateuTalSpaK38LnIFUcpx3OReA9l3QJp6Z8DV0y5f6NijKHaxHf3r0GQYeO9c1VDUhkO2DfFsAYa+e2DUtTy5x3uK6AL7QsHJRoAp1+sGNE8MqeheXvaYOKLAygNLOAbVhWenEGLXXGyUaHEiPaWdBzqlDfXAkp8tUYBpVUZ3g6gDDCgjBYtmscIeDD577//SuDy1atXUe7usMtD+XahdLRo32VT7FM6WzLafjpqSxUyQ4T1rUatpoa5KXPw9/I5UtDmkzdNsZVVjUAZp9Lo9e7mF3crT/UnbPapO2RN49OMB5AAmedvPzbUngMoDZnrjaoMeTpG0B4VJmaURD4qzMkZo74FjALKoQ3zU6vSmfUb9qGGle9+H4YT0EtsP/KGVrdeiYoJO1ZuKt5D2ahIevp7zyU9Ewr9XiZbctp2+pZQ3WCvhIxjJn3HxqrG2RlMc4F2d4VR69xDWti1nE8clGgAmcYg2IauNCuIq7x011gmtQMog2mH2DuW3rVDaOSyo/Z26vTmWMCgBYwCymEN81NLB1AatLLx6rYDSuNDjBpX2AUomxbLQH/tumiKUdQ4LU1pPAgaCSaNZDVzbPmyKpX7aq375wVdy/nEQYkGEEcLfj1eAQQZrdfuGyNwPj2iDmXrszQIVtAZgt0W+LxWbhq9wpXJ7xTHAsFqAdCh4eNblLYMalYtSjkeSqvXM2CA8vjx4xQWFkZXrlyhtGnTUokSJSh37txWz9ey9u0ClM2LZ6C54eYAyqhIq2NkAQtmSGxYx9pI+2bU3d67GpUeucbd1PwuZanxxK0+NZ07dUJJtpH3SILonJcsE2nYzD0m0p9TJ3gs8GmNXDRm1fHgGZAzEscCChYokD4xLe4uruWNDPz3S2WyxJZWvvstGbCFjdoOKEFq3qFDB5o7dy69fv2aYsaMSS9evCDEViIDfMqUKVGS3NzKTcV/hb1TPKNHtps/eyMqxBj6Mz9fKHP86c+XaxH7WXJ4JKCc17kMNZm0zZemCPO9+/gFXb3/1H099L3xN6MlTsxophM/Gx2DU99+C0SFMBH7reL0GGwWYETloh7KkY0L0HslHUBp9TraDiihhjNv3jwaN24cNW7cmJIkSUJ3796V/tazZ09q0qQJTZs2zep5m96+XYDy3RIZac7OCz6PP27M6PTkhSvpSU11x+fGg+zCXKkTeFDoBNnwpOHs6ledig2L1Ir+q1MZavqjb4AS7UEt5zqnUewAymBcdWdMjgUcC/hjASQgrvussvCR91eNC9C7DqD0x+RC19oOKBMlSkRff/01de7c2WuAEydOpN69e9O9e/eEBh9MlewClO+VzOhB8GrUBgljx6AHz1watm86oIwKusSgOyo6dJV7Ged2LEPNJ/sOKOVH3InjxpRk5oyW2DGi0TOTlUSMjsGp71jAsUDUsIDdPMgpEsSm8H7VhQElVIzeKeF4KK3eTbYDSvBOzpgxg0JDQ73mtnz5cmrZsiXdvBncVC9Ki2IfoMxEv4dFapoa3SA8wKiYKyVtPH7DaBNRoj7IcOsVSkfN/PD22THRvQNqUOEhkYASRzO95/sufQct79uPnruHnihODLr/1PUB4RTHAo4FHAtYYYH+9fLS0CX28ZciJOfo0NrCgBI6682LZ7Ri6o70ImdV2wHlF198QUePHpXkF+UclA0aNJASc0aPHm3JwlvZaFQBlDzgiIrShKJr2LFiNqqZL7XP8Yii/fhbDwo/hYas9LcZ9/Vyj6QDKE0zrdOQYwFbLGCFLKTVAx9QLy8NsRFQYj6QZJyx7ZzQ1EY3LUjNHEApZCt/KtkOKHHcPWHCBIodOzbVr19fUsq5fv06LV68mJ49e0bdu3enGDFiSHMC4ERcZVQodgFKZKrN3uG7hzJlwth0IyLGrnLulLT+2JvpoexQMRuF5k/jc8a0XXtu/6CaVHCQeYASWt4POI+k/N92zcvpx7GAY4H/jgUG1s9Lg4NYYembZoUIlHtWFCvf/VaM18o2bQeUcqUcrclFJdUcKzcVn8nWolQmmuUHoEyTKI47C7hK7pS07g0FlKDjQbZzwx+2WHn/+N32gUE1qYCJgDJ+rOj06Hmk0pQDKP1eIqcBxwKOBXQsAK3sgYsOBa2dxjQrRE0cQGn5+tgOKC2fUYA6sAtQtiydiX7b7ruHkldOiQpa174sZ9uyWWjQ2/nowMV7PssY+tKvL9ccGlyL8g1c4culitfwWfyowCdhmdaJ05BjAccCjgU4C0Are8DC4AWUY5sXosZFHQ+l1ZvWAZQmWdguQAk90pnbxeJGlKbGaztXC0lFa45eN8kCwdPMx9VyUs8auejgpXtU73vfdLHtms3hIbUo7wDzAGWsGNHoOZednSB2DIns3CmOBRwL+G+BhoXT0YK9l/1v6A1rYWiDfNQ/iAHlt+8UokZFHEBp9bZzAKVJFrYLUBoJRFaaGvi7ztx8JP1UPU8qWn3kzQOUPavnoo+r56TDl+9TnfGbTFpha5o5OjSUQvovN63xGNHeopev/3W35wBK00zrNORYgBoXSU/rjl2nOz6IBbzJ5hvaMD/1X3AwaKcICrmGRdJbMj4r3/2WDNjCRh1AaZJxrdxUfAylv4CS5wurnic1rT5yzSQL6DcD/dV9F+7qV/SzBuTjulfLSUev3qfQcW8OoAQVUu40iajP3+q0QvIMUXlMpZ+mdS53LGCLBYI107lx0fS07qgDKOWbYFjD/NQviAHld+8WpgaFHUBp9c3rAEqTLGwXoASo+FWQKkFpatB7PnbtgfRTjbypadVh+wDl7A9L0ftTdvhl8bSJ49CVe5HSgkqNfV4rN3WtkoOOX3tANb/d6Fd/Vl98fFhtytVvmVA3iA3NnSahIZ5KB1AKmZZg2+lbz4pVtqmWPB7Wpm6FusmQNC5dvPNEqK4vlTZ/UYXKf73Ol0stvaZJ0Qy09ug1x0MpszK0srU+dC1dFIHGx79XhN4ulE6gpvEqVr77jY8msFfYAijv379PCRMm9OCdDOy0ze/dyk3Feyj9ffHlSZuIjly5LxmgVr7UtOJQ1AKURTIloT3ntb2cvUJzU5fKOejk9QdUfWxwA8oTw2tTzr7WAcp4saLTYy7r2/yd/2a0CK/2mFXHg2oywfwxsOzjClT7O+u8/3IFqWBZGFDP4FTnrnPk7bEk/goyWL2+379XhOo7gNJqM5MtgDJ69Oi0bds2KlmyJFWtWpUgsRgSEmL55OzswEpAOXXTafp6+VH67t0iFHbmts+eFBwj5U2biA5ddgHK0HxpaPmhq7aZ6fcPS9N7U7b71Z8Id+aXtUOoU6XsdOrGQ6o2ZoNf/Vl98cnhtenIlQdC2ejtymUheJi/NKCkEwxerszJ49G5W4+tNqVf7fepE0Ijlh71qw2zLw7mDP0VPSrS0gNXaNKGU5QoTky6+fCZqdPf078GFeEkSU1t3I/GmhXLQKscQOllQWhlG3ku+bEEPl064f0iVK+g46H0yXgGLrIFUMaPH5+WLVtGFStWJPBQbt++XQKXb1KxElDCTs9evqLYMaLT4MWHaNoW347mokd7i/KkTUgHL7kAZe38aWjZQfsA5ZwOpendn/wDlCKZ6QAHHSpml5KPqnyz3udttrBrOWpgMY/l6RF1KFq0t4QkxKIqoAS4/3HDKZ/XQe1CM5PKgpFHL5hVjlb2rEi5UieUnktfzjtAf++5ZOr6yiVJTW3cj8beKZ6RVhy+6ngoZTaEVvYX83yXjPVjSYQu/eH9olS3YFqhukYrWf3uNzqeQNa3BVCWKVOGXr9+TbVr16YhQ4ZQ+/btKUMG5RR+kJn3798/kDbxqW+7NpU/gDJmdADKRLT/4j1pjnUKpKGlByIBpb8cl3qGm9uxDDWfvE2vmubvNfOmppU6cZ/96uah9hWy0blbj6jSaN8BJbwwtcZZe2R+ZmQdKRSED2tQMwAAZUiahIYe3NC8ffritV829/fiblVy0IR1J/1txuP6NZ9WoszJ4lEOwXABvc75I7v86RO5P7r0rrPy9yTxYgYtcFn9SUXKkSqhNP2ef+w1HVCaLUlq1jq9WyKjdKrjHHl7WhRa2b3+2m+WmU1vZ2KLolSngAMoTTesrEFbAOWePXuA1j7+AAAgAElEQVSoa9eudOTIEUI8ZZw4cSRPpVLByxV11Mqff/5Js2bNol27dtHt27cpe/bs1LlzZ+rYsaNHm0uXLqW+fftKfQK8fvLJJ9SlSxevZr/55htJCvLq1atUoEABSUe8cuXKhu1uF6Acsvgw/bLljOHx4QJwFOZJk5D2RQDKugXS0j8HrrjbgprAp3/uM9y2aEbmn53KULMflQFl8vix6Naj57p9y0Gw0gX96+WlD8pnpQu3H1OFUb4H9uOlaXUM5tmv6kpTeJMB5UfVctL4NSd019ZIBSN2E2l3cbfy7rADES+4SJv+1kmRIBZVzJWS5u821/vn77hwPQB99pQJpKZ6zNljOjdjsALK90pmlE51HEDpuYuglV06W3K/nresRYRGdZ29m24rvA9w2jFj21nDceGTWhSl2g6gNOPW12zDFkDJj8DfI+/SpUtT5syZqVGjRpQ6dWpat24djRw5knr06CGBQRTEa+J4vXXr1tSyZUvasmULDRw4kCZPnix5R1kBmOzTpw+NGDGCihYtSlOmTKEFCxZQWFiYBC6NFLsA5dAlh+nnzb4BSnirQDvDqHtwBPDP/khA6WtgNY7SX3Hch6ofAxqAUiR7G+0isHrxPm1i4UH181Lbclnp0t0nVO6rtUaW0aPu2k8rUVWLYzCNACN4KPOkSUS95ol7AmLHiEbPOKJzn43hx4Wf1MhFY01OeDFiN5Gho72/91ykODGi05ZTN/1SoxLpT6QOAGV4vxpUafS6oItBxb2RLQJQfjxnDy00mex738CaVGiweRr3IvYWqfNeyUy07OAVB1DKjMW0skU+jPXsjISsh09fUsXR3s6ALpWz02/bz9H9p8bEGn5sWYxC86fR69qn3+169/s0OJsvsh1QbtiwgYoVK0YJEri+bo2WGzduUMqUKT0ug/dx0qRJdPfuXYodO7Z0tA7v5Y4dkRQ1HTp0oCVLlhAWH6D22bNnEiDF30eNGiW19+rVKwlIFixYkObMmWNoaHZtqmFLDtNUHwElMn6LZU5Km07clOZWr2BaWsIBym29q1KZkcYBGF58Nx/qexf/6lSGmqp4KLMkj0dnBRI3GhVJr3u8hni4NmWz0JV7T3yaD1v49Z9Vpsp+xGCKbCAjwMgXQClXzhEZk9l1QOM0esUxU5s1YjeRjll7qDtg4UGa4Qc1l0h/InUYoKw4ah2dvx1cSU24N7KkiC9N46Pf99AinY88kfnydfYPqkkFTdS4N9q/Wv33S2WSPsLvPXlhVpNvRDtMK9sMQIn42STxYlHLqTto80nXu4qVrlWy05ywC0KnWfx1k1sVo1r5HEBp9WazHVCyCW3cuJE2bdokAb9kyZJRhQoVJK+iL2XmzJmSN/Ly5ctSW4kSJaKvvvqKevbs6W4OQBZH2eHh4RKghWcTGee7d++mIkWKuOsNHjyYxowZQ/fu3TNEc2QXoBz+z2Gassk3DyVUU1qUzkSTN5yW5iv39uGlev7WY8UvQ611EQUMWoAyV+oEdPzaQ93lBw/cvN0XNesxQHnt/lMqNWKNbptqFXwF2EY6NAKM/lcuK4WkTWgoVikYACWy7r9aZiyDWu8Dw4jdRNaDB5QDFx70i+tVpD+ROmYBSiu81Bs+r0yZk7sAZfff9+ieGojMl69zYFBNKhCEgLJFqUzSR7gDKD1XlGllmwEoEe6QOF5Mav9ruJfwRveqOeiPnRfo+gNjrAI/tSpGNR1AafQ2NFzfdkD56NEj6bh6zZo1BDqh5MmT061btyTvYPXq1envv/+mePHiGZoIvIzz5s2j69ev07FjxyhfvnxSVnloaKi7HXg2U6VKRQCfOAYHdRHiOh8/fkxx48Z110OMZvPmzenChQuqiUNKgwsEoDQa65UwTgyCLOGQJYelKYDolfcssJeq0RcEuBRn7zhPAxcd0lw3LUBZIH1iOnDJlSykVZoXz0Bzw7UBJTvyvv7gKZUc7jugtIMLzwgwkjyUaRMZA5TRo9HzV4FNykH8UudZu/WW1uP3sL7VNNfOiN1EOg5OQBmbwvtVpwqj1tKF276TiIvGJ4vYidXZ1KsKZUzmek53m73b46TDSDtqdQ8OrkX5B5qncW/GmNAGEhcX7zMfUGZLGZ9O33BJ4kbFwqQN1x+7Tm2n7fRrCvBOg4rqg+k7ac1RT2lgxGPP23VRCmcyUqa0Li4JeVhR7Hr3WzF2s9u0HVB269ZNAnWIZ2zWrJkEKgEm//rrLymxBp7G8ePHC88THkdkkSNGsl+/flK8ZPny5aU4SsRbsvLy5UuKGTMmfffdd/TRRx/R8OHDaejQofT0qafqyurVq6lGjRq0b98+6ehbrSBxiE8eunLlikSFZBSICk80ouKIpUfop40uDyPIWgH+REviuDEJ8WwM+DUonM4j9om9VPH1/cvmM/SdYCIFrrv18BkVG7ZacyhagBJH8bvO3dGdCmKYfg87r1lvYP281K5cVokbr7jOmLQaOjS4FuWz+KVmBBgBUIJH9HMD2ZSxggBQgmJmxcGrhojDw/pUo5Ia3mUjdtPdVERkBFCWzJpM4oO1uqRIYA6gtIJvVg9Q+usVVbv34KH6fq25jAFG1rFV6czSR7jZHsrsKePTqSgMKHlpQ38FJfAxgdO0dtPCaN2xGx7L06N6TinkySiv7dTWxam6AyiNbHWf6toOKNOmTUsDBgyQMrPlBV5DgDyAM5GCzOxSpUpJnsT169dLgJEBSnBd4jc5oARY7d69uwQohw0bRk+eeH7prFq1imrWrEn79+/XTMwZNGgQ4XhcXqwGlCOXHqHJEYASZK3dZosDyqTxYtInNXNT/wjN1YaF03lkZ/IvVcyrxPDVdEPjaCFTsniERJ5yOVKQiDdQC1CWzZ6ctp66pbvseKDP3H5Os96Aennpf+WzSlmCRf0gRwbpuAgtDQDs4MUur69aUfNAGAFGOPIGj6gRQAmqqBev/tW1q5UVVvWsSInixjQUfrCjTzXN+kbsJjI3fu8PWnRIUzxAhLpKpE+9OmYBSis87Vu+rErpk7hOdpCRyyf34W9aSTUi4gSHh9SivAO8PZQfV8sp/KGrZ19ffm9dJjMt2HPJcFKIXl88oIz2FpFAjqNek7b+zjyU6PTszUd+xZ7jYyJ+7BjU+pcw2njcE1DihG3RvkuGwffPbYpTtTyOh9LqTWE7oMTxMjKpa9Wq5TW3FStWUMOGDb1AnpIREOOImEh4GDdv3iwdnaMcPnzY0JE3ACVojFgRPfIOlIdy5LIj7hhIkLXiYS5aksWPRZ/VzO3WXJUnuMgBZakRq+naffVYlSXdy1P+9Iml7kUSYLQApaiuuIj0JKMNuvv4ORUeskrUPF71YA+9mCBoGs/8oJQugfrSjypQnfHeUnVGgBEAZd50iegzA9ROMaK9RS8D/HYC/VKy+LGFwf2oJgUlupzSI9XDFYzYTWQDBDOgLP/1Wp90s0HndXhwKMWNFV13H4vYiK+z9cuqlI4Bylm7PejHJFChce/Mal+KWkyNTJhU6vvIkFDKM2C510/wUI1bbYyCChyEXQyGXKjZo02ZzJKHzGiWsZ59c6RKQCevu2LIg+EjUG+88t9ZDCX+7i9dG9Yee7bVzzvcCaSsP0ikguru6NUHhob4S9viVDXEAZSGjOZDZdsBJeh5ILs4e/Zsr+G+//77UgwkOCa1CkAkAOmJEyeko23QCLGC7O03OSkHyQ1MdcTogxJB/gCUTCKrcZH0NJ9TuJADyrIj19Dle54hAfy6/PNRecqXzgUocQTEaD7UNIi1AKWoNnD78ll1s9wZoOTH5MO9oflSZO0BUM74X0ldeqHlPSpQ6Lj/LqBMlSiOUNYu+CALZEis+4ESSEAp+vHjy57jr0mZMDbt7FudfAWUpbMlozkdykhN6n0YGR3r9t7VKE1i14d4l1m7PAQS9ACl1nOAjUMNUMJD9e1qY5rrah9z6AvPsLrjNwtPHx+083dftBRQBkOYirBBIioy2iD8U8S5oNX+0aGhFCdmdMUsbySAgraJqb2JjnNauxJUJXcq0eqG6jkxlJHmsh1QwjvZpEkT6TgaMZRp0qSha9eu0dy5cyX+RyTXNGjQQHVBEQvZuHFjQpY4/qcU5wjaIFAIAWyy0qlTJ1q0aJEXbRD+joxwFMRyoj1QBwUrbRAPKI0mO+AIrVdobndSR+Oi6T1Ik+WAEhyOWsHPAEkhaRK5bYwH7ZV7T+nY1QeKNCJqL5L5XcpSluTxhTxYHStmcx/5q20SBigfPnvpV2C/iIcyY7K49Gs7fUCpprpjBBjBQ5kvXSJD5POiHKGGnqAGK6/+pBIBeIf09/Y4yZtigPLy3SdUVoND1IjdRIZrxENZPU9qr+xTkT6M1vEXUJbJlpx+7+CKI7cSUHb+bZeXhKvWvcOTyKvZBKBCab/4Aii1Pla1wKbS2BDHjKQQsz2UOVMloBMRHkozmRkg34m4cxYmZXQPitYHsXmz4hml6giTQriUr+XYsFBJZvi9n7bTttOeYVAAlFBKY1zKon04gFLUUv7Vsx1QYrgAdog/3Lt3L/37778SPU/hwoUJcYn169fXnBESd3766SeJOxJUQ3zJmzev5J1kxOZt27alFi1aSHGViNtUIzYHMTo8p1OnTqX58+cHNbH518uP0qT1Ll3kH1sWpU6/iR954wX1ZWiIG5DIKXjkgFKP/47p+coXTI2XblG3cvT2hC1e64ujcxyficQ7gth2YsT81TYKk158/PylYhyW6C0jCihnty/tVohQkzqErWp+6y3jaAQYRVVAaUQmkYVRXLzzmMp/ra5yZMRuIuvN7/1P5u7VVKcxU0Nca2z+AkrEJc/+0AUoa327kY5dM3ZMqDW2zV9UoQxJXVnenWbukuQI+aJ174hImgJQdpi5yyuGzheSfC1AKXoywuaGe/CvXResBZQmJtLhQ27zF1VN/6CQ7w2EqTQv4QKUdx49pyJ+xK6DNSRm9Gj0zuRttEOW/PZFaIj0MSeSwMmPcVrbElQlxPFQijwL/akTEEDJBgwKIXgSkyRJQvHjuzjN9EqWLFno3DnlpAxwSzLZREgvQgWHl14ETRBfAGaZ9CK8pPBMAqhWqVJFbxhev9vl9h61/KgbUIH9v9Nv2uEB/EBTJYxNveuEUM8/XPKKTYtloL92RVLwyAFllW/W05mb6lQWvJ4v348aoIRuddbeS71sh2OndInjCj2ERGKoGKB8+uKVkFdMbbFFACX4Etd9Vpm6/b6Hrtx9QrnTJKTfwy54NYnElBoagBJKMnryhL7EUAZDgD9UVbKmiK+49nJDMUCpp3JkJaBUoivhx2k3oNQ7KVDbv+VzpKDf2rsSExH+sef8HUkBCC9kfwvPQ2kUUIoIBsBLBfUteWIOYujGGFRd0gKNIuCWtxUkXf8MNx9Q8jy8ZsZQ2gUov2pcgN4tmUky1f2nL4TCW9T2IJIhY0SPRs1/3EZhZz3ZFHrXDpES5nASZqQ4MZRGrOV73YACSt+HHXxX2gUoR684Sj+sc3kowf7fcaY4oEyTKI4EKD+es1e6vlmxDPSnBqD84q/99Ee4Nzhi1ufl1/gVUZNiAwioNma9V4Yejp3SJYkjlECzp38NqjZ2g6LOKxtD3zp56MOK2ejZy1eUu5/+MSviolYdvuZ1vK8FKEHFsu7YdZrbsQwVypjEPX01Umwc+1Yfu8Fr4/Igvv2vO2n1EU/eNfnLDDyURpJyRHXWrbyj2D4ROXZlgBIfew1+2EL7I3Tn5eOzElA2nriFdp+/q2oSuwAlPgDD+laX5EO1Qk9KZEkqiRQMWOjJA1shZwopYcxjDylw+/my9uAJTZXQFUPZcWY4rTjkCVK17h0lwQDs6yNX7ruHcnxYbcLRr3zP+AsoEQbAH6OqnRyo2QQx3HMtBpRmJtLZBShHNCpAUBFC8fdk6PSIOhQt2lvUZNJWL08knu3j156gBwalF50sb1/ucuPXOIDSuM0Ur7ALUH6z4hhNWOfiYQP7P46FRAv0svvWzeOmGpKThMs9lHpfmrz8miigrDpmvReBL2IxAXZFMrIxRnge1x69rpq52adOCHWomJ1evHpNOfsu0zUP4qKWHbhKV+9HfvWyl7EaCII8GALH8T++KMn2QeKyZ41cVE1BF5y3+YczwiVgq1bgHQEP5acGsryDAVDCgwsPpQig5BO94KGas/M89f37oCoQF2lTdwPIeCiVaHD4NowKCoj0r1RHFFACZLUtl8VLWWb2h6WobPYUHk2DP7P55MjYciNjw7zzpU9MSO5rXSaL+9IOM8KluDa+aAHKXf2qe3HWFs+cVJLTYyci7NhTvr6f1cxF36w0lpTDeyjhteXl/NRODtTs8mGFrJJSi9kxlLlTJ3SHJERFQDmsYX5qWdqVHCv6Ia9mY5xkIQyu0cQttEf2YYfTJ4Q8gRLOSHEApRFr+V7XAZS+287jSrsA5ZiVx9zEvmD/BwgRLekSx6F+9fK6gdg7xTN6eCDlgBLtKt3UrL+Nn1ehTMm9VY20PJQNf9hCey94en/gJUidMA4VGrJSdypsjOC9U6NMwrFIx0rZpSOz7H28j9jlneAoefH+y27OTbyAsqdMoOghYdcyeTB5W3JACe8nHoLQYq4aASjBSYn4syq5U0oE7KwoeXr49gEojXoodQ1qQwX24SEC/uCtBjUSKwjJUPLIWumhRJYq+qxXMB31nn/Ay0LBBiiRqADteqYsg4+hIQ3ySyBeqQC0IZzFaIFndmqbEl6XKX0IaQFKJRUcOaBkx56+AMpa+VJLPI4Pnr6Q7q+MSeO5KbtAR8VzGyJsp/pY79hmNdt0qJiN5oSdtxRQmplIp+WhNDO5bGjD/ASOYJTXr/+lbALPXTUbs3tb6V2BhMvJG04Zll50iM2N3u2+1XcApW9287oqEIASN0l7A4ASRMS4IVnc5bslMtKcnZFH2kqA8rvVJ9w0HXJybj4wnzdIjzl7PAjT2W9oH1xrrX/e4UFHhId6SgDKwZGAUo1vko1x8b7LqipBDFDi2FQpZlO+eABqICuGhwSFfSHjv9VAEJMH0wOUIH5HliVP9ls0UxKa36Wc1x5SypbVApTyGFgAVLmyhEnb269mWLydL4ASGbVKHlmjgFJP2k5p76utf7ABSmil42XOVJ1q509Dk1oW01wzkbWQN6AGKJU0l3lAKRdQwHF2rn6eJwdQyoLXiXkoT42oQwBW8nECPI9ecUxzbvK1PHz5vhtQyu8RtVAUtQ7AMjE77LzhI1e9GygkTUI3t6KZcc9godjUSzkpBx+6w/45ojc0od+HNMjn4bX2ZX+hI8z99Mi6Up9vT9jsFfICEYkpG09r0tkpDdgBlELL6HclB1D6bUJXA3YByrErj9H4COkxuPEBKP8VFELB1+rA+vncXk25jKHSSxXHF9O3nKXiWZJJyhg4sn78/JU0Z14tgzejlocS9QD0ENvINKaRBYyM1oKDIgEllDIQfA2gd/yai/AXhY1x4d5L7lhQ+RLiBdupUnbpzyIPNsRFzdt9ke48fuHRh9b1TB5M3jdUiHglHxasfu7WI6o02uUVUgOUXRUIovn2AXzx4mFKOVAEYrrssN/KHhWFEptM2vLCzRgBlPAOwwvLCqioPpnrSiLji2FAmSI+ndZIMDMCKKuGpJJCLqwuqRPFph19qpMeHyxCPHDcyBJY6hRIQxNbaAPKGdvOesVc6s1HHVB6x/7Cngcu3qNNJ29IyX8lh0eS1Csl59kFKOUfA3j2KIWiqNmiY6VsNHtHcALKJPFi0t2IZxgbv12AclD9vNSWO20Ree4q2RgfEfiYQKn3/SYvvsnBb+ejqZtPG9a2d7S89e5uc353AKU5drQPUHLZwMhcA+fXF/O8j+WUpoWHy6D6+eiDX13H5AiixsORFbWXKt/WD+tOuj0ECK5Pm9glv8YXPUCJujn6LHUruCBpI4UMULKx9Pxjr6RMIR/j33suurPV5f37Aij5QHveDmoPRiYPJu+734IDUiYtK183KUDvlMhE5289poqjXTQ4aoCy2+zdtGS/uuwogC+yyBmgRID68KUuD8PwRvmpRanMQgDapC0v3AwLjRB5yVgFKHH8q8VYYARQ2uUJFgWU8DRh7ZmyTN2CaQkqWnrlp42naMTSo3rV3L+rAUqlrHjenk+ev/JQvVE6DgegvPXwGZ299VjqjyVmmOGhRLJP7e9cogLyY148e1goioghOlfOTr9tP2eph9LXuGeleFC7ACWTu2U2FLnXleyNDPcTw12Ass53m+gwl6iFv8ETOmWTcUCJfIOa+dKILLHhOnY5kwwPLAAXBARQgjQcEofQvYbqDV8QjLtvn7dHIgC2MdSlXZvq21XH3Vq2AJS3Hj4X1naG9vbgBvmo3bSd0txEPJRyI3y/5oSbtoNXy+DraR15s3o8oESMXfIEsTySCtgLSd4W+zuOzpWyptE+uMrw4EcRebAh0B7A+lGE51UEUDJ5MLl95ICS8bPxcmSFMyahBV29j7y7/76HcJSvVuSAEiT1o5a7jv9YP1rzBZF9+Nk7UjynnWVTryqUMVk8obWQk+VP23JGUSfdqIcS9E4MrCjN3UpAubBrOYmNgU/6ErE/A5RlRq7RpEnByxwfh4wIHBnf379XRLeLuTsvUK95+3XrsQpqgPJ/03d6eWx5ez5/+drjiFsJUAIo/LL5jHuNWNiJfD/ze15t4FpH3oiv5DPSkTBmJJ4UPLg4gRDJMp75QUlq9XOYkH35I2+hCxQqYd5y7mC7ACUTk2DDEnnuKs0TCUknIzyUoeM2ekksIlYTH0IXbj8xZCYwotRyAKUhm/lS2XZACW5IKNMUK1aMcuXKRbFixfIa97Rp03yZS0CvsQtQ8nyFIGtF3JFo1i9eqgjWb/2L6yEnEkMpNyq4EjEGlB19qlHqRJE66KyuCKBEsgySZlDgwUqWIJaHqo0eoMR1m07cUHxg46XTpXIOqW2RBxsC7WduO0dPXriO8kUAJZMHk9un798HaBbn9R3VtCA1L56ReOUX0AwBZMiLmmeX1ZMDSp7k+dt3ClGjIhlIy3OLefHxZHbdMEYAJXgB4YVlhedd5cerBijVuFnxMaUFpI0Aysq5U9L6YzfcwwEYklP2yMeqx+mqtBaigBLHjeAAZICyQeF09N27AoAy/IJbNYvvXz4/9hu8e1PbFPcaqh6glCdpyAHl+PeKUN0CaaVwmnMRHkq19fUXUCIcYOmBSBJ2hGOwUBSR+6Frlew0Y5sYoPyjQ2l656ftIs1KYR48bZLQRbJKsFml0evcNsTPWkk5ZsZQMu5fNiSR567aHNna1x2/iQ5djqSSQn2cxEB62AGUvuwQ66+xHVCmTp2aunfvTv369bN+djb2EBBA2a4E3X38XPXoVz59HPsNbZCfWv68Q/pJjzZIyXzjVh+ncatPSD/xXHR8XRFAmbX3P+7YTwCOZPFjuZMK0BZ7qMhBlvzFj8xWSCzyBYH7Xat4AsoC6RMTjtUQlykvCLSftvUswZPC943/Vnswqnko1QDltftPqdQIVxxZwQyJaVG38l7jkB/vyyvATuDQ6/WXy6v0UdUc7njaCe8XkbKStcYM2/HHf3bdHv4ASmRb8+T7bMxqgEPJ+4Vr4KnRegkZBZQbjt9w71+1PvmxVh69TtNDqrQWDFBWGLVWc+xDG+STVEoY5yqSYMb5ASjVtMrVAGW7aWFeyWBye/L3kdxerC4PhtTWF6cPUAvTKloeSlB48WEl+JhloSgi90O3Kjno121nhTyUczqUpndtBpTyDxe7ACWjamM2NANQgodWLrGIJMeJ608aBpT40AzN7xx5i+xxf+rYDijTpk1LM2bMoBo1avgz7qC7NhCAcnq7EpICBiMq542iFBuULUV8GtYoP70/xQUo9YjNlYzMe0jD+1Un6IPLiwig5B84SO5JEjemIqCUq+7IXxYFBq6gB4KAsnzOFG7ZSn7MCLT/edMZd0yniIeSZaHK597n7wMecalM45bXt82fPhEt6e4pG4p21CT/2pTJTB9XzyWBbsR6MkCJODlGncSHH6g9zDGvo1fvU+g4VzyZ0QIyd7nEnkgbjA1A5CUj91BuPXXTvV/5vowCSiSUaZGDGwGUlXKllLgMmYfdKkAJbtbtfapJ8X9a3it4beAFZ5yrjYukp7HvFNZdGii+sHhcvjJiMEHLJS8AmkhukJe208I8PLb43RdAyR/Xqq0v4qO/WmYMUPLhMfIwH3zsVBilLvEpn6sEKLee9XrmKBnbCKAc06yQ+6QJfMFGlWCYzat+s94j+cwuQMnHrWMsIve62gZla9900lYKP3fHoxqSHMHDfPGOsSNvyBSH5k+re0/4UsGud78vY7P7GtsBZf/+/aUElqh4rK21OHZtKh7Q/fq/khKgBOiSl30Da3rQ8OB3UKeMbFTAfQyjp+WtNF8+yxwExclNAJRI7kkcN6aHzBp7qMjjCr0A5aAVXt4CNQ/l24XSuZNY+LkhI3zyxlMeHif2u9KDUS27HdeAt/D3sMiknG+aFZKyXBGawLTKQU6+9GNvQMl74xLEjkGrPqkoJZKUzJJMkiJD4ePeQFyNkAMcKeZMHXlMrAUoj119QLXGifPu8XZSo3LSe2gxe6lxSmLtsY9R5PrwYASAag1ejMzDywMW+VzVwB2OKpnClNJ4jQJKAN0Xr1whG1YDSiUJOn4OeMlij+WIIPFHrOzY5vqAUm09AEjnc4lwrC81QNnmlzCCx5YvwQQosYfw0X3z4TPqXjUnvTcl8hjaKKDEs2IWYihlH7FKe+r3D0t79KV2n/zZqQyBhxMUbogZHvdOETfNkd69Jbe5XInMLkDJhxlhTGYAyvd+2u6haoR2ESsOpRwHUBrZGfbVtR1QSjf3xx9TeHg4Va9eXdLx5guScnr27GmfBUzqKRCAcsb/StJdFUB5YFBNL+WMHKkSEI4Mmv3oUsqQvzjUXqKYinIAACAASURBVKq8iXhidcggJo3vHQNr1EMJ7xpABctS5QGDHqAsOGiFF8kw1DS6Vc0pDZs92HDkjVhJtCcvSOCZtN4lZ8n3rfZg1LKTHFDC89CkWAa69/iFm7gdAfjLe1T0Gkevv/bR3HCXtrra8SIPKPHCKpM9uVc7WoDy+LUHVFNBU1zkNvAVUG79siqlS+JiA1AaG78PkanKg2N+XPJjU6X25OAOPIM7z96mRkXSe+i6x4oezU1bJV9ztT7Z30GOvf3ULff1VgNKpRhFfox4yWKPMRJ/gEt8yOgVNY5PeWw1a6dm3tT0k4KHEjHZPFm4kj1FjrxFPJTgmB1p0EPJ22HXuTuSpB8r8J6X/1rZQ4l1xkcoss9Zn3iG/A7aIBMBpdLzxBdAhnaQqAiPLCt2AUr+I17tPtfbj/Lnb6ufd9CmEzc9LsOJz3drjAPKSS2KUu0CjodSdA18rWc7oFy5ciU1bdqUHj6M3PT84AEoX71yJUdEpWIboOR4KJFFCN4xJZCkpEaRM1UC+govn4gHKl6ySpQ8WnbntcQhP5gknjegVIsFVDtKDutTjRLGUQaUPJUOQMDx4bU9hgcydObdihMzGj198Zp4wnUeUAJM1/t+s9f0kLkJOS/5A03twagNKPfT72GRZPFjmxeixkUzSKodBSJ4NnOlTkAre1byGseX8/a7iebVXt5/7DzvponCkVrpbMYA5YlrD6jGfwhQqu05eID52FsjHkoo0UDC8BkXc6sFANC2PFlC5NnGjrzlYR/yawEeAcqZOglio0c19R1QyunE9ACl0ovfLA8ljreRhMGKv4ASKl1QYGEF3nNopSsVJCdNb1eSdpy+5T7VQWIcpBflgFIppAInCCy8SGu9zQSUNb/d4MHba1eWN/8Rj7nqedVF7KFER4UPdJzSaYWv4F6A8hk7QUBfDqAUeeL4X8d2QJk7d27KkCEDjR8/Xsryjhkzpv+zCIIWAgEof/ugFN15/FwRUIInkSlnMPMAyOBFwx6oyAZduDeSpkbEQ8k/kJnertz8RgHlzr7VKWGcGG4PEnjYzkSoJfC6ykra4fAsIkg/abyYtLFXFQLnXSou85y96Fnc4tAlh+nnzWc8hiw/DtWLoTQCKFn29ePnL91H+vAUw3MmL73nR4JRxCv+2MqbnJoHlMgiLWUQUJ68/sCQ1Bw/RsRy/rrtnOG7jecrVQJefOgFVJNypIo8vuc788VDqbaWiEfl9YCNAkqEIrBjNxEPpS+AErF023pXoxFLj9BPG0+r2h0fLfg4ZKpQ8DDiw1GvqJHGtyydyYNLVQ9QKqlW+QIo+eQjdj1OtAYtOuTed0j+0OPO1Lo/91+8S29PiASU8J6X1QGU4WdvU9OIUx3ItCKOWZ4ICBYB0IEt4mi/AgEo5fuMAUqEJOC5x3uS5Vne4GrsMHOX3rZR/B2MEx9Vc50Kobx49Zr+3n3JEC0Vu5atX6eZu7xitrHXx6zUBpRQMSs2dJUHoJzYoijVcTyUPq2tkYtsB5Tx48enBQsWOEk5RlaJq8vHMM5q7wKU3WZ7H+MiC5k/QkYTuVMnpNHNCrofqOCr43kPRQAl2sERIl7I0LtWKkYBJWIx48dWAZScegw8mTxYRN8vX72mTSdvEo60lRKE5IAS18j1tq0ElOPeKUwNi6Snpy9euQEzkqPWflbZy3R8Qg9oVH5o4U1O7T+gVOfv1NuSvgJKvYQhT0BZiQC4lYoeoGQAm2kAsyxp1hZ/PX67dv+ZuxujgLJ37TzU8bdwalMmC7WvkE0zZsxXDyUDlPDAw+MDdoA/d7lCIvjy3buFqUHh9O4xIPEE3ni9okYx1bpMZokaR17UvOYAfYWHrHKfFOA6swAl2pq+5QwNWnxYGg5P5q82P63n2MFL9zxOKbA3S4+MVPHh2wSB/bR2JWn3+TvUeKLrmBxhH4g9lQNKALciGZN6Asr2pej9qa4ESK1ipodS/sHGAKXSPSAHlBiHElWP3vjxe8/quejj6pGAEn/bduqWUAypvH1mDyVeXjxPIb2pl2DH8xyjfSQwItnMimKXM8mKsZvdpu2AEnGTjRo1oq5du5o9l4C2Z9em4gHlbAlQvnBn+vIGAE8i46VjfwfXGWJQ2LGvnEJDFFDqGdoooEQsZrzY0d20J7z8Fi9HqEZTpDUeJUApl0dE5iYyB1nxx0PJH1ujPfayB/BlSRPgA13/eRWvYTMghB8QpP9X57JedXhAObdjGSqZNZlXHa0YylM3HhqSmuMbBz1N/4WH9Jbf63eer1RpbLwmOaTw1D5U9AAlWzckXyA+sF6hdJJcqNLLVM5LaRRQzvyglMc8rTjyZoCS7whMAKC96VUrt1uHGSTm+DhkY5DL4KktGGRNe/yx1+tntVhZkIJPbuWd5Y0G+JME/NsXQFn+67UeXl82MGRVD1zk2nf+Ako5bRb2Jp/sxRuDAUpQ14DCBgUfVfN2X1IElEUzJfU48cEHf4soBigBuBESVT1vaq9M/3ixortld5md8JED7lCcHsnj6dUYGngbK2Xts73zyR97vZLD8DyFmINRQMnTqhl+gOlcYNe73+xxW9Ge7YBy586d1K5dOykxp1q1al5JOZhksmTeL0krJm9mm3ZtKg9A+WEpKYayy6zdXlNRApS4+b9uUtAtQwYv2D8HIulBAgUoEYsZN1YkoOTVEnhAiaNxaFYbKUqAUs4V2b1qDvo+Qh9d/jKUA4VWpTMT1BrUyhd/7ac/wiNjKBmghBeHHUkCzOB4Xl7y9F/uJlcHZyhUPORlTth5+nK+S2oT2aElshgDlKdvPDQkNcf6R+Ywkj8YKFaavxrdiR6g5OmrzACUamvDr2WiODE8krmsBpRyBRORPawEKLGPHj9/Ra///dcdk8vou+A5O3T5HvWpk4diRrACaPWj5qFsVy4LTdvizdeqBSh3nbtNTSa5kv3k9xD+LZKUowYoed1xETJureeYnOUApx4lI/hh5baCZvsvbUsQ79WE93b+GwwoYYNHz15S7BjRvO510GUBNH+72iVsgQL6NBxvx4kZ3WuriQBKpfVk6yd/lqIDfDwhrtYBlCJPEPvr2A4oo0Vz0Z+gIAFHqThJOeobgQeUyPIFsXlnBUB5fFhtD7kztIgYHyi3sCzfYAGUoDiKGzO6e7w8oOwya5db2UKNpkjrtmEvssFv56M2ZbNIVeVckTxBuPxlyL8I//moPIWkSUTwoKoV+UOQj91hbbHMS3kbufouc2cOq2WC84DyL9CNGASUiP0zIjXHxog1Qia+lidOTT6OD1VQup4n2LcSUCL5gr2IUiSIRTcfPncvgZ2AUp4EpraX0iWOQ1t7V1Pda9fvP6VX//5LaRNHemGNvELUsrw/KJ/VK84Y7arF9bI+lTzIar8p1eXXh18PMwGlPIYYpx4lhysfeTNAyatLIb50wZ7LXh5KeMJLZElKC7iYdMS4MxEJrXWx88i7+eRtUkIZEo7K50jh9nLLn3vgWGWsAWzsAJSgquO5f7XAuwigHFg/r5e0Kmtz3bHrbplgNgZ4Gkcu1QeUvBIbrnU8lEaeDL7XtR1QTp8+XRVIsmm0adPG9xkF6MpAeCiR5XvnkTKgRMIMIzpmJsEx6ldNCriTMuB1wtc2K2Z5KJWOKrSAGiiO8IXLxhsz+lt0YngdaVidf9tFyw66pNJ2968hxW4aKdDQhtemRt40biAop/ZBMDkkJZXsoPWSVBoHT/2D30EEDf4+FNYWXj7ILpWXbL3/oQg1SikmdHF3bzUdcFxi/CjzOpehYpmNeSjP3nxElb9Zb8SEUl0keSHOVQtQYswHLt3zapsPVVC6/p3iGd1eXRDyZ1OJzRU98labHI7C4c0/cPEeTWpZlNpGaNrL9yZ/vdJ48SL+rb2xI285JQ5PUK82XiUPpeGF07hAjYcS2vZTNnkmrqGZQAHKmdvOukMt/PVQykM+cOpRYvhqRSsxQMl7NZEBv2ivN6DEWpXKmswWQAm2jhMcNRD/3NKLoUQ8LhJ0cJyPbPVh/xxRfO5pAkqO+1frnXHu1iNdWUuc9iAEiS98m2uPXqMtJ2+5P3DwgT5syWG6fO+p6s7G9XJAycJCzLx/WFt2vfutGLvZbdoOKM2eQLC0Z9em4j2UAJTwUHb6zfvI++Tw2l5HFnjgIVi/6pgNktnk9CCBApQAK/wRC08PxGf6qdEUGd0DfDY1rv24Wk6J24x/MLP/9hdQQs2ocu5UUnOsLTWgwPelpqbjCSjLSnKS8qIVQynykFeyJ9Mu1wKURTIloT3n73pdzocqKF3P8x5aCSjZwHBsjNMRkbWFzBtitvhSLkdymtW+tMff9GIoGfUMtKQntigmkdEj6zZXmoQEYnClwmiDjO5v0fpqSjmQIp2skFVeO38amtTSm3lA5F4ROfJW81DO3H7ODTr618tLYGrQKlrPMfkHFcIFig9TBpTVQlLRz21LEO/VfK9kRlq874qXhxKhOPjQ4GnYQOvW6mfltVUDUEq2lM9V7SQA89YDlHxbUzedVgWUcv11XAdeTnAf89y/eu8MtThdNg7Yv/Lo9ZoUXjz3LhRvhizWB5T8xzn6gl48OEWtKHa9+60Yu9ltOoDSJIvatal4QImsVmR5KwHK0yPquHnp2BTLZEsueSgrjXZ5qECB8cuWSE+E3sNB1FRGPZTISIdXksXnxYoRjXBkj9JxZjitOHRN+u99A2pS4nj+00zJE2d6VM/p1idHP2pJOSL2kXsokThVNkcKafzsYZ8qYWwK61vdy5z8yyBfukT0z0feajo8oJzfpawU0yQvWoDy/K3HhrSLWdtYD6yLFnDCkd/Os55SabjeCKBE3CjiR5WKvx5KLTtprS2IrYtxoEMJUHaYEU4rD7v2qbywtnGagLCBaLKQCTYvhFIwOUe0Ic9SF73/ROvxL2r+GqjB8NyP7LeoACjh7f66qTplEk4seKlFhNHwa8vbgQFK3quJ9hF3Ls/yTh4/lgS43hRAycd8M5swQMlz/4o8E+t8t4kOX7mvuC3xnnry4pWi7C67gPfmT25VjAYvOqTroZTTJ7FYdtF7w0g9u979RsYUqLq2A8qsWbPqHnmfPq3OuRYoQ+n1a9em4gElsnzBpdfpN2/usDMj67iTQNjY4aEECTJ7oGoReuvNV+t3o4AS3i8kEbCYHXgrj0UASv5FDX6xRHH8B5TyOEdQXvCB5v4Ays//3OdB7cInzjDgAHojfJlrARxk5C9TkGcMFKBk2uVagBIfLNtO3/KaF6/5rnQ9vD6MDF6Ja5Q1GChAif75vstmT06zP/T0UD57+Uo6SkfcFy/xKMJ/99PGU/TPgauE5KR+3PGf2oeHP/cmfy0fj8v/XS3GM1CA8rft59x2GVAvLw3R8FCqceOy+V2889hDGQdhNEwSVW7X6nlS0dQ2JYj3amKNEIIjB5RI8oK6FS9ZCW8eVIT0itEYStC/Hbv2QPHDRX5/qSUA4mItD6UdgFIttlpuD96TDq5MZPxraZ3jeiiCgXCfUYOB91eNjkxvffR+t+vdrzeOYPjddkDZrVs3L0B5+/Zt2rRpk/T3Bg0aSKTnUa3Ytal4QAmwAkDZUYGMVun4AxQz375T2K0MATJaqA6wIvK1KbIuRgHlsWGhFDNaNLdHFYo3R4e6PJQ8oESsJRR1/C1yL6KWHUSORfnx8Hrc+PvCruWoUEaXvChrC96MXf1raAJKtaSc2TvOS0lFKH93KUtFDHoo5R4aUVviA0V+TCy/Fgoycqk01OGTqaBV/M2KY5KnjsVBgTeR6Z9HBUCJmFvsGaUiZ2Eom93lnRYp645ep3bTd7qr4hgV3l2rCv9xwvchp9Fiv/kDKOHx/H7NCYl2qHzOFIrhBmpH3rN2nKO+f7vi7JSSOPix6z3Drtx7QmVGRirjIIwGHJpKhQFK/p4BZ+ryg1fo0XNPNbf4saJTrXxpPAAlEljUwhn0xqz14QaBiuPXvJXmzDzy5p9XbKzMQ1l4yEqJXQRFz96oo+ah5K/lj6jlbfKxvlNbF6f+Cw/qAko2ZnBh4mOPhR1ZcS/Z9e63Yuxmt2k7oFSbwLNnzyQwWa9ePQLojGrFrk3Fa2kjyxeAUkndQOnhgiNJeCjZkbdcdULk4SCyLkYBJbwKyOxmtDrI+D4yNFTq6sMZ4bQq4iiRJYaIjEGrjtyLCHCAZB94hxBXOqJRJCm0v4By6UcVKG+6RNJwWFtJ4sWkvQNqeg2R70tNnpEHlAu6lpMy9+VF68jbV0DJ9obWiw6Zo+uP3fAajzyZCse6sDUDkfC0I/s0YewYtHtADVXKm2DxULJ4UqU9Jg9JUVIyUtub8qxWNU+2v/ufXc/vJb5NOesB+43Ff6r1r3evYN0ZQ4JSXVFAWTBDEhq8+BDtv+idAKb3DENmPE8ThDCaQkNWKk6JAUowAzB5Rsj6rTh01QtQ4lQFrBm8h/JNBZRFhqyU+I/NBJQ5+y51K9vI15BnI/ilbXFJxWnt0euqt4HeHjDr/mHt2PXuN3vcVrQXNIASk1u6dCl16tSJzp8/b8VcLW3Trk3FA0pk+d5+9EICXfKiBCiRwAFAyWhjQHL++V/73ZeadSOCfJnPHmcdqB0l4zgVYWUMUIJA9/AQF6Bs/2s4rT7iik07PKQWxYsVw+91lHsRP62Ri7pXyylRyoCqhaez0ntJygfz6dx9NG93pJoJLyXI2sLx2P5BtbzmAQlJSEmiqMkz8t4a3vvJN6YFKOVHfqLGFAGUiDlbo/CgV8rO5zPt4ZVMFDcmwTOttb5Ka1H7u00Esmoc0/YKDRGdjlQP/ILjVh+nzpVzKCY3KdlUi4sR9fn7U40nVG2Q649d98g8B7VReD9vT7ahSWpU5vcSX02epMZ+8xdQqu1RtrfKjlzj9lrzzwoe+DLS9q0nbyqq0Og9w248eOaR1Q06LMQEKhUcYU9tU5yu3nvqVtNpWDidBGRP33zkcQliwOsXTOcBKJGQxzMJqC2F0phB7YOPLKUSaA8lQgSYbKmevTF+do8qvaPY33L3W0bPXr6W/qkFKKe1LUGIL4eXksXWa7Vr1r2i1Y5d73475uJvH0EFKGfMmCERnt+54x3Y7+9Erb7erk3lCSjLSje2KKBEFu78zmUlVQtw8IEnLVe/ZW7TiDwcROxo1EMpP07F8dEhN6DcSauPuL5GtTxDIuNideSg77OauahbVU/ZMFbXX0C54fPKlDm5K8mEtQVP3IHB3oASRzO5+y2X6mZLGZ/WfupNbM6DgEXdyhG8NfKiBSh5b4sRm4kAStAjMW8y3zaUkOQqGjyg5G2kNSaltbj/9AVByaR0tuRCZN5G5szXZX3rUefgOJ+pLs3rrJyFrzYG6CzzMXdqoRG+zkF+HZ89zf/WoWI2Re3wQAFK/mieAUo1WT+9Zxiel3zMJOKyCw6KBJQlsySjsLO3JXMwQHn9wVM3VyUyhbtVzSHRT53kqHtAqdyocHrTACX6l5+ksDXyhzaIX2etGErUw7MCYBrKOSjsyBs62bceuThc9eyNOiKAMu+A5W4VHi1AybNm1Bi7QZU+yax7RKQdu979ImMJdB3bAeX8+fO95vz8+XM6cuQITZgwgSpXrkzz5s0LtF0M92/XpuIBJbJ8bz98Tu0FPZSI5YNXS+lFKfpwEDGMUUApBysJYseggxGAq/2v5gNKuQf181q5qWuVHIpT8xdQbv2yKqWLkP9jbfGAWd4pq6OmlMMDysXdylOBDIm9xq0FKC/ffUJlv4qMIRNZT35vaB15A2wtP+TiDOWLEt0TT920qVcVypgsnu5QjK6FboMGKvgCKNViXNW63XTihgfNDMIw4N21qvD8jnwfIN6/eOeJV7dq+vKsopH1UapbZuQad2wcDyp4QMkECngPXrYU8d0eQz2Ac+/xC48jbsRlF+AAJbzKzX50eQbxgQQeWT7LHxKX4DREkd8Lcl7fae1KeBFzK62l2pjl8Z7sWpxe8GCW/d3sGEr5ujJA2W32bkn+U/SdIQIoC2hwW/IxlEh0wjhQan67QTWW1Kp7Rqldu979ds7J175sB5S8Ug4/6FixYkka399//z2lSCEeyO7rxM2+zq5NxQNKvLBuGQGUGRLTwm6eZNlGXgKiNjN65C0HlLwHjweUSN6JHcNb4kt0XKyeHPCaCSjlc1eizOFjROVjZ+uROXk82qCg920UUA5pkI8GLDzkfjmqvaT0bCjioYQHa+kBb0CpRPfkC6DUirPSG7+/v4sCytErjrqzvNVCEtTGsvnETQ9llaTxYtIehVhbf+fCrucVaPg2EZcLkIZ4QHjvv1npStyrWzAt/fB+UdXujTxLjABKPhsd+7l1mSwUfvY2NY0Afl83KSABLJD8h+ZPo2keeLR5jyQ+XPMPXOG+Zkn38lTv+83SvxmgBNcvS9zhbWA1oMQYlD7gsqeMT6dueB65M3Anr58xWVza1MtbRAH15RyRasCWtckAJewxbvUJqpk3tZsSTcvoSoASXnBIhLKilejDZ3njVA1JXSgOoDTrSWBeO7YDynPnznmNPk6cOJQqVSpdOiHzpm1+S3YBSv5IDUkZ+Hr+4FexGEol9RUjLwFRq5kJKD+YvtMdl6dHCSI6vh5z9ngoWvQKzU1dKpvjoZSDVR5MMVvztEhqgFLtRcBTqODllz+9tocS4QR4+WRJHo9iRI/mEQ8mai/2slJ7wbF26hVM6/Zc8G3rAcrNX1ShDEn1PZSIeRy57AiBJ7FCTpeXwq7C1g7EyqH506p2ywNKtfVRu1geF6iWvGXWnH/delaiYJEXCCZAIx4ynfCUMzovPUAJPe8xK49Tj+q5CIwSWsUIoPxj53n6Yp6L2YAByl3n7riPYsc2L0SNi2YQMgt0qvNxABKJfvy/lQAlD0J5L60cvCEDnI+fRrwfn7WvNkAtr6oSoMSx/6DFkeTuUL2BLGLbclkNEZvL1XBEAaWQoblKPKAE7+Tpmw8pe8oEHu/7H9adpNErjlHCODHogCy+nAeUs9qXonIRvL61vt2oSp9kdIz+1Lfr3e/PGO261nZAadfE7O7Hrk3FA0p4QG49ekb/my4GKJXIsoMSUHIPlf9N3+nO6IP6D0CRv0UOKL8IDaHOlbMrNmvUPnJAyScSsbZ4aUl5p6yOmt63UUApf0lcu/+USo1Q1i6Wj6VopiS0O0L5RsRDifiyRfsue9mR6YDzP/Dk8nxYgL9ra9X1IIQ/c+sRVcyZQvPDd9TyozQxIrEK2u/50nkDflVAeeomvT9lh/tnUCvBdlYVUDgN5oAJ62d772qUJnEc6Z88H6EeoDQyTl8B5dAG+ahVmSzEe9rVQj+UxvP0xSsK6e+KU0bB/Zl3gLKHEh64n1oXJx6E8tRJeoASGclKz2b5uIwCSiQx8jrbasmO6EeLhxK/izzfWB3Qgs38wFNyVGTNeUCpNteXr15LDBEFMySmVIlce48Vnth89oeliFFxhY7bSEevKvNxiozLrDp2vfvNGq+V7QQEUL569Yp27NhBWIinT701OVu3bm3lnC1p265NxQNKJGXgyFvpK1gpniZv2kS0VEaWLfJAMWowedILu17twScHK3wWdLtpYbQugooGX7dylRGjY0P9j+fsoYV7I4HPl7VDJK+XUjFqn55/7PVQy+C9qqwtUKfgpaDVn5reN59IoQZYtMYsp03Rst+kFkWlwPyimZO6FXmUPCasDWTALuDsyv6uREjPA8ptvatS2sRxfVnKoLuGz9QHMT0I6kXL9tO36N2ftrurq7EBiLanV++XzWcUScJ5wmm0wdbcakBZesQaunrf9T7gnxW8og8DlKgj8UE+e0VNiol5J3ENn/iGfyPRjweYvIeSAUoehPJZ/vJ7oWmxDIR4P1b4ttTWAtRtHSoqP3t427Pr2SmT2j0uH1NUAZRae5Vff3jPkYCH4gBKvTvc/t9tB5S7d++mxo0b04ULF6SvX3kBZQsAZ1QrgQCU+DK/+eiZYuC3EqBUIss2CphE1sXfI2/eM9N2Wpib25Blg4uMQavOR7/v8fCk9a4dQh0tApT8mJmtkRF6ZmRdxSGyOmp63zyg5Dku+cY0ASWXsapnx5/bFKdqeVJ7VNMClODo43n42IVKgJJXK9rRpxqllnkl9MYWrL/zgHJlz4qUK3VC4aEyvW92gdLxn3BjAhWhJa6ki83H/fKgBiENEzRiKAW6dFdR2qOqgDL8AvWKoDcb2jA/tSqd2UhXHnXhCWMSr/hBDijxkVZ3vCuGkoHH5y9fu9kwWFwlbxfWgRxQ4hk8fs0J6ec95+9IH8ZtymSmX7e5wr70KKiU+oAHb1G38qqexTcRUPIhD+CsZeEUIsk+Pm8UAxfa9e43MKSAVbUdUJYqVYoeP35M48aNo7x58xKSceQleXLXF0hUKnZtKj4pRw9QTlx/kkBFsv20iwYDkl0relb0MGswAko+dgxKExuOu8iy9TI4RfdL99/30GLuaFbLS2DUPvxxupx/UaQtVkdNx5nPzFXzgGn1I+fh07IZjuyqhogDSv6FCtqj0xGJA0wHnO+LB5Ryj5joOgZjva+WHXXrYPMcpCJjlXMPqtFLibQlUkdOG8Ou4ZWNeFBjNaAsNWK1WyrPw0PJAcphDfNTSz8ApVxSEIl+jKoLc8VHWp3xmzwAHx9ryMjOlcCeEqBkNgUoPXb1AaVLEsetHY6YyzHNC2kulRwg/tcBJc/tKqLAI3If+FvHrne/v+O043rbAWWCBAlo7ty5VKeO8pGfHZO2og+7NhV/5I0jlZsPnymS5yodLyupr4iAHKP28vfIm89utQJQ8rQXmJuZgJI/TpcDYBFbszpqsns8oFzeowKFpPE+UtXqB/ul+LDVQkuKpIIqIalUP0DkjTQvnoHmhruO/NqWzUJQzoG3WUke0gNQ9q1GqRJ6xk0JDTAIK/GAcu2nlShbygTCo+Qzl3ERT58l3IiBilM2nqbhS494XSHnSCHoCgAAIABJREFUDWX7KVCAkk/K8BdQyoEgPnZ4Ll4lQMmD0KohqeiXtiUkm8nBHnS+/+SOvJU+gO88ek5FhrqkHgFAITShVeR9FIpg6gjEkTfL8jawxaSqIjGUWm3yWf4Q80A2P0rd8Zvo0OX7Xpea5XgQnadd737R8QSynu2AsmjRovTFF1/QO++8E8h5m963XZtKDigRNVB/guuIhi/8TcWod5SUO0RAjlFj+Qsoef49ED3Dy4pi1oMCxO7/RPCood2+dfLQhxWzKU7TqH1EACXzMih1yPpTU0nxF1DynHp66woevSq5xQHleyUz0u9hF6RmcbQ3uEF+1S54PfXwftUJMoNvQkEW+uQNp6WpaOmSK80VWdJNJkWqo/CKUVbYZvKGUzRy2VGvpuVZ+fW/30wHLt0jox5XrTEr3Vclh6+m6w+eed3rPA/h8Eb5qUUp34+80XhI/2X09IVLlQUxzjn7Roo7wOsPAITCk9iz8eIjaXq7ktLvvgBKnoLIJ0AZwSX8XwKUPA8puJeLZkoq2d8BlFY8Ffxr03ZAuWHDBkkNZ86cORQSYkwqzb+pWnt1IAAlS8rgqUrYLHnwhSMbPMiSK7y0jQImESvKpQ2VxqTUL/sbrxDS6ucdtOnETa+XjMg41Op0mbXLgy+xX9081L6COYByK5epKwfAOLpHxuKXoSGqRN7MBvnTJ6Il3St4TYEHlCt6VKTcabxj9LTWVK4UomVHXpWC1dOKoWxRKhPN2uGSTTUCKOVHrP6sbaCvHbn0CE3e6AKUGz+vQpmS69MhsTHzVDj4mxZfqRnz/HHDKYJHVV7kMa8vXr2mh09feqkd+TMGXwHliEYF6P1SmfzpWlIKYx+UYI7gYyr1ACXoeaDRrQQoeQ89flf6APYXUIIjFHRxZgFKpj0PFbW/u3iKXsjveSuzvLUWlJfexNxhA5R632+ig5dcHkqEf03acJI6VsxOEPCws9j17rdzTr72ZTugLFCgAF29elWSV0yXLh0lSeK5+EjK2bdvn6/zCdh1dm0q3kPJJ2XIX/Si3jwrAKW/HkreO2cFoOz82y5adjCSgNtMQIkNuP/iXYl2xZdjXJBNzwm7QN+/X0TiapMXnozaF0DJH7np3SxGASWSJZA0ZBRQKml9640tWH/nAaUovyabCxI3Gk10ydyhQNv86NDalk0VuvFIIpIXkH3juN3KovTcUYuhnLfrIn36p+udYAag5JPy5BQ8CCMJHefyUCpRBPGgip8DjmL/2HnBHfKB65WewbxSjy8eSgb8RAElVHVWf1JJdSnhbECoBRS34sVSXnPWVzAASl4sgHnOMTmzEjZ92fN2vft9GZvd19gOKNu2batLYD5t2jS77eB3f3ZtKh5Q8kkZDqAUX8JOM3d5SAT2r5eXPiifVbEBKwC3+Ei9a/KAUi2LWGvMVgJKxE1O33pWCFDyOsVK0oz+2CiQ145YesStg22UX3PvhbvU8Ict7uFrEeCbMUck7Y1afsyrKZ471Yx+lNow4qGcv/sifTLXBShHNi5A75X0z0PJJ86Biixbn6XuIeIjrda4jdK/lUjMy+dIQb+1d3ExyufAxwWrAsonL6jQYJd2uNWAEh8FoNlREj8wsq6BBpRq6mBvT9hM+y/ek6Yi6kAxMm/Runa9+0XHE8h6tgPKQE7Wyr7t2lQ8oOSTMoIJUPp75I14OsTVobScuoM2nzT3yLvjzHBaceiaezsMqJeX/hdFACWvbuILoLQyhrJduSw0bYsLULYuk5mGaMRQ8oBSSUnHynvVyraH/3OYpmw6I3VhlA5p34W71IADlLFiRCMkjVhVmDqJvH1Q6cSJ6b/Eqda4Awkoea5YeLay9lYBlJzUJBtv2ezJafaHpQMGKCE2ML+L2JG3WcpibO44gQBtk9HC80X6AvzUxBwaTNhM+xxAaXQ5LK3vAEqTzGsXoOTjJfkjz2AClJtO3KBWP4d5WVaU2JzPcG4xdTttOXnL1K/QDjPCaeXhSEA5sH5ealcuangoeUC5qmdFyqnAc6jloTRCG2Q0Kad9+aw0dbMLTOkBSv6jQ4mn0qTb0vZmeEAZZjB7/cDFex4JdrGiR6Pjw60DlBPWnnDrdPOGApVO7Bj2A8oSw1cT9icK/6z4e89F6vmHy0P5VeMC9K6fHkqeJ1fO14uPtJrfujyUfFY7u6dKZ0tGczqUUQSUPFm/fA7MvryMoy8eymohqejntiWEYih9AW9KNwwAHZ7po5oWklgbjBZ/AaUa9y4+vvARpmZro+P0tb5d735fx2fndQ6gNMnadm0qHlDyHioeRBjxjFh1pKuUvCEKKFMljE1hfV0eSisA5YczwmkVByihjQsdXKVilX183Xb+AkojSjlGAWWHitncx7163gweUB4YVJMSxjH+ovLVhlZeN2zJYTeoNpq9Dq3yet9HMjZoSXSaMYfv15ygMauOezVllmdLa4xK95UIoPy6SQF6p4R/R9783tMClJASHf9eEQ/wyFOMyefAA8pobxGdVhAvePD0BRUY5NuRd/aU8WnGB6UIKloiMZRmAUp/95qZgJIP80J4CMJEHEDp7wqZd70DKE2yZSAAJe+hGrvqOE1cd5J+al3Mi4za6IPdDJP4Ayh5Uu/3p2ynrafM9VC2/zWcVh+J9FAOfjsftSmb5T8BKK/ee0qlR4ppeRsFlB0rZXNT5hgBlHYkgZixp0Xa4AGlnM9R7/pDl++5VVpQN0a0t+ikikSnXlsiv0PFBc8NeUGiCuRBrSxKgAj8qOBJlQOEBXsuUY8/9kp/NwNQ8gpBckCJZ2qNCA9lg8Lp6Lt3XYAS3uO9F+5I3tGY0aN5gEw2Xh5QgiNYKXbRH0Cp90GOcQTbBzDG1OfvA4RMbV+BnxqzRaOJW2jPeQdQWnmfGm3bAZRGLaZSPxCAUs4L9+T5K4oby9hRlVUPIH8AZZpEcWh7n2qSpd/7aTttO202oNxJq49cd69kVAKU07ecoUGLD0tj9+XIGxQwjHfvx5ZFqdNvuwnJH89eunj5+GIUUHaunJ2QOYzSsnQmGtawgOrdxTMB2JEEYtJtrtsMDyiNxoYevnzfrdKCjrQ033UHIlBh3OrjNG61SxqQL3ZkzBoBlAv3XqKP57gA5agmBal5iYwCs1Ovgntg1PKjlC9dYmpYJL0HCMMztfpY15F3oyLp6dt3Cqs2pOWhVGMuePjsJeUfuEJq0+iRd1QFlADRiP2vlDulIWcHM7xaImLjiVtotwMo/boXzL7YAZQmWTQwgLISgRbCnxKMgJLXsX73p21u6UizjnA+mL6T1hyNBJRDGuSj1mWihoeSB5RqRNN6awouPLzYMiSNR+dvPaYk8WNSwYhjOA9AaVApp1uVHDRh3UmpCXBSDm8kBiiPDAk1/CHkz5638lpoY8MDhmI0NvTIlftuUm1cr3Zsatb4v111nL6L0Jrm2zTrPtMaJ0jcv15+jHpUz0lls6eQqqp5KM0GlPJx5e63zP1BxQNKaNOP9RFQqnmnHz17Sfn+Y4DS3/2qFubjAEp/LWv+9Q6gNMmmgQCUaz6tpMhVaGRKeuDDSFt8XX88lOkSx6GtvV0eSisA5f+m76S1HKAc2iAftYoigHLaljM0OMJDqQYoa4zdQCeuPyQ+FlVvHZXWy6j04kfVchKOUY0CSjuyivXmb9bvQxYfpl+2uADlocG1KL4BPkdoPTPKGlz/1lvg16tr1tC82mGk1vIf7ACUSpMSApRNC1Lz4v55KOV9A9zCU9+5UnaCvGD1sRukKnpa2/JnZ+/5+91KUSKAElKNow1IL0ZVD6W/G5gHlPwzr8mkrQQxAJRA7Vn0bde731872nG9AyhNsrJdm4pPynlTASWCzrd8WVVamXcmb6MdZ26b+tBoNy2M1h1zyTmigAoDMX9KxSrA7eu28wSUyh5qxElCkadx0fSSF1KkmAEo4W1iR6hQMwEJtVrhM23tyCoWsYEZdQYvPuSmTjLqeT1+7YE7w5iNxcoX5ZiVx+j7tS6PMl+s7FPLxsWHraKbD5973euL9l0mkJGjjG5akJqZDCjRLvS6Iapx8vpDN6DUA3xagFKNW/Xx85eUd8B/68jb3/tK7VSm6aStFO4ASn/Na+r1UQ5Qnjx5kr755hvavn07HTx4UJJvxP/Ly9KlS6lv37505MgRypAhA33yySfUpUsXr3poa8KECZJ6D1R8Ro8eTZUrVzZs5EAAyrWfVqJsCmoqRgZvFWDyx0PJA8rmk7dRmMmAsu20MFrPAcphDfNTyygCKH/ZfIaGLHHFUEIBw9+QB7ZXzACUn9bI5c4aNgIowbUIzsU3ofCA0ihQPnHtgTshxA5AqSTZin7/i4CS2fvUjYdUbYzLQwkpRVDlqBVvQHmAoDuNogYoEeeeZ8Byqc5/JYbS3/ta7SN69eFr1H5GOCWNF5P2DKjpbzc+X2/Xu9/nAdp4YZQDlAsXLqRu3bpRqVKl6Pjx4/T69WsvQLlt2zaqWLEitW7dmlq2bElbtmyhgQMH0uTJk6l9+/Zu8wJM9unTh0aMGEFFixalKVOm0IIFCygsLEwCl0aKXZuKfwms+6wyZU0R38gwveoGI6DMkDQubf7C5aHkSW3NetG1/iWMNh6P9FAOb5SfWpSKGh7KYAaUn9XM5eY1hJoJVE3Uyid/7KX5ey5JP0NPOUZE5qxfmzkILuYBpVGgzHvH7ACU207dovembJe6AqB/HpGYZdZ9ZnQ5ig1dRbceeXsoF++7TN0jPJTfNCskATGryukbD6lqBKB8t0RG+qpJQZ8ApVpCFg8o9Tyg6NgoPZBVz3Or7C3SLv/M40/l4FU+dPk+ZUwWzyd+TJG+RerY9e4XGUug60Q5QAkAGS2ay5sBGcfw8HAvQFm7dm26ffs27dixw23fDh060JIlS6R4B1z/7NkzSp06NeHvo0aNkuq9evVKApIFCxakOXPmGFobuzYVDyjXf1aZsrzhgBJas/8cuEJ50yYyzRvH64NjkbX0gYPtAW0noPylbXGvrEwlTya7UT6vlZtGr3BJ+ekRUPOA0g6aGkM3sx+VeZlAo/PivWN2AEr0sfLQVUl3/v0pO6RELZRgA5RL9l+mbrNdR952Asr3SmakkY3FASVPj6MGKJ++eEUh/R0PpZFbjH/mmXEqZ6Rvkbp2vftFxhLoOlEOUPIGUwKUAIqJEiWir776inr27OmuvmHDBukoGwC0WLFitG7dOqpatSrt3r2bihRxcY2hDB48mMaMGUP37t3T1Rznx2LXpgLdxcQIapYNn1emzMnfPA9lpmTxaGOvKpbdG3JAqaUPHMyA0owYWmZkJaDoC6DMnTohHbv2gDpVyq7JZcgDSugpR7OY99CyzSRr+PXrf2nShlPSx0+tfGkMdct7x+wClKwfUNkEGlAWHbqKbit4KHlAOaZZIWpioYfyzM1HVOWb9ZJZ9Lzs8meDB6AcWFPRa+YASkO3hFSZ5w11AKVx+9l5xRsHKA8fPkz58uWjZcuWUWhoqNuWN27coFSpUtHMmTOlY/CJEydS165d6fHjxxQ3blx3vT///JOaN29OFy5ckGIvRUsgAOXGz6tQpuRiSRdq87AKMPkTQ5k5eTza8Ll1gJLXB4ddtLxpVtlHdF/J6/EP10AASj6LWT62XqG5qUvlHEJTk+spIyHiv17O3nxElSPAjN2AMt+A5fTo+Sup20B5KNUA5T/7r1DX2bulsY17p7DEHWlV4QGlHvWVFqBUo4ziAaVz5C22ilM3naZh/xyRKpsR5iXWq3gtu9794iMKXM03DlAiXrJ8+fKEOMrSpUu7Lfvy5UuKGTMmfffdd/TRRx/R8OHDaejQofT06VMP669evZpq1KhB+/btk46+1cr9+/cJ/2PlypUrVLJkScNA1OjS8x7KTb2qSPEj/hSrAFMwA0pezjEqA0ozv9ZFPZQghd555jbdfvzcfQzJ9t8XoSEEcnORwgPKQAEYkXHaWefC7cdUYdQ6jy7tsk3eAcvpcZACyqUHrlCXWS5A+f17Rah+oXSWLQsP6vXI+eXPzr5/H6BZEYowaoDy2ctXlLuf68jbAZRiy8gDSjPCvMR6Fa/lAMpIW72xgBJZ4EjcYYUByvHjx1P37t0lQDls2DB68uSJx85ZtWoV1axZk/bv36+ZmDNo0CDpeFxejHo2xbetq+Z/AVBmSR6P1lvooZQDSi05N6sAt9F1Z/X5h2sgACUbx7qj16nd9J0e03AApa+r6rru8t0nVPartQEBlHn6L6cnL4LTQ8kDyoktilKdAmn9M7TG1eduPaJKo11H3q3LZKYhDfKr1tYClGr69A6gNL50/DPPjDAv4yPQvsIBlG8woDR65A1AGSdOHLdFRI+8g8FDufmLKsI8g2q3hFWAyR8PJTLXcbRhVeH1wSWQriHnZpV9fJ2bnYDy5zbFqVqe1IpDXXfsOrWb5gkov6wdIsVOipQec/bQgr2Xpap2eeFExhXIOtfvP6WSIzx11u2yTTADymUHrlDnCA/l5FbFDMemGllTHlC2KZOZBhsAlP0WHKDftrtog9QAJTLpc/VbJtVxPJRiKzNl42kavtR15O0ASjGbBarWG+eh/C8l5YD8G5yN/hSrAJMeoGz241baefaORL49trlLL5ddky1FfFprIaDk1XckQKmhvmGVfXxds2ABlBuO36A2v4R5TKN37RDq6ABKX5eWbj18RsWGrfa43i5AGdJ/GT194dJzt6tPuaGKDFlJdx6/8BrD8oNXJCUblKmti1P1vMofOT4bnrsQUqQVR7vCDtqWzUKD3s6n2qz82cADyoODa1ECBZUkhIzk7OsASiNr9dPGUzRi6VHpEjPyBoz0LVLX8VBGWumNA5SYGmiD7t69K8VRstKpUydatGiRF20Q/o6McBTQBiFuEtRBwUob9PXyozQpIst765dVKV2QAsq64zdJHGF84V9UD56+oB2nb1P5nCkoTszonoAyZXxa+6l1HkpefQcda6lvBDOgNDNAnc2zUMYktO/CXWk9tDyU4PEEnydf+tQJoQ4VHQ+lyEtIqc69xy+o0JCVqveMr+2KXMfrWQcboOQ9lEpyoCLzE63DA8p25bLQwPrigLL/goM0c/s5qSsHUIpaXL/e5A2naOQyF6A0I29Av0djNRxAGYUBJbKyoYKD8sMPP9CpU6do7Nix0r8rVapEKVOmlIAkiM1BK9SiRQuJ2HzAgAGqxOYjR46UiM2nTp1K8+fPD2picx5QbutdldImDk4P5fUHT+nv3ZfcDwKsj96LioGa7Cnj0xoLASWvvoNxaXHb/VcA5YGL9yjs7G3KmDQudZi5SxdQbjpxg1r97Ako+9bJQx9WzCb0NEYCCnSri2VOSjM/iIx1Frr4Da0E2h7Q9/BF754xyxQ4hg00sXnhISvproKHks/ynvG/kpLetlWFT4z6oHxW6l8vr2pX8mcDDyjVdNxfvnpNORwPpaHlcwClIXMFtHKU81CePXuWsmbNqmg0cEsy2USATqjg8NKLoAniC5j2mfTitWvXJM8kSM6rVDFOWWPXVwoPKLf3riaREvtTrAZMRtpndcHhB1lBq0rzH7dJ4IkVEUAJmsTTI+taNSThdq3OeFx/7Dq1jYiN1PJQbj5xk1r+HCkcgAn0q5uH2lcQA5SoD13jODGivzEclMKLqFKRp5RhVWwDlH2X0fNXgT3ybvjDFtob4R3n581rec9qX4rK5Ujhr6lVr+cB5YcVslLfuuKAcsDCgzRjm8tDqQYoIdSQvY/LIWJFDOXYlcdo/NqTVLdAWvqhRVHL7GRnw3PDL1Cvv/ZLXYb1qUapEvn3zjN77Ha9+80etxXtRTlAaYURzGjTrk3FZ3m/qYAyZ6oEtMpCQMniN9m6ax15I5v5h3UnJU8FjoMDXfgAdSsoNHhAqRWvtvXkTXp/qn+AMtC2DLb++fg6uwFlzr5L6cWrf6Vu7QKxcvvjuPmzv/ZJQKtZ8YzunxfsuUQ9/tgr/XtOh9JUOltyy5aOB5QdKmajPnXyqPYl/1jmAeXhIbUoXqwYXtdaDShBrH/4yn3KnSYhxXxD5ExxX3SdtVtKQB1QXx3gW7YpdBq2690fqPkZ6dcBlEaspVHXrk3FA8odfapRaj+/1qBCgaOaYQ0LUN2C5tNx+OKhtBpQNp20lcLP3XGvphZtkEnbw7RmrM545JNtNAHlqZuSXB9fALpxTOgU3ywAMJAtwnv1XwSUalbj5Sz/7FSGSmRJ5puBBa66eOcxlf/alZTTsWI26u0joDwyJJTixnLFhvOFX2MrPJQCU3SqmGwBu979Jg/bkuYcQGmSWe3aVPyRt1nufxz9W6VU4gughHzfip4VTVoZ72b4bEz8qqXlbdkgfGw4WADltlO36L0p2z1mMaBeXvqfAyh9XFnXZXJ2BLu8hTn6LKWXrwProVQz3J/hF+jziCPP+V3KUtFMSf2ysdbFPKAEBRaosNSKlodSDVDiWZu1t3VH3pYZxmlY1QJ2vfujwhI4gNKkVbJrU3kAyr7VKFXC4IonkZvTF0AZkiYhLe9hHaC8//QF1Ru/mc7ffiwNd1jD/NSydGaTdoK1zQQLoNx++ha9+5MnoBxYPy+1K+d4KP3ZAfz90rN6Lvq4ek5/mhO+FnF9OI5FsQvEig7uj53n6Yt5B6TqC7uWszT05NLdJ1Quglweqk8g6/cFUB4dGupmr+Cv5wFl8+IZaFTTQppmUHt+GnmuitrZqeebBex69/s2OnuvcgClSfa2a1PxgHJn3+qUMmFsk2ZgTTNGHnysrtWAEjPdde42NZnkopUa2iAftSqTxRoDmNwqz8lmBcmv6JH3jtO36B0ZoBxUPy+1dQClXyvO3y+nR9SxLWEpW+9/KAJPBh2gnL3jPPX52wUol3QvT/nTJ/bLxloX82pFXatkp89riQPKgQsP0q8RSTlqgBJ9szV2jrwtW0ZbG7br3W/rpHzszAGUPhpOfpldm4oHlOH9qlOKBA6g9GUJ95y/Q40mbpUuHfx2PmpTNuoBSitIfnlAOaV1caqhQiK98+xtavZjJM8r7OgASl92ouc1DGxkSBqXNn9R1f8GBVvI2vsf+tfloAw6QAluR8R5oyz7uALlSZtIcFbGq/GAsluVHPRZrdyqjcg/lu0ElEgUnLjuJE1pU5zKZrcu6924Bf97V9j17o8KlnUApUmrZNem4gHlrn7VKfkbCCjxwsCLw8qy/+JdenvCFqmLqBT7Z7VqBE9Y/lOrYlQzXxrFZQg/e5uaOoDS9C3KQEq6xHFoa+9qprev1qCRkwTbBhXR0a9bz9LARYekf63qWZFypk5o2RCu3HtCZUa69NQ/qpqDPqmpDyihVgbVskGLDtH0rWela0U8lP4ceaMPhChEB5+ZUwJqAbve/QGdpGDnDqAUNJReNbs21VfLjtKPG05Jw9ndvwYlix9Lb2gB/d3Ii4qpdWhJIZo1mYOX7lG97zdLzRnlTzRrDL60YzXJL09YrgUo+ZABNo+o5On1xfZ2XMPul1QJY1NY3+p2dCn1YeQ+tW1QER1N23KGBi8+LP1rzaeVKHvKBJYNwQNQVstJn9TIpdoXvPkzt52VqIWypUzgASiPDQul2DG8s7x5W/tz5G2ZAZyGDVvArne/4YEF4AIHUJpkdLs2FQ8o9/SvQUnfIEB58+EzOn7tAZXOmtzy2LEjV+5T7e82SatvROHFpO3iczN8Uo4VMmTigPIONZnkChlgZUiDfNQ6isSi+rwAFl/IgF2KBLEovF8Ni3uLbD6YASVP5m9F3DBv5Kv3nlLpkWukP31cLSf11ACU8sXhPZQOoLRt6wa8I7ve/QGfqMAAHEApYCSRKnZtKh5Q7h1Qg5LEe3M8lCJ2NqvOiWsPqMa3G6XmetcOoY6VxDSozerf13aQoV5tzAbCkeiCruVMp3sSBZR8DCqbS1RKbvLV/lZfx4Bd0ngxac+AmlZ3524/mAElH+ZhxUcUb+Rr959SqREuQGk0y54HlMeH1aZYMaIprh+ztUioTTCvi22bM8g7suvdH+RmkIbnAEqTVsmuTcUDyn0DalLieDFNmoE1zQTrA/HUjYcSMEPpFZqbulTOYY0BLGgVmssxor1liRf3wdMXVGDQSmnUWvsLEnmQyuPL0Ib5qVUUoV+yYFlMaZLdLwnjxKADg2qZ0qZII8F6n2Lsk9afIsSOoyBWETGLVhUeUH5aIxd1ryZO2yQKKCEleejSPSnhR0/NJpjXxao1iGrt2vXujwp2cQClSatk16byAJQDa1LiuA6g9GUJz958RJW/WS9d+lnNXNStqviLw5f+otI1CD1Axq8WJdW+C3epgQxQammiR6X5B3KsDEDEjxWdDg0JtW0owQxckNE8esUxyRZmyM1qGZUHlJ/Xyk1dq4h/aA5efIimbXEl5Wh5KI0sajCvi5F5vMl17Xr3RwUbOoDSpFWya1PxgHL/oJqUKI4DKH1ZQl6zF4H3HxnwRPjS35t2DZ8lj7nlT5+I5nUuq5qI8KbN36r5MAARO0Y0OjastlXdeLUbzMBl/JoTNHbVcWnMYRaLOfBJOf4ASq0YSiOLGszrYmQeb3Jdu979UcGGDqA0aZXs2lQOoDRnwXhFjB7Vc1KP6urZnOb0+Ga1cuDiPao/wZUlXzJrMprbscybNcEAzYYBCIQ0nBxRx7ZRrDt2nT6YvlNSOoImezCVb1cdp+/WnJCGZDVV2v/bOxcwm6r3j78yImKKmXHNrV/EkHJtmKLEMIrUv/5FKpdUSBf6Ny7FyHTTVUWX6aJQ/mVKKeRSGSrpQkhScvmFMJXb5O73fBf7/PY5zpmzz7HOnn32/q7nmcdjZu+19/rsd6/13e9a633NcSgjXQoz+asNMvJ4vMy1OZ3DTmdbYUxBaYVS8R5j19hfvK20dnUKSmucwh5ll1GZBeWK0R2lPD2UYZ9NsAPMuznhnSwqPEhUF3D5SeawS+3qJ8vrvVu6vMX2NM/RUjnpAAAWjElEQVQQEHWTy8mCIe3suejxq2DDlxNnPJ78ZI2MX/CLustYb0Q0f2gijzfyeVsthw4fkbEfrZZ6lctLj1Y1rZ5W5HEUlFowxrQSu8b+mDZCU+UUlJpA2mVUFJR6Htj23fulRc48Vdkdl/5LhhQRwFjPFd1Vy6rNO6XL+GMeyszGVWRCz2buamAxtea97/8tb365QRCL9V8psQvgXUzNi+qy4+b8JM9/eiz27vIYrxs3L4VxQvQHCsqoTMbWk+wa+21tVJQXo6CMElzgaXYZFQWlngf2594D0vTBuaqyAe3Olv/rFDpnr54ruquWHzfvkszxx+J4/k+zGoINOSwkEAsC5j5vZXaGnF46IRaXUXWaBaUT4tNSUMbsUWur2K6xX9sNx7AiCkpNcO0yKgpKPQ9sZ+FBaTLmWHgcTGtheovFOgFzYPibW9eW0V1TrZ/MI0kgAgLGRplalcrKZ0PbaY+9ar6VDQV7pe24Y9EfnJBBy8k51iN4hK4+1K6xPx4gUlBqekp2GdXDs1bLi5+vU3fNNZTRPzxzvMVbL64rwzIbRF+ZB89cs3W3ZDx9LDA8lwx40ABsbvLfhQfktFNLxjyKgDmcmBMEJWYCnl2wViVeOP+sM2ymzstZIWDX2G/lXor7GApKTU/ALqMyC8pYT//oQOPUKZvCA4ek4QNzVBP7pdeRkQ7b2aqDfSzrQIrMjsczDTlhajCWbWXd3iHw2469csnx+LRWMtl4hwxbGoqAXWN/PDwBCkpNT8kuo4o3Qdn1uUXyw793KsrrH+miifbJV7Pv4GE59/7ZqqLebWrLqCs4ZRsJVXMcykevbiz/20LPrtZI7oHHkoBuAgjq33zssc16k/q0lLb1knVfgvW5jIBdY388YKOg1PSU7DKqSV+sl1EfrFJ3vXpMJzUN5OSC8Dwv56+Tq5pWl9RqiY651YOHj8g5I2ap++EawMgfy6K1O+SGV5aoEyf2bCqdG1eNvBKeQQIOJPD/32ySgj0H5La2dWO6XtOBTectRUHArrE/iluz/RQKSk3I7TIq5HEe+s5yqV+lfERpwTQ10zXVHD16VOoM+1i1Z1r/C6VV3UquaZsdDdm2e5+0zJmvLrVkeHupXKGMHZflNUiABEjAUQTsGvsd1egQN0NBqekp0ag0gbSxmo0FhbJ11z6V6YUlcgIIbn5KiRLSsFqFyE/mGSRAAiTgAgIc+//7ECkoNRk0jUoTSFZDAiRAAiRAAnFCgGM/BaV2U6VRaUfKCkmABEiABEjA0QQ49lNQajdQGpV2pKyQBEiABEiABBxNgGM/BaV2A6VRaUfKCkmABEiABEjA0QQ49lNQajdQGpV2pKyQBEiABEiABBxNgGM/BaV2A6VRaUfKCkmABEiABEjA0QQ49lNQajdQGpV2pKyQBEiABEiABBxNgGM/BaV2A6VRaUfKCkmABEiABEjA0QQ49lNQajdQGpV2pKyQBEiABEiABBxNgGM/BaV2A6VRaUfKCkmABEiABEjA0QQ49lNQajdQGpV2pKyQBEiABEiABBxNgGM/BaV2A6VRaUfKCkmABEiABEjA0QQ49lNQajdQGpV2pKyQBEiABEiABBxNgGM/BaV2A6VRaUfKCkmABEiABEjA0QQ49lNQajdQGpV2pKyQBEiABEiABBxNgGM/BaV2A6VRaUfKCkmABEiABEjA0QQ49lNQajdQGpV2pKyQBEiABEiABBxNgGM/BaV2A6VRaUfKCkmABEiABEjA0QQ49lNQajdQGpV2pKyQBEiABEiABBxNgGM/BaV2A6VRaUfKCkmABEiABEjA0QQ49lNQajdQGpV2pKyQBEiABEiABBxNgGM/BaV2A6VRaUfKCkmABEiABEjA0QQ49lNQ+gj8/PPPMnjwYMnPz5dy5crJ9ddfL4888oicdtppERkxjSoiXDyYBEiABEiABOKeAMd+CkpF4O+//5ZGjRpJrVq15P7775dt27bJPffcI506dZLJkydHZOg0qohw8WASIAESIAESiHsCHPspKBWBRx99VMaMGSMbNmyQpKQk9bupU6dKz5495ccff5QGDRpYNnYalWVUPJAESIAESIAEXEGAYz8FpSLQtm1bOeOMM2TGjBk+Ivv375fExETJycmRIUOGWDZ4GpVlVDyQBEiABEiABFxBgGM/BaUikJKSIn369FFrJs0lNTVV0tLSJDc317LB06gso+KBJEACJEACJOAKAhz7KSgVgVKlSsmDDz4oWVlZfoadnp6uxGZeXl5Ig9+1a5fgxyibNm2S1q1by9dffy1Vq1Z1xYvCRpAACZAACZAACYQmsGXLFmnZsqX89ttvUrt2bU+jKnH06NGjXiUAQTl27Fi57777/BC0adNGqlSpItOnTw+JZvTo0ZKdne1VdGw3CZAACZAACZDAcQJwJrVo0cLTPDwtKE9myjvQQ7lv3z6Bl7JOnTqSkJCg1aiMLyB6P/+LlUz8TYw8TnzlyIRMrHTEtBPaycnYyaFDh2T79u3SuHFjKVOmjJWqXHuMpwWlzk05sbQQrtE4kS6Z+DMhD9qIlT6IdkI7oZ1YIUA7iYaSpwUlwgZhDSXCBlWqVEnxe/vtt1Vw80jDBkUD3+o5HAT4coezFdoIbSScjeDvtBPaCe3ECgHaSTSUPC0ojcDmWEhrDmyekZERcWDzaOBbPYeDAF/ucLZCG6GNhLMRCsrghPju8N3hu2OFQPhjPC0ogQepF++44w5ZtGiRlC1bVnkn4bmMNPVieNTRH4H1mk8++aTK4lOhQoXoK3LRmWTi/zDJ40TjJhMysdLl0U5oJ7QTKwTCH+N5QRkeEY8gARIgARIgARIgARIoigAFJe2DBEiABEiABEiABEjgpAhQUJ4UPp5MAiRAAiRAAiRAAiRAQUkbIAESIAESIAESIAESOCkCFJQnhS+2J2PD0ODBgyU/P1/KlSunNgwh77iTNgzpIPDOO+/IlClT5Ntvv5U///xTzj77bLn99tvl1ltvlVNOOcV3iY8//lhGjBghq1evlho1aqhNSgMGDDjhFh5//HF57rnnZOvWrSrY7Lhx46Rdu3Y6brXY6tizZ4+ce+658vvvv8vSpUulefPmnuXyyiuvyPjx42XNmjVqk9qFF14oH3zwgSd5vP/++/Lwww+rdwL9ArJ84f/169f3s1W3vju//PKL4H3/6quvZOXKleodwb+BRWf7d+/eLUOHDpV3331X9u/fL5deeqk8++yzUqtWrWLrH8wXDsfk8OHD8sQTT8hHH32kwuMhMDf6yVGjRkn79u2j6k/jnUlgozEWIZ0i3in0vYHFyhjjdCaxMFYKylhQ1VCnEdIInZQ5pFGnTp0cFdJIQ1OVIEA7u3fvLpUrV5ZPP/1UDYp33XWXEoMoX375pVx88cVy4403yg033CCLFy9WHeCLL74o/fr1890GXvThw4fLQw89JE2bNpWXX35ZMOgiyxA6zXgtSA86adIk+eOPP/wEpde4IOXpU089pT4sWrVqpT5AZs+erezAa3Yyb9486dixo3ofevXqJegzwAe7lletWuWLCOFmG5kxY4YMGjRI2QI+wI8cOXKCoNTd/ssvv1y+++47JcrwQfPAAw8o5j/88IMjPvbDMYFAwgf5TTfdJB06dBCkIH799ddl2rRp6sMM7TOK1f403pmYxwVko27durXKzQ1WgYLSLUxiMRZSUMaCqoY6EbpozJgxKuh6UlKSqnHq1KnSs2dPRwVd19BUlbYqOTnZryp4HydOnKgGydKlS0vnzp2VeFiyZInvuP79+8vMmTNVsGZ4MuEtgCDF7x977DF1HL7GISTPO+88FbQ+HstPP/2kPJIYwG677TY/QeklLvDC4VnC2wQhFax4iQc+pObPny/r1q2TEiVKKBz4cIK4AiOwQHEzEwhIYxbj5ptvlm+++eYEQamz/eh/8AEM715mZqbiu3HjRjWrAi8l3s/iLuGYoE+EAD7zzDN9twoRhT4GAhkf9ChW+1M3MDE/s1dffVU5NK655ho1E2IWlG5iEgs7paCMBVUNdcZLWkgNTQ1axZtvvqm8kZs3b5aKFSuqjg7T/Xfffbfv+M8//1xNZWMQadasmeoIMf0E78EFF1zgOy47O1uJsZ07d/oG3ljddyzqhXiCIIYX4JJLLvEJSnRuXuICL+17772nPFHBitd4wMP0/fffK8+YUcAG092G4PESk2CCUnf7MSsCkYGPW0PEgz3ey9NPP10+/PDDWHQBUdcZSmQHq7Bv374qHjOWkqBY7U/dxAQOjHr16glEJcYVeCPNgtKtTKI2sIATKSh1kdRcT0pKivTp00eJKHNJTU2VtLQ0yc3N1XxFZ1UHL+P06dNl27ZtqoNDu2fNmiWY8jcKPJvgBPGJab8JEybIwIEDpbCw0G/qCWs0r732Wtm0aZOa6omngnVaaNPatWuVUDYLSqx/8hIXfGTBW3/++ecrbxA6f7wLzzzzjPqd13gsXLhQrXlD0gNjyhtrrrGGDkITnn0vMQkmnnS3H/0IPJJYs2kueEfnzJmj2DupWBWU8GqiL8HHCJYIoVjtT93EBMsnfv31VzXWYPlIoKB0KxNdNktBqYuk5nqwrgV5xrOysvxqTk9PVyIqLy9P8xWdUx2+DCEU8OU7cuRItV4S7cZaKEw3GQWLycEJggIDaU5OjmK2b98+v8ZgrRnWCi1fvlx5+uKlQBhjkwE6NnxcfPbZZ36C0mtcMNjBY129enW1RvbUU08VeJ/Xr1+vBDfWDXrNTrDko0ePHoINACgNGzZUwsb4cPKSjQQTT7rbj36kZMmSat2uuaCfgtiA59JJxaqgRB+K2R/0MVirjmK1P3ULk2XLlqlxBx9jRr8bKCjdykSXzVJQ6iKpuR4IpbFjxwqm+cwFuzirVKmivHduLNiZjTVgGBDRuYGDMSjAK4C/BQpKTEEhfSZedjD7559//NDMnTtXrbnD1GA8bczB5iLcO9YoYZ1YKEHpFS7nnHOO8gBhFy+8KShbtmyROnXqqPXGeDcgKL3C44svvlDr+CAaunbtqpZ0QGjjgwrvDJZDeOndKUpQ6rIJiKeEhATlwTIXbBJ74YUXpKCgwFHdshVBiaVD6B/vvPNO39pzQ1Ba6U/dwARrSC+66CI1vmB5FEowD6XVMSbemOgyWgpKXSQ11+PFKW8MiFgTiQERa3kqVaqkqEY6bQVBWaZMGd8Ticcpb2zGwloerBnEjkMUMLniiivU2iYsoMfUWyRT3vHOBZ09uOCjw1ww3d2kSRP18eUlHrCBmjVr+s1WYBkIPsawqQAb27z07uiY8g73jrhpehfvED6y4ZHMyMhQmxbN60KN6V0vMEHbEYIO3snExETVvWC5GcLPYdMnxhP8uJWJLvlCQamLpOZ6vLYpByISnRqmLjG1bY7ppnthveZHFZPqDG9kqMohruBZ8NKmHAgGTDUGCkqISYSIgofISzzKli2rlsQgbI25wGMLz+Xzzz+vdup6hQk35ZzYWxTlocRaQXj0GzRooN4rLCExF7duQAnGBN5ILJ8JVfCxCoHpVia6BjEKSl0kNdeDsEFYDwiPjOGpw1cUgpvD64BOwC0FayGvuuoqwSYD/ARb54jQH9iEAbFpFIToQNy0wLBB+L2xmQkhMlAfprrjKWwQ2oo1PeaC/2OdE4RTixYtlIjyEhdsUEIojxUrVkijRo0UGgR6r1u3ruB9QdxSL/FAHwAvNuIOGgViG15LTM3de++96tdeYVJU2CBdfYcRIse8QRCb/WCDTgkbZO4zQjGBnWCJCD42jA/TwPHECJETrj91AxOsw8aPuRixOfGs8U7hGbuViS4tQUGpi6TmeozA5rVr1/YLbA4v3uTJkzVfrXirQ0acl156Sa3fwToWc8EmA3R6RnBidJCIxYm1YfDMhApsjik/CC7shscGpngPbA4mgWso8TsvccHHAbJXYAMK1nbBo4K1k0YkAGST8hIPTMdh7TB2GHfr1k19cGENJT5CsUGpatWq6lVyMxNsXEPMTRR4ZOF1w653FMzyIL6t7vYjfBemRs2BzbFcxymBzcMxQXgjbD4BK4wliN1rLuaNj0YQ73D9abwzCYyDDB7B1lDi925hEotRn4IyFlQ11YmYchgwsHYO01vwTsIT47bUixDNGASDFUwxGGkTMXBgo4o59SIGU3PB4mojLRayysAzCaGKcDvxXoIJSrTJS1wgHuGlRZzFgwcPKtGAzDnmVINe4QFbRyYorOvCZiUIBQhueCcDN5+5lQm8Spjit7PvQFBwI/XigQMHHJd6MRwT9LehmIEj7MooVvvTeGcSLDVvKEHpFiaxGA8pKGNBlXWSAAmQAAmQAAmQgIcIUFB66GGzqSRAAiRAAiRAAiQQCwIUlLGgyjpJgARIgARIgARIwEMEKCg99LDZVBIgARIgARIgARKIBQEKylhQZZ0kQAIkQAIkQAIk4CECFJQeethsKgmQQHQEQu2wj642a2cha8m4cePUjuJQBXnvEZPUHA3BWu08igRIgAT0EqCg1MuTtZEACbiQAMKiIKEAwvEg3qUdBfmnkTHKiCcZ7JoUlHY8CV6DBEjACgEKSiuUeAwJkEDcEECcOMQHLF26dNzcc7Q3SkEZLTmeRwIkoJsABaVuoqyPBEjAVgJGejkEsB82bJgKfD9lyhSVphFZUkaMGCFID5eQkCBdunSRp59+WlJSUnz3iNSdSC83f/58qVixokrhiED7M2fO9KVjCzbljfzzqBspPXfs2KHSIOL6PXr08NVt3Buy2iAgO5IVpKamysSJE6VZs2ZFcgo25Y0MQahrz5490rFjR+nXr59qE6e8bTU5XowESCAIAQpKmgUJkEBcE4BoQ0535LwfOXKknHXWWeoHIg8ZMDIzM6Vv376yd+9e9Xcch+lkFHgzmzdvLsiqhPRyiYmJKrMSBGXJkiWLFJRXX321ylIEkYfc4hCWyP/7xhtvSK9evVT9uDcI02rVqklWVpZKI4p/MYWO1HelSpUKyT5QUBqpFrGm8rLLLpNPPvlEpk2bpvKZU1DGtQnz5knAFQQoKF3xGNkIEvAuAYi2SZMmKS8kUg8aBWkZDx06pFKXQpyhIMc11kFC5EFoQhDCw7dw4UJfHnmIvRo1aihvJdLYoQR6KJG3uUmTJip/9IABA3zXzMjIkDVr1vjOw71BYK5YsUJ5JlHmzZsnHTp0kPz8fElPT7ckKJHHvGbNmtK+fXtVn1HgDX3rrbcoKL1r/mw5CTiGAAWlYx4Fb4QESCAaAhBtyO29fft23+mFhYXKG4i87oMGDfKrFhtd+vfvL6NGjZLs7Gw1Bf7XX3/5HdOtWzdZvnx5SEEJIYl64QWFx9Mo8FD27t1bNm7cqLykuDcISEyrG2Xz5s1SvXp1JQSvu+46S4ISHlPkYM7Ly5Pu3bv7zsH/4SmlhzIay+E5JEACOglQUOqkybpIgARsJwDRtnTpUuV9NAqmgeFlDFUwBZ6bm6vWTi5YsECtbTSXW265RebOnRtSUGKaG2L04MGDfufNnj1bOnfuLMuWLVMeTGMN5cqVK33HQYQmJyfLa6+9pv4eqpinvDFFn5aWprytbdq08Z2yePFi5eWkoLTd7HhBEiCBAAIUlDQJEiCBuCYQTLRhvWT58uVl+PDhcuWVV57QvqSkJOXxi9ZDOWHCBBk4cKAUFBSoqXGjBPNQYif2yQpKeijj2kR58yTgCQL/AfxoIFZTV3YzAAAAAElFTkSuQmCC\" width=\"599.9999869953505\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pb_num_dict = {_k:len(_v.kept_probes) for _k,_v in filtered_pb_dict.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(pb_nums)\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"region id\", labelpad=1)\n",
    "plt.ylim([0, num_probes_kept*1.1])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 118,
   "id": "d87753a7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\chr_2_final_region_num.pkl\n"
     ]
    }
   ],
   "source": [
    "chr_2_region_savefile = os.path.join(library_folder, 'chr_2_final_region_num.pkl')\n",
    "overwrite = True\n",
    "\n",
    "filtered_reg_names = {}\n",
    "for _reg, _pbde in filtered_pb_dict.items():\n",
    "    if len(_pbde.kept_probes) > 0:\n",
    "        filtered_reg_names[_reg] = list(_pbde.kept_probes.values())[0]['reg_name']\n",
    "\n",
    "if not os.path.exists(chr_2_region_savefile) or overwrite:\n",
    "    chr_2_region_num = {}\n",
    "    for _reg, _info in filtered_reg_names.items():\n",
    "        _chr = _info.split('_')[0].split(':')[0]\n",
    "        if _chr not in chr_2_region_num:\n",
    "            chr_2_region_num[_chr] = 1\n",
    "        else:\n",
    "            chr_2_region_num[_chr] += 1\n",
    "    print(f\"Saving into file: {chr_2_region_savefile}\")\n",
    "\n",
    "    pickle.dump(chr_2_region_num, open(chr_2_region_savefile, 'wb'))\n",
    "else:\n",
    "    print(f\"loading from file: {chr_2_region_savefile}\")\n",
    "    chr_2_region_num = pickle.load(open(chr_2_region_savefile, 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "id": "28386f9b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'1': 84,\n",
       " '2': 90,\n",
       " '14': 56,\n",
       " '15': 49,\n",
       " '16': 48,\n",
       " '17': 46,\n",
       " '18': 47,\n",
       " '19': 53,\n",
       " '20': 60,\n",
       " '21': 30,\n",
       " '22': 29,\n",
       " 'X': 59,\n",
       " 'Y': 14,\n",
       " '3': 77,\n",
       " '4': 74,\n",
       " '5': 69,\n",
       " '6': 64,\n",
       " '7': 59,\n",
       " '8': 55,\n",
       " '9': 53,\n",
       " '10': 65,\n",
       " '11': 61,\n",
       " '12': 62,\n",
       " '13': 48}"
      ]
     },
     "execution_count": 119,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chr_2_region_num"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "f9b134b0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'1': 84,\n",
       " '2': 90,\n",
       " '3': 77,\n",
       " '4': 74,\n",
       " '5': 69,\n",
       " '6': 64,\n",
       " '7': 59,\n",
       " '8': 55,\n",
       " '9': 53,\n",
       " '10': 65,\n",
       " '11': 61,\n",
       " '12': 62,\n",
       " '13': 48,\n",
       " '14': 56,\n",
       " '15': 49,\n",
       " '16': 48,\n",
       " '17': 46,\n",
       " '18': 47,\n",
       " '19': 53,\n",
       " '20': 60,\n",
       " '21': 30,\n",
       " '22': 29,\n",
       " 'X': 59,\n",
       " 'Y': 14}"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chr_2_region_num"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "id": "4a37d6da",
   "metadata": {},
   "outputs": [],
   "source": [
    "#\n",
    "reg_2_start = {}\n",
    "\n",
    "for _name, _info in filtered_reg_names.items():\n",
    "    _locus = _info.split('_')[0]\n",
    "    _start,_end = _locus.split(':')[1].split('-')\n",
    "    reg_2_start[_name] = int(_start)\n",
    "#    \n",
    "chr_2_region_names = {}\n",
    "\n",
    "for _reg, _info in filtered_reg_names.items():\n",
    "    _chr = _info.split('_')[0].split(':')[0]\n",
    "    if _chr not in chr_2_region_names:\n",
    "        chr_2_region_names[_chr] = [_reg]\n",
    "    else:\n",
    "        chr_2_region_names[_chr].append(_reg)\n",
    "# sort\n",
    "for _chr, _regs in chr_2_region_names.items():\n",
    "    _sorted_regs = sorted(_regs, key=lambda _r: reg_2_start[_r])\n",
    "    chr_2_region_names[_chr] = _sorted_regs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "id": "f467b939",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'1': ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31', '32', '33', '34', '35', '36', '37', '38', '39', '40', '42', '43', '44', '45', '46', '47', '57', '60', '62', '63', '64', '65', '66', '67', '68', '69', '70', '71', '72', '73', '74', '75', '76', '77', '78', '79', '80', '81', '83', '84', '85', '86', '87', '88', '89', '90', '91', '92', '93', '94', '95', '96', '98'], '2': ['100', '101', '102', '103', '104', '105', '106', '107', '108', '109', '110', '111', '112', '114', '115', '116', '117', '118', '119', '120', '121', '122', '123', '124', '125', '126', '127', '128', '129', '130', '131', '132', '133', '134', '137', '138', '139', '140', '141', '143', '144', '145', '146', '147', '148', '149', '150', '152', '153', '154', '155', '156', '157', '158', '159', '160', '161', '162', '163', '164', '165', '166', '167', '168', '169', '170', '171', '172', '173', '174', '175', '176', '177', '178', '179', '180', '181', '182', '183', '184', '185', '186', '187', '188', '189', '190', '191', '192', '193', '194'], '14': ['949', '950', '951', '952', '953', '954', '955', '956', '957', '958', '959', '960', '961', '962', '963', '964', '965', '966', '967', '968', '969', '970', '971', '972', '973', '974', '975', '976', '977', '978', '979', '980', '981', '982', '983', '984', '985', '986', '987', '988', '989', '990', '991', '992', '993', '994', '995', '996', '997', '999', '1000', '1001', '1002', '1003', '1004', '1005'], '15': ['1020', '1023', '1024', '1025', '1029', '1030', '1031', '1032', '1033', '1034', '1035', '1036', '1037', '1038', '1039', '1040', '1041', '1042', '1043', '1044', '1045', '1046', '1047', '1048', '1049', '1050', '1051', '1052', '1053', '1054', '1055', '1056', '1057', '1058', '1059', '1060', '1061', '1062', '1063', '1064', '1065', '1066', '1067', '1068', '1069', '1070', '1071', '1072', '1073'], '16': ['1074', '1075', '1076', '1077', '1078', '1079', '1080', '1081', '1082', '1083', '1085', '1087', '1089', '1090', '1091', '1092', '1093', '1094', '1097', '1105', '1106', '1107', '1108', '1109', '1110', '1111', '1112', '1113', '1114', '1115', '1116', '1117', '1118', '1119', '1120', '1121', '1122', '1123', '1124', '1125', '1126', '1127', '1128', '1129', '1130', '1131', '1132', '1133'], '17': ['1134', '1135', '1136', '1137', '1138', '1139', '1140', '1141', '1142', '1143', '1144', '1145', '1146', '1147', '1148', '1152', '1153', '1154', '1155', '1156', '1157', '1160', '1162', '1163', '1165', '1166', '1167', '1168', '1169', '1170', '1171', '1172', '1173', '1174', '1176', '1177', '1178', '1179', '1180', '1181', '1182', '1183', '1184', '1186', '1187', '1188'], '18': ['1189', '1190', '1191', '1192', '1193', '1194', '1195', '1196', '1197', '1203', '1204', '1205', '1206', '1207', '1208', '1209', '1210', '1211', '1212', '1213', '1214', '1215', '1216', '1217', '1218', '1219', '1220', '1221', '1222', '1224', '1225', '1226', '1227', '1228', '1229', '1230', '1231', '1232', '1233', '1234', '1235', '1236', '1237', '1238', '1239', '1240', '1241'], '19': ['1242', '1243', '1244', '1245', '1246', '1247', '1248', '1249', '1250', '1251', '1252', '1253', '1254', '1255', '1256', '1257', '1258', '1259', '1260', '1261', '1262', '1263', '1264', '1265', '1269', '1270', '1271', '1272', '1273', '1274', '1275', '1276', '1277', '1278', '1279', '1280', '1281', '1282', '1284', '1285', '1286', '1287', '1288', '1289', '1290', '1291', '1292', '1293', '1294', '1295', '1297', '1298', '1299'], '20': ['1300', '1301', '1302', '1303', '1304', '1305', '1306', '1307', '1308', '1309', '1310', '1311', '1312', '1313', '1314', '1315', '1316', '1317', '1318', '1319', '1320', '1321', '1322', '1323', '1324', '1325', '1330', '1331', '1332', '1333', '1334', '1335', '1336', '1337', '1338', '1339', '1340', '1341', '1342', '1343', '1344', '1345', '1346', '1347', '1348', '1349', '1350', '1351', '1352', '1353', '1354', '1355', '1356', '1357', '1358', '1359', '1360', '1361', '1362', '1363'], '21': ['1378', '1379', '1380', '1381', '1382', '1383', '1384', '1385', '1386', '1387', '1388', '1389', '1390', '1391', '1392', '1393', '1394', '1395', '1396', '1398', '1399', '1400', '1401', '1402', '1403', '1404', '1405', '1406', '1408', '1409'], '22': ['1427', '1429', '1430', '1434', '1435', '1436', '1437', '1438', '1439', '1440', '1441', '1442', '1443', '1444', '1445', '1446', '1447', '1448', '1449', '1450', '1451', '1452', '1453', '1454', '1455', '1456', '1457', '1458', '1459'], 'X': ['1461', '1462', '1463', '1464', '1465', '1466', '1467', '1468', '1469', '1470', '1471', '1472', '1473', '1474', '1475', '1476', '1477', '1478', '1479', '1480', '1481', '1482', '1485', '1486', '1487', '1488', '1489', '1490', '1491', '1492', '1493', '1494', '1495', '1496', '1497', '1498', '1499', '1500', '1501', '1502', '1503', '1504', '1505', '1506', '1507', '1508', '1509', '1510', '1511', '1512', '1513', '1514', '1515', '1516', '1517', '1518', '1519', '1520', '1521'], 'Y': ['1525', '1526', '1527', '1529', '1530', '1533', '1534', '1535', '1536', '1537', '1538', '1541', '1542', '1548'], '3': ['195', '196', '197', '198', '199', '200', '201', '202', '203', '204', '205', '206', '207', '208', '209', '210', '211', '212', '213', '214', '215', '216', '217', '218', '219', '220', '221', '222', '223', '224', '225', '226', '227', '228', '229', '230', '233', '234', '235', '236', '237', '238', '239', '240', '241', '242', '243', '244', '245', '246', '247', '248', '249', '250', '251', '252', '253', '254', '255', '256', '257', '258', '259', '260', '261', '262', '263', '264', '265', '266', '267', '268', '269', '270', '271', '272', '273'], '4': ['274', '275', '276', '277', '278', '279', '280', '281', '282', '283', '284', '285', '286', '287', '288', '289', '290', '291', '292', '293', '295', '296', '297', '298', '299', '300', '302', '303', '304', '305', '306', '307', '308', '309', '310', '311', '312', '313', '314', '315', '316', '317', '318', '319', '320', '321', '322', '323', '324', '325', '326', '327', '328', '329', '330', '331', '332', '333', '334', '335', '336', '337', '338', '339', '340', '341', '342', '343', '344', '345', '346', '347', '348', '349'], '5': ['350', '351', '352', '353', '354', '355', '356', '357', '358', '359', '360', '361', '362', '363', '364', '365', '366', '367', '368', '370', '371', '372', '373', '374', '375', '376', '377', '379', '380', '381', '382', '383', '384', '385', '386', '387', '388', '389', '390', '391', '392', '393', '394', '395', '396', '397', '398', '399', '400', '401', '402', '403', '404', '405', '406', '407', '408', '409', '410', '411', '412', '413', '414', '415', '416', '417', '418', '419', '421'], '6': ['422', '423', '424', '425', '426', '427', '428', '429', '430', '431', '432', '435', '436', '437', '438', '439', '440', '441', '442', '443', '444', '446', '447', '448', '449', '450', '451', '452', '453', '454', '455', '456', '457', '458', '459', '460', '461', '462', '463', '464', '465', '466', '467', '468', '469', '470', '471', '472', '473', '474', '475', '476', '477', '478', '479', '480', '481', '482', '483', '484', '485', '486', '487', '489'], '7': ['490', '491', '492', '493', '494', '495', '496', '497', '498', '499', '500', '501', '502', '503', '504', '505', '506', '507', '508', '509', '510', '511', '512', '516', '517', '518', '519', '520', '521', '522', '523', '524', '525', '526', '527', '528', '529', '530', '531', '532', '533', '534', '535', '536', '537', '538', '539', '540', '541', '542', '543', '544', '545', '546', '548', '549', '550', '551', '552'], '8': ['554', '555', '556', '557', '558', '559', '560', '561', '562', '563', '564', '565', '566', '567', '568', '569', '570', '571', '572', '573', '574', '575', '576', '577', '578', '579', '580', '581', '582', '583', '584', '585', '586', '587', '588', '589', '590', '591', '592', '593', '594', '595', '596', '597', '598', '599', '600', '601', '602', '603', '605', '606', '607', '608', '609'], '9': ['611', '612', '613', '614', '615', '616', '617', '618', '619', '620', '621', '622', '623', '624', '625', '626', '627', '628', '629', '645', '646', '647', '648', '649', '650', '651', '652', '653', '654', '655', '656', '657', '658', '659', '660', '662', '663', '664', '665', '666', '667', '668', '669', '670', '671', '672', '673', '674', '675', '676', '677', '678', '679'], '10': ['680', '681', '682', '683', '684', '685', '686', '687', '688', '689', '690', '691', '692', '693', '694', '695', '696', '697', '698', '699', '701', '702', '703', '704', '705', '706', '707', '708', '709', '710', '711', '712', '713', '714', '715', '716', '717', '718', '719', '720', '721', '722', '723', '724', '725', '726', '727', '728', '729', '730', '731', '732', '733', '734', '735', '736', '737', '738', '739', '740', '741', '742', '743', '744', '745'], '11': ['748', '749', '750', '751', '752', '753', '754', '755', '756', '757', '758', '759', '760', '761', '762', '763', '764', '765', '766', '767', '768', '769', '770', '774', '775', '776', '777', '778', '779', '780', '782', '783', '784', '785', '786', '787', '788', '789', '790', '791', '792', '793', '794', '795', '796', '797', '798', '799', '800', '801', '802', '803', '804', '805', '806', '807', '808', '809', '810', '811', '812'], '12': ['813', '814', '815', '816', '817', '819', '820', '821', '822', '823', '824', '825', '826', '827', '828', '829', '832', '833', '834', '835', '836', '837', '838', '839', '840', '841', '843', '844', '845', '846', '847', '848', '849', '850', '851', '852', '853', '854', '855', '856', '857', '858', '859', '860', '861', '862', '863', '864', '865', '866', '867', '868', '869', '870', '871', '872', '873', '874', '875', '876', '877', '878'], '13': ['888', '889', '890', '891', '892', '893', '894', '895', '896', '897', '898', '899', '900', '901', '902', '903', '904', '905', '906', '907', '908', '909', '910', '911', '912', '913', '914', '915', '916', '917', '918', '919', '920', '921', '922', '923', '924', '925', '926', '927', '928', '929', '930', '931', '932', '933', '934', '935']}\n"
     ]
    }
   ],
   "source": [
    "print(chr_2_region_names)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "id": "5469fa71",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\chr_2_region_names.pkl\n"
     ]
    }
   ],
   "source": [
    "overwrite_region_names = True\n",
    "chr_2_region_name_savefile = os.path.join(library_folder, 'chr_2_region_names.pkl')\n",
    "\n",
    "if not os.path.exists(chr_2_region_name_savefile) or overwrite_region_names:\n",
    "    print(chr_2_region_name_savefile)\n",
    "    pickle.dump(chr_2_region_names, open(chr_2_region_name_savefile, 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "id": "79fb2c86",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'name': '15:80235001-80265000_strand_+_gene_1060_reg_1060_pb_23',\n",
       " 'reg_index': 1060,\n",
       " 'reg_name': '15:80235001-80265000_strand_+_gene_1060',\n",
       " 'pb_index': 23,\n",
       " 'strand': '+',\n",
       " 'gc': 0.5714285714285715,\n",
       " 'tm': 82.43016918757803,\n",
       " 'map_genome': 46,\n",
       " 'map_transcriptome': 0,\n",
       " 'map_rep_genome': 0,\n",
       " 'map_self_sequences': 26}"
      ]
     },
     "execution_count": 124,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(list(filtered_pb_dict.values())[51].kept_probes.values())[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "id": "42c9910a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# clear memory\n",
    "del(_pbde)\n",
    "del(_pb_class)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2c3d9549",
   "metadata": {},
   "source": [
    "<a id='3'></a>\n",
    "# 3. Load and asssemble probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d373567a",
   "metadata": {},
   "source": [
    "<a id='3.1'></a>\n",
    "## 3.1 design readout mapping dict (region_to_readout)\n",
    "\n",
    "load directly from http://localhost:8889/notebooks/Chromatin_Analysis_Scripts/Library_design/CTP-10_Aire/CTP-10_Aire_introns_DNA-MERFISH.ipynb#3.3-assemble-probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "id": "136b756f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1352"
      ]
     },
     "execution_count": 128,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# summarize total readout usage\n",
    "region_2_readout_dict = pickle.load(open(os.path.join(library_folder, 'region_2_readout.pkl'), 'rb'))\n",
    "region_2_readout_dict = {str(_r):_bs for _r,_bs in region_2_readout_dict.items()}\n",
    "len(region_2_readout_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 129,
   "id": "55ba1861",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'100': ['c28', 'c43', 'c97'],\n",
       " '101': ['c5', 'c12', 'c60'],\n",
       " '102': ['c15', 'c53', 'c67'],\n",
       " '103': ['c3', 'c57', 'c80'],\n",
       " '104': ['c4', 'c27', 'c87'],\n",
       " '105': ['c37', 'c47', 'c69'],\n",
       " '106': ['c1', 'c68', 'c85'],\n",
       " '107': ['c21', 'c33', 'c91'],\n",
       " '108': ['c9', 'c70', 'c94'],\n",
       " '109': ['c45', 'c56', 'c65'],\n",
       " '110': ['c13', 'c76', 'c84'],\n",
       " '111': ['c22', 'c25', 'c78'],\n",
       " '112': ['c11', 'c26', 'c49'],\n",
       " '114': ['c46', 'c62', 'c88'],\n",
       " '115': ['c6', 'c24', 'c30'],\n",
       " '116': ['c7', 'c29', 'c32'],\n",
       " '117': ['c16', 'c90', 'c95'],\n",
       " '118': ['c34', 'c44', 'c93'],\n",
       " '119': ['c19', 'c83', 'c85'],\n",
       " '120': ['c8', 'c18', 'c58'],\n",
       " '121': ['c35', 'c39', 'c72'],\n",
       " '122': ['c45', 'c57', 'c81'],\n",
       " '123': ['c41', 'c71', 'c97'],\n",
       " '124': ['c17', 'c43', 'c96'],\n",
       " '125': ['c20', 'c28', 'c36'],\n",
       " '126': ['c23', 'c31', 'c55'],\n",
       " '127': ['c53', 'c73', 'c82'],\n",
       " '128': ['c62', 'c64', 'c79'],\n",
       " '129': ['c74', 'c84', 'c95'],\n",
       " '130': ['c60', 'c61', 'c89'],\n",
       " '131': ['c0', 'c24', 'c25'],\n",
       " '132': ['c5', 'c10', 'c54'],\n",
       " '133': ['c4', 'c40', 'c70'],\n",
       " '134': ['c48', 'c66', 'c85'],\n",
       " '137': ['c16', 'c63', 'c76'],\n",
       " '138': ['c78', 'c86', 'c94'],\n",
       " '139': ['c15', 'c36', 'c56'],\n",
       " '140': ['c13', 'c81', 'c91'],\n",
       " '141': ['c8', 'c35', 'c68'],\n",
       " '143': ['c75', 'c77', 'c83'],\n",
       " '144': ['c2', 'c33', 'c71'],\n",
       " '145': ['c11', 'c17', 'c29'],\n",
       " '146': ['c67', 'c92', 'c98'],\n",
       " '147': ['c7', 'c47', 'c72'],\n",
       " '148': ['c10', 'c50', 'c74'],\n",
       " '149': ['c9', 'c41', 'c79'],\n",
       " '150': ['c49', 'c58', 'c65'],\n",
       " '152': ['c0', 'c51', 'c59'],\n",
       " '153': ['c44', 'c69', 'c80'],\n",
       " '154': ['c22', 'c26', 'c40'],\n",
       " '155': ['c14', 'c18', 'c27'],\n",
       " '156': ['c32', 'c86', 'c88'],\n",
       " '157': ['c3', 'c91', 'c93'],\n",
       " '158': ['c21', 'c30', 'c70'],\n",
       " '159': ['c12', 'c34', 'c42'],\n",
       " '160': ['c31', 'c37', 'c78'],\n",
       " '161': ['c15', 'c23', 'c50'],\n",
       " '162': ['c19', 'c24', 'c84'],\n",
       " '163': ['c38', 'c42', 'c98'],\n",
       " '164': ['c52', 'c55', 'c73'],\n",
       " '165': ['c28', 'c29', 'c63'],\n",
       " '166': ['c2', 'c43', 'c59'],\n",
       " '167': ['c6', 'c67', 'c87'],\n",
       " '168': ['c17', 'c64', 'c90'],\n",
       " '169': ['c14', 'c48', 'c68'],\n",
       " '170': ['c26', 'c39', 'c92'],\n",
       " '171': ['c3', 'c86', 'c89'],\n",
       " '172': ['c4', 'c54', 'c81'],\n",
       " '173': ['c61', 'c74', 'c79'],\n",
       " '174': ['c2', 'c57', 'c82'],\n",
       " '175': ['c38', 'c53', 'c72'],\n",
       " '176': ['c1', 'c31', 'c52'],\n",
       " '177': ['c20', 'c21', 'c46'],\n",
       " '178': ['c51', 'c65', 'c76'],\n",
       " '179': ['c5', 'c66', 'c88'],\n",
       " '180': ['c56', 'c96', 'c97'],\n",
       " '181': ['c9', 'c47', 'c75'],\n",
       " '182': ['c22', 'c71', 'c77'],\n",
       " '183': ['c37', 'c58', 'c63'],\n",
       " '184': ['c40', 'c44', 'c60'],\n",
       " '185': ['c8', 'c80', 'c87'],\n",
       " '186': ['c19', 'c64', 'c93'],\n",
       " '187': ['c12', 'c62', 'c95'],\n",
       " '188': ['c6', 'c45', 'c83'],\n",
       " '189': ['c7', 'c33', 'c35'],\n",
       " '190': ['c11', 'c36', 'c94'],\n",
       " '191': ['c0', 'c30', 'c69'],\n",
       " '192': ['c25', 'c49', 'c90'],\n",
       " '193': ['c16', 'c27', 'c32'],\n",
       " '194': ['c1', 'c13', 'c46'],\n",
       " '0': ['c13', 'c47', 'c55'],\n",
       " '1': ['c10', 'c46', 'c72'],\n",
       " '2': ['c12', 'c58', 'c81'],\n",
       " '3': ['c1', 'c65', 'c78'],\n",
       " '4': ['c7', 'c73', 'c89'],\n",
       " '5': ['c14', 'c52', 'c54'],\n",
       " '6': ['c42', 'c50', 'c66'],\n",
       " '7': ['c23', 'c34', 'c38'],\n",
       " '8': ['c26', 'c28', 'c60'],\n",
       " '9': ['c45', 'c85', 'c86'],\n",
       " '10': ['c22', 'c31', 'c74'],\n",
       " '11': ['c39', 'c77', 'c90'],\n",
       " '12': ['c18', 'c79', 'c96'],\n",
       " '13': ['c25', 'c47', 'c93'],\n",
       " '14': ['c24', 'c46', 'c53'],\n",
       " '15': ['c8', 'c88', 'c97'],\n",
       " '16': ['c15', 'c55', 'c61'],\n",
       " '17': ['c14', 'c89', 'c98'],\n",
       " '18': ['c16', 'c75', 'c92'],\n",
       " '19': ['c17', 'c26', 'c65'],\n",
       " '20': ['c49', 'c83', 'c94'],\n",
       " '21': ['c11', 'c19', 'c40'],\n",
       " '22': ['c41', 'c42', 'c48'],\n",
       " '23': ['c21', 'c62', 'c68'],\n",
       " '24': ['c34', 'c35', 'c57'],\n",
       " '25': ['c51', 'c58', 'c73'],\n",
       " '26': ['c66', 'c67', 'c71'],\n",
       " '27': ['c3', 'c72', 'c87'],\n",
       " '28': ['c0', 'c27', 'c84'],\n",
       " '29': ['c69', 'c81', 'c90'],\n",
       " '30': ['c5', 'c26', 'c70'],\n",
       " '31': ['c7', 'c25', 'c31'],\n",
       " '32': ['c20', 'c77', 'c82'],\n",
       " '33': ['c19', 'c91', 'c98'],\n",
       " '34': ['c10', 'c23', 'c89'],\n",
       " '35': ['c29', 'c44', 'c56'],\n",
       " '36': ['c2', 'c4', 'c32'],\n",
       " '37': ['c6', 'c9', 'c64'],\n",
       " '38': ['c18', 'c41', 'c59'],\n",
       " '39': ['c12', 'c13', 'c28'],\n",
       " '40': ['c21', 'c43', 'c57'],\n",
       " '42': ['c22', 'c36', 'c48'],\n",
       " '43': ['c24', 'c78', 'c83'],\n",
       " '44': ['c3', 'c30', 'c92'],\n",
       " '45': ['c77', 'c86', 'c96'],\n",
       " '46': ['c38', 'c54', 'c97'],\n",
       " '47': ['c0', 'c52', 'c75'],\n",
       " '57': ['c29', 'c76', 'c94'],\n",
       " '60': ['c9', 'c33', 'c37'],\n",
       " '62': ['c25', 'c51', 'c55'],\n",
       " '63': ['c17', 'c46', 'c61'],\n",
       " '64': ['c43', 'c63', 'c80'],\n",
       " '65': ['c2', 'c44', 'c45'],\n",
       " '66': ['c6', 'c49', 'c95'],\n",
       " '67': ['c12', 'c40', 'c85'],\n",
       " '68': ['c37', 'c53', 'c76'],\n",
       " '69': ['c18', 'c66', 'c82'],\n",
       " '70': ['c5', 'c27', 'c33'],\n",
       " '71': ['c4', 'c74', 'c86'],\n",
       " '72': ['c36', 'c81', 'c88'],\n",
       " '73': ['c20', 'c30', 'c32'],\n",
       " '74': ['c11', 'c79', 'c98'],\n",
       " '75': ['c1', 'c16', 'c87'],\n",
       " '76': ['c50', 'c59', 'c92'],\n",
       " '77': ['c24', 'c35', 'c71'],\n",
       " '78': ['c0', 'c80', 'c95'],\n",
       " '79': ['c23', 'c70', 'c84'],\n",
       " '80': ['c10', 'c39', 'c64'],\n",
       " '81': ['c31', 'c56', 'c83'],\n",
       " '83': ['c60', 'c62', 'c85'],\n",
       " '84': ['c41', 'c52', 'c68'],\n",
       " '85': ['c15', 'c42', 'c82'],\n",
       " '86': ['c63', 'c72', 'c78'],\n",
       " '87': ['c47', 'c65', 'c67'],\n",
       " '88': ['c7', 'c91', 'c97'],\n",
       " '89': ['c13', 'c50', 'c93'],\n",
       " '90': ['c8', 'c36', 'c59'],\n",
       " '91': ['c28', 'c69', 'c88'],\n",
       " '92': ['c48', 'c51', 'c96'],\n",
       " '93': ['c17', 'c40', 'c58'],\n",
       " '94': ['c14', 'c38', 'c61'],\n",
       " '95': ['c54', 'c73', 'c75'],\n",
       " '96': ['c20', 'c34', 'c39'],\n",
       " '98': ['c1', 'c22', 'c62'],\n",
       " '195': ['c11', 'c77', 'c93'],\n",
       " '196': ['c8', 'c44', 'c65'],\n",
       " '197': ['c50', 'c87', 'c89'],\n",
       " '198': ['c21', 'c35', 'c41'],\n",
       " '199': ['c52', 'c60', 'c69'],\n",
       " '200': ['c2', 'c54', 'c55'],\n",
       " '201': ['c47', 'c62', 'c91'],\n",
       " '202': ['c3', 'c25', 'c40'],\n",
       " '203': ['c28', 'c39', 'c79'],\n",
       " '204': ['c20', 'c33', 'c59'],\n",
       " '205': ['c37', 'c70', 'c96'],\n",
       " '206': ['c31', 'c36', 'c84'],\n",
       " '207': ['c51', 'c64', 'c75'],\n",
       " '208': ['c27', 'c34', 'c63'],\n",
       " '209': ['c10', 'c44', 'c73'],\n",
       " '210': ['c9', 'c74', 'c92'],\n",
       " '211': ['c32', 'c93', 'c98'],\n",
       " '212': ['c19', 'c21', 'c94'],\n",
       " '213': ['c17', 'c72', 'c83'],\n",
       " '214': ['c1', 'c24', 'c26'],\n",
       " '215': ['c46', 'c76', 'c97'],\n",
       " '216': ['c16', 'c78', 'c91'],\n",
       " '217': ['c30', 'c55', 'c77'],\n",
       " '218': ['c29', 'c45', 'c49'],\n",
       " '219': ['c38', 'c57', 'c68'],\n",
       " '220': ['c6', 'c23', 'c42'],\n",
       " '221': ['c48', 'c67', 'c82'],\n",
       " '222': ['c53', 'c61', 'c85'],\n",
       " '223': ['c5', 'c64', 'c73'],\n",
       " '224': ['c11', 'c15', 'c27'],\n",
       " '225': ['c43', 'c50', 'c94'],\n",
       " '226': ['c18', 'c65', 'c81'],\n",
       " '227': ['c10', 'c56', 'c69'],\n",
       " '228': ['c22', 'c32', 'c72'],\n",
       " '229': ['c2', 'c75', 'c80'],\n",
       " '230': ['c34', 'c66', 'c95'],\n",
       " '233': ['c3', 'c9', 'c19'],\n",
       " '234': ['c14', 'c90', 'c92'],\n",
       " '235': ['c12', 'c87', 'c91'],\n",
       " '236': ['c46', 'c58', 'c78'],\n",
       " '237': ['c7', 'c13', 'c83'],\n",
       " '238': ['c30', 'c35', 'c84'],\n",
       " '239': ['c60', 'c64', 'c71'],\n",
       " '240': ['c0', 'c86', 'c97'],\n",
       " '241': ['c24', 'c28', 'c40'],\n",
       " '242': ['c16', 'c21', 'c77'],\n",
       " '243': ['c8', 'c20', 'c56'],\n",
       " '244': ['c41', 'c43', 'c69'],\n",
       " '245': ['c25', 'c52', 'c88'],\n",
       " '246': ['c4', 'c5', 'c15'],\n",
       " '247': ['c37', 'c42', 'c89'],\n",
       " '248': ['c45', 'c67', 'c79'],\n",
       " '249': ['c36', 'c58', 'c85'],\n",
       " '250': ['c27', 'c54', 'c95'],\n",
       " '251': ['c38', 'c44', 'c70'],\n",
       " '252': ['c23', 'c39', 'c48'],\n",
       " '253': ['c59', 'c76', 'c96'],\n",
       " '254': ['c49', 'c60', 'c66'],\n",
       " '255': ['c6', 'c71', 'c80'],\n",
       " '256': ['c29', 'c68', 'c90'],\n",
       " '257': ['c3', 'c18', 'c33'],\n",
       " '258': ['c20', 'c53', 'c57'],\n",
       " '259': ['c26', 'c41', 'c61'],\n",
       " '260': ['c1', 'c47', 'c94'],\n",
       " '261': ['c10', 'c51', 'c62'],\n",
       " '262': ['c4', 'c63', 'c93'],\n",
       " '263': ['c12', 'c43', 'c98'],\n",
       " '264': ['c7', 'c22', 'c30'],\n",
       " '265': ['c0', 'c2', 'c31'],\n",
       " '266': ['c16', 'c81', 'c86'],\n",
       " '267': ['c9', 'c13', 'c88'],\n",
       " '268': ['c8', 'c82', 'c95'],\n",
       " '269': ['c11', 'c14', 'c73'],\n",
       " '270': ['c17', 'c19', 'c71'],\n",
       " '271': ['c54', 'c56', 'c74'],\n",
       " '272': ['c50', 'c84', 'c90'],\n",
       " '273': ['c39', 'c87', 'c96'],\n",
       " '274': ['c5', 'c6', 'c13'],\n",
       " '275': ['c48', 'c79', 'c80'],\n",
       " '276': ['c59', 'c61', 'c75'],\n",
       " '277': ['c49', 'c55', 'c82'],\n",
       " '278': ['c58', 'c72', 'c90'],\n",
       " '279': ['c2', 'c10', 'c17'],\n",
       " '280': ['c16', 'c51', 'c69'],\n",
       " '281': ['c12', 'c14', 'c31'],\n",
       " '282': ['c38', 'c81', 'c83'],\n",
       " '283': ['c43', 'c52', 'c84'],\n",
       " '284': ['c21', 'c26', 'c63'],\n",
       " '285': ['c23', 'c27', 'c85'],\n",
       " '286': ['c7', 'c39', 'c62'],\n",
       " '287': ['c86', 'c87', 'c93'],\n",
       " '288': ['c8', 'c19', 'c46'],\n",
       " '289': ['c42', 'c59', 'c68'],\n",
       " '290': ['c49', 'c74', 'c80'],\n",
       " '291': ['c5', 'c55', 'c67'],\n",
       " '292': ['c30', 'c47', 'c73'],\n",
       " '293': ['c22', 'c44', 'c64'],\n",
       " '295': ['c6', 'c32', 'c34'],\n",
       " '296': ['c25', 'c36', 'c96'],\n",
       " '297': ['c3', 'c56', 'c94'],\n",
       " '298': ['c9', 'c20', 'c95'],\n",
       " '299': ['c40', 'c65', 'c77'],\n",
       " '300': ['c24', 'c54', 'c60'],\n",
       " '302': ['c28', 'c50', 'c78'],\n",
       " '303': ['c0', 'c72', 'c85'],\n",
       " '304': ['c48', 'c63', 'c92'],\n",
       " '305': ['c18', 'c38', 'c67'],\n",
       " '306': ['c49', 'c68', 'c71'],\n",
       " '307': ['c4', 'c57', 'c97'],\n",
       " '308': ['c15', 'c41', 'c66'],\n",
       " '309': ['c53', 'c89', 'c91'],\n",
       " '310': ['c11', 'c33', 'c45'],\n",
       " '311': ['c14', 'c42', 'c76'],\n",
       " '312': ['c35', 'c70', 'c98'],\n",
       " '313': ['c32', 'c37', 'c82'],\n",
       " '314': ['c34', 'c55', 'c74'],\n",
       " '315': ['c1', 'c29', 'c83'],\n",
       " '316': ['c39', 'c84', 'c88'],\n",
       " '317': ['c41', 'c78', 'c93'],\n",
       " '318': ['c46', 'c92', 'c96'],\n",
       " '319': ['c3', 'c26', 'c90'],\n",
       " '320': ['c16', 'c28', 'c94'],\n",
       " '321': ['c0', 'c12', 'c73'],\n",
       " '322': ['c7', 'c60', 'c65'],\n",
       " '323': ['c9', 'c24', 'c77'],\n",
       " '324': ['c13', 'c21', 'c58'],\n",
       " '325': ['c51', 'c57', 'c67'],\n",
       " '326': ['c23', 'c37', 'c61'],\n",
       " '327': ['c52', 'c59', 'c66'],\n",
       " '328': ['c30', 'c31', 'c62'],\n",
       " '329': ['c1', 'c2', 'c27'],\n",
       " '330': ['c47', 'c50', 'c86'],\n",
       " '331': ['c10', 'c71', 'c91'],\n",
       " '332': ['c17', 'c36', 'c87'],\n",
       " '333': ['c8', 'c25', 'c64'],\n",
       " '334': ['c15', 'c54', 'c69'],\n",
       " '335': ['c75', 'c79', 'c97'],\n",
       " '336': ['c4', 'c33', 'c76'],\n",
       " '337': ['c29', 'c53', 'c92'],\n",
       " '338': ['c70', 'c80', 'c89'],\n",
       " '339': ['c18', 'c23', 'c40'],\n",
       " '340': ['c2', 'c19', 'c22'],\n",
       " '341': ['c11', 'c52', 'c56'],\n",
       " '342': ['c51', 'c81', 'c95'],\n",
       " '343': ['c44', 'c72', 'c88'],\n",
       " '344': ['c37', 'c68', 'c98'],\n",
       " '345': ['c5', 'c35', 'c74'],\n",
       " '346': ['c14', 'c32', 'c45'],\n",
       " '347': ['c34', 'c43', 'c48'],\n",
       " '348': ['c20', 'c42', 'c61'],\n",
       " '349': ['c6', 'c38', 'c63'],\n",
       " '350': ['c13', 'c78', 'c79'],\n",
       " '351': ['c15', 'c76', 'c85'],\n",
       " '352': ['c10', 'c32', 'c36'],\n",
       " '353': ['c41', 'c47', 'c60'],\n",
       " '354': ['c26', 'c51', 'c88'],\n",
       " '355': ['c12', 'c68', 'c96'],\n",
       " '356': ['c4', 'c18', 'c21'],\n",
       " '357': ['c30', 'c49', 'c93'],\n",
       " '358': ['c6', 'c44', 'c59'],\n",
       " '359': ['c37', 'c67', 'c97'],\n",
       " '360': ['c3', 'c28', 'c65'],\n",
       " '361': ['c27', 'c42', 'c69'],\n",
       " '362': ['c9', 'c45', 'c87'],\n",
       " '363': ['c17', 'c57', 'c66'],\n",
       " '364': ['c34', 'c72', 'c82'],\n",
       " '365': ['c16', 'c20', 'c35'],\n",
       " '366': ['c33', 'c50', 'c55'],\n",
       " '367': ['c2', 'c7', 'c86'],\n",
       " '368': ['c43', 'c46', 'c73'],\n",
       " '370': ['c12', 'c19', 'c61'],\n",
       " '371': ['c24', 'c63', 'c91'],\n",
       " '372': ['c13', 'c48', 'c62'],\n",
       " '373': ['c52', 'c77', 'c80'],\n",
       " '374': ['c25', 'c37', 'c84'],\n",
       " '375': ['c4', 'c29', 'c89'],\n",
       " '376': ['c21', 'c23', 'c32'],\n",
       " '377': ['c27', 'c71', 'c94'],\n",
       " '379': ['c6', 'c17', 'c54'],\n",
       " '380': ['c56', 'c92', 'c93'],\n",
       " '381': ['c14', 'c87', 'c95'],\n",
       " '382': ['c15', 'c22', 'c98'],\n",
       " '383': ['c59', 'c83', 'c97'],\n",
       " '384': ['c0', 'c39', 'c40'],\n",
       " '385': ['c5', 'c42', 'c65'],\n",
       " '386': ['c1', 'c49', 'c61'],\n",
       " '387': ['c9', 'c78', 'c90'],\n",
       " '388': ['c8', 'c28', 'c67'],\n",
       " '389': ['c3', 'c38', 'c64'],\n",
       " '390': ['c11', 'c31', 'c69'],\n",
       " '391': ['c44', 'c53', 'c75'],\n",
       " '392': ['c30', 'c33', 'c57'],\n",
       " '393': ['c35', 'c81', 'c82'],\n",
       " '394': ['c29', 'c70', 'c79'],\n",
       " '395': ['c19', 'c45', 'c66'],\n",
       " '396': ['c74', 'c76', 'c89'],\n",
       " '397': ['c47', 'c85', 'c98'],\n",
       " '398': ['c15', 'c18', 'c72'],\n",
       " '399': ['c14', 'c23', 'c58'],\n",
       " '400': ['c39', 'c52', 'c95'],\n",
       " '401': ['c26', 'c34', 'c86'],\n",
       " '402': ['c60', 'c68', 'c73'],\n",
       " '403': ['c50', 'c63', 'c83'],\n",
       " '404': ['c8', 'c51', 'c91'],\n",
       " '405': ['c5', 'c48', 'c94'],\n",
       " '406': ['c54', 'c80', 'c96'],\n",
       " '407': ['c36', 'c43', 'c71'],\n",
       " '408': ['c7', 'c16', 'c88'],\n",
       " '409': ['c0', 'c1', 'c56'],\n",
       " '410': ['c20', 'c62', 'c92'],\n",
       " '411': ['c11', 'c55', 'c64'],\n",
       " '412': ['c2', 'c38', 'c74'],\n",
       " '413': ['c10', 'c53', 'c79'],\n",
       " '414': ['c46', 'c75', 'c98'],\n",
       " '415': ['c18', 'c70', 'c90'],\n",
       " '416': ['c32', 'c77', 'c81'],\n",
       " '417': ['c40', 'c51', 'c84'],\n",
       " '418': ['c12', 'c25', 'c41'],\n",
       " '419': ['c31', 'c47', 'c58'],\n",
       " '421': ['c22', 'c24', 'c29'],\n",
       " '680': ['c53', 'c64', 'c70'],\n",
       " '681': ['c22', 'c57', 'c89'],\n",
       " '682': ['c71', 'c73', 'c74'],\n",
       " '683': ['c1', 'c80', 'c86'],\n",
       " '684': ['c7', 'c43', 'c79'],\n",
       " '685': ['c13', 'c36', 'c41'],\n",
       " '686': ['c8', 'c61', 'c94'],\n",
       " '687': ['c21', 'c42', 'c52'],\n",
       " '688': ['c40', 'c54', 'c62'],\n",
       " '689': ['c3', 'c58', 'c84'],\n",
       " '690': ['c14', 'c16', 'c29'],\n",
       " '691': ['c39', 'c53', 'c78'],\n",
       " '692': ['c23', 'c25', 'c98'],\n",
       " '693': ['c4', 'c35', 'c46'],\n",
       " '694': ['c26', 'c45', 'c75'],\n",
       " '695': ['c28', 'c51', 'c66'],\n",
       " '696': ['c12', 'c67', 'c69'],\n",
       " '697': ['c32', 'c33', 'c96'],\n",
       " '698': ['c9', 'c68', 'c89'],\n",
       " '699': ['c18', 'c49', 'c76'],\n",
       " '701': ['c0', 'c21', 'c48'],\n",
       " '702': ['c11', 'c85', 'c87'],\n",
       " '703': ['c17', 'c20', 'c91'],\n",
       " '704': ['c19', 'c34', 'c54'],\n",
       " '705': ['c56', 'c59', 'c78'],\n",
       " '706': ['c50', 'c88', 'c95'],\n",
       " '707': ['c16', 'c64', 'c84'],\n",
       " '708': ['c15', 'c46', 'c63'],\n",
       " '709': ['c14', 'c25', 'c79'],\n",
       " '710': ['c24', 'c80', 'c92'],\n",
       " '711': ['c38', 'c52', 'c65'],\n",
       " '712': ['c30', 'c41', 'c94'],\n",
       " '713': ['c44', 'c77', 'c98'],\n",
       " '714': ['c10', 'c26', 'c31'],\n",
       " '715': ['c1', 'c5', 'c7'],\n",
       " '716': ['c3', 'c55', 'c68'],\n",
       " '717': ['c28', 'c61', 'c73'],\n",
       " '718': ['c13', 'c23', 'c90'],\n",
       " '719': ['c8', 'c43', 'c74'],\n",
       " '720': ['c27', 'c37', 'c60'],\n",
       " '721': ['c33', 'c83', 'c86'],\n",
       " '722': ['c45', 'c62', 'c76'],\n",
       " '723': ['c4', 'c58', 'c75'],\n",
       " '724': ['c69', 'c70', 'c82'],\n",
       " '725': ['c9', 'c35', 'c66'],\n",
       " '726': ['c47', 'c81', 'c97'],\n",
       " '727': ['c2', 'c39', 'c42'],\n",
       " '728': ['c71', 'c81', 'c93'],\n",
       " '729': ['c6', 'c29', 'c72'],\n",
       " '730': ['c0', 'c11', 'c57'],\n",
       " '731': ['c44', 'c49', 'c92'],\n",
       " '732': ['c37', 'c48', 'c56'],\n",
       " '733': ['c17', 'c22', 'c67'],\n",
       " '734': ['c10', 'c20', 'c90'],\n",
       " '735': ['c30', 'c60', 'c91'],\n",
       " '736': ['c15', 'c24', 'c88'],\n",
       " '737': ['c19', 'c63', 'c95'],\n",
       " '738': ['c31', 'c38', 'c50'],\n",
       " '739': ['c34', 'c65', 'c96'],\n",
       " '740': ['c18', 'c55', 'c83'],\n",
       " '741': ['c12', 'c27', 'c77'],\n",
       " '742': ['c51', 'c87', 'c97'],\n",
       " '743': ['c32', 'c59', 'c85'],\n",
       " '744': ['c36', 'c40', 'c53'],\n",
       " '745': ['c5', 'c82', 'c89'],\n",
       " '422': ['c23', 'c35', 'c64'],\n",
       " '423': ['c53', 'c86', 'c95'],\n",
       " '424': ['c7', 'c34', 'c77'],\n",
       " '425': ['c15', 'c29', 'c78'],\n",
       " '426': ['c13', 'c24', 'c52'],\n",
       " '427': ['c55', 'c89', 'c97'],\n",
       " '428': ['c3', 'c27', 'c59'],\n",
       " '429': ['c18', 'c20', 'c37'],\n",
       " '430': ['c6', 'c36', 'c66'],\n",
       " '431': ['c41', 'c45', 'c82'],\n",
       " '432': ['c4', 'c47', 'c92'],\n",
       " '435': ['c32', 'c65', 'c68'],\n",
       " '436': ['c9', 'c31', 'c80'],\n",
       " '437': ['c1', 'c23', 'c81'],\n",
       " '438': ['c62', 'c94', 'c98'],\n",
       " '439': ['c8', 'c30', 'c83'],\n",
       " '440': ['c21', 'c39', 'c61'],\n",
       " '441': ['c40', 'c59', 'c90'],\n",
       " '442': ['c0', 'c10', 'c49'],\n",
       " '443': ['c71', 'c78', 'c87'],\n",
       " '444': ['c16', 'c46', 'c48'],\n",
       " '446': ['c7', 'c26', 'c67'],\n",
       " '447': ['c31', 'c70', 'c93'],\n",
       " '448': ['c11', 'c43', 'c58'],\n",
       " '449': ['c38', 'c69', 'c96'],\n",
       " '450': ['c3', 'c5', 'c44'],\n",
       " '451': ['c18', 'c53', 'c84'],\n",
       " '452': ['c9', 'c22', 'c28'],\n",
       " '453': ['c14', 'c34', 'c85'],\n",
       " '454': ['c13', 'c25', 'c54'],\n",
       " '455': ['c20', 'c52', 'c63'],\n",
       " '456': ['c2', 'c15', 'c95'],\n",
       " '457': ['c41', 'c55', 'c57'],\n",
       " '458': ['c4', 'c66', 'c73'],\n",
       " '459': ['c36', 'c70', 'c76'],\n",
       " '460': ['c35', 'c45', 'c60'],\n",
       " '461': ['c33', 'c82', 'c93'],\n",
       " '462': ['c29', 'c42', 'c51'],\n",
       " '463': ['c12', 'c74', 'c75'],\n",
       " '464': ['c50', 'c56', 'c77'],\n",
       " '465': ['c19', 'c27', 'c88'],\n",
       " '466': ['c17', 'c24', 'c37'],\n",
       " '467': ['c5', 'c79', 'c91'],\n",
       " '468': ['c6', 'c28', 'c75'],\n",
       " '469': ['c42', 'c72', 'c97'],\n",
       " '470': ['c49', 'c64', 'c89'],\n",
       " '471': ['c10', 'c57', 'c76'],\n",
       " '472': ['c33', 'c62', 'c87'],\n",
       " '473': ['c46', 'c74', 'c81'],\n",
       " '474': ['c2', 'c16', 'c67'],\n",
       " '475': ['c54', 'c90', 'c94'],\n",
       " '476': ['c48', 'c58', 'c83'],\n",
       " '477': ['c71', 'c85', 'c92'],\n",
       " '478': ['c11', 'c30', 'c61'],\n",
       " '479': ['c26', 'c80', 'c84'],\n",
       " '480': ['c21', 'c88', 'c98'],\n",
       " '481': ['c56', 'c63', 'c68'],\n",
       " '482': ['c65', 'c72', 'c73'],\n",
       " '483': ['c8', 'c47', 'c96'],\n",
       " '484': ['c0', 'c43', 'c44'],\n",
       " '485': ['c14', 'c39', 'c69'],\n",
       " '486': ['c25', 'c38', 'c60'],\n",
       " '487': ['c1', 'c32', 'c40'],\n",
       " '489': ['c22', 'c35', 'c86'],\n",
       " '813': ['c25', 'c39', 'c65'],\n",
       " '814': ['c59', 'c64', 'c88'],\n",
       " '815': ['c23', 'c36', 'c63'],\n",
       " '816': ['c26', 'c35', 'c42'],\n",
       " '817': ['c40', 'c73', 'c98'],\n",
       " '819': ['c19', 'c38', 'c80'],\n",
       " '820': ['c7', 'c17', 'c93'],\n",
       " '821': ['c14', 'c33', 'c66'],\n",
       " '822': ['c4', 'c45', 'c90'],\n",
       " '823': ['c50', 'c57', 'c85'],\n",
       " '824': ['c16', 'c43', 'c60'],\n",
       " '825': ['c11', 'c70', 'c86'],\n",
       " '826': ['c27', 'c68', 'c92'],\n",
       " '827': ['c44', 'c55', 'c58'],\n",
       " '828': ['c34', 'c62', 'c81'],\n",
       " '829': ['c3', 'c29', 'c41'],\n",
       " '832': ['c10', 'c18', 'c78'],\n",
       " '833': ['c5', 'c31', 'c40'],\n",
       " '834': ['c30', 'c54', 'c72'],\n",
       " '835': ['c21', 'c22', 'c84'],\n",
       " '836': ['c6', 'c43', 'c82'],\n",
       " '837': ['c24', 'c38', 'c47'],\n",
       " '838': ['c42', 'c56', 'c88'],\n",
       " '839': ['c26', 'c64', 'c94'],\n",
       " '840': ['c69', 'c73', 'c83'],\n",
       " '841': ['c7', 'c37', 'c49'],\n",
       " '843': ['c9', 'c15', 'c51'],\n",
       " '844': ['c2', 'c23', 'c93'],\n",
       " '845': ['c8', 'c33', 'c48'],\n",
       " '846': ['c11', 'c18', 'c95'],\n",
       " '847': ['c1', 'c17', 'c45'],\n",
       " '848': ['c19', 'c82', 'c97'],\n",
       " '849': ['c29', 'c54', 'c77'],\n",
       " '850': ['c66', 'c75', 'c90'],\n",
       " '851': ['c57', 'c70', 'c91'],\n",
       " '852': ['c20', 'c76', 'c80'],\n",
       " '853': ['c36', 'c46', 'c79'],\n",
       " '854': ['c13', 'c63', 'c89'],\n",
       " '855': ['c52', 'c61', 'c96'],\n",
       " '856': ['c25', 'c28', 'c87'],\n",
       " '857': ['c14', 'c53', 'c59'],\n",
       " '858': ['c0', 'c65', 'c98'],\n",
       " '859': ['c16', 'c39', 'c74'],\n",
       " '860': ['c35', 'c67', 'c95'],\n",
       " '861': ['c12', 'c32', 'c71'],\n",
       " '862': ['c4', 'c50', 'c79'],\n",
       " '863': ['c6', 'c76', 'c77'],\n",
       " '864': ['c68', 'c75', 'c91'],\n",
       " '865': ['c44', 'c47', 'c83'],\n",
       " '866': ['c48', 'c61', 'c81'],\n",
       " '867': ['c31', 'c41', 'c72'],\n",
       " '868': ['c2', 'c85', 'c96'],\n",
       " '869': ['c51', 'c60', 'c78'],\n",
       " '870': ['c13', 'c56', 'c86'],\n",
       " '871': ['c15', 'c30', 'c34'],\n",
       " '872': ['c22', 'c58', 'c69'],\n",
       " '873': ['c8', 'c62', 'c84'],\n",
       " '874': ['c1', 'c3', 'c10'],\n",
       " '875': ['c9', 'c21', 'c49'],\n",
       " '876': ['c55', 'c87', 'c94'],\n",
       " '877': ['c5', 'c37', 'c92'],\n",
       " '878': ['c24', 'c27', 'c97'],\n",
       " '748': ['c10', 'c55', 'c59'],\n",
       " '749': ['c40', 'c43', 'c72'],\n",
       " '750': ['c37', 'c54', 'c88'],\n",
       " '751': ['c3', 'c21', 'c78'],\n",
       " '752': ['c14', 'c56', 'c84'],\n",
       " '753': ['c15', 'c32', 'c48'],\n",
       " '754': ['c31', 'c65', 'c97'],\n",
       " '755': ['c34', 'c87', 'c92'],\n",
       " '756': ['c26', 'c44', 'c62'],\n",
       " '757': ['c16', 'c22', 'c23'],\n",
       " '758': ['c5', 'c45', 'c77'],\n",
       " '759': ['c20', 'c50', 'c75'],\n",
       " '760': ['c52', 'c71', 'c79'],\n",
       " '761': ['c0', 'c66', 'c91'],\n",
       " '762': ['c12', 'c33', 'c53'],\n",
       " '763': ['c19', 'c47', 'c57'],\n",
       " '764': ['c42', 'c64', 'c85'],\n",
       " '765': ['c9', 'c10', 'c27'],\n",
       " '766': ['c8', 'c54', 'c98'],\n",
       " '767': ['c61', 'c69', 'c86'],\n",
       " '768': ['c39', 'c73', 'c94'],\n",
       " '769': ['c15', 'c26', 'c97'],\n",
       " '770': ['c5', 'c78', 'c84'],\n",
       " '774': ['c31', 'c49', 'c87'],\n",
       " '775': ['c2', 'c13', 'c29'],\n",
       " '776': ['c16', 'c40', 'c55'],\n",
       " '777': ['c33', 'c67', 'c77'],\n",
       " '778': ['c6', 'c41', 'c70'],\n",
       " '779': ['c3', 'c66', 'c79'],\n",
       " '780': ['c28', 'c74', 'c93'],\n",
       " '782': ['c17', 'c50', 'c82'],\n",
       " '783': ['c19', 'c75', 'c89'],\n",
       " '784': ['c12', 'c46', 'c52'],\n",
       " '785': ['c0', 'c4', 'c20'],\n",
       " '786': ['c1', 'c34', 'c51'],\n",
       " '787': ['c25', 'c35', 'c56'],\n",
       " '788': ['c38', 'c58', 'c88'],\n",
       " '789': ['c36', 'c83', 'c90'],\n",
       " '790': ['c11', 'c48', 'c60'],\n",
       " '791': ['c22', 'c43', 'c81'],\n",
       " '792': ['c21', 'c72', 'c80'],\n",
       " '793': ['c23', 'c24', 'c57'],\n",
       " '794': ['c44', 'c68', 'c76'],\n",
       " '795': ['c53', 'c63', 'c71'],\n",
       " '796': ['c18', 'c45', 'c91'],\n",
       " '797': ['c7', 'c58', 'c96'],\n",
       " '798': ['c30', 'c37', 'c95'],\n",
       " '799': ['c28', 'c32', 'c92'],\n",
       " '800': ['c59', 'c62', 'c89'],\n",
       " '801': ['c14', 'c65', 'c74'],\n",
       " '802': ['c46', 'c67', 'c93'],\n",
       " '803': ['c17', 'c70', 'c85'],\n",
       " '804': ['c47', 'c76', 'c82'],\n",
       " '805': ['c4', 'c6', 'c94'],\n",
       " '806': ['c1', 'c9', 'c73'],\n",
       " '807': ['c39', 'c41', 'c98'],\n",
       " '808': ['c8', 'c60', 'c86'],\n",
       " '809': ['c29', 'c64', 'c69'],\n",
       " '810': ['c36', 'c42', 'c49'],\n",
       " '811': ['c2', 'c61', 'c83'],\n",
       " '812': ['c13', 'c27', 'c35'],\n",
       " '1300': ['c2', 'c35', 'c87'],\n",
       " '1301': ['c7', 'c42', 'c80'],\n",
       " '1302': ['c12', 'c38', 'c51'],\n",
       " '1303': ['c25', 'c33', 'c89'],\n",
       " '1304': ['c17', 'c63', 'c81'],\n",
       " '1305': ['c18', 'c28', 'c57'],\n",
       " '1306': ['c56', 'c62', 'c66'],\n",
       " '1307': ['c23', 'c26', 'c83'],\n",
       " '1308': ['c10', 'c13', 'c98'],\n",
       " '1309': ['c41', 'c49', 'c54'],\n",
       " '1310': ['c29', 'c43', 'c85'],\n",
       " '1311': ['c58', 'c64', 'c77'],\n",
       " '1312': ['c14', 'c55', 'c86'],\n",
       " '1313': ['c22', 'c45', 'c97'],\n",
       " '1314': ['c24', 'c59', 'c70'],\n",
       " '1315': ['c34', 'c80', 'c91'],\n",
       " '1316': ['c46', 'c69', 'c71'],\n",
       " '1317': ['c19', 'c52', 'c67'],\n",
       " '1318': ['c12', 'c18', 'c50'],\n",
       " '1319': ['c75', 'c76', 'c95'],\n",
       " '1320': ['c57', 'c90', 'c96'],\n",
       " '1321': ['c17', 'c68', 'c79'],\n",
       " '1322': ['c4', 'c25', 'c53'],\n",
       " '1323': ['c7', 'c74', 'c82'],\n",
       " '1324': ['c0', 'c38', 'c93'],\n",
       " '1325': ['c10', 'c84', 'c94'],\n",
       " '1330': ['c11', 'c66', 'c81'],\n",
       " '1331': ['c14', 'c15', 'c77'],\n",
       " '1332': ['c21', 'c36', 'c54'],\n",
       " '1333': ['c6', 'c37', 'c40'],\n",
       " '1334': ['c27', 'c46', 'c65'],\n",
       " '1335': ['c5', 'c47', 'c71'],\n",
       " '1336': ['c1', 'c30', 'c39'],\n",
       " '1337': ['c2', 'c73', 'c92'],\n",
       " '1338': ['c35', 'c61', 'c63'],\n",
       " '1339': ['c20', 'c44', 'c87'],\n",
       " '1340': ['c32', 'c42', 'c78'],\n",
       " '1341': ['c3', 'c8', 'c16'],\n",
       " '1342': ['c31', 'c48', 'c88'],\n",
       " '1343': ['c9', 'c60', 'c72'],\n",
       " '1344': ['c28', 'c30', 'c89'],\n",
       " '1345': ['c6', 'c33', 'c51'],\n",
       " '1346': ['c15', 'c79', 'c94'],\n",
       " '1347': ['c68', 'c84', 'c93'],\n",
       " '1348': ['c11', 'c82', 'c90'],\n",
       " '1349': ['c0', 'c67', 'c76'],\n",
       " '1350': ['c24', 'c45', 'c74'],\n",
       " '1351': ['c19', 'c49', 'c70'],\n",
       " '1352': ['c22', 'c75', 'c96'],\n",
       " '1353': ['c59', 'c91', 'c95'],\n",
       " '1354': ['c50', 'c52', 'c97'],\n",
       " '1355': ['c4', 'c13', 'c43'],\n",
       " '1356': ['c20', 'c29', 'c98'],\n",
       " '1357': ['c26', 'c53', 'c69'],\n",
       " '1358': ['c23', 'c41', 'c62'],\n",
       " '1359': ['c16', 'c65', 'c83'],\n",
       " '1360': ['c58', 'c86', 'c92'],\n",
       " '1361': ['c1', 'c36', 'c64'],\n",
       " '1362': ['c37', 'c55', 'c85'],\n",
       " '1363': ['c34', 'c47', 'c56'],\n",
       " '490': ['c60', 'c63', 'c75'],\n",
       " '491': ['c41', 'c77', 'c89'],\n",
       " '492': ['c2', 'c58', 'c97'],\n",
       " '493': ['c4', 'c56', 'c91'],\n",
       " '494': ['c12', 'c57', 'c72'],\n",
       " '495': ['c27', 'c70', 'c73'],\n",
       " '496': ['c9', 'c25', 'c76'],\n",
       " '497': ['c28', 'c44', 'c96'],\n",
       " '498': ['c20', 'c48', 'c74'],\n",
       " '499': ['c7', 'c46', 'c68'],\n",
       " '500': ['c32', 'c51', 'c79'],\n",
       " '501': ['c6', 'c18', 'c19'],\n",
       " '502': ['c3', 'c24', 'c75'],\n",
       " '503': ['c17', 'c60', 'c88'],\n",
       " '504': ['c61', 'c71', 'c72'],\n",
       " '505': ['c30', 'c40', 'c81'],\n",
       " '506': ['c52', 'c90', 'c93'],\n",
       " '507': ['c8', 'c11', 'c50'],\n",
       " '508': ['c15', 'c38', 'c62'],\n",
       " '509': ['c39', 'c67', 'c70'],\n",
       " '510': ['c31', 'c35', 'c85'],\n",
       " '511': ['c42', 'c83', 'c92'],\n",
       " '512': ['c1', 'c37', 'c43'],\n",
       " '516': ['c33', 'c54', 'c78'],\n",
       " '517': ['c23', 'c45', 'c53'],\n",
       " '518': ['c5', 'c61', 'c87'],\n",
       " '519': ['c21', 'c64', 'c82'],\n",
       " '520': ['c0', 'c36', 'c89'],\n",
       " '521': ['c13', 'c66', 'c80'],\n",
       " '522': ['c14', 'c22', 'c49'],\n",
       " '523': ['c65', 'c69', 'c94'],\n",
       " '524': ['c10', 'c29', 'c34'],\n",
       " '525': ['c26', 'c47', 'c95'],\n",
       " '526': ['c59', 'c84', 'c86'],\n",
       " '527': ['c31', 'c63', 'c67'],\n",
       " '528': ['c38', 'c39', 'c71'],\n",
       " '529': ['c16', 'c24', 'c98'],\n",
       " '530': ['c55', 'c88', 'c96'],\n",
       " '531': ['c8', 'c17', 'c78'],\n",
       " '532': ['c27', 'c76', 'c91'],\n",
       " '533': ['c11', 'c68', 'c74'],\n",
       " '534': ['c0', 'c5', 'c32'],\n",
       " '535': ['c30', 'c50', 'c51'],\n",
       " '536': ['c9', 'c44', 'c46'],\n",
       " '537': ['c33', 'c79', 'c90'],\n",
       " '538': ['c7', 'c53', 'c66'],\n",
       " '539': ['c21', 'c25', 'c45'],\n",
       " '540': ['c18', 'c80', 'c93'],\n",
       " '541': ['c3', 'c48', 'c52'],\n",
       " '542': ['c2', 'c20', 'c40'],\n",
       " '543': ['c10', 'c28', 'c82'],\n",
       " '544': ['c19', 'c73', 'c77'],\n",
       " '545': ['c4', 'c12', 'c37'],\n",
       " '546': ['c6', 'c56', 'c98'],\n",
       " '548': ['c42', 'c55', 'c81'],\n",
       " '549': ['c49', 'c57', 'c62'],\n",
       " '550': ['c43', 'c83', 'c95'],\n",
       " '551': ['c35', 'c69', 'c97'],\n",
       " '552': ['c16', 'c41', 'c58'],\n",
       " '1461': ['c24', 'c51', 'c85'],\n",
       " '1462': ['c4', 'c23', 'c28'],\n",
       " '1463': ['c25', 'c29', 'c82'],\n",
       " '1464': ['c36', 'c68', 'c95'],\n",
       " '1465': ['c1', 'c54', 'c92'],\n",
       " '1466': ['c9', 'c91', 'c96'],\n",
       " '1467': ['c11', 'c47', 'c59'],\n",
       " '1468': ['c7', 'c18', 'c87'],\n",
       " '1469': ['c15', 'c52', 'c64'],\n",
       " '1470': ['c32', 'c63', 'c73'],\n",
       " '1471': ['c8', 'c38', 'c89'],\n",
       " '1472': ['c26', 'c48', 'c72'],\n",
       " '1473': ['c12', 'c20', 'c23'],\n",
       " '1474': ['c27', 'c53', 'c93'],\n",
       " '1475': ['c31', 'c66', 'c94'],\n",
       " '1476': ['c13', 'c44', 'c74'],\n",
       " '1477': ['c34', 'c45', 'c84'],\n",
       " '1478': ['c6', 'c39', 'c46'],\n",
       " '1479': ['c57', 'c75', 'c88'],\n",
       " '1480': ['c60', 'c76', 'c90'],\n",
       " '1481': ['c25', 'c80', 'c83'],\n",
       " '1482': ['c43', 'c56', 'c61'],\n",
       " '1485': ['c14', 'c40', 'c41'],\n",
       " '1486': ['c0', 'c58', 'c62'],\n",
       " '1487': ['c7', 'c21', 'c38'],\n",
       " '1488': ['c16', 'c19', 'c33'],\n",
       " '1489': ['c4', 'c42', 'c71'],\n",
       " '1490': ['c44', 'c52', 'c86'],\n",
       " '1491': ['c2', 'c70', 'c81'],\n",
       " '1492': ['c22', 'c50', 'c65'],\n",
       " '1493': ['c5', 'c17', 'c51'],\n",
       " '1494': ['c10', 'c68', 'c97'],\n",
       " '1495': ['c55', 'c90', 'c98'],\n",
       " '1496': ['c20', 'c49', 'c79'],\n",
       " '1497': ['c3', 'c37', 'c77'],\n",
       " '1498': ['c30', 'c67', 'c78'],\n",
       " '1499': ['c33', 'c36', 'c69'],\n",
       " '1500': ['c14', 'c35', 'c47'],\n",
       " '1501': ['c24', 'c61', 'c82'],\n",
       " '1502': ['c21', 'c40', 'c86'],\n",
       " '1503': ['c28', 'c80', 'c85'],\n",
       " '1504': ['c32', 'c84', 'c89'],\n",
       " '1505': ['c0', 'c19', 'c92'],\n",
       " '1506': ['c12', 'c22', 'c88'],\n",
       " '1507': ['c15', 'c31', 'c59'],\n",
       " '1508': ['c34', 'c73', 'c78'],\n",
       " '1509': ['c5', 'c95', 'c96'],\n",
       " '1510': ['c18', 'c48', 'c71'],\n",
       " '1511': ['c8', 'c70', 'c75'],\n",
       " '1512': ['c26', 'c29', 'c74'],\n",
       " '1513': ['c27', 'c64', 'c66'],\n",
       " '1514': ['c3', 'c45', 'c54'],\n",
       " '1515': ['c53', 'c60', 'c94'],\n",
       " '1516': ['c46', 'c57', 'c87'],\n",
       " '1517': ['c9', 'c39', 'c63'],\n",
       " '1518': ['c11', 'c13', 'c72'],\n",
       " '1519': ['c1', 'c6', 'c58'],\n",
       " '1520': ['c43', 'c76', 'c93'],\n",
       " '1521': ['c2', 'c65', 'c91'],\n",
       " '949': ['c14', 'c70', 'c78'],\n",
       " '950': ['c15', 'c73', 'c84'],\n",
       " '951': ['c27', 'c57', 'c83'],\n",
       " '952': ['c23', 'c30', 'c80'],\n",
       " '953': ['c21', 'c81', 'c92'],\n",
       " '954': ['c0', 'c26', 'c82'],\n",
       " '955': ['c13', 'c19', 'c69'],\n",
       " '956': ['c46', 'c54', 'c85'],\n",
       " '957': ['c33', 'c38', 'c49'],\n",
       " '958': ['c5', 'c53', 'c62'],\n",
       " '959': ['c1', 'c25', 'c77'],\n",
       " '960': ['c45', 'c61', 'c68'],\n",
       " '961': ['c21', 'c59', 'c79'],\n",
       " '962': ['c50', 'c58', 'c76'],\n",
       " '963': ['c42', 'c43', 'c67'],\n",
       " '964': ['c40', 'c74', 'c97'],\n",
       " '965': ['c4', 'c24', 'c65'],\n",
       " '966': ['c32', 'c69', 'c87'],\n",
       " '967': ['c2', 'c28', 'c41'],\n",
       " '968': ['c29', 'c33', 'c52'],\n",
       " '969': ['c27', 'c89', 'c90'],\n",
       " '970': ['c66', 'c84', 'c92'],\n",
       " '971': ['c14', 'c60', 'c96'],\n",
       " '972': ['c3', 'c35', 'c49'],\n",
       " '973': ['c44', 'c63', 'c94'],\n",
       " '974': ['c64', 'c78', 'c95'],\n",
       " '975': ['c6', 'c48', 'c93'],\n",
       " '976': ['c47', 'c53', 'c54'],\n",
       " '977': ['c61', 'c77', 'c88'],\n",
       " '978': ['c16', 'c85', 'c97'],\n",
       " '979': ['c12', 'c17', 'c30'],\n",
       " '980': ['c29', 'c36', 'c75'],\n",
       " '981': ['c19', 'c28', 'c37'],\n",
       " '982': ['c60', 'c82', 'c98'],\n",
       " '983': ['c9', 'c11', 'c32'],\n",
       " '984': ['c8', 'c10', 'c15'],\n",
       " '985': ['c39', 'c86', 'c91'],\n",
       " '986': ['c22', 'c34', 'c76'],\n",
       " '987': ['c56', 'c70', 'c72'],\n",
       " '988': ['c7', 'c23', 'c51'],\n",
       " '989': ['c16', 'c18', 'c31'],\n",
       " '990': ['c20', 'c55', 'c71'],\n",
       " '991': ['c3', 'c67', 'c81'],\n",
       " '992': ['c50', 'c80', 'c98'],\n",
       " '993': ['c35', 'c40', 'c83'],\n",
       " '994': ['c37', 'c65', 'c87'],\n",
       " '995': ['c47', 'c79', 'c88'],\n",
       " '996': ['c5', 'c52', 'c57'],\n",
       " '997': ['c17', 'c62', 'c73'],\n",
       " '999': ['c24', 'c68', 'c94'],\n",
       " '1000': ['c0', 'c42', 'c90'],\n",
       " '1001': ['c38', 'c46', 'c95'],\n",
       " '1002': ['c1', 'c41', 'c44'],\n",
       " '1003': ['c6', 'c25', 'c86'],\n",
       " '1004': ['c13', 'c45', 'c59'],\n",
       " '1005': ['c26', 'c55', 'c93'],\n",
       " '554': ['c30', 'c56', 'c64'],\n",
       " '555': ['c23', 'c43', 'c78'],\n",
       " '556': ['c0', 'c35', 'c94'],\n",
       " '557': ['c57', 'c61', 'c92'],\n",
       " '558': ['c21', 'c50', 'c96'],\n",
       " '559': ['c10', 'c22', 'c47'],\n",
       " '560': ['c25', 'c66', 'c72'],\n",
       " '561': ['c18', 'c34', 'c89'],\n",
       " '562': ['c67', 'c74', 'c91'],\n",
       " '563': ['c4', 'c9', 'c48'],\n",
       " '564': ['c39', 'c56', 'c75'],\n",
       " '565': ['c33', 'c84', 'c85'],\n",
       " '566': ['c20', 'c65', 'c86'],\n",
       " '567': ['c28', 'c58', 'c71'],\n",
       " '568': ['c26', 'c37', 'c79'],\n",
       " '569': ['c16', 'c54', 'c68'],\n",
       " '570': ['c45', 'c51', 'c92'],\n",
       " '571': ['c2', 'c60', 'c77'],\n",
       " '572': ['c53', 'c87', 'c88'],\n",
       " '573': ['c24', 'c32', 'c55'],\n",
       " '574': ['c29', 'c46', 'c80'],\n",
       " '575': ['c10', 'c12', 'c66'],\n",
       " '576': ['c8', 'c21', 'c34'],\n",
       " '577': ['c1', 'c15', 'c90'],\n",
       " '578': ['c72', 'c81', 'c89'],\n",
       " '579': ['c6', 'c14', 'c62'],\n",
       " '580': ['c17', 'c31', 'c76'],\n",
       " '581': ['c30', 'c36', 'c38'],\n",
       " '582': ['c18', 'c63', 'c77'],\n",
       " '583': ['c7', 'c40', 'c71'],\n",
       " '584': ['c11', 'c44', 'c51'],\n",
       " '585': ['c4', 'c19', 'c41'],\n",
       " '586': ['c9', 'c83', 'c93'],\n",
       " '587': ['c52', 'c82', 'c85'],\n",
       " '588': ['c5', 'c59', 'c69'],\n",
       " '589': ['c27', 'c78', 'c98'],\n",
       " '590': ['c8', 'c13', 'c49'],\n",
       " '591': ['c3', 'c42', 'c73'],\n",
       " '592': ['c70', 'c95', 'c97'],\n",
       " '593': ['c12', 'c63', 'c79'],\n",
       " '594': ['c20', 'c64', 'c81'],\n",
       " '595': ['c36', 'c45', 'c50'],\n",
       " '596': ['c11', 'c22', 'c39'],\n",
       " '597': ['c7', 'c24', 'c48'],\n",
       " '598': ['c17', 'c74', 'c94'],\n",
       " '599': ['c62', 'c67', 'c75'],\n",
       " '600': ['c31', 'c46', 'c91'],\n",
       " '601': ['c15', 'c35', 'c96'],\n",
       " '602': ['c61', 'c65', 'c80'],\n",
       " '603': ['c6', 'c16', 'c47'],\n",
       " '605': ['c26', 'c57', 'c59'],\n",
       " '606': ['c0', 'c23', 'c87'],\n",
       " '607': ['c43', 'c53', 'c90'],\n",
       " '608': ['c55', 'c60', 'c84'],\n",
       " '609': ['c25', 'c58', 'c68'],\n",
       " '611': ['c3', 'c83', 'c98'],\n",
       " '612': ['c12', 'c39', 'c82'],\n",
       " '613': ['c18', 'c64', 'c86'],\n",
       " '614': ['c36', 'c51', 'c52'],\n",
       " '615': ['c19', 'c42', 'c79'],\n",
       " '616': ['c9', 'c38', 'c56'],\n",
       " '617': ['c17', 'c44', 'c89'],\n",
       " '618': ['c20', 'c22', 'c93'],\n",
       " '619': ['c5', 'c50', 'c81'],\n",
       " '620': ['c14', 'c30', 'c75'],\n",
       " '621': ['c33', 'c34', 'c94'],\n",
       " '622': ['c29', 'c47', 'c87'],\n",
       " '623': ['c2', 'c62', 'c63'],\n",
       " '624': ['c21', 'c28', 'c55'],\n",
       " '625': ['c7', 'c27', 'c61'],\n",
       " '626': ['c13', 'c31', 'c77'],\n",
       " '627': ['c25', 'c69', 'c91'],\n",
       " '628': ['c8', 'c37', 'c72'],\n",
       " '629': ['c3', 'c53', 'c97'],\n",
       " '645': ['c32', 'c41', 'c95'],\n",
       " '646': ['c4', 'c10', 'c11'],\n",
       " '647': ['c1', 'c71', 'c76'],\n",
       " '648': ['c38', 'c43', 'c45'],\n",
       " '649': ['c14', 'c44', 'c67'],\n",
       " '650': ['c5', 'c85', 'c93'],\n",
       " '651': ['c46', 'c49', 'c84'],\n",
       " '652': ['c23', 'c68', 'c88'],\n",
       " '653': ['c52', 'c78', 'c92'],\n",
       " '654': ['c0', 'c60', 'c74'],\n",
       " '655': ['c29', 'c35', 'c48'],\n",
       " '656': ['c54', 'c66', 'c70'],\n",
       " '657': ['c40', 'c49', 'c96'],\n",
       " '658': ['c15', 'c57', 'c65'],\n",
       " '659': ['c58', 'c59', 'c80'],\n",
       " '660': ['c16', 'c26', 'c73'],\n",
       " '662': ['c6', 'c74', 'c90'],\n",
       " '663': ['c2', 'c24', 'c50'],\n",
       " '664': ['c33', 'c63', 'c97'],\n",
       " '665': ['c34', 'c67', 'c88'],\n",
       " '666': ['c10', 'c30', 'c85'],\n",
       " '667': ['c9', 'c53', 'c81'],\n",
       " '668': ['c8', 'c32', 'c39'],\n",
       " '669': ['c20', 'c51', 'c54'],\n",
       " '670': ['c4', 'c82', 'c83'],\n",
       " '671': ['c36', 'c37', 'c86'],\n",
       " '672': ['c69', 'c79', 'c89'],\n",
       " '673': ['c7', 'c12', 'c56'],\n",
       " '674': ['c28', 'c64', 'c91'],\n",
       " '675': ['c1', 'c18', 'c75'],\n",
       " '676': ['c13', 'c22', 'c42'],\n",
       " '677': ['c31', 'c95', 'c98'],\n",
       " '678': ['c19', 'c76', 'c87'],\n",
       " '679': ['c17', 'c21', 'c27'],\n",
       " '1242': ['c31', 'c32', 'c90'],\n",
       " '1243': ['c11', 'c25', 'c42'],\n",
       " '1244': ['c0', 'c28', 'c70'],\n",
       " '1245': ['c38', 'c48', 'c73'],\n",
       " '1246': ['c36', 'c39', 'c80'],\n",
       " '1247': ['c8', 'c40', 'c57'],\n",
       " '1248': ['c35', 'c50', 'c62'],\n",
       " '1249': ['c12', 'c64', 'c83'],\n",
       " '1250': ['c4', 'c7', 'c55'],\n",
       " '1251': ['c6', 'c84', 'c97'],\n",
       " '1252': ['c5', 'c14', 'c46'],\n",
       " '1253': ['c11', 'c37', 'c41'],\n",
       " ...}"
      ]
     },
     "execution_count": 129,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_2_readout_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a4dd2783",
   "metadata": {},
   "source": [
    "<a id='3.2'></a>\n",
    "## 3.2 loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8e899029",
   "metadata": {},
   "source": [
    "### primers\n",
    "pick primer set 2,9"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "id": "a69492f8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1A03_primer_2\n",
      "Name: W1A03_primer_2\n",
      "Description: W1A03_primer_2\n",
      "Number of features: 0\n",
      "Seq('CCCGCAATGGCTGACAACCG')\n",
      "- Picked primer: ID: W1A10_primer_9\n",
      "Name: W1A10_primer_9\n",
      "Description: W1A10_primer_9\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGATTGCCGCATGGTTTCCG')\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Primers'\n",
    "primers = library_tools.assemble.load_primers([2,9], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3dfa7962",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "id": "152a93c4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# collect types from region_2_readout_dict\n",
    "readout_usage_by_type = {}\n",
    "for _region, _infos in region_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in readout_usage_by_type:\n",
    "            readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in readout_usage_by_type[_info[0]]:\n",
    "            readout_usage_by_type[_info[0]].append(int(_info[1:]))\n",
    "\n",
    "readout_usage_dict = {_type:len(_v) for _type,_v in readout_usage_by_type.items()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "f3648545",
   "metadata": {},
   "outputs": [],
   "source": [
    "readout_dict ={}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "id": "54fa06dc",
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3.classes import _allowed_kwds\n",
    "\n",
    "readout_dict ={}\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts'\n",
    "\n",
    "# combo\n",
    "readout_type = 'combo'\n",
    "# \n",
    "_type_char = _allowed_kwds[readout_type]\n",
    "_readout_start = 0\n",
    "_num_color = 2\n",
    "\n",
    "readout_dict[_type_char] = library_tools.assemble.load_readouts(readout_usage_dict[_type_char], \n",
    "                                                       _type='snr_after_630', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=_num_color, # 750,647,561\n",
    "                                                       _start_id=_readout_start,\n",
    "                                                       )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 133,
   "id": "dd5038ee",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['NDB_784',\n",
       " 'NDB_755',\n",
       " 'NDB_826',\n",
       " 'NDB_713',\n",
       " 'NDB_865',\n",
       " 'NDB_725',\n",
       " 'NDB_817',\n",
       " 'NDB_710',\n",
       " 'NDB_652',\n",
       " 'NDB_971',\n",
       " 'NDB_718',\n",
       " 'NDB_743',\n",
       " 'NDB_847',\n",
       " 'NDB_683',\n",
       " 'NDB_643',\n",
       " 'NDB_815',\n",
       " 'NDB_760',\n",
       " 'NDB_965',\n",
       " 'NDB_790',\n",
       " 'NDB_917',\n",
       " 'NDB_778',\n",
       " 'NDB_728',\n",
       " 'NDB_844',\n",
       " 'NDB_974',\n",
       " 'NDB_961',\n",
       " 'NDB_746',\n",
       " 'NDB_661',\n",
       " 'NDB_875',\n",
       " 'NDB_901',\n",
       " 'NDB_773',\n",
       " 'NDB_868',\n",
       " 'NDB_812',\n",
       " 'NDB_1027',\n",
       " 'NDB_716',\n",
       " 'NDB_754',\n",
       " 'NDB_860',\n",
       " 'NDB_856',\n",
       " 'NDB_680',\n",
       " 'NDB_634',\n",
       " 'NDB_737',\n",
       " 'NDB_715',\n",
       " 'NDB_698',\n",
       " 'NDB_883',\n",
       " 'NDB_863',\n",
       " 'NDB_1033',\n",
       " 'NDB_878',\n",
       " 'NDB_631',\n",
       " 'NDB_758',\n",
       " 'NDB_1075',\n",
       " 'NDB_653',\n",
       " 'NDB_1060',\n",
       " 'NDB_905',\n",
       " 'NDB_805',\n",
       " 'NDB_866',\n",
       " 'NDB_730',\n",
       " 'NDB_887',\n",
       " 'NDB_880',\n",
       " 'NDB_752',\n",
       " 'NDB_832',\n",
       " 'NDB_896',\n",
       " 'NDB_835',\n",
       " 'NDB_989',\n",
       " 'NDB_946',\n",
       " 'NDB_641',\n",
       " 'NDB_721',\n",
       " 'NDB_899',\n",
       " 'NDB_853',\n",
       " 'NDB_941',\n",
       " 'NDB_838',\n",
       " 'NDB_647',\n",
       " 'NDB_994',\n",
       " 'NDB_881',\n",
       " 'NDB_1066',\n",
       " 'NDB_665',\n",
       " 'NDB_637',\n",
       " 'NDB_734',\n",
       " 'NDB_706',\n",
       " 'NDB_935',\n",
       " 'NDB_889',\n",
       " 'NDB_788',\n",
       " 'NDB_862',\n",
       " 'NDB_779',\n",
       " 'NDB_694',\n",
       " 'NDB_977',\n",
       " 'NDB_751',\n",
       " 'NDB_836',\n",
       " 'NDB_742',\n",
       " 'NDB_929',\n",
       " 'NDB_958',\n",
       " 'NDB_1028',\n",
       " 'NDB_925',\n",
       " 'NDB_824',\n",
       " 'NDB_712',\n",
       " 'NDB_818',\n",
       " 'NDB_940',\n",
       " 'NDB_872',\n",
       " 'NDB_934',\n",
       " 'NDB_926',\n",
       " 'NDB_910']"
      ]
     },
     "execution_count": 133,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[_r.id for _r in readout_dict[_type_char]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "id": "3f1ffff7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save\n"
     ]
    }
   ],
   "source": [
    "# save\n",
    "readout_usage_file = os.path.join(library_folder, 'readout_usage.pkl')\n",
    "if not os.path.isfile(readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save')\n",
    "    pickle.dump(readout_dict, open(readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3d1eeb3b",
   "metadata": {},
   "source": [
    "<a id='3.3'></a>\n",
    "## 3.3 assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "id": "c37342ca",
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "# params\n",
    "assemble_params = {\n",
    "    'primer_len':20,\n",
    "    'num_readout_per_probe':4, \n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "id": "ec61f5f6",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['c']\n",
      "--- assemblying 344 probes in region: 0\n",
      "--- assemblying 319 probes in region: 1\n",
      "--- assemblying 216 probes in region: 10\n",
      "--- assemblying 344 probes in region: 100\n",
      "--- assemblying 326 probes in region: 1000\n",
      "--- assemblying 341 probes in region: 1001\n",
      "--- assemblying 381 probes in region: 1002\n",
      "--- assemblying 356 probes in region: 1003\n",
      "--- assemblying 283 probes in region: 1004\n",
      "--- assemblying 321 probes in region: 1005\n",
      "--- assemblying 286 probes in region: 101\n",
      "--- assemblying 240 probes in region: 102\n",
      "--- assemblying 52 probes in region: 1020\n",
      "--- assemblying 217 probes in region: 1023\n",
      "--- assemblying 242 probes in region: 1024\n",
      "--- assemblying 306 probes in region: 1025\n",
      "--- assemblying 235 probes in region: 1029\n",
      "--- assemblying 205 probes in region: 103\n",
      "--- assemblying 209 probes in region: 1030\n",
      "--- assemblying 290 probes in region: 1031\n",
      "--- assemblying 275 probes in region: 1032\n",
      "--- assemblying 234 probes in region: 1033\n",
      "--- assemblying 183 probes in region: 1034\n",
      "--- assemblying 201 probes in region: 1035\n",
      "--- assemblying 130 probes in region: 1036\n",
      "--- assemblying 237 probes in region: 1037\n",
      "--- assemblying 227 probes in region: 1038\n",
      "--- assemblying 246 probes in region: 1039\n",
      "--- assemblying 119 probes in region: 104\n",
      "--- assemblying 350 probes in region: 1040\n",
      "--- assemblying 254 probes in region: 1041\n",
      "--- assemblying 325 probes in region: 1042\n",
      "--- assemblying 122 probes in region: 1043\n",
      "--- assemblying 253 probes in region: 1044\n",
      "--- assemblying 347 probes in region: 1045\n",
      "--- assemblying 263 probes in region: 1046\n",
      "--- assemblying 350 probes in region: 1047\n",
      "--- assemblying 296 probes in region: 1048\n",
      "--- assemblying 176 probes in region: 1049\n",
      "--- assemblying 352 probes in region: 105\n",
      "--- assemblying 267 probes in region: 1050\n",
      "--- assemblying 384 probes in region: 1051\n",
      "--- assemblying 135 probes in region: 1052\n",
      "--- assemblying 351 probes in region: 1053\n",
      "--- assemblying 320 probes in region: 1054\n",
      "--- assemblying 268 probes in region: 1055\n",
      "--- assemblying 361 probes in region: 1056\n",
      "--- assemblying 155 probes in region: 1057\n",
      "--- assemblying 131 probes in region: 1058\n",
      "--- assemblying 164 probes in region: 1059\n",
      "--- assemblying 244 probes in region: 106\n",
      "--- assemblying 315 probes in region: 1060\n",
      "--- assemblying 309 probes in region: 1061\n",
      "--- assemblying 240 probes in region: 1062\n",
      "--- assemblying 266 probes in region: 1063\n",
      "--- assemblying 327 probes in region: 1064\n",
      "--- assemblying 318 probes in region: 1065\n",
      "--- assemblying 207 probes in region: 1066\n",
      "--- assemblying 204 probes in region: 1067\n",
      "--- assemblying 147 probes in region: 1068\n",
      "--- assemblying 160 probes in region: 1069\n",
      "--- assemblying 305 probes in region: 107\n",
      "--- assemblying 250 probes in region: 1070\n",
      "--- assemblying 291 probes in region: 1071\n",
      "--- assemblying 259 probes in region: 1072\n",
      "--- assemblying 263 probes in region: 1073\n",
      "--- assemblying 273 probes in region: 1074\n",
      "--- assemblying 256 probes in region: 1075\n",
      "--- assemblying 293 probes in region: 1076\n",
      "--- assemblying 88 probes in region: 1077\n",
      "--- assemblying 241 probes in region: 1078\n",
      "--- assemblying 302 probes in region: 1079\n",
      "--- assemblying 213 probes in region: 108\n",
      "--- assemblying 362 probes in region: 1080\n",
      "--- assemblying 298 probes in region: 1081\n",
      "--- assemblying 263 probes in region: 1082\n",
      "--- assemblying 325 probes in region: 1083\n",
      "--- assemblying 341 probes in region: 1085\n",
      "--- assemblying 336 probes in region: 1087\n",
      "--- assemblying 207 probes in region: 1089\n",
      "--- assemblying 225 probes in region: 109\n",
      "--- assemblying 261 probes in region: 1090\n",
      "--- assemblying 226 probes in region: 1091\n",
      "--- assemblying 355 probes in region: 1092\n",
      "--- assemblying 203 probes in region: 1093\n",
      "--- assemblying 317 probes in region: 1094\n",
      "--- assemblying 230 probes in region: 1097\n",
      "--- assemblying 212 probes in region: 11\n",
      "--- assemblying 230 probes in region: 110\n",
      "--- assemblying 223 probes in region: 1105\n",
      "--- assemblying 189 probes in region: 1106\n",
      "--- assemblying 267 probes in region: 1107\n",
      "--- assemblying 295 probes in region: 1108\n",
      "--- assemblying 184 probes in region: 1109\n",
      "--- assemblying 358 probes in region: 111\n",
      "--- assemblying 336 probes in region: 1110\n",
      "--- assemblying 381 probes in region: 1111\n",
      "--- assemblying 398 probes in region: 1112\n",
      "--- assemblying 266 probes in region: 1113\n",
      "--- assemblying 217 probes in region: 1114\n",
      "--- assemblying 253 probes in region: 1115\n",
      "--- assemblying 184 probes in region: 1116\n",
      "--- assemblying 288 probes in region: 1117\n",
      "--- assemblying 231 probes in region: 1118\n",
      "--- assemblying 328 probes in region: 1119\n",
      "--- assemblying 167 probes in region: 112\n",
      "--- assemblying 228 probes in region: 1120\n",
      "--- assemblying 298 probes in region: 1121\n",
      "--- assemblying 227 probes in region: 1122\n",
      "--- assemblying 267 probes in region: 1123\n",
      "--- assemblying 201 probes in region: 1124\n",
      "--- assemblying 208 probes in region: 1125\n",
      "--- assemblying 320 probes in region: 1126\n",
      "--- assemblying 241 probes in region: 1127\n",
      "--- assemblying 235 probes in region: 1128\n",
      "--- assemblying 335 probes in region: 1129\n",
      "--- assemblying 313 probes in region: 1130\n",
      "--- assemblying 272 probes in region: 1131\n",
      "--- assemblying 290 probes in region: 1132\n",
      "--- assemblying 205 probes in region: 1133\n",
      "--- assemblying 331 probes in region: 1134\n",
      "--- assemblying 268 probes in region: 1135\n",
      "--- assemblying 294 probes in region: 1136\n",
      "--- assemblying 196 probes in region: 1137\n",
      "--- assemblying 222 probes in region: 1138\n",
      "--- assemblying 256 probes in region: 1139\n",
      "--- assemblying 215 probes in region: 114\n",
      "--- assemblying 242 probes in region: 1140\n",
      "--- assemblying 375 probes in region: 1141\n",
      "--- assemblying 263 probes in region: 1142\n",
      "--- assemblying 206 probes in region: 1143\n",
      "--- assemblying 122 probes in region: 1144\n",
      "--- assemblying 215 probes in region: 1145\n",
      "--- assemblying 94 probes in region: 1146\n",
      "--- assemblying 257 probes in region: 1147\n",
      "--- assemblying 101 probes in region: 1148\n",
      "--- assemblying 224 probes in region: 115\n",
      "--- assemblying 194 probes in region: 1152\n",
      "--- assemblying 269 probes in region: 1153\n",
      "--- assemblying 54 probes in region: 1154\n",
      "--- assemblying 254 probes in region: 1155\n",
      "--- assemblying 327 probes in region: 1156\n",
      "--- assemblying 276 probes in region: 1157\n",
      "--- assemblying 228 probes in region: 116\n",
      "--- assemblying 317 probes in region: 1160\n",
      "--- assemblying 222 probes in region: 1162\n",
      "--- assemblying 272 probes in region: 1163\n",
      "--- assemblying 270 probes in region: 1165\n",
      "--- assemblying 342 probes in region: 1166\n",
      "--- assemblying 192 probes in region: 1167\n",
      "--- assemblying 346 probes in region: 1168\n",
      "--- assemblying 185 probes in region: 1169\n",
      "--- assemblying 276 probes in region: 117\n",
      "--- assemblying 232 probes in region: 1170\n",
      "--- assemblying 268 probes in region: 1171\n",
      "--- assemblying 329 probes in region: 1172\n",
      "--- assemblying 96 probes in region: 1173\n",
      "--- assemblying 128 probes in region: 1174\n",
      "--- assemblying 212 probes in region: 1176\n",
      "--- assemblying 241 probes in region: 1177\n",
      "--- assemblying 321 probes in region: 1178\n",
      "--- assemblying 197 probes in region: 1179\n",
      "--- assemblying 216 probes in region: 118\n",
      "--- assemblying 213 probes in region: 1180\n",
      "--- assemblying 271 probes in region: 1181\n",
      "--- assemblying 291 probes in region: 1182\n",
      "--- assemblying 337 probes in region: 1183\n",
      "--- assemblying 310 probes in region: 1184\n",
      "--- assemblying 265 probes in region: 1186\n",
      "--- assemblying 285 probes in region: 1187\n",
      "--- assemblying 77 probes in region: 1188\n",
      "--- assemblying 171 probes in region: 1189\n",
      "--- assemblying 221 probes in region: 119\n",
      "--- assemblying 187 probes in region: 1190\n",
      "--- assemblying 194 probes in region: 1191\n",
      "--- assemblying 242 probes in region: 1192\n",
      "--- assemblying 279 probes in region: 1193\n",
      "--- assemblying 291 probes in region: 1194\n",
      "--- assemblying 288 probes in region: 1195\n",
      "--- assemblying 277 probes in region: 1196\n",
      "--- assemblying 184 probes in region: 1197\n",
      "--- assemblying 240 probes in region: 12\n",
      "--- assemblying 200 probes in region: 120\n",
      "--- assemblying 191 probes in region: 1203\n",
      "--- assemblying 346 probes in region: 1204\n",
      "--- assemblying 321 probes in region: 1205\n",
      "--- assemblying 108 probes in region: 1206\n",
      "--- assemblying 213 probes in region: 1207\n",
      "--- assemblying 208 probes in region: 1208\n",
      "--- assemblying 169 probes in region: 1209\n",
      "--- assemblying 295 probes in region: 121\n",
      "--- assemblying 270 probes in region: 1210\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 302 probes in region: 1211\n",
      "--- assemblying 278 probes in region: 1212\n",
      "--- assemblying 293 probes in region: 1213\n",
      "--- assemblying 317 probes in region: 1214\n",
      "--- assemblying 221 probes in region: 1215\n",
      "--- assemblying 216 probes in region: 1216\n",
      "--- assemblying 190 probes in region: 1217\n",
      "--- assemblying 235 probes in region: 1218\n",
      "--- assemblying 315 probes in region: 1219\n",
      "--- assemblying 270 probes in region: 122\n",
      "--- assemblying 358 probes in region: 1220\n",
      "--- assemblying 400 probes in region: 1221\n",
      "--- assemblying 179 probes in region: 1222\n",
      "--- assemblying 231 probes in region: 1224\n",
      "--- assemblying 282 probes in region: 1225\n",
      "--- assemblying 238 probes in region: 1226\n",
      "--- assemblying 215 probes in region: 1227\n",
      "--- assemblying 167 probes in region: 1228\n",
      "--- assemblying 235 probes in region: 1229\n",
      "--- assemblying 182 probes in region: 123\n",
      "--- assemblying 223 probes in region: 1230\n",
      "--- assemblying 241 probes in region: 1231\n",
      "--- assemblying 237 probes in region: 1232\n",
      "--- assemblying 223 probes in region: 1233\n",
      "--- assemblying 169 probes in region: 1234\n",
      "--- assemblying 241 probes in region: 1235\n",
      "--- assemblying 271 probes in region: 1236\n",
      "--- assemblying 228 probes in region: 1237\n",
      "--- assemblying 287 probes in region: 1238\n",
      "--- assemblying 249 probes in region: 1239\n",
      "--- assemblying 235 probes in region: 124\n",
      "--- assemblying 297 probes in region: 1240\n",
      "--- assemblying 344 probes in region: 1241\n",
      "--- assemblying 279 probes in region: 1242\n",
      "--- assemblying 297 probes in region: 1243\n",
      "--- assemblying 260 probes in region: 1244\n",
      "--- assemblying 283 probes in region: 1245\n",
      "--- assemblying 305 probes in region: 1246\n",
      "--- assemblying 234 probes in region: 1247\n",
      "--- assemblying 160 probes in region: 1248\n",
      "--- assemblying 237 probes in region: 1249\n",
      "--- assemblying 251 probes in region: 125\n",
      "--- assemblying 310 probes in region: 1250\n",
      "--- assemblying 181 probes in region: 1251\n",
      "--- assemblying 181 probes in region: 1252\n",
      "--- assemblying 246 probes in region: 1253\n",
      "--- assemblying 178 probes in region: 1254\n",
      "--- assemblying 303 probes in region: 1255\n",
      "--- assemblying 186 probes in region: 1256\n",
      "--- assemblying 173 probes in region: 1257\n",
      "--- assemblying 168 probes in region: 1258\n",
      "--- assemblying 216 probes in region: 1259\n",
      "--- assemblying 291 probes in region: 126\n",
      "--- assemblying 364 probes in region: 1260\n",
      "--- assemblying 377 probes in region: 1261\n",
      "--- assemblying 85 probes in region: 1262\n",
      "--- assemblying 189 probes in region: 1263\n",
      "--- assemblying 279 probes in region: 1264\n",
      "--- assemblying 99 probes in region: 1265\n",
      "--- assemblying 51 probes in region: 1269\n",
      "--- assemblying 222 probes in region: 127\n",
      "--- assemblying 305 probes in region: 1270\n",
      "--- assemblying 312 probes in region: 1271\n",
      "--- assemblying 317 probes in region: 1272\n",
      "--- assemblying 297 probes in region: 1273\n",
      "--- assemblying 192 probes in region: 1274\n",
      "--- assemblying 356 probes in region: 1275\n",
      "--- assemblying 244 probes in region: 1276\n",
      "--- assemblying 362 probes in region: 1277\n",
      "--- assemblying 150 probes in region: 1278\n",
      "--- assemblying 193 probes in region: 1279\n",
      "--- assemblying 275 probes in region: 128\n",
      "--- assemblying 296 probes in region: 1280\n",
      "--- assemblying 275 probes in region: 1281\n",
      "--- assemblying 234 probes in region: 1282\n",
      "--- assemblying 188 probes in region: 1284\n",
      "--- assemblying 206 probes in region: 1285\n",
      "--- assemblying 263 probes in region: 1286\n",
      "--- assemblying 287 probes in region: 1287\n",
      "--- assemblying 239 probes in region: 1288\n",
      "--- assemblying 368 probes in region: 1289\n",
      "--- assemblying 216 probes in region: 129\n",
      "--- assemblying 315 probes in region: 1290\n",
      "--- assemblying 273 probes in region: 1291\n",
      "--- assemblying 248 probes in region: 1292\n",
      "--- assemblying 312 probes in region: 1293\n",
      "--- assemblying 90 probes in region: 1294\n",
      "--- assemblying 167 probes in region: 1295\n",
      "--- assemblying 312 probes in region: 1297\n",
      "--- assemblying 282 probes in region: 1298\n",
      "--- assemblying 267 probes in region: 1299\n",
      "--- assemblying 341 probes in region: 13\n",
      "--- assemblying 162 probes in region: 130\n",
      "--- assemblying 266 probes in region: 1300\n",
      "--- assemblying 252 probes in region: 1301\n",
      "--- assemblying 220 probes in region: 1302\n",
      "--- assemblying 174 probes in region: 1303\n",
      "--- assemblying 268 probes in region: 1304\n",
      "--- assemblying 297 probes in region: 1305\n",
      "--- assemblying 285 probes in region: 1306\n",
      "--- assemblying 271 probes in region: 1307\n",
      "--- assemblying 243 probes in region: 1308\n",
      "--- assemblying 325 probes in region: 1309\n",
      "--- assemblying 190 probes in region: 131\n",
      "--- assemblying 285 probes in region: 1310\n",
      "--- assemblying 211 probes in region: 1311\n",
      "--- assemblying 257 probes in region: 1312\n",
      "--- assemblying 201 probes in region: 1313\n",
      "--- assemblying 272 probes in region: 1314\n",
      "--- assemblying 347 probes in region: 1315\n",
      "--- assemblying 301 probes in region: 1316\n",
      "--- assemblying 344 probes in region: 1317\n",
      "--- assemblying 253 probes in region: 1318\n",
      "--- assemblying 385 probes in region: 1319\n",
      "--- assemblying 192 probes in region: 132\n",
      "--- assemblying 286 probes in region: 1320\n",
      "--- assemblying 321 probes in region: 1321\n",
      "--- assemblying 310 probes in region: 1322\n",
      "--- assemblying 306 probes in region: 1323\n",
      "--- assemblying 321 probes in region: 1324\n",
      "--- assemblying 367 probes in region: 1325\n",
      "--- assemblying 312 probes in region: 133\n",
      "--- assemblying 129 probes in region: 1330\n",
      "--- assemblying 242 probes in region: 1331\n",
      "--- assemblying 278 probes in region: 1332\n",
      "--- assemblying 136 probes in region: 1333\n",
      "--- assemblying 161 probes in region: 1334\n",
      "--- assemblying 326 probes in region: 1335\n",
      "--- assemblying 266 probes in region: 1336\n",
      "--- assemblying 339 probes in region: 1337\n",
      "--- assemblying 188 probes in region: 1338\n",
      "--- assemblying 238 probes in region: 1339\n",
      "--- assemblying 237 probes in region: 134\n",
      "--- assemblying 344 probes in region: 1340\n",
      "--- assemblying 308 probes in region: 1341\n",
      "--- assemblying 261 probes in region: 1342\n",
      "--- assemblying 201 probes in region: 1343\n",
      "--- assemblying 274 probes in region: 1344\n",
      "--- assemblying 312 probes in region: 1345\n",
      "--- assemblying 298 probes in region: 1346\n",
      "--- assemblying 179 probes in region: 1347\n",
      "--- assemblying 130 probes in region: 1348\n",
      "--- assemblying 302 probes in region: 1349\n",
      "--- assemblying 265 probes in region: 1350\n",
      "--- assemblying 353 probes in region: 1351\n",
      "--- assemblying 243 probes in region: 1352\n",
      "--- assemblying 192 probes in region: 1353\n",
      "--- assemblying 278 probes in region: 1354\n",
      "--- assemblying 187 probes in region: 1355\n",
      "--- assemblying 291 probes in region: 1356\n",
      "--- assemblying 248 probes in region: 1357\n",
      "--- assemblying 327 probes in region: 1358\n",
      "--- assemblying 267 probes in region: 1359\n",
      "--- assemblying 324 probes in region: 1360\n",
      "--- assemblying 397 probes in region: 1361\n",
      "--- assemblying 385 probes in region: 1362\n",
      "--- assemblying 314 probes in region: 1363\n",
      "--- assemblying 315 probes in region: 137\n",
      "--- assemblying 217 probes in region: 1378\n",
      "--- assemblying 292 probes in region: 1379\n",
      "--- assemblying 285 probes in region: 138\n",
      "--- assemblying 192 probes in region: 1380\n",
      "--- assemblying 214 probes in region: 1381\n",
      "--- assemblying 210 probes in region: 1382\n",
      "--- assemblying 230 probes in region: 1383\n",
      "--- assemblying 148 probes in region: 1384\n",
      "--- assemblying 163 probes in region: 1385\n",
      "--- assemblying 128 probes in region: 1386\n",
      "--- assemblying 190 probes in region: 1387\n",
      "--- assemblying 176 probes in region: 1388\n",
      "--- assemblying 251 probes in region: 1389\n",
      "--- assemblying 174 probes in region: 139\n",
      "--- assemblying 288 probes in region: 1390\n",
      "--- assemblying 233 probes in region: 1391\n",
      "--- assemblying 244 probes in region: 1392\n",
      "--- assemblying 323 probes in region: 1393\n",
      "--- assemblying 174 probes in region: 1394\n",
      "--- assemblying 282 probes in region: 1395\n",
      "--- assemblying 192 probes in region: 1396\n",
      "--- assemblying 184 probes in region: 1398\n",
      "--- assemblying 268 probes in region: 1399\n",
      "--- assemblying 176 probes in region: 14\n",
      "--- assemblying 261 probes in region: 140\n",
      "--- assemblying 308 probes in region: 1400\n",
      "--- assemblying 268 probes in region: 1401\n",
      "--- assemblying 308 probes in region: 1402\n",
      "--- assemblying 223 probes in region: 1403\n",
      "--- assemblying 235 probes in region: 1404\n",
      "--- assemblying 336 probes in region: 1405\n",
      "--- assemblying 328 probes in region: 1406\n",
      "--- assemblying 354 probes in region: 1408\n",
      "--- assemblying 374 probes in region: 1409\n",
      "--- assemblying 74 probes in region: 141\n",
      "--- assemblying 253 probes in region: 1427\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 266 probes in region: 1429\n",
      "--- assemblying 186 probes in region: 143\n",
      "--- assemblying 240 probes in region: 1430\n",
      "--- assemblying 295 probes in region: 1434\n",
      "--- assemblying 255 probes in region: 1435\n",
      "--- assemblying 298 probes in region: 1436\n",
      "--- assemblying 360 probes in region: 1437\n",
      "--- assemblying 174 probes in region: 1438\n",
      "--- assemblying 198 probes in region: 1439\n",
      "--- assemblying 187 probes in region: 144\n",
      "--- assemblying 236 probes in region: 1440\n",
      "--- assemblying 184 probes in region: 1441\n",
      "--- assemblying 248 probes in region: 1442\n",
      "--- assemblying 316 probes in region: 1443\n",
      "--- assemblying 314 probes in region: 1444\n",
      "--- assemblying 316 probes in region: 1445\n",
      "--- assemblying 274 probes in region: 1446\n",
      "--- assemblying 380 probes in region: 1447\n",
      "--- assemblying 267 probes in region: 1448\n",
      "--- assemblying 343 probes in region: 1449\n",
      "--- assemblying 178 probes in region: 145\n",
      "--- assemblying 211 probes in region: 1450\n",
      "--- assemblying 255 probes in region: 1451\n",
      "--- assemblying 113 probes in region: 1452\n",
      "--- assemblying 304 probes in region: 1453\n",
      "--- assemblying 277 probes in region: 1454\n",
      "--- assemblying 290 probes in region: 1455\n",
      "--- assemblying 270 probes in region: 1456\n",
      "--- assemblying 281 probes in region: 1457\n",
      "--- assemblying 396 probes in region: 1458\n",
      "--- assemblying 326 probes in region: 1459\n",
      "--- assemblying 294 probes in region: 146\n",
      "--- assemblying 143 probes in region: 1461\n",
      "--- assemblying 238 probes in region: 1462\n",
      "--- assemblying 268 probes in region: 1463\n",
      "--- assemblying 281 probes in region: 1464\n",
      "--- assemblying 231 probes in region: 1465\n",
      "--- assemblying 286 probes in region: 1466\n",
      "--- assemblying 193 probes in region: 1467\n",
      "--- assemblying 180 probes in region: 1468\n",
      "--- assemblying 202 probes in region: 1469\n",
      "--- assemblying 355 probes in region: 147\n",
      "--- assemblying 194 probes in region: 1470\n",
      "--- assemblying 221 probes in region: 1471\n",
      "--- assemblying 278 probes in region: 1472\n",
      "--- assemblying 117 probes in region: 1473\n",
      "--- assemblying 173 probes in region: 1474\n",
      "--- assemblying 234 probes in region: 1475\n",
      "--- assemblying 197 probes in region: 1476\n",
      "--- assemblying 341 probes in region: 1477\n",
      "--- assemblying 230 probes in region: 1478\n",
      "--- assemblying 216 probes in region: 1479\n",
      "--- assemblying 193 probes in region: 148\n",
      "--- assemblying 117 probes in region: 1480\n",
      "--- assemblying 198 probes in region: 1481\n",
      "--- assemblying 212 probes in region: 1482\n",
      "--- assemblying 351 probes in region: 1485\n",
      "--- assemblying 227 probes in region: 1486\n",
      "--- assemblying 250 probes in region: 1487\n",
      "--- assemblying 311 probes in region: 1488\n",
      "--- assemblying 94 probes in region: 1489\n",
      "--- assemblying 309 probes in region: 149\n",
      "--- assemblying 128 probes in region: 1490\n",
      "--- assemblying 239 probes in region: 1491\n",
      "--- assemblying 238 probes in region: 1492\n",
      "--- assemblying 129 probes in region: 1493\n",
      "--- assemblying 189 probes in region: 1494\n",
      "--- assemblying 199 probes in region: 1495\n",
      "--- assemblying 72 probes in region: 1496\n",
      "--- assemblying 230 probes in region: 1497\n",
      "--- assemblying 146 probes in region: 1498\n",
      "--- assemblying 160 probes in region: 1499\n",
      "--- assemblying 343 probes in region: 15\n",
      "--- assemblying 240 probes in region: 150\n",
      "--- assemblying 369 probes in region: 1500\n",
      "--- assemblying 253 probes in region: 1501\n",
      "--- assemblying 139 probes in region: 1502\n",
      "--- assemblying 186 probes in region: 1503\n",
      "--- assemblying 285 probes in region: 1504\n",
      "--- assemblying 239 probes in region: 1505\n",
      "--- assemblying 163 probes in region: 1506\n",
      "--- assemblying 242 probes in region: 1507\n",
      "--- assemblying 189 probes in region: 1508\n",
      "--- assemblying 188 probes in region: 1509\n",
      "--- assemblying 176 probes in region: 1510\n",
      "--- assemblying 218 probes in region: 1511\n",
      "--- assemblying 248 probes in region: 1512\n",
      "--- assemblying 245 probes in region: 1513\n",
      "--- assemblying 262 probes in region: 1514\n",
      "--- assemblying 357 probes in region: 1515\n",
      "--- assemblying 103 probes in region: 1516\n",
      "--- assemblying 232 probes in region: 1517\n",
      "--- assemblying 217 probes in region: 1518\n",
      "--- assemblying 239 probes in region: 1519\n",
      "--- assemblying 248 probes in region: 152\n",
      "--- assemblying 221 probes in region: 1520\n",
      "--- assemblying 176 probes in region: 1521\n",
      "--- assemblying 51 probes in region: 1525\n",
      "--- assemblying 60 probes in region: 1526\n",
      "--- assemblying 72 probes in region: 1527\n",
      "--- assemblying 166 probes in region: 1529\n",
      "--- assemblying 307 probes in region: 153\n",
      "--- assemblying 218 probes in region: 1530\n",
      "--- assemblying 69 probes in region: 1533\n",
      "--- assemblying 149 probes in region: 1534\n",
      "--- assemblying 187 probes in region: 1535\n",
      "--- assemblying 187 probes in region: 1536\n",
      "--- assemblying 170 probes in region: 1537\n",
      "--- assemblying 164 probes in region: 1538\n",
      "--- assemblying 283 probes in region: 154\n",
      "--- assemblying 251 probes in region: 1541\n",
      "--- assemblying 204 probes in region: 1542\n",
      "--- assemblying 106 probes in region: 1548\n",
      "--- assemblying 252 probes in region: 155\n",
      "--- assemblying 292 probes in region: 156\n",
      "--- assemblying 255 probes in region: 157\n",
      "--- assemblying 227 probes in region: 158\n",
      "--- assemblying 231 probes in region: 159\n",
      "--- assemblying 245 probes in region: 16\n",
      "--- assemblying 136 probes in region: 160\n",
      "--- assemblying 134 probes in region: 161\n",
      "--- assemblying 338 probes in region: 162\n",
      "--- assemblying 234 probes in region: 163\n",
      "--- assemblying 303 probes in region: 164\n",
      "--- assemblying 244 probes in region: 165\n",
      "--- assemblying 304 probes in region: 166\n",
      "--- assemblying 217 probes in region: 167\n",
      "--- assemblying 301 probes in region: 168\n",
      "--- assemblying 238 probes in region: 169\n",
      "--- assemblying 203 probes in region: 17\n",
      "--- assemblying 303 probes in region: 170\n",
      "--- assemblying 272 probes in region: 171\n",
      "--- assemblying 200 probes in region: 172\n",
      "--- assemblying 173 probes in region: 173\n",
      "--- assemblying 198 probes in region: 174\n",
      "--- assemblying 317 probes in region: 175\n",
      "--- assemblying 152 probes in region: 176\n",
      "--- assemblying 279 probes in region: 177\n",
      "--- assemblying 280 probes in region: 178\n",
      "--- assemblying 230 probes in region: 179\n",
      "--- assemblying 204 probes in region: 18\n",
      "--- assemblying 226 probes in region: 180\n",
      "--- assemblying 248 probes in region: 181\n",
      "--- assemblying 93 probes in region: 182\n",
      "--- assemblying 204 probes in region: 183\n",
      "--- assemblying 173 probes in region: 184\n",
      "--- assemblying 313 probes in region: 185\n",
      "--- assemblying 301 probes in region: 186\n",
      "--- assemblying 239 probes in region: 187\n",
      "--- assemblying 233 probes in region: 188\n",
      "--- assemblying 191 probes in region: 189\n",
      "--- assemblying 391 probes in region: 19\n",
      "--- assemblying 150 probes in region: 190\n",
      "--- assemblying 274 probes in region: 191\n",
      "--- assemblying 270 probes in region: 192\n",
      "--- assemblying 357 probes in region: 193\n",
      "--- assemblying 297 probes in region: 194\n",
      "--- assemblying 199 probes in region: 195\n",
      "--- assemblying 317 probes in region: 196\n",
      "--- assemblying 251 probes in region: 197\n",
      "--- assemblying 379 probes in region: 198\n",
      "--- assemblying 275 probes in region: 199\n",
      "--- assemblying 377 probes in region: 2\n",
      "--- assemblying 206 probes in region: 20\n",
      "--- assemblying 229 probes in region: 200\n",
      "--- assemblying 361 probes in region: 201\n",
      "--- assemblying 331 probes in region: 202\n",
      "--- assemblying 190 probes in region: 203\n",
      "--- assemblying 265 probes in region: 204\n",
      "--- assemblying 241 probes in region: 205\n",
      "--- assemblying 262 probes in region: 206\n",
      "--- assemblying 324 probes in region: 207\n",
      "--- assemblying 174 probes in region: 208\n",
      "--- assemblying 180 probes in region: 209\n",
      "--- assemblying 255 probes in region: 21\n",
      "--- assemblying 365 probes in region: 210\n",
      "--- assemblying 349 probes in region: 211\n",
      "--- assemblying 330 probes in region: 212\n",
      "--- assemblying 200 probes in region: 213\n",
      "--- assemblying 219 probes in region: 214\n",
      "--- assemblying 142 probes in region: 215\n",
      "--- assemblying 400 probes in region: 216\n",
      "--- assemblying 278 probes in region: 217\n",
      "--- assemblying 252 probes in region: 218\n",
      "--- assemblying 258 probes in region: 219\n",
      "--- assemblying 303 probes in region: 22\n",
      "--- assemblying 214 probes in region: 220\n",
      "--- assemblying 256 probes in region: 221\n",
      "--- assemblying 292 probes in region: 222\n",
      "--- assemblying 318 probes in region: 223\n",
      "--- assemblying 322 probes in region: 224\n",
      "--- assemblying 234 probes in region: 225\n",
      "--- assemblying 220 probes in region: 226\n",
      "--- assemblying 242 probes in region: 227\n",
      "--- assemblying 148 probes in region: 228\n",
      "--- assemblying 197 probes in region: 229\n",
      "--- assemblying 333 probes in region: 23\n",
      "--- assemblying 166 probes in region: 230\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 196 probes in region: 233\n",
      "--- assemblying 255 probes in region: 234\n",
      "--- assemblying 233 probes in region: 235\n",
      "--- assemblying 182 probes in region: 236\n",
      "--- assemblying 179 probes in region: 237\n",
      "--- assemblying 251 probes in region: 238\n",
      "--- assemblying 231 probes in region: 239\n",
      "--- assemblying 279 probes in region: 24\n",
      "--- assemblying 234 probes in region: 240\n",
      "--- assemblying 272 probes in region: 241\n",
      "--- assemblying 288 probes in region: 242\n",
      "--- assemblying 165 probes in region: 243\n",
      "--- assemblying 333 probes in region: 244\n",
      "--- assemblying 238 probes in region: 245\n",
      "--- assemblying 205 probes in region: 246\n",
      "--- assemblying 234 probes in region: 247\n",
      "--- assemblying 277 probes in region: 248\n",
      "--- assemblying 242 probes in region: 249\n",
      "--- assemblying 238 probes in region: 25\n",
      "--- assemblying 172 probes in region: 250\n",
      "--- assemblying 357 probes in region: 251\n",
      "--- assemblying 287 probes in region: 252\n",
      "--- assemblying 188 probes in region: 253\n",
      "--- assemblying 282 probes in region: 254\n",
      "--- assemblying 225 probes in region: 255\n",
      "--- assemblying 218 probes in region: 256\n",
      "--- assemblying 288 probes in region: 257\n",
      "--- assemblying 314 probes in region: 258\n",
      "--- assemblying 160 probes in region: 259\n",
      "--- assemblying 328 probes in region: 26\n",
      "--- assemblying 162 probes in region: 260\n",
      "--- assemblying 159 probes in region: 261\n",
      "--- assemblying 179 probes in region: 262\n",
      "--- assemblying 324 probes in region: 263\n",
      "--- assemblying 259 probes in region: 264\n",
      "--- assemblying 220 probes in region: 265\n",
      "--- assemblying 251 probes in region: 266\n",
      "--- assemblying 263 probes in region: 267\n",
      "--- assemblying 175 probes in region: 268\n",
      "--- assemblying 351 probes in region: 269\n",
      "--- assemblying 149 probes in region: 27\n",
      "--- assemblying 313 probes in region: 270\n",
      "--- assemblying 220 probes in region: 271\n",
      "--- assemblying 232 probes in region: 272\n",
      "--- assemblying 231 probes in region: 273\n",
      "--- assemblying 330 probes in region: 274\n",
      "--- assemblying 390 probes in region: 275\n",
      "--- assemblying 325 probes in region: 276\n",
      "--- assemblying 313 probes in region: 277\n",
      "--- assemblying 224 probes in region: 278\n",
      "--- assemblying 210 probes in region: 279\n",
      "--- assemblying 262 probes in region: 28\n",
      "--- assemblying 309 probes in region: 280\n",
      "--- assemblying 239 probes in region: 281\n",
      "--- assemblying 262 probes in region: 282\n",
      "--- assemblying 293 probes in region: 283\n",
      "--- assemblying 215 probes in region: 284\n",
      "--- assemblying 234 probes in region: 285\n",
      "--- assemblying 187 probes in region: 286\n",
      "--- assemblying 194 probes in region: 287\n",
      "--- assemblying 249 probes in region: 288\n",
      "--- assemblying 250 probes in region: 289\n",
      "--- assemblying 199 probes in region: 29\n",
      "--- assemblying 253 probes in region: 290\n",
      "--- assemblying 195 probes in region: 291\n",
      "--- assemblying 226 probes in region: 292\n",
      "--- assemblying 157 probes in region: 293\n",
      "--- assemblying 209 probes in region: 295\n",
      "--- assemblying 274 probes in region: 296\n",
      "--- assemblying 194 probes in region: 297\n",
      "--- assemblying 253 probes in region: 298\n",
      "--- assemblying 141 probes in region: 299\n",
      "--- assemblying 242 probes in region: 3\n",
      "--- assemblying 263 probes in region: 30\n",
      "--- assemblying 151 probes in region: 300\n",
      "--- assemblying 328 probes in region: 302\n",
      "--- assemblying 213 probes in region: 303\n",
      "--- assemblying 232 probes in region: 304\n",
      "--- assemblying 222 probes in region: 305\n",
      "--- assemblying 242 probes in region: 306\n",
      "--- assemblying 231 probes in region: 307\n",
      "--- assemblying 203 probes in region: 308\n",
      "--- assemblying 343 probes in region: 309\n",
      "--- assemblying 215 probes in region: 31\n",
      "--- assemblying 166 probes in region: 310\n",
      "--- assemblying 267 probes in region: 311\n",
      "--- assemblying 241 probes in region: 312\n",
      "--- assemblying 182 probes in region: 313\n",
      "--- assemblying 253 probes in region: 314\n",
      "--- assemblying 228 probes in region: 315\n",
      "--- assemblying 183 probes in region: 316\n",
      "--- assemblying 251 probes in region: 317\n",
      "--- assemblying 178 probes in region: 318\n",
      "--- assemblying 249 probes in region: 319\n",
      "--- assemblying 190 probes in region: 32\n",
      "--- assemblying 155 probes in region: 320\n",
      "--- assemblying 279 probes in region: 321\n",
      "--- assemblying 258 probes in region: 322\n",
      "--- assemblying 248 probes in region: 323\n",
      "--- assemblying 171 probes in region: 324\n",
      "--- assemblying 176 probes in region: 325\n",
      "--- assemblying 191 probes in region: 326\n",
      "--- assemblying 106 probes in region: 327\n",
      "--- assemblying 170 probes in region: 328\n",
      "--- assemblying 295 probes in region: 329\n",
      "--- assemblying 338 probes in region: 33\n",
      "--- assemblying 218 probes in region: 330\n",
      "--- assemblying 154 probes in region: 331\n",
      "--- assemblying 377 probes in region: 332\n",
      "--- assemblying 214 probes in region: 333\n",
      "--- assemblying 234 probes in region: 334\n",
      "--- assemblying 271 probes in region: 335\n",
      "--- assemblying 244 probes in region: 336\n",
      "--- assemblying 181 probes in region: 337\n",
      "--- assemblying 222 probes in region: 338\n",
      "--- assemblying 257 probes in region: 339\n",
      "--- assemblying 230 probes in region: 34\n",
      "--- assemblying 201 probes in region: 340\n",
      "--- assemblying 284 probes in region: 341\n",
      "--- assemblying 157 probes in region: 342\n",
      "--- assemblying 260 probes in region: 343\n",
      "--- assemblying 234 probes in region: 344\n",
      "--- assemblying 225 probes in region: 345\n",
      "--- assemblying 279 probes in region: 346\n",
      "--- assemblying 202 probes in region: 347\n",
      "--- assemblying 223 probes in region: 348\n",
      "--- assemblying 161 probes in region: 349\n",
      "--- assemblying 203 probes in region: 35\n",
      "--- assemblying 400 probes in region: 350\n",
      "--- assemblying 319 probes in region: 351\n",
      "--- assemblying 325 probes in region: 352\n",
      "--- assemblying 158 probes in region: 353\n",
      "--- assemblying 293 probes in region: 354\n",
      "--- assemblying 234 probes in region: 355\n",
      "--- assemblying 257 probes in region: 356\n",
      "--- assemblying 170 probes in region: 357\n",
      "--- assemblying 216 probes in region: 358\n",
      "--- assemblying 179 probes in region: 359\n",
      "--- assemblying 154 probes in region: 36\n",
      "--- assemblying 190 probes in region: 360\n",
      "--- assemblying 175 probes in region: 361\n",
      "--- assemblying 315 probes in region: 362\n",
      "--- assemblying 350 probes in region: 363\n",
      "--- assemblying 253 probes in region: 364\n",
      "--- assemblying 284 probes in region: 365\n",
      "--- assemblying 214 probes in region: 366\n",
      "--- assemblying 102 probes in region: 367\n",
      "--- assemblying 64 probes in region: 368\n",
      "--- assemblying 358 probes in region: 37\n",
      "--- assemblying 158 probes in region: 370\n",
      "--- assemblying 84 probes in region: 371\n",
      "--- assemblying 236 probes in region: 372\n",
      "--- assemblying 270 probes in region: 373\n",
      "--- assemblying 224 probes in region: 374\n",
      "--- assemblying 226 probes in region: 375\n",
      "--- assemblying 222 probes in region: 376\n",
      "--- assemblying 340 probes in region: 377\n",
      "--- assemblying 287 probes in region: 379\n",
      "--- assemblying 211 probes in region: 38\n",
      "--- assemblying 224 probes in region: 380\n",
      "--- assemblying 278 probes in region: 381\n",
      "--- assemblying 318 probes in region: 382\n",
      "--- assemblying 176 probes in region: 383\n",
      "--- assemblying 166 probes in region: 384\n",
      "--- assemblying 272 probes in region: 385\n",
      "--- assemblying 215 probes in region: 386\n",
      "--- assemblying 192 probes in region: 387\n",
      "--- assemblying 277 probes in region: 388\n",
      "--- assemblying 170 probes in region: 389\n",
      "--- assemblying 237 probes in region: 39\n",
      "--- assemblying 164 probes in region: 390\n",
      "--- assemblying 257 probes in region: 391\n",
      "--- assemblying 202 probes in region: 392\n",
      "--- assemblying 229 probes in region: 393\n",
      "--- assemblying 309 probes in region: 394\n",
      "--- assemblying 256 probes in region: 395\n",
      "--- assemblying 182 probes in region: 396\n",
      "--- assemblying 245 probes in region: 397\n",
      "--- assemblying 162 probes in region: 398\n",
      "--- assemblying 196 probes in region: 399\n",
      "--- assemblying 256 probes in region: 4\n",
      "--- assemblying 293 probes in region: 40\n",
      "--- assemblying 222 probes in region: 400\n",
      "--- assemblying 237 probes in region: 401\n",
      "--- assemblying 191 probes in region: 402\n",
      "--- assemblying 377 probes in region: 403\n",
      "--- assemblying 331 probes in region: 404\n",
      "--- assemblying 205 probes in region: 405\n",
      "--- assemblying 261 probes in region: 406\n",
      "--- assemblying 306 probes in region: 407\n",
      "--- assemblying 163 probes in region: 408\n",
      "--- assemblying 221 probes in region: 409\n",
      "--- assemblying 271 probes in region: 410\n",
      "--- assemblying 282 probes in region: 411\n",
      "--- assemblying 212 probes in region: 412\n",
      "--- assemblying 339 probes in region: 413\n",
      "--- assemblying 188 probes in region: 414\n",
      "--- assemblying 252 probes in region: 415\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 252 probes in region: 416\n",
      "--- assemblying 315 probes in region: 417\n",
      "--- assemblying 200 probes in region: 418\n",
      "--- assemblying 355 probes in region: 419\n",
      "--- assemblying 201 probes in region: 42\n",
      "--- assemblying 212 probes in region: 421\n",
      "--- assemblying 298 probes in region: 422\n",
      "--- assemblying 345 probes in region: 423\n",
      "--- assemblying 323 probes in region: 424\n",
      "--- assemblying 221 probes in region: 425\n",
      "--- assemblying 350 probes in region: 426\n",
      "--- assemblying 217 probes in region: 427\n",
      "--- assemblying 243 probes in region: 428\n",
      "--- assemblying 193 probes in region: 429\n",
      "--- assemblying 235 probes in region: 43\n",
      "--- assemblying 236 probes in region: 430\n",
      "--- assemblying 237 probes in region: 431\n",
      "--- assemblying 270 probes in region: 432\n",
      "--- assemblying 357 probes in region: 435\n",
      "--- assemblying 294 probes in region: 436\n",
      "--- assemblying 188 probes in region: 437\n",
      "--- assemblying 206 probes in region: 438\n",
      "--- assemblying 310 probes in region: 439\n",
      "--- assemblying 317 probes in region: 44\n",
      "--- assemblying 321 probes in region: 440\n",
      "--- assemblying 183 probes in region: 441\n",
      "--- assemblying 276 probes in region: 442\n",
      "--- assemblying 241 probes in region: 443\n",
      "--- assemblying 307 probes in region: 444\n",
      "--- assemblying 108 probes in region: 446\n",
      "--- assemblying 181 probes in region: 447\n",
      "--- assemblying 127 probes in region: 448\n",
      "--- assemblying 228 probes in region: 449\n",
      "--- assemblying 187 probes in region: 45\n",
      "--- assemblying 268 probes in region: 450\n",
      "--- assemblying 258 probes in region: 451\n",
      "--- assemblying 197 probes in region: 452\n",
      "--- assemblying 67 probes in region: 453\n",
      "--- assemblying 145 probes in region: 454\n",
      "--- assemblying 142 probes in region: 455\n",
      "--- assemblying 217 probes in region: 456\n",
      "--- assemblying 207 probes in region: 457\n",
      "--- assemblying 236 probes in region: 458\n",
      "--- assemblying 180 probes in region: 459\n",
      "--- assemblying 309 probes in region: 46\n",
      "--- assemblying 202 probes in region: 460\n",
      "--- assemblying 209 probes in region: 461\n",
      "--- assemblying 170 probes in region: 462\n",
      "--- assemblying 143 probes in region: 463\n",
      "--- assemblying 211 probes in region: 464\n",
      "--- assemblying 352 probes in region: 465\n",
      "--- assemblying 189 probes in region: 466\n",
      "--- assemblying 200 probes in region: 467\n",
      "--- assemblying 222 probes in region: 468\n",
      "--- assemblying 260 probes in region: 469\n",
      "--- assemblying 213 probes in region: 47\n",
      "--- assemblying 131 probes in region: 470\n",
      "--- assemblying 199 probes in region: 471\n",
      "--- assemblying 203 probes in region: 472\n",
      "--- assemblying 203 probes in region: 473\n",
      "--- assemblying 235 probes in region: 474\n",
      "--- assemblying 263 probes in region: 475\n",
      "--- assemblying 236 probes in region: 476\n",
      "--- assemblying 215 probes in region: 477\n",
      "--- assemblying 189 probes in region: 478\n",
      "--- assemblying 287 probes in region: 479\n",
      "--- assemblying 257 probes in region: 480\n",
      "--- assemblying 269 probes in region: 481\n",
      "--- assemblying 230 probes in region: 482\n",
      "--- assemblying 220 probes in region: 483\n",
      "--- assemblying 265 probes in region: 484\n",
      "--- assemblying 255 probes in region: 485\n",
      "--- assemblying 294 probes in region: 486\n",
      "--- assemblying 342 probes in region: 487\n",
      "--- assemblying 216 probes in region: 489\n",
      "--- assemblying 384 probes in region: 490\n",
      "--- assemblying 276 probes in region: 491\n",
      "--- assemblying 229 probes in region: 492\n",
      "--- assemblying 239 probes in region: 493\n",
      "--- assemblying 293 probes in region: 494\n",
      "--- assemblying 259 probes in region: 495\n",
      "--- assemblying 167 probes in region: 496\n",
      "--- assemblying 228 probes in region: 497\n",
      "--- assemblying 262 probes in region: 498\n",
      "--- assemblying 162 probes in region: 499\n",
      "--- assemblying 303 probes in region: 5\n",
      "--- assemblying 142 probes in region: 500\n",
      "--- assemblying 304 probes in region: 501\n",
      "--- assemblying 269 probes in region: 502\n",
      "--- assemblying 318 probes in region: 503\n",
      "--- assemblying 220 probes in region: 504\n",
      "--- assemblying 308 probes in region: 505\n",
      "--- assemblying 306 probes in region: 506\n",
      "--- assemblying 202 probes in region: 507\n",
      "--- assemblying 221 probes in region: 508\n",
      "--- assemblying 272 probes in region: 509\n",
      "--- assemblying 336 probes in region: 510\n",
      "--- assemblying 184 probes in region: 511\n",
      "--- assemblying 197 probes in region: 512\n",
      "--- assemblying 227 probes in region: 516\n",
      "--- assemblying 262 probes in region: 517\n",
      "--- assemblying 264 probes in region: 518\n",
      "--- assemblying 188 probes in region: 519\n",
      "--- assemblying 212 probes in region: 520\n",
      "--- assemblying 265 probes in region: 521\n",
      "--- assemblying 187 probes in region: 522\n",
      "--- assemblying 152 probes in region: 523\n",
      "--- assemblying 196 probes in region: 524\n",
      "--- assemblying 216 probes in region: 525\n",
      "--- assemblying 223 probes in region: 526\n",
      "--- assemblying 255 probes in region: 527\n",
      "--- assemblying 249 probes in region: 528\n",
      "--- assemblying 269 probes in region: 529\n",
      "--- assemblying 223 probes in region: 530\n",
      "--- assemblying 201 probes in region: 531\n",
      "--- assemblying 232 probes in region: 532\n",
      "--- assemblying 201 probes in region: 533\n",
      "--- assemblying 181 probes in region: 534\n",
      "--- assemblying 116 probes in region: 535\n",
      "--- assemblying 259 probes in region: 536\n",
      "--- assemblying 231 probes in region: 537\n",
      "--- assemblying 214 probes in region: 538\n",
      "--- assemblying 242 probes in region: 539\n",
      "--- assemblying 179 probes in region: 540\n",
      "--- assemblying 248 probes in region: 541\n",
      "--- assemblying 227 probes in region: 542\n",
      "--- assemblying 302 probes in region: 543\n",
      "--- assemblying 276 probes in region: 544\n",
      "--- assemblying 267 probes in region: 545\n",
      "--- assemblying 305 probes in region: 546\n",
      "--- assemblying 191 probes in region: 548\n",
      "--- assemblying 206 probes in region: 549\n",
      "--- assemblying 378 probes in region: 550\n",
      "--- assemblying 90 probes in region: 551\n",
      "--- assemblying 264 probes in region: 552\n",
      "--- assemblying 306 probes in region: 554\n",
      "--- assemblying 241 probes in region: 555\n",
      "--- assemblying 234 probes in region: 556\n",
      "--- assemblying 273 probes in region: 557\n",
      "--- assemblying 251 probes in region: 558\n",
      "--- assemblying 221 probes in region: 559\n",
      "--- assemblying 261 probes in region: 560\n",
      "--- assemblying 240 probes in region: 561\n",
      "--- assemblying 249 probes in region: 562\n",
      "--- assemblying 193 probes in region: 563\n",
      "--- assemblying 270 probes in region: 564\n",
      "--- assemblying 218 probes in region: 565\n",
      "--- assemblying 169 probes in region: 566\n",
      "--- assemblying 229 probes in region: 567\n",
      "--- assemblying 222 probes in region: 568\n",
      "--- assemblying 292 probes in region: 569\n",
      "--- assemblying 60 probes in region: 57\n",
      "--- assemblying 104 probes in region: 570\n",
      "--- assemblying 356 probes in region: 571\n",
      "--- assemblying 280 probes in region: 572\n",
      "--- assemblying 206 probes in region: 573\n",
      "--- assemblying 258 probes in region: 574\n",
      "--- assemblying 204 probes in region: 575\n",
      "--- assemblying 291 probes in region: 576\n",
      "--- assemblying 207 probes in region: 577\n",
      "--- assemblying 246 probes in region: 578\n",
      "--- assemblying 151 probes in region: 579\n",
      "--- assemblying 202 probes in region: 580\n",
      "--- assemblying 250 probes in region: 581\n",
      "--- assemblying 199 probes in region: 582\n",
      "--- assemblying 181 probes in region: 583\n",
      "--- assemblying 139 probes in region: 584\n",
      "--- assemblying 241 probes in region: 585\n",
      "--- assemblying 218 probes in region: 586\n",
      "--- assemblying 156 probes in region: 587\n",
      "--- assemblying 177 probes in region: 588\n",
      "--- assemblying 275 probes in region: 589\n",
      "--- assemblying 225 probes in region: 590\n",
      "--- assemblying 283 probes in region: 591\n",
      "--- assemblying 144 probes in region: 592\n",
      "--- assemblying 306 probes in region: 593\n",
      "--- assemblying 155 probes in region: 594\n",
      "--- assemblying 180 probes in region: 595\n",
      "--- assemblying 254 probes in region: 596\n",
      "--- assemblying 277 probes in region: 597\n",
      "--- assemblying 226 probes in region: 598\n",
      "--- assemblying 304 probes in region: 599\n",
      "--- assemblying 300 probes in region: 6\n",
      "--- assemblying 187 probes in region: 60\n",
      "--- assemblying 294 probes in region: 600\n",
      "--- assemblying 299 probes in region: 601\n",
      "--- assemblying 261 probes in region: 602\n",
      "--- assemblying 251 probes in region: 603\n",
      "--- assemblying 187 probes in region: 605\n",
      "--- assemblying 293 probes in region: 606\n",
      "--- assemblying 281 probes in region: 607\n",
      "--- assemblying 355 probes in region: 608\n",
      "--- assemblying 323 probes in region: 609\n",
      "--- assemblying 302 probes in region: 611\n",
      "--- assemblying 311 probes in region: 612\n",
      "--- assemblying 200 probes in region: 613\n",
      "--- assemblying 277 probes in region: 614\n",
      "--- assemblying 258 probes in region: 615\n",
      "--- assemblying 173 probes in region: 616\n",
      "--- assemblying 254 probes in region: 617\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 192 probes in region: 618\n",
      "--- assemblying 282 probes in region: 619\n",
      "--- assemblying 330 probes in region: 62\n",
      "--- assemblying 196 probes in region: 620\n",
      "--- assemblying 248 probes in region: 621\n",
      "--- assemblying 247 probes in region: 622\n",
      "--- assemblying 185 probes in region: 623\n",
      "--- assemblying 132 probes in region: 624\n",
      "--- assemblying 168 probes in region: 625\n",
      "--- assemblying 118 probes in region: 626\n",
      "--- assemblying 253 probes in region: 627\n",
      "--- assemblying 273 probes in region: 628\n",
      "--- assemblying 360 probes in region: 629\n",
      "--- assemblying 241 probes in region: 63\n",
      "--- assemblying 199 probes in region: 64\n",
      "--- assemblying 257 probes in region: 645\n",
      "--- assemblying 241 probes in region: 646\n",
      "--- assemblying 219 probes in region: 647\n",
      "--- assemblying 215 probes in region: 648\n",
      "--- assemblying 262 probes in region: 649\n",
      "--- assemblying 114 probes in region: 65\n",
      "--- assemblying 241 probes in region: 650\n",
      "--- assemblying 146 probes in region: 651\n",
      "--- assemblying 306 probes in region: 652\n",
      "--- assemblying 373 probes in region: 653\n",
      "--- assemblying 271 probes in region: 654\n",
      "--- assemblying 400 probes in region: 655\n",
      "--- assemblying 292 probes in region: 656\n",
      "--- assemblying 186 probes in region: 657\n",
      "--- assemblying 311 probes in region: 658\n",
      "--- assemblying 54 probes in region: 659\n",
      "--- assemblying 127 probes in region: 66\n",
      "--- assemblying 278 probes in region: 660\n",
      "--- assemblying 184 probes in region: 662\n",
      "--- assemblying 205 probes in region: 663\n",
      "--- assemblying 255 probes in region: 664\n",
      "--- assemblying 266 probes in region: 665\n",
      "--- assemblying 273 probes in region: 666\n",
      "--- assemblying 172 probes in region: 667\n",
      "--- assemblying 328 probes in region: 668\n",
      "--- assemblying 312 probes in region: 669\n",
      "--- assemblying 271 probes in region: 67\n",
      "--- assemblying 311 probes in region: 670\n",
      "--- assemblying 199 probes in region: 671\n",
      "--- assemblying 171 probes in region: 672\n",
      "--- assemblying 120 probes in region: 673\n",
      "--- assemblying 258 probes in region: 674\n",
      "--- assemblying 327 probes in region: 675\n",
      "--- assemblying 330 probes in region: 676\n",
      "--- assemblying 308 probes in region: 677\n",
      "--- assemblying 347 probes in region: 678\n",
      "--- assemblying 320 probes in region: 679\n",
      "--- assemblying 226 probes in region: 68\n",
      "--- assemblying 294 probes in region: 680\n",
      "--- assemblying 308 probes in region: 681\n",
      "--- assemblying 147 probes in region: 682\n",
      "--- assemblying 285 probes in region: 683\n",
      "--- assemblying 253 probes in region: 684\n",
      "--- assemblying 328 probes in region: 685\n",
      "--- assemblying 266 probes in region: 686\n",
      "--- assemblying 78 probes in region: 687\n",
      "--- assemblying 278 probes in region: 688\n",
      "--- assemblying 160 probes in region: 689\n",
      "--- assemblying 128 probes in region: 69\n",
      "--- assemblying 289 probes in region: 690\n",
      "--- assemblying 270 probes in region: 691\n",
      "--- assemblying 189 probes in region: 692\n",
      "--- assemblying 246 probes in region: 693\n",
      "--- assemblying 236 probes in region: 694\n",
      "--- assemblying 246 probes in region: 695\n",
      "--- assemblying 170 probes in region: 696\n",
      "--- assemblying 220 probes in region: 697\n",
      "--- assemblying 227 probes in region: 698\n",
      "--- assemblying 66 probes in region: 699\n",
      "--- assemblying 341 probes in region: 7\n",
      "--- assemblying 120 probes in region: 70\n",
      "--- assemblying 289 probes in region: 701\n",
      "--- assemblying 291 probes in region: 702\n",
      "--- assemblying 181 probes in region: 703\n",
      "--- assemblying 284 probes in region: 704\n",
      "--- assemblying 292 probes in region: 705\n",
      "--- assemblying 246 probes in region: 706\n",
      "--- assemblying 182 probes in region: 707\n",
      "--- assemblying 213 probes in region: 708\n",
      "--- assemblying 289 probes in region: 709\n",
      "--- assemblying 137 probes in region: 71\n",
      "--- assemblying 234 probes in region: 710\n",
      "--- assemblying 290 probes in region: 711\n",
      "--- assemblying 201 probes in region: 712\n",
      "--- assemblying 236 probes in region: 713\n",
      "--- assemblying 196 probes in region: 714\n",
      "--- assemblying 353 probes in region: 715\n",
      "--- assemblying 136 probes in region: 716\n",
      "--- assemblying 282 probes in region: 717\n",
      "--- assemblying 391 probes in region: 718\n",
      "--- assemblying 376 probes in region: 719\n",
      "--- assemblying 267 probes in region: 72\n",
      "--- assemblying 218 probes in region: 720\n",
      "--- assemblying 270 probes in region: 721\n",
      "--- assemblying 309 probes in region: 722\n",
      "--- assemblying 73 probes in region: 723\n",
      "--- assemblying 288 probes in region: 724\n",
      "--- assemblying 271 probes in region: 725\n",
      "--- assemblying 209 probes in region: 726\n",
      "--- assemblying 138 probes in region: 727\n",
      "--- assemblying 365 probes in region: 728\n",
      "--- assemblying 200 probes in region: 729\n",
      "--- assemblying 198 probes in region: 73\n",
      "--- assemblying 379 probes in region: 730\n",
      "--- assemblying 174 probes in region: 731\n",
      "--- assemblying 296 probes in region: 732\n",
      "--- assemblying 274 probes in region: 733\n",
      "--- assemblying 225 probes in region: 734\n",
      "--- assemblying 206 probes in region: 735\n",
      "--- assemblying 362 probes in region: 736\n",
      "--- assemblying 252 probes in region: 737\n",
      "--- assemblying 222 probes in region: 738\n",
      "--- assemblying 229 probes in region: 739\n",
      "--- assemblying 173 probes in region: 74\n",
      "--- assemblying 249 probes in region: 740\n",
      "--- assemblying 266 probes in region: 741\n",
      "--- assemblying 353 probes in region: 742\n",
      "--- assemblying 309 probes in region: 743\n",
      "--- assemblying 308 probes in region: 744\n",
      "--- assemblying 367 probes in region: 745\n",
      "--- assemblying 254 probes in region: 748\n",
      "--- assemblying 199 probes in region: 749\n",
      "--- assemblying 149 probes in region: 75\n",
      "--- assemblying 287 probes in region: 750\n",
      "--- assemblying 293 probes in region: 751\n",
      "--- assemblying 307 probes in region: 752\n",
      "--- assemblying 145 probes in region: 753\n",
      "--- assemblying 344 probes in region: 754\n",
      "--- assemblying 59 probes in region: 755\n",
      "--- assemblying 319 probes in region: 756\n",
      "--- assemblying 192 probes in region: 757\n",
      "--- assemblying 141 probes in region: 758\n",
      "--- assemblying 306 probes in region: 759\n",
      "--- assemblying 121 probes in region: 76\n",
      "--- assemblying 220 probes in region: 760\n",
      "--- assemblying 222 probes in region: 761\n",
      "--- assemblying 141 probes in region: 762\n",
      "--- assemblying 270 probes in region: 763\n",
      "--- assemblying 261 probes in region: 764\n",
      "--- assemblying 191 probes in region: 765\n",
      "--- assemblying 254 probes in region: 766\n",
      "--- assemblying 199 probes in region: 767\n",
      "--- assemblying 278 probes in region: 768\n",
      "--- assemblying 160 probes in region: 769\n",
      "--- assemblying 218 probes in region: 77\n",
      "--- assemblying 81 probes in region: 770\n",
      "--- assemblying 202 probes in region: 774\n",
      "--- assemblying 202 probes in region: 775\n",
      "--- assemblying 317 probes in region: 776\n",
      "--- assemblying 373 probes in region: 777\n",
      "--- assemblying 252 probes in region: 778\n",
      "--- assemblying 208 probes in region: 779\n",
      "--- assemblying 142 probes in region: 78\n",
      "--- assemblying 349 probes in region: 780\n",
      "--- assemblying 206 probes in region: 782\n",
      "--- assemblying 286 probes in region: 783\n",
      "--- assemblying 196 probes in region: 784\n",
      "--- assemblying 295 probes in region: 785\n",
      "--- assemblying 234 probes in region: 786\n",
      "--- assemblying 293 probes in region: 787\n",
      "--- assemblying 182 probes in region: 788\n",
      "--- assemblying 209 probes in region: 789\n",
      "--- assemblying 198 probes in region: 79\n",
      "--- assemblying 91 probes in region: 790\n",
      "--- assemblying 164 probes in region: 791\n",
      "--- assemblying 168 probes in region: 792\n",
      "--- assemblying 231 probes in region: 793\n",
      "--- assemblying 298 probes in region: 794\n",
      "--- assemblying 126 probes in region: 795\n",
      "--- assemblying 289 probes in region: 796\n",
      "--- assemblying 249 probes in region: 797\n",
      "--- assemblying 244 probes in region: 798\n",
      "--- assemblying 289 probes in region: 799\n",
      "--- assemblying 323 probes in region: 8\n",
      "--- assemblying 247 probes in region: 80\n",
      "--- assemblying 196 probes in region: 800\n",
      "--- assemblying 320 probes in region: 801\n",
      "--- assemblying 257 probes in region: 802\n",
      "--- assemblying 289 probes in region: 803\n",
      "--- assemblying 220 probes in region: 804\n",
      "--- assemblying 168 probes in region: 805\n",
      "--- assemblying 291 probes in region: 806\n",
      "--- assemblying 233 probes in region: 807\n",
      "--- assemblying 197 probes in region: 808\n",
      "--- assemblying 400 probes in region: 809\n",
      "--- assemblying 223 probes in region: 81\n",
      "--- assemblying 160 probes in region: 810\n",
      "--- assemblying 234 probes in region: 811\n",
      "--- assemblying 358 probes in region: 812\n",
      "--- assemblying 187 probes in region: 813\n",
      "--- assemblying 275 probes in region: 814\n",
      "--- assemblying 332 probes in region: 815\n",
      "--- assemblying 209 probes in region: 816\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 204 probes in region: 817\n",
      "--- assemblying 268 probes in region: 819\n",
      "--- assemblying 255 probes in region: 820\n",
      "--- assemblying 142 probes in region: 821\n",
      "--- assemblying 257 probes in region: 822\n",
      "--- assemblying 129 probes in region: 823\n",
      "--- assemblying 288 probes in region: 824\n",
      "--- assemblying 225 probes in region: 825\n",
      "--- assemblying 263 probes in region: 826\n",
      "--- assemblying 283 probes in region: 827\n",
      "--- assemblying 377 probes in region: 828\n",
      "--- assemblying 237 probes in region: 829\n",
      "--- assemblying 259 probes in region: 83\n",
      "--- assemblying 168 probes in region: 832\n",
      "--- assemblying 204 probes in region: 833\n",
      "--- assemblying 310 probes in region: 834\n",
      "--- assemblying 236 probes in region: 835\n",
      "--- assemblying 164 probes in region: 836\n",
      "--- assemblying 302 probes in region: 837\n",
      "--- assemblying 226 probes in region: 838\n",
      "--- assemblying 194 probes in region: 839\n",
      "--- assemblying 191 probes in region: 84\n",
      "--- assemblying 264 probes in region: 840\n",
      "--- assemblying 373 probes in region: 841\n",
      "--- assemblying 148 probes in region: 843\n",
      "--- assemblying 256 probes in region: 844\n",
      "--- assemblying 300 probes in region: 845\n",
      "--- assemblying 261 probes in region: 846\n",
      "--- assemblying 174 probes in region: 847\n",
      "--- assemblying 294 probes in region: 848\n",
      "--- assemblying 123 probes in region: 849\n",
      "--- assemblying 313 probes in region: 85\n",
      "--- assemblying 181 probes in region: 850\n",
      "--- assemblying 241 probes in region: 851\n",
      "--- assemblying 279 probes in region: 852\n",
      "--- assemblying 194 probes in region: 853\n",
      "--- assemblying 215 probes in region: 854\n",
      "--- assemblying 199 probes in region: 855\n",
      "--- assemblying 163 probes in region: 856\n",
      "--- assemblying 300 probes in region: 857\n",
      "--- assemblying 252 probes in region: 858\n",
      "--- assemblying 240 probes in region: 859\n",
      "--- assemblying 237 probes in region: 86\n",
      "--- assemblying 144 probes in region: 860\n",
      "--- assemblying 333 probes in region: 861\n",
      "--- assemblying 233 probes in region: 862\n",
      "--- assemblying 282 probes in region: 863\n",
      "--- assemblying 250 probes in region: 864\n",
      "--- assemblying 223 probes in region: 865\n",
      "--- assemblying 216 probes in region: 866\n",
      "--- assemblying 164 probes in region: 867\n",
      "--- assemblying 289 probes in region: 868\n",
      "--- assemblying 301 probes in region: 869\n",
      "--- assemblying 236 probes in region: 87\n",
      "--- assemblying 324 probes in region: 870\n",
      "--- assemblying 329 probes in region: 871\n",
      "--- assemblying 360 probes in region: 872\n",
      "--- assemblying 265 probes in region: 873\n",
      "--- assemblying 370 probes in region: 874\n",
      "--- assemblying 212 probes in region: 875\n",
      "--- assemblying 262 probes in region: 876\n",
      "--- assemblying 268 probes in region: 877\n",
      "--- assemblying 347 probes in region: 878\n",
      "--- assemblying 204 probes in region: 88\n",
      "--- assemblying 215 probes in region: 888\n",
      "--- assemblying 279 probes in region: 889\n",
      "--- assemblying 335 probes in region: 89\n",
      "--- assemblying 205 probes in region: 890\n",
      "--- assemblying 169 probes in region: 891\n",
      "--- assemblying 314 probes in region: 892\n",
      "--- assemblying 252 probes in region: 893\n",
      "--- assemblying 308 probes in region: 894\n",
      "--- assemblying 294 probes in region: 895\n",
      "--- assemblying 158 probes in region: 896\n",
      "--- assemblying 286 probes in region: 897\n",
      "--- assemblying 182 probes in region: 898\n",
      "--- assemblying 211 probes in region: 899\n",
      "--- assemblying 264 probes in region: 9\n",
      "--- assemblying 118 probes in region: 90\n",
      "--- assemblying 251 probes in region: 900\n",
      "--- assemblying 230 probes in region: 901\n",
      "--- assemblying 271 probes in region: 902\n",
      "--- assemblying 144 probes in region: 903\n",
      "--- assemblying 320 probes in region: 904\n",
      "--- assemblying 311 probes in region: 905\n",
      "--- assemblying 194 probes in region: 906\n",
      "--- assemblying 143 probes in region: 907\n",
      "--- assemblying 216 probes in region: 908\n",
      "--- assemblying 181 probes in region: 909\n",
      "--- assemblying 363 probes in region: 91\n",
      "--- assemblying 155 probes in region: 910\n",
      "--- assemblying 187 probes in region: 911\n",
      "--- assemblying 230 probes in region: 912\n",
      "--- assemblying 169 probes in region: 913\n",
      "--- assemblying 176 probes in region: 914\n",
      "--- assemblying 166 probes in region: 915\n",
      "--- assemblying 250 probes in region: 916\n",
      "--- assemblying 250 probes in region: 917\n",
      "--- assemblying 259 probes in region: 918\n",
      "--- assemblying 157 probes in region: 919\n",
      "--- assemblying 271 probes in region: 92\n",
      "--- assemblying 187 probes in region: 920\n",
      "--- assemblying 173 probes in region: 921\n",
      "--- assemblying 174 probes in region: 922\n",
      "--- assemblying 141 probes in region: 923\n",
      "--- assemblying 236 probes in region: 924\n",
      "--- assemblying 214 probes in region: 925\n",
      "--- assemblying 293 probes in region: 926\n",
      "--- assemblying 314 probes in region: 927\n",
      "--- assemblying 334 probes in region: 928\n",
      "--- assemblying 261 probes in region: 929\n",
      "--- assemblying 212 probes in region: 93\n",
      "--- assemblying 266 probes in region: 930\n",
      "--- assemblying 249 probes in region: 931\n",
      "--- assemblying 297 probes in region: 932\n",
      "--- assemblying 224 probes in region: 933\n",
      "--- assemblying 209 probes in region: 934\n",
      "--- assemblying 400 probes in region: 935\n",
      "--- assemblying 178 probes in region: 94\n",
      "--- assemblying 160 probes in region: 949\n",
      "--- assemblying 214 probes in region: 95\n",
      "--- assemblying 178 probes in region: 950\n",
      "--- assemblying 188 probes in region: 951\n",
      "--- assemblying 190 probes in region: 952\n",
      "--- assemblying 245 probes in region: 953\n",
      "--- assemblying 90 probes in region: 954\n",
      "--- assemblying 174 probes in region: 955\n",
      "--- assemblying 265 probes in region: 956\n",
      "--- assemblying 258 probes in region: 957\n",
      "--- assemblying 328 probes in region: 958\n",
      "--- assemblying 213 probes in region: 959\n",
      "--- assemblying 258 probes in region: 96\n",
      "--- assemblying 279 probes in region: 960\n",
      "--- assemblying 237 probes in region: 961\n",
      "--- assemblying 226 probes in region: 962\n",
      "--- assemblying 195 probes in region: 963\n",
      "--- assemblying 149 probes in region: 964\n",
      "--- assemblying 191 probes in region: 965\n",
      "--- assemblying 161 probes in region: 966\n",
      "--- assemblying 229 probes in region: 967\n",
      "--- assemblying 228 probes in region: 968\n",
      "--- assemblying 156 probes in region: 969\n",
      "--- assemblying 110 probes in region: 970\n",
      "--- assemblying 173 probes in region: 971\n",
      "--- assemblying 392 probes in region: 972\n",
      "--- assemblying 270 probes in region: 973\n",
      "--- assemblying 209 probes in region: 974\n",
      "--- assemblying 315 probes in region: 975\n",
      "--- assemblying 194 probes in region: 976\n",
      "--- assemblying 297 probes in region: 977\n",
      "--- assemblying 130 probes in region: 978\n",
      "--- assemblying 255 probes in region: 979\n",
      "--- assemblying 168 probes in region: 98\n",
      "--- assemblying 173 probes in region: 980\n",
      "--- assemblying 302 probes in region: 981\n",
      "--- assemblying 171 probes in region: 982\n",
      "--- assemblying 236 probes in region: 983\n",
      "--- assemblying 269 probes in region: 984\n",
      "--- assemblying 342 probes in region: 985\n",
      "--- assemblying 233 probes in region: 986\n",
      "--- assemblying 374 probes in region: 987\n",
      "--- assemblying 348 probes in region: 988\n",
      "--- assemblying 154 probes in region: 989\n",
      "--- assemblying 281 probes in region: 990\n",
      "--- assemblying 247 probes in region: 991\n",
      "--- assemblying 237 probes in region: 992\n",
      "--- assemblying 153 probes in region: 993\n",
      "--- assemblying 286 probes in region: 994\n",
      "--- assemblying 337 probes in region: 995\n",
      "--- assemblying 255 probes in region: 996\n",
      "--- assemblying 304 probes in region: 997\n",
      "--- assemblying 341 probes in region: 999\n",
      "-- 327516 probes assembled in total.\n",
      "-- saving 327516 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "# run assembly\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dict, \n",
    "                                               region_2_readout_dict, readout_dict, primers, \n",
    "                                               primer_len=assemble_params['primer_len'], \n",
    "                                               num_readout_per_probe=assemble_params['num_readout_per_probe'], \n",
    "                                               unique_readout_per_probe=True,\n",
    "                                               rc_targets=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9265b92d",
   "metadata": {},
   "source": [
    "# 4.Check quality"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8dca4823",
   "metadata": {},
   "source": [
    "\n",
    "## 4.1 Load probes\n",
    "\n",
    "Checking procedures could restart from here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 137,
   "id": "a651e7eb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 327516\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# candidate full-length probe filename\n",
    "candidate_basename = 'candidate_probes.fasta' # old version\n",
    "# load full probes\n",
    "full_records = []\n",
    "with open(os.path.join(library_folder, candidate_basename), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fe532250",
   "metadata": {},
   "source": [
    "### Manually input the following info to double check!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 138,
   "id": "cf1d4bf8",
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 4\n",
    "primer_len=20\n",
    "target_len=42\n",
    "min_num_probes = 40\n",
    "# merfish related\n",
    "num_on_bits = 3"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2da410b1",
   "metadata": {},
   "source": [
    "### 4.1.1 check primer usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "id": "3cba3f0b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 327516\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n",
    "print(primer_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "30ce1025",
   "metadata": {},
   "source": [
    "### 4.1.2 check region probe number"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "id": "761720bd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: 0 -> 344\n",
      "gene: 1 -> 319\n",
      "gene: 10 -> 216\n",
      "gene: 100 -> 344\n",
      "gene: 1000 -> 326\n",
      "gene: 1001 -> 341\n",
      "gene: 1002 -> 381\n",
      "gene: 1003 -> 356\n",
      "gene: 1004 -> 283\n",
      "gene: 1005 -> 321\n",
      "gene: 101 -> 286\n",
      "gene: 102 -> 240\n",
      "gene: 1020 -> 52\n",
      "gene: 1023 -> 217\n",
      "gene: 1024 -> 242\n",
      "gene: 1025 -> 306\n",
      "gene: 1029 -> 235\n",
      "gene: 103 -> 205\n",
      "gene: 1030 -> 209\n",
      "gene: 1031 -> 290\n",
      "gene: 1032 -> 275\n",
      "gene: 1033 -> 234\n",
      "gene: 1034 -> 183\n",
      "gene: 1035 -> 201\n",
      "gene: 1036 -> 130\n",
      "gene: 1037 -> 237\n",
      "gene: 1038 -> 227\n",
      "gene: 1039 -> 246\n",
      "gene: 104 -> 119\n",
      "gene: 1040 -> 350\n",
      "gene: 1041 -> 254\n",
      "gene: 1042 -> 325\n",
      "gene: 1043 -> 122\n",
      "gene: 1044 -> 253\n",
      "gene: 1045 -> 347\n",
      "gene: 1046 -> 263\n",
      "gene: 1047 -> 350\n",
      "gene: 1048 -> 296\n",
      "gene: 1049 -> 176\n",
      "gene: 105 -> 352\n",
      "gene: 1050 -> 267\n",
      "gene: 1051 -> 384\n",
      "gene: 1052 -> 135\n",
      "gene: 1053 -> 351\n",
      "gene: 1054 -> 320\n",
      "gene: 1055 -> 268\n",
      "gene: 1056 -> 361\n",
      "gene: 1057 -> 155\n",
      "gene: 1058 -> 131\n",
      "gene: 1059 -> 164\n",
      "gene: 106 -> 244\n",
      "gene: 1060 -> 315\n",
      "gene: 1061 -> 309\n",
      "gene: 1062 -> 240\n",
      "gene: 1063 -> 266\n",
      "gene: 1064 -> 327\n",
      "gene: 1065 -> 318\n",
      "gene: 1066 -> 207\n",
      "gene: 1067 -> 204\n",
      "gene: 1068 -> 147\n",
      "gene: 1069 -> 160\n",
      "gene: 107 -> 305\n",
      "gene: 1070 -> 250\n",
      "gene: 1071 -> 291\n",
      "gene: 1072 -> 259\n",
      "gene: 1073 -> 263\n",
      "gene: 1074 -> 273\n",
      "gene: 1075 -> 256\n",
      "gene: 1076 -> 293\n",
      "gene: 1077 -> 88\n",
      "gene: 1078 -> 241\n",
      "gene: 1079 -> 302\n",
      "gene: 108 -> 213\n",
      "gene: 1080 -> 362\n",
      "gene: 1081 -> 298\n",
      "gene: 1082 -> 263\n",
      "gene: 1083 -> 325\n",
      "gene: 1085 -> 341\n",
      "gene: 1087 -> 336\n",
      "gene: 1089 -> 207\n",
      "gene: 109 -> 225\n",
      "gene: 1090 -> 261\n",
      "gene: 1091 -> 226\n",
      "gene: 1092 -> 355\n",
      "gene: 1093 -> 203\n",
      "gene: 1094 -> 317\n",
      "gene: 1097 -> 230\n",
      "gene: 11 -> 212\n",
      "gene: 110 -> 230\n",
      "gene: 1105 -> 223\n",
      "gene: 1106 -> 189\n",
      "gene: 1107 -> 267\n",
      "gene: 1108 -> 295\n",
      "gene: 1109 -> 184\n",
      "gene: 111 -> 358\n",
      "gene: 1110 -> 336\n",
      "gene: 1111 -> 381\n",
      "gene: 1112 -> 398\n",
      "gene: 1113 -> 266\n",
      "gene: 1114 -> 217\n",
      "gene: 1115 -> 253\n",
      "gene: 1116 -> 184\n",
      "gene: 1117 -> 288\n",
      "gene: 1118 -> 231\n",
      "gene: 1119 -> 328\n",
      "gene: 112 -> 167\n",
      "gene: 1120 -> 228\n",
      "gene: 1121 -> 298\n",
      "gene: 1122 -> 227\n",
      "gene: 1123 -> 267\n",
      "gene: 1124 -> 201\n",
      "gene: 1125 -> 208\n",
      "gene: 1126 -> 320\n",
      "gene: 1127 -> 241\n",
      "gene: 1128 -> 235\n",
      "gene: 1129 -> 335\n",
      "gene: 1130 -> 313\n",
      "gene: 1131 -> 272\n",
      "gene: 1132 -> 290\n",
      "gene: 1133 -> 205\n",
      "gene: 1134 -> 331\n",
      "gene: 1135 -> 268\n",
      "gene: 1136 -> 294\n",
      "gene: 1137 -> 196\n",
      "gene: 1138 -> 222\n",
      "gene: 1139 -> 256\n",
      "gene: 114 -> 215\n",
      "gene: 1140 -> 242\n",
      "gene: 1141 -> 375\n",
      "gene: 1142 -> 263\n",
      "gene: 1143 -> 206\n",
      "gene: 1144 -> 122\n",
      "gene: 1145 -> 215\n",
      "gene: 1146 -> 94\n",
      "gene: 1147 -> 257\n",
      "gene: 1148 -> 101\n",
      "gene: 115 -> 224\n",
      "gene: 1152 -> 194\n",
      "gene: 1153 -> 269\n",
      "gene: 1154 -> 54\n",
      "gene: 1155 -> 254\n",
      "gene: 1156 -> 327\n",
      "gene: 1157 -> 276\n",
      "gene: 116 -> 228\n",
      "gene: 1160 -> 317\n",
      "gene: 1162 -> 222\n",
      "gene: 1163 -> 272\n",
      "gene: 1165 -> 270\n",
      "gene: 1166 -> 342\n",
      "gene: 1167 -> 192\n",
      "gene: 1168 -> 346\n",
      "gene: 1169 -> 185\n",
      "gene: 117 -> 276\n",
      "gene: 1170 -> 232\n",
      "gene: 1171 -> 268\n",
      "gene: 1172 -> 329\n",
      "gene: 1173 -> 96\n",
      "gene: 1174 -> 128\n",
      "gene: 1176 -> 212\n",
      "gene: 1177 -> 241\n",
      "gene: 1178 -> 321\n",
      "gene: 1179 -> 197\n",
      "gene: 118 -> 216\n",
      "gene: 1180 -> 213\n",
      "gene: 1181 -> 271\n",
      "gene: 1182 -> 291\n",
      "gene: 1183 -> 337\n",
      "gene: 1184 -> 310\n",
      "gene: 1186 -> 265\n",
      "gene: 1187 -> 285\n",
      "gene: 1188 -> 77\n",
      "gene: 1189 -> 171\n",
      "gene: 119 -> 221\n",
      "gene: 1190 -> 187\n",
      "gene: 1191 -> 194\n",
      "gene: 1192 -> 242\n",
      "gene: 1193 -> 279\n",
      "gene: 1194 -> 291\n",
      "gene: 1195 -> 288\n",
      "gene: 1196 -> 277\n",
      "gene: 1197 -> 184\n",
      "gene: 12 -> 240\n",
      "gene: 120 -> 200\n",
      "gene: 1203 -> 191\n",
      "gene: 1204 -> 346\n",
      "gene: 1205 -> 321\n",
      "gene: 1206 -> 108\n",
      "gene: 1207 -> 213\n",
      "gene: 1208 -> 208\n",
      "gene: 1209 -> 169\n",
      "gene: 121 -> 295\n",
      "gene: 1210 -> 270\n",
      "gene: 1211 -> 302\n",
      "gene: 1212 -> 278\n",
      "gene: 1213 -> 293\n",
      "gene: 1214 -> 317\n",
      "gene: 1215 -> 221\n",
      "gene: 1216 -> 216\n",
      "gene: 1217 -> 190\n",
      "gene: 1218 -> 235\n",
      "gene: 1219 -> 315\n",
      "gene: 122 -> 270\n",
      "gene: 1220 -> 358\n",
      "gene: 1221 -> 400\n",
      "gene: 1222 -> 179\n",
      "gene: 1224 -> 231\n",
      "gene: 1225 -> 282\n",
      "gene: 1226 -> 238\n",
      "gene: 1227 -> 215\n",
      "gene: 1228 -> 167\n",
      "gene: 1229 -> 235\n",
      "gene: 123 -> 182\n",
      "gene: 1230 -> 223\n",
      "gene: 1231 -> 241\n",
      "gene: 1232 -> 237\n",
      "gene: 1233 -> 223\n",
      "gene: 1234 -> 169\n",
      "gene: 1235 -> 241\n",
      "gene: 1236 -> 271\n",
      "gene: 1237 -> 228\n",
      "gene: 1238 -> 287\n",
      "gene: 1239 -> 249\n",
      "gene: 124 -> 235\n",
      "gene: 1240 -> 297\n",
      "gene: 1241 -> 344\n",
      "gene: 1242 -> 279\n",
      "gene: 1243 -> 297\n",
      "gene: 1244 -> 260\n",
      "gene: 1245 -> 283\n",
      "gene: 1246 -> 305\n",
      "gene: 1247 -> 234\n",
      "gene: 1248 -> 160\n",
      "gene: 1249 -> 237\n",
      "gene: 125 -> 251\n",
      "gene: 1250 -> 310\n",
      "gene: 1251 -> 181\n",
      "gene: 1252 -> 181\n",
      "gene: 1253 -> 246\n",
      "gene: 1254 -> 178\n",
      "gene: 1255 -> 303\n",
      "gene: 1256 -> 186\n",
      "gene: 1257 -> 173\n",
      "gene: 1258 -> 168\n",
      "gene: 1259 -> 216\n",
      "gene: 126 -> 291\n",
      "gene: 1260 -> 364\n",
      "gene: 1261 -> 377\n",
      "gene: 1262 -> 85\n",
      "gene: 1263 -> 189\n",
      "gene: 1264 -> 279\n",
      "gene: 1265 -> 99\n",
      "gene: 1269 -> 51\n",
      "gene: 127 -> 222\n",
      "gene: 1270 -> 305\n",
      "gene: 1271 -> 312\n",
      "gene: 1272 -> 317\n",
      "gene: 1273 -> 297\n",
      "gene: 1274 -> 192\n",
      "gene: 1275 -> 356\n",
      "gene: 1276 -> 244\n",
      "gene: 1277 -> 362\n",
      "gene: 1278 -> 150\n",
      "gene: 1279 -> 193\n",
      "gene: 128 -> 275\n",
      "gene: 1280 -> 296\n",
      "gene: 1281 -> 275\n",
      "gene: 1282 -> 234\n",
      "gene: 1284 -> 188\n",
      "gene: 1285 -> 206\n",
      "gene: 1286 -> 263\n",
      "gene: 1287 -> 287\n",
      "gene: 1288 -> 239\n",
      "gene: 1289 -> 368\n",
      "gene: 129 -> 216\n",
      "gene: 1290 -> 315\n",
      "gene: 1291 -> 273\n",
      "gene: 1292 -> 248\n",
      "gene: 1293 -> 312\n",
      "gene: 1294 -> 90\n",
      "gene: 1295 -> 167\n",
      "gene: 1297 -> 312\n",
      "gene: 1298 -> 282\n",
      "gene: 1299 -> 267\n",
      "gene: 13 -> 341\n",
      "gene: 130 -> 162\n",
      "gene: 1300 -> 266\n",
      "gene: 1301 -> 252\n",
      "gene: 1302 -> 220\n",
      "gene: 1303 -> 174\n",
      "gene: 1304 -> 268\n",
      "gene: 1305 -> 297\n",
      "gene: 1306 -> 285\n",
      "gene: 1307 -> 271\n",
      "gene: 1308 -> 243\n",
      "gene: 1309 -> 325\n",
      "gene: 131 -> 190\n",
      "gene: 1310 -> 285\n",
      "gene: 1311 -> 211\n",
      "gene: 1312 -> 257\n",
      "gene: 1313 -> 201\n",
      "gene: 1314 -> 272\n",
      "gene: 1315 -> 347\n",
      "gene: 1316 -> 301\n",
      "gene: 1317 -> 344\n",
      "gene: 1318 -> 253\n",
      "gene: 1319 -> 385\n",
      "gene: 132 -> 192\n",
      "gene: 1320 -> 286\n",
      "gene: 1321 -> 321\n",
      "gene: 1322 -> 310\n",
      "gene: 1323 -> 306\n",
      "gene: 1324 -> 321\n",
      "gene: 1325 -> 367\n",
      "gene: 133 -> 312\n",
      "gene: 1330 -> 129\n",
      "gene: 1331 -> 242\n",
      "gene: 1332 -> 278\n",
      "gene: 1333 -> 136\n",
      "gene: 1334 -> 161\n",
      "gene: 1335 -> 326\n",
      "gene: 1336 -> 266\n",
      "gene: 1337 -> 339\n",
      "gene: 1338 -> 188\n",
      "gene: 1339 -> 238\n",
      "gene: 134 -> 237\n",
      "gene: 1340 -> 344\n",
      "gene: 1341 -> 308\n",
      "gene: 1342 -> 261\n",
      "gene: 1343 -> 201\n",
      "gene: 1344 -> 274\n",
      "gene: 1345 -> 312\n",
      "gene: 1346 -> 298\n",
      "gene: 1347 -> 179\n",
      "gene: 1348 -> 130\n",
      "gene: 1349 -> 302\n",
      "gene: 1350 -> 265\n",
      "gene: 1351 -> 353\n",
      "gene: 1352 -> 243\n",
      "gene: 1353 -> 192\n",
      "gene: 1354 -> 278\n",
      "gene: 1355 -> 187\n",
      "gene: 1356 -> 291\n",
      "gene: 1357 -> 248\n",
      "gene: 1358 -> 327\n",
      "gene: 1359 -> 267\n",
      "gene: 1360 -> 324\n",
      "gene: 1361 -> 397\n",
      "gene: 1362 -> 385\n",
      "gene: 1363 -> 314\n",
      "gene: 137 -> 315\n",
      "gene: 1378 -> 217\n",
      "gene: 1379 -> 292\n",
      "gene: 138 -> 285\n",
      "gene: 1380 -> 192\n",
      "gene: 1381 -> 214\n",
      "gene: 1382 -> 210\n",
      "gene: 1383 -> 230\n",
      "gene: 1384 -> 148\n",
      "gene: 1385 -> 163\n",
      "gene: 1386 -> 128\n",
      "gene: 1387 -> 190\n",
      "gene: 1388 -> 176\n",
      "gene: 1389 -> 251\n",
      "gene: 139 -> 174\n",
      "gene: 1390 -> 288\n",
      "gene: 1391 -> 233\n",
      "gene: 1392 -> 244\n",
      "gene: 1393 -> 323\n",
      "gene: 1394 -> 174\n",
      "gene: 1395 -> 282\n",
      "gene: 1396 -> 192\n",
      "gene: 1398 -> 184\n",
      "gene: 1399 -> 268\n",
      "gene: 14 -> 176\n",
      "gene: 140 -> 261\n",
      "gene: 1400 -> 308\n",
      "gene: 1401 -> 268\n",
      "gene: 1402 -> 308\n",
      "gene: 1403 -> 223\n",
      "gene: 1404 -> 235\n",
      "gene: 1405 -> 336\n",
      "gene: 1406 -> 328\n",
      "gene: 1408 -> 354\n",
      "gene: 1409 -> 374\n",
      "gene: 141 -> 74\n",
      "gene: 1427 -> 253\n",
      "gene: 1429 -> 266\n",
      "gene: 143 -> 186\n",
      "gene: 1430 -> 240\n",
      "gene: 1434 -> 295\n",
      "gene: 1435 -> 255\n",
      "gene: 1436 -> 298\n",
      "gene: 1437 -> 360\n",
      "gene: 1438 -> 174\n",
      "gene: 1439 -> 198\n",
      "gene: 144 -> 187\n",
      "gene: 1440 -> 236\n",
      "gene: 1441 -> 184\n",
      "gene: 1442 -> 248\n",
      "gene: 1443 -> 316\n",
      "gene: 1444 -> 314\n",
      "gene: 1445 -> 316\n",
      "gene: 1446 -> 274\n",
      "gene: 1447 -> 380\n",
      "gene: 1448 -> 267\n",
      "gene: 1449 -> 343\n",
      "gene: 145 -> 178\n",
      "gene: 1450 -> 211\n",
      "gene: 1451 -> 255\n",
      "gene: 1452 -> 113\n",
      "gene: 1453 -> 304\n",
      "gene: 1454 -> 277\n",
      "gene: 1455 -> 290\n",
      "gene: 1456 -> 270\n",
      "gene: 1457 -> 281\n",
      "gene: 1458 -> 396\n",
      "gene: 1459 -> 326\n",
      "gene: 146 -> 294\n",
      "gene: 1461 -> 143\n",
      "gene: 1462 -> 238\n",
      "gene: 1463 -> 268\n",
      "gene: 1464 -> 281\n",
      "gene: 1465 -> 231\n",
      "gene: 1466 -> 286\n",
      "gene: 1467 -> 193\n",
      "gene: 1468 -> 180\n",
      "gene: 1469 -> 202\n",
      "gene: 147 -> 355\n",
      "gene: 1470 -> 194\n",
      "gene: 1471 -> 221\n",
      "gene: 1472 -> 278\n",
      "gene: 1473 -> 117\n",
      "gene: 1474 -> 173\n",
      "gene: 1475 -> 234\n",
      "gene: 1476 -> 197\n",
      "gene: 1477 -> 341\n",
      "gene: 1478 -> 230\n",
      "gene: 1479 -> 216\n",
      "gene: 148 -> 193\n",
      "gene: 1480 -> 117\n",
      "gene: 1481 -> 198\n",
      "gene: 1482 -> 212\n",
      "gene: 1485 -> 351\n",
      "gene: 1486 -> 227\n",
      "gene: 1487 -> 250\n",
      "gene: 1488 -> 311\n",
      "gene: 1489 -> 94\n",
      "gene: 149 -> 309\n",
      "gene: 1490 -> 128\n",
      "gene: 1491 -> 239\n",
      "gene: 1492 -> 238\n",
      "gene: 1493 -> 129\n",
      "gene: 1494 -> 189\n",
      "gene: 1495 -> 199\n",
      "gene: 1496 -> 72\n",
      "gene: 1497 -> 230\n",
      "gene: 1498 -> 146\n",
      "gene: 1499 -> 160\n",
      "gene: 15 -> 343\n",
      "gene: 150 -> 240\n",
      "gene: 1500 -> 369\n",
      "gene: 1501 -> 253\n",
      "gene: 1502 -> 139\n",
      "gene: 1503 -> 186\n",
      "gene: 1504 -> 285\n",
      "gene: 1505 -> 239\n",
      "gene: 1506 -> 163\n",
      "gene: 1507 -> 242\n",
      "gene: 1508 -> 189\n",
      "gene: 1509 -> 188\n",
      "gene: 1510 -> 176\n",
      "gene: 1511 -> 218\n",
      "gene: 1512 -> 248\n",
      "gene: 1513 -> 245\n",
      "gene: 1514 -> 262\n",
      "gene: 1515 -> 357\n",
      "gene: 1516 -> 103\n",
      "gene: 1517 -> 232\n",
      "gene: 1518 -> 217\n",
      "gene: 1519 -> 239\n",
      "gene: 152 -> 248\n",
      "gene: 1520 -> 221\n",
      "gene: 1521 -> 176\n",
      "gene: 1525 -> 51\n",
      "gene: 1526 -> 60\n",
      "gene: 1527 -> 72\n",
      "gene: 1529 -> 166\n",
      "gene: 153 -> 307\n",
      "gene: 1530 -> 218\n",
      "gene: 1533 -> 69\n",
      "gene: 1534 -> 149\n",
      "gene: 1535 -> 187\n",
      "gene: 1536 -> 187\n",
      "gene: 1537 -> 170\n",
      "gene: 1538 -> 164\n",
      "gene: 154 -> 283\n",
      "gene: 1541 -> 251\n",
      "gene: 1542 -> 204\n",
      "gene: 1548 -> 106\n",
      "gene: 155 -> 252\n",
      "gene: 156 -> 292\n",
      "gene: 157 -> 255\n",
      "gene: 158 -> 227\n",
      "gene: 159 -> 231\n",
      "gene: 16 -> 245\n",
      "gene: 160 -> 136\n",
      "gene: 161 -> 134\n",
      "gene: 162 -> 338\n",
      "gene: 163 -> 234\n",
      "gene: 164 -> 303\n",
      "gene: 165 -> 244\n",
      "gene: 166 -> 304\n",
      "gene: 167 -> 217\n",
      "gene: 168 -> 301\n",
      "gene: 169 -> 238\n",
      "gene: 17 -> 203\n",
      "gene: 170 -> 303\n",
      "gene: 171 -> 272\n",
      "gene: 172 -> 200\n",
      "gene: 173 -> 173\n",
      "gene: 174 -> 198\n",
      "gene: 175 -> 317\n",
      "gene: 176 -> 152\n",
      "gene: 177 -> 279\n",
      "gene: 178 -> 280\n",
      "gene: 179 -> 230\n",
      "gene: 18 -> 204\n",
      "gene: 180 -> 226\n",
      "gene: 181 -> 248\n",
      "gene: 182 -> 93\n",
      "gene: 183 -> 204\n",
      "gene: 184 -> 173\n",
      "gene: 185 -> 313\n",
      "gene: 186 -> 301\n",
      "gene: 187 -> 239\n",
      "gene: 188 -> 233\n",
      "gene: 189 -> 191\n",
      "gene: 19 -> 391\n",
      "gene: 190 -> 150\n",
      "gene: 191 -> 274\n",
      "gene: 192 -> 270\n",
      "gene: 193 -> 357\n",
      "gene: 194 -> 297\n",
      "gene: 195 -> 199\n",
      "gene: 196 -> 317\n",
      "gene: 197 -> 251\n",
      "gene: 198 -> 379\n",
      "gene: 199 -> 275\n",
      "gene: 2 -> 377\n",
      "gene: 20 -> 206\n",
      "gene: 200 -> 229\n",
      "gene: 201 -> 361\n",
      "gene: 202 -> 331\n",
      "gene: 203 -> 190\n",
      "gene: 204 -> 265\n",
      "gene: 205 -> 241\n",
      "gene: 206 -> 262\n",
      "gene: 207 -> 324\n",
      "gene: 208 -> 174\n",
      "gene: 209 -> 180\n",
      "gene: 21 -> 255\n",
      "gene: 210 -> 365\n",
      "gene: 211 -> 349\n",
      "gene: 212 -> 330\n",
      "gene: 213 -> 200\n",
      "gene: 214 -> 219\n",
      "gene: 215 -> 142\n",
      "gene: 216 -> 400\n",
      "gene: 217 -> 278\n",
      "gene: 218 -> 252\n",
      "gene: 219 -> 258\n",
      "gene: 22 -> 303\n",
      "gene: 220 -> 214\n",
      "gene: 221 -> 256\n",
      "gene: 222 -> 292\n",
      "gene: 223 -> 318\n",
      "gene: 224 -> 322\n",
      "gene: 225 -> 234\n",
      "gene: 226 -> 220\n",
      "gene: 227 -> 242\n",
      "gene: 228 -> 148\n",
      "gene: 229 -> 197\n",
      "gene: 23 -> 333\n",
      "gene: 230 -> 166\n",
      "gene: 233 -> 196\n",
      "gene: 234 -> 255\n",
      "gene: 235 -> 233\n",
      "gene: 236 -> 182\n",
      "gene: 237 -> 179\n",
      "gene: 238 -> 251\n",
      "gene: 239 -> 231\n",
      "gene: 24 -> 279\n",
      "gene: 240 -> 234\n",
      "gene: 241 -> 272\n",
      "gene: 242 -> 288\n",
      "gene: 243 -> 165\n",
      "gene: 244 -> 333\n",
      "gene: 245 -> 238\n",
      "gene: 246 -> 205\n",
      "gene: 247 -> 234\n",
      "gene: 248 -> 277\n",
      "gene: 249 -> 242\n",
      "gene: 25 -> 238\n",
      "gene: 250 -> 172\n",
      "gene: 251 -> 357\n",
      "gene: 252 -> 287\n",
      "gene: 253 -> 188\n",
      "gene: 254 -> 282\n",
      "gene: 255 -> 225\n",
      "gene: 256 -> 218\n",
      "gene: 257 -> 288\n",
      "gene: 258 -> 314\n",
      "gene: 259 -> 160\n",
      "gene: 26 -> 328\n",
      "gene: 260 -> 162\n",
      "gene: 261 -> 159\n",
      "gene: 262 -> 179\n",
      "gene: 263 -> 324\n",
      "gene: 264 -> 259\n",
      "gene: 265 -> 220\n",
      "gene: 266 -> 251\n",
      "gene: 267 -> 263\n",
      "gene: 268 -> 175\n",
      "gene: 269 -> 351\n",
      "gene: 27 -> 149\n",
      "gene: 270 -> 313\n",
      "gene: 271 -> 220\n",
      "gene: 272 -> 232\n",
      "gene: 273 -> 231\n",
      "gene: 274 -> 330\n",
      "gene: 275 -> 390\n",
      "gene: 276 -> 325\n",
      "gene: 277 -> 313\n",
      "gene: 278 -> 224\n",
      "gene: 279 -> 210\n",
      "gene: 28 -> 262\n",
      "gene: 280 -> 309\n",
      "gene: 281 -> 239\n",
      "gene: 282 -> 262\n",
      "gene: 283 -> 293\n",
      "gene: 284 -> 215\n",
      "gene: 285 -> 234\n",
      "gene: 286 -> 187\n",
      "gene: 287 -> 194\n",
      "gene: 288 -> 249\n",
      "gene: 289 -> 250\n",
      "gene: 29 -> 199\n",
      "gene: 290 -> 253\n",
      "gene: 291 -> 195\n",
      "gene: 292 -> 226\n",
      "gene: 293 -> 157\n",
      "gene: 295 -> 209\n",
      "gene: 296 -> 274\n",
      "gene: 297 -> 194\n",
      "gene: 298 -> 253\n",
      "gene: 299 -> 141\n",
      "gene: 3 -> 242\n",
      "gene: 30 -> 263\n",
      "gene: 300 -> 151\n",
      "gene: 302 -> 328\n",
      "gene: 303 -> 213\n",
      "gene: 304 -> 232\n",
      "gene: 305 -> 222\n",
      "gene: 306 -> 242\n",
      "gene: 307 -> 231\n",
      "gene: 308 -> 203\n",
      "gene: 309 -> 343\n",
      "gene: 31 -> 215\n",
      "gene: 310 -> 166\n",
      "gene: 311 -> 267\n",
      "gene: 312 -> 241\n",
      "gene: 313 -> 182\n",
      "gene: 314 -> 253\n",
      "gene: 315 -> 228\n",
      "gene: 316 -> 183\n",
      "gene: 317 -> 251\n",
      "gene: 318 -> 178\n",
      "gene: 319 -> 249\n",
      "gene: 32 -> 190\n",
      "gene: 320 -> 155\n",
      "gene: 321 -> 279\n",
      "gene: 322 -> 258\n",
      "gene: 323 -> 248\n",
      "gene: 324 -> 171\n",
      "gene: 325 -> 176\n",
      "gene: 326 -> 191\n",
      "gene: 327 -> 106\n",
      "gene: 328 -> 170\n",
      "gene: 329 -> 295\n",
      "gene: 33 -> 338\n",
      "gene: 330 -> 218\n",
      "gene: 331 -> 154\n",
      "gene: 332 -> 377\n",
      "gene: 333 -> 214\n",
      "gene: 334 -> 234\n",
      "gene: 335 -> 271\n",
      "gene: 336 -> 244\n",
      "gene: 337 -> 181\n",
      "gene: 338 -> 222\n",
      "gene: 339 -> 257\n",
      "gene: 34 -> 230\n",
      "gene: 340 -> 201\n",
      "gene: 341 -> 284\n",
      "gene: 342 -> 157\n",
      "gene: 343 -> 260\n",
      "gene: 344 -> 234\n",
      "gene: 345 -> 225\n",
      "gene: 346 -> 279\n",
      "gene: 347 -> 202\n",
      "gene: 348 -> 223\n",
      "gene: 349 -> 161\n",
      "gene: 35 -> 203\n",
      "gene: 350 -> 400\n",
      "gene: 351 -> 319\n",
      "gene: 352 -> 325\n",
      "gene: 353 -> 158\n",
      "gene: 354 -> 293\n",
      "gene: 355 -> 234\n",
      "gene: 356 -> 257\n",
      "gene: 357 -> 170\n",
      "gene: 358 -> 216\n",
      "gene: 359 -> 179\n",
      "gene: 36 -> 154\n",
      "gene: 360 -> 190\n",
      "gene: 361 -> 175\n",
      "gene: 362 -> 315\n",
      "gene: 363 -> 350\n",
      "gene: 364 -> 253\n",
      "gene: 365 -> 284\n",
      "gene: 366 -> 214\n",
      "gene: 367 -> 102\n",
      "gene: 368 -> 64\n",
      "gene: 37 -> 358\n",
      "gene: 370 -> 158\n",
      "gene: 371 -> 84\n",
      "gene: 372 -> 236\n",
      "gene: 373 -> 270\n",
      "gene: 374 -> 224\n",
      "gene: 375 -> 226\n",
      "gene: 376 -> 222\n",
      "gene: 377 -> 340\n",
      "gene: 379 -> 287\n",
      "gene: 38 -> 211\n",
      "gene: 380 -> 224\n",
      "gene: 381 -> 278\n",
      "gene: 382 -> 318\n",
      "gene: 383 -> 176\n",
      "gene: 384 -> 166\n",
      "gene: 385 -> 272\n",
      "gene: 386 -> 215\n",
      "gene: 387 -> 192\n",
      "gene: 388 -> 277\n",
      "gene: 389 -> 170\n",
      "gene: 39 -> 237\n",
      "gene: 390 -> 164\n",
      "gene: 391 -> 257\n",
      "gene: 392 -> 202\n",
      "gene: 393 -> 229\n",
      "gene: 394 -> 309\n",
      "gene: 395 -> 256\n",
      "gene: 396 -> 182\n",
      "gene: 397 -> 245\n",
      "gene: 398 -> 162\n",
      "gene: 399 -> 196\n",
      "gene: 4 -> 256\n",
      "gene: 40 -> 293\n",
      "gene: 400 -> 222\n",
      "gene: 401 -> 237\n",
      "gene: 402 -> 191\n",
      "gene: 403 -> 377\n",
      "gene: 404 -> 331\n",
      "gene: 405 -> 205\n",
      "gene: 406 -> 261\n",
      "gene: 407 -> 306\n",
      "gene: 408 -> 163\n",
      "gene: 409 -> 221\n",
      "gene: 410 -> 271\n",
      "gene: 411 -> 282\n",
      "gene: 412 -> 212\n",
      "gene: 413 -> 339\n",
      "gene: 414 -> 188\n",
      "gene: 415 -> 252\n",
      "gene: 416 -> 252\n",
      "gene: 417 -> 315\n",
      "gene: 418 -> 200\n",
      "gene: 419 -> 355\n",
      "gene: 42 -> 201\n",
      "gene: 421 -> 212\n",
      "gene: 422 -> 298\n",
      "gene: 423 -> 345\n",
      "gene: 424 -> 323\n",
      "gene: 425 -> 221\n",
      "gene: 426 -> 350\n",
      "gene: 427 -> 217\n",
      "gene: 428 -> 243\n",
      "gene: 429 -> 193\n",
      "gene: 43 -> 235\n",
      "gene: 430 -> 236\n",
      "gene: 431 -> 237\n",
      "gene: 432 -> 270\n",
      "gene: 435 -> 357\n",
      "gene: 436 -> 294\n",
      "gene: 437 -> 188\n",
      "gene: 438 -> 206\n",
      "gene: 439 -> 310\n",
      "gene: 44 -> 317\n",
      "gene: 440 -> 321\n",
      "gene: 441 -> 183\n",
      "gene: 442 -> 276\n",
      "gene: 443 -> 241\n",
      "gene: 444 -> 307\n",
      "gene: 446 -> 108\n",
      "gene: 447 -> 181\n",
      "gene: 448 -> 127\n",
      "gene: 449 -> 228\n",
      "gene: 45 -> 187\n",
      "gene: 450 -> 268\n",
      "gene: 451 -> 258\n",
      "gene: 452 -> 197\n",
      "gene: 453 -> 67\n",
      "gene: 454 -> 145\n",
      "gene: 455 -> 142\n",
      "gene: 456 -> 217\n",
      "gene: 457 -> 207\n",
      "gene: 458 -> 236\n",
      "gene: 459 -> 180\n",
      "gene: 46 -> 309\n",
      "gene: 460 -> 202\n",
      "gene: 461 -> 209\n",
      "gene: 462 -> 170\n",
      "gene: 463 -> 143\n",
      "gene: 464 -> 211\n",
      "gene: 465 -> 352\n",
      "gene: 466 -> 189\n",
      "gene: 467 -> 200\n",
      "gene: 468 -> 222\n",
      "gene: 469 -> 260\n",
      "gene: 47 -> 213\n",
      "gene: 470 -> 131\n",
      "gene: 471 -> 199\n",
      "gene: 472 -> 203\n",
      "gene: 473 -> 203\n",
      "gene: 474 -> 235\n",
      "gene: 475 -> 263\n",
      "gene: 476 -> 236\n",
      "gene: 477 -> 215\n",
      "gene: 478 -> 189\n",
      "gene: 479 -> 287\n",
      "gene: 480 -> 257\n",
      "gene: 481 -> 269\n",
      "gene: 482 -> 230\n",
      "gene: 483 -> 220\n",
      "gene: 484 -> 265\n",
      "gene: 485 -> 255\n",
      "gene: 486 -> 294\n",
      "gene: 487 -> 342\n",
      "gene: 489 -> 216\n",
      "gene: 490 -> 384\n",
      "gene: 491 -> 276\n",
      "gene: 492 -> 229\n",
      "gene: 493 -> 239\n",
      "gene: 494 -> 293\n",
      "gene: 495 -> 259\n",
      "gene: 496 -> 167\n",
      "gene: 497 -> 228\n",
      "gene: 498 -> 262\n",
      "gene: 499 -> 162\n",
      "gene: 5 -> 303\n",
      "gene: 500 -> 142\n",
      "gene: 501 -> 304\n",
      "gene: 502 -> 269\n",
      "gene: 503 -> 318\n",
      "gene: 504 -> 220\n",
      "gene: 505 -> 308\n",
      "gene: 506 -> 306\n",
      "gene: 507 -> 202\n",
      "gene: 508 -> 221\n",
      "gene: 509 -> 272\n",
      "gene: 510 -> 336\n",
      "gene: 511 -> 184\n",
      "gene: 512 -> 197\n",
      "gene: 516 -> 227\n",
      "gene: 517 -> 262\n",
      "gene: 518 -> 264\n",
      "gene: 519 -> 188\n",
      "gene: 520 -> 212\n",
      "gene: 521 -> 265\n",
      "gene: 522 -> 187\n",
      "gene: 523 -> 152\n",
      "gene: 524 -> 196\n",
      "gene: 525 -> 216\n",
      "gene: 526 -> 223\n",
      "gene: 527 -> 255\n",
      "gene: 528 -> 249\n",
      "gene: 529 -> 269\n",
      "gene: 530 -> 223\n",
      "gene: 531 -> 201\n",
      "gene: 532 -> 232\n",
      "gene: 533 -> 201\n",
      "gene: 534 -> 181\n",
      "gene: 535 -> 116\n",
      "gene: 536 -> 259\n",
      "gene: 537 -> 231\n",
      "gene: 538 -> 214\n",
      "gene: 539 -> 242\n",
      "gene: 540 -> 179\n",
      "gene: 541 -> 248\n",
      "gene: 542 -> 227\n",
      "gene: 543 -> 302\n",
      "gene: 544 -> 276\n",
      "gene: 545 -> 267\n",
      "gene: 546 -> 305\n",
      "gene: 548 -> 191\n",
      "gene: 549 -> 206\n",
      "gene: 550 -> 378\n",
      "gene: 551 -> 90\n",
      "gene: 552 -> 264\n",
      "gene: 554 -> 306\n",
      "gene: 555 -> 241\n",
      "gene: 556 -> 234\n",
      "gene: 557 -> 273\n",
      "gene: 558 -> 251\n",
      "gene: 559 -> 221\n",
      "gene: 560 -> 261\n",
      "gene: 561 -> 240\n",
      "gene: 562 -> 249\n",
      "gene: 563 -> 193\n",
      "gene: 564 -> 270\n",
      "gene: 565 -> 218\n",
      "gene: 566 -> 169\n",
      "gene: 567 -> 229\n",
      "gene: 568 -> 222\n",
      "gene: 569 -> 292\n",
      "gene: 57 -> 60\n",
      "gene: 570 -> 104\n",
      "gene: 571 -> 356\n",
      "gene: 572 -> 280\n",
      "gene: 573 -> 206\n",
      "gene: 574 -> 258\n",
      "gene: 575 -> 204\n",
      "gene: 576 -> 291\n",
      "gene: 577 -> 207\n",
      "gene: 578 -> 246\n",
      "gene: 579 -> 151\n",
      "gene: 580 -> 202\n",
      "gene: 581 -> 250\n",
      "gene: 582 -> 199\n",
      "gene: 583 -> 181\n",
      "gene: 584 -> 139\n",
      "gene: 585 -> 241\n",
      "gene: 586 -> 218\n",
      "gene: 587 -> 156\n",
      "gene: 588 -> 177\n",
      "gene: 589 -> 275\n",
      "gene: 590 -> 225\n",
      "gene: 591 -> 283\n",
      "gene: 592 -> 144\n",
      "gene: 593 -> 306\n",
      "gene: 594 -> 155\n",
      "gene: 595 -> 180\n",
      "gene: 596 -> 254\n",
      "gene: 597 -> 277\n",
      "gene: 598 -> 226\n",
      "gene: 599 -> 304\n",
      "gene: 6 -> 300\n",
      "gene: 60 -> 187\n",
      "gene: 600 -> 294\n",
      "gene: 601 -> 299\n",
      "gene: 602 -> 261\n",
      "gene: 603 -> 251\n",
      "gene: 605 -> 187\n",
      "gene: 606 -> 293\n",
      "gene: 607 -> 281\n",
      "gene: 608 -> 355\n",
      "gene: 609 -> 323\n",
      "gene: 611 -> 302\n",
      "gene: 612 -> 311\n",
      "gene: 613 -> 200\n",
      "gene: 614 -> 277\n",
      "gene: 615 -> 258\n",
      "gene: 616 -> 173\n",
      "gene: 617 -> 254\n",
      "gene: 618 -> 192\n",
      "gene: 619 -> 282\n",
      "gene: 62 -> 330\n",
      "gene: 620 -> 196\n",
      "gene: 621 -> 248\n",
      "gene: 622 -> 247\n",
      "gene: 623 -> 185\n",
      "gene: 624 -> 132\n",
      "gene: 625 -> 168\n",
      "gene: 626 -> 118\n",
      "gene: 627 -> 253\n",
      "gene: 628 -> 273\n",
      "gene: 629 -> 360\n",
      "gene: 63 -> 241\n",
      "gene: 64 -> 199\n",
      "gene: 645 -> 257\n",
      "gene: 646 -> 241\n",
      "gene: 647 -> 219\n",
      "gene: 648 -> 215\n",
      "gene: 649 -> 262\n",
      "gene: 65 -> 114\n",
      "gene: 650 -> 241\n",
      "gene: 651 -> 146\n",
      "gene: 652 -> 306\n",
      "gene: 653 -> 373\n",
      "gene: 654 -> 271\n",
      "gene: 655 -> 400\n",
      "gene: 656 -> 292\n",
      "gene: 657 -> 186\n",
      "gene: 658 -> 311\n",
      "gene: 659 -> 54\n",
      "gene: 66 -> 127\n",
      "gene: 660 -> 278\n",
      "gene: 662 -> 184\n",
      "gene: 663 -> 205\n",
      "gene: 664 -> 255\n",
      "gene: 665 -> 266\n",
      "gene: 666 -> 273\n",
      "gene: 667 -> 172\n",
      "gene: 668 -> 328\n",
      "gene: 669 -> 312\n",
      "gene: 67 -> 271\n",
      "gene: 670 -> 311\n",
      "gene: 671 -> 199\n",
      "gene: 672 -> 171\n",
      "gene: 673 -> 120\n",
      "gene: 674 -> 258\n",
      "gene: 675 -> 327\n",
      "gene: 676 -> 330\n",
      "gene: 677 -> 308\n",
      "gene: 678 -> 347\n",
      "gene: 679 -> 320\n",
      "gene: 68 -> 226\n",
      "gene: 680 -> 294\n",
      "gene: 681 -> 308\n",
      "gene: 682 -> 147\n",
      "gene: 683 -> 285\n",
      "gene: 684 -> 253\n",
      "gene: 685 -> 328\n",
      "gene: 686 -> 266\n",
      "gene: 687 -> 78\n",
      "gene: 688 -> 278\n",
      "gene: 689 -> 160\n",
      "gene: 69 -> 128\n",
      "gene: 690 -> 289\n",
      "gene: 691 -> 270\n",
      "gene: 692 -> 189\n",
      "gene: 693 -> 246\n",
      "gene: 694 -> 236\n",
      "gene: 695 -> 246\n",
      "gene: 696 -> 170\n",
      "gene: 697 -> 220\n",
      "gene: 698 -> 227\n",
      "gene: 699 -> 66\n",
      "gene: 7 -> 341\n",
      "gene: 70 -> 120\n",
      "gene: 701 -> 289\n",
      "gene: 702 -> 291\n",
      "gene: 703 -> 181\n",
      "gene: 704 -> 284\n",
      "gene: 705 -> 292\n",
      "gene: 706 -> 246\n",
      "gene: 707 -> 182\n",
      "gene: 708 -> 213\n",
      "gene: 709 -> 289\n",
      "gene: 71 -> 137\n",
      "gene: 710 -> 234\n",
      "gene: 711 -> 290\n",
      "gene: 712 -> 201\n",
      "gene: 713 -> 236\n",
      "gene: 714 -> 196\n",
      "gene: 715 -> 353\n",
      "gene: 716 -> 136\n",
      "gene: 717 -> 282\n",
      "gene: 718 -> 391\n",
      "gene: 719 -> 376\n",
      "gene: 72 -> 267\n",
      "gene: 720 -> 218\n",
      "gene: 721 -> 270\n",
      "gene: 722 -> 309\n",
      "gene: 723 -> 73\n",
      "gene: 724 -> 288\n",
      "gene: 725 -> 271\n",
      "gene: 726 -> 209\n",
      "gene: 727 -> 138\n",
      "gene: 728 -> 365\n",
      "gene: 729 -> 200\n",
      "gene: 73 -> 198\n",
      "gene: 730 -> 379\n",
      "gene: 731 -> 174\n",
      "gene: 732 -> 296\n",
      "gene: 733 -> 274\n",
      "gene: 734 -> 225\n",
      "gene: 735 -> 206\n",
      "gene: 736 -> 362\n",
      "gene: 737 -> 252\n",
      "gene: 738 -> 222\n",
      "gene: 739 -> 229\n",
      "gene: 74 -> 173\n",
      "gene: 740 -> 249\n",
      "gene: 741 -> 266\n",
      "gene: 742 -> 353\n",
      "gene: 743 -> 309\n",
      "gene: 744 -> 308\n",
      "gene: 745 -> 367\n",
      "gene: 748 -> 254\n",
      "gene: 749 -> 199\n",
      "gene: 75 -> 149\n",
      "gene: 750 -> 287\n",
      "gene: 751 -> 293\n",
      "gene: 752 -> 307\n",
      "gene: 753 -> 145\n",
      "gene: 754 -> 344\n",
      "gene: 755 -> 59\n",
      "gene: 756 -> 319\n",
      "gene: 757 -> 192\n",
      "gene: 758 -> 141\n",
      "gene: 759 -> 306\n",
      "gene: 76 -> 121\n",
      "gene: 760 -> 220\n",
      "gene: 761 -> 222\n",
      "gene: 762 -> 141\n",
      "gene: 763 -> 270\n",
      "gene: 764 -> 261\n",
      "gene: 765 -> 191\n",
      "gene: 766 -> 254\n",
      "gene: 767 -> 199\n",
      "gene: 768 -> 278\n",
      "gene: 769 -> 160\n",
      "gene: 77 -> 218\n",
      "gene: 770 -> 81\n",
      "gene: 774 -> 202\n",
      "gene: 775 -> 202\n",
      "gene: 776 -> 317\n",
      "gene: 777 -> 373\n",
      "gene: 778 -> 252\n",
      "gene: 779 -> 208\n",
      "gene: 78 -> 142\n",
      "gene: 780 -> 349\n",
      "gene: 782 -> 206\n",
      "gene: 783 -> 286\n",
      "gene: 784 -> 196\n",
      "gene: 785 -> 295\n",
      "gene: 786 -> 234\n",
      "gene: 787 -> 293\n",
      "gene: 788 -> 182\n",
      "gene: 789 -> 209\n",
      "gene: 79 -> 198\n",
      "gene: 790 -> 91\n",
      "gene: 791 -> 164\n",
      "gene: 792 -> 168\n",
      "gene: 793 -> 231\n",
      "gene: 794 -> 298\n",
      "gene: 795 -> 126\n",
      "gene: 796 -> 289\n",
      "gene: 797 -> 249\n",
      "gene: 798 -> 244\n",
      "gene: 799 -> 289\n",
      "gene: 8 -> 323\n",
      "gene: 80 -> 247\n",
      "gene: 800 -> 196\n",
      "gene: 801 -> 320\n",
      "gene: 802 -> 257\n",
      "gene: 803 -> 289\n",
      "gene: 804 -> 220\n",
      "gene: 805 -> 168\n",
      "gene: 806 -> 291\n",
      "gene: 807 -> 233\n",
      "gene: 808 -> 197\n",
      "gene: 809 -> 400\n",
      "gene: 81 -> 223\n",
      "gene: 810 -> 160\n",
      "gene: 811 -> 234\n",
      "gene: 812 -> 358\n",
      "gene: 813 -> 187\n",
      "gene: 814 -> 275\n",
      "gene: 815 -> 332\n",
      "gene: 816 -> 209\n",
      "gene: 817 -> 204\n",
      "gene: 819 -> 268\n",
      "gene: 820 -> 255\n",
      "gene: 821 -> 142\n",
      "gene: 822 -> 257\n",
      "gene: 823 -> 129\n",
      "gene: 824 -> 288\n",
      "gene: 825 -> 225\n",
      "gene: 826 -> 263\n",
      "gene: 827 -> 283\n",
      "gene: 828 -> 377\n",
      "gene: 829 -> 237\n",
      "gene: 83 -> 259\n",
      "gene: 832 -> 168\n",
      "gene: 833 -> 204\n",
      "gene: 834 -> 310\n",
      "gene: 835 -> 236\n",
      "gene: 836 -> 164\n",
      "gene: 837 -> 302\n",
      "gene: 838 -> 226\n",
      "gene: 839 -> 194\n",
      "gene: 84 -> 191\n",
      "gene: 840 -> 264\n",
      "gene: 841 -> 373\n",
      "gene: 843 -> 148\n",
      "gene: 844 -> 256\n",
      "gene: 845 -> 300\n",
      "gene: 846 -> 261\n",
      "gene: 847 -> 174\n",
      "gene: 848 -> 294\n",
      "gene: 849 -> 123\n",
      "gene: 85 -> 313\n",
      "gene: 850 -> 181\n",
      "gene: 851 -> 241\n",
      "gene: 852 -> 279\n",
      "gene: 853 -> 194\n",
      "gene: 854 -> 215\n",
      "gene: 855 -> 199\n",
      "gene: 856 -> 163\n",
      "gene: 857 -> 300\n",
      "gene: 858 -> 252\n",
      "gene: 859 -> 240\n",
      "gene: 86 -> 237\n",
      "gene: 860 -> 144\n",
      "gene: 861 -> 333\n",
      "gene: 862 -> 233\n",
      "gene: 863 -> 282\n",
      "gene: 864 -> 250\n",
      "gene: 865 -> 223\n",
      "gene: 866 -> 216\n",
      "gene: 867 -> 164\n",
      "gene: 868 -> 289\n",
      "gene: 869 -> 301\n",
      "gene: 87 -> 236\n",
      "gene: 870 -> 324\n",
      "gene: 871 -> 329\n",
      "gene: 872 -> 360\n",
      "gene: 873 -> 265\n",
      "gene: 874 -> 370\n",
      "gene: 875 -> 212\n",
      "gene: 876 -> 262\n",
      "gene: 877 -> 268\n",
      "gene: 878 -> 347\n",
      "gene: 88 -> 204\n",
      "gene: 888 -> 215\n",
      "gene: 889 -> 279\n",
      "gene: 89 -> 335\n",
      "gene: 890 -> 205\n",
      "gene: 891 -> 169\n",
      "gene: 892 -> 314\n",
      "gene: 893 -> 252\n",
      "gene: 894 -> 308\n",
      "gene: 895 -> 294\n",
      "gene: 896 -> 158\n",
      "gene: 897 -> 286\n",
      "gene: 898 -> 182\n",
      "gene: 899 -> 211\n",
      "gene: 9 -> 264\n",
      "gene: 90 -> 118\n",
      "gene: 900 -> 251\n",
      "gene: 901 -> 230\n",
      "gene: 902 -> 271\n",
      "gene: 903 -> 144\n",
      "gene: 904 -> 320\n",
      "gene: 905 -> 311\n",
      "gene: 906 -> 194\n",
      "gene: 907 -> 143\n",
      "gene: 908 -> 216\n",
      "gene: 909 -> 181\n",
      "gene: 91 -> 363\n",
      "gene: 910 -> 155\n",
      "gene: 911 -> 187\n",
      "gene: 912 -> 230\n",
      "gene: 913 -> 169\n",
      "gene: 914 -> 176\n",
      "gene: 915 -> 166\n",
      "gene: 916 -> 250\n",
      "gene: 917 -> 250\n",
      "gene: 918 -> 259\n",
      "gene: 919 -> 157\n",
      "gene: 92 -> 271\n",
      "gene: 920 -> 187\n",
      "gene: 921 -> 173\n",
      "gene: 922 -> 174\n",
      "gene: 923 -> 141\n",
      "gene: 924 -> 236\n",
      "gene: 925 -> 214\n",
      "gene: 926 -> 293\n",
      "gene: 927 -> 314\n",
      "gene: 928 -> 334\n",
      "gene: 929 -> 261\n",
      "gene: 93 -> 212\n",
      "gene: 930 -> 266\n",
      "gene: 931 -> 249\n",
      "gene: 932 -> 297\n",
      "gene: 933 -> 224\n",
      "gene: 934 -> 209\n",
      "gene: 935 -> 400\n",
      "gene: 94 -> 178\n",
      "gene: 949 -> 160\n",
      "gene: 95 -> 214\n",
      "gene: 950 -> 178\n",
      "gene: 951 -> 188\n",
      "gene: 952 -> 190\n",
      "gene: 953 -> 245\n",
      "gene: 954 -> 90\n",
      "gene: 955 -> 174\n",
      "gene: 956 -> 265\n",
      "gene: 957 -> 258\n",
      "gene: 958 -> 328\n",
      "gene: 959 -> 213\n",
      "gene: 96 -> 258\n",
      "gene: 960 -> 279\n",
      "gene: 961 -> 237\n",
      "gene: 962 -> 226\n",
      "gene: 963 -> 195\n",
      "gene: 964 -> 149\n",
      "gene: 965 -> 191\n",
      "gene: 966 -> 161\n",
      "gene: 967 -> 229\n",
      "gene: 968 -> 228\n",
      "gene: 969 -> 156\n",
      "gene: 970 -> 110\n",
      "gene: 971 -> 173\n",
      "gene: 972 -> 392\n",
      "gene: 973 -> 270\n",
      "gene: 974 -> 209\n",
      "gene: 975 -> 315\n",
      "gene: 976 -> 194\n",
      "gene: 977 -> 297\n",
      "gene: 978 -> 130\n",
      "gene: 979 -> 255\n",
      "gene: 98 -> 168\n",
      "gene: 980 -> 173\n",
      "gene: 981 -> 302\n",
      "gene: 982 -> 171\n",
      "gene: 983 -> 236\n",
      "gene: 984 -> 269\n",
      "gene: 985 -> 342\n",
      "gene: 986 -> 233\n",
      "gene: 987 -> 374\n",
      "gene: 988 -> 348\n",
      "gene: 989 -> 154\n",
      "gene: 990 -> 281\n",
      "gene: 991 -> 247\n",
      "gene: 992 -> 237\n",
      "gene: 993 -> 153\n",
      "gene: 994 -> 286\n",
      "gene: 995 -> 337\n",
      "gene: 996 -> 255\n",
      "gene: 997 -> 304\n",
      "gene: 999 -> 341\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 141,
   "id": "95f60f31",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7t3QuwbmVdP/AHAq8NmiBKokIWXhArkZsgYk4IpZV2mSmIyCFECBLMZMC4qMBRUwfyhtpMDmo1CiiDoGmjIAQomnkBFVPwpAehIUTHYDy2//Os3Pt/9mHvs9daZz3rty6fd4ZxhPWu33o+v9/77u9+9nvZZmFhYSG5ESBAgAABAgQIEJiJwDYC8Ew6bZkECBAgQIAAAQKVgABsEAgQIECAAAECBGYlIADPqt0WS4AAAQIECBAgIACbAQIECBAgQIAAgVkJCMCzarfFEiBAgAABAgQICMBmgAABAgQIECBAYFYCAvCs2m2xBAgQIECAAAECArAZIECAAAECBAgQmJWAADyrdlssAQIECBAgQICAAGwGCBAgQIAAAQIEZiUgAM+q3RZLgAABAgQIECAgAJsBAgQIECBAgACBWQkIwLNqt8USIECAAAECBAgIwGaAAAECBAgQIEBgVgIC8KzabbEECBAgQIAAAQICsBkgQIAAAQIECBCYlYAAPKt2WywBAgQIECBAgIAAbAYIECBAgAABAgRmJSAAz6rdFkuAAAECBAgQICAAmwECBAgQIECAAIFZCQjAs2q3xRIgQIAAAQIECAjAZoAAAQIECBAgQGBWAgLwrNptsQQIECBAgAABAgKwGSBAgAABAgQIEJiVgAA8q3ZbLAECBAgQIECAgABsBggQIECAAAECBGYlIADPqt0WS4AAAQIECBAgIACbAQIECBAgQIAAgVkJCMCzarfFEiBAgAABAgQICMBmgAABAgQIECBAYFYCAvCs2m2xBAgQIECAAAECArAZIECAAAECBAgQmJWAADyrdlssAQIECBAgQICAAGwGCBAgQIAAAQIEZiUgAM+q3RZLgAABAgQIECAgAJsBAgQIECBAgACBWQkIwLNqt8USIECAAAECBAgIwGaAAAECBAgQIEBgVgIC8KzabbEECBAgQIAAAQICsBkgQIAAAQIECBCYlYAAPKt2WywBAgQIECBAgIAAbAYIECBAgAABAgRmJSAAz6rdFkuAAAECBAgQICAAmwECBAgQIECAAIFZCQjAs2q3xRIgQIAAAQIECAjAZoAAAQIECBAgQGBWAgLwrNptsQQIECBAgAABAgKwGSBAgAABAgQIEJiVgAA8q3ZbLAECBAgQIECAgABsBggQIECAAAECBGYlIADPqt0WS4AAAQIECBAgIACbAQIECBAgQIAAgVkJCMCzarfFEiBAgAABAgQICMBmgAABAgQIECBAYFYCAvCs2m2xBAgQIECAAAECArAZIECAAAECBAgQmJWAADyrdlssAQIECBAgQICAAGwGCBAgQIAAAQIEZiUgAM+q3RZLgAABAgQIECAgAJsBAgQIECBAgACBWQkIwLNqt8USIECAAAECBAgIwAVn4N57701f+tKX0iMf+ci03XbbFazk1AQIECBAgACB5QIbN25Md955Z9prr73Sgx70IDybCAjABcfhs5/9bNp3330LVnBqAgQIECBAgMCWBT7zmc+kffbZB5MA3M8M3HrrrWn33XdPefB22WWXfoqqQoAAAQIECBBIKW3YsKHaiPvWt76VdtttNyYCcD8z8J//+Z/psY99bFq/fn3adddd+ymqCgECBAgQIEAgpSSHrD4GXgJR8CFi8AriOjUBAgQIECCwRQE5RAAOeYgYvBB2RQkQIECAAAE7wFucATvABR8iAnBBXKcmQIAAAQIE7AC3nAEBuCVcnbsJwHWUHEOAAAECBAiUEJBDVlcVgEtM3E/PafAK4jo1AQIECBAgYAe45QwIwC3h6txNAK6j5BgCBAgQIECghIAcYge4xFyteU6DtyaRAwgQIECAAIFCAnKIAFxotLZ8WoMXwq4oAQIECBAg4FMgtjgDXgJR8CEiABfEdWoCBAgQIEBgiwJyiB3gkIeIwQthV5QAAQIECBCwA2wHOOpRIABHyatLgAABAgQIyCF2gEMeBQYvhF1RAgQIECBAwA6wHeCoR4EAHCWvLgECBAgQICCH2AEOeRQYvBB2RQkQIECAAAE7wHaAox4FAnCUvLoECBAgQICAHGIHOORRYPBC2BUlQIAAAQIE7ADbAY56FAjAUfLqEiBAgAABAnKIHeCQR4HBC2FXlACBQIHdTv1I4+q3rvvNxvdxBwIE1haQQwTgtaekwBEGrwCqUxIgMGgBAXjQ7XFxMxOQQwTgkJE3eCHsihIgECggAAfiK01gMwE5RAAOeVAYvBB2RQkQCBQQgAPxlSYgANeegW0WFhYWah/twEYCAnAjLgcTIDABAQF4Ak20hMkIyCF2gEOG2eCFsCtKgECggAAciK80ATvAtWfADnBtquYHCsDNzdyDAIFxCwjA4+6fq5+WgBxiBzhkog1eCLuiBAgECgjAgfhKE7ADXHsG7ADXpmp+oADc3Mw9CBAYt4AAPO7+ufppCcghdoBDJtrghbArSoBAoIAAHIivNAE7wLVnwA5wbarmBwrAzc3cgwCBcQsIwOPun6ufloAcYgc4ZKINXgi7ogQIBAoIwIH4ShOwA1x7BuwA16ZqfqAA3NzMPQgQGLeAADzu/rn6aQnIIXaAQyba4IWwK0qAQKCAAByIrzQBO8C1Z8AOcG2q5gcKwM3N3IMAgXELCMDj7p+rn5aAHGIHOGSiDV4Iu6IECAQKCMCB+EoTsANcewbsANeman6gANzczD0IEBi3gAA87v65+mkJyCF2gEMm2uCFsCtKgECggAAciK80ATvAtWfADnBtquYHCsDNzdyDAIFxCwjA4+6fq5+WgBxiBzhkog1eCLuiBAgECgjAgfhKE7ADXHsG7ADXpmp+oADc3Mw9CBAYt4AAPO7+ufppCcghdoBDJtrghbArSoBAoIAAHIivNAE7wLVnwA5wbarmBwrAzc3cgwCBcQsIwOPun6ufloAcYgc4ZKINXgi7ogQIBAoIwIH4ShOwA1x7BuwA16ZqfqAA3NzMPQgQGLeAADzu/rn6aQnIIXaAQyba4IWwK0qAQKCAAByIrzQBO8C1Z8AOcG2q5gcKwM3N3IMAgXELCMDj7p+rn5aAHGIHOGSiDV4Iu6IECAQKCMCB+EoTsANcewbsANeman6gANzczD0IEBi3gAA87v65+mkJyCF2gEMm2uCFsCtKgECggAAciK80ATvAtWfADnBtquYHCsDNzdyDAIFxC/QVgJvWuXXdb44b1tUTaCEgh9gBbjE2W38Xg7f1hs5AgMC4BJoG07y6NuG0aZ02NcYl72oJ3F9ADhGAQx4XBi+EXVECBAIFmgZTATiwWUpPXkAOEYBDhtzghbArSoBAoIAAHIivNIHNBOQQATjkQWHwQtgVJUAgUEAADsRXmoAAXHsGvAmuNlXzAwXg5mbuQYDAuAUE4HH3z9VPS0AOsQMcMtEGL4RdUQIEAgUE4EB8pQnYAa49A7PbAf7Qhz6UzjvvvHTzzTenBz/4wenAAw+s/v8Tn/jEZWhXXHFFOv3006vjdt1113TKKaek448/vjZsPlAAbsTlYAIEJiAgAE+giZYwGQE5xA5wJfCJT3wiHXrooenII49Mf/zHf5zuvvvudNZZZ6V77rknfeUrX0k77LBDddx1112XDj744HTUUUdVx1577bXpzDPPTBdeeGE65phjaj8wDF5tKgcSIDARAQF4Io20jEkIyCECcCWQw+u//Mu/pG9+85tpm222qf7dZz7zmbTffvulvON7+OGHV/8u/+9dd92VbrjhhiW5Y489Nl1++eXVru62225b64Fh8GoxOYgAgQkJCMATaqaljF5ADhGAK4E/+ZM/Sf/2b/+WvvjFLy6JfP3rX69e/vCRj3wk/cZv/Ea67777qp3gdevWpZNPPnnpuKuuuiodcsgh6cYbb0x77713rQeFwavF5CACBCYkIABPqJmWMnoBOUQArgSuvvrq9NznPje96U1vWnoJxEknnZS+8Y1vVMH4gQ98YLrpppvSnnvuma688sp02GGHLcndeeedaeedd04XXXRR9bKIOjeDV0fJMQQITElAAJ5SN61l7AJyiAC8JJBfxvBHf/RH6Qc/+EH1757ylKekj33sY9Ub3fItv973oIMOql4HvP/++y/db+PGjWn77bdP559/fsqheaVbfi1x/mfxtmHDhrTvvvum9evXL51/7A8m10+AAIEtCQjA5oPAcAQEYAG4EvjXf/3X6mUORx99dPqt3/qt9P3vfz+de+656d57762Cb37pw2IAvv7666vXBi/eFgPwBRdckE488cQVRfMb6s4+++z7/TcBeDhPBq6EAIGyAgJwWV9nJ9BEQAAWgCuBZzzjGelxj3tcuuSSS5ZE8ksb8u5v/ii0/FFnW/MSCDvATR6WjiVAYIoCAvAUu2pNYxUQgAXgSuAhD3lIOvXUU9MZZ5yxTGT33Xevdobf+ta3ehPcWB/lrpsAgUEICMCDaIOLIFAJCMACcCXw5Cc/Oe2xxx7pwx/+8JLI7bffXu0Kn3POOekVr3hF9e/zx6DlzwjOrwNevB133HHpsssu8zFonlQIECCwBQEB2HgQGI6AACwAVwJvectbqtfvnnDCCem3f/u3q5CbXwN82223VV+Escsuu1THLX4RRn6t8BFHHFG9LjjvGvsijOE8qF0JAQLDFBCAh9kXVzVPAQFYAK4EFhYW0rve9a70tre9rfros5/92Z+tPqUh7/7utddey5TyF2Ocdtppy74KOQfnJjeD10TLsQQITEFAAJ5CF61hKgJyiAAcMssGL4RdUQIEAgUE4EB8pQlsJiCHCMAhDwqDF8KuKAECgQICcCC+0gQE4NozsM1Cfl2AWxEBAbgIq5MSIDBgAQF4wM1xabMTkEPsAIcMvcELYVeUAIFAAQE4EF9pAnaAa8+AHeDaVM0PFICbm7kHAQLjFhCAx90/Vz8tATnEDnDIRBu8EHZFCRAIFBCAA/GVJmAHuPYM2AGuTdX8QAG4uZl7ECAwbgEBeNz9c/XTEpBD7ACHTLTBC2FXlACBQAEBOBBfaQJ2gGvPgB3g2lTNDxSAm5u5BwEC4xYQgMfdP1c/LQE5xA5wyEQbvBB2RQkQCBQQgAPxlSZgB7j2DNgBrk3V/EABuLmZexAgMG4BAXjc/XP10xKQQ+wAh0y0wQthV5QAgQ4F2gTapuVvXfebTe+Sml5XmxqNL8odCAxMQA4RgENG0uCFsCtKgECHAk2DZpvSbcJp0+tqU6PNWtyHwJAE5BABOGQeDV4Iu6IECHQo0DRotindJpw2va42NdqsxX0IDElADhGAQ+bR4IWwK0qAQIcCTYNmm9JtwmnT62pTo81a3IfAkATkEAE4ZB4NXgi7ogQIdCjQNGi2Kd0mnDa9rj5q5LW3qdPGzH0I1BGQQwTgOnPS+TEGr3NSJyRAoGeBpkGzzeW1CY1Nr6uPGgJwm+67T0kBOUQALjlfq57b4IWwK0qAQIcCTYNmm9J9hNM+agjAbbrvPiUF5BABuOR8CcAhuooSINCHgADcTLlN0G5WwdEE6gsIwAJw/Wnp8EiD1yGmUxEgECIgADdjF4CbeTm6rIAcIgCXnbBVzm7wQtgVJUCgQwEBuBmmANzMy9FlBeQQAbjshAnAIb6KEiBQXkAAbmYsADfzcnRZAQFYAC47YQJwiK+iBAiUFxCAmxkLwM28HF1WQAAWgMtOmAAc4qsoAQLlBQTgZsYCcDMvR5cVEIAF4LITJgCH+CpKgEB5gakE4PJS/1dBAO5LWp06AgKwAFxnTjo/xuB1TuqEBAj0LCAANwMXgJt5ObqsgBwiAJedMDvAIb6KEiBQXkAAbmYsADfzcnRZAQFYAC47YQJwiK+iBAiUFxCAmxkLwM28HF1WQAAWgMtOmAAc4qsoAQLlBQTgZsYCcDMvR5cVEIAF4LITJgCH+CpKgEB5AQG4mbEA3MzL0WUFBGABuOyECcAhvooSIFBeQABuZiwAN/NydFkBAVgALjthAnCIr6IECJQXEICbGQvAzbwcXVZAABaAy06YABziqygBAuUFBOBmxgJwMy9HlxUQgAXgshMmAIf4KkqAQHkBAbiZsQDczMvRZQUEYAG47IQJwCG+ihIgUF5AAG5mLAA383J0WQEBWAAuO2ECcIivogQIlBcQgJsZC8DNvBxdVkAAFoDLTpgAHOKrKAEC5QUE4GbGAnAzL0eXFRCABeCyEyYAh/gqSoBAeYE+AnD5VfRXQQDuz1qltQUEYAF47SkpcITBK4DqlAQI9CogADfjFoCbeTm6rIAcIgCXnTA7wCG+ihIgUF5AAG5mLAA383J0WQEBWAAuO2ECcIivogQIlBcQgJsZC8DNvBxdVkAAFoDLTpgAHOKrKIGpCLQJmX0FrTbXNpW+tFlHX31pc23uMz8BAVgADpl6gxfCriiB0Qm0CZl9Ba021za6BnR4wX31pcNLdqoJC8ghAnDIeBu8EHZFCYxOoE3I7Ctotbm20TWgwwvuqy8dXrJTTVhADhGAQ8bb4IWwK0pgdAJtQmZfQavNtY2uAR1ecF996fCSnWrCAnKIABwy3gYvhF1RAqMTaBMy+wpaba5tdA3o8IL76kuHl+xUExaQQwTgkPE2eCHsihIYnUCbkNlX0GpzbaNrQIcX3FdfOrxkp5qwgBwiAIeMt8ELYVeUwOgE2oTMvoJWm2sbXQM6vOC++tLhJTvVhAXkEAE4ZLwNXgi7ogRGJ9AmZPYVtNpc2+ga0OEF99WXDi/ZqSYsIIcIwCHjbfBC2BUlMDqBNiGzr6DV5tpG14AOL7ivvnR4yU41YQE5RAAOGW+DF8KuKIHRCQiZo2vZqhcsAE+nl1NYiRwiAIfMscELYVeUwOgEBODRtUwAnk7LJr0SOUQADhlwgxfCriiB0QkIwKNrmQA8nZZNeiVyiAAcMuAGL4RdUQKjExCAR9cyAXg6LZv0SuQQAThkwA1eCLuiBEYnIACPrmUC8HRaNumVyCECcMiAG7wQdkUJjE5AAB5dywTg6bRs0iuRQwTgkAE3eCHsihIYnYAAPLqWCcDTadmkVyKHCMAhA27wQtgVJTA6AQF4dC0TgKfTskmvRA4RgEMG3OCFsCtKYHQCAvDoWiYAT6dlk16JHCIAhwy4wQthV5TA6AQE4NG1TACeTssmvRI5RAAOGXCDF8KuKIHRCQjAo2uZADydlk16JXKIABwy4AYvhF1RAqMTEIBH1zIBeDotm/RK5BABOGTADV4Iu6IERicgAI+uZQLwdFo26ZXIIQJwyIAbvBB2RQmECwi04S0Iu4Bb1/1mWG2FCWwuIIcIwCGPCoMXwq4ogXABATi8BWEXIACH0Su8goAcIgCHPDAMXgi7ogTCBQTg8BaEXYAAHEavsADcaAa2WVhYWGh0DwfXFhCAa1M5kMCkBATgSbWz0WIE4EZcDi4sIIfYAS48Yiuf3uCFsCtKIFxAAA5vQdgFCMBh9ArbAW40A3aAG3E1O1gAbublaAJTERCAp9LJ5usQgJubuUc5ATnEDnC56drCmQ1eCLuiBMIFBODwFoRdgAAcRq+wHeBGM2AHuBFXs4MF4GZejiYwFQEBeCqdbL4OAbi5mXuUE5BD7ACXmy47wCG2ihIYsoAAPOTulL02Abisr7M3ExCABeBmE9PR0QavI0inITAyAQF4ZA3r8HIF4A4xnWqrBeQQAXirh6jNCQxeGzX3ITB+AQF4/D1suwIBuK2c+5UQkEME4GUCf/d3f5cuuOCC9LWvfS3tsMMOaf/990+XXXbZ0jFXXHFFOv3009PNN9+cdt1113TKKaek448/vvFsGrzGZO5AYBICAvAk2thqEQJwKzZ3KiQghwjASwJnnXVWevOb31wF3P322y/ddddd6aMf/Wi68MILq2Ouu+66dPDBB6ejjjoqHXnkkenaa69NZ555ZvXfjznmmEYjavAacTmYwGQEBODJtLLxQgTgxmTuUFBADhGAK4G8o7vXXnulvMN76KGHrqhy+OGHV6H4hhtuWPrvxx57bLr88stTHqRtt9229qgavNpUDiQwKQEBeFLtbLQYAbgRl4MLC8ghAnAl8MpXvjJdeuml6etf//qKIvfdd1/1koh169alk08+eemYq666Kh1yyCHpxhtvTHvvvXftcTV4takcSGBSAgLwpNrZaDECcCMuBxcWkEME4Erg2c9+dtppp53Sr/zKr6S//du/TXfffXc64IAD0vnnn1/9u5tuuintueee6corr0yHHXbYktqdd96Zdt5553TRRRdVL4uoezN4daUcR2BaAgLwtPrZZDUCcBMtx5YWkEME4ErgiU98Yvrud7+bHvOYx6Rzzz03PeABD0hnn312uvXWW9Mtt9ySvvKVr6SDDjqoeh1wfmPc4m3jxo1p++23r4LySSedtKrmPffck/I/i7cNGzakfffdN61fv756M50bAQLzEBCA59HnlVYpAM+390NcuQAsAFcCv/RLv5S+8Y1vpC9/+cvVTm++5ZC6++67p1e/+tXpwAMPrALw9ddfX71BbvMAnD854sQTT1xVM7/BLgfqzW8C8BCfFlwTgXICAnA526GfWQAeeofmdX0CsABcCeRQe9ttt6Xbb799mUh++cMv//IvV68R3pqXQNgBntcTi9USWE1AAJ7vbAjA8+39EFcuAAvAlcDRRx9dfeTZ5gE4h9+nP/3p6R3veIc3wQ3xEeyaCIxMQAAeWcM6vFwBuENMp9pqAQFYAK4EPvjBD6bf//3fT1/60pfSU5/61Orffec730m/8Au/kF73utell73sZSl/DFp+c1x+HfDi7bjjjqu+KMPHoG31Y9EJCMxCQACeRZtXXKQAPN/eD3HlArAAXAn85Cc/qd6U9oMf/CC99rWvrd4El1/7e8cdd1TfCvfQhz506Ysw8m7xEUccUX0RxhlnnOGLMIb4yHZNBAYqIAAPtDE9XJYA3AOyErUFBGABeEkgh938Gb8f+chH0o9//OPqo9HyN8PlT4hYvOUvyjjttNOWfRXyCSecUHvgFg80eI3J3IHAJAQE4Em0sdUiBOBWbO5USEAOEYALjdaWT2vwQtgVJRAuIACHtyDsAgTgMHqFVxCQQwTgkAeGwQthV5RAuIAAHN6CsAsQgMPoFRaAG83ANgsLCwuN7uHg2gICcG0qBxKYlIAAPKl2NlpMmwDcdF7a1Gi0CAdPRkAOsQMcMswGL4RdUQLhAk0DTfgFu4DOBNqE06bz0qZGZwt0olEJyCECcMjAGrwQdkUJhAs0DTThF+wCOhNoE06bzkubGp0t0IlGJSCHCMAhA2vwQtgVJRAu0DTQhF+wC+hMoE04bTovbWp0tkAnGpWAHCIAhwyswQthV5RAuEDTQBN+wS6gM4E24bTpvLSp0dkCnWhUAnKIABwysAYvhF1RAuECTQNN+AW7gM4E2oTTpvPSpkZnC3SiUQnIIQJwyMAavBB2RQmECzQNNOEX7AI6E2gTTpvOS5sanS3QiUYlIIcIwCEDa/BC2BUlEC7QNNCEX7AL6EygTThtOi9tanS2QCcalYAcIgCHDKzBC2FXlEC4QNNAE37BLqAzgTbhtOm8tKnR2QKdaFQCcogAHDKwBi+EXVEC4QJNA034BbuAzgTahNOm89KmRmcLdKJRCcghAnDIwBq8EHZFCYQLNA004RfsAjoTaBNOm85LmxqdLdCJRiUghwjAIQNr8ELYFSUQLtA00IRfsAvoTKBNOG06L21qdLZAJxqVgBwiAIcMrMELYVeUQLhA00ATfsEuoDOBNuG06by0qdHZAp1oVAJyiAAcMrAGL4RdUQLhAk0DTfgFu4DOBNqE06bz0qZGZwt0olEJyCECcMjAGrwQdkUJhAs0DTThF+wCOhNoE06bzkubGp0t0IlGJSCHCMAhA2vwQtgVJRAu0DTQhF+wC+hMoE04bTovbWp0tkAnGpWAHCIAhwyswQthV5RAuEDTQBN+wS6gM4E24bTpvLSp0dkCnWhUAnKIABwysAYvhF1RAuECTQNN+AW7gM4E2oTTpvPSpkZnC3SiUQnIIQJwyMAavBB2RQmECzQNNOEX7AImLyA0T77FKy5QDhGAQybf4IWwK0ogXEAADm+BC9hMQACe50jIIQJwyOQbvBB2RQmECwjA4S1wAQKwGUgpySECcMgDweCFsCtKIFxAAA5vgQsQgM2AALzFGdhmYWFhwZSUERCAy7g6K4GhCwjAQ+/Q/K7PSyDm1/O8YjnEDnDI5Bu8EHZFCYQLCMDhLXABdoDNgABsBzjqUSAAR8mrSyBWQACO9Vf9/gJ2gOc5FXKIHeCQyTd4IeyKEggXEIDDW+AC7ACbATvAdoCjHgUCcJS8ugS6ExBmu7N0pjgBO8Bx9pGV5RA7wCHzZ/BC2BUl0KmAANwpp5MFCQjAQfDBZeUQAThkBA1eCLuiBDoVEIA75XSyIAEBOAg+uKwcIgCHjKDBC2FXlECnAgJwp5xOFiQgAAfBB5eVQwTgkBE0eCHsihLoVEAA7pTTyYIEBOAg+OCycogAHDKCBi+EXVECnQoIwJ1yOlmQgAAcBB9cVg4RgENG0OCFsCtKoFMBAbhTTicLEhCAg+CDy8ohAnDICBq8EHZFCXQqIAB3yulkQQICcBB8cFk5RAAOGUGDF8KuKIFOBQTgTjmdLEhAAA6CDy4rhwjAISNo8ELYFSXQqYAA3Cmnk41IQGgeUbNWuVQ5RAAOmWKDF8KuKIFOBQTgTjmdbEQCAvCImiUAN27WNgsLCwuN7+UOtQQE4FpMDiIwaAEBeNDtcXEFBQTggrg9nVoOsQPc06gtL2PwQtgVJdCpgADcKaeTjUhAAB5Rs+wAN26WHeDGZPXvIADXt3IkgaEKCMBD7YzrKi0gAJcWLn9+OcQOcPkpW6GCwQthV5RApwICcKecTjYiAQF4RM2yA9y4WXaAG5PVv4MAXN/KkQSGKiAAD7Uzrqu0gABcWrj8+eUQO8Dlp8wOcIixogRKCwjApYWdf6jjJnKDAAAgAElEQVQCAvBQO1P/ugRgAbj+tHR4pMHrENOpCAQJCMBB8MqGCwjA4S3Y6guQQwTgrR6iNicweG3U3IfAsAQE4GH1w9X0JyAA92ddqpIcIgCXmq0tntfghbArSqBTAQG4U04nG5GAADyiZq1yqXKIABwyxQYvhF1RAp0KCMCdcjrZiAQE4BE1SwBu3CyfAtGYrP4dBOD6Vo4kMFQBAXionXFdpQUE4NLC5c8vh9gBLj9lK1QweCHsihLoVEAA7pTTyUYkIACPqFl2gBs3yw5wY7L6dxCA61s5ksBQBQTgoXbGdZUWEIBLC5c/vxxiB7j8lNkBDjFWlEBpAQG4tLDzD1VAAB5qZ+pflwAsANeflg6PNHgdYjoVgSABATgIXtlwAQE4vAVbfQFyiAC81UPU5gQGr42a+xAoJyDMlrN15ukJCMDj76kcIgCHTLHBC2FXlMCqAgKw4SBQX0AArm811CPlEAE4ZDYNXgi7ogQEYDNAoAMBAbgDxOBTyCECcMgIGrwQdkUJCMBmgEAHAgJwB4jBp5BDBOCQETR4IeyKEhCAzQCBDgQE4A4Qg08hhwjAISNo8ELYFSUgAJsBAh0ICMAdIAafQg4RgENG0OCFsCtKQAA2AwQ6EBCAO0AMPoUcIgCHjKDBC2FXlIAAbAYIdCAgAHeAGHwKOUQADhlBgxfCrigBAdgMEOhAQADuADH4FHKIABwyggYvhF1RAgKwGSDQgYAA3AFi8CnkEAE4ZAQNXgi7ogQEYDNAoAMBAbgDxOBTyCECcMgIGrwQdkUJCMBmgEAHAgJwB4jBp5BDBOCQETR4IeyKEhCAzQCBDgQE4A4Qg08hhwjAISNo8ELYFSUgAJsBAh0ICMAdIAafQg4RgENG0OCFsCtKQAA2AwQ6EBCAO0AMPoUcIgCHjKDBC2FXlIAAbAYIdCAgAHeAGHwKOUQADhlBgxfCrigBAdgMEOhAQADuADH4FHKIABwyggYvhF1RAgKwGSDQgYAA3AFi8CnkEAE4ZAQNXgi7ogQEYDNAoAMBAbgDxOBTyCECcMgIGrwQdkUJCMBmgEAHAgJwB4jBp5BDBOAVBX74wx+mJz3pSek73/lO+uxnP5ue8YxnLB13xRVXpNNPPz3dfPPNadddd02nnHJKOv744xuNssFrxOVgAsUFdjv1I8VrKEBgKgIC8Pg7KYcIwCsKvPKVr0zvec970ve+971lAfi6665LBx98cDrqqKPSkUcema699tp05plnpgsvvDAdc8wxtR8RBq82lQMJ9CIgAPfCrMhEBATg8TdSDhGA7yfw1a9+tdrxfeMb35iOO+64ZQH48MMPT3fddVe64YYblu537LHHpssvvzzlYdp2221rPSoMXi0mBxHoTUAA7o1aoRkKCMzDa7ocIgDfT+DQQw9NT3va09Lzn//89JznPGcpAN93331phx12SOvWrUsnn3zy0v2uuuqqdMghh6Qbb7wx7b333rWm3ODVYnIQgd4EBODeqBWaoYAAPLymyyEC8DKBD37wg+mEE05It9xyS/r85z+/LADfdNNNac8990xXXnllOuyww5bud+edd6add945XXTRRdXLIurcDF4dJccQ6E9AAO7PWqX5CQjAw+u5HCIALwn86Ec/qt74dtZZZ6UXv/jF6VOf+tSyAJxf73vQQQel/Drg/ffff+l+GzduTNtvv306//zz00knnbSi6D333JPyP4u3DRs2pH333TetX7++eiOdGwECsQICcKy/6tMWEICH118BWABeEjjttNPSxz/+8er1vfm1vKsF4Ouvvz7tt99+9wvAF1xwQTrxxBNXFM2h+uyzz77ffxOAh/ek4IrmKSAAz7PvVt2PgADcj3OTKgKwAFwJ3HbbbWmPPfZIl156aXrmM59Z/btrrrkmveAFL0if/OQnqzfFffvb3279Egg7wE0elo4l0L+AANy/uYrzERCAh9drAVgArgQWd3tX48g7vvnNbt4EN7wHsSsi0IWAANyFonMQWFlAAB7eZAjAAnAlcPfdd6cvfOELyzTy/8+f9vCOd7wj7bPPPunpT396yh+Dlo/NrwNevOWPSrvssst8DNrwHt+uiEBtAQG4NpUDCTQWEIAbkxW/gwAsAK8qsPlrgPOBi1+EcfTRR6cjjjii+iKMM844wxdhFH+oKkCgrIAAXNbX2ectIAAPr/8CsADcKADng/NXIec3zG36Vcj5o9Oa3AxeEy3HEigvIACXN1ZhvgIC8PB6L4cIwCFTafBC2BUlsKqAAGw4CJQTEIDL2bY9sxwiALedna26n8HbKj53JtC5gADcOakTElgSEICHNwxyiAAcMpUGL4RdUQJ2gM0AgQABATgAfY2ScogAHDKVBi+EXVECArAZIBAgIAAHoAvArdG3WVhYWGh9b3fcooAAbEAIDEvASyCG1Q9XMy0BAXh4/ZRD7ACHTKXBC2FXlIAdYDNAIEBAAA5AtwPcGt0OcGu6te8oAK9t5AgCfQrYAe5TW625CQjAw+u4HGIHOGQqDV4Iu6IE7ACbAQIBAgJwALod4NbodoBb0619RwF4bSNHEOhTwA5wn9pqzU1AAB5ex+UQO8AhU2nwQtgVJWAH2AwQCBAQgAPQ7QC3RrcD3Jpu7TsKwGsbOYJAnwJ2gPvUVmtuAgLw8Douh9gBDplKgxfCrigBO8BmgECAgAAcgG4HuDW6HeDWdGvfUQBe28gRBPoUsAPcp7ZacxMQgIfXcTnEDnDIVBq8EHZFCdgBNgMEAgQE4AB0O8Ct0e0At6Zb+44C8NpGjiDQp4Ad4D611ZqbgAA8vI7LIXaAQ6bS4IWwK0rADrAZIBAgIAAHoNsBbo1uB7g13dp3FIDXNnIEgT4F7AD3qa3W3AQE4OF1XA6xAxwylQYvhF1RAnaAzQCBAAEBOADdDnBrdDvArenWvqMAvLaRIwj0KWAHuE9tteYmIAAPr+NyiB3gkKk0eCHsihKwA2wGCAQICMAB6HaAW6PbAW5Nt/YdBeC1jRxBoE8BO8B9aqs1NwEBeHgdl0PsAIdMpcELYVeUgB1gM0AgQEAADkC3A9wa3Q5wa7q17ygAr23kCAJ9CtgB7lNbrbkJCMDD67gcYgc4ZCoNXgi7ogTsAJsBAgECAnAAuh3g1uh2gFvTrX1HAXhtI0cQ6FPADnCf2mrNTUAAHl7H5RA7wCFTafBC2BUlYAfYDBAIEBCAA9DtALdGtwPcmm7tOwrAaxs5gkCfAnaA+9RWa24CAvDwOi6H2AEOmUqDF8KuKAE7wGaAQICAAByAbge4Nbod4NZ0a99RAF7byBEE+hSwA9yntlpzExCAh9dxOcQOcMhUGrwQdkUJ2AE2AwQCBATgAHQ7wK3R7QC3plv7jgLw2kaOINCngB3gPrXVmpuAADy8jsshdoBDptLghbArSsAOsBkgECAgAAeg2wFujW4HuDXd2ncUgNc2cgSBPgXsAPeprdbcBATg4XVcDrEDHDKVBi+EXVECdoDNAIEAAQE4AN0OcGt0O8Ct6da+owC8tpEjCPQpYAe4T2215iYgAA+v43KIHeCQqTR4IeyKErADbAYIBAgIwAHodoBbo9sBbk239h0F4LWNHEGgTwE7wH1qqzU3AQF4eB2XQ+wAh0ylwQthV5SAHWAzQCBAQAAOQLcD3BrdDnBrurXvKACvbeQIAn0K2AHuU1utuQkIwMPruBxiBzhkKg1eCLuiBOwAmwECAQICcAC6HeDW6HaAW9OtfUcBeG0jRxDoU8AOcJ/aas1NQAAeXsflEDvAIVNp8ELYFSVgB9gMEAgQaBOA2/xS2qZOAMcgSsohAnDIIBq8EHZFCQjAZoBAgECbYCoAl22UHCIAl52wVc5u8ELYFSUgAJsBAgECAnAA+hol5RABOGQqDV4Iu6IEBGAzQCBAQAAOQBeAW6N7E1xrurXvKACvbeQIAosCffwptE0NHSJAoJ5AXwG43tUsP6rNtbWpM7T7yCF2gENm0uCFsCs6UoE24bTpD7U2NUbK6bIJ9C7Q9PGYL7Cvx2Sba+sdsEBBOUQALjBWa5/S4K1t5AgCdoDNAIFpCLQJmQJw2d7LIQJw2Qlb5ewGL4Rd0ZEKtPlB2PQHbpsaI+V02QR6F2j6eLQDXL5FcogAXH7KVqhg8ELYFR2pQJtw2vQHbpsaI+V02QR6F2j6eBSAy7dIDhGAy0+ZABxirOh0BNqE06Y/cNvUmI6wlRAoK9D08SgAl+1HPrsALACXnzIBOMRY0ekItAmnTX/gtqkxHWErIVBWoOnjUQAu2w8BeMu+Pgat4Pz5zasgrlNPTqBNOG36A7dNjclBWxCBQgJNH48CcKFGbHJaOcQOcPkpswMcYqzodATahNOmP3Db1JiOsJUQKCvQ9PEoAJfthx1gO8DlJ2yVCn7zCqNXeIQCbcJp0x+4bWqMkNIlEwgRaPp4FIDLt0kOsQNcfsrsAIcYKzodgTbhtOkP3DY1piNsJQTKCjR9PArAZfthB9gOcPkJswMcZqzwdATahNOmP3Db1JiOsJUQKCvQ9PEoAJfthwAsAJefMAE4zFjh6Qi0CadNf+C2qTEdYSshUFag6eNRAC7bDwFYAC4/YQJwmLHC0xFoE06b/sBtU2M6wlZCoKxA08ejAFy2HwKwAFx+wgTgMGOFpyPQJpw2/YHbpsZ0hK2EQFmBpo9HAbhsPwRgAbj8hAnAYcYKT0egTTht+gO3TY3pCFsJgbICTR+PAnDZfgjAAnD5CROAw4wVno5Am3Da9AdumxrTEbYSAmUFmj4eBeCy/RCABeDyEyYAhxkrPB2BNuG06Q/cNjWmI2wlBMoKNH08CsBl+yEAC8DlJ0wADjNWeDoCbcJp0x+4bWpMR9hKCMxXoOlzxVSkfBHG6p3cZmFhYWEqjR7aOgze0DrieoYs0CacNv2h1qbGkM1cGwEC9QSaPlfUO+vwj5JDBOCQKTV4IeyKjlSgTTht+kOtTY2RcrpsAgQ2EWj6XDEVPDlEAA6ZZYMXwq7oSAXahNOmP9Ta1Bgpp8smQEAATnKIABzyRGDwQtgVHamAcDrSxrlsAiMQaPrL8giWVOsS5RABuNagdH2Qweta1PmmLCAAT7m71kYgVkAAXp923XXX2CYMrLo3wRVsiABcENepJycgAE+upRZEYDACArAAvPkwCsAFH54CcEFcp56cgAA8uZZaEIHBCAjAArAA3OPDUQDuEVup0QsIwKNvoQUQGKyAACwAC8A9PjwF4B6xlRq9gAA8+hZaAIHBCgjAArAA3OPDUwDuEVup0QsIwKNvoQUQGKyAACwAzzoAf+ADH0jve9/70uc+97l01113pSc84QnppS99aXrJS16Stt122yWbK664Ip1++unp5ptvrt41ecopp6Tjjz++8QNbAG5M5g4zFhCAZ9x8SydQWEAAFoBnHYD333//9PjHPz698IUvTI961KPSJz/5yXTeeeell73sZekNb3hDZXPdddelgw8+OB111FHpyCOPTNdee20688wz04UXXpiOOeaYRg9RAbgRl4NnLiAAz3wALJ9AQQEBWACedQC+88470yMf+chlBnl39+1vf3u6++670wMf+MB0+OGHV7vDN9xww9Jxxx57bLr88surb1TZdKd4rceqALyWkP9O4P8LCMCmgQCBUgICsAA86wC80gProosuqnZ7v/vd76ZHPOIRaYcddkjr1q1LJ5988tLhV111VTrkkEPSjTfemPbee+/aj08BuDaVAwkkAdgQECBQSkAAFoAF4M0E8u7uxRdfnO644470ta99Le25557pyiuvTIcddtjSkXnneOedd045LOeXRdS9CcB1pRxHIAnAhoAAgWICArAALABvIpB3dA844IDqNb6vetWrqtf7HnTQQdXrgPPrhRdvGzduTNtvv306//zz00knnbTqA/See+5J+Z/F24YNG9K+++6b1q83eMWe1Zx4MgJ2gCfTSgshMDgBAVgOEYB/KnD77ben/fbbr/qUh0996lNVwF0MwNdff3313zYPwBdccEE68cQTV31gn3XWWenss8++338XgAf3XOiCBiggAA+wKS6JwEQEBGABWABOKX3/+9+vXtN77733pmuuuSbtuOOOlctNN920VS+BsAM8kWdKywgREIBD2BUlMAsBAVgAnn0AzqH3ec97Xrrllluqlzrkj0VbvN13333eBDeLp0KLHKKAADzErrgmAtMQEIAF4FkH4Pxa3he96EXp6quvrv552tOedr9Hdv4YtPyRaDkcL96OO+64dNlll/kYtGk8D1rFQAUE4IE2xmURmICAACwAzzoA5298e+c735le//rXp2c961nLLJ7ylKdUu7+LX4Rx9NFHpyOOOKJ6XfAZZ5zhizAm8ARoCcMWEICH3R9XR2DMAgKwADzrALzbbrul2267bcXHcP5WuPy64HzLX4V82mmnLfsq5BNOOKHxY9/HoDUmc4cZCwjAM26+pRMoLCAAC8CzDsCFH1/3O70A3Le4emMWEIDH3D3XTmDYAgKwACwA9/gYFYB7xFZq9AIC8OhbaAEEBisgAAvAAnCPD08BuEdspUYvIACPvoUWQGCwAgKwACwA9/jwFIB7xFZq9AIC8OhbaAEEBisgAAvAAnCPD08BuEdspUYvIACPvoUWQGCwAgKwACwA9/jwFIB7xFZq9AIC8OhbaAEEBisgAAvAAnCPD08BuEdspUYvIACPvoUWQGCwAgKwACwA9/jwFIB7xFZq9AIC8OhbaAEEBisgAAvAAnCPD08BuEdspUYvIACPvoUWQGCwAgKwACwA9/jwFIB7xFZq9AIC8OhbaAEEBisgAAvAAnCPD08BuEdspUYvIACPvoUWQGCwAgKwACwA9/jwFIB7xFaqmECbYNrmh02bOsUW7cQECBBoKNDmea9hicaHyyGrk22zsLCw0FjUHWoJGLxaTA4auECbYNrmB0GbOgOnc3kECMxIoM3zXmkeOUQALj1jK57f4IWwK9qxQJtg2uYHQZs6HS/V6QgQINBaoM3zXutiNe8ohwjANUel28MMXreezhYj0CaYtvlB0KZOjIiqBAgQuL9Am+e90o5yiABcesbsAIcIK9qHgGDah7IaBAiMXUAAHlcHvQa4YL/85lUQ16l7ExCAe6NWiACBEQsIwONqngBcsF8CcEFcp+5NQADujVohAgRGLCAAj6t5AnDBfgnABXGdujcBAbg3aoUIEBixgAA8ruYJwAX7JQAXxHXq3gQE4N6oFSJAYMQCAvC4micAF+yXAFwQ16l7ExCAe6NWiACBEQsIwONqngBcsF8CcEFcp+5NQADujVohAgRGLCAAj6t5AnDBfgnABXGdujcBAbg3aoUIEBixgAA8ruYJwAX7JQAXxHXq3gQE4N6oFSJAYMQCAvC4micAF+yXAFwQ16l7ExCAe6NWiACBEQsIwONqngBcsF8CcEFcp+5NQADujVohAgRGLCAAj6t5AnDBfgnABXGdujcBAbg3aoUIEBixgAA8ruYJwAX7JQAXxHXq3gQE4N6oFSJAYMQCAvC4micAF+yXAFwQ16l7ExCAe6NWiACBEQsIwONqngBcsF8CcEFcp+5NQADujVohAgRGLCAAj6t5AnDBfgnABXGdujcBAbg3aoUIEBixgAA8ruYJwAX7JQAXxHXq3gQE4N6oFSJAYMQCAvC4micAF+yXAFwQ16l7ExCAe6NWiACBEQsIwONqngBcsF8CcEFcp+5NQADujVohAgRGLCAAj6t5AnDBfgnABXGdujcBAbg3aoUIEBixgAA8ruYJwAX7JQAXxHXq3gQE4N6oFSJAYMQCAvC4micAF+yXAFwQ16l7ExCAe6NWiACBEQsIwONqngBcsF8CcEFcp+5NQADujVohAgRGLCAAj6t5AnDBfgnABXGdujcBAbg3aoUIEBixgAA8ruYJwAX7JQAXxHXq3gQE4N6oFSJAYMQCAvC4micAF+yXAFwQ16l7ExCAe6NWiACBEQsIwONqngBcsF8CcEHcCZ66TdBs84Tbps4EuS2JAAECnQq0eT7u9AJWOJkcsrqwAFxw+gxeQdwJnrpNMG3zhNumzgS5LYkAAQKdCrR5Pu70AgTgRpwCcCOuZgcLwM285n50m2Da5gm3TZ2598b6CRAgsJZAm+fjtc65tf9dDrEDvLUz1Or+Bq8V22zv1CaYtnnCbVNntk2xcAIECNQUaPN8XPPUrQ+TQwTg1sOzNXc0eFujN7/7tgmmbZ5w29SZXzesmAABAs0E2jwfN6vQ/Gg5RABuPjUd3MPgdYA4o1MIpjNqtqUSIEAgpVQ6NMshAnDIA83ghbCPtqgAPNrWuXACBAi0EhCAW7F1cidvguuEceWTCMAFcSd4agF4gk21JAIECGxBQACOGw8BuKC9AFwQd4KnFoAn2FRLIkCAgAA8yBkQgAu2RQAuiDvBUwvAE2yqJREgQEAAHuQMCMAF2yIAF8Sd4KkF4Ak21ZIIECAgAA9yBgTggm0RgAviTvDUAvAEm2pJBAgQEIAHOQMCcMG2CMAFcSd4agF4gk21JAIECAjAg5wBAbhgWwTggrg/PfWQQ2PTd/cOeS3lO6kCAQIE5ifQ9OdEUyE5ZHUxAbjpNDU43uA1wGp56JBDY9MntiGvpWV73I0AAQIE7AAPcgYE4IJtEYAL4toBLo+rAgECBAgUFWi6UdL0YuQQO8BNZ6aT4w1eJ4xbPMmQd02bPrENeS3lO6kCAQIE5ifQ9OdEUyE5RABuOjOdHG/wOmEUgMszqkCAAAECAQICcAD6T0t6CURBewG4IO5PTz3kXdOmT2xDXkv5TqpAgACB+Qk0/TnRVEgOsQPcdGY6Od7gdcJoB7g8owoECBAgECAgAAeg2wEujy4Alzce8q5p0ye2Ia+lfCdVIECAwPwEmv6caCokh9gBbjoznRxv8DphtANcnlEFAgQIEAgQEIAD0O0Al0cXgMsb2zUtb6wCAQIECJQREIDLuNY5qzfB1VFqeYwA3BKuwd0E4AZYDiVAgACBQQkIwHHtEIAL2gvABXF/emoBuLyxCgQIECBQRkAALuNa56wCcB2llscIwC3hGtxNAG6A5VACBAgQGJSAABzXDgG4oL0AXBDXDnB5XBUIECBAoKiAAFyUd4snF4AL2gvABXEF4PK4KhAgQIBAUQEBuCivABzFKwCXl/cSiPLGKhAgQIBAGQEBuIxrnbPaAa6j1PIYAbglXIO7CcANsBxKgAABAoMSEIDj2iEAF7QXgAvi/vTUAnB5YxUIECBAoIyAAFzGtc5ZBeA6Si2PEYBbwjW4mwDcAMuhBAgQIDAoAQE4rh0CcEF7Abggrh3g8rgqECBAgEBRAQG4KO8WTy4Ar8Lz9a9/PZ100knp05/+dHroQx+a/vAP/zCtW7cuPfjBD67dLQG4NlXrA+0At6ZzRwIECBAIFhCA4xogAK9gf/fdd6enPvWp6fGPf3z667/+63THHXekU045JR122GHpve99b+1uCcC1qVofKAC3pnNHAgQIEAgWEIDjGiAAr2D/ute9Lr361a9Ot912W9ppp52qI97//venI444It10003pyU9+cq2OCcC1mLbqIAF4q/jcmQABAgQCBQTgOHwBeAX7Zz/72enhD394+vCHP7z0X++77770sIc9LJ1zzjnp5S9/ea2OCcC1mLbqIAF4q/jcmQABAgQCBQTgOHwBeAX7nXfeOb34xS+uXvO76W3PPfdMBxxwQHr3u99dq2MCcC2mrTpIAN4qPncmQIAAgUABATgOXwBewX777bdPr3nNa9Kpp5667L8edNBBKYfjSy65ZMWO3XPPPSn/s3hbv359euYzn5k+85nPpF122SWuyxOuvP+5/zLh1VkaAQIECExZ4PrTnlt0eRs2bEj77rtv+ta3vpV22223orXGdnIBeJUA/NrXvja98pWvXPZfDzzwwPToRz86XXzxxSv2+ayzzkpnn3322GbA9RIgQIAAAQITFsgbcfvss8+EV9h8aQLwCmZtXwKx+Q7wvffem/Iu8O67756222675t2pcY/F3+7muMs857Xn0bD+/9vZMPvz++vSnGd/zmv3vNf8eX/jxo3pzjvvTHvttVd60IMeVCNVzOcQAXiFXnf1Jrg+xmjOrzOe89rzbFn/f6bHPvax1S+Zu+66ax8Pt8HU0Hu9n+Pce97zvN/lk7AAvIJm/hi0/Brg/DFoO+64Y3XEP/7jP1ZfhtHkY9C6bNRq55rzD8I5r90Pgnn/IDD7ArAAPL9ffD3vd5uqBOAVPBe/CCO/YHzTL8J43vOe1+iLMLpt1cpnm/MPwjmv3ROhADzX3e+5z77nvfn+8jP32e86UwnAq4jmr0I+8cQT0zXXXJMe8pCHVLu/eWe4yVchd92slc6XX3f8pje9qfqmuh122KGPkoOpMee15yZYv9mf4+N+7rPvcT/fx/3cZ7/r8CEAdy3qfAQIECBAgAABAoMWEIAH3R4XR4AAAQIECBAg0LWAANy1qPMRIECAAAECBAgMWkAAHnR7XBwBAgQIECBAgEDXAgJw16LOR4AAAQIECBAgMGgBAXjQ7Vn94vKnVJx00knp05/+dHroQx9afUrFunXrBvcpFVvL+/d///fpT//0T+93mvw11Xm9i7crrrginX766enmm2+uvhQhvzv++OOP39ryvd7/G9/4Rvqbv/mbdP3116cvf/nL6UlPelL1v5vf6q41n+stb3lLuv3226tvAXrDG96QDjnkkF7X1KRYnfUfffTR6T3vec/9TnvllVemww47bNm/H9P6P/CBD6T3ve996XOf+1y666670hOe8IT00pe+NL3kJS9J2267beM5n9rap9r33NiPfexj6dxzz60+Yz5/wsNjHvOY9Du/8zvpzDPPTA972MMm3fs6a59y7zd/IvvhD39YPe9/5zvfSZ/97GfTM57xjEn3v8nPhxLHCsAlVAufc/Fzih//+Mcv+5ziHFjit3YAAAtJSURBVADe+973Fq7e7+kXA/BHP/rRZT8M8g+J/Dmo+Xbdddelgw8+OB111FHpyCOPTNdee231w+PCCy9MxxxzTL8XvBXVPvzhD6c///M/T/vtt1/Kv+D87//+7/0CcN215gB02mmnVT9Yn/70p6d3vetd6UMf+lD1tcE5DA/xVmf9+Ydh/qUvh8VNb09+8pOXzcfY1r///vun/Hh+4QtfmB71qEelT37yk+m8885LL3vZy6pfXJrM+RTXPtW+577+wz/8Q/riF79Yfa33z/3cz1WP+bPOOqt63P7zP//zpHtfZ+1T7v3mz8N5Yyf/gv+9731vWQCe8vN+5M8iAThSv2Xt/HnEr371q6tvqttpp52qs7z//e9PRxxxxOC+qa7lEpfuthiA83eZL65183Mefvjh1a7ZDTfcsPSfjj322HT55ZdXXxe86Q7a1l5PyfvnwLt4rflJ/8Ybb7xfAK6z1vvuu68KUdng9a9/fXXJP/nJT6rg+7SnPa36VsMh3uqsfzWXTdczxvXn+X7kIx+5rC35rxhvf/vbU/6F94EPfGCaau/rrH2qfV/tcZh/Yc2P37wT+PM///OT7f1K69987XPp/Ve/+tVqx/eNb3xjOu6445YF4Kk+9qN/DgnA0R1oUf/Zz352evjDH57yjtniLf/Qz38uO+ecc9LLX/7yFmcd5l3WCsB53fkLQPLLIU4++eSlRVx11VXVn/tziNx7772HubgtXNVKT/p115p3D3/t134tff7zn0+/+qu/ulTl7LPPrp5cv//976dtttlm0Car/dCr88NwCuvPzbnooouqv2p897vfTY94xCNqzfkU177LLrukOfU99/6SSy5Jv/u7v5tuvfXW9OhHP3pWvd907fmvInPp/aGHHlptUDz/+c9Pz3nOc5YC8Jye9/v+oSQA9y3eQb2dd945vfjFL172Gth82j333DMdcMAB6d3vfncHVYZxisUAnNf8X//1X9Wfif/sz/4s/dVf/VX6mZ/5mWrHO69789eA5l2lfJ8cIvLLIsZ2W+lJv+5a3/a2t6UTTjgh/ehHP1r2mvD8OtM/+IM/SOvXr69eJz3k25YC8D/90z9VO6J5fXlXO39deX7N5OJtCuvPa8k7gBdffHG644470te+9rVacz7FtefHeZ6Hqfc9/5Xmxz/+cfWclp/fH/e4x6XLLrus9nPcmHu/2trz42AOvf/gBz9YPWffcsst1cbFpgF4Ts/7ff9MEoD7Fu+g3vbbb59e85rXpFNPPXXZ2Q466KAq9OXfoKdyy2+SyC9tyK+LzbuW+QdC/rNwfoNQfoNXfr1vXnd+jVR+HeXibePGjSk7nX/++dWbBcd2WykA1l1r/itAno9777132bI/8YlPpF//9V9P//7v/17tNAz5tloAzv3cbrvtqjCYXxqQZyG/TjKH+9/7vd+rljSF9ee/XORfZvNr2V/1qlfVnvMprj33dA59z7+U5pc85Ft+P0cORfkNznN43K+29jn0Pv8in9/4ll/3nX/x+dSnPrUsAM+h/1E/iwTgKPmtqJuD3Wtf+9qUXzC/6e3AAw+s/lyWd42mfHvFK16R3vzmN1c7md/85jerAJw/OSGH5M0D8AUXXJBOPPHE0XFsKQCvtdYcgvJ8/M///M+ydX/84x9P+c9s+Q03Q30j3OIF1/mzZz42v274mc98ZvXu+bxTshiAx7z+/KkdeZZzKMg/DPPjffGH4NR7v9LaV3rwTrHv+XGZPwXgK1/5SvUL7C/+4i+m/JjNPa/zHDfmx/1qa8+7/5vfptb7/Gbl3Oe80ZPfA7JaAJ76Yz/ih7QAHKG+lTXn9BKIlajyx8Pkd0znjwPLL4nwEojlL/dY/FNoDsAPetCDlgin8BKIleYhf0pCfknM4ks+xrz+/Prs/Nr1vHt/zTXXpB133LFactM/g46x96utfbWnyyn1ffM15o/Dy2+Iyo/ZpzzlKbWe48Y895uuf9O1L/5VZ3OfqfQ+v5F9jz32SJdeemn1i3y+5cf9C17wguqTYPIMfPvb355V/7cyHjW6uwDciGsYB8/pTXArieeP8so7ZDkA5zd7eRPc8jf8TeGNUHV3gPN85E+6yH8NWQzAY11/Dr3Pe97zqtcB5pf05F/uFm9TfyPMlta+2rPuVPq+0vrya2Lz69zzXzLym3vrPMeNde43X/+ma9/8ZX6Lx06l94u7vavNeP45l9/QPaf+95myBOA+tTuqlT8GLf+JLP/2uLhDlD/aKn8ZRt4pyp+JOuVb/pSL/JrA/BFn+SUf+SNi8utBc2hYvOWPkcmvFx7Tx6Bt2rMtfQzaWmtd/BiwbLD4ZSH5h0p+3W9+6cNQPwatzvo3n+v859D82u8cfhe/NGSM68+vWX/Ri16Urr766uqflV6jXWfOp7r2qfZ9tefpvAv4rGc9q3rjX37j6lR7v9L6N1/7lHufn8u/8IUvLFti/v/5l553vOMdaZ999qk+D3pO/e8zuwjAfWp3VGvxizB22223ZV+EkXePpvZFGHlNz33uc9NTn/rUSi+H2ne+853pL/7iL6rXAefb4oeE59CYPws5v17yjDPOGN0XYeQQl3e18+2tb31r+o//+I/0pje9qfr/edc/f05s3bUufhlC/jKF/ASaPxkkvzlyyF+Esdb683/PPc6/6OVvSvvv//7v6k1weecrv+49f4nE4m1s68/f+JbnOu9s5eCz6S3/CTzvAE2192utPfd5qn3Pfc6/+OQ/dedfeh784AdXb1LNc5A/yzu/3OsBD3jAZHu/1to3bNgw6d6vFAk2fw1wk59xY3ve6ygStT6NANyaLvaO+ZvC8pu78m/LD3nIQ6pQkHeG8xPolG456OaPOMs7uXm3L79eKn+7W177pp9lm4NjfjPBpl+FnD9WZky3/Jmfu++++4qXnEPe4tcY11nrwsJC9bXK+ZMy8rcK5Z3f/EM1f7zOUG9rrT8HhPy12Pk1gvlj7nIwyMEh/5k0/6K06W1s68+/zOa/6Kx0m3rv11r7lPue+53/SpN3evMvvPk5LnvkYPiXf/mX1S8+i7cpPu7XWnv+gqOpPuZXex5eKQDnY6fY/+ifRQJwdAfUJ0CAAAECBAgQ6FVAAO6VWzECBAgQIECAAIFoAQE4ugPqEyBAgAABAgQI9CogAPfKrRgBAgQIECBAgEC0gAAc3QH1CRAgQIAAAQIEehUQgHvlVowAAQIECBAgQCBaQACO7oD6BAgQIECAAAECvQoIwL1yK0aAAAECBAgQIBAtIABHd0B9AgQIECBAgACBXgUE4F65FSNAgAABAgQIEIgWEICjO6A+AQIECBAgQIBArwICcK/cihEgQIAAAQIECEQLCMDRHVCfAAECBAgQIECgVwEBuFduxQgQIECAAAECBKIFBODoDqhPgAABAgQIECDQq4AA3Cu3YgQIECBAgAABAtECAnB0B9QnQIAAAQIECBDoVUAA7pVbMQIECBAgQIAAgWgBATi6A+oTIECAAAECBAj0KiAA98qtGAECBAgQIECAQLSAABzdAfUJECBAgAABAgR6FRCAe+VWjAABAgQIECBAIFpAAI7ugPoECBAgQIAAAQK9CgjAvXIrRoAAAQIECBAgEC0gAEd3QH0CBAgQIECAAIFeBQTgXrkVI0CAAAECBAgQiBYQgKM7oD4BAgQIECBAgECvAgJwr9yKESBAgAABAgQIRAsIwNEdUJ8AAQIECBAgQKBXAQG4V27FCBAgQIAAAQIEogUE4OgOqE+AAAECBAgQINCrgADcK7diBAgQIECAAAEC0QICcHQH1CdAgAABAgQIEOhVQADulVsxAgQIECBAgACBaAEBOLoD6hMgQIAAAQIECPQq8P8AQ7h+v/Vv5pcAAAAASUVORK5CYII=\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.figure()\n",
    "plt.hist(list(reg_size_dic.values()), bins=np.arange(0,400,10))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2db97b4d",
   "metadata": {},
   "source": [
    "### 4.1.3 check readout usage for each gene"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "id": "8d0a6c3d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0 -> ['NDB_683', 'NDB_758', 'NDB_887']\n",
      "1 -> ['NDB_718', 'NDB_631', 'NDB_1066']\n",
      "10 -> ['NDB_844', 'NDB_812', 'NDB_637']\n",
      "100 -> ['NDB_901', 'NDB_863', 'NDB_926']\n",
      "1000 -> ['NDB_784', 'NDB_883', 'NDB_925']\n",
      "1001 -> ['NDB_634', 'NDB_631', 'NDB_872']\n",
      "1002 -> ['NDB_755', 'NDB_698', 'NDB_1033']\n",
      "1003 -> ['NDB_817', 'NDB_746', 'NDB_742']\n",
      "1004 -> ['NDB_683', 'NDB_878', 'NDB_896']\n",
      "1005 -> ['NDB_661', 'NDB_887', 'NDB_818']\n",
      "101 -> ['NDB_725', 'NDB_847', 'NDB_835']\n",
      "102 -> ['NDB_815', 'NDB_866', 'NDB_941']\n",
      "1020 -> ['NDB_971', 'NDB_754', 'NDB_715']\n",
      "1023 -> ['NDB_868', 'NDB_832', 'NDB_929']\n",
      "1024 -> ['NDB_1033', 'NDB_1075', 'NDB_872']\n",
      "1025 -> ['NDB_743', 'NDB_641', 'NDB_958']\n",
      "1029 -> ['NDB_680', 'NDB_896', 'NDB_818']\n",
      "103 -> ['NDB_713', 'NDB_752', 'NDB_862']\n",
      "1030 -> ['NDB_713', 'NDB_815', 'NDB_881']\n",
      "1031 -> ['NDB_730', 'NDB_694', 'NDB_824']\n",
      "1032 -> ['NDB_784', 'NDB_698', 'NDB_631']\n",
      "1033 -> ['NDB_880', 'NDB_752', 'NDB_788']\n",
      "1034 -> ['NDB_875', 'NDB_653', 'NDB_941']\n",
      "1035 -> ['NDB_974', 'NDB_853', 'NDB_934']\n",
      "1036 -> ['NDB_917', 'NDB_838', 'NDB_1066']\n",
      "1037 -> ['NDB_683', 'NDB_716', 'NDB_899']\n",
      "1038 -> ['NDB_835', 'NDB_712', 'NDB_926']\n",
      "1039 -> ['NDB_878', 'NDB_721', 'NDB_910']\n",
      "104 -> ['NDB_865', 'NDB_875', 'NDB_929']\n",
      "1040 -> ['NDB_773', 'NDB_634', 'NDB_742']\n",
      "1041 -> ['NDB_737', 'NDB_706', 'NDB_977']\n",
      "1042 -> ['NDB_790', 'NDB_1033', 'NDB_989']\n",
      "1043 -> ['NDB_901', 'NDB_860', 'NDB_935']\n",
      "1044 -> ['NDB_817', 'NDB_710', 'NDB_778']\n",
      "1045 -> ['NDB_994', 'NDB_637', 'NDB_958']\n",
      "1046 -> ['NDB_826', 'NDB_754', 'NDB_647']\n",
      "1047 -> ['NDB_865', 'NDB_643', 'NDB_905']\n",
      "1048 -> ['NDB_965', 'NDB_698', 'NDB_751']\n",
      "1049 -> ['NDB_880', 'NDB_941', 'NDB_665']\n",
      "105 -> ['NDB_680', 'NDB_758', 'NDB_647']\n",
      "1050 -> ['NDB_917', 'NDB_779', 'NDB_934']\n",
      "1051 -> ['NDB_1075', 'NDB_899', 'NDB_1028']\n",
      "1052 -> ['NDB_652', 'NDB_661', 'NDB_836']\n",
      "1053 -> ['NDB_844', 'NDB_716', 'NDB_872']\n",
      "1054 -> ['NDB_725', 'NDB_746', 'NDB_863']\n",
      "1055 -> ['NDB_755', 'NDB_743', 'NDB_778']\n",
      "1056 -> ['NDB_758', 'NDB_653', 'NDB_641']\n",
      "1057 -> ['NDB_887', 'NDB_889', 'NDB_862']\n",
      "1058 -> ['NDB_631', 'NDB_905', 'NDB_940']\n",
      "1059 -> ['NDB_812', 'NDB_883', 'NDB_734']\n",
      "106 -> ['NDB_755', 'NDB_838', 'NDB_836']\n",
      "1060 -> ['NDB_1027', 'NDB_1060', 'NDB_866']\n",
      "1061 -> ['NDB_760', 'NDB_805', 'NDB_946']\n",
      "1062 -> ['NDB_847', 'NDB_728', 'NDB_925']\n",
      "1063 -> ['NDB_718', 'NDB_994', 'NDB_935']\n",
      "1064 -> ['NDB_790', 'NDB_856', 'NDB_926']\n",
      "1065 -> ['NDB_961', 'NDB_721', 'NDB_929']\n",
      "1066 -> ['NDB_755', 'NDB_1028', 'NDB_818']\n",
      "1067 -> ['NDB_896', 'NDB_665', 'NDB_779']\n",
      "1068 -> ['NDB_868', 'NDB_881', 'NDB_742']\n",
      "1069 -> ['NDB_715', 'NDB_989', 'NDB_706']\n",
      "107 -> ['NDB_728', 'NDB_716', 'NDB_824']\n",
      "1070 -> ['NDB_844', 'NDB_824', 'NDB_940']\n",
      "1071 -> ['NDB_971', 'NDB_832', 'NDB_694']\n",
      "1072 -> ['NDB_815', 'NDB_758', 'NDB_637']\n",
      "1073 -> ['NDB_974', 'NDB_1066', 'NDB_788']\n",
      "1074 -> ['NDB_725', 'NDB_838', 'NDB_742']\n",
      "1075 -> ['NDB_1027', 'NDB_863', 'NDB_653']\n",
      "1076 -> ['NDB_643', 'NDB_779', 'NDB_940']\n",
      "1077 -> ['NDB_760', 'NDB_853', 'NDB_818']\n",
      "1078 -> ['NDB_844', 'NDB_634', 'NDB_896']\n",
      "1079 -> ['NDB_784', 'NDB_889', 'NDB_934']\n",
      "108 -> ['NDB_971', 'NDB_994', 'NDB_940']\n",
      "1080 -> ['NDB_847', 'NDB_661', 'NDB_1028']\n",
      "1081 -> ['NDB_683', 'NDB_754', 'NDB_866']\n",
      "1082 -> ['NDB_965', 'NDB_901', 'NDB_758']\n",
      "1083 -> ['NDB_718', 'NDB_805', 'NDB_832']\n",
      "1085 -> ['NDB_755', 'NDB_865', 'NDB_872']\n",
      "1087 -> ['NDB_875', 'NDB_698', 'NDB_742']\n",
      "1089 -> ['NDB_778', 'NDB_961', 'NDB_812']\n",
      "109 -> ['NDB_878', 'NDB_880', 'NDB_899']\n",
      "1090 -> ['NDB_746', 'NDB_1075', 'NDB_926']\n",
      "1091 -> ['NDB_773', 'NDB_881', 'NDB_958']\n",
      "1092 -> ['NDB_847', 'NDB_860', 'NDB_712']\n",
      "1093 -> ['NDB_887', 'NDB_880', 'NDB_706']\n",
      "1094 -> ['NDB_917', 'NDB_856', 'NDB_835']\n",
      "1097 -> ['NDB_737', 'NDB_838', 'NDB_779']\n",
      "11 -> ['NDB_737', 'NDB_935', 'NDB_925']\n",
      "110 -> ['NDB_683', 'NDB_706', 'NDB_751']\n",
      "1105 -> ['NDB_743', 'NDB_946', 'NDB_899']\n",
      "1106 -> ['NDB_716', 'NDB_863', 'NDB_994']\n",
      "1107 -> ['NDB_683', 'NDB_760', 'NDB_868']\n",
      "1108 -> ['NDB_815', 'NDB_680', 'NDB_1033']\n",
      "1109 -> ['NDB_698', 'NDB_941', 'NDB_925']\n",
      "111 -> ['NDB_844', 'NDB_746', 'NDB_889']\n",
      "1110 -> ['NDB_641', 'NDB_929', 'NDB_910']\n",
      "1111 -> ['NDB_713', 'NDB_1027', 'NDB_758']\n",
      "1112 -> ['NDB_790', 'NDB_860', 'NDB_730']\n",
      "1113 -> ['NDB_832', 'NDB_788', 'NDB_818']\n",
      "1114 -> ['NDB_905', 'NDB_1066', 'NDB_935']\n",
      "1115 -> ['NDB_971', 'NDB_989', 'NDB_751']\n",
      "1116 -> ['NDB_652', 'NDB_883', 'NDB_878']\n",
      "1117 -> ['NDB_817', 'NDB_1060', 'NDB_665']\n",
      "1118 -> ['NDB_710', 'NDB_734', 'NDB_836']\n",
      "1119 -> ['NDB_728', 'NDB_647', 'NDB_637']\n",
      "112 -> ['NDB_743', 'NDB_661', 'NDB_653']\n",
      "1120 -> ['NDB_974', 'NDB_631', 'NDB_694']\n",
      "1121 -> ['NDB_815', 'NDB_715', 'NDB_824']\n",
      "1122 -> ['NDB_826', 'NDB_746', 'NDB_868']\n",
      "1123 -> ['NDB_812', 'NDB_737', 'NDB_752']\n",
      "1124 -> ['NDB_875', 'NDB_946', 'NDB_862']\n",
      "1125 -> ['NDB_784', 'NDB_716', 'NDB_721']\n",
      "1126 -> ['NDB_805', 'NDB_866', 'NDB_977']\n",
      "1127 -> ['NDB_713', 'NDB_706', 'NDB_958']\n",
      "1128 -> ['NDB_634', 'NDB_788', 'NDB_712']\n",
      "1129 -> ['NDB_844', 'NDB_835', 'NDB_994']\n",
      "1130 -> ['NDB_725', 'NDB_653', 'NDB_910']\n",
      "1131 -> ['NDB_1028', 'NDB_940', 'NDB_872']\n",
      "1132 -> ['NDB_755', 'NDB_643', 'NDB_901']\n",
      "1133 -> ['NDB_853', 'NDB_935', 'NDB_889']\n",
      "1134 -> ['NDB_721', 'NDB_706', 'NDB_712']\n",
      "1135 -> ['NDB_905', 'NDB_880', 'NDB_881']\n",
      "1136 -> ['NDB_755', 'NDB_652', 'NDB_847']\n",
      "1137 -> ['NDB_728', 'NDB_899', 'NDB_818']\n",
      "1138 -> ['NDB_826', 'NDB_817', 'NDB_805']\n",
      "1139 -> ['NDB_875', 'NDB_868', 'NDB_788']\n",
      "114 -> ['NDB_631', 'NDB_946', 'NDB_958']\n",
      "1140 -> ['NDB_773', 'NDB_737', 'NDB_887']\n",
      "1141 -> ['NDB_715', 'NDB_838', 'NDB_694']\n",
      "1142 -> ['NDB_965', 'NDB_883', 'NDB_742']\n",
      "1143 -> ['NDB_698', 'NDB_637', 'NDB_934']\n",
      "1144 -> ['NDB_856', 'NDB_752', 'NDB_910']\n",
      "1145 -> ['NDB_680', 'NDB_946', 'NDB_881']\n",
      "1146 -> ['NDB_901', 'NDB_1075', 'NDB_751']\n",
      "1147 -> ['NDB_812', 'NDB_716', 'NDB_665']\n",
      "1148 -> ['NDB_653', 'NDB_836', 'NDB_958']\n",
      "115 -> ['NDB_817', 'NDB_961', 'NDB_868']\n",
      "1152 -> ['NDB_917', 'NDB_778', 'NDB_863']\n",
      "1153 -> ['NDB_865', 'NDB_989', 'NDB_721']\n",
      "1154 -> ['NDB_710', 'NDB_878', 'NDB_889']\n",
      "1155 -> ['NDB_718', 'NDB_961', 'NDB_872']\n",
      "1156 -> ['NDB_844', 'NDB_641', 'NDB_853']\n",
      "1157 -> ['NDB_647', 'NDB_1066', 'NDB_706']\n",
      "116 -> ['NDB_710', 'NDB_773', 'NDB_1027']\n",
      "1160 -> ['NDB_683', 'NDB_1027', 'NDB_835']\n",
      "1162 -> ['NDB_698', 'NDB_977', 'NDB_824']\n",
      "1163 -> ['NDB_847', 'NDB_887', 'NDB_994']\n",
      "1165 -> ['NDB_868', 'NDB_926', 'NDB_910']\n",
      "1166 -> ['NDB_880', 'NDB_832', 'NDB_1028']\n",
      "1167 -> ['NDB_860', 'NDB_1033', 'NDB_925']\n",
      "1168 -> ['NDB_784', 'NDB_758', 'NDB_935']\n",
      "1169 -> ['NDB_971', 'NDB_896', 'NDB_941']\n",
      "117 -> ['NDB_760', 'NDB_925', 'NDB_872']\n",
      "1170 -> ['NDB_790', 'NDB_746', 'NDB_631']\n",
      "1171 -> ['NDB_743', 'NDB_866', 'NDB_862']\n",
      "1172 -> ['NDB_661', 'NDB_1060', 'NDB_730']\n",
      "1173 -> ['NDB_683', 'NDB_694', 'NDB_940']\n",
      "1174 -> ['NDB_725', 'NDB_634', 'NDB_734']\n",
      "1176 -> ['NDB_815', 'NDB_779', 'NDB_929']\n",
      "1177 -> ['NDB_901', 'NDB_754', 'NDB_977']\n",
      "1178 -> ['NDB_713', 'NDB_863', 'NDB_905']\n",
      "1179 -> ['NDB_974', 'NDB_665', 'NDB_824']\n",
      "118 -> ['NDB_754', 'NDB_1033', 'NDB_818']\n",
      "1180 -> ['NDB_643', 'NDB_965', 'NDB_926']\n",
      "1181 -> ['NDB_760', 'NDB_883', 'NDB_752']\n",
      "1182 -> ['NDB_826', 'NDB_958', 'NDB_940']\n",
      "1183 -> ['NDB_899', 'NDB_994', 'NDB_712']\n",
      "1184 -> ['NDB_755', 'NDB_641', 'NDB_637']\n",
      "1186 -> ['NDB_652', 'NDB_773', 'NDB_818']\n",
      "1187 -> ['NDB_715', 'NDB_788', 'NDB_872']\n",
      "1188 -> ['NDB_680', 'NDB_878', 'NDB_805']\n",
      "1189 -> ['NDB_790', 'NDB_961', 'NDB_737']\n",
      "119 -> ['NDB_917', 'NDB_977', 'NDB_836']\n",
      "1190 -> ['NDB_718', 'NDB_728', 'NDB_734']\n",
      "1191 -> ['NDB_752', 'NDB_935', 'NDB_751']\n",
      "1192 -> ['NDB_865', 'NDB_941', 'NDB_836']\n",
      "1193 -> ['NDB_758', 'NDB_1075', 'NDB_721']\n",
      "1194 -> ['NDB_1060', 'NDB_989', 'NDB_824']\n",
      "1195 -> ['NDB_817', 'NDB_844', 'NDB_838']\n",
      "1196 -> ['NDB_847', 'NDB_760', 'NDB_1033']\n",
      "1197 -> ['NDB_778', 'NDB_634', 'NDB_853']\n",
      "12 -> ['NDB_790', 'NDB_788', 'NDB_934']\n",
      "120 -> ['NDB_652', 'NDB_790', 'NDB_832']\n",
      "1203 -> ['NDB_905', 'NDB_862', 'NDB_694']\n",
      "1204 -> ['NDB_661', 'NDB_716', 'NDB_880']\n",
      "1205 -> ['NDB_653', 'NDB_889', 'NDB_926']\n",
      "1206 -> ['NDB_710', 'NDB_815', 'NDB_994']\n",
      "1207 -> ['NDB_755', 'NDB_917', 'NDB_866']\n",
      "1208 -> ['NDB_812', 'NDB_878', 'NDB_881']\n",
      "1209 -> ['NDB_817', 'NDB_743', 'NDB_860']\n",
      "121 -> ['NDB_860', 'NDB_737', 'NDB_1066']\n",
      "1210 -> ['NDB_826', 'NDB_725', 'NDB_652']\n",
      "1211 -> ['NDB_683', 'NDB_680', 'NDB_734']\n",
      "1212 -> ['NDB_746', 'NDB_875', 'NDB_1060']\n",
      "1213 -> ['NDB_961', 'NDB_883', 'NDB_946']\n",
      "1214 -> ['NDB_971', 'NDB_868', 'NDB_863']\n",
      "1215 -> ['NDB_901', 'NDB_896', 'NDB_910']\n",
      "1216 -> ['NDB_641', 'NDB_647', 'NDB_751']\n",
      "1217 -> ['NDB_974', 'NDB_706', 'NDB_742']\n",
      "1218 -> ['NDB_860', 'NDB_805', 'NDB_1028']\n",
      "1219 -> ['NDB_812', 'NDB_754', 'NDB_788']\n",
      "122 -> ['NDB_878', 'NDB_752', 'NDB_779']\n",
      "1220 -> ['NDB_865', 'NDB_965', 'NDB_653']\n",
      "1221 -> ['NDB_758', 'NDB_862', 'NDB_925']\n",
      "1222 -> ['NDB_844', 'NDB_694', 'NDB_929']\n",
      "1224 -> ['NDB_728', 'NDB_835', 'NDB_977']\n",
      "1225 -> ['NDB_790', 'NDB_773', 'NDB_665']\n",
      "1226 -> ['NDB_698', 'NDB_779', 'NDB_836']\n",
      "1227 -> ['NDB_746', 'NDB_1027', 'NDB_940']\n",
      "1228 -> ['NDB_974', 'NDB_1033', 'NDB_730']\n",
      "1229 -> ['NDB_713', 'NDB_856', 'NDB_637']\n",
      "123 -> ['NDB_698', 'NDB_881', 'NDB_926']\n",
      "1230 -> ['NDB_715', 'NDB_958', 'NDB_818']\n",
      "1231 -> ['NDB_887', 'NDB_899', 'NDB_872']\n",
      "1232 -> ['NDB_826', 'NDB_643', 'NDB_1066']\n",
      "1233 -> ['NDB_710', 'NDB_743', 'NDB_712']\n",
      "1234 -> ['NDB_917', 'NDB_737', 'NDB_832']\n",
      "1235 -> ['NDB_661', 'NDB_631', 'NDB_935']\n",
      "1236 -> ['NDB_778', 'NDB_875', 'NDB_934']\n",
      "1237 -> ['NDB_784', 'NDB_971', 'NDB_965']\n",
      "1238 -> ['NDB_1075', 'NDB_730', 'NDB_752']\n",
      "1239 -> ['NDB_718', 'NDB_989', 'NDB_941']\n",
      "124 -> ['NDB_965', 'NDB_863', 'NDB_934']\n",
      "1240 -> ['NDB_634', 'NDB_929', 'NDB_925']\n",
      "1241 -> ['NDB_760', 'NDB_896', 'NDB_1066']\n",
      "1242 -> ['NDB_812', 'NDB_1027', 'NDB_925']\n",
      "1243 -> ['NDB_743', 'NDB_746', 'NDB_883']\n",
      "1244 -> ['NDB_784', 'NDB_901', 'NDB_994']\n",
      "1245 -> ['NDB_634', 'NDB_1075', 'NDB_665']\n",
      "1246 -> ['NDB_856', 'NDB_737', 'NDB_862']\n",
      "1247 -> ['NDB_652', 'NDB_715', 'NDB_752']\n",
      "1248 -> ['NDB_860', 'NDB_1060', 'NDB_946']\n",
      "1249 -> ['NDB_847', 'NDB_721', 'NDB_977']\n",
      "125 -> ['NDB_778', 'NDB_901', 'NDB_856']\n",
      "1250 -> ['NDB_865', 'NDB_710', 'NDB_887']\n",
      "1251 -> ['NDB_817', 'NDB_751', 'NDB_926']\n",
      "1252 -> ['NDB_725', 'NDB_643', 'NDB_631']\n",
      "1253 -> ['NDB_743', 'NDB_680', 'NDB_698']\n",
      "1254 -> ['NDB_878', 'NDB_665', 'NDB_934']\n",
      "1255 -> ['NDB_826', 'NDB_866', 'NDB_910']\n",
      "1256 -> ['NDB_778', 'NDB_889', 'NDB_836']\n",
      "1257 -> ['NDB_961', 'NDB_863', 'NDB_742']\n",
      "1258 -> ['NDB_875', 'NDB_812', 'NDB_1033']\n",
      "1259 -> ['NDB_784', 'NDB_713', 'NDB_844']\n",
      "126 -> ['NDB_974', 'NDB_812', 'NDB_887']\n",
      "1260 -> ['NDB_773', 'NDB_896', 'NDB_835']\n",
      "1261 -> ['NDB_661', 'NDB_634', 'NDB_706']\n",
      "1262 -> ['NDB_974', 'NDB_637', 'NDB_935']\n",
      "1263 -> ['NDB_710', 'NDB_1060', 'NDB_647']\n",
      "1264 -> ['NDB_965', 'NDB_754', 'NDB_805']\n",
      "1265 -> ['NDB_683', 'NDB_994', 'NDB_881']\n",
      "1269 -> ['NDB_865', 'NDB_946', 'NDB_1066']\n",
      "127 -> ['NDB_866', 'NDB_665', 'NDB_694']\n",
      "1270 -> ['NDB_728', 'NDB_853', 'NDB_929']\n",
      "1271 -> ['NDB_760', 'NDB_788', 'NDB_694']\n",
      "1272 -> ['NDB_958', 'NDB_1028', 'NDB_712']\n",
      "1273 -> ['NDB_734', 'NDB_818', 'NDB_940']\n",
      "1274 -> ['NDB_815', 'NDB_901', 'NDB_653']\n",
      "1275 -> ['NDB_868', 'NDB_883', 'NDB_641']\n",
      "1276 -> ['NDB_971', 'NDB_856', 'NDB_899']\n",
      "1277 -> ['NDB_905', 'NDB_989', 'NDB_925']\n",
      "1278 -> ['NDB_716', 'NDB_862', 'NDB_779']\n",
      "1279 -> ['NDB_730', 'NDB_832', 'NDB_941']\n",
      "128 -> ['NDB_946', 'NDB_721', 'NDB_788']\n",
      "1280 -> ['NDB_790', 'NDB_838', 'NDB_647']\n",
      "1281 -> ['NDB_917', 'NDB_1027', 'NDB_880']\n",
      "1282 -> ['NDB_755', 'NDB_1075', 'NDB_824']\n",
      "1284 -> ['NDB_715', 'NDB_758', 'NDB_853']\n",
      "1285 -> ['NDB_1066', 'NDB_712', 'NDB_872']\n",
      "1286 -> ['NDB_718', 'NDB_760', 'NDB_746']\n",
      "1287 -> ['NDB_661', 'NDB_881', 'NDB_934']\n",
      "1288 -> ['NDB_652', 'NDB_961', 'NDB_698']\n",
      "1289 -> ['NDB_805', 'NDB_637', 'NDB_910']\n",
      "129 -> ['NDB_637', 'NDB_751', 'NDB_872']\n",
      "1290 -> ['NDB_826', 'NDB_847', 'NDB_751']\n",
      "1291 -> ['NDB_875', 'NDB_878', 'NDB_887']\n",
      "1292 -> ['NDB_737', 'NDB_863', 'NDB_730']\n",
      "1293 -> ['NDB_754', 'NDB_721', 'NDB_926']\n",
      "1294 -> ['NDB_680', 'NDB_631', 'NDB_977']\n",
      "1295 -> ['NDB_860', 'NDB_734', 'NDB_889']\n",
      "1297 -> ['NDB_725', 'NDB_974', 'NDB_773']\n",
      "1298 -> ['NDB_713', 'NDB_643', 'NDB_778']\n",
      "1299 -> ['NDB_817', 'NDB_752', 'NDB_835']\n",
      "13 -> ['NDB_746', 'NDB_758', 'NDB_818']\n",
      "130 -> ['NDB_835', 'NDB_989', 'NDB_1028']\n",
      "1300 -> ['NDB_826', 'NDB_860', 'NDB_929']\n",
      "1301 -> ['NDB_710', 'NDB_883', 'NDB_862']\n",
      "1302 -> ['NDB_847', 'NDB_634', 'NDB_905']\n",
      "1303 -> ['NDB_746', 'NDB_716', 'NDB_1028']\n",
      "1304 -> ['NDB_965', 'NDB_641', 'NDB_779']\n",
      "1305 -> ['NDB_790', 'NDB_901', 'NDB_752']\n",
      "1306 -> ['NDB_880', 'NDB_946', 'NDB_853']\n",
      "1307 -> ['NDB_974', 'NDB_661', 'NDB_977']\n",
      "1308 -> ['NDB_718', 'NDB_683', 'NDB_910']\n",
      "1309 -> ['NDB_698', 'NDB_653', 'NDB_730']\n",
      "131 -> ['NDB_784', 'NDB_961', 'NDB_746']\n",
      "1310 -> ['NDB_773', 'NDB_863', 'NDB_836']\n",
      "1311 -> ['NDB_832', 'NDB_721', 'NDB_935']\n",
      "1312 -> ['NDB_643', 'NDB_887', 'NDB_742']\n",
      "1313 -> ['NDB_844', 'NDB_878', 'NDB_926']\n",
      "1314 -> ['NDB_961', 'NDB_896', 'NDB_994']\n",
      "1315 -> ['NDB_754', 'NDB_862', 'NDB_824']\n",
      "1316 -> ['NDB_631', 'NDB_647', 'NDB_881']\n",
      "1317 -> ['NDB_917', 'NDB_805', 'NDB_941']\n",
      "1318 -> ['NDB_847', 'NDB_790', 'NDB_1060']\n",
      "1319 -> ['NDB_734', 'NDB_706', 'NDB_872']\n",
      "132 -> ['NDB_725', 'NDB_718', 'NDB_730']\n",
      "1320 -> ['NDB_752', 'NDB_925', 'NDB_934']\n",
      "1321 -> ['NDB_965', 'NDB_838', 'NDB_788']\n",
      "1322 -> ['NDB_865', 'NDB_746', 'NDB_866']\n",
      "1323 -> ['NDB_710', 'NDB_637', 'NDB_694']\n",
      "1324 -> ['NDB_784', 'NDB_634', 'NDB_818']\n",
      "1325 -> ['NDB_718', 'NDB_751', 'NDB_940']\n",
      "133 -> ['NDB_865', 'NDB_715', 'NDB_994']\n",
      "1330 -> ['NDB_743', 'NDB_853', 'NDB_779']\n",
      "1331 -> ['NDB_643', 'NDB_815', 'NDB_935']\n",
      "1332 -> ['NDB_728', 'NDB_856', 'NDB_730']\n",
      "1333 -> ['NDB_817', 'NDB_680', 'NDB_715']\n",
      "1334 -> ['NDB_875', 'NDB_631', 'NDB_899']\n",
      "1335 -> ['NDB_725', 'NDB_758', 'NDB_881']\n",
      "1336 -> ['NDB_755', 'NDB_868', 'NDB_737']\n",
      "1337 -> ['NDB_826', 'NDB_665', 'NDB_712']\n",
      "1338 -> ['NDB_860', 'NDB_989', 'NDB_641']\n",
      "1339 -> ['NDB_778', 'NDB_1033', 'NDB_929']\n",
      "134 -> ['NDB_1075', 'NDB_853', 'NDB_836']\n",
      "1340 -> ['NDB_1027', 'NDB_883', 'NDB_889']\n",
      "1341 -> ['NDB_713', 'NDB_652', 'NDB_760']\n",
      "1342 -> ['NDB_812', 'NDB_1075', 'NDB_958']\n",
      "1343 -> ['NDB_971', 'NDB_835', 'NDB_1066']\n",
      "1344 -> ['NDB_901', 'NDB_868', 'NDB_1028']\n",
      "1345 -> ['NDB_817', 'NDB_716', 'NDB_905']\n",
      "1346 -> ['NDB_815', 'NDB_788', 'NDB_940']\n",
      "1347 -> ['NDB_838', 'NDB_751', 'NDB_818']\n",
      "1348 -> ['NDB_743', 'NDB_694', 'NDB_925']\n",
      "1349 -> ['NDB_784', 'NDB_941', 'NDB_706']\n",
      "1350 -> ['NDB_961', 'NDB_878', 'NDB_637']\n",
      "1351 -> ['NDB_917', 'NDB_653', 'NDB_994']\n",
      "1352 -> ['NDB_844', 'NDB_734', 'NDB_934']\n",
      "1353 -> ['NDB_896', 'NDB_824', 'NDB_872']\n",
      "1354 -> ['NDB_1060', 'NDB_805', 'NDB_926']\n",
      "1355 -> ['NDB_865', 'NDB_683', 'NDB_863']\n",
      "1356 -> ['NDB_778', 'NDB_773', 'NDB_910']\n",
      "1357 -> ['NDB_661', 'NDB_866', 'NDB_647']\n",
      "1358 -> ['NDB_974', 'NDB_698', 'NDB_946']\n",
      "1359 -> ['NDB_760', 'NDB_899', 'NDB_977']\n",
      "1360 -> ['NDB_832', 'NDB_742', 'NDB_712']\n",
      "1361 -> ['NDB_755', 'NDB_856', 'NDB_721']\n",
      "1362 -> ['NDB_680', 'NDB_887', 'NDB_836']\n",
      "1363 -> ['NDB_754', 'NDB_758', 'NDB_880']\n",
      "137 -> ['NDB_760', 'NDB_641', 'NDB_706']\n",
      "1378 -> ['NDB_1066', 'NDB_751', 'NDB_934']\n",
      "1379 -> ['NDB_883', 'NDB_1033', 'NDB_824']\n",
      "138 -> ['NDB_889', 'NDB_742', 'NDB_940']\n",
      "1380 -> ['NDB_826', 'NDB_1075', 'NDB_925']\n",
      "1381 -> ['NDB_728', 'NDB_758', 'NDB_905']\n",
      "1382 -> ['NDB_760', 'NDB_878', 'NDB_994']\n",
      "1383 -> ['NDB_917', 'NDB_860', 'NDB_887']\n",
      "1384 -> ['NDB_935', 'NDB_940', 'NDB_926']\n",
      "1385 -> ['NDB_844', 'NDB_788', 'NDB_977']\n",
      "1386 -> ['NDB_746', 'NDB_752', 'NDB_641']\n",
      "1387 -> ['NDB_971', 'NDB_974', 'NDB_881']\n",
      "1388 -> ['NDB_715', 'NDB_880', 'NDB_929']\n",
      "1389 -> ['NDB_847', 'NDB_773', 'NDB_899']\n",
      "139 -> ['NDB_815', 'NDB_856', 'NDB_880']\n",
      "1390 -> ['NDB_989', 'NDB_818', 'NDB_872']\n",
      "1391 -> ['NDB_683', 'NDB_721', 'NDB_941']\n",
      "1392 -> ['NDB_875', 'NDB_901', 'NDB_634']\n",
      "1393 -> ['NDB_755', 'NDB_896', 'NDB_694']\n",
      "1394 -> ['NDB_784', 'NDB_790', 'NDB_958']\n",
      "1395 -> ['NDB_961', 'NDB_653', 'NDB_665']\n",
      "1396 -> ['NDB_710', 'NDB_805', 'NDB_779']\n",
      "1398 -> ['NDB_737', 'NDB_836', 'NDB_1028']\n",
      "1399 -> ['NDB_868', 'NDB_866', 'NDB_637']\n",
      "14 -> ['NDB_961', 'NDB_631', 'NDB_866']\n",
      "140 -> ['NDB_683', 'NDB_779', 'NDB_824']\n",
      "1400 -> ['NDB_1027', 'NDB_730', 'NDB_706']\n",
      "1401 -> ['NDB_718', 'NDB_643', 'NDB_862']\n",
      "1402 -> ['NDB_725', 'NDB_778', 'NDB_832']\n",
      "1403 -> ['NDB_865', 'NDB_661', 'NDB_856']\n",
      "1404 -> ['NDB_815', 'NDB_863', 'NDB_838']\n",
      "1405 -> ['NDB_713', 'NDB_754', 'NDB_835']\n",
      "1406 -> ['NDB_946', 'NDB_647', 'NDB_889']\n",
      "1408 -> ['NDB_631', 'NDB_853', 'NDB_742']\n",
      "1409 -> ['NDB_716', 'NDB_1066', 'NDB_734']\n",
      "141 -> ['NDB_652', 'NDB_860', 'NDB_838']\n",
      "1427 -> ['NDB_698', 'NDB_1060', 'NDB_721']\n",
      "1429 -> ['NDB_643', 'NDB_680', 'NDB_824']\n",
      "143 -> ['NDB_734', 'NDB_935', 'NDB_977']\n",
      "1430 -> ['NDB_725', 'NDB_754', 'NDB_856']\n",
      "1434 -> ['NDB_773', 'NDB_715', 'NDB_941']\n",
      "1435 -> ['NDB_737', 'NDB_653', 'NDB_896']\n",
      "1436 -> ['NDB_746', 'NDB_1033', 'NDB_836']\n",
      "1437 -> ['NDB_826', 'NDB_971', 'NDB_661']\n",
      "1438 -> ['NDB_905', 'NDB_641', 'NDB_994']\n",
      "1439 -> ['NDB_790', 'NDB_805', 'NDB_940']\n",
      "144 -> ['NDB_826', 'NDB_716', 'NDB_881']\n",
      "1440 -> ['NDB_868', 'NDB_878', 'NDB_1075']\n",
      "1441 -> ['NDB_755', 'NDB_634', 'NDB_751']\n",
      "1442 -> ['NDB_683', 'NDB_989', 'NDB_926']\n",
      "1443 -> ['NDB_784', 'NDB_881', 'NDB_977']\n",
      "1444 -> ['NDB_652', 'NDB_665', 'NDB_925']\n",
      "1445 -> ['NDB_961', 'NDB_716', 'NDB_832']\n",
      "1446 -> ['NDB_844', 'NDB_631', 'NDB_887']\n",
      "1447 -> ['NDB_718', 'NDB_863', 'NDB_712']\n",
      "1448 -> ['NDB_743', 'NDB_901', 'NDB_730']\n",
      "1449 -> ['NDB_865', 'NDB_889', 'NDB_958']\n",
      "145 -> ['NDB_743', 'NDB_965', 'NDB_773']\n",
      "1450 -> ['NDB_1027', 'NDB_946', 'NDB_637']\n",
      "1451 -> ['NDB_815', 'NDB_917', 'NDB_742']\n",
      "1452 -> ['NDB_713', 'NDB_965', 'NDB_974']\n",
      "1453 -> ['NDB_866', 'NDB_838', 'NDB_935']\n",
      "1454 -> ['NDB_817', 'NDB_1028', 'NDB_934']\n",
      "1455 -> ['NDB_812', 'NDB_835', 'NDB_779']\n",
      "1456 -> ['NDB_853', 'NDB_647', 'NDB_910']\n",
      "1457 -> ['NDB_698', 'NDB_880', 'NDB_862']\n",
      "1458 -> ['NDB_643', 'NDB_752', 'NDB_818']\n",
      "1459 -> ['NDB_875', 'NDB_856', 'NDB_1066']\n",
      "146 -> ['NDB_941', 'NDB_712', 'NDB_910']\n",
      "1461 -> ['NDB_961', 'NDB_905', 'NDB_836']\n",
      "1462 -> ['NDB_865', 'NDB_974', 'NDB_901']\n",
      "1463 -> ['NDB_746', 'NDB_773', 'NDB_694']\n",
      "1464 -> ['NDB_856', 'NDB_838', 'NDB_872']\n",
      "1465 -> ['NDB_755', 'NDB_730', 'NDB_712']\n",
      "1466 -> ['NDB_971', 'NDB_824', 'NDB_934']\n",
      "1467 -> ['NDB_743', 'NDB_758', 'NDB_896']\n",
      "1468 -> ['NDB_710', 'NDB_790', 'NDB_929']\n",
      "1469 -> ['NDB_815', 'NDB_805', 'NDB_721']\n",
      "147 -> ['NDB_710', 'NDB_758', 'NDB_1066']\n",
      "1470 -> ['NDB_1027', 'NDB_641', 'NDB_665']\n",
      "1471 -> ['NDB_652', 'NDB_634', 'NDB_1028']\n",
      "1472 -> ['NDB_661', 'NDB_1075', 'NDB_1066']\n",
      "1473 -> ['NDB_847', 'NDB_778', 'NDB_974']\n",
      "1474 -> ['NDB_875', 'NDB_866', 'NDB_818']\n",
      "1475 -> ['NDB_812', 'NDB_853', 'NDB_940']\n",
      "1476 -> ['NDB_683', 'NDB_1033', 'NDB_637']\n",
      "1477 -> ['NDB_754', 'NDB_878', 'NDB_751']\n",
      "1478 -> ['NDB_817', 'NDB_737', 'NDB_631']\n",
      "1479 -> ['NDB_752', 'NDB_734', 'NDB_958']\n",
      "148 -> ['NDB_718', 'NDB_1060', 'NDB_637']\n",
      "1480 -> ['NDB_835', 'NDB_706', 'NDB_925']\n",
      "1481 -> ['NDB_746', 'NDB_862', 'NDB_977']\n",
      "1482 -> ['NDB_863', 'NDB_880', 'NDB_989']\n",
      "1485 -> ['NDB_643', 'NDB_715', 'NDB_698']\n",
      "1486 -> ['NDB_784', 'NDB_832', 'NDB_946']\n",
      "1487 -> ['NDB_710', 'NDB_728', 'NDB_634']\n",
      "1488 -> ['NDB_760', 'NDB_917', 'NDB_716']\n",
      "1489 -> ['NDB_865', 'NDB_883', 'NDB_881']\n",
      "149 -> ['NDB_971', 'NDB_698', 'NDB_788']\n",
      "1490 -> ['NDB_1033', 'NDB_805', 'NDB_742']\n",
      "1491 -> ['NDB_826', 'NDB_994', 'NDB_779']\n",
      "1492 -> ['NDB_844', 'NDB_1060', 'NDB_899']\n",
      "1493 -> ['NDB_725', 'NDB_965', 'NDB_905']\n",
      "1494 -> ['NDB_718', 'NDB_838', 'NDB_926']\n",
      "1495 -> ['NDB_887', 'NDB_925', 'NDB_910']\n",
      "1496 -> ['NDB_778', 'NDB_653', 'NDB_788']\n",
      "1497 -> ['NDB_713', 'NDB_680', 'NDB_935']\n",
      "1498 -> ['NDB_868', 'NDB_941', 'NDB_889']\n",
      "1499 -> ['NDB_716', 'NDB_856', 'NDB_647']\n",
      "15 -> ['NDB_652', 'NDB_958', 'NDB_926']\n",
      "150 -> ['NDB_653', 'NDB_832', 'NDB_899']\n",
      "1500 -> ['NDB_643', 'NDB_860', 'NDB_758']\n",
      "1501 -> ['NDB_961', 'NDB_989', 'NDB_694']\n",
      "1502 -> ['NDB_728', 'NDB_715', 'NDB_742']\n",
      "1503 -> ['NDB_901', 'NDB_862', 'NDB_836']\n",
      "1504 -> ['NDB_1027', 'NDB_751', 'NDB_1028']\n",
      "1505 -> ['NDB_784', 'NDB_917', 'NDB_712']\n",
      "1506 -> ['NDB_847', 'NDB_844', 'NDB_958']\n",
      "1507 -> ['NDB_815', 'NDB_812', 'NDB_896']\n",
      "1508 -> ['NDB_754', 'NDB_665', 'NDB_889']\n",
      "1509 -> ['NDB_725', 'NDB_872', 'NDB_934']\n",
      "1510 -> ['NDB_790', 'NDB_1075', 'NDB_881']\n",
      "1511 -> ['NDB_652', 'NDB_994', 'NDB_734']\n",
      "1512 -> ['NDB_661', 'NDB_773', 'NDB_637']\n",
      "1513 -> ['NDB_875', 'NDB_721', 'NDB_853']\n",
      "1514 -> ['NDB_713', 'NDB_878', 'NDB_730']\n",
      "1515 -> ['NDB_866', 'NDB_835', 'NDB_940']\n",
      "1516 -> ['NDB_631', 'NDB_752', 'NDB_929']\n",
      "1517 -> ['NDB_971', 'NDB_737', 'NDB_641']\n",
      "1518 -> ['NDB_743', 'NDB_683', 'NDB_1066']\n",
      "1519 -> ['NDB_755', 'NDB_817', 'NDB_832']\n",
      "152 -> ['NDB_784', 'NDB_905', 'NDB_896']\n",
      "1520 -> ['NDB_863', 'NDB_706', 'NDB_818']\n",
      "1521 -> ['NDB_826', 'NDB_899', 'NDB_824']\n",
      "1525 -> ['NDB_661', 'NDB_838', 'NDB_889']\n",
      "1526 -> ['NDB_815', 'NDB_746', 'NDB_734']\n",
      "1527 -> ['NDB_965', 'NDB_860', 'NDB_862']\n",
      "1529 -> ['NDB_1027', 'NDB_989', 'NDB_853']\n",
      "153 -> ['NDB_1033', 'NDB_647', 'NDB_862']\n",
      "1530 -> ['NDB_760', 'NDB_680', 'NDB_634']\n",
      "1533 -> ['NDB_779', 'NDB_751', 'NDB_910']\n",
      "1534 -> ['NDB_718', 'NDB_883', 'NDB_929']\n",
      "1535 -> ['NDB_725', 'NDB_728', 'NDB_706']\n",
      "1536 -> ['NDB_743', 'NDB_754', 'NDB_631']\n",
      "1537 -> ['NDB_778', 'NDB_758', 'NDB_1028']\n",
      "1538 -> ['NDB_832', 'NDB_835', 'NDB_872']\n",
      "154 -> ['NDB_844', 'NDB_661', 'NDB_715']\n",
      "1541 -> ['NDB_713', 'NDB_865', 'NDB_647']\n",
      "1542 -> ['NDB_817', 'NDB_652', 'NDB_866']\n",
      "1548 -> ['NDB_812', 'NDB_730', 'NDB_742']\n",
      "155 -> ['NDB_643', 'NDB_790', 'NDB_875']\n",
      "156 -> ['NDB_1027', 'NDB_742', 'NDB_958']\n",
      "157 -> ['NDB_713', 'NDB_824', 'NDB_818']\n",
      "158 -> ['NDB_728', 'NDB_868', 'NDB_994']\n",
      "159 -> ['NDB_847', 'NDB_754', 'NDB_883']\n",
      "16 -> ['NDB_815', 'NDB_887', 'NDB_989']\n",
      "160 -> ['NDB_812', 'NDB_680', 'NDB_889']\n",
      "161 -> ['NDB_815', 'NDB_974', 'NDB_1060']\n",
      "162 -> ['NDB_917', 'NDB_961', 'NDB_751']\n",
      "163 -> ['NDB_634', 'NDB_883', 'NDB_910']\n",
      "164 -> ['NDB_805', 'NDB_887', 'NDB_665']\n",
      "165 -> ['NDB_901', 'NDB_773', 'NDB_641']\n",
      "166 -> ['NDB_826', 'NDB_863', 'NDB_896']\n",
      "167 -> ['NDB_817', 'NDB_941', 'NDB_929']\n",
      "168 -> ['NDB_965', 'NDB_721', 'NDB_925']\n",
      "169 -> ['NDB_643', 'NDB_1075', 'NDB_838']\n",
      "17 -> ['NDB_643', 'NDB_1028', 'NDB_910']\n",
      "170 -> ['NDB_661', 'NDB_737', 'NDB_712']\n",
      "171 -> ['NDB_713', 'NDB_742', 'NDB_1028']\n",
      "172 -> ['NDB_865', 'NDB_730', 'NDB_779']\n",
      "173 -> ['NDB_989', 'NDB_637', 'NDB_788']\n",
      "174 -> ['NDB_826', 'NDB_752', 'NDB_694']\n",
      "175 -> ['NDB_634', 'NDB_866', 'NDB_1066']\n",
      "176 -> ['NDB_755', 'NDB_812', 'NDB_805']\n",
      "177 -> ['NDB_778', 'NDB_728', 'NDB_631']\n",
      "178 -> ['NDB_905', 'NDB_899', 'NDB_706']\n",
      "179 -> ['NDB_725', 'NDB_853', 'NDB_958']\n",
      "18 -> ['NDB_760', 'NDB_734', 'NDB_712']\n",
      "180 -> ['NDB_880', 'NDB_934', 'NDB_926']\n",
      "181 -> ['NDB_971', 'NDB_758', 'NDB_734']\n",
      "182 -> ['NDB_844', 'NDB_881', 'NDB_935']\n",
      "183 -> ['NDB_680', 'NDB_832', 'NDB_641']\n",
      "184 -> ['NDB_715', 'NDB_1033', 'NDB_835']\n",
      "185 -> ['NDB_652', 'NDB_862', 'NDB_929']\n",
      "186 -> ['NDB_917', 'NDB_721', 'NDB_818']\n",
      "187 -> ['NDB_847', 'NDB_946', 'NDB_872']\n",
      "188 -> ['NDB_817', 'NDB_878', 'NDB_977']\n",
      "189 -> ['NDB_710', 'NDB_716', 'NDB_860']\n",
      "19 -> ['NDB_965', 'NDB_661', 'NDB_899']\n",
      "190 -> ['NDB_743', 'NDB_856', 'NDB_940']\n",
      "191 -> ['NDB_784', 'NDB_868', 'NDB_647']\n",
      "192 -> ['NDB_746', 'NDB_653', 'NDB_925']\n",
      "193 -> ['NDB_760', 'NDB_875', 'NDB_1027']\n",
      "194 -> ['NDB_755', 'NDB_683', 'NDB_631']\n",
      "195 -> ['NDB_743', 'NDB_935', 'NDB_818']\n",
      "196 -> ['NDB_652', 'NDB_1033', 'NDB_899']\n",
      "197 -> ['NDB_1060', 'NDB_929', 'NDB_1028']\n",
      "198 -> ['NDB_728', 'NDB_860', 'NDB_698']\n",
      "199 -> ['NDB_805', 'NDB_835', 'NDB_647']\n",
      "2 -> ['NDB_847', 'NDB_832', 'NDB_779']\n",
      "20 -> ['NDB_653', 'NDB_977', 'NDB_940']\n",
      "200 -> ['NDB_826', 'NDB_730', 'NDB_887']\n",
      "201 -> ['NDB_758', 'NDB_946', 'NDB_824']\n",
      "202 -> ['NDB_713', 'NDB_746', 'NDB_715']\n",
      "203 -> ['NDB_901', 'NDB_737', 'NDB_788']\n",
      "204 -> ['NDB_778', 'NDB_716', 'NDB_896']\n",
      "205 -> ['NDB_680', 'NDB_994', 'NDB_934']\n",
      "206 -> ['NDB_812', 'NDB_856', 'NDB_751']\n",
      "207 -> ['NDB_905', 'NDB_721', 'NDB_734']\n",
      "208 -> ['NDB_875', 'NDB_754', 'NDB_641']\n",
      "209 -> ['NDB_718', 'NDB_1033', 'NDB_665']\n",
      "21 -> ['NDB_743', 'NDB_917', 'NDB_715']\n",
      "210 -> ['NDB_971', 'NDB_637', 'NDB_712']\n",
      "211 -> ['NDB_1027', 'NDB_818', 'NDB_910']\n",
      "212 -> ['NDB_917', 'NDB_728', 'NDB_940']\n",
      "213 -> ['NDB_965', 'NDB_1066', 'NDB_977']\n",
      "214 -> ['NDB_755', 'NDB_961', 'NDB_661']\n",
      "215 -> ['NDB_631', 'NDB_706', 'NDB_926']\n",
      "216 -> ['NDB_760', 'NDB_889', 'NDB_824']\n",
      "217 -> ['NDB_868', 'NDB_887', 'NDB_935']\n",
      "218 -> ['NDB_773', 'NDB_878', 'NDB_653']\n",
      "219 -> ['NDB_634', 'NDB_752', 'NDB_838']\n",
      "22 -> ['NDB_698', 'NDB_883', 'NDB_1075']\n",
      "220 -> ['NDB_817', 'NDB_974', 'NDB_883']\n",
      "221 -> ['NDB_1075', 'NDB_941', 'NDB_694']\n",
      "222 -> ['NDB_866', 'NDB_989', 'NDB_836']\n",
      "223 -> ['NDB_725', 'NDB_721', 'NDB_665']\n",
      "224 -> ['NDB_743', 'NDB_815', 'NDB_875']\n",
      "225 -> ['NDB_863', 'NDB_1060', 'NDB_940']\n",
      "226 -> ['NDB_790', 'NDB_899', 'NDB_779']\n",
      "227 -> ['NDB_718', 'NDB_880', 'NDB_647']\n",
      "228 -> ['NDB_844', 'NDB_1027', 'NDB_1066']\n",
      "229 -> ['NDB_826', 'NDB_734', 'NDB_862']\n",
      "23 -> ['NDB_728', 'NDB_946', 'NDB_838']\n",
      "230 -> ['NDB_754', 'NDB_853', 'NDB_872']\n",
      "233 -> ['NDB_713', 'NDB_971', 'NDB_917']\n",
      "234 -> ['NDB_643', 'NDB_925', 'NDB_712']\n",
      "235 -> ['NDB_847', 'NDB_929', 'NDB_824']\n",
      "236 -> ['NDB_631', 'NDB_832', 'NDB_889']\n",
      "237 -> ['NDB_710', 'NDB_683', 'NDB_977']\n",
      "238 -> ['NDB_868', 'NDB_860', 'NDB_751']\n",
      "239 -> ['NDB_835', 'NDB_721', 'NDB_881']\n",
      "24 -> ['NDB_754', 'NDB_860', 'NDB_752']\n",
      "240 -> ['NDB_784', 'NDB_742', 'NDB_926']\n",
      "241 -> ['NDB_961', 'NDB_901', 'NDB_715']\n",
      "242 -> ['NDB_760', 'NDB_728', 'NDB_935']\n",
      "243 -> ['NDB_652', 'NDB_778', 'NDB_880']\n",
      "244 -> ['NDB_698', 'NDB_863', 'NDB_647']\n",
      "245 -> ['NDB_746', 'NDB_805', 'NDB_958']\n",
      "246 -> ['NDB_865', 'NDB_725', 'NDB_815']\n",
      "247 -> ['NDB_680', 'NDB_883', 'NDB_1028']\n",
      "248 -> ['NDB_878', 'NDB_941', 'NDB_788']\n",
      "249 -> ['NDB_856', 'NDB_832', 'NDB_836']\n",
      "25 -> ['NDB_905', 'NDB_832', 'NDB_665']\n",
      "250 -> ['NDB_875', 'NDB_730', 'NDB_872']\n",
      "251 -> ['NDB_634', 'NDB_1033', 'NDB_994']\n",
      "252 -> ['NDB_974', 'NDB_737', 'NDB_1075']\n",
      "253 -> ['NDB_896', 'NDB_706', 'NDB_934']\n",
      "254 -> ['NDB_653', 'NDB_835', 'NDB_853']\n",
      "255 -> ['NDB_817', 'NDB_881', 'NDB_862']\n",
      "256 -> ['NDB_773', 'NDB_838', 'NDB_925']\n",
      "257 -> ['NDB_713', 'NDB_790', 'NDB_716']\n",
      "258 -> ['NDB_778', 'NDB_866', 'NDB_752']\n",
      "259 -> ['NDB_661', 'NDB_698', 'NDB_989']\n",
      "26 -> ['NDB_853', 'NDB_941', 'NDB_881']\n",
      "260 -> ['NDB_755', 'NDB_758', 'NDB_940']\n",
      "261 -> ['NDB_718', 'NDB_905', 'NDB_946']\n",
      "262 -> ['NDB_865', 'NDB_641', 'NDB_818']\n",
      "263 -> ['NDB_847', 'NDB_863', 'NDB_910']\n",
      "264 -> ['NDB_710', 'NDB_844', 'NDB_868']\n",
      "265 -> ['NDB_784', 'NDB_826', 'NDB_812']\n",
      "266 -> ['NDB_760', 'NDB_779', 'NDB_742']\n",
      "267 -> ['NDB_971', 'NDB_683', 'NDB_958']\n",
      "268 -> ['NDB_652', 'NDB_694', 'NDB_872']\n",
      "269 -> ['NDB_743', 'NDB_643', 'NDB_665']\n",
      "27 -> ['NDB_713', 'NDB_1066', 'NDB_929']\n",
      "270 -> ['NDB_965', 'NDB_917', 'NDB_881']\n",
      "271 -> ['NDB_730', 'NDB_880', 'NDB_637']\n",
      "272 -> ['NDB_1060', 'NDB_751', 'NDB_925']\n",
      "273 -> ['NDB_737', 'NDB_929', 'NDB_934']\n",
      "274 -> ['NDB_725', 'NDB_817', 'NDB_683']\n",
      "275 -> ['NDB_1075', 'NDB_788', 'NDB_862']\n",
      "276 -> ['NDB_896', 'NDB_989', 'NDB_734']\n",
      "277 -> ['NDB_653', 'NDB_887', 'NDB_694']\n",
      "278 -> ['NDB_832', 'NDB_1066', 'NDB_925']\n",
      "279 -> ['NDB_826', 'NDB_718', 'NDB_965']\n",
      "28 -> ['NDB_784', 'NDB_875', 'NDB_751']\n",
      "280 -> ['NDB_760', 'NDB_905', 'NDB_647']\n",
      "281 -> ['NDB_847', 'NDB_643', 'NDB_812']\n",
      "282 -> ['NDB_634', 'NDB_779', 'NDB_977']\n",
      "283 -> ['NDB_863', 'NDB_805', 'NDB_751']\n",
      "284 -> ['NDB_728', 'NDB_661', 'NDB_641']\n",
      "285 -> ['NDB_974', 'NDB_875', 'NDB_836']\n",
      "286 -> ['NDB_710', 'NDB_737', 'NDB_946']\n",
      "287 -> ['NDB_742', 'NDB_929', 'NDB_818']\n",
      "288 -> ['NDB_652', 'NDB_917', 'NDB_631']\n",
      "289 -> ['NDB_883', 'NDB_896', 'NDB_838']\n",
      "29 -> ['NDB_647', 'NDB_779', 'NDB_925']\n",
      "290 -> ['NDB_653', 'NDB_637', 'NDB_862']\n",
      "291 -> ['NDB_725', 'NDB_887', 'NDB_941']\n",
      "292 -> ['NDB_868', 'NDB_758', 'NDB_665']\n",
      "293 -> ['NDB_844', 'NDB_1033', 'NDB_721']\n",
      "295 -> ['NDB_817', 'NDB_1027', 'NDB_754']\n",
      "296 -> ['NDB_746', 'NDB_856', 'NDB_934']\n",
      "297 -> ['NDB_713', 'NDB_880', 'NDB_940']\n",
      "298 -> ['NDB_971', 'NDB_778', 'NDB_872']\n",
      "299 -> ['NDB_715', 'NDB_899', 'NDB_935']\n",
      "3 -> ['NDB_755', 'NDB_899', 'NDB_889']\n",
      "30 -> ['NDB_725', 'NDB_661', 'NDB_994']\n",
      "300 -> ['NDB_961', 'NDB_730', 'NDB_835']\n",
      "302 -> ['NDB_901', 'NDB_1060', 'NDB_889']\n",
      "303 -> ['NDB_784', 'NDB_1066', 'NDB_836']\n",
      "304 -> ['NDB_1075', 'NDB_641', 'NDB_712']\n",
      "305 -> ['NDB_790', 'NDB_634', 'NDB_941']\n",
      "306 -> ['NDB_653', 'NDB_838', 'NDB_881']\n",
      "307 -> ['NDB_865', 'NDB_752', 'NDB_926']\n",
      "308 -> ['NDB_815', 'NDB_698', 'NDB_853']\n",
      "309 -> ['NDB_866', 'NDB_1028', 'NDB_824']\n",
      "31 -> ['NDB_710', 'NDB_746', 'NDB_812']\n",
      "310 -> ['NDB_743', 'NDB_716', 'NDB_878']\n",
      "311 -> ['NDB_643', 'NDB_883', 'NDB_706']\n",
      "312 -> ['NDB_860', 'NDB_994', 'NDB_910']\n",
      "313 -> ['NDB_1027', 'NDB_680', 'NDB_694']\n",
      "314 -> ['NDB_754', 'NDB_887', 'NDB_637']\n",
      "315 -> ['NDB_755', 'NDB_773', 'NDB_977']\n",
      "316 -> ['NDB_737', 'NDB_751', 'NDB_958']\n",
      "317 -> ['NDB_698', 'NDB_889', 'NDB_818']\n",
      "318 -> ['NDB_631', 'NDB_712', 'NDB_934']\n",
      "319 -> ['NDB_713', 'NDB_661', 'NDB_925']\n",
      "32 -> ['NDB_778', 'NDB_935', 'NDB_694']\n",
      "320 -> ['NDB_760', 'NDB_901', 'NDB_940']\n",
      "321 -> ['NDB_784', 'NDB_847', 'NDB_665']\n",
      "322 -> ['NDB_710', 'NDB_835', 'NDB_899']\n",
      "323 -> ['NDB_971', 'NDB_961', 'NDB_935']\n",
      "324 -> ['NDB_683', 'NDB_728', 'NDB_832']\n",
      "325 -> ['NDB_905', 'NDB_752', 'NDB_941']\n",
      "326 -> ['NDB_974', 'NDB_680', 'NDB_989']\n",
      "327 -> ['NDB_805', 'NDB_896', 'NDB_853']\n",
      "328 -> ['NDB_868', 'NDB_812', 'NDB_946']\n",
      "329 -> ['NDB_755', 'NDB_826', 'NDB_875']\n",
      "33 -> ['NDB_917', 'NDB_824', 'NDB_910']\n",
      "330 -> ['NDB_758', 'NDB_1060', 'NDB_742']\n",
      "331 -> ['NDB_718', 'NDB_881', 'NDB_824']\n",
      "332 -> ['NDB_965', 'NDB_856', 'NDB_929']\n",
      "333 -> ['NDB_652', 'NDB_746', 'NDB_721']\n",
      "334 -> ['NDB_815', 'NDB_730', 'NDB_647']\n",
      "335 -> ['NDB_734', 'NDB_788', 'NDB_926']\n",
      "336 -> ['NDB_865', 'NDB_716', 'NDB_706']\n",
      "337 -> ['NDB_773', 'NDB_866', 'NDB_712']\n",
      "338 -> ['NDB_994', 'NDB_862', 'NDB_1028']\n",
      "339 -> ['NDB_790', 'NDB_974', 'NDB_715']\n",
      "34 -> ['NDB_718', 'NDB_974', 'NDB_1028']\n",
      "340 -> ['NDB_826', 'NDB_917', 'NDB_844']\n",
      "341 -> ['NDB_743', 'NDB_805', 'NDB_880']\n",
      "342 -> ['NDB_905', 'NDB_779', 'NDB_872']\n",
      "343 -> ['NDB_1033', 'NDB_1066', 'NDB_958']\n",
      "344 -> ['NDB_680', 'NDB_838', 'NDB_910']\n",
      "345 -> ['NDB_725', 'NDB_860', 'NDB_637']\n",
      "346 -> ['NDB_643', 'NDB_1027', 'NDB_878']\n",
      "347 -> ['NDB_754', 'NDB_863', 'NDB_1075']\n",
      "348 -> ['NDB_778', 'NDB_883', 'NDB_989']\n",
      "349 -> ['NDB_817', 'NDB_634', 'NDB_641']\n",
      "35 -> ['NDB_773', 'NDB_1033', 'NDB_880']\n",
      "350 -> ['NDB_683', 'NDB_889', 'NDB_788']\n",
      "351 -> ['NDB_815', 'NDB_706', 'NDB_836']\n",
      "352 -> ['NDB_718', 'NDB_1027', 'NDB_856']\n",
      "353 -> ['NDB_698', 'NDB_758', 'NDB_835']\n",
      "354 -> ['NDB_661', 'NDB_905', 'NDB_958']\n",
      "355 -> ['NDB_847', 'NDB_838', 'NDB_934']\n",
      "356 -> ['NDB_865', 'NDB_790', 'NDB_728']\n",
      "357 -> ['NDB_868', 'NDB_653', 'NDB_818']\n",
      "358 -> ['NDB_817', 'NDB_1033', 'NDB_896']\n",
      "359 -> ['NDB_680', 'NDB_941', 'NDB_926']\n",
      "36 -> ['NDB_826', 'NDB_865', 'NDB_1027']\n",
      "360 -> ['NDB_713', 'NDB_901', 'NDB_899']\n",
      "361 -> ['NDB_875', 'NDB_883', 'NDB_647']\n",
      "362 -> ['NDB_971', 'NDB_878', 'NDB_929']\n",
      "363 -> ['NDB_965', 'NDB_752', 'NDB_853']\n",
      "364 -> ['NDB_754', 'NDB_1066', 'NDB_694']\n",
      "365 -> ['NDB_760', 'NDB_778', 'NDB_860']\n",
      "366 -> ['NDB_716', 'NDB_1060', 'NDB_887']\n",
      "367 -> ['NDB_826', 'NDB_710', 'NDB_742']\n",
      "368 -> ['NDB_863', 'NDB_631', 'NDB_665']\n",
      "37 -> ['NDB_817', 'NDB_971', 'NDB_721']\n",
      "370 -> ['NDB_847', 'NDB_917', 'NDB_989']\n",
      "371 -> ['NDB_961', 'NDB_641', 'NDB_824']\n",
      "372 -> ['NDB_683', 'NDB_1075', 'NDB_946']\n",
      "373 -> ['NDB_805', 'NDB_935', 'NDB_862']\n",
      "374 -> ['NDB_746', 'NDB_680', 'NDB_751']\n",
      "375 -> ['NDB_865', 'NDB_773', 'NDB_1028']\n",
      "376 -> ['NDB_728', 'NDB_974', 'NDB_1027']\n",
      "377 -> ['NDB_875', 'NDB_881', 'NDB_940']\n",
      "379 -> ['NDB_817', 'NDB_965', 'NDB_730']\n",
      "38 -> ['NDB_790', 'NDB_698', 'NDB_896']\n",
      "380 -> ['NDB_880', 'NDB_712', 'NDB_818']\n",
      "381 -> ['NDB_643', 'NDB_929', 'NDB_872']\n",
      "382 -> ['NDB_815', 'NDB_844', 'NDB_910']\n",
      "383 -> ['NDB_896', 'NDB_977', 'NDB_926']\n",
      "384 -> ['NDB_784', 'NDB_737', 'NDB_715']\n",
      "385 -> ['NDB_725', 'NDB_883', 'NDB_899']\n",
      "386 -> ['NDB_755', 'NDB_653', 'NDB_989']\n",
      "387 -> ['NDB_971', 'NDB_889', 'NDB_925']\n",
      "388 -> ['NDB_652', 'NDB_901', 'NDB_941']\n",
      "389 -> ['NDB_713', 'NDB_634', 'NDB_721']\n",
      "39 -> ['NDB_847', 'NDB_683', 'NDB_901']\n",
      "390 -> ['NDB_743', 'NDB_812', 'NDB_647']\n",
      "391 -> ['NDB_1033', 'NDB_866', 'NDB_734']\n",
      "392 -> ['NDB_868', 'NDB_716', 'NDB_752']\n",
      "393 -> ['NDB_860', 'NDB_779', 'NDB_694']\n",
      "394 -> ['NDB_773', 'NDB_994', 'NDB_788']\n",
      "395 -> ['NDB_917', 'NDB_878', 'NDB_853']\n",
      "396 -> ['NDB_637', 'NDB_706', 'NDB_1028']\n",
      "397 -> ['NDB_758', 'NDB_836', 'NDB_910']\n",
      "398 -> ['NDB_815', 'NDB_790', 'NDB_1066']\n",
      "399 -> ['NDB_643', 'NDB_974', 'NDB_832']\n",
      "4 -> ['NDB_710', 'NDB_665', 'NDB_1028']\n",
      "40 -> ['NDB_728', 'NDB_863', 'NDB_752']\n",
      "400 -> ['NDB_737', 'NDB_805', 'NDB_872']\n",
      "401 -> ['NDB_661', 'NDB_754', 'NDB_742']\n",
      "402 -> ['NDB_835', 'NDB_838', 'NDB_665']\n",
      "403 -> ['NDB_1060', 'NDB_641', 'NDB_977']\n",
      "404 -> ['NDB_652', 'NDB_905', 'NDB_824']\n",
      "405 -> ['NDB_725', 'NDB_1075', 'NDB_940']\n",
      "406 -> ['NDB_730', 'NDB_862', 'NDB_934']\n",
      "407 -> ['NDB_856', 'NDB_863', 'NDB_881']\n",
      "408 -> ['NDB_710', 'NDB_760', 'NDB_958']\n",
      "409 -> ['NDB_784', 'NDB_755', 'NDB_880']\n",
      "410 -> ['NDB_778', 'NDB_946', 'NDB_712']\n",
      "411 -> ['NDB_743', 'NDB_887', 'NDB_721']\n",
      "412 -> ['NDB_826', 'NDB_634', 'NDB_637']\n",
      "413 -> ['NDB_718', 'NDB_866', 'NDB_788']\n",
      "414 -> ['NDB_631', 'NDB_734', 'NDB_910']\n",
      "415 -> ['NDB_790', 'NDB_994', 'NDB_925']\n",
      "416 -> ['NDB_1027', 'NDB_935', 'NDB_779']\n",
      "417 -> ['NDB_715', 'NDB_905', 'NDB_751']\n",
      "418 -> ['NDB_847', 'NDB_746', 'NDB_698']\n",
      "419 -> ['NDB_812', 'NDB_758', 'NDB_832']\n",
      "42 -> ['NDB_844', 'NDB_856', 'NDB_1075']\n",
      "421 -> ['NDB_844', 'NDB_961', 'NDB_773']\n",
      "422 -> ['NDB_974', 'NDB_860', 'NDB_721']\n",
      "423 -> ['NDB_866', 'NDB_742', 'NDB_872']\n",
      "424 -> ['NDB_710', 'NDB_754', 'NDB_935']\n",
      "425 -> ['NDB_815', 'NDB_773', 'NDB_889']\n",
      "426 -> ['NDB_683', 'NDB_961', 'NDB_805']\n",
      "427 -> ['NDB_887', 'NDB_1028', 'NDB_926']\n",
      "428 -> ['NDB_713', 'NDB_875', 'NDB_896']\n",
      "429 -> ['NDB_790', 'NDB_778', 'NDB_680']\n",
      "43 -> ['NDB_961', 'NDB_889', 'NDB_977']\n",
      "430 -> ['NDB_817', 'NDB_856', 'NDB_853']\n",
      "431 -> ['NDB_698', 'NDB_878', 'NDB_694']\n",
      "432 -> ['NDB_865', 'NDB_758', 'NDB_712']\n",
      "435 -> ['NDB_1027', 'NDB_899', 'NDB_838']\n",
      "436 -> ['NDB_971', 'NDB_812', 'NDB_862']\n",
      "437 -> ['NDB_755', 'NDB_974', 'NDB_779']\n",
      "438 -> ['NDB_946', 'NDB_940', 'NDB_910']\n",
      "439 -> ['NDB_652', 'NDB_868', 'NDB_977']\n",
      "44 -> ['NDB_713', 'NDB_868', 'NDB_712']\n",
      "440 -> ['NDB_728', 'NDB_737', 'NDB_989']\n",
      "441 -> ['NDB_715', 'NDB_896', 'NDB_925']\n",
      "442 -> ['NDB_784', 'NDB_718', 'NDB_653']\n",
      "443 -> ['NDB_881', 'NDB_889', 'NDB_929']\n",
      "444 -> ['NDB_760', 'NDB_631', 'NDB_1075']\n",
      "446 -> ['NDB_710', 'NDB_661', 'NDB_941']\n",
      "447 -> ['NDB_812', 'NDB_994', 'NDB_818']\n",
      "448 -> ['NDB_743', 'NDB_863', 'NDB_832']\n",
      "449 -> ['NDB_634', 'NDB_647', 'NDB_934']\n",
      "45 -> ['NDB_935', 'NDB_742', 'NDB_934']\n",
      "450 -> ['NDB_713', 'NDB_725', 'NDB_1033']\n",
      "451 -> ['NDB_790', 'NDB_866', 'NDB_751']\n",
      "452 -> ['NDB_971', 'NDB_844', 'NDB_901']\n",
      "453 -> ['NDB_643', 'NDB_754', 'NDB_836']\n",
      "454 -> ['NDB_683', 'NDB_746', 'NDB_730']\n",
      "455 -> ['NDB_778', 'NDB_805', 'NDB_641']\n",
      "456 -> ['NDB_826', 'NDB_815', 'NDB_872']\n",
      "457 -> ['NDB_698', 'NDB_887', 'NDB_752']\n",
      "458 -> ['NDB_865', 'NDB_853', 'NDB_665']\n",
      "459 -> ['NDB_856', 'NDB_994', 'NDB_706']\n",
      "46 -> ['NDB_634', 'NDB_730', 'NDB_926']\n",
      "460 -> ['NDB_860', 'NDB_878', 'NDB_835']\n",
      "461 -> ['NDB_716', 'NDB_694', 'NDB_818']\n",
      "462 -> ['NDB_773', 'NDB_883', 'NDB_905']\n",
      "463 -> ['NDB_847', 'NDB_637', 'NDB_734']\n",
      "464 -> ['NDB_1060', 'NDB_880', 'NDB_935']\n",
      "465 -> ['NDB_917', 'NDB_875', 'NDB_958']\n",
      "466 -> ['NDB_965', 'NDB_961', 'NDB_680']\n",
      "467 -> ['NDB_725', 'NDB_788', 'NDB_824']\n",
      "468 -> ['NDB_817', 'NDB_901', 'NDB_734']\n",
      "469 -> ['NDB_883', 'NDB_1066', 'NDB_926']\n",
      "47 -> ['NDB_784', 'NDB_805', 'NDB_734']\n",
      "470 -> ['NDB_653', 'NDB_721', 'NDB_1028']\n",
      "471 -> ['NDB_718', 'NDB_752', 'NDB_706']\n",
      "472 -> ['NDB_716', 'NDB_946', 'NDB_929']\n",
      "473 -> ['NDB_631', 'NDB_637', 'NDB_779']\n",
      "474 -> ['NDB_826', 'NDB_760', 'NDB_941']\n",
      "475 -> ['NDB_730', 'NDB_925', 'NDB_940']\n",
      "476 -> ['NDB_1075', 'NDB_832', 'NDB_977']\n",
      "477 -> ['NDB_881', 'NDB_836', 'NDB_712']\n",
      "478 -> ['NDB_743', 'NDB_868', 'NDB_989']\n",
      "479 -> ['NDB_661', 'NDB_862', 'NDB_751']\n",
      "480 -> ['NDB_728', 'NDB_958', 'NDB_910']\n",
      "481 -> ['NDB_880', 'NDB_641', 'NDB_838']\n",
      "482 -> ['NDB_899', 'NDB_1066', 'NDB_665']\n",
      "483 -> ['NDB_652', 'NDB_758', 'NDB_934']\n",
      "484 -> ['NDB_784', 'NDB_863', 'NDB_1033']\n",
      "485 -> ['NDB_643', 'NDB_737', 'NDB_647']\n",
      "486 -> ['NDB_746', 'NDB_634', 'NDB_835']\n",
      "487 -> ['NDB_755', 'NDB_1027', 'NDB_715']\n",
      "489 -> ['NDB_844', 'NDB_860', 'NDB_742']\n",
      "490 -> ['NDB_835', 'NDB_641', 'NDB_734']\n",
      "491 -> ['NDB_698', 'NDB_935', 'NDB_1028']\n",
      "492 -> ['NDB_826', 'NDB_832', 'NDB_926']\n",
      "493 -> ['NDB_865', 'NDB_880', 'NDB_824']\n",
      "494 -> ['NDB_847', 'NDB_752', 'NDB_1066']\n",
      "495 -> ['NDB_875', 'NDB_994', 'NDB_665']\n",
      "496 -> ['NDB_971', 'NDB_746', 'NDB_706']\n",
      "497 -> ['NDB_901', 'NDB_1033', 'NDB_934']\n",
      "498 -> ['NDB_778', 'NDB_1075', 'NDB_637']\n",
      "499 -> ['NDB_710', 'NDB_631', 'NDB_838']\n",
      "5 -> ['NDB_643', 'NDB_805', 'NDB_730']\n",
      "500 -> ['NDB_1027', 'NDB_905', 'NDB_788']\n",
      "501 -> ['NDB_817', 'NDB_790', 'NDB_917']\n",
      "502 -> ['NDB_713', 'NDB_961', 'NDB_734']\n",
      "503 -> ['NDB_965', 'NDB_835', 'NDB_958']\n",
      "504 -> ['NDB_989', 'NDB_881', 'NDB_1066']\n",
      "505 -> ['NDB_868', 'NDB_715', 'NDB_779']\n",
      "506 -> ['NDB_805', 'NDB_925', 'NDB_818']\n",
      "507 -> ['NDB_652', 'NDB_743', 'NDB_1060']\n",
      "508 -> ['NDB_815', 'NDB_634', 'NDB_946']\n",
      "509 -> ['NDB_737', 'NDB_941', 'NDB_994']\n",
      "510 -> ['NDB_812', 'NDB_860', 'NDB_836']\n",
      "511 -> ['NDB_883', 'NDB_977', 'NDB_712']\n",
      "512 -> ['NDB_755', 'NDB_680', 'NDB_863']\n",
      "516 -> ['NDB_716', 'NDB_730', 'NDB_889']\n",
      "517 -> ['NDB_974', 'NDB_878', 'NDB_866']\n",
      "518 -> ['NDB_725', 'NDB_989', 'NDB_929']\n",
      "519 -> ['NDB_728', 'NDB_721', 'NDB_694']\n",
      "520 -> ['NDB_784', 'NDB_856', 'NDB_1028']\n",
      "521 -> ['NDB_683', 'NDB_853', 'NDB_862']\n",
      "522 -> ['NDB_643', 'NDB_844', 'NDB_653']\n",
      "523 -> ['NDB_899', 'NDB_647', 'NDB_940']\n",
      "524 -> ['NDB_718', 'NDB_773', 'NDB_754']\n",
      "525 -> ['NDB_661', 'NDB_758', 'NDB_872']\n",
      "526 -> ['NDB_896', 'NDB_751', 'NDB_742']\n",
      "527 -> ['NDB_812', 'NDB_641', 'NDB_941']\n",
      "528 -> ['NDB_634', 'NDB_737', 'NDB_881']\n",
      "529 -> ['NDB_760', 'NDB_961', 'NDB_910']\n",
      "530 -> ['NDB_887', 'NDB_958', 'NDB_934']\n",
      "531 -> ['NDB_652', 'NDB_965', 'NDB_889']\n",
      "532 -> ['NDB_875', 'NDB_706', 'NDB_824']\n",
      "533 -> ['NDB_743', 'NDB_838', 'NDB_637']\n",
      "534 -> ['NDB_784', 'NDB_725', 'NDB_1027']\n",
      "535 -> ['NDB_868', 'NDB_1060', 'NDB_905']\n",
      "536 -> ['NDB_971', 'NDB_1033', 'NDB_631']\n",
      "537 -> ['NDB_716', 'NDB_788', 'NDB_925']\n",
      "538 -> ['NDB_710', 'NDB_866', 'NDB_853']\n",
      "539 -> ['NDB_728', 'NDB_746', 'NDB_878']\n",
      "540 -> ['NDB_790', 'NDB_862', 'NDB_818']\n",
      "541 -> ['NDB_713', 'NDB_1075', 'NDB_805']\n",
      "542 -> ['NDB_826', 'NDB_778', 'NDB_715']\n",
      "543 -> ['NDB_718', 'NDB_901', 'NDB_694']\n",
      "544 -> ['NDB_917', 'NDB_665', 'NDB_935']\n",
      "545 -> ['NDB_865', 'NDB_847', 'NDB_680']\n",
      "546 -> ['NDB_817', 'NDB_880', 'NDB_910']\n",
      "548 -> ['NDB_883', 'NDB_887', 'NDB_779']\n",
      "549 -> ['NDB_653', 'NDB_752', 'NDB_946']\n",
      "550 -> ['NDB_863', 'NDB_977', 'NDB_872']\n",
      "551 -> ['NDB_860', 'NDB_647', 'NDB_926']\n",
      "552 -> ['NDB_760', 'NDB_698', 'NDB_832']\n",
      "554 -> ['NDB_868', 'NDB_880', 'NDB_721']\n",
      "555 -> ['NDB_974', 'NDB_863', 'NDB_889']\n",
      "556 -> ['NDB_784', 'NDB_860', 'NDB_940']\n",
      "557 -> ['NDB_752', 'NDB_989', 'NDB_712']\n",
      "558 -> ['NDB_728', 'NDB_1060', 'NDB_934']\n",
      "559 -> ['NDB_718', 'NDB_844', 'NDB_758']\n",
      "560 -> ['NDB_746', 'NDB_853', 'NDB_1066']\n",
      "561 -> ['NDB_790', 'NDB_754', 'NDB_1028']\n",
      "562 -> ['NDB_941', 'NDB_637', 'NDB_824']\n",
      "563 -> ['NDB_865', 'NDB_971', 'NDB_1075']\n",
      "564 -> ['NDB_737', 'NDB_880', 'NDB_734']\n",
      "565 -> ['NDB_716', 'NDB_751', 'NDB_836']\n",
      "566 -> ['NDB_778', 'NDB_899', 'NDB_742']\n",
      "567 -> ['NDB_901', 'NDB_832', 'NDB_881']\n",
      "568 -> ['NDB_661', 'NDB_680', 'NDB_788']\n",
      "569 -> ['NDB_760', 'NDB_730', 'NDB_838']\n",
      "57 -> ['NDB_773', 'NDB_706', 'NDB_940']\n",
      "570 -> ['NDB_878', 'NDB_905', 'NDB_712']\n",
      "571 -> ['NDB_826', 'NDB_835', 'NDB_935']\n",
      "572 -> ['NDB_866', 'NDB_929', 'NDB_958']\n",
      "573 -> ['NDB_961', 'NDB_1027', 'NDB_887']\n",
      "574 -> ['NDB_773', 'NDB_631', 'NDB_862']\n",
      "575 -> ['NDB_718', 'NDB_847', 'NDB_853']\n",
      "576 -> ['NDB_652', 'NDB_728', 'NDB_754']\n",
      "577 -> ['NDB_755', 'NDB_815', 'NDB_925']\n",
      "578 -> ['NDB_1066', 'NDB_779', 'NDB_1028']\n",
      "579 -> ['NDB_817', 'NDB_643', 'NDB_946']\n",
      "580 -> ['NDB_965', 'NDB_812', 'NDB_706']\n",
      "581 -> ['NDB_868', 'NDB_856', 'NDB_634']\n",
      "582 -> ['NDB_790', 'NDB_641', 'NDB_935']\n",
      "583 -> ['NDB_710', 'NDB_715', 'NDB_881']\n",
      "584 -> ['NDB_743', 'NDB_1033', 'NDB_905']\n",
      "585 -> ['NDB_865', 'NDB_917', 'NDB_698']\n",
      "586 -> ['NDB_971', 'NDB_977', 'NDB_818']\n",
      "587 -> ['NDB_805', 'NDB_694', 'NDB_836']\n",
      "588 -> ['NDB_725', 'NDB_896', 'NDB_647']\n",
      "589 -> ['NDB_875', 'NDB_889', 'NDB_910']\n",
      "590 -> ['NDB_652', 'NDB_683', 'NDB_653']\n",
      "591 -> ['NDB_713', 'NDB_883', 'NDB_665']\n",
      "592 -> ['NDB_994', 'NDB_872', 'NDB_926']\n",
      "593 -> ['NDB_847', 'NDB_641', 'NDB_788']\n",
      "594 -> ['NDB_778', 'NDB_721', 'NDB_779']\n",
      "595 -> ['NDB_856', 'NDB_878', 'NDB_1060']\n",
      "596 -> ['NDB_743', 'NDB_844', 'NDB_737']\n",
      "597 -> ['NDB_710', 'NDB_961', 'NDB_1075']\n",
      "598 -> ['NDB_965', 'NDB_637', 'NDB_940']\n",
      "599 -> ['NDB_946', 'NDB_941', 'NDB_734']\n",
      "6 -> ['NDB_883', 'NDB_1060', 'NDB_853']\n",
      "60 -> ['NDB_971', 'NDB_716', 'NDB_680']\n",
      "600 -> ['NDB_812', 'NDB_631', 'NDB_824']\n",
      "601 -> ['NDB_815', 'NDB_860', 'NDB_934']\n",
      "602 -> ['NDB_989', 'NDB_899', 'NDB_862']\n",
      "603 -> ['NDB_817', 'NDB_760', 'NDB_758']\n",
      "605 -> ['NDB_661', 'NDB_752', 'NDB_896']\n",
      "606 -> ['NDB_784', 'NDB_974', 'NDB_929']\n",
      "607 -> ['NDB_863', 'NDB_866', 'NDB_925']\n",
      "608 -> ['NDB_887', 'NDB_835', 'NDB_751']\n",
      "609 -> ['NDB_746', 'NDB_832', 'NDB_838']\n",
      "611 -> ['NDB_713', 'NDB_977', 'NDB_910']\n",
      "612 -> ['NDB_847', 'NDB_737', 'NDB_694']\n",
      "613 -> ['NDB_790', 'NDB_721', 'NDB_742']\n",
      "614 -> ['NDB_856', 'NDB_905', 'NDB_805']\n",
      "615 -> ['NDB_917', 'NDB_883', 'NDB_788']\n",
      "616 -> ['NDB_971', 'NDB_634', 'NDB_880']\n",
      "617 -> ['NDB_965', 'NDB_1033', 'NDB_1028']\n",
      "618 -> ['NDB_778', 'NDB_844', 'NDB_818']\n",
      "619 -> ['NDB_725', 'NDB_1060', 'NDB_779']\n",
      "62 -> ['NDB_746', 'NDB_905', 'NDB_887']\n",
      "620 -> ['NDB_643', 'NDB_868', 'NDB_734']\n",
      "621 -> ['NDB_716', 'NDB_754', 'NDB_940']\n",
      "622 -> ['NDB_773', 'NDB_758', 'NDB_929']\n",
      "623 -> ['NDB_826', 'NDB_946', 'NDB_641']\n",
      "624 -> ['NDB_728', 'NDB_901', 'NDB_887']\n",
      "625 -> ['NDB_710', 'NDB_875', 'NDB_989']\n",
      "626 -> ['NDB_683', 'NDB_812', 'NDB_935']\n",
      "627 -> ['NDB_746', 'NDB_647', 'NDB_824']\n",
      "628 -> ['NDB_652', 'NDB_680', 'NDB_1066']\n",
      "629 -> ['NDB_713', 'NDB_866', 'NDB_926']\n",
      "63 -> ['NDB_965', 'NDB_631', 'NDB_989']\n",
      "64 -> ['NDB_863', 'NDB_641', 'NDB_862']\n",
      "645 -> ['NDB_1027', 'NDB_698', 'NDB_872']\n",
      "646 -> ['NDB_865', 'NDB_718', 'NDB_743']\n",
      "647 -> ['NDB_755', 'NDB_881', 'NDB_706']\n",
      "648 -> ['NDB_634', 'NDB_863', 'NDB_878']\n",
      "649 -> ['NDB_643', 'NDB_1033', 'NDB_941']\n",
      "65 -> ['NDB_826', 'NDB_1033', 'NDB_878']\n",
      "650 -> ['NDB_725', 'NDB_836', 'NDB_818']\n",
      "651 -> ['NDB_631', 'NDB_653', 'NDB_751']\n",
      "652 -> ['NDB_974', 'NDB_838', 'NDB_958']\n",
      "653 -> ['NDB_805', 'NDB_889', 'NDB_712']\n",
      "654 -> ['NDB_784', 'NDB_835', 'NDB_637']\n",
      "655 -> ['NDB_773', 'NDB_860', 'NDB_1075']\n",
      "656 -> ['NDB_730', 'NDB_853', 'NDB_994']\n",
      "657 -> ['NDB_715', 'NDB_653', 'NDB_934']\n",
      "658 -> ['NDB_815', 'NDB_752', 'NDB_899']\n",
      "659 -> ['NDB_832', 'NDB_896', 'NDB_862']\n",
      "66 -> ['NDB_817', 'NDB_653', 'NDB_872']\n",
      "660 -> ['NDB_760', 'NDB_661', 'NDB_665']\n",
      "662 -> ['NDB_817', 'NDB_637', 'NDB_925']\n",
      "663 -> ['NDB_826', 'NDB_961', 'NDB_1060']\n",
      "664 -> ['NDB_716', 'NDB_641', 'NDB_926']\n",
      "665 -> ['NDB_754', 'NDB_941', 'NDB_958']\n",
      "666 -> ['NDB_718', 'NDB_868', 'NDB_836']\n",
      "667 -> ['NDB_971', 'NDB_866', 'NDB_779']\n",
      "668 -> ['NDB_652', 'NDB_1027', 'NDB_737']\n",
      "669 -> ['NDB_778', 'NDB_905', 'NDB_730']\n",
      "67 -> ['NDB_847', 'NDB_715', 'NDB_836']\n",
      "670 -> ['NDB_865', 'NDB_694', 'NDB_977']\n",
      "671 -> ['NDB_856', 'NDB_680', 'NDB_742']\n",
      "672 -> ['NDB_647', 'NDB_788', 'NDB_1028']\n",
      "673 -> ['NDB_710', 'NDB_847', 'NDB_880']\n",
      "674 -> ['NDB_901', 'NDB_721', 'NDB_824']\n",
      "675 -> ['NDB_755', 'NDB_790', 'NDB_734']\n",
      "676 -> ['NDB_683', 'NDB_844', 'NDB_883']\n",
      "677 -> ['NDB_812', 'NDB_872', 'NDB_910']\n",
      "678 -> ['NDB_917', 'NDB_706', 'NDB_929']\n",
      "679 -> ['NDB_965', 'NDB_728', 'NDB_875']\n",
      "68 -> ['NDB_680', 'NDB_866', 'NDB_706']\n",
      "680 -> ['NDB_866', 'NDB_721', 'NDB_994']\n",
      "681 -> ['NDB_844', 'NDB_752', 'NDB_1028']\n",
      "682 -> ['NDB_881', 'NDB_665', 'NDB_637']\n",
      "683 -> ['NDB_755', 'NDB_862', 'NDB_742']\n",
      "684 -> ['NDB_710', 'NDB_863', 'NDB_788']\n",
      "685 -> ['NDB_683', 'NDB_856', 'NDB_698']\n",
      "686 -> ['NDB_652', 'NDB_989', 'NDB_940']\n",
      "687 -> ['NDB_728', 'NDB_883', 'NDB_805']\n",
      "688 -> ['NDB_715', 'NDB_730', 'NDB_946']\n",
      "689 -> ['NDB_713', 'NDB_832', 'NDB_751']\n",
      "69 -> ['NDB_790', 'NDB_853', 'NDB_694']\n",
      "690 -> ['NDB_643', 'NDB_760', 'NDB_773']\n",
      "691 -> ['NDB_737', 'NDB_866', 'NDB_889']\n",
      "692 -> ['NDB_974', 'NDB_746', 'NDB_910']\n",
      "693 -> ['NDB_865', 'NDB_860', 'NDB_631']\n",
      "694 -> ['NDB_661', 'NDB_878', 'NDB_734']\n",
      "695 -> ['NDB_901', 'NDB_905', 'NDB_853']\n",
      "696 -> ['NDB_847', 'NDB_941', 'NDB_647']\n",
      "697 -> ['NDB_1027', 'NDB_716', 'NDB_934']\n",
      "698 -> ['NDB_971', 'NDB_838', 'NDB_1028']\n",
      "699 -> ['NDB_790', 'NDB_653', 'NDB_706']\n",
      "7 -> ['NDB_974', 'NDB_754', 'NDB_634']\n",
      "70 -> ['NDB_725', 'NDB_875', 'NDB_716']\n",
      "701 -> ['NDB_784', 'NDB_728', 'NDB_1075']\n",
      "702 -> ['NDB_743', 'NDB_836', 'NDB_929']\n",
      "703 -> ['NDB_965', 'NDB_778', 'NDB_824']\n",
      "704 -> ['NDB_917', 'NDB_754', 'NDB_730']\n",
      "705 -> ['NDB_880', 'NDB_896', 'NDB_889']\n",
      "706 -> ['NDB_1060', 'NDB_958', 'NDB_872']\n",
      "707 -> ['NDB_760', 'NDB_721', 'NDB_751']\n",
      "708 -> ['NDB_815', 'NDB_631', 'NDB_641']\n",
      "709 -> ['NDB_643', 'NDB_746', 'NDB_788']\n",
      "71 -> ['NDB_865', 'NDB_637', 'NDB_742']\n",
      "710 -> ['NDB_961', 'NDB_862', 'NDB_712']\n",
      "711 -> ['NDB_634', 'NDB_805', 'NDB_899']\n",
      "712 -> ['NDB_868', 'NDB_698', 'NDB_940']\n",
      "713 -> ['NDB_1033', 'NDB_935', 'NDB_910']\n",
      "714 -> ['NDB_718', 'NDB_661', 'NDB_812']\n",
      "715 -> ['NDB_755', 'NDB_725', 'NDB_710']\n",
      "716 -> ['NDB_713', 'NDB_887', 'NDB_838']\n",
      "717 -> ['NDB_901', 'NDB_989', 'NDB_665']\n",
      "718 -> ['NDB_683', 'NDB_974', 'NDB_925']\n",
      "719 -> ['NDB_652', 'NDB_863', 'NDB_637']\n",
      "72 -> ['NDB_856', 'NDB_779', 'NDB_958']\n",
      "720 -> ['NDB_875', 'NDB_680', 'NDB_835']\n",
      "721 -> ['NDB_716', 'NDB_977', 'NDB_742']\n",
      "722 -> ['NDB_878', 'NDB_946', 'NDB_706']\n",
      "723 -> ['NDB_865', 'NDB_832', 'NDB_734']\n",
      "724 -> ['NDB_647', 'NDB_994', 'NDB_694']\n",
      "725 -> ['NDB_971', 'NDB_860', 'NDB_853']\n",
      "726 -> ['NDB_758', 'NDB_779', 'NDB_926']\n",
      "727 -> ['NDB_826', 'NDB_737', 'NDB_883']\n",
      "728 -> ['NDB_881', 'NDB_779', 'NDB_818']\n",
      "729 -> ['NDB_817', 'NDB_773', 'NDB_1066']\n",
      "73 -> ['NDB_778', 'NDB_868', 'NDB_1027']\n",
      "730 -> ['NDB_784', 'NDB_743', 'NDB_752']\n",
      "731 -> ['NDB_1033', 'NDB_653', 'NDB_712']\n",
      "732 -> ['NDB_680', 'NDB_1075', 'NDB_880']\n",
      "733 -> ['NDB_965', 'NDB_844', 'NDB_941']\n",
      "734 -> ['NDB_718', 'NDB_778', 'NDB_925']\n",
      "735 -> ['NDB_868', 'NDB_835', 'NDB_824']\n",
      "736 -> ['NDB_815', 'NDB_961', 'NDB_958']\n",
      "737 -> ['NDB_917', 'NDB_641', 'NDB_872']\n",
      "738 -> ['NDB_812', 'NDB_634', 'NDB_1060']\n",
      "739 -> ['NDB_754', 'NDB_899', 'NDB_934']\n",
      "74 -> ['NDB_743', 'NDB_788', 'NDB_910']\n",
      "740 -> ['NDB_790', 'NDB_887', 'NDB_977']\n",
      "741 -> ['NDB_847', 'NDB_875', 'NDB_935']\n",
      "742 -> ['NDB_905', 'NDB_929', 'NDB_926']\n",
      "743 -> ['NDB_1027', 'NDB_896', 'NDB_836']\n",
      "744 -> ['NDB_856', 'NDB_715', 'NDB_866']\n",
      "745 -> ['NDB_725', 'NDB_694', 'NDB_1028']\n",
      "748 -> ['NDB_718', 'NDB_887', 'NDB_896']\n",
      "749 -> ['NDB_715', 'NDB_863', 'NDB_1066']\n",
      "75 -> ['NDB_755', 'NDB_760', 'NDB_929']\n",
      "750 -> ['NDB_680', 'NDB_730', 'NDB_958']\n",
      "751 -> ['NDB_713', 'NDB_728', 'NDB_889']\n",
      "752 -> ['NDB_643', 'NDB_880', 'NDB_751']\n",
      "753 -> ['NDB_815', 'NDB_1027', 'NDB_1075']\n",
      "754 -> ['NDB_812', 'NDB_899', 'NDB_926']\n",
      "755 -> ['NDB_754', 'NDB_929', 'NDB_712']\n",
      "756 -> ['NDB_661', 'NDB_1033', 'NDB_946']\n",
      "757 -> ['NDB_760', 'NDB_844', 'NDB_974']\n",
      "758 -> ['NDB_725', 'NDB_878', 'NDB_935']\n",
      "759 -> ['NDB_778', 'NDB_1060', 'NDB_734']\n",
      "76 -> ['NDB_1060', 'NDB_896', 'NDB_712']\n",
      "760 -> ['NDB_805', 'NDB_881', 'NDB_788']\n",
      "761 -> ['NDB_784', 'NDB_853', 'NDB_824']\n",
      "762 -> ['NDB_847', 'NDB_716', 'NDB_866']\n",
      "763 -> ['NDB_917', 'NDB_758', 'NDB_752']\n",
      "764 -> ['NDB_883', 'NDB_721', 'NDB_836']\n",
      "765 -> ['NDB_971', 'NDB_718', 'NDB_875']\n",
      "766 -> ['NDB_652', 'NDB_730', 'NDB_910']\n",
      "767 -> ['NDB_989', 'NDB_647', 'NDB_742']\n",
      "768 -> ['NDB_737', 'NDB_665', 'NDB_940']\n",
      "769 -> ['NDB_815', 'NDB_661', 'NDB_926']\n",
      "77 -> ['NDB_961', 'NDB_860', 'NDB_881']\n",
      "770 -> ['NDB_725', 'NDB_889', 'NDB_751']\n",
      "774 -> ['NDB_812', 'NDB_653', 'NDB_929']\n",
      "775 -> ['NDB_826', 'NDB_683', 'NDB_773']\n",
      "776 -> ['NDB_760', 'NDB_715', 'NDB_887']\n",
      "777 -> ['NDB_716', 'NDB_941', 'NDB_935']\n",
      "778 -> ['NDB_817', 'NDB_698', 'NDB_994']\n",
      "779 -> ['NDB_713', 'NDB_853', 'NDB_788']\n",
      "78 -> ['NDB_784', 'NDB_862', 'NDB_872']\n",
      "780 -> ['NDB_901', 'NDB_637', 'NDB_818']\n",
      "782 -> ['NDB_965', 'NDB_1060', 'NDB_694']\n",
      "783 -> ['NDB_917', 'NDB_734', 'NDB_1028']\n",
      "784 -> ['NDB_847', 'NDB_631', 'NDB_805']\n",
      "785 -> ['NDB_784', 'NDB_865', 'NDB_778']\n",
      "786 -> ['NDB_755', 'NDB_754', 'NDB_905']\n",
      "787 -> ['NDB_746', 'NDB_860', 'NDB_880']\n",
      "788 -> ['NDB_634', 'NDB_832', 'NDB_958']\n",
      "789 -> ['NDB_856', 'NDB_977', 'NDB_925']\n",
      "79 -> ['NDB_974', 'NDB_994', 'NDB_751']\n",
      "790 -> ['NDB_743', 'NDB_1075', 'NDB_835']\n",
      "791 -> ['NDB_844', 'NDB_863', 'NDB_779']\n",
      "792 -> ['NDB_728', 'NDB_1066', 'NDB_862']\n",
      "793 -> ['NDB_974', 'NDB_961', 'NDB_752']\n",
      "794 -> ['NDB_1033', 'NDB_838', 'NDB_706']\n",
      "795 -> ['NDB_866', 'NDB_641', 'NDB_881']\n",
      "796 -> ['NDB_790', 'NDB_878', 'NDB_824']\n",
      "797 -> ['NDB_710', 'NDB_832', 'NDB_934']\n",
      "798 -> ['NDB_868', 'NDB_680', 'NDB_872']\n",
      "799 -> ['NDB_901', 'NDB_1027', 'NDB_712']\n",
      "8 -> ['NDB_661', 'NDB_901', 'NDB_835']\n",
      "80 -> ['NDB_718', 'NDB_737', 'NDB_721']\n",
      "800 -> ['NDB_896', 'NDB_946', 'NDB_1028']\n",
      "801 -> ['NDB_643', 'NDB_899', 'NDB_637']\n",
      "802 -> ['NDB_631', 'NDB_941', 'NDB_818']\n",
      "803 -> ['NDB_965', 'NDB_994', 'NDB_836']\n",
      "804 -> ['NDB_758', 'NDB_706', 'NDB_694']\n",
      "805 -> ['NDB_865', 'NDB_817', 'NDB_940']\n",
      "806 -> ['NDB_755', 'NDB_971', 'NDB_665']\n",
      "807 -> ['NDB_737', 'NDB_698', 'NDB_910']\n",
      "808 -> ['NDB_652', 'NDB_835', 'NDB_742']\n",
      "809 -> ['NDB_773', 'NDB_721', 'NDB_647']\n",
      "81 -> ['NDB_812', 'NDB_880', 'NDB_977']\n",
      "810 -> ['NDB_856', 'NDB_883', 'NDB_653']\n",
      "811 -> ['NDB_826', 'NDB_989', 'NDB_977']\n",
      "812 -> ['NDB_683', 'NDB_875', 'NDB_860']\n",
      "813 -> ['NDB_746', 'NDB_737', 'NDB_899']\n",
      "814 -> ['NDB_896', 'NDB_721', 'NDB_958']\n",
      "815 -> ['NDB_974', 'NDB_856', 'NDB_641']\n",
      "816 -> ['NDB_661', 'NDB_860', 'NDB_883']\n",
      "817 -> ['NDB_715', 'NDB_665', 'NDB_910']\n",
      "819 -> ['NDB_917', 'NDB_634', 'NDB_862']\n",
      "820 -> ['NDB_710', 'NDB_965', 'NDB_818']\n",
      "821 -> ['NDB_643', 'NDB_716', 'NDB_853']\n",
      "822 -> ['NDB_865', 'NDB_878', 'NDB_925']\n",
      "823 -> ['NDB_1060', 'NDB_752', 'NDB_836']\n",
      "824 -> ['NDB_760', 'NDB_863', 'NDB_835']\n",
      "825 -> ['NDB_743', 'NDB_994', 'NDB_742']\n",
      "826 -> ['NDB_875', 'NDB_838', 'NDB_712']\n",
      "827 -> ['NDB_1033', 'NDB_887', 'NDB_832']\n",
      "828 -> ['NDB_754', 'NDB_946', 'NDB_779']\n",
      "829 -> ['NDB_713', 'NDB_773', 'NDB_698']\n",
      "83 -> ['NDB_835', 'NDB_946', 'NDB_836']\n",
      "832 -> ['NDB_718', 'NDB_790', 'NDB_889']\n",
      "833 -> ['NDB_725', 'NDB_812', 'NDB_715']\n",
      "834 -> ['NDB_868', 'NDB_730', 'NDB_1066']\n",
      "835 -> ['NDB_728', 'NDB_844', 'NDB_751']\n",
      "836 -> ['NDB_817', 'NDB_863', 'NDB_694']\n",
      "837 -> ['NDB_961', 'NDB_634', 'NDB_758']\n",
      "838 -> ['NDB_883', 'NDB_880', 'NDB_958']\n",
      "839 -> ['NDB_661', 'NDB_721', 'NDB_940']\n",
      "84 -> ['NDB_698', 'NDB_805', 'NDB_838']\n",
      "840 -> ['NDB_647', 'NDB_665', 'NDB_977']\n",
      "841 -> ['NDB_710', 'NDB_680', 'NDB_653']\n",
      "843 -> ['NDB_971', 'NDB_815', 'NDB_905']\n",
      "844 -> ['NDB_826', 'NDB_974', 'NDB_818']\n",
      "845 -> ['NDB_652', 'NDB_716', 'NDB_1075']\n",
      "846 -> ['NDB_743', 'NDB_790', 'NDB_872']\n",
      "847 -> ['NDB_755', 'NDB_965', 'NDB_878']\n",
      "848 -> ['NDB_917', 'NDB_694', 'NDB_926']\n",
      "849 -> ['NDB_773', 'NDB_730', 'NDB_935']\n",
      "85 -> ['NDB_815', 'NDB_883', 'NDB_694']\n",
      "850 -> ['NDB_853', 'NDB_734', 'NDB_925']\n",
      "851 -> ['NDB_752', 'NDB_994', 'NDB_824']\n",
      "852 -> ['NDB_778', 'NDB_706', 'NDB_862']\n",
      "853 -> ['NDB_856', 'NDB_631', 'NDB_788']\n",
      "854 -> ['NDB_683', 'NDB_641', 'NDB_1028']\n",
      "855 -> ['NDB_805', 'NDB_989', 'NDB_934']\n",
      "856 -> ['NDB_746', 'NDB_901', 'NDB_929']\n",
      "857 -> ['NDB_643', 'NDB_866', 'NDB_896']\n",
      "858 -> ['NDB_784', 'NDB_899', 'NDB_910']\n",
      "859 -> ['NDB_760', 'NDB_737', 'NDB_637']\n",
      "86 -> ['NDB_641', 'NDB_1066', 'NDB_889']\n",
      "860 -> ['NDB_860', 'NDB_941', 'NDB_872']\n",
      "861 -> ['NDB_847', 'NDB_1027', 'NDB_881']\n",
      "862 -> ['NDB_865', 'NDB_1060', 'NDB_788']\n",
      "863 -> ['NDB_817', 'NDB_706', 'NDB_935']\n",
      "864 -> ['NDB_838', 'NDB_734', 'NDB_824']\n",
      "865 -> ['NDB_1033', 'NDB_758', 'NDB_977']\n",
      "866 -> ['NDB_1075', 'NDB_989', 'NDB_779']\n",
      "867 -> ['NDB_812', 'NDB_698', 'NDB_1066']\n",
      "868 -> ['NDB_826', 'NDB_836', 'NDB_934']\n",
      "869 -> ['NDB_905', 'NDB_835', 'NDB_889']\n",
      "87 -> ['NDB_758', 'NDB_899', 'NDB_941']\n",
      "870 -> ['NDB_683', 'NDB_880', 'NDB_742']\n",
      "871 -> ['NDB_815', 'NDB_868', 'NDB_754']\n",
      "872 -> ['NDB_844', 'NDB_832', 'NDB_647']\n",
      "873 -> ['NDB_652', 'NDB_946', 'NDB_751']\n",
      "874 -> ['NDB_755', 'NDB_713', 'NDB_718']\n",
      "875 -> ['NDB_971', 'NDB_728', 'NDB_653']\n",
      "876 -> ['NDB_887', 'NDB_929', 'NDB_940']\n",
      "877 -> ['NDB_725', 'NDB_680', 'NDB_712']\n",
      "878 -> ['NDB_961', 'NDB_875', 'NDB_926']\n",
      "88 -> ['NDB_710', 'NDB_824', 'NDB_926']\n",
      "888 -> ['NDB_917', 'NDB_1033', 'NDB_1060']\n",
      "889 -> ['NDB_743', 'NDB_881', 'NDB_734']\n",
      "89 -> ['NDB_683', 'NDB_1060', 'NDB_818']\n",
      "890 -> ['NDB_905', 'NDB_838', 'NDB_977']\n",
      "891 -> ['NDB_631', 'NDB_880', 'NDB_925']\n",
      "892 -> ['NDB_716', 'NDB_989', 'NDB_910']\n",
      "893 -> ['NDB_715', 'NDB_641', 'NDB_721']\n",
      "894 -> ['NDB_643', 'NDB_728', 'NDB_961']\n",
      "895 -> ['NDB_835', 'NDB_941', 'NDB_862']\n",
      "896 -> ['NDB_883', 'NDB_758', 'NDB_751']\n",
      "897 -> ['NDB_917', 'NDB_1075', 'NDB_889']\n",
      "898 -> ['NDB_971', 'NDB_887', 'NDB_946']\n",
      "899 -> ['NDB_773', 'NDB_680', 'NDB_752']\n",
      "9 -> ['NDB_878', 'NDB_836', 'NDB_742']\n",
      "90 -> ['NDB_652', 'NDB_856', 'NDB_896']\n",
      "900 -> ['NDB_817', 'NDB_958', 'NDB_824']\n",
      "901 -> ['NDB_652', 'NDB_647', 'NDB_935']\n",
      "902 -> ['NDB_683', 'NDB_965', 'NDB_737']\n",
      "903 -> ['NDB_826', 'NDB_853', 'NDB_838']\n",
      "904 -> ['NDB_713', 'NDB_710', 'NDB_872']\n",
      "905 -> ['NDB_634', 'NDB_836', 'NDB_940']\n",
      "906 -> ['NDB_760', 'NDB_866', 'NDB_934']\n",
      "907 -> ['NDB_743', 'NDB_974', 'NDB_926']\n",
      "908 -> ['NDB_865', 'NDB_754', 'NDB_896']\n",
      "909 -> ['NDB_718', 'NDB_641', 'NDB_899']\n",
      "91 -> ['NDB_901', 'NDB_647', 'NDB_958']\n",
      "910 -> ['NDB_725', 'NDB_790', 'NDB_868']\n",
      "911 -> ['NDB_728', 'NDB_680', 'NDB_665']\n",
      "912 -> ['NDB_961', 'NDB_856', 'NDB_941']\n",
      "913 -> ['NDB_716', 'NDB_698', 'NDB_712']\n",
      "914 -> ['NDB_815', 'NDB_878', 'NDB_832']\n",
      "915 -> ['NDB_746', 'NDB_661', 'NDB_779']\n",
      "916 -> ['NDB_875', 'NDB_880', 'NDB_694']\n",
      "917 -> ['NDB_755', 'NDB_1060', 'NDB_994']\n",
      "918 -> ['NDB_784', 'NDB_730', 'NDB_788']\n",
      "919 -> ['NDB_860', 'NDB_905', 'NDB_818']\n",
      "92 -> ['NDB_1075', 'NDB_905', 'NDB_934']\n",
      "920 -> ['NDB_812', 'NDB_863', 'NDB_1028']\n",
      "921 -> ['NDB_901', 'NDB_805', 'NDB_1066']\n",
      "922 -> ['NDB_847', 'NDB_653', 'NDB_742']\n",
      "923 -> ['NDB_896', 'NDB_637', 'NDB_929']\n",
      "924 -> ['NDB_844', 'NDB_665', 'NDB_836']\n",
      "925 -> ['NDB_965', 'NDB_1027', 'NDB_734']\n",
      "926 -> ['NDB_652', 'NDB_853', 'NDB_706']\n",
      "927 -> ['NDB_683', 'NDB_778', 'NDB_661']\n",
      "928 -> ['NDB_718', 'NDB_977', 'NDB_934']\n",
      "929 -> ['NDB_710', 'NDB_856', 'NDB_1033']\n",
      "93 -> ['NDB_965', 'NDB_715', 'NDB_832']\n",
      "930 -> ['NDB_899', 'NDB_881', 'NDB_925']\n",
      "931 -> ['NDB_971', 'NDB_790', 'NDB_883']\n",
      "932 -> ['NDB_773', 'NDB_751', 'NDB_824']\n",
      "933 -> ['NDB_715', 'NDB_878', 'NDB_862']\n",
      "934 -> ['NDB_1075', 'NDB_887', 'NDB_647']\n",
      "935 -> ['NDB_784', 'NDB_817', 'NDB_989']\n",
      "94 -> ['NDB_643', 'NDB_634', 'NDB_989']\n",
      "949 -> ['NDB_643', 'NDB_994', 'NDB_889']\n",
      "95 -> ['NDB_730', 'NDB_665', 'NDB_734']\n",
      "950 -> ['NDB_815', 'NDB_665', 'NDB_751']\n",
      "951 -> ['NDB_875', 'NDB_752', 'NDB_977']\n",
      "952 -> ['NDB_974', 'NDB_868', 'NDB_862']\n",
      "953 -> ['NDB_728', 'NDB_779', 'NDB_712']\n",
      "954 -> ['NDB_784', 'NDB_661', 'NDB_694']\n",
      "955 -> ['NDB_683', 'NDB_917', 'NDB_647']\n",
      "956 -> ['NDB_631', 'NDB_730', 'NDB_836']\n",
      "957 -> ['NDB_716', 'NDB_634', 'NDB_653']\n",
      "958 -> ['NDB_725', 'NDB_866', 'NDB_946']\n",
      "959 -> ['NDB_755', 'NDB_746', 'NDB_935']\n",
      "96 -> ['NDB_778', 'NDB_754', 'NDB_737']\n",
      "960 -> ['NDB_878', 'NDB_989', 'NDB_838']\n",
      "961 -> ['NDB_728', 'NDB_896', 'NDB_788']\n",
      "962 -> ['NDB_1060', 'NDB_832', 'NDB_706']\n",
      "963 -> ['NDB_883', 'NDB_863', 'NDB_941']\n",
      "964 -> ['NDB_715', 'NDB_637', 'NDB_926']\n",
      "965 -> ['NDB_865', 'NDB_961', 'NDB_899']\n",
      "966 -> ['NDB_1027', 'NDB_647', 'NDB_929']\n",
      "967 -> ['NDB_826', 'NDB_901', 'NDB_698']\n",
      "968 -> ['NDB_773', 'NDB_716', 'NDB_805']\n",
      "969 -> ['NDB_875', 'NDB_1028', 'NDB_925']\n",
      "970 -> ['NDB_853', 'NDB_751', 'NDB_712']\n",
      "971 -> ['NDB_643', 'NDB_835', 'NDB_934']\n",
      "972 -> ['NDB_713', 'NDB_860', 'NDB_653']\n",
      "973 -> ['NDB_1033', 'NDB_641', 'NDB_940']\n",
      "974 -> ['NDB_721', 'NDB_889', 'NDB_872']\n",
      "975 -> ['NDB_817', 'NDB_1075', 'NDB_818']\n",
      "976 -> ['NDB_758', 'NDB_866', 'NDB_730']\n",
      "977 -> ['NDB_989', 'NDB_935', 'NDB_958']\n",
      "978 -> ['NDB_760', 'NDB_836', 'NDB_926']\n",
      "979 -> ['NDB_847', 'NDB_965', 'NDB_868']\n",
      "98 -> ['NDB_755', 'NDB_844', 'NDB_946']\n",
      "980 -> ['NDB_773', 'NDB_856', 'NDB_734']\n",
      "981 -> ['NDB_917', 'NDB_901', 'NDB_680']\n",
      "982 -> ['NDB_835', 'NDB_694', 'NDB_910']\n",
      "983 -> ['NDB_971', 'NDB_743', 'NDB_1027']\n",
      "984 -> ['NDB_652', 'NDB_718', 'NDB_815']\n",
      "985 -> ['NDB_737', 'NDB_742', 'NDB_824']\n",
      "986 -> ['NDB_844', 'NDB_754', 'NDB_706']\n",
      "987 -> ['NDB_880', 'NDB_994', 'NDB_1066']\n",
      "988 -> ['NDB_710', 'NDB_974', 'NDB_905']\n",
      "989 -> ['NDB_760', 'NDB_790', 'NDB_812']\n",
      "990 -> ['NDB_778', 'NDB_887', 'NDB_881']\n",
      "991 -> ['NDB_713', 'NDB_941', 'NDB_779']\n",
      "992 -> ['NDB_1060', 'NDB_862', 'NDB_910']\n",
      "993 -> ['NDB_860', 'NDB_715', 'NDB_977']\n",
      "994 -> ['NDB_680', 'NDB_899', 'NDB_929']\n",
      "995 -> ['NDB_758', 'NDB_788', 'NDB_958']\n",
      "996 -> ['NDB_725', 'NDB_805', 'NDB_752']\n",
      "997 -> ['NDB_965', 'NDB_946', 'NDB_665']\n",
      "999 -> ['NDB_961', 'NDB_838', 'NDB_940']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "check_readout_dict = pickle.load(open(os.path.join(library_folder, 'readout_usage.pkl'), 'rb'))\n",
    "\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(full_records, check_readout_dict, \n",
    "                                                                     total_readout_num=num_on_bits)\n",
    "print(reg2readout_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3e64afd0",
   "metadata": {},
   "source": [
    "### 4.1.4 check readout usage in the library"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "id": "1647e94f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- extracting probes from >region:0, region:1, region:10, region:100, region:1000, region:1001, region:1002, region:1003, region:1004, region:1005, region:101, region:102, region:1020, region:1023, region:1024, region:1025, region:1029, region:103, region:1030, region:1031, region:1032, region:1033, region:1034, region:1035, region:1036, region:1037, region:1038, region:1039, region:104, region:1040, region:1041, region:1042, region:1043, region:1044, region:1045, region:1046, region:1047, region:1048, region:1049, region:105, region:1050, region:1051, region:1052, region:1053, region:1054, region:1055, region:1056, region:1057, region:1058, region:1059, region:106, region:1060, region:1061, region:1062, region:1063, region:1064, region:1065, region:1066, region:1067, region:1068, region:1069, region:107, region:1070, region:1071, region:1072, region:1073, region:1074, region:1075, region:1076, region:1077, region:1078, region:1079, region:108, region:1080, region:1081, region:1082, region:1083, region:1085, region:1087, region:1089, region:109, region:1090, region:1091, region:1092, region:1093, region:1094, region:1097, region:11, region:110, region:1105, region:1106, region:1107, region:1108, region:1109, region:111, region:1110, region:1111, region:1112, region:1113, region:1114, region:1115, region:1116, region:1117, region:1118, region:1119, region:112, region:1120, region:1121, region:1122, region:1123, region:1124, region:1125, region:1126, region:1127, region:1128, region:1129, region:1130, region:1131, region:1132, region:1133, region:1134, region:1135, region:1136, region:1137, region:1138, region:1139, region:114, region:1140, region:1141, region:1142, region:1143, region:1144, region:1145, region:1146, region:1147, region:1148, region:115, region:1152, region:1153, region:1154, region:1155, region:1156, region:1157, region:116, region:1160, region:1162, region:1163, region:1165, region:1166, region:1167, region:1168, region:1169, region:117, region:1170, region:1171, region:1172, region:1173, region:1174, region:1176, region:1177, region:1178, region:1179, region:118, region:1180, region:1181, region:1182, region:1183, region:1184, region:1186, region:1187, region:1188, region:1189, region:119, region:1190, region:1191, region:1192, region:1193, region:1194, region:1195, region:1196, region:1197, region:12, region:120, region:1203, region:1204, region:1205, region:1206, region:1207, region:1208, region:1209, region:121, region:1210, region:1211, region:1212, region:1213, region:1214, region:1215, region:1216, region:1217, region:1218, region:1219, region:122, region:1220, region:1221, region:1222, region:1224, region:1225, region:1226, region:1227, region:1228, region:1229, region:123, region:1230, region:1231, region:1232, region:1233, region:1234, region:1235, region:1236, region:1237, region:1238, region:1239, region:124, region:1240, region:1241, region:1242, region:1243, region:1244, region:1245, region:1246, region:1247, region:1248, region:1249, region:125, region:1250, region:1251, region:1252, region:1253, region:1254, region:1255, region:1256, region:1257, region:1258, region:1259, region:126, region:1260, region:1261, region:1262, region:1263, region:1264, region:1265, region:1269, region:127, region:1270, region:1271, region:1272, region:1273, region:1274, region:1275, region:1276, region:1277, region:1278, region:1279, region:128, region:1280, region:1281, region:1282, region:1284, region:1285, region:1286, region:1287, region:1288, region:1289, region:129, region:1290, region:1291, region:1292, region:1293, region:1294, region:1295, region:1297, region:1298, region:1299, region:13, region:130, region:1300, region:1301, region:1302, region:1303, region:1304, region:1305, region:1306, region:1307, region:1308, region:1309, region:131, region:1310, region:1311, region:1312, region:1313, region:1314, region:1315, region:1316, region:1317, region:1318, region:1319, region:132, region:1320, region:1321, region:1322, region:1323, region:1324, region:1325, region:133, region:1330, region:1331, region:1332, region:1333, region:1334, region:1335, region:1336, region:1337, region:1338, region:1339, region:134, region:1340, region:1341, region:1342, region:1343, region:1344, region:1345, region:1346, region:1347, region:1348, region:1349, region:1350, region:1351, region:1352, region:1353, region:1354, region:1355, region:1356, region:1357, region:1358, region:1359, region:1360, region:1361, region:1362, region:1363, region:137, region:1378, region:1379, region:138, region:1380, region:1381, region:1382, region:1383, region:1384, region:1385, region:1386, region:1387, region:1388, region:1389, region:139, region:1390, region:1391, region:1392, region:1393, region:1394, region:1395, region:1396, region:1398, region:1399, region:14, region:140, region:1400, region:1401, region:1402, region:1403, region:1404, region:1405, region:1406, region:1408, region:1409, region:141, region:1427, region:1429, region:143, region:1430, region:1434, region:1435, region:1436, region:1437, region:1438, region:1439, region:144, region:1440, region:1441, region:1442, region:1443, region:1444, region:1445, region:1446, region:1447, region:1448, region:1449, region:145, region:1450, region:1451, region:1452, region:1453, region:1454, region:1455, region:1456, region:1457, region:1458, region:1459, region:146, region:1461, region:1462, region:1463, region:1464, region:1465, region:1466, region:1467, region:1468, region:1469, region:147, region:1470, region:1471, region:1472, region:1473, region:1474, region:1475, region:1476, region:1477, region:1478, region:1479, region:148, region:1480, region:1481, region:1482, region:1485, region:1486, region:1487, region:1488, region:1489, region:149, region:1490, region:1491, region:1492, region:1493, region:1494, region:1495, region:1496, region:1497, region:1498, region:1499, region:15, region:150, region:1500, region:1501, region:1502, region:1503, region:1504, region:1505, region:1506, region:1507, region:1508, region:1509, region:1510, region:1511, region:1512, region:1513, region:1514, region:1515, region:1516, region:1517, region:1518, region:1519, region:152, region:1520, region:1521, region:1525, region:1526, region:1527, region:1529, region:153, region:1530, region:1533, region:1534, region:1535, region:1536, region:1537, region:1538, region:154, region:1541, region:1542, region:1548, region:155, region:156, region:157, region:158, region:159, region:16, region:160, region:161, region:162, region:163, region:164, region:165, region:166, region:167, region:168, region:169, region:17, region:170, region:171, region:172, region:173, region:174, region:175, region:176, region:177, region:178, region:179, region:18, region:180, region:181, region:182, region:183, region:184, region:185, region:186, region:187, region:188, region:189, region:19, region:190, region:191, region:192, region:193, region:194, region:195, region:196, region:197, region:198, region:199, region:2, region:20, region:200, region:201, region:202, region:203, region:204, region:205, region:206, region:207, region:208, region:209, region:21, region:210, region:211, region:212, region:213, region:214, region:215, region:216, region:217, region:218, region:219, region:22, region:220, region:221, region:222, region:223, region:224, region:225, region:226, region:227, region:228, region:229, region:23, region:230, region:233, region:234, region:235, region:236, region:237, region:238, region:239, region:24, region:240, region:241, region:242, region:243, region:244, region:245, region:246, region:247, region:248, region:249, region:25, region:250, region:251, region:252, region:253, region:254, region:255, region:256, region:257, region:258, region:259, region:26, region:260, region:261, region:262, region:263, region:264, region:265, region:266, region:267, region:268, region:269, region:27, region:270, region:271, region:272, region:273, region:274, region:275, region:276, region:277, region:278, region:279, region:28, region:280, region:281, region:282, region:283, region:284, region:285, region:286, region:287, region:288, region:289, region:29, region:290, region:291, region:292, region:293, region:295, region:296, region:297, region:298, region:299, region:3, region:30, "
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "region:300, region:302, region:303, region:304, region:305, region:306, region:307, region:308, region:309, region:31, region:310, region:311, region:312, region:313, region:314, region:315, region:316, region:317, region:318, region:319, region:32, region:320, region:321, region:322, region:323, region:324, region:325, region:326, region:327, region:328, region:329, region:33, region:330, region:331, region:332, region:333, region:334, region:335, region:336, region:337, region:338, region:339, region:34, region:340, region:341, region:342, region:343, region:344, region:345, region:346, region:347, region:348, region:349, region:35, region:350, region:351, region:352, region:353, region:354, region:355, region:356, region:357, region:358, region:359, region:36, region:360, region:361, region:362, region:363, region:364, region:365, region:366, region:367, region:368, region:37, region:370, region:371, region:372, region:373, region:374, region:375, region:376, region:377, region:379, region:38, region:380, region:381, region:382, region:383, region:384, region:385, region:386, region:387, region:388, region:389, region:39, region:390, region:391, region:392, region:393, region:394, region:395, region:396, region:397, region:398, region:399, region:4, region:40, region:400, region:401, region:402, region:403, region:404, region:405, region:406, region:407, region:408, region:409, region:410, region:411, region:412, region:413, region:414, region:415, region:416, region:417, region:418, region:419, region:42, region:421, region:422, region:423, region:424, region:425, region:426, region:427, region:428, region:429, region:43, region:430, region:431, region:432, region:435, region:436, region:437, region:438, region:439, region:44, region:440, region:441, region:442, region:443, region:444, region:446, region:447, region:448, region:449, region:45, region:450, region:451, region:452, region:453, region:454, region:455, region:456, region:457, region:458, region:459, region:46, region:460, region:461, region:462, region:463, region:464, region:465, region:466, region:467, region:468, region:469, region:47, region:470, region:471, region:472, region:473, region:474, region:475, region:476, region:477, region:478, region:479, region:480, region:481, region:482, region:483, region:484, region:485, region:486, region:487, region:489, region:490, region:491, region:492, region:493, region:494, region:495, region:496, region:497, region:498, region:499, region:5, region:500, region:501, region:502, region:503, region:504, region:505, region:506, region:507, region:508, region:509, region:510, region:511, region:512, region:516, region:517, region:518, region:519, region:520, region:521, region:522, region:523, region:524, region:525, region:526, region:527, region:528, region:529, region:530, region:531, region:532, region:533, region:534, region:535, region:536, region:537, region:538, region:539, region:540, region:541, region:542, region:543, region:544, region:545, region:546, region:548, region:549, region:550, region:551, region:552, region:554, region:555, region:556, region:557, region:558, region:559, region:560, region:561, region:562, region:563, region:564, region:565, region:566, region:567, region:568, region:569, region:57, region:570, region:571, region:572, region:573, region:574, region:575, region:576, region:577, region:578, region:579, region:580, region:581, region:582, region:583, region:584, region:585, region:586, region:587, region:588, region:589, region:590, region:591, region:592, region:593, region:594, region:595, region:596, region:597, region:598, region:599, region:6, region:60, region:600, region:601, region:602, region:603, region:605, region:606, region:607, region:608, region:609, region:611, region:612, region:613, region:614, region:615, region:616, region:617, region:618, region:619, region:62, region:620, region:621, region:622, region:623, region:624, region:625, region:626, region:627, region:628, region:629, region:63, region:64, region:645, region:646, region:647, region:648, region:649, region:65, region:650, region:651, region:652, region:653, region:654, region:655, region:656, region:657, region:658, region:659, region:66, region:660, region:662, region:663, region:664, region:665, region:666, region:667, region:668, region:669, region:67, region:670, region:671, region:672, region:673, region:674, region:675, region:676, region:677, region:678, region:679, region:68, region:680, region:681, region:682, region:683, region:684, region:685, region:686, region:687, region:688, region:689, region:69, region:690, region:691, region:692, region:693, region:694, region:695, region:696, region:697, region:698, region:699, region:7, region:70, region:701, region:702, region:703, region:704, region:705, region:706, region:707, region:708, region:709, region:71, region:710, region:711, region:712, region:713, region:714, region:715, region:716, region:717, region:718, region:719, region:72, region:720, region:721, region:722, region:723, region:724, region:725, region:726, region:727, region:728, region:729, region:73, region:730, region:731, region:732, region:733, region:734, region:735, region:736, region:737, region:738, region:739, region:74, region:740, region:741, region:742, region:743, region:744, region:745, region:748, region:749, region:75, region:750, region:751, region:752, region:753, region:754, region:755, region:756, region:757, region:758, region:759, region:76, region:760, region:761, region:762, region:763, region:764, region:765, region:766, region:767, region:768, region:769, region:77, region:770, region:774, region:775, region:776, region:777, region:778, region:779, region:78, region:780, region:782, region:783, region:784, region:785, region:786, region:787, region:788, region:789, region:79, region:790, region:791, region:792, region:793, region:794, region:795, region:796, region:797, region:798, region:799, region:8, region:80, region:800, region:801, region:802, region:803, region:804, region:805, region:806, region:807, region:808, region:809, region:81, region:810, region:811, region:812, region:813, region:814, region:815, region:816, region:817, region:819, region:820, region:821, region:822, region:823, region:824, region:825, region:826, region:827, region:828, region:829, region:83, region:832, region:833, region:834, region:835, region:836, region:837, region:838, region:839, region:84, region:840, region:841, region:843, region:844, region:845, region:846, region:847, region:848, region:849, region:85, region:850, region:851, region:852, region:853, region:854, region:855, region:856, region:857, region:858, region:859, region:86, region:860, region:861, region:862, region:863, region:864, region:865, region:866, region:867, region:868, region:869, region:87, region:870, region:871, region:872, region:873, region:874, region:875, region:876, region:877, region:878, region:88, region:888, region:889, region:89, region:890, region:891, region:892, region:893, region:894, region:895, region:896, region:897, region:898, region:899, region:9, region:90, region:900, region:901, region:902, region:903, region:904, region:905, region:906, region:907, region:908, region:909, region:91, region:910, region:911, region:912, region:913, region:914, region:915, region:916, region:917, region:918, region:919, region:92, region:920, region:921, region:922, region:923, region:924, region:925, region:926, region:927, region:928, region:929, region:93, region:930, region:931, region:932, region:933, region:934, region:935, region:94, region:949, region:95, region:950, region:951, region:952, region:953, region:954, region:955, region:956, region:957, region:958, region:959, region:96, region:960, region:961, region:962, region:963, region:964, region:965, region:966, region:967, region:968, region:969, region:970, region:971, region:972, region:973, region:974, region:975, region:976, region:977, region:978, region:979, region:98, region:980, region:981, region:982, region:983, region:984, region:985, region:986, region:987, region:988, region:989, region:990, region:991, region:992, region:993, region:994, region:995, region:996, region:997, region:999, {'NDB_683': ['0', '0', '0', '0', '1004', '1004', '1004', '1004', '1037', '1037', '1037', '1037', '1081', '1081', '1081', '1081', '110', '110', '110', '110', '1107', '1107', '1107', '1107', '1160', '1160', '1160', '1160', '1173', '1173', '1173', '1173', '1211', '1211', '1211', '1211', '1265', '1265', '1265', '1265', '1308', '1308', '1308', '1308', '1355', '1355', '1355', '1355', '1391', '1391', '1391', '1391', '140', '140', '140', '140', '1442', '1442', '1442', '1442', '1476', '1476', '1476', '1476', '1518', '1518', '1518', '1518', '194', '194', '194', '194', '237', '237', '237', '237', '267', '267', '267', '267', '274', '274', '274', '274', '324', '324', '324', '324', '350', '350', '350', '350', '372', '372', '372', '372', '39', '39', '39', '39', '426', '426', '426', '426', '454', '454', '454', '454', '521', '521', '521', '521', '590', '590', '590', '590', '626', '626', '626', '626', '676', '676', '676', '676', '685', '685', '685', '685', '718', '718', '718', '718', '775', '775', '775', '775', '812', '812', '812', '812', '854', '854', '854', '854', '870', '870', '870', '870', '89', '89', '89', '89', '902', '902', '902', '902', '927', '927', '927', '927', '955', '955', '955', '955'], 'NDB_758': ['0', '0', '0', '0', '105', '105', '105', '105', '1056', '1056', '1056', '1056', '1072', '1072', '1072', '1072', '1082', '1082', '1082', '1082', '1111', '1111', '1111', '1111', '1168', '1168', '1168', '1168', '1193', '1193', '1193', '1193', '1221', '1221', '1221', '1221', '1284', '1284', '1284', '1284', '13', '13', '13', '13', '1335', '1335', '1335', '1335', '1363', '1363', '1363', '1363', '1381', '1381', '1381', '1381', '1467', '1467', '1467', '1467', '147', '147', '147', '147', '1500', '1500', '1500', '1500', '1537', '1537', '1537', '1537', '181', '181', '181', '181', '201', '201', '201', '201', '260', '260', '260', '260', '292', '292', '292', '292', '330', '330', '330', '330', '353', '353', '353', '353', '397', '397', '397', '397', '419', '419', '419', '419', '432', '432', '432', '432', '483', '483', '483', '483', '525', '525', '525', '525', '559', '559', '559', '559', '603', '603', '603', '603', '622', '622', '622', '622', '726', '726', '726', '726', '763', '763', '763', '763', '804', '804', '804', '804', '837', '837', '837', '837', '865', '865', '865', '865', '87', '87', '87', '87', '896', '896', '896', '896', '976', '976', '976', '976', '995', '995', '995', '995'], 'NDB_887': ['0', '0', '0', '0', '1005', '1005', '1005', '1005', '1057', '1057', '1057', '1057', '1093', '1093', '1093', '1093', '1140', '1140', '1140', '1140', '1163', '1163', '1163', '1163', '1231', '1231', '1231', '1231', '1250', '1250', '1250', '1250', '126', '126', '126', '126', '1291', '1291', '1291', '1291', '1312', '1312', '1312', '1312', '1362', '1362', '1362', '1362', '1383', '1383', '1383', '1383', '1446', '1446', '1446', '1446', '1495', '1495', '1495', '1495', '16', '16', '16', '16', '164', '164', '164', '164', '200', '200', '200', '200', '217', '217', '217', '217', '277', '277', '277', '277', '291', '291', '291', '291', '314', '314', '314', '314', '366', '366', '366', '366', '411', '411', '411', '411', '427', '427', '427', '427', '457', '457', '457', '457', '530', '530', '530', '530', '548', '548', '548', '548', '573', '573', '573', '573', '608', '608', '608', '608', '62', '62', '62', '62', '624', '624', '624', '624', '716', '716', '716', '716', '740', '740', '740', '740', '748', '748', '748', '748', '776', '776', '776', '776', '827', '827', '827', '827', '876', '876', '876', '876', '898', '898', '898', '898', '934', '934', '934', '934', '990', '990', '990', '990'], 'NDB_718': ['1', '1', '1', '1', '1063', '1063', '1063', '1063', '1083', '1083', '1083', '1083', '1155', '1155', '1155', '1155', '1190', '1190', '1190', '1190', '1239', '1239', '1239', '1239', '1286', '1286', '1286', '1286', '1308', '1308', '1308', '1308', '132', '132', '132', '132', '1325', '1325', '1325', '1325', '1401', '1401', '1401', '1401', '1447', '1447', '1447', '1447', '148', '148', '148', '148', '1494', '1494', '1494', '1494', '1534', '1534', '1534', '1534', '209', '209', '209', '209', '227', '227', '227', '227', '261', '261', '261', '261', '279', '279', '279', '279', '331', '331', '331', '331', '34', '34', '34', '34', '352', '352', '352', '352', '413', '413', '413', '413', '442', '442', '442', '442', '471', '471', '471', '471', '524', '524', '524', '524', '543', '543', '543', '543', '559', '559', '559', '559', '575', '575', '575', '575', '646', '646', '646', '646', '666', '666', '666', '666', '714', '714', '714', '714', '734', '734', '734', '734', '748', '748', '748', '748', '765', '765', '765', '765', '80', '80', '80', '80', '832', '832', '832', '832', '874', '874', '874', '874', '909', '909', '909', '909', '928', '928', '928', '928', '984', '984', '984', '984'], 'NDB_631': ['1', '1', '1', '1', '1001', '1001', '1001', '1001', '1032', '1032', '1032', '1032', '1058', '1058', '1058', '1058', '1120', '1120', '1120', '1120', '114', '114', '114', '114', '1170', '1170', '1170', '1170', '1235', '1235', '1235', '1235', '1252', '1252', '1252', '1252', '1294', '1294', '1294', '1294', '1316', '1316', '1316', '1316', '1334', '1334', '1334', '1334', '14', '14', '14', '14', '1408', '1408', '1408', '1408', '1446', '1446', '1446', '1446', '1478', '1478', '1478', '1478', '1516', '1516', '1516', '1516', '1536', '1536', '1536', '1536', '177', '177', '177', '177', '194', '194', '194', '194', '215', '215', '215', '215', '236', '236', '236', '236', '288', '288', '288', '288', '318', '318', '318', '318', '368', '368', '368', '368', '414', '414', '414', '414', '444', '444', '444', '444', '473', '473', '473', '473', '499', '499', '499', '499', '536', '536', '536', '536', '574', '574', '574', '574', '600', '600', '600', '600', '63', '63', '63', '63', '651', '651', '651', '651', '693', '693', '693', '693', '708', '708', '708', '708', '784', '784', '784', '784', '802', '802', '802', '802', '853', '853', '853', '853', '891', '891', '891', '891', '956', '956', '956', '956'], 'NDB_1066': ['1', '1', '1', '1', '1036', '1036', '1036', '1036', '1073', '1073', '1073', '1073', '1114', '1114', '1114', '1114', '1157', '1157', '1157', '1157', '121', '121', '121', '121', '1232', '1232', '1232', '1232', '1241', '1241', '1241', '1241', '1269', '1269', '1269', '1269', '1285', '1285', '1285', '1285', '1343', '1343', '1343', '1343', '1378', '1378', '1378', '1378', '1409', '1409', '1409', '1409', '1459', '1459', '1459', '1459', '147', '147', '147', '147', '1472', '1472', '1472', '1472', '1518', '1518', '1518', '1518', '175', '175', '175', '175', '213', '213', '213', '213', '228', '228', '228', '228', '27', '27', '27', '27', '278', '278', '278', '278', '303', '303', '303', '303', '343', '343', '343', '343', '364', '364', '364', '364', '398', '398', '398', '398', '469', '469', '469', '469', '482', '482', '482', '482', '494', '494', '494', '494', '504', '504', '504', '504', '560', '560', '560', '560', '578', '578', '578', '578', '628', '628', '628', '628', '729', '729', '729', '729', '749', '749', '749', '749', '792', '792', '792', '792', '834', '834', '834', '834', '86', '86', '86', '86', '867', '867', '867', '867', '921', '921', '921', '921', '987', '987', '987', '987'], 'NDB_844': ['10', '10', '10', '10', '1053', '1053', '1053', '1053', '1070', '1070', '1070', '1070', '1078', '1078', '1078', '1078', '111', '111', '111', '111', '1129', '1129', '1129', '1129', '1156', '1156', '1156', '1156', '1195', '1195', '1195', '1195', '1222', '1222', '1222', '1222', '1259', '1259', '1259', '1259', '1313', '1313', '1313', '1313', '1352', '1352', '1352', '1352', '1385', '1385', '1385', '1385', '1446', '1446', '1446', '1446', '1492', '1492', '1492', '1492', '1506', '1506', '1506', '1506', '154', '154', '154', '154', '182', '182', '182', '182', '228', '228', '228', '228', '264', '264', '264', '264', '293', '293', '293', '293', '340', '340', '340', '340', '382', '382', '382', '382', '42', '42', '42', '42', '421', '421', '421', '421', '452', '452', '452', '452', '489', '489', '489', '489', '522', '522', '522', '522', '559', '559', '559', '559', '596', '596', '596', '596', '618', '618', '618', '618', '676', '676', '676', '676', '681', '681', '681', '681', '733', '733', '733', '733', '757', '757', '757', '757', '791', '791', '791', '791', '835', '835', '835', '835', '872', '872', '872', '872', '924', '924', '924', '924', '98', '98', '98', '98', '986', '986', '986', '986'], 'NDB_812': ['10', '10', '10', '10', '1059', '1059', '1059', '1059', '1089', '1089', '1089', '1089', '1123', '1123', '1123', '1123', '1147', '1147', '1147', '1147', '1208', '1208', '1208', '1208', '1219', '1219', '1219', '1219', '1242', '1242', '1242', '1242', '1258', '1258', '1258', '1258', '126', '126', '126', '126', '1342', '1342', '1342', '1342', '1455', '1455', '1455', '1455', '1475', '1475', '1475', '1475', '1507', '1507', '1507', '1507', '1548', '1548', '1548', '1548', '160', '160', '160', '160', '176', '176', '176', '176', '206', '206', '206', '206', '265', '265', '265', '265', '281', '281', '281', '281', '31', '31', '31', '31', '328', '328', '328', '328', '390', '390', '390', '390', '419', '419', '419', '419', '436', '436', '436', '436', '447', '447', '447', '447', '510', '510', '510', '510', '527', '527', '527', '527', '580', '580', '580', '580', '600', '600', '600', '600', '626', '626', '626', '626', '677', '677', '677', '677', '714', '714', '714', '714', '738', '738', '738', '738', '754', '754', '754', '754', '774', '774', '774', '774', '81', '81', '81', '81', '833', '833', '833', '833', '867', '867', '867', '867', '920', '920', '920', '920', '989', '989', '989', '989'], 'NDB_637': ['10', '10', '10', '10', '1045', '1045', '1045', '1045', '1072', '1072', '1072', '1072', '1119', '1119', '1119', '1119', '1143', '1143', '1143', '1143', '1184', '1184', '1184', '1184', '1229', '1229', '1229', '1229', '1262', '1262', '1262', '1262', '1289', '1289', '1289', '1289', '129', '129', '129', '129', '1323', '1323', '1323', '1323', '1350', '1350', '1350', '1350', '1399', '1399', '1399', '1399', '1450', '1450', '1450', '1450', '1476', '1476', '1476', '1476', '148', '148', '148', '148', '1512', '1512', '1512', '1512', '173', '173', '173', '173', '210', '210', '210', '210', '271', '271', '271', '271', '290', '290', '290', '290', '314', '314', '314', '314', '345', '345', '345', '345', '396', '396', '396', '396', '412', '412', '412', '412', '463', '463', '463', '463', '473', '473', '473', '473', '498', '498', '498', '498', '533', '533', '533', '533', '562', '562', '562', '562', '598', '598', '598', '598', '654', '654', '654', '654', '662', '662', '662', '662', '682', '682', '682', '682', '71', '71', '71', '71', '719', '719', '719', '719', '780', '780', '780', '780', '801', '801', '801', '801', '859', '859', '859', '859', '923', '923', '923', '923', '964', '964', '964', '964'], 'NDB_901': ['100', '100', '100', '100', '1043', '1043', '1043', '1043', '1082', '1082', '1082', '1082', '1132', '1132', '1132', '1132', '1146', '1146', '1146', '1146', '1177', '1177', '1177', '1177', '1215', '1215', '1215', '1215', '1244', '1244', '1244', '1244', '125', '125', '125', '125', '1274', '1274', '1274', '1274', '1305', '1305', '1305', '1305', '1344', '1344', '1344', '1344', '1392', '1392', '1392', '1392', '1448', '1448', '1448', '1448', '1462', '1462', '1462', '1462', '1503', '1503', '1503', '1503', '165', '165', '165', '165', '203', '203', '203', '203', '241', '241', '241', '241', '302', '302', '302', '302', '320', '320', '320', '320', '360', '360', '360', '360', '388', '388', '388', '388', '39', '39', '39', '39', '452', '452', '452', '452', '468', '468', '468', '468', '497', '497', '497', '497', '543', '543', '543', '543', '567', '567', '567', '567', '624', '624', '624', '624', '674', '674', '674', '674', '695', '695', '695', '695', '717', '717', '717', '717', '780', '780', '780', '780', '799', '799', '799', '799', '8', '8', '8', '8', '856', '856', '856', '856', '91', '91', '91', '91', '921', '921', '921', '921', '967', '967', '967', '967', '981', '981', '981', '981'], 'NDB_863': ['100', '100', '100', '100', '1054', '1054', '1054', '1054', '1075', '1075', '1075', '1075', '1106', '1106', '1106', '1106', '1152', '1152', '1152', '1152', '1178', '1178', '1178', '1178', '1214', '1214', '1214', '1214', '124', '124', '124', '124', '1257', '1257', '1257', '1257', '1292', '1292', '1292', '1292', '1310', '1310', '1310', '1310', '1355', '1355', '1355', '1355', '1404', '1404', '1404', '1404', '1447', '1447', '1447', '1447', '1482', '1482', '1482', '1482', '1520', '1520', '1520', '1520', '166', '166', '166', '166', '225', '225', '225', '225', '244', '244', '244', '244', '263', '263', '263', '263', '283', '283', '283', '283', '347', '347', '347', '347', '368', '368', '368', '368', '40', '40', '40', '40', '407', '407', '407', '407', '448', '448', '448', '448', '484', '484', '484', '484', '512', '512', '512', '512', '550', '550', '550', '550', '555', '555', '555', '555', '607', '607', '607', '607', '64', '64', '64', '64', '648', '648', '648', '648', '684', '684', '684', '684', '719', '719', '719', '719', '749', '749', '749', '749', '791', '791', '791', '791', '824', '824', '824', '824', '836', '836', '836', '836', '920', '920', '920', '920', '963', '963', '963', '963'], 'NDB_926': ['100', '100', '100', '100', '1038', '1038', '1038', '1038', '1064', '1064', '1064', '1064', '1090', '1090', '1090', '1090', '1165', '1165', '1165', '1165', '1180', '1180', '1180', '1180', '1205', '1205', '1205', '1205', '123', '123', '123', '123', '1251', '1251', '1251', '1251', '1293', '1293', '1293', '1293', '1313', '1313', '1313', '1313', '1354', '1354', '1354', '1354', '1384', '1384', '1384', '1384', '1442', '1442', '1442', '1442', '1494', '1494', '1494', '1494', '15', '15', '15', '15', '180', '180', '180', '180', '215', '215', '215', '215', '240', '240', '240', '240', '307', '307', '307', '307', '335', '335', '335', '335', '359', '359', '359', '359', '383', '383', '383', '383', '427', '427', '427', '427', '46', '46', '46', '46', '469', '469', '469', '469', '492', '492', '492', '492', '551', '551', '551', '551', '592', '592', '592', '592', '629', '629', '629', '629', '664', '664', '664', '664', '726', '726', '726', '726', '742', '742', '742', '742', '754', '754', '754', '754', '769', '769', '769', '769', '848', '848', '848', '848', '878', '878', '878', '878', '88', '88', '88', '88', '907', '907', '907', '907', '964', '964', '964', '964', '978', '978', '978', '978'], 'NDB_784': ['1000', '1000', '1000', '1000', '1032', '1032', '1032', '1032', '1079', '1079', '1079', '1079', '1125', '1125', '1125', '1125', '1168', '1168', '1168', '1168', '1237', '1237', '1237', '1237', '1244', '1244', '1244', '1244', '1259', '1259', '1259', '1259', '131', '131', '131', '131', '1324', '1324', '1324', '1324', '1349', '1349', '1349', '1349', '1394', '1394', '1394', '1394', '1443', '1443', '1443', '1443', '1486', '1486', '1486', '1486', '1505', '1505', '1505', '1505', '152', '152', '152', '152', '191', '191', '191', '191', '240', '240', '240', '240', '265', '265', '265', '265', '28', '28', '28', '28', '303', '303', '303', '303', '321', '321', '321', '321', '384', '384', '384', '384', '409', '409', '409', '409', '442', '442', '442', '442', '47', '47', '47', '47', '484', '484', '484', '484', '520', '520', '520', '520', '534', '534', '534', '534', '556', '556', '556', '556', '606', '606', '606', '606', '654', '654', '654', '654', '701', '701', '701', '701', '730', '730', '730', '730', '761', '761', '761', '761', '78', '78', '78', '78', '785', '785', '785', '785', '858', '858', '858', '858', '918', '918', '918', '918', '935', '935', '935', '935', '954', '954', '954', '954'], 'NDB_883': ['1000', '1000', '1000', '1000', '1059', '1059', '1059', '1059', '1116', '1116', '1116', '1116', '1142', '1142', '1142', '1142', '1181', '1181', '1181', '1181', '1213', '1213', '1213', '1213', '1243', '1243', '1243', '1243', '1275', '1275', '1275', '1275', '1301', '1301', '1301', '1301', '1340', '1340', '1340', '1340', '1379', '1379', '1379', '1379', '1489', '1489', '1489', '1489', '1534', '1534', '1534', '1534', '159', '159', '159', '159', '163', '163', '163', '163', '22', '22', '22', '22', '220', '220', '220', '220', '247', '247', '247', '247', '289', '289', '289', '289', '311', '311', '311', '311', '348', '348', '348', '348', '361', '361', '361', '361', '385', '385', '385', '385', '462', '462', '462', '462', '469', '469', '469', '469', '511', '511', '511', '511', '548', '548', '548', '548', '591', '591', '591', '591', '6', '6', '6', '6', '615', '615', '615', '615', '676', '676', '676', '676', '687', '687', '687', '687', '727', '727', '727', '727', '764', '764', '764', '764', '810', '810', '810', '810', '816', '816', '816', '816', '838', '838', '838', '838', '85', '85', '85', '85', '896', '896', '896', '896', '931', '931', '931', '931', '963', '963', '963', '963'], 'NDB_925': ['1000', '1000', '1000', '1000', '1062', '1062', '1062', '1062', '11', '11', '11', '11', '1109', '1109', '1109', '1109', '1167', '1167', '1167', '1167', '117', '117', '117', '117', '1221', '1221', '1221', '1221', '1240', '1240', '1240', '1240', '1242', '1242', '1242', '1242', '1277', '1277', '1277', '1277', '1320', '1320', '1320', '1320', '1348', '1348', '1348', '1348', '1380', '1380', '1380', '1380', '1444', '1444', '1444', '1444', '1480', '1480', '1480', '1480', '1495', '1495', '1495', '1495', '168', '168', '168', '168', '192', '192', '192', '192', '234', '234', '234', '234', '256', '256', '256', '256', '272', '272', '272', '272', '278', '278', '278', '278', '29', '29', '29', '29', '319', '319', '319', '319', '387', '387', '387', '387', '415', '415', '415', '415', '441', '441', '441', '441', '475', '475', '475', '475', '506', '506', '506', '506', '537', '537', '537', '537', '577', '577', '577', '577', '607', '607', '607', '607', '662', '662', '662', '662', '718', '718', '718', '718', '734', '734', '734', '734', '789', '789', '789', '789', '822', '822', '822', '822', '850', '850', '850', '850', '891', '891', '891', '891', '930', '930', '930', '930', '969', '969', '969', '969'], 'NDB_634': ['1001', '1001', '1001', '1001', '1040', '1040', '1040', '1040', '1078', '1078', '1078', '1078', '1128', '1128', '1128', '1128', '1174', '1174', '1174', '1174', '1197', '1197', '1197', '1197', '1240', '1240', '1240', '1240', '1245', '1245', '1245', '1245', '1261', '1261', '1261', '1261', '1302', '1302', '1302', '1302', '1324', '1324', '1324', '1324', '1392', '1392', '1392', '1392', '1441', '1441', '1441', '1441', '1471', '1471', '1471', '1471', '1487', '1487', '1487', '1487', '1530', '1530', '1530', '1530', '163', '163', '163', '163', '175', '175', '175', '175', '219', '219', '219', '219', '251', '251', '251', '251', '282', '282', '282', '282', '305', '305', '305', '305', '349', '349', '349', '349', '389', '389', '389', '389', '412', '412', '412', '412', '449', '449', '449', '449', '46', '46', '46', '46', '486', '486', '486', '486', '508', '508', '508', '508', '528', '528', '528', '528', '581', '581', '581', '581', '616', '616', '616', '616', '648', '648', '648', '648', '7', '7', '7', '7', '711', '711', '711', '711', '738', '738', '738', '738', '788', '788', '788', '788', '819', '819', '819', '819', '837', '837', '837', '837', '905', '905', '905', '905', '94', '94', '94', '94', '957', '957', '957', '957'], 'NDB_872': ['1001', '1001', '1001', '1001', '1024', '1024', '1024', '1024', '1053', '1053', '1053', '1053', '1085', '1085', '1085', '1085', '1131', '1131', '1131', '1131', '1155', '1155', '1155', '1155', '117', '117', '117', '117', '1187', '1187', '1187', '1187', '1231', '1231', '1231', '1231', '1285', '1285', '1285', '1285', '129', '129', '129', '129', '1319', '1319', '1319', '1319', '1353', '1353', '1353', '1353', '1390', '1390', '1390', '1390', '1464', '1464', '1464', '1464', '1509', '1509', '1509', '1509', '1538', '1538', '1538', '1538', '187', '187', '187', '187', '230', '230', '230', '230', '250', '250', '250', '250', '268', '268', '268', '268', '298', '298', '298', '298', '342', '342', '342', '342', '381', '381', '381', '381', '400', '400', '400', '400', '423', '423', '423', '423', '456', '456', '456', '456', '525', '525', '525', '525', '550', '550', '550', '550', '592', '592', '592', '592', '645', '645', '645', '645', '66', '66', '66', '66', '677', '677', '677', '677', '706', '706', '706', '706', '737', '737', '737', '737', '78', '78', '78', '78', '798', '798', '798', '798', '846', '846', '846', '846', '860', '860', '860', '860', '904', '904', '904', '904', '974', '974', '974', '974'], 'NDB_755': ['1002', '1002', '1002', '1002', '1055', '1055', '1055', '1055', '106', '106', '106', '106', '1066', '1066', '1066', '1066', '1085', '1085', '1085', '1085', '1132', '1132', '1132', '1132', '1136', '1136', '1136', '1136', '1184', '1184', '1184', '1184', '1207', '1207', '1207', '1207', '1282', '1282', '1282', '1282', '1336', '1336', '1336', '1336', '1361', '1361', '1361', '1361', '1393', '1393', '1393', '1393', '1441', '1441', '1441', '1441', '1465', '1465', '1465', '1465', '1519', '1519', '1519', '1519', '176', '176', '176', '176', '194', '194', '194', '194', '214', '214', '214', '214', '260', '260', '260', '260', '3', '3', '3', '3', '315', '315', '315', '315', '329', '329', '329', '329', '386', '386', '386', '386', '409', '409', '409', '409', '437', '437', '437', '437', '487', '487', '487', '487', '512', '512', '512', '512', '577', '577', '577', '577', '647', '647', '647', '647', '675', '675', '675', '675', '683', '683', '683', '683', '715', '715', '715', '715', '75', '75', '75', '75', '786', '786', '786', '786', '806', '806', '806', '806', '847', '847', '847', '847', '874', '874', '874', '874', '917', '917', '917', '917', '959', '959', '959', '959', '98', '98', '98', '98'], 'NDB_698': ['1002', '1002', '1002', '1002', '1032', '1032', '1032', '1032', '1048', '1048', '1048', '1048', '1087', '1087', '1087', '1087', '1109', '1109', '1109', '1109', '1143', '1143', '1143', '1143', '1162', '1162', '1162', '1162', '1226', '1226', '1226', '1226', '123', '123', '123', '123', '1253', '1253', '1253', '1253', '1288', '1288', '1288', '1288', '1309', '1309', '1309', '1309', '1358', '1358', '1358', '1358', '1427', '1427', '1427', '1427', '1457', '1457', '1457', '1457', '1485', '1485', '1485', '1485', '149', '149', '149', '149', '198', '198', '198', '198', '22', '22', '22', '22', '244', '244', '244', '244', '259', '259', '259', '259', '308', '308', '308', '308', '317', '317', '317', '317', '353', '353', '353', '353', '38', '38', '38', '38', '418', '418', '418', '418', '431', '431', '431', '431', '457', '457', '457', '457', '491', '491', '491', '491', '552', '552', '552', '552', '585', '585', '585', '585', '645', '645', '645', '645', '685', '685', '685', '685', '712', '712', '712', '712', '778', '778', '778', '778', '807', '807', '807', '807', '829', '829', '829', '829', '84', '84', '84', '84', '867', '867', '867', '867', '913', '913', '913', '913', '967', '967', '967', '967'], 'NDB_1033': ['1002', '1002', '1002', '1002', '1024', '1024', '1024', '1024', '1042', '1042', '1042', '1042', '1108', '1108', '1108', '1108', '1167', '1167', '1167', '1167', '118', '118', '118', '118', '1196', '1196', '1196', '1196', '1228', '1228', '1228', '1228', '1258', '1258', '1258', '1258', '1339', '1339', '1339', '1339', '1379', '1379', '1379', '1379', '1436', '1436', '1436', '1436', '1476', '1476', '1476', '1476', '1490', '1490', '1490', '1490', '153', '153', '153', '153', '184', '184', '184', '184', '196', '196', '196', '196', '209', '209', '209', '209', '251', '251', '251', '251', '293', '293', '293', '293', '343', '343', '343', '343', '35', '35', '35', '35', '358', '358', '358', '358', '391', '391', '391', '391', '450', '450', '450', '450', '484', '484', '484', '484', '497', '497', '497', '497', '536', '536', '536', '536', '584', '584', '584', '584', '617', '617', '617', '617', '649', '649', '649', '649', '65', '65', '65', '65', '713', '713', '713', '713', '731', '731', '731', '731', '756', '756', '756', '756', '794', '794', '794', '794', '827', '827', '827', '827', '865', '865', '865', '865', '888', '888', '888', '888', '929', '929', '929', '929', '973', '973', '973', '973'], 'NDB_817': ['1003', '1003', '1003', '1003', '1044', '1044', '1044', '1044', '1117', '1117', '1117', '1117', '1138', '1138', '1138', '1138', '115', '115', '115', '115', '1195', '1195', '1195', '1195', '1209', '1209', '1209', '1209', '1251', '1251', '1251', '1251', '1299', '1299', '1299', '1299', '1333', '1333', '1333', '1333', '1345', '1345', '1345', '1345', '1454', '1454', '1454', '1454', '1478', '1478', '1478', '1478', '1519', '1519', '1519', '1519', '1542', '1542', '1542', '1542', '167', '167', '167', '167', '188', '188', '188', '188', '220', '220', '220', '220', '255', '255', '255', '255', '274', '274', '274', '274', '295', '295', '295', '295', '349', '349', '349', '349', '358', '358', '358', '358', '37', '37', '37', '37', '379', '379', '379', '379', '430', '430', '430', '430', '468', '468', '468', '468', '501', '501', '501', '501', '546', '546', '546', '546', '579', '579', '579', '579', '603', '603', '603', '603', '66', '66', '66', '66', '662', '662', '662', '662', '729', '729', '729', '729', '778', '778', '778', '778', '805', '805', '805', '805', '836', '836', '836', '836', '863', '863', '863', '863', '900', '900', '900', '900', '935', '935', '935', '935', '975', '975', '975', '975'], 'NDB_746': ['1003', '1003', '1003', '1003', '1054', '1054', '1054', '1054', '1090', '1090', '1090', '1090', '111', '111', '111', '111', '1122', '1122', '1122', '1122', '1170', '1170', '1170', '1170', '1212', '1212', '1212', '1212', '1227', '1227', '1227', '1227', '1243', '1243', '1243', '1243', '1286', '1286', '1286', '1286', '13', '13', '13', '13', '1303', '1303', '1303', '1303', '131', '131', '131', '131', '1322', '1322', '1322', '1322', '1386', '1386', '1386', '1386', '1436', '1436', '1436', '1436', '1463', '1463', '1463', '1463', '1481', '1481', '1481', '1481', '1526', '1526', '1526', '1526', '192', '192', '192', '192', '202', '202', '202', '202', '245', '245', '245', '245', '296', '296', '296', '296', '31', '31', '31', '31', '333', '333', '333', '333', '374', '374', '374', '374', '418', '418', '418', '418', '454', '454', '454', '454', '486', '486', '486', '486', '496', '496', '496', '496', '539', '539', '539', '539', '560', '560', '560', '560', '609', '609', '609', '609', '62', '62', '62', '62', '627', '627', '627', '627', '692', '692', '692', '692', '709', '709', '709', '709', '787', '787', '787', '787', '813', '813', '813', '813', '856', '856', '856', '856', '915', '915', '915', '915', '959', '959', '959', '959'], 'NDB_742': ['1003', '1003', '1003', '1003', '1040', '1040', '1040', '1040', '1068', '1068', '1068', '1068', '1074', '1074', '1074', '1074', '1087', '1087', '1087', '1087', '1142', '1142', '1142', '1142', '1217', '1217', '1217', '1217', '1257', '1257', '1257', '1257', '1312', '1312', '1312', '1312', '1360', '1360', '1360', '1360', '138', '138', '138', '138', '1408', '1408', '1408', '1408', '1451', '1451', '1451', '1451', '1490', '1490', '1490', '1490', '1502', '1502', '1502', '1502', '1548', '1548', '1548', '1548', '156', '156', '156', '156', '171', '171', '171', '171', '240', '240', '240', '240', '266', '266', '266', '266', '287', '287', '287', '287', '330', '330', '330', '330', '367', '367', '367', '367', '401', '401', '401', '401', '423', '423', '423', '423', '45', '45', '45', '45', '489', '489', '489', '489', '526', '526', '526', '526', '566', '566', '566', '566', '613', '613', '613', '613', '671', '671', '671', '671', '683', '683', '683', '683', '71', '71', '71', '71', '721', '721', '721', '721', '767', '767', '767', '767', '808', '808', '808', '808', '825', '825', '825', '825', '870', '870', '870', '870', '9', '9', '9', '9', '922', '922', '922', '922', '985', '985', '985', '985'], 'NDB_878': ['1004', '1004', '1004', '1004', '1039', '1039', '1039', '1039', '109', '109', '109', '109', '1116', '1116', '1116', '1116', '1154', '1154', '1154', '1154', '1188', '1188', '1188', '1188', '1208', '1208', '1208', '1208', '122', '122', '122', '122', '1254', '1254', '1254', '1254', '1291', '1291', '1291', '1291', '1313', '1313', '1313', '1313', '1350', '1350', '1350', '1350', '1382', '1382', '1382', '1382', '1440', '1440', '1440', '1440', '1477', '1477', '1477', '1477', '1514', '1514', '1514', '1514', '188', '188', '188', '188', '218', '218', '218', '218', '248', '248', '248', '248', '310', '310', '310', '310', '346', '346', '346', '346', '362', '362', '362', '362', '395', '395', '395', '395', '431', '431', '431', '431', '460', '460', '460', '460', '517', '517', '517', '517', '539', '539', '539', '539', '570', '570', '570', '570', '595', '595', '595', '595', '648', '648', '648', '648', '65', '65', '65', '65', '694', '694', '694', '694', '722', '722', '722', '722', '758', '758', '758', '758', '796', '796', '796', '796', '822', '822', '822', '822', '847', '847', '847', '847', '9', '9', '9', '9', '914', '914', '914', '914', '933', '933', '933', '933', '960', '960', '960', '960'], 'NDB_896': ['1004', '1004', '1004', '1004', '1029', '1029', '1029', '1029', '1067', '1067', '1067', '1067', '1078', '1078', '1078', '1078', '1169', '1169', '1169', '1169', '1215', '1215', '1215', '1215', '1241', '1241', '1241', '1241', '1260', '1260', '1260', '1260', '1314', '1314', '1314', '1314', '1353', '1353', '1353', '1353', '1393', '1393', '1393', '1393', '1435', '1435', '1435', '1435', '1467', '1467', '1467', '1467', '1507', '1507', '1507', '1507', '152', '152', '152', '152', '166', '166', '166', '166', '204', '204', '204', '204', '253', '253', '253', '253', '276', '276', '276', '276', '289', '289', '289', '289', '327', '327', '327', '327', '358', '358', '358', '358', '38', '38', '38', '38', '383', '383', '383', '383', '428', '428', '428', '428', '441', '441', '441', '441', '526', '526', '526', '526', '588', '588', '588', '588', '605', '605', '605', '605', '659', '659', '659', '659', '705', '705', '705', '705', '743', '743', '743', '743', '748', '748', '748', '748', '76', '76', '76', '76', '800', '800', '800', '800', '814', '814', '814', '814', '857', '857', '857', '857', '90', '90', '90', '90', '908', '908', '908', '908', '923', '923', '923', '923', '961', '961', '961', '961'], 'NDB_661': ['1005', '1005', '1005', '1005', '1052', '1052', '1052', '1052', '1080', '1080', '1080', '1080', '112', '112', '112', '112', '1172', '1172', '1172', '1172', '1204', '1204', '1204', '1204', '1235', '1235', '1235', '1235', '1261', '1261', '1261', '1261', '1287', '1287', '1287', '1287', '1307', '1307', '1307', '1307', '1357', '1357', '1357', '1357', '1403', '1403', '1403', '1403', '1437', '1437', '1437', '1437', '1472', '1472', '1472', '1472', '1512', '1512', '1512', '1512', '1525', '1525', '1525', '1525', '154', '154', '154', '154', '170', '170', '170', '170', '19', '19', '19', '19', '214', '214', '214', '214', '259', '259', '259', '259', '284', '284', '284', '284', '30', '30', '30', '30', '319', '319', '319', '319', '354', '354', '354', '354', '401', '401', '401', '401', '446', '446', '446', '446', '479', '479', '479', '479', '525', '525', '525', '525', '568', '568', '568', '568', '605', '605', '605', '605', '660', '660', '660', '660', '694', '694', '694', '694', '714', '714', '714', '714', '756', '756', '756', '756', '769', '769', '769', '769', '8', '8', '8', '8', '816', '816', '816', '816', '839', '839', '839', '839', '915', '915', '915', '915', '927', '927', '927', '927', '954', '954', '954', '954'], 'NDB_818': ['1005', '1005', '1005', '1005', '1029', '1029', '1029', '1029', '1066', '1066', '1066', '1066', '1077', '1077', '1077', '1077', '1113', '1113', '1113', '1113', '1137', '1137', '1137', '1137', '118', '118', '118', '118', '1186', '1186', '1186', '1186', '1230', '1230', '1230', '1230', '1273', '1273', '1273', '1273', '13', '13', '13', '13', '1324', '1324', '1324', '1324', '1347', '1347', '1347', '1347', '1390', '1390', '1390', '1390', '1458', '1458', '1458', '1458', '1474', '1474', '1474', '1474', '1520', '1520', '1520', '1520', '157', '157', '157', '157', '186', '186', '186', '186', '195', '195', '195', '195', '211', '211', '211', '211', '262', '262', '262', '262', '287', '287', '287', '287', '317', '317', '317', '317', '357', '357', '357', '357', '380', '380', '380', '380', '447', '447', '447', '447', '461', '461', '461', '461', '506', '506', '506', '506', '540', '540', '540', '540', '586', '586', '586', '586', '618', '618', '618', '618', '650', '650', '650', '650', '728', '728', '728', '728', '780', '780', '780', '780', '802', '802', '802', '802', '820', '820', '820', '820', '844', '844', '844', '844', '89', '89', '89', '89', '919', '919', '919', '919', '975', '975', '975', '975'], 'NDB_725': ['101', '101', '101', '101', '1054', '1054', '1054', '1054', '1074', '1074', '1074', '1074', '1130', '1130', '1130', '1130', '1174', '1174', '1174', '1174', '1210', '1210', '1210', '1210', '1252', '1252', '1252', '1252', '1297', '1297', '1297', '1297', '132', '132', '132', '132', '1335', '1335', '1335', '1335', '1402', '1402', '1402', '1402', '1430', '1430', '1430', '1430', '1493', '1493', '1493', '1493', '1509', '1509', '1509', '1509', '1535', '1535', '1535', '1535', '179', '179', '179', '179', '223', '223', '223', '223', '246', '246', '246', '246', '274', '274', '274', '274', '291', '291', '291', '291', '30', '30', '30', '30', '345', '345', '345', '345', '385', '385', '385', '385', '405', '405', '405', '405', '450', '450', '450', '450', '467', '467', '467', '467', '518', '518', '518', '518', '534', '534', '534', '534', '588', '588', '588', '588', '619', '619', '619', '619', '650', '650', '650', '650', '70', '70', '70', '70', '715', '715', '715', '715', '745', '745', '745', '745', '758', '758', '758', '758', '770', '770', '770', '770', '833', '833', '833', '833', '877', '877', '877', '877', '910', '910', '910', '910', '958', '958', '958', '958', '996', '996', '996', '996'], 'NDB_847': ['101', '101', '101', '101', '1062', '1062', '1062', '1062', '1080', '1080', '1080', '1080', '1092', '1092', '1092', '1092', '1136', '1136', '1136', '1136', '1163', '1163', '1163', '1163', '1196', '1196', '1196', '1196', '1249', '1249', '1249', '1249', '1290', '1290', '1290', '1290', '1302', '1302', '1302', '1302', '1318', '1318', '1318', '1318', '1389', '1389', '1389', '1389', '1473', '1473', '1473', '1473', '1506', '1506', '1506', '1506', '159', '159', '159', '159', '187', '187', '187', '187', '2', '2', '2', '2', '235', '235', '235', '235', '263', '263', '263', '263', '281', '281', '281', '281', '321', '321', '321', '321', '355', '355', '355', '355', '370', '370', '370', '370', '39', '39', '39', '39', '418', '418', '418', '418', '463', '463', '463', '463', '494', '494', '494', '494', '545', '545', '545', '545', '575', '575', '575', '575', '593', '593', '593', '593', '612', '612', '612', '612', '67', '67', '67', '67', '673', '673', '673', '673', '696', '696', '696', '696', '741', '741', '741', '741', '762', '762', '762', '762', '784', '784', '784', '784', '861', '861', '861', '861', '922', '922', '922', '922', '979', '979', '979', '979'], 'NDB_835': ['101', '101', '101', '101', '1038', '1038', '1038', '1038', '1094', '1094', '1094', '1094', '1129', '1129', '1129', '1129', '1160', '1160', '1160', '1160', '1224', '1224', '1224', '1224', '1260', '1260', '1260', '1260', '1299', '1299', '1299', '1299', '130', '130', '130', '130', '1343', '1343', '1343', '1343', '1405', '1405', '1405', '1405', '1455', '1455', '1455', '1455', '1480', '1480', '1480', '1480', '1515', '1515', '1515', '1515', '1538', '1538', '1538', '1538', '184', '184', '184', '184', '199', '199', '199', '199', '239', '239', '239', '239', '254', '254', '254', '254', '300', '300', '300', '300', '322', '322', '322', '322', '353', '353', '353', '353', '402', '402', '402', '402', '460', '460', '460', '460', '486', '486', '486', '486', '490', '490', '490', '490', '503', '503', '503', '503', '571', '571', '571', '571', '608', '608', '608', '608', '654', '654', '654', '654', '720', '720', '720', '720', '735', '735', '735', '735', '790', '790', '790', '790', '8', '8', '8', '8', '808', '808', '808', '808', '824', '824', '824', '824', '83', '83', '83', '83', '869', '869', '869', '869', '895', '895', '895', '895', '971', '971', '971', '971', '982', '982', '982', '982'], 'NDB_815': ['102', '102', '102', '102', '1030', '1030', '1030', '1030', '1072', '1072', '1072', '1072', '1108', '1108', '1108', '1108', '1121', '1121', '1121', '1121', '1176', '1176', '1176', '1176', '1206', '1206', '1206', '1206', '1274', '1274', '1274', '1274', '1331', '1331', '1331', '1331', '1346', '1346', '1346', '1346', '139', '139', '139', '139', '1404', '1404', '1404', '1404', '1451', '1451', '1451', '1451', '1469', '1469', '1469', '1469', '1507', '1507', '1507', '1507', '1526', '1526', '1526', '1526', '16', '16', '16', '16', '161', '161', '161', '161', '224', '224', '224', '224', '246', '246', '246', '246', '308', '308', '308', '308', '334', '334', '334', '334', '351', '351', '351', '351', '382', '382', '382', '382', '398', '398', '398', '398', '425', '425', '425', '425', '456', '456', '456', '456', '508', '508', '508', '508', '577', '577', '577', '577', '601', '601', '601', '601', '658', '658', '658', '658', '708', '708', '708', '708', '736', '736', '736', '736', '753', '753', '753', '753', '769', '769', '769', '769', '843', '843', '843', '843', '85', '85', '85', '85', '871', '871', '871', '871', '914', '914', '914', '914', '950', '950', '950', '950', '984', '984', '984', '984'], 'NDB_866': ['102', '102', '102', '102', '1060', '1060', '1060', '1060', '1081', '1081', '1081', '1081', '1126', '1126', '1126', '1126', '1171', '1171', '1171', '1171', '1207', '1207', '1207', '1207', '1255', '1255', '1255', '1255', '127', '127', '127', '127', '1322', '1322', '1322', '1322', '1357', '1357', '1357', '1357', '1399', '1399', '1399', '1399', '14', '14', '14', '14', '1453', '1453', '1453', '1453', '1474', '1474', '1474', '1474', '1515', '1515', '1515', '1515', '1542', '1542', '1542', '1542', '175', '175', '175', '175', '222', '222', '222', '222', '258', '258', '258', '258', '309', '309', '309', '309', '337', '337', '337', '337', '391', '391', '391', '391', '413', '413', '413', '413', '423', '423', '423', '423', '451', '451', '451', '451', '517', '517', '517', '517', '538', '538', '538', '538', '572', '572', '572', '572', '607', '607', '607', '607', '629', '629', '629', '629', '667', '667', '667', '667', '68', '68', '68', '68', '680', '680', '680', '680', '691', '691', '691', '691', '744', '744', '744', '744', '762', '762', '762', '762', '795', '795', '795', '795', '857', '857', '857', '857', '906', '906', '906', '906', '958', '958', '958', '958', '976', '976', '976', '976'], 'NDB_941': ['102', '102', '102', '102', '1034', '1034', '1034', '1034', '1049', '1049', '1049', '1049', '1109', '1109', '1109', '1109', '1169', '1169', '1169', '1169', '1192', '1192', '1192', '1192', '1239', '1239', '1239', '1239', '1279', '1279', '1279', '1279', '1317', '1317', '1317', '1317', '1349', '1349', '1349', '1349', '1391', '1391', '1391', '1391', '1434', '1434', '1434', '1434', '146', '146', '146', '146', '1498', '1498', '1498', '1498', '167', '167', '167', '167', '221', '221', '221', '221', '248', '248', '248', '248', '26', '26', '26', '26', '291', '291', '291', '291', '305', '305', '305', '305', '325', '325', '325', '325', '359', '359', '359', '359', '388', '388', '388', '388', '446', '446', '446', '446', '474', '474', '474', '474', '509', '509', '509', '509', '527', '527', '527', '527', '562', '562', '562', '562', '599', '599', '599', '599', '649', '649', '649', '649', '665', '665', '665', '665', '696', '696', '696', '696', '733', '733', '733', '733', '777', '777', '777', '777', '802', '802', '802', '802', '860', '860', '860', '860', '87', '87', '87', '87', '895', '895', '895', '895', '912', '912', '912', '912', '963', '963', '963', '963', '991', '991', '991', '991'], 'NDB_971': ['1020', '1020', '1020', '1020', '1071', '1071', '1071', '1071', '108', '108', '108', '108', '1115', '1115', '1115', '1115', '1169', '1169', '1169', '1169', '1214', '1214', '1214', '1214', '1237', '1237', '1237', '1237', '1276', '1276', '1276', '1276', '1343', '1343', '1343', '1343', '1387', '1387', '1387', '1387', '1437', '1437', '1437', '1437', '1466', '1466', '1466', '1466', '149', '149', '149', '149', '1517', '1517', '1517', '1517', '181', '181', '181', '181', '210', '210', '210', '210', '233', '233', '233', '233', '267', '267', '267', '267', '298', '298', '298', '298', '323', '323', '323', '323', '362', '362', '362', '362', '37', '37', '37', '37', '387', '387', '387', '387', '436', '436', '436', '436', '452', '452', '452', '452', '496', '496', '496', '496', '536', '536', '536', '536', '563', '563', '563', '563', '586', '586', '586', '586', '60', '60', '60', '60', '616', '616', '616', '616', '667', '667', '667', '667', '698', '698', '698', '698', '725', '725', '725', '725', '765', '765', '765', '765', '806', '806', '806', '806', '843', '843', '843', '843', '875', '875', '875', '875', '898', '898', '898', '898', '931', '931', '931', '931', '983', '983', '983', '983'], 'NDB_754': ['1020', '1020', '1020', '1020', '1046', '1046', '1046', '1046', '1081', '1081', '1081', '1081', '1177', '1177', '1177', '1177', '118', '118', '118', '118', '1219', '1219', '1219', '1219', '1264', '1264', '1264', '1264', '1293', '1293', '1293', '1293', '1315', '1315', '1315', '1315', '1363', '1363', '1363', '1363', '1405', '1405', '1405', '1405', '1430', '1430', '1430', '1430', '1477', '1477', '1477', '1477', '1508', '1508', '1508', '1508', '1536', '1536', '1536', '1536', '159', '159', '159', '159', '208', '208', '208', '208', '230', '230', '230', '230', '24', '24', '24', '24', '295', '295', '295', '295', '314', '314', '314', '314', '347', '347', '347', '347', '364', '364', '364', '364', '401', '401', '401', '401', '424', '424', '424', '424', '453', '453', '453', '453', '524', '524', '524', '524', '561', '561', '561', '561', '576', '576', '576', '576', '621', '621', '621', '621', '665', '665', '665', '665', '7', '7', '7', '7', '704', '704', '704', '704', '739', '739', '739', '739', '755', '755', '755', '755', '786', '786', '786', '786', '828', '828', '828', '828', '871', '871', '871', '871', '908', '908', '908', '908', '96', '96', '96', '96', '986', '986', '986', '986'], 'NDB_715': ['1020', '1020', '1020', '1020', '1069', '1069', '1069', '1069', '1121', '1121', '1121', '1121', '1141', '1141', '1141', '1141', '1187', '1187', '1187', '1187', '1230', '1230', '1230', '1230', '1247', '1247', '1247', '1247', '1284', '1284', '1284', '1284', '133', '133', '133', '133', '1333', '1333', '1333', '1333', '1388', '1388', '1388', '1388', '1434', '1434', '1434', '1434', '1485', '1485', '1485', '1485', '1502', '1502', '1502', '1502', '154', '154', '154', '154', '184', '184', '184', '184', '202', '202', '202', '202', '21', '21', '21', '21', '241', '241', '241', '241', '299', '299', '299', '299', '339', '339', '339', '339', '384', '384', '384', '384', '417', '417', '417', '417', '441', '441', '441', '441', '487', '487', '487', '487', '505', '505', '505', '505', '542', '542', '542', '542', '583', '583', '583', '583', '657', '657', '657', '657', '67', '67', '67', '67', '688', '688', '688', '688', '744', '744', '744', '744', '749', '749', '749', '749', '776', '776', '776', '776', '817', '817', '817', '817', '833', '833', '833', '833', '893', '893', '893', '893', '93', '93', '93', '93', '933', '933', '933', '933', '964', '964', '964', '964', '993', '993', '993', '993'], 'NDB_868': ['1023', '1023', '1023', '1023', '1068', '1068', '1068', '1068', '1107', '1107', '1107', '1107', '1122', '1122', '1122', '1122', '1139', '1139', '1139', '1139', '115', '115', '115', '115', '1165', '1165', '1165', '1165', '1214', '1214', '1214', '1214', '1275', '1275', '1275', '1275', '1336', '1336', '1336', '1336', '1344', '1344', '1344', '1344', '1399', '1399', '1399', '1399', '1440', '1440', '1440', '1440', '1498', '1498', '1498', '1498', '158', '158', '158', '158', '191', '191', '191', '191', '217', '217', '217', '217', '238', '238', '238', '238', '264', '264', '264', '264', '292', '292', '292', '292', '328', '328', '328', '328', '357', '357', '357', '357', '392', '392', '392', '392', '439', '439', '439', '439', '44', '44', '44', '44', '478', '478', '478', '478', '505', '505', '505', '505', '535', '535', '535', '535', '554', '554', '554', '554', '581', '581', '581', '581', '620', '620', '620', '620', '666', '666', '666', '666', '712', '712', '712', '712', '73', '73', '73', '73', '735', '735', '735', '735', '798', '798', '798', '798', '834', '834', '834', '834', '871', '871', '871', '871', '910', '910', '910', '910', '952', '952', '952', '952', '979', '979', '979', '979'], 'NDB_832': ['1023', '1023', '1023', '1023', '1071', '1071', '1071', '1071', '1083', '1083', '1083', '1083', '1113', '1113', '1113', '1113', '1166', '1166', '1166', '1166', '120', '120', '120', '120', '1234', '1234', '1234', '1234', '1279', '1279', '1279', '1279', '1311', '1311', '1311', '1311', '1360', '1360', '1360', '1360', '1402', '1402', '1402', '1402', '1445', '1445', '1445', '1445', '1486', '1486', '1486', '1486', '150', '150', '150', '150', '1519', '1519', '1519', '1519', '1538', '1538', '1538', '1538', '183', '183', '183', '183', '2', '2', '2', '2', '236', '236', '236', '236', '249', '249', '249', '249', '25', '25', '25', '25', '278', '278', '278', '278', '324', '324', '324', '324', '399', '399', '399', '399', '419', '419', '419', '419', '448', '448', '448', '448', '476', '476', '476', '476', '492', '492', '492', '492', '552', '552', '552', '552', '567', '567', '567', '567', '609', '609', '609', '609', '659', '659', '659', '659', '689', '689', '689', '689', '723', '723', '723', '723', '788', '788', '788', '788', '797', '797', '797', '797', '827', '827', '827', '827', '872', '872', '872', '872', '914', '914', '914', '914', '93', '93', '93', '93', '962', '962', '962', '962'], 'NDB_929': ['1023', '1023', '1023', '1023', '104', '104', '104', '104', '1065', '1065', '1065', '1065', '1110', '1110', '1110', '1110', '1176', '1176', '1176', '1176', '1222', '1222', '1222', '1222', '1240', '1240', '1240', '1240', '1270', '1270', '1270', '1270', '1300', '1300', '1300', '1300', '1339', '1339', '1339', '1339', '1388', '1388', '1388', '1388', '1468', '1468', '1468', '1468', '1516', '1516', '1516', '1516', '1534', '1534', '1534', '1534', '167', '167', '167', '167', '185', '185', '185', '185', '197', '197', '197', '197', '235', '235', '235', '235', '27', '27', '27', '27', '273', '273', '273', '273', '287', '287', '287', '287', '332', '332', '332', '332', '362', '362', '362', '362', '381', '381', '381', '381', '443', '443', '443', '443', '472', '472', '472', '472', '518', '518', '518', '518', '572', '572', '572', '572', '606', '606', '606', '606', '622', '622', '622', '622', '678', '678', '678', '678', '702', '702', '702', '702', '742', '742', '742', '742', '75', '75', '75', '75', '755', '755', '755', '755', '774', '774', '774', '774', '856', '856', '856', '856', '876', '876', '876', '876', '923', '923', '923', '923', '966', '966', '966', '966', '994', '994', '994', '994'], 'NDB_1075': ['1024', '1024', '1024', '1024', '1051', '1051', '1051', '1051', '1090', '1090', '1090', '1090', '1146', '1146', '1146', '1146', '1193', '1193', '1193', '1193', '1238', '1238', '1238', '1238', '1245', '1245', '1245', '1245', '1282', '1282', '1282', '1282', '134', '134', '134', '134', '1342', '1342', '1342', '1342', '1380', '1380', '1380', '1380', '1440', '1440', '1440', '1440', '1472', '1472', '1472', '1472', '1510', '1510', '1510', '1510', '169', '169', '169', '169', '22', '22', '22', '22', '221', '221', '221', '221', '252', '252', '252', '252', '275', '275', '275', '275', '304', '304', '304', '304', '347', '347', '347', '347', '372', '372', '372', '372', '405', '405', '405', '405', '42', '42', '42', '42', '444', '444', '444', '444', '476', '476', '476', '476', '498', '498', '498', '498', '541', '541', '541', '541', '563', '563', '563', '563', '597', '597', '597', '597', '655', '655', '655', '655', '701', '701', '701', '701', '732', '732', '732', '732', '753', '753', '753', '753', '790', '790', '790', '790', '845', '845', '845', '845', '866', '866', '866', '866', '897', '897', '897', '897', '92', '92', '92', '92', '934', '934', '934', '934', '975', '975', '975', '975'], 'NDB_743': ['1025', '1025', '1025', '1025', '1055', '1055', '1055', '1055', '1105', '1105', '1105', '1105', '112', '112', '112', '112', '1171', '1171', '1171', '1171', '1209', '1209', '1209', '1209', '1233', '1233', '1233', '1233', '1243', '1243', '1243', '1243', '1253', '1253', '1253', '1253', '1330', '1330', '1330', '1330', '1348', '1348', '1348', '1348', '1448', '1448', '1448', '1448', '145', '145', '145', '145', '1467', '1467', '1467', '1467', '1518', '1518', '1518', '1518', '1536', '1536', '1536', '1536', '190', '190', '190', '190', '195', '195', '195', '195', '21', '21', '21', '21', '224', '224', '224', '224', '269', '269', '269', '269', '310', '310', '310', '310', '341', '341', '341', '341', '390', '390', '390', '390', '411', '411', '411', '411', '448', '448', '448', '448', '478', '478', '478', '478', '507', '507', '507', '507', '533', '533', '533', '533', '584', '584', '584', '584', '596', '596', '596', '596', '646', '646', '646', '646', '702', '702', '702', '702', '730', '730', '730', '730', '74', '74', '74', '74', '790', '790', '790', '790', '825', '825', '825', '825', '846', '846', '846', '846', '889', '889', '889', '889', '907', '907', '907', '907', '983', '983', '983', '983'], 'NDB_641': ['1025', '1025', '1025', '1025', '1056', '1056', '1056', '1056', '1110', '1110', '1110', '1110', '1156', '1156', '1156', '1156', '1184', '1184', '1184', '1184', '1216', '1216', '1216', '1216', '1275', '1275', '1275', '1275', '1304', '1304', '1304', '1304', '1338', '1338', '1338', '1338', '137', '137', '137', '137', '1386', '1386', '1386', '1386', '1438', '1438', '1438', '1438', '1470', '1470', '1470', '1470', '1517', '1517', '1517', '1517', '165', '165', '165', '165', '183', '183', '183', '183', '208', '208', '208', '208', '262', '262', '262', '262', '284', '284', '284', '284', '304', '304', '304', '304', '349', '349', '349', '349', '371', '371', '371', '371', '403', '403', '403', '403', '455', '455', '455', '455', '481', '481', '481', '481', '490', '490', '490', '490', '527', '527', '527', '527', '582', '582', '582', '582', '593', '593', '593', '593', '623', '623', '623', '623', '64', '64', '64', '64', '664', '664', '664', '664', '708', '708', '708', '708', '737', '737', '737', '737', '795', '795', '795', '795', '815', '815', '815', '815', '854', '854', '854', '854', '86', '86', '86', '86', '893', '893', '893', '893', '909', '909', '909', '909', '973', '973', '973', '973'], 'NDB_958': ['1025', '1025', '1025', '1025', '1045', '1045', '1045', '1045', '1091', '1091', '1091', '1091', '1127', '1127', '1127', '1127', '114', '114', '114', '114', '1148', '1148', '1148', '1148', '1182', '1182', '1182', '1182', '1230', '1230', '1230', '1230', '1272', '1272', '1272', '1272', '1342', '1342', '1342', '1342', '1394', '1394', '1394', '1394', '1449', '1449', '1449', '1449', '1479', '1479', '1479', '1479', '15', '15', '15', '15', '1506', '1506', '1506', '1506', '156', '156', '156', '156', '179', '179', '179', '179', '245', '245', '245', '245', '267', '267', '267', '267', '316', '316', '316', '316', '343', '343', '343', '343', '354', '354', '354', '354', '408', '408', '408', '408', '465', '465', '465', '465', '480', '480', '480', '480', '503', '503', '503', '503', '530', '530', '530', '530', '572', '572', '572', '572', '652', '652', '652', '652', '665', '665', '665', '665', '706', '706', '706', '706', '72', '72', '72', '72', '736', '736', '736', '736', '750', '750', '750', '750', '788', '788', '788', '788', '814', '814', '814', '814', '838', '838', '838', '838', '900', '900', '900', '900', '91', '91', '91', '91', '977', '977', '977', '977', '995', '995', '995', '995'], 'NDB_680': ['1029', '1029', '1029', '1029', '105', '105', '105', '105', '1108', '1108', '1108', '1108', '1145', '1145', '1145', '1145', '1188', '1188', '1188', '1188', '1211', '1211', '1211', '1211', '1253', '1253', '1253', '1253', '1294', '1294', '1294', '1294', '1333', '1333', '1333', '1333', '1362', '1362', '1362', '1362', '1429', '1429', '1429', '1429', '1497', '1497', '1497', '1497', '1530', '1530', '1530', '1530', '160', '160', '160', '160', '183', '183', '183', '183', '205', '205', '205', '205', '247', '247', '247', '247', '313', '313', '313', '313', '326', '326', '326', '326', '344', '344', '344', '344', '359', '359', '359', '359', '374', '374', '374', '374', '429', '429', '429', '429', '466', '466', '466', '466', '512', '512', '512', '512', '545', '545', '545', '545', '568', '568', '568', '568', '60', '60', '60', '60', '628', '628', '628', '628', '671', '671', '671', '671', '68', '68', '68', '68', '720', '720', '720', '720', '732', '732', '732', '732', '750', '750', '750', '750', '798', '798', '798', '798', '841', '841', '841', '841', '877', '877', '877', '877', '899', '899', '899', '899', '911', '911', '911', '911', '981', '981', '981', '981', '994', '994', '994', '994'], 'NDB_713': ['103', '103', '103', '103', '1030', '1030', '1030', '1030', '1111', '1111', '1111', '1111', '1127', '1127', '1127', '1127', '1178', '1178', '1178', '1178', '1229', '1229', '1229', '1229', '1259', '1259', '1259', '1259', '1298', '1298', '1298', '1298', '1341', '1341', '1341', '1341', '1405', '1405', '1405', '1405', '1452', '1452', '1452', '1452', '1497', '1497', '1497', '1497', '1514', '1514', '1514', '1514', '1541', '1541', '1541', '1541', '157', '157', '157', '157', '171', '171', '171', '171', '202', '202', '202', '202', '233', '233', '233', '233', '257', '257', '257', '257', '27', '27', '27', '27', '297', '297', '297', '297', '319', '319', '319', '319', '360', '360', '360', '360', '389', '389', '389', '389', '428', '428', '428', '428', '44', '44', '44', '44', '450', '450', '450', '450', '502', '502', '502', '502', '541', '541', '541', '541', '591', '591', '591', '591', '611', '611', '611', '611', '629', '629', '629', '629', '689', '689', '689', '689', '716', '716', '716', '716', '751', '751', '751', '751', '779', '779', '779', '779', '829', '829', '829', '829', '874', '874', '874', '874', '904', '904', '904', '904', '972', '972', '972', '972', '991', '991', '991', '991'], 'NDB_752': ['103', '103', '103', '103', '1033', '1033', '1033', '1033', '1123', '1123', '1123', '1123', '1144', '1144', '1144', '1144', '1181', '1181', '1181', '1181', '1191', '1191', '1191', '1191', '122', '122', '122', '122', '1238', '1238', '1238', '1238', '1247', '1247', '1247', '1247', '1299', '1299', '1299', '1299', '1305', '1305', '1305', '1305', '1320', '1320', '1320', '1320', '1386', '1386', '1386', '1386', '1458', '1458', '1458', '1458', '1479', '1479', '1479', '1479', '1516', '1516', '1516', '1516', '174', '174', '174', '174', '219', '219', '219', '219', '24', '24', '24', '24', '258', '258', '258', '258', '307', '307', '307', '307', '325', '325', '325', '325', '363', '363', '363', '363', '392', '392', '392', '392', '40', '40', '40', '40', '457', '457', '457', '457', '471', '471', '471', '471', '494', '494', '494', '494', '549', '549', '549', '549', '557', '557', '557', '557', '605', '605', '605', '605', '658', '658', '658', '658', '681', '681', '681', '681', '730', '730', '730', '730', '763', '763', '763', '763', '793', '793', '793', '793', '823', '823', '823', '823', '851', '851', '851', '851', '899', '899', '899', '899', '951', '951', '951', '951', '996', '996', '996', '996'], 'NDB_862': ['103', '103', '103', '103', '1057', '1057', '1057', '1057', '1124', '1124', '1124', '1124', '1171', '1171', '1171', '1171', '1203', '1203', '1203', '1203', '1221', '1221', '1221', '1221', '1246', '1246', '1246', '1246', '1278', '1278', '1278', '1278', '1301', '1301', '1301', '1301', '1315', '1315', '1315', '1315', '1401', '1401', '1401', '1401', '1457', '1457', '1457', '1457', '1481', '1481', '1481', '1481', '1503', '1503', '1503', '1503', '1527', '1527', '1527', '1527', '153', '153', '153', '153', '185', '185', '185', '185', '229', '229', '229', '229', '255', '255', '255', '255', '275', '275', '275', '275', '290', '290', '290', '290', '338', '338', '338', '338', '373', '373', '373', '373', '406', '406', '406', '406', '436', '436', '436', '436', '479', '479', '479', '479', '521', '521', '521', '521', '540', '540', '540', '540', '574', '574', '574', '574', '602', '602', '602', '602', '64', '64', '64', '64', '659', '659', '659', '659', '683', '683', '683', '683', '710', '710', '710', '710', '78', '78', '78', '78', '792', '792', '792', '792', '819', '819', '819', '819', '852', '852', '852', '852', '895', '895', '895', '895', '933', '933', '933', '933', '952', '952', '952', '952', '992', '992', '992', '992'], 'NDB_881': ['1030', '1030', '1030', '1030', '1068', '1068', '1068', '1068', '1091', '1091', '1091', '1091', '1135', '1135', '1135', '1135', '1145', '1145', '1145', '1145', '1208', '1208', '1208', '1208', '123', '123', '123', '123', '1265', '1265', '1265', '1265', '1287', '1287', '1287', '1287', '1316', '1316', '1316', '1316', '1335', '1335', '1335', '1335', '1387', '1387', '1387', '1387', '144', '144', '144', '144', '1443', '1443', '1443', '1443', '1489', '1489', '1489', '1489', '1510', '1510', '1510', '1510', '182', '182', '182', '182', '239', '239', '239', '239', '255', '255', '255', '255', '26', '26', '26', '26', '270', '270', '270', '270', '306', '306', '306', '306', '331', '331', '331', '331', '377', '377', '377', '377', '407', '407', '407', '407', '443', '443', '443', '443', '477', '477', '477', '477', '504', '504', '504', '504', '528', '528', '528', '528', '567', '567', '567', '567', '583', '583', '583', '583', '647', '647', '647', '647', '682', '682', '682', '682', '728', '728', '728', '728', '760', '760', '760', '760', '77', '77', '77', '77', '795', '795', '795', '795', '861', '861', '861', '861', '889', '889', '889', '889', '930', '930', '930', '930', '990', '990', '990', '990'], 'NDB_730': ['1031', '1031', '1031', '1031', '1112', '1112', '1112', '1112', '1172', '1172', '1172', '1172', '1228', '1228', '1228', '1228', '1238', '1238', '1238', '1238', '1279', '1279', '1279', '1279', '1292', '1292', '1292', '1292', '1309', '1309', '1309', '1309', '132', '132', '132', '132', '1332', '1332', '1332', '1332', '1400', '1400', '1400', '1400', '1448', '1448', '1448', '1448', '1465', '1465', '1465', '1465', '1514', '1514', '1514', '1514', '1548', '1548', '1548', '1548', '172', '172', '172', '172', '200', '200', '200', '200', '250', '250', '250', '250', '271', '271', '271', '271', '300', '300', '300', '300', '334', '334', '334', '334', '379', '379', '379', '379', '406', '406', '406', '406', '454', '454', '454', '454', '46', '46', '46', '46', '475', '475', '475', '475', '5', '5', '5', '5', '516', '516', '516', '516', '569', '569', '569', '569', '656', '656', '656', '656', '669', '669', '669', '669', '688', '688', '688', '688', '704', '704', '704', '704', '750', '750', '750', '750', '766', '766', '766', '766', '834', '834', '834', '834', '849', '849', '849', '849', '918', '918', '918', '918', '95', '95', '95', '95', '956', '956', '956', '956', '976', '976', '976', '976'], 'NDB_694': ['1031', '1031', '1031', '1031', '1071', '1071', '1071', '1071', '1120', '1120', '1120', '1120', '1141', '1141', '1141', '1141', '1173', '1173', '1173', '1173', '1203', '1203', '1203', '1203', '1222', '1222', '1222', '1222', '127', '127', '127', '127', '1271', '1271', '1271', '1271', '1323', '1323', '1323', '1323', '1348', '1348', '1348', '1348', '1393', '1393', '1393', '1393', '1463', '1463', '1463', '1463', '1501', '1501', '1501', '1501', '174', '174', '174', '174', '221', '221', '221', '221', '268', '268', '268', '268', '277', '277', '277', '277', '313', '313', '313', '313', '32', '32', '32', '32', '364', '364', '364', '364', '393', '393', '393', '393', '431', '431', '431', '431', '461', '461', '461', '461', '519', '519', '519', '519', '543', '543', '543', '543', '587', '587', '587', '587', '612', '612', '612', '612', '670', '670', '670', '670', '69', '69', '69', '69', '724', '724', '724', '724', '745', '745', '745', '745', '782', '782', '782', '782', '804', '804', '804', '804', '836', '836', '836', '836', '848', '848', '848', '848', '85', '85', '85', '85', '916', '916', '916', '916', '954', '954', '954', '954', '982', '982', '982', '982'], 'NDB_824': ['1031', '1031', '1031', '1031', '107', '107', '107', '107', '1070', '1070', '1070', '1070', '1121', '1121', '1121', '1121', '1162', '1162', '1162', '1162', '1179', '1179', '1179', '1179', '1194', '1194', '1194', '1194', '1282', '1282', '1282', '1282', '1315', '1315', '1315', '1315', '1353', '1353', '1353', '1353', '1379', '1379', '1379', '1379', '140', '140', '140', '140', '1429', '1429', '1429', '1429', '1466', '1466', '1466', '1466', '1521', '1521', '1521', '1521', '157', '157', '157', '157', '201', '201', '201', '201', '216', '216', '216', '216', '235', '235', '235', '235', '309', '309', '309', '309', '33', '33', '33', '33', '331', '331', '331', '331', '371', '371', '371', '371', '404', '404', '404', '404', '467', '467', '467', '467', '493', '493', '493', '493', '532', '532', '532', '532', '562', '562', '562', '562', '600', '600', '600', '600', '627', '627', '627', '627', '674', '674', '674', '674', '703', '703', '703', '703', '735', '735', '735', '735', '761', '761', '761', '761', '796', '796', '796', '796', '851', '851', '851', '851', '864', '864', '864', '864', '88', '88', '88', '88', '900', '900', '900', '900', '932', '932', '932', '932', '985', '985', '985', '985'], 'NDB_880': ['1033', '1033', '1033', '1033', '1049', '1049', '1049', '1049', '109', '109', '109', '109', '1093', '1093', '1093', '1093', '1135', '1135', '1135', '1135', '1166', '1166', '1166', '1166', '1204', '1204', '1204', '1204', '1281', '1281', '1281', '1281', '1306', '1306', '1306', '1306', '1363', '1363', '1363', '1363', '1388', '1388', '1388', '1388', '139', '139', '139', '139', '1457', '1457', '1457', '1457', '1482', '1482', '1482', '1482', '180', '180', '180', '180', '227', '227', '227', '227', '243', '243', '243', '243', '271', '271', '271', '271', '297', '297', '297', '297', '341', '341', '341', '341', '35', '35', '35', '35', '380', '380', '380', '380', '409', '409', '409', '409', '464', '464', '464', '464', '481', '481', '481', '481', '493', '493', '493', '493', '546', '546', '546', '546', '554', '554', '554', '554', '564', '564', '564', '564', '616', '616', '616', '616', '673', '673', '673', '673', '705', '705', '705', '705', '732', '732', '732', '732', '752', '752', '752', '752', '787', '787', '787', '787', '81', '81', '81', '81', '838', '838', '838', '838', '870', '870', '870', '870', '891', '891', '891', '891', '916', '916', '916', '916', '987', '987', '987', '987'], 'NDB_788': ['1033', '1033', '1033', '1033', '1073', '1073', '1073', '1073', '1113', '1113', '1113', '1113', '1128', '1128', '1128', '1128', '1139', '1139', '1139', '1139', '1187', '1187', '1187', '1187', '12', '12', '12', '12', '1219', '1219', '1219', '1219', '1271', '1271', '1271', '1271', '128', '128', '128', '128', '1321', '1321', '1321', '1321', '1346', '1346', '1346', '1346', '1385', '1385', '1385', '1385', '149', '149', '149', '149', '1496', '1496', '1496', '1496', '173', '173', '173', '173', '203', '203', '203', '203', '248', '248', '248', '248', '275', '275', '275', '275', '335', '335', '335', '335', '350', '350', '350', '350', '394', '394', '394', '394', '413', '413', '413', '413', '467', '467', '467', '467', '500', '500', '500', '500', '537', '537', '537', '537', '568', '568', '568', '568', '593', '593', '593', '593', '615', '615', '615', '615', '672', '672', '672', '672', '684', '684', '684', '684', '709', '709', '709', '709', '74', '74', '74', '74', '760', '760', '760', '760', '779', '779', '779', '779', '853', '853', '853', '853', '862', '862', '862', '862', '918', '918', '918', '918', '961', '961', '961', '961', '995', '995', '995', '995'], 'NDB_875': ['1034', '1034', '1034', '1034', '104', '104', '104', '104', '1087', '1087', '1087', '1087', '1124', '1124', '1124', '1124', '1139', '1139', '1139', '1139', '1212', '1212', '1212', '1212', '1236', '1236', '1236', '1236', '1258', '1258', '1258', '1258', '1291', '1291', '1291', '1291', '1334', '1334', '1334', '1334', '1392', '1392', '1392', '1392', '1459', '1459', '1459', '1459', '1474', '1474', '1474', '1474', '1513', '1513', '1513', '1513', '155', '155', '155', '155', '193', '193', '193', '193', '208', '208', '208', '208', '224', '224', '224', '224', '250', '250', '250', '250', '28', '28', '28', '28', '285', '285', '285', '285', '329', '329', '329', '329', '361', '361', '361', '361', '377', '377', '377', '377', '428', '428', '428', '428', '465', '465', '465', '465', '495', '495', '495', '495', '532', '532', '532', '532', '589', '589', '589', '589', '625', '625', '625', '625', '679', '679', '679', '679', '70', '70', '70', '70', '720', '720', '720', '720', '741', '741', '741', '741', '765', '765', '765', '765', '812', '812', '812', '812', '826', '826', '826', '826', '878', '878', '878', '878', '916', '916', '916', '916', '951', '951', '951', '951', '969', '969', '969', '969'], 'NDB_653': ['1034', '1034', '1034', '1034', '1056', '1056', '1056', '1056', '1075', '1075', '1075', '1075', '112', '112', '112', '112', '1130', '1130', '1130', '1130', '1148', '1148', '1148', '1148', '1205', '1205', '1205', '1205', '1220', '1220', '1220', '1220', '1274', '1274', '1274', '1274', '1309', '1309', '1309', '1309', '1351', '1351', '1351', '1351', '1395', '1395', '1395', '1395', '1435', '1435', '1435', '1435', '1496', '1496', '1496', '1496', '150', '150', '150', '150', '192', '192', '192', '192', '20', '20', '20', '20', '218', '218', '218', '218', '254', '254', '254', '254', '277', '277', '277', '277', '290', '290', '290', '290', '306', '306', '306', '306', '357', '357', '357', '357', '386', '386', '386', '386', '442', '442', '442', '442', '470', '470', '470', '470', '522', '522', '522', '522', '549', '549', '549', '549', '590', '590', '590', '590', '651', '651', '651', '651', '657', '657', '657', '657', '66', '66', '66', '66', '699', '699', '699', '699', '731', '731', '731', '731', '774', '774', '774', '774', '810', '810', '810', '810', '841', '841', '841', '841', '875', '875', '875', '875', '922', '922', '922', '922', '957', '957', '957', '957', '972', '972', '972', '972'], 'NDB_974': ['1035', '1035', '1035', '1035', '1073', '1073', '1073', '1073', '1120', '1120', '1120', '1120', '1179', '1179', '1179', '1179', '1217', '1217', '1217', '1217', '1228', '1228', '1228', '1228', '126', '126', '126', '126', '1262', '1262', '1262', '1262', '1297', '1297', '1297', '1297', '1307', '1307', '1307', '1307', '1358', '1358', '1358', '1358', '1387', '1387', '1387', '1387', '1452', '1452', '1452', '1452', '1462', '1462', '1462', '1462', '1473', '1473', '1473', '1473', '161', '161', '161', '161', '220', '220', '220', '220', '252', '252', '252', '252', '285', '285', '285', '285', '326', '326', '326', '326', '339', '339', '339', '339', '34', '34', '34', '34', '376', '376', '376', '376', '399', '399', '399', '399', '422', '422', '422', '422', '437', '437', '437', '437', '517', '517', '517', '517', '555', '555', '555', '555', '606', '606', '606', '606', '652', '652', '652', '652', '692', '692', '692', '692', '7', '7', '7', '7', '718', '718', '718', '718', '757', '757', '757', '757', '79', '79', '79', '79', '793', '793', '793', '793', '815', '815', '815', '815', '844', '844', '844', '844', '907', '907', '907', '907', '952', '952', '952', '952', '988', '988', '988', '988'], 'NDB_853': ['1035', '1035', '1035', '1035', '1077', '1077', '1077', '1077', '1133', '1133', '1133', '1133', '1156', '1156', '1156', '1156', '1197', '1197', '1197', '1197', '1270', '1270', '1270', '1270', '1284', '1284', '1284', '1284', '1306', '1306', '1306', '1306', '1330', '1330', '1330', '1330', '134', '134', '134', '134', '1408', '1408', '1408', '1408', '1456', '1456', '1456', '1456', '1475', '1475', '1475', '1475', '1513', '1513', '1513', '1513', '1529', '1529', '1529', '1529', '179', '179', '179', '179', '230', '230', '230', '230', '254', '254', '254', '254', '26', '26', '26', '26', '308', '308', '308', '308', '327', '327', '327', '327', '363', '363', '363', '363', '395', '395', '395', '395', '430', '430', '430', '430', '458', '458', '458', '458', '521', '521', '521', '521', '538', '538', '538', '538', '560', '560', '560', '560', '575', '575', '575', '575', '6', '6', '6', '6', '656', '656', '656', '656', '69', '69', '69', '69', '695', '695', '695', '695', '725', '725', '725', '725', '761', '761', '761', '761', '779', '779', '779', '779', '821', '821', '821', '821', '850', '850', '850', '850', '903', '903', '903', '903', '926', '926', '926', '926', '970', '970', '970', '970'], 'NDB_934': ['1035', '1035', '1035', '1035', '1050', '1050', '1050', '1050', '1079', '1079', '1079', '1079', '1143', '1143', '1143', '1143', '12', '12', '12', '12', '1236', '1236', '1236', '1236', '124', '124', '124', '124', '1254', '1254', '1254', '1254', '1287', '1287', '1287', '1287', '1320', '1320', '1320', '1320', '1352', '1352', '1352', '1352', '1378', '1378', '1378', '1378', '1454', '1454', '1454', '1454', '1466', '1466', '1466', '1466', '1509', '1509', '1509', '1509', '180', '180', '180', '180', '205', '205', '205', '205', '253', '253', '253', '253', '273', '273', '273', '273', '296', '296', '296', '296', '318', '318', '318', '318', '355', '355', '355', '355', '406', '406', '406', '406', '449', '449', '449', '449', '45', '45', '45', '45', '483', '483', '483', '483', '497', '497', '497', '497', '530', '530', '530', '530', '558', '558', '558', '558', '601', '601', '601', '601', '657', '657', '657', '657', '697', '697', '697', '697', '739', '739', '739', '739', '797', '797', '797', '797', '855', '855', '855', '855', '868', '868', '868', '868', '906', '906', '906', '906', '92', '92', '92', '92', '928', '928', '928', '928', '971', '971', '971', '971'], 'NDB_917': ['1036', '1036', '1036', '1036', '1050', '1050', '1050', '1050', '1094', '1094', '1094', '1094', '1152', '1152', '1152', '1152', '119', '119', '119', '119', '1207', '1207', '1207', '1207', '1234', '1234', '1234', '1234', '1281', '1281', '1281', '1281', '1317', '1317', '1317', '1317', '1351', '1351', '1351', '1351', '1383', '1383', '1383', '1383', '1451', '1451', '1451', '1451', '1488', '1488', '1488', '1488', '1505', '1505', '1505', '1505', '162', '162', '162', '162', '186', '186', '186', '186', '21', '21', '21', '21', '212', '212', '212', '212', '233', '233', '233', '233', '270', '270', '270', '270', '288', '288', '288', '288', '33', '33', '33', '33', '340', '340', '340', '340', '370', '370', '370', '370', '395', '395', '395', '395', '465', '465', '465', '465', '501', '501', '501', '501', '544', '544', '544', '544', '585', '585', '585', '585', '615', '615', '615', '615', '678', '678', '678', '678', '704', '704', '704', '704', '737', '737', '737', '737', '763', '763', '763', '763', '783', '783', '783', '783', '819', '819', '819', '819', '848', '848', '848', '848', '888', '888', '888', '888', '897', '897', '897', '897', '955', '955', '955', '955', '981', '981', '981', '981'], 'NDB_838': ['1036', '1036', '1036', '1036', '106', '106', '106', '106', '1074', '1074', '1074', '1074', '1097', '1097', '1097', '1097', '1141', '1141', '1141', '1141', '1195', '1195', '1195', '1195', '1280', '1280', '1280', '1280', '1321', '1321', '1321', '1321', '1347', '1347', '1347', '1347', '1404', '1404', '1404', '1404', '141', '141', '141', '141', '1453', '1453', '1453', '1453', '1464', '1464', '1464', '1464', '1494', '1494', '1494', '1494', '1525', '1525', '1525', '1525', '169', '169', '169', '169', '219', '219', '219', '219', '23', '23', '23', '23', '256', '256', '256', '256', '289', '289', '289', '289', '306', '306', '306', '306', '344', '344', '344', '344', '355', '355', '355', '355', '402', '402', '402', '402', '435', '435', '435', '435', '481', '481', '481', '481', '499', '499', '499', '499', '533', '533', '533', '533', '569', '569', '569', '569', '609', '609', '609', '609', '652', '652', '652', '652', '698', '698', '698', '698', '716', '716', '716', '716', '794', '794', '794', '794', '826', '826', '826', '826', '84', '84', '84', '84', '864', '864', '864', '864', '890', '890', '890', '890', '903', '903', '903', '903', '960', '960', '960', '960', '999', '999', '999', '999'], 'NDB_716': ['1037', '1037', '1037', '1037', '1053', '1053', '1053', '1053', '107', '107', '107', '107', '1106', '1106', '1106', '1106', '1125', '1125', '1125', '1125', '1147', '1147', '1147', '1147', '1204', '1204', '1204', '1204', '1278', '1278', '1278', '1278', '1303', '1303', '1303', '1303', '1345', '1345', '1345', '1345', '1409', '1409', '1409', '1409', '144', '144', '144', '144', '1445', '1445', '1445', '1445', '1488', '1488', '1488', '1488', '1499', '1499', '1499', '1499', '189', '189', '189', '189', '204', '204', '204', '204', '257', '257', '257', '257', '310', '310', '310', '310', '336', '336', '336', '336', '366', '366', '366', '366', '392', '392', '392', '392', '461', '461', '461', '461', '472', '472', '472', '472', '516', '516', '516', '516', '537', '537', '537', '537', '565', '565', '565', '565', '60', '60', '60', '60', '621', '621', '621', '621', '664', '664', '664', '664', '697', '697', '697', '697', '70', '70', '70', '70', '721', '721', '721', '721', '762', '762', '762', '762', '777', '777', '777', '777', '821', '821', '821', '821', '845', '845', '845', '845', '892', '892', '892', '892', '913', '913', '913', '913', '957', '957', '957', '957', '968', '968', '968', '968'], 'NDB_899': ['1037', '1037', '1037', '1037', '1051', '1051', '1051', '1051', '109', '109', '109', '109', '1105', '1105', '1105', '1105', '1137', '1137', '1137', '1137', '1183', '1183', '1183', '1183', '1231', '1231', '1231', '1231', '1276', '1276', '1276', '1276', '1334', '1334', '1334', '1334', '1359', '1359', '1359', '1359', '1389', '1389', '1389', '1389', '1492', '1492', '1492', '1492', '150', '150', '150', '150', '1521', '1521', '1521', '1521', '178', '178', '178', '178', '19', '19', '19', '19', '196', '196', '196', '196', '226', '226', '226', '226', '299', '299', '299', '299', '3', '3', '3', '3', '322', '322', '322', '322', '360', '360', '360', '360', '385', '385', '385', '385', '435', '435', '435', '435', '482', '482', '482', '482', '523', '523', '523', '523', '566', '566', '566', '566', '602', '602', '602', '602', '658', '658', '658', '658', '711', '711', '711', '711', '739', '739', '739', '739', '754', '754', '754', '754', '801', '801', '801', '801', '813', '813', '813', '813', '858', '858', '858', '858', '87', '87', '87', '87', '909', '909', '909', '909', '930', '930', '930', '930', '965', '965', '965', '965', '994', '994', '994', '994'], 'NDB_712': ['1038', '1038', '1038', '1038', '1092', '1092', '1092', '1092', '1128', '1128', '1128', '1128', '1134', '1134', '1134', '1134', '1183', '1183', '1183', '1183', '1233', '1233', '1233', '1233', '1272', '1272', '1272', '1272', '1285', '1285', '1285', '1285', '1337', '1337', '1337', '1337', '1360', '1360', '1360', '1360', '1447', '1447', '1447', '1447', '146', '146', '146', '146', '1465', '1465', '1465', '1465', '1505', '1505', '1505', '1505', '170', '170', '170', '170', '18', '18', '18', '18', '210', '210', '210', '210', '234', '234', '234', '234', '304', '304', '304', '304', '318', '318', '318', '318', '337', '337', '337', '337', '380', '380', '380', '380', '410', '410', '410', '410', '432', '432', '432', '432', '44', '44', '44', '44', '477', '477', '477', '477', '511', '511', '511', '511', '557', '557', '557', '557', '570', '570', '570', '570', '653', '653', '653', '653', '710', '710', '710', '710', '731', '731', '731', '731', '755', '755', '755', '755', '76', '76', '76', '76', '799', '799', '799', '799', '826', '826', '826', '826', '877', '877', '877', '877', '913', '913', '913', '913', '953', '953', '953', '953', '970', '970', '970', '970'], 'NDB_721': ['1039', '1039', '1039', '1039', '1065', '1065', '1065', '1065', '1125', '1125', '1125', '1125', '1134', '1134', '1134', '1134', '1153', '1153', '1153', '1153', '1193', '1193', '1193', '1193', '1249', '1249', '1249', '1249', '128', '128', '128', '128', '1293', '1293', '1293', '1293', '1311', '1311', '1311', '1311', '1361', '1361', '1361', '1361', '1391', '1391', '1391', '1391', '1427', '1427', '1427', '1427', '1469', '1469', '1469', '1469', '1513', '1513', '1513', '1513', '168', '168', '168', '168', '186', '186', '186', '186', '207', '207', '207', '207', '223', '223', '223', '223', '239', '239', '239', '239', '293', '293', '293', '293', '333', '333', '333', '333', '37', '37', '37', '37', '389', '389', '389', '389', '411', '411', '411', '411', '422', '422', '422', '422', '470', '470', '470', '470', '519', '519', '519', '519', '554', '554', '554', '554', '594', '594', '594', '594', '613', '613', '613', '613', '674', '674', '674', '674', '680', '680', '680', '680', '707', '707', '707', '707', '764', '764', '764', '764', '80', '80', '80', '80', '809', '809', '809', '809', '814', '814', '814', '814', '839', '839', '839', '839', '893', '893', '893', '893', '974', '974', '974', '974'], 'NDB_910': ['1039', '1039', '1039', '1039', '1110', '1110', '1110', '1110', '1130', '1130', '1130', '1130', '1144', '1144', '1144', '1144', '1165', '1165', '1165', '1165', '1215', '1215', '1215', '1215', '1255', '1255', '1255', '1255', '1289', '1289', '1289', '1289', '1308', '1308', '1308', '1308', '1356', '1356', '1356', '1356', '1456', '1456', '1456', '1456', '146', '146', '146', '146', '1495', '1495', '1495', '1495', '1533', '1533', '1533', '1533', '163', '163', '163', '163', '17', '17', '17', '17', '211', '211', '211', '211', '263', '263', '263', '263', '312', '312', '312', '312', '33', '33', '33', '33', '344', '344', '344', '344', '382', '382', '382', '382', '397', '397', '397', '397', '414', '414', '414', '414', '438', '438', '438', '438', '480', '480', '480', '480', '529', '529', '529', '529', '546', '546', '546', '546', '589', '589', '589', '589', '611', '611', '611', '611', '677', '677', '677', '677', '692', '692', '692', '692', '713', '713', '713', '713', '74', '74', '74', '74', '766', '766', '766', '766', '807', '807', '807', '807', '817', '817', '817', '817', '858', '858', '858', '858', '892', '892', '892', '892', '982', '982', '982', '982', '992', '992', '992', '992'], 'NDB_865': ['104', '104', '104', '104', '1047', '1047', '1047', '1047', '1085', '1085', '1085', '1085', '1153', '1153', '1153', '1153', '1192', '1192', '1192', '1192', '1220', '1220', '1220', '1220', '1250', '1250', '1250', '1250', '1269', '1269', '1269', '1269', '1322', '1322', '1322', '1322', '133', '133', '133', '133', '1355', '1355', '1355', '1355', '1403', '1403', '1403', '1403', '1449', '1449', '1449', '1449', '1462', '1462', '1462', '1462', '1489', '1489', '1489', '1489', '1541', '1541', '1541', '1541', '172', '172', '172', '172', '246', '246', '246', '246', '262', '262', '262', '262', '307', '307', '307', '307', '336', '336', '336', '336', '356', '356', '356', '356', '36', '36', '36', '36', '375', '375', '375', '375', '432', '432', '432', '432', '458', '458', '458', '458', '493', '493', '493', '493', '545', '545', '545', '545', '563', '563', '563', '563', '585', '585', '585', '585', '646', '646', '646', '646', '670', '670', '670', '670', '693', '693', '693', '693', '71', '71', '71', '71', '723', '723', '723', '723', '785', '785', '785', '785', '805', '805', '805', '805', '822', '822', '822', '822', '862', '862', '862', '862', '908', '908', '908', '908', '965', '965', '965', '965'], 'NDB_773': ['1040', '1040', '1040', '1040', '1091', '1091', '1091', '1091', '1140', '1140', '1140', '1140', '116', '116', '116', '116', '1186', '1186', '1186', '1186', '1225', '1225', '1225', '1225', '1260', '1260', '1260', '1260', '1297', '1297', '1297', '1297', '1310', '1310', '1310', '1310', '1356', '1356', '1356', '1356', '1389', '1389', '1389', '1389', '1434', '1434', '1434', '1434', '145', '145', '145', '145', '1463', '1463', '1463', '1463', '1512', '1512', '1512', '1512', '165', '165', '165', '165', '218', '218', '218', '218', '256', '256', '256', '256', '315', '315', '315', '315', '337', '337', '337', '337', '35', '35', '35', '35', '375', '375', '375', '375', '394', '394', '394', '394', '421', '421', '421', '421', '425', '425', '425', '425', '462', '462', '462', '462', '524', '524', '524', '524', '57', '57', '57', '57', '574', '574', '574', '574', '622', '622', '622', '622', '655', '655', '655', '655', '690', '690', '690', '690', '729', '729', '729', '729', '775', '775', '775', '775', '809', '809', '809', '809', '829', '829', '829', '829', '849', '849', '849', '849', '899', '899', '899', '899', '932', '932', '932', '932', '968', '968', '968', '968', '980', '980', '980', '980'], 'NDB_737': ['1041', '1041', '1041', '1041', '1097', '1097', '1097', '1097', '11', '11', '11', '11', '1123', '1123', '1123', '1123', '1140', '1140', '1140', '1140', '1189', '1189', '1189', '1189', '121', '121', '121', '121', '1234', '1234', '1234', '1234', '1246', '1246', '1246', '1246', '1292', '1292', '1292', '1292', '1336', '1336', '1336', '1336', '1398', '1398', '1398', '1398', '1435', '1435', '1435', '1435', '1478', '1478', '1478', '1478', '1517', '1517', '1517', '1517', '170', '170', '170', '170', '203', '203', '203', '203', '252', '252', '252', '252', '273', '273', '273', '273', '286', '286', '286', '286', '316', '316', '316', '316', '384', '384', '384', '384', '400', '400', '400', '400', '440', '440', '440', '440', '485', '485', '485', '485', '509', '509', '509', '509', '528', '528', '528', '528', '564', '564', '564', '564', '596', '596', '596', '596', '612', '612', '612', '612', '668', '668', '668', '668', '691', '691', '691', '691', '727', '727', '727', '727', '768', '768', '768', '768', '80', '80', '80', '80', '807', '807', '807', '807', '813', '813', '813', '813', '859', '859', '859', '859', '902', '902', '902', '902', '96', '96', '96', '96', '985', '985', '985', '985'], 'NDB_706': ['1041', '1041', '1041', '1041', '1069', '1069', '1069', '1069', '1093', '1093', '1093', '1093', '110', '110', '110', '110', '1127', '1127', '1127', '1127', '1134', '1134', '1134', '1134', '1157', '1157', '1157', '1157', '1217', '1217', '1217', '1217', '1261', '1261', '1261', '1261', '1319', '1319', '1319', '1319', '1349', '1349', '1349', '1349', '137', '137', '137', '137', '1400', '1400', '1400', '1400', '1480', '1480', '1480', '1480', '1520', '1520', '1520', '1520', '1535', '1535', '1535', '1535', '178', '178', '178', '178', '215', '215', '215', '215', '253', '253', '253', '253', '311', '311', '311', '311', '336', '336', '336', '336', '351', '351', '351', '351', '396', '396', '396', '396', '459', '459', '459', '459', '471', '471', '471', '471', '496', '496', '496', '496', '532', '532', '532', '532', '57', '57', '57', '57', '580', '580', '580', '580', '647', '647', '647', '647', '678', '678', '678', '678', '68', '68', '68', '68', '699', '699', '699', '699', '722', '722', '722', '722', '794', '794', '794', '794', '804', '804', '804', '804', '852', '852', '852', '852', '863', '863', '863', '863', '926', '926', '926', '926', '962', '962', '962', '962', '986', '986', '986', '986'], 'NDB_977': ['1041', '1041', '1041', '1041', '1126', '1126', '1126', '1126', '1162', '1162', '1162', '1162', '1177', '1177', '1177', '1177', '119', '119', '119', '119', '1224', '1224', '1224', '1224', '1249', '1249', '1249', '1249', '1294', '1294', '1294', '1294', '1307', '1307', '1307', '1307', '1359', '1359', '1359', '1359', '1385', '1385', '1385', '1385', '143', '143', '143', '143', '1443', '1443', '1443', '1443', '1481', '1481', '1481', '1481', '188', '188', '188', '188', '20', '20', '20', '20', '213', '213', '213', '213', '237', '237', '237', '237', '282', '282', '282', '282', '315', '315', '315', '315', '383', '383', '383', '383', '403', '403', '403', '403', '43', '43', '43', '43', '439', '439', '439', '439', '476', '476', '476', '476', '511', '511', '511', '511', '550', '550', '550', '550', '586', '586', '586', '586', '611', '611', '611', '611', '670', '670', '670', '670', '721', '721', '721', '721', '740', '740', '740', '740', '789', '789', '789', '789', '81', '81', '81', '81', '811', '811', '811', '811', '840', '840', '840', '840', '865', '865', '865', '865', '890', '890', '890', '890', '928', '928', '928', '928', '951', '951', '951', '951', '993', '993', '993', '993'], 'NDB_790': ['1042', '1042', '1042', '1042', '1064', '1064', '1064', '1064', '1112', '1112', '1112', '1112', '1170', '1170', '1170', '1170', '1189', '1189', '1189', '1189', '12', '12', '12', '12', '120', '120', '120', '120', '1225', '1225', '1225', '1225', '1280', '1280', '1280', '1280', '1305', '1305', '1305', '1305', '1318', '1318', '1318', '1318', '1394', '1394', '1394', '1394', '1439', '1439', '1439', '1439', '1468', '1468', '1468', '1468', '1510', '1510', '1510', '1510', '155', '155', '155', '155', '226', '226', '226', '226', '257', '257', '257', '257', '305', '305', '305', '305', '339', '339', '339', '339', '356', '356', '356', '356', '38', '38', '38', '38', '398', '398', '398', '398', '415', '415', '415', '415', '429', '429', '429', '429', '451', '451', '451', '451', '501', '501', '501', '501', '540', '540', '540', '540', '561', '561', '561', '561', '582', '582', '582', '582', '613', '613', '613', '613', '675', '675', '675', '675', '69', '69', '69', '69', '699', '699', '699', '699', '740', '740', '740', '740', '796', '796', '796', '796', '832', '832', '832', '832', '846', '846', '846', '846', '910', '910', '910', '910', '931', '931', '931', '931', '989', '989', '989', '989'], 'NDB_989': ['1042', '1042', '1042', '1042', '1069', '1069', '1069', '1069', '1115', '1115', '1115', '1115', '1153', '1153', '1153', '1153', '1194', '1194', '1194', '1194', '1239', '1239', '1239', '1239', '1277', '1277', '1277', '1277', '130', '130', '130', '130', '1338', '1338', '1338', '1338', '1390', '1390', '1390', '1390', '1442', '1442', '1442', '1442', '1482', '1482', '1482', '1482', '1501', '1501', '1501', '1501', '1529', '1529', '1529', '1529', '16', '16', '16', '16', '173', '173', '173', '173', '222', '222', '222', '222', '259', '259', '259', '259', '276', '276', '276', '276', '326', '326', '326', '326', '348', '348', '348', '348', '370', '370', '370', '370', '386', '386', '386', '386', '440', '440', '440', '440', '478', '478', '478', '478', '504', '504', '504', '504', '518', '518', '518', '518', '557', '557', '557', '557', '602', '602', '602', '602', '625', '625', '625', '625', '63', '63', '63', '63', '686', '686', '686', '686', '717', '717', '717', '717', '767', '767', '767', '767', '811', '811', '811', '811', '855', '855', '855', '855', '866', '866', '866', '866', '892', '892', '892', '892', '935', '935', '935', '935', '94', '94', '94', '94', '960', '960', '960', '960', '977', '977', '977', '977'], 'NDB_860': ['1043', '1043', '1043', '1043', '1092', '1092', '1092', '1092', '1112', '1112', '1112', '1112', '1167', '1167', '1167', '1167', '1209', '1209', '1209', '1209', '121', '121', '121', '121', '1218', '1218', '1218', '1218', '1248', '1248', '1248', '1248', '1295', '1295', '1295', '1295', '1300', '1300', '1300', '1300', '1338', '1338', '1338', '1338', '1383', '1383', '1383', '1383', '141', '141', '141', '141', '1500', '1500', '1500', '1500', '1527', '1527', '1527', '1527', '189', '189', '189', '189', '198', '198', '198', '198', '238', '238', '238', '238', '24', '24', '24', '24', '312', '312', '312', '312', '345', '345', '345', '345', '365', '365', '365', '365', '393', '393', '393', '393', '422', '422', '422', '422', '460', '460', '460', '460', '489', '489', '489', '489', '510', '510', '510', '510', '551', '551', '551', '551', '556', '556', '556', '556', '601', '601', '601', '601', '655', '655', '655', '655', '693', '693', '693', '693', '725', '725', '725', '725', '77', '77', '77', '77', '787', '787', '787', '787', '812', '812', '812', '812', '816', '816', '816', '816', '860', '860', '860', '860', '919', '919', '919', '919', '972', '972', '972', '972', '993', '993', '993', '993'], 'NDB_935': ['1043', '1043', '1043', '1043', '1063', '1063', '1063', '1063', '11', '11', '11', '11', '1114', '1114', '1114', '1114', '1133', '1133', '1133', '1133', '1168', '1168', '1168', '1168', '1191', '1191', '1191', '1191', '1235', '1235', '1235', '1235', '1262', '1262', '1262', '1262', '1311', '1311', '1311', '1311', '1331', '1331', '1331', '1331', '1384', '1384', '1384', '1384', '143', '143', '143', '143', '1453', '1453', '1453', '1453', '1497', '1497', '1497', '1497', '182', '182', '182', '182', '195', '195', '195', '195', '217', '217', '217', '217', '242', '242', '242', '242', '299', '299', '299', '299', '32', '32', '32', '32', '323', '323', '323', '323', '373', '373', '373', '373', '416', '416', '416', '416', '424', '424', '424', '424', '45', '45', '45', '45', '464', '464', '464', '464', '491', '491', '491', '491', '544', '544', '544', '544', '571', '571', '571', '571', '582', '582', '582', '582', '626', '626', '626', '626', '713', '713', '713', '713', '741', '741', '741', '741', '758', '758', '758', '758', '777', '777', '777', '777', '849', '849', '849', '849', '863', '863', '863', '863', '901', '901', '901', '901', '959', '959', '959', '959', '977', '977', '977', '977'], 'NDB_710': ['1044', '1044', '1044', '1044', '1118', '1118', '1118', '1118', '1154', '1154', '1154', '1154', '116', '116', '116', '116', '1206', '1206', '1206', '1206', '1233', '1233', '1233', '1233', '1250', '1250', '1250', '1250', '1263', '1263', '1263', '1263', '1301', '1301', '1301', '1301', '1323', '1323', '1323', '1323', '1396', '1396', '1396', '1396', '1468', '1468', '1468', '1468', '147', '147', '147', '147', '1487', '1487', '1487', '1487', '189', '189', '189', '189', '237', '237', '237', '237', '264', '264', '264', '264', '286', '286', '286', '286', '31', '31', '31', '31', '322', '322', '322', '322', '367', '367', '367', '367', '4', '4', '4', '4', '408', '408', '408', '408', '424', '424', '424', '424', '446', '446', '446', '446', '499', '499', '499', '499', '538', '538', '538', '538', '583', '583', '583', '583', '597', '597', '597', '597', '625', '625', '625', '625', '673', '673', '673', '673', '684', '684', '684', '684', '715', '715', '715', '715', '797', '797', '797', '797', '820', '820', '820', '820', '841', '841', '841', '841', '88', '88', '88', '88', '904', '904', '904', '904', '929', '929', '929', '929', '988', '988', '988', '988'], 'NDB_778': ['1044', '1044', '1044', '1044', '1055', '1055', '1055', '1055', '1089', '1089', '1089', '1089', '1152', '1152', '1152', '1152', '1197', '1197', '1197', '1197', '1236', '1236', '1236', '1236', '125', '125', '125', '125', '1256', '1256', '1256', '1256', '1298', '1298', '1298', '1298', '1339', '1339', '1339', '1339', '1356', '1356', '1356', '1356', '1402', '1402', '1402', '1402', '1473', '1473', '1473', '1473', '1496', '1496', '1496', '1496', '1537', '1537', '1537', '1537', '177', '177', '177', '177', '204', '204', '204', '204', '243', '243', '243', '243', '258', '258', '258', '258', '298', '298', '298', '298', '32', '32', '32', '32', '348', '348', '348', '348', '365', '365', '365', '365', '410', '410', '410', '410', '429', '429', '429', '429', '455', '455', '455', '455', '498', '498', '498', '498', '542', '542', '542', '542', '566', '566', '566', '566', '594', '594', '594', '594', '618', '618', '618', '618', '669', '669', '669', '669', '703', '703', '703', '703', '73', '73', '73', '73', '734', '734', '734', '734', '759', '759', '759', '759', '785', '785', '785', '785', '852', '852', '852', '852', '927', '927', '927', '927', '96', '96', '96', '96', '990', '990', '990', '990'], 'NDB_994': ['1045', '1045', '1045', '1045', '1063', '1063', '1063', '1063', '108', '108', '108', '108', '1106', '1106', '1106', '1106', '1129', '1129', '1129', '1129', '1163', '1163', '1163', '1163', '1183', '1183', '1183', '1183', '1206', '1206', '1206', '1206', '1244', '1244', '1244', '1244', '1265', '1265', '1265', '1265', '1314', '1314', '1314', '1314', '133', '133', '133', '133', '1351', '1351', '1351', '1351', '1382', '1382', '1382', '1382', '1438', '1438', '1438', '1438', '1491', '1491', '1491', '1491', '1511', '1511', '1511', '1511', '158', '158', '158', '158', '205', '205', '205', '205', '251', '251', '251', '251', '30', '30', '30', '30', '312', '312', '312', '312', '338', '338', '338', '338', '394', '394', '394', '394', '415', '415', '415', '415', '447', '447', '447', '447', '459', '459', '459', '459', '495', '495', '495', '495', '509', '509', '509', '509', '592', '592', '592', '592', '656', '656', '656', '656', '680', '680', '680', '680', '724', '724', '724', '724', '778', '778', '778', '778', '79', '79', '79', '79', '803', '803', '803', '803', '825', '825', '825', '825', '851', '851', '851', '851', '917', '917', '917', '917', '949', '949', '949', '949', '987', '987', '987', '987'], 'NDB_826': ['1046', '1046', '1046', '1046', '1122', '1122', '1122', '1122', '1138', '1138', '1138', '1138', '1182', '1182', '1182', '1182', '1210', '1210', '1210', '1210', '1232', '1232', '1232', '1232', '1255', '1255', '1255', '1255', '1290', '1290', '1290', '1290', '1300', '1300', '1300', '1300', '1337', '1337', '1337', '1337', '1380', '1380', '1380', '1380', '1437', '1437', '1437', '1437', '144', '144', '144', '144', '1491', '1491', '1491', '1491', '1521', '1521', '1521', '1521', '166', '166', '166', '166', '174', '174', '174', '174', '200', '200', '200', '200', '229', '229', '229', '229', '265', '265', '265', '265', '279', '279', '279', '279', '329', '329', '329', '329', '340', '340', '340', '340', '36', '36', '36', '36', '367', '367', '367', '367', '412', '412', '412', '412', '456', '456', '456', '456', '474', '474', '474', '474', '492', '492', '492', '492', '542', '542', '542', '542', '571', '571', '571', '571', '623', '623', '623', '623', '65', '65', '65', '65', '663', '663', '663', '663', '727', '727', '727', '727', '775', '775', '775', '775', '811', '811', '811', '811', '844', '844', '844', '844', '868', '868', '868', '868', '903', '903', '903', '903', '967', '967', '967', '967'], 'NDB_647': ['1046', '1046', '1046', '1046', '105', '105', '105', '105', '1119', '1119', '1119', '1119', '1157', '1157', '1157', '1157', '1216', '1216', '1216', '1216', '1263', '1263', '1263', '1263', '1280', '1280', '1280', '1280', '1316', '1316', '1316', '1316', '1357', '1357', '1357', '1357', '1406', '1406', '1406', '1406', '1456', '1456', '1456', '1456', '1499', '1499', '1499', '1499', '153', '153', '153', '153', '1541', '1541', '1541', '1541', '191', '191', '191', '191', '199', '199', '199', '199', '227', '227', '227', '227', '244', '244', '244', '244', '280', '280', '280', '280', '29', '29', '29', '29', '334', '334', '334', '334', '361', '361', '361', '361', '390', '390', '390', '390', '449', '449', '449', '449', '485', '485', '485', '485', '523', '523', '523', '523', '551', '551', '551', '551', '588', '588', '588', '588', '627', '627', '627', '627', '672', '672', '672', '672', '696', '696', '696', '696', '724', '724', '724', '724', '767', '767', '767', '767', '809', '809', '809', '809', '840', '840', '840', '840', '872', '872', '872', '872', '901', '901', '901', '901', '91', '91', '91', '91', '934', '934', '934', '934', '955', '955', '955', '955', '966', '966', '966', '966'], 'NDB_643': ['1047', '1047', '1047', '1047', '1076', '1076', '1076', '1076', '1132', '1132', '1132', '1132', '1180', '1180', '1180', '1180', '1232', '1232', '1232', '1232', '1252', '1252', '1252', '1252', '1298', '1298', '1298', '1298', '1312', '1312', '1312', '1312', '1331', '1331', '1331', '1331', '1401', '1401', '1401', '1401', '1429', '1429', '1429', '1429', '1458', '1458', '1458', '1458', '1485', '1485', '1485', '1485', '1500', '1500', '1500', '1500', '155', '155', '155', '155', '169', '169', '169', '169', '17', '17', '17', '17', '234', '234', '234', '234', '269', '269', '269', '269', '281', '281', '281', '281', '311', '311', '311', '311', '346', '346', '346', '346', '381', '381', '381', '381', '399', '399', '399', '399', '453', '453', '453', '453', '485', '485', '485', '485', '5', '5', '5', '5', '522', '522', '522', '522', '579', '579', '579', '579', '620', '620', '620', '620', '649', '649', '649', '649', '690', '690', '690', '690', '709', '709', '709', '709', '752', '752', '752', '752', '801', '801', '801', '801', '821', '821', '821', '821', '857', '857', '857', '857', '894', '894', '894', '894', '94', '94', '94', '94', '949', '949', '949', '949', '971', '971', '971', '971'], 'NDB_905': ['1047', '1047', '1047', '1047', '1058', '1058', '1058', '1058', '1114', '1114', '1114', '1114', '1135', '1135', '1135', '1135', '1178', '1178', '1178', '1178', '1203', '1203', '1203', '1203', '1277', '1277', '1277', '1277', '1302', '1302', '1302', '1302', '1345', '1345', '1345', '1345', '1381', '1381', '1381', '1381', '1438', '1438', '1438', '1438', '1461', '1461', '1461', '1461', '1493', '1493', '1493', '1493', '152', '152', '152', '152', '178', '178', '178', '178', '207', '207', '207', '207', '25', '25', '25', '25', '261', '261', '261', '261', '280', '280', '280', '280', '325', '325', '325', '325', '342', '342', '342', '342', '354', '354', '354', '354', '404', '404', '404', '404', '417', '417', '417', '417', '462', '462', '462', '462', '500', '500', '500', '500', '535', '535', '535', '535', '570', '570', '570', '570', '584', '584', '584', '584', '614', '614', '614', '614', '62', '62', '62', '62', '669', '669', '669', '669', '695', '695', '695', '695', '742', '742', '742', '742', '786', '786', '786', '786', '843', '843', '843', '843', '869', '869', '869', '869', '890', '890', '890', '890', '919', '919', '919', '919', '92', '92', '92', '92', '988', '988', '988', '988'], 'NDB_965': ['1048', '1048', '1048', '1048', '1082', '1082', '1082', '1082', '1142', '1142', '1142', '1142', '1180', '1180', '1180', '1180', '1220', '1220', '1220', '1220', '1237', '1237', '1237', '1237', '124', '124', '124', '124', '1264', '1264', '1264', '1264', '1304', '1304', '1304', '1304', '1321', '1321', '1321', '1321', '145', '145', '145', '145', '1452', '1452', '1452', '1452', '1493', '1493', '1493', '1493', '1527', '1527', '1527', '1527', '168', '168', '168', '168', '19', '19', '19', '19', '213', '213', '213', '213', '270', '270', '270', '270', '279', '279', '279', '279', '332', '332', '332', '332', '363', '363', '363', '363', '379', '379', '379', '379', '466', '466', '466', '466', '503', '503', '503', '503', '531', '531', '531', '531', '580', '580', '580', '580', '598', '598', '598', '598', '617', '617', '617', '617', '63', '63', '63', '63', '679', '679', '679', '679', '703', '703', '703', '703', '733', '733', '733', '733', '782', '782', '782', '782', '803', '803', '803', '803', '820', '820', '820', '820', '847', '847', '847', '847', '902', '902', '902', '902', '925', '925', '925', '925', '93', '93', '93', '93', '979', '979', '979', '979', '997', '997', '997', '997'], 'NDB_751': ['1048', '1048', '1048', '1048', '110', '110', '110', '110', '1115', '1115', '1115', '1115', '1146', '1146', '1146', '1146', '1191', '1191', '1191', '1191', '1216', '1216', '1216', '1216', '1251', '1251', '1251', '1251', '129', '129', '129', '129', '1290', '1290', '1290', '1290', '1325', '1325', '1325', '1325', '1347', '1347', '1347', '1347', '1378', '1378', '1378', '1378', '1441', '1441', '1441', '1441', '1477', '1477', '1477', '1477', '1504', '1504', '1504', '1504', '1533', '1533', '1533', '1533', '162', '162', '162', '162', '206', '206', '206', '206', '238', '238', '238', '238', '272', '272', '272', '272', '28', '28', '28', '28', '283', '283', '283', '283', '316', '316', '316', '316', '374', '374', '374', '374', '417', '417', '417', '417', '451', '451', '451', '451', '479', '479', '479', '479', '526', '526', '526', '526', '565', '565', '565', '565', '608', '608', '608', '608', '651', '651', '651', '651', '689', '689', '689', '689', '707', '707', '707', '707', '752', '752', '752', '752', '770', '770', '770', '770', '79', '79', '79', '79', '835', '835', '835', '835', '873', '873', '873', '873', '896', '896', '896', '896', '932', '932', '932', '932', '950', '950', '950', '950', '970', '970', '970', '970'], 'NDB_665': ['1049', '1049', '1049', '1049', '1067', '1067', '1067', '1067', '1117', '1117', '1117', '1117', '1147', '1147', '1147', '1147', '1179', '1179', '1179', '1179', '1225', '1225', '1225', '1225', '1245', '1245', '1245', '1245', '1254', '1254', '1254', '1254', '127', '127', '127', '127', '1337', '1337', '1337', '1337', '1395', '1395', '1395', '1395', '1444', '1444', '1444', '1444', '1470', '1470', '1470', '1470', '1508', '1508', '1508', '1508', '164', '164', '164', '164', '209', '209', '209', '209', '223', '223', '223', '223', '25', '25', '25', '25', '269', '269', '269', '269', '292', '292', '292', '292', '321', '321', '321', '321', '368', '368', '368', '368', '4', '4', '4', '4', '402', '402', '402', '402', '458', '458', '458', '458', '482', '482', '482', '482', '495', '495', '495', '495', '544', '544', '544', '544', '591', '591', '591', '591', '660', '660', '660', '660', '682', '682', '682', '682', '717', '717', '717', '717', '768', '768', '768', '768', '806', '806', '806', '806', '817', '817', '817', '817', '840', '840', '840', '840', '911', '911', '911', '911', '924', '924', '924', '924', '95', '95', '95', '95', '950', '950', '950', '950', '997', '997', '997', '997'], 'NDB_779': ['1050', '1050', '1050', '1050', '1067', '1067', '1067', '1067', '1076', '1076', '1076', '1076', '1097', '1097', '1097', '1097', '1176', '1176', '1176', '1176', '122', '122', '122', '122', '1226', '1226', '1226', '1226', '1278', '1278', '1278', '1278', '1304', '1304', '1304', '1304', '1330', '1330', '1330', '1330', '1396', '1396', '1396', '1396', '140', '140', '140', '140', '1455', '1455', '1455', '1455', '1491', '1491', '1491', '1491', '1533', '1533', '1533', '1533', '172', '172', '172', '172', '2', '2', '2', '2', '226', '226', '226', '226', '266', '266', '266', '266', '282', '282', '282', '282', '29', '29', '29', '29', '342', '342', '342', '342', '393', '393', '393', '393', '416', '416', '416', '416', '437', '437', '437', '437', '473', '473', '473', '473', '505', '505', '505', '505', '548', '548', '548', '548', '578', '578', '578', '578', '594', '594', '594', '594', '619', '619', '619', '619', '667', '667', '667', '667', '72', '72', '72', '72', '726', '726', '726', '726', '728', '728', '728', '728', '791', '791', '791', '791', '828', '828', '828', '828', '866', '866', '866', '866', '915', '915', '915', '915', '953', '953', '953', '953', '991', '991', '991', '991'], 'NDB_1028': ['1051', '1051', '1051', '1051', '1066', '1066', '1066', '1066', '1080', '1080', '1080', '1080', '1131', '1131', '1131', '1131', '1166', '1166', '1166', '1166', '1218', '1218', '1218', '1218', '1272', '1272', '1272', '1272', '130', '130', '130', '130', '1303', '1303', '1303', '1303', '1344', '1344', '1344', '1344', '1398', '1398', '1398', '1398', '1454', '1454', '1454', '1454', '1471', '1471', '1471', '1471', '1504', '1504', '1504', '1504', '1537', '1537', '1537', '1537', '17', '17', '17', '17', '171', '171', '171', '171', '197', '197', '197', '197', '247', '247', '247', '247', '309', '309', '309', '309', '338', '338', '338', '338', '34', '34', '34', '34', '375', '375', '375', '375', '396', '396', '396', '396', '4', '4', '4', '4', '427', '427', '427', '427', '470', '470', '470', '470', '491', '491', '491', '491', '520', '520', '520', '520', '561', '561', '561', '561', '578', '578', '578', '578', '617', '617', '617', '617', '672', '672', '672', '672', '681', '681', '681', '681', '698', '698', '698', '698', '745', '745', '745', '745', '783', '783', '783', '783', '800', '800', '800', '800', '854', '854', '854', '854', '920', '920', '920', '920', '969', '969', '969', '969'], 'NDB_652': ['1052', '1052', '1052', '1052', '1116', '1116', '1116', '1116', '1136', '1136', '1136', '1136', '1186', '1186', '1186', '1186', '120', '120', '120', '120', '1210', '1210', '1210', '1210', '1247', '1247', '1247', '1247', '1288', '1288', '1288', '1288', '1341', '1341', '1341', '1341', '141', '141', '141', '141', '1444', '1444', '1444', '1444', '1471', '1471', '1471', '1471', '15', '15', '15', '15', '1511', '1511', '1511', '1511', '1542', '1542', '1542', '1542', '185', '185', '185', '185', '196', '196', '196', '196', '243', '243', '243', '243', '268', '268', '268', '268', '288', '288', '288', '288', '333', '333', '333', '333', '388', '388', '388', '388', '404', '404', '404', '404', '439', '439', '439', '439', '483', '483', '483', '483', '507', '507', '507', '507', '531', '531', '531', '531', '576', '576', '576', '576', '590', '590', '590', '590', '628', '628', '628', '628', '668', '668', '668', '668', '686', '686', '686', '686', '719', '719', '719', '719', '766', '766', '766', '766', '808', '808', '808', '808', '845', '845', '845', '845', '873', '873', '873', '873', '90', '90', '90', '90', '901', '901', '901', '901', '926', '926', '926', '926', '984', '984', '984', '984'], 'NDB_836': ['1052', '1052', '1052', '1052', '106', '106', '106', '106', '1118', '1118', '1118', '1118', '1148', '1148', '1148', '1148', '119', '119', '119', '119', '1192', '1192', '1192', '1192', '1226', '1226', '1226', '1226', '1256', '1256', '1256', '1256', '1310', '1310', '1310', '1310', '134', '134', '134', '134', '1362', '1362', '1362', '1362', '1398', '1398', '1398', '1398', '1436', '1436', '1436', '1436', '1461', '1461', '1461', '1461', '1503', '1503', '1503', '1503', '222', '222', '222', '222', '249', '249', '249', '249', '285', '285', '285', '285', '303', '303', '303', '303', '351', '351', '351', '351', '397', '397', '397', '397', '453', '453', '453', '453', '477', '477', '477', '477', '510', '510', '510', '510', '565', '565', '565', '565', '587', '587', '587', '587', '650', '650', '650', '650', '666', '666', '666', '666', '67', '67', '67', '67', '702', '702', '702', '702', '743', '743', '743', '743', '764', '764', '764', '764', '803', '803', '803', '803', '823', '823', '823', '823', '83', '83', '83', '83', '868', '868', '868', '868', '9', '9', '9', '9', '905', '905', '905', '905', '924', '924', '924', '924', '956', '956', '956', '956', '978', '978', '978', '978'], 'NDB_889': ['1057', '1057', '1057', '1057', '1079', '1079', '1079', '1079', '111', '111', '111', '111', '1133', '1133', '1133', '1133', '1154', '1154', '1154', '1154', '1205', '1205', '1205', '1205', '1256', '1256', '1256', '1256', '1295', '1295', '1295', '1295', '1340', '1340', '1340', '1340', '138', '138', '138', '138', '1406', '1406', '1406', '1406', '1449', '1449', '1449', '1449', '1498', '1498', '1498', '1498', '1508', '1508', '1508', '1508', '1525', '1525', '1525', '1525', '160', '160', '160', '160', '216', '216', '216', '216', '236', '236', '236', '236', '3', '3', '3', '3', '302', '302', '302', '302', '317', '317', '317', '317', '350', '350', '350', '350', '387', '387', '387', '387', '425', '425', '425', '425', '43', '43', '43', '43', '443', '443', '443', '443', '516', '516', '516', '516', '531', '531', '531', '531', '555', '555', '555', '555', '589', '589', '589', '589', '653', '653', '653', '653', '691', '691', '691', '691', '705', '705', '705', '705', '751', '751', '751', '751', '770', '770', '770', '770', '832', '832', '832', '832', '86', '86', '86', '86', '869', '869', '869', '869', '897', '897', '897', '897', '949', '949', '949', '949', '974', '974', '974', '974'], 'NDB_940': ['1058', '1058', '1058', '1058', '1070', '1070', '1070', '1070', '1076', '1076', '1076', '1076', '108', '108', '108', '108', '1131', '1131', '1131', '1131', '1173', '1173', '1173', '1173', '1182', '1182', '1182', '1182', '1227', '1227', '1227', '1227', '1273', '1273', '1273', '1273', '1325', '1325', '1325', '1325', '1346', '1346', '1346', '1346', '138', '138', '138', '138', '1384', '1384', '1384', '1384', '1439', '1439', '1439', '1439', '1475', '1475', '1475', '1475', '1515', '1515', '1515', '1515', '190', '190', '190', '190', '20', '20', '20', '20', '212', '212', '212', '212', '225', '225', '225', '225', '260', '260', '260', '260', '297', '297', '297', '297', '320', '320', '320', '320', '377', '377', '377', '377', '405', '405', '405', '405', '438', '438', '438', '438', '475', '475', '475', '475', '523', '523', '523', '523', '556', '556', '556', '556', '57', '57', '57', '57', '598', '598', '598', '598', '621', '621', '621', '621', '686', '686', '686', '686', '712', '712', '712', '712', '768', '768', '768', '768', '805', '805', '805', '805', '839', '839', '839', '839', '876', '876', '876', '876', '905', '905', '905', '905', '973', '973', '973', '973', '999', '999', '999', '999'], 'NDB_734': ['1059', '1059', '1059', '1059', '1118', '1118', '1118', '1118', '1174', '1174', '1174', '1174', '1190', '1190', '1190', '1190', '1211', '1211', '1211', '1211', '1273', '1273', '1273', '1273', '1295', '1295', '1295', '1295', '1319', '1319', '1319', '1319', '1352', '1352', '1352', '1352', '1409', '1409', '1409', '1409', '143', '143', '143', '143', '1479', '1479', '1479', '1479', '1511', '1511', '1511', '1511', '1526', '1526', '1526', '1526', '18', '18', '18', '18', '181', '181', '181', '181', '207', '207', '207', '207', '229', '229', '229', '229', '276', '276', '276', '276', '335', '335', '335', '335', '391', '391', '391', '391', '414', '414', '414', '414', '463', '463', '463', '463', '468', '468', '468', '468', '47', '47', '47', '47', '490', '490', '490', '490', '502', '502', '502', '502', '564', '564', '564', '564', '599', '599', '599', '599', '620', '620', '620', '620', '675', '675', '675', '675', '694', '694', '694', '694', '723', '723', '723', '723', '759', '759', '759', '759', '783', '783', '783', '783', '850', '850', '850', '850', '864', '864', '864', '864', '889', '889', '889', '889', '925', '925', '925', '925', '95', '95', '95', '95', '980', '980', '980', '980'], 'NDB_1027': ['1060', '1060', '1060', '1060', '1075', '1075', '1075', '1075', '1111', '1111', '1111', '1111', '116', '116', '116', '116', '1160', '1160', '1160', '1160', '1227', '1227', '1227', '1227', '1242', '1242', '1242', '1242', '1281', '1281', '1281', '1281', '1340', '1340', '1340', '1340', '1400', '1400', '1400', '1400', '1450', '1450', '1450', '1450', '1470', '1470', '1470', '1470', '1504', '1504', '1504', '1504', '1529', '1529', '1529', '1529', '156', '156', '156', '156', '193', '193', '193', '193', '211', '211', '211', '211', '228', '228', '228', '228', '295', '295', '295', '295', '313', '313', '313', '313', '346', '346', '346', '346', '352', '352', '352', '352', '36', '36', '36', '36', '376', '376', '376', '376', '416', '416', '416', '416', '435', '435', '435', '435', '487', '487', '487', '487', '500', '500', '500', '500', '534', '534', '534', '534', '573', '573', '573', '573', '645', '645', '645', '645', '668', '668', '668', '668', '697', '697', '697', '697', '73', '73', '73', '73', '743', '743', '743', '743', '753', '753', '753', '753', '799', '799', '799', '799', '861', '861', '861', '861', '925', '925', '925', '925', '966', '966', '966', '966', '983', '983', '983', '983'], 'NDB_1060': ['1060', '1060', '1060', '1060', '1117', '1117', '1117', '1117', '1172', '1172', '1172', '1172', '1194', '1194', '1194', '1194', '1212', '1212', '1212', '1212', '1248', '1248', '1248', '1248', '1263', '1263', '1263', '1263', '1318', '1318', '1318', '1318', '1354', '1354', '1354', '1354', '1427', '1427', '1427', '1427', '148', '148', '148', '148', '1492', '1492', '1492', '1492', '161', '161', '161', '161', '197', '197', '197', '197', '225', '225', '225', '225', '272', '272', '272', '272', '302', '302', '302', '302', '330', '330', '330', '330', '366', '366', '366', '366', '403', '403', '403', '403', '464', '464', '464', '464', '507', '507', '507', '507', '535', '535', '535', '535', '558', '558', '558', '558', '595', '595', '595', '595', '6', '6', '6', '6', '619', '619', '619', '619', '663', '663', '663', '663', '706', '706', '706', '706', '738', '738', '738', '738', '759', '759', '759', '759', '76', '76', '76', '76', '782', '782', '782', '782', '823', '823', '823', '823', '862', '862', '862', '862', '888', '888', '888', '888', '89', '89', '89', '89', '917', '917', '917', '917', '962', '962', '962', '962', '992', '992', '992', '992'], 'NDB_760': ['1061', '1061', '1061', '1061', '1077', '1077', '1077', '1077', '1107', '1107', '1107', '1107', '117', '117', '117', '117', '1181', '1181', '1181', '1181', '1196', '1196', '1196', '1196', '1241', '1241', '1241', '1241', '1271', '1271', '1271', '1271', '1286', '1286', '1286', '1286', '1341', '1341', '1341', '1341', '1359', '1359', '1359', '1359', '137', '137', '137', '137', '1382', '1382', '1382', '1382', '1488', '1488', '1488', '1488', '1530', '1530', '1530', '1530', '18', '18', '18', '18', '193', '193', '193', '193', '216', '216', '216', '216', '242', '242', '242', '242', '266', '266', '266', '266', '280', '280', '280', '280', '320', '320', '320', '320', '365', '365', '365', '365', '408', '408', '408', '408', '444', '444', '444', '444', '474', '474', '474', '474', '529', '529', '529', '529', '552', '552', '552', '552', '569', '569', '569', '569', '603', '603', '603', '603', '660', '660', '660', '660', '690', '690', '690', '690', '707', '707', '707', '707', '75', '75', '75', '75', '757', '757', '757', '757', '776', '776', '776', '776', '824', '824', '824', '824', '859', '859', '859', '859', '906', '906', '906', '906', '978', '978', '978', '978', '989', '989', '989', '989'], 'NDB_805': ['1061', '1061', '1061', '1061', '1083', '1083', '1083', '1083', '1126', '1126', '1126', '1126', '1138', '1138', '1138', '1138', '1188', '1188', '1188', '1188', '1218', '1218', '1218', '1218', '1264', '1264', '1264', '1264', '1289', '1289', '1289', '1289', '1317', '1317', '1317', '1317', '1354', '1354', '1354', '1354', '1396', '1396', '1396', '1396', '1439', '1439', '1439', '1439', '1469', '1469', '1469', '1469', '1490', '1490', '1490', '1490', '164', '164', '164', '164', '176', '176', '176', '176', '199', '199', '199', '199', '245', '245', '245', '245', '283', '283', '283', '283', '327', '327', '327', '327', '341', '341', '341', '341', '373', '373', '373', '373', '400', '400', '400', '400', '426', '426', '426', '426', '455', '455', '455', '455', '47', '47', '47', '47', '5', '5', '5', '5', '506', '506', '506', '506', '541', '541', '541', '541', '587', '587', '587', '587', '614', '614', '614', '614', '653', '653', '653', '653', '687', '687', '687', '687', '711', '711', '711', '711', '760', '760', '760', '760', '784', '784', '784', '784', '84', '84', '84', '84', '855', '855', '855', '855', '921', '921', '921', '921', '968', '968', '968', '968', '996', '996', '996', '996'], 'NDB_946': ['1061', '1061', '1061', '1061', '1105', '1105', '1105', '1105', '1124', '1124', '1124', '1124', '114', '114', '114', '114', '1145', '1145', '1145', '1145', '1213', '1213', '1213', '1213', '1248', '1248', '1248', '1248', '1269', '1269', '1269', '1269', '128', '128', '128', '128', '1306', '1306', '1306', '1306', '1358', '1358', '1358', '1358', '1406', '1406', '1406', '1406', '1450', '1450', '1450', '1450', '1486', '1486', '1486', '1486', '187', '187', '187', '187', '201', '201', '201', '201', '23', '23', '23', '23', '261', '261', '261', '261', '286', '286', '286', '286', '328', '328', '328', '328', '372', '372', '372', '372', '410', '410', '410', '410', '438', '438', '438', '438', '472', '472', '472', '472', '508', '508', '508', '508', '549', '549', '549', '549', '579', '579', '579', '579', '599', '599', '599', '599', '623', '623', '623', '623', '688', '688', '688', '688', '722', '722', '722', '722', '756', '756', '756', '756', '800', '800', '800', '800', '828', '828', '828', '828', '83', '83', '83', '83', '873', '873', '873', '873', '898', '898', '898', '898', '958', '958', '958', '958', '98', '98', '98', '98', '997', '997', '997', '997'], 'NDB_728': ['1062', '1062', '1062', '1062', '107', '107', '107', '107', '1119', '1119', '1119', '1119', '1137', '1137', '1137', '1137', '1190', '1190', '1190', '1190', '1224', '1224', '1224', '1224', '1270', '1270', '1270', '1270', '1332', '1332', '1332', '1332', '1381', '1381', '1381', '1381', '1487', '1487', '1487', '1487', '1502', '1502', '1502', '1502', '1535', '1535', '1535', '1535', '158', '158', '158', '158', '177', '177', '177', '177', '198', '198', '198', '198', '212', '212', '212', '212', '23', '23', '23', '23', '242', '242', '242', '242', '284', '284', '284', '284', '324', '324', '324', '324', '356', '356', '356', '356', '376', '376', '376', '376', '40', '40', '40', '40', '440', '440', '440', '440', '480', '480', '480', '480', '519', '519', '519', '519', '539', '539', '539', '539', '558', '558', '558', '558', '576', '576', '576', '576', '624', '624', '624', '624', '679', '679', '679', '679', '687', '687', '687', '687', '701', '701', '701', '701', '751', '751', '751', '751', '792', '792', '792', '792', '835', '835', '835', '835', '875', '875', '875', '875', '894', '894', '894', '894', '911', '911', '911', '911', '953', '953', '953', '953', '961', '961', '961', '961'], 'NDB_856': ['1064', '1064', '1064', '1064', '1094', '1094', '1094', '1094', '1144', '1144', '1144', '1144', '1229', '1229', '1229', '1229', '1246', '1246', '1246', '1246', '125', '125', '125', '125', '1276', '1276', '1276', '1276', '1332', '1332', '1332', '1332', '1361', '1361', '1361', '1361', '139', '139', '139', '139', '1403', '1403', '1403', '1403', '1430', '1430', '1430', '1430', '1459', '1459', '1459', '1459', '1464', '1464', '1464', '1464', '1499', '1499', '1499', '1499', '190', '190', '190', '190', '206', '206', '206', '206', '249', '249', '249', '249', '296', '296', '296', '296', '332', '332', '332', '332', '352', '352', '352', '352', '407', '407', '407', '407', '42', '42', '42', '42', '430', '430', '430', '430', '459', '459', '459', '459', '520', '520', '520', '520', '581', '581', '581', '581', '595', '595', '595', '595', '614', '614', '614', '614', '671', '671', '671', '671', '685', '685', '685', '685', '72', '72', '72', '72', '744', '744', '744', '744', '789', '789', '789', '789', '810', '810', '810', '810', '815', '815', '815', '815', '853', '853', '853', '853', '90', '90', '90', '90', '912', '912', '912', '912', '929', '929', '929', '929', '980', '980', '980', '980'], 'NDB_961': ['1065', '1065', '1065', '1065', '1089', '1089', '1089', '1089', '115', '115', '115', '115', '1155', '1155', '1155', '1155', '1189', '1189', '1189', '1189', '1213', '1213', '1213', '1213', '1257', '1257', '1257', '1257', '1288', '1288', '1288', '1288', '131', '131', '131', '131', '1314', '1314', '1314', '1314', '1350', '1350', '1350', '1350', '1395', '1395', '1395', '1395', '14', '14', '14', '14', '1445', '1445', '1445', '1445', '1461', '1461', '1461', '1461', '1501', '1501', '1501', '1501', '162', '162', '162', '162', '214', '214', '214', '214', '241', '241', '241', '241', '300', '300', '300', '300', '323', '323', '323', '323', '371', '371', '371', '371', '421', '421', '421', '421', '426', '426', '426', '426', '43', '43', '43', '43', '466', '466', '466', '466', '502', '502', '502', '502', '529', '529', '529', '529', '573', '573', '573', '573', '597', '597', '597', '597', '663', '663', '663', '663', '710', '710', '710', '710', '736', '736', '736', '736', '77', '77', '77', '77', '793', '793', '793', '793', '837', '837', '837', '837', '878', '878', '878', '878', '894', '894', '894', '894', '912', '912', '912', '912', '965', '965', '965', '965', '999', '999', '999', '999']}\n",
      " > Done.\n",
      "-- inverting region_to_readout dict\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NDB_1027 -> ['1060', '1075', '1111', '116', '1160', '1227', '1242', '1281', '1340', '1400', '1450', '1470', '1504', '1529', '156', '193', '211', '228', '295', '313', '346', '352', '36', '376', '416', '435', '487', '500', '534', '573', '645', '668', '697', '73', '743', '753', '799', '861', '925', '966', '983']\n",
      "NDB_1028 -> ['1051', '1066', '1080', '1131', '1166', '1218', '1272', '130', '1303', '1344', '1398', '1454', '1471', '1504', '1537', '17', '171', '197', '247', '309', '338', '34', '375', '396', '4', '427', '470', '491', '520', '561', '578', '617', '672', '681', '698', '745', '783', '800', '854', '920', '969']\n",
      "NDB_1033 -> ['1002', '1024', '1042', '1108', '1167', '118', '1196', '1228', '1258', '1339', '1379', '1436', '1476', '1490', '153', '184', '196', '209', '251', '293', '343', '35', '358', '391', '450', '484', '497', '536', '584', '617', '649', '65', '713', '731', '756', '794', '827', '865', '888', '929', '973']\n",
      "NDB_1060 -> ['1060', '1117', '1172', '1194', '1212', '1248', '1263', '1318', '1354', '1427', '148', '1492', '161', '197', '225', '272', '302', '330', '366', '403', '464', '507', '535', '558', '595', '6', '619', '663', '706', '738', '759', '76', '782', '823', '862', '888', '89', '917', '962', '992']\n",
      "NDB_1066 -> ['1', '1036', '1073', '1114', '1157', '121', '1232', '1241', '1269', '1285', '1343', '1378', '1409', '1459', '147', '1472', '1518', '175', '213', '228', '27', '278', '303', '343', '364', '398', '469', '482', '494', '504', '560', '578', '628', '729', '749', '792', '834', '86', '867', '921', '987']\n",
      "NDB_1075 -> ['1024', '1051', '1090', '1146', '1193', '1238', '1245', '1282', '134', '1342', '1380', '1440', '1472', '1510', '169', '22', '221', '252', '275', '304', '347', '372', '405', '42', '444', '476', '498', '541', '563', '597', '655', '701', '732', '753', '790', '845', '866', '897', '92', '934', '975']\n",
      "NDB_631 -> ['1', '1001', '1032', '1058', '1120', '114', '1170', '1235', '1252', '1294', '1316', '1334', '14', '1408', '1446', '1478', '1516', '1536', '177', '194', '215', '236', '288', '318', '368', '414', '444', '473', '499', '536', '574', '600', '63', '651', '693', '708', '784', '802', '853', '891', '956']\n",
      "NDB_634 -> ['1001', '1040', '1078', '1128', '1174', '1197', '1240', '1245', '1261', '1302', '1324', '1392', '1441', '1471', '1487', '1530', '163', '175', '219', '251', '282', '305', '349', '389', '412', '449', '46', '486', '508', '528', '581', '616', '648', '7', '711', '738', '788', '819', '837', '905', '94', '957']\n",
      "NDB_637 -> ['10', '1045', '1072', '1119', '1143', '1184', '1229', '1262', '1289', '129', '1323', '1350', '1399', '1450', '1476', '148', '1512', '173', '210', '271', '290', '314', '345', '396', '412', '463', '473', '498', '533', '562', '598', '654', '662', '682', '71', '719', '780', '801', '859', '923', '964']\n",
      "NDB_641 -> ['1025', '1056', '1110', '1156', '1184', '1216', '1275', '1304', '1338', '137', '1386', '1438', '1470', '1517', '165', '183', '208', '262', '284', '304', '349', '371', '403', '455', '481', '490', '527', '582', '593', '623', '64', '664', '708', '737', '795', '815', '854', '86', '893', '909', '973']\n",
      "NDB_643 -> ['1047', '1076', '1132', '1180', '1232', '1252', '1298', '1312', '1331', '1401', '1429', '1458', '1485', '1500', '155', '169', '17', '234', '269', '281', '311', '346', '381', '399', '453', '485', '5', '522', '579', '620', '649', '690', '709', '752', '801', '821', '857', '894', '94', '949', '971']\n",
      "NDB_647 -> ['1046', '105', '1119', '1157', '1216', '1263', '1280', '1316', '1357', '1406', '1456', '1499', '153', '1541', '191', '199', '227', '244', '280', '29', '334', '361', '390', '449', '485', '523', '551', '588', '627', '672', '696', '724', '767', '809', '840', '872', '901', '91', '934', '955', '966']\n",
      "NDB_652 -> ['1052', '1116', '1136', '1186', '120', '1210', '1247', '1288', '1341', '141', '1444', '1471', '15', '1511', '1542', '185', '196', '243', '268', '288', '333', '388', '404', '439', '483', '507', '531', '576', '590', '628', '668', '686', '719', '766', '808', '845', '873', '90', '901', '926', '984']\n",
      "NDB_653 -> ['1034', '1056', '1075', '112', '1130', '1148', '1205', '1220', '1274', '1309', '1351', '1395', '1435', '1496', '150', '192', '20', '218', '254', '277', '290', '306', '357', '386', '442', '470', '522', '549', '590', '651', '657', '66', '699', '731', '774', '810', '841', '875', '922', '957', '972']\n",
      "NDB_661 -> ['1005', '1052', '1080', '112', '1172', '1204', '1235', '1261', '1287', '1307', '1357', '1403', '1437', '1472', '1512', '1525', '154', '170', '19', '214', '259', '284', '30', '319', '354', '401', '446', '479', '525', '568', '605', '660', '694', '714', '756', '769', '8', '816', '839', '915', '927', '954']\n",
      "NDB_665 -> ['1049', '1067', '1117', '1147', '1179', '1225', '1245', '1254', '127', '1337', '1395', '1444', '1470', '1508', '164', '209', '223', '25', '269', '292', '321', '368', '4', '402', '458', '482', '495', '544', '591', '660', '682', '717', '768', '806', '817', '840', '911', '924', '95', '950', '997']\n",
      "NDB_680 -> ['1029', '105', '1108', '1145', '1188', '1211', '1253', '1294', '1333', '1362', '1429', '1497', '1530', '160', '183', '205', '247', '313', '326', '344', '359', '374', '429', '466', '512', '545', '568', '60', '628', '671', '68', '720', '732', '750', '798', '841', '877', '899', '911', '981', '994']\n",
      "NDB_683 -> ['0', '1004', '1037', '1081', '110', '1107', '1160', '1173', '1211', '1265', '1308', '1355', '1391', '140', '1442', '1476', '1518', '194', '237', '267', '274', '324', '350', '372', '39', '426', '454', '521', '590', '626', '676', '685', '718', '775', '812', '854', '870', '89', '902', '927', '955']\n",
      "NDB_694 -> ['1031', '1071', '1120', '1141', '1173', '1203', '1222', '127', '1271', '1323', '1348', '1393', '1463', '1501', '174', '221', '268', '277', '313', '32', '364', '393', '431', '461', '519', '543', '587', '612', '670', '69', '724', '745', '782', '804', '836', '848', '85', '916', '954', '982']\n",
      "NDB_698 -> ['1002', '1032', '1048', '1087', '1109', '1143', '1162', '1226', '123', '1253', '1288', '1309', '1358', '1427', '1457', '1485', '149', '198', '22', '244', '259', '308', '317', '353', '38', '418', '431', '457', '491', '552', '585', '645', '685', '712', '778', '807', '829', '84', '867', '913', '967']\n",
      "NDB_706 -> ['1041', '1069', '1093', '110', '1127', '1134', '1157', '1217', '1261', '1319', '1349', '137', '1400', '1480', '1520', '1535', '178', '215', '253', '311', '336', '351', '396', '459', '471', '496', '532', '57', '580', '647', '678', '68', '699', '722', '794', '804', '852', '863', '926', '962', '986']\n",
      "NDB_710 -> ['1044', '1118', '1154', '116', '1206', '1233', '1250', '1263', '1301', '1323', '1396', '1468', '147', '1487', '189', '237', '264', '286', '31', '322', '367', '4', '408', '424', '446', '499', '538', '583', '597', '625', '673', '684', '715', '797', '820', '841', '88', '904', '929', '988']\n",
      "NDB_712 -> ['1038', '1092', '1128', '1134', '1183', '1233', '1272', '1285', '1337', '1360', '1447', '146', '1465', '1505', '170', '18', '210', '234', '304', '318', '337', '380', '410', '432', '44', '477', '511', '557', '570', '653', '710', '731', '755', '76', '799', '826', '877', '913', '953', '970']\n",
      "NDB_713 -> ['103', '1030', '1111', '1127', '1178', '1229', '1259', '1298', '1341', '1405', '1452', '1497', '1514', '1541', '157', '171', '202', '233', '257', '27', '297', '319', '360', '389', '428', '44', '450', '502', '541', '591', '611', '629', '689', '716', '751', '779', '829', '874', '904', '972', '991']\n",
      "NDB_715 -> ['1020', '1069', '1121', '1141', '1187', '1230', '1247', '1284', '133', '1333', '1388', '1434', '1485', '1502', '154', '184', '202', '21', '241', '299', '339', '384', '417', '441', '487', '505', '542', '583', '657', '67', '688', '744', '749', '776', '817', '833', '893', '93', '933', '964', '993']\n",
      "NDB_716 -> ['1037', '1053', '107', '1106', '1125', '1147', '1204', '1278', '1303', '1345', '1409', '144', '1445', '1488', '1499', '189', '204', '257', '310', '336', '366', '392', '461', '472', '516', '537', '565', '60', '621', '664', '697', '70', '721', '762', '777', '821', '845', '892', '913', '957', '968']\n",
      "NDB_718 -> ['1', '1063', '1083', '1155', '1190', '1239', '1286', '1308', '132', '1325', '1401', '1447', '148', '1494', '1534', '209', '227', '261', '279', '331', '34', '352', '413', '442', '471', '524', '543', '559', '575', '646', '666', '714', '734', '748', '765', '80', '832', '874', '909', '928', '984']\n",
      "NDB_721 -> ['1039', '1065', '1125', '1134', '1153', '1193', '1249', '128', '1293', '1311', '1361', '1391', '1427', '1469', '1513', '168', '186', '207', '223', '239', '293', '333', '37', '389', '411', '422', '470', '519', '554', '594', '613', '674', '680', '707', '764', '80', '809', '814', '839', '893', '974']\n",
      "NDB_725 -> ['101', '1054', '1074', '1130', '1174', '1210', '1252', '1297', '132', '1335', '1402', '1430', '1493', '1509', '1535', '179', '223', '246', '274', '291', '30', '345', '385', '405', '450', '467', '518', '534', '588', '619', '650', '70', '715', '745', '758', '770', '833', '877', '910', '958', '996']\n",
      "NDB_728 -> ['1062', '107', '1119', '1137', '1190', '1224', '1270', '1332', '1381', '1487', '1502', '1535', '158', '177', '198', '212', '23', '242', '284', '324', '356', '376', '40', '440', '480', '519', '539', '558', '576', '624', '679', '687', '701', '751', '792', '835', '875', '894', '911', '953', '961']\n",
      "NDB_730 -> ['1031', '1112', '1172', '1228', '1238', '1279', '1292', '1309', '132', '1332', '1400', '1448', '1465', '1514', '1548', '172', '200', '250', '271', '300', '334', '379', '406', '454', '46', '475', '5', '516', '569', '656', '669', '688', '704', '750', '766', '834', '849', '918', '95', '956', '976']\n",
      "NDB_734 -> ['1059', '1118', '1174', '1190', '1211', '1273', '1295', '1319', '1352', '1409', '143', '1479', '1511', '1526', '18', '181', '207', '229', '276', '335', '391', '414', '463', '468', '47', '490', '502', '564', '599', '620', '675', '694', '723', '759', '783', '850', '864', '889', '925', '95', '980']\n",
      "NDB_737 -> ['1041', '1097', '11', '1123', '1140', '1189', '121', '1234', '1246', '1292', '1336', '1398', '1435', '1478', '1517', '170', '203', '252', '273', '286', '316', '384', '400', '440', '485', '509', '528', '564', '596', '612', '668', '691', '727', '768', '80', '807', '813', '859', '902', '96', '985']\n",
      "NDB_742 -> ['1003', '1040', '1068', '1074', '1087', '1142', '1217', '1257', '1312', '1360', '138', '1408', '1451', '1490', '1502', '1548', '156', '171', '240', '266', '287', '330', '367', '401', '423', '45', '489', '526', '566', '613', '671', '683', '71', '721', '767', '808', '825', '870', '9', '922', '985']\n",
      "NDB_743 -> ['1025', '1055', '1105', '112', '1171', '1209', '1233', '1243', '1253', '1330', '1348', '1448', '145', '1467', '1518', '1536', '190', '195', '21', '224', '269', '310', '341', '390', '411', '448', '478', '507', '533', '584', '596', '646', '702', '730', '74', '790', '825', '846', '889', '907', '983']\n",
      "NDB_746 -> ['1003', '1054', '1090', '111', '1122', '1170', '1212', '1227', '1243', '1286', '13', '1303', '131', '1322', '1386', '1436', '1463', '1481', '1526', '192', '202', '245', '296', '31', '333', '374', '418', '454', '486', '496', '539', '560', '609', '62', '627', '692', '709', '787', '813', '856', '915', '959']\n",
      "NDB_751 -> ['1048', '110', '1115', '1146', '1191', '1216', '1251', '129', '1290', '1325', '1347', '1378', '1441', '1477', '1504', '1533', '162', '206', '238', '272', '28', '283', '316', '374', '417', '451', '479', '526', '565', '608', '651', '689', '707', '752', '770', '79', '835', '873', '896', '932', '950', '970']\n",
      "NDB_752 -> ['103', '1033', '1123', '1144', '1181', '1191', '122', '1238', '1247', '1299', '1305', '1320', '1386', '1458', '1479', '1516', '174', '219', '24', '258', '307', '325', '363', '392', '40', '457', '471', '494', '549', '557', '605', '658', '681', '730', '763', '793', '823', '851', '899', '951', '996']\n",
      "NDB_754 -> ['1020', '1046', '1081', '1177', '118', '1219', '1264', '1293', '1315', '1363', '1405', '1430', '1477', '1508', '1536', '159', '208', '230', '24', '295', '314', '347', '364', '401', '424', '453', '524', '561', '576', '621', '665', '7', '704', '739', '755', '786', '828', '871', '908', '96', '986']\n",
      "NDB_755 -> ['1002', '1055', '106', '1066', '1085', '1132', '1136', '1184', '1207', '1282', '1336', '1361', '1393', '1441', '1465', '1519', '176', '194', '214', '260', '3', '315', '329', '386', '409', '437', '487', '512', '577', '647', '675', '683', '715', '75', '786', '806', '847', '874', '917', '959', '98']\n",
      "NDB_758 -> ['0', '105', '1056', '1072', '1082', '1111', '1168', '1193', '1221', '1284', '13', '1335', '1363', '1381', '1467', '147', '1500', '1537', '181', '201', '260', '292', '330', '353', '397', '419', '432', '483', '525', '559', '603', '622', '726', '763', '804', '837', '865', '87', '896', '976', '995']\n",
      "NDB_760 -> ['1061', '1077', '1107', '117', '1181', '1196', '1241', '1271', '1286', '1341', '1359', '137', '1382', '1488', '1530', '18', '193', '216', '242', '266', '280', '320', '365', '408', '444', '474', '529', '552', '569', '603', '660', '690', '707', '75', '757', '776', '824', '859', '906', '978', '989']\n",
      "NDB_773 -> ['1040', '1091', '1140', '116', '1186', '1225', '1260', '1297', '1310', '1356', '1389', '1434', '145', '1463', '1512', '165', '218', '256', '315', '337', '35', '375', '394', '421', '425', '462', '524', '57', '574', '622', '655', '690', '729', '775', '809', '829', '849', '899', '932', '968', '980']\n",
      "NDB_778 -> ['1044', '1055', '1089', '1152', '1197', '1236', '125', '1256', '1298', '1339', '1356', '1402', '1473', '1496', '1537', '177', '204', '243', '258', '298', '32', '348', '365', '410', '429', '455', '498', '542', '566', '594', '618', '669', '703', '73', '734', '759', '785', '852', '927', '96', '990']\n",
      "NDB_779 -> ['1050', '1067', '1076', '1097', '1176', '122', '1226', '1278', '1304', '1330', '1396', '140', '1455', '1491', '1533', '172', '2', '226', '266', '282', '29', '342', '393', '416', '437', '473', '505', '548', '578', '594', '619', '667', '72', '726', '728', '791', '828', '866', '915', '953', '991']\n",
      "NDB_784 -> ['1000', '1032', '1079', '1125', '1168', '1237', '1244', '1259', '131', '1324', '1349', '1394', '1443', '1486', '1505', '152', '191', '240', '265', '28', '303', '321', '384', '409', '442', '47', '484', '520', '534', '556', '606', '654', '701', '730', '761', '78', '785', '858', '918', '935', '954']\n",
      "NDB_788 -> ['1033', '1073', '1113', '1128', '1139', '1187', '12', '1219', '1271', '128', '1321', '1346', '1385', '149', '1496', '173', '203', '248', '275', '335', '350', '394', '413', '467', '500', '537', '568', '593', '615', '672', '684', '709', '74', '760', '779', '853', '862', '918', '961', '995']\n",
      "NDB_790 -> ['1042', '1064', '1112', '1170', '1189', '12', '120', '1225', '1280', '1305', '1318', '1394', '1439', '1468', '1510', '155', '226', '257', '305', '339', '356', '38', '398', '415', '429', '451', '501', '540', '561', '582', '613', '675', '69', '699', '740', '796', '832', '846', '910', '931', '989']\n",
      "NDB_805 -> ['1061', '1083', '1126', '1138', '1188', '1218', '1264', '1289', '1317', '1354', '1396', '1439', '1469', '1490', '164', '176', '199', '245', '283', '327', '341', '373', '400', '426', '455', '47', '5', '506', '541', '587', '614', '653', '687', '711', '760', '784', '84', '855', '921', '968', '996']\n",
      "NDB_812 -> ['10', '1059', '1089', '1123', '1147', '1208', '1219', '1242', '1258', '126', '1342', '1455', '1475', '1507', '1548', '160', '176', '206', '265', '281', '31', '328', '390', '419', '436', '447', '510', '527', '580', '600', '626', '677', '714', '738', '754', '774', '81', '833', '867', '920', '989']\n",
      "NDB_815 -> ['102', '1030', '1072', '1108', '1121', '1176', '1206', '1274', '1331', '1346', '139', '1404', '1451', '1469', '1507', '1526', '16', '161', '224', '246', '308', '334', '351', '382', '398', '425', '456', '508', '577', '601', '658', '708', '736', '753', '769', '843', '85', '871', '914', '950', '984']\n",
      "NDB_817 -> ['1003', '1044', '1117', '1138', '115', '1195', '1209', '1251', '1299', '1333', '1345', '1454', '1478', '1519', '1542', '167', '188', '220', '255', '274', '295', '349', '358', '37', '379', '430', '468', '501', '546', '579', '603', '66', '662', '729', '778', '805', '836', '863', '900', '935', '975']\n",
      "NDB_818 -> ['1005', '1029', '1066', '1077', '1113', '1137', '118', '1186', '1230', '1273', '13', '1324', '1347', '1390', '1458', '1474', '1520', '157', '186', '195', '211', '262', '287', '317', '357', '380', '447', '461', '506', '540', '586', '618', '650', '728', '780', '802', '820', '844', '89', '919', '975']\n",
      "NDB_824 -> ['1031', '107', '1070', '1121', '1162', '1179', '1194', '1282', '1315', '1353', '1379', '140', '1429', '1466', '1521', '157', '201', '216', '235', '309', '33', '331', '371', '404', '467', '493', '532', '562', '600', '627', '674', '703', '735', '761', '796', '851', '864', '88', '900', '932', '985']\n",
      "NDB_826 -> ['1046', '1122', '1138', '1182', '1210', '1232', '1255', '1290', '1300', '1337', '1380', '1437', '144', '1491', '1521', '166', '174', '200', '229', '265', '279', '329', '340', '36', '367', '412', '456', '474', '492', '542', '571', '623', '65', '663', '727', '775', '811', '844', '868', '903', '967']\n",
      "NDB_832 -> ['1023', '1071', '1083', '1113', '1166', '120', '1234', '1279', '1311', '1360', '1402', '1445', '1486', '150', '1519', '1538', '183', '2', '236', '249', '25', '278', '324', '399', '419', '448', '476', '492', '552', '567', '609', '659', '689', '723', '788', '797', '827', '872', '914', '93', '962']\n",
      "NDB_835 -> ['101', '1038', '1094', '1129', '1160', '1224', '1260', '1299', '130', '1343', '1405', '1455', '1480', '1515', '1538', '184', '199', '239', '254', '300', '322', '353', '402', '460', '486', '490', '503', '571', '608', '654', '720', '735', '790', '8', '808', '824', '83', '869', '895', '971', '982']\n",
      "NDB_836 -> ['1052', '106', '1118', '1148', '119', '1192', '1226', '1256', '1310', '134', '1362', '1398', '1436', '1461', '1503', '222', '249', '285', '303', '351', '397', '453', '477', '510', '565', '587', '650', '666', '67', '702', '743', '764', '803', '823', '83', '868', '9', '905', '924', '956', '978']\n",
      "NDB_838 -> ['1036', '106', '1074', '1097', '1141', '1195', '1280', '1321', '1347', '1404', '141', '1453', '1464', '1494', '1525', '169', '219', '23', '256', '289', '306', '344', '355', '402', '435', '481', '499', '533', '569', '609', '652', '698', '716', '794', '826', '84', '864', '890', '903', '960', '999']\n",
      "NDB_844 -> ['10', '1053', '1070', '1078', '111', '1129', '1156', '1195', '1222', '1259', '1313', '1352', '1385', '1446', '1492', '1506', '154', '182', '228', '264', '293', '340', '382', '42', '421', '452', '489', '522', '559', '596', '618', '676', '681', '733', '757', '791', '835', '872', '924', '98', '986']\n",
      "NDB_847 -> ['101', '1062', '1080', '1092', '1136', '1163', '1196', '1249', '1290', '1302', '1318', '1389', '1473', '1506', '159', '187', '2', '235', '263', '281', '321', '355', '370', '39', '418', '463', '494', '545', '575', '593', '612', '67', '673', '696', '741', '762', '784', '861', '922', '979']\n",
      "NDB_853 -> ['1035', '1077', '1133', '1156', '1197', '1270', '1284', '1306', '1330', '134', '1408', '1456', '1475', '1513', '1529', '179', '230', '254', '26', '308', '327', '363', '395', '430', '458', '521', '538', '560', '575', '6', '656', '69', '695', '725', '761', '779', '821', '850', '903', '926', '970']\n",
      "NDB_856 -> ['1064', '1094', '1144', '1229', '1246', '125', '1276', '1332', '1361', '139', '1403', '1430', '1459', '1464', '1499', '190', '206', '249', '296', '332', '352', '407', '42', '430', '459', '520', '581', '595', '614', '671', '685', '72', '744', '789', '810', '815', '853', '90', '912', '929', '980']\n",
      "NDB_860 -> ['1043', '1092', '1112', '1167', '1209', '121', '1218', '1248', '1295', '1300', '1338', '1383', '141', '1500', '1527', '189', '198', '238', '24', '312', '345', '365', '393', '422', '460', '489', '510', '551', '556', '601', '655', '693', '725', '77', '787', '812', '816', '860', '919', '972', '993']\n",
      "NDB_862 -> ['103', '1057', '1124', '1171', '1203', '1221', '1246', '1278', '1301', '1315', '1401', '1457', '1481', '1503', '1527', '153', '185', '229', '255', '275', '290', '338', '373', '406', '436', '479', '521', '540', '574', '602', '64', '659', '683', '710', '78', '792', '819', '852', '895', '933', '952', '992']\n",
      "NDB_863 -> ['100', '1054', '1075', '1106', '1152', '1178', '1214', '124', '1257', '1292', '1310', '1355', '1404', '1447', '1482', '1520', '166', '225', '244', '263', '283', '347', '368', '40', '407', '448', '484', '512', '550', '555', '607', '64', '648', '684', '719', '749', '791', '824', '836', '920', '963']\n",
      "NDB_865 -> ['104', '1047', '1085', '1153', '1192', '1220', '1250', '1269', '1322', '133', '1355', '1403', '1449', '1462', '1489', '1541', '172', '246', '262', '307', '336', '356', '36', '375', '432', '458', '493', '545', '563', '585', '646', '670', '693', '71', '723', '785', '805', '822', '862', '908', '965']\n",
      "NDB_866 -> ['102', '1060', '1081', '1126', '1171', '1207', '1255', '127', '1322', '1357', '1399', '14', '1453', '1474', '1515', '1542', '175', '222', '258', '309', '337', '391', '413', '423', '451', '517', '538', '572', '607', '629', '667', '68', '680', '691', '744', '762', '795', '857', '906', '958', '976']\n",
      "NDB_868 -> ['1023', '1068', '1107', '1122', '1139', '115', '1165', '1214', '1275', '1336', '1344', '1399', '1440', '1498', '158', '191', '217', '238', '264', '292', '328', '357', '392', '439', '44', '478', '505', '535', '554', '581', '620', '666', '712', '73', '735', '798', '834', '871', '910', '952', '979']\n",
      "NDB_872 -> ['1001', '1024', '1053', '1085', '1131', '1155', '117', '1187', '1231', '1285', '129', '1319', '1353', '1390', '1464', '1509', '1538', '187', '230', '250', '268', '298', '342', '381', '400', '423', '456', '525', '550', '592', '645', '66', '677', '706', '737', '78', '798', '846', '860', '904', '974']\n",
      "NDB_875 -> ['1034', '104', '1087', '1124', '1139', '1212', '1236', '1258', '1291', '1334', '1392', '1459', '1474', '1513', '155', '193', '208', '224', '250', '28', '285', '329', '361', '377', '428', '465', '495', '532', '589', '625', '679', '70', '720', '741', '765', '812', '826', '878', '916', '951', '969']\n",
      "NDB_878 -> ['1004', '1039', '109', '1116', '1154', '1188', '1208', '122', '1254', '1291', '1313', '1350', '1382', '1440', '1477', '1514', '188', '218', '248', '310', '346', '362', '395', '431', '460', '517', '539', '570', '595', '648', '65', '694', '722', '758', '796', '822', '847', '9', '914', '933', '960']\n",
      "NDB_880 -> ['1033', '1049', '109', '1093', '1135', '1166', '1204', '1281', '1306', '1363', '1388', '139', '1457', '1482', '180', '227', '243', '271', '297', '341', '35', '380', '409', '464', '481', '493', '546', '554', '564', '616', '673', '705', '732', '752', '787', '81', '838', '870', '891', '916', '987']\n",
      "NDB_881 -> ['1030', '1068', '1091', '1135', '1145', '1208', '123', '1265', '1287', '1316', '1335', '1387', '144', '1443', '1489', '1510', '182', '239', '255', '26', '270', '306', '331', '377', '407', '443', '477', '504', '528', '567', '583', '647', '682', '728', '760', '77', '795', '861', '889', '930', '990']\n",
      "NDB_883 -> ['1000', '1059', '1116', '1142', '1181', '1213', '1243', '1275', '1301', '1340', '1379', '1489', '1534', '159', '163', '22', '220', '247', '289', '311', '348', '361', '385', '462', '469', '511', '548', '591', '6', '615', '676', '687', '727', '764', '810', '816', '838', '85', '896', '931', '963']\n",
      "NDB_887 -> ['0', '1005', '1057', '1093', '1140', '1163', '1231', '1250', '126', '1291', '1312', '1362', '1383', '1446', '1495', '16', '164', '200', '217', '277', '291', '314', '366', '411', '427', '457', '530', '548', '573', '608', '62', '624', '716', '740', '748', '776', '827', '876', '898', '934', '990']\n",
      "NDB_889 -> ['1057', '1079', '111', '1133', '1154', '1205', '1256', '1295', '1340', '138', '1406', '1449', '1498', '1508', '1525', '160', '216', '236', '3', '302', '317', '350', '387', '425', '43', '443', '516', '531', '555', '589', '653', '691', '705', '751', '770', '832', '86', '869', '897', '949', '974']\n",
      "NDB_896 -> ['1004', '1029', '1067', '1078', '1169', '1215', '1241', '1260', '1314', '1353', '1393', '1435', '1467', '1507', '152', '166', '204', '253', '276', '289', '327', '358', '38', '383', '428', '441', '526', '588', '605', '659', '705', '743', '748', '76', '800', '814', '857', '90', '908', '923', '961']\n",
      "NDB_899 -> ['1037', '1051', '109', '1105', '1137', '1183', '1231', '1276', '1334', '1359', '1389', '1492', '150', '1521', '178', '19', '196', '226', '299', '3', '322', '360', '385', '435', '482', '523', '566', '602', '658', '711', '739', '754', '801', '813', '858', '87', '909', '930', '965', '994']\n",
      "NDB_901 -> ['100', '1043', '1082', '1132', '1146', '1177', '1215', '1244', '125', '1274', '1305', '1344', '1392', '1448', '1462', '1503', '165', '203', '241', '302', '320', '360', '388', '39', '452', '468', '497', '543', '567', '624', '674', '695', '717', '780', '799', '8', '856', '91', '921', '967', '981']\n",
      "NDB_905 -> ['1047', '1058', '1114', '1135', '1178', '1203', '1277', '1302', '1345', '1381', '1438', '1461', '1493', '152', '178', '207', '25', '261', '280', '325', '342', '354', '404', '417', '462', '500', '535', '570', '584', '614', '62', '669', '695', '742', '786', '843', '869', '890', '919', '92', '988']\n",
      "NDB_910 -> ['1039', '1110', '1130', '1144', '1165', '1215', '1255', '1289', '1308', '1356', '1456', '146', '1495', '1533', '163', '17', '211', '263', '312', '33', '344', '382', '397', '414', '438', '480', '529', '546', '589', '611', '677', '692', '713', '74', '766', '807', '817', '858', '892', '982', '992']\n",
      "NDB_917 -> ['1036', '1050', '1094', '1152', '119', '1207', '1234', '1281', '1317', '1351', '1383', '1451', '1488', '1505', '162', '186', '21', '212', '233', '270', '288', '33', '340', '370', '395', '465', '501', '544', '585', '615', '678', '704', '737', '763', '783', '819', '848', '888', '897', '955', '981']\n",
      "NDB_925 -> ['1000', '1062', '11', '1109', '1167', '117', '1221', '1240', '1242', '1277', '1320', '1348', '1380', '1444', '1480', '1495', '168', '192', '234', '256', '272', '278', '29', '319', '387', '415', '441', '475', '506', '537', '577', '607', '662', '718', '734', '789', '822', '850', '891', '930', '969']\n",
      "NDB_926 -> ['100', '1038', '1064', '1090', '1165', '1180', '1205', '123', '1251', '1293', '1313', '1354', '1384', '1442', '1494', '15', '180', '215', '240', '307', '335', '359', '383', '427', '46', '469', '492', '551', '592', '629', '664', '726', '742', '754', '769', '848', '878', '88', '907', '964', '978']\n",
      "NDB_929 -> ['1023', '104', '1065', '1110', '1176', '1222', '1240', '1270', '1300', '1339', '1388', '1468', '1516', '1534', '167', '185', '197', '235', '27', '273', '287', '332', '362', '381', '443', '472', '518', '572', '606', '622', '678', '702', '742', '75', '755', '774', '856', '876', '923', '966', '994']\n",
      "NDB_934 -> ['1035', '1050', '1079', '1143', '12', '1236', '124', '1254', '1287', '1320', '1352', '1378', '1454', '1466', '1509', '180', '205', '253', '273', '296', '318', '355', '406', '449', '45', '483', '497', '530', '558', '601', '657', '697', '739', '797', '855', '868', '906', '92', '928', '971']\n",
      "NDB_935 -> ['1043', '1063', '11', '1114', '1133', '1168', '1191', '1235', '1262', '1311', '1331', '1384', '143', '1453', '1497', '182', '195', '217', '242', '299', '32', '323', '373', '416', '424', '45', '464', '491', '544', '571', '582', '626', '713', '741', '758', '777', '849', '863', '901', '959', '977']\n",
      "NDB_940 -> ['1058', '1070', '1076', '108', '1131', '1173', '1182', '1227', '1273', '1325', '1346', '138', '1384', '1439', '1475', '1515', '190', '20', '212', '225', '260', '297', '320', '377', '405', '438', '475', '523', '556', '57', '598', '621', '686', '712', '768', '805', '839', '876', '905', '973', '999']\n",
      "NDB_941 -> ['102', '1034', '1049', '1109', '1169', '1192', '1239', '1279', '1317', '1349', '1391', '1434', '146', '1498', '167', '221', '248', '26', '291', '305', '325', '359', '388', '446', '474', '509', '527', '562', '599', '649', '665', '696', '733', '777', '802', '860', '87', '895', '912', '963', '991']\n",
      "NDB_946 -> ['1061', '1105', '1124', '114', '1145', '1213', '1248', '1269', '128', '1306', '1358', '1406', '1450', '1486', '187', '201', '23', '261', '286', '328', '372', '410', '438', '472', '508', '549', '579', '599', '623', '688', '722', '756', '800', '828', '83', '873', '898', '958', '98', '997']\n",
      "NDB_958 -> ['1025', '1045', '1091', '1127', '114', '1148', '1182', '1230', '1272', '1342', '1394', '1449', '1479', '15', '1506', '156', '179', '245', '267', '316', '343', '354', '408', '465', '480', '503', '530', '572', '652', '665', '706', '72', '736', '750', '788', '814', '838', '900', '91', '977', '995']\n",
      "NDB_961 -> ['1065', '1089', '115', '1155', '1189', '1213', '1257', '1288', '131', '1314', '1350', '1395', '14', '1445', '1461', '1501', '162', '214', '241', '300', '323', '371', '421', '426', '43', '466', '502', '529', '573', '597', '663', '710', '736', '77', '793', '837', '878', '894', '912', '965', '999']\n",
      "NDB_965 -> ['1048', '1082', '1142', '1180', '1220', '1237', '124', '1264', '1304', '1321', '145', '1452', '1493', '1527', '168', '19', '213', '270', '279', '332', '363', '379', '466', '503', '531', '580', '598', '617', '63', '679', '703', '733', '782', '803', '820', '847', '902', '925', '93', '979', '997']\n",
      "NDB_971 -> ['1020', '1071', '108', '1115', '1169', '1214', '1237', '1276', '1343', '1387', '1437', '1466', '149', '1517', '181', '210', '233', '267', '298', '323', '362', '37', '387', '436', '452', '496', '536', '563', '586', '60', '616', '667', '698', '725', '765', '806', '843', '875', '898', '931', '983']\n",
      "NDB_974 -> ['1035', '1073', '1120', '1179', '1217', '1228', '126', '1262', '1297', '1307', '1358', '1387', '1452', '1462', '1473', '161', '220', '252', '285', '326', '339', '34', '376', '399', '422', '437', '517', '555', '606', '652', '692', '7', '718', '757', '79', '793', '815', '844', '907', '952', '988']\n",
      "NDB_977 -> ['1041', '1126', '1162', '1177', '119', '1224', '1249', '1294', '1307', '1359', '1385', '143', '1443', '1481', '188', '20', '213', '237', '282', '315', '383', '403', '43', '439', '476', '511', '550', '586', '611', '670', '721', '740', '789', '81', '811', '840', '865', '890', '928', '951', '993']\n",
      "NDB_989 -> ['1042', '1069', '1115', '1153', '1194', '1239', '1277', '130', '1338', '1390', '1442', '1482', '1501', '1529', '16', '173', '222', '259', '276', '326', '348', '370', '386', '440', '478', '504', '518', '557', '602', '625', '63', '686', '717', '767', '811', '855', '866', '892', '935', '94', '960', '977']\n",
      "NDB_994 -> ['1045', '1063', '108', '1106', '1129', '1163', '1183', '1206', '1244', '1265', '1314', '133', '1351', '1382', '1438', '1491', '1511', '158', '205', '251', '30', '312', '338', '394', '415', '447', '459', '495', '509', '592', '656', '680', '724', '778', '79', '803', '825', '851', '917', '949', '987']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(reg_readout_dic, full_records, \n",
    "                                                                    check_readout_dict, primer_len=primer_len, \n",
    "                                                                    target_len=target_len,\n",
    "                                                                    probe_readout_num=num_readout_per_probe)\n",
    "print(readout2reg_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7445b10e",
   "metadata": {},
   "source": [
    "### 4.1.5 check bewteen probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 145,
   "id": "3a6a50cc",
   "metadata": {},
   "outputs": [],
   "source": [
    "# construct internal map\n",
    "reload(check)\n",
    "int_map = check._construct_internal_map(full_records, library_folder, save=False) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 146,
   "id": "11132a68",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n"
     ]
    }
   ],
   "source": [
    "# internal check probes\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, int_map, readout_dict)\n",
    "print(readout_count_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 147,
   "id": "056864be",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_1:1235001-1265000_gene_0_pb_183_pos_13330_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_1:1235001-1265000_gene_0_pb_187_pos_13649_strand_+_readouts_[NDB_758_c,NDB_758_c,NDB_758_c,NDB_758_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_1:1235001-1265000_gene_0_pb_188_pos_13789_strand_+_readouts_[NDB_887_c,NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_1:1235001-1265000_gene_0_pb_189_pos_13855_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:84, dumped\n",
      "--- Sequence:loc_1:1235001-1265000_gene_0_pb_191_pos_14659_strand_-_readouts_[NDB_887_c,NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_56_pos_4401_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_57_pos_4503_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_58_pos_4557_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_59_pos_4601_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_60_pos_4656_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_61_pos_4700_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_62_pos_4773_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_63_pos_4817_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_64_pos_4900_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:119, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_65_pos_4947_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:92, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_66_pos_5012_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_67_pos_5084_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_68_pos_5133_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_126_pos_10141_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:201, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_127_pos_10221_strand_+_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:186, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_128_pos_10273_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:144, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_129_pos_10317_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:198, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_130_pos_10363_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:132, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_131_pos_10423_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:93, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_132_pos_10468_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:132, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_133_pos_10512_strand_+_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:194, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_134_pos_10573_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_135_pos_10644_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:113, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_136_pos_10734_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:129, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_137_pos_10851_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:191, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_138_pos_10925_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_139_pos_11094_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:205, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_140_pos_11155_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:176, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_141_pos_11221_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:192, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_142_pos_11265_strand_+_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:93, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_143_pos_11311_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_144_pos_11382_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:105, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_145_pos_11443_strand_+_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:178, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_147_pos_11566_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:209, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_148_pos_11625_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_149_pos_11688_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:158, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_150_pos_11744_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:167, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_178_pos_15818_strand_+_readouts_[NDB_887_c,NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:96, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_179_pos_15865_strand_+_readouts_[NDB_818_c,NDB_818_c,NDB_818_c,NDB_818_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_181_pos_16197_strand_-_readouts_[NDB_887_c,NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:87, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_182_pos_16852_strand_+_readouts_[NDB_818_c,NDB_818_c,NDB_818_c,NDB_818_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:99, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_183_pos_16897_strand_-_readouts_[NDB_661_c,NDB_661_c,NDB_661_c,NDB_661_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_184_pos_16946_strand_+_readouts_[NDB_887_c,NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_185_pos_16993_strand_+_readouts_[NDB_818_c,NDB_818_c,NDB_818_c,NDB_818_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_186_pos_17039_strand_+_readouts_[NDB_661_c,NDB_661_c,NDB_661_c,NDB_661_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_2:6235001-6265000_gene_101_pb_36_pos_3462_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_2:6235001-6265000_gene_101_pb_37_pos_3528_strand_-_readouts_[NDB_847_c,NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_2:6235001-6265000_gene_101_pb_38_pos_3816_strand_-_readouts_[NDB_835_c,NDB_835_c,NDB_835_c,NDB_835_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_2:6235001-6265000_gene_101_pb_39_pos_3860_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_15:62235001-62265000_gene_1048_pb_100_pos_10851_strand_-_readouts_[NDB_698_c,NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_15:62235001-62265000_gene_1048_pb_103_pos_11083_strand_-_readouts_[NDB_698_c,NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_15:62235001-62265000_gene_1048_pb_107_pos_11397_strand_-_readouts_[NDB_751_c,NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_15:62235001-62265000_gene_1048_pb_155_pos_16135_strand_-_readouts_[NDB_751_c,NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_102_pos_11145_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_104_pos_11273_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_105_pos_11363_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_106_pos_11450_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_107_pos_11499_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_108_pos_11580_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_109_pos_11629_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_16:60735001-60765000_gene_1114_pb_118_pos_16300_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_16:60735001-60765000_gene_1114_pb_132_pos_18301_strand_-_readouts_[NDB_905_c,NDB_905_c,NDB_905_c,NDB_905_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_16:84735001-84765000_gene_1130_pb_302_pos_28753_strand_+_readouts_[NDB_910_c,NDB_910_c,NDB_910_c,NDB_910_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_16:84735001-84765000_gene_1130_pb_304_pos_28847_strand_-_readouts_[NDB_653_c,NDB_653_c,NDB_653_c,NDB_653_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_16:87735001-87765000_gene_1132_pb_38_pos_3587_strand_-_readouts_[NDB_901_c,NDB_901_c,NDB_901_c,NDB_901_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_16:87735001-87765000_gene_1132_pb_39_pos_4304_strand_-_readouts_[NDB_755_c,NDB_755_c,NDB_755_c,NDB_755_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_37_pos_5569_strand_+_readouts_[NDB_706_c,NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_38_pos_5613_strand_+_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_39_pos_5664_strand_+_readouts_[NDB_721_c,NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_40_pos_5708_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_41_pos_5752_strand_+_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_157_pos_13640_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:130, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_158_pos_13697_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:131, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_159_pos_13743_strand_-_readouts_[NDB_721_c,NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:145, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_160_pos_13799_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_161_pos_13910_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:135, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_162_pos_13998_strand_-_readouts_[NDB_721_c,NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_163_pos_14089_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:155, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_164_pos_14187_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:158, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_165_pos_14263_strand_-_readouts_[NDB_721_c,NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:111, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_166_pos_14320_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:145, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_167_pos_14364_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_169_pos_14477_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:132, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_171_pos_14590_strand_-_readouts_[NDB_721_c,NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_172_pos_14634_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_173_pos_14710_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:126, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_219_pos_18063_strand_+_readouts_[NDB_721_c,NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_221_pos_18339_strand_+_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_17:3735001-3765000_gene_1136_pb_158_pos_16971_strand_+_readouts_[NDB_847_c,NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_17:3735001-3765000_gene_1136_pb_159_pos_17016_strand_-_readouts_[NDB_755_c,NDB_755_c,NDB_755_c,NDB_755_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_17:3735001-3765000_gene_1136_pb_160_pos_17060_strand_+_readouts_[NDB_652_c,NDB_652_c,NDB_652_c,NDB_652_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_17:3735001-3765000_gene_1136_pb_161_pos_17107_strand_+_readouts_[NDB_847_c,NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_17:5235001-5265000_gene_1137_pb_46_pos_5724_strand_-_readouts_[NDB_899_c,NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_17:5235001-5265000_gene_1137_pb_48_pos_6055_strand_-_readouts_[NDB_728_c,NDB_728_c,NDB_728_c,NDB_728_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_17:5235001-5265000_gene_1137_pb_49_pos_6179_strand_-_readouts_[NDB_899_c,NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_146_pos_23288_strand_+_readouts_[NDB_805_c,NDB_805_c,NDB_805_c,NDB_805_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_147_pos_23342_strand_+_readouts_[NDB_826_c,NDB_826_c,NDB_826_c,NDB_826_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_148_pos_23387_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_149_pos_23437_strand_+_readouts_[NDB_805_c,NDB_805_c,NDB_805_c,NDB_805_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_150_pos_23486_strand_+_readouts_[NDB_826_c,NDB_826_c,NDB_826_c,NDB_826_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_151_pos_23533_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_152_pos_23582_strand_+_readouts_[NDB_805_c,NDB_805_c,NDB_805_c,NDB_805_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_153_pos_23680_strand_+_readouts_[NDB_826_c,NDB_826_c,NDB_826_c,NDB_826_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_154_pos_23729_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_156_pos_23875_strand_+_readouts_[NDB_826_c,NDB_826_c,NDB_826_c,NDB_826_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_17:66735001-66765000_gene_1178_pb_160_pos_13995_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c,NDB_863_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_17:66735001-66765000_gene_1178_pb_162_pos_14146_strand_+_readouts_[NDB_713_c,NDB_713_c,NDB_713_c,NDB_713_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_17:80235001-80265000_gene_1187_pb_7_pos_1300_strand_-_readouts_[NDB_788_c,NDB_788_c,NDB_788_c,NDB_788_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_226_pos_19397_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_236_pos_20036_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_238_pos_20157_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_239_pos_20222_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:109, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_242_pos_20393_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_243_pos_20437_strand_-_readouts_[NDB_760_c,NDB_760_c,NDB_760_c,NDB_760_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_244_pos_20489_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_246_pos_20611_strand_+_readouts_[NDB_760_c,NDB_760_c,NDB_760_c,NDB_760_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_247_pos_20659_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_248_pos_20745_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_249_pos_20796_strand_+_readouts_[NDB_760_c,NDB_760_c,NDB_760_c,NDB_760_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:108, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_251_pos_20922_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_253_pos_21085_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_254_pos_21143_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_257_pos_26196_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_263_pos_26520_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_266_pos_26721_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_267_pos_26805_strand_-_readouts_[NDB_856_c,NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_268_pos_26859_strand_-_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_274_pos_27156_strand_-_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_279_pos_27512_strand_-_readouts_[NDB_856_c,NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_287_pos_28065_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_19:5485001-5515000_gene_1247_pb_105_pos_13735_strand_+_readouts_[NDB_652_c,NDB_652_c,NDB_652_c,NDB_652_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_19:5485001-5515000_gene_1247_pb_106_pos_13837_strand_-_readouts_[NDB_715_c,NDB_715_c,NDB_715_c,NDB_715_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_19:30485001-30515000_gene_1272_pb_166_pos_15605_strand_-_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_19:31485001-31515000_gene_1273_pb_212_pos_21958_strand_+_readouts_[NDB_940_c,NDB_940_c,NDB_940_c,NDB_940_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_19:31485001-31515000_gene_1273_pb_214_pos_22419_strand_+_readouts_[NDB_818_c,NDB_818_c,NDB_818_c,NDB_818_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_230_pos_20742_strand_-_readouts_[NDB_641_c,NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_231_pos_20787_strand_+_readouts_[NDB_868_c,NDB_868_c,NDB_868_c,NDB_868_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_232_pos_20900_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c,NDB_883_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_234_pos_21183_strand_-_readouts_[NDB_868_c,NDB_868_c,NDB_868_c,NDB_868_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_235_pos_21456_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c,NDB_883_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_236_pos_21579_strand_+_readouts_[NDB_641_c,NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_237_pos_21869_strand_+_readouts_[NDB_868_c,NDB_868_c,NDB_868_c,NDB_868_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_19:47485001-47515000_gene_1289_pb_205_pos_16691_strand_+_readouts_[NDB_637_c,NDB_637_c,NDB_637_c,NDB_637_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_20:8485001-8515000_gene_1308_pb_72_pos_8628_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_20:8485001-8515000_gene_1308_pb_73_pos_8794_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_20:8485001-8515000_gene_1308_pb_74_pos_8867_strand_+_readouts_[NDB_910_c,NDB_910_c,NDB_910_c,NDB_910_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_20:8485001-8515000_gene_1308_pb_76_pos_9020_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_20:18485001-18515000_gene_1318_pb_125_pos_16718_strand_+_readouts_[NDB_1060_c,NDB_1060_c,NDB_1060_c,NDB_1060_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_20:18485001-18515000_gene_1318_pb_126_pos_16781_strand_-_readouts_[NDB_847_c,NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_20:18485001-18515000_gene_1318_pb_127_pos_16829_strand_-_readouts_[NDB_790_c,NDB_790_c,NDB_790_c,NDB_790_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_221_pos_18146_strand_+_readouts_[NDB_940_c,NDB_940_c,NDB_940_c,NDB_940_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_222_pos_18203_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:106, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_223_pos_18322_strand_+_readouts_[NDB_751_c,NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_224_pos_18437_strand_-_readouts_[NDB_940_c,NDB_940_c,NDB_940_c,NDB_940_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:107, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_225_pos_18484_strand_+_readouts_[NDB_718_c,NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_226_pos_18551_strand_-_readouts_[NDB_751_c,NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:107, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_20:37485001-37515000_gene_1337_pb_226_pos_21169_strand_+_readouts_[NDB_665_c,NDB_665_c,NDB_665_c,NDB_665_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_98_pos_20150_strand_+_readouts_[NDB_925_c,NDB_925_c,NDB_925_c,NDB_925_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_99_pos_21022_strand_+_readouts_[NDB_743_c,NDB_743_c,NDB_743_c,NDB_743_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_102_pos_22426_strand_+_readouts_[NDB_743_c,NDB_743_c,NDB_743_c,NDB_743_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_111_pos_25553_strand_+_readouts_[NDB_743_c,NDB_743_c,NDB_743_c,NDB_743_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_122_pos_27710_strand_+_readouts_[NDB_925_c,NDB_925_c,NDB_925_c,NDB_925_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:86, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_127_pos_29552_strand_-_readouts_[NDB_694_c,NDB_694_c,NDB_694_c,NDB_694_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_129_pos_29957_strand_+_readouts_[NDB_743_c,NDB_743_c,NDB_743_c,NDB_743_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_20:57485001-57515000_gene_1357_pb_54_pos_9610_strand_-_readouts_[NDB_661_c,NDB_661_c,NDB_661_c,NDB_661_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_20:57485001-57515000_gene_1357_pb_83_pos_13447_strand_-_readouts_[NDB_647_c,NDB_647_c,NDB_647_c,NDB_647_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_20:60485001-60515000_gene_1360_pb_173_pos_15690_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_208_pos_15898_strand_+_readouts_[NDB_887_c,NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_209_pos_15942_strand_+_readouts_[NDB_836_c,NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_210_pos_16086_strand_-_readouts_[NDB_680_c,NDB_680_c,NDB_680_c,NDB_680_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_211_pos_16142_strand_+_readouts_[NDB_887_c,NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_214_pos_16331_strand_-_readouts_[NDB_887_c,NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_215_pos_16382_strand_-_readouts_[NDB_836_c,NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_20:63485001-63515000_gene_1363_pb_267_pos_22445_strand_-_readouts_[NDB_754_c,NDB_754_c,NDB_754_c,NDB_754_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_20:63485001-63515000_gene_1363_pb_268_pos_22519_strand_+_readouts_[NDB_758_c,NDB_758_c,NDB_758_c,NDB_758_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_20:63485001-63515000_gene_1363_pb_270_pos_22640_strand_-_readouts_[NDB_754_c,NDB_754_c,NDB_754_c,NDB_754_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_20:63485001-63515000_gene_1363_pb_271_pos_22706_strand_-_readouts_[NDB_758_c,NDB_758_c,NDB_758_c,NDB_758_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_186_pos_17824_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:104, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_187_pos_17870_strand_-_readouts_[NDB_853_c,NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:132, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_189_pos_17963_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_190_pos_18007_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:180, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_191_pos_18053_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_192_pos_18097_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_193_pos_18176_strand_-_readouts_[NDB_853_c,NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_194_pos_18222_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_195_pos_18267_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:177, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_196_pos_18314_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_197_pos_18450_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:150, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_198_pos_18510_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_199_pos_18567_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:103, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_200_pos_18648_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_201_pos_18692_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_202_pos_18736_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_203_pos_18785_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:174, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_204_pos_18861_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_205_pos_18906_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:119, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_207_pos_18994_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:100, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_209_pos_19151_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_210_pos_19213_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:134, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_211_pos_19284_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_212_pos_19334_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:130, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_213_pos_19378_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:134, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_214_pos_19454_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_215_pos_19509_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:100, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_216_pos_19690_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:109, dumped\n",
      "--- Sequence:loc_21:45485001-45515000_gene_1409_pb_14_pos_1380_strand_-_readouts_[NDB_734_c,NDB_734_c,NDB_734_c,NDB_734_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_21:45485001-45515000_gene_1409_pb_222_pos_16939_strand_-_readouts_[NDB_716_c,NDB_716_c,NDB_716_c,NDB_716_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_21:45485001-45515000_gene_1409_pb_223_pos_16986_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_21:45485001-45515000_gene_1409_pb_224_pos_17035_strand_+_readouts_[NDB_734_c,NDB_734_c,NDB_734_c,NDB_734_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_21:45485001-45515000_gene_1409_pb_225_pos_17079_strand_-_readouts_[NDB_716_c,NDB_716_c,NDB_716_c,NDB_716_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_22:29485001-29515000_gene_1439_pb_152_pos_24719_strand_-_readouts_[NDB_940_c,NDB_940_c,NDB_940_c,NDB_940_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_22:29485001-29515000_gene_1439_pb_153_pos_24869_strand_+_readouts_[NDB_790_c,NDB_790_c,NDB_790_c,NDB_790_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_22:35485001-35515000_gene_1445_pb_73_pos_6569_strand_+_readouts_[NDB_716_c,NDB_716_c,NDB_716_c,NDB_716_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_22:35485001-35515000_gene_1445_pb_74_pos_6800_strand_+_readouts_[NDB_832_c,NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_22:35485001-35515000_gene_1445_pb_75_pos_6856_strand_+_readouts_[NDB_961_c,NDB_961_c,NDB_961_c,NDB_961_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_22:35485001-35515000_gene_1445_pb_76_pos_6937_strand_+_readouts_[NDB_716_c,NDB_716_c,NDB_716_c,NDB_716_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_22:39485001-39515000_gene_1449_pb_289_pos_25786_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_22:39485001-39515000_gene_1449_pb_290_pos_25854_strand_+_readouts_[NDB_958_c,NDB_958_c,NDB_958_c,NDB_958_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_22:49485001-49515000_gene_1459_pb_49_pos_4444_strand_+_readouts_[NDB_856_c,NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_22:49485001-49515000_gene_1459_pb_50_pos_4510_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_22:49485001-49515000_gene_1459_pb_52_pos_4620_strand_+_readouts_[NDB_856_c,NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_X:116235001-116265000_gene_1506_pb_77_pos_7640_strand_+_readouts_[NDB_958_c,NDB_958_c,NDB_958_c,NDB_958_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_2:168735001-168765000_gene_166_pb_84_pos_7550_strand_+_readouts_[NDB_826_c,NDB_826_c,NDB_826_c,NDB_826_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_2:178735001-178765000_gene_170_pb_168_pos_16820_strand_-_readouts_[NDB_661_c,NDB_661_c,NDB_661_c,NDB_661_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:141, dumped\n",
      "--- Sequence:loc_3:76235001-76265000_gene_225_pb_25_pos_3536_strand_+_readouts_[NDB_1060_c,NDB_1060_c,NDB_1060_c,NDB_1060_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_3:171235001-171265000_gene_263_pb_154_pos_15442_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c,NDB_863_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_136_pos_14761_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_137_pos_14877_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:133, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_138_pos_14942_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_139_pos_14986_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_140_pos_15057_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:100, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_142_pos_15216_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_143_pos_15260_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_144_pos_15408_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:161, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_145_pos_15483_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:135, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_146_pos_15530_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:125, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_147_pos_15585_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:141, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_148_pos_15652_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_149_pos_15711_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:134, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_150_pos_15769_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:136, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_151_pos_15828_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:135, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_152_pos_15918_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:155, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_154_pos_16125_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_155_pos_16235_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:116, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_156_pos_16285_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:120, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_157_pos_16344_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:113, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_25_pos_1515_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:135, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_26_pos_1589_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:167, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_27_pos_1653_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:122, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_28_pos_1726_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:107, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_29_pos_1826_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:113, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_30_pos_1898_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:130, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_33_pos_2069_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_34_pos_2164_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_35_pos_2234_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_36_pos_2304_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:117, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_38_pos_2511_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:95, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_39_pos_2588_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_40_pos_2692_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:166, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_41_pos_2759_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:153, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_42_pos_2803_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_44_pos_2938_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:152, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_45_pos_3014_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_81_pos_5590_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_4:3735001-3765000_gene_275_pb_140_pos_11505_strand_+_readouts_[NDB_862_c,NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_4:3735001-3765000_gene_275_pb_141_pos_11598_strand_+_readouts_[NDB_1075_c,NDB_1075_c,NDB_1075_c,NDB_1075_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_4:83735001-83765000_gene_307_pb_125_pos_20855_strand_-_readouts_[NDB_926_c,NDB_926_c,NDB_926_c,NDB_926_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:153, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_36_pos_4620_strand_+_readouts_[NDB_643_c,NDB_643_c,NDB_643_c,NDB_643_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_37_pos_4701_strand_-_readouts_[NDB_1027_c,NDB_1027_c,NDB_1027_c,NDB_1027_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_38_pos_5467_strand_-_readouts_[NDB_878_c,NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:84, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_39_pos_5728_strand_+_readouts_[NDB_643_c,NDB_643_c,NDB_643_c,NDB_643_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_40_pos_6638_strand_-_readouts_[NDB_1027_c,NDB_1027_c,NDB_1027_c,NDB_1027_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_41_pos_6697_strand_-_readouts_[NDB_878_c,NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_322_pos_23951_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_324_pos_24083_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:86, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_325_pos_24141_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:111, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_327_pos_24259_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_328_pos_24316_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_331_pos_24475_strand_-_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_333_pos_24604_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_334_pos_24666_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_335_pos_24726_strand_+_readouts_[NDB_788_c,NDB_788_c,NDB_788_c,NDB_788_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:93, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_336_pos_24833_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_337_pos_24934_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_338_pos_24985_strand_-_readouts_[NDB_788_c,NDB_788_c,NDB_788_c,NDB_788_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_272_pos_26376_strand_-_readouts_[NDB_836_c,NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_273_pos_26420_strand_+_readouts_[NDB_815_c,NDB_815_c,NDB_815_c,NDB_815_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_275_pos_26524_strand_-_readouts_[NDB_836_c,NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_276_pos_26606_strand_-_readouts_[NDB_815_c,NDB_815_c,NDB_815_c,NDB_815_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_278_pos_26734_strand_-_readouts_[NDB_836_c,NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_279_pos_26778_strand_-_readouts_[NDB_815_c,NDB_815_c,NDB_815_c,NDB_815_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_5:26235001-26265000_gene_360_pb_5_pos_876_strand_+_readouts_[NDB_899_c,NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_5:26235001-26265000_gene_360_pb_7_pos_1193_strand_+_readouts_[NDB_901_c,NDB_901_c,NDB_901_c,NDB_901_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_5:26235001-26265000_gene_360_pb_8_pos_1297_strand_+_readouts_[NDB_899_c,NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_5:143735001-143765000_gene_407_pb_169_pos_16397_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c,NDB_863_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_5:146235001-146265000_gene_408_pb_105_pos_21980_strand_-_readouts_[NDB_710_c,NDB_710_c,NDB_710_c,NDB_710_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:87, dumped\n",
      "--- Sequence:loc_6:1235001-1265000_gene_422_pb_244_pos_25980_strand_+_readouts_[NDB_860_c,NDB_860_c,NDB_860_c,NDB_860_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_6:1235001-1265000_gene_422_pb_245_pos_26025_strand_+_readouts_[NDB_721_c,NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_6:158735001-158765000_gene_485_pb_158_pos_21795_strand_-_readouts_[NDB_647_c,NDB_647_c,NDB_647_c,NDB_647_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:239, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_335_pos_26194_strand_+_readouts_[NDB_734_c,NDB_734_c,NDB_734_c,NDB_734_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:165, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_336_pos_26259_strand_+_readouts_[NDB_835_c,NDB_835_c,NDB_835_c,NDB_835_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:140, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_337_pos_26354_strand_-_readouts_[NDB_641_c,NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:159, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_338_pos_26424_strand_-_readouts_[NDB_734_c,NDB_734_c,NDB_734_c,NDB_734_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:118, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_340_pos_26524_strand_+_readouts_[NDB_641_c,NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:171, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_341_pos_26586_strand_+_readouts_[NDB_734_c,NDB_734_c,NDB_734_c,NDB_734_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:148, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_342_pos_26632_strand_-_readouts_[NDB_835_c,NDB_835_c,NDB_835_c,NDB_835_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_200_pos_20841_strand_+_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_201_pos_20896_strand_+_readouts_[NDB_698_c,NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_202_pos_20951_strand_+_readouts_[NDB_935_c,NDB_935_c,NDB_935_c,NDB_935_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:125, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_203_pos_21000_strand_+_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:121, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_204_pos_21062_strand_+_readouts_[NDB_698_c,NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:126, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_205_pos_21179_strand_+_readouts_[NDB_935_c,NDB_935_c,NDB_935_c,NDB_935_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:96, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_206_pos_21240_strand_-_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_207_pos_21302_strand_-_readouts_[NDB_698_c,NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_208_pos_21367_strand_-_readouts_[NDB_935_c,NDB_935_c,NDB_935_c,NDB_935_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_209_pos_21414_strand_+_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:128, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_210_pos_21474_strand_+_readouts_[NDB_698_c,NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_24_pos_2347_strand_-_readouts_[NDB_974_c,NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_25_pos_2407_strand_-_readouts_[NDB_878_c,NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_26_pos_2479_strand_-_readouts_[NDB_866_c,NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_29_pos_2681_strand_+_readouts_[NDB_866_c,NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_30_pos_2755_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_31_pos_2825_strand_+_readouts_[NDB_878_c,NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_32_pos_2882_strand_-_readouts_[NDB_866_c,NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_33_pos_2968_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:115, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_36_pos_3235_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_37_pos_3308_strand_+_readouts_[NDB_878_c,NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:96, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_38_pos_3378_strand_+_readouts_[NDB_866_c,NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_39_pos_3443_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_40_pos_3518_strand_+_readouts_[NDB_878_c,NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_41_pos_3594_strand_+_readouts_[NDB_866_c,NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_201_pos_17367_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_202_pos_17440_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_203_pos_17498_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:159, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_204_pos_17567_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:139, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_205_pos_17612_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:144, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_206_pos_17716_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:162, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_207_pos_17792_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:106, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_209_pos_17892_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_210_pos_17936_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_211_pos_17986_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:115, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_212_pos_18069_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:161, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_213_pos_18113_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:149, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_214_pos_18163_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_215_pos_18207_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:132, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_216_pos_18256_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:161, dumped\n",
      "--- Sequence:loc_1:156235001-156265000_gene_62_pb_184_pos_15904_strand_-_readouts_[NDB_905_c,NDB_905_c,NDB_905_c,NDB_905_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_1:168735001-168765000_gene_67_pb_101_pos_9463_strand_+_readouts_[NDB_836_c,NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_1:168735001-168765000_gene_67_pb_102_pos_9523_strand_+_readouts_[NDB_847_c,NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_9:130985001-131015000_gene_676_pb_149_pos_11443_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c,NDB_883_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_10:985001-1015000_gene_680_pb_254_pos_26107_strand_-_readouts_[NDB_994_c,NDB_994_c,NDB_994_c,NDB_994_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_10:2985001-3015000_gene_681_pb_158_pos_17054_strand_-_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_10:2985001-3015000_gene_681_pb_161_pos_17237_strand_-_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_10:88985001-89015000_gene_724_pb_20_pos_1662_strand_+_readouts_[NDB_694_c,NDB_694_c,NDB_694_c,NDB_694_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_10:116985001-117015000_gene_738_pb_163_pos_19989_strand_-_readouts_[NDB_634_c,NDB_634_c,NDB_634_c,NDB_634_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:372, dumped\n",
      "--- Sequence:loc_10:126985001-127015000_gene_743_pb_178_pos_18183_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_10:126985001-127015000_gene_743_pb_179_pos_18261_strand_+_readouts_[NDB_836_c,NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_10:126985001-127015000_gene_743_pb_180_pos_18332_strand_+_readouts_[NDB_1027_c,NDB_1027_c,NDB_1027_c,NDB_1027_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_178_pos_16773_strand_+_readouts_[NDB_728_c,NDB_728_c,NDB_728_c,NDB_728_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_179_pos_16846_strand_-_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_180_pos_16892_strand_+_readouts_[NDB_713_c,NDB_713_c,NDB_713_c,NDB_713_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_182_pos_17002_strand_-_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_183_pos_17084_strand_+_readouts_[NDB_713_c,NDB_713_c,NDB_713_c,NDB_713_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_184_pos_17128_strand_-_readouts_[NDB_728_c,NDB_728_c,NDB_728_c,NDB_728_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_185_pos_17200_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_186_pos_17244_strand_+_readouts_[NDB_713_c,NDB_713_c,NDB_713_c,NDB_713_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_187_pos_17288_strand_+_readouts_[NDB_728_c,NDB_728_c,NDB_728_c,NDB_728_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_191_pos_17568_strand_-_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:95, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_192_pos_17612_strand_-_readouts_[NDB_713_c,NDB_713_c,NDB_713_c,NDB_713_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_11:32985001-33015000_gene_762_pb_93_pos_21050_strand_+_readouts_[NDB_847_c,NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:121, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_11:62985001-63015000_gene_777_pb_48_pos_4612_strand_-_readouts_[NDB_716_c,NDB_716_c,NDB_716_c,NDB_716_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_11:74985001-75015000_gene_783_pb_197_pos_23135_strand_+_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:129, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_152_pos_21019_strand_-_readouts_[NDB_721_c,NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_154_pos_21355_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_157_pos_22005_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_158_pos_22153_strand_+_readouts_[NDB_721_c,NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_159_pos_22288_strand_+_readouts_[NDB_718_c,NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_164_pos_23479_strand_+_readouts_[NDB_721_c,NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_12:4985001-5015000_gene_815_pb_289_pos_25626_strand_+_readouts_[NDB_856_c,NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_12:4985001-5015000_gene_815_pb_291_pos_25784_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_12:4985001-5015000_gene_815_pb_292_pos_25887_strand_+_readouts_[NDB_856_c,NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_12:4985001-5015000_gene_815_pb_293_pos_25939_strand_+_readouts_[NDB_641_c,NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_12:118985001-119015000_gene_872_pb_290_pos_23481_strand_-_readouts_[NDB_647_c,NDB_647_c,NDB_647_c,NDB_647_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_12:120985001-121015000_gene_873_pb_79_pos_9825_strand_+_readouts_[NDB_946_c,NDB_946_c,NDB_946_c,NDB_946_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_12:120985001-121015000_gene_873_pb_80_pos_9884_strand_+_readouts_[NDB_751_c,NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_12:120985001-121015000_gene_873_pb_83_pos_10046_strand_-_readouts_[NDB_751_c,NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_12:120985001-121015000_gene_873_pb_84_pos_10105_strand_+_readouts_[NDB_652_c,NDB_652_c,NDB_652_c,NDB_652_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_164_pos_19738_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:139, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_165_pos_19825_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:148, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_166_pos_20061_strand_+_readouts_[NDB_680_c,NDB_680_c,NDB_680_c,NDB_680_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_167_pos_20118_strand_+_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_168_pos_20162_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:129, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_169_pos_20230_strand_+_readouts_[NDB_680_c,NDB_680_c,NDB_680_c,NDB_680_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_170_pos_20292_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:132, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_171_pos_20346_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:149, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_177_pos_20690_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_13:22985001-23015000_gene_890_pb_119_pos_16706_strand_+_readouts_[NDB_977_c,NDB_977_c,NDB_977_c,NDB_977_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_60_pos_6577_strand_+_readouts_[NDB_715_c,NDB_715_c,NDB_715_c,NDB_715_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_61_pos_6683_strand_+_readouts_[NDB_878_c,NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_62_pos_6738_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_185_pos_23270_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_186_pos_23660_strand_+_readouts_[NDB_715_c,NDB_715_c,NDB_715_c,NDB_715_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_187_pos_23785_strand_-_readouts_[NDB_878_c,NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_56_pos_3544_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:117, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_58_pos_3721_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:116, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_60_pos_3826_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_62_pos_4004_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_63_pos_4056_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:145, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_64_pos_4232_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:125, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_65_pos_4304_strand_-_readouts_[NDB_989_c,NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_66_pos_4380_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:141, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_67_pos_4428_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:127, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_69_pos_4567_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:138, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_70_pos_4637_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_71_pos_4681_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:111, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_275_pos_20501_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_276_pos_20550_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_355_pos_26406_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_356_pos_26460_strand_-_readouts_[NDB_989_c,NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_357_pos_26572_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_358_pos_26643_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_359_pos_26687_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:93, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_360_pos_26731_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_361_pos_26910_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_362_pos_27020_strand_-_readouts_[NDB_989_c,NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_363_pos_27146_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_365_pos_27360_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:106, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_366_pos_27420_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:69, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_367_pos_27464_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_369_pos_27591_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:119, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_370_pos_27647_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:157, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_371_pos_27727_strand_-_readouts_[NDB_989_c,NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:120, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_372_pos_27864_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_373_pos_27954_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:157, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_14:50235001-50265000_gene_969_pb_21_pos_3641_strand_-_readouts_[NDB_875_c,NDB_875_c,NDB_875_c,NDB_875_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_14:57735001-57765000_gene_974_pb_121_pos_18282_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_14:59235001-59265000_gene_975_pb_60_pos_5799_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_1:246235001-246265000_gene_98_pb_98_pos_16513_strand_+_readouts_[NDB_946_c,NDB_946_c,NDB_946_c,NDB_946_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_1:246235001-246265000_gene_98_pb_100_pos_17135_strand_-_readouts_[NDB_844_c,NDB_844_c,NDB_844_c,NDB_844_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_7_pos_405_strand_+_readouts_[NDB_994_c,NDB_994_c,NDB_994_c,NDB_994_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_9_pos_565_strand_+_readouts_[NDB_880_c,NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_10_pos_669_strand_-_readouts_[NDB_994_c,NDB_994_c,NDB_994_c,NDB_994_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_12_pos_765_strand_-_readouts_[NDB_880_c,NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_13_pos_898_strand_-_readouts_[NDB_994_c,NDB_994_c,NDB_994_c,NDB_994_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_14_pos_985_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_15_pos_1101_strand_+_readouts_[NDB_880_c,NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_17_pos_1273_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_18_pos_1386_strand_-_readouts_[NDB_880_c,NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "-- total probes removed by internal screening: 453\n"
     ]
    }
   ],
   "source": [
    "# check between probes\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map, \n",
    "                                                          primer_len=primer_len, \n",
    "                                                          target_len=target_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "id": "495f8a7d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# clear memory\n",
    "del(int_map)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ef0c0c1c",
   "metadata": {},
   "source": [
    "### 4.1.6 finish checking, save"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 149,
   "id": "0b11845a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "327063 327516\n"
     ]
    }
   ],
   "source": [
    "print(len(kept_records), len(cand_probes))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 150,
   "id": "56768f43",
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4c40e93f",
   "metadata": {},
   "source": [
    "## 4.2 BLAST screening of probes\n",
    "\n",
    "create the local blast database by:\n",
    "```bash\n",
    "makeblastdb -in C:\\Users\\puzheng\\Documents\\Tools\\lowercase_masked\\Mus_musculus.GRCm38.dna_sm.merged.fa -dbtype nucl -parse_seqids -out GRCm38_ensembl -title \"Mouse chromosomes soft masked\" -blastdb_version 4![image.png](attachment:image.png)\n",
    "```\n",
    "\n",
    "The outputs are:\n",
    "```bash\n",
    "Building a new DB, current time: 03/30/2021 21:49:18\n",
    "New DB name:   C:\\Users\\puzheng\\Documents\\Tools\\GRCm38_ensembl\n",
    "New DB title:  Mouse chromosomes soft masked\n",
    "Sequence type: Nucleotide\n",
    "Keep MBits: T\n",
    "Maximum file size: 1000000000B\n",
    "Adding sequences from FASTA; added 66 sequences in 42.2668 seconds.\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 151,
   "id": "5045ff8f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<module 'ImageAnalysis3.library_tools.quality_check' from 'E:\\\\Users\\\\puzheng_new\\\\Documents\\\\Chromatin_Analysis_Scripts\\\\Library_design\\\\CTP-14-human_brain_genome\\\\..\\\\..\\\\..\\\\..\\\\Documents\\\\ImageAnalysis3\\\\library_tools\\\\quality_check.py'>"
      ]
     },
     "execution_count": 151,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 152,
   "id": "bef107d6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "327063\n"
     ]
    }
   ],
   "source": [
    "if 'kept_records' not in locals() or len(kept_records) == 0:\n",
    "    kept_records = []\n",
    "    with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'r') as handle:\n",
    "        for record in SeqIO.parse(handle, \"fasta\"):\n",
    "            kept_records.append(record)\n",
    "print(len(kept_records))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 153,
   "id": "75e15498",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\blast_database\\hg38\n",
      "-- number of probes loaded: 327063\n",
      "-- processing region: 0 with 339 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_0.fasta\n",
      "--- *blasting region: 0\n",
      "--- total time for blast 0: 22.302s.\n",
      "-- processing region: 1 with 306 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1.fasta\n",
      "--- *blasting region: 1\n",
      "--- total time for blast 1: 14.681s.\n",
      "-- processing region: 10 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_10.fasta\n",
      "--- *blasting region: 10\n",
      "--- total time for blast 10: 14.056s.\n",
      "-- processing region: 100 with 344 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_100.fasta\n",
      "--- *blasting region: 100\n",
      "--- total time for blast 100: 15.268s.\n",
      "-- processing region: 1000 with 326 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1000.fasta\n",
      "--- *blasting region: 1000\n",
      "--- total time for blast 1000: 15.705s.\n",
      "-- processing region: 1001 with 341 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1001.fasta\n",
      "--- *blasting region: 1001\n",
      "--- total time for blast 1001: 16.193s.\n",
      "-- processing region: 1002 with 381 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1002.fasta\n",
      "--- *blasting region: 1002\n",
      "--- total time for blast 1002: 24.550s.\n",
      "-- processing region: 1003 with 332 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1003.fasta\n",
      "--- *blasting region: 1003\n",
      "--- total time for blast 1003: 15.257s.\n",
      "-- processing region: 1004 with 283 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1004.fasta\n",
      "--- *blasting region: 1004\n",
      "--- total time for blast 1004: 12.697s.\n",
      "-- processing region: 1005 with 313 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1005.fasta\n",
      "--- *blasting region: 1005\n",
      "--- total time for blast 1005: 13.109s.\n",
      "-- processing region: 101 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_101.fasta\n",
      "--- *blasting region: 101\n",
      "--- total time for blast 101: 11.627s.\n",
      "-- processing region: 102 with 240 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_102.fasta\n",
      "--- *blasting region: 102\n",
      "--- total time for blast 102: 13.493s.\n",
      "-- processing region: 1020 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1020.fasta\n",
      "--- *blasting region: 1020\n",
      "--- total time for blast 1020: 3.695s.\n",
      "-- processing region: 1023 with 217 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1023.fasta\n",
      "--- *blasting region: 1023\n",
      "--- total time for blast 1023: 14.572s.\n",
      "-- processing region: 1024 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1024.fasta\n",
      "--- *blasting region: 1024\n",
      "--- total time for blast 1024: 16.394s.\n",
      "-- processing region: 1025 with 306 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1025.fasta\n",
      "--- *blasting region: 1025\n",
      "--- total time for blast 1025: 12.987s.\n",
      "-- processing region: 1029 with 235 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1029.fasta\n",
      "--- *blasting region: 1029\n",
      "--- total time for blast 1029: 10.540s.\n",
      "-- processing region: 103 with 205 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_103.fasta\n",
      "--- *blasting region: 103\n",
      "--- total time for blast 103: 9.908s.\n",
      "-- processing region: 1030 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1030.fasta\n",
      "--- *blasting region: 1030\n",
      "--- total time for blast 1030: 9.291s.\n",
      "-- processing region: 1031 with 290 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1031.fasta\n",
      "--- *blasting region: 1031\n",
      "--- total time for blast 1031: 12.072s.\n",
      "-- processing region: 1032 with 275 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1032.fasta\n",
      "--- *blasting region: 1032\n",
      "--- total time for blast 1032: 13.239s.\n",
      "-- processing region: 1033 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1033.fasta\n",
      "--- *blasting region: 1033\n",
      "--- total time for blast 1033: 17.489s.\n",
      "-- processing region: 1034 with 183 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1034.fasta\n",
      "--- *blasting region: 1034\n",
      "--- total time for blast 1034: 12.127s.\n",
      "-- processing region: 1035 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1035.fasta\n",
      "--- *blasting region: 1035\n",
      "--- total time for blast 1035: 9.717s.\n",
      "-- processing region: 1036 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1036.fasta\n",
      "--- *blasting region: 1036\n",
      "--- total time for blast 1036: 6.710s.\n",
      "-- processing region: 1037 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1037.fasta\n",
      "--- *blasting region: 1037\n",
      "--- total time for blast 1037: 11.006s.\n",
      "-- processing region: 1038 with 227 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1038.fasta\n",
      "--- *blasting region: 1038\n",
      "--- total time for blast 1038: 11.730s.\n",
      "-- processing region: 1039 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1039.fasta\n",
      "--- *blasting region: 1039\n",
      "--- total time for blast 1039: 9.456s.\n",
      "-- processing region: 104 with 119 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_104.fasta\n",
      "--- *blasting region: 104\n",
      "--- total time for blast 104: 9.020s.\n",
      "-- processing region: 1040 with 350 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1040.fasta\n",
      "--- *blasting region: 1040\n",
      "--- total time for blast 1040: 13.761s.\n",
      "-- processing region: 1041 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1041.fasta\n",
      "--- *blasting region: 1041\n",
      "--- total time for blast 1041: 12.501s.\n",
      "-- processing region: 1042 with 325 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1042.fasta\n",
      "--- *blasting region: 1042\n",
      "--- total time for blast 1042: 18.851s.\n",
      "-- processing region: 1043 with 122 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1043.fasta\n",
      "--- *blasting region: 1043\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1043: 13.599s.\n",
      "-- processing region: 1044 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1044.fasta\n",
      "--- *blasting region: 1044\n",
      "--- total time for blast 1044: 13.597s.\n",
      "-- processing region: 1045 with 347 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1045.fasta\n",
      "--- *blasting region: 1045\n",
      "--- total time for blast 1045: 15.639s.\n",
      "-- processing region: 1046 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1046.fasta\n",
      "--- *blasting region: 1046\n",
      "--- total time for blast 1046: 13.001s.\n",
      "-- processing region: 1047 with 350 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1047.fasta\n",
      "--- *blasting region: 1047\n",
      "--- total time for blast 1047: 13.573s.\n",
      "-- processing region: 1048 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1048.fasta\n",
      "--- *blasting region: 1048\n",
      "--- total time for blast 1048: 14.589s.\n",
      "-- processing region: 1049 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1049.fasta\n",
      "--- *blasting region: 1049\n",
      "--- total time for blast 1049: 10.830s.\n",
      "-- processing region: 105 with 352 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_105.fasta\n",
      "--- *blasting region: 105\n",
      "--- total time for blast 105: 14.477s.\n",
      "-- processing region: 1050 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1050.fasta\n",
      "--- *blasting region: 1050\n",
      "--- total time for blast 1050: 12.147s.\n",
      "-- processing region: 1051 with 384 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1051.fasta\n",
      "--- *blasting region: 1051\n",
      "--- total time for blast 1051: 16.152s.\n",
      "-- processing region: 1052 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1052.fasta\n",
      "--- *blasting region: 1052\n",
      "--- total time for blast 1052: 11.911s.\n",
      "-- processing region: 1053 with 351 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1053.fasta\n",
      "--- *blasting region: 1053\n",
      "--- total time for blast 1053: 15.928s.\n",
      "-- processing region: 1054 with 320 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1054.fasta\n",
      "--- *blasting region: 1054\n",
      "--- total time for blast 1054: 15.323s.\n",
      "-- processing region: 1055 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1055.fasta\n",
      "--- *blasting region: 1055\n",
      "--- total time for blast 1055: 15.041s.\n",
      "-- processing region: 1056 with 361 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1056.fasta\n",
      "--- *blasting region: 1056\n",
      "--- total time for blast 1056: 13.961s.\n",
      "-- processing region: 1057 with 155 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1057.fasta\n",
      "--- *blasting region: 1057\n",
      "--- total time for blast 1057: 10.119s.\n",
      "-- processing region: 1058 with 131 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1058.fasta\n",
      "--- *blasting region: 1058\n",
      "--- total time for blast 1058: 7.323s.\n",
      "-- processing region: 1059 with 164 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1059.fasta\n",
      "--- *blasting region: 1059\n",
      "--- total time for blast 1059: 6.870s.\n",
      "-- processing region: 106 with 244 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_106.fasta\n",
      "--- *blasting region: 106\n",
      "--- total time for blast 106: 11.998s.\n",
      "-- processing region: 1060 with 315 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1060.fasta\n",
      "--- *blasting region: 1060\n",
      "--- total time for blast 1060: 14.230s.\n",
      "-- processing region: 1061 with 309 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1061.fasta\n",
      "--- *blasting region: 1061\n",
      "--- total time for blast 1061: 15.299s.\n",
      "-- processing region: 1062 with 240 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1062.fasta\n",
      "--- *blasting region: 1062\n",
      "--- total time for blast 1062: 11.739s.\n",
      "-- processing region: 1063 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1063.fasta\n",
      "--- *blasting region: 1063\n",
      "--- total time for blast 1063: 12.864s.\n",
      "-- processing region: 1064 with 327 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1064.fasta\n",
      "--- *blasting region: 1064\n",
      "--- total time for blast 1064: 15.196s.\n",
      "-- processing region: 1065 with 318 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1065.fasta\n",
      "--- *blasting region: 1065\n",
      "--- total time for blast 1065: 14.651s.\n",
      "-- processing region: 1066 with 207 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1066.fasta\n",
      "--- *blasting region: 1066\n",
      "--- total time for blast 1066: 15.596s.\n",
      "-- processing region: 1067 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1067.fasta\n",
      "--- *blasting region: 1067\n",
      "--- total time for blast 1067: 9.747s.\n",
      "-- processing region: 1068 with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1068.fasta\n",
      "--- *blasting region: 1068\n",
      "--- total time for blast 1068: 11.449s.\n",
      "-- processing region: 1069 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1069.fasta\n",
      "--- *blasting region: 1069\n",
      "--- total time for blast 1069: 11.694s.\n",
      "-- processing region: 107 with 305 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_107.fasta\n",
      "--- *blasting region: 107\n",
      "--- total time for blast 107: 14.071s.\n",
      "-- processing region: 1070 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1070.fasta\n",
      "--- *blasting region: 1070\n",
      "--- total time for blast 1070: 12.018s.\n",
      "-- processing region: 1071 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1071.fasta\n",
      "--- *blasting region: 1071\n",
      "--- total time for blast 1071: 13.396s.\n",
      "-- processing region: 1072 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1072.fasta\n",
      "--- *blasting region: 1072\n",
      "--- total time for blast 1072: 12.703s.\n",
      "-- processing region: 1073 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1073.fasta\n",
      "--- *blasting region: 1073\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1073: 13.600s.\n",
      "-- processing region: 1074 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1074.fasta\n",
      "--- *blasting region: 1074\n",
      "--- total time for blast 1074: 10.206s.\n",
      "-- processing region: 1075 with 256 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1075.fasta\n",
      "--- *blasting region: 1075\n",
      "--- total time for blast 1075: 10.883s.\n",
      "-- processing region: 1076 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1076.fasta\n",
      "--- *blasting region: 1076\n",
      "--- total time for blast 1076: 13.714s.\n",
      "-- processing region: 1077 with 88 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1077.fasta\n",
      "--- *blasting region: 1077\n",
      "--- total time for blast 1077: 4.567s.\n",
      "-- processing region: 1078 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1078.fasta\n",
      "--- *blasting region: 1078\n",
      "--- total time for blast 1078: 11.669s.\n",
      "-- processing region: 1079 with 302 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1079.fasta\n",
      "--- *blasting region: 1079\n",
      "--- total time for blast 1079: 16.668s.\n",
      "-- processing region: 108 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_108.fasta\n",
      "--- *blasting region: 108\n",
      "--- total time for blast 108: 10.385s.\n",
      "-- processing region: 1080 with 362 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1080.fasta\n",
      "--- *blasting region: 1080\n",
      "--- total time for blast 1080: 14.498s.\n",
      "-- processing region: 1081 with 298 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1081.fasta\n",
      "--- *blasting region: 1081\n",
      "--- total time for blast 1081: 12.701s.\n",
      "-- processing region: 1082 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1082.fasta\n",
      "--- *blasting region: 1082\n",
      "--- total time for blast 1082: 13.069s.\n",
      "-- processing region: 1083 with 325 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1083.fasta\n",
      "--- *blasting region: 1083\n",
      "--- total time for blast 1083: 13.334s.\n",
      "-- processing region: 1085 with 341 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1085.fasta\n",
      "--- *blasting region: 1085\n",
      "--- total time for blast 1085: 17.391s.\n",
      "-- processing region: 1087 with 336 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1087.fasta\n",
      "--- *blasting region: 1087\n",
      "--- total time for blast 1087: 13.448s.\n",
      "-- processing region: 1089 with 207 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1089.fasta\n",
      "--- *blasting region: 1089\n",
      "--- total time for blast 1089: 11.137s.\n",
      "-- processing region: 109 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_109.fasta\n",
      "--- *blasting region: 109\n",
      "--- total time for blast 109: 13.634s.\n",
      "-- processing region: 1090 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1090.fasta\n",
      "--- *blasting region: 1090\n",
      "--- total time for blast 1090: 11.575s.\n",
      "-- processing region: 1091 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1091.fasta\n",
      "--- *blasting region: 1091\n",
      "--- total time for blast 1091: 14.635s.\n",
      "-- processing region: 1092 with 355 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1092.fasta\n",
      "--- *blasting region: 1092\n",
      "--- total time for blast 1092: 16.009s.\n",
      "-- processing region: 1093 with 203 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1093.fasta\n",
      "--- *blasting region: 1093\n",
      "--- total time for blast 1093: 9.057s.\n",
      "-- processing region: 1094 with 317 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1094.fasta\n",
      "--- *blasting region: 1094\n",
      "--- total time for blast 1094: 11.931s.\n",
      "-- processing region: 1097 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1097.fasta\n",
      "--- *blasting region: 1097\n",
      "--- total time for blast 1097: 10.400s.\n",
      "-- processing region: 11 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_11.fasta\n",
      "--- *blasting region: 11\n",
      "--- total time for blast 11: 9.665s.\n",
      "-- processing region: 110 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_110.fasta\n",
      "--- *blasting region: 110\n",
      "--- total time for blast 110: 13.655s.\n",
      "-- processing region: 1105 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1105.fasta\n",
      "--- *blasting region: 1105\n",
      "--- total time for blast 1105: 9.489s.\n",
      "-- processing region: 1106 with 189 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1106.fasta\n",
      "--- *blasting region: 1106\n",
      "--- total time for blast 1106: 19.614s.\n",
      "-- processing region: 1107 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1107.fasta\n",
      "--- *blasting region: 1107\n",
      "--- total time for blast 1107: 11.564s.\n",
      "-- processing region: 1108 with 295 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1108.fasta\n",
      "--- *blasting region: 1108\n",
      "--- total time for blast 1108: 12.629s.\n",
      "-- processing region: 1109 with 184 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1109.fasta\n",
      "--- *blasting region: 1109\n",
      "--- total time for blast 1109: 8.549s.\n",
      "-- processing region: 111 with 358 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_111.fasta\n",
      "--- *blasting region: 111\n",
      "--- total time for blast 111: 13.178s.\n",
      "-- processing region: 1110 with 336 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1110.fasta\n",
      "--- *blasting region: 1110\n",
      "--- total time for blast 1110: 16.276s.\n",
      "-- processing region: 1111 with 381 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1111.fasta\n",
      "--- *blasting region: 1111\n",
      "--- total time for blast 1111: 16.231s.\n",
      "-- processing region: 1112 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1112.fasta\n",
      "--- *blasting region: 1112\n",
      "--- total time for blast 1112: 15.681s.\n",
      "-- processing region: 1113 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1113.fasta\n",
      "--- *blasting region: 1113\n",
      "--- total time for blast 1113: 13.921s.\n",
      "-- processing region: 1114 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1114.fasta\n",
      "--- *blasting region: 1114\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1114: 9.357s.\n",
      "-- processing region: 1115 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1115.fasta\n",
      "--- *blasting region: 1115\n",
      "--- total time for blast 1115: 14.674s.\n",
      "-- processing region: 1116 with 184 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1116.fasta\n",
      "--- *blasting region: 1116\n",
      "--- total time for blast 1116: 16.571s.\n",
      "-- processing region: 1117 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1117.fasta\n",
      "--- *blasting region: 1117\n",
      "--- total time for blast 1117: 14.012s.\n",
      "-- processing region: 1118 with 231 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1118.fasta\n",
      "--- *blasting region: 1118\n",
      "--- total time for blast 1118: 13.010s.\n",
      "-- processing region: 1119 with 328 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1119.fasta\n",
      "--- *blasting region: 1119\n",
      "--- total time for blast 1119: 11.454s.\n",
      "-- processing region: 112 with 167 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_112.fasta\n",
      "--- *blasting region: 112\n",
      "--- total time for blast 112: 10.972s.\n",
      "-- processing region: 1120 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1120.fasta\n",
      "--- *blasting region: 1120\n",
      "--- total time for blast 1120: 18.504s.\n",
      "-- processing region: 1121 with 298 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1121.fasta\n",
      "--- *blasting region: 1121\n",
      "--- total time for blast 1121: 11.720s.\n",
      "-- processing region: 1122 with 227 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1122.fasta\n",
      "--- *blasting region: 1122\n",
      "--- total time for blast 1122: 13.259s.\n",
      "-- processing region: 1123 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1123.fasta\n",
      "--- *blasting region: 1123\n",
      "--- total time for blast 1123: 11.773s.\n",
      "-- processing region: 1124 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1124.fasta\n",
      "--- *blasting region: 1124\n",
      "--- total time for blast 1124: 9.292s.\n",
      "-- processing region: 1125 with 208 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1125.fasta\n",
      "--- *blasting region: 1125\n",
      "--- total time for blast 1125: 15.356s.\n",
      "-- processing region: 1126 with 320 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1126.fasta\n",
      "--- *blasting region: 1126\n",
      "--- total time for blast 1126: 17.990s.\n",
      "-- processing region: 1127 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1127.fasta\n",
      "--- *blasting region: 1127\n",
      "--- total time for blast 1127: 12.243s.\n",
      "-- processing region: 1128 with 235 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1128.fasta\n",
      "--- *blasting region: 1128\n",
      "--- total time for blast 1128: 21.994s.\n",
      "-- processing region: 1129 with 335 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1129.fasta\n",
      "--- *blasting region: 1129\n",
      "--- total time for blast 1129: 15.704s.\n",
      "-- processing region: 1130 with 311 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1130.fasta\n",
      "--- *blasting region: 1130\n",
      "--- total time for blast 1130: 12.445s.\n",
      "-- processing region: 1131 with 272 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1131.fasta\n",
      "--- *blasting region: 1131\n",
      "--- total time for blast 1131: 11.269s.\n",
      "-- processing region: 1132 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1132.fasta\n",
      "--- *blasting region: 1132\n",
      "--- total time for blast 1132: 10.150s.\n",
      "-- processing region: 1133 with 205 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1133.fasta\n",
      "--- *blasting region: 1133\n",
      "--- total time for blast 1133: 13.515s.\n",
      "-- processing region: 1134 with 309 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1134.fasta\n",
      "--- *blasting region: 1134\n",
      "--- total time for blast 1134: 16.170s.\n",
      "-- processing region: 1135 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1135.fasta\n",
      "--- *blasting region: 1135\n",
      "--- total time for blast 1135: 36.326s.\n",
      "-- processing region: 1136 with 290 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1136.fasta\n",
      "--- *blasting region: 1136\n",
      "--- total time for blast 1136: 11.129s.\n",
      "-- processing region: 1137 with 193 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1137.fasta\n",
      "--- *blasting region: 1137\n",
      "--- total time for blast 1137: 17.653s.\n",
      "-- processing region: 1138 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1138.fasta\n",
      "--- *blasting region: 1138\n",
      "--- total time for blast 1138: 12.078s.\n",
      "-- processing region: 1139 with 256 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1139.fasta\n",
      "--- *blasting region: 1139\n",
      "--- total time for blast 1139: 13.292s.\n",
      "-- processing region: 114 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_114.fasta\n",
      "--- *blasting region: 114\n",
      "--- total time for blast 114: 10.788s.\n",
      "-- processing region: 1140 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1140.fasta\n",
      "--- *blasting region: 1140\n",
      "--- total time for blast 1140: 9.288s.\n",
      "-- processing region: 1141 with 375 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1141.fasta\n",
      "--- *blasting region: 1141\n",
      "--- total time for blast 1141: 15.180s.\n",
      "-- processing region: 1142 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1142.fasta\n",
      "--- *blasting region: 1142\n",
      "--- total time for blast 1142: 15.311s.\n",
      "-- processing region: 1143 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1143.fasta\n",
      "--- *blasting region: 1143\n",
      "--- total time for blast 1143: 12.230s.\n",
      "-- processing region: 1144 with 122 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1144.fasta\n",
      "--- *blasting region: 1144\n",
      "--- total time for blast 1144: 6.906s.\n",
      "-- processing region: 1145 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1145.fasta\n",
      "--- *blasting region: 1145\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1145: 8.923s.\n",
      "-- processing region: 1146 with 94 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1146.fasta\n",
      "--- *blasting region: 1146\n",
      "--- total time for blast 1146: 4.739s.\n",
      "-- processing region: 1147 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1147.fasta\n",
      "--- *blasting region: 1147\n",
      "--- total time for blast 1147: 12.955s.\n",
      "-- processing region: 1148 with 101 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1148.fasta\n",
      "--- *blasting region: 1148\n",
      "--- total time for blast 1148: 5.540s.\n",
      "-- processing region: 115 with 224 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_115.fasta\n",
      "--- *blasting region: 115\n",
      "--- total time for blast 115: 12.135s.\n",
      "-- processing region: 1152 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1152.fasta\n",
      "--- *blasting region: 1152\n",
      "--- total time for blast 1152: 10.543s.\n",
      "-- processing region: 1153 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1153.fasta\n",
      "--- *blasting region: 1153\n",
      "--- total time for blast 1153: 11.833s.\n",
      "-- processing region: 1154 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1154.fasta\n",
      "--- *blasting region: 1154\n",
      "--- total time for blast 1154: 4.089s.\n",
      "-- processing region: 1155 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1155.fasta\n",
      "--- *blasting region: 1155\n",
      "--- total time for blast 1155: 12.078s.\n",
      "-- processing region: 1156 with 327 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1156.fasta\n",
      "--- *blasting region: 1156\n",
      "--- total time for blast 1156: 14.267s.\n",
      "-- processing region: 1157 with 276 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1157.fasta\n",
      "--- *blasting region: 1157\n",
      "--- total time for blast 1157: 12.274s.\n",
      "-- processing region: 116 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_116.fasta\n",
      "--- *blasting region: 116\n",
      "--- total time for blast 116: 12.624s.\n",
      "-- processing region: 1160 with 317 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1160.fasta\n",
      "--- *blasting region: 1160\n",
      "--- total time for blast 1160: 12.561s.\n",
      "-- processing region: 1162 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1162.fasta\n",
      "--- *blasting region: 1162\n",
      "--- total time for blast 1162: 9.967s.\n",
      "-- processing region: 1163 with 272 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1163.fasta\n",
      "--- *blasting region: 1163\n",
      "--- total time for blast 1163: 10.881s.\n",
      "-- processing region: 1165 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1165.fasta\n",
      "--- *blasting region: 1165\n",
      "--- total time for blast 1165: 12.125s.\n",
      "-- processing region: 1166 with 342 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1166.fasta\n",
      "--- *blasting region: 1166\n",
      "--- total time for blast 1166: 17.090s.\n",
      "-- processing region: 1167 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1167.fasta\n",
      "--- *blasting region: 1167\n",
      "--- total time for blast 1167: 13.112s.\n",
      "-- processing region: 1168 with 346 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1168.fasta\n",
      "--- *blasting region: 1168\n",
      "--- total time for blast 1168: 15.908s.\n",
      "-- processing region: 1169 with 185 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1169.fasta\n",
      "--- *blasting region: 1169\n",
      "--- total time for blast 1169: 11.457s.\n",
      "-- processing region: 117 with 276 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_117.fasta\n",
      "--- *blasting region: 117\n",
      "--- total time for blast 117: 13.410s.\n",
      "-- processing region: 1170 with 232 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1170.fasta\n",
      "--- *blasting region: 1170\n",
      "--- total time for blast 1170: 13.042s.\n",
      "-- processing region: 1171 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1171.fasta\n",
      "--- *blasting region: 1171\n",
      "--- total time for blast 1171: 20.453s.\n",
      "-- processing region: 1172 with 329 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1172.fasta\n",
      "--- *blasting region: 1172\n",
      "--- total time for blast 1172: 14.124s.\n",
      "-- processing region: 1173 with 96 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1173.fasta\n",
      "--- *blasting region: 1173\n",
      "--- total time for blast 1173: 7.540s.\n",
      "-- processing region: 1174 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1174.fasta\n",
      "--- *blasting region: 1174\n",
      "--- total time for blast 1174: 13.164s.\n",
      "-- processing region: 1176 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1176.fasta\n",
      "--- *blasting region: 1176\n",
      "--- total time for blast 1176: 11.725s.\n",
      "-- processing region: 1177 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1177.fasta\n",
      "--- *blasting region: 1177\n",
      "--- total time for blast 1177: 10.516s.\n",
      "-- processing region: 1178 with 319 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1178.fasta\n",
      "--- *blasting region: 1178\n",
      "--- total time for blast 1178: 14.645s.\n",
      "-- processing region: 1179 with 197 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1179.fasta\n",
      "--- *blasting region: 1179\n",
      "--- total time for blast 1179: 16.580s.\n",
      "-- processing region: 118 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_118.fasta\n",
      "--- *blasting region: 118\n",
      "--- total time for blast 118: 10.874s.\n",
      "-- processing region: 1180 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1180.fasta\n",
      "--- *blasting region: 1180\n",
      "--- total time for blast 1180: 10.117s.\n",
      "-- processing region: 1181 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1181.fasta\n",
      "--- *blasting region: 1181\n",
      "--- total time for blast 1181: 12.748s.\n",
      "-- processing region: 1182 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1182.fasta\n",
      "--- *blasting region: 1182\n",
      "--- total time for blast 1182: 12.942s.\n",
      "-- processing region: 1183 with 337 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1183.fasta\n",
      "--- *blasting region: 1183\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1183: 15.056s.\n",
      "-- processing region: 1184 with 310 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1184.fasta\n",
      "--- *blasting region: 1184\n",
      "--- total time for blast 1184: 12.921s.\n",
      "-- processing region: 1186 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1186.fasta\n",
      "--- *blasting region: 1186\n",
      "--- total time for blast 1186: 31.931s.\n",
      "-- processing region: 1187 with 284 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1187.fasta\n",
      "--- *blasting region: 1187\n",
      "--- total time for blast 1187: 12.868s.\n",
      "-- processing region: 1188 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1188.fasta\n",
      "--- *blasting region: 1188\n",
      "--- total time for blast 1188: 3.880s.\n",
      "-- processing region: 1189 with 171 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1189.fasta\n",
      "--- *blasting region: 1189\n",
      "--- total time for blast 1189: 12.538s.\n",
      "-- processing region: 119 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_119.fasta\n",
      "--- *blasting region: 119\n",
      "--- total time for blast 119: 11.066s.\n",
      "-- processing region: 1190 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1190.fasta\n",
      "--- *blasting region: 1190\n",
      "--- total time for blast 1190: 10.523s.\n",
      "-- processing region: 1191 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1191.fasta\n",
      "--- *blasting region: 1191\n",
      "--- total time for blast 1191: 11.589s.\n",
      "-- processing region: 1192 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1192.fasta\n",
      "--- *blasting region: 1192\n",
      "--- total time for blast 1192: 12.610s.\n",
      "-- processing region: 1193 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1193.fasta\n",
      "--- *blasting region: 1193\n",
      "--- total time for blast 1193: 11.722s.\n",
      "-- processing region: 1194 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1194.fasta\n",
      "--- *blasting region: 1194\n",
      "--- total time for blast 1194: 15.213s.\n",
      "-- processing region: 1195 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1195.fasta\n",
      "--- *blasting region: 1195\n",
      "--- total time for blast 1195: 13.456s.\n",
      "-- processing region: 1196 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1196.fasta\n",
      "--- *blasting region: 1196\n",
      "--- total time for blast 1196: 15.031s.\n",
      "-- processing region: 1197 with 184 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1197.fasta\n",
      "--- *blasting region: 1197\n",
      "--- total time for blast 1197: 14.751s.\n",
      "-- processing region: 12 with 240 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_12.fasta\n",
      "--- *blasting region: 12\n",
      "--- total time for blast 12: 14.767s.\n",
      "-- processing region: 120 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_120.fasta\n",
      "--- *blasting region: 120\n",
      "--- total time for blast 120: 18.497s.\n",
      "-- processing region: 1203 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1203.fasta\n",
      "--- *blasting region: 1203\n",
      "--- total time for blast 1203: 9.042s.\n",
      "-- processing region: 1204 with 346 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1204.fasta\n",
      "--- *blasting region: 1204\n",
      "--- total time for blast 1204: 15.420s.\n",
      "-- processing region: 1205 with 321 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1205.fasta\n",
      "--- *blasting region: 1205\n",
      "--- total time for blast 1205: 12.949s.\n",
      "-- processing region: 1206 with 108 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1206.fasta\n",
      "--- *blasting region: 1206\n",
      "--- total time for blast 1206: 10.887s.\n",
      "-- processing region: 1207 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1207.fasta\n",
      "--- *blasting region: 1207\n",
      "--- total time for blast 1207: 14.364s.\n",
      "-- processing region: 1208 with 208 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1208.fasta\n",
      "--- *blasting region: 1208\n",
      "--- total time for blast 1208: 16.206s.\n",
      "-- processing region: 1209 with 169 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1209.fasta\n",
      "--- *blasting region: 1209\n",
      "--- total time for blast 1209: 11.258s.\n",
      "-- processing region: 121 with 295 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_121.fasta\n",
      "--- *blasting region: 121\n",
      "--- total time for blast 121: 11.691s.\n",
      "-- processing region: 1210 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1210.fasta\n",
      "--- *blasting region: 1210\n",
      "--- total time for blast 1210: 13.079s.\n",
      "-- processing region: 1211 with 302 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1211.fasta\n",
      "--- *blasting region: 1211\n",
      "--- total time for blast 1211: 14.701s.\n",
      "-- processing region: 1212 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1212.fasta\n",
      "--- *blasting region: 1212\n",
      "--- total time for blast 1212: 11.243s.\n",
      "-- processing region: 1213 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1213.fasta\n",
      "--- *blasting region: 1213\n",
      "--- total time for blast 1213: 13.427s.\n",
      "-- processing region: 1214 with 317 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1214.fasta\n",
      "--- *blasting region: 1214\n",
      "--- total time for blast 1214: 16.460s.\n",
      "-- processing region: 1215 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1215.fasta\n",
      "--- *blasting region: 1215\n",
      "--- total time for blast 1215: 13.946s.\n",
      "-- processing region: 1216 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1216.fasta\n",
      "--- *blasting region: 1216\n",
      "--- total time for blast 1216: 11.011s.\n",
      "-- processing region: 1217 with 190 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1217.fasta\n",
      "--- *blasting region: 1217\n",
      "--- total time for blast 1217: 9.880s.\n",
      "-- processing region: 1218 with 235 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1218.fasta\n",
      "--- *blasting region: 1218\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1218: 10.745s.\n",
      "-- processing region: 1219 with 315 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1219.fasta\n",
      "--- *blasting region: 1219\n",
      "--- total time for blast 1219: 12.031s.\n",
      "-- processing region: 122 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_122.fasta\n",
      "--- *blasting region: 122\n",
      "--- total time for blast 122: 14.132s.\n",
      "-- processing region: 1220 with 358 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1220.fasta\n",
      "--- *blasting region: 1220\n",
      "--- total time for blast 1220: 12.276s.\n",
      "-- processing region: 1221 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1221.fasta\n",
      "--- *blasting region: 1221\n",
      "--- total time for blast 1221: 15.696s.\n",
      "-- processing region: 1222 with 179 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1222.fasta\n",
      "--- *blasting region: 1222\n",
      "--- total time for blast 1222: 10.211s.\n",
      "-- processing region: 1224 with 231 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1224.fasta\n",
      "--- *blasting region: 1224\n",
      "--- total time for blast 1224: 11.403s.\n",
      "-- processing region: 1225 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1225.fasta\n",
      "--- *blasting region: 1225\n",
      "--- total time for blast 1225: 15.474s.\n",
      "-- processing region: 1226 with 238 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1226.fasta\n",
      "--- *blasting region: 1226\n",
      "--- total time for blast 1226: 12.171s.\n",
      "-- processing region: 1227 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1227.fasta\n",
      "--- *blasting region: 1227\n",
      "--- total time for blast 1227: 11.441s.\n",
      "-- processing region: 1228 with 167 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1228.fasta\n",
      "--- *blasting region: 1228\n",
      "--- total time for blast 1228: 9.892s.\n",
      "-- processing region: 1229 with 235 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1229.fasta\n",
      "--- *blasting region: 1229\n",
      "--- total time for blast 1229: 13.505s.\n",
      "-- processing region: 123 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_123.fasta\n",
      "--- *blasting region: 123\n",
      "--- total time for blast 123: 10.632s.\n",
      "-- processing region: 1230 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1230.fasta\n",
      "--- *blasting region: 1230\n",
      "--- total time for blast 1230: 11.516s.\n",
      "-- processing region: 1231 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1231.fasta\n",
      "--- *blasting region: 1231\n",
      "--- total time for blast 1231: 10.743s.\n",
      "-- processing region: 1232 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1232.fasta\n",
      "--- *blasting region: 1232\n",
      "--- total time for blast 1232: 13.129s.\n",
      "-- processing region: 1233 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1233.fasta\n",
      "--- *blasting region: 1233\n",
      "--- total time for blast 1233: 12.297s.\n",
      "-- processing region: 1234 with 169 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1234.fasta\n",
      "--- *blasting region: 1234\n",
      "--- total time for blast 1234: 12.144s.\n",
      "-- processing region: 1235 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1235.fasta\n",
      "--- *blasting region: 1235\n",
      "--- total time for blast 1235: 11.698s.\n",
      "-- processing region: 1236 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1236.fasta\n",
      "--- *blasting region: 1236\n",
      "--- total time for blast 1236: 12.674s.\n",
      "-- processing region: 1237 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1237.fasta\n",
      "--- *blasting region: 1237\n",
      "--- total time for blast 1237: 11.871s.\n",
      "-- processing region: 1238 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1238.fasta\n",
      "--- *blasting region: 1238\n",
      "--- total time for blast 1238: 12.474s.\n",
      "-- processing region: 1239 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1239.fasta\n",
      "--- *blasting region: 1239\n",
      "--- total time for blast 1239: 13.249s.\n",
      "-- processing region: 124 with 235 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_124.fasta\n",
      "--- *blasting region: 124\n",
      "--- total time for blast 124: 11.916s.\n",
      "-- processing region: 1240 with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1240.fasta\n",
      "--- *blasting region: 1240\n",
      "--- total time for blast 1240: 12.148s.\n",
      "-- processing region: 1241 with 330 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1241.fasta\n",
      "--- *blasting region: 1241\n",
      "--- total time for blast 1241: 12.550s.\n",
      "-- processing region: 1242 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1242.fasta\n",
      "--- *blasting region: 1242\n",
      "--- total time for blast 1242: 10.109s.\n",
      "-- processing region: 1243 with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1243.fasta\n",
      "--- *blasting region: 1243\n",
      "--- total time for blast 1243: 9.930s.\n",
      "-- processing region: 1244 with 260 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1244.fasta\n",
      "--- *blasting region: 1244\n",
      "--- total time for blast 1244: 12.330s.\n",
      "-- processing region: 1245 with 283 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1245.fasta\n",
      "--- *blasting region: 1245\n",
      "--- total time for blast 1245: 11.680s.\n",
      "-- processing region: 1246 with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1246.fasta\n",
      "--- *blasting region: 1246\n",
      "--- total time for blast 1246: 15.806s.\n",
      "-- processing region: 1247 with 232 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1247.fasta\n",
      "--- *blasting region: 1247\n",
      "--- total time for blast 1247: 10.664s.\n",
      "-- processing region: 1248 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1248.fasta\n",
      "--- *blasting region: 1248\n",
      "--- total time for blast 1248: 7.993s.\n",
      "-- processing region: 1249 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1249.fasta\n",
      "--- *blasting region: 1249\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1249: 10.018s.\n",
      "-- processing region: 125 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_125.fasta\n",
      "--- *blasting region: 125\n",
      "--- total time for blast 125: 16.371s.\n",
      "-- processing region: 1250 with 310 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1250.fasta\n",
      "--- *blasting region: 1250\n",
      "--- total time for blast 1250: 11.473s.\n",
      "-- processing region: 1251 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1251.fasta\n",
      "--- *blasting region: 1251\n",
      "--- total time for blast 1251: 14.856s.\n",
      "-- processing region: 1252 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1252.fasta\n",
      "--- *blasting region: 1252\n",
      "--- total time for blast 1252: 12.120s.\n",
      "-- processing region: 1253 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1253.fasta\n",
      "--- *blasting region: 1253\n",
      "--- total time for blast 1253: 10.713s.\n",
      "-- processing region: 1254 with 178 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1254.fasta\n",
      "--- *blasting region: 1254\n",
      "--- total time for blast 1254: 9.227s.\n",
      "-- processing region: 1255 with 303 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1255.fasta\n",
      "--- *blasting region: 1255\n",
      "--- total time for blast 1255: 15.254s.\n",
      "-- processing region: 1256 with 186 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1256.fasta\n",
      "--- *blasting region: 1256\n",
      "--- total time for blast 1256: 27.267s.\n",
      "-- processing region: 1257 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1257.fasta\n",
      "--- *blasting region: 1257\n",
      "--- total time for blast 1257: 9.341s.\n",
      "-- processing region: 1258 with 168 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1258.fasta\n",
      "--- *blasting region: 1258\n",
      "--- total time for blast 1258: 19.999s.\n",
      "-- processing region: 1259 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1259.fasta\n",
      "--- *blasting region: 1259\n",
      "--- total time for blast 1259: 11.474s.\n",
      "-- processing region: 126 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_126.fasta\n",
      "--- *blasting region: 126\n",
      "--- total time for blast 126: 12.117s.\n",
      "-- processing region: 1260 with 364 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1260.fasta\n",
      "--- *blasting region: 1260\n",
      "--- total time for blast 1260: 13.404s.\n",
      "-- processing region: 1261 with 377 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1261.fasta\n",
      "--- *blasting region: 1261\n",
      "--- total time for blast 1261: 13.533s.\n",
      "-- processing region: 1262 with 85 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1262.fasta\n",
      "--- *blasting region: 1262\n",
      "--- total time for blast 1262: 6.235s.\n",
      "-- processing region: 1263 with 189 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1263.fasta\n",
      "--- *blasting region: 1263\n",
      "--- total time for blast 1263: 10.282s.\n",
      "-- processing region: 1264 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1264.fasta\n",
      "--- *blasting region: 1264\n",
      "--- total time for blast 1264: 11.924s.\n",
      "-- processing region: 1265 with 99 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1265.fasta\n",
      "--- *blasting region: 1265\n",
      "--- total time for blast 1265: 17.539s.\n",
      "-- processing region: 1269 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1269.fasta\n",
      "--- *blasting region: 1269\n",
      "--- total time for blast 1269: 11.400s.\n",
      "-- processing region: 127 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_127.fasta\n",
      "--- *blasting region: 127\n",
      "--- total time for blast 127: 10.805s.\n",
      "-- processing region: 1270 with 305 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1270.fasta\n",
      "--- *blasting region: 1270\n",
      "--- total time for blast 1270: 13.274s.\n",
      "-- processing region: 1271 with 312 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1271.fasta\n",
      "--- *blasting region: 1271\n",
      "--- total time for blast 1271: 12.976s.\n",
      "-- processing region: 1272 with 316 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1272.fasta\n",
      "--- *blasting region: 1272\n",
      "--- total time for blast 1272: 13.158s.\n",
      "-- processing region: 1273 with 295 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1273.fasta\n",
      "--- *blasting region: 1273\n",
      "--- total time for blast 1273: 15.750s.\n",
      "-- processing region: 1274 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1274.fasta\n",
      "--- *blasting region: 1274\n",
      "--- total time for blast 1274: 10.421s.\n",
      "-- processing region: 1275 with 349 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1275.fasta\n",
      "--- *blasting region: 1275\n",
      "--- total time for blast 1275: 13.178s.\n",
      "-- processing region: 1276 with 244 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1276.fasta\n",
      "--- *blasting region: 1276\n",
      "--- total time for blast 1276: 11.702s.\n",
      "-- processing region: 1277 with 362 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1277.fasta\n",
      "--- *blasting region: 1277\n",
      "--- total time for blast 1277: 12.610s.\n",
      "-- processing region: 1278 with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1278.fasta\n",
      "--- *blasting region: 1278\n",
      "--- total time for blast 1278: 8.129s.\n",
      "-- processing region: 1279 with 193 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1279.fasta\n",
      "--- *blasting region: 1279\n",
      "--- total time for blast 1279: 9.554s.\n",
      "-- processing region: 128 with 275 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_128.fasta\n",
      "--- *blasting region: 128\n",
      "--- total time for blast 128: 12.051s.\n",
      "-- processing region: 1280 with 296 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1280.fasta\n",
      "--- *blasting region: 1280\n",
      "--- total time for blast 1280: 12.989s.\n",
      "-- processing region: 1281 with 275 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1281.fasta\n",
      "--- *blasting region: 1281\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1281: 12.844s.\n",
      "-- processing region: 1282 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1282.fasta\n",
      "--- *blasting region: 1282\n",
      "--- total time for blast 1282: 9.515s.\n",
      "-- processing region: 1284 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1284.fasta\n",
      "--- *blasting region: 1284\n",
      "--- total time for blast 1284: 9.634s.\n",
      "-- processing region: 1285 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1285.fasta\n",
      "--- *blasting region: 1285\n",
      "--- total time for blast 1285: 10.428s.\n",
      "-- processing region: 1286 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1286.fasta\n",
      "--- *blasting region: 1286\n",
      "--- total time for blast 1286: 11.829s.\n",
      "-- processing region: 1287 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1287.fasta\n",
      "--- *blasting region: 1287\n",
      "--- total time for blast 1287: 39.271s.\n",
      "-- processing region: 1288 with 239 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1288.fasta\n",
      "--- *blasting region: 1288\n",
      "--- total time for blast 1288: 9.917s.\n",
      "-- processing region: 1289 with 367 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1289.fasta\n",
      "--- *blasting region: 1289\n",
      "--- total time for blast 1289: 14.706s.\n",
      "-- processing region: 129 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_129.fasta\n",
      "--- *blasting region: 129\n",
      "--- total time for blast 129: 14.375s.\n",
      "-- processing region: 1290 with 315 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1290.fasta\n",
      "--- *blasting region: 1290\n",
      "--- total time for blast 1290: 14.525s.\n",
      "-- processing region: 1291 with 273 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1291.fasta\n",
      "--- *blasting region: 1291\n",
      "--- total time for blast 1291: 13.074s.\n",
      "-- processing region: 1292 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1292.fasta\n",
      "--- *blasting region: 1292\n",
      "--- total time for blast 1292: 10.240s.\n",
      "-- processing region: 1293 with 312 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1293.fasta\n",
      "--- *blasting region: 1293\n",
      "--- total time for blast 1293: 27.704s.\n",
      "-- processing region: 1294 with 90 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1294.fasta\n",
      "--- *blasting region: 1294\n",
      "--- total time for blast 1294: 14.865s.\n",
      "-- processing region: 1295 with 167 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1295.fasta\n",
      "--- *blasting region: 1295\n",
      "--- total time for blast 1295: 8.113s.\n",
      "-- processing region: 1297 with 312 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1297.fasta\n",
      "--- *blasting region: 1297\n",
      "--- total time for blast 1297: 12.622s.\n",
      "-- processing region: 1298 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1298.fasta\n",
      "--- *blasting region: 1298\n",
      "--- total time for blast 1298: 17.822s.\n",
      "-- processing region: 1299 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1299.fasta\n",
      "--- *blasting region: 1299\n",
      "--- total time for blast 1299: 12.159s.\n",
      "-- processing region: 13 with 341 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_13.fasta\n",
      "--- *blasting region: 13\n",
      "--- total time for blast 13: 16.116s.\n",
      "-- processing region: 130 with 162 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_130.fasta\n",
      "--- *blasting region: 130\n",
      "--- total time for blast 130: 10.654s.\n",
      "-- processing region: 1300 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1300.fasta\n",
      "--- *blasting region: 1300\n",
      "--- total time for blast 1300: 18.198s.\n",
      "-- processing region: 1301 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1301.fasta\n",
      "--- *blasting region: 1301\n",
      "--- total time for blast 1301: 15.732s.\n",
      "-- processing region: 1302 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1302.fasta\n",
      "--- *blasting region: 1302\n",
      "--- total time for blast 1302: 11.158s.\n",
      "-- processing region: 1303 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1303.fasta\n",
      "--- *blasting region: 1303\n",
      "--- total time for blast 1303: 8.637s.\n",
      "-- processing region: 1304 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1304.fasta\n",
      "--- *blasting region: 1304\n",
      "--- total time for blast 1304: 17.242s.\n",
      "-- processing region: 1305 with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1305.fasta\n",
      "--- *blasting region: 1305\n",
      "--- total time for blast 1305: 14.241s.\n",
      "-- processing region: 1306 with 285 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1306.fasta\n",
      "--- *blasting region: 1306\n",
      "--- total time for blast 1306: 14.024s.\n",
      "-- processing region: 1307 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1307.fasta\n",
      "--- *blasting region: 1307\n",
      "--- total time for blast 1307: 17.783s.\n",
      "-- processing region: 1308 with 239 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1308.fasta\n",
      "--- *blasting region: 1308\n",
      "--- total time for blast 1308: 12.364s.\n",
      "-- processing region: 1309 with 325 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1309.fasta\n",
      "--- *blasting region: 1309\n",
      "--- total time for blast 1309: 13.714s.\n",
      "-- processing region: 131 with 190 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_131.fasta\n",
      "--- *blasting region: 131\n",
      "--- total time for blast 131: 11.405s.\n",
      "-- processing region: 1310 with 285 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1310.fasta\n",
      "--- *blasting region: 1310\n",
      "--- total time for blast 1310: 15.144s.\n",
      "-- processing region: 1311 with 211 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1311.fasta\n",
      "--- *blasting region: 1311\n",
      "--- total time for blast 1311: 9.980s.\n",
      "-- processing region: 1312 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1312.fasta\n",
      "--- *blasting region: 1312\n",
      "--- total time for blast 1312: 10.143s.\n",
      "-- processing region: 1313 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1313.fasta\n",
      "--- *blasting region: 1313\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1313: 10.548s.\n",
      "-- processing region: 1314 with 272 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1314.fasta\n",
      "--- *blasting region: 1314\n",
      "--- total time for blast 1314: 14.549s.\n",
      "-- processing region: 1315 with 347 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1315.fasta\n",
      "--- *blasting region: 1315\n",
      "--- total time for blast 1315: 14.307s.\n",
      "-- processing region: 1316 with 301 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1316.fasta\n",
      "--- *blasting region: 1316\n",
      "--- total time for blast 1316: 14.602s.\n",
      "-- processing region: 1317 with 344 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1317.fasta\n",
      "--- *blasting region: 1317\n",
      "--- total time for blast 1317: 13.728s.\n",
      "-- processing region: 1318 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1318.fasta\n",
      "--- *blasting region: 1318\n",
      "--- total time for blast 1318: 12.390s.\n",
      "-- processing region: 1319 with 385 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1319.fasta\n",
      "--- *blasting region: 1319\n",
      "--- total time for blast 1319: 16.746s.\n",
      "-- processing region: 132 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_132.fasta\n",
      "--- *blasting region: 132\n",
      "--- total time for blast 132: 11.447s.\n",
      "-- processing region: 1320 with 286 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1320.fasta\n",
      "--- *blasting region: 1320\n",
      "--- total time for blast 1320: 36.063s.\n",
      "-- processing region: 1321 with 321 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1321.fasta\n",
      "--- *blasting region: 1321\n",
      "--- total time for blast 1321: 13.180s.\n",
      "-- processing region: 1322 with 310 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1322.fasta\n",
      "--- *blasting region: 1322\n",
      "--- total time for blast 1322: 16.257s.\n",
      "-- processing region: 1323 with 306 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1323.fasta\n",
      "--- *blasting region: 1323\n",
      "--- total time for blast 1323: 15.821s.\n",
      "-- processing region: 1324 with 321 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1324.fasta\n",
      "--- *blasting region: 1324\n",
      "--- total time for blast 1324: 13.992s.\n",
      "-- processing region: 1325 with 361 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1325.fasta\n",
      "--- *blasting region: 1325\n",
      "--- total time for blast 1325: 14.520s.\n",
      "-- processing region: 133 with 312 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_133.fasta\n",
      "--- *blasting region: 133\n",
      "--- total time for blast 133: 12.645s.\n",
      "-- processing region: 1330 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1330.fasta\n",
      "--- *blasting region: 1330\n",
      "--- total time for blast 1330: 5.998s.\n",
      "-- processing region: 1331 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1331.fasta\n",
      "--- *blasting region: 1331\n",
      "--- total time for blast 1331: 9.755s.\n",
      "-- processing region: 1332 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1332.fasta\n",
      "--- *blasting region: 1332\n",
      "--- total time for blast 1332: 12.206s.\n",
      "-- processing region: 1333 with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1333.fasta\n",
      "--- *blasting region: 1333\n",
      "--- total time for blast 1333: 9.208s.\n",
      "-- processing region: 1334 with 161 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1334.fasta\n",
      "--- *blasting region: 1334\n",
      "--- total time for blast 1334: 11.904s.\n",
      "-- processing region: 1335 with 326 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1335.fasta\n",
      "--- *blasting region: 1335\n",
      "--- total time for blast 1335: 14.540s.\n",
      "-- processing region: 1336 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1336.fasta\n",
      "--- *blasting region: 1336\n",
      "--- total time for blast 1336: 12.778s.\n",
      "-- processing region: 1337 with 338 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1337.fasta\n",
      "--- *blasting region: 1337\n",
      "--- total time for blast 1337: 16.020s.\n",
      "-- processing region: 1338 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1338.fasta\n",
      "--- *blasting region: 1338\n",
      "--- total time for blast 1338: 9.670s.\n",
      "-- processing region: 1339 with 238 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1339.fasta\n",
      "--- *blasting region: 1339\n",
      "--- total time for blast 1339: 14.918s.\n",
      "-- processing region: 134 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_134.fasta\n",
      "--- *blasting region: 134\n",
      "--- total time for blast 134: 12.502s.\n",
      "-- processing region: 1340 with 344 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1340.fasta\n",
      "--- *blasting region: 1340\n",
      "--- total time for blast 1340: 13.032s.\n",
      "-- processing region: 1341 with 308 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1341.fasta\n",
      "--- *blasting region: 1341\n",
      "--- total time for blast 1341: 14.363s.\n",
      "-- processing region: 1342 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1342.fasta\n",
      "--- *blasting region: 1342\n",
      "--- total time for blast 1342: 11.809s.\n",
      "-- processing region: 1343 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1343.fasta\n",
      "--- *blasting region: 1343\n",
      "--- total time for blast 1343: 12.664s.\n",
      "-- processing region: 1344 with 274 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1344.fasta\n",
      "--- *blasting region: 1344\n",
      "--- total time for blast 1344: 13.752s.\n",
      "-- processing region: 1345 with 312 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1345.fasta\n",
      "--- *blasting region: 1345\n",
      "--- total time for blast 1345: 14.712s.\n",
      "-- processing region: 1346 with 298 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1346.fasta\n",
      "--- *blasting region: 1346\n",
      "--- total time for blast 1346: 12.172s.\n",
      "-- processing region: 1347 with 179 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1347.fasta\n",
      "--- *blasting region: 1347\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1347: 14.652s.\n",
      "-- processing region: 1348 with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1348.fasta\n",
      "--- *blasting region: 1348\n",
      "--- total time for blast 1348: 6.957s.\n",
      "-- processing region: 1349 with 302 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1349.fasta\n",
      "--- *blasting region: 1349\n",
      "--- total time for blast 1349: 14.482s.\n",
      "-- processing region: 1350 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1350.fasta\n",
      "--- *blasting region: 1350\n",
      "--- total time for blast 1350: 12.624s.\n",
      "-- processing region: 1351 with 353 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1351.fasta\n",
      "--- *blasting region: 1351\n",
      "--- total time for blast 1351: 13.398s.\n",
      "-- processing region: 1352 with 243 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1352.fasta\n",
      "--- *blasting region: 1352\n",
      "--- total time for blast 1352: 14.216s.\n",
      "-- processing region: 1353 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1353.fasta\n",
      "--- *blasting region: 1353\n",
      "--- total time for blast 1353: 8.868s.\n",
      "-- processing region: 1354 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1354.fasta\n",
      "--- *blasting region: 1354\n",
      "--- total time for blast 1354: 11.392s.\n",
      "-- processing region: 1355 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1355.fasta\n",
      "--- *blasting region: 1355\n",
      "--- total time for blast 1355: 12.457s.\n",
      "-- processing region: 1356 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1356.fasta\n",
      "--- *blasting region: 1356\n",
      "--- total time for blast 1356: 13.366s.\n",
      "-- processing region: 1357 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1357.fasta\n",
      "--- *blasting region: 1357\n",
      "--- total time for blast 1357: 13.683s.\n",
      "-- processing region: 1358 with 327 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1358.fasta\n",
      "--- *blasting region: 1358\n",
      "--- total time for blast 1358: 14.876s.\n",
      "-- processing region: 1359 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1359.fasta\n",
      "--- *blasting region: 1359\n",
      "--- total time for blast 1359: 14.780s.\n",
      "-- processing region: 1360 with 323 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1360.fasta\n",
      "--- *blasting region: 1360\n",
      "--- total time for blast 1360: 14.382s.\n",
      "-- processing region: 1361 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1361.fasta\n",
      "--- *blasting region: 1361\n",
      "--- total time for blast 1361: 16.465s.\n",
      "-- processing region: 1362 with 379 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1362.fasta\n",
      "--- *blasting region: 1362\n",
      "--- total time for blast 1362: 13.498s.\n",
      "-- processing region: 1363 with 310 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1363.fasta\n",
      "--- *blasting region: 1363\n",
      "--- total time for blast 1363: 12.206s.\n",
      "-- processing region: 137 with 315 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_137.fasta\n",
      "--- *blasting region: 137\n",
      "--- total time for blast 137: 12.279s.\n",
      "-- processing region: 1378 with 217 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1378.fasta\n",
      "--- *blasting region: 1378\n",
      "--- total time for blast 1378: 13.629s.\n",
      "-- processing region: 1379 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1379.fasta\n",
      "--- *blasting region: 1379\n",
      "--- total time for blast 1379: 14.165s.\n",
      "-- processing region: 138 with 285 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_138.fasta\n",
      "--- *blasting region: 138\n",
      "--- total time for blast 138: 11.847s.\n",
      "-- processing region: 1380 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1380.fasta\n",
      "--- *blasting region: 1380\n",
      "--- total time for blast 1380: 9.968s.\n",
      "-- processing region: 1381 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1381.fasta\n",
      "--- *blasting region: 1381\n",
      "--- total time for blast 1381: 9.254s.\n",
      "-- processing region: 1382 with 210 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1382.fasta\n",
      "--- *blasting region: 1382\n",
      "--- total time for blast 1382: 11.453s.\n",
      "-- processing region: 1383 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1383.fasta\n",
      "--- *blasting region: 1383\n",
      "--- total time for blast 1383: 11.222s.\n",
      "-- processing region: 1384 with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1384.fasta\n",
      "--- *blasting region: 1384\n",
      "--- total time for blast 1384: 10.777s.\n",
      "-- processing region: 1385 with 163 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1385.fasta\n",
      "--- *blasting region: 1385\n",
      "--- total time for blast 1385: 10.221s.\n",
      "-- processing region: 1386 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1386.fasta\n",
      "--- *blasting region: 1386\n",
      "--- total time for blast 1386: 11.236s.\n",
      "-- processing region: 1387 with 190 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1387.fasta\n",
      "--- *blasting region: 1387\n",
      "--- total time for blast 1387: 11.314s.\n",
      "-- processing region: 1388 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1388.fasta\n",
      "--- *blasting region: 1388\n",
      "--- total time for blast 1388: 10.838s.\n",
      "-- processing region: 1389 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1389.fasta\n",
      "--- *blasting region: 1389\n",
      "--- total time for blast 1389: 11.660s.\n",
      "-- processing region: 139 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_139.fasta\n",
      "--- *blasting region: 139\n",
      "--- total time for blast 139: 9.331s.\n",
      "-- processing region: 1390 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1390.fasta\n",
      "--- *blasting region: 1390\n",
      "--- total time for blast 1390: 13.513s.\n",
      "-- processing region: 1391 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1391.fasta\n",
      "--- *blasting region: 1391\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1391: 13.706s.\n",
      "-- processing region: 1392 with 244 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1392.fasta\n",
      "--- *blasting region: 1392\n",
      "--- total time for blast 1392: 15.894s.\n",
      "-- processing region: 1393 with 323 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1393.fasta\n",
      "--- *blasting region: 1393\n",
      "--- total time for blast 1393: 14.248s.\n",
      "-- processing region: 1394 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1394.fasta\n",
      "--- *blasting region: 1394\n",
      "--- total time for blast 1394: 13.469s.\n",
      "-- processing region: 1395 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1395.fasta\n",
      "--- *blasting region: 1395\n",
      "--- total time for blast 1395: 31.003s.\n",
      "-- processing region: 1396 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1396.fasta\n",
      "--- *blasting region: 1396\n",
      "--- total time for blast 1396: 17.512s.\n",
      "-- processing region: 1398 with 184 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1398.fasta\n",
      "--- *blasting region: 1398\n",
      "--- total time for blast 1398: 12.702s.\n",
      "-- processing region: 1399 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1399.fasta\n",
      "--- *blasting region: 1399\n",
      "--- total time for blast 1399: 17.298s.\n",
      "-- processing region: 14 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_14.fasta\n",
      "--- *blasting region: 14\n",
      "--- total time for blast 14: 13.346s.\n",
      "-- processing region: 140 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_140.fasta\n",
      "--- *blasting region: 140\n",
      "--- total time for blast 140: 11.949s.\n",
      "-- processing region: 1400 with 308 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1400.fasta\n",
      "--- *blasting region: 1400\n",
      "--- total time for blast 1400: 11.643s.\n",
      "-- processing region: 1401 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1401.fasta\n",
      "--- *blasting region: 1401\n",
      "--- total time for blast 1401: 11.900s.\n",
      "-- processing region: 1402 with 308 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1402.fasta\n",
      "--- *blasting region: 1402\n",
      "--- total time for blast 1402: 16.230s.\n",
      "-- processing region: 1403 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1403.fasta\n",
      "--- *blasting region: 1403\n",
      "--- total time for blast 1403: 11.797s.\n",
      "-- processing region: 1404 with 235 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1404.fasta\n",
      "--- *blasting region: 1404\n",
      "--- total time for blast 1404: 12.326s.\n",
      "-- processing region: 1405 with 336 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1405.fasta\n",
      "--- *blasting region: 1405\n",
      "--- total time for blast 1405: 14.686s.\n",
      "-- processing region: 1406 with 328 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1406.fasta\n",
      "--- *blasting region: 1406\n",
      "--- total time for blast 1406: 13.158s.\n",
      "-- processing region: 1408 with 326 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1408.fasta\n",
      "--- *blasting region: 1408\n",
      "--- total time for blast 1408: 14.842s.\n",
      "-- processing region: 1409 with 369 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1409.fasta\n",
      "--- *blasting region: 1409\n",
      "--- total time for blast 1409: 13.437s.\n",
      "-- processing region: 141 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_141.fasta\n",
      "--- *blasting region: 141\n",
      "--- total time for blast 141: 5.491s.\n",
      "-- processing region: 1427 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1427.fasta\n",
      "--- *blasting region: 1427\n",
      "--- total time for blast 1427: 11.699s.\n",
      "-- processing region: 1429 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1429.fasta\n",
      "--- *blasting region: 1429\n",
      "--- total time for blast 1429: 11.094s.\n",
      "-- processing region: 143 with 186 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_143.fasta\n",
      "--- *blasting region: 143\n",
      "--- total time for blast 143: 8.972s.\n",
      "-- processing region: 1430 with 240 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1430.fasta\n",
      "--- *blasting region: 1430\n",
      "--- total time for blast 1430: 10.633s.\n",
      "-- processing region: 1434 with 295 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1434.fasta\n",
      "--- *blasting region: 1434\n",
      "--- total time for blast 1434: 11.592s.\n",
      "-- processing region: 1435 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1435.fasta\n",
      "--- *blasting region: 1435\n",
      "--- total time for blast 1435: 9.885s.\n",
      "-- processing region: 1436 with 298 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1436.fasta\n",
      "--- *blasting region: 1436\n",
      "--- total time for blast 1436: 14.886s.\n",
      "-- processing region: 1437 with 360 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1437.fasta\n",
      "--- *blasting region: 1437\n",
      "--- total time for blast 1437: 17.083s.\n",
      "-- processing region: 1438 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1438.fasta\n",
      "--- *blasting region: 1438\n",
      "--- total time for blast 1438: 15.175s.\n",
      "-- processing region: 1439 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1439.fasta\n",
      "--- *blasting region: 1439\n",
      "--- total time for blast 1439: 13.445s.\n",
      "-- processing region: 144 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_144.fasta\n",
      "--- *blasting region: 144\n",
      "--- total time for blast 144: 12.146s.\n",
      "-- processing region: 1440 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1440.fasta\n",
      "--- *blasting region: 1440\n",
      "--- total time for blast 1440: 16.306s.\n",
      "-- processing region: 1441 with 184 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1441.fasta\n",
      "--- *blasting region: 1441\n",
      "--- total time for blast 1441: 11.101s.\n",
      "-- processing region: 1442 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1442.fasta\n",
      "--- *blasting region: 1442\n",
      "--- total time for blast 1442: 11.040s.\n",
      "-- processing region: 1443 with 316 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1443.fasta\n",
      "--- *blasting region: 1443\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1443: 15.534s.\n",
      "-- processing region: 1444 with 314 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1444.fasta\n",
      "--- *blasting region: 1444\n",
      "--- total time for blast 1444: 16.360s.\n",
      "-- processing region: 1445 with 312 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1445.fasta\n",
      "--- *blasting region: 1445\n",
      "--- total time for blast 1445: 29.889s.\n",
      "-- processing region: 1446 with 274 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1446.fasta\n",
      "--- *blasting region: 1446\n",
      "--- total time for blast 1446: 11.993s.\n",
      "-- processing region: 1447 with 380 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1447.fasta\n",
      "--- *blasting region: 1447\n",
      "--- total time for blast 1447: 15.641s.\n",
      "-- processing region: 1448 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1448.fasta\n",
      "--- *blasting region: 1448\n",
      "--- total time for blast 1448: 14.766s.\n",
      "-- processing region: 1449 with 341 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1449.fasta\n",
      "--- *blasting region: 1449\n",
      "--- total time for blast 1449: 24.194s.\n",
      "-- processing region: 145 with 178 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_145.fasta\n",
      "--- *blasting region: 145\n",
      "--- total time for blast 145: 13.688s.\n",
      "-- processing region: 1450 with 211 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1450.fasta\n",
      "--- *blasting region: 1450\n",
      "--- total time for blast 1450: 10.210s.\n",
      "-- processing region: 1451 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1451.fasta\n",
      "--- *blasting region: 1451\n",
      "--- total time for blast 1451: 9.658s.\n",
      "-- processing region: 1452 with 113 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1452.fasta\n",
      "--- *blasting region: 1452\n",
      "--- total time for blast 1452: 8.305s.\n",
      "-- processing region: 1453 with 304 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1453.fasta\n",
      "--- *blasting region: 1453\n",
      "--- total time for blast 1453: 13.212s.\n",
      "-- processing region: 1454 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1454.fasta\n",
      "--- *blasting region: 1454\n",
      "--- total time for blast 1454: 13.634s.\n",
      "-- processing region: 1455 with 290 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1455.fasta\n",
      "--- *blasting region: 1455\n",
      "--- total time for blast 1455: 10.437s.\n",
      "-- processing region: 1456 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1456.fasta\n",
      "--- *blasting region: 1456\n",
      "--- total time for blast 1456: 11.059s.\n",
      "-- processing region: 1457 with 281 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1457.fasta\n",
      "--- *blasting region: 1457\n",
      "--- total time for blast 1457: 14.687s.\n",
      "-- processing region: 1458 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1458.fasta\n",
      "--- *blasting region: 1458\n",
      "--- total time for blast 1458: 15.915s.\n",
      "-- processing region: 1459 with 323 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1459.fasta\n",
      "--- *blasting region: 1459\n",
      "--- total time for blast 1459: 14.681s.\n",
      "-- processing region: 146 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_146.fasta\n",
      "--- *blasting region: 146\n",
      "--- total time for blast 146: 19.766s.\n",
      "-- processing region: 1461 with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1461.fasta\n",
      "--- *blasting region: 1461\n",
      "--- total time for blast 1461: 9.771s.\n",
      "-- processing region: 1462 with 238 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1462.fasta\n",
      "--- *blasting region: 1462\n",
      "--- total time for blast 1462: 10.936s.\n",
      "-- processing region: 1463 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1463.fasta\n",
      "--- *blasting region: 1463\n",
      "--- total time for blast 1463: 14.660s.\n",
      "-- processing region: 1464 with 281 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1464.fasta\n",
      "--- *blasting region: 1464\n",
      "--- total time for blast 1464: 16.143s.\n",
      "-- processing region: 1465 with 231 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1465.fasta\n",
      "--- *blasting region: 1465\n",
      "--- total time for blast 1465: 11.375s.\n",
      "-- processing region: 1466 with 286 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1466.fasta\n",
      "--- *blasting region: 1466\n",
      "--- total time for blast 1466: 14.809s.\n",
      "-- processing region: 1467 with 193 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1467.fasta\n",
      "--- *blasting region: 1467\n",
      "--- total time for blast 1467: 9.908s.\n",
      "-- processing region: 1468 with 180 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1468.fasta\n",
      "--- *blasting region: 1468\n",
      "--- total time for blast 1468: 11.055s.\n",
      "-- processing region: 1469 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1469.fasta\n",
      "--- *blasting region: 1469\n",
      "--- total time for blast 1469: 8.551s.\n",
      "-- processing region: 147 with 355 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_147.fasta\n",
      "--- *blasting region: 147\n",
      "--- total time for blast 147: 16.185s.\n",
      "-- processing region: 1470 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1470.fasta\n",
      "--- *blasting region: 1470\n",
      "--- total time for blast 1470: 11.115s.\n",
      "-- processing region: 1471 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1471.fasta\n",
      "--- *blasting region: 1471\n",
      "--- total time for blast 1471: 14.854s.\n",
      "-- processing region: 1472 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1472.fasta\n",
      "--- *blasting region: 1472\n",
      "--- total time for blast 1472: 13.439s.\n",
      "-- processing region: 1473 with 117 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1473.fasta\n",
      "--- *blasting region: 1473\n",
      "--- total time for blast 1473: 12.678s.\n",
      "-- processing region: 1474 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1474.fasta\n",
      "--- *blasting region: 1474\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1474: 10.897s.\n",
      "-- processing region: 1475 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1475.fasta\n",
      "--- *blasting region: 1475\n",
      "--- total time for blast 1475: 11.868s.\n",
      "-- processing region: 1476 with 197 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1476.fasta\n",
      "--- *blasting region: 1476\n",
      "--- total time for blast 1476: 16.659s.\n",
      "-- processing region: 1477 with 341 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1477.fasta\n",
      "--- *blasting region: 1477\n",
      "--- total time for blast 1477: 14.337s.\n",
      "-- processing region: 1478 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1478.fasta\n",
      "--- *blasting region: 1478\n",
      "--- total time for blast 1478: 11.888s.\n",
      "-- processing region: 1479 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1479.fasta\n",
      "--- *blasting region: 1479\n",
      "--- total time for blast 1479: 10.799s.\n",
      "-- processing region: 148 with 193 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_148.fasta\n",
      "--- *blasting region: 148\n",
      "--- total time for blast 148: 10.152s.\n",
      "-- processing region: 1480 with 117 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1480.fasta\n",
      "--- *blasting region: 1480\n",
      "--- total time for blast 1480: 7.322s.\n",
      "-- processing region: 1481 with 198 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1481.fasta\n",
      "--- *blasting region: 1481\n",
      "--- total time for blast 1481: 10.557s.\n",
      "-- processing region: 1482 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1482.fasta\n",
      "--- *blasting region: 1482\n",
      "--- total time for blast 1482: 13.179s.\n",
      "-- processing region: 1485 with 351 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1485.fasta\n",
      "--- *blasting region: 1485\n",
      "--- total time for blast 1485: 14.254s.\n",
      "-- processing region: 1486 with 227 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1486.fasta\n",
      "--- *blasting region: 1486\n",
      "--- total time for blast 1486: 15.702s.\n",
      "-- processing region: 1487 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1487.fasta\n",
      "--- *blasting region: 1487\n",
      "--- total time for blast 1487: 10.845s.\n",
      "-- processing region: 1488 with 311 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1488.fasta\n",
      "--- *blasting region: 1488\n",
      "--- total time for blast 1488: 18.031s.\n",
      "-- processing region: 1489 with 94 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1489.fasta\n",
      "--- *blasting region: 1489\n",
      "--- total time for blast 1489: 12.343s.\n",
      "-- processing region: 149 with 309 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_149.fasta\n",
      "--- *blasting region: 149\n",
      "--- total time for blast 149: 13.960s.\n",
      "-- processing region: 1490 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1490.fasta\n",
      "--- *blasting region: 1490\n",
      "--- total time for blast 1490: 13.887s.\n",
      "-- processing region: 1491 with 239 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1491.fasta\n",
      "--- *blasting region: 1491\n",
      "--- total time for blast 1491: 13.800s.\n",
      "-- processing region: 1492 with 238 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1492.fasta\n",
      "--- *blasting region: 1492\n",
      "--- total time for blast 1492: 12.519s.\n",
      "-- processing region: 1493 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1493.fasta\n",
      "--- *blasting region: 1493\n",
      "--- total time for blast 1493: 8.716s.\n",
      "-- processing region: 1494 with 189 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1494.fasta\n",
      "--- *blasting region: 1494\n",
      "--- total time for blast 1494: 13.211s.\n",
      "-- processing region: 1495 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1495.fasta\n",
      "--- *blasting region: 1495\n",
      "--- total time for blast 1495: 9.968s.\n",
      "-- processing region: 1496 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1496.fasta\n",
      "--- *blasting region: 1496\n",
      "--- total time for blast 1496: 5.689s.\n",
      "-- processing region: 1497 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1497.fasta\n",
      "--- *blasting region: 1497\n",
      "--- total time for blast 1497: 12.981s.\n",
      "-- processing region: 1498 with 146 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1498.fasta\n",
      "--- *blasting region: 1498\n",
      "--- total time for blast 1498: 14.212s.\n",
      "-- processing region: 1499 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1499.fasta\n",
      "--- *blasting region: 1499\n",
      "--- total time for blast 1499: 16.558s.\n",
      "-- processing region: 15 with 343 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_15.fasta\n",
      "--- *blasting region: 15\n",
      "--- total time for blast 15: 14.307s.\n",
      "-- processing region: 150 with 240 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_150.fasta\n",
      "--- *blasting region: 150\n",
      "--- total time for blast 150: 13.016s.\n",
      "-- processing region: 1500 with 369 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1500.fasta\n",
      "--- *blasting region: 1500\n",
      "--- total time for blast 1500: 13.144s.\n",
      "-- processing region: 1501 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1501.fasta\n",
      "--- *blasting region: 1501\n",
      "--- total time for blast 1501: 14.905s.\n",
      "-- processing region: 1502 with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1502.fasta\n",
      "--- *blasting region: 1502\n",
      "--- total time for blast 1502: 10.839s.\n",
      "-- processing region: 1503 with 186 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1503.fasta\n",
      "--- *blasting region: 1503\n",
      "--- total time for blast 1503: 9.554s.\n",
      "-- processing region: 1504 with 285 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1504.fasta\n",
      "--- *blasting region: 1504\n",
      "--- total time for blast 1504: 12.514s.\n",
      "-- processing region: 1505 with 239 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1505.fasta\n",
      "--- *blasting region: 1505\n",
      "--- total time for blast 1505: 14.101s.\n",
      "-- processing region: 1506 with 162 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1506.fasta\n",
      "--- *blasting region: 1506\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1506: 10.992s.\n",
      "-- processing region: 1507 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1507.fasta\n",
      "--- *blasting region: 1507\n",
      "--- total time for blast 1507: 9.786s.\n",
      "-- processing region: 1508 with 189 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1508.fasta\n",
      "--- *blasting region: 1508\n",
      "--- total time for blast 1508: 9.758s.\n",
      "-- processing region: 1509 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1509.fasta\n",
      "--- *blasting region: 1509\n",
      "--- total time for blast 1509: 10.959s.\n",
      "-- processing region: 1510 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1510.fasta\n",
      "--- *blasting region: 1510\n",
      "--- total time for blast 1510: 14.554s.\n",
      "-- processing region: 1511 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1511.fasta\n",
      "--- *blasting region: 1511\n",
      "--- total time for blast 1511: 11.601s.\n",
      "-- processing region: 1512 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1512.fasta\n",
      "--- *blasting region: 1512\n",
      "--- total time for blast 1512: 12.995s.\n",
      "-- processing region: 1513 with 245 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1513.fasta\n",
      "--- *blasting region: 1513\n",
      "--- total time for blast 1513: 10.344s.\n",
      "-- processing region: 1514 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1514.fasta\n",
      "--- *blasting region: 1514\n",
      "--- total time for blast 1514: 11.537s.\n",
      "-- processing region: 1515 with 357 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1515.fasta\n",
      "--- *blasting region: 1515\n",
      "--- total time for blast 1515: 15.482s.\n",
      "-- processing region: 1516 with 103 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1516.fasta\n",
      "--- *blasting region: 1516\n",
      "--- total time for blast 1516: 6.631s.\n",
      "-- processing region: 1517 with 232 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1517.fasta\n",
      "--- *blasting region: 1517\n",
      "--- total time for blast 1517: 9.970s.\n",
      "-- processing region: 1518 with 217 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1518.fasta\n",
      "--- *blasting region: 1518\n",
      "--- total time for blast 1518: 14.441s.\n",
      "-- processing region: 1519 with 239 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1519.fasta\n",
      "--- *blasting region: 1519\n",
      "--- total time for blast 1519: 12.991s.\n",
      "-- processing region: 152 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_152.fasta\n",
      "--- *blasting region: 152\n",
      "--- total time for blast 152: 12.510s.\n",
      "-- processing region: 1520 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1520.fasta\n",
      "--- *blasting region: 1520\n",
      "--- total time for blast 1520: 12.129s.\n",
      "-- processing region: 1521 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1521.fasta\n",
      "--- *blasting region: 1521\n",
      "--- total time for blast 1521: 8.047s.\n",
      "-- processing region: 1525 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1525.fasta\n",
      "--- *blasting region: 1525\n",
      "--- total time for blast 1525: 3.797s.\n",
      "-- processing region: 1526 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1526.fasta\n",
      "--- *blasting region: 1526\n",
      "--- total time for blast 1526: 6.068s.\n",
      "-- processing region: 1527 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1527.fasta\n",
      "--- *blasting region: 1527\n",
      "--- total time for blast 1527: 4.302s.\n",
      "-- processing region: 1529 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1529.fasta\n",
      "--- *blasting region: 1529\n",
      "--- total time for blast 1529: 16.869s.\n",
      "-- processing region: 153 with 307 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_153.fasta\n",
      "--- *blasting region: 153\n",
      "--- total time for blast 153: 15.574s.\n",
      "-- processing region: 1530 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1530.fasta\n",
      "--- *blasting region: 1530\n",
      "--- total time for blast 1530: 15.122s.\n",
      "-- processing region: 1533 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1533.fasta\n",
      "--- *blasting region: 1533\n",
      "--- total time for blast 1533: 11.740s.\n",
      "-- processing region: 1534 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1534.fasta\n",
      "--- *blasting region: 1534\n",
      "--- total time for blast 1534: 15.131s.\n",
      "-- processing region: 1535 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1535.fasta\n",
      "--- *blasting region: 1535\n",
      "--- total time for blast 1535: 16.267s.\n",
      "-- processing region: 1536 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1536.fasta\n",
      "--- *blasting region: 1536\n",
      "--- total time for blast 1536: 15.857s.\n",
      "-- processing region: 1537 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1537.fasta\n",
      "--- *blasting region: 1537\n",
      "--- total time for blast 1537: 13.596s.\n",
      "-- processing region: 1538 with 164 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1538.fasta\n",
      "--- *blasting region: 1538\n",
      "--- total time for blast 1538: 22.959s.\n",
      "-- processing region: 154 with 283 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_154.fasta\n",
      "--- *blasting region: 154\n",
      "--- total time for blast 154: 12.476s.\n",
      "-- processing region: 1541 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1541.fasta\n",
      "--- *blasting region: 1541\n",
      "--- total time for blast 1541: 20.504s.\n",
      "-- processing region: 1542 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1542.fasta\n",
      "--- *blasting region: 1542\n",
      "--- total time for blast 1542: 16.077s.\n",
      "-- processing region: 1548 with 106 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_1548.fasta\n",
      "--- *blasting region: 1548\n",
      "--- total time for blast 1548: 5.289s.\n",
      "-- processing region: 155 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_155.fasta\n",
      "--- *blasting region: 155\n",
      "--- total time for blast 155: 12.166s.\n",
      "-- processing region: 156 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_156.fasta\n",
      "--- *blasting region: 156\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 156: 19.448s.\n",
      "-- processing region: 157 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_157.fasta\n",
      "--- *blasting region: 157\n",
      "--- total time for blast 157: 11.701s.\n",
      "-- processing region: 158 with 227 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_158.fasta\n",
      "--- *blasting region: 158\n",
      "--- total time for blast 158: 10.510s.\n",
      "-- processing region: 159 with 231 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_159.fasta\n",
      "--- *blasting region: 159\n",
      "--- total time for blast 159: 9.822s.\n",
      "-- processing region: 16 with 245 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_16.fasta\n",
      "--- *blasting region: 16\n",
      "--- total time for blast 16: 45.792s.\n",
      "-- processing region: 160 with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_160.fasta\n",
      "--- *blasting region: 160\n",
      "--- total time for blast 160: 33.548s.\n",
      "-- processing region: 161 with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_161.fasta\n",
      "--- *blasting region: 161\n",
      "--- total time for blast 161: 25.479s.\n",
      "-- processing region: 162 with 338 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_162.fasta\n",
      "--- *blasting region: 162\n",
      "--- total time for blast 162: 59.749s.\n",
      "-- processing region: 163 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_163.fasta\n",
      "--- *blasting region: 163\n",
      "--- total time for blast 163: 10.330s.\n",
      "-- processing region: 164 with 303 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_164.fasta\n",
      "--- *blasting region: 164\n",
      "--- total time for blast 164: 18.615s.\n",
      "-- processing region: 165 with 244 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_165.fasta\n",
      "--- *blasting region: 165\n",
      "--- total time for blast 165: 13.350s.\n",
      "-- processing region: 166 with 303 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_166.fasta\n",
      "--- *blasting region: 166\n",
      "--- total time for blast 166: 25.959s.\n",
      "-- processing region: 167 with 217 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_167.fasta\n",
      "--- *blasting region: 167\n",
      "--- total time for blast 167: 11.898s.\n",
      "-- processing region: 168 with 301 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_168.fasta\n",
      "--- *blasting region: 168\n",
      "--- total time for blast 168: 11.052s.\n",
      "-- processing region: 169 with 238 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_169.fasta\n",
      "--- *blasting region: 169\n",
      "--- total time for blast 169: 9.920s.\n",
      "-- processing region: 17 with 203 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_17.fasta\n",
      "--- *blasting region: 17\n",
      "--- total time for blast 17: 11.330s.\n",
      "-- processing region: 170 with 302 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_170.fasta\n",
      "--- *blasting region: 170\n",
      "--- total time for blast 170: 13.553s.\n",
      "-- processing region: 171 with 272 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_171.fasta\n",
      "--- *blasting region: 171\n",
      "--- total time for blast 171: 12.832s.\n",
      "-- processing region: 172 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_172.fasta\n",
      "--- *blasting region: 172\n",
      "--- total time for blast 172: 10.331s.\n",
      "-- processing region: 173 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_173.fasta\n",
      "--- *blasting region: 173\n",
      "--- total time for blast 173: 11.501s.\n",
      "-- processing region: 174 with 198 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_174.fasta\n",
      "--- *blasting region: 174\n",
      "--- total time for blast 174: 13.306s.\n",
      "-- processing region: 175 with 317 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_175.fasta\n",
      "--- *blasting region: 175\n",
      "--- total time for blast 175: 13.703s.\n",
      "-- processing region: 176 with 152 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_176.fasta\n",
      "--- *blasting region: 176\n",
      "--- total time for blast 176: 12.456s.\n",
      "-- processing region: 177 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_177.fasta\n",
      "--- *blasting region: 177\n",
      "--- total time for blast 177: 14.157s.\n",
      "-- processing region: 178 with 280 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_178.fasta\n",
      "--- *blasting region: 178\n",
      "--- total time for blast 178: 10.823s.\n",
      "-- processing region: 179 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_179.fasta\n",
      "--- *blasting region: 179\n",
      "--- total time for blast 179: 15.880s.\n",
      "-- processing region: 18 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_18.fasta\n",
      "--- *blasting region: 18\n",
      "--- total time for blast 18: 13.797s.\n",
      "-- processing region: 180 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_180.fasta\n",
      "--- *blasting region: 180\n",
      "--- total time for blast 180: 14.251s.\n",
      "-- processing region: 181 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_181.fasta\n",
      "--- *blasting region: 181\n",
      "--- total time for blast 181: 12.423s.\n",
      "-- processing region: 182 with 93 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_182.fasta\n",
      "--- *blasting region: 182\n",
      "--- total time for blast 182: 7.115s.\n",
      "-- processing region: 183 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_183.fasta\n",
      "--- *blasting region: 183\n",
      "--- total time for blast 183: 9.456s.\n",
      "-- processing region: 184 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_184.fasta\n",
      "--- *blasting region: 184\n",
      "--- total time for blast 184: 9.630s.\n",
      "-- processing region: 185 with 313 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_185.fasta\n",
      "--- *blasting region: 185\n",
      "--- total time for blast 185: 13.838s.\n",
      "-- processing region: 186 with 301 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_186.fasta\n",
      "--- *blasting region: 186\n",
      "--- total time for blast 186: 12.711s.\n",
      "-- processing region: 187 with 239 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_187.fasta\n",
      "--- *blasting region: 187\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 187: 12.567s.\n",
      "-- processing region: 188 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_188.fasta\n",
      "--- *blasting region: 188\n",
      "--- total time for blast 188: 10.894s.\n",
      "-- processing region: 189 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_189.fasta\n",
      "--- *blasting region: 189\n",
      "--- total time for blast 189: 12.040s.\n",
      "-- processing region: 19 with 391 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_19.fasta\n",
      "--- *blasting region: 19\n",
      "--- total time for blast 19: 15.727s.\n",
      "-- processing region: 190 with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_190.fasta\n",
      "--- *blasting region: 190\n",
      "--- total time for blast 190: 9.572s.\n",
      "-- processing region: 191 with 274 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_191.fasta\n",
      "--- *blasting region: 191\n",
      "--- total time for blast 191: 14.297s.\n",
      "-- processing region: 192 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_192.fasta\n",
      "--- *blasting region: 192\n",
      "--- total time for blast 192: 17.842s.\n",
      "-- processing region: 193 with 357 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_193.fasta\n",
      "--- *blasting region: 193\n",
      "--- total time for blast 193: 13.281s.\n",
      "-- processing region: 194 with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_194.fasta\n",
      "--- *blasting region: 194\n",
      "--- total time for blast 194: 13.235s.\n",
      "-- processing region: 195 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_195.fasta\n",
      "--- *blasting region: 195\n",
      "--- total time for blast 195: 10.247s.\n",
      "-- processing region: 196 with 317 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_196.fasta\n",
      "--- *blasting region: 196\n",
      "--- total time for blast 196: 16.644s.\n",
      "-- processing region: 197 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_197.fasta\n",
      "--- *blasting region: 197\n",
      "--- total time for blast 197: 14.540s.\n",
      "-- processing region: 198 with 379 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_198.fasta\n",
      "--- *blasting region: 198\n",
      "--- total time for blast 198: 16.334s.\n",
      "-- processing region: 199 with 275 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_199.fasta\n",
      "--- *blasting region: 199\n",
      "--- total time for blast 199: 15.933s.\n",
      "-- processing region: 2 with 377 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_2.fasta\n",
      "--- *blasting region: 2\n",
      "--- total time for blast 2: 15.920s.\n",
      "-- processing region: 20 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_20.fasta\n",
      "--- *blasting region: 20\n",
      "--- total time for blast 20: 13.132s.\n",
      "-- processing region: 200 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_200.fasta\n",
      "--- *blasting region: 200\n",
      "--- total time for blast 200: 11.410s.\n",
      "-- processing region: 201 with 361 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_201.fasta\n",
      "--- *blasting region: 201\n",
      "--- total time for blast 201: 13.712s.\n",
      "-- processing region: 202 with 331 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_202.fasta\n",
      "--- *blasting region: 202\n",
      "--- total time for blast 202: 14.555s.\n",
      "-- processing region: 203 with 190 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_203.fasta\n",
      "--- *blasting region: 203\n",
      "--- total time for blast 203: 11.158s.\n",
      "-- processing region: 204 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_204.fasta\n",
      "--- *blasting region: 204\n",
      "--- total time for blast 204: 13.508s.\n",
      "-- processing region: 205 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_205.fasta\n",
      "--- *blasting region: 205\n",
      "--- total time for blast 205: 13.376s.\n",
      "-- processing region: 206 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_206.fasta\n",
      "--- *blasting region: 206\n",
      "--- total time for blast 206: 14.369s.\n",
      "-- processing region: 207 with 324 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_207.fasta\n",
      "--- *blasting region: 207\n",
      "--- total time for blast 207: 13.454s.\n",
      "-- processing region: 208 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_208.fasta\n",
      "--- *blasting region: 208\n",
      "--- total time for blast 208: 9.102s.\n",
      "-- processing region: 209 with 180 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_209.fasta\n",
      "--- *blasting region: 209\n",
      "--- total time for blast 209: 12.885s.\n",
      "-- processing region: 21 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_21.fasta\n",
      "--- *blasting region: 21\n",
      "--- total time for blast 21: 9.824s.\n",
      "-- processing region: 210 with 365 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_210.fasta\n",
      "--- *blasting region: 210\n",
      "--- total time for blast 210: 16.412s.\n",
      "-- processing region: 211 with 349 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_211.fasta\n",
      "--- *blasting region: 211\n",
      "--- total time for blast 211: 14.157s.\n",
      "-- processing region: 212 with 330 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_212.fasta\n",
      "--- *blasting region: 212\n",
      "--- total time for blast 212: 15.146s.\n",
      "-- processing region: 213 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_213.fasta\n",
      "--- *blasting region: 213\n",
      "--- total time for blast 213: 10.851s.\n",
      "-- processing region: 214 with 219 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_214.fasta\n",
      "--- *blasting region: 214\n",
      "--- total time for blast 214: 9.663s.\n",
      "-- processing region: 215 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_215.fasta\n",
      "--- *blasting region: 215\n",
      "--- total time for blast 215: 7.970s.\n",
      "-- processing region: 216 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_216.fasta\n",
      "--- *blasting region: 216\n",
      "--- total time for blast 216: 12.918s.\n",
      "-- processing region: 217 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_217.fasta\n",
      "--- *blasting region: 217\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 217: 12.038s.\n",
      "-- processing region: 218 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_218.fasta\n",
      "--- *blasting region: 218\n",
      "--- total time for blast 218: 11.127s.\n",
      "-- processing region: 219 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_219.fasta\n",
      "--- *blasting region: 219\n",
      "--- total time for blast 219: 15.148s.\n",
      "-- processing region: 22 with 303 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_22.fasta\n",
      "--- *blasting region: 22\n",
      "--- total time for blast 22: 13.972s.\n",
      "-- processing region: 220 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_220.fasta\n",
      "--- *blasting region: 220\n",
      "--- total time for blast 220: 11.972s.\n",
      "-- processing region: 221 with 256 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_221.fasta\n",
      "--- *blasting region: 221\n",
      "--- total time for blast 221: 11.595s.\n",
      "-- processing region: 222 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_222.fasta\n",
      "--- *blasting region: 222\n",
      "--- total time for blast 222: 17.512s.\n",
      "-- processing region: 223 with 318 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_223.fasta\n",
      "--- *blasting region: 223\n",
      "--- total time for blast 223: 12.511s.\n",
      "-- processing region: 224 with 322 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_224.fasta\n",
      "--- *blasting region: 224\n",
      "--- total time for blast 224: 13.085s.\n",
      "-- processing region: 225 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_225.fasta\n",
      "--- *blasting region: 225\n",
      "--- total time for blast 225: 13.086s.\n",
      "-- processing region: 226 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_226.fasta\n",
      "--- *blasting region: 226\n",
      "--- total time for blast 226: 12.767s.\n",
      "-- processing region: 227 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_227.fasta\n",
      "--- *blasting region: 227\n",
      "--- total time for blast 227: 12.434s.\n",
      "-- processing region: 228 with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_228.fasta\n",
      "--- *blasting region: 228\n",
      "--- total time for blast 228: 9.875s.\n",
      "-- processing region: 229 with 197 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_229.fasta\n",
      "--- *blasting region: 229\n",
      "--- total time for blast 229: 9.783s.\n",
      "-- processing region: 23 with 333 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_23.fasta\n",
      "--- *blasting region: 23\n",
      "--- total time for blast 23: 14.293s.\n",
      "-- processing region: 230 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_230.fasta\n",
      "--- *blasting region: 230\n",
      "--- total time for blast 230: 14.166s.\n",
      "-- processing region: 233 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_233.fasta\n",
      "--- *blasting region: 233\n",
      "--- total time for blast 233: 10.123s.\n",
      "-- processing region: 234 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_234.fasta\n",
      "--- *blasting region: 234\n",
      "--- total time for blast 234: 12.370s.\n",
      "-- processing region: 235 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_235.fasta\n",
      "--- *blasting region: 235\n",
      "--- total time for blast 235: 18.873s.\n",
      "-- processing region: 236 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_236.fasta\n",
      "--- *blasting region: 236\n",
      "--- total time for blast 236: 16.140s.\n",
      "-- processing region: 237 with 179 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_237.fasta\n",
      "--- *blasting region: 237\n",
      "--- total time for blast 237: 9.997s.\n",
      "-- processing region: 238 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_238.fasta\n",
      "--- *blasting region: 238\n",
      "--- total time for blast 238: 11.152s.\n",
      "-- processing region: 239 with 231 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_239.fasta\n",
      "--- *blasting region: 239\n",
      "--- total time for blast 239: 11.753s.\n",
      "-- processing region: 24 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_24.fasta\n",
      "--- *blasting region: 24\n",
      "--- total time for blast 24: 10.840s.\n",
      "-- processing region: 240 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_240.fasta\n",
      "--- *blasting region: 240\n",
      "--- total time for blast 240: 10.824s.\n",
      "-- processing region: 241 with 272 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_241.fasta\n",
      "--- *blasting region: 241\n",
      "--- total time for blast 241: 23.881s.\n",
      "-- processing region: 242 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_242.fasta\n",
      "--- *blasting region: 242\n",
      "--- total time for blast 242: 12.499s.\n",
      "-- processing region: 243 with 165 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_243.fasta\n",
      "--- *blasting region: 243\n",
      "--- total time for blast 243: 11.774s.\n",
      "-- processing region: 244 with 333 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_244.fasta\n",
      "--- *blasting region: 244\n",
      "--- total time for blast 244: 17.832s.\n",
      "-- processing region: 245 with 238 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_245.fasta\n",
      "--- *blasting region: 245\n",
      "--- total time for blast 245: 13.297s.\n",
      "-- processing region: 246 with 205 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_246.fasta\n",
      "--- *blasting region: 246\n",
      "--- total time for blast 246: 10.918s.\n",
      "-- processing region: 247 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_247.fasta\n",
      "--- *blasting region: 247\n",
      "--- total time for blast 247: 11.408s.\n",
      "-- processing region: 248 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_248.fasta\n",
      "--- *blasting region: 248\n",
      "--- total time for blast 248: 13.036s.\n",
      "-- processing region: 249 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_249.fasta\n",
      "--- *blasting region: 249\n",
      "--- total time for blast 249: 12.097s.\n",
      "-- processing region: 25 with 238 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_25.fasta\n",
      "--- *blasting region: 25\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 25: 11.189s.\n",
      "-- processing region: 250 with 172 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_250.fasta\n",
      "--- *blasting region: 250\n",
      "--- total time for blast 250: 9.275s.\n",
      "-- processing region: 251 with 357 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_251.fasta\n",
      "--- *blasting region: 251\n",
      "--- total time for blast 251: 17.015s.\n",
      "-- processing region: 252 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_252.fasta\n",
      "--- *blasting region: 252\n",
      "--- total time for blast 252: 14.354s.\n",
      "-- processing region: 253 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_253.fasta\n",
      "--- *blasting region: 253\n",
      "--- total time for blast 253: 12.209s.\n",
      "-- processing region: 254 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_254.fasta\n",
      "--- *blasting region: 254\n",
      "--- total time for blast 254: 11.903s.\n",
      "-- processing region: 255 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_255.fasta\n",
      "--- *blasting region: 255\n",
      "--- total time for blast 255: 11.418s.\n",
      "-- processing region: 256 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_256.fasta\n",
      "--- *blasting region: 256\n",
      "--- total time for blast 256: 9.253s.\n",
      "-- processing region: 257 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_257.fasta\n",
      "--- *blasting region: 257\n",
      "--- total time for blast 257: 15.172s.\n",
      "-- processing region: 258 with 314 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_258.fasta\n",
      "--- *blasting region: 258\n",
      "--- total time for blast 258: 21.809s.\n",
      "-- processing region: 259 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_259.fasta\n",
      "--- *blasting region: 259\n",
      "--- total time for blast 259: 10.521s.\n",
      "-- processing region: 26 with 328 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_26.fasta\n",
      "--- *blasting region: 26\n",
      "--- total time for blast 26: 14.420s.\n",
      "-- processing region: 260 with 162 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_260.fasta\n",
      "--- *blasting region: 260\n",
      "--- total time for blast 260: 17.809s.\n",
      "-- processing region: 261 with 159 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_261.fasta\n",
      "--- *blasting region: 261\n",
      "--- total time for blast 261: 9.904s.\n",
      "-- processing region: 262 with 179 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_262.fasta\n",
      "--- *blasting region: 262\n",
      "--- total time for blast 262: 9.059s.\n",
      "-- processing region: 263 with 323 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_263.fasta\n",
      "--- *blasting region: 263\n",
      "--- total time for blast 263: 14.318s.\n",
      "-- processing region: 264 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_264.fasta\n",
      "--- *blasting region: 264\n",
      "--- total time for blast 264: 12.898s.\n",
      "-- processing region: 265 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_265.fasta\n",
      "--- *blasting region: 265\n",
      "--- total time for blast 265: 12.435s.\n",
      "-- processing region: 266 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_266.fasta\n",
      "--- *blasting region: 266\n",
      "--- total time for blast 266: 11.605s.\n",
      "-- processing region: 267 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_267.fasta\n",
      "--- *blasting region: 267\n",
      "--- total time for blast 267: 12.137s.\n",
      "-- processing region: 268 with 175 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_268.fasta\n",
      "--- *blasting region: 268\n",
      "--- total time for blast 268: 29.732s.\n",
      "-- processing region: 269 with 351 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_269.fasta\n",
      "--- *blasting region: 269\n",
      "--- total time for blast 269: 13.604s.\n",
      "-- processing region: 27 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_27.fasta\n",
      "--- *blasting region: 27\n",
      "--- total time for blast 27: 11.214s.\n",
      "-- processing region: 270 with 313 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_270.fasta\n",
      "--- *blasting region: 270\n",
      "--- total time for blast 270: 13.868s.\n",
      "-- processing region: 271 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_271.fasta\n",
      "--- *blasting region: 271\n",
      "--- total time for blast 271: 13.676s.\n",
      "-- processing region: 272 with 232 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_272.fasta\n",
      "--- *blasting region: 272\n",
      "--- total time for blast 272: 11.608s.\n",
      "-- processing region: 273 with 211 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_273.fasta\n",
      "--- *blasting region: 273\n",
      "--- total time for blast 273: 9.771s.\n",
      "-- processing region: 274 with 312 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_274.fasta\n",
      "--- *blasting region: 274\n",
      "--- total time for blast 274: 12.884s.\n",
      "-- processing region: 275 with 388 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_275.fasta\n",
      "--- *blasting region: 275\n",
      "--- total time for blast 275: 14.256s.\n",
      "-- processing region: 276 with 325 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_276.fasta\n",
      "--- *blasting region: 276\n",
      "--- total time for blast 276: 20.985s.\n",
      "-- processing region: 277 with 313 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_277.fasta\n",
      "--- *blasting region: 277\n",
      "--- total time for blast 277: 13.161s.\n",
      "-- processing region: 278 with 224 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_278.fasta\n",
      "--- *blasting region: 278\n",
      "--- total time for blast 278: 13.990s.\n",
      "-- processing region: 279 with 210 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_279.fasta\n",
      "--- *blasting region: 279\n",
      "--- total time for blast 279: 12.254s.\n",
      "-- processing region: 28 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_28.fasta\n",
      "--- *blasting region: 28\n",
      "--- total time for blast 28: 16.214s.\n",
      "-- processing region: 280 with 309 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_280.fasta\n",
      "--- *blasting region: 280\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 280: 13.028s.\n",
      "-- processing region: 281 with 239 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_281.fasta\n",
      "--- *blasting region: 281\n",
      "--- total time for blast 281: 11.015s.\n",
      "-- processing region: 282 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_282.fasta\n",
      "--- *blasting region: 282\n",
      "--- total time for blast 282: 10.765s.\n",
      "-- processing region: 283 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_283.fasta\n",
      "--- *blasting region: 283\n",
      "--- total time for blast 283: 15.666s.\n",
      "-- processing region: 284 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_284.fasta\n",
      "--- *blasting region: 284\n",
      "--- total time for blast 284: 10.867s.\n",
      "-- processing region: 285 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_285.fasta\n",
      "--- *blasting region: 285\n",
      "--- total time for blast 285: 12.158s.\n",
      "-- processing region: 286 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_286.fasta\n",
      "--- *blasting region: 286\n",
      "--- total time for blast 286: 11.761s.\n",
      "-- processing region: 287 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_287.fasta\n",
      "--- *blasting region: 287\n",
      "--- total time for blast 287: 9.946s.\n",
      "-- processing region: 288 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_288.fasta\n",
      "--- *blasting region: 288\n",
      "--- total time for blast 288: 12.476s.\n",
      "-- processing region: 289 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_289.fasta\n",
      "--- *blasting region: 289\n",
      "--- total time for blast 289: 11.774s.\n",
      "-- processing region: 29 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_29.fasta\n",
      "--- *blasting region: 29\n",
      "--- total time for blast 29: 10.124s.\n",
      "-- processing region: 290 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_290.fasta\n",
      "--- *blasting region: 290\n",
      "--- total time for blast 290: 14.376s.\n",
      "-- processing region: 291 with 195 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_291.fasta\n",
      "--- *blasting region: 291\n",
      "--- total time for blast 291: 9.997s.\n",
      "-- processing region: 292 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_292.fasta\n",
      "--- *blasting region: 292\n",
      "--- total time for blast 292: 10.155s.\n",
      "-- processing region: 293 with 157 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_293.fasta\n",
      "--- *blasting region: 293\n",
      "--- total time for blast 293: 10.894s.\n",
      "-- processing region: 295 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_295.fasta\n",
      "--- *blasting region: 295\n",
      "--- total time for blast 295: 12.244s.\n",
      "-- processing region: 296 with 274 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_296.fasta\n",
      "--- *blasting region: 296\n",
      "--- total time for blast 296: 14.518s.\n",
      "-- processing region: 297 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_297.fasta\n",
      "--- *blasting region: 297\n",
      "--- total time for blast 297: 9.983s.\n",
      "-- processing region: 298 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_298.fasta\n",
      "--- *blasting region: 298\n",
      "--- total time for blast 298: 13.738s.\n",
      "-- processing region: 299 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_299.fasta\n",
      "--- *blasting region: 299\n",
      "--- total time for blast 299: 6.827s.\n",
      "-- processing region: 3 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_3.fasta\n",
      "--- *blasting region: 3\n",
      "--- total time for blast 3: 10.398s.\n",
      "-- processing region: 30 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_30.fasta\n",
      "--- *blasting region: 30\n",
      "--- total time for blast 30: 15.778s.\n",
      "-- processing region: 300 with 151 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_300.fasta\n",
      "--- *blasting region: 300\n",
      "--- total time for blast 300: 10.523s.\n",
      "-- processing region: 302 with 328 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_302.fasta\n",
      "--- *blasting region: 302\n",
      "--- total time for blast 302: 12.480s.\n",
      "-- processing region: 303 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_303.fasta\n",
      "--- *blasting region: 303\n",
      "--- total time for blast 303: 12.937s.\n",
      "-- processing region: 304 with 232 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_304.fasta\n",
      "--- *blasting region: 304\n",
      "--- total time for blast 304: 11.913s.\n",
      "-- processing region: 305 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_305.fasta\n",
      "--- *blasting region: 305\n",
      "--- total time for blast 305: 12.715s.\n",
      "-- processing region: 306 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_306.fasta\n",
      "--- *blasting region: 306\n",
      "--- total time for blast 306: 14.108s.\n",
      "-- processing region: 307 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_307.fasta\n",
      "--- *blasting region: 307\n",
      "--- total time for blast 307: 10.918s.\n",
      "-- processing region: 308 with 203 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_308.fasta\n",
      "--- *blasting region: 308\n",
      "--- total time for blast 308: 9.453s.\n",
      "-- processing region: 309 with 343 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_309.fasta\n",
      "--- *blasting region: 309\n",
      "--- total time for blast 309: 14.685s.\n",
      "-- processing region: 31 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_31.fasta\n",
      "--- *blasting region: 31\n",
      "--- total time for blast 31: 13.218s.\n",
      "-- processing region: 310 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_310.fasta\n",
      "--- *blasting region: 310\n",
      "--- total time for blast 310: 9.003s.\n",
      "-- processing region: 311 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_311.fasta\n",
      "--- *blasting region: 311\n",
      "--- total time for blast 311: 10.563s.\n",
      "-- processing region: 312 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_312.fasta\n",
      "--- *blasting region: 312\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 312: 12.488s.\n",
      "-- processing region: 313 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_313.fasta\n",
      "--- *blasting region: 313\n",
      "--- total time for blast 313: 10.169s.\n",
      "-- processing region: 314 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_314.fasta\n",
      "--- *blasting region: 314\n",
      "--- total time for blast 314: 11.588s.\n",
      "-- processing region: 315 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_315.fasta\n",
      "--- *blasting region: 315\n",
      "--- total time for blast 315: 11.046s.\n",
      "-- processing region: 316 with 183 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_316.fasta\n",
      "--- *blasting region: 316\n",
      "--- total time for blast 316: 9.325s.\n",
      "-- processing region: 317 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_317.fasta\n",
      "--- *blasting region: 317\n",
      "--- total time for blast 317: 10.910s.\n",
      "-- processing region: 318 with 178 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_318.fasta\n",
      "--- *blasting region: 318\n",
      "--- total time for blast 318: 10.978s.\n",
      "-- processing region: 319 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_319.fasta\n",
      "--- *blasting region: 319\n",
      "--- total time for blast 319: 13.454s.\n",
      "-- processing region: 32 with 190 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_32.fasta\n",
      "--- *blasting region: 32\n",
      "--- total time for blast 32: 10.686s.\n",
      "-- processing region: 320 with 155 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_320.fasta\n",
      "--- *blasting region: 320\n",
      "--- total time for blast 320: 9.107s.\n",
      "-- processing region: 321 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_321.fasta\n",
      "--- *blasting region: 321\n",
      "--- total time for blast 321: 13.163s.\n",
      "-- processing region: 322 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_322.fasta\n",
      "--- *blasting region: 322\n",
      "--- total time for blast 322: 13.868s.\n",
      "-- processing region: 323 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_323.fasta\n",
      "--- *blasting region: 323\n",
      "--- total time for blast 323: 11.047s.\n",
      "-- processing region: 324 with 171 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_324.fasta\n",
      "--- *blasting region: 324\n",
      "--- total time for blast 324: 11.554s.\n",
      "-- processing region: 325 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_325.fasta\n",
      "--- *blasting region: 325\n",
      "--- total time for blast 325: 11.378s.\n",
      "-- processing region: 326 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_326.fasta\n",
      "--- *blasting region: 326\n",
      "--- total time for blast 326: 12.819s.\n",
      "-- processing region: 327 with 106 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_327.fasta\n",
      "--- *blasting region: 327\n",
      "--- total time for blast 327: 10.431s.\n",
      "-- processing region: 328 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_328.fasta\n",
      "--- *blasting region: 328\n",
      "--- total time for blast 328: 12.335s.\n",
      "-- processing region: 329 with 295 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_329.fasta\n",
      "--- *blasting region: 329\n",
      "--- total time for blast 329: 13.362s.\n",
      "-- processing region: 33 with 338 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_33.fasta\n",
      "--- *blasting region: 33\n",
      "--- total time for blast 33: 13.778s.\n",
      "-- processing region: 330 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_330.fasta\n",
      "--- *blasting region: 330\n",
      "--- total time for blast 330: 12.383s.\n",
      "-- processing region: 331 with 154 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_331.fasta\n",
      "--- *blasting region: 331\n",
      "--- total time for blast 331: 10.267s.\n",
      "-- processing region: 332 with 377 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_332.fasta\n",
      "--- *blasting region: 332\n",
      "--- total time for blast 332: 15.830s.\n",
      "-- processing region: 333 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_333.fasta\n",
      "--- *blasting region: 333\n",
      "--- total time for blast 333: 9.766s.\n",
      "-- processing region: 334 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_334.fasta\n",
      "--- *blasting region: 334\n",
      "--- total time for blast 334: 31.454s.\n",
      "-- processing region: 335 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_335.fasta\n",
      "--- *blasting region: 335\n",
      "--- total time for blast 335: 13.966s.\n",
      "-- processing region: 336 with 244 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_336.fasta\n",
      "--- *blasting region: 336\n",
      "--- total time for blast 336: 17.287s.\n",
      "-- processing region: 337 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_337.fasta\n",
      "--- *blasting region: 337\n",
      "--- total time for blast 337: 12.834s.\n",
      "-- processing region: 338 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_338.fasta\n",
      "--- *blasting region: 338\n",
      "--- total time for blast 338: 11.898s.\n",
      "-- processing region: 339 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_339.fasta\n",
      "--- *blasting region: 339\n",
      "--- total time for blast 339: 16.749s.\n",
      "-- processing region: 34 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_34.fasta\n",
      "--- *blasting region: 34\n",
      "--- total time for blast 34: 12.135s.\n",
      "-- processing region: 340 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_340.fasta\n",
      "--- *blasting region: 340\n",
      "--- total time for blast 340: 9.924s.\n",
      "-- processing region: 341 with 284 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_341.fasta\n",
      "--- *blasting region: 341\n",
      "--- total time for blast 341: 13.422s.\n",
      "-- processing region: 342 with 157 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_342.fasta\n",
      "--- *blasting region: 342\n",
      "--- total time for blast 342: 11.143s.\n",
      "-- processing region: 343 with 260 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_343.fasta\n",
      "--- *blasting region: 343\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 343: 13.137s.\n",
      "-- processing region: 344 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_344.fasta\n",
      "--- *blasting region: 344\n",
      "--- total time for blast 344: 10.430s.\n",
      "-- processing region: 345 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_345.fasta\n",
      "--- *blasting region: 345\n",
      "--- total time for blast 345: 10.024s.\n",
      "-- processing region: 346 with 273 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_346.fasta\n",
      "--- *blasting region: 346\n",
      "--- total time for blast 346: 10.344s.\n",
      "-- processing region: 347 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_347.fasta\n",
      "--- *blasting region: 347\n",
      "--- total time for blast 347: 10.057s.\n",
      "-- processing region: 348 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_348.fasta\n",
      "--- *blasting region: 348\n",
      "--- total time for blast 348: 13.462s.\n",
      "-- processing region: 349 with 161 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_349.fasta\n",
      "--- *blasting region: 349\n",
      "--- total time for blast 349: 11.120s.\n",
      "-- processing region: 35 with 203 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_35.fasta\n",
      "--- *blasting region: 35\n",
      "--- total time for blast 35: 12.959s.\n",
      "-- processing region: 350 with 388 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_350.fasta\n",
      "--- *blasting region: 350\n",
      "--- total time for blast 350: 14.353s.\n",
      "-- processing region: 351 with 313 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_351.fasta\n",
      "--- *blasting region: 351\n",
      "--- total time for blast 351: 39.006s.\n",
      "-- processing region: 352 with 325 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_352.fasta\n",
      "--- *blasting region: 352\n",
      "--- total time for blast 352: 15.490s.\n",
      "-- processing region: 353 with 158 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_353.fasta\n",
      "--- *blasting region: 353\n",
      "--- total time for blast 353: 14.084s.\n",
      "-- processing region: 354 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_354.fasta\n",
      "--- *blasting region: 354\n",
      "--- total time for blast 354: 14.835s.\n",
      "-- processing region: 355 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_355.fasta\n",
      "--- *blasting region: 355\n",
      "--- total time for blast 355: 11.297s.\n",
      "-- processing region: 356 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_356.fasta\n",
      "--- *blasting region: 356\n",
      "--- total time for blast 356: 14.457s.\n",
      "-- processing region: 357 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_357.fasta\n",
      "--- *blasting region: 357\n",
      "--- total time for blast 357: 9.679s.\n",
      "-- processing region: 358 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_358.fasta\n",
      "--- *blasting region: 358\n",
      "--- total time for blast 358: 16.905s.\n",
      "-- processing region: 359 with 179 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_359.fasta\n",
      "--- *blasting region: 359\n",
      "--- total time for blast 359: 10.638s.\n",
      "-- processing region: 36 with 154 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_36.fasta\n",
      "--- *blasting region: 36\n",
      "--- total time for blast 36: 11.201s.\n",
      "-- processing region: 360 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_360.fasta\n",
      "--- *blasting region: 360\n",
      "--- total time for blast 360: 11.010s.\n",
      "-- processing region: 361 with 175 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_361.fasta\n",
      "--- *blasting region: 361\n",
      "--- total time for blast 361: 8.377s.\n",
      "-- processing region: 362 with 315 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_362.fasta\n",
      "--- *blasting region: 362\n",
      "--- total time for blast 362: 13.761s.\n",
      "-- processing region: 363 with 350 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_363.fasta\n",
      "--- *blasting region: 363\n",
      "--- total time for blast 363: 15.261s.\n",
      "-- processing region: 364 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_364.fasta\n",
      "--- *blasting region: 364\n",
      "--- total time for blast 364: 10.849s.\n",
      "-- processing region: 365 with 284 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_365.fasta\n",
      "--- *blasting region: 365\n",
      "--- total time for blast 365: 12.029s.\n",
      "-- processing region: 366 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_366.fasta\n",
      "--- *blasting region: 366\n",
      "--- total time for blast 366: 10.348s.\n",
      "-- processing region: 367 with 102 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_367.fasta\n",
      "--- *blasting region: 367\n",
      "--- total time for blast 367: 13.678s.\n",
      "-- processing region: 368 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_368.fasta\n",
      "--- *blasting region: 368\n",
      "--- total time for blast 368: 16.275s.\n",
      "-- processing region: 37 with 358 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_37.fasta\n",
      "--- *blasting region: 37\n",
      "--- total time for blast 37: 17.254s.\n",
      "-- processing region: 370 with 158 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_370.fasta\n",
      "--- *blasting region: 370\n",
      "--- total time for blast 370: 8.780s.\n",
      "-- processing region: 371 with 84 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_371.fasta\n",
      "--- *blasting region: 371\n",
      "--- total time for blast 371: 10.386s.\n",
      "-- processing region: 372 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_372.fasta\n",
      "--- *blasting region: 372\n",
      "--- total time for blast 372: 11.365s.\n",
      "-- processing region: 373 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_373.fasta\n",
      "--- *blasting region: 373\n",
      "--- total time for blast 373: 11.875s.\n",
      "-- processing region: 374 with 224 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_374.fasta\n",
      "--- *blasting region: 374\n",
      "--- total time for blast 374: 10.988s.\n",
      "-- processing region: 375 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_375.fasta\n",
      "--- *blasting region: 375\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 375: 10.095s.\n",
      "-- processing region: 376 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_376.fasta\n",
      "--- *blasting region: 376\n",
      "--- total time for blast 376: 10.468s.\n",
      "-- processing region: 377 with 340 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_377.fasta\n",
      "--- *blasting region: 377\n",
      "--- total time for blast 377: 13.108s.\n",
      "-- processing region: 379 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_379.fasta\n",
      "--- *blasting region: 379\n",
      "--- total time for blast 379: 19.842s.\n",
      "-- processing region: 38 with 211 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_38.fasta\n",
      "--- *blasting region: 38\n",
      "--- total time for blast 38: 11.321s.\n",
      "-- processing region: 380 with 224 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_380.fasta\n",
      "--- *blasting region: 380\n",
      "--- total time for blast 380: 12.957s.\n",
      "-- processing region: 381 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_381.fasta\n",
      "--- *blasting region: 381\n",
      "--- total time for blast 381: 13.155s.\n",
      "-- processing region: 382 with 318 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_382.fasta\n",
      "--- *blasting region: 382\n",
      "--- total time for blast 382: 11.807s.\n",
      "-- processing region: 383 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_383.fasta\n",
      "--- *blasting region: 383\n",
      "--- total time for blast 383: 9.687s.\n",
      "-- processing region: 384 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_384.fasta\n",
      "--- *blasting region: 384\n",
      "--- total time for blast 384: 9.540s.\n",
      "-- processing region: 385 with 272 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_385.fasta\n",
      "--- *blasting region: 385\n",
      "--- total time for blast 385: 12.343s.\n",
      "-- processing region: 386 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_386.fasta\n",
      "--- *blasting region: 386\n",
      "--- total time for blast 386: 11.000s.\n",
      "-- processing region: 387 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_387.fasta\n",
      "--- *blasting region: 387\n",
      "--- total time for blast 387: 9.432s.\n",
      "-- processing region: 388 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_388.fasta\n",
      "--- *blasting region: 388\n",
      "--- total time for blast 388: 11.767s.\n",
      "-- processing region: 389 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_389.fasta\n",
      "--- *blasting region: 389\n",
      "--- total time for blast 389: 10.170s.\n",
      "-- processing region: 39 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_39.fasta\n",
      "--- *blasting region: 39\n",
      "--- total time for blast 39: 10.778s.\n",
      "-- processing region: 390 with 164 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_390.fasta\n",
      "--- *blasting region: 390\n",
      "--- total time for blast 390: 12.478s.\n",
      "-- processing region: 391 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_391.fasta\n",
      "--- *blasting region: 391\n",
      "--- total time for blast 391: 16.628s.\n",
      "-- processing region: 392 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_392.fasta\n",
      "--- *blasting region: 392\n",
      "--- total time for blast 392: 13.526s.\n",
      "-- processing region: 393 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_393.fasta\n",
      "--- *blasting region: 393\n",
      "--- total time for blast 393: 9.832s.\n",
      "-- processing region: 394 with 309 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_394.fasta\n",
      "--- *blasting region: 394\n",
      "--- total time for blast 394: 14.972s.\n",
      "-- processing region: 395 with 256 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_395.fasta\n",
      "--- *blasting region: 395\n",
      "--- total time for blast 395: 13.929s.\n",
      "-- processing region: 396 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_396.fasta\n",
      "--- *blasting region: 396\n",
      "--- total time for blast 396: 9.269s.\n",
      "-- processing region: 397 with 245 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_397.fasta\n",
      "--- *blasting region: 397\n",
      "--- total time for blast 397: 12.759s.\n",
      "-- processing region: 398 with 162 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_398.fasta\n",
      "--- *blasting region: 398\n",
      "--- total time for blast 398: 11.527s.\n",
      "-- processing region: 399 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_399.fasta\n",
      "--- *blasting region: 399\n",
      "--- total time for blast 399: 10.865s.\n",
      "-- processing region: 4 with 256 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_4.fasta\n",
      "--- *blasting region: 4\n",
      "--- total time for blast 4: 12.176s.\n",
      "-- processing region: 40 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_40.fasta\n",
      "--- *blasting region: 40\n",
      "--- total time for blast 40: 13.767s.\n",
      "-- processing region: 400 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_400.fasta\n",
      "--- *blasting region: 400\n",
      "--- total time for blast 400: 15.795s.\n",
      "-- processing region: 401 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_401.fasta\n",
      "--- *blasting region: 401\n",
      "--- total time for blast 401: 15.309s.\n",
      "-- processing region: 402 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_402.fasta\n",
      "--- *blasting region: 402\n",
      "--- total time for blast 402: 12.641s.\n",
      "-- processing region: 403 with 377 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_403.fasta\n",
      "--- *blasting region: 403\n",
      "--- total time for blast 403: 15.807s.\n",
      "-- processing region: 404 with 331 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_404.fasta\n",
      "--- *blasting region: 404\n",
      "--- total time for blast 404: 12.943s.\n",
      "-- processing region: 405 with 205 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_405.fasta\n",
      "--- *blasting region: 405\n",
      "--- total time for blast 405: 9.758s.\n",
      "-- processing region: 406 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_406.fasta\n",
      "--- *blasting region: 406\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 406: 12.584s.\n",
      "-- processing region: 407 with 305 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_407.fasta\n",
      "--- *blasting region: 407\n",
      "--- total time for blast 407: 15.877s.\n",
      "-- processing region: 408 with 162 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_408.fasta\n",
      "--- *blasting region: 408\n",
      "--- total time for blast 408: 11.295s.\n",
      "-- processing region: 409 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_409.fasta\n",
      "--- *blasting region: 409\n",
      "--- total time for blast 409: 17.430s.\n",
      "-- processing region: 410 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_410.fasta\n",
      "--- *blasting region: 410\n",
      "--- total time for blast 410: 13.465s.\n",
      "-- processing region: 411 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_411.fasta\n",
      "--- *blasting region: 411\n",
      "--- total time for blast 411: 12.064s.\n",
      "-- processing region: 412 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_412.fasta\n",
      "--- *blasting region: 412\n",
      "--- total time for blast 412: 23.137s.\n",
      "-- processing region: 413 with 339 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_413.fasta\n",
      "--- *blasting region: 413\n",
      "--- total time for blast 413: 15.998s.\n",
      "-- processing region: 414 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_414.fasta\n",
      "--- *blasting region: 414\n",
      "--- total time for blast 414: 11.036s.\n",
      "-- processing region: 415 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_415.fasta\n",
      "--- *blasting region: 415\n",
      "--- total time for blast 415: 16.447s.\n",
      "-- processing region: 416 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_416.fasta\n",
      "--- *blasting region: 416\n",
      "--- total time for blast 416: 13.080s.\n",
      "-- processing region: 417 with 315 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_417.fasta\n",
      "--- *blasting region: 417\n",
      "--- total time for blast 417: 13.375s.\n",
      "-- processing region: 418 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_418.fasta\n",
      "--- *blasting region: 418\n",
      "--- total time for blast 418: 10.350s.\n",
      "-- processing region: 419 with 355 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_419.fasta\n",
      "--- *blasting region: 419\n",
      "--- total time for blast 419: 14.073s.\n",
      "-- processing region: 42 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_42.fasta\n",
      "--- *blasting region: 42\n",
      "--- total time for blast 42: 12.973s.\n",
      "-- processing region: 421 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_421.fasta\n",
      "--- *blasting region: 421\n",
      "--- total time for blast 421: 16.611s.\n",
      "-- processing region: 422 with 296 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_422.fasta\n",
      "--- *blasting region: 422\n",
      "--- total time for blast 422: 14.237s.\n",
      "-- processing region: 423 with 345 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_423.fasta\n",
      "--- *blasting region: 423\n",
      "--- total time for blast 423: 15.950s.\n",
      "-- processing region: 424 with 323 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_424.fasta\n",
      "--- *blasting region: 424\n",
      "--- total time for blast 424: 14.932s.\n",
      "-- processing region: 425 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_425.fasta\n",
      "--- *blasting region: 425\n",
      "--- total time for blast 425: 10.066s.\n",
      "-- processing region: 426 with 350 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_426.fasta\n",
      "--- *blasting region: 426\n",
      "--- total time for blast 426: 14.409s.\n",
      "-- processing region: 427 with 217 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_427.fasta\n",
      "--- *blasting region: 427\n",
      "--- total time for blast 427: 17.649s.\n",
      "-- processing region: 428 with 243 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_428.fasta\n",
      "--- *blasting region: 428\n",
      "--- total time for blast 428: 11.403s.\n",
      "-- processing region: 429 with 193 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_429.fasta\n",
      "--- *blasting region: 429\n",
      "--- total time for blast 429: 13.457s.\n",
      "-- processing region: 43 with 235 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_43.fasta\n",
      "--- *blasting region: 43\n",
      "--- total time for blast 43: 11.197s.\n",
      "-- processing region: 430 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_430.fasta\n",
      "--- *blasting region: 430\n",
      "--- total time for blast 430: 12.100s.\n",
      "-- processing region: 431 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_431.fasta\n",
      "--- *blasting region: 431\n",
      "--- total time for blast 431: 13.339s.\n",
      "-- processing region: 432 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_432.fasta\n",
      "--- *blasting region: 432\n",
      "--- total time for blast 432: 13.921s.\n",
      "-- processing region: 435 with 357 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_435.fasta\n",
      "--- *blasting region: 435\n",
      "--- total time for blast 435: 13.737s.\n",
      "-- processing region: 436 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_436.fasta\n",
      "--- *blasting region: 436\n",
      "--- total time for blast 436: 13.735s.\n",
      "-- processing region: 437 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_437.fasta\n",
      "--- *blasting region: 437\n",
      "--- total time for blast 437: 21.877s.\n",
      "-- processing region: 438 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_438.fasta\n",
      "--- *blasting region: 438\n",
      "--- total time for blast 438: 10.083s.\n",
      "-- processing region: 439 with 310 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_439.fasta\n",
      "--- *blasting region: 439\n",
      "--- total time for blast 439: 12.954s.\n",
      "-- processing region: 44 with 317 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_44.fasta\n",
      "--- *blasting region: 44\n",
      "--- total time for blast 44: 15.515s.\n",
      "-- processing region: 440 with 321 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_440.fasta\n",
      "--- *blasting region: 440\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 440: 14.953s.\n",
      "-- processing region: 441 with 183 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_441.fasta\n",
      "--- *blasting region: 441\n",
      "--- total time for blast 441: 8.192s.\n",
      "-- processing region: 442 with 276 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_442.fasta\n",
      "--- *blasting region: 442\n",
      "--- total time for blast 442: 13.975s.\n",
      "-- processing region: 443 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_443.fasta\n",
      "--- *blasting region: 443\n",
      "--- total time for blast 443: 11.495s.\n",
      "-- processing region: 444 with 307 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_444.fasta\n",
      "--- *blasting region: 444\n",
      "--- total time for blast 444: 14.833s.\n",
      "-- processing region: 446 with 108 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_446.fasta\n",
      "--- *blasting region: 446\n",
      "--- total time for blast 446: 15.845s.\n",
      "-- processing region: 447 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_447.fasta\n",
      "--- *blasting region: 447\n",
      "--- total time for blast 447: 10.070s.\n",
      "-- processing region: 448 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_448.fasta\n",
      "--- *blasting region: 448\n",
      "--- total time for blast 448: 8.614s.\n",
      "-- processing region: 449 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_449.fasta\n",
      "--- *blasting region: 449\n",
      "--- total time for blast 449: 11.251s.\n",
      "-- processing region: 45 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_45.fasta\n",
      "--- *blasting region: 45\n",
      "--- total time for blast 45: 12.551s.\n",
      "-- processing region: 450 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_450.fasta\n",
      "--- *blasting region: 450\n",
      "--- total time for blast 450: 17.181s.\n",
      "-- processing region: 451 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_451.fasta\n",
      "--- *blasting region: 451\n",
      "--- total time for blast 451: 15.062s.\n",
      "-- processing region: 452 with 197 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_452.fasta\n",
      "--- *blasting region: 452\n",
      "--- total time for blast 452: 13.766s.\n",
      "-- processing region: 453 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_453.fasta\n",
      "--- *blasting region: 453\n",
      "--- total time for blast 453: 8.008s.\n",
      "-- processing region: 454 with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_454.fasta\n",
      "--- *blasting region: 454\n",
      "--- total time for blast 454: 10.066s.\n",
      "-- processing region: 455 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_455.fasta\n",
      "--- *blasting region: 455\n",
      "--- total time for blast 455: 10.607s.\n",
      "-- processing region: 456 with 217 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_456.fasta\n",
      "--- *blasting region: 456\n",
      "--- total time for blast 456: 10.500s.\n",
      "-- processing region: 457 with 207 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_457.fasta\n",
      "--- *blasting region: 457\n",
      "--- total time for blast 457: 12.223s.\n",
      "-- processing region: 458 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_458.fasta\n",
      "--- *blasting region: 458\n",
      "--- total time for blast 458: 13.691s.\n",
      "-- processing region: 459 with 180 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_459.fasta\n",
      "--- *blasting region: 459\n",
      "--- total time for blast 459: 9.860s.\n",
      "-- processing region: 46 with 309 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_46.fasta\n",
      "--- *blasting region: 46\n",
      "--- total time for blast 46: 12.670s.\n",
      "-- processing region: 460 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_460.fasta\n",
      "--- *blasting region: 460\n",
      "--- total time for blast 460: 11.073s.\n",
      "-- processing region: 461 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_461.fasta\n",
      "--- *blasting region: 461\n",
      "--- total time for blast 461: 11.090s.\n",
      "-- processing region: 462 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_462.fasta\n",
      "--- *blasting region: 462\n",
      "--- total time for blast 462: 8.423s.\n",
      "-- processing region: 463 with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_463.fasta\n",
      "--- *blasting region: 463\n",
      "--- total time for blast 463: 10.069s.\n",
      "-- processing region: 464 with 211 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_464.fasta\n",
      "--- *blasting region: 464\n",
      "--- total time for blast 464: 26.402s.\n",
      "-- processing region: 465 with 352 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_465.fasta\n",
      "--- *blasting region: 465\n",
      "--- total time for blast 465: 15.640s.\n",
      "-- processing region: 466 with 189 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_466.fasta\n",
      "--- *blasting region: 466\n",
      "--- total time for blast 466: 8.657s.\n",
      "-- processing region: 467 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_467.fasta\n",
      "--- *blasting region: 467\n",
      "--- total time for blast 467: 10.267s.\n",
      "-- processing region: 468 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_468.fasta\n",
      "--- *blasting region: 468\n",
      "--- total time for blast 468: 10.574s.\n",
      "-- processing region: 469 with 260 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_469.fasta\n",
      "--- *blasting region: 469\n",
      "--- total time for blast 469: 13.514s.\n",
      "-- processing region: 47 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_47.fasta\n",
      "--- *blasting region: 47\n",
      "--- total time for blast 47: 13.044s.\n",
      "-- processing region: 470 with 131 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_470.fasta\n",
      "--- *blasting region: 470\n",
      "--- total time for blast 470: 7.604s.\n",
      "-- processing region: 471 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_471.fasta\n",
      "--- *blasting region: 471\n",
      "--- total time for blast 471: 10.999s.\n",
      "-- processing region: 472 with 203 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_472.fasta\n",
      "--- *blasting region: 472\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 472: 17.860s.\n",
      "-- processing region: 473 with 203 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_473.fasta\n",
      "--- *blasting region: 473\n",
      "--- total time for blast 473: 15.078s.\n",
      "-- processing region: 474 with 235 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_474.fasta\n",
      "--- *blasting region: 474\n",
      "--- total time for blast 474: 11.658s.\n",
      "-- processing region: 475 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_475.fasta\n",
      "--- *blasting region: 475\n",
      "--- total time for blast 475: 11.948s.\n",
      "-- processing region: 476 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_476.fasta\n",
      "--- *blasting region: 476\n",
      "--- total time for blast 476: 26.253s.\n",
      "-- processing region: 477 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_477.fasta\n",
      "--- *blasting region: 477\n",
      "--- total time for blast 477: 13.433s.\n",
      "-- processing region: 478 with 189 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_478.fasta\n",
      "--- *blasting region: 478\n",
      "--- total time for blast 478: 10.968s.\n",
      "-- processing region: 479 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_479.fasta\n",
      "--- *blasting region: 479\n",
      "--- total time for blast 479: 13.516s.\n",
      "-- processing region: 480 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_480.fasta\n",
      "--- *blasting region: 480\n",
      "--- total time for blast 480: 12.922s.\n",
      "-- processing region: 481 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_481.fasta\n",
      "--- *blasting region: 481\n",
      "--- total time for blast 481: 13.775s.\n",
      "-- processing region: 482 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_482.fasta\n",
      "--- *blasting region: 482\n",
      "--- total time for blast 482: 9.940s.\n",
      "-- processing region: 483 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_483.fasta\n",
      "--- *blasting region: 483\n",
      "--- total time for blast 483: 11.017s.\n",
      "-- processing region: 484 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_484.fasta\n",
      "--- *blasting region: 484\n",
      "--- total time for blast 484: 15.278s.\n",
      "-- processing region: 485 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_485.fasta\n",
      "--- *blasting region: 485\n",
      "--- total time for blast 485: 13.780s.\n",
      "-- processing region: 486 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_486.fasta\n",
      "--- *blasting region: 486\n",
      "--- total time for blast 486: 18.155s.\n",
      "-- processing region: 487 with 342 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_487.fasta\n",
      "--- *blasting region: 487\n",
      "--- total time for blast 487: 15.130s.\n",
      "-- processing region: 489 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_489.fasta\n",
      "--- *blasting region: 489\n",
      "--- total time for blast 489: 10.269s.\n",
      "-- processing region: 490 with 377 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_490.fasta\n",
      "--- *blasting region: 490\n",
      "--- total time for blast 490: 13.119s.\n",
      "-- processing region: 491 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_491.fasta\n",
      "--- *blasting region: 491\n",
      "--- total time for blast 491: 12.638s.\n",
      "-- processing region: 492 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_492.fasta\n",
      "--- *blasting region: 492\n",
      "--- total time for blast 492: 11.658s.\n",
      "-- processing region: 493 with 239 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_493.fasta\n",
      "--- *blasting region: 493\n",
      "--- total time for blast 493: 12.937s.\n",
      "-- processing region: 494 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_494.fasta\n",
      "--- *blasting region: 494\n",
      "--- total time for blast 494: 15.018s.\n",
      "-- processing region: 495 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_495.fasta\n",
      "--- *blasting region: 495\n",
      "--- total time for blast 495: 12.709s.\n",
      "-- processing region: 496 with 167 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_496.fasta\n",
      "--- *blasting region: 496\n",
      "--- total time for blast 496: 10.769s.\n",
      "-- processing region: 497 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_497.fasta\n",
      "--- *blasting region: 497\n",
      "--- total time for blast 497: 17.308s.\n",
      "-- processing region: 498 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_498.fasta\n",
      "--- *blasting region: 498\n",
      "--- total time for blast 498: 13.599s.\n",
      "-- processing region: 499 with 162 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_499.fasta\n",
      "--- *blasting region: 499\n",
      "--- total time for blast 499: 11.521s.\n",
      "-- processing region: 5 with 303 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_5.fasta\n",
      "--- *blasting region: 5\n",
      "--- total time for blast 5: 12.611s.\n",
      "-- processing region: 500 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_500.fasta\n",
      "--- *blasting region: 500\n",
      "--- total time for blast 500: 9.645s.\n",
      "-- processing region: 501 with 304 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_501.fasta\n",
      "--- *blasting region: 501\n",
      "--- total time for blast 501: 15.856s.\n",
      "-- processing region: 502 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_502.fasta\n",
      "--- *blasting region: 502\n",
      "--- total time for blast 502: 13.842s.\n",
      "-- processing region: 503 with 318 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_503.fasta\n",
      "--- *blasting region: 503\n",
      "--- total time for blast 503: 14.581s.\n",
      "-- processing region: 504 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_504.fasta\n",
      "--- *blasting region: 504\n",
      "--- total time for blast 504: 15.270s.\n",
      "-- processing region: 505 with 308 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_505.fasta\n",
      "--- *blasting region: 505\n",
      "--- total time for blast 505: 13.213s.\n",
      "-- processing region: 506 with 306 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_506.fasta\n",
      "--- *blasting region: 506\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 506: 12.657s.\n",
      "-- processing region: 507 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_507.fasta\n",
      "--- *blasting region: 507\n",
      "--- total time for blast 507: 12.830s.\n",
      "-- processing region: 508 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_508.fasta\n",
      "--- *blasting region: 508\n",
      "--- total time for blast 508: 10.776s.\n",
      "-- processing region: 509 with 272 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_509.fasta\n",
      "--- *blasting region: 509\n",
      "--- total time for blast 509: 15.105s.\n",
      "-- processing region: 510 with 336 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_510.fasta\n",
      "--- *blasting region: 510\n",
      "--- total time for blast 510: 16.276s.\n",
      "-- processing region: 511 with 184 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_511.fasta\n",
      "--- *blasting region: 511\n",
      "--- total time for blast 511: 14.508s.\n",
      "-- processing region: 512 with 197 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_512.fasta\n",
      "--- *blasting region: 512\n",
      "--- total time for blast 512: 12.032s.\n",
      "-- processing region: 516 with 227 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_516.fasta\n",
      "--- *blasting region: 516\n",
      "--- total time for blast 516: 11.493s.\n",
      "-- processing region: 517 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_517.fasta\n",
      "--- *blasting region: 517\n",
      "--- total time for blast 517: 14.049s.\n",
      "-- processing region: 518 with 264 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_518.fasta\n",
      "--- *blasting region: 518\n",
      "--- total time for blast 518: 14.691s.\n",
      "-- processing region: 519 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_519.fasta\n",
      "--- *blasting region: 519\n",
      "--- total time for blast 519: 22.042s.\n",
      "-- processing region: 520 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_520.fasta\n",
      "--- *blasting region: 520\n",
      "--- total time for blast 520: 14.975s.\n",
      "-- processing region: 521 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_521.fasta\n",
      "--- *blasting region: 521\n",
      "--- total time for blast 521: 12.212s.\n",
      "-- processing region: 522 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_522.fasta\n",
      "--- *blasting region: 522\n",
      "--- total time for blast 522: 9.227s.\n",
      "-- processing region: 523 with 152 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_523.fasta\n",
      "--- *blasting region: 523\n",
      "--- total time for blast 523: 7.234s.\n",
      "-- processing region: 524 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_524.fasta\n",
      "--- *blasting region: 524\n",
      "--- total time for blast 524: 9.507s.\n",
      "-- processing region: 525 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_525.fasta\n",
      "--- *blasting region: 525\n",
      "--- total time for blast 525: 11.562s.\n",
      "-- processing region: 526 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_526.fasta\n",
      "--- *blasting region: 526\n",
      "--- total time for blast 526: 14.457s.\n",
      "-- processing region: 527 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_527.fasta\n",
      "--- *blasting region: 527\n",
      "--- total time for blast 527: 13.263s.\n",
      "-- processing region: 528 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_528.fasta\n",
      "--- *blasting region: 528\n",
      "--- total time for blast 528: 12.084s.\n",
      "-- processing region: 529 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_529.fasta\n",
      "--- *blasting region: 529\n",
      "--- total time for blast 529: 10.639s.\n",
      "-- processing region: 530 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_530.fasta\n",
      "--- *blasting region: 530\n",
      "--- total time for blast 530: 12.060s.\n",
      "-- processing region: 531 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_531.fasta\n",
      "--- *blasting region: 531\n",
      "--- total time for blast 531: 8.079s.\n",
      "-- processing region: 532 with 232 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_532.fasta\n",
      "--- *blasting region: 532\n",
      "--- total time for blast 532: 9.151s.\n",
      "-- processing region: 533 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_533.fasta\n",
      "--- *blasting region: 533\n",
      "--- total time for blast 533: 17.107s.\n",
      "-- processing region: 534 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_534.fasta\n",
      "--- *blasting region: 534\n",
      "--- total time for blast 534: 14.766s.\n",
      "-- processing region: 535 with 116 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_535.fasta\n",
      "--- *blasting region: 535\n",
      "--- total time for blast 535: 6.341s.\n",
      "-- processing region: 536 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_536.fasta\n",
      "--- *blasting region: 536\n",
      "--- total time for blast 536: 22.413s.\n",
      "-- processing region: 537 with 231 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_537.fasta\n",
      "--- *blasting region: 537\n",
      "--- total time for blast 537: 12.321s.\n",
      "-- processing region: 538 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_538.fasta\n",
      "--- *blasting region: 538\n",
      "--- total time for blast 538: 10.681s.\n",
      "-- processing region: 539 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_539.fasta\n",
      "--- *blasting region: 539\n",
      "--- total time for blast 539: 10.345s.\n",
      "-- processing region: 540 with 179 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_540.fasta\n",
      "--- *blasting region: 540\n",
      "--- total time for blast 540: 11.038s.\n",
      "-- processing region: 541 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_541.fasta\n",
      "--- *blasting region: 541\n",
      "--- total time for blast 541: 10.626s.\n",
      "-- processing region: 542 with 227 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_542.fasta\n",
      "--- *blasting region: 542\n",
      "--- total time for blast 542: 10.706s.\n",
      "-- processing region: 543 with 302 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_543.fasta\n",
      "--- *blasting region: 543\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 543: 14.275s.\n",
      "-- processing region: 544 with 276 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_544.fasta\n",
      "--- *blasting region: 544\n",
      "--- total time for blast 544: 14.393s.\n",
      "-- processing region: 545 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_545.fasta\n",
      "--- *blasting region: 545\n",
      "--- total time for blast 545: 12.068s.\n",
      "-- processing region: 546 with 305 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_546.fasta\n",
      "--- *blasting region: 546\n",
      "--- total time for blast 546: 14.944s.\n",
      "-- processing region: 548 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_548.fasta\n",
      "--- *blasting region: 548\n",
      "--- total time for blast 548: 11.282s.\n",
      "-- processing region: 549 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_549.fasta\n",
      "--- *blasting region: 549\n",
      "--- total time for blast 549: 14.041s.\n",
      "-- processing region: 550 with 378 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_550.fasta\n",
      "--- *blasting region: 550\n",
      "--- total time for blast 550: 15.511s.\n",
      "-- processing region: 551 with 90 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_551.fasta\n",
      "--- *blasting region: 551\n",
      "--- total time for blast 551: 4.783s.\n",
      "-- processing region: 552 with 264 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_552.fasta\n",
      "--- *blasting region: 552\n",
      "--- total time for blast 552: 11.812s.\n",
      "-- processing region: 554 with 306 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_554.fasta\n",
      "--- *blasting region: 554\n",
      "--- total time for blast 554: 14.054s.\n",
      "-- processing region: 555 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_555.fasta\n",
      "--- *blasting region: 555\n",
      "--- total time for blast 555: 15.118s.\n",
      "-- processing region: 556 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_556.fasta\n",
      "--- *blasting region: 556\n",
      "--- total time for blast 556: 12.872s.\n",
      "-- processing region: 557 with 273 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_557.fasta\n",
      "--- *blasting region: 557\n",
      "--- total time for blast 557: 14.404s.\n",
      "-- processing region: 558 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_558.fasta\n",
      "--- *blasting region: 558\n",
      "--- total time for blast 558: 13.885s.\n",
      "-- processing region: 559 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_559.fasta\n",
      "--- *blasting region: 559\n",
      "--- total time for blast 559: 11.578s.\n",
      "-- processing region: 560 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_560.fasta\n",
      "--- *blasting region: 560\n",
      "--- total time for blast 560: 13.541s.\n",
      "-- processing region: 561 with 240 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_561.fasta\n",
      "--- *blasting region: 561\n",
      "--- total time for blast 561: 15.010s.\n",
      "-- processing region: 562 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_562.fasta\n",
      "--- *blasting region: 562\n",
      "--- total time for blast 562: 14.016s.\n",
      "-- processing region: 563 with 193 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_563.fasta\n",
      "--- *blasting region: 563\n",
      "--- total time for blast 563: 10.402s.\n",
      "-- processing region: 564 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_564.fasta\n",
      "--- *blasting region: 564\n",
      "--- total time for blast 564: 12.531s.\n",
      "-- processing region: 565 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_565.fasta\n",
      "--- *blasting region: 565\n",
      "--- total time for blast 565: 13.729s.\n",
      "-- processing region: 566 with 169 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_566.fasta\n",
      "--- *blasting region: 566\n",
      "--- total time for blast 566: 10.807s.\n",
      "-- processing region: 567 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_567.fasta\n",
      "--- *blasting region: 567\n",
      "--- total time for blast 567: 11.063s.\n",
      "-- processing region: 568 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_568.fasta\n",
      "--- *blasting region: 568\n",
      "--- total time for blast 568: 12.386s.\n",
      "-- processing region: 569 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_569.fasta\n",
      "--- *blasting region: 569\n",
      "--- total time for blast 569: 13.279s.\n",
      "-- processing region: 57 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_57.fasta\n",
      "--- *blasting region: 57\n",
      "--- total time for blast 57: 3.656s.\n",
      "-- processing region: 570 with 104 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_570.fasta\n",
      "--- *blasting region: 570\n",
      "--- total time for blast 570: 11.857s.\n",
      "-- processing region: 571 with 356 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_571.fasta\n",
      "--- *blasting region: 571\n",
      "--- total time for blast 571: 15.380s.\n",
      "-- processing region: 572 with 280 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_572.fasta\n",
      "--- *blasting region: 572\n",
      "--- total time for blast 572: 15.400s.\n",
      "-- processing region: 573 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_573.fasta\n",
      "--- *blasting region: 573\n",
      "--- total time for blast 573: 12.868s.\n",
      "-- processing region: 574 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_574.fasta\n",
      "--- *blasting region: 574\n",
      "--- total time for blast 574: 15.250s.\n",
      "-- processing region: 575 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_575.fasta\n",
      "--- *blasting region: 575\n",
      "--- total time for blast 575: 10.711s.\n",
      "-- processing region: 576 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_576.fasta\n",
      "--- *blasting region: 576\n",
      "--- total time for blast 576: 12.054s.\n",
      "-- processing region: 577 with 207 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_577.fasta\n",
      "--- *blasting region: 577\n",
      "--- total time for blast 577: 9.310s.\n",
      "-- processing region: 578 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_578.fasta\n",
      "--- *blasting region: 578\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 578: 14.839s.\n",
      "-- processing region: 579 with 151 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_579.fasta\n",
      "--- *blasting region: 579\n",
      "--- total time for blast 579: 12.749s.\n",
      "-- processing region: 580 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_580.fasta\n",
      "--- *blasting region: 580\n",
      "--- total time for blast 580: 9.367s.\n",
      "-- processing region: 581 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_581.fasta\n",
      "--- *blasting region: 581\n",
      "--- total time for blast 581: 18.260s.\n",
      "-- processing region: 582 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_582.fasta\n",
      "--- *blasting region: 582\n",
      "--- total time for blast 582: 10.748s.\n",
      "-- processing region: 583 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_583.fasta\n",
      "--- *blasting region: 583\n",
      "--- total time for blast 583: 10.192s.\n",
      "-- processing region: 584 with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_584.fasta\n",
      "--- *blasting region: 584\n",
      "--- total time for blast 584: 12.972s.\n",
      "-- processing region: 585 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_585.fasta\n",
      "--- *blasting region: 585\n",
      "--- total time for blast 585: 11.832s.\n",
      "-- processing region: 586 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_586.fasta\n",
      "--- *blasting region: 586\n",
      "--- total time for blast 586: 10.778s.\n",
      "-- processing region: 587 with 156 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_587.fasta\n",
      "--- *blasting region: 587\n",
      "--- total time for blast 587: 11.576s.\n",
      "-- processing region: 588 with 177 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_588.fasta\n",
      "--- *blasting region: 588\n",
      "--- total time for blast 588: 9.553s.\n",
      "-- processing region: 589 with 275 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_589.fasta\n",
      "--- *blasting region: 589\n",
      "--- total time for blast 589: 11.850s.\n",
      "-- processing region: 590 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_590.fasta\n",
      "--- *blasting region: 590\n",
      "--- total time for blast 590: 10.734s.\n",
      "-- processing region: 591 with 283 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_591.fasta\n",
      "--- *blasting region: 591\n",
      "--- total time for blast 591: 12.575s.\n",
      "-- processing region: 592 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_592.fasta\n",
      "--- *blasting region: 592\n",
      "--- total time for blast 592: 13.820s.\n",
      "-- processing region: 593 with 306 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_593.fasta\n",
      "--- *blasting region: 593\n",
      "--- total time for blast 593: 15.587s.\n",
      "-- processing region: 594 with 155 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_594.fasta\n",
      "--- *blasting region: 594\n",
      "--- total time for blast 594: 9.466s.\n",
      "-- processing region: 595 with 180 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_595.fasta\n",
      "--- *blasting region: 595\n",
      "--- total time for blast 595: 13.380s.\n",
      "-- processing region: 596 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_596.fasta\n",
      "--- *blasting region: 596\n",
      "--- total time for blast 596: 10.311s.\n",
      "-- processing region: 597 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_597.fasta\n",
      "--- *blasting region: 597\n",
      "--- total time for blast 597: 17.226s.\n",
      "-- processing region: 598 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_598.fasta\n",
      "--- *blasting region: 598\n",
      "--- total time for blast 598: 13.253s.\n",
      "-- processing region: 599 with 304 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_599.fasta\n",
      "--- *blasting region: 599\n",
      "--- total time for blast 599: 13.642s.\n",
      "-- processing region: 6 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_6.fasta\n",
      "--- *blasting region: 6\n",
      "--- total time for blast 6: 16.064s.\n",
      "-- processing region: 60 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_60.fasta\n",
      "--- *blasting region: 60\n",
      "--- total time for blast 60: 9.935s.\n",
      "-- processing region: 600 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_600.fasta\n",
      "--- *blasting region: 600\n",
      "--- total time for blast 600: 11.803s.\n",
      "-- processing region: 601 with 299 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_601.fasta\n",
      "--- *blasting region: 601\n",
      "--- total time for blast 601: 13.163s.\n",
      "-- processing region: 602 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_602.fasta\n",
      "--- *blasting region: 602\n",
      "--- total time for blast 602: 12.984s.\n",
      "-- processing region: 603 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_603.fasta\n",
      "--- *blasting region: 603\n",
      "--- total time for blast 603: 12.251s.\n",
      "-- processing region: 605 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_605.fasta\n",
      "--- *blasting region: 605\n",
      "--- total time for blast 605: 10.720s.\n",
      "-- processing region: 606 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_606.fasta\n",
      "--- *blasting region: 606\n",
      "--- total time for blast 606: 16.172s.\n",
      "-- processing region: 607 with 281 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_607.fasta\n",
      "--- *blasting region: 607\n",
      "--- total time for blast 607: 14.477s.\n",
      "-- processing region: 608 with 355 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_608.fasta\n",
      "--- *blasting region: 608\n",
      "--- total time for blast 608: 15.547s.\n",
      "-- processing region: 609 with 308 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_609.fasta\n",
      "--- *blasting region: 609\n",
      "--- total time for blast 609: 12.732s.\n",
      "-- processing region: 611 with 302 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_611.fasta\n",
      "--- *blasting region: 611\n",
      "--- total time for blast 611: 12.876s.\n",
      "-- processing region: 612 with 311 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_612.fasta\n",
      "--- *blasting region: 612\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 612: 12.898s.\n",
      "-- processing region: 613 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_613.fasta\n",
      "--- *blasting region: 613\n",
      "--- total time for blast 613: 10.522s.\n",
      "-- processing region: 614 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_614.fasta\n",
      "--- *blasting region: 614\n",
      "--- total time for blast 614: 12.343s.\n",
      "-- processing region: 615 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_615.fasta\n",
      "--- *blasting region: 615\n",
      "--- total time for blast 615: 12.191s.\n",
      "-- processing region: 616 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_616.fasta\n",
      "--- *blasting region: 616\n",
      "--- total time for blast 616: 11.903s.\n",
      "-- processing region: 617 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_617.fasta\n",
      "--- *blasting region: 617\n",
      "--- total time for blast 617: 14.994s.\n",
      "-- processing region: 618 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_618.fasta\n",
      "--- *blasting region: 618\n",
      "--- total time for blast 618: 9.375s.\n",
      "-- processing region: 619 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_619.fasta\n",
      "--- *blasting region: 619\n",
      "--- total time for blast 619: 11.969s.\n",
      "-- processing region: 62 with 329 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_62.fasta\n",
      "--- *blasting region: 62\n",
      "--- total time for blast 62: 11.880s.\n",
      "-- processing region: 620 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_620.fasta\n",
      "--- *blasting region: 620\n",
      "--- total time for blast 620: 10.609s.\n",
      "-- processing region: 621 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_621.fasta\n",
      "--- *blasting region: 621\n",
      "--- total time for blast 621: 11.178s.\n",
      "-- processing region: 622 with 247 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_622.fasta\n",
      "--- *blasting region: 622\n",
      "--- total time for blast 622: 14.452s.\n",
      "-- processing region: 623 with 185 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_623.fasta\n",
      "--- *blasting region: 623\n",
      "--- total time for blast 623: 11.627s.\n",
      "-- processing region: 624 with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_624.fasta\n",
      "--- *blasting region: 624\n",
      "--- total time for blast 624: 6.436s.\n",
      "-- processing region: 625 with 168 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_625.fasta\n",
      "--- *blasting region: 625\n",
      "--- total time for blast 625: 8.541s.\n",
      "-- processing region: 626 with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_626.fasta\n",
      "--- *blasting region: 626\n",
      "--- total time for blast 626: 16.805s.\n",
      "-- processing region: 627 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_627.fasta\n",
      "--- *blasting region: 627\n",
      "--- total time for blast 627: 17.079s.\n",
      "-- processing region: 628 with 273 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_628.fasta\n",
      "--- *blasting region: 628\n",
      "--- total time for blast 628: 11.117s.\n",
      "-- processing region: 629 with 360 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_629.fasta\n",
      "--- *blasting region: 629\n",
      "--- total time for blast 629: 15.127s.\n",
      "-- processing region: 63 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_63.fasta\n",
      "--- *blasting region: 63\n",
      "--- total time for blast 63: 12.944s.\n",
      "-- processing region: 64 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_64.fasta\n",
      "--- *blasting region: 64\n",
      "--- total time for blast 64: 10.229s.\n",
      "-- processing region: 645 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_645.fasta\n",
      "--- *blasting region: 645\n",
      "--- total time for blast 645: 11.547s.\n",
      "-- processing region: 646 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_646.fasta\n",
      "--- *blasting region: 646\n",
      "--- total time for blast 646: 10.984s.\n",
      "-- processing region: 647 with 219 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_647.fasta\n",
      "--- *blasting region: 647\n",
      "--- total time for blast 647: 11.878s.\n",
      "-- processing region: 648 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_648.fasta\n",
      "--- *blasting region: 648\n",
      "--- total time for blast 648: 10.446s.\n",
      "-- processing region: 649 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_649.fasta\n",
      "--- *blasting region: 649\n",
      "--- total time for blast 649: 12.724s.\n",
      "-- processing region: 65 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_65.fasta\n",
      "--- *blasting region: 65\n",
      "--- total time for blast 65: 9.893s.\n",
      "-- processing region: 650 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_650.fasta\n",
      "--- *blasting region: 650\n",
      "--- total time for blast 650: 12.195s.\n",
      "-- processing region: 651 with 146 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_651.fasta\n",
      "--- *blasting region: 651\n",
      "--- total time for blast 651: 9.341s.\n",
      "-- processing region: 652 with 306 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_652.fasta\n",
      "--- *blasting region: 652\n",
      "--- total time for blast 652: 13.448s.\n",
      "-- processing region: 653 with 373 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_653.fasta\n",
      "--- *blasting region: 653\n",
      "--- total time for blast 653: 15.429s.\n",
      "-- processing region: 654 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_654.fasta\n",
      "--- *blasting region: 654\n",
      "--- total time for blast 654: 14.940s.\n",
      "-- processing region: 655 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_655.fasta\n",
      "--- *blasting region: 655\n",
      "--- total time for blast 655: 14.634s.\n",
      "-- processing region: 656 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_656.fasta\n",
      "--- *blasting region: 656\n",
      "--- total time for blast 656: 13.609s.\n",
      "-- processing region: 657 with 186 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_657.fasta\n",
      "--- *blasting region: 657\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 657: 9.316s.\n",
      "-- processing region: 658 with 311 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_658.fasta\n",
      "--- *blasting region: 658\n",
      "--- total time for blast 658: 14.533s.\n",
      "-- processing region: 659 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_659.fasta\n",
      "--- *blasting region: 659\n",
      "--- total time for blast 659: 4.842s.\n",
      "-- processing region: 66 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_66.fasta\n",
      "--- *blasting region: 66\n",
      "--- total time for blast 66: 8.271s.\n",
      "-- processing region: 660 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_660.fasta\n",
      "--- *blasting region: 660\n",
      "--- total time for blast 660: 12.512s.\n",
      "-- processing region: 662 with 184 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_662.fasta\n",
      "--- *blasting region: 662\n",
      "--- total time for blast 662: 14.580s.\n",
      "-- processing region: 663 with 205 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_663.fasta\n",
      "--- *blasting region: 663\n",
      "--- total time for blast 663: 13.173s.\n",
      "-- processing region: 664 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_664.fasta\n",
      "--- *blasting region: 664\n",
      "--- total time for blast 664: 10.950s.\n",
      "-- processing region: 665 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_665.fasta\n",
      "--- *blasting region: 665\n",
      "--- total time for blast 665: 11.553s.\n",
      "-- processing region: 666 with 273 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_666.fasta\n",
      "--- *blasting region: 666\n",
      "--- total time for blast 666: 14.209s.\n",
      "-- processing region: 667 with 172 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_667.fasta\n",
      "--- *blasting region: 667\n",
      "--- total time for blast 667: 14.667s.\n",
      "-- processing region: 668 with 328 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_668.fasta\n",
      "--- *blasting region: 668\n",
      "--- total time for blast 668: 13.214s.\n",
      "-- processing region: 669 with 312 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_669.fasta\n",
      "--- *blasting region: 669\n",
      "--- total time for blast 669: 14.903s.\n",
      "-- processing region: 67 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_67.fasta\n",
      "--- *blasting region: 67\n",
      "--- total time for blast 67: 12.412s.\n",
      "-- processing region: 670 with 311 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_670.fasta\n",
      "--- *blasting region: 670\n",
      "--- total time for blast 670: 13.621s.\n",
      "-- processing region: 671 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_671.fasta\n",
      "--- *blasting region: 671\n",
      "--- total time for blast 671: 10.456s.\n",
      "-- processing region: 672 with 171 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_672.fasta\n",
      "--- *blasting region: 672\n",
      "--- total time for blast 672: 8.815s.\n",
      "-- processing region: 673 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_673.fasta\n",
      "--- *blasting region: 673\n",
      "--- total time for blast 673: 8.838s.\n",
      "-- processing region: 674 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_674.fasta\n",
      "--- *blasting region: 674\n",
      "--- total time for blast 674: 9.728s.\n",
      "-- processing region: 675 with 327 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_675.fasta\n",
      "--- *blasting region: 675\n",
      "--- total time for blast 675: 14.706s.\n",
      "-- processing region: 676 with 329 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_676.fasta\n",
      "--- *blasting region: 676\n",
      "--- total time for blast 676: 15.681s.\n",
      "-- processing region: 677 with 308 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_677.fasta\n",
      "--- *blasting region: 677\n",
      "--- total time for blast 677: 15.383s.\n",
      "-- processing region: 678 with 347 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_678.fasta\n",
      "--- *blasting region: 678\n",
      "--- total time for blast 678: 13.799s.\n",
      "-- processing region: 679 with 320 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_679.fasta\n",
      "--- *blasting region: 679\n",
      "--- total time for blast 679: 19.393s.\n",
      "-- processing region: 68 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_68.fasta\n",
      "--- *blasting region: 68\n",
      "--- total time for blast 68: 10.746s.\n",
      "-- processing region: 680 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_680.fasta\n",
      "--- *blasting region: 680\n",
      "--- total time for blast 680: 13.644s.\n",
      "-- processing region: 681 with 306 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_681.fasta\n",
      "--- *blasting region: 681\n",
      "--- total time for blast 681: 12.722s.\n",
      "-- processing region: 682 with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_682.fasta\n",
      "--- *blasting region: 682\n",
      "--- total time for blast 682: 8.988s.\n",
      "-- processing region: 683 with 285 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_683.fasta\n",
      "--- *blasting region: 683\n",
      "--- total time for blast 683: 12.083s.\n",
      "-- processing region: 684 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_684.fasta\n",
      "--- *blasting region: 684\n",
      "--- total time for blast 684: 14.860s.\n",
      "-- processing region: 685 with 328 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_685.fasta\n",
      "--- *blasting region: 685\n",
      "--- total time for blast 685: 14.861s.\n",
      "-- processing region: 686 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_686.fasta\n",
      "--- *blasting region: 686\n",
      "--- total time for blast 686: 13.957s.\n",
      "-- processing region: 687 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_687.fasta\n",
      "--- *blasting region: 687\n",
      "--- total time for blast 687: 4.568s.\n",
      "-- processing region: 688 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_688.fasta\n",
      "--- *blasting region: 688\n",
      "--- total time for blast 688: 11.549s.\n",
      "-- processing region: 689 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_689.fasta\n",
      "--- *blasting region: 689\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 689: 10.560s.\n",
      "-- processing region: 69 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_69.fasta\n",
      "--- *blasting region: 69\n",
      "--- total time for blast 69: 7.935s.\n",
      "-- processing region: 690 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_690.fasta\n",
      "--- *blasting region: 690\n",
      "--- total time for blast 690: 11.958s.\n",
      "-- processing region: 691 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_691.fasta\n",
      "--- *blasting region: 691\n",
      "--- total time for blast 691: 12.350s.\n",
      "-- processing region: 692 with 189 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_692.fasta\n",
      "--- *blasting region: 692\n",
      "--- total time for blast 692: 16.364s.\n",
      "-- processing region: 693 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_693.fasta\n",
      "--- *blasting region: 693\n",
      "--- total time for blast 693: 11.514s.\n",
      "-- processing region: 694 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_694.fasta\n",
      "--- *blasting region: 694\n",
      "--- total time for blast 694: 11.929s.\n",
      "-- processing region: 695 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_695.fasta\n",
      "--- *blasting region: 695\n",
      "--- total time for blast 695: 12.084s.\n",
      "-- processing region: 696 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_696.fasta\n",
      "--- *blasting region: 696\n",
      "--- total time for blast 696: 9.924s.\n",
      "-- processing region: 697 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_697.fasta\n",
      "--- *blasting region: 697\n",
      "--- total time for blast 697: 11.156s.\n",
      "-- processing region: 698 with 227 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_698.fasta\n",
      "--- *blasting region: 698\n",
      "--- total time for blast 698: 10.550s.\n",
      "-- processing region: 699 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_699.fasta\n",
      "--- *blasting region: 699\n",
      "--- total time for blast 699: 9.164s.\n",
      "-- processing region: 7 with 341 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_7.fasta\n",
      "--- *blasting region: 7\n",
      "--- total time for blast 7: 15.320s.\n",
      "-- processing region: 70 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_70.fasta\n",
      "--- *blasting region: 70\n",
      "--- total time for blast 70: 8.008s.\n",
      "-- processing region: 701 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_701.fasta\n",
      "--- *blasting region: 701\n",
      "--- total time for blast 701: 14.804s.\n",
      "-- processing region: 702 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_702.fasta\n",
      "--- *blasting region: 702\n",
      "--- total time for blast 702: 15.368s.\n",
      "-- processing region: 703 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_703.fasta\n",
      "--- *blasting region: 703\n",
      "--- total time for blast 703: 10.510s.\n",
      "-- processing region: 704 with 284 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_704.fasta\n",
      "--- *blasting region: 704\n",
      "--- total time for blast 704: 13.041s.\n",
      "-- processing region: 705 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_705.fasta\n",
      "--- *blasting region: 705\n",
      "--- total time for blast 705: 13.595s.\n",
      "-- processing region: 706 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_706.fasta\n",
      "--- *blasting region: 706\n",
      "--- total time for blast 706: 14.269s.\n",
      "-- processing region: 707 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_707.fasta\n",
      "--- *blasting region: 707\n",
      "--- total time for blast 707: 12.327s.\n",
      "-- processing region: 708 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_708.fasta\n",
      "--- *blasting region: 708\n",
      "--- total time for blast 708: 9.940s.\n",
      "-- processing region: 709 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_709.fasta\n",
      "--- *blasting region: 709\n",
      "--- total time for blast 709: 15.215s.\n",
      "-- processing region: 71 with 137 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_71.fasta\n",
      "--- *blasting region: 71\n",
      "--- total time for blast 71: 9.721s.\n",
      "-- processing region: 710 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_710.fasta\n",
      "--- *blasting region: 710\n",
      "--- total time for blast 710: 16.077s.\n",
      "-- processing region: 711 with 290 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_711.fasta\n",
      "--- *blasting region: 711\n",
      "--- total time for blast 711: 12.930s.\n",
      "-- processing region: 712 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_712.fasta\n",
      "--- *blasting region: 712\n",
      "--- total time for blast 712: 11.016s.\n",
      "-- processing region: 713 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_713.fasta\n",
      "--- *blasting region: 713\n",
      "--- total time for blast 713: 12.102s.\n",
      "-- processing region: 714 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_714.fasta\n",
      "--- *blasting region: 714\n",
      "--- total time for blast 714: 15.703s.\n",
      "-- processing region: 715 with 353 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_715.fasta\n",
      "--- *blasting region: 715\n",
      "--- total time for blast 715: 15.199s.\n",
      "-- processing region: 716 with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_716.fasta\n",
      "--- *blasting region: 716\n",
      "--- total time for blast 716: 8.117s.\n",
      "-- processing region: 717 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_717.fasta\n",
      "--- *blasting region: 717\n",
      "--- total time for blast 717: 12.539s.\n",
      "-- processing region: 718 with 391 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_718.fasta\n",
      "--- *blasting region: 718\n",
      "--- total time for blast 718: 16.410s.\n",
      "-- processing region: 719 with 376 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_719.fasta\n",
      "--- *blasting region: 719\n",
      "--- total time for blast 719: 17.064s.\n",
      "-- processing region: 72 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_72.fasta\n",
      "--- *blasting region: 72\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 72: 16.159s.\n",
      "-- processing region: 720 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_720.fasta\n",
      "--- *blasting region: 720\n",
      "--- total time for blast 720: 10.393s.\n",
      "-- processing region: 721 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_721.fasta\n",
      "--- *blasting region: 721\n",
      "--- total time for blast 721: 12.045s.\n",
      "-- processing region: 722 with 309 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_722.fasta\n",
      "--- *blasting region: 722\n",
      "--- total time for blast 722: 13.129s.\n",
      "-- processing region: 723 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_723.fasta\n",
      "--- *blasting region: 723\n",
      "--- total time for blast 723: 6.230s.\n",
      "-- processing region: 724 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_724.fasta\n",
      "--- *blasting region: 724\n",
      "--- total time for blast 724: 13.606s.\n",
      "-- processing region: 725 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_725.fasta\n",
      "--- *blasting region: 725\n",
      "--- total time for blast 725: 12.718s.\n",
      "-- processing region: 726 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_726.fasta\n",
      "--- *blasting region: 726\n",
      "--- total time for blast 726: 9.345s.\n",
      "-- processing region: 727 with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_727.fasta\n",
      "--- *blasting region: 727\n",
      "--- total time for blast 727: 7.144s.\n",
      "-- processing region: 728 with 365 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_728.fasta\n",
      "--- *blasting region: 728\n",
      "--- total time for blast 728: 16.362s.\n",
      "-- processing region: 729 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_729.fasta\n",
      "--- *blasting region: 729\n",
      "--- total time for blast 729: 10.260s.\n",
      "-- processing region: 73 with 198 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_73.fasta\n",
      "--- *blasting region: 73\n",
      "--- total time for blast 73: 11.165s.\n",
      "-- processing region: 730 with 379 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_730.fasta\n",
      "--- *blasting region: 730\n",
      "--- total time for blast 730: 14.735s.\n",
      "-- processing region: 731 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_731.fasta\n",
      "--- *blasting region: 731\n",
      "--- total time for blast 731: 10.277s.\n",
      "-- processing region: 732 with 296 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_732.fasta\n",
      "--- *blasting region: 732\n",
      "--- total time for blast 732: 13.473s.\n",
      "-- processing region: 733 with 274 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_733.fasta\n",
      "--- *blasting region: 733\n",
      "--- total time for blast 733: 12.363s.\n",
      "-- processing region: 734 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_734.fasta\n",
      "--- *blasting region: 734\n",
      "--- total time for blast 734: 15.441s.\n",
      "-- processing region: 735 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_735.fasta\n",
      "--- *blasting region: 735\n",
      "--- total time for blast 735: 10.262s.\n",
      "-- processing region: 736 with 362 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_736.fasta\n",
      "--- *blasting region: 736\n",
      "--- total time for blast 736: 14.132s.\n",
      "-- processing region: 737 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_737.fasta\n",
      "--- *blasting region: 737\n",
      "--- total time for blast 737: 12.335s.\n",
      "-- processing region: 738 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_738.fasta\n",
      "--- *blasting region: 738\n",
      "--- total time for blast 738: 10.428s.\n",
      "-- processing region: 739 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_739.fasta\n",
      "--- *blasting region: 739\n",
      "--- total time for blast 739: 12.899s.\n",
      "-- processing region: 74 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_74.fasta\n",
      "--- *blasting region: 74\n",
      "--- total time for blast 74: 9.285s.\n",
      "-- processing region: 740 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_740.fasta\n",
      "--- *blasting region: 740\n",
      "--- total time for blast 740: 14.628s.\n",
      "-- processing region: 741 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_741.fasta\n",
      "--- *blasting region: 741\n",
      "--- total time for blast 741: 11.264s.\n",
      "-- processing region: 742 with 353 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_742.fasta\n",
      "--- *blasting region: 742\n",
      "--- total time for blast 742: 13.378s.\n",
      "-- processing region: 743 with 306 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_743.fasta\n",
      "--- *blasting region: 743\n",
      "--- total time for blast 743: 13.711s.\n",
      "-- processing region: 744 with 308 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_744.fasta\n",
      "--- *blasting region: 744\n",
      "--- total time for blast 744: 15.328s.\n",
      "-- processing region: 745 with 367 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_745.fasta\n",
      "--- *blasting region: 745\n",
      "--- total time for blast 745: 15.038s.\n",
      "-- processing region: 748 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_748.fasta\n",
      "--- *blasting region: 748\n",
      "--- total time for blast 748: 11.535s.\n",
      "-- processing region: 749 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_749.fasta\n",
      "--- *blasting region: 749\n",
      "--- total time for blast 749: 10.980s.\n",
      "-- processing region: 75 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_75.fasta\n",
      "--- *blasting region: 75\n",
      "--- total time for blast 75: 9.784s.\n",
      "-- processing region: 750 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_750.fasta\n",
      "--- *blasting region: 750\n",
      "--- total time for blast 750: 17.444s.\n",
      "-- processing region: 751 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_751.fasta\n",
      "--- *blasting region: 751\n",
      "--- total time for blast 751: 14.028s.\n",
      "-- processing region: 752 with 307 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_752.fasta\n",
      "--- *blasting region: 752\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 752: 14.818s.\n",
      "-- processing region: 753 with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_753.fasta\n",
      "--- *blasting region: 753\n",
      "--- total time for blast 753: 6.926s.\n",
      "-- processing region: 754 with 344 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_754.fasta\n",
      "--- *blasting region: 754\n",
      "--- total time for blast 754: 11.577s.\n",
      "-- processing region: 755 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_755.fasta\n",
      "--- *blasting region: 755\n",
      "--- total time for blast 755: 7.157s.\n",
      "-- processing region: 756 with 319 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_756.fasta\n",
      "--- *blasting region: 756\n",
      "--- total time for blast 756: 17.817s.\n",
      "-- processing region: 757 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_757.fasta\n",
      "--- *blasting region: 757\n",
      "--- total time for blast 757: 12.907s.\n",
      "-- processing region: 758 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_758.fasta\n",
      "--- *blasting region: 758\n",
      "--- total time for blast 758: 9.294s.\n",
      "-- processing region: 759 with 306 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_759.fasta\n",
      "--- *blasting region: 759\n",
      "--- total time for blast 759: 15.087s.\n",
      "-- processing region: 76 with 121 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_76.fasta\n",
      "--- *blasting region: 76\n",
      "--- total time for blast 76: 13.802s.\n",
      "-- processing region: 760 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_760.fasta\n",
      "--- *blasting region: 760\n",
      "--- total time for blast 760: 13.167s.\n",
      "-- processing region: 761 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_761.fasta\n",
      "--- *blasting region: 761\n",
      "--- total time for blast 761: 13.080s.\n",
      "-- processing region: 762 with 140 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_762.fasta\n",
      "--- *blasting region: 762\n",
      "--- total time for blast 762: 10.989s.\n",
      "-- processing region: 763 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_763.fasta\n",
      "--- *blasting region: 763\n",
      "--- total time for blast 763: 13.993s.\n",
      "-- processing region: 764 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_764.fasta\n",
      "--- *blasting region: 764\n",
      "--- total time for blast 764: 14.003s.\n",
      "-- processing region: 765 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_765.fasta\n",
      "--- *blasting region: 765\n",
      "--- total time for blast 765: 11.663s.\n",
      "-- processing region: 766 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_766.fasta\n",
      "--- *blasting region: 766\n",
      "--- total time for blast 766: 11.483s.\n",
      "-- processing region: 767 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_767.fasta\n",
      "--- *blasting region: 767\n",
      "--- total time for blast 767: 13.169s.\n",
      "-- processing region: 768 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_768.fasta\n",
      "--- *blasting region: 768\n",
      "--- total time for blast 768: 11.162s.\n",
      "-- processing region: 769 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_769.fasta\n",
      "--- *blasting region: 769\n",
      "--- total time for blast 769: 8.737s.\n",
      "-- processing region: 77 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_77.fasta\n",
      "--- *blasting region: 77\n",
      "--- total time for blast 77: 11.871s.\n",
      "-- processing region: 770 with 81 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_770.fasta\n",
      "--- *blasting region: 770\n",
      "--- total time for blast 770: 7.853s.\n",
      "-- processing region: 774 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_774.fasta\n",
      "--- *blasting region: 774\n",
      "--- total time for blast 774: 10.443s.\n",
      "-- processing region: 775 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_775.fasta\n",
      "--- *blasting region: 775\n",
      "--- total time for blast 775: 17.970s.\n",
      "-- processing region: 776 with 317 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_776.fasta\n",
      "--- *blasting region: 776\n",
      "--- total time for blast 776: 10.809s.\n",
      "-- processing region: 777 with 372 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_777.fasta\n",
      "--- *blasting region: 777\n",
      "--- total time for blast 777: 14.831s.\n",
      "-- processing region: 778 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_778.fasta\n",
      "--- *blasting region: 778\n",
      "--- total time for blast 778: 13.039s.\n",
      "-- processing region: 779 with 208 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_779.fasta\n",
      "--- *blasting region: 779\n",
      "--- total time for blast 779: 10.149s.\n",
      "-- processing region: 78 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_78.fasta\n",
      "--- *blasting region: 78\n",
      "--- total time for blast 78: 10.406s.\n",
      "-- processing region: 780 with 349 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_780.fasta\n",
      "--- *blasting region: 780\n",
      "--- total time for blast 780: 13.690s.\n",
      "-- processing region: 782 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_782.fasta\n",
      "--- *blasting region: 782\n",
      "--- total time for blast 782: 8.914s.\n",
      "-- processing region: 783 with 285 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_783.fasta\n",
      "--- *blasting region: 783\n",
      "--- total time for blast 783: 13.376s.\n",
      "-- processing region: 784 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_784.fasta\n",
      "--- *blasting region: 784\n",
      "--- total time for blast 784: 15.671s.\n",
      "-- processing region: 785 with 295 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_785.fasta\n",
      "--- *blasting region: 785\n",
      "--- total time for blast 785: 18.024s.\n",
      "-- processing region: 786 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_786.fasta\n",
      "--- *blasting region: 786\n",
      "--- total time for blast 786: 9.962s.\n",
      "-- processing region: 787 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_787.fasta\n",
      "--- *blasting region: 787\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 787: 13.512s.\n",
      "-- processing region: 788 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_788.fasta\n",
      "--- *blasting region: 788\n",
      "--- total time for blast 788: 13.638s.\n",
      "-- processing region: 789 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_789.fasta\n",
      "--- *blasting region: 789\n",
      "--- total time for blast 789: 16.602s.\n",
      "-- processing region: 79 with 198 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_79.fasta\n",
      "--- *blasting region: 79\n",
      "--- total time for blast 79: 13.171s.\n",
      "-- processing region: 790 with 91 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_790.fasta\n",
      "--- *blasting region: 790\n",
      "--- total time for blast 790: 4.876s.\n",
      "-- processing region: 791 with 164 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_791.fasta\n",
      "--- *blasting region: 791\n",
      "--- total time for blast 791: 10.202s.\n",
      "-- processing region: 792 with 168 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_792.fasta\n",
      "--- *blasting region: 792\n",
      "--- total time for blast 792: 12.266s.\n",
      "-- processing region: 793 with 231 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_793.fasta\n",
      "--- *blasting region: 793\n",
      "--- total time for blast 793: 13.909s.\n",
      "-- processing region: 794 with 298 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_794.fasta\n",
      "--- *blasting region: 794\n",
      "--- total time for blast 794: 14.155s.\n",
      "-- processing region: 795 with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_795.fasta\n",
      "--- *blasting region: 795\n",
      "--- total time for blast 795: 10.794s.\n",
      "-- processing region: 796 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_796.fasta\n",
      "--- *blasting region: 796\n",
      "--- total time for blast 796: 13.857s.\n",
      "-- processing region: 797 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_797.fasta\n",
      "--- *blasting region: 797\n",
      "--- total time for blast 797: 14.031s.\n",
      "-- processing region: 798 with 244 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_798.fasta\n",
      "--- *blasting region: 798\n",
      "--- total time for blast 798: 13.380s.\n",
      "-- processing region: 799 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_799.fasta\n",
      "--- *blasting region: 799\n",
      "--- total time for blast 799: 12.063s.\n",
      "-- processing region: 8 with 323 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_8.fasta\n",
      "--- *blasting region: 8\n",
      "--- total time for blast 8: 12.458s.\n",
      "-- processing region: 80 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_80.fasta\n",
      "--- *blasting region: 80\n",
      "--- total time for blast 80: 10.023s.\n",
      "-- processing region: 800 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_800.fasta\n",
      "--- *blasting region: 800\n",
      "--- total time for blast 800: 11.823s.\n",
      "-- processing region: 801 with 320 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_801.fasta\n",
      "--- *blasting region: 801\n",
      "--- total time for blast 801: 14.577s.\n",
      "-- processing region: 802 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_802.fasta\n",
      "--- *blasting region: 802\n",
      "--- total time for blast 802: 12.338s.\n",
      "-- processing region: 803 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_803.fasta\n",
      "--- *blasting region: 803\n",
      "--- total time for blast 803: 14.181s.\n",
      "-- processing region: 804 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_804.fasta\n",
      "--- *blasting region: 804\n",
      "--- total time for blast 804: 9.672s.\n",
      "-- processing region: 805 with 168 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_805.fasta\n",
      "--- *blasting region: 805\n",
      "--- total time for blast 805: 10.091s.\n",
      "-- processing region: 806 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_806.fasta\n",
      "--- *blasting region: 806\n",
      "--- total time for blast 806: 12.918s.\n",
      "-- processing region: 807 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_807.fasta\n",
      "--- *blasting region: 807\n",
      "--- total time for blast 807: 10.336s.\n",
      "-- processing region: 808 with 197 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_808.fasta\n",
      "--- *blasting region: 808\n",
      "--- total time for blast 808: 11.163s.\n",
      "-- processing region: 809 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_809.fasta\n",
      "--- *blasting region: 809\n",
      "--- total time for blast 809: 15.882s.\n",
      "-- processing region: 81 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_81.fasta\n",
      "--- *blasting region: 81\n",
      "--- total time for blast 81: 11.372s.\n",
      "-- processing region: 810 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_810.fasta\n",
      "--- *blasting region: 810\n",
      "--- total time for blast 810: 10.946s.\n",
      "-- processing region: 811 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_811.fasta\n",
      "--- *blasting region: 811\n",
      "--- total time for blast 811: 12.061s.\n",
      "-- processing region: 812 with 358 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_812.fasta\n",
      "--- *blasting region: 812\n",
      "--- total time for blast 812: 13.458s.\n",
      "-- processing region: 813 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_813.fasta\n",
      "--- *blasting region: 813\n",
      "--- total time for blast 813: 9.692s.\n",
      "-- processing region: 814 with 275 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_814.fasta\n",
      "--- *blasting region: 814\n",
      "--- total time for blast 814: 14.607s.\n",
      "-- processing region: 815 with 328 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_815.fasta\n",
      "--- *blasting region: 815\n",
      "--- total time for blast 815: 15.463s.\n",
      "-- processing region: 816 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_816.fasta\n",
      "--- *blasting region: 816\n",
      "--- total time for blast 816: 10.419s.\n",
      "-- processing region: 817 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_817.fasta\n",
      "--- *blasting region: 817\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 817: 9.868s.\n",
      "-- processing region: 819 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_819.fasta\n",
      "--- *blasting region: 819\n",
      "--- total time for blast 819: 33.025s.\n",
      "-- processing region: 820 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_820.fasta\n",
      "--- *blasting region: 820\n",
      "--- total time for blast 820: 14.272s.\n",
      "-- processing region: 821 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_821.fasta\n",
      "--- *blasting region: 821\n",
      "--- total time for blast 821: 8.587s.\n",
      "-- processing region: 822 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_822.fasta\n",
      "--- *blasting region: 822\n",
      "--- total time for blast 822: 11.873s.\n",
      "-- processing region: 823 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_823.fasta\n",
      "--- *blasting region: 823\n",
      "--- total time for blast 823: 14.676s.\n",
      "-- processing region: 824 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_824.fasta\n",
      "--- *blasting region: 824\n",
      "--- total time for blast 824: 19.780s.\n",
      "-- processing region: 825 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_825.fasta\n",
      "--- *blasting region: 825\n",
      "--- total time for blast 825: 12.570s.\n",
      "-- processing region: 826 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_826.fasta\n",
      "--- *blasting region: 826\n",
      "--- total time for blast 826: 13.286s.\n",
      "-- processing region: 827 with 283 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_827.fasta\n",
      "--- *blasting region: 827\n",
      "--- total time for blast 827: 15.461s.\n",
      "-- processing region: 828 with 377 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_828.fasta\n",
      "--- *blasting region: 828\n",
      "--- total time for blast 828: 13.406s.\n",
      "-- processing region: 829 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_829.fasta\n",
      "--- *blasting region: 829\n",
      "--- total time for blast 829: 12.952s.\n",
      "-- processing region: 83 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_83.fasta\n",
      "--- *blasting region: 83\n",
      "--- total time for blast 83: 13.387s.\n",
      "-- processing region: 832 with 168 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_832.fasta\n",
      "--- *blasting region: 832\n",
      "--- total time for blast 832: 9.967s.\n",
      "-- processing region: 833 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_833.fasta\n",
      "--- *blasting region: 833\n",
      "--- total time for blast 833: 9.034s.\n",
      "-- processing region: 834 with 310 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_834.fasta\n",
      "--- *blasting region: 834\n",
      "--- total time for blast 834: 14.087s.\n",
      "-- processing region: 835 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_835.fasta\n",
      "--- *blasting region: 835\n",
      "--- total time for blast 835: 13.538s.\n",
      "-- processing region: 836 with 164 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_836.fasta\n",
      "--- *blasting region: 836\n",
      "--- total time for blast 836: 14.976s.\n",
      "-- processing region: 837 with 302 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_837.fasta\n",
      "--- *blasting region: 837\n",
      "--- total time for blast 837: 13.171s.\n",
      "-- processing region: 838 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_838.fasta\n",
      "--- *blasting region: 838\n",
      "--- total time for blast 838: 12.200s.\n",
      "-- processing region: 839 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_839.fasta\n",
      "--- *blasting region: 839\n",
      "--- total time for blast 839: 34.053s.\n",
      "-- processing region: 84 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_84.fasta\n",
      "--- *blasting region: 84\n",
      "--- total time for blast 84: 10.219s.\n",
      "-- processing region: 840 with 264 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_840.fasta\n",
      "--- *blasting region: 840\n",
      "--- total time for blast 840: 14.245s.\n",
      "-- processing region: 841 with 373 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_841.fasta\n",
      "--- *blasting region: 841\n",
      "--- total time for blast 841: 13.773s.\n",
      "-- processing region: 843 with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_843.fasta\n",
      "--- *blasting region: 843\n",
      "--- total time for blast 843: 7.607s.\n",
      "-- processing region: 844 with 256 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_844.fasta\n",
      "--- *blasting region: 844\n",
      "--- total time for blast 844: 14.349s.\n",
      "-- processing region: 845 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_845.fasta\n",
      "--- *blasting region: 845\n",
      "--- total time for blast 845: 13.629s.\n",
      "-- processing region: 846 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_846.fasta\n",
      "--- *blasting region: 846\n",
      "--- total time for blast 846: 15.145s.\n",
      "-- processing region: 847 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_847.fasta\n",
      "--- *blasting region: 847\n",
      "--- total time for blast 847: 10.019s.\n",
      "-- processing region: 848 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_848.fasta\n",
      "--- *blasting region: 848\n",
      "--- total time for blast 848: 13.827s.\n",
      "-- processing region: 849 with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_849.fasta\n",
      "--- *blasting region: 849\n",
      "--- total time for blast 849: 8.497s.\n",
      "-- processing region: 85 with 313 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_85.fasta\n",
      "--- *blasting region: 85\n",
      "--- total time for blast 85: 13.928s.\n",
      "-- processing region: 850 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_850.fasta\n",
      "--- *blasting region: 850\n",
      "--- total time for blast 850: 9.809s.\n",
      "-- processing region: 851 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_851.fasta\n",
      "--- *blasting region: 851\n",
      "--- total time for blast 851: 13.777s.\n",
      "-- processing region: 852 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_852.fasta\n",
      "--- *blasting region: 852\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 852: 13.778s.\n",
      "-- processing region: 853 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_853.fasta\n",
      "--- *blasting region: 853\n",
      "--- total time for blast 853: 13.035s.\n",
      "-- processing region: 854 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_854.fasta\n",
      "--- *blasting region: 854\n",
      "--- total time for blast 854: 15.737s.\n",
      "-- processing region: 855 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_855.fasta\n",
      "--- *blasting region: 855\n",
      "--- total time for blast 855: 15.706s.\n",
      "-- processing region: 856 with 163 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_856.fasta\n",
      "--- *blasting region: 856\n",
      "--- total time for blast 856: 12.760s.\n",
      "-- processing region: 857 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_857.fasta\n",
      "--- *blasting region: 857\n",
      "--- total time for blast 857: 13.964s.\n",
      "-- processing region: 858 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_858.fasta\n",
      "--- *blasting region: 858\n",
      "--- total time for blast 858: 14.295s.\n",
      "-- processing region: 859 with 240 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_859.fasta\n",
      "--- *blasting region: 859\n",
      "--- total time for blast 859: 18.574s.\n",
      "-- processing region: 86 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_86.fasta\n",
      "--- *blasting region: 86\n",
      "--- total time for blast 86: 11.786s.\n",
      "-- processing region: 860 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_860.fasta\n",
      "--- *blasting region: 860\n",
      "--- total time for blast 860: 12.160s.\n",
      "-- processing region: 861 with 333 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_861.fasta\n",
      "--- *blasting region: 861\n",
      "--- total time for blast 861: 15.781s.\n",
      "-- processing region: 862 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_862.fasta\n",
      "--- *blasting region: 862\n",
      "--- total time for blast 862: 11.651s.\n",
      "-- processing region: 863 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_863.fasta\n",
      "--- *blasting region: 863\n",
      "--- total time for blast 863: 12.912s.\n",
      "-- processing region: 864 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_864.fasta\n",
      "--- *blasting region: 864\n",
      "--- total time for blast 864: 13.241s.\n",
      "-- processing region: 865 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_865.fasta\n",
      "--- *blasting region: 865\n",
      "--- total time for blast 865: 16.009s.\n",
      "-- processing region: 866 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_866.fasta\n",
      "--- *blasting region: 866\n",
      "--- total time for blast 866: 11.038s.\n",
      "-- processing region: 867 with 164 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_867.fasta\n",
      "--- *blasting region: 867\n",
      "--- total time for blast 867: 8.194s.\n",
      "-- processing region: 868 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_868.fasta\n",
      "--- *blasting region: 868\n",
      "--- total time for blast 868: 25.649s.\n",
      "-- processing region: 869 with 301 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_869.fasta\n",
      "--- *blasting region: 869\n",
      "--- total time for blast 869: 15.993s.\n",
      "-- processing region: 87 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_87.fasta\n",
      "--- *blasting region: 87\n",
      "--- total time for blast 87: 12.717s.\n",
      "-- processing region: 870 with 324 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_870.fasta\n",
      "--- *blasting region: 870\n",
      "--- total time for blast 870: 14.201s.\n",
      "-- processing region: 871 with 329 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_871.fasta\n",
      "--- *blasting region: 871\n",
      "--- total time for blast 871: 12.567s.\n",
      "-- processing region: 872 with 359 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_872.fasta\n",
      "--- *blasting region: 872\n",
      "--- total time for blast 872: 15.886s.\n",
      "-- processing region: 873 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_873.fasta\n",
      "--- *blasting region: 873\n",
      "--- total time for blast 873: 13.522s.\n",
      "-- processing region: 874 with 370 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_874.fasta\n",
      "--- *blasting region: 874\n",
      "--- total time for blast 874: 17.692s.\n",
      "-- processing region: 875 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_875.fasta\n",
      "--- *blasting region: 875\n",
      "--- total time for blast 875: 12.630s.\n",
      "-- processing region: 876 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_876.fasta\n",
      "--- *blasting region: 876\n",
      "--- total time for blast 876: 12.546s.\n",
      "-- processing region: 877 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_877.fasta\n",
      "--- *blasting region: 877\n",
      "--- total time for blast 877: 14.495s.\n",
      "-- processing region: 878 with 347 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_878.fasta\n",
      "--- *blasting region: 878\n",
      "--- total time for blast 878: 11.956s.\n",
      "-- processing region: 88 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_88.fasta\n",
      "--- *blasting region: 88\n",
      "--- total time for blast 88: 8.681s.\n",
      "-- processing region: 888 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_888.fasta\n",
      "--- *blasting region: 888\n",
      "--- total time for blast 888: 23.963s.\n",
      "-- processing region: 889 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_889.fasta\n",
      "--- *blasting region: 889\n",
      "--- total time for blast 889: 11.743s.\n",
      "-- processing region: 89 with 335 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_89.fasta\n",
      "--- *blasting region: 89\n",
      "--- total time for blast 89: 12.855s.\n",
      "-- processing region: 890 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_890.fasta\n",
      "--- *blasting region: 890\n",
      "--- total time for blast 890: 10.951s.\n",
      "-- processing region: 891 with 169 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_891.fasta\n",
      "--- *blasting region: 891\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 891: 9.538s.\n",
      "-- processing region: 892 with 314 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_892.fasta\n",
      "--- *blasting region: 892\n",
      "--- total time for blast 892: 13.865s.\n",
      "-- processing region: 893 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_893.fasta\n",
      "--- *blasting region: 893\n",
      "--- total time for blast 893: 10.892s.\n",
      "-- processing region: 894 with 308 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_894.fasta\n",
      "--- *blasting region: 894\n",
      "--- total time for blast 894: 13.146s.\n",
      "-- processing region: 895 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_895.fasta\n",
      "--- *blasting region: 895\n",
      "--- total time for blast 895: 14.493s.\n",
      "-- processing region: 896 with 158 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_896.fasta\n",
      "--- *blasting region: 896\n",
      "--- total time for blast 896: 11.949s.\n",
      "-- processing region: 897 with 286 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_897.fasta\n",
      "--- *blasting region: 897\n",
      "--- total time for blast 897: 12.084s.\n",
      "-- processing region: 898 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_898.fasta\n",
      "--- *blasting region: 898\n",
      "--- total time for blast 898: 14.470s.\n",
      "-- processing region: 899 with 211 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_899.fasta\n",
      "--- *blasting region: 899\n",
      "--- total time for blast 899: 11.265s.\n",
      "-- processing region: 9 with 264 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_9.fasta\n",
      "--- *blasting region: 9\n",
      "--- total time for blast 9: 12.250s.\n",
      "-- processing region: 90 with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_90.fasta\n",
      "--- *blasting region: 90\n",
      "--- total time for blast 90: 8.147s.\n",
      "-- processing region: 900 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_900.fasta\n",
      "--- *blasting region: 900\n",
      "--- total time for blast 900: 12.176s.\n",
      "-- processing region: 901 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_901.fasta\n",
      "--- *blasting region: 901\n",
      "--- total time for blast 901: 11.050s.\n",
      "-- processing region: 902 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_902.fasta\n",
      "--- *blasting region: 902\n",
      "--- total time for blast 902: 12.190s.\n",
      "-- processing region: 903 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_903.fasta\n",
      "--- *blasting region: 903\n",
      "--- total time for blast 903: 9.058s.\n",
      "-- processing region: 904 with 320 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_904.fasta\n",
      "--- *blasting region: 904\n",
      "--- total time for blast 904: 14.499s.\n",
      "-- processing region: 905 with 311 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_905.fasta\n",
      "--- *blasting region: 905\n",
      "--- total time for blast 905: 13.481s.\n",
      "-- processing region: 906 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_906.fasta\n",
      "--- *blasting region: 906\n",
      "--- total time for blast 906: 14.344s.\n",
      "-- processing region: 907 with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_907.fasta\n",
      "--- *blasting region: 907\n",
      "--- total time for blast 907: 7.740s.\n",
      "-- processing region: 908 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_908.fasta\n",
      "--- *blasting region: 908\n",
      "--- total time for blast 908: 13.846s.\n",
      "-- processing region: 909 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_909.fasta\n",
      "--- *blasting region: 909\n",
      "--- total time for blast 909: 11.287s.\n",
      "-- processing region: 91 with 363 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_91.fasta\n",
      "--- *blasting region: 91\n",
      "--- total time for blast 91: 15.038s.\n",
      "-- processing region: 910 with 155 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_910.fasta\n",
      "--- *blasting region: 910\n",
      "--- total time for blast 910: 15.510s.\n",
      "-- processing region: 911 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_911.fasta\n",
      "--- *blasting region: 911\n",
      "--- total time for blast 911: 9.284s.\n",
      "-- processing region: 912 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_912.fasta\n",
      "--- *blasting region: 912\n",
      "--- total time for blast 912: 11.500s.\n",
      "-- processing region: 913 with 169 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_913.fasta\n",
      "--- *blasting region: 913\n",
      "--- total time for blast 913: 13.959s.\n",
      "-- processing region: 914 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_914.fasta\n",
      "--- *blasting region: 914\n",
      "--- total time for blast 914: 10.322s.\n",
      "-- processing region: 915 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_915.fasta\n",
      "--- *blasting region: 915\n",
      "--- total time for blast 915: 8.457s.\n",
      "-- processing region: 916 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_916.fasta\n",
      "--- *blasting region: 916\n",
      "--- total time for blast 916: 12.113s.\n",
      "-- processing region: 917 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_917.fasta\n",
      "--- *blasting region: 917\n",
      "--- total time for blast 917: 12.289s.\n",
      "-- processing region: 918 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_918.fasta\n",
      "--- *blasting region: 918\n",
      "--- total time for blast 918: 14.937s.\n",
      "-- processing region: 919 with 157 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_919.fasta\n",
      "--- *blasting region: 919\n",
      "--- total time for blast 919: 16.749s.\n",
      "-- processing region: 92 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_92.fasta\n",
      "--- *blasting region: 92\n",
      "--- total time for blast 92: 12.884s.\n",
      "-- processing region: 920 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_920.fasta\n",
      "--- *blasting region: 920\n",
      "--- total time for blast 920: 11.062s.\n",
      "-- processing region: 921 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_921.fasta\n",
      "--- *blasting region: 921\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 921: 8.878s.\n",
      "-- processing region: 922 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_922.fasta\n",
      "--- *blasting region: 922\n",
      "--- total time for blast 922: 16.517s.\n",
      "-- processing region: 923 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_923.fasta\n",
      "--- *blasting region: 923\n",
      "--- total time for blast 923: 15.412s.\n",
      "-- processing region: 924 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_924.fasta\n",
      "--- *blasting region: 924\n",
      "--- total time for blast 924: 12.257s.\n",
      "-- processing region: 925 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_925.fasta\n",
      "--- *blasting region: 925\n",
      "--- total time for blast 925: 11.612s.\n",
      "-- processing region: 926 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_926.fasta\n",
      "--- *blasting region: 926\n",
      "--- total time for blast 926: 11.956s.\n",
      "-- processing region: 927 with 314 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_927.fasta\n",
      "--- *blasting region: 927\n",
      "--- total time for blast 927: 16.651s.\n",
      "-- processing region: 928 with 334 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_928.fasta\n",
      "--- *blasting region: 928\n",
      "--- total time for blast 928: 15.292s.\n",
      "-- processing region: 929 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_929.fasta\n",
      "--- *blasting region: 929\n",
      "--- total time for blast 929: 15.765s.\n",
      "-- processing region: 93 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_93.fasta\n",
      "--- *blasting region: 93\n",
      "--- total time for blast 93: 9.715s.\n",
      "-- processing region: 930 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_930.fasta\n",
      "--- *blasting region: 930\n",
      "--- total time for blast 930: 13.497s.\n",
      "-- processing region: 931 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_931.fasta\n",
      "--- *blasting region: 931\n",
      "--- total time for blast 931: 13.182s.\n",
      "-- processing region: 932 with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_932.fasta\n",
      "--- *blasting region: 932\n",
      "--- total time for blast 932: 13.127s.\n",
      "-- processing region: 933 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_933.fasta\n",
      "--- *blasting region: 933\n",
      "--- total time for blast 933: 8.735s.\n",
      "-- processing region: 934 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_934.fasta\n",
      "--- *blasting region: 934\n",
      "--- total time for blast 934: 10.184s.\n",
      "-- processing region: 935 with 369 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_935.fasta\n",
      "--- *blasting region: 935\n",
      "--- total time for blast 935: 16.458s.\n",
      "-- processing region: 94 with 178 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_94.fasta\n",
      "--- *blasting region: 94\n",
      "--- total time for blast 94: 9.873s.\n",
      "-- processing region: 949 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_949.fasta\n",
      "--- *blasting region: 949\n",
      "--- total time for blast 949: 9.167s.\n",
      "-- processing region: 95 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_95.fasta\n",
      "--- *blasting region: 95\n",
      "--- total time for blast 95: 11.864s.\n",
      "-- processing region: 950 with 178 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_950.fasta\n",
      "--- *blasting region: 950\n",
      "--- total time for blast 950: 11.407s.\n",
      "-- processing region: 951 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_951.fasta\n",
      "--- *blasting region: 951\n",
      "--- total time for blast 951: 9.798s.\n",
      "-- processing region: 952 with 190 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_952.fasta\n",
      "--- *blasting region: 952\n",
      "--- total time for blast 952: 12.579s.\n",
      "-- processing region: 953 with 245 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_953.fasta\n",
      "--- *blasting region: 953\n",
      "--- total time for blast 953: 14.555s.\n",
      "-- processing region: 954 with 90 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_954.fasta\n",
      "--- *blasting region: 954\n",
      "--- total time for blast 954: 7.479s.\n",
      "-- processing region: 955 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_955.fasta\n",
      "--- *blasting region: 955\n",
      "--- total time for blast 955: 13.318s.\n",
      "-- processing region: 956 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_956.fasta\n",
      "--- *blasting region: 956\n",
      "--- total time for blast 956: 12.689s.\n",
      "-- processing region: 957 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_957.fasta\n",
      "--- *blasting region: 957\n",
      "--- total time for blast 957: 10.775s.\n",
      "-- processing region: 958 with 328 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_958.fasta\n",
      "--- *blasting region: 958\n",
      "--- total time for blast 958: 16.898s.\n",
      "-- processing region: 959 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_959.fasta\n",
      "--- *blasting region: 959\n",
      "--- total time for blast 959: 10.409s.\n",
      "-- processing region: 96 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_96.fasta\n",
      "--- *blasting region: 96\n",
      "--- total time for blast 96: 11.828s.\n",
      "-- processing region: 960 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_960.fasta\n",
      "--- *blasting region: 960\n",
      "--- total time for blast 960: 12.384s.\n",
      "-- processing region: 961 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_961.fasta\n",
      "--- *blasting region: 961\n",
      "--- total time for blast 961: 10.585s.\n",
      "-- processing region: 962 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_962.fasta\n",
      "--- *blasting region: 962\n",
      "--- total time for blast 962: 10.276s.\n",
      "-- processing region: 963 with 195 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_963.fasta\n",
      "--- *blasting region: 963\n",
      "--- total time for blast 963: 12.401s.\n",
      "-- processing region: 964 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_964.fasta\n",
      "--- *blasting region: 964\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 964: 11.505s.\n",
      "-- processing region: 965 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_965.fasta\n",
      "--- *blasting region: 965\n",
      "--- total time for blast 965: 11.129s.\n",
      "-- processing region: 966 with 161 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_966.fasta\n",
      "--- *blasting region: 966\n",
      "--- total time for blast 966: 12.232s.\n",
      "-- processing region: 967 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_967.fasta\n",
      "--- *blasting region: 967\n",
      "--- total time for blast 967: 14.380s.\n",
      "-- processing region: 968 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_968.fasta\n",
      "--- *blasting region: 968\n",
      "--- total time for blast 968: 14.207s.\n",
      "-- processing region: 969 with 155 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_969.fasta\n",
      "--- *blasting region: 969\n",
      "--- total time for blast 969: 11.942s.\n",
      "-- processing region: 970 with 110 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_970.fasta\n",
      "--- *blasting region: 970\n",
      "--- total time for blast 970: 6.988s.\n",
      "-- processing region: 971 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_971.fasta\n",
      "--- *blasting region: 971\n",
      "--- total time for blast 971: 9.399s.\n",
      "-- processing region: 972 with 392 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_972.fasta\n",
      "--- *blasting region: 972\n",
      "--- total time for blast 972: 14.476s.\n",
      "-- processing region: 973 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_973.fasta\n",
      "--- *blasting region: 973\n",
      "--- total time for blast 973: 13.383s.\n",
      "-- processing region: 974 with 208 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_974.fasta\n",
      "--- *blasting region: 974\n",
      "--- total time for blast 974: 10.152s.\n",
      "-- processing region: 975 with 314 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_975.fasta\n",
      "--- *blasting region: 975\n",
      "--- total time for blast 975: 13.287s.\n",
      "-- processing region: 976 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_976.fasta\n",
      "--- *blasting region: 976\n",
      "--- total time for blast 976: 12.284s.\n",
      "-- processing region: 977 with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_977.fasta\n",
      "--- *blasting region: 977\n",
      "--- total time for blast 977: 13.924s.\n",
      "-- processing region: 978 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_978.fasta\n",
      "--- *blasting region: 978\n",
      "--- total time for blast 978: 16.178s.\n",
      "-- processing region: 979 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_979.fasta\n",
      "--- *blasting region: 979\n",
      "--- total time for blast 979: 12.638s.\n",
      "-- processing region: 98 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_98.fasta\n",
      "--- *blasting region: 98\n",
      "--- total time for blast 98: 9.189s.\n",
      "-- processing region: 980 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_980.fasta\n",
      "--- *blasting region: 980\n",
      "--- total time for blast 980: 11.915s.\n",
      "-- processing region: 981 with 302 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_981.fasta\n",
      "--- *blasting region: 981\n",
      "--- total time for blast 981: 12.863s.\n",
      "-- processing region: 982 with 171 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_982.fasta\n",
      "--- *blasting region: 982\n",
      "--- total time for blast 982: 9.658s.\n",
      "-- processing region: 983 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_983.fasta\n",
      "--- *blasting region: 983\n",
      "--- total time for blast 983: 9.973s.\n",
      "-- processing region: 984 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_984.fasta\n",
      "--- *blasting region: 984\n",
      "--- total time for blast 984: 10.956s.\n",
      "-- processing region: 985 with 342 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_985.fasta\n",
      "--- *blasting region: 985\n",
      "--- total time for blast 985: 11.179s.\n",
      "-- processing region: 986 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_986.fasta\n",
      "--- *blasting region: 986\n",
      "--- total time for blast 986: 9.334s.\n",
      "-- processing region: 987 with 365 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_987.fasta\n",
      "--- *blasting region: 987\n",
      "--- total time for blast 987: 15.701s.\n",
      "-- processing region: 988 with 348 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_988.fasta\n",
      "--- *blasting region: 988\n",
      "--- total time for blast 988: 15.180s.\n",
      "-- processing region: 989 with 154 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_989.fasta\n",
      "--- *blasting region: 989\n",
      "--- total time for blast 989: 10.948s.\n",
      "-- processing region: 990 with 281 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_990.fasta\n",
      "--- *blasting region: 990\n",
      "--- total time for blast 990: 14.252s.\n",
      "-- processing region: 991 with 247 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_991.fasta\n",
      "--- *blasting region: 991\n",
      "--- total time for blast 991: 13.184s.\n",
      "-- processing region: 992 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_992.fasta\n",
      "--- *blasting region: 992\n",
      "--- total time for blast 992: 9.793s.\n",
      "-- processing region: 993 with 153 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_993.fasta\n",
      "--- *blasting region: 993\n",
      "--- total time for blast 993: 7.010s.\n",
      "-- processing region: 994 with 286 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_994.fasta\n",
      "--- *blasting region: 994\n",
      "--- total time for blast 994: 13.090s.\n",
      "-- processing region: 995 with 337 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_995.fasta\n",
      "--- *blasting region: 995\n",
      "--- total time for blast 995: 14.696s.\n",
      "-- processing region: 996 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_996.fasta\n",
      "--- *blasting region: 996\n",
      "--- total time for blast 996: 12.941s.\n",
      "-- processing region: 997 with 304 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_997.fasta\n",
      "--- *blasting region: 997\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 997: 12.652s.\n",
      "-- processing region: 999 with 341 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_999.fasta\n",
      "--- *blasting region: 999\n",
      "--- total time for blast 999: 13.933s.\n"
     ]
    }
   ],
   "source": [
    "blast_database = os.path.join(reference_folder, 'blast_database', 'hg38')\n",
    "\n",
    "check.Blast_probes(kept_records, library_folder, num_threads=44, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7c6a9506",
   "metadata": {},
   "source": [
    "### 4.2.1 screen probe by blast results\n",
    "\n",
    "This step has 2 different cutoffs:\n",
    "\n",
    "hard_cutoff: whether a probe has any ~40mer hit somewhere else, if so just dump the probe\n",
    "\n",
    "soft_cutoff: whether a probe has a lot of ~20mer off-target hits, if exceed certain amount then dump the probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 154,
   "id": "6f6749b3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 327063\n",
      "- Number of regions in this library: 1352\n",
      "-- checking probes in region: 0\n",
      "--- gene=0, id=179 removed by soft count = 135\n",
      "--- gene=0, id=186 removed by hard count = 12\n",
      "--- gene=0, id=200 removed by soft count = 61\n",
      "--- gene=0, id=201 removed by soft count = 87\n",
      "--- gene=0, id=202 removed by soft count = 232\n",
      "--- number of probes: 339 , kept by blast: 334 , if remove dups: 334\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1\n",
      "--- gene=1, id=67 removed by soft count = 197\n",
      "--- gene=1, id=68 removed by soft count = 113\n",
      "--- gene=1, id=150 removed by soft count = 106\n",
      "--- gene=1, id=158 removed by soft count = 104\n",
      "--- gene=1, id=161 removed by soft count = 134\n",
      "--- gene=1, id=162 removed by soft count = 168\n",
      "--- gene=1, id=163 removed by soft count = 144\n",
      "--- gene=1, id=164 removed by soft count = 132\n",
      "--- gene=1, id=165 removed by soft count = 307\n",
      "--- number of probes: 306 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 10\n",
      "--- gene=10, id=17 removed by soft count = 360\n",
      "--- gene=10, id=20 removed by soft count = 126\n",
      "--- gene=10, id=21 removed by soft count = 123\n",
      "--- gene=10, id=22 removed by soft count = 119\n",
      "--- gene=10, id=89 removed by soft count = 77\n",
      "--- gene=10, id=107 removed by soft count = 1509\n",
      "--- gene=10, id=120 removed by soft count = 99\n",
      "--- gene=10, id=208 removed by soft count = 66\n",
      "--- number of probes: 216 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 100\n",
      "--- gene=100, id=47 removed by soft count = 103\n",
      "--- gene=100, id=48 removed by soft count = 92\n",
      "--- gene=100, id=49 removed by soft count = 63\n",
      "--- gene=100, id=51 removed by soft count = 86\n",
      "--- gene=100, id=52 removed by soft count = 197\n",
      "--- gene=100, id=53 removed by soft count = 466\n",
      "--- gene=100, id=54 removed by soft count = 273\n",
      "--- gene=100, id=313 removed by soft count = 2348\n",
      "--- number of probes: 344 , kept by blast: 336 , if remove dups: 336\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1000\n",
      "--- gene=1000, id=183 removed by soft count = 122\n",
      "--- gene=1000, id=191 removed by soft count = 276\n",
      "--- gene=1000, id=204 removed by soft count = 158\n",
      "--- number of probes: 326 , kept by blast: 323 , if remove dups: 323\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1001\n",
      "--- gene=1001, id=30 removed by soft count = 169\n",
      "--- gene=1001, id=34 removed by soft count = 1070\n",
      "--- gene=1001, id=108 removed by soft count = 236\n",
      "--- gene=1001, id=253 removed by soft count = 54\n",
      "--- gene=1001, id=261 removed by soft count = 72\n",
      "--- gene=1001, id=263 removed by soft count = 224\n",
      "--- gene=1001, id=266 removed by soft count = 185\n",
      "--- gene=1001, id=267 removed by soft count = 76\n",
      "--- gene=1001, id=280 removed by soft count = 154\n",
      "--- gene=1001, id=294 removed by soft count = 354\n",
      "--- gene=1001, id=296 removed by soft count = 57\n",
      "--- gene=1001, id=316 removed by soft count = 83\n",
      "--- gene=1001, id=333 removed by soft count = 100\n",
      "--- gene=1001, id=335 removed by soft count = 215\n",
      "--- number of probes: 341 , kept by blast: 327 , if remove dups: 327\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1002\n",
      "--- gene=1002, id=173 removed by soft count = 506\n",
      "--- gene=1002, id=179 removed by soft count = 93\n",
      "--- gene=1002, id=357 removed by soft count = 3368\n",
      "--- gene=1002, id=376 removed by soft count = 71\n",
      "--- number of probes: 381 , kept by blast: 377 , if remove dups: 377\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1003\n",
      "--- gene=1003, id=76 removed by soft count = 113\n",
      "--- gene=1003, id=86 removed by soft count = 141\n",
      "--- gene=1003, id=111 removed by soft count = 680\n",
      "--- gene=1003, id=113 removed by soft count = 862\n",
      "--- gene=1003, id=126 removed by hard count = 6\n",
      "--- gene=1003, id=310 removed by soft count = 75\n",
      "--- gene=1003, id=317 removed by soft count = 106\n",
      "--- number of probes: 332 , kept by blast: 325 , if remove dups: 325\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1004\n",
      "--- number of probes: 283 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1005\n",
      "--- gene=1005, id=229 removed by soft count = 61\n",
      "--- gene=1005, id=230 removed by soft count = 302\n",
      "--- gene=1005, id=251 removed by soft count = 67\n",
      "--- gene=1005, id=257 removed by soft count = 387\n",
      "--- number of probes: 313 , kept by blast: 309 , if remove dups: 308\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 101\n",
      "--- gene=101, id=79 removed by soft count = 190\n",
      "--- gene=101, id=104 removed by soft count = 97\n",
      "--- gene=101, id=107 removed by soft count = 78\n",
      "--- gene=101, id=169 removed by soft count = 148\n",
      "--- number of probes: 282 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 102\n",
      "--- gene=102, id=167 removed by soft count = 1149\n",
      "--- gene=102, id=170 removed by soft count = 89\n",
      "--- gene=102, id=175 removed by soft count = 187\n",
      "--- gene=102, id=189 removed by soft count = 282\n",
      "--- gene=102, id=232 removed by soft count = 588\n",
      "--- number of probes: 240 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1020\n",
      "--- gene=1020, id=19 removed by soft count = 261\n",
      "--- gene=1020, id=21 removed by hard count = 3\n",
      "--- gene=1020, id=25 removed by hard count = 3\n",
      "--- number of probes: 52 , kept by blast: 49 , if remove dups: 49\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: 1023\n",
      "--- gene=1023, id=20 removed by soft count = 492\n",
      "--- gene=1023, id=74 removed by soft count = 528\n",
      "--- gene=1023, id=91 removed by soft count = 69\n",
      "--- gene=1023, id=97 removed by soft count = 93\n",
      "--- gene=1023, id=98 removed by soft count = 74\n",
      "--- gene=1023, id=99 removed by soft count = 51\n",
      "--- gene=1023, id=100 removed by soft count = 52\n",
      "--- gene=1023, id=101 removed by soft count = 107\n",
      "--- gene=1023, id=109 removed by soft count = 182\n",
      "--- gene=1023, id=112 removed by soft count = 89\n",
      "--- gene=1023, id=113 removed by soft count = 82\n",
      "--- gene=1023, id=164 removed by soft count = 107\n",
      "--- gene=1023, id=165 removed by soft count = 74\n",
      "--- gene=1023, id=167 removed by soft count = 975\n",
      "--- gene=1023, id=191 removed by soft count = 79\n",
      "--- number of probes: 217 , kept by blast: 202 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1024\n",
      "--- gene=1024, id=17 removed by soft count = 561\n",
      "--- gene=1024, id=46 removed by soft count = 96\n",
      "--- gene=1024, id=47 removed by soft count = 1358\n",
      "--- gene=1024, id=53 removed by soft count = 158\n",
      "--- gene=1024, id=91 removed by soft count = 617\n",
      "--- gene=1024, id=92 removed by soft count = 98\n",
      "--- gene=1024, id=110 removed by soft count = 109\n",
      "--- gene=1024, id=117 removed by soft count = 251\n",
      "--- gene=1024, id=118 removed by soft count = 67\n",
      "--- gene=1024, id=124 removed by soft count = 88\n",
      "--- gene=1024, id=130 removed by soft count = 51\n",
      "--- gene=1024, id=137 removed by soft count = 68\n",
      "--- gene=1024, id=172 removed by soft count = 78\n",
      "--- number of probes: 242 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1025\n",
      "--- gene=1025, id=41 removed by soft count = 72\n",
      "--- gene=1025, id=107 removed by soft count = 51\n",
      "--- gene=1025, id=108 removed by soft count = 151\n",
      "--- gene=1025, id=109 removed by soft count = 100\n",
      "--- gene=1025, id=110 removed by soft count = 54\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1025, id=283 removed by soft count = 76\n",
      "--- number of probes: 306 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1029\n",
      "--- gene=1029, id=159 removed by soft count = 57\n",
      "--- gene=1029, id=230 removed by soft count = 69\n",
      "--- gene=1029, id=232 removed by soft count = 361\n",
      "--- number of probes: 235 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 103\n",
      "--- gene=103, id=21 removed by soft count = 76\n",
      "--- gene=103, id=29 removed by soft count = 63\n",
      "--- number of probes: 205 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1030\n",
      "--- gene=1030, id=72 removed by soft count = 99\n",
      "--- number of probes: 209 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1031\n",
      "--- gene=1031, id=62 removed by soft count = 433\n",
      "--- gene=1031, id=104 removed by soft count = 64\n",
      "--- gene=1031, id=219 removed by soft count = 150\n",
      "--- gene=1031, id=224 removed by soft count = 113\n",
      "--- number of probes: 290 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1032\n",
      "--- gene=1032, id=259 removed by soft count = 445\n",
      "--- number of probes: 275 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1033\n",
      "--- gene=1033, id=2 removed by soft count = 442\n",
      "--- gene=1033, id=4 removed by soft count = 574\n",
      "--- gene=1033, id=27 removed by soft count = 1796\n",
      "--- gene=1033, id=120 removed by soft count = 71\n",
      "--- gene=1033, id=121 removed by soft count = 365\n",
      "--- gene=1033, id=122 removed by soft count = 191\n",
      "--- gene=1033, id=123 removed by soft count = 1418\n",
      "--- gene=1033, id=124 removed by soft count = 195\n",
      "--- gene=1033, id=125 removed by soft count = 122\n",
      "--- gene=1033, id=129 removed by soft count = 183\n",
      "--- gene=1033, id=196 removed by soft count = 56\n",
      "--- gene=1033, id=200 removed by soft count = 874\n",
      "--- gene=1033, id=202 removed by soft count = 77\n",
      "--- number of probes: 234 , kept by blast: 221 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1034\n",
      "--- gene=1034, id=9 removed by soft count = 674\n",
      "--- gene=1034, id=17 removed by soft count = 295\n",
      "--- gene=1034, id=72 removed by soft count = 152\n",
      "--- number of probes: 183 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1035\n",
      "--- number of probes: 201 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1036\n",
      "--- number of probes: 130 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1037\n",
      "--- gene=1037, id=68 removed by soft count = 344\n",
      "--- gene=1037, id=142 removed by soft count = 69\n",
      "--- gene=1037, id=145 removed by soft count = 159\n",
      "--- gene=1037, id=230 removed by soft count = 53\n",
      "--- number of probes: 237 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1038\n",
      "--- gene=1038, id=45 removed by soft count = 838\n",
      "--- gene=1038, id=121 removed by soft count = 57\n",
      "--- gene=1038, id=123 removed by soft count = 426\n",
      "--- gene=1038, id=175 removed by soft count = 236\n",
      "--- gene=1038, id=177 removed by soft count = 63\n",
      "--- gene=1038, id=178 removed by soft count = 111\n",
      "--- gene=1038, id=183 removed by soft count = 86\n",
      "--- gene=1038, id=195 removed by soft count = 100\n",
      "--- gene=1038, id=199 removed by soft count = 84\n",
      "--- gene=1038, id=225 removed by soft count = 205\n",
      "--- number of probes: 227 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1039\n",
      "--- gene=1039, id=113 removed by soft count = 56\n",
      "--- gene=1039, id=114 removed by soft count = 229\n",
      "--- number of probes: 246 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 104\n",
      "--- gene=104, id=2 removed by soft count = 135\n",
      "--- gene=104, id=4 removed by soft count = 167\n",
      "--- gene=104, id=5 removed by soft count = 1193\n",
      "--- gene=104, id=6 removed by soft count = 848\n",
      "--- gene=104, id=7 removed by soft count = 508\n",
      "--- gene=104, id=8 removed by soft count = 136\n",
      "--- gene=104, id=9 removed by soft count = 66\n",
      "--- gene=104, id=11 removed by soft count = 88\n",
      "--- gene=104, id=13 removed by soft count = 95\n",
      "--- gene=104, id=14 removed by soft count = 226\n",
      "--- gene=104, id=15 removed by soft count = 177\n",
      "--- gene=104, id=16 removed by soft count = 203\n",
      "--- gene=104, id=18 removed by soft count = 223\n",
      "--- gene=104, id=20 removed by soft count = 1021\n",
      "--- gene=104, id=21 removed by soft count = 113\n",
      "--- gene=104, id=23 removed by soft count = 52\n",
      "--- gene=104, id=24 removed by soft count = 88\n",
      "--- gene=104, id=25 removed by soft count = 63\n",
      "--- gene=104, id=26 removed by soft count = 85\n",
      "--- gene=104, id=27 removed by soft count = 79\n",
      "--- gene=104, id=28 removed by soft count = 72\n",
      "--- gene=104, id=29 removed by soft count = 590\n",
      "--- gene=104, id=30 removed by soft count = 105\n",
      "--- number of probes: 119 , kept by blast: 96 , if remove dups: 96\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 96\n",
      "-- checking probes in region: 1040\n",
      "--- gene=1040, id=223 removed by soft count = 63\n",
      "--- number of probes: 350 , kept by blast: 349 , if remove dups: 349\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1041\n",
      "--- gene=1041, id=70 removed by soft count = 1141\n",
      "--- gene=1041, id=73 removed by soft count = 203\n",
      "--- gene=1041, id=233 removed by soft count = 466\n",
      "--- number of probes: 254 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1042\n",
      "--- gene=1042, id=64 removed by soft count = 58\n",
      "--- gene=1042, id=65 removed by soft count = 94\n",
      "--- gene=1042, id=100 removed by soft count = 56\n",
      "--- gene=1042, id=107 removed by soft count = 130\n",
      "--- gene=1042, id=108 removed by soft count = 159\n",
      "--- gene=1042, id=311 removed by soft count = 233\n",
      "--- number of probes: 325 , kept by blast: 319 , if remove dups: 319\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1043\n",
      "--- gene=1043, id=47 removed by soft count = 83\n",
      "--- gene=1043, id=53 removed by soft count = 259\n",
      "--- gene=1043, id=60 removed by soft count = 439\n",
      "--- gene=1043, id=61 removed by soft count = 71\n",
      "--- gene=1043, id=65 removed by soft count = 899\n",
      "--- gene=1043, id=68 removed by soft count = 60\n",
      "--- gene=1043, id=84 removed by soft count = 15390\n",
      "--- gene=1043, id=98 removed by soft count = 324\n",
      "--- gene=1043, id=99 removed by soft count = 77\n",
      "--- gene=1043, id=100 removed by soft count = 403\n",
      "--- gene=1043, id=102 removed by soft count = 383\n",
      "--- gene=1043, id=106 removed by soft count = 856\n",
      "--- gene=1043, id=110 removed by soft count = 60\n",
      "--- gene=1043, id=112 removed by soft count = 88\n",
      "--- gene=1043, id=113 removed by soft count = 111\n",
      "--- gene=1043, id=118 removed by soft count = 75\n",
      "--- gene=1043, id=120 removed by soft count = 768\n",
      "--- number of probes: 122 , kept by blast: 105 , if remove dups: 105\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 105\n",
      "-- checking probes in region: 1044\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1044, id=35 removed by soft count = 301\n",
      "--- gene=1044, id=74 removed by hard count = 0\n",
      "--- gene=1044, id=246 removed by soft count = 72\n",
      "--- gene=1044, id=248 removed by soft count = 67\n",
      "--- number of probes: 253 , kept by blast: 249 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1045\n",
      "--- number of probes: 347 , kept by blast: 347 , if remove dups: 347\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1046\n",
      "--- gene=1046, id=216 removed by soft count = 60\n",
      "--- gene=1046, id=217 removed by soft count = 216\n",
      "--- gene=1046, id=218 removed by soft count = 157\n",
      "--- number of probes: 263 , kept by blast: 260 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1047\n",
      "--- gene=1047, id=27 removed by soft count = 133\n",
      "--- number of probes: 350 , kept by blast: 349 , if remove dups: 349\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1048\n",
      "--- gene=1048, id=0 removed by soft count = 369\n",
      "--- gene=1048, id=16 removed by soft count = 1010\n",
      "--- gene=1048, id=53 removed by soft count = 53\n",
      "--- gene=1048, id=71 removed by soft count = 52\n",
      "--- gene=1048, id=72 removed by soft count = 54\n",
      "--- gene=1048, id=80 removed by soft count = 60\n",
      "--- gene=1048, id=81 removed by soft count = 58\n",
      "--- gene=1048, id=84 removed by soft count = 72\n",
      "--- gene=1048, id=88 removed by soft count = 62\n",
      "--- gene=1048, id=91 removed by soft count = 82\n",
      "--- gene=1048, id=93 removed by soft count = 62\n",
      "--- gene=1048, id=108 removed by soft count = 52\n",
      "--- gene=1048, id=119 removed by soft count = 51\n",
      "--- gene=1048, id=134 removed by soft count = 61\n",
      "--- gene=1048, id=135 removed by soft count = 55\n",
      "--- gene=1048, id=136 removed by soft count = 99\n",
      "--- gene=1048, id=137 removed by soft count = 247\n",
      "--- gene=1048, id=140 removed by soft count = 52\n",
      "--- gene=1048, id=142 removed by soft count = 63\n",
      "--- gene=1048, id=143 removed by soft count = 57\n",
      "--- gene=1048, id=147 removed by soft count = 67\n",
      "--- gene=1048, id=148 removed by soft count = 52\n",
      "--- gene=1048, id=149 removed by soft count = 61\n",
      "--- gene=1048, id=151 removed by soft count = 68\n",
      "--- gene=1048, id=153 removed by soft count = 54\n",
      "--- gene=1048, id=157 removed by soft count = 51\n",
      "--- gene=1048, id=158 removed by soft count = 53\n",
      "--- gene=1048, id=163 removed by soft count = 59\n",
      "--- gene=1048, id=164 removed by soft count = 75\n",
      "--- gene=1048, id=185 removed by soft count = 69\n",
      "--- gene=1048, id=187 removed by soft count = 51\n",
      "--- gene=1048, id=191 removed by soft count = 69\n",
      "--- gene=1048, id=193 removed by soft count = 70\n",
      "--- gene=1048, id=204 removed by soft count = 60\n",
      "--- gene=1048, id=208 removed by soft count = 56\n",
      "--- gene=1048, id=212 removed by soft count = 196\n",
      "--- gene=1048, id=227 removed by soft count = 60\n",
      "--- gene=1048, id=232 removed by soft count = 60\n",
      "--- gene=1048, id=233 removed by soft count = 59\n",
      "--- gene=1048, id=235 removed by soft count = 70\n",
      "--- gene=1048, id=238 removed by soft count = 59\n",
      "--- gene=1048, id=239 removed by soft count = 67\n",
      "--- gene=1048, id=249 removed by soft count = 59\n",
      "--- gene=1048, id=251 removed by soft count = 74\n",
      "--- gene=1048, id=255 removed by soft count = 76\n",
      "--- gene=1048, id=256 removed by soft count = 61\n",
      "--- gene=1048, id=258 removed by soft count = 66\n",
      "--- gene=1048, id=263 removed by soft count = 53\n",
      "--- gene=1048, id=270 removed by soft count = 56\n",
      "--- gene=1048, id=271 removed by soft count = 57\n",
      "--- number of probes: 292 , kept by blast: 242 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1049\n",
      "--- gene=1049, id=31 removed by soft count = 463\n",
      "--- gene=1049, id=39 removed by soft count = 159\n",
      "--- gene=1049, id=41 removed by soft count = 154\n",
      "--- gene=1049, id=45 removed by soft count = 184\n",
      "--- gene=1049, id=50 removed by soft count = 181\n",
      "--- gene=1049, id=85 removed by soft count = 125\n",
      "--- gene=1049, id=175 removed by soft count = 600\n",
      "--- number of probes: 176 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 105\n",
      "--- gene=105, id=136 removed by soft count = 73\n",
      "--- gene=105, id=243 removed by soft count = 72\n",
      "--- gene=105, id=248 removed by soft count = 90\n",
      "--- gene=105, id=254 removed by soft count = 145\n",
      "--- gene=105, id=255 removed by soft count = 99\n",
      "--- gene=105, id=259 removed by soft count = 76\n",
      "--- gene=105, id=260 removed by soft count = 60\n",
      "--- gene=105, id=262 removed by soft count = 51\n",
      "--- number of probes: 352 , kept by blast: 344 , if remove dups: 344\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1050\n",
      "--- gene=1050, id=24 removed by soft count = 51\n",
      "--- gene=1050, id=28 removed by soft count = 51\n",
      "--- number of probes: 267 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1051\n",
      "--- gene=1051, id=36 removed by soft count = 137\n",
      "--- gene=1051, id=60 removed by soft count = 430\n",
      "--- gene=1051, id=302 removed by soft count = 1323\n",
      "--- number of probes: 384 , kept by blast: 381 , if remove dups: 381\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1052\n",
      "--- gene=1052, id=11 removed by soft count = 88\n",
      "--- gene=1052, id=13 removed by soft count = 498\n",
      "--- gene=1052, id=17 removed by soft count = 1851\n",
      "--- gene=1052, id=32 removed by soft count = 98\n",
      "--- gene=1052, id=44 removed by soft count = 54\n",
      "--- gene=1052, id=45 removed by soft count = 79\n",
      "--- gene=1052, id=47 removed by soft count = 79\n",
      "--- gene=1052, id=49 removed by soft count = 74\n",
      "--- gene=1052, id=52 removed by soft count = 705\n",
      "--- gene=1052, id=84 removed by soft count = 340\n",
      "--- gene=1052, id=113 removed by soft count = 59\n",
      "--- number of probes: 135 , kept by blast: 124 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1053\n",
      "--- number of probes: 351 , kept by blast: 351 , if remove dups: 351\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1054\n",
      "--- gene=1054, id=115 removed by soft count = 74\n",
      "--- gene=1054, id=277 removed by soft count = 329\n",
      "--- gene=1054, id=278 removed by soft count = 118\n",
      "--- gene=1054, id=279 removed by soft count = 52\n",
      "--- number of probes: 320 , kept by blast: 316 , if remove dups: 316\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1055\n",
      "--- gene=1055, id=73 removed by soft count = 1593\n",
      "--- gene=1055, id=74 removed by soft count = 170\n",
      "--- gene=1055, id=77 removed by soft count = 544\n",
      "--- gene=1055, id=83 removed by soft count = 86\n",
      "--- gene=1055, id=85 removed by soft count = 290\n",
      "--- gene=1055, id=158 removed by soft count = 51\n",
      "--- gene=1055, id=265 removed by soft count = 147\n",
      "--- number of probes: 268 , kept by blast: 261 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1056\n",
      "--- gene=1056, id=137 removed by soft count = 80\n",
      "--- gene=1056, id=151 removed by soft count = 79\n",
      "--- gene=1056, id=152 removed by soft count = 75\n",
      "--- gene=1056, id=153 removed by soft count = 92\n",
      "--- gene=1056, id=154 removed by soft count = 73\n",
      "--- number of probes: 361 , kept by blast: 356 , if remove dups: 356\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1057\n",
      "--- gene=1057, id=0 removed by soft count = 144\n",
      "--- gene=1057, id=57 removed by soft count = 335\n",
      "--- gene=1057, id=62 removed by soft count = 56\n",
      "--- gene=1057, id=64 removed by soft count = 69\n",
      "--- gene=1057, id=70 removed by soft count = 94\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1057, id=74 removed by soft count = 172\n",
      "--- gene=1057, id=106 removed by soft count = 759\n",
      "--- number of probes: 155 , kept by blast: 148 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1058\n",
      "--- gene=1058, id=1 removed by soft count = 509\n",
      "--- gene=1058, id=2 removed by soft count = 203\n",
      "--- gene=1058, id=36 removed by soft count = 76\n",
      "--- gene=1058, id=62 removed by soft count = 106\n",
      "--- gene=1058, id=69 removed by soft count = 134\n",
      "--- gene=1058, id=86 removed by soft count = 82\n",
      "--- gene=1058, id=87 removed by soft count = 167\n",
      "--- number of probes: 131 , kept by blast: 124 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1059\n",
      "--- gene=1059, id=2 removed by soft count = 56\n",
      "--- gene=1059, id=3 removed by soft count = 62\n",
      "--- gene=1059, id=4 removed by soft count = 79\n",
      "--- gene=1059, id=5 removed by soft count = 65\n",
      "--- gene=1059, id=16 removed by soft count = 51\n",
      "--- number of probes: 164 , kept by blast: 159 , if remove dups: 154\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 106\n",
      "--- gene=106, id=179 removed by soft count = 243\n",
      "--- gene=106, id=183 removed by soft count = 501\n",
      "--- gene=106, id=240 removed by soft count = 273\n",
      "--- number of probes: 244 , kept by blast: 241 , if remove dups: 241\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1060\n",
      "--- gene=1060, id=21 removed by soft count = 65\n",
      "--- gene=1060, id=24 removed by soft count = 183\n",
      "--- gene=1060, id=27 removed by soft count = 348\n",
      "--- gene=1060, id=28 removed by soft count = 195\n",
      "--- gene=1060, id=313 removed by soft count = 184\n",
      "--- number of probes: 315 , kept by blast: 310 , if remove dups: 310\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1061\n",
      "--- gene=1061, id=3 removed by soft count = 146\n",
      "--- gene=1061, id=7 removed by soft count = 2973\n",
      "--- gene=1061, id=97 removed by soft count = 567\n",
      "--- gene=1061, id=216 removed by soft count = 511\n",
      "--- gene=1061, id=242 removed by soft count = 531\n",
      "--- gene=1061, id=243 removed by soft count = 127\n",
      "--- gene=1061, id=244 removed by soft count = 71\n",
      "--- gene=1061, id=294 removed by hard count = 4\n",
      "--- number of probes: 309 , kept by blast: 301 , if remove dups: 301\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1062\n",
      "--- gene=1062, id=22 removed by soft count = 132\n",
      "--- gene=1062, id=24 removed by soft count = 656\n",
      "--- gene=1062, id=26 removed by soft count = 136\n",
      "--- gene=1062, id=43 removed by soft count = 88\n",
      "--- gene=1062, id=44 removed by soft count = 125\n",
      "--- number of probes: 240 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1063\n",
      "--- gene=1063, id=85 removed by soft count = 259\n",
      "--- gene=1063, id=186 removed by soft count = 675\n",
      "--- number of probes: 266 , kept by blast: 264 , if remove dups: 264\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1064\n",
      "--- gene=1064, id=21 removed by soft count = 128\n",
      "--- gene=1064, id=23 removed by soft count = 61\n",
      "--- gene=1064, id=305 removed by soft count = 108\n",
      "--- gene=1064, id=306 removed by soft count = 69\n",
      "--- number of probes: 327 , kept by blast: 323 , if remove dups: 323\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1065\n",
      "--- gene=1065, id=33 removed by soft count = 189\n",
      "--- gene=1065, id=92 removed by soft count = 116\n",
      "--- gene=1065, id=230 removed by soft count = 143\n",
      "--- number of probes: 318 , kept by blast: 315 , if remove dups: 315\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1066\n",
      "--- gene=1066, id=0 removed by soft count = 97\n",
      "--- gene=1066, id=2 removed by soft count = 126\n",
      "--- gene=1066, id=4 removed by soft count = 82\n",
      "--- gene=1066, id=5 removed by soft count = 95\n",
      "--- gene=1066, id=6 removed by soft count = 53\n",
      "--- gene=1066, id=174 removed by soft count = 9327\n",
      "--- number of probes: 207 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1067\n",
      "--- gene=1067, id=12 removed by soft count = 115\n",
      "--- gene=1067, id=14 removed by soft count = 240\n",
      "--- gene=1067, id=80 removed by soft count = 104\n",
      "--- number of probes: 204 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1068\n",
      "--- gene=1068, id=0 removed by soft count = 260\n",
      "--- gene=1068, id=1 removed by soft count = 122\n",
      "--- gene=1068, id=4 removed by soft count = 120\n",
      "--- gene=1068, id=10 removed by soft count = 584\n",
      "--- gene=1068, id=11 removed by soft count = 292\n",
      "--- gene=1068, id=12 removed by soft count = 103\n",
      "--- gene=1068, id=13 removed by soft count = 3098\n",
      "--- gene=1068, id=15 removed by soft count = 99\n",
      "--- gene=1068, id=16 removed by soft count = 66\n",
      "--- gene=1068, id=17 removed by soft count = 288\n",
      "--- gene=1068, id=18 removed by soft count = 124\n",
      "--- gene=1068, id=19 removed by soft count = 107\n",
      "--- gene=1068, id=25 removed by soft count = 65\n",
      "--- gene=1068, id=106 removed by soft count = 906\n",
      "--- gene=1068, id=112 removed by soft count = 538\n",
      "--- gene=1068, id=126 removed by soft count = 667\n",
      "--- gene=1068, id=127 removed by soft count = 896\n",
      "--- gene=1068, id=128 removed by soft count = 59\n",
      "--- number of probes: 147 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1069\n",
      "--- gene=1069, id=0 removed by soft count = 514\n",
      "--- gene=1069, id=2 removed by soft count = 90\n",
      "--- gene=1069, id=9 removed by soft count = 59\n",
      "--- gene=1069, id=10 removed by soft count = 57\n",
      "--- gene=1069, id=12 removed by soft count = 140\n",
      "--- gene=1069, id=15 removed by soft count = 61\n",
      "--- gene=1069, id=17 removed by soft count = 125\n",
      "--- gene=1069, id=18 removed by soft count = 55\n",
      "--- gene=1069, id=20 removed by soft count = 160\n",
      "--- gene=1069, id=49 removed by soft count = 183\n",
      "--- gene=1069, id=56 removed by soft count = 82\n",
      "--- gene=1069, id=58 removed by soft count = 106\n",
      "--- gene=1069, id=59 removed by soft count = 306\n",
      "--- gene=1069, id=61 removed by soft count = 522\n",
      "--- gene=1069, id=63 removed by soft count = 5687\n",
      "--- gene=1069, id=64 removed by soft count = 309\n",
      "--- gene=1069, id=65 removed by soft count = 280\n",
      "--- gene=1069, id=66 removed by soft count = 674\n",
      "--- gene=1069, id=71 removed by soft count = 56\n",
      "--- gene=1069, id=139 removed by soft count = 391\n",
      "--- number of probes: 160 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 107\n",
      "--- gene=107, id=4 removed by soft count = 62\n",
      "--- gene=107, id=41 removed by soft count = 3348\n",
      "--- gene=107, id=42 removed by soft count = 284\n",
      "--- number of probes: 305 , kept by blast: 302 , if remove dups: 302\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1070\n",
      "--- gene=1070, id=62 removed by soft count = 598\n",
      "--- gene=1070, id=63 removed by soft count = 278\n",
      "--- gene=1070, id=64 removed by soft count = 227\n",
      "--- gene=1070, id=74 removed by soft count = 609\n",
      "--- gene=1070, id=75 removed by soft count = 1061\n",
      "--- gene=1070, id=210 removed by soft count = 149\n",
      "--- gene=1070, id=238 removed by soft count = 196\n",
      "--- number of probes: 250 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1071\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1071, id=13 removed by soft count = 63\n",
      "--- gene=1071, id=28 removed by soft count = 51\n",
      "--- gene=1071, id=194 removed by soft count = 51\n",
      "--- gene=1071, id=196 removed by soft count = 359\n",
      "--- gene=1071, id=238 removed by soft count = 128\n",
      "--- number of probes: 291 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1072\n",
      "--- gene=1072, id=74 removed by soft count = 97\n",
      "--- gene=1072, id=76 removed by soft count = 80\n",
      "--- gene=1072, id=82 removed by soft count = 685\n",
      "--- gene=1072, id=84 removed by soft count = 124\n",
      "--- gene=1072, id=86 removed by soft count = 114\n",
      "--- gene=1072, id=87 removed by soft count = 94\n",
      "--- gene=1072, id=256 removed by soft count = 52\n",
      "--- number of probes: 259 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1073\n",
      "--- gene=1073, id=81 removed by soft count = 135\n",
      "--- gene=1073, id=121 removed by soft count = 95\n",
      "--- gene=1073, id=123 removed by soft count = 69\n",
      "--- gene=1073, id=125 removed by soft count = 68\n",
      "--- gene=1073, id=146 removed by soft count = 162\n",
      "--- gene=1073, id=168 removed by soft count = 334\n",
      "--- gene=1073, id=261 removed by soft count = 129\n",
      "--- gene=1073, id=262 removed by soft count = 60\n",
      "--- number of probes: 263 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1074\n",
      "--- gene=1074, id=107 removed by soft count = 289\n",
      "--- gene=1074, id=111 removed by soft count = 101\n",
      "--- gene=1074, id=155 removed by soft count = 123\n",
      "--- gene=1074, id=163 removed by soft count = 109\n",
      "--- gene=1074, id=168 removed by soft count = 70\n",
      "--- number of probes: 266 , kept by blast: 261 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1075\n",
      "--- gene=1075, id=171 removed by soft count = 470\n",
      "--- gene=1075, id=221 removed by soft count = 69\n",
      "--- number of probes: 256 , kept by blast: 254 , if remove dups: 254\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1076\n",
      "--- gene=1076, id=38 removed by soft count = 186\n",
      "--- gene=1076, id=53 removed by soft count = 290\n",
      "--- gene=1076, id=239 removed by soft count = 1888\n",
      "--- number of probes: 293 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1077\n",
      "--- gene=1077, id=24 removed by soft count = 58\n",
      "--- gene=1077, id=76 removed by soft count = 154\n",
      "--- number of probes: 88 , kept by blast: 86 , if remove dups: 85\n",
      "-- number of probes kept for this region: 86\n",
      "-- checking probes in region: 1078\n",
      "--- gene=1078, id=0 removed by soft count = 68\n",
      "--- gene=1078, id=38 removed by soft count = 72\n",
      "--- gene=1078, id=56 removed by soft count = 82\n",
      "--- gene=1078, id=65 removed by soft count = 123\n",
      "--- gene=1078, id=166 removed by soft count = 58\n",
      "--- number of probes: 241 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1079\n",
      "--- gene=1079, id=62 removed by soft count = 95\n",
      "--- gene=1079, id=97 removed by soft count = 377\n",
      "--- gene=1079, id=99 removed by soft count = 696\n",
      "--- gene=1079, id=100 removed by soft count = 59\n",
      "--- gene=1079, id=187 removed by soft count = 175\n",
      "--- gene=1079, id=206 removed by soft count = 558\n",
      "--- gene=1079, id=244 removed by soft count = 54\n",
      "--- gene=1079, id=248 removed by soft count = 129\n",
      "--- gene=1079, id=249 removed by soft count = 77\n",
      "--- gene=1079, id=298 removed by soft count = 275\n",
      "--- gene=1079, id=299 removed by soft count = 70\n",
      "--- number of probes: 302 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 108\n",
      "--- gene=108, id=180 removed by soft count = 283\n",
      "--- gene=108, id=181 removed by soft count = 192\n",
      "--- number of probes: 213 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1080\n",
      "--- gene=1080, id=86 removed by soft count = 62\n",
      "--- gene=1080, id=207 removed by soft count = 51\n",
      "--- gene=1080, id=209 removed by soft count = 82\n",
      "--- number of probes: 362 , kept by blast: 359 , if remove dups: 359\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1081\n",
      "--- gene=1081, id=20 removed by soft count = 66\n",
      "--- number of probes: 298 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1082\n",
      "--- gene=1082, id=74 removed by soft count = 68\n",
      "--- gene=1082, id=94 removed by soft count = 296\n",
      "--- gene=1082, id=105 removed by soft count = 2737\n",
      "--- gene=1082, id=107 removed by soft count = 190\n",
      "--- gene=1082, id=121 removed by soft count = 782\n",
      "--- gene=1082, id=123 removed by soft count = 61\n",
      "--- gene=1082, id=250 removed by soft count = 58\n",
      "--- gene=1082, id=251 removed by soft count = 120\n",
      "--- number of probes: 263 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1083\n",
      "--- gene=1083, id=248 removed by soft count = 132\n",
      "--- number of probes: 325 , kept by blast: 324 , if remove dups: 324\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1085\n",
      "--- gene=1085, id=76 removed by soft count = 1264\n",
      "--- gene=1085, id=150 removed by soft count = 390\n",
      "--- gene=1085, id=227 removed by soft count = 65\n",
      "--- gene=1085, id=291 removed by soft count = 189\n",
      "--- number of probes: 341 , kept by blast: 337 , if remove dups: 337\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1087\n",
      "--- gene=1087, id=252 removed by soft count = 322\n",
      "--- gene=1087, id=292 removed by soft count = 57\n",
      "--- gene=1087, id=324 removed by soft count = 59\n",
      "--- number of probes: 336 , kept by blast: 333 , if remove dups: 333\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1089\n",
      "--- gene=1089, id=6 removed by soft count = 282\n",
      "--- gene=1089, id=48 removed by soft count = 61\n",
      "--- gene=1089, id=51 removed by soft count = 54\n",
      "--- gene=1089, id=88 removed by soft count = 51\n",
      "--- gene=1089, id=93 removed by soft count = 845\n",
      "--- gene=1089, id=112 removed by soft count = 196\n",
      "--- gene=1089, id=113 removed by soft count = 88\n",
      "--- gene=1089, id=119 removed by soft count = 91\n",
      "--- gene=1089, id=155 removed by soft count = 120\n",
      "--- gene=1089, id=205 removed by soft count = 71\n",
      "--- number of probes: 207 , kept by blast: 197 , if remove dups: 197\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 109\n",
      "--- gene=109, id=175 removed by soft count = 74\n",
      "--- gene=109, id=176 removed by soft count = 93\n",
      "--- gene=109, id=179 removed by soft count = 91\n",
      "--- gene=109, id=181 removed by soft count = 649\n",
      "--- gene=109, id=186 removed by soft count = 104\n",
      "--- number of probes: 225 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1090\n",
      "--- gene=1090, id=27 removed by soft count = 57\n",
      "--- gene=1090, id=34 removed by soft count = 54\n",
      "--- gene=1090, id=57 removed by soft count = 54\n",
      "--- number of probes: 261 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1091\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1091, id=34 removed by soft count = 107\n",
      "--- gene=1091, id=39 removed by soft count = 127\n",
      "--- gene=1091, id=41 removed by soft count = 396\n",
      "--- gene=1091, id=43 removed by soft count = 203\n",
      "--- gene=1091, id=44 removed by soft count = 640\n",
      "--- gene=1091, id=45 removed by soft count = 385\n",
      "--- gene=1091, id=47 removed by soft count = 158\n",
      "--- gene=1091, id=50 removed by soft count = 162\n",
      "--- gene=1091, id=65 removed by soft count = 544\n",
      "--- gene=1091, id=122 removed by soft count = 142\n",
      "--- gene=1091, id=143 removed by soft count = 78\n",
      "--- gene=1091, id=147 removed by soft count = 606\n",
      "--- gene=1091, id=160 removed by soft count = 82\n",
      "--- gene=1091, id=204 removed by soft count = 355\n",
      "--- gene=1091, id=205 removed by soft count = 816\n",
      "--- number of probes: 226 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1092\n",
      "--- gene=1092, id=25 removed by soft count = 4840\n",
      "--- gene=1092, id=285 removed by soft count = 70\n",
      "--- number of probes: 355 , kept by blast: 353 , if remove dups: 353\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1093\n",
      "--- gene=1093, id=106 removed by soft count = 96\n",
      "--- gene=1093, id=189 removed by soft count = 116\n",
      "--- gene=1093, id=192 removed by soft count = 143\n",
      "--- gene=1093, id=194 removed by soft count = 93\n",
      "--- number of probes: 203 , kept by blast: 199 , if remove dups: 195\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1094\n",
      "--- number of probes: 317 , kept by blast: 317 , if remove dups: 317\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1097\n",
      "--- gene=1097, id=15 removed by soft count = 74\n",
      "--- gene=1097, id=39 removed by soft count = 82\n",
      "--- gene=1097, id=40 removed by soft count = 95\n",
      "--- gene=1097, id=183 removed by soft count = 565\n",
      "--- gene=1097, id=208 removed by soft count = 256\n",
      "--- gene=1097, id=216 removed by soft count = 188\n",
      "--- gene=1097, id=217 removed by soft count = 122\n",
      "--- gene=1097, id=219 removed by soft count = 149\n",
      "--- number of probes: 230 , kept by blast: 222 , if remove dups: 222\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 11\n",
      "--- gene=11, id=185 removed by soft count = 525\n",
      "--- gene=11, id=198 removed by soft count = 1270\n",
      "--- gene=11, id=199 removed by soft count = 382\n",
      "--- gene=11, id=200 removed by soft count = 76\n",
      "--- gene=11, id=201 removed by soft count = 197\n",
      "--- number of probes: 212 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 110\n",
      "--- gene=110, id=43 removed by soft count = 3166\n",
      "--- gene=110, id=176 removed by soft count = 226\n",
      "--- gene=110, id=197 removed by soft count = 385\n",
      "--- gene=110, id=200 removed by soft count = 447\n",
      "--- gene=110, id=228 removed by soft count = 1155\n",
      "--- number of probes: 230 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1105\n",
      "--- gene=1105, id=37 removed by soft count = 82\n",
      "--- gene=1105, id=47 removed by soft count = 74\n",
      "--- gene=1105, id=48 removed by soft count = 238\n",
      "--- gene=1105, id=58 removed by soft count = 95\n",
      "--- gene=1105, id=112 removed by soft count = 142\n",
      "--- gene=1105, id=157 removed by soft count = 51\n",
      "--- number of probes: 223 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1106\n",
      "--- gene=1106, id=167 removed by soft count = 79\n",
      "--- gene=1106, id=168 removed by soft count = 420\n",
      "--- gene=1106, id=169 removed by soft count = 77\n",
      "--- gene=1106, id=173 removed by soft count = 440\n",
      "--- gene=1106, id=174 removed by soft count = 55\n",
      "--- gene=1106, id=176 removed by soft count = 138\n",
      "--- gene=1106, id=181 removed by soft count = 174\n",
      "--- gene=1106, id=185 removed by soft count = 4057\n",
      "--- gene=1106, id=186 removed by soft count = 174\n",
      "--- gene=1106, id=187 removed by soft count = 2824\n",
      "--- gene=1106, id=188 removed by soft count = 25737\n",
      "--- number of probes: 189 , kept by blast: 178 , if remove dups: 178\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1107\n",
      "--- number of probes: 267 , kept by blast: 267 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1108\n",
      "--- gene=1108, id=75 removed by soft count = 112\n",
      "--- gene=1108, id=76 removed by soft count = 73\n",
      "--- gene=1108, id=77 removed by soft count = 144\n",
      "--- number of probes: 295 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1109\n",
      "--- gene=1109, id=157 removed by soft count = 128\n",
      "--- number of probes: 184 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 111\n",
      "--- gene=111, id=98 removed by soft count = 218\n",
      "--- gene=111, id=288 removed by soft count = 54\n",
      "--- number of probes: 358 , kept by blast: 356 , if remove dups: 356\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1110\n",
      "--- number of probes: 336 , kept by blast: 336 , if remove dups: 336\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1111\n",
      "--- gene=1111, id=94 removed by soft count = 289\n",
      "--- gene=1111, id=95 removed by soft count = 58\n",
      "--- number of probes: 381 , kept by blast: 379 , if remove dups: 379\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1112\n",
      "--- gene=1112, id=150 removed by hard count = 3\n",
      "--- gene=1112, id=164 removed by soft count = 80\n",
      "--- number of probes: 398 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1113\n",
      "--- gene=1113, id=14 removed by soft count = 73\n",
      "--- gene=1113, id=120 removed by soft count = 149\n",
      "--- gene=1113, id=213 removed by soft count = 114\n",
      "--- gene=1113, id=230 removed by soft count = 206\n",
      "--- gene=1113, id=232 removed by soft count = 664\n",
      "--- number of probes: 266 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1114\n",
      "--- gene=1114, id=8 removed by soft count = 51\n",
      "--- gene=1114, id=9 removed by soft count = 72\n",
      "--- gene=1114, id=79 removed by soft count = 186\n",
      "--- gene=1114, id=106 removed by soft count = 108\n",
      "--- gene=1114, id=188 removed by soft count = 388\n",
      "--- number of probes: 215 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1115\n",
      "--- gene=1115, id=24 removed by soft count = 92\n",
      "--- gene=1115, id=62 removed by soft count = 120\n",
      "--- gene=1115, id=68 removed by soft count = 240\n",
      "--- gene=1115, id=83 removed by soft count = 123\n",
      "--- gene=1115, id=118 removed by soft count = 96\n",
      "--- gene=1115, id=201 removed by soft count = 130\n",
      "--- gene=1115, id=215 removed by soft count = 1139\n",
      "--- number of probes: 253 , kept by blast: 246 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1116\n",
      "--- gene=1116, id=36 removed by soft count = 73\n",
      "--- gene=1116, id=47 removed by soft count = 59\n",
      "--- gene=1116, id=56 removed by soft count = 60\n",
      "--- gene=1116, id=75 removed by soft count = 62\n",
      "--- gene=1116, id=77 removed by soft count = 84\n",
      "--- gene=1116, id=78 removed by soft count = 73\n",
      "--- gene=1116, id=79 removed by soft count = 545\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1116, id=93 removed by soft count = 102\n",
      "--- gene=1116, id=116 removed by soft count = 185\n",
      "--- gene=1116, id=117 removed by soft count = 498\n",
      "--- gene=1116, id=123 removed by soft count = 235\n",
      "--- gene=1116, id=125 removed by soft count = 192\n",
      "--- gene=1116, id=145 removed by soft count = 246\n",
      "--- gene=1116, id=146 removed by soft count = 59\n",
      "--- gene=1116, id=148 removed by soft count = 60\n",
      "--- gene=1116, id=149 removed by soft count = 290\n",
      "--- gene=1116, id=150 removed by soft count = 53\n",
      "--- gene=1116, id=157 removed by soft count = 8907\n",
      "--- gene=1116, id=158 removed by soft count = 89\n",
      "--- gene=1116, id=159 removed by soft count = 75\n",
      "--- gene=1116, id=161 removed by soft count = 60\n",
      "--- gene=1116, id=162 removed by soft count = 81\n",
      "--- number of probes: 184 , kept by blast: 162 , if remove dups: 162\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1117\n",
      "--- gene=1117, id=69 removed by soft count = 57\n",
      "--- gene=1117, id=265 removed by soft count = 340\n",
      "--- number of probes: 288 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1118\n",
      "--- gene=1118, id=94 removed by soft count = 82\n",
      "--- gene=1118, id=171 removed by soft count = 138\n",
      "--- gene=1118, id=185 removed by soft count = 91\n",
      "--- number of probes: 231 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1119\n",
      "--- gene=1119, id=80 removed by soft count = 136\n",
      "--- gene=1119, id=225 removed by soft count = 80\n",
      "--- gene=1119, id=228 removed by soft count = 107\n",
      "--- number of probes: 328 , kept by blast: 325 , if remove dups: 325\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 112\n",
      "--- gene=112, id=3 removed by soft count = 82\n",
      "--- gene=112, id=7 removed by soft count = 70\n",
      "--- gene=112, id=9 removed by soft count = 55\n",
      "--- gene=112, id=15 removed by soft count = 118\n",
      "--- gene=112, id=16 removed by soft count = 2980\n",
      "--- gene=112, id=17 removed by soft count = 1739\n",
      "--- gene=112, id=19 removed by soft count = 282\n",
      "--- gene=112, id=20 removed by soft count = 66\n",
      "--- gene=112, id=22 removed by soft count = 193\n",
      "--- gene=112, id=24 removed by soft count = 148\n",
      "--- gene=112, id=26 removed by soft count = 420\n",
      "--- gene=112, id=27 removed by soft count = 149\n",
      "--- gene=112, id=29 removed by soft count = 187\n",
      "--- gene=112, id=121 removed by soft count = 88\n",
      "--- gene=112, id=123 removed by soft count = 146\n",
      "--- gene=112, id=138 removed by soft count = 1289\n",
      "--- number of probes: 167 , kept by blast: 151 , if remove dups: 151\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1120\n",
      "--- gene=1120, id=1 removed by soft count = 61\n",
      "--- gene=1120, id=107 removed by soft count = 1154\n",
      "--- gene=1120, id=183 removed by soft count = 1288\n",
      "--- gene=1120, id=190 removed by soft count = 3716\n",
      "--- gene=1120, id=193 removed by soft count = 51\n",
      "--- number of probes: 228 , kept by blast: 223 , if remove dups: 222\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1121\n",
      "--- gene=1121, id=20 removed by soft count = 111\n",
      "--- gene=1121, id=23 removed by soft count = 55\n",
      "--- gene=1121, id=156 removed by soft count = 54\n",
      "--- gene=1121, id=197 removed by soft count = 265\n",
      "--- gene=1121, id=256 removed by soft count = 56\n",
      "--- number of probes: 298 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1122\n",
      "--- gene=1122, id=1 removed by soft count = 89\n",
      "--- gene=1122, id=2 removed by soft count = 157\n",
      "--- gene=1122, id=72 removed by soft count = 56\n",
      "--- gene=1122, id=93 removed by soft count = 168\n",
      "--- gene=1122, id=139 removed by soft count = 666\n",
      "--- gene=1122, id=162 removed by soft count = 218\n",
      "--- number of probes: 227 , kept by blast: 221 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1123\n",
      "--- gene=1123, id=18 removed by soft count = 80\n",
      "--- gene=1123, id=22 removed by soft count = 67\n",
      "--- gene=1123, id=55 removed by soft count = 99\n",
      "--- gene=1123, id=56 removed by soft count = 93\n",
      "--- gene=1123, id=107 removed by soft count = 93\n",
      "--- gene=1123, id=108 removed by soft count = 121\n",
      "--- gene=1123, id=110 removed by soft count = 86\n",
      "--- gene=1123, id=230 removed by soft count = 159\n",
      "--- gene=1123, id=237 removed by soft count = 60\n",
      "--- gene=1123, id=244 removed by soft count = 227\n",
      "--- gene=1123, id=249 removed by soft count = 116\n",
      "--- number of probes: 267 , kept by blast: 256 , if remove dups: 256\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1124\n",
      "--- gene=1124, id=0 removed by soft count = 906\n",
      "--- gene=1124, id=3 removed by soft count = 141\n",
      "--- gene=1124, id=8 removed by soft count = 69\n",
      "--- gene=1124, id=44 removed by soft count = 198\n",
      "--- number of probes: 201 , kept by blast: 197 , if remove dups: 197\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1125\n",
      "--- gene=1125, id=0 removed by soft count = 62\n",
      "--- gene=1125, id=71 removed by soft count = 217\n",
      "--- gene=1125, id=172 removed by soft count = 117\n",
      "--- gene=1125, id=173 removed by soft count = 55\n",
      "--- gene=1125, id=174 removed by soft count = 246\n",
      "--- gene=1125, id=175 removed by soft count = 2410\n",
      "--- gene=1125, id=176 removed by soft count = 14407\n",
      "--- gene=1125, id=179 removed by soft count = 75\n",
      "--- gene=1125, id=180 removed by soft count = 161\n",
      "--- gene=1125, id=181 removed by soft count = 355\n",
      "--- number of probes: 208 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1126\n",
      "--- gene=1126, id=18 removed by soft count = 3677\n",
      "--- gene=1126, id=93 removed by soft count = 242\n",
      "--- gene=1126, id=94 removed by soft count = 112\n",
      "--- gene=1126, id=95 removed by soft count = 69\n",
      "--- gene=1126, id=96 removed by soft count = 150\n",
      "--- gene=1126, id=173 removed by soft count = 388\n",
      "--- gene=1126, id=240 removed by soft count = 173\n",
      "--- number of probes: 320 , kept by blast: 313 , if remove dups: 313\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1127\n",
      "--- gene=1127, id=32 removed by soft count = 87\n",
      "--- gene=1127, id=130 removed by soft count = 657\n",
      "--- gene=1127, id=151 removed by soft count = 84\n",
      "--- number of probes: 241 , kept by blast: 238 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1128\n",
      "--- gene=1128, id=14 removed by soft count = 153\n",
      "--- gene=1128, id=17 removed by soft count = 231\n",
      "--- gene=1128, id=72 removed by soft count = 12558\n",
      "--- gene=1128, id=139 removed by soft count = 135\n",
      "--- gene=1128, id=171 removed by soft count = 62\n",
      "--- gene=1128, id=197 removed by soft count = 79\n",
      "--- gene=1128, id=219 removed by soft count = 1481\n",
      "--- number of probes: 235 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1129\n",
      "--- gene=1129, id=0 removed by soft count = 391\n",
      "--- gene=1129, id=66 removed by soft count = 113\n",
      "--- gene=1129, id=67 removed by soft count = 139\n",
      "--- gene=1129, id=70 removed by soft count = 88\n",
      "--- number of probes: 335 , kept by blast: 331 , if remove dups: 330\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1130\n",
      "--- gene=1130, id=77 removed by soft count = 65\n",
      "--- gene=1130, id=122 removed by soft count = 116\n",
      "--- gene=1130, id=123 removed by soft count = 132\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1130, id=158 removed by soft count = 1076\n",
      "--- number of probes: 311 , kept by blast: 307 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1131\n",
      "--- gene=1131, id=170 removed by soft count = 68\n",
      "--- gene=1131, id=171 removed by soft count = 83\n",
      "--- number of probes: 272 , kept by blast: 270 , if remove dups: 269\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1132\n",
      "--- gene=1132, id=38 removed by soft count = 56\n",
      "--- gene=1132, id=39 removed by soft count = 61\n",
      "--- gene=1132, id=40 removed by soft count = 57\n",
      "--- gene=1132, id=94 removed by soft count = 88\n",
      "--- gene=1132, id=178 removed by soft count = 180\n",
      "--- number of probes: 288 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1133\n",
      "--- gene=1133, id=41 removed by soft count = 71\n",
      "--- gene=1133, id=42 removed by soft count = 998\n",
      "--- gene=1133, id=55 removed by soft count = 153\n",
      "--- gene=1133, id=69 removed by soft count = 3373\n",
      "--- gene=1133, id=110 removed by soft count = 76\n",
      "--- gene=1133, id=116 removed by soft count = 175\n",
      "--- gene=1133, id=118 removed by soft count = 179\n",
      "--- gene=1133, id=124 removed by soft count = 61\n",
      "--- gene=1133, id=130 removed by soft count = 2806\n",
      "--- gene=1133, id=157 removed by soft count = 77\n",
      "--- gene=1133, id=174 removed by soft count = 78\n",
      "--- gene=1133, id=189 removed by soft count = 65\n",
      "--- gene=1133, id=190 removed by soft count = 118\n",
      "--- gene=1133, id=198 removed by soft count = 216\n",
      "--- number of probes: 205 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1134\n",
      "--- gene=1134, id=12 removed by soft count = 1513\n",
      "--- gene=1134, id=200 removed by hard count = 3\n",
      "--- gene=1134, id=223 removed by soft count = 332\n",
      "--- gene=1134, id=235 removed by soft count = 83\n",
      "--- gene=1134, id=236 removed by soft count = 120\n",
      "--- gene=1134, id=248 removed by soft count = 2950\n",
      "--- gene=1134, id=299 removed by soft count = 105\n",
      "--- gene=1134, id=305 removed by soft count = 83\n",
      "--- number of probes: 309 , kept by blast: 301 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1135\n",
      "--- gene=1135, id=8 removed by soft count = 321\n",
      "--- gene=1135, id=37 removed by soft count = 70\n",
      "--- gene=1135, id=85 removed by soft count = 42296\n",
      "--- number of probes: 268 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1136\n",
      "--- number of probes: 290 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1137\n",
      "--- gene=1137, id=46 removed by hard count = 4\n",
      "--- gene=1137, id=47 removed by soft count = 62\n",
      "--- gene=1137, id=70 removed by soft count = 65\n",
      "--- gene=1137, id=75 removed by soft count = 56\n",
      "--- gene=1137, id=77 removed by soft count = 79\n",
      "--- gene=1137, id=81 removed by soft count = 340\n",
      "--- gene=1137, id=82 removed by soft count = 56\n",
      "--- gene=1137, id=96 removed by soft count = 293\n",
      "--- gene=1137, id=185 removed by soft count = 99\n",
      "--- gene=1137, id=187 removed by soft count = 4550\n",
      "--- number of probes: 193 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1138\n",
      "--- gene=1138, id=0 removed by soft count = 140\n",
      "--- gene=1138, id=35 removed by soft count = 698\n",
      "--- gene=1138, id=36 removed by soft count = 298\n",
      "--- gene=1138, id=37 removed by soft count = 97\n",
      "--- gene=1138, id=44 removed by soft count = 452\n",
      "--- gene=1138, id=45 removed by soft count = 696\n",
      "--- gene=1138, id=46 removed by soft count = 229\n",
      "--- gene=1138, id=63 removed by soft count = 84\n",
      "--- gene=1138, id=64 removed by soft count = 75\n",
      "--- gene=1138, id=147 removed by hard count = 3\n",
      "--- number of probes: 212 , kept by blast: 202 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1139\n",
      "--- gene=1139, id=77 removed by soft count = 238\n",
      "--- gene=1139, id=128 removed by soft count = 75\n",
      "--- gene=1139, id=209 removed by soft count = 74\n",
      "--- number of probes: 256 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 114\n",
      "--- gene=114, id=9 removed by soft count = 74\n",
      "--- gene=114, id=181 removed by soft count = 331\n",
      "--- number of probes: 215 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1140\n",
      "--- number of probes: 242 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1141\n",
      "--- gene=1141, id=156 removed by soft count = 197\n",
      "--- gene=1141, id=163 removed by soft count = 121\n",
      "--- gene=1141, id=335 removed by soft count = 56\n",
      "--- number of probes: 375 , kept by blast: 372 , if remove dups: 372\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1142\n",
      "--- gene=1142, id=54 removed by soft count = 279\n",
      "--- gene=1142, id=55 removed by soft count = 176\n",
      "--- gene=1142, id=106 removed by soft count = 64\n",
      "--- gene=1142, id=218 removed by soft count = 338\n",
      "--- number of probes: 263 , kept by blast: 259 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1143\n",
      "--- gene=1143, id=16 removed by soft count = 184\n",
      "--- gene=1143, id=88 removed by soft count = 128\n",
      "--- gene=1143, id=115 removed by soft count = 75\n",
      "--- gene=1143, id=166 removed by soft count = 102\n",
      "--- gene=1143, id=176 removed by soft count = 60\n",
      "--- gene=1143, id=194 removed by soft count = 113\n",
      "--- gene=1143, id=196 removed by soft count = 1375\n",
      "--- gene=1143, id=197 removed by soft count = 91\n",
      "--- gene=1143, id=198 removed by soft count = 155\n",
      "--- number of probes: 206 , kept by blast: 197 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1144\n",
      "--- gene=1144, id=88 removed by soft count = 236\n",
      "--- gene=1144, id=89 removed by soft count = 178\n",
      "--- number of probes: 122 , kept by blast: 120 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 117\n",
      "-- checking probes in region: 1145\n",
      "--- gene=1145, id=135 removed by soft count = 85\n",
      "--- number of probes: 215 , kept by blast: 214 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1146\n",
      "--- gene=1146, id=39 removed by soft count = 171\n",
      "--- number of probes: 94 , kept by blast: 93 , if remove dups: 92\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 92\n",
      "-- checking probes in region: 1147\n",
      "--- gene=1147, id=144 removed by soft count = 51\n",
      "--- gene=1147, id=147 removed by soft count = 247\n",
      "--- gene=1147, id=212 removed by soft count = 1134\n",
      "--- gene=1147, id=254 removed by soft count = 164\n",
      "--- gene=1147, id=255 removed by soft count = 165\n",
      "--- number of probes: 257 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1148\n",
      "--- gene=1148, id=4 removed by hard count = 3\n",
      "--- number of probes: 101 , kept by blast: 100 , if remove dups: 98\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 98\n",
      "-- checking probes in region: 115\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=115, id=6 removed by soft count = 843\n",
      "--- gene=115, id=11 removed by soft count = 99\n",
      "--- gene=115, id=101 removed by soft count = 123\n",
      "--- gene=115, id=102 removed by soft count = 53\n",
      "--- number of probes: 224 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1152\n",
      "--- gene=1152, id=0 removed by soft count = 112\n",
      "--- gene=1152, id=3 removed by soft count = 304\n",
      "--- gene=1152, id=5 removed by soft count = 353\n",
      "--- gene=1152, id=6 removed by soft count = 490\n",
      "--- gene=1152, id=9 removed by soft count = 507\n",
      "--- number of probes: 194 , kept by blast: 189 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1153\n",
      "--- gene=1153, id=72 removed by soft count = 76\n",
      "--- gene=1153, id=145 removed by soft count = 407\n",
      "--- number of probes: 269 , kept by blast: 267 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1154\n",
      "--- gene=1154, id=30 removed by soft count = 160\n",
      "--- gene=1154, id=31 removed by soft count = 160\n",
      "--- number of probes: 54 , kept by blast: 52 , if remove dups: 51\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: 1155\n",
      "--- gene=1155, id=17 removed by soft count = 234\n",
      "--- gene=1155, id=79 removed by soft count = 85\n",
      "--- number of probes: 254 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1156\n",
      "--- gene=1156, id=113 removed by soft count = 100\n",
      "--- gene=1156, id=205 removed by soft count = 265\n",
      "--- gene=1156, id=206 removed by soft count = 84\n",
      "--- gene=1156, id=207 removed by soft count = 310\n",
      "--- gene=1156, id=242 removed by soft count = 77\n",
      "--- gene=1156, id=260 removed by soft count = 113\n",
      "--- gene=1156, id=267 removed by soft count = 211\n",
      "--- gene=1156, id=322 removed by soft count = 88\n",
      "--- gene=1156, id=323 removed by soft count = 96\n",
      "--- number of probes: 327 , kept by blast: 318 , if remove dups: 318\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1157\n",
      "--- gene=1157, id=127 removed by soft count = 73\n",
      "--- gene=1157, id=157 removed by soft count = 112\n",
      "--- gene=1157, id=185 removed by soft count = 217\n",
      "--- gene=1157, id=219 removed by soft count = 2586\n",
      "--- number of probes: 276 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 116\n",
      "--- gene=116, id=1 removed by soft count = 145\n",
      "--- gene=116, id=103 removed by soft count = 524\n",
      "--- gene=116, id=104 removed by soft count = 1086\n",
      "--- number of probes: 228 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1160\n",
      "--- number of probes: 317 , kept by blast: 317 , if remove dups: 317\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1162\n",
      "--- gene=1162, id=4 removed by soft count = 108\n",
      "--- gene=1162, id=12 removed by soft count = 281\n",
      "--- gene=1162, id=155 removed by soft count = 99\n",
      "--- number of probes: 222 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1163\n",
      "--- number of probes: 272 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1165\n",
      "--- gene=1165, id=1 removed by soft count = 727\n",
      "--- gene=1165, id=16 removed by soft count = 129\n",
      "--- gene=1165, id=20 removed by soft count = 151\n",
      "--- gene=1165, id=21 removed by soft count = 110\n",
      "--- gene=1165, id=117 removed by soft count = 223\n",
      "--- gene=1165, id=118 removed by soft count = 96\n",
      "--- gene=1165, id=131 removed by soft count = 61\n",
      "--- gene=1165, id=263 removed by soft count = 58\n",
      "--- number of probes: 270 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1166\n",
      "--- gene=1166, id=27 removed by soft count = 823\n",
      "--- gene=1166, id=149 removed by soft count = 88\n",
      "--- number of probes: 342 , kept by blast: 340 , if remove dups: 340\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1167\n",
      "--- gene=1167, id=0 removed by soft count = 71\n",
      "--- gene=1167, id=1 removed by soft count = 57\n",
      "--- gene=1167, id=12 removed by soft count = 460\n",
      "--- gene=1167, id=13 removed by soft count = 125\n",
      "--- gene=1167, id=14 removed by soft count = 57\n",
      "--- gene=1167, id=15 removed by soft count = 74\n",
      "--- gene=1167, id=17 removed by soft count = 159\n",
      "--- gene=1167, id=23 removed by soft count = 57\n",
      "--- gene=1167, id=31 removed by soft count = 880\n",
      "--- gene=1167, id=40 removed by soft count = 1849\n",
      "--- gene=1167, id=53 removed by soft count = 67\n",
      "--- gene=1167, id=64 removed by soft count = 347\n",
      "--- gene=1167, id=65 removed by soft count = 1248\n",
      "--- gene=1167, id=66 removed by soft count = 193\n",
      "--- gene=1167, id=67 removed by soft count = 220\n",
      "--- gene=1167, id=71 removed by soft count = 181\n",
      "--- gene=1167, id=72 removed by soft count = 66\n",
      "--- number of probes: 192 , kept by blast: 175 , if remove dups: 175\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1168\n",
      "--- gene=1168, id=5 removed by soft count = 466\n",
      "--- number of probes: 346 , kept by blast: 345 , if remove dups: 345\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1169\n",
      "--- gene=1169, id=5 removed by soft count = 60\n",
      "--- gene=1169, id=68 removed by soft count = 76\n",
      "--- gene=1169, id=123 removed by soft count = 61\n",
      "--- gene=1169, id=140 removed by soft count = 233\n",
      "--- gene=1169, id=142 removed by soft count = 61\n",
      "--- gene=1169, id=145 removed by soft count = 263\n",
      "--- gene=1169, id=148 removed by soft count = 254\n",
      "--- gene=1169, id=150 removed by soft count = 321\n",
      "--- gene=1169, id=152 removed by soft count = 239\n",
      "--- number of probes: 185 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 117\n",
      "--- gene=117, id=100 removed by soft count = 129\n",
      "--- gene=117, id=106 removed by soft count = 674\n",
      "--- gene=117, id=209 removed by soft count = 73\n",
      "--- gene=117, id=268 removed by soft count = 142\n",
      "--- number of probes: 276 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1170\n",
      "--- gene=1170, id=0 removed by soft count = 61\n",
      "--- gene=1170, id=114 removed by soft count = 643\n",
      "--- gene=1170, id=118 removed by soft count = 105\n",
      "--- gene=1170, id=119 removed by soft count = 129\n",
      "--- gene=1170, id=121 removed by soft count = 365\n",
      "--- gene=1170, id=123 removed by soft count = 68\n",
      "--- number of probes: 232 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1171\n",
      "--- gene=1171, id=0 removed by soft count = 8357\n",
      "--- gene=1171, id=1 removed by soft count = 7962\n",
      "--- gene=1171, id=2 removed by soft count = 211\n",
      "--- gene=1171, id=3 removed by soft count = 52\n",
      "--- gene=1171, id=4 removed by soft count = 928\n",
      "--- gene=1171, id=5 removed by soft count = 448\n",
      "--- gene=1171, id=28 removed by soft count = 121\n",
      "--- gene=1171, id=30 removed by soft count = 107\n",
      "--- gene=1171, id=32 removed by soft count = 177\n",
      "--- gene=1171, id=129 removed by soft count = 4150\n",
      "--- gene=1171, id=227 removed by soft count = 167\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 268 , kept by blast: 257 , if remove dups: 257\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1172\n",
      "--- gene=1172, id=216 removed by soft count = 233\n",
      "--- number of probes: 329 , kept by blast: 328 , if remove dups: 328\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1173\n",
      "--- gene=1173, id=18 removed by soft count = 280\n",
      "--- gene=1173, id=24 removed by soft count = 158\n",
      "--- gene=1173, id=25 removed by soft count = 64\n",
      "--- gene=1173, id=26 removed by soft count = 167\n",
      "--- gene=1173, id=32 removed by soft count = 334\n",
      "--- gene=1173, id=87 removed by soft count = 141\n",
      "--- number of probes: 96 , kept by blast: 90 , if remove dups: 90\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 90\n",
      "-- checking probes in region: 1174\n",
      "--- gene=1174, id=6 removed by soft count = 89\n",
      "--- gene=1174, id=11 removed by soft count = 120\n",
      "--- gene=1174, id=15 removed by soft count = 75\n",
      "--- gene=1174, id=103 removed by soft count = 4389\n",
      "--- gene=1174, id=115 removed by soft count = 144\n",
      "--- gene=1174, id=117 removed by soft count = 3714\n",
      "--- gene=1174, id=118 removed by soft count = 184\n",
      "--- gene=1174, id=121 removed by soft count = 73\n",
      "--- gene=1174, id=122 removed by soft count = 383\n",
      "--- gene=1174, id=123 removed by soft count = 322\n",
      "--- gene=1174, id=124 removed by soft count = 58\n",
      "--- gene=1174, id=125 removed by soft count = 135\n",
      "--- gene=1174, id=127 removed by soft count = 82\n",
      "--- number of probes: 128 , kept by blast: 115 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 115\n",
      "-- checking probes in region: 1176\n",
      "--- gene=1176, id=159 removed by soft count = 56\n",
      "--- gene=1176, id=163 removed by soft count = 88\n",
      "--- number of probes: 212 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1177\n",
      "--- gene=1177, id=224 removed by soft count = 64\n",
      "--- gene=1177, id=225 removed by soft count = 540\n",
      "--- gene=1177, id=226 removed by soft count = 1267\n",
      "--- gene=1177, id=228 removed by soft count = 116\n",
      "--- number of probes: 241 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1178\n",
      "--- gene=1178, id=114 removed by soft count = 68\n",
      "--- gene=1178, id=116 removed by soft count = 65\n",
      "--- gene=1178, id=193 removed by soft count = 146\n",
      "--- gene=1178, id=250 removed by soft count = 51\n",
      "--- gene=1178, id=252 removed by soft count = 207\n",
      "--- gene=1178, id=264 removed by soft count = 55\n",
      "--- gene=1178, id=266 removed by soft count = 83\n",
      "--- number of probes: 319 , kept by blast: 312 , if remove dups: 311\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1179\n",
      "--- gene=1179, id=8 removed by soft count = 3994\n",
      "--- gene=1179, id=40 removed by soft count = 770\n",
      "--- gene=1179, id=128 removed by soft count = 88\n",
      "--- gene=1179, id=130 removed by soft count = 81\n",
      "--- number of probes: 197 , kept by blast: 193 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 118\n",
      "--- number of probes: 216 , kept by blast: 216 , if remove dups: 216\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1180\n",
      "--- gene=1180, id=6 removed by soft count = 419\n",
      "--- gene=1180, id=7 removed by soft count = 600\n",
      "--- gene=1180, id=8 removed by soft count = 255\n",
      "--- gene=1180, id=9 removed by soft count = 366\n",
      "--- gene=1180, id=10 removed by soft count = 368\n",
      "--- gene=1180, id=11 removed by soft count = 149\n",
      "--- gene=1180, id=62 removed by soft count = 61\n",
      "--- gene=1180, id=79 removed by soft count = 188\n",
      "--- number of probes: 213 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1181\n",
      "--- number of probes: 271 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1182\n",
      "--- gene=1182, id=58 removed by soft count = 55\n",
      "--- gene=1182, id=63 removed by soft count = 112\n",
      "--- gene=1182, id=101 removed by soft count = 86\n",
      "--- gene=1182, id=171 removed by soft count = 98\n",
      "--- gene=1182, id=183 removed by soft count = 403\n",
      "--- gene=1182, id=257 removed by soft count = 597\n",
      "--- number of probes: 291 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1183\n",
      "--- gene=1183, id=84 removed by soft count = 105\n",
      "--- gene=1183, id=170 removed by soft count = 139\n",
      "--- number of probes: 337 , kept by blast: 335 , if remove dups: 335\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1184\n",
      "--- gene=1184, id=39 removed by soft count = 67\n",
      "--- gene=1184, id=110 removed by soft count = 319\n",
      "--- number of probes: 310 , kept by blast: 308 , if remove dups: 308\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1186\n",
      "--- gene=1186, id=133 removed by soft count = 55\n",
      "--- gene=1186, id=141 removed by soft count = 142\n",
      "--- gene=1186, id=143 removed by soft count = 116\n",
      "--- gene=1186, id=166 removed by soft count = 129\n",
      "--- gene=1186, id=184 removed by soft count = 39141\n",
      "--- gene=1186, id=258 removed by soft count = 64\n",
      "--- number of probes: 265 , kept by blast: 259 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1187\n",
      "--- gene=1187, id=34 removed by soft count = 270\n",
      "--- gene=1187, id=43 removed by soft count = 56\n",
      "--- gene=1187, id=47 removed by soft count = 184\n",
      "--- gene=1187, id=207 removed by soft count = 87\n",
      "--- gene=1187, id=216 removed by soft count = 67\n",
      "--- number of probes: 284 , kept by blast: 279 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1188\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "-- number of probes kept for this region: 77\n",
      "-- checking probes in region: 1189\n",
      "--- number of probes: 171 , kept by blast: 171 , if remove dups: 171\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 119\n",
      "--- gene=119, id=90 removed by soft count = 142\n",
      "--- gene=119, id=142 removed by soft count = 133\n",
      "--- gene=119, id=148 removed by soft count = 75\n",
      "--- gene=119, id=150 removed by soft count = 89\n",
      "--- gene=119, id=154 removed by soft count = 117\n",
      "--- gene=119, id=155 removed by soft count = 74\n",
      "--- gene=119, id=201 removed by soft count = 90\n",
      "--- number of probes: 221 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1190\n",
      "--- gene=1190, id=28 removed by soft count = 88\n",
      "--- gene=1190, id=84 removed by soft count = 139\n",
      "--- gene=1190, id=93 removed by soft count = 347\n",
      "--- gene=1190, id=94 removed by soft count = 460\n",
      "--- gene=1190, id=96 removed by soft count = 88\n",
      "--- gene=1190, id=97 removed by soft count = 897\n",
      "--- gene=1190, id=98 removed by soft count = 166\n",
      "--- gene=1190, id=99 removed by soft count = 130\n",
      "--- gene=1190, id=101 removed by soft count = 528\n",
      "--- gene=1190, id=102 removed by soft count = 563\n",
      "--- gene=1190, id=103 removed by soft count = 265\n",
      "--- number of probes: 187 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1191\n",
      "--- gene=1191, id=39 removed by soft count = 267\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1191, id=92 removed by soft count = 633\n",
      "--- gene=1191, id=106 removed by soft count = 85\n",
      "--- gene=1191, id=107 removed by soft count = 90\n",
      "--- gene=1191, id=161 removed by soft count = 130\n",
      "--- gene=1191, id=162 removed by soft count = 61\n",
      "--- gene=1191, id=177 removed by soft count = 181\n",
      "--- gene=1191, id=178 removed by soft count = 1557\n",
      "--- number of probes: 194 , kept by blast: 186 , if remove dups: 186\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1192\n",
      "--- gene=1192, id=68 removed by soft count = 102\n",
      "--- gene=1192, id=85 removed by soft count = 52\n",
      "--- gene=1192, id=115 removed by soft count = 215\n",
      "--- gene=1192, id=116 removed by soft count = 59\n",
      "--- gene=1192, id=131 removed by soft count = 63\n",
      "--- gene=1192, id=162 removed by soft count = 88\n",
      "--- gene=1192, id=164 removed by soft count = 59\n",
      "--- gene=1192, id=183 removed by soft count = 78\n",
      "--- gene=1192, id=186 removed by soft count = 205\n",
      "--- gene=1192, id=212 removed by soft count = 69\n",
      "--- gene=1192, id=215 removed by soft count = 55\n",
      "--- gene=1192, id=217 removed by soft count = 254\n",
      "--- gene=1192, id=218 removed by soft count = 255\n",
      "--- gene=1192, id=219 removed by soft count = 116\n",
      "--- gene=1192, id=220 removed by soft count = 330\n",
      "--- gene=1192, id=221 removed by soft count = 437\n",
      "--- gene=1192, id=225 removed by soft count = 60\n",
      "--- gene=1192, id=227 removed by soft count = 89\n",
      "--- gene=1192, id=237 removed by soft count = 85\n",
      "--- gene=1192, id=238 removed by soft count = 69\n",
      "--- gene=1192, id=239 removed by soft count = 283\n",
      "--- gene=1192, id=240 removed by soft count = 560\n",
      "--- number of probes: 242 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1193\n",
      "--- gene=1193, id=85 removed by soft count = 63\n",
      "--- gene=1193, id=196 removed by soft count = 293\n",
      "--- gene=1193, id=270 removed by soft count = 123\n",
      "--- number of probes: 279 , kept by blast: 276 , if remove dups: 276\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1194\n",
      "--- gene=1194, id=0 removed by soft count = 60\n",
      "--- gene=1194, id=1 removed by soft count = 196\n",
      "--- gene=1194, id=2 removed by soft count = 78\n",
      "--- gene=1194, id=4 removed by soft count = 195\n",
      "--- gene=1194, id=8 removed by soft count = 53\n",
      "--- gene=1194, id=11 removed by soft count = 77\n",
      "--- gene=1194, id=14 removed by soft count = 55\n",
      "--- gene=1194, id=27 removed by soft count = 238\n",
      "--- gene=1194, id=238 removed by soft count = 2972\n",
      "--- number of probes: 291 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1195\n",
      "--- gene=1195, id=31 removed by soft count = 77\n",
      "--- gene=1195, id=33 removed by soft count = 346\n",
      "--- gene=1195, id=93 removed by soft count = 84\n",
      "--- gene=1195, id=95 removed by soft count = 93\n",
      "--- gene=1195, id=96 removed by soft count = 100\n",
      "--- gene=1195, id=97 removed by soft count = 130\n",
      "--- gene=1195, id=210 removed by soft count = 122\n",
      "--- number of probes: 288 , kept by blast: 281 , if remove dups: 281\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1196\n",
      "--- gene=1196, id=28 removed by soft count = 101\n",
      "--- gene=1196, id=29 removed by soft count = 133\n",
      "--- gene=1196, id=71 removed by soft count = 65\n",
      "--- gene=1196, id=72 removed by soft count = 69\n",
      "--- gene=1196, id=103 removed by hard count = 0\n",
      "--- gene=1196, id=111 removed by soft count = 368\n",
      "--- gene=1196, id=274 removed by soft count = 76\n",
      "--- gene=1196, id=276 removed by soft count = 211\n",
      "--- number of probes: 277 , kept by blast: 269 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1197\n",
      "--- gene=1197, id=2 removed by soft count = 283\n",
      "--- gene=1197, id=52 removed by soft count = 103\n",
      "--- gene=1197, id=94 removed by soft count = 133\n",
      "--- gene=1197, id=148 removed by soft count = 5147\n",
      "--- gene=1197, id=150 removed by soft count = 507\n",
      "--- gene=1197, id=151 removed by soft count = 413\n",
      "--- gene=1197, id=152 removed by soft count = 72\n",
      "--- gene=1197, id=153 removed by soft count = 527\n",
      "--- gene=1197, id=154 removed by soft count = 416\n",
      "--- gene=1197, id=155 removed by soft count = 93\n",
      "--- gene=1197, id=156 removed by soft count = 374\n",
      "--- gene=1197, id=174 removed by soft count = 210\n",
      "--- gene=1197, id=177 removed by soft count = 251\n",
      "--- number of probes: 184 , kept by blast: 171 , if remove dups: 171\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 12\n",
      "--- gene=12, id=8 removed by soft count = 84\n",
      "--- gene=12, id=92 removed by soft count = 54\n",
      "--- gene=12, id=171 removed by soft count = 247\n",
      "--- gene=12, id=172 removed by soft count = 1643\n",
      "--- number of probes: 240 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 120\n",
      "--- gene=120, id=3 removed by soft count = 136\n",
      "--- gene=120, id=11 removed by soft count = 1962\n",
      "--- gene=120, id=105 removed by soft count = 70\n",
      "--- gene=120, id=126 removed by soft count = 211\n",
      "--- gene=120, id=127 removed by soft count = 56\n",
      "--- gene=120, id=128 removed by soft count = 73\n",
      "--- gene=120, id=159 removed by soft count = 241\n",
      "--- gene=120, id=160 removed by soft count = 585\n",
      "--- gene=120, id=163 removed by soft count = 76\n",
      "--- gene=120, id=164 removed by soft count = 119\n",
      "--- gene=120, id=166 removed by soft count = 82\n",
      "--- gene=120, id=167 removed by soft count = 63\n",
      "--- gene=120, id=168 removed by soft count = 124\n",
      "--- gene=120, id=169 removed by soft count = 135\n",
      "--- gene=120, id=170 removed by soft count = 113\n",
      "--- gene=120, id=171 removed by soft count = 121\n",
      "--- gene=120, id=173 removed by soft count = 169\n",
      "--- gene=120, id=185 removed by soft count = 76\n",
      "--- number of probes: 200 , kept by blast: 182 , if remove dups: 182\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1203\n",
      "--- gene=1203, id=0 removed by soft count = 58\n",
      "--- gene=1203, id=1 removed by soft count = 128\n",
      "--- gene=1203, id=39 removed by soft count = 69\n",
      "--- number of probes: 191 , kept by blast: 188 , if remove dups: 188\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1204\n",
      "--- gene=1204, id=230 removed by soft count = 266\n",
      "--- gene=1204, id=307 removed by soft count = 53\n",
      "--- gene=1204, id=344 removed by soft count = 120\n",
      "--- number of probes: 346 , kept by blast: 343 , if remove dups: 343\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1205\n",
      "--- gene=1205, id=9 removed by hard count = 0\n",
      "--- gene=1205, id=108 removed by soft count = 243\n",
      "--- gene=1205, id=144 removed by soft count = 175\n",
      "--- gene=1205, id=210 removed by soft count = 242\n",
      "--- gene=1205, id=211 removed by soft count = 356\n",
      "--- gene=1205, id=273 removed by soft count = 245\n",
      "--- gene=1205, id=276 removed by soft count = 55\n",
      "--- number of probes: 321 , kept by blast: 314 , if remove dups: 314\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1206\n",
      "--- gene=1206, id=0 removed by soft count = 188\n",
      "--- gene=1206, id=1 removed by soft count = 276\n",
      "--- gene=1206, id=2 removed by soft count = 204\n",
      "--- gene=1206, id=3 removed by soft count = 316\n",
      "--- gene=1206, id=4 removed by soft count = 191\n",
      "--- gene=1206, id=5 removed by soft count = 283\n",
      "--- gene=1206, id=6 removed by soft count = 349\n",
      "--- gene=1206, id=8 removed by soft count = 268\n",
      "--- gene=1206, id=9 removed by soft count = 64\n",
      "--- gene=1206, id=10 removed by soft count = 172\n",
      "--- gene=1206, id=11 removed by soft count = 4481\n",
      "--- gene=1206, id=12 removed by soft count = 281\n",
      "--- gene=1206, id=13 removed by soft count = 306\n",
      "--- gene=1206, id=14 removed by soft count = 196\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1206, id=16 removed by soft count = 275\n",
      "--- gene=1206, id=26 removed by soft count = 103\n",
      "--- gene=1206, id=31 removed by hard count = 3\n",
      "--- gene=1206, id=51 removed by soft count = 268\n",
      "--- gene=1206, id=56 removed by soft count = 121\n",
      "--- gene=1206, id=96 removed by soft count = 1537\n",
      "--- gene=1206, id=100 removed by soft count = 93\n",
      "--- gene=1206, id=102 removed by soft count = 161\n",
      "--- number of probes: 108 , kept by blast: 86 , if remove dups: 86\n",
      "-- number of probes kept for this region: 86\n",
      "-- checking probes in region: 1207\n",
      "--- gene=1207, id=18 removed by soft count = 106\n",
      "--- gene=1207, id=26 removed by soft count = 60\n",
      "--- gene=1207, id=32 removed by soft count = 2179\n",
      "--- gene=1207, id=33 removed by soft count = 1001\n",
      "--- gene=1207, id=177 removed by soft count = 52\n",
      "--- number of probes: 213 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1208\n",
      "--- gene=1208, id=5 removed by soft count = 146\n",
      "--- gene=1208, id=9 removed by soft count = 134\n",
      "--- gene=1208, id=26 removed by soft count = 2753\n",
      "--- gene=1208, id=27 removed by soft count = 59\n",
      "--- gene=1208, id=32 removed by soft count = 826\n",
      "--- gene=1208, id=55 removed by soft count = 88\n",
      "--- gene=1208, id=122 removed by soft count = 1488\n",
      "--- gene=1208, id=202 removed by soft count = 847\n",
      "--- number of probes: 208 , kept by blast: 200 , if remove dups: 200\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1209\n",
      "--- gene=1209, id=32 removed by soft count = 330\n",
      "--- gene=1209, id=34 removed by soft count = 60\n",
      "--- gene=1209, id=72 removed by soft count = 385\n",
      "--- gene=1209, id=114 removed by soft count = 243\n",
      "--- gene=1209, id=116 removed by soft count = 2926\n",
      "--- gene=1209, id=118 removed by soft count = 165\n",
      "--- gene=1209, id=121 removed by soft count = 115\n",
      "--- gene=1209, id=161 removed by soft count = 60\n",
      "--- number of probes: 169 , kept by blast: 161 , if remove dups: 161\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 121\n",
      "--- gene=121, id=62 removed by soft count = 293\n",
      "--- gene=121, id=63 removed by soft count = 79\n",
      "--- gene=121, id=65 removed by soft count = 138\n",
      "--- gene=121, id=67 removed by soft count = 106\n",
      "--- gene=121, id=70 removed by soft count = 128\n",
      "--- gene=121, id=71 removed by soft count = 165\n",
      "--- gene=121, id=72 removed by soft count = 255\n",
      "--- gene=121, id=76 removed by soft count = 63\n",
      "--- gene=121, id=148 removed by soft count = 97\n",
      "--- number of probes: 295 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1210\n",
      "--- gene=1210, id=34 removed by soft count = 56\n",
      "--- gene=1210, id=115 removed by soft count = 70\n",
      "--- gene=1210, id=153 removed by soft count = 2310\n",
      "--- number of probes: 270 , kept by blast: 267 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1211\n",
      "--- gene=1211, id=25 removed by soft count = 271\n",
      "--- number of probes: 302 , kept by blast: 301 , if remove dups: 301\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1212\n",
      "--- gene=1212, id=123 removed by soft count = 297\n",
      "--- gene=1212, id=172 removed by soft count = 82\n",
      "--- number of probes: 278 , kept by blast: 276 , if remove dups: 276\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1213\n",
      "--- gene=1213, id=15 removed by soft count = 107\n",
      "--- gene=1213, id=18 removed by soft count = 347\n",
      "--- gene=1213, id=42 removed by soft count = 220\n",
      "--- gene=1213, id=148 removed by soft count = 113\n",
      "--- gene=1213, id=150 removed by soft count = 254\n",
      "--- gene=1213, id=153 removed by soft count = 114\n",
      "--- gene=1213, id=167 removed by soft count = 130\n",
      "--- number of probes: 293 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1214\n",
      "--- gene=1214, id=151 removed by soft count = 718\n",
      "--- gene=1214, id=153 removed by soft count = 562\n",
      "--- gene=1214, id=183 removed by soft count = 51\n",
      "--- number of probes: 317 , kept by blast: 314 , if remove dups: 314\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1215\n",
      "--- gene=1215, id=16 removed by soft count = 1095\n",
      "--- gene=1215, id=118 removed by soft count = 79\n",
      "--- gene=1215, id=119 removed by soft count = 237\n",
      "--- gene=1215, id=128 removed by soft count = 107\n",
      "--- gene=1215, id=131 removed by soft count = 159\n",
      "--- gene=1215, id=161 removed by soft count = 141\n",
      "--- gene=1215, id=173 removed by soft count = 577\n",
      "--- gene=1215, id=174 removed by soft count = 131\n",
      "--- gene=1215, id=175 removed by soft count = 399\n",
      "--- gene=1215, id=176 removed by soft count = 1636\n",
      "--- gene=1215, id=177 removed by soft count = 396\n",
      "--- gene=1215, id=178 removed by soft count = 1550\n",
      "--- gene=1215, id=182 removed by soft count = 86\n",
      "--- gene=1215, id=218 removed by soft count = 218\n",
      "--- number of probes: 221 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1216\n",
      "--- gene=1216, id=18 removed by soft count = 327\n",
      "--- gene=1216, id=23 removed by soft count = 184\n",
      "--- gene=1216, id=89 removed by soft count = 64\n",
      "--- number of probes: 216 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1217\n",
      "--- gene=1217, id=21 removed by soft count = 204\n",
      "--- gene=1217, id=22 removed by soft count = 408\n",
      "--- gene=1217, id=23 removed by soft count = 805\n",
      "--- gene=1217, id=166 removed by soft count = 159\n",
      "--- gene=1217, id=178 removed by soft count = 68\n",
      "--- number of probes: 190 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1218\n",
      "--- gene=1218, id=4 removed by soft count = 73\n",
      "--- gene=1218, id=5 removed by soft count = 51\n",
      "--- gene=1218, id=17 removed by soft count = 78\n",
      "--- gene=1218, id=144 removed by soft count = 1430\n",
      "--- gene=1218, id=145 removed by soft count = 225\n",
      "--- gene=1218, id=232 removed by soft count = 84\n",
      "--- number of probes: 235 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1219\n",
      "--- gene=1219, id=66 removed by soft count = 149\n",
      "--- gene=1219, id=246 removed by soft count = 220\n",
      "--- gene=1219, id=247 removed by soft count = 63\n",
      "--- gene=1219, id=248 removed by soft count = 116\n",
      "--- number of probes: 315 , kept by blast: 311 , if remove dups: 311\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 122\n",
      "--- gene=122, id=5 removed by soft count = 130\n",
      "--- gene=122, id=98 removed by soft count = 3966\n",
      "--- gene=122, id=161 removed by soft count = 170\n",
      "--- number of probes: 270 , kept by blast: 267 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1220\n",
      "--- number of probes: 358 , kept by blast: 358 , if remove dups: 358\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1221\n",
      "--- gene=1221, id=285 removed by soft count = 133\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1222\n",
      "--- gene=1222, id=7 removed by soft count = 62\n",
      "--- gene=1222, id=33 removed by soft count = 256\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1222, id=34 removed by soft count = 474\n",
      "--- gene=1222, id=35 removed by soft count = 213\n",
      "--- gene=1222, id=36 removed by soft count = 52\n",
      "--- gene=1222, id=63 removed by soft count = 197\n",
      "--- gene=1222, id=68 removed by soft count = 124\n",
      "--- gene=1222, id=74 removed by soft count = 71\n",
      "--- gene=1222, id=105 removed by soft count = 168\n",
      "--- gene=1222, id=107 removed by soft count = 126\n",
      "--- gene=1222, id=110 removed by soft count = 71\n",
      "--- number of probes: 179 , kept by blast: 168 , if remove dups: 168\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1224\n",
      "--- gene=1224, id=205 removed by soft count = 452\n",
      "--- number of probes: 231 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1225\n",
      "--- gene=1225, id=83 removed by soft count = 57\n",
      "--- gene=1225, id=91 removed by soft count = 62\n",
      "--- gene=1225, id=177 removed by soft count = 852\n",
      "--- gene=1225, id=222 removed by soft count = 99\n",
      "--- gene=1225, id=247 removed by soft count = 160\n",
      "--- gene=1225, id=250 removed by soft count = 121\n",
      "--- number of probes: 282 , kept by blast: 276 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1226\n",
      "--- gene=1226, id=229 removed by soft count = 135\n",
      "--- gene=1226, id=231 removed by soft count = 93\n",
      "--- number of probes: 238 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1227\n",
      "--- gene=1227, id=2 removed by soft count = 100\n",
      "--- gene=1227, id=5 removed by soft count = 408\n",
      "--- gene=1227, id=6 removed by soft count = 207\n",
      "--- gene=1227, id=40 removed by soft count = 261\n",
      "--- gene=1227, id=167 removed by soft count = 143\n",
      "--- gene=1227, id=172 removed by soft count = 209\n",
      "--- number of probes: 215 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1228\n",
      "--- gene=1228, id=58 removed by soft count = 169\n",
      "--- gene=1228, id=64 removed by soft count = 60\n",
      "--- number of probes: 167 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1229\n",
      "--- gene=1229, id=29 removed by soft count = 134\n",
      "--- gene=1229, id=30 removed by soft count = 130\n",
      "--- gene=1229, id=34 removed by soft count = 190\n",
      "--- gene=1229, id=37 removed by soft count = 55\n",
      "--- gene=1229, id=40 removed by soft count = 92\n",
      "--- gene=1229, id=43 removed by soft count = 59\n",
      "--- gene=1229, id=78 removed by soft count = 64\n",
      "--- gene=1229, id=114 removed by soft count = 108\n",
      "--- gene=1229, id=146 removed by soft count = 371\n",
      "--- gene=1229, id=165 removed by soft count = 310\n",
      "--- gene=1229, id=208 removed by soft count = 1211\n",
      "--- number of probes: 235 , kept by blast: 224 , if remove dups: 224\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 123\n",
      "--- gene=123, id=23 removed by soft count = 498\n",
      "--- gene=123, id=24 removed by soft count = 171\n",
      "--- gene=123, id=48 removed by soft count = 640\n",
      "--- gene=123, id=168 removed by soft count = 67\n",
      "--- gene=123, id=170 removed by soft count = 151\n",
      "--- gene=123, id=174 removed by soft count = 172\n",
      "--- gene=123, id=176 removed by soft count = 94\n",
      "--- number of probes: 182 , kept by blast: 175 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1230\n",
      "--- gene=1230, id=184 removed by soft count = 271\n",
      "--- gene=1230, id=185 removed by soft count = 1295\n",
      "--- gene=1230, id=187 removed by soft count = 279\n",
      "--- gene=1230, id=204 removed by soft count = 548\n",
      "--- number of probes: 223 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1231\n",
      "--- gene=1231, id=0 removed by soft count = 92\n",
      "--- gene=1231, id=32 removed by soft count = 222\n",
      "--- number of probes: 241 , kept by blast: 239 , if remove dups: 239\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1232\n",
      "--- gene=1232, id=127 removed by soft count = 597\n",
      "--- gene=1232, id=172 removed by soft count = 86\n",
      "--- gene=1232, id=189 removed by soft count = 81\n",
      "--- number of probes: 237 , kept by blast: 234 , if remove dups: 234\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1233\n",
      "--- gene=1233, id=88 removed by soft count = 153\n",
      "--- gene=1233, id=93 removed by soft count = 1344\n",
      "--- gene=1233, id=157 removed by soft count = 141\n",
      "--- gene=1233, id=197 removed by soft count = 58\n",
      "--- gene=1233, id=198 removed by soft count = 268\n",
      "--- number of probes: 223 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1234\n",
      "--- gene=1234, id=24 removed by soft count = 343\n",
      "--- gene=1234, id=46 removed by soft count = 100\n",
      "--- gene=1234, id=85 removed by soft count = 4133\n",
      "--- number of probes: 169 , kept by blast: 166 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1235\n",
      "--- gene=1235, id=118 removed by soft count = 253\n",
      "--- gene=1235, id=145 removed by soft count = 55\n",
      "--- gene=1235, id=206 removed by hard count = 3\n",
      "--- gene=1235, id=207 removed by soft count = 109\n",
      "--- gene=1235, id=214 removed by soft count = 305\n",
      "--- gene=1235, id=237 removed by soft count = 74\n",
      "--- number of probes: 241 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1236\n",
      "--- gene=1236, id=11 removed by soft count = 63\n",
      "--- gene=1236, id=24 removed by soft count = 59\n",
      "--- gene=1236, id=30 removed by soft count = 106\n",
      "--- gene=1236, id=245 removed by soft count = 146\n",
      "--- number of probes: 271 , kept by blast: 267 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1237\n",
      "--- gene=1237, id=26 removed by soft count = 94\n",
      "--- number of probes: 228 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1238\n",
      "--- gene=1238, id=96 removed by soft count = 189\n",
      "--- gene=1238, id=98 removed by soft count = 248\n",
      "--- gene=1238, id=275 removed by soft count = 102\n",
      "--- number of probes: 287 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1239\n",
      "--- gene=1239, id=6 removed by soft count = 175\n",
      "--- gene=1239, id=12 removed by soft count = 99\n",
      "--- gene=1239, id=43 removed by soft count = 81\n",
      "--- gene=1239, id=142 removed by soft count = 92\n",
      "--- gene=1239, id=145 removed by soft count = 123\n",
      "--- number of probes: 249 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 124\n",
      "--- gene=124, id=93 removed by soft count = 142\n",
      "--- gene=124, id=96 removed by soft count = 915\n",
      "--- gene=124, id=113 removed by soft count = 91\n",
      "--- gene=124, id=124 removed by soft count = 61\n",
      "--- gene=124, id=125 removed by soft count = 67\n",
      "--- gene=124, id=127 removed by soft count = 81\n",
      "--- gene=124, id=128 removed by soft count = 79\n",
      "--- number of probes: 235 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1240\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1240, id=37 removed by soft count = 194\n",
      "--- gene=1240, id=38 removed by soft count = 343\n",
      "--- gene=1240, id=120 removed by soft count = 231\n",
      "--- gene=1240, id=255 removed by soft count = 88\n",
      "--- gene=1240, id=257 removed by soft count = 85\n",
      "--- gene=1240, id=266 removed by soft count = 53\n",
      "--- gene=1240, id=269 removed by soft count = 150\n",
      "--- number of probes: 297 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1241\n",
      "--- gene=1241, id=131 removed by soft count = 54\n",
      "--- gene=1241, id=283 removed by soft count = 84\n",
      "--- gene=1241, id=301 removed by soft count = 828\n",
      "--- gene=1241, id=323 removed by soft count = 80\n",
      "--- number of probes: 330 , kept by blast: 326 , if remove dups: 326\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1242\n",
      "--- gene=1242, id=29 removed by soft count = 620\n",
      "--- gene=1242, id=106 removed by hard count = 0\n",
      "--- gene=1242, id=155 removed by soft count = 73\n",
      "--- gene=1242, id=156 removed by soft count = 67\n",
      "--- gene=1242, id=169 removed by soft count = 121\n",
      "--- gene=1242, id=172 removed by soft count = 215\n",
      "--- gene=1242, id=173 removed by soft count = 212\n",
      "--- gene=1242, id=217 removed by soft count = 108\n",
      "--- number of probes: 279 , kept by blast: 271 , if remove dups: 269\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1243\n",
      "--- gene=1243, id=161 removed by soft count = 514\n",
      "--- number of probes: 297 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1244\n",
      "--- gene=1244, id=196 removed by soft count = 72\n",
      "--- number of probes: 260 , kept by blast: 259 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1245\n",
      "--- gene=1245, id=11 removed by soft count = 197\n",
      "--- gene=1245, id=17 removed by soft count = 77\n",
      "--- gene=1245, id=112 removed by soft count = 91\n",
      "--- gene=1245, id=150 removed by soft count = 52\n",
      "--- gene=1245, id=152 removed by soft count = 207\n",
      "--- gene=1245, id=155 removed by soft count = 172\n",
      "--- gene=1245, id=213 removed by soft count = 273\n",
      "--- gene=1245, id=277 removed by soft count = 145\n",
      "--- number of probes: 283 , kept by blast: 275 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1246\n",
      "--- gene=1246, id=11 removed by soft count = 329\n",
      "--- gene=1246, id=36 removed by soft count = 79\n",
      "--- gene=1246, id=52 removed by soft count = 52\n",
      "--- gene=1246, id=227 removed by soft count = 295\n",
      "--- gene=1246, id=280 removed by hard count = 3\n",
      "--- number of probes: 297 , kept by blast: 292 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1247\n",
      "--- gene=1247, id=145 removed by soft count = 55\n",
      "--- gene=1247, id=163 removed by soft count = 85\n",
      "--- gene=1247, id=192 removed by soft count = 152\n",
      "--- number of probes: 232 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1248\n",
      "--- gene=1248, id=0 removed by soft count = 223\n",
      "--- gene=1248, id=56 removed by soft count = 77\n",
      "--- gene=1248, id=120 removed by soft count = 1561\n",
      "--- gene=1248, id=125 removed by soft count = 83\n",
      "--- number of probes: 160 , kept by blast: 156 , if remove dups: 156\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1249\n",
      "--- gene=1249, id=50 removed by soft count = 51\n",
      "--- gene=1249, id=51 removed by soft count = 139\n",
      "--- gene=1249, id=52 removed by soft count = 125\n",
      "--- gene=1249, id=53 removed by soft count = 171\n",
      "--- gene=1249, id=114 removed by soft count = 110\n",
      "--- gene=1249, id=116 removed by soft count = 199\n",
      "--- number of probes: 237 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 125\n",
      "--- gene=125, id=24 removed by soft count = 4093\n",
      "--- gene=125, id=100 removed by soft count = 798\n",
      "--- gene=125, id=113 removed by soft count = 52\n",
      "--- number of probes: 251 , kept by blast: 248 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1250\n",
      "--- gene=1250, id=130 removed by soft count = 735\n",
      "--- gene=1250, id=157 removed by hard count = 3\n",
      "--- gene=1250, id=166 removed by hard count = 0\n",
      "--- number of probes: 310 , kept by blast: 307 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1251\n",
      "--- gene=1251, id=21 removed by soft count = 200\n",
      "--- gene=1251, id=30 removed by soft count = 135\n",
      "--- gene=1251, id=31 removed by soft count = 639\n",
      "--- gene=1251, id=32 removed by soft count = 2208\n",
      "--- gene=1251, id=33 removed by soft count = 1622\n",
      "--- gene=1251, id=106 removed by soft count = 433\n",
      "--- gene=1251, id=111 removed by soft count = 292\n",
      "--- gene=1251, id=115 removed by soft count = 75\n",
      "--- gene=1251, id=116 removed by soft count = 2066\n",
      "--- number of probes: 181 , kept by blast: 172 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1252\n",
      "--- gene=1252, id=0 removed by soft count = 451\n",
      "--- gene=1252, id=55 removed by soft count = 122\n",
      "--- gene=1252, id=132 removed by soft count = 3285\n",
      "--- gene=1252, id=157 removed by soft count = 79\n",
      "--- number of probes: 181 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1253\n",
      "--- gene=1253, id=43 removed by soft count = 60\n",
      "--- gene=1253, id=126 removed by soft count = 1513\n",
      "--- number of probes: 246 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1254\n",
      "--- gene=1254, id=26 removed by soft count = 1118\n",
      "--- gene=1254, id=116 removed by soft count = 70\n",
      "--- gene=1254, id=117 removed by soft count = 82\n",
      "--- number of probes: 178 , kept by blast: 175 , if remove dups: 175\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1255\n",
      "--- gene=1255, id=48 removed by soft count = 91\n",
      "--- gene=1255, id=159 removed by soft count = 207\n",
      "--- gene=1255, id=162 removed by soft count = 380\n",
      "--- number of probes: 303 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1256\n",
      "--- gene=1256, id=12 removed by soft count = 21712\n",
      "--- gene=1256, id=175 removed by soft count = 265\n",
      "--- number of probes: 186 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1257\n",
      "--- gene=1257, id=0 removed by soft count = 62\n",
      "--- gene=1257, id=6 removed by soft count = 53\n",
      "--- gene=1257, id=34 removed by soft count = 167\n",
      "--- number of probes: 173 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1258\n",
      "--- gene=1258, id=11 removed by soft count = 8477\n",
      "--- gene=1258, id=16 removed by soft count = 7234\n",
      "--- gene=1258, id=17 removed by soft count = 125\n",
      "--- gene=1258, id=64 removed by soft count = 73\n",
      "--- gene=1258, id=116 removed by soft count = 460\n",
      "--- gene=1258, id=136 removed by soft count = 54\n",
      "--- number of probes: 168 , kept by blast: 162 , if remove dups: 162\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1259\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1259, id=60 removed by soft count = 76\n",
      "--- gene=1259, id=69 removed by soft count = 136\n",
      "--- gene=1259, id=214 removed by soft count = 275\n",
      "--- gene=1259, id=215 removed by soft count = 1289\n",
      "--- number of probes: 216 , kept by blast: 212 , if remove dups: 212\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 126\n",
      "--- gene=126, id=249 removed by soft count = 69\n",
      "--- gene=126, id=251 removed by soft count = 51\n",
      "--- gene=126, id=259 removed by soft count = 109\n",
      "--- gene=126, id=263 removed by soft count = 65\n",
      "--- gene=126, id=265 removed by soft count = 56\n",
      "--- number of probes: 291 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1260\n",
      "--- gene=1260, id=147 removed by soft count = 146\n",
      "--- gene=1260, id=152 removed by soft count = 189\n",
      "--- number of probes: 364 , kept by blast: 362 , if remove dups: 362\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1261\n",
      "--- gene=1261, id=37 removed by hard count = 0\n",
      "--- gene=1261, id=356 removed by soft count = 154\n",
      "--- number of probes: 377 , kept by blast: 375 , if remove dups: 375\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1262\n",
      "--- gene=1262, id=2 removed by soft count = 950\n",
      "--- gene=1262, id=5 removed by soft count = 53\n",
      "--- gene=1262, id=15 removed by soft count = 310\n",
      "--- gene=1262, id=18 removed by soft count = 61\n",
      "--- gene=1262, id=23 removed by soft count = 57\n",
      "--- gene=1262, id=45 removed by soft count = 52\n",
      "--- gene=1262, id=48 removed by soft count = 61\n",
      "--- gene=1262, id=60 removed by soft count = 618\n",
      "--- gene=1262, id=61 removed by soft count = 356\n",
      "--- gene=1262, id=63 removed by soft count = 76\n",
      "--- gene=1262, id=64 removed by soft count = 129\n",
      "--- gene=1262, id=65 removed by soft count = 82\n",
      "--- gene=1262, id=66 removed by soft count = 95\n",
      "--- gene=1262, id=68 removed by soft count = 127\n",
      "--- gene=1262, id=70 removed by soft count = 88\n",
      "--- gene=1262, id=71 removed by soft count = 66\n",
      "--- gene=1262, id=72 removed by soft count = 91\n",
      "--- gene=1262, id=73 removed by soft count = 58\n",
      "--- gene=1262, id=74 removed by soft count = 61\n",
      "--- gene=1262, id=77 removed by soft count = 58\n",
      "--- gene=1262, id=80 removed by soft count = 68\n",
      "--- gene=1262, id=82 removed by soft count = 51\n",
      "--- number of probes: 85 , kept by blast: 63 , if remove dups: 63\n",
      "-- number of probes kept for this region: 63\n",
      "-- checking probes in region: 1263\n",
      "--- gene=1263, id=0 removed by soft count = 109\n",
      "--- gene=1263, id=8 removed by soft count = 82\n",
      "--- gene=1263, id=65 removed by soft count = 58\n",
      "--- gene=1263, id=101 removed by soft count = 584\n",
      "--- gene=1263, id=114 removed by soft count = 643\n",
      "--- gene=1263, id=117 removed by soft count = 478\n",
      "--- gene=1263, id=126 removed by soft count = 107\n",
      "--- gene=1263, id=131 removed by soft count = 64\n",
      "--- gene=1263, id=132 removed by soft count = 61\n",
      "--- gene=1263, id=144 removed by soft count = 61\n",
      "--- gene=1263, id=148 removed by soft count = 73\n",
      "--- gene=1263, id=171 removed by soft count = 58\n",
      "--- gene=1263, id=176 removed by soft count = 493\n",
      "--- number of probes: 189 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1264\n",
      "--- gene=1264, id=3 removed by soft count = 77\n",
      "--- gene=1264, id=27 removed by soft count = 59\n",
      "--- gene=1264, id=63 removed by soft count = 129\n",
      "--- gene=1264, id=83 removed by soft count = 73\n",
      "--- gene=1264, id=85 removed by soft count = 92\n",
      "--- gene=1264, id=93 removed by soft count = 73\n",
      "--- gene=1264, id=121 removed by soft count = 242\n",
      "--- gene=1264, id=150 removed by soft count = 72\n",
      "--- number of probes: 279 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1265\n",
      "--- gene=1265, id=0 removed by soft count = 408\n",
      "--- gene=1265, id=1 removed by soft count = 123\n",
      "--- gene=1265, id=2 removed by soft count = 729\n",
      "--- gene=1265, id=5 removed by soft count = 477\n",
      "--- gene=1265, id=6 removed by soft count = 353\n",
      "--- gene=1265, id=7 removed by soft count = 560\n",
      "--- gene=1265, id=13 removed by soft count = 893\n",
      "--- gene=1265, id=16 removed by soft count = 107\n",
      "--- gene=1265, id=17 removed by soft count = 95\n",
      "--- gene=1265, id=18 removed by soft count = 67\n",
      "--- gene=1265, id=28 removed by soft count = 92\n",
      "--- gene=1265, id=54 removed by soft count = 55\n",
      "--- gene=1265, id=76 removed by soft count = 85\n",
      "--- gene=1265, id=80 removed by soft count = 55\n",
      "--- gene=1265, id=84 removed by soft count = 10868\n",
      "--- gene=1265, id=86 removed by soft count = 91\n",
      "--- gene=1265, id=87 removed by soft count = 190\n",
      "--- gene=1265, id=88 removed by soft count = 104\n",
      "--- number of probes: 99 , kept by blast: 81 , if remove dups: 81\n",
      "-- number of probes kept for this region: 81\n",
      "-- checking probes in region: 1269\n",
      "--- gene=1269, id=0 removed by soft count = 1746\n",
      "--- gene=1269, id=1 removed by soft count = 1408\n",
      "--- gene=1269, id=5 removed by soft count = 1243\n",
      "--- gene=1269, id=6 removed by soft count = 188\n",
      "--- gene=1269, id=7 removed by soft count = 463\n",
      "--- gene=1269, id=8 removed by soft count = 663\n",
      "--- gene=1269, id=10 removed by soft count = 317\n",
      "--- gene=1269, id=11 removed by soft count = 939\n",
      "--- gene=1269, id=12 removed by soft count = 57\n",
      "--- gene=1269, id=13 removed by soft count = 306\n",
      "--- gene=1269, id=14 removed by soft count = 106\n",
      "--- gene=1269, id=15 removed by soft count = 488\n",
      "--- gene=1269, id=17 removed by soft count = 308\n",
      "--- gene=1269, id=19 removed by soft count = 171\n",
      "--- gene=1269, id=21 removed by soft count = 91\n",
      "--- gene=1269, id=22 removed by soft count = 167\n",
      "--- gene=1269, id=23 removed by soft count = 465\n",
      "--- gene=1269, id=25 removed by soft count = 196\n",
      "--- gene=1269, id=27 removed by soft count = 224\n",
      "--- gene=1269, id=30 removed by soft count = 1968\n",
      "--- gene=1269, id=33 removed by soft count = 99\n",
      "--- gene=1269, id=35 removed by soft count = 586\n",
      "--- gene=1269, id=36 removed by soft count = 167\n",
      "--- gene=1269, id=37 removed by soft count = 533\n",
      "--- gene=1269, id=38 removed by soft count = 154\n",
      "--- gene=1269, id=43 removed by soft count = 640\n",
      "--- gene=1269, id=44 removed by soft count = 108\n",
      "--- gene=1269, id=45 removed by soft count = 3061\n",
      "--- gene=1269, id=49 removed by soft count = 63\n",
      "--- gene=1269, id=50 removed by soft count = 742\n",
      "--- number of probes: 51 , kept by blast: 21 , if remove dups: 21\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: 127\n",
      "--- gene=127, id=2 removed by soft count = 52\n",
      "--- gene=127, id=17 removed by soft count = 64\n",
      "--- gene=127, id=116 removed by soft count = 290\n",
      "--- gene=127, id=117 removed by soft count = 162\n",
      "--- gene=127, id=178 removed by soft count = 85\n",
      "--- number of probes: 222 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1270\n",
      "--- gene=1270, id=110 removed by soft count = 119\n",
      "--- gene=1270, id=112 removed by soft count = 187\n",
      "--- gene=1270, id=230 removed by soft count = 108\n",
      "--- number of probes: 305 , kept by blast: 302 , if remove dups: 302\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1271\n",
      "--- gene=1271, id=265 removed by soft count = 74\n",
      "--- gene=1271, id=307 removed by soft count = 104\n",
      "--- gene=1271, id=308 removed by hard count = 0\n",
      "--- number of probes: 312 , kept by blast: 309 , if remove dups: 309\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1272\n",
      "--- number of probes: 316 , kept by blast: 316 , if remove dups: 315\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1273\n",
      "--- gene=1273, id=0 removed by soft count = 381\n",
      "--- gene=1273, id=4 removed by soft count = 676\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1273, id=34 removed by soft count = 155\n",
      "--- gene=1273, id=53 removed by soft count = 145\n",
      "--- gene=1273, id=64 removed by soft count = 513\n",
      "--- gene=1273, id=84 removed by soft count = 64\n",
      "--- gene=1273, id=192 removed by soft count = 84\n",
      "--- gene=1273, id=212 removed by hard count = 6\n",
      "--- gene=1273, id=276 removed by soft count = 99\n",
      "--- number of probes: 295 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1274\n",
      "--- gene=1274, id=63 removed by soft count = 377\n",
      "--- gene=1274, id=65 removed by soft count = 147\n",
      "--- gene=1274, id=85 removed by soft count = 66\n",
      "--- gene=1274, id=99 removed by soft count = 354\n",
      "--- gene=1274, id=100 removed by soft count = 75\n",
      "--- gene=1274, id=108 removed by soft count = 100\n",
      "--- gene=1274, id=109 removed by soft count = 66\n",
      "--- gene=1274, id=149 removed by soft count = 143\n",
      "--- gene=1274, id=158 removed by soft count = 701\n",
      "--- gene=1274, id=180 removed by soft count = 335\n",
      "--- gene=1274, id=186 removed by soft count = 82\n",
      "--- gene=1274, id=187 removed by soft count = 94\n",
      "--- gene=1274, id=188 removed by soft count = 57\n",
      "--- number of probes: 192 , kept by blast: 179 , if remove dups: 179\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1275\n",
      "--- gene=1275, id=72 removed by soft count = 301\n",
      "--- gene=1275, id=283 removed by soft count = 132\n",
      "--- gene=1275, id=284 removed by soft count = 55\n",
      "--- number of probes: 349 , kept by blast: 346 , if remove dups: 346\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1276\n",
      "--- gene=1276, id=109 removed by soft count = 476\n",
      "--- gene=1276, id=242 removed by soft count = 603\n",
      "--- gene=1276, id=243 removed by soft count = 637\n",
      "--- number of probes: 244 , kept by blast: 241 , if remove dups: 241\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1277\n",
      "--- gene=1277, id=13 removed by soft count = 113\n",
      "--- number of probes: 362 , kept by blast: 361 , if remove dups: 361\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1278\n",
      "--- gene=1278, id=39 removed by soft count = 411\n",
      "--- gene=1278, id=40 removed by soft count = 231\n",
      "--- gene=1278, id=84 removed by soft count = 96\n",
      "--- gene=1278, id=88 removed by soft count = 91\n",
      "--- gene=1278, id=110 removed by soft count = 200\n",
      "--- gene=1278, id=113 removed by soft count = 111\n",
      "--- gene=1278, id=120 removed by soft count = 66\n",
      "--- number of probes: 150 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1279\n",
      "--- gene=1279, id=31 removed by soft count = 107\n",
      "--- gene=1279, id=32 removed by soft count = 98\n",
      "--- gene=1279, id=33 removed by soft count = 57\n",
      "--- gene=1279, id=34 removed by soft count = 813\n",
      "--- gene=1279, id=66 removed by soft count = 104\n",
      "--- gene=1279, id=69 removed by soft count = 53\n",
      "--- number of probes: 193 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 128\n",
      "--- gene=128, id=24 removed by soft count = 565\n",
      "--- gene=128, id=28 removed by soft count = 126\n",
      "--- gene=128, id=172 removed by soft count = 362\n",
      "--- number of probes: 275 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1280\n",
      "--- number of probes: 296 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1281\n",
      "--- gene=1281, id=260 removed by soft count = 173\n",
      "--- gene=1281, id=262 removed by soft count = 119\n",
      "--- number of probes: 275 , kept by blast: 273 , if remove dups: 273\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1282\n",
      "--- number of probes: 234 , kept by blast: 234 , if remove dups: 234\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1284\n",
      "--- gene=1284, id=1 removed by soft count = 81\n",
      "--- gene=1284, id=3 removed by soft count = 158\n",
      "--- gene=1284, id=4 removed by soft count = 145\n",
      "--- gene=1284, id=7 removed by soft count = 173\n",
      "--- gene=1284, id=9 removed by soft count = 116\n",
      "--- gene=1284, id=11 removed by soft count = 56\n",
      "--- gene=1284, id=12 removed by soft count = 54\n",
      "--- gene=1284, id=18 removed by soft count = 63\n",
      "--- gene=1284, id=21 removed by soft count = 72\n",
      "--- gene=1284, id=22 removed by soft count = 127\n",
      "--- gene=1284, id=23 removed by soft count = 115\n",
      "--- gene=1284, id=26 removed by soft count = 61\n",
      "--- gene=1284, id=28 removed by soft count = 57\n",
      "--- gene=1284, id=30 removed by soft count = 131\n",
      "--- gene=1284, id=33 removed by soft count = 79\n",
      "--- gene=1284, id=35 removed by soft count = 67\n",
      "--- gene=1284, id=36 removed by soft count = 77\n",
      "--- gene=1284, id=37 removed by soft count = 1292\n",
      "--- gene=1284, id=38 removed by soft count = 320\n",
      "--- gene=1284, id=89 removed by soft count = 135\n",
      "--- number of probes: 188 , kept by blast: 168 , if remove dups: 168\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1285\n",
      "--- gene=1285, id=22 removed by soft count = 142\n",
      "--- gene=1285, id=50 removed by soft count = 335\n",
      "--- gene=1285, id=99 removed by soft count = 388\n",
      "--- number of probes: 206 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1286\n",
      "--- gene=1286, id=61 removed by soft count = 1000\n",
      "--- gene=1286, id=258 removed by soft count = 100\n",
      "--- number of probes: 263 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1287\n",
      "--- gene=1287, id=189 removed by soft count = 70\n",
      "--- gene=1287, id=195 removed by soft count = 56791\n",
      "--- number of probes: 287 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1288\n",
      "--- gene=1288, id=155 removed by soft count = 129\n",
      "--- gene=1288, id=195 removed by soft count = 285\n",
      "--- number of probes: 239 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1289\n",
      "--- gene=1289, id=59 removed by hard count = 0\n",
      "--- gene=1289, id=60 removed by hard count = 0\n",
      "--- number of probes: 367 , kept by blast: 365 , if remove dups: 365\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 129\n",
      "--- gene=129, id=49 removed by soft count = 92\n",
      "--- gene=129, id=55 removed by soft count = 134\n",
      "--- gene=129, id=56 removed by soft count = 181\n",
      "--- number of probes: 216 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1290\n",
      "--- gene=1290, id=9 removed by hard count = 0\n",
      "--- gene=1290, id=215 removed by soft count = 249\n",
      "--- number of probes: 315 , kept by blast: 313 , if remove dups: 313\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1291\n",
      "--- gene=1291, id=17 removed by soft count = 91\n",
      "--- gene=1291, id=19 removed by soft count = 68\n",
      "--- gene=1291, id=49 removed by soft count = 1428\n",
      "--- gene=1291, id=99 removed by soft count = 59\n",
      "--- gene=1291, id=105 removed by soft count = 171\n",
      "--- gene=1291, id=143 removed by soft count = 104\n",
      "--- gene=1291, id=145 removed by soft count = 76\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1291, id=254 removed by soft count = 476\n",
      "--- number of probes: 273 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1292\n",
      "--- number of probes: 248 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1293\n",
      "--- gene=1293, id=22 removed by soft count = 1524\n",
      "--- gene=1293, id=67 removed by soft count = 76\n",
      "--- gene=1293, id=81 removed by soft count = 109\n",
      "--- gene=1293, id=83 removed by soft count = 65\n",
      "--- gene=1293, id=102 removed by soft count = 65\n",
      "--- gene=1293, id=197 removed by soft count = 51\n",
      "--- gene=1293, id=219 removed by soft count = 87\n",
      "--- gene=1293, id=220 removed by soft count = 67\n",
      "--- gene=1293, id=221 removed by soft count = 885\n",
      "--- gene=1293, id=222 removed by soft count = 57\n",
      "--- gene=1293, id=277 removed by soft count = 212\n",
      "--- number of probes: 312 , kept by blast: 301 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1294\n",
      "--- gene=1294, id=1 removed by soft count = 203\n",
      "--- gene=1294, id=3 removed by soft count = 2809\n",
      "--- gene=1294, id=31 removed by soft count = 74\n",
      "--- gene=1294, id=33 removed by soft count = 119\n",
      "--- gene=1294, id=34 removed by soft count = 236\n",
      "--- gene=1294, id=35 removed by soft count = 209\n",
      "--- gene=1294, id=46 removed by soft count = 453\n",
      "--- gene=1294, id=47 removed by soft count = 5567\n",
      "--- gene=1294, id=66 removed by soft count = 72\n",
      "--- gene=1294, id=71 removed by soft count = 317\n",
      "--- gene=1294, id=76 removed by soft count = 53\n",
      "--- gene=1294, id=83 removed by soft count = 108\n",
      "--- number of probes: 90 , kept by blast: 78 , if remove dups: 78\n",
      "-- number of probes kept for this region: 78\n",
      "-- checking probes in region: 1295\n",
      "--- gene=1295, id=25 removed by soft count = 51\n",
      "--- gene=1295, id=28 removed by soft count = 171\n",
      "--- gene=1295, id=49 removed by soft count = 76\n",
      "--- number of probes: 167 , kept by blast: 164 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1297\n",
      "--- gene=1297, id=264 removed by hard count = 4\n",
      "--- gene=1297, id=303 removed by soft count = 62\n",
      "--- number of probes: 312 , kept by blast: 310 , if remove dups: 310\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1298\n",
      "--- gene=1298, id=56 removed by soft count = 1252\n",
      "--- gene=1298, id=130 removed by soft count = 607\n",
      "--- gene=1298, id=134 removed by soft count = 172\n",
      "--- gene=1298, id=190 removed by soft count = 62\n",
      "--- gene=1298, id=198 removed by soft count = 54\n",
      "--- gene=1298, id=235 removed by soft count = 56\n",
      "--- gene=1298, id=278 removed by soft count = 53\n",
      "--- number of probes: 282 , kept by blast: 275 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1299\n",
      "--- gene=1299, id=232 removed by soft count = 98\n",
      "--- gene=1299, id=248 removed by soft count = 52\n",
      "--- number of probes: 267 , kept by blast: 265 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 13\n",
      "--- gene=13, id=197 removed by soft count = 52\n",
      "--- gene=13, id=202 removed by soft count = 290\n",
      "--- gene=13, id=204 removed by soft count = 74\n",
      "--- gene=13, id=205 removed by soft count = 246\n",
      "--- gene=13, id=206 removed by soft count = 52\n",
      "--- gene=13, id=207 removed by soft count = 890\n",
      "--- gene=13, id=208 removed by soft count = 141\n",
      "--- gene=13, id=210 removed by soft count = 159\n",
      "--- number of probes: 341 , kept by blast: 333 , if remove dups: 331\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 130\n",
      "--- gene=130, id=59 removed by soft count = 61\n",
      "--- gene=130, id=70 removed by soft count = 365\n",
      "--- gene=130, id=92 removed by soft count = 74\n",
      "--- gene=130, id=127 removed by soft count = 688\n",
      "--- gene=130, id=145 removed by soft count = 72\n",
      "--- gene=130, id=152 removed by soft count = 124\n",
      "--- gene=130, id=153 removed by soft count = 250\n",
      "--- gene=130, id=154 removed by soft count = 608\n",
      "--- gene=130, id=155 removed by soft count = 271\n",
      "--- gene=130, id=156 removed by soft count = 136\n",
      "--- gene=130, id=157 removed by soft count = 195\n",
      "--- gene=130, id=158 removed by soft count = 214\n",
      "--- gene=130, id=159 removed by soft count = 376\n",
      "--- number of probes: 162 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1300\n",
      "--- gene=1300, id=39 removed by soft count = 673\n",
      "--- gene=1300, id=66 removed by soft count = 9304\n",
      "--- gene=1300, id=87 removed by soft count = 120\n",
      "--- gene=1300, id=108 removed by soft count = 95\n",
      "--- gene=1300, id=173 removed by soft count = 80\n",
      "--- gene=1300, id=174 removed by soft count = 249\n",
      "--- gene=1300, id=216 removed by soft count = 74\n",
      "--- number of probes: 266 , kept by blast: 259 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1301\n",
      "--- gene=1301, id=2 removed by soft count = 391\n",
      "--- gene=1301, id=7 removed by soft count = 57\n",
      "--- gene=1301, id=15 removed by soft count = 108\n",
      "--- gene=1301, id=23 removed by soft count = 66\n",
      "--- gene=1301, id=49 removed by soft count = 144\n",
      "--- gene=1301, id=94 removed by soft count = 841\n",
      "--- gene=1301, id=95 removed by soft count = 336\n",
      "--- gene=1301, id=99 removed by soft count = 134\n",
      "--- gene=1301, id=101 removed by soft count = 63\n",
      "--- gene=1301, id=102 removed by soft count = 54\n",
      "--- gene=1301, id=104 removed by soft count = 51\n",
      "--- gene=1301, id=106 removed by soft count = 105\n",
      "--- gene=1301, id=109 removed by soft count = 134\n",
      "--- gene=1301, id=167 removed by soft count = 65\n",
      "--- gene=1301, id=226 removed by soft count = 121\n",
      "--- gene=1301, id=228 removed by soft count = 152\n",
      "--- number of probes: 252 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1302\n",
      "--- gene=1302, id=68 removed by soft count = 87\n",
      "--- gene=1302, id=154 removed by soft count = 3767\n",
      "--- gene=1302, id=205 removed by soft count = 254\n",
      "--- gene=1302, id=207 removed by soft count = 113\n",
      "--- gene=1302, id=211 removed by soft count = 157\n",
      "--- gene=1302, id=212 removed by soft count = 51\n",
      "--- number of probes: 220 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1303\n",
      "--- gene=1303, id=123 removed by soft count = 137\n",
      "--- gene=1303, id=125 removed by soft count = 185\n",
      "--- gene=1303, id=126 removed by soft count = 105\n",
      "--- gene=1303, id=133 removed by soft count = 86\n",
      "--- gene=1303, id=155 removed by soft count = 82\n",
      "--- number of probes: 174 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1304\n",
      "--- gene=1304, id=127 removed by soft count = 18531\n",
      "--- gene=1304, id=128 removed by soft count = 1216\n",
      "--- gene=1304, id=168 removed by soft count = 1422\n",
      "--- number of probes: 268 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1305\n",
      "--- gene=1305, id=133 removed by soft count = 147\n",
      "--- number of probes: 297 , kept by blast: 296 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1306\n",
      "--- gene=1306, id=272 removed by soft count = 289\n",
      "--- number of probes: 285 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1307\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1307, id=13 removed by soft count = 52\n",
      "--- gene=1307, id=14 removed by soft count = 88\n",
      "--- gene=1307, id=16 removed by soft count = 58\n",
      "--- gene=1307, id=18 removed by soft count = 59\n",
      "--- gene=1307, id=61 removed by soft count = 77\n",
      "--- gene=1307, id=63 removed by soft count = 3097\n",
      "--- gene=1307, id=64 removed by soft count = 51\n",
      "--- gene=1307, id=144 removed by soft count = 4086\n",
      "--- gene=1307, id=168 removed by soft count = 104\n",
      "--- gene=1307, id=169 removed by soft count = 52\n",
      "--- number of probes: 271 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1308\n",
      "--- gene=1308, id=168 removed by soft count = 52\n",
      "--- gene=1308, id=191 removed by soft count = 80\n",
      "--- gene=1308, id=229 removed by soft count = 240\n",
      "--- number of probes: 239 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1309\n",
      "--- gene=1309, id=1 removed by soft count = 118\n",
      "--- gene=1309, id=127 removed by soft count = 134\n",
      "--- gene=1309, id=128 removed by soft count = 100\n",
      "--- gene=1309, id=200 removed by soft count = 101\n",
      "--- gene=1309, id=201 removed by soft count = 55\n",
      "--- gene=1309, id=216 removed by soft count = 55\n",
      "--- number of probes: 325 , kept by blast: 319 , if remove dups: 319\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 131\n",
      "--- gene=131, id=38 removed by soft count = 783\n",
      "--- gene=131, id=131 removed by soft count = 352\n",
      "--- gene=131, id=132 removed by soft count = 267\n",
      "--- gene=131, id=134 removed by soft count = 529\n",
      "--- gene=131, id=138 removed by soft count = 81\n",
      "--- gene=131, id=187 removed by soft count = 76\n",
      "--- gene=131, id=189 removed by soft count = 150\n",
      "--- number of probes: 190 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1310\n",
      "--- gene=1310, id=86 removed by soft count = 69\n",
      "--- gene=1310, id=196 removed by soft count = 160\n",
      "--- gene=1310, id=257 removed by soft count = 106\n",
      "--- number of probes: 285 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1311\n",
      "--- gene=1311, id=2 removed by soft count = 92\n",
      "--- gene=1311, id=3 removed by soft count = 90\n",
      "--- gene=1311, id=187 removed by soft count = 70\n",
      "--- gene=1311, id=188 removed by soft count = 52\n",
      "--- gene=1311, id=189 removed by soft count = 125\n",
      "--- number of probes: 211 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1312\n",
      "--- gene=1312, id=21 removed by soft count = 184\n",
      "--- gene=1312, id=227 removed by soft count = 149\n",
      "--- gene=1312, id=231 removed by soft count = 174\n",
      "--- number of probes: 257 , kept by blast: 254 , if remove dups: 254\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1313\n",
      "--- gene=1313, id=7 removed by soft count = 102\n",
      "--- gene=1313, id=79 removed by soft count = 378\n",
      "--- gene=1313, id=119 removed by soft count = 419\n",
      "--- gene=1313, id=124 removed by soft count = 308\n",
      "--- gene=1313, id=153 removed by soft count = 83\n",
      "--- gene=1313, id=154 removed by soft count = 152\n",
      "--- gene=1313, id=163 removed by soft count = 65\n",
      "--- gene=1313, id=165 removed by soft count = 122\n",
      "--- gene=1313, id=166 removed by soft count = 220\n",
      "--- gene=1313, id=167 removed by soft count = 544\n",
      "--- number of probes: 201 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1314\n",
      "--- gene=1314, id=193 removed by soft count = 1573\n",
      "--- number of probes: 272 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1315\n",
      "--- gene=1315, id=68 removed by soft count = 91\n",
      "--- gene=1315, id=163 removed by soft count = 71\n",
      "--- gene=1315, id=340 removed by soft count = 61\n",
      "--- gene=1315, id=341 removed by soft count = 942\n",
      "--- gene=1315, id=342 removed by soft count = 139\n",
      "--- gene=1315, id=343 removed by soft count = 336\n",
      "--- number of probes: 347 , kept by blast: 341 , if remove dups: 341\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1316\n",
      "--- gene=1316, id=238 removed by soft count = 783\n",
      "--- gene=1316, id=239 removed by soft count = 844\n",
      "--- gene=1316, id=240 removed by soft count = 58\n",
      "--- number of probes: 301 , kept by blast: 298 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1317\n",
      "--- gene=1317, id=62 removed by soft count = 53\n",
      "--- number of probes: 344 , kept by blast: 343 , if remove dups: 343\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1318\n",
      "--- gene=1318, id=2 removed by soft count = 82\n",
      "--- gene=1318, id=3 removed by soft count = 110\n",
      "--- gene=1318, id=112 removed by soft count = 102\n",
      "--- gene=1318, id=113 removed by soft count = 74\n",
      "--- gene=1318, id=114 removed by soft count = 69\n",
      "--- gene=1318, id=115 removed by soft count = 128\n",
      "--- gene=1318, id=116 removed by soft count = 172\n",
      "--- gene=1318, id=118 removed by soft count = 100\n",
      "--- gene=1318, id=120 removed by soft count = 344\n",
      "--- number of probes: 250 , kept by blast: 241 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1319\n",
      "--- gene=1319, id=55 removed by soft count = 212\n",
      "--- gene=1319, id=184 removed by soft count = 427\n",
      "--- number of probes: 385 , kept by blast: 383 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 132\n",
      "--- gene=132, id=35 removed by soft count = 76\n",
      "--- gene=132, id=36 removed by soft count = 923\n",
      "--- gene=132, id=37 removed by soft count = 201\n",
      "--- gene=132, id=38 removed by soft count = 56\n",
      "--- gene=132, id=39 removed by soft count = 84\n",
      "--- gene=132, id=105 removed by soft count = 58\n",
      "--- gene=132, id=106 removed by soft count = 69\n",
      "--- gene=132, id=108 removed by soft count = 110\n",
      "--- gene=132, id=110 removed by soft count = 122\n",
      "--- gene=132, id=111 removed by soft count = 116\n",
      "--- gene=132, id=112 removed by soft count = 318\n",
      "--- gene=132, id=113 removed by soft count = 265\n",
      "--- gene=132, id=114 removed by soft count = 238\n",
      "--- gene=132, id=115 removed by soft count = 63\n",
      "--- gene=132, id=169 removed by soft count = 62\n",
      "--- gene=132, id=174 removed by soft count = 64\n",
      "--- gene=132, id=183 removed by soft count = 67\n",
      "--- number of probes: 192 , kept by blast: 175 , if remove dups: 175\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1320\n",
      "--- gene=1320, id=2 removed by soft count = 53\n",
      "--- gene=1320, id=10 removed by soft count = 136\n",
      "--- gene=1320, id=20 removed by soft count = 56\n",
      "--- gene=1320, id=148 removed by soft count = 863\n",
      "--- gene=1320, id=150 removed by soft count = 72\n",
      "--- gene=1320, id=151 removed by soft count = 441\n",
      "--- gene=1320, id=152 removed by soft count = 359\n",
      "--- gene=1320, id=202 removed by soft count = 47177\n",
      "--- gene=1320, id=272 removed by soft count = 81\n",
      "--- number of probes: 286 , kept by blast: 277 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1321\n",
      "--- gene=1321, id=0 removed by soft count = 864\n",
      "--- gene=1321, id=1 removed by soft count = 68\n",
      "--- gene=1321, id=38 removed by soft count = 51\n",
      "--- gene=1321, id=81 removed by soft count = 60\n",
      "--- gene=1321, id=82 removed by soft count = 167\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1321, id=176 removed by soft count = 363\n",
      "--- number of probes: 321 , kept by blast: 315 , if remove dups: 315\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1322\n",
      "--- gene=1322, id=131 removed by soft count = 868\n",
      "--- gene=1322, id=132 removed by soft count = 349\n",
      "--- gene=1322, id=133 removed by soft count = 108\n",
      "--- number of probes: 310 , kept by blast: 307 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1323\n",
      "--- gene=1323, id=0 removed by soft count = 214\n",
      "--- gene=1323, id=3 removed by soft count = 117\n",
      "--- gene=1323, id=7 removed by soft count = 802\n",
      "--- gene=1323, id=8 removed by soft count = 310\n",
      "--- gene=1323, id=11 removed by soft count = 89\n",
      "--- gene=1323, id=12 removed by soft count = 229\n",
      "--- gene=1323, id=35 removed by soft count = 2992\n",
      "--- gene=1323, id=101 removed by soft count = 186\n",
      "--- gene=1323, id=185 removed by soft count = 66\n",
      "--- gene=1323, id=221 removed by soft count = 123\n",
      "--- gene=1323, id=291 removed by soft count = 322\n",
      "--- gene=1323, id=302 removed by soft count = 90\n",
      "--- number of probes: 306 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1324\n",
      "--- gene=1324, id=102 removed by soft count = 97\n",
      "--- gene=1324, id=161 removed by soft count = 116\n",
      "--- number of probes: 321 , kept by blast: 319 , if remove dups: 319\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1325\n",
      "--- gene=1325, id=260 removed by soft count = 65\n",
      "--- gene=1325, id=359 removed by soft count = 1869\n",
      "--- number of probes: 361 , kept by blast: 359 , if remove dups: 359\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 133\n",
      "--- gene=133, id=122 removed by soft count = 131\n",
      "--- gene=133, id=250 removed by soft count = 143\n",
      "--- gene=133, id=277 removed by soft count = 110\n",
      "--- number of probes: 312 , kept by blast: 309 , if remove dups: 309\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1330\n",
      "--- gene=1330, id=73 removed by soft count = 62\n",
      "--- gene=1330, id=128 removed by soft count = 86\n",
      "--- number of probes: 129 , kept by blast: 127 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1331\n",
      "--- number of probes: 242 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1332\n",
      "--- gene=1332, id=152 removed by soft count = 269\n",
      "--- gene=1332, id=153 removed by soft count = 338\n",
      "--- number of probes: 278 , kept by blast: 276 , if remove dups: 276\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1333\n",
      "--- gene=1333, id=38 removed by soft count = 2539\n",
      "--- gene=1333, id=91 removed by soft count = 105\n",
      "--- gene=1333, id=127 removed by soft count = 91\n",
      "--- number of probes: 136 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1334\n",
      "--- gene=1334, id=3 removed by soft count = 373\n",
      "--- gene=1334, id=21 removed by soft count = 143\n",
      "--- gene=1334, id=50 removed by soft count = 365\n",
      "--- gene=1334, id=56 removed by soft count = 481\n",
      "--- gene=1334, id=57 removed by soft count = 158\n",
      "--- gene=1334, id=58 removed by soft count = 80\n",
      "--- gene=1334, id=59 removed by soft count = 147\n",
      "--- gene=1334, id=84 removed by soft count = 2515\n",
      "--- gene=1334, id=85 removed by soft count = 2223\n",
      "--- gene=1334, id=125 removed by soft count = 56\n",
      "--- gene=1334, id=127 removed by soft count = 70\n",
      "--- gene=1334, id=159 removed by soft count = 64\n",
      "--- number of probes: 161 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1335\n",
      "--- gene=1335, id=142 removed by soft count = 489\n",
      "--- number of probes: 326 , kept by blast: 325 , if remove dups: 324\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1336\n",
      "--- gene=1336, id=38 removed by soft count = 99\n",
      "--- gene=1336, id=201 removed by soft count = 418\n",
      "--- gene=1336, id=205 removed by soft count = 680\n",
      "--- gene=1336, id=208 removed by soft count = 79\n",
      "--- gene=1336, id=212 removed by soft count = 176\n",
      "--- gene=1336, id=213 removed by soft count = 127\n",
      "--- gene=1336, id=228 removed by soft count = 67\n",
      "--- gene=1336, id=246 removed by soft count = 57\n",
      "--- number of probes: 266 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1337\n",
      "--- gene=1337, id=171 removed by soft count = 973\n",
      "--- number of probes: 338 , kept by blast: 337 , if remove dups: 332\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1338\n",
      "--- gene=1338, id=130 removed by soft count = 331\n",
      "--- gene=1338, id=133 removed by soft count = 217\n",
      "--- gene=1338, id=140 removed by soft count = 96\n",
      "--- gene=1338, id=142 removed by soft count = 364\n",
      "--- gene=1338, id=153 removed by soft count = 82\n",
      "--- number of probes: 188 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1339\n",
      "--- gene=1339, id=7 removed by soft count = 108\n",
      "--- gene=1339, id=9 removed by soft count = 86\n",
      "--- gene=1339, id=10 removed by soft count = 90\n",
      "--- gene=1339, id=65 removed by soft count = 59\n",
      "--- gene=1339, id=203 removed by soft count = 71\n",
      "--- gene=1339, id=231 removed by soft count = 118\n",
      "--- gene=1339, id=236 removed by soft count = 1651\n",
      "--- gene=1339, id=237 removed by soft count = 325\n",
      "--- number of probes: 238 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 134\n",
      "--- gene=134, id=82 removed by soft count = 1065\n",
      "--- gene=134, id=86 removed by soft count = 139\n",
      "--- gene=134, id=88 removed by soft count = 158\n",
      "--- gene=134, id=213 removed by soft count = 215\n",
      "--- gene=134, id=224 removed by soft count = 736\n",
      "--- number of probes: 237 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1340\n",
      "--- gene=1340, id=59 removed by soft count = 71\n",
      "--- gene=1340, id=267 removed by soft count = 81\n",
      "--- number of probes: 344 , kept by blast: 342 , if remove dups: 342\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1341\n",
      "--- gene=1341, id=60 removed by soft count = 455\n",
      "--- gene=1341, id=61 removed by soft count = 401\n",
      "--- gene=1341, id=62 removed by soft count = 537\n",
      "--- gene=1341, id=108 removed by soft count = 63\n",
      "--- gene=1341, id=147 removed by soft count = 178\n",
      "--- number of probes: 308 , kept by blast: 303 , if remove dups: 303\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1342\n",
      "--- gene=1342, id=59 removed by soft count = 51\n",
      "--- gene=1342, id=67 removed by soft count = 149\n",
      "--- gene=1342, id=105 removed by soft count = 69\n",
      "--- gene=1342, id=106 removed by soft count = 176\n",
      "--- gene=1342, id=108 removed by soft count = 81\n",
      "--- gene=1342, id=110 removed by soft count = 66\n",
      "--- gene=1342, id=117 removed by soft count = 113\n",
      "--- gene=1342, id=123 removed by soft count = 123\n",
      "--- gene=1342, id=128 removed by soft count = 106\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1342, id=229 removed by soft count = 154\n",
      "--- number of probes: 261 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1343\n",
      "--- gene=1343, id=16 removed by soft count = 249\n",
      "--- gene=1343, id=45 removed by soft count = 452\n",
      "--- gene=1343, id=50 removed by soft count = 239\n",
      "--- gene=1343, id=62 removed by soft count = 124\n",
      "--- gene=1343, id=79 removed by soft count = 765\n",
      "--- gene=1343, id=86 removed by soft count = 337\n",
      "--- gene=1343, id=95 removed by soft count = 64\n",
      "--- gene=1343, id=98 removed by soft count = 128\n",
      "--- gene=1343, id=99 removed by soft count = 313\n",
      "--- gene=1343, id=159 removed by soft count = 221\n",
      "--- gene=1343, id=165 removed by soft count = 114\n",
      "--- number of probes: 201 , kept by blast: 190 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1344\n",
      "--- gene=1344, id=190 removed by soft count = 2146\n",
      "--- gene=1344, id=221 removed by hard count = 0\n",
      "--- number of probes: 274 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1345\n",
      "--- gene=1345, id=1 removed by soft count = 87\n",
      "--- gene=1345, id=2 removed by soft count = 71\n",
      "--- gene=1345, id=5 removed by soft count = 77\n",
      "--- gene=1345, id=11 removed by soft count = 162\n",
      "--- gene=1345, id=24 removed by soft count = 101\n",
      "--- gene=1345, id=27 removed by soft count = 1051\n",
      "--- gene=1345, id=28 removed by soft count = 224\n",
      "--- gene=1345, id=219 removed by soft count = 69\n",
      "--- number of probes: 312 , kept by blast: 304 , if remove dups: 304\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1346\n",
      "--- gene=1346, id=61 removed by soft count = 116\n",
      "--- gene=1346, id=167 removed by soft count = 70\n",
      "--- gene=1346, id=290 removed by soft count = 82\n",
      "--- gene=1346, id=291 removed by soft count = 72\n",
      "--- number of probes: 298 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1347\n",
      "--- gene=1347, id=23 removed by soft count = 94\n",
      "--- gene=1347, id=31 removed by soft count = 4777\n",
      "--- gene=1347, id=167 removed by soft count = 2013\n",
      "--- number of probes: 179 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1348\n",
      "--- gene=1348, id=76 removed by soft count = 105\n",
      "--- gene=1348, id=86 removed by soft count = 83\n",
      "--- gene=1348, id=88 removed by soft count = 103\n",
      "--- gene=1348, id=89 removed by soft count = 96\n",
      "--- gene=1348, id=91 removed by soft count = 130\n",
      "--- gene=1348, id=92 removed by soft count = 52\n",
      "--- gene=1348, id=93 removed by soft count = 208\n",
      "--- gene=1348, id=94 removed by soft count = 88\n",
      "--- gene=1348, id=95 removed by soft count = 64\n",
      "--- gene=1348, id=96 removed by soft count = 113\n",
      "--- gene=1348, id=97 removed by soft count = 128\n",
      "--- gene=1348, id=98 removed by soft count = 140\n",
      "--- gene=1348, id=99 removed by soft count = 141\n",
      "--- gene=1348, id=100 removed by soft count = 119\n",
      "--- gene=1348, id=101 removed by soft count = 120\n",
      "--- gene=1348, id=102 removed by soft count = 136\n",
      "--- gene=1348, id=103 removed by soft count = 118\n",
      "--- gene=1348, id=104 removed by soft count = 125\n",
      "--- gene=1348, id=105 removed by soft count = 113\n",
      "--- gene=1348, id=106 removed by soft count = 326\n",
      "--- gene=1348, id=107 removed by soft count = 128\n",
      "--- gene=1348, id=108 removed by soft count = 203\n",
      "--- gene=1348, id=109 removed by soft count = 56\n",
      "--- gene=1348, id=110 removed by soft count = 296\n",
      "--- gene=1348, id=112 removed by soft count = 148\n",
      "--- gene=1348, id=113 removed by hard count = 4\n",
      "--- gene=1348, id=114 removed by soft count = 145\n",
      "--- gene=1348, id=115 removed by hard count = 4\n",
      "--- gene=1348, id=116 removed by soft count = 258\n",
      "--- gene=1348, id=117 removed by soft count = 124\n",
      "--- gene=1348, id=118 removed by soft count = 51\n",
      "--- gene=1348, id=119 removed by soft count = 144\n",
      "--- gene=1348, id=120 removed by soft count = 129\n",
      "--- gene=1348, id=121 removed by soft count = 157\n",
      "--- gene=1348, id=122 removed by soft count = 105\n",
      "--- number of probes: 123 , kept by blast: 88 , if remove dups: 88\n",
      "-- number of probes kept for this region: 88\n",
      "-- checking probes in region: 1349\n",
      "--- gene=1349, id=38 removed by soft count = 146\n",
      "--- gene=1349, id=39 removed by soft count = 872\n",
      "--- gene=1349, id=132 removed by soft count = 62\n",
      "--- gene=1349, id=144 removed by soft count = 168\n",
      "--- gene=1349, id=160 removed by soft count = 169\n",
      "--- gene=1349, id=161 removed by soft count = 70\n",
      "--- gene=1349, id=175 removed by soft count = 1238\n",
      "--- number of probes: 302 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1350\n",
      "--- gene=1350, id=0 removed by soft count = 67\n",
      "--- gene=1350, id=20 removed by soft count = 81\n",
      "--- gene=1350, id=35 removed by soft count = 66\n",
      "--- number of probes: 265 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1351\n",
      "--- gene=1351, id=204 removed by soft count = 125\n",
      "--- number of probes: 353 , kept by blast: 352 , if remove dups: 352\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1352\n",
      "--- gene=1352, id=13 removed by soft count = 182\n",
      "--- gene=1352, id=184 removed by soft count = 88\n",
      "--- gene=1352, id=186 removed by soft count = 307\n",
      "--- gene=1352, id=187 removed by soft count = 889\n",
      "--- gene=1352, id=189 removed by soft count = 323\n",
      "--- gene=1352, id=190 removed by soft count = 672\n",
      "--- gene=1352, id=192 removed by soft count = 323\n",
      "--- gene=1352, id=193 removed by soft count = 63\n",
      "--- number of probes: 243 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1353\n",
      "--- number of probes: 192 , kept by blast: 192 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1354\n",
      "--- number of probes: 278 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1355\n",
      "--- gene=1355, id=65 removed by soft count = 1364\n",
      "--- gene=1355, id=66 removed by soft count = 2750\n",
      "--- gene=1355, id=67 removed by soft count = 96\n",
      "--- gene=1355, id=68 removed by soft count = 113\n",
      "--- gene=1355, id=69 removed by soft count = 97\n",
      "--- gene=1355, id=70 removed by soft count = 488\n",
      "--- gene=1355, id=71 removed by soft count = 96\n",
      "--- gene=1355, id=73 removed by soft count = 94\n",
      "--- gene=1355, id=163 removed by soft count = 106\n",
      "--- gene=1355, id=164 removed by soft count = 98\n",
      "--- gene=1355, id=168 removed by soft count = 148\n",
      "--- gene=1355, id=171 removed by soft count = 86\n",
      "--- gene=1355, id=173 removed by soft count = 220\n",
      "--- gene=1355, id=175 removed by soft count = 51\n",
      "--- number of probes: 187 , kept by blast: 173 , if remove dups: 173\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1356\n",
      "--- gene=1356, id=227 removed by soft count = 66\n",
      "--- gene=1356, id=228 removed by soft count = 193\n",
      "--- gene=1356, id=245 removed by soft count = 128\n",
      "--- number of probes: 291 , kept by blast: 288 , if remove dups: 288\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1357\n",
      "--- gene=1357, id=3 removed by soft count = 729\n",
      "--- gene=1357, id=22 removed by soft count = 94\n",
      "--- gene=1357, id=94 removed by soft count = 620\n",
      "--- gene=1357, id=124 removed by soft count = 160\n",
      "--- gene=1357, id=131 removed by soft count = 114\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1357, id=165 removed by soft count = 75\n",
      "--- number of probes: 246 , kept by blast: 240 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1358\n",
      "--- gene=1358, id=24 removed by soft count = 127\n",
      "--- gene=1358, id=65 removed by soft count = 561\n",
      "--- gene=1358, id=289 removed by soft count = 71\n",
      "--- number of probes: 327 , kept by blast: 324 , if remove dups: 324\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1359\n",
      "--- gene=1359, id=38 removed by soft count = 76\n",
      "--- gene=1359, id=134 removed by soft count = 268\n",
      "--- gene=1359, id=202 removed by soft count = 64\n",
      "--- gene=1359, id=212 removed by soft count = 85\n",
      "--- gene=1359, id=239 removed by soft count = 838\n",
      "--- gene=1359, id=266 removed by soft count = 69\n",
      "--- number of probes: 267 , kept by blast: 261 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1360\n",
      "--- gene=1360, id=16 removed by soft count = 63\n",
      "--- gene=1360, id=43 removed by soft count = 141\n",
      "--- number of probes: 323 , kept by blast: 321 , if remove dups: 317\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1361\n",
      "--- gene=1361, id=339 removed by soft count = 79\n",
      "--- gene=1361, id=389 removed by soft count = 55\n",
      "--- gene=1361, id=391 removed by soft count = 71\n",
      "--- number of probes: 397 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1362\n",
      "--- gene=1362, id=170 removed by soft count = 79\n",
      "--- gene=1362, id=171 removed by soft count = 74\n",
      "--- gene=1362, id=172 removed by soft count = 69\n",
      "--- gene=1362, id=173 removed by soft count = 65\n",
      "--- gene=1362, id=174 removed by soft count = 71\n",
      "--- gene=1362, id=175 removed by soft count = 70\n",
      "--- gene=1362, id=176 removed by soft count = 64\n",
      "--- gene=1362, id=179 removed by soft count = 73\n",
      "--- gene=1362, id=205 removed by hard count = 0\n",
      "--- gene=1362, id=218 removed by soft count = 140\n",
      "--- number of probes: 379 , kept by blast: 369 , if remove dups: 368\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1363\n",
      "--- gene=1363, id=247 removed by soft count = 80\n",
      "--- gene=1363, id=284 removed by soft count = 113\n",
      "--- gene=1363, id=285 removed by soft count = 73\n",
      "--- gene=1363, id=295 removed by soft count = 133\n",
      "--- gene=1363, id=299 removed by hard count = 3\n",
      "--- gene=1363, id=303 removed by soft count = 98\n",
      "--- gene=1363, id=309 removed by soft count = 181\n",
      "--- number of probes: 310 , kept by blast: 303 , if remove dups: 301\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 137\n",
      "--- gene=137, id=93 removed by soft count = 2696\n",
      "--- number of probes: 315 , kept by blast: 314 , if remove dups: 314\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1378\n",
      "--- gene=1378, id=155 removed by soft count = 164\n",
      "--- gene=1378, id=157 removed by soft count = 350\n",
      "--- gene=1378, id=158 removed by soft count = 1095\n",
      "--- gene=1378, id=159 removed by soft count = 2260\n",
      "--- number of probes: 217 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1379\n",
      "--- gene=1379, id=175 removed by soft count = 60\n",
      "--- number of probes: 292 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 138\n",
      "--- gene=138, id=143 removed by soft count = 71\n",
      "--- gene=138, id=144 removed by soft count = 101\n",
      "--- gene=138, id=158 removed by soft count = 54\n",
      "--- gene=138, id=166 removed by soft count = 80\n",
      "--- number of probes: 285 , kept by blast: 281 , if remove dups: 281\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1380\n",
      "--- gene=1380, id=4 removed by soft count = 92\n",
      "--- gene=1380, id=44 removed by soft count = 193\n",
      "--- gene=1380, id=190 removed by soft count = 182\n",
      "--- gene=1380, id=191 removed by soft count = 104\n",
      "--- number of probes: 192 , kept by blast: 188 , if remove dups: 188\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1381\n",
      "--- gene=1381, id=117 removed by soft count = 55\n",
      "--- gene=1381, id=141 removed by soft count = 130\n",
      "--- gene=1381, id=149 removed by soft count = 236\n",
      "--- gene=1381, id=151 removed by soft count = 887\n",
      "--- number of probes: 214 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1382\n",
      "--- gene=1382, id=28 removed by soft count = 554\n",
      "--- gene=1382, id=104 removed by soft count = 324\n",
      "--- gene=1382, id=148 removed by soft count = 52\n",
      "--- gene=1382, id=149 removed by soft count = 305\n",
      "--- gene=1382, id=175 removed by soft count = 1963\n",
      "--- gene=1382, id=178 removed by soft count = 190\n",
      "--- number of probes: 210 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1383\n",
      "--- gene=1383, id=20 removed by soft count = 53\n",
      "--- gene=1383, id=73 removed by soft count = 1164\n",
      "--- gene=1383, id=74 removed by soft count = 238\n",
      "--- gene=1383, id=110 removed by soft count = 125\n",
      "--- gene=1383, id=111 removed by soft count = 94\n",
      "--- gene=1383, id=112 removed by soft count = 107\n",
      "--- gene=1383, id=139 removed by soft count = 643\n",
      "--- gene=1383, id=140 removed by soft count = 59\n",
      "--- gene=1383, id=141 removed by soft count = 65\n",
      "--- gene=1383, id=146 removed by soft count = 69\n",
      "--- gene=1383, id=147 removed by soft count = 281\n",
      "--- gene=1383, id=148 removed by soft count = 93\n",
      "--- gene=1383, id=152 removed by soft count = 124\n",
      "--- gene=1383, id=187 removed by soft count = 182\n",
      "--- gene=1383, id=192 removed by soft count = 93\n",
      "--- gene=1383, id=228 removed by soft count = 301\n",
      "--- gene=1383, id=229 removed by soft count = 59\n",
      "--- number of probes: 230 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1384\n",
      "--- gene=1384, id=33 removed by soft count = 479\n",
      "--- gene=1384, id=39 removed by soft count = 92\n",
      "--- gene=1384, id=58 removed by soft count = 189\n",
      "--- gene=1384, id=72 removed by soft count = 171\n",
      "--- gene=1384, id=73 removed by soft count = 129\n",
      "--- gene=1384, id=75 removed by soft count = 4197\n",
      "--- gene=1384, id=76 removed by soft count = 272\n",
      "--- gene=1384, id=77 removed by soft count = 968\n",
      "--- gene=1384, id=78 removed by soft count = 420\n",
      "--- gene=1384, id=79 removed by soft count = 597\n",
      "--- gene=1384, id=83 removed by soft count = 232\n",
      "--- gene=1384, id=84 removed by soft count = 256\n",
      "--- gene=1384, id=87 removed by soft count = 63\n",
      "--- gene=1384, id=88 removed by soft count = 302\n",
      "--- gene=1384, id=123 removed by soft count = 66\n",
      "--- gene=1384, id=131 removed by soft count = 414\n",
      "--- number of probes: 148 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1385\n",
      "--- gene=1385, id=19 removed by soft count = 71\n",
      "--- gene=1385, id=41 removed by soft count = 73\n",
      "--- gene=1385, id=43 removed by soft count = 55\n",
      "--- gene=1385, id=56 removed by soft count = 416\n",
      "--- gene=1385, id=57 removed by soft count = 307\n",
      "--- gene=1385, id=105 removed by soft count = 109\n",
      "--- gene=1385, id=148 removed by soft count = 164\n",
      "--- gene=1385, id=149 removed by soft count = 55\n",
      "--- gene=1385, id=152 removed by soft count = 314\n",
      "--- gene=1385, id=153 removed by soft count = 624\n",
      "--- number of probes: 163 , kept by blast: 153 , if remove dups: 153\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1386\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1386, id=8 removed by soft count = 102\n",
      "--- gene=1386, id=36 removed by soft count = 130\n",
      "--- gene=1386, id=37 removed by soft count = 176\n",
      "--- gene=1386, id=48 removed by soft count = 1000\n",
      "--- gene=1386, id=49 removed by soft count = 357\n",
      "--- gene=1386, id=50 removed by soft count = 274\n",
      "--- gene=1386, id=104 removed by soft count = 721\n",
      "--- gene=1386, id=105 removed by soft count = 875\n",
      "--- gene=1386, id=107 removed by soft count = 56\n",
      "--- gene=1386, id=108 removed by soft count = 93\n",
      "--- gene=1386, id=113 removed by soft count = 168\n",
      "--- number of probes: 128 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 117\n",
      "-- checking probes in region: 1387\n",
      "--- gene=1387, id=16 removed by soft count = 73\n",
      "--- gene=1387, id=91 removed by soft count = 912\n",
      "--- gene=1387, id=92 removed by soft count = 106\n",
      "--- gene=1387, id=93 removed by soft count = 185\n",
      "--- gene=1387, id=94 removed by soft count = 384\n",
      "--- gene=1387, id=100 removed by soft count = 1346\n",
      "--- number of probes: 190 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1388\n",
      "--- gene=1388, id=21 removed by soft count = 665\n",
      "--- gene=1388, id=96 removed by soft count = 69\n",
      "--- gene=1388, id=103 removed by soft count = 122\n",
      "--- gene=1388, id=104 removed by soft count = 762\n",
      "--- gene=1388, id=121 removed by soft count = 98\n",
      "--- number of probes: 176 , kept by blast: 171 , if remove dups: 171\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1389\n",
      "--- gene=1389, id=68 removed by soft count = 833\n",
      "--- gene=1389, id=157 removed by soft count = 53\n",
      "--- gene=1389, id=159 removed by soft count = 52\n",
      "--- gene=1389, id=232 removed by soft count = 73\n",
      "--- gene=1389, id=235 removed by soft count = 126\n",
      "--- gene=1389, id=236 removed by soft count = 171\n",
      "--- number of probes: 251 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 139\n",
      "--- gene=139, id=0 removed by soft count = 205\n",
      "--- gene=139, id=2 removed by soft count = 93\n",
      "--- gene=139, id=4 removed by soft count = 221\n",
      "--- gene=139, id=5 removed by soft count = 144\n",
      "--- gene=139, id=15 removed by soft count = 59\n",
      "--- gene=139, id=30 removed by soft count = 294\n",
      "--- gene=139, id=59 removed by soft count = 87\n",
      "--- gene=139, id=119 removed by soft count = 89\n",
      "--- gene=139, id=145 removed by soft count = 55\n",
      "--- number of probes: 174 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1390\n",
      "--- gene=1390, id=119 removed by soft count = 381\n",
      "--- gene=1390, id=130 removed by soft count = 104\n",
      "--- number of probes: 288 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1391\n",
      "--- gene=1391, id=83 removed by soft count = 71\n",
      "--- gene=1391, id=160 removed by soft count = 307\n",
      "--- gene=1391, id=187 removed by soft count = 2665\n",
      "--- gene=1391, id=195 removed by soft count = 64\n",
      "--- gene=1391, id=206 removed by soft count = 387\n",
      "--- number of probes: 233 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1392\n",
      "--- gene=1392, id=103 removed by soft count = 136\n",
      "--- gene=1392, id=117 removed by soft count = 3754\n",
      "--- gene=1392, id=159 removed by soft count = 491\n",
      "--- gene=1392, id=161 removed by soft count = 357\n",
      "--- gene=1392, id=165 removed by soft count = 242\n",
      "--- gene=1392, id=166 removed by soft count = 431\n",
      "--- gene=1392, id=167 removed by soft count = 105\n",
      "--- number of probes: 244 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1393\n",
      "--- gene=1393, id=41 removed by soft count = 188\n",
      "--- gene=1393, id=42 removed by soft count = 75\n",
      "--- gene=1393, id=43 removed by soft count = 102\n",
      "--- gene=1393, id=44 removed by soft count = 188\n",
      "--- gene=1393, id=242 removed by soft count = 58\n",
      "--- number of probes: 323 , kept by blast: 318 , if remove dups: 315\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1394\n",
      "--- gene=1394, id=2 removed by soft count = 120\n",
      "--- gene=1394, id=4 removed by soft count = 145\n",
      "--- gene=1394, id=5 removed by soft count = 86\n",
      "--- gene=1394, id=6 removed by soft count = 130\n",
      "--- gene=1394, id=7 removed by soft count = 185\n",
      "--- gene=1394, id=81 removed by soft count = 72\n",
      "--- gene=1394, id=99 removed by soft count = 1279\n",
      "--- gene=1394, id=100 removed by soft count = 286\n",
      "--- gene=1394, id=138 removed by soft count = 798\n",
      "--- number of probes: 174 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1395\n",
      "--- gene=1395, id=41 removed by soft count = 429\n",
      "--- gene=1395, id=73 removed by soft count = 262\n",
      "--- gene=1395, id=75 removed by soft count = 211\n",
      "--- gene=1395, id=98 removed by soft count = 63\n",
      "--- gene=1395, id=105 removed by soft count = 64\n",
      "--- gene=1395, id=106 removed by soft count = 433\n",
      "--- gene=1395, id=109 removed by soft count = 74\n",
      "--- gene=1395, id=206 removed by soft count = 38962\n",
      "--- gene=1395, id=241 removed by soft count = 56\n",
      "--- gene=1395, id=243 removed by soft count = 67\n",
      "--- gene=1395, id=246 removed by soft count = 88\n",
      "--- number of probes: 282 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1396\n",
      "--- gene=1396, id=25 removed by soft count = 147\n",
      "--- gene=1396, id=26 removed by soft count = 95\n",
      "--- gene=1396, id=40 removed by soft count = 357\n",
      "--- gene=1396, id=139 removed by soft count = 215\n",
      "--- gene=1396, id=146 removed by soft count = 256\n",
      "--- gene=1396, id=161 removed by soft count = 78\n",
      "--- gene=1396, id=165 removed by soft count = 82\n",
      "--- gene=1396, id=183 removed by soft count = 268\n",
      "--- number of probes: 192 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1398\n",
      "--- gene=1398, id=0 removed by soft count = 113\n",
      "--- gene=1398, id=2 removed by soft count = 619\n",
      "--- gene=1398, id=3 removed by soft count = 281\n",
      "--- gene=1398, id=4 removed by soft count = 158\n",
      "--- gene=1398, id=5 removed by soft count = 86\n",
      "--- gene=1398, id=6 removed by soft count = 120\n",
      "--- gene=1398, id=7 removed by soft count = 58\n",
      "--- gene=1398, id=8 removed by soft count = 139\n",
      "--- gene=1398, id=13 removed by soft count = 51\n",
      "--- gene=1398, id=14 removed by soft count = 112\n",
      "--- gene=1398, id=15 removed by soft count = 265\n",
      "--- gene=1398, id=16 removed by soft count = 92\n",
      "--- gene=1398, id=73 removed by soft count = 82\n",
      "--- gene=1398, id=74 removed by soft count = 482\n",
      "--- gene=1398, id=122 removed by soft count = 58\n",
      "--- number of probes: 184 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1399\n",
      "--- gene=1399, id=58 removed by soft count = 58\n",
      "--- gene=1399, id=119 removed by soft count = 97\n",
      "--- number of probes: 268 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 14\n",
      "--- gene=14, id=35 removed by soft count = 319\n",
      "--- gene=14, id=56 removed by soft count = 179\n",
      "--- gene=14, id=174 removed by soft count = 124\n",
      "--- number of probes: 176 , kept by blast: 173 , if remove dups: 173\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 140\n",
      "--- gene=140, id=121 removed by soft count = 124\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=140, id=215 removed by soft count = 164\n",
      "--- gene=140, id=234 removed by soft count = 217\n",
      "--- number of probes: 261 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1400\n",
      "--- gene=1400, id=1 removed by soft count = 56\n",
      "--- gene=1400, id=65 removed by soft count = 66\n",
      "--- gene=1400, id=179 removed by soft count = 121\n",
      "--- number of probes: 308 , kept by blast: 305 , if remove dups: 305\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1401\n",
      "--- gene=1401, id=91 removed by soft count = 73\n",
      "--- gene=1401, id=113 removed by soft count = 175\n",
      "--- number of probes: 268 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1402\n",
      "--- gene=1402, id=0 removed by soft count = 332\n",
      "--- gene=1402, id=1 removed by soft count = 262\n",
      "--- gene=1402, id=4 removed by soft count = 64\n",
      "--- gene=1402, id=7 removed by soft count = 76\n",
      "--- gene=1402, id=260 removed by soft count = 66\n",
      "--- number of probes: 308 , kept by blast: 303 , if remove dups: 303\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1403\n",
      "--- gene=1403, id=38 removed by soft count = 80\n",
      "--- gene=1403, id=184 removed by hard count = 0\n",
      "--- number of probes: 223 , kept by blast: 221 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1404\n",
      "--- gene=1404, id=73 removed by soft count = 153\n",
      "--- gene=1404, id=79 removed by soft count = 74\n",
      "--- gene=1404, id=81 removed by soft count = 248\n",
      "--- gene=1404, id=82 removed by soft count = 60\n",
      "--- gene=1404, id=83 removed by soft count = 82\n",
      "--- gene=1404, id=119 removed by soft count = 52\n",
      "--- gene=1404, id=218 removed by soft count = 712\n",
      "--- gene=1404, id=219 removed by soft count = 95\n",
      "--- gene=1404, id=220 removed by soft count = 293\n",
      "--- number of probes: 235 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1405\n",
      "--- gene=1405, id=17 removed by soft count = 67\n",
      "--- gene=1405, id=18 removed by soft count = 538\n",
      "--- gene=1405, id=47 removed by soft count = 52\n",
      "--- gene=1405, id=153 removed by soft count = 90\n",
      "--- gene=1405, id=155 removed by soft count = 55\n",
      "--- gene=1405, id=286 removed by soft count = 77\n",
      "--- gene=1405, id=300 removed by soft count = 63\n",
      "--- gene=1405, id=305 removed by soft count = 109\n",
      "--- gene=1405, id=309 removed by soft count = 493\n",
      "--- gene=1405, id=310 removed by soft count = 555\n",
      "--- gene=1405, id=312 removed by soft count = 65\n",
      "--- gene=1405, id=314 removed by soft count = 238\n",
      "--- gene=1405, id=316 removed by soft count = 203\n",
      "--- gene=1405, id=318 removed by soft count = 78\n",
      "--- gene=1405, id=320 removed by soft count = 115\n",
      "--- gene=1405, id=322 removed by soft count = 176\n",
      "--- gene=1405, id=323 removed by soft count = 343\n",
      "--- gene=1405, id=324 removed by soft count = 495\n",
      "--- gene=1405, id=325 removed by soft count = 78\n",
      "--- gene=1405, id=326 removed by soft count = 146\n",
      "--- gene=1405, id=329 removed by soft count = 69\n",
      "--- gene=1405, id=331 removed by soft count = 332\n",
      "--- gene=1405, id=334 removed by soft count = 167\n",
      "--- number of probes: 336 , kept by blast: 313 , if remove dups: 313\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1406\n",
      "--- gene=1406, id=50 removed by soft count = 95\n",
      "--- gene=1406, id=51 removed by soft count = 175\n",
      "--- gene=1406, id=73 removed by soft count = 51\n",
      "--- gene=1406, id=74 removed by soft count = 92\n",
      "--- gene=1406, id=164 removed by soft count = 63\n",
      "--- gene=1406, id=258 removed by soft count = 160\n",
      "--- gene=1406, id=259 removed by soft count = 62\n",
      "--- gene=1406, id=261 removed by soft count = 290\n",
      "--- gene=1406, id=287 removed by soft count = 95\n",
      "--- number of probes: 328 , kept by blast: 319 , if remove dups: 319\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1408\n",
      "--- gene=1408, id=6 removed by soft count = 64\n",
      "--- gene=1408, id=20 removed by soft count = 63\n",
      "--- gene=1408, id=22 removed by soft count = 70\n",
      "--- gene=1408, id=25 removed by soft count = 80\n",
      "--- gene=1408, id=28 removed by soft count = 54\n",
      "--- gene=1408, id=29 removed by soft count = 74\n",
      "--- gene=1408, id=31 removed by soft count = 151\n",
      "--- gene=1408, id=32 removed by soft count = 120\n",
      "--- gene=1408, id=36 removed by soft count = 51\n",
      "--- gene=1408, id=37 removed by soft count = 641\n",
      "--- gene=1408, id=38 removed by soft count = 189\n",
      "--- gene=1408, id=54 removed by soft count = 105\n",
      "--- gene=1408, id=55 removed by soft count = 278\n",
      "--- gene=1408, id=64 removed by soft count = 58\n",
      "--- gene=1408, id=118 removed by soft count = 106\n",
      "--- gene=1408, id=195 removed by soft count = 52\n",
      "--- number of probes: 326 , kept by blast: 310 , if remove dups: 308\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1409\n",
      "--- gene=1409, id=178 removed by hard count = 0\n",
      "--- gene=1409, id=210 removed by soft count = 120\n",
      "--- gene=1409, id=211 removed by hard count = 0\n",
      "--- gene=1409, id=291 removed by soft count = 60\n",
      "--- gene=1409, id=293 removed by soft count = 56\n",
      "--- number of probes: 369 , kept by blast: 364 , if remove dups: 364\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 141\n",
      "--- gene=141, id=0 removed by soft count = 405\n",
      "--- gene=141, id=1 removed by soft count = 181\n",
      "--- gene=141, id=2 removed by soft count = 305\n",
      "--- gene=141, id=5 removed by soft count = 79\n",
      "--- gene=141, id=12 removed by soft count = 79\n",
      "--- gene=141, id=19 removed by soft count = 109\n",
      "--- number of probes: 74 , kept by blast: 68 , if remove dups: 65\n",
      "-- number of probes kept for this region: 68\n",
      "-- checking probes in region: 1427\n",
      "--- gene=1427, id=9 removed by soft count = 315\n",
      "--- gene=1427, id=45 removed by soft count = 243\n",
      "--- gene=1427, id=185 removed by soft count = 549\n",
      "--- gene=1427, id=228 removed by soft count = 329\n",
      "--- number of probes: 253 , kept by blast: 249 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1429\n",
      "--- gene=1429, id=26 removed by soft count = 119\n",
      "--- gene=1429, id=134 removed by soft count = 53\n",
      "--- number of probes: 266 , kept by blast: 264 , if remove dups: 263\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 143\n",
      "--- gene=143, id=151 removed by soft count = 280\n",
      "--- number of probes: 186 , kept by blast: 185 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1430\n",
      "--- gene=1430, id=48 removed by soft count = 629\n",
      "--- gene=1430, id=49 removed by soft count = 54\n",
      "--- gene=1430, id=50 removed by soft count = 113\n",
      "--- gene=1430, id=52 removed by soft count = 196\n",
      "--- gene=1430, id=53 removed by soft count = 53\n",
      "--- gene=1430, id=154 removed by soft count = 58\n",
      "--- gene=1430, id=156 removed by soft count = 62\n",
      "--- number of probes: 240 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1434\n",
      "--- gene=1434, id=92 removed by soft count = 191\n",
      "--- gene=1434, id=191 removed by soft count = 58\n",
      "--- number of probes: 295 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1435\n",
      "--- gene=1435, id=28 removed by soft count = 93\n",
      "--- gene=1435, id=29 removed by soft count = 119\n",
      "--- gene=1435, id=30 removed by soft count = 108\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 255 , kept by blast: 252 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1436\n",
      "--- gene=1436, id=127 removed by soft count = 632\n",
      "--- number of probes: 298 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1437\n",
      "--- gene=1437, id=0 removed by soft count = 123\n",
      "--- gene=1437, id=69 removed by soft count = 84\n",
      "--- gene=1437, id=85 removed by soft count = 129\n",
      "--- number of probes: 360 , kept by blast: 357 , if remove dups: 357\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1438\n",
      "--- gene=1438, id=67 removed by soft count = 108\n",
      "--- gene=1438, id=154 removed by soft count = 82\n",
      "--- gene=1438, id=155 removed by soft count = 93\n",
      "--- gene=1438, id=156 removed by soft count = 94\n",
      "--- gene=1438, id=159 removed by soft count = 148\n",
      "--- gene=1438, id=160 removed by soft count = 885\n",
      "--- gene=1438, id=161 removed by soft count = 11701\n",
      "--- gene=1438, id=162 removed by soft count = 1195\n",
      "--- number of probes: 174 , kept by blast: 166 , if remove dups: 166\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1439\n",
      "--- gene=1439, id=93 removed by soft count = 2103\n",
      "--- gene=1439, id=126 removed by soft count = 51\n",
      "--- gene=1439, id=194 removed by soft count = 53\n",
      "--- number of probes: 196 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 144\n",
      "--- gene=144, id=5 removed by soft count = 103\n",
      "--- gene=144, id=29 removed by soft count = 68\n",
      "--- gene=144, id=37 removed by soft count = 347\n",
      "--- gene=144, id=84 removed by soft count = 83\n",
      "--- gene=144, id=184 removed by soft count = 176\n",
      "--- gene=144, id=186 removed by soft count = 78\n",
      "--- number of probes: 187 , kept by blast: 181 , if remove dups: 181\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1440\n",
      "--- gene=1440, id=188 removed by soft count = 388\n",
      "--- gene=1440, id=199 removed by soft count = 213\n",
      "--- gene=1440, id=207 removed by soft count = 2561\n",
      "--- gene=1440, id=213 removed by soft count = 98\n",
      "--- gene=1440, id=215 removed by soft count = 178\n",
      "--- number of probes: 236 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1441\n",
      "--- gene=1441, id=117 removed by soft count = 597\n",
      "--- gene=1441, id=120 removed by soft count = 276\n",
      "--- gene=1441, id=140 removed by soft count = 87\n",
      "--- gene=1441, id=149 removed by soft count = 108\n",
      "--- gene=1441, id=152 removed by soft count = 117\n",
      "--- gene=1441, id=164 removed by soft count = 113\n",
      "--- gene=1441, id=167 removed by soft count = 156\n",
      "--- number of probes: 184 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1442\n",
      "--- gene=1442, id=107 removed by soft count = 426\n",
      "--- gene=1442, id=109 removed by soft count = 249\n",
      "--- gene=1442, id=165 removed by soft count = 839\n",
      "--- gene=1442, id=166 removed by soft count = 212\n",
      "--- gene=1442, id=168 removed by soft count = 147\n",
      "--- number of probes: 248 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1443\n",
      "--- gene=1443, id=124 removed by soft count = 70\n",
      "--- gene=1443, id=308 removed by soft count = 61\n",
      "--- number of probes: 316 , kept by blast: 314 , if remove dups: 314\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1444\n",
      "--- gene=1444, id=6 removed by soft count = 134\n",
      "--- gene=1444, id=132 removed by soft count = 286\n",
      "--- gene=1444, id=133 removed by soft count = 221\n",
      "--- gene=1444, id=137 removed by soft count = 161\n",
      "--- gene=1444, id=138 removed by soft count = 58\n",
      "--- gene=1444, id=139 removed by soft count = 190\n",
      "--- gene=1444, id=239 removed by soft count = 427\n",
      "--- gene=1444, id=240 removed by soft count = 3684\n",
      "--- number of probes: 314 , kept by blast: 306 , if remove dups: 306\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1445\n",
      "--- gene=1445, id=83 removed by soft count = 95\n",
      "--- gene=1445, id=146 removed by soft count = 47085\n",
      "--- gene=1445, id=168 removed by soft count = 693\n",
      "--- gene=1445, id=192 removed by soft count = 900\n",
      "--- gene=1445, id=218 removed by soft count = 250\n",
      "--- gene=1445, id=219 removed by soft count = 319\n",
      "--- number of probes: 312 , kept by blast: 306 , if remove dups: 306\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1446\n",
      "--- gene=1446, id=12 removed by soft count = 134\n",
      "--- gene=1446, id=19 removed by soft count = 239\n",
      "--- gene=1446, id=53 removed by soft count = 332\n",
      "--- number of probes: 274 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1447\n",
      "--- number of probes: 380 , kept by blast: 380 , if remove dups: 379\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1448\n",
      "--- gene=1448, id=111 removed by soft count = 59\n",
      "--- gene=1448, id=228 removed by soft count = 729\n",
      "--- gene=1448, id=229 removed by soft count = 53\n",
      "--- gene=1448, id=231 removed by soft count = 52\n",
      "--- gene=1448, id=243 removed by soft count = 2794\n",
      "--- gene=1448, id=265 removed by soft count = 191\n",
      "--- gene=1448, id=266 removed by soft count = 850\n",
      "--- number of probes: 267 , kept by blast: 260 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1449\n",
      "--- gene=1449, id=65 removed by soft count = 53\n",
      "--- gene=1449, id=231 removed by soft count = 9384\n",
      "--- gene=1449, id=251 removed by soft count = 79\n",
      "--- number of probes: 341 , kept by blast: 338 , if remove dups: 338\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 145\n",
      "--- gene=145, id=0 removed by soft count = 3822\n",
      "--- gene=145, id=29 removed by soft count = 52\n",
      "--- gene=145, id=50 removed by soft count = 74\n",
      "--- gene=145, id=59 removed by soft count = 83\n",
      "--- gene=145, id=76 removed by soft count = 68\n",
      "--- gene=145, id=120 removed by soft count = 615\n",
      "--- gene=145, id=121 removed by soft count = 71\n",
      "--- gene=145, id=122 removed by soft count = 201\n",
      "--- gene=145, id=123 removed by soft count = 199\n",
      "--- gene=145, id=124 removed by soft count = 101\n",
      "--- gene=145, id=125 removed by soft count = 143\n",
      "--- gene=145, id=126 removed by soft count = 141\n",
      "--- gene=145, id=135 removed by soft count = 244\n",
      "--- gene=145, id=136 removed by soft count = 384\n",
      "--- gene=145, id=138 removed by soft count = 105\n",
      "--- gene=145, id=140 removed by soft count = 74\n",
      "--- gene=145, id=141 removed by soft count = 419\n",
      "--- gene=145, id=142 removed by soft count = 1371\n",
      "--- gene=145, id=153 removed by soft count = 143\n",
      "--- gene=145, id=159 removed by soft count = 188\n",
      "--- gene=145, id=175 removed by soft count = 51\n",
      "--- number of probes: 178 , kept by blast: 157 , if remove dups: 157\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1450\n",
      "--- gene=1450, id=88 removed by soft count = 91\n",
      "--- gene=1450, id=116 removed by soft count = 388\n",
      "--- number of probes: 211 , kept by blast: 209 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1451\n",
      "--- gene=1451, id=61 removed by soft count = 250\n",
      "--- gene=1451, id=62 removed by soft count = 79\n",
      "--- gene=1451, id=63 removed by soft count = 153\n",
      "--- gene=1451, id=64 removed by soft count = 102\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1451, id=120 removed by soft count = 78\n",
      "--- number of probes: 255 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1452\n",
      "--- gene=1452, id=49 removed by soft count = 227\n",
      "--- gene=1452, id=52 removed by soft count = 2181\n",
      "--- number of probes: 113 , kept by blast: 111 , if remove dups: 109\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 109\n",
      "-- checking probes in region: 1453\n",
      "--- gene=1453, id=77 removed by soft count = 300\n",
      "--- gene=1453, id=109 removed by soft count = 54\n",
      "--- gene=1453, id=110 removed by soft count = 122\n",
      "--- gene=1453, id=193 removed by soft count = 341\n",
      "--- gene=1453, id=240 removed by soft count = 52\n",
      "--- gene=1453, id=293 removed by soft count = 64\n",
      "--- number of probes: 304 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1454\n",
      "--- gene=1454, id=28 removed by soft count = 256\n",
      "--- gene=1454, id=82 removed by soft count = 51\n",
      "--- gene=1454, id=184 removed by soft count = 81\n",
      "--- gene=1454, id=185 removed by soft count = 151\n",
      "--- gene=1454, id=186 removed by soft count = 59\n",
      "--- gene=1454, id=213 removed by soft count = 52\n",
      "--- gene=1454, id=214 removed by soft count = 106\n",
      "--- gene=1454, id=219 removed by soft count = 166\n",
      "--- gene=1454, id=231 removed by soft count = 59\n",
      "--- number of probes: 277 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1455\n",
      "--- gene=1455, id=2 removed by soft count = 59\n",
      "--- gene=1455, id=20 removed by soft count = 79\n",
      "--- number of probes: 290 , kept by blast: 288 , if remove dups: 288\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1456\n",
      "--- gene=1456, id=78 removed by soft count = 179\n",
      "--- gene=1456, id=83 removed by soft count = 98\n",
      "--- number of probes: 270 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1457\n",
      "--- gene=1457, id=29 removed by soft count = 410\n",
      "--- gene=1457, id=65 removed by soft count = 240\n",
      "--- gene=1457, id=162 removed by soft count = 116\n",
      "--- gene=1457, id=186 removed by soft count = 83\n",
      "--- gene=1457, id=187 removed by soft count = 115\n",
      "--- gene=1457, id=201 removed by soft count = 213\n",
      "--- gene=1457, id=202 removed by soft count = 359\n",
      "--- gene=1457, id=203 removed by soft count = 91\n",
      "--- gene=1457, id=245 removed by soft count = 112\n",
      "--- number of probes: 281 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1458\n",
      "--- gene=1458, id=166 removed by hard count = 6\n",
      "--- gene=1458, id=329 removed by soft count = 157\n",
      "--- gene=1458, id=334 removed by soft count = 700\n",
      "--- gene=1458, id=335 removed by soft count = 62\n",
      "--- number of probes: 396 , kept by blast: 392 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1459\n",
      "--- gene=1459, id=9 removed by soft count = 254\n",
      "--- gene=1459, id=141 removed by soft count = 120\n",
      "--- gene=1459, id=266 removed by soft count = 85\n",
      "--- gene=1459, id=277 removed by soft count = 261\n",
      "--- gene=1459, id=280 removed by soft count = 52\n",
      "--- gene=1459, id=298 removed by soft count = 115\n",
      "--- gene=1459, id=299 removed by soft count = 69\n",
      "--- gene=1459, id=301 removed by soft count = 55\n",
      "--- gene=1459, id=302 removed by soft count = 346\n",
      "--- number of probes: 323 , kept by blast: 314 , if remove dups: 314\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 146\n",
      "--- gene=146, id=0 removed by soft count = 1060\n",
      "--- gene=146, id=33 removed by soft count = 319\n",
      "--- gene=146, id=36 removed by soft count = 102\n",
      "--- gene=146, id=72 removed by soft count = 1061\n",
      "--- gene=146, id=73 removed by soft count = 70\n",
      "--- gene=146, id=74 removed by soft count = 193\n",
      "--- gene=146, id=75 removed by soft count = 2239\n",
      "--- gene=146, id=244 removed by soft count = 365\n",
      "--- gene=146, id=245 removed by soft count = 80\n",
      "--- gene=146, id=247 removed by soft count = 2202\n",
      "--- number of probes: 294 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1461\n",
      "--- gene=1461, id=21 removed by soft count = 453\n",
      "--- gene=1461, id=24 removed by soft count = 247\n",
      "--- gene=1461, id=38 removed by soft count = 270\n",
      "--- gene=1461, id=39 removed by soft count = 272\n",
      "--- gene=1461, id=65 removed by soft count = 268\n",
      "--- gene=1461, id=70 removed by soft count = 215\n",
      "--- gene=1461, id=78 removed by soft count = 131\n",
      "--- gene=1461, id=79 removed by soft count = 53\n",
      "--- gene=1461, id=83 removed by soft count = 73\n",
      "--- gene=1461, id=106 removed by soft count = 374\n",
      "--- gene=1461, id=114 removed by soft count = 63\n",
      "--- number of probes: 143 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1462\n",
      "--- gene=1462, id=25 removed by soft count = 84\n",
      "--- gene=1462, id=53 removed by soft count = 93\n",
      "--- gene=1462, id=188 removed by soft count = 235\n",
      "--- number of probes: 238 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1463\n",
      "--- gene=1463, id=8 removed by soft count = 101\n",
      "--- gene=1463, id=164 removed by soft count = 489\n",
      "--- gene=1463, id=210 removed by soft count = 134\n",
      "--- gene=1463, id=242 removed by soft count = 150\n",
      "--- number of probes: 268 , kept by blast: 264 , if remove dups: 263\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1464\n",
      "--- gene=1464, id=0 removed by soft count = 56\n",
      "--- gene=1464, id=22 removed by soft count = 176\n",
      "--- gene=1464, id=23 removed by soft count = 127\n",
      "--- gene=1464, id=37 removed by soft count = 62\n",
      "--- gene=1464, id=83 removed by soft count = 128\n",
      "--- gene=1464, id=119 removed by soft count = 51\n",
      "--- gene=1464, id=130 removed by soft count = 815\n",
      "--- gene=1464, id=136 removed by soft count = 71\n",
      "--- gene=1464, id=233 removed by soft count = 212\n",
      "--- number of probes: 281 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1465\n",
      "--- gene=1465, id=19 removed by soft count = 158\n",
      "--- gene=1465, id=40 removed by soft count = 141\n",
      "--- gene=1465, id=42 removed by soft count = 93\n",
      "--- number of probes: 231 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1466\n",
      "--- gene=1466, id=2 removed by soft count = 68\n",
      "--- gene=1466, id=51 removed by soft count = 60\n",
      "--- gene=1466, id=179 removed by soft count = 90\n",
      "--- number of probes: 286 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1467\n",
      "--- gene=1467, id=0 removed by soft count = 430\n",
      "--- gene=1467, id=14 removed by soft count = 135\n",
      "--- gene=1467, id=39 removed by soft count = 60\n",
      "--- gene=1467, id=95 removed by soft count = 100\n",
      "--- gene=1467, id=96 removed by soft count = 139\n",
      "--- gene=1467, id=97 removed by soft count = 68\n",
      "--- gene=1467, id=98 removed by soft count = 96\n",
      "--- gene=1467, id=100 removed by soft count = 131\n",
      "--- gene=1467, id=172 removed by soft count = 336\n",
      "--- number of probes: 193 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1468\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1468, id=6 removed by soft count = 598\n",
      "--- gene=1468, id=7 removed by soft count = 81\n",
      "--- gene=1468, id=9 removed by soft count = 99\n",
      "--- gene=1468, id=10 removed by soft count = 114\n",
      "--- gene=1468, id=12 removed by soft count = 342\n",
      "--- gene=1468, id=15 removed by soft count = 224\n",
      "--- gene=1468, id=19 removed by soft count = 56\n",
      "--- gene=1468, id=20 removed by soft count = 324\n",
      "--- gene=1468, id=21 removed by soft count = 90\n",
      "--- gene=1468, id=22 removed by soft count = 126\n",
      "--- gene=1468, id=25 removed by soft count = 63\n",
      "--- gene=1468, id=28 removed by soft count = 149\n",
      "--- gene=1468, id=29 removed by soft count = 72\n",
      "--- gene=1468, id=32 removed by soft count = 146\n",
      "--- gene=1468, id=34 removed by soft count = 57\n",
      "--- gene=1468, id=41 removed by soft count = 81\n",
      "--- gene=1468, id=42 removed by soft count = 84\n",
      "--- gene=1468, id=47 removed by soft count = 55\n",
      "--- gene=1468, id=54 removed by soft count = 61\n",
      "--- number of probes: 180 , kept by blast: 161 , if remove dups: 161\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1469\n",
      "--- gene=1469, id=96 removed by soft count = 344\n",
      "--- number of probes: 202 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 147\n",
      "--- gene=147, id=114 removed by soft count = 317\n",
      "--- gene=147, id=116 removed by soft count = 262\n",
      "--- gene=147, id=135 removed by soft count = 108\n",
      "--- gene=147, id=207 removed by soft count = 1150\n",
      "--- gene=147, id=220 removed by soft count = 249\n",
      "--- gene=147, id=288 removed by soft count = 54\n",
      "--- number of probes: 355 , kept by blast: 349 , if remove dups: 349\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1470\n",
      "--- gene=1470, id=18 removed by soft count = 701\n",
      "--- gene=1470, id=20 removed by soft count = 1099\n",
      "--- gene=1470, id=88 removed by soft count = 386\n",
      "--- gene=1470, id=89 removed by soft count = 414\n",
      "--- gene=1470, id=90 removed by soft count = 368\n",
      "--- gene=1470, id=91 removed by soft count = 224\n",
      "--- gene=1470, id=167 removed by soft count = 86\n",
      "--- gene=1470, id=183 removed by soft count = 406\n",
      "--- number of probes: 194 , kept by blast: 186 , if remove dups: 186\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1471\n",
      "--- gene=1471, id=99 removed by soft count = 11759\n",
      "--- gene=1471, id=149 removed by soft count = 79\n",
      "--- gene=1471, id=191 removed by soft count = 77\n",
      "--- gene=1471, id=192 removed by soft count = 66\n",
      "--- number of probes: 221 , kept by blast: 217 , if remove dups: 216\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1472\n",
      "--- gene=1472, id=104 removed by soft count = 72\n",
      "--- gene=1472, id=107 removed by soft count = 180\n",
      "--- gene=1472, id=108 removed by soft count = 453\n",
      "--- gene=1472, id=109 removed by soft count = 309\n",
      "--- number of probes: 278 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1473\n",
      "--- gene=1473, id=42 removed by soft count = 189\n",
      "--- gene=1473, id=44 removed by soft count = 183\n",
      "--- gene=1473, id=45 removed by soft count = 256\n",
      "--- gene=1473, id=64 removed by soft count = 196\n",
      "--- gene=1473, id=67 removed by soft count = 353\n",
      "--- gene=1473, id=70 removed by soft count = 285\n",
      "--- gene=1473, id=73 removed by soft count = 1114\n",
      "--- gene=1473, id=74 removed by soft count = 74\n",
      "--- gene=1473, id=75 removed by soft count = 234\n",
      "--- gene=1473, id=76 removed by soft count = 200\n",
      "--- gene=1473, id=77 removed by soft count = 1264\n",
      "--- gene=1473, id=79 removed by soft count = 627\n",
      "--- gene=1473, id=83 removed by soft count = 274\n",
      "--- gene=1473, id=87 removed by soft count = 474\n",
      "--- gene=1473, id=88 removed by soft count = 70\n",
      "--- gene=1473, id=89 removed by soft count = 69\n",
      "--- gene=1473, id=96 removed by soft count = 59\n",
      "--- number of probes: 117 , kept by blast: 100 , if remove dups: 100\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: 1474\n",
      "--- gene=1474, id=32 removed by soft count = 66\n",
      "--- gene=1474, id=45 removed by soft count = 60\n",
      "--- gene=1474, id=51 removed by soft count = 270\n",
      "--- gene=1474, id=53 removed by soft count = 180\n",
      "--- gene=1474, id=54 removed by soft count = 66\n",
      "--- gene=1474, id=61 removed by soft count = 117\n",
      "--- gene=1474, id=64 removed by soft count = 301\n",
      "--- gene=1474, id=158 removed by soft count = 709\n",
      "--- gene=1474, id=159 removed by soft count = 211\n",
      "--- gene=1474, id=160 removed by soft count = 127\n",
      "--- gene=1474, id=167 removed by soft count = 226\n",
      "--- gene=1474, id=170 removed by soft count = 151\n",
      "--- gene=1474, id=172 removed by soft count = 91\n",
      "--- number of probes: 173 , kept by blast: 160 , if remove dups: 160\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1475\n",
      "--- gene=1475, id=40 removed by soft count = 383\n",
      "--- gene=1475, id=41 removed by soft count = 1627\n",
      "--- gene=1475, id=43 removed by soft count = 1222\n",
      "--- gene=1475, id=44 removed by soft count = 84\n",
      "--- gene=1475, id=48 removed by soft count = 142\n",
      "--- gene=1475, id=50 removed by soft count = 329\n",
      "--- gene=1475, id=51 removed by soft count = 639\n",
      "--- gene=1475, id=52 removed by soft count = 301\n",
      "--- gene=1475, id=110 removed by soft count = 439\n",
      "--- number of probes: 234 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1476\n",
      "--- gene=1476, id=155 removed by soft count = 54\n",
      "--- gene=1476, id=184 removed by soft count = 3859\n",
      "--- gene=1476, id=186 removed by soft count = 60\n",
      "--- gene=1476, id=189 removed by soft count = 51\n",
      "--- gene=1476, id=191 removed by soft count = 283\n",
      "--- gene=1476, id=196 removed by soft count = 105\n",
      "--- number of probes: 197 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1477\n",
      "--- gene=1477, id=184 removed by soft count = 130\n",
      "--- number of probes: 341 , kept by blast: 340 , if remove dups: 340\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1478\n",
      "--- gene=1478, id=17 removed by soft count = 343\n",
      "--- gene=1478, id=18 removed by soft count = 51\n",
      "--- gene=1478, id=19 removed by soft count = 129\n",
      "--- gene=1478, id=26 removed by soft count = 104\n",
      "--- gene=1478, id=172 removed by soft count = 79\n",
      "--- gene=1478, id=174 removed by soft count = 472\n",
      "--- gene=1478, id=175 removed by soft count = 154\n",
      "--- gene=1478, id=177 removed by soft count = 115\n",
      "--- gene=1478, id=178 removed by soft count = 56\n",
      "--- gene=1478, id=179 removed by soft count = 58\n",
      "--- gene=1478, id=204 removed by soft count = 227\n",
      "--- gene=1478, id=211 removed by soft count = 140\n",
      "--- number of probes: 230 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1479\n",
      "--- gene=1479, id=44 removed by soft count = 57\n",
      "--- gene=1479, id=77 removed by soft count = 175\n",
      "--- gene=1479, id=98 removed by soft count = 54\n",
      "--- gene=1479, id=99 removed by soft count = 232\n",
      "--- gene=1479, id=163 removed by soft count = 260\n",
      "--- gene=1479, id=164 removed by soft count = 185\n",
      "--- gene=1479, id=165 removed by soft count = 158\n",
      "--- gene=1479, id=166 removed by soft count = 60\n",
      "--- gene=1479, id=171 removed by soft count = 339\n",
      "--- gene=1479, id=173 removed by soft count = 158\n",
      "--- gene=1479, id=183 removed by soft count = 352\n",
      "--- number of probes: 216 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 148\n",
      "--- gene=148, id=190 removed by soft count = 188\n",
      "--- number of probes: 193 , kept by blast: 192 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1480\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1480, id=12 removed by soft count = 64\n",
      "--- gene=1480, id=24 removed by soft count = 77\n",
      "--- gene=1480, id=28 removed by soft count = 64\n",
      "--- gene=1480, id=31 removed by soft count = 162\n",
      "--- gene=1480, id=32 removed by soft count = 348\n",
      "--- gene=1480, id=33 removed by soft count = 51\n",
      "--- gene=1480, id=36 removed by soft count = 256\n",
      "--- gene=1480, id=37 removed by soft count = 85\n",
      "--- gene=1480, id=38 removed by soft count = 96\n",
      "--- gene=1480, id=40 removed by soft count = 54\n",
      "--- gene=1480, id=56 removed by soft count = 207\n",
      "--- gene=1480, id=62 removed by soft count = 82\n",
      "--- gene=1480, id=68 removed by soft count = 75\n",
      "--- number of probes: 117 , kept by blast: 104 , if remove dups: 104\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 104\n",
      "-- checking probes in region: 1481\n",
      "--- gene=1481, id=59 removed by soft count = 83\n",
      "--- gene=1481, id=82 removed by soft count = 501\n",
      "--- gene=1481, id=84 removed by soft count = 92\n",
      "--- gene=1481, id=92 removed by soft count = 123\n",
      "--- gene=1481, id=93 removed by soft count = 70\n",
      "--- number of probes: 198 , kept by blast: 193 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1482\n",
      "--- gene=1482, id=66 removed by soft count = 58\n",
      "--- gene=1482, id=67 removed by soft count = 72\n",
      "--- gene=1482, id=68 removed by soft count = 66\n",
      "--- gene=1482, id=69 removed by soft count = 61\n",
      "--- gene=1482, id=109 removed by soft count = 287\n",
      "--- gene=1482, id=110 removed by soft count = 244\n",
      "--- gene=1482, id=136 removed by soft count = 345\n",
      "--- gene=1482, id=139 removed by soft count = 173\n",
      "--- gene=1482, id=146 removed by soft count = 533\n",
      "--- gene=1482, id=147 removed by soft count = 164\n",
      "--- gene=1482, id=148 removed by soft count = 174\n",
      "--- gene=1482, id=149 removed by soft count = 89\n",
      "--- number of probes: 212 , kept by blast: 200 , if remove dups: 200\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1485\n",
      "--- gene=1485, id=343 removed by soft count = 308\n",
      "--- gene=1485, id=344 removed by soft count = 243\n",
      "--- gene=1485, id=350 removed by soft count = 78\n",
      "--- number of probes: 351 , kept by blast: 348 , if remove dups: 348\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1486\n",
      "--- gene=1486, id=1 removed by soft count = 259\n",
      "--- gene=1486, id=2 removed by soft count = 1141\n",
      "--- gene=1486, id=3 removed by soft count = 333\n",
      "--- gene=1486, id=5 removed by soft count = 74\n",
      "--- gene=1486, id=6 removed by soft count = 98\n",
      "--- gene=1486, id=7 removed by soft count = 335\n",
      "--- gene=1486, id=9 removed by soft count = 126\n",
      "--- gene=1486, id=21 removed by soft count = 169\n",
      "--- gene=1486, id=24 removed by soft count = 129\n",
      "--- gene=1486, id=25 removed by soft count = 75\n",
      "--- gene=1486, id=26 removed by soft count = 230\n",
      "--- gene=1486, id=27 removed by soft count = 88\n",
      "--- gene=1486, id=31 removed by soft count = 1933\n",
      "--- gene=1486, id=32 removed by soft count = 672\n",
      "--- gene=1486, id=33 removed by soft count = 73\n",
      "--- gene=1486, id=34 removed by soft count = 276\n",
      "--- gene=1486, id=35 removed by soft count = 129\n",
      "--- gene=1486, id=37 removed by soft count = 52\n",
      "--- gene=1486, id=51 removed by soft count = 83\n",
      "--- gene=1486, id=62 removed by soft count = 128\n",
      "--- gene=1486, id=87 removed by soft count = 199\n",
      "--- number of probes: 227 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1487\n",
      "--- gene=1487, id=2 removed by soft count = 469\n",
      "--- gene=1487, id=101 removed by soft count = 216\n",
      "--- gene=1487, id=249 removed by soft count = 260\n",
      "--- number of probes: 250 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1488\n",
      "--- gene=1488, id=28 removed by soft count = 656\n",
      "--- gene=1488, id=289 removed by soft count = 8615\n",
      "--- gene=1488, id=290 removed by soft count = 306\n",
      "--- gene=1488, id=294 removed by soft count = 51\n",
      "--- number of probes: 311 , kept by blast: 307 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1489\n",
      "--- gene=1489, id=0 removed by soft count = 229\n",
      "--- gene=1489, id=39 removed by soft count = 171\n",
      "--- gene=1489, id=40 removed by soft count = 158\n",
      "--- gene=1489, id=42 removed by soft count = 276\n",
      "--- gene=1489, id=44 removed by soft count = 56\n",
      "--- gene=1489, id=47 removed by soft count = 781\n",
      "--- gene=1489, id=49 removed by soft count = 149\n",
      "--- gene=1489, id=50 removed by soft count = 185\n",
      "--- gene=1489, id=52 removed by soft count = 112\n",
      "--- gene=1489, id=53 removed by soft count = 120\n",
      "--- gene=1489, id=54 removed by soft count = 77\n",
      "--- gene=1489, id=56 removed by soft count = 217\n",
      "--- gene=1489, id=57 removed by soft count = 267\n",
      "--- gene=1489, id=69 removed by soft count = 224\n",
      "--- gene=1489, id=71 removed by soft count = 91\n",
      "--- gene=1489, id=77 removed by soft count = 56\n",
      "--- gene=1489, id=83 removed by soft count = 201\n",
      "--- gene=1489, id=85 removed by soft count = 331\n",
      "--- gene=1489, id=86 removed by soft count = 13653\n",
      "--- gene=1489, id=88 removed by soft count = 92\n",
      "--- gene=1489, id=91 removed by soft count = 56\n",
      "--- number of probes: 94 , kept by blast: 73 , if remove dups: 73\n",
      "-- number of probes kept for this region: 73\n",
      "-- checking probes in region: 149\n",
      "--- number of probes: 309 , kept by blast: 309 , if remove dups: 309\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1490\n",
      "--- gene=1490, id=1 removed by soft count = 106\n",
      "--- gene=1490, id=2 removed by soft count = 14436\n",
      "--- gene=1490, id=6 removed by soft count = 587\n",
      "--- gene=1490, id=7 removed by soft count = 135\n",
      "--- gene=1490, id=10 removed by soft count = 257\n",
      "--- gene=1490, id=12 removed by soft count = 113\n",
      "--- gene=1490, id=13 removed by soft count = 373\n",
      "--- gene=1490, id=19 removed by soft count = 55\n",
      "--- gene=1490, id=41 removed by soft count = 72\n",
      "--- gene=1490, id=93 removed by soft count = 100\n",
      "--- gene=1490, id=121 removed by soft count = 115\n",
      "--- gene=1490, id=123 removed by soft count = 120\n",
      "--- gene=1490, id=125 removed by soft count = 194\n",
      "--- gene=1490, id=126 removed by soft count = 217\n",
      "--- gene=1490, id=127 removed by soft count = 88\n",
      "--- number of probes: 128 , kept by blast: 113 , if remove dups: 113\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 113\n",
      "-- checking probes in region: 1491\n",
      "--- gene=1491, id=8 removed by soft count = 89\n",
      "--- gene=1491, id=11 removed by soft count = 80\n",
      "--- gene=1491, id=12 removed by soft count = 60\n",
      "--- gene=1491, id=17 removed by soft count = 62\n",
      "--- gene=1491, id=20 removed by soft count = 612\n",
      "--- gene=1491, id=23 removed by soft count = 51\n",
      "--- gene=1491, id=24 removed by soft count = 66\n",
      "--- gene=1491, id=28 removed by soft count = 104\n",
      "--- gene=1491, id=31 removed by soft count = 165\n",
      "--- gene=1491, id=32 removed by soft count = 124\n",
      "--- gene=1491, id=226 removed by soft count = 52\n",
      "--- gene=1491, id=228 removed by soft count = 52\n",
      "--- number of probes: 239 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1492\n",
      "--- gene=1492, id=26 removed by soft count = 139\n",
      "--- gene=1492, id=28 removed by soft count = 62\n",
      "--- gene=1492, id=30 removed by soft count = 54\n",
      "--- gene=1492, id=32 removed by soft count = 117\n",
      "--- gene=1492, id=34 removed by soft count = 104\n",
      "--- gene=1492, id=35 removed by soft count = 257\n",
      "--- gene=1492, id=37 removed by soft count = 53\n",
      "--- gene=1492, id=39 removed by soft count = 1073\n",
      "--- gene=1492, id=41 removed by soft count = 440\n",
      "--- gene=1492, id=46 removed by soft count = 132\n",
      "--- gene=1492, id=47 removed by soft count = 257\n",
      "--- gene=1492, id=48 removed by soft count = 66\n",
      "--- gene=1492, id=49 removed by soft count = 117\n",
      "--- gene=1492, id=52 removed by soft count = 75\n",
      "--- gene=1492, id=57 removed by soft count = 1588\n",
      "--- gene=1492, id=59 removed by soft count = 137\n",
      "--- gene=1492, id=61 removed by soft count = 57\n",
      "--- gene=1492, id=79 removed by soft count = 66\n",
      "--- gene=1492, id=80 removed by soft count = 69\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1492, id=82 removed by soft count = 72\n",
      "--- gene=1492, id=83 removed by soft count = 201\n",
      "--- gene=1492, id=93 removed by soft count = 79\n",
      "--- gene=1492, id=152 removed by soft count = 91\n",
      "--- number of probes: 238 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1493\n",
      "--- gene=1493, id=0 removed by soft count = 546\n",
      "--- gene=1493, id=1 removed by soft count = 278\n",
      "--- gene=1493, id=10 removed by soft count = 58\n",
      "--- gene=1493, id=13 removed by soft count = 248\n",
      "--- gene=1493, id=14 removed by soft count = 189\n",
      "--- gene=1493, id=17 removed by soft count = 223\n",
      "--- gene=1493, id=24 removed by soft count = 133\n",
      "--- gene=1493, id=25 removed by soft count = 83\n",
      "--- gene=1493, id=26 removed by soft count = 652\n",
      "--- gene=1493, id=28 removed by soft count = 109\n",
      "--- gene=1493, id=29 removed by soft count = 86\n",
      "--- gene=1493, id=30 removed by soft count = 93\n",
      "--- gene=1493, id=32 removed by soft count = 119\n",
      "--- gene=1493, id=37 removed by soft count = 81\n",
      "--- gene=1493, id=55 removed by soft count = 185\n",
      "--- number of probes: 129 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 114\n",
      "-- checking probes in region: 1494\n",
      "--- gene=1494, id=10 removed by soft count = 1026\n",
      "--- gene=1494, id=12 removed by soft count = 64\n",
      "--- gene=1494, id=13 removed by soft count = 191\n",
      "--- gene=1494, id=17 removed by soft count = 109\n",
      "--- gene=1494, id=37 removed by soft count = 313\n",
      "--- gene=1494, id=38 removed by soft count = 2134\n",
      "--- gene=1494, id=39 removed by soft count = 430\n",
      "--- gene=1494, id=40 removed by soft count = 468\n",
      "--- gene=1494, id=60 removed by soft count = 99\n",
      "--- gene=1494, id=62 removed by soft count = 148\n",
      "--- gene=1494, id=63 removed by soft count = 56\n",
      "--- gene=1494, id=64 removed by soft count = 208\n",
      "--- gene=1494, id=65 removed by soft count = 102\n",
      "--- gene=1494, id=67 removed by soft count = 620\n",
      "--- gene=1494, id=68 removed by soft count = 1947\n",
      "--- gene=1494, id=70 removed by soft count = 116\n",
      "--- gene=1494, id=71 removed by soft count = 123\n",
      "--- gene=1494, id=81 removed by soft count = 2077\n",
      "--- number of probes: 189 , kept by blast: 171 , if remove dups: 171\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1495\n",
      "--- gene=1495, id=0 removed by soft count = 86\n",
      "--- gene=1495, id=13 removed by soft count = 59\n",
      "--- gene=1495, id=35 removed by soft count = 129\n",
      "--- gene=1495, id=36 removed by soft count = 138\n",
      "--- gene=1495, id=40 removed by soft count = 53\n",
      "--- gene=1495, id=181 removed by soft count = 183\n",
      "--- gene=1495, id=183 removed by soft count = 207\n",
      "--- gene=1495, id=187 removed by soft count = 69\n",
      "--- number of probes: 199 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1496\n",
      "--- gene=1496, id=7 removed by soft count = 141\n",
      "--- gene=1496, id=8 removed by soft count = 367\n",
      "--- gene=1496, id=9 removed by soft count = 64\n",
      "--- gene=1496, id=10 removed by soft count = 96\n",
      "--- gene=1496, id=19 removed by soft count = 273\n",
      "--- gene=1496, id=36 removed by soft count = 67\n",
      "--- gene=1496, id=61 removed by soft count = 69\n",
      "--- gene=1496, id=65 removed by soft count = 63\n",
      "--- gene=1496, id=68 removed by soft count = 109\n",
      "--- number of probes: 72 , kept by blast: 63 , if remove dups: 62\n",
      "-- number of probes kept for this region: 63\n",
      "-- checking probes in region: 1497\n",
      "--- gene=1497, id=0 removed by soft count = 917\n",
      "--- gene=1497, id=1 removed by soft count = 111\n",
      "--- gene=1497, id=2 removed by soft count = 148\n",
      "--- gene=1497, id=3 removed by soft count = 354\n",
      "--- gene=1497, id=4 removed by soft count = 107\n",
      "--- gene=1497, id=6 removed by soft count = 329\n",
      "--- gene=1497, id=7 removed by soft count = 255\n",
      "--- gene=1497, id=10 removed by soft count = 54\n",
      "--- gene=1497, id=35 removed by soft count = 477\n",
      "--- gene=1497, id=41 removed by soft count = 58\n",
      "--- gene=1497, id=47 removed by soft count = 52\n",
      "--- gene=1497, id=185 removed by soft count = 152\n",
      "--- gene=1497, id=208 removed by soft count = 110\n",
      "--- gene=1497, id=211 removed by soft count = 77\n",
      "--- gene=1497, id=215 removed by soft count = 52\n",
      "--- number of probes: 230 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1498\n",
      "--- gene=1498, id=15 removed by soft count = 754\n",
      "--- gene=1498, id=42 removed by soft count = 117\n",
      "--- gene=1498, id=51 removed by soft count = 63\n",
      "--- gene=1498, id=116 removed by soft count = 131\n",
      "--- gene=1498, id=125 removed by soft count = 17446\n",
      "--- gene=1498, id=134 removed by soft count = 356\n",
      "--- gene=1498, id=135 removed by soft count = 78\n",
      "--- number of probes: 146 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1499\n",
      "--- gene=1499, id=34 removed by soft count = 138\n",
      "--- gene=1499, id=74 removed by soft count = 241\n",
      "--- gene=1499, id=75 removed by soft count = 346\n",
      "--- gene=1499, id=77 removed by soft count = 93\n",
      "--- gene=1499, id=78 removed by soft count = 197\n",
      "--- gene=1499, id=80 removed by soft count = 180\n",
      "--- gene=1499, id=81 removed by soft count = 110\n",
      "--- gene=1499, id=83 removed by soft count = 74\n",
      "--- gene=1499, id=89 removed by soft count = 1209\n",
      "--- gene=1499, id=90 removed by soft count = 68\n",
      "--- gene=1499, id=91 removed by soft count = 57\n",
      "--- gene=1499, id=93 removed by soft count = 131\n",
      "--- gene=1499, id=95 removed by soft count = 255\n",
      "--- gene=1499, id=97 removed by soft count = 81\n",
      "--- gene=1499, id=101 removed by soft count = 102\n",
      "--- gene=1499, id=102 removed by soft count = 188\n",
      "--- gene=1499, id=103 removed by soft count = 124\n",
      "--- gene=1499, id=105 removed by soft count = 105\n",
      "--- gene=1499, id=144 removed by soft count = 217\n",
      "--- gene=1499, id=146 removed by soft count = 365\n",
      "--- gene=1499, id=153 removed by soft count = 6628\n",
      "--- gene=1499, id=154 removed by soft count = 384\n",
      "--- gene=1499, id=155 removed by soft count = 199\n",
      "--- number of probes: 160 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 15\n",
      "--- gene=15, id=102 removed by soft count = 655\n",
      "--- gene=15, id=295 removed by soft count = 100\n",
      "--- gene=15, id=307 removed by soft count = 63\n",
      "--- gene=15, id=310 removed by soft count = 78\n",
      "--- number of probes: 343 , kept by blast: 339 , if remove dups: 339\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 150\n",
      "--- gene=150, id=66 removed by soft count = 163\n",
      "--- gene=150, id=94 removed by soft count = 505\n",
      "--- gene=150, id=228 removed by soft count = 56\n",
      "--- gene=150, id=230 removed by soft count = 191\n",
      "--- gene=150, id=233 removed by soft count = 60\n",
      "--- gene=150, id=236 removed by soft count = 81\n",
      "--- gene=150, id=238 removed by soft count = 8688\n",
      "--- number of probes: 240 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1500\n",
      "--- gene=1500, id=267 removed by soft count = 78\n",
      "--- number of probes: 369 , kept by blast: 368 , if remove dups: 368\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1501\n",
      "--- gene=1501, id=70 removed by soft count = 166\n",
      "--- gene=1501, id=71 removed by soft count = 71\n",
      "--- gene=1501, id=72 removed by soft count = 63\n",
      "--- gene=1501, id=134 removed by soft count = 904\n",
      "--- gene=1501, id=141 removed by soft count = 81\n",
      "--- gene=1501, id=218 removed by soft count = 107\n",
      "--- gene=1501, id=224 removed by soft count = 323\n",
      "--- gene=1501, id=225 removed by soft count = 99\n",
      "--- gene=1501, id=226 removed by soft count = 263\n",
      "--- gene=1501, id=232 removed by soft count = 325\n",
      "--- gene=1501, id=249 removed by soft count = 180\n",
      "--- gene=1501, id=250 removed by soft count = 839\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1501, id=251 removed by soft count = 1001\n",
      "--- gene=1501, id=252 removed by soft count = 239\n",
      "--- number of probes: 253 , kept by blast: 239 , if remove dups: 239\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1502\n",
      "--- gene=1502, id=0 removed by soft count = 276\n",
      "--- gene=1502, id=2 removed by soft count = 320\n",
      "--- gene=1502, id=80 removed by soft count = 122\n",
      "--- gene=1502, id=81 removed by soft count = 109\n",
      "--- gene=1502, id=88 removed by soft count = 303\n",
      "--- gene=1502, id=90 removed by soft count = 185\n",
      "--- gene=1502, id=94 removed by soft count = 106\n",
      "--- gene=1502, id=99 removed by soft count = 641\n",
      "--- gene=1502, id=100 removed by soft count = 254\n",
      "--- gene=1502, id=101 removed by soft count = 58\n",
      "--- gene=1502, id=105 removed by soft count = 106\n",
      "--- gene=1502, id=106 removed by soft count = 119\n",
      "--- gene=1502, id=107 removed by soft count = 83\n",
      "--- gene=1502, id=108 removed by soft count = 98\n",
      "--- gene=1502, id=109 removed by soft count = 59\n",
      "--- gene=1502, id=112 removed by soft count = 81\n",
      "--- gene=1502, id=113 removed by soft count = 153\n",
      "--- gene=1502, id=114 removed by soft count = 200\n",
      "--- gene=1502, id=115 removed by soft count = 284\n",
      "--- gene=1502, id=116 removed by soft count = 122\n",
      "--- gene=1502, id=120 removed by soft count = 60\n",
      "--- gene=1502, id=122 removed by soft count = 118\n",
      "--- gene=1502, id=128 removed by soft count = 108\n",
      "--- gene=1502, id=130 removed by soft count = 59\n",
      "--- gene=1502, id=131 removed by soft count = 60\n",
      "--- gene=1502, id=132 removed by soft count = 68\n",
      "--- gene=1502, id=136 removed by soft count = 134\n",
      "--- gene=1502, id=138 removed by soft count = 268\n",
      "--- number of probes: 139 , kept by blast: 111 , if remove dups: 110\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 110\n",
      "-- checking probes in region: 1503\n",
      "--- gene=1503, id=121 removed by soft count = 88\n",
      "--- gene=1503, id=122 removed by soft count = 61\n",
      "--- gene=1503, id=146 removed by soft count = 106\n",
      "--- gene=1503, id=147 removed by soft count = 313\n",
      "--- gene=1503, id=148 removed by soft count = 200\n",
      "--- gene=1503, id=150 removed by soft count = 81\n",
      "--- gene=1503, id=151 removed by soft count = 263\n",
      "--- gene=1503, id=169 removed by soft count = 181\n",
      "--- gene=1503, id=171 removed by soft count = 103\n",
      "--- gene=1503, id=177 removed by soft count = 57\n",
      "--- number of probes: 186 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1504\n",
      "--- gene=1504, id=5 removed by soft count = 218\n",
      "--- gene=1504, id=11 removed by soft count = 266\n",
      "--- gene=1504, id=13 removed by soft count = 139\n",
      "--- gene=1504, id=14 removed by soft count = 190\n",
      "--- gene=1504, id=15 removed by soft count = 158\n",
      "--- number of probes: 285 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1505\n",
      "--- gene=1505, id=0 removed by soft count = 272\n",
      "--- gene=1505, id=1 removed by soft count = 510\n",
      "--- gene=1505, id=2 removed by soft count = 662\n",
      "--- gene=1505, id=3 removed by soft count = 462\n",
      "--- gene=1505, id=156 removed by soft count = 61\n",
      "--- gene=1505, id=229 removed by soft count = 143\n",
      "--- gene=1505, id=230 removed by soft count = 149\n",
      "--- gene=1505, id=231 removed by soft count = 402\n",
      "--- gene=1505, id=232 removed by soft count = 401\n",
      "--- gene=1505, id=233 removed by soft count = 239\n",
      "--- gene=1505, id=234 removed by soft count = 396\n",
      "--- gene=1505, id=235 removed by soft count = 180\n",
      "--- gene=1505, id=236 removed by soft count = 227\n",
      "--- gene=1505, id=237 removed by soft count = 132\n",
      "--- gene=1505, id=238 removed by soft count = 67\n",
      "--- number of probes: 239 , kept by blast: 224 , if remove dups: 224\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1506\n",
      "--- gene=1506, id=0 removed by soft count = 77\n",
      "--- gene=1506, id=78 removed by soft count = 57\n",
      "--- gene=1506, id=80 removed by soft count = 352\n",
      "--- gene=1506, id=81 removed by soft count = 177\n",
      "--- gene=1506, id=82 removed by soft count = 62\n",
      "--- gene=1506, id=83 removed by soft count = 77\n",
      "--- gene=1506, id=84 removed by soft count = 80\n",
      "--- gene=1506, id=85 removed by soft count = 54\n",
      "--- gene=1506, id=86 removed by soft count = 63\n",
      "--- gene=1506, id=87 removed by soft count = 75\n",
      "--- gene=1506, id=88 removed by soft count = 81\n",
      "--- gene=1506, id=89 removed by soft count = 96\n",
      "--- gene=1506, id=100 removed by soft count = 5297\n",
      "--- gene=1506, id=152 removed by soft count = 78\n",
      "--- gene=1506, id=154 removed by soft count = 302\n",
      "--- number of probes: 162 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1507\n",
      "--- gene=1507, id=0 removed by soft count = 125\n",
      "--- gene=1507, id=1 removed by soft count = 61\n",
      "--- gene=1507, id=18 removed by soft count = 88\n",
      "--- gene=1507, id=24 removed by soft count = 83\n",
      "--- gene=1507, id=153 removed by soft count = 140\n",
      "--- number of probes: 242 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1508\n",
      "--- gene=1508, id=0 removed by soft count = 261\n",
      "--- gene=1508, id=1 removed by soft count = 65\n",
      "--- gene=1508, id=39 removed by soft count = 199\n",
      "--- gene=1508, id=89 removed by soft count = 354\n",
      "--- gene=1508, id=90 removed by soft count = 219\n",
      "--- gene=1508, id=91 removed by soft count = 133\n",
      "--- gene=1508, id=93 removed by soft count = 1037\n",
      "--- gene=1508, id=99 removed by soft count = 189\n",
      "--- gene=1508, id=102 removed by soft count = 176\n",
      "--- gene=1508, id=103 removed by soft count = 181\n",
      "--- gene=1508, id=105 removed by soft count = 150\n",
      "--- gene=1508, id=106 removed by soft count = 134\n",
      "--- gene=1508, id=188 removed by soft count = 68\n",
      "--- number of probes: 189 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1509\n",
      "--- gene=1509, id=49 removed by soft count = 84\n",
      "--- gene=1509, id=164 removed by soft count = 401\n",
      "--- gene=1509, id=165 removed by soft count = 1161\n",
      "--- gene=1509, id=166 removed by soft count = 89\n",
      "--- gene=1509, id=167 removed by soft count = 159\n",
      "--- gene=1509, id=181 removed by soft count = 51\n",
      "--- gene=1509, id=182 removed by soft count = 95\n",
      "--- number of probes: 188 , kept by blast: 181 , if remove dups: 181\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1510\n",
      "--- gene=1510, id=22 removed by soft count = 53\n",
      "--- gene=1510, id=67 removed by soft count = 306\n",
      "--- gene=1510, id=109 removed by soft count = 68\n",
      "--- gene=1510, id=112 removed by soft count = 202\n",
      "--- gene=1510, id=113 removed by soft count = 8952\n",
      "--- gene=1510, id=114 removed by soft count = 1347\n",
      "--- gene=1510, id=116 removed by soft count = 57\n",
      "--- gene=1510, id=119 removed by soft count = 122\n",
      "--- gene=1510, id=135 removed by soft count = 72\n",
      "--- gene=1510, id=136 removed by soft count = 51\n",
      "--- gene=1510, id=167 removed by soft count = 414\n",
      "--- gene=1510, id=169 removed by soft count = 393\n",
      "--- gene=1510, id=170 removed by soft count = 69\n",
      "--- gene=1510, id=171 removed by soft count = 124\n",
      "--- gene=1510, id=172 removed by soft count = 112\n",
      "--- gene=1510, id=173 removed by soft count = 785\n",
      "--- gene=1510, id=174 removed by soft count = 89\n",
      "--- gene=1510, id=175 removed by soft count = 95\n",
      "--- number of probes: 176 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1511\n",
      "--- gene=1511, id=62 removed by soft count = 60\n",
      "--- gene=1511, id=63 removed by soft count = 109\n",
      "--- gene=1511, id=64 removed by soft count = 129\n",
      "--- gene=1511, id=112 removed by soft count = 121\n",
      "--- gene=1511, id=115 removed by soft count = 202\n",
      "--- gene=1511, id=157 removed by soft count = 302\n",
      "--- gene=1511, id=177 removed by soft count = 59\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 218 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1512\n",
      "--- number of probes: 248 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1513\n",
      "--- gene=1513, id=0 removed by soft count = 285\n",
      "--- gene=1513, id=2 removed by soft count = 218\n",
      "--- gene=1513, id=3 removed by soft count = 56\n",
      "--- gene=1513, id=209 removed by soft count = 52\n",
      "--- gene=1513, id=232 removed by soft count = 56\n",
      "--- gene=1513, id=236 removed by soft count = 98\n",
      "--- number of probes: 245 , kept by blast: 239 , if remove dups: 239\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1514\n",
      "--- gene=1514, id=11 removed by soft count = 65\n",
      "--- number of probes: 262 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1515\n",
      "--- gene=1515, id=96 removed by soft count = 117\n",
      "--- gene=1515, id=162 removed by soft count = 67\n",
      "--- number of probes: 357 , kept by blast: 355 , if remove dups: 355\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1516\n",
      "--- gene=1516, id=3 removed by soft count = 78\n",
      "--- gene=1516, id=4 removed by soft count = 59\n",
      "--- gene=1516, id=6 removed by soft count = 300\n",
      "--- number of probes: 103 , kept by blast: 100 , if remove dups: 100\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: 1517\n",
      "--- gene=1517, id=31 removed by soft count = 81\n",
      "--- gene=1517, id=35 removed by soft count = 153\n",
      "--- gene=1517, id=50 removed by soft count = 107\n",
      "--- gene=1517, id=189 removed by soft count = 57\n",
      "--- gene=1517, id=194 removed by soft count = 235\n",
      "--- number of probes: 232 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1518\n",
      "--- gene=1518, id=67 removed by soft count = 963\n",
      "--- gene=1518, id=68 removed by soft count = 118\n",
      "--- gene=1518, id=108 removed by soft count = 84\n",
      "--- gene=1518, id=109 removed by soft count = 61\n",
      "--- gene=1518, id=111 removed by soft count = 315\n",
      "--- gene=1518, id=113 removed by soft count = 58\n",
      "--- gene=1518, id=114 removed by soft count = 85\n",
      "--- gene=1518, id=115 removed by soft count = 158\n",
      "--- gene=1518, id=116 removed by soft count = 65\n",
      "--- gene=1518, id=120 removed by soft count = 76\n",
      "--- gene=1518, id=122 removed by soft count = 296\n",
      "--- gene=1518, id=123 removed by soft count = 59\n",
      "--- gene=1518, id=124 removed by soft count = 239\n",
      "--- gene=1518, id=126 removed by soft count = 189\n",
      "--- gene=1518, id=129 removed by soft count = 141\n",
      "--- gene=1518, id=130 removed by soft count = 133\n",
      "--- gene=1518, id=131 removed by soft count = 80\n",
      "--- gene=1518, id=133 removed by soft count = 228\n",
      "--- gene=1518, id=135 removed by soft count = 59\n",
      "--- gene=1518, id=138 removed by soft count = 685\n",
      "--- gene=1518, id=140 removed by soft count = 342\n",
      "--- gene=1518, id=145 removed by soft count = 846\n",
      "--- gene=1518, id=146 removed by soft count = 150\n",
      "--- gene=1518, id=147 removed by soft count = 470\n",
      "--- gene=1518, id=148 removed by soft count = 66\n",
      "--- gene=1518, id=150 removed by soft count = 57\n",
      "--- gene=1518, id=151 removed by soft count = 779\n",
      "--- gene=1518, id=153 removed by soft count = 1064\n",
      "--- number of probes: 217 , kept by blast: 189 , if remove dups: 189\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1519\n",
      "--- gene=1519, id=6 removed by soft count = 336\n",
      "--- gene=1519, id=8 removed by soft count = 324\n",
      "--- gene=1519, id=9 removed by soft count = 78\n",
      "--- gene=1519, id=10 removed by soft count = 127\n",
      "--- number of probes: 239 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 152\n",
      "--- gene=152, id=60 removed by soft count = 248\n",
      "--- gene=152, id=177 removed by soft count = 62\n",
      "--- gene=152, id=196 removed by soft count = 103\n",
      "--- number of probes: 248 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1520\n",
      "--- gene=1520, id=77 removed by soft count = 461\n",
      "--- gene=1520, id=163 removed by soft count = 108\n",
      "--- gene=1520, id=164 removed by soft count = 656\n",
      "--- gene=1520, id=165 removed by soft count = 251\n",
      "--- gene=1520, id=166 removed by soft count = 498\n",
      "--- gene=1520, id=167 removed by soft count = 185\n",
      "--- number of probes: 221 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1521\n",
      "--- gene=1521, id=6 removed by soft count = 83\n",
      "--- gene=1521, id=48 removed by soft count = 68\n",
      "--- gene=1521, id=53 removed by soft count = 72\n",
      "--- gene=1521, id=116 removed by soft count = 291\n",
      "--- gene=1521, id=117 removed by soft count = 263\n",
      "--- gene=1521, id=122 removed by soft count = 192\n",
      "--- gene=1521, id=126 removed by soft count = 71\n",
      "--- number of probes: 176 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1525\n",
      "--- gene=1525, id=9 removed by soft count = 53\n",
      "--- gene=1525, id=12 removed by soft count = 412\n",
      "--- gene=1525, id=13 removed by soft count = 60\n",
      "--- gene=1525, id=45 removed by soft count = 149\n",
      "--- number of probes: 51 , kept by blast: 47 , if remove dups: 45\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: 1526\n",
      "--- gene=1526, id=31 removed by soft count = 80\n",
      "--- gene=1526, id=32 removed by soft count = 573\n",
      "--- gene=1526, id=45 removed by soft count = 56\n",
      "--- gene=1526, id=52 removed by soft count = 1147\n",
      "--- gene=1526, id=56 removed by soft count = 78\n",
      "--- number of probes: 60 , kept by blast: 55 , if remove dups: 54\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: 1527\n",
      "--- gene=1527, id=6 removed by soft count = 135\n",
      "--- gene=1527, id=11 removed by soft count = 102\n",
      "--- gene=1527, id=12 removed by soft count = 131\n",
      "--- gene=1527, id=32 removed by soft count = 460\n",
      "--- number of probes: 72 , kept by blast: 68 , if remove dups: 63\n",
      "-- number of probes kept for this region: 68\n",
      "-- checking probes in region: 1529\n",
      "--- gene=1529, id=13 removed by soft count = 105\n",
      "--- gene=1529, id=14 removed by soft count = 185\n",
      "--- gene=1529, id=40 removed by soft count = 727\n",
      "--- gene=1529, id=41 removed by soft count = 67\n",
      "--- gene=1529, id=89 removed by soft count = 5673\n",
      "--- gene=1529, id=90 removed by soft count = 409\n",
      "--- gene=1529, id=97 removed by soft count = 398\n",
      "--- gene=1529, id=101 removed by soft count = 52\n",
      "--- gene=1529, id=103 removed by soft count = 83\n",
      "--- gene=1529, id=104 removed by soft count = 56\n",
      "--- gene=1529, id=107 removed by soft count = 1831\n",
      "--- gene=1529, id=108 removed by soft count = 204\n",
      "--- gene=1529, id=110 removed by soft count = 90\n",
      "--- gene=1529, id=111 removed by soft count = 214\n",
      "--- gene=1529, id=112 removed by soft count = 422\n",
      "--- gene=1529, id=113 removed by soft count = 85\n",
      "--- gene=1529, id=114 removed by soft count = 51\n",
      "--- gene=1529, id=115 removed by soft count = 58\n",
      "--- gene=1529, id=116 removed by soft count = 159\n",
      "--- gene=1529, id=118 removed by soft count = 251\n",
      "--- gene=1529, id=120 removed by soft count = 781\n",
      "--- gene=1529, id=121 removed by soft count = 159\n",
      "--- gene=1529, id=122 removed by soft count = 201\n",
      "--- gene=1529, id=123 removed by soft count = 941\n",
      "--- gene=1529, id=124 removed by soft count = 247\n",
      "--- gene=1529, id=125 removed by soft count = 77\n",
      "--- gene=1529, id=126 removed by soft count = 154\n",
      "--- gene=1529, id=127 removed by soft count = 607\n",
      "--- gene=1529, id=128 removed by soft count = 238\n",
      "--- gene=1529, id=129 removed by soft count = 91\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1529, id=130 removed by soft count = 165\n",
      "--- gene=1529, id=151 removed by soft count = 12629\n",
      "--- gene=1529, id=152 removed by soft count = 112\n",
      "--- gene=1529, id=154 removed by soft count = 744\n",
      "--- number of probes: 166 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 153\n",
      "--- number of probes: 307 , kept by blast: 307 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1530\n",
      "--- gene=1530, id=1 removed by soft count = 487\n",
      "--- gene=1530, id=14 removed by soft count = 67\n",
      "--- gene=1530, id=36 removed by soft count = 57\n",
      "--- gene=1530, id=57 removed by soft count = 55\n",
      "--- gene=1530, id=62 removed by soft count = 70\n",
      "--- gene=1530, id=100 removed by soft count = 70\n",
      "--- gene=1530, id=103 removed by soft count = 74\n",
      "--- gene=1530, id=134 removed by soft count = 97\n",
      "--- gene=1530, id=152 removed by soft count = 15792\n",
      "--- gene=1530, id=187 removed by soft count = 51\n",
      "--- gene=1530, id=188 removed by soft count = 443\n",
      "--- number of probes: 218 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1533\n",
      "--- gene=1533, id=0 removed by soft count = 52\n",
      "--- gene=1533, id=1 removed by soft count = 2704\n",
      "--- gene=1533, id=2 removed by soft count = 655\n",
      "--- gene=1533, id=3 removed by soft count = 191\n",
      "--- gene=1533, id=4 removed by soft count = 658\n",
      "--- gene=1533, id=5 removed by soft count = 135\n",
      "--- gene=1533, id=8 removed by soft count = 498\n",
      "--- gene=1533, id=9 removed by soft count = 121\n",
      "--- gene=1533, id=10 removed by soft count = 722\n",
      "--- gene=1533, id=12 removed by soft count = 330\n",
      "--- gene=1533, id=16 removed by soft count = 83\n",
      "--- gene=1533, id=17 removed by soft count = 512\n",
      "--- gene=1533, id=18 removed by soft count = 82\n",
      "--- gene=1533, id=21 removed by soft count = 240\n",
      "--- gene=1533, id=23 removed by soft count = 51\n",
      "--- gene=1533, id=24 removed by soft count = 142\n",
      "--- gene=1533, id=26 removed by soft count = 161\n",
      "--- gene=1533, id=30 removed by soft count = 219\n",
      "--- gene=1533, id=31 removed by soft count = 120\n",
      "--- gene=1533, id=33 removed by soft count = 175\n",
      "--- gene=1533, id=35 removed by soft count = 86\n",
      "--- gene=1533, id=38 removed by soft count = 96\n",
      "--- gene=1533, id=39 removed by soft count = 376\n",
      "--- gene=1533, id=41 removed by soft count = 396\n",
      "--- gene=1533, id=43 removed by soft count = 2544\n",
      "--- gene=1533, id=45 removed by soft count = 55\n",
      "--- gene=1533, id=50 removed by soft count = 312\n",
      "--- gene=1533, id=51 removed by soft count = 72\n",
      "--- gene=1533, id=54 removed by soft count = 168\n",
      "--- gene=1533, id=55 removed by soft count = 77\n",
      "--- gene=1533, id=56 removed by soft count = 148\n",
      "--- gene=1533, id=57 removed by soft count = 1146\n",
      "--- gene=1533, id=59 removed by soft count = 266\n",
      "--- gene=1533, id=60 removed by soft count = 165\n",
      "--- gene=1533, id=61 removed by soft count = 188\n",
      "--- gene=1533, id=64 removed by soft count = 176\n",
      "--- gene=1533, id=66 removed by soft count = 528\n",
      "--- gene=1533, id=67 removed by soft count = 322\n",
      "--- gene=1533, id=68 removed by soft count = 157\n",
      "--- number of probes: 69 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: 1534\n",
      "--- gene=1534, id=55 removed by soft count = 106\n",
      "--- gene=1534, id=57 removed by soft count = 59\n",
      "--- gene=1534, id=60 removed by soft count = 91\n",
      "--- gene=1534, id=70 removed by soft count = 327\n",
      "--- gene=1534, id=71 removed by soft count = 1673\n",
      "--- gene=1534, id=76 removed by soft count = 62\n",
      "--- gene=1534, id=94 removed by soft count = 151\n",
      "--- gene=1534, id=115 removed by hard count = 3\n",
      "--- gene=1534, id=126 removed by soft count = 12176\n",
      "--- gene=1534, id=132 removed by soft count = 51\n",
      "--- gene=1534, id=142 removed by soft count = 77\n",
      "--- gene=1534, id=144 removed by soft count = 654\n",
      "--- number of probes: 149 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1535\n",
      "--- gene=1535, id=1 removed by soft count = 189\n",
      "--- gene=1535, id=2 removed by soft count = 6620\n",
      "--- gene=1535, id=3 removed by soft count = 9875\n",
      "--- gene=1535, id=4 removed by soft count = 415\n",
      "--- gene=1535, id=5 removed by soft count = 2112\n",
      "--- gene=1535, id=9 removed by soft count = 52\n",
      "--- gene=1535, id=25 removed by soft count = 52\n",
      "--- gene=1535, id=52 removed by soft count = 61\n",
      "--- gene=1535, id=56 removed by soft count = 464\n",
      "--- gene=1535, id=57 removed by soft count = 1195\n",
      "--- gene=1535, id=58 removed by soft count = 455\n",
      "--- gene=1535, id=69 removed by soft count = 136\n",
      "--- gene=1535, id=76 removed by soft count = 164\n",
      "--- gene=1535, id=103 removed by hard count = 0\n",
      "--- gene=1535, id=127 removed by soft count = 62\n",
      "--- gene=1535, id=135 removed by soft count = 57\n",
      "--- gene=1535, id=137 removed by soft count = 242\n",
      "--- gene=1535, id=138 removed by soft count = 112\n",
      "--- gene=1535, id=141 removed by soft count = 154\n",
      "--- gene=1535, id=143 removed by soft count = 1492\n",
      "--- gene=1535, id=144 removed by soft count = 343\n",
      "--- gene=1535, id=145 removed by soft count = 1519\n",
      "--- number of probes: 187 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1536\n",
      "--- gene=1536, id=15 removed by soft count = 73\n",
      "--- gene=1536, id=45 removed by soft count = 6555\n",
      "--- gene=1536, id=48 removed by soft count = 497\n",
      "--- gene=1536, id=64 removed by soft count = 52\n",
      "--- gene=1536, id=71 removed by soft count = 74\n",
      "--- gene=1536, id=72 removed by soft count = 1531\n",
      "--- gene=1536, id=73 removed by soft count = 77\n",
      "--- gene=1536, id=104 removed by soft count = 68\n",
      "--- gene=1536, id=118 removed by soft count = 213\n",
      "--- gene=1536, id=128 removed by soft count = 258\n",
      "--- gene=1536, id=132 removed by soft count = 150\n",
      "--- gene=1536, id=154 removed by soft count = 104\n",
      "--- gene=1536, id=159 removed by soft count = 853\n",
      "--- gene=1536, id=160 removed by soft count = 184\n",
      "--- gene=1536, id=161 removed by soft count = 211\n",
      "--- gene=1536, id=162 removed by soft count = 265\n",
      "--- gene=1536, id=163 removed by soft count = 151\n",
      "--- gene=1536, id=164 removed by soft count = 187\n",
      "--- gene=1536, id=167 removed by soft count = 283\n",
      "--- gene=1536, id=168 removed by soft count = 132\n",
      "--- gene=1536, id=169 removed by soft count = 70\n",
      "--- gene=1536, id=170 removed by soft count = 203\n",
      "--- gene=1536, id=172 removed by soft count = 287\n",
      "--- gene=1536, id=173 removed by soft count = 133\n",
      "--- gene=1536, id=174 removed by soft count = 883\n",
      "--- number of probes: 187 , kept by blast: 162 , if remove dups: 162\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1537\n",
      "--- gene=1537, id=25 removed by soft count = 64\n",
      "--- gene=1537, id=47 removed by soft count = 87\n",
      "--- gene=1537, id=112 removed by soft count = 1086\n",
      "--- gene=1537, id=115 removed by soft count = 58\n",
      "--- gene=1537, id=118 removed by soft count = 66\n",
      "--- gene=1537, id=119 removed by soft count = 58\n",
      "--- gene=1537, id=136 removed by soft count = 2390\n",
      "--- gene=1537, id=137 removed by soft count = 137\n",
      "--- gene=1537, id=143 removed by soft count = 837\n",
      "--- gene=1537, id=144 removed by soft count = 757\n",
      "--- gene=1537, id=145 removed by soft count = 81\n",
      "--- gene=1537, id=146 removed by soft count = 73\n",
      "--- gene=1537, id=147 removed by soft count = 160\n",
      "--- gene=1537, id=148 removed by soft count = 653\n",
      "--- gene=1537, id=151 removed by soft count = 69\n",
      "--- gene=1537, id=164 removed by soft count = 725\n",
      "--- gene=1537, id=165 removed by soft count = 748\n",
      "--- gene=1537, id=166 removed by soft count = 798\n",
      "--- gene=1537, id=167 removed by soft count = 596\n",
      "--- gene=1537, id=168 removed by soft count = 233\n",
      "--- gene=1537, id=169 removed by soft count = 935\n",
      "--- number of probes: 170 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1538\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1538, id=30 removed by soft count = 785\n",
      "--- gene=1538, id=31 removed by soft count = 474\n",
      "--- gene=1538, id=33 removed by soft count = 11068\n",
      "--- gene=1538, id=42 removed by soft count = 312\n",
      "--- gene=1538, id=43 removed by soft count = 505\n",
      "--- gene=1538, id=52 removed by soft count = 86\n",
      "--- gene=1538, id=53 removed by soft count = 61\n",
      "--- gene=1538, id=56 removed by soft count = 231\n",
      "--- gene=1538, id=59 removed by soft count = 373\n",
      "--- gene=1538, id=92 removed by soft count = 2449\n",
      "--- gene=1538, id=93 removed by soft count = 260\n",
      "--- gene=1538, id=96 removed by soft count = 176\n",
      "--- gene=1538, id=138 removed by soft count = 2964\n",
      "--- gene=1538, id=159 removed by soft count = 1681\n",
      "--- gene=1538, id=161 removed by soft count = 7822\n",
      "--- gene=1538, id=163 removed by soft count = 2989\n",
      "--- number of probes: 164 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 154\n",
      "--- gene=154, id=3 removed by soft count = 188\n",
      "--- gene=154, id=160 removed by soft count = 94\n",
      "--- gene=154, id=162 removed by soft count = 99\n",
      "--- gene=154, id=167 removed by soft count = 83\n",
      "--- gene=154, id=180 removed by soft count = 101\n",
      "--- number of probes: 283 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1541\n",
      "--- gene=1541, id=9 removed by soft count = 1194\n",
      "--- gene=1541, id=41 removed by soft count = 366\n",
      "--- gene=1541, id=42 removed by soft count = 210\n",
      "--- gene=1541, id=54 removed by soft count = 2573\n",
      "--- gene=1541, id=63 removed by soft count = 115\n",
      "--- gene=1541, id=209 removed by soft count = 6814\n",
      "--- gene=1541, id=210 removed by soft count = 2005\n",
      "--- gene=1541, id=211 removed by soft count = 825\n",
      "--- gene=1541, id=212 removed by soft count = 1125\n",
      "--- gene=1541, id=215 removed by soft count = 1273\n",
      "--- gene=1541, id=217 removed by soft count = 76\n",
      "--- gene=1541, id=219 removed by soft count = 230\n",
      "--- gene=1541, id=247 removed by soft count = 594\n",
      "--- gene=1541, id=250 removed by soft count = 190\n",
      "--- number of probes: 251 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1542\n",
      "--- gene=1542, id=21 removed by soft count = 804\n",
      "--- gene=1542, id=28 removed by soft count = 8954\n",
      "--- gene=1542, id=37 removed by soft count = 137\n",
      "--- gene=1542, id=39 removed by soft count = 80\n",
      "--- gene=1542, id=51 removed by soft count = 81\n",
      "--- gene=1542, id=52 removed by soft count = 401\n",
      "--- gene=1542, id=70 removed by soft count = 84\n",
      "--- gene=1542, id=72 removed by soft count = 570\n",
      "--- gene=1542, id=77 removed by soft count = 68\n",
      "--- gene=1542, id=81 removed by soft count = 68\n",
      "--- gene=1542, id=83 removed by soft count = 151\n",
      "--- gene=1542, id=84 removed by soft count = 419\n",
      "--- gene=1542, id=85 removed by soft count = 156\n",
      "--- gene=1542, id=86 removed by soft count = 266\n",
      "--- gene=1542, id=90 removed by soft count = 53\n",
      "--- gene=1542, id=91 removed by soft count = 116\n",
      "--- gene=1542, id=104 removed by soft count = 59\n",
      "--- gene=1542, id=108 removed by soft count = 61\n",
      "--- gene=1542, id=188 removed by soft count = 64\n",
      "--- number of probes: 204 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1548\n",
      "--- gene=1548, id=11 removed by soft count = 60\n",
      "--- gene=1548, id=40 removed by hard count = 4\n",
      "--- number of probes: 106 , kept by blast: 104 , if remove dups: 102\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 102\n",
      "-- checking probes in region: 155\n",
      "--- gene=155, id=133 removed by soft count = 60\n",
      "--- gene=155, id=134 removed by soft count = 61\n",
      "--- gene=155, id=136 removed by soft count = 74\n",
      "--- number of probes: 252 , kept by blast: 249 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 156\n",
      "--- gene=156, id=125 removed by soft count = 368\n",
      "--- gene=156, id=231 removed by soft count = 5316\n",
      "--- number of probes: 292 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 157\n",
      "--- number of probes: 255 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 158\n",
      "--- gene=158, id=105 removed by soft count = 72\n",
      "--- number of probes: 227 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 159\n",
      "--- gene=159, id=3 removed by soft count = 132\n",
      "--- gene=159, id=57 removed by soft count = 92\n",
      "--- gene=159, id=122 removed by soft count = 97\n",
      "--- number of probes: 231 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 16\n",
      "--- gene=16, id=154 removed by soft count = 4544\n",
      "--- gene=16, id=161 removed by soft count = 331\n",
      "--- gene=16, id=162 removed by soft count = 889\n",
      "--- gene=16, id=163 removed by soft count = 87\n",
      "--- gene=16, id=166 removed by soft count = 55\n",
      "--- gene=16, id=167 removed by soft count = 101\n",
      "--- gene=16, id=169 removed by soft count = 90\n",
      "--- gene=16, id=170 removed by soft count = 141\n",
      "--- gene=16, id=172 removed by soft count = 64\n",
      "--- gene=16, id=173 removed by soft count = 57\n",
      "--- gene=16, id=174 removed by soft count = 52\n",
      "--- gene=16, id=176 removed by soft count = 197\n",
      "--- gene=16, id=179 removed by soft count = 63\n",
      "--- gene=16, id=181 removed by soft count = 114\n",
      "--- gene=16, id=183 removed by soft count = 76\n",
      "--- number of probes: 245 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 160\n",
      "--- gene=160, id=68 removed by soft count = 112\n",
      "--- gene=160, id=69 removed by soft count = 149\n",
      "--- gene=160, id=73 removed by soft count = 78\n",
      "--- gene=160, id=75 removed by soft count = 398\n",
      "--- gene=160, id=77 removed by soft count = 123\n",
      "--- gene=160, id=86 removed by soft count = 325\n",
      "--- gene=160, id=87 removed by soft count = 100\n",
      "--- gene=160, id=90 removed by soft count = 163\n",
      "--- gene=160, id=92 removed by soft count = 137\n",
      "--- gene=160, id=106 removed by soft count = 54\n",
      "--- gene=160, id=134 removed by soft count = 175\n",
      "--- gene=160, id=135 removed by soft count = 473\n",
      "--- number of probes: 136 , kept by blast: 124 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 161\n",
      "--- gene=161, id=122 removed by soft count = 56\n",
      "--- number of probes: 134 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 162\n",
      "--- gene=162, id=53 removed by soft count = 154\n",
      "--- gene=162, id=79 removed by soft count = 201\n",
      "--- gene=162, id=80 removed by soft count = 60\n",
      "--- gene=162, id=82 removed by soft count = 57\n",
      "--- gene=162, id=247 removed by soft count = 79\n",
      "--- gene=162, id=265 removed by soft count = 224\n",
      "--- gene=162, id=268 removed by soft count = 469\n",
      "--- gene=162, id=337 removed by soft count = 63\n",
      "--- number of probes: 338 , kept by blast: 330 , if remove dups: 330\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 163\n",
      "--- gene=163, id=215 removed by soft count = 132\n",
      "--- gene=163, id=220 removed by soft count = 165\n",
      "--- gene=163, id=230 removed by soft count = 172\n",
      "--- number of probes: 234 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 164\n",
      "--- gene=164, id=57 removed by soft count = 727\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=164, id=133 removed by soft count = 181\n",
      "--- number of probes: 303 , kept by blast: 301 , if remove dups: 301\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 165\n",
      "--- gene=165, id=230 removed by soft count = 373\n",
      "--- number of probes: 244 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 166\n",
      "--- gene=166, id=10 removed by soft count = 275\n",
      "--- gene=166, id=14 removed by soft count = 8707\n",
      "--- gene=166, id=271 removed by soft count = 1650\n",
      "--- number of probes: 303 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 167\n",
      "--- gene=167, id=1 removed by soft count = 64\n",
      "--- gene=167, id=3 removed by soft count = 66\n",
      "--- gene=167, id=4 removed by soft count = 71\n",
      "--- gene=167, id=39 removed by soft count = 138\n",
      "--- gene=167, id=65 removed by soft count = 94\n",
      "--- gene=167, id=79 removed by soft count = 52\n",
      "--- gene=167, id=81 removed by soft count = 64\n",
      "--- gene=167, id=83 removed by soft count = 137\n",
      "--- gene=167, id=91 removed by soft count = 109\n",
      "--- gene=167, id=93 removed by soft count = 526\n",
      "--- gene=167, id=184 removed by soft count = 168\n",
      "--- gene=167, id=186 removed by soft count = 93\n",
      "--- number of probes: 217 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 168\n",
      "--- number of probes: 301 , kept by blast: 301 , if remove dups: 301\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 169\n",
      "--- gene=169, id=15 removed by soft count = 60\n",
      "--- gene=169, id=50 removed by soft count = 54\n",
      "--- gene=169, id=87 removed by soft count = 174\n",
      "--- gene=169, id=128 removed by soft count = 300\n",
      "--- gene=169, id=129 removed by soft count = 428\n",
      "--- number of probes: 238 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 17\n",
      "--- gene=17, id=2 removed by soft count = 143\n",
      "--- gene=17, id=82 removed by soft count = 130\n",
      "--- gene=17, id=86 removed by soft count = 104\n",
      "--- gene=17, id=89 removed by soft count = 62\n",
      "--- gene=17, id=99 removed by soft count = 172\n",
      "--- number of probes: 203 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 170\n",
      "--- number of probes: 302 , kept by blast: 302 , if remove dups: 302\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 171\n",
      "--- number of probes: 272 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 172\n",
      "--- gene=172, id=134 removed by soft count = 516\n",
      "--- gene=172, id=135 removed by soft count = 120\n",
      "--- number of probes: 200 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 173\n",
      "--- gene=173, id=0 removed by soft count = 61\n",
      "--- gene=173, id=1 removed by soft count = 292\n",
      "--- gene=173, id=2 removed by soft count = 528\n",
      "--- gene=173, id=3 removed by soft count = 177\n",
      "--- gene=173, id=6 removed by soft count = 1383\n",
      "--- gene=173, id=52 removed by soft count = 216\n",
      "--- gene=173, id=112 removed by soft count = 218\n",
      "--- gene=173, id=159 removed by soft count = 142\n",
      "--- gene=173, id=162 removed by soft count = 163\n",
      "--- gene=173, id=164 removed by soft count = 127\n",
      "--- gene=173, id=166 removed by soft count = 123\n",
      "--- gene=173, id=167 removed by soft count = 118\n",
      "--- gene=173, id=172 removed by soft count = 153\n",
      "--- number of probes: 173 , kept by blast: 160 , if remove dups: 160\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 174\n",
      "--- gene=174, id=136 removed by soft count = 5795\n",
      "--- gene=174, id=138 removed by soft count = 1335\n",
      "--- number of probes: 198 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 175\n",
      "--- gene=175, id=140 removed by soft count = 52\n",
      "--- gene=175, id=242 removed by soft count = 121\n",
      "--- gene=175, id=244 removed by soft count = 190\n",
      "--- gene=175, id=246 removed by soft count = 222\n",
      "--- number of probes: 317 , kept by blast: 313 , if remove dups: 313\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 176\n",
      "--- gene=176, id=17 removed by soft count = 109\n",
      "--- gene=176, id=18 removed by soft count = 269\n",
      "--- gene=176, id=20 removed by soft count = 153\n",
      "--- gene=176, id=23 removed by soft count = 239\n",
      "--- gene=176, id=24 removed by soft count = 110\n",
      "--- gene=176, id=25 removed by soft count = 276\n",
      "--- gene=176, id=26 removed by soft count = 369\n",
      "--- gene=176, id=28 removed by soft count = 53\n",
      "--- gene=176, id=29 removed by soft count = 100\n",
      "--- gene=176, id=39 removed by soft count = 208\n",
      "--- gene=176, id=41 removed by soft count = 241\n",
      "--- gene=176, id=42 removed by soft count = 480\n",
      "--- gene=176, id=44 removed by soft count = 103\n",
      "--- gene=176, id=45 removed by soft count = 4188\n",
      "--- gene=176, id=91 removed by soft count = 462\n",
      "--- gene=176, id=100 removed by soft count = 130\n",
      "--- number of probes: 152 , kept by blast: 136 , if remove dups: 136\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 177\n",
      "--- gene=177, id=8 removed by soft count = 211\n",
      "--- gene=177, id=9 removed by soft count = 81\n",
      "--- number of probes: 279 , kept by blast: 277 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 178\n",
      "--- gene=178, id=15 removed by soft count = 68\n",
      "--- number of probes: 280 , kept by blast: 279 , if remove dups: 279\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 179\n",
      "--- gene=179, id=7 removed by soft count = 79\n",
      "--- gene=179, id=12 removed by soft count = 208\n",
      "--- gene=179, id=39 removed by soft count = 53\n",
      "--- gene=179, id=43 removed by soft count = 504\n",
      "--- gene=179, id=113 removed by soft count = 59\n",
      "--- gene=179, id=121 removed by soft count = 344\n",
      "--- gene=179, id=150 removed by soft count = 2762\n",
      "--- number of probes: 230 , kept by blast: 223 , if remove dups: 223\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 18\n",
      "--- gene=18, id=4 removed by soft count = 99\n",
      "--- gene=18, id=5 removed by soft count = 411\n",
      "--- gene=18, id=22 removed by soft count = 85\n",
      "--- gene=18, id=82 removed by soft count = 323\n",
      "--- gene=18, id=94 removed by soft count = 4875\n",
      "--- gene=18, id=103 removed by soft count = 237\n",
      "--- gene=18, id=175 removed by soft count = 90\n",
      "--- gene=18, id=176 removed by soft count = 82\n",
      "--- number of probes: 204 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 180\n",
      "--- gene=180, id=107 removed by soft count = 161\n",
      "--- gene=180, id=170 removed by soft count = 263\n",
      "--- gene=180, id=171 removed by soft count = 2219\n",
      "--- gene=180, id=172 removed by soft count = 1278\n",
      "--- gene=180, id=174 removed by soft count = 105\n",
      "--- gene=180, id=175 removed by soft count = 372\n",
      "--- gene=180, id=179 removed by soft count = 304\n",
      "--- gene=180, id=180 removed by soft count = 839\n",
      "--- gene=180, id=182 removed by soft count = 186\n",
      "--- gene=180, id=222 removed by soft count = 71\n",
      "--- gene=180, id=224 removed by soft count = 53\n",
      "--- gene=180, id=225 removed by soft count = 55\n",
      "--- number of probes: 226 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 181\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=181, id=21 removed by soft count = 61\n",
      "--- gene=181, id=92 removed by soft count = 93\n",
      "--- gene=181, id=187 removed by soft count = 178\n",
      "--- gene=181, id=189 removed by soft count = 59\n",
      "--- gene=181, id=247 removed by soft count = 138\n",
      "--- number of probes: 248 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 182\n",
      "--- gene=182, id=0 removed by soft count = 141\n",
      "--- gene=182, id=2 removed by soft count = 184\n",
      "--- gene=182, id=3 removed by soft count = 70\n",
      "--- gene=182, id=4 removed by soft count = 102\n",
      "--- gene=182, id=5 removed by soft count = 253\n",
      "--- gene=182, id=7 removed by soft count = 67\n",
      "--- gene=182, id=11 removed by soft count = 54\n",
      "--- gene=182, id=12 removed by soft count = 192\n",
      "--- gene=182, id=14 removed by soft count = 128\n",
      "--- gene=182, id=15 removed by soft count = 117\n",
      "--- gene=182, id=62 removed by soft count = 513\n",
      "--- gene=182, id=65 removed by soft count = 162\n",
      "--- number of probes: 93 , kept by blast: 81 , if remove dups: 81\n",
      "-- number of probes kept for this region: 81\n",
      "-- checking probes in region: 183\n",
      "--- gene=183, id=201 removed by soft count = 753\n",
      "--- gene=183, id=202 removed by soft count = 587\n",
      "--- gene=183, id=203 removed by soft count = 1064\n",
      "--- number of probes: 204 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 184\n",
      "--- gene=184, id=63 removed by soft count = 189\n",
      "--- gene=184, id=64 removed by soft count = 382\n",
      "--- gene=184, id=149 removed by soft count = 291\n",
      "--- gene=184, id=150 removed by soft count = 111\n",
      "--- gene=184, id=154 removed by soft count = 464\n",
      "--- gene=184, id=163 removed by soft count = 295\n",
      "--- number of probes: 173 , kept by blast: 167 , if remove dups: 167\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 185\n",
      "--- number of probes: 313 , kept by blast: 313 , if remove dups: 313\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 186\n",
      "--- gene=186, id=160 removed by soft count = 73\n",
      "--- gene=186, id=289 removed by soft count = 92\n",
      "--- gene=186, id=298 removed by soft count = 71\n",
      "--- number of probes: 301 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 187\n",
      "--- gene=187, id=132 removed by soft count = 568\n",
      "--- gene=187, id=147 removed by soft count = 88\n",
      "--- gene=187, id=188 removed by soft count = 77\n",
      "--- gene=187, id=218 removed by soft count = 64\n",
      "--- number of probes: 239 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 188\n",
      "--- gene=188, id=58 removed by soft count = 203\n",
      "--- gene=188, id=60 removed by soft count = 184\n",
      "--- gene=188, id=61 removed by soft count = 460\n",
      "--- gene=188, id=92 removed by soft count = 70\n",
      "--- number of probes: 233 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 189\n",
      "--- gene=189, id=27 removed by soft count = 143\n",
      "--- gene=189, id=32 removed by soft count = 2028\n",
      "--- gene=189, id=38 removed by soft count = 546\n",
      "--- gene=189, id=39 removed by soft count = 116\n",
      "--- gene=189, id=106 removed by soft count = 311\n",
      "--- gene=189, id=107 removed by soft count = 101\n",
      "--- gene=189, id=110 removed by soft count = 1312\n",
      "--- number of probes: 191 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 19\n",
      "--- number of probes: 391 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 190\n",
      "--- gene=190, id=10 removed by soft count = 122\n",
      "--- gene=190, id=28 removed by soft count = 423\n",
      "--- gene=190, id=29 removed by soft count = 113\n",
      "--- gene=190, id=31 removed by soft count = 858\n",
      "--- gene=190, id=38 removed by soft count = 62\n",
      "--- gene=190, id=126 removed by soft count = 65\n",
      "--- gene=190, id=130 removed by soft count = 96\n",
      "--- gene=190, id=132 removed by soft count = 67\n",
      "--- gene=190, id=135 removed by soft count = 83\n",
      "--- gene=190, id=148 removed by soft count = 54\n",
      "--- number of probes: 150 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 191\n",
      "--- gene=191, id=141 removed by hard count = 5\n",
      "--- gene=191, id=146 removed by soft count = 67\n",
      "--- gene=191, id=167 removed by soft count = 180\n",
      "--- gene=191, id=207 removed by soft count = 51\n",
      "--- number of probes: 274 , kept by blast: 270 , if remove dups: 270\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 192\n",
      "--- gene=192, id=0 removed by soft count = 2798\n",
      "--- gene=192, id=1 removed by soft count = 859\n",
      "--- gene=192, id=3 removed by soft count = 1433\n",
      "--- gene=192, id=8 removed by soft count = 264\n",
      "--- gene=192, id=9 removed by soft count = 4654\n",
      "--- gene=192, id=10 removed by soft count = 359\n",
      "--- gene=192, id=11 removed by soft count = 2463\n",
      "--- gene=192, id=12 removed by soft count = 266\n",
      "--- gene=192, id=13 removed by soft count = 2188\n",
      "--- gene=192, id=110 removed by soft count = 2555\n",
      "--- gene=192, id=112 removed by soft count = 240\n",
      "--- gene=192, id=197 removed by soft count = 64\n",
      "--- number of probes: 270 , kept by blast: 258 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 193\n",
      "--- gene=193, id=1 removed by soft count = 136\n",
      "--- gene=193, id=2 removed by soft count = 87\n",
      "--- gene=193, id=237 removed by soft count = 302\n",
      "--- number of probes: 357 , kept by blast: 354 , if remove dups: 354\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 194\n",
      "--- gene=194, id=52 removed by soft count = 53\n",
      "--- gene=194, id=101 removed by soft count = 296\n",
      "--- gene=194, id=181 removed by soft count = 70\n",
      "--- gene=194, id=187 removed by soft count = 76\n",
      "--- number of probes: 297 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 195\n",
      "--- gene=195, id=7 removed by soft count = 99\n",
      "--- gene=195, id=8 removed by soft count = 67\n",
      "--- gene=195, id=9 removed by soft count = 157\n",
      "--- gene=195, id=10 removed by soft count = 152\n",
      "--- gene=195, id=11 removed by soft count = 383\n",
      "--- gene=195, id=12 removed by soft count = 271\n",
      "--- gene=195, id=13 removed by soft count = 191\n",
      "--- gene=195, id=14 removed by soft count = 125\n",
      "--- gene=195, id=17 removed by soft count = 365\n",
      "--- gene=195, id=18 removed by soft count = 752\n",
      "--- gene=195, id=20 removed by soft count = 704\n",
      "--- gene=195, id=95 removed by soft count = 351\n",
      "--- number of probes: 199 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 196\n",
      "--- gene=196, id=32 removed by soft count = 59\n",
      "--- gene=196, id=121 removed by soft count = 214\n",
      "--- gene=196, id=133 removed by soft count = 52\n",
      "--- gene=196, id=172 removed by soft count = 53\n",
      "--- gene=196, id=177 removed by soft count = 236\n",
      "--- gene=196, id=178 removed by soft count = 899\n",
      "--- gene=196, id=204 removed by soft count = 102\n",
      "--- gene=196, id=277 removed by soft count = 303\n",
      "--- gene=196, id=278 removed by soft count = 182\n",
      "--- number of probes: 317 , kept by blast: 308 , if remove dups: 308\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 197\n",
      "--- gene=197, id=10 removed by soft count = 76\n",
      "--- gene=197, id=14 removed by soft count = 159\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=197, id=166 removed by soft count = 55\n",
      "--- gene=197, id=186 removed by soft count = 53\n",
      "--- gene=197, id=197 removed by soft count = 422\n",
      "--- gene=197, id=198 removed by soft count = 212\n",
      "--- number of probes: 251 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 198\n",
      "--- gene=198, id=190 removed by soft count = 104\n",
      "--- gene=198, id=193 removed by soft count = 99\n",
      "--- gene=198, id=194 removed by soft count = 134\n",
      "--- gene=198, id=195 removed by soft count = 162\n",
      "--- number of probes: 379 , kept by blast: 375 , if remove dups: 375\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 199\n",
      "--- gene=199, id=7 removed by soft count = 121\n",
      "--- gene=199, id=33 removed by soft count = 259\n",
      "--- gene=199, id=54 removed by soft count = 56\n",
      "--- gene=199, id=55 removed by soft count = 1149\n",
      "--- gene=199, id=56 removed by soft count = 205\n",
      "--- gene=199, id=105 removed by soft count = 168\n",
      "--- gene=199, id=110 removed by soft count = 60\n",
      "--- gene=199, id=115 removed by soft count = 82\n",
      "--- gene=199, id=116 removed by soft count = 195\n",
      "--- gene=199, id=117 removed by soft count = 69\n",
      "--- gene=199, id=120 removed by soft count = 100\n",
      "--- gene=199, id=143 removed by soft count = 160\n",
      "--- gene=199, id=144 removed by soft count = 58\n",
      "--- gene=199, id=151 removed by soft count = 52\n",
      "--- gene=199, id=154 removed by soft count = 79\n",
      "--- gene=199, id=161 removed by soft count = 51\n",
      "--- gene=199, id=164 removed by soft count = 58\n",
      "--- gene=199, id=167 removed by soft count = 55\n",
      "--- gene=199, id=171 removed by soft count = 282\n",
      "--- gene=199, id=172 removed by soft count = 1048\n",
      "--- gene=199, id=268 removed by soft count = 72\n",
      "--- gene=199, id=269 removed by soft count = 170\n",
      "--- number of probes: 275 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 2\n",
      "--- gene=2, id=261 removed by soft count = 65\n",
      "--- number of probes: 377 , kept by blast: 376 , if remove dups: 376\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 20\n",
      "--- gene=20, id=97 removed by soft count = 1203\n",
      "--- gene=20, id=112 removed by soft count = 139\n",
      "--- gene=20, id=128 removed by soft count = 212\n",
      "--- number of probes: 206 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 200\n",
      "--- gene=200, id=0 removed by soft count = 699\n",
      "--- gene=200, id=1 removed by soft count = 1180\n",
      "--- gene=200, id=3 removed by soft count = 64\n",
      "--- gene=200, id=4 removed by soft count = 119\n",
      "--- gene=200, id=5 removed by soft count = 469\n",
      "--- gene=200, id=6 removed by soft count = 52\n",
      "--- gene=200, id=7 removed by soft count = 90\n",
      "--- gene=200, id=8 removed by soft count = 104\n",
      "--- gene=200, id=172 removed by soft count = 308\n",
      "--- number of probes: 229 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 201\n",
      "--- number of probes: 361 , kept by blast: 361 , if remove dups: 361\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 202\n",
      "--- gene=202, id=143 removed by soft count = 133\n",
      "--- gene=202, id=281 removed by soft count = 55\n",
      "--- gene=202, id=312 removed by soft count = 59\n",
      "--- number of probes: 331 , kept by blast: 328 , if remove dups: 328\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 203\n",
      "--- gene=203, id=11 removed by soft count = 112\n",
      "--- gene=203, id=81 removed by soft count = 930\n",
      "--- gene=203, id=84 removed by soft count = 129\n",
      "--- gene=203, id=92 removed by soft count = 671\n",
      "--- gene=203, id=160 removed by soft count = 311\n",
      "--- number of probes: 190 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 204\n",
      "--- gene=204, id=129 removed by soft count = 599\n",
      "--- number of probes: 265 , kept by blast: 264 , if remove dups: 264\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 205\n",
      "--- gene=205, id=127 removed by soft count = 71\n",
      "--- gene=205, id=128 removed by soft count = 56\n",
      "--- gene=205, id=134 removed by soft count = 83\n",
      "--- gene=205, id=136 removed by soft count = 733\n",
      "--- gene=205, id=137 removed by soft count = 310\n",
      "--- number of probes: 241 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 206\n",
      "--- gene=206, id=0 removed by soft count = 180\n",
      "--- gene=206, id=1 removed by soft count = 691\n",
      "--- gene=206, id=2 removed by soft count = 168\n",
      "--- gene=206, id=3 removed by soft count = 143\n",
      "--- gene=206, id=4 removed by soft count = 73\n",
      "--- gene=206, id=5 removed by soft count = 60\n",
      "--- gene=206, id=8 removed by soft count = 64\n",
      "--- gene=206, id=144 removed by soft count = 2871\n",
      "--- gene=206, id=156 removed by soft count = 120\n",
      "--- gene=206, id=258 removed by soft count = 147\n",
      "--- gene=206, id=259 removed by soft count = 246\n",
      "--- gene=206, id=260 removed by soft count = 62\n",
      "--- number of probes: 262 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 207\n",
      "--- gene=207, id=273 removed by soft count = 92\n",
      "--- gene=207, id=278 removed by soft count = 343\n",
      "--- number of probes: 324 , kept by blast: 322 , if remove dups: 322\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 208\n",
      "--- gene=208, id=56 removed by soft count = 78\n",
      "--- gene=208, id=58 removed by soft count = 257\n",
      "--- gene=208, id=59 removed by soft count = 450\n",
      "--- gene=208, id=60 removed by soft count = 65\n",
      "--- gene=208, id=122 removed by soft count = 494\n",
      "--- gene=208, id=126 removed by soft count = 163\n",
      "--- number of probes: 174 , kept by blast: 168 , if remove dups: 168\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 209\n",
      "--- gene=209, id=0 removed by soft count = 230\n",
      "--- gene=209, id=29 removed by soft count = 77\n",
      "--- gene=209, id=30 removed by soft count = 65\n",
      "--- gene=209, id=91 removed by soft count = 83\n",
      "--- gene=209, id=92 removed by soft count = 59\n",
      "--- gene=209, id=94 removed by soft count = 52\n",
      "--- gene=209, id=95 removed by soft count = 84\n",
      "--- gene=209, id=97 removed by soft count = 84\n",
      "--- gene=209, id=101 removed by soft count = 253\n",
      "--- gene=209, id=102 removed by soft count = 325\n",
      "--- gene=209, id=103 removed by soft count = 158\n",
      "--- gene=209, id=104 removed by soft count = 353\n",
      "--- gene=209, id=105 removed by soft count = 410\n",
      "--- gene=209, id=107 removed by soft count = 167\n",
      "--- gene=209, id=109 removed by soft count = 1218\n",
      "--- gene=209, id=167 removed by soft count = 457\n",
      "--- gene=209, id=168 removed by soft count = 205\n",
      "--- number of probes: 180 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 21\n",
      "--- gene=21, id=108 removed by soft count = 61\n",
      "--- gene=21, id=220 removed by soft count = 390\n",
      "--- number of probes: 255 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 210\n",
      "--- gene=210, id=168 removed by soft count = 71\n",
      "--- gene=210, id=363 removed by soft count = 194\n",
      "--- number of probes: 365 , kept by blast: 363 , if remove dups: 363\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 211\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=211, id=250 removed by soft count = 75\n",
      "--- number of probes: 349 , kept by blast: 348 , if remove dups: 348\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 212\n",
      "--- gene=212, id=104 removed by soft count = 207\n",
      "--- gene=212, id=105 removed by soft count = 171\n",
      "--- gene=212, id=117 removed by soft count = 234\n",
      "--- gene=212, id=162 removed by soft count = 58\n",
      "--- gene=212, id=203 removed by soft count = 142\n",
      "--- gene=212, id=255 removed by soft count = 58\n",
      "--- number of probes: 330 , kept by blast: 324 , if remove dups: 324\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 213\n",
      "--- gene=213, id=26 removed by soft count = 151\n",
      "--- gene=213, id=81 removed by soft count = 283\n",
      "--- gene=213, id=84 removed by soft count = 579\n",
      "--- gene=213, id=103 removed by soft count = 865\n",
      "--- gene=213, id=104 removed by soft count = 694\n",
      "--- gene=213, id=112 removed by soft count = 56\n",
      "--- gene=213, id=141 removed by soft count = 191\n",
      "--- gene=213, id=142 removed by soft count = 75\n",
      "--- gene=213, id=146 removed by soft count = 210\n",
      "--- number of probes: 200 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 214\n",
      "--- gene=214, id=0 removed by soft count = 85\n",
      "--- gene=214, id=1 removed by soft count = 155\n",
      "--- gene=214, id=2 removed by soft count = 239\n",
      "--- gene=214, id=4 removed by soft count = 146\n",
      "--- gene=214, id=201 removed by soft count = 100\n",
      "--- number of probes: 219 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 215\n",
      "--- gene=215, id=31 removed by soft count = 205\n",
      "--- gene=215, id=33 removed by soft count = 67\n",
      "--- gene=215, id=34 removed by soft count = 2263\n",
      "--- gene=215, id=35 removed by soft count = 128\n",
      "--- gene=215, id=48 removed by soft count = 151\n",
      "--- gene=215, id=49 removed by soft count = 140\n",
      "--- gene=215, id=53 removed by soft count = 162\n",
      "--- gene=215, id=89 removed by soft count = 391\n",
      "--- gene=215, id=140 removed by soft count = 920\n",
      "--- gene=215, id=141 removed by soft count = 79\n",
      "--- number of probes: 142 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 216\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 217\n",
      "--- gene=217, id=63 removed by soft count = 159\n",
      "--- gene=217, id=82 removed by soft count = 935\n",
      "--- gene=217, id=84 removed by soft count = 157\n",
      "--- gene=217, id=169 removed by soft count = 64\n",
      "--- number of probes: 278 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 218\n",
      "--- gene=218, id=229 removed by soft count = 72\n",
      "--- number of probes: 252 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 219\n",
      "--- gene=219, id=83 removed by soft count = 286\n",
      "--- gene=219, id=84 removed by soft count = 336\n",
      "--- gene=219, id=85 removed by soft count = 467\n",
      "--- gene=219, id=112 removed by soft count = 259\n",
      "--- gene=219, id=245 removed by soft count = 11800\n",
      "--- gene=219, id=246 removed by soft count = 825\n",
      "--- number of probes: 258 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 22\n",
      "--- gene=22, id=72 removed by soft count = 145\n",
      "--- gene=22, id=164 removed by soft count = 84\n",
      "--- gene=22, id=165 removed by soft count = 56\n",
      "--- number of probes: 303 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 220\n",
      "--- number of probes: 214 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 221\n",
      "--- gene=221, id=24 removed by soft count = 72\n",
      "--- gene=221, id=117 removed by soft count = 111\n",
      "--- gene=221, id=165 removed by soft count = 216\n",
      "--- gene=221, id=176 removed by soft count = 147\n",
      "--- gene=221, id=223 removed by soft count = 111\n",
      "--- number of probes: 256 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 222\n",
      "--- gene=222, id=62 removed by soft count = 86\n",
      "--- gene=222, id=66 removed by soft count = 76\n",
      "--- gene=222, id=71 removed by soft count = 58\n",
      "--- gene=222, id=102 removed by soft count = 1693\n",
      "--- gene=222, id=183 removed by soft count = 56\n",
      "--- gene=222, id=230 removed by soft count = 123\n",
      "--- gene=222, id=281 removed by soft count = 288\n",
      "--- number of probes: 292 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 223\n",
      "--- number of probes: 318 , kept by blast: 318 , if remove dups: 318\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 224\n",
      "--- gene=224, id=69 removed by soft count = 113\n",
      "--- gene=224, id=170 removed by soft count = 72\n",
      "--- gene=224, id=268 removed by soft count = 456\n",
      "--- gene=224, id=286 removed by soft count = 264\n",
      "--- gene=224, id=300 removed by soft count = 144\n",
      "--- gene=224, id=302 removed by soft count = 120\n",
      "--- gene=224, id=307 removed by soft count = 76\n",
      "--- number of probes: 322 , kept by blast: 315 , if remove dups: 315\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 225\n",
      "--- gene=225, id=4 removed by soft count = 62\n",
      "--- gene=225, id=25 removed by soft count = 54\n",
      "--- gene=225, id=26 removed by soft count = 103\n",
      "--- gene=225, id=64 removed by soft count = 1379\n",
      "--- gene=225, id=176 removed by soft count = 352\n",
      "--- number of probes: 233 , kept by blast: 228 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 226\n",
      "--- gene=226, id=186 removed by soft count = 393\n",
      "--- gene=226, id=190 removed by soft count = 488\n",
      "--- number of probes: 220 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 227\n",
      "--- gene=227, id=126 removed by soft count = 171\n",
      "--- gene=227, id=128 removed by soft count = 266\n",
      "--- gene=227, id=129 removed by soft count = 160\n",
      "--- gene=227, id=148 removed by soft count = 166\n",
      "--- number of probes: 242 , kept by blast: 238 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 228\n",
      "--- gene=228, id=42 removed by soft count = 123\n",
      "--- gene=228, id=67 removed by soft count = 469\n",
      "--- gene=228, id=97 removed by soft count = 224\n",
      "--- gene=228, id=108 removed by soft count = 66\n",
      "--- gene=228, id=109 removed by soft count = 376\n",
      "--- gene=228, id=110 removed by soft count = 346\n",
      "--- gene=228, id=111 removed by soft count = 64\n",
      "--- gene=228, id=112 removed by soft count = 353\n",
      "--- number of probes: 148 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 229\n",
      "--- gene=229, id=40 removed by soft count = 93\n",
      "--- gene=229, id=186 removed by soft count = 76\n",
      "--- number of probes: 197 , kept by blast: 195 , if remove dups: 195\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 23\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 333 , kept by blast: 333 , if remove dups: 333\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 230\n",
      "--- gene=230, id=0 removed by soft count = 789\n",
      "--- gene=230, id=1 removed by soft count = 186\n",
      "--- gene=230, id=92 removed by soft count = 1267\n",
      "--- gene=230, id=93 removed by soft count = 122\n",
      "--- gene=230, id=94 removed by soft count = 3630\n",
      "--- gene=230, id=152 removed by soft count = 68\n",
      "--- gene=230, id=157 removed by soft count = 2464\n",
      "--- gene=230, id=162 removed by soft count = 168\n",
      "--- gene=230, id=165 removed by soft count = 3321\n",
      "--- number of probes: 166 , kept by blast: 157 , if remove dups: 157\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 233\n",
      "--- gene=233, id=58 removed by soft count = 216\n",
      "--- gene=233, id=114 removed by soft count = 649\n",
      "--- gene=233, id=157 removed by soft count = 207\n",
      "--- gene=233, id=186 removed by soft count = 96\n",
      "--- gene=233, id=194 removed by soft count = 207\n",
      "--- number of probes: 196 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 234\n",
      "--- gene=234, id=35 removed by soft count = 409\n",
      "--- gene=234, id=79 removed by soft count = 278\n",
      "--- gene=234, id=152 removed by soft count = 215\n",
      "--- gene=234, id=153 removed by soft count = 140\n",
      "--- gene=234, id=154 removed by soft count = 188\n",
      "--- gene=234, id=155 removed by soft count = 324\n",
      "--- gene=234, id=156 removed by soft count = 471\n",
      "--- gene=234, id=158 removed by soft count = 69\n",
      "--- gene=234, id=160 removed by soft count = 139\n",
      "--- number of probes: 255 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 235\n",
      "--- gene=235, id=14 removed by soft count = 914\n",
      "--- gene=235, id=15 removed by soft count = 438\n",
      "--- gene=235, id=16 removed by soft count = 634\n",
      "--- gene=235, id=55 removed by soft count = 1526\n",
      "--- gene=235, id=91 removed by soft count = 381\n",
      "--- number of probes: 233 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 236\n",
      "--- gene=236, id=31 removed by soft count = 98\n",
      "--- gene=236, id=59 removed by soft count = 141\n",
      "--- gene=236, id=69 removed by soft count = 100\n",
      "--- gene=236, id=70 removed by soft count = 237\n",
      "--- gene=236, id=75 removed by soft count = 263\n",
      "--- gene=236, id=79 removed by soft count = 97\n",
      "--- gene=236, id=96 removed by soft count = 179\n",
      "--- gene=236, id=97 removed by soft count = 245\n",
      "--- gene=236, id=133 removed by soft count = 57\n",
      "--- gene=236, id=134 removed by soft count = 454\n",
      "--- gene=236, id=142 removed by soft count = 189\n",
      "--- gene=236, id=147 removed by soft count = 14074\n",
      "--- gene=236, id=148 removed by soft count = 764\n",
      "--- number of probes: 182 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 237\n",
      "--- gene=237, id=13 removed by soft count = 91\n",
      "--- gene=237, id=16 removed by soft count = 231\n",
      "--- gene=237, id=123 removed by soft count = 64\n",
      "--- gene=237, id=130 removed by soft count = 110\n",
      "--- gene=237, id=131 removed by soft count = 69\n",
      "--- gene=237, id=132 removed by soft count = 58\n",
      "--- gene=237, id=135 removed by soft count = 208\n",
      "--- gene=237, id=137 removed by soft count = 58\n",
      "--- gene=237, id=146 removed by soft count = 82\n",
      "--- gene=237, id=147 removed by soft count = 58\n",
      "--- gene=237, id=148 removed by soft count = 71\n",
      "--- gene=237, id=150 removed by soft count = 183\n",
      "--- gene=237, id=151 removed by soft count = 202\n",
      "--- gene=237, id=155 removed by soft count = 155\n",
      "--- gene=237, id=156 removed by soft count = 365\n",
      "--- number of probes: 179 , kept by blast: 164 , if remove dups: 164\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 238\n",
      "--- gene=238, id=22 removed by soft count = 98\n",
      "--- gene=238, id=27 removed by soft count = 181\n",
      "--- gene=238, id=28 removed by soft count = 57\n",
      "--- gene=238, id=249 removed by soft count = 748\n",
      "--- gene=238, id=250 removed by soft count = 75\n",
      "--- number of probes: 251 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 239\n",
      "--- gene=239, id=3 removed by soft count = 60\n",
      "--- gene=239, id=4 removed by soft count = 55\n",
      "--- gene=239, id=5 removed by soft count = 105\n",
      "--- gene=239, id=37 removed by soft count = 580\n",
      "--- gene=239, id=47 removed by soft count = 74\n",
      "--- gene=239, id=50 removed by soft count = 57\n",
      "--- gene=239, id=148 removed by soft count = 74\n",
      "--- gene=239, id=192 removed by soft count = 57\n",
      "--- gene=239, id=197 removed by soft count = 55\n",
      "--- gene=239, id=198 removed by soft count = 81\n",
      "--- gene=239, id=217 removed by soft count = 114\n",
      "--- number of probes: 231 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 24\n",
      "--- gene=24, id=249 removed by soft count = 105\n",
      "--- number of probes: 279 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 240\n",
      "--- gene=240, id=204 removed by soft count = 52\n",
      "--- number of probes: 234 , kept by blast: 233 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 241\n",
      "--- gene=241, id=269 removed by soft count = 20024\n",
      "--- number of probes: 272 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 242\n",
      "--- gene=242, id=13 removed by soft count = 114\n",
      "--- gene=242, id=15 removed by soft count = 380\n",
      "--- gene=242, id=16 removed by soft count = 79\n",
      "--- gene=242, id=65 removed by soft count = 106\n",
      "--- gene=242, id=168 removed by soft count = 71\n",
      "--- gene=242, id=287 removed by soft count = 272\n",
      "--- number of probes: 288 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 243\n",
      "--- gene=243, id=7 removed by soft count = 56\n",
      "--- gene=243, id=11 removed by soft count = 60\n",
      "--- gene=243, id=22 removed by soft count = 109\n",
      "--- gene=243, id=23 removed by soft count = 281\n",
      "--- gene=243, id=42 removed by soft count = 56\n",
      "--- gene=243, id=56 removed by soft count = 77\n",
      "--- gene=243, id=57 removed by soft count = 100\n",
      "--- gene=243, id=58 removed by soft count = 88\n",
      "--- gene=243, id=67 removed by soft count = 61\n",
      "--- gene=243, id=71 removed by soft count = 79\n",
      "--- gene=243, id=76 removed by soft count = 272\n",
      "--- gene=243, id=78 removed by soft count = 2167\n",
      "--- gene=243, id=79 removed by soft count = 86\n",
      "--- gene=243, id=84 removed by soft count = 83\n",
      "--- number of probes: 165 , kept by blast: 151 , if remove dups: 151\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 244\n",
      "--- gene=244, id=93 removed by soft count = 74\n",
      "--- gene=244, id=307 removed by soft count = 115\n",
      "--- number of probes: 333 , kept by blast: 331 , if remove dups: 330\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 245\n",
      "--- gene=245, id=11 removed by soft count = 77\n",
      "--- gene=245, id=13 removed by soft count = 753\n",
      "--- gene=245, id=14 removed by soft count = 231\n",
      "--- gene=245, id=33 removed by soft count = 137\n",
      "--- gene=245, id=39 removed by soft count = 105\n",
      "--- gene=245, id=41 removed by soft count = 73\n",
      "--- gene=245, id=48 removed by soft count = 66\n",
      "--- gene=245, id=49 removed by soft count = 61\n",
      "--- gene=245, id=69 removed by soft count = 140\n",
      "--- gene=245, id=70 removed by soft count = 232\n",
      "--- gene=245, id=73 removed by soft count = 266\n",
      "--- gene=245, id=74 removed by soft count = 80\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=245, id=90 removed by soft count = 375\n",
      "--- gene=245, id=101 removed by soft count = 221\n",
      "--- gene=245, id=109 removed by soft count = 103\n",
      "--- gene=245, id=114 removed by soft count = 214\n",
      "--- gene=245, id=115 removed by soft count = 155\n",
      "--- gene=245, id=120 removed by soft count = 165\n",
      "--- gene=245, id=122 removed by soft count = 66\n",
      "--- gene=245, id=124 removed by soft count = 99\n",
      "--- gene=245, id=125 removed by soft count = 51\n",
      "--- number of probes: 238 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 246\n",
      "--- gene=246, id=48 removed by soft count = 567\n",
      "--- gene=246, id=49 removed by soft count = 105\n",
      "--- gene=246, id=53 removed by soft count = 312\n",
      "--- gene=246, id=99 removed by soft count = 437\n",
      "--- gene=246, id=129 removed by soft count = 88\n",
      "--- number of probes: 205 , kept by blast: 200 , if remove dups: 200\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 247\n",
      "--- gene=247, id=99 removed by soft count = 192\n",
      "--- gene=247, id=101 removed by soft count = 225\n",
      "--- gene=247, id=102 removed by soft count = 117\n",
      "--- gene=247, id=145 removed by soft count = 76\n",
      "--- gene=247, id=189 removed by soft count = 647\n",
      "--- gene=247, id=217 removed by soft count = 253\n",
      "--- gene=247, id=233 removed by soft count = 87\n",
      "--- number of probes: 234 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 248\n",
      "--- gene=248, id=0 removed by soft count = 173\n",
      "--- gene=248, id=1 removed by soft count = 1871\n",
      "--- number of probes: 277 , kept by blast: 275 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 249\n",
      "--- gene=249, id=9 removed by soft count = 99\n",
      "--- gene=249, id=115 removed by soft count = 59\n",
      "--- number of probes: 242 , kept by blast: 240 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 25\n",
      "--- gene=25, id=2 removed by soft count = 78\n",
      "--- gene=25, id=3 removed by soft count = 173\n",
      "--- gene=25, id=4 removed by soft count = 85\n",
      "--- gene=25, id=14 removed by soft count = 70\n",
      "--- gene=25, id=15 removed by soft count = 592\n",
      "--- gene=25, id=16 removed by soft count = 61\n",
      "--- gene=25, id=17 removed by soft count = 432\n",
      "--- gene=25, id=19 removed by soft count = 78\n",
      "--- gene=25, id=147 removed by soft count = 128\n",
      "--- gene=25, id=221 removed by soft count = 57\n",
      "--- number of probes: 238 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 250\n",
      "--- gene=250, id=29 removed by soft count = 67\n",
      "--- gene=250, id=55 removed by soft count = 68\n",
      "--- gene=250, id=56 removed by soft count = 80\n",
      "--- gene=250, id=58 removed by soft count = 142\n",
      "--- gene=250, id=60 removed by soft count = 151\n",
      "--- gene=250, id=61 removed by soft count = 149\n",
      "--- gene=250, id=63 removed by soft count = 65\n",
      "--- gene=250, id=65 removed by soft count = 142\n",
      "--- gene=250, id=86 removed by soft count = 171\n",
      "--- number of probes: 172 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 251\n",
      "--- gene=251, id=50 removed by soft count = 547\n",
      "--- gene=251, id=219 removed by soft count = 54\n",
      "--- gene=251, id=220 removed by soft count = 115\n",
      "--- gene=251, id=355 removed by soft count = 85\n",
      "--- number of probes: 357 , kept by blast: 353 , if remove dups: 353\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 252\n",
      "--- gene=252, id=34 removed by soft count = 57\n",
      "--- gene=252, id=63 removed by soft count = 217\n",
      "--- gene=252, id=65 removed by soft count = 91\n",
      "--- gene=252, id=74 removed by soft count = 52\n",
      "--- number of probes: 287 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 253\n",
      "--- gene=253, id=1 removed by soft count = 90\n",
      "--- gene=253, id=2 removed by soft count = 146\n",
      "--- gene=253, id=3 removed by soft count = 306\n",
      "--- gene=253, id=4 removed by soft count = 265\n",
      "--- gene=253, id=15 removed by soft count = 106\n",
      "--- gene=253, id=16 removed by soft count = 235\n",
      "--- gene=253, id=20 removed by soft count = 344\n",
      "--- gene=253, id=28 removed by soft count = 110\n",
      "--- gene=253, id=49 removed by soft count = 70\n",
      "--- gene=253, id=51 removed by soft count = 206\n",
      "--- gene=253, id=68 removed by soft count = 129\n",
      "--- gene=253, id=105 removed by soft count = 735\n",
      "--- gene=253, id=131 removed by soft count = 919\n",
      "--- gene=253, id=132 removed by soft count = 84\n",
      "--- gene=253, id=133 removed by soft count = 208\n",
      "--- gene=253, id=135 removed by soft count = 61\n",
      "--- gene=253, id=139 removed by soft count = 150\n",
      "--- gene=253, id=145 removed by soft count = 343\n",
      "--- number of probes: 188 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 254\n",
      "--- number of probes: 282 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 255\n",
      "--- gene=255, id=75 removed by soft count = 119\n",
      "--- gene=255, id=94 removed by soft count = 285\n",
      "--- number of probes: 225 , kept by blast: 223 , if remove dups: 223\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 256\n",
      "--- gene=256, id=5 removed by soft count = 68\n",
      "--- gene=256, id=175 removed by soft count = 136\n",
      "--- number of probes: 218 , kept by blast: 216 , if remove dups: 216\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 257\n",
      "--- gene=257, id=278 removed by soft count = 94\n",
      "--- gene=257, id=279 removed by soft count = 75\n",
      "--- number of probes: 288 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 258\n",
      "--- gene=258, id=132 removed by soft count = 1593\n",
      "--- gene=258, id=136 removed by soft count = 113\n",
      "--- gene=258, id=203 removed by soft count = 180\n",
      "--- gene=258, id=213 removed by soft count = 620\n",
      "--- gene=258, id=220 removed by soft count = 112\n",
      "--- gene=258, id=224 removed by soft count = 92\n",
      "--- gene=258, id=225 removed by soft count = 482\n",
      "--- number of probes: 314 , kept by blast: 307 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 259\n",
      "--- gene=259, id=51 removed by soft count = 157\n",
      "--- gene=259, id=114 removed by soft count = 350\n",
      "--- gene=259, id=126 removed by soft count = 77\n",
      "--- gene=259, id=129 removed by soft count = 148\n",
      "--- gene=259, id=135 removed by soft count = 107\n",
      "--- gene=259, id=139 removed by soft count = 68\n",
      "--- gene=259, id=141 removed by soft count = 772\n",
      "--- number of probes: 160 , kept by blast: 153 , if remove dups: 153\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 26\n",
      "--- number of probes: 328 , kept by blast: 328 , if remove dups: 328\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 260\n",
      "--- gene=260, id=40 removed by soft count = 194\n",
      "--- gene=260, id=41 removed by soft count = 1671\n",
      "--- gene=260, id=45 removed by soft count = 434\n",
      "--- gene=260, id=46 removed by soft count = 260\n",
      "--- gene=260, id=56 removed by soft count = 264\n",
      "--- gene=260, id=59 removed by soft count = 198\n",
      "--- gene=260, id=125 removed by soft count = 4490\n",
      "--- gene=260, id=126 removed by soft count = 806\n",
      "--- gene=260, id=134 removed by soft count = 86\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=260, id=138 removed by soft count = 190\n",
      "--- gene=260, id=142 removed by soft count = 51\n",
      "--- gene=260, id=144 removed by soft count = 58\n",
      "--- gene=260, id=145 removed by soft count = 1327\n",
      "--- number of probes: 162 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 261\n",
      "--- gene=261, id=74 removed by soft count = 851\n",
      "--- gene=261, id=77 removed by soft count = 63\n",
      "--- gene=261, id=78 removed by soft count = 325\n",
      "--- gene=261, id=106 removed by soft count = 271\n",
      "--- gene=261, id=107 removed by soft count = 477\n",
      "--- gene=261, id=118 removed by soft count = 288\n",
      "--- gene=261, id=119 removed by soft count = 359\n",
      "--- gene=261, id=131 removed by soft count = 58\n",
      "--- gene=261, id=142 removed by soft count = 115\n",
      "--- number of probes: 159 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 262\n",
      "--- gene=262, id=79 removed by soft count = 985\n",
      "--- gene=262, id=80 removed by soft count = 94\n",
      "--- gene=262, id=150 removed by soft count = 207\n",
      "--- gene=262, id=151 removed by soft count = 80\n",
      "--- gene=262, id=154 removed by soft count = 71\n",
      "--- gene=262, id=156 removed by soft count = 138\n",
      "--- number of probes: 179 , kept by blast: 173 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 263\n",
      "--- gene=263, id=263 removed by soft count = 53\n",
      "--- gene=263, id=265 removed by soft count = 291\n",
      "--- number of probes: 323 , kept by blast: 321 , if remove dups: 319\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 264\n",
      "--- gene=264, id=25 removed by soft count = 832\n",
      "--- gene=264, id=27 removed by soft count = 180\n",
      "--- gene=264, id=194 removed by soft count = 90\n",
      "--- number of probes: 259 , kept by blast: 256 , if remove dups: 256\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 265\n",
      "--- gene=265, id=58 removed by soft count = 185\n",
      "--- gene=265, id=64 removed by soft count = 95\n",
      "--- gene=265, id=189 removed by soft count = 129\n",
      "--- gene=265, id=200 removed by soft count = 84\n",
      "--- gene=265, id=202 removed by soft count = 58\n",
      "--- gene=265, id=209 removed by soft count = 73\n",
      "--- gene=265, id=211 removed by soft count = 146\n",
      "--- gene=265, id=213 removed by soft count = 241\n",
      "--- gene=265, id=214 removed by soft count = 447\n",
      "--- gene=265, id=216 removed by soft count = 249\n",
      "--- gene=265, id=219 removed by soft count = 109\n",
      "--- number of probes: 220 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 266\n",
      "--- gene=266, id=118 removed by soft count = 275\n",
      "--- gene=266, id=119 removed by soft count = 153\n",
      "--- gene=266, id=120 removed by soft count = 834\n",
      "--- gene=266, id=121 removed by soft count = 2334\n",
      "--- number of probes: 251 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 267\n",
      "--- gene=267, id=101 removed by soft count = 303\n",
      "--- gene=267, id=202 removed by soft count = 634\n",
      "--- number of probes: 263 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 268\n",
      "--- gene=268, id=24 removed by soft count = 61\n",
      "--- gene=268, id=26 removed by soft count = 498\n",
      "--- gene=268, id=27 removed by soft count = 418\n",
      "--- gene=268, id=28 removed by soft count = 376\n",
      "--- gene=268, id=30 removed by soft count = 3067\n",
      "--- gene=268, id=31 removed by soft count = 78\n",
      "--- gene=268, id=32 removed by soft count = 809\n",
      "--- gene=268, id=70 removed by soft count = 40011\n",
      "--- gene=268, id=71 removed by soft count = 53\n",
      "--- gene=268, id=130 removed by soft count = 3300\n",
      "--- gene=268, id=167 removed by soft count = 477\n",
      "--- number of probes: 175 , kept by blast: 164 , if remove dups: 164\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 269\n",
      "--- gene=269, id=205 removed by soft count = 87\n",
      "--- gene=269, id=327 removed by soft count = 72\n",
      "--- number of probes: 351 , kept by blast: 349 , if remove dups: 349\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 27\n",
      "--- gene=27, id=17 removed by soft count = 461\n",
      "--- gene=27, id=18 removed by soft count = 325\n",
      "--- gene=27, id=19 removed by soft count = 603\n",
      "--- gene=27, id=20 removed by soft count = 888\n",
      "--- gene=27, id=135 removed by soft count = 69\n",
      "--- gene=27, id=137 removed by soft count = 147\n",
      "--- gene=27, id=138 removed by soft count = 88\n",
      "--- gene=27, id=140 removed by soft count = 144\n",
      "--- gene=27, id=141 removed by soft count = 84\n",
      "--- gene=27, id=142 removed by soft count = 754\n",
      "--- gene=27, id=143 removed by soft count = 277\n",
      "--- gene=27, id=144 removed by soft count = 137\n",
      "--- gene=27, id=145 removed by soft count = 133\n",
      "--- gene=27, id=146 removed by soft count = 65\n",
      "--- gene=27, id=147 removed by soft count = 1940\n",
      "--- gene=27, id=148 removed by soft count = 1515\n",
      "--- number of probes: 149 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 270\n",
      "--- gene=270, id=67 removed by soft count = 166\n",
      "--- gene=270, id=99 removed by soft count = 209\n",
      "--- number of probes: 313 , kept by blast: 311 , if remove dups: 311\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 271\n",
      "--- gene=271, id=16 removed by soft count = 141\n",
      "--- gene=271, id=18 removed by soft count = 67\n",
      "--- gene=271, id=24 removed by soft count = 53\n",
      "--- gene=271, id=26 removed by soft count = 109\n",
      "--- gene=271, id=42 removed by soft count = 95\n",
      "--- gene=271, id=43 removed by soft count = 70\n",
      "--- gene=271, id=68 removed by soft count = 108\n",
      "--- gene=271, id=71 removed by soft count = 101\n",
      "--- gene=271, id=105 removed by soft count = 151\n",
      "--- gene=271, id=144 removed by soft count = 52\n",
      "--- gene=271, id=147 removed by soft count = 52\n",
      "--- gene=271, id=148 removed by soft count = 167\n",
      "--- gene=271, id=155 removed by soft count = 269\n",
      "--- gene=271, id=160 removed by soft count = 459\n",
      "--- gene=271, id=161 removed by soft count = 81\n",
      "--- gene=271, id=193 removed by soft count = 65\n",
      "--- number of probes: 220 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 272\n",
      "--- gene=272, id=46 removed by soft count = 257\n",
      "--- gene=272, id=53 removed by soft count = 195\n",
      "--- gene=272, id=111 removed by soft count = 235\n",
      "--- gene=272, id=112 removed by soft count = 95\n",
      "--- gene=272, id=117 removed by soft count = 57\n",
      "--- gene=272, id=136 removed by soft count = 788\n",
      "--- gene=272, id=137 removed by soft count = 567\n",
      "--- gene=272, id=138 removed by soft count = 86\n",
      "--- gene=272, id=208 removed by soft count = 73\n",
      "--- gene=272, id=209 removed by soft count = 88\n",
      "--- number of probes: 232 , kept by blast: 222 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 273\n",
      "--- gene=273, id=109 removed by hard count = 4\n",
      "--- gene=273, id=136 removed by soft count = 51\n",
      "--- gene=273, id=137 removed by soft count = 59\n",
      "--- gene=273, id=186 removed by soft count = 138\n",
      "--- gene=273, id=188 removed by soft count = 196\n",
      "--- number of probes: 211 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 274\n",
      "--- gene=274, id=28 removed by soft count = 51\n",
      "--- gene=274, id=220 removed by soft count = 149\n",
      "--- gene=274, id=221 removed by soft count = 168\n",
      "--- gene=274, id=222 removed by soft count = 203\n",
      "--- gene=274, id=228 removed by soft count = 81\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=274, id=260 removed by soft count = 53\n",
      "--- gene=274, id=261 removed by soft count = 62\n",
      "--- gene=274, id=269 removed by soft count = 51\n",
      "--- gene=274, id=270 removed by soft count = 119\n",
      "--- gene=274, id=307 removed by soft count = 113\n",
      "--- gene=274, id=310 removed by soft count = 249\n",
      "--- number of probes: 312 , kept by blast: 301 , if remove dups: 301\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 275\n",
      "--- gene=275, id=140 removed by hard count = 5\n",
      "--- gene=275, id=182 removed by soft count = 83\n",
      "--- gene=275, id=256 removed by soft count = 54\n",
      "--- number of probes: 388 , kept by blast: 385 , if remove dups: 384\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 276\n",
      "--- gene=276, id=20 removed by soft count = 1382\n",
      "--- gene=276, id=25 removed by soft count = 483\n",
      "--- gene=276, id=26 removed by soft count = 556\n",
      "--- gene=276, id=39 removed by soft count = 1074\n",
      "--- gene=276, id=106 removed by soft count = 142\n",
      "--- gene=276, id=107 removed by soft count = 59\n",
      "--- gene=276, id=132 removed by soft count = 454\n",
      "--- gene=276, id=133 removed by soft count = 69\n",
      "--- gene=276, id=134 removed by soft count = 104\n",
      "--- gene=276, id=135 removed by soft count = 187\n",
      "--- gene=276, id=136 removed by soft count = 115\n",
      "--- gene=276, id=236 removed by soft count = 6667\n",
      "--- gene=276, id=267 removed by soft count = 60\n",
      "--- number of probes: 325 , kept by blast: 312 , if remove dups: 312\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 277\n",
      "--- gene=277, id=90 removed by soft count = 460\n",
      "--- gene=277, id=91 removed by soft count = 51\n",
      "--- gene=277, id=204 removed by soft count = 62\n",
      "--- gene=277, id=215 removed by soft count = 125\n",
      "--- gene=277, id=218 removed by soft count = 68\n",
      "--- gene=277, id=220 removed by soft count = 125\n",
      "--- gene=277, id=229 removed by soft count = 157\n",
      "--- gene=277, id=231 removed by soft count = 539\n",
      "--- gene=277, id=232 removed by soft count = 612\n",
      "--- number of probes: 313 , kept by blast: 304 , if remove dups: 304\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 278\n",
      "--- gene=278, id=18 removed by soft count = 57\n",
      "--- gene=278, id=20 removed by soft count = 54\n",
      "--- gene=278, id=22 removed by soft count = 82\n",
      "--- gene=278, id=45 removed by soft count = 53\n",
      "--- gene=278, id=128 removed by soft count = 154\n",
      "--- gene=278, id=164 removed by soft count = 715\n",
      "--- gene=278, id=165 removed by soft count = 132\n",
      "--- gene=278, id=166 removed by soft count = 81\n",
      "--- gene=278, id=206 removed by soft count = 94\n",
      "--- number of probes: 224 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 279\n",
      "--- gene=279, id=44 removed by soft count = 641\n",
      "--- gene=279, id=47 removed by soft count = 333\n",
      "--- gene=279, id=49 removed by soft count = 163\n",
      "--- gene=279, id=50 removed by soft count = 336\n",
      "--- gene=279, id=147 removed by soft count = 296\n",
      "--- gene=279, id=149 removed by soft count = 118\n",
      "--- gene=279, id=150 removed by soft count = 298\n",
      "--- gene=279, id=151 removed by soft count = 528\n",
      "--- gene=279, id=152 removed by soft count = 191\n",
      "--- gene=279, id=153 removed by soft count = 178\n",
      "--- gene=279, id=155 removed by soft count = 407\n",
      "--- number of probes: 210 , kept by blast: 199 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 28\n",
      "--- gene=28, id=8 removed by soft count = 1410\n",
      "--- gene=28, id=226 removed by soft count = 359\n",
      "--- gene=28, id=227 removed by soft count = 241\n",
      "--- number of probes: 262 , kept by blast: 259 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 280\n",
      "--- gene=280, id=106 removed by soft count = 94\n",
      "--- gene=280, id=193 removed by soft count = 154\n",
      "--- number of probes: 309 , kept by blast: 307 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 281\n",
      "--- gene=281, id=23 removed by soft count = 98\n",
      "--- gene=281, id=25 removed by soft count = 125\n",
      "--- gene=281, id=37 removed by soft count = 110\n",
      "--- gene=281, id=38 removed by soft count = 317\n",
      "--- gene=281, id=126 removed by soft count = 73\n",
      "--- gene=281, id=127 removed by soft count = 59\n",
      "--- gene=281, id=152 removed by soft count = 79\n",
      "--- gene=281, id=183 removed by soft count = 52\n",
      "--- gene=281, id=194 removed by soft count = 1011\n",
      "--- number of probes: 239 , kept by blast: 230 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 282\n",
      "--- gene=282, id=36 removed by soft count = 147\n",
      "--- gene=282, id=39 removed by soft count = 51\n",
      "--- number of probes: 262 , kept by blast: 260 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 283\n",
      "--- gene=283, id=41 removed by soft count = 288\n",
      "--- gene=283, id=271 removed by soft count = 506\n",
      "--- gene=283, id=272 removed by soft count = 289\n",
      "--- gene=283, id=273 removed by soft count = 553\n",
      "--- gene=283, id=274 removed by soft count = 452\n",
      "--- gene=283, id=275 removed by soft count = 2699\n",
      "--- number of probes: 293 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 284\n",
      "--- gene=284, id=20 removed by soft count = 61\n",
      "--- gene=284, id=84 removed by soft count = 76\n",
      "--- gene=284, id=92 removed by soft count = 102\n",
      "--- gene=284, id=93 removed by soft count = 273\n",
      "--- gene=284, id=165 removed by soft count = 112\n",
      "--- gene=284, id=167 removed by soft count = 160\n",
      "--- gene=284, id=170 removed by soft count = 295\n",
      "--- number of probes: 215 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 285\n",
      "--- gene=285, id=49 removed by soft count = 133\n",
      "--- gene=285, id=198 removed by soft count = 71\n",
      "--- gene=285, id=202 removed by soft count = 131\n",
      "--- gene=285, id=224 removed by soft count = 73\n",
      "--- gene=285, id=228 removed by soft count = 103\n",
      "--- number of probes: 234 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 286\n",
      "--- gene=286, id=30 removed by soft count = 109\n",
      "--- gene=286, id=79 removed by soft count = 55\n",
      "--- gene=286, id=90 removed by soft count = 158\n",
      "--- gene=286, id=92 removed by soft count = 108\n",
      "--- gene=286, id=139 removed by soft count = 2463\n",
      "--- gene=286, id=140 removed by soft count = 388\n",
      "--- gene=286, id=144 removed by soft count = 1968\n",
      "--- number of probes: 187 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 287\n",
      "--- gene=287, id=0 removed by soft count = 266\n",
      "--- gene=287, id=1 removed by soft count = 75\n",
      "--- gene=287, id=2 removed by soft count = 342\n",
      "--- gene=287, id=3 removed by soft count = 262\n",
      "--- gene=287, id=25 removed by soft count = 127\n",
      "--- gene=287, id=26 removed by soft count = 171\n",
      "--- gene=287, id=30 removed by soft count = 88\n",
      "--- gene=287, id=32 removed by soft count = 59\n",
      "--- gene=287, id=36 removed by soft count = 116\n",
      "--- gene=287, id=37 removed by soft count = 165\n",
      "--- gene=287, id=38 removed by soft count = 215\n",
      "--- gene=287, id=39 removed by soft count = 130\n",
      "--- gene=287, id=52 removed by soft count = 90\n",
      "--- gene=287, id=71 removed by soft count = 341\n",
      "--- gene=287, id=166 removed by soft count = 79\n",
      "--- gene=287, id=184 removed by soft count = 324\n",
      "--- gene=287, id=185 removed by soft count = 248\n",
      "--- gene=287, id=186 removed by soft count = 111\n",
      "--- number of probes: 194 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 288\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=288, id=98 removed by soft count = 3639\n",
      "--- gene=288, id=138 removed by soft count = 57\n",
      "--- gene=288, id=139 removed by soft count = 254\n",
      "--- gene=288, id=141 removed by soft count = 78\n",
      "--- gene=288, id=142 removed by soft count = 51\n",
      "--- number of probes: 249 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 289\n",
      "--- gene=289, id=90 removed by soft count = 55\n",
      "--- gene=289, id=91 removed by soft count = 71\n",
      "--- gene=289, id=93 removed by soft count = 98\n",
      "--- gene=289, id=94 removed by soft count = 683\n",
      "--- gene=289, id=123 removed by soft count = 133\n",
      "--- gene=289, id=125 removed by soft count = 71\n",
      "--- gene=289, id=126 removed by soft count = 161\n",
      "--- gene=289, id=206 removed by soft count = 568\n",
      "--- number of probes: 250 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 29\n",
      "--- gene=29, id=26 removed by soft count = 52\n",
      "--- gene=29, id=33 removed by soft count = 257\n",
      "--- gene=29, id=67 removed by soft count = 102\n",
      "--- gene=29, id=76 removed by soft count = 139\n",
      "--- gene=29, id=106 removed by soft count = 100\n",
      "--- gene=29, id=133 removed by soft count = 61\n",
      "--- number of probes: 199 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 290\n",
      "--- gene=290, id=31 removed by soft count = 77\n",
      "--- gene=290, id=87 removed by soft count = 60\n",
      "--- gene=290, id=88 removed by soft count = 271\n",
      "--- gene=290, id=89 removed by soft count = 85\n",
      "--- gene=290, id=91 removed by soft count = 493\n",
      "--- gene=290, id=109 removed by soft count = 380\n",
      "--- gene=290, id=110 removed by soft count = 148\n",
      "--- gene=290, id=111 removed by soft count = 66\n",
      "--- number of probes: 253 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 291\n",
      "--- gene=291, id=0 removed by soft count = 111\n",
      "--- gene=291, id=32 removed by soft count = 51\n",
      "--- gene=291, id=47 removed by soft count = 64\n",
      "--- gene=291, id=48 removed by soft count = 71\n",
      "--- gene=291, id=99 removed by soft count = 139\n",
      "--- gene=291, id=100 removed by soft count = 88\n",
      "--- gene=291, id=113 removed by soft count = 81\n",
      "--- number of probes: 195 , kept by blast: 188 , if remove dups: 188\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 292\n",
      "--- gene=292, id=222 removed by soft count = 124\n",
      "--- number of probes: 226 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 293\n",
      "--- gene=293, id=5 removed by soft count = 120\n",
      "--- gene=293, id=9 removed by soft count = 383\n",
      "--- gene=293, id=13 removed by soft count = 57\n",
      "--- gene=293, id=25 removed by soft count = 611\n",
      "--- gene=293, id=107 removed by soft count = 278\n",
      "--- gene=293, id=109 removed by soft count = 95\n",
      "--- gene=293, id=142 removed by soft count = 297\n",
      "--- gene=293, id=144 removed by soft count = 51\n",
      "--- gene=293, id=154 removed by soft count = 69\n",
      "--- number of probes: 157 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 295\n",
      "--- gene=295, id=20 removed by soft count = 396\n",
      "--- gene=295, id=73 removed by soft count = 79\n",
      "--- gene=295, id=74 removed by soft count = 87\n",
      "--- gene=295, id=75 removed by soft count = 392\n",
      "--- gene=295, id=179 removed by soft count = 1702\n",
      "--- gene=295, id=180 removed by soft count = 58\n",
      "--- number of probes: 209 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 296\n",
      "--- gene=296, id=11 removed by soft count = 78\n",
      "--- gene=296, id=13 removed by soft count = 318\n",
      "--- gene=296, id=208 removed by soft count = 227\n",
      "--- gene=296, id=213 removed by soft count = 182\n",
      "--- gene=296, id=216 removed by soft count = 310\n",
      "--- gene=296, id=217 removed by soft count = 130\n",
      "--- gene=296, id=218 removed by soft count = 92\n",
      "--- gene=296, id=220 removed by soft count = 147\n",
      "--- gene=296, id=238 removed by soft count = 117\n",
      "--- number of probes: 274 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 297\n",
      "--- gene=297, id=99 removed by soft count = 55\n",
      "--- gene=297, id=155 removed by soft count = 475\n",
      "--- gene=297, id=181 removed by soft count = 305\n",
      "--- gene=297, id=192 removed by soft count = 86\n",
      "--- number of probes: 194 , kept by blast: 190 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 298\n",
      "--- gene=298, id=41 removed by soft count = 126\n",
      "--- gene=298, id=216 removed by soft count = 430\n",
      "--- gene=298, id=236 removed by soft count = 78\n",
      "--- number of probes: 253 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 299\n",
      "--- gene=299, id=12 removed by soft count = 241\n",
      "--- gene=299, id=16 removed by soft count = 89\n",
      "--- number of probes: 141 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 3\n",
      "--- gene=3, id=6 removed by soft count = 164\n",
      "--- gene=3, id=49 removed by soft count = 242\n",
      "--- gene=3, id=55 removed by soft count = 294\n",
      "--- gene=3, id=70 removed by soft count = 140\n",
      "--- number of probes: 242 , kept by blast: 238 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 30\n",
      "--- gene=30, id=37 removed by soft count = 162\n",
      "--- gene=30, id=55 removed by soft count = 75\n",
      "--- gene=30, id=62 removed by soft count = 2849\n",
      "--- gene=30, id=64 removed by soft count = 547\n",
      "--- gene=30, id=65 removed by soft count = 368\n",
      "--- gene=30, id=67 removed by soft count = 102\n",
      "--- gene=30, id=68 removed by soft count = 152\n",
      "--- gene=30, id=69 removed by soft count = 141\n",
      "--- gene=30, id=80 removed by soft count = 76\n",
      "--- gene=30, id=81 removed by soft count = 97\n",
      "--- number of probes: 263 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 300\n",
      "--- gene=300, id=71 removed by soft count = 1797\n",
      "--- gene=300, id=98 removed by soft count = 5357\n",
      "--- gene=300, id=99 removed by soft count = 52\n",
      "--- gene=300, id=127 removed by soft count = 56\n",
      "--- number of probes: 151 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 302\n",
      "--- number of probes: 328 , kept by blast: 328 , if remove dups: 328\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 303\n",
      "--- gene=303, id=74 removed by soft count = 116\n",
      "--- gene=303, id=134 removed by soft count = 63\n",
      "--- gene=303, id=200 removed by soft count = 71\n",
      "--- gene=303, id=212 removed by soft count = 164\n",
      "--- number of probes: 213 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 304\n",
      "--- gene=304, id=15 removed by soft count = 116\n",
      "--- gene=304, id=31 removed by soft count = 197\n",
      "--- gene=304, id=53 removed by soft count = 58\n",
      "--- gene=304, id=80 removed by soft count = 145\n",
      "--- gene=304, id=125 removed by soft count = 441\n",
      "--- gene=304, id=126 removed by soft count = 86\n",
      "--- gene=304, id=191 removed by soft count = 54\n",
      "--- gene=304, id=192 removed by soft count = 52\n",
      "--- gene=304, id=193 removed by soft count = 271\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=304, id=202 removed by soft count = 726\n",
      "--- gene=304, id=212 removed by soft count = 53\n",
      "--- number of probes: 232 , kept by blast: 221 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 305\n",
      "--- gene=305, id=24 removed by soft count = 99\n",
      "--- gene=305, id=67 removed by soft count = 69\n",
      "--- gene=305, id=175 removed by soft count = 78\n",
      "--- gene=305, id=220 removed by soft count = 106\n",
      "--- number of probes: 222 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 306\n",
      "--- gene=306, id=7 removed by soft count = 530\n",
      "--- gene=306, id=8 removed by soft count = 375\n",
      "--- gene=306, id=10 removed by soft count = 479\n",
      "--- gene=306, id=12 removed by soft count = 52\n",
      "--- gene=306, id=16 removed by soft count = 253\n",
      "--- gene=306, id=107 removed by soft count = 425\n",
      "--- gene=306, id=108 removed by soft count = 919\n",
      "--- gene=306, id=109 removed by soft count = 88\n",
      "--- gene=306, id=110 removed by soft count = 106\n",
      "--- gene=306, id=134 removed by soft count = 411\n",
      "--- number of probes: 242 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 307\n",
      "--- gene=307, id=35 removed by soft count = 82\n",
      "--- gene=307, id=37 removed by soft count = 679\n",
      "--- gene=307, id=54 removed by soft count = 100\n",
      "--- gene=307, id=55 removed by soft count = 132\n",
      "--- gene=307, id=85 removed by soft count = 163\n",
      "--- gene=307, id=86 removed by soft count = 157\n",
      "--- gene=307, id=89 removed by soft count = 290\n",
      "--- gene=307, id=91 removed by soft count = 490\n",
      "--- gene=307, id=94 removed by soft count = 53\n",
      "--- gene=307, id=95 removed by soft count = 92\n",
      "--- gene=307, id=166 removed by soft count = 188\n",
      "--- number of probes: 230 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 308\n",
      "--- gene=308, id=9 removed by soft count = 51\n",
      "--- gene=308, id=17 removed by soft count = 1093\n",
      "--- gene=308, id=18 removed by soft count = 466\n",
      "--- gene=308, id=19 removed by soft count = 108\n",
      "--- gene=308, id=20 removed by soft count = 291\n",
      "--- gene=308, id=21 removed by soft count = 73\n",
      "--- gene=308, id=24 removed by soft count = 112\n",
      "--- number of probes: 203 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 309\n",
      "--- gene=309, id=77 removed by soft count = 54\n",
      "--- number of probes: 343 , kept by blast: 342 , if remove dups: 342\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 31\n",
      "--- gene=31, id=29 removed by soft count = 351\n",
      "--- gene=31, id=131 removed by soft count = 2515\n",
      "--- gene=31, id=132 removed by soft count = 192\n",
      "--- gene=31, id=133 removed by soft count = 1415\n",
      "--- number of probes: 215 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 310\n",
      "--- gene=310, id=42 removed by soft count = 69\n",
      "--- gene=310, id=43 removed by soft count = 98\n",
      "--- gene=310, id=44 removed by soft count = 62\n",
      "--- gene=310, id=45 removed by soft count = 63\n",
      "--- gene=310, id=48 removed by soft count = 132\n",
      "--- gene=310, id=51 removed by soft count = 198\n",
      "--- gene=310, id=56 removed by soft count = 73\n",
      "--- gene=310, id=57 removed by soft count = 595\n",
      "--- gene=310, id=59 removed by soft count = 100\n",
      "--- gene=310, id=67 removed by soft count = 86\n",
      "--- gene=310, id=71 removed by soft count = 203\n",
      "--- gene=310, id=97 removed by soft count = 154\n",
      "--- gene=310, id=111 removed by soft count = 156\n",
      "--- gene=310, id=112 removed by soft count = 76\n",
      "--- number of probes: 166 , kept by blast: 152 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 311\n",
      "--- gene=311, id=51 removed by soft count = 82\n",
      "--- gene=311, id=70 removed by soft count = 157\n",
      "--- gene=311, id=74 removed by soft count = 435\n",
      "--- gene=311, id=75 removed by soft count = 112\n",
      "--- gene=311, id=158 removed by soft count = 70\n",
      "--- gene=311, id=229 removed by soft count = 230\n",
      "--- number of probes: 267 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 312\n",
      "--- gene=312, id=3 removed by soft count = 133\n",
      "--- gene=312, id=5 removed by soft count = 1439\n",
      "--- gene=312, id=6 removed by soft count = 97\n",
      "--- gene=312, id=9 removed by soft count = 404\n",
      "--- gene=312, id=82 removed by soft count = 141\n",
      "--- gene=312, id=99 removed by soft count = 81\n",
      "--- gene=312, id=119 removed by soft count = 123\n",
      "--- gene=312, id=214 removed by soft count = 51\n",
      "--- number of probes: 241 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 313\n",
      "--- gene=313, id=0 removed by soft count = 99\n",
      "--- gene=313, id=16 removed by soft count = 1808\n",
      "--- gene=313, id=18 removed by soft count = 68\n",
      "--- gene=313, id=46 removed by soft count = 192\n",
      "--- gene=313, id=49 removed by soft count = 497\n",
      "--- gene=313, id=127 removed by soft count = 180\n",
      "--- number of probes: 182 , kept by blast: 176 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 314\n",
      "--- gene=314, id=71 removed by soft count = 76\n",
      "--- number of probes: 253 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 315\n",
      "--- gene=315, id=60 removed by soft count = 211\n",
      "--- gene=315, id=85 removed by soft count = 59\n",
      "--- gene=315, id=90 removed by soft count = 59\n",
      "--- number of probes: 228 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 316\n",
      "--- gene=316, id=82 removed by soft count = 163\n",
      "--- gene=316, id=125 removed by soft count = 90\n",
      "--- gene=316, id=126 removed by soft count = 219\n",
      "--- gene=316, id=129 removed by soft count = 160\n",
      "--- number of probes: 183 , kept by blast: 179 , if remove dups: 179\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 317\n",
      "--- gene=317, id=6 removed by soft count = 56\n",
      "--- gene=317, id=94 removed by soft count = 209\n",
      "--- gene=317, id=123 removed by soft count = 962\n",
      "--- number of probes: 251 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 318\n",
      "--- gene=318, id=21 removed by soft count = 106\n",
      "--- gene=318, id=64 removed by soft count = 639\n",
      "--- gene=318, id=68 removed by soft count = 214\n",
      "--- gene=318, id=70 removed by soft count = 344\n",
      "--- gene=318, id=77 removed by soft count = 2870\n",
      "--- number of probes: 178 , kept by blast: 173 , if remove dups: 173\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 319\n",
      "--- gene=319, id=13 removed by soft count = 486\n",
      "--- gene=319, id=14 removed by soft count = 62\n",
      "--- gene=319, id=121 removed by soft count = 74\n",
      "--- gene=319, id=122 removed by soft count = 895\n",
      "--- number of probes: 249 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 32\n",
      "--- gene=32, id=78 removed by soft count = 767\n",
      "--- gene=32, id=79 removed by soft count = 813\n",
      "--- gene=32, id=80 removed by soft count = 571\n",
      "--- gene=32, id=81 removed by soft count = 603\n",
      "--- gene=32, id=85 removed by soft count = 61\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=32, id=135 removed by soft count = 85\n",
      "--- gene=32, id=168 removed by soft count = 209\n",
      "--- gene=32, id=169 removed by soft count = 181\n",
      "--- number of probes: 190 , kept by blast: 182 , if remove dups: 182\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 320\n",
      "--- gene=320, id=22 removed by soft count = 421\n",
      "--- gene=320, id=27 removed by soft count = 69\n",
      "--- gene=320, id=34 removed by soft count = 244\n",
      "--- gene=320, id=36 removed by soft count = 284\n",
      "--- gene=320, id=44 removed by soft count = 248\n",
      "--- gene=320, id=118 removed by soft count = 415\n",
      "--- gene=320, id=134 removed by soft count = 62\n",
      "--- gene=320, id=135 removed by soft count = 338\n",
      "--- gene=320, id=136 removed by soft count = 651\n",
      "--- gene=320, id=137 removed by soft count = 122\n",
      "--- gene=320, id=138 removed by soft count = 206\n",
      "--- gene=320, id=149 removed by soft count = 1462\n",
      "--- gene=320, id=151 removed by soft count = 188\n",
      "--- gene=320, id=153 removed by soft count = 412\n",
      "--- gene=320, id=154 removed by soft count = 443\n",
      "--- number of probes: 155 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 321\n",
      "--- gene=321, id=45 removed by soft count = 51\n",
      "--- gene=321, id=235 removed by soft count = 54\n",
      "--- number of probes: 279 , kept by blast: 277 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 322\n",
      "--- gene=322, id=32 removed by soft count = 158\n",
      "--- gene=322, id=124 removed by soft count = 202\n",
      "--- gene=322, id=166 removed by soft count = 90\n",
      "--- gene=322, id=214 removed by soft count = 6828\n",
      "--- gene=322, id=255 removed by soft count = 67\n",
      "--- number of probes: 258 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 323\n",
      "--- gene=323, id=0 removed by soft count = 299\n",
      "--- gene=323, id=95 removed by soft count = 131\n",
      "--- gene=323, id=238 removed by soft count = 53\n",
      "--- number of probes: 248 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 324\n",
      "--- gene=324, id=5 removed by soft count = 111\n",
      "--- gene=324, id=6 removed by soft count = 52\n",
      "--- gene=324, id=58 removed by soft count = 208\n",
      "--- gene=324, id=66 removed by soft count = 440\n",
      "--- gene=324, id=84 removed by soft count = 76\n",
      "--- gene=324, id=91 removed by soft count = 202\n",
      "--- gene=324, id=92 removed by soft count = 70\n",
      "--- gene=324, id=94 removed by soft count = 78\n",
      "--- gene=324, id=98 removed by soft count = 493\n",
      "--- gene=324, id=99 removed by soft count = 74\n",
      "--- gene=324, id=105 removed by soft count = 716\n",
      "--- number of probes: 171 , kept by blast: 160 , if remove dups: 160\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 325\n",
      "--- gene=325, id=1 removed by soft count = 75\n",
      "--- gene=325, id=3 removed by soft count = 67\n",
      "--- gene=325, id=11 removed by soft count = 60\n",
      "--- gene=325, id=12 removed by soft count = 51\n",
      "--- gene=325, id=14 removed by soft count = 160\n",
      "--- gene=325, id=16 removed by soft count = 96\n",
      "--- gene=325, id=35 removed by soft count = 64\n",
      "--- gene=325, id=40 removed by soft count = 909\n",
      "--- gene=325, id=61 removed by soft count = 355\n",
      "--- gene=325, id=65 removed by soft count = 54\n",
      "--- gene=325, id=76 removed by soft count = 53\n",
      "--- gene=325, id=89 removed by soft count = 64\n",
      "--- gene=325, id=112 removed by soft count = 69\n",
      "--- number of probes: 176 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 326\n",
      "--- gene=326, id=45 removed by soft count = 1811\n",
      "--- gene=326, id=70 removed by soft count = 80\n",
      "--- gene=326, id=94 removed by soft count = 376\n",
      "--- gene=326, id=95 removed by soft count = 617\n",
      "--- gene=326, id=96 removed by soft count = 281\n",
      "--- number of probes: 191 , kept by blast: 186 , if remove dups: 186\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 327\n",
      "--- gene=327, id=26 removed by soft count = 191\n",
      "--- gene=327, id=48 removed by soft count = 188\n",
      "--- gene=327, id=52 removed by soft count = 76\n",
      "--- gene=327, id=53 removed by soft count = 86\n",
      "--- gene=327, id=57 removed by soft count = 265\n",
      "--- gene=327, id=63 removed by soft count = 228\n",
      "--- gene=327, id=65 removed by soft count = 676\n",
      "--- gene=327, id=67 removed by soft count = 235\n",
      "--- gene=327, id=69 removed by soft count = 495\n",
      "--- gene=327, id=70 removed by soft count = 1985\n",
      "--- gene=327, id=71 removed by soft count = 129\n",
      "--- gene=327, id=73 removed by soft count = 76\n",
      "--- gene=327, id=74 removed by soft count = 61\n",
      "--- gene=327, id=75 removed by soft count = 74\n",
      "--- gene=327, id=78 removed by soft count = 56\n",
      "--- number of probes: 106 , kept by blast: 91 , if remove dups: 91\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 91\n",
      "-- checking probes in region: 328\n",
      "--- gene=328, id=11 removed by soft count = 1216\n",
      "--- gene=328, id=14 removed by soft count = 233\n",
      "--- gene=328, id=15 removed by soft count = 781\n",
      "--- gene=328, id=16 removed by soft count = 1380\n",
      "--- gene=328, id=23 removed by soft count = 258\n",
      "--- gene=328, id=28 removed by soft count = 139\n",
      "--- gene=328, id=29 removed by soft count = 105\n",
      "--- gene=328, id=30 removed by soft count = 62\n",
      "--- gene=328, id=32 removed by soft count = 170\n",
      "--- gene=328, id=33 removed by soft count = 59\n",
      "--- gene=328, id=34 removed by soft count = 125\n",
      "--- gene=328, id=36 removed by soft count = 77\n",
      "--- gene=328, id=37 removed by soft count = 325\n",
      "--- gene=328, id=46 removed by soft count = 74\n",
      "--- gene=328, id=55 removed by soft count = 68\n",
      "--- gene=328, id=104 removed by soft count = 247\n",
      "--- gene=328, id=105 removed by soft count = 181\n",
      "--- gene=328, id=106 removed by soft count = 60\n",
      "--- gene=328, id=107 removed by soft count = 606\n",
      "--- gene=328, id=114 removed by soft count = 168\n",
      "--- gene=328, id=155 removed by soft count = 220\n",
      "--- gene=328, id=160 removed by soft count = 175\n",
      "--- gene=328, id=161 removed by soft count = 52\n",
      "--- gene=328, id=162 removed by soft count = 151\n",
      "--- gene=328, id=165 removed by soft count = 162\n",
      "--- gene=328, id=166 removed by soft count = 58\n",
      "--- number of probes: 170 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 329\n",
      "--- gene=329, id=254 removed by soft count = 420\n",
      "--- number of probes: 295 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 33\n",
      "--- gene=33, id=34 removed by soft count = 408\n",
      "--- gene=33, id=35 removed by soft count = 371\n",
      "--- gene=33, id=255 removed by soft count = 369\n",
      "--- gene=33, id=260 removed by soft count = 94\n",
      "--- number of probes: 338 , kept by blast: 334 , if remove dups: 334\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 330\n",
      "--- gene=330, id=52 removed by soft count = 323\n",
      "--- gene=330, id=93 removed by soft count = 84\n",
      "--- gene=330, id=112 removed by soft count = 105\n",
      "--- gene=330, id=147 removed by soft count = 183\n",
      "--- gene=330, id=148 removed by soft count = 113\n",
      "--- gene=330, id=149 removed by soft count = 283\n",
      "--- gene=330, id=151 removed by soft count = 74\n",
      "--- gene=330, id=152 removed by soft count = 63\n",
      "--- gene=330, id=157 removed by soft count = 59\n",
      "--- gene=330, id=159 removed by soft count = 59\n",
      "--- gene=330, id=161 removed by soft count = 98\n",
      "--- gene=330, id=203 removed by soft count = 118\n",
      "--- gene=330, id=204 removed by soft count = 57\n",
      "--- gene=330, id=205 removed by soft count = 139\n",
      "--- gene=330, id=209 removed by soft count = 242\n",
      "--- gene=330, id=211 removed by soft count = 179\n",
      "--- gene=330, id=212 removed by soft count = 299\n",
      "--- gene=330, id=214 removed by soft count = 472\n",
      "--- gene=330, id=215 removed by soft count = 185\n",
      "--- gene=330, id=216 removed by soft count = 123\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=330, id=217 removed by soft count = 191\n",
      "--- number of probes: 218 , kept by blast: 197 , if remove dups: 197\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 331\n",
      "--- gene=331, id=0 removed by soft count = 114\n",
      "--- gene=331, id=1 removed by soft count = 448\n",
      "--- gene=331, id=30 removed by soft count = 166\n",
      "--- gene=331, id=31 removed by soft count = 167\n",
      "--- gene=331, id=72 removed by soft count = 230\n",
      "--- gene=331, id=73 removed by soft count = 128\n",
      "--- gene=331, id=92 removed by soft count = 64\n",
      "--- gene=331, id=93 removed by soft count = 308\n",
      "--- gene=331, id=94 removed by soft count = 109\n",
      "--- gene=331, id=97 removed by soft count = 134\n",
      "--- gene=331, id=98 removed by soft count = 291\n",
      "--- gene=331, id=99 removed by soft count = 208\n",
      "--- gene=331, id=100 removed by soft count = 864\n",
      "--- gene=331, id=101 removed by soft count = 890\n",
      "--- gene=331, id=125 removed by soft count = 75\n",
      "--- gene=331, id=146 removed by soft count = 330\n",
      "--- number of probes: 154 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 332\n",
      "--- number of probes: 377 , kept by blast: 377 , if remove dups: 377\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 333\n",
      "--- gene=333, id=38 removed by soft count = 458\n",
      "--- gene=333, id=207 removed by soft count = 139\n",
      "--- gene=333, id=209 removed by soft count = 65\n",
      "--- gene=333, id=210 removed by soft count = 106\n",
      "--- gene=333, id=212 removed by soft count = 93\n",
      "--- number of probes: 214 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 334\n",
      "--- gene=334, id=7 removed by soft count = 61\n",
      "--- gene=334, id=43 removed by soft count = 1283\n",
      "--- gene=334, id=46 removed by soft count = 90\n",
      "--- gene=334, id=48 removed by soft count = 32158\n",
      "--- gene=334, id=131 removed by soft count = 217\n",
      "--- gene=334, id=173 removed by soft count = 72\n",
      "--- gene=334, id=229 removed by soft count = 157\n",
      "--- number of probes: 234 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 335\n",
      "--- gene=335, id=2 removed by soft count = 54\n",
      "--- gene=335, id=3 removed by soft count = 51\n",
      "--- gene=335, id=5 removed by soft count = 265\n",
      "--- gene=335, id=63 removed by soft count = 53\n",
      "--- gene=335, id=64 removed by soft count = 104\n",
      "--- gene=335, id=77 removed by soft count = 81\n",
      "--- gene=335, id=81 removed by soft count = 55\n",
      "--- gene=335, id=84 removed by soft count = 177\n",
      "--- gene=335, id=149 removed by soft count = 58\n",
      "--- gene=335, id=189 removed by soft count = 75\n",
      "--- number of probes: 271 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 336\n",
      "--- gene=336, id=4 removed by soft count = 58\n",
      "--- gene=336, id=10 removed by soft count = 56\n",
      "--- gene=336, id=26 removed by soft count = 2547\n",
      "--- gene=336, id=73 removed by soft count = 1613\n",
      "--- gene=336, id=75 removed by soft count = 493\n",
      "--- gene=336, id=76 removed by soft count = 178\n",
      "--- gene=336, id=147 removed by soft count = 92\n",
      "--- gene=336, id=219 removed by soft count = 53\n",
      "--- gene=336, id=222 removed by soft count = 102\n",
      "--- gene=336, id=223 removed by soft count = 73\n",
      "--- number of probes: 244 , kept by blast: 234 , if remove dups: 234\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 337\n",
      "--- gene=337, id=50 removed by soft count = 133\n",
      "--- gene=337, id=60 removed by soft count = 97\n",
      "--- gene=337, id=68 removed by soft count = 79\n",
      "--- gene=337, id=71 removed by soft count = 61\n",
      "--- gene=337, id=105 removed by soft count = 63\n",
      "--- gene=337, id=109 removed by soft count = 329\n",
      "--- gene=337, id=148 removed by soft count = 442\n",
      "--- number of probes: 181 , kept by blast: 174 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 338\n",
      "--- gene=338, id=52 removed by soft count = 73\n",
      "--- gene=338, id=53 removed by soft count = 369\n",
      "--- gene=338, id=75 removed by soft count = 59\n",
      "--- gene=338, id=77 removed by soft count = 110\n",
      "--- gene=338, id=79 removed by soft count = 64\n",
      "--- gene=338, id=100 removed by soft count = 55\n",
      "--- gene=338, id=107 removed by soft count = 58\n",
      "--- gene=338, id=108 removed by soft count = 68\n",
      "--- gene=338, id=109 removed by soft count = 63\n",
      "--- gene=338, id=111 removed by soft count = 267\n",
      "--- gene=338, id=112 removed by soft count = 89\n",
      "--- gene=338, id=128 removed by soft count = 209\n",
      "--- gene=338, id=134 removed by soft count = 57\n",
      "--- gene=338, id=139 removed by soft count = 54\n",
      "--- gene=338, id=156 removed by soft count = 68\n",
      "--- gene=338, id=175 removed by soft count = 115\n",
      "--- number of probes: 222 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 339\n",
      "--- gene=339, id=7 removed by soft count = 155\n",
      "--- gene=339, id=63 removed by soft count = 2802\n",
      "--- gene=339, id=96 removed by soft count = 207\n",
      "--- gene=339, id=100 removed by soft count = 57\n",
      "--- gene=339, id=116 removed by soft count = 88\n",
      "--- gene=339, id=117 removed by soft count = 71\n",
      "--- number of probes: 257 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 34\n",
      "--- gene=34, id=34 removed by soft count = 66\n",
      "--- gene=34, id=109 removed by soft count = 62\n",
      "--- number of probes: 230 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 340\n",
      "--- gene=340, id=7 removed by soft count = 160\n",
      "--- number of probes: 201 , kept by blast: 200 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 341\n",
      "--- gene=341, id=45 removed by soft count = 1041\n",
      "--- gene=341, id=104 removed by soft count = 294\n",
      "--- number of probes: 284 , kept by blast: 282 , if remove dups: 281\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 342\n",
      "--- gene=342, id=26 removed by soft count = 118\n",
      "--- gene=342, id=63 removed by soft count = 53\n",
      "--- gene=342, id=64 removed by soft count = 93\n",
      "--- gene=342, id=75 removed by soft count = 323\n",
      "--- gene=342, id=77 removed by soft count = 448\n",
      "--- gene=342, id=78 removed by soft count = 54\n",
      "--- gene=342, id=79 removed by soft count = 618\n",
      "--- gene=342, id=85 removed by soft count = 93\n",
      "--- gene=342, id=86 removed by soft count = 582\n",
      "--- gene=342, id=88 removed by soft count = 169\n",
      "--- gene=342, id=139 removed by soft count = 53\n",
      "--- gene=342, id=141 removed by soft count = 53\n",
      "--- gene=342, id=142 removed by soft count = 154\n",
      "--- gene=342, id=143 removed by soft count = 93\n",
      "--- number of probes: 157 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 343\n",
      "--- gene=343, id=54 removed by soft count = 73\n",
      "--- gene=343, id=56 removed by soft count = 369\n",
      "--- number of probes: 260 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 344\n",
      "--- gene=344, id=82 removed by soft count = 92\n",
      "--- gene=344, id=148 removed by soft count = 97\n",
      "--- gene=344, id=149 removed by soft count = 819\n",
      "--- gene=344, id=151 removed by soft count = 52\n",
      "--- gene=344, id=152 removed by soft count = 81\n",
      "--- number of probes: 234 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 345\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=345, id=50 removed by soft count = 60\n",
      "--- gene=345, id=53 removed by soft count = 538\n",
      "--- number of probes: 225 , kept by blast: 223 , if remove dups: 223\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 346\n",
      "--- gene=346, id=35 removed by soft count = 54\n",
      "--- gene=346, id=36 removed by soft count = 51\n",
      "--- number of probes: 273 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 347\n",
      "--- gene=347, id=12 removed by soft count = 255\n",
      "--- gene=347, id=58 removed by soft count = 190\n",
      "--- gene=347, id=89 removed by soft count = 226\n",
      "--- gene=347, id=118 removed by soft count = 241\n",
      "--- gene=347, id=164 removed by soft count = 56\n",
      "--- number of probes: 202 , kept by blast: 197 , if remove dups: 197\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 348\n",
      "--- gene=348, id=61 removed by soft count = 431\n",
      "--- gene=348, id=62 removed by soft count = 179\n",
      "--- gene=348, id=63 removed by soft count = 138\n",
      "--- gene=348, id=64 removed by soft count = 657\n",
      "--- gene=348, id=67 removed by soft count = 92\n",
      "--- gene=348, id=72 removed by soft count = 64\n",
      "--- gene=348, id=75 removed by soft count = 115\n",
      "--- gene=348, id=181 removed by soft count = 1799\n",
      "--- gene=348, id=204 removed by soft count = 446\n",
      "--- gene=348, id=208 removed by soft count = 546\n",
      "--- gene=348, id=209 removed by soft count = 71\n",
      "--- gene=348, id=211 removed by soft count = 57\n",
      "--- number of probes: 223 , kept by blast: 211 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 349\n",
      "--- gene=349, id=0 removed by soft count = 70\n",
      "--- gene=349, id=88 removed by soft count = 782\n",
      "--- gene=349, id=115 removed by soft count = 79\n",
      "--- gene=349, id=146 removed by soft count = 190\n",
      "--- gene=349, id=157 removed by soft count = 352\n",
      "--- number of probes: 161 , kept by blast: 156 , if remove dups: 155\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 35\n",
      "--- gene=35, id=6 removed by soft count = 105\n",
      "--- gene=35, id=74 removed by soft count = 245\n",
      "--- gene=35, id=75 removed by soft count = 135\n",
      "--- gene=35, id=77 removed by soft count = 302\n",
      "--- number of probes: 203 , kept by blast: 199 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 350\n",
      "--- gene=350, id=97 removed by hard count = 7\n",
      "--- gene=350, id=202 removed by soft count = 53\n",
      "--- number of probes: 388 , kept by blast: 386 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 351\n",
      "--- gene=351, id=143 removed by soft count = 146\n",
      "--- gene=351, id=144 removed by soft count = 89\n",
      "--- gene=351, id=146 removed by soft count = 295\n",
      "--- gene=351, id=147 removed by soft count = 245\n",
      "--- gene=351, id=148 removed by soft count = 131\n",
      "--- gene=351, id=149 removed by soft count = 759\n",
      "--- gene=351, id=251 removed by soft count = 115523\n",
      "--- number of probes: 313 , kept by blast: 306 , if remove dups: 306\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 352\n",
      "--- gene=352, id=40 removed by soft count = 247\n",
      "--- gene=352, id=124 removed by soft count = 149\n",
      "--- gene=352, id=127 removed by soft count = 72\n",
      "--- gene=352, id=128 removed by soft count = 191\n",
      "--- gene=352, id=288 removed by soft count = 70\n",
      "--- number of probes: 325 , kept by blast: 320 , if remove dups: 320\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 353\n",
      "--- gene=353, id=14 removed by soft count = 200\n",
      "--- gene=353, id=15 removed by soft count = 113\n",
      "--- gene=353, id=16 removed by soft count = 378\n",
      "--- gene=353, id=18 removed by soft count = 608\n",
      "--- gene=353, id=30 removed by soft count = 84\n",
      "--- gene=353, id=31 removed by soft count = 175\n",
      "--- gene=353, id=71 removed by soft count = 471\n",
      "--- gene=353, id=84 removed by soft count = 554\n",
      "--- gene=353, id=87 removed by soft count = 115\n",
      "--- gene=353, id=89 removed by soft count = 160\n",
      "--- gene=353, id=102 removed by soft count = 403\n",
      "--- gene=353, id=143 removed by soft count = 1923\n",
      "--- gene=353, id=144 removed by soft count = 206\n",
      "--- gene=353, id=145 removed by soft count = 3282\n",
      "--- gene=353, id=146 removed by soft count = 293\n",
      "--- gene=353, id=148 removed by soft count = 1283\n",
      "--- gene=353, id=156 removed by soft count = 261\n",
      "--- number of probes: 158 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 354\n",
      "--- gene=354, id=5 removed by soft count = 94\n",
      "--- gene=354, id=52 removed by soft count = 133\n",
      "--- gene=354, id=94 removed by soft count = 60\n",
      "--- gene=354, id=174 removed by soft count = 756\n",
      "--- gene=354, id=210 removed by soft count = 53\n",
      "--- gene=354, id=218 removed by soft count = 852\n",
      "--- number of probes: 293 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 355\n",
      "--- gene=355, id=167 removed by soft count = 165\n",
      "--- number of probes: 234 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 356\n",
      "--- gene=356, id=47 removed by soft count = 82\n",
      "--- gene=356, id=48 removed by soft count = 53\n",
      "--- gene=356, id=67 removed by soft count = 931\n",
      "--- gene=356, id=79 removed by soft count = 195\n",
      "--- gene=356, id=80 removed by soft count = 152\n",
      "--- gene=356, id=155 removed by soft count = 355\n",
      "--- gene=356, id=156 removed by soft count = 295\n",
      "--- gene=356, id=157 removed by soft count = 208\n",
      "--- gene=356, id=158 removed by soft count = 286\n",
      "--- number of probes: 257 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 357\n",
      "--- gene=357, id=49 removed by soft count = 303\n",
      "--- gene=357, id=56 removed by soft count = 95\n",
      "--- gene=357, id=59 removed by soft count = 82\n",
      "--- gene=357, id=60 removed by soft count = 197\n",
      "--- gene=357, id=61 removed by soft count = 64\n",
      "--- gene=357, id=67 removed by soft count = 56\n",
      "--- gene=357, id=70 removed by soft count = 112\n",
      "--- gene=357, id=84 removed by soft count = 69\n",
      "--- gene=357, id=88 removed by soft count = 118\n",
      "--- gene=357, id=92 removed by soft count = 246\n",
      "--- gene=357, id=93 removed by soft count = 187\n",
      "--- gene=357, id=94 removed by soft count = 333\n",
      "--- gene=357, id=95 removed by soft count = 136\n",
      "--- gene=357, id=96 removed by soft count = 295\n",
      "--- gene=357, id=97 removed by soft count = 255\n",
      "--- gene=357, id=98 removed by soft count = 263\n",
      "--- gene=357, id=99 removed by soft count = 332\n",
      "--- gene=357, id=100 removed by soft count = 344\n",
      "--- gene=357, id=101 removed by soft count = 581\n",
      "--- gene=357, id=121 removed by soft count = 108\n",
      "--- gene=357, id=122 removed by soft count = 70\n",
      "--- gene=357, id=123 removed by soft count = 191\n",
      "--- gene=357, id=143 removed by soft count = 178\n",
      "--- number of probes: 170 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 358\n",
      "--- gene=358, id=80 removed by soft count = 369\n",
      "--- gene=358, id=81 removed by soft count = 96\n",
      "--- gene=358, id=84 removed by soft count = 217\n",
      "--- gene=358, id=88 removed by soft count = 227\n",
      "--- gene=358, id=92 removed by soft count = 63\n",
      "--- gene=358, id=107 removed by soft count = 64\n",
      "--- gene=358, id=113 removed by soft count = 406\n",
      "--- gene=358, id=114 removed by soft count = 290\n",
      "--- gene=358, id=115 removed by soft count = 108\n",
      "--- gene=358, id=190 removed by soft count = 199\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=358, id=195 removed by soft count = 6111\n",
      "--- number of probes: 216 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 359\n",
      "--- gene=359, id=75 removed by soft count = 53\n",
      "--- gene=359, id=77 removed by soft count = 158\n",
      "--- gene=359, id=78 removed by soft count = 65\n",
      "--- gene=359, id=81 removed by soft count = 452\n",
      "--- gene=359, id=84 removed by soft count = 1135\n",
      "--- gene=359, id=85 removed by soft count = 1095\n",
      "--- gene=359, id=164 removed by soft count = 243\n",
      "--- number of probes: 179 , kept by blast: 172 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 36\n",
      "--- gene=36, id=5 removed by soft count = 75\n",
      "--- gene=36, id=6 removed by soft count = 292\n",
      "--- gene=36, id=108 removed by soft count = 68\n",
      "--- gene=36, id=109 removed by soft count = 126\n",
      "--- gene=36, id=110 removed by soft count = 94\n",
      "--- gene=36, id=115 removed by soft count = 103\n",
      "--- gene=36, id=117 removed by soft count = 58\n",
      "--- gene=36, id=120 removed by soft count = 116\n",
      "--- gene=36, id=121 removed by soft count = 245\n",
      "--- gene=36, id=124 removed by soft count = 240\n",
      "--- gene=36, id=125 removed by soft count = 1218\n",
      "--- gene=36, id=126 removed by soft count = 990\n",
      "--- gene=36, id=127 removed by soft count = 3009\n",
      "--- gene=36, id=128 removed by soft count = 868\n",
      "--- gene=36, id=135 removed by soft count = 63\n",
      "--- number of probes: 154 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 360\n",
      "--- gene=360, id=3 removed by hard count = 0\n",
      "--- gene=360, id=5 removed by hard count = 0\n",
      "--- gene=360, id=131 removed by soft count = 85\n",
      "--- gene=360, id=132 removed by soft count = 502\n",
      "--- gene=360, id=133 removed by soft count = 260\n",
      "--- gene=360, id=168 removed by soft count = 442\n",
      "--- gene=360, id=170 removed by soft count = 140\n",
      "--- gene=360, id=171 removed by soft count = 567\n",
      "--- gene=360, id=174 removed by soft count = 394\n",
      "--- gene=360, id=175 removed by soft count = 562\n",
      "--- gene=360, id=182 removed by soft count = 61\n",
      "--- gene=360, id=184 removed by soft count = 107\n",
      "--- number of probes: 187 , kept by blast: 175 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 361\n",
      "--- gene=361, id=95 removed by soft count = 287\n",
      "--- gene=361, id=96 removed by soft count = 93\n",
      "--- gene=361, id=97 removed by soft count = 193\n",
      "--- gene=361, id=163 removed by soft count = 65\n",
      "--- gene=361, id=164 removed by soft count = 51\n",
      "--- number of probes: 175 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 362\n",
      "--- gene=362, id=219 removed by soft count = 107\n",
      "--- gene=362, id=304 removed by soft count = 51\n",
      "--- number of probes: 315 , kept by blast: 313 , if remove dups: 313\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 363\n",
      "--- gene=363, id=112 removed by soft count = 775\n",
      "--- gene=363, id=254 removed by soft count = 71\n",
      "--- gene=363, id=327 removed by soft count = 248\n",
      "--- gene=363, id=330 removed by soft count = 60\n",
      "--- number of probes: 350 , kept by blast: 346 , if remove dups: 346\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 364\n",
      "--- gene=364, id=120 removed by soft count = 168\n",
      "--- gene=364, id=121 removed by soft count = 442\n",
      "--- number of probes: 253 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 365\n",
      "--- gene=365, id=273 removed by soft count = 116\n",
      "--- number of probes: 284 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 366\n",
      "--- gene=366, id=3 removed by soft count = 60\n",
      "--- gene=366, id=66 removed by soft count = 97\n",
      "--- gene=366, id=106 removed by soft count = 70\n",
      "--- gene=366, id=108 removed by soft count = 83\n",
      "--- gene=366, id=146 removed by soft count = 887\n",
      "--- number of probes: 214 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 367\n",
      "--- gene=367, id=56 removed by soft count = 19588\n",
      "--- gene=367, id=57 removed by soft count = 379\n",
      "--- gene=367, id=65 removed by soft count = 1348\n",
      "--- gene=367, id=66 removed by soft count = 1317\n",
      "--- gene=367, id=67 removed by soft count = 359\n",
      "--- gene=367, id=68 removed by soft count = 444\n",
      "--- gene=367, id=69 removed by soft count = 189\n",
      "--- gene=367, id=71 removed by soft count = 130\n",
      "--- gene=367, id=72 removed by soft count = 152\n",
      "--- gene=367, id=82 removed by soft count = 93\n",
      "--- gene=367, id=84 removed by soft count = 93\n",
      "--- gene=367, id=85 removed by soft count = 154\n",
      "--- gene=367, id=86 removed by soft count = 53\n",
      "--- gene=367, id=87 removed by soft count = 281\n",
      "--- gene=367, id=88 removed by soft count = 79\n",
      "--- gene=367, id=91 removed by soft count = 125\n",
      "--- gene=367, id=93 removed by soft count = 97\n",
      "--- gene=367, id=94 removed by soft count = 62\n",
      "--- gene=367, id=99 removed by soft count = 78\n",
      "--- number of probes: 102 , kept by blast: 83 , if remove dups: 83\n",
      "-- number of probes kept for this region: 83\n",
      "-- checking probes in region: 368\n",
      "--- gene=368, id=1 removed by soft count = 1163\n",
      "--- gene=368, id=2 removed by soft count = 76\n",
      "--- gene=368, id=3 removed by soft count = 612\n",
      "--- gene=368, id=4 removed by soft count = 1258\n",
      "--- gene=368, id=5 removed by soft count = 279\n",
      "--- gene=368, id=6 removed by soft count = 84\n",
      "--- gene=368, id=7 removed by soft count = 473\n",
      "--- gene=368, id=8 removed by soft count = 158\n",
      "--- gene=368, id=9 removed by soft count = 257\n",
      "--- gene=368, id=11 removed by soft count = 888\n",
      "--- gene=368, id=13 removed by soft count = 61\n",
      "--- gene=368, id=14 removed by soft count = 9082\n",
      "--- gene=368, id=16 removed by soft count = 354\n",
      "--- gene=368, id=17 removed by soft count = 75\n",
      "--- gene=368, id=18 removed by soft count = 154\n",
      "--- gene=368, id=22 removed by soft count = 68\n",
      "--- gene=368, id=23 removed by soft count = 57\n",
      "--- gene=368, id=25 removed by soft count = 65\n",
      "--- gene=368, id=27 removed by soft count = 288\n",
      "--- gene=368, id=29 removed by soft count = 493\n",
      "--- gene=368, id=30 removed by soft count = 157\n",
      "--- gene=368, id=33 removed by soft count = 68\n",
      "--- gene=368, id=35 removed by soft count = 103\n",
      "--- gene=368, id=40 removed by soft count = 896\n",
      "--- gene=368, id=43 removed by soft count = 365\n",
      "--- gene=368, id=44 removed by soft count = 948\n",
      "--- gene=368, id=50 removed by soft count = 66\n",
      "--- gene=368, id=56 removed by soft count = 187\n",
      "--- gene=368, id=58 removed by soft count = 197\n",
      "--- gene=368, id=59 removed by soft count = 396\n",
      "--- gene=368, id=60 removed by soft count = 1065\n",
      "--- gene=368, id=61 removed by soft count = 167\n",
      "--- gene=368, id=62 removed by soft count = 12348\n",
      "--- gene=368, id=63 removed by soft count = 469\n",
      "--- number of probes: 64 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: 37\n",
      "--- gene=37, id=23 removed by soft count = 384\n",
      "--- gene=37, id=232 removed by soft count = 71\n",
      "--- number of probes: 358 , kept by blast: 356 , if remove dups: 356\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 370\n",
      "--- gene=370, id=152 removed by soft count = 436\n",
      "--- number of probes: 158 , kept by blast: 157 , if remove dups: 157\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 371\n",
      "--- gene=371, id=3 removed by soft count = 5476\n",
      "--- gene=371, id=10 removed by soft count = 56\n",
      "--- gene=371, id=11 removed by soft count = 63\n",
      "--- gene=371, id=12 removed by soft count = 526\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=371, id=13 removed by soft count = 724\n",
      "--- gene=371, id=14 removed by soft count = 675\n",
      "--- gene=371, id=27 removed by soft count = 93\n",
      "--- gene=371, id=42 removed by soft count = 386\n",
      "--- gene=371, id=43 removed by soft count = 2481\n",
      "--- gene=371, id=52 removed by soft count = 126\n",
      "--- number of probes: 84 , kept by blast: 74 , if remove dups: 74\n",
      "-- number of probes kept for this region: 74\n",
      "-- checking probes in region: 372\n",
      "--- gene=372, id=4 removed by soft count = 80\n",
      "--- gene=372, id=11 removed by soft count = 107\n",
      "--- gene=372, id=13 removed by soft count = 77\n",
      "--- gene=372, id=14 removed by soft count = 79\n",
      "--- gene=372, id=17 removed by soft count = 87\n",
      "--- gene=372, id=20 removed by soft count = 57\n",
      "--- gene=372, id=29 removed by soft count = 437\n",
      "--- gene=372, id=30 removed by soft count = 328\n",
      "--- gene=372, id=84 removed by soft count = 575\n",
      "--- gene=372, id=181 removed by soft count = 129\n",
      "--- number of probes: 236 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 373\n",
      "--- gene=373, id=66 removed by soft count = 69\n",
      "--- number of probes: 270 , kept by blast: 269 , if remove dups: 269\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 374\n",
      "--- gene=374, id=73 removed by soft count = 1101\n",
      "--- gene=374, id=74 removed by soft count = 1226\n",
      "--- gene=374, id=179 removed by soft count = 111\n",
      "--- gene=374, id=200 removed by soft count = 61\n",
      "--- number of probes: 224 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 375\n",
      "--- gene=375, id=37 removed by soft count = 66\n",
      "--- gene=375, id=38 removed by soft count = 70\n",
      "--- gene=375, id=84 removed by soft count = 57\n",
      "--- gene=375, id=88 removed by soft count = 68\n",
      "--- gene=375, id=89 removed by soft count = 115\n",
      "--- gene=375, id=138 removed by soft count = 60\n",
      "--- gene=375, id=139 removed by soft count = 127\n",
      "--- gene=375, id=140 removed by soft count = 84\n",
      "--- number of probes: 226 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 376\n",
      "--- gene=376, id=4 removed by soft count = 127\n",
      "--- gene=376, id=138 removed by soft count = 151\n",
      "--- number of probes: 222 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 377\n",
      "--- number of probes: 340 , kept by blast: 340 , if remove dups: 340\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 379\n",
      "--- gene=379, id=48 removed by soft count = 4068\n",
      "--- gene=379, id=179 removed by soft count = 4153\n",
      "--- gene=379, id=188 removed by soft count = 426\n",
      "--- gene=379, id=231 removed by soft count = 51\n",
      "--- number of probes: 287 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 38\n",
      "--- gene=38, id=151 removed by soft count = 972\n",
      "--- gene=38, id=174 removed by soft count = 95\n",
      "--- number of probes: 211 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 380\n",
      "--- gene=380, id=112 removed by soft count = 63\n",
      "--- gene=380, id=222 removed by soft count = 1715\n",
      "--- gene=380, id=223 removed by soft count = 316\n",
      "--- number of probes: 224 , kept by blast: 221 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 381\n",
      "--- gene=381, id=5 removed by soft count = 101\n",
      "--- gene=381, id=266 removed by soft count = 114\n",
      "--- gene=381, id=267 removed by soft count = 66\n",
      "--- gene=381, id=268 removed by soft count = 63\n",
      "--- number of probes: 278 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 382\n",
      "--- gene=382, id=288 removed by soft count = 182\n",
      "--- gene=382, id=290 removed by soft count = 59\n",
      "--- gene=382, id=299 removed by soft count = 71\n",
      "--- number of probes: 318 , kept by blast: 315 , if remove dups: 315\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 383\n",
      "--- gene=383, id=0 removed by soft count = 308\n",
      "--- gene=383, id=1 removed by soft count = 225\n",
      "--- gene=383, id=2 removed by soft count = 175\n",
      "--- gene=383, id=3 removed by soft count = 70\n",
      "--- gene=383, id=4 removed by soft count = 58\n",
      "--- gene=383, id=83 removed by soft count = 177\n",
      "--- gene=383, id=91 removed by soft count = 258\n",
      "--- gene=383, id=120 removed by soft count = 206\n",
      "--- gene=383, id=121 removed by soft count = 176\n",
      "--- gene=383, id=122 removed by soft count = 718\n",
      "--- number of probes: 176 , kept by blast: 166 , if remove dups: 166\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 384\n",
      "--- gene=384, id=72 removed by soft count = 1291\n",
      "--- gene=384, id=73 removed by soft count = 83\n",
      "--- gene=384, id=76 removed by soft count = 55\n",
      "--- gene=384, id=145 removed by soft count = 56\n",
      "--- gene=384, id=148 removed by soft count = 53\n",
      "--- gene=384, id=151 removed by soft count = 66\n",
      "--- gene=384, id=161 removed by soft count = 68\n",
      "--- gene=384, id=162 removed by soft count = 55\n",
      "--- number of probes: 166 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 385\n",
      "--- number of probes: 272 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 386\n",
      "--- gene=386, id=10 removed by soft count = 52\n",
      "--- gene=386, id=13 removed by soft count = 52\n",
      "--- gene=386, id=16 removed by soft count = 51\n",
      "--- gene=386, id=21 removed by soft count = 160\n",
      "--- gene=386, id=125 removed by soft count = 1076\n",
      "--- gene=386, id=201 removed by soft count = 300\n",
      "--- number of probes: 215 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 387\n",
      "--- gene=387, id=24 removed by soft count = 266\n",
      "--- gene=387, id=186 removed by soft count = 124\n",
      "--- gene=387, id=187 removed by soft count = 230\n",
      "--- gene=387, id=188 removed by soft count = 473\n",
      "--- gene=387, id=189 removed by soft count = 191\n",
      "--- gene=387, id=190 removed by soft count = 358\n",
      "--- gene=387, id=191 removed by soft count = 1884\n",
      "--- number of probes: 192 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 388\n",
      "--- gene=388, id=20 removed by soft count = 103\n",
      "--- number of probes: 277 , kept by blast: 276 , if remove dups: 276\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 389\n",
      "--- gene=389, id=31 removed by soft count = 460\n",
      "--- gene=389, id=35 removed by soft count = 123\n",
      "--- gene=389, id=38 removed by soft count = 227\n",
      "--- gene=389, id=40 removed by soft count = 219\n",
      "--- gene=389, id=41 removed by soft count = 100\n",
      "--- number of probes: 170 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 39\n",
      "--- gene=39, id=165 removed by soft count = 72\n",
      "--- gene=39, id=178 removed by soft count = 78\n",
      "--- number of probes: 237 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 390\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=390, id=1 removed by soft count = 52\n",
      "--- gene=390, id=2 removed by soft count = 174\n",
      "--- gene=390, id=4 removed by soft count = 3050\n",
      "--- gene=390, id=5 removed by soft count = 175\n",
      "--- gene=390, id=6 removed by soft count = 469\n",
      "--- gene=390, id=8 removed by soft count = 95\n",
      "--- gene=390, id=9 removed by soft count = 163\n",
      "--- gene=390, id=13 removed by soft count = 127\n",
      "--- gene=390, id=15 removed by soft count = 100\n",
      "--- gene=390, id=29 removed by soft count = 83\n",
      "--- gene=390, id=70 removed by soft count = 64\n",
      "--- gene=390, id=90 removed by soft count = 114\n",
      "--- gene=390, id=102 removed by soft count = 58\n",
      "--- gene=390, id=112 removed by soft count = 70\n",
      "--- gene=390, id=116 removed by soft count = 76\n",
      "--- gene=390, id=136 removed by soft count = 115\n",
      "--- gene=390, id=138 removed by soft count = 61\n",
      "--- gene=390, id=139 removed by soft count = 2035\n",
      "--- gene=390, id=140 removed by soft count = 883\n",
      "--- gene=390, id=147 removed by soft count = 76\n",
      "--- gene=390, id=150 removed by soft count = 54\n",
      "--- gene=390, id=154 removed by soft count = 68\n",
      "--- gene=390, id=160 removed by soft count = 69\n",
      "--- number of probes: 164 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 391\n",
      "--- gene=391, id=9 removed by soft count = 52\n",
      "--- gene=391, id=31 removed by soft count = 56\n",
      "--- gene=391, id=69 removed by soft count = 139\n",
      "--- gene=391, id=141 removed by soft count = 108\n",
      "--- gene=391, id=241 removed by soft count = 210\n",
      "--- gene=391, id=246 removed by soft count = 164\n",
      "--- gene=391, id=248 removed by soft count = 70\n",
      "--- gene=391, id=253 removed by soft count = 95\n",
      "--- number of probes: 257 , kept by blast: 249 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 392\n",
      "--- gene=392, id=82 removed by soft count = 296\n",
      "--- gene=392, id=109 removed by soft count = 533\n",
      "--- gene=392, id=143 removed by soft count = 689\n",
      "--- gene=392, id=144 removed by soft count = 261\n",
      "--- gene=392, id=145 removed by soft count = 389\n",
      "--- gene=392, id=146 removed by soft count = 64\n",
      "--- gene=392, id=148 removed by soft count = 98\n",
      "--- gene=392, id=150 removed by soft count = 157\n",
      "--- gene=392, id=166 removed by soft count = 466\n",
      "--- gene=392, id=167 removed by soft count = 2605\n",
      "--- gene=392, id=183 removed by soft count = 294\n",
      "--- number of probes: 202 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 393\n",
      "--- gene=393, id=12 removed by soft count = 398\n",
      "--- gene=393, id=23 removed by soft count = 240\n",
      "--- gene=393, id=200 removed by soft count = 59\n",
      "--- number of probes: 229 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 394\n",
      "--- gene=394, id=44 removed by soft count = 71\n",
      "--- gene=394, id=100 removed by soft count = 176\n",
      "--- gene=394, id=142 removed by soft count = 149\n",
      "--- gene=394, id=233 removed by soft count = 366\n",
      "--- number of probes: 309 , kept by blast: 305 , if remove dups: 305\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 395\n",
      "--- gene=395, id=12 removed by soft count = 152\n",
      "--- gene=395, id=13 removed by soft count = 541\n",
      "--- gene=395, id=105 removed by soft count = 69\n",
      "--- gene=395, id=106 removed by soft count = 293\n",
      "--- gene=395, id=107 removed by soft count = 216\n",
      "--- gene=395, id=127 removed by soft count = 125\n",
      "--- gene=395, id=134 removed by soft count = 421\n",
      "--- gene=395, id=136 removed by soft count = 269\n",
      "--- gene=395, id=155 removed by soft count = 239\n",
      "--- gene=395, id=173 removed by soft count = 118\n",
      "--- gene=395, id=175 removed by soft count = 259\n",
      "--- gene=395, id=233 removed by soft count = 65\n",
      "--- gene=395, id=235 removed by soft count = 249\n",
      "--- number of probes: 256 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 396\n",
      "--- gene=396, id=9 removed by soft count = 351\n",
      "--- gene=396, id=15 removed by soft count = 124\n",
      "--- gene=396, id=76 removed by soft count = 78\n",
      "--- gene=396, id=77 removed by soft count = 67\n",
      "--- gene=396, id=78 removed by soft count = 76\n",
      "--- number of probes: 182 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 397\n",
      "--- gene=397, id=60 removed by soft count = 197\n",
      "--- gene=397, id=177 removed by soft count = 138\n",
      "--- gene=397, id=220 removed by soft count = 488\n",
      "--- gene=397, id=236 removed by soft count = 66\n",
      "--- gene=397, id=244 removed by soft count = 480\n",
      "--- number of probes: 245 , kept by blast: 240 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 398\n",
      "--- gene=398, id=5 removed by soft count = 113\n",
      "--- gene=398, id=23 removed by soft count = 216\n",
      "--- gene=398, id=27 removed by soft count = 631\n",
      "--- gene=398, id=31 removed by soft count = 238\n",
      "--- gene=398, id=32 removed by soft count = 124\n",
      "--- gene=398, id=33 removed by soft count = 70\n",
      "--- gene=398, id=43 removed by soft count = 61\n",
      "--- gene=398, id=58 removed by soft count = 366\n",
      "--- gene=398, id=126 removed by soft count = 106\n",
      "--- gene=398, id=127 removed by soft count = 63\n",
      "--- gene=398, id=128 removed by soft count = 499\n",
      "--- gene=398, id=130 removed by soft count = 1185\n",
      "--- gene=398, id=131 removed by soft count = 1381\n",
      "--- gene=398, id=133 removed by soft count = 72\n",
      "--- gene=398, id=137 removed by soft count = 206\n",
      "--- gene=398, id=138 removed by soft count = 122\n",
      "--- gene=398, id=154 removed by soft count = 58\n",
      "--- number of probes: 162 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 399\n",
      "--- gene=399, id=124 removed by soft count = 1868\n",
      "--- gene=399, id=173 removed by soft count = 61\n",
      "--- gene=399, id=191 removed by soft count = 140\n",
      "--- number of probes: 196 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 4\n",
      "--- gene=4, id=82 removed by soft count = 480\n",
      "--- gene=4, id=154 removed by soft count = 353\n",
      "--- number of probes: 256 , kept by blast: 254 , if remove dups: 254\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 40\n",
      "--- gene=40, id=131 removed by soft count = 60\n",
      "--- gene=40, id=288 removed by soft count = 529\n",
      "--- number of probes: 293 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 400\n",
      "--- gene=400, id=138 removed by soft count = 304\n",
      "--- gene=400, id=142 removed by soft count = 120\n",
      "--- gene=400, id=173 removed by soft count = 91\n",
      "--- gene=400, id=174 removed by soft count = 94\n",
      "--- gene=400, id=202 removed by soft count = 54\n",
      "--- gene=400, id=205 removed by soft count = 132\n",
      "--- gene=400, id=206 removed by soft count = 9438\n",
      "--- number of probes: 222 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 401\n",
      "--- gene=401, id=25 removed by soft count = 183\n",
      "--- gene=401, id=70 removed by soft count = 94\n",
      "--- gene=401, id=71 removed by soft count = 198\n",
      "--- gene=401, id=109 removed by soft count = 79\n",
      "--- gene=401, id=194 removed by soft count = 4117\n",
      "--- gene=401, id=195 removed by soft count = 139\n",
      "--- gene=401, id=196 removed by soft count = 158\n",
      "--- gene=401, id=197 removed by soft count = 579\n",
      "--- gene=401, id=198 removed by soft count = 664\n",
      "--- gene=401, id=208 removed by soft count = 109\n",
      "--- number of probes: 237 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 402\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=402, id=1 removed by soft count = 320\n",
      "--- gene=402, id=11 removed by soft count = 1086\n",
      "--- gene=402, id=78 removed by soft count = 120\n",
      "--- gene=402, id=83 removed by soft count = 100\n",
      "--- gene=402, id=89 removed by soft count = 154\n",
      "--- gene=402, id=164 removed by soft count = 135\n",
      "--- gene=402, id=166 removed by soft count = 583\n",
      "--- number of probes: 191 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 403\n",
      "--- number of probes: 377 , kept by blast: 377 , if remove dups: 377\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 404\n",
      "--- gene=404, id=219 removed by soft count = 151\n",
      "--- gene=404, id=222 removed by soft count = 56\n",
      "--- gene=404, id=223 removed by soft count = 100\n",
      "--- gene=404, id=224 removed by soft count = 289\n",
      "--- gene=404, id=225 removed by soft count = 74\n",
      "--- gene=404, id=226 removed by soft count = 64\n",
      "--- number of probes: 331 , kept by blast: 325 , if remove dups: 325\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 405\n",
      "--- gene=405, id=56 removed by soft count = 359\n",
      "--- gene=405, id=105 removed by hard count = 0\n",
      "--- gene=405, id=152 removed by soft count = 244\n",
      "--- gene=405, id=164 removed by soft count = 108\n",
      "--- gene=405, id=168 removed by soft count = 55\n",
      "--- gene=405, id=184 removed by soft count = 683\n",
      "--- number of probes: 205 , kept by blast: 199 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 406\n",
      "--- gene=406, id=151 removed by soft count = 135\n",
      "--- gene=406, id=208 removed by soft count = 320\n",
      "--- gene=406, id=209 removed by soft count = 210\n",
      "--- gene=406, id=223 removed by soft count = 255\n",
      "--- number of probes: 261 , kept by blast: 257 , if remove dups: 257\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 407\n",
      "--- gene=407, id=37 removed by soft count = 55\n",
      "--- gene=407, id=42 removed by soft count = 55\n",
      "--- gene=407, id=300 removed by soft count = 133\n",
      "--- number of probes: 305 , kept by blast: 302 , if remove dups: 302\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 408\n",
      "--- gene=408, id=0 removed by soft count = 1290\n",
      "--- gene=408, id=28 removed by soft count = 54\n",
      "--- gene=408, id=43 removed by soft count = 711\n",
      "--- gene=408, id=161 removed by soft count = 58\n",
      "--- number of probes: 162 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 409\n",
      "--- gene=409, id=89 removed by soft count = 77\n",
      "--- gene=409, id=90 removed by soft count = 3755\n",
      "--- gene=409, id=91 removed by soft count = 233\n",
      "--- gene=409, id=92 removed by soft count = 118\n",
      "--- gene=409, id=93 removed by soft count = 1478\n",
      "--- gene=409, id=100 removed by soft count = 74\n",
      "--- gene=409, id=158 removed by soft count = 162\n",
      "--- gene=409, id=161 removed by soft count = 387\n",
      "--- gene=409, id=162 removed by soft count = 91\n",
      "--- gene=409, id=164 removed by soft count = 182\n",
      "--- gene=409, id=168 removed by soft count = 198\n",
      "--- gene=409, id=169 removed by soft count = 77\n",
      "--- number of probes: 221 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 410\n",
      "--- gene=410, id=86 removed by soft count = 139\n",
      "--- gene=410, id=88 removed by soft count = 58\n",
      "--- gene=410, id=90 removed by soft count = 59\n",
      "--- gene=410, id=91 removed by soft count = 145\n",
      "--- gene=410, id=92 removed by soft count = 81\n",
      "--- gene=410, id=134 removed by soft count = 80\n",
      "--- gene=410, id=165 removed by soft count = 369\n",
      "--- gene=410, id=183 removed by soft count = 130\n",
      "--- number of probes: 271 , kept by blast: 263 , if remove dups: 263\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 411\n",
      "--- gene=411, id=120 removed by soft count = 379\n",
      "--- gene=411, id=121 removed by soft count = 342\n",
      "--- gene=411, id=212 removed by soft count = 90\n",
      "--- gene=411, id=213 removed by soft count = 72\n",
      "--- gene=411, id=239 removed by soft count = 214\n",
      "--- gene=411, id=242 removed by soft count = 217\n",
      "--- gene=411, id=243 removed by soft count = 75\n",
      "--- gene=411, id=244 removed by soft count = 230\n",
      "--- gene=411, id=245 removed by soft count = 124\n",
      "--- gene=411, id=246 removed by soft count = 423\n",
      "--- number of probes: 282 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 412\n",
      "--- gene=412, id=12 removed by soft count = 5174\n",
      "--- gene=412, id=55 removed by soft count = 94\n",
      "--- gene=412, id=166 removed by soft count = 23483\n",
      "--- gene=412, id=177 removed by soft count = 53\n",
      "--- gene=412, id=188 removed by soft count = 54\n",
      "--- gene=412, id=189 removed by soft count = 54\n",
      "--- gene=412, id=198 removed by soft count = 60\n",
      "--- gene=412, id=199 removed by soft count = 54\n",
      "--- gene=412, id=200 removed by soft count = 125\n",
      "--- gene=412, id=201 removed by soft count = 918\n",
      "--- number of probes: 212 , kept by blast: 202 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 413\n",
      "--- number of probes: 339 , kept by blast: 339 , if remove dups: 339\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 414\n",
      "--- gene=414, id=46 removed by soft count = 1322\n",
      "--- gene=414, id=47 removed by soft count = 453\n",
      "--- gene=414, id=48 removed by soft count = 110\n",
      "--- gene=414, id=49 removed by soft count = 479\n",
      "--- gene=414, id=50 removed by soft count = 854\n",
      "--- gene=414, id=51 removed by soft count = 492\n",
      "--- gene=414, id=52 removed by soft count = 853\n",
      "--- gene=414, id=53 removed by soft count = 230\n",
      "--- gene=414, id=78 removed by soft count = 217\n",
      "--- gene=414, id=79 removed by soft count = 469\n",
      "--- gene=414, id=80 removed by soft count = 242\n",
      "--- gene=414, id=81 removed by soft count = 79\n",
      "--- number of probes: 188 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 415\n",
      "--- gene=415, id=0 removed by soft count = 126\n",
      "--- gene=415, id=81 removed by soft count = 305\n",
      "--- gene=415, id=82 removed by soft count = 1323\n",
      "--- gene=415, id=84 removed by soft count = 115\n",
      "--- gene=415, id=132 removed by soft count = 90\n",
      "--- gene=415, id=133 removed by soft count = 77\n",
      "--- gene=415, id=169 removed by soft count = 53\n",
      "--- gene=415, id=171 removed by soft count = 254\n",
      "--- gene=415, id=180 removed by soft count = 58\n",
      "--- number of probes: 252 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 416\n",
      "--- gene=416, id=9 removed by soft count = 185\n",
      "--- gene=416, id=10 removed by soft count = 1921\n",
      "--- gene=416, id=12 removed by soft count = 56\n",
      "--- gene=416, id=14 removed by soft count = 193\n",
      "--- number of probes: 252 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 417\n",
      "--- gene=417, id=76 removed by soft count = 204\n",
      "--- gene=417, id=77 removed by soft count = 164\n",
      "--- gene=417, id=80 removed by soft count = 345\n",
      "--- gene=417, id=81 removed by soft count = 794\n",
      "--- gene=417, id=82 removed by soft count = 95\n",
      "--- gene=417, id=84 removed by soft count = 58\n",
      "--- gene=417, id=181 removed by soft count = 124\n",
      "--- number of probes: 315 , kept by blast: 308 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 418\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=418, id=34 removed by soft count = 574\n",
      "--- gene=418, id=120 removed by soft count = 85\n",
      "--- gene=418, id=121 removed by soft count = 171\n",
      "--- gene=418, id=123 removed by soft count = 65\n",
      "--- gene=418, id=124 removed by soft count = 287\n",
      "--- gene=418, id=151 removed by soft count = 550\n",
      "--- gene=418, id=152 removed by soft count = 112\n",
      "--- gene=418, id=154 removed by soft count = 357\n",
      "--- gene=418, id=193 removed by soft count = 249\n",
      "--- number of probes: 200 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 419\n",
      "--- gene=419, id=64 removed by soft count = 88\n",
      "--- gene=419, id=128 removed by soft count = 157\n",
      "--- gene=419, id=129 removed by soft count = 568\n",
      "--- gene=419, id=155 removed by soft count = 179\n",
      "--- gene=419, id=294 removed by soft count = 84\n",
      "--- number of probes: 355 , kept by blast: 350 , if remove dups: 350\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 42\n",
      "--- gene=42, id=38 removed by soft count = 58\n",
      "--- gene=42, id=85 removed by soft count = 88\n",
      "--- gene=42, id=161 removed by soft count = 131\n",
      "--- gene=42, id=162 removed by soft count = 91\n",
      "--- gene=42, id=163 removed by soft count = 53\n",
      "--- gene=42, id=164 removed by soft count = 60\n",
      "--- gene=42, id=166 removed by soft count = 161\n",
      "--- gene=42, id=167 removed by soft count = 82\n",
      "--- gene=42, id=168 removed by soft count = 70\n",
      "--- gene=42, id=169 removed by soft count = 97\n",
      "--- gene=42, id=170 removed by soft count = 106\n",
      "--- gene=42, id=175 removed by soft count = 201\n",
      "--- gene=42, id=178 removed by soft count = 134\n",
      "--- number of probes: 201 , kept by blast: 188 , if remove dups: 188\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 421\n",
      "--- gene=421, id=0 removed by soft count = 177\n",
      "--- gene=421, id=1 removed by soft count = 15293\n",
      "--- gene=421, id=46 removed by soft count = 68\n",
      "--- gene=421, id=51 removed by soft count = 85\n",
      "--- gene=421, id=54 removed by soft count = 485\n",
      "--- gene=421, id=164 removed by soft count = 553\n",
      "--- gene=421, id=165 removed by soft count = 253\n",
      "--- gene=421, id=166 removed by soft count = 101\n",
      "--- gene=421, id=211 removed by soft count = 177\n",
      "--- number of probes: 212 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 422\n",
      "--- gene=422, id=106 removed by hard count = 4\n",
      "--- gene=422, id=147 removed by soft count = 105\n",
      "--- gene=422, id=232 removed by soft count = 93\n",
      "--- gene=422, id=235 removed by soft count = 439\n",
      "--- gene=422, id=236 removed by soft count = 470\n",
      "--- gene=422, id=237 removed by soft count = 263\n",
      "--- gene=422, id=242 removed by soft count = 59\n",
      "--- gene=422, id=245 removed by soft count = 133\n",
      "--- number of probes: 296 , kept by blast: 288 , if remove dups: 288\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 423\n",
      "--- gene=423, id=241 removed by soft count = 106\n",
      "--- gene=423, id=278 removed by soft count = 296\n",
      "--- gene=423, id=279 removed by soft count = 376\n",
      "--- gene=423, id=280 removed by soft count = 842\n",
      "--- gene=423, id=282 removed by soft count = 276\n",
      "--- gene=423, id=304 removed by soft count = 279\n",
      "--- gene=423, id=305 removed by soft count = 313\n",
      "--- gene=423, id=306 removed by soft count = 510\n",
      "--- gene=423, id=308 removed by soft count = 258\n",
      "--- gene=423, id=310 removed by soft count = 300\n",
      "--- gene=423, id=312 removed by soft count = 167\n",
      "--- number of probes: 345 , kept by blast: 334 , if remove dups: 334\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 424\n",
      "--- gene=424, id=5 removed by soft count = 73\n",
      "--- number of probes: 323 , kept by blast: 322 , if remove dups: 322\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 425\n",
      "--- gene=425, id=22 removed by soft count = 63\n",
      "--- gene=425, id=32 removed by soft count = 83\n",
      "--- number of probes: 221 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 426\n",
      "--- gene=426, id=104 removed by soft count = 82\n",
      "--- gene=426, id=160 removed by soft count = 117\n",
      "--- gene=426, id=256 removed by soft count = 63\n",
      "--- number of probes: 350 , kept by blast: 347 , if remove dups: 347\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 427\n",
      "--- gene=427, id=41 removed by soft count = 720\n",
      "--- gene=427, id=42 removed by soft count = 307\n",
      "--- gene=427, id=44 removed by soft count = 85\n",
      "--- gene=427, id=48 removed by soft count = 527\n",
      "--- gene=427, id=58 removed by soft count = 361\n",
      "--- gene=427, id=139 removed by soft count = 55\n",
      "--- number of probes: 217 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 428\n",
      "--- gene=428, id=4 removed by soft count = 120\n",
      "--- gene=428, id=6 removed by soft count = 57\n",
      "--- gene=428, id=7 removed by soft count = 90\n",
      "--- gene=428, id=75 removed by soft count = 408\n",
      "--- gene=428, id=76 removed by soft count = 91\n",
      "--- gene=428, id=173 removed by soft count = 125\n",
      "--- gene=428, id=215 removed by soft count = 180\n",
      "--- gene=428, id=216 removed by soft count = 161\n",
      "--- gene=428, id=220 removed by soft count = 220\n",
      "--- gene=428, id=239 removed by soft count = 568\n",
      "--- gene=428, id=240 removed by soft count = 385\n",
      "--- number of probes: 243 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 429\n",
      "--- gene=429, id=19 removed by soft count = 447\n",
      "--- gene=429, id=22 removed by soft count = 106\n",
      "--- gene=429, id=154 removed by soft count = 295\n",
      "--- gene=429, id=155 removed by soft count = 800\n",
      "--- gene=429, id=185 removed by soft count = 52\n",
      "--- number of probes: 193 , kept by blast: 188 , if remove dups: 188\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 43\n",
      "--- gene=43, id=45 removed by soft count = 108\n",
      "--- gene=43, id=58 removed by soft count = 58\n",
      "--- gene=43, id=163 removed by soft count = 185\n",
      "--- gene=43, id=234 removed by soft count = 187\n",
      "--- number of probes: 235 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 430\n",
      "--- gene=430, id=26 removed by soft count = 117\n",
      "--- gene=430, id=29 removed by soft count = 118\n",
      "--- gene=430, id=123 removed by soft count = 110\n",
      "--- gene=430, id=125 removed by soft count = 92\n",
      "--- gene=430, id=126 removed by soft count = 189\n",
      "--- gene=430, id=127 removed by soft count = 267\n",
      "--- gene=430, id=128 removed by soft count = 115\n",
      "--- number of probes: 236 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 431\n",
      "--- gene=431, id=54 removed by soft count = 76\n",
      "--- gene=431, id=55 removed by soft count = 147\n",
      "--- gene=431, id=98 removed by soft count = 1497\n",
      "--- gene=431, id=129 removed by soft count = 484\n",
      "--- gene=431, id=142 removed by soft count = 242\n",
      "--- gene=431, id=143 removed by soft count = 197\n",
      "--- gene=431, id=154 removed by soft count = 110\n",
      "--- gene=431, id=170 removed by soft count = 258\n",
      "--- gene=431, id=174 removed by soft count = 202\n",
      "--- gene=431, id=176 removed by soft count = 147\n",
      "--- gene=431, id=204 removed by soft count = 121\n",
      "--- gene=431, id=217 removed by soft count = 835\n",
      "--- number of probes: 237 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 432\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=432, id=88 removed by soft count = 137\n",
      "--- gene=432, id=209 removed by soft count = 105\n",
      "--- gene=432, id=211 removed by soft count = 88\n",
      "--- gene=432, id=212 removed by soft count = 101\n",
      "--- gene=432, id=215 removed by soft count = 51\n",
      "--- gene=432, id=227 removed by soft count = 81\n",
      "--- gene=432, id=228 removed by soft count = 58\n",
      "--- gene=432, id=269 removed by soft count = 177\n",
      "--- number of probes: 270 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 435\n",
      "--- gene=435, id=130 removed by soft count = 61\n",
      "--- number of probes: 357 , kept by blast: 356 , if remove dups: 356\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 436\n",
      "--- gene=436, id=8 removed by soft count = 124\n",
      "--- gene=436, id=11 removed by soft count = 211\n",
      "--- gene=436, id=12 removed by soft count = 97\n",
      "--- gene=436, id=102 removed by soft count = 89\n",
      "--- gene=436, id=244 removed by soft count = 1328\n",
      "--- number of probes: 294 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 437\n",
      "--- gene=437, id=54 removed by soft count = 130\n",
      "--- gene=437, id=60 removed by soft count = 11376\n",
      "--- gene=437, id=67 removed by soft count = 166\n",
      "--- gene=437, id=72 removed by soft count = 83\n",
      "--- gene=437, id=156 removed by soft count = 51\n",
      "--- number of probes: 188 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 438\n",
      "--- gene=438, id=36 removed by soft count = 109\n",
      "--- gene=438, id=183 removed by soft count = 133\n",
      "--- gene=438, id=184 removed by soft count = 272\n",
      "--- gene=438, id=187 removed by soft count = 118\n",
      "--- gene=438, id=188 removed by soft count = 115\n",
      "--- gene=438, id=191 removed by soft count = 83\n",
      "--- gene=438, id=192 removed by soft count = 81\n",
      "--- gene=438, id=194 removed by soft count = 134\n",
      "--- gene=438, id=195 removed by soft count = 71\n",
      "--- gene=438, id=196 removed by soft count = 144\n",
      "--- gene=438, id=199 removed by soft count = 768\n",
      "--- gene=438, id=202 removed by soft count = 167\n",
      "--- gene=438, id=205 removed by soft count = 51\n",
      "--- number of probes: 206 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 439\n",
      "--- gene=439, id=167 removed by soft count = 60\n",
      "--- number of probes: 310 , kept by blast: 309 , if remove dups: 309\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 44\n",
      "--- gene=44, id=25 removed by soft count = 95\n",
      "--- gene=44, id=26 removed by soft count = 66\n",
      "--- gene=44, id=28 removed by soft count = 234\n",
      "--- gene=44, id=29 removed by soft count = 212\n",
      "--- gene=44, id=34 removed by soft count = 97\n",
      "--- gene=44, id=309 removed by soft count = 1355\n",
      "--- gene=44, id=311 removed by soft count = 146\n",
      "--- gene=44, id=312 removed by soft count = 141\n",
      "--- gene=44, id=313 removed by soft count = 174\n",
      "--- gene=44, id=316 removed by soft count = 133\n",
      "--- number of probes: 317 , kept by blast: 307 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 440\n",
      "--- gene=440, id=39 removed by soft count = 51\n",
      "--- gene=440, id=149 removed by soft count = 139\n",
      "--- gene=440, id=213 removed by soft count = 94\n",
      "--- gene=440, id=260 removed by soft count = 92\n",
      "--- gene=440, id=262 removed by soft count = 263\n",
      "--- number of probes: 321 , kept by blast: 316 , if remove dups: 316\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 441\n",
      "--- gene=441, id=130 removed by soft count = 117\n",
      "--- number of probes: 183 , kept by blast: 182 , if remove dups: 182\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 442\n",
      "--- gene=442, id=75 removed by soft count = 355\n",
      "--- gene=442, id=129 removed by soft count = 76\n",
      "--- gene=442, id=177 removed by soft count = 305\n",
      "--- gene=442, id=267 removed by soft count = 155\n",
      "--- gene=442, id=269 removed by soft count = 286\n",
      "--- number of probes: 276 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 443\n",
      "--- gene=443, id=228 removed by soft count = 54\n",
      "--- number of probes: 241 , kept by blast: 240 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 444\n",
      "--- gene=444, id=55 removed by soft count = 6432\n",
      "--- gene=444, id=151 removed by soft count = 89\n",
      "--- gene=444, id=265 removed by soft count = 126\n",
      "--- number of probes: 307 , kept by blast: 304 , if remove dups: 304\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 446\n",
      "--- gene=446, id=1 removed by soft count = 83\n",
      "--- gene=446, id=2 removed by soft count = 892\n",
      "--- gene=446, id=3 removed by soft count = 108\n",
      "--- gene=446, id=4 removed by soft count = 1214\n",
      "--- gene=446, id=59 removed by soft count = 7035\n",
      "--- gene=446, id=60 removed by soft count = 391\n",
      "--- gene=446, id=61 removed by soft count = 66\n",
      "--- gene=446, id=62 removed by soft count = 87\n",
      "--- gene=446, id=63 removed by soft count = 8922\n",
      "--- gene=446, id=89 removed by soft count = 114\n",
      "--- gene=446, id=91 removed by soft count = 285\n",
      "--- gene=446, id=92 removed by soft count = 87\n",
      "--- gene=446, id=93 removed by soft count = 166\n",
      "--- gene=446, id=94 removed by soft count = 539\n",
      "--- gene=446, id=96 removed by soft count = 265\n",
      "--- gene=446, id=100 removed by soft count = 1524\n",
      "--- gene=446, id=101 removed by soft count = 8945\n",
      "--- gene=446, id=103 removed by soft count = 73\n",
      "--- number of probes: 108 , kept by blast: 90 , if remove dups: 90\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 90\n",
      "-- checking probes in region: 447\n",
      "--- gene=447, id=49 removed by soft count = 706\n",
      "--- gene=447, id=74 removed by soft count = 105\n",
      "--- gene=447, id=78 removed by soft count = 75\n",
      "--- gene=447, id=114 removed by soft count = 475\n",
      "--- number of probes: 181 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 448\n",
      "--- gene=448, id=87 removed by soft count = 122\n",
      "--- gene=448, id=91 removed by soft count = 55\n",
      "--- gene=448, id=93 removed by soft count = 468\n",
      "--- gene=448, id=94 removed by soft count = 1719\n",
      "--- gene=448, id=96 removed by soft count = 139\n",
      "--- gene=448, id=97 removed by soft count = 690\n",
      "--- gene=448, id=108 removed by soft count = 272\n",
      "--- gene=448, id=117 removed by soft count = 179\n",
      "--- gene=448, id=118 removed by soft count = 164\n",
      "--- gene=448, id=120 removed by soft count = 162\n",
      "--- gene=448, id=122 removed by soft count = 171\n",
      "--- gene=448, id=123 removed by soft count = 95\n",
      "--- gene=448, id=124 removed by soft count = 238\n",
      "--- gene=448, id=125 removed by soft count = 87\n",
      "--- gene=448, id=126 removed by soft count = 250\n",
      "--- number of probes: 127 , kept by blast: 112 , if remove dups: 112\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 112\n",
      "-- checking probes in region: 449\n",
      "--- gene=449, id=5 removed by soft count = 361\n",
      "--- gene=449, id=84 removed by soft count = 585\n",
      "--- number of probes: 228 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 45\n",
      "--- gene=45, id=27 removed by soft count = 51\n",
      "--- gene=45, id=29 removed by soft count = 58\n",
      "--- gene=45, id=37 removed by soft count = 485\n",
      "--- gene=45, id=40 removed by soft count = 238\n",
      "--- gene=45, id=41 removed by soft count = 62\n",
      "--- gene=45, id=74 removed by soft count = 968\n",
      "--- gene=45, id=85 removed by soft count = 189\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=45, id=98 removed by soft count = 901\n",
      "--- number of probes: 187 , kept by blast: 179 , if remove dups: 179\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 450\n",
      "--- gene=450, id=39 removed by soft count = 86\n",
      "--- gene=450, id=41 removed by soft count = 61\n",
      "--- gene=450, id=42 removed by soft count = 95\n",
      "--- gene=450, id=44 removed by soft count = 589\n",
      "--- gene=450, id=45 removed by soft count = 1811\n",
      "--- gene=450, id=46 removed by soft count = 1287\n",
      "--- gene=450, id=47 removed by soft count = 446\n",
      "--- gene=450, id=63 removed by soft count = 85\n",
      "--- gene=450, id=64 removed by soft count = 57\n",
      "--- gene=450, id=65 removed by soft count = 437\n",
      "--- gene=450, id=67 removed by soft count = 744\n",
      "--- gene=450, id=68 removed by soft count = 204\n",
      "--- gene=450, id=71 removed by soft count = 115\n",
      "--- gene=450, id=72 removed by soft count = 271\n",
      "--- gene=450, id=145 removed by soft count = 59\n",
      "--- number of probes: 268 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 451\n",
      "--- number of probes: 258 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 452\n",
      "--- gene=452, id=2 removed by soft count = 69\n",
      "--- gene=452, id=15 removed by soft count = 122\n",
      "--- gene=452, id=17 removed by soft count = 90\n",
      "--- gene=452, id=19 removed by soft count = 127\n",
      "--- gene=452, id=94 removed by soft count = 108\n",
      "--- gene=452, id=123 removed by soft count = 165\n",
      "--- gene=452, id=146 removed by soft count = 916\n",
      "--- gene=452, id=148 removed by soft count = 186\n",
      "--- gene=452, id=149 removed by soft count = 77\n",
      "--- gene=452, id=151 removed by soft count = 70\n",
      "--- gene=452, id=152 removed by soft count = 95\n",
      "--- gene=452, id=179 removed by soft count = 1125\n",
      "--- gene=452, id=180 removed by soft count = 143\n",
      "--- gene=452, id=181 removed by soft count = 2151\n",
      "--- number of probes: 197 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 453\n",
      "--- gene=453, id=1 removed by soft count = 64\n",
      "--- gene=453, id=2 removed by soft count = 54\n",
      "--- gene=453, id=7 removed by soft count = 123\n",
      "--- gene=453, id=9 removed by soft count = 66\n",
      "--- gene=453, id=20 removed by soft count = 209\n",
      "--- gene=453, id=22 removed by soft count = 96\n",
      "--- gene=453, id=44 removed by soft count = 76\n",
      "--- gene=453, id=45 removed by soft count = 77\n",
      "--- gene=453, id=46 removed by soft count = 84\n",
      "--- gene=453, id=49 removed by soft count = 126\n",
      "--- gene=453, id=54 removed by soft count = 85\n",
      "--- gene=453, id=57 removed by soft count = 163\n",
      "--- gene=453, id=58 removed by soft count = 715\n",
      "--- gene=453, id=59 removed by soft count = 73\n",
      "--- gene=453, id=62 removed by soft count = 51\n",
      "--- gene=453, id=63 removed by soft count = 877\n",
      "--- gene=453, id=64 removed by soft count = 476\n",
      "--- gene=453, id=65 removed by soft count = 255\n",
      "--- gene=453, id=66 removed by soft count = 121\n",
      "--- number of probes: 67 , kept by blast: 48 , if remove dups: 48\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: 454\n",
      "--- gene=454, id=104 removed by soft count = 7466\n",
      "--- gene=454, id=124 removed by soft count = 160\n",
      "--- gene=454, id=129 removed by soft count = 84\n",
      "--- gene=454, id=132 removed by soft count = 134\n",
      "--- number of probes: 145 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 455\n",
      "--- gene=455, id=14 removed by soft count = 619\n",
      "--- gene=455, id=15 removed by soft count = 1030\n",
      "--- gene=455, id=16 removed by soft count = 352\n",
      "--- gene=455, id=17 removed by soft count = 97\n",
      "--- gene=455, id=19 removed by soft count = 174\n",
      "--- gene=455, id=20 removed by soft count = 125\n",
      "--- gene=455, id=21 removed by soft count = 157\n",
      "--- gene=455, id=26 removed by soft count = 60\n",
      "--- gene=455, id=42 removed by soft count = 195\n",
      "--- gene=455, id=50 removed by soft count = 189\n",
      "--- gene=455, id=61 removed by soft count = 181\n",
      "--- gene=455, id=63 removed by soft count = 492\n",
      "--- gene=455, id=64 removed by soft count = 137\n",
      "--- gene=455, id=69 removed by soft count = 123\n",
      "--- gene=455, id=70 removed by soft count = 236\n",
      "--- gene=455, id=135 removed by soft count = 169\n",
      "--- number of probes: 142 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 456\n",
      "--- gene=456, id=12 removed by soft count = 464\n",
      "--- gene=456, id=13 removed by soft count = 51\n",
      "--- gene=456, id=14 removed by soft count = 185\n",
      "--- gene=456, id=16 removed by hard count = 6\n",
      "--- number of probes: 217 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 457\n",
      "--- gene=457, id=9 removed by soft count = 63\n",
      "--- gene=457, id=10 removed by soft count = 106\n",
      "--- gene=457, id=12 removed by soft count = 65\n",
      "--- gene=457, id=16 removed by soft count = 60\n",
      "--- gene=457, id=27 removed by soft count = 108\n",
      "--- gene=457, id=28 removed by soft count = 329\n",
      "--- gene=457, id=61 removed by soft count = 1151\n",
      "--- gene=457, id=175 removed by soft count = 84\n",
      "--- gene=457, id=191 removed by soft count = 2121\n",
      "--- number of probes: 207 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 458\n",
      "--- gene=458, id=70 removed by soft count = 116\n",
      "--- gene=458, id=171 removed by soft count = 934\n",
      "--- gene=458, id=172 removed by soft count = 297\n",
      "--- gene=458, id=173 removed by soft count = 571\n",
      "--- gene=458, id=174 removed by soft count = 953\n",
      "--- gene=458, id=181 removed by soft count = 62\n",
      "--- gene=458, id=197 removed by soft count = 54\n",
      "--- gene=458, id=198 removed by soft count = 80\n",
      "--- gene=458, id=211 removed by soft count = 78\n",
      "--- number of probes: 236 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 459\n",
      "--- gene=459, id=164 removed by soft count = 175\n",
      "--- number of probes: 180 , kept by blast: 179 , if remove dups: 179\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 46\n",
      "--- gene=46, id=3 removed by soft count = 331\n",
      "--- gene=46, id=34 removed by soft count = 71\n",
      "--- gene=46, id=37 removed by soft count = 233\n",
      "--- gene=46, id=52 removed by soft count = 62\n",
      "--- gene=46, id=103 removed by soft count = 119\n",
      "--- gene=46, id=141 removed by soft count = 112\n",
      "--- gene=46, id=180 removed by soft count = 208\n",
      "--- gene=46, id=181 removed by soft count = 209\n",
      "--- gene=46, id=183 removed by soft count = 61\n",
      "--- gene=46, id=224 removed by soft count = 60\n",
      "--- number of probes: 309 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 460\n",
      "--- gene=460, id=143 removed by soft count = 166\n",
      "--- gene=460, id=144 removed by soft count = 100\n",
      "--- gene=460, id=145 removed by soft count = 143\n",
      "--- gene=460, id=146 removed by soft count = 293\n",
      "--- gene=460, id=147 removed by soft count = 51\n",
      "--- gene=460, id=149 removed by soft count = 306\n",
      "--- gene=460, id=183 removed by soft count = 144\n",
      "--- gene=460, id=187 removed by soft count = 1561\n",
      "--- gene=460, id=188 removed by soft count = 1096\n",
      "--- gene=460, id=189 removed by soft count = 712\n",
      "--- gene=460, id=192 removed by soft count = 83\n",
      "--- gene=460, id=194 removed by soft count = 637\n",
      "--- gene=460, id=196 removed by soft count = 57\n",
      "--- number of probes: 202 , kept by blast: 189 , if remove dups: 189\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 461\n",
      "--- gene=461, id=86 removed by soft count = 1594\n",
      "--- gene=461, id=87 removed by soft count = 578\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=461, id=88 removed by soft count = 397\n",
      "--- gene=461, id=146 removed by soft count = 69\n",
      "--- gene=461, id=195 removed by soft count = 608\n",
      "--- number of probes: 209 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 462\n",
      "--- gene=462, id=9 removed by soft count = 115\n",
      "--- gene=462, id=23 removed by soft count = 158\n",
      "--- gene=462, id=27 removed by soft count = 236\n",
      "--- gene=462, id=124 removed by soft count = 99\n",
      "--- gene=462, id=125 removed by soft count = 73\n",
      "--- gene=462, id=126 removed by soft count = 79\n",
      "--- gene=462, id=129 removed by soft count = 61\n",
      "--- gene=462, id=132 removed by soft count = 62\n",
      "--- gene=462, id=136 removed by soft count = 85\n",
      "--- gene=462, id=138 removed by soft count = 55\n",
      "--- gene=462, id=139 removed by soft count = 57\n",
      "--- gene=462, id=140 removed by soft count = 79\n",
      "--- number of probes: 170 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 463\n",
      "--- gene=463, id=0 removed by soft count = 430\n",
      "--- gene=463, id=2 removed by soft count = 444\n",
      "--- gene=463, id=67 removed by soft count = 94\n",
      "--- gene=463, id=89 removed by soft count = 187\n",
      "--- gene=463, id=90 removed by soft count = 81\n",
      "--- gene=463, id=111 removed by soft count = 128\n",
      "--- gene=463, id=112 removed by soft count = 234\n",
      "--- gene=463, id=119 removed by soft count = 77\n",
      "--- gene=463, id=123 removed by soft count = 104\n",
      "--- gene=463, id=124 removed by soft count = 604\n",
      "--- gene=463, id=128 removed by soft count = 58\n",
      "--- gene=463, id=130 removed by soft count = 52\n",
      "--- gene=463, id=134 removed by soft count = 136\n",
      "--- number of probes: 143 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 464\n",
      "--- gene=464, id=1 removed by soft count = 101\n",
      "--- gene=464, id=36 removed by soft count = 105\n",
      "--- gene=464, id=38 removed by soft count = 240\n",
      "--- gene=464, id=89 removed by soft count = 74\n",
      "--- gene=464, id=91 removed by soft count = 334\n",
      "--- gene=464, id=110 removed by soft count = 123\n",
      "--- gene=464, id=162 removed by soft count = 292\n",
      "--- gene=464, id=174 removed by soft count = 23840\n",
      "--- gene=464, id=203 removed by soft count = 349\n",
      "--- gene=464, id=207 removed by soft count = 663\n",
      "--- gene=464, id=208 removed by soft count = 156\n",
      "--- gene=464, id=209 removed by soft count = 85\n",
      "--- gene=464, id=210 removed by soft count = 63\n",
      "--- number of probes: 211 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 465\n",
      "--- gene=465, id=39 removed by soft count = 87\n",
      "--- gene=465, id=41 removed by soft count = 84\n",
      "--- number of probes: 352 , kept by blast: 350 , if remove dups: 350\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 466\n",
      "--- gene=466, id=24 removed by soft count = 116\n",
      "--- gene=466, id=25 removed by soft count = 142\n",
      "--- number of probes: 189 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 467\n",
      "--- gene=467, id=93 removed by soft count = 265\n",
      "--- gene=467, id=115 removed by soft count = 624\n",
      "--- gene=467, id=116 removed by soft count = 71\n",
      "--- gene=467, id=146 removed by soft count = 253\n",
      "--- number of probes: 200 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 468\n",
      "--- gene=468, id=48 removed by soft count = 74\n",
      "--- gene=468, id=194 removed by soft count = 84\n",
      "--- gene=468, id=204 removed by soft count = 80\n",
      "--- gene=468, id=213 removed by soft count = 239\n",
      "--- gene=468, id=215 removed by soft count = 273\n",
      "--- gene=468, id=217 removed by soft count = 62\n",
      "--- gene=468, id=219 removed by soft count = 145\n",
      "--- gene=468, id=221 removed by soft count = 56\n",
      "--- number of probes: 222 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 469\n",
      "--- gene=469, id=213 removed by soft count = 375\n",
      "--- gene=469, id=242 removed by soft count = 379\n",
      "--- number of probes: 260 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 47\n",
      "--- gene=47, id=96 removed by soft count = 84\n",
      "--- gene=47, id=97 removed by soft count = 78\n",
      "--- gene=47, id=207 removed by soft count = 134\n",
      "--- gene=47, id=209 removed by soft count = 66\n",
      "--- gene=47, id=211 removed by soft count = 165\n",
      "--- number of probes: 213 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 470\n",
      "--- gene=470, id=0 removed by soft count = 379\n",
      "--- gene=470, id=1 removed by soft count = 211\n",
      "--- gene=470, id=2 removed by soft count = 70\n",
      "--- gene=470, id=3 removed by soft count = 86\n",
      "--- gene=470, id=4 removed by soft count = 156\n",
      "--- gene=470, id=5 removed by soft count = 211\n",
      "--- gene=470, id=64 removed by soft count = 64\n",
      "--- gene=470, id=73 removed by soft count = 81\n",
      "--- gene=470, id=76 removed by soft count = 406\n",
      "--- gene=470, id=77 removed by soft count = 264\n",
      "--- gene=470, id=78 removed by soft count = 279\n",
      "--- gene=470, id=80 removed by soft count = 296\n",
      "--- gene=470, id=81 removed by soft count = 1591\n",
      "--- gene=470, id=122 removed by soft count = 107\n",
      "--- gene=470, id=127 removed by soft count = 91\n",
      "--- gene=470, id=128 removed by soft count = 120\n",
      "--- gene=470, id=130 removed by soft count = 343\n",
      "--- number of probes: 131 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 114\n",
      "-- checking probes in region: 471\n",
      "--- gene=471, id=4 removed by soft count = 84\n",
      "--- gene=471, id=7 removed by soft count = 53\n",
      "--- gene=471, id=18 removed by soft count = 57\n",
      "--- gene=471, id=29 removed by soft count = 102\n",
      "--- gene=471, id=134 removed by soft count = 51\n",
      "--- gene=471, id=139 removed by soft count = 110\n",
      "--- gene=471, id=141 removed by soft count = 63\n",
      "--- gene=471, id=144 removed by soft count = 1060\n",
      "--- number of probes: 199 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 472\n",
      "--- gene=472, id=0 removed by soft count = 621\n",
      "--- gene=472, id=95 removed by soft count = 300\n",
      "--- gene=472, id=121 removed by soft count = 117\n",
      "--- gene=472, id=136 removed by soft count = 12970\n",
      "--- gene=472, id=154 removed by soft count = 77\n",
      "--- gene=472, id=156 removed by soft count = 106\n",
      "--- gene=472, id=199 removed by soft count = 54\n",
      "--- number of probes: 203 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 473\n",
      "--- gene=473, id=17 removed by soft count = 120\n",
      "--- gene=473, id=19 removed by soft count = 56\n",
      "--- gene=473, id=40 removed by soft count = 51\n",
      "--- gene=473, id=64 removed by soft count = 127\n",
      "--- gene=473, id=65 removed by soft count = 73\n",
      "--- gene=473, id=92 removed by soft count = 130\n",
      "--- gene=473, id=182 removed by soft count = 417\n",
      "--- gene=473, id=185 removed by soft count = 78\n",
      "--- gene=473, id=199 removed by soft count = 73\n",
      "--- gene=473, id=202 removed by soft count = 5597\n",
      "--- number of probes: 203 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 474\n",
      "--- gene=474, id=193 removed by soft count = 131\n",
      "--- number of probes: 235 , kept by blast: 234 , if remove dups: 234\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 475\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=475, id=0 removed by soft count = 101\n",
      "--- gene=475, id=65 removed by soft count = 89\n",
      "--- gene=475, id=66 removed by soft count = 65\n",
      "--- gene=475, id=73 removed by soft count = 103\n",
      "--- gene=475, id=109 removed by soft count = 431\n",
      "--- number of probes: 263 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 476\n",
      "--- gene=476, id=49 removed by soft count = 33510\n",
      "--- gene=476, id=65 removed by soft count = 99\n",
      "--- gene=476, id=66 removed by soft count = 576\n",
      "--- gene=476, id=67 removed by soft count = 433\n",
      "--- gene=476, id=68 removed by soft count = 108\n",
      "--- number of probes: 236 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 477\n",
      "--- gene=477, id=6 removed by soft count = 397\n",
      "--- gene=477, id=128 removed by soft count = 137\n",
      "--- gene=477, id=175 removed by soft count = 237\n",
      "--- number of probes: 215 , kept by blast: 212 , if remove dups: 212\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 478\n",
      "--- gene=478, id=63 removed by soft count = 63\n",
      "--- gene=478, id=64 removed by soft count = 132\n",
      "--- gene=478, id=78 removed by soft count = 213\n",
      "--- gene=478, id=82 removed by soft count = 64\n",
      "--- gene=478, id=107 removed by soft count = 113\n",
      "--- gene=478, id=148 removed by soft count = 143\n",
      "--- gene=478, id=152 removed by soft count = 58\n",
      "--- gene=478, id=160 removed by soft count = 113\n",
      "--- gene=478, id=165 removed by soft count = 465\n",
      "--- gene=478, id=166 removed by soft count = 69\n",
      "--- gene=478, id=172 removed by soft count = 152\n",
      "--- number of probes: 189 , kept by blast: 178 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 479\n",
      "--- gene=479, id=30 removed by soft count = 389\n",
      "--- gene=479, id=31 removed by soft count = 1231\n",
      "--- number of probes: 287 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 480\n",
      "--- gene=480, id=92 removed by soft count = 206\n",
      "--- gene=480, id=128 removed by soft count = 57\n",
      "--- gene=480, id=130 removed by soft count = 182\n",
      "--- gene=480, id=131 removed by soft count = 129\n",
      "--- gene=480, id=132 removed by soft count = 919\n",
      "--- gene=480, id=133 removed by soft count = 2071\n",
      "--- gene=480, id=227 removed by soft count = 76\n",
      "--- number of probes: 257 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 481\n",
      "--- gene=481, id=190 removed by soft count = 85\n",
      "--- gene=481, id=207 removed by soft count = 173\n",
      "--- gene=481, id=268 removed by soft count = 1520\n",
      "--- number of probes: 269 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 482\n",
      "--- gene=482, id=33 removed by soft count = 159\n",
      "--- gene=482, id=133 removed by soft count = 464\n",
      "--- gene=482, id=159 removed by soft count = 328\n",
      "--- number of probes: 230 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 483\n",
      "--- gene=483, id=7 removed by soft count = 54\n",
      "--- gene=483, id=29 removed by soft count = 66\n",
      "--- gene=483, id=31 removed by soft count = 681\n",
      "--- gene=483, id=32 removed by soft count = 261\n",
      "--- gene=483, id=126 removed by soft count = 876\n",
      "--- gene=483, id=187 removed by soft count = 80\n",
      "--- gene=483, id=200 removed by soft count = 174\n",
      "--- gene=483, id=202 removed by soft count = 52\n",
      "--- gene=483, id=203 removed by soft count = 80\n",
      "--- gene=483, id=208 removed by soft count = 65\n",
      "--- gene=483, id=209 removed by soft count = 76\n",
      "--- gene=483, id=214 removed by soft count = 61\n",
      "--- gene=483, id=215 removed by soft count = 84\n",
      "--- number of probes: 220 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 484\n",
      "--- gene=484, id=85 removed by soft count = 128\n",
      "--- gene=484, id=154 removed by soft count = 101\n",
      "--- number of probes: 265 , kept by blast: 263 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 485\n",
      "--- gene=485, id=46 removed by soft count = 102\n",
      "--- gene=485, id=48 removed by soft count = 103\n",
      "--- gene=485, id=49 removed by soft count = 73\n",
      "--- gene=485, id=222 removed by soft count = 1779\n",
      "--- number of probes: 254 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 486\n",
      "--- gene=486, id=7 removed by soft count = 102\n",
      "--- gene=486, id=43 removed by soft count = 2391\n",
      "--- gene=486, id=137 removed by soft count = 21035\n",
      "--- gene=486, id=188 removed by soft count = 117\n",
      "--- number of probes: 294 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 487\n",
      "--- gene=487, id=243 removed by soft count = 121\n",
      "--- gene=487, id=247 removed by soft count = 70\n",
      "--- gene=487, id=262 removed by soft count = 108\n",
      "--- gene=487, id=273 removed by soft count = 816\n",
      "--- gene=487, id=276 removed by soft count = 263\n",
      "--- gene=487, id=277 removed by soft count = 484\n",
      "--- gene=487, id=278 removed by soft count = 175\n",
      "--- number of probes: 342 , kept by blast: 335 , if remove dups: 335\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 489\n",
      "--- gene=489, id=55 removed by soft count = 402\n",
      "--- gene=489, id=116 removed by soft count = 182\n",
      "--- gene=489, id=118 removed by soft count = 144\n",
      "--- gene=489, id=141 removed by soft count = 67\n",
      "--- gene=489, id=142 removed by soft count = 87\n",
      "--- gene=489, id=143 removed by soft count = 84\n",
      "--- gene=489, id=160 removed by soft count = 69\n",
      "--- gene=489, id=167 removed by soft count = 103\n",
      "--- gene=489, id=198 removed by soft count = 69\n",
      "--- gene=489, id=203 removed by soft count = 92\n",
      "--- gene=489, id=206 removed by soft count = 277\n",
      "--- gene=489, id=208 removed by soft count = 267\n",
      "--- gene=489, id=209 removed by soft count = 253\n",
      "--- gene=489, id=211 removed by soft count = 99\n",
      "--- number of probes: 216 , kept by blast: 202 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 490\n",
      "--- gene=490, id=212 removed by soft count = 201\n",
      "--- number of probes: 377 , kept by blast: 376 , if remove dups: 376\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 491\n",
      "--- gene=491, id=6 removed by soft count = 82\n",
      "--- gene=491, id=24 removed by soft count = 61\n",
      "--- gene=491, id=98 removed by soft count = 63\n",
      "--- gene=491, id=102 removed by soft count = 164\n",
      "--- gene=491, id=103 removed by soft count = 94\n",
      "--- gene=491, id=134 removed by soft count = 70\n",
      "--- gene=491, id=136 removed by soft count = 160\n",
      "--- gene=491, id=209 removed by soft count = 118\n",
      "--- gene=491, id=227 removed by soft count = 57\n",
      "--- number of probes: 265 , kept by blast: 256 , if remove dups: 256\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 492\n",
      "--- gene=492, id=76 removed by soft count = 204\n",
      "--- gene=492, id=112 removed by soft count = 67\n",
      "--- number of probes: 229 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 493\n",
      "--- gene=493, id=0 removed by soft count = 60\n",
      "--- gene=493, id=1 removed by soft count = 718\n",
      "--- gene=493, id=2 removed by soft count = 182\n",
      "--- gene=493, id=3 removed by soft count = 107\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=493, id=4 removed by soft count = 1124\n",
      "--- gene=493, id=15 removed by soft count = 60\n",
      "--- gene=493, id=43 removed by soft count = 1141\n",
      "--- gene=493, id=53 removed by soft count = 185\n",
      "--- gene=493, id=82 removed by soft count = 115\n",
      "--- number of probes: 239 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 494\n",
      "--- gene=494, id=0 removed by soft count = 144\n",
      "--- gene=494, id=1 removed by soft count = 143\n",
      "--- gene=494, id=2 removed by soft count = 66\n",
      "--- gene=494, id=7 removed by soft count = 837\n",
      "--- gene=494, id=268 removed by soft count = 164\n",
      "--- gene=494, id=290 removed by soft count = 714\n",
      "--- number of probes: 293 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 495\n",
      "--- gene=495, id=220 removed by soft count = 143\n",
      "--- gene=495, id=225 removed by soft count = 56\n",
      "--- gene=495, id=226 removed by soft count = 101\n",
      "--- gene=495, id=227 removed by soft count = 100\n",
      "--- gene=495, id=240 removed by soft count = 664\n",
      "--- gene=495, id=241 removed by soft count = 61\n",
      "--- gene=495, id=257 removed by soft count = 186\n",
      "--- gene=495, id=258 removed by soft count = 87\n",
      "--- number of probes: 259 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 496\n",
      "--- gene=496, id=6 removed by soft count = 79\n",
      "--- gene=496, id=24 removed by soft count = 51\n",
      "--- gene=496, id=59 removed by soft count = 414\n",
      "--- gene=496, id=62 removed by soft count = 102\n",
      "--- gene=496, id=82 removed by soft count = 412\n",
      "--- gene=496, id=85 removed by soft count = 296\n",
      "--- gene=496, id=86 removed by soft count = 87\n",
      "--- gene=496, id=88 removed by soft count = 276\n",
      "--- gene=496, id=97 removed by soft count = 80\n",
      "--- gene=496, id=99 removed by soft count = 205\n",
      "--- gene=496, id=100 removed by soft count = 1136\n",
      "--- number of probes: 167 , kept by blast: 156 , if remove dups: 156\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 497\n",
      "--- gene=497, id=0 removed by soft count = 12319\n",
      "--- gene=497, id=1 removed by soft count = 1303\n",
      "--- gene=497, id=3 removed by soft count = 57\n",
      "--- gene=497, id=7 removed by soft count = 177\n",
      "--- gene=497, id=8 removed by soft count = 65\n",
      "--- gene=497, id=9 removed by soft count = 392\n",
      "--- gene=497, id=10 removed by soft count = 93\n",
      "--- gene=497, id=11 removed by soft count = 651\n",
      "--- gene=497, id=175 removed by soft count = 2015\n",
      "--- number of probes: 228 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 498\n",
      "--- number of probes: 262 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 499\n",
      "--- gene=499, id=55 removed by soft count = 271\n",
      "--- gene=499, id=56 removed by soft count = 55\n",
      "--- gene=499, id=58 removed by soft count = 95\n",
      "--- gene=499, id=83 removed by soft count = 69\n",
      "--- gene=499, id=85 removed by soft count = 101\n",
      "--- gene=499, id=91 removed by soft count = 125\n",
      "--- gene=499, id=94 removed by soft count = 67\n",
      "--- gene=499, id=102 removed by soft count = 312\n",
      "--- gene=499, id=107 removed by soft count = 64\n",
      "--- gene=499, id=126 removed by soft count = 1101\n",
      "--- gene=499, id=128 removed by soft count = 274\n",
      "--- gene=499, id=129 removed by soft count = 5042\n",
      "--- number of probes: 162 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 5\n",
      "--- gene=5, id=67 removed by soft count = 114\n",
      "--- gene=5, id=122 removed by soft count = 69\n",
      "--- number of probes: 303 , kept by blast: 301 , if remove dups: 301\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 500\n",
      "--- gene=500, id=15 removed by soft count = 60\n",
      "--- gene=500, id=32 removed by soft count = 454\n",
      "--- gene=500, id=80 removed by soft count = 157\n",
      "--- gene=500, id=81 removed by soft count = 94\n",
      "--- gene=500, id=100 removed by soft count = 177\n",
      "--- gene=500, id=108 removed by soft count = 59\n",
      "--- gene=500, id=125 removed by soft count = 1794\n",
      "--- gene=500, id=140 removed by soft count = 215\n",
      "--- gene=500, id=141 removed by soft count = 83\n",
      "--- number of probes: 142 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 501\n",
      "--- gene=501, id=296 removed by soft count = 356\n",
      "--- number of probes: 304 , kept by blast: 303 , if remove dups: 303\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 502\n",
      "--- gene=502, id=58 removed by soft count = 71\n",
      "--- gene=502, id=199 removed by soft count = 65\n",
      "--- gene=502, id=201 removed by soft count = 78\n",
      "--- gene=502, id=251 removed by soft count = 57\n",
      "--- gene=502, id=262 removed by soft count = 129\n",
      "--- number of probes: 269 , kept by blast: 264 , if remove dups: 264\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 503\n",
      "--- gene=503, id=14 removed by soft count = 103\n",
      "--- gene=503, id=34 removed by soft count = 93\n",
      "--- gene=503, id=58 removed by soft count = 325\n",
      "--- gene=503, id=59 removed by soft count = 70\n",
      "--- gene=503, id=160 removed by soft count = 209\n",
      "--- gene=503, id=161 removed by soft count = 350\n",
      "--- gene=503, id=172 removed by soft count = 220\n",
      "--- number of probes: 318 , kept by blast: 311 , if remove dups: 311\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 504\n",
      "--- gene=504, id=10 removed by soft count = 161\n",
      "--- gene=504, id=12 removed by soft count = 51\n",
      "--- gene=504, id=14 removed by soft count = 413\n",
      "--- gene=504, id=15 removed by soft count = 78\n",
      "--- gene=504, id=16 removed by soft count = 240\n",
      "--- gene=504, id=159 removed by soft count = 189\n",
      "--- gene=504, id=162 removed by soft count = 436\n",
      "--- gene=504, id=164 removed by soft count = 127\n",
      "--- gene=504, id=190 removed by soft count = 79\n",
      "--- gene=504, id=191 removed by soft count = 77\n",
      "--- gene=504, id=192 removed by soft count = 68\n",
      "--- gene=504, id=193 removed by soft count = 276\n",
      "--- gene=504, id=195 removed by soft count = 5308\n",
      "--- number of probes: 220 , kept by blast: 207 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 505\n",
      "--- gene=505, id=132 removed by soft count = 125\n",
      "--- gene=505, id=256 removed by soft count = 98\n",
      "--- gene=505, id=257 removed by soft count = 162\n",
      "--- gene=505, id=260 removed by soft count = 113\n",
      "--- gene=505, id=274 removed by soft count = 78\n",
      "--- number of probes: 308 , kept by blast: 303 , if remove dups: 303\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 506\n",
      "--- gene=506, id=220 removed by soft count = 136\n",
      "--- gene=506, id=259 removed by soft count = 258\n",
      "--- number of probes: 306 , kept by blast: 304 , if remove dups: 304\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 507\n",
      "--- gene=507, id=0 removed by soft count = 139\n",
      "--- gene=507, id=1 removed by soft count = 165\n",
      "--- gene=507, id=9 removed by soft count = 509\n",
      "--- gene=507, id=28 removed by soft count = 136\n",
      "--- gene=507, id=78 removed by soft count = 55\n",
      "--- gene=507, id=80 removed by soft count = 95\n",
      "--- gene=507, id=82 removed by soft count = 228\n",
      "--- gene=507, id=97 removed by soft count = 93\n",
      "--- gene=507, id=105 removed by soft count = 110\n",
      "--- gene=507, id=106 removed by soft count = 704\n",
      "--- gene=507, id=161 removed by soft count = 146\n",
      "--- number of probes: 202 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 508\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=508, id=15 removed by soft count = 83\n",
      "--- gene=508, id=36 removed by soft count = 69\n",
      "--- gene=508, id=41 removed by soft count = 67\n",
      "--- gene=508, id=42 removed by soft count = 66\n",
      "--- gene=508, id=46 removed by soft count = 149\n",
      "--- gene=508, id=90 removed by soft count = 98\n",
      "--- gene=508, id=93 removed by soft count = 51\n",
      "--- gene=508, id=134 removed by soft count = 63\n",
      "--- gene=508, id=142 removed by soft count = 53\n",
      "--- gene=508, id=156 removed by soft count = 56\n",
      "--- gene=508, id=170 removed by soft count = 89\n",
      "--- number of probes: 221 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 509\n",
      "--- gene=509, id=18 removed by soft count = 56\n",
      "--- gene=509, id=19 removed by soft count = 52\n",
      "--- gene=509, id=80 removed by soft count = 1044\n",
      "--- gene=509, id=81 removed by soft count = 3951\n",
      "--- gene=509, id=82 removed by soft count = 57\n",
      "--- gene=509, id=94 removed by soft count = 75\n",
      "--- gene=509, id=96 removed by soft count = 70\n",
      "--- gene=509, id=184 removed by soft count = 126\n",
      "--- gene=509, id=185 removed by soft count = 507\n",
      "--- gene=509, id=206 removed by soft count = 151\n",
      "--- gene=509, id=208 removed by soft count = 57\n",
      "--- number of probes: 272 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 510\n",
      "--- gene=510, id=47 removed by soft count = 519\n",
      "--- gene=510, id=54 removed by soft count = 1440\n",
      "--- gene=510, id=55 removed by soft count = 506\n",
      "--- gene=510, id=79 removed by soft count = 68\n",
      "--- gene=510, id=197 removed by soft count = 216\n",
      "--- number of probes: 336 , kept by blast: 331 , if remove dups: 331\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 511\n",
      "--- gene=511, id=0 removed by soft count = 51\n",
      "--- gene=511, id=19 removed by soft count = 369\n",
      "--- gene=511, id=21 removed by soft count = 105\n",
      "--- gene=511, id=23 removed by soft count = 237\n",
      "--- gene=511, id=25 removed by soft count = 63\n",
      "--- gene=511, id=26 removed by soft count = 1045\n",
      "--- gene=511, id=28 removed by soft count = 287\n",
      "--- gene=511, id=40 removed by soft count = 123\n",
      "--- gene=511, id=48 removed by soft count = 1475\n",
      "--- gene=511, id=81 removed by soft count = 119\n",
      "--- gene=511, id=85 removed by soft count = 179\n",
      "--- gene=511, id=86 removed by soft count = 231\n",
      "--- gene=511, id=89 removed by soft count = 76\n",
      "--- gene=511, id=90 removed by soft count = 60\n",
      "--- gene=511, id=93 removed by soft count = 147\n",
      "--- gene=511, id=104 removed by soft count = 56\n",
      "--- gene=511, id=113 removed by soft count = 134\n",
      "--- gene=511, id=121 removed by soft count = 75\n",
      "--- gene=511, id=182 removed by soft count = 427\n",
      "--- number of probes: 184 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 512\n",
      "--- gene=512, id=6 removed by soft count = 58\n",
      "--- gene=512, id=7 removed by soft count = 88\n",
      "--- gene=512, id=8 removed by soft count = 164\n",
      "--- gene=512, id=41 removed by soft count = 190\n",
      "--- gene=512, id=52 removed by soft count = 436\n",
      "--- gene=512, id=53 removed by soft count = 57\n",
      "--- gene=512, id=101 removed by soft count = 275\n",
      "--- gene=512, id=136 removed by soft count = 99\n",
      "--- gene=512, id=146 removed by soft count = 53\n",
      "--- gene=512, id=168 removed by soft count = 1020\n",
      "--- gene=512, id=185 removed by soft count = 51\n",
      "--- number of probes: 197 , kept by blast: 186 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 516\n",
      "--- gene=516, id=2 removed by soft count = 354\n",
      "--- gene=516, id=3 removed by soft count = 675\n",
      "--- gene=516, id=21 removed by soft count = 219\n",
      "--- gene=516, id=79 removed by soft count = 138\n",
      "--- gene=516, id=81 removed by soft count = 135\n",
      "--- gene=516, id=107 removed by soft count = 216\n",
      "--- gene=516, id=222 removed by soft count = 56\n",
      "--- gene=516, id=226 removed by soft count = 52\n",
      "--- number of probes: 227 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 517\n",
      "--- gene=517, id=104 removed by soft count = 232\n",
      "--- gene=517, id=136 removed by soft count = 80\n",
      "--- gene=517, id=157 removed by soft count = 153\n",
      "--- gene=517, id=168 removed by soft count = 233\n",
      "--- gene=517, id=169 removed by soft count = 51\n",
      "--- gene=517, id=170 removed by soft count = 130\n",
      "--- gene=517, id=171 removed by soft count = 56\n",
      "--- number of probes: 248 , kept by blast: 241 , if remove dups: 241\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 518\n",
      "--- gene=518, id=55 removed by soft count = 60\n",
      "--- gene=518, id=65 removed by soft count = 220\n",
      "--- gene=518, id=66 removed by soft count = 115\n",
      "--- gene=518, id=111 removed by soft count = 240\n",
      "--- gene=518, id=112 removed by soft count = 223\n",
      "--- gene=518, id=153 removed by soft count = 199\n",
      "--- gene=518, id=156 removed by soft count = 56\n",
      "--- gene=518, id=162 removed by soft count = 94\n",
      "--- gene=518, id=165 removed by soft count = 133\n",
      "--- gene=518, id=179 removed by soft count = 180\n",
      "--- gene=518, id=182 removed by soft count = 691\n",
      "--- number of probes: 264 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 519\n",
      "--- gene=519, id=67 removed by soft count = 181\n",
      "--- gene=519, id=68 removed by soft count = 22325\n",
      "--- gene=519, id=117 removed by soft count = 126\n",
      "--- gene=519, id=129 removed by soft count = 93\n",
      "--- gene=519, id=130 removed by soft count = 194\n",
      "--- gene=519, id=131 removed by soft count = 153\n",
      "--- gene=519, id=132 removed by soft count = 322\n",
      "--- gene=519, id=134 removed by soft count = 274\n",
      "--- gene=519, id=135 removed by soft count = 178\n",
      "--- gene=519, id=137 removed by soft count = 944\n",
      "--- gene=519, id=138 removed by soft count = 1336\n",
      "--- number of probes: 188 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 520\n",
      "--- gene=520, id=187 removed by soft count = 5861\n",
      "--- number of probes: 212 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 521\n",
      "--- number of probes: 265 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 522\n",
      "--- gene=522, id=0 removed by soft count = 260\n",
      "--- gene=522, id=33 removed by soft count = 118\n",
      "--- gene=522, id=49 removed by soft count = 196\n",
      "--- gene=522, id=136 removed by soft count = 130\n",
      "--- gene=522, id=172 removed by soft count = 72\n",
      "--- gene=522, id=174 removed by soft count = 65\n",
      "--- gene=522, id=178 removed by soft count = 70\n",
      "--- number of probes: 187 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 523\n",
      "--- gene=523, id=134 removed by soft count = 85\n",
      "--- number of probes: 152 , kept by blast: 151 , if remove dups: 151\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 524\n",
      "--- gene=524, id=149 removed by soft count = 152\n",
      "--- gene=524, id=152 removed by soft count = 122\n",
      "--- gene=524, id=155 removed by soft count = 78\n",
      "--- gene=524, id=157 removed by soft count = 67\n",
      "--- number of probes: 196 , kept by blast: 192 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 525\n",
      "--- gene=525, id=54 removed by soft count = 76\n",
      "--- gene=525, id=105 removed by soft count = 74\n",
      "--- gene=525, id=124 removed by soft count = 94\n",
      "--- gene=525, id=126 removed by soft count = 66\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=525, id=127 removed by soft count = 329\n",
      "--- number of probes: 216 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 526\n",
      "--- gene=526, id=134 removed by soft count = 190\n",
      "--- gene=526, id=150 removed by soft count = 163\n",
      "--- gene=526, id=151 removed by soft count = 9041\n",
      "--- gene=526, id=152 removed by soft count = 3819\n",
      "--- gene=526, id=153 removed by soft count = 524\n",
      "--- gene=526, id=154 removed by soft count = 272\n",
      "--- number of probes: 223 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 527\n",
      "--- gene=527, id=54 removed by soft count = 55\n",
      "--- gene=527, id=66 removed by soft count = 69\n",
      "--- gene=527, id=67 removed by soft count = 146\n",
      "--- gene=527, id=93 removed by soft count = 73\n",
      "--- gene=527, id=210 removed by soft count = 64\n",
      "--- gene=527, id=211 removed by soft count = 138\n",
      "--- gene=527, id=252 removed by soft count = 1998\n",
      "--- gene=527, id=254 removed by soft count = 486\n",
      "--- number of probes: 255 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 528\n",
      "--- gene=528, id=41 removed by soft count = 111\n",
      "--- gene=528, id=81 removed by soft count = 54\n",
      "--- gene=528, id=217 removed by soft count = 68\n",
      "--- number of probes: 249 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 529\n",
      "--- gene=529, id=122 removed by soft count = 80\n",
      "--- gene=529, id=149 removed by soft count = 74\n",
      "--- gene=529, id=150 removed by soft count = 166\n",
      "--- gene=529, id=152 removed by soft count = 91\n",
      "--- number of probes: 269 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 530\n",
      "--- gene=530, id=92 removed by soft count = 276\n",
      "--- gene=530, id=149 removed by soft count = 1775\n",
      "--- gene=530, id=150 removed by soft count = 92\n",
      "--- gene=530, id=200 removed by soft count = 377\n",
      "--- number of probes: 223 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 531\n",
      "--- number of probes: 201 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 532\n",
      "--- gene=532, id=1 removed by soft count = 56\n",
      "--- gene=532, id=2 removed by soft count = 63\n",
      "--- gene=532, id=178 removed by soft count = 52\n",
      "--- gene=532, id=195 removed by soft count = 144\n",
      "--- gene=532, id=196 removed by soft count = 301\n",
      "--- gene=532, id=198 removed by soft count = 251\n",
      "--- gene=532, id=201 removed by soft count = 155\n",
      "--- gene=532, id=202 removed by soft count = 105\n",
      "--- gene=532, id=203 removed by soft count = 148\n",
      "--- number of probes: 232 , kept by blast: 223 , if remove dups: 223\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 533\n",
      "--- gene=533, id=6 removed by soft count = 270\n",
      "--- gene=533, id=41 removed by soft count = 138\n",
      "--- gene=533, id=42 removed by soft count = 192\n",
      "--- gene=533, id=45 removed by soft count = 209\n",
      "--- gene=533, id=46 removed by soft count = 75\n",
      "--- gene=533, id=51 removed by soft count = 107\n",
      "--- gene=533, id=53 removed by soft count = 98\n",
      "--- gene=533, id=54 removed by soft count = 54\n",
      "--- gene=533, id=62 removed by soft count = 103\n",
      "--- gene=533, id=64 removed by soft count = 54\n",
      "--- gene=533, id=66 removed by soft count = 59\n",
      "--- gene=533, id=68 removed by soft count = 61\n",
      "--- gene=533, id=127 removed by soft count = 1119\n",
      "--- gene=533, id=169 removed by soft count = 58\n",
      "--- gene=533, id=173 removed by soft count = 121\n",
      "--- gene=533, id=174 removed by soft count = 165\n",
      "--- gene=533, id=175 removed by soft count = 154\n",
      "--- gene=533, id=176 removed by soft count = 58\n",
      "--- gene=533, id=177 removed by soft count = 233\n",
      "--- gene=533, id=178 removed by soft count = 4482\n",
      "--- gene=533, id=179 removed by soft count = 761\n",
      "--- number of probes: 201 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 534\n",
      "--- gene=534, id=73 removed by soft count = 561\n",
      "--- gene=534, id=74 removed by soft count = 443\n",
      "--- gene=534, id=75 removed by soft count = 176\n",
      "--- gene=534, id=76 removed by soft count = 7913\n",
      "--- number of probes: 181 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 535\n",
      "--- gene=535, id=20 removed by soft count = 89\n",
      "--- gene=535, id=23 removed by soft count = 73\n",
      "--- gene=535, id=24 removed by soft count = 56\n",
      "--- gene=535, id=32 removed by soft count = 54\n",
      "--- number of probes: 116 , kept by blast: 112 , if remove dups: 112\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 112\n",
      "-- checking probes in region: 536\n",
      "--- gene=536, id=22 removed by soft count = 155\n",
      "--- gene=536, id=73 removed by soft count = 312\n",
      "--- gene=536, id=76 removed by soft count = 382\n",
      "--- gene=536, id=97 removed by soft count = 324\n",
      "--- gene=536, id=98 removed by soft count = 348\n",
      "--- gene=536, id=99 removed by soft count = 18666\n",
      "--- gene=536, id=242 removed by soft count = 77\n",
      "--- number of probes: 259 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 537\n",
      "--- gene=537, id=0 removed by soft count = 396\n",
      "--- gene=537, id=1 removed by soft count = 76\n",
      "--- gene=537, id=2 removed by soft count = 77\n",
      "--- gene=537, id=4 removed by soft count = 105\n",
      "--- gene=537, id=94 removed by soft count = 126\n",
      "--- gene=537, id=165 removed by soft count = 65\n",
      "--- number of probes: 231 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 538\n",
      "--- gene=538, id=116 removed by soft count = 82\n",
      "--- gene=538, id=132 removed by soft count = 61\n",
      "--- gene=538, id=133 removed by soft count = 68\n",
      "--- gene=538, id=135 removed by soft count = 93\n",
      "--- gene=538, id=139 removed by soft count = 450\n",
      "--- gene=538, id=140 removed by soft count = 122\n",
      "--- gene=538, id=144 removed by soft count = 135\n",
      "--- gene=538, id=204 removed by soft count = 306\n",
      "--- number of probes: 214 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 539\n",
      "--- gene=539, id=18 removed by soft count = 71\n",
      "--- gene=539, id=28 removed by soft count = 104\n",
      "--- gene=539, id=30 removed by soft count = 59\n",
      "--- gene=539, id=229 removed by soft count = 1088\n",
      "--- gene=539, id=230 removed by soft count = 96\n",
      "--- number of probes: 242 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 540\n",
      "--- gene=540, id=42 removed by soft count = 464\n",
      "--- gene=540, id=43 removed by soft count = 82\n",
      "--- gene=540, id=45 removed by soft count = 348\n",
      "--- gene=540, id=61 removed by soft count = 81\n",
      "--- gene=540, id=63 removed by soft count = 105\n",
      "--- gene=540, id=64 removed by soft count = 77\n",
      "--- gene=540, id=66 removed by soft count = 94\n",
      "--- gene=540, id=71 removed by soft count = 73\n",
      "--- gene=540, id=109 removed by soft count = 145\n",
      "--- number of probes: 179 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 541\n",
      "--- gene=541, id=173 removed by soft count = 193\n",
      "--- gene=541, id=225 removed by soft count = 129\n",
      "--- gene=541, id=226 removed by soft count = 54\n",
      "--- gene=541, id=244 removed by soft count = 65\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 248 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 542\n",
      "--- gene=542, id=53 removed by soft count = 120\n",
      "--- gene=542, id=67 removed by hard count = 3\n",
      "--- gene=542, id=184 removed by soft count = 124\n",
      "--- gene=542, id=188 removed by soft count = 109\n",
      "--- number of probes: 227 , kept by blast: 223 , if remove dups: 222\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 543\n",
      "--- gene=543, id=5 removed by soft count = 66\n",
      "--- gene=543, id=236 removed by soft count = 977\n",
      "--- number of probes: 302 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 544\n",
      "--- gene=544, id=3 removed by soft count = 102\n",
      "--- gene=544, id=13 removed by soft count = 109\n",
      "--- gene=544, id=15 removed by soft count = 221\n",
      "--- gene=544, id=36 removed by soft count = 426\n",
      "--- gene=544, id=164 removed by soft count = 697\n",
      "--- gene=544, id=178 removed by soft count = 617\n",
      "--- gene=544, id=183 removed by soft count = 887\n",
      "--- gene=544, id=258 removed by soft count = 58\n",
      "--- gene=544, id=259 removed by soft count = 231\n",
      "--- gene=544, id=272 removed by soft count = 54\n",
      "--- gene=544, id=273 removed by soft count = 86\n",
      "--- number of probes: 276 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 545\n",
      "--- gene=545, id=156 removed by soft count = 451\n",
      "--- gene=545, id=182 removed by soft count = 394\n",
      "--- number of probes: 267 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 546\n",
      "--- gene=546, id=130 removed by soft count = 329\n",
      "--- gene=546, id=131 removed by soft count = 170\n",
      "--- number of probes: 305 , kept by blast: 303 , if remove dups: 303\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 548\n",
      "--- gene=548, id=46 removed by soft count = 427\n",
      "--- gene=548, id=59 removed by soft count = 969\n",
      "--- gene=548, id=60 removed by soft count = 530\n",
      "--- gene=548, id=61 removed by soft count = 720\n",
      "--- gene=548, id=62 removed by soft count = 565\n",
      "--- gene=548, id=63 removed by soft count = 221\n",
      "--- gene=548, id=64 removed by soft count = 840\n",
      "--- gene=548, id=65 removed by soft count = 1103\n",
      "--- gene=548, id=150 removed by soft count = 58\n",
      "--- gene=548, id=151 removed by soft count = 428\n",
      "--- gene=548, id=152 removed by soft count = 272\n",
      "--- number of probes: 191 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 549\n",
      "--- gene=549, id=47 removed by soft count = 113\n",
      "--- gene=549, id=48 removed by soft count = 2229\n",
      "--- gene=549, id=50 removed by soft count = 862\n",
      "--- gene=549, id=53 removed by soft count = 143\n",
      "--- gene=549, id=105 removed by soft count = 123\n",
      "--- gene=549, id=108 removed by soft count = 125\n",
      "--- gene=549, id=147 removed by soft count = 93\n",
      "--- gene=549, id=151 removed by soft count = 974\n",
      "--- gene=549, id=173 removed by soft count = 114\n",
      "--- gene=549, id=175 removed by soft count = 175\n",
      "--- number of probes: 206 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 550\n",
      "--- number of probes: 378 , kept by blast: 378 , if remove dups: 378\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 551\n",
      "--- number of probes: 90 , kept by blast: 90 , if remove dups: 88\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 88\n",
      "-- checking probes in region: 552\n",
      "--- gene=552, id=34 removed by soft count = 56\n",
      "--- gene=552, id=37 removed by soft count = 63\n",
      "--- gene=552, id=39 removed by hard count = 0\n",
      "--- gene=552, id=87 removed by soft count = 258\n",
      "--- gene=552, id=89 removed by soft count = 52\n",
      "--- gene=552, id=90 removed by soft count = 1019\n",
      "--- gene=552, id=94 removed by soft count = 84\n",
      "--- gene=552, id=95 removed by soft count = 152\n",
      "--- gene=552, id=96 removed by soft count = 218\n",
      "--- gene=552, id=169 removed by soft count = 55\n",
      "--- gene=552, id=209 removed by soft count = 61\n",
      "--- number of probes: 264 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 554\n",
      "--- gene=554, id=14 removed by soft count = 418\n",
      "--- gene=554, id=89 removed by soft count = 81\n",
      "--- gene=554, id=91 removed by soft count = 57\n",
      "--- gene=554, id=93 removed by soft count = 128\n",
      "--- gene=554, id=275 removed by soft count = 75\n",
      "--- gene=554, id=278 removed by soft count = 141\n",
      "--- gene=554, id=286 removed by soft count = 204\n",
      "--- number of probes: 306 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 555\n",
      "--- gene=555, id=49 removed by soft count = 190\n",
      "--- gene=555, id=113 removed by soft count = 383\n",
      "--- gene=555, id=136 removed by soft count = 318\n",
      "--- gene=555, id=141 removed by soft count = 212\n",
      "--- gene=555, id=142 removed by soft count = 301\n",
      "--- gene=555, id=171 removed by soft count = 166\n",
      "--- gene=555, id=172 removed by soft count = 84\n",
      "--- gene=555, id=175 removed by soft count = 170\n",
      "--- gene=555, id=203 removed by soft count = 178\n",
      "--- gene=555, id=204 removed by soft count = 99\n",
      "--- gene=555, id=211 removed by soft count = 1090\n",
      "--- gene=555, id=212 removed by soft count = 229\n",
      "--- gene=555, id=213 removed by soft count = 313\n",
      "--- gene=555, id=234 removed by soft count = 253\n",
      "--- gene=555, id=235 removed by soft count = 77\n",
      "--- gene=555, id=236 removed by soft count = 267\n",
      "--- number of probes: 241 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 556\n",
      "--- gene=556, id=34 removed by soft count = 69\n",
      "--- gene=556, id=42 removed by soft count = 56\n",
      "--- gene=556, id=44 removed by soft count = 64\n",
      "--- gene=556, id=87 removed by soft count = 350\n",
      "--- number of probes: 234 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 557\n",
      "--- gene=557, id=47 removed by soft count = 200\n",
      "--- gene=557, id=57 removed by soft count = 675\n",
      "--- gene=557, id=58 removed by soft count = 323\n",
      "--- gene=557, id=161 removed by soft count = 433\n",
      "--- gene=557, id=162 removed by soft count = 119\n",
      "--- number of probes: 273 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 558\n",
      "--- gene=558, id=12 removed by soft count = 64\n",
      "--- gene=558, id=117 removed by soft count = 807\n",
      "--- gene=558, id=120 removed by soft count = 239\n",
      "--- gene=558, id=121 removed by soft count = 407\n",
      "--- gene=558, id=131 removed by soft count = 224\n",
      "--- gene=558, id=156 removed by soft count = 64\n",
      "--- gene=558, id=213 removed by soft count = 52\n",
      "--- gene=558, id=234 removed by soft count = 81\n",
      "--- gene=558, id=250 removed by soft count = 980\n",
      "--- number of probes: 251 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 559\n",
      "--- gene=559, id=80 removed by soft count = 52\n",
      "--- gene=559, id=86 removed by soft count = 96\n",
      "--- gene=559, id=113 removed by soft count = 1746\n",
      "--- gene=559, id=217 removed by soft count = 87\n",
      "--- number of probes: 221 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 560\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=560, id=104 removed by soft count = 409\n",
      "--- gene=560, id=140 removed by soft count = 65\n",
      "--- gene=560, id=199 removed by soft count = 62\n",
      "--- gene=560, id=216 removed by soft count = 129\n",
      "--- number of probes: 261 , kept by blast: 257 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 561\n",
      "--- gene=561, id=39 removed by soft count = 1343\n",
      "--- gene=561, id=41 removed by soft count = 8743\n",
      "--- gene=561, id=42 removed by soft count = 304\n",
      "--- gene=561, id=171 removed by soft count = 60\n",
      "--- number of probes: 240 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 562\n",
      "--- gene=562, id=24 removed by soft count = 1444\n",
      "--- gene=562, id=137 removed by soft count = 54\n",
      "--- gene=562, id=139 removed by soft count = 53\n",
      "--- gene=562, id=182 removed by soft count = 102\n",
      "--- gene=562, id=234 removed by soft count = 67\n",
      "--- number of probes: 249 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 563\n",
      "--- gene=563, id=101 removed by soft count = 96\n",
      "--- gene=563, id=111 removed by soft count = 352\n",
      "--- gene=563, id=192 removed by soft count = 170\n",
      "--- number of probes: 193 , kept by blast: 190 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 564\n",
      "--- gene=564, id=225 removed by soft count = 67\n",
      "--- gene=564, id=226 removed by soft count = 52\n",
      "--- number of probes: 270 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 565\n",
      "--- gene=565, id=0 removed by soft count = 811\n",
      "--- gene=565, id=7 removed by soft count = 558\n",
      "--- gene=565, id=45 removed by soft count = 103\n",
      "--- gene=565, id=152 removed by soft count = 321\n",
      "--- gene=565, id=153 removed by soft count = 83\n",
      "--- gene=565, id=157 removed by soft count = 95\n",
      "--- gene=565, id=186 removed by soft count = 67\n",
      "--- gene=565, id=197 removed by soft count = 420\n",
      "--- gene=565, id=207 removed by soft count = 789\n",
      "--- gene=565, id=208 removed by soft count = 67\n",
      "--- gene=565, id=211 removed by soft count = 323\n",
      "--- number of probes: 218 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 566\n",
      "--- gene=566, id=2 removed by soft count = 82\n",
      "--- gene=566, id=53 removed by soft count = 283\n",
      "--- gene=566, id=103 removed by soft count = 374\n",
      "--- gene=566, id=104 removed by soft count = 891\n",
      "--- gene=566, id=105 removed by soft count = 196\n",
      "--- gene=566, id=106 removed by soft count = 440\n",
      "--- gene=566, id=107 removed by soft count = 56\n",
      "--- gene=566, id=108 removed by soft count = 77\n",
      "--- gene=566, id=109 removed by soft count = 277\n",
      "--- gene=566, id=135 removed by soft count = 66\n",
      "--- gene=566, id=161 removed by soft count = 218\n",
      "--- gene=566, id=162 removed by soft count = 172\n",
      "--- number of probes: 169 , kept by blast: 157 , if remove dups: 157\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 567\n",
      "--- gene=567, id=87 removed by soft count = 75\n",
      "--- gene=567, id=135 removed by soft count = 71\n",
      "--- gene=567, id=136 removed by soft count = 97\n",
      "--- gene=567, id=182 removed by soft count = 152\n",
      "--- gene=567, id=183 removed by soft count = 99\n",
      "--- gene=567, id=185 removed by soft count = 83\n",
      "--- gene=567, id=186 removed by soft count = 68\n",
      "--- gene=567, id=201 removed by soft count = 66\n",
      "--- gene=567, id=202 removed by soft count = 86\n",
      "--- gene=567, id=205 removed by soft count = 52\n",
      "--- gene=567, id=206 removed by soft count = 128\n",
      "--- gene=567, id=212 removed by soft count = 174\n",
      "--- gene=567, id=215 removed by soft count = 97\n",
      "--- gene=567, id=218 removed by soft count = 90\n",
      "--- number of probes: 229 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 568\n",
      "--- gene=568, id=13 removed by soft count = 225\n",
      "--- gene=568, id=34 removed by soft count = 69\n",
      "--- gene=568, id=36 removed by soft count = 256\n",
      "--- gene=568, id=38 removed by soft count = 145\n",
      "--- gene=568, id=39 removed by soft count = 230\n",
      "--- gene=568, id=43 removed by soft count = 71\n",
      "--- gene=568, id=45 removed by soft count = 74\n",
      "--- gene=568, id=87 removed by soft count = 99\n",
      "--- gene=568, id=183 removed by soft count = 771\n",
      "--- gene=568, id=215 removed by soft count = 87\n",
      "--- gene=568, id=217 removed by soft count = 316\n",
      "--- number of probes: 222 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 569\n",
      "--- gene=569, id=38 removed by soft count = 283\n",
      "--- gene=569, id=45 removed by soft count = 54\n",
      "--- gene=569, id=49 removed by soft count = 125\n",
      "--- gene=569, id=51 removed by soft count = 170\n",
      "--- gene=569, id=66 removed by soft count = 54\n",
      "--- gene=569, id=96 removed by soft count = 296\n",
      "--- gene=569, id=199 removed by soft count = 1031\n",
      "--- gene=569, id=221 removed by soft count = 62\n",
      "--- number of probes: 292 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 57\n",
      "--- gene=57, id=50 removed by soft count = 311\n",
      "--- number of probes: 60 , kept by blast: 59 , if remove dups: 59\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: 570\n",
      "--- gene=570, id=1 removed by soft count = 58\n",
      "--- gene=570, id=2 removed by soft count = 901\n",
      "--- gene=570, id=3 removed by soft count = 256\n",
      "--- gene=570, id=5 removed by soft count = 388\n",
      "--- gene=570, id=6 removed by soft count = 599\n",
      "--- gene=570, id=8 removed by soft count = 140\n",
      "--- gene=570, id=9 removed by soft count = 86\n",
      "--- gene=570, id=10 removed by soft count = 242\n",
      "--- gene=570, id=11 removed by soft count = 63\n",
      "--- gene=570, id=14 removed by soft count = 64\n",
      "--- gene=570, id=16 removed by soft count = 1310\n",
      "--- gene=570, id=18 removed by soft count = 108\n",
      "--- gene=570, id=19 removed by soft count = 329\n",
      "--- gene=570, id=25 removed by soft count = 127\n",
      "--- gene=570, id=26 removed by soft count = 369\n",
      "--- gene=570, id=27 removed by soft count = 112\n",
      "--- gene=570, id=30 removed by soft count = 138\n",
      "--- gene=570, id=32 removed by soft count = 181\n",
      "--- gene=570, id=33 removed by soft count = 88\n",
      "--- gene=570, id=35 removed by soft count = 98\n",
      "--- gene=570, id=38 removed by soft count = 79\n",
      "--- gene=570, id=41 removed by soft count = 517\n",
      "--- gene=570, id=42 removed by soft count = 96\n",
      "--- gene=570, id=43 removed by soft count = 64\n",
      "--- gene=570, id=44 removed by soft count = 128\n",
      "--- gene=570, id=45 removed by soft count = 661\n",
      "--- gene=570, id=48 removed by soft count = 150\n",
      "--- gene=570, id=49 removed by soft count = 56\n",
      "--- gene=570, id=50 removed by soft count = 191\n",
      "--- gene=570, id=51 removed by soft count = 73\n",
      "--- gene=570, id=52 removed by soft count = 101\n",
      "--- gene=570, id=53 removed by soft count = 331\n",
      "--- gene=570, id=54 removed by soft count = 147\n",
      "--- gene=570, id=55 removed by soft count = 1233\n",
      "--- gene=570, id=56 removed by soft count = 67\n",
      "--- gene=570, id=57 removed by soft count = 661\n",
      "--- gene=570, id=58 removed by soft count = 78\n",
      "--- gene=570, id=59 removed by soft count = 59\n",
      "--- gene=570, id=60 removed by soft count = 796\n",
      "--- gene=570, id=61 removed by soft count = 574\n",
      "--- gene=570, id=62 removed by soft count = 57\n",
      "--- gene=570, id=65 removed by soft count = 292\n",
      "--- gene=570, id=68 removed by soft count = 100\n",
      "--- gene=570, id=69 removed by soft count = 287\n",
      "--- gene=570, id=70 removed by soft count = 444\n",
      "--- gene=570, id=71 removed by soft count = 580\n",
      "--- gene=570, id=72 removed by soft count = 253\n",
      "--- gene=570, id=74 removed by soft count = 527\n",
      "--- gene=570, id=76 removed by soft count = 148\n",
      "--- gene=570, id=78 removed by soft count = 110\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=570, id=79 removed by soft count = 354\n",
      "--- gene=570, id=81 removed by soft count = 74\n",
      "--- gene=570, id=83 removed by soft count = 180\n",
      "--- gene=570, id=86 removed by soft count = 86\n",
      "--- gene=570, id=90 removed by soft count = 1107\n",
      "--- gene=570, id=93 removed by soft count = 416\n",
      "--- gene=570, id=97 removed by soft count = 79\n",
      "--- gene=570, id=101 removed by soft count = 52\n",
      "--- number of probes: 104 , kept by blast: 46 , if remove dups: 46\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: 571\n",
      "--- gene=571, id=6 removed by soft count = 64\n",
      "--- gene=571, id=17 removed by soft count = 114\n",
      "--- gene=571, id=22 removed by soft count = 56\n",
      "--- gene=571, id=28 removed by soft count = 101\n",
      "--- gene=571, id=34 removed by soft count = 54\n",
      "--- gene=571, id=35 removed by soft count = 174\n",
      "--- gene=571, id=45 removed by soft count = 114\n",
      "--- gene=571, id=53 removed by soft count = 56\n",
      "--- gene=571, id=55 removed by soft count = 288\n",
      "--- gene=571, id=58 removed by soft count = 127\n",
      "--- gene=571, id=59 removed by soft count = 115\n",
      "--- gene=571, id=60 removed by soft count = 73\n",
      "--- gene=571, id=61 removed by soft count = 218\n",
      "--- gene=571, id=64 removed by soft count = 95\n",
      "--- gene=571, id=66 removed by soft count = 90\n",
      "--- gene=571, id=68 removed by soft count = 145\n",
      "--- gene=571, id=76 removed by soft count = 184\n",
      "--- gene=571, id=77 removed by soft count = 121\n",
      "--- gene=571, id=87 removed by soft count = 167\n",
      "--- gene=571, id=88 removed by soft count = 92\n",
      "--- gene=571, id=90 removed by soft count = 57\n",
      "--- gene=571, id=94 removed by soft count = 79\n",
      "--- gene=571, id=95 removed by soft count = 130\n",
      "--- gene=571, id=97 removed by soft count = 53\n",
      "--- gene=571, id=106 removed by soft count = 188\n",
      "--- gene=571, id=110 removed by soft count = 221\n",
      "--- gene=571, id=111 removed by soft count = 98\n",
      "--- gene=571, id=112 removed by soft count = 78\n",
      "--- gene=571, id=113 removed by soft count = 181\n",
      "--- gene=571, id=126 removed by soft count = 86\n",
      "--- gene=571, id=128 removed by soft count = 120\n",
      "--- gene=571, id=146 removed by soft count = 95\n",
      "--- gene=571, id=158 removed by soft count = 55\n",
      "--- gene=571, id=167 removed by soft count = 190\n",
      "--- gene=571, id=173 removed by soft count = 229\n",
      "--- gene=571, id=174 removed by soft count = 77\n",
      "--- gene=571, id=177 removed by soft count = 79\n",
      "--- gene=571, id=178 removed by soft count = 103\n",
      "--- gene=571, id=183 removed by soft count = 83\n",
      "--- gene=571, id=186 removed by soft count = 160\n",
      "--- gene=571, id=193 removed by soft count = 62\n",
      "--- gene=571, id=199 removed by soft count = 77\n",
      "--- gene=571, id=202 removed by soft count = 78\n",
      "--- gene=571, id=205 removed by soft count = 122\n",
      "--- gene=571, id=210 removed by soft count = 120\n",
      "--- gene=571, id=214 removed by soft count = 87\n",
      "--- gene=571, id=216 removed by soft count = 58\n",
      "--- gene=571, id=224 removed by soft count = 55\n",
      "--- gene=571, id=232 removed by soft count = 124\n",
      "--- gene=571, id=236 removed by soft count = 126\n",
      "--- gene=571, id=237 removed by soft count = 110\n",
      "--- gene=571, id=238 removed by soft count = 176\n",
      "--- gene=571, id=241 removed by soft count = 94\n",
      "--- gene=571, id=243 removed by soft count = 187\n",
      "--- gene=571, id=253 removed by soft count = 63\n",
      "--- gene=571, id=257 removed by soft count = 87\n",
      "--- gene=571, id=261 removed by soft count = 93\n",
      "--- gene=571, id=265 removed by soft count = 66\n",
      "--- gene=571, id=269 removed by soft count = 117\n",
      "--- gene=571, id=279 removed by soft count = 85\n",
      "--- gene=571, id=283 removed by soft count = 179\n",
      "--- gene=571, id=285 removed by soft count = 149\n",
      "--- gene=571, id=288 removed by soft count = 78\n",
      "--- gene=571, id=294 removed by soft count = 88\n",
      "--- gene=571, id=303 removed by soft count = 147\n",
      "--- gene=571, id=306 removed by soft count = 76\n",
      "--- gene=571, id=309 removed by soft count = 154\n",
      "--- gene=571, id=311 removed by soft count = 73\n",
      "--- gene=571, id=312 removed by soft count = 63\n",
      "--- gene=571, id=318 removed by soft count = 74\n",
      "--- gene=571, id=324 removed by soft count = 73\n",
      "--- gene=571, id=329 removed by soft count = 198\n",
      "--- gene=571, id=331 removed by soft count = 241\n",
      "--- gene=571, id=333 removed by soft count = 105\n",
      "--- gene=571, id=341 removed by soft count = 149\n",
      "--- gene=571, id=343 removed by soft count = 122\n",
      "--- gene=571, id=344 removed by soft count = 70\n",
      "--- gene=571, id=345 removed by soft count = 196\n",
      "--- gene=571, id=346 removed by soft count = 74\n",
      "--- gene=571, id=353 removed by soft count = 345\n",
      "--- gene=571, id=354 removed by soft count = 318\n",
      "--- gene=571, id=355 removed by soft count = 216\n",
      "--- number of probes: 356 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 572\n",
      "--- gene=572, id=14 removed by soft count = 208\n",
      "--- gene=572, id=15 removed by soft count = 62\n",
      "--- gene=572, id=107 removed by soft count = 105\n",
      "--- gene=572, id=123 removed by soft count = 77\n",
      "--- gene=572, id=125 removed by soft count = 69\n",
      "--- gene=572, id=126 removed by soft count = 72\n",
      "--- gene=572, id=133 removed by soft count = 53\n",
      "--- gene=572, id=138 removed by soft count = 154\n",
      "--- gene=572, id=147 removed by soft count = 77\n",
      "--- gene=572, id=194 removed by soft count = 52\n",
      "--- gene=572, id=243 removed by soft count = 113\n",
      "--- gene=572, id=252 removed by soft count = 51\n",
      "--- gene=572, id=253 removed by soft count = 55\n",
      "--- gene=572, id=256 removed by soft count = 71\n",
      "--- number of probes: 280 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 573\n",
      "--- gene=573, id=86 removed by soft count = 302\n",
      "--- gene=573, id=116 removed by soft count = 1527\n",
      "--- gene=573, id=192 removed by soft count = 110\n",
      "--- gene=573, id=202 removed by soft count = 125\n",
      "--- number of probes: 206 , kept by blast: 202 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 574\n",
      "--- gene=574, id=1 removed by soft count = 94\n",
      "--- gene=574, id=2 removed by soft count = 80\n",
      "--- gene=574, id=3 removed by soft count = 51\n",
      "--- gene=574, id=9 removed by soft count = 91\n",
      "--- gene=574, id=15 removed by soft count = 62\n",
      "--- gene=574, id=113 removed by soft count = 129\n",
      "--- gene=574, id=168 removed by soft count = 169\n",
      "--- gene=574, id=215 removed by soft count = 162\n",
      "--- gene=574, id=216 removed by soft count = 143\n",
      "--- gene=574, id=231 removed by soft count = 114\n",
      "--- gene=574, id=232 removed by soft count = 88\n",
      "--- gene=574, id=246 removed by soft count = 79\n",
      "--- gene=574, id=250 removed by soft count = 101\n",
      "--- gene=574, id=254 removed by soft count = 78\n",
      "--- gene=574, id=256 removed by soft count = 179\n",
      "--- number of probes: 258 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 575\n",
      "--- gene=575, id=1 removed by soft count = 138\n",
      "--- gene=575, id=2 removed by soft count = 178\n",
      "--- gene=575, id=52 removed by soft count = 63\n",
      "--- gene=575, id=104 removed by soft count = 117\n",
      "--- gene=575, id=143 removed by soft count = 57\n",
      "--- gene=575, id=169 removed by soft count = 255\n",
      "--- gene=575, id=200 removed by soft count = 238\n",
      "--- number of probes: 204 , kept by blast: 197 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 576\n",
      "--- gene=576, id=40 removed by soft count = 64\n",
      "--- gene=576, id=215 removed by soft count = 96\n",
      "--- gene=576, id=257 removed by soft count = 820\n",
      "--- gene=576, id=258 removed by soft count = 354\n",
      "--- number of probes: 291 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 577\n",
      "--- gene=577, id=41 removed by soft count = 78\n",
      "--- gene=577, id=86 removed by soft count = 283\n",
      "--- gene=577, id=89 removed by soft count = 776\n",
      "--- gene=577, id=105 removed by soft count = 70\n",
      "--- gene=577, id=170 removed by soft count = 209\n",
      "--- gene=577, id=174 removed by soft count = 56\n",
      "--- gene=577, id=201 removed by soft count = 126\n",
      "--- gene=577, id=203 removed by soft count = 53\n",
      "--- number of probes: 207 , kept by blast: 199 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 578\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=578, id=210 removed by soft count = 62\n",
      "--- gene=578, id=211 removed by soft count = 121\n",
      "--- gene=578, id=212 removed by soft count = 89\n",
      "--- gene=578, id=213 removed by soft count = 463\n",
      "--- gene=578, id=214 removed by soft count = 11773\n",
      "--- gene=578, id=215 removed by soft count = 176\n",
      "--- gene=578, id=216 removed by soft count = 364\n",
      "--- gene=578, id=227 removed by soft count = 66\n",
      "--- gene=578, id=233 removed by soft count = 115\n",
      "--- gene=578, id=235 removed by soft count = 99\n",
      "--- gene=578, id=236 removed by soft count = 151\n",
      "--- gene=578, id=241 removed by soft count = 87\n",
      "--- number of probes: 246 , kept by blast: 234 , if remove dups: 234\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 579\n",
      "--- gene=579, id=1 removed by soft count = 264\n",
      "--- gene=579, id=3 removed by soft count = 80\n",
      "--- gene=579, id=5 removed by soft count = 197\n",
      "--- gene=579, id=6 removed by soft count = 88\n",
      "--- gene=579, id=7 removed by soft count = 324\n",
      "--- gene=579, id=8 removed by soft count = 76\n",
      "--- gene=579, id=10 removed by soft count = 78\n",
      "--- gene=579, id=11 removed by soft count = 137\n",
      "--- gene=579, id=16 removed by soft count = 122\n",
      "--- gene=579, id=17 removed by soft count = 270\n",
      "--- gene=579, id=41 removed by soft count = 80\n",
      "--- gene=579, id=62 removed by soft count = 1690\n",
      "--- gene=579, id=63 removed by soft count = 231\n",
      "--- gene=579, id=69 removed by soft count = 136\n",
      "--- gene=579, id=70 removed by soft count = 328\n",
      "--- gene=579, id=72 removed by soft count = 378\n",
      "--- gene=579, id=78 removed by soft count = 225\n",
      "--- gene=579, id=82 removed by soft count = 187\n",
      "--- gene=579, id=90 removed by soft count = 126\n",
      "--- gene=579, id=131 removed by soft count = 666\n",
      "--- gene=579, id=147 removed by soft count = 344\n",
      "--- gene=579, id=148 removed by soft count = 94\n",
      "--- gene=579, id=150 removed by soft count = 510\n",
      "--- number of probes: 151 , kept by blast: 128 , if remove dups: 128\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 580\n",
      "--- gene=580, id=24 removed by soft count = 563\n",
      "--- gene=580, id=25 removed by soft count = 135\n",
      "--- gene=580, id=26 removed by soft count = 82\n",
      "--- gene=580, id=29 removed by soft count = 347\n",
      "--- gene=580, id=30 removed by soft count = 117\n",
      "--- gene=580, id=31 removed by soft count = 99\n",
      "--- gene=580, id=55 removed by soft count = 70\n",
      "--- number of probes: 202 , kept by blast: 195 , if remove dups: 195\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 581\n",
      "--- gene=581, id=10 removed by soft count = 6865\n",
      "--- gene=581, id=69 removed by soft count = 483\n",
      "--- gene=581, id=162 removed by soft count = 795\n",
      "--- number of probes: 250 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 582\n",
      "--- gene=582, id=108 removed by soft count = 58\n",
      "--- number of probes: 199 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 583\n",
      "--- gene=583, id=44 removed by soft count = 243\n",
      "--- gene=583, id=49 removed by soft count = 62\n",
      "--- gene=583, id=117 removed by soft count = 69\n",
      "--- gene=583, id=118 removed by soft count = 374\n",
      "--- gene=583, id=146 removed by soft count = 51\n",
      "--- gene=583, id=147 removed by soft count = 78\n",
      "--- gene=583, id=148 removed by soft count = 62\n",
      "--- gene=583, id=149 removed by soft count = 124\n",
      "--- gene=583, id=150 removed by soft count = 52\n",
      "--- gene=583, id=153 removed by soft count = 64\n",
      "--- gene=583, id=156 removed by soft count = 520\n",
      "--- gene=583, id=158 removed by soft count = 430\n",
      "--- number of probes: 181 , kept by blast: 169 , if remove dups: 168\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 584\n",
      "--- gene=584, id=56 removed by soft count = 300\n",
      "--- gene=584, id=58 removed by soft count = 194\n",
      "--- gene=584, id=64 removed by soft count = 103\n",
      "--- gene=584, id=104 removed by soft count = 121\n",
      "--- gene=584, id=107 removed by soft count = 183\n",
      "--- gene=584, id=108 removed by soft count = 186\n",
      "--- gene=584, id=109 removed by soft count = 203\n",
      "--- gene=584, id=110 removed by soft count = 12991\n",
      "--- gene=584, id=113 removed by soft count = 677\n",
      "--- gene=584, id=115 removed by soft count = 85\n",
      "--- gene=584, id=118 removed by soft count = 51\n",
      "--- gene=584, id=119 removed by soft count = 57\n",
      "--- gene=584, id=129 removed by soft count = 64\n",
      "--- number of probes: 139 , kept by blast: 126 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 585\n",
      "--- gene=585, id=26 removed by soft count = 143\n",
      "--- gene=585, id=71 removed by soft count = 330\n",
      "--- gene=585, id=103 removed by soft count = 139\n",
      "--- gene=585, id=105 removed by soft count = 101\n",
      "--- gene=585, id=106 removed by soft count = 134\n",
      "--- gene=585, id=108 removed by soft count = 125\n",
      "--- gene=585, id=109 removed by soft count = 116\n",
      "--- gene=585, id=110 removed by soft count = 80\n",
      "--- gene=585, id=111 removed by soft count = 54\n",
      "--- gene=585, id=112 removed by soft count = 84\n",
      "--- gene=585, id=113 removed by soft count = 57\n",
      "--- gene=585, id=115 removed by soft count = 71\n",
      "--- gene=585, id=117 removed by soft count = 58\n",
      "--- gene=585, id=191 removed by soft count = 105\n",
      "--- number of probes: 241 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 586\n",
      "--- gene=586, id=36 removed by soft count = 388\n",
      "--- gene=586, id=39 removed by soft count = 85\n",
      "--- gene=586, id=91 removed by soft count = 113\n",
      "--- gene=586, id=156 removed by soft count = 107\n",
      "--- gene=586, id=213 removed by soft count = 89\n",
      "--- number of probes: 218 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 587\n",
      "--- gene=587, id=2 removed by soft count = 62\n",
      "--- gene=587, id=3 removed by soft count = 142\n",
      "--- gene=587, id=4 removed by soft count = 108\n",
      "--- gene=587, id=6 removed by soft count = 120\n",
      "--- gene=587, id=9 removed by soft count = 86\n",
      "--- gene=587, id=13 removed by soft count = 729\n",
      "--- gene=587, id=29 removed by soft count = 291\n",
      "--- gene=587, id=38 removed by soft count = 82\n",
      "--- gene=587, id=39 removed by soft count = 109\n",
      "--- gene=587, id=44 removed by soft count = 1491\n",
      "--- gene=587, id=46 removed by soft count = 291\n",
      "--- gene=587, id=49 removed by soft count = 382\n",
      "--- gene=587, id=50 removed by soft count = 51\n",
      "--- gene=587, id=55 removed by soft count = 56\n",
      "--- gene=587, id=56 removed by soft count = 283\n",
      "--- gene=587, id=61 removed by soft count = 69\n",
      "--- gene=587, id=67 removed by soft count = 107\n",
      "--- gene=587, id=143 removed by soft count = 163\n",
      "--- gene=587, id=148 removed by soft count = 55\n",
      "--- gene=587, id=149 removed by soft count = 57\n",
      "--- gene=587, id=153 removed by soft count = 67\n",
      "--- gene=587, id=155 removed by soft count = 134\n",
      "--- number of probes: 156 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 588\n",
      "--- gene=588, id=36 removed by soft count = 200\n",
      "--- gene=588, id=37 removed by soft count = 386\n",
      "--- gene=588, id=56 removed by soft count = 528\n",
      "--- gene=588, id=57 removed by soft count = 751\n",
      "--- gene=588, id=58 removed by soft count = 385\n",
      "--- gene=588, id=59 removed by soft count = 290\n",
      "--- gene=588, id=97 removed by soft count = 179\n",
      "--- gene=588, id=100 removed by soft count = 58\n",
      "--- gene=588, id=116 removed by soft count = 108\n",
      "--- gene=588, id=135 removed by soft count = 54\n",
      "--- number of probes: 177 , kept by blast: 167 , if remove dups: 167\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 589\n",
      "--- gene=589, id=224 removed by soft count = 73\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 275 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 590\n",
      "--- gene=590, id=88 removed by soft count = 68\n",
      "--- gene=590, id=114 removed by soft count = 86\n",
      "--- gene=590, id=143 removed by soft count = 117\n",
      "--- gene=590, id=144 removed by soft count = 98\n",
      "--- gene=590, id=148 removed by soft count = 914\n",
      "--- gene=590, id=149 removed by soft count = 388\n",
      "--- gene=590, id=164 removed by soft count = 57\n",
      "--- number of probes: 225 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 591\n",
      "--- number of probes: 283 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 592\n",
      "--- gene=592, id=26 removed by soft count = 3229\n",
      "--- gene=592, id=33 removed by soft count = 151\n",
      "--- gene=592, id=40 removed by soft count = 841\n",
      "--- gene=592, id=67 removed by soft count = 187\n",
      "--- gene=592, id=68 removed by soft count = 2835\n",
      "--- gene=592, id=105 removed by soft count = 6940\n",
      "--- gene=592, id=106 removed by soft count = 926\n",
      "--- gene=592, id=107 removed by soft count = 433\n",
      "--- gene=592, id=108 removed by soft count = 314\n",
      "--- gene=592, id=111 removed by soft count = 296\n",
      "--- number of probes: 144 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 593\n",
      "--- gene=593, id=30 removed by soft count = 86\n",
      "--- gene=593, id=54 removed by soft count = 65\n",
      "--- gene=593, id=109 removed by soft count = 145\n",
      "--- gene=593, id=164 removed by soft count = 383\n",
      "--- gene=593, id=186 removed by soft count = 56\n",
      "--- gene=593, id=187 removed by soft count = 60\n",
      "--- gene=593, id=198 removed by soft count = 180\n",
      "--- number of probes: 306 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 594\n",
      "--- gene=594, id=83 removed by soft count = 119\n",
      "--- gene=594, id=84 removed by soft count = 241\n",
      "--- gene=594, id=85 removed by soft count = 817\n",
      "--- number of probes: 155 , kept by blast: 152 , if remove dups: 152\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 595\n",
      "--- gene=595, id=61 removed by soft count = 874\n",
      "--- gene=595, id=64 removed by soft count = 87\n",
      "--- gene=595, id=65 removed by soft count = 62\n",
      "--- gene=595, id=66 removed by soft count = 72\n",
      "--- gene=595, id=68 removed by soft count = 58\n",
      "--- gene=595, id=69 removed by soft count = 71\n",
      "--- gene=595, id=70 removed by soft count = 55\n",
      "--- gene=595, id=75 removed by soft count = 54\n",
      "--- gene=595, id=77 removed by soft count = 62\n",
      "--- gene=595, id=80 removed by soft count = 57\n",
      "--- gene=595, id=87 removed by soft count = 91\n",
      "--- gene=595, id=88 removed by soft count = 73\n",
      "--- gene=595, id=91 removed by soft count = 102\n",
      "--- gene=595, id=94 removed by soft count = 500\n",
      "--- gene=595, id=96 removed by soft count = 1814\n",
      "--- gene=595, id=133 removed by soft count = 642\n",
      "--- gene=595, id=134 removed by soft count = 388\n",
      "--- gene=595, id=135 removed by soft count = 382\n",
      "--- gene=595, id=136 removed by soft count = 233\n",
      "--- gene=595, id=137 removed by soft count = 394\n",
      "--- gene=595, id=138 removed by soft count = 176\n",
      "--- gene=595, id=139 removed by soft count = 858\n",
      "--- gene=595, id=141 removed by soft count = 295\n",
      "--- gene=595, id=142 removed by soft count = 135\n",
      "--- gene=595, id=144 removed by soft count = 75\n",
      "--- gene=595, id=146 removed by soft count = 63\n",
      "--- gene=595, id=148 removed by soft count = 90\n",
      "--- gene=595, id=177 removed by soft count = 60\n",
      "--- number of probes: 180 , kept by blast: 152 , if remove dups: 152\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 596\n",
      "--- gene=596, id=46 removed by soft count = 103\n",
      "--- gene=596, id=53 removed by soft count = 136\n",
      "--- gene=596, id=84 removed by soft count = 54\n",
      "--- gene=596, id=93 removed by soft count = 356\n",
      "--- gene=596, id=117 removed by soft count = 147\n",
      "--- gene=596, id=118 removed by soft count = 123\n",
      "--- gene=596, id=120 removed by soft count = 96\n",
      "--- gene=596, id=121 removed by soft count = 73\n",
      "--- gene=596, id=122 removed by soft count = 102\n",
      "--- gene=596, id=163 removed by soft count = 137\n",
      "--- gene=596, id=166 removed by soft count = 60\n",
      "--- number of probes: 254 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 597\n",
      "--- gene=597, id=0 removed by soft count = 121\n",
      "--- gene=597, id=106 removed by soft count = 431\n",
      "--- gene=597, id=108 removed by soft count = 60\n",
      "--- gene=597, id=109 removed by soft count = 112\n",
      "--- gene=597, id=132 removed by soft count = 139\n",
      "--- gene=597, id=147 removed by soft count = 13424\n",
      "--- gene=597, id=150 removed by soft count = 1163\n",
      "--- gene=597, id=212 removed by soft count = 89\n",
      "--- gene=597, id=266 removed by soft count = 70\n",
      "--- number of probes: 277 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 598\n",
      "--- gene=598, id=7 removed by soft count = 3075\n",
      "--- gene=598, id=158 removed by soft count = 174\n",
      "--- number of probes: 226 , kept by blast: 224 , if remove dups: 224\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 599\n",
      "--- gene=599, id=107 removed by soft count = 61\n",
      "--- number of probes: 304 , kept by blast: 303 , if remove dups: 303\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 6\n",
      "--- gene=6, id=53 removed by soft count = 123\n",
      "--- gene=6, id=85 removed by soft count = 128\n",
      "--- gene=6, id=264 removed by soft count = 110\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 60\n",
      "--- gene=60, id=59 removed by soft count = 76\n",
      "--- gene=60, id=68 removed by soft count = 864\n",
      "--- gene=60, id=69 removed by soft count = 265\n",
      "--- gene=60, id=79 removed by soft count = 60\n",
      "--- number of probes: 187 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 600\n",
      "--- gene=600, id=126 removed by soft count = 86\n",
      "--- gene=600, id=157 removed by soft count = 94\n",
      "--- gene=600, id=290 removed by soft count = 77\n",
      "--- number of probes: 294 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 601\n",
      "--- gene=601, id=7 removed by soft count = 1484\n",
      "--- gene=601, id=86 removed by soft count = 429\n",
      "--- gene=601, id=91 removed by soft count = 179\n",
      "--- gene=601, id=284 removed by soft count = 78\n",
      "--- number of probes: 299 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 602\n",
      "--- gene=602, id=63 removed by soft count = 75\n",
      "--- gene=602, id=141 removed by soft count = 571\n",
      "--- gene=602, id=168 removed by soft count = 58\n",
      "--- gene=602, id=170 removed by soft count = 198\n",
      "--- gene=602, id=174 removed by soft count = 79\n",
      "--- gene=602, id=182 removed by soft count = 160\n",
      "--- number of probes: 261 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 603\n",
      "--- gene=603, id=173 removed by soft count = 69\n",
      "--- gene=603, id=181 removed by soft count = 74\n",
      "--- gene=603, id=208 removed by soft count = 106\n",
      "--- number of probes: 251 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 605\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=605, id=0 removed by soft count = 3596\n",
      "--- gene=605, id=160 removed by soft count = 113\n",
      "--- number of probes: 187 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 606\n",
      "--- gene=606, id=56 removed by soft count = 81\n",
      "--- gene=606, id=113 removed by soft count = 76\n",
      "--- gene=606, id=149 removed by soft count = 84\n",
      "--- gene=606, id=167 removed by soft count = 71\n",
      "--- gene=606, id=175 removed by soft count = 73\n",
      "--- gene=606, id=184 removed by soft count = 373\n",
      "--- number of probes: 293 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 607\n",
      "--- gene=607, id=108 removed by soft count = 194\n",
      "--- gene=607, id=109 removed by soft count = 149\n",
      "--- gene=607, id=110 removed by soft count = 97\n",
      "--- gene=607, id=111 removed by soft count = 290\n",
      "--- gene=607, id=118 removed by soft count = 74\n",
      "--- gene=607, id=136 removed by soft count = 64\n",
      "--- number of probes: 281 , kept by blast: 275 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 608\n",
      "--- gene=608, id=85 removed by soft count = 438\n",
      "--- gene=608, id=86 removed by soft count = 155\n",
      "--- number of probes: 355 , kept by blast: 353 , if remove dups: 353\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 609\n",
      "--- gene=609, id=114 removed by soft count = 99\n",
      "--- gene=609, id=115 removed by soft count = 101\n",
      "--- number of probes: 308 , kept by blast: 306 , if remove dups: 306\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 611\n",
      "--- gene=611, id=144 removed by hard count = 5\n",
      "--- gene=611, id=145 removed by hard count = 5\n",
      "--- gene=611, id=249 removed by soft count = 238\n",
      "--- number of probes: 302 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 612\n",
      "--- gene=612, id=41 removed by soft count = 245\n",
      "--- gene=612, id=190 removed by soft count = 62\n",
      "--- gene=612, id=209 removed by soft count = 60\n",
      "--- number of probes: 311 , kept by blast: 308 , if remove dups: 308\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 613\n",
      "--- gene=613, id=22 removed by soft count = 257\n",
      "--- gene=613, id=148 removed by soft count = 166\n",
      "--- gene=613, id=168 removed by soft count = 211\n",
      "--- number of probes: 200 , kept by blast: 197 , if remove dups: 197\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 614\n",
      "--- gene=614, id=161 removed by soft count = 90\n",
      "--- gene=614, id=182 removed by soft count = 116\n",
      "--- gene=614, id=251 removed by soft count = 78\n",
      "--- number of probes: 277 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 615\n",
      "--- gene=615, id=47 removed by soft count = 85\n",
      "--- gene=615, id=49 removed by soft count = 127\n",
      "--- gene=615, id=100 removed by soft count = 62\n",
      "--- gene=615, id=103 removed by soft count = 326\n",
      "--- gene=615, id=104 removed by soft count = 132\n",
      "--- gene=615, id=105 removed by soft count = 672\n",
      "--- gene=615, id=106 removed by soft count = 326\n",
      "--- number of probes: 258 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 616\n",
      "--- gene=616, id=1 removed by soft count = 2796\n",
      "--- gene=616, id=2 removed by soft count = 319\n",
      "--- gene=616, id=3 removed by soft count = 210\n",
      "--- gene=616, id=4 removed by soft count = 307\n",
      "--- gene=616, id=5 removed by soft count = 1253\n",
      "--- gene=616, id=6 removed by soft count = 191\n",
      "--- gene=616, id=44 removed by soft count = 310\n",
      "--- gene=616, id=123 removed by soft count = 51\n",
      "--- gene=616, id=138 removed by soft count = 125\n",
      "--- gene=616, id=139 removed by soft count = 2447\n",
      "--- number of probes: 173 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 617\n",
      "--- gene=617, id=115 removed by soft count = 123\n",
      "--- gene=617, id=127 removed by soft count = 995\n",
      "--- gene=617, id=231 removed by soft count = 110\n",
      "--- gene=617, id=232 removed by soft count = 94\n",
      "--- number of probes: 254 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 618\n",
      "--- number of probes: 192 , kept by blast: 192 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 619\n",
      "--- gene=619, id=59 removed by soft count = 180\n",
      "--- gene=619, id=110 removed by soft count = 52\n",
      "--- gene=619, id=273 removed by soft count = 187\n",
      "--- number of probes: 282 , kept by blast: 279 , if remove dups: 279\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 62\n",
      "--- number of probes: 329 , kept by blast: 329 , if remove dups: 329\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 620\n",
      "--- gene=620, id=0 removed by soft count = 124\n",
      "--- gene=620, id=1 removed by soft count = 80\n",
      "--- gene=620, id=61 removed by soft count = 104\n",
      "--- gene=620, id=62 removed by soft count = 500\n",
      "--- gene=620, id=114 removed by soft count = 807\n",
      "--- gene=620, id=115 removed by soft count = 76\n",
      "--- gene=620, id=116 removed by soft count = 201\n",
      "--- gene=620, id=117 removed by soft count = 422\n",
      "--- gene=620, id=118 removed by soft count = 98\n",
      "--- gene=620, id=119 removed by soft count = 131\n",
      "--- gene=620, id=121 removed by soft count = 53\n",
      "--- gene=620, id=123 removed by soft count = 86\n",
      "--- number of probes: 196 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 621\n",
      "--- gene=621, id=80 removed by soft count = 329\n",
      "--- gene=621, id=215 removed by soft count = 354\n",
      "--- gene=621, id=216 removed by soft count = 605\n",
      "--- gene=621, id=240 removed by soft count = 192\n",
      "--- gene=621, id=241 removed by soft count = 133\n",
      "--- gene=621, id=244 removed by soft count = 116\n",
      "--- number of probes: 248 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 622\n",
      "--- gene=622, id=107 removed by soft count = 263\n",
      "--- gene=622, id=108 removed by soft count = 1118\n",
      "--- gene=622, id=109 removed by soft count = 3750\n",
      "--- gene=622, id=221 removed by soft count = 58\n",
      "--- gene=622, id=225 removed by soft count = 153\n",
      "--- number of probes: 247 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 623\n",
      "--- gene=623, id=18 removed by soft count = 1960\n",
      "--- gene=623, id=20 removed by soft count = 2208\n",
      "--- gene=623, id=105 removed by soft count = 150\n",
      "--- gene=623, id=106 removed by soft count = 254\n",
      "--- gene=623, id=111 removed by soft count = 83\n",
      "--- gene=623, id=142 removed by soft count = 560\n",
      "--- gene=623, id=146 removed by soft count = 200\n",
      "--- gene=623, id=150 removed by soft count = 59\n",
      "--- number of probes: 185 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 624\n",
      "--- gene=624, id=0 removed by soft count = 417\n",
      "--- gene=624, id=76 removed by soft count = 121\n",
      "--- number of probes: 132 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 625\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=625, id=0 removed by soft count = 69\n",
      "--- gene=625, id=1 removed by soft count = 119\n",
      "--- gene=625, id=6 removed by soft count = 56\n",
      "--- gene=625, id=105 removed by soft count = 126\n",
      "--- gene=625, id=106 removed by soft count = 168\n",
      "--- number of probes: 168 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 626\n",
      "--- gene=626, id=0 removed by soft count = 385\n",
      "--- gene=626, id=1 removed by soft count = 70\n",
      "--- gene=626, id=2 removed by soft count = 13782\n",
      "--- gene=626, id=18 removed by soft count = 1605\n",
      "--- gene=626, id=47 removed by soft count = 617\n",
      "--- gene=626, id=54 removed by soft count = 65\n",
      "--- gene=626, id=56 removed by soft count = 56\n",
      "--- gene=626, id=89 removed by soft count = 106\n",
      "--- gene=626, id=96 removed by soft count = 9064\n",
      "--- gene=626, id=105 removed by soft count = 73\n",
      "--- gene=626, id=107 removed by soft count = 608\n",
      "--- gene=626, id=108 removed by soft count = 54\n",
      "--- gene=626, id=109 removed by soft count = 352\n",
      "--- gene=626, id=113 removed by soft count = 263\n",
      "--- gene=626, id=115 removed by soft count = 393\n",
      "--- number of probes: 118 , kept by blast: 103 , if remove dups: 103\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 103\n",
      "-- checking probes in region: 627\n",
      "--- gene=627, id=96 removed by soft count = 59\n",
      "--- gene=627, id=97 removed by soft count = 83\n",
      "--- gene=627, id=99 removed by soft count = 907\n",
      "--- gene=627, id=100 removed by soft count = 149\n",
      "--- gene=627, id=169 removed by soft count = 1608\n",
      "--- number of probes: 253 , kept by blast: 248 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 628\n",
      "--- gene=628, id=268 removed by soft count = 582\n",
      "--- gene=628, id=270 removed by soft count = 613\n",
      "--- gene=628, id=272 removed by soft count = 372\n",
      "--- number of probes: 273 , kept by blast: 270 , if remove dups: 270\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 629\n",
      "--- gene=629, id=162 removed by soft count = 57\n",
      "--- number of probes: 360 , kept by blast: 359 , if remove dups: 359\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 63\n",
      "--- gene=63, id=29 removed by soft count = 128\n",
      "--- gene=63, id=33 removed by soft count = 271\n",
      "--- gene=63, id=34 removed by soft count = 87\n",
      "--- gene=63, id=120 removed by soft count = 80\n",
      "--- gene=63, id=149 removed by soft count = 63\n",
      "--- gene=63, id=154 removed by soft count = 161\n",
      "--- number of probes: 241 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 64\n",
      "--- gene=64, id=111 removed by soft count = 52\n",
      "--- gene=64, id=112 removed by soft count = 239\n",
      "--- gene=64, id=113 removed by soft count = 275\n",
      "--- gene=64, id=114 removed by soft count = 172\n",
      "--- gene=64, id=117 removed by soft count = 176\n",
      "--- gene=64, id=118 removed by soft count = 185\n",
      "--- gene=64, id=119 removed by soft count = 201\n",
      "--- gene=64, id=120 removed by soft count = 72\n",
      "--- gene=64, id=121 removed by soft count = 198\n",
      "--- gene=64, id=122 removed by soft count = 197\n",
      "--- gene=64, id=123 removed by soft count = 178\n",
      "--- gene=64, id=124 removed by soft count = 237\n",
      "--- gene=64, id=125 removed by soft count = 266\n",
      "--- gene=64, id=126 removed by soft count = 205\n",
      "--- gene=64, id=128 removed by soft count = 119\n",
      "--- number of probes: 199 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 645\n",
      "--- number of probes: 257 , kept by blast: 257 , if remove dups: 257\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 646\n",
      "--- gene=646, id=37 removed by soft count = 58\n",
      "--- gene=646, id=219 removed by soft count = 668\n",
      "--- gene=646, id=220 removed by soft count = 86\n",
      "--- gene=646, id=223 removed by soft count = 82\n",
      "--- gene=646, id=230 removed by soft count = 53\n",
      "--- number of probes: 241 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 647\n",
      "--- gene=647, id=80 removed by soft count = 118\n",
      "--- gene=647, id=81 removed by soft count = 594\n",
      "--- gene=647, id=82 removed by soft count = 166\n",
      "--- gene=647, id=83 removed by soft count = 537\n",
      "--- gene=647, id=109 removed by soft count = 321\n",
      "--- gene=647, id=112 removed by soft count = 546\n",
      "--- gene=647, id=113 removed by soft count = 116\n",
      "--- gene=647, id=116 removed by soft count = 477\n",
      "--- gene=647, id=118 removed by soft count = 168\n",
      "--- gene=647, id=122 removed by soft count = 59\n",
      "--- gene=647, id=123 removed by soft count = 134\n",
      "--- gene=647, id=131 removed by soft count = 114\n",
      "--- gene=647, id=133 removed by soft count = 78\n",
      "--- gene=647, id=175 removed by soft count = 75\n",
      "--- number of probes: 219 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 648\n",
      "--- number of probes: 215 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 649\n",
      "--- gene=649, id=51 removed by soft count = 68\n",
      "--- gene=649, id=70 removed by soft count = 260\n",
      "--- gene=649, id=71 removed by soft count = 251\n",
      "--- gene=649, id=137 removed by soft count = 135\n",
      "--- number of probes: 262 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 65\n",
      "--- gene=65, id=16 removed by soft count = 395\n",
      "--- gene=65, id=19 removed by soft count = 78\n",
      "--- gene=65, id=23 removed by soft count = 59\n",
      "--- gene=65, id=25 removed by soft count = 172\n",
      "--- gene=65, id=26 removed by soft count = 78\n",
      "--- gene=65, id=27 removed by soft count = 88\n",
      "--- gene=65, id=29 removed by soft count = 138\n",
      "--- gene=65, id=30 removed by soft count = 286\n",
      "--- gene=65, id=102 removed by soft count = 64\n",
      "--- gene=65, id=107 removed by soft count = 187\n",
      "--- gene=65, id=108 removed by soft count = 392\n",
      "--- gene=65, id=109 removed by soft count = 85\n",
      "--- gene=65, id=110 removed by soft count = 2081\n",
      "--- number of probes: 114 , kept by blast: 101 , if remove dups: 101\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 101\n",
      "-- checking probes in region: 650\n",
      "--- gene=650, id=6 removed by soft count = 106\n",
      "--- gene=650, id=144 removed by soft count = 119\n",
      "--- gene=650, id=176 removed by soft count = 95\n",
      "--- number of probes: 241 , kept by blast: 238 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 651\n",
      "--- gene=651, id=17 removed by soft count = 89\n",
      "--- gene=651, id=57 removed by soft count = 104\n",
      "--- gene=651, id=58 removed by soft count = 604\n",
      "--- gene=651, id=59 removed by soft count = 232\n",
      "--- gene=651, id=60 removed by soft count = 153\n",
      "--- gene=651, id=61 removed by soft count = 130\n",
      "--- gene=651, id=62 removed by soft count = 80\n",
      "--- gene=651, id=118 removed by soft count = 84\n",
      "--- gene=651, id=127 removed by soft count = 57\n",
      "--- gene=651, id=136 removed by soft count = 70\n",
      "--- gene=651, id=140 removed by soft count = 142\n",
      "--- gene=651, id=144 removed by soft count = 667\n",
      "--- gene=651, id=145 removed by soft count = 573\n",
      "--- number of probes: 146 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 652\n",
      "--- gene=652, id=38 removed by soft count = 278\n",
      "--- gene=652, id=39 removed by soft count = 310\n",
      "--- number of probes: 306 , kept by blast: 304 , if remove dups: 304\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 653\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=653, id=342 removed by soft count = 65\n",
      "--- number of probes: 373 , kept by blast: 372 , if remove dups: 372\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 654\n",
      "--- gene=654, id=49 removed by soft count = 218\n",
      "--- gene=654, id=63 removed by soft count = 444\n",
      "--- gene=654, id=140 removed by soft count = 76\n",
      "--- gene=654, id=151 removed by soft count = 52\n",
      "--- number of probes: 271 , kept by blast: 267 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 655\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 656\n",
      "--- gene=656, id=54 removed by soft count = 87\n",
      "--- gene=656, id=75 removed by soft count = 376\n",
      "--- gene=656, id=80 removed by soft count = 60\n",
      "--- number of probes: 292 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 657\n",
      "--- gene=657, id=13 removed by soft count = 79\n",
      "--- gene=657, id=15 removed by soft count = 82\n",
      "--- gene=657, id=25 removed by soft count = 92\n",
      "--- gene=657, id=26 removed by soft count = 869\n",
      "--- gene=657, id=32 removed by soft count = 72\n",
      "--- gene=657, id=37 removed by soft count = 68\n",
      "--- gene=657, id=38 removed by soft count = 759\n",
      "--- gene=657, id=44 removed by soft count = 51\n",
      "--- gene=657, id=45 removed by soft count = 55\n",
      "--- number of probes: 186 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 658\n",
      "--- gene=658, id=182 removed by soft count = 1675\n",
      "--- number of probes: 311 , kept by blast: 310 , if remove dups: 310\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 659\n",
      "--- gene=659, id=2 removed by soft count = 266\n",
      "--- gene=659, id=4 removed by soft count = 104\n",
      "--- gene=659, id=7 removed by soft count = 199\n",
      "--- gene=659, id=8 removed by soft count = 79\n",
      "--- gene=659, id=30 removed by soft count = 116\n",
      "--- gene=659, id=33 removed by soft count = 93\n",
      "--- number of probes: 54 , kept by blast: 48 , if remove dups: 42\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: 66\n",
      "--- gene=66, id=41 removed by soft count = 51\n",
      "--- gene=66, id=42 removed by soft count = 3372\n",
      "--- gene=66, id=44 removed by soft count = 279\n",
      "--- gene=66, id=124 removed by soft count = 85\n",
      "--- number of probes: 127 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 660\n",
      "--- gene=660, id=76 removed by soft count = 76\n",
      "--- gene=660, id=78 removed by soft count = 1259\n",
      "--- gene=660, id=253 removed by soft count = 524\n",
      "--- gene=660, id=254 removed by soft count = 75\n",
      "--- gene=660, id=258 removed by soft count = 122\n",
      "--- gene=660, id=259 removed by soft count = 93\n",
      "--- gene=660, id=260 removed by soft count = 190\n",
      "--- gene=660, id=262 removed by soft count = 61\n",
      "--- gene=660, id=265 removed by soft count = 76\n",
      "--- gene=660, id=266 removed by soft count = 69\n",
      "--- gene=660, id=268 removed by soft count = 51\n",
      "--- gene=660, id=276 removed by soft count = 124\n",
      "--- gene=660, id=277 removed by soft count = 165\n",
      "--- number of probes: 278 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 662\n",
      "--- gene=662, id=14 removed by soft count = 155\n",
      "--- gene=662, id=17 removed by soft count = 147\n",
      "--- gene=662, id=23 removed by soft count = 108\n",
      "--- gene=662, id=27 removed by soft count = 716\n",
      "--- gene=662, id=29 removed by soft count = 107\n",
      "--- gene=662, id=31 removed by soft count = 51\n",
      "--- gene=662, id=32 removed by soft count = 56\n",
      "--- gene=662, id=34 removed by soft count = 146\n",
      "--- gene=662, id=40 removed by soft count = 2404\n",
      "--- gene=662, id=44 removed by soft count = 139\n",
      "--- gene=662, id=46 removed by soft count = 64\n",
      "--- gene=662, id=48 removed by soft count = 76\n",
      "--- gene=662, id=49 removed by soft count = 79\n",
      "--- gene=662, id=163 removed by soft count = 766\n",
      "--- number of probes: 184 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 663\n",
      "--- gene=663, id=24 removed by soft count = 144\n",
      "--- gene=663, id=83 removed by soft count = 84\n",
      "--- gene=663, id=85 removed by soft count = 73\n",
      "--- gene=663, id=199 removed by soft count = 484\n",
      "--- gene=663, id=200 removed by soft count = 460\n",
      "--- gene=663, id=202 removed by soft count = 1144\n",
      "--- number of probes: 205 , kept by blast: 199 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 664\n",
      "--- gene=664, id=18 removed by soft count = 240\n",
      "--- gene=664, id=19 removed by soft count = 157\n",
      "--- gene=664, id=22 removed by soft count = 153\n",
      "--- gene=664, id=146 removed by soft count = 146\n",
      "--- number of probes: 255 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 665\n",
      "--- gene=665, id=105 removed by soft count = 274\n",
      "--- gene=665, id=107 removed by soft count = 204\n",
      "--- gene=665, id=170 removed by soft count = 318\n",
      "--- gene=665, id=171 removed by soft count = 56\n",
      "--- gene=665, id=172 removed by soft count = 103\n",
      "--- number of probes: 266 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 666\n",
      "--- gene=666, id=31 removed by soft count = 65\n",
      "--- gene=666, id=43 removed by soft count = 56\n",
      "--- gene=666, id=51 removed by soft count = 55\n",
      "--- gene=666, id=116 removed by soft count = 639\n",
      "--- gene=666, id=119 removed by soft count = 171\n",
      "--- gene=666, id=176 removed by soft count = 76\n",
      "--- number of probes: 273 , kept by blast: 267 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 667\n",
      "--- gene=667, id=9 removed by soft count = 2334\n",
      "--- gene=667, id=10 removed by soft count = 520\n",
      "--- gene=667, id=12 removed by soft count = 422\n",
      "--- gene=667, id=13 removed by soft count = 80\n",
      "--- gene=667, id=20 removed by soft count = 51\n",
      "--- gene=667, id=47 removed by soft count = 86\n",
      "--- gene=667, id=49 removed by soft count = 153\n",
      "--- gene=667, id=74 removed by soft count = 76\n",
      "--- gene=667, id=75 removed by soft count = 78\n",
      "--- gene=667, id=76 removed by soft count = 298\n",
      "--- gene=667, id=81 removed by soft count = 133\n",
      "--- gene=667, id=86 removed by soft count = 273\n",
      "--- gene=667, id=100 removed by soft count = 57\n",
      "--- gene=667, id=131 removed by soft count = 52\n",
      "--- number of probes: 172 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 668\n",
      "--- gene=668, id=135 removed by soft count = 212\n",
      "--- gene=668, id=137 removed by soft count = 58\n",
      "--- gene=668, id=139 removed by soft count = 480\n",
      "--- gene=668, id=141 removed by soft count = 526\n",
      "--- gene=668, id=142 removed by soft count = 96\n",
      "--- number of probes: 328 , kept by blast: 323 , if remove dups: 323\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 669\n",
      "--- gene=669, id=32 removed by soft count = 88\n",
      "--- gene=669, id=35 removed by soft count = 135\n",
      "--- gene=669, id=309 removed by soft count = 264\n",
      "--- gene=669, id=310 removed by soft count = 448\n",
      "--- number of probes: 312 , kept by blast: 308 , if remove dups: 308\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 67\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=67, id=66 removed by soft count = 183\n",
      "--- gene=67, id=113 removed by soft count = 104\n",
      "--- gene=67, id=114 removed by soft count = 711\n",
      "--- number of probes: 269 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 670\n",
      "--- gene=670, id=52 removed by soft count = 58\n",
      "--- gene=670, id=223 removed by soft count = 131\n",
      "--- gene=670, id=268 removed by soft count = 171\n",
      "--- number of probes: 311 , kept by blast: 308 , if remove dups: 308\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 671\n",
      "--- gene=671, id=9 removed by soft count = 51\n",
      "--- gene=671, id=125 removed by soft count = 242\n",
      "--- gene=671, id=131 removed by soft count = 79\n",
      "--- gene=671, id=180 removed by soft count = 306\n",
      "--- gene=671, id=182 removed by soft count = 141\n",
      "--- gene=671, id=183 removed by soft count = 185\n",
      "--- number of probes: 199 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 672\n",
      "--- gene=672, id=120 removed by soft count = 79\n",
      "--- gene=672, id=128 removed by soft count = 75\n",
      "--- number of probes: 171 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 673\n",
      "--- gene=673, id=69 removed by soft count = 172\n",
      "--- gene=673, id=116 removed by soft count = 248\n",
      "--- gene=673, id=118 removed by soft count = 218\n",
      "--- gene=673, id=119 removed by soft count = 5650\n",
      "--- number of probes: 120 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 116\n",
      "-- checking probes in region: 674\n",
      "--- number of probes: 258 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 675\n",
      "--- number of probes: 327 , kept by blast: 327 , if remove dups: 327\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 676\n",
      "--- gene=676, id=131 removed by soft count = 68\n",
      "--- gene=676, id=148 removed by hard count = 0\n",
      "--- gene=676, id=149 removed by hard count = 0\n",
      "--- gene=676, id=150 removed by hard count = 0\n",
      "--- gene=676, id=199 removed by soft count = 796\n",
      "--- number of probes: 329 , kept by blast: 324 , if remove dups: 324\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 677\n",
      "--- gene=677, id=20 removed by soft count = 85\n",
      "--- gene=677, id=246 removed by soft count = 191\n",
      "--- gene=677, id=298 removed by soft count = 282\n",
      "--- number of probes: 308 , kept by blast: 305 , if remove dups: 305\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 678\n",
      "--- gene=678, id=45 removed by soft count = 67\n",
      "--- gene=678, id=192 removed by soft count = 83\n",
      "--- gene=678, id=238 removed by soft count = 191\n",
      "--- gene=678, id=272 removed by soft count = 223\n",
      "--- gene=678, id=288 removed by soft count = 58\n",
      "--- number of probes: 347 , kept by blast: 342 , if remove dups: 341\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 679\n",
      "--- gene=679, id=40 removed by soft count = 116\n",
      "--- gene=679, id=158 removed by soft count = 106\n",
      "--- gene=679, id=161 removed by soft count = 11385\n",
      "--- gene=679, id=174 removed by soft count = 63\n",
      "--- gene=679, id=185 removed by soft count = 209\n",
      "--- number of probes: 320 , kept by blast: 315 , if remove dups: 315\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 68\n",
      "--- gene=68, id=11 removed by soft count = 473\n",
      "--- gene=68, id=12 removed by soft count = 416\n",
      "--- gene=68, id=15 removed by soft count = 143\n",
      "--- gene=68, id=18 removed by soft count = 269\n",
      "--- gene=68, id=19 removed by soft count = 108\n",
      "--- gene=68, id=45 removed by soft count = 82\n",
      "--- gene=68, id=46 removed by soft count = 102\n",
      "--- gene=68, id=83 removed by soft count = 87\n",
      "--- number of probes: 226 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 680\n",
      "--- gene=680, id=162 removed by soft count = 137\n",
      "--- gene=680, id=165 removed by soft count = 452\n",
      "--- gene=680, id=169 removed by soft count = 80\n",
      "--- gene=680, id=224 removed by soft count = 208\n",
      "--- gene=680, id=243 removed by hard count = 3\n",
      "--- number of probes: 293 , kept by blast: 288 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 681\n",
      "--- gene=681, id=84 removed by soft count = 58\n",
      "--- gene=681, id=88 removed by soft count = 228\n",
      "--- gene=681, id=89 removed by soft count = 386\n",
      "--- gene=681, id=90 removed by soft count = 295\n",
      "--- gene=681, id=122 removed by soft count = 80\n",
      "--- gene=681, id=123 removed by soft count = 433\n",
      "--- gene=681, id=124 removed by soft count = 137\n",
      "--- gene=681, id=148 removed by soft count = 61\n",
      "--- number of probes: 306 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 682\n",
      "--- gene=682, id=10 removed by soft count = 89\n",
      "--- gene=682, id=11 removed by soft count = 189\n",
      "--- gene=682, id=14 removed by soft count = 150\n",
      "--- gene=682, id=111 removed by soft count = 63\n",
      "--- number of probes: 147 , kept by blast: 143 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 683\n",
      "--- gene=683, id=64 removed by soft count = 106\n",
      "--- gene=683, id=123 removed by soft count = 60\n",
      "--- gene=683, id=234 removed by soft count = 105\n",
      "--- gene=683, id=235 removed by soft count = 54\n",
      "--- gene=683, id=245 removed by soft count = 117\n",
      "--- number of probes: 285 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 684\n",
      "--- gene=684, id=63 removed by soft count = 132\n",
      "--- gene=684, id=76 removed by soft count = 130\n",
      "--- gene=684, id=210 removed by soft count = 923\n",
      "--- number of probes: 253 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 685\n",
      "--- gene=685, id=59 removed by soft count = 93\n",
      "--- gene=685, id=79 removed by soft count = 407\n",
      "--- number of probes: 328 , kept by blast: 326 , if remove dups: 324\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 686\n",
      "--- gene=686, id=50 removed by soft count = 88\n",
      "--- gene=686, id=98 removed by soft count = 178\n",
      "--- gene=686, id=99 removed by soft count = 685\n",
      "--- gene=686, id=187 removed by soft count = 74\n",
      "--- gene=686, id=190 removed by soft count = 295\n",
      "--- gene=686, id=216 removed by soft count = 77\n",
      "--- gene=686, id=253 removed by soft count = 1287\n",
      "--- gene=686, id=255 removed by soft count = 1605\n",
      "--- number of probes: 266 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 687\n",
      "--- gene=687, id=0 removed by soft count = 109\n",
      "--- gene=687, id=3 removed by soft count = 77\n",
      "--- gene=687, id=4 removed by soft count = 55\n",
      "--- gene=687, id=9 removed by soft count = 93\n",
      "--- gene=687, id=10 removed by soft count = 60\n",
      "--- gene=687, id=13 removed by soft count = 67\n",
      "--- gene=687, id=14 removed by soft count = 56\n",
      "--- gene=687, id=15 removed by soft count = 121\n",
      "--- gene=687, id=16 removed by soft count = 115\n",
      "--- gene=687, id=17 removed by soft count = 70\n",
      "--- gene=687, id=20 removed by soft count = 55\n",
      "--- gene=687, id=21 removed by soft count = 118\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=687, id=28 removed by soft count = 58\n",
      "--- gene=687, id=33 removed by soft count = 66\n",
      "--- gene=687, id=34 removed by soft count = 51\n",
      "--- gene=687, id=38 removed by soft count = 82\n",
      "--- gene=687, id=40 removed by soft count = 110\n",
      "--- gene=687, id=42 removed by soft count = 76\n",
      "--- gene=687, id=48 removed by soft count = 55\n",
      "--- gene=687, id=49 removed by soft count = 58\n",
      "--- gene=687, id=51 removed by soft count = 55\n",
      "--- gene=687, id=57 removed by soft count = 53\n",
      "--- gene=687, id=58 removed by soft count = 54\n",
      "--- gene=687, id=62 removed by soft count = 307\n",
      "--- gene=687, id=64 removed by soft count = 51\n",
      "--- gene=687, id=65 removed by soft count = 77\n",
      "--- gene=687, id=66 removed by soft count = 57\n",
      "--- gene=687, id=68 removed by soft count = 61\n",
      "--- gene=687, id=69 removed by soft count = 59\n",
      "--- gene=687, id=71 removed by soft count = 86\n",
      "--- gene=687, id=72 removed by soft count = 67\n",
      "--- gene=687, id=74 removed by soft count = 90\n",
      "--- gene=687, id=75 removed by soft count = 134\n",
      "--- gene=687, id=76 removed by soft count = 147\n",
      "--- gene=687, id=77 removed by soft count = 150\n",
      "--- number of probes: 78 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: 688\n",
      "--- gene=688, id=2 removed by soft count = 61\n",
      "--- gene=688, id=3 removed by soft count = 95\n",
      "--- gene=688, id=161 removed by soft count = 275\n",
      "--- gene=688, id=162 removed by soft count = 931\n",
      "--- gene=688, id=271 removed by soft count = 270\n",
      "--- gene=688, id=272 removed by soft count = 114\n",
      "--- gene=688, id=273 removed by soft count = 309\n",
      "--- gene=688, id=274 removed by soft count = 189\n",
      "--- gene=688, id=275 removed by soft count = 100\n",
      "--- gene=688, id=276 removed by soft count = 186\n",
      "--- gene=688, id=277 removed by soft count = 103\n",
      "--- number of probes: 278 , kept by blast: 267 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 689\n",
      "--- gene=689, id=20 removed by soft count = 351\n",
      "--- gene=689, id=21 removed by soft count = 374\n",
      "--- gene=689, id=22 removed by soft count = 2988\n",
      "--- gene=689, id=56 removed by soft count = 66\n",
      "--- gene=689, id=115 removed by soft count = 71\n",
      "--- gene=689, id=122 removed by soft count = 136\n",
      "--- gene=689, id=123 removed by soft count = 169\n",
      "--- gene=689, id=126 removed by soft count = 113\n",
      "--- number of probes: 160 , kept by blast: 152 , if remove dups: 152\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 69\n",
      "--- gene=69, id=6 removed by soft count = 128\n",
      "--- gene=69, id=7 removed by soft count = 94\n",
      "--- gene=69, id=9 removed by soft count = 126\n",
      "--- gene=69, id=119 removed by soft count = 356\n",
      "--- gene=69, id=120 removed by soft count = 413\n",
      "--- gene=69, id=121 removed by soft count = 149\n",
      "--- gene=69, id=123 removed by soft count = 461\n",
      "--- gene=69, id=124 removed by soft count = 789\n",
      "--- gene=69, id=125 removed by soft count = 222\n",
      "--- gene=69, id=127 removed by soft count = 230\n",
      "--- number of probes: 128 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 118\n",
      "-- checking probes in region: 690\n",
      "--- gene=690, id=57 removed by soft count = 89\n",
      "--- gene=690, id=84 removed by soft count = 58\n",
      "--- gene=690, id=141 removed by hard count = 0\n",
      "--- gene=690, id=162 removed by soft count = 339\n",
      "--- gene=690, id=224 removed by soft count = 67\n",
      "--- number of probes: 289 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 691\n",
      "--- number of probes: 270 , kept by blast: 270 , if remove dups: 270\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 692\n",
      "--- gene=692, id=23 removed by soft count = 567\n",
      "--- gene=692, id=49 removed by soft count = 81\n",
      "--- gene=692, id=50 removed by soft count = 242\n",
      "--- gene=692, id=51 removed by soft count = 159\n",
      "--- gene=692, id=52 removed by soft count = 158\n",
      "--- gene=692, id=89 removed by soft count = 75\n",
      "--- gene=692, id=138 removed by soft count = 11235\n",
      "--- gene=692, id=172 removed by soft count = 496\n",
      "--- gene=692, id=173 removed by soft count = 308\n",
      "--- number of probes: 189 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 693\n",
      "--- gene=693, id=55 removed by soft count = 299\n",
      "--- gene=693, id=57 removed by soft count = 102\n",
      "--- gene=693, id=120 removed by soft count = 194\n",
      "--- gene=693, id=159 removed by soft count = 204\n",
      "--- gene=693, id=168 removed by soft count = 64\n",
      "--- gene=693, id=174 removed by soft count = 71\n",
      "--- number of probes: 246 , kept by blast: 240 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 694\n",
      "--- gene=694, id=35 removed by soft count = 93\n",
      "--- gene=694, id=169 removed by soft count = 213\n",
      "--- gene=694, id=170 removed by soft count = 100\n",
      "--- gene=694, id=209 removed by soft count = 122\n",
      "--- gene=694, id=217 removed by soft count = 124\n",
      "--- gene=694, id=218 removed by soft count = 478\n",
      "--- gene=694, id=219 removed by soft count = 145\n",
      "--- number of probes: 236 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 695\n",
      "--- gene=695, id=9 removed by soft count = 259\n",
      "--- gene=695, id=12 removed by soft count = 78\n",
      "--- gene=695, id=18 removed by soft count = 195\n",
      "--- gene=695, id=62 removed by soft count = 256\n",
      "--- gene=695, id=73 removed by soft count = 75\n",
      "--- gene=695, id=158 removed by soft count = 74\n",
      "--- gene=695, id=168 removed by soft count = 317\n",
      "--- gene=695, id=185 removed by soft count = 128\n",
      "--- gene=695, id=186 removed by soft count = 118\n",
      "--- gene=695, id=188 removed by soft count = 163\n",
      "--- gene=695, id=240 removed by soft count = 64\n",
      "--- gene=695, id=241 removed by soft count = 785\n",
      "--- gene=695, id=242 removed by soft count = 53\n",
      "--- number of probes: 246 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 696\n",
      "--- gene=696, id=48 removed by soft count = 63\n",
      "--- gene=696, id=84 removed by soft count = 151\n",
      "--- gene=696, id=86 removed by soft count = 1362\n",
      "--- gene=696, id=87 removed by soft count = 245\n",
      "--- number of probes: 170 , kept by blast: 166 , if remove dups: 166\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 697\n",
      "--- gene=697, id=36 removed by soft count = 64\n",
      "--- gene=697, id=76 removed by soft count = 390\n",
      "--- gene=697, id=98 removed by soft count = 65\n",
      "--- gene=697, id=105 removed by soft count = 51\n",
      "--- gene=697, id=106 removed by soft count = 74\n",
      "--- gene=697, id=115 removed by soft count = 281\n",
      "--- number of probes: 220 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 698\n",
      "--- gene=698, id=2 removed by soft count = 86\n",
      "--- gene=698, id=31 removed by soft count = 136\n",
      "--- gene=698, id=33 removed by soft count = 644\n",
      "--- gene=698, id=36 removed by soft count = 238\n",
      "--- gene=698, id=142 removed by soft count = 92\n",
      "--- gene=698, id=178 removed by soft count = 212\n",
      "--- gene=698, id=179 removed by soft count = 163\n",
      "--- number of probes: 227 , kept by blast: 220 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 699\n",
      "--- gene=699, id=10 removed by soft count = 229\n",
      "--- gene=699, id=14 removed by soft count = 966\n",
      "--- gene=699, id=15 removed by soft count = 1033\n",
      "--- gene=699, id=38 removed by soft count = 704\n",
      "--- gene=699, id=39 removed by soft count = 782\n",
      "--- gene=699, id=42 removed by soft count = 1892\n",
      "--- gene=699, id=46 removed by soft count = 454\n",
      "--- gene=699, id=47 removed by soft count = 243\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=699, id=48 removed by soft count = 171\n",
      "--- gene=699, id=51 removed by soft count = 381\n",
      "--- gene=699, id=52 removed by soft count = 1851\n",
      "--- gene=699, id=54 removed by soft count = 153\n",
      "--- gene=699, id=55 removed by soft count = 64\n",
      "--- gene=699, id=58 removed by soft count = 247\n",
      "--- gene=699, id=59 removed by soft count = 108\n",
      "--- gene=699, id=60 removed by soft count = 58\n",
      "--- gene=699, id=61 removed by soft count = 101\n",
      "--- gene=699, id=64 removed by soft count = 151\n",
      "--- gene=699, id=65 removed by soft count = 65\n",
      "--- number of probes: 66 , kept by blast: 47 , if remove dups: 47\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: 7\n",
      "--- gene=7, id=104 removed by soft count = 281\n",
      "--- gene=7, id=162 removed by hard count = 0\n",
      "--- gene=7, id=238 removed by soft count = 61\n",
      "--- gene=7, id=249 removed by soft count = 56\n",
      "--- gene=7, id=253 removed by soft count = 265\n",
      "--- number of probes: 341 , kept by blast: 336 , if remove dups: 336\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 70\n",
      "--- gene=70, id=2 removed by soft count = 138\n",
      "--- gene=70, id=6 removed by soft count = 53\n",
      "--- gene=70, id=10 removed by soft count = 54\n",
      "--- gene=70, id=18 removed by soft count = 101\n",
      "--- gene=70, id=22 removed by soft count = 51\n",
      "--- gene=70, id=41 removed by soft count = 101\n",
      "--- gene=70, id=85 removed by soft count = 166\n",
      "--- gene=70, id=89 removed by soft count = 253\n",
      "--- gene=70, id=90 removed by soft count = 432\n",
      "--- number of probes: 120 , kept by blast: 111 , if remove dups: 111\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 111\n",
      "-- checking probes in region: 701\n",
      "--- gene=701, id=4 removed by soft count = 445\n",
      "--- gene=701, id=118 removed by soft count = 102\n",
      "--- gene=701, id=159 removed by soft count = 175\n",
      "--- gene=701, id=241 removed by soft count = 290\n",
      "--- gene=701, id=244 removed by soft count = 413\n",
      "--- gene=701, id=245 removed by soft count = 129\n",
      "--- gene=701, id=248 removed by soft count = 90\n",
      "--- gene=701, id=249 removed by soft count = 638\n",
      "--- gene=701, id=266 removed by soft count = 221\n",
      "--- gene=701, id=285 removed by soft count = 74\n",
      "--- number of probes: 289 , kept by blast: 279 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 702\n",
      "--- gene=702, id=33 removed by soft count = 544\n",
      "--- gene=702, id=34 removed by soft count = 154\n",
      "--- gene=702, id=169 removed by soft count = 559\n",
      "--- gene=702, id=172 removed by soft count = 285\n",
      "--- gene=702, id=174 removed by soft count = 65\n",
      "--- gene=702, id=289 removed by soft count = 86\n",
      "--- gene=702, id=290 removed by soft count = 5296\n",
      "--- number of probes: 291 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 703\n",
      "--- gene=703, id=2 removed by soft count = 71\n",
      "--- gene=703, id=14 removed by soft count = 80\n",
      "--- gene=703, id=18 removed by soft count = 51\n",
      "--- gene=703, id=40 removed by soft count = 98\n",
      "--- gene=703, id=132 removed by soft count = 710\n",
      "--- gene=703, id=133 removed by soft count = 185\n",
      "--- gene=703, id=143 removed by soft count = 63\n",
      "--- number of probes: 181 , kept by blast: 174 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 704\n",
      "--- gene=704, id=71 removed by soft count = 1242\n",
      "--- gene=704, id=72 removed by soft count = 558\n",
      "--- gene=704, id=73 removed by soft count = 606\n",
      "--- gene=704, id=74 removed by soft count = 439\n",
      "--- gene=704, id=75 removed by soft count = 500\n",
      "--- gene=704, id=76 removed by soft count = 51\n",
      "--- gene=704, id=77 removed by soft count = 321\n",
      "--- gene=704, id=78 removed by soft count = 287\n",
      "--- gene=704, id=79 removed by soft count = 456\n",
      "--- gene=704, id=80 removed by soft count = 405\n",
      "--- gene=704, id=81 removed by soft count = 55\n",
      "--- gene=704, id=82 removed by soft count = 148\n",
      "--- gene=704, id=83 removed by soft count = 396\n",
      "--- gene=704, id=84 removed by soft count = 276\n",
      "--- gene=704, id=85 removed by soft count = 369\n",
      "--- gene=704, id=164 removed by soft count = 62\n",
      "--- gene=704, id=165 removed by soft count = 86\n",
      "--- gene=704, id=166 removed by soft count = 64\n",
      "--- gene=704, id=169 removed by soft count = 55\n",
      "--- gene=704, id=170 removed by soft count = 92\n",
      "--- gene=704, id=175 removed by soft count = 57\n",
      "--- gene=704, id=176 removed by soft count = 147\n",
      "--- number of probes: 284 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 705\n",
      "--- gene=705, id=236 removed by soft count = 233\n",
      "--- gene=705, id=265 removed by soft count = 597\n",
      "--- number of probes: 292 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 706\n",
      "--- gene=706, id=13 removed by soft count = 1658\n",
      "--- gene=706, id=34 removed by soft count = 255\n",
      "--- gene=706, id=104 removed by soft count = 428\n",
      "--- gene=706, id=192 removed by soft count = 72\n",
      "--- number of probes: 246 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 707\n",
      "--- gene=707, id=13 removed by soft count = 280\n",
      "--- gene=707, id=14 removed by soft count = 763\n",
      "--- gene=707, id=16 removed by soft count = 2466\n",
      "--- gene=707, id=69 removed by soft count = 97\n",
      "--- gene=707, id=78 removed by soft count = 52\n",
      "--- gene=707, id=82 removed by soft count = 126\n",
      "--- gene=707, id=83 removed by soft count = 469\n",
      "--- gene=707, id=86 removed by soft count = 58\n",
      "--- gene=707, id=101 removed by soft count = 122\n",
      "--- gene=707, id=106 removed by soft count = 54\n",
      "--- gene=707, id=113 removed by soft count = 479\n",
      "--- gene=707, id=123 removed by soft count = 62\n",
      "--- gene=707, id=126 removed by soft count = 58\n",
      "--- number of probes: 182 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 708\n",
      "--- gene=708, id=56 removed by soft count = 56\n",
      "--- gene=708, id=68 removed by soft count = 73\n",
      "--- gene=708, id=79 removed by soft count = 172\n",
      "--- gene=708, id=80 removed by soft count = 55\n",
      "--- gene=708, id=87 removed by soft count = 125\n",
      "--- gene=708, id=138 removed by soft count = 52\n",
      "--- number of probes: 213 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 709\n",
      "--- gene=709, id=19 removed by soft count = 90\n",
      "--- gene=709, id=49 removed by soft count = 2653\n",
      "--- gene=709, id=50 removed by soft count = 79\n",
      "--- gene=709, id=51 removed by soft count = 383\n",
      "--- gene=709, id=52 removed by soft count = 1075\n",
      "--- gene=709, id=155 removed by soft count = 56\n",
      "--- gene=709, id=242 removed by soft count = 90\n",
      "--- number of probes: 289 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 71\n",
      "--- gene=71, id=116 removed by soft count = 87\n",
      "--- gene=71, id=122 removed by soft count = 236\n",
      "--- gene=71, id=123 removed by soft count = 56\n",
      "--- gene=71, id=124 removed by soft count = 109\n",
      "--- gene=71, id=126 removed by soft count = 314\n",
      "--- gene=71, id=129 removed by soft count = 64\n",
      "--- gene=71, id=130 removed by soft count = 103\n",
      "--- gene=71, id=134 removed by soft count = 123\n",
      "--- gene=71, id=135 removed by soft count = 62\n",
      "--- gene=71, id=136 removed by soft count = 106\n",
      "--- number of probes: 137 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 710\n",
      "--- gene=710, id=178 removed by soft count = 1231\n",
      "--- gene=710, id=180 removed by soft count = 5243\n",
      "--- gene=710, id=181 removed by soft count = 5157\n",
      "--- gene=710, id=182 removed by soft count = 1410\n",
      "--- gene=710, id=183 removed by soft count = 488\n",
      "--- gene=710, id=184 removed by soft count = 142\n",
      "--- gene=710, id=185 removed by soft count = 201\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 234 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 711\n",
      "--- gene=711, id=194 removed by soft count = 238\n",
      "--- gene=711, id=214 removed by soft count = 320\n",
      "--- number of probes: 290 , kept by blast: 288 , if remove dups: 288\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 712\n",
      "--- gene=712, id=2 removed by soft count = 68\n",
      "--- gene=712, id=27 removed by soft count = 154\n",
      "--- gene=712, id=43 removed by soft count = 98\n",
      "--- gene=712, id=115 removed by soft count = 95\n",
      "--- gene=712, id=142 removed by soft count = 159\n",
      "--- gene=712, id=149 removed by soft count = 143\n",
      "--- gene=712, id=154 removed by soft count = 91\n",
      "--- number of probes: 201 , kept by blast: 194 , if remove dups: 194\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 713\n",
      "--- number of probes: 236 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 714\n",
      "--- gene=714, id=3 removed by soft count = 84\n",
      "--- gene=714, id=6 removed by soft count = 137\n",
      "--- gene=714, id=18 removed by soft count = 1417\n",
      "--- gene=714, id=54 removed by soft count = 3577\n",
      "--- gene=714, id=112 removed by soft count = 1352\n",
      "--- number of probes: 196 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 715\n",
      "--- gene=715, id=1 removed by soft count = 109\n",
      "--- gene=715, id=4 removed by soft count = 628\n",
      "--- gene=715, id=173 removed by soft count = 505\n",
      "--- gene=715, id=179 removed by soft count = 93\n",
      "--- number of probes: 353 , kept by blast: 349 , if remove dups: 349\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 716\n",
      "--- gene=716, id=18 removed by soft count = 101\n",
      "--- gene=716, id=36 removed by soft count = 54\n",
      "--- gene=716, id=37 removed by soft count = 361\n",
      "--- number of probes: 136 , kept by blast: 133 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 717\n",
      "--- gene=717, id=82 removed by soft count = 58\n",
      "--- gene=717, id=252 removed by soft count = 157\n",
      "--- number of probes: 282 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 718\n",
      "--- gene=718, id=384 removed by soft count = 101\n",
      "--- number of probes: 391 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 719\n",
      "--- gene=719, id=62 removed by soft count = 128\n",
      "--- number of probes: 376 , kept by blast: 375 , if remove dups: 375\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 72\n",
      "--- gene=72, id=61 removed by soft count = 2033\n",
      "--- gene=72, id=117 removed by soft count = 57\n",
      "--- gene=72, id=129 removed by soft count = 52\n",
      "--- gene=72, id=136 removed by soft count = 195\n",
      "--- gene=72, id=137 removed by soft count = 59\n",
      "--- gene=72, id=143 removed by soft count = 51\n",
      "--- gene=72, id=144 removed by soft count = 83\n",
      "--- gene=72, id=145 removed by soft count = 488\n",
      "--- gene=72, id=146 removed by soft count = 333\n",
      "--- number of probes: 267 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 720\n",
      "--- gene=720, id=53 removed by soft count = 745\n",
      "--- gene=720, id=54 removed by soft count = 233\n",
      "--- gene=720, id=56 removed by soft count = 79\n",
      "--- gene=720, id=58 removed by soft count = 113\n",
      "--- gene=720, id=61 removed by soft count = 75\n",
      "--- gene=720, id=62 removed by soft count = 148\n",
      "--- gene=720, id=138 removed by soft count = 682\n",
      "--- gene=720, id=197 removed by soft count = 64\n",
      "--- number of probes: 218 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 721\n",
      "--- gene=721, id=102 removed by soft count = 170\n",
      "--- gene=721, id=228 removed by soft count = 557\n",
      "--- number of probes: 270 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 722\n",
      "--- gene=722, id=6 removed by soft count = 111\n",
      "--- gene=722, id=8 removed by soft count = 260\n",
      "--- gene=722, id=11 removed by soft count = 841\n",
      "--- gene=722, id=28 removed by soft count = 61\n",
      "--- gene=722, id=87 removed by soft count = 76\n",
      "--- number of probes: 309 , kept by blast: 304 , if remove dups: 304\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 723\n",
      "--- gene=723, id=6 removed by soft count = 785\n",
      "--- gene=723, id=8 removed by soft count = 85\n",
      "--- gene=723, id=14 removed by soft count = 760\n",
      "--- gene=723, id=25 removed by soft count = 76\n",
      "--- gene=723, id=49 removed by soft count = 241\n",
      "--- number of probes: 73 , kept by blast: 68 , if remove dups: 68\n",
      "-- number of probes kept for this region: 68\n",
      "-- checking probes in region: 724\n",
      "--- gene=724, id=20 removed by soft count = 66\n",
      "--- gene=724, id=35 removed by soft count = 68\n",
      "--- gene=724, id=122 removed by soft count = 80\n",
      "--- gene=724, id=140 removed by soft count = 79\n",
      "--- number of probes: 287 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 725\n",
      "--- gene=725, id=163 removed by soft count = 688\n",
      "--- gene=725, id=209 removed by soft count = 144\n",
      "--- number of probes: 271 , kept by blast: 269 , if remove dups: 269\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 726\n",
      "--- gene=726, id=53 removed by soft count = 203\n",
      "--- number of probes: 209 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 727\n",
      "--- gene=727, id=22 removed by soft count = 76\n",
      "--- gene=727, id=48 removed by soft count = 73\n",
      "--- gene=727, id=51 removed by soft count = 195\n",
      "--- gene=727, id=67 removed by soft count = 190\n",
      "--- gene=727, id=68 removed by soft count = 118\n",
      "--- gene=727, id=75 removed by soft count = 217\n",
      "--- gene=727, id=82 removed by soft count = 115\n",
      "--- gene=727, id=102 removed by soft count = 115\n",
      "--- gene=727, id=103 removed by soft count = 377\n",
      "--- number of probes: 138 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 728\n",
      "--- gene=728, id=73 removed by soft count = 208\n",
      "--- gene=728, id=298 removed by soft count = 70\n",
      "--- gene=728, id=302 removed by soft count = 434\n",
      "--- number of probes: 365 , kept by blast: 362 , if remove dups: 362\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 729\n",
      "--- gene=729, id=1 removed by soft count = 74\n",
      "--- gene=729, id=4 removed by soft count = 58\n",
      "--- gene=729, id=5 removed by soft count = 69\n",
      "--- gene=729, id=7 removed by soft count = 63\n",
      "--- gene=729, id=119 removed by soft count = 56\n",
      "--- gene=729, id=122 removed by soft count = 253\n",
      "--- number of probes: 200 , kept by blast: 194 , if remove dups: 194\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 73\n",
      "--- gene=73, id=99 removed by soft count = 115\n",
      "--- gene=73, id=100 removed by soft count = 159\n",
      "--- gene=73, id=106 removed by soft count = 92\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=73, id=107 removed by soft count = 147\n",
      "--- gene=73, id=108 removed by soft count = 268\n",
      "--- gene=73, id=109 removed by soft count = 620\n",
      "--- gene=73, id=185 removed by soft count = 114\n",
      "--- number of probes: 198 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 730\n",
      "--- number of probes: 379 , kept by blast: 379 , if remove dups: 379\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 731\n",
      "--- gene=731, id=119 removed by soft count = 654\n",
      "--- number of probes: 174 , kept by blast: 173 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 732\n",
      "--- gene=732, id=0 removed by soft count = 89\n",
      "--- gene=732, id=4 removed by soft count = 236\n",
      "--- gene=732, id=6 removed by soft count = 114\n",
      "--- number of probes: 296 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 733\n",
      "--- gene=733, id=174 removed by soft count = 115\n",
      "--- number of probes: 274 , kept by blast: 273 , if remove dups: 273\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 734\n",
      "--- gene=734, id=83 removed by soft count = 663\n",
      "--- gene=734, id=84 removed by soft count = 59\n",
      "--- gene=734, id=131 removed by soft count = 4395\n",
      "--- gene=734, id=132 removed by soft count = 51\n",
      "--- gene=734, id=133 removed by soft count = 248\n",
      "--- number of probes: 225 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 735\n",
      "--- gene=735, id=37 removed by soft count = 307\n",
      "--- gene=735, id=144 removed by soft count = 89\n",
      "--- number of probes: 206 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 736\n",
      "--- gene=736, id=207 removed by soft count = 96\n",
      "--- gene=736, id=260 removed by soft count = 99\n",
      "--- gene=736, id=347 removed by soft count = 60\n",
      "--- gene=736, id=348 removed by soft count = 197\n",
      "--- number of probes: 362 , kept by blast: 358 , if remove dups: 357\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 737\n",
      "--- gene=737, id=0 removed by soft count = 216\n",
      "--- gene=737, id=1 removed by soft count = 288\n",
      "--- gene=737, id=3 removed by soft count = 64\n",
      "--- gene=737, id=223 removed by soft count = 256\n",
      "--- number of probes: 252 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 738\n",
      "--- gene=738, id=19 removed by soft count = 591\n",
      "--- gene=738, id=61 removed by soft count = 55\n",
      "--- gene=738, id=70 removed by soft count = 169\n",
      "--- gene=738, id=91 removed by soft count = 188\n",
      "--- gene=738, id=102 removed by soft count = 222\n",
      "--- gene=738, id=201 removed by soft count = 174\n",
      "--- gene=738, id=202 removed by soft count = 67\n",
      "--- gene=738, id=205 removed by soft count = 232\n",
      "--- gene=738, id=208 removed by soft count = 150\n",
      "--- number of probes: 221 , kept by blast: 212 , if remove dups: 212\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 739\n",
      "--- gene=739, id=19 removed by soft count = 228\n",
      "--- gene=739, id=160 removed by soft count = 4032\n",
      "--- gene=739, id=197 removed by soft count = 68\n",
      "--- gene=739, id=220 removed by soft count = 858\n",
      "--- number of probes: 229 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 74\n",
      "--- gene=74, id=82 removed by soft count = 1181\n",
      "--- gene=74, id=84 removed by soft count = 115\n",
      "--- gene=74, id=86 removed by soft count = 92\n",
      "--- gene=74, id=87 removed by soft count = 2074\n",
      "--- gene=74, id=88 removed by soft count = 310\n",
      "--- gene=74, id=90 removed by soft count = 73\n",
      "--- gene=74, id=97 removed by soft count = 172\n",
      "--- number of probes: 173 , kept by blast: 166 , if remove dups: 166\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 740\n",
      "--- gene=740, id=2 removed by soft count = 87\n",
      "--- gene=740, id=13 removed by soft count = 703\n",
      "--- gene=740, id=24 removed by soft count = 67\n",
      "--- gene=740, id=35 removed by soft count = 68\n",
      "--- gene=740, id=51 removed by soft count = 141\n",
      "--- gene=740, id=56 removed by soft count = 165\n",
      "--- gene=740, id=59 removed by soft count = 405\n",
      "--- gene=740, id=60 removed by soft count = 65\n",
      "--- gene=740, id=61 removed by soft count = 87\n",
      "--- gene=740, id=62 removed by soft count = 111\n",
      "--- gene=740, id=63 removed by soft count = 121\n",
      "--- gene=740, id=64 removed by soft count = 101\n",
      "--- gene=740, id=65 removed by soft count = 66\n",
      "--- gene=740, id=66 removed by soft count = 430\n",
      "--- gene=740, id=67 removed by soft count = 147\n",
      "--- gene=740, id=69 removed by soft count = 184\n",
      "--- gene=740, id=71 removed by soft count = 74\n",
      "--- gene=740, id=72 removed by soft count = 141\n",
      "--- gene=740, id=73 removed by soft count = 123\n",
      "--- gene=740, id=74 removed by soft count = 55\n",
      "--- gene=740, id=75 removed by soft count = 118\n",
      "--- gene=740, id=180 removed by soft count = 210\n",
      "--- gene=740, id=183 removed by soft count = 494\n",
      "--- gene=740, id=187 removed by soft count = 175\n",
      "--- gene=740, id=190 removed by soft count = 103\n",
      "--- number of probes: 249 , kept by blast: 224 , if remove dups: 224\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 741\n",
      "--- gene=741, id=154 removed by soft count = 70\n",
      "--- gene=741, id=176 removed by soft count = 445\n",
      "--- gene=741, id=255 removed by soft count = 62\n",
      "--- gene=741, id=256 removed by soft count = 137\n",
      "--- number of probes: 266 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 742\n",
      "--- gene=742, id=11 removed by soft count = 129\n",
      "--- gene=742, id=266 removed by soft count = 239\n",
      "--- number of probes: 353 , kept by blast: 351 , if remove dups: 351\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 743\n",
      "--- gene=743, id=112 removed by soft count = 198\n",
      "--- gene=743, id=156 removed by soft count = 97\n",
      "--- gene=743, id=194 removed by soft count = 152\n",
      "--- number of probes: 306 , kept by blast: 303 , if remove dups: 303\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 744\n",
      "--- gene=744, id=35 removed by soft count = 498\n",
      "--- gene=744, id=36 removed by soft count = 67\n",
      "--- gene=744, id=220 removed by soft count = 100\n",
      "--- gene=744, id=261 removed by soft count = 185\n",
      "--- number of probes: 308 , kept by blast: 304 , if remove dups: 303\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 745\n",
      "--- gene=745, id=270 removed by soft count = 95\n",
      "--- gene=745, id=297 removed by soft count = 106\n",
      "--- gene=745, id=298 removed by soft count = 144\n",
      "--- gene=745, id=300 removed by soft count = 228\n",
      "--- number of probes: 367 , kept by blast: 363 , if remove dups: 363\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 748\n",
      "--- gene=748, id=97 removed by soft count = 53\n",
      "--- gene=748, id=161 removed by soft count = 78\n",
      "--- gene=748, id=162 removed by soft count = 176\n",
      "--- gene=748, id=163 removed by soft count = 63\n",
      "--- gene=748, id=187 removed by soft count = 363\n",
      "--- gene=748, id=188 removed by soft count = 503\n",
      "--- gene=748, id=194 removed by soft count = 74\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=748, id=196 removed by soft count = 52\n",
      "--- number of probes: 254 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 749\n",
      "--- gene=749, id=0 removed by soft count = 546\n",
      "--- gene=749, id=1 removed by soft count = 63\n",
      "--- gene=749, id=2 removed by soft count = 134\n",
      "--- gene=749, id=3 removed by soft count = 141\n",
      "--- gene=749, id=4 removed by soft count = 426\n",
      "--- gene=749, id=34 removed by soft count = 55\n",
      "--- gene=749, id=35 removed by soft count = 62\n",
      "--- gene=749, id=65 removed by soft count = 54\n",
      "--- gene=749, id=73 removed by soft count = 188\n",
      "--- number of probes: 199 , kept by blast: 190 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 75\n",
      "--- gene=75, id=2 removed by soft count = 244\n",
      "--- gene=75, id=30 removed by soft count = 241\n",
      "--- gene=75, id=81 removed by soft count = 421\n",
      "--- gene=75, id=82 removed by soft count = 142\n",
      "--- gene=75, id=83 removed by soft count = 1686\n",
      "--- gene=75, id=88 removed by soft count = 65\n",
      "--- gene=75, id=89 removed by soft count = 93\n",
      "--- gene=75, id=90 removed by soft count = 69\n",
      "--- gene=75, id=92 removed by soft count = 485\n",
      "--- gene=75, id=139 removed by soft count = 95\n",
      "--- number of probes: 149 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 750\n",
      "--- gene=750, id=261 removed by soft count = 24370\n",
      "--- gene=750, id=265 removed by soft count = 83\n",
      "--- number of probes: 287 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 751\n",
      "--- gene=751, id=29 removed by soft count = 294\n",
      "--- gene=751, id=115 removed by soft count = 77\n",
      "--- gene=751, id=117 removed by soft count = 78\n",
      "--- gene=751, id=118 removed by soft count = 72\n",
      "--- gene=751, id=119 removed by soft count = 181\n",
      "--- gene=751, id=120 removed by soft count = 68\n",
      "--- gene=751, id=123 removed by soft count = 51\n",
      "--- gene=751, id=124 removed by soft count = 89\n",
      "--- gene=751, id=199 removed by soft count = 445\n",
      "--- gene=751, id=200 removed by soft count = 61\n",
      "--- gene=751, id=201 removed by soft count = 383\n",
      "--- gene=751, id=202 removed by soft count = 171\n",
      "--- gene=751, id=240 removed by soft count = 315\n",
      "--- number of probes: 282 , kept by blast: 269 , if remove dups: 269\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 752\n",
      "--- gene=752, id=17 removed by soft count = 107\n",
      "--- gene=752, id=54 removed by soft count = 155\n",
      "--- gene=752, id=55 removed by soft count = 82\n",
      "--- gene=752, id=96 removed by soft count = 745\n",
      "--- number of probes: 307 , kept by blast: 303 , if remove dups: 303\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 753\n",
      "--- gene=753, id=5 removed by soft count = 64\n",
      "--- gene=753, id=6 removed by soft count = 57\n",
      "--- gene=753, id=7 removed by soft count = 58\n",
      "--- gene=753, id=8 removed by soft count = 54\n",
      "--- gene=753, id=9 removed by soft count = 51\n",
      "--- gene=753, id=10 removed by soft count = 96\n",
      "--- gene=753, id=11 removed by soft count = 54\n",
      "--- gene=753, id=16 removed by soft count = 253\n",
      "--- gene=753, id=19 removed by soft count = 69\n",
      "--- gene=753, id=20 removed by soft count = 181\n",
      "--- gene=753, id=21 removed by soft count = 374\n",
      "--- gene=753, id=22 removed by soft count = 91\n",
      "--- gene=753, id=28 removed by soft count = 66\n",
      "--- gene=753, id=40 removed by soft count = 92\n",
      "--- gene=753, id=100 removed by soft count = 57\n",
      "--- gene=753, id=124 removed by soft count = 83\n",
      "--- gene=753, id=125 removed by soft count = 188\n",
      "--- gene=753, id=126 removed by soft count = 256\n",
      "--- gene=753, id=139 removed by soft count = 77\n",
      "--- number of probes: 145 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 754\n",
      "--- gene=754, id=209 removed by hard count = 3\n",
      "--- number of probes: 344 , kept by blast: 343 , if remove dups: 343\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 755\n",
      "--- gene=755, id=0 removed by soft count = 359\n",
      "--- gene=755, id=1 removed by soft count = 320\n",
      "--- gene=755, id=3 removed by soft count = 61\n",
      "--- gene=755, id=4 removed by soft count = 56\n",
      "--- gene=755, id=7 removed by soft count = 126\n",
      "--- gene=755, id=16 removed by soft count = 528\n",
      "--- gene=755, id=18 removed by soft count = 734\n",
      "--- gene=755, id=20 removed by soft count = 355\n",
      "--- gene=755, id=22 removed by soft count = 267\n",
      "--- gene=755, id=25 removed by soft count = 84\n",
      "--- gene=755, id=26 removed by soft count = 207\n",
      "--- gene=755, id=29 removed by soft count = 61\n",
      "--- gene=755, id=32 removed by soft count = 74\n",
      "--- gene=755, id=36 removed by soft count = 146\n",
      "--- gene=755, id=39 removed by soft count = 83\n",
      "--- gene=755, id=41 removed by soft count = 92\n",
      "--- gene=755, id=43 removed by soft count = 345\n",
      "--- gene=755, id=44 removed by soft count = 107\n",
      "--- gene=755, id=46 removed by soft count = 74\n",
      "--- gene=755, id=49 removed by soft count = 520\n",
      "--- gene=755, id=51 removed by soft count = 743\n",
      "--- gene=755, id=54 removed by soft count = 259\n",
      "--- gene=755, id=57 removed by soft count = 177\n",
      "--- number of probes: 59 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: 756\n",
      "--- gene=756, id=21 removed by soft count = 232\n",
      "--- gene=756, id=88 removed by soft count = 95\n",
      "--- gene=756, id=131 removed by soft count = 153\n",
      "--- gene=756, id=299 removed by soft count = 92\n",
      "--- gene=756, id=300 removed by soft count = 88\n",
      "--- number of probes: 319 , kept by blast: 314 , if remove dups: 314\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 757\n",
      "--- gene=757, id=2 removed by soft count = 232\n",
      "--- gene=757, id=22 removed by soft count = 93\n",
      "--- gene=757, id=47 removed by soft count = 81\n",
      "--- gene=757, id=48 removed by soft count = 115\n",
      "--- gene=757, id=59 removed by soft count = 125\n",
      "--- gene=757, id=92 removed by soft count = 92\n",
      "--- gene=757, id=113 removed by soft count = 57\n",
      "--- gene=757, id=114 removed by soft count = 57\n",
      "--- gene=757, id=142 removed by soft count = 410\n",
      "--- gene=757, id=143 removed by soft count = 398\n",
      "--- gene=757, id=145 removed by soft count = 389\n",
      "--- gene=757, id=146 removed by soft count = 100\n",
      "--- gene=757, id=149 removed by soft count = 1095\n",
      "--- gene=757, id=187 removed by soft count = 172\n",
      "--- number of probes: 192 , kept by blast: 178 , if remove dups: 178\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 758\n",
      "--- gene=758, id=59 removed by soft count = 70\n",
      "--- gene=758, id=77 removed by soft count = 132\n",
      "--- gene=758, id=78 removed by soft count = 207\n",
      "--- gene=758, id=79 removed by soft count = 112\n",
      "--- gene=758, id=81 removed by soft count = 63\n",
      "--- gene=758, id=82 removed by soft count = 359\n",
      "--- gene=758, id=83 removed by soft count = 272\n",
      "--- gene=758, id=84 removed by soft count = 233\n",
      "--- gene=758, id=85 removed by soft count = 248\n",
      "--- gene=758, id=86 removed by soft count = 89\n",
      "--- gene=758, id=87 removed by soft count = 237\n",
      "--- gene=758, id=88 removed by soft count = 272\n",
      "--- gene=758, id=89 removed by soft count = 66\n",
      "--- gene=758, id=90 removed by soft count = 286\n",
      "--- gene=758, id=91 removed by soft count = 330\n",
      "--- gene=758, id=96 removed by soft count = 67\n",
      "--- gene=758, id=124 removed by soft count = 136\n",
      "--- gene=758, id=134 removed by soft count = 1080\n",
      "--- gene=758, id=135 removed by soft count = 147\n",
      "--- gene=758, id=136 removed by soft count = 75\n",
      "--- gene=758, id=137 removed by soft count = 53\n",
      "--- number of probes: 141 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 759\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=759, id=213 removed by soft count = 54\n",
      "--- gene=759, id=217 removed by soft count = 72\n",
      "--- number of probes: 306 , kept by blast: 304 , if remove dups: 304\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 76\n",
      "--- gene=76, id=1 removed by soft count = 285\n",
      "--- gene=76, id=3 removed by soft count = 406\n",
      "--- gene=76, id=4 removed by soft count = 90\n",
      "--- gene=76, id=5 removed by soft count = 432\n",
      "--- gene=76, id=6 removed by soft count = 279\n",
      "--- gene=76, id=7 removed by soft count = 417\n",
      "--- gene=76, id=8 removed by soft count = 111\n",
      "--- gene=76, id=9 removed by soft count = 293\n",
      "--- gene=76, id=10 removed by soft count = 363\n",
      "--- gene=76, id=11 removed by soft count = 232\n",
      "--- gene=76, id=12 removed by soft count = 488\n",
      "--- gene=76, id=13 removed by soft count = 493\n",
      "--- gene=76, id=14 removed by soft count = 288\n",
      "--- gene=76, id=15 removed by soft count = 389\n",
      "--- gene=76, id=17 removed by soft count = 425\n",
      "--- gene=76, id=18 removed by soft count = 591\n",
      "--- gene=76, id=19 removed by soft count = 350\n",
      "--- gene=76, id=20 removed by soft count = 57\n",
      "--- gene=76, id=35 removed by soft count = 2224\n",
      "--- gene=76, id=36 removed by soft count = 692\n",
      "--- gene=76, id=38 removed by soft count = 5399\n",
      "--- gene=76, id=40 removed by soft count = 476\n",
      "--- gene=76, id=46 removed by soft count = 488\n",
      "--- gene=76, id=53 removed by soft count = 389\n",
      "--- gene=76, id=55 removed by soft count = 148\n",
      "--- gene=76, id=58 removed by soft count = 462\n",
      "--- gene=76, id=60 removed by soft count = 224\n",
      "--- gene=76, id=61 removed by soft count = 231\n",
      "--- gene=76, id=62 removed by soft count = 321\n",
      "--- gene=76, id=66 removed by soft count = 83\n",
      "--- gene=76, id=67 removed by soft count = 65\n",
      "--- gene=76, id=69 removed by soft count = 305\n",
      "--- gene=76, id=70 removed by soft count = 152\n",
      "--- gene=76, id=117 removed by soft count = 72\n",
      "--- gene=76, id=120 removed by soft count = 60\n",
      "--- number of probes: 121 , kept by blast: 86 , if remove dups: 86\n",
      "-- number of probes kept for this region: 86\n",
      "-- checking probes in region: 760\n",
      "--- gene=760, id=198 removed by soft count = 2081\n",
      "--- gene=760, id=206 removed by soft count = 378\n",
      "--- gene=760, id=207 removed by soft count = 64\n",
      "--- number of probes: 220 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 761\n",
      "--- gene=761, id=80 removed by soft count = 280\n",
      "--- gene=761, id=83 removed by soft count = 108\n",
      "--- gene=761, id=84 removed by soft count = 65\n",
      "--- gene=761, id=86 removed by soft count = 61\n",
      "--- gene=761, id=192 removed by soft count = 86\n",
      "--- number of probes: 222 , kept by blast: 217 , if remove dups: 216\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 762\n",
      "--- gene=762, id=7 removed by soft count = 74\n",
      "--- gene=762, id=50 removed by soft count = 5734\n",
      "--- gene=762, id=60 removed by soft count = 112\n",
      "--- gene=762, id=62 removed by soft count = 108\n",
      "--- gene=762, id=100 removed by soft count = 415\n",
      "--- number of probes: 140 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 763\n",
      "--- gene=763, id=31 removed by soft count = 74\n",
      "--- gene=763, id=62 removed by soft count = 392\n",
      "--- gene=763, id=70 removed by soft count = 335\n",
      "--- gene=763, id=71 removed by soft count = 511\n",
      "--- gene=763, id=72 removed by soft count = 64\n",
      "--- number of probes: 270 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 764\n",
      "--- gene=764, id=18 removed by soft count = 387\n",
      "--- gene=764, id=23 removed by soft count = 63\n",
      "--- gene=764, id=206 removed by soft count = 1364\n",
      "--- gene=764, id=207 removed by soft count = 253\n",
      "--- gene=764, id=223 removed by soft count = 132\n",
      "--- gene=764, id=246 removed by soft count = 898\n",
      "--- gene=764, id=249 removed by soft count = 146\n",
      "--- number of probes: 261 , kept by blast: 254 , if remove dups: 254\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 765\n",
      "--- gene=765, id=3 removed by soft count = 226\n",
      "--- gene=765, id=100 removed by soft count = 93\n",
      "--- gene=765, id=102 removed by soft count = 187\n",
      "--- gene=765, id=103 removed by soft count = 74\n",
      "--- gene=765, id=114 removed by soft count = 1030\n",
      "--- gene=765, id=145 removed by soft count = 534\n",
      "--- gene=765, id=146 removed by soft count = 623\n",
      "--- gene=765, id=149 removed by soft count = 168\n",
      "--- number of probes: 191 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 766\n",
      "--- gene=766, id=55 removed by soft count = 254\n",
      "--- gene=766, id=57 removed by soft count = 112\n",
      "--- gene=766, id=58 removed by soft count = 247\n",
      "--- gene=766, id=99 removed by soft count = 152\n",
      "--- gene=766, id=100 removed by soft count = 238\n",
      "--- gene=766, id=216 removed by soft count = 66\n",
      "--- gene=766, id=240 removed by soft count = 150\n",
      "--- gene=766, id=242 removed by soft count = 166\n",
      "--- number of probes: 254 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 767\n",
      "--- gene=767, id=128 removed by soft count = 3389\n",
      "--- gene=767, id=160 removed by soft count = 371\n",
      "--- gene=767, id=161 removed by soft count = 58\n",
      "--- gene=767, id=163 removed by soft count = 667\n",
      "--- gene=767, id=164 removed by soft count = 171\n",
      "--- number of probes: 199 , kept by blast: 194 , if remove dups: 194\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 768\n",
      "--- gene=768, id=23 removed by soft count = 150\n",
      "--- gene=768, id=24 removed by soft count = 52\n",
      "--- gene=768, id=26 removed by soft count = 69\n",
      "--- gene=768, id=227 removed by soft count = 88\n",
      "--- number of probes: 278 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 769\n",
      "--- gene=769, id=31 removed by soft count = 63\n",
      "--- gene=769, id=40 removed by soft count = 918\n",
      "--- gene=769, id=41 removed by soft count = 217\n",
      "--- number of probes: 160 , kept by blast: 157 , if remove dups: 157\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 77\n",
      "--- gene=77, id=51 removed by soft count = 143\n",
      "--- gene=77, id=88 removed by soft count = 285\n",
      "--- gene=77, id=89 removed by soft count = 187\n",
      "--- gene=77, id=130 removed by soft count = 69\n",
      "--- gene=77, id=171 removed by soft count = 53\n",
      "--- gene=77, id=172 removed by soft count = 221\n",
      "--- gene=77, id=181 removed by soft count = 105\n",
      "--- gene=77, id=184 removed by soft count = 129\n",
      "--- number of probes: 218 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 770\n",
      "--- gene=770, id=13 removed by soft count = 1666\n",
      "--- gene=770, id=69 removed by soft count = 85\n",
      "--- number of probes: 81 , kept by blast: 79 , if remove dups: 79\n",
      "-- number of probes kept for this region: 79\n",
      "-- checking probes in region: 774\n",
      "--- gene=774, id=0 removed by soft count = 148\n",
      "--- gene=774, id=6 removed by soft count = 83\n",
      "--- gene=774, id=29 removed by soft count = 78\n",
      "--- gene=774, id=31 removed by soft count = 412\n",
      "--- gene=774, id=35 removed by soft count = 112\n",
      "--- gene=774, id=36 removed by soft count = 306\n",
      "--- gene=774, id=38 removed by soft count = 59\n",
      "--- gene=774, id=43 removed by soft count = 115\n",
      "--- gene=774, id=44 removed by soft count = 311\n",
      "--- gene=774, id=45 removed by soft count = 60\n",
      "--- gene=774, id=47 removed by soft count = 53\n",
      "--- gene=774, id=64 removed by soft count = 168\n",
      "--- gene=774, id=67 removed by soft count = 73\n",
      "--- gene=774, id=71 removed by soft count = 51\n",
      "--- gene=774, id=76 removed by soft count = 61\n",
      "--- gene=774, id=86 removed by soft count = 120\n",
      "--- gene=774, id=87 removed by soft count = 74\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=774, id=92 removed by soft count = 60\n",
      "--- gene=774, id=112 removed by soft count = 90\n",
      "--- gene=774, id=113 removed by soft count = 240\n",
      "--- gene=774, id=114 removed by soft count = 100\n",
      "--- gene=774, id=133 removed by soft count = 107\n",
      "--- gene=774, id=134 removed by soft count = 96\n",
      "--- gene=774, id=135 removed by soft count = 123\n",
      "--- gene=774, id=137 removed by soft count = 264\n",
      "--- gene=774, id=140 removed by soft count = 51\n",
      "--- gene=774, id=141 removed by soft count = 246\n",
      "--- gene=774, id=143 removed by soft count = 91\n",
      "--- gene=774, id=146 removed by soft count = 79\n",
      "--- gene=774, id=190 removed by soft count = 59\n",
      "--- gene=774, id=196 removed by soft count = 328\n",
      "--- number of probes: 202 , kept by blast: 171 , if remove dups: 171\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 775\n",
      "--- gene=775, id=0 removed by soft count = 1471\n",
      "--- gene=775, id=1 removed by soft count = 12738\n",
      "--- gene=775, id=2 removed by soft count = 4524\n",
      "--- gene=775, id=3 removed by soft count = 4275\n",
      "--- gene=775, id=4 removed by soft count = 375\n",
      "--- gene=775, id=5 removed by soft count = 78\n",
      "--- gene=775, id=6 removed by soft count = 203\n",
      "--- gene=775, id=10 removed by soft count = 381\n",
      "--- gene=775, id=85 removed by soft count = 119\n",
      "--- gene=775, id=104 removed by soft count = 51\n",
      "--- gene=775, id=106 removed by soft count = 65\n",
      "--- gene=775, id=107 removed by soft count = 53\n",
      "--- gene=775, id=111 removed by soft count = 51\n",
      "--- gene=775, id=112 removed by soft count = 76\n",
      "--- gene=775, id=113 removed by soft count = 56\n",
      "--- gene=775, id=119 removed by soft count = 69\n",
      "--- gene=775, id=120 removed by soft count = 62\n",
      "--- gene=775, id=121 removed by soft count = 212\n",
      "--- gene=775, id=122 removed by soft count = 392\n",
      "--- gene=775, id=123 removed by soft count = 311\n",
      "--- gene=775, id=124 removed by soft count = 390\n",
      "--- gene=775, id=125 removed by soft count = 273\n",
      "--- gene=775, id=174 removed by soft count = 70\n",
      "--- gene=775, id=175 removed by soft count = 94\n",
      "--- gene=775, id=191 removed by soft count = 57\n",
      "--- gene=775, id=193 removed by soft count = 51\n",
      "--- gene=775, id=195 removed by soft count = 54\n",
      "--- gene=775, id=197 removed by soft count = 63\n",
      "--- gene=775, id=200 removed by soft count = 220\n",
      "--- gene=775, id=201 removed by soft count = 162\n",
      "--- number of probes: 202 , kept by blast: 172 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 776\n",
      "--- gene=776, id=0 removed by soft count = 87\n",
      "--- gene=776, id=17 removed by soft count = 84\n",
      "--- number of probes: 317 , kept by blast: 315 , if remove dups: 315\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 777\n",
      "--- gene=777, id=68 removed by soft count = 56\n",
      "--- gene=777, id=212 removed by soft count = 60\n",
      "--- number of probes: 372 , kept by blast: 370 , if remove dups: 370\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 778\n",
      "--- gene=778, id=14 removed by soft count = 99\n",
      "--- gene=778, id=221 removed by soft count = 848\n",
      "--- number of probes: 252 , kept by blast: 250 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 779\n",
      "--- gene=779, id=87 removed by soft count = 118\n",
      "--- gene=779, id=120 removed by soft count = 63\n",
      "--- gene=779, id=121 removed by soft count = 153\n",
      "--- gene=779, id=124 removed by soft count = 538\n",
      "--- number of probes: 208 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 78\n",
      "--- gene=78, id=30 removed by soft count = 63\n",
      "--- gene=78, id=56 removed by soft count = 131\n",
      "--- gene=78, id=60 removed by soft count = 57\n",
      "--- gene=78, id=61 removed by soft count = 372\n",
      "--- gene=78, id=65 removed by soft count = 80\n",
      "--- gene=78, id=66 removed by soft count = 409\n",
      "--- gene=78, id=67 removed by soft count = 63\n",
      "--- gene=78, id=71 removed by soft count = 136\n",
      "--- gene=78, id=73 removed by soft count = 711\n",
      "--- gene=78, id=74 removed by soft count = 54\n",
      "--- gene=78, id=75 removed by soft count = 479\n",
      "--- gene=78, id=78 removed by soft count = 227\n",
      "--- gene=78, id=79 removed by soft count = 518\n",
      "--- number of probes: 142 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 780\n",
      "--- gene=780, id=65 removed by soft count = 63\n",
      "--- gene=780, id=75 removed by soft count = 166\n",
      "--- gene=780, id=138 removed by soft count = 58\n",
      "--- gene=780, id=348 removed by soft count = 52\n",
      "--- number of probes: 349 , kept by blast: 345 , if remove dups: 345\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 782\n",
      "--- gene=782, id=163 removed by soft count = 148\n",
      "--- gene=782, id=164 removed by soft count = 100\n",
      "--- gene=782, id=165 removed by soft count = 55\n",
      "--- number of probes: 206 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 783\n",
      "--- gene=783, id=78 removed by soft count = 202\n",
      "--- gene=783, id=79 removed by soft count = 80\n",
      "--- gene=783, id=103 removed by soft count = 498\n",
      "--- gene=783, id=104 removed by soft count = 644\n",
      "--- gene=783, id=106 removed by soft count = 55\n",
      "--- gene=783, id=127 removed by soft count = 65\n",
      "--- number of probes: 285 , kept by blast: 279 , if remove dups: 279\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 784\n",
      "--- gene=784, id=28 removed by soft count = 184\n",
      "--- gene=784, id=114 removed by soft count = 247\n",
      "--- gene=784, id=115 removed by soft count = 199\n",
      "--- gene=784, id=122 removed by soft count = 109\n",
      "--- gene=784, id=137 removed by soft count = 74\n",
      "--- gene=784, id=142 removed by soft count = 886\n",
      "--- gene=784, id=158 removed by soft count = 347\n",
      "--- gene=784, id=185 removed by soft count = 4776\n",
      "--- gene=784, id=187 removed by soft count = 169\n",
      "--- number of probes: 196 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 785\n",
      "--- gene=785, id=15 removed by soft count = 94\n",
      "--- gene=785, id=84 removed by soft count = 64\n",
      "--- gene=785, id=135 removed by soft count = 76\n",
      "--- gene=785, id=159 removed by soft count = 136\n",
      "--- gene=785, id=162 removed by soft count = 52\n",
      "--- gene=785, id=173 removed by soft count = 95\n",
      "--- gene=785, id=176 removed by soft count = 170\n",
      "--- gene=785, id=177 removed by soft count = 232\n",
      "--- gene=785, id=178 removed by soft count = 126\n",
      "--- gene=785, id=180 removed by soft count = 66\n",
      "--- gene=785, id=183 removed by soft count = 1307\n",
      "--- gene=785, id=185 removed by soft count = 145\n",
      "--- gene=785, id=195 removed by soft count = 132\n",
      "--- number of probes: 295 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 786\n",
      "--- gene=786, id=129 removed by soft count = 118\n",
      "--- gene=786, id=130 removed by soft count = 77\n",
      "--- number of probes: 234 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 787\n",
      "--- gene=787, id=214 removed by soft count = 134\n",
      "--- number of probes: 293 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 788\n",
      "--- gene=788, id=1 removed by soft count = 164\n",
      "--- gene=788, id=2 removed by soft count = 115\n",
      "--- gene=788, id=7 removed by soft count = 236\n",
      "--- gene=788, id=9 removed by soft count = 325\n",
      "--- gene=788, id=102 removed by soft count = 63\n",
      "--- gene=788, id=103 removed by soft count = 108\n",
      "--- gene=788, id=114 removed by soft count = 364\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=788, id=120 removed by soft count = 4304\n",
      "--- number of probes: 182 , kept by blast: 174 , if remove dups: 173\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 789\n",
      "--- gene=789, id=30 removed by soft count = 124\n",
      "--- gene=789, id=57 removed by soft count = 2456\n",
      "--- gene=789, id=81 removed by soft count = 9818\n",
      "--- gene=789, id=89 removed by soft count = 51\n",
      "--- gene=789, id=95 removed by soft count = 397\n",
      "--- gene=789, id=173 removed by soft count = 242\n",
      "--- gene=789, id=174 removed by soft count = 112\n",
      "--- number of probes: 209 , kept by blast: 202 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 79\n",
      "--- gene=79, id=6 removed by soft count = 153\n",
      "--- gene=79, id=7 removed by soft count = 227\n",
      "--- gene=79, id=9 removed by soft count = 551\n",
      "--- gene=79, id=11 removed by soft count = 110\n",
      "--- gene=79, id=173 removed by soft count = 135\n",
      "--- gene=79, id=174 removed by soft count = 765\n",
      "--- gene=79, id=175 removed by soft count = 635\n",
      "--- number of probes: 198 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 790\n",
      "--- gene=790, id=22 removed by soft count = 64\n",
      "--- number of probes: 91 , kept by blast: 90 , if remove dups: 87\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 87\n",
      "-- checking probes in region: 791\n",
      "--- gene=791, id=31 removed by soft count = 60\n",
      "--- gene=791, id=84 removed by soft count = 364\n",
      "--- gene=791, id=149 removed by soft count = 345\n",
      "--- gene=791, id=154 removed by soft count = 60\n",
      "--- number of probes: 164 , kept by blast: 160 , if remove dups: 160\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 792\n",
      "--- gene=792, id=10 removed by soft count = 227\n",
      "--- gene=792, id=11 removed by soft count = 474\n",
      "--- gene=792, id=12 removed by soft count = 483\n",
      "--- gene=792, id=14 removed by soft count = 321\n",
      "--- gene=792, id=15 removed by soft count = 570\n",
      "--- gene=792, id=17 removed by soft count = 53\n",
      "--- gene=792, id=35 removed by soft count = 51\n",
      "--- gene=792, id=37 removed by hard count = 3\n",
      "--- gene=792, id=38 removed by soft count = 105\n",
      "--- gene=792, id=40 removed by soft count = 3262\n",
      "--- gene=792, id=42 removed by soft count = 52\n",
      "--- gene=792, id=45 removed by soft count = 127\n",
      "--- gene=792, id=47 removed by soft count = 102\n",
      "--- gene=792, id=48 removed by soft count = 795\n",
      "--- gene=792, id=50 removed by soft count = 195\n",
      "--- gene=792, id=56 removed by soft count = 62\n",
      "--- gene=792, id=58 removed by soft count = 83\n",
      "--- gene=792, id=70 removed by soft count = 64\n",
      "--- gene=792, id=167 removed by soft count = 152\n",
      "--- number of probes: 168 , kept by blast: 149 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 793\n",
      "--- gene=793, id=130 removed by soft count = 1245\n",
      "--- gene=793, id=133 removed by soft count = 53\n",
      "--- gene=793, id=183 removed by soft count = 142\n",
      "--- gene=793, id=184 removed by soft count = 219\n",
      "--- gene=793, id=186 removed by soft count = 98\n",
      "--- number of probes: 231 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 794\n",
      "--- gene=794, id=30 removed by soft count = 62\n",
      "--- gene=794, id=53 removed by soft count = 55\n",
      "--- gene=794, id=59 removed by soft count = 104\n",
      "--- gene=794, id=81 removed by soft count = 199\n",
      "--- gene=794, id=88 removed by soft count = 80\n",
      "--- gene=794, id=261 removed by soft count = 109\n",
      "--- number of probes: 298 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 795\n",
      "--- gene=795, id=1 removed by soft count = 68\n",
      "--- gene=795, id=3 removed by soft count = 155\n",
      "--- gene=795, id=4 removed by soft count = 88\n",
      "--- gene=795, id=5 removed by soft count = 114\n",
      "--- gene=795, id=14 removed by soft count = 60\n",
      "--- gene=795, id=16 removed by soft count = 117\n",
      "--- gene=795, id=47 removed by soft count = 4904\n",
      "--- gene=795, id=48 removed by soft count = 4149\n",
      "--- gene=795, id=49 removed by soft count = 465\n",
      "--- gene=795, id=57 removed by soft count = 95\n",
      "--- gene=795, id=110 removed by soft count = 226\n",
      "--- gene=795, id=112 removed by soft count = 52\n",
      "--- number of probes: 126 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 114\n",
      "-- checking probes in region: 796\n",
      "--- gene=796, id=36 removed by soft count = 556\n",
      "--- gene=796, id=37 removed by soft count = 62\n",
      "--- gene=796, id=101 removed by soft count = 195\n",
      "--- gene=796, id=102 removed by soft count = 174\n",
      "--- gene=796, id=182 removed by soft count = 140\n",
      "--- gene=796, id=200 removed by soft count = 79\n",
      "--- number of probes: 289 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 797\n",
      "--- gene=797, id=80 removed by soft count = 503\n",
      "--- gene=797, id=147 removed by soft count = 119\n",
      "--- gene=797, id=148 removed by soft count = 84\n",
      "--- gene=797, id=154 removed by soft count = 936\n",
      "--- gene=797, id=192 removed by soft count = 86\n",
      "--- gene=797, id=201 removed by soft count = 128\n",
      "--- gene=797, id=204 removed by soft count = 96\n",
      "--- gene=797, id=205 removed by soft count = 149\n",
      "--- gene=797, id=206 removed by soft count = 237\n",
      "--- gene=797, id=218 removed by soft count = 203\n",
      "--- number of probes: 249 , kept by blast: 239 , if remove dups: 239\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 798\n",
      "--- gene=798, id=5 removed by soft count = 54\n",
      "--- gene=798, id=11 removed by soft count = 92\n",
      "--- gene=798, id=14 removed by soft count = 164\n",
      "--- gene=798, id=19 removed by soft count = 73\n",
      "--- gene=798, id=20 removed by soft count = 249\n",
      "--- gene=798, id=21 removed by soft count = 115\n",
      "--- gene=798, id=22 removed by soft count = 54\n",
      "--- gene=798, id=23 removed by soft count = 67\n",
      "--- gene=798, id=61 removed by soft count = 115\n",
      "--- gene=798, id=137 removed by soft count = 59\n",
      "--- gene=798, id=225 removed by soft count = 1054\n",
      "--- gene=798, id=227 removed by soft count = 61\n",
      "--- number of probes: 244 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 799\n",
      "--- number of probes: 289 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 8\n",
      "--- number of probes: 323 , kept by blast: 323 , if remove dups: 323\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 80\n",
      "--- gene=80, id=34 removed by soft count = 174\n",
      "--- gene=80, id=72 removed by soft count = 60\n",
      "--- gene=80, id=151 removed by soft count = 122\n",
      "--- gene=80, id=152 removed by soft count = 215\n",
      "--- gene=80, id=153 removed by soft count = 195\n",
      "--- gene=80, id=154 removed by soft count = 125\n",
      "--- gene=80, id=155 removed by soft count = 133\n",
      "--- gene=80, id=156 removed by soft count = 148\n",
      "--- gene=80, id=157 removed by soft count = 191\n",
      "--- gene=80, id=158 removed by soft count = 162\n",
      "--- number of probes: 241 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 800\n",
      "--- gene=800, id=35 removed by soft count = 156\n",
      "--- gene=800, id=80 removed by soft count = 316\n",
      "--- gene=800, id=81 removed by soft count = 1001\n",
      "--- gene=800, id=83 removed by soft count = 133\n",
      "--- gene=800, id=87 removed by soft count = 102\n",
      "--- gene=800, id=93 removed by soft count = 166\n",
      "--- gene=800, id=95 removed by soft count = 121\n",
      "--- gene=800, id=96 removed by soft count = 307\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=800, id=131 removed by soft count = 446\n",
      "--- gene=800, id=144 removed by soft count = 95\n",
      "--- gene=800, id=146 removed by soft count = 801\n",
      "--- number of probes: 196 , kept by blast: 185 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 801\n",
      "--- gene=801, id=55 removed by soft count = 180\n",
      "--- gene=801, id=120 removed by soft count = 76\n",
      "--- gene=801, id=123 removed by soft count = 54\n",
      "--- gene=801, id=124 removed by soft count = 63\n",
      "--- gene=801, id=171 removed by soft count = 54\n",
      "--- gene=801, id=173 removed by soft count = 61\n",
      "--- number of probes: 320 , kept by blast: 314 , if remove dups: 314\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 802\n",
      "--- gene=802, id=3 removed by soft count = 63\n",
      "--- gene=802, id=12 removed by soft count = 64\n",
      "--- number of probes: 257 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 803\n",
      "--- gene=803, id=146 removed by soft count = 110\n",
      "--- gene=803, id=184 removed by soft count = 210\n",
      "--- gene=803, id=185 removed by soft count = 351\n",
      "--- gene=803, id=186 removed by soft count = 101\n",
      "--- number of probes: 289 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 804\n",
      "--- gene=804, id=126 removed by soft count = 189\n",
      "--- number of probes: 220 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 805\n",
      "--- gene=805, id=25 removed by soft count = 4171\n",
      "--- gene=805, id=35 removed by soft count = 90\n",
      "--- gene=805, id=47 removed by soft count = 156\n",
      "--- gene=805, id=91 removed by soft count = 56\n",
      "--- gene=805, id=94 removed by soft count = 76\n",
      "--- gene=805, id=95 removed by soft count = 55\n",
      "--- gene=805, id=96 removed by soft count = 78\n",
      "--- gene=805, id=102 removed by soft count = 89\n",
      "--- gene=805, id=115 removed by soft count = 277\n",
      "--- gene=805, id=132 removed by soft count = 92\n",
      "--- number of probes: 168 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 806\n",
      "--- gene=806, id=281 removed by soft count = 169\n",
      "--- gene=806, id=282 removed by soft count = 354\n",
      "--- number of probes: 291 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 807\n",
      "--- number of probes: 233 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 808\n",
      "--- gene=808, id=26 removed by soft count = 70\n",
      "--- gene=808, id=27 removed by soft count = 503\n",
      "--- gene=808, id=28 removed by soft count = 789\n",
      "--- gene=808, id=30 removed by soft count = 128\n",
      "--- gene=808, id=31 removed by soft count = 1010\n",
      "--- gene=808, id=33 removed by soft count = 53\n",
      "--- gene=808, id=123 removed by soft count = 76\n",
      "--- gene=808, id=129 removed by soft count = 148\n",
      "--- gene=808, id=130 removed by soft count = 87\n",
      "--- gene=808, id=153 removed by soft count = 766\n",
      "--- gene=808, id=158 removed by soft count = 102\n",
      "--- number of probes: 197 , kept by blast: 186 , if remove dups: 186\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 809\n",
      "--- gene=809, id=311 removed by soft count = 579\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 81\n",
      "--- gene=81, id=181 removed by soft count = 216\n",
      "--- gene=81, id=182 removed by soft count = 1061\n",
      "--- gene=81, id=189 removed by soft count = 814\n",
      "--- gene=81, id=199 removed by soft count = 1580\n",
      "--- number of probes: 223 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 810\n",
      "--- gene=810, id=71 removed by soft count = 1201\n",
      "--- gene=810, id=95 removed by soft count = 430\n",
      "--- gene=810, id=97 removed by soft count = 653\n",
      "--- gene=810, id=101 removed by soft count = 156\n",
      "--- gene=810, id=107 removed by soft count = 150\n",
      "--- number of probes: 160 , kept by blast: 155 , if remove dups: 155\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 811\n",
      "--- gene=811, id=173 removed by soft count = 183\n",
      "--- gene=811, id=200 removed by soft count = 58\n",
      "--- number of probes: 234 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 812\n",
      "--- gene=812, id=168 removed by soft count = 148\n",
      "--- gene=812, id=195 removed by soft count = 765\n",
      "--- gene=812, id=255 removed by soft count = 79\n",
      "--- number of probes: 358 , kept by blast: 355 , if remove dups: 355\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 813\n",
      "--- gene=813, id=6 removed by soft count = 959\n",
      "--- gene=813, id=9 removed by soft count = 100\n",
      "--- gene=813, id=124 removed by soft count = 66\n",
      "--- number of probes: 187 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 814\n",
      "--- gene=814, id=29 removed by soft count = 239\n",
      "--- gene=814, id=92 removed by soft count = 101\n",
      "--- gene=814, id=122 removed by soft count = 380\n",
      "--- gene=814, id=126 removed by soft count = 231\n",
      "--- gene=814, id=136 removed by soft count = 87\n",
      "--- gene=814, id=177 removed by soft count = 101\n",
      "--- gene=814, id=179 removed by soft count = 160\n",
      "--- gene=814, id=258 removed by soft count = 2208\n",
      "--- gene=814, id=259 removed by soft count = 299\n",
      "--- number of probes: 275 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 815\n",
      "--- gene=815, id=243 removed by soft count = 64\n",
      "--- gene=815, id=260 removed by soft count = 66\n",
      "--- number of probes: 328 , kept by blast: 326 , if remove dups: 325\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 816\n",
      "--- gene=816, id=42 removed by soft count = 132\n",
      "--- gene=816, id=43 removed by soft count = 77\n",
      "--- gene=816, id=130 removed by soft count = 314\n",
      "--- gene=816, id=131 removed by soft count = 191\n",
      "--- number of probes: 209 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 817\n",
      "--- gene=817, id=1 removed by soft count = 70\n",
      "--- gene=817, id=4 removed by soft count = 157\n",
      "--- gene=817, id=115 removed by soft count = 208\n",
      "--- gene=817, id=116 removed by soft count = 99\n",
      "--- gene=817, id=137 removed by soft count = 121\n",
      "--- gene=817, id=159 removed by soft count = 379\n",
      "--- gene=817, id=192 removed by soft count = 129\n",
      "--- gene=817, id=195 removed by soft count = 165\n",
      "--- gene=817, id=200 removed by soft count = 212\n",
      "--- gene=817, id=201 removed by soft count = 53\n",
      "--- gene=817, id=202 removed by soft count = 187\n",
      "--- number of probes: 204 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 819\n",
      "--- gene=819, id=51 removed by soft count = 159\n",
      "--- gene=819, id=53 removed by soft count = 333\n",
      "--- gene=819, id=54 removed by soft count = 114\n",
      "--- gene=819, id=144 removed by soft count = 54\n",
      "--- gene=819, id=214 removed by soft count = 20431\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 268 , kept by blast: 263 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 820\n",
      "--- gene=820, id=28 removed by soft count = 410\n",
      "--- gene=820, id=29 removed by soft count = 383\n",
      "--- gene=820, id=34 removed by soft count = 8096\n",
      "--- gene=820, id=35 removed by soft count = 319\n",
      "--- gene=820, id=53 removed by soft count = 139\n",
      "--- gene=820, id=56 removed by soft count = 137\n",
      "--- gene=820, id=58 removed by soft count = 188\n",
      "--- gene=820, id=60 removed by soft count = 81\n",
      "--- number of probes: 255 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 821\n",
      "--- gene=821, id=107 removed by soft count = 86\n",
      "--- gene=821, id=117 removed by soft count = 1684\n",
      "--- gene=821, id=131 removed by soft count = 95\n",
      "--- gene=821, id=134 removed by soft count = 895\n",
      "--- gene=821, id=135 removed by soft count = 96\n",
      "--- gene=821, id=136 removed by soft count = 910\n",
      "--- gene=821, id=138 removed by soft count = 111\n",
      "--- gene=821, id=141 removed by soft count = 124\n",
      "--- number of probes: 142 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 822\n",
      "--- gene=822, id=17 removed by soft count = 433\n",
      "--- gene=822, id=211 removed by soft count = 112\n",
      "--- gene=822, id=217 removed by soft count = 83\n",
      "--- gene=822, id=218 removed by soft count = 120\n",
      "--- gene=822, id=219 removed by soft count = 71\n",
      "--- gene=822, id=220 removed by soft count = 52\n",
      "--- number of probes: 257 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 823\n",
      "--- gene=823, id=0 removed by soft count = 431\n",
      "--- gene=823, id=3 removed by soft count = 73\n",
      "--- gene=823, id=5 removed by soft count = 251\n",
      "--- gene=823, id=8 removed by soft count = 103\n",
      "--- gene=823, id=11 removed by soft count = 58\n",
      "--- gene=823, id=20 removed by soft count = 57\n",
      "--- gene=823, id=25 removed by soft count = 115\n",
      "--- gene=823, id=40 removed by soft count = 66\n",
      "--- gene=823, id=43 removed by soft count = 58\n",
      "--- gene=823, id=44 removed by soft count = 2181\n",
      "--- gene=823, id=46 removed by soft count = 57\n",
      "--- gene=823, id=47 removed by soft count = 63\n",
      "--- gene=823, id=50 removed by soft count = 75\n",
      "--- gene=823, id=52 removed by soft count = 150\n",
      "--- gene=823, id=53 removed by soft count = 57\n",
      "--- gene=823, id=54 removed by soft count = 488\n",
      "--- gene=823, id=69 removed by soft count = 63\n",
      "--- gene=823, id=70 removed by soft count = 85\n",
      "--- gene=823, id=84 removed by soft count = 59\n",
      "--- gene=823, id=92 removed by soft count = 18008\n",
      "--- gene=823, id=114 removed by soft count = 154\n",
      "--- gene=823, id=117 removed by soft count = 630\n",
      "--- number of probes: 129 , kept by blast: 107 , if remove dups: 107\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 107\n",
      "-- checking probes in region: 824\n",
      "--- gene=824, id=137 removed by soft count = 58\n",
      "--- gene=824, id=188 removed by soft count = 185\n",
      "--- gene=824, id=274 removed by soft count = 15799\n",
      "--- gene=824, id=275 removed by soft count = 8462\n",
      "--- number of probes: 288 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 825\n",
      "--- gene=825, id=147 removed by soft count = 356\n",
      "--- gene=825, id=173 removed by soft count = 103\n",
      "--- gene=825, id=174 removed by soft count = 366\n",
      "--- gene=825, id=175 removed by soft count = 68\n",
      "--- gene=825, id=184 removed by soft count = 417\n",
      "--- gene=825, id=187 removed by soft count = 134\n",
      "--- number of probes: 225 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 826\n",
      "--- gene=826, id=4 removed by soft count = 201\n",
      "--- gene=826, id=77 removed by soft count = 138\n",
      "--- gene=826, id=78 removed by soft count = 57\n",
      "--- gene=826, id=79 removed by soft count = 387\n",
      "--- gene=826, id=92 removed by soft count = 100\n",
      "--- gene=826, id=137 removed by soft count = 232\n",
      "--- gene=826, id=195 removed by soft count = 63\n",
      "--- gene=826, id=200 removed by soft count = 431\n",
      "--- number of probes: 263 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 827\n",
      "--- gene=827, id=205 removed by soft count = 742\n",
      "--- gene=827, id=225 removed by soft count = 66\n",
      "--- gene=827, id=246 removed by soft count = 832\n",
      "--- number of probes: 283 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 828\n",
      "--- number of probes: 377 , kept by blast: 377 , if remove dups: 377\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 829\n",
      "--- gene=829, id=172 removed by soft count = 155\n",
      "--- gene=829, id=217 removed by soft count = 51\n",
      "--- gene=829, id=220 removed by soft count = 94\n",
      "--- gene=829, id=230 removed by soft count = 595\n",
      "--- gene=829, id=231 removed by soft count = 1828\n",
      "--- number of probes: 237 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 83\n",
      "--- gene=83, id=31 removed by soft count = 313\n",
      "--- gene=83, id=32 removed by soft count = 1845\n",
      "--- gene=83, id=58 removed by soft count = 76\n",
      "--- gene=83, id=60 removed by soft count = 187\n",
      "--- gene=83, id=104 removed by soft count = 199\n",
      "--- number of probes: 259 , kept by blast: 254 , if remove dups: 254\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 832\n",
      "--- gene=832, id=8 removed by soft count = 241\n",
      "--- number of probes: 168 , kept by blast: 167 , if remove dups: 167\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 833\n",
      "--- gene=833, id=122 removed by soft count = 130\n",
      "--- gene=833, id=124 removed by soft count = 271\n",
      "--- gene=833, id=125 removed by soft count = 490\n",
      "--- gene=833, id=126 removed by soft count = 140\n",
      "--- gene=833, id=127 removed by soft count = 100\n",
      "--- gene=833, id=128 removed by soft count = 94\n",
      "--- gene=833, id=129 removed by soft count = 209\n",
      "--- gene=833, id=141 removed by soft count = 202\n",
      "--- gene=833, id=144 removed by soft count = 110\n",
      "--- gene=833, id=145 removed by soft count = 76\n",
      "--- gene=833, id=177 removed by soft count = 70\n",
      "--- gene=833, id=180 removed by soft count = 53\n",
      "--- gene=833, id=189 removed by soft count = 82\n",
      "--- number of probes: 204 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 834\n",
      "--- gene=834, id=179 removed by soft count = 480\n",
      "--- gene=834, id=193 removed by soft count = 1109\n",
      "--- gene=834, id=232 removed by soft count = 54\n",
      "--- gene=834, id=309 removed by soft count = 170\n",
      "--- number of probes: 310 , kept by blast: 306 , if remove dups: 306\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 835\n",
      "--- gene=835, id=8 removed by soft count = 961\n",
      "--- gene=835, id=17 removed by soft count = 79\n",
      "--- gene=835, id=20 removed by soft count = 583\n",
      "--- gene=835, id=170 removed by soft count = 485\n",
      "--- number of probes: 236 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 836\n",
      "--- gene=836, id=0 removed by soft count = 81\n",
      "--- gene=836, id=2 removed by soft count = 311\n",
      "--- gene=836, id=3 removed by soft count = 134\n",
      "--- gene=836, id=6 removed by soft count = 98\n",
      "--- gene=836, id=7 removed by soft count = 723\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=836, id=11 removed by soft count = 91\n",
      "--- gene=836, id=48 removed by soft count = 1046\n",
      "--- gene=836, id=79 removed by soft count = 87\n",
      "--- number of probes: 164 , kept by blast: 156 , if remove dups: 156\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 837\n",
      "--- gene=837, id=223 removed by soft count = 311\n",
      "--- number of probes: 302 , kept by blast: 301 , if remove dups: 301\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 838\n",
      "--- gene=838, id=45 removed by soft count = 899\n",
      "--- gene=838, id=173 removed by soft count = 221\n",
      "--- gene=838, id=174 removed by soft count = 259\n",
      "--- gene=838, id=175 removed by soft count = 319\n",
      "--- number of probes: 226 , kept by blast: 222 , if remove dups: 222\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 839\n",
      "--- gene=839, id=13 removed by soft count = 6873\n",
      "--- gene=839, id=60 removed by soft count = 74\n",
      "--- gene=839, id=61 removed by soft count = 63\n",
      "--- gene=839, id=65 removed by soft count = 100\n",
      "--- gene=839, id=66 removed by soft count = 37323\n",
      "--- gene=839, id=89 removed by soft count = 393\n",
      "--- gene=839, id=90 removed by soft count = 508\n",
      "--- number of probes: 194 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 84\n",
      "--- gene=84, id=21 removed by soft count = 74\n",
      "--- gene=84, id=67 removed by soft count = 698\n",
      "--- gene=84, id=85 removed by soft count = 93\n",
      "--- gene=84, id=86 removed by soft count = 63\n",
      "--- gene=84, id=87 removed by soft count = 1439\n",
      "--- gene=84, id=188 removed by soft count = 155\n",
      "--- number of probes: 191 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 840\n",
      "--- gene=840, id=36 removed by soft count = 89\n",
      "--- gene=840, id=48 removed by soft count = 92\n",
      "--- gene=840, id=67 removed by soft count = 1103\n",
      "--- gene=840, id=68 removed by soft count = 144\n",
      "--- gene=840, id=69 removed by soft count = 175\n",
      "--- gene=840, id=70 removed by soft count = 277\n",
      "--- gene=840, id=71 removed by soft count = 124\n",
      "--- gene=840, id=81 removed by soft count = 333\n",
      "--- gene=840, id=200 removed by soft count = 142\n",
      "--- gene=840, id=256 removed by soft count = 595\n",
      "--- gene=840, id=257 removed by soft count = 461\n",
      "--- gene=840, id=258 removed by soft count = 224\n",
      "--- number of probes: 264 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 841\n",
      "--- gene=841, id=50 removed by soft count = 228\n",
      "--- gene=841, id=51 removed by soft count = 118\n",
      "--- gene=841, id=69 removed by soft count = 210\n",
      "--- number of probes: 373 , kept by blast: 370 , if remove dups: 370\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 843\n",
      "--- gene=843, id=1 removed by soft count = 123\n",
      "--- gene=843, id=4 removed by soft count = 645\n",
      "--- gene=843, id=17 removed by soft count = 71\n",
      "--- number of probes: 148 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 844\n",
      "--- gene=844, id=36 removed by soft count = 120\n",
      "--- gene=844, id=37 removed by soft count = 57\n",
      "--- gene=844, id=39 removed by soft count = 60\n",
      "--- gene=844, id=43 removed by soft count = 58\n",
      "--- gene=844, id=49 removed by soft count = 79\n",
      "--- gene=844, id=51 removed by soft count = 422\n",
      "--- gene=844, id=115 removed by soft count = 511\n",
      "--- gene=844, id=181 removed by soft count = 57\n",
      "--- number of probes: 256 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 845\n",
      "--- gene=845, id=24 removed by soft count = 80\n",
      "--- gene=845, id=34 removed by soft count = 125\n",
      "--- gene=845, id=50 removed by soft count = 76\n",
      "--- gene=845, id=172 removed by soft count = 257\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 846\n",
      "--- gene=846, id=65 removed by soft count = 115\n",
      "--- gene=846, id=69 removed by soft count = 126\n",
      "--- gene=846, id=74 removed by soft count = 2160\n",
      "--- gene=846, id=230 removed by soft count = 91\n",
      "--- number of probes: 261 , kept by blast: 257 , if remove dups: 257\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 847\n",
      "--- gene=847, id=5 removed by soft count = 129\n",
      "--- gene=847, id=7 removed by soft count = 83\n",
      "--- gene=847, id=9 removed by soft count = 161\n",
      "--- gene=847, id=10 removed by soft count = 172\n",
      "--- gene=847, id=11 removed by soft count = 85\n",
      "--- gene=847, id=12 removed by soft count = 134\n",
      "--- gene=847, id=14 removed by soft count = 85\n",
      "--- gene=847, id=15 removed by soft count = 170\n",
      "--- gene=847, id=16 removed by soft count = 508\n",
      "--- gene=847, id=68 removed by soft count = 108\n",
      "--- gene=847, id=71 removed by soft count = 122\n",
      "--- gene=847, id=144 removed by soft count = 96\n",
      "--- gene=847, id=154 removed by soft count = 89\n",
      "--- gene=847, id=160 removed by soft count = 459\n",
      "--- gene=847, id=161 removed by soft count = 51\n",
      "--- gene=847, id=162 removed by soft count = 222\n",
      "--- gene=847, id=163 removed by soft count = 86\n",
      "--- gene=847, id=164 removed by soft count = 65\n",
      "--- gene=847, id=167 removed by soft count = 128\n",
      "--- number of probes: 174 , kept by blast: 155 , if remove dups: 155\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 848\n",
      "--- gene=848, id=139 removed by soft count = 56\n",
      "--- gene=848, id=195 removed by soft count = 289\n",
      "--- gene=848, id=198 removed by soft count = 1994\n",
      "--- gene=848, id=203 removed by soft count = 69\n",
      "--- gene=848, id=206 removed by soft count = 80\n",
      "--- number of probes: 294 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 849\n",
      "--- gene=849, id=92 removed by soft count = 56\n",
      "--- gene=849, id=93 removed by soft count = 843\n",
      "--- gene=849, id=94 removed by soft count = 142\n",
      "--- gene=849, id=95 removed by soft count = 268\n",
      "--- gene=849, id=96 removed by soft count = 90\n",
      "--- gene=849, id=97 removed by soft count = 534\n",
      "--- gene=849, id=98 removed by soft count = 2008\n",
      "--- gene=849, id=99 removed by soft count = 159\n",
      "--- gene=849, id=112 removed by soft count = 229\n",
      "--- gene=849, id=113 removed by soft count = 53\n",
      "--- gene=849, id=115 removed by soft count = 104\n",
      "--- gene=849, id=116 removed by soft count = 226\n",
      "--- gene=849, id=117 removed by soft count = 240\n",
      "--- gene=849, id=121 removed by soft count = 63\n",
      "--- number of probes: 123 , kept by blast: 109 , if remove dups: 109\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 109\n",
      "-- checking probes in region: 85\n",
      "--- gene=85, id=45 removed by soft count = 118\n",
      "--- gene=85, id=56 removed by soft count = 534\n",
      "--- gene=85, id=79 removed by soft count = 98\n",
      "--- gene=85, id=216 removed by soft count = 590\n",
      "--- gene=85, id=239 removed by soft count = 108\n",
      "--- gene=85, id=241 removed by soft count = 64\n",
      "--- gene=85, id=242 removed by soft count = 368\n",
      "--- gene=85, id=243 removed by soft count = 111\n",
      "--- number of probes: 313 , kept by blast: 305 , if remove dups: 305\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 850\n",
      "--- gene=850, id=0 removed by soft count = 290\n",
      "--- gene=850, id=3 removed by soft count = 114\n",
      "--- gene=850, id=5 removed by soft count = 140\n",
      "--- gene=850, id=130 removed by soft count = 105\n",
      "--- gene=850, id=132 removed by soft count = 328\n",
      "--- gene=850, id=176 removed by soft count = 450\n",
      "--- gene=850, id=177 removed by soft count = 82\n",
      "--- gene=850, id=178 removed by soft count = 240\n",
      "--- gene=850, id=180 removed by soft count = 289\n",
      "--- number of probes: 181 , kept by blast: 172 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 851\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=851, id=65 removed by soft count = 149\n",
      "--- gene=851, id=67 removed by soft count = 1758\n",
      "--- gene=851, id=68 removed by soft count = 284\n",
      "--- gene=851, id=70 removed by soft count = 68\n",
      "--- gene=851, id=80 removed by soft count = 105\n",
      "--- gene=851, id=116 removed by soft count = 300\n",
      "--- gene=851, id=125 removed by soft count = 61\n",
      "--- gene=851, id=126 removed by soft count = 199\n",
      "--- number of probes: 241 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 852\n",
      "--- gene=852, id=19 removed by soft count = 138\n",
      "--- number of probes: 279 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 853\n",
      "--- gene=853, id=18 removed by soft count = 59\n",
      "--- gene=853, id=26 removed by soft count = 63\n",
      "--- gene=853, id=28 removed by soft count = 58\n",
      "--- gene=853, id=32 removed by soft count = 60\n",
      "--- gene=853, id=35 removed by soft count = 69\n",
      "--- gene=853, id=88 removed by soft count = 784\n",
      "--- gene=853, id=179 removed by soft count = 695\n",
      "--- gene=853, id=191 removed by soft count = 171\n",
      "--- gene=853, id=192 removed by soft count = 1163\n",
      "--- number of probes: 194 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 854\n",
      "--- gene=854, id=0 removed by soft count = 144\n",
      "--- gene=854, id=84 removed by soft count = 1186\n",
      "--- gene=854, id=100 removed by soft count = 414\n",
      "--- gene=854, id=125 removed by soft count = 103\n",
      "--- gene=854, id=126 removed by soft count = 1009\n",
      "--- gene=854, id=136 removed by soft count = 586\n",
      "--- gene=854, id=139 removed by soft count = 174\n",
      "--- gene=854, id=183 removed by soft count = 180\n",
      "--- gene=854, id=185 removed by soft count = 71\n",
      "--- gene=854, id=188 removed by soft count = 100\n",
      "--- number of probes: 215 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 855\n",
      "--- gene=855, id=75 removed by soft count = 51\n",
      "--- gene=855, id=77 removed by soft count = 158\n",
      "--- gene=855, id=81 removed by soft count = 1576\n",
      "--- gene=855, id=86 removed by soft count = 521\n",
      "--- gene=855, id=88 removed by soft count = 74\n",
      "--- gene=855, id=142 removed by soft count = 422\n",
      "--- gene=855, id=144 removed by soft count = 297\n",
      "--- gene=855, id=145 removed by soft count = 1732\n",
      "--- gene=855, id=147 removed by soft count = 108\n",
      "--- number of probes: 199 , kept by blast: 190 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 856\n",
      "--- gene=856, id=71 removed by soft count = 65\n",
      "--- gene=856, id=72 removed by soft count = 172\n",
      "--- gene=856, id=74 removed by soft count = 61\n",
      "--- gene=856, id=75 removed by soft count = 64\n",
      "--- gene=856, id=77 removed by soft count = 57\n",
      "--- gene=856, id=79 removed by soft count = 77\n",
      "--- gene=856, id=81 removed by soft count = 146\n",
      "--- gene=856, id=82 removed by soft count = 627\n",
      "--- gene=856, id=83 removed by soft count = 199\n",
      "--- gene=856, id=84 removed by soft count = 835\n",
      "--- gene=856, id=86 removed by soft count = 123\n",
      "--- gene=856, id=88 removed by soft count = 82\n",
      "--- gene=856, id=90 removed by soft count = 131\n",
      "--- gene=856, id=91 removed by soft count = 193\n",
      "--- gene=856, id=98 removed by soft count = 346\n",
      "--- gene=856, id=100 removed by soft count = 66\n",
      "--- gene=856, id=101 removed by soft count = 322\n",
      "--- gene=856, id=106 removed by soft count = 62\n",
      "--- gene=856, id=107 removed by soft count = 721\n",
      "--- gene=856, id=109 removed by soft count = 220\n",
      "--- gene=856, id=112 removed by soft count = 72\n",
      "--- gene=856, id=113 removed by soft count = 605\n",
      "--- gene=856, id=133 removed by soft count = 143\n",
      "--- number of probes: 163 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 857\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 858\n",
      "--- gene=858, id=1 removed by soft count = 271\n",
      "--- gene=858, id=2 removed by soft count = 246\n",
      "--- gene=858, id=3 removed by soft count = 167\n",
      "--- gene=858, id=12 removed by soft count = 149\n",
      "--- gene=858, id=13 removed by soft count = 146\n",
      "--- gene=858, id=56 removed by soft count = 582\n",
      "--- gene=858, id=249 removed by soft count = 825\n",
      "--- gene=858, id=250 removed by soft count = 447\n",
      "--- gene=858, id=251 removed by soft count = 146\n",
      "--- number of probes: 252 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 859\n",
      "--- gene=859, id=1 removed by soft count = 153\n",
      "--- gene=859, id=16 removed by soft count = 232\n",
      "--- gene=859, id=69 removed by soft count = 220\n",
      "--- gene=859, id=122 removed by soft count = 55\n",
      "--- gene=859, id=123 removed by soft count = 185\n",
      "--- gene=859, id=124 removed by soft count = 84\n",
      "--- gene=859, id=151 removed by soft count = 155\n",
      "--- gene=859, id=213 removed by soft count = 15840\n",
      "--- gene=859, id=225 removed by soft count = 209\n",
      "--- number of probes: 240 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 86\n",
      "--- gene=86, id=232 removed by soft count = 328\n",
      "--- gene=86, id=233 removed by soft count = 164\n",
      "--- gene=86, id=235 removed by soft count = 263\n",
      "--- gene=86, id=236 removed by soft count = 190\n",
      "--- number of probes: 237 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 860\n",
      "--- gene=860, id=1 removed by soft count = 898\n",
      "--- gene=860, id=2 removed by soft count = 145\n",
      "--- gene=860, id=23 removed by soft count = 140\n",
      "--- gene=860, id=81 removed by soft count = 57\n",
      "--- gene=860, id=82 removed by soft count = 973\n",
      "--- gene=860, id=109 removed by soft count = 284\n",
      "--- gene=860, id=112 removed by soft count = 90\n",
      "--- gene=860, id=114 removed by soft count = 528\n",
      "--- gene=860, id=115 removed by soft count = 267\n",
      "--- gene=860, id=117 removed by soft count = 412\n",
      "--- gene=860, id=120 removed by soft count = 72\n",
      "--- gene=860, id=121 removed by soft count = 90\n",
      "--- gene=860, id=124 removed by soft count = 310\n",
      "--- gene=860, id=125 removed by soft count = 69\n",
      "--- number of probes: 144 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 861\n",
      "--- gene=861, id=204 removed by soft count = 1109\n",
      "--- gene=861, id=205 removed by soft count = 111\n",
      "--- gene=861, id=303 removed by soft count = 78\n",
      "--- number of probes: 333 , kept by blast: 330 , if remove dups: 330\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 862\n",
      "--- gene=862, id=11 removed by soft count = 273\n",
      "--- gene=862, id=51 removed by soft count = 60\n",
      "--- gene=862, id=107 removed by soft count = 172\n",
      "--- gene=862, id=109 removed by soft count = 81\n",
      "--- gene=862, id=112 removed by soft count = 165\n",
      "--- number of probes: 233 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 863\n",
      "--- gene=863, id=14 removed by soft count = 51\n",
      "--- gene=863, id=125 removed by soft count = 923\n",
      "--- number of probes: 282 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 864\n",
      "--- gene=864, id=97 removed by soft count = 54\n",
      "--- gene=864, id=109 removed by soft count = 1147\n",
      "--- gene=864, id=110 removed by soft count = 273\n",
      "--- gene=864, id=112 removed by soft count = 86\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=864, id=159 removed by soft count = 227\n",
      "--- gene=864, id=160 removed by soft count = 2268\n",
      "--- number of probes: 250 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 865\n",
      "--- gene=865, id=0 removed by soft count = 57\n",
      "--- gene=865, id=2 removed by soft count = 199\n",
      "--- gene=865, id=119 removed by soft count = 58\n",
      "--- gene=865, id=120 removed by soft count = 88\n",
      "--- gene=865, id=123 removed by soft count = 129\n",
      "--- gene=865, id=125 removed by soft count = 70\n",
      "--- gene=865, id=126 removed by soft count = 128\n",
      "--- gene=865, id=144 removed by soft count = 107\n",
      "--- gene=865, id=145 removed by soft count = 102\n",
      "--- gene=865, id=146 removed by soft count = 105\n",
      "--- gene=865, id=197 removed by soft count = 344\n",
      "--- gene=865, id=199 removed by soft count = 81\n",
      "--- gene=865, id=200 removed by soft count = 90\n",
      "--- gene=865, id=202 removed by soft count = 126\n",
      "--- gene=865, id=204 removed by soft count = 72\n",
      "--- gene=865, id=209 removed by soft count = 5318\n",
      "--- gene=865, id=210 removed by soft count = 888\n",
      "--- gene=865, id=211 removed by soft count = 322\n",
      "--- gene=865, id=212 removed by soft count = 121\n",
      "--- gene=865, id=214 removed by soft count = 102\n",
      "--- number of probes: 223 , kept by blast: 203 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 866\n",
      "--- gene=866, id=156 removed by soft count = 55\n",
      "--- gene=866, id=190 removed by soft count = 56\n",
      "--- gene=866, id=192 removed by soft count = 98\n",
      "--- gene=866, id=195 removed by soft count = 65\n",
      "--- gene=866, id=198 removed by soft count = 72\n",
      "--- gene=866, id=199 removed by soft count = 151\n",
      "--- gene=866, id=201 removed by soft count = 528\n",
      "--- gene=866, id=202 removed by soft count = 83\n",
      "--- number of probes: 216 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 867\n",
      "--- gene=867, id=83 removed by soft count = 323\n",
      "--- gene=867, id=90 removed by soft count = 189\n",
      "--- gene=867, id=110 removed by soft count = 53\n",
      "--- gene=867, id=152 removed by soft count = 87\n",
      "--- gene=867, id=162 removed by soft count = 311\n",
      "--- gene=867, id=163 removed by soft count = 54\n",
      "--- number of probes: 164 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 868\n",
      "--- gene=868, id=65 removed by soft count = 325\n",
      "--- gene=868, id=66 removed by soft count = 749\n",
      "--- gene=868, id=97 removed by soft count = 51\n",
      "--- gene=868, id=100 removed by soft count = 17263\n",
      "--- gene=868, id=101 removed by soft count = 56\n",
      "--- gene=868, id=132 removed by soft count = 651\n",
      "--- gene=868, id=136 removed by soft count = 135\n",
      "--- gene=868, id=171 removed by soft count = 189\n",
      "--- gene=868, id=275 removed by soft count = 178\n",
      "--- gene=868, id=276 removed by soft count = 77\n",
      "--- gene=868, id=278 removed by soft count = 732\n",
      "--- number of probes: 289 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 869\n",
      "--- gene=869, id=1 removed by soft count = 115\n",
      "--- gene=869, id=2 removed by soft count = 67\n",
      "--- gene=869, id=37 removed by soft count = 663\n",
      "--- gene=869, id=40 removed by soft count = 818\n",
      "--- gene=869, id=41 removed by soft count = 647\n",
      "--- gene=869, id=45 removed by soft count = 268\n",
      "--- gene=869, id=54 removed by soft count = 348\n",
      "--- gene=869, id=73 removed by soft count = 56\n",
      "--- gene=869, id=75 removed by soft count = 541\n",
      "--- gene=869, id=77 removed by soft count = 1403\n",
      "--- gene=869, id=102 removed by soft count = 112\n",
      "--- gene=869, id=104 removed by soft count = 64\n",
      "--- gene=869, id=124 removed by soft count = 120\n",
      "--- gene=869, id=125 removed by soft count = 53\n",
      "--- gene=869, id=126 removed by soft count = 164\n",
      "--- gene=869, id=159 removed by soft count = 356\n",
      "--- gene=869, id=250 removed by soft count = 55\n",
      "--- gene=869, id=257 removed by soft count = 75\n",
      "--- gene=869, id=290 removed by soft count = 95\n",
      "--- number of probes: 301 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 87\n",
      "--- gene=87, id=65 removed by soft count = 430\n",
      "--- gene=87, id=77 removed by soft count = 330\n",
      "--- gene=87, id=79 removed by soft count = 61\n",
      "--- gene=87, id=82 removed by soft count = 74\n",
      "--- gene=87, id=87 removed by soft count = 116\n",
      "--- gene=87, id=88 removed by soft count = 110\n",
      "--- gene=87, id=91 removed by soft count = 163\n",
      "--- gene=87, id=92 removed by soft count = 329\n",
      "--- gene=87, id=93 removed by soft count = 109\n",
      "--- gene=87, id=94 removed by soft count = 167\n",
      "--- gene=87, id=96 removed by soft count = 191\n",
      "--- gene=87, id=103 removed by soft count = 62\n",
      "--- gene=87, id=104 removed by soft count = 68\n",
      "--- gene=87, id=107 removed by soft count = 99\n",
      "--- gene=87, id=108 removed by soft count = 78\n",
      "--- gene=87, id=111 removed by soft count = 72\n",
      "--- gene=87, id=116 removed by soft count = 96\n",
      "--- gene=87, id=117 removed by soft count = 53\n",
      "--- gene=87, id=118 removed by soft count = 81\n",
      "--- gene=87, id=119 removed by soft count = 59\n",
      "--- gene=87, id=129 removed by soft count = 86\n",
      "--- gene=87, id=130 removed by soft count = 95\n",
      "--- gene=87, id=131 removed by soft count = 53\n",
      "--- gene=87, id=140 removed by soft count = 105\n",
      "--- gene=87, id=143 removed by soft count = 77\n",
      "--- gene=87, id=144 removed by soft count = 150\n",
      "--- gene=87, id=150 removed by soft count = 579\n",
      "--- number of probes: 236 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 870\n",
      "--- gene=870, id=109 removed by soft count = 151\n",
      "--- gene=870, id=320 removed by soft count = 60\n",
      "--- number of probes: 324 , kept by blast: 322 , if remove dups: 321\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 871\n",
      "--- gene=871, id=0 removed by soft count = 89\n",
      "--- gene=871, id=296 removed by soft count = 78\n",
      "--- number of probes: 329 , kept by blast: 327 , if remove dups: 327\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 872\n",
      "--- gene=872, id=111 removed by soft count = 132\n",
      "--- gene=872, id=112 removed by soft count = 172\n",
      "--- gene=872, id=289 removed by hard count = 3\n",
      "--- number of probes: 359 , kept by blast: 356 , if remove dups: 356\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 873\n",
      "--- gene=873, id=74 removed by soft count = 107\n",
      "--- gene=873, id=76 removed by soft count = 573\n",
      "--- gene=873, id=82 removed by soft count = 447\n",
      "--- gene=873, id=85 removed by soft count = 155\n",
      "--- gene=873, id=86 removed by soft count = 250\n",
      "--- gene=873, id=144 removed by soft count = 51\n",
      "--- gene=873, id=204 removed by soft count = 66\n",
      "--- gene=873, id=217 removed by soft count = 803\n",
      "--- number of probes: 261 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 874\n",
      "--- gene=874, id=216 removed by soft count = 107\n",
      "--- gene=874, id=310 removed by soft count = 394\n",
      "--- gene=874, id=329 removed by soft count = 110\n",
      "--- gene=874, id=369 removed by soft count = 1471\n",
      "--- number of probes: 370 , kept by blast: 366 , if remove dups: 366\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 875\n",
      "--- gene=875, id=92 removed by soft count = 2099\n",
      "--- gene=875, id=93 removed by soft count = 588\n",
      "--- gene=875, id=98 removed by soft count = 103\n",
      "--- gene=875, id=127 removed by soft count = 61\n",
      "--- gene=875, id=156 removed by soft count = 121\n",
      "--- number of probes: 212 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 876\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=876, id=58 removed by soft count = 703\n",
      "--- gene=876, id=59 removed by soft count = 318\n",
      "--- gene=876, id=60 removed by soft count = 269\n",
      "--- gene=876, id=70 removed by soft count = 165\n",
      "--- gene=876, id=71 removed by soft count = 269\n",
      "--- gene=876, id=96 removed by soft count = 798\n",
      "--- gene=876, id=97 removed by soft count = 196\n",
      "--- gene=876, id=98 removed by soft count = 135\n",
      "--- gene=876, id=139 removed by soft count = 1698\n",
      "--- gene=876, id=225 removed by soft count = 221\n",
      "--- gene=876, id=247 removed by soft count = 72\n",
      "--- number of probes: 262 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 877\n",
      "--- gene=877, id=74 removed by soft count = 54\n",
      "--- gene=877, id=75 removed by soft count = 231\n",
      "--- gene=877, id=76 removed by soft count = 79\n",
      "--- gene=877, id=77 removed by soft count = 559\n",
      "--- gene=877, id=119 removed by soft count = 122\n",
      "--- gene=877, id=196 removed by soft count = 174\n",
      "--- gene=877, id=198 removed by soft count = 209\n",
      "--- gene=877, id=199 removed by soft count = 1266\n",
      "--- gene=877, id=203 removed by soft count = 1799\n",
      "--- number of probes: 259 , kept by blast: 250 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 878\n",
      "--- gene=878, id=124 removed by soft count = 63\n",
      "--- number of probes: 347 , kept by blast: 346 , if remove dups: 346\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 88\n",
      "--- number of probes: 204 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 888\n",
      "--- gene=888, id=9 removed by soft count = 72\n",
      "--- gene=888, id=11 removed by soft count = 65\n",
      "--- gene=888, id=114 removed by soft count = 3389\n",
      "--- gene=888, id=115 removed by soft count = 782\n",
      "--- gene=888, id=116 removed by soft count = 589\n",
      "--- gene=888, id=119 removed by soft count = 8226\n",
      "--- gene=888, id=120 removed by soft count = 13847\n",
      "--- gene=888, id=121 removed by soft count = 9290\n",
      "--- gene=888, id=122 removed by soft count = 5275\n",
      "--- gene=888, id=124 removed by soft count = 4195\n",
      "--- gene=888, id=125 removed by soft count = 1517\n",
      "--- gene=888, id=126 removed by soft count = 1614\n",
      "--- gene=888, id=127 removed by soft count = 1843\n",
      "--- gene=888, id=128 removed by soft count = 263\n",
      "--- gene=888, id=129 removed by soft count = 472\n",
      "--- gene=888, id=130 removed by soft count = 56\n",
      "--- gene=888, id=169 removed by soft count = 175\n",
      "--- gene=888, id=206 removed by soft count = 143\n",
      "--- gene=888, id=207 removed by soft count = 361\n",
      "--- number of probes: 215 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 889\n",
      "--- gene=889, id=252 removed by soft count = 94\n",
      "--- number of probes: 279 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 89\n",
      "--- gene=89, id=118 removed by soft count = 60\n",
      "--- gene=89, id=266 removed by soft count = 125\n",
      "--- number of probes: 335 , kept by blast: 333 , if remove dups: 333\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 890\n",
      "--- gene=890, id=42 removed by soft count = 287\n",
      "--- gene=890, id=143 removed by soft count = 66\n",
      "--- gene=890, id=148 removed by soft count = 126\n",
      "--- gene=890, id=149 removed by soft count = 260\n",
      "--- gene=890, id=150 removed by soft count = 1556\n",
      "--- gene=890, id=163 removed by soft count = 109\n",
      "--- gene=890, id=164 removed by soft count = 192\n",
      "--- gene=890, id=165 removed by soft count = 63\n",
      "--- gene=890, id=166 removed by soft count = 166\n",
      "--- gene=890, id=168 removed by soft count = 195\n",
      "--- gene=890, id=169 removed by soft count = 142\n",
      "--- gene=890, id=170 removed by soft count = 161\n",
      "--- gene=890, id=202 removed by soft count = 64\n",
      "--- number of probes: 204 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 891\n",
      "--- gene=891, id=10 removed by soft count = 98\n",
      "--- gene=891, id=47 removed by soft count = 78\n",
      "--- gene=891, id=78 removed by soft count = 81\n",
      "--- gene=891, id=86 removed by soft count = 163\n",
      "--- gene=891, id=90 removed by soft count = 73\n",
      "--- gene=891, id=91 removed by soft count = 74\n",
      "--- gene=891, id=92 removed by soft count = 151\n",
      "--- gene=891, id=94 removed by soft count = 94\n",
      "--- gene=891, id=95 removed by soft count = 83\n",
      "--- gene=891, id=96 removed by soft count = 174\n",
      "--- gene=891, id=143 removed by soft count = 126\n",
      "--- gene=891, id=144 removed by soft count = 212\n",
      "--- number of probes: 169 , kept by blast: 157 , if remove dups: 155\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 892\n",
      "--- gene=892, id=121 removed by soft count = 235\n",
      "--- number of probes: 314 , kept by blast: 313 , if remove dups: 313\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 893\n",
      "--- gene=893, id=8 removed by soft count = 65\n",
      "--- gene=893, id=69 removed by soft count = 575\n",
      "--- gene=893, id=90 removed by soft count = 261\n",
      "--- gene=893, id=91 removed by soft count = 2341\n",
      "--- gene=893, id=92 removed by soft count = 274\n",
      "--- number of probes: 252 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 894\n",
      "--- gene=894, id=31 removed by soft count = 55\n",
      "--- number of probes: 308 , kept by blast: 307 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 895\n",
      "--- gene=895, id=98 removed by soft count = 113\n",
      "--- gene=895, id=105 removed by soft count = 187\n",
      "--- gene=895, id=106 removed by soft count = 53\n",
      "--- gene=895, id=111 removed by soft count = 86\n",
      "--- gene=895, id=126 removed by soft count = 94\n",
      "--- gene=895, id=169 removed by soft count = 177\n",
      "--- gene=895, id=232 removed by soft count = 84\n",
      "--- gene=895, id=272 removed by soft count = 101\n",
      "--- gene=895, id=275 removed by soft count = 239\n",
      "--- number of probes: 294 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 896\n",
      "--- gene=896, id=0 removed by soft count = 5592\n",
      "--- gene=896, id=1 removed by soft count = 1088\n",
      "--- gene=896, id=2 removed by soft count = 779\n",
      "--- gene=896, id=5 removed by soft count = 77\n",
      "--- gene=896, id=71 removed by soft count = 131\n",
      "--- gene=896, id=137 removed by soft count = 118\n",
      "--- number of probes: 158 , kept by blast: 152 , if remove dups: 152\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 897\n",
      "--- gene=897, id=10 removed by soft count = 61\n",
      "--- number of probes: 286 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 898\n",
      "--- gene=898, id=2 removed by soft count = 6630\n",
      "--- gene=898, id=66 removed by soft count = 86\n",
      "--- gene=898, id=67 removed by soft count = 70\n",
      "--- gene=898, id=68 removed by soft count = 72\n",
      "--- gene=898, id=72 removed by soft count = 94\n",
      "--- gene=898, id=79 removed by soft count = 65\n",
      "--- gene=898, id=101 removed by soft count = 56\n",
      "--- gene=898, id=108 removed by soft count = 197\n",
      "--- gene=898, id=109 removed by soft count = 219\n",
      "--- number of probes: 182 , kept by blast: 173 , if remove dups: 173\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 899\n",
      "--- gene=899, id=24 removed by soft count = 200\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 211 , kept by blast: 210 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 9\n",
      "--- gene=9, id=159 removed by soft count = 104\n",
      "--- number of probes: 264 , kept by blast: 263 , if remove dups: 263\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 90\n",
      "--- gene=90, id=18 removed by soft count = 71\n",
      "--- gene=90, id=45 removed by soft count = 744\n",
      "--- gene=90, id=91 removed by soft count = 58\n",
      "--- gene=90, id=97 removed by soft count = 102\n",
      "--- gene=90, id=100 removed by soft count = 148\n",
      "--- gene=90, id=107 removed by soft count = 166\n",
      "--- gene=90, id=111 removed by soft count = 101\n",
      "--- gene=90, id=112 removed by soft count = 87\n",
      "--- number of probes: 118 , kept by blast: 110 , if remove dups: 110\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 110\n",
      "-- checking probes in region: 900\n",
      "--- gene=900, id=3 removed by soft count = 162\n",
      "--- gene=900, id=32 removed by soft count = 102\n",
      "--- gene=900, id=157 removed by soft count = 244\n",
      "--- gene=900, id=164 removed by soft count = 233\n",
      "--- gene=900, id=224 removed by soft count = 93\n",
      "--- gene=900, id=234 removed by soft count = 83\n",
      "--- gene=900, id=235 removed by soft count = 72\n",
      "--- gene=900, id=240 removed by soft count = 67\n",
      "--- number of probes: 251 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 901\n",
      "--- gene=901, id=4 removed by soft count = 95\n",
      "--- gene=901, id=5 removed by soft count = 71\n",
      "--- gene=901, id=6 removed by soft count = 237\n",
      "--- gene=901, id=8 removed by soft count = 600\n",
      "--- gene=901, id=19 removed by soft count = 492\n",
      "--- gene=901, id=64 removed by soft count = 446\n",
      "--- gene=901, id=83 removed by soft count = 229\n",
      "--- gene=901, id=127 removed by soft count = 128\n",
      "--- number of probes: 230 , kept by blast: 222 , if remove dups: 222\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 902\n",
      "--- gene=902, id=9 removed by soft count = 79\n",
      "--- gene=902, id=82 removed by soft count = 396\n",
      "--- gene=902, id=116 removed by soft count = 217\n",
      "--- gene=902, id=120 removed by soft count = 353\n",
      "--- gene=902, id=197 removed by soft count = 112\n",
      "--- gene=902, id=217 removed by soft count = 107\n",
      "--- gene=902, id=235 removed by soft count = 62\n",
      "--- gene=902, id=259 removed by soft count = 268\n",
      "--- gene=902, id=260 removed by soft count = 161\n",
      "--- number of probes: 271 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 903\n",
      "--- gene=903, id=32 removed by soft count = 56\n",
      "--- gene=903, id=52 removed by soft count = 76\n",
      "--- gene=903, id=63 removed by soft count = 74\n",
      "--- gene=903, id=74 removed by soft count = 1183\n",
      "--- gene=903, id=75 removed by soft count = 679\n",
      "--- gene=903, id=139 removed by soft count = 61\n",
      "--- number of probes: 144 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 904\n",
      "--- gene=904, id=65 removed by soft count = 153\n",
      "--- gene=904, id=80 removed by soft count = 66\n",
      "--- number of probes: 320 , kept by blast: 318 , if remove dups: 318\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 905\n",
      "--- gene=905, id=299 removed by soft count = 60\n",
      "--- number of probes: 311 , kept by blast: 310 , if remove dups: 310\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 906\n",
      "--- gene=906, id=20 removed by soft count = 51\n",
      "--- gene=906, id=23 removed by soft count = 171\n",
      "--- gene=906, id=67 removed by soft count = 98\n",
      "--- gene=906, id=68 removed by soft count = 298\n",
      "--- gene=906, id=86 removed by soft count = 197\n",
      "--- gene=906, id=88 removed by soft count = 304\n",
      "--- gene=906, id=121 removed by soft count = 54\n",
      "--- gene=906, id=124 removed by soft count = 236\n",
      "--- gene=906, id=129 removed by soft count = 86\n",
      "--- gene=906, id=144 removed by soft count = 156\n",
      "--- gene=906, id=145 removed by soft count = 840\n",
      "--- gene=906, id=158 removed by soft count = 60\n",
      "--- gene=906, id=164 removed by soft count = 6146\n",
      "--- gene=906, id=165 removed by soft count = 4590\n",
      "--- gene=906, id=166 removed by soft count = 603\n",
      "--- gene=906, id=183 removed by soft count = 71\n",
      "--- number of probes: 194 , kept by blast: 178 , if remove dups: 178\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 907\n",
      "--- gene=907, id=31 removed by soft count = 56\n",
      "--- gene=907, id=33 removed by soft count = 257\n",
      "--- gene=907, id=99 removed by soft count = 401\n",
      "--- number of probes: 143 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 908\n",
      "--- gene=908, id=56 removed by soft count = 176\n",
      "--- gene=908, id=113 removed by soft count = 60\n",
      "--- gene=908, id=118 removed by soft count = 70\n",
      "--- gene=908, id=124 removed by soft count = 8508\n",
      "--- gene=908, id=169 removed by soft count = 1171\n",
      "--- gene=908, id=178 removed by soft count = 2476\n",
      "--- number of probes: 216 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 909\n",
      "--- gene=909, id=69 removed by soft count = 325\n",
      "--- gene=909, id=78 removed by soft count = 161\n",
      "--- gene=909, id=80 removed by soft count = 280\n",
      "--- gene=909, id=83 removed by soft count = 63\n",
      "--- gene=909, id=85 removed by soft count = 78\n",
      "--- gene=909, id=86 removed by soft count = 176\n",
      "--- gene=909, id=90 removed by soft count = 53\n",
      "--- gene=909, id=92 removed by soft count = 131\n",
      "--- gene=909, id=93 removed by soft count = 95\n",
      "--- gene=909, id=110 removed by soft count = 73\n",
      "--- gene=909, id=112 removed by soft count = 226\n",
      "--- gene=909, id=113 removed by soft count = 119\n",
      "--- gene=909, id=120 removed by soft count = 153\n",
      "--- gene=909, id=121 removed by soft count = 199\n",
      "--- gene=909, id=122 removed by soft count = 468\n",
      "--- gene=909, id=123 removed by soft count = 525\n",
      "--- gene=909, id=129 removed by soft count = 72\n",
      "--- gene=909, id=130 removed by soft count = 61\n",
      "--- number of probes: 181 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 91\n",
      "--- gene=91, id=356 removed by soft count = 241\n",
      "--- number of probes: 363 , kept by blast: 362 , if remove dups: 362\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 910\n",
      "--- gene=910, id=49 removed by soft count = 189\n",
      "--- gene=910, id=52 removed by soft count = 806\n",
      "--- gene=910, id=53 removed by soft count = 723\n",
      "--- gene=910, id=57 removed by soft count = 73\n",
      "--- gene=910, id=60 removed by soft count = 75\n",
      "--- gene=910, id=62 removed by soft count = 89\n",
      "--- gene=910, id=63 removed by soft count = 113\n",
      "--- gene=910, id=65 removed by soft count = 76\n",
      "--- gene=910, id=70 removed by soft count = 82\n",
      "--- gene=910, id=79 removed by soft count = 79\n",
      "--- gene=910, id=93 removed by soft count = 113\n",
      "--- gene=910, id=94 removed by soft count = 79\n",
      "--- gene=910, id=98 removed by soft count = 263\n",
      "--- gene=910, id=100 removed by soft count = 54\n",
      "--- gene=910, id=104 removed by soft count = 113\n",
      "--- gene=910, id=105 removed by soft count = 64\n",
      "--- gene=910, id=106 removed by soft count = 246\n",
      "--- gene=910, id=108 removed by soft count = 87\n",
      "--- gene=910, id=109 removed by soft count = 114\n",
      "--- gene=910, id=110 removed by soft count = 272\n",
      "--- gene=910, id=111 removed by soft count = 326\n",
      "--- gene=910, id=112 removed by soft count = 482\n",
      "--- gene=910, id=113 removed by soft count = 133\n",
      "--- gene=910, id=114 removed by soft count = 586\n",
      "--- gene=910, id=115 removed by soft count = 213\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=910, id=116 removed by soft count = 612\n",
      "--- gene=910, id=117 removed by soft count = 294\n",
      "--- gene=910, id=118 removed by soft count = 116\n",
      "--- gene=910, id=119 removed by soft count = 265\n",
      "--- gene=910, id=120 removed by soft count = 1217\n",
      "--- gene=910, id=121 removed by soft count = 345\n",
      "--- gene=910, id=122 removed by soft count = 174\n",
      "--- gene=910, id=125 removed by soft count = 70\n",
      "--- gene=910, id=131 removed by soft count = 195\n",
      "--- gene=910, id=134 removed by soft count = 1473\n",
      "--- number of probes: 155 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 911\n",
      "--- gene=911, id=39 removed by soft count = 212\n",
      "--- gene=911, id=56 removed by soft count = 70\n",
      "--- number of probes: 187 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 912\n",
      "--- number of probes: 230 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 913\n",
      "--- gene=913, id=26 removed by soft count = 128\n",
      "--- gene=913, id=29 removed by soft count = 1741\n",
      "--- gene=913, id=72 removed by soft count = 151\n",
      "--- gene=913, id=87 removed by soft count = 1662\n",
      "--- gene=913, id=89 removed by soft count = 367\n",
      "--- gene=913, id=90 removed by soft count = 108\n",
      "--- gene=913, id=114 removed by soft count = 113\n",
      "--- number of probes: 169 , kept by blast: 162 , if remove dups: 161\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 914\n",
      "--- gene=914, id=0 removed by soft count = 1597\n",
      "--- gene=914, id=1 removed by soft count = 52\n",
      "--- gene=914, id=2 removed by soft count = 180\n",
      "--- gene=914, id=3 removed by soft count = 989\n",
      "--- gene=914, id=4 removed by soft count = 323\n",
      "--- gene=914, id=7 removed by soft count = 83\n",
      "--- gene=914, id=135 removed by soft count = 85\n",
      "--- number of probes: 176 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 915\n",
      "--- gene=915, id=136 removed by soft count = 148\n",
      "--- gene=915, id=158 removed by soft count = 74\n",
      "--- gene=915, id=160 removed by soft count = 66\n",
      "--- number of probes: 166 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 916\n",
      "--- gene=916, id=104 removed by soft count = 131\n",
      "--- gene=916, id=152 removed by soft count = 56\n",
      "--- gene=916, id=154 removed by soft count = 89\n",
      "--- gene=916, id=177 removed by soft count = 405\n",
      "--- gene=916, id=194 removed by soft count = 74\n",
      "--- gene=916, id=236 removed by soft count = 102\n",
      "--- number of probes: 250 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 917\n",
      "--- gene=917, id=57 removed by soft count = 283\n",
      "--- gene=917, id=62 removed by hard count = 3\n",
      "--- gene=917, id=135 removed by soft count = 119\n",
      "--- gene=917, id=136 removed by soft count = 212\n",
      "--- gene=917, id=163 removed by soft count = 1057\n",
      "--- gene=917, id=222 removed by soft count = 79\n",
      "--- gene=917, id=233 removed by soft count = 128\n",
      "--- gene=917, id=234 removed by soft count = 218\n",
      "--- gene=917, id=235 removed by soft count = 78\n",
      "--- number of probes: 250 , kept by blast: 241 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 918\n",
      "--- gene=918, id=70 removed by soft count = 217\n",
      "--- gene=918, id=71 removed by soft count = 1154\n",
      "--- gene=918, id=233 removed by soft count = 337\n",
      "--- gene=918, id=234 removed by soft count = 538\n",
      "--- gene=918, id=254 removed by soft count = 248\n",
      "--- gene=918, id=255 removed by soft count = 725\n",
      "--- gene=918, id=256 removed by soft count = 181\n",
      "--- gene=918, id=258 removed by soft count = 157\n",
      "--- number of probes: 259 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 919\n",
      "--- gene=919, id=31 removed by soft count = 96\n",
      "--- gene=919, id=34 removed by soft count = 54\n",
      "--- gene=919, id=37 removed by soft count = 90\n",
      "--- gene=919, id=39 removed by soft count = 1457\n",
      "--- gene=919, id=45 removed by soft count = 141\n",
      "--- gene=919, id=46 removed by soft count = 1581\n",
      "--- gene=919, id=68 removed by soft count = 272\n",
      "--- gene=919, id=69 removed by soft count = 105\n",
      "--- gene=919, id=93 removed by soft count = 79\n",
      "--- gene=919, id=94 removed by soft count = 1485\n",
      "--- gene=919, id=109 removed by soft count = 327\n",
      "--- gene=919, id=119 removed by soft count = 728\n",
      "--- gene=919, id=120 removed by soft count = 3860\n",
      "--- gene=919, id=122 removed by soft count = 596\n",
      "--- gene=919, id=123 removed by soft count = 81\n",
      "--- gene=919, id=124 removed by soft count = 189\n",
      "--- gene=919, id=125 removed by soft count = 311\n",
      "--- gene=919, id=126 removed by soft count = 103\n",
      "--- gene=919, id=127 removed by soft count = 69\n",
      "--- gene=919, id=128 removed by soft count = 69\n",
      "--- gene=919, id=131 removed by soft count = 476\n",
      "--- gene=919, id=133 removed by soft count = 75\n",
      "--- gene=919, id=135 removed by soft count = 109\n",
      "--- gene=919, id=144 removed by soft count = 67\n",
      "--- gene=919, id=146 removed by soft count = 115\n",
      "--- gene=919, id=147 removed by soft count = 179\n",
      "--- gene=919, id=149 removed by soft count = 565\n",
      "--- gene=919, id=150 removed by soft count = 149\n",
      "--- gene=919, id=151 removed by soft count = 79\n",
      "--- gene=919, id=152 removed by soft count = 58\n",
      "--- gene=919, id=153 removed by soft count = 500\n",
      "--- number of probes: 157 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 92\n",
      "--- gene=92, id=2 removed by soft count = 557\n",
      "--- gene=92, id=134 removed by soft count = 85\n",
      "--- gene=92, id=263 removed by soft count = 191\n",
      "--- number of probes: 271 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 920\n",
      "--- gene=920, id=93 removed by soft count = 2707\n",
      "--- gene=920, id=116 removed by soft count = 118\n",
      "--- number of probes: 187 , kept by blast: 185 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 921\n",
      "--- gene=921, id=34 removed by soft count = 165\n",
      "--- gene=921, id=146 removed by soft count = 62\n",
      "--- gene=921, id=165 removed by soft count = 121\n",
      "--- number of probes: 173 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 922\n",
      "--- gene=922, id=45 removed by soft count = 134\n",
      "--- gene=922, id=46 removed by soft count = 228\n",
      "--- gene=922, id=49 removed by soft count = 64\n",
      "--- gene=922, id=50 removed by soft count = 5440\n",
      "--- gene=922, id=51 removed by soft count = 1565\n",
      "--- gene=922, id=52 removed by soft count = 9894\n",
      "--- gene=922, id=53 removed by soft count = 14157\n",
      "--- gene=922, id=54 removed by soft count = 163\n",
      "--- gene=922, id=55 removed by soft count = 321\n",
      "--- gene=922, id=57 removed by soft count = 314\n",
      "--- gene=922, id=70 removed by soft count = 221\n",
      "--- gene=922, id=72 removed by soft count = 65\n",
      "--- gene=922, id=74 removed by soft count = 165\n",
      "--- gene=922, id=139 removed by soft count = 193\n",
      "--- number of probes: 174 , kept by blast: 160 , if remove dups: 160\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 923\n",
      "--- gene=923, id=32 removed by soft count = 866\n",
      "--- gene=923, id=33 removed by soft count = 896\n",
      "--- gene=923, id=34 removed by soft count = 1591\n",
      "--- gene=923, id=35 removed by soft count = 331\n",
      "--- gene=923, id=36 removed by soft count = 189\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=923, id=80 removed by soft count = 452\n",
      "--- gene=923, id=81 removed by soft count = 1499\n",
      "--- gene=923, id=82 removed by soft count = 2697\n",
      "--- gene=923, id=131 removed by soft count = 101\n",
      "--- number of probes: 141 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 924\n",
      "--- gene=924, id=99 removed by soft count = 123\n",
      "--- gene=924, id=100 removed by soft count = 140\n",
      "--- gene=924, id=115 removed by soft count = 110\n",
      "--- gene=924, id=179 removed by soft count = 82\n",
      "--- number of probes: 236 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 925\n",
      "--- gene=925, id=10 removed by soft count = 164\n",
      "--- gene=925, id=111 removed by soft count = 348\n",
      "--- gene=925, id=113 removed by soft count = 74\n",
      "--- gene=925, id=146 removed by soft count = 309\n",
      "--- gene=925, id=159 removed by soft count = 1266\n",
      "--- gene=925, id=161 removed by soft count = 252\n",
      "--- gene=925, id=162 removed by soft count = 473\n",
      "--- number of probes: 214 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 926\n",
      "--- gene=926, id=62 removed by soft count = 128\n",
      "--- gene=926, id=116 removed by soft count = 490\n",
      "--- gene=926, id=129 removed by soft count = 80\n",
      "--- gene=926, id=133 removed by soft count = 63\n",
      "--- gene=926, id=241 removed by soft count = 51\n",
      "--- number of probes: 293 , kept by blast: 288 , if remove dups: 288\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 927\n",
      "--- gene=927, id=58 removed by soft count = 54\n",
      "--- gene=927, id=75 removed by soft count = 257\n",
      "--- gene=927, id=76 removed by soft count = 1126\n",
      "--- gene=927, id=121 removed by soft count = 146\n",
      "--- gene=927, id=122 removed by soft count = 85\n",
      "--- gene=927, id=141 removed by soft count = 75\n",
      "--- gene=927, id=145 removed by soft count = 849\n",
      "--- number of probes: 314 , kept by blast: 307 , if remove dups: 307\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 928\n",
      "--- gene=928, id=267 removed by soft count = 134\n",
      "--- number of probes: 334 , kept by blast: 333 , if remove dups: 333\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 929\n",
      "--- gene=929, id=6 removed by soft count = 60\n",
      "--- gene=929, id=12 removed by soft count = 143\n",
      "--- gene=929, id=60 removed by soft count = 179\n",
      "--- gene=929, id=62 removed by soft count = 166\n",
      "--- gene=929, id=143 removed by soft count = 418\n",
      "--- gene=929, id=184 removed by soft count = 1233\n",
      "--- number of probes: 261 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 93\n",
      "--- gene=93, id=42 removed by soft count = 57\n",
      "--- gene=93, id=132 removed by soft count = 166\n",
      "--- number of probes: 212 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 930\n",
      "--- gene=930, id=13 removed by soft count = 1348\n",
      "--- gene=930, id=14 removed by soft count = 1079\n",
      "--- gene=930, id=15 removed by soft count = 287\n",
      "--- gene=930, id=16 removed by soft count = 83\n",
      "--- gene=930, id=46 removed by soft count = 51\n",
      "--- gene=930, id=50 removed by soft count = 186\n",
      "--- gene=930, id=51 removed by soft count = 501\n",
      "--- gene=930, id=53 removed by soft count = 257\n",
      "--- gene=930, id=88 removed by soft count = 279\n",
      "--- gene=930, id=91 removed by soft count = 53\n",
      "--- number of probes: 266 , kept by blast: 256 , if remove dups: 256\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 931\n",
      "--- gene=931, id=33 removed by soft count = 101\n",
      "--- gene=931, id=196 removed by soft count = 184\n",
      "--- gene=931, id=198 removed by soft count = 195\n",
      "--- number of probes: 249 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 932\n",
      "--- gene=932, id=46 removed by soft count = 86\n",
      "--- gene=932, id=286 removed by soft count = 75\n",
      "--- number of probes: 297 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 933\n",
      "--- gene=933, id=214 removed by soft count = 150\n",
      "--- gene=933, id=215 removed by soft count = 134\n",
      "--- number of probes: 218 , kept by blast: 216 , if remove dups: 216\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 934\n",
      "--- gene=934, id=0 removed by soft count = 51\n",
      "--- gene=934, id=4 removed by soft count = 139\n",
      "--- gene=934, id=8 removed by soft count = 62\n",
      "--- gene=934, id=63 removed by soft count = 74\n",
      "--- gene=934, id=67 removed by soft count = 104\n",
      "--- gene=934, id=68 removed by soft count = 69\n",
      "--- gene=934, id=69 removed by soft count = 167\n",
      "--- gene=934, id=71 removed by soft count = 191\n",
      "--- gene=934, id=100 removed by soft count = 266\n",
      "--- gene=934, id=102 removed by soft count = 191\n",
      "--- gene=934, id=141 removed by soft count = 104\n",
      "--- gene=934, id=148 removed by soft count = 201\n",
      "--- gene=934, id=149 removed by soft count = 87\n",
      "--- gene=934, id=150 removed by soft count = 103\n",
      "--- gene=934, id=151 removed by soft count = 80\n",
      "--- gene=934, id=152 removed by soft count = 110\n",
      "--- gene=934, id=154 removed by soft count = 96\n",
      "--- gene=934, id=158 removed by soft count = 151\n",
      "--- gene=934, id=160 removed by soft count = 83\n",
      "--- gene=934, id=161 removed by soft count = 68\n",
      "--- number of probes: 209 , kept by blast: 189 , if remove dups: 189\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 935\n",
      "--- gene=935, id=59 removed by hard count = 6\n",
      "--- gene=935, id=61 removed by soft count = 201\n",
      "--- gene=935, id=62 removed by soft count = 79\n",
      "--- gene=935, id=88 removed by soft count = 202\n",
      "--- gene=935, id=89 removed by soft count = 212\n",
      "--- gene=935, id=222 removed by soft count = 70\n",
      "--- gene=935, id=252 removed by soft count = 99\n",
      "--- gene=935, id=257 removed by soft count = 118\n",
      "--- gene=935, id=341 removed by hard count = 3\n",
      "--- number of probes: 369 , kept by blast: 360 , if remove dups: 359\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 94\n",
      "--- gene=94, id=163 removed by soft count = 107\n",
      "--- gene=94, id=164 removed by soft count = 159\n",
      "--- number of probes: 178 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 949\n",
      "--- gene=949, id=28 removed by soft count = 364\n",
      "--- gene=949, id=63 removed by soft count = 58\n",
      "--- gene=949, id=85 removed by soft count = 55\n",
      "--- gene=949, id=95 removed by soft count = 847\n",
      "--- gene=949, id=102 removed by soft count = 109\n",
      "--- gene=949, id=106 removed by soft count = 88\n",
      "--- gene=949, id=138 removed by soft count = 89\n",
      "--- number of probes: 160 , kept by blast: 153 , if remove dups: 153\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 95\n",
      "--- gene=95, id=27 removed by soft count = 138\n",
      "--- gene=95, id=40 removed by soft count = 59\n",
      "--- gene=95, id=48 removed by soft count = 72\n",
      "--- gene=95, id=50 removed by soft count = 64\n",
      "--- gene=95, id=52 removed by soft count = 63\n",
      "--- gene=95, id=54 removed by soft count = 74\n",
      "--- gene=95, id=74 removed by soft count = 54\n",
      "--- gene=95, id=101 removed by soft count = 54\n",
      "--- gene=95, id=119 removed by soft count = 138\n",
      "--- gene=95, id=200 removed by soft count = 534\n",
      "--- gene=95, id=201 removed by soft count = 791\n",
      "--- gene=95, id=202 removed by soft count = 90\n",
      "--- gene=95, id=204 removed by soft count = 53\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=95, id=206 removed by soft count = 107\n",
      "--- gene=95, id=207 removed by soft count = 402\n",
      "--- gene=95, id=208 removed by soft count = 76\n",
      "--- gene=95, id=210 removed by soft count = 65\n",
      "--- gene=95, id=212 removed by soft count = 545\n",
      "--- number of probes: 214 , kept by blast: 196 , if remove dups: 195\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 950\n",
      "--- gene=950, id=55 removed by soft count = 787\n",
      "--- gene=950, id=96 removed by soft count = 98\n",
      "--- gene=950, id=165 removed by soft count = 398\n",
      "--- gene=950, id=166 removed by soft count = 353\n",
      "--- gene=950, id=167 removed by soft count = 1332\n",
      "--- gene=950, id=168 removed by soft count = 2273\n",
      "--- gene=950, id=174 removed by soft count = 157\n",
      "--- gene=950, id=177 removed by soft count = 285\n",
      "--- number of probes: 178 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 951\n",
      "--- gene=951, id=0 removed by soft count = 68\n",
      "--- gene=951, id=79 removed by soft count = 79\n",
      "--- gene=951, id=80 removed by soft count = 153\n",
      "--- gene=951, id=81 removed by soft count = 129\n",
      "--- gene=951, id=121 removed by soft count = 82\n",
      "--- gene=951, id=163 removed by soft count = 304\n",
      "--- gene=951, id=164 removed by soft count = 127\n",
      "--- gene=951, id=179 removed by soft count = 55\n",
      "--- gene=951, id=180 removed by soft count = 54\n",
      "--- number of probes: 188 , kept by blast: 179 , if remove dups: 179\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 952\n",
      "--- gene=952, id=17 removed by soft count = 54\n",
      "--- gene=952, id=20 removed by soft count = 262\n",
      "--- gene=952, id=21 removed by soft count = 135\n",
      "--- gene=952, id=52 removed by soft count = 635\n",
      "--- gene=952, id=99 removed by soft count = 1135\n",
      "--- number of probes: 190 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 953\n",
      "--- gene=953, id=89 removed by soft count = 89\n",
      "--- gene=953, id=90 removed by soft count = 279\n",
      "--- gene=953, id=91 removed by soft count = 4446\n",
      "--- gene=953, id=92 removed by soft count = 2596\n",
      "--- gene=953, id=199 removed by soft count = 189\n",
      "--- gene=953, id=200 removed by soft count = 174\n",
      "--- gene=953, id=201 removed by soft count = 398\n",
      "--- gene=953, id=202 removed by soft count = 251\n",
      "--- number of probes: 245 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 954\n",
      "--- gene=954, id=25 removed by soft count = 64\n",
      "--- gene=954, id=26 removed by soft count = 58\n",
      "--- gene=954, id=27 removed by soft count = 363\n",
      "--- gene=954, id=31 removed by soft count = 208\n",
      "--- gene=954, id=52 removed by soft count = 133\n",
      "--- gene=954, id=80 removed by soft count = 96\n",
      "--- gene=954, id=85 removed by soft count = 139\n",
      "--- gene=954, id=86 removed by soft count = 124\n",
      "--- gene=954, id=87 removed by soft count = 160\n",
      "--- number of probes: 90 , kept by blast: 81 , if remove dups: 80\n",
      "-- number of probes kept for this region: 81\n",
      "-- checking probes in region: 955\n",
      "--- gene=955, id=94 removed by soft count = 75\n",
      "--- gene=955, id=102 removed by soft count = 53\n",
      "--- gene=955, id=105 removed by soft count = 123\n",
      "--- gene=955, id=128 removed by soft count = 157\n",
      "--- gene=955, id=129 removed by soft count = 1042\n",
      "--- gene=955, id=130 removed by soft count = 350\n",
      "--- gene=955, id=131 removed by soft count = 5262\n",
      "--- gene=955, id=141 removed by soft count = 81\n",
      "--- gene=955, id=170 removed by soft count = 162\n",
      "--- number of probes: 174 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 956\n",
      "--- gene=956, id=29 removed by soft count = 65\n",
      "--- gene=956, id=52 removed by soft count = 108\n",
      "--- gene=956, id=98 removed by soft count = 107\n",
      "--- gene=956, id=232 removed by soft count = 135\n",
      "--- number of probes: 265 , kept by blast: 261 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 957\n",
      "--- gene=957, id=0 removed by soft count = 110\n",
      "--- gene=957, id=1 removed by soft count = 447\n",
      "--- gene=957, id=2 removed by soft count = 250\n",
      "--- gene=957, id=8 removed by soft count = 80\n",
      "--- gene=957, id=113 removed by soft count = 171\n",
      "--- gene=957, id=257 removed by soft count = 63\n",
      "--- number of probes: 258 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 958\n",
      "--- gene=958, id=68 removed by soft count = 254\n",
      "--- gene=958, id=93 removed by soft count = 139\n",
      "--- gene=958, id=94 removed by soft count = 82\n",
      "--- gene=958, id=96 removed by soft count = 159\n",
      "--- gene=958, id=118 removed by soft count = 527\n",
      "--- gene=958, id=213 removed by soft count = 227\n",
      "--- gene=958, id=215 removed by soft count = 188\n",
      "--- gene=958, id=271 removed by soft count = 90\n",
      "--- number of probes: 328 , kept by blast: 320 , if remove dups: 320\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 959\n",
      "--- gene=959, id=3 removed by soft count = 54\n",
      "--- gene=959, id=173 removed by soft count = 53\n",
      "--- gene=959, id=174 removed by soft count = 272\n",
      "--- gene=959, id=187 removed by soft count = 62\n",
      "--- gene=959, id=193 removed by soft count = 72\n",
      "--- gene=959, id=196 removed by soft count = 76\n",
      "--- number of probes: 213 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 96\n",
      "--- gene=96, id=71 removed by soft count = 348\n",
      "--- gene=96, id=82 removed by soft count = 132\n",
      "--- gene=96, id=105 removed by soft count = 143\n",
      "--- gene=96, id=163 removed by soft count = 268\n",
      "--- gene=96, id=185 removed by soft count = 282\n",
      "--- number of probes: 258 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 960\n",
      "--- gene=960, id=94 removed by soft count = 205\n",
      "--- number of probes: 279 , kept by blast: 278 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 961\n",
      "--- gene=961, id=5 removed by soft count = 206\n",
      "--- gene=961, id=111 removed by soft count = 69\n",
      "--- gene=961, id=112 removed by soft count = 113\n",
      "--- gene=961, id=137 removed by soft count = 83\n",
      "--- gene=961, id=138 removed by soft count = 58\n",
      "--- gene=961, id=152 removed by soft count = 127\n",
      "--- gene=961, id=229 removed by soft count = 125\n",
      "--- gene=961, id=231 removed by soft count = 290\n",
      "--- gene=961, id=233 removed by soft count = 343\n",
      "--- gene=961, id=236 removed by soft count = 156\n",
      "--- number of probes: 237 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 962\n",
      "--- gene=962, id=0 removed by soft count = 258\n",
      "--- gene=962, id=14 removed by soft count = 152\n",
      "--- gene=962, id=27 removed by soft count = 52\n",
      "--- gene=962, id=31 removed by soft count = 66\n",
      "--- gene=962, id=35 removed by soft count = 491\n",
      "--- gene=962, id=37 removed by soft count = 180\n",
      "--- gene=962, id=39 removed by soft count = 123\n",
      "--- gene=962, id=171 removed by soft count = 189\n",
      "--- gene=962, id=172 removed by soft count = 54\n",
      "--- number of probes: 226 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 963\n",
      "--- gene=963, id=0 removed by soft count = 335\n",
      "--- gene=963, id=1 removed by soft count = 146\n",
      "--- gene=963, id=2 removed by soft count = 201\n",
      "--- gene=963, id=3 removed by soft count = 473\n",
      "--- gene=963, id=6 removed by soft count = 128\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=963, id=7 removed by soft count = 368\n",
      "--- gene=963, id=8 removed by soft count = 419\n",
      "--- gene=963, id=9 removed by soft count = 94\n",
      "--- gene=963, id=10 removed by soft count = 435\n",
      "--- gene=963, id=11 removed by soft count = 370\n",
      "--- gene=963, id=12 removed by soft count = 199\n",
      "--- gene=963, id=13 removed by soft count = 187\n",
      "--- gene=963, id=14 removed by soft count = 350\n",
      "--- gene=963, id=15 removed by soft count = 63\n",
      "--- gene=963, id=16 removed by soft count = 559\n",
      "--- gene=963, id=17 removed by soft count = 381\n",
      "--- gene=963, id=18 removed by soft count = 163\n",
      "--- gene=963, id=19 removed by soft count = 506\n",
      "--- gene=963, id=20 removed by soft count = 57\n",
      "--- gene=963, id=21 removed by soft count = 853\n",
      "--- gene=963, id=40 removed by soft count = 176\n",
      "--- gene=963, id=41 removed by soft count = 69\n",
      "--- gene=963, id=42 removed by soft count = 105\n",
      "--- gene=963, id=43 removed by soft count = 91\n",
      "--- gene=963, id=45 removed by soft count = 98\n",
      "--- gene=963, id=66 removed by soft count = 87\n",
      "--- gene=963, id=101 removed by soft count = 69\n",
      "--- gene=963, id=104 removed by soft count = 339\n",
      "--- gene=963, id=107 removed by soft count = 63\n",
      "--- gene=963, id=175 removed by soft count = 1046\n",
      "--- gene=963, id=178 removed by soft count = 83\n",
      "--- number of probes: 195 , kept by blast: 164 , if remove dups: 164\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 964\n",
      "--- gene=964, id=30 removed by soft count = 457\n",
      "--- gene=964, id=33 removed by soft count = 163\n",
      "--- gene=964, id=45 removed by soft count = 681\n",
      "--- gene=964, id=49 removed by soft count = 52\n",
      "--- gene=964, id=51 removed by soft count = 82\n",
      "--- gene=964, id=53 removed by soft count = 104\n",
      "--- gene=964, id=54 removed by soft count = 115\n",
      "--- gene=964, id=58 removed by soft count = 658\n",
      "--- gene=964, id=59 removed by soft count = 245\n",
      "--- gene=964, id=104 removed by soft count = 2581\n",
      "--- gene=964, id=108 removed by soft count = 69\n",
      "--- gene=964, id=137 removed by soft count = 68\n",
      "--- gene=964, id=139 removed by soft count = 217\n",
      "--- gene=964, id=140 removed by soft count = 152\n",
      "--- gene=964, id=141 removed by soft count = 781\n",
      "--- gene=964, id=145 removed by soft count = 113\n",
      "--- number of probes: 149 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 965\n",
      "--- gene=965, id=30 removed by soft count = 151\n",
      "--- gene=965, id=38 removed by soft count = 78\n",
      "--- gene=965, id=40 removed by soft count = 51\n",
      "--- gene=965, id=45 removed by soft count = 101\n",
      "--- gene=965, id=47 removed by soft count = 72\n",
      "--- gene=965, id=48 removed by soft count = 162\n",
      "--- gene=965, id=49 removed by soft count = 78\n",
      "--- gene=965, id=114 removed by soft count = 82\n",
      "--- gene=965, id=116 removed by soft count = 128\n",
      "--- gene=965, id=128 removed by soft count = 128\n",
      "--- gene=965, id=153 removed by soft count = 57\n",
      "--- gene=965, id=156 removed by soft count = 74\n",
      "--- gene=965, id=159 removed by soft count = 99\n",
      "--- number of probes: 191 , kept by blast: 178 , if remove dups: 178\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 966\n",
      "--- gene=966, id=107 removed by soft count = 3315\n",
      "--- gene=966, id=108 removed by soft count = 6698\n",
      "--- gene=966, id=110 removed by soft count = 276\n",
      "--- number of probes: 161 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 967\n",
      "--- gene=967, id=66 removed by soft count = 1248\n",
      "--- gene=967, id=100 removed by soft count = 438\n",
      "--- gene=967, id=119 removed by soft count = 125\n",
      "--- gene=967, id=135 removed by soft count = 742\n",
      "--- gene=967, id=136 removed by soft count = 1539\n",
      "--- gene=967, id=138 removed by soft count = 1168\n",
      "--- gene=967, id=139 removed by soft count = 2877\n",
      "--- gene=967, id=140 removed by soft count = 3602\n",
      "--- gene=967, id=141 removed by soft count = 163\n",
      "--- gene=967, id=142 removed by soft count = 51\n",
      "--- number of probes: 229 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 968\n",
      "--- gene=968, id=0 removed by soft count = 83\n",
      "--- gene=968, id=5 removed by soft count = 2873\n",
      "--- gene=968, id=102 removed by soft count = 57\n",
      "--- gene=968, id=137 removed by soft count = 174\n",
      "--- gene=968, id=139 removed by soft count = 724\n",
      "--- gene=968, id=208 removed by soft count = 107\n",
      "--- gene=968, id=209 removed by soft count = 705\n",
      "--- gene=968, id=212 removed by soft count = 141\n",
      "--- number of probes: 228 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 969\n",
      "--- gene=969, id=4 removed by soft count = 126\n",
      "--- gene=969, id=36 removed by soft count = 668\n",
      "--- gene=969, id=67 removed by soft count = 538\n",
      "--- gene=969, id=84 removed by soft count = 60\n",
      "--- gene=969, id=90 removed by soft count = 347\n",
      "--- gene=969, id=93 removed by soft count = 54\n",
      "--- gene=969, id=95 removed by soft count = 53\n",
      "--- gene=969, id=98 removed by soft count = 100\n",
      "--- gene=969, id=99 removed by soft count = 73\n",
      "--- gene=969, id=100 removed by soft count = 112\n",
      "--- gene=969, id=130 removed by soft count = 62\n",
      "--- gene=969, id=133 removed by soft count = 370\n",
      "--- number of probes: 155 , kept by blast: 143 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 970\n",
      "--- gene=970, id=50 removed by soft count = 89\n",
      "--- gene=970, id=51 removed by soft count = 132\n",
      "--- gene=970, id=53 removed by soft count = 59\n",
      "--- gene=970, id=54 removed by soft count = 83\n",
      "--- gene=970, id=56 removed by soft count = 165\n",
      "--- gene=970, id=57 removed by soft count = 61\n",
      "--- gene=970, id=58 removed by soft count = 116\n",
      "--- gene=970, id=59 removed by soft count = 82\n",
      "--- gene=970, id=60 removed by soft count = 187\n",
      "--- gene=970, id=63 removed by soft count = 81\n",
      "--- gene=970, id=70 removed by soft count = 149\n",
      "--- gene=970, id=71 removed by soft count = 80\n",
      "--- gene=970, id=72 removed by soft count = 73\n",
      "--- gene=970, id=75 removed by soft count = 70\n",
      "--- gene=970, id=76 removed by soft count = 88\n",
      "--- gene=970, id=77 removed by soft count = 56\n",
      "--- gene=970, id=78 removed by soft count = 57\n",
      "--- gene=970, id=79 removed by soft count = 88\n",
      "--- gene=970, id=80 removed by soft count = 93\n",
      "--- gene=970, id=81 removed by soft count = 65\n",
      "--- gene=970, id=82 removed by soft count = 98\n",
      "--- gene=970, id=84 removed by soft count = 57\n",
      "--- gene=970, id=86 removed by soft count = 56\n",
      "--- gene=970, id=88 removed by soft count = 252\n",
      "--- gene=970, id=91 removed by soft count = 150\n",
      "--- gene=970, id=96 removed by soft count = 59\n",
      "--- gene=970, id=97 removed by soft count = 131\n",
      "--- gene=970, id=100 removed by soft count = 183\n",
      "--- gene=970, id=103 removed by soft count = 116\n",
      "--- gene=970, id=106 removed by soft count = 53\n",
      "--- number of probes: 110 , kept by blast: 80 , if remove dups: 80\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 971\n",
      "--- gene=971, id=80 removed by soft count = 160\n",
      "--- gene=971, id=81 removed by soft count = 301\n",
      "--- gene=971, id=87 removed by soft count = 107\n",
      "--- gene=971, id=155 removed by soft count = 217\n",
      "--- gene=971, id=156 removed by soft count = 546\n",
      "--- gene=971, id=157 removed by soft count = 99\n",
      "--- gene=971, id=159 removed by soft count = 105\n",
      "--- gene=971, id=161 removed by soft count = 87\n",
      "--- gene=971, id=164 removed by soft count = 87\n",
      "--- gene=971, id=166 removed by soft count = 80\n",
      "--- gene=971, id=167 removed by soft count = 1198\n",
      "--- number of probes: 173 , kept by blast: 162 , if remove dups: 162\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 972\n",
      "--- number of probes: 392 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 973\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=973, id=83 removed by soft count = 1321\n",
      "--- gene=973, id=173 removed by soft count = 993\n",
      "--- gene=973, id=197 removed by soft count = 53\n",
      "--- gene=973, id=222 removed by soft count = 110\n",
      "--- gene=973, id=261 removed by soft count = 55\n",
      "--- number of probes: 270 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 974\n",
      "--- gene=974, id=162 removed by soft count = 74\n",
      "--- gene=974, id=187 removed by soft count = 351\n",
      "--- number of probes: 208 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 975\n",
      "--- number of probes: 314 , kept by blast: 314 , if remove dups: 314\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 976\n",
      "--- gene=976, id=21 removed by soft count = 92\n",
      "--- gene=976, id=69 removed by soft count = 144\n",
      "--- gene=976, id=70 removed by soft count = 210\n",
      "--- gene=976, id=71 removed by soft count = 281\n",
      "--- gene=976, id=73 removed by soft count = 286\n",
      "--- gene=976, id=74 removed by soft count = 267\n",
      "--- gene=976, id=99 removed by soft count = 95\n",
      "--- number of probes: 194 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 977\n",
      "--- gene=977, id=96 removed by soft count = 76\n",
      "--- gene=977, id=97 removed by soft count = 295\n",
      "--- gene=977, id=264 removed by soft count = 61\n",
      "--- gene=977, id=268 removed by soft count = 52\n",
      "--- gene=977, id=269 removed by soft count = 102\n",
      "--- number of probes: 297 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 978\n",
      "--- gene=978, id=39 removed by soft count = 126\n",
      "--- gene=978, id=55 removed by soft count = 4622\n",
      "--- gene=978, id=63 removed by soft count = 79\n",
      "--- gene=978, id=64 removed by soft count = 122\n",
      "--- gene=978, id=78 removed by soft count = 70\n",
      "--- gene=978, id=79 removed by soft count = 131\n",
      "--- gene=978, id=84 removed by soft count = 366\n",
      "--- gene=978, id=109 removed by soft count = 145\n",
      "--- gene=978, id=122 removed by soft count = 6323\n",
      "--- number of probes: 130 , kept by blast: 121 , if remove dups: 121\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 979\n",
      "--- gene=979, id=112 removed by soft count = 57\n",
      "--- gene=979, id=115 removed by soft count = 57\n",
      "--- gene=979, id=118 removed by soft count = 95\n",
      "--- gene=979, id=119 removed by soft count = 58\n",
      "--- gene=979, id=120 removed by soft count = 129\n",
      "--- gene=979, id=123 removed by soft count = 452\n",
      "--- gene=979, id=127 removed by soft count = 322\n",
      "--- gene=979, id=129 removed by soft count = 432\n",
      "--- gene=979, id=130 removed by soft count = 256\n",
      "--- gene=979, id=131 removed by soft count = 320\n",
      "--- gene=979, id=139 removed by soft count = 128\n",
      "--- gene=979, id=142 removed by soft count = 89\n",
      "--- gene=979, id=143 removed by soft count = 58\n",
      "--- gene=979, id=144 removed by soft count = 312\n",
      "--- gene=979, id=147 removed by soft count = 98\n",
      "--- gene=979, id=148 removed by soft count = 85\n",
      "--- gene=979, id=149 removed by soft count = 92\n",
      "--- gene=979, id=204 removed by soft count = 72\n",
      "--- gene=979, id=206 removed by soft count = 58\n",
      "--- number of probes: 255 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 98\n",
      "--- gene=98, id=27 removed by soft count = 69\n",
      "--- gene=98, id=107 removed by soft count = 120\n",
      "--- gene=98, id=114 removed by soft count = 146\n",
      "--- gene=98, id=115 removed by soft count = 155\n",
      "--- gene=98, id=116 removed by soft count = 115\n",
      "--- gene=98, id=118 removed by soft count = 99\n",
      "--- gene=98, id=121 removed by soft count = 107\n",
      "--- gene=98, id=135 removed by soft count = 126\n",
      "--- number of probes: 166 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 980\n",
      "--- gene=980, id=38 removed by soft count = 70\n",
      "--- gene=980, id=73 removed by soft count = 95\n",
      "--- gene=980, id=91 removed by soft count = 122\n",
      "--- gene=980, id=93 removed by soft count = 181\n",
      "--- gene=980, id=142 removed by soft count = 233\n",
      "--- gene=980, id=143 removed by soft count = 54\n",
      "--- gene=980, id=144 removed by soft count = 101\n",
      "--- gene=980, id=146 removed by soft count = 162\n",
      "--- gene=980, id=147 removed by soft count = 125\n",
      "--- gene=980, id=148 removed by soft count = 434\n",
      "--- gene=980, id=166 removed by soft count = 540\n",
      "--- gene=980, id=167 removed by soft count = 1017\n",
      "--- number of probes: 173 , kept by blast: 161 , if remove dups: 161\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 981\n",
      "--- gene=981, id=5 removed by soft count = 77\n",
      "--- gene=981, id=248 removed by soft count = 61\n",
      "--- gene=981, id=278 removed by soft count = 114\n",
      "--- number of probes: 302 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 982\n",
      "--- gene=982, id=0 removed by soft count = 474\n",
      "--- gene=982, id=36 removed by soft count = 151\n",
      "--- gene=982, id=40 removed by soft count = 167\n",
      "--- gene=982, id=42 removed by soft count = 55\n",
      "--- gene=982, id=43 removed by soft count = 90\n",
      "--- gene=982, id=46 removed by soft count = 172\n",
      "--- gene=982, id=47 removed by soft count = 186\n",
      "--- gene=982, id=131 removed by soft count = 810\n",
      "--- number of probes: 171 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 983\n",
      "--- gene=983, id=157 removed by soft count = 842\n",
      "--- gene=983, id=159 removed by soft count = 157\n",
      "--- gene=983, id=161 removed by soft count = 58\n",
      "--- gene=983, id=162 removed by soft count = 81\n",
      "--- number of probes: 236 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 984\n",
      "--- gene=984, id=157 removed by soft count = 603\n",
      "--- gene=984, id=168 removed by soft count = 65\n",
      "--- gene=984, id=246 removed by soft count = 55\n",
      "--- gene=984, id=257 removed by soft count = 163\n",
      "--- gene=984, id=258 removed by soft count = 85\n",
      "--- gene=984, id=260 removed by soft count = 74\n",
      "--- gene=984, id=267 removed by soft count = 68\n",
      "--- number of probes: 269 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 985\n",
      "--- gene=985, id=254 removed by hard count = 0\n",
      "--- number of probes: 342 , kept by blast: 341 , if remove dups: 341\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 986\n",
      "--- gene=986, id=133 removed by soft count = 54\n",
      "--- gene=986, id=135 removed by soft count = 254\n",
      "--- number of probes: 233 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 987\n",
      "--- gene=987, id=9 removed by hard count = 0\n",
      "--- gene=987, id=144 removed by soft count = 71\n",
      "--- number of probes: 365 , kept by blast: 363 , if remove dups: 362\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 988\n",
      "--- gene=988, id=258 removed by soft count = 106\n",
      "--- number of probes: 348 , kept by blast: 347 , if remove dups: 347\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 989\n",
      "--- gene=989, id=6 removed by soft count = 1378\n",
      "--- gene=989, id=7 removed by soft count = 1230\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=989, id=8 removed by soft count = 208\n",
      "--- gene=989, id=9 removed by soft count = 183\n",
      "--- gene=989, id=29 removed by soft count = 65\n",
      "--- gene=989, id=30 removed by soft count = 119\n",
      "--- gene=989, id=51 removed by soft count = 115\n",
      "--- gene=989, id=78 removed by soft count = 121\n",
      "--- gene=989, id=79 removed by soft count = 625\n",
      "--- gene=989, id=80 removed by soft count = 921\n",
      "--- gene=989, id=82 removed by soft count = 718\n",
      "--- number of probes: 154 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 990\n",
      "--- gene=990, id=27 removed by soft count = 65\n",
      "--- gene=990, id=31 removed by soft count = 118\n",
      "--- gene=990, id=74 removed by soft count = 72\n",
      "--- gene=990, id=223 removed by soft count = 107\n",
      "--- gene=990, id=250 removed by soft count = 131\n",
      "--- gene=990, id=251 removed by soft count = 173\n",
      "--- gene=990, id=252 removed by soft count = 54\n",
      "--- number of probes: 281 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 991\n",
      "--- gene=991, id=24 removed by soft count = 119\n",
      "--- gene=991, id=91 removed by soft count = 347\n",
      "--- gene=991, id=94 removed by soft count = 104\n",
      "--- gene=991, id=96 removed by soft count = 362\n",
      "--- gene=991, id=183 removed by soft count = 95\n",
      "--- gene=991, id=211 removed by soft count = 245\n",
      "--- gene=991, id=214 removed by soft count = 78\n",
      "--- gene=991, id=221 removed by soft count = 56\n",
      "--- gene=991, id=222 removed by soft count = 189\n",
      "--- number of probes: 247 , kept by blast: 238 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 992\n",
      "--- gene=992, id=20 removed by soft count = 62\n",
      "--- number of probes: 237 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 993\n",
      "--- gene=993, id=1 removed by soft count = 204\n",
      "--- gene=993, id=35 removed by soft count = 156\n",
      "--- gene=993, id=142 removed by soft count = 97\n",
      "--- number of probes: 153 , kept by blast: 150 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 994\n",
      "--- gene=994, id=37 removed by soft count = 128\n",
      "--- gene=994, id=61 removed by soft count = 88\n",
      "--- gene=994, id=131 removed by soft count = 70\n",
      "--- gene=994, id=182 removed by soft count = 52\n",
      "--- gene=994, id=268 removed by soft count = 80\n",
      "--- gene=994, id=269 removed by soft count = 78\n",
      "--- gene=994, id=271 removed by soft count = 70\n",
      "--- gene=994, id=272 removed by soft count = 91\n",
      "--- gene=994, id=276 removed by soft count = 66\n",
      "--- number of probes: 286 , kept by blast: 277 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 995\n",
      "--- gene=995, id=26 removed by soft count = 177\n",
      "--- number of probes: 337 , kept by blast: 336 , if remove dups: 336\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 996\n",
      "--- gene=996, id=83 removed by soft count = 57\n",
      "--- gene=996, id=95 removed by soft count = 74\n",
      "--- gene=996, id=103 removed by soft count = 53\n",
      "--- gene=996, id=159 removed by soft count = 69\n",
      "--- gene=996, id=252 removed by soft count = 236\n",
      "--- gene=996, id=253 removed by soft count = 176\n",
      "--- number of probes: 255 , kept by blast: 249 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 997\n",
      "--- gene=997, id=115 removed by soft count = 71\n",
      "--- gene=997, id=116 removed by soft count = 90\n",
      "--- gene=997, id=117 removed by soft count = 78\n",
      "--- gene=997, id=223 removed by soft count = 107\n",
      "--- gene=997, id=292 removed by soft count = 72\n",
      "--- number of probes: 304 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 999\n",
      "--- gene=999, id=67 removed by soft count = 57\n",
      "--- gene=999, id=107 removed by soft count = 187\n",
      "--- gene=999, id=114 removed by soft count = 57\n",
      "--- gene=999, id=244 removed by soft count = 66\n",
      "--- gene=999, id=297 removed by soft count = 171\n",
      "--- number of probes: 341 , kept by blast: 336 , if remove dups: 336\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "- summarize\n",
      "-- region: 0 , number of probes: 120\n",
      "-- region: 1 , number of probes: 120\n",
      "-- region: 10 , number of probes: 120\n",
      "-- region: 100 , number of probes: 120\n",
      "-- region: 1000 , number of probes: 120\n",
      "-- region: 1001 , number of probes: 120\n",
      "-- region: 1002 , number of probes: 120\n",
      "-- region: 1003 , number of probes: 120\n",
      "-- region: 1004 , number of probes: 120\n",
      "-- region: 1005 , number of probes: 120\n",
      "-- region: 101 , number of probes: 120\n",
      "-- region: 102 , number of probes: 120\n",
      "-- region: 1020 , number of probes: 49\n",
      "-- region: 1023 , number of probes: 120\n",
      "-- region: 1024 , number of probes: 120\n",
      "-- region: 1025 , number of probes: 120\n",
      "-- region: 1029 , number of probes: 120\n",
      "-- region: 103 , number of probes: 120\n",
      "-- region: 1030 , number of probes: 120\n",
      "-- region: 1031 , number of probes: 120\n",
      "-- region: 1032 , number of probes: 120\n",
      "-- region: 1033 , number of probes: 120\n",
      "-- region: 1034 , number of probes: 120\n",
      "-- region: 1035 , number of probes: 120\n",
      "-- region: 1036 , number of probes: 120\n",
      "-- region: 1037 , number of probes: 120\n",
      "-- region: 1038 , number of probes: 120\n",
      "-- region: 1039 , number of probes: 120\n",
      "-- region: 104 , number of probes: 96\n",
      "-- region: 1040 , number of probes: 120\n",
      "-- region: 1041 , number of probes: 120\n",
      "-- region: 1042 , number of probes: 120\n",
      "-- region: 1043 , number of probes: 105\n",
      "-- region: 1044 , number of probes: 120\n",
      "-- region: 1045 , number of probes: 120\n",
      "-- region: 1046 , number of probes: 120\n",
      "-- region: 1047 , number of probes: 120\n",
      "-- region: 1048 , number of probes: 120\n",
      "-- region: 1049 , number of probes: 120\n",
      "-- region: 105 , number of probes: 120\n",
      "-- region: 1050 , number of probes: 120\n",
      "-- region: 1051 , number of probes: 120\n",
      "-- region: 1052 , number of probes: 120\n",
      "-- region: 1053 , number of probes: 120\n",
      "-- region: 1054 , number of probes: 120\n",
      "-- region: 1055 , number of probes: 120\n",
      "-- region: 1056 , number of probes: 120\n",
      "-- region: 1057 , number of probes: 120\n",
      "-- region: 1058 , number of probes: 120\n",
      "-- region: 1059 , number of probes: 120\n",
      "-- region: 106 , number of probes: 120\n",
      "-- region: 1060 , number of probes: 120\n",
      "-- region: 1061 , number of probes: 120\n",
      "-- region: 1062 , number of probes: 120\n",
      "-- region: 1063 , number of probes: 120\n",
      "-- region: 1064 , number of probes: 120\n",
      "-- region: 1065 , number of probes: 120\n",
      "-- region: 1066 , number of probes: 120\n",
      "-- region: 1067 , number of probes: 120\n",
      "-- region: 1068 , number of probes: 120\n",
      "-- region: 1069 , number of probes: 120\n",
      "-- region: 107 , number of probes: 120\n",
      "-- region: 1070 , number of probes: 120\n",
      "-- region: 1071 , number of probes: 120\n",
      "-- region: 1072 , number of probes: 120\n",
      "-- region: 1073 , number of probes: 120\n",
      "-- region: 1074 , number of probes: 120\n",
      "-- region: 1075 , number of probes: 120\n",
      "-- region: 1076 , number of probes: 120\n",
      "-- region: 1077 , number of probes: 86\n",
      "-- region: 1078 , number of probes: 120\n",
      "-- region: 1079 , number of probes: 120\n",
      "-- region: 108 , number of probes: 120\n",
      "-- region: 1080 , number of probes: 120\n",
      "-- region: 1081 , number of probes: 120\n",
      "-- region: 1082 , number of probes: 120\n",
      "-- region: 1083 , number of probes: 120\n",
      "-- region: 1085 , number of probes: 120\n",
      "-- region: 1087 , number of probes: 120\n",
      "-- region: 1089 , number of probes: 120\n",
      "-- region: 109 , number of probes: 120\n",
      "-- region: 1090 , number of probes: 120\n",
      "-- region: 1091 , number of probes: 120\n",
      "-- region: 1092 , number of probes: 120\n",
      "-- region: 1093 , number of probes: 120\n",
      "-- region: 1094 , number of probes: 120\n",
      "-- region: 1097 , number of probes: 120\n",
      "-- region: 11 , number of probes: 120\n",
      "-- region: 110 , number of probes: 120\n",
      "-- region: 1105 , number of probes: 120\n",
      "-- region: 1106 , number of probes: 120\n",
      "-- region: 1107 , number of probes: 120\n",
      "-- region: 1108 , number of probes: 120\n",
      "-- region: 1109 , number of probes: 120\n",
      "-- region: 111 , number of probes: 120\n",
      "-- region: 1110 , number of probes: 120\n",
      "-- region: 1111 , number of probes: 120\n",
      "-- region: 1112 , number of probes: 120\n",
      "-- region: 1113 , number of probes: 120\n",
      "-- region: 1114 , number of probes: 120\n",
      "-- region: 1115 , number of probes: 120\n",
      "-- region: 1116 , number of probes: 120\n",
      "-- region: 1117 , number of probes: 120\n",
      "-- region: 1118 , number of probes: 120\n",
      "-- region: 1119 , number of probes: 120\n",
      "-- region: 112 , number of probes: 120\n",
      "-- region: 1120 , number of probes: 120\n",
      "-- region: 1121 , number of probes: 120\n",
      "-- region: 1122 , number of probes: 120\n",
      "-- region: 1123 , number of probes: 120\n",
      "-- region: 1124 , number of probes: 120\n",
      "-- region: 1125 , number of probes: 120\n",
      "-- region: 1126 , number of probes: 120\n",
      "-- region: 1127 , number of probes: 120\n",
      "-- region: 1128 , number of probes: 120\n",
      "-- region: 1129 , number of probes: 120\n",
      "-- region: 1130 , number of probes: 120\n",
      "-- region: 1131 , number of probes: 120\n",
      "-- region: 1132 , number of probes: 120\n",
      "-- region: 1133 , number of probes: 120\n",
      "-- region: 1134 , number of probes: 120\n",
      "-- region: 1135 , number of probes: 120\n",
      "-- region: 1136 , number of probes: 120\n",
      "-- region: 1137 , number of probes: 120\n",
      "-- region: 1138 , number of probes: 120\n",
      "-- region: 1139 , number of probes: 120\n",
      "-- region: 114 , number of probes: 120\n",
      "-- region: 1140 , number of probes: 120\n",
      "-- region: 1141 , number of probes: 120\n",
      "-- region: 1142 , number of probes: 120\n",
      "-- region: 1143 , number of probes: 120\n",
      "-- region: 1144 , number of probes: 117\n",
      "-- region: 1145 , number of probes: 120\n",
      "-- region: 1146 , number of probes: 92\n",
      "-- region: 1147 , number of probes: 120\n",
      "-- region: 1148 , number of probes: 98\n",
      "-- region: 115 , number of probes: 120\n",
      "-- region: 1152 , number of probes: 120\n",
      "-- region: 1153 , number of probes: 120\n",
      "-- region: 1154 , number of probes: 52\n",
      "-- region: 1155 , number of probes: 120\n",
      "-- region: 1156 , number of probes: 120\n",
      "-- region: 1157 , number of probes: 120\n",
      "-- region: 116 , number of probes: 120\n",
      "-- region: 1160 , number of probes: 120\n",
      "-- region: 1162 , number of probes: 120\n",
      "-- region: 1163 , number of probes: 120\n",
      "-- region: 1165 , number of probes: 120\n",
      "-- region: 1166 , number of probes: 120\n",
      "-- region: 1167 , number of probes: 120\n",
      "-- region: 1168 , number of probes: 120\n",
      "-- region: 1169 , number of probes: 120\n",
      "-- region: 117 , number of probes: 120\n",
      "-- region: 1170 , number of probes: 120\n",
      "-- region: 1171 , number of probes: 120\n",
      "-- region: 1172 , number of probes: 120\n",
      "-- region: 1173 , number of probes: 90\n",
      "-- region: 1174 , number of probes: 115\n",
      "-- region: 1176 , number of probes: 120\n",
      "-- region: 1177 , number of probes: 120\n",
      "-- region: 1178 , number of probes: 120\n",
      "-- region: 1179 , number of probes: 120\n",
      "-- region: 118 , number of probes: 120\n",
      "-- region: 1180 , number of probes: 120\n",
      "-- region: 1181 , number of probes: 120\n",
      "-- region: 1182 , number of probes: 120\n",
      "-- region: 1183 , number of probes: 120\n",
      "-- region: 1184 , number of probes: 120\n",
      "-- region: 1186 , number of probes: 120\n",
      "-- region: 1187 , number of probes: 120\n",
      "-- region: 1188 , number of probes: 77\n",
      "-- region: 1189 , number of probes: 120\n",
      "-- region: 119 , number of probes: 120\n",
      "-- region: 1190 , number of probes: 120\n",
      "-- region: 1191 , number of probes: 120\n",
      "-- region: 1192 , number of probes: 120\n",
      "-- region: 1193 , number of probes: 120\n",
      "-- region: 1194 , number of probes: 120\n",
      "-- region: 1195 , number of probes: 120\n",
      "-- region: 1196 , number of probes: 120\n",
      "-- region: 1197 , number of probes: 120\n",
      "-- region: 12 , number of probes: 120\n",
      "-- region: 120 , number of probes: 120\n",
      "-- region: 1203 , number of probes: 120\n",
      "-- region: 1204 , number of probes: 120\n",
      "-- region: 1205 , number of probes: 120\n",
      "-- region: 1206 , number of probes: 86\n",
      "-- region: 1207 , number of probes: 120\n",
      "-- region: 1208 , number of probes: 120\n",
      "-- region: 1209 , number of probes: 120\n",
      "-- region: 121 , number of probes: 120\n",
      "-- region: 1210 , number of probes: 120\n",
      "-- region: 1211 , number of probes: 120\n",
      "-- region: 1212 , number of probes: 120\n",
      "-- region: 1213 , number of probes: 120\n",
      "-- region: 1214 , number of probes: 120\n",
      "-- region: 1215 , number of probes: 120\n",
      "-- region: 1216 , number of probes: 120\n",
      "-- region: 1217 , number of probes: 120\n",
      "-- region: 1218 , number of probes: 120\n",
      "-- region: 1219 , number of probes: 120\n",
      "-- region: 122 , number of probes: 120\n",
      "-- region: 1220 , number of probes: 120\n",
      "-- region: 1221 , number of probes: 120\n",
      "-- region: 1222 , number of probes: 120\n",
      "-- region: 1224 , number of probes: 120\n",
      "-- region: 1225 , number of probes: 120\n",
      "-- region: 1226 , number of probes: 120\n",
      "-- region: 1227 , number of probes: 120\n",
      "-- region: 1228 , number of probes: 120\n",
      "-- region: 1229 , number of probes: 120\n",
      "-- region: 123 , number of probes: 120\n",
      "-- region: 1230 , number of probes: 120\n",
      "-- region: 1231 , number of probes: 120\n",
      "-- region: 1232 , number of probes: 120\n",
      "-- region: 1233 , number of probes: 120\n",
      "-- region: 1234 , number of probes: 120\n",
      "-- region: 1235 , number of probes: 120\n",
      "-- region: 1236 , number of probes: 120\n",
      "-- region: 1237 , number of probes: 120\n",
      "-- region: 1238 , number of probes: 120\n",
      "-- region: 1239 , number of probes: 120\n",
      "-- region: 124 , number of probes: 120\n",
      "-- region: 1240 , number of probes: 120\n",
      "-- region: 1241 , number of probes: 120\n",
      "-- region: 1242 , number of probes: 120\n",
      "-- region: 1243 , number of probes: 120\n",
      "-- region: 1244 , number of probes: 120\n",
      "-- region: 1245 , number of probes: 120\n",
      "-- region: 1246 , number of probes: 120\n",
      "-- region: 1247 , number of probes: 120\n",
      "-- region: 1248 , number of probes: 120\n",
      "-- region: 1249 , number of probes: 120\n",
      "-- region: 125 , number of probes: 120\n",
      "-- region: 1250 , number of probes: 120\n",
      "-- region: 1251 , number of probes: 120\n",
      "-- region: 1252 , number of probes: 120\n",
      "-- region: 1253 , number of probes: 120\n",
      "-- region: 1254 , number of probes: 120\n",
      "-- region: 1255 , number of probes: 120\n",
      "-- region: 1256 , number of probes: 120\n",
      "-- region: 1257 , number of probes: 120\n",
      "-- region: 1258 , number of probes: 120\n",
      "-- region: 1259 , number of probes: 120\n",
      "-- region: 126 , number of probes: 120\n",
      "-- region: 1260 , number of probes: 120\n",
      "-- region: 1261 , number of probes: 120\n",
      "-- region: 1262 , number of probes: 63\n",
      "-- region: 1263 , number of probes: 120\n",
      "-- region: 1264 , number of probes: 120\n",
      "-- region: 1265 , number of probes: 81\n",
      "-- region: 1269 , number of probes: 21\n",
      "-- region: 127 , number of probes: 120\n",
      "-- region: 1270 , number of probes: 120\n",
      "-- region: 1271 , number of probes: 120\n",
      "-- region: 1272 , number of probes: 120\n",
      "-- region: 1273 , number of probes: 120\n",
      "-- region: 1274 , number of probes: 120\n",
      "-- region: 1275 , number of probes: 120\n",
      "-- region: 1276 , number of probes: 120\n",
      "-- region: 1277 , number of probes: 120\n",
      "-- region: 1278 , number of probes: 120\n",
      "-- region: 1279 , number of probes: 120\n",
      "-- region: 128 , number of probes: 120\n",
      "-- region: 1280 , number of probes: 120\n",
      "-- region: 1281 , number of probes: 120\n",
      "-- region: 1282 , number of probes: 120\n",
      "-- region: 1284 , number of probes: 120\n",
      "-- region: 1285 , number of probes: 120\n",
      "-- region: 1286 , number of probes: 120\n",
      "-- region: 1287 , number of probes: 120\n",
      "-- region: 1288 , number of probes: 120\n",
      "-- region: 1289 , number of probes: 120\n",
      "-- region: 129 , number of probes: 120\n",
      "-- region: 1290 , number of probes: 120\n",
      "-- region: 1291 , number of probes: 120\n",
      "-- region: 1292 , number of probes: 120\n",
      "-- region: 1293 , number of probes: 120\n",
      "-- region: 1294 , number of probes: 78\n",
      "-- region: 1295 , number of probes: 120\n",
      "-- region: 1297 , number of probes: 120\n",
      "-- region: 1298 , number of probes: 120\n",
      "-- region: 1299 , number of probes: 120\n",
      "-- region: 13 , number of probes: 120\n",
      "-- region: 130 , number of probes: 120\n",
      "-- region: 1300 , number of probes: 120\n",
      "-- region: 1301 , number of probes: 120\n",
      "-- region: 1302 , number of probes: 120\n",
      "-- region: 1303 , number of probes: 120\n",
      "-- region: 1304 , number of probes: 120\n",
      "-- region: 1305 , number of probes: 120\n",
      "-- region: 1306 , number of probes: 120\n",
      "-- region: 1307 , number of probes: 120\n",
      "-- region: 1308 , number of probes: 120\n",
      "-- region: 1309 , number of probes: 120\n",
      "-- region: 131 , number of probes: 120\n",
      "-- region: 1310 , number of probes: 120\n",
      "-- region: 1311 , number of probes: 120\n",
      "-- region: 1312 , number of probes: 120\n",
      "-- region: 1313 , number of probes: 120\n",
      "-- region: 1314 , number of probes: 120\n",
      "-- region: 1315 , number of probes: 120\n",
      "-- region: 1316 , number of probes: 120\n",
      "-- region: 1317 , number of probes: 120\n",
      "-- region: 1318 , number of probes: 120\n",
      "-- region: 1319 , number of probes: 120\n",
      "-- region: 132 , number of probes: 120\n",
      "-- region: 1320 , number of probes: 120\n",
      "-- region: 1321 , number of probes: 120\n",
      "-- region: 1322 , number of probes: 120\n",
      "-- region: 1323 , number of probes: 120\n",
      "-- region: 1324 , number of probes: 120\n",
      "-- region: 1325 , number of probes: 120\n",
      "-- region: 133 , number of probes: 120\n",
      "-- region: 1330 , number of probes: 120\n",
      "-- region: 1331 , number of probes: 120\n",
      "-- region: 1332 , number of probes: 120\n",
      "-- region: 1333 , number of probes: 120\n",
      "-- region: 1334 , number of probes: 120\n",
      "-- region: 1335 , number of probes: 120\n",
      "-- region: 1336 , number of probes: 120\n",
      "-- region: 1337 , number of probes: 120\n",
      "-- region: 1338 , number of probes: 120\n",
      "-- region: 1339 , number of probes: 120\n",
      "-- region: 134 , number of probes: 120\n",
      "-- region: 1340 , number of probes: 120\n",
      "-- region: 1341 , number of probes: 120\n",
      "-- region: 1342 , number of probes: 120\n",
      "-- region: 1343 , number of probes: 120\n",
      "-- region: 1344 , number of probes: 120\n",
      "-- region: 1345 , number of probes: 120\n",
      "-- region: 1346 , number of probes: 120\n",
      "-- region: 1347 , number of probes: 120\n",
      "-- region: 1348 , number of probes: 88\n",
      "-- region: 1349 , number of probes: 120\n",
      "-- region: 1350 , number of probes: 120\n",
      "-- region: 1351 , number of probes: 120\n",
      "-- region: 1352 , number of probes: 120\n",
      "-- region: 1353 , number of probes: 120\n",
      "-- region: 1354 , number of probes: 120\n",
      "-- region: 1355 , number of probes: 120\n",
      "-- region: 1356 , number of probes: 120\n",
      "-- region: 1357 , number of probes: 120\n",
      "-- region: 1358 , number of probes: 120\n",
      "-- region: 1359 , number of probes: 120\n",
      "-- region: 1360 , number of probes: 120\n",
      "-- region: 1361 , number of probes: 120\n",
      "-- region: 1362 , number of probes: 120\n",
      "-- region: 1363 , number of probes: 120\n",
      "-- region: 137 , number of probes: 120\n",
      "-- region: 1378 , number of probes: 120\n",
      "-- region: 1379 , number of probes: 120\n",
      "-- region: 138 , number of probes: 120\n",
      "-- region: 1380 , number of probes: 120\n",
      "-- region: 1381 , number of probes: 120\n",
      "-- region: 1382 , number of probes: 120\n",
      "-- region: 1383 , number of probes: 120\n",
      "-- region: 1384 , number of probes: 120\n",
      "-- region: 1385 , number of probes: 120\n",
      "-- region: 1386 , number of probes: 117\n",
      "-- region: 1387 , number of probes: 120\n",
      "-- region: 1388 , number of probes: 120\n",
      "-- region: 1389 , number of probes: 120\n",
      "-- region: 139 , number of probes: 120\n",
      "-- region: 1390 , number of probes: 120\n",
      "-- region: 1391 , number of probes: 120\n",
      "-- region: 1392 , number of probes: 120\n",
      "-- region: 1393 , number of probes: 120\n",
      "-- region: 1394 , number of probes: 120\n",
      "-- region: 1395 , number of probes: 120\n",
      "-- region: 1396 , number of probes: 120\n",
      "-- region: 1398 , number of probes: 120\n",
      "-- region: 1399 , number of probes: 120\n",
      "-- region: 14 , number of probes: 120\n",
      "-- region: 140 , number of probes: 120\n",
      "-- region: 1400 , number of probes: 120\n",
      "-- region: 1401 , number of probes: 120\n",
      "-- region: 1402 , number of probes: 120\n",
      "-- region: 1403 , number of probes: 120\n",
      "-- region: 1404 , number of probes: 120\n",
      "-- region: 1405 , number of probes: 120\n",
      "-- region: 1406 , number of probes: 120\n",
      "-- region: 1408 , number of probes: 120\n",
      "-- region: 1409 , number of probes: 120\n",
      "-- region: 141 , number of probes: 68\n",
      "-- region: 1427 , number of probes: 120\n",
      "-- region: 1429 , number of probes: 120\n",
      "-- region: 143 , number of probes: 120\n",
      "-- region: 1430 , number of probes: 120\n",
      "-- region: 1434 , number of probes: 120\n",
      "-- region: 1435 , number of probes: 120\n",
      "-- region: 1436 , number of probes: 120\n",
      "-- region: 1437 , number of probes: 120\n",
      "-- region: 1438 , number of probes: 120\n",
      "-- region: 1439 , number of probes: 120\n",
      "-- region: 144 , number of probes: 120\n",
      "-- region: 1440 , number of probes: 120\n",
      "-- region: 1441 , number of probes: 120\n",
      "-- region: 1442 , number of probes: 120\n",
      "-- region: 1443 , number of probes: 120\n",
      "-- region: 1444 , number of probes: 120\n",
      "-- region: 1445 , number of probes: 120\n",
      "-- region: 1446 , number of probes: 120\n",
      "-- region: 1447 , number of probes: 120\n",
      "-- region: 1448 , number of probes: 120\n",
      "-- region: 1449 , number of probes: 120\n",
      "-- region: 145 , number of probes: 120\n",
      "-- region: 1450 , number of probes: 120\n",
      "-- region: 1451 , number of probes: 120\n",
      "-- region: 1452 , number of probes: 109\n",
      "-- region: 1453 , number of probes: 120\n",
      "-- region: 1454 , number of probes: 120\n",
      "-- region: 1455 , number of probes: 120\n",
      "-- region: 1456 , number of probes: 120\n",
      "-- region: 1457 , number of probes: 120\n",
      "-- region: 1458 , number of probes: 120\n",
      "-- region: 1459 , number of probes: 120\n",
      "-- region: 146 , number of probes: 120\n",
      "-- region: 1461 , number of probes: 120\n",
      "-- region: 1462 , number of probes: 120\n",
      "-- region: 1463 , number of probes: 120\n",
      "-- region: 1464 , number of probes: 120\n",
      "-- region: 1465 , number of probes: 120\n",
      "-- region: 1466 , number of probes: 120\n",
      "-- region: 1467 , number of probes: 120\n",
      "-- region: 1468 , number of probes: 120\n",
      "-- region: 1469 , number of probes: 120\n",
      "-- region: 147 , number of probes: 120\n",
      "-- region: 1470 , number of probes: 120\n",
      "-- region: 1471 , number of probes: 120\n",
      "-- region: 1472 , number of probes: 120\n",
      "-- region: 1473 , number of probes: 100\n",
      "-- region: 1474 , number of probes: 120\n",
      "-- region: 1475 , number of probes: 120\n",
      "-- region: 1476 , number of probes: 120\n",
      "-- region: 1477 , number of probes: 120\n",
      "-- region: 1478 , number of probes: 120\n",
      "-- region: 1479 , number of probes: 120\n",
      "-- region: 148 , number of probes: 120\n",
      "-- region: 1480 , number of probes: 104\n",
      "-- region: 1481 , number of probes: 120\n",
      "-- region: 1482 , number of probes: 120\n",
      "-- region: 1485 , number of probes: 120\n",
      "-- region: 1486 , number of probes: 120\n",
      "-- region: 1487 , number of probes: 120\n",
      "-- region: 1488 , number of probes: 120\n",
      "-- region: 1489 , number of probes: 73\n",
      "-- region: 149 , number of probes: 120\n",
      "-- region: 1490 , number of probes: 113\n",
      "-- region: 1491 , number of probes: 120\n",
      "-- region: 1492 , number of probes: 120\n",
      "-- region: 1493 , number of probes: 114\n",
      "-- region: 1494 , number of probes: 120\n",
      "-- region: 1495 , number of probes: 120\n",
      "-- region: 1496 , number of probes: 63\n",
      "-- region: 1497 , number of probes: 120\n",
      "-- region: 1498 , number of probes: 120\n",
      "-- region: 1499 , number of probes: 120\n",
      "-- region: 15 , number of probes: 120\n",
      "-- region: 150 , number of probes: 120\n",
      "-- region: 1500 , number of probes: 120\n",
      "-- region: 1501 , number of probes: 120\n",
      "-- region: 1502 , number of probes: 110\n",
      "-- region: 1503 , number of probes: 120\n",
      "-- region: 1504 , number of probes: 120\n",
      "-- region: 1505 , number of probes: 120\n",
      "-- region: 1506 , number of probes: 120\n",
      "-- region: 1507 , number of probes: 120\n",
      "-- region: 1508 , number of probes: 120\n",
      "-- region: 1509 , number of probes: 120\n",
      "-- region: 1510 , number of probes: 120\n",
      "-- region: 1511 , number of probes: 120\n",
      "-- region: 1512 , number of probes: 120\n",
      "-- region: 1513 , number of probes: 120\n",
      "-- region: 1514 , number of probes: 120\n",
      "-- region: 1515 , number of probes: 120\n",
      "-- region: 1516 , number of probes: 100\n",
      "-- region: 1517 , number of probes: 120\n",
      "-- region: 1518 , number of probes: 120\n",
      "-- region: 1519 , number of probes: 120\n",
      "-- region: 152 , number of probes: 120\n",
      "-- region: 1520 , number of probes: 120\n",
      "-- region: 1521 , number of probes: 120\n",
      "-- region: 1525 , number of probes: 47\n",
      "-- region: 1526 , number of probes: 55\n",
      "-- region: 1527 , number of probes: 68\n",
      "-- region: 1529 , number of probes: 120\n",
      "-- region: 153 , number of probes: 120\n",
      "-- region: 1530 , number of probes: 120\n",
      "-- region: 1533 , number of probes: 30\n",
      "-- region: 1534 , number of probes: 120\n",
      "-- region: 1535 , number of probes: 120\n",
      "-- region: 1536 , number of probes: 120\n",
      "-- region: 1537 , number of probes: 120\n",
      "-- region: 1538 , number of probes: 120\n",
      "-- region: 154 , number of probes: 120\n",
      "-- region: 1541 , number of probes: 120\n",
      "-- region: 1542 , number of probes: 120\n",
      "-- region: 1548 , number of probes: 102\n",
      "-- region: 155 , number of probes: 120\n",
      "-- region: 156 , number of probes: 120\n",
      "-- region: 157 , number of probes: 120\n",
      "-- region: 158 , number of probes: 120\n",
      "-- region: 159 , number of probes: 120\n",
      "-- region: 16 , number of probes: 120\n",
      "-- region: 160 , number of probes: 120\n",
      "-- region: 161 , number of probes: 120\n",
      "-- region: 162 , number of probes: 120\n",
      "-- region: 163 , number of probes: 120\n",
      "-- region: 164 , number of probes: 120\n",
      "-- region: 165 , number of probes: 120\n",
      "-- region: 166 , number of probes: 120\n",
      "-- region: 167 , number of probes: 120\n",
      "-- region: 168 , number of probes: 120\n",
      "-- region: 169 , number of probes: 120\n",
      "-- region: 17 , number of probes: 120\n",
      "-- region: 170 , number of probes: 120\n",
      "-- region: 171 , number of probes: 120\n",
      "-- region: 172 , number of probes: 120\n",
      "-- region: 173 , number of probes: 120\n",
      "-- region: 174 , number of probes: 120\n",
      "-- region: 175 , number of probes: 120\n",
      "-- region: 176 , number of probes: 120\n",
      "-- region: 177 , number of probes: 120\n",
      "-- region: 178 , number of probes: 120\n",
      "-- region: 179 , number of probes: 120\n",
      "-- region: 18 , number of probes: 120\n",
      "-- region: 180 , number of probes: 120\n",
      "-- region: 181 , number of probes: 120\n",
      "-- region: 182 , number of probes: 81\n",
      "-- region: 183 , number of probes: 120\n",
      "-- region: 184 , number of probes: 120\n",
      "-- region: 185 , number of probes: 120\n",
      "-- region: 186 , number of probes: 120\n",
      "-- region: 187 , number of probes: 120\n",
      "-- region: 188 , number of probes: 120\n",
      "-- region: 189 , number of probes: 120\n",
      "-- region: 19 , number of probes: 120\n",
      "-- region: 190 , number of probes: 120\n",
      "-- region: 191 , number of probes: 120\n",
      "-- region: 192 , number of probes: 120\n",
      "-- region: 193 , number of probes: 120\n",
      "-- region: 194 , number of probes: 120\n",
      "-- region: 195 , number of probes: 120\n",
      "-- region: 196 , number of probes: 120\n",
      "-- region: 197 , number of probes: 120\n",
      "-- region: 198 , number of probes: 120\n",
      "-- region: 199 , number of probes: 120\n",
      "-- region: 2 , number of probes: 120\n",
      "-- region: 20 , number of probes: 120\n",
      "-- region: 200 , number of probes: 120\n",
      "-- region: 201 , number of probes: 120\n",
      "-- region: 202 , number of probes: 120\n",
      "-- region: 203 , number of probes: 120\n",
      "-- region: 204 , number of probes: 120\n",
      "-- region: 205 , number of probes: 120\n",
      "-- region: 206 , number of probes: 120\n",
      "-- region: 207 , number of probes: 120\n",
      "-- region: 208 , number of probes: 120\n",
      "-- region: 209 , number of probes: 120\n",
      "-- region: 21 , number of probes: 120\n",
      "-- region: 210 , number of probes: 120\n",
      "-- region: 211 , number of probes: 120\n",
      "-- region: 212 , number of probes: 120\n",
      "-- region: 213 , number of probes: 120\n",
      "-- region: 214 , number of probes: 120\n",
      "-- region: 215 , number of probes: 120\n",
      "-- region: 216 , number of probes: 120\n",
      "-- region: 217 , number of probes: 120\n",
      "-- region: 218 , number of probes: 120\n",
      "-- region: 219 , number of probes: 120\n",
      "-- region: 22 , number of probes: 120\n",
      "-- region: 220 , number of probes: 120\n",
      "-- region: 221 , number of probes: 120\n",
      "-- region: 222 , number of probes: 120\n",
      "-- region: 223 , number of probes: 120\n",
      "-- region: 224 , number of probes: 120\n",
      "-- region: 225 , number of probes: 120\n",
      "-- region: 226 , number of probes: 120\n",
      "-- region: 227 , number of probes: 120\n",
      "-- region: 228 , number of probes: 120\n",
      "-- region: 229 , number of probes: 120\n",
      "-- region: 23 , number of probes: 120\n",
      "-- region: 230 , number of probes: 120\n",
      "-- region: 233 , number of probes: 120\n",
      "-- region: 234 , number of probes: 120\n",
      "-- region: 235 , number of probes: 120\n",
      "-- region: 236 , number of probes: 120\n",
      "-- region: 237 , number of probes: 120\n",
      "-- region: 238 , number of probes: 120\n",
      "-- region: 239 , number of probes: 120\n",
      "-- region: 24 , number of probes: 120\n",
      "-- region: 240 , number of probes: 120\n",
      "-- region: 241 , number of probes: 120\n",
      "-- region: 242 , number of probes: 120\n",
      "-- region: 243 , number of probes: 120\n",
      "-- region: 244 , number of probes: 120\n",
      "-- region: 245 , number of probes: 120\n",
      "-- region: 246 , number of probes: 120\n",
      "-- region: 247 , number of probes: 120\n",
      "-- region: 248 , number of probes: 120\n",
      "-- region: 249 , number of probes: 120\n",
      "-- region: 25 , number of probes: 120\n",
      "-- region: 250 , number of probes: 120\n",
      "-- region: 251 , number of probes: 120\n",
      "-- region: 252 , number of probes: 120\n",
      "-- region: 253 , number of probes: 120\n",
      "-- region: 254 , number of probes: 120\n",
      "-- region: 255 , number of probes: 120\n",
      "-- region: 256 , number of probes: 120\n",
      "-- region: 257 , number of probes: 120\n",
      "-- region: 258 , number of probes: 120\n",
      "-- region: 259 , number of probes: 120\n",
      "-- region: 26 , number of probes: 120\n",
      "-- region: 260 , number of probes: 120\n",
      "-- region: 261 , number of probes: 120\n",
      "-- region: 262 , number of probes: 120\n",
      "-- region: 263 , number of probes: 120\n",
      "-- region: 264 , number of probes: 120\n",
      "-- region: 265 , number of probes: 120\n",
      "-- region: 266 , number of probes: 120\n",
      "-- region: 267 , number of probes: 120\n",
      "-- region: 268 , number of probes: 120\n",
      "-- region: 269 , number of probes: 120\n",
      "-- region: 27 , number of probes: 120\n",
      "-- region: 270 , number of probes: 120\n",
      "-- region: 271 , number of probes: 120\n",
      "-- region: 272 , number of probes: 120\n",
      "-- region: 273 , number of probes: 120\n",
      "-- region: 274 , number of probes: 120\n",
      "-- region: 275 , number of probes: 120\n",
      "-- region: 276 , number of probes: 120\n",
      "-- region: 277 , number of probes: 120\n",
      "-- region: 278 , number of probes: 120\n",
      "-- region: 279 , number of probes: 120\n",
      "-- region: 28 , number of probes: 120\n",
      "-- region: 280 , number of probes: 120\n",
      "-- region: 281 , number of probes: 120\n",
      "-- region: 282 , number of probes: 120\n",
      "-- region: 283 , number of probes: 120\n",
      "-- region: 284 , number of probes: 120\n",
      "-- region: 285 , number of probes: 120\n",
      "-- region: 286 , number of probes: 120\n",
      "-- region: 287 , number of probes: 120\n",
      "-- region: 288 , number of probes: 120\n",
      "-- region: 289 , number of probes: 120\n",
      "-- region: 29 , number of probes: 120\n",
      "-- region: 290 , number of probes: 120\n",
      "-- region: 291 , number of probes: 120\n",
      "-- region: 292 , number of probes: 120\n",
      "-- region: 293 , number of probes: 120\n",
      "-- region: 295 , number of probes: 120\n",
      "-- region: 296 , number of probes: 120\n",
      "-- region: 297 , number of probes: 120\n",
      "-- region: 298 , number of probes: 120\n",
      "-- region: 299 , number of probes: 120\n",
      "-- region: 3 , number of probes: 120\n",
      "-- region: 30 , number of probes: 120\n",
      "-- region: 300 , number of probes: 120\n",
      "-- region: 302 , number of probes: 120\n",
      "-- region: 303 , number of probes: 120\n",
      "-- region: 304 , number of probes: 120\n",
      "-- region: 305 , number of probes: 120\n",
      "-- region: 306 , number of probes: 120\n",
      "-- region: 307 , number of probes: 120\n",
      "-- region: 308 , number of probes: 120\n",
      "-- region: 309 , number of probes: 120\n",
      "-- region: 31 , number of probes: 120\n",
      "-- region: 310 , number of probes: 120\n",
      "-- region: 311 , number of probes: 120\n",
      "-- region: 312 , number of probes: 120\n",
      "-- region: 313 , number of probes: 120\n",
      "-- region: 314 , number of probes: 120\n",
      "-- region: 315 , number of probes: 120\n",
      "-- region: 316 , number of probes: 120\n",
      "-- region: 317 , number of probes: 120\n",
      "-- region: 318 , number of probes: 120\n",
      "-- region: 319 , number of probes: 120\n",
      "-- region: 32 , number of probes: 120\n",
      "-- region: 320 , number of probes: 120\n",
      "-- region: 321 , number of probes: 120\n",
      "-- region: 322 , number of probes: 120\n",
      "-- region: 323 , number of probes: 120\n",
      "-- region: 324 , number of probes: 120\n",
      "-- region: 325 , number of probes: 120\n",
      "-- region: 326 , number of probes: 120\n",
      "-- region: 327 , number of probes: 91\n",
      "-- region: 328 , number of probes: 120\n",
      "-- region: 329 , number of probes: 120\n",
      "-- region: 33 , number of probes: 120\n",
      "-- region: 330 , number of probes: 120\n",
      "-- region: 331 , number of probes: 120\n",
      "-- region: 332 , number of probes: 120\n",
      "-- region: 333 , number of probes: 120\n",
      "-- region: 334 , number of probes: 120\n",
      "-- region: 335 , number of probes: 120\n",
      "-- region: 336 , number of probes: 120\n",
      "-- region: 337 , number of probes: 120\n",
      "-- region: 338 , number of probes: 120\n",
      "-- region: 339 , number of probes: 120\n",
      "-- region: 34 , number of probes: 120\n",
      "-- region: 340 , number of probes: 120\n",
      "-- region: 341 , number of probes: 120\n",
      "-- region: 342 , number of probes: 120\n",
      "-- region: 343 , number of probes: 120\n",
      "-- region: 344 , number of probes: 120\n",
      "-- region: 345 , number of probes: 120\n",
      "-- region: 346 , number of probes: 120\n",
      "-- region: 347 , number of probes: 120\n",
      "-- region: 348 , number of probes: 120\n",
      "-- region: 349 , number of probes: 120\n",
      "-- region: 35 , number of probes: 120\n",
      "-- region: 350 , number of probes: 120\n",
      "-- region: 351 , number of probes: 120\n",
      "-- region: 352 , number of probes: 120\n",
      "-- region: 353 , number of probes: 120\n",
      "-- region: 354 , number of probes: 120\n",
      "-- region: 355 , number of probes: 120\n",
      "-- region: 356 , number of probes: 120\n",
      "-- region: 357 , number of probes: 120\n",
      "-- region: 358 , number of probes: 120\n",
      "-- region: 359 , number of probes: 120\n",
      "-- region: 36 , number of probes: 120\n",
      "-- region: 360 , number of probes: 120\n",
      "-- region: 361 , number of probes: 120\n",
      "-- region: 362 , number of probes: 120\n",
      "-- region: 363 , number of probes: 120\n",
      "-- region: 364 , number of probes: 120\n",
      "-- region: 365 , number of probes: 120\n",
      "-- region: 366 , number of probes: 120\n",
      "-- region: 367 , number of probes: 83\n",
      "-- region: 368 , number of probes: 30\n",
      "-- region: 37 , number of probes: 120\n",
      "-- region: 370 , number of probes: 120\n",
      "-- region: 371 , number of probes: 74\n",
      "-- region: 372 , number of probes: 120\n",
      "-- region: 373 , number of probes: 120\n",
      "-- region: 374 , number of probes: 120\n",
      "-- region: 375 , number of probes: 120\n",
      "-- region: 376 , number of probes: 120\n",
      "-- region: 377 , number of probes: 120\n",
      "-- region: 379 , number of probes: 120\n",
      "-- region: 38 , number of probes: 120\n",
      "-- region: 380 , number of probes: 120\n",
      "-- region: 381 , number of probes: 120\n",
      "-- region: 382 , number of probes: 120\n",
      "-- region: 383 , number of probes: 120\n",
      "-- region: 384 , number of probes: 120\n",
      "-- region: 385 , number of probes: 120\n",
      "-- region: 386 , number of probes: 120\n",
      "-- region: 387 , number of probes: 120\n",
      "-- region: 388 , number of probes: 120\n",
      "-- region: 389 , number of probes: 120\n",
      "-- region: 39 , number of probes: 120\n",
      "-- region: 390 , number of probes: 120\n",
      "-- region: 391 , number of probes: 120\n",
      "-- region: 392 , number of probes: 120\n",
      "-- region: 393 , number of probes: 120\n",
      "-- region: 394 , number of probes: 120\n",
      "-- region: 395 , number of probes: 120\n",
      "-- region: 396 , number of probes: 120\n",
      "-- region: 397 , number of probes: 120\n",
      "-- region: 398 , number of probes: 120\n",
      "-- region: 399 , number of probes: 120\n",
      "-- region: 4 , number of probes: 120\n",
      "-- region: 40 , number of probes: 120\n",
      "-- region: 400 , number of probes: 120\n",
      "-- region: 401 , number of probes: 120\n",
      "-- region: 402 , number of probes: 120\n",
      "-- region: 403 , number of probes: 120\n",
      "-- region: 404 , number of probes: 120\n",
      "-- region: 405 , number of probes: 120\n",
      "-- region: 406 , number of probes: 120\n",
      "-- region: 407 , number of probes: 120\n",
      "-- region: 408 , number of probes: 120\n",
      "-- region: 409 , number of probes: 120\n",
      "-- region: 410 , number of probes: 120\n",
      "-- region: 411 , number of probes: 120\n",
      "-- region: 412 , number of probes: 120\n",
      "-- region: 413 , number of probes: 120\n",
      "-- region: 414 , number of probes: 120\n",
      "-- region: 415 , number of probes: 120\n",
      "-- region: 416 , number of probes: 120\n",
      "-- region: 417 , number of probes: 120\n",
      "-- region: 418 , number of probes: 120\n",
      "-- region: 419 , number of probes: 120\n",
      "-- region: 42 , number of probes: 120\n",
      "-- region: 421 , number of probes: 120\n",
      "-- region: 422 , number of probes: 120\n",
      "-- region: 423 , number of probes: 120\n",
      "-- region: 424 , number of probes: 120\n",
      "-- region: 425 , number of probes: 120\n",
      "-- region: 426 , number of probes: 120\n",
      "-- region: 427 , number of probes: 120\n",
      "-- region: 428 , number of probes: 120\n",
      "-- region: 429 , number of probes: 120\n",
      "-- region: 43 , number of probes: 120\n",
      "-- region: 430 , number of probes: 120\n",
      "-- region: 431 , number of probes: 120\n",
      "-- region: 432 , number of probes: 120\n",
      "-- region: 435 , number of probes: 120\n",
      "-- region: 436 , number of probes: 120\n",
      "-- region: 437 , number of probes: 120\n",
      "-- region: 438 , number of probes: 120\n",
      "-- region: 439 , number of probes: 120\n",
      "-- region: 44 , number of probes: 120\n",
      "-- region: 440 , number of probes: 120\n",
      "-- region: 441 , number of probes: 120\n",
      "-- region: 442 , number of probes: 120\n",
      "-- region: 443 , number of probes: 120\n",
      "-- region: 444 , number of probes: 120\n",
      "-- region: 446 , number of probes: 90\n",
      "-- region: 447 , number of probes: 120\n",
      "-- region: 448 , number of probes: 112\n",
      "-- region: 449 , number of probes: 120\n",
      "-- region: 45 , number of probes: 120\n",
      "-- region: 450 , number of probes: 120\n",
      "-- region: 451 , number of probes: 120\n",
      "-- region: 452 , number of probes: 120\n",
      "-- region: 453 , number of probes: 48\n",
      "-- region: 454 , number of probes: 120\n",
      "-- region: 455 , number of probes: 120\n",
      "-- region: 456 , number of probes: 120\n",
      "-- region: 457 , number of probes: 120\n",
      "-- region: 458 , number of probes: 120\n",
      "-- region: 459 , number of probes: 120\n",
      "-- region: 46 , number of probes: 120\n",
      "-- region: 460 , number of probes: 120\n",
      "-- region: 461 , number of probes: 120\n",
      "-- region: 462 , number of probes: 120\n",
      "-- region: 463 , number of probes: 120\n",
      "-- region: 464 , number of probes: 120\n",
      "-- region: 465 , number of probes: 120\n",
      "-- region: 466 , number of probes: 120\n",
      "-- region: 467 , number of probes: 120\n",
      "-- region: 468 , number of probes: 120\n",
      "-- region: 469 , number of probes: 120\n",
      "-- region: 47 , number of probes: 120\n",
      "-- region: 470 , number of probes: 114\n",
      "-- region: 471 , number of probes: 120\n",
      "-- region: 472 , number of probes: 120\n",
      "-- region: 473 , number of probes: 120\n",
      "-- region: 474 , number of probes: 120\n",
      "-- region: 475 , number of probes: 120\n",
      "-- region: 476 , number of probes: 120\n",
      "-- region: 477 , number of probes: 120\n",
      "-- region: 478 , number of probes: 120\n",
      "-- region: 479 , number of probes: 120\n",
      "-- region: 480 , number of probes: 120\n",
      "-- region: 481 , number of probes: 120\n",
      "-- region: 482 , number of probes: 120\n",
      "-- region: 483 , number of probes: 120\n",
      "-- region: 484 , number of probes: 120\n",
      "-- region: 485 , number of probes: 120\n",
      "-- region: 486 , number of probes: 120\n",
      "-- region: 487 , number of probes: 120\n",
      "-- region: 489 , number of probes: 120\n",
      "-- region: 490 , number of probes: 120\n",
      "-- region: 491 , number of probes: 120\n",
      "-- region: 492 , number of probes: 120\n",
      "-- region: 493 , number of probes: 120\n",
      "-- region: 494 , number of probes: 120\n",
      "-- region: 495 , number of probes: 120\n",
      "-- region: 496 , number of probes: 120\n",
      "-- region: 497 , number of probes: 120\n",
      "-- region: 498 , number of probes: 120\n",
      "-- region: 499 , number of probes: 120\n",
      "-- region: 5 , number of probes: 120\n",
      "-- region: 500 , number of probes: 120\n",
      "-- region: 501 , number of probes: 120\n",
      "-- region: 502 , number of probes: 120\n",
      "-- region: 503 , number of probes: 120\n",
      "-- region: 504 , number of probes: 120\n",
      "-- region: 505 , number of probes: 120\n",
      "-- region: 506 , number of probes: 120\n",
      "-- region: 507 , number of probes: 120\n",
      "-- region: 508 , number of probes: 120\n",
      "-- region: 509 , number of probes: 120\n",
      "-- region: 510 , number of probes: 120\n",
      "-- region: 511 , number of probes: 120\n",
      "-- region: 512 , number of probes: 120\n",
      "-- region: 516 , number of probes: 120\n",
      "-- region: 517 , number of probes: 120\n",
      "-- region: 518 , number of probes: 120\n",
      "-- region: 519 , number of probes: 120\n",
      "-- region: 520 , number of probes: 120\n",
      "-- region: 521 , number of probes: 120\n",
      "-- region: 522 , number of probes: 120\n",
      "-- region: 523 , number of probes: 120\n",
      "-- region: 524 , number of probes: 120\n",
      "-- region: 525 , number of probes: 120\n",
      "-- region: 526 , number of probes: 120\n",
      "-- region: 527 , number of probes: 120\n",
      "-- region: 528 , number of probes: 120\n",
      "-- region: 529 , number of probes: 120\n",
      "-- region: 530 , number of probes: 120\n",
      "-- region: 531 , number of probes: 120\n",
      "-- region: 532 , number of probes: 120\n",
      "-- region: 533 , number of probes: 120\n",
      "-- region: 534 , number of probes: 120\n",
      "-- region: 535 , number of probes: 112\n",
      "-- region: 536 , number of probes: 120\n",
      "-- region: 537 , number of probes: 120\n",
      "-- region: 538 , number of probes: 120\n",
      "-- region: 539 , number of probes: 120\n",
      "-- region: 540 , number of probes: 120\n",
      "-- region: 541 , number of probes: 120\n",
      "-- region: 542 , number of probes: 120\n",
      "-- region: 543 , number of probes: 120\n",
      "-- region: 544 , number of probes: 120\n",
      "-- region: 545 , number of probes: 120\n",
      "-- region: 546 , number of probes: 120\n",
      "-- region: 548 , number of probes: 120\n",
      "-- region: 549 , number of probes: 120\n",
      "-- region: 550 , number of probes: 120\n",
      "-- region: 551 , number of probes: 88\n",
      "-- region: 552 , number of probes: 120\n",
      "-- region: 554 , number of probes: 120\n",
      "-- region: 555 , number of probes: 120\n",
      "-- region: 556 , number of probes: 120\n",
      "-- region: 557 , number of probes: 120\n",
      "-- region: 558 , number of probes: 120\n",
      "-- region: 559 , number of probes: 120\n",
      "-- region: 560 , number of probes: 120\n",
      "-- region: 561 , number of probes: 120\n",
      "-- region: 562 , number of probes: 120\n",
      "-- region: 563 , number of probes: 120\n",
      "-- region: 564 , number of probes: 120\n",
      "-- region: 565 , number of probes: 120\n",
      "-- region: 566 , number of probes: 120\n",
      "-- region: 567 , number of probes: 120\n",
      "-- region: 568 , number of probes: 120\n",
      "-- region: 569 , number of probes: 120\n",
      "-- region: 57 , number of probes: 59\n",
      "-- region: 570 , number of probes: 46\n",
      "-- region: 571 , number of probes: 120\n",
      "-- region: 572 , number of probes: 120\n",
      "-- region: 573 , number of probes: 120\n",
      "-- region: 574 , number of probes: 120\n",
      "-- region: 575 , number of probes: 120\n",
      "-- region: 576 , number of probes: 120\n",
      "-- region: 577 , number of probes: 120\n",
      "-- region: 578 , number of probes: 120\n",
      "-- region: 579 , number of probes: 120\n",
      "-- region: 580 , number of probes: 120\n",
      "-- region: 581 , number of probes: 120\n",
      "-- region: 582 , number of probes: 120\n",
      "-- region: 583 , number of probes: 120\n",
      "-- region: 584 , number of probes: 120\n",
      "-- region: 585 , number of probes: 120\n",
      "-- region: 586 , number of probes: 120\n",
      "-- region: 587 , number of probes: 120\n",
      "-- region: 588 , number of probes: 120\n",
      "-- region: 589 , number of probes: 120\n",
      "-- region: 590 , number of probes: 120\n",
      "-- region: 591 , number of probes: 120\n",
      "-- region: 592 , number of probes: 120\n",
      "-- region: 593 , number of probes: 120\n",
      "-- region: 594 , number of probes: 120\n",
      "-- region: 595 , number of probes: 120\n",
      "-- region: 596 , number of probes: 120\n",
      "-- region: 597 , number of probes: 120\n",
      "-- region: 598 , number of probes: 120\n",
      "-- region: 599 , number of probes: 120\n",
      "-- region: 6 , number of probes: 120\n",
      "-- region: 60 , number of probes: 120\n",
      "-- region: 600 , number of probes: 120\n",
      "-- region: 601 , number of probes: 120\n",
      "-- region: 602 , number of probes: 120\n",
      "-- region: 603 , number of probes: 120\n",
      "-- region: 605 , number of probes: 120\n",
      "-- region: 606 , number of probes: 120\n",
      "-- region: 607 , number of probes: 120\n",
      "-- region: 608 , number of probes: 120\n",
      "-- region: 609 , number of probes: 120\n",
      "-- region: 611 , number of probes: 120\n",
      "-- region: 612 , number of probes: 120\n",
      "-- region: 613 , number of probes: 120\n",
      "-- region: 614 , number of probes: 120\n",
      "-- region: 615 , number of probes: 120\n",
      "-- region: 616 , number of probes: 120\n",
      "-- region: 617 , number of probes: 120\n",
      "-- region: 618 , number of probes: 120\n",
      "-- region: 619 , number of probes: 120\n",
      "-- region: 62 , number of probes: 120\n",
      "-- region: 620 , number of probes: 120\n",
      "-- region: 621 , number of probes: 120\n",
      "-- region: 622 , number of probes: 120\n",
      "-- region: 623 , number of probes: 120\n",
      "-- region: 624 , number of probes: 120\n",
      "-- region: 625 , number of probes: 120\n",
      "-- region: 626 , number of probes: 103\n",
      "-- region: 627 , number of probes: 120\n",
      "-- region: 628 , number of probes: 120\n",
      "-- region: 629 , number of probes: 120\n",
      "-- region: 63 , number of probes: 120\n",
      "-- region: 64 , number of probes: 120\n",
      "-- region: 645 , number of probes: 120\n",
      "-- region: 646 , number of probes: 120\n",
      "-- region: 647 , number of probes: 120\n",
      "-- region: 648 , number of probes: 120\n",
      "-- region: 649 , number of probes: 120\n",
      "-- region: 65 , number of probes: 101\n",
      "-- region: 650 , number of probes: 120\n",
      "-- region: 651 , number of probes: 120\n",
      "-- region: 652 , number of probes: 120\n",
      "-- region: 653 , number of probes: 120\n",
      "-- region: 654 , number of probes: 120\n",
      "-- region: 655 , number of probes: 120\n",
      "-- region: 656 , number of probes: 120\n",
      "-- region: 657 , number of probes: 120\n",
      "-- region: 658 , number of probes: 120\n",
      "-- region: 659 , number of probes: 48\n",
      "-- region: 66 , number of probes: 120\n",
      "-- region: 660 , number of probes: 120\n",
      "-- region: 662 , number of probes: 120\n",
      "-- region: 663 , number of probes: 120\n",
      "-- region: 664 , number of probes: 120\n",
      "-- region: 665 , number of probes: 120\n",
      "-- region: 666 , number of probes: 120\n",
      "-- region: 667 , number of probes: 120\n",
      "-- region: 668 , number of probes: 120\n",
      "-- region: 669 , number of probes: 120\n",
      "-- region: 67 , number of probes: 120\n",
      "-- region: 670 , number of probes: 120\n",
      "-- region: 671 , number of probes: 120\n",
      "-- region: 672 , number of probes: 120\n",
      "-- region: 673 , number of probes: 116\n",
      "-- region: 674 , number of probes: 120\n",
      "-- region: 675 , number of probes: 120\n",
      "-- region: 676 , number of probes: 120\n",
      "-- region: 677 , number of probes: 120\n",
      "-- region: 678 , number of probes: 120\n",
      "-- region: 679 , number of probes: 120\n",
      "-- region: 68 , number of probes: 120\n",
      "-- region: 680 , number of probes: 120\n",
      "-- region: 681 , number of probes: 120\n",
      "-- region: 682 , number of probes: 120\n",
      "-- region: 683 , number of probes: 120\n",
      "-- region: 684 , number of probes: 120\n",
      "-- region: 685 , number of probes: 120\n",
      "-- region: 686 , number of probes: 120\n",
      "-- region: 687 , number of probes: 43\n",
      "-- region: 688 , number of probes: 120\n",
      "-- region: 689 , number of probes: 120\n",
      "-- region: 69 , number of probes: 118\n",
      "-- region: 690 , number of probes: 120\n",
      "-- region: 691 , number of probes: 120\n",
      "-- region: 692 , number of probes: 120\n",
      "-- region: 693 , number of probes: 120\n",
      "-- region: 694 , number of probes: 120\n",
      "-- region: 695 , number of probes: 120\n",
      "-- region: 696 , number of probes: 120\n",
      "-- region: 697 , number of probes: 120\n",
      "-- region: 698 , number of probes: 120\n",
      "-- region: 699 , number of probes: 47\n",
      "-- region: 7 , number of probes: 120\n",
      "-- region: 70 , number of probes: 111\n",
      "-- region: 701 , number of probes: 120\n",
      "-- region: 702 , number of probes: 120\n",
      "-- region: 703 , number of probes: 120\n",
      "-- region: 704 , number of probes: 120\n",
      "-- region: 705 , number of probes: 120\n",
      "-- region: 706 , number of probes: 120\n",
      "-- region: 707 , number of probes: 120\n",
      "-- region: 708 , number of probes: 120\n",
      "-- region: 709 , number of probes: 120\n",
      "-- region: 71 , number of probes: 120\n",
      "-- region: 710 , number of probes: 120\n",
      "-- region: 711 , number of probes: 120\n",
      "-- region: 712 , number of probes: 120\n",
      "-- region: 713 , number of probes: 120\n",
      "-- region: 714 , number of probes: 120\n",
      "-- region: 715 , number of probes: 120\n",
      "-- region: 716 , number of probes: 120\n",
      "-- region: 717 , number of probes: 120\n",
      "-- region: 718 , number of probes: 120\n",
      "-- region: 719 , number of probes: 120\n",
      "-- region: 72 , number of probes: 120\n",
      "-- region: 720 , number of probes: 120\n",
      "-- region: 721 , number of probes: 120\n",
      "-- region: 722 , number of probes: 120\n",
      "-- region: 723 , number of probes: 68\n",
      "-- region: 724 , number of probes: 120\n",
      "-- region: 725 , number of probes: 120\n",
      "-- region: 726 , number of probes: 120\n",
      "-- region: 727 , number of probes: 120\n",
      "-- region: 728 , number of probes: 120\n",
      "-- region: 729 , number of probes: 120\n",
      "-- region: 73 , number of probes: 120\n",
      "-- region: 730 , number of probes: 120\n",
      "-- region: 731 , number of probes: 120\n",
      "-- region: 732 , number of probes: 120\n",
      "-- region: 733 , number of probes: 120\n",
      "-- region: 734 , number of probes: 120\n",
      "-- region: 735 , number of probes: 120\n",
      "-- region: 736 , number of probes: 120\n",
      "-- region: 737 , number of probes: 120\n",
      "-- region: 738 , number of probes: 120\n",
      "-- region: 739 , number of probes: 120\n",
      "-- region: 74 , number of probes: 120\n",
      "-- region: 740 , number of probes: 120\n",
      "-- region: 741 , number of probes: 120\n",
      "-- region: 742 , number of probes: 120\n",
      "-- region: 743 , number of probes: 120\n",
      "-- region: 744 , number of probes: 120\n",
      "-- region: 745 , number of probes: 120\n",
      "-- region: 748 , number of probes: 120\n",
      "-- region: 749 , number of probes: 120\n",
      "-- region: 75 , number of probes: 120\n",
      "-- region: 750 , number of probes: 120\n",
      "-- region: 751 , number of probes: 120\n",
      "-- region: 752 , number of probes: 120\n",
      "-- region: 753 , number of probes: 120\n",
      "-- region: 754 , number of probes: 120\n",
      "-- region: 755 , number of probes: 36\n",
      "-- region: 756 , number of probes: 120\n",
      "-- region: 757 , number of probes: 120\n",
      "-- region: 758 , number of probes: 120\n",
      "-- region: 759 , number of probes: 120\n",
      "-- region: 76 , number of probes: 86\n",
      "-- region: 760 , number of probes: 120\n",
      "-- region: 761 , number of probes: 120\n",
      "-- region: 762 , number of probes: 120\n",
      "-- region: 763 , number of probes: 120\n",
      "-- region: 764 , number of probes: 120\n",
      "-- region: 765 , number of probes: 120\n",
      "-- region: 766 , number of probes: 120\n",
      "-- region: 767 , number of probes: 120\n",
      "-- region: 768 , number of probes: 120\n",
      "-- region: 769 , number of probes: 120\n",
      "-- region: 77 , number of probes: 120\n",
      "-- region: 770 , number of probes: 79\n",
      "-- region: 774 , number of probes: 120\n",
      "-- region: 775 , number of probes: 120\n",
      "-- region: 776 , number of probes: 120\n",
      "-- region: 777 , number of probes: 120\n",
      "-- region: 778 , number of probes: 120\n",
      "-- region: 779 , number of probes: 120\n",
      "-- region: 78 , number of probes: 120\n",
      "-- region: 780 , number of probes: 120\n",
      "-- region: 782 , number of probes: 120\n",
      "-- region: 783 , number of probes: 120\n",
      "-- region: 784 , number of probes: 120\n",
      "-- region: 785 , number of probes: 120\n",
      "-- region: 786 , number of probes: 120\n",
      "-- region: 787 , number of probes: 120\n",
      "-- region: 788 , number of probes: 120\n",
      "-- region: 789 , number of probes: 120\n",
      "-- region: 79 , number of probes: 120\n",
      "-- region: 790 , number of probes: 87\n",
      "-- region: 791 , number of probes: 120\n",
      "-- region: 792 , number of probes: 120\n",
      "-- region: 793 , number of probes: 120\n",
      "-- region: 794 , number of probes: 120\n",
      "-- region: 795 , number of probes: 114\n",
      "-- region: 796 , number of probes: 120\n",
      "-- region: 797 , number of probes: 120\n",
      "-- region: 798 , number of probes: 120\n",
      "-- region: 799 , number of probes: 120\n",
      "-- region: 8 , number of probes: 120\n",
      "-- region: 80 , number of probes: 120\n",
      "-- region: 800 , number of probes: 120\n",
      "-- region: 801 , number of probes: 120\n",
      "-- region: 802 , number of probes: 120\n",
      "-- region: 803 , number of probes: 120\n",
      "-- region: 804 , number of probes: 120\n",
      "-- region: 805 , number of probes: 120\n",
      "-- region: 806 , number of probes: 120\n",
      "-- region: 807 , number of probes: 120\n",
      "-- region: 808 , number of probes: 120\n",
      "-- region: 809 , number of probes: 120\n",
      "-- region: 81 , number of probes: 120\n",
      "-- region: 810 , number of probes: 120\n",
      "-- region: 811 , number of probes: 120\n",
      "-- region: 812 , number of probes: 120\n",
      "-- region: 813 , number of probes: 120\n",
      "-- region: 814 , number of probes: 120\n",
      "-- region: 815 , number of probes: 120\n",
      "-- region: 816 , number of probes: 120\n",
      "-- region: 817 , number of probes: 120\n",
      "-- region: 819 , number of probes: 120\n",
      "-- region: 820 , number of probes: 120\n",
      "-- region: 821 , number of probes: 120\n",
      "-- region: 822 , number of probes: 120\n",
      "-- region: 823 , number of probes: 107\n",
      "-- region: 824 , number of probes: 120\n",
      "-- region: 825 , number of probes: 120\n",
      "-- region: 826 , number of probes: 120\n",
      "-- region: 827 , number of probes: 120\n",
      "-- region: 828 , number of probes: 120\n",
      "-- region: 829 , number of probes: 120\n",
      "-- region: 83 , number of probes: 120\n",
      "-- region: 832 , number of probes: 120\n",
      "-- region: 833 , number of probes: 120\n",
      "-- region: 834 , number of probes: 120\n",
      "-- region: 835 , number of probes: 120\n",
      "-- region: 836 , number of probes: 120\n",
      "-- region: 837 , number of probes: 120\n",
      "-- region: 838 , number of probes: 120\n",
      "-- region: 839 , number of probes: 120\n",
      "-- region: 84 , number of probes: 120\n",
      "-- region: 840 , number of probes: 120\n",
      "-- region: 841 , number of probes: 120\n",
      "-- region: 843 , number of probes: 120\n",
      "-- region: 844 , number of probes: 120\n",
      "-- region: 845 , number of probes: 120\n",
      "-- region: 846 , number of probes: 120\n",
      "-- region: 847 , number of probes: 120\n",
      "-- region: 848 , number of probes: 120\n",
      "-- region: 849 , number of probes: 109\n",
      "-- region: 85 , number of probes: 120\n",
      "-- region: 850 , number of probes: 120\n",
      "-- region: 851 , number of probes: 120\n",
      "-- region: 852 , number of probes: 120\n",
      "-- region: 853 , number of probes: 120\n",
      "-- region: 854 , number of probes: 120\n",
      "-- region: 855 , number of probes: 120\n",
      "-- region: 856 , number of probes: 120\n",
      "-- region: 857 , number of probes: 120\n",
      "-- region: 858 , number of probes: 120\n",
      "-- region: 859 , number of probes: 120\n",
      "-- region: 86 , number of probes: 120\n",
      "-- region: 860 , number of probes: 120\n",
      "-- region: 861 , number of probes: 120\n",
      "-- region: 862 , number of probes: 120\n",
      "-- region: 863 , number of probes: 120\n",
      "-- region: 864 , number of probes: 120\n",
      "-- region: 865 , number of probes: 120\n",
      "-- region: 866 , number of probes: 120\n",
      "-- region: 867 , number of probes: 120\n",
      "-- region: 868 , number of probes: 120\n",
      "-- region: 869 , number of probes: 120\n",
      "-- region: 87 , number of probes: 120\n",
      "-- region: 870 , number of probes: 120\n",
      "-- region: 871 , number of probes: 120\n",
      "-- region: 872 , number of probes: 120\n",
      "-- region: 873 , number of probes: 120\n",
      "-- region: 874 , number of probes: 120\n",
      "-- region: 875 , number of probes: 120\n",
      "-- region: 876 , number of probes: 120\n",
      "-- region: 877 , number of probes: 120\n",
      "-- region: 878 , number of probes: 120\n",
      "-- region: 88 , number of probes: 120\n",
      "-- region: 888 , number of probes: 120\n",
      "-- region: 889 , number of probes: 120\n",
      "-- region: 89 , number of probes: 120\n",
      "-- region: 890 , number of probes: 120\n",
      "-- region: 891 , number of probes: 120\n",
      "-- region: 892 , number of probes: 120\n",
      "-- region: 893 , number of probes: 120\n",
      "-- region: 894 , number of probes: 120\n",
      "-- region: 895 , number of probes: 120\n",
      "-- region: 896 , number of probes: 120\n",
      "-- region: 897 , number of probes: 120\n",
      "-- region: 898 , number of probes: 120\n",
      "-- region: 899 , number of probes: 120\n",
      "-- region: 9 , number of probes: 120\n",
      "-- region: 90 , number of probes: 110\n",
      "-- region: 900 , number of probes: 120\n",
      "-- region: 901 , number of probes: 120\n",
      "-- region: 902 , number of probes: 120\n",
      "-- region: 903 , number of probes: 120\n",
      "-- region: 904 , number of probes: 120\n",
      "-- region: 905 , number of probes: 120\n",
      "-- region: 906 , number of probes: 120\n",
      "-- region: 907 , number of probes: 120\n",
      "-- region: 908 , number of probes: 120\n",
      "-- region: 909 , number of probes: 120\n",
      "-- region: 91 , number of probes: 120\n",
      "-- region: 910 , number of probes: 120\n",
      "-- region: 911 , number of probes: 120\n",
      "-- region: 912 , number of probes: 120\n",
      "-- region: 913 , number of probes: 120\n",
      "-- region: 914 , number of probes: 120\n",
      "-- region: 915 , number of probes: 120\n",
      "-- region: 916 , number of probes: 120\n",
      "-- region: 917 , number of probes: 120\n",
      "-- region: 918 , number of probes: 120\n",
      "-- region: 919 , number of probes: 120\n",
      "-- region: 92 , number of probes: 120\n",
      "-- region: 920 , number of probes: 120\n",
      "-- region: 921 , number of probes: 120\n",
      "-- region: 922 , number of probes: 120\n",
      "-- region: 923 , number of probes: 120\n",
      "-- region: 924 , number of probes: 120\n",
      "-- region: 925 , number of probes: 120\n",
      "-- region: 926 , number of probes: 120\n",
      "-- region: 927 , number of probes: 120\n",
      "-- region: 928 , number of probes: 120\n",
      "-- region: 929 , number of probes: 120\n",
      "-- region: 93 , number of probes: 120\n",
      "-- region: 930 , number of probes: 120\n",
      "-- region: 931 , number of probes: 120\n",
      "-- region: 932 , number of probes: 120\n",
      "-- region: 933 , number of probes: 120\n",
      "-- region: 934 , number of probes: 120\n",
      "-- region: 935 , number of probes: 120\n",
      "-- region: 94 , number of probes: 120\n",
      "-- region: 949 , number of probes: 120\n",
      "-- region: 95 , number of probes: 120\n",
      "-- region: 950 , number of probes: 120\n",
      "-- region: 951 , number of probes: 120\n",
      "-- region: 952 , number of probes: 120\n",
      "-- region: 953 , number of probes: 120\n",
      "-- region: 954 , number of probes: 81\n",
      "-- region: 955 , number of probes: 120\n",
      "-- region: 956 , number of probes: 120\n",
      "-- region: 957 , number of probes: 120\n",
      "-- region: 958 , number of probes: 120\n",
      "-- region: 959 , number of probes: 120\n",
      "-- region: 96 , number of probes: 120\n",
      "-- region: 960 , number of probes: 120\n",
      "-- region: 961 , number of probes: 120\n",
      "-- region: 962 , number of probes: 120\n",
      "-- region: 963 , number of probes: 120\n",
      "-- region: 964 , number of probes: 120\n",
      "-- region: 965 , number of probes: 120\n",
      "-- region: 966 , number of probes: 120\n",
      "-- region: 967 , number of probes: 120\n",
      "-- region: 968 , number of probes: 120\n",
      "-- region: 969 , number of probes: 120\n",
      "-- region: 970 , number of probes: 80\n",
      "-- region: 971 , number of probes: 120\n",
      "-- region: 972 , number of probes: 120\n",
      "-- region: 973 , number of probes: 120\n",
      "-- region: 974 , number of probes: 120\n",
      "-- region: 975 , number of probes: 120\n",
      "-- region: 976 , number of probes: 120\n",
      "-- region: 977 , number of probes: 120\n",
      "-- region: 978 , number of probes: 120\n",
      "-- region: 979 , number of probes: 120\n",
      "-- region: 98 , number of probes: 120\n",
      "-- region: 980 , number of probes: 120\n",
      "-- region: 981 , number of probes: 120\n",
      "-- region: 982 , number of probes: 120\n",
      "-- region: 983 , number of probes: 120\n",
      "-- region: 984 , number of probes: 120\n",
      "-- region: 985 , number of probes: 120\n",
      "-- region: 986 , number of probes: 120\n",
      "-- region: 987 , number of probes: 120\n",
      "-- region: 988 , number of probes: 120\n",
      "-- region: 989 , number of probes: 120\n",
      "-- region: 990 , number of probes: 120\n",
      "-- region: 991 , number of probes: 120\n",
      "-- region: 992 , number of probes: 120\n",
      "-- region: 993 , number of probes: 120\n",
      "-- region: 994 , number of probes: 120\n",
      "-- region: 995 , number of probes: 120\n",
      "-- region: 996 , number of probes: 120\n",
      "-- region: 997 , number of probes: 120\n",
      "-- region: 999 , number of probes: 120\n",
      "- Number of probes kept: 159879\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\.\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "num_probes_per_region = 120\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(library_folder, \n",
    "                                                                                             num_probes_per_region,\n",
    "                                                                                             hard_thres=target_len,\n",
    "                                                                                             soft_count_th=50,\n",
    "                                                                                             )"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d06655b7",
   "metadata": {},
   "source": [
    "# 5. Summary Table and plots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "id": "b792aa76",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "159879\n"
     ]
    }
   ],
   "source": [
    "blasted_records = []\n",
    "with open(os.path.join(library_folder, 'blast_full_probes.fasta'), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        blasted_records.append(record)\n",
    "print(len(blasted_records))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 156,
   "id": "27cafd06",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "\n",
    "region_info_dict = {\n",
    "    'gene_name':[],\n",
    "    'locus_info':[],\n",
    "    'chr':[],\n",
    "    #'chr_order':[],\n",
    "    'num_probes':[],\n",
    "    'readouts': [],\n",
    "    #'type':[],\n",
    "}\n",
    "\n",
    "probe_name_re_string = r'loc_(?P<loc>.+)_gene_(?P<gene>.+)_pb_(?P<pb>[0-9]+)_pos_(?P<pos>[0-9]+)(.+)_readouts_\\[(?P<readouts>.+?)\\]_primers_\\[(?P<fwd_primers>.+?),(?P<rev_primers>.+?)\\]'\n",
    "\n",
    "for _pb in blasted_records:\n",
    "    \n",
    "    _match = re.match(probe_name_re_string, _pb.id, )\n",
    "    # append all if not exist\n",
    "    if _match.groupdict()['gene'] not in region_info_dict['gene_name']:\n",
    "        # append new\n",
    "        region_info_dict['gene_name'].append(_match.groupdict()['gene'])\n",
    "        region_info_dict['locus_info'].append(_match.groupdict()['loc'])\n",
    "        region_info_dict['chr'].append(_match.groupdict()['loc'].split(':')[0])\n",
    "        region_info_dict['num_probes'].append(1)\n",
    "        region_info_dict['readouts'].append(list(np.unique(_match.groupdict()['readouts'].split(','))))\n",
    "    else:\n",
    "        _ind = region_info_dict['gene_name'].index(_match.groupdict()['gene'])\n",
    "        region_info_dict['num_probes'][_ind] += 1\n",
    "        for _rd in np.unique(_match.groupdict()['readouts'].split(',')):\n",
    "            if _rd not in region_info_dict['readouts'][_ind]:\n",
    "                region_info_dict['readouts'][_ind].append(_rd)\n",
    "        \n",
    "    #break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 157,
   "id": "2979cac5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene_name</th>\n",
       "      <th>locus_info</th>\n",
       "      <th>chr</th>\n",
       "      <th>num_probes</th>\n",
       "      <th>readouts</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>1:1235001-1265000</td>\n",
       "      <td>1</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_683_c, NDB_758_c, NDB_887_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>1:3735001-3765000</td>\n",
       "      <td>1</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_631_c, NDB_1066_c, NDB_718_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10</td>\n",
       "      <td>1:26235001-26265000</td>\n",
       "      <td>1</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_812_c, NDB_637_c, NDB_844_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100</td>\n",
       "      <td>2:3735001-3765000</td>\n",
       "      <td>2</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_863_c, NDB_926_c, NDB_901_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1000</td>\n",
       "      <td>14:96735001-96765000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_784_c, NDB_883_c, NDB_925_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1347</th>\n",
       "      <td>994</td>\n",
       "      <td>14:87735001-87765000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_929_c, NDB_680_c, NDB_899_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1348</th>\n",
       "      <td>995</td>\n",
       "      <td>14:89235001-89265000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_758_c, NDB_788_c, NDB_958_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1349</th>\n",
       "      <td>996</td>\n",
       "      <td>14:90735001-90765000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_725_c, NDB_805_c, NDB_752_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1350</th>\n",
       "      <td>997</td>\n",
       "      <td>14:92235001-92265000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_965_c, NDB_946_c, NDB_665_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1351</th>\n",
       "      <td>999</td>\n",
       "      <td>14:95235001-95265000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_940_c, NDB_961_c, NDB_838_c]</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1352 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     gene_name            locus_info chr  num_probes  \\\n",
       "0            0     1:1235001-1265000   1         120   \n",
       "1            1     1:3735001-3765000   1         120   \n",
       "2           10   1:26235001-26265000   1         120   \n",
       "3          100     2:3735001-3765000   2         120   \n",
       "4         1000  14:96735001-96765000  14         120   \n",
       "...        ...                   ...  ..         ...   \n",
       "1347       994  14:87735001-87765000  14         120   \n",
       "1348       995  14:89235001-89265000  14         120   \n",
       "1349       996  14:90735001-90765000  14         120   \n",
       "1350       997  14:92235001-92265000  14         120   \n",
       "1351       999  14:95235001-95265000  14         120   \n",
       "\n",
       "                                readouts  \n",
       "0      [NDB_683_c, NDB_758_c, NDB_887_c]  \n",
       "1     [NDB_631_c, NDB_1066_c, NDB_718_c]  \n",
       "2      [NDB_812_c, NDB_637_c, NDB_844_c]  \n",
       "3      [NDB_863_c, NDB_926_c, NDB_901_c]  \n",
       "4      [NDB_784_c, NDB_883_c, NDB_925_c]  \n",
       "...                                  ...  \n",
       "1347   [NDB_929_c, NDB_680_c, NDB_899_c]  \n",
       "1348   [NDB_758_c, NDB_788_c, NDB_958_c]  \n",
       "1349   [NDB_725_c, NDB_805_c, NDB_752_c]  \n",
       "1350   [NDB_965_c, NDB_946_c, NDB_665_c]  \n",
       "1351   [NDB_940_c, NDB_961_c, NDB_838_c]  \n",
       "\n",
       "[1352 rows x 5 columns]"
      ]
     },
     "execution_count": 157,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_info_df = pd.DataFrame(region_info_dict)\n",
    "region_info_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 158,
   "id": "f822e25a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Add additional information\n",
    "_types = []\n",
    "for _g in region_info_df['gene_name']:\n",
    "    if 'hSE-' in _g:\n",
    "        _types.append('SE')\n",
    "    elif 'enhancer-' in _g:\n",
    "        _types.append('enhancer')\n",
    "    else:\n",
    "        _types.append('genome')\n",
    "# add to this df\n",
    "region_info_df['type'] = _types\n",
    "\n",
    "region_info_df['chr_order'] = -1\n",
    "for _chr in np.unique(region_info_df['chr']):\n",
    "    _reg_starts = [int(_info.split(':')[1].split('-')[0])\n",
    "                   for _info in region_info_df.loc[region_info_df['chr']==_chr,\n",
    "                                                   'locus_info']\n",
    "                  ]\n",
    "    _reg_orders = np.array(_reg_starts).argsort().argsort().astype(np.int32)\n",
    "    region_info_df.loc[region_info_df['chr']==_chr,'chr_order'] = _reg_orders"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 159,
   "id": "d319419e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene_name</th>\n",
       "      <th>locus_info</th>\n",
       "      <th>chr</th>\n",
       "      <th>num_probes</th>\n",
       "      <th>readouts</th>\n",
       "      <th>type</th>\n",
       "      <th>chr_order</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>1:1235001-1265000</td>\n",
       "      <td>1</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_683_c, NDB_758_c, NDB_887_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>1:3735001-3765000</td>\n",
       "      <td>1</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_631_c, NDB_1066_c, NDB_718_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10</td>\n",
       "      <td>1:26235001-26265000</td>\n",
       "      <td>1</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_812_c, NDB_637_c, NDB_844_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100</td>\n",
       "      <td>2:3735001-3765000</td>\n",
       "      <td>2</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_863_c, NDB_926_c, NDB_901_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1000</td>\n",
       "      <td>14:96735001-96765000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_784_c, NDB_883_c, NDB_925_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1347</th>\n",
       "      <td>994</td>\n",
       "      <td>14:87735001-87765000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_929_c, NDB_680_c, NDB_899_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>45</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1348</th>\n",
       "      <td>995</td>\n",
       "      <td>14:89235001-89265000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_758_c, NDB_788_c, NDB_958_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>46</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1349</th>\n",
       "      <td>996</td>\n",
       "      <td>14:90735001-90765000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_725_c, NDB_805_c, NDB_752_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1350</th>\n",
       "      <td>997</td>\n",
       "      <td>14:92235001-92265000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_965_c, NDB_946_c, NDB_665_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>48</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1351</th>\n",
       "      <td>999</td>\n",
       "      <td>14:95235001-95265000</td>\n",
       "      <td>14</td>\n",
       "      <td>120</td>\n",
       "      <td>[NDB_940_c, NDB_961_c, NDB_838_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>49</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1352 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     gene_name            locus_info chr  num_probes  \\\n",
       "0            0     1:1235001-1265000   1         120   \n",
       "1            1     1:3735001-3765000   1         120   \n",
       "2           10   1:26235001-26265000   1         120   \n",
       "3          100     2:3735001-3765000   2         120   \n",
       "4         1000  14:96735001-96765000  14         120   \n",
       "...        ...                   ...  ..         ...   \n",
       "1347       994  14:87735001-87765000  14         120   \n",
       "1348       995  14:89235001-89265000  14         120   \n",
       "1349       996  14:90735001-90765000  14         120   \n",
       "1350       997  14:92235001-92265000  14         120   \n",
       "1351       999  14:95235001-95265000  14         120   \n",
       "\n",
       "                                readouts    type  chr_order  \n",
       "0      [NDB_683_c, NDB_758_c, NDB_887_c]  genome          0  \n",
       "1     [NDB_631_c, NDB_1066_c, NDB_718_c]  genome          1  \n",
       "2      [NDB_812_c, NDB_637_c, NDB_844_c]  genome         10  \n",
       "3      [NDB_863_c, NDB_926_c, NDB_901_c]  genome          0  \n",
       "4      [NDB_784_c, NDB_883_c, NDB_925_c]  genome         50  \n",
       "...                                  ...     ...        ...  \n",
       "1347   [NDB_929_c, NDB_680_c, NDB_899_c]  genome         45  \n",
       "1348   [NDB_758_c, NDB_788_c, NDB_958_c]  genome         46  \n",
       "1349   [NDB_725_c, NDB_805_c, NDB_752_c]  genome         47  \n",
       "1350   [NDB_965_c, NDB_946_c, NDB_665_c]  genome         48  \n",
       "1351   [NDB_940_c, NDB_961_c, NDB_838_c]  genome         49  \n",
       "\n",
       "[1352 rows x 7 columns]"
      ]
     },
     "execution_count": 159,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_info_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 160,
   "id": "807aa9f2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "saving region info to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\summary_region_info.csv\n"
     ]
    }
   ],
   "source": [
    "overwrite_region_info = True\n",
    "region_info_filename = os.path.join(library_folder, r'summary_region_info.csv')\n",
    "\n",
    "if not os.path.exists(region_info_filename) or overwrite_region_info:\n",
    "    print(f\"saving region info to file: {region_info_filename}\")\n",
    "    region_info_df.to_csv(region_info_filename, index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0d1a3125",
   "metadata": {},
   "source": [
    "## Generate summary plots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 161,
   "id": "1bbf42ac",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: 0 -> 120\n",
      "gene: 1 -> 120\n",
      "gene: 10 -> 120\n",
      "gene: 100 -> 120\n",
      "gene: 1000 -> 120\n",
      "gene: 1001 -> 120\n",
      "gene: 1002 -> 120\n",
      "gene: 1003 -> 120\n",
      "gene: 1004 -> 120\n",
      "gene: 1005 -> 120\n",
      "gene: 101 -> 120\n",
      "gene: 102 -> 120\n",
      "gene: 1020 -> 49\n",
      "gene: 1023 -> 120\n",
      "gene: 1024 -> 120\n",
      "gene: 1025 -> 120\n",
      "gene: 1029 -> 120\n",
      "gene: 103 -> 120\n",
      "gene: 1030 -> 120\n",
      "gene: 1031 -> 120\n",
      "gene: 1032 -> 120\n",
      "gene: 1033 -> 120\n",
      "gene: 1034 -> 120\n",
      "gene: 1035 -> 120\n",
      "gene: 1036 -> 120\n",
      "gene: 1037 -> 120\n",
      "gene: 1038 -> 120\n",
      "gene: 1039 -> 120\n",
      "gene: 104 -> 96\n",
      "gene: 1040 -> 120\n",
      "gene: 1041 -> 120\n",
      "gene: 1042 -> 120\n",
      "gene: 1043 -> 105\n",
      "gene: 1044 -> 120\n",
      "gene: 1045 -> 120\n",
      "gene: 1046 -> 120\n",
      "gene: 1047 -> 120\n",
      "gene: 1048 -> 120\n",
      "gene: 1049 -> 120\n",
      "gene: 105 -> 120\n",
      "gene: 1050 -> 120\n",
      "gene: 1051 -> 120\n",
      "gene: 1052 -> 120\n",
      "gene: 1053 -> 120\n",
      "gene: 1054 -> 120\n",
      "gene: 1055 -> 120\n",
      "gene: 1056 -> 120\n",
      "gene: 1057 -> 120\n",
      "gene: 1058 -> 120\n",
      "gene: 1059 -> 120\n",
      "gene: 106 -> 120\n",
      "gene: 1060 -> 120\n",
      "gene: 1061 -> 120\n",
      "gene: 1062 -> 120\n",
      "gene: 1063 -> 120\n",
      "gene: 1064 -> 120\n",
      "gene: 1065 -> 120\n",
      "gene: 1066 -> 120\n",
      "gene: 1067 -> 120\n",
      "gene: 1068 -> 120\n",
      "gene: 1069 -> 120\n",
      "gene: 107 -> 120\n",
      "gene: 1070 -> 120\n",
      "gene: 1071 -> 120\n",
      "gene: 1072 -> 120\n",
      "gene: 1073 -> 120\n",
      "gene: 1074 -> 120\n",
      "gene: 1075 -> 120\n",
      "gene: 1076 -> 120\n",
      "gene: 1077 -> 86\n",
      "gene: 1078 -> 120\n",
      "gene: 1079 -> 120\n",
      "gene: 108 -> 120\n",
      "gene: 1080 -> 120\n",
      "gene: 1081 -> 120\n",
      "gene: 1082 -> 120\n",
      "gene: 1083 -> 120\n",
      "gene: 1085 -> 120\n",
      "gene: 1087 -> 120\n",
      "gene: 1089 -> 120\n",
      "gene: 109 -> 120\n",
      "gene: 1090 -> 120\n",
      "gene: 1091 -> 120\n",
      "gene: 1092 -> 120\n",
      "gene: 1093 -> 120\n",
      "gene: 1094 -> 120\n",
      "gene: 1097 -> 120\n",
      "gene: 11 -> 120\n",
      "gene: 110 -> 120\n",
      "gene: 1105 -> 120\n",
      "gene: 1106 -> 120\n",
      "gene: 1107 -> 120\n",
      "gene: 1108 -> 120\n",
      "gene: 1109 -> 120\n",
      "gene: 111 -> 120\n",
      "gene: 1110 -> 120\n",
      "gene: 1111 -> 120\n",
      "gene: 1112 -> 120\n",
      "gene: 1113 -> 120\n",
      "gene: 1114 -> 120\n",
      "gene: 1115 -> 120\n",
      "gene: 1116 -> 120\n",
      "gene: 1117 -> 120\n",
      "gene: 1118 -> 120\n",
      "gene: 1119 -> 120\n",
      "gene: 112 -> 120\n",
      "gene: 1120 -> 120\n",
      "gene: 1121 -> 120\n",
      "gene: 1122 -> 120\n",
      "gene: 1123 -> 120\n",
      "gene: 1124 -> 120\n",
      "gene: 1125 -> 120\n",
      "gene: 1126 -> 120\n",
      "gene: 1127 -> 120\n",
      "gene: 1128 -> 120\n",
      "gene: 1129 -> 120\n",
      "gene: 1130 -> 120\n",
      "gene: 1131 -> 120\n",
      "gene: 1132 -> 120\n",
      "gene: 1133 -> 120\n",
      "gene: 1134 -> 120\n",
      "gene: 1135 -> 120\n",
      "gene: 1136 -> 120\n",
      "gene: 1137 -> 120\n",
      "gene: 1138 -> 120\n",
      "gene: 1139 -> 120\n",
      "gene: 114 -> 120\n",
      "gene: 1140 -> 120\n",
      "gene: 1141 -> 120\n",
      "gene: 1142 -> 120\n",
      "gene: 1143 -> 120\n",
      "gene: 1144 -> 117\n",
      "gene: 1145 -> 120\n",
      "gene: 1146 -> 92\n",
      "gene: 1147 -> 120\n",
      "gene: 1148 -> 98\n",
      "gene: 115 -> 120\n",
      "gene: 1152 -> 120\n",
      "gene: 1153 -> 120\n",
      "gene: 1154 -> 52\n",
      "gene: 1155 -> 120\n",
      "gene: 1156 -> 120\n",
      "gene: 1157 -> 120\n",
      "gene: 116 -> 120\n",
      "gene: 1160 -> 120\n",
      "gene: 1162 -> 120\n",
      "gene: 1163 -> 120\n",
      "gene: 1165 -> 120\n",
      "gene: 1166 -> 120\n",
      "gene: 1167 -> 120\n",
      "gene: 1168 -> 120\n",
      "gene: 1169 -> 120\n",
      "gene: 117 -> 120\n",
      "gene: 1170 -> 120\n",
      "gene: 1171 -> 120\n",
      "gene: 1172 -> 120\n",
      "gene: 1173 -> 90\n",
      "gene: 1174 -> 115\n",
      "gene: 1176 -> 120\n",
      "gene: 1177 -> 120\n",
      "gene: 1178 -> 120\n",
      "gene: 1179 -> 120\n",
      "gene: 118 -> 120\n",
      "gene: 1180 -> 120\n",
      "gene: 1181 -> 120\n",
      "gene: 1182 -> 120\n",
      "gene: 1183 -> 120\n",
      "gene: 1184 -> 120\n",
      "gene: 1186 -> 120\n",
      "gene: 1187 -> 120\n",
      "gene: 1188 -> 77\n",
      "gene: 1189 -> 120\n",
      "gene: 119 -> 120\n",
      "gene: 1190 -> 120\n",
      "gene: 1191 -> 120\n",
      "gene: 1192 -> 120\n",
      "gene: 1193 -> 120\n",
      "gene: 1194 -> 120\n",
      "gene: 1195 -> 120\n",
      "gene: 1196 -> 120\n",
      "gene: 1197 -> 120\n",
      "gene: 12 -> 120\n",
      "gene: 120 -> 120\n",
      "gene: 1203 -> 120\n",
      "gene: 1204 -> 120\n",
      "gene: 1205 -> 120\n",
      "gene: 1206 -> 86\n",
      "gene: 1207 -> 120\n",
      "gene: 1208 -> 120\n",
      "gene: 1209 -> 120\n",
      "gene: 121 -> 120\n",
      "gene: 1210 -> 120\n",
      "gene: 1211 -> 120\n",
      "gene: 1212 -> 120\n",
      "gene: 1213 -> 120\n",
      "gene: 1214 -> 120\n",
      "gene: 1215 -> 120\n",
      "gene: 1216 -> 120\n",
      "gene: 1217 -> 120\n",
      "gene: 1218 -> 120\n",
      "gene: 1219 -> 120\n",
      "gene: 122 -> 120\n",
      "gene: 1220 -> 120\n",
      "gene: 1221 -> 120\n",
      "gene: 1222 -> 120\n",
      "gene: 1224 -> 120\n",
      "gene: 1225 -> 120\n",
      "gene: 1226 -> 120\n",
      "gene: 1227 -> 120\n",
      "gene: 1228 -> 120\n",
      "gene: 1229 -> 120\n",
      "gene: 123 -> 120\n",
      "gene: 1230 -> 120\n",
      "gene: 1231 -> 120\n",
      "gene: 1232 -> 120\n",
      "gene: 1233 -> 120\n",
      "gene: 1234 -> 120\n",
      "gene: 1235 -> 120\n",
      "gene: 1236 -> 120\n",
      "gene: 1237 -> 120\n",
      "gene: 1238 -> 120\n",
      "gene: 1239 -> 120\n",
      "gene: 124 -> 120\n",
      "gene: 1240 -> 120\n",
      "gene: 1241 -> 120\n",
      "gene: 1242 -> 120\n",
      "gene: 1243 -> 120\n",
      "gene: 1244 -> 120\n",
      "gene: 1245 -> 120\n",
      "gene: 1246 -> 120\n",
      "gene: 1247 -> 120\n",
      "gene: 1248 -> 120\n",
      "gene: 1249 -> 120\n",
      "gene: 125 -> 120\n",
      "gene: 1250 -> 120\n",
      "gene: 1251 -> 120\n",
      "gene: 1252 -> 120\n",
      "gene: 1253 -> 120\n",
      "gene: 1254 -> 120\n",
      "gene: 1255 -> 120\n",
      "gene: 1256 -> 120\n",
      "gene: 1257 -> 120\n",
      "gene: 1258 -> 120\n",
      "gene: 1259 -> 120\n",
      "gene: 126 -> 120\n",
      "gene: 1260 -> 120\n",
      "gene: 1261 -> 120\n",
      "gene: 1262 -> 63\n",
      "gene: 1263 -> 120\n",
      "gene: 1264 -> 120\n",
      "gene: 1265 -> 81\n",
      "gene: 1269 -> 21\n",
      "gene: 127 -> 120\n",
      "gene: 1270 -> 120\n",
      "gene: 1271 -> 120\n",
      "gene: 1272 -> 120\n",
      "gene: 1273 -> 120\n",
      "gene: 1274 -> 120\n",
      "gene: 1275 -> 120\n",
      "gene: 1276 -> 120\n",
      "gene: 1277 -> 120\n",
      "gene: 1278 -> 120\n",
      "gene: 1279 -> 120\n",
      "gene: 128 -> 120\n",
      "gene: 1280 -> 120\n",
      "gene: 1281 -> 120\n",
      "gene: 1282 -> 120\n",
      "gene: 1284 -> 120\n",
      "gene: 1285 -> 120\n",
      "gene: 1286 -> 120\n",
      "gene: 1287 -> 120\n",
      "gene: 1288 -> 120\n",
      "gene: 1289 -> 120\n",
      "gene: 129 -> 120\n",
      "gene: 1290 -> 120\n",
      "gene: 1291 -> 120\n",
      "gene: 1292 -> 120\n",
      "gene: 1293 -> 120\n",
      "gene: 1294 -> 78\n",
      "gene: 1295 -> 120\n",
      "gene: 1297 -> 120\n",
      "gene: 1298 -> 120\n",
      "gene: 1299 -> 120\n",
      "gene: 13 -> 120\n",
      "gene: 130 -> 120\n",
      "gene: 1300 -> 120\n",
      "gene: 1301 -> 120\n",
      "gene: 1302 -> 120\n",
      "gene: 1303 -> 120\n",
      "gene: 1304 -> 120\n",
      "gene: 1305 -> 120\n",
      "gene: 1306 -> 120\n",
      "gene: 1307 -> 120\n",
      "gene: 1308 -> 120\n",
      "gene: 1309 -> 120\n",
      "gene: 131 -> 120\n",
      "gene: 1310 -> 120\n",
      "gene: 1311 -> 120\n",
      "gene: 1312 -> 120\n",
      "gene: 1313 -> 120\n",
      "gene: 1314 -> 120\n",
      "gene: 1315 -> 120\n",
      "gene: 1316 -> 120\n",
      "gene: 1317 -> 120\n",
      "gene: 1318 -> 120\n",
      "gene: 1319 -> 120\n",
      "gene: 132 -> 120\n",
      "gene: 1320 -> 120\n",
      "gene: 1321 -> 120\n",
      "gene: 1322 -> 120\n",
      "gene: 1323 -> 120\n",
      "gene: 1324 -> 120\n",
      "gene: 1325 -> 120\n",
      "gene: 133 -> 120\n",
      "gene: 1330 -> 120\n",
      "gene: 1331 -> 120\n",
      "gene: 1332 -> 120\n",
      "gene: 1333 -> 120\n",
      "gene: 1334 -> 120\n",
      "gene: 1335 -> 120\n",
      "gene: 1336 -> 120\n",
      "gene: 1337 -> 120\n",
      "gene: 1338 -> 120\n",
      "gene: 1339 -> 120\n",
      "gene: 134 -> 120\n",
      "gene: 1340 -> 120\n",
      "gene: 1341 -> 120\n",
      "gene: 1342 -> 120\n",
      "gene: 1343 -> 120\n",
      "gene: 1344 -> 120\n",
      "gene: 1345 -> 120\n",
      "gene: 1346 -> 120\n",
      "gene: 1347 -> 120\n",
      "gene: 1348 -> 88\n",
      "gene: 1349 -> 120\n",
      "gene: 1350 -> 120\n",
      "gene: 1351 -> 120\n",
      "gene: 1352 -> 120\n",
      "gene: 1353 -> 120\n",
      "gene: 1354 -> 120\n",
      "gene: 1355 -> 120\n",
      "gene: 1356 -> 120\n",
      "gene: 1357 -> 120\n",
      "gene: 1358 -> 120\n",
      "gene: 1359 -> 120\n",
      "gene: 1360 -> 120\n",
      "gene: 1361 -> 120\n",
      "gene: 1362 -> 120\n",
      "gene: 1363 -> 120\n",
      "gene: 137 -> 120\n",
      "gene: 1378 -> 120\n",
      "gene: 1379 -> 120\n",
      "gene: 138 -> 120\n",
      "gene: 1380 -> 120\n",
      "gene: 1381 -> 120\n",
      "gene: 1382 -> 120\n",
      "gene: 1383 -> 120\n",
      "gene: 1384 -> 120\n",
      "gene: 1385 -> 120\n",
      "gene: 1386 -> 117\n",
      "gene: 1387 -> 120\n",
      "gene: 1388 -> 120\n",
      "gene: 1389 -> 120\n",
      "gene: 139 -> 120\n",
      "gene: 1390 -> 120\n",
      "gene: 1391 -> 120\n",
      "gene: 1392 -> 120\n",
      "gene: 1393 -> 120\n",
      "gene: 1394 -> 120\n",
      "gene: 1395 -> 120\n",
      "gene: 1396 -> 120\n",
      "gene: 1398 -> 120\n",
      "gene: 1399 -> 120\n",
      "gene: 14 -> 120\n",
      "gene: 140 -> 120\n",
      "gene: 1400 -> 120\n",
      "gene: 1401 -> 120\n",
      "gene: 1402 -> 120\n",
      "gene: 1403 -> 120\n",
      "gene: 1404 -> 120\n",
      "gene: 1405 -> 120\n",
      "gene: 1406 -> 120\n",
      "gene: 1408 -> 120\n",
      "gene: 1409 -> 120\n",
      "gene: 141 -> 68\n",
      "gene: 1427 -> 120\n",
      "gene: 1429 -> 120\n",
      "gene: 143 -> 120\n",
      "gene: 1430 -> 120\n",
      "gene: 1434 -> 120\n",
      "gene: 1435 -> 120\n",
      "gene: 1436 -> 120\n",
      "gene: 1437 -> 120\n",
      "gene: 1438 -> 120\n",
      "gene: 1439 -> 120\n",
      "gene: 144 -> 120\n",
      "gene: 1440 -> 120\n",
      "gene: 1441 -> 120\n",
      "gene: 1442 -> 120\n",
      "gene: 1443 -> 120\n",
      "gene: 1444 -> 120\n",
      "gene: 1445 -> 120\n",
      "gene: 1446 -> 120\n",
      "gene: 1447 -> 120\n",
      "gene: 1448 -> 120\n",
      "gene: 1449 -> 120\n",
      "gene: 145 -> 120\n",
      "gene: 1450 -> 120\n",
      "gene: 1451 -> 120\n",
      "gene: 1452 -> 109\n",
      "gene: 1453 -> 120\n",
      "gene: 1454 -> 120\n",
      "gene: 1455 -> 120\n",
      "gene: 1456 -> 120\n",
      "gene: 1457 -> 120\n",
      "gene: 1458 -> 120\n",
      "gene: 1459 -> 120\n",
      "gene: 146 -> 120\n",
      "gene: 1461 -> 120\n",
      "gene: 1462 -> 120\n",
      "gene: 1463 -> 120\n",
      "gene: 1464 -> 120\n",
      "gene: 1465 -> 120\n",
      "gene: 1466 -> 120\n",
      "gene: 1467 -> 120\n",
      "gene: 1468 -> 120\n",
      "gene: 1469 -> 120\n",
      "gene: 147 -> 120\n",
      "gene: 1470 -> 120\n",
      "gene: 1471 -> 120\n",
      "gene: 1472 -> 120\n",
      "gene: 1473 -> 100\n",
      "gene: 1474 -> 120\n",
      "gene: 1475 -> 120\n",
      "gene: 1476 -> 120\n",
      "gene: 1477 -> 120\n",
      "gene: 1478 -> 120\n",
      "gene: 1479 -> 120\n",
      "gene: 148 -> 120\n",
      "gene: 1480 -> 104\n",
      "gene: 1481 -> 120\n",
      "gene: 1482 -> 120\n",
      "gene: 1485 -> 120\n",
      "gene: 1486 -> 120\n",
      "gene: 1487 -> 120\n",
      "gene: 1488 -> 120\n",
      "gene: 1489 -> 73\n",
      "gene: 149 -> 120\n",
      "gene: 1490 -> 113\n",
      "gene: 1491 -> 120\n",
      "gene: 1492 -> 120\n",
      "gene: 1493 -> 114\n",
      "gene: 1494 -> 120\n",
      "gene: 1495 -> 120\n",
      "gene: 1496 -> 63\n",
      "gene: 1497 -> 120\n",
      "gene: 1498 -> 120\n",
      "gene: 1499 -> 120\n",
      "gene: 15 -> 120\n",
      "gene: 150 -> 120\n",
      "gene: 1500 -> 120\n",
      "gene: 1501 -> 120\n",
      "gene: 1502 -> 110\n",
      "gene: 1503 -> 120\n",
      "gene: 1504 -> 120\n",
      "gene: 1505 -> 120\n",
      "gene: 1506 -> 120\n",
      "gene: 1507 -> 120\n",
      "gene: 1508 -> 120\n",
      "gene: 1509 -> 120\n",
      "gene: 1510 -> 120\n",
      "gene: 1511 -> 120\n",
      "gene: 1512 -> 120\n",
      "gene: 1513 -> 120\n",
      "gene: 1514 -> 120\n",
      "gene: 1515 -> 120\n",
      "gene: 1516 -> 100\n",
      "gene: 1517 -> 120\n",
      "gene: 1518 -> 120\n",
      "gene: 1519 -> 120\n",
      "gene: 152 -> 120\n",
      "gene: 1520 -> 120\n",
      "gene: 1521 -> 120\n",
      "gene: 1525 -> 47\n",
      "gene: 1526 -> 55\n",
      "gene: 1527 -> 68\n",
      "gene: 1529 -> 120\n",
      "gene: 153 -> 120\n",
      "gene: 1530 -> 120\n",
      "gene: 1533 -> 30\n",
      "gene: 1534 -> 120\n",
      "gene: 1535 -> 120\n",
      "gene: 1536 -> 120\n",
      "gene: 1537 -> 120\n",
      "gene: 1538 -> 120\n",
      "gene: 154 -> 120\n",
      "gene: 1541 -> 120\n",
      "gene: 1542 -> 120\n",
      "gene: 1548 -> 102\n",
      "gene: 155 -> 120\n",
      "gene: 156 -> 120\n",
      "gene: 157 -> 120\n",
      "gene: 158 -> 120\n",
      "gene: 159 -> 120\n",
      "gene: 16 -> 120\n",
      "gene: 160 -> 120\n",
      "gene: 161 -> 120\n",
      "gene: 162 -> 120\n",
      "gene: 163 -> 120\n",
      "gene: 164 -> 120\n",
      "gene: 165 -> 120\n",
      "gene: 166 -> 120\n",
      "gene: 167 -> 120\n",
      "gene: 168 -> 120\n",
      "gene: 169 -> 120\n",
      "gene: 17 -> 120\n",
      "gene: 170 -> 120\n",
      "gene: 171 -> 120\n",
      "gene: 172 -> 120\n",
      "gene: 173 -> 120\n",
      "gene: 174 -> 120\n",
      "gene: 175 -> 120\n",
      "gene: 176 -> 120\n",
      "gene: 177 -> 120\n",
      "gene: 178 -> 120\n",
      "gene: 179 -> 120\n",
      "gene: 18 -> 120\n",
      "gene: 180 -> 120\n",
      "gene: 181 -> 120\n",
      "gene: 182 -> 81\n",
      "gene: 183 -> 120\n",
      "gene: 184 -> 120\n",
      "gene: 185 -> 120\n",
      "gene: 186 -> 120\n",
      "gene: 187 -> 120\n",
      "gene: 188 -> 120\n",
      "gene: 189 -> 120\n",
      "gene: 19 -> 120\n",
      "gene: 190 -> 120\n",
      "gene: 191 -> 120\n",
      "gene: 192 -> 120\n",
      "gene: 193 -> 120\n",
      "gene: 194 -> 120\n",
      "gene: 195 -> 120\n",
      "gene: 196 -> 120\n",
      "gene: 197 -> 120\n",
      "gene: 198 -> 120\n",
      "gene: 199 -> 120\n",
      "gene: 2 -> 120\n",
      "gene: 20 -> 120\n",
      "gene: 200 -> 120\n",
      "gene: 201 -> 120\n",
      "gene: 202 -> 120\n",
      "gene: 203 -> 120\n",
      "gene: 204 -> 120\n",
      "gene: 205 -> 120\n",
      "gene: 206 -> 120\n",
      "gene: 207 -> 120\n",
      "gene: 208 -> 120\n",
      "gene: 209 -> 120\n",
      "gene: 21 -> 120\n",
      "gene: 210 -> 120\n",
      "gene: 211 -> 120\n",
      "gene: 212 -> 120\n",
      "gene: 213 -> 120\n",
      "gene: 214 -> 120\n",
      "gene: 215 -> 120\n",
      "gene: 216 -> 120\n",
      "gene: 217 -> 120\n",
      "gene: 218 -> 120\n",
      "gene: 219 -> 120\n",
      "gene: 22 -> 120\n",
      "gene: 220 -> 120\n",
      "gene: 221 -> 120\n",
      "gene: 222 -> 120\n",
      "gene: 223 -> 120\n",
      "gene: 224 -> 120\n",
      "gene: 225 -> 120\n",
      "gene: 226 -> 120\n",
      "gene: 227 -> 120\n",
      "gene: 228 -> 120\n",
      "gene: 229 -> 120\n",
      "gene: 23 -> 120\n",
      "gene: 230 -> 120\n",
      "gene: 233 -> 120\n",
      "gene: 234 -> 120\n",
      "gene: 235 -> 120\n",
      "gene: 236 -> 120\n",
      "gene: 237 -> 120\n",
      "gene: 238 -> 120\n",
      "gene: 239 -> 120\n",
      "gene: 24 -> 120\n",
      "gene: 240 -> 120\n",
      "gene: 241 -> 120\n",
      "gene: 242 -> 120\n",
      "gene: 243 -> 120\n",
      "gene: 244 -> 120\n",
      "gene: 245 -> 120\n",
      "gene: 246 -> 120\n",
      "gene: 247 -> 120\n",
      "gene: 248 -> 120\n",
      "gene: 249 -> 120\n",
      "gene: 25 -> 120\n",
      "gene: 250 -> 120\n",
      "gene: 251 -> 120\n",
      "gene: 252 -> 120\n",
      "gene: 253 -> 120\n",
      "gene: 254 -> 120\n",
      "gene: 255 -> 120\n",
      "gene: 256 -> 120\n",
      "gene: 257 -> 120\n",
      "gene: 258 -> 120\n",
      "gene: 259 -> 120\n",
      "gene: 26 -> 120\n",
      "gene: 260 -> 120\n",
      "gene: 261 -> 120\n",
      "gene: 262 -> 120\n",
      "gene: 263 -> 120\n",
      "gene: 264 -> 120\n",
      "gene: 265 -> 120\n",
      "gene: 266 -> 120\n",
      "gene: 267 -> 120\n",
      "gene: 268 -> 120\n",
      "gene: 269 -> 120\n",
      "gene: 27 -> 120\n",
      "gene: 270 -> 120\n",
      "gene: 271 -> 120\n",
      "gene: 272 -> 120\n",
      "gene: 273 -> 120\n",
      "gene: 274 -> 120\n",
      "gene: 275 -> 120\n",
      "gene: 276 -> 120\n",
      "gene: 277 -> 120\n",
      "gene: 278 -> 120\n",
      "gene: 279 -> 120\n",
      "gene: 28 -> 120\n",
      "gene: 280 -> 120\n",
      "gene: 281 -> 120\n",
      "gene: 282 -> 120\n",
      "gene: 283 -> 120\n",
      "gene: 284 -> 120\n",
      "gene: 285 -> 120\n",
      "gene: 286 -> 120\n",
      "gene: 287 -> 120\n",
      "gene: 288 -> 120\n",
      "gene: 289 -> 120\n",
      "gene: 29 -> 120\n",
      "gene: 290 -> 120\n",
      "gene: 291 -> 120\n",
      "gene: 292 -> 120\n",
      "gene: 293 -> 120\n",
      "gene: 295 -> 120\n",
      "gene: 296 -> 120\n",
      "gene: 297 -> 120\n",
      "gene: 298 -> 120\n",
      "gene: 299 -> 120\n",
      "gene: 3 -> 120\n",
      "gene: 30 -> 120\n",
      "gene: 300 -> 120\n",
      "gene: 302 -> 120\n",
      "gene: 303 -> 120\n",
      "gene: 304 -> 120\n",
      "gene: 305 -> 120\n",
      "gene: 306 -> 120\n",
      "gene: 307 -> 120\n",
      "gene: 308 -> 120\n",
      "gene: 309 -> 120\n",
      "gene: 31 -> 120\n",
      "gene: 310 -> 120\n",
      "gene: 311 -> 120\n",
      "gene: 312 -> 120\n",
      "gene: 313 -> 120\n",
      "gene: 314 -> 120\n",
      "gene: 315 -> 120\n",
      "gene: 316 -> 120\n",
      "gene: 317 -> 120\n",
      "gene: 318 -> 120\n",
      "gene: 319 -> 120\n",
      "gene: 32 -> 120\n",
      "gene: 320 -> 120\n",
      "gene: 321 -> 120\n",
      "gene: 322 -> 120\n",
      "gene: 323 -> 120\n",
      "gene: 324 -> 120\n",
      "gene: 325 -> 120\n",
      "gene: 326 -> 120\n",
      "gene: 327 -> 91\n",
      "gene: 328 -> 120\n",
      "gene: 329 -> 120\n",
      "gene: 33 -> 120\n",
      "gene: 330 -> 120\n",
      "gene: 331 -> 120\n",
      "gene: 332 -> 120\n",
      "gene: 333 -> 120\n",
      "gene: 334 -> 120\n",
      "gene: 335 -> 120\n",
      "gene: 336 -> 120\n",
      "gene: 337 -> 120\n",
      "gene: 338 -> 120\n",
      "gene: 339 -> 120\n",
      "gene: 34 -> 120\n",
      "gene: 340 -> 120\n",
      "gene: 341 -> 120\n",
      "gene: 342 -> 120\n",
      "gene: 343 -> 120\n",
      "gene: 344 -> 120\n",
      "gene: 345 -> 120\n",
      "gene: 346 -> 120\n",
      "gene: 347 -> 120\n",
      "gene: 348 -> 120\n",
      "gene: 349 -> 120\n",
      "gene: 35 -> 120\n",
      "gene: 350 -> 120\n",
      "gene: 351 -> 120\n",
      "gene: 352 -> 120\n",
      "gene: 353 -> 120\n",
      "gene: 354 -> 120\n",
      "gene: 355 -> 120\n",
      "gene: 356 -> 120\n",
      "gene: 357 -> 120\n",
      "gene: 358 -> 120\n",
      "gene: 359 -> 120\n",
      "gene: 36 -> 120\n",
      "gene: 360 -> 120\n",
      "gene: 361 -> 120\n",
      "gene: 362 -> 120\n",
      "gene: 363 -> 120\n",
      "gene: 364 -> 120\n",
      "gene: 365 -> 120\n",
      "gene: 366 -> 120\n",
      "gene: 367 -> 83\n",
      "gene: 368 -> 30\n",
      "gene: 37 -> 120\n",
      "gene: 370 -> 120\n",
      "gene: 371 -> 74\n",
      "gene: 372 -> 120\n",
      "gene: 373 -> 120\n",
      "gene: 374 -> 120\n",
      "gene: 375 -> 120\n",
      "gene: 376 -> 120\n",
      "gene: 377 -> 120\n",
      "gene: 379 -> 120\n",
      "gene: 38 -> 120\n",
      "gene: 380 -> 120\n",
      "gene: 381 -> 120\n",
      "gene: 382 -> 120\n",
      "gene: 383 -> 120\n",
      "gene: 384 -> 120\n",
      "gene: 385 -> 120\n",
      "gene: 386 -> 120\n",
      "gene: 387 -> 120\n",
      "gene: 388 -> 120\n",
      "gene: 389 -> 120\n",
      "gene: 39 -> 120\n",
      "gene: 390 -> 120\n",
      "gene: 391 -> 120\n",
      "gene: 392 -> 120\n",
      "gene: 393 -> 120\n",
      "gene: 394 -> 120\n",
      "gene: 395 -> 120\n",
      "gene: 396 -> 120\n",
      "gene: 397 -> 120\n",
      "gene: 398 -> 120\n",
      "gene: 399 -> 120\n",
      "gene: 4 -> 120\n",
      "gene: 40 -> 120\n",
      "gene: 400 -> 120\n",
      "gene: 401 -> 120\n",
      "gene: 402 -> 120\n",
      "gene: 403 -> 120\n",
      "gene: 404 -> 120\n",
      "gene: 405 -> 120\n",
      "gene: 406 -> 120\n",
      "gene: 407 -> 120\n",
      "gene: 408 -> 120\n",
      "gene: 409 -> 120\n",
      "gene: 410 -> 120\n",
      "gene: 411 -> 120\n",
      "gene: 412 -> 120\n",
      "gene: 413 -> 120\n",
      "gene: 414 -> 120\n",
      "gene: 415 -> 120\n",
      "gene: 416 -> 120\n",
      "gene: 417 -> 120\n",
      "gene: 418 -> 120\n",
      "gene: 419 -> 120\n",
      "gene: 42 -> 120\n",
      "gene: 421 -> 120\n",
      "gene: 422 -> 120\n",
      "gene: 423 -> 120\n",
      "gene: 424 -> 120\n",
      "gene: 425 -> 120\n",
      "gene: 426 -> 120\n",
      "gene: 427 -> 120\n",
      "gene: 428 -> 120\n",
      "gene: 429 -> 120\n",
      "gene: 43 -> 120\n",
      "gene: 430 -> 120\n",
      "gene: 431 -> 120\n",
      "gene: 432 -> 120\n",
      "gene: 435 -> 120\n",
      "gene: 436 -> 120\n",
      "gene: 437 -> 120\n",
      "gene: 438 -> 120\n",
      "gene: 439 -> 120\n",
      "gene: 44 -> 120\n",
      "gene: 440 -> 120\n",
      "gene: 441 -> 120\n",
      "gene: 442 -> 120\n",
      "gene: 443 -> 120\n",
      "gene: 444 -> 120\n",
      "gene: 446 -> 90\n",
      "gene: 447 -> 120\n",
      "gene: 448 -> 112\n",
      "gene: 449 -> 120\n",
      "gene: 45 -> 120\n",
      "gene: 450 -> 120\n",
      "gene: 451 -> 120\n",
      "gene: 452 -> 120\n",
      "gene: 453 -> 48\n",
      "gene: 454 -> 120\n",
      "gene: 455 -> 120\n",
      "gene: 456 -> 120\n",
      "gene: 457 -> 120\n",
      "gene: 458 -> 120\n",
      "gene: 459 -> 120\n",
      "gene: 46 -> 120\n",
      "gene: 460 -> 120\n",
      "gene: 461 -> 120\n",
      "gene: 462 -> 120\n",
      "gene: 463 -> 120\n",
      "gene: 464 -> 120\n",
      "gene: 465 -> 120\n",
      "gene: 466 -> 120\n",
      "gene: 467 -> 120\n",
      "gene: 468 -> 120\n",
      "gene: 469 -> 120\n",
      "gene: 47 -> 120\n",
      "gene: 470 -> 114\n",
      "gene: 471 -> 120\n",
      "gene: 472 -> 120\n",
      "gene: 473 -> 120\n",
      "gene: 474 -> 120\n",
      "gene: 475 -> 120\n",
      "gene: 476 -> 120\n",
      "gene: 477 -> 120\n",
      "gene: 478 -> 120\n",
      "gene: 479 -> 120\n",
      "gene: 480 -> 120\n",
      "gene: 481 -> 120\n",
      "gene: 482 -> 120\n",
      "gene: 483 -> 120\n",
      "gene: 484 -> 120\n",
      "gene: 485 -> 120\n",
      "gene: 486 -> 120\n",
      "gene: 487 -> 120\n",
      "gene: 489 -> 120\n",
      "gene: 490 -> 120\n",
      "gene: 491 -> 120\n",
      "gene: 492 -> 120\n",
      "gene: 493 -> 120\n",
      "gene: 494 -> 120\n",
      "gene: 495 -> 120\n",
      "gene: 496 -> 120\n",
      "gene: 497 -> 120\n",
      "gene: 498 -> 120\n",
      "gene: 499 -> 120\n",
      "gene: 5 -> 120\n",
      "gene: 500 -> 120\n",
      "gene: 501 -> 120\n",
      "gene: 502 -> 120\n",
      "gene: 503 -> 120\n",
      "gene: 504 -> 120\n",
      "gene: 505 -> 120\n",
      "gene: 506 -> 120\n",
      "gene: 507 -> 120\n",
      "gene: 508 -> 120\n",
      "gene: 509 -> 120\n",
      "gene: 510 -> 120\n",
      "gene: 511 -> 120\n",
      "gene: 512 -> 120\n",
      "gene: 516 -> 120\n",
      "gene: 517 -> 120\n",
      "gene: 518 -> 120\n",
      "gene: 519 -> 120\n",
      "gene: 520 -> 120\n",
      "gene: 521 -> 120\n",
      "gene: 522 -> 120\n",
      "gene: 523 -> 120\n",
      "gene: 524 -> 120\n",
      "gene: 525 -> 120\n",
      "gene: 526 -> 120\n",
      "gene: 527 -> 120\n",
      "gene: 528 -> 120\n",
      "gene: 529 -> 120\n",
      "gene: 530 -> 120\n",
      "gene: 531 -> 120\n",
      "gene: 532 -> 120\n",
      "gene: 533 -> 120\n",
      "gene: 534 -> 120\n",
      "gene: 535 -> 112\n",
      "gene: 536 -> 120\n",
      "gene: 537 -> 120\n",
      "gene: 538 -> 120\n",
      "gene: 539 -> 120\n",
      "gene: 540 -> 120\n",
      "gene: 541 -> 120\n",
      "gene: 542 -> 120\n",
      "gene: 543 -> 120\n",
      "gene: 544 -> 120\n",
      "gene: 545 -> 120\n",
      "gene: 546 -> 120\n",
      "gene: 548 -> 120\n",
      "gene: 549 -> 120\n",
      "gene: 550 -> 120\n",
      "gene: 551 -> 88\n",
      "gene: 552 -> 120\n",
      "gene: 554 -> 120\n",
      "gene: 555 -> 120\n",
      "gene: 556 -> 120\n",
      "gene: 557 -> 120\n",
      "gene: 558 -> 120\n",
      "gene: 559 -> 120\n",
      "gene: 560 -> 120\n",
      "gene: 561 -> 120\n",
      "gene: 562 -> 120\n",
      "gene: 563 -> 120\n",
      "gene: 564 -> 120\n",
      "gene: 565 -> 120\n",
      "gene: 566 -> 120\n",
      "gene: 567 -> 120\n",
      "gene: 568 -> 120\n",
      "gene: 569 -> 120\n",
      "gene: 57 -> 59\n",
      "gene: 570 -> 46\n",
      "gene: 571 -> 120\n",
      "gene: 572 -> 120\n",
      "gene: 573 -> 120\n",
      "gene: 574 -> 120\n",
      "gene: 575 -> 120\n",
      "gene: 576 -> 120\n",
      "gene: 577 -> 120\n",
      "gene: 578 -> 120\n",
      "gene: 579 -> 120\n",
      "gene: 580 -> 120\n",
      "gene: 581 -> 120\n",
      "gene: 582 -> 120\n",
      "gene: 583 -> 120\n",
      "gene: 584 -> 120\n",
      "gene: 585 -> 120\n",
      "gene: 586 -> 120\n",
      "gene: 587 -> 120\n",
      "gene: 588 -> 120\n",
      "gene: 589 -> 120\n",
      "gene: 590 -> 120\n",
      "gene: 591 -> 120\n",
      "gene: 592 -> 120\n",
      "gene: 593 -> 120\n",
      "gene: 594 -> 120\n",
      "gene: 595 -> 120\n",
      "gene: 596 -> 120\n",
      "gene: 597 -> 120\n",
      "gene: 598 -> 120\n",
      "gene: 599 -> 120\n",
      "gene: 6 -> 120\n",
      "gene: 60 -> 120\n",
      "gene: 600 -> 120\n",
      "gene: 601 -> 120\n",
      "gene: 602 -> 120\n",
      "gene: 603 -> 120\n",
      "gene: 605 -> 120\n",
      "gene: 606 -> 120\n",
      "gene: 607 -> 120\n",
      "gene: 608 -> 120\n",
      "gene: 609 -> 120\n",
      "gene: 611 -> 120\n",
      "gene: 612 -> 120\n",
      "gene: 613 -> 120\n",
      "gene: 614 -> 120\n",
      "gene: 615 -> 120\n",
      "gene: 616 -> 120\n",
      "gene: 617 -> 120\n",
      "gene: 618 -> 120\n",
      "gene: 619 -> 120\n",
      "gene: 62 -> 120\n",
      "gene: 620 -> 120\n",
      "gene: 621 -> 120\n",
      "gene: 622 -> 120\n",
      "gene: 623 -> 120\n",
      "gene: 624 -> 120\n",
      "gene: 625 -> 120\n",
      "gene: 626 -> 103\n",
      "gene: 627 -> 120\n",
      "gene: 628 -> 120\n",
      "gene: 629 -> 120\n",
      "gene: 63 -> 120\n",
      "gene: 64 -> 120\n",
      "gene: 645 -> 120\n",
      "gene: 646 -> 120\n",
      "gene: 647 -> 120\n",
      "gene: 648 -> 120\n",
      "gene: 649 -> 120\n",
      "gene: 65 -> 101\n",
      "gene: 650 -> 120\n",
      "gene: 651 -> 120\n",
      "gene: 652 -> 120\n",
      "gene: 653 -> 120\n",
      "gene: 654 -> 120\n",
      "gene: 655 -> 120\n",
      "gene: 656 -> 120\n",
      "gene: 657 -> 120\n",
      "gene: 658 -> 120\n",
      "gene: 659 -> 48\n",
      "gene: 66 -> 120\n",
      "gene: 660 -> 120\n",
      "gene: 662 -> 120\n",
      "gene: 663 -> 120\n",
      "gene: 664 -> 120\n",
      "gene: 665 -> 120\n",
      "gene: 666 -> 120\n",
      "gene: 667 -> 120\n",
      "gene: 668 -> 120\n",
      "gene: 669 -> 120\n",
      "gene: 67 -> 120\n",
      "gene: 670 -> 120\n",
      "gene: 671 -> 120\n",
      "gene: 672 -> 120\n",
      "gene: 673 -> 116\n",
      "gene: 674 -> 120\n",
      "gene: 675 -> 120\n",
      "gene: 676 -> 120\n",
      "gene: 677 -> 120\n",
      "gene: 678 -> 120\n",
      "gene: 679 -> 120\n",
      "gene: 68 -> 120\n",
      "gene: 680 -> 120\n",
      "gene: 681 -> 120\n",
      "gene: 682 -> 120\n",
      "gene: 683 -> 120\n",
      "gene: 684 -> 120\n",
      "gene: 685 -> 120\n",
      "gene: 686 -> 120\n",
      "gene: 687 -> 43\n",
      "gene: 688 -> 120\n",
      "gene: 689 -> 120\n",
      "gene: 69 -> 118\n",
      "gene: 690 -> 120\n",
      "gene: 691 -> 120\n",
      "gene: 692 -> 120\n",
      "gene: 693 -> 120\n",
      "gene: 694 -> 120\n",
      "gene: 695 -> 120\n",
      "gene: 696 -> 120\n",
      "gene: 697 -> 120\n",
      "gene: 698 -> 120\n",
      "gene: 699 -> 47\n",
      "gene: 7 -> 120\n",
      "gene: 70 -> 111\n",
      "gene: 701 -> 120\n",
      "gene: 702 -> 120\n",
      "gene: 703 -> 120\n",
      "gene: 704 -> 120\n",
      "gene: 705 -> 120\n",
      "gene: 706 -> 120\n",
      "gene: 707 -> 120\n",
      "gene: 708 -> 120\n",
      "gene: 709 -> 120\n",
      "gene: 71 -> 120\n",
      "gene: 710 -> 120\n",
      "gene: 711 -> 120\n",
      "gene: 712 -> 120\n",
      "gene: 713 -> 120\n",
      "gene: 714 -> 120\n",
      "gene: 715 -> 120\n",
      "gene: 716 -> 120\n",
      "gene: 717 -> 120\n",
      "gene: 718 -> 120\n",
      "gene: 719 -> 120\n",
      "gene: 72 -> 120\n",
      "gene: 720 -> 120\n",
      "gene: 721 -> 120\n",
      "gene: 722 -> 120\n",
      "gene: 723 -> 68\n",
      "gene: 724 -> 120\n",
      "gene: 725 -> 120\n",
      "gene: 726 -> 120\n",
      "gene: 727 -> 120\n",
      "gene: 728 -> 120\n",
      "gene: 729 -> 120\n",
      "gene: 73 -> 120\n",
      "gene: 730 -> 120\n",
      "gene: 731 -> 120\n",
      "gene: 732 -> 120\n",
      "gene: 733 -> 120\n",
      "gene: 734 -> 120\n",
      "gene: 735 -> 120\n",
      "gene: 736 -> 120\n",
      "gene: 737 -> 120\n",
      "gene: 738 -> 120\n",
      "gene: 739 -> 120\n",
      "gene: 74 -> 120\n",
      "gene: 740 -> 120\n",
      "gene: 741 -> 120\n",
      "gene: 742 -> 120\n",
      "gene: 743 -> 120\n",
      "gene: 744 -> 120\n",
      "gene: 745 -> 120\n",
      "gene: 748 -> 120\n",
      "gene: 749 -> 120\n",
      "gene: 75 -> 120\n",
      "gene: 750 -> 120\n",
      "gene: 751 -> 120\n",
      "gene: 752 -> 120\n",
      "gene: 753 -> 120\n",
      "gene: 754 -> 120\n",
      "gene: 755 -> 36\n",
      "gene: 756 -> 120\n",
      "gene: 757 -> 120\n",
      "gene: 758 -> 120\n",
      "gene: 759 -> 120\n",
      "gene: 76 -> 86\n",
      "gene: 760 -> 120\n",
      "gene: 761 -> 120\n",
      "gene: 762 -> 120\n",
      "gene: 763 -> 120\n",
      "gene: 764 -> 120\n",
      "gene: 765 -> 120\n",
      "gene: 766 -> 120\n",
      "gene: 767 -> 120\n",
      "gene: 768 -> 120\n",
      "gene: 769 -> 120\n",
      "gene: 77 -> 120\n",
      "gene: 770 -> 79\n",
      "gene: 774 -> 120\n",
      "gene: 775 -> 120\n",
      "gene: 776 -> 120\n",
      "gene: 777 -> 120\n",
      "gene: 778 -> 120\n",
      "gene: 779 -> 120\n",
      "gene: 78 -> 120\n",
      "gene: 780 -> 120\n",
      "gene: 782 -> 120\n",
      "gene: 783 -> 120\n",
      "gene: 784 -> 120\n",
      "gene: 785 -> 120\n",
      "gene: 786 -> 120\n",
      "gene: 787 -> 120\n",
      "gene: 788 -> 120\n",
      "gene: 789 -> 120\n",
      "gene: 79 -> 120\n",
      "gene: 790 -> 87\n",
      "gene: 791 -> 120\n",
      "gene: 792 -> 120\n",
      "gene: 793 -> 120\n",
      "gene: 794 -> 120\n",
      "gene: 795 -> 114\n",
      "gene: 796 -> 120\n",
      "gene: 797 -> 120\n",
      "gene: 798 -> 120\n",
      "gene: 799 -> 120\n",
      "gene: 8 -> 120\n",
      "gene: 80 -> 120\n",
      "gene: 800 -> 120\n",
      "gene: 801 -> 120\n",
      "gene: 802 -> 120\n",
      "gene: 803 -> 120\n",
      "gene: 804 -> 120\n",
      "gene: 805 -> 120\n",
      "gene: 806 -> 120\n",
      "gene: 807 -> 120\n",
      "gene: 808 -> 120\n",
      "gene: 809 -> 120\n",
      "gene: 81 -> 120\n",
      "gene: 810 -> 120\n",
      "gene: 811 -> 120\n",
      "gene: 812 -> 120\n",
      "gene: 813 -> 120\n",
      "gene: 814 -> 120\n",
      "gene: 815 -> 120\n",
      "gene: 816 -> 120\n",
      "gene: 817 -> 120\n",
      "gene: 819 -> 120\n",
      "gene: 820 -> 120\n",
      "gene: 821 -> 120\n",
      "gene: 822 -> 120\n",
      "gene: 823 -> 107\n",
      "gene: 824 -> 120\n",
      "gene: 825 -> 120\n",
      "gene: 826 -> 120\n",
      "gene: 827 -> 120\n",
      "gene: 828 -> 120\n",
      "gene: 829 -> 120\n",
      "gene: 83 -> 120\n",
      "gene: 832 -> 120\n",
      "gene: 833 -> 120\n",
      "gene: 834 -> 120\n",
      "gene: 835 -> 120\n",
      "gene: 836 -> 120\n",
      "gene: 837 -> 120\n",
      "gene: 838 -> 120\n",
      "gene: 839 -> 120\n",
      "gene: 84 -> 120\n",
      "gene: 840 -> 120\n",
      "gene: 841 -> 120\n",
      "gene: 843 -> 120\n",
      "gene: 844 -> 120\n",
      "gene: 845 -> 120\n",
      "gene: 846 -> 120\n",
      "gene: 847 -> 120\n",
      "gene: 848 -> 120\n",
      "gene: 849 -> 109\n",
      "gene: 85 -> 120\n",
      "gene: 850 -> 120\n",
      "gene: 851 -> 120\n",
      "gene: 852 -> 120\n",
      "gene: 853 -> 120\n",
      "gene: 854 -> 120\n",
      "gene: 855 -> 120\n",
      "gene: 856 -> 120\n",
      "gene: 857 -> 120\n",
      "gene: 858 -> 120\n",
      "gene: 859 -> 120\n",
      "gene: 86 -> 120\n",
      "gene: 860 -> 120\n",
      "gene: 861 -> 120\n",
      "gene: 862 -> 120\n",
      "gene: 863 -> 120\n",
      "gene: 864 -> 120\n",
      "gene: 865 -> 120\n",
      "gene: 866 -> 120\n",
      "gene: 867 -> 120\n",
      "gene: 868 -> 120\n",
      "gene: 869 -> 120\n",
      "gene: 87 -> 120\n",
      "gene: 870 -> 120\n",
      "gene: 871 -> 120\n",
      "gene: 872 -> 120\n",
      "gene: 873 -> 120\n",
      "gene: 874 -> 120\n",
      "gene: 875 -> 120\n",
      "gene: 876 -> 120\n",
      "gene: 877 -> 120\n",
      "gene: 878 -> 120\n",
      "gene: 88 -> 120\n",
      "gene: 888 -> 120\n",
      "gene: 889 -> 120\n",
      "gene: 89 -> 120\n",
      "gene: 890 -> 120\n",
      "gene: 891 -> 120\n",
      "gene: 892 -> 120\n",
      "gene: 893 -> 120\n",
      "gene: 894 -> 120\n",
      "gene: 895 -> 120\n",
      "gene: 896 -> 120\n",
      "gene: 897 -> 120\n",
      "gene: 898 -> 120\n",
      "gene: 899 -> 120\n",
      "gene: 9 -> 120\n",
      "gene: 90 -> 110\n",
      "gene: 900 -> 120\n",
      "gene: 901 -> 120\n",
      "gene: 902 -> 120\n",
      "gene: 903 -> 120\n",
      "gene: 904 -> 120\n",
      "gene: 905 -> 120\n",
      "gene: 906 -> 120\n",
      "gene: 907 -> 120\n",
      "gene: 908 -> 120\n",
      "gene: 909 -> 120\n",
      "gene: 91 -> 120\n",
      "gene: 910 -> 120\n",
      "gene: 911 -> 120\n",
      "gene: 912 -> 120\n",
      "gene: 913 -> 120\n",
      "gene: 914 -> 120\n",
      "gene: 915 -> 120\n",
      "gene: 916 -> 120\n",
      "gene: 917 -> 120\n",
      "gene: 918 -> 120\n",
      "gene: 919 -> 120\n",
      "gene: 92 -> 120\n",
      "gene: 920 -> 120\n",
      "gene: 921 -> 120\n",
      "gene: 922 -> 120\n",
      "gene: 923 -> 120\n",
      "gene: 924 -> 120\n",
      "gene: 925 -> 120\n",
      "gene: 926 -> 120\n",
      "gene: 927 -> 120\n",
      "gene: 928 -> 120\n",
      "gene: 929 -> 120\n",
      "gene: 93 -> 120\n",
      "gene: 930 -> 120\n",
      "gene: 931 -> 120\n",
      "gene: 932 -> 120\n",
      "gene: 933 -> 120\n",
      "gene: 934 -> 120\n",
      "gene: 935 -> 120\n",
      "gene: 94 -> 120\n",
      "gene: 949 -> 120\n",
      "gene: 95 -> 120\n",
      "gene: 950 -> 120\n",
      "gene: 951 -> 120\n",
      "gene: 952 -> 120\n",
      "gene: 953 -> 120\n",
      "gene: 954 -> 81\n",
      "gene: 955 -> 120\n",
      "gene: 956 -> 120\n",
      "gene: 957 -> 120\n",
      "gene: 958 -> 120\n",
      "gene: 959 -> 120\n",
      "gene: 96 -> 120\n",
      "gene: 960 -> 120\n",
      "gene: 961 -> 120\n",
      "gene: 962 -> 120\n",
      "gene: 963 -> 120\n",
      "gene: 964 -> 120\n",
      "gene: 965 -> 120\n",
      "gene: 966 -> 120\n",
      "gene: 967 -> 120\n",
      "gene: 968 -> 120\n",
      "gene: 969 -> 120\n",
      "gene: 970 -> 80\n",
      "gene: 971 -> 120\n",
      "gene: 972 -> 120\n",
      "gene: 973 -> 120\n",
      "gene: 974 -> 120\n",
      "gene: 975 -> 120\n",
      "gene: 976 -> 120\n",
      "gene: 977 -> 120\n",
      "gene: 978 -> 120\n",
      "gene: 979 -> 120\n",
      "gene: 98 -> 120\n",
      "gene: 980 -> 120\n",
      "gene: 981 -> 120\n",
      "gene: 982 -> 120\n",
      "gene: 983 -> 120\n",
      "gene: 984 -> 120\n",
      "gene: 985 -> 120\n",
      "gene: 986 -> 120\n",
      "gene: 987 -> 120\n",
      "gene: 988 -> 120\n",
      "gene: 989 -> 120\n",
      "gene: 990 -> 120\n",
      "gene: 991 -> 120\n",
      "gene: 992 -> 120\n",
      "gene: 993 -> 120\n",
      "gene: 994 -> 120\n",
      "gene: 995 -> 120\n",
      "gene: 996 -> 120\n",
      "gene: 997 -> 120\n",
      "gene: 999 -> 120\n",
      "False\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "# get region size dict\n",
    "min_num_probes = 50\n",
    "reg_size_dic, len_check = check._check_region_size(blasted_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 162,
   "id": "a330f8a4",
   "metadata": {},
   "outputs": [],
   "source": [
    "plt.style.use('default')\n",
    "#plt.style.use('dark_background')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 163,
   "id": "e4fb5b25",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAApQAAAFKCAYAAACjE8/+AAAAAXNSR0IArs4c6QAAIABJREFUeF7snQd0VEUXx//SOwjSe++99xZCEaUpFpSiiHQFG703AREQUMSOn1IEROkgTXqV3iEQWkJNAiGQwnfmbXYzW97uK7Ozu+HOOR41OzNv3n/mvfm9O3fuPPf06dOnoEQKkAKkAClACpACpAApQAoYVOA5AkqDylExUoAUIAVIAVKAFCAFSAFFAQJKGgikAClACpACpAApQAqQAqYUIKA0JR8VJgVIAVKAFCAFSAFSgBQgoKQxQAqQAqQAKUAKkAKkAClgSgECSlPyUWFSgBQgBUgBUoAUIAVIAQJKGgOkAClACpACpAApQAqQAqYUIKA0JR8VJgVIAVKAFCAFSAFSgBQgoKQxQAqQAqQAKUAKkAKkAClgSgECSlPyUWFSgBQgBUgBUoAUIAVIAQJKGgOkAClACpACpAApQAqQAqYUIKA0JR8VJgVIAVKAFCAFSAFSgBQgoKQxQAqQAqQAKUAKkAKkAClgSgECSlPyUWFSgBQgBUgBUoAUIAVIAQJKGgOkAClACpACpAApQAqQAqYUIKA0JR8VJgVIAVKAFCAFSAFSgBQgoKQxQAqQAqQAKUAKkAKkAClgSgECSlPyUWFSgBQgBUgBUoAUIAVIAQJKGgOkAClACpACpAApQAqQAqYUIKA0JR8VJgVIAVKAFCAFSAFSgBQgoKQxQAqQAqQAKUAKkAKkAClgSgECSlPyUWFSgBQgBUgBUoAUIAVIAQJKGgOkAClACpACpAApQAqQAqYUIKA0JR8VJgVIAVKAFCAFSAFSgBQgoKQxQAqQAqQAKUAKkAKkAClgSgECSlPyUWFSgBQgBUgBUoAUIAVIAQJKGgOkAClACpACpAApQAqQAqYUIKA0JR8VJgVIAVKAFCAFSAFSgBQgoKQxQAqQAqQAKUAKkAKkAClgSgECSlPyUWFSgBQgBUgBUoAUIAVIAQJKGgOkAClACpACpAApQAqQAqYUIKA0JR8VJgVIAVKAFCAFSAFSgBQgoKQxQAqQAqQAKUAKkAKkAClgSgECSlPyUWFSgBQgBUgBUoAUIAVIAQJKGgOkAClACpACpAApQAqQAqYUIKA0JR8VJgVIAVKAFCAFSAFSgBQgoKQxQAqQAqQAKUAKkAKkAClgSgECSlPyUWFSgBQgBUgBUoAUIAVIAQJKGgOkAClACpACpAApQAqQAqYUIKA0JV9S4ZiYGBw7dgw5c+ZEqlSpBNVK1ZACpAApQAqQAqSAvyoQFxeHW7duoWLFikiXLp2/NlNKuwgoBcm8f/9+1KpVS1BtVA0pQAqQAqQAKUAKBIoC+/btQ82aNQOluV5pJwGlIFlDQkJQtGhRsEGVN29eQbVSNaQAKUAKkAKkACngrwrcuHFDMSZdunQJRYoU8ddmSmkXAaUgma9evYqCBQsiNDQUBQoUEFQrVUMKkAKkAClACpAC/qoAzf1JPUNAKWiU0qASJCRVQwqQAqQAKUAKBIgCNPcTUAofqjSohEtKFZICpAApQAqQAn6tAM39BJTCBygNKuGSUoWkAClACpACpIBfK0BzPwGl8AFKg0q4pFQhKUAKkAKkACng1wrQ3E9AKXyA0qASLilVSAqQAqQAKUAK+LUCNPcTUAofoDSohEtKFZICpAApQAqQAn6tAM39BJTCBygNKuGSUoWkAClACpACpIBfK0BzPwGl8AFKg0q4pFQhKUAKkAKkACng1wrQ3E9AKXyA0qASLilVSAqQAqQAKUAK+LUCNPcTUAofoDSohEtKFZICpAApQAqQAn6tAM39BJTCBygNKuGSUoWkAClACpACpIBfK0BzPwGl8AFKg0q4pFQhKUAKkAKkACng1wrQ3E9AKXyA0qASLilVSAqQAqQAKUAK+LUCNPcTUAofoDSohEtKFZICpAApQAqQAn6tAM39AQyU58+fx/Tp07Fnzx4cP34cZcqUUf5tTfHx8fjiiy+wevVqnDx5EnFxcahYsSJGjx6N5s2bOw1MVtecOXNw8+ZNJd+0adPQpEkT3QOYBpVuyagAKUAKkAKkACkQ0ArQ3B/AQLly5Ur0798ftWvXxtmzZ5GQkGAHlA8ePECBAgXQrVs3tGjRAqlTp8ZPP/2ExYsX46+//kLbtm1td89gctiwYZg0aRKqVauGBQsW4M8//8S+ffsUuNSTaFDpUYvykgKkAClACpACga8Azf0BDJQMIFOkSKHcQffu3XHgwAEnC2VkZCSef/55210+ffoUNWrUQJYsWbBlyxbl748fP0bu3LnRq1cvTJ06Vfkbs24ykKxUqRIWLVqka6TToNIlF2UmBUgBUoAUIAUCXgGa+wMYKPnR5woo1Ubnu+++ix07duDMmTNKFgaWzZo1w6FDh1C1alVbsbFjxypL5hEREXjuuec0D3YaVJqlooykAClACpACpECyUIDm/mcMKJlVs3z58ihdurSypM3SvHnz0K9fP0RHRyN9+vQ2RZYuXYrOnTsjNDRUWTrXmrw1qE7fjMSOc7ftmhEe9RjpUqdElnSpbH+PS3iKq/eikSZlSjyfITXSp0mpqelPnwIXbz9EsRcyQgc/e6z7RkQMrtyNRsHnM+BJfLzy75Qp7AGdXZu/ZkxsPG4/eIICzyf1B7vQtfuPkD1DGs335LFxOupMePoUIXeiFX3Mptj4p7h+/xEK58igWtXjuASERcagUHb1PLeiHiNNqhTImj61aj0Xbqn3aeSjWLDr5MycVrlWxrSpkCrFczgb9gCZEscUqztHxjQutb90+6HSPjYGc2dJhxPXI8Du7YVMafDgcRwePo5T/u4p3YyIQeZ0qZExrbax6qk+f/mdaZohTSpk5p5PPW1jWl67H40iOcyPOU/XZe8S9qxWzJ8FV+89Qt6s6ZE6pf1z+vBxPCJjYpE3q+c+9XQ9/vcLtx6g2AuZ7N4B7J3Bxk7aVJZVKD45vi/0XMsxLxt7bKxnSmt5h16+E438z6dXnoOIR7G28Wwt9+hJPO48tH83xSc8Vd5x7B1RPGcmJSvTk9VdIX8WpFB5obJn5EFMHPJo1JO9n/NmSYe70U+QP5vl3cje99fuuX6XsOeT9RV7TxR08x6x3lvInYco8HwG5d5dpegn8bgfHYt82Yz3Pxtb7P3A3hlGkto7T8/cwJ7JN2sXMnJ5t2W8NfcLb6iECp97ytaDAzRptVDOmjULgwYNwtatW9GoUSPlbidOnIjx48cjJibG7u43bdqk+F4eOXJEWfpWS2xZnf1jTTdu3ECtWrV0g6gn6X/fdwVDlx/zlI1+JwVIAVKAFCAFSAEVBXJnSYu9w4KE60NAmSRpsgfKbdu2ITg4GB988IHNV9IKlBMmTMCjR4/sBtjGjRuV/EePHnW7MWfMmDFgy+OOSa9l09PoXnX0OmZsOGvLxr7Uoh7HKf//Qqa0ipWSfa2yL2U+sS/U9Bq+BtnXrzWJsMKxum5GxoB91TomZplL+dxzyhc9s/pZE2sr+za+HpEE99a28O3Tek+eNOXrZHmZlSJX5rROxRzbaUaf2IQEhN5NGmuu6tKS59aDx4iKsfZ/GmRJ52yldNenkTFxuP3gsSeJnH63au+oHZ+RWTjYWHSnqTU/s+I9TBwjTHurpUh3w/ysALNQMQsUS8wCnDnRAqalmfein+BedKxdVjNjztM1+XcJn7dg9vRIneinzp5j9jyzlCFNSuTRYHn2dF32u6sxyv+NWeKsVkq+neydZtZSyvcRG3vs/62JWeaZhZIl9m5l71hmzWeWMP496eqdy8awte9Z3qJs1cdBDPa79XoZ01gs/O6Sq+eN9Y/au8RVfndjiM/PVh0crZSPYuMVCzZLbIWpsAaLp+P98NfIlzWdbiul2jtP79yQI1MaLO1dT8vw1JWHgDJJrmQNlAwKmUWyZcuWyiYb3ifSuuTNgDJduqSHWuuStywLpePIHrLsKBbtD1X+PKNzZXSsVkBZ2qg5cZNd1qW966JmkeweH4wiQ1bb8oRMedFjfi0ZPlpyBMsOXXXKenhkCzyfMQ3Y8lHZUetsv6/oWw+MQzp9vUv5W7m8WbDmg4bKf3f+Zjf2hdxV/vv39+qgbvEcWprgNs8rX+/Cgcv3bHk6VSuALzpXdirDXv4VRq8Xog+bkOpP2ey2rnsPn6Dq+I1KHua2cHhUsFObxvx1Aj/tClH+PrFDBXSpXdgpj7s+XXIgFJ/+cVS3hsv61EP1ws+Dr9uxEjYhWT9sOlbLjxmdq6hep99vh7D66A3l92/fro7g8nl0t8kfC3yy9AiWHrSM/dlvVMXLlfNpbubcLecxbb3Fx9uaRD2TrhoxbMUx/Lb3itNP/37a1LZUuvP8bXT5bq+Sh90LuycRydUYrTx2gw3mNg5qhJK5MyuX+uyPo1h8wPLOa1o6J37sUctUEwYv/g/LD19T6pj/dnW8v/Cgrb7+TUtgzpbzyv+/26AoRrYth1M3ItF61r/K32oVzY4l79dFeFQMak38x64dr1YvYOt79sP5ia2RKqX90v3aYzfQ53+HlHLtquTDrNfd6+n4vFUrlA1z3qyGeirvEsf8DD7//bSZql58/kMjWyB7xjR2efdcvIPXv92j/K1hyRew8N3aurVvNXM7Tt+MUsr93b8BKhbIqqsO/p03qUNF27J15/m7se+S2LlBV8MSMxNQJqmWbIHywoULaNCgAcqWLYt169YhTRr7ByVQN+W4Aso7Dx6j+gR7oLQCgKcHxBdAyXwly4xMAsq1HzQE+xLuOM89UC7qVQd1iskDSuYLWF4QUN6IeIS6k90DZVx8AkoMX6t0mRpQjv37BH7caQFK/uXK97O7Pv3j4FV8vPSIp2Hh9Ptf/eujUoFsboGSWWSY/xZLr1QvgOmvOkO6teL+vx3CqkSgXNC1BlqUy627Tf5YgIefOW9WRdtK/guUI/48hl/3uAfKAyF38co3uxWpZQLlpsGNUSKXxS9RNFCy8c+eA5a+61oDPX85YBtKA5uXxOx/zin/bwXKy3ceovG0rcrf9ADlpcltnDZ2rjt+A71/tQBl+yr5MNMAUM7tUk31XWIGKI+MCkbWDPYrHnz/iwBK9q4vmzeLrkeXf+dN7lgRb9Sy+EHyQClqbtDVMAJKJ7mSJVCyIOX169dXwgSxJW/2b8dkDRvUu3dvTJkyRfmZhQ1ifpMsdJC/hg0auvwoft9n+Vr/8rXK6FC1AHjLlvU+/+xXH1UKZvP4fHgDKPkXNt+A/0a1QLYMafA4Lh6lRyQB5ZaPm+B+9BN08ACUzDLAXuhmk1YLZfSTOJQbJcZCyRz160xOsmioWZ6s/cEsBcxi4JjMAuXyQ1cxeIl+oFw9sAHK58vqFiiL58wIthmIpddqFMTnr6j7IA/4/TD+PnLdNqkHJROg5D/45r5ZDS9Wyqt5uM7beh5T18mzUI788zgW7rns1L4dnzVVNmmwdOxqBF6as0P5b5lA+c9HjW0bXXhNm5XJhR+619SsqauMvBX5h+418M5PSUA5KKgUvtxkcTHq2aAoRrQtZ7cCVKtIdizpXddpVah8vixg/yw5kLQy4+oZFwGU87pUV32XmAHK42NbOrme/Bd6H+3n7lT0EAGUvOVZayeO+/skfth5Sck+pWNFvO4CKBf3qoPaAowNWtvE5yMLZZIaAQeUbFf2mjVrlDuYO3cumCVyxowZyv83btwYmTJlQt26dZW///rrr0qsST7VqVPH9r/WwOaTJ09WApt/9913WL58uV8HNmcbdNhGHZZmvlYF7avmV2CsyjjLUqk1aV1akAmU1i/g2PgElEy0xLH27hnaXPHTsr641Ja8tS7je3opOAKlmjXNcWnezPIj8xmsPck8UPIvV/5rnb9nd3268r9r+GDRf54kcvp9w6BGKJU7s1ugLJU7k7JTnKU3ahXE5I7qQDnw98P4KxEov+9WA83LJg8LJf98ft2lGlpX1A6U32y7gClrT9tpb2bMeerk0SuP4+fdzkC5c0gz227iMzej0HLmdqUqmUC5+aPGKJa4c5r/iBYBlJ/+ccQGfj/1qInuP+63SfVxcClMT/RZf69hUQx/sZwStcC6UqEGlMxN4KvN5zQA5U30/tWyxN6han58+Zq6WwjL42rJ++u3qqu+S8wA5alxrZwiaRy/FoG2X1k+KEQAJTMesJUMPWn8qpP4focFKD/vVBGv1Uy0UHLuUKKMDXraZc1LQJmkWsABZUhICIoWLeqy39kydpEiRVR/Z4X4Te3sv61HL4aFhSmWSRbkvGnTprrHlaxB5QoomRM58z/ik9Wi5OlGvAGUi/dfwWfLnHemW4GSX9pl7WOWSxa2o13il7AaUP7Ruy5qaPAL9XTPWoHScWnezOTu6HPlyULJlnUYMDom/uXKf61rBUoGcQzm9CbrBO/Oh7JMnsw2Xyk9QPlj95poWiaX3ib5Zf7hK47hf4l+id+8VR2tKmj3DZ2/7QImSwRK3jeNF5MHypDbD9FkumW5VyZQbv24CYokggf/zmteJhe+N2mh5IFy4bu18Pb3+2y3/2mr0jYrsRUoExKeotgwixHDCpRsY1sNzs2IPc/80jzL6+oZX3/ips1ns2PV/JhhACjZuKql8nFqBijPTmithCPjE+8/KgIoef9crQ/whFUn8V0iUE59pRI61yioFOX960UZG7S2ic8na+430jbZZQIOKGULpPV6sgYV70g/6/UqaFclP6JiYlFxjD1QrvuwIcrk8eyr4g2gZPHZiie+gHn9jowOVmIn8i9o9vvp8a1wNiwKL8+xLK2oAaVWv1BPfeYIlMyZfpoLfz+RQOm4cUoNKA9fuYctp8PRs1Exlzu4+Zcr/7WuFShZ1ID+v+kHyu2fNEWhHBncWijZkt+J65ZQWp6A8oNFh7HyP8uS9489aqJp6eQBlLxfItvw0VLHZqNvt1/ApDXyLJS8+wQ/fnYNaYZ8ifEOed9fmUC57ZMmKJwYh5MHyqCyufBdN3NL3jxQ/q9nbdumI6bB0NZlbFDfq1ExDGtTVpHG+p60AqWj3zp7nvmleTWg3HDiJnolbgLytHGNv661f9imnG/erm63IYh/l5gByguT2jjFCj4XFoUWX1os1CKAcvfQZmCxTvWkSWtO4dvtF5Ui016phFddAOWyPnVRvbB5dyg97bLmlTX3G2mb7DIElIIUlzWoXAGl4+YRdktafVW8AZSuXoTsb0fHBCuQxCzDRYdavvhZYs7rx69F2ny11IByed96qFYo6UhNo12nFSgdfT3NWChdTUBG2j9x9Uks+Ney/DO1UyV0rmn5WueTuz5dc+wG+ibuMtVzfStkuLNQViqQFUevRijV6gFKtuzYJJkA5aiVx/FL4jKy3s1GC7ZfxMQ1p+y6xcyY89S/vPsEn5cHSn7cygRK3pLFv/NEAyWLHPHGAssuZpZGvFgWE1Zb+sAVULLfezYsBqNAufFkGN5L3ASkFl1C7Vlmf2dA+W3XGk7WUWsZM0DpahPRxVsP0OyLbUr1IoBy//AgJZyWnjR5zSnMTwRKttGPuSixxFsoRRkb9LTLmlfW3G+kbbLLEFAKUlzWoOKX1KwWSkdfP3ZLvFO7u1v0BVCy9jhel/fVUQNKrRuNPHWpI1B2rlEAU19x3pH8JC4BpUZYdl2zZGZyv/vwCaolhgQyUxf/tc4v/6hNQo5tXnc8yYfLk0787/uGNUeuLOncWijZJjDmxM+SHqD8+Z1aaFwqp57m+G1e3i+R7SDWs9nou38v2mBGxJjzJBJv7ebz8lYkPnSWr4CSf+cFlc2N77rV8HRrbn/nN+Uw3zu2W9iaRrUth3GrTir/+36jYhiaaKE8evW+MraZK0rqlCng6nnmfT3VnvFNJ8Nsu8o9RUJwfE+y/2dAyT5U+KgeoiyUrt5vV+5Eo9G0LYoeIoDSGjpOTwdOXnsK87dZLJRfvFoZnaxAyYUNEmVs0NMua15Zc7+RtskuQ0ApSHFZg4pfUrPGuXNcmmW3xC8ZubtFmUB5bEywctSe44uSvchUgZJ7aazsVx+VNexc99SlWoHScfOQGaB03DhltC7+a51f/uHv2V2f8ktunnTif7fGqHNnoWRxKg8mxvf0BJT8ppxf3qmFRskEKHm/RLaDuFkZ7ZuN2MYD5iPLJ6PjREvf8tZuPj/bJGc9FpB/BmQCJe/Hyb/zWHgpBlRmEg+UzC/bGhaJ1Tn25fIY/dcJC1A2LoahrS1L3o7JNVAmbZhk+V313T+nwvDuz5Zd5WquNmrPMvs7e8bY/at9nJqxULpqLzsq1hrzUgRQ8nOA1j5kG9XYhjWWrLGX2X/zYYNYLOOqAlavtLaJzydr7jfSNtllCCgFKS5rULkCSkdLGrslrc7PMoGSD0vheF12HvSLsy27CVmcMhavTHlpcDv5rLEQzXaZI1Cqhbhx9AU1M7lHRMei8rgkP1ejdfFf6/zyj9ok5Hgd3kKiR0er/6s7oGQhnayBhvUAJdsY0bBk8rBQ8kCpd7PRDzsu2axj1r4xOk609C3/ccLn54GSd0+RCZT8srtooOR9KJllyxr/lmkwvn0FsHBKLLkDSsdQbayfeF9PNaDcfDrMFqZIbWXEHVAyC+X33WraDkBwvI5ooAyPjLFtABIBlK52knsaq5+vO42vt1qA0hoqz3FuELV65aktrn6XNfcbaZvsMgSUghSXNaj42HFfvVEVL1XOB8dd0+yW+Beyu1v0F6A8eT0SbWZbTqNQA8pVAxqgQn59pyy4unetQOm4ecjM5O64E99oXWaBkm346fFTUpgUrcP/5LiWyJAmldsl77rFcmD3xTtKlXqAkm2MqF/iBa1N8et8PFDq9Q39cecljP1bnoWSH0u8qHuHNbc7EtD6jtB7P3rfO/xJOTzU8u+84HK5FR9CM4kHSrbqYY0uwepkhwUwn02WejcujiGty7i8lKsVB8eTh1w94yKAksXh5MPEuVvyHt+uPN6uW0RVLk/vf95XVARQnpvYWnEZ0JOmrjuNeYlAaXXzcgRKUatXetplzStr7jfSNtllCCgFKS5rUPFO/9aTOBzBh92S46SgdpueXihG5XFlyToxtiUyJp5t7HhdPjyFHVByS97eAsrXaxbElE7OMRMdNw8ZhUCmoeNOfKN18cs/vD8R30/u+nTrmXC7uHta+9caUsSdhbJBiRew4/xtTUDJ7/L+rWdt1EuGQKl3Kf/nXSG25VZrvxgdJ1r6lbf88Pmt/rLWv4XejUbovWiwDwb++Fot19Dz3uGBkn9/8e880UDJ3inWOIusrSwU15DlFqDs06Q4PmslFij5DzojFkq25P1Dt5qqqx3888nGH/tQY2dw6+kHPi+/siICKF1t/PE0jqavP2M7DlMNKEWtXnlqi6vfZc39RtomuwwBpSDFZQ0qV0DpCD7slrTuppMJlFYrF2uf43X5AMpqFkqtsTU9damjhVINKF2101Pdar+LOhechwDen0grUG4/ewtdf0iKu6f1fi5OaoMUKZ5za6FkfpCsfpY8WSjtgPK92qhXPPlZKPUu5f+yOwSjVlr896zJm0DJW37cAaXWMaInn6v3Dg+UPNTyG51als+N+W+Ls1Cyd4rV1UYPULpyYeE3D7G6XPXdljPh6JEYSN0IULIlb3aWOR93WM1CqWXseHr/8+8tEUCppU2O4+iLDWfw1WbL+erWfQPsv3l3KFHGBj1j2JpX1txvpG2yyxBQClJc1qDiX6780W6OliPrJgpPt+fpheKpvNrvrixZ7oCSxaEMTox3prbL28g5sK7a5wiU7uBHlD6ijnHkIYD3J9IKlDvO3cZb3+/V3a3WicCdhbJp6ZzYckY/ULLQLXWLmz+jXfdNeaEAv+Stdyl/4e4QjJQIlNPWn8bcLRbfNDugHN4cuTKn84I6SVV6Akr+g5h/57Uqn0eJw2gm8UveLF5vq5kWVxuWWGxX66EMfZsUx6cqFkpXQOl4NroreOJXCIwC5U/v1EIlLu6wN4GS3/DpK6CcseEMZicCpdXNi4DSzBPgvbIElIK0lQWU/IQ1r0s1tEk82s1xoreeSuPp9kQBk+N1XIEH75DteF0+gK4dUHJL3lqDtXu6Z18ApahjHHkIsB696U57x0lt14XbeHOBd4CSxQjcdCpcaY4nCyW/y3tRrzqo46NzeD2NFb2/88+n3qV8dq62dUOI9bpGLDpa26wGlFpXN7Rex1W+Lzeexax/ztnFeuQtlAdGBOGFTJZ4hbymooGSxeu1Bu5m12KxXT9ddlS5br+mxfFJS9dL3q58orUA5bazt9AtcYXACFCyJW/my8ofZOFNoOT9830GlBvPYvY/55Q+sbp5KUDJzQ2iVq+MjGlZc7+RtskuQ0ApSHFZg0orUGoNzyATKNmJOOlSp1QUd7zu+fAoBM2wnMigZqFc/2EjlM6T2XSPdfp6ly28jSf4EaWPqFN31PyJeFHctXnPxTt4/dukQM5axdRioWT+bRtOhukGSl+ew6v1/rXm459PvZbX/+29jOErLDuMkztQMjedkDvRKJIjg80vkwdKfoWF17R1hTxgZ1mbSbyFctPgxgiaYQnczZJWoIyMiXWyEvKbh1hdrj4GeJcTI0DJlrwXvlvbdra443X0vq885efdqXwFlNaPD3avvBGFX/JeM7AhyuXzfDKcmXGjVlbW3O+Ntouuk4BSkKKyBhX/cv26SzW0VrFQ8svL7m7R0wvFqDyuLJTugfKB7cWuBpRaT//x1GZnoHR9bjarR5Q+ok7d4f2JeAd1rUDJwvrwgZw9aeUINu6WvNlkv/b4zUSgVNeUZeB9KH15Dq/W+9eaj38+9Vpef9t7xbbD2JdAyVsHtd63iHw8UPIBsPkjIttUzIN5XcQB5ZaPm6Bp4lnlClC+Ugmf/mGxUPZvWgIftyzt8tZcASXv314hfxasGmAJfcanf8/dsp0dbhQof+1ZG+VGrbdV600LJf8O9BVQztx0FjM3WSyU/JzHA6Uodygj41jW3G+kbbLLEFAKUlzkCRr/AAAgAElEQVTWoOJfrt+8VQ2tKuRV7sBxoufhzd0tigImx2voBcoLtx6geeIRX2pL3psGN0KJXOYtlM5L3t4HSlGn7vD+RLyDulagPHj5Ljp9nXQyiNbhr8VC+WKlvFh99EYiUBbE5I7OO+et1+OBkgWXrlHEN+fwar1/rfl4oNRref193xUlliGffLHk7Q9AybvseBMo2Rn11pNg9AClq6gNPFCqnePO+zAbAUq25P3ru7VRdtS6gAHKdnN24EjikaxGxjNb7p6x8axyv/ycxwOlKHcorc85n0/W3G+kbbLLEFAKUlzWoNIKlFrjfckEyjMTWiFtKtdL3vyZsWoWSrY8VSJXJtM95giUb9YupMSfc5VE6eMYK9TIi5W1zyxQHrpyzy6Qszsx2TFzfx+5juDyuTGjcxWXHy58eRb4+q8j13UD5bI+dVG9cPIDSr2gvGjfFVvIGquuRseJlodEzYfy4Igg5Ej0X9RSj6g8vIXSGkif1c2fOf5ixbyY26WaqUvyS97sAIiGUy1HC7LETp/6JNFCOaBZCXwU7NpC6Qoo+c1D375dHcHl8zi10yxQsiXv396rgzIjAwcoT9+MRKd5u5SYya7Cs3nqzK/+OYcvbEBZHa0qWHTlgVKUO5Sntrj6Xdbcb6RtsssQUApSXNag4l+u37yV9HA5WgStYV483Z4oYHK8jisLpTugvHT7oW3pSQ0oN3/UGMVyBiZQijp1h32pWx3U+R2PvP7u+pSdR9x+7k5Pw0L5fWTbcni7TmGkSZUUiNjdkneHqvmx4vA1A0BZTzlSLjkk3kKpF5SX7A+1bQjxJVBqjRAhur94oDw6JhhZEo9pZcdRsmMpWRINlOwACOvRgqx+dvrUx0uPKNca2KwEBqsApaswYFqAcuf52+jynWVTnBELpS+B0oz/Kluh4d8jesbOnM3nMH2DxULJW355oBTlDqWnXda8suZ+I22TXYaAUpDisgaVVqDUatmQCZTW4NhMcsfrhtx+iCaJvkxqS97M36noCxlN95ijD2WX2oUw0csWSlFB0nkHdX7HIy/KuuM3FV+80S+VQ7sq+e30Onr1Pl6eow0oR7Uth3caFLUr7w4oO1UrgGWHrmoCSn6XNzv+rpqPzuE1PZgcKuCBUu99LTkQavPfexaBssq4DbgfHavcOn9M64RVJ/GdFSgr5QULl2Ym8RZKFkC99qR/bNVpBcqHj+OcNsbwfc/O22bnjjumXedv400TQMk+vH57rzZKj5BnoWRL+RtPhoH5Ohd4PoMZ6Q2VnbvlPKatP6OU5S2/PFCKcocy0kBZc7+RtskuQ0ApSHFZg4r/Wue/1hwnen8ESn4Z3hEoL995iMbTtiq9oWah3PpxExQJUKB0BdFGhh7voM7HIXWsiwGsq1NNjl+LsDsZxF0bGJD2qK8dKJnFZckB/UDpy3N4jfSBuzI8VKzoWw9VdYDy0gOhtuVWXwIlvyFGtD7u6uMtlPypWhNXn8SCfy0WyraV8mKOQKDcN7w5ak1MAkp2+tRHVgtl85IY3KKUyyYbBkoubJdRC+WiXnVRasRaW7u8vSmHXUjtfSJjfPBA+V3XGghKBHV7oBTjDmXkfmTN/UbaJrsMAaUgxWUNKv5rnf9aC3SgvHIn2uYcrwaUzIG+UA7zX8iOPpRv1SmECe2960MpCihnbTqHLzdZln/4EBpahzF/ZrqnMnqBksWe/H1fqFKtpziU/KYcX57D60kDvb/zQKn3vv44eNW23OpLoPxvVAtky5BG762bzs8DJR+z1ptAyfxFq0/YZGs7O31q8BLLkvcHzUtikApQujqogO97Hnx4YXZfuIM3FljCdhkFysXv10XJ4XKB0nTnmqhg3tbzmLrOYqH8vlsNNC9rsfzyQPnPR41RXIA7lJFmypr7jbRNdhkCSkGKyxpUgQyU5ye2RqqUFn88RwslOy/Y6hyvBpTMgb5g9mcbKPkdj3wIDa3DmDnI8yeDuCs35qVy6K7DQslcB/6394puoPTlObxaddOaj4cKvfe17OBVm3XsWQdKPkrFpDWn8O32i4okbGMH8x02kz5ZegRLD1os6Qyeq4zbKAQo+Q2TPPjwbeXjwBoFyqW966H4sDW2as1YKGtN3ITwqMdKXVpXtcxob6TsN9suYMra00rRH7rXQLMyzkApyr/eSPtkzf1G2ia7DAGlIMVlDSr+a5330wkEC+WFSW2QMsVzhoFyx2dNhfjwOFoo2caT8e0ruBwJIn1MRdRlv+MxKWyU1mHMH3HpqczYl8ujW70idtnc+VB2rVsYv+y+rBso/+7fABULZPXUnID4nQdKvecLrzh8FYMWW6xj1uTNSV5tl7fWU7ZEdwhvoeQ38E1ecwrzE4GSRRJg4bLMJB4o2W5y/lxsdpyptQ8+DCqJD4NcL3m7OvmK929XA8q9F+/gtcSDBYwC5R+966GYIKBkH5gjVhzHuw2K2mIam9HWG2V5oPyxe000LZNLuQxvoRTlX2+k/bLmfiNtk12GgFKQ4rIGFf+1zi+rBDpQXr0XjQafW8J3lM2bBSxQrfLS4I7X2jmkGfJnS2+6xxw35QQSUPI7Hvld/lpFOR+eFEDeU5lx7cqja13tQNmjfhH8uDMkESjdBzbnN+XoBS9P7fbl7zxQ6j0O7s/D1/Dh4v98D5Sjg5E1fWrpMvJAyW/gm7z2FOZvs1goRQAlvymHbf6pMDopSDg7ztTaB4OCSuGDoJIudXB18hUPlLwlja+AP1jg1eoFMO3Vym51dnyvs005LBxV0aFiLJTSO9nABe2AskdNNC3tDJSi/OsNNA+y5n4jbZNdhoBSkOKyBlUgAyUfysjRWnft/iPUn7LZGSi/2Y19IXeVv7MQH/m8AJTMsjauXWBYKM0CJR/v09PQH9+uPN7WAZQ9GxS17cZlMSwnd3Ttl8qu+ywApd7j4Fb+dw0fLCKgZOODd4/hgbJdlXyY9bo5CyUPlMxXkw8SLgooeUuaSKBkYYOW962veoKXiFUQT+8F2b/zQMnOMW/iAihF+dcbuTdZc7+RtskuQ0ApSHFZg4pf/uGXVQLBQnlpchvbzmPHF9+NiEeoO9k9UO4Z2hx5sqYz3WOOS96BBJT8jke10zjcCcSHZ/IkJHMDYNZbPrlb8n6/UTHb0qQeoNQLXp7a7cvfeQul3tM7/AUo+RiQMrXkwwbx7jHMf45BBUuigZItrfMheNhxplaoZzu8BzY3ZqFUA8r9IXfx6jeWk6qMLnk/y0D5yzu10KhUTot+nLFBlH+9kfEua+430jbZZQgoBSkua1DxX+v8sgo/0evZrOGtL1pX4OEOKG9GxKDOZEv4DrU4lCxmXO4sBJSuYrJpHcb85idPZfQCZZ8mxfH1VsvE7wko+V3evjyH15MGen/ngVLv6R3sVKIBvx+2u6QvfCiPjQlG5sSg4nrv30x+fsmbX834fN1p27hqXyUfZgq0ULJQZvyOaR4oP2pRCgOMAiW3NMtrciDkLl4xAZRsyXtZn3rPrIVy4bu10LAkAaWZ58ybZQkoBanrT0CpZxKSCZTudiOGRcbYAgyr7fLeN6w5cnkBKLvVLYyxAbLkzYfQUAue7G5I876qzB+VuRqopQntK+AtHRbK/k1LYM6W87qBUq8lT9Aj65VqeKDUe3rHqqPX0f83AkrWMfzH59R1pzEv8UOFncb05WuWY0CNJn7Jm4Erv8FFK1A+jot3Ci7O+1D+qAKUBy/fRaevyUKpp+/4JW92jnmDki8oxXkLpagNm3raZc0ra+430jbZZQgoBSkua1DxFkp+WcUoGBot50k2VxZKd0AZHhmDWoknVqgB5f7hQciZOa2nS3v83XFTTiABJbMAMosNS2qx7twJcP3+I9tRc2zTAUtZ0qfC2L9POhWb2KECutTWvuTNlgetx0LqiUOp15LnsYN9mIEHSr2nd6w+egP9fjtk13o9H4d6b1ttlzd/So3eOs3k5y2U/H17EyjZdfh3FdtBzvx7Wfo4uBT6N3O95M2OEnQMLs4DJe/rx2ty8PI9sPePAkQ1CmDqK/o25TzrPpT/61kb9Us4A6WoDZtGxq+sud9I22SXCTigPH/+PKZPn449e/bg+PHjKFOmjPJvx7RmzRoMHz4cp06dQoECBTB48GD07dvXKR+ra86cObh58yYqVqyIadOmoUmTJrr7QdagSrZAGRVjO7FCbcn7wIggvJBJPFB2r1cEY14u77LPRQK3iLr4r3W10CTuBi/vq2qdMPm/8WUndaiIN2sXsqvOnQ8lA1Rr0HVPQMlvytkwqBFK5c6s+5nzxwI8UOoNtrzm2A30/R8BJetXHih58O1YNT9mmLRQ8vE+vQWUP79TC40Tff34cXroyj10nGccKJ/1Je/fetZGPRdAKWrDppF3iqy530jbZJcJOKBcuXIl+vfvj9q1a+Ps2bNISEhwAsrdu3ejUaNG6Nq1K9566y3s3LkTo0ePxvz589GzZ0+bxgwmhw0bhkmTJqFatWpYsGAB/vzzT+zbt0+BSz1J1qCyA0puWcUorBgt50kbvRbKW1GPUXOi5cQKNQslO9UixzMOlPO3XcBkF0F+PfWH9XfeV9UKlPzf+HrYLm3mC8knd0DJfM6+2Gg5xceTDyUPlHoteVrv1Rf5eKDUGxtv3fEb6P2r74GSP/ZQpoZqFsrp68/YXCk6VsuPGZ3NLXknJDzFwj2XUTJ3JtQr/oKdhZIFTbf6sX7SsjT6NS3hUoLY+ASn02p4C6UaUB6+cg8dTADls26h/P29OqhbPIfSJ/ySt6gNm0bGu6y530jbZJcJOKBkAJkiheW0le7du+PAgQNOQNm6dWvcvXsXe/futenZq1cvrFq1SokZxco/fvwYuXPnBvv71KlTlXzx8fEKSFaqVAmLFi3S1ReyBhUPlPyyilEwNFrOkzh6gfL2g8eokXgEmhpQijpj2HGXdyBZKHmgVNtJ6q5vXAEl779qBigZoE7fYAFKTzEy+U05mwb77hxeT+NY7+88UOqNjbfu+E30/vWg3SV9seR9clxLZEiTSu+tm86vBpRfbDiDrzZbfHNFAKVjQ/l31Zw3q9r8WM0AJb8bmb/ef6H30X7uTgsQ0ZK3pjHDr8os6lUHdYo5A6WoDZuaGuSQSdbcb6RtsssEHFDyArkCSgaKWbJkwZQpUzBo0CBb9m3btilL2QxAq1evji1btqBZs2Y4dOgQqlZNims2duxYfPHFF4iIiLCFuNHSKbIGVXIFyjsPHtvO1FVb8hZ1xnCyAUoVx3+9QMn7r/Jlp3SsiNd1WCjZBNyoZE5cuvMQL1XK6/b54YFS79KwlufRV3l4oNQbG2/9iZt4f+GzC5R82CAepHmg7FStAL7o7N7vUG/fGwHKuPgElHA4T5u3UPK7kfn2HAm9j3YmgPJZXPLm/cYX96qD2i6AUtSGTb1jh+WXNfcbaZvsMskOKE+ePIny5ctj7dq1aNWqlU3PW7duIVeuXFi4cKGyDD5v3jz069cP0dHRSJ8+6fSVpUuXonPnzggNDVV8L7UmWYOKB0p+WcWopdFoOU+66LVQ3n34BNXGW87UVbNQijoSznFTDjvhZfRLgeFDaWehFAWUnP8q36+fd6qI12pqX/J2Z9FxHC88UPryHF5P41jv7zxQ6o2Nt+HETfTyA6Bkwb7Tp0mp99ZN51cDyhkbzmB2ooXSG0A5YdVJJSB/gefTY2jrsraNUZ+2Ko2+TVwveccnPHU6T1sGUD6LS948UC55vy5qFc2ujLVXv9mF/SH3lP/eN7w5cmU2H1LOyCCWNfcbaZvsMskOKJm/ZIMGDcD8KOvUqWPTMy4uDqlTp8asWbMwcOBATJw4EePHj0dMTIyd5ps2bUKLFi1w5MgRZelbLUVGRoL9Y003btxArVq1dIOo3g7ngZJfVjEKhkbLeWq3XqC89/AJqnoASlEBl5MLUKrtJHXXN66WvHn/Vb7s1E6V0LlmQbvq3PlQGgVKvb6GnsaeL3/ngVJvKJNNJ8PQ85cDds33xZL36fGtkC61HwHlxrO26AGvVC+A6R6OK9Tb/ywE0NYzt1Cj8PPYe+mubWPUZ63KgMVWdZU8ASUf3oYvf/Tqfbw8h5a89fQRv+S9tHdd1CziDJSiIoDoaZc1LwFlkmrJFijZLnC2ccearEA5e/ZsDBgwQAHKCRMm4NEj+zh8GzduRHBwMI4ePep2Y86YMWPAlscdk17Lpt4BnFyB8n70E1QZ58JCyZ3lLSrgsiNQvlO/KEa9VM5lV4gEbhF18RZKNcd/d2OKj59n3V3N+6/aAeUrldC5hj1Q8kd/Ol5HD1Dym3L0+hrqfWZk5ueBUu/O039OheHdn59doFTzofxy41nM+uec0o1azr8209/8Tnt3QMk29vDxKxn48xZKNaA8djUCL83ZoTTRiA/ls7jkzQMlO8e8RiJQ8ptyREUAMTJ2CCiTMVDqXfJmQJkuXZKpXOuStz9YKHk/HaOwYrScpwdPr4UyIjoWlcdtUKpVW/IWFR/P0YcyUIFSzfHfU9+wYOaRj2JRNm8WJSvvv+oJKJnvGIuld+xaBCasPmV3KXdLhI5t4oFSr6+hp/vz5e88UO4e2gx5sya503hq1+bTYXjnJwJKphNvmZUJlGuP3UCfxNBNQ1qXQe/Gri2UnoCSj5fI9/vxaxFo+5VxoHwWl7x5oGSnBDGoVoCcO3pRVAQQT8+oq98JKJMxUD5Lm3L4r2CjYGi0nKcHTzdQPooFs1C4A0pRu0+TC1CqOf576hvH33n/Vf63aa9UwqsOFkrr77/tvYJhK47ZVeXOouN4Td6HUq+vod77k5mfB0q9O0+3nA5Hj5/22zXXF0ve7HzrtKnkL3mrWShnbjqLmZssFkotVj0z/c2HbhraugzeVwHKp0+foujQNbZLOVoo+XiJBJRmegTKOe7sPHeWlveth2qFnIFSVAQQIy0loEzGQMlujYUNun//vuJHaU29e/fGX3/95RQ2iP2d7QhniYUNYn6TLHRQIIQNSk5AGRkTi0pjXAAlt+QtarOA45L3uw2KYmTbwFvyVltW0/tS5P1X+bLMV435rLlKv++7gqHLCSgdteGBUu/O061nwtH9RwJKpikP0rM2nbMFzH+tRkF8/oq6b7vese+YXytQsnKOH+P8krcaUJ64HoEXZxu3UD6LS978ppwVfeuhqgugFBUBxMj4IaAMYKBku7LZKTgszZ07FxcuXMCMGTOU/2/cuDFy5sypgCQLbM7CCnXp0kUJbD5q1CjVwOaTJ09WApt/9913WL58ecAENueXVYxaGo2W8/Tg6bVQRsXEomIiULKl2LUfNLRYJDigFLVZILkApdqymqe+cfyd91/lf/vi1cropAMo3S0ROl6Tt1Dq3byi9/5k5ueBUu9GgW1nb6HbD/vsmusLC+XZCa2RJpUl1q/MpLbLmx3nOSMxYL73gTIpFuiwNmXQq5HrJW+PQPlebSVoumM6eT0SbWb/a3m3GYhD+awD5Z/96qNKwWyKfvwub1ERQIyMdwLKAAbKkJAQFC1a1GW/s9iS1mMTGXSyU3D4oxdZmCA+sWUL69GLYWFhimWSBTlv2rSp7nEla1Dxm3L4r2CjYGi0nCeB9ALlg8dxqDB6vVKtGlCKWopzBMqeDYpiRABaKNWsIJ76xvF33n+V/21G58roWM21hXLRvisY4mChNAqUvjyHV69WnvLzQKl3o8D2s7fQlYBSkZgHaR4oX69ZEFM6edNCKQYo+RNd+DFz6kYkWs8ioPT0HPG/80veK/vVR2VXQDk6GFnTp9ZTrbC8suZ+YQ32YkUBvcvbi7rorlrWoEquQBn9JA7lRrkHSlGWE0cfymceKDn/VTNA6c7nzPGB4jfl6N0NrfvhlFiAB8pDI1sge8Y0mq/+77lbePt731soz01sjdQp/cdC+dU/52xHenobKPng8sPblMV7jYqp9p+7JW8CSs3D3mNGHij/6l8flQo4WyhFhZTz2BgXGWTN/UbaJrsMAaUgxWUNKjug5JZVjFoajZbzJJteC+WjJ/EoO2qdUq3aSTnnJ7ZGKgETnSNQvtewKIa/GHg+lL+pLKt56hsnC6UKUH75WmV0qOraQrl4/xV8tszeh9IoUPryHF69WnnKzwOlXr+unedvo8t3ScfFOlrqPF1b7+/T1p/G3C0XnIqJes70tkdtU86czedsR3q+UasgJnf0noWSDy4/4sWy6NnQGFDyRwTyOpy+GYlWM8lCqWds8EC5akADVMifVSnO7/IWFVJOT7useWXN/UbaJrsMAaUgxWUNKjWg/O7fi0rw32/eru7Sd0ftNv0FKGNi41FmpHugvDCpDVKmeM50jzkueQcqUKpZQfQKxG+I4svOfK0K2lfN77K6JftD8emyo3a/efI54zPzPpR6d0PrvT+Z+Xmg1OvXtev8bbxJQKl0F7/kPXfLeUxbf0b5+xu1CmFyx4pe61JvA+WZm1FoOXO7BYjIh1JTP/KbctSAUlRIOU0Ncsgka+430jbZZQgoBSkua1DxQOkIFCw2WgqdwBVIQHlxUhvd9+eqex2BslejYhjWpqzLkSBSHxF18YHN1awgeoc0vyGKLzvr9SpoV8X7QKl3N7Te+5OZnwdKvctwuy7cxpsLfG+hFPXhpld3NQulTKDceDIM7yWeVmTGQsmfOc3rcDYsCsFfElDqGRs8UK4e2ADl8zlbKEWFlNPTLmteWXO/kbbJLuM3QPnkyROkSaPd30i2UJ6uJ2tQ8UApAihEQI4rbfQuefMnuKgteV+a3AbPPSfeQhmoQKk2aXkaq46/8xuiNAPlgVB8+oe9hdKTzxlfN2+h9OU5vHq18pSfB0q9y3C7L9zBGwv22F3CF7u8RX24edLK8XctQPlm7UKY1MF7Fkr++EsWSoyFFFNL7nwo+TOn+fLnwqLQgoBS19DggXLNwIYol89yIAO/5C0qpJyuhiVmljX3G2mb7DLSgXLhwoVKjEh2/CFLx48fR4cOHXDp0iXlDO4lS5YgV65csnUwfT1Zgyq5AuWTuASUGrFW6Qc1oBQ1uTr6UL7fqBiGBqCFUm3S0juYHz6OQ/nEHfZ82dlvVMXLlfO5rG7pgVB8Iggo9YbX0Xt/MvPzQHlibEtkTJtK8+X3XryD1759doFSLWzQvK3nMXWdZck70IHyfHgUgmaQhVLzQwH7wOYsnJz1hC8+bJCokHJ62mXNK2vuN9I22WWkAyULHP7+++/DGsKHhegJDw9Hnz598NVXX6Fhw4ZKPMhAS7IGFQ+UIixU/mKhjI1PQMnhBJSexj2/5L20d13UTDzX1lM5d7/zO+zNAKWnJUK+bt5C6ctj08zo5qosD5R6l+H2XbqrxF3lk6iPKFdtVduUI2olQK+2akDJW6i61C6EiV60UPLnqY9qWw7vGLRQqj2b58MfIGjGNkUa8qHUNkL4TTnrP2yE0nkyKwUJKLXpJzOXdKDMkiULVq5cqcR6vH37NvLkyYNVq1ahVatWWLx4MT7++GOEhobK1EDItXwBlCIsVP4ClOyM6BKSgNLRh/L9xsUwtHXg+VD+0bsuaggASn6HPf8wfPVGVbykYqH84+BVfLz0iN2zYxQo9YbXEfLAeqkSHij1Wk32h9zFq9/4Hii9CbHuZNcClG/VKYQJ7b235M2fpz76pXLoUd/Ykrfas3nh1gM0/4KAUs/jx39QbBjUCKVyOwOlqBjFetpFFkpntaQDZbZs2RRwbNmyJZYuXYquXbsqS+Bp06bF9u3blb8/evTISL/6tAwBpb38en0o4xOeovgwywlI3l7yJqC07yt+hz3/y5w3q6JtJddL3iKB0pfn8Ip+afBAqXeSOxByF68QUCpdwkMtb6EioHwey/rUczr20TqOvWUgEP2c6KmPB8qNgxqhpAugFBWjWE+7CCj9ACjZ8YgMKidNmgR2jjazWK5evVpp2a+//ooRI0aAnYYTaMkXQCliydNbL6CyI9fhUWy8XTfyk4TjddkO9WKugPKb3dgXctdpkjEzPpILUC7rUxfVC2c3I4VSVg0o575ZDS9Wyuuy/mUHr+IjQRZKvfEaTd+wFyvggVLvJHfw8l10+poslI5Aybt5vF2nMMa3r+C1HtxyOhw9frKcpz7mpXLobtBCqfZsXrz1AM1MWCirFcqG5X3rP7NAuWlwI5TIZbFQ8ptyfBWMn7VD1tzvtUEvsGLpFkp2rnbbtm0RGRmJzJkzY9OmTahRo4ZyS506dULKlCmVjTmBlmQNKt6HUsSSp7eAksVbY870K/+7butKd0DJjsEsOtSFhVICUPZuXBzs2EBXSaQ+IuriJ1dmqWBn+5pN/A57vq55XaqhTUXXQLn80FUMXmK/5O1pVyxfN+9DqTdeo9n79WZ5Hij1TnKHrtxDx3m7VD/CRLdbzYfSV0vearu8v91+AZPWnFZuv2vdwhjXzotAeSYcPX60AKWZJW+1Z/PS7YdoOn2rBYgMxKEkoGyMErkyOQGlr0JdEVDav5WkAyW7fFRUFM6ePYvixYsr1kprYudvlyhRAqVKlRL97vR6fQSUriVWgyhXf7f+zdtL3o67vAMVKJf3rYdqhcwDJb/D3idA6cNzeEW/GPiA73onucNX7qEDAaXSJTzUJiegDLn9EE1MACX7gHyWl7z/+agxiue0ACW/KcdXoa4IKP0AKEW/xP2hPl8ApYglTxFWM3f6mwZKCRbKPk2K47NWgWehXNG3HqoKAEp+hz3fl193qYbWKhbKFYevYtBiewulp12xahZKvfEa/eF5V2sD8wWeuems4uelFnJJrex/offRfu5Ou5+9aS30Nwul2qacBdsvYuKaU4ou3eoWxlgvWii3nglHdwEWSrWPvct3HqLxNLJQ6nmGeR/aLR83QdEXMjoBpa8iExBQ+gFQnjhxAuPHj8f+/fsV/4Pdu3ejWrVqGD58uBKLsnXr1nrGm1/k9Q1Qml/y9BVQlhu1DtFP4ov6hmEAACAASURBVFG5QFas7N9A6UOXFkoCSrvxzS95/9mvPqoUTLLwG30Q+B32fB3fvFUNrSq4XvL+8/A1fLj4P7tLGgVKXx6bZlQzb5Q7Enof7SQC5foTN/H+woNOt+JNiHWnmxpQsmNlJ6xOHkB55U40Gk3boshAS97aniJ+U87Wj5ugiAug9NWYJaD0MVBu3LgRL774ogKQzZs3x+TJk3HgwAHl/8eMGYODBw/i77//1jbS/CgXAaXrzlADVrb0w/wr36hdELkyp7MDSha4lgWwVV6683eDxedjSdRLw3FTTt8mxfFpAFooV/arj8oCgJLfYW8PlNXRqkIelx3rCig9+ZzxFfE+lHoDgPvRYy+0KceuRuClOTvs6hQ15l01lPkt/7wrBAWzZ8C7Px+wZfHmNc0CZfd6RTDm5fJCdecr23b2Frr9sE/5k6fx7O6kHLXVg9C70Wg41ThQPutL3ts+aYLCOZwtlL4aswSUPgZKtgGnXLly+OWXXxAXF6cct2gFShafkgU8Z3AWaMkXQCnCh85XFkpX/WttCwGl+ujnLZR/9a+PSgXMWyj5Hfb8lee/XR0ty7sGypX/XcMHi+wtlJ4mYDWg1BsAPNDeDVrbe/xaBNp+JQ8o+XZ5+z2gRQM1C+X3Oy5h/KqTShXeBsrtZ2+hqwCgVFs9IKDUMhLs8/AWyu2fNEWhHBmUDLwPJQGlfl29UUL6ppx06dIpFsgWLVogPj4eqVOntgHltm3blDiUMTEx3rhXr9ZJQOlaXj0Tla825fRrWhyftAw8H8q/+zdAxQJZTY9rfoe9GaD0FGZFDSh9eQ6vafEEVkBAuQH3o2MVRXlASE5AefVeNBp8ThZKPY8ND5T/ftpUsaizxIcNIqDUo6j38koHynz58inL3N26dXMCygULFijxKdm53oGWfAGUIjZl6AE+I32ip35ZQOm45C0LKKuP34g7D5+gR/0iGP2SsWU73kIpCihZv7oKRK/XQqkHKAf+fhh/HbGElNJ7ooyRcRgIZU5cj8CLs59dC+WXG89i1j/nkCZlCpydmORH/8OOSxiXaKE08+xoGQP/nruFt7+3LHl7Gs/ulrzV3FEIKLX0grqFkgdKslDq19LbJaQDZd++fcHCA61fv14JEcQslMxvMm/evGBBz1966SVMnz7d2/ctvH5fAKWITRl6gM+IaHrq9xVQ9m9aAh+3LO3y9vS035M+4VExij9oUNncSJc6pafsLn/ngXLVgAaokN+8hVINKL99uzqCVZa8GQwyKOTT2JfLo1u9IpruiwdKvSfKaLpAAGY6dSMSrWf9a9dyWZYXkePcqPQsfNXGk2GoVjgb8mZNb6vmx52XMPZvy5L3O/WLYtRL5YxewmO5Hedu463v9yr5PLlwuANKNXeUa/cfof6UzUr9RjblPOs+lASUHoewTzNIB8qIiAgEBQXh6NGjqFixIg4dOoTKlSvj4sWLKF26NDZv3oxMmSxxpgIpEVC67i09E1VyB0oR4zm5ACW/KUfviTIidPTHOk7fjESrmc8uUKr1yU87L2FMMgHK6/cfoZ4XgXLkn8excM9l1CmWHYt61fXHYa67TfyS947PmqLA8+RDqVtESQWkAyW7r9jYWOWYRbbj+/bt28iePbsCmexcb7ZJJxCTL4BSxC5fPcBnpF/01O8roBzQrAQ+Cva+hdKIfo5leKBcPbAByufznoXS3ZL330euY4CDhXJcu/LoWlebhZIHSr0nyojQ0R/rYKdLtZy53a5pz5KFUgtQvtugKNiJTN5KO8/fRpfvLBZKM0veau4oNyIeoe5k71komZV376U7yoEHGdOm8pZMUusloJQqt6mL+QQoTbXYTwv7AihF7PIdtuIYftt7BV1qF8LEDhWFq2sWKK2BhnNlTot9w4OEtM/xpJxABco1AxuiXL4sQjTR60O56uh19P/NfsnbKFCen9gaqVKmEHIfgVzJ2bAoBH9JQOnYhyy00ei/Tih/9jZQ7jp/G28mAqWZJW81oLwZEYM6k/9R7sXIkvcP3WugWZncqmd5B/L4V2s7AWXg9KrPgJIdvbhv3z7cuHEDbKMOCyfElrwDNQUqULIYhMx3i4XqSZniOeHymwVK1qBzYVHImy09Mgn64k4uQMlidbJ+E5FEAOX4duXxtgELpd4jCkXcrz/WcT48CkEzCCgd++aX3SEYtdIClD0bFMUIL1ooRQGlmn9zWGQMak8yBpQbBjVCqdyZFR30vFf9cazraRMBpR61fJtXOlA+ePAAvXr1wpIlS5CQkKBsymFL4ClSpMCrr74KttObfCjVB8Xktacwf9tFJYPIXb7eGoZ6Xnyulry90S7HXd4Dm5XA4ABc8l73YUOUyeM9oHS3KWf10Rvo99shu+4xCpS+PDbNG+PLaJ3nwx8gaMY2u+K05A0s3B2CkYlA+V7Dohj+oveWvHdduI03F5jflKMGlOGRMahlECj5saDnvWp0PPpLOQJKf+kJz+2QDpQ9evTAsmXLMHPmTHTs2BHZsmXD/fv3lb8NGjQInTp1wo8//ui55X6WwxcWSpG7fL0lp54XHwGl517gfSjXf9gIpfNYLBZmkysLpW6gbF8Bb9cprKkpvA+lLGjS1DAfZrpw6wGaf0FA6dgFyQooo2JQa6IxCyUBJUCbcnz4gtJwaelAmSVLFnz++efo06ePU/PmzZuHoUOHgu0ED7REQOm6xwICKJuXxOAWpVzegJ72yxiz/gKUa47dQN//OVgoCShNDYGLtx6gGQGlk4Zs1zLbvcxSr0bFMKxNWVM6uyu8+8IdvLFgj5LFjA+l2oY5FjqMgFJf95GFUp9evswtHShz5cqlHLvYqlUrp/tet24d3nrrLWXnd6AlAkoCShljlgdK3qfK7LX1+lCuPXYDfRyAckL7CniLLJSGu4Kdb99k+la78rKst/724cSL8OueyxiRCJTvNyqGoV4Eyj0X7+D1b80DpdqGuVtRj1Fz4ibl9vRuyiELJVkoDb9cJBWUDpSfffYZTp8+jT///BPPPZe0CYQd/9auXTtlY860adNM3z6rn53Ic+rUKaRPnx7169dX/t9x4w8Lsj58+HAlX4ECBTB48GCw4Ot6ky+AUmTYGL33qzW/nolK1pK346acD5qXxKAAtFBuHNQIJROd9LX2h1o+AkqzCpovf/nOQzSeRkDpqOT/9l7G8BUWC2WgA+XtB49RYwIBpZ6nhSyUetTybV4pQDljxgzbXcbFxWH27NlgZ3qzU3GYxTI8PFw53/vx48cYOHAgPvnkE1OqbNq0CcHBwYq18+2331Z8NMeMGYPIyEicOHECbNmdpd27d6NRo0ZK/EuWd+fOnRg9ejTmz5+Pnj176moDAWXgWigDFSg3DW6EErl840O57vgN9P7VfsmbLJS6XhlOmQkoXetnB5SNi2Foa+8tee+9eAevJVoozcShVIvAcOfBY1QnoNT1oBBQ6pLLp5mlACXbwa01MatlfHy81uwu8zEY/Oeff5TTd6xWUBaiqHbt2sqxj61bW86JZf++e/cu9u617Opjie1AX7VqFRgg6mm3L4BSZBxCU4K7KeyPFkrHXd4fBpXEh0GB50O5aXBjlMgl5lQpvZty1h2/id6/HrTreQJKc0/RlTvRaDRti10ltOQNJU4ui5fL0vteBkp2NGrn+buVa5nxoVQDyrsPn6Da+I1K/bTkre15IaDUppM/5JIClLJvtFu3bjh8+LByvKM1sbiXbLl79erVaNOmjWINZZbKKVOmKLvLrWnbtm1o0qQJDhw4gOrVq2tuOgGla6kIKDUPIU0ZeR/Kfz5qjOI5AxMox/19Ej/svKTcsyxo0iSwDzOF3o1Gw6kElI5d8Pu+Kxi63AKUvRsXx5DWZbzWS6KAUi2k172HT1CVgFJX//EW6r3DmiN3lnRK+Ve/2YX9Ifd8/g6RNffrEs1HmZMlUG7fvh3NmzcHW2q3LnmzpfTz588roJk2bVqcPHkS5cuXx9q1a+02CN26dUtZhl+4cKGyDK41yRpUfBxKkYGttd6n3nyBAJSDgkrhg6CSLm9NT/v1amMkPw+Umz9qjGJeBEp3Ry+uP3ET7y80bqGMeBSLHj/uQ+1iOfBZK+8BghGNfVWGgNK18ov2XcGQRKDs06S4V8fL/pC7ePUbi4XSzJK3Wkiv+9FPUGUcWSj1PGPsOMkeP+1DgWwZ8PkrlWxFCSj1qCgnr0+A8s6dO5g7dy7+/fdfZcmZneXNfBnZZpgcOXIIuXO2bP3mm28iKipKqa9cuXJYv369svGGJeYv2aBBA8WPsk6dOrZrMh9PFmx91qxZij+nWmL+mOwfa2In/tSqVQuhoaG2awi5EYdKeKAUGdjaG21ldeoBMl9tyglUoNzycRMUfSGjkK7TuynHLFAKaXQyq+TqvWg0+JwslI7dKhMoD4TcxSuJQGlmyVsNKCOiY1F53AblFmnJ29wDPHjJf1h+6JpSiS9XOWQZk8ypJae0dKC8cOGCAo8sNFC9evWQJ08e3Lx5E7t27cILL7wAZl0sXry4qbtndbFl7e7du+Pll19W4lpOmjQJMTExCkiypW4rUO7Zs0fxrbQmK1CyjUMDBgxQbQfb5DN27Fin3wko7SXxR6B09KEkoLQHf2sPugtsvuHETfQyYaE09YAn08LX7j9C/Smb7e5O1kSp5zmVLf/i/Vfw2TLLknffJsXxqRct2qKAUi2kFwGluNHD/FEHLf4PzcrkQrd6RcRVrLMmAsokwaQDZfv27cH8GVnMyUKFCtlawkCMxaYsVaoUVqxYobNL7bOzc8FZ3cuXL7f9wJaymXWShQ5ioYHMLnn7g4VS5EkppgR3U1jPRCXLQukIlCyo+cDmgbfkvfXjJijiRQulO6DceDIM7/1ywK7n9WzK8dZ4C+R6r99/hHoElE5duGR/KD5dZvGH79e0OD5p6T0XiYOX76LT1+aXvNVCejFXj8pjxVkoy+bNAub6RMl3ChBQ+hAos2bNiu+//x6vvPKK0whYunSpEq7H7Ek5GTJkwJAhQzBq1Ci7axQtWlSxXLLl9uSwKUdkYGtvPY4ElGKV5X0ot33SBIVzeG/Jm4BSbN95qo2A0rVCSw6E4tM/ZAHlPbAPTpbM+FCqAWVkTCwqjTEPlNvP3sLiA6EY0qoMCmbP4Glo0e9eVICA0odAmTlzZmXDC7NUOiZmmWQxIa1+j0bHQNmyZRVL58qVK21VsGV1ZrWcOHGiLc4lCxvEYlQyP0pr6t27N/7666+ACBtEQGlshDhaKD9qUQoDPFgoC+fIgG2fNDV2QYGleKDc/klTFMohZjLR60O56WQYepKFUmDPAjciHqHuZFrydhSVB8r+TUvg45alherOV3bwshigVIsRGxUTi4oCgNJrAlDFuhUgoPQhULJlbbb8vHHjRmUzjjXdu3cPLVq0QM6cOZWd12bSnDlzFP/Hfv36KafvMGhkPpSXL19WApvnzZtXqd4a2Jz5Wnbp0kXxq2RWzUAJbC7ypBQzersr648WSseTctwB5fFrEfht3xX0aVzcLywBPFD++2lTYW3SC5T/nArDuz/TkrfI5+ZmRAzqTP7HrkryoQSWHgjFJ4kWSm8D5aEr99BxnsVCaWZTjhpQPngchwqj1yv1m9mUI3LcUV3mFCCg9CFQHj9+HA0bNgTb/NKsWTNlU05YWJgSiDxNmjTKphwWzsdMYsc4LliwAPPmzVNCBWXKlEnZgc2skxUrVrSrmgU6HzZsmN3RiwxE9SZZg4rf5S3ypBS996s1vxGg9Pbxao4Wyo+DS6F/M9c+lFrvU1Y+fwbKiR0qoEvtwrKkSHbXCYuMQe1JBJSOHfvHwav4eOkR5c8DmpXAR8Hes1CKA0rXhw48fByH8gSUyerZlTX3B4Jo0jflMFFYB7AYkTt27ACzTDJLJQvhwwKMW8P6BIJ4fBtlDarkDJQsqPCuC7fxfqPiSJ8mpdeGAAGls7R6T8rZfDoM7/xkb6EkoDQ3ZMMjY1CLgNJJxGUHr+KjRKAc2KwEBnsRKA9fuYcOiRZKMz6UaocORD+JQ7lRZKE096T4V2lZc79/3bXr1kgFSha2h53TzYKNM4thckqyBpU9UIo7es9bfaHHQumtNjjW6wiUn7QsjX5NS8i6vKnryLRQutuUQ0BpqhtdFg6PikGtiWShdBRHJlD+F3of7efuVJpgZslbDSgfPYlH2VHrlPppyVv8M+SLGmXN/b64N73XlAqUrHFs+Zkdf9i4cWO9bfXr/LIGFQ+UIo/e85a4BJRilfUXoNxyOhw9ftpvd3NkoTTX17eiHqPmxE12lZAPJbD80FUMXmJZ8mbhvViYL28lUUCpdooVAaW3es539cqa+313h9qvLB0og4ODERQUhE8//VR7KwMgp6xBRUBpfjCQhdJZQ72bcracCUePHwkozY/GpBoIKF2rueLwVQxabAHKD5qXxCAvAuWR0Ptol2ihNLPkrXaKVUxsPMqMJAulyOfG13XJmvt9fZ9ari8dKNnJNOyMbHbMIgvbw87Nfu655+zayu/+1nIT/pBH1qDigVLkWc7e0pAslGKVlWmhdHeWNwGl2H5ltd1+8Bg1JpCF0lHZPw9fw4eL/5MClEev3sfLc8wveRNQin8+/LVGWXO/v94/3y7pQJkiRQrb9R1B0vpDfHx8IGhn10ZZg4qA0vzQcLRQftqqNPo2IR9KR2Xd+VBuPROO7mShND8YuRruPHiM6hxQvtewKIa/WE7oNdQq88cPP2tbeaD8MKgkPgzy3pK3KKBUO8XqcVw8So8gC6WUQS3pIrLmfkm3Y+oy0oHyp59+crJIOt5Bt27dTN2ULwrLGlQ8UKp9Bfvi/gNpoiKgdO4tvbu8t529hW4/7LOriHwozT157GziauM32io5O6E10qRK+gA3V7v70v4MlCv/u4YPFlkslN4GymNXI/DSnB3KtcwseasB5ZO4BJQaYYmzTJtyvDmi5dUta+6Xd0fGryQdKI031b9LyhpUPFCKPMvZW+r640RFQOnc22wzCPPhe71mQSzaH6pkcGehJKAU/8Tce/gEVROBMn+29Ng5pJn4i6jU6I/PqbWpPFAOCiqFD4K8FzOWHWTQ9ivzQKl2LGpsfAJKDieglDawJVxI1twv4VZMX8JnQMnO6z527Bhu3LihnFzDAo6zc74DNckaVASU5keII1B+1qoM+jQpbr5iCTV4y4eSnTF85U40mJWsa6Ll0Z0PJTtL2JrPettkoTQ3AO5HP0GVcRYLZb6s6bBraHNzFeoo7c9A+deR6xj4+2HlbtgOb7bT21tJFFCqHYsaF5+AEgSU3uo+n9Qra+73yc3pvKh0oExISMCIESPw1Vdf4eHDh7bmZsyYEf3798eECROQMqX3glrr1EdzdlmDigdKta9gzY2WkNEfJyoCSvWO//fcLbz9vWUpm4BSwgPCXSIiOhaVx20goHSQ3VdAaSYOpRpQxic8RfFha5Q7pCVvuc+Xt64ma+73VvtF1isdKD/66CMFJocMGYKOHTsqu7zDw8OxbNkyTJ06VYHKL774QuQ9SqlL1qDigVLtpSXlhjVeJBCAckjrMujd+Nm2UFq7kwdKd0vefD5rWbJQanwoVLJFPIpF5bEWoMybNR12k4VS0eLvI9cxINFC+VGLUhggyUJpBij//bQpCmbP4NTTCQlPUYyA0tyD4melZc39fnbbLpsjHShz5MihxKD87LPPnBo0efJkTJ8+HXfu3AkE7ezaKGtQEVCaHxqOFkoCyiRNtQLljnO38db3e+06g4DS3NgkoHSt36qj19H/N8uS98fBpdC/mfeWvE9cj8CLs837UBJQmnsWAqm0rLk/EDSRDpTZsmXD0qVL0aJFCyd9NmzYgM6dO+P+/fuBoJ3PgVLtpeVP4pGFUmxveMuHUq+FkoBSbL+y2pgfa6UxZKF0VHb10Rvo99shKUB58nok2sz+V7mWNyyUT58+RdGhtOQt/unxXY0ElEnaSwfKnj17gj1U33//vdMIeOedd5S//fDDD74bHQavLGtQ8RZKAkpjnUUWSnXdtPpQ7jx/G12+IwulsRHoulRUTCwqElA6ibPm2A30/Z8FKD9pWRr9mnovZqwooNzxWVMUeN55yZvdg/Ujm3woRT49vqtL1tzvuzvUfmXpQPnLL79g+PDhKFGiBNq3b2/zoVyxYgUuXLiAiRMnKud9WxPzswyEJGtQEVCaHw0ElOaBctf523iTgNL8YORqePA4DhVGr1f+kidLOuwZRru8mRYygfLUjUi0nmWxUJqJQ0lAKfTR8OvKZM39fi1CYuOkAyV/Uo4ngdhJOoFyao6sQcUDpbuXlidtZf1OS95ilfaXJW8CSrH9ymp7+DgO5QkonYRde+wG+kiyUJ6+GYlWM80vebMYoiyWqKtEFkrxz44va5Q19/vyHrVeWzpQXr58WWvblHyFCxfWld9XmWUNKh4o3b20fKWD43UDASgDKQ7l//ZexvAVxxWZdw1phnwqk5bR/te6KWfXhdt4cwEteRvV2VW56CdxKDfKYqHMnSUt9g4LElm927r88Tm1Nnjd8Rvo/atlydvbx6SKAkp3z6ZVay1Ha/pzv0gbnH5+IVlzv5/LoDRPOlAGgihG2ihrUBFQGukd+zKBfFIOOwv4re/2ouDzGTDjtSrmxXCogYBSuKSaK3z0JB5lR1nOeSagTJJt3fGb6P3rQSlAeeZmFFrO3K5cy8yStzugHPv3CRwIuYdfe9ZG1vSpAxb0NQ/sZJ5R1twfCDISUArqJVmDigdKb1ioBMlhq8Yfv7ADGShF949jfVo35ey+cAdvLNhjV5zCBpnrHR4oc2VOi33DyULJFOWB0turCWfDohD8pXmg3D20GfJmdb3krWeU+OP7U0/7n4W8sub+QNCSgFJQL8kaVASU5juMgFJdQ61AuefiHbz+LQGl+dGYVENMbDzKjLRYKAkok3RZf+Im3l9osVAGClDuGdocebKmMz08CChNS+j1CmTN/V6/EQEXIKAUICKrQtag4oFS1FewIAlcVuOPL0QCSm1A6e6kHAJK8U8NAaVrTTecuIleiUDp7UMIzoVFoYUACyUBpfjnw19rlDX3++v98+0ioBTUS7IGFQ+Uol5agiQIWKD0dmw7b+orum6tPpR7L97Ba2ShFCo/848tPYIslI6ibjwZhvd+OaD8eWjrMnjfi8ekng+PQtAM80vee4c1R+4sZKEU+oD4aWWy5n4/vX27ZkkByuzZs2PTpk2oVq0aWPDykSNHomjRooGgj+Y2yhpUBJSau0Q1o6OFkoAySSqtQLnv0l10nr/bTmPyoTQ3Np/EJaDUiLVKJTkzp8V+8qFUtCCgtIyrkCkvmhtgVNorCsia+73SeMGVSgHKdOnSYfPmzahXrx5YHMo9e/agVq1agm/Ft9XJGlQ8UIr6CvamcoGw5E1ASUDpzWdAa92x8QkoOZyA0lGvTSfD0DPRQjmsTRn0alRcq6S6850Pf4CgGduUcmaOXtw3rDlykYVSt/6BWEDW3B8I2kgBysqVK6NQoULo1KmTYqEcNWoUihUrpqpP165dA0E7uzbKGlQ8UIp6aXlTbAJKb6orvm6tFsr9IXfx6jf2FsoJ7SvgrTqBETdWvHLma4yLT0AJAkonIf85FYZ3f7YseQ9vUxbvNVKfO8z2gjCgHN4cuTLTkrfZ/giE8rLm/kDQQgpQbty4Ed26dcPNmzfBTr9hZ3mrpUA6HYe/B1mDioDS/GNFS97qGhJQmh9fRmvggfKFTGlxYASFDWJabj4dhnd+kgOUF249QPMvLBZKM3Eo9xFQGn0MAq6crLk/EISRApRWIe7fvw/mT7l+/XrFn1It5ciRQ4h233//PWbPno0zZ84gS5YsqFOnDv766y9b3WvWrFHOFT916hQKFCiAwYMHo2/fvoauLWtQ2QGloJeWoRvWWMgfLZRzNp/D9A1nbXdAS95JnXn5zkM0nrZV+cPGQY1QMndmlz19IOQuXiELpcanQFu2+ISnKD5sjZKZgDJJMx4oR7xYFj0b+r+Fkvm/Mj9Ys8kf359m7ym5lZc19weCblKBkgny888/o23bthAFjWoijxkzBl9++aUCjLVr18bdu3exbt06zJ8/Xymye/duNGrUCGx5/a233sLOnTsxevRo5feePXvq7jtZg4oHSlEvLd03q6OAP74Q2eaH/r8dwoaTYcqdEFDad+jqozcQl5CAdlXyq/b0wct30elrWvLW8Sh4zJqQ8BTFCCiddNpyOhw9ftqv/D1QgJJZl9lHgdnkj+9Ps/eU3MrLmvsDQTfpQGkVJSQkRIE4BnrMatmgQQNh53Yzi2PFihXBLJDBwcEu+6F169bKtffuTTqPuFevXli1apUSU5JtHtKTZA0qAko9vaKelw97Q0CpX1MCSv2aeSrBXIGKDiULpaNOW86Eo8ePcoBS1JI3AaWn0Z58fpc19weCYtKBMj4+Hr1798aPP/6IhIQEm0YM4NiGnW+++UY3zDkK/dlnn2HFihU4ezZpWZPP8/jxY2UJfMqUKRg0aJDtp23btqFJkyY4cOAAqlevrqv/ZA0qHihFvbR03ajOzP76hc2Hvfk4uBT6Nyup886e7ewHL98D80XlE23KMT8mrM/LC5nS4MCIFuYr1FiDvz6nrPlbz4Sje4AB5cERQchBFkqNoy+ws8ma+wNBJelAyWJQTp06FWPHjsUbb7yBPHnyKJt1Fi1apCw5Mxhkv5lJjRs3xgsvvIAqVargq6++AvPdrFu3LmbNmqX87eTJkyhfvjzWrl2LVq1a2S5169Yt5MqVCwsXLlSWwfUkWYOKgFJPr6jn5Xcpf9SiFAY0J6DUo+yhK/fQcR4BpR7NtOQloHRWiYDSognFodTyBMnPI2vul39n+q8oHSgLFy6MPn36YMiQIU6tZRbDr7/+GpcvX9Z/J1yJ0qVL4/r168ifPz8mTZqENGnSKJDKltnPv0cBQgAAIABJREFUnTuHEydOKEvszI+SbdSxpri4OKROnVoBz4EDB7ptQ2RkJNg/1nTjxg0ltmZoaKiywcdbiQdKUV/B3morq9dfLR8ElOZ6nYDSnH5qpa3PS46MaXBwJFkomU7bzt5Ctx/2KZJ524dS1JL3oZEtkD1jGtODxF/fn6ZvLBlVQECZ1JnSgZIFOf/777/RooXzy5KFF3rppZcQExNjariVLFkS58+fx/HjxxVLJEsM+NjpPOPGjUP9+vUVoGQB1tmGHUegZDvDBwwY4LYNbNOPK0uqTKAU9dIyJbaHwv76QuR3KZOFUv8IOHzlHjqQhVK/cBqfFwLKJKG2n72FrolA6e04lKKA8vDIFniegFL48+GPFRJQ+hAoy5Urh6CgICWcj2NiVkF2RCNbkjaTGCQyKydbSucTW+5mQdbZsrrZJW9/sFASUBofJfymEgJK/ToSUOrXTEsJ6wcYs26x51tW8tcPP3b/fGxUAkpZI4Kuo1UBAkofAuW3336rbMp57bXX0LlzZ8WHMiwsDEuWLMHixYuVTTnvvfee1r50ma979+5KiCBHoGQwyeJfsmskh005or6CTYmt0eLCsvmTDxC/qWRwi1IYSD6UuobBf6H30X7uTrsytClHl4QuMxNQOsuy8/xtdPnOEo1jaOsyeL+x945eFGWh/G9UC2TLQEve5p8I/6+BgNKHQMkuzTbKjB8/Hrdv37adnJMzZ07lSMZ+/fqZHkF//PEHXn31VRw7dgwVKlRQ6rt27Zpy3OPnn3+ODz/8ECxsENusw/worYmBLgt8Hihhg0S9tEwL7qYCf7V8EFCa6/UjoffRjoDSnIguShNQOouy+8IdvLFgj/LDZ63KoE8T/wfKI6OCkTVDatPjw1/fn6ZvLBlVQEDpY6Bkl2chg06fPo179+4pcSjZRhq9sR/VxiQLTcQ2yERFRWHChAnKphzmOxkeHq6cmpMxY0ZbYHNmzezSpYsSE5MBbSAFNiegNP5W4jeVkIVSv44ElPo101KCgNJZJX4DnbdjxoqyUBJQahntySMPAaUfAKW3hxKDRxZjcvXq1YiNjQULJcROzmHgak0s8PmwYcPsjl40aiGVNaj4Xd6iXlre7At//cLmfQAJKPWPgKNX7+PlObTkrV859yWsz8vzGVLj8CjXhzKIviarz1+fU9Y2masJ58MfIGiG5Szv0S+VQ4/6RVXldtRs3N8n8cPOS0r+I6ODkTU9WSi9MVb9rU5Zc7+/3ber9kjf5R0Iohhpo6xBZQeUgl5aRu5Xaxl/nah4H8BBQaXwQRDFodTapywfAaUetbTnJaB01oq3hn/QvCQGtSilXVCdOQkodQpG2RUXuYIFC3o9ZGAgSE1AKaiXZA0qAkoxHUZAaU7HY1cj8NKcHXaV0KYcc5rylsJsGVLjP7JQKoIevxaBtl9ZxtrAZiUwODhplcm84vY1iFryPjomGFnSkYVSdP/4Y32y5n5/vHfHNhFQCuolWYOKB0pRLy1BErisxl8tlLzVgyyU+kcAP8lbSxNQ6tfRsYT1eSGgTFLm1I1ItJ71r/KHfk2L45OWZcwLrVKDKKA8NiYYmQkovdZP/lSxrLnfn+5ZrS0ElIJ6Sdag4oFS1EtLkAQBBZT8ki0Bpf4RQECpXzMtJaxAyfzvmB+erOSvH37s/s+GRSH4y+2KFL0bF8eQ1gSUssYFXcezArLmfs8t8X0OAkpBfSBrUBFQiukwfsn2w6CS+DDIe35ZYlrsX7UQUHqnPwgonXXlrYa9GhXDsDZlvSM+AFEWyuNjWyJT2lSm2+nPoG/65pJJBbLm/kCQyydAyY48XLp0qeLE6njM4nPPPYeVK1cGgnZ2bZQ1qHigFPXS8qbY/vpC5IHI247+3tTXV3WfuB6BF2eTD6Vo/QkonRUNuf0QTaZvVX54t0FRjGxbTrTstvpEAeWJsS2RkYDSa/3kTxXLmvv96Z7V2iIdKGfNmqWE88mVKxeKFy+uxIh0TFu2bAkE7QgoNfQSAaUGkQIwiyugnNihArrULhyAd+M/TSagdO6L0LvRaDjVMid0r1cEY14u77UOI6D0mrTJtmICyqSulQ6UbHt9hw4dMHPmTGGBzP1hpMoaVGShFNPbZKE0p+PJ65FoM9uyUcKaCCjNacpKW4EyS7pUODqmpfkKNdbgrx9+rPnX7z9CvSmblTvpWrcwxrWznH7mjSQKKE+Oa4kMaWjJ2xt95G91ypr7/e2+XbVHOlDmyJFDObe7efPmgaCP5jbKGlQElJq7xG1G3sJGS976NSWg1K+ZlhIElM4qhUfGoNakf5QfutQuhIkdKmqR0lAeAkpDsj3ThWTN/YEgsnSg7NOnj3L04fTp0wNBH81tlDWoAg0om0zbgpA70YqOIVNe1KyntzPyQDSweUmw03IoaVeAD+ViLUUWSu36qeW0AmXmdKlwjCyUiky3HzxGjQmblP9+o1YhTO7o/0B5alwrpE+T0vSA8GfLsembSyYVyJr7A0Eu6UD5+PFjvPvuu8pZ3kFBQciWLZuTTh07dgwE7ezaKGtQBRpQXr7zEF9uPIsudQqjZpHsftOvBJTmuuL0zUi0mklL3uZUdC5NQOmsyf3oJ6gybqPyw2s1CuLzVyqJlt1Wn6iTck6Pb4V0qQkovdZRflSxrLnfj25ZtSnSgfLw4cOKD+WVK1dcNort8o6Pjw8E7QgoA66XkhrMW9jIQqm/Iwko9WumpYQNKNOmwrGx5EPJNIuMiUWlMRsU+V6pXgDTX62sRUpDeQgoDcn2TBcioEzqfulAWa1aNcTFxWHy5MkoVaqUy13ehQsH3k5RWYMq0CyU/vqm4YGIgFJ/LxFQ6tdMSwkCSmeVop/Eodyo9coPHavmx4zXqmiR0lAeUT6UZKE0JH9AFpI19weCONKBMkOGDFi+fDlatWoVCPpobqOsQUVAqblL3GY8czMKLWdaTt/w9vnAYlrsX7W4CmxOPpTm+4iA0lnDmNh4lBm5TvmhXZV8mPV6VfNCq9QgCijPTGiFtKloydtrHeVHFcua+/3ollWbIh0o69Spg759+6Jr166BoI/mNsoaVASUmruEgFKMVC5rOXzlHjrM22X3GwGlecEJKJ01jItPQInha5Uf2lbKizlvVjMvNAGl1zR81iqWNfcHgq7SgfLQoUPKppypU6eiadOmSJXKfKwufxBa1qDaciYcPX7cr9zy2QmtkSZVCn+4/YBrA38+MFko9Xff/pC7ePWb3QSU+qVzW8IKlOzYPnYSlqzkz7uJnz59iqJD1yhStKmYB/O6VPeaLGSh9Jq0ybZiWXN/IAgoHSgzZ86M2NhY5Z8UKVIgffr0djqxTTkRERGBoJ1dG2UNKvZyXfnfdeR/Pr1f7ZoOtA47FxaFFl9alrwHNCuBj4JLB9ot+LS9u87fxpvf7SWgFNwLBJSuBbXq0rJ8bsx/u4Zg1ZOqI6D0mrTJtmJZc38gCCgdKMeMGQMGje7S6NGjA0E7nwBlwAnjpw0+Hx6FoBkElEa7Z9vZW+j2wz4CSqMCqpSzglPGNClxYpw8P/Pxq07i+x2XkD1jGhwa2ULwXZmvzqpLi3K5saCr94AyKiYWFRN3lK8Z2BDl8mVRbbyjVXfc3/9v72zAbKr2P/4zZiTCxZheuDF5CSMqjLfxWhmTl+I+3ecWycuEiBvVPyGMiFL3xr2FUl1XdHuKrrkuFfJ+caOUwjS94TKMyDDGDDP8n+/SPp0zzpk5c87a5+x99nc9zzweM3v/9t6f9dtrffdv/dZae+XNrT+o45lDGXyd28UCBeWvNRVyQWkXJynrfdKpykosvMe7C8pHuzaQJ5IZoSxLjazbd0yGLtpJQVkWaH4cGy5BWVBYJGv3Zkvr+OoSV6WiH3ca2kMMLnc0jpM3BrU29eJ7/pcjEJbtG8SWeB0KSlOrwTbG2fdTUGp3VjqVdqSmGnRfb46Csuyos8/kS+KMy9vhGYWTcsrOsfgZhkipVKG87A1hhDL4OzfXgsGlc6NasmhIorkX89N6cUH52cGfpd8vE9W+f+5uiYoqeSTOn8tYObfVn/t3wjHs+8MoKLt161aqj33yySelHmO1A+hUVquRku/HPVeKgjKwusPSS5jcNPqdz5UBCsrAOLqfRUHpnaHBpWPDWFk8tE3woDVY8Cb2dh86JbHXVJA61StpuIIIBaUWjKYaYd8fRkHZq1evK3IoT548KV9++aXahvG2226T9PR0Ux3ADON0KjOommfz++O50u2ljeoCFJSBc3Zf4JyCMnCOxpkUlN4ZYi9v7Om9eGiidGxYK3jQGiyEQuyF4hoaUDjaBPv+MApKX56XnZ0tffr0kccff1zuu+8+2zkoncpeVeYuKEd1rS9PJje21wNY5G4pKPVWBAWld57IaTyaky8Nr62iF3gQ1kIh9kJxjSAQ8FQRYd9vQUGJW/rggw9k/PjxkpGRYTtHpVPZq8p++OmsdH1xg7ppCsrA646CMnB23s40BMTVMeVl37Ohm+Wt9ymcYS0UYi8U13BGbZn3lOz7LSooly1bJoMHD5bTp0+bV/smWaZTmQTWJLM//nRWulBQBk2XgjJohB4GKCj18jTTWijEXiiuYSYjJ9hm3x9GQYmdcoqX8+fPy759+yQtLU0aNGgga9eutZ0f0qnsVWUHTpyVzrMZoQy21igogyXoeb4hICrGRMn+Z1P0Gqc1rQRCIfZCcQ2tUBxojH1/GAUldscpvrA5dn9BadOmjSxdulTi4+Nt55Z0KntV2cETedJp9np10yO71Jf/68EcykBqkIIyEGq+z6Gg1MvTTGuhEHuhuIaZjJxgm31/GAXlxo2XZ9a6l4oVK0qdOnWkdu3apvhfbm6uNG7cWA4fPiyffvqptGr1604Lq1atkokTJ6oIKe5h3LhxMnLkyDLfB52qzMjCegIFpR78FJR6OBpWDAFxVXSUZExnhFIvXb3WXtv0nby8NlPmD2gpnRqZM/OcglJvnZlhjX1/GAWlGRVams2nnnpKFi1aJMeOHfMQlNu2bZNOnTrJwIEDZcCAAbJ161bBto8LFiyQ1NTU0sx6/J1OVSZcYT/40Mk86fgCI5TBVgQFZbAEPc+noNTL02xrhUUXJbp8lGmXoaA0Da02w+z7HSQo9+/fryKSL730kowYMcJDUKakpAjWwNyxY4eLyLBhw2TlypVqKQAMz/tb6FT+krLGce6C8pEu9eUpDnkHVDEUlAFh83kSBaVenna3RkFp/Rpk3x9iQVmlSpUr8iZ9uQnyK3NycrR5Uffu3aV58+aCBdW7du3qEpQFBQVStWpVmTVrlowdO9Z1PQzJd+nSRXbu3CktW7b0+z7oVH6jssSB//s5T5KevxyhpKAMvEooKANn5+1MCkq9PO1ujYLS+jXIvj/EgnLq1KmlCsotW7bIunXr1HFFRUVavOj999+XUaNGSWZmpmB2ubug3Lt3ryQkJMjq1aulR49f13s7fvy4xMXFyeLFi9UwuL+FTuUvKWsc5y4oR3SuL+NTOCknkJqhoAyEmu9zDAFRITpKvmEOpV64NrRGQWn9SmPfH2JBWZJLIG8RghNiEpHEyZMnS79+/YL2ory8PDURB7aHDBkiGzZs8BCUuG5SUpIgj7Jt27au6xUWFkpMTIzMmTNHxowZ4/M+sFam+3qZWVlZkpiYKIcOHVKTe1isTeDwqXPSYdblPeMpKAOvKwrKwNmVFKGkoNTL1a7WKCitX3MUlBYQlJs3b1brTq5fv15atGihhOS9996rzXsmTJgga9asUfmRyIX0JSi3b9+ulisyiiEo586dK6NHj/Z5PxCquP/ihYJSWxWaaujIqXPSnoIyaMYUlEEj9DDgilCWj5JvZnCWt1669rNGQWn9OqOgDKOg3LRpk4oaQuDdfvvtSkhiD2+d5cCBA9KoUSO1lWP79u2VaQyp9+7dWwlYTNI5ePBgUEPejFDqrLHQ28rKOSftZjJCGSx5CspgCXqeT0Gpl6fdrVFQWr8GKSjDICghIBHRw6SX1q1bKyHZs2dPU7zFiEb6Mo6IJO6Dk3JMwW8Lo+6Ccnjnm+TplCa2uG+r3SQFpd4aMQRETPlykjnjbr3Gac12BCgorV9lFJQhFpSYNY0hbgg5CEn3STBmuMupU6dk9+7dHqbxf8zmnj9/vhK0iI5i2SAcizxKo2BpofT0dC4bZEbFWMjm0Zx8aTtznbojCsrAK4aCMnB23s6koNTL0+7WKCitX4MUlCEWlMZ6jpUrVy51trfuZYOMRy2eQ4nfGwubDxo0SPr3768WNofg5cLm1n+Jg73DY6fzpc1zFJTBcqSgDJag5/kUlHp52t0aBaX1a5CCMsSC0tvklZLcBLvV6C7eBCWuga0XMYHHfetFLDVU1kKnKiux8B6ffTpfEikog64ECsqgEXoYoKDUy9Pu1igorV+D7PtDLCit7xLB3yGdKniGobTgISg73SRP380cykD4U1AGQs33OYaAiI4qJ98+xxxKvXTtZ42C0vp1xr6fglK7l9KptCM11WD2mXxJnPHLkDcFZcCsKSgDRuf1RApKvTztbq3n3M3y9ZHTUrdmJdn4ZFe7P05E3j/7fgpK7Y5Np9KO1FSDx88USOsZa9U1hlNQBsyagjJgdCUKyvJR5eQ7Rij1wrWhNbRTK3Yfll7Nb5DrqlW04RNE/i2z76eg1O7ldCrtSE01+FNugbSafllQDut0k0zgkHdAvCkoA8Lm8yQjQklBqZcrrZGAWQTY91NQavctOpV2pKYapKDUg5eCUg9Hw4ohKKPKiXw/05x1evXeMa2RgLMJsO+noNT+BtCptCM11eCJ3AJp+UuE8uGO8TKxZ1NTrxepxiko9dYsBaVenrRGAmYTYN9PQandx+hU2pGaavDk2fNy+7Nr1DUoKANHTUEZODtvZ1JQ6uVJayRgNgH2/RSU2n2MTqUdqakGfz57Xm6joAyaMQVl0Ag9DFBQ6uVJayRgNgH2/RSU2n2MTqUdqakGT+Wdl1unMUIZLGQKymAJep5vCMpy5UR+YA6lXri0RgImEGDfT0Gp3a3oVNqRmmrQXVCmJsXLpF7MoQwEOAVlINR8n0NBqZcnrZGA2QTY91NQavcxOpV2pKYazMm7IC2mfayuQUEZOGoKysDZeTuTglIvT1ojAbMJsO+noNTuY3Qq7UhNNZhz7oK0SKOgDBYyBWWwBD3P51Z7ennSGgmYTYB9PwWldh+jU2lHaqrB0/kXpPlUCspgIWccPSPJL29SZmb2u0XuT7wxWJOOPt8QlBXKR8k3M1IczYIPTwJ2IMC+n4JSu5/SqbQjNdWgu6AcmhQvzzCHMiDeFy9ekj6vbJGfz16QNeM6SaUK0QHZ4UmXCbz4UYa8vvl7eWtwa2lfP5ZYSIAELE6AfT8FpXYXpVNpR2qqwTP5F+SWXyKUQzrEy+TenJQTKHCIyqJLlySmfFSgJnieG4HzhRelQjRZ0ilIwA4E2PdTUGr3UzqVdqSmGswtKJRmUz5S16CgNBU1jZMACZBAxBJg309Bqd256VTakZpq8GxBoSRQUJrKmMZJgARIINIJsO+noNTu43Qq7UhNNUhBaSpeGicBEiABRxBg309Bqd3R6VTakZpqMO98oTSdfHnIe3CHejKld4Kp16NxEiABEiCByCPAvp+CUrtX06m0IzXV4LnzRdJk8ocUlKZSpnESIAESiGwC7PspKLV7OJ1KO1JTDeZfKJLGz1BQmgqZxkmABEggwgmw76eg1O7idCrtSE01SEFpKl4aJwESIAFHEGDfT0Gp3dHpVNqRmmrQXVAOal9PpvZhDqWpwGmcBEiABCKQAPt+Ckrtbk2n0o7UVIMFhUVy86TLQ94UlKaipnESIAESiFgC7PspKLU7N51KO1JTDWI3kkaTVlNQmkqZxkmABEggsgmw76eg1O7hdCrtSE01SEFpKl4aJwESIAFHEGDfH+GC8r333pMlS5bIrl275OTJk1K/fn155JFHZPjw4RIV9eseuatWrZKJEyfKvn37pE6dOjJu3DgZOXJkQC8BnSogbGE76ULRRWk4kRHKsFUAL0wCJEACEUCAfX+EC8q2bdtK3bp1pW/fvnLttdfK+vXrZebMmfLYY4/J7Nmz1dNv27ZNOnXqJAMHDpQBAwbI1q1bZcqUKbJgwQJJTU0ts5vTqcqMLKwnFBZdlAa/CMqH2tWVtHuahfV+eHESIAESIAH7EWDfH+GC8vjx41KrVi0Pz0T0cd68eXLq1Cm56qqrJCUlRUUvd+zY4Tpu2LBhsnLlSoGDuEcy/XFxOpU/lKxzTNHFS1J/wip1QxSU1qkX3gkJkAAJ2IkA+/4IF5TenHHx4sUqGnnkyBGpUaOGVK1aVWbNmiVjx451Hb5x40bp0qWL7Ny5U1q2bFkmn6ZTlQlX2A+moAx7FfAGSIAESMD2BNj3O1BQIvq4bNkyyc7OloyMDElISJDVq1dLjx49XDQQ2YyLixOITwyDl6XQqcpCK/zHXrp0SeKfvhyhXJLaRjo0iA3/TfEOSIAESIAEbEWAfb/DBCUiju3atVM5kpMmTVL5kklJSSqPEvmWRiksLJSYmBiZM2eOjBkzpkSnPn36tODHKFlZWZKYmCiHDh1SE3xYrE/gu+O5knUqX5IaUkxav7Z4hyRAAiRgPQIUlA4SlEePHpU2bdookbdhwwYlGA1BuX37dvW34oJy7ty5Mnr06BI9d+rUqZKWlnbFMRSU1nvheUckQAIkQAIkYAYBCkqHCMqcnByVE5mfny9btmyRmjVrqiffu3dv0EPejFCa8WrSJgmQAAmQAAnYhwAFpQMEJURkcnKyZGZmqqFtLCNklIKCAk7Ksc/7yjslARIgARIgAUsSoKCMcEGJXMh+/frJpk2b1E/z5s2vcEQsG4QlhCA2jTJixAhJT0/nskGWfG15UyRAAiRAAiRgLQIUlBEuKLEjzmuvvSYvvPCCdOzY0cP7mjZtqqKTxsLmgwYNkv79+6u8ysmTJ3Nhc2u9q7wbEiABEiABErAsAQrKCBeU9erVkwMHDnh1QOyag7xKFGy9OGHCBI+tF0eNGhWQ49KpAsLGk0iABEiABEjAtgTY90e4oAyHZ9KpwkGd1yQBEiABEiCB8BFg309Bqd376FTakdIgCZAACZAACViaAPt+CkrtDkqn0o6UBkmABEiABEjA0gTY91NQandQOpV2pDRIAiRAAiRAApYmwL6fglK7g9KptCOlQRIgARIgARKwNAH2/RSU2h2UTqUdKQ2SAAmQAAmQgKUJsO+noNTuoHQq7UhpkARIgARIgAQsTYB9PwWldgelU2lHSoMkQAIkQAIkYGkC7PspKLU7KJ1KO1IaJAESIAESIAFLE2DfT0Gp3UHpVNqR0iAJkAAJkAAJWJoA+34KSu0OSqfSjpQGSYAESIAESMDSBNj3U1Bqd1A6lXakNEgCJEACJEAClibAvp+CUruD0qm0I6VBEiABEiABErA0Afb9FJTaHZROpR0pDZIACZAACZCApQmw76eg1O6gdCrtSGmQBEiABEiABCxNgH0/BaV2B6VTaUdKgyRAAiRAAiRgaQLs+ykotTsonUo7UhokARIgARIgAUsTYN9PQandQelU2pHSIAmQAAmQAAlYmgD7fgpK7Q5Kp9KOlAZJgARIgARIwNIE2PdTUGp3UDqVdqQ0SAIkQAIkQAKWJsC+n4JSu4PSqbQjpUESIAESIAESsDQB9v0UlNodlE6lHSkNkgAJkAAJkIClCbDvp6DU7qB0Ku1IaZAESIAESIAELE2AfT8FpXYHpVNpR0qDJEACJEACJGBpAuz7KSi1OyidSjtSGiQBEiABEiABSxNg309Bqd1B6VTakdIgCZAACZAACViaAPt+CkrtDkqn0o6UBkmABEiABEjA0gTY91NQugh88803MmbMGNm8ebNUrlxZ7r//fpk1a5ZcffXVZXJiOlWZcPFgEiABEiABErA9Afb9FJSKwKlTp6RZs2ZSt25deeaZZyQ7O1vGjRsnPXr0kLfffrtMjk6nKhMuHkwCJEACJEACtifAvp+CUhF4/vnnZdq0aXLgwAGJjY1Vv1u6dKn0799f9u7dK02aNPHb2elUfqPigSRAAiRAAiQQEQTY91NQKgKdO3eW3/zmN7JixQoXkYKCAqlWrZrMmDFDHn/8cb8dnk7lNyoeSAIkQAIkQAIRQYB9PwWlIhAXFydDhgxROZPuJSEhQdq1aycLFy702+HpVH6j4oEkQAIkQAIkEBEE2PdTUCoCMTEx8uyzz8r48eM9HDspKUmJzeXLl/t0+NOnTwt+jHLo0CFp3769/Pe//5Xrr78+Il4UPgQJkAAJkAAJkIBvAllZWZKYmCg//PCD1KtXz9Goyl26dOmSUwlAUE6fPl2eeuopDwQdOnSQ6667TpYtW+YTzdSpUyUtLc2p6PjcJEACJEACJEACvxBAMKl169aO5uFoQRnMkHfxCGV+fr4gShkfHy/R0dFancr4AmL081esZOLpYuRx5StHJmTiT0NMP6GfBOMnhYWFcvz4cbnlllukYsWK/piK2GMcLSh1Tsox00OYo3ElXTLxZEIe9BF/2iD6Cf2EfuIPAfpJIJQcLSixbBByKLFsUM2aNRW/f/zjH2px87IuGxQIfH/PYSfAl7s0X6GP0EdK8xH8nX5CP6Gf+EOAfhIIJUcLSmNhcyTSui9snpycXOaFzQOB7+857AT4cpfmK/QR+khpPkJB6Z0Q3x2+O3x3/CFQ+jGOFpTAg60XR48eLVu2bJFKlSqp6CQil2XderF01IEfgXzNP/3pT2oXn6pVqwZuKILOJBPPyiSPK52bTMjEnyaPfkI/oZ/4Q6D0YxwvKEtHxCNIgARIgARIgARIgARKIkBBSf8gARIgARIgARIgARIIigAFZVD4eDIJkAAJkAAJkAAJkAAFJX2ABEiABEiABEiABEggKAIUlEHhM/dkTBgaM2YHTXtEAAAOV0lEQVSMbN68WSpXrqwmDGHfcStNGNJB4L333pMlS5bIrl275OTJk1K/fn155JFHZPjw4RIVFeW6xKpVq2TixImyb98+qVOnjpqkNHLkyCtu4cUXX5S//vWvcvToUbXY7OzZs6VLly46bjVsNnJzc6Vx48Zy+PBh+fTTT6VVq1aO5fLGG2/I3LlzJSMjQ01Sa9u2raSnpzuSxz//+U+ZOXOmeifQLmCXL/z/5ptv9vDVSH13vv32W8H7vn37dvnqq6/UO4J/ixedz3/mzBl54okn5P3335eCggLp1q2b/OUvf5G6deuGrX1wv3BpTIqKiuSll16Sf//732p5PCzMjXZyypQpcscddwTUntqdSfGHRl+E7RTxTqHtLV786WOszsQMZ6WgNIOqBpvGkkZopNyXNOrRo4elljTS8KhKEOA5+/btK9dee62sX79edYqPPfaYEoMo27Ztk06dOsnAgQNlwIABsnXrVtUALliwQFJTU123gRd9woQJ8txzz8ntt98ur7/+uqDTxS5DaDTtWrA96KJFi+TYsWMegtJpXLDl6Z///Gf1YdGmTRv1AfLhhx8qP3Can6xdu1a6d++u3ocHH3xQ0GaAD2Ytf/31164VISLZR1asWCGPPvqo8gV8gF+8ePEKQan7+Xv16iWfffaZEmX4oJk8ebJi/uWXX1riY780JhBI+CB/6KGH5K677hJsQfy3v/1N3n33XfVhhuczir/tqd2ZuPcL2I26ffv2am9usCouKCOFiRl9IQWlGVQ12MTSRdOmTVOLrsfGxiqLS5culf79+1tq0XUNj6q2rapVq5aHKUQf582bpzrJq666SlJSUpR42LFjh+u4YcOGycqVK9VizYhkIloAQYrfv/DCC+o4fI1DSDZv3lwtWm/Hsn//fhWRRAc2YsQID0HpJC6IwqEuEW2CkPJWnMQDH1Lr1q2T77//XsqVK6dw4MMJ4gqMwAIlkplAQBqjGIMGDZKdO3deISh1Pj/aH3wAI7p39913K74HDx5UoyqIUuL9DHcpjQnaRAjg6tWru24VIgptDAQyPuhR/G1PI4GJe529+eabKqBx3333qZEQd0EZSUzM8FMKSjOoarBpl20hNTyqVxOLFy9W0cgjR45IjRo1VEOH4f6xY8e6jt+4caMaykYn0rJlS9UQYvgJ0YPbbrvNdVxaWpoSYzk5Oa6O16z7NsMuxBMEMaIAXbt2dQlKNG5O4oIo7QcffKAiUd6K03ggwvT555+ryJhRwAbD3YbgcRITb4JS9/NjVAQiAx+3hogHe7yX11xzjfzrX/8yowkI2KYvke3N4NChQ9V6zEglQfG3PY0kJghgNGrUSCAq0a8gGukuKCOVScAOVuxECkpdJDXbiYuLkyFDhigR5V4SEhKkXbt2snDhQs1XtJY5RBmXLVsm2dnZqoHDc69evVow5G8URDbBCeITw36vvvqqjBo1SvLy8jyGnpCj+fvf/14OHTqkhnrsVJCnhWfKzMxUQtldUCL/yUlc8JGFaP2tt96qokFo/PEuzJkzR/3OaTw2bdqkct6w6YEx5I2ca+TQQWgisu8kJt7Ek+7nRzuCiCRyNt0L3tGPPvpIsbdS8VdQIqqJtgQfI0gRQvG3PY0kJkif+O6771Rfg/SR4oIyUpno8lkKSl0kNdtBXgv2GR8/fryH5aSkJCWili9frvmK1jGHL0MIBXz5Tpo0SeVL4rmRC4XhJqMgmRycICjQkc6YMUMxy8/P93gY5JohV+iLL75QkT67FAhjTDJAw4aPiw0bNngISqdxQWeHiHXt2rVVjmyFChUE0ecff/xRCW7kDTrNT5Dy8cADDwgmAKA0bdpUCRvjw8lJPuJNPOl+frQj5cuXV3m77gXtFMQGIpdWKv4KSrShGP1BG4NcdRR/29NIYbJ7927V7+BjzGh3iwvKSGWiy2cpKHWR1GwHQmn69OmCYT73glmc1113nYreRWLBzGzkgKFDROMGDkangKgA/lZcUGIICttn4mUHs3PnznmgWbNmjcq5w9CgnSbmYHIR7h05SsgT8yUoncKlYcOGKgKEWbyIpqBkZWVJfHy8yjfGuwFB6RQe//nPf1QeH0RDnz59VEoHhDY+qPDOIB3CSe9OSYJSl09APEVHR6sIlnvBJLH58+fLiRMnLNUs+yMokTqE9vGPf/yjK/fcEJT+tKeRwAQ5pB07dlT9C9KjULxFKP3tY+zGRJfTUlDqIqnZjhOHvNEhIicSHSJyeWrWrKmolnXYCoKyYsWKrhqx45A3JmMhlwc5g5hxiAImvXv3VrlNSKDH0FtZhrztzgWNPbjgo8O9YLi7RYsW6uPLSTzgAzfeeKPHaAXSQPAxhkkFmNjmpHdHx5B3ae9IJA3v4h3CRzYiksnJyWrSonteqDG86wQmeHYsQYfoZLVq1VTzgnQzLD+HSZ/oT/ATqUx0yRcKSl0kNdtx2qQciEg0ahi6xNC2+5puuhPrNVeVKeaMaKQv4xBXiCw4aVIOBAOGGosLSohJLBGFCJGTeFSqVEmlxGDZGveCiC0il6+88oqaqesUJpyUc2VrUVKEErmCiOg3adJEvVdIIXEvkToBxRsTRCORPuOr4GMVAjNSmejqxCgodZHUbAfLBiEfEBEZI1KHrygsbo6oAxqBSCnIhezXr59gkgF+vOU5YukPTMKA2DQKlujAumnFlw3C743JTFgiA/Yw1G2nZYPwrMjpcS/4P/KcIJxat26tRJSTuGCCEpby2LNnjzRr1kyhwULvN910k+B9wbqlTuKBNgBRbKw7aBSIbUQtMTT35JNPql87hUlJywbpajuMJXLcJwhish980CrLBrm3Gb6YwE+QIoKPDePDtHh/YiyRU1p7GglMkIeNH/dirM2JusY7hTqOVCa6tAQFpS6Smu0YC5vXq1fPY2FzRPHefvttzVcLrznsiPPaa6+p/B3ksbgXTDJAo2csTowGEmtxIjcMkRlfC5tjyA+CC7PhMYHJ7gubg0nxHEr8zklc8HGA3SswAQW5XYioIHfSWAkAu0k5iQeG45A7jBnG99xzj/rgQg4lPkIxQen6669Xr1IkM8HENay5iYKILKJumPWOglEerG+r+/mxfBeGRt0XNke6jlUWNi+NCZY3wuQTsEJfgrV73Yv7xEdjEe/S2lO7Mym+DjJ4eMuhxO8jhYkZvT4FpRlUNdnEmnLoMJA7h+EtRCcRiYm0rRchmtEJeisYYjC2TUTHgYkq7lsvojN1L0iuNrbFwq4yiExCqGK5HbsXb4ISz+QkLhCPiNJincULFy4o0YCdc9y3GnQKD/g6doJCXhcmK0EoQHAjOll88lmkMkFUCUP8oWw7sCi4sfXi+fPnLbf1YmlM0N76YgaO8Cuj+Nue2p2Jt615fQnKSGFiRn9IQWkGVdokARIgARIgARIgAQcRoKB0UGXzUUmABEiABEiABEjADAIUlGZQpU0SIAESIAESIAEScBABCkoHVTYflQRIgARIgARIgATMIEBBaQZV2iQBEiABEiABEiABBxGgoHRQZfNRSSCSCBRfjBjrtWIPXqwEgIW9zSi+ZtqbcS3DJnYvmT17tppZ7Kvs3LlTrU3qviqCmfdE2yRAAiRQnAAFJX2CBEjAlgQgKLEk1CeffKLu/8iRI2pB+127dsnmzZtdW1bqfDgsj4KNBbAsD9a9DEXBPtTYOcpYV9LbNSkoQ1ETvAYJkEBJBCgo6R8kQAK2JOBtnbisrCypXbu2PPzww2rRe6cUCkqn1DSfkwSsS4CC0rp1wzsjARIogYCvhYex8wd2ScKWaUbBbikTJ04UbBMXHR0tPXv2lJdfflni4uJcx2ALT2wzt27dOqlRo4bayhEL7q9cudK1LZu3IW/sQw/b2Nrzp59+UtshPv300/LAAw+4bBtb4GF3GyzMjk0LEhISZN68edKyZcsS69nbkDd2CoKt3Nxc6d69u6Smpqpn4pA3XxkSIIFwEaCgDBd5XpcESCAoAt4EJQRW9erVBdt5QnChQExiJwzkVQ4dOlTOnj0rkyZNEuRcYjgZBbtftGrVSrC7EraZq1atmhpOh6AsX758iYLyd7/7ndqtCCIPe4xDWGIf4L///e/y4IMPKvsQlBCmN9xwg4wfP15tJ4p/MYSOLfBiYmJ8siguKI0tF5FTeeedd8rHH38s7777rtrXnIIyKJfiySRAAkEQoKAMAh5PJQESCB8BQ1BiD2sUDHdDpH344YdKKDZs2FD9HtszFhYWqi1MIc5QsNc18iAh8iA0IQgR4du0aZNrP3mIvTp16qhoJbazQykeocT+zS1atFD7SI8cOdIFIzk5WTIyMlznQVBCYO7Zs0dFJlHWrl0rd911l8r3TEpK8ktQYj/zG2+8Ue644w5lzyiIhr7zzjsUlOFzR16ZBBxPgILS8S5AACRgTwLFZ3njKRBNTE9Pd83yzsvLU9FA7O/+6KOPejwoJroMGzZMpkyZImlpaWoI/Oeff/Y45p577pEvvvjCp6CEkIRdDHUj4mkURCgHDx4sBw8elN/+9rcqQgkBiWF1o2ASEfI9IQT/8Ic/+CUoETHFXszLly+Xvn37us7B/xEpZYTSnr7MuyaBSCBAQRkJtchnIAEHEjBmeSOqePHiRcnMzFQRSojIr776Ss2KxjAwooy+CobAFy5cqHInMVscuY3uBZN71qxZ41NQYpgbYvTChQse5yFKmpKSIrt371YRTCOHEvdlFIjQWrVqyVtvvaX+7qu4D3kj8tquXTsVbe3QoYPrlK1bt6ooJwWlA18EPjIJWIQABaVFKoK3QQIkUDYC3nIosWRQYmKiijxiwgvyJatUqaLWprz33nuvuEBsbKyK+AUaoXz11Vdl1KhRcuLECTU0bhRvEUrMxA5WUDJCWTYf4dEkQAKhI/D/maE6SnLC/jwAAAAASUVORK5CYII=\" width=\"599.9999869953505\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Plot number of probes per region\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(list(reg_size_dic.values()))\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"Num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"Region id\", labelpad=1)\n",
    "plt.ylim([0, np.max(list(reg_size_dic.values()))*1.1])\n",
    "plt.savefig(os.path.join(report_folder, 'afterblast_nProbes_per_region.png'),\n",
    "            transparent=True, dpi=300)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5d0d61c0",
   "metadata": {},
   "source": [
    "## number and location on each chr"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 164,
   "id": "6e65ec21",
   "metadata": {},
   "outputs": [],
   "source": [
    "chrs = []\n",
    "chr_2_centers = {}\n",
    "chr_2_types = {}\n",
    "for _loc_name, _type in zip(region_info_df['locus_info'], region_info_df['type']):\n",
    "    _chr = _loc_name.split(':')[0]\n",
    "    _start, _end = _loc_name.split(':')[1].split('-')\n",
    "    _center = (int(_start)+int(_end))/2\n",
    "    if _chr not in chrs:\n",
    "        chrs.append(_chr)\n",
    "    # create\n",
    "    if _chr not in chr_2_centers:\n",
    "        chr_2_centers[_chr] = []\n",
    "        chr_2_types[_chr] = []\n",
    "    # append\n",
    "    if _center not in chr_2_centers[_chr]:\n",
    "        chr_2_centers[_chr].append(_center)\n",
    "        chr_2_types[_chr].append(_type)\n",
    "# region number\n",
    "chr_2_region_num = {_chr:len(cts) for _chr, cts in chr_2_centers.items()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 165,
   "id": "9f5911e4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load chromosome sizes\n",
    "import pandas as pd\n",
    "from ImageAnalysis3.structure_tools.distance import sort_chr\n",
    "\n",
    "sorted_chrs = sorted(chrs, key=lambda _c: sort_chr(_c))\n",
    "\n",
    "chrom_size_df = pd.read_csv(r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\hg38.chrom.sizes',\n",
    "            delimiter='\\t', header=None)\n",
    "\n",
    "chr_2_len = {}\n",
    "for _chr in chrs:\n",
    "    chr_2_len[_chr] = chrom_size_df.loc[chrom_size_df[0]==f'chr{_chr}', 1].values[0]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 166,
   "id": "932982a9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAxwAAAIqCAYAAACuU0wMAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjUuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8qNh9FAAAACXBIWXMAABcSAAAXEgFnn9JSAACH40lEQVR4nO3deXxMV/8H8M9NJBNGrNkIEYmtsaRUELWklqIoYimlRVHL0wXV6tNfPWi6eErRvVpatKUPaqlSfbT2iiCKBqESsdSSCCozkYTk/P7IMyOTzCSz3Jk7y+f9es3LzDn3fs/33jsTc+aec68khBAgIiIiIiKyAy+lEyAiIiIiIvfFDgcREREREdkNOxxERERERGQ37HAQEREREZHdsMNBRERERER2ww4HERERERHZDTscRERERERkN+xwEBERERGR3bDDQUREREREdsMOBxERERER2Q07HEREREREZDfscBARERERkd1UUjoBdxISEgKtVouwsDClUyEiIiIiks2FCxegVqtx9epVi9flGQ4ZabVa3L17V+k0iIiIiIhkdffuXWi1WqvW5RkOGenObJw4cULhTIiIiIiI5NO8eXOr1+UZDiIiIiIisht2OIiIiIiIyG7Y4SAiIiIiIrthh4OIiIiIiOyGHQ4iIiIiIrIbdjiIiIiIiMhu2OEgIiIiIiK74X04iIiIiJyIEAJCCKXTIDclSRIkSXJom+xwEBERESmssLAQ2dnZyMnJQUFBgdLpkJvz9vZGlSpVUK1aNfj7+9u9A8IOBxEREZGCCgsLceHCBeTl5SmdCnmIwsJC5OTkICcnBzVq1EBwcDC8vOw304IdDiIiIiIFZWdnIy8vD97e3ggODoZarbbrlz/ybEII5OfnIycnBzdu3MCtW7fg5+eHmjVr2q1NdjiIiIiIFJSTkwMACA4ORvXq1RXOhjxBlSpVUKVKFVSqVAmZmZm4efOmXTscHtF9jo+PhyRJmDlzZrnLDRs2DJIk4aWXXnJQZkREROTJhBD6ORtqtVrhbMjTVKtWDQCQn59v1wsVeESH4+OPP0b16tXx3nvv4ciRI0aX2bx5M9auXYuGDRsiISHBwRkSERGRJyr5JY/DqMjRvL299c/Z4bBRnTp18O6776KwsBDjxo3DvXv3DOpzcnIwZcoUAMCSJUtQpUoVJdIkIiIiInI7HtHhAIAJEyYgLi4OR48exYIFCwzqXn31VVy6dAmjR49Gz549FcrQ/rRaQJKKH5mZ95/rHlpt+cvp6iuKZyx2ee2ZilnecpZsqzXrExEREZE8PKbDIUkSPv/8c/j5+WHu3Ln4888/AQCJiYn47LPPEBQUhIULFyqcJRERERGRe/Goq1Q1btwY//rXv/Daa69hwoQJ+PnnnzF+/HgUFRXh/fffR61atZRO0WZaIz/nlzxzoZOaWnbdkyeBgAAgN/d+2fnz959nZgJqdXEc3b/G4hmL/fvvZcsyMoDAQMOyrKzyl6toPp2xbdXlbc76VIwTF4mIiEgukrDnDBEndO/ePcTExODo0aNo164dDh48iH79+mHz5s1mx2jevLnR8rS0NERGRuLEiRNypWsxR9+qntyTh/1ZICJSTFFREU6fPg0AaNq0qUMmjmu1QNWqxc81Gv4Y58ksef/pvv9a8z3XY4ZU6VSqVAlLly6Ft7c3Dh48CH9/f3z66adKp0VERERE5JY8akiVzkMPPYRBgwZh3bp1mDhxIurVq2fR+qZ6dqbOfDiSRqMpU1ZymFFERPHz3buBrl0Nlzt48P6QqhYt7pe1a1f8PD297JAqY/GMxd62Dejd27AsJcX4kCpd28aWs2RIlS43Xd7mrE9ERERE8vLIDgdwf4y6u41VN7Y9uqKS8xqaNSu7blRU2bkZDRrcfx4UZPiF3VQ8Y7Fbty5bFh5etgNg7HAYW84UY9taOm8iIiIichyPG1JFRERERK5hzZo1iImJQeXKlREcHIyxY8fi2rVrGDNmDCRJwq5duwyWz8rKwowZM9C0aVP4+fmhZs2a6NOnD/bs2VMm9q5duyBJEsaMGYMbN25g8uTJqFOnDlQqFVq0aIEvv/zSZF6JiYkYMGAAAgMDoVKpEB4ejilTpuDy5ctlll2+fDkkScKcOXOQlpaGYcOGISAgANWqVUOfPn1w8uRJAMXzjN9++200adIEfn5+aNSoET755BOTOWRkZGDixIkIDw+HSqVCYGAghgwZguPHj5u5dx3HY89weCK1Gig5F9jUvGA5ljO2jjnzkEvHtJZccYiIiEgZixcvxrRp0+Dt7Y24uDgEBARg+/bt2LVrF1q1alVm+dTUVPTo0QN//fUXIiMj8dhjjyE7Oxs7duzAf//7X3z99dd48skny6x369YtxMbG4u+//0a7du2g0WiwZ88ejBs3DkVFRRg/frzB8t988w3GjBmDoqIidOzYEfXr18eRI0fw6aefYv369di1axeaGRnuce7cObRr1w41atRA165d8eeff2Lbtm1ITk7G8ePHMWnSJOzYsQOxsbGIiIjAzp078Y9//AM+Pj6YMGGCQax9+/ahb9++uH37Npo3b47HH38cf/31F9avX4+tW7diy5YteOSRR2w8AjISHmr06NECgJg9e7ZsMaOiokRUVJRs8YiIiMi9FRYWipMnT4qTJ0+KwsJCWWNrNMYf164JUfyzXPFzU8spKS0tTfj6+go/Pz+xZ88effmdO3dE3759BQABQOzcuVMIIcS9e/dEixYtBADx/vvvi6KiIv06R44cEbVr1xZqtVpcu3ZNX75z5059nMGDBwtNiY3euHGjACDCwsIM8rpw4YKoXLmyqFSpkti8ebO+vLCwUEydOlUAEDExMQbrfPXVV/p2pk+frj/ORUVFYsyYMQKAiIqKEi1atBAXL17Ur/fLL78IAKJBgwYG8f7++28REhIifHx8xNq1aw3qtm/fLnx9fUVoaKjIz8+vcD9b8v6z5Xsuh1QRERERuaGqVY0/goPvLxMcbHo5JX355ZcoKCjA6NGj0blzZ325n58f3n///TKXb928eTNSUlIwYsQIvPDCCwa3CWjdujVmzZoFrVaLb775pkxb1apVw+eff24wD3bAgAFo2bIlLly4gIyMDH350qVLcefOHYwYMQL9+vXTl3t5eWHevHmoW7cuDh06hAMHDpRpJzIyEv/+97/1uUuShOnTpwMATp48iQ8++MDgQkbdu3dH69atcf78eYMcvvzyS1y9ehUzZszAkCFDDNro0aMHpkyZgr/++gs//vij0X2rBHY4iIiIiMip7N+/HwAwdOjQMnWRkZFoXepqNNu3bwcADBw40Gi8Tp06AQAOHTpUpq5t27ZGb/7cpEkTAMCVK1f0ZXv37gUAjBw5sszyKpVKn69uuZLi4uJQqZLhbIaI/11S09fXF11LX+ITxdtaOgdbtlUpnMNBRERE5IaMXCkfQPGVHHVnOa5dc84rOeomX9evX99ofVhYGJKTk/WvdWcAnnjiCTzxxBMm416/fr1MmanbI1T932me/Pz8MnmFh4cbXUdXbmzyeGhoaJky3VmVkJAQozfd09WXzEG3re3btzeag46xbVWKx3Y4li9fjuXLlyudBhEREZFdmNORUKuds8OhU3JoVEmi1JVhCgsLAQB9+vRBUFCQyXjGJnObasOavMqrL28dS3LQbevQoUNRpUoVk8tV1CFxJI/tcBARERGRc6pTpw5Onz6NCxcuoHHjxmXqL168aPBad5Zi0qRJePzxx+2WV926dXH69GmcO3dOP+SqpPPnzwMozt9e6tWrh9OnT+P11183erUuZ+QRczhWrFgBSZLw4IMP4t69e0aX+e233+Dl5YWQkBDcvHnTwRkSERERkU7Hjh0BAOvWrStTl56ejt9//92grEePHgCAjRs32jUv3QT2b7/9tkxdQUEB1q5da7CcPThqW+XkER2O0aNHo0ePHjh27Bjee++9MvUFBQWYMGEChBB4//33UbNmTQWyJCIiIiIAGDt2LHx8fLB8+XL9BHIAyMvLw9SpU1FUVGSw/JAhQ9CsWTMsX74c//73v3H37l2D+oKCAqxfvx5//PGHTXmNGzcOlStXxurVq7FlyxZ9eVFREV577TX89ddfiImJQYcOHWxqpzwTJ05EYGAg3n77bXz11VdlhpdptVqsXLkSly5dslsOlvKIDgcAfPbZZ6hcuTLmzp2LtLQ0g7q3334bp06dQt++fcudaERERERE9teoUSO8/fbbyMvLQ5cuXdCzZ08MHz4cjRs3xrFjx9C/f38AxVd3AoBKlSphw4YNqF+/Pl599VU0aNAAvXv3xrBhwxAbG4vg4GAMHjy4zHdAS4WFheHzzz+HEAL9+/dH586d8eSTTyIqKgrvvfcegoODsXLlSpu3vzw1a9bEhg0boFar8cwzz6Bhw4bo168fBg8ejJiYGAQHB2P06NFONWncYzockZGRmD17Nu7cuYNJkybpy0+ePIl33nkHVatWLff28e5CqwUkqfih1Vpf7qh1PCGW0u2XF8tSnrJfXK19Z42ldPuevi1Ezm7GjBlYvXo1oqOjsXfvXuzYsQOPPPIIDhw4gDt37gAAateurV++WbNmOHr0KObMmYOgoCDs27cPW7ZsQVZWFrp06YKvvvpKPxzJFqNGjcKePXvQr18/nDp1CuvWrcOdO3cwefJkJCcnG52YLreHH34Yf/zxB1566SVUrlxZfzf127dvo1+/fvjPf/6DqKgou+dhLo+aNP7SSy9h9erV+OWXX7BixQo8/fTTmDBhAgoKCjB//nyEhYUpnSIRERGRXanVxfcZdwXDhw/H8OHDDcq0Wi2OHj0KtVqNRo0aGdTVrFkTs2fPxuzZsyuMHRcXV2Y4UknlXdG0Y8eO+OGHHyreAABjxozBmDFjTNZbm0PdunWxYMECLFiwwKw8lORRHY5KlSph6dKlaN++PV566SVkZGRg//79aNeuHZ577jml05OF1sTPV7riktWZmcZjlFeuVhfH0P1rSSxr1pGzfWeLpXT75cUCLLtMoqn3V+l8zG3fWfeLOx1juWMp3b7SsZRu35JYpT/jaks+7EQOlJ6ejtq1a6N69er6Mo1Gg8mTJ+P69et45pln4O3trWCGZC5JlNetclPTpk3D4sWLARR3QpKTky26rFjz5s2NlqelpSEyMhInTpyQI02rWHMtaSIi8lwe+DXAqRQVFeH06dMAgKZNmxq9+ZunmjdvHubMmYOHHnoI9erVw82bN/H777/j+vXrCA8Px4EDBxCsu4MhWcWS95/u+68133M98l09bdo0/fNnn33WZa5hTEREROQpunfvjoEDB+Kvv/7C5s2bsXfvXtSqVQsvvfQSDh48yM6GC/GoIVU6c+bM0T/funUrcnNzy71TY2mmenamznw4kkajMVpecshLRETx8/R0w2XMKS99mt6SWNasI2f7zhZL6fbLiwVYP6TKWF5KbguPsWNiKd2+0rGUbt+SWNZ8xomUEBMTg++++07pNEgGHtfh2LlzJ7766ivUq1cPrVu3xubNmzFnzhy8++67SqcmC1NjcXXFJb8YBAVZX+6odTwhltLtlxfLXJ62X1ytfWeNpXT7nr4tRESO4lFDqvLy8jBx4kQAwEcffYQlS5agevXqWLRoEY4dO6ZwdkRERERE7sejJo2/9tpreOeddzBo0CCsX78eQPENASdPnox27dohMTHRpslatkymISIiIs8jhEBqaioAoHHjxqhUyeMGn5CC7t69i7NnzwIovo9JeRcf4qRxM6SkpGDBggWoVq0aPvzwQ335xIkT0bFjRxw8eBAff/yxghkSERGRp5EkSX+3bFOXtieyl9u3bwMAVCqVXa906hHd6KKiIkyYMAF3797FokWLEBoaqq+TJAmff/45Wrdujf/7v/9DfHy8QT0RERGRPfn7+yM7OxvXrl0DUDwfk5fHJXsRQiA/Px85OTm4ceMGgOIbJtqTR3Q4Pv74Yxw4cAAdOnTA5MmTy9Q3b94cL7/8Mt5++208//zz+uFWRERERPZWu3ZtaLVa5OXl4fLly0qnQx6mRo0aBjdXtAe3n8Nx6dIlREVF4c6dOzhy5AhatmxpdLm8vDy0bNkSZ8+excaNGzFgwACL2+IcDiIiIrJGYWEhsrOzkZOTg4KCAqXTITfn7e0NtVoNf39/+Pv7mzWcypbvuW5/hqNevXr68Wnl8fPzw59//umAjIiIiIgMeXt7IygoCEFBQRBC8A7wZDeSJNl1voYxbt/hICIiInIlSnwhJLInzkgiIiIiIiK7cckOR25uLjZu3Ihx48ahVatWqFatGtRqNaKjo/HGG29Ao9EYLF9UVIS9e/filVdeQfv27REUFASVSoXIyEhMmjQJ586dU2hLiIiIiIjcm0tOGl+6dCkmTJgAoHgCS1RUFG7fvo39+/cjJycHzZo1w+7duxEUFAQAOHv2LBo3bgwACA0NRdu2beHl5YWDBw/ir7/+gr+/P7Zu3YpOnTrZlBcnjRMRERGRO/K4G//5+vpi8uTJOHPmDFJSUrBmzRps27YNp0+fRuvWrZGamoqpU6fql5ckCb169cLu3btx6dIlbNy4EevXr0daWhrGjBmDnJwcjBw5Enfv3lVuo4iIiIiI3JBLnuEoT2JiIjp27AiVSoXbt2/r795pSl5eHkJCQvD3339j165d6Nq1q9Vt8wyHZbRaoGrV4ucaDaBWG5bp6OrMiaNz7RoQHGyfMnvGtqW9ivYTERERkbU87gxHeaKjowEA+fn5yM7OrnB5Pz8/NGnSBAB4sx0iIiIiIpm53WVx09PTAQA+Pj6oVatWhcsXFhbi/PnzAICQkBC75uYIWq1W6RQqpEuxZKoZGUCVKsD162WXz8gAAgOLn+t+wS+5blZW2XV+/13estRU48+dqb2S+ykoiKc6iIiIyDm43ZCqCRMmYOnSpejfvz9++OGHCpf/9ttvMWrUKAQGBuLixYtQqVQVrqM7pVRaWloaIiMjFR1Sxet2EwDeMIqIiIhkxSFV/7N161YsW7YMPj4+SEhIqHD5ixcv6ieXv/HGG2Z1NoiIiIiIyHxuM6Tq1KlTGDVqFIQQmD9/vn4uhylarRaDBg3C9evXMXDgQEyaNMnstkz17Eyd+XCk0vcgcUYlh1RFRBQ/T0m5P6SqXTvD5VNSKh5S1aKF4TrbtgG9e8tXtns3oLueQMnnztReyf1ERERE5CzcosNx6dIl9O7dGzdv3sT06dPx4osvlrv83bt3MXjwYCQnJ6NTp05YtWqVgzK1P7ULXKbIWKchPLy4/H+3TjGgqzMWo/Rzndat5S1r1sz4c2dqz9h+IiIiIlKayw+pun79Onr27IkLFy5g7NixWLBgQbnLFxUVYdSoUfj5558RHR2NzZs3o3Llyg7KloiIiIjIs7j0GY6cnBz06dMHqampiI+PxxdffFHhpOkpU6ZgzZo1aNKkCf773/+iRo0ajkmWylCrgdJzm42VWRMHsG+Zs7ZHRERE5Gxc9gxHfn4+BgwYgMOHD6NXr15YvXo1vL29y13ntddew5IlSxAWFobt27cjyNj4HSIiIiIiko1LdjgKCwsxYsQI7Ny5E507d8b69esrvKP4woUL8c477yAkJAS//PILwsLCHJQtEREREZHncskhVR999BE2bNgAAAgICMCUKVOMLrdgwQIEBATg6NGjmDFjBgCgYcOGeOutt4wuP378eHTq1Mk+SRMREREReSCX7HDcvHlT/1zX8TBmzpw5CAgIwK1bt/Q3QktMTERiYqLR5ePi4tjhICIiIiKSkdvdaVxJttyBkYiIiIjIWfFO40RERERE5JRcssORm5uLjRs3Yty4cWjVqhWqVasGtVqN6OhovPHGG0bvtn3x4kV88sknGD16NB544AF4eXlBkiQcOHBAgS0gIiIiIvIMLjmHY9WqVZgwYQKA4tM7vXv3xu3bt7F//37Mnj0bq1evxu7duw0ue/v9999j2rRpSqVMREREROSRXPIMh6+vLyZPnowzZ84gJSUFa9aswbZt23D69Gm0bt0aqampmDp1qsE6ERERmDZtGlatWoU///wTXbt2VSZ5IiIiIiIP4pIdjqeffhqffPIJGjdubFBep04dfPzxxwCA9evXo6CgQF/3+OOPY+HChRgxYgQaNWrk0Hw9lVYLSFLxIzPz/nO5y+wZW6u1/7ZU1IaunoiIiMgVuWSHozzR0dEAiu9Enp2drXA2RERERESezSXncJQnPT0dAODj44NatWopnI3jaZ3g5/CSv9jrpKaWXe733+Upkzt2yXgZGUCVKkBurv3aO3kSCAgwbOP8+fvPMzMBtRpQq9VQq8uuT0REROTM3O4+HBMmTMDSpUvRv39//PDDDyaXi4uLw+7du5GYmIgOHTpY1IbuOsSlpaWlITIyUtH7cEiSpFjbZG8C7vVpJSIiIlfB+3D8z9atW7Fs2TL4+PggISFB6XSIiIiIiDye2wypOnXqFEaNGgUhBObPn6+fy2EPpnp2ps58OJKxe5A4WskhVRERxc937wZKXxhs2zagd2/by+SOXTJeSsr9IVUtWtinvYMH7w+p0rVx8CDQrl3x8/R03ZAqEBEREbkct+hwXLp0Cb1798bNmzcxffp0vPjii0qnpBi1E3wr1aVQcg5Hs2Zll2vdWp4yuWOXjBceXrw99tyWqKiybTRocP95UBA7G0REROS6XH5I1fXr19GzZ09cuHABY8eOxYIFC5ROiYiIiIiI/selz3Dk5OSgT58+SE1NRXx8PL744gtOmnYiajUMJjkbm/AsV5k9YwP23xZz2yAiIiJyNS57hiM/Px8DBgzA4cOH0atXL6xevRre3t5Kp0VERERERCW4ZIejsLAQI0aMwM6dO9G5c2esX78evr6+SqdFRERERESluOSQqo8++ggbNmwAAAQEBGDKlClGl1uwYAECAgIAAFeuXMGgQYP0dSdPngQAjB8/HlWrVgUA9O3bF7NmzbJn6kREREREHsUlOxw3b97UP9d1PIyZM2eOvsORn5+PpKSkMsuUvMRtM2OXHyIiIiIiIqu53Z3GlWTLHRiJiIiIiJwV7zROREREREROySU7HLm5udi4cSPGjRuHVq1aoVq1alCr1YiOjsYbb7xR7t22V65ciXbt2qFq1aqoVasWHnvsMezfv9+B2RMREREReQ6XHFK1dOlSTJgwAUDx6Z2oqCjcvn0b+/fvR05ODpo1a4bdu3cjKCjIYL3p06dj0aJFqFy5Mh599FHk5eXh119/hRACa9euNZhUbg0OqSIiIiIid+RxQ6p8fX0xefJknDlzBikpKVizZg22bduG06dPo3Xr1khNTcXUqVMN1tmxYwcWLVqE2rVr49ixY9i4cSO2bduGPXv2wNvbG2PHjjWYjE5ERERERLZzyQ7H008/jU8++QSNGzc2KK9Tpw4+/vhjAMD69etRUFCgr3vvvfcAAK+//rrBerGxsZg0aRL+/vtvfPnllw7I3n60WkCSih+ZmfefGyurqN6WMnvGNrc9rdbyfeZK2+cK7Zl7DIiIiMi9uWSHozzR0dEAii+Dm52dDQD6oVMAMGTIkDLr6Mo2b97soCyJiIiIiDyDS96Hozzp6ekAAB8fH9SqVQsAkJqaivz8fAQGBqJevXpl1mnTpg0A4Pjx445LVEa6X5KL/y1+kZpadrmSZb//XrZerjJjbTu6vYwMIDCw+LlaXbZet8+yssxrr2Qbjtg+JduTK05mJhAUVLzzjR0DIiIi8gwuOWm8PBMmTMDSpUvRv39//PDDDwCAH374AQMGDEDr1q1x5MgRo+vVrFkTt27dwu3bt+Hv719uG7pJM6WlpaUhMjLS4ZPGJcnglUPbJipf8Z8X9/orQ0RE5Hk8btK4KVu3bsWyZcvg4+ODhIQEfbnuMrlVqlQxua76fz/BlndJXSIiIiIisozbdDhOnTqFUaNGQQiB+fPn6+dyAIDuJI4kmf7135ITPSdOnDD6iIyMtH4DbKDRFD+uXQMADQANdu/W6J8bK9u2rWy9XGXG2nZ0eykpGly7VvzQaMo+dHUpKea1V7INR2yfku3JFSc9XaN/bxIREZHncos5HJcuXULv3r1x8+ZNTJ8+HS+++KJBvW6IlLacy+bk5uYCAKpWrWq/RO3EcHx88YtmzcouV7Ksdeuy9XKVGWvb0e2Fh5c/b0BXZ2wZY+2VbMMR26dke3LFCQri3A0iIiJygzMc169fR8+ePXHhwgWMHTsWCxYsKLNMWFgYgOKOiTFarRa3bt1CjRo1Kpy/QURERERE5nPpMxw5OTno06cPUlNTER8fjy+++MLosKmmTZtCpVIhKysLly5dKnOlKt1E8latWjkkb3tRqw0n5xobJWZJvS1l9oxtbnvmKL3PbGnP3fenLWVERETkuVz2DEd+fj4GDBiAw4cPo1evXli9ejW8vb2NLlu5cmV069YNALBu3boy9bqyfv362S9hIiIiIiIP5JIdjsLCQowYMQI7d+5E586dsX79evj6+pa7zvTp0wEAb775Jv788099eWJiIpYsWYJq1aph3Lhxds2biIiIiMjTuOSQqo8++ggbNmwAAAQEBGDKlClGl1uwYAECAgIAAD169MCLL76I999/Hw8++CB69uyJgoICbN++HUVFRfj222/1NwokIiIiIiJ5uGSH4+bNm/rnuo6HMXPmzNF3OABg8eLFePDBB/HRRx9h+/bt8PHxQffu3fH666+jU6dOds2ZiIiIiMgTud2dxpVkyx0YiYiIiIicFe80TkRERERETsllOxzJycmYN28e4uPjERoaCkmS4OfnV+46ly5dwqRJkxAWFgaVSoW6detizJgxyMjIcEzSREREREQexiXncABAQkICNm3aZPbyKSkp6NatG7KystCwYUP069cPZ8+exYoVK7Bx40bs3bsXLVu2tGPGRERERESex2XPcMTGxuJf//oXNm/ejKtXr5a7rBACI0eORFZWFp555hmcOXMG33//PY4dO4aFCxfi77//xogRI1BUVOSg7ImIiIiIPIPLdjhmzpyJuXPnol+/fggODi532d9++w3Hjx9HzZo1sXjxYlSqdP/EzrRp0xATE4MTJ07gxx9/tHfadqXVApJU/MjMvP/cWFlF9baU2TO2sTKtVp594ojt0+Wq5LGqKAdz9y0RERGROVy2w2GJ5ORkAEDbtm3h7+9fpr5r164AYNEQLSIiIiIiqpjLzuGwhPZ/P9PWrFnTaL3uhn/Hjh1zWE5yKvmLNVD8IjW17HIly37/vWy9XGXG2pa7vZJtZGYCanXxc92/hvvE9rxKrmtLnJMngYAAIDe3/HhytWduDufPG28vMxMICireqbp9S0RERGQJt7kPhyRJUKlUyMvLK1P3xRdf4Nlnn0X79u1x4MCBMvWTJk3CkiVLULt2bVy/fr3CtnTXIS4tLS0NkZGRDr8PhyQZvHJo2+QJiv9EuMdfCiIiIrIG78NRgS5dugAADh06hJMnTxrUaTQarFu3DgCQk5Pj8NyIiIiIiNyZR3Q4mjZtisGDB6OoqAgDBgzAzp07odFocPToUfTt2xd///03AMDLy7zdceLECaOPyMhIe26GSRpN8ePaNQDQANBg926N/rmxsm3bytbLVWasbbnbK9lGeroG164VPzSa4ofudXq6PHmVXNeWOAcPFueUklJ+PLnaMzeHgwdN71vd+4uIiIjIGh4xhwMAli5diuzsbOzatQvdunXTl1epUgVvv/02XnnlFZNzPJyd4dj64hfNmpVdrmRZ69Zl6+UqM9a23O2VbCMoqOz8gtJzOWzNq+S6tsSJiirOraK85GrP3BwaNDDenrF9S0RERGQJj+lw1KhRAzt27MDPP/+MHTt24O+//0Z4eDiefPJJ/TArU3MziIiIiIjIOh7T4QCKJ5b37t0bvXv3Nij/4IMPAABxcXEKZCUftdpwYq+xSb6W1NtSZs/Y5ZWVZuk+MbfM1hyVPlYV5cAJ4kRERCQXj5jDUZ4bN25gxYoV8PX1xejRo5VOh4iIiIjIrXhMh+PMmTO4ffu2QVlmZibi4+ORnZ2N1157DfXq1VMoOyIiIiIi9+SyQ6q2bNmChIQEg7KCggJ06NBB/3rWrFno27cvAGDVqlV499130bZtW4SGhuLGjRvYu3cv7ty5gzFjxmDWrFkOzZ+IiIiIyBO4bIcjKysLSUlJBmVCCIOyrKws/fNu3brh6NGjSE5ORlJSEqpWrYrOnTtj0qRJGDRokMPyJiIiIiLyJG5zp3FnYMsdGImIiIiInBXvNE5ERERERE6JHQ4iIiIiIrIbl+1wJCcnY968eYiPj0doaCgkSYKfn1+56/z111+YPHkyIiIioFKpULVqVbRp0wbz589Hfn6+gzInIiIiIvIcLjuHY+DAgdi0aZNBmUqlQl5entHlz5w5g4cffhjXr19HREQEWrduDa1Wi3379kGj0aBTp07YsWMHfHx8rM6JcziIiIiIyB3Z8j3XZa9SFRsbi+joaMTExCAmJgYhISHlLv/qq6/i+vXreO6557B48WJ4e3sDKL4XR6dOnbBv3z588803GDt2rCPSJyIiIiLyCC47pGrmzJmYO3cu+vXrh+Dg4AqX37NnDwDg9ddf13c2ACAoKAhTpkwBABw6dMg+yTqQVgtIUvFDqy3/dWam+cta89pe2yVn3vbcB0rmacn7whP3p9zvTyIiIjLNZTscllKpVBUuU6tWLQdkQkRERETkOVx2SJWlevbsiRUrVuCtt97C4sWL4eVV3NfKzMzEJ598gkqVKmHkyJEKZ2k93S+2mZn3f7rNyDBcpvTr8+dN19n6OjMTUKuLn+v+tVTJX6FLPpczb3vuA6XyLLnvdUruP2fJU8nYJfdRUJCVb1AiIiIyi8tOGi9NkqRyJ41fuXIFPXv2xIkTJxAREYE2bdpAq9Vi7969qF27Nj7++GP07dvXrLZ0k2ZKS0tLQ2RkpCKTxiVJ/8zhbRO5Mjf5E0hERGRXHjlp3FJ16tTB7t27MWLECGzfvh3p6ekAijsqgwcPRlRUlMIZEhERERG5H4/pcBw/fhx9+/aFt7c3Nm3ahC5dukCr1WLdunX45z//iW3btuG3335DZGRkhbFM9exMnflwBI2m+N/MTA0iIoqfp6QU/9uihfHXBw8C7dqZt6ylr9PT5R9SpdsuOfO25z5QKs+S+16n5P5zljyVjG1sHxEREZF9eMSQqrt376JFixY4e/YsDh8+jNatWxvUL1y4EC+99BJGjBiBVatWWZ2DM9yHQ6sFqlYtfq7rhJh6fe0aoLvAV0XLWvNazi90JbdLzrztuQ+UzNNYh8MZ81QyNjscRERE5rPle65HXKXqwIEDOHPmjP6Gf6UNGzYMALBr1y4HZ0ZERERE5N48YkjVpUuXAADVqlUzWq8rv3HjhsNyshe1Gih9zqq815Ysa81ruZTeLjnztOc+UDJWSRW9L5x1m+0dm4iIiOzPI85w6O5Cfvr0aeTk5JSp193wLzw83JFpERERERG5PY/ocMTGxiIoKAharRbPPfcc8vPz9XWXL1/GtGnTAABDhgxRKkUiIiIiIrfkskOqtmzZgoSEBIOygoICdOjQQf961qxZ6Nu3L/z8/LBkyRIMHToUK1euxK+//oq2bdvizp07SExMRE5ODtq0aYNXX33V0ZtBREREROTWXLbDkZWVhaSkJIMyIYRBWVZWlv75wIEDcfDgQSxYsAB79uzB1q1b4evri8aNG2PYsGGYOnUqKleu7LD8iYiIiIg8gdtcFtcZOMNlcYmIiIiI5MbL4hIRERERkVNy2Q5HcnIy5s2bh/j4eISGhkKSJPj5+ZlcXpKkCh/dunVz4BYQEREREbk/l53DkZCQgE2bNpm9/OjRo03WbdmyBdevX0fnzp3lSI2IiIiIiP7HZTscsbGxiI6ORkxMDGJiYvT32jBl+fLlRstv3bqF7777DgAwatQoudMkIiIiIvJoLtvhmDlzpixx1q5di/z8fHTo0AGNGzeWJaaStFqgatXi5xpN8b+mXl+7BgQHm7espa/tGVujKb5ztrnK2ye25lleHqXbLb1sRceq5PKWHFdHHytnzdOS/V06tiXvLyIiIiqfy87hkMs333wDAHjqqacUzoSIiIiIyP247BkOOVy4cAF79+6Fj48PnnjiCaXTsYlWW/xvZqZWX5aRYbhM6dfnz5uus/W1PWNnZt7/BbqiMwwl/5U7T1N5GGu35LKllysvtlZ7/19r87TnsXLWPC3d3yVjl1w3KIinOoiIiGzlNvfhkCQJKpUKeXl5Zq/zzjvv4LXXXsPjjz9u0QR03XWIS0tLS0NkZKQi9+GQJP0zh7dN5K7c5M8jERGRzXgfDitxOBURERERkX157JCqI0eO4OTJk6hRowb69+9v0bqmenamznw4gm4SbGamBhERxc9TUor/bdHC+OuDB4F27cxb1tLX9oydnm75kCpT+8SWPE3lYazdksuWXM5UXrrlSw5VMve4OvJYOWuelu7vkrGNrUtERETW89gOh+7sxtChQ6FSqRTOxnbGxpyHhxsuU/p1gwam62x9bc/YQUHmfSHULVNy7L6ceZrKw1i7xpYtL6/Sy5e3rJLHylnztHR/l4xt7vuLiIiIzOORQ6oKCwt57w0iIiIiIgfwyDMcv/76K65cuYIGDRq43d3F1Wqg9DzX8l5bsqylr+0Z2xIV7RN75WGsXUvysnRZJY+Vs+ZpSW6cH05ERGQfHnmGQzecatSoUZAkXtWJiIiIiMhePK7DkZubiw0bNgDgcCoiIiIiIntz2SFVW7ZsQUJCgkFZQUEBOnTooH89a9Ys9O3b12CZjRs3QqPRICYmBs2aNXNIrkREREREnsplOxxZWVlISkoyKBNCGJRlZWWVWa/kcCoiIiIiIrIvt7nTuDOw5Q6MRERERETOincaJyIiIiIip+SyHY7k5GTMmzcP8fHxCA0NhSRJ8PPzq3C9/Px8LFiwAG3btkW1atVQtWpVNG3aFOPGjcNff/3lgMyJiIiIiDyHy87hSEhIwKZNmyxaJzMzEz169MAff/yBkJAQ9OjRAwBw9uxZfPnllxg7dixCQ0PtkS4RERERkUdy2Q5HbGwsoqOjERMTg5iYGISEhJS7fFFREQYMGIA//vgD//d//4c5c+agUqX7m5+eno5q1arZO20iIiIiIo/ish2OmTNnWrT88uXLceDAAQwePBhvvvlmmfqIiAi5UlOUVgtUrVr8XKMp/tfU62vXgOBg85a19HXp2Gp1+XlaUm/JNtqaZ2kV5W0vcm6zPd8HjnxPOWtsS99TRERE7s5l53BYasmSJQCAl156SeFMiIiIiIg8h8ue4bBETk4ODh8+DH9/f7Rv3x6JiYn44YcfcOPGDYSFhWHAgAFo0aKF0mnaRKst/jczU6svy8gwXKb06/PnTdfZ+rpk7MxM42cwLK3Xau//a+88TeVjKm97/YptrF17HitniuWqsY29p9RqNc90EBGRx3Kb+3BIkgSVSoW8vLwydUlJSejQoQNat26Njh074uOPPy6z7owZM/Duu++a1ZbuOsSlpaWlITIyUpH7cEiS/pnD2yaiigi4x19aIiLyVE53H46ffvoJAwcORGhoKFQqFcaNG2dQN336dFy+fNkeTRt18+ZNAMAff/yBjz/+GDNmzMC5c+eQlZWFL774ApUrV8b8+fPx2WefOSwnIiIiIiJPIPuQqilTpmDJkiUQQsDf3x93795FyZMoNWrUwOLFi1GvXj1Mnz5d7uaNKiwsBADcu3cPI0aMwPz58/V148ePR35+Pp577jm89dZbmDRpUoXxTPXsTJ35cATdxNXMTA10899TUor/1Y0WK/364EGgXTvzlrX0dcnY6enGhyjp8jS3vuSQKnO30do8yxtSZSxvRwypkmub7fk+cNR7ypljG3tPcTgVERF5Mlk7HF9++SU+++wztG/fHkuWLEGrVq3g5WV4EiU2NhahoaHYvHmzwzoc/v7++ufPPPNMmfqxY8fi+eefx6VLl3D27Fk0atTIIXnJSfeFJijo/jeb8HDDZUq/btDAdJ2tr0vGDgoqf46GpfUl6+ydp46uvKK85WasXXseK2eK5aqxzX1PEREReQpZh1QtWbIEtWrVwo8//ohWrVqZXK5Ro0ZIT0+Xs+lyhZf4RtCg5LeB/6lSpQoCAwMBFN8ckIiIiIiI5CHrGY4TJ07gkUceQe3atctdLiQkBAcOHJCz6XKFhYWhdu3ayM7Oxo0bN8rUFxUV4datWwCAqrqL6bsotRplJqeW99qSZS19Xd4kWWN5mltv6TbakqcledmT3Ntc+rWzxnLV2JwgTkREdJ+sZzi8vLxQVFRU4XKXL1+G2sHjDPr37w8A2LlzZ5m6/fv3o6CgAJUrV0azZs0cmhcRERERkTuTtcPRrFkzHD58GLm5uSaXyc7OxtGjR8sdcmUPL7/8Mry9vTF//nz8/vvv+vLMzEy8+OKLAIrnd/j6+jo0LyIiIiIidyZrh2PkyJHIysrCP/7xD9y7d69MvRACL7zwAjQaDZ566imb2tqyZQs6dOigfwBAQUGBQdmWLVv0y0dFRWHRokW4ceMGYmNj8cgjj6B///5o1qwZjhw5gjZt2uCdd96xKSciIiIiIjIk6xyOKVOm4Pvvv8eKFSuwb98+9OrVCwBw/PhxzJgxAz/++CPOnDmDbt26YfTo0Ta1lZWVhaSkJIMyIYRBWVZWlkH9888/j6ZNm2LBggU4ePAg8vLyEBkZialTp2LGjBmoUqWKTTkREREREZEh2e80npubixkzZmDZsmW4e/euQZ23tzfGjBmDDz74AJUrV5azWadgyx0YiYiIiIiclS3fc2XvcOhkZWVh9+7dyMjIQGFhIerVq4dHHnkEdevWtUdzToEdDiIiIiJyR7Z8z5X9TuM6gYGBGDJkiL3CIzk5Gdu3b8fBgweRlJSEy5cvQ6VSIS8vz+jyc+bMwdy5c03GmzlzJubNm2evdImIiIiIPJLdOhz2lpCQgE2bNlm83sMPP2z0TuIPPfSQHGkREREREVEJdulw7N69G7t378aVK1eQn59vdBlJkrBs2TKr24iNjUV0dDRiYmIQExODkJAQs9YbP348xowZY3W7RERERERkPlk7HDdu3MCgQYOwb98+VDQ1xNYOx8yZM61el5yfVgvobvqu0RTfadvcZYHyX1tyz0lLY5d8fe0aEBxs3br2jMU83XcfOPh+qkRERGaRtcMxbdo07N27F82bN8ezzz6LyMhIh99RnIiIiIiInIesHY7NmzejXr16SExMRFXdz25OZseOHTh69Cjy8vJQr1499OnTx63mb2i1WqVTsIku/ZKbkZl5/5fbkv1XY8tmZBjGK/3aVCxz8qgodunX589bv649YzFP99wHxt7b/MGHiIicgayXxa1evToeffRRrF27Vq6QZpMkyeqrVA0ePBjLly83u5OkuyxYaWlpaYiMjFT0sriSJCnWNhE5Fztd9ZyIiDyQLZfF9ZIzkTZt2uB8yZ/gnEijRo2wYMECnDhxAhqNBhcvXsS3336L0NBQfP/993jqqaeUTpGIiIiIyO3IeoZjx44d6N27N9asWYOBAwfKFdYsFZ3hMOXKlSto2bIlsrOz8dtvv6Fjx45W5+AMN/5zpyFVERHFz9PTKx5SpVs2JaX43xYtjL82FcucPCqKXfr1wYNAu3bWrWvPWMzTPfeBsfc2h1QREZFcnObGf926dcOqVavw7LPP4ocffkCPHj0QGhpqcphPly5d5GzeKnXq1MHYsWOxYMEC/PzzzzZ1OJyBq3/B0KVfst8UFGS8c2Bs2fBww2VKvzYVS47YpV83aGD9uvaMxTzdcx+Y+94mIiJyNNnvw6HRaODj44MVK1ZgxYoV5S5bWFgod/NWady4MYDisx1ERERERCQfWTscy5cvx7hx4yCEQJs2bRAREeESv7jfvHkTAJz2ylqeSK0GzB3sZ2zZil7bkoclr21Z156xmKd9YyuZJxERkbORtcPx7rvvQqVSYevWrYiLi5MztN0IIbBhwwYAcKvL4xIREREROQNZr1KVkZGBrl27Ol1n4/r161i5ciXy8/MNyjUaDSZPnoykpCSEhIRg0KBBCmVIREREROSeZD3DERoaKme4cm3ZsgUJCQkGZQUFBejQoYP+9axZs9C3b19oNBqMHj0azz//PB544AGEhYXh1q1bOHLkCLKzs1GjRg2sW7cOVapUcVj+RERERESeQNYOx9NPP4333nsPV65cQZ06deQMXUZWVhaSkpIMyoQQBmVZWVkAgNq1a2PmzJk4cOAAzp49i6NHj8Lb2xsNGzbEmDFjMG3aNId2loiIiIiIPIWs9+EoLCzEsGHDcPbsWXz44YdOcdlbR3KG+3AQEREREcnNae7Dobu87Pnz5/HII4/A19cXISEhRu/DIUkS0tLS5GyeiIiIiIicjKwdjoyMDIPX+fn5OH/+vJxNEBERERGRC5H1KlVFRUUWPWyRnJyMefPmIT4+Xn83cz8/P4ti9OjRA5IkQZIkXL161aZ8iIiIiIioLNnvNO4oCQkJ2LRpk9XrL1++HL/++iskSYKM01iIiIiIiKgEl+1wxMbGIjo6GjExMYiJiUFISIjZ62ZlZWHGjBl49NFHcfr0aQ77IiIiIiKyE1mHVOmkpKRgypQpaNmyJWrXro2AgAC0bNkS//jHP5CSkiJLGzNnzsTcuXPRr18/BAcHW7Tu1KlTodVq8cknn8iSC8lPqwUkqfih1Rq+zsw0XWfpazljMU/XydOZ9oFS731jbRMREdmD7Gc43n//fbz88ssoLCw0GKp048YNnDhxAl988QXmz5+PF198Ue6mzfLzzz9j1apVSEhIQGRkpCI5EBERERF5Clk7HNu3b8e0adNQpUoVTJo0CU899RTCw8MhSRIyMjLw9ddf47PPPsP06dPRokULdO/eXc7mK5Sbm4tJkyahWbNmeOWVVxzatqNoXfxnS136JTej1MXPUHIEXOk6S1/LGYt5uk6ezrIPMjMBtdqwzFHv/ZJtl8xBXTohIiIiG8l6478+ffrg119/xa5du9CxY0ejyyQmJqJLly7o2bMntm7dKlfTkCQJKpUKeXl5Jpd56aWXsHDhQuzcuRNxcXEAgPDwcJw/fx5Xrlwxex6I7sYnpaWlpSEyMlLRG/8Zu+cJEZG5eBENIiIyxpYb/8k6h+PgwYPo2rWryc4GUDzZOy4uDklJSXI2XaEjR47g/fffx+jRo/WdDSIiIiIisi9Zh1Tl5uYiMDCwwuUCAwORm5srZ9PlKiwsxIQJE1CjRg0sWLDA5nimenamznw4kkajUToFm5QcUhURUfxcd52BFi2K/z14EGjXznidpa/ljMU8XSdPZ9kH6enGh1Q54r1fsm2OoiIiInuStcNRv359JCYmorCwEN7e3kaXuXfvHhITE1G/fn05my7X4sWLceTIESxbtgwBAQEOa1cJrj7+Wpd+yXHs4eGGyzRoYLrO0tdyxmKerpOns+yDoKDy53DYM09jbRMREdmDrEOqBgwYgPPnz2P8+PG4fft2mfrbt29jwoQJuHDhAgYOHChn0+XavHkzJEnCihUrEBcXZ/DQ3WE8Pj4ecXFx2Ldvn8PyIiIiIiJyd7JOGr9x4wZiYmKQkZGBatWq4bHHHtNfpercuXPYsmULbt++jYiICBw6dAg1a9aUq+lyJ43HxcVh9+7dZsXZsGGD1Z0hWybTEBERERE5K1u+58ra4QCAy5cvY+LEidiyZYvR+r59+2LJkiWoW7eunM2adZUqY6y5SpUp7HAQERERkTuy5Xuu7Df+q1u3LjZv3oxz585h3759uHz5sr68U6dOaNiwodxNEhERERGRk5K9w6HTsGFDu3YutmzZgoSEBIOygoICdOjQQf961qxZ6Nu3r91yICIiIiKi8tmtw1FaamoqUlJSEBYWhna66zraICsrq8y9PIQQBmVZWVk2t0NERERERNaT9SpV//nPf9CtW7cyHYGZM2eiefPmeOKJJxAbG4uhQ4eiqKjIprbGjBkDIUS5jzFjxlQYJyMjA0IIm+dvEBERERFRWbJ2OL755hv8/vvvePDBB/VlSUlJmD9/Pvz9/TF8+HCEh4dj/fr1WL16tZxNExERERGRE5K1w5GSkoJWrVpBpVLpy1auXAlJkrBmzRp8++23OHz4MPz9/fH555/b1FZycjLmzZuH+Ph4hIaGQpIk+Pn5mVz+hx9+wOjRo9GyZUsEBATAx8cHQUFBeOyxx0xeUYuIiIiIiGwj6xyOzMxMxMbGGpTt3LkTQUFBePTRRwEANWvWROfOnZGcnGxTWwkJCdi0aZPZy69cuRLr169H8+bN0b59e/j7+yMjIwM//fQTfvrpJ8yaNQtvvPGGTTkREREREZEhWTscVapUQW5urv71jRs3cPr0aQwdOtRguRo1auDmzZs2tRUbG4vo6GjExMQgJiamwjkY//d//4clS5agdu3aBuVJSUno0aMH3nzzTQwfPhxRUVE25UVERERERPfJOqQqIiICiYmJuHv3LgBg/fr1AKA/u6Fz9epVBAUF2dTWzJkzMXfuXPTr1w/BwcEVLt+6desynQ0AaN++PYYPHw4hBHbt2mVTTmQbrRaQpOKHVuteZc6WD7fPM7ZFye0jIiLSkbXD8cwzzyArKwtdunTB9OnT8fLLL6Nq1aoYMGCAfpl79+7h8OHDaNKkiZxN28Tb2xsA4Ovrq3AmRERERETuRdYhVePGjcPOnTuxZs0aJCUlQa1W44svvjA4s7B582b8/fff6Natm5xNW+348eP4z3/+Ax8fH3Tv3l3pdGymdbGfF0v/6qqTkQFUqQKUGKHnkmWZmca325lylGv7nCEfOcpKUzofV3svZmYCajWgVquhVhtvk4iIPIskhBByBz1//jwyMzPRrFkz+Pv7G9QdPXoU58+fR4cOHcwaCmUuSZKgUqmQl5dX7nKbN2/G999/j7t37+LChQvYv38/fHx88Nlnn5l13w4AaN68udHytLQ0REZG4sSJE5amLxtJkhRrm4joPgH5/3chIiKl6L7/WvM91y53Gm/QoAEaNGhgtO7BBx80uE+Hox07dgwrVqzQv/bz88PixYvx9NNPK5YTEREREZG7sssZDqB4rsaxY8dw+fJlSJKEOnXqIDo6GpUq2aWPY/YZDp28vDycPXsWn376KT799FP07dsX33//vU3zOGzp+cnF1YdURUQUP09JuT9Uo0UL1y1LTzfcXnfePmfIR44yTzhW9ty+9HQOqSIickc2fc8VMsvLyxMzZ84U1atXF15eXgaP6tWri5kzZ4o7d+7I3awAIFQqlVXrvvDCCwKAWLBggU05REVFiaioKJtieDKNRgig+KHRuFeZs+XD7fOMbVFy+4iIyL3Y8j1X1qtU5efno3v37pg/fz5ycnLQqlUrPP7443j88ccRHR2NnJwczJ8/Hz169EB+fr6cTdtk1KhRAGDRjQSJiIiIiKhisg6pmjdvHl577TV06tQJn3zyCVrozrH/T0pKCp577jns3bsXb7/9NmbOnClX0xYPqSrp3LlziIiIQLNmzXDq1Cmrc3CGIVVERERERHKz5XuurGc4Vq9ejcDAQGzdurVMZwMAWrRogR9//BEBAQH49ttv5WzaJrt37wYAREZGKpwJEREREZF7kbXDcfbsWcTFxaFq1aoml6latSri4uKQlpYmZ9PlyszMxHvvvYdbt26Vqdu+fTteeeUVAMDYsWMdlhMRERERkSeQ9ZJRlSpVQm7JO0CZkJuba/PVqrZs2YKEhASDsoKCAnTo0EH/etasWejbty9yc3MxY8YMzJo1C23btkW9evWg1Wpx5swZpKamAgCmTZuGwYMH25QTEREREREZkrXD0bJlS+zYsQPnzp1Dw4YNjS5z7tw57NixA23atLGpraysLCQlJRmUCSEMyrKysgAAQUFBePfdd7Fr1y6cOHEChw8fRlFREerUqYPhw4dj4sSJiIuLsykfIiIiIiIqS9ZJ419//TVGjx6N+vXr44033sCIESP097XIz8/H6tWrMXv2bFy6dAkrV67EyJEj5WraKXDSOBERERG5I1u+58p+47+JEyfiiy++gCRJkCQJwcHBkCQJV69ehRACQghMnDgRn376qZzNOgV2OIiIiIjIHTnNVaoAYMmSJVi7di06deqESpUq4cqVK7h8+TIqVaqEzp07Y+3atbJ0NpKTkzFv3jzEx8cjNDQUkiTBz8/P6LJFRUXYu3cvXnnlFbRv3x5BQUFQqVSIjIzEpEmTcO7cOZvzISIiIiKismQ/w1HSvXv3kJ2dDQCoXbu2zRPFSxo4cGCZG/WZug/H2bNn0bhxYwBAaGgo2rZtCy8vLxw8eBB//fUX/P39sXXrVnTq1MmmnHiGg4iIiIjckS3fc2WdNF4meKVKCA4Otkvs2NhYREdHIyYmBjExMQgJCTG5rCRJ6NWrF1577TV06dJFX56fn49JkyZh+fLlGDlyJM6ePQsfHx+75EtERERE5IlkH1LlKDNnzsTcuXPRr1+/Cjs1kZGR2LZtm0FnAyg+I/Lpp5+ievXquHDhAvbv32/PlB1GqwUkqfih1dqnTO52HZW33Nts7n5wtm2R+zg7W97u/l7k9tlv+4iISH6ydzhOnDiBsWPHIiIiApUrV4a3t7fRh5zDq2zh5+eHJk2aAAAuX76scDZERERERO5F1m/9u3fvRp8+fZCXlwdJklC7dm2o1Wo5m5BdYWEhzp8/DwDlDstyBVotoNVqDX6py8gAqlQBSt6P0ZayzExAd0h1/5b8tdDaNkqTO2+5t9kYY/shM9P4ss60fbYc55Lb5wx5y1FWmtL5yHlMjXGmHOXaPrne887+/xcRkauQddJ4+/btcejQIbz++uuYMWMGqlWrJlfoCkmSZHLSeHm+/fZbjBo1CoGBgbh48SJUKlWF6+gmzZSWlpaGyMhIxSaNSxIASIq0TUTkbux4TRUiIpfjNJPGjx8/jg4dOuCNN96QM6zdXLx4EVOnTgUAvPHGG2Z1NoiIiIiIyHyydjhq1aqFOnXqyBnSbrRaLQYNGoTr169j4MCBmDRpktnrmurZmTrz4SgaDaDVaqDVAhERxWUpKfeHDrRoYXtZenr5Q22sbVfHXnnLvc3GGNsP6emGyzjj9tlynEtunzPkLUeZqx0rS45pSe68ffb8nBMRkRWEjJ599llRr149kZ+fL2dYswAQKpXKrGULCgpEr169BADRqVMnkZubK0sOUVFRIioqSpZYttBohACKHxqNfcrkbtdRecu9zebuB2fbFrmPs7Pl7e7vRW6f/baPiIiMs+V7rqxXqXrzzTfh6+uLsWPH4vbt23KGlk1RURFGjRqFn3/+GdHR0di8eTMqV66sdFpERERERG5J9juNZ2dno2vXrrh8+TLatm2L0NBQSFLZicySJGHZsmWytWvupPFJkyZhyZIlaNKkCfbu3YugoCDZcuCdxomIiIjIHTnNpPHbt29j6NChOHXqFIQQ+OWXX0wuK3eHwxyvvfYalixZgrCwMGzfvl3WzgYREREREZUla4djxowZ2LVrF1q0aIFnn30WERERTnMd84ULF+Kdd95BSEgIfvnlF4SFhSmdEhERERGR25O1w7Fp0ybUr18fiYmJdu9obNmyBQkJCQZlBQUF6NChg/71rFmz0LdvXxw9ehQzZswAADRs2BBvvfWW0Zjjx49Hp06d7Jc0EREREZGHkbXDcefOHXTp0sUhZzWysrKQlJRkUCaEMCjLysoCANy6dUt/A6fExEQkJiYajRkXF8cOBxERERGRjGSdNN61a1fcuXMHBw8elCukS+GkcSIiIiJyR7Z8z5X1srizZs3CkSNHsHHjRjnDEhERERGRi5J1SJWvry+ee+45DBs2DKNGjUKPHj1MXhYXALp06WJ1W8nJydi+fTsOHjyIpKQkXL58udzL4l68eBGbN29GUlISDh48iNOnT0MIgcTERIN5H0REREREJB9ZOxxxcXGQJAlCCCxfvhwrVqwod/nCwkKr20pISMCmTZvMXv7777/HtGnTrG6PiIiIiIgsJ2uH4+mnnzZ5NkNusbGxiI6ORkxMDGJiYhASElLu8hEREZg2bZp++fHjx2P37t0OyZWIiIiIyGMJNwFAqFQqs5fv2rWrACASExNlyyEqKkpERUXJFs/eNBohgOKHRqNsmbPl46rbZwtn2z65c3S2Y+Xu70V32j4iIrLte66sk8aJiIiIiIhKknVIVUn37t3DsWPHcPnyZUiShDp16iA6OhqVKtmtSQKg1WrNWMbwXwDIyACqVAFyc5UpK03pfOQqy8wsu232akd3+xtLb4Njy/uh5PbZc1sc+Z4tzZneT67wXnSG7bPXe9ER95giInJHst6HAwDy8/Mxe/ZsfPbZZ8jJyTGo8/f3x6RJkzBnzhz4+fnJ2SwkSSr3KlWlxcXFYffu3VZdpUp3HeLS0tLSEBkZqeh9OBw1h4aIyNPI/N8lEZFLseU+HLKebsjPz0f37t31d/Ju1aoVwsPDAQDnz5/HsWPHMH/+fOzbtw+//vorVCqVnM0TEREREZGTkbXDsWjRIuzfvx+dOnXCJ598ghYtWhjUp6Sk4LnnnsPevXuxePFizJw5U87mHcZUz87UmQ9H0mg0FS5TcnhKRETx85SU+8MJdIfNkWU6zpKPXGXp6Yb73l7bl54uz5AqS/MruX323BZHvmd1+F50rTJHvReJiMgKcs5eb9WqlQgKChI5OTkml8nJyRFBQUGiZcuWcjbNq1RZwd2vLONMZY5sx1rOtn1y5+hsx8rd34vutH1EROREV6k6e/Ys4uLiULVqVZPLVK1aFXFxcUhLS5OzaSIiIiIickKydjgqVaqE3JKX+DAhNzeXV6siIiIiIvIAsl6lqlOnTvj999+RkpKChg0bGl3m3LlzaNGiBdq0aYO9e/fK1bRDr1Jlii2z94mIiIiInJUt33NlPcMxceJE3LlzB3FxcVixYgUKCgr0dfn5+Vi+fDni4uKQl5eHSZMmydk0ERERERE5IdnvwzFx4kR88cUXkCQJkiQhODgYkiTh6tWrEEJACIGJEyfi008/tamdLVu2ICEhQf86KSkJkiShXbt2+rJZs2ahb9++AIArV65g0KBB+rqTJ08iJycHzZs318856du3L2bNmmV1TjzDQURERETuyGnuwwEAS5YswaOPPooPPvgASUlJuHLlCgDA19cXHTp0wPPPP4/Bgwfb3E5WVhaSkpIMyoQQBmVZWVn65/n5+WWWBwx3WrNmzWzOi4iIiIiI7pP1DMfx48fh5eWlv//GvXv3kJ2dDQCoXbu2208U5xkOIiIiInJHTnOG48EHH0TXrl2xc+fO4uCVKiE4OFjOJoiIiIiIyIXIOmm8Vq1aqFu3rpwhiYiIiIjIhcna4ejQoQP++OMPOUOalJycjHnz5iE+Ph6hoaGQJAl+fn4Vrrdy5Uq0a9cOVatWRa1atfDYY49h//79DsiYiIiIiMjzyNrhmD17Nk6fPo333ntPzrBGJSQk4J///Cc2bNiAy5cvm7XO9OnTMXr0aKSkpKBHjx5o164dtm/fji5dumDDhg12zpiIiIiIyPPIOofj1KlTGDVqFF555RV8/fXX6NevH8LCwkyeeXj66aetbis2NhbR0dGIiYlBTEwMQkJCyl1+x44dWLRoEWrXro3ExEQ0btwYAJCYmIi4uDiMHTsWcXFxqFmzptU5ERERERFRKUJGkiQJLy8vIUmS/uHl5VXmoSuXEwChUqlM1j/22GMCgFi0aFGZuhdeeEEAEAsWLLAph6ioKBEVFWVTDDloNEIAxQ+NxjXKnC0fe2yfuxwrdz9+zpYPt8+x20dERMbZ8j1X1jMc//rXvyBJkpwhZZGXl4dff/0VADBkyJAy9UOGDMEHH3yAzZs346WXXnJ0ekREREREbkvWDsecOXPkDCeb1NRU5OfnIzAwEPXq1StT36ZNGwDF9xFxZVotoNVqodXeL8vIAKpUAXJznbusNKXzkassMxNQq4uf6/4FoD9GrnisSm+fjjPkw/ei6bKSx8rdt0/Oz6q65AsiIrKKrDf+U5IkSVCpVMjLyytT98MPP2DAgAFo3bo1jhw5YnT9mjVr4tatW7h9+zb8/f3LbUt345PS0tLSEBkZqdiN/4pPLjnfGSYiIlflJv9FEhHZzJYb/8l6lSpnpdFoAABVjP18+T+6X7F0yxIRERERke1kHVIFAJmZmfjkk0+wZ88eXLlyBfn5+UaXkyQJaWlpcjdvlO4XqvLml1jyK5apnp2pMx+OotEAWq0GWi0QEVFclpJyf+hAixbOW6bjanlXVJaeXvGQKlfe5vT0+9vkDPnwvWjesXL37ZPzs0pERDKQb+66ECdPnhSBgYFlrlRl6iEnwPRVqjZt2iQAiNatW5tcv0aNGgKAuH37ttU58CpVvHJOedvnLsfK3Y+fs+XD7XPs9hERkXG2fM+VdUjVyy+/jOvXryM+Ph7JycnIyclBUVGRyYejhIWFAQAuXbpktF6r1eLWrVuoUaNGhfM3iIiIiIjIfLJOGq9evTrq1q2LkydPOvzyuOVNGr9z5w5q1qyJ/Px8XLx4scyVqvbu3YsuXbqgS5cu2L17t9U52DKZhoiIiIjIWTnNpHEhBFq2bOl09+KoXLkyunXrBgBYt25dmXpdWb9+/RyaFxERERGRu5O1w9G2bVucPXtWzpCymT59OgDgzTffxJ9//qkvT0xMxJIlS1CtWjWMGzdOqfSIiIiIiNySrB2OOXPm4I8//sCaNWvkDGvUli1b0KFDB/0DAAoKCgzKtmzZol++R48eePHFF5GdnY0HH3wQAwcOxGOPPYYuXbrg7t27+PLLL1GrVi27501ERERE5Elsuizunj17ypS9+OKLGDlyJLZu3YqePXuiXr16JodYdenSxeq2s7KykJSUZFAmhDAoy8rKMqhfvHgxHnzwQXz00UfYvn07fHx80L17d7z++uvo1KmT1bkQEREREZFxNk0a9/LyMtqZ0IWsaC5HYWGhtU07JU4aJyIiIiJ3ZMv3XJvOcDz99NNON0GciIiIiIich00djuXLl8uUhuMcOHAA77zzDn777TdoNBqEhYVh2LBheO2111Cl5G2GiYiIiIjIZjZPGt+xYwdWrlyJkydPVrjsyZMnsXLlSuzcudPWZq3y7bffolOnTvjhhx8QHh6Oxx57DHl5eXjrrbfQsWNH5OTkKJIXEREREZG7sukMx8WLF9G3b1/Ur18fycnJFS5fv359DBo0CJcuXcKff/6JunXr2tK8RS5duoTx48ejsLAQX375JcaOHQsAyM/Px1NPPYW1a9filVdewaeffuqwnIiIiIiI3J1NZziWLl2KgoICvPvuu/D3969weX9/f8yfPx937tzBsmXLbGnaYsuXL0deXh569uyp72wAgEqlwscff4wqVapg2bJlyM7Odmhe9qDVApJU/NBqXaPM2fKxdVvIMs52/JwpH1ved+60fa5w/IiIyDibOhzbt29HYGAgBg4caPY6jz/+OIKDg/HTTz/Z0rTFdGdg4uLiytQFBgYiKioKd+/exdatWx2aFxERERGRO7NpSFVqaioefvhhi9dr27Yt9u/fb0vTFtP+7yeomjVrGq3X3fTv2LFjeOqppxyWl5y02uLtLPlrW0YGUKUKkJvr3GWlKZ2PNWWZmYBaXfxc9y+Vr+SvxTpKH9PSlM7Hmvedsf2amVl225xtW2zdDzpKHoOgIH74iYhKs+k+HCqVCkOHDsU333xj0XqjRo3C2rVrkZ+fb23TFhs5ciRWrVqFmTNnYt68eWXqmzVrhtOnT2Pw4MFYt25dubF01yEuLS0tDZGRkYrdh6P4CsW8TDERkVJs+C+ViMip2XIfDpuGVNWsWRPXrl2zeL1r166ZPNNgL127dgUArF69GgUFBQZ1Bw4cwOnTpwGAV6oiIiIiIpKRTUOqoqKicODAAdy5cweVK1c2a53c3FwkJiaiXbt2tjRtsZEjR+Ktt97ChQsXMGDAACxYsABhYWH47bffMGHCBFSqVAn37t2Dl1fFfTBTPTtTZz4cRaMBtFoNtFogIqK4LCXl/un/Fi2ct0zH1fIuWZaeziFVlio59MdZjr2Os+RjzfvO2H5NTzfc966yfZbsBx0ljwERERkhbLBw4UIhSZJ47bXXzF7nn//8p/Dy8hILFy60pWmrHD9+XISFhQkABo+wsDDxyiuvCABixIgRVsePiooSUVFRMmZsHY1GCKD4odG4Rpmz5WPrtpBlnO34OVM+trzv3Gn7XOH4ERG5M1u+59o0pGrixIkICQnBvHnz8Oabb6KoqMjkskVFRUhISMC8efMQHByMiRMn2tK0VVq2bInU1FSsWLECzz//PCZPnozPPvsMKSkp+mWUPktBRERERORObJo0DhTPf+jevTvy8vJQr149DB06FG3atEFgYCAAICsrC0eOHMHatWtx6dIlqFQq/Prrr4iNjZVlA+Ty0EMP4ciRI9i3b59VV94CbJtMQ0RERETkrGz5nmtzhwMAjh8/jlGjRiElJQWSVPYqSbommjdvjm+++QbR0dG2Nimr3bt3Iy4uDs2bNzc422EpdjiIiIiIyB3Z8j3XpknjOq1atcLx48fx888/Y8uWLfj999+RnZ0NIQQCAgLw4IMPom/fvujdu7cczVnt6NGjaNGiBSpVur/ZR44cwZNPPglJkvDhhx8qmB0RERERkfuRpcOh06tXL/Tq1UvOkLKaOnUqTp48iQcffBABAQHIyMhAUlISvLy8sGTJEjzyyCNKp0hERERE5FZk7XA4u1GjRuGbb77B0aNHcevWLQQGBmL48OF4+eWX8eCDDyqdHhERERGR25FlDgcV4xwOIiIiInJHit1pnIiIiIiIqDwe1+E4cOAABg8ejJCQEPj4+KBWrVro3r071q1bp3RqRERERERux6M6HGvXrsXDDz+M9evXo379+hg8eDBatGiBXbt2YejQoXj11VeVTpGIiIiIyK14zByOe/fuoW7dusjKysJ3332HJ554Ql+XmJiIbt26IT8/H3/++SciIyOtaoNzOIiIiIjIHXEOhxlSU1ORlZWFZs2aGXQ2ACA2Nha9evWCEALJyckKZegYWi0gScUPrdb6cket4wmxlG7fWWMp3T63xX32iylKbwsRkafwmA6HSqUya7latWrZORMiIiIiIs/hMffhiIiIQEREBFJTU7FmzRoMGzZMX5eYmIiff/4ZDRs2RJcuXRTM0nZaEz+f6YpLVmdmGo9RXrlaXRxD968lsaxZR872nS2W0u07MpbS7fMY2z+W0u0bi6VWFz/X/atj6u+ho/6+mcrrfiwTFURELspj5nAAwN69e9G/f3/8/fffaNu2LSIjI3HlyhXs27cP7dq1w9dff41GjRpVGEc3hq20tLQ0REZGKjqHQ5IkxdomIiLbedB/y0TkQmyZw+ExZzgAoHPnzti9ezcGDRqEw4cP4/DhwwAAf39/9OjRA3Xr1lU4QyIiIiIi9+JRHY7Vq1dj7Nix6NChA7777js0b94cly9fxoIFC/Dmm2/i119/xe7du+Hj41NuHFM9O1NnPhxJo9EYLS85hCAiovh5errhMuaUlx5yYEksa9aRs31ni6V0+46MpXT7PMaeeYzNGVKlxN+3ioZUERG5HeEhzpw5I3x8fES9evWERqMpU9+/f38BQCxZssTqNqKiokRUVJQtadqdRiMEUPwouRssLXfUOp4QS+n2nTWW0u1zW9xnv5ii9LYQEbkSW77nesxVqr777jvcvXsXvXv3NjohTzeJfNeuXQ7OjIiIiIjIfXnMkKpLly4BAKpVq2a0Xld+48YNh+WkBLW6+Hc2W8sdtY4nxFK6fWeNpXT73Bb32S+mKL0tRESewmPOcISEhACAfqJ4aYcOHQIAhIeHOyolIiIiIiK35zEdjgEDBgAA9uzZg08//dSg7sCBA1i0aBEAYMiQIQ7PjYiIiIjIXXlMh6NNmzaYMWMGAGDKlClo0aIFhg0bhk6dOuHhhx+GVqvFs88+ix49eiicKRERERGR+/CYORwAMH/+fHTs2BGfffYZkpOTcfr0afj7+6Nr164YP348nnzySaVTJCIiIiJyKx7V4QCAQYMGYdCgQUqnQURERETkETxmSBURERERETmex3Q4du3aBUmSKny88cYbSqdKREREROQ2PGZIVUhICEaPHm20rrCwEN988w0AoHPnzo5Mi4iIiIjIrUlC8HZEP/30Ex577DHUr18fGRkZ8PKy7sRP8+bNAQAnTpyQMz0iIiIiIkXZ8j3XY4ZUlUd3dmPkyJFWdzaciVYLSFLxQ6s1r87Scket4wmxlG6/vFhyUnpbeIydL5bS7Su9LeVReluIiOTk+t+ubaTVarFp0yYAwKhRoxTOhoiIiIjIvXh8h2P9+vXQarVo3bq1/lQRERERERHJw2MmjZuiG0711FNPKZyJ7bTa4jM2JU+NZ2YCanVxne7fknXGlFduSyyl23e2WEq3X14s4P6/ttLl4w77xZ2OsdyxlG5f6ViWvi9065Rm6eelvDaseb8ay0st1x8DIvJYHj1p/OrVq6hXrx4A4NKlSwgJCTFrPVNnQtLS0hAZGanYpHFJAgBJkbaJiMg9efDXBCIqgZPGrbRq1SoUFhaiZ8+eZnc2iIiIiIjIfB49pMra4VSmenZKzwHRaACtVgOtFoiIKC5LTy97ar1kXUnmlNsSS+n2nS2W0u2XFwuwz5AqV98v7nSM5Y6ldPtKx7L0faFbpzRLPy9ybovcn30iIh2P7XCcOnUKv//+O6pWrYqBAwcqnY4s1OrisbYl//MPCjL8z8NUnaXljlrHE2Ip3X55seTgLNvCY+x8sZRuX+ltMcbanO2dFxGRLTx2SNXXX38NAIiPj0eVKlUUzoaIiIiIyD155KRxIQQaNmyI8+fPY/v27ejRo4cscXmncSIiIiJyR5w0bqG9e/fi/PnzqFu3Lrp166Z0OkREREREbssjOxy6yeIjR46El5dH7gIiIiIiIofwuG/b+fn5WLduHQBg1KhRCmdDREREROTePO4qVSqVCjdu3FA6DSIiIiIij+BxZziIiIiIiMhx2OEgIiIiIiK78cgOx9WrVzFt2jQ0adIElStXRq1atfDQQw/hlVdeUTo1IiIiIiK34nEdjsTERDzwwANYvHgxfHx88Pjjj6NDhw7Izs7GwoULlU6PiIiIiMiteNSk8cuXL+Oxxx5Dfn4+1q9fj0GDBhnUHzx4UKHMiIiIiIjck0ed4Xj11Vdx69YtvPvuu2U6GwDQrl07BbKSn1YLSFLxQ6s1r87ScketI3f7lvKE/WLN/vL0/eJq7TtrLKXb57Yo+3dXzr/VROTcPKbDcfPmTaxZswbVq1fH+PHjlU6HiIiIiMgjeMyQqt9++w35+fno0aMHfHx8sG7dOuzbtw93795Fs2bNMGzYMAQHByudpk20WkCr1Rr8UpSZCajVxXW6f0vWGVNeuS2xlGpfrb7fvrl0sS1pw5q8nOG4GIsFGN9f1u4XS9t31v3iTsdY7lhKt690LKXbd7ZY1vwdKbmO2pI/2ETk9CQhhFA6CUf497//jVdffRXPPPMMTp06hcTERIN6tVqNr776CkOHDq0wVvPmzY2Wp6WlITIyEidOnJAlZ0tJEgBIirRNREQkFw/5akLkUnTff635nutRQ6oAYOXKlTh+/DiWLVuGrKwsnDt3DtOnT4dWq8WoUaNw/PhxhTMlIiIiInIfHjOkqrCwEABw7949fPzxx3jmmWcAAAEBAXjvvfdw4cIFrFu3Du+++y6++eabcmOZ6tmZOvPhKBoNoNVqoNUCERHFZenpZU+Hl6wryZxyW2Ip1b6tQ6rsmZczHBdjsYCKh0JYkpeS26JkLKXbd2QspdtXOpbS7TtbLGv+jljzt5qIXIPHdDj8/f0BAF5eXhg9enSZ+meeeQbr1q3Drl27HJyZfHTjXkv+5x8UZPjH21SdpeWOWkfu9s3lSfvFkv3F/eKa7TtrLKXb57bYP5Yxcv6tJiLX4DFDqsLDwwEAISEhUKlUJuszTc2CIyIiIiIii3nMpPE//vgDrVq1QuXKlaHVaiFJhpOr9+3bh86dO6NmzZq4ceOGVW3YMpmGiIiIiMhZcdK4GVq2bImGDRvizp07SEpKKlOvG0rVpk0bB2dGREREROS+PKbDAQAzZ84EALzwwgu4fv26vjw5ORnvvfceAGDSpEmK5EZERERE5I48ZtI4AEyYMAG//vor1q5di6ZNm6Jjx47QaDTYv38/CgoKMGHCBAwZMkTpNImIiIiI3IZHdTi8vLzw3XffIS4uDkuXLsWOHTsgSRLatm2LSZMm4amnnlI6RSIiIiIit+Ixk8YdgZPGiYiIiMgdcdI4ERERERE5JY/qcMTFxUGSJJOPbdu2KZ0iEREREZFb8ag5HDqDBw9G1apVy5SHhoYqkA0RERERkfvyyA7HggUL9HcWJyIiIiIi+/GoIVWeQqsFJKn4odWaV2dpuaPW8YRY1q5jijvtF7m20Rm2xV2OC/eLc7bvrLEc1b4p1vx9ISL5scNBRERERER245FDqpYtW4bs7Gx4eXmhSZMmGDhwIMLCwpROy2ZaLaDVag1+xcnMBNTq4jrdvyXrjCmv3JZYSrfvbLGsXQe4/29JuhjOuF8sXQeQZxtd8Ri7aiyl21c6ltLtO1ssR7Vv6u+Fqb8VxpZXG/tjQ0Sy8qj7cMTFxWH37t1lyn18fDBr1izMmjXLrDi66xCXlpaWhsjISMXuwyFJACAp0jYREZEr8qCvQUQ24X04zNSlSxd8/fXXSEtLQ25uLk6fPo233noLlSpVwr/+9S+8//77SqdIRERERORWPOoMhyn//e9/0atXL1SvXh1XrlxB5cqVrYqj9J3GSw6piogoLktPL3s6umRdSeaU2xJL6fadLZa16wAVDzdytv1i6TpybaMrHmNXjaV0+0rHUrp9Z4vlqPbNGVJV0fIcUkVkHpu+5woSQgjRtm1bAUDs2LHD6hhRUVEiKipKxqyso9EIARQ/NBrz6iwtd9Q6nhDL2nVMcaf9Itc2OsO2uMtx4X5xzvadNZaj2jfFmr8vRGScLd9zPWpIVXkaN24MALhy5YrCmRARERERuQ8OqfqfPn36YNu2bdi0aRMef/xxq2IoPaSKiIiIiMgeOGncRllZWdi7dy8AoE2bNgpnQ0RERETkPjymw3HgwAHs3LkTpU/oZGRkYNCgQdBqtXj88cdRr149hTIkIiIiInI/HnPjv9TUVIwdOxZ16tRBkyZNEBISgkuXLiE5ORl5eXlo3rw5vvjiC6XTJCIiIiJyKx7T4Wjfvj0mT56MpKQknDx5Er/99hvUajUefPBBDB06FJMnT7b6crhERERERGScx3Q4HnjgAXzyySdKp0FERERE5FE8Zg4HERERERE5nkd3OG7cuIGgoCBIkoRmzZopnQ4RERERkdvx6A7H9OnTcf36daXTICIiIiJyWx7b4fj111+xYsUKTJgwQelUiIiIiIjclkd2OO7cuYNJkyYhKioKM2bMUDod2Wm1gCQVP7Ra8+osLXfUOp4QS+72TVE6L0fkrPS2OGv7zhpL6fa5LdwvSv59I/IkHnOVqpLmzp2LtLQ07Nq1Cz4+PkqnQ0RERETktjyuw3H8+HG89957GDt2LLp06YKMjAylU5KNVgtotVqDX1cyMwG1urhO92/JOmPKK7clltLtO1ssudsH7v9bki62UnkZ+7VP7pwrat9djrEzx1K6faVjKd2+s8VSun1HxjJVBwBBQUb+wBF5IEkIIZROwlGKiooQGxuL9PR0pKamonbt2sjIyEDDhg3RtGlTpKammhWnefPmRsvT0tIQGRmJEydOyJm22SQJACRF2iYiIiJDHvQVizyA7vuvNd9zPeoMx4cffoiDBw/iq6++Qu3atZVOh4iIiIjI7XlMh+PixYt4/fXX0bVrV4wZM8amWKZ6dqbOfDiKRgNotRpotUBERHFZenrZU8gl60oyp9yWWEq372yx5G4fqHh4klL7xd45O3JblDzGzhxL6faVjqV0+84WS+n2HRnLVB0RlSA8RL9+/YSvr684efKkQfm5c+cEANG0aVOb24iKihJRUVE2x7GVRiMEUPzQaMyrs7TcUet4Qiy52zdF6bwckbPS2+Ks7TtrLKXb57Zwvyj5943I1djyPddjznD8+OOPqFGjBiZPnmxQnpeXBwC4cOEC4uLi9MtWrVrV0SkSEREREbkdj5k0LknmT6a+efMmatSoYXEbtkymISIiIiJyVrZ8z/WYG/8JIYw+zp07BwBo2rSpvsyazgYREREREZXlMR0OIiIiIiJyPHY4iIiIiIjIbtjhICIiIiIiu/GYq1SZEh4eDg+ZN09ERERE5HA8w0FERERERHbjUR2OhQsXIj4+Ho0bN0b16tWhUqnQoEEDjB49mpeyJSIiIiKyA4+5DwcABAQEQKvVolWrVggNDQVQfC3hM2fOwNfXFxs3bkSfPn2sjs/7cBARERGRO7Lle65HzeHYtGkTHnroIfj5+RmUf/rpp5gyZQrGjx+PCxcuwNvbW6EMiYiIiIjci0cNqXr44YfLdDYAYPLkyWjUqBEuX76M06dPK5AZEREREZF78qgOR3l0ZzV8fX0VzsR2Wi0gScUPrda8OkvLHbWOJ8RSun1njaV0+3Jvi6WcdVt4jJ2zfWeNpXT7zhrL2nWIXBU7HABWrlyJ06dPo0mTJoiIiFA6HSIiIiIit+FRczh05s+fjxMnTkCr1eLUqVM4ceIE6tati1WrVsHLy3X7YFotoNVqDX4RycwE1OriOt2/JeuMKa/cllhKt+9ssZRu35GxlG5fyWMM3P/XHLrYzvo5NhVL6faVjqV0+84WS+n2HRnL3u2X/DuituSPCZET8airVOn06NEDv/76q/51/fr18fXXX6Nr165mra+bpV9aWloaIiMjFbtKlSQBgKRI20RERGRfHviVjZyILVepct2f823wyy+/QAiBmzdvYs+ePWjatCni4uLw1ltvKZ0aEREREZFb8cgzHKXdvXsXsbGxOHLkCJKSkhATE2NVHKXvw1FySJVuKkp6etlTuCXrSjKn3JZYSrfvbLGUbt+RsZRuX8ljDFg/pMqZtqWiWEq3r3Qspdt3tlhKt+/IWPZun0OqyFnY9D1XkBBCiHfffVcAELNmzbI6RlRUlIiKipIxK+toNEIAxQ+Nxrw6S8sdtY4nxFK6fWeNpXT7cm+LpZx1W3iMnbN9Z42ldPvOGsvadYiUZMv3XI8cUmVMQEAAACArK0vhTIiIiIiI3AeHVP3PmDFjsGLFCsyfPx8zZsywKobSQ6qIiIiIiOyBk8bNsHfvXvznP//BvXv3DMrv3r2LDz/8EF9//TUqV66MJ554QqEMiYiIiIjcj8fchyMtLQ1jx45FQEAAHnroIdSuXRvXr1/HH3/8gStXrsDPzw/Lly9H/fr1lU6ViIiIiMhteEyHo2vXrnjttdewe/duHD9+HNevX4evry/Cw8MxZMgQvPDCC2jUqJHSaRIRERERuRWP6XA0bNiQ99kgIiIiInIwj5nDQUREREREjscOBxERERER2Y3HdDhyc3OxceNGjBs3Dq1atUK1atWgVqsRHR2NN954AxqNRukUiYiIiIjcjsd0OFatWoVBgwbhyy+/RFFREXr37o3OnTvj3LlzmD17NmJiYpCZmal0mkREREREbsVjOhy+vr6YPHkyzpw5g5SUFKxZswbbtm3D6dOn0bp1a6SmpmLq1KlKp0lERERE5FY8psPx9NNP45NPPkHjxo0NyuvUqYOPP/4YALB+/XoUFBQokZ7DaLWAJBU/tFrryx21jrPGKo+rbYuzxlK6fUe9X0xRelsckZejtoWfY36OnS2W0u1zW5zz774785gOR3mio6MBAPn5+cjOzlY4GyIiIiIi9+Ex9+EoT3p6OgDAx8cHtWrVUjgb22hNdK91xSWrTU1ZKa9crS6OofvXkljWrCNn+3LEUqvvb0tplu5jpbfFkbGUbt8Z3y+mmHofOWpbTOVcUV5ytS/ntpS3DiDP51jHHd6vcsZSun1HxlK6fR5j+8eydB3A+N8XtSX/GbgZSQghlE5CaRMmTMDSpUvRv39//PDDDxUu37x5c6PlaWlpiIyMxIkTJ+RO0WySJCnWNhEREREZ5+pfuXXff635nuvxQ6q2bt2KZcuWwcfHBwkJCUqnQ0RERETkVjx6SNWpU6cwatQoCCEwf/58/VyOipjq2Zk68+FIpu4nUnKYQERE8fP/jSTTM6e89KlKS2JZs46c7csRy9yhGK6wLY6MpXT7zvh+McXU+8hR22LOkCpXP8Zyfo513OH9Kmcspdt3ZCyl2+cxdr5jDFj2d98jCA918eJFERYWJgCI6dOnyxIzKipKREVFyRLLXjQaIYDih0Zjfbmj1nHWWOVxtW1x1lhKt++o94spSm+LI/Jy1Lbwc8zPsbPFUrp9botz/t13drZ8z/XIIVXXr19Hz549ceHCBYwdOxYLFixQOiUiIiIiIrfkcZPGc3Jy0K1bNxw+fBjx8fFYs2YNvL29ZYlty2QaIiIiIiJnxUnjZsrPz8eAAQNw+PBh9OrVC6tXr5ats0FERERERGV5TIejsLAQI0aMwM6dO9G5c2esX78evr6+SqdFREREROTWPOYqVR999BE2bNgAAAgICMCUKVOMLrdgwQIEBAQ4MjUiIiIiIrflMR2Omzdv6p/rOh7GzJkzhx0OIiIiIiKZeMyQqjlz5kAIUeEjPDxc6VSJiIiIiNyGx3Q4iIiIiIjI8TxmSBUAJCcnY/v27Th48CCSkpJw+fJlqFQq5OXlKZ0aEREREZFb8qgOR0JCAjZt2qR0GkREREREHsOjOhyxsbGIjo5GTEwMYmJiEBISonRKRERERERuzaPmcMycORNz585Fv379EBwcrHQ6dqPVApJU/NBqzauztNxR63hCLKXbd9ZYSrfvTttSHqXzsiZndyHntjvi2DtqHU+I5Qztm+Jq2+KssSqqM8ad/x56VIeDiIiIiIgcy6OGVLk7rRbQarUGveLMTECtLq7T/Vuyzpjyym2JpXT7zhZL6fYdGUvp9j31GKvVxc91/5aki+Fsx7i8nN2FqX1vzbZbE8vSY++K731HxlK6faX+Jii9LY6MJVf7pva/JZ9jtYv+cZSEEELpJJQiSZJVV6lq3ry50fK0tDRERkbixIkTcqRnMUkCAEmRtomIiIjIvpT82q77/mvN91wOqSIiIiIiIrvhkCormOrZmTrz4SgaDaDVaqDVAhERxWXp6WVP75WsK8mccltiKd2+s8VSun1HxlK6fU89xuYOn3CmY+xpQ6ps3XZrYll67F3xve/IWEq3r9TfBKW3xZGx5GrfnCFVbvv3UHgwAEKlUskWLyoqSkRFRckWz1oajRBA8UOjMa/O0nJHreMJsZRu31ljKd2+O21LeZTOy5qc3YWc2+6IY++odTwhljO0b4qrbYuzxqqozhhn/3toy/dcDqkiIiIiIiK74aRxKyaNm2LLZBoiIiIiImfFSeNEREREROSU2OEgIiIiIiK78airVG3ZsgUJCQkGZQUFBejQoYP+9axZs9C3b19Hp0ZERERE5JY8qsORlZWFpKQkgzIhhEFZVlaWo9MiIiIiInJbHj1pXG6cNE5ERERE7oiTxomIiIiIyCmxw0FERERERHbDDgcREREREdkNOxxERERERGQ37HAQEREREZHdsMNBRERERER2ww4HERERERHZDTscRERERERkN+xwEBERERGR3bDDQUREREREdiMJIYTSSbgLf39/3L17F5GRkUqnQkREREQkm7S0NPj4+CAnJ8fidXmGQ0ZqtRo+Pj6KtZ+Wloa0tDTF2ifl8Nh7Nh5/z8Vj79l4/D2XEsfex8cHarXaqnV5hsONNG/eHABw4sQJhTMhR+Ox92w8/p6Lx96z8fh7Llc79jzDQUREREREdsMOBxERERER2Q07HEREREREZDfscBARERERkd2ww0FERERERHbDq1QREREREZHd8AwHERERERHZDTscRERERERkN+xwEBERERGR3bDDQUREREREdsMOBxERERER2Q07HEREREREZDfscBARERERkd2ww0FERERERHbDDocTy8vLw+zZs9GkSRP4+fmhbt26eOaZZ3Dp0iWLY926dQtTp05FgwYNoFKp0KBBA7z44ou4deuW/ImTzeQ69uHh4ZAkyeQjNTXVTltA1kpOTsa8efMQHx+P0NBQSJIEPz8/q+Pxs+865Dz2/Oy7jtzcXGzcuBHjxo1Dq1atUK1aNajVakRHR+ONN96ARqOxOCY/965D7uPvrJ993mncSeXl5aF79+7Yv38/6tSpg86dOyMjIwMHDx5EYGAgEhMTERkZaVas7OxsxMbG4s8//0RERATatm2LEydO4MSJE2jUqBEOHDiA2rVr23mLyFxyHvvw8HCcP38eo0ePNlr/zjvvoE6dOnKmTzYaOHAgNm3aZFCmUqmQl5dncSx+9l2LnMeen33XsXTpUkyYMAEA0Lx5c0RFReH27dvYv38/cnJy0KxZM+zevRtBQUFmxePn3rXIffyd9rMvyCnNmjVLABCxsbEiJydHX/7ee+8JAKJLly5mx3rqqacEABEfHy/u3r2rL3/++ecFAPH000/LmjvZRs5j36BBA8GPuWuZN2+e+Ne//iU2b94srl69KgAIlUplVSx+9l2LnMeen33XsWLFCjF58mRx5swZg/LLly+L1q1bCwBixIgRZsfj5961yH38nfWzzzMcTuju3bsICgrCrVu3cOTIEbRu3dqgPjo6GsePH8fhw4fx0EMPlRvr6tWrCA0Nhbe3Ny5evIjg4GB9XX5+PurXr48bN27gr7/+MqgjZch57IH7v3TwY+66JEmy6ldufvZdn7XHHuBn310kJiaiY8eOUKlUuH37Nnx9fctdnp9792Lp8Qec97PPORxOaN++fbh16xYiIyPLfOEEgCFDhgAANm/eXGGsn376CUVFRejSpUuZPy4qlQr9+/dHYWEhfvrpJ3mSJ5vIeezJs/GzT+T6oqOjARR3FrKzsytcnp9792Lp8XdmlZROgMo6duwYAKBNmzZG63XluuVsjfXll1+aFYvsT85jX9L8+fORlpYGlUqF5s2bY9CgQQgMDLQtWXJq/OwTwM++q0tPTwcA+Pj4oFatWhUuz8+9e7H0+JfkbJ99djic0IULFwAA9erVM1qvK9ct56hYZH/2Ol6vvPKKwetp06bhgw8+wLhx46zIklwBP/sE8LPv6t5//30AQO/evaFSqSpcnp9792Lp8S/J2T77HFLlhHSXQKtSpYrRerVabbCco2KR/cl9vB5//HGsX78e58+fR25uLlJSUjB9+nTk5+dj/Pjx2Lhxoyx5k/PhZ9+z8bPv+rZu3Yply5bBx8cHCQkJZq3Dz737sOb4A8772ecZDiekm+gjSVK59Y6ORfYn9/H64IMPDF43b94c7733Hpo2bYqJEydi5syZGDhwoFW5knPjZ9+z8bPv2k6dOoVRo0ZBCIH58+frx/JXhJ9792Dt8Qec97PPMxxOyN/fHwCg1WqN1ufm5gIAqlat6tBYZH+OOl7jx49HUFAQzpw5g3PnztkUi5wTP/tkDD/7zu/SpUvo3bs3bt68ienTp+PFF180e11+7l2fLce/PEp/9tnhcEJhYWEAYPKu0rpy3XKOikX256jj5eXlpb954JUrV2yKRc6Jn30yhp9953b9+nX07NkTFy5cwNixY7FgwQKL1ufn3rXZevzLo/Rnnx0OJ6Q7dXbkyBGj9bryVq1aOTQW2Z8jj9fNmzcB8Jcud8XPPpnCz75zysnJQZ8+fZCamor4+Hh88cUXJodGmcLPveuS4/hXRNHPvsNvNUgVys/PF9WrVxcAxJEjR8rUt2rVSgAQBw8erDDW5cuXhZeXl/D19RXXrl0zqMvLyxOBgYHCy8tLXLlyRbb8yXpyHvvypKSkCEmSRJUqVUR+fr5Nsci+YOXdpvnZd33WHvvy8LPvnPLy8sQjjzwiAIhevXpZfWz4uXdNch3/8ij92WeHw0n93//9nwAgOnbsKDQajb78vffeEwBEp06dDJb/8MMPRdOmTcWrr75aJtbIkSMFADF48GBx9+5dffkLL7wgAIhRo0bZb0PIYnId+23btonDhw+XiX/s2DHxwAMPCADihRdesM9GkGwq+tLJz777svbY87PvWu7duycGDRokAIjOnTsLrVZb4Tr83LsPOY+/M3/2eZUqJ/X666/jl19+wf79+9G4cWN07twZ58+fR1JSEmrXro2vvvrKYPnr16/j9OnTRsflLV68GAcOHMD333+PZs2aoW3btjhx4gRSUlIQGRmJRYsWOWqzyAxyHfvExETMnTsXDRo0QGRkJAIDA3Hu3DkcOXIE9+7dQ9euXfHOO+84ctPIDFu2bClzCcSCggJ06NBB/3rWrFno27cvAH723Ylcx56ffdfy0UcfYcOGDQCAgIAATJkyxehyCxYsQEBAAAB+7t2JnMffmT/77HA4KT8/P+zcuRPvvPMOVq1ahY0bN6JmzZoYPXo0EhISUL9+fbNjBQQE4NChQ5g9ezY2btyIDRs2IDg4GM899xzmzp1r8d0ryb7kOva9evXCxYsXcejQIRw7dgx///03qlWrhk6dOmHkyJEYO3YsvL297bw1ZKmsrCwkJSUZlAkhDMqysrLMisXPvmuR69jzs+9adOPqAei/eBozZ84c/RfO8vBz71rkPP7O/NmXhOBFmYmIiIiIyD54lSoiIiIiIrIbdjiIiIiIiMhu2OEgIiIiIiK7YYeDiIiIiIjshh0OIiIiIiKyG3Y4iIiIiIjIbtjhICIiIiIiu2GHg4iIiIiI7IYdDiIiIiIisht2OIiIiIiInExycjLmzZuH+Ph4hIaGQpIk+Pn5yd7OgQMHMHjwYISEhMDHxwe1atVC9+7dsW7dOtnakIQQQrZoRERERERks4EDB2LTpk0GZSqVCnl5ebK1sXbtWgwfPhxFRUVo27YtIiMjcfnyZfz2228oKirCzJkzMW/ePJvb4RkOIiI7yc3NxYcffohHH30UderUga+vL/z9/REVFYWxY8di8+bNKCoqUjpNxWRkZECSJMTFxdkl/q5du+waX2n23n9EpKzY2Fj861//wubNm3H16lXZ49+7dw//+Mc/UFRUhO+++w6HDh3Cd999hz179mDfvn3w8/PDu+++i7S0NJvbYoeDiMgOEhMT0bhxY7zwwgvYu3cvmjRpgsGDB6NHjx7w8vLC8uXL8fjjj6N169ZKp0pOavny5ZAkCXPmzFE6FSJSwMyZMzF37lz069cPwcHBssdPTU1FVlYWmjVrhieeeMKgLjY2Fr169YIQAsnJyTa3VcnmCEREZODo0aPo1q0b8vPz8eqrr+K1116Dv7+/wTIZGRlYuHAhvvrqK4WyVF5oaChOnTqFKlWqKJ2KS+L+I6LSNBoNFi5ciLVr1yItLQ2VKlVC69atMW3aNAwcONBgWZVKZVbMWrVq2ZwXz3AQEclICIFRo0YhLy8Pb7/9Nt55550ynQ0ACA8PxwcffIBdu3Y5Pkkn4ePjg2bNmiEsLEzpVFwS9x8RlXTt2jW0b98es2fPxs2bN9GzZ0+0b98eycnJGDRoUJm5GBEREYiIiEBqairWrFljUJeYmIiff/4ZDRs2RJcuXWzOjR0OIiIZbd26FSdOnECDBg3wyiuvVLj8Qw89ZLQ8IyMDEydORHh4OFQqFQIDAzFkyBAcP368zLIlh95cuHABTz75JAIDA1G5cmW0bdsWmzdvLjffnj17ombNmvDz80PTpk3x6quv4tatW2WWnTNnDiRJwvLly5GcnIw+ffqgRo0aqFWrFoYNG4ZLly4BALRaLV5++WWEh4fDz88PLVq0MHq1k4rmICQmJmLYsGGoW7cuVCoVQkND0atXL3zzzTcmt8dWluwPnZ9++gn9+vVDUFAQVCoVwsLCMHDgQGzZssVguS1btuCZZ57BAw88gGrVqkGtViM6Ohpvv/028vPzDZaNi4vD2LFjAQBz586FJEn6x/LlywFUvP++/vprdOrUCdWqVUOVKlXQqlUrvPPOO0YnnI4ZMwaSJGHXrl3Ys2cPunXrBn9/f1SrVg19+/bFyZMnzd+JRKSIsWPH4uTJk3jllVdw7tw5bNq0Cdu3b8fx48cRGRmJ119/3eD/EG9vbyxfvhzVq1fHE088gZiYGAwfPhxdu3ZFp06d8OCDD+K///0vfH19bU9OEBGRbP7xj38IAOKll16yOsbevXtFtWrVBADRvHlzMWTIEBEbGyskSRKVK1cWO3bsMFj+q6++EgDE6NGjRVBQkAgLCxMDBw4UsbGxAoDw8vISP//8c5l23n77bQFAVKpUSXTv3l088cQTol69egKAaNKkibh69arB8rNnzxYAxKRJk4RKpRLNmzcXgwcPFo0aNdKvc+vWLRETEyNq164t+vXrJ+Li4oQkSUKSJLFt2zaDeOfOnRMARNeuXcvktmjRIiFJkgAgYmJixPDhw0W3bt1EQECAaNCggVn7cefOnSbjG2Pp/hBCiOnTpwsAwtvbW3Tq1EkMHz5cdOnSRVSrVq1Mu8HBwaJq1aqiffv2YujQoaJXr16iZs2aAoDo1q2buHfvnn7Zd955Rzz88MMCgIiOjhajR4/WP/bu3Vvh/nv22WcFAOHn5ycee+wxMWTIEBEQECAAiNjYWJGbm2uw/OjRowUAMX36dOHt7S2io6PF4MGDRZMmTQQAUbt2bXHlyhWz9iMR2QcAoVKpjNb9/vvvAoDo2LGjKCoqKlO/ceNGAUA8//zzZeqOHj0qGjZsKADoH/7+/uL1118XWq1WntxliUJEREIIITp16iQAiG+++caq9f/++28REhIifHx8xNq1aw3qtm/fLnx9fUVoaKjIz8/Xl+s6HLr/TO7evauvW7x4sQAgOnfubBDr4MGDwsvLS/j7+4ukpCR9eV5enhg6dKgAIIYOHWqwjq7DAUAsWrRIX15QUCB69OghAIioqCgRFxcnbty4oa9funSpACC6dOliEM/UF+bdu3cLSZJEtWrVxM6dOw3q8vPzy3RcTLGkw2HN/vj6668FAFGvXj1x7NgxgzqNRiN+/fVXg7INGzYIjUZjUHb79m3Rr18/AUCsWLHCoE53XGfPnm00Z1P7b926dQKACA0NFX/++ae+/O+//9a/P19++WWDdXQdDi8vL7Fq1Sp9+b1798TgwYMFADFr1iyjeRCRY5TX4Xj33XcFAPHvf//baP3169cFANGhQweD8lWrVgmVSiW6du0qkpKShEajEWfOnNH/aBEbGysKCgpsz93mCEREpNesWTMBwOSX4pK/VOseBw4c0NcvWrRIABD//Oc/ja4/depUAUB8//33+jLdF9OIiIgy/zHcvXtX1KxZU/j4+Bh0Up5++mmTXyKvXbsmKleuLLy8vMSlS5f05boOR+mOgxBC/PDDD/pf+kt+yRWi+EtrQECA8PHxMcjP1BfmPn36CABiwYIFRveBuSzpcFizPx544AEBQKxbt86mPP/8808BQMTHxxuUW9vh6NKliwAgli1bVmad48ePC0mShL+/v8H7QdfhGDVqVJl1kpOTLTpTRET2UV6HY8qUKQZnKEw9GjVqpF/nzJkzwsfHR9SrV6/MjyFCCNG/f38BQCxZssTm3HmVKiIiGYn/3UtVkiSj9StWrChT1rt3b7Rv3x4AsH37dgAoczURnU6dOmHx4sU4dOgQ4uPjDeri4uLg4+NjUFapUiVEREQgOTkZ2dnZqFOnDgBg7969AICRI0eWaSMoKAiPPvooNm3ahP3792Po0KEG9T179iyzTkREBIDiyfCNGjUyqPP29kZ4eDgOHz6M69ev63MwprCwUD+R/tlnnzW5nNws3R+XL1/GqVOnULt2bQwePNjsdv78809s3boVZ8+ehVarRVFRkf498+eff9q8HXfv3sWBAwcgSRKefPLJMvUtW7ZEq1atcOzYMRw7dgwxMTEG9Y8++miZdZo0aQIAuHLlis35EZF9FBYWAgA6d+6s/3tsTEBAgP75d999h7t376J3795Qq9Vllh02bBg2b96MXbt22fz3mB0OIiIZBQQE4PTp07h+/brRet2XS6B4om7pDkhGRgYA6DsgphiLX69ePaPLVq1aFQAMJiZfvnwZkiShQYMGRtcJDw/XL1daaGhomTLdf1bG6krWl54cXdr169dx584dBAUFGb26l71Yuj8uXrwIAIiMjDQrvhACM2bMwKJFiwzeAyXl5ORYmHVZ2dnZKCgoQEhICPz8/IwuEx4ejmPHjhk9tsbeQ8beP0TkXHSf3SFDhuCFF14wax3dhT6qVatmtF5XfuPGDZvzY4eDiEhG0dHR+O2333DkyBGjvzBXRPcr1dChQ8u9v4KxDompsyq2MBazvHbkysEe2yKH0nmZm+d//vMfLFy4EPXq1cPixYsRGxuLwMBA+Pj4oKCgACqVymRHRI48zV3GWfc7EZWvR48emDVrFjZu3Gh2hyMkJAQAcPjwYaP1hw4dAnD/Bxdb8LK4REQy6tOnDwBg3bp1KCoqsnh93a9Ur7/+OpYvX27yMX78eJvyrFu3LoQQOH/+vNF6XXl5w5/sISAgAJUrV8a1a9dk+cXfXJbuj/r16wMAzp49a1b8DRs2AAA+/fRTDB48GHXr1tUPf0tPT7cp95Jq164NX19fXL16FXfu3DG6jFLHlojsp0OHDujevTt27tyJadOmQaPRGNQXFRXhv//9L/bt26cvGzBgAABgz549+PTTTw2WP3DgABYtWgSg+KyJrdjhICKS0WOPPYYHHngA58+fx7///W+L1+/RowcAYOPGjTJnZqhz584AgG+//bZMXVZWFv773//Cy8sLHTt2tGsepXl7e+vvK/HFF184rF1L90fdunXxwAMPIDs7G+vXr68w/s2bNwHc76iUVPqGWzq6a9/fu3fPvI1A8c0AO3ToACEEVq9eXaY+JSUFx44dg7+/P6Kjo82OS0SOt2XLFnTo0EH/AICCggKDspL3+/n222/RqlUrLF68GA0aNED37t0xfPhwdO7cGSEhIejVq5fB2Yw2bdpgxowZAIApU6agRYsWGDZsGDp16oSHH34YWq0Wzz77rP7/JVuww0FEJCMvLy98/fXXUKlU+L//+z/885//NPpL/fnz53HmzJky5RMnTkRgYCDefvttfPXVV2WG2Wi1WqxcuVI/9tZa//jHP+Dl5YX333/f4D+ggoICPP/888jNzUV8fLzJORn2NHPmTEiShISEBP1kbp27d+/i559/lr1Na/bHq6++CgCYOnUqTpw4YRBPq9Vix44d+te6ideff/65wTHdu3cv5s+fbzSnunXrAgBOnz5t0bY8//zzAIDZs2cbnD3JycnBc889ByEEJk6cKM/NvIjIbrKyspCUlKR/AMXzwUqWZWVl6ZcPDg7GgQMHsHDhQjRu3BiHDh3Cxo0bcenSJbRu3Roff/wxRo0aZdDG/PnzsX79ejz66KO4evUqNmzYgJMnT6Jr16749ttvsWTJEnk2xubrXBERURl79+4VwcHB+puvdenSRQwfPlwMGDBAPPTQQ8LLy0t/Y79Tp04ZrLtv3z5Rq1YtAUA0aNBA9O3bV8THx4u2bdsKtVotAIjff/9dv3xFl0/t2rWrACDOnTtnUP7WW2/pb3TXo0cPMXz4cFG/fn0BQDRu3Njkjf+++uqrMm2UdxM6UzmUt878+fP1l3Fs166dGDFihOjevbtVN/7z9/cX7du3N/k4c+aMVftDCCGee+45/eWAO3fuLEaMGCG6du1a5sZ/p0+f1h+7qKgoMXz4cNG5c2chSZKYMWOG/liXdOfOHREUFKTfR2PHjhXjxo0Tv/32W4X7T3cN/cqVK4u+ffuKoUOHisDAQP11+EvfzEt3WdzS9z3RMZYfEZG52OEgIrITjUYjFi1aJLp16yaCgoKEj4+PqFq1qmjWrJl46qmnxA8//GBwd+mS/vrrL/HSSy+JZs2aicqVK4uqVauKJk2aiCeeeEL85z//MXrjP0s7HEII8eOPP4ru3buL6tWrC19fX9GoUSPxyiuvGNy4T8eRHQ4hhNi1a5cYMGCACAwMFD4+PiI0NFT06tVLfPvtt0aXL03X4ajoUbLzZsn+0NmwYYN49NFHRc2aNYWvr68ICwsTgwYNElu3bjVY7uTJk6J///4iKChIVKlSRbRu3Vp8/vnnQgjTX+gPHTokevbsKapXr66/87pu/1e0/1auXCk6duwoqlatKvz8/ETz5s3FW2+9VeYu40Kww0FE9iUJIeNlMYiIiIiIiErgHA4iIiIiIrIbdjiIiIiIiMhu2OEgIiIiIiK7YYeDiIiIiIjshh0OIiIiIiKyG3Y4iIiIiIjIbtjhICIiIiIiu2GHg4iIiIiI7IYdDiIiIiIisht2OIiIiIiIyG7Y4SAiIiIiIrthh4OIiIiIiOyGHQ4iIiIiIrIbdjiIiIiIiMhu2OEgIiIiIiK7YYeDiIiIiIjshh0OIiIiIiKym/8H66gD8CMCd+EAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 900x600 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "%matplotlib inline\n",
    "color_map = {\n",
    "    'genome':'blue',\n",
    "}\n",
    "#plt.style.use('dark_background')\n",
    "from matplotlib.lines import Line2D\n",
    "\n",
    "fig, ax = plt.subplots(figsize=(6,4),dpi=150)\n",
    "for _i, _chr in enumerate(sorted_chrs):\n",
    "    ax.hlines(_i, 0, chr_2_len[_chr], color='black', linewidth=1)\n",
    "    ax.scatter(chr_2_centers[_chr], \n",
    "               np.ones(len(chr_2_centers[_chr]))* _i, \n",
    "               marker='+', linewidth=1,\n",
    "               s=15, c=[color_map[_t] for _t in chr_2_types[_chr]],\n",
    "               )\n",
    "# create legend patch\n",
    "legend_element = [Line2D([0],[0], color=[0,0,0,0], \n",
    "                         label=_type, marker='+', markersize=8,\n",
    "                         markeredgecolor=_c, markerfacecolor=_c,)\n",
    "                  for _type,_c in color_map.items()]\n",
    "ax.legend(handles=legend_element)\n",
    "ax.set_yticks(np.arange(len(sorted_chrs)))\n",
    "ax.set_yticklabels(sorted_chrs)\n",
    "#ax.set_xlim([0, chr_2_len['1']])\n",
    "ax.set_xlabel(\"Genomic Location\")\n",
    "ax.set_ylabel(\"Chromosome\")\n",
    "#ax.legend([_sc])\n",
    "plt.savefig(os.path.join(report_folder, 'target_location_summary.png'), \n",
    "            dpi=300, transparent=True)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 167,
   "id": "7b68187e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAA2gAAAIkCAYAAABr+wieAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjUuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8qNh9FAAAACXBIWXMAABcSAAAXEgFnn9JSAACl10lEQVR4nOzdd1zV1R/H8ddlC6KI4ATElbhHjtw7za3ZMM1dmg1HZVqWprY0c2SWv9QsR45cucqtmWZuc+BWxIkyFJB9f38QF0hQ1Av3Au/n43Ef3u/5nnvO5/5+WXw45/s5BqPRaEREREREREQszsbSAYiIiIiIiEgiJWgiIiIiIiJWQgmaiIiIiIiIlVCCJiIiIiIiYiWUoImIiIiIiFgJJWgiIiIiIiJWQgmaiIiIiIiIlVCCJiIiIiIiYiWUoImIiIiIiFgJJWgiIiIiIiJWQgmaiIiIiIiIlVCCJiIiIiIiYiWUoImIiIiIiFgJJWgiIiIiIiJWQgmaiIiIiIiV2b9/P59//jldunShePHiGAwGnJyczD7PX3/9xbPPPkuRIkWwt7fH3d2d5s2b88svv5h9LskYg9FoNFo6CBERERERSdapUydWrVqVqs3R0ZGoqCizzbF06VJefPFFEhISqFmzJqVLl+bKlSv8+eefJCQk8N577/H555+bbT7JGCVoIiIiIiJW5osvviAyMpJatWpRq1YtihQpYtYELS4ujmLFihEUFMSiRYt44YUXTPd2795Ns2bNiI6O5vTp05QuXdosc0rGKEETEREREbFyBoPBrAna0aNHqVy5Mn5+fpw4ceKe+0kreIsXL+b55583y5ySMXoGTUREREQkBwgPD2fs2LFUrlwZZ2dn8uXLR+PGjVm5cuU9fR0dHTM0pru7u5mjlAfRCpqIiIiIiJV70Ara9evXadasGcePH6d48eI8+eSTREZGsnv3biIiIvjss88YMWKEqX98fDxPPPEE586du2eVLGmLY9GiRfH398fBwSHTv58kU4ImIiIiImLlHpSgtWnThvXr1zN8+HDGjx+Pvb09AOfOnePpp5/mwoULHDhwgCpVqpg+88cff9C+fXvCwsJMRUKuXr3Kzp07qV27NvPmzaNMmTJZ8v0kmbY4ioiIiIhkY4cOHWL9+vXUq1ePzz//3JScAZQqVYpJkyYRHx/PrFmzUn2uYcOGbN++nZIlS7Jv3z4WL17Mjh07cHFxoUWLFhQrViyrv4qgBE1EREREJFvbuHEjAB07dsRgMNxzv0GDBgDs3bs3VfvPP/9MnTp18PHxYc+ePYSHh3Pq1Cm6devG+PHjadGiBbGxsZn/BSQVbXEUEREREbFy99vi+PrrrzNjxowHjlGmTBlOnz4NwOnTp6lYsSKFCxfG398fFxeXVH07dOjA6tWrmTlzJq+++qp5voRkiJ2lAxARERERkUcXHx8PJG5ZLFWqVLr9PDw8TO8XLVpEbGwsrVu3vic5A3j++edZvXo127ZtU4KWxZSgiYiIiIhkY15eXgB07dqVt956K0OfCQwMBCBfvnxp3k9qDw4ONkOE8jD0DJqIiIiISDbWokULgDTPO0tPkSJFANi3b1+a95OeV/P19X2s2OThKUETEREREcnGnnrqKZo3b87WrVsZOnQo4eHhqe4nJCSwYcMGdu7caWrr2LEjADt27ODbb79N1f+vv/5i8uTJQOKqnGQtFQkREREREbEya9euZdy4cabrPXv2YDAYqF27tqntww8/pG3btkDiQdVPP/00R44cwd3dnWrVquHp6cnly5c5efIkQUFBTJ48mSFDhpg+/+677/Lll18CULFiRSpUqMCVK1fYvXs3CQkJvPrqq8ycOTNrvrCY6Bk0ERERERErExQUxJ49e1K1GY3GVG1BQUGm94ULF+avv/7iu+++Y/Hixezdu5eYmBiKFi1K9erV6dixI88//3yq8SZOnEi9evX47rvv2L9/PydPnsTV1ZXGjRvTv39/Xnrppcz9kpImraCJiIiIiIhYCT2DJiIiIiIiYiWUoImIiIiIiFgJJWgiIiIiIiJWQkVCzMjGxgaj0Yijo6OlQxERERERMZvo6GgMBgMJCQmWDiXH0wqaGaneioiIiIjkVPpZN2toBc2MklbOoqKiLByJiIiIiIj5ODk5WTqEXEMraCIiIiIiIlZCCZqIiIiIiIiVUIImIiIiIiJiJZSgiYiIiIiIWAklaCIiIiIiIlZCCZqIiIiIiIiVUIImIiIiIiJiJZSgiYiIiIiIWAklaCIiIiIiIlZCCZqIiIiIiIiVUIImIiIiIiJiJZSgiYiIiIiIWAklaCIiIiIiIlZCCZqIiIiIiIiVUIImIiIiIiJiJZSgiYiIiIiIWAklaCIiIiIiIlZCCZqIiIiIiIiVUIImIiIiIiJiJZSgiYiIiIiIWAk7Swcg2dfkjafMPubQlk+YfUwRERERkexCK2giIiIiIiJWQgmaiIiIiIiIlVCCJiIiIiIiYiWUoImIiIiIiFgJJWgiIiIiIiJWQgmaiIiIiIiIlVCCJiIiIiIiYiWUoImIiIiIiFgJJWgiIiIiIiJWQgmaiIiIiIiIlVCCJiIiIiIimS4yMpKVK1fSr18/qlSpQr58+XBxcaFq1aqMHTuW8PDwhxrP19cXg8GQ7svf3z+TvknmsrN0ACIiIiIikvMtXLiQV155BYCKFSvSunVrbt++za5duxg9ejQ///wz27dvp1ChQg81bq9evdJsz58//2PHbAlK0EREREREJNM5ODjw2muvMXToUMqWLWtqv3r1Km3btuXgwYMMGTKEhQsXPtS4c+fONXOklmUwGo1GSweRUzg5OQEQFRVl4UiyxuSNp8w+5tCWT5h9TBERERF5PJn9c+7u3bupV68ejo6O3L59GwcHhwd+xtfXl4sXL5LT0hk9gyYiIiIiIhZVtWpVAKKjo7l165aFo7EsbXEUERERERGLOnfuHAD29va4u7s/1GcnTpzI2bNncXR0pGLFinTu3BlPT8/MCDNLKEETERERERGLmjp1KgCtW7fG0dHxoT47fPjwVNdDhw5l2rRp9OvXz2zxZSUlaLlRQACsXw89eoCLi6WjEREREZFsICYmhooVK6Z579ixY4887rp165g9ezb29vaMGzcuw5/r0KEDTZs25cknn8TT05Nz584xZ84cpk6dSv/+/SlYsCCdOnV65LgsRc+g5UYzZsDAgeDlBe+8A+fPWzoiEREREcmFTpw4QY8ePTAajUycONH0LFpGTJs2jc6dO+Pj40OePHmoWLEikyZNYsaMGQC89957mRV2plIVRzPKFlUc795NTMyCg5PbDAbo0AHefBOaNUu8zgBVcRQRERHJHTLj59zAwEDq169PQEAAw4YNY9KkSWYZNyEhgaJFi3Ljxg3OnTtHyZIlzTJuVtEKWm5z9y507w6ursltRiOsWgUtWkDlyjBzJkREWC5GEREREcnRbt68ScuWLQkICKBPnz58+eWXZhvbxsaG0qVLA4lnrGU3StByG3d3mDYNAgMT/3ziPytWx45p+6OIiIiIZJo7d+7wzDPP4O/vT5cuXfj+++8xZHAHV0aFhIQAkDdvXrOOmxWUoOVW+fIlbmk8cSKxYMgzz6S+HxoKkyZB6dLQsSNs3py40iYiIiIi8oiio6Pp2LEj+/bto1WrVvz888/Y2tqadY5jx45x8uRJnJ2d8fPzM+vYWUEJWm5nYwOtW8O6dXDyJLz11r3bH3/9NXH7Y6VK8N132v4oIiIiIg8tPj6ebt26sXXrVho2bMjy5ctxcHC472emT5+On58fI0eOTNX++++/s3///nv6HzlyhOeeew6j0Uj//v0fOL41Upl9SfbEEzB1KowbBz/9BF9/DadSFAI5fhxeew1GjIB+/chftQ1hRb0tF6+IiIiIZBvTp09nxYoVAHh4eDBo0KA0+3355Zd4eHgAic+qnTx58p5nyXbv3s3HH39MiRIlKF26NJ6enpw/f54DBw4QFxdH48aN+eyzzzL3C2USJWhyr3z54I03YNAg2LAhMVFbty75flgYfPUVfQyTOVenCYc69SSget0MV38UERERkdwn6bkwwJSopWXMmDGmBC09rVq14tKlS+zdu5fDhw8TFhZGvnz5aNCgAd27d6dPnz5m3zqZVVRm34yyRZn9R3XqFHzzDfzwA9y5c8/tWyXKcKhDd0606Ehsnkc//Fpl9kVERESsT47+OdfKKEEzo1zxD+6dO/Djj/duf/xXlIsrx1o9y+EO3Qkr5vPQwytBExEREbE+ueLnXCuhBM2MctU/uAkJLP/iB6qtnEepv7ffc9toMCRuf+z4MgE16mV4+6MSNBERERHrk6t+zrUwJWhmZA3/4E7eeO+q1uNKL2lKmsvt8gWq/rqAir8vwzHy3gqPt3xKs23QqMRE7RHnEhERERHLsYafc3MLldmXxxZa3Jftr33A9wt3sOX1Dwn2KpnqfsGAs7T5ZCi2MdEWilBEREREJHtQgiZmE+ucl8Mde/DjrHUs/3QW5+o0Md3LcyeUkmlshRQRERERkWRK0MT8bGy4WLMhq8bN5ESz9qZmv82/WjAoERERERHrpwRNMtWJ5h1M70vt2Ybj7VDLBSMiIiIiYuWUoEmmCqhRjwi3ggDYxsXyxI7fLByRiIiIiIj1UoImmcpoa8fJpm1N1+W3aJujiIiIiEh6lKBJpvNPsc2x+NH95Lt6yYLRiIiIiIhYLyVokumul62UqvS+39Y1FoxGRERERMR6KUGTzGcwpCoW4rf5V9D56CIiIiIi91CCJlnCP0W5/YKXzlHo9DELRiMiIiIiYp2UoEmWuF3Um8uVnjRdl9eZaCIiIiIi98jWCdpff/3Fs88+S5EiRbC3t8fd3Z3mzZvzyy+/pPuZn376idq1a5M3b17c3d1p06YNu3btysKoc68TzZK3OZbbthZDfJwFoxERERERsT7ZNkFbunQp9evXZ/ny5Xh7e/Pss89SqVIltm3bxnPPPceIESPu+cywYcPo1asXR48epUWLFtSuXZuNGzfSqFEjVqxYYYFvkbucatSaeDt7AFxCbuJzcLeFIxIRERERsS7ZMkGLi4vj9ddfJyEhgUWLFrF3714WLVrEjh072LlzJ05OTkyYMIGzZ8+aPrNlyxYmT55MwYIFOXz4MCtXruS3335jx44d2Nra0qdPH0JCQiz4rXK+6HxunK/d2HRdftMqC0YjIiIiImJ9smWC5u/vT1BQEH5+frzwwgup7tWtW5dWrVphNBrZv3+/qX3SpEkAjBo1irJly6bqP3DgQMLCwpgzZ07WfIFcLGU1xzJ/bsL+boQFoxERERERsS7ZMkFzdHTMUD93d3cAoqKi2Lx5MwBdu3a9p19S2+rVq80UoaTnfJ0mRLm4AmAffZfSf26ycEQiIiIiItYjWyZopUqVolSpUvj7+7NkyZJU93bv3s3vv/9OyZIladSoEZC44hYdHY2npydeXl73jFejRg0Ajhw5kvnB53LxDo6cbtTadF1+i6o5ioiIiIgksbN0AI/C1taWuXPn0r59e1544QUmTpxI6dKluXr1Kjt37qR27drMmzcPBwcHAAICAgDSTM4AXFxccHNzIyQkhDt37uDq6nrf+StWrJhme0xMjGlOSd+J5h2ovH4pAD4HduEcHESku6eFoxIRERERsbxsuYIG0LBhQ7Zv307JkiXZt28fixcvZseOHbi4uNCiRQuKFStm6hseHg6As7NzuuO5uLik6iuZ53KlmtwulPj/j01CAuW2rbVwRCIiIiIi1iHbJmg///wzderUwcfHhz179hAeHs6pU6fo1q0b48ePp0WLFsTGxgJgNBoBMBgM6Y6X1Ccjjh07luZLq2cZZGODf7P2pksdWi0iIiIikihbJminT5+mV69eeHp6snbtWmrXro2Liwtly5Zl5syZtG/fnt27d/PDDz8AmLYsRkSkXzEwMjISgLx582b+F5BU1RwLnz6Ge8DZ+/QWEREREckdsmWCtmjRImJjY2ndurVpa2JKzz//PADbtm0DwMfHB4DAwMA0x4uIiCA0NBQ3N7cHPn8m5hFcogzXy1QwXftpFU1EREREJHsmaEmJVr58+dK8n9QeHBwMQLly5XB0dCQoKCjNJO3AgQMAVKlSJTPClXT4N0teRfPbshoSEiwYjYiIiIiI5WXLBK1IkSIA7Nu3L837e/fuBcDX1xeAPHny0KxZMwB++eWXe/ontbVr187cocp9nGzahgSbxH8E81+/TLHjBywckYiIiIiIZWXLBK1jx44A7Nixg2+//TbVvb/++ovJkycDqQ+lHjZsGADjx4/n9OnTpvbdu3czc+ZM8uXLR79+/TI7dEkhomBhLlWra7ouv0nbHEVEREQkd8uWCVqNGjV45513ABg0aBCVKlXi+eefp0GDBtSvX5+IiAheffVVWrRoYfpMixYtGDx4MLdu3aJatWp06tSJNm3a0KhRI2JjY5kzZw7u7u6W+kq5VspiIU/sWA/R0RaMRkRERETEsrJlggYwceJEli9fztNPP821a9dYsWIFx48fp3HjxixYsICZM2fe85kpU6bwww8/UL58eTZu3MiuXbto3rw527dv59lnn7XAt5Az9VsQ6+gEgFP4bVi3zsIRiYiIiIhYjsH4MAeAyX05OSUmGlFRURaLYfLGU2Yfc2jLJzJ1rmc+exu/rWsSL7p0gWXLzDKuiIiIiJiHNfycm1tk2xU0yTlSbnNkzRoICbFcMCIiIiIiFqQETSzu4pP1icz/7/N/MTGQRqVNEREREZHcQAmaWJzR1o6TTdsmN8yfb7lgREREREQsSAmaWIVU2xx37ICLFy0XjIiIiIiIhShBE6tw/YnKBHv5JjcsXGixWERERERELMXO0gGIAGAw4N+sA/V+mpZ4PW8ejBgBBkOmVKaE9KtTioiIiIhYilbQxGr4p9zmeOIEHDxouWBERERERCxACZpYjbCi3lCvXnKDioWIiIiISC6jBE2sS48eye9//hni4iwXi4iIiIhIFlOCJtbl+efB3j7x/bVrsGWLZeMREREREclCStDEuhQsCG3aJF9rm6OIiIiI5CJK0MT6pNzmuHw5dncjLReLiIiIiEgWUoIm1qddO8iXL/F9RASld2+2bDwiIiIiIllECZpYHycneO4502X5zb9aMBgRERERkayjBE2sU4ptjiX2/4lzyE0LBiMiIiIikjWUoIl1atQIvLwAsEmI54lt6ywckIiIiIhI5lOCJtbJxga6dzddlt+8yoLBiIiIiIhkDSVoYr1SbHMscuooBQLOWjAYEREREZHMpwRNrFelSlC1qunSb8tqCwYjIiIiIpL5lKCJdUuxilZ+869gNFowGBERERGRzGVn6QBE7uullzAOH47BaCT/9csUO36AKxWfzLTpJm88ZfYxh7Z8wuxjioiIiEjOpBU0sW7FihFQra7p0k9noomIiIhIDqYETayef/MOpvfltq/HJjbGgtGIiIiIiGQeJWhi9c7Ub0msoxMATnfC8N27w8IRiYiIiIhkDiVoYvViXPJytm5z03V5bXMUERERkRxKCZpkCym3OZb6ayuO4bctGI2IiIiISOZQgibZwsUn6xOZvwAAdrExlP3jdwtHJCIiIiJifkrQJFtIsLPnZJO2pmtVcxQRERGRnEgJmmQb/s2Stzl6H/kb1xtXLBiNiIiIiIj5KUGTbOOaXxVCivuarv22rLFcMCIiIiIimUAJmmQfBgMnmrc3XZbfvAqMRgsGJCIiIiJiXkrQJFtJuc2x4MUzeJ7zt2A0IiIiIiLmpQRNspWwYj5cqVDddK1iISIiIiKSkyhBk2znRLPkbY5+W1ZjiI+3YDQiIiIiIuajBE2ynVONnyHe1g6AvMFBeB/eY+GIRERERETMQwmaZDtR+d25UKuh6VrbHEVEREQkp1CCJtmSf/PkYiFld/6OXdRdC0YjIiIiImIeStAkWzr7VDOinV0AcLgbSendmy0ckYiIiIjcT2RkJCtXrqRfv35UqVKFfPny4eLiQtWqVRk7dizh4eEPPWZoaChDhgyhRIkSODo6UqJECQYPHkxoaKj5v0AWUYIm2VK8oxOnG7YyXWubo4iIiIh1W7hwIZ07d2bOnDkkJCTQunVrGjZsyPnz5xk9ejS1atXixo0bGR7v1q1b1K5dm6lTp2JnZ0enTp1wdXVl2rRp1KpVi1u3bmXit8k8StAk2zrRvKPpve++neQJyZ5/CUVERERyAwcHB1577TVOnTrF0aNHWbJkCb/99hsnT56kevXq+Pv7M2TIkAyPN3ToUE6fPk2XLl04efIkixcv5ujRo7z55pucOXOGYcOGZd6XyURK0CTbCqxSmzseRQCwSYin3PZ1Fo5IRERERNLTs2dPZsyYQdmyZVO1Fy1alG+++QaA5cuXExMT88Cxrl27xoIFC7C3t2fGjBnY2dmZ7k2cOBFPT08WLFjA9evXzfslsoASNMm+bGzwb9bOdKltjiIiIiLZU9WqVQGIjo7O0NbE9evXk5CQQKNGjShcuHCqe46OjrRv3574+HjWr1+fKfFmJiVokq2dSFHNsejJI7gFnrdgNCIiIiLyKM6dOweAvb097u7uD+x/+PBhAGrUqJHm/aT2pH7Zid2Du4hYr1slyxFUqhye504CUH7Lanb3fMvCUYmIiIjkPDExMVSsWDHNe8eOHXussadOnQpA69atcXR0fGD/gIAAALy8vNK8n9Se1C87UYIm2d6JZh3wPDcRSNzmuPvlN8FgsHBUDzZ54ymzjzm05RNmH1NEREQkM61bt47Zs2djb2/PuHHjMvSZpJL8zs7Oad53cXFJ1S87UYIm2d7Jpu1oOPtLDEYjblcv4XNwNwE16lk6LBEREZEcxcHB4bFXyv7rxIkT9OjRA6PRyMSJE03Poj2I0WgEwJDOL+WT7mdHegZNsr1wzyJcqlrHdN35/X60mvAe+a9kvyVtERERkdwiMDCQ1q1bExISwrBhwxg8eHCGP+vq6gpAREREmvcjIyMByJs37+MHmsWUoEmOcKBLb9N7m4QEKmxaSa9+z9Bi8ihcb1yxXGAiIiIico+bN2/SsmVLAgIC6NOnD19++eVDfd7HxwdITPLSktSe1C87UYImOcL5p5qy+qOvCSnua2qzjY+j8vql9On9NE2//hiXm9nvHAwRERGRnObOnTs888wz+Pv706VLF77//vt0tyqmJ2kr5IEDB9K8n9RepUqVxwvWApSgSY5xpsHT/DhrLb+/8xlhRZIr+tjGxVJt9UL69mpB428/xTnkpgWjFBEREcm9oqOj6dixI/v27aNVq1b8/PPP2NraPvQ4rVu3xsbGhj/++IMbN27cM8fq1auxsbHhmWeeMVfoWUYJmuQoRls7jj/dhblzfmPT4LHc9ixqumcXG0ONFT/St2cLGsyaCBk4BFFEREREzCM+Pp5u3bqxdetWGjZsyPLly3FwcLjvZ6ZPn46fnx8jR45M1V60aFG6detGTEwMgwYNIi4uznRv+PDhBAUF8dJLL1GkSJFM+S6ZSVUcJUdKsLPnn7YvcLxlZyqtX0Ltn78jb3AQAPbRd6m1ZBasWwRDh8KwYeDmZtmARURERHK46dOns2LFCgA8PDwYNGhQmv2+/PJLPDw8gMRn1U6ePMnVq1fv6TdlyhT++usvli1bhp+fHzVr1uTYsWMcPXqU0qVLM3ny5Mz7MplICZrkaPEODhzu2IOjrbtSdfXP1Fr8P5zDghNvhofDuHEwbRq8/TY8ROUgEREREXk4ISEhpvdJiVpaxowZY0rQ7sfDw4O9e/cyevRoVq5cyYoVKyhcuDBvvPEGH3/8Me7u7maJO6sZjNn5kAAr4+TkBEBUVJTFYsjKw4+zai5zzmN/N4Jqqxbw5NLZ5LkTmvqmuzt/dO7DoQ49iMuT9qGHj8LS//uJiIiIPC5r+Dk3t9AzaJKrxOZxYe+LrzLnp83w8ceQL1/yzeBgGs6eRN9eLai+bC620foXkIiIiIhkLSVokivFuOSFjz6CCxfggw8gxSGGLqG3aDLzM/r2bkmVXxdgGxNjuUBFREREJFdRgia5W4ECMH48nD8P775LrKOT6VbeWzdoPn0svfu2otK6JdjExVowUBERERHJDZSgiQB4eMCECcz5cRMHOvckzj655Gu+G1doOeVDevV7hgobVmCIj7vPQCIiIiIij04JmkgKke6ebH/tA36Yu5HD7boRb2dvuud29RKtvhxBz1faUW7rGkhIsGCkIiIiIpITKUETSUO4ZxG2vDWGuXN+45/WXUmwST7h3j3wPG0+e5ser3WkxN4doEKoIiIiImImStBE7uN2ES82DfuEubPXc7xFRxJskv/KeJ4/RZcPXuHZEX3wPHPcglGKiIiISE6hBE0kA8KKl+D34ROY9781nGzcJtU9n4O76f56F1pNGI7rjSsWilBEREREcgIlaCIPIdinNOs+mMyCb5YTUO0pU7vBaKTCplX07tOKBrMm4hh+24JRioiIiEh2pQRN5BHcKFuRZV/MZcX4/3HT9wlTu11sDLWWzKJPr5ZUXz4Xm1idoSYiIiIiGWdn6QBEsi2DgQu1G3PxyQZU2LCcej9OJW9wEAB57oTS5LvPqLZqPkyeCM8/DwaDhQMWazd54ymzjzm05RMP7iQiIiJWQytoIo/JaGvLsWee44cfNvBnr8HE5HE23XO7eglefBGeegr++MOCUYqIiIhIdqAETcRM4vI483f3QfwwdyOH2r+UqjQ/f/8NjRpBx47g72+5IEVERETEqilBEzGzyAIebH1zND99v4Yz9Vqkvvnrr1CpErz2Gs7/bocUEREREUmiBE0kk4R4l2L1mG8StzY+lVzxkfh4+O47+vR+mjrzp2N3N9JyQYqIiIiIVVGREJHM1qAB7NoFy5bBiBFw9iwADlGR1Pvpa6quWcSunm9xrFUXjLbm/yuZGYUnQMUnRERERDJDlqyg/f7773z55ZcsWbKEuLi4rJhSxLoYDNC1Kxw/DlOnQsGCplsuwUG0nPIhLw/sSMm/toLRaMFARURERMSSzJagzZgxg1KlSrFz585U7d26daNNmza89957dOvWjUaNGhEdHW2uaUWyFwcHeOstOHuWv194lTgHR9OtghfP0OmjgXQd3pPCp/6xYJAiIiIiYilmS9BWrFhBREQE9erVM7Vt3LiRxYsXU7x4cUaMGEHt2rXZs2cPs2fPNte0ItlT/vz82e9tfpjzO8dadsaY4ow078N/89IbXXnms7fJdy3QgkGKiIiISFYzW4J28uRJKlWqhI1N8pALFy7EYDDwyy+/8Mknn7Bt2zY8PDz46aefzDWtSLYWXqgoG979nAUzVnCxRv1U9/y2rqFXv9Y0/N8XOIbftlCEIiIiIpKVzJagBQUFUbRo0VRtO3bswMfHh9q1awPg6OhIvXr1OH/+vLmmFckRgkqXZ/nnc1j26WyCSpUztdvFxlLzlzm8/Go7SuzTQdciIiIiOZ3ZEjQ3NzdCQ0NN11evXuX8+fM0btw4VT8XFxfCw8PNNa1IjhJQswELvlnB7+98zh2PIqZ215vX6fJ+f5pP/Qj7SP39EREREcmpzJaglS1blp07dxIWFgbAggULMBgMtG7dOlW/wMBAihQpktYQIgIYbW05/nRnfvjhd3b1fIs4e3vTvSprF/PywI4UP7LXghGKiIiISGYxW4I2aNAgbt++zZNPPkmXLl344IMP8PT0pF27dqY+d+/eZd++fVSoUMFc04rkWPGOTuzp8ToLpy/jRunypvb81wJ57t2XafTdZ9hGR1kwQhERERExN7Odivviiy9y6NAhpk6dyrlz5/Dy8uLHH38kb968pj5LliwhMjKSZs2amWtakRzvVsly/DxtCXV+/o7aC7/DJiEeg9HIk8vnUvLv7fw2fALX/apYOkyRNOmgdBERkYdj1oOqP//8c0JDQ7l+/ToBAQE0bdo01f1mzZpx8OBBXnnlFXNOK5LjJdg7sLvnWyyatphbPqVN7e6B53lxyAvU+2EyNrExFoxQRERERMzBrAkaJFZq9PT0TPOet7c3VatWTbWqJiIZd/2JyiyYsYJ9Xfuazk6zSUigzs/f8dKbXeHwYQtHKCIiIiKPw+wJWla7du0aQ4cO5YknniBPnjy4u7vz5JNPMnz48DT7//TTT9SuXZu8efPi7u5OmzZt2LVrVxZHLfLo4h0c+ePV91gyaT6hRb1N7Z7nTkKtWvDppxAXZ8EIRURERORRmTVBu3HjBmPGjKFZs2aUL1+eUqVKpfkqXbr0gwfLgN27d1O+fHmmTJmCvb09HTp04KmnnuLWrVt89dVX9/QfNmwYvXr14ujRo7Ro0YLatWuzceNGGjVqxIoVK8wSk0hWuVKpJvO/W8Wh9i8lN8bGwgcfQP364O9vueBERERE5JGYrUjIiRMnaNy4Mbdu3cJoNJpr2HRduXKFNm3aEB0dzfLly+ncuXOq+3///Xeq6y1btjB58mQKFizI7t27KVu2LJCY5DVp0oQ+ffrQpEkTChQokOmxi5hLbB4Xtr45mrP1WvD0pPdxvXkt8cbff0P16omraRWeAZtsv1guZpYZxTtUuENEROTxme2ntnfffZebN2/SpUsX9u/fz507d0hISEj39bhGjBhBaGgoEyZMuCc5A6hdu3aq60mTJgEwatQoU3IGULduXQYOHEhYWBhz5sx57LhELCHgyfrM+99q6N07uTEqCoYN47l3e5Lv6iWLxSYiIiIiGWe2BO2PP/6gXLlyLFmyhOrVq+Pi4mKuoe8REhLCkiVLyJ8/P/37939g/6ioKDZv3gxA165d77mf1LZ69WrzBiqShaLz5oMffoBVq6BwYVO71z97eXlAByqvXQxZsLotIiIiIo/ObAma0WikcuXKGP6tLJeZ/vzzT6Kjo2nQoAH29vb88ssvDBkyhNdff52vv/6a69evp+rv7+9PdHQ0np6eeHl53TNejRo1ADhy5Eimxy6S6Tp0gKNH4bnnTE0OUZG0mPoRnT/oT96gaxYMTkRERETux2zPoNWsWZMzZ86Ya7j7OnbsGACFCxemYcOG7N69O9X9kSNH8sMPP/Dcvz+gBgQEAKSZnAG4uLjg5uZGSEgId+7cwdXV9b7zV6xYMc32mJgYHBwcHuq7iGQKDw9YsgQWL+buKwPJcycUAN99O3n51XZse30UJ5p3hCz4hYqIiIiIZJzZErQxY8bQvHlzlixZwvPPP2+uYdMUEhICJJbMd3R0ZPbs2XTo0IHw8HC+/vprvvrqK3r06EG5cuWoUqUK4eHhADg7O6c7pouLC6GhoYSHhz8wQRPJNl54gXnG4rSY8hGl9mwFwCniDq0nvEeZnRvZPPhjIgt4WDhIERHJqMwo8AMq8iPyXzExMezatYs//viDw4cPExQURFhYGPnz58fT05Nq1arRsGFD6tata/YFGrMlaACDBw+me/furFu3jpYtW+Ll5ZXulsdGjRo98jzx8fEAxMXF8c0339C3b18APDw8mDRpEgEBAfzyyy9MmDCB+fPnm6pK3m/75cNUnkxawfsvJyenDI8hklUiChZi1dhvqbBhOU2+/QTHyAgAyuzaRLFj+9n81secadjKwlGKiIiIWN6JEyeYOXMm8+fPJyQkJN0cYfny5RgMBtzc3OjZsyevvvoq5cuXN0sMZkvQmjRpgsFgwGg08tNPPzFv3rz79k9Ksh5F0gqXjY0NvXr1uud+3759+eWXX9i2bVuq/hEREemOGRkZCUDevHkfOS4Rq2UwcLzVs1yqXpenvxyJz6G/AHAOC6H9uLc40bQdW1//kOh8bpaNU0RERMQCAgMDGTVqFPPnzychIQEfHx/atm1L7dq18fPzw93dnXz58hEWFkZISAgnTpzg77//Zvv27UydOpWvv/6al19+mXHjxqX7WFVGmS1B69mzZ5YUCAHw9fUFoEiRIjg6OqZ7/8aNGwD4+PgAif/DpyUiIoLQ0FDc3Ny0vVFytDuFirHs8x+ouuZnGn4/EfvouwCU37oG78N72DjsEy7UbmzhKEVERESy1hNPJG7zfeWVV+jRowf169e/b//mzZub3u/cuZN58+Yxb948li5danq86lGZLUGbO3euuYZ6oOrVqwOYlh3/mxjeunULSF4NK1euHI6OjgQFBREYGHhPVnvgwAEAqlSpktmhi1iejQ2HO3Tn4pP1aTVxBMWOHwQgb3AQnUe9yqEO3dnxynDiHbVlV0RERHKHV199lREjRlCkSJGH/myDBg1o0KABY8aMYcKECY8di1mfQcsqlStXpmTJkpw/f549e/bw1FNPpbqftLUxqXx+njx5aNasGevXrzeV5E/pl19+AaBdu3aZHruItQgt7suSSQuosewH6v04BbvYWACq/bqA4v/sZd37kwkuUeahxsyMh9f14LqIiJiLiqxIeqZMmfLYYxQtWpTJkyc/9jhmOwctpbi4OPbv38/q1atZs2YN+/fvJy4uzqxzvPfeewC89dZb3Lx509S+f/9+Jk2aBMDAgQNN7cOGDQNg/PjxnD592tS+e/duZs6cSb58+ejXr59ZYxSxdkZbW/Y/35+F36zgRunkB1s9z5+i++tdqLxmkQ63FhEREclCZl1Bi46OZvTo0Xz33XfcuXMn1T1XV1cGDhzImDFjzFLt8JVXXmHz5s0sXbqUcuXKUa9ePcLDw9m1axcxMTG88sordO3a1dS/RYsWDB48mKlTp1KtWjVatmxJTEwMGzduJCEhgQULFuDu7v7YcYlkR7d8y7Jo6hIazP6SGit+BMAuJpoW00ZT4sCfbBwyTgVEREREJFeKjY3lyJEjXLlyBYBixYpRpUoV7O3tM2U+syVo0dHRNG/e3HRodJUqVUzFOi5evMjhw4eZOHEiO3fuZPPmzWkW93gYNjY2LFq0iCZNmjBr1iy2bNmCwWCgZs2aDBw4kJdffvmez0yZMoVq1aoxffp0Nm7ciL29Pc2bN2fUqFE0aNDgseIRye7iHRzY/tr7BNSox9NfjsA5LPG8wbI7N1D45D/8NuJLLleuaeEoRURERLJGREQEo0ePZvbs2dy+fTvVPVdXV/r168eYMWPMXmTQbAna5MmT2bVrFw0aNGDGjBlUqlQp1f2jR4/yxhtv8McffzBlyhTTFsXHYWNjw6BBgxg0aFCGP9O7d2969+792HOL5FTn6zRh/ne/0nrCcHwOJv7CJV/QVbq++zJ7XhrEnu6vYbTNlo+vioiIiGRIWFgYTZo04ciRIwBUrVoVX19fjEYjAQEBHDp0iClTprBp0yZ27NhB/vz5zTa32Z5B+/nnn/H09GTdunX3JGcAlSpVYs2aNXh4eLBgwQJzTSsimSCiYCGWfTaHP/q9Tfy/yZhNQgJ150/nuXd74nrjioUjFBEREck8H330EYcPH6Zp06YcO3aMAwcOsHz5clasWMH+/fs5duwYzZo14+jRo3z00UdmndtsCdqZM2do0qTJfQ96zps3L02aNOHs2bPmmlZEMouNDfteeJXFk38mtKi3qbn40f30GNgRli2zYHAiIiIimWfZsmUUKVKEX3/9FT8/v3vu+/n5sWrVKgoXLswyM/9MZLYEzc7OjsjIyAf2i4yMxM5O26NEsovrflVYMGMlJ5q1N7U5hd+Grl1h4EDIwN97ERERkezk1q1bNG7cGGdn53T7ODs706hRI4KDg806t9kStMqVK7NlyxbOnz+fbp/z58+zZcsWHQgtks3EuOTltxFf8tvwL4jJk+JfVDNnQq1a8O/+bBEREZGcoFSpUoSEhDywX1hYGCVLljTr3GZL0AYMGMDdu3dp0qQJP/74IzExMaZ70dHRzJ07lyZNmhAVFZXqfDIRyT5OtOjEghkruF62YnLj8eNQuzZ8843OTBMREZEc4dVXX2Xbtm0cOnQo3T6HDh1iy5Yt9O/f36xzmy1Be/nll3nllVe4dOkSffv2xdnZmeLFi+Pl5YWLiwv9+vXj0qVLvPrqq3Tv3t1c04pIFgst7suiKYvgnXeSG6Oj4Y036DBmEE63H/zbJhERERFrNnjwYF555RWaNm3KRx99xLFjxwgPDyc8PJxjx47x0Ucf0axZMwYMGMDQoUPNOrfZEjSAmTNnsnTpUho0aICdnR1Xr17lypUr2NnZ0bBhQ5YuXcq3335rzilFxAIS7B1g4kT47TcoXNjUXnr3FnoM7IjX4T0WjE5ERETk8dja2jJjxgzCwsL45JNPqFKlCvnz5yd//vxUqVKFTz75hNDQUL755htsbW1TvR633obZq3U8++yzPPvss8TFxXHr1i0AChYsqMIgIjlRq1Zw+DD07p2YrAGuN6/TdXgv/n5xAH+9/AYJdvaWjVFERETkIXl7e2MwGCwyd6ZlTXZ2dhRO8Zt1EcmhCheGtWthyhTi3xuBbVwsBqOROj9/h/fhPawf8SW3i3hZOkoRERGRDLtw4YLF5jbrFkcRyaVsbGDYMBZNXUxIcV9Tc7HjB+kxsCNPbFtnudhEREREspFHXkFr1qwZBoOBH3/8ES8vL5o1a5bhzxoMBjZv3vyoU4uIlbpRtiILZiyn6TfjqbhhOQCOkeG0/XQoJfbvZOugUcTlSf88EZHsYPLGU5ky7tCWT2TKuCIiYh537twBwNXVNVPneeQEbdu2bRgMBtPh1Nu2bcvwZy21n1NEMl9sHhc2vPMZF2vUo/m00ThGRgBQ6fdlFDt2gHXvf0VQmQoWjlJERETkwdasWcM333zDrl27CA8PB8DFxYX69eszaNAg2rdvb/Y5HzlBSzqQunjx4qmuRUQATjZrz7Xy1Xjms7cp6n8YAPfA87w4+Hl29nuHg517gX5ZIyIiIlbIaDTSv39/5s6di/Hfc17d3NwwGo2EhYXx+++/s2HDBl5++WV++OEHsy5APXKCVqJEiftei4iEFfVmyVcLqPvT19Ra/D8MRiN2sbE0+e4zvA/9xfqRk4jN42LpMEVERERSmTp1Kj/88APFihXjww8/5KWXXjJtbbxz5w4LFy5k3LhxzJs3j2rVqjFkyBCzza0iISKSqRLs7Pmz7zCWff4D4e6epvbSf22l6/Be5AkNtmB0IiIiIvf63//+h7OzM3/88QcDBgxI9dyZq6srAwYM4I8//iBPnjz873//M+vcOpxMRLLEpep1mf/drzw9aSSl9mwDoMjJf3hhaDeWfzqL20W9H2q8zCjUoCINIiIiAomPbz399NOULFky3T4lS5akefPmbNiwwaxzm20F7b8naKf3cnJyonjx4rRt25aFCxeaa3oRyQbuurnz65gZHOrQ3dRW4PIFXhzaDc+zJywYmYiIiEgyT09PHBwcHtjPwcEBDw8Ps85ttgTN29sbHx8fjEaj6eXm5kb+/PlTtRUpUoTg4GDWr1/Pyy+/TIcOHYiPjzdXGCJi5Yy2tmx9/UN29hlqanMJDuK5t7vjfXC3BSMTERERSdS5c2e2bNlCSEhIun2Cg4PZsmULnTp1MuvcZkvQzpw5Q7Vq1ShRogRz5szhzp073Lp1i+DgYO7cucOcOXMoWbIk1apVIywsjN27d1O1alXWrl3LjBkzzBWGiGQHBgN7uw3k97c/JcHGFgDHyAg6f/CKDrUWERHLMhopfmQvHuf8LR2JWND48eMpVaoUzZo1Y8uWLffc37JlCy1btqRUqVJ8+umnZp3bbM+gffLJJ2zZsoXjx4+bSu8ncXFxoXfv3rRo0YIKFSrw6aefMmbMGJYvX06FChVYsGABb775prlCEZFs4nirZ7nrVpC24wdjHx2FbVwsbT8diktIUGIZfhERkSxWaf1SWk75EIBFUxZxtUJ1C0ckWaFZs2b3tDk4OLB//35atmyJu7u7qWp9QEAAt27dAuCpp56iU6dObN682WyxmC1B++mnn2jWrNk9yVlKXl5eNG/enHnz5jFmzBh8fX2pWbMmhw8fNlcYIpLNnK/ThF8m/EinDweQ53YoAE2+/RSX4CB29n3bssGJiEiuU3bn78nv//hdCVousW3btnTvGY1Gbt26ZUrKUtq9e7dZz0ADMyZoV65coVq1ag/sZ2Njw5UrV0zXXl5e7N2711xhiEg2dK18NRZP/pnO7/cn//XLANRa/D0ut4Kg6RKwt7dwhCIiklsUuHTe9N494KwFI5GsdP78+Qd3yiJmS9C8vLzYvHkzN27coFChQmn2uX79Ops3b8bLy8vUduPGDdzd3c0VhohkUyHepVg85Wc6f/AKnudOAlBh00ro0AGWLoW8eS0boIiI5Hi2MdHku5G8kOB+6ZwFo5GslLR90RqYrUhI7969uX37No0aNWLp0qXExcWZ7sXFxbF06VKaNGnCnTt36N27t6n98OHDVK5c2VxhiEg2FlGwMEsmLeBS1drJjb/9Bs2aQVCQ5QITEZFcwe3yRQxGo+k63/XL2EZHWTCinGf//v18/vnndOnSheLFi2MwGHBycnqksXx9fTEYDOm+/P2zZ6EXs62gvffee+zdu5dff/2VF198ERsbGwoXLozBYODatWskJCRgNBpp37497733HgD+/v7UrFmTPn36mCsMEcnmYlxcWfHJLFp/8S5P/PHvcwB790L9+onJWqlSWRpPZhyIDToUWySnyYx/V+jfE1mvwOULqa4NRiPugecJKl3eMgHlQOPGjWPVqlVmHbNXr7QLi+XPnz/DYwQHB5tlV585xjFbgmZnZ8fKlSuZP38+3333Hfv27TM9a2Zvb0+dOnUYMGAAL7/8sukzlSpVYv369eYKQURyiHgHR9a9P5mI7z6l+qr5iY2nT0O9erB+PVTXA9siImJ+BQIv3NPmHnBWCZoZ1a1bl6pVq1KrVi1q1apFkSJFHnvMuXPnPvYYJUuW5I033mDYsGEULFjwoT8fFBTEpEmTmDFjBrdv336sWMyWoCXp0aMHPXr0IC4uzlTppGDBgtjZmX0qEcnBjLa2bBs0iup1KsD77yc2Xr8OjRvDihWAt0XjExGRnKdA4L2FIlQoxLySdtJZmy5duvDFF18wadIk2rRpQ/fu3WncuDEeHh7pfubatWts3bqVhQsXsmHDBuLj49NdzXsYmZY12dnZUbhw4cwaXkRyA4MBRo6EokWhf3+Ij4c7d+CZZ3jinS841bStpSMUEZEcRAla7vXDDz8wbNgwRo0axa+//mrahunj40O5cuUoUKAArq6u3L59m+DgYPz9/bl8ObHytI2NDe3bt2fcuHFUrFjxsWMxe4IWFxfHmjVr2Lt3Lzdv3qROnTr07dsXSCzFf/PmTSpUqKAVNRHJuN69oVAheO45iIyE2FjafjYs8UDrLr0tHZ2IiOQQaSdoquRo7SZOnMjZs2dxdHSkYsWKdO7cGU9Pz4cep3LlyqxatYrAwEBmz57NmjVrOHz4MBcvXrynr52dHTVr1qRt27b07ds3VZX6x2XWLGn79u28/PLLXL58GaPRiMFgIDY21pSgbd68md69e7NkyRKeffZZc04tIjldmzawZQu0bQv/bp9u8t1nyQda25itKK2IiORCTrdDyHM79J72ApcvYIiPw2irxYWYmJh0V4iOHTuWxdEkGz58eKrroUOHMm3aNPr16/dI43l5eTF69GhGjx5NREQEx48f58aNG4SFhZE/f34KFSpExYoVcXZ2Nkf49zDbTzT//PMPbdq04caNGwwePJilS5diTFGmFODZZ5/F2dmZZcuWmWtaEclN6tSBP/+EFGeV1Foyi1ZfjsAmLtaCgYmISHaXskBIjJMzRoMBANu4WPJfvWShqOR+OnTowPLly7l48SKRkZEcPXqUYcOGER0dTf/+/Vm5cuVjz+Hi4kKtWrVo27YtL730Em3btqVWrVqZlpyBGVfQxo4dS3R0NBs2bKBZs2Zp9nF2dqZ8+fIcPHjQXNOKSG5Trhzs2sWNhs0pdC7xfJMKm1bhHHqLNR9OIzaPi4UDFBGR7ChlghZcojROt0Nx+zcxcw84S6hXSQtFZj0cHBwsulL2X9OmTUt1XbFiRSZNmkS5cuUYMGAA7733Hp06dbJMcI/BbCto27dv56mnnko3OUvi4+NjKr8vIvJIihVj6aT5qQ609t23k67v9iRPyC0LBiYiItmVW4oz0EKK+xLsU9p0XVDPoWUr/fv3p1ChQpw6dYrz5+99rtDamS1Bu337NsWLF39gv+joaOLj4801rYjkUokHWs/mVKPWprYip47ywtBu2ooiIiIPzf1S8g/yIV6+BHuXMl0XuKQELTuxsbGhdOnEBPvq1asWjubhmS1BK1q0KCdOnHhgv6NHj1IixfMjIiKPKt7BgbXvT+Zgx5dNbQWuXOSFIS9S6LT1bMEQERHr53Y5ZYJWKtUKmkrtZz8hISEA5M2b18KRPDyzJWhPP/00x44dY8WKFen2mTt3LhcvXqRtW51dJCJmYmPDtkEfJFZy/JdLyE2ee6cHPgd2WTAwERHJNhISKHA5uZR6iFfqLY7ul87Cf4rfifU6duwYJ0+exNnZGT8/P0uH89DMlqC9//775M2bl27duvHhhx+yb98+AFNFlfHjxzNo0CAKFizIsGHDzDWtiAgYDOx98VV+f+czEmxsAXC4G0mnUa9SbusaCwcnIiLWzjXoKnYx0abr0OIlUiVojpERuNy6YYnQcr3p06fj5+fHyJEjU7X//vvv7N+//57+R44c4bnnnsNoNNK/f38cHByyKlSzMVuCVqJECdauXUuBAgX45JNPqFOnDgaDgaVLl1K1alU++ugjXF1dWbVqFUWKFDHXtCIiJsef7sKqsd8S65gHSCyN3Oazt3n6y5F4njlu4ehERMRapazgeMejMLF5XIjOm48I9+TDjgtqm6NZrF27lqeeesr0gsTz1VK2rV271tT/5s2bnDx58p5nyXbv3k3NmjXx9fWlefPmvPjii9SpU4cnn3ySEydO0LhxYz777LMs/W7mYtYT9xo0aMCpU6eYPXs2mzZt4sKFC8THx+Pl5UWLFi0YMGAAbm5u5pxSRCSVC7Ub88uEH+n04aumA0crblhOxQ3LuVzpSQ517MGZ+i1JsLO3bKAiImI1CgSmeP6suK/pfbB3SVyCg4DE59ACatTL6tBynKCgIPbs2ZOqzWg0pmoLCgp64DitWrXi0qVL7N27l8OHDxMWFka+fPlo0KAB3bt3p0+fPtja2pot7ujoaIKDg3F0dMTd3d1s46bFbAnatGnTcHZ2pn///gwZMoQhQ4aYa2gRkYdyrXxVFk1ZRKdRAyhwJfmZguJH91P86H7ueBTmSLtu/PPM89wtUNCCkYqIiDVIuYIW4p183lmwd2m8D/8NqFCIufTu3ZvevXtnuP+YMWMYM2bMPe1169albt265gssHd9++y0zZ87k6NGjGI1GevXqxZw5cwBYsmQJixYt4osvvqBs2bJmm9NsWxzffvttVq9eba7hREQeS6hXSeb9bw3rh0/garkqqe653rxO/blT6N+jMa0mvEfhU/9YKEoREbEGqVfQUiRo/y0UIrlGXFwc7du354033uDkyZNUqFAB438KxZQvX56VK1eyePFis85ttgStSJEiODk5mWs4EZHHFu/ggH+Ljiz6eik/T13CiWbtiU+xtdEuNpYKm1by0htdeWHIi/DzzxATY8GIRUTEElIlaN7pJGg6rDpXmTp1KmvXrqVdu3ZcvHiRI0eO3NOncuXKlCxZkvXr15t1brMlaK1atWLnzp3E6IcbEbFC18pX5bcRXzJr/lZ29Xwz1YPfAMWOH4SXXgJfXxg7Fq5ft0ygIiKSpWyjo8h344rpOuUzaLdSJGguITdxvBOWlaGJBf30008ULVqURYsWUahQoXT7lSpViosXL6Z7/1GYLUH75JNPsLW1pXv37tnyxG4RyR0i3T3Z0+MNZs3bwrqRk7hSoXrqDlevwujR4O0NL79MYf97f2MmIiI5h9uVAAz/bl2Lt7PndpHipnsRBQsR7exiutZzaLnH6dOnqVOnDnny5LlvPw8PD27evGnWuc1WJGTkyJFUrVqV5cuXs3btWmrUqIGPj0+a2x4NBgOzZ88219QiIg8twd6Bk03bcbJpOwqf+odqK+dTYce65C2OsbEwfz4vzZ/P1XJVONTpZU41ak2CffY7T0Wyr8kbT5l9zKEtnzD7mCLZWYHLF0zvQ4t6Y7RN8eOxwUCwd2mKnkz8ZZ37pXNcrVgjiyMUS3B0dCQ8PPyB/QICAsifP79Z5zZbgjZ37lzT+6ioKHbt2sWuXbvS7KsETUSsyfUnKvP78C+osGAmfP89zJgBV5K3uxQ9eYSiX7xLo/99wT9tX+BI2xeJKJj+dgcREck+ClxKfv4s1Mv3nvvBPikSNK2g5RqVK1dm79693Lp1i4IF0674HBAQwIEDB2jWrJlZ5zZbgrZ161ZzDSUiYhmFCsEHH8Dw4bBiBXz9NezcabrtEnKTp+Z/Q62fZ3K6UWsOdezB1fLVwGCwXMwiIvJYUq6ghXiVvOd+sE8p03v3SyoUklv069ePPn360KNHDxYuXEiBAgVS3Q8PD6d///7ExMTQv39/s85ttgStcePG5hpKRMSy7O3h+efh+eeZ/+1Kqq2ah9+WNdjFJm5/tI2Pw2/rGvy2ruF62Yoc7NSTU43bEO+g7Y8iItlNqgqOaSZoKSs5agUtt+jVqxdr167ll19+oWTJkjRo0ACAXbt20bVrV7Zu3UpISAgvvfQSnTt3NuvcZisSIiKSEwWVqcDGtz/j+4Xb2dn3be54FEl1v/DpY7Se+B79uzemwayJFDu2H0N8vIWiFRGRh/UwCVr+a4HYRkdlSVxieYsXL+bzzz/HwcGBdevWAXDq1CmWL19OQkIC48aNY968eWaf12wraCIiOVlUfnf2vvgq+57rS+ldm6m2aj7eR/423XcOC6bWklnUWjKLu65uXKzZgPO1G3OhZgOi8rtbMPLcQwU1Ho/+95PcyOl2CHluh5quQ9J4Bi2siBdx9vbYxcZiMBopEHiBm6X9si5IsRiDwcDw4cN5++23OXjwIBcuXCA+Ph4vLy9q1aqFQybtnFGCJiLyEIy2dpxp2IozDVvhcc6faqvm47dlNfYpfqOa506oaQuk0WDgql9VztduzPk6TQgqXV7PrImIWIkCgRdM76NcXIl0u7cYhNHWjtBivnhcPA0kbnNUgpa72NraUrNmTWrWrJkl8ylBExF5RDdL+bFp6Hj+6PcOFTavotRfWyn+zz5s42JNfQxGI8VOHKLYiUPU/3Eq4e6enK/dGO50g5YtwdXVgt9ARCR3S5mghXr5pvsLtGCfUqYEreAlPYeWG/Tt2xc7OzuGDx9OmTJl0u03e/Zs/vzzT+bMmWO2uZWgiYg8puh8bhzs3IuDnXthHxmOz4HdlPx7GyX/3k7e4KBUffMGB1H5t1/gt18Si5E0bAht20KbNmA0aHVNRCQLpXr+rLhvuv1UKCT3mTt3LgaDgeXLl7N48WKaN2+eZr+dO3fy008/KUETEbFWsc55OdugJWcbtASjEc+zJyj593ZK/r2doicOYTAaU3SOhS1bEl9vv02fot6JWyFrNyawah3iHRwt90UAEhKwSYgnwc7esnGIiGSSlCtoaRUISZI6QVOp/dyiVKlSBAYG8swzz/DVV1/xxhtvZMm8j5yg9e3blwYNGtC3b18g8aC2vHnz4u6uh+FFRAAwGAgqU4GgMhX4+6XXcAoLxnffTnz/3k75w7sgODhVd7erl6i+aj7VV80n1tGJS9Xqcr5OYsJ2p1Cxx48nNhZu3oSgILwPHiRPaDDOYcHkCQsmT1iI6U/nsGDyhAbjdCcUm4QEIgp4cMezCHcKFUv807ModwoVTfzTsyiRBTww2to+fnwiZqbCJ/IgD6rgmCRlguZ2+TyG+DiMtlrnyOkaNmzIgAED6NSpE4MHD+aff/7hm2++wc4uc/+/f+TR586dC2BK0EqWLEnv3r2ZPXu2WQITEclpovK749+8A/7NO1C+WWnYswfWroV16+DQoVR97aOjKLVnK6X2bAXgpu8TnK/diPO1m3C1QjUS7OyxjY4yJVN5wv5NtkKTEq3EZIsxdyEoKPEVGmoav+tDxO0SchOXkJsUOXU0zfvxtnaEexSGJ0qBt3fiy8cHvL3xvJzAbc+iRLvm1/ZNEbEuCQm4pTyk2vs+CZpXSYwGAwajEbvYWPJdu0xY8RJZEKRYWp06ddi7dy+dOnVi1qxZnDx5kmXLllGw4L0FZczlkRM0e3t7oqKSq5YZjUaMKbfuiIhI+mxtoV69xNcnn8Dly2z88gdK/r0dnwO7cIiKTNXd48IpPC6cotaSWcQ4OQPc08dSbOPjyH/9Mly/fM+9Hv/+GeuYJ/1VuH//jHPKk7WBi0iu5hp0FbvYGNN1aDGfdPvGOzpxu3Bx8l8LBKBgwFklaLmIl5cXO3fupFevXixdupTatWuzatUqKlWqlCnzPXKC5u3tzR9//MHFixcpUUL/gIqIPJbixTna5nmOtnke25gYih/dR8k92/Ddux33FM9IgBkSszx5uO1agLv5CxCZ352of/+8m9+du27uROYvYHpvtLElb9A1XIOu4hp0DdcbVxL/DEr80+lOWIamtI++i3vgedxTbCf6rwtPNmDDO58RUbDQ430/EZEMKHAp+d9HdzwKE5vH5b79b/mUNiVo7pfOcq5us0yNT6yLk5MTixcvplKlSowZM4b69evz448/Zspcj5ygvfTSS4wfP55SpUqZ2n788ccMBWowGIiLi3vUqUVEcrR4BwcCatQjoEY9tr/2PvkvXzQVGvE6sge72NhU/aOd8/6bWLlzN3+Bf1+JCVej+hXA0zP1y8WF2Q/xbE5YUe9079nfjcA16Bp5b1zlWc8EuHQp8RUQAJcuEXsxINUZcffju38n7ce8ztIv5xHv6JTh+EREHkWBlNsb7/P8WZJgn9KU+ns7oEIhudmHH35IpUqV6NmzJ127dqVw4cJmn+ORE7QxY8bg5ubGqlWrCAwM5Pz58zg7O+Ph4WHO+EREcr2w4iU41Lknhzr3xO5uJJ7n/IlzypO4+pWvAPEODul+tlEmFzSIzeNCsE/pxAfo05hr+oaTON0O+c/q21Vcb1w1rcrlvXkdm4R4AIqePMLTX33A+hFf6pm1HCozCneAinfIw0tdIMT3gf2DvZMXJVRqP3fr3Lkzf/75Jx06dCAgIACDmf979cgJmo2NDcOGDWPYsGGm6+eee86sZwCIiEhqcXmcuVqxhqXDyDiDgaj87kTldyeoTIW0u8TH0fSbcVRdswgAv61ruOVTmr+7D8rKSEUkl8loif0k95yFZjTqF0k52NatWylSpEi696tUqcK+ffv47rvviI+PN+vcZqsR2atXLxo0aGCu4UREJJcw2tqxbdAoCly+iM/B3QDU/3EqId6lON2otYWjE5GcKqMl9pOkXEFzjAzHJfgGEQXNv71NrEPjxo0f2MfDw4NRo0aZfW4bcw30ww8/mErui4iIPIwEO3vWjJpKSHFfU1urie9RKJ3S/iIij8M2Oop8N66YrjOSoEXncyPCLbm0up5Dk8xi9lPWYmNjWbFiBX/88QdXrlzBYDBQtGhRGjZsSOfOnbG3tzf3lCIikgNEu+Zn1dhveXHwCziF38Y+OoqOo19j4fRf9FtqETErtysBGP49Hirezp7bhYtl6HPBPqVxCb0FJG5zvFS9bqbFKFmrb9++GAwGPv30UwoXLvxQC08Gg8GsZ0GbNUH7888/eemllwgMDLznTLQZM2bg7e3NwoULqVevnjmnFRGRHCLEuxRrR02l8/v9sUmIJ++tG3QYPYilX87XOWkiYjYptzeGFvXGaJuxH4mDfUrhfeRvQIVCcpq5c+diMBh47733KFy4MHPnzs3wZ602QTt16hTPPPMM4eHhPPnkk/To0QNfX18ALl68yPz589m3bx/PPPMM+/bto2zZsuaaWkREcpCAGvXY+voomn/9MQBFTh3l6UkjWff+ZD2QLyJm8bAFQpIEe/+nUIjkGFu3bgXAx8cn1bUlmC1B++STTwgPD2fy5MkMHjz4nvtvvfUW06ZNY8iQIXzyyScPlZWKiEjucqT9S7gHnKX6qvkAlNu+nmCfMvz18hsWjkxEcoJUK2gZKLGfJFUlx0t6Bi0n+W9RkIwUCcksZisSsnnzZqpXr55mcpbkrbfeonr16mzatMlc04qISA61feBILtaob7quO+9rnti2zoIRiUhOkfKQ6uCHWUFLkaDlDQ7CIeKOOcMSAcyYoAUFBeHn5/fAfn5+fty8edNc04qISA5ltLVj7QeTU/3w1OrLERQ+ecSCUYlITlDgUsoVtIwnaOEehYl2djFda5tjzhUZGUlAQAARERGp2sPCwhg5ciTt2rXj9ddf5/z58+mM8OjMlqAVLFiQU6dOPbDfqVOncHd3N9e0IiKSg0W75mfVuO+Ics0PgF1MNB1GD4LLly0cmYhkV063Q8hzJ9R0/TAraBgMhKQ4D00JWs41fvx4SpYsyYkTJ0xtMTEx1K1blwkTJrBu3Tq+/fZb6taty/Xr1806t9kStKZNm3LgwAG+//77dPt8//337N+/n2bNmplrWhERyeFCi/uy+sNpxP9bZS1vcBB06ACRkRaOTESyo5QFQqJcXLnr9nALBykLhRRUgpZjbd68mZIlS1KzZk1T28KFC/H396dp06b8/vvvDBkyhBs3bjB58mSzzm22BG3UqFE4OTkxcOBAmjZtyvfff8/GjRvZtGkT33//PU2aNGHgwIE4OzvzwQcfmGtaERHJBQKrPcXW1z9MbjhwAHr1goQEywUlItnSPQVCHrI6bKpCITqsOscKCAjgiSeeSNW2cuVKbGxsmDt3Li1btuSrr76iXLlyrF271qxzm62KY/ny5fn111/p3r0727dvZ8eOHanuG41GChcuzIIFCyhfvry5phURkVzin3YvUjDgDNVXzkts+OUXGDMGxo61aFwikr08aon9JLd8UmxxvKQVtJwqJCSEAgUKpGrbtWsXlStXxsvLy9RWpUoVfv/9d7PObdaDqps3b865c+dYsmQJf/zxB1euXAGgWLFiNGzYkOeffx5nZ2dzTikiIrnI9gEjKBB4Ht99OxMbxo2D8uWhWzfLBiYi2UbKFbSQ4r4P/fmUz6DluxaIbUy0OcISK1OkSBFTLgNw7Ngxbt68yUsvvZSqnyETzuc0a4IG4OzsTO/evendu7e5hxYRkVwusbLjFF5//2VIenC7Tx8oVQoocN/PiojAf1bQvB9+BS20mA/xdvbYxsVik5Dwb8JX2XwBilWoXr06a9eu5dChQ1SrVo3JkydjMBho165dqn6nT5+mWLFiZp3bbM+giYiIZIUYF1dYvRqSKgJHR0OnTuS9cdWygYmI9UtIwC3FGWiPssXRaGtHaPESpms9h5YzjRgxgoSEBGrWrEnBggWZM2cOVatWTVXs8MaNGxw+fJgnn3zSrHMrQRMRkeyndGlYvhzs/t0Icu0aHUe/ht1dVXYUkfTlu3EFu9gY03VIsRL36Z2+lJUcVWo/Z6pTpw6rVq2iQYMGFClShB49evDrr79iY5OcPi1cuBBXV1dat25t1rmVoImISPbUuDF8+63pstDZE7SeMFyVHUUkXW4ptjfe8ShCXJ5Hq42QulCIVtByqrZt27Jt2zaOHTvGTz/9lKo4CMCQIUMICQmhe/fuZp1XCZqIiGRf/fvD0KGmy7J/bqTej1MtGJCIWDP3lAVCvHwfeZxUpfZVyVHMTAmaiIhkbxMnQps2pss6P3+H3+ZfLRiQiFirx33+LEnKBK3ApfMQH/84YYmkogRNRESyN1tb+PlnbpUoY2pq+dUHFD1+0IJBiYg1cr9knhW0EK+SGP8tr24XGwMXLjxmZCLJckSCFhwcTKFChTAYDPj5+d23708//UTt2rXJmzcv7u7utGnThl27dmVRpCIikiny5WPV2O+IzJ9Yat8uNob2Y17H9fplCwcmItbEXCtocU55uF0oRWn1pGM/RMwgRyRow4YN4+bNmxnq16tXL44ePUqLFi2oXbs2GzdupFGjRqxYsSILIhURkcwSVtSbNR9NJ97OHgCX0Ft0HP0a9ncjLByZiFiFu3fJdyP54OHHSdAg9TZHJWhiTlmaoO3bt48dO3awY8cOs425efNmfvzxR1555ZX79tuyZQuTJ0+mYMGCHD58mJUrV/Lbb7+xY8cObG1t6dOnDyEhIWaLS0REst7lyjXZ/NYY07XnuZO0/vxdVXYUEThzBoPRCEC8nT23Cz/e4cJK0CSzZGmC9vLLL9O0adNUB7w9jrt37zJw4EAqVKjAO++8c9++kyZNAmDUqFGULVvW1F63bl0GDhxIWFgYc+bMMUtcIiJiOcdad2Vf176m6zK7N1P/h8kWjEhErMKpU6a3ocV8MNraPdZwwd7JpfaVoOU8R44c4ejRoxaZO0sTtGLFiuHj44O3t7dZxvv44485e/Ys3377Lfb29un2i4qKYvPmzQB07dr1nvtJbatXrzZLXCIiYlk7+73DuTpNTde1F/+P8htXWi4gEbG8kydNb0OK+z72cME+/0nQ/l2dk5yhWrVqvPXWWxaZO0sTtM2bN3P+/HnOnz//4M4PcOTIESZNmkSfPn1o1KjRffv6+/sTHR2Np6fnPQfMAdSoUcM0poiIZH9GW1vWjfySm75PmNpaTBlFsWP7LRiViFhUihW0EO/He/4M/rPFMSwMrl177DHFeri7u1O0aFGLzJ0ti4QkJCTwyiuv4ObmxoQJEx7YPyAgACDN5AzAxcUFNzc3QkJCuHPnzgPHq1ixYpqvmJiYh/siIiKSaWKd87Jq7LdE5ncHwC42lvZj3iDftUALRyYiFmHmFbSofAVM/34BtM0xh3nqqaf4559/LDJ3tkzQvv76a/7++28mTpxIwYIFH9g/PDwcAGdn53T7uLi4pOorIiLZ3+0iXqwe/TVx/26Ddw4LpsPo17CP1L/rRXIdM6+ggQqF5GSjR4/m5MmTpjoWWenxno5MR3x8PDdv3iQ6OjrdPj4+Po809qVLlxg1ahSNGzemd+/eGfqM8d89wYZ/DxS8X5+MOHbsWJrtTk5OGR5DRESyxpVKNdk8eBytvhwBgOf5Uzz3zsvs6f46Z+s2A5ts+btKEXkYt25BcLDpMqS4uRK0Unj9szfxwt/fLGOKdThx4gQ9evRg+PDhzJs3j3bt2uHj45Puz/s9e/Y029xmTdA2btzIp59+yu7du4mNjU23n8FgIC4u7pHmGDRoEDExMXz77bcZ/oyrqysAERHpn4UTGRkJQN68eR8pLhERsV7Hn+6M+6Wz1Fr8PQCFzxynw8evE+xVkn3P9cO/eUfiHRwsHKWIZJoU2xujXFy56+Z+n84ZpxW0nKt3794YDAaMRiNHjhzhyJEjaS72GI1GDAaDdSZoy5Yt44UXXiAhIQEPDw98fX1N2wbNac2aNbi5ufHaa6+lao+KigISnzdr0qSJqW/evHlNq3WBgWk/dxAREUFoaChubm6mZE5ERHKWnX2G4Rxyi4oblpva3APP8/TkUdT7cSoHu/TiSNsXiXHRfwdEcpyU2xu9SsJ9dlU9jFveStByqo8++ui+u+8yk9kStDFjxgAwa9YsevfujU0mbhkJDQ1l+/btad67e/eu6V7SKl25cuVwdHQkKCiIwMDAe4qFHDhwAIAqVapkWswiImJhNjZseOczDrfrRq0lsyjz5wbTobV5g4NoOOtLai/8jiPtu3GwU08iChaycMAiYjYpVtBCvXzNNmyqFbQrVxKrOebPb7bxxXKSchtLMFsWdebMGRo3bkzfvn0zNTkzGo1pvpJK95crV87U5ubmBkCePHlMh2P/8ssv94yZ1NauXbtMi1tERKzDdb8qrPloGnNnr+dImxdMBUQAHCPDqbX4e/r2bEaLyaNS/dZdRLKxFH+Xg73M8/wZQLhnEWLypChCp+fQxAzMlkkVKVLEqp/fGjZsGADjx4/n9OnTpvbdu3czc+ZM8uXLR79+/SwVnoiIZLFQr5JsHjKW2fO28veLA4hKsbXRLjaWyuuXgp8fPPss7NljwUhF5LGlSNDMuYKGwUCw938OrBZ5TGbb4ti1a1d+/PFHbt++Tb58+cw1rNm0aNGCwYMHM3XqVKpVq0bLli2JiYlh48aNJCQksGDBAtzdzfPAqIiIZB+R7p782XcYe194lcrrFlNj+Vzy3rqReNNohOXLE1+NG8N774FtKbM9vyIiWSA+HlL8cj7EjCtokLjNscipo4kXStBylBs3bjBjxgx27NjB1atX061QbzAYOHv2rNnmNVuC9vHHH7N9+3Y6d+7MrFmzKFnSvP/wm8OUKVOoVq0a06dPZ+PGjdjb29O8eXNGjRpFgwYNLB2eiIhYUIxLXvY/149DHV+m3NbV1Fwyi4KXziV32L4dtm+nR8kn2Pdcf041aUOCnX36A4qIdbh0CVL8YB1SrIRZh9cKWs504sQJGjduzK1btx7qOC5zMFuC5uzszKZNm6hXrx7lypXD19eX4sWLp1n9xGAwsHnzZnNNDYCvr2+G/sfr3bt3hs9PExGR3CfewYHjrZ7leMvODI0+BV98Abt2me57nj/FMxOGU3/uFA4825t/Wj9HXMpnUETEuqQoEHLHo4jZ/74Gq5JjjvTuu+9y8+ZNnn32Wd5//32eeOKJTKlQnxazJWiXL1+mRYsWnDp1CqPRyJkzZzhz5kyafS1VslJERCTDbGygQ4fE186dMGECrF5tup3vxhWafPspdebP4HDH7hzq0MNsZyuJiBmlLLHvbf4dXsE+KVbQzp2DqChI5zBjyT7++OMPypUrx5IlS7I8dzFbgjZs2DBOnjxJixYteOuttyhVqlSWZZkiIiKZqkGDxNexYxwb+iF+W1ZjG594lEueO6E8Nf8bnlw6m2OtnmX/s324XdTbwgGLiEmKFbSQ4r5mHz6smA/Y2UFcHCQkJD7vVrmy2eeRrGU0GqlcubJFFpbMlqBt2rSJsmXLsn79emxtbc01rIiIiPWoWJEN737Orl6DqbHiRyqvW4zD3UgA7KOjqPbrAqqsWcSpxs+w77l+BJWpYOGARSSzV9AS7OyhbNnk7Y3+/krQcoCaNWumuxsws5mtzH5CQgJVq1ZVciYiIjleeKGi7Bgwglnzt7Gzz1Ai3Aqa7tkkxOO3dQ09BnWmy3t9qLBhOU63QywYrUgulzJBy4QVNADKl09+r+fQcoQxY8bwzz//sGTJkiyf22wraE899RT+OpxPRERykWjX/OztNpADXXpTceMKnvxlDm5XAkz3SxzcRYmDu0iwsSWwam1O12/J2fotiChY2IJRi+Qid+9CQPLfSXOX2DdRgpbt7dix4562wYMH0717d9atW0fLli3x8vJKd8tjo0aNzBaL2RK0Tz75hPr16/P111/z5ptvmmtYERERqxfv6MSRdt3455nnKfPnBmot/p7Cp4+Z7tskxONzcDc+B3fTfPpYrlSozpl6LTjT4OnE51dEJHOcOZN4niGAvT23CxfPnHmUoGV7TZo0STP5MhqN/PTTT8ybN+++n4+PjzdbLGZL0I4ePUqfPn0YMmQIK1asoEWLFumW2Qfo2bOnuaYWERGxCkZbW043eobTDVtT/J+9lNu2jjK7NuESHJSqX7HjByl2/CCNZk0kqFQ5ztR/Goq8ApUq6RBsEXNKUSCEMmUwZtajOCkTtJMnEw/H1mM/2UrPnj2tptK82RK03r17YzAYMBqNbNu2je3bt6fZz2g0YjAYlKCJiEjOZTBwuUptLlepzZY3PqLoiUOU+XMjZXduIP+1wFRdPc+dxPPcSZj3NZQpA126JL5q1bJQ8CI5SIrnzyhXLvPmSTl2VBRcvAilSqXfX6zO3LlzLR2CidkStI8++shqsk4RERGrYWPD1Yo1uFqxBn+8MhzPc/6U+XMjZXZuxOPCqdR9z5xJPG9twgQoXpymTzbhTP2WBFaphdHWbP/JFsk9Uq6gPfFE5s3j4gIlSiQmZpC4zVEJmjwis/3bfsyYMeYaSkREJGcyGAgqXZ6g0uXZ3fMt3ALPm5K1oiePpO57+TLVLi+g2q8LuJvPjbN1m3Gm/tME1KhHvIOjZeIXyW6yagUNwM8vdYLWtm3mzidmt2XLFgIDA6lZsyYVKtz/mJTjx4+zb98+vL29adq0qVnjMFuZfREREXk4oV4l2ffCqyz6emlipblp06BJE7BJ/Z/nPLdDqfT7cjp9NJCBzz1Fm0+G8sS2ddhHhlsmcJHsImWClpkraKBCIRm0f/9+Pv/8c7p06WKqV+Hk5PTI44WGhjJkyBBKlCiBo6MjJUqUYPDgwYSGhj7UOJcuXaJt27aMHz8eb2/vB/b39vbmk08+oV27dly5cuURo0+bEjQRERFr4O0Nb74JW7fCtWswaxbn6jQhzt4+VTeHu5GU276Otp8OZeBzden44UAq/L4Mx/DbFgpcxErdvAnBwcnXStCswrhx4xg5ciQrVqx47MTm1q1b1K5dm6lTp2JnZ0enTp1wdXVl2rRp1KpVi1u3bmV4rFmzZhETE8OECRNwdXV9YH9XV1cmTpzI3bt3mT179uN8jXuYbYtjs2bNMtzXYDCwefNmc00tIiKSs3h6Qr9+rPJpiENEOL5/b6fsnxvw/XsHDlGRpm52sTGU2rOVUnu2EvW/L9gw9BPONmhpwcBFrEjK1TM3t8S/V4Rm3nz/TdCMRlVlTUPdunWpWrUqtWrVolatWhQpUuSRxxo6dCinT5+mS5cuLF68GDu7xNTmrbfe4uuvv2bYsGH8+OOPGRpr48aNeHp60qlTpwzP36FDBwoXLsz69ev58MMPH+UrpMlsCdq2bdse2CepyqOKiYiIiGRMjEteTjVty6mmbbGNjqLEgT8ps3Mjpf/agtOdMFM/pzthdBj7Boc6dGfHq+/pOTWR/xYIyeyfP1MmaKGhcOMGFNah9P/13nvvmWWca9eusWDBAuzt7ZkxY4YpOQOYOHEiixYtYsGCBUyYMIHCGfj/wd/fn/r16z90HDVr1mTXrl0P/bn7MdsWx/Pnz6f5Onv2LNu2bWPkyJHkyZOHd999l3PnzplrWhERkVwj3tGJc3Wbs+Hdz5m5+E+Wff4Dh9t1I8bJ2dSn2q8LeHHwC7gFnrdgpCJWICsLhAB4eCS+kmibY6Zav349CQkJNGrU6J4EzNHRkfbt2xMfH8/69eszNF5ERAT58+d/6Djy589PeLh5nwc2W4JWokSJNF8lS5akUaNGfPLJJ6xcuZKvvvqKAwcOmGtaERGRXCnBzp6AGvXY8tYYFsxYzvUyyRXHCp09QfdBXfDbtMqCEYpYWFYWCEmi59CyzOHDhwGoUaNGmveT2pP6PUiBAgW4fv36Q8dx/fp1ChQo8NCfu58sLRLSokULnnzyST7//POsnFZERCRHC/UqyeIpiznY8WVTm0NUJM9MGM7TE0dgfzfCgtGJWEhWnYGWUg5P0GJiYqhYsWKar6wWEBAAgJeXV5r3k9qT+j1IhQoV+Ouvv7h7926GY4iMjGT37t0PLMn/sLK8iqO3tzfHjh3L6mlFRERytHgHB7a9PopfR39DlGvyNp2KG1fw0uvP4nHO34LRiWSx+PjEg9+TZMUWR8jxCZo1SdpW6OzsnOZ9FxeXVP0epH379kRERDB+/PgMxzB+/Hju3r1L+/btM/yZjMjSBO3u3bvs3bv3sc46EBERkfSdrd+C+d+u5EqF6qY298DzdHvzOaqsXphYWU4kpwsIgOjo5OsyZbJm3hyeoDk4OHDs2LE0X1nN+O+/y9IrPmh8yH/XDRgwgCJFivD5558zfvx4EhIS0u2bkJDAuHHj+PzzzylcuDADBgx4qLkexGxVHO+3fBgeHs6pU6eYNGkSly5dolu3buaaVkRERP7jTqFiLP1yHk/Nm07tRTMxGI3YxcbQ/OuP4cpRmDUrsey4SE6V8vkzb2/4dzUl06VM0C5fhtu3IV++rJk7l0k6qywiIu0t3JGRiUeS5M2bN0PjOTs7s3z5cpo3b87o0aP5/vvvee6556hRowaenp4ABAUFceDAAZYuXUpgYCBOTk4sW7Ys3VW8R2W2BM3X1/eB5fONRiPlypVj4sSJ5ppWRERE0pBgZ8+uPkMJrFKb1hOG4xJyM/HGsmWwfz8sWgR16lg2SJHMYonnzwC8vBKTwaSkwd8fatfOuvlzER8fHwACAwPTvJ/UntQvI5566il2795Njx49OHr0KJMnT76nT9LKXMWKFZk/fz5Vq1Z92NAfyGwJWqNGjdJN0BwcHChatCiNGzemW7du2uIoIiKSRQKerM/8b1fSesJ7lDjwZ2LjhQvQoAF8+im8/TbYZPkj6SKZK6tL7CexsUmcL6li+YkTStAySVJilF51+KT2KlWqPNS4VapU4ciRI/z++++sXbuWgwcPcuvWLYxGIx4eHlSrVo22bdvSunXrx/sC95GlB1WLiIhI1ot092T5p7Ootfh7Gvw0NbGAQlwcDB8OW7bAjz9aOkQR87JEif0k5cunTtAkU7Ru3RobGxv++OMPbty4QaFChUz3oqOjWb16NTY2NjzzzDOPNH6rVq1o1aqVucJ9KPqVmYiISG5gY8PebgNg+/bEZ3KS/PYbVKuG98HdlotNxNwstcURcnyhkKw2ffp0/Pz8GDlyZKr2okWL0q1bN2JiYhg0aBBxcXGme8OHDycoKIiXXnqJIkWKZHXIj00JmoiISG5Svz4cOgSdOiW3Xb3KsyP6UPfHqRji49L7pEj2cPduYhXHJFm5xRFSJ2j+Ot7iv9auXctTTz1lekHi+Wop29auXWvqf/PmTU6ePMnVq1fvGWvKlCmULl2aZcuW4efnx4svvkjlypWZNm0apUuXTvMZsuzgkbc4Tpgw4bEmHj58+GN9XkRERB6RuzssXw7Tp8M770BMDAajkacWzMDr8N+sHzmJcM/s91tnEQBOn05+7+AAJUpk7fwpE7SzZyEmJjEOARIrIe7ZsydVm9FoTNUWFBSUobE8PDzYu3cvo0ePZuXKlaxYsYLChQvzxhtv8PHHH+Pu7m7W2LPKIydoI0aMeGDVxiRpnVOgBE1ERMSCDAZ4883EFbUXXjAd6ut1dB89XuvI7+98zvmnmlo4SJFHkPL5szJlwNY2a+cvUwbs7BKf84yPT0wYK1bM2hisWO/evendu3eG+48ZM4YxY8ake79AgQJMmzaNadOmPX5wVuKRE7SPPvoowwkawPXr1/nxxx+5e/fuQ31OREREMlGNGnDgACc6daf8ltUA5LkdSqePBnKgcy/+6P8OCfb67b9kI5Z8/gzA3j4xSUva3njihBI0eSiPnKDdL5NN6datW3zxxRfMmzePu3fv4uLiwuuvv/6o04qIiIi5ubry23sTCahel2bTx2EffReAGit+pPjRfaz9YAphxTJ+lpCIRVmygmOS8uVTJ2giDyHTioQEBwczcuRISpYsyaRJkzAajbzzzjucP3+ezz//PLOmFRERkUdhMHC81bMs/GYZQSWTf6gtfPoY3Qd14omta+/zYRErYqkz0FJSJUd5DGZP0EJCQvjggw8oWbIkX3zxBfHx8QwdOpTz588zYcIEPDw8zD2liIiImEmwT2l+nraUw+1eNLU5RkbQ9rNhtJg8CruouxaMTuQBjEbLb3EEJWjyWMyWoIWGhvLhhx9SsmRJPv/8c2JjYxkyZAjnzp3jyy+/xNPT01xTiYiISCaKd3Riy1sfs2bUFKKd85raK69fyktvdqXo8YMWjE7kPm7dgpCQ5GtrWEE7eRISEiwTh2RLj52ghYWFMXr0aHx9ffn000+JiYnhzTff5Ny5c3z11VcULlzYHHGKiIhIFjvd6Bnmf7uSq+WqmNoKXjzDi0Ne5Pmh3SizcwOG+HgLRijyHylXz9zcwFI7t1ImhnfvwsWLlolDsqVHTtBu377Nxx9/jK+vL+PGjSM6OprXX3+ds2fPMmXKlGx5areIiIikdruoN0u+WsC+rn1TtRc/doD2Y9+kd7/WVF01H7u7kRaKUCSF/z5/ZqnK4Xnzgrd38rW2OcpDeOQEzdfXl7FjxxIVFcXrr7/OuXPnmDZtGkWLFjVnfCIiImJhCfYO/PHqe/zyxVwuV3oy1T23KwE0+2Ycr/RoQv05X+Fy67qFohTBOio4JtFzaPKIHjlBCw0NBSAuLo7Zs2dTunRpnJ2dM/RycXExV/wiIiKSRS5Vr8uSrxaycNpSTjZuQ4JN8gHATnfCqL1oJv1ebk6rCe/hcdbfgpFKrmUNBUKSpEzQ/PX3QTLukc9BAzAajcTHxxOv/eciIiK5xnW/Kqz7YDL5rgVSbeU8Kv22FMfICABs42KpsGklFTathGUtYNgwaN3aclvNJHexhhL7SbSCJo/okVfQEhISHuslIiIi2dvtIl7sGDiSWQu2s+OV4dzx+M/z55s2QZs2UKkSzJ4NUVGWCVRyh/h4OHMm+dqaVtBOnEg8AkAkAzLtoGoRERHJHWJcXNn/XD/m/LSJdSO+5HqZiqk7HD8O/ftDiRLUmT8dp7BgywQqOVtAAERHJ1+XLWu5WCB1ghYcDEFBlotFspXH2uIoIiIikiTBzp6Tzdpzsmk7iv+zl+e3L4HVq5M73LhBvZ++pvai/3G8ZScOdOlNiHcpywUs9zV546kHd3pIQ1tm4qpWyufPvL3B2Tnz5soIT08oWDDxbDZIXEUrVMiyMUm2oBU0ERERMS+DgctVasOvvyYWRxg4EJycTLftYqKpsnYxvfs9Q4ePBuJ1eI+2f8njs6bnz5LoOTR5BErQREREJPOUKwfffguXLsHYsUS4FUx1u/RfW3nu3Z689Pqz+G3+FZu4WAsFKtmeNZXYT6IETR6BEjQRERHJfB4e8OGHzJ6/lQ1Dx3OrRJlUtwufOcYzX7xL354teHLJLBzDb1soUMm2rKnEfhIlaPII9AyaiIiIZJl4B0eOPfMcx1p3pcS+P3jylx8ocXCX6b7rzWs0mjWRpxZ8w4nmHbjmV42g0n4Ee5cm3sHBgpGL1dMWR8khlKCJiIhI1jMYuFirERdrNcLjrD81ls/Fb+sabP/d4uhwN5KqaxZRdc0iAOJt7QjxLkVQqXIElfLj5r9/Rrp7WvJbiJk8bkESu6i7vBkQYLqec8Oevo8blDn4+SW/DwyEO3fA1dVy8Ui2oARNRERELOpmaT82vPs5f/YdSrVVC6iydhFOd8JS9bGNj8Pjwik8Lpyi/JbkypARbgW5WcoPfn8KqlZNfPn5gVbbchW3KxdN7+Ps7bldqJgFo0nBxyexmmRkZOK1vz/UqmXZmMTqKUETERERqxBRsDB/9h3Gnm4D8du6huJH9+Nx/iQFL54xraz9l0voLVwO/AkH/kxutLeHChVo5en772qbH0Ely3G3QME0x5Dsr0DgedP7sGIlMNraWjCaFGxsErdbHjyYeK0ETTJACZqIiIhYlbg8zhxt8zxH2zwPgE1cLAUuncPzrD+e5/zxOH8Kz3P+uITcTHuA2Fg4fJgKHAZWmZoj3D2Tt0iWTPwzxLskCXb2WfCtJDMVCLxgeh9S3NdicaSpfPnkBE3PoUkGKEETERERq5ZgZ8+tkuW4VbIc/nQ0tTuH3MTjrD+e50/S6O4VOHw48QfguLg0x3EJDsIlOAjffTtNbXH29twqUZYj7V/iaOuuYDBk+vcR80uVoHmXtFwgaVGhEHlIStBEREQkW4os4EFAzQYE1GxAo5b/llWPjk7cRnb4MPt/3Za44nbOH+ewkDTHsIuNpfCZ47ScPAqMRtOqnWQvKbc4WuUKWhIlaJIBStBEREQk53B0NBUL2VH0qcQ2oxGX4CA8zv27RfLcSTzPncT90jlsEuJNH23y7SdcqViD4P+c0SZWzmhMnaBZ8wramTMQE6MiNnJfStBEREQkZzMYiChYiIiChbhYq5Gp2TYmGs+zJ+j40UCcw0Kwj46izadD+fnrX4h3cLRgwPIw8oSF4JTiYPOQ4laWoJUpA7a2EB+f+DpzBipUsHRUYsVsLB2AiIiIiCXEOzhyrXw1NrzzuanN8/wpGn4/0YJRycMqcDl59SzKNT938xewYDRpcHBITNKSaJujPIASNBEREcnVztdpwoHOPU3X1VfNo9TuLRaMSB6G238rOFpjoRc9hyYPQQmaiIiI5Ho7+73LjVJ+puunJ43E5eZ1C0YkGeWe8vkzL1/LBXI/StDkIShBExERkVwv3sGBde9/RaxjHgDy3A6l9RfvYoiPf8AnxdJSraB5WdnzZ0mUoMlDUIImIiIiAoT4lGbroA9M1z6H91BzyfcWjEgyIlusoPklr87i7w8JCZaLRayeEjQRERGRfx1r3ZWTjZ8xXdf7cRpFThyyXEByX4b4ePJfuWi6ttoVtJQJ2t27cOmS5WIRq6cETURERCSJwcDmwWMJK1wcAJuEeNp8OgyHiDsWDkzSku/GFexiY03XocVKWDCa+3B1BS+v5Gttc5T7UIImIiIikkJ03nysH/ElCTa2AOS/fpkWUz4Co9HCkcl/uaXY3ni7UDHinPJYMJoH0HNokkFK0ERERET+42rFGux++Q3Tdbnt66iwYbkFI5K0FPhviX1rpgRNMkgJmoiIiEga9r44gEtVapuum00fBydPWjAi+a8C2aFASBIlaJJBStBERERE0mC0teW39yZy19UNAPvou9CtG0RHWzYwMSmQHUrsJ1GCJhmkBE1EREQkHeGeRdjw9ifJDQcPwsiRlgtIUsm2K2i3bkFQkOViEaumBE1ERETkPs7Va8GhDt2TGyZPhnXrLBeQAGAXdZd8QVdN11a/gubpCe7uyddaRZN0KEETEREReYAdrwznpu8TyQ29e8PVq+n2l8zndjn5/LM4e3vuFCpmwWgywGDQNkfJECVoIiIiIg8Q7+jEuve/AienxIagIOjZExISLBtYLlbgcvL2xrBiJTDa2lowmgxSgiYZoARNREREJANu+ZaFKVOSGzZtgi+/tFg8uV3KAiHB1r69MYmfX/J7JWiSDiVoIiIiIhn16qvQpUvy9QcfwN69losnF0tZICTU2guEJNEKmmSAEjQRERGRjDIY4Pvvwds78TouDrp1wyEi3LJx5ULZcgUtZYJ26RKE658buZcSNBEREZGH4e4OCxaAzb8/Rp09S7PpH1s2ptzGaMyeK2glSkCePMnXOvhc0qAETURERORhNWwIH35ouiy/+VfKb1ppuXhymTxhITiF3zZdZ5sVNBsbKFcu+VrbHCUNStBEREREHsWoUdCggemy2dcfkz9F6XfJPCkrOEa55icqXwELRvOQ9ByaPIASNBEREZFHYWeXuNXRzQ0Ah7uRtPlsGDaxMZaNKxcocCk5QQsp7pv4bGB2oQRNHkAJmoiIiMij8vGBWbNMl0VOHaX+3CmWiyeXKHD5gul9SHbZ3phECZo8gBI0ERERkcfx7LMcafuC6bLm0tn47NtpkVCcwoKpumo+z77bkw4fDSTvjasWiSOzpVpByy4FQpKkTNDOnIHYWMvFIlbJztIBiIiIiGR32weMpPjR/RS8eAaA1hPfY/53q4gs4JHpc9vExlByz3YqbFxByb+3YxsfZ7rndiWAxV8tJDqfW6bHkZWy9Qpa2bJgawvx8YnHNJw5kzppk1xPK2giIiIijynOKQ/rRk4izt4BAJeQmzw9cQQkJGTOhEYjhf2P0HT6WF59sSEdxr5Bmd2bUyVnAAUDztJx9GvYRkdlThwWYIiPJ/+V5GIs2W4FzcEBSpdOvtY2R/kPJWgiIiIiZnCzlB87BowwXZfc9wfVV/xk1jny3rhKrZ9n0rN/G1566zmq/bqAPHdCU/UJd/fkXO3Gpuvixw7wzBfvYoiPN2sslpLvxhXsUmwLDCnua7lgHpWeQ5P7UIImIiIiYiaH27/E2brNTNcNZ39JodPHHm/Q8HD46Sdo0YL+LzelwQ9fUfDSuVRdYh2dONG0Hcs/ncWsBdtZNW4mR1s9a7pfducGmnz7CRiNjxeLFXBLcUD17ULFiHd0smA0j8jPL/m9EjT5j2yZoEVGRrJy5Ur69etHlSpVyJcvHy4uLlStWpWxY8cSHh6e7md/+uknateuTd68eXF3d6dNmzbs2rUrC6MXERGRHMtgYMPbn3LHozAAtnGxtPl0GPZ3Ix5unIQE2LIFevWCIkUS/9y8GcN/EqzAyrXYMOwT/rfoT34bOYmLNRtitLUFg4HNgz9OtZJW7dcF1Fzy/WN/RUtzD/xPif3sKOUKmr+/5eIQq5QtE7SFCxfSuXNn5syZQ0JCAq1bt6Zhw4acP3+e0aNHU6tWLW7cuHHP54YNG0avXr04evQoLVq0oHbt2mzcuJFGjRqxYsUKC3wTERERyWmi8hXgt/cmYvz3bK4Cly/Q9JvxGfpsgUvnqPfDZPq93AyaN09cOYtIndyFFvNhV883mf3jJpZOms+x1l2Jccl7z1gJdvas/WAK18pVNrU1nD2J8ptWPvqXswJugRdM70O8s1mBkCT/TdAy61lFyZayZRVHBwcHXnvtNYYOHUrZsmVN7VevXqVt27YcPHiQIUOGsHDhQtO9LVu2MHnyZAoWLMju3btNn9u9ezdNmjShT58+NGnShAIFstFJ9CIiImKVAqvW4e8XB1Dn5+8AqLhhORefrM/Jpu3u6et4O5Ry29ZSYdMqivofTnvA/PnhhRdYXL4JVyrUyPDBzHF5nFk5diYvDn0RtysBALSc9AERBTwJeLL+o305C8sRK2gptzhGREBgYOKZeiJk0xW0nj17MmPGjFTJGUDRokX55ptvAFi+fDkxMTGme5MmTQJg1KhRqT5Xt25dBg4cSFhYGHPmzMmC6EVERCQ3+OvlN7hSobrpuvnU0eS/eglILI1fatcm2o19kwEvNqD59LH3Jme2ttC2LSxeDFevwsyZXKn4ZIaTsyR3CxRk+SeziMzvnjhsfBztx76B55njj/cFLSRHrKDlywfFiydf6zk0SSFbJmj3U7VqVQCio6O5desWAFFRUWzevBmArl273vOZpLbVq1dnUZQiIiKS0yXY2bN+xJdEOyduP3SMDKfNp0Np8s14XnmpER3HvE7ZnRuwjUt9UPGNUn5sGzAycVVlzRp4/nnIk+exYgkrXoKV42cS65g4jsPdSDqPepV81wIfa9ysZnc3knxByYdvZ9sVNFAlR0lXjkvQzp1LrGpkb2+Pu3vib4r8/f2Jjo7G09MTLy+vez5To0YNAI4cOZJ1gYqIiEiOd7uIF5uGjDNdFzn5D9VXzcM5LCRVv4gCHux/tg/zvl3Fgu9WcfDZ3onFQczoerkqrPlwKgk2tgC4BAfR+f3+ON0OecAnrUfSNk2AOHsH7hQqZsFoHpMSNElHjkvQpk6dCkDr1q1xdHQEICAg8S9zWskZgIuLC25uboSEhHDnzp2sCVRERERyhVNN2vBP63t38MTZO3CycRtWjP8f3y/czo4BI7hZ2i+NEcznQu3GbBoy1nTtHniejh+9BnfvZuq85lLgcvLzZ6HFSiRWrMyulKBJOrJlkZD0rFu3jtmzZ2Nvb8+4ccm/rUoqu+/s7JzuZ11cXAgNDSU8PBxXV9f7zlOxYsU022NiYnBwcHiEyEVERCQn2/baB+S/FojPob+4XOlJjrfoxOlGrYnOmy/LYznWuit5b16n3k/TACh2/CB06wbLliU+92bFClxKUSDEy9dygZiDEjRJR45J0E6cOEGPHj0wGo1MnDjR9CwagPHfM0MM93mo1pgDDm4UERER6xSXx5llX8xNPCjaxvIbmPZ0H0Tem9epsm5xYsOqVfDmm/DNNw9dhCQrFbh8wfQ+xCubFghJkjJBu3kz8eXhYbl4xGrkiAQtMDCQ1q1bExISwrBhwxg8eHCq+0krYhER6R8SGRkZCUDevPeeI/Jfx44dS7PdySkbnmQvIiIiWcNgsJ7kx2Bgy5sf4RJ8g9J/bU1s+/Zb8PKC99+3bGz3kaNW0AoVggIFIOTfZwBPnICGDS0bk1gFy/8K5zHdvHmTli1bEhAQQJ8+ffjyyy/v6ePz77kSgYFpVyqKiIggNDQUNze3B25vFBEREckJjLZ2rHt/Mlf9kncd8cEHMHeuxWK6L6MxZ62gGQwwbBh88QX8+itUqmTpiMRKZOsVtDt37vDMM8/g7+9Ply5d+P7779PcxliuXDkcHR0JCgoiMDDwnmIhBw4cAKBKlSpZEreIiIiINYhzysPKcd/x2vs94fTpxMb+/RMrSNqWsmxw/5EnLASn8Num62y/ggYwapSlIxArlG1X0KKjo+nYsSP79u2jVatW/Pzzz9im82Brnjx5aNasGQC//PLLPfeT2tq1a5d5AYuIiIhYoaj87vDbb1C4cGJDfDx07UqhU0ctG9h/FAg8Z3of5ZqfqHwFLBiNSObJlglafHw83bp1Y+vWrTRs2JDly5c/sHrisGHDABg/fjynk35DBOzevZuZM2eSL18++vXrl6lxi4iIiFilUqVg3TpIehY/IoJOo14l/9VLlo0rhQKBF0zvg71KWs/zfCJmli0TtOnTp7NixQoAPDw8GDRoEL17977ndfPmTdNnWrRoweDBg7l16xbVqlWjU6dOtGnThkaNGhEbG8ucOXNMB1uLiIiI5Do1aiSW2rdLfALGJfQWnd/vT57QYAsHlqhAYIoz0Ir7Wi4QeWxRUVGMHj2aJ554AicnJ4oVK0bfvn3TrReRHl9fXwwGQ7ovf3//TPoGmStbPoMWEpJ84n1SopaWMWPG4JGiXOmUKVOoVq0a06dPZ+PGjdjb29O8eXNGjRpFgwYNMjVmEREREav39NMwezb06gUklrXv+OEAfpnwI3F50j9PNiukWkHzzuYFQnKxqKgomjdvzq5duyhatCgdO3bkwoUL/PDDD6xZs4bdu3dTunTphxqz17//vP5X/vz5zRFylsuWCdqYMWMYM2bMI302aXVNRERERNLQsydcvmwqt1/05BHafjqMX8dMx2hruR8dtYKWM3z66afs2rWLunXrsmHDBtMRV1999RVvv/02ffv2Zfv27Q815lxrrTz6iLLlFkcRERERyUQjRnCo/Uumy1J7ttJ82seJB21bgCE+HrcrAaZrraBlT7GxsXz99dcAfPPNN6nOHx42bBhVqlRhx44d7N+/31IhWgUlaCIiIiKSmsHAtkGjOFOvhamp8vol1FnwTdaGER9HkROHeWre19jGxZraQ4uVyNI4xDx27txJaGgopUuXpnr16vfc79q1KwCrV6/O6tCsSrbc4igiIiIimctoa8u6kZPo+l5vih0/CEC9n74mvGBhjj3zXCZNaqTghVP4HPoL74O78TqyF8fI8FRdbhcqRryjU+bML5nq8OHDANSoUSPN+0ntSf0yauLEiZw9exZHR0cqVqxI586d8fT0fLxgLUgJmoiIiIikKd7RiVVjv+X5oS9R8FLiOWQtpo4m0t2T83WamGeSc+eotG4J3of+wvvQX7iE3rpv92NPdzHPvPLQYmJiqFixYpr3jh079sDPBwQkblP18vJK835Se1K/jBo+fHiq66FDhzJt2rRse4SWEjQRERERSVdUvgKs+HQWLw5+gbzBQdgkxNN2/BCWTvyJ635VHn7Aa9dgyxbYvDnxzwsXaHmf7vF29lz1q8ql6nW5ULMB18pXe9SvIhYWHp64GursnHZFUBcXl1T9HqRDhw40bdqUJ598Ek9PT86dO8ecOXOYOnUq/fv3p2DBgnTq1MkssWclJWgiIiIicl93Chdn5Sff89zb3XGMjMA++i6dPnyVRVMWE1b8/s+DOYbfxuvw33gf+guG7Ifjx+/b32gwcKNMBS5Ve4qA6nW5XPFJi5f4l0QODg4ZWilLj/HfIjOGdA4ZNz5kEZpp06aluq5YsSKTJk2iXLlyDBgwgPfee08JmoiIiIjkTEGly7N69Dd0/uAVbONicQ4Locv7/Vg8ZRGRBZLPnbWLukuxYwfwObQb70N/Uej0MWwSEu479i3vUlyqXpeAanUJrFKL6HxumfxtxBJcXV0BiIiISPN+ZGQkQKrqjo+if//+fPjhh5w6dYrz589TsmT2qvqpBE1EREREMuRS9bpsePtTnvniXQDcrl6i44cD2PHqe3gd2Yv3ob8oeuIgdrGx9x/I2xuaN098NWvGT8cytqVNsjcfHx8AAgMD07yf1J7U71HZ2NhQunRpbty4wdWrV5WgiYiIiEjO5d+8Ay63btBo1kQAipw6yvPvvHzfz0TmL8Clak9RrlvHxKSsdGlIuc3t2KnMDFmsRNWqVQE4cOBAmveT2qtUeYRnG/8jJCQEePzVOEtQgiYiIiIiD2X/c/1wvXmN6ivnpXk/Jo8zgVVqE1CtLpeqP8VN3yfAxoZyLZ/I4kjFmtSvX5/8+fNz9uxZDh48eM9ZaL/88gsA7dq1e6x5jh07xsmTJ3F2dsbPz++xxrIEHVQtIiIiIg/HYGD7gJEcb9EJgDh7BwKq1uHP3kNYNGUR3y77m1XjZnLw2d7cLOUHNvqRUxKLjLzxxhsAvPHGG6meRfvqq684cuQIDRo0oFatWqb26dOn4+fnx8iRI1ON9fvvv7N///575jhy5AjPPfccRqOR/v374+DgkEnfJvNoBU1EREREHprR1pbfh3/Bn32Gcjefmw6PlgwZNWoUmzZtYteuXZQtW5aGDRty8eJF9uzZQ8GCBfnhhx9S9b958yYnT57k6tWrqdp3797Nxx9/TIkSJShdujSenp6cP3+eAwcOEBcXR+PGjfnss8+y8quZjX6dISIiIiKPLNyziJIzyTAnJye2bt3Khx9+iLOzMytXruTChQv06tWLgwcPUqZMmQyN06pVK/r27Uu+fPk4fPgwy5Yt48yZMzRo0IDvv/+ezZs3p3vemrXTCpqIiIiIiGSZPHnyMHbsWMaOHfvAvmPGjGHMmDH3tNetW5e6detmQnSWpxU0ERERERERK6EETURERERExEooQRMRERER+X97dx4e073/Afw92SarIIkGkdQWKrVFELW2duVGUG1VO6WoKG0tpbettvRSj8bWotR1NVou6lFLWz9Va0iilVijBLWvQVT2RPL5/eGZuZKZLDNzZhzJ+/U884fznXzf54w533M+M+d8h0glWKARERERERGpBAs0IiIiIiIilWCBRkREREREpBKcZp+IiIiISGHztqco3uf47sGK90nqw2/QiIiIiIiIVIIFGhERERERkUqwQCMiIiIiIlIJFmhEREREREQqwQKNiIiIiIhIJVigERERERERqQQLNCIiIiIiIpVggUZERERERKQSLNCIiIiIiIhUggUaERERERGRSrBAIyIiIiIiUgkWaERERERERCrBAo2IiIiIiEglWKARERERERGpBAs0IiIiIiIilWCBRkREREREpBIs0IiIiIiIiFSCBRoREREREZFKsEAjIiIiIiJSCRZoREREREREKsECjYiIiIiISCVYoBEREREREakECzQiIiIiIiKVYIFGRERERESkEizQiIiIiIiIVIIFGhERERERkUqwQCMiIiIiIlIJFmhEREREREQqwQKNiIiIiIhIJVigERERERERqQQLNCIiIiIiIpVggUZERERERKQSLNCIiIiIiIhUggUaERERERGRSrBAIyIiIiIiUgkWaERERERERCrBAo2IiIiIiEglWKARERERERGpBAs0IiIiIiIilWCBRkREREREpBIs0IiIiIiIiFSCBRoREREREZFKsEAjIiIiIiJSCRZoREREREREKsECjYiIiIiISCVYoBEREREREakECzQiIiIiIiKVYIFGRERERESkEizQiIiIiIiIVIIFGhERERERkUqwQCMiIiIiIlIJFmhEREREREQqwQKNiIiIiIhIJVigERERERERqQQLNCIiIiIiIpVggUZERERERKQSLNCIiIiIiIhUggUaERERERGRSrBAIyIiIiIiUgkWaERERERERCpR6Qq0nJwcfPLJJwgODoarqytq1aqF4cOH4/Lly4961YiIiIiIKjwlz8fv3r2Ld999F0FBQdBqtQgKCsI777yDu3fvKr/idlKpCrScnBx07doV06dPR0ZGBiIiIlCnTh2sWLECoaGhOHv27KNeRSIiIiKiCkvJ8/Hbt2+jTZs2WLBgAZycnNC/f394eXnhyy+/ROvWrXH79m0bbontVKoCbebMmYiLi0O7du2QkpKCtWvX4sCBA5gzZw5SU1MxfPjwR72KREREREQVlpLn4+PHj8fp06cxYMAAnDp1CmvXrsXx48cxbtw4nDlzBhMmTLDhlthOpSnQ8vPz8dVXXwEAFi1aBE9PT0PbhAkT0KxZM+zduxeJiYmPahWJiIiIiCosJc/Hr1+/jlWrVsHZ2RmLFy+Gk5OToe2LL76An58fVq1ahRs3bii/ITZWaQq0ffv24e7du6hfvz5atmxp1D5o0CAAwJYtW+y9akREREREFZ6S5+Nbt25FYWEhOnXqhCeeeKJIm1arRb9+/VBQUICtW7cqs/J2VGkKtCNHjgAAQkNDTbbrl+ufR0REREREylHyfLwin9s7lf2UiuHixYsAgICAAJPt+uX655UmJCTE5PLc3FwAgKurqyWrqIj7haJ4n/900DzSLFvk2DOLr5/yOfbM4utnXVZFfP0q4jbZM4uvn3VZfP2sy6qor5896M9zSzoPTk5OLrMPJc/HlexLbSpNgZaRkQEAcHd3N9nu4eFR5HmPK6dy7rh5eXkAABcXF5tm2SunomZVxG2yZ1ZF3CZ7ZnGbLM+pqFkVcZvsmVURt8meWRVxm5TKelwoeT5ekc/tK02BJvLgUwyNxvTOom8vj/J8QqB2+k8/bL0t9sqpqFkVcZvsmVURt8meWdwmZj2qnIqaVRG3yZ5ZFXGb7J31qCl5Pq5kX2pTae5B8/LyAgBkZmaabM/KygKAIrPJEBERERGRMpQ8H6/I5/aVpkALDAwEgBJ/oVy/XP88IiIiIiJSjpLn4xX53L7SFGjNmzcHACQlJZls1y9v1qyZ3daJiIiIiKiyUPJ8vCKf21eaAq19+/bw9vbG2bNncejQIaP29evXAwD69u1r71UjIiIiIqrwlDwf79WrFxwcHBAbG4ubN28WacvNzcWWLVvg4OCA3r17K7PydlRpCjQXFxeMHTsWADB27Ngi16vOnTsXR48eRYcOHdC6detHtYpERERERBWWJefjCxcuROPGjfHPf/6zSF81a9bEyy+/jLy8PIwZMwb37983tE2ePBmpqakYMmQI/P39bbxVytPI4zzFiZlycnLQpUsXHDhwADVr1kTHjh1x4cIFHDhwAD4+PkhISECDBg0e9WoSEREREVVI5p6Pf/rpp5g2bRp0Oh2+/fbbIn3dunUL4eHhOHv2LOrXr4+wsDAkJyfj+PHjqF+/PhISEuDr62vnLbRepfkGDXjwA9K7du3C1KlT4e7ujo0bN+L8+fPQ6XQ4dOgQizMiIiIiIhtS8nzc19cXf/zxB8aNG4e8vDz8+OOP+PvvvzF27Fj8/vvvj2VxBlSyb9CIiIiIiIjUrFJ9g0ZERERERKRmLNCIiIiIiIhUggUaERERERGRSrBAIyIiIiIiUgkWaERERERERCrBAo2IiIiIiEglWKBVEomJiZg1axYGDBiA2rVrQ6PRwNXVVfGcrKwsbNy4EW+88QaaNWuGKlWqwMPDA82bN8f06dORkZGhaN7cuXMxYMAANGzYEN7e3tBqtQgKCoJOp0NycrKiWQ+7c+cOatSoAY1Gg8aNGyvad5cuXaDRaEp8/N///Z+ieQBw/fp1jB8/HsHBwXBzc0P16tXRqlUrTJ48WZH+d+/eXeo26R/Tp09XJC8hIQEDBw6Ev78/nJ2dUb16dXTt2hXr169XpP/iWREREfD19YWrqyuCg4Px0UcfISsry+y+LN1PV65ciTZt2sDT0xPVq1dHnz59EBcXp1jOpUuXsHjxYuh0Ojz11FNwcHCARqNBQkKCottUWFiI2NhYTJ48GW3btkWNGjWg1WpRv359jB49GufOnVMsCwA2b94MnU6Hpk2bwtfXF87OzqhRowb69OmDn3/+WbEcU7p162Z431+/fl2xrE8//bTUfez9999XdJtyc3MRHR2NsLAwVKlSBZ6enmjUqBHeeOMNXLlyRZGs8owdzz33nGLbdeXKFURFRaFevXrQarXw9PREaGgovvjiC+Tm5iqWc/nyZYwePRqBgYHQarWoVasWXn/9dZw/f77Ev7HmGGvuOGFJlqVjhblZ1o4VxcXExECj0aBFixa4f/++yefs378fDg4O8Pf3R1pamln96w0YMAAajQZTpkwp9XmDBw+GRqPBxIkTLcqhCkCoUoiIiBAARR5arVbxnGXLlhn6DwkJkRdeeEF69uwpXl5eAkAaN24sN27cUCzPx8dHXF1dpU2bNhIZGSmRkZESHBwsAMTFxUV++eUXxbIeptPpRKPRCABp1KiRon137txZAMjAgQNFp9MZPY4ePapoXlxcnFStWlUASJMmTWTw4MHSu3dvCQoKEkdHR0Uy/vzzT5PbotPpZOjQoYb3zM6dO63OWrdunTg4OAgACQsLkxdffFE6duxoWDZlyhQFtuiB77//XhwdHQWAtGrVSiIjI6VOnToCQJo3by737t0zqz9L9tPx48cLAHFzc5OIiAjp2bOnODk5iaOjo2zYsEGRnHnz5hk9H4DEx8cruk2nT582PKd27doSEREhkZGRUrt2bQEgXl5eEhsbq0iWiMjAgQNFo9HI008/LX369JEXX3xR2rZta/jbqVOnKpJT3IoVKwSAYQy5du2aYtv0ySefCABp3769yf1t3bp1im3TjRs3pGnTpgJA/P39DWOwfllJ/1fmZpU0duh0OvH19RUA8vHHHyuSderUKUOf9erVk4EDB0qvXr3E09NTAEiHDh0kLy/P6pxjx46Jn5+fAJC6devKgAEDpFmzZgJAvL29SxznLT3GWjJOWJJl6Vhhbpa1Y4Up3bp1EwAya9Yso7bc3Fx56qmnBICsWbPGrH4fdvXqVfH29hZHR0dJTEw0+ZzNmzcb3heZmZkWZ9HjjQVaJTFr1iz5+OOPZcuWLXL9+nWbFWgxMTESFRUlKSkpRZZfvXpVWrZsKQDk5ZdfVixv3759kp2dbbR88eLFAkBq1aol9+/fVyxPROS3334TADJq1CibFmjnzp1TtF9Trly5IlWrVhU3NzeTB+kDBw7YfB1++eUXASB16tSRgoICq/rKz883nPQUP4jGxcWJq6uraDQaOXPmjFU5IiKXLl0SV1dXASD/+c9/DMtzcnLkhRdeEAAyevRos/o0dz/dsWOHABAfH58i+1xcXJy4uLiIt7e33Llzx+qcTZs2yfjx42X16tVy+vRpw3u0PAWaOVlnzpyRnj17yp49e4osz8nJkddff10ASGBgoMkTZEu2KykpSW7dumW0PCEhQTw9PUWj0UhycrLVOQ+7efOm+Pj4SI8ePSQoKKjMAs3cLH2BtmLFinKtj6U5BQUFEh4eLgDkww8/lPz8/CLtZ8+eldTUVEWySpKWliZarVYAGB1zLM2KjIwUADJ27Ngix44bN25Iw4YNjfZ3S3IKCwsNxdjw4cOLvHZz5841FCmmxkNLjrGWjhOWZFk6VpibZe1YYcqZM2fEzc1N3NzcjI4R+v3q+eefL3d/JVm6dKkAkBYtWhjtN/fu3ZOAgAABIL/++qvVWfT4YoFWSdmqQCtNXFycITc3N9fmeQ0aNBAAJk+wLJWVlSUNGjSQJk2aSEpKymNfoL366qsCQL766iubZ5VkyJAhAkDef/99q/s6duyY4dNWU/Sfcq9du9bqrM8++0wASPfu3Y3abt68Ke7u7uLs7GyyACivsvbTPn36CACZN2+eUdvbb78tACQ6OtrqnOLMKdCszdLLzs4Wb29vASC7d++2aZaIyIgRIwSALFq0SNGcIUOGiKurq5w5c6ZcBZq5WZYWaObmLF++XIAH3/Rby9L/p2+++UYASHh4uGJZPj4+AkCuX79u1Kb/digqKsqqnNjYWAEg1apVM/kte+vWrQWAbNq0qcych5V0jFVqnChPVnHWjBXmZulZMlbozZo1SwBIt27dDMuSk5PFxcVFPD095cKFC2avf3GFhYXSpUsXASCff/55kbYxY8YIANHpdFbn0OON96CR3TRv3hzAg3sWbt++bfM8R0dHAICLi4tifU6bNg1nz57F119/DWdnZ8X6fRTS0tKwbt06eHt7Y8SIEY9kHTIzM7Fp0yYAwNChQ63uT6vVlut51atXtzorMTERwIN7Bovz8/NDkyZNkJ+fj19++cXqLFNycnKwY8cOAMCgQYOM2vXLtmzZYpN8e9Pf3wcAV69etXmeLcaPbdu2YfXq1fjwww9Rv359xfp9FJYuXQoAj/Qeme+//x4A8OqrryrWZ3nGEGvHD/3YERYWBi8vL6P2zp07A4BhbCwvU8dYW40T9jyem5tlzVgxceJENG/eHL/99htiYmIgIhg5ciTy8vIwY8YMBAYGmr8BxWg0GnzzzTdwdXXFtGnTcPr0aQBAfHw8lixZgho1amDu3LlW59DjzelRrwBVHn/99RcAGCZtsKWVK1fi1KlTCA4ORr169RTp8+jRo5gzZw6GDRuGTp06lXojtxKWL1+O27dvw8HBAcHBwejfv78iBwe9/fv3Izc3F926dYOzszPWr1+Pffv2IT8/H40bN8bgwYPxxBNPKJZnyoYNG5CZmYmWLVsiJCTE6v7q1auHevXq4eTJk1i3bh0GDx5saIuPj8e2bdtQt25ddOrUyeqszMxMAEC1atVMtuvf40eOHFH0BFLv5MmTyM3NhZ+fHwICAozaQ0NDATx431YEBQUFuHDhAgDA39/fpllHjx7F2rVr4ezsjK5duyrSZ1ZWFkaPHo3GjRsrNvlOaXbu3InDhw8jJycHAQEB6N27N1q1aqVI3+np6Th48CC8vLzQtm1bxMfHY/Pmzbhz5w4CAwMRERGBp59+WpGskly8eBGxsbFwdnbGiy++qFi/3bt3R0xMDGbMmIH58+fDweHB59g3b97E4sWL4eTkhFdeecWqDHPGDnOYOsbaapyw5/Hc3CxrxgonJyf8+9//Rtu2bTFx4kScP38ecXFxaNOmDcaOHWv+ypegYcOG+Pjjj/HBBx9g5MiR2LZtG0aMGIHCwkIsWLDA5q8pPQYe9Vd49GjgEVziqL9kqF+/for3PXv2bNHpdDJo0CAJCQkx3H928OBBRfovKCiQNm3aiK+vr+GStXPnztn0EsfiD2dnZ5k+fbpiOfpLOYYPHy7t2rUzyvPw8ChxQgGl9OjRQwDI3LlzFetz7969hstb9JOEdOrUSRwcHCQ8PFxOnz6tSI7+0sySJh1p1KiR1ZeAlbafbtq0SQBIy5YtS/x7/eQvZU1WYu548Cgucfz+++8FgPj5+UlOTo6iWZs3bxadTidDhgyRDh06iIODg2i12nJfJlienAkTJggA2bVrl2GZLS9xNPUYOHCgpKenW52TkJBgeO+99dZbRjkajUbee+89xbbJlJkzZwoA+cc//mHW35WVdfXqVcMxpF69ejJo0CDp3bu3eHp6SlBQkPz0009W5+gvzWzbtq3J9jfffNNwz5g5TB1jlRwnysoyRYlLHM09d7BkrCju3XffNbyfnZyc5MiRIxb1U5r8/Hxp0aKFAJA2bdoIAOnbt6/iOfR4YoFWSdm7QPv5559Fo9GIs7OzHD58WPH+u3btWuQEoU6dOmZfe16a+fPnG93XYasCberUqfLdd9/J2bNnJSsrS06dOiUzZswQNzc3ASDz589XJGfKlCmGg4+Hh4csX75cUlNT5dy5c4aTSRcXF5scmERErl27Jo6OjuLo6GjWCWp5HD58WOrWrVvkPeHl5SUfffSRYrNi6W/0DgwMNLovIj4+3pDbo0cPizNK209XrVolwIPZ+kqin9Hs6tWrFueYYu8C7eLFi4aZ9b7++mvFs/T3E+ofrq6usmTJknJPWlNWTmJiojg6OhrdV2KLAu27776T6OhoSU5OloyMDLl06ZKsWrXK8F7o37+/1Tlbt241jB0AZNKkSXLu3DlJTU2VZcuWibu7u1n/V5a8J5o0aSIA5IcffjDr78qTdevWLenevbtR0anT6eSvv/6yOufkyZMCQBwcHIzukU5PTzfcB+fi4lK+jZKSj7FKjhNlZZlibYFm7rmDpWNFcRcuXDD8348ZM8bifspy8OBBw0zAXl5ecunSJZtl0eOFBVolZc8C7cSJE1KtWjVFi4uSpKWlyd69ew3T5f7rX/+yus+LFy+Kp6endO7cuchyWxVoJdm2bZsAD6ZgzsrKsrq/SZMmGQ5AS5cuNWofNGiQAJBXXnnF6ixT5syZIwCkV69eiva7evVq0Wq10rlzZzlw4IBkZGRISkqKYdbNdu3amTWzV0kyMjIkMDDQsA3Hjx+Xe/fuydatWyUgIMBw8mrN9pW2n+o/Je7QoUOJf1+rVq3HvkDLyMiQVq1amVVcWJqVnZ0tx44dkzFjxohGo5G+ffuWa1KC0nLu378voaGh4uPjYzSroS0KtJJcvXrVcOK/f/9+q3J++uknw9hhalbehQsXCgAJCAgo17qZu02JiYkCQKpWrWr2NyRlZR05ckQCAgIkKChINm3aJGlpaXL58mWZP3++uLm5yRNPPFGuWWDLyhk4cKAAkAYNGsjOnTslPT1dDh06JJ06dTKMHa6uruXaptKOsUqOE2VlmWLNWGFuljVjRXHDhg0zvMeffPJJm053rz/WTpo0yWYZ9PhhgVZJ2atAu3TpkuEkdsKECTbP08vLy5NWrVqJRqOR33//3aq++vbtKy4uLnLixIkiy+1doImIhIWFCaDM74VNmzbN8CmuqZMc/fT3tWvXtjrLFP3UyatWrVKsz5SUFHF2dpaAgADJyMgwau/Xr1+JBakljh49anh/P/wIDAyUyZMnl3gCW16l7aeV4RLHvLw86dmzp+EE09wPJqwZ55SaBTM6OloAyPLly43a7FmgifzvQ5mSfjOsvDl79uwxvNe3b99u1J6ZmWn4jbfyXFJs7jbpf9Nr5MiR5f6b8mTl5eVJcHCwODg4SFJSklG7/kOl8uzTZW1TWlqaYSa/hx/u7u4ye/ZsASA1a9YsM6esY6yS44Qlx3NLxwpzs6wdKx62c+dOwwcM+mOGOZfsmkun0wkA+eSTT2yWQY8fzuJINnPr1i10794dFy9exLBhwxAdHW23bP2N4yJi9Sx2P/30E9zd3REVFYUuXboYHi+99BKABzer65dlZGQosfolatiwIQDg2rVrVvf15JNPAnhwE7Wpmcv07Tdv3rQ6q7g///wThw4dgqenJ/r3769Yv2vWrEF+fj569eoFDw8Po3b9pCG7d+9WJK9p06Y4efIkYmJiMG7cOERFRWHJkiU4fvy44TlKTH5iin7CmMuXL5tsz8zMxN27d1G1alWTM8WpXWFhIYYOHYpt27ahefPm2LJlC9zc3OyWr59V1NyZ9IrbsmULNBoNYmJiiowfXbp0wfXr1wEAAwYMQJcuXbBv3z6r17s0So0f+rEBAIKCgoza3d3d4efnB0D58aOgoABr1qwBoMzMrw9LSEhASkoK6tWrh5YtWxq1Kzl+VK1aFTt37sTWrVvx3nvvYdSoUZg5cyZOnDhhmGClrLGjPMdYpcYJex7Pzc1ScqzIycnBm2++CQBYuHAhli5dCm9vb8ybN8/sSVuIrMFZHMkm0tPT0bt3b5w8eRIDBgzAsmXLoNFo7LoOvr6+AIDU1FSr+7p79y727Nljsi07O9vQdv/+fauzSpOWlgYA8PT0tLov/QlIWloaRMTo/0c/nbESWcV99913AB6cmLq7uyvWr/4kpEqVKibb9cvv3LmjWKabmxtee+01vPbaa0WW//bbbwBMT8OvhEaNGkGr1SI1NRWXL182mqEtKSkJANCsWTOb5NvamDFjsG7dOgQHB+PXX39F1apV7Zqv5PghIti7d2+J7fHx8QAenJjaklLjR2BgIHx8fHD79m2T+1JhYSHu3r2rSFZxO3bswLVr1xAUFISOHTsq2re9xw+NRoNevXqhV69eRZZ/+eWXAEofO8p7jFVinLDn8dySLCXHiunTp+P06dOIjIxEREQEAGDWrFmIiorCqFGjEB8fb5jZk8iW+C4jxeXm5iIiIgIHDx5Ez5498d///tfwm0L2pC+arP29IXlwKbDR49y5cwAeHAD1y2x5EpmamorY2FgA/5sa2RpNmzZF3bp1kZ2djQMHDhi16z8lViLrYSKC1atXA1D294uA/02pfPDgQZPtf/zxB4Ci3wDYwp49e5CUlISQkBC0b9/eJhlubm547rnnAADr1683atcv69u3r03ybemDDz7A0qVLERgYiO3bt6NGjRp2Xwelxo/du3eXOIbov326du0aRETRb5OLExH8+OOPAKDIdPv9+vUDAOzatcuoLS4uDnl5eXBzc0Pjxo2tznqY/rfPhg4dqniRoB8/Tp06hfT0dKN2e4wfd+7cQUxMDFxcXKDT6Uw+x5xjrLXjhD2P55ZkKTlWHD9+HNHR0ahSpQq++uorw/I333wTzzzzDH7//XcsWrTI4v6JzGLfKypJLWCje9Du378vkZGRAkA6duxo0xtr9+7dK2vWrJH8/Pwiy/Py8uTLL78UBwcHcXNzk4sXL9ok3xb3oMXHx8vOnTulsLDQKKt9+/YCmD+tdGmWLFkiAKR169ZFJjA4ePCg4b4Ec2dJK4v+/pVatWqVe5a88tJPHgBAFi9eXKQtPj5ePDw8SrxvxhKHDh0yev8lJiZKrVq1RKPRWH2vYFn76fbt2wV4MB13SkqKYXlcXJxotVqpUqWK3L592+qc4mx5D5r+Ph9/f/8i22SJ0rJu3Lgh0dHRkpaWZtT266+/ip+fnwCQ9evXW5VTGqXvQUtNTZWYmBije0rT09MNU7f7+/uXa1wua5uSk5PF0dFRqlevXuR+rRs3bkhoaKgAkLfeesvqbXpYZmameHp6CgD5888/y9W3OVnZ2dlSo0YNASCvvfZakdfxypUr0rRpUwEgH374oVU5IiKnTp2Sv//+u8iyGzduGPatTz/91OTfWXKMtXScUOJ4Xt6xwpIsJceKgoICCQ8PFwCycOFCo/bjx4+Ls7OzeHl5yeXLl63KKo73oJEpvMSxkvj555/x2WefFVmWl5eH8PBww7+nTp2K559/3qqchQsXGj6l9fX1xZgxY0w+Lzo62nAJkaXOnj2LYcOGwdfXF61atYKPjw9u3bqFY8eO4dq1a3B1dcW3336LOnXqWJVjTydPnsSwYcNQs2ZNBAcHw9/fH5cvX0ZiYiJycnIQEhKCZcuWKZY3cuRI7NixAz/88AMaNWqEZ555BhkZGYZPwEeOHIlBgwYplgf87xPwV155RfFLRUJDQzFp0iRER0djzJgxWLRoEZo0aYKrV68iPj4ehYWFGDVqFLp166ZI3rvvvosTJ06gRYsW8PX1xfnz53HgwAE4ODhg6dKlePbZZ83qz9z9tFu3bnjnnXewYMECtGjRAt27d0deXh62b9+OwsJCrFq1yuQPnpqbc+3aNURGRhraTpw4AQAYMWKE4RK2559/HlOnTrUq6/Dhw5g0aRIAoG7dupgxY4bJ12nEiBHo0KGDVVlZWVmYNGkSpk6dirCwMAQEBCAzMxMpKSk4efIkAGD8+PEYOHCgVTnWMicrIyMDOp0O48aNw1NPPYXAwEDcvXsXSUlJuH37NqpWrYr169ebvKzY3G1q0qQJ5s2bh7fffhvt2rVDu3bt4Onpif379yMtLQ2hoaH4/PPPrd6mh23cuBEZGRlo3bp1ub+ZMyfL1dUVS5cuxQsvvICVK1dix44dCAsLQ3Z2NuLj45Geno7Q0FC8//77Vm/T6tWrMXv2bISFhaF27dq4c+cOYmNjkZ2djddff93kvgRYdoy1dJywJMvSscLcLGvHiuIWLVqEhIQEhIeHIyoqyqg9JCQE7733HmbOnIlx48Zhw4YNZfZJZJVHXSGSfaxYscJotqjij/L+KGtpSvuR1Icf586dszrrr7/+kg8++EDat28vNWvWFGdnZ/Hw8JCQkBAZN26cYj9IXBJbfIN24sQJiYqKktDQUPHz8xMnJyfx9vaW8PBwmTNnjiLT6xdXUFAgixYtkpYtW4q7u7t4eHjIM888IytXrlQ8KycnxzBtsq1+X01EZMOGDdKjRw/x8fERJycnqVatmjz77LOKzhgpIrJs2TLp3Lmz+Pn5ibOzs9SqVUuGDBkihw4dsqg/S/fTFStWSKtWrcTd3V28vb2lZ8+eEhsbq1iO/r1e2qP4b3xZkrVr165yjR8ljVXmZGVmZsrs2bOlT58+EhQUJG5ubqLVauXJJ5+Ul156qcgPSiv1/1Rceb5BMyfr3r17MmXKFOncubPUrl1btFqtuLu7S0hIiEycOLHUT/4t3aZt27ZJ9+7dxdvbW7RarTRp0kSmTZtW6jcglmb17t1bAMiCBQtK7FuJrKSkJBkyZIgEBAQYjistWrSQmTNnljgGm5uzZ88eiYiIkICAAHFxcZHq1atLjx49ZMOGDaVujzXHWHPHCUuyLB0rzM2ydqx42KVLl8TLy0ucnJzk6NGjJT4vOztbGjRoIABk48aNZfZbXvwGjUzRiIiAiIiIiIiIHjlOEkJERERERKQSLNCIiIiIiIhUggUaERERERGRSrBAIyIiIiIiUgkWaERERERERCrBAo2IiIiIiEglWKARERERERGpBAs0IiIiIiIilWCBRkREREREpBIs0IiIiIiIiFSCBRoREREREZFKsEAjIiIiIiJSCRZoREREREREKsECjYiIiIiISCVYoBEREREREakECzQiIiIiIiKVYIFGRERERESkEv8Pbo9SiKSA2QcAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 900x600 with 2 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "fig, ax1 = plt.subplots(figsize=(6,4),dpi=150)\n",
    "ax2 = ax1.twinx()\n",
    "_bar_p = ax1.bar(sorted_chrs, \n",
    "                 [chr_2_region_num[_c] for _c in sorted_chrs], alpha=0.5)\n",
    "ax1.set_ylabel(\"Num. of regions\")\n",
    "ax2.plot(sorted_chrs, [chr_2_len[_c] for _c in sorted_chrs], color='r')\n",
    "ax2.set_ylim([0, np.max(list(chr_2_len.values()))*1.1])\n",
    "ax2.set_ylabel(\"Chr size (bp)\")\n",
    "plt.savefig(os.path.join(report_folder, 'target_chr_stat_summary.png'), \n",
    "            dpi=300, transparent=True)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a9429fce",
   "metadata": {},
   "source": [
    "# Create count table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 169,
   "id": "3cbb34c9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\ctp14-human_genome_library_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "from tqdm import tqdm\n",
    "overwrite_table = False\n",
    "# library\n",
    "library_table_file = os.path.join(reference_folder, 'ctp14-human_genome_library_17w.npy')\n",
    "\n",
    "if not os.path.exists(library_table_file) or overwrite_table:\n",
    "    # library\n",
    "    _library_filenames = [os.path.join(library_folder, 'blast_full_probes.fasta')]\n",
    "    print(len(_library_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=library_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_library_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=44) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {library_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "153261f4",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "postanalysis",
   "language": "python",
   "name": "postanalysis"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
