{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "989d42da",
   "metadata": {},
   "source": [
    "# Design CTP-14, mouse, long-5kb-multiplex-tracing library\n",
    "\n",
    "by Pu Zheng\n",
    "\n",
    "2022.12.12\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8d307bec",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3b459614",
   "metadata": {},
   "source": [
    "<a id='0'></a>\n",
    "# 0 Minimum required packages and settings"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4846ea43",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "986751f2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "9948\n"
     ]
    }
   ],
   "source": [
    "%run \"..\\..\\Startup_py3.py\"\n",
    "sys.path.append(r\"..\\..\\..\\..\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "from ImageAnalysis3 import library_tools\n",
    "print(os.getpid())"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bd4cf4d8",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "9e5e2942",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Use pool_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\n"
     ]
    }
   ],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain'\n",
    "if not os.path.exists(pool_folder):\n",
    "    print(f\"Create pool_folder: {pool_folder}\")\n",
    "    os.makedirs(pool_folder)\n",
    "else:\n",
    "    print(f\"Use pool_folder: {pool_folder}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "9864b112",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "create library folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\n",
      "create sequence folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\n",
      "create report folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\reports\n",
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\reports\n"
     ]
    }
   ],
   "source": [
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'mouse_long_5kb')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7e248709",
   "metadata": {},
   "source": [
    "<a id='1.1'></a>\n",
    "## 1.1 select regions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "id": "7fe0571f",
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm.notebook import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "id": "5da19f2a",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- load sequence: 1, size=195471971\n",
      "-- load sequence: 2, size=182113224\n",
      "-- load sequence: 3, size=160039680\n",
      "-- load sequence: 4, size=156508116\n",
      "-- load sequence: 5, size=151834684\n",
      "-- load sequence: 6, size=149736546\n",
      "-- load sequence: 7, size=145441459\n",
      "-- load sequence: 8, size=129401213\n",
      "-- load sequence: 9, size=124595110\n",
      "-- load sequence: 10, size=130694993\n",
      "-- load sequence: 11, size=122082543\n",
      "-- load sequence: 12, size=120129022\n",
      "-- load sequence: 13, size=120421639\n",
      "-- load sequence: 14, size=124902244\n",
      "-- load sequence: 15, size=104043685\n",
      "-- load sequence: 16, size=98207768\n",
      "-- load sequence: 17, size=94987271\n",
      "-- load sequence: 18, size=90702639\n",
      "-- load sequence: 19, size=61431566\n",
      "-- load sequence: MT, size=16299\n",
      "-- load sequence: X, size=171031299\n",
      "-- load sequence: Y, size=91744698\n",
      "-- load sequence: JH584299.1, size=953012\n",
      "-- load sequence: GL456233.1, size=336933\n",
      "-- load sequence: JH584301.1, size=259875\n",
      "-- load sequence: GL456211.1, size=241735\n",
      "-- load sequence: GL456350.1, size=227966\n",
      "-- load sequence: JH584293.1, size=207968\n",
      "-- load sequence: GL456221.1, size=206961\n",
      "-- load sequence: JH584297.1, size=205776\n",
      "-- load sequence: JH584296.1, size=199368\n",
      "-- load sequence: GL456354.1, size=195993\n",
      "-- load sequence: JH584294.1, size=191905\n",
      "-- load sequence: JH584298.1, size=184189\n",
      "-- load sequence: JH584300.1, size=182347\n",
      "-- load sequence: GL456219.1, size=175968\n",
      "-- load sequence: GL456210.1, size=169725\n",
      "-- load sequence: JH584303.1, size=158099\n",
      "-- load sequence: JH584302.1, size=155838\n",
      "-- load sequence: GL456212.1, size=153618\n",
      "-- load sequence: JH584304.1, size=114452\n",
      "-- load sequence: GL456379.1, size=72385\n",
      "-- load sequence: GL456216.1, size=66673\n",
      "-- load sequence: GL456393.1, size=55711\n",
      "-- load sequence: GL456366.1, size=47073\n",
      "-- load sequence: GL456367.1, size=42057\n",
      "-- load sequence: GL456239.1, size=40056\n",
      "-- load sequence: GL456213.1, size=39340\n",
      "-- load sequence: GL456383.1, size=38659\n",
      "-- load sequence: GL456385.1, size=35240\n",
      "-- load sequence: GL456360.1, size=31704\n",
      "-- load sequence: GL456378.1, size=31602\n",
      "-- load sequence: GL456389.1, size=28772\n",
      "-- load sequence: GL456372.1, size=28664\n",
      "-- load sequence: GL456370.1, size=26764\n",
      "-- load sequence: GL456381.1, size=25871\n",
      "-- load sequence: GL456387.1, size=24685\n",
      "-- load sequence: GL456390.1, size=24668\n",
      "-- load sequence: GL456394.1, size=24323\n",
      "-- load sequence: GL456392.1, size=23629\n",
      "-- load sequence: GL456382.1, size=23158\n",
      "-- load sequence: GL456359.1, size=22974\n",
      "-- load sequence: GL456396.1, size=21240\n",
      "-- load sequence: GL456368.1, size=20208\n",
      "-- load sequence: JH584292.1, size=14945\n",
      "-- load sequence: JH584295.1, size=1976\n",
      "- 66 sequences loaded.\n"
     ]
    }
   ],
   "source": [
    "# prepare sequence reader\n",
    "reload(ia.library_tools.sequences)\n",
    "resolution = 5000\n",
    "flanking = 0\n",
    "\n",
    "seq_rd = ia.library_tools.sequences.sequence_reader(genome_folder, resolution=resolution, flanking=flanking)\n",
    "seq_rd.load_ref_sequences() # load genomic sequences"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f6ad39aa",
   "metadata": {},
   "source": [
    "<a id='1.2'></a>\n",
    "## 1.2 manually input region information"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "id": "1b010660",
   "metadata": {},
   "outputs": [],
   "source": [
    "region_dicts = [\n",
    "    {\n",
    "        'Chr': '7',\n",
    "        'Start':'44000001',\n",
    "        'End':'49000000',\n",
    "        'Region':f\"7:44000000-50000000\",\n",
    "        'Strand':'+',\n",
    "        'Gene': f\"Chr7-6Mb\",\n",
    "    }\n",
    "]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 137,
   "id": "ac32b5ca",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n"
     ]
    }
   ],
   "source": [
    "seqs = seq_rd.find_sequence_for_region(region_dicts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 138,
   "id": "6848aac9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Chr7-6Mb': [SeqRecord(seq=Seq('GGTGGGGTTGAAACAAATGCAGACAAAAGGCTCAGACAAACTCAGGAATGCTCG...AGC'), id='7:43999999-44004999_strand_+_gene_Chr7-6Mb-seg-1', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAAGCCCTGAATCATGGACTTGCCTAATCCCACACTACTGTCATTGGCTGTCT...GTA'), id='7:44004999-44009999_strand_+_gene_Chr7-6Mb-seg-2', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCCAGCAGCTGGGGAGGAGTGTCTTTACAAATCCTGGATGAGCAGTTTAGAGA...ACT'), id='7:44009999-44014999_strand_+_gene_Chr7-6Mb-seg-3', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGGACAGTGGGGTGGGACTGGCTGAAAGTCTACCCATGGCCTCCTGGACTCT...CTC'), id='7:44014999-44019999_strand_+_gene_Chr7-6Mb-seg-4', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCTGGAATAGTCTGAATTTTCATACTTTTGAATAAAAGAGTGATCAAGCCATG...TTG'), id='7:44019999-44024999_strand_+_gene_Chr7-6Mb-seg-5', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATTCTGGAGACAACACATCCCTCACTTGGGTGTGTTACTTAGGCCTATTTACC...GCC'), id='7:44024999-44029999_strand_+_gene_Chr7-6Mb-seg-6', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAGAGATTTCAAAGGAGAAGAATTTTAGAATGTGGCATGAAGACTGCTTTTGT...GCT'), id='7:44029999-44034999_strand_+_gene_Chr7-6Mb-seg-7', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAGACATCTGAGCACCTTCCCCACCAGAGAAGAGTTATCTGCTTAAGAGGGCT...AAA'), id='7:44034999-44039999_strand_+_gene_Chr7-6Mb-seg-8', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGTACATTTACACAATGGAGTACTACTCAGCTATTAAAAACAATGAATTTATG...CTG'), id='7:44039999-44044999_strand_+_gene_Chr7-6Mb-seg-9', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTCAAGAGTTCTTTCTGTCCCCACAGCATCGATCTCCCTATCCAGGACCAGCC...GTC'), id='7:44044999-44049999_strand_+_gene_Chr7-6Mb-seg-10', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCAATGAGGTTCTTCTGGGGTTGTGGCTCAGTGAGTGAGACCCTGTGAAGTGT...TGC'), id='7:44049999-44054999_strand_+_gene_Chr7-6Mb-seg-11', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTCACTGAGGGTAGGCTGTGAGAGAAGTCCTCACTCTCCTCAGAGCAGCTCTG...TTG'), id='7:44054999-44059999_strand_+_gene_Chr7-6Mb-seg-12', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGCTTACAGGACACACATGCATACACCGAGGTAACATGTGCTAACTACCATGA...ATA'), id='7:44059999-44064999_strand_+_gene_Chr7-6Mb-seg-13', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGCAGAGGAAAATGACCAGTACTAAAGGGATTGAAGTTTGGGCGGGTCCCGA...AAC'), id='7:44064999-44069999_strand_+_gene_Chr7-6Mb-seg-14', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGGAGGGAAGTTAGAGAGATAACTAGTCACTCCCACATCTCTCTCTCTCTCT...AGG'), id='7:44069999-44074999_strand_+_gene_Chr7-6Mb-seg-15', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGAACTCTCTGCTCAGCAGCACCTTGTCAGTAAAAGCTTCCCTCACCCTGGCT...AAG'), id='7:44074999-44079999_strand_+_gene_Chr7-6Mb-seg-16', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCTTGCCCTTCTAGAGGTATGGTCTCCAACCTCTGTCCCCCCCTATTAATTAA...CAA'), id='7:44079999-44084999_strand_+_gene_Chr7-6Mb-seg-17', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGTCCTCTGCCATTTCTCAGTCCCGTGTGGGCACCAGATCTGTTGTGTCTGAC...GAC'), id='7:44084999-44089999_strand_+_gene_Chr7-6Mb-seg-18', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGTCCTATATAATCTGAATCTATGACTCCAGGATGGACTATCAGGCCCTGCAT...TTC'), id='7:44089999-44094999_strand_+_gene_Chr7-6Mb-seg-19', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGTGGAAGCTGTTTGAGCCTCATCTCCAGTACCACTTTAACTCTACCCATTC...AAG'), id='7:44094999-44099999_strand_+_gene_Chr7-6Mb-seg-20', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGCAGGAAATGGTTCAGAGATTCAGCACCTGCCTAGAGTCTCCAATGAGGTTC...GAG'), id='7:44099999-44104999_strand_+_gene_Chr7-6Mb-seg-21', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCTGGATGATGAGGGATGAGTCGAGAGAGTTCCGACCTTGGGCTGCAGGCCAG...GAA'), id='7:44104999-44109999_strand_+_gene_Chr7-6Mb-seg-22', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTAGGGGCCCAATTGATCACCAGCTCCTCACTTACCCTGGATTAGGGCTAGCCT...GGT'), id='7:44109999-44114999_strand_+_gene_Chr7-6Mb-seg-23', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGTGTTCAGAGTCCTGTTTCTTTTTTTTTTTCTCTCTTATTATCTGTCTCTCT...TAC'), id='7:44114999-44119999_strand_+_gene_Chr7-6Mb-seg-24', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCTATCTCACACTCCAGTGCTATGCACACTCCAGTGAAAGTCTGCTTGACAGG...AAC'), id='7:44119999-44124999_strand_+_gene_Chr7-6Mb-seg-25', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGCTTGTTATCACCAACGTCCAGCCGATGGTAATGAACTGAAACAAATGCGCT...GTC'), id='7:44124999-44129999_strand_+_gene_Chr7-6Mb-seg-26', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGCAAGGACACCAGGGGAGAGCCCTAATTGTTTTATGTCTGCAGCGAAATTG...AGA'), id='7:44129999-44134999_strand_+_gene_Chr7-6Mb-seg-27', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTCCTCCAATGAGGTTCTTCTGGGGTTGTGACTCAGTGGTAGAGACCCTGTGGA...AGA'), id='7:44134999-44139999_strand_+_gene_Chr7-6Mb-seg-28', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGGGCTGGTCAGAACTGGAGCTGGGTGAGGTAACTGAGTACAGAGAGAGCTG...GGC'), id='7:44139999-44144999_strand_+_gene_Chr7-6Mb-seg-29', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACTATACCCAGCCAAACTCTCAATTACCATAGATGGAGAAACCAAAGTGTTCC...CAG'), id='7:44144999-44149999_strand_+_gene_Chr7-6Mb-seg-30', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTATGATTGAATCAGAATAACTGGTTACATTAAAGGAAAGGGAAGATTCATCA...AAA'), id='7:44149999-44154999_strand_+_gene_Chr7-6Mb-seg-31', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATCTGGTCCAATAGGGAGGAGTTTTGGTGCTCCCCCACCTCCTCCGCCTTATG...GAA'), id='7:44154999-44159999_strand_+_gene_Chr7-6Mb-seg-32', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTGCTCAAATGCCTATGGTTTCTACTCCTGTTACAATGCCATCTGCTGCCAAG...CAC'), id='7:44159999-44164999_strand_+_gene_Chr7-6Mb-seg-33', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATACATCATTACCTGGACCTATGGTTAAGGGAAATGCAATTGCTGATTCAGCCA...ATA'), id='7:44164999-44169999_strand_+_gene_Chr7-6Mb-seg-34', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTGAAATCTACTTGGTCTCCTGCCTCTCCTTCACTTCCTCCCATTTCCTTCCA...ACT'), id='7:44169999-44174999_strand_+_gene_Chr7-6Mb-seg-35', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATAGAGGAGAAAGTGGGGGAAAGCCTTGAAGATATGGACACAGGGGAAAAAAA...ACT'), id='7:44174999-44179999_strand_+_gene_Chr7-6Mb-seg-36', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCTCCGTAGTACAAGGACAATTGGCATATTAGGGTGGGAGGCTGTGAAGGGCT...ATC'), id='7:44179999-44184999_strand_+_gene_Chr7-6Mb-seg-37', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAACTCTGCCGAATGCTAGCTAAACTTGCACCAATATCTCCTTTGACTGGCAA...TTC'), id='7:44184999-44189999_strand_+_gene_Chr7-6Mb-seg-38', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTCTCATTGGACTAAAATTGAAGCTTACCTGCCATCCCTTAACCTGAAACTCT...CAT'), id='7:44189999-44194999_strand_+_gene_Chr7-6Mb-seg-39', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGCTGGAGGCCAGCTTGGGTTACATAGGCCTGTCACAAACAAGGTAGAGGAA...CTT'), id='7:44194999-44199999_strand_+_gene_Chr7-6Mb-seg-40', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCTGTCTCTCTCATTGCCCTTCCAACCCTTGGTCCAAAACAATAAAGTGGCC...CAA'), id='7:44199999-44204999_strand_+_gene_Chr7-6Mb-seg-41', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCCTCCCGACAGGTGCATACAAATCAAAGTGCCGGCAGACTGACCAATCCCAG...CCC'), id='7:44204999-44209999_strand_+_gene_Chr7-6Mb-seg-42', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTGGGACTCTGCAAAGCCCTGTCTGGGTCTGGGTCTGGGTCTCATGTGTTCTC...TGA'), id='7:44209999-44214999_strand_+_gene_Chr7-6Mb-seg-43', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGCGTGCTTCCTCTCTTCTCTGTTAGAGCAAAGAACCGTGAAGTTTTCAGGAA...CCG'), id='7:44214999-44219999_strand_+_gene_Chr7-6Mb-seg-44', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGATGATCTCCAGTGTGTGAACTTCAAGCTCCTGCCTAATGAGGACTGTGTCA...AGC'), id='7:44219999-44224999_strand_+_gene_Chr7-6Mb-seg-45', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGCAGCCCTTGTGGCTATCAACGGCAAAGAATTTCTACTAACTGCTGGTTTCT...GGG'), id='7:44224999-44229999_strand_+_gene_Chr7-6Mb-seg-46', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGAGTCTGTCCACAGAAGGGCAGGGACTCAGCCCAGGCTGCAGACTGTGCACC...AAA'), id='7:44229999-44234999_strand_+_gene_Chr7-6Mb-seg-47', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATAAATGGAGGCTGGAGAGGGGACTCAGCAGTTGAGAGCACTGGATGCTCTTC...GGA'), id='7:44234999-44239999_strand_+_gene_Chr7-6Mb-seg-48', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATATGCAAACGAGATACTCTATGTGGTCTCCCTTCAAAGAAGTCATGAATGGT...GGG'), id='7:44239999-44244999_strand_+_gene_Chr7-6Mb-seg-49', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGAACCCAGCGCTTCCTTCATGTTGGACAGGCACTCTACCAACTGAGCCACAT...TTC'), id='7:44244999-44249999_strand_+_gene_Chr7-6Mb-seg-50', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCTCTCCCTCCTGCCAACAGACAGAGGCAGTTCCTTCAGACAACCAGGCTTCC...TGG'), id='7:44249999-44254999_strand_+_gene_Chr7-6Mb-seg-51', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGATCAAGGTATCCAGAACTTTCCATGCTCTCCGGAGTGGCTCTCCAACCAGG...AGC'), id='7:44254999-44259999_strand_+_gene_Chr7-6Mb-seg-52', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCAGCTGCTCTTCCAGAGGTCCTGAGTTCAATTCCCAGCAACCACATGGTGAC...TCC'), id='7:44259999-44264999_strand_+_gene_Chr7-6Mb-seg-53', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTCATCCTCAAATAATGACACTCCCTGGTGACTTAGCATTTAAATAAATGAG...AGA'), id='7:44264999-44269999_strand_+_gene_Chr7-6Mb-seg-54', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTACAGAAGGGTGTAAGCCACCGTGTTGGGGCTTGGAACTGAACCTGGGTCCT...AGG'), id='7:44269999-44274999_strand_+_gene_Chr7-6Mb-seg-55', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGACCGAGCATCTGACCAAGGAGTCCTATGTAACTTTAACTTTTTTCAACATCT...CAG'), id='7:44274999-44279999_strand_+_gene_Chr7-6Mb-seg-56', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGTGATTTTGCACATTTGTAATCTCAGTCTCAGGGAGGCAGAGACAGATAGAT...GAA'), id='7:44279999-44284999_strand_+_gene_Chr7-6Mb-seg-57', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCCCTATAACTAAAATGTCCTCCGAGTGCATGTGTGGCAGACAGAAGGCTGGT...ATG'), id='7:44284999-44289999_strand_+_gene_Chr7-6Mb-seg-58', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAAGTACACTGTAGCTGTCTTCAGACACTCCAGAAGAGGGCATCAGATCTCATT...ATG'), id='7:44289999-44294999_strand_+_gene_Chr7-6Mb-seg-59', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGGAGAGAGTTAGATTAAAAAAGGAAATATTTTCTATGTTAAAAATGGAGAA...CCC'), id='7:44294999-44299999_strand_+_gene_Chr7-6Mb-seg-60', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTGAAGAGATAATAATTTTAAGGTAAAATGTGGTGCCAGCGCCTGAATTCCT...GGT'), id='7:44299999-44304999_strand_+_gene_Chr7-6Mb-seg-61', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGGCGGAGTCAGAACGTTGCGGGTAGGAGGGGGCGAGGCCAAAGCTAGGCCTT...GAG'), id='7:44304999-44309999_strand_+_gene_Chr7-6Mb-seg-62', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGAAGGGTGGAGGAGGGCGAGGGGGGGGGTGGAGCCGCGCGTGTGTTGGGGGG...ACC'), id='7:44309999-44314999_strand_+_gene_Chr7-6Mb-seg-63', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAAGCAGCTAGAAACGTTAGAAGCTGGCATTTACCTGCCATTGGCAGGTGCTG...CAC'), id='7:44314999-44319999_strand_+_gene_Chr7-6Mb-seg-64', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCATTCATCAGTACATCTAGAAACTCATCTACCCATCCATCCACTTATCTATCT...TCA'), id='7:44319999-44324999_strand_+_gene_Chr7-6Mb-seg-65', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTTTTACTTTTTATCCTGAGACAGCACTTTGCTAAGTTGCCCATGCTGGCCTT...TAT'), id='7:44324999-44329999_strand_+_gene_Chr7-6Mb-seg-66', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCCCCATGCCCAGCCTCTTCTGTTCAAGCCCCTTGATCGCTGTAGGTGTTCCT...TTG'), id='7:44329999-44334999_strand_+_gene_Chr7-6Mb-seg-67', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGCCTTGCAATTTTTGTCCTTTTCCCTCCTCTCCCGGGGCTGAGGTCCATCCA...GGG'), id='7:44334999-44339999_strand_+_gene_Chr7-6Mb-seg-68', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGAGGGAGAGGAGGTAGAAACCTGTGGACAGGTTTATTGACCTCTGGATAAGT...AAA'), id='7:44339999-44344999_strand_+_gene_Chr7-6Mb-seg-69', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGATTCTTTGCCACTGAGGTAGGTGTGTGGGAAGGAAACTAAGGAAAGACTCCC...TAA'), id='7:44344999-44349999_strand_+_gene_Chr7-6Mb-seg-70', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGATGAATAGCCAGACAAACGAAATGGTGGATGGAAGATACACGGACGGAATG...GAT'), id='7:44349999-44354999_strand_+_gene_Chr7-6Mb-seg-71', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAACCACTGAGCTACACCCCCAGCTGTATTATGTTTTGAGTCAGGGCCTAAGT...TCG'), id='7:44354999-44359999_strand_+_gene_Chr7-6Mb-seg-72', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCCGCCCCCCACGCCCAACACGACTTTGCCGAGAGAAATGGCTTTTGTATCAG...ACG'), id='7:44359999-44364999_strand_+_gene_Chr7-6Mb-seg-73', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGTCTCCACATCACTGAGCTGACCAATCTCCCCGAGAATACTGCAGAATCTCT...CAG'), id='7:44364999-44369999_strand_+_gene_Chr7-6Mb-seg-74', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATGCAAATTAGAATGAACGGAAATCAAATAAGTTACATGAGCGAGTTGGGAAC...GCA'), id='7:44369999-44374999_strand_+_gene_Chr7-6Mb-seg-75', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCGGCTAGACTCCTGGAGACAAACAGAAACTGAGTTCTCTAGGGGGGAGGGGAC...CCT'), id='7:44374999-44379999_strand_+_gene_Chr7-6Mb-seg-76', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAATAAGCTCACAAGTTCAGAAGCCTTCCTTGACATCTTTCCCAGTACTCTCA...GGC'), id='7:44379999-44384999_strand_+_gene_Chr7-6Mb-seg-77', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTCTTGACTGTCTGGCCCTCGTTCTTTCCCTCGGGGGGGGGGGGGGCTCTTT...TGT'), id='7:44384999-44389999_strand_+_gene_Chr7-6Mb-seg-78', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTAGAGCAGGACTGATATGTTGTCAATCTCCCAGGGTACCCTGAGGTACTAG...AGG'), id='7:44389999-44394999_strand_+_gene_Chr7-6Mb-seg-79', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGAGCATGATTGCTCCCCCTTCTCGATAGGTCCCTACCTCAGTCTCTGACAAC...CAT'), id='7:44394999-44399999_strand_+_gene_Chr7-6Mb-seg-80', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACCAGAAATAAACACACTCTGTGAGTCTCATCCCACCTGTCTCCCTGGGGCCA...CAG'), id='7:44399999-44404999_strand_+_gene_Chr7-6Mb-seg-81', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAAGTCTGGGTGGTGGCAGCTTGGAACAGTGAGTCACCTAGATAACCCCTCCC...ATA'), id='7:44404999-44409999_strand_+_gene_Chr7-6Mb-seg-82', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAACCAGAGAACCAAGGCAACACGGCCTCCCTGGTAACCAGGAAAGGCTCAGA...AAT'), id='7:44409999-44414999_strand_+_gene_Chr7-6Mb-seg-83', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATACCAACCAGAAAAAAAAAAAACTAAGACAATCTTTTAAATTTTTATTTTAA...TGA'), id='7:44414999-44419999_strand_+_gene_Chr7-6Mb-seg-84', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGTGTGTGTTCGTGGGTTCGCACCTGTGTGAAAGGCAGAGACACCCTTGGCT...ATG'), id='7:44419999-44424999_strand_+_gene_Chr7-6Mb-seg-85', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCAGGTATGTCTGTGTGCACATGTATATATGTGTGAAGGCATATGTGTGTATG...CTC'), id='7:44424999-44429999_strand_+_gene_Chr7-6Mb-seg-86', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCTTTCTCCTAGCTGTCAATCCTATGCCAGAAGCTCAGGCTCTGTCCAGGGTC...CAG'), id='7:44429999-44434999_strand_+_gene_Chr7-6Mb-seg-87', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTTTTGTTGCCCTGGACAGGCTGAGGCAGAGGGTTCACTCTTCCTAGCACGGA...AGA'), id='7:44434999-44439999_strand_+_gene_Chr7-6Mb-seg-88', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCAGAGATGGATCCTAACAGCGCAGGTGTGAAGCCAGGCATCCATACAGTGTG...TGG'), id='7:44439999-44444999_strand_+_gene_Chr7-6Mb-seg-89', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTTCTCCCCACCCCTGAGGGCTGGTGGGGGCGGTGTGGCCGTGACCTCAGCC...GAA'), id='7:44444999-44449999_strand_+_gene_Chr7-6Mb-seg-90', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCCCCACACCCTCTCTGCGAATTCTGAATAGGTATTTACAATGTGGTCTTAA...GAG'), id='7:44449999-44454999_strand_+_gene_Chr7-6Mb-seg-91', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAGGGAGAGAGAGAGAGAGAGAGAGAGAGAGAGAGGGGGGGGGGGGGAGAGAG...TCA'), id='7:44454999-44459999_strand_+_gene_Chr7-6Mb-seg-92', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTAGGTATACATTTTGTAAGGGGTAGGAGAATATTGTGGCAAGCTGTGAATGG...ACA'), id='7:44459999-44464999_strand_+_gene_Chr7-6Mb-seg-93', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTGGGGCCTGCAGGACCACATACATTCCAGCCAGACCTTTCCACTTAGACTTT...GTC'), id='7:44464999-44469999_strand_+_gene_Chr7-6Mb-seg-94', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGGGTCTGAAGGCTCCTCTCCCAGAAGAAATGACATCTGAAGCTGGTCTGGAG...AGA'), id='7:44469999-44474999_strand_+_gene_Chr7-6Mb-seg-95', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGAATGAACACACCTTGAGCCTCTACTGTATACCAGGGGTTCCACTAACTTCT...CCT'), id='7:44474999-44479999_strand_+_gene_Chr7-6Mb-seg-96', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTCCTCCTCCTCTTCCTCCTTCTCTGGCTCTGTCTCCCCTTCCTTGCCCTCTC...AGA'), id='7:44479999-44484999_strand_+_gene_Chr7-6Mb-seg-97', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTGTGTGCAATCCTGGGCTGGCCAGCGATGGGCTCTGGGACTCTTGTGAGATT...CTT'), id='7:44484999-44489999_strand_+_gene_Chr7-6Mb-seg-98', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAATCTGCTCAGGCACCCCTCTGTAACATACCCTAAGAAAGCCATTCAAACATA...CAG'), id='7:44489999-44494999_strand_+_gene_Chr7-6Mb-seg-99', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACTGCCTCTCATGACACAGTCCATTTCCATAGCCCAAAGAAAACTCTAAAAGC...TTG'), id='7:44494999-44499999_strand_+_gene_Chr7-6Mb-seg-100', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGATAGGGCAGGACCAGGATGCGAGAGAGGTTAAGACAGAAATCTGGTGGCTG...ACC'), id='7:44499999-44504999_strand_+_gene_Chr7-6Mb-seg-101', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTAGGGTGGCCTCTGACGGCACAGTTGAGAGCTGCGGTATATCCCGCCACCACG...GTT'), id='7:44504999-44509999_strand_+_gene_Chr7-6Mb-seg-102', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGGACTCACCCCGACTTTCCTTCTTCACACCCTTCTTGCCCACCACACACATC...AGT'), id='7:44509999-44514999_strand_+_gene_Chr7-6Mb-seg-103', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCTCCACACCATCTTTCATCCTGAGTGGAGGGAAGGCGGCCAGCGGTCTCAG...AGG'), id='7:44514999-44519999_strand_+_gene_Chr7-6Mb-seg-104', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAGGCCAGGCAAGGCAATGTAGTGAGACACTGTCTCAAAAATTATACACACAC...GAG'), id='7:44519999-44524999_strand_+_gene_Chr7-6Mb-seg-105', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCACTTGTGTGGGGGCAAAGAACGAGGAGGAAAAGAGAAAAGGAAAAAAGGGA...GAC'), id='7:44524999-44529999_strand_+_gene_Chr7-6Mb-seg-106', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGACAGGTGAGAGAAGGGAAGGGCTGATGGGGACAGAAACAGATAGAGACACAG...ACA'), id='7:44529999-44534999_strand_+_gene_Chr7-6Mb-seg-107', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACACACACACACACACCGGTCCACGAGGATCCGGCGCAGAGAGGATGTAACGA...TAA'), id='7:44534999-44539999_strand_+_gene_Chr7-6Mb-seg-108', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGAGCTTGCCCTAGAAAAAGGACGCAGGGAGACTTGACTGAGGCACAGAAGAA...GGG'), id='7:44539999-44544999_strand_+_gene_Chr7-6Mb-seg-109', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACACGACCTGTGCTGAGTTGTGGTAGTGGTACACACCTGGAGGCCAGCCTGAT...ATC'), id='7:44544999-44549999_strand_+_gene_Chr7-6Mb-seg-110', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAGATCTCGGACAGCAAGGGCGGCAGCTTCTTGTCCTGGAGTCGCAATGCAAA...TCT'), id='7:44549999-44554999_strand_+_gene_Chr7-6Mb-seg-111', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCATAAAAGTAATAATACCAGCTTTTTCACAAAAGATTGAAATACTTGTGTGA...GCT'), id='7:44554999-44559999_strand_+_gene_Chr7-6Mb-seg-112', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAAAAATAGCTGGGCGTGGTGGCACACGCCTTTAATCCCAGCGCTTGGGAGGC...TTT'), id='7:44559999-44564999_strand_+_gene_Chr7-6Mb-seg-113', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTCACCCCGTGGGATTAACACCTCTGTCTCCGACCCTCTCCCTATCTCTCTT...ATA'), id='7:44564999-44569999_strand_+_gene_Chr7-6Mb-seg-114', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATGCTGGGGCCCCGATAAGTGGGGTTGTTGGTTGTTTAGGGAAGTTGGTTTGT...ACC'), id='7:44569999-44574999_strand_+_gene_Chr7-6Mb-seg-115', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAGACTTCCAGGTCCACGCCAAGAACAGTAGATATAGGATACCCAGGTCATTC...TTA'), id='7:44574999-44579999_strand_+_gene_Chr7-6Mb-seg-116', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTATATGTAAGTACACTGTAGCTGTCTTCAGACACTCCAGAAGAGAGAGTCAG...AGG'), id='7:44579999-44584999_strand_+_gene_Chr7-6Mb-seg-117', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATTCTGAAGGGTCTGATGGGGGAGAGCTGGTCCTAGGGGGCTCAGACCCCGCT...TGT'), id='7:44584999-44589999_strand_+_gene_Chr7-6Mb-seg-118', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAAGCAGGAGGACAGCGAACCTAAGACCTCCCTCAACTCCACAACTTGCAGCC...TCA'), id='7:44589999-44594999_strand_+_gene_Chr7-6Mb-seg-119', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAACTGATTGCTAGAAGTGGCGGAAGCATTAAATGGGAACGCTTGGATTCTTGT...GGT'), id='7:44594999-44599999_strand_+_gene_Chr7-6Mb-seg-120', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGTGGTCTACTTCTAGAAACAACACAGCACCTTCTTCCAAAATAGAATCACGT...GGG'), id='7:44599999-44604999_strand_+_gene_Chr7-6Mb-seg-121', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCAATTTCCAGAGCTACAAAAATAAAACAACAAAAAGGTTACAAAGATGGCAG...CCT'), id='7:44604999-44609999_strand_+_gene_Chr7-6Mb-seg-122', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAGGGACCCCTCAGTTCACGTTTCCTTTTCTTTGGTTCGTTTTGTTTGTTTGG...CTA'), id='7:44609999-44614999_strand_+_gene_Chr7-6Mb-seg-123', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAGAATTATTCAAGGCGCTAGGTTCCATCTTCGGTGGCACTAAATTTATTATC...TCA'), id='7:44614999-44619999_strand_+_gene_Chr7-6Mb-seg-124', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTTGAGATTTAAAAAAAAATGGAAAAGCTGGTTGTTGTAGTGCATGCCTTTAG...GAC'), id='7:44619999-44624999_strand_+_gene_Chr7-6Mb-seg-125', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGAGCCCGCTCCGAATCACTGGATCGGCCCTGGACCTCCTGTAGCTGGGACTCC...CGG'), id='7:44624999-44629999_strand_+_gene_Chr7-6Mb-seg-126', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTCAGAGCAGAGCTCGGCTTCTGCTCGAAGCTGCTCAGCCAGGCGCGTGCGCT...GGT'), id='7:44629999-44634999_strand_+_gene_Chr7-6Mb-seg-127', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTGGGTTCAAATCCCGACTCTACCACTTACTGACTGTGTGACCTTGAGCAAGC...CTC'), id='7:44634999-44639999_strand_+_gene_Chr7-6Mb-seg-128', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTCGATCCTTTGTTTTGACATCACTAAACCTCCTCCGCAGAGGCATCGCTAAA...TTC'), id='7:44639999-44644999_strand_+_gene_Chr7-6Mb-seg-129', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTAGGGTTGTGTTGGGTTTCTGTTTGTTTTGTTTTGATTTGGTTTGATTTTTGC...TGA'), id='7:44644999-44649999_strand_+_gene_Chr7-6Mb-seg-130', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGGAGATGGGACCCAGGCAAGACTTCCTGACCTTTTGTCTCATCTTTGATATG...CTC'), id='7:44649999-44654999_strand_+_gene_Chr7-6Mb-seg-131', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGGACTACTATGCCCCCACTCCACAAAGCACAGCCAAGAGTCTGACCACGGG...AAC'), id='7:44654999-44659999_strand_+_gene_Chr7-6Mb-seg-132', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACACCACAGGACTATGACAAGGTTAAATAATGTAACCAAGACTGCTATCCTAG...ACT'), id='7:44659999-44664999_strand_+_gene_Chr7-6Mb-seg-133', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTTGGAGGACAGTCACAGGACGTGAGGCAGATACTCTAGACCACGCCCTGCTA...CCA'), id='7:44664999-44669999_strand_+_gene_Chr7-6Mb-seg-134', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTCCCTGTGTTCCCCACCTCGGTCCAGTCTATATCTTCACTCTCTGTGAGATG...CTA'), id='7:44669999-44674999_strand_+_gene_Chr7-6Mb-seg-135', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGGTTAAGATCACCAGCTGTTCTTGTGGTGAACCCAGCTTCTATTGTAGCATC...TGG'), id='7:44674999-44679999_strand_+_gene_Chr7-6Mb-seg-136', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGGTTTGCCTGTTGCTTGGTGCTTCTGGAGGACAGAAGAGGCTGACAGATCCC...ACA'), id='7:44679999-44684999_strand_+_gene_Chr7-6Mb-seg-137', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTCCCTGCGCCTGCTCTCTCCCCACCACCGATCTCTGCTCAGATGCTTCCCAG...GAC'), id='7:44684999-44689999_strand_+_gene_Chr7-6Mb-seg-138', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAAGTAGGAGACGAGCACTGAGCTTTGATTTGCTTTGCTCCGCCACTCTATTC...ACT'), id='7:44689999-44694999_strand_+_gene_Chr7-6Mb-seg-139', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGGGACTGGACTGCCCTTCTCTATATAATCCAGCCATTTGGGGTACATGGTCT...GGT'), id='7:44694999-44699999_strand_+_gene_Chr7-6Mb-seg-140', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGCGCCGCCGATGCCCAGTGTAAAAAAGGTTTTTAGATGTTGTGGCTTGCTGG...CCC'), id='7:44699999-44704999_strand_+_gene_Chr7-6Mb-seg-141', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCATCTCCCAAATAATGACACAGAGACCTTTTATATTTTTATGAAAGCTTTAG...TTG'), id='7:44704999-44709999_strand_+_gene_Chr7-6Mb-seg-142', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCTGCATGTATTATATGCCATTTGTGCATGCAGTACCTGTGGGTGTCAGAAG...CAG'), id='7:44709999-44714999_strand_+_gene_Chr7-6Mb-seg-143', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAGTAGCCCAGACTGCCCTCAACTCAGAGTGGTCCTCCTGCCTCAGTATCCCA...GTG'), id='7:44714999-44719999_strand_+_gene_Chr7-6Mb-seg-144', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGAGGGCTGTCTTTGGCTTTTTTTTTTTTAAAGACAGGATGTGATGGTTAGTG...TGG'), id='7:44719999-44724999_strand_+_gene_Chr7-6Mb-seg-145', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATGCTCACAGGTCCTTCCTGTCTCCATTTCTGTCTCACGCGGATCCCATTACA...TGC'), id='7:44724999-44729999_strand_+_gene_Chr7-6Mb-seg-146', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGAAGTAAACACTGATTGAAAGAGAATGGTCAACATTATAGCCTGGACCCAAA...TGG'), id='7:44729999-44734999_strand_+_gene_Chr7-6Mb-seg-147', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAAGGAGTTGGCAGGGTGACAAACAGACACTGACAGAGAGGGTGCTGATCTGA...TGT'), id='7:44734999-44739999_strand_+_gene_Chr7-6Mb-seg-148', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGTTTGCTTTGTTATTCAAGACCAGAGTTTCCAGGCAGCCCTGGTTGTCCCAT...TCC'), id='7:44739999-44744999_strand_+_gene_Chr7-6Mb-seg-149', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCCTCCCGCATTCTCTTACTACCAGTTCCCAGCCCCGAGAAAAGTGCCCACCT...GGG'), id='7:44744999-44749999_strand_+_gene_Chr7-6Mb-seg-150', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATAGGGGGTGAAATGGGAAGGGGAGAATAAAGGCAGGAAAAAGAACTCACAAAG...AGC'), id='7:44749999-44754999_strand_+_gene_Chr7-6Mb-seg-151', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGTCCCCTCCACACAGTCCCAGCTCCTCAGCAGACTGTAATGCATTTATGGT...GGG'), id='7:44754999-44759999_strand_+_gene_Chr7-6Mb-seg-152', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGGAGGAACCATAAAGCAGAGCAGCCCAGACCACATATCCAGGAATCAGCATT...CCT'), id='7:44759999-44764999_strand_+_gene_Chr7-6Mb-seg-153', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGGGGTGATGATGCCTCGTAAGATCAGCCAGGAGGTCTAGCCTGGAACAGAGG...CAG'), id='7:44764999-44769999_strand_+_gene_Chr7-6Mb-seg-154', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTTTAATGCTCATTGCATGCAACACACCACAACTTATTCACAGGCTGTAGTCT...AAT'), id='7:44769999-44774999_strand_+_gene_Chr7-6Mb-seg-155', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCACATGGGGCTTGCCTGCCATTCATGCCGGTCGGTCACCGTGATGTGATGCC...GTT'), id='7:44774999-44779999_strand_+_gene_Chr7-6Mb-seg-156', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTAGTATCAGACAGCTTCCATAAGTCTTTGGCTTATAGTTATTAGGTCTCACCA...GGG'), id='7:44779999-44784999_strand_+_gene_Chr7-6Mb-seg-157', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGTAGGGGAGTAGGGTGGGGGGAGGATATAGGGGACTTTCAGGATAGCATTT...CAC'), id='7:44784999-44789999_strand_+_gene_Chr7-6Mb-seg-158', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAATTATTTTTGTTGCTACTTCATAACTGTAATTTTGCTCCTGTTATGAATCA...GGC'), id='7:44789999-44794999_strand_+_gene_Chr7-6Mb-seg-159', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGATATCACTGAGGACTGCAGGCTGAGGGCCCAGAGCTCTCCATGGGTCTGGG...TAC'), id='7:44794999-44799999_strand_+_gene_Chr7-6Mb-seg-160', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGTATCGATTGTAATTGGTGGTAATCAAATATTAGCTACATATTCTAGTTATT...TTG'), id='7:44799999-44804999_strand_+_gene_Chr7-6Mb-seg-161', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGCCTCCATATGGATGCTAGGAATCAAACCTGGGTCCTCTGCAAGAACAGCC...GCC'), id='7:44804999-44809999_strand_+_gene_Chr7-6Mb-seg-162', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGATTCTTCTCTCTTGAAATAAGAAAGTATTTAATCTTTTATTTTACAGAAGC...GGC'), id='7:44809999-44814999_strand_+_gene_Chr7-6Mb-seg-163', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCCCCCCCCTGGAAGCCCGCCCTCCAGGGCACCCCCAAGGACCTCATAGGGG...TAA'), id='7:44814999-44819999_strand_+_gene_Chr7-6Mb-seg-164', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCCACACAGTGCTAGGGAACAAGTAAAACTGGGTCACGAAGGCCAGAGGGAC...AGT'), id='7:44819999-44824999_strand_+_gene_Chr7-6Mb-seg-165', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTTAAAACAACTACCTCACACCAGAAGGTTCCCTCAGAGCTAGCTGATACCCA...TGC'), id='7:44824999-44829999_strand_+_gene_Chr7-6Mb-seg-166', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGATCCTCAATGCACACATGGACTCTCTTCAGTGGGTGGACCAGAGCTCTGCC...GGG'), id='7:44829999-44834999_strand_+_gene_Chr7-6Mb-seg-167', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAACAGACAGACATAAACATGGTACATAAACATTCATAAAGGCAAAACTCAACC...CAC'), id='7:44834999-44839999_strand_+_gene_Chr7-6Mb-seg-168', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGGTCGGACGCTGCAGCCCCCTTCGCTGGACTGAGCACCGACCAGACCCTGCG...TCA'), id='7:44839999-44844999_strand_+_gene_Chr7-6Mb-seg-169', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTGACACTCACGCAAGAAATGACCTCAAAGCCAGGCTCTGGTTCATCATCTGG...CTA'), id='7:44844999-44849999_strand_+_gene_Chr7-6Mb-seg-170', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGGGCACGTGAGTGGAGGCGGCGTTGTGAGCTGGGGTTTGGGGAGGGTCTGGCC...CCG'), id='7:44849999-44854999_strand_+_gene_Chr7-6Mb-seg-171', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTGGTCCCGGCCTCCTGGGAGTGTGGCCAAGCCCTCCCTAACCCAGAATTGGC...TGT'), id='7:44854999-44859999_strand_+_gene_Chr7-6Mb-seg-172', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGTGTGTGTGCGCGGACACATGCATACACATGCCTGGGTTGAGACAGGGCTCC...CAC'), id='7:44859999-44864999_strand_+_gene_Chr7-6Mb-seg-173', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCCTATGGACAGGAAACCACCATGCGGTCAGCAGCTTTCACTCACAATTCCCC...GGA'), id='7:44864999-44869999_strand_+_gene_Chr7-6Mb-seg-174', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAATGGGTGGCGCGACGGAACCTACCACCAGACTGAGGGGTAGATGTAGGAACC...CTG'), id='7:44869999-44874999_strand_+_gene_Chr7-6Mb-seg-175', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTCACCCTGGCTGTCCTGTGTAGCCCTGGCTGTCCTGTGTCACCCTGGCTGTC...CCC'), id='7:44874999-44879999_strand_+_gene_Chr7-6Mb-seg-176', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGCTGGGGCTGCCCCATGCTCTGATGTGGTGGCAGCAATGTAGGGGCCCCTGG...GAC'), id='7:44879999-44884999_strand_+_gene_Chr7-6Mb-seg-177', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAGGAGCTTTGACTCCACCCCCACAGAGGCCAAGTCTCCACAGGGCCCCAGGA...GAG'), id='7:44884999-44889999_strand_+_gene_Chr7-6Mb-seg-178', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATCTGATGGCCTCTTTTGGTCTCAAAGAGCACCAGGCACAAGTGTTGCATATA...GCA'), id='7:44889999-44894999_strand_+_gene_Chr7-6Mb-seg-179', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTCCCAGGTCTGCATGGGCTACATGAGACCACACCTCAAAAATACACTAAACA...GTC'), id='7:44894999-44899999_strand_+_gene_Chr7-6Mb-seg-180', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATAGGAAGACCGCAAGGCTAGGCGGGAAGAAGCTGATCTGGGAGTTACTGTGGA...GAA'), id='7:44899999-44904999_strand_+_gene_Chr7-6Mb-seg-181', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGGGAGGGCACTGGGCAGGGGAGTTATCTTTGATTACTTCCTAGACAGCCATG...GCA'), id='7:44904999-44909999_strand_+_gene_Chr7-6Mb-seg-182', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGCCAGGCAAAATGTGCAGGCTGGCTCGGCCCTGCATCGTGAGCACCCTCGTG...CAG'), id='7:44909999-44914999_strand_+_gene_Chr7-6Mb-seg-183', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGTTAAGAGCACTGACTGCTCTTCCAGAGGTCCTGAGTTCCCACATGGTGGCT...GCA'), id='7:44914999-44919999_strand_+_gene_Chr7-6Mb-seg-184', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAACTGCTCTTCCAGAAGACCTGGGTTCAATTTCCAGCACCCATGTGACAGCT...AGT'), id='7:44919999-44924999_strand_+_gene_Chr7-6Mb-seg-185', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCAGGATAGACAGGACTACACAGAGAAACCTCTGTATGTGTGTGTGAAGAGAG...AAT'), id='7:44924999-44929999_strand_+_gene_Chr7-6Mb-seg-186', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCCTGCTGCAGAGAGGGGCTGGGCGAGGCGGTAGACCGTGAGGTGTCCACTAC...ACC'), id='7:44929999-44934999_strand_+_gene_Chr7-6Mb-seg-187', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACATAATGTCTTAACTTTTCTGTTTGGCCTTCCAGTCCCACACATTTAACCCAT...GTG'), id='7:44934999-44939999_strand_+_gene_Chr7-6Mb-seg-188', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGGCACACACCTTCAATCCCAGCACTCGGGAGGCAGAGGCAGGCGGGTTTCTG...TAT'), id='7:44939999-44944999_strand_+_gene_Chr7-6Mb-seg-189', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAGTTTACAAGTCCAATGGGCCTCTCTTTCCAGTGATGGCTGACTAGGCCATC...ATG'), id='7:44944999-44949999_strand_+_gene_Chr7-6Mb-seg-190', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGTGTCCACACCATACGGCCCATCCACGTCGCCCACCCACCTGTCATGGCCAC...GCT'), id='7:44949999-44954999_strand_+_gene_Chr7-6Mb-seg-191', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGAGAGACTGTCTTAAAAAATAAGGTAGAAGGCCGGGCATGATGGCAAACGCC...CCG'), id='7:44954999-44959999_strand_+_gene_Chr7-6Mb-seg-192', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCCCCACCCTCCAGCTGCCCAGAGGCTGGACCCAGTCCAGTTCACTCACAGGC...TGC'), id='7:44959999-44964999_strand_+_gene_Chr7-6Mb-seg-193', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGGCATGAGGTCTGGCTGCCCCCTGTGGAGAAGCCCTAGGTCTCCATAAGTG...GTT'), id='7:44964999-44969999_strand_+_gene_Chr7-6Mb-seg-194', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTTAAGAAGCTCAACTCAAAGTGCAGCCCCAGGTCTAGCTCCCAAACTAGCTA...TCC'), id='7:44969999-44974999_strand_+_gene_Chr7-6Mb-seg-195', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCCCTGCCCTCTTTTAGGTACTACATTATGTCATGTATGTCACCGTATATCTA...CAC'), id='7:44974999-44979999_strand_+_gene_Chr7-6Mb-seg-196', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCCAGGGTAGAATGAGATCTATCCTCTGACCTCTACATGTGCCCCACCCAATA...AGC'), id='7:44979999-44984999_strand_+_gene_Chr7-6Mb-seg-197', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGGGAGGTGAGATTCCCACGTGGGCCCCACTTTTTCAATAGTACAGACGGACA...ACA'), id='7:44984999-44989999_strand_+_gene_Chr7-6Mb-seg-198', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAGTTTAAAAAGAAAAAATAAACAGTCCAGCTGGATCCTGCCACAAGCCAATT...TGC'), id='7:44989999-44994999_strand_+_gene_Chr7-6Mb-seg-199', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGATGATGCCCTCAGAGGCCATAAGGAGGCCCCTTTCCCTAATCCCTAATGTG...CGG'), id='7:44994999-44999999_strand_+_gene_Chr7-6Mb-seg-200', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTAGCTGCTGACAATAGCAAGGACCCTTATGACCCTCATAAAGTGTATGAGTTT...AGG'), id='7:44999999-45004999_strand_+_gene_Chr7-6Mb-seg-201', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGGAGCAACTGAGGTAATTAAAAGAGGCATTTATACATACATGTATACATAT...AGG'), id='7:45004999-45009999_strand_+_gene_Chr7-6Mb-seg-202', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTCACACCCTTCAGGGAGCTCACTGGTCCTGAGGACAGGGTTCACACCCAGAG...AAA'), id='7:45009999-45014999_strand_+_gene_Chr7-6Mb-seg-203', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACTTTCCAAGTTTCAAATGGAACCATGAGTTTGCAGGACTGGAGTAATATTAG...CAG'), id='7:45014999-45019999_strand_+_gene_Chr7-6Mb-seg-204', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAGAGGGGGGACAGAGACTTGGAGAAGATATGGGACAGGACTCACACACACAC...TTT'), id='7:45019999-45024999_strand_+_gene_Chr7-6Mb-seg-205', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAACTCCAGCCCTCAGCCAACACACCAGGGTGGAACTTACTGTGAGTGTCCTGT...AAA'), id='7:45024999-45029999_strand_+_gene_Chr7-6Mb-seg-206', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACCTAAAAAACATAGTAACCAGAATCCTTAGAACCGGACAAGTCTATACTTTC...TGA'), id='7:45029999-45034999_strand_+_gene_Chr7-6Mb-seg-207', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAGGAAGAGACCAGGGAGACCTACAAGGTTACCCTCACCCCTGCCCTGCTGGA...CAC'), id='7:45034999-45039999_strand_+_gene_Chr7-6Mb-seg-208', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTGAAGGGCTGGAATGTGACTCAAGGGTAAAGACCCTGCTGGGAACTCCCCAG...GGC'), id='7:45039999-45044999_strand_+_gene_Chr7-6Mb-seg-209', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGGTGGCGCACGCCTTTAATCCCAGCACTTGGGAGGCAGAGCCAGGCAGATTT...GCA'), id='7:45044999-45049999_strand_+_gene_Chr7-6Mb-seg-210', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGGGAAGGGGCGGCTGCCGAAGGAAGCCGGGTGCTGGTAGGCAGACAGGGCAG...CCA'), id='7:45049999-45054999_strand_+_gene_Chr7-6Mb-seg-211', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGCTACACAGAGAAACCCTGTCTCATAAAAAAACAAAACAAAAACACACAAAA...TTC'), id='7:45054999-45059999_strand_+_gene_Chr7-6Mb-seg-212', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAGAAGCGCTCCTAGGGTTAGAGAAGAGAAGGAAAAGATAGCAACTGAACACG...ACA'), id='7:45059999-45064999_strand_+_gene_Chr7-6Mb-seg-213', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTGCTCTCAGCCAGGGCCTGCTTTGTTTGTTTATATATTAATAGTAAGTTACT...CAT'), id='7:45064999-45069999_strand_+_gene_Chr7-6Mb-seg-214', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAGGCAAAACTCCCACGCACATCAAAATAAATCTTTTTTTTTAAAGATTTATT...CTG'), id='7:45069999-45074999_strand_+_gene_Chr7-6Mb-seg-215', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGCCCTCGTACTCACTGCTGAGTCCACACTGGACCGCAGGCGCGCAGCCCACG...TTC'), id='7:45074999-45079999_strand_+_gene_Chr7-6Mb-seg-216', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAGCCGGTAGTCCCAGAACTCGCCAGCAGAGGGCACGCGCGCCATTCCATGCC...TAC'), id='7:45079999-45084999_strand_+_gene_Chr7-6Mb-seg-217', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTACAGCTGAGCACTGCCCACGGGGAGGAGAGGCGGGGCCTTGGGATGTTACTG...CAG'), id='7:45084999-45089999_strand_+_gene_Chr7-6Mb-seg-218', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAACCACATGGTGGCTCACAACCATCAGTAATGAGATCTGACGCCCTTTTCTGG...AAC'), id='7:45089999-45094999_strand_+_gene_Chr7-6Mb-seg-219', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACATGGTGGCTCACAACTGTCTGTAATAGGATCTGATGCTCTCTTCTGGTGTG...GGC'), id='7:45094999-45099999_strand_+_gene_Chr7-6Mb-seg-220', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTGAGCATGTGGACAGATGCTCACCCCATACACAGCACAGGTATATATCACA...TCC'), id='7:45099999-45104999_strand_+_gene_Chr7-6Mb-seg-221', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTCCCTCTGGTCTATATCAAGTGTGTGTACAAGTGTGTGCGTGTGTAATAAT...AAC'), id='7:45104999-45109999_strand_+_gene_Chr7-6Mb-seg-222', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCAACCAGCAACCAACCCACAACCCACTTCTCTTGGGGCCTTAGCACTTATAT...TAC'), id='7:45109999-45114999_strand_+_gene_Chr7-6Mb-seg-223', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTGCCAGGCTTTTGCACAGGTGCTGGGGATCTTAACTGCTCACGTTTGCACAG...AGA'), id='7:45114999-45119999_strand_+_gene_Chr7-6Mb-seg-224', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGGAGACTCAAGGTCTATGGCTGAGTGAGCAGTGAGGCTGGTCAGTGCTGAG...ATT'), id='7:45119999-45124999_strand_+_gene_Chr7-6Mb-seg-225', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGAGTGGTGCTGGATCTTTTTAACCCCAGCACTCACGGAGGCATAAGGACTTG...GTA'), id='7:45124999-45129999_strand_+_gene_Chr7-6Mb-seg-226', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAGGGGGTGATCAATGGTCCCTGAGGACTGGAGTTCTCACGTGACACTCTACC...AAA'), id='7:45129999-45134999_strand_+_gene_Chr7-6Mb-seg-227', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGGTCTAGATTTTCAGGTCTGAGGGTGCAGGAGGTAGCCCACTAGTCTTGTC...ACC'), id='7:45134999-45139999_strand_+_gene_Chr7-6Mb-seg-228', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCAGGGTCACTGGGAGGAGGCTTTGTTGTCTCTAAAAAGCTCTCATGCCATTG...AAC'), id='7:45139999-45144999_strand_+_gene_Chr7-6Mb-seg-229', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCCTCCGACAAACAGCCCATAGGGAGGGGCTGGGCTGCAGAGTGGGGTGGGAG...TGA'), id='7:45144999-45149999_strand_+_gene_Chr7-6Mb-seg-230', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACTCTTTACGTTAGCTAGTACTGCTGCTGCAGCCTATATATATATAGTATAGT...CTT'), id='7:45149999-45154999_strand_+_gene_Chr7-6Mb-seg-231', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTAGACCAGGCTGGCCTCGAACTCAGAAATCCGCCTGCCTCTGCTTCCGGAGT...GTT'), id='7:45154999-45159999_strand_+_gene_Chr7-6Mb-seg-232', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCAGATGTGGAGAAGATGTTGGTGTCTTCTCTAAGTGAAATAAACAGGTTTC...CAG'), id='7:45159999-45164999_strand_+_gene_Chr7-6Mb-seg-233', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTTGAGAGGCCTAAAAATATCTATCCACAGTGCGGGGAGGAGATGGCCTCGT...GTG'), id='7:45164999-45169999_strand_+_gene_Chr7-6Mb-seg-234', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCTTGGGTTTGGCTCTGGGAAATTGGGAGGGTCCTTGTTTGAGGGCATATCTT...AGC'), id='7:45169999-45174999_strand_+_gene_Chr7-6Mb-seg-235', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAGTGCAGCCTCCGAGGCCCCCGCCCCCTGTCCGGGACTACTGACCACGGGCC...ACT'), id='7:45174999-45179999_strand_+_gene_Chr7-6Mb-seg-236', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGAGACAGAGGCCAGTGGATCTGGGTTCTAGGCCAGCCTGGCCTACAGAGCA...GGG'), id='7:45179999-45184999_strand_+_gene_Chr7-6Mb-seg-237', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTCTGTGGTGCAATTGGGGTGGAATCTGTGAGATCCCAGGCTCAGAGTGGAAA...CTC'), id='7:45184999-45189999_strand_+_gene_Chr7-6Mb-seg-238', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCTCTCTCAGCTAGCAGCTTCTCATTCTGCAGGGAGGACAGCCATGCTCACC...CAA'), id='7:45189999-45194999_strand_+_gene_Chr7-6Mb-seg-239', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAAACAACCCTCTGAAACCAAACCAACCAACAAACAAAAAAAGAATGCCAAGA...AAG'), id='7:45194999-45199999_strand_+_gene_Chr7-6Mb-seg-240', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCAGTAGGAGGGATAGAGGTGACAGGGTACCTGTTTTACCCAAAGTAAGAGGT...ACT'), id='7:45199999-45204999_strand_+_gene_Chr7-6Mb-seg-241', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTATAGTTTCTACTTACTGCTTAGCTATGTAGCCCAGGATGGCTTCGGACTTG...TGG'), id='7:45204999-45209999_strand_+_gene_Chr7-6Mb-seg-242', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTAAAATACTTCATTACAGCAGCAGAGAGGAAACTGGAACAGTATGGCACAGG...AAA'), id='7:45209999-45214999_strand_+_gene_Chr7-6Mb-seg-243', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATACAGATAAAATGTAAACGTTAAAAAACAGAGTAAAATAAAAACACAAAATG...GGA'), id='7:45214999-45219999_strand_+_gene_Chr7-6Mb-seg-244', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACGCTGTGTCCAGGCTTTTCTTGACTTAGAAAACCATAAGCGTCTTATTGTGG...TGT'), id='7:45219999-45224999_strand_+_gene_Chr7-6Mb-seg-245', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGGGCCCATTGCAGATGGGAGTTGGTCATGGGCCAAAGCTTTAGTCGGCCAAAT...TAC'), id='7:45224999-45229999_strand_+_gene_Chr7-6Mb-seg-246', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCATGGGTAGTCACCACTTGATTTCACGGCCAGTCTGGGGAGGGTTGAGGGAGG...CTT'), id='7:45229999-45234999_strand_+_gene_Chr7-6Mb-seg-247', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGATCTGCTGGCATACTATGCCCCTGAGTAGGTCACGCCTACATATAAGGGAC...TTT'), id='7:45234999-45239999_strand_+_gene_Chr7-6Mb-seg-248', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACACACAGTGTGGTCACATTTAGAAGGGCACAAAGCGATCTGGTGATCCCCTTC...ATG'), id='7:45239999-45244999_strand_+_gene_Chr7-6Mb-seg-249', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTAAGGATGTTGAACATTTATTTGTTTCTCAGCTATTTGAAATTCCTCTACTG...ACG'), id='7:45244999-45249999_strand_+_gene_Chr7-6Mb-seg-250', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAGAGACACATCTTCACATAGTGTACAAAAACATCACTCCAACACAGTCTGTA...AGC'), id='7:45249999-45254999_strand_+_gene_Chr7-6Mb-seg-251', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTAGAGACTGGAGCTAGGCTCGGTTTTTGTTGGGGGGCAACTTGGAACCTAAGC...GTG'), id='7:45254999-45259999_strand_+_gene_Chr7-6Mb-seg-252', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGTACAGCATGGTTATGGTGAGGAGCCCCAGGCTTCACAAACCTGCCTGCCCC...GCC'), id='7:45259999-45264999_strand_+_gene_Chr7-6Mb-seg-253', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTGGCCACCCACACTGCAGTGGTTGGAGCTAGCCCAGGGCCTTGTGTGTGAAA...TAG'), id='7:45264999-45269999_strand_+_gene_Chr7-6Mb-seg-254', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATTAGGAGTTTTGAGGACAGAAGGTGGGGCCTTGTAAACCATGATATGACCTA...TTC'), id='7:45269999-45274999_strand_+_gene_Chr7-6Mb-seg-255', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTACCCTGTGAATTCTTTTCTGGTAGCTTTGTCTTCATTTTTTTTTCTTTTTAC...CAG'), id='7:45274999-45279999_strand_+_gene_Chr7-6Mb-seg-256', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAGGTCCGCGTTACCAAGATCCCAAACCACTTCCAGGCTAAGAAGACGGAGAA...TCC'), id='7:45279999-45284999_strand_+_gene_Chr7-6Mb-seg-257', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAAAGTTGTATGTCGATGAGGGTGTCTGAGCACCGGTGACAGTTAGACAAGGT...GCA'), id='7:45284999-45289999_strand_+_gene_Chr7-6Mb-seg-258', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGGATTGGAGTTTGGTTCTCCAACACACACCCTGACAGCTGATGTGTCCTAAT...GGT'), id='7:45289999-45294999_strand_+_gene_Chr7-6Mb-seg-259', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCTCACAACCACCTGTAATGAGATCTGGCTCCCTCTTCTGGTGCATCTGAAGA...AGG'), id='7:45294999-45299999_strand_+_gene_Chr7-6Mb-seg-260', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGTCTGTGAATTGAGGAACCAGCAATGTGGGGGGTGAGGAGGTGGCTAAGATT...GAG'), id='7:45299999-45304999_strand_+_gene_Chr7-6Mb-seg-261', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTGTGCCAACAGGAAGTTCTCTTTGTGCACAGACTCCCAAGTCAGCAGCTTCC...ACA'), id='7:45304999-45309999_strand_+_gene_Chr7-6Mb-seg-262', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAACACTAAATAAAGGCTGGGACAAGCAGAAATATTAGTTTGCTAAGTCCACTT...TGG'), id='7:45309999-45314999_strand_+_gene_Chr7-6Mb-seg-263', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTAGATGAGAGGCGGGCAGAAGAAGGCGAGAAGCAGGGCCCAGATGGGAGTTG...AGT'), id='7:45314999-45319999_strand_+_gene_Chr7-6Mb-seg-264', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCACTGGCCGTAGCACATCTCAGAGCATGTGTCACTTCAGTCTCTTCATCTGC...GGT'), id='7:45319999-45324999_strand_+_gene_Chr7-6Mb-seg-265', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCCTACATTTCCTTCGGGCCCCATGCTGGAAAGCTGGTCGCAATTGTAGATGT...TCA'), id='7:45324999-45329999_strand_+_gene_Chr7-6Mb-seg-266', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAGCTCTGGAATTACAGGTGTGCACTACCAGCCCTGGCTTATTTTGGTGCTGA...GAC'), id='7:45329999-45334999_strand_+_gene_Chr7-6Mb-seg-267', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTCTAAACGTTCAGCCTTTGTGCTAAAAATGCCAAACTTGAGCTTGTCACGTC...GGT'), id='7:45334999-45339999_strand_+_gene_Chr7-6Mb-seg-268', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCCTGCTAGACAATGCTCTGGAAAGGAGGGAAAACGGCCCAGCCTTCGGAGGG...GGC'), id='7:45339999-45344999_strand_+_gene_Chr7-6Mb-seg-269', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTACGGATCATGAAAATCTGTATAGAGTGAGACCATTCGGCTCCTGATGAGTGC...TAG'), id='7:45344999-45349999_strand_+_gene_Chr7-6Mb-seg-270', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTATCTGTGGGAAGTTCTGTTTCTGACCTCTCCCACCTACACTGGGCTGGGGTG...GCT'), id='7:45349999-45354999_strand_+_gene_Chr7-6Mb-seg-271', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTGGAATGGCCAAGCAAAATGGTGCCTTCAGTCACAGAGCAGAGTTTGTGGGT...TCC'), id='7:45354999-45359999_strand_+_gene_Chr7-6Mb-seg-272', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACTTAGTTCAGTTGAAGTGGACCATAGGTCTTCTCTTTGTCATCCTCTTACTC...CAA'), id='7:45359999-45364999_strand_+_gene_Chr7-6Mb-seg-273', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATCCTGCCACTGGGGATGGGAAAGGAGCAAGTAAAAAAAAAGACATGCTTAGA...GGG'), id='7:45364999-45369999_strand_+_gene_Chr7-6Mb-seg-274', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACACGGCAACGTCAGAGAGCTGCCAGTCCACAGCCCTGGACACCGGTGTCCTC...ATG'), id='7:45369999-45374999_strand_+_gene_Chr7-6Mb-seg-275', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATAAGTGGATTGGAGAGATGACTCGGCTGTTAAAAGAGCACTGACTGCTCTTGC...AGT'), id='7:45374999-45379999_strand_+_gene_Chr7-6Mb-seg-276', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGCTACTCACACAAAAGACCTAAACCCAGCCAGGGCCTAATTTGAGGCATCA...CGA'), id='7:45379999-45384999_strand_+_gene_Chr7-6Mb-seg-277', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGATTACAGCTATGAGCCACCATATTTAAAGTTAGCATGTTTTCTTTTATTTCT...AAA'), id='7:45384999-45389999_strand_+_gene_Chr7-6Mb-seg-278', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAATAAATGGTATTGGGATGGCTAATCTTAGTTGACAACTTTCATTAATCTGGA...AAC'), id='7:45389999-45394999_strand_+_gene_Chr7-6Mb-seg-279', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTAAGGCTGGTAACAGAAAACAAAGGGGAAAAGGAGTAGGAGGTGTGGGACCCA...TCT'), id='7:45394999-45399999_strand_+_gene_Chr7-6Mb-seg-280', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCAGAGGACCTGAGTTCAATTCCTAGCACCCACGTGGCAGCTCACAACTGTCT...GAA'), id='7:45399999-45404999_strand_+_gene_Chr7-6Mb-seg-281', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTTATGGGCAGACCAGGCTGGCCTCAAATCCACAGGTTGGCTTGCCTCTGCTT...CTC'), id='7:45404999-45409999_strand_+_gene_Chr7-6Mb-seg-282', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATGGGTAGCACTGTAAGAGACTTGGCCATAGATTGTGAGTTTCCTAGGTGTTC...GCT'), id='7:45409999-45414999_strand_+_gene_Chr7-6Mb-seg-283', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTCCAGGTTATAGGAACACGAGGGCCAAGTGGCACTGCACGGATTTCAGATCT...GAA'), id='7:45414999-45419999_strand_+_gene_Chr7-6Mb-seg-284', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTGCTGCACTTGGACCTCCTCACCTGCTGCCCCGCCCACAGGAACGTGTCTG...CCC'), id='7:45419999-45424999_strand_+_gene_Chr7-6Mb-seg-285', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCTTGGAGATCTTGCCGGTGGCCTTGTCAATGGTGATGACGTCTCTGAAGGAA...TGG'), id='7:45424999-45429999_strand_+_gene_Chr7-6Mb-seg-286', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTAGGCAGCACTGGCTGGTCTAAGCAGCACCGTGTGGCTGCTGAGCTCAGGCT...AGC'), id='7:45429999-45434999_strand_+_gene_Chr7-6Mb-seg-287', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGATCTCTCTGGTTTCCAGTTTTGGTGCTCGAAGTCCCCTGCCCGCAGTAGCCA...CGT'), id='7:45434999-45439999_strand_+_gene_Chr7-6Mb-seg-288', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCCCAGATCACCGCAATCGAGGCTCAACACCTCCTTAAGAGAAAACCAGGTAG...ATA'), id='7:45439999-45444999_strand_+_gene_Chr7-6Mb-seg-289', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTAAAACACACCCACCACATCACAAAGGGTAAGATGGTTTATTCCGGAACCAAG...GTC'), id='7:45444999-45449999_strand_+_gene_Chr7-6Mb-seg-290', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCTTTGGCTACATAGTGGGCTCAGGGCCATCCTGAGCTACATAAGATTCTGTC...ATG'), id='7:45449999-45454999_strand_+_gene_Chr7-6Mb-seg-291', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCACCCATTCATTTTGGTTTGTGCCTGCTCTGGGTATGCAGACCCAGGGGTACC...CCC'), id='7:45454999-45459999_strand_+_gene_Chr7-6Mb-seg-292', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTCCCATGAGATGAAGTCACGGGACTGGAGCGCGAGGCCTTGTGAAATGGAGG...CAA'), id='7:45459999-45464999_strand_+_gene_Chr7-6Mb-seg-293', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAAGGGCTAGCTTGCGCTGTTTGGAGCAGAGTTCAGGTGCTGGAGATAGCCCA...CCA'), id='7:45464999-45469999_strand_+_gene_Chr7-6Mb-seg-294', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTAAGCCTTCTCCCTATCCTTGTCTTCTCTGTGGTGTGTGTACACGTGTAGTG...TGT'), id='7:45469999-45474999_strand_+_gene_Chr7-6Mb-seg-295', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGTGTGTTTCCCCCTAAAGTAAAGAGAGAACCTGTTTGTCCTAGAAAGAAGCT...AAA'), id='7:45474999-45479999_strand_+_gene_Chr7-6Mb-seg-296', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAAAATAAATCTTAAAAGAAACAAAACAATTAACTGGCAGGGTGGTGGTGCAC...TGT'), id='7:45479999-45484999_strand_+_gene_Chr7-6Mb-seg-297', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGCCACCATGTGGTTGCTGGGATTTGAACTCAGGACCTTTGGAAGAGCAGTCA...TCC'), id='7:45484999-45489999_strand_+_gene_Chr7-6Mb-seg-298', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCTTCACGTAAGCCTTTGCCAGGAAAGACCGTTCCCTGTCACAGGCGCCACA...GAA'), id='7:45489999-45494999_strand_+_gene_Chr7-6Mb-seg-299', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAAAGGCTGGACCCGGGAGACACTAAGGTCTGAGGGTTTCCCGTGTCCTCACC...GCC'), id='7:45494999-45499999_strand_+_gene_Chr7-6Mb-seg-300', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAGAGGCCACGCAGGCAAAAAGGATGGCACATGGGCCTGGAGGATGTCTGAGG...TGG'), id='7:45499999-45504999_strand_+_gene_Chr7-6Mb-seg-301', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACTGGAGTTATAGGTGGTTGTGAGCTGCACTGTGGGTGCTGGGAACCAAGCCT...TGG'), id='7:45504999-45509999_strand_+_gene_Chr7-6Mb-seg-302', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTAAGGATGTCCCAACCCCGTATCCGGTTGGGGAAGCGAGTGTGCACGCCACCC...TGT'), id='7:45509999-45514999_strand_+_gene_Chr7-6Mb-seg-303', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGTGGAGACAACTTGCAGGGGCAGGTTCTTTCCTGTGACAACATTAGGACTCG...TTC'), id='7:45514999-45519999_strand_+_gene_Chr7-6Mb-seg-304', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACAGTTGGGGAGGAAAAAGATGAAAGTTTTGGATGCCATGAATACAGACTAGC...CGT'), id='7:45519999-45524999_strand_+_gene_Chr7-6Mb-seg-305', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCATTTCCTTCTCTCTAAGGTCATCTTGTCCTGGCTTTGCATTGTACTCATCA...GGA'), id='7:45524999-45529999_strand_+_gene_Chr7-6Mb-seg-306', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGACAGAGAGAGGGAGAGAAAGAACTCACAAGTGAGACAGAGAAGAAAGCAGA...TTC'), id='7:45529999-45534999_strand_+_gene_Chr7-6Mb-seg-307', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTTAACCACTGAACCACCTCTCCAGCCCTCAAAGGGGTTCTCTTTTAAGATCT...GTG'), id='7:45534999-45539999_strand_+_gene_Chr7-6Mb-seg-308', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGGGTCCTGAACCCTGGGCTTCAGAAAGACCAATAGATGCCTTTCTTCCCCCA...TGT'), id='7:45539999-45544999_strand_+_gene_Chr7-6Mb-seg-309', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTGGTTTTTTGAGACAGGGTTTCTCTGTGTAGCCATGGCTGTCCTGGAACTCA...GAA'), id='7:45544999-45549999_strand_+_gene_Chr7-6Mb-seg-310', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCTCTGCTTCTAGATTGACTCTCAAGTTCTTCCTTTCTTCCTCCCTCCTTTCC...TAT'), id='7:45549999-45554999_strand_+_gene_Chr7-6Mb-seg-311', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTGGGAAGGTGGTGGTGGTGACTGGAGGCAGCCGGGGCATCGGGGCTGCGATC...CAT'), id='7:45554999-45559999_strand_+_gene_Chr7-6Mb-seg-312', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGGGAAGCTACAGCCTCCTCACTATCTCCCAGAGCTTTCTAGGTTTGCTTTTG...CTG'), id='7:45559999-45564999_strand_+_gene_Chr7-6Mb-seg-313', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCAGGAAAACAGGTTCAGCCTCTCAGCAGGTAGTGCCATCTTGGAGGAAGAAG...CAT'), id='7:45564999-45569999_strand_+_gene_Chr7-6Mb-seg-314', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATACAAAATTTAGAGGTCTGGATCTGTTCCCAGGGGTTAGAGCGCTTCCCAAAC...CGG'), id='7:45569999-45574999_strand_+_gene_Chr7-6Mb-seg-315', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCCAGGGTCTGAGGGAGGAGATCTGAGGTCTGGGCTGATGAATTCTAGGAGA...ACT'), id='7:45574999-45579999_strand_+_gene_Chr7-6Mb-seg-316', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAATCCCTTGTCAACTTGACACAAATAATATCTCATGTTCACATGAAACAATA...GCT'), id='7:45579999-45584999_strand_+_gene_Chr7-6Mb-seg-317', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGTTCTGCTAGGCGCCTCTGCCTGCCAGTAAGTGGCCAGTCTGTCCCAGCAGA...CCT'), id='7:45584999-45589999_strand_+_gene_Chr7-6Mb-seg-318', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACTTCCTGGCCTGGTCTTCTGCCTGTCTCTATGCTAGCTTCCTCTTTGGCCTT...GAG'), id='7:45589999-45594999_strand_+_gene_Chr7-6Mb-seg-319', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATTAAAAACAGACCAAAAATTGTATGTTAAAATACACAGCAGTTAATAGATAA...TTG'), id='7:45594999-45599999_strand_+_gene_Chr7-6Mb-seg-320', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAAGTTTTTGTTTTTTTAAAGAATTATTTATTCATTTTATGTATCTGAGTACAC...AGA'), id='7:45599999-45604999_strand_+_gene_Chr7-6Mb-seg-321', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTTGTTTCAAAACAGAAGTAAAATGTAGCTGGGGATGTGGCTCAACGCCACTC...GAC'), id='7:45604999-45609999_strand_+_gene_Chr7-6Mb-seg-322', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAGAGGGCCCTGTCCCCAGACCCTGCAGACCTCCAACTGTTTCAAGGACCCAA...GTC'), id='7:45609999-45614999_strand_+_gene_Chr7-6Mb-seg-323', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTCCACAGTGGTGAGAACCCAGGAGCCTCCTCCTCTCAACCTCCATGAGGCCC...CTG'), id='7:45614999-45619999_strand_+_gene_Chr7-6Mb-seg-324', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTTAGGGAGGCTGAAGAAAATCCAGACTTACCAACTGGTCTCCTGGATTTCTG...CTA'), id='7:45619999-45624999_strand_+_gene_Chr7-6Mb-seg-325', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATTGTGGAGGTGAGGATGCCACCCAGCATAAGAGCCCAGTGTGAGAGGGGGGA...ATT'), id='7:45624999-45629999_strand_+_gene_Chr7-6Mb-seg-326', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCTGGGCCAGTCGCTGACACACAGTGATTGCTCCTAAATGCATGTTAAAAGAA...GAC'), id='7:45629999-45634999_strand_+_gene_Chr7-6Mb-seg-327', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCTGAGGCTGTGTCTGTGGAGCTGCGACCACTCATACTATGGATGGCCAACAC...AGG'), id='7:45634999-45639999_strand_+_gene_Chr7-6Mb-seg-328', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACAGCTGAGTAGCTGAGAGGAGCCTACCTGGTGAGAAGCCCCTGGGGTAGATA...TTG'), id='7:45639999-45644999_strand_+_gene_Chr7-6Mb-seg-329', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTCTAAGGAGAGGTCATCTGCATCCACAGGGACCCTCCAACTACAGAGTACTG...TAA'), id='7:45644999-45649999_strand_+_gene_Chr7-6Mb-seg-330', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACACCTTTTCTTGCTGCCTCTGGAAGAGACTGTCTGTGGGTGGCAGCCCGGCT...TAC'), id='7:45649999-45654999_strand_+_gene_Chr7-6Mb-seg-331', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCAGCTCGGGAGTGCTGAGATCATTGGCTGGTGTCACCACACCAGTCTGAATC...GAG'), id='7:45654999-45659999_strand_+_gene_Chr7-6Mb-seg-332', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAACTACTAATCTGATTCTGGTGAGGCAATGAATCTACTGCTGCTACTATCATA...GGA'), id='7:45659999-45664999_strand_+_gene_Chr7-6Mb-seg-333', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGGAGCCAGATTCACATCACTTGTGATCCAAAGGCTAGGTAGGCACTTGGGTG...GTT'), id='7:45664999-45669999_strand_+_gene_Chr7-6Mb-seg-334', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAATAAGGGCTGGAAAACGTAACTCTTAATTACTTTATAATCCTCTGGCAGCT...CAT'), id='7:45669999-45674999_strand_+_gene_Chr7-6Mb-seg-335', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAAGATGCTGTTGTCTGATTCTACTGGAGCAGAGATCCAAGCCAGGCAAGTCC...TTA'), id='7:45674999-45679999_strand_+_gene_Chr7-6Mb-seg-336', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGCTAGCCCTGCCCCCCTGCCCCCCACCCACCGCAGCTCTGGTGAGTCACCCT...AGG'), id='7:45679999-45684999_strand_+_gene_Chr7-6Mb-seg-337', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTGGATTTCTGGTTTCTAAGTGCCTTGGCATCGGGCACACACCTTATGCCAC...ACG'), id='7:45684999-45689999_strand_+_gene_Chr7-6Mb-seg-338', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCACTGGGGGAGCCAGGAATAGTTACAACACGGGTTGTCTCTTCACAGGAAGA...AGC'), id='7:45689999-45694999_strand_+_gene_Chr7-6Mb-seg-339', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAAACCTATTAATCGACATTAAGGCTTTGGAACAGGCCTGGCCCAGAGCAAGC...GGT'), id='7:45694999-45699999_strand_+_gene_Chr7-6Mb-seg-340', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGAGCGAAGAAGGGGTGTGCATTGGCACCGAGAGGGACCCGGGGCCGGCGAG...GCT'), id='7:45699999-45704999_strand_+_gene_Chr7-6Mb-seg-341', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGCGTCTCAAGGGCTCGAGACGTGATTGGTTCTTGCCAGAAGCCGGAGGAAGG...GCA'), id='7:45704999-45709999_strand_+_gene_Chr7-6Mb-seg-342', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACAAATACATTTATATTTTTAAGAAAAAGCTAGCCTTCCCCCACCTCCCTTGT...CCG'), id='7:45709999-45714999_strand_+_gene_Chr7-6Mb-seg-343', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTCTCCCCAGCCGGTACAACCACTCCAGGGCCGCAACGCAGGACCAGATCTGAG...TCT'), id='7:45714999-45719999_strand_+_gene_Chr7-6Mb-seg-344', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAAGTTCCCAAGCTGAAGGTAAGTGACCAGCATGGGCCAAAGGGACATGTCTC...ACA'), id='7:45719999-45724999_strand_+_gene_Chr7-6Mb-seg-345', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTGAGCCATGAGAGCCACAGGCCATGCCCCACATGAAGGCAAGCCAGGGGCTT...AGT'), id='7:45724999-45729999_strand_+_gene_Chr7-6Mb-seg-346', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTATTGTGAGGATCCTGGGTTGGGGTCATCAGAAGCTGGGCTGGGGCTGGGCTC...CTG'), id='7:45729999-45734999_strand_+_gene_Chr7-6Mb-seg-347', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTTGGTTTTTCAAGTACTGGAGTGACAGTCATACATGACCATGAGCATAGAGT...AAG'), id='7:45734999-45739999_strand_+_gene_Chr7-6Mb-seg-348', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGAATCTGGAGACAGGAGCAAGGAAGTCCCGCCCCGCAATCCATGGCTGCGTG...TGG'), id='7:45739999-45744999_strand_+_gene_Chr7-6Mb-seg-349', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAACTGAGTCACCTTCCAAGGAAGGATCTTGTGCCCGTAGGCACCCTGAAATC...CAG'), id='7:45744999-45749999_strand_+_gene_Chr7-6Mb-seg-350', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGACCTTGAGAAAGAAGCCAGTGAAGGACACAAGCTCTACACTGCAGCATTGA...GGC'), id='7:45749999-45754999_strand_+_gene_Chr7-6Mb-seg-351', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCTTCTGCCTTCCCCCAACCCCTTGTCAGAACAGACCCTTCCATATGGGGCAG...ACT'), id='7:45754999-45759999_strand_+_gene_Chr7-6Mb-seg-352', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTTTAAAAGCTTTATTTATTACCTCCCTCACAAGGGCAGGTCTACCTCGTGTC...TGT'), id='7:45759999-45764999_strand_+_gene_Chr7-6Mb-seg-353', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATCCCCAAGTCATAGTGACAGAGACTCTGTCGTCAAACAAAGGTGGAGACTCC...GAC'), id='7:45764999-45769999_strand_+_gene_Chr7-6Mb-seg-354', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGGGGTTTTCAGAGATGGGGGCATCTACATTTCCAATAAAAACCGAGAGTGG...ACC'), id='7:45769999-45774999_strand_+_gene_Chr7-6Mb-seg-355', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACTAAGGTTTTTTTTTCCCTCCTGGTCACAGAAAATGAGTTTAGAAGACTCTG...GCG'), id='7:45774999-45779999_strand_+_gene_Chr7-6Mb-seg-356', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGGAGAAGGCAGATATGGGGCATAAAAATACAAAGAAGAGAGCTGGAGCGATG...TGA'), id='7:45779999-45784999_strand_+_gene_Chr7-6Mb-seg-357', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGTTCCTGTGACATGTCTCAGCTGTGTGTGGGCTGCGTGGGCTTGCTATCTGC...AGC'), id='7:45784999-45789999_strand_+_gene_Chr7-6Mb-seg-358', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAACCCTCCCGTAAACACATAGACTAGCAGAGCCCGTCCTCACCATGAACAGC...GGG'), id='7:45789999-45794999_strand_+_gene_Chr7-6Mb-seg-359', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCTGGGAGAAGACGCCCGAGACTGGGGGTCTAGCTCCCGAGACCCTGCTGGAT...CTT'), id='7:45794999-45799999_strand_+_gene_Chr7-6Mb-seg-360', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCAGACTCAAGGTGGGACTTTGAGAGTTTGAGTGGGACTGGGATAATTGTCCC...TCA'), id='7:45799999-45804999_strand_+_gene_Chr7-6Mb-seg-361', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAGGATTGGATTGCTGCCTGGCAAGCGGCGACATGATTGAGAGGCAGGTCTGCC...TGA'), id='7:45804999-45809999_strand_+_gene_Chr7-6Mb-seg-362', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCAGCCCCATTTTTCAGTTGTGGAAACTTGAGGTTCTCAAAGGTGCAAAAAC...AAA'), id='7:45809999-45814999_strand_+_gene_Chr7-6Mb-seg-363', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCTGCCTTCCCCAGGCATTCTCCCAAGACACCCCTTTTTCCTCAGAATCTTCT...CCC'), id='7:45814999-45819999_strand_+_gene_Chr7-6Mb-seg-364', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCTCTTCCTCATCCCCAGCTCGACTGTTTCCTGGCTCCAGCGCGCCGCTCAGG...CTG'), id='7:45819999-45824999_strand_+_gene_Chr7-6Mb-seg-365', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTAAGTGATAAAGGGGAGGAAGTGTGGGTTCTCAGCTCCCCTAGGATGATGCC...GCC'), id='7:45824999-45829999_strand_+_gene_Chr7-6Mb-seg-366', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCGTAGTGAGGAACCATGGCCAGTTCCAGCTGAGGTTTCTGCTCCTCCTCATC...CCA'), id='7:45829999-45834999_strand_+_gene_Chr7-6Mb-seg-367', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTTTTTGCAGAAGTGATAGTGCCTGGTGGGAAGGCGGTGCTGCAGAAATGACA...TGA'), id='7:45834999-45839999_strand_+_gene_Chr7-6Mb-seg-368', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACTGACTAGATTTCACTCTCACTAGGCCTTTCTCTGCCAGAGTAATCAATAAA...CAG'), id='7:45839999-45844999_strand_+_gene_Chr7-6Mb-seg-369', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCAGTGCCTAGCGCAAGAAGACCAACAGATATTTGGTGAGGGCAGAGGGATGA...AAC'), id='7:45844999-45849999_strand_+_gene_Chr7-6Mb-seg-370', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGATGAAGAGGCCAGGAGTCCCACCTTTAGTCCCAGCACCCAGGAGGCAGAGTC...AGC'), id='7:45849999-45854999_strand_+_gene_Chr7-6Mb-seg-371', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCAGGTCACAGCCCCCTGCTCCTTGATCCCGTCCCTCAAGCTCCTTGGGCCTG...TGC'), id='7:45854999-45859999_strand_+_gene_Chr7-6Mb-seg-372', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGGCACATAGATTCTCAGTAGCAGGTTTTTCAGTGGATGGATGTTTGGCCAGT...GGC'), id='7:45859999-45864999_strand_+_gene_Chr7-6Mb-seg-373', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCATCCTTCAGCCAACCCCACTTCTAGGTTTCACTTGCTCCACATCTAGTACA...ACA'), id='7:45864999-45869999_strand_+_gene_Chr7-6Mb-seg-374', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATGTTGCGCTGGAAGTTGGGCCCCGGACATTGCGGAGGCTGTCGGGGTGTCCT...GCA'), id='7:45869999-45874999_strand_+_gene_Chr7-6Mb-seg-375', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCTTCTGTAAGGTGCAGGATGGAGCCTAGGCCCTGTGCACCACTGAGCGCTCC...ACC'), id='7:45874999-45879999_strand_+_gene_Chr7-6Mb-seg-376', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATACATTTCAGATGTTTCCTTTCTAAGCTTGCTTGATCAAAATGCTCTTCAAGA...ACT'), id='7:45879999-45884999_strand_+_gene_Chr7-6Mb-seg-377', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTATTTGGCCCTCAGCCCTGTTTCATAAATGAGAAAACTGAGGCATGGAATAA...AAG'), id='7:45884999-45889999_strand_+_gene_Chr7-6Mb-seg-378', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTAACAGGTTGTGGACACCAAATCCAGGTTCTCTGCAAGAGTAGCCAGCGCTCT...TAG'), id='7:45889999-45894999_strand_+_gene_Chr7-6Mb-seg-379', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCTAGTCCCTGTTCACCCCTCCCCTACCTGAGAAATAGAAAATGGATGGGTCC...ACT'), id='7:45894999-45899999_strand_+_gene_Chr7-6Mb-seg-380', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTGCCTCCTCTGTACTGCGCCTGGAGGAAGCTAGTTATTCCAGTTGGACAGCA...TGT'), id='7:45899999-45904999_strand_+_gene_Chr7-6Mb-seg-381', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTCAACAGTCTGAAAAGTTTTGTTTTAGCCATCCCAGGCTGGTCTTGAACTTG...TTT'), id='7:45904999-45909999_strand_+_gene_Chr7-6Mb-seg-382', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTCCCTACGTTTCCTTCCTGGAGGGTATTATCCTCCCTTTCTGAGCAGCAGCGT...GCC'), id='7:45909999-45914999_strand_+_gene_Chr7-6Mb-seg-383', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACACAACCATGTTGGAAGGTCCCCTGATGTGTCAGTGCAGGCTGGATTTTGGCA...CAC'), id='7:45914999-45919999_strand_+_gene_Chr7-6Mb-seg-384', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGTTCAGGGACTCTTTGTCTGGGAGAGTCCACCAGGACTGGAAGGGACAGAGA...GCT'), id='7:45919999-45924999_strand_+_gene_Chr7-6Mb-seg-385', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGGAGGCGGAGATAGTTTGAGACCAACCAGGGCTACATAGTAAAACCCTACAT...TAG'), id='7:45924999-45929999_strand_+_gene_Chr7-6Mb-seg-386', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCTTTAAAAAAATTTTTTTTTAATCTTTATGTACACTGGTGTTTTCCCTTCAT...CTG'), id='7:45929999-45934999_strand_+_gene_Chr7-6Mb-seg-387', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGCCTCTGCCTCCTGAGTGCTGGGATTAAGTGTTTGCTGCTGCCACTTACTG...GGC'), id='7:45934999-45939999_strand_+_gene_Chr7-6Mb-seg-388', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCCTCGCTATCTAGGCTTTTTGGCTCTTGGCCCTTGGCTCAGTGCCTGCTTC...TCT'), id='7:45939999-45944999_strand_+_gene_Chr7-6Mb-seg-389', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAAAGACCCCAGCATTCCAAATGTCACAGATCACAGAAACTATCTGCAGCTGG...CCG'), id='7:45944999-45949999_strand_+_gene_Chr7-6Mb-seg-390', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCATGCTCTTGCCCCACGCTGACAGGACAACCTCAACCTGGAGAGGCAGTCC...AGA'), id='7:45949999-45954999_strand_+_gene_Chr7-6Mb-seg-391', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACACAAAGAGACAGAGAGACAGAGAGAGACAGGGAGAGAAACACAAAGAGACA...CAG'), id='7:45954999-45959999_strand_+_gene_Chr7-6Mb-seg-392', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTGCTTTCTTATAGAACCCAGGACCACCAGCCTAGGGTGGTGCATAAATCCAT...CTA'), id='7:45959999-45964999_strand_+_gene_Chr7-6Mb-seg-393', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATGTACATGTACACAGATATAAATTTACAACACGCTCAGATGTGCTGCAACAA...AGA'), id='7:45964999-45969999_strand_+_gene_Chr7-6Mb-seg-394', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCCACACTCGTGGGATTATTAGACAGGCATGAGGACATATTAGTACCCCTCT...CTG'), id='7:45969999-45974999_strand_+_gene_Chr7-6Mb-seg-395', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTCTTCTGAGTGACCTGCAGTTACAGCCATGCCTGGTACACAGTAGTACCTGC...GAG'), id='7:45974999-45979999_strand_+_gene_Chr7-6Mb-seg-396', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACTTCCCGGCCCCTGTTCTGCCCACGATGCCCACCTGCCAGGTAACGGAAGAC...CTA'), id='7:45979999-45984999_strand_+_gene_Chr7-6Mb-seg-397', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGAGTGCAGCTCTTGTGGGGAGTGGGTTCCAGATCTACTCTGATCCCAGGCTC...AGA'), id='7:45984999-45989999_strand_+_gene_Chr7-6Mb-seg-398', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAAGGCAGTATGGAAGGAGAGATGATACTGTGGATTTCAGATTTGGACTTTCC...TGT'), id='7:45989999-45994999_strand_+_gene_Chr7-6Mb-seg-399', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTCGGCTCCTCCTTTGGGGGGGTGTGTGGCCTGTGTGCAGGACTGCAGTGCCA...CTT'), id='7:45994999-45999999_strand_+_gene_Chr7-6Mb-seg-400', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTTCCTTCCTTTTTCTTTCTTTTTTCTTCAAGACAGGGTTTCTCTGTGTAGCC...CTG'), id='7:45999999-46004999_strand_+_gene_Chr7-6Mb-seg-401', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCCACTTCTGATGGTCTGGCTCTACTCTAGCCCTTTCCCTCCCCTCCTCTGCT...CAG'), id='7:46004999-46009999_strand_+_gene_Chr7-6Mb-seg-402', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTTGGCCTGCTCAGTGCTCCCAATTGTTTACCACATCCTCCCTCCTTCCCTGT...CCA'), id='7:46009999-46014999_strand_+_gene_Chr7-6Mb-seg-403', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTAGTACTTGAACAGTCCTGCCCTAGTCTCTTAGAGATGGAAATTGCATGGAGC...GTA'), id='7:46014999-46019999_strand_+_gene_Chr7-6Mb-seg-404', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAAATAATCATGTATCAGTTCAAAACAAAACAAAGTATGTGGGATTGGCGGTAC...TAG'), id='7:46019999-46024999_strand_+_gene_Chr7-6Mb-seg-405', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAACATGAATCAGTAGCGGTGATATTACGTAGCAGAGACAGCCAGGCCTCCGC...AAA'), id='7:46024999-46029999_strand_+_gene_Chr7-6Mb-seg-406', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTGGGAAGGTTTGGGGAGCGGCCCACTAGCACTATGCAAATAACAAAGTCCCC...GCA'), id='7:46029999-46034999_strand_+_gene_Chr7-6Mb-seg-407', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGGGCATCTCAGAAGCTGCCCTGGAAGGCTTTTCAGCTTGAGCTTTCTGCAGC...CAC'), id='7:46034999-46039999_strand_+_gene_Chr7-6Mb-seg-408', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTTTTTCTTATACATCTTTCCAGGGATGACATCACCCACAGGGGGTGGACCCT...CCC'), id='7:46039999-46044999_strand_+_gene_Chr7-6Mb-seg-409', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTCTCCAAACAAACAAACAGCCCAAAGAATATTTAAAAGGTTTTAAGTGTTTG...TCA'), id='7:46044999-46049999_strand_+_gene_Chr7-6Mb-seg-410', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGGATGCAGAAGTCAGGTCGGGGTTGCCAGGGAGACCCACAAGCCAGAGGCAG...CCC'), id='7:46049999-46054999_strand_+_gene_Chr7-6Mb-seg-411', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAAACTGAAAGAGCAAGGGTTTTCCAGAGTGTTAGGAGCTATAGCGGGGCACA...TCT'), id='7:46054999-46059999_strand_+_gene_Chr7-6Mb-seg-412', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAAGGCCAGGCTGCTGCCTCAGCTGCTACAAACCCCGAAGTACCCTTTTTATT...GTG'), id='7:46059999-46064999_strand_+_gene_Chr7-6Mb-seg-413', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTGCTGTGCCTTTTCTGTTATCACGCTCGCCTGCCTGTGCCTGCCTGCCTGCC...CCT'), id='7:46064999-46069999_strand_+_gene_Chr7-6Mb-seg-414', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACCCAAGACAACGGCATTCTGACTTTCTCAAACCTGGTAAGGCCCTGCCAGGG...GGT'), id='7:46069999-46074999_strand_+_gene_Chr7-6Mb-seg-415', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTCTATGTTCTGTGACATAGGTGTGCAATCTTAGCAGTAGGCACTGCCATCA...TGT'), id='7:46074999-46079999_strand_+_gene_Chr7-6Mb-seg-416', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTTGTATGTATGTGCACATGTGTGCTCGTGTGTATGCATTTGTATGTGTCTAT...GGC'), id='7:46079999-46084999_strand_+_gene_Chr7-6Mb-seg-417', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGAGGAAACCAGCTCCGGCTGGTTTCCGCAGGGAGGGGAGTCTTTGCCTTGCT...ATC'), id='7:46084999-46089999_strand_+_gene_Chr7-6Mb-seg-418', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTGTTTATCCATTGTCTGTCTGTCTCTATCTGTCTGCCGGCCTGTGTGTCTGT...GCC'), id='7:46089999-46094999_strand_+_gene_Chr7-6Mb-seg-419', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCGAGGAAGCCTTGGATACTTCCTTAAGGGAATAGAAGCTAAGGCATCACAGC...CCT'), id='7:46094999-46099999_strand_+_gene_Chr7-6Mb-seg-420', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGGTTACCCTAAGCTTGTACCAACCTCTAGGCTGGTATGCCCCTCAGCTGGGT...CTG'), id='7:46099999-46104999_strand_+_gene_Chr7-6Mb-seg-421', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCTGCAGTGCTGGGGATTGAAGCCATGGCCTCATTCCTTCTACACAACCCAA...CCC'), id='7:46104999-46109999_strand_+_gene_Chr7-6Mb-seg-422', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCAGGCCATGGCATCATCCCCTGCCCCCAAACACTTGCATTCTTTCCATCAC...AGA'), id='7:46109999-46114999_strand_+_gene_Chr7-6Mb-seg-423', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGCCACGAGGAACACAGGTGTGACGTAGGAGATGACAGTCAAGGCGGAGACAC...AAA'), id='7:46114999-46119999_strand_+_gene_Chr7-6Mb-seg-424', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGATCATACATAGCCAGGGGAAGCAGGCAGGACAAGGCCTTCTAGACGGTAAG...TAT'), id='7:46119999-46124999_strand_+_gene_Chr7-6Mb-seg-425', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCCACACATGAGATTCTAGATGGGGTTAGCCCAAAGAGCCTCACTGCCAGGCC...GCC'), id='7:46124999-46129999_strand_+_gene_Chr7-6Mb-seg-426', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTTTAAGGGCAGAGTCTCACTCATGTGGCTCAGGCTGACCTGGAACTCACTG...ACC'), id='7:46129999-46134999_strand_+_gene_Chr7-6Mb-seg-427', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATGTTGATTAGGAGGAAATATAGCCCAGGAAATATGAGATACAGGGGAACGCC...CCA'), id='7:46134999-46139999_strand_+_gene_Chr7-6Mb-seg-428', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCTCAGGCTGGGTAGCTGCTATCTGGAGTAGAGGATCCTTCCTCATTCTCTCA...CTG'), id='7:46139999-46144999_strand_+_gene_Chr7-6Mb-seg-429', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCCAGAAGTAAACCCAGCGGCCTGACACATGACACAGAGTTTTCTAGAGTTGG...CTT'), id='7:46144999-46149999_strand_+_gene_Chr7-6Mb-seg-430', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGATAGTGAATCTTCTCTGCTGGATTTTAACTTCTTTGTTAATGGTTAGCTCAG...AAG'), id='7:46149999-46154999_strand_+_gene_Chr7-6Mb-seg-431', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGTCAGCCTTCAGCAGACCTGAAGTGAGATCCGTCACGAGGGTCAGTTGCCCG...ATA'), id='7:46154999-46159999_strand_+_gene_Chr7-6Mb-seg-432', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTTTTTAGTGCAGCTTTAAAGGGGTGGAGCCAGAACATCTAGGTTTTTGTGAA...TGT'), id='7:46159999-46164999_strand_+_gene_Chr7-6Mb-seg-433', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACACTATCATTGTGACATCATTGTAACACTATCATTGTGACCTCATCATTGTA...AGG'), id='7:46164999-46169999_strand_+_gene_Chr7-6Mb-seg-434', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAGAGGGGAGGAGAGGATGCTGATCACACACGTGTGCACCTCTACAGGCATGC...CTG'), id='7:46169999-46174999_strand_+_gene_Chr7-6Mb-seg-435', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGCCATCTTAGTGACCAGGGACACTTTTCACTGAGTAGTCAGACTCCACATTT...CTC'), id='7:46174999-46179999_strand_+_gene_Chr7-6Mb-seg-436', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTCCTCGCAGCACCCTGGGCTCCCGCACGGCCCGCCCCGCCCCGCCCCGGCC...CAA'), id='7:46179999-46184999_strand_+_gene_Chr7-6Mb-seg-437', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACTATGGAAGTCTGTACTATGGAAGCAGGGTATCTGGCCAGCTATGGGATCAT...ATT'), id='7:46184999-46189999_strand_+_gene_Chr7-6Mb-seg-438', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACTATATAGAATAAGGGTTTGTATTTTATGGACCTATTGCACTTCAATCATAG...GGT'), id='7:46189999-46194999_strand_+_gene_Chr7-6Mb-seg-439', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATGCTGGGCTACGTTCTGTGCAGTGGTAAACCTTTGCCCTCCTTATCCTATCC...CAT'), id='7:46194999-46199999_strand_+_gene_Chr7-6Mb-seg-440', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTATGTTTGAAGCAGGGTCTCGTGTAGCTCAGGTTGGCCTCAACTCACCGTGTT...CGA'), id='7:46199999-46204999_strand_+_gene_Chr7-6Mb-seg-441', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGAACTAGAATTGTACGGGTCCATACCCACTGCAAATCATGTGGATTCATCCC...ATG'), id='7:46204999-46209999_strand_+_gene_Chr7-6Mb-seg-442', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAAAAGGAGGCAACAAAGGGGGGGGGGCACACCGAGTCACATGGGTAGCCACA...ACA'), id='7:46209999-46214999_strand_+_gene_Chr7-6Mb-seg-443', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACGATGAGATGCAGAATCCACAGGCAACAGTTGGGGGTTGCGTTCCCAAGGTA...ACG'), id='7:46214999-46219999_strand_+_gene_Chr7-6Mb-seg-444', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGTGTATAGAGTGAACCCTGAAACTCACCCAGCATACTAGAATAGTGAAGGG...ACA'), id='7:46219999-46224999_strand_+_gene_Chr7-6Mb-seg-445', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACAGATTAAGTAGCCAATATTCATGGAGACTGGAGACATCATATCCTTGCGGT...GCC'), id='7:46224999-46229999_strand_+_gene_Chr7-6Mb-seg-446', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACTCTGAATTCCTGTGGCACAGAGGGCCTGCCTGGGCTTGGTGACACAGACAC...CCT'), id='7:46229999-46234999_strand_+_gene_Chr7-6Mb-seg-447', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGAGACCTTCACCACCAAGGTCATGTACATAGCCTTCATCTTGTCTGGAGTCT...TTC'), id='7:46234999-46239999_strand_+_gene_Chr7-6Mb-seg-448', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTAGCCACTTCTGAATTTCCAAAGTATATCACATGGCTGTTGGTTTCATATTT...GCT'), id='7:46239999-46244999_strand_+_gene_Chr7-6Mb-seg-449', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGTAAGAATTTCTGGGGAGATCCCAGGATCCAGGGGCCAAAGTGGGGCAAGAA...AGG'), id='7:46244999-46249999_strand_+_gene_Chr7-6Mb-seg-450', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCCGTCAGCCTTCACCCTGGCCTGGGATGGTATCTCAGCCATTTACATCAAG...CCG'), id='7:46249999-46254999_strand_+_gene_Chr7-6Mb-seg-451', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTACACATTCCCTGCCACGTGTCAGTACATCCTCGCCAAGAGCCGTTCCTCAGG...ATG'), id='7:46254999-46259999_strand_+_gene_Chr7-6Mb-seg-452', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTGGAGGATACCGTGGGCCTCTGTGGCACCTTCAACGGCAACACACAGGACGA...TTT'), id='7:46259999-46264999_strand_+_gene_Chr7-6Mb-seg-453', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGCCATGTACCCACCAACGCCCTTAATAAAAGCCTCAGAGAGATATACAGGAA...CCA'), id='7:46264999-46269999_strand_+_gene_Chr7-6Mb-seg-454', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATTAGAACCCAGATTGGTTTCAATGGTGACAGGCTGGGGCCCAGGAGCCTAGC...GGT'), id='7:46269999-46274999_strand_+_gene_Chr7-6Mb-seg-455', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACTATCAGAGTCACGGCCTCGGAGATGGAGAAGATAAGCAGAAACACAGGGGT...AAT'), id='7:46274999-46279999_strand_+_gene_Chr7-6Mb-seg-456', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCGTCCTCAGTGTTTACCTCAGGTGCACCCAGCTGGTCCCTCGTAACTTTGCGT...CTT'), id='7:46279999-46284999_strand_+_gene_Chr7-6Mb-seg-457', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCATCAGCACGCTTCCTTCTCGCTGCACCGGGGGACATGGCAGGCTGGCCTGGT...GAA'), id='7:46284999-46289999_strand_+_gene_Chr7-6Mb-seg-458', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCTCCAGCCCCCAGGGTGTTTGACTCACCTGCCCATCTTCCTGGTGGCCTTGG...TCC'), id='7:46289999-46294999_strand_+_gene_Chr7-6Mb-seg-459', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTAGGTCAGCAATTCACATGCTGGGTGCCTCGTGGCGTGATTAATACAATAGT...CTC'), id='7:46294999-46299999_strand_+_gene_Chr7-6Mb-seg-460', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTACAGCCTTCCTGTGCTCCAGTGACTCTACCTACCAGGCCTGTGTGGCAGCTT...GGA'), id='7:46299999-46304999_strand_+_gene_Chr7-6Mb-seg-461', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTGGGTAAGGTGTGAGCCAAGTGGGTCTCTGTGGTGGTGGTTTCATGTCTTTC...GCC'), id='7:46304999-46309999_strand_+_gene_Chr7-6Mb-seg-462', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTCTGCCAGCATCTACAACCACAACATCAACACCTATGCCCGCTTCTGCAAGT...CCT'), id='7:46309999-46314999_strand_+_gene_Chr7-6Mb-seg-463', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGAAAGTCCATTTTCTAAAGCTAAGCTGCCTTCAAGCTTGCTGTGCATTCTCT...CAT'), id='7:46314999-46319999_strand_+_gene_Chr7-6Mb-seg-464', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAACAAGAAAACACCTTCCAGGTAAGATGTCTCCACTCAGGCAATAGTGGCAC...TCT'), id='7:46319999-46324999_strand_+_gene_Chr7-6Mb-seg-465', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCAGCCATGCCCCTGCAGCTAGCTATTTTGCTTAGCATATTCCTCTCCCTCA...TTG'), id='7:46324999-46329999_strand_+_gene_Chr7-6Mb-seg-466', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTTCTTGGTCCCTAATTGGTGAGACTGTCTGGGAAGGATTAGAAGGCGTGTCC...GTC'), id='7:46329999-46334999_strand_+_gene_Chr7-6Mb-seg-467', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAGACACATTGATCTTTTCTCATCTTTTGCTCTTGATAGAGGAGCTCTCTGCT...AGC'), id='7:46334999-46339999_strand_+_gene_Chr7-6Mb-seg-468', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATGGGGGCTGGAAAACCAGAGCTAATGCAAGCCAGCTGGGGGCCAGGACAAGA...CTC'), id='7:46339999-46344999_strand_+_gene_Chr7-6Mb-seg-469', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCTCTCTCTCTCTCTCTCTCTCTCTTTCTCTCTCTCTCTCTCTGTGTGTGTG...CCG'), id='7:46344999-46349999_strand_+_gene_Chr7-6Mb-seg-470', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACCAGAACTGGAAGCAGTAATGTTGGATGTTAGTACAACTGAGACACTGGGAG...AAT'), id='7:46349999-46354999_strand_+_gene_Chr7-6Mb-seg-471', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTGTCACCGAGGCACATCCCGAGATCACAAACTGATTGCAAGCATTTCTGACT...AAC'), id='7:46354999-46359999_strand_+_gene_Chr7-6Mb-seg-472', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCTCCACTTTTCAAATGTGGAATGTGTGGCGAGGCAGAGGGACCAGAGTCGCA...CTA'), id='7:46359999-46364999_strand_+_gene_Chr7-6Mb-seg-473', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATATCACAGTGAGCTCAGGCTCAGACTTCAGGTTCTCAGTACACCAGCTGGACC...ATA'), id='7:46364999-46369999_strand_+_gene_Chr7-6Mb-seg-474', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCACTAGACAGTGAGGCATGGCCTCCTTGTCACTCCCAGAAGAGACATCCTTCC...AGA'), id='7:46369999-46374999_strand_+_gene_Chr7-6Mb-seg-475', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGGGCTTTCCAGTTTGTAAAAAGGAACAGTTAAAATGGCTTCCAGAAGTTAGT...TTA'), id='7:46374999-46379999_strand_+_gene_Chr7-6Mb-seg-476', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGATAGTGGGACTGCACAGGGACAGCACAGTCAGGAGCCAACCCTCAGAACAT...CCG'), id='7:46379999-46384999_strand_+_gene_Chr7-6Mb-seg-477', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGATCCATCCCATAATCAGCCACCAAACGCAGACACTATTGCACATGCCAGCA...ACA'), id='7:46384999-46389999_strand_+_gene_Chr7-6Mb-seg-478', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACACACACACACACACACACACACACCCCAGTCAGCTAAGTAGCCTGCAGGTA...CAG'), id='7:46389999-46394999_strand_+_gene_Chr7-6Mb-seg-479', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCCAGGCTGGAGGTGCATCACTCTGGACCAGGAGATGGGACAGACTCTGGCCA...CTT'), id='7:46394999-46399999_strand_+_gene_Chr7-6Mb-seg-480', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCACTCACATTTGACCTTGGCCTGAAGTAGGCCCAACTTCTTTAAGGTAGTTGC...TGG'), id='7:46399999-46404999_strand_+_gene_Chr7-6Mb-seg-481', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTAATGGAACTCTTAAAGAAGAAACTGTCTCCACAGTGCCCAGCCATCTCCCA...ATT'), id='7:46404999-46409999_strand_+_gene_Chr7-6Mb-seg-482', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTTCAGGAAGGTGTTGTGTGAGTGTGTTCTGCTGAAGTCACTAGCCTTGGTGG...GTC'), id='7:46409999-46414999_strand_+_gene_Chr7-6Mb-seg-483', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCTCTTAGCACCTGAGCCTACCAACTGCCCAACATGTAGGGGCTGAGTGTGCC...CTC'), id='7:46414999-46419999_strand_+_gene_Chr7-6Mb-seg-484', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTTTCCCTGGTGACCACCAGATCAGTAGCCTCCTTTCTCTGGCTGCCAGCTGC...GAG'), id='7:46419999-46424999_strand_+_gene_Chr7-6Mb-seg-485', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGGCCTTTCTCTGGACCTCCATGTCCCTATGAAGATCAGTATTGTGAGGGGAC...TCC'), id='7:46424999-46429999_strand_+_gene_Chr7-6Mb-seg-486', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAGCTTTGGATGGTCTACCCTCTCCTCAACTCCACTGTGAGCCCCAGCAGGAA...TCA'), id='7:46429999-46434999_strand_+_gene_Chr7-6Mb-seg-487', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAGCCCACCCCAGCCTTGCACCCCCCTTCCATGGAGCACCCACTTTTCTATCC...TCA'), id='7:46434999-46439999_strand_+_gene_Chr7-6Mb-seg-488', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTTCTGTTCTGTTCCAGATCTCTCCTGATTCACAGCTGGACACACCCACCCAC...GGG'), id='7:46439999-46444999_strand_+_gene_Chr7-6Mb-seg-489', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAGCCATCTTGAAGACACTCGAGAACTCTGTGAGGACTGCCAGGATGCTCTTT...CTA'), id='7:46444999-46449999_strand_+_gene_Chr7-6Mb-seg-490', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGGTGTGAATGAGAATAGTTTTATTTCTTTGAAGACCTTCCCTAAGGATGACC...CTC'), id='7:46449999-46454999_strand_+_gene_Chr7-6Mb-seg-491', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGAGGAGGGGGAGATAGAGAGAGGTAGAAAGATCTGAATCCAATTTCTTCTT...TCT'), id='7:46454999-46459999_strand_+_gene_Chr7-6Mb-seg-492', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTGCTTGCTTACTCTGCCCCCCAGTAGAGAGCCTGAGGTAGGGTAATGCCTCA...TAT'), id='7:46459999-46464999_strand_+_gene_Chr7-6Mb-seg-493', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAGATATGATTACAATAAGGGTTCTGGGGTAGGGGGTAAGCAGGTTGTTCTGG...ACC'), id='7:46464999-46469999_strand_+_gene_Chr7-6Mb-seg-494', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATCACCTATCTATCTATTTAAACATATTCTACCCGGTGCTTCTTGATGCCTGA...TTG'), id='7:46469999-46474999_strand_+_gene_Chr7-6Mb-seg-495', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGCTTTACCTCAAAACACAGGAGCCACTCTTTTACCATCCCCCCAGGTTGGTA...CTC'), id='7:46474999-46479999_strand_+_gene_Chr7-6Mb-seg-496', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGCCAATTCCACTTCTGACTGGAAGTGGCTACCTGGAGTGCTGTGTGGGGGAT...TGC'), id='7:46479999-46484999_strand_+_gene_Chr7-6Mb-seg-497', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATATTAAACAAAACTATCTCACCTATACATGTGAATATGACATCTAAACATACA...TAA'), id='7:46484999-46489999_strand_+_gene_Chr7-6Mb-seg-498', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTGTCTTCAGTACCGTACAGAGCATGAGGAAGCTTGCATGTGATATTTTACAT...TCT'), id='7:46489999-46494999_strand_+_gene_Chr7-6Mb-seg-499', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCAGTGATCTCCCTGCCTTCCTTTCCTTGGTGCTCCTGGGGACTGAACCCAG...CGT'), id='7:46494999-46499999_strand_+_gene_Chr7-6Mb-seg-500', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTATAGGAAACTGGTGTCGTCCCTTGGCCGTATGAAGGGTGTTTCCCGAGCAGT...GAT'), id='7:46499999-46504999_strand_+_gene_Chr7-6Mb-seg-501', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTGCTAAATTATCCAAGTGGGTCCAATATAATCATGAGGCTAAATGGGGACCA...AGA'), id='7:46504999-46509999_strand_+_gene_Chr7-6Mb-seg-502', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCACTTCTGGGGTGGGCACTGTGTTGGCAAGGTTACAGAGCGGGAATCCAGCT...AGT'), id='7:46509999-46514999_strand_+_gene_Chr7-6Mb-seg-503', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATACTGGGTGTAAGTAGATCCTGAGCAGTGATTGCCCCACTGTGCTCTTTAGC...AAC'), id='7:46514999-46519999_strand_+_gene_Chr7-6Mb-seg-504', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTGTGTCTTGTCTGCCTTACAGAAGTGCTGAGGGACAGATGGTGGCCCTCCTT...CTG'), id='7:46519999-46524999_strand_+_gene_Chr7-6Mb-seg-505', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGATCAAAAGTCCTGGATGAGTGGCAGGCTCACACAGCCCCAGCCATCTCCTC...AAA'), id='7:46524999-46529999_strand_+_gene_Chr7-6Mb-seg-506', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCTCAGGCATCAGTTCTTGCCCCCACGCCTTCCCTGCAGCCACCTCTGTCTCC...GCC'), id='7:46529999-46534999_strand_+_gene_Chr7-6Mb-seg-507', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCACAAGGCAGTCCTGAGATCTGCACGGCAAGCTCTGACCAGTTTTCCTCCAT...CTA'), id='7:46534999-46539999_strand_+_gene_Chr7-6Mb-seg-508', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTATCCACCTATCTCTAAAGCTATCCATATACCCACCCACCCTCTCTTCTACT...TAC'), id='7:46539999-46544999_strand_+_gene_Chr7-6Mb-seg-509', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGATTCCTTTTAATGTTTCAGTGAAGGCTTGGTTGGTGGCAGGCACTATCCTA...CAC'), id='7:46544999-46549999_strand_+_gene_Chr7-6Mb-seg-510', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAGTAGGTTAAAGACAATGGGAACGAGAGGGAAGGAAGGGGGAGAGAGATGAC...TCA'), id='7:46549999-46554999_strand_+_gene_Chr7-6Mb-seg-511', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCCAGCATGGTGCAAGCACACACCAGCACGGTGCAAGCTCATCCCAGCACAGT...GGA'), id='7:46554999-46559999_strand_+_gene_Chr7-6Mb-seg-512', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGGCTAGGGCTAAACTAGTCCTGTGGGAAAGAAGCGCAAGCAAGGAAGAAGCC...TGG'), id='7:46559999-46564999_strand_+_gene_Chr7-6Mb-seg-513', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAAAGTGGGCTGAGCTCTGGTCTGCATACCCCTTCTCTCTGTCCTGAATCTGA...GAG'), id='7:46564999-46569999_strand_+_gene_Chr7-6Mb-seg-514', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAGAGAGAGAGAGAGAGAGAGAGAGAGAGAGAGAGAGAGAGAAAGGAGGGAGG...TCA'), id='7:46569999-46574999_strand_+_gene_Chr7-6Mb-seg-515', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAGTGCCTGGCAGTTTATCATCTTTATTGTCATCACCATTACTATCACTACCA...AGC'), id='7:46574999-46579999_strand_+_gene_Chr7-6Mb-seg-516', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGCTGATGGTGCTAAGTGACAGGCTATGGAACTATTAGAAGGTGGATCTTTG...GAA'), id='7:46579999-46584999_strand_+_gene_Chr7-6Mb-seg-517', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAAACAAATAAAACAAACCATACAGCAGGATCTCAAAGGATACAGTTTGCCAGA...TGT'), id='7:46584999-46589999_strand_+_gene_Chr7-6Mb-seg-518', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAGAATACTTAGCTCAGCACTAAACACCTTCTAGGTCGGGTCACAGCCCCCCA...TGA'), id='7:46589999-46594999_strand_+_gene_Chr7-6Mb-seg-519', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGAGGGTCCCAGGGGGCAGTCTGGGCTTCAGGACAGCCAGGCTCACCTGGCAC...AAA'), id='7:46594999-46599999_strand_+_gene_Chr7-6Mb-seg-520', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTTCAAACCCGTGGAAGGCCATGGAGTAAACCCTAGTCCTAACTCATTTAGTG...GTG'), id='7:46599999-46604999_strand_+_gene_Chr7-6Mb-seg-521', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTGTGTGTGAAGGGATGCAGACAGAAACTATGAATAAGTGAACAGCCCAACCC...ATT'), id='7:46604999-46609999_strand_+_gene_Chr7-6Mb-seg-522', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATACTGCCAACCTATTCTGGATGGGATCTTGGCCCCAAAACAGAAAAATCATC...CAT'), id='7:46609999-46614999_strand_+_gene_Chr7-6Mb-seg-523', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTCATGCCTGGGGCTGGGGGAGTTGTGGGTACCGGTGAACAGAGAACATCGTG...TAC'), id='7:46614999-46619999_strand_+_gene_Chr7-6Mb-seg-524', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTGGCAGAGCTGAGTCTACCCACAGGTACAGTGGCAGAGATGAGAGTCTACCC...GCT'), id='7:46619999-46624999_strand_+_gene_Chr7-6Mb-seg-525', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGCACCAAACCTGGATCCTCGGGAAGAGCGGCCGGTGCTCTTAACCACGAGTC...ACT'), id='7:46624999-46629999_strand_+_gene_Chr7-6Mb-seg-526', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAATTGTAAATCTCTCAGGTGCTAAGACGGGTCTTAACGACTAGTGTCCTTGCT...AGG'), id='7:46629999-46634999_strand_+_gene_Chr7-6Mb-seg-527', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCAGCACTCTCTGAACTAGGTACCAGGACAGAAAATTAGGGATGAGGTAAGGT...CCA'), id='7:46634999-46639999_strand_+_gene_Chr7-6Mb-seg-528', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGAACACACCCGGAGGTGCCTTACTACACCCACACTCGTAATTCTGCACCATG...TTG'), id='7:46639999-46644999_strand_+_gene_Chr7-6Mb-seg-529', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGATCATGAACTTTAAGTTTATTACCCAGTGGAACAAATTCTTAAATTATAAGT...CTG'), id='7:46644999-46649999_strand_+_gene_Chr7-6Mb-seg-530', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGCTCGTGATGAGACATTCCTGTTTACAGGCAATCTTGGGATCAAAGGGCTT...GAT'), id='7:46649999-46654999_strand_+_gene_Chr7-6Mb-seg-531', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGGGGCTACCAGGTTCAGTAGCTCAAGAGTTCACAGTGTACTCTGTCTTGTAC...GAG'), id='7:46654999-46659999_strand_+_gene_Chr7-6Mb-seg-532', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAGGGCAATCACAGTGAGTATAGTCAAAACATCTTCCATATGTATGAAAATAT...CTG'), id='7:46659999-46664999_strand_+_gene_Chr7-6Mb-seg-533', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAGAACCACAGGGGTTTTTAACCACTGAGCTATTTTTCCACTCTTCCTGAACA...CCA'), id='7:46664999-46669999_strand_+_gene_Chr7-6Mb-seg-534', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGGAGAAACAATGGTTTGGATTTTTTTTTAATGCTAAATCTTCCCTGGCTAGC...GCT'), id='7:46669999-46674999_strand_+_gene_Chr7-6Mb-seg-535', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCCTACCAGCCACCACTGGCAAAAGCAACTCAGGGTGGGAGTTTCTCAGGCCC...GAG'), id='7:46674999-46679999_strand_+_gene_Chr7-6Mb-seg-536', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGTTCCAGGACAGCCAGGACTACAGAGAGAAACCCTGTCTTGAAAAACAAAA...TGC'), id='7:46679999-46684999_strand_+_gene_Chr7-6Mb-seg-537', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTATAACCCAGGCTGGACTTGAACTCGTGACTCTCCTGCCTTGGCCTCCAGAG...CAG'), id='7:46684999-46689999_strand_+_gene_Chr7-6Mb-seg-538', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCCTGAACCTCCGCACAGCTAATCTCAGTGAAAGGCAAAGCTCCCCTGCTTC...CAT'), id='7:46689999-46694999_strand_+_gene_Chr7-6Mb-seg-539', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGCACAGGGGAAATTTTCCTGAACAGAACATCAATGGCTTATGTTCTAAAGAT...GAG'), id='7:46694999-46699999_strand_+_gene_Chr7-6Mb-seg-540', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTGTGAATACATGGGAAGTGTGTGAATGTACGTACGCGTGATACTGAGACATG...CAG'), id='7:46699999-46704999_strand_+_gene_Chr7-6Mb-seg-541', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTAAAGCCTCTGGAGCCAGCATTTGCCAAGGGCACACTGATCAGTGTGATAAT...CTA'), id='7:46704999-46709999_strand_+_gene_Chr7-6Mb-seg-542', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGCAACAAGAACAACCTTTAAAAATAATAAAAAGGAAGGGTTTTTTTTTTTT...CCT'), id='7:46709999-46714999_strand_+_gene_Chr7-6Mb-seg-543', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAAGCCTACCTTGACCAGCTTCTTTCATGAACTGGACCCATCTTTGGCTGTCA...ATC'), id='7:46714999-46719999_strand_+_gene_Chr7-6Mb-seg-544', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTTTTTCCCTGGTGCCTACAGCCAAGAAGGCGAGGAGCCTCTAACTGCCCCCA...ACA'), id='7:46719999-46724999_strand_+_gene_Chr7-6Mb-seg-545', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACTAAGCTTTCTTTAGCTTAAAAACAGATTCATTCCATTGCTTATTATTCTGT...AGG'), id='7:46724999-46729999_strand_+_gene_Chr7-6Mb-seg-546', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGGAACAGTGAAAGGCATGAAGCACCAGACCATTACCTGATTATTTTAGCAGC...CAT'), id='7:46729999-46734999_strand_+_gene_Chr7-6Mb-seg-547', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGATGGCTTTAGGAGGAACCAGAAATTGCCAGGAAGGGGGTGGAGGGCTGGAA...AGG'), id='7:46734999-46739999_strand_+_gene_Chr7-6Mb-seg-548', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTTGCCCAATCAAAACCATCACATACAAGGGACTCTACTTATTTTCTAATACT...CAA'), id='7:46739999-46744999_strand_+_gene_Chr7-6Mb-seg-549', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAAATGAGGGCAGTTATGGCCAATGTGACCTGGCTAGCAAATTATGAATCTGA...GAG'), id='7:46744999-46749999_strand_+_gene_Chr7-6Mb-seg-550', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACAGCATTCTCGGTTTCCTTCCGGCTCTCTAGCTCCTGTCTCACAGGGCTTTT...TCC'), id='7:46749999-46754999_strand_+_gene_Chr7-6Mb-seg-551', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGCATTACTTGATGTTCCAGGATGGGGGGATACTGAAGAGGGCGCTCCACCCT...TGG'), id='7:46754999-46759999_strand_+_gene_Chr7-6Mb-seg-552', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTAGGAATACTGTTTGGGAAGTGAGCAGTGTGGTCACTTGCCCTCGGAGGGGC...CAC'), id='7:46759999-46764999_strand_+_gene_Chr7-6Mb-seg-553', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACATGCTCAGTAAAGCTTAAGGTAAAAGTCAGCAAAAATTAAAACTCGGCCTTC...ATG'), id='7:46764999-46769999_strand_+_gene_Chr7-6Mb-seg-554', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTGCACTGTGTGTGTGTGTACCATGTATGTGTGTGCTTATGAGTCACATATCT...TAA'), id='7:46769999-46774999_strand_+_gene_Chr7-6Mb-seg-555', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAAATTCAGAAATGAATAAACCAAAGGAAATGAAGGATTCAATACCAAAATCA...TAC'), id='7:46774999-46779999_strand_+_gene_Chr7-6Mb-seg-556', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAGCCCCAGTCCTCCACTGGCCTCTCTGTTCCTCTCATCTTTCTCTTCTAAAT...GTT'), id='7:46779999-46784999_strand_+_gene_Chr7-6Mb-seg-557', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGCCCATGGGAAGCACTCAGTTTAGTTTTGTTTCAGACAGGGTCTCACTAAGC...CTG'), id='7:46784999-46789999_strand_+_gene_Chr7-6Mb-seg-558', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATTGAGACATCAGGCTTAGGCCTGCAATCCCAGCACTGAGGAGGGTGAGGCAG...AAA'), id='7:46789999-46794999_strand_+_gene_Chr7-6Mb-seg-559', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAAACAAAAAAACAGAAACAAAACAAACAAACAAAAAAAACAGAAACAAAACA...TAC'), id='7:46794999-46799999_strand_+_gene_Chr7-6Mb-seg-560', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGTAACTAATGCTGTTTCGTTCTTCTTTTTTTTTAAGATTTATTTATGTAATG...TAA'), id='7:46799999-46804999_strand_+_gene_Chr7-6Mb-seg-561', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGACCTTGTTGTGAGCCAAGTGATCAGTGCTGAGGAATTCTGGGCCAATCGTTT...AGC'), id='7:46804999-46809999_strand_+_gene_Chr7-6Mb-seg-562', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGATTGGCACTTGTCAAATGAAAAGACTAGTGCCTTTTTTTCCCCTGAGGGTAA...ACT'), id='7:46809999-46814999_strand_+_gene_Chr7-6Mb-seg-563', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAGAGATGGCTCAGTGGTTAAGAGCACTGGTTGCTCCTCCAGAGGACCTGGGT...TTA'), id='7:46814999-46819999_strand_+_gene_Chr7-6Mb-seg-564', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTAGTGTGTTAAAGCAGGATGGAGGTGTGTATTTGCTACACTCAGAAGAAACG...TGG'), id='7:46819999-46824999_strand_+_gene_Chr7-6Mb-seg-565', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCAGTGGGTAAGAGCACCCGACTGCTCTTCTGAAGGTCTGGAGTTCAAATCCC...GCC'), id='7:46824999-46829999_strand_+_gene_Chr7-6Mb-seg-566', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCGGGCCTCTGGGAGTAACGGAGTTAGCCTTGGCTGCAGACGAGACGTGGCAT...CTG'), id='7:46829999-46834999_strand_+_gene_Chr7-6Mb-seg-567', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCTCTGTATTACCCATGAACCCTGTGTTGTACCCCGAGGCCCTGAGCTCCGGA...GGG'), id='7:46834999-46839999_strand_+_gene_Chr7-6Mb-seg-568', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGAGGGAGGGGCGGGTTGGATGTGACACAACTGATAGGGTGCCTCCATCATGC...TAA'), id='7:46839999-46844999_strand_+_gene_Chr7-6Mb-seg-569', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAGACCTTAATCTGCAGTGAACACATCCCTTCCGGGGAGATGGGTGAGCAGGGT...AAA'), id='7:46844999-46849999_strand_+_gene_Chr7-6Mb-seg-570', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAAAAACAAAACAAAACTATGCACCAAAGCGAGGGTCTGAACGGAAACAGGAC...GGA'), id='7:46849999-46854999_strand_+_gene_Chr7-6Mb-seg-571', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTCTTCCTCAGTGTCCCATGTATCCTGGGACAAAATGGAATCTCGGATGTTGT...CAC'), id='7:46854999-46859999_strand_+_gene_Chr7-6Mb-seg-572', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCACTGGTTAAATGGCAGCAGCCTGGAGGGCTAGGGAGAGCTCTCCAGAAGGG...CAC'), id='7:46859999-46864999_strand_+_gene_Chr7-6Mb-seg-573', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCTGGTTTGGTGGCACACATCTGCAATCCCAGCACTCAGAAGGCAGAGATAGG...AAA'), id='7:46864999-46869999_strand_+_gene_Chr7-6Mb-seg-574', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCAGACCAAAACCAGACTTGAGACAAACTTTTCTACCCAGCCCCATTATAGTG...CAG'), id='7:46869999-46874999_strand_+_gene_Chr7-6Mb-seg-575', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAATATGGATCCAGCATCAGAGGATCCAACACTCTCTTCAGGGCCCAAACCCA...TAT'), id='7:46874999-46879999_strand_+_gene_Chr7-6Mb-seg-576', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGCTCTGACATGGTTCACAACTGGCTCTAGCTCTGGGGAACTGAAGGCTGCCT...GGA'), id='7:46879999-46884999_strand_+_gene_Chr7-6Mb-seg-577', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTGGTAGGAGCTGGACTTAGAACATTTGCAGCAAATGTACAGCTTGGTCTTCA...AGC'), id='7:46884999-46889999_strand_+_gene_Chr7-6Mb-seg-578', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGGCTGTCCTGGCACTCACTTTGTAGACCAGGCTGGCCTCGAACTCAGAAAT...AGG'), id='7:46889999-46894999_strand_+_gene_Chr7-6Mb-seg-579', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGTCCCTGTCCACTCTAGTTTACGTAGCATCTATAAAGTCTGTGGGAGGCTAA...TAA'), id='7:46894999-46899999_strand_+_gene_Chr7-6Mb-seg-580', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCTCTATCGCTAGAGTGAGGAATCGAAGTGCCAAGAACTAATCGGGCATTACA...GTC'), id='7:46899999-46904999_strand_+_gene_Chr7-6Mb-seg-581', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTATCACTCACTTCCATGCTGGCTTCTCCAATATATAAAGCACCATCTCTGAG...ATA'), id='7:46904999-46909999_strand_+_gene_Chr7-6Mb-seg-582', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTGCTCCAAGCTGATGGTGATATTCTGGCACCTGTTACAGGACTATGTCACTA...CCC'), id='7:46909999-46914999_strand_+_gene_Chr7-6Mb-seg-583', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACGCTGTCAGCATCACTGATTAATGTCTAGAACAATCTTCTTCCAGCCTACTT...TCA'), id='7:46914999-46919999_strand_+_gene_Chr7-6Mb-seg-584', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTCCGGGGCCGCTGGTCTGCGGTGCGAAGTCGGGCGAGAGATCAGGTCCGGTC...GGC'), id='7:46919999-46924999_strand_+_gene_Chr7-6Mb-seg-585', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGATATGATGGCCTACACTTTTAATCCCATTGAGAGACAGAAGCAGGCAGTTCT...GGT'), id='7:46924999-46929999_strand_+_gene_Chr7-6Mb-seg-586', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGTCTGAAGACAGCTACAGTGTATACATATACCTTAAATGAATGAATGAATGA...GCC'), id='7:46929999-46934999_strand_+_gene_Chr7-6Mb-seg-587', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTGGGGGACGGACGCCTGGCACAGCACCAGGCAGCTGAAGCTGAGGAAGGAAG...GGT'), id='7:46934999-46939999_strand_+_gene_Chr7-6Mb-seg-588', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATTCTAAGAGCAGTCTTACTGAATATTCCTTACACACCACGGCGATTTGGCTT...GTG'), id='7:46939999-46944999_strand_+_gene_Chr7-6Mb-seg-589', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTGATTGACAGATCCTGCCTCAGTGAAGAAAGTGGAAGAGCAGTTGAACATAA...TGC'), id='7:46944999-46949999_strand_+_gene_Chr7-6Mb-seg-590', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCACTGCAAAAACAAAAACCTTAGCAAGGCCTGGAGTACGGAATAACAGCTATT...CAC'), id='7:46949999-46954999_strand_+_gene_Chr7-6Mb-seg-591', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAAGAACTAGCTTCCGTACTTACCCTGATACATCACAGGAATTGTACCGGTGA...CTC'), id='7:46954999-46959999_strand_+_gene_Chr7-6Mb-seg-592', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGGCTATGTATAACTCTAATTTTAGGGGATCTGATGCCCTTGTGCCCTCCCTC...CCC'), id='7:46959999-46964999_strand_+_gene_Chr7-6Mb-seg-593', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTGAGTGAGCCTTTACTGTACTCATTTGTATTTCATGATGCCCTAAATGCATA...AAT'), id='7:46964999-46969999_strand_+_gene_Chr7-6Mb-seg-594', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAACCAGAAGTAAAACTCCAACCCAGAGAAGGTAAGAACACACCAGAAAGTACA...CAC'), id='7:46969999-46974999_strand_+_gene_Chr7-6Mb-seg-595', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGAGCGGAGAAACTTTCAGTGATCAAGAAAAAAACAAAAAACAAAACAAAAAA...TTT'), id='7:46974999-46979999_strand_+_gene_Chr7-6Mb-seg-596', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTTTCCCAGAGTGAATTTCTGGTTTATCAAAAATTAGGTGTCAATATATGTAT...TTA'), id='7:46979999-46984999_strand_+_gene_Chr7-6Mb-seg-597', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCTTCTACCCACCCAACCAGCAGCTGCATCTGTCTTGGCAACACCTTTGTAAC...CAG'), id='7:46984999-46989999_strand_+_gene_Chr7-6Mb-seg-598', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTTGTTTTGTTTTGAGATTTTATGTTTTGAGTGTTCTGCCTGCATGGTACTGG...GGA'), id='7:46989999-46994999_strand_+_gene_Chr7-6Mb-seg-599', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCTTAGGTGTCTACAAGGCAAGTCTTCAGAACCCAGCATTAGCAAAAGCACCA...TAG'), id='7:46994999-46999999_strand_+_gene_Chr7-6Mb-seg-600', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGTTCAATCCCCAGCCAGTACTGCAGAAAACACAGAAAACAAAGCGAGGAAAG...AAG'), id='7:46999999-47004999_strand_+_gene_Chr7-6Mb-seg-601', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGAAAACTGCATATTTGCCTTCACAACTTGGGTGAAAGTCAATGCAGGGGCGC...TTT'), id='7:47004999-47009999_strand_+_gene_Chr7-6Mb-seg-602', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTGTGATTATTTACAGTTTGTAGTCCAACGATGGCCAACTATGAATGGGAAGT...AAG'), id='7:47009999-47014999_strand_+_gene_Chr7-6Mb-seg-603', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAACTAAGATATTTTCGTTAAACTAAAGGGCCAGAGGCCAGGAAGTGACACAA...CAG'), id='7:47014999-47019999_strand_+_gene_Chr7-6Mb-seg-604', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCTTTAAGTAAATCACTGAATAAGCCAACATCTAACAATCAGTCCTTTCGGCT...GAG'), id='7:47019999-47024999_strand_+_gene_Chr7-6Mb-seg-605', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGGGAGGGTGGATCTTACAGAGTTACAGAGAGGGTGGGGCGAGTAAAGGTGAT...TCC'), id='7:47024999-47029999_strand_+_gene_Chr7-6Mb-seg-606', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGTCCAACCATTTGCCTAGGAATTTCATAAATTCATTCTTTTTAATAGCTGAG...TAA'), id='7:47029999-47034999_strand_+_gene_Chr7-6Mb-seg-607', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTTTTACTTAGTTGTTCCATCTGGATGCATTGGACCTTTGTCTACTCTCTTC...AGT'), id='7:47034999-47039999_strand_+_gene_Chr7-6Mb-seg-608', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTTCTCCATTGTGTAAATGTACCACATTTTCTGTATCCATTCCTCTGTTGAGG...TAA'), id='7:47039999-47044999_strand_+_gene_Chr7-6Mb-seg-609', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCCACATATTATCTTGCACAGTGTCAATATTTAGCCCTCACCGAGGGTCCAGA...GCC'), id='7:47044999-47049999_strand_+_gene_Chr7-6Mb-seg-610', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTAATTTGTTTACTCATTGCTGTTATGGGGTCCACATACAGGCTGCTGCTTGC...GTG'), id='7:47049999-47054999_strand_+_gene_Chr7-6Mb-seg-611', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTGGCTGTGTGGTTGTGTGATTATAGTCAGGAGGGGGAGGGTGGGGATGAAT...ATT'), id='7:47054999-47059999_strand_+_gene_Chr7-6Mb-seg-612', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATATATCCAACCCCATGCCTAGTTTCTATACTTAGTATACAGGAACAGGAGTC...ATA'), id='7:47059999-47064999_strand_+_gene_Chr7-6Mb-seg-613', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCATGTATTCCTCACCTGTTTTACCATTGACACACCAGGCAAGTCTGTGATCT...GGG'), id='7:47064999-47069999_strand_+_gene_Chr7-6Mb-seg-614', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATAAGGAAGGAACCACCATTCACAGAGCACTCATTATGTGTCTGGGCCCAGGA...GGA'), id='7:47069999-47074999_strand_+_gene_Chr7-6Mb-seg-615', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAATAAGGGGCTTAAACTTACTCAGCTGAATAGGGCTCACAGAGGGCTGGAGGA...ATG'), id='7:47074999-47079999_strand_+_gene_Chr7-6Mb-seg-616', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTGTCTGCCCACCCCACTGTCCTCATTAGCTGCTGTCAGCCTCCTCTCTGCT...CCT'), id='7:47079999-47084999_strand_+_gene_Chr7-6Mb-seg-617', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTAGGTTCAGGGTATGGTTGGAGGAGGTGGGAACCTGTCAACAATCCAGCAGGA...GTG'), id='7:47084999-47089999_strand_+_gene_Chr7-6Mb-seg-618', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTTGAGGAGGTGCCACACCTGGTTGGGAGCACACAGCATCATGGACTGGGGAC...GAG'), id='7:47089999-47094999_strand_+_gene_Chr7-6Mb-seg-619', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGGGTGGCTCACCCATGATGGGCACAGCTGGTGACCAGTAAGTTCAGTTTGAT...CAG'), id='7:47094999-47099999_strand_+_gene_Chr7-6Mb-seg-620', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTTCTCATTCTGGCTCTGTTACCTACCCTAGGCTGCTACTGCATTTGCCTCTC...GCC'), id='7:47099999-47104999_strand_+_gene_Chr7-6Mb-seg-621', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGCACTTGGGCAGATGTGCTCAGCACTGCCATTACACCAGGAGAATGGCTCTT...GGC'), id='7:47104999-47109999_strand_+_gene_Chr7-6Mb-seg-622', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAGACCATGTGTGAGCCTCCATGTGAATACTGGGATTGAACCTGGACCCTCTG...TCT'), id='7:47109999-47114999_strand_+_gene_Chr7-6Mb-seg-623', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCTATGGGTGAATAATCTCCCACTGTATGGACAGACAACATTTCATTATCCA...TAT'), id='7:47114999-47119999_strand_+_gene_Chr7-6Mb-seg-624', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGTCTCTCTGGGAAATACTAATTTTCACTTTAAGAGAAGTGGGACCTGTTTGA...TGA'), id='7:47119999-47124999_strand_+_gene_Chr7-6Mb-seg-625', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCTTAAATTTGTGTTTTTATGTTTATTTATTCTTGTGGTCCTGAGGATTGAAT...TGA'), id='7:47124999-47129999_strand_+_gene_Chr7-6Mb-seg-626', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAAGGGTAAGAAAAGCCTTGACACTGTGTAAAGATGCATCACCTAATCATTCA...CCT'), id='7:47129999-47134999_strand_+_gene_Chr7-6Mb-seg-627', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCTGTCAATGTAGCTATTGATGCCCATCTCAAAGAGAGAGACAAGGAAGACCC...CTT'), id='7:47134999-47139999_strand_+_gene_Chr7-6Mb-seg-628', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTTCAGGCATTTGAGTGCTTGGTCCCCAGCTGGTAGCACTGTTTGGCCTTGCT...CTT'), id='7:47139999-47144999_strand_+_gene_Chr7-6Mb-seg-629', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTCTTCTTCTTCTTCTTCTTCTTCTTCTTCTTCTTCTTCTTCTTCTTCTTCTT...CTT'), id='7:47144999-47149999_strand_+_gene_Chr7-6Mb-seg-630', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTGCCTCGTCTTATTTTTGAGCATGACTTTGAACCAACCCATGGCTAGTGAAG...TTT'), id='7:47149999-47154999_strand_+_gene_Chr7-6Mb-seg-631', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGGACATTCTCAAGCATTGTTTATTTTGGTGGTTCTCCTTCACTTCCTCGTG...GGG'), id='7:47154999-47159999_strand_+_gene_Chr7-6Mb-seg-632', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATGAAATTATTATCTCACTACCCCCTGCTGATTAGGGGCATTGAGTTCCTTGG...CAG'), id='7:47159999-47164999_strand_+_gene_Chr7-6Mb-seg-633', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACATCCAGTGCGATGACATTAAGTCCAGCAGCCTTCTTTCTTTGTATGAGACAG...GAC'), id='7:47164999-47169999_strand_+_gene_Chr7-6Mb-seg-634', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACATTGTCCTTGAACTCACATAAATCTATTTACCTCTGTCTCCCAAGTGCTGG...CCA'), id='7:47169999-47174999_strand_+_gene_Chr7-6Mb-seg-635', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCCCCCCAATCCCCTACCCATCCACTCCCCCTTTTGGGCCCTGGCGTTCCCCT...TTC'), id='7:47174999-47179999_strand_+_gene_Chr7-6Mb-seg-636', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTCTCATCCACTCCTATTATCCGTAGGTTTGGTGTTCTCATTGTGTCCCGGAT...AAT'), id='7:47179999-47184999_strand_+_gene_Chr7-6Mb-seg-637', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCAAGAGGTTTAGTGTTCCAGTGTTCATGGAATTTCTCAGGCTAGTAGGGAGA...AGG'), id='7:47184999-47189999_strand_+_gene_Chr7-6Mb-seg-638', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTGGCAGAGCCTCTCAGGAGACAGCTATATCAGGCTCCTGTAAGCATGCATT...CAT'), id='7:47189999-47194999_strand_+_gene_Chr7-6Mb-seg-639', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTAGTCAGACTAATTGGCCTGAAAGCCCCCAAGGACCCTCCTCTCTGTGCTTCC...TGT'), id='7:47194999-47199999_strand_+_gene_Chr7-6Mb-seg-640', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAGAAAGAGGAACACTCCTCCATTGTTGGTGGGATTGCAAGCTTGTACAACCA...TTT'), id='7:47199999-47204999_strand_+_gene_Chr7-6Mb-seg-641', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCAGTACCCGAGACAGTCCTGTGGTTTCCTGCTCAGTGTCCTATTGGCAAGTG...ATT'), id='7:47204999-47209999_strand_+_gene_Chr7-6Mb-seg-642', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATATGTTGTCATACAAGTTTCAAAATGTGTGTGTGTAGGTGTGTATGTATATAT...CAT'), id='7:47209999-47214999_strand_+_gene_Chr7-6Mb-seg-643', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTATGCTCCTTGGGTTTTGGCTCAATCTCTGAGATCCCTGCAAGGGTCCAGGTT...CTT'), id='7:47214999-47219999_strand_+_gene_Chr7-6Mb-seg-644', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGATAATTAATGAATGTTATTAGTAAGCCTGAGACAGAGGATTTGAGAAAGATA...CTA'), id='7:47219999-47224999_strand_+_gene_Chr7-6Mb-seg-645', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGGTTTGCTGTAGATTGCTTTTATCATGTTTATGTATGGGCCTTGAATTCCTG...TCC'), id='7:47224999-47229999_strand_+_gene_Chr7-6Mb-seg-646', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAGTTTACCCCCCTCAAATGCATGGTTGTAACTATAGTTTATAACCAAGATGC...AAC'), id='7:47229999-47234999_strand_+_gene_Chr7-6Mb-seg-647', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATTTTGAGGGTCTGGTGCTTCAACCGATGCCTGAATGAGCCCACGAAGAAGTA...CGT'), id='7:47234999-47239999_strand_+_gene_Chr7-6Mb-seg-648', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTATCTGCTAAACCTTTGGGGACTTTTGCAGTTGGACTGGGTAGAGGACTGGG...GCA'), id='7:47239999-47244999_strand_+_gene_Chr7-6Mb-seg-649', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTTTCTAGCTGTGGTGTACTCATAGCTCTCATACAATGGTTTGGTGTTGATT...ATA'), id='7:47244999-47249999_strand_+_gene_Chr7-6Mb-seg-650', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTCAATGACCCAGGAAATTCCTAGGCGTAAGTATAACCAGGGTCAGAAAGAATG...GGG'), id='7:47249999-47254999_strand_+_gene_Chr7-6Mb-seg-651', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAGTACCAACTGTGGACAGGTGGGCCGGGGTTGCATAAGAAAACAAGCTGAGCA...CAT'), id='7:47254999-47259999_strand_+_gene_Chr7-6Mb-seg-652', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCTGTATCCATTCCTCTGTTGAGGGGCATCTAGGTTCTTTCCAGCTTCTGGC...GTT'), id='7:47259999-47264999_strand_+_gene_Chr7-6Mb-seg-653', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGTTCCACTCACTAGAGGTCTTAGGATCACGTGTGGAATCCTGTGTGGGCCCT...AGA'), id='7:47264999-47269999_strand_+_gene_Chr7-6Mb-seg-654', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACAGAGACAGATACACATAGAGACAGAGACACAGAAAGAGACAAAAAGAGACA...TTC'), id='7:47269999-47274999_strand_+_gene_Chr7-6Mb-seg-655', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAAGAAAGGGTAAAGTGTTCAAACTTTGGTCTTCGTTCTTCTTGAATTTCATGC...ATT'), id='7:47274999-47279999_strand_+_gene_Chr7-6Mb-seg-656', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGGATTTTTGCATCAATATTCATAAGGGAAATTGGTCTGAAGTTCTCTATCTA...GAC'), id='7:47279999-47284999_strand_+_gene_Chr7-6Mb-seg-657', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTGGGTTGGAAGTCGATTTTATTTGATATTAGAATGGCAACTCCATCTTGTTT...AAA'), id='7:47284999-47289999_strand_+_gene_Chr7-6Mb-seg-658', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATTAAGTGTAAAGAGTGCCATAGATACAGCTACTCTTGGCACTGAGGGCAGAG...CGC'), id='7:47289999-47294999_strand_+_gene_Chr7-6Mb-seg-659', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACTGAAACTGGTAGCTGTCCAGGCCTTACTGTGACTGAGACTGGTCGCTGTCC...CAT'), id='7:47294999-47299999_strand_+_gene_Chr7-6Mb-seg-660', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTAATCTTAATTCTGTCTGTCCCACGGATGAAGTCCTAGTATCTAAAAAATTA...TAG'), id='7:47299999-47304999_strand_+_gene_Chr7-6Mb-seg-661', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTTCAAGTGGCCAGAGTATCAAGGTCAAAGCATGCCCTTCCTTAACTAGATTA...CTC'), id='7:47304999-47309999_strand_+_gene_Chr7-6Mb-seg-662', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGCCTGCAATGTAGAGAACCAGCATGACGCTGAAGAAGCACTTGCGAAAGATA...GAA'), id='7:47309999-47314999_strand_+_gene_Chr7-6Mb-seg-663', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAATGAGACAAAAACAACAACAACAACAACAACAACAACAACAACAACAACAAC...CTT'), id='7:47314999-47319999_strand_+_gene_Chr7-6Mb-seg-664', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTGTCACAGCAATGAGGAAAGTAACTAATACAAACCCTGTTTCTGCTTTCCTT...TGT'), id='7:47319999-47324999_strand_+_gene_Chr7-6Mb-seg-665', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTGGGGCATATGATATTCACAAGACCAAGGGCCTCTCTTCCCACTGATGGCCA...CCC'), id='7:47324999-47329999_strand_+_gene_Chr7-6Mb-seg-666', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTTAGTTTCTGTTTCCACGATCTGTCCATTGATGAAAGTGGTGTGTTGAAGT...CAG'), id='7:47329999-47334999_strand_+_gene_Chr7-6Mb-seg-667', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCATTCTGGAGAACCATTTTGAGGGTCTGGTGCTTCAACCGATGCCTGAAGGA...TTT'), id='7:47334999-47339999_strand_+_gene_Chr7-6Mb-seg-668', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATAATGTTGGGTCTGATAAAACTTTGGGGACTTTTGAAGTGGAATTGGGCAGTG...AAC'), id='7:47339999-47344999_strand_+_gene_Chr7-6Mb-seg-669', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCAGCTCCTCCTGTAACATGCATATCTGCCTTGATGCTGCCATGCTCCTGCC...CCC'), id='7:47344999-47349999_strand_+_gene_Chr7-6Mb-seg-670', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGCGATTCAACAGGTGTGAAGCCCTGCACTTTGGGGCAAAATGCACAAACAGG...AAT'), id='7:47349999-47354999_strand_+_gene_Chr7-6Mb-seg-671', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGTCTCTGTTAACAATTATACAATCCAACTCCATGAGATGTAAAAATGTTGTA...TGG'), id='7:47354999-47359999_strand_+_gene_Chr7-6Mb-seg-672', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATATATGCCCAGGAGGGGTATTGAGAGATCCTCTGGTAGTACTATGTCCAATTT...ACT'), id='7:47359999-47364999_strand_+_gene_Chr7-6Mb-seg-673', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCCTTAATATCTCCTAATACCAATGGACTCAATTCCCCAAATAAAGAGACAT...AAG'), id='7:47364999-47369999_strand_+_gene_Chr7-6Mb-seg-674', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCCTGATCGTCGACCTTCAGCTGCTCCCTACCTGCTGCTGTATTGAAGTCCCC...GCA'), id='7:47369999-47374999_strand_+_gene_Chr7-6Mb-seg-675', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGACATGAAATGGTGATGAGCGTGGATGTGGAAATGGAGATGAGATAAAGAAGT...TGT'), id='7:47374999-47379999_strand_+_gene_Chr7-6Mb-seg-676', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCACTGGCTTACTCTCTGTAGCTTGCTGTGCTTGCTTTCTTATACACCCCTGGA...AAA'), id='7:47379999-47384999_strand_+_gene_Chr7-6Mb-seg-677', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATAGACACTACCTCAGAATGAAAGGCTGGAAAACAATTTTCCAAGCAAATGGT...AGA'), id='7:47384999-47389999_strand_+_gene_Chr7-6Mb-seg-678', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACAAAGTTTGGAGCTGAGACAAAAGGATGGACCATCTAGAGACTGCCATATCC...CCT'), id='7:47389999-47394999_strand_+_gene_Chr7-6Mb-seg-679', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATACTATCCCACTCCCCCTGCCTCTATGAGAGTGTTCCTTCAACTACCCATCCA...TTT'), id='7:47394999-47399999_strand_+_gene_Chr7-6Mb-seg-680', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAAGACTTTCCCGTGAGACAAATGGCACCCAAAGATATTATTATACCAGTGGA...TGT'), id='7:47399999-47404999_strand_+_gene_Chr7-6Mb-seg-681', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTATTGACGGTGTCTTTTGCCTTACAGAAGCTTTGCAGTTTCATGAGGTCCCA...TCC'), id='7:47404999-47409999_strand_+_gene_Chr7-6Mb-seg-682', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTTGGCCTGGCTTTTTCTTTTCTCTACTGAAGCACATATGTCAGCACATTTCC...ACC'), id='7:47409999-47414999_strand_+_gene_Chr7-6Mb-seg-683', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGTGGAAACAAGAAGAACTATTCAAAGAATTAACCAAACGAGGAGTTGGTTTT...ACA'), id='7:47414999-47419999_strand_+_gene_Chr7-6Mb-seg-684', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTACTTGCTTTGGAAGGGCTAGCTTGAAAGTTTCTGTTTTTGTTGTTGCTTTT...CTA'), id='7:47419999-47424999_strand_+_gene_Chr7-6Mb-seg-685', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATATTTTTATGGATTGGTCTGATTAAAAGTTCATATAGAAAGATGTTAAGCCT...CTT'), id='7:47424999-47429999_strand_+_gene_Chr7-6Mb-seg-686', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGAAATCAGTCTGGCAGTTCCTCAGAAAATTGGACATAGTACTACCAGAGGAT...AAA'), id='7:47429999-47434999_strand_+_gene_Chr7-6Mb-seg-687', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAATGAATTTATGAAATTCTGAGGCAAATGGATGGATCTGAAGGATATCTTCC...CAA'), id='7:47434999-47439999_strand_+_gene_Chr7-6Mb-seg-688', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATACAAGAAGCCTACAGAACTCCAAATAGACTGGACCAGAAAAGAAATTCCTC...ATA'), id='7:47439999-47444999_strand_+_gene_Chr7-6Mb-seg-689', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACCAAAATTGTCAGAAATGCCAGATAAATAATTCAATAGCCCAGTGTTAAAGA...TTT'), id='7:47444999-47449999_strand_+_gene_Chr7-6Mb-seg-690', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGTGTACAGCTCAAGAAATCAAAGTCAGTATACAATAGAGACAATTATAACT...GGT'), id='7:47449999-47454999_strand_+_gene_Chr7-6Mb-seg-691', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCTGTGGAGGTCAGAACTGGGCATTAGTCCCCTGAAGCTGGAGTTATAGTCAG...CTA'), id='7:47454999-47459999_strand_+_gene_Chr7-6Mb-seg-692', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGCTTTCAGATGTGTTTTTAGGCTGGTAGTGTGTGCTCTCTCCAATTTCTTTA...AGG'), id='7:47459999-47464999_strand_+_gene_Chr7-6Mb-seg-693', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTTCCCTATGAACTCACCTTTTGTTTTCCTTTTTTGTTGGCTTTTTAAAAAAA...AAG'), id='7:47464999-47469999_strand_+_gene_Chr7-6Mb-seg-694', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCTACTAAACAAAACTGGAAAACCTGCATGAAATGGACAAATTCCTAGACAGA...CCC'), id='7:47469999-47474999_strand_+_gene_Chr7-6Mb-seg-695', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTGGGTGGAGTCTTCTGTCATGGTGTGGTGGGGTAGAGATGGGAGAGACAGCC...AAA'), id='7:47474999-47479999_strand_+_gene_Chr7-6Mb-seg-696', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAAGAAGTCAAAATATCACTTTTTGCAGATGATATGATAGTATATATAAGTGA...AAA'), id='7:47479999-47484999_strand_+_gene_Chr7-6Mb-seg-697', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAATAAATAACCAAATTCTTTACAAATAGAAATAGATTGAAATGATTTCTTTT...TTA'), id='7:47484999-47489999_strand_+_gene_Chr7-6Mb-seg-698', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCTAAACATCTTAATAGATGAGCTTATAGTTTTGATCAGGAAACATTCATTT...TGT'), id='7:47489999-47494999_strand_+_gene_Chr7-6Mb-seg-699', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTGGGGCATATAAAGTTTGCAATACCAAGGGGCATCTCTTCCCATTGATGGTC...GGG'), id='7:47494999-47499999_strand_+_gene_Chr7-6Mb-seg-700', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTTCCCAGCTCCTGGTTTTGTTGATTATATTTATAGTTCTTTTGGTTTCTAC...GCA'), id='7:47499999-47504999_strand_+_gene_Chr7-6Mb-seg-701', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTATGTTGGATTTAACAGACCTCATGGCATACTAACTATATAAGGTGATGAAT...AAA'), id='7:47504999-47509999_strand_+_gene_Chr7-6Mb-seg-702', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGTCTCTAATGAATGTGGCTGCCCTTGCATTTGGTGCGTAGATATTCAGAATT...TGG'), id='7:47509999-47514999_strand_+_gene_Chr7-6Mb-seg-703', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGGACTGAAGCTGAGGCTTGTGTGTGCTAAATAGACATACCACCCTCAGGCC...ACA'), id='7:47514999-47519999_strand_+_gene_Chr7-6Mb-seg-704', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGAGAGAGATAGAGAGGCAGAGAGAGAGACAGAGAGAGAGAGACAGAGAGAGA...AAT'), id='7:47519999-47524999_strand_+_gene_Chr7-6Mb-seg-705', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAAGGCAGGAACATGGCAGCATCCAGCCAGGTATGTCGCAGACAGAGCTGAGAG...AGA'), id='7:47524999-47529999_strand_+_gene_Chr7-6Mb-seg-706', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACTGTGGACTCTGAATGTAGCAAGCCTGGACTTGATATGTATTGTCAAGAGAT...GGT'), id='7:47529999-47534999_strand_+_gene_Chr7-6Mb-seg-707', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAAGACTCATTCAGAGTATAATGCAGGTTTGGCAAAGGCCACAGTTACCCACA...AGA'), id='7:47534999-47539999_strand_+_gene_Chr7-6Mb-seg-708', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAAGAAGGAATCTGCCAGAGCCAGATTGAGGACATAGTCTGAGATGGGCTATC...AGA'), id='7:47539999-47544999_strand_+_gene_Chr7-6Mb-seg-709', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAGAAAGTAGGGAAAAGCCTGGAAGATATGGCCACAGGGGAAAAATTCCTGAA...TTA'), id='7:47544999-47549999_strand_+_gene_Chr7-6Mb-seg-710', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGTTTATTATTTTCATTATTATTAACAAAATGTCACAGTGATTCCTCCATCAT...GTG'), id='7:47549999-47554999_strand_+_gene_Chr7-6Mb-seg-711', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAGAGGATCATCTTCTCTCTCCTTTTTCTTTACATTCAGGTACAGAACAATCA...ATT'), id='7:47554999-47559999_strand_+_gene_Chr7-6Mb-seg-712', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGACATATCGATTCTGGATGCATCTTGTGGTGCATCCAGAATACAAAAGTTGCT...TAA'), id='7:47559999-47564999_strand_+_gene_Chr7-6Mb-seg-713', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGCTATCCATCCTCTAGGCATATTTGAACAAACTGAGGAAGAGAGGGTGTCAG...CTC'), id='7:47564999-47569999_strand_+_gene_Chr7-6Mb-seg-714', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTGTCTCTCCCTCTC...CTT'), id='7:47569999-47574999_strand_+_gene_Chr7-6Mb-seg-715', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAGCTTCTTTGGTACTTTCTCTAGCTCCTTCATTGGGGGCCATGTGATCCATCC...CAT'), id='7:47574999-47579999_strand_+_gene_Chr7-6Mb-seg-716', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGCTTTCTTGGGCTGTGGATTAATGAGATACCAGGTAGATGTGAGAGACAATT...CCC'), id='7:47579999-47584999_strand_+_gene_Chr7-6Mb-seg-717', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCCATGTCTCCCCTTTTGGAAGCTCCCTATACCATCCTCACTTCCCCTGCCT...AAT'), id='7:47584999-47589999_strand_+_gene_Chr7-6Mb-seg-718', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCATTTCCTGTCAGCCCGACCAGTCCGAAGATGATGATCATCAAGTCTGGGAT...AAC'), id='7:47589999-47594999_strand_+_gene_Chr7-6Mb-seg-719', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAAAGCAGCAGTAGCAATAGTACCACCACCACCAGCAGCAGCAGCAGCGGCTGT...CCC'), id='7:47594999-47599999_strand_+_gene_Chr7-6Mb-seg-720', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACCCAAGTCGTTTTTGGTCATGGTTTCATCACAGCAATAGTCACCCAAAGTAA...TCT'), id='7:47599999-47604999_strand_+_gene_Chr7-6Mb-seg-721', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTACTTGTCTTATCCTCCTCAACATTAGATCTTAATACCCTAACATGCCATGCA...CAT'), id='7:47604999-47609999_strand_+_gene_Chr7-6Mb-seg-722', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTTCTACCTTCTCACTGCCAGTTGAACCATGTTGAAAATAATGCTGTCTTATT...CTA'), id='7:47609999-47614999_strand_+_gene_Chr7-6Mb-seg-723', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAATTTTATAAATTCATTGTTTTTAATAGCTGAGTAGTACTCCATTGTAAATG...TCC'), id='7:47614999-47619999_strand_+_gene_Chr7-6Mb-seg-724', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTTTCCTGACTTTTTCTTAATTACAAAAATTGGAGTATTCCAAGGTGAGGTAG...AGA'), id='7:47619999-47624999_strand_+_gene_Chr7-6Mb-seg-725', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTTTTAAAGAAATATATACACAATTTGCTTGAATTAAGCTTAAGAAATGAAA...ATA'), id='7:47624999-47629999_strand_+_gene_Chr7-6Mb-seg-726', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGAAGAGGTTATGGATTACATGGGGAAGCACATGTGAGGGGGACAAAGAGGAG...TTT'), id='7:47629999-47634999_strand_+_gene_Chr7-6Mb-seg-727', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATCAGCAAACATTCATTGTCATTTTAGAATTTGTCTCTGTTAACAATTATACA...GTA'), id='7:47634999-47639999_strand_+_gene_Chr7-6Mb-seg-728', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTTATATTGTTTTGATTATTGTTAAAGGATGTTGTTTCCCTAATTACTATCTC...TCA'), id='7:47639999-47644999_strand_+_gene_Chr7-6Mb-seg-729', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGATGTGCTTTGATAGGTAATTTTATGTGTCAACTGGGTCAGGTTATGTTGTA...TTG'), id='7:47644999-47649999_strand_+_gene_Chr7-6Mb-seg-730', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCCTTCAATATAGTCATATTGAGTAAGATTCGAGAAAGTAGATACCCTTGTCT...GCG'), id='7:47649999-47654999_strand_+_gene_Chr7-6Mb-seg-731', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGGTCACTTGTTCAGTCAACAAAAGGCTCTGATAATGTGTCATATGAGCATTG...ATG'), id='7:47654999-47659999_strand_+_gene_Chr7-6Mb-seg-732', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTATCTTACAATAGACAGGCATGGTGTTGGAGCAGTAGCTGTGAGTTTTACAA...CTC'), id='7:47659999-47664999_strand_+_gene_Chr7-6Mb-seg-733', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTCCTCTCCTCTTCTCTTCTCTTCTCTTCTCTTCTCTTTTCTTAGATATTTT...TGC'), id='7:47664999-47669999_strand_+_gene_Chr7-6Mb-seg-734', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGGAGTGATATTGCTGGATCTTCTGTTAGTACTATGTCCAATTTTCTAAATA...TGG'), id='7:47669999-47674999_strand_+_gene_Chr7-6Mb-seg-735', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAATTGTTTTCCAGTCTTTCATTCTGAGGTAGTGTCTGTCTTTTTCTCTGATA...TGG'), id='7:47674999-47679999_strand_+_gene_Chr7-6Mb-seg-736', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTAAGGGTGATCTTCAATAACAACATAAATAATGGAAAGCCAACATTCACGTG...TAA'), id='7:47679999-47684999_strand_+_gene_Chr7-6Mb-seg-737', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTAAGCAGAATCATTTTTAGTAGAACCAATCCCTCTGGCAGCTTGAGGAATACC...GCC'), id='7:47684999-47689999_strand_+_gene_Chr7-6Mb-seg-738', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGAATTTAATAAAAAGAAACACAGGAAGTTTGGATGTCTCTCATGTCAACCAA...CAA'), id='7:47689999-47694999_strand_+_gene_Chr7-6Mb-seg-739', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGGTTATTTCAGATTGAGAAATACTGAGGGTATTGGAGAACTAAAGAATTAT...TGT'), id='7:47694999-47699999_strand_+_gene_Chr7-6Mb-seg-740', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGATTTAGGTTAAAAATTCTCTTGGAAATAGGACAGAACCTGGTGTGTACCTGT...TTT'), id='7:47699999-47704999_strand_+_gene_Chr7-6Mb-seg-741', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTAGCATTTCCCTGATGACTAAGGATGTTGAGCATTTCTTTAAGTGTTTCTT...CCT'), id='7:47704999-47709999_strand_+_gene_Chr7-6Mb-seg-742', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAATTGACAGGTATTGTCAAGAGACTTGGTTGAATGTGGGGAGGCTATTGGAG...ATC'), id='7:47709999-47714999_strand_+_gene_Chr7-6Mb-seg-743', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTTACTATGTTCTCTTCCTTTTCTCCTGCTACTCTACATGAAGCACCCCTGTT...ACA'), id='7:47714999-47719999_strand_+_gene_Chr7-6Mb-seg-744', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGACACTGGTCTGTGCAACAGATTACAGATCACATGCATTTACCTCCTCCCTTG...CTA'), id='7:47719999-47724999_strand_+_gene_Chr7-6Mb-seg-745', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAACTTCACTTGAAGTACTCCCGGAGGTGTAGCGAATATAAGAATTTGGTCACA...CAT'), id='7:47724999-47729999_strand_+_gene_Chr7-6Mb-seg-746', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGTATCACATTTTCTGTATTCATTACTTTGTTGAGGGACATCTTGGTTCTTTT...TTC'), id='7:47729999-47734999_strand_+_gene_Chr7-6Mb-seg-747', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTATCCATTCCTCTGGTGAGGGAAATCTAGTTTTTTTCCAGCTCCTGGGTATT...GGT'), id='7:47734999-47739999_strand_+_gene_Chr7-6Mb-seg-748', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGGACTGAAGCTGAGGCTCATGTGTGCTAAAAACTCTTACTACACTCAGGGCT...TAT'), id='7:47739999-47744999_strand_+_gene_Chr7-6Mb-seg-749', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACCCAATTAGAAATAATTTGAAATGATTTCTATGTCTTACAAGATCATCGTAG...GTT'), id='7:47744999-47749999_strand_+_gene_Chr7-6Mb-seg-750', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGCAGAAATTTTTTTTTTAATATTGGTGAAGAAATGCTACAGCAACAAGAAAA...TGA'), id='7:47749999-47754999_strand_+_gene_Chr7-6Mb-seg-751', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTTTCTCTTTGTTTCTAATTGTGACTTTCTTCACAATTTTGACTGCCAATTCT...ATT'), id='7:47754999-47759999_strand_+_gene_Chr7-6Mb-seg-752', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAGTGTAGAAAGGTTTATTTGGGCTTAGGGTTCAGGGGATAGAGTCCATCAGA...AGC'), id='7:47759999-47764999_strand_+_gene_Chr7-6Mb-seg-753', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACATGAATGGATTCCATTCTTTCTCCCAGGAGGTCTTTAGTGGTATTTACTTT...GGG'), id='7:47764999-47769999_strand_+_gene_Chr7-6Mb-seg-754', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCATCAATAATCGATGACGGCCAAGACTGACGATTCACCGATGGCTTAGTTTCA...CAG'), id='7:47769999-47774999_strand_+_gene_Chr7-6Mb-seg-755', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAACCATGGGGCTCCTTTGTATTTTTTTTCTTTTAAAATAATTAATTAGAGGA...TCA'), id='7:47774999-47779999_strand_+_gene_Chr7-6Mb-seg-756', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCCACAGATAAAGAATTCTGCAGACAAAAGGCACAGATACTAGGTGGAACCAG...CAC'), id='7:47779999-47784999_strand_+_gene_Chr7-6Mb-seg-757', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAAGTTACCTCAGGTCTTCAGGAGCACTACATGATTGCATTCTCTGGTCATAA...CTT'), id='7:47784999-47789999_strand_+_gene_Chr7-6Mb-seg-758', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATAGTCCTCAGGGATGTATCTAAAGCTGGTCTCTCTGATATAATAATAATCTT...TTC'), id='7:47789999-47794999_strand_+_gene_Chr7-6Mb-seg-759', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAGATTCGGTTTGCCGCTCTAAAAGAAATTATGCTGCGTTATGCCGTGGGGTG...GAT'), id='7:47794999-47799999_strand_+_gene_Chr7-6Mb-seg-760', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTCATACCAATCCATTGTACTGAGCACAGAGTTCCCTATGAAGGAGCTAGAGG...TGT'), id='7:47799999-47804999_strand_+_gene_Chr7-6Mb-seg-761', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAAAATCGGCATGGTACTGGTATAGCGACAGACAAGTAGACCAAGGGAATTGAA...GCC'), id='7:47804999-47809999_strand_+_gene_Chr7-6Mb-seg-762', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTTTCTCAGGGACACCACTGGAAGATTTTTAAACATGGATCCAAACATCTCA...TCC'), id='7:47809999-47814999_strand_+_gene_Chr7-6Mb-seg-763', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGATGTTCCATGGGTCATAGACACAGAAGCTATGATTTAGATGTATTCATTAG...GTG'), id='7:47814999-47819999_strand_+_gene_Chr7-6Mb-seg-764', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTATCCTGAAAGTTCCCTATACTGCCCCCCACCCTGCTCTCTGACCCACCCACT...TAC'), id='7:47819999-47824999_strand_+_gene_Chr7-6Mb-seg-765', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTGGATGAATCCCCAGGTGGAACAATCTCTGGGTGACTTTTCCTTTAGTGTCT...TGA'), id='7:47824999-47829999_strand_+_gene_Chr7-6Mb-seg-766', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATAATCTTCCCATAATCTCCTTCCTTTATTTTGCTCATTAGTGAAATTCATGAT...AGA'), id='7:47829999-47834999_strand_+_gene_Chr7-6Mb-seg-767', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAGTGGAGTGAGAAGATAATCTGTGGAAATGATATACATATAGTACTCATGTA...AAA'), id='7:47834999-47839999_strand_+_gene_Chr7-6Mb-seg-768', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATTCATTTTCTTGTCTTTTTCTTCCAATGCTGGGTTAATAGGCATATGCCACC...ATT'), id='7:47839999-47844999_strand_+_gene_Chr7-6Mb-seg-769', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTTTAGGAGATATGGGACTGTTTAGATGATTAATCTCATTCTGACATAACTTT...ATA'), id='7:47844999-47849999_strand_+_gene_Chr7-6Mb-seg-770', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATATATACATATATATATATATAAAATCATGAATTTATACTGATAATTCTGTT...TTC'), id='7:47849999-47854999_strand_+_gene_Chr7-6Mb-seg-771', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCAGGCTTTCACTGCTGGACTTGGATACACCCCTTCTGGACTTAGCTTCATCC...TGA'), id='7:47854999-47859999_strand_+_gene_Chr7-6Mb-seg-772', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAAATGTCTCCATGGGATCCAGTTGTAAGGCATTTTCTCAATTAGAGATCCAT...AGT'), id='7:47859999-47864999_strand_+_gene_Chr7-6Mb-seg-773', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACACTCGTATAAGAATTTAGCAAGGCCTAGGAAATGGGGGAGGTTGTGATATT...TCT'), id='7:47864999-47869999_strand_+_gene_Chr7-6Mb-seg-774', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCTGTCCCAAAAGCTTTGTAGCTTCTGTAGTTGGCATTCTCACCTACACAGAC...TCC'), id='7:47869999-47874999_strand_+_gene_Chr7-6Mb-seg-775', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGCCCAGCTTGCCTGATAGGAATAATGGTTTTTGTGTTTCACTGGAGGTGATA...GGG'), id='7:47874999-47879999_strand_+_gene_Chr7-6Mb-seg-776', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGCACCACCCACAGTGGGAATGGCTCTCCCACATCATTCATTGATTAAGAAAA...ATG'), id='7:47879999-47884999_strand_+_gene_Chr7-6Mb-seg-777', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGCAATGCTGGAGATAGAAACCCTAGGAAAGAAATCTAAAACCATAGATAAGA...TGA'), id='7:47884999-47889999_strand_+_gene_Chr7-6Mb-seg-778', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTATCCATACTTTGGTTGTCCTTCTTCAGTTTCATGACTTTTCCAAATTGTAT...CTG'), id='7:47889999-47894999_strand_+_gene_Chr7-6Mb-seg-779', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGCAATCCCACTAAAATCAGGGGCTACACAAGGCTGTACACTCTCTCCCTAT...TTC'), id='7:47894999-47899999_strand_+_gene_Chr7-6Mb-seg-780', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTTTGAAGTTATAAGGTGGGATGCTGCCAGAACAGAAGGATCTCAAAGTAGGC...TTC'), id='7:47899999-47904999_strand_+_gene_Chr7-6Mb-seg-781', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATACCAAGGGGCCTCTCTTCCCAATGATGGCCGAATAGGACTGATATAGCTGT...GAG'), id='7:47904999-47909999_strand_+_gene_Chr7-6Mb-seg-782', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGAAGGTAAATCCTTTTGTTTTAGGATAAAATGTTCTGTAGATATCTGTTAAG...TCA'), id='7:47909999-47914999_strand_+_gene_Chr7-6Mb-seg-783', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGACAAGAGAGTAATTGTGCTGAACCTAAGCTATTACCATGGCAACAAAAGTT...AAG'), id='7:47914999-47919999_strand_+_gene_Chr7-6Mb-seg-784', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTAGAAAGTCTGGCAGTGTGTAGCAGTGTAGCATTCTTTACAGAGGTGAGCCTC...CAA'), id='7:47919999-47924999_strand_+_gene_Chr7-6Mb-seg-785', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAAAGAAAAGACAGTTGGATATAACTGCAACTTTTGGGAACTCTGGGCCACTAT...GGA'), id='7:47924999-47929999_strand_+_gene_Chr7-6Mb-seg-786', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGGATGATGCTCATTTTTTTGTCTCGTGGTCCTAATGACAGCTTGAATATGCT...GTC'), id='7:47929999-47934999_strand_+_gene_Chr7-6Mb-seg-787', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCGTTTGGAAGCTGATTATGGGATGGATCCCCAGGTATGGCAGTCTCCAGATG...GGA'), id='7:47934999-47939999_strand_+_gene_Chr7-6Mb-seg-788', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAAGGGCCTCTCCTCCCATTGATGACTGATTAGGTCATCCACTGCTACATATG...CAG'), id='7:47939999-47944999_strand_+_gene_Chr7-6Mb-seg-789', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAAATACAAAAAGATTCAGCATGTAGGGTCATGTGATTTAGCTTAAAGATGAA...CTT'), id='7:47944999-47949999_strand_+_gene_Chr7-6Mb-seg-790', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCCCAGGGATGCATGGATGGTTTAATATATGGAAATCTATCAATAAAATGCA...CCA'), id='7:47949999-47954999_strand_+_gene_Chr7-6Mb-seg-791', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGCCCACAGCCAGAACACAACTGCATTTTCTGCTAGCCCAGCCCAGGCACTG...GAG'), id='7:47954999-47959999_strand_+_gene_Chr7-6Mb-seg-792', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGAAGTCTCATTTAGAGGATATTGCAGGGTTGGCACAGGCTAAAGTTACCCA...GGA'), id='7:47959999-47964999_strand_+_gene_Chr7-6Mb-seg-793', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGGTAAAGATCTTTTACCAATCTGTTGGTTGCCATTTTGTCCTATTGACAGTA...AAA'), id='7:47964999-47969999_strand_+_gene_Chr7-6Mb-seg-794', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATAGTTTTTTATTTTTATTTACTTATTTATTTATATAGATTTATTTATCTCCA...ACA'), id='7:47969999-47974999_strand_+_gene_Chr7-6Mb-seg-795', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAACTTACGGCAATTAAGCAACATTTAAATTTCTATCTATTCTCCAGAATGCT...TAG'), id='7:47974999-47979999_strand_+_gene_Chr7-6Mb-seg-796', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTCCAGCACCATGACTCTCTTGTTAAATAAGGGAGTTATTTATGTCTTTCTTAA...TCT'), id='7:47979999-47984999_strand_+_gene_Chr7-6Mb-seg-797', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCATTTCTCCATTTATCCCCATTGATATGGTGATGTTTACTCTGTGTCACAGTT...ACG'), id='7:47984999-47989999_strand_+_gene_Chr7-6Mb-seg-798', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCACCACAGACTCCCAAGAGATCCTACATCTTAGAGGAAGAAAGATAGCTCCA...AGG'), id='7:47989999-47994999_strand_+_gene_Chr7-6Mb-seg-799', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGATTGGACTGGTCTGGATTTTCTATCATTACTCTCCCAGTTTTAAAATCTAT...GCA'), id='7:47994999-47999999_strand_+_gene_Chr7-6Mb-seg-800', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGGGACAAAATCCTAATTAACAAAATCAGAAATGAAAAGGGAGACATAGCAAC...ATT'), id='7:47999999-48004999_strand_+_gene_Chr7-6Mb-seg-801', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTAGAAGGAAGTTTGTACCCTGCAGATGTCTTCCCCCTTTGTTTTTTTCTAGA...TGA'), id='7:48004999-48009999_strand_+_gene_Chr7-6Mb-seg-802', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTCTTCTGTGACTCGGTTACCTCACTCAGGATGATATCTTCCAGATCCACCCA...TAG'), id='7:48009999-48014999_strand_+_gene_Chr7-6Mb-seg-803', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGAGGTAAGTGTTTTAACGTATATAACTTATAATGTATACACTTATTCCATTA...CTA'), id='7:48014999-48019999_strand_+_gene_Chr7-6Mb-seg-804', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCCCCCACCCACATCCCACCACAACATGCACCTATCCCCCACCAACTCCCCCT...CCC'), id='7:48019999-48024999_strand_+_gene_Chr7-6Mb-seg-805', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGGAAAAGAGGAAGAGAAATTAAAACAAAACCACAAAAGGAATGCTTAATTTG...GCA'), id='7:48024999-48029999_strand_+_gene_Chr7-6Mb-seg-806', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGTCCCTTCTCCCAATTTCATTTTGTTCTTTCCTTCTTTCGAAAAAAAAAATC...TTG'), id='7:48029999-48034999_strand_+_gene_Chr7-6Mb-seg-807', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTGTGAGCATGTGATTTTAAGGCTGAAAGCTTGATATTGTATAGCCAAGTAGG...TAG'), id='7:48034999-48039999_strand_+_gene_Chr7-6Mb-seg-808', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTTAGGGTTAGGGTTAGGGTTTCTCCCAAGGTTGAAGGTCTCTTCACTGTTAG...CAG'), id='7:48039999-48044999_strand_+_gene_Chr7-6Mb-seg-809', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAACTGATGTATAATAAGATACCTGTATATCAAGAAGGAAATAATGTAATAGT...ATG'), id='7:48044999-48049999_strand_+_gene_Chr7-6Mb-seg-810', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGTTGAACAACAATCTACTCGATGATAACTTGGTCAAGGATGAATTAAAGAA...GAG'), id='7:48049999-48054999_strand_+_gene_Chr7-6Mb-seg-811', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGTGTGGGGAGTCCCATCCTTCTTCAACTGGGGGGGGGGTCATGCCTATTTAT...GGA'), id='7:48054999-48059999_strand_+_gene_Chr7-6Mb-seg-812', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGCATTCGATTGTATATATACCATATGTTCATTTTCCAGACATCAGTTGAAGA...GAA'), id='7:48059999-48064999_strand_+_gene_Chr7-6Mb-seg-813', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTTTTTAAAAATCCAGTTTGTACTATTTGGTCTATCTGCAGACCTATATAATA...CAG'), id='7:48064999-48069999_strand_+_gene_Chr7-6Mb-seg-814', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCTTCAAAGAAGACCTAATTCCAACTCTCCTCAAACTATTCCACAAAATAGAA...TAG'), id='7:48069999-48074999_strand_+_gene_Chr7-6Mb-seg-815', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAGATGCAGTTATTTGCACCCATCCAATGGACAGAAGCAGCTGACCCCTGTTG...GAG'), id='7:48074999-48079999_strand_+_gene_Chr7-6Mb-seg-816', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTTTTTTTTAAATCTCCCTCTGCTGTGTTGTATATCAACAGCAGCAGAGATTT...TAT'), id='7:48079999-48084999_strand_+_gene_Chr7-6Mb-seg-817', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGAGCCAGGAGAGAAGGACCAATTGTTGTGTATGCCTTCTGCTCATGAAGAA...GAC'), id='7:48084999-48089999_strand_+_gene_Chr7-6Mb-seg-818', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACGATAATACTATAGGCCTCTTGAAAAGCTATATTTAGAGCTAAGGCTTCCAA...GAG'), id='7:48089999-48094999_strand_+_gene_Chr7-6Mb-seg-819', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGTTACAGGAAATCCAAGGCTACACAGAGAAACCCTGTCTCAAAAAATAAAT...AGC'), id='7:48094999-48099999_strand_+_gene_Chr7-6Mb-seg-820', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTTGTCTTTCTGGATATGACTTAAATATGCAAAGAAATTGAGGCAAACAATTT...GAC'), id='7:48099999-48104999_strand_+_gene_Chr7-6Mb-seg-821', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTGCTACAGGATCCTTCTGCACACGTTTATTCAGTCCTGTTTCTTCTTTCTCT...AAA'), id='7:48104999-48109999_strand_+_gene_Chr7-6Mb-seg-822', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAACTTCTCTACTGCTAGTGAATAATGTGATGTCCAGAGCATAGCTACTCAGCA...CCA'), id='7:48109999-48114999_strand_+_gene_Chr7-6Mb-seg-823', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAACCTGTTAGGTAAAAATGATGGTGTGTGTGTGTGTGTGTGTGTGTGTGTGT...CAT'), id='7:48114999-48119999_strand_+_gene_Chr7-6Mb-seg-824', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAATGCTGCTCAGTCACTGTTTGCCAGCCCATGCAGCAGCAGTGTTCCTAGTGT...GTC'), id='7:48119999-48124999_strand_+_gene_Chr7-6Mb-seg-825', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACAACACAGGGTTGCGCCAAACAGGCCCACTCCCACCCACTCTTTAAAATGTG...ACT'), id='7:48124999-48129999_strand_+_gene_Chr7-6Mb-seg-826', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTAGTGTCCGCCTGTTGTGAGCTGGGATCCCATATGTTTTGGGGCCCTGGGACC...CAG'), id='7:48129999-48134999_strand_+_gene_Chr7-6Mb-seg-827', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTTTGTCTCTGTATCTCCTTCCATGGGTGTTTTGTTCCCACTTCTAAGAAGGG...ACT'), id='7:48134999-48139999_strand_+_gene_Chr7-6Mb-seg-828', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCGTGCTCAAGATGGCACAGGGTTGGATTATTATTTTTAAGAAGGCTTTAAGTT...TTT'), id='7:48139999-48144999_strand_+_gene_Chr7-6Mb-seg-829', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGTTGATGAAAATCAGTACATTATCCACTCTGATTTTTGTTAAGGGAGTTGAA...TTG'), id='7:48144999-48149999_strand_+_gene_Chr7-6Mb-seg-830', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGTCAATTTTATCCGATATTAGAATGGCTACTCCAGCTTGTTTCTTCAGACC...ATT'), id='7:48149999-48154999_strand_+_gene_Chr7-6Mb-seg-831', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAATATATCAGCATTTAAGATTAGAACATTGTTTTATAGGAAGTGTTATCTCAG...ACA'), id='7:48154999-48159999_strand_+_gene_Chr7-6Mb-seg-832', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATAGAGCACATTACACCTTTAAGACCAGTAAAAAAACAAACAAACAAACAAAC...AAT'), id='7:48159999-48164999_strand_+_gene_Chr7-6Mb-seg-833', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATTATTTTCTGAGGGAAAACAGCTTGATCTATAAGCCTAAAACCTAAATAATT...AGG'), id='7:48164999-48169999_strand_+_gene_Chr7-6Mb-seg-834', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAGTTCCTGTGCTTGCGGTTGTTTATCCTGAACAGTATTTTGTAATGAGAGCT...TGA'), id='7:48169999-48174999_strand_+_gene_Chr7-6Mb-seg-835', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCACACTTCGACTGTAGTCAATAAGAACTATGCTTCCCTGCACACACTTCAGTG...ACT'), id='7:48174999-48179999_strand_+_gene_Chr7-6Mb-seg-836', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGGTTCTATAGACTCCCAACCCTCAAACACTGCAGGCCAATGCTCTTCATCAG...TTC'), id='7:48179999-48184999_strand_+_gene_Chr7-6Mb-seg-837', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCCAACAACTCTGGAGCACATCCTTCTAAGCAACACACAAAAATTCCACTGCA...TAT'), id='7:48184999-48189999_strand_+_gene_Chr7-6Mb-seg-838', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAATATAGTACTTGAAGTCATAGCCAGAGCTATTCGACAACAAATTGCTAAGG...ATC'), id='7:48189999-48194999_strand_+_gene_Chr7-6Mb-seg-839', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCACTCAATATACATTTTCTAGATTTACCTGCAAACCTCCTGATTTCATTTTTT...GAT'), id='7:48194999-48199999_strand_+_gene_Chr7-6Mb-seg-840', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAAACTGTCTGGAAAAGAGAGCAGTGTAGCTGGACAGGTAGCATCATGTAGAA...CTG'), id='7:48199999-48204999_strand_+_gene_Chr7-6Mb-seg-841', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCTTCCCTACACTGACACTCTGACATGGATGAGAGTCCTTGCCCACAGAACA...AGG'), id='7:48204999-48209999_strand_+_gene_Chr7-6Mb-seg-842', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTAATATAGTAAAAATGGTTATCTTGTTAAATGCAATCTACAGAGTCAATGCA...GTC'), id='7:48209999-48214999_strand_+_gene_Chr7-6Mb-seg-843', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACAAGATAGATAAACCCTTAGCCAGATTCATTAGAGAACACAGGGACAGCATC...TGT'), id='7:48214999-48219999_strand_+_gene_Chr7-6Mb-seg-844', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATATGCTGTAACATCATTTTGCTTATGCTCAGGAGTGGTATAGCTGGGTTCTC...GGT'), id='7:48219999-48224999_strand_+_gene_Chr7-6Mb-seg-845', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGGGAATGCTGTGGTTTTATTCTCATTTAGTTATAAAAGGTTTTACTTTTTCT...AAA'), id='7:48224999-48229999_strand_+_gene_Chr7-6Mb-seg-846', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGGAAAAACATAAATCCCTAAAAGAAATACAGTAAAACACAATCAAATAGGTG...CAA'), id='7:48229999-48234999_strand_+_gene_Chr7-6Mb-seg-847', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGGATATAAATTGGAAAGGAAGTCAAAATATCACTATTTGCAGATGATAAGAT...CCA'), id='7:48234999-48239999_strand_+_gene_Chr7-6Mb-seg-848', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAGGATCTAACCTGAAAAATCCCATGACACGTGCCTCTTAAGGAAGTGAGTCCA...AAC'), id='7:48239999-48244999_strand_+_gene_Chr7-6Mb-seg-849', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATAATTTACAAACATGTCTTTTCACTAACTCTACTTTCTCACACTGGCACCTCT...GGT'), id='7:48244999-48249999_strand_+_gene_Chr7-6Mb-seg-850', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTTCTCATTTCTGCCTGATAGAGAAAAAAGACTCTATGTTTTGGGTGTCATA...ATC'), id='7:48249999-48254999_strand_+_gene_Chr7-6Mb-seg-851', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGCCAAAAGCAATCTACAGATTCAATGCAATCCCCATCAAAATTCCAACTCAA...GTA'), id='7:48254999-48259999_strand_+_gene_Chr7-6Mb-seg-852', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCCAGACCCAAAAGGACATGCATGACAGGAGCCTAGCATGGCTGTCCTCTGAG...GGT'), id='7:48259999-48264999_strand_+_gene_Chr7-6Mb-seg-853', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTGGTAATAATTTTGGAACATTCTCAATCAATTTTAAGTGTTTTTAAATGTT...GAA'), id='7:48264999-48269999_strand_+_gene_Chr7-6Mb-seg-854', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACAATACCTATGGAAGGGGTTACAGAGACAAAGTTTGGAGCAGAGCCTGAAGG...AGT'), id='7:48269999-48274999_strand_+_gene_Chr7-6Mb-seg-855', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTTTTCATATGCAAACATACTCTGCTTTTGGACCTAGGAGATCATCTGCATTT...AGT'), id='7:48274999-48279999_strand_+_gene_Chr7-6Mb-seg-856', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGGAACAGACCTAGCTTATGACCTAGATATTTATCAATAAATGATAACTCTC...AAG'), id='7:48279999-48284999_strand_+_gene_Chr7-6Mb-seg-857', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGAGGAAACCAGCTGGGCAAACTCCAAACTGTATCTCCATGGCTGACATCAAAG...ATG'), id='7:48284999-48289999_strand_+_gene_Chr7-6Mb-seg-858', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTATTCTTTGGTTGGTGCTTTGGTCCCTGCAAGTACTTGGAATACTGGTTAGT...GGT'), id='7:48289999-48294999_strand_+_gene_Chr7-6Mb-seg-859', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTACCTACCTTGGGCTATGCTTGCTTTCAATAACTTATTTGAAATATAATTCC...CCA'), id='7:48294999-48299999_strand_+_gene_Chr7-6Mb-seg-860', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTTTACAATGTTGGTCCTGCCAATCCATGAGCATGGGAGATCTTTCCATCTTC...GGC'), id='7:48299999-48304999_strand_+_gene_Chr7-6Mb-seg-861', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACCGCTGTTGCTGTGGCTTAGGTCGCCTCCCCAGCCGGGCGGGCACCTGTCCT...CCC'), id='7:48304999-48309999_strand_+_gene_Chr7-6Mb-seg-862', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCATATGCCTCATTCATTATTTCCATATTTTTAAATCTCATAGCCTCCCTTAT...TTC'), id='7:48309999-48314999_strand_+_gene_Chr7-6Mb-seg-863', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCGGTCCTCTCCCCACCTCTCTTCCACCCACATCCTTCCCTCCTGACTCCCAT...TGG'), id='7:48314999-48319999_strand_+_gene_Chr7-6Mb-seg-864', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGCCATGGAACCCACCTCTCCAGGGGGTGGACAAGGGTCTGTCCAAGGTCTC...TAA'), id='7:48319999-48324999_strand_+_gene_Chr7-6Mb-seg-865', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTTATGATCTAAGCAGTTATTCATACTTCGTGACTGTATGCCTTTTATTTAGC...GTT'), id='7:48324999-48329999_strand_+_gene_Chr7-6Mb-seg-866', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTATTCAGGAATTTTTCCCCGTGCCCATCTCTTCGAGGCTTTTCCCCACTTTCT...AAC'), id='7:48329999-48334999_strand_+_gene_Chr7-6Mb-seg-867', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAGTCAAGATGTAATGTTTTAGACCTCCATGAGTTCCTCTAATACCTCCAAAT...CCA'), id='7:48334999-48339999_strand_+_gene_Chr7-6Mb-seg-868', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCAAACATGACTAAGAAGGGTTGATAGCTCCTGGTTCCAAGCTGTTTGTTCTT...AGG'), id='7:48339999-48344999_strand_+_gene_Chr7-6Mb-seg-869', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCTTCTACAAATTACCAGAGTCCTGGAGGTGAGACACTCTCAGGACTCATTGG...AAG'), id='7:48344999-48349999_strand_+_gene_Chr7-6Mb-seg-870', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGATAGCCCCTCTGGCTATCTATCCCTTGCTTCAGAGAAGATGAGTGGGGATGA...GCA'), id='7:48349999-48354999_strand_+_gene_Chr7-6Mb-seg-871', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACATTGCACCTCACACAATAATAGTGGGAGACTTCAACACACCACTTTCACCA...TTT'), id='7:48354999-48359999_strand_+_gene_Chr7-6Mb-seg-872', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGTTGTTCAGAATTGTTTCAGTATCCTTGATTTTTTCTTATTCCATACGATAA...GTC'), id='7:48359999-48364999_strand_+_gene_Chr7-6Mb-seg-873', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCATGCTTTTTAAAATTCCAATAATTTAAATATTAAGGTCAATTTAAAGATCTA...AGG'), id='7:48364999-48369999_strand_+_gene_Chr7-6Mb-seg-874', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCAAATGCTGCATAATATCCCTTCCCCAGAGGTTAACCGGGAGTGGGAGCACA...ATC'), id='7:48369999-48374999_strand_+_gene_Chr7-6Mb-seg-875', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGAGACTGCCGAACCTGGGGATCTATCGCATAAACAATCACCAAACCCAAACA...TCT'), id='7:48374999-48379999_strand_+_gene_Chr7-6Mb-seg-876', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGTTTGGATCTGTAGCCAACAGGAAGAGACAGCGACACTAGCCTTGGTTTCAG...TCC'), id='7:48379999-48384999_strand_+_gene_Chr7-6Mb-seg-877', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTTGTGTAGCCAGTCTATTAGTCTATGTCTTTTTATTGGGGAATTAAGTCCA...CCA'), id='7:48384999-48389999_strand_+_gene_Chr7-6Mb-seg-878', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATTTGTAGTTTCCTTTAGTATTTGTATGACTTAGAATATTCTCTAACTTAATA...CTA'), id='7:48389999-48394999_strand_+_gene_Chr7-6Mb-seg-879', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGGGTTCCCCATCTCCTAACTTCCAAGGTTGCCTGTATTCCATTCTTTCTGCT...TAC'), id='7:48394999-48399999_strand_+_gene_Chr7-6Mb-seg-880', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATTTATGCTTCCTCTATTCCTTCTTATAGTGAACTAATTTATTATCAGTGAGG...GTT'), id='7:48399999-48404999_strand_+_gene_Chr7-6Mb-seg-881', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGAGCCGCCCCCACATTCGCCGTCACAAGATGGCGCTGACATCCTGTGTTCTA...CCT'), id='7:48404999-48409999_strand_+_gene_Chr7-6Mb-seg-882', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGGAGCACACTGCAAGCACTGGAACCCTGCTAGAGGAGTAAATTCAACAACAC...CAT'), id='7:48409999-48414999_strand_+_gene_Chr7-6Mb-seg-883', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTACTAAGGCTGATCTTCAATAACAAAAAAAATAACAGAAAGCCAACACTCACG...ACC'), id='7:48414999-48419999_strand_+_gene_Chr7-6Mb-seg-884', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAAGAAATCTACAGGTTCAATGCAATTCTCATCAAAATTCCAACGCATTTTTT...AAT'), id='7:48419999-48424999_strand_+_gene_Chr7-6Mb-seg-885', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAAGTACAAACAATAAAGAAAACCCAAACAGAGACAATTCCAGAGATAGATAT...TAA'), id='7:48424999-48429999_strand_+_gene_Chr7-6Mb-seg-886', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCACTAAATAAACAAATTCAAAGACAAAATTGACATGATCATCTCTTTAGATG...CAA'), id='7:48429999-48434999_strand_+_gene_Chr7-6Mb-seg-887', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCTATTTGGTGTTCTATAGACTCCTTGTGCATTTATGTACATTTCTTTAGGTT...TAC'), id='7:48434999-48439999_strand_+_gene_Chr7-6Mb-seg-888', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGTTTTGAAGGCATGAAGGGTTCATTAAGAACGACTGAGGCTTGGCACTGTGA...TCT'), id='7:48439999-48444999_strand_+_gene_Chr7-6Mb-seg-889', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATATAATGACTATATATGACTATATATAAGGAAATAATGCACTTCTTATCAAAT...AGA'), id='7:48444999-48449999_strand_+_gene_Chr7-6Mb-seg-890', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGACTCTCAGGCACAGAAGATATGGTAGAAGAAATCGATACATCAGACAAAGA...CCC'), id='7:48449999-48454999_strand_+_gene_Chr7-6Mb-seg-891', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCTGGTTGGGTCTCAGTGGGAGAGAAGGTGTCTAGGCTTCCTGGAATTTGAT...TGG'), id='7:48454999-48459999_strand_+_gene_Chr7-6Mb-seg-892', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAATACAAACTGTGTGAAGGCTCATGGAAGACATAATATATGTTCATTTTCTAT...TTT'), id='7:48459999-48464999_strand_+_gene_Chr7-6Mb-seg-893', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTAAAAGAAACATTATTAAAGCTTAAATCACAAATCAAACCCCACAGTTTAATA...AAG'), id='7:48464999-48469999_strand_+_gene_Chr7-6Mb-seg-894', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCAAACCTCAAAGCATATGAGGAAATTAACAAGACACCTGAAATCTCTAGAAC...GTT'), id='7:48469999-48474999_strand_+_gene_Chr7-6Mb-seg-895', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAAAATTTCTGCCTCTGTTCCAAAAAAGTAGAGAAGACCAGAGTTCTAGCCCA...AGC'), id='7:48474999-48479999_strand_+_gene_Chr7-6Mb-seg-896', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCAAAATTAAAAATCCTGCTTCTCCTTTTCTCTTCAATTGTACATTTTATATT...CTC'), id='7:48479999-48484999_strand_+_gene_Chr7-6Mb-seg-897', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTACACACCTTCATAGGACCTGTTAATATATGTATGATTTCCTCTCTGCACCC...TTT'), id='7:48484999-48489999_strand_+_gene_Chr7-6Mb-seg-898', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTACTGTTATTTGTGGAAACACAACAGATTGATTTTGGGCAAACTGTAAGATT...TAT'), id='7:48489999-48494999_strand_+_gene_Chr7-6Mb-seg-899', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCGAAATCCATCTATGTAATCCACCATAAAAATAAACTGAAATAACAAAATCA...GCT'), id='7:48494999-48499999_strand_+_gene_Chr7-6Mb-seg-900', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTTAGTTGGTGGTTTAGTCTCTGTGGACACCAATTGGACCAGGTTAGTTGATT...GAT'), id='7:48499999-48504999_strand_+_gene_Chr7-6Mb-seg-901', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGATGATACATTGTTTAAAATCCACAAAGTCTCTTGTCCTCTCTTAGGCCAACT...TTG'), id='7:48504999-48509999_strand_+_gene_Chr7-6Mb-seg-902', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAATGTATATAAAGAAAATATATAATAAGAAAACATAAAAATAACTAAAACATT...TCA'), id='7:48509999-48514999_strand_+_gene_Chr7-6Mb-seg-903', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGTTACCACCCCCACAGCCCCCACAGGAGAAGCATGGCTTTTAGTCTCGTAGA...AAA'), id='7:48514999-48519999_strand_+_gene_Chr7-6Mb-seg-904', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATCCTCATCTGACTCTTTCAGCCGCTTGTTGGGGATTTTGGAGGACAGTCATG...ATG'), id='7:48519999-48524999_strand_+_gene_Chr7-6Mb-seg-905', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGCTGACCCAATCAGAATCAGCGTTGGGTTAGCACTGATTCAAGCTTAATTCG...TAC'), id='7:48524999-48529999_strand_+_gene_Chr7-6Mb-seg-906', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATAGTATGCCCAAGGATGCCTACTCTCACCATATCTATTCAATATAATACTCAA...ACA'), id='7:48529999-48534999_strand_+_gene_Chr7-6Mb-seg-907', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAACCAAGACAACTTACCATCATCAAAACCCAGCACTCCCACTTTGCCCAGTC...AAG'), id='7:48534999-48539999_strand_+_gene_Chr7-6Mb-seg-908', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATATAAGTTACAATTTGCAAAACACATGAAACTGAAAAAGAACAAAGACCAAAG...TCA'), id='7:48539999-48544999_strand_+_gene_Chr7-6Mb-seg-909', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATATAGTGCACCAATGTGATATTTTGTGGAAGATCCAAACGATCAAGATCCCTT...ATA'), id='7:48544999-48549999_strand_+_gene_Chr7-6Mb-seg-910', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCCTGAGTAACATCACTCAGTCACAAAAGAACACACATGGTATCTACACACTT...GGT'), id='7:48549999-48554999_strand_+_gene_Chr7-6Mb-seg-911', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAAAGGAACAAATACAAGCATATTCAGGAGTTCATCGAATTCAAAGAGGACTT...TAG'), id='7:48554999-48559999_strand_+_gene_Chr7-6Mb-seg-912', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTATTCTTTGGTTGTGTGGTTGGTTGTTCAATATTTGGTTGACTTTCATTAAC...GAT'), id='7:48559999-48564999_strand_+_gene_Chr7-6Mb-seg-913', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGATTCTGCACTGAACCCATCTGATCCTGGGCTGCTTTGATTTTTTGTTTCTT...AGG'), id='7:48564999-48569999_strand_+_gene_Chr7-6Mb-seg-914', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAATAGACAAGGCTGTTCACTCTCTCCCTACCTATTCAACATTGTACTTGAAGT...GGT'), id='7:48569999-48574999_strand_+_gene_Chr7-6Mb-seg-915', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACTACTGTGAAGGTTGTCATTTCCCTAATTTCTTTCTCAGATTGTTTATCCTT...ACC'), id='7:48574999-48579999_strand_+_gene_Chr7-6Mb-seg-916', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCATAAATCTCCAACATTATCATACTATATACTGGATGGACCTTTCCAGTCAC...CTG'), id='7:48579999-48584999_strand_+_gene_Chr7-6Mb-seg-917', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAATCAGTCTGACAGTTCCTCAGAAAATTGAAAAGATTATTACTGGAAGATCC...AAA'), id='7:48584999-48589999_strand_+_gene_Chr7-6Mb-seg-918', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACATTTCATTTAAATTTGTAACTACCGTTGGATTTATCATTAATGTATTCATTT...TTG'), id='7:48589999-48594999_strand_+_gene_Chr7-6Mb-seg-919', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGGATGTCTCTTGGAGGAGTTGGGGCACAAAAGGGAAAGAAGAATGTGATATA...TCC'), id='7:48594999-48599999_strand_+_gene_Chr7-6Mb-seg-920', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCACAAAAAAGACGTAGACTAACAGACTGGCTACACAAACAGGACCCAAAATTT...TGT'), id='7:48599999-48604999_strand_+_gene_Chr7-6Mb-seg-921', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCAATGAACTCTTCACTGGTGAGACTGAAGTTTTAAAGTTCATCCTGAGTGAG...AGC'), id='7:48604999-48609999_strand_+_gene_Chr7-6Mb-seg-922', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGAACAAACATGATTCCCCTCCAAAACTGCAAGCACAGGGGCAGAACTCAAAG...ATT'), id='7:48609999-48614999_strand_+_gene_Chr7-6Mb-seg-923', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAGTGCCAGGAAGCAGGAGTGGGTGGGTTGATGAGCAGGAGGAGGGGGAAGGG...AAC'), id='7:48614999-48619999_strand_+_gene_Chr7-6Mb-seg-924', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAATTATAGAAGAAAACTTCCCAAACCTAAAGAAAGACATGCCCATGAACATA...CTG'), id='7:48619999-48624999_strand_+_gene_Chr7-6Mb-seg-925', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATATAGCTGTCTCTTGTGAGGCTATGCCGGGGCCTAGCAAACACAGAAATAGAT...GGC'), id='7:48624999-48629999_strand_+_gene_Chr7-6Mb-seg-926', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGTGTGCTCTTGAGTGAACCTGCCTTAGAGAGTCACCATAGAGAAAATCCAAA...AGA'), id='7:48629999-48634999_strand_+_gene_Chr7-6Mb-seg-927', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTATGAAAGCCAGAAGAGCCTGGTCAGATGTTATACAGACACTAAGAGAACATA...TAC'), id='7:48634999-48639999_strand_+_gene_Chr7-6Mb-seg-928', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTAATAAATAAATAAATAAATGAATAAAAAAAAAAAAAATCCATGAGCCTATGG...TTT'), id='7:48639999-48644999_strand_+_gene_Chr7-6Mb-seg-929', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAACTCAATTATTCATACAGTTAGAAAGAGCAATTTACAAATTCATCTGGAATA...ACA'), id='7:48644999-48649999_strand_+_gene_Chr7-6Mb-seg-930', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAAGGATAAACAGGCTGAGAAAGAAATTAGGGAAACAACACCCTTCTCAATAG...TTA'), id='7:48649999-48654999_strand_+_gene_Chr7-6Mb-seg-931', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGATTCTACCTTATCCCAGACAGAGTGACATCAAGAACCCAGTAGACAAGAAGT...CAT'), id='7:48654999-48659999_strand_+_gene_Chr7-6Mb-seg-932', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCTTTTGTGTTTGGGAGAAATGTCCTGCAGATATCTTCTAAGTCCATTTGGT...AAG'), id='7:48659999-48664999_strand_+_gene_Chr7-6Mb-seg-933', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCACAGTCCAATTAATAAAGACAATAGTACCCTTAAACACCTAGACTGATTCT...ATA'), id='7:48664999-48669999_strand_+_gene_Chr7-6Mb-seg-934', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTGGTGAGTAAGGAACAGAAAACAGAAAGTATAGGTCATAATAATGAGTAACA...TCG'), id='7:48669999-48674999_strand_+_gene_Chr7-6Mb-seg-935', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCTGGAGGATAGCATTCAAGTGAGGTAACCCAATCACAAAATAACACACATGA...ACT'), id='7:48674999-48679999_strand_+_gene_Chr7-6Mb-seg-936', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATAAAGAAAACCCAAGGTGAGGCAATGCTGGAGATAGAAAACCTTGAAAAGAAA...ACT'), id='7:48679999-48684999_strand_+_gene_Chr7-6Mb-seg-937', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGACCCACAGAGAACCACAAATAAAACAATTGACCACGCACAAATGATTAAATC...CTT'), id='7:48684999-48689999_strand_+_gene_Chr7-6Mb-seg-938', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATAAATGACTGTGAGCATCCACTTCTGTATTTGCCAGGCACTGGCATAGAGTGC...AAG'), id='7:48689999-48694999_strand_+_gene_Chr7-6Mb-seg-939', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGATAACCACACAAACATAAAAAATAACATCAAAAGTAACAGGAAGCAACAAT...ACC'), id='7:48694999-48699999_strand_+_gene_Chr7-6Mb-seg-940', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAGAATAAAAGGCTGGAAAACAATTTTCCAAGCAAATGGTATGAAGAAACAAG...TAA'), id='7:48699999-48704999_strand_+_gene_Chr7-6Mb-seg-941', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACCCACCCCCATGGGAGAAGGCCAACTCCTGACATTAATGATATTCTGCTGTG...GTA'), id='7:48704999-48709999_strand_+_gene_Chr7-6Mb-seg-942', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGCAACGGGTTCCCTGACTCCACCGAGACAAGTCTGCTCAGGTGAGAGTGTGG...AGA'), id='7:48709999-48714999_strand_+_gene_Chr7-6Mb-seg-943', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAAAAATTCAGGTGACAGCAGATACTGGTGAGGATGTGGAGAAAGAGGAACAC...AAG'), id='7:48714999-48719999_strand_+_gene_Chr7-6Mb-seg-944', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCAGATGTGAAGACTTGAAGAGAGCAGGAGATTTTTTAAAAATGAAATCATGT...ACA'), id='7:48719999-48724999_strand_+_gene_Chr7-6Mb-seg-945', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGGTAAGGAGGATCTCCTTCAATTTAAGGTTTATCTAGATAATAAGTTCCAAG...ATC'), id='7:48724999-48729999_strand_+_gene_Chr7-6Mb-seg-946', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCCAATCATCTTAGCTGCTTTCTGTGACCTCTGGGCCAAGCCAAGCCTACTAT...TAC'), id='7:48729999-48734999_strand_+_gene_Chr7-6Mb-seg-947', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCAAAAGCCTATGCGCCACAAAATTGGAATATCTTAAGGAAATGGATGATTTG...AGT'), id='7:48734999-48739999_strand_+_gene_Chr7-6Mb-seg-948', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAGTCTATTTTTCAGTAGATGTCAGGTCTGTCTTCTCATGAGAGGACTAGTTT...CAT'), id='7:48739999-48744999_strand_+_gene_Chr7-6Mb-seg-949', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACTCTTCTCAAGGATAAAAGAACCTCTAGTGGAATCACCATGCCTGACCTAAA...CCC'), id='7:48744999-48749999_strand_+_gene_Chr7-6Mb-seg-950', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGTGGGTGTGGGAGGGGTAGTAATGGTGCCAAGGTGGGGCAGGGTGTGTCTG...CAC'), id='7:48749999-48754999_strand_+_gene_Chr7-6Mb-seg-951', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTTCACTCTCTAGTCAGTCCTCAGCACCAAACAAAGTACTTGGCAAAAATAAG...GAC'), id='7:48754999-48759999_strand_+_gene_Chr7-6Mb-seg-952', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGCAGGGACGAGAAGCTGTGGCCTTGCTTCTCTCACCTTCTTGATCTCTGGAG...CAT'), id='7:48759999-48764999_strand_+_gene_Chr7-6Mb-seg-953', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTAAGCAAATGCTTATAGCCTTTTTGTTGTTACGGCCTAGCTGACAAAGGTTA...ATG'), id='7:48764999-48769999_strand_+_gene_Chr7-6Mb-seg-954', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTATTTTTAAGCTATCATTACTGAATGCTTAGTCTGTGTTTGCTCACATTTAGC...AGA'), id='7:48769999-48774999_strand_+_gene_Chr7-6Mb-seg-955', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTGAGCCACCATGTGGTTGCTGGGAATTGAACTCAGGACCTCTGGAAAAGCAG...TTT'), id='7:48774999-48779999_strand_+_gene_Chr7-6Mb-seg-956', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAAGATTTTTTTATATTTCTTGATTGATGAATTGACTGATTTTAGGAGCCATGC...CAC'), id='7:48779999-48784999_strand_+_gene_Chr7-6Mb-seg-957', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACATTCATAGTGTTGTTTAAAATAGCTCACATCTCATGACTAGATCTTGATGTT...AAC'), id='7:48784999-48789999_strand_+_gene_Chr7-6Mb-seg-958', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGTTTGGTGTAGGTTTAAGTGGAAAATAACATGCTGAGGTGTTTTTATTGGGA...GGA'), id='7:48789999-48794999_strand_+_gene_Chr7-6Mb-seg-959', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGATCCAAGTAAGTGGCCACATAAATAAGTTGATGGTGGGCCACATAAATAAGT...TGT'), id='7:48794999-48799999_strand_+_gene_Chr7-6Mb-seg-960', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGCACGCATCTGTGCACATGTAGTGATCTTTATGATTCCAGAACTCTCTTTTG...TTT'), id='7:48799999-48804999_strand_+_gene_Chr7-6Mb-seg-961', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTAAGGACCATCTTTCTTGTTTCTTTGATCTTTCCTGTCAATAACTAGACTAGA...ACC'), id='7:48804999-48809999_strand_+_gene_Chr7-6Mb-seg-962', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGTGCCAAGTTGAGGTCCTTGGTTCTTCTTAGATATGCTTTTATCTTAATTAT...TAG'), id='7:48809999-48814999_strand_+_gene_Chr7-6Mb-seg-963', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAGAAACTCATGTAAGCATTTATCTACTTTACAGGGAAAGTGCATTGAAATCT...GTG'), id='7:48814999-48819999_strand_+_gene_Chr7-6Mb-seg-964', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCATTCTCTAGGGAATGCTTTATGGAACATAACCATGGCCGGCTCAGACAGGT...CTG'), id='7:48819999-48824999_strand_+_gene_Chr7-6Mb-seg-965', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAAGGAGTCAGTGTGAGATAGTTAAGCTGGCATGTGTCTGACAAGCCAGCCTC...TGT'), id='7:48824999-48829999_strand_+_gene_Chr7-6Mb-seg-966', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAATAAGAGGAAACCATCATCTTGTCATTTGTAACAGGATCATAGGCCAAGTC...TTC'), id='7:48829999-48834999_strand_+_gene_Chr7-6Mb-seg-967', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCCTTCAGAGGCTCATCCCCAGACCCTTCCCTCGCATTGCAGACGTTACTCA...GGC'), id='7:48834999-48839999_strand_+_gene_Chr7-6Mb-seg-968', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTGTGGATGCTGCAAATAGTGGCTTGAAGTCTTATCCAGTTGGCAAAAATATA...CGC'), id='7:48839999-48844999_strand_+_gene_Chr7-6Mb-seg-969', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCTCAGGCTGCCTGTATCACTGACTGAGTTGGCCTCCGACCCACTCTGCCTCT...ATC'), id='7:48844999-48849999_strand_+_gene_Chr7-6Mb-seg-970', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGTGAGTCCATCAAAAAAACACAAAGATATTTACATTTTGATTAATAACAGTA...CTG'), id='7:48849999-48854999_strand_+_gene_Chr7-6Mb-seg-971', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTTGGTGCTGGATGGTGGGTAACAACCCAAGGAGAAAAAAGAGACTTCAGAAA...TAG'), id='7:48854999-48859999_strand_+_gene_Chr7-6Mb-seg-972', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTCAAGAACTCTATTCTATACTAGCCTCGGGAAAAAGAACCCTCTGAAATTGT...TCA'), id='7:48859999-48864999_strand_+_gene_Chr7-6Mb-seg-973', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAAACAATCTAAAGATACTAACAACTTTGGCAGTCCCTGCACATGGCCATTTG...GTC'), id='7:48864999-48869999_strand_+_gene_Chr7-6Mb-seg-974', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGTGACTCAAGGGGGACTTCTGAGTAAGACTTTCGAGTCTACAACACACAATG...TTC'), id='7:48869999-48874999_strand_+_gene_Chr7-6Mb-seg-975', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACGTGGTCTTCCCCAATTAAAATCTTGGCAGCAATTTCCAGGCTCACTATCTG...TAA'), id='7:48874999-48879999_strand_+_gene_Chr7-6Mb-seg-976', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATATACATAATTTTTTAAGCACAGAAAAAACCCCAAAGCTTTTTCCATATATA...GGG'), id='7:48879999-48884999_strand_+_gene_Chr7-6Mb-seg-977', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCGGTGAACACTGATTGGCTAAGGTTCTACCTTAGCAGGCTATCCTACCAAGT...AGT'), id='7:48884999-48889999_strand_+_gene_Chr7-6Mb-seg-978', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCAGGCATCACTCCTGTGAGCCTTGCAACAGCCTTGACAGGTAGTGATAGGCT...ACG'), id='7:48889999-48894999_strand_+_gene_Chr7-6Mb-seg-979', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAATATGTGGGAAGTCATTTCTTGCTGGAATCCAGAGATACTCCCGTACCAAAC...CTT'), id='7:48894999-48899999_strand_+_gene_Chr7-6Mb-seg-980', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTTTTTTGAGACGGGGTCTTGCTGAGTTTCTCCAGCTATTTTTGAACATATCT...AGC'), id='7:48899999-48904999_strand_+_gene_Chr7-6Mb-seg-981', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATAGACAGGGATACTGGGGAGTCGGGCATGTTCAGCCTGGAGATGAAGACACTT...GCT'), id='7:48904999-48909999_strand_+_gene_Chr7-6Mb-seg-982', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTCTTACTGCATCTGCAGTTAGGAGGCAGAGAGATGACTGGTACTCAGCTCAC...ATT'), id='7:48909999-48914999_strand_+_gene_Chr7-6Mb-seg-983', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTTTTTATTATTTCTCTTTACTTAGATCTCTGTGAGAATCATGATGTTTCCAA...CTC'), id='7:48914999-48919999_strand_+_gene_Chr7-6Mb-seg-984', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGAAGACTGTGCAGAGCCTGTCATTGTAGAGTGTGTGCTGTTCGTATTTGATG...TTC'), id='7:48919999-48924999_strand_+_gene_Chr7-6Mb-seg-985', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCCTCAACTTCCTTCTGGTCATGGTGGAACCCCAACAAAGACACTTTGCTCAG...AGA'), id='7:48924999-48929999_strand_+_gene_Chr7-6Mb-seg-986', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCTTGACGGTGGAAGACTTCCTCAGCTGGTAGTACTATTTTGGGAGGTTCTGG...GAC'), id='7:48929999-48934999_strand_+_gene_Chr7-6Mb-seg-987', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGCATATGGGGCAAAGGTGAAATTGGAGCCAAACTTGCCAGCAGAGATCTCTC...GTT'), id='7:48934999-48939999_strand_+_gene_Chr7-6Mb-seg-988', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAGTGGAACGCAAGGGCTCAACGCCACACCCAGAGTGACTACAAACTTCTTTT...CTA'), id='7:48939999-48944999_strand_+_gene_Chr7-6Mb-seg-989', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCTCCCTTCCTTCCCTGGCCACATCAAATTTCCCTGCCATCTGCTTTTCCTGC...CTC'), id='7:48944999-48949999_strand_+_gene_Chr7-6Mb-seg-990', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTTCATCTACGTTCAGACTACTTACTCAAAACAATGGCATACAATCACTTAAC...CTC'), id='7:48949999-48954999_strand_+_gene_Chr7-6Mb-seg-991', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGAGAGAAACTACATGCACTTAGTCACCTTCAGCAAACTGTGGAAGGGCGGTAT...CTC'), id='7:48954999-48959999_strand_+_gene_Chr7-6Mb-seg-992', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTGGATGTGTGTGTCCCTGTGTG...CTT'), id='7:48959999-48964999_strand_+_gene_Chr7-6Mb-seg-993', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAACTCTCCTTCAAAAGTGAAACTCACTGGGGAGTGATATGCTATAGCTGATTG...TGG'), id='7:48964999-48969999_strand_+_gene_Chr7-6Mb-seg-994', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTAGAGGGGGGAAAATAGGATTCAAGCAGACATTCAGGGAGGTTCTGTGGAGGA...CTG'), id='7:48969999-48974999_strand_+_gene_Chr7-6Mb-seg-995', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCACAGATGCCCCTTTCTTTCTGAGCTTCCCTATGCCTGTGGTAATGATAACCT...CTG'), id='7:48974999-48979999_strand_+_gene_Chr7-6Mb-seg-996', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTTTTGATCACCATTTTCCTAGGGAGATATTTGGGCAAAATGTTATCATCATT...AAG'), id='7:48979999-48984999_strand_+_gene_Chr7-6Mb-seg-997', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTCCCTCCAGCTCCACAGCAGAGTCTGATTCTAATGACCTGTAGTCAGTTACC...TTT'), id='7:48984999-48989999_strand_+_gene_Chr7-6Mb-seg-998', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGATCCTCCTGCCTTAGCCTCCTGAGTATGGGATTATAGGCACTTGCCATTAC...GCA'), id='7:48989999-48994999_strand_+_gene_Chr7-6Mb-seg-999', name='', description='', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTTTCAGAGATGCCCTTCCTGGGTTGCTCAGGGCAGAACAGGATGGGACAGCA...CAG'), id='7:48994999-48999999_strand_+_gene_Chr7-6Mb-seg-1000', name='', description='', dbxrefs=[]),\n",
       "  ...]}"
      ]
     },
     "execution_count": 138,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "seq_rd.seq_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "id": "e3624291",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-2.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-3.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-4.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-5.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-6.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-7.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-8.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-9.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-10.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-11.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-12.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-13.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-14.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-15.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-16.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-17.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-18.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-19.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-20.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-21.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-22.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-23.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-24.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-25.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-26.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-27.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-28.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-29.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-30.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-31.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-32.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-33.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-34.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-35.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-36.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-37.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-38.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-39.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-40.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-41.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-42.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-43.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-44.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-45.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-46.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-47.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-48.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-49.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-50.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-51.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-52.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-53.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-54.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-55.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-56.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-57.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-58.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-59.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-60.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-61.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-62.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-63.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-64.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-65.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-66.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-67.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-68.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-69.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-70.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-71.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-72.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-73.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-74.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-75.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-76.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-77.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-78.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-79.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-80.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-81.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-82.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-83.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-84.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-85.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-86.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-87.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-88.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-89.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-90.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-91.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-92.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-93.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-94.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-95.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-96.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-97.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-98.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-99.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-100.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-101.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-102.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-103.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-104.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-105.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-106.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-107.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-108.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-109.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-110.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-111.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-112.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-113.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-114.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-115.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-116.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-117.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-118.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-119.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-120.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-121.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-122.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-123.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-124.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-125.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-126.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-127.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-128.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-129.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-130.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-131.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-132.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-133.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-134.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-135.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-136.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-137.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-138.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-139.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-140.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-141.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-142.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-143.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-144.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-145.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-146.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-147.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-148.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-149.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-150.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-151.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-152.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-153.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-154.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-155.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-156.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-157.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-158.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-159.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-160.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-161.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-162.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-163.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-164.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-165.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-166.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-167.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-168.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-169.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-170.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-171.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-172.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-173.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-174.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-175.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-176.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-177.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-178.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-179.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-180.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-181.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-182.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-183.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-184.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-185.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-186.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-187.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-188.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-189.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-190.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-191.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-192.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-193.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-194.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-195.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-196.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-197.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-198.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-199.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-200.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-201.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-202.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-203.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-204.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-205.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-206.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-207.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-208.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-209.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-210.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-211.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-212.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-213.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-214.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-215.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-216.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-217.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-218.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-219.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-220.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-221.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-222.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-223.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-224.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-225.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-226.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-227.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-228.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-229.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-230.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-231.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-232.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-233.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-234.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-235.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-236.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-237.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-238.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-239.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-240.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-241.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-242.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-243.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-244.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-245.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-246.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-247.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-248.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-249.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-250.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-251.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-252.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-253.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-254.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-255.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-256.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-257.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-258.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-259.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-260.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-261.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-262.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-263.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-264.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-265.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-266.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-267.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-268.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-269.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-270.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-271.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-272.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-273.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-274.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-275.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-276.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-277.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-278.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-279.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-280.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-281.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-282.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-283.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-284.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-285.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-286.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-287.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-288.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-289.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-290.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-291.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-292.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-293.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-294.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-295.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-296.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-297.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-298.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-299.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-300.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-301.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-302.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-303.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-304.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-305.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-306.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-307.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-308.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-309.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-310.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-311.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-312.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-313.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-314.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-315.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-316.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-317.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-318.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-319.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-320.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-321.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-322.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-323.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-324.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-325.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-326.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-327.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-328.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-329.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-330.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-331.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-332.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-333.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-334.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-335.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-336.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-337.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-338.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-339.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-340.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-341.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-342.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-343.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-344.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-345.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-346.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-347.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-348.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-349.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-350.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-351.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-352.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-353.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-354.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-355.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-356.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-357.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-358.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-359.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-360.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-361.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-362.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-363.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-364.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-365.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-366.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-367.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-368.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-369.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-370.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-371.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-372.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-373.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-374.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-375.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-376.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-377.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-378.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-379.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-380.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-381.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-382.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-383.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-384.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-385.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-386.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-387.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-388.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-389.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-390.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-391.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-392.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-393.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-394.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-395.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-396.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-397.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-398.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-399.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-400.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-401.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-402.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-403.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-404.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-405.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-406.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-407.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-408.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-409.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-410.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-411.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-412.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-413.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-414.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-415.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-416.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-417.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-418.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-419.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-420.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-421.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-422.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-423.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-424.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-425.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-426.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-427.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-428.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-429.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-430.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-431.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-432.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-433.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-434.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-435.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-436.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-437.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-438.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-439.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-440.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-441.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-442.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-443.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-444.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-445.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-446.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-447.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-448.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-449.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-450.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-451.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-452.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-453.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-454.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-455.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-456.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-457.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-458.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-459.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-460.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-461.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-462.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-463.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-464.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-465.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-466.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-467.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-468.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-469.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-470.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-471.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-472.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-473.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-474.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-475.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-476.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-477.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-478.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-479.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-480.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-481.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-482.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-483.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-484.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-485.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-486.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-487.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-488.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-489.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-490.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-491.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-492.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-493.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-494.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-495.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-496.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-497.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-498.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-499.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-500.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-501.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-502.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-503.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-504.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-505.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-506.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-507.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-508.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-509.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-510.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-511.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-512.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-513.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-514.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-515.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-516.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-517.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-518.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-519.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-520.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-521.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-522.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-523.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-524.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-525.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-526.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-527.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-528.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-529.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-530.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-531.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-532.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-533.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-534.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-535.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-536.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-537.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-538.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-539.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-540.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-541.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-542.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-543.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-544.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-545.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-546.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-547.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-548.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-549.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-550.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-551.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-552.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-553.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-554.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-555.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-556.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-557.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-558.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-559.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-560.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-561.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-562.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-563.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-564.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-565.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-566.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-567.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-568.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-569.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-570.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-571.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-572.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-573.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-574.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-575.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-576.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-577.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-578.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-579.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-580.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-581.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-582.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-583.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-584.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-585.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-586.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-587.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-588.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-589.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-590.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-591.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-592.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-593.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-594.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-595.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-596.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-597.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-598.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-599.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-600.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-601.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-602.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-603.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-604.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-605.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-606.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-607.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-608.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-609.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-610.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-611.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-612.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-613.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-614.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-615.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-616.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-617.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-618.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-619.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-620.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-621.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-622.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-623.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-624.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-625.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-626.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-627.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-628.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-629.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-630.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-631.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-632.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-633.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-634.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-635.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-636.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-637.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-638.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-639.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-640.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-641.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-642.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-643.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-644.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-645.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-646.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-647.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-648.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-649.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-650.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-651.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-652.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-653.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-654.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-655.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-656.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-657.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-658.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-659.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-660.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-661.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-662.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-663.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-664.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-665.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-666.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-667.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-668.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-669.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-670.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-671.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-672.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-673.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-674.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-675.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-676.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-677.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-678.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-679.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-680.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-681.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-682.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-683.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-684.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-685.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-686.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-687.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-688.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-689.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-690.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-691.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-692.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-693.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-694.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-695.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-696.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-697.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-698.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-699.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-700.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-701.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-702.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-703.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-704.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-705.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-706.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-707.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-708.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-709.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-710.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-711.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-712.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-713.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-714.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-715.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-716.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-717.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-718.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-719.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-720.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-721.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-722.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-723.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-724.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-725.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-726.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-727.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-728.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-729.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-730.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-731.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-732.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-733.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-734.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-735.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-736.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-737.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-738.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-739.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-740.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-741.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-742.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-743.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-744.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-745.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-746.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-747.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-748.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-749.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-750.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-751.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-752.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-753.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-754.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-755.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-756.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-757.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-758.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-759.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-760.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-761.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-762.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-763.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-764.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-765.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-766.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-767.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-768.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-769.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-770.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-771.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-772.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-773.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-774.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-775.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-776.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-777.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-778.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-779.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-780.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-781.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-782.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-783.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-784.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-785.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-786.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-787.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-788.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-789.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-790.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-791.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-792.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-793.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-794.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-795.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-796.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-797.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-798.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-799.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-800.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-801.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-802.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-803.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-804.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-805.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-806.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-807.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-808.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-809.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-810.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-811.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-812.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-813.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-814.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-815.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-816.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-817.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-818.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-819.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-820.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-821.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-822.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-823.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-824.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-825.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-826.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-827.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-828.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-829.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-830.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-831.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-832.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-833.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-834.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-835.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-836.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-837.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-838.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-839.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-840.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-841.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-842.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-843.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-844.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-845.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-846.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-847.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-848.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-849.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-850.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-851.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-852.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-853.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-854.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-855.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-856.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-857.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-858.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-859.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-860.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-861.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-862.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-863.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-864.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-865.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-866.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-867.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-868.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-869.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-870.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-871.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-872.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-873.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-874.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-875.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-876.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-877.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-878.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-879.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-880.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-881.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-882.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-883.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-884.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-885.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-886.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-887.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-888.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-889.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-890.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-891.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-892.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-893.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-894.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-895.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-896.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-897.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-898.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-899.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-900.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-901.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-902.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-903.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-904.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-905.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-906.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-907.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-908.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-909.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-910.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-911.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-912.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-913.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-914.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-915.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-916.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-917.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-918.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-919.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-920.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-921.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-922.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-923.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-924.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-925.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-926.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-927.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-928.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-929.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-930.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-931.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-932.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-933.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-934.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-935.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-936.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-937.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-938.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-939.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-940.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-941.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-942.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-943.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-944.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-945.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-946.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-947.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-948.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-949.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-950.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-951.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-952.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-953.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-954.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-955.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-956.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-957.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-958.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-959.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-960.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-961.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-962.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-963.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-964.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-965.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-966.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-967.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-968.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-969.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-970.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-971.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-972.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-973.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-974.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-975.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-976.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-977.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-978.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-979.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-980.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-981.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-982.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-983.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-984.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-985.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-986.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-987.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-988.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-989.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-990.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-991.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-992.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-993.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-994.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-995.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-996.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-997.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-998.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-999.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1000.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1001.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1002.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1003.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1004.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1005.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1006.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1007.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1008.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1009.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1010.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1011.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1012.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1013.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1014.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1015.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1016.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1017.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1018.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1019.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1020.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1021.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1022.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1023.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1024.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1025.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1026.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1027.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1028.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1029.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1030.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1031.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1032.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1033.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1034.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1035.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1036.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1037.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1038.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1039.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1040.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1041.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1042.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1043.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1044.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1045.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1046.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1047.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1048.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1049.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1050.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1051.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1052.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1053.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1054.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1055.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1056.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1057.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1058.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1059.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1060.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1061.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1062.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1063.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1064.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1065.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1066.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1067.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1068.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1069.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1070.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1071.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1072.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1073.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1074.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1075.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1076.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1077.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1078.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1079.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1080.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1081.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1082.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1083.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1084.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1085.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1086.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1087.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1088.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1089.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1090.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1091.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1092.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1093.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1094.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1095.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1096.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1097.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1098.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1099.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1100.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1101.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1102.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1103.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1104.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1105.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1106.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1107.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1108.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1109.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1110.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1111.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1112.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1113.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1114.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1115.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1116.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1117.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1118.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1119.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1120.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1121.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1122.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1123.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1124.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1125.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1126.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1127.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1128.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1129.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1130.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1131.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1132.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1133.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1134.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1135.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1136.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1137.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1138.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1139.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1140.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1141.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1142.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1143.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1144.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1145.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1146.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1147.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1148.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1149.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1150.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1151.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1152.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1153.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1154.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1155.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1156.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1157.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1158.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1159.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1160.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1161.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1162.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1163.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1164.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1165.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1166.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1167.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1168.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1169.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1170.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1171.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1172.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1173.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1174.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1175.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1176.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1177.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1178.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1179.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1180.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1181.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1182.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1183.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1184.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1185.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1186.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1187.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1188.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1189.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1190.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1191.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1192.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1193.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1194.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1195.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1196.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1197.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1198.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb-seg-1199.fasta\n"
     ]
    }
   ],
   "source": [
    "seq_rd.save_sequences(save_folder=sequence_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9252f4d3",
   "metadata": {},
   "source": [
    "<a id='2'></a>\n",
    "# 2. Design probe targeting sequences by probe_designer\n",
    "\n",
    "run probe_designer, remeber to clear the memory usage because each table should take ~32GB"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8e85674f",
   "metadata": {},
   "source": [
    "<a id='2.1'></a>\n",
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires mm10 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "49eb3323",
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False\n",
    "from ImageAnalysis3 import library_tools"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2e5c8ab3",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "id": "80be081a",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\GRCm38_genome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {genome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1acc14b4",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 141,
   "id": "1e244aad",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\GRCm38_transcriptome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {transcriptome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0b477fab",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "id": "556095da",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Repbase_v2603_repeat_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {repeat_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d38ed029",
   "metadata": {},
   "source": [
    "<a id='2.2'></a>\n",
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "id": "8e476dee",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1200 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\reports\\mouse_5Mb_5kb.pbr\n"
     ]
    }
   ],
   "source": [
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "#transcriptome_index = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "#ref_merfish_index = os.path.join(reference_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "# get input files \n",
    "input_files = [os.path.join(sequence_folder, _fl) for _fl in os.listdir(sequence_folder) if '.fasta' in _fl]\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "probe_savefile = os.path.join(report_folder, f'mouse_5Mb_5kb.pbr')\n",
    "print(probe_savefile)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "722e6b4b",
   "metadata": {},
   "source": [
    "### create pb_designer class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "eb3702cf",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_complement (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 1200\n",
      "    - save_file location: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\reports\\mouse_5Mb_5kb.pbr\n",
      "\n",
      "- Fail to load from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\reports\\mouse_5Mb_5kb.pbr, file doesn't exist.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             #'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             #'ref_merfish':{'file':ref_merfish_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=probe_savefile,\n",
    "    params_dic={'word_size':17,'pb_len':42,'buffer_len':0,'max_count':2**16-1,\n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): 50, # 25\n",
    "               'rep_genome': 0,\n",
    "               #'transcriptome': 25, # 14\n",
    "               #'ref_merfish': 14,\n",
    "               'gc':[0.25,0.85],'tm': 37+0.62*50+5,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b84dae63",
   "metadata": {},
   "source": [
    "### calculate probe reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "e40a3f50",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 61.312s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 62.570s.\n",
      "Time(s): 123.88362693786621\n",
      "- Designing targeting sequence for 1200 regions\n",
      "-- designing region: 7:43999999-44004999_strand_+_gene_Chr7-6Mb_reg_1 -- region: 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.392s.\n",
      "-- designing region: 7:44004999-44009999_strand_+_gene_Chr7-6Mb_reg_2 -- region: 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.715s.\n",
      "-- designing region: 7:44009999-44014999_strand_+_gene_Chr7-6Mb_reg_3 -- region: 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_2.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.874s.\n",
      "-- designing region: 7:44014999-44019999_strand_+_gene_Chr7-6Mb_reg_4 -- region: 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_3.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 7:44019999-44024999_strand_+_gene_Chr7-6Mb_reg_5 -- region: 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_4.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.102s.\n",
      "- Designed 9918 candidate probes in 3.505s.\n",
      "-- designing region: 7:44024999-44029999_strand_+_gene_Chr7-6Mb_reg_6 -- region: 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_5.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.216s.\n",
      "- Designed 9918 candidate probes in 3.620s.\n",
      "-- designing region: 7:44029999-44034999_strand_+_gene_Chr7-6Mb_reg_7 -- region: 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_6.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.207s.\n",
      "- Designed 9918 candidate probes in 3.635s.\n",
      "-- designing region: 7:44034999-44039999_strand_+_gene_Chr7-6Mb_reg_8 -- region: 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_7.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.210s.\n",
      "- Designed 9918 candidate probes in 3.538s.\n",
      "-- designing region: 7:44039999-44044999_strand_+_gene_Chr7-6Mb_reg_9 -- region: 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_8.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.216s.\n",
      "- Designed 9918 candidate probes in 3.520s.\n",
      "-- designing region: 7:44044999-44049999_strand_+_gene_Chr7-6Mb_reg_10 -- region: 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_9.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.207s.\n",
      "- Designed 9918 candidate probes in 3.502s.\n",
      "-- designing region: 7:44049999-44054999_strand_+_gene_Chr7-6Mb_reg_11 -- region: 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_10.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.201s.\n",
      "- Designed 9918 candidate probes in 3.501s.\n",
      "-- designing region: 7:44054999-44059999_strand_+_gene_Chr7-6Mb_reg_12 -- region: 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_11.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "- Designed 9918 candidate probes in 3.630s.\n",
      "-- designing region: 7:44059999-44064999_strand_+_gene_Chr7-6Mb_reg_13 -- region: 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_12.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.209s.\n",
      "- Designed 9918 candidate probes in 3.621s.\n",
      "-- designing region: 7:44064999-44069999_strand_+_gene_Chr7-6Mb_reg_14 -- region: 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_13.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.208s.\n",
      "- Designed 9918 candidate probes in 3.646s.\n",
      "-- designing region: 7:44069999-44074999_strand_+_gene_Chr7-6Mb_reg_15 -- region: 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_14.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.207s.\n",
      "- Designed 9918 candidate probes in 3.572s.\n",
      "-- designing region: 7:44074999-44079999_strand_+_gene_Chr7-6Mb_reg_16 -- region: 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_15.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.207s.\n",
      "- Designed 9918 candidate probes in 3.639s.\n",
      "-- designing region: 7:44079999-44084999_strand_+_gene_Chr7-6Mb_reg_17 -- region: 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_16.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.157s.\n",
      "- Designed 9918 candidate probes in 2.808s.\n",
      "-- designing region: 7:44084999-44089999_strand_+_gene_Chr7-6Mb_reg_18 -- region: 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_17.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.629s.\n",
      "-- designing region: 7:44089999-44094999_strand_+_gene_Chr7-6Mb_reg_19 -- region: 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_18.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:44094999-44099999_strand_+_gene_Chr7-6Mb_reg_20 -- region: 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_19.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 7:44099999-44104999_strand_+_gene_Chr7-6Mb_reg_21 -- region: 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_20.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.173s.\n",
      "- Designed 9918 candidate probes in 2.747s.\n",
      "-- designing region: 7:44104999-44109999_strand_+_gene_Chr7-6Mb_reg_22 -- region: 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_21.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.860s.\n",
      "-- designing region: 7:44109999-44114999_strand_+_gene_Chr7-6Mb_reg_23 -- region: 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_22.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.749s.\n",
      "-- designing region: 7:44114999-44119999_strand_+_gene_Chr7-6Mb_reg_24 -- region: 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_23.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:44119999-44124999_strand_+_gene_Chr7-6Mb_reg_25 -- region: 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_24.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.095s.\n",
      "- Designed 9918 candidate probes in 2.671s.\n",
      "-- designing region: 7:44124999-44129999_strand_+_gene_Chr7-6Mb_reg_26 -- region: 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_25.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.677s.\n",
      "-- designing region: 7:44129999-44134999_strand_+_gene_Chr7-6Mb_reg_27 -- region: 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_26.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:44134999-44139999_strand_+_gene_Chr7-6Mb_reg_28 -- region: 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_27.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:44139999-44144999_strand_+_gene_Chr7-6Mb_reg_29 -- region: 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_28.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.170s.\n",
      "- Designed 9918 candidate probes in 2.744s.\n",
      "-- designing region: 7:44144999-44149999_strand_+_gene_Chr7-6Mb_reg_30 -- region: 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_29.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.708s.\n",
      "-- designing region: 7:44149999-44154999_strand_+_gene_Chr7-6Mb_reg_31 -- region: 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_30.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 7:44154999-44159999_strand_+_gene_Chr7-6Mb_reg_32 -- region: 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_31.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 7:44159999-44164999_strand_+_gene_Chr7-6Mb_reg_33 -- region: 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_32.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.158s.\n",
      "- Designed 9918 candidate probes in 2.939s.\n",
      "-- designing region: 7:44164999-44169999_strand_+_gene_Chr7-6Mb_reg_34 -- region: 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_33.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.615s.\n",
      "-- designing region: 7:44169999-44174999_strand_+_gene_Chr7-6Mb_reg_35 -- region: 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_34.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 7:44174999-44179999_strand_+_gene_Chr7-6Mb_reg_36 -- region: 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_35.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 7:44179999-44184999_strand_+_gene_Chr7-6Mb_reg_37 -- region: 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_36.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "- Designed 9918 candidate probes in 2.721s.\n",
      "-- designing region: 7:44184999-44189999_strand_+_gene_Chr7-6Mb_reg_38 -- region: 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_37.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.683s.\n",
      "-- designing region: 7:44189999-44194999_strand_+_gene_Chr7-6Mb_reg_39 -- region: 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_38.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:44194999-44199999_strand_+_gene_Chr7-6Mb_reg_40 -- region: 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_39.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:44199999-44204999_strand_+_gene_Chr7-6Mb_reg_41 -- region: 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_40.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.167s.\n",
      "- Designed 9918 candidate probes in 2.744s.\n",
      "-- designing region: 7:44204999-44209999_strand_+_gene_Chr7-6Mb_reg_42 -- region: 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_41.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.711s.\n",
      "-- designing region: 7:44209999-44214999_strand_+_gene_Chr7-6Mb_reg_43 -- region: 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_42.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:44214999-44219999_strand_+_gene_Chr7-6Mb_reg_44 -- region: 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_43.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:44219999-44224999_strand_+_gene_Chr7-6Mb_reg_45 -- region: 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_44.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.171s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.588s.\n",
      "-- designing region: 7:44224999-44229999_strand_+_gene_Chr7-6Mb_reg_46 -- region: 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_45.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:44229999-44234999_strand_+_gene_Chr7-6Mb_reg_47 -- region: 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_46.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 3.025s.\n",
      "-- designing region: 7:44234999-44239999_strand_+_gene_Chr7-6Mb_reg_48 -- region: 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_47.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 7:44239999-44244999_strand_+_gene_Chr7-6Mb_reg_49 -- region: 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_48.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 7:44244999-44249999_strand_+_gene_Chr7-6Mb_reg_50 -- region: 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_49.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.631s.\n",
      "-- designing region: 7:44249999-44254999_strand_+_gene_Chr7-6Mb_reg_51 -- region: 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_50.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:44254999-44259999_strand_+_gene_Chr7-6Mb_reg_52 -- region: 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_51.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:44259999-44264999_strand_+_gene_Chr7-6Mb_reg_53 -- region: 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_52.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "- Designed 9918 candidate probes in 2.712s.\n",
      "-- designing region: 7:44264999-44269999_strand_+_gene_Chr7-6Mb_reg_54 -- region: 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_53.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.703s.\n",
      "-- designing region: 7:44269999-44274999_strand_+_gene_Chr7-6Mb_reg_55 -- region: 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_54.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:44274999-44279999_strand_+_gene_Chr7-6Mb_reg_56 -- region: 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_55.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.745s.\n",
      "-- designing region: 7:44279999-44284999_strand_+_gene_Chr7-6Mb_reg_57 -- region: 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_56.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:44284999-44289999_strand_+_gene_Chr7-6Mb_reg_58 -- region: 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_57.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.690s.\n",
      "-- designing region: 7:44289999-44294999_strand_+_gene_Chr7-6Mb_reg_59 -- region: 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_58.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.752s.\n",
      "-- designing region: 7:44294999-44299999_strand_+_gene_Chr7-6Mb_reg_60 -- region: 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_59.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:44299999-44304999_strand_+_gene_Chr7-6Mb_reg_61 -- region: 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_60.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.620s.\n",
      "-- designing region: 7:44304999-44309999_strand_+_gene_Chr7-6Mb_reg_62 -- region: 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_61.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 7:44309999-44314999_strand_+_gene_Chr7-6Mb_reg_63 -- region: 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_62.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 3.095s.\n",
      "-- designing region: 7:44314999-44319999_strand_+_gene_Chr7-6Mb_reg_64 -- region: 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_63.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:44319999-44324999_strand_+_gene_Chr7-6Mb_reg_65 -- region: 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_64.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:44324999-44329999_strand_+_gene_Chr7-6Mb_reg_66 -- region: 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_65.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 7:44329999-44334999_strand_+_gene_Chr7-6Mb_reg_67 -- region: 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_66.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.696s.\n",
      "-- designing region: 7:44334999-44339999_strand_+_gene_Chr7-6Mb_reg_68 -- region: 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_67.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 7:44339999-44344999_strand_+_gene_Chr7-6Mb_reg_69 -- region: 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_68.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 7:44344999-44349999_strand_+_gene_Chr7-6Mb_reg_70 -- region: 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_69.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.600s.\n",
      "-- designing region: 7:44349999-44354999_strand_+_gene_Chr7-6Mb_reg_71 -- region: 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_70.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:44354999-44359999_strand_+_gene_Chr7-6Mb_reg_72 -- region: 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_71.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:44359999-44364999_strand_+_gene_Chr7-6Mb_reg_73 -- region: 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_72.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Designed 9918 candidate probes in 2.703s.\n",
      "-- designing region: 7:44364999-44369999_strand_+_gene_Chr7-6Mb_reg_74 -- region: 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_73.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.677s.\n",
      "-- designing region: 7:44369999-44374999_strand_+_gene_Chr7-6Mb_reg_75 -- region: 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_74.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 7:44374999-44379999_strand_+_gene_Chr7-6Mb_reg_76 -- region: 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_75.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.748s.\n",
      "-- designing region: 7:44379999-44384999_strand_+_gene_Chr7-6Mb_reg_77 -- region: 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_76.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.163s.\n",
      "- Designed 9918 candidate probes in 2.730s.\n",
      "-- designing region: 7:44384999-44389999_strand_+_gene_Chr7-6Mb_reg_78 -- region: 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_77.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.689s.\n",
      "-- designing region: 7:44389999-44394999_strand_+_gene_Chr7-6Mb_reg_79 -- region: 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_78.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:44394999-44399999_strand_+_gene_Chr7-6Mb_reg_80 -- region: 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_79.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.603s.\n",
      "-- designing region: 7:44399999-44404999_strand_+_gene_Chr7-6Mb_reg_81 -- region: 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_80.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.668s.\n",
      "-- designing region: 7:44404999-44409999_strand_+_gene_Chr7-6Mb_reg_82 -- region: 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_81.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:44409999-44414999_strand_+_gene_Chr7-6Mb_reg_83 -- region: 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_82.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 3.187s.\n",
      "-- designing region: 7:44414999-44419999_strand_+_gene_Chr7-6Mb_reg_84 -- region: 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_83.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:44419999-44424999_strand_+_gene_Chr7-6Mb_reg_85 -- region: 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_84.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 7:44424999-44429999_strand_+_gene_Chr7-6Mb_reg_86 -- region: 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_85.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.630s.\n",
      "-- designing region: 7:44429999-44434999_strand_+_gene_Chr7-6Mb_reg_87 -- region: 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_86.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 7:44434999-44439999_strand_+_gene_Chr7-6Mb_reg_88 -- region: 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_87.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 7:44439999-44444999_strand_+_gene_Chr7-6Mb_reg_89 -- region: 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_88.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.141s.\n",
      "- Designed 9918 candidate probes in 2.715s.\n",
      "-- designing region: 7:44444999-44449999_strand_+_gene_Chr7-6Mb_reg_90 -- region: 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_89.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.672s.\n",
      "-- designing region: 7:44449999-44454999_strand_+_gene_Chr7-6Mb_reg_91 -- region: 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_90.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.749s.\n",
      "-- designing region: 7:44454999-44459999_strand_+_gene_Chr7-6Mb_reg_92 -- region: 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_91.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 7:44459999-44464999_strand_+_gene_Chr7-6Mb_reg_93 -- region: 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_92.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.597s.\n",
      "-- designing region: 7:44464999-44469999_strand_+_gene_Chr7-6Mb_reg_94 -- region: 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_93.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 7:44469999-44474999_strand_+_gene_Chr7-6Mb_reg_95 -- region: 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_94.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 7:44474999-44479999_strand_+_gene_Chr7-6Mb_reg_96 -- region: 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_95.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.153s.\n",
      "- Designed 9918 candidate probes in 2.726s.\n",
      "-- designing region: 7:44479999-44484999_strand_+_gene_Chr7-6Mb_reg_97 -- region: 96, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_96.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.682s.\n",
      "-- designing region: 7:44484999-44489999_strand_+_gene_Chr7-6Mb_reg_98 -- region: 97, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_97.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 7:44489999-44494999_strand_+_gene_Chr7-6Mb_reg_99 -- region: 98, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_98.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:44494999-44499999_strand_+_gene_Chr7-6Mb_reg_100 -- region: 99, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_99.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.166s.\n",
      "- Designed 9918 candidate probes in 2.739s.\n",
      "-- designing region: 7:44499999-44504999_strand_+_gene_Chr7-6Mb_reg_101 -- region: 100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_100.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.719s.\n",
      "-- designing region: 7:44504999-44509999_strand_+_gene_Chr7-6Mb_reg_102 -- region: 101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_101.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:44509999-44514999_strand_+_gene_Chr7-6Mb_reg_103 -- region: 102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_102.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.749s.\n",
      "-- designing region: 7:44514999-44519999_strand_+_gene_Chr7-6Mb_reg_104 -- region: 103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_103.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.612s.\n",
      "-- designing region: 7:44519999-44524999_strand_+_gene_Chr7-6Mb_reg_105 -- region: 104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_104.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.196s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:44524999-44529999_strand_+_gene_Chr7-6Mb_reg_106 -- region: 105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_105.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 7:44529999-44534999_strand_+_gene_Chr7-6Mb_reg_107 -- region: 106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_106.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.126s.\n",
      "- Designed 9918 candidate probes in 2.682s.\n",
      "-- designing region: 7:44534999-44539999_strand_+_gene_Chr7-6Mb_reg_108 -- region: 107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_107.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 3.222s.\n",
      "-- designing region: 7:44539999-44544999_strand_+_gene_Chr7-6Mb_reg_109 -- region: 108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_108.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 7:44544999-44549999_strand_+_gene_Chr7-6Mb_reg_110 -- region: 109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_109.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.656s.\n",
      "-- designing region: 7:44549999-44554999_strand_+_gene_Chr7-6Mb_reg_111 -- region: 110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_110.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.729s.\n",
      "-- designing region: 7:44554999-44559999_strand_+_gene_Chr7-6Mb_reg_112 -- region: 111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_111.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.100s.\n",
      "- Designed 9918 candidate probes in 2.670s.\n",
      "-- designing region: 7:44559999-44564999_strand_+_gene_Chr7-6Mb_reg_113 -- region: 112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_112.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "- Designed 9918 candidate probes in 2.673s.\n",
      "-- designing region: 7:44564999-44569999_strand_+_gene_Chr7-6Mb_reg_114 -- region: 113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_113.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:44569999-44574999_strand_+_gene_Chr7-6Mb_reg_115 -- region: 114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_114.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 7:44574999-44579999_strand_+_gene_Chr7-6Mb_reg_116 -- region: 115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_115.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.177s.\n",
      "- Designed 9918 candidate probes in 2.746s.\n",
      "-- designing region: 7:44579999-44584999_strand_+_gene_Chr7-6Mb_reg_117 -- region: 116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_116.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.706s.\n",
      "-- designing region: 7:44584999-44589999_strand_+_gene_Chr7-6Mb_reg_118 -- region: 117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_117.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.631s.\n",
      "-- designing region: 7:44589999-44594999_strand_+_gene_Chr7-6Mb_reg_119 -- region: 118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_118.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.125s.\n",
      "- Designed 9918 candidate probes in 2.696s.\n",
      "-- designing region: 7:44594999-44599999_strand_+_gene_Chr7-6Mb_reg_120 -- region: 119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_119.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.690s.\n",
      "-- designing region: 7:44599999-44604999_strand_+_gene_Chr7-6Mb_reg_121 -- region: 120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_120.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 7:44604999-44609999_strand_+_gene_Chr7-6Mb_reg_122 -- region: 121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_121.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:44609999-44614999_strand_+_gene_Chr7-6Mb_reg_123 -- region: 122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_122.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:44614999-44619999_strand_+_gene_Chr7-6Mb_reg_124 -- region: 123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_123.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.692s.\n",
      "-- designing region: 7:44619999-44624999_strand_+_gene_Chr7-6Mb_reg_125 -- region: 124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_124.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 7:44624999-44629999_strand_+_gene_Chr7-6Mb_reg_126 -- region: 125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_125.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:44629999-44634999_strand_+_gene_Chr7-6Mb_reg_127 -- region: 126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_126.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.596s.\n",
      "-- designing region: 7:44634999-44639999_strand_+_gene_Chr7-6Mb_reg_128 -- region: 127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_127.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:44639999-44644999_strand_+_gene_Chr7-6Mb_reg_129 -- region: 128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_128.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.196s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:44644999-44649999_strand_+_gene_Chr7-6Mb_reg_130 -- region: 129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_129.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Designed 9918 candidate probes in 2.678s.\n",
      "-- designing region: 7:44649999-44654999_strand_+_gene_Chr7-6Mb_reg_131 -- region: 130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_130.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.656s.\n",
      "-- designing region: 7:44654999-44659999_strand_+_gene_Chr7-6Mb_reg_132 -- region: 131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_131.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:44659999-44664999_strand_+_gene_Chr7-6Mb_reg_133 -- region: 132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_132.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 7:44664999-44669999_strand_+_gene_Chr7-6Mb_reg_134 -- region: 133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_133.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 7:44669999-44674999_strand_+_gene_Chr7-6Mb_reg_135 -- region: 134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_134.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.679s.\n",
      "-- designing region: 7:44674999-44679999_strand_+_gene_Chr7-6Mb_reg_136 -- region: 135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_135.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 7:44679999-44684999_strand_+_gene_Chr7-6Mb_reg_137 -- region: 136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_136.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:44684999-44689999_strand_+_gene_Chr7-6Mb_reg_138 -- region: 137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_137.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.614s.\n",
      "-- designing region: 7:44689999-44694999_strand_+_gene_Chr7-6Mb_reg_139 -- region: 138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_138.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.722s.\n",
      "-- designing region: 7:44694999-44699999_strand_+_gene_Chr7-6Mb_reg_140 -- region: 139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_139.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.726s.\n",
      "-- designing region: 7:44699999-44704999_strand_+_gene_Chr7-6Mb_reg_141 -- region: 140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_140.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.169s.\n",
      "- Designed 9918 candidate probes in 3.274s.\n",
      "-- designing region: 7:44704999-44709999_strand_+_gene_Chr7-6Mb_reg_142 -- region: 141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_141.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.725s.\n",
      "-- designing region: 7:44709999-44714999_strand_+_gene_Chr7-6Mb_reg_143 -- region: 142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_142.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.166s.\n",
      "- Designed 9918 candidate probes in 2.746s.\n",
      "-- designing region: 7:44714999-44719999_strand_+_gene_Chr7-6Mb_reg_144 -- region: 143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_143.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.707s.\n",
      "-- designing region: 7:44719999-44724999_strand_+_gene_Chr7-6Mb_reg_145 -- region: 144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_144.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.199s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:44724999-44729999_strand_+_gene_Chr7-6Mb_reg_146 -- region: 145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_145.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.862s.\n",
      "-- designing region: 7:44729999-44734999_strand_+_gene_Chr7-6Mb_reg_147 -- region: 146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_146.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "- Designed 9918 candidate probes in 2.714s.\n",
      "-- designing region: 7:44734999-44739999_strand_+_gene_Chr7-6Mb_reg_148 -- region: 147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_147.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.653s.\n",
      "-- designing region: 7:44739999-44744999_strand_+_gene_Chr7-6Mb_reg_149 -- region: 148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_148.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:44744999-44749999_strand_+_gene_Chr7-6Mb_reg_150 -- region: 149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_149.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 4.969s.\n",
      "-- designing region: 7:44749999-44754999_strand_+_gene_Chr7-6Mb_reg_151 -- region: 150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_150.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 11.051s.\n",
      "- Designed 9918 candidate probes in 23.577s.\n",
      "-- designing region: 7:44754999-44759999_strand_+_gene_Chr7-6Mb_reg_152 -- region: 151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_151.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.345s.\n",
      "- Designed 9918 candidate probes in 4.807s.\n",
      "-- designing region: 7:44759999-44764999_strand_+_gene_Chr7-6Mb_reg_153 -- region: 152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_152.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.361s.\n",
      "- Designed 9918 candidate probes in 5.219s.\n",
      "-- designing region: 7:44764999-44769999_strand_+_gene_Chr7-6Mb_reg_154 -- region: 153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_153.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.679s.\n",
      "-- designing region: 7:44769999-44774999_strand_+_gene_Chr7-6Mb_reg_155 -- region: 154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_154.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:44774999-44779999_strand_+_gene_Chr7-6Mb_reg_156 -- region: 155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_155.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 3.361s.\n",
      "-- designing region: 7:44779999-44784999_strand_+_gene_Chr7-6Mb_reg_157 -- region: 156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_156.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.343s.\n",
      "- Designed 9918 candidate probes in 5.230s.\n",
      "-- designing region: 7:44784999-44789999_strand_+_gene_Chr7-6Mb_reg_158 -- region: 157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_157.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.361s.\n",
      "- Designed 9918 candidate probes in 5.609s.\n",
      "-- designing region: 7:44789999-44794999_strand_+_gene_Chr7-6Mb_reg_159 -- region: 158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_158.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.353s.\n",
      "- Designed 9918 candidate probes in 5.009s.\n",
      "-- designing region: 7:44794999-44799999_strand_+_gene_Chr7-6Mb_reg_160 -- region: 159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_159.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:44799999-44804999_strand_+_gene_Chr7-6Mb_reg_161 -- region: 160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_160.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:44804999-44809999_strand_+_gene_Chr7-6Mb_reg_162 -- region: 161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_161.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:44809999-44814999_strand_+_gene_Chr7-6Mb_reg_163 -- region: 162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_162.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.707s.\n",
      "-- designing region: 7:44814999-44819999_strand_+_gene_Chr7-6Mb_reg_164 -- region: 163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_163.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:44819999-44824999_strand_+_gene_Chr7-6Mb_reg_165 -- region: 164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_164.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:44824999-44829999_strand_+_gene_Chr7-6Mb_reg_166 -- region: 165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_165.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.148s.\n",
      "- Designed 9918 candidate probes in 2.748s.\n",
      "-- designing region: 7:44829999-44834999_strand_+_gene_Chr7-6Mb_reg_167 -- region: 166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_166.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.701s.\n",
      "-- designing region: 7:44834999-44839999_strand_+_gene_Chr7-6Mb_reg_168 -- region: 167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_167.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:44839999-44844999_strand_+_gene_Chr7-6Mb_reg_169 -- region: 168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_168.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.196s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:44844999-44849999_strand_+_gene_Chr7-6Mb_reg_170 -- region: 169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_169.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.148s.\n",
      "- Designed 9918 candidate probes in 2.746s.\n",
      "-- designing region: 7:44849999-44854999_strand_+_gene_Chr7-6Mb_reg_171 -- region: 170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_170.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.679s.\n",
      "-- designing region: 7:44854999-44859999_strand_+_gene_Chr7-6Mb_reg_172 -- region: 171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_171.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:44859999-44864999_strand_+_gene_Chr7-6Mb_reg_173 -- region: 172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_172.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:44864999-44869999_strand_+_gene_Chr7-6Mb_reg_174 -- region: 173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_173.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "- Designed 9918 candidate probes in 2.739s.\n",
      "-- designing region: 7:44869999-44874999_strand_+_gene_Chr7-6Mb_reg_175 -- region: 174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_174.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:44874999-44879999_strand_+_gene_Chr7-6Mb_reg_176 -- region: 175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_175.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 7:44879999-44884999_strand_+_gene_Chr7-6Mb_reg_177 -- region: 176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_176.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:44884999-44889999_strand_+_gene_Chr7-6Mb_reg_178 -- region: 177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_177.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.125s.\n",
      "- Designed 9918 candidate probes in 2.721s.\n",
      "-- designing region: 7:44889999-44894999_strand_+_gene_Chr7-6Mb_reg_179 -- region: 178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_178.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.677s.\n",
      "-- designing region: 7:44894999-44899999_strand_+_gene_Chr7-6Mb_reg_180 -- region: 179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_179.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 7:44899999-44904999_strand_+_gene_Chr7-6Mb_reg_181 -- region: 180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_180.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 3.652s.\n",
      "-- designing region: 7:44904999-44909999_strand_+_gene_Chr7-6Mb_reg_182 -- region: 181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_181.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.638s.\n",
      "-- designing region: 7:44909999-44914999_strand_+_gene_Chr7-6Mb_reg_183 -- region: 182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_182.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:44914999-44919999_strand_+_gene_Chr7-6Mb_reg_184 -- region: 183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_183.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:44919999-44924999_strand_+_gene_Chr7-6Mb_reg_185 -- region: 184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_184.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.104s.\n",
      "- Designed 9918 candidate probes in 2.690s.\n",
      "-- designing region: 7:44924999-44929999_strand_+_gene_Chr7-6Mb_reg_186 -- region: 185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_185.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.674s.\n",
      "-- designing region: 7:44929999-44934999_strand_+_gene_Chr7-6Mb_reg_187 -- region: 186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_186.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:44934999-44939999_strand_+_gene_Chr7-6Mb_reg_188 -- region: 187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_187.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:44939999-44944999_strand_+_gene_Chr7-6Mb_reg_189 -- region: 188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_188.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.166s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 7:44944999-44949999_strand_+_gene_Chr7-6Mb_reg_190 -- region: 189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_189.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.692s.\n",
      "-- designing region: 7:44949999-44954999_strand_+_gene_Chr7-6Mb_reg_191 -- region: 190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_190.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:44954999-44959999_strand_+_gene_Chr7-6Mb_reg_192 -- region: 191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_191.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "- Designed 9918 candidate probes in 2.751s.\n",
      "-- designing region: 7:44959999-44964999_strand_+_gene_Chr7-6Mb_reg_193 -- region: 192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_192.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:44964999-44969999_strand_+_gene_Chr7-6Mb_reg_194 -- region: 193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_193.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.694s.\n",
      "-- designing region: 7:44969999-44974999_strand_+_gene_Chr7-6Mb_reg_195 -- region: 194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_194.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:44974999-44979999_strand_+_gene_Chr7-6Mb_reg_196 -- region: 195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_195.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:44979999-44984999_strand_+_gene_Chr7-6Mb_reg_197 -- region: 196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_196.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.145s.\n",
      "- Designed 9918 candidate probes in 2.588s.\n",
      "-- designing region: 7:44984999-44989999_strand_+_gene_Chr7-6Mb_reg_198 -- region: 197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_197.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:44989999-44994999_strand_+_gene_Chr7-6Mb_reg_199 -- region: 198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_198.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:44994999-44999999_strand_+_gene_Chr7-6Mb_reg_200 -- region: 199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_199.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 7:44999999-45004999_strand_+_gene_Chr7-6Mb_reg_201 -- region: 200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_200.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.195s.\n",
      "- Designed 9918 candidate probes in 2.637s.\n",
      "-- designing region: 7:45004999-45009999_strand_+_gene_Chr7-6Mb_reg_202 -- region: 201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_201.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:45009999-45014999_strand_+_gene_Chr7-6Mb_reg_203 -- region: 202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_202.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 7:45014999-45019999_strand_+_gene_Chr7-6Mb_reg_204 -- region: 203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_203.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:45019999-45024999_strand_+_gene_Chr7-6Mb_reg_205 -- region: 204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_204.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 7:45024999-45029999_strand_+_gene_Chr7-6Mb_reg_206 -- region: 205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_205.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.725s.\n",
      "-- designing region: 7:45029999-45034999_strand_+_gene_Chr7-6Mb_reg_207 -- region: 206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_206.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:45034999-45039999_strand_+_gene_Chr7-6Mb_reg_208 -- region: 207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_207.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:45039999-45044999_strand_+_gene_Chr7-6Mb_reg_209 -- region: 208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_208.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.168s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:45044999-45049999_strand_+_gene_Chr7-6Mb_reg_210 -- region: 209, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_209.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.701s.\n",
      "-- designing region: 7:45049999-45054999_strand_+_gene_Chr7-6Mb_reg_211 -- region: 210, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_210.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:45054999-45059999_strand_+_gene_Chr7-6Mb_reg_212 -- region: 211, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_211.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:45059999-45064999_strand_+_gene_Chr7-6Mb_reg_213 -- region: 212, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_212.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:45064999-45069999_strand_+_gene_Chr7-6Mb_reg_214 -- region: 213, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_213.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.676s.\n",
      "-- designing region: 7:45069999-45074999_strand_+_gene_Chr7-6Mb_reg_215 -- region: 214, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_214.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:45074999-45079999_strand_+_gene_Chr7-6Mb_reg_216 -- region: 215, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_215.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:45079999-45084999_strand_+_gene_Chr7-6Mb_reg_217 -- region: 216, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_216.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "- Designed 9918 candidate probes in 2.734s.\n",
      "-- designing region: 7:45084999-45089999_strand_+_gene_Chr7-6Mb_reg_218 -- region: 217, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_217.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.690s.\n",
      "-- designing region: 7:45089999-45094999_strand_+_gene_Chr7-6Mb_reg_219 -- region: 218, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_218.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:45094999-45099999_strand_+_gene_Chr7-6Mb_reg_220 -- region: 219, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_219.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:45099999-45104999_strand_+_gene_Chr7-6Mb_reg_221 -- region: 220, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_220.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.161s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 7:45104999-45109999_strand_+_gene_Chr7-6Mb_reg_222 -- region: 221, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_221.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.690s.\n",
      "-- designing region: 7:45109999-45114999_strand_+_gene_Chr7-6Mb_reg_223 -- region: 222, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_222.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:45114999-45119999_strand_+_gene_Chr7-6Mb_reg_224 -- region: 223, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_223.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:45119999-45124999_strand_+_gene_Chr7-6Mb_reg_225 -- region: 224, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_224.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.155s.\n",
      "- Designed 9918 candidate probes in 2.752s.\n",
      "-- designing region: 7:45124999-45129999_strand_+_gene_Chr7-6Mb_reg_226 -- region: 225, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_225.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.685s.\n",
      "-- designing region: 7:45129999-45134999_strand_+_gene_Chr7-6Mb_reg_227 -- region: 226, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_226.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:45134999-45139999_strand_+_gene_Chr7-6Mb_reg_228 -- region: 227, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_227.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:45139999-45144999_strand_+_gene_Chr7-6Mb_reg_229 -- region: 228, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_228.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.732s.\n",
      "-- designing region: 7:45144999-45149999_strand_+_gene_Chr7-6Mb_reg_230 -- region: 229, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_229.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 3.782s.\n",
      "-- designing region: 7:45149999-45154999_strand_+_gene_Chr7-6Mb_reg_231 -- region: 230, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_230.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:45154999-45159999_strand_+_gene_Chr7-6Mb_reg_232 -- region: 231, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_231.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:45159999-45164999_strand_+_gene_Chr7-6Mb_reg_233 -- region: 232, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_232.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.119s.\n",
      "- Designed 9918 candidate probes in 2.713s.\n",
      "-- designing region: 7:45164999-45169999_strand_+_gene_Chr7-6Mb_reg_234 -- region: 233, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_233.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.680s.\n",
      "-- designing region: 7:45169999-45174999_strand_+_gene_Chr7-6Mb_reg_235 -- region: 234, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_234.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:45174999-45179999_strand_+_gene_Chr7-6Mb_reg_236 -- region: 235, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_235.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:45179999-45184999_strand_+_gene_Chr7-6Mb_reg_237 -- region: 236, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_236.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Designed 9918 candidate probes in 2.727s.\n",
      "-- designing region: 7:45184999-45189999_strand_+_gene_Chr7-6Mb_reg_238 -- region: 237, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_237.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.699s.\n",
      "-- designing region: 7:45189999-45194999_strand_+_gene_Chr7-6Mb_reg_239 -- region: 238, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_238.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:45194999-45199999_strand_+_gene_Chr7-6Mb_reg_240 -- region: 239, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_239.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:45199999-45204999_strand_+_gene_Chr7-6Mb_reg_241 -- region: 240, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_240.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:45204999-45209999_strand_+_gene_Chr7-6Mb_reg_242 -- region: 241, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_241.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.666s.\n",
      "-- designing region: 7:45209999-45214999_strand_+_gene_Chr7-6Mb_reg_243 -- region: 242, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_242.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:45214999-45219999_strand_+_gene_Chr7-6Mb_reg_244 -- region: 243, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_243.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 7:45219999-45224999_strand_+_gene_Chr7-6Mb_reg_245 -- region: 244, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_244.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.113s.\n",
      "- Designed 9918 candidate probes in 2.706s.\n",
      "-- designing region: 7:45224999-45229999_strand_+_gene_Chr7-6Mb_reg_246 -- region: 245, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_245.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.688s.\n",
      "-- designing region: 7:45229999-45234999_strand_+_gene_Chr7-6Mb_reg_247 -- region: 246, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_246.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:45234999-45239999_strand_+_gene_Chr7-6Mb_reg_248 -- region: 247, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_247.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:45239999-45244999_strand_+_gene_Chr7-6Mb_reg_249 -- region: 248, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_248.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Designed 9918 candidate probes in 2.735s.\n",
      "-- designing region: 7:45244999-45249999_strand_+_gene_Chr7-6Mb_reg_250 -- region: 249, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_249.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.683s.\n",
      "-- designing region: 7:45249999-45254999_strand_+_gene_Chr7-6Mb_reg_251 -- region: 250, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_250.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:45254999-45259999_strand_+_gene_Chr7-6Mb_reg_252 -- region: 251, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_251.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:45259999-45264999_strand_+_gene_Chr7-6Mb_reg_253 -- region: 252, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_252.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "- Designed 9918 candidate probes in 2.745s.\n",
      "-- designing region: 7:45264999-45269999_strand_+_gene_Chr7-6Mb_reg_254 -- region: 253, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_253.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.689s.\n",
      "-- designing region: 7:45269999-45274999_strand_+_gene_Chr7-6Mb_reg_255 -- region: 254, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_254.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:45274999-45279999_strand_+_gene_Chr7-6Mb_reg_256 -- region: 255, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_255.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:45279999-45284999_strand_+_gene_Chr7-6Mb_reg_257 -- region: 256, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_256.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.151s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 7:45284999-45289999_strand_+_gene_Chr7-6Mb_reg_258 -- region: 257, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_257.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.696s.\n",
      "-- designing region: 7:45289999-45294999_strand_+_gene_Chr7-6Mb_reg_259 -- region: 258, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_258.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:45294999-45299999_strand_+_gene_Chr7-6Mb_reg_260 -- region: 259, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_259.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:45299999-45304999_strand_+_gene_Chr7-6Mb_reg_261 -- region: 260, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_260.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "- Designed 9918 candidate probes in 2.723s.\n",
      "-- designing region: 7:45304999-45309999_strand_+_gene_Chr7-6Mb_reg_262 -- region: 261, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_261.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.694s.\n",
      "-- designing region: 7:45309999-45314999_strand_+_gene_Chr7-6Mb_reg_263 -- region: 262, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_262.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:45314999-45319999_strand_+_gene_Chr7-6Mb_reg_264 -- region: 263, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_263.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:45319999-45324999_strand_+_gene_Chr7-6Mb_reg_265 -- region: 264, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_264.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.155s.\n",
      "- Designed 9918 candidate probes in 2.748s.\n",
      "-- designing region: 7:45324999-45329999_strand_+_gene_Chr7-6Mb_reg_266 -- region: 265, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_265.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.677s.\n",
      "-- designing region: 7:45329999-45334999_strand_+_gene_Chr7-6Mb_reg_267 -- region: 266, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_266.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:45334999-45339999_strand_+_gene_Chr7-6Mb_reg_268 -- region: 267, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_267.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:45339999-45344999_strand_+_gene_Chr7-6Mb_reg_269 -- region: 268, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_268.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.171s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 7:45344999-45349999_strand_+_gene_Chr7-6Mb_reg_270 -- region: 269, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_269.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.686s.\n",
      "-- designing region: 7:45349999-45354999_strand_+_gene_Chr7-6Mb_reg_271 -- region: 270, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_270.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:45354999-45359999_strand_+_gene_Chr7-6Mb_reg_272 -- region: 271, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_271.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:45359999-45364999_strand_+_gene_Chr7-6Mb_reg_273 -- region: 272, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_272.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "- Designed 9918 candidate probes in 2.734s.\n",
      "-- designing region: 7:45364999-45369999_strand_+_gene_Chr7-6Mb_reg_274 -- region: 273, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_273.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.693s.\n",
      "-- designing region: 7:45369999-45374999_strand_+_gene_Chr7-6Mb_reg_275 -- region: 274, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_274.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:45374999-45379999_strand_+_gene_Chr7-6Mb_reg_276 -- region: 275, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_275.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:45379999-45384999_strand_+_gene_Chr7-6Mb_reg_277 -- region: 276, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_276.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.169s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 7:45384999-45389999_strand_+_gene_Chr7-6Mb_reg_278 -- region: 277, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_277.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:45389999-45394999_strand_+_gene_Chr7-6Mb_reg_279 -- region: 278, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_278.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.649s.\n",
      "-- designing region: 7:45394999-45399999_strand_+_gene_Chr7-6Mb_reg_280 -- region: 279, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_279.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.110s.\n",
      "- Designed 9918 candidate probes in 2.687s.\n",
      "-- designing region: 7:45399999-45404999_strand_+_gene_Chr7-6Mb_reg_281 -- region: 280, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_280.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.687s.\n",
      "-- designing region: 7:45404999-45409999_strand_+_gene_Chr7-6Mb_reg_282 -- region: 281, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_281.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:45409999-45414999_strand_+_gene_Chr7-6Mb_reg_283 -- region: 282, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_282.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:45414999-45419999_strand_+_gene_Chr7-6Mb_reg_284 -- region: 283, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_283.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.145s.\n",
      "- Designed 9918 candidate probes in 2.740s.\n",
      "-- designing region: 7:45419999-45424999_strand_+_gene_Chr7-6Mb_reg_285 -- region: 284, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_284.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.699s.\n",
      "-- designing region: 7:45424999-45429999_strand_+_gene_Chr7-6Mb_reg_286 -- region: 285, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_285.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:45429999-45434999_strand_+_gene_Chr7-6Mb_reg_287 -- region: 286, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_286.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:45434999-45439999_strand_+_gene_Chr7-6Mb_reg_288 -- region: 287, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_287.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "- Designed 9918 candidate probes in 2.740s.\n",
      "-- designing region: 7:45439999-45444999_strand_+_gene_Chr7-6Mb_reg_289 -- region: 288, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_288.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.884s.\n",
      "-- designing region: 7:45444999-45449999_strand_+_gene_Chr7-6Mb_reg_290 -- region: 289, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_289.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:45449999-45454999_strand_+_gene_Chr7-6Mb_reg_291 -- region: 290, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_290.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:45454999-45459999_strand_+_gene_Chr7-6Mb_reg_292 -- region: 291, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_291.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 4.147s.\n",
      "-- designing region: 7:45459999-45464999_strand_+_gene_Chr7-6Mb_reg_293 -- region: 292, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_292.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.195s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:45464999-45469999_strand_+_gene_Chr7-6Mb_reg_294 -- region: 293, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_293.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:45469999-45474999_strand_+_gene_Chr7-6Mb_reg_295 -- region: 294, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_294.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.624s.\n",
      "-- designing region: 7:45474999-45479999_strand_+_gene_Chr7-6Mb_reg_296 -- region: 295, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_295.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:45479999-45484999_strand_+_gene_Chr7-6Mb_reg_297 -- region: 296, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_296.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:45484999-45489999_strand_+_gene_Chr7-6Mb_reg_298 -- region: 297, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_297.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.688s.\n",
      "-- designing region: 7:45489999-45494999_strand_+_gene_Chr7-6Mb_reg_299 -- region: 298, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_298.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.673s.\n",
      "-- designing region: 7:45494999-45499999_strand_+_gene_Chr7-6Mb_reg_300 -- region: 299, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_299.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:45499999-45504999_strand_+_gene_Chr7-6Mb_reg_301 -- region: 300, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_300.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:45504999-45509999_strand_+_gene_Chr7-6Mb_reg_302 -- region: 301, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_301.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Designed 9918 candidate probes in 2.732s.\n",
      "-- designing region: 7:45509999-45514999_strand_+_gene_Chr7-6Mb_reg_303 -- region: 302, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_302.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.684s.\n",
      "-- designing region: 7:45514999-45519999_strand_+_gene_Chr7-6Mb_reg_304 -- region: 303, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_303.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:45519999-45524999_strand_+_gene_Chr7-6Mb_reg_305 -- region: 304, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_304.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:45524999-45529999_strand_+_gene_Chr7-6Mb_reg_306 -- region: 305, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_305.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.158s.\n",
      "- Designed 9918 candidate probes in 2.742s.\n",
      "-- designing region: 7:45529999-45534999_strand_+_gene_Chr7-6Mb_reg_307 -- region: 306, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_306.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.697s.\n",
      "-- designing region: 7:45534999-45539999_strand_+_gene_Chr7-6Mb_reg_308 -- region: 307, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_307.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:45539999-45544999_strand_+_gene_Chr7-6Mb_reg_309 -- region: 308, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_308.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:45544999-45549999_strand_+_gene_Chr7-6Mb_reg_310 -- region: 309, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_309.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.146s.\n",
      "- Designed 9918 candidate probes in 2.731s.\n",
      "-- designing region: 7:45549999-45554999_strand_+_gene_Chr7-6Mb_reg_311 -- region: 310, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_310.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.692s.\n",
      "-- designing region: 7:45554999-45559999_strand_+_gene_Chr7-6Mb_reg_312 -- region: 311, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_311.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:45559999-45564999_strand_+_gene_Chr7-6Mb_reg_313 -- region: 312, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_312.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:45564999-45569999_strand_+_gene_Chr7-6Mb_reg_314 -- region: 313, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_313.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.117s.\n",
      "- Designed 9918 candidate probes in 2.706s.\n",
      "-- designing region: 7:45569999-45574999_strand_+_gene_Chr7-6Mb_reg_315 -- region: 314, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_314.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.709s.\n",
      "-- designing region: 7:45574999-45579999_strand_+_gene_Chr7-6Mb_reg_316 -- region: 315, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_315.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:45579999-45584999_strand_+_gene_Chr7-6Mb_reg_317 -- region: 316, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_316.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:45584999-45589999_strand_+_gene_Chr7-6Mb_reg_318 -- region: 317, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_317.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.619s.\n",
      "-- designing region: 7:45589999-45594999_strand_+_gene_Chr7-6Mb_reg_319 -- region: 318, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_318.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 7:45594999-45599999_strand_+_gene_Chr7-6Mb_reg_320 -- region: 319, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_319.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:45599999-45604999_strand_+_gene_Chr7-6Mb_reg_321 -- region: 320, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_320.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.110s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.701s.\n",
      "-- designing region: 7:45604999-45609999_strand_+_gene_Chr7-6Mb_reg_322 -- region: 321, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_321.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.660s.\n",
      "-- designing region: 7:45609999-45614999_strand_+_gene_Chr7-6Mb_reg_323 -- region: 322, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_322.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 7:45614999-45619999_strand_+_gene_Chr7-6Mb_reg_324 -- region: 323, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_323.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 7:45619999-45624999_strand_+_gene_Chr7-6Mb_reg_325 -- region: 324, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_324.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.110s.\n",
      "- Designed 9918 candidate probes in 2.708s.\n",
      "-- designing region: 7:45624999-45629999_strand_+_gene_Chr7-6Mb_reg_326 -- region: 325, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_325.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.692s.\n",
      "-- designing region: 7:45629999-45634999_strand_+_gene_Chr7-6Mb_reg_327 -- region: 326, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_326.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 7:45634999-45639999_strand_+_gene_Chr7-6Mb_reg_328 -- region: 327, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_327.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.125s.\n",
      "- Designed 9918 candidate probes in 2.741s.\n",
      "-- designing region: 7:45639999-45644999_strand_+_gene_Chr7-6Mb_reg_329 -- region: 328, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_328.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.143s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 7:45644999-45649999_strand_+_gene_Chr7-6Mb_reg_330 -- region: 329, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_329.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.707s.\n",
      "-- designing region: 7:45649999-45654999_strand_+_gene_Chr7-6Mb_reg_331 -- region: 330, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_330.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:45654999-45659999_strand_+_gene_Chr7-6Mb_reg_332 -- region: 331, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_331.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.806s.\n",
      "-- designing region: 7:45659999-45664999_strand_+_gene_Chr7-6Mb_reg_333 -- region: 332, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_332.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 7:45664999-45669999_strand_+_gene_Chr7-6Mb_reg_334 -- region: 333, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_333.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.662s.\n",
      "-- designing region: 7:45669999-45674999_strand_+_gene_Chr7-6Mb_reg_335 -- region: 334, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_334.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:45674999-45679999_strand_+_gene_Chr7-6Mb_reg_336 -- region: 335, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_335.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.812s.\n",
      "-- designing region: 7:45679999-45684999_strand_+_gene_Chr7-6Mb_reg_337 -- region: 336, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_336.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.802s.\n",
      "-- designing region: 7:45684999-45689999_strand_+_gene_Chr7-6Mb_reg_338 -- region: 337, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_337.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.173s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:45689999-45694999_strand_+_gene_Chr7-6Mb_reg_339 -- region: 338, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_338.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.718s.\n",
      "-- designing region: 7:45694999-45699999_strand_+_gene_Chr7-6Mb_reg_340 -- region: 339, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_339.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.796s.\n",
      "-- designing region: 7:45699999-45704999_strand_+_gene_Chr7-6Mb_reg_341 -- region: 340, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_340.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.807s.\n",
      "-- designing region: 7:45704999-45709999_strand_+_gene_Chr7-6Mb_reg_342 -- region: 341, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_341.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Designed 9918 candidate probes in 2.745s.\n",
      "-- designing region: 7:45709999-45714999_strand_+_gene_Chr7-6Mb_reg_343 -- region: 342, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_342.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.707s.\n",
      "-- designing region: 7:45714999-45719999_strand_+_gene_Chr7-6Mb_reg_344 -- region: 343, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_343.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:45719999-45724999_strand_+_gene_Chr7-6Mb_reg_345 -- region: 344, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_344.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "- Designed 9918 candidate probes in 2.841s.\n",
      "-- designing region: 7:45724999-45729999_strand_+_gene_Chr7-6Mb_reg_346 -- region: 345, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_345.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:45729999-45734999_strand_+_gene_Chr7-6Mb_reg_347 -- region: 346, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_346.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.167s.\n",
      "- Designed 9918 candidate probes in 2.625s.\n",
      "-- designing region: 7:45734999-45739999_strand_+_gene_Chr7-6Mb_reg_348 -- region: 347, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_347.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:45739999-45744999_strand_+_gene_Chr7-6Mb_reg_349 -- region: 348, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_348.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:45744999-45749999_strand_+_gene_Chr7-6Mb_reg_350 -- region: 349, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_349.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 7:45749999-45754999_strand_+_gene_Chr7-6Mb_reg_351 -- region: 350, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_350.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.169s.\n",
      "- Designed 9918 candidate probes in 2.619s.\n",
      "-- designing region: 7:45754999-45759999_strand_+_gene_Chr7-6Mb_reg_352 -- region: 351, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_351.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:45759999-45764999_strand_+_gene_Chr7-6Mb_reg_353 -- region: 352, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_352.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.216s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 7:45764999-45769999_strand_+_gene_Chr7-6Mb_reg_354 -- region: 353, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_353.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:45769999-45774999_strand_+_gene_Chr7-6Mb_reg_355 -- region: 354, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_354.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.176s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 7:45774999-45779999_strand_+_gene_Chr7-6Mb_reg_356 -- region: 355, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_355.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.718s.\n",
      "-- designing region: 7:45779999-45784999_strand_+_gene_Chr7-6Mb_reg_357 -- region: 356, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_356.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.805s.\n",
      "-- designing region: 7:45784999-45789999_strand_+_gene_Chr7-6Mb_reg_358 -- region: 357, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_357.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:45789999-45794999_strand_+_gene_Chr7-6Mb_reg_359 -- region: 358, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_358.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.119s.\n",
      "- Designed 9918 candidate probes in 2.740s.\n",
      "-- designing region: 7:45794999-45799999_strand_+_gene_Chr7-6Mb_reg_360 -- region: 359, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_359.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.691s.\n",
      "-- designing region: 7:45799999-45804999_strand_+_gene_Chr7-6Mb_reg_361 -- region: 360, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_360.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.215s.\n",
      "- Designed 9918 candidate probes in 2.684s.\n",
      "-- designing region: 7:45804999-45809999_strand_+_gene_Chr7-6Mb_reg_362 -- region: 361, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_361.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.195s.\n",
      "- Designed 9918 candidate probes in 2.815s.\n",
      "-- designing region: 7:45809999-45814999_strand_+_gene_Chr7-6Mb_reg_363 -- region: 362, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_362.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.161s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:45814999-45819999_strand_+_gene_Chr7-6Mb_reg_364 -- region: 363, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_363.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.709s.\n",
      "-- designing region: 7:45819999-45824999_strand_+_gene_Chr7-6Mb_reg_365 -- region: 364, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_364.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 7:45824999-45829999_strand_+_gene_Chr7-6Mb_reg_366 -- region: 365, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_365.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.202s.\n",
      "- Designed 9918 candidate probes in 2.822s.\n",
      "-- designing region: 7:45829999-45834999_strand_+_gene_Chr7-6Mb_reg_367 -- region: 366, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_366.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 7:45834999-45839999_strand_+_gene_Chr7-6Mb_reg_368 -- region: 367, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_367.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.666s.\n",
      "-- designing region: 7:45839999-45844999_strand_+_gene_Chr7-6Mb_reg_369 -- region: 368, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_368.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.208s.\n",
      "- Designed 9918 candidate probes in 4.523s.\n",
      "-- designing region: 7:45844999-45849999_strand_+_gene_Chr7-6Mb_reg_370 -- region: 369, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_369.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.116s.\n",
      "- Designed 9918 candidate probes in 2.732s.\n",
      "-- designing region: 7:45849999-45854999_strand_+_gene_Chr7-6Mb_reg_371 -- region: 370, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_370.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.688s.\n",
      "-- designing region: 7:45854999-45859999_strand_+_gene_Chr7-6Mb_reg_372 -- region: 371, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_371.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:45859999-45864999_strand_+_gene_Chr7-6Mb_reg_373 -- region: 372, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_372.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.806s.\n",
      "-- designing region: 7:45864999-45869999_strand_+_gene_Chr7-6Mb_reg_374 -- region: 373, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_373.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.114s.\n",
      "- Designed 9918 candidate probes in 2.732s.\n",
      "-- designing region: 7:45869999-45874999_strand_+_gene_Chr7-6Mb_reg_375 -- region: 374, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_374.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.688s.\n",
      "-- designing region: 7:45874999-45879999_strand_+_gene_Chr7-6Mb_reg_376 -- region: 375, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_375.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.805s.\n",
      "-- designing region: 7:45879999-45884999_strand_+_gene_Chr7-6Mb_reg_377 -- region: 376, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_376.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.222s.\n",
      "- Designed 9918 candidate probes in 2.837s.\n",
      "-- designing region: 7:45884999-45889999_strand_+_gene_Chr7-6Mb_reg_378 -- region: 377, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_377.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:45889999-45894999_strand_+_gene_Chr7-6Mb_reg_379 -- region: 378, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_378.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.174s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:45894999-45899999_strand_+_gene_Chr7-6Mb_reg_380 -- region: 379, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_379.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.707s.\n",
      "-- designing region: 7:45899999-45904999_strand_+_gene_Chr7-6Mb_reg_381 -- region: 380, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_380.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:45904999-45909999_strand_+_gene_Chr7-6Mb_reg_382 -- region: 381, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_381.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.808s.\n",
      "-- designing region: 7:45909999-45914999_strand_+_gene_Chr7-6Mb_reg_383 -- region: 382, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_382.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.113s.\n",
      "- Designed 9918 candidate probes in 2.734s.\n",
      "-- designing region: 7:45914999-45919999_strand_+_gene_Chr7-6Mb_reg_384 -- region: 383, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_383.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.712s.\n",
      "-- designing region: 7:45919999-45924999_strand_+_gene_Chr7-6Mb_reg_385 -- region: 384, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_384.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "- Designed 9918 candidate probes in 2.684s.\n",
      "-- designing region: 7:45924999-45929999_strand_+_gene_Chr7-6Mb_reg_386 -- region: 385, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_385.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.802s.\n",
      "-- designing region: 7:45929999-45934999_strand_+_gene_Chr7-6Mb_reg_387 -- region: 386, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_386.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:45934999-45939999_strand_+_gene_Chr7-6Mb_reg_388 -- region: 387, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_387.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.689s.\n",
      "-- designing region: 7:45939999-45944999_strand_+_gene_Chr7-6Mb_reg_389 -- region: 388, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_388.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:45944999-45949999_strand_+_gene_Chr7-6Mb_reg_390 -- region: 389, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_389.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.812s.\n",
      "-- designing region: 7:45949999-45954999_strand_+_gene_Chr7-6Mb_reg_391 -- region: 390, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_390.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.109s.\n",
      "- Designed 9918 candidate probes in 2.725s.\n",
      "-- designing region: 7:45954999-45959999_strand_+_gene_Chr7-6Mb_reg_392 -- region: 391, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_391.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.691s.\n",
      "-- designing region: 7:45959999-45964999_strand_+_gene_Chr7-6Mb_reg_393 -- region: 392, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_392.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:45964999-45969999_strand_+_gene_Chr7-6Mb_reg_394 -- region: 393, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_393.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.807s.\n",
      "-- designing region: 7:45969999-45974999_strand_+_gene_Chr7-6Mb_reg_395 -- region: 394, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_394.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.099s.\n",
      "- Designed 9918 candidate probes in 2.715s.\n",
      "-- designing region: 7:45974999-45979999_strand_+_gene_Chr7-6Mb_reg_396 -- region: 395, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_395.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.675s.\n",
      "-- designing region: 7:45979999-45984999_strand_+_gene_Chr7-6Mb_reg_397 -- region: 396, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_396.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.818s.\n",
      "-- designing region: 7:45984999-45989999_strand_+_gene_Chr7-6Mb_reg_398 -- region: 397, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_397.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.810s.\n",
      "-- designing region: 7:45989999-45994999_strand_+_gene_Chr7-6Mb_reg_399 -- region: 398, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_398.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.812s.\n",
      "-- designing region: 7:45994999-45999999_strand_+_gene_Chr7-6Mb_reg_400 -- region: 399, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_399.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.802s.\n",
      "-- designing region: 7:45999999-46004999_strand_+_gene_Chr7-6Mb_reg_401 -- region: 400, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_400.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "- Designed 9918 candidate probes in 2.703s.\n",
      "-- designing region: 7:46004999-46009999_strand_+_gene_Chr7-6Mb_reg_402 -- region: 401, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_401.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 7:46009999-46014999_strand_+_gene_Chr7-6Mb_reg_403 -- region: 402, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_402.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.806s.\n",
      "-- designing region: 7:46014999-46019999_strand_+_gene_Chr7-6Mb_reg_404 -- region: 403, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_403.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.807s.\n",
      "-- designing region: 7:46019999-46024999_strand_+_gene_Chr7-6Mb_reg_405 -- region: 404, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_404.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.661s.\n",
      "-- designing region: 7:46024999-46029999_strand_+_gene_Chr7-6Mb_reg_406 -- region: 405, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_405.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:46029999-46034999_strand_+_gene_Chr7-6Mb_reg_407 -- region: 406, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_406.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.808s.\n",
      "-- designing region: 7:46034999-46039999_strand_+_gene_Chr7-6Mb_reg_408 -- region: 407, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_407.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:46039999-46044999_strand_+_gene_Chr7-6Mb_reg_409 -- region: 408, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_408.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.175s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:46044999-46049999_strand_+_gene_Chr7-6Mb_reg_410 -- region: 409, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_409.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.734s.\n",
      "-- designing region: 7:46049999-46054999_strand_+_gene_Chr7-6Mb_reg_411 -- region: 410, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_410.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:46054999-46059999_strand_+_gene_Chr7-6Mb_reg_412 -- region: 411, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_411.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:46059999-46064999_strand_+_gene_Chr7-6Mb_reg_413 -- region: 412, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_412.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 7:46064999-46069999_strand_+_gene_Chr7-6Mb_reg_414 -- region: 413, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_413.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.683s.\n",
      "-- designing region: 7:46069999-46074999_strand_+_gene_Chr7-6Mb_reg_415 -- region: 414, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_414.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.811s.\n",
      "-- designing region: 7:46074999-46079999_strand_+_gene_Chr7-6Mb_reg_416 -- region: 415, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_415.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.811s.\n",
      "-- designing region: 7:46079999-46084999_strand_+_gene_Chr7-6Mb_reg_417 -- region: 416, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_416.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.103s.\n",
      "- Designed 9918 candidate probes in 2.715s.\n",
      "-- designing region: 7:46084999-46089999_strand_+_gene_Chr7-6Mb_reg_418 -- region: 417, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_417.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.688s.\n",
      "-- designing region: 7:46089999-46094999_strand_+_gene_Chr7-6Mb_reg_419 -- region: 418, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_418.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:46094999-46099999_strand_+_gene_Chr7-6Mb_reg_420 -- region: 419, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_419.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.814s.\n",
      "-- designing region: 7:46099999-46104999_strand_+_gene_Chr7-6Mb_reg_421 -- region: 420, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_420.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.809s.\n",
      "-- designing region: 7:46104999-46109999_strand_+_gene_Chr7-6Mb_reg_422 -- region: 421, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_421.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:46109999-46114999_strand_+_gene_Chr7-6Mb_reg_423 -- region: 422, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_422.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.711s.\n",
      "-- designing region: 7:46114999-46119999_strand_+_gene_Chr7-6Mb_reg_424 -- region: 423, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_423.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.817s.\n",
      "-- designing region: 7:46119999-46124999_strand_+_gene_Chr7-6Mb_reg_425 -- region: 424, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_424.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.808s.\n",
      "-- designing region: 7:46124999-46129999_strand_+_gene_Chr7-6Mb_reg_426 -- region: 425, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_425.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.112s.\n",
      "- Designed 9918 candidate probes in 2.722s.\n",
      "-- designing region: 7:46129999-46134999_strand_+_gene_Chr7-6Mb_reg_427 -- region: 426, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_426.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.683s.\n",
      "-- designing region: 7:46134999-46139999_strand_+_gene_Chr7-6Mb_reg_428 -- region: 427, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_427.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.810s.\n",
      "-- designing region: 7:46139999-46144999_strand_+_gene_Chr7-6Mb_reg_429 -- region: 428, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_428.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.806s.\n",
      "-- designing region: 7:46144999-46149999_strand_+_gene_Chr7-6Mb_reg_430 -- region: 429, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_429.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:46149999-46154999_strand_+_gene_Chr7-6Mb_reg_431 -- region: 430, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_430.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.663s.\n",
      "-- designing region: 7:46154999-46159999_strand_+_gene_Chr7-6Mb_reg_432 -- region: 431, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_431.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:46159999-46164999_strand_+_gene_Chr7-6Mb_reg_433 -- region: 432, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_432.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "- Designed 9918 candidate probes in 2.815s.\n",
      "-- designing region: 7:46164999-46169999_strand_+_gene_Chr7-6Mb_reg_434 -- region: 433, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_433.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 7:46169999-46174999_strand_+_gene_Chr7-6Mb_reg_435 -- region: 434, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_434.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.151s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:46174999-46179999_strand_+_gene_Chr7-6Mb_reg_436 -- region: 435, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_435.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.729s.\n",
      "-- designing region: 7:46179999-46184999_strand_+_gene_Chr7-6Mb_reg_437 -- region: 436, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_436.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 7:46184999-46189999_strand_+_gene_Chr7-6Mb_reg_438 -- region: 437, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_437.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.802s.\n",
      "-- designing region: 7:46189999-46194999_strand_+_gene_Chr7-6Mb_reg_439 -- region: 438, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_438.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.106s.\n",
      "- Designed 9918 candidate probes in 2.726s.\n",
      "-- designing region: 7:46194999-46199999_strand_+_gene_Chr7-6Mb_reg_440 -- region: 439, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_439.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.709s.\n",
      "-- designing region: 7:46199999-46204999_strand_+_gene_Chr7-6Mb_reg_441 -- region: 440, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_440.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "- Designed 9918 candidate probes in 2.810s.\n",
      "-- designing region: 7:46204999-46209999_strand_+_gene_Chr7-6Mb_reg_442 -- region: 441, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_441.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.819s.\n",
      "-- designing region: 7:46209999-46214999_strand_+_gene_Chr7-6Mb_reg_443 -- region: 442, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_442.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.811s.\n",
      "-- designing region: 7:46214999-46219999_strand_+_gene_Chr7-6Mb_reg_444 -- region: 443, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_443.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.175s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:46219999-46224999_strand_+_gene_Chr7-6Mb_reg_445 -- region: 444, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_444.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.704s.\n",
      "-- designing region: 7:46224999-46229999_strand_+_gene_Chr7-6Mb_reg_446 -- region: 445, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_445.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.808s.\n",
      "-- designing region: 7:46229999-46234999_strand_+_gene_Chr7-6Mb_reg_447 -- region: 446, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_446.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.811s.\n",
      "-- designing region: 7:46234999-46239999_strand_+_gene_Chr7-6Mb_reg_448 -- region: 447, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_447.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.117s.\n",
      "- Designed 9918 candidate probes in 2.735s.\n",
      "-- designing region: 7:46239999-46244999_strand_+_gene_Chr7-6Mb_reg_449 -- region: 448, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_448.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.198s.\n",
      "- Designed 9918 candidate probes in 2.689s.\n",
      "-- designing region: 7:46244999-46249999_strand_+_gene_Chr7-6Mb_reg_450 -- region: 449, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_449.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:46249999-46254999_strand_+_gene_Chr7-6Mb_reg_451 -- region: 450, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_450.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.814s.\n",
      "-- designing region: 7:46254999-46259999_strand_+_gene_Chr7-6Mb_reg_452 -- region: 451, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_451.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.811s.\n",
      "-- designing region: 7:46259999-46264999_strand_+_gene_Chr7-6Mb_reg_453 -- region: 452, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_452.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.175s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 7:46264999-46269999_strand_+_gene_Chr7-6Mb_reg_454 -- region: 453, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_453.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.713s.\n",
      "-- designing region: 7:46269999-46274999_strand_+_gene_Chr7-6Mb_reg_455 -- region: 454, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_454.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.809s.\n",
      "-- designing region: 7:46274999-46279999_strand_+_gene_Chr7-6Mb_reg_456 -- region: 455, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_455.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 7:46279999-46284999_strand_+_gene_Chr7-6Mb_reg_457 -- region: 456, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_456.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.116s.\n",
      "- Designed 9918 candidate probes in 2.736s.\n",
      "-- designing region: 7:46284999-46289999_strand_+_gene_Chr7-6Mb_reg_458 -- region: 457, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_457.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.716s.\n",
      "-- designing region: 7:46289999-46294999_strand_+_gene_Chr7-6Mb_reg_459 -- region: 458, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_458.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 7:46294999-46299999_strand_+_gene_Chr7-6Mb_reg_460 -- region: 459, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_459.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.810s.\n",
      "-- designing region: 7:46299999-46304999_strand_+_gene_Chr7-6Mb_reg_461 -- region: 460, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_460.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.108s.\n",
      "- Designed 9918 candidate probes in 2.724s.\n",
      "-- designing region: 7:46304999-46309999_strand_+_gene_Chr7-6Mb_reg_462 -- region: 461, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_461.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.675s.\n",
      "-- designing region: 7:46309999-46314999_strand_+_gene_Chr7-6Mb_reg_463 -- region: 462, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_462.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.195s.\n",
      "- Designed 9918 candidate probes in 2.817s.\n",
      "-- designing region: 7:46314999-46319999_strand_+_gene_Chr7-6Mb_reg_464 -- region: 463, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_463.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.805s.\n",
      "-- designing region: 7:46319999-46324999_strand_+_gene_Chr7-6Mb_reg_465 -- region: 464, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_464.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.811s.\n",
      "-- designing region: 7:46324999-46329999_strand_+_gene_Chr7-6Mb_reg_466 -- region: 465, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_465.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 4.797s.\n",
      "-- designing region: 7:46329999-46334999_strand_+_gene_Chr7-6Mb_reg_467 -- region: 466, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_466.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.725s.\n",
      "-- designing region: 7:46334999-46339999_strand_+_gene_Chr7-6Mb_reg_468 -- region: 467, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_467.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.802s.\n",
      "-- designing region: 7:46339999-46344999_strand_+_gene_Chr7-6Mb_reg_469 -- region: 468, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_468.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.805s.\n",
      "-- designing region: 7:46344999-46349999_strand_+_gene_Chr7-6Mb_reg_470 -- region: 469, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_469.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.110s.\n",
      "- Designed 9918 candidate probes in 2.730s.\n",
      "-- designing region: 7:46349999-46354999_strand_+_gene_Chr7-6Mb_reg_471 -- region: 470, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_470.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "- Designed 9918 candidate probes in 2.705s.\n",
      "-- designing region: 7:46354999-46359999_strand_+_gene_Chr7-6Mb_reg_472 -- region: 471, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_471.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.805s.\n",
      "-- designing region: 7:46359999-46364999_strand_+_gene_Chr7-6Mb_reg_473 -- region: 472, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_472.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 7:46364999-46369999_strand_+_gene_Chr7-6Mb_reg_474 -- region: 473, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_473.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 3.601s.\n",
      "-- designing region: 7:46369999-46374999_strand_+_gene_Chr7-6Mb_reg_475 -- region: 474, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_474.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 4.256s.\n",
      "- Designed 9918 candidate probes in 6.780s.\n",
      "-- designing region: 7:46374999-46379999_strand_+_gene_Chr7-6Mb_reg_476 -- region: 475, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_475.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.199s.\n",
      "- Designed 9918 candidate probes in 2.813s.\n",
      "-- designing region: 7:46379999-46384999_strand_+_gene_Chr7-6Mb_reg_477 -- region: 476, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_476.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:46384999-46389999_strand_+_gene_Chr7-6Mb_reg_478 -- region: 477, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_477.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:46389999-46394999_strand_+_gene_Chr7-6Mb_reg_479 -- region: 478, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_478.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.644s.\n",
      "-- designing region: 7:46394999-46399999_strand_+_gene_Chr7-6Mb_reg_480 -- region: 479, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_479.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:46399999-46404999_strand_+_gene_Chr7-6Mb_reg_481 -- region: 480, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_480.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:46404999-46409999_strand_+_gene_Chr7-6Mb_reg_482 -- region: 481, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_481.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:46409999-46414999_strand_+_gene_Chr7-6Mb_reg_483 -- region: 482, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_482.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 7:46414999-46419999_strand_+_gene_Chr7-6Mb_reg_484 -- region: 483, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_483.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.740s.\n",
      "-- designing region: 7:46419999-46424999_strand_+_gene_Chr7-6Mb_reg_485 -- region: 484, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_484.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:46424999-46429999_strand_+_gene_Chr7-6Mb_reg_486 -- region: 485, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_485.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:46429999-46434999_strand_+_gene_Chr7-6Mb_reg_487 -- region: 486, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_486.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:46434999-46439999_strand_+_gene_Chr7-6Mb_reg_488 -- region: 487, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_487.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.674s.\n",
      "-- designing region: 7:46439999-46444999_strand_+_gene_Chr7-6Mb_reg_489 -- region: 488, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_488.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:46444999-46449999_strand_+_gene_Chr7-6Mb_reg_490 -- region: 489, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_489.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 7:46449999-46454999_strand_+_gene_Chr7-6Mb_reg_491 -- region: 490, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_490.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.106s.\n",
      "- Designed 9918 candidate probes in 2.694s.\n",
      "-- designing region: 7:46454999-46459999_strand_+_gene_Chr7-6Mb_reg_492 -- region: 491, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_491.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.697s.\n",
      "-- designing region: 7:46459999-46464999_strand_+_gene_Chr7-6Mb_reg_493 -- region: 492, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_492.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 7:46464999-46469999_strand_+_gene_Chr7-6Mb_reg_494 -- region: 493, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_493.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 7:46469999-46474999_strand_+_gene_Chr7-6Mb_reg_495 -- region: 494, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_494.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.107s.\n",
      "- Designed 9918 candidate probes in 2.713s.\n",
      "-- designing region: 7:46474999-46479999_strand_+_gene_Chr7-6Mb_reg_496 -- region: 495, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_495.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.689s.\n",
      "-- designing region: 7:46479999-46484999_strand_+_gene_Chr7-6Mb_reg_497 -- region: 496, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_496.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:46484999-46489999_strand_+_gene_Chr7-6Mb_reg_498 -- region: 497, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_497.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:46489999-46494999_strand_+_gene_Chr7-6Mb_reg_499 -- region: 498, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_498.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "- Designed 9918 candidate probes in 2.739s.\n",
      "-- designing region: 7:46494999-46499999_strand_+_gene_Chr7-6Mb_reg_500 -- region: 499, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_499.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:46499999-46504999_strand_+_gene_Chr7-6Mb_reg_501 -- region: 500, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_500.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:46504999-46509999_strand_+_gene_Chr7-6Mb_reg_502 -- region: 501, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_501.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:46509999-46514999_strand_+_gene_Chr7-6Mb_reg_503 -- region: 502, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_502.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.126s.\n",
      "- Designed 9918 candidate probes in 2.725s.\n",
      "-- designing region: 7:46514999-46519999_strand_+_gene_Chr7-6Mb_reg_504 -- region: 503, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_503.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.701s.\n",
      "-- designing region: 7:46519999-46524999_strand_+_gene_Chr7-6Mb_reg_505 -- region: 504, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_504.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:46524999-46529999_strand_+_gene_Chr7-6Mb_reg_506 -- region: 505, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_505.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:46529999-46534999_strand_+_gene_Chr7-6Mb_reg_507 -- region: 506, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_506.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "- Designed 9918 candidate probes in 2.730s.\n",
      "-- designing region: 7:46534999-46539999_strand_+_gene_Chr7-6Mb_reg_508 -- region: 507, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_507.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.694s.\n",
      "-- designing region: 7:46539999-46544999_strand_+_gene_Chr7-6Mb_reg_509 -- region: 508, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_508.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:46544999-46549999_strand_+_gene_Chr7-6Mb_reg_510 -- region: 509, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_509.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:46549999-46554999_strand_+_gene_Chr7-6Mb_reg_511 -- region: 510, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_510.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "- Designed 9918 candidate probes in 2.746s.\n",
      "-- designing region: 7:46554999-46559999_strand_+_gene_Chr7-6Mb_reg_512 -- region: 511, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_511.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.690s.\n",
      "-- designing region: 7:46559999-46564999_strand_+_gene_Chr7-6Mb_reg_513 -- region: 512, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_512.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:46564999-46569999_strand_+_gene_Chr7-6Mb_reg_514 -- region: 513, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_513.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:46569999-46574999_strand_+_gene_Chr7-6Mb_reg_515 -- region: 514, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_514.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Designed 9918 candidate probes in 2.729s.\n",
      "-- designing region: 7:46574999-46579999_strand_+_gene_Chr7-6Mb_reg_516 -- region: 515, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_515.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.684s.\n",
      "-- designing region: 7:46579999-46584999_strand_+_gene_Chr7-6Mb_reg_517 -- region: 516, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_516.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:46584999-46589999_strand_+_gene_Chr7-6Mb_reg_518 -- region: 517, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_517.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:46589999-46594999_strand_+_gene_Chr7-6Mb_reg_519 -- region: 518, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_518.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "- Designed 9918 candidate probes in 2.736s.\n",
      "-- designing region: 7:46594999-46599999_strand_+_gene_Chr7-6Mb_reg_520 -- region: 519, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_519.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.689s.\n",
      "-- designing region: 7:46599999-46604999_strand_+_gene_Chr7-6Mb_reg_521 -- region: 520, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_520.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:46604999-46609999_strand_+_gene_Chr7-6Mb_reg_522 -- region: 521, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_521.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 7:46609999-46614999_strand_+_gene_Chr7-6Mb_reg_523 -- region: 522, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_522.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.146s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 7:46614999-46619999_strand_+_gene_Chr7-6Mb_reg_524 -- region: 523, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_523.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.682s.\n",
      "-- designing region: 7:46619999-46624999_strand_+_gene_Chr7-6Mb_reg_525 -- region: 524, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_524.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:46624999-46629999_strand_+_gene_Chr7-6Mb_reg_526 -- region: 525, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_525.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:46629999-46634999_strand_+_gene_Chr7-6Mb_reg_527 -- region: 526, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_526.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Designed 9918 candidate probes in 2.746s.\n",
      "-- designing region: 7:46634999-46639999_strand_+_gene_Chr7-6Mb_reg_528 -- region: 527, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_527.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.704s.\n",
      "-- designing region: 7:46639999-46644999_strand_+_gene_Chr7-6Mb_reg_529 -- region: 528, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_528.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:46644999-46649999_strand_+_gene_Chr7-6Mb_reg_530 -- region: 529, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_529.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:46649999-46654999_strand_+_gene_Chr7-6Mb_reg_531 -- region: 530, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_530.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.124s.\n",
      "- Designed 9918 candidate probes in 2.717s.\n",
      "-- designing region: 7:46654999-46659999_strand_+_gene_Chr7-6Mb_reg_532 -- region: 531, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_531.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.691s.\n",
      "-- designing region: 7:46659999-46664999_strand_+_gene_Chr7-6Mb_reg_533 -- region: 532, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_532.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 7:46664999-46669999_strand_+_gene_Chr7-6Mb_reg_534 -- region: 533, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_533.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:46669999-46674999_strand_+_gene_Chr7-6Mb_reg_535 -- region: 534, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_534.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Designed 9918 candidate probes in 2.729s.\n",
      "-- designing region: 7:46674999-46679999_strand_+_gene_Chr7-6Mb_reg_536 -- region: 535, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_535.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.672s.\n",
      "-- designing region: 7:46679999-46684999_strand_+_gene_Chr7-6Mb_reg_537 -- region: 536, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_536.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:46684999-46689999_strand_+_gene_Chr7-6Mb_reg_538 -- region: 537, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_537.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 7:46689999-46694999_strand_+_gene_Chr7-6Mb_reg_539 -- region: 538, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_538.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.122s.\n",
      "- Designed 9918 candidate probes in 2.718s.\n",
      "-- designing region: 7:46694999-46699999_strand_+_gene_Chr7-6Mb_reg_540 -- region: 539, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_539.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.687s.\n",
      "-- designing region: 7:46699999-46704999_strand_+_gene_Chr7-6Mb_reg_541 -- region: 540, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_540.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:46704999-46709999_strand_+_gene_Chr7-6Mb_reg_542 -- region: 541, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_541.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:46709999-46714999_strand_+_gene_Chr7-6Mb_reg_543 -- region: 542, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_542.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Designed 9918 candidate probes in 2.726s.\n",
      "-- designing region: 7:46714999-46719999_strand_+_gene_Chr7-6Mb_reg_544 -- region: 543, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_543.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:46719999-46724999_strand_+_gene_Chr7-6Mb_reg_545 -- region: 544, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_544.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:46724999-46729999_strand_+_gene_Chr7-6Mb_reg_546 -- region: 545, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_545.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:46729999-46734999_strand_+_gene_Chr7-6Mb_reg_547 -- region: 546, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_546.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "- Designed 9918 candidate probes in 2.731s.\n",
      "-- designing region: 7:46734999-46739999_strand_+_gene_Chr7-6Mb_reg_548 -- region: 547, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_547.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.700s.\n",
      "-- designing region: 7:46739999-46744999_strand_+_gene_Chr7-6Mb_reg_549 -- region: 548, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_548.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:46744999-46749999_strand_+_gene_Chr7-6Mb_reg_550 -- region: 549, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_549.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 7:46749999-46754999_strand_+_gene_Chr7-6Mb_reg_551 -- region: 550, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_550.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.146s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.740s.\n",
      "-- designing region: 7:46754999-46759999_strand_+_gene_Chr7-6Mb_reg_552 -- region: 551, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_551.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.706s.\n",
      "-- designing region: 7:46759999-46764999_strand_+_gene_Chr7-6Mb_reg_553 -- region: 552, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_552.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:46764999-46769999_strand_+_gene_Chr7-6Mb_reg_554 -- region: 553, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_553.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:46769999-46774999_strand_+_gene_Chr7-6Mb_reg_555 -- region: 554, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_554.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "- Designed 9918 candidate probes in 2.727s.\n",
      "-- designing region: 7:46774999-46779999_strand_+_gene_Chr7-6Mb_reg_556 -- region: 555, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_555.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.704s.\n",
      "-- designing region: 7:46779999-46784999_strand_+_gene_Chr7-6Mb_reg_557 -- region: 556, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_556.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:46784999-46789999_strand_+_gene_Chr7-6Mb_reg_558 -- region: 557, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_557.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.730s.\n",
      "-- designing region: 7:46789999-46794999_strand_+_gene_Chr7-6Mb_reg_559 -- region: 558, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_558.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:46794999-46799999_strand_+_gene_Chr7-6Mb_reg_560 -- region: 559, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_559.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.703s.\n",
      "-- designing region: 7:46799999-46804999_strand_+_gene_Chr7-6Mb_reg_561 -- region: 560, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_560.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:46804999-46809999_strand_+_gene_Chr7-6Mb_reg_562 -- region: 561, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_561.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:46809999-46814999_strand_+_gene_Chr7-6Mb_reg_563 -- region: 562, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_562.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "- Designed 9918 candidate probes in 2.733s.\n",
      "-- designing region: 7:46814999-46819999_strand_+_gene_Chr7-6Mb_reg_564 -- region: 563, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_563.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:46819999-46824999_strand_+_gene_Chr7-6Mb_reg_565 -- region: 564, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_564.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:46824999-46829999_strand_+_gene_Chr7-6Mb_reg_566 -- region: 565, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_565.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:46829999-46834999_strand_+_gene_Chr7-6Mb_reg_567 -- region: 566, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_566.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.149s.\n",
      "- Designed 9918 candidate probes in 2.745s.\n",
      "-- designing region: 7:46834999-46839999_strand_+_gene_Chr7-6Mb_reg_568 -- region: 567, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_567.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.669s.\n",
      "-- designing region: 7:46839999-46844999_strand_+_gene_Chr7-6Mb_reg_569 -- region: 568, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_568.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.796s.\n",
      "-- designing region: 7:46844999-46849999_strand_+_gene_Chr7-6Mb_reg_570 -- region: 569, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_569.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 7:46849999-46854999_strand_+_gene_Chr7-6Mb_reg_571 -- region: 570, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_570.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Designed 9918 candidate probes in 2.738s.\n",
      "-- designing region: 7:46854999-46859999_strand_+_gene_Chr7-6Mb_reg_572 -- region: 571, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_571.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.636s.\n",
      "-- designing region: 7:46859999-46864999_strand_+_gene_Chr7-6Mb_reg_573 -- region: 572, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_572.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:46864999-46869999_strand_+_gene_Chr7-6Mb_reg_574 -- region: 573, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_573.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 3.216s.\n",
      "-- designing region: 7:46869999-46874999_strand_+_gene_Chr7-6Mb_reg_575 -- region: 574, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_574.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:46874999-46879999_strand_+_gene_Chr7-6Mb_reg_576 -- region: 575, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_575.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:46879999-46884999_strand_+_gene_Chr7-6Mb_reg_577 -- region: 576, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_576.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.173s.\n",
      "- Designed 9796 candidate probes in 2.734s.\n",
      "-- designing region: 7:46884999-46889999_strand_+_gene_Chr7-6Mb_reg_578 -- region: 577, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_577.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.707s.\n",
      "-- designing region: 7:46889999-46894999_strand_+_gene_Chr7-6Mb_reg_579 -- region: 578, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_578.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:46894999-46899999_strand_+_gene_Chr7-6Mb_reg_580 -- region: 579, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_579.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:46899999-46904999_strand_+_gene_Chr7-6Mb_reg_581 -- region: 580, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_580.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:46904999-46909999_strand_+_gene_Chr7-6Mb_reg_582 -- region: 581, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_581.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.681s.\n",
      "-- designing region: 7:46909999-46914999_strand_+_gene_Chr7-6Mb_reg_583 -- region: 582, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_582.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:46914999-46919999_strand_+_gene_Chr7-6Mb_reg_584 -- region: 583, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_583.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:46919999-46924999_strand_+_gene_Chr7-6Mb_reg_585 -- region: 584, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_584.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.155s.\n",
      "- Designed 9918 candidate probes in 2.746s.\n",
      "-- designing region: 7:46924999-46929999_strand_+_gene_Chr7-6Mb_reg_586 -- region: 585, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_585.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 5.388s.\n",
      "-- designing region: 7:46929999-46934999_strand_+_gene_Chr7-6Mb_reg_587 -- region: 586, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_586.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Designed 9918 candidate probes in 2.687s.\n",
      "-- designing region: 7:46934999-46939999_strand_+_gene_Chr7-6Mb_reg_588 -- region: 587, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_587.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.665s.\n",
      "-- designing region: 7:46939999-46944999_strand_+_gene_Chr7-6Mb_reg_589 -- region: 588, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_588.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:46944999-46949999_strand_+_gene_Chr7-6Mb_reg_590 -- region: 589, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_589.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:46949999-46954999_strand_+_gene_Chr7-6Mb_reg_591 -- region: 590, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_590.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.125s.\n",
      "- Designed 9918 candidate probes in 2.715s.\n",
      "-- designing region: 7:46954999-46959999_strand_+_gene_Chr7-6Mb_reg_592 -- region: 591, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_591.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.688s.\n",
      "-- designing region: 7:46959999-46964999_strand_+_gene_Chr7-6Mb_reg_593 -- region: 592, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_592.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:46964999-46969999_strand_+_gene_Chr7-6Mb_reg_594 -- region: 593, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_593.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:46969999-46974999_strand_+_gene_Chr7-6Mb_reg_595 -- region: 594, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_594.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "- Designed 9918 candidate probes in 2.729s.\n",
      "-- designing region: 7:46974999-46979999_strand_+_gene_Chr7-6Mb_reg_596 -- region: 595, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_595.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.690s.\n",
      "-- designing region: 7:46979999-46984999_strand_+_gene_Chr7-6Mb_reg_597 -- region: 596, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_596.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:46984999-46989999_strand_+_gene_Chr7-6Mb_reg_598 -- region: 597, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_597.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.815s.\n",
      "-- designing region: 7:46989999-46994999_strand_+_gene_Chr7-6Mb_reg_599 -- region: 598, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_598.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "- Designed 9918 candidate probes in 2.708s.\n",
      "-- designing region: 7:46994999-46999999_strand_+_gene_Chr7-6Mb_reg_600 -- region: 599, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_599.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.686s.\n",
      "-- designing region: 7:46999999-47004999_strand_+_gene_Chr7-6Mb_reg_601 -- region: 600, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_600.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:47004999-47009999_strand_+_gene_Chr7-6Mb_reg_602 -- region: 601, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_601.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:47009999-47014999_strand_+_gene_Chr7-6Mb_reg_603 -- region: 602, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_602.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.153s.\n",
      "- Designed 9918 candidate probes in 2.745s.\n",
      "-- designing region: 7:47014999-47019999_strand_+_gene_Chr7-6Mb_reg_604 -- region: 603, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_603.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.702s.\n",
      "-- designing region: 7:47019999-47024999_strand_+_gene_Chr7-6Mb_reg_605 -- region: 604, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_604.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:47024999-47029999_strand_+_gene_Chr7-6Mb_reg_606 -- region: 605, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_605.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:47029999-47034999_strand_+_gene_Chr7-6Mb_reg_607 -- region: 606, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_606.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.175s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:47034999-47039999_strand_+_gene_Chr7-6Mb_reg_608 -- region: 607, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_607.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.686s.\n",
      "-- designing region: 7:47039999-47044999_strand_+_gene_Chr7-6Mb_reg_609 -- region: 608, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_608.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:47044999-47049999_strand_+_gene_Chr7-6Mb_reg_610 -- region: 609, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_609.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:47049999-47054999_strand_+_gene_Chr7-6Mb_reg_611 -- region: 610, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_610.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.157s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 7:47054999-47059999_strand_+_gene_Chr7-6Mb_reg_612 -- region: 611, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_611.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.707s.\n",
      "-- designing region: 7:47059999-47064999_strand_+_gene_Chr7-6Mb_reg_613 -- region: 612, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_612.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 7:47064999-47069999_strand_+_gene_Chr7-6Mb_reg_614 -- region: 613, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_613.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:47069999-47074999_strand_+_gene_Chr7-6Mb_reg_615 -- region: 614, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_614.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Designed 9918 candidate probes in 2.728s.\n",
      "-- designing region: 7:47074999-47079999_strand_+_gene_Chr7-6Mb_reg_616 -- region: 615, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_615.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.699s.\n",
      "-- designing region: 7:47079999-47084999_strand_+_gene_Chr7-6Mb_reg_617 -- region: 616, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_616.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:47084999-47089999_strand_+_gene_Chr7-6Mb_reg_618 -- region: 617, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_617.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:47089999-47094999_strand_+_gene_Chr7-6Mb_reg_619 -- region: 618, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_618.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "- Designed 9918 candidate probes in 2.735s.\n",
      "-- designing region: 7:47094999-47099999_strand_+_gene_Chr7-6Mb_reg_620 -- region: 619, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_619.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.683s.\n",
      "-- designing region: 7:47099999-47104999_strand_+_gene_Chr7-6Mb_reg_621 -- region: 620, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_620.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:47104999-47109999_strand_+_gene_Chr7-6Mb_reg_622 -- region: 621, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_621.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:47109999-47114999_strand_+_gene_Chr7-6Mb_reg_623 -- region: 622, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_622.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.153s.\n",
      "- Designed 9918 candidate probes in 2.745s.\n",
      "-- designing region: 7:47114999-47119999_strand_+_gene_Chr7-6Mb_reg_624 -- region: 623, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_623.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.688s.\n",
      "-- designing region: 7:47119999-47124999_strand_+_gene_Chr7-6Mb_reg_625 -- region: 624, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_624.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:47124999-47129999_strand_+_gene_Chr7-6Mb_reg_626 -- region: 625, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_625.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:47129999-47134999_strand_+_gene_Chr7-6Mb_reg_627 -- region: 626, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_626.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "- Designed 9918 candidate probes in 2.734s.\n",
      "-- designing region: 7:47134999-47139999_strand_+_gene_Chr7-6Mb_reg_628 -- region: 627, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_627.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.665s.\n",
      "-- designing region: 7:47139999-47144999_strand_+_gene_Chr7-6Mb_reg_629 -- region: 628, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_628.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:47144999-47149999_strand_+_gene_Chr7-6Mb_reg_630 -- region: 629, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_629.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:47149999-47154999_strand_+_gene_Chr7-6Mb_reg_631 -- region: 630, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_630.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.174s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:47154999-47159999_strand_+_gene_Chr7-6Mb_reg_632 -- region: 631, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_631.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.693s.\n",
      "-- designing region: 7:47159999-47164999_strand_+_gene_Chr7-6Mb_reg_633 -- region: 632, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_632.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:47164999-47169999_strand_+_gene_Chr7-6Mb_reg_634 -- region: 633, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_633.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:47169999-47174999_strand_+_gene_Chr7-6Mb_reg_635 -- region: 634, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_634.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.145s.\n",
      "- Designed 9918 candidate probes in 2.734s.\n",
      "-- designing region: 7:47174999-47179999_strand_+_gene_Chr7-6Mb_reg_636 -- region: 635, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_635.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.689s.\n",
      "-- designing region: 7:47179999-47184999_strand_+_gene_Chr7-6Mb_reg_637 -- region: 636, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_636.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:47184999-47189999_strand_+_gene_Chr7-6Mb_reg_638 -- region: 637, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_637.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:47189999-47194999_strand_+_gene_Chr7-6Mb_reg_639 -- region: 638, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_638.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.155s.\n",
      "- Designed 9918 candidate probes in 2.744s.\n",
      "-- designing region: 7:47194999-47199999_strand_+_gene_Chr7-6Mb_reg_640 -- region: 639, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_639.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.693s.\n",
      "-- designing region: 7:47199999-47204999_strand_+_gene_Chr7-6Mb_reg_641 -- region: 640, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_640.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:47204999-47209999_strand_+_gene_Chr7-6Mb_reg_642 -- region: 641, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_641.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:47209999-47214999_strand_+_gene_Chr7-6Mb_reg_643 -- region: 642, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_642.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.741s.\n",
      "-- designing region: 7:47214999-47219999_strand_+_gene_Chr7-6Mb_reg_644 -- region: 643, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_643.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.696s.\n",
      "-- designing region: 7:47219999-47224999_strand_+_gene_Chr7-6Mb_reg_645 -- region: 644, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_644.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:47224999-47229999_strand_+_gene_Chr7-6Mb_reg_646 -- region: 645, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_645.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:47229999-47234999_strand_+_gene_Chr7-6Mb_reg_647 -- region: 646, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_646.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.164s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:47234999-47239999_strand_+_gene_Chr7-6Mb_reg_648 -- region: 647, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_647.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.676s.\n",
      "-- designing region: 7:47239999-47244999_strand_+_gene_Chr7-6Mb_reg_649 -- region: 648, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_648.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:47244999-47249999_strand_+_gene_Chr7-6Mb_reg_650 -- region: 649, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_649.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:47249999-47254999_strand_+_gene_Chr7-6Mb_reg_651 -- region: 650, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_650.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.162s.\n",
      "- Designed 9918 candidate probes in 2.749s.\n",
      "-- designing region: 7:47254999-47259999_strand_+_gene_Chr7-6Mb_reg_652 -- region: 651, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_651.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:47259999-47264999_strand_+_gene_Chr7-6Mb_reg_653 -- region: 652, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_652.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:47264999-47269999_strand_+_gene_Chr7-6Mb_reg_654 -- region: 653, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_653.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:47269999-47274999_strand_+_gene_Chr7-6Mb_reg_655 -- region: 654, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_654.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.167s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 7:47274999-47279999_strand_+_gene_Chr7-6Mb_reg_656 -- region: 655, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_655.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:47279999-47284999_strand_+_gene_Chr7-6Mb_reg_657 -- region: 656, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_656.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:47284999-47289999_strand_+_gene_Chr7-6Mb_reg_658 -- region: 657, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_657.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:47289999-47294999_strand_+_gene_Chr7-6Mb_reg_659 -- region: 658, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_658.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.168s.\n",
      "- Designed 9918 candidate probes in 2.732s.\n",
      "-- designing region: 7:47294999-47299999_strand_+_gene_Chr7-6Mb_reg_660 -- region: 659, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_659.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.703s.\n",
      "-- designing region: 7:47299999-47304999_strand_+_gene_Chr7-6Mb_reg_661 -- region: 660, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_660.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:47304999-47309999_strand_+_gene_Chr7-6Mb_reg_662 -- region: 661, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_661.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:47309999-47314999_strand_+_gene_Chr7-6Mb_reg_663 -- region: 662, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_662.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:47314999-47319999_strand_+_gene_Chr7-6Mb_reg_664 -- region: 663, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_663.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.702s.\n",
      "-- designing region: 7:47319999-47324999_strand_+_gene_Chr7-6Mb_reg_665 -- region: 664, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_664.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:47324999-47329999_strand_+_gene_Chr7-6Mb_reg_666 -- region: 665, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_665.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.744s.\n",
      "-- designing region: 7:47329999-47334999_strand_+_gene_Chr7-6Mb_reg_667 -- region: 666, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_666.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:47334999-47339999_strand_+_gene_Chr7-6Mb_reg_668 -- region: 667, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_667.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.711s.\n",
      "-- designing region: 7:47339999-47344999_strand_+_gene_Chr7-6Mb_reg_669 -- region: 668, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_668.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 7:47344999-47349999_strand_+_gene_Chr7-6Mb_reg_670 -- region: 669, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_669.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:47349999-47354999_strand_+_gene_Chr7-6Mb_reg_671 -- region: 670, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_670.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:47354999-47359999_strand_+_gene_Chr7-6Mb_reg_672 -- region: 671, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_671.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.713s.\n",
      "-- designing region: 7:47359999-47364999_strand_+_gene_Chr7-6Mb_reg_673 -- region: 672, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_672.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:47364999-47369999_strand_+_gene_Chr7-6Mb_reg_674 -- region: 673, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_673.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:47369999-47374999_strand_+_gene_Chr7-6Mb_reg_675 -- region: 674, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_674.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.169s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 7:47374999-47379999_strand_+_gene_Chr7-6Mb_reg_676 -- region: 675, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_675.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.709s.\n",
      "-- designing region: 7:47379999-47384999_strand_+_gene_Chr7-6Mb_reg_677 -- region: 676, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_676.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:47384999-47389999_strand_+_gene_Chr7-6Mb_reg_678 -- region: 677, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_677.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 7:47389999-47394999_strand_+_gene_Chr7-6Mb_reg_679 -- region: 678, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_678.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.160s.\n",
      "- Designed 9918 candidate probes in 2.738s.\n",
      "-- designing region: 7:47394999-47399999_strand_+_gene_Chr7-6Mb_reg_680 -- region: 679, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_679.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:47399999-47404999_strand_+_gene_Chr7-6Mb_reg_681 -- region: 680, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_680.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:47404999-47409999_strand_+_gene_Chr7-6Mb_reg_682 -- region: 681, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_681.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:47409999-47414999_strand_+_gene_Chr7-6Mb_reg_683 -- region: 682, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_682.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.597s.\n",
      "-- designing region: 7:47414999-47419999_strand_+_gene_Chr7-6Mb_reg_684 -- region: 683, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_683.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:47419999-47424999_strand_+_gene_Chr7-6Mb_reg_685 -- region: 684, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_684.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:47424999-47429999_strand_+_gene_Chr7-6Mb_reg_686 -- region: 685, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_685.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.116s.\n",
      "- Designed 9918 candidate probes in 2.704s.\n",
      "-- designing region: 7:47429999-47434999_strand_+_gene_Chr7-6Mb_reg_687 -- region: 686, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_686.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.653s.\n",
      "-- designing region: 7:47434999-47439999_strand_+_gene_Chr7-6Mb_reg_688 -- region: 687, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_687.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.632s.\n",
      "-- designing region: 7:47439999-47444999_strand_+_gene_Chr7-6Mb_reg_689 -- region: 688, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_688.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.116s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.693s.\n",
      "-- designing region: 7:47444999-47449999_strand_+_gene_Chr7-6Mb_reg_690 -- region: 689, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_689.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.676s.\n",
      "-- designing region: 7:47449999-47454999_strand_+_gene_Chr7-6Mb_reg_691 -- region: 690, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_690.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:47454999-47459999_strand_+_gene_Chr7-6Mb_reg_692 -- region: 691, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_691.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:47459999-47464999_strand_+_gene_Chr7-6Mb_reg_693 -- region: 692, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_692.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.169s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 7:47464999-47469999_strand_+_gene_Chr7-6Mb_reg_694 -- region: 693, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_693.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.714s.\n",
      "-- designing region: 7:47469999-47474999_strand_+_gene_Chr7-6Mb_reg_695 -- region: 694, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_694.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:47474999-47479999_strand_+_gene_Chr7-6Mb_reg_696 -- region: 695, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_695.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:47479999-47484999_strand_+_gene_Chr7-6Mb_reg_697 -- region: 696, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_696.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.166s.\n",
      "- Designed 9918 candidate probes in 2.741s.\n",
      "-- designing region: 7:47484999-47489999_strand_+_gene_Chr7-6Mb_reg_698 -- region: 697, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_697.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.697s.\n",
      "-- designing region: 7:47489999-47494999_strand_+_gene_Chr7-6Mb_reg_699 -- region: 698, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_698.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:47494999-47499999_strand_+_gene_Chr7-6Mb_reg_700 -- region: 699, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_699.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:47499999-47504999_strand_+_gene_Chr7-6Mb_reg_701 -- region: 700, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_700.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.610s.\n",
      "-- designing region: 7:47504999-47509999_strand_+_gene_Chr7-6Mb_reg_702 -- region: 701, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_701.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.696s.\n",
      "-- designing region: 7:47509999-47514999_strand_+_gene_Chr7-6Mb_reg_703 -- region: 702, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_702.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:47514999-47519999_strand_+_gene_Chr7-6Mb_reg_704 -- region: 703, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_703.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "- Designed 9918 candidate probes in 2.725s.\n",
      "-- designing region: 7:47519999-47524999_strand_+_gene_Chr7-6Mb_reg_705 -- region: 704, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_704.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.677s.\n",
      "-- designing region: 7:47524999-47529999_strand_+_gene_Chr7-6Mb_reg_706 -- region: 705, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_705.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 7:47529999-47534999_strand_+_gene_Chr7-6Mb_reg_707 -- region: 706, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_706.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 7:47534999-47539999_strand_+_gene_Chr7-6Mb_reg_708 -- region: 707, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_707.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.177s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:47539999-47544999_strand_+_gene_Chr7-6Mb_reg_709 -- region: 708, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_708.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.680s.\n",
      "-- designing region: 7:47544999-47549999_strand_+_gene_Chr7-6Mb_reg_710 -- region: 709, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_709.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.650s.\n",
      "-- designing region: 7:47549999-47554999_strand_+_gene_Chr7-6Mb_reg_711 -- region: 710, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_710.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Designed 9918 candidate probes in 2.711s.\n",
      "-- designing region: 7:47554999-47559999_strand_+_gene_Chr7-6Mb_reg_712 -- region: 711, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_711.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.675s.\n",
      "-- designing region: 7:47559999-47564999_strand_+_gene_Chr7-6Mb_reg_713 -- region: 712, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_712.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:47564999-47569999_strand_+_gene_Chr7-6Mb_reg_714 -- region: 713, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_713.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:47569999-47574999_strand_+_gene_Chr7-6Mb_reg_715 -- region: 714, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_714.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.169s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 7:47574999-47579999_strand_+_gene_Chr7-6Mb_reg_716 -- region: 715, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_715.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.702s.\n",
      "-- designing region: 7:47579999-47584999_strand_+_gene_Chr7-6Mb_reg_717 -- region: 716, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_716.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.744s.\n",
      "-- designing region: 7:47584999-47589999_strand_+_gene_Chr7-6Mb_reg_718 -- region: 717, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_717.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 7:47589999-47594999_strand_+_gene_Chr7-6Mb_reg_719 -- region: 718, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_718.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.169s.\n",
      "- Designed 9918 candidate probes in 2.588s.\n",
      "-- designing region: 7:47594999-47599999_strand_+_gene_Chr7-6Mb_reg_720 -- region: 719, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_719.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:47599999-47604999_strand_+_gene_Chr7-6Mb_reg_721 -- region: 720, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_720.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:47604999-47609999_strand_+_gene_Chr7-6Mb_reg_722 -- region: 721, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_721.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Designed 9918 candidate probes in 2.708s.\n",
      "-- designing region: 7:47609999-47614999_strand_+_gene_Chr7-6Mb_reg_723 -- region: 722, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_722.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.661s.\n",
      "-- designing region: 7:47614999-47619999_strand_+_gene_Chr7-6Mb_reg_724 -- region: 723, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_723.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:47619999-47624999_strand_+_gene_Chr7-6Mb_reg_725 -- region: 724, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_724.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:47624999-47629999_strand_+_gene_Chr7-6Mb_reg_726 -- region: 725, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_725.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.153s.\n",
      "- Designed 9918 candidate probes in 2.741s.\n",
      "-- designing region: 7:47629999-47634999_strand_+_gene_Chr7-6Mb_reg_727 -- region: 726, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_726.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.691s.\n",
      "-- designing region: 7:47634999-47639999_strand_+_gene_Chr7-6Mb_reg_728 -- region: 727, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_727.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 7:47639999-47644999_strand_+_gene_Chr7-6Mb_reg_729 -- region: 728, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_728.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:47644999-47649999_strand_+_gene_Chr7-6Mb_reg_730 -- region: 729, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_729.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.176s.\n",
      "- Designed 9918 candidate probes in 2.604s.\n",
      "-- designing region: 7:47649999-47654999_strand_+_gene_Chr7-6Mb_reg_731 -- region: 730, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_730.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 7:47654999-47659999_strand_+_gene_Chr7-6Mb_reg_732 -- region: 731, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_731.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.664s.\n",
      "-- designing region: 7:47659999-47664999_strand_+_gene_Chr7-6Mb_reg_733 -- region: 732, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_732.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.151s.\n",
      "- Designed 9918 candidate probes in 2.573s.\n",
      "-- designing region: 7:47664999-47669999_strand_+_gene_Chr7-6Mb_reg_734 -- region: 733, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_733.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:47669999-47674999_strand_+_gene_Chr7-6Mb_reg_735 -- region: 734, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_734.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 6.059s.\n",
      "-- designing region: 7:47674999-47679999_strand_+_gene_Chr7-6Mb_reg_736 -- region: 735, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_735.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:47679999-47684999_strand_+_gene_Chr7-6Mb_reg_737 -- region: 736, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_736.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:47684999-47689999_strand_+_gene_Chr7-6Mb_reg_738 -- region: 737, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_737.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.617s.\n",
      "-- designing region: 7:47689999-47694999_strand_+_gene_Chr7-6Mb_reg_739 -- region: 738, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_738.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 7:47694999-47699999_strand_+_gene_Chr7-6Mb_reg_740 -- region: 739, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_739.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:47699999-47704999_strand_+_gene_Chr7-6Mb_reg_741 -- region: 740, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_740.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Designed 9918 candidate probes in 2.729s.\n",
      "-- designing region: 7:47704999-47709999_strand_+_gene_Chr7-6Mb_reg_742 -- region: 741, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_741.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.679s.\n",
      "-- designing region: 7:47709999-47714999_strand_+_gene_Chr7-6Mb_reg_743 -- region: 742, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_742.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:47714999-47719999_strand_+_gene_Chr7-6Mb_reg_744 -- region: 743, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_743.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 7:47719999-47724999_strand_+_gene_Chr7-6Mb_reg_745 -- region: 744, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_744.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "- Designed 9918 candidate probes in 2.727s.\n",
      "-- designing region: 7:47724999-47729999_strand_+_gene_Chr7-6Mb_reg_746 -- region: 745, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_745.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.694s.\n",
      "-- designing region: 7:47729999-47734999_strand_+_gene_Chr7-6Mb_reg_747 -- region: 746, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_746.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 7:47734999-47739999_strand_+_gene_Chr7-6Mb_reg_748 -- region: 747, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_747.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 7:47739999-47744999_strand_+_gene_Chr7-6Mb_reg_749 -- region: 748, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_748.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.123s.\n",
      "- Designed 9918 candidate probes in 2.726s.\n",
      "-- designing region: 7:47744999-47749999_strand_+_gene_Chr7-6Mb_reg_750 -- region: 749, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_749.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.692s.\n",
      "-- designing region: 7:47749999-47754999_strand_+_gene_Chr7-6Mb_reg_751 -- region: 750, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_750.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:47754999-47759999_strand_+_gene_Chr7-6Mb_reg_752 -- region: 751, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_751.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:47759999-47764999_strand_+_gene_Chr7-6Mb_reg_753 -- region: 752, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_752.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "- Designed 9918 candidate probes in 2.730s.\n",
      "-- designing region: 7:47764999-47769999_strand_+_gene_Chr7-6Mb_reg_754 -- region: 753, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_753.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.710s.\n",
      "-- designing region: 7:47769999-47774999_strand_+_gene_Chr7-6Mb_reg_755 -- region: 754, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_754.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 7:47774999-47779999_strand_+_gene_Chr7-6Mb_reg_756 -- region: 755, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_755.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:47779999-47784999_strand_+_gene_Chr7-6Mb_reg_757 -- region: 756, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_756.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.100s.\n",
      "- Designed 9918 candidate probes in 2.704s.\n",
      "-- designing region: 7:47784999-47789999_strand_+_gene_Chr7-6Mb_reg_758 -- region: 757, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_757.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.685s.\n",
      "-- designing region: 7:47789999-47794999_strand_+_gene_Chr7-6Mb_reg_759 -- region: 758, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_758.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:47794999-47799999_strand_+_gene_Chr7-6Mb_reg_760 -- region: 759, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_759.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 7:47799999-47804999_strand_+_gene_Chr7-6Mb_reg_761 -- region: 760, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_760.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.117s.\n",
      "- Designed 9918 candidate probes in 2.729s.\n",
      "-- designing region: 7:47804999-47809999_strand_+_gene_Chr7-6Mb_reg_762 -- region: 761, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_761.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.688s.\n",
      "-- designing region: 7:47809999-47814999_strand_+_gene_Chr7-6Mb_reg_763 -- region: 762, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_762.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 7:47814999-47819999_strand_+_gene_Chr7-6Mb_reg_764 -- region: 763, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_763.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.195s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:47819999-47824999_strand_+_gene_Chr7-6Mb_reg_765 -- region: 764, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_764.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.099s.\n",
      "- Designed 9918 candidate probes in 2.706s.\n",
      "-- designing region: 7:47824999-47829999_strand_+_gene_Chr7-6Mb_reg_766 -- region: 765, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_765.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.691s.\n",
      "-- designing region: 7:47829999-47834999_strand_+_gene_Chr7-6Mb_reg_767 -- region: 766, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_766.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.890s.\n",
      "-- designing region: 7:47834999-47839999_strand_+_gene_Chr7-6Mb_reg_768 -- region: 767, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_767.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.195s.\n",
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 7:47839999-47844999_strand_+_gene_Chr7-6Mb_reg_769 -- region: 768, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_768.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:47844999-47849999_strand_+_gene_Chr7-6Mb_reg_770 -- region: 769, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_769.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.168s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:47849999-47854999_strand_+_gene_Chr7-6Mb_reg_771 -- region: 770, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_770.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.708s.\n",
      "-- designing region: 7:47854999-47859999_strand_+_gene_Chr7-6Mb_reg_772 -- region: 771, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_771.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 7:47859999-47864999_strand_+_gene_Chr7-6Mb_reg_773 -- region: 772, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_772.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:47864999-47869999_strand_+_gene_Chr7-6Mb_reg_774 -- region: 773, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_773.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Designed 9918 candidate probes in 2.741s.\n",
      "-- designing region: 7:47869999-47874999_strand_+_gene_Chr7-6Mb_reg_775 -- region: 774, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_774.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.697s.\n",
      "-- designing region: 7:47874999-47879999_strand_+_gene_Chr7-6Mb_reg_776 -- region: 775, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_775.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 7:47879999-47884999_strand_+_gene_Chr7-6Mb_reg_777 -- region: 776, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_776.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:47884999-47889999_strand_+_gene_Chr7-6Mb_reg_778 -- region: 777, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_777.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.103s.\n",
      "- Designed 9918 candidate probes in 2.708s.\n",
      "-- designing region: 7:47889999-47894999_strand_+_gene_Chr7-6Mb_reg_779 -- region: 778, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_778.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.694s.\n",
      "-- designing region: 7:47894999-47899999_strand_+_gene_Chr7-6Mb_reg_780 -- region: 779, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_779.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 7:47899999-47904999_strand_+_gene_Chr7-6Mb_reg_781 -- region: 780, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_780.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 7:47904999-47909999_strand_+_gene_Chr7-6Mb_reg_782 -- region: 781, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_781.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.122s.\n",
      "- Designed 9918 candidate probes in 2.732s.\n",
      "-- designing region: 7:47909999-47914999_strand_+_gene_Chr7-6Mb_reg_783 -- region: 782, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_782.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.680s.\n",
      "-- designing region: 7:47914999-47919999_strand_+_gene_Chr7-6Mb_reg_784 -- region: 783, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_783.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:47919999-47924999_strand_+_gene_Chr7-6Mb_reg_785 -- region: 784, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_784.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:47924999-47929999_strand_+_gene_Chr7-6Mb_reg_786 -- region: 785, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_785.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.108s.\n",
      "- Designed 9918 candidate probes in 2.723s.\n",
      "-- designing region: 7:47929999-47934999_strand_+_gene_Chr7-6Mb_reg_787 -- region: 786, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_786.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.691s.\n",
      "-- designing region: 7:47934999-47939999_strand_+_gene_Chr7-6Mb_reg_788 -- region: 787, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_787.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:47939999-47944999_strand_+_gene_Chr7-6Mb_reg_789 -- region: 788, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_788.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 7:47944999-47949999_strand_+_gene_Chr7-6Mb_reg_790 -- region: 789, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_789.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.120s.\n",
      "- Designed 9918 candidate probes in 2.731s.\n",
      "-- designing region: 7:47949999-47954999_strand_+_gene_Chr7-6Mb_reg_791 -- region: 790, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_790.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.681s.\n",
      "-- designing region: 7:47954999-47959999_strand_+_gene_Chr7-6Mb_reg_792 -- region: 791, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_791.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 7:47959999-47964999_strand_+_gene_Chr7-6Mb_reg_793 -- region: 792, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_792.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:47964999-47969999_strand_+_gene_Chr7-6Mb_reg_794 -- region: 793, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_793.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.104s.\n",
      "- Designed 9918 candidate probes in 2.710s.\n",
      "-- designing region: 7:47969999-47974999_strand_+_gene_Chr7-6Mb_reg_795 -- region: 794, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_794.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.694s.\n",
      "-- designing region: 7:47974999-47979999_strand_+_gene_Chr7-6Mb_reg_796 -- region: 795, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_795.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:47979999-47984999_strand_+_gene_Chr7-6Mb_reg_797 -- region: 796, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_796.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:47984999-47989999_strand_+_gene_Chr7-6Mb_reg_798 -- region: 797, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_797.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.114s.\n",
      "- Designed 9918 candidate probes in 2.724s.\n",
      "-- designing region: 7:47989999-47994999_strand_+_gene_Chr7-6Mb_reg_799 -- region: 798, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_798.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:47994999-47999999_strand_+_gene_Chr7-6Mb_reg_800 -- region: 799, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_799.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:47999999-48004999_strand_+_gene_Chr7-6Mb_reg_801 -- region: 800, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_800.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:48004999-48009999_strand_+_gene_Chr7-6Mb_reg_802 -- region: 801, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_801.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:48009999-48014999_strand_+_gene_Chr7-6Mb_reg_803 -- region: 802, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_802.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:48014999-48019999_strand_+_gene_Chr7-6Mb_reg_804 -- region: 803, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_803.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.708s.\n",
      "-- designing region: 7:48019999-48024999_strand_+_gene_Chr7-6Mb_reg_805 -- region: 804, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_804.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 6.814s.\n",
      "-- designing region: 7:48024999-48029999_strand_+_gene_Chr7-6Mb_reg_806 -- region: 805, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_805.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.808s.\n",
      "-- designing region: 7:48029999-48034999_strand_+_gene_Chr7-6Mb_reg_807 -- region: 806, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_806.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.108s.\n",
      "- Designed 9918 candidate probes in 2.705s.\n",
      "-- designing region: 7:48034999-48039999_strand_+_gene_Chr7-6Mb_reg_808 -- region: 807, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_807.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.679s.\n",
      "-- designing region: 7:48039999-48044999_strand_+_gene_Chr7-6Mb_reg_809 -- region: 808, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_808.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:48044999-48049999_strand_+_gene_Chr7-6Mb_reg_810 -- region: 809, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_809.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:48049999-48054999_strand_+_gene_Chr7-6Mb_reg_811 -- region: 810, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_810.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.126s.\n",
      "- Designed 9918 candidate probes in 2.716s.\n",
      "-- designing region: 7:48054999-48059999_strand_+_gene_Chr7-6Mb_reg_812 -- region: 811, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_811.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.696s.\n",
      "-- designing region: 7:48059999-48064999_strand_+_gene_Chr7-6Mb_reg_813 -- region: 812, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_812.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:48064999-48069999_strand_+_gene_Chr7-6Mb_reg_814 -- region: 813, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_813.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:48069999-48074999_strand_+_gene_Chr7-6Mb_reg_815 -- region: 814, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_814.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Designed 9918 candidate probes in 2.732s.\n",
      "-- designing region: 7:48074999-48079999_strand_+_gene_Chr7-6Mb_reg_816 -- region: 815, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_815.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.699s.\n",
      "-- designing region: 7:48079999-48084999_strand_+_gene_Chr7-6Mb_reg_817 -- region: 816, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_816.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:48084999-48089999_strand_+_gene_Chr7-6Mb_reg_818 -- region: 817, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_817.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:48089999-48094999_strand_+_gene_Chr7-6Mb_reg_819 -- region: 818, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_818.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "- Designed 9918 candidate probes in 2.568s.\n",
      "-- designing region: 7:48094999-48099999_strand_+_gene_Chr7-6Mb_reg_820 -- region: 819, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_819.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:48099999-48104999_strand_+_gene_Chr7-6Mb_reg_821 -- region: 820, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_820.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:48104999-48109999_strand_+_gene_Chr7-6Mb_reg_822 -- region: 821, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_821.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:48109999-48114999_strand_+_gene_Chr7-6Mb_reg_823 -- region: 822, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_822.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.629s.\n",
      "-- designing region: 7:48114999-48119999_strand_+_gene_Chr7-6Mb_reg_824 -- region: 823, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_823.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:48119999-48124999_strand_+_gene_Chr7-6Mb_reg_825 -- region: 824, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_824.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:48124999-48129999_strand_+_gene_Chr7-6Mb_reg_826 -- region: 825, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_825.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:48129999-48134999_strand_+_gene_Chr7-6Mb_reg_827 -- region: 826, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_826.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.631s.\n",
      "-- designing region: 7:48134999-48139999_strand_+_gene_Chr7-6Mb_reg_828 -- region: 827, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_827.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:48139999-48144999_strand_+_gene_Chr7-6Mb_reg_829 -- region: 828, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_828.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 7:48144999-48149999_strand_+_gene_Chr7-6Mb_reg_830 -- region: 829, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_829.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.115s.\n",
      "- Designed 9918 candidate probes in 2.709s.\n",
      "-- designing region: 7:48149999-48154999_strand_+_gene_Chr7-6Mb_reg_831 -- region: 830, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_830.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.650s.\n",
      "-- designing region: 7:48154999-48159999_strand_+_gene_Chr7-6Mb_reg_832 -- region: 831, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_831.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:48159999-48164999_strand_+_gene_Chr7-6Mb_reg_833 -- region: 832, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_832.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:48164999-48169999_strand_+_gene_Chr7-6Mb_reg_834 -- region: 833, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_833.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.168s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:48169999-48174999_strand_+_gene_Chr7-6Mb_reg_835 -- region: 834, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_834.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.678s.\n",
      "-- designing region: 7:48174999-48179999_strand_+_gene_Chr7-6Mb_reg_836 -- region: 835, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_835.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:48179999-48184999_strand_+_gene_Chr7-6Mb_reg_837 -- region: 836, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_836.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:48184999-48189999_strand_+_gene_Chr7-6Mb_reg_838 -- region: 837, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_837.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.157s.\n",
      "- Designed 9918 candidate probes in 2.746s.\n",
      "-- designing region: 7:48189999-48194999_strand_+_gene_Chr7-6Mb_reg_839 -- region: 838, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_838.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.677s.\n",
      "-- designing region: 7:48194999-48199999_strand_+_gene_Chr7-6Mb_reg_840 -- region: 839, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_839.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:48199999-48204999_strand_+_gene_Chr7-6Mb_reg_841 -- region: 840, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_840.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:48204999-48209999_strand_+_gene_Chr7-6Mb_reg_842 -- region: 841, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_841.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.172s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:48209999-48214999_strand_+_gene_Chr7-6Mb_reg_843 -- region: 842, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_842.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.684s.\n",
      "-- designing region: 7:48214999-48219999_strand_+_gene_Chr7-6Mb_reg_844 -- region: 843, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_843.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:48219999-48224999_strand_+_gene_Chr7-6Mb_reg_845 -- region: 844, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_844.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:48224999-48229999_strand_+_gene_Chr7-6Mb_reg_846 -- region: 845, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_845.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.145s.\n",
      "- Designed 9918 candidate probes in 2.739s.\n",
      "-- designing region: 7:48229999-48234999_strand_+_gene_Chr7-6Mb_reg_847 -- region: 846, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_846.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.705s.\n",
      "-- designing region: 7:48234999-48239999_strand_+_gene_Chr7-6Mb_reg_848 -- region: 847, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_847.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:48239999-48244999_strand_+_gene_Chr7-6Mb_reg_849 -- region: 848, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_848.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:48244999-48249999_strand_+_gene_Chr7-6Mb_reg_850 -- region: 849, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_849.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.176s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:48249999-48254999_strand_+_gene_Chr7-6Mb_reg_851 -- region: 850, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_850.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.696s.\n",
      "-- designing region: 7:48254999-48259999_strand_+_gene_Chr7-6Mb_reg_852 -- region: 851, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_851.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:48259999-48264999_strand_+_gene_Chr7-6Mb_reg_853 -- region: 852, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_852.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:48264999-48269999_strand_+_gene_Chr7-6Mb_reg_854 -- region: 853, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_853.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.159s.\n",
      "- Designed 9918 candidate probes in 2.739s.\n",
      "-- designing region: 7:48269999-48274999_strand_+_gene_Chr7-6Mb_reg_855 -- region: 854, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_854.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.701s.\n",
      "-- designing region: 7:48274999-48279999_strand_+_gene_Chr7-6Mb_reg_856 -- region: 855, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_855.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:48279999-48284999_strand_+_gene_Chr7-6Mb_reg_857 -- region: 856, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_856.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:48284999-48289999_strand_+_gene_Chr7-6Mb_reg_858 -- region: 857, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_857.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.163s.\n",
      "- Designed 9918 candidate probes in 2.752s.\n",
      "-- designing region: 7:48289999-48294999_strand_+_gene_Chr7-6Mb_reg_859 -- region: 858, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_858.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.700s.\n",
      "-- designing region: 7:48294999-48299999_strand_+_gene_Chr7-6Mb_reg_860 -- region: 859, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_859.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:48299999-48304999_strand_+_gene_Chr7-6Mb_reg_861 -- region: 860, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_860.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:48304999-48309999_strand_+_gene_Chr7-6Mb_reg_862 -- region: 861, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_861.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.171s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 7:48309999-48314999_strand_+_gene_Chr7-6Mb_reg_863 -- region: 862, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_862.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.700s.\n",
      "-- designing region: 7:48314999-48319999_strand_+_gene_Chr7-6Mb_reg_864 -- region: 863, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_863.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:48319999-48324999_strand_+_gene_Chr7-6Mb_reg_865 -- region: 864, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_864.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 7:48324999-48329999_strand_+_gene_Chr7-6Mb_reg_866 -- region: 865, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_865.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.149s.\n",
      "- Designed 9918 candidate probes in 2.646s.\n",
      "-- designing region: 7:48329999-48334999_strand_+_gene_Chr7-6Mb_reg_867 -- region: 866, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_866.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.738s.\n",
      "-- designing region: 7:48334999-48339999_strand_+_gene_Chr7-6Mb_reg_868 -- region: 867, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_867.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:48339999-48344999_strand_+_gene_Chr7-6Mb_reg_869 -- region: 868, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_868.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:48344999-48349999_strand_+_gene_Chr7-6Mb_reg_870 -- region: 869, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_869.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.168s.\n",
      "- Designed 9918 candidate probes in 2.596s.\n",
      "-- designing region: 7:48349999-48354999_strand_+_gene_Chr7-6Mb_reg_871 -- region: 870, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_870.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:48354999-48359999_strand_+_gene_Chr7-6Mb_reg_872 -- region: 871, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_871.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:48359999-48364999_strand_+_gene_Chr7-6Mb_reg_873 -- region: 872, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_872.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.106s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.697s.\n",
      "-- designing region: 7:48364999-48369999_strand_+_gene_Chr7-6Mb_reg_874 -- region: 873, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_873.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.674s.\n",
      "-- designing region: 7:48369999-48374999_strand_+_gene_Chr7-6Mb_reg_875 -- region: 874, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_874.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:48374999-48379999_strand_+_gene_Chr7-6Mb_reg_876 -- region: 875, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_875.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:48379999-48384999_strand_+_gene_Chr7-6Mb_reg_877 -- region: 876, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_876.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "- Designed 9918 candidate probes in 2.721s.\n",
      "-- designing region: 7:48384999-48389999_strand_+_gene_Chr7-6Mb_reg_878 -- region: 877, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_877.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.700s.\n",
      "-- designing region: 7:48389999-48394999_strand_+_gene_Chr7-6Mb_reg_879 -- region: 878, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_878.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:48394999-48399999_strand_+_gene_Chr7-6Mb_reg_880 -- region: 879, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_879.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:48399999-48404999_strand_+_gene_Chr7-6Mb_reg_881 -- region: 880, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_880.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.161s.\n",
      "- Designed 9918 candidate probes in 2.742s.\n",
      "-- designing region: 7:48404999-48409999_strand_+_gene_Chr7-6Mb_reg_882 -- region: 881, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_881.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.682s.\n",
      "-- designing region: 7:48409999-48414999_strand_+_gene_Chr7-6Mb_reg_883 -- region: 882, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_882.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:48414999-48419999_strand_+_gene_Chr7-6Mb_reg_884 -- region: 883, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_883.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 7:48419999-48424999_strand_+_gene_Chr7-6Mb_reg_885 -- region: 884, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_884.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.172s.\n",
      "- Designed 9918 candidate probes in 2.610s.\n",
      "-- designing region: 7:48424999-48429999_strand_+_gene_Chr7-6Mb_reg_886 -- region: 885, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_885.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:48429999-48434999_strand_+_gene_Chr7-6Mb_reg_887 -- region: 886, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_886.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 7:48434999-48439999_strand_+_gene_Chr7-6Mb_reg_888 -- region: 887, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_887.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.109s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:48439999-48444999_strand_+_gene_Chr7-6Mb_reg_889 -- region: 888, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_888.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.659s.\n",
      "-- designing region: 7:48444999-48449999_strand_+_gene_Chr7-6Mb_reg_890 -- region: 889, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_889.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:48449999-48454999_strand_+_gene_Chr7-6Mb_reg_891 -- region: 890, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_890.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:48454999-48459999_strand_+_gene_Chr7-6Mb_reg_892 -- region: 891, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_891.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.165s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 7:48459999-48464999_strand_+_gene_Chr7-6Mb_reg_893 -- region: 892, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_892.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.679s.\n",
      "-- designing region: 7:48464999-48469999_strand_+_gene_Chr7-6Mb_reg_894 -- region: 893, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_893.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:48469999-48474999_strand_+_gene_Chr7-6Mb_reg_895 -- region: 894, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_894.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:48474999-48479999_strand_+_gene_Chr7-6Mb_reg_896 -- region: 895, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_895.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.157s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.587s.\n",
      "-- designing region: 7:48479999-48484999_strand_+_gene_Chr7-6Mb_reg_897 -- region: 896, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_896.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:48484999-48489999_strand_+_gene_Chr7-6Mb_reg_898 -- region: 897, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_897.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:48489999-48494999_strand_+_gene_Chr7-6Mb_reg_899 -- region: 898, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_898.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.102s.\n",
      "- Designed 9918 candidate probes in 2.688s.\n",
      "-- designing region: 7:48494999-48499999_strand_+_gene_Chr7-6Mb_reg_900 -- region: 899, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_899.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.678s.\n",
      "-- designing region: 7:48499999-48504999_strand_+_gene_Chr7-6Mb_reg_901 -- region: 900, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_900.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:48504999-48509999_strand_+_gene_Chr7-6Mb_reg_902 -- region: 901, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_901.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:48509999-48514999_strand_+_gene_Chr7-6Mb_reg_903 -- region: 902, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_902.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.122s.\n",
      "- Designed 9918 candidate probes in 2.702s.\n",
      "-- designing region: 7:48514999-48519999_strand_+_gene_Chr7-6Mb_reg_904 -- region: 903, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_903.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.698s.\n",
      "-- designing region: 7:48519999-48524999_strand_+_gene_Chr7-6Mb_reg_905 -- region: 904, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_904.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:48524999-48529999_strand_+_gene_Chr7-6Mb_reg_906 -- region: 905, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_905.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:48529999-48534999_strand_+_gene_Chr7-6Mb_reg_907 -- region: 906, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_906.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.170s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 7:48534999-48539999_strand_+_gene_Chr7-6Mb_reg_908 -- region: 907, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_907.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.686s.\n",
      "-- designing region: 7:48539999-48544999_strand_+_gene_Chr7-6Mb_reg_909 -- region: 908, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_908.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:48544999-48549999_strand_+_gene_Chr7-6Mb_reg_910 -- region: 909, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_909.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "- Designed 9918 candidate probes in 2.742s.\n",
      "-- designing region: 7:48549999-48554999_strand_+_gene_Chr7-6Mb_reg_911 -- region: 910, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_910.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.615s.\n",
      "-- designing region: 7:48554999-48559999_strand_+_gene_Chr7-6Mb_reg_912 -- region: 911, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_911.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:48559999-48564999_strand_+_gene_Chr7-6Mb_reg_913 -- region: 912, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_912.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:48564999-48569999_strand_+_gene_Chr7-6Mb_reg_914 -- region: 913, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_913.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.109s.\n",
      "- Designed 9918 candidate probes in 2.691s.\n",
      "-- designing region: 7:48569999-48574999_strand_+_gene_Chr7-6Mb_reg_915 -- region: 914, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_914.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.665s.\n",
      "-- designing region: 7:48574999-48579999_strand_+_gene_Chr7-6Mb_reg_916 -- region: 915, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_915.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:48579999-48584999_strand_+_gene_Chr7-6Mb_reg_917 -- region: 916, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_916.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:48584999-48589999_strand_+_gene_Chr7-6Mb_reg_918 -- region: 917, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_917.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "- Designed 9918 candidate probes in 6.603s.\n",
      "-- designing region: 7:48589999-48594999_strand_+_gene_Chr7-6Mb_reg_919 -- region: 918, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_918.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.748s.\n",
      "-- designing region: 7:48594999-48599999_strand_+_gene_Chr7-6Mb_reg_920 -- region: 919, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_919.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:48599999-48604999_strand_+_gene_Chr7-6Mb_reg_921 -- region: 920, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_920.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.099s.\n",
      "- Designed 9918 candidate probes in 2.692s.\n",
      "-- designing region: 7:48604999-48609999_strand_+_gene_Chr7-6Mb_reg_922 -- region: 921, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_921.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.654s.\n",
      "-- designing region: 7:48609999-48614999_strand_+_gene_Chr7-6Mb_reg_923 -- region: 922, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_922.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:48614999-48619999_strand_+_gene_Chr7-6Mb_reg_924 -- region: 923, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_923.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:48619999-48624999_strand_+_gene_Chr7-6Mb_reg_925 -- region: 924, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_924.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "- Designed 9918 candidate probes in 2.719s.\n",
      "-- designing region: 7:48624999-48629999_strand_+_gene_Chr7-6Mb_reg_926 -- region: 925, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_925.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.697s.\n",
      "-- designing region: 7:48629999-48634999_strand_+_gene_Chr7-6Mb_reg_927 -- region: 926, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_926.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:48634999-48639999_strand_+_gene_Chr7-6Mb_reg_928 -- region: 927, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_927.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:48639999-48644999_strand_+_gene_Chr7-6Mb_reg_929 -- region: 928, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_928.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.154s.\n",
      "- Designed 9918 candidate probes in 2.743s.\n",
      "-- designing region: 7:48644999-48649999_strand_+_gene_Chr7-6Mb_reg_930 -- region: 929, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_929.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.691s.\n",
      "-- designing region: 7:48649999-48654999_strand_+_gene_Chr7-6Mb_reg_931 -- region: 930, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_930.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:48654999-48659999_strand_+_gene_Chr7-6Mb_reg_932 -- region: 931, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_931.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:48659999-48664999_strand_+_gene_Chr7-6Mb_reg_933 -- region: 932, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_932.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.171s.\n",
      "- Designed 9918 candidate probes in 2.601s.\n",
      "-- designing region: 7:48664999-48669999_strand_+_gene_Chr7-6Mb_reg_934 -- region: 933, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_933.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:48669999-48674999_strand_+_gene_Chr7-6Mb_reg_935 -- region: 934, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_934.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:48674999-48679999_strand_+_gene_Chr7-6Mb_reg_936 -- region: 935, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_935.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.103s.\n",
      "- Designed 9918 candidate probes in 2.684s.\n",
      "-- designing region: 7:48679999-48684999_strand_+_gene_Chr7-6Mb_reg_937 -- region: 936, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_936.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.671s.\n",
      "-- designing region: 7:48684999-48689999_strand_+_gene_Chr7-6Mb_reg_938 -- region: 937, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_937.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:48689999-48694999_strand_+_gene_Chr7-6Mb_reg_939 -- region: 938, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_938.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:48694999-48699999_strand_+_gene_Chr7-6Mb_reg_940 -- region: 939, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_939.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.157s.\n",
      "- Designed 9918 candidate probes in 2.739s.\n",
      "-- designing region: 7:48699999-48704999_strand_+_gene_Chr7-6Mb_reg_941 -- region: 940, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_940.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.698s.\n",
      "-- designing region: 7:48704999-48709999_strand_+_gene_Chr7-6Mb_reg_942 -- region: 941, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_941.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:48709999-48714999_strand_+_gene_Chr7-6Mb_reg_943 -- region: 942, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_942.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:48714999-48719999_strand_+_gene_Chr7-6Mb_reg_944 -- region: 943, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_943.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.168s.\n",
      "- Designed 9918 candidate probes in 2.595s.\n",
      "-- designing region: 7:48719999-48724999_strand_+_gene_Chr7-6Mb_reg_945 -- region: 944, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_944.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:48724999-48729999_strand_+_gene_Chr7-6Mb_reg_946 -- region: 945, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_945.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:48729999-48734999_strand_+_gene_Chr7-6Mb_reg_947 -- region: 946, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_946.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 7:48734999-48739999_strand_+_gene_Chr7-6Mb_reg_948 -- region: 947, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_947.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.635s.\n",
      "-- designing region: 7:48739999-48744999_strand_+_gene_Chr7-6Mb_reg_949 -- region: 948, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_948.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:48744999-48749999_strand_+_gene_Chr7-6Mb_reg_950 -- region: 949, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_949.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:48749999-48754999_strand_+_gene_Chr7-6Mb_reg_951 -- region: 950, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_950.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.117s.\n",
      "- Designed 9918 candidate probes in 2.708s.\n",
      "-- designing region: 7:48754999-48759999_strand_+_gene_Chr7-6Mb_reg_952 -- region: 951, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_951.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.692s.\n",
      "-- designing region: 7:48759999-48764999_strand_+_gene_Chr7-6Mb_reg_953 -- region: 952, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_952.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:48764999-48769999_strand_+_gene_Chr7-6Mb_reg_954 -- region: 953, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_953.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:48769999-48774999_strand_+_gene_Chr7-6Mb_reg_955 -- region: 954, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_954.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "- Designed 9918 candidate probes in 2.721s.\n",
      "-- designing region: 7:48774999-48779999_strand_+_gene_Chr7-6Mb_reg_956 -- region: 955, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_955.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.692s.\n",
      "-- designing region: 7:48779999-48784999_strand_+_gene_Chr7-6Mb_reg_957 -- region: 956, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_956.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:48784999-48789999_strand_+_gene_Chr7-6Mb_reg_958 -- region: 957, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_957.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:48789999-48794999_strand_+_gene_Chr7-6Mb_reg_959 -- region: 958, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_958.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.156s.\n",
      "- Designed 9918 candidate probes in 2.742s.\n",
      "-- designing region: 7:48794999-48799999_strand_+_gene_Chr7-6Mb_reg_960 -- region: 959, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_959.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.642s.\n",
      "-- designing region: 7:48799999-48804999_strand_+_gene_Chr7-6Mb_reg_961 -- region: 960, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_960.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:48804999-48809999_strand_+_gene_Chr7-6Mb_reg_962 -- region: 961, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_961.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:48809999-48814999_strand_+_gene_Chr7-6Mb_reg_963 -- region: 962, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_962.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.617s.\n",
      "-- designing region: 7:48814999-48819999_strand_+_gene_Chr7-6Mb_reg_964 -- region: 963, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_963.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:48819999-48824999_strand_+_gene_Chr7-6Mb_reg_965 -- region: 964, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_964.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:48824999-48829999_strand_+_gene_Chr7-6Mb_reg_966 -- region: 965, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_965.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Designed 9918 candidate probes in 2.687s.\n",
      "-- designing region: 7:48829999-48834999_strand_+_gene_Chr7-6Mb_reg_967 -- region: 966, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_966.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.690s.\n",
      "-- designing region: 7:48834999-48839999_strand_+_gene_Chr7-6Mb_reg_968 -- region: 967, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_967.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:48839999-48844999_strand_+_gene_Chr7-6Mb_reg_969 -- region: 968, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_968.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:48844999-48849999_strand_+_gene_Chr7-6Mb_reg_970 -- region: 969, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_969.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.157s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 7:48849999-48854999_strand_+_gene_Chr7-6Mb_reg_971 -- region: 970, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_970.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.672s.\n",
      "-- designing region: 7:48854999-48859999_strand_+_gene_Chr7-6Mb_reg_972 -- region: 971, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_971.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:48859999-48864999_strand_+_gene_Chr7-6Mb_reg_973 -- region: 972, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_972.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:48864999-48869999_strand_+_gene_Chr7-6Mb_reg_974 -- region: 973, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_973.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.159s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 7:48869999-48874999_strand_+_gene_Chr7-6Mb_reg_975 -- region: 974, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_974.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.692s.\n",
      "-- designing region: 7:48874999-48879999_strand_+_gene_Chr7-6Mb_reg_976 -- region: 975, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_975.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:48879999-48884999_strand_+_gene_Chr7-6Mb_reg_977 -- region: 976, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_976.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:48884999-48889999_strand_+_gene_Chr7-6Mb_reg_978 -- region: 977, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_977.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.161s.\n",
      "- Designed 9918 candidate probes in 2.749s.\n",
      "-- designing region: 7:48889999-48894999_strand_+_gene_Chr7-6Mb_reg_979 -- region: 978, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_978.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.701s.\n",
      "-- designing region: 7:48894999-48899999_strand_+_gene_Chr7-6Mb_reg_980 -- region: 979, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_979.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:48899999-48904999_strand_+_gene_Chr7-6Mb_reg_981 -- region: 980, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_980.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:48904999-48909999_strand_+_gene_Chr7-6Mb_reg_982 -- region: 981, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_981.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Designed 9918 candidate probes in 2.747s.\n",
      "-- designing region: 7:48909999-48914999_strand_+_gene_Chr7-6Mb_reg_983 -- region: 982, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_982.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.690s.\n",
      "-- designing region: 7:48914999-48919999_strand_+_gene_Chr7-6Mb_reg_984 -- region: 983, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_983.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:48919999-48924999_strand_+_gene_Chr7-6Mb_reg_985 -- region: 984, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_984.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:48924999-48929999_strand_+_gene_Chr7-6Mb_reg_986 -- region: 985, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_985.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.159s.\n",
      "- Designed 9918 candidate probes in 2.751s.\n",
      "-- designing region: 7:48929999-48934999_strand_+_gene_Chr7-6Mb_reg_987 -- region: 986, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_986.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.679s.\n",
      "-- designing region: 7:48934999-48939999_strand_+_gene_Chr7-6Mb_reg_988 -- region: 987, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_987.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:48939999-48944999_strand_+_gene_Chr7-6Mb_reg_989 -- region: 988, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_988.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:48944999-48949999_strand_+_gene_Chr7-6Mb_reg_990 -- region: 989, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_989.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.157s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 7:48949999-48954999_strand_+_gene_Chr7-6Mb_reg_991 -- region: 990, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_990.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.686s.\n",
      "-- designing region: 7:48954999-48959999_strand_+_gene_Chr7-6Mb_reg_992 -- region: 991, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_991.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 7:48959999-48964999_strand_+_gene_Chr7-6Mb_reg_993 -- region: 992, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_992.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:48964999-48969999_strand_+_gene_Chr7-6Mb_reg_994 -- region: 993, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_993.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.168s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 7:48969999-48974999_strand_+_gene_Chr7-6Mb_reg_995 -- region: 994, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_994.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.710s.\n",
      "-- designing region: 7:48974999-48979999_strand_+_gene_Chr7-6Mb_reg_996 -- region: 995, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_995.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:48979999-48984999_strand_+_gene_Chr7-6Mb_reg_997 -- region: 996, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_996.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:48984999-48989999_strand_+_gene_Chr7-6Mb_reg_998 -- region: 997, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_997.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.155s.\n",
      "- Designed 9918 candidate probes in 2.745s.\n",
      "-- designing region: 7:48989999-48994999_strand_+_gene_Chr7-6Mb_reg_999 -- region: 998, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_998.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.706s.\n",
      "-- designing region: 7:48994999-48999999_strand_+_gene_Chr7-6Mb_reg_1000 -- region: 999, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_999.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:48999999-49004999_strand_+_gene_Chr7-6Mb_reg_1001 -- region: 1000, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1000.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:49004999-49009999_strand_+_gene_Chr7-6Mb_reg_1002 -- region: 1001, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1001.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.163s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 7:49009999-49014999_strand_+_gene_Chr7-6Mb_reg_1003 -- region: 1002, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1002.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.706s.\n",
      "-- designing region: 7:49014999-49019999_strand_+_gene_Chr7-6Mb_reg_1004 -- region: 1003, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1003.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 7:49019999-49024999_strand_+_gene_Chr7-6Mb_reg_1005 -- region: 1004, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1004.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:49024999-49029999_strand_+_gene_Chr7-6Mb_reg_1006 -- region: 1005, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1005.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "- Designed 9918 candidate probes in 2.734s.\n",
      "-- designing region: 7:49029999-49034999_strand_+_gene_Chr7-6Mb_reg_1007 -- region: 1006, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1006.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.705s.\n",
      "-- designing region: 7:49034999-49039999_strand_+_gene_Chr7-6Mb_reg_1008 -- region: 1007, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1007.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:49039999-49044999_strand_+_gene_Chr7-6Mb_reg_1009 -- region: 1008, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1008.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:49044999-49049999_strand_+_gene_Chr7-6Mb_reg_1010 -- region: 1009, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1009.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.158s.\n",
      "- Designed 9918 candidate probes in 2.745s.\n",
      "-- designing region: 7:49049999-49054999_strand_+_gene_Chr7-6Mb_reg_1011 -- region: 1010, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1010.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:49054999-49059999_strand_+_gene_Chr7-6Mb_reg_1012 -- region: 1011, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1011.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:49059999-49064999_strand_+_gene_Chr7-6Mb_reg_1013 -- region: 1012, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1012.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:49064999-49069999_strand_+_gene_Chr7-6Mb_reg_1014 -- region: 1013, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1013.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.112s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:49069999-49074999_strand_+_gene_Chr7-6Mb_reg_1015 -- region: 1014, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1014.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.691s.\n",
      "-- designing region: 7:49074999-49079999_strand_+_gene_Chr7-6Mb_reg_1016 -- region: 1015, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1015.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:49079999-49084999_strand_+_gene_Chr7-6Mb_reg_1017 -- region: 1016, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1016.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:49084999-49089999_strand_+_gene_Chr7-6Mb_reg_1018 -- region: 1017, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1017.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "- Designed 9918 candidate probes in 2.659s.\n",
      "-- designing region: 7:49089999-49094999_strand_+_gene_Chr7-6Mb_reg_1019 -- region: 1018, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1018.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.729s.\n",
      "-- designing region: 7:49094999-49099999_strand_+_gene_Chr7-6Mb_reg_1020 -- region: 1019, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1019.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:49099999-49104999_strand_+_gene_Chr7-6Mb_reg_1021 -- region: 1020, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1020.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:49104999-49109999_strand_+_gene_Chr7-6Mb_reg_1022 -- region: 1021, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1021.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.169s.\n",
      "- Designed 9918 candidate probes in 2.596s.\n",
      "-- designing region: 7:49109999-49114999_strand_+_gene_Chr7-6Mb_reg_1023 -- region: 1022, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1022.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 7:49114999-49119999_strand_+_gene_Chr7-6Mb_reg_1024 -- region: 1023, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1023.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:49119999-49124999_strand_+_gene_Chr7-6Mb_reg_1025 -- region: 1024, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1024.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.102s.\n",
      "- Designed 9918 candidate probes in 2.683s.\n",
      "-- designing region: 7:49124999-49129999_strand_+_gene_Chr7-6Mb_reg_1026 -- region: 1025, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1025.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.671s.\n",
      "-- designing region: 7:49129999-49134999_strand_+_gene_Chr7-6Mb_reg_1027 -- region: 1026, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1026.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:49134999-49139999_strand_+_gene_Chr7-6Mb_reg_1028 -- region: 1027, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1027.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 7:49139999-49144999_strand_+_gene_Chr7-6Mb_reg_1029 -- region: 1028, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1028.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.164s.\n",
      "- Designed 9918 candidate probes in 2.746s.\n",
      "-- designing region: 7:49144999-49149999_strand_+_gene_Chr7-6Mb_reg_1030 -- region: 1029, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1029.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.684s.\n",
      "-- designing region: 7:49149999-49154999_strand_+_gene_Chr7-6Mb_reg_1031 -- region: 1030, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1030.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 7:49154999-49159999_strand_+_gene_Chr7-6Mb_reg_1032 -- region: 1031, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1031.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 7:49159999-49164999_strand_+_gene_Chr7-6Mb_reg_1033 -- region: 1032, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1032.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.175s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 7:49164999-49169999_strand_+_gene_Chr7-6Mb_reg_1034 -- region: 1033, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1033.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.711s.\n",
      "-- designing region: 7:49169999-49174999_strand_+_gene_Chr7-6Mb_reg_1035 -- region: 1034, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1034.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:49174999-49179999_strand_+_gene_Chr7-6Mb_reg_1036 -- region: 1035, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1035.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 7:49179999-49184999_strand_+_gene_Chr7-6Mb_reg_1037 -- region: 1036, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1036.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.611s.\n",
      "-- designing region: 7:49184999-49189999_strand_+_gene_Chr7-6Mb_reg_1038 -- region: 1037, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1037.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 7:49189999-49194999_strand_+_gene_Chr7-6Mb_reg_1039 -- region: 1038, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1038.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:49194999-49199999_strand_+_gene_Chr7-6Mb_reg_1040 -- region: 1039, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1039.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Designed 9918 candidate probes in 2.686s.\n",
      "-- designing region: 7:49199999-49204999_strand_+_gene_Chr7-6Mb_reg_1041 -- region: 1040, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1040.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.641s.\n",
      "-- designing region: 7:49204999-49209999_strand_+_gene_Chr7-6Mb_reg_1042 -- region: 1041, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1041.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.748s.\n",
      "-- designing region: 7:49209999-49214999_strand_+_gene_Chr7-6Mb_reg_1043 -- region: 1042, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1042.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 7:49214999-49219999_strand_+_gene_Chr7-6Mb_reg_1044 -- region: 1043, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1043.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:49219999-49224999_strand_+_gene_Chr7-6Mb_reg_1045 -- region: 1044, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1044.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.684s.\n",
      "-- designing region: 7:49224999-49229999_strand_+_gene_Chr7-6Mb_reg_1046 -- region: 1045, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1045.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:49229999-49234999_strand_+_gene_Chr7-6Mb_reg_1047 -- region: 1046, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1046.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 7:49234999-49239999_strand_+_gene_Chr7-6Mb_reg_1048 -- region: 1047, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1047.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.177s.\n",
      "- Designed 9918 candidate probes in 2.731s.\n",
      "-- designing region: 7:49239999-49244999_strand_+_gene_Chr7-6Mb_reg_1049 -- region: 1048, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1048.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.730s.\n",
      "-- designing region: 7:49244999-49249999_strand_+_gene_Chr7-6Mb_reg_1050 -- region: 1049, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1049.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:49249999-49254999_strand_+_gene_Chr7-6Mb_reg_1051 -- region: 1050, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1050.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:49254999-49259999_strand_+_gene_Chr7-6Mb_reg_1052 -- region: 1051, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1051.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.161s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 7:49259999-49264999_strand_+_gene_Chr7-6Mb_reg_1053 -- region: 1052, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1052.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.712s.\n",
      "-- designing region: 7:49264999-49269999_strand_+_gene_Chr7-6Mb_reg_1054 -- region: 1053, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1053.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:49269999-49274999_strand_+_gene_Chr7-6Mb_reg_1055 -- region: 1054, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1054.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:49274999-49279999_strand_+_gene_Chr7-6Mb_reg_1056 -- region: 1055, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1055.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.141s.\n",
      "- Designed 9918 candidate probes in 2.714s.\n",
      "-- designing region: 7:49279999-49284999_strand_+_gene_Chr7-6Mb_reg_1057 -- region: 1056, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1056.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.733s.\n",
      "-- designing region: 7:49284999-49289999_strand_+_gene_Chr7-6Mb_reg_1058 -- region: 1057, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1057.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.807s.\n",
      "-- designing region: 7:49289999-49294999_strand_+_gene_Chr7-6Mb_reg_1059 -- region: 1058, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1058.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 7:49294999-49299999_strand_+_gene_Chr7-6Mb_reg_1060 -- region: 1059, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1059.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "- Designed 9918 candidate probes in 2.748s.\n",
      "-- designing region: 7:49299999-49304999_strand_+_gene_Chr7-6Mb_reg_1061 -- region: 1060, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1060.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.702s.\n",
      "-- designing region: 7:49304999-49309999_strand_+_gene_Chr7-6Mb_reg_1062 -- region: 1061, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1061.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:49309999-49314999_strand_+_gene_Chr7-6Mb_reg_1063 -- region: 1062, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1062.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:49314999-49319999_strand_+_gene_Chr7-6Mb_reg_1064 -- region: 1063, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1063.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Designed 9918 candidate probes in 2.720s.\n",
      "-- designing region: 7:49319999-49324999_strand_+_gene_Chr7-6Mb_reg_1065 -- region: 1064, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1064.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.693s.\n",
      "-- designing region: 7:49324999-49329999_strand_+_gene_Chr7-6Mb_reg_1066 -- region: 1065, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1065.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:49329999-49334999_strand_+_gene_Chr7-6Mb_reg_1067 -- region: 1066, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1066.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:49334999-49339999_strand_+_gene_Chr7-6Mb_reg_1068 -- region: 1067, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1067.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.109s.\n",
      "- Designed 9918 candidate probes in 2.719s.\n",
      "-- designing region: 7:49339999-49344999_strand_+_gene_Chr7-6Mb_reg_1069 -- region: 1068, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1068.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.686s.\n",
      "-- designing region: 7:49344999-49349999_strand_+_gene_Chr7-6Mb_reg_1070 -- region: 1069, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1069.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:49349999-49354999_strand_+_gene_Chr7-6Mb_reg_1071 -- region: 1070, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1070.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 7:49354999-49359999_strand_+_gene_Chr7-6Mb_reg_1072 -- region: 1071, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1071.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.114s.\n",
      "- Designed 9918 candidate probes in 2.724s.\n",
      "-- designing region: 7:49359999-49364999_strand_+_gene_Chr7-6Mb_reg_1073 -- region: 1072, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1072.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.680s.\n",
      "-- designing region: 7:49364999-49369999_strand_+_gene_Chr7-6Mb_reg_1074 -- region: 1073, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1073.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:49369999-49374999_strand_+_gene_Chr7-6Mb_reg_1075 -- region: 1074, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1074.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.802s.\n",
      "-- designing region: 7:49374999-49379999_strand_+_gene_Chr7-6Mb_reg_1076 -- region: 1075, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1075.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:49379999-49384999_strand_+_gene_Chr7-6Mb_reg_1077 -- region: 1076, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1076.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.643s.\n",
      "-- designing region: 7:49384999-49389999_strand_+_gene_Chr7-6Mb_reg_1078 -- region: 1077, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1077.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.164s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:49389999-49394999_strand_+_gene_Chr7-6Mb_reg_1079 -- region: 1078, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1078.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 7:49394999-49399999_strand_+_gene_Chr7-6Mb_reg_1080 -- region: 1079, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1079.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 7:49399999-49404999_strand_+_gene_Chr7-6Mb_reg_1081 -- region: 1080, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1080.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.648s.\n",
      "-- designing region: 7:49404999-49409999_strand_+_gene_Chr7-6Mb_reg_1082 -- region: 1081, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1081.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 7:49409999-49414999_strand_+_gene_Chr7-6Mb_reg_1083 -- region: 1082, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1082.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.806s.\n",
      "-- designing region: 7:49414999-49419999_strand_+_gene_Chr7-6Mb_reg_1084 -- region: 1083, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1083.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:49419999-49424999_strand_+_gene_Chr7-6Mb_reg_1085 -- region: 1084, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1084.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:49424999-49429999_strand_+_gene_Chr7-6Mb_reg_1086 -- region: 1085, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1085.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.714s.\n",
      "-- designing region: 7:49429999-49434999_strand_+_gene_Chr7-6Mb_reg_1087 -- region: 1086, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1086.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.805s.\n",
      "-- designing region: 7:49434999-49439999_strand_+_gene_Chr7-6Mb_reg_1088 -- region: 1087, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1087.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:49439999-49444999_strand_+_gene_Chr7-6Mb_reg_1089 -- region: 1088, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1088.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.105s.\n",
      "- Designed 9918 candidate probes in 2.713s.\n",
      "-- designing region: 7:49444999-49449999_strand_+_gene_Chr7-6Mb_reg_1090 -- region: 1089, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1089.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.704s.\n",
      "-- designing region: 7:49449999-49454999_strand_+_gene_Chr7-6Mb_reg_1091 -- region: 1090, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1090.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:49454999-49459999_strand_+_gene_Chr7-6Mb_reg_1092 -- region: 1091, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1091.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:49459999-49464999_strand_+_gene_Chr7-6Mb_reg_1093 -- region: 1092, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1092.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.120s.\n",
      "- Designed 9918 candidate probes in 2.724s.\n",
      "-- designing region: 7:49464999-49469999_strand_+_gene_Chr7-6Mb_reg_1094 -- region: 1093, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1093.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.694s.\n",
      "-- designing region: 7:49469999-49474999_strand_+_gene_Chr7-6Mb_reg_1095 -- region: 1094, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1094.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.802s.\n",
      "-- designing region: 7:49474999-49479999_strand_+_gene_Chr7-6Mb_reg_1096 -- region: 1095, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1095.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:49479999-49484999_strand_+_gene_Chr7-6Mb_reg_1097 -- region: 1096, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1096.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.117s.\n",
      "- Designed 9918 candidate probes in 2.721s.\n",
      "-- designing region: 7:49484999-49489999_strand_+_gene_Chr7-6Mb_reg_1098 -- region: 1097, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1097.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:49489999-49494999_strand_+_gene_Chr7-6Mb_reg_1099 -- region: 1098, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1098.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 7:49494999-49499999_strand_+_gene_Chr7-6Mb_reg_1100 -- region: 1099, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1099.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:49499999-49504999_strand_+_gene_Chr7-6Mb_reg_1101 -- region: 1100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1100.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:49504999-49509999_strand_+_gene_Chr7-6Mb_reg_1102 -- region: 1101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1101.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.159s.\n",
      "- Designed 9918 candidate probes in 2.620s.\n",
      "-- designing region: 7:49509999-49514999_strand_+_gene_Chr7-6Mb_reg_1103 -- region: 1102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1102.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 7:49514999-49519999_strand_+_gene_Chr7-6Mb_reg_1104 -- region: 1103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1103.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:49519999-49524999_strand_+_gene_Chr7-6Mb_reg_1105 -- region: 1104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1104.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 7:49524999-49529999_strand_+_gene_Chr7-6Mb_reg_1106 -- region: 1105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1105.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 7:49529999-49534999_strand_+_gene_Chr7-6Mb_reg_1107 -- region: 1106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1106.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.711s.\n",
      "-- designing region: 7:49534999-49539999_strand_+_gene_Chr7-6Mb_reg_1108 -- region: 1107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1107.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:49539999-49544999_strand_+_gene_Chr7-6Mb_reg_1109 -- region: 1108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1108.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 7:49544999-49549999_strand_+_gene_Chr7-6Mb_reg_1110 -- region: 1109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1109.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "- Designed 9918 candidate probes in 2.738s.\n",
      "-- designing region: 7:49549999-49554999_strand_+_gene_Chr7-6Mb_reg_1111 -- region: 1110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1110.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.706s.\n",
      "-- designing region: 7:49554999-49559999_strand_+_gene_Chr7-6Mb_reg_1112 -- region: 1111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1111.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 7:49559999-49564999_strand_+_gene_Chr7-6Mb_reg_1113 -- region: 1112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1112.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 7:49564999-49569999_strand_+_gene_Chr7-6Mb_reg_1114 -- region: 1113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1113.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "- Designed 9918 candidate probes in 2.726s.\n",
      "-- designing region: 7:49569999-49574999_strand_+_gene_Chr7-6Mb_reg_1115 -- region: 1114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1114.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.698s.\n",
      "-- designing region: 7:49574999-49579999_strand_+_gene_Chr7-6Mb_reg_1116 -- region: 1115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1115.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 7:49579999-49584999_strand_+_gene_Chr7-6Mb_reg_1117 -- region: 1116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1116.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:49584999-49589999_strand_+_gene_Chr7-6Mb_reg_1118 -- region: 1117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1117.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Designed 9918 candidate probes in 2.713s.\n",
      "-- designing region: 7:49589999-49594999_strand_+_gene_Chr7-6Mb_reg_1119 -- region: 1118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1118.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.682s.\n",
      "-- designing region: 7:49594999-49599999_strand_+_gene_Chr7-6Mb_reg_1120 -- region: 1119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1119.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.806s.\n",
      "-- designing region: 7:49599999-49604999_strand_+_gene_Chr7-6Mb_reg_1121 -- region: 1120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1120.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 7:49604999-49609999_strand_+_gene_Chr7-6Mb_reg_1122 -- region: 1121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1121.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 7:49609999-49614999_strand_+_gene_Chr7-6Mb_reg_1123 -- region: 1122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1122.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 7:49614999-49619999_strand_+_gene_Chr7-6Mb_reg_1124 -- region: 1123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1123.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.725s.\n",
      "-- designing region: 7:49619999-49624999_strand_+_gene_Chr7-6Mb_reg_1125 -- region: 1124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1124.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 7:49624999-49629999_strand_+_gene_Chr7-6Mb_reg_1126 -- region: 1125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1125.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 7:49629999-49634999_strand_+_gene_Chr7-6Mb_reg_1127 -- region: 1126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1126.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "- Designed 9918 candidate probes in 2.636s.\n",
      "-- designing region: 7:49634999-49639999_strand_+_gene_Chr7-6Mb_reg_1128 -- region: 1127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1127.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.745s.\n",
      "-- designing region: 7:49639999-49644999_strand_+_gene_Chr7-6Mb_reg_1129 -- region: 1128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1128.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 7:49644999-49649999_strand_+_gene_Chr7-6Mb_reg_1130 -- region: 1129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1129.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 7:49649999-49654999_strand_+_gene_Chr7-6Mb_reg_1131 -- region: 1130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1130.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "- Designed 9918 candidate probes in 2.739s.\n",
      "-- designing region: 7:49654999-49659999_strand_+_gene_Chr7-6Mb_reg_1132 -- region: 1131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1131.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.702s.\n",
      "-- designing region: 7:49659999-49664999_strand_+_gene_Chr7-6Mb_reg_1133 -- region: 1132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1132.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.812s.\n",
      "-- designing region: 7:49664999-49669999_strand_+_gene_Chr7-6Mb_reg_1134 -- region: 1133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1133.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.802s.\n",
      "-- designing region: 7:49669999-49674999_strand_+_gene_Chr7-6Mb_reg_1135 -- region: 1134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1134.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "- Designed 9918 candidate probes in 2.752s.\n",
      "-- designing region: 7:49674999-49679999_strand_+_gene_Chr7-6Mb_reg_1136 -- region: 1135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1135.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.691s.\n",
      "-- designing region: 7:49679999-49684999_strand_+_gene_Chr7-6Mb_reg_1137 -- region: 1136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1136.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 7.379s.\n",
      "-- designing region: 7:49684999-49689999_strand_+_gene_Chr7-6Mb_reg_1138 -- region: 1137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1137.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.679s.\n",
      "-- designing region: 7:49689999-49694999_strand_+_gene_Chr7-6Mb_reg_1139 -- region: 1138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1138.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 7:49694999-49699999_strand_+_gene_Chr7-6Mb_reg_1140 -- region: 1139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1139.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:49699999-49704999_strand_+_gene_Chr7-6Mb_reg_1141 -- region: 1140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1140.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.096s.\n",
      "- Designed 9918 candidate probes in 2.689s.\n",
      "-- designing region: 7:49704999-49709999_strand_+_gene_Chr7-6Mb_reg_1142 -- region: 1141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1141.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 7.729s.\n",
      "-- designing region: 7:49709999-49714999_strand_+_gene_Chr7-6Mb_reg_1143 -- region: 1142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1142.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 7:49714999-49719999_strand_+_gene_Chr7-6Mb_reg_1144 -- region: 1143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1143.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:49719999-49724999_strand_+_gene_Chr7-6Mb_reg_1145 -- region: 1144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1144.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Designed 9918 candidate probes in 2.723s.\n",
      "-- designing region: 7:49724999-49729999_strand_+_gene_Chr7-6Mb_reg_1146 -- region: 1145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1145.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.667s.\n",
      "-- designing region: 7:49729999-49734999_strand_+_gene_Chr7-6Mb_reg_1147 -- region: 1146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1146.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:49734999-49739999_strand_+_gene_Chr7-6Mb_reg_1148 -- region: 1147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1147.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:49739999-49744999_strand_+_gene_Chr7-6Mb_reg_1149 -- region: 1148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1148.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "- Designed 9918 candidate probes in 2.707s.\n",
      "-- designing region: 7:49744999-49749999_strand_+_gene_Chr7-6Mb_reg_1150 -- region: 1149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1149.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.697s.\n",
      "-- designing region: 7:49749999-49754999_strand_+_gene_Chr7-6Mb_reg_1151 -- region: 1150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1150.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:49754999-49759999_strand_+_gene_Chr7-6Mb_reg_1152 -- region: 1151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1151.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:49759999-49764999_strand_+_gene_Chr7-6Mb_reg_1153 -- region: 1152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1152.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.177s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 7:49764999-49769999_strand_+_gene_Chr7-6Mb_reg_1154 -- region: 1153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1153.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.694s.\n",
      "-- designing region: 7:49769999-49774999_strand_+_gene_Chr7-6Mb_reg_1155 -- region: 1154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1154.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:49774999-49779999_strand_+_gene_Chr7-6Mb_reg_1156 -- region: 1155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1155.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:49779999-49784999_strand_+_gene_Chr7-6Mb_reg_1157 -- region: 1156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1156.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.163s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 7:49784999-49789999_strand_+_gene_Chr7-6Mb_reg_1158 -- region: 1157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1157.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.696s.\n",
      "-- designing region: 7:49789999-49794999_strand_+_gene_Chr7-6Mb_reg_1159 -- region: 1158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1158.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:49794999-49799999_strand_+_gene_Chr7-6Mb_reg_1160 -- region: 1159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1159.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.746s.\n",
      "-- designing region: 7:49799999-49804999_strand_+_gene_Chr7-6Mb_reg_1161 -- region: 1160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1160.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.175s.\n",
      "- Designed 9918 candidate probes in 2.614s.\n",
      "-- designing region: 7:49804999-49809999_strand_+_gene_Chr7-6Mb_reg_1162 -- region: 1161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1161.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:49809999-49814999_strand_+_gene_Chr7-6Mb_reg_1163 -- region: 1162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1162.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 7:49814999-49819999_strand_+_gene_Chr7-6Mb_reg_1164 -- region: 1163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1163.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.104s.\n",
      "- Designed 9918 candidate probes in 2.681s.\n",
      "-- designing region: 7:49819999-49824999_strand_+_gene_Chr7-6Mb_reg_1165 -- region: 1164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1164.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.681s.\n",
      "-- designing region: 7:49824999-49829999_strand_+_gene_Chr7-6Mb_reg_1166 -- region: 1165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1165.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 7:49829999-49834999_strand_+_gene_Chr7-6Mb_reg_1167 -- region: 1166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1166.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:49834999-49839999_strand_+_gene_Chr7-6Mb_reg_1168 -- region: 1167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1167.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.145s.\n",
      "- Designed 9918 candidate probes in 2.738s.\n",
      "-- designing region: 7:49839999-49844999_strand_+_gene_Chr7-6Mb_reg_1169 -- region: 1168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1168.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.687s.\n",
      "-- designing region: 7:49844999-49849999_strand_+_gene_Chr7-6Mb_reg_1170 -- region: 1169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1169.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:49849999-49854999_strand_+_gene_Chr7-6Mb_reg_1171 -- region: 1170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1170.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 7:49854999-49859999_strand_+_gene_Chr7-6Mb_reg_1172 -- region: 1171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1171.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.153s.\n",
      "- Designed 9918 candidate probes in 2.743s.\n",
      "-- designing region: 7:49859999-49864999_strand_+_gene_Chr7-6Mb_reg_1173 -- region: 1172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1172.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.695s.\n",
      "-- designing region: 7:49864999-49869999_strand_+_gene_Chr7-6Mb_reg_1174 -- region: 1173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1173.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 7:49869999-49874999_strand_+_gene_Chr7-6Mb_reg_1175 -- region: 1174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1174.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 7:49874999-49879999_strand_+_gene_Chr7-6Mb_reg_1176 -- region: 1175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1175.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.149s.\n",
      "- Designed 9918 candidate probes in 2.733s.\n",
      "-- designing region: 7:49879999-49884999_strand_+_gene_Chr7-6Mb_reg_1177 -- region: 1176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1176.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.683s.\n",
      "-- designing region: 7:49884999-49889999_strand_+_gene_Chr7-6Mb_reg_1178 -- region: 1177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1177.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 7:49889999-49894999_strand_+_gene_Chr7-6Mb_reg_1179 -- region: 1178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1178.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 3.704s.\n",
      "-- designing region: 7:49894999-49899999_strand_+_gene_Chr7-6Mb_reg_1180 -- region: 1179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1179.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 7:49899999-49904999_strand_+_gene_Chr7-6Mb_reg_1181 -- region: 1180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1180.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.719s.\n",
      "-- designing region: 7:49904999-49909999_strand_+_gene_Chr7-6Mb_reg_1182 -- region: 1181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1181.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:49909999-49914999_strand_+_gene_Chr7-6Mb_reg_1183 -- region: 1182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1182.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.745s.\n",
      "-- designing region: 7:49914999-49919999_strand_+_gene_Chr7-6Mb_reg_1184 -- region: 1183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1183.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.625s.\n",
      "-- designing region: 7:49919999-49924999_strand_+_gene_Chr7-6Mb_reg_1185 -- region: 1184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1184.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 7:49924999-49929999_strand_+_gene_Chr7-6Mb_reg_1186 -- region: 1185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1185.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 7:49929999-49934999_strand_+_gene_Chr7-6Mb_reg_1187 -- region: 1186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1186.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:49934999-49939999_strand_+_gene_Chr7-6Mb_reg_1188 -- region: 1187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1187.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.622s.\n",
      "-- designing region: 7:49939999-49944999_strand_+_gene_Chr7-6Mb_reg_1189 -- region: 1188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1188.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 7:49944999-49949999_strand_+_gene_Chr7-6Mb_reg_1190 -- region: 1189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1189.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 7:49949999-49954999_strand_+_gene_Chr7-6Mb_reg_1191 -- region: 1190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1190.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.103s.\n",
      "- Designed 9918 candidate probes in 2.692s.\n",
      "-- designing region: 7:49954999-49959999_strand_+_gene_Chr7-6Mb_reg_1192 -- region: 1191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1191.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.683s.\n",
      "-- designing region: 7:49959999-49964999_strand_+_gene_Chr7-6Mb_reg_1193 -- region: 1192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1192.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 7:49964999-49969999_strand_+_gene_Chr7-6Mb_reg_1194 -- region: 1193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1193.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 7:49969999-49974999_strand_+_gene_Chr7-6Mb_reg_1195 -- region: 1194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1194.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.146s.\n",
      "- Designed 9918 candidate probes in 2.736s.\n",
      "-- designing region: 7:49974999-49979999_strand_+_gene_Chr7-6Mb_reg_1196 -- region: 1195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1195.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.707s.\n",
      "-- designing region: 7:49979999-49984999_strand_+_gene_Chr7-6Mb_reg_1197 -- region: 1196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1196.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 7:49984999-49989999_strand_+_gene_Chr7-6Mb_reg_1198 -- region: 1197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1197.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 7:49989999-49994999_strand_+_gene_Chr7-6Mb_reg_1199 -- region: 1198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1198.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.148s.\n",
      "- Designed 9918 candidate probes in 2.735s.\n",
      "-- designing region: 7:49994999-49999999_strand_+_gene_Chr7-6Mb_reg_1200 -- region: 1199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\sequences\\Chr7-6Mb_reg_1199.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.689s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\reports\\mouse_5Mb_5kb.pbr\n",
      "Wall time: 1h 1min 1s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "pb_designer.computeOTmaps() # load the tables \n",
    "pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e23bacce",
   "metadata": {},
   "source": [
    "### check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "b2307763",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 7:43999999-44004999_strand_+_gene_Chr7-6Mb_reg_1, 7920 candidate probes\n",
      "--- 2978 probes passed check_dic selection.\n",
      "finish in 0.332s, 52 probes kept.\n",
      "-- check region:1 7:44004999-44009999_strand_+_gene_Chr7-6Mb_reg_2, 9432 candidate probes\n",
      "--- 3482 probes passed check_dic selection.\n",
      "finish in 0.293s, 60 probes kept.\n",
      "-- check region:2 7:44009999-44014999_strand_+_gene_Chr7-6Mb_reg_3, 7796 candidate probes\n",
      "--- 2450 probes passed check_dic selection.\n",
      "finish in 0.212s, 49 probes kept.\n",
      "-- check region:3 7:44014999-44019999_strand_+_gene_Chr7-6Mb_reg_4, 8486 candidate probes\n",
      "--- 2820 probes passed check_dic selection.\n",
      "finish in 0.237s, 52 probes kept.\n",
      "-- check region:4 7:44019999-44024999_strand_+_gene_Chr7-6Mb_reg_5, 5230 candidate probes\n",
      "--- 1386 probes passed check_dic selection.\n",
      "finish in 0.135s, 18 probes kept.\n",
      "-- check region:5 7:44024999-44029999_strand_+_gene_Chr7-6Mb_reg_6, 6334 candidate probes\n",
      "--- 1102 probes passed check_dic selection.\n",
      "finish in 0.142s, 19 probes kept.\n",
      "-- check region:6 7:44029999-44034999_strand_+_gene_Chr7-6Mb_reg_7, 9378 candidate probes\n",
      "--- 802 probes passed check_dic selection.\n",
      "finish in 0.179s, 30 probes kept.\n",
      "-- check region:7 7:44034999-44039999_strand_+_gene_Chr7-6Mb_reg_8, 8912 candidate probes\n",
      "--- 86 probes passed check_dic selection.\n",
      "finish in 0.147s, 6 probes kept.\n",
      "-- check region:8 7:44039999-44044999_strand_+_gene_Chr7-6Mb_reg_9, 9638 candidate probes\n",
      "--- 4428 probes passed check_dic selection.\n",
      "finish in 0.306s, 60 probes kept.\n",
      "-- check region:9 7:44044999-44049999_strand_+_gene_Chr7-6Mb_reg_10, 8056 candidate probes\n",
      "--- 2074 probes passed check_dic selection.\n",
      "finish in 0.203s, 41 probes kept.\n",
      "-- check region:10 7:44049999-44054999_strand_+_gene_Chr7-6Mb_reg_11, 8148 candidate probes\n",
      "--- 2380 probes passed check_dic selection.\n",
      "finish in 0.213s, 44 probes kept.\n",
      "-- check region:11 7:44054999-44059999_strand_+_gene_Chr7-6Mb_reg_12, 7668 candidate probes\n",
      "--- 1524 probes passed check_dic selection.\n",
      "finish in 0.176s, 40 probes kept.\n",
      "-- check region:12 7:44059999-44064999_strand_+_gene_Chr7-6Mb_reg_13, 9532 candidate probes\n",
      "--- 674 probes passed check_dic selection.\n",
      "finish in 0.182s, 20 probes kept.\n",
      "-- check region:13 7:44064999-44069999_strand_+_gene_Chr7-6Mb_reg_14, 8228 candidate probes\n",
      "--- 936 probes passed check_dic selection.\n",
      "finish in 0.168s, 26 probes kept.\n",
      "-- check region:14 7:44069999-44074999_strand_+_gene_Chr7-6Mb_reg_15, 9358 candidate probes\n",
      "--- 4554 probes passed check_dic selection.\n",
      "finish in 0.307s, 66 probes kept.\n",
      "-- check region:15 7:44074999-44079999_strand_+_gene_Chr7-6Mb_reg_16, 9060 candidate probes\n",
      "--- 4050 probes passed check_dic selection.\n",
      "finish in 0.287s, 58 probes kept.\n",
      "-- check region:16 7:44079999-44084999_strand_+_gene_Chr7-6Mb_reg_17, 9846 candidate probes\n",
      "--- 496 probes passed check_dic selection.\n",
      "finish in 0.170s, 14 probes kept.\n",
      "-- check region:17 7:44084999-44089999_strand_+_gene_Chr7-6Mb_reg_18, 9482 candidate probes\n",
      "--- 338 probes passed check_dic selection.\n",
      "finish in 0.167s, 13 probes kept.\n",
      "-- check region:18 7:44089999-44094999_strand_+_gene_Chr7-6Mb_reg_19, 9460 candidate probes\n",
      "--- 1672 probes passed check_dic selection.\n",
      "finish in 0.214s, 40 probes kept.\n",
      "-- check region:19 7:44094999-44099999_strand_+_gene_Chr7-6Mb_reg_20, 8740 candidate probes\n",
      "--- 2886 probes passed check_dic selection.\n",
      "finish in 0.241s, 49 probes kept.\n",
      "-- check region:20 7:44099999-44104999_strand_+_gene_Chr7-6Mb_reg_21, 8008 candidate probes\n",
      "--- 1942 probes passed check_dic selection.\n",
      "finish in 0.196s, 40 probes kept.\n",
      "-- check region:21 7:44104999-44109999_strand_+_gene_Chr7-6Mb_reg_22, 8570 candidate probes\n",
      "--- 3044 probes passed check_dic selection.\n",
      "finish in 0.244s, 51 probes kept.\n",
      "-- check region:22 7:44109999-44114999_strand_+_gene_Chr7-6Mb_reg_23, 9126 candidate probes\n",
      "--- 2224 probes passed check_dic selection.\n",
      "finish in 0.225s, 42 probes kept.\n",
      "-- check region:23 7:44114999-44119999_strand_+_gene_Chr7-6Mb_reg_24, 8608 candidate probes\n",
      "--- 3004 probes passed check_dic selection.\n",
      "finish in 0.243s, 51 probes kept.\n",
      "-- check region:24 7:44119999-44124999_strand_+_gene_Chr7-6Mb_reg_25, 9064 candidate probes\n",
      "--- 2896 probes passed check_dic selection.\n",
      "finish in 0.245s, 46 probes kept.\n",
      "-- check region:25 7:44124999-44129999_strand_+_gene_Chr7-6Mb_reg_26, 9692 candidate probes\n",
      "--- 52 probes passed check_dic selection.\n",
      "finish in 0.155s, 3 probes kept.\n",
      "-- check region:26 7:44129999-44134999_strand_+_gene_Chr7-6Mb_reg_27, 9540 candidate probes\n",
      "--- 2490 probes passed check_dic selection.\n",
      "finish in 0.240s, 47 probes kept.\n",
      "-- check region:27 7:44134999-44139999_strand_+_gene_Chr7-6Mb_reg_28, 9496 candidate probes\n",
      "--- 3238 probes passed check_dic selection.\n",
      "finish in 0.265s, 60 probes kept.\n",
      "-- check region:28 7:44139999-44144999_strand_+_gene_Chr7-6Mb_reg_29, 7554 candidate probes\n",
      "--- 2250 probes passed check_dic selection.\n",
      "finish in 0.199s, 40 probes kept.\n",
      "-- check region:29 7:44144999-44149999_strand_+_gene_Chr7-6Mb_reg_30, 2550 candidate probes\n",
      "--- 46 probes passed check_dic selection.\n",
      "finish in 0.043s, 1 probes kept.\n",
      "-- check region:30 7:44149999-44154999_strand_+_gene_Chr7-6Mb_reg_31, 9342 candidate probes\n",
      "--- 734 probes passed check_dic selection.\n",
      "finish in 0.173s, 20 probes kept.\n",
      "-- check region:31 7:44154999-44159999_strand_+_gene_Chr7-6Mb_reg_32, 3908 candidate probes\n",
      "--- 34 probes passed check_dic selection.\n",
      "finish in 0.067s, 2 probes kept.\n",
      "-- check region:32 7:44159999-44164999_strand_+_gene_Chr7-6Mb_reg_33, 6178 candidate probes\n",
      "--- 72 probes passed check_dic selection.\n",
      "finish in 0.103s, 2 probes kept.\n",
      "-- check region:33 7:44164999-44169999_strand_+_gene_Chr7-6Mb_reg_34, 9898 candidate probes\n",
      "--- 482 probes passed check_dic selection.\n",
      "finish in 0.177s, 15 probes kept.\n",
      "-- check region:34 7:44169999-44174999_strand_+_gene_Chr7-6Mb_reg_35, 9236 candidate probes\n",
      "--- 242 probes passed check_dic selection.\n",
      "finish in 0.157s, 6 probes kept.\n",
      "-- check region:35 7:44174999-44179999_strand_+_gene_Chr7-6Mb_reg_36, 9610 candidate probes\n",
      "--- 1316 probes passed check_dic selection.\n",
      "finish in 0.203s, 31 probes kept.\n",
      "-- check region:36 7:44179999-44184999_strand_+_gene_Chr7-6Mb_reg_37, 9734 candidate probes\n",
      "--- 230 probes passed check_dic selection.\n",
      "finish in 0.166s, 11 probes kept.\n",
      "-- check region:37 7:44184999-44189999_strand_+_gene_Chr7-6Mb_reg_38, 9262 candidate probes\n",
      "--- 1708 probes passed check_dic selection.\n",
      "finish in 0.211s, 39 probes kept.\n",
      "-- check region:38 7:44189999-44194999_strand_+_gene_Chr7-6Mb_reg_39, 9282 candidate probes\n",
      "--- 2148 probes passed check_dic selection.\n",
      "finish in 0.227s, 39 probes kept.\n",
      "-- check region:39 7:44194999-44199999_strand_+_gene_Chr7-6Mb_reg_40, 9784 candidate probes\n",
      "--- 3026 probes passed check_dic selection.\n",
      "finish in 0.262s, 51 probes kept.\n",
      "-- check region:40 7:44199999-44204999_strand_+_gene_Chr7-6Mb_reg_41, 9310 candidate probes\n",
      "--- 3904 probes passed check_dic selection.\n",
      "finish in 0.288s, 60 probes kept.\n",
      "-- check region:41 7:44204999-44209999_strand_+_gene_Chr7-6Mb_reg_42, 9422 candidate probes\n",
      "--- 3230 probes passed check_dic selection.\n",
      "finish in 0.267s, 56 probes kept.\n",
      "-- check region:42 7:44209999-44214999_strand_+_gene_Chr7-6Mb_reg_43, 9250 candidate probes\n",
      "--- 3862 probes passed check_dic selection.\n",
      "finish in 0.283s, 64 probes kept.\n",
      "-- check region:43 7:44214999-44219999_strand_+_gene_Chr7-6Mb_reg_44, 9200 candidate probes\n",
      "--- 3090 probes passed check_dic selection.\n",
      "finish in 0.258s, 49 probes kept.\n",
      "-- check region:44 7:44219999-44224999_strand_+_gene_Chr7-6Mb_reg_45, 9014 candidate probes\n",
      "--- 3312 probes passed check_dic selection.\n",
      "finish in 0.261s, 50 probes kept.\n",
      "-- check region:45 7:44224999-44229999_strand_+_gene_Chr7-6Mb_reg_46, 9918 candidate probes\n",
      "--- 3986 probes passed check_dic selection.\n",
      "finish in 0.300s, 60 probes kept.\n",
      "-- check region:46 7:44229999-44234999_strand_+_gene_Chr7-6Mb_reg_47, 9876 candidate probes\n",
      "--- 3342 probes passed check_dic selection.\n",
      "finish in 0.275s, 58 probes kept.\n",
      "-- check region:47 7:44234999-44239999_strand_+_gene_Chr7-6Mb_reg_48, 9754 candidate probes\n",
      "--- 5728 probes passed check_dic selection.\n",
      "finish in 0.354s, 71 probes kept.\n",
      "-- check region:48 7:44239999-44244999_strand_+_gene_Chr7-6Mb_reg_49, 9896 candidate probes\n",
      "--- 4536 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.316s, 67 probes kept.\n",
      "-- check region:49 7:44244999-44249999_strand_+_gene_Chr7-6Mb_reg_50, 9910 candidate probes\n",
      "--- 7232 probes passed check_dic selection.\n",
      "finish in 0.407s, 84 probes kept.\n",
      "-- check region:50 7:44249999-44254999_strand_+_gene_Chr7-6Mb_reg_51, 9918 candidate probes\n",
      "--- 7132 probes passed check_dic selection.\n",
      "finish in 0.412s, 81 probes kept.\n",
      "-- check region:51 7:44254999-44259999_strand_+_gene_Chr7-6Mb_reg_52, 9918 candidate probes\n",
      "--- 6728 probes passed check_dic selection.\n",
      "finish in 0.392s, 79 probes kept.\n",
      "-- check region:52 7:44259999-44264999_strand_+_gene_Chr7-6Mb_reg_53, 9742 candidate probes\n",
      "--- 6234 probes passed check_dic selection.\n",
      "finish in 0.372s, 74 probes kept.\n",
      "-- check region:53 7:44264999-44269999_strand_+_gene_Chr7-6Mb_reg_54, 9600 candidate probes\n",
      "--- 6036 probes passed check_dic selection.\n",
      "finish in 0.363s, 70 probes kept.\n",
      "-- check region:54 7:44269999-44274999_strand_+_gene_Chr7-6Mb_reg_55, 9908 candidate probes\n",
      "--- 5836 probes passed check_dic selection.\n",
      "finish in 0.359s, 70 probes kept.\n",
      "-- check region:55 7:44274999-44279999_strand_+_gene_Chr7-6Mb_reg_56, 9724 candidate probes\n",
      "--- 5110 probes passed check_dic selection.\n",
      "finish in 0.334s, 68 probes kept.\n",
      "-- check region:56 7:44279999-44284999_strand_+_gene_Chr7-6Mb_reg_57, 9884 candidate probes\n",
      "--- 5532 probes passed check_dic selection.\n",
      "finish in 0.348s, 68 probes kept.\n",
      "-- check region:57 7:44284999-44289999_strand_+_gene_Chr7-6Mb_reg_58, 9512 candidate probes\n",
      "--- 4464 probes passed check_dic selection.\n",
      "finish in 0.304s, 62 probes kept.\n",
      "-- check region:58 7:44289999-44294999_strand_+_gene_Chr7-6Mb_reg_59, 9538 candidate probes\n",
      "--- 3780 probes passed check_dic selection.\n",
      "finish in 0.285s, 58 probes kept.\n",
      "-- check region:59 7:44294999-44299999_strand_+_gene_Chr7-6Mb_reg_60, 9844 candidate probes\n",
      "--- 2872 probes passed check_dic selection.\n",
      "finish in 0.255s, 48 probes kept.\n",
      "-- check region:60 7:44299999-44304999_strand_+_gene_Chr7-6Mb_reg_61, 9908 candidate probes\n",
      "--- 6578 probes passed check_dic selection.\n",
      "finish in 0.384s, 80 probes kept.\n",
      "-- check region:61 7:44304999-44309999_strand_+_gene_Chr7-6Mb_reg_62, 9918 candidate probes\n",
      "--- 8432 probes passed check_dic selection.\n",
      "finish in 0.451s, 88 probes kept.\n",
      "-- check region:62 7:44309999-44314999_strand_+_gene_Chr7-6Mb_reg_63, 9896 candidate probes\n",
      "--- 8268 probes passed check_dic selection.\n",
      "finish in 0.441s, 80 probes kept.\n",
      "-- check region:63 7:44314999-44319999_strand_+_gene_Chr7-6Mb_reg_64, 9898 candidate probes\n",
      "--- 5860 probes passed check_dic selection.\n",
      "finish in 0.362s, 75 probes kept.\n",
      "-- check region:64 7:44319999-44324999_strand_+_gene_Chr7-6Mb_reg_65, 9890 candidate probes\n",
      "--- 6980 probes passed check_dic selection.\n",
      "finish in 0.400s, 83 probes kept.\n",
      "-- check region:65 7:44324999-44329999_strand_+_gene_Chr7-6Mb_reg_66, 9738 candidate probes\n",
      "--- 6102 probes passed check_dic selection.\n",
      "finish in 0.369s, 70 probes kept.\n",
      "-- check region:66 7:44329999-44334999_strand_+_gene_Chr7-6Mb_reg_67, 9706 candidate probes\n",
      "--- 5412 probes passed check_dic selection.\n",
      "finish in 0.344s, 72 probes kept.\n",
      "-- check region:67 7:44334999-44339999_strand_+_gene_Chr7-6Mb_reg_68, 9918 candidate probes\n",
      "--- 6850 probes passed check_dic selection.\n",
      "finish in 0.396s, 84 probes kept.\n",
      "-- check region:68 7:44339999-44344999_strand_+_gene_Chr7-6Mb_reg_69, 9918 candidate probes\n",
      "--- 7480 probes passed check_dic selection.\n",
      "finish in 0.417s, 84 probes kept.\n",
      "-- check region:69 7:44344999-44349999_strand_+_gene_Chr7-6Mb_reg_70, 9800 candidate probes\n",
      "--- 5128 probes passed check_dic selection.\n",
      "finish in 0.334s, 68 probes kept.\n",
      "-- check region:70 7:44349999-44354999_strand_+_gene_Chr7-6Mb_reg_71, 9918 candidate probes\n",
      "--- 8144 probes passed check_dic selection.\n",
      "finish in 0.441s, 88 probes kept.\n",
      "-- check region:71 7:44354999-44359999_strand_+_gene_Chr7-6Mb_reg_72, 9666 candidate probes\n",
      "--- 6262 probes passed check_dic selection.\n",
      "finish in 0.370s, 74 probes kept.\n",
      "-- check region:72 7:44359999-44364999_strand_+_gene_Chr7-6Mb_reg_73, 9556 candidate probes\n",
      "--- 6502 probes passed check_dic selection.\n",
      "finish in 0.379s, 69 probes kept.\n",
      "-- check region:73 7:44364999-44369999_strand_+_gene_Chr7-6Mb_reg_74, 9662 candidate probes\n",
      "--- 4032 probes passed check_dic selection.\n",
      "finish in 0.294s, 64 probes kept.\n",
      "-- check region:74 7:44369999-44374999_strand_+_gene_Chr7-6Mb_reg_75, 9776 candidate probes\n",
      "--- 6080 probes passed check_dic selection.\n",
      "finish in 0.367s, 76 probes kept.\n",
      "-- check region:75 7:44374999-44379999_strand_+_gene_Chr7-6Mb_reg_76, 9772 candidate probes\n",
      "--- 5028 probes passed check_dic selection.\n",
      "finish in 0.331s, 67 probes kept.\n",
      "-- check region:76 7:44379999-44384999_strand_+_gene_Chr7-6Mb_reg_77, 9672 candidate probes\n",
      "--- 6314 probes passed check_dic selection.\n",
      "finish in 0.372s, 81 probes kept.\n",
      "-- check region:77 7:44384999-44389999_strand_+_gene_Chr7-6Mb_reg_78, 9740 candidate probes\n",
      "--- 5410 probes passed check_dic selection.\n",
      "finish in 0.342s, 69 probes kept.\n",
      "-- check region:78 7:44389999-44394999_strand_+_gene_Chr7-6Mb_reg_79, 9918 candidate probes\n",
      "--- 7674 probes passed check_dic selection.\n",
      "finish in 0.424s, 82 probes kept.\n",
      "-- check region:79 7:44394999-44399999_strand_+_gene_Chr7-6Mb_reg_80, 9872 candidate probes\n",
      "--- 6440 probes passed check_dic selection.\n",
      "finish in 0.381s, 77 probes kept.\n",
      "-- check region:80 7:44399999-44404999_strand_+_gene_Chr7-6Mb_reg_81, 9590 candidate probes\n",
      "--- 5126 probes passed check_dic selection.\n",
      "finish in 0.330s, 66 probes kept.\n",
      "-- check region:81 7:44404999-44409999_strand_+_gene_Chr7-6Mb_reg_82, 9866 candidate probes\n",
      "--- 6542 probes passed check_dic selection.\n",
      "finish in 0.387s, 77 probes kept.\n",
      "-- check region:82 7:44409999-44414999_strand_+_gene_Chr7-6Mb_reg_83, 9442 candidate probes\n",
      "--- 4744 probes passed check_dic selection.\n",
      "finish in 0.316s, 64 probes kept.\n",
      "-- check region:83 7:44414999-44419999_strand_+_gene_Chr7-6Mb_reg_84, 9400 candidate probes\n",
      "--- 5670 probes passed check_dic selection.\n",
      "finish in 0.346s, 63 probes kept.\n",
      "-- check region:84 7:44419999-44424999_strand_+_gene_Chr7-6Mb_reg_85, 9858 candidate probes\n",
      "--- 5978 probes passed check_dic selection.\n",
      "finish in 0.364s, 74 probes kept.\n",
      "-- check region:85 7:44424999-44429999_strand_+_gene_Chr7-6Mb_reg_86, 9862 candidate probes\n",
      "--- 6364 probes passed check_dic selection.\n",
      "finish in 0.373s, 73 probes kept.\n",
      "-- check region:86 7:44429999-44434999_strand_+_gene_Chr7-6Mb_reg_87, 9868 candidate probes\n",
      "--- 6700 probes passed check_dic selection.\n",
      "finish in 0.389s, 79 probes kept.\n",
      "-- check region:87 7:44434999-44439999_strand_+_gene_Chr7-6Mb_reg_88, 9610 candidate probes\n",
      "--- 5872 probes passed check_dic selection.\n",
      "finish in 0.356s, 71 probes kept.\n",
      "-- check region:88 7:44439999-44444999_strand_+_gene_Chr7-6Mb_reg_89, 9850 candidate probes\n",
      "--- 6888 probes passed check_dic selection.\n",
      "finish in 0.398s, 80 probes kept.\n",
      "-- check region:89 7:44444999-44449999_strand_+_gene_Chr7-6Mb_reg_90, 9870 candidate probes\n",
      "--- 6560 probes passed check_dic selection.\n",
      "finish in 0.386s, 79 probes kept.\n",
      "-- check region:90 7:44449999-44454999_strand_+_gene_Chr7-6Mb_reg_91, 9334 candidate probes\n",
      "--- 3718 probes passed check_dic selection.\n",
      "finish in 0.280s, 52 probes kept.\n",
      "-- check region:91 7:44454999-44459999_strand_+_gene_Chr7-6Mb_reg_92, 9706 candidate probes\n",
      "--- 4814 probes passed check_dic selection.\n",
      "finish in 0.321s, 69 probes kept.\n",
      "-- check region:92 7:44459999-44464999_strand_+_gene_Chr7-6Mb_reg_93, 9918 candidate probes\n",
      "--- 8250 probes passed check_dic selection.\n",
      "finish in 0.441s, 86 probes kept.\n",
      "-- check region:93 7:44464999-44469999_strand_+_gene_Chr7-6Mb_reg_94, 9918 candidate probes\n",
      "--- 7764 probes passed check_dic selection.\n",
      "finish in 0.426s, 83 probes kept.\n",
      "-- check region:94 7:44469999-44474999_strand_+_gene_Chr7-6Mb_reg_95, 9906 candidate probes\n",
      "--- 6998 probes passed check_dic selection.\n",
      "finish in 0.398s, 83 probes kept.\n",
      "-- check region:95 7:44474999-44479999_strand_+_gene_Chr7-6Mb_reg_96, 9912 candidate probes\n",
      "--- 2562 probes passed check_dic selection.\n",
      "finish in 0.248s, 49 probes kept.\n",
      "-- check region:96 7:44479999-44484999_strand_+_gene_Chr7-6Mb_reg_97, 9018 candidate probes\n",
      "--- 3470 probes passed check_dic selection.\n",
      "finish in 0.266s, 61 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:97 7:44484999-44489999_strand_+_gene_Chr7-6Mb_reg_98, 9804 candidate probes\n",
      "--- 6822 probes passed check_dic selection.\n",
      "finish in 0.395s, 78 probes kept.\n",
      "-- check region:98 7:44489999-44494999_strand_+_gene_Chr7-6Mb_reg_99, 9918 candidate probes\n",
      "--- 7550 probes passed check_dic selection.\n",
      "finish in 0.419s, 85 probes kept.\n",
      "-- check region:99 7:44494999-44499999_strand_+_gene_Chr7-6Mb_reg_100, 9870 candidate probes\n",
      "--- 6658 probes passed check_dic selection.\n",
      "finish in 0.389s, 80 probes kept.\n",
      "-- check region:100 7:44499999-44504999_strand_+_gene_Chr7-6Mb_reg_101, 9860 candidate probes\n",
      "--- 6208 probes passed check_dic selection.\n",
      "finish in 0.373s, 75 probes kept.\n",
      "-- check region:101 7:44504999-44509999_strand_+_gene_Chr7-6Mb_reg_102, 9896 candidate probes\n",
      "--- 7914 probes passed check_dic selection.\n",
      "finish in 0.434s, 82 probes kept.\n",
      "-- check region:102 7:44509999-44514999_strand_+_gene_Chr7-6Mb_reg_103, 9810 candidate probes\n",
      "--- 5604 probes passed check_dic selection.\n",
      "finish in 0.352s, 66 probes kept.\n",
      "-- check region:103 7:44514999-44519999_strand_+_gene_Chr7-6Mb_reg_104, 9824 candidate probes\n",
      "--- 6502 probes passed check_dic selection.\n",
      "finish in 0.383s, 75 probes kept.\n",
      "-- check region:104 7:44519999-44524999_strand_+_gene_Chr7-6Mb_reg_105, 9788 candidate probes\n",
      "--- 7150 probes passed check_dic selection.\n",
      "finish in 0.404s, 75 probes kept.\n",
      "-- check region:105 7:44524999-44529999_strand_+_gene_Chr7-6Mb_reg_106, 9872 candidate probes\n",
      "--- 7554 probes passed check_dic selection.\n",
      "finish in 0.419s, 87 probes kept.\n",
      "-- check region:106 7:44529999-44534999_strand_+_gene_Chr7-6Mb_reg_107, 9910 candidate probes\n",
      "--- 8002 probes passed check_dic selection.\n",
      "finish in 0.442s, 86 probes kept.\n",
      "-- check region:107 7:44534999-44539999_strand_+_gene_Chr7-6Mb_reg_108, 9918 candidate probes\n",
      "--- 8528 probes passed check_dic selection.\n",
      "finish in 0.458s, 87 probes kept.\n",
      "-- check region:108 7:44539999-44544999_strand_+_gene_Chr7-6Mb_reg_109, 9648 candidate probes\n",
      "--- 7870 probes passed check_dic selection.\n",
      "finish in 0.428s, 81 probes kept.\n",
      "-- check region:109 7:44544999-44549999_strand_+_gene_Chr7-6Mb_reg_110, 9470 candidate probes\n",
      "--- 6128 probes passed check_dic selection.\n",
      "finish in 0.365s, 70 probes kept.\n",
      "-- check region:110 7:44549999-44554999_strand_+_gene_Chr7-6Mb_reg_111, 9898 candidate probes\n",
      "--- 7932 probes passed check_dic selection.\n",
      "finish in 0.431s, 82 probes kept.\n",
      "-- check region:111 7:44554999-44559999_strand_+_gene_Chr7-6Mb_reg_112, 9746 candidate probes\n",
      "--- 4052 probes passed check_dic selection.\n",
      "finish in 0.295s, 59 probes kept.\n",
      "-- check region:112 7:44559999-44564999_strand_+_gene_Chr7-6Mb_reg_113, 9604 candidate probes\n",
      "--- 5192 probes passed check_dic selection.\n",
      "finish in 0.333s, 68 probes kept.\n",
      "-- check region:113 7:44564999-44569999_strand_+_gene_Chr7-6Mb_reg_114, 9684 candidate probes\n",
      "--- 3300 probes passed check_dic selection.\n",
      "finish in 0.266s, 59 probes kept.\n",
      "-- check region:114 7:44569999-44574999_strand_+_gene_Chr7-6Mb_reg_115, 9726 candidate probes\n",
      "--- 4028 probes passed check_dic selection.\n",
      "finish in 0.296s, 64 probes kept.\n",
      "-- check region:115 7:44574999-44579999_strand_+_gene_Chr7-6Mb_reg_116, 9304 candidate probes\n",
      "--- 2366 probes passed check_dic selection.\n",
      "finish in 0.231s, 39 probes kept.\n",
      "-- check region:116 7:44579999-44584999_strand_+_gene_Chr7-6Mb_reg_117, 9664 candidate probes\n",
      "--- 5342 probes passed check_dic selection.\n",
      "finish in 0.340s, 61 probes kept.\n",
      "-- check region:117 7:44584999-44589999_strand_+_gene_Chr7-6Mb_reg_118, 9886 candidate probes\n",
      "--- 7208 probes passed check_dic selection.\n",
      "finish in 0.408s, 81 probes kept.\n",
      "-- check region:118 7:44589999-44594999_strand_+_gene_Chr7-6Mb_reg_119, 9918 candidate probes\n",
      "--- 7946 probes passed check_dic selection.\n",
      "finish in 0.429s, 84 probes kept.\n",
      "-- check region:119 7:44594999-44599999_strand_+_gene_Chr7-6Mb_reg_120, 9862 candidate probes\n",
      "--- 6964 probes passed check_dic selection.\n",
      "finish in 0.401s, 82 probes kept.\n",
      "-- check region:120 7:44599999-44604999_strand_+_gene_Chr7-6Mb_reg_121, 9918 candidate probes\n",
      "--- 7536 probes passed check_dic selection.\n",
      "finish in 0.427s, 87 probes kept.\n",
      "-- check region:121 7:44604999-44609999_strand_+_gene_Chr7-6Mb_reg_122, 9918 candidate probes\n",
      "--- 7158 probes passed check_dic selection.\n",
      "finish in 0.407s, 81 probes kept.\n",
      "-- check region:122 7:44609999-44614999_strand_+_gene_Chr7-6Mb_reg_123, 9774 candidate probes\n",
      "--- 6840 probes passed check_dic selection.\n",
      "finish in 0.395s, 76 probes kept.\n",
      "-- check region:123 7:44614999-44619999_strand_+_gene_Chr7-6Mb_reg_124, 9792 candidate probes\n",
      "--- 5116 probes passed check_dic selection.\n",
      "finish in 0.334s, 61 probes kept.\n",
      "-- check region:124 7:44619999-44624999_strand_+_gene_Chr7-6Mb_reg_125, 9124 candidate probes\n",
      "--- 5852 probes passed check_dic selection.\n",
      "finish in 0.348s, 69 probes kept.\n",
      "-- check region:125 7:44624999-44629999_strand_+_gene_Chr7-6Mb_reg_126, 9588 candidate probes\n",
      "--- 5674 probes passed check_dic selection.\n",
      "finish in 0.349s, 66 probes kept.\n",
      "-- check region:126 7:44629999-44634999_strand_+_gene_Chr7-6Mb_reg_127, 9870 candidate probes\n",
      "--- 7332 probes passed check_dic selection.\n",
      "finish in 0.411s, 83 probes kept.\n",
      "-- check region:127 7:44634999-44639999_strand_+_gene_Chr7-6Mb_reg_128, 9766 candidate probes\n",
      "--- 6640 probes passed check_dic selection.\n",
      "finish in 0.383s, 72 probes kept.\n",
      "-- check region:128 7:44639999-44644999_strand_+_gene_Chr7-6Mb_reg_129, 9872 candidate probes\n",
      "--- 5010 probes passed check_dic selection.\n",
      "finish in 0.329s, 66 probes kept.\n",
      "-- check region:129 7:44644999-44649999_strand_+_gene_Chr7-6Mb_reg_130, 9842 candidate probes\n",
      "--- 5626 probes passed check_dic selection.\n",
      "finish in 0.350s, 69 probes kept.\n",
      "-- check region:130 7:44649999-44654999_strand_+_gene_Chr7-6Mb_reg_131, 9684 candidate probes\n",
      "--- 5276 probes passed check_dic selection.\n",
      "finish in 0.340s, 69 probes kept.\n",
      "-- check region:131 7:44654999-44659999_strand_+_gene_Chr7-6Mb_reg_132, 9892 candidate probes\n",
      "--- 7534 probes passed check_dic selection.\n",
      "finish in 0.425s, 79 probes kept.\n",
      "-- check region:132 7:44659999-44664999_strand_+_gene_Chr7-6Mb_reg_133, 9918 candidate probes\n",
      "--- 6486 probes passed check_dic selection.\n",
      "finish in 0.384s, 76 probes kept.\n",
      "-- check region:133 7:44664999-44669999_strand_+_gene_Chr7-6Mb_reg_134, 9860 candidate probes\n",
      "--- 6172 probes passed check_dic selection.\n",
      "finish in 0.373s, 71 probes kept.\n",
      "-- check region:134 7:44669999-44674999_strand_+_gene_Chr7-6Mb_reg_135, 9918 candidate probes\n",
      "--- 5376 probes passed check_dic selection.\n",
      "finish in 0.342s, 75 probes kept.\n",
      "-- check region:135 7:44674999-44679999_strand_+_gene_Chr7-6Mb_reg_136, 9702 candidate probes\n",
      "--- 5168 probes passed check_dic selection.\n",
      "finish in 0.332s, 64 probes kept.\n",
      "-- check region:136 7:44679999-44684999_strand_+_gene_Chr7-6Mb_reg_137, 9820 candidate probes\n",
      "--- 5478 probes passed check_dic selection.\n",
      "finish in 0.346s, 66 probes kept.\n",
      "-- check region:137 7:44684999-44689999_strand_+_gene_Chr7-6Mb_reg_138, 9764 candidate probes\n",
      "--- 5470 probes passed check_dic selection.\n",
      "finish in 0.347s, 67 probes kept.\n",
      "-- check region:138 7:44689999-44694999_strand_+_gene_Chr7-6Mb_reg_139, 9918 candidate probes\n",
      "--- 5206 probes passed check_dic selection.\n",
      "finish in 0.341s, 72 probes kept.\n",
      "-- check region:139 7:44694999-44699999_strand_+_gene_Chr7-6Mb_reg_140, 9390 candidate probes\n",
      "--- 4614 probes passed check_dic selection.\n",
      "finish in 0.311s, 64 probes kept.\n",
      "-- check region:140 7:44699999-44704999_strand_+_gene_Chr7-6Mb_reg_141, 9610 candidate probes\n",
      "--- 4360 probes passed check_dic selection.\n",
      "finish in 0.305s, 66 probes kept.\n",
      "-- check region:141 7:44704999-44709999_strand_+_gene_Chr7-6Mb_reg_142, 9634 candidate probes\n",
      "--- 3664 probes passed check_dic selection.\n",
      "finish in 0.281s, 57 probes kept.\n",
      "-- check region:142 7:44709999-44714999_strand_+_gene_Chr7-6Mb_reg_143, 9860 candidate probes\n",
      "--- 4486 probes passed check_dic selection.\n",
      "finish in 0.314s, 65 probes kept.\n",
      "-- check region:143 7:44714999-44719999_strand_+_gene_Chr7-6Mb_reg_144, 9760 candidate probes\n",
      "--- 4946 probes passed check_dic selection.\n",
      "finish in 0.327s, 69 probes kept.\n",
      "-- check region:144 7:44719999-44724999_strand_+_gene_Chr7-6Mb_reg_145, 9546 candidate probes\n",
      "--- 5840 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.356s, 69 probes kept.\n",
      "-- check region:145 7:44724999-44729999_strand_+_gene_Chr7-6Mb_reg_146, 9868 candidate probes\n",
      "--- 4670 probes passed check_dic selection.\n",
      "finish in 0.319s, 61 probes kept.\n",
      "-- check region:146 7:44729999-44734999_strand_+_gene_Chr7-6Mb_reg_147, 9664 candidate probes\n",
      "--- 6834 probes passed check_dic selection.\n",
      "finish in 0.389s, 73 probes kept.\n",
      "-- check region:147 7:44734999-44739999_strand_+_gene_Chr7-6Mb_reg_148, 9318 candidate probes\n",
      "--- 5822 probes passed check_dic selection.\n",
      "finish in 0.350s, 72 probes kept.\n",
      "-- check region:148 7:44739999-44744999_strand_+_gene_Chr7-6Mb_reg_149, 9726 candidate probes\n",
      "--- 3982 probes passed check_dic selection.\n",
      "finish in 0.292s, 59 probes kept.\n",
      "-- check region:149 7:44744999-44749999_strand_+_gene_Chr7-6Mb_reg_150, 9904 candidate probes\n",
      "--- 5094 probes passed check_dic selection.\n",
      "finish in 0.330s, 63 probes kept.\n",
      "-- check region:150 7:44749999-44754999_strand_+_gene_Chr7-6Mb_reg_151, 9880 candidate probes\n",
      "--- 4416 probes passed check_dic selection.\n",
      "finish in 0.313s, 53 probes kept.\n",
      "-- check region:151 7:44754999-44759999_strand_+_gene_Chr7-6Mb_reg_152, 9832 candidate probes\n",
      "--- 4622 probes passed check_dic selection.\n",
      "finish in 0.319s, 61 probes kept.\n",
      "-- check region:152 7:44759999-44764999_strand_+_gene_Chr7-6Mb_reg_153, 9904 candidate probes\n",
      "--- 6296 probes passed check_dic selection.\n",
      "finish in 0.376s, 70 probes kept.\n",
      "-- check region:153 7:44764999-44769999_strand_+_gene_Chr7-6Mb_reg_154, 9918 candidate probes\n",
      "--- 7216 probes passed check_dic selection.\n",
      "finish in 0.407s, 84 probes kept.\n",
      "-- check region:154 7:44769999-44774999_strand_+_gene_Chr7-6Mb_reg_155, 9850 candidate probes\n",
      "--- 5078 probes passed check_dic selection.\n",
      "finish in 0.331s, 67 probes kept.\n",
      "-- check region:155 7:44774999-44779999_strand_+_gene_Chr7-6Mb_reg_156, 9404 candidate probes\n",
      "--- 5056 probes passed check_dic selection.\n",
      "finish in 0.326s, 64 probes kept.\n",
      "-- check region:156 7:44779999-44784999_strand_+_gene_Chr7-6Mb_reg_157, 9716 candidate probes\n",
      "--- 6556 probes passed check_dic selection.\n",
      "finish in 0.386s, 72 probes kept.\n",
      "-- check region:157 7:44784999-44789999_strand_+_gene_Chr7-6Mb_reg_158, 9840 candidate probes\n",
      "--- 5834 probes passed check_dic selection.\n",
      "finish in 0.360s, 72 probes kept.\n",
      "-- check region:158 7:44789999-44794999_strand_+_gene_Chr7-6Mb_reg_159, 9632 candidate probes\n",
      "--- 4306 probes passed check_dic selection.\n",
      "finish in 0.306s, 62 probes kept.\n",
      "-- check region:159 7:44794999-44799999_strand_+_gene_Chr7-6Mb_reg_160, 8574 candidate probes\n",
      "--- 294 probes passed check_dic selection.\n",
      "finish in 0.146s, 11 probes kept.\n",
      "-- check region:160 7:44799999-44804999_strand_+_gene_Chr7-6Mb_reg_161, 9908 candidate probes\n",
      "--- 1964 probes passed check_dic selection.\n",
      "finish in 0.225s, 29 probes kept.\n",
      "-- check region:161 7:44804999-44809999_strand_+_gene_Chr7-6Mb_reg_162, 9828 candidate probes\n",
      "--- 3510 probes passed check_dic selection.\n",
      "finish in 0.278s, 61 probes kept.\n",
      "-- check region:162 7:44809999-44814999_strand_+_gene_Chr7-6Mb_reg_163, 9918 candidate probes\n",
      "--- 6294 probes passed check_dic selection.\n",
      "finish in 0.378s, 75 probes kept.\n",
      "-- check region:163 7:44814999-44819999_strand_+_gene_Chr7-6Mb_reg_164, 9918 candidate probes\n",
      "--- 7742 probes passed check_dic selection.\n",
      "finish in 0.425s, 82 probes kept.\n",
      "-- check region:164 7:44819999-44824999_strand_+_gene_Chr7-6Mb_reg_165, 9666 candidate probes\n",
      "--- 6486 probes passed check_dic selection.\n",
      "finish in 0.375s, 71 probes kept.\n",
      "-- check region:165 7:44824999-44829999_strand_+_gene_Chr7-6Mb_reg_166, 9918 candidate probes\n",
      "--- 8212 probes passed check_dic selection.\n",
      "finish in 0.440s, 86 probes kept.\n",
      "-- check region:166 7:44829999-44834999_strand_+_gene_Chr7-6Mb_reg_167, 9552 candidate probes\n",
      "--- 6954 probes passed check_dic selection.\n",
      "finish in 0.392s, 78 probes kept.\n",
      "-- check region:167 7:44834999-44839999_strand_+_gene_Chr7-6Mb_reg_168, 9918 candidate probes\n",
      "--- 8552 probes passed check_dic selection.\n",
      "finish in 0.454s, 87 probes kept.\n",
      "-- check region:168 7:44839999-44844999_strand_+_gene_Chr7-6Mb_reg_169, 9870 candidate probes\n",
      "--- 8768 probes passed check_dic selection.\n",
      "finish in 0.467s, 87 probes kept.\n",
      "-- check region:169 7:44844999-44849999_strand_+_gene_Chr7-6Mb_reg_170, 9908 candidate probes\n",
      "--- 8112 probes passed check_dic selection.\n",
      "finish in 0.439s, 81 probes kept.\n",
      "-- check region:170 7:44849999-44854999_strand_+_gene_Chr7-6Mb_reg_171, 9918 candidate probes\n",
      "--- 8832 probes passed check_dic selection.\n",
      "finish in 0.463s, 84 probes kept.\n",
      "-- check region:171 7:44854999-44859999_strand_+_gene_Chr7-6Mb_reg_172, 9790 candidate probes\n",
      "--- 6604 probes passed check_dic selection.\n",
      "finish in 0.391s, 78 probes kept.\n",
      "-- check region:172 7:44859999-44864999_strand_+_gene_Chr7-6Mb_reg_173, 9848 candidate probes\n",
      "--- 8686 probes passed check_dic selection.\n",
      "finish in 0.464s, 90 probes kept.\n",
      "-- check region:173 7:44864999-44869999_strand_+_gene_Chr7-6Mb_reg_174, 9858 candidate probes\n",
      "--- 8592 probes passed check_dic selection.\n",
      "finish in 0.452s, 88 probes kept.\n",
      "-- check region:174 7:44869999-44874999_strand_+_gene_Chr7-6Mb_reg_175, 9750 candidate probes\n",
      "--- 4416 probes passed check_dic selection.\n",
      "finish in 0.308s, 59 probes kept.\n",
      "-- check region:175 7:44874999-44879999_strand_+_gene_Chr7-6Mb_reg_176, 9842 candidate probes\n",
      "--- 7370 probes passed check_dic selection.\n",
      "finish in 0.413s, 79 probes kept.\n",
      "-- check region:176 7:44879999-44884999_strand_+_gene_Chr7-6Mb_reg_177, 9544 candidate probes\n",
      "--- 7616 probes passed check_dic selection.\n",
      "finish in 0.418s, 79 probes kept.\n",
      "-- check region:177 7:44884999-44889999_strand_+_gene_Chr7-6Mb_reg_178, 9898 candidate probes\n",
      "--- 7282 probes passed check_dic selection.\n",
      "finish in 0.412s, 80 probes kept.\n",
      "-- check region:178 7:44889999-44894999_strand_+_gene_Chr7-6Mb_reg_179, 9552 candidate probes\n",
      "--- 5152 probes passed check_dic selection.\n",
      "finish in 0.330s, 67 probes kept.\n",
      "-- check region:179 7:44894999-44899999_strand_+_gene_Chr7-6Mb_reg_180, 9918 candidate probes\n",
      "--- 7952 probes passed check_dic selection.\n",
      "finish in 0.436s, 85 probes kept.\n",
      "-- check region:180 7:44899999-44904999_strand_+_gene_Chr7-6Mb_reg_181, 9918 candidate probes\n",
      "--- 8400 probes passed check_dic selection.\n",
      "finish in 0.454s, 86 probes kept.\n",
      "-- check region:181 7:44904999-44909999_strand_+_gene_Chr7-6Mb_reg_182, 9918 candidate probes\n",
      "--- 8102 probes passed check_dic selection.\n",
      "finish in 0.439s, 85 probes kept.\n",
      "-- check region:182 7:44909999-44914999_strand_+_gene_Chr7-6Mb_reg_183, 9776 candidate probes\n",
      "--- 5436 probes passed check_dic selection.\n",
      "finish in 0.342s, 70 probes kept.\n",
      "-- check region:183 7:44914999-44919999_strand_+_gene_Chr7-6Mb_reg_184, 9622 candidate probes\n",
      "--- 6354 probes passed check_dic selection.\n",
      "finish in 0.372s, 75 probes kept.\n",
      "-- check region:184 7:44919999-44924999_strand_+_gene_Chr7-6Mb_reg_185, 9522 candidate probes\n",
      "--- 3852 probes passed check_dic selection.\n",
      "finish in 0.285s, 57 probes kept.\n",
      "-- check region:185 7:44924999-44929999_strand_+_gene_Chr7-6Mb_reg_186, 9648 candidate probes\n",
      "--- 6352 probes passed check_dic selection.\n",
      "finish in 0.375s, 72 probes kept.\n",
      "-- check region:186 7:44929999-44934999_strand_+_gene_Chr7-6Mb_reg_187, 9686 candidate probes\n",
      "--- 3762 probes passed check_dic selection.\n",
      "finish in 0.284s, 56 probes kept.\n",
      "-- check region:187 7:44934999-44939999_strand_+_gene_Chr7-6Mb_reg_188, 9624 candidate probes\n",
      "--- 3036 probes passed check_dic selection.\n",
      "finish in 0.257s, 54 probes kept.\n",
      "-- check region:188 7:44939999-44944999_strand_+_gene_Chr7-6Mb_reg_189, 9744 candidate probes\n",
      "--- 4002 probes passed check_dic selection.\n",
      "finish in 0.294s, 57 probes kept.\n",
      "-- check region:189 7:44944999-44949999_strand_+_gene_Chr7-6Mb_reg_190, 9234 candidate probes\n",
      "--- 4094 probes passed check_dic selection.\n",
      "finish in 0.291s, 54 probes kept.\n",
      "-- check region:190 7:44949999-44954999_strand_+_gene_Chr7-6Mb_reg_191, 9698 candidate probes\n",
      "--- 6580 probes passed check_dic selection.\n",
      "finish in 0.383s, 69 probes kept.\n",
      "-- check region:191 7:44954999-44959999_strand_+_gene_Chr7-6Mb_reg_192, 9674 candidate probes\n",
      "--- 5908 probes passed check_dic selection.\n",
      "finish in 0.361s, 75 probes kept.\n",
      "-- check region:192 7:44959999-44964999_strand_+_gene_Chr7-6Mb_reg_193, 9436 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 6292 probes passed check_dic selection.\n",
      "finish in 0.370s, 71 probes kept.\n",
      "-- check region:193 7:44964999-44969999_strand_+_gene_Chr7-6Mb_reg_194, 9196 candidate probes\n",
      "--- 4796 probes passed check_dic selection.\n",
      "finish in 0.314s, 60 probes kept.\n",
      "-- check region:194 7:44969999-44974999_strand_+_gene_Chr7-6Mb_reg_195, 9902 candidate probes\n",
      "--- 7272 probes passed check_dic selection.\n",
      "finish in 0.409s, 75 probes kept.\n",
      "-- check region:195 7:44974999-44979999_strand_+_gene_Chr7-6Mb_reg_196, 9818 candidate probes\n",
      "--- 7308 probes passed check_dic selection.\n",
      "finish in 0.411s, 73 probes kept.\n",
      "-- check region:196 7:44979999-44984999_strand_+_gene_Chr7-6Mb_reg_197, 9540 candidate probes\n",
      "--- 6292 probes passed check_dic selection.\n",
      "finish in 0.369s, 74 probes kept.\n",
      "-- check region:197 7:44984999-44989999_strand_+_gene_Chr7-6Mb_reg_198, 9698 candidate probes\n",
      "--- 6068 probes passed check_dic selection.\n",
      "finish in 0.362s, 67 probes kept.\n",
      "-- check region:198 7:44989999-44994999_strand_+_gene_Chr7-6Mb_reg_199, 9892 candidate probes\n",
      "--- 5954 probes passed check_dic selection.\n",
      "finish in 0.366s, 76 probes kept.\n",
      "-- check region:199 7:44994999-44999999_strand_+_gene_Chr7-6Mb_reg_200, 9668 candidate probes\n",
      "--- 7222 probes passed check_dic selection.\n",
      "finish in 0.405s, 81 probes kept.\n",
      "-- check region:200 7:44999999-45004999_strand_+_gene_Chr7-6Mb_reg_201, 9844 candidate probes\n",
      "--- 8184 probes passed check_dic selection.\n",
      "finish in 0.443s, 87 probes kept.\n",
      "-- check region:201 7:45004999-45009999_strand_+_gene_Chr7-6Mb_reg_202, 9804 candidate probes\n",
      "--- 7824 probes passed check_dic selection.\n",
      "finish in 0.428s, 83 probes kept.\n",
      "-- check region:202 7:45009999-45014999_strand_+_gene_Chr7-6Mb_reg_203, 9804 candidate probes\n",
      "--- 7416 probes passed check_dic selection.\n",
      "finish in 0.412s, 79 probes kept.\n",
      "-- check region:203 7:45014999-45019999_strand_+_gene_Chr7-6Mb_reg_204, 9918 candidate probes\n",
      "--- 7726 probes passed check_dic selection.\n",
      "finish in 0.428s, 80 probes kept.\n",
      "-- check region:204 7:45019999-45024999_strand_+_gene_Chr7-6Mb_reg_205, 9658 candidate probes\n",
      "--- 4604 probes passed check_dic selection.\n",
      "finish in 0.313s, 63 probes kept.\n",
      "-- check region:205 7:45024999-45029999_strand_+_gene_Chr7-6Mb_reg_206, 9838 candidate probes\n",
      "--- 7320 probes passed check_dic selection.\n",
      "finish in 0.407s, 78 probes kept.\n",
      "-- check region:206 7:45029999-45034999_strand_+_gene_Chr7-6Mb_reg_207, 9802 candidate probes\n",
      "--- 6560 probes passed check_dic selection.\n",
      "finish in 0.384s, 71 probes kept.\n",
      "-- check region:207 7:45034999-45039999_strand_+_gene_Chr7-6Mb_reg_208, 9810 candidate probes\n",
      "--- 5854 probes passed check_dic selection.\n",
      "finish in 0.358s, 76 probes kept.\n",
      "-- check region:208 7:45039999-45044999_strand_+_gene_Chr7-6Mb_reg_209, 9904 candidate probes\n",
      "--- 5890 probes passed check_dic selection.\n",
      "finish in 0.363s, 71 probes kept.\n",
      "-- check region:209 7:45044999-45049999_strand_+_gene_Chr7-6Mb_reg_210, 9684 candidate probes\n",
      "--- 7984 probes passed check_dic selection.\n",
      "finish in 0.434s, 82 probes kept.\n",
      "-- check region:210 7:45049999-45054999_strand_+_gene_Chr7-6Mb_reg_211, 9892 candidate probes\n",
      "--- 7454 probes passed check_dic selection.\n",
      "finish in 0.417s, 82 probes kept.\n",
      "-- check region:211 7:45054999-45059999_strand_+_gene_Chr7-6Mb_reg_212, 9798 candidate probes\n",
      "--- 5470 probes passed check_dic selection.\n",
      "finish in 0.347s, 71 probes kept.\n",
      "-- check region:212 7:45059999-45064999_strand_+_gene_Chr7-6Mb_reg_213, 9886 candidate probes\n",
      "--- 6050 probes passed check_dic selection.\n",
      "finish in 0.366s, 66 probes kept.\n",
      "-- check region:213 7:45064999-45069999_strand_+_gene_Chr7-6Mb_reg_214, 9896 candidate probes\n",
      "--- 3778 probes passed check_dic selection.\n",
      "finish in 0.289s, 61 probes kept.\n",
      "-- check region:214 7:45069999-45074999_strand_+_gene_Chr7-6Mb_reg_215, 9846 candidate probes\n",
      "--- 4362 probes passed check_dic selection.\n",
      "finish in 0.307s, 55 probes kept.\n",
      "-- check region:215 7:45074999-45079999_strand_+_gene_Chr7-6Mb_reg_216, 9820 candidate probes\n",
      "--- 7030 probes passed check_dic selection.\n",
      "finish in 0.398s, 79 probes kept.\n",
      "-- check region:216 7:45079999-45084999_strand_+_gene_Chr7-6Mb_reg_217, 9724 candidate probes\n",
      "--- 4876 probes passed check_dic selection.\n",
      "finish in 0.325s, 61 probes kept.\n",
      "-- check region:217 7:45084999-45089999_strand_+_gene_Chr7-6Mb_reg_218, 9640 candidate probes\n",
      "--- 4474 probes passed check_dic selection.\n",
      "finish in 0.309s, 60 probes kept.\n",
      "-- check region:218 7:45089999-45094999_strand_+_gene_Chr7-6Mb_reg_219, 9848 candidate probes\n",
      "--- 6160 probes passed check_dic selection.\n",
      "finish in 0.368s, 70 probes kept.\n",
      "-- check region:219 7:45094999-45099999_strand_+_gene_Chr7-6Mb_reg_220, 9602 candidate probes\n",
      "--- 4186 probes passed check_dic selection.\n",
      "finish in 0.300s, 54 probes kept.\n",
      "-- check region:220 7:45099999-45104999_strand_+_gene_Chr7-6Mb_reg_221, 9574 candidate probes\n",
      "--- 5218 probes passed check_dic selection.\n",
      "finish in 0.331s, 67 probes kept.\n",
      "-- check region:221 7:45104999-45109999_strand_+_gene_Chr7-6Mb_reg_222, 9506 candidate probes\n",
      "--- 3296 probes passed check_dic selection.\n",
      "finish in 0.267s, 51 probes kept.\n",
      "-- check region:222 7:45109999-45114999_strand_+_gene_Chr7-6Mb_reg_223, 9482 candidate probes\n",
      "--- 3912 probes passed check_dic selection.\n",
      "finish in 0.287s, 54 probes kept.\n",
      "-- check region:223 7:45114999-45119999_strand_+_gene_Chr7-6Mb_reg_224, 9584 candidate probes\n",
      "--- 4072 probes passed check_dic selection.\n",
      "finish in 0.294s, 57 probes kept.\n",
      "-- check region:224 7:45119999-45124999_strand_+_gene_Chr7-6Mb_reg_225, 9918 candidate probes\n",
      "--- 7486 probes passed check_dic selection.\n",
      "finish in 0.416s, 76 probes kept.\n",
      "-- check region:225 7:45124999-45129999_strand_+_gene_Chr7-6Mb_reg_226, 9918 candidate probes\n",
      "--- 7710 probes passed check_dic selection.\n",
      "finish in 0.426s, 81 probes kept.\n",
      "-- check region:226 7:45129999-45134999_strand_+_gene_Chr7-6Mb_reg_227, 9724 candidate probes\n",
      "--- 5526 probes passed check_dic selection.\n",
      "finish in 0.346s, 73 probes kept.\n",
      "-- check region:227 7:45134999-45139999_strand_+_gene_Chr7-6Mb_reg_228, 9772 candidate probes\n",
      "--- 6146 probes passed check_dic selection.\n",
      "finish in 0.368s, 68 probes kept.\n",
      "-- check region:228 7:45139999-45144999_strand_+_gene_Chr7-6Mb_reg_229, 9886 candidate probes\n",
      "--- 6754 probes passed check_dic selection.\n",
      "finish in 0.393s, 76 probes kept.\n",
      "-- check region:229 7:45144999-45149999_strand_+_gene_Chr7-6Mb_reg_230, 9918 candidate probes\n",
      "--- 7562 probes passed check_dic selection.\n",
      "finish in 0.420s, 74 probes kept.\n",
      "-- check region:230 7:45149999-45154999_strand_+_gene_Chr7-6Mb_reg_231, 9528 candidate probes\n",
      "--- 5240 probes passed check_dic selection.\n",
      "finish in 0.334s, 70 probes kept.\n",
      "-- check region:231 7:45154999-45159999_strand_+_gene_Chr7-6Mb_reg_232, 9776 candidate probes\n",
      "--- 6464 probes passed check_dic selection.\n",
      "finish in 0.382s, 79 probes kept.\n",
      "-- check region:232 7:45159999-45164999_strand_+_gene_Chr7-6Mb_reg_233, 9882 candidate probes\n",
      "--- 5754 probes passed check_dic selection.\n",
      "finish in 0.357s, 73 probes kept.\n",
      "-- check region:233 7:45164999-45169999_strand_+_gene_Chr7-6Mb_reg_234, 9876 candidate probes\n",
      "--- 7004 probes passed check_dic selection.\n",
      "finish in 0.398s, 83 probes kept.\n",
      "-- check region:234 7:45169999-45174999_strand_+_gene_Chr7-6Mb_reg_235, 9592 candidate probes\n",
      "--- 6634 probes passed check_dic selection.\n",
      "finish in 0.386s, 72 probes kept.\n",
      "-- check region:235 7:45174999-45179999_strand_+_gene_Chr7-6Mb_reg_236, 9898 candidate probes\n",
      "--- 8344 probes passed check_dic selection.\n",
      "finish in 0.452s, 85 probes kept.\n",
      "-- check region:236 7:45179999-45184999_strand_+_gene_Chr7-6Mb_reg_237, 9864 candidate probes\n",
      "--- 7738 probes passed check_dic selection.\n",
      "finish in 0.426s, 81 probes kept.\n",
      "-- check region:237 7:45184999-45189999_strand_+_gene_Chr7-6Mb_reg_238, 9862 candidate probes\n",
      "--- 7788 probes passed check_dic selection.\n",
      "finish in 0.430s, 80 probes kept.\n",
      "-- check region:238 7:45189999-45194999_strand_+_gene_Chr7-6Mb_reg_239, 9802 candidate probes\n",
      "--- 6532 probes passed check_dic selection.\n",
      "finish in 0.381s, 71 probes kept.\n",
      "-- check region:239 7:45194999-45199999_strand_+_gene_Chr7-6Mb_reg_240, 9876 candidate probes\n",
      "--- 7300 probes passed check_dic selection.\n",
      "finish in 0.415s, 80 probes kept.\n",
      "-- check region:240 7:45199999-45204999_strand_+_gene_Chr7-6Mb_reg_241, 9608 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 5918 probes passed check_dic selection.\n",
      "finish in 0.359s, 69 probes kept.\n",
      "-- check region:241 7:45204999-45209999_strand_+_gene_Chr7-6Mb_reg_242, 9798 candidate probes\n",
      "--- 4368 probes passed check_dic selection.\n",
      "finish in 0.309s, 58 probes kept.\n",
      "-- check region:242 7:45209999-45214999_strand_+_gene_Chr7-6Mb_reg_243, 9578 candidate probes\n",
      "--- 5824 probes passed check_dic selection.\n",
      "finish in 0.357s, 71 probes kept.\n",
      "-- check region:243 7:45214999-45219999_strand_+_gene_Chr7-6Mb_reg_244, 9864 candidate probes\n",
      "--- 6256 probes passed check_dic selection.\n",
      "finish in 0.373s, 71 probes kept.\n",
      "-- check region:244 7:45219999-45224999_strand_+_gene_Chr7-6Mb_reg_245, 9816 candidate probes\n",
      "--- 6128 probes passed check_dic selection.\n",
      "finish in 0.368s, 74 probes kept.\n",
      "-- check region:245 7:45224999-45229999_strand_+_gene_Chr7-6Mb_reg_246, 9816 candidate probes\n",
      "--- 6210 probes passed check_dic selection.\n",
      "finish in 0.370s, 69 probes kept.\n",
      "-- check region:246 7:45229999-45234999_strand_+_gene_Chr7-6Mb_reg_247, 9430 candidate probes\n",
      "--- 7018 probes passed check_dic selection.\n",
      "finish in 0.394s, 74 probes kept.\n",
      "-- check region:247 7:45234999-45239999_strand_+_gene_Chr7-6Mb_reg_248, 9810 candidate probes\n",
      "--- 6802 probes passed check_dic selection.\n",
      "finish in 0.393s, 77 probes kept.\n",
      "-- check region:248 7:45239999-45244999_strand_+_gene_Chr7-6Mb_reg_249, 9606 candidate probes\n",
      "--- 3880 probes passed check_dic selection.\n",
      "finish in 0.289s, 55 probes kept.\n",
      "-- check region:249 7:45244999-45249999_strand_+_gene_Chr7-6Mb_reg_250, 9402 candidate probes\n",
      "--- 854 probes passed check_dic selection.\n",
      "finish in 0.178s, 21 probes kept.\n",
      "-- check region:250 7:45249999-45254999_strand_+_gene_Chr7-6Mb_reg_251, 9586 candidate probes\n",
      "--- 2352 probes passed check_dic selection.\n",
      "finish in 0.235s, 46 probes kept.\n",
      "-- check region:251 7:45254999-45259999_strand_+_gene_Chr7-6Mb_reg_252, 9518 candidate probes\n",
      "--- 5550 probes passed check_dic selection.\n",
      "finish in 0.344s, 70 probes kept.\n",
      "-- check region:252 7:45259999-45264999_strand_+_gene_Chr7-6Mb_reg_253, 9904 candidate probes\n",
      "--- 6522 probes passed check_dic selection.\n",
      "finish in 0.389s, 79 probes kept.\n",
      "-- check region:253 7:45264999-45269999_strand_+_gene_Chr7-6Mb_reg_254, 9912 candidate probes\n",
      "--- 5468 probes passed check_dic selection.\n",
      "finish in 0.349s, 62 probes kept.\n",
      "-- check region:254 7:45269999-45274999_strand_+_gene_Chr7-6Mb_reg_255, 9768 candidate probes\n",
      "--- 6880 probes passed check_dic selection.\n",
      "finish in 0.392s, 74 probes kept.\n",
      "-- check region:255 7:45274999-45279999_strand_+_gene_Chr7-6Mb_reg_256, 9876 candidate probes\n",
      "--- 4888 probes passed check_dic selection.\n",
      "finish in 0.325s, 71 probes kept.\n",
      "-- check region:256 7:45279999-45284999_strand_+_gene_Chr7-6Mb_reg_257, 9876 candidate probes\n",
      "--- 6158 probes passed check_dic selection.\n",
      "finish in 0.372s, 71 probes kept.\n",
      "-- check region:257 7:45284999-45289999_strand_+_gene_Chr7-6Mb_reg_258, 9776 candidate probes\n",
      "--- 6892 probes passed check_dic selection.\n",
      "finish in 0.395s, 75 probes kept.\n",
      "-- check region:258 7:45289999-45294999_strand_+_gene_Chr7-6Mb_reg_259, 9912 candidate probes\n",
      "--- 4348 probes passed check_dic selection.\n",
      "finish in 0.310s, 68 probes kept.\n",
      "-- check region:259 7:45294999-45299999_strand_+_gene_Chr7-6Mb_reg_260, 9908 candidate probes\n",
      "--- 4844 probes passed check_dic selection.\n",
      "finish in 0.325s, 63 probes kept.\n",
      "-- check region:260 7:45299999-45304999_strand_+_gene_Chr7-6Mb_reg_261, 9892 candidate probes\n",
      "--- 6942 probes passed check_dic selection.\n",
      "finish in 0.401s, 75 probes kept.\n",
      "-- check region:261 7:45304999-45309999_strand_+_gene_Chr7-6Mb_reg_262, 9868 candidate probes\n",
      "--- 5896 probes passed check_dic selection.\n",
      "finish in 0.363s, 71 probes kept.\n",
      "-- check region:262 7:45309999-45314999_strand_+_gene_Chr7-6Mb_reg_263, 9818 candidate probes\n",
      "--- 6020 probes passed check_dic selection.\n",
      "finish in 0.365s, 74 probes kept.\n",
      "-- check region:263 7:45314999-45319999_strand_+_gene_Chr7-6Mb_reg_264, 9656 candidate probes\n",
      "--- 5680 probes passed check_dic selection.\n",
      "finish in 0.350s, 67 probes kept.\n",
      "-- check region:264 7:45319999-45324999_strand_+_gene_Chr7-6Mb_reg_265, 9670 candidate probes\n",
      "--- 5532 probes passed check_dic selection.\n",
      "finish in 0.348s, 69 probes kept.\n",
      "-- check region:265 7:45324999-45329999_strand_+_gene_Chr7-6Mb_reg_266, 9406 candidate probes\n",
      "--- 4736 probes passed check_dic selection.\n",
      "finish in 0.314s, 56 probes kept.\n",
      "-- check region:266 7:45329999-45334999_strand_+_gene_Chr7-6Mb_reg_267, 9874 candidate probes\n",
      "--- 6872 probes passed check_dic selection.\n",
      "finish in 0.394s, 76 probes kept.\n",
      "-- check region:267 7:45334999-45339999_strand_+_gene_Chr7-6Mb_reg_268, 9894 candidate probes\n",
      "--- 7492 probes passed check_dic selection.\n",
      "finish in 0.420s, 80 probes kept.\n",
      "-- check region:268 7:45339999-45344999_strand_+_gene_Chr7-6Mb_reg_269, 9766 candidate probes\n",
      "--- 6638 probes passed check_dic selection.\n",
      "finish in 0.386s, 71 probes kept.\n",
      "-- check region:269 7:45344999-45349999_strand_+_gene_Chr7-6Mb_reg_270, 9870 candidate probes\n",
      "--- 5106 probes passed check_dic selection.\n",
      "finish in 0.334s, 64 probes kept.\n",
      "-- check region:270 7:45349999-45354999_strand_+_gene_Chr7-6Mb_reg_271, 9772 candidate probes\n",
      "--- 5598 probes passed check_dic selection.\n",
      "finish in 0.350s, 69 probes kept.\n",
      "-- check region:271 7:45354999-45359999_strand_+_gene_Chr7-6Mb_reg_272, 9556 candidate probes\n",
      "--- 6724 probes passed check_dic selection.\n",
      "finish in 0.389s, 77 probes kept.\n",
      "-- check region:272 7:45359999-45364999_strand_+_gene_Chr7-6Mb_reg_273, 9882 candidate probes\n",
      "--- 7072 probes passed check_dic selection.\n",
      "finish in 0.404s, 77 probes kept.\n",
      "-- check region:273 7:45364999-45369999_strand_+_gene_Chr7-6Mb_reg_274, 9918 candidate probes\n",
      "--- 8926 probes passed check_dic selection.\n",
      "finish in 0.470s, 87 probes kept.\n",
      "-- check region:274 7:45369999-45374999_strand_+_gene_Chr7-6Mb_reg_275, 9904 candidate probes\n",
      "--- 6262 probes passed check_dic selection.\n",
      "finish in 0.376s, 71 probes kept.\n",
      "-- check region:275 7:45374999-45379999_strand_+_gene_Chr7-6Mb_reg_276, 9690 candidate probes\n",
      "--- 5980 probes passed check_dic selection.\n",
      "finish in 0.360s, 67 probes kept.\n",
      "-- check region:276 7:45379999-45384999_strand_+_gene_Chr7-6Mb_reg_277, 9918 candidate probes\n",
      "--- 8086 probes passed check_dic selection.\n",
      "finish in 0.445s, 86 probes kept.\n",
      "-- check region:277 7:45384999-45389999_strand_+_gene_Chr7-6Mb_reg_278, 9798 candidate probes\n",
      "--- 5408 probes passed check_dic selection.\n",
      "finish in 0.341s, 70 probes kept.\n",
      "-- check region:278 7:45389999-45394999_strand_+_gene_Chr7-6Mb_reg_279, 9736 candidate probes\n",
      "--- 4884 probes passed check_dic selection.\n",
      "finish in 0.324s, 65 probes kept.\n",
      "-- check region:279 7:45394999-45399999_strand_+_gene_Chr7-6Mb_reg_280, 9619 candidate probes\n",
      "--- 3224 probes passed check_dic selection.\n",
      "finish in 0.268s, 48 probes kept.\n",
      "-- check region:280 7:45399999-45404999_strand_+_gene_Chr7-6Mb_reg_281, 9404 candidate probes\n",
      "--- 3064 probes passed check_dic selection.\n",
      "finish in 0.256s, 48 probes kept.\n",
      "-- check region:281 7:45404999-45409999_strand_+_gene_Chr7-6Mb_reg_282, 9918 candidate probes\n",
      "--- 7020 probes passed check_dic selection.\n",
      "finish in 0.398s, 72 probes kept.\n",
      "-- check region:282 7:45409999-45414999_strand_+_gene_Chr7-6Mb_reg_283, 9776 candidate probes\n",
      "--- 6944 probes passed check_dic selection.\n",
      "finish in 0.395s, 74 probes kept.\n",
      "-- check region:283 7:45414999-45419999_strand_+_gene_Chr7-6Mb_reg_284, 9884 candidate probes\n",
      "--- 7006 probes passed check_dic selection.\n",
      "finish in 0.406s, 85 probes kept.\n",
      "-- check region:284 7:45419999-45424999_strand_+_gene_Chr7-6Mb_reg_285, 9918 candidate probes\n",
      "--- 8386 probes passed check_dic selection.\n",
      "finish in 0.456s, 86 probes kept.\n",
      "-- check region:285 7:45424999-45429999_strand_+_gene_Chr7-6Mb_reg_286, 9882 candidate probes\n",
      "--- 6412 probes passed check_dic selection.\n",
      "finish in 0.380s, 73 probes kept.\n",
      "-- check region:286 7:45429999-45434999_strand_+_gene_Chr7-6Mb_reg_287, 9846 candidate probes\n",
      "--- 7642 probes passed check_dic selection.\n",
      "finish in 0.421s, 86 probes kept.\n",
      "-- check region:287 7:45434999-45439999_strand_+_gene_Chr7-6Mb_reg_288, 9798 candidate probes\n",
      "--- 7396 probes passed check_dic selection.\n",
      "finish in 0.413s, 80 probes kept.\n",
      "-- check region:288 7:45439999-45444999_strand_+_gene_Chr7-6Mb_reg_289, 9664 candidate probes\n",
      "--- 5840 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.359s, 69 probes kept.\n",
      "-- check region:289 7:45444999-45449999_strand_+_gene_Chr7-6Mb_reg_290, 9752 candidate probes\n",
      "--- 4534 probes passed check_dic selection.\n",
      "finish in 0.313s, 60 probes kept.\n",
      "-- check region:290 7:45449999-45454999_strand_+_gene_Chr7-6Mb_reg_291, 9402 candidate probes\n",
      "--- 3268 probes passed check_dic selection.\n",
      "finish in 0.263s, 46 probes kept.\n",
      "-- check region:291 7:45454999-45459999_strand_+_gene_Chr7-6Mb_reg_292, 9918 candidate probes\n",
      "--- 6352 probes passed check_dic selection.\n",
      "finish in 0.380s, 71 probes kept.\n",
      "-- check region:292 7:45459999-45464999_strand_+_gene_Chr7-6Mb_reg_293, 9782 candidate probes\n",
      "--- 5916 probes passed check_dic selection.\n",
      "finish in 0.359s, 72 probes kept.\n",
      "-- check region:293 7:45464999-45469999_strand_+_gene_Chr7-6Mb_reg_294, 9670 candidate probes\n",
      "--- 5802 probes passed check_dic selection.\n",
      "finish in 0.355s, 70 probes kept.\n",
      "-- check region:294 7:45469999-45474999_strand_+_gene_Chr7-6Mb_reg_295, 9462 candidate probes\n",
      "--- 2892 probes passed check_dic selection.\n",
      "finish in 0.252s, 46 probes kept.\n",
      "-- check region:295 7:45474999-45479999_strand_+_gene_Chr7-6Mb_reg_296, 9554 candidate probes\n",
      "--- 5322 probes passed check_dic selection.\n",
      "finish in 0.336s, 61 probes kept.\n",
      "-- check region:296 7:45479999-45484999_strand_+_gene_Chr7-6Mb_reg_297, 9312 candidate probes\n",
      "--- 2622 probes passed check_dic selection.\n",
      "finish in 0.239s, 36 probes kept.\n",
      "-- check region:297 7:45484999-45489999_strand_+_gene_Chr7-6Mb_reg_298, 9752 candidate probes\n",
      "--- 4676 probes passed check_dic selection.\n",
      "finish in 0.317s, 61 probes kept.\n",
      "-- check region:298 7:45489999-45494999_strand_+_gene_Chr7-6Mb_reg_299, 9882 candidate probes\n",
      "--- 6110 probes passed check_dic selection.\n",
      "finish in 0.369s, 73 probes kept.\n",
      "-- check region:299 7:45494999-45499999_strand_+_gene_Chr7-6Mb_reg_300, 9912 candidate probes\n",
      "--- 6470 probes passed check_dic selection.\n",
      "finish in 0.382s, 71 probes kept.\n",
      "-- check region:300 7:45499999-45504999_strand_+_gene_Chr7-6Mb_reg_301, 9800 candidate probes\n",
      "--- 5846 probes passed check_dic selection.\n",
      "finish in 0.359s, 69 probes kept.\n",
      "-- check region:301 7:45504999-45509999_strand_+_gene_Chr7-6Mb_reg_302, 9836 candidate probes\n",
      "--- 5630 probes passed check_dic selection.\n",
      "finish in 0.354s, 70 probes kept.\n",
      "-- check region:302 7:45509999-45514999_strand_+_gene_Chr7-6Mb_reg_303, 9618 candidate probes\n",
      "--- 4284 probes passed check_dic selection.\n",
      "finish in 0.300s, 48 probes kept.\n",
      "-- check region:303 7:45514999-45519999_strand_+_gene_Chr7-6Mb_reg_304, 9856 candidate probes\n",
      "--- 6940 probes passed check_dic selection.\n",
      "finish in 0.397s, 79 probes kept.\n",
      "-- check region:304 7:45519999-45524999_strand_+_gene_Chr7-6Mb_reg_305, 9342 candidate probes\n",
      "--- 5894 probes passed check_dic selection.\n",
      "finish in 0.349s, 67 probes kept.\n",
      "-- check region:305 7:45524999-45529999_strand_+_gene_Chr7-6Mb_reg_306, 9804 candidate probes\n",
      "--- 5236 probes passed check_dic selection.\n",
      "finish in 0.339s, 61 probes kept.\n",
      "-- check region:306 7:45529999-45534999_strand_+_gene_Chr7-6Mb_reg_307, 9722 candidate probes\n",
      "--- 5688 probes passed check_dic selection.\n",
      "finish in 0.355s, 70 probes kept.\n",
      "-- check region:307 7:45534999-45539999_strand_+_gene_Chr7-6Mb_reg_308, 9760 candidate probes\n",
      "--- 5238 probes passed check_dic selection.\n",
      "finish in 0.339s, 62 probes kept.\n",
      "-- check region:308 7:45539999-45544999_strand_+_gene_Chr7-6Mb_reg_309, 9816 candidate probes\n",
      "--- 4112 probes passed check_dic selection.\n",
      "finish in 0.301s, 54 probes kept.\n",
      "-- check region:309 7:45544999-45549999_strand_+_gene_Chr7-6Mb_reg_310, 9450 candidate probes\n",
      "--- 4714 probes passed check_dic selection.\n",
      "finish in 0.314s, 56 probes kept.\n",
      "-- check region:310 7:45549999-45554999_strand_+_gene_Chr7-6Mb_reg_311, 9790 candidate probes\n",
      "--- 6050 probes passed check_dic selection.\n",
      "finish in 0.368s, 74 probes kept.\n",
      "-- check region:311 7:45554999-45559999_strand_+_gene_Chr7-6Mb_reg_312, 9368 candidate probes\n",
      "--- 4340 probes passed check_dic selection.\n",
      "finish in 0.543s, 59 probes kept.\n",
      "-- check region:312 7:45559999-45564999_strand_+_gene_Chr7-6Mb_reg_313, 9692 candidate probes\n",
      "--- 5000 probes passed check_dic selection.\n",
      "finish in 0.329s, 60 probes kept.\n",
      "-- check region:313 7:45564999-45569999_strand_+_gene_Chr7-6Mb_reg_314, 9644 candidate probes\n",
      "--- 4896 probes passed check_dic selection.\n",
      "finish in 0.324s, 64 probes kept.\n",
      "-- check region:314 7:45569999-45574999_strand_+_gene_Chr7-6Mb_reg_315, 9814 candidate probes\n",
      "--- 6254 probes passed check_dic selection.\n",
      "finish in 0.371s, 72 probes kept.\n",
      "-- check region:315 7:45574999-45579999_strand_+_gene_Chr7-6Mb_reg_316, 9758 candidate probes\n",
      "--- 3652 probes passed check_dic selection.\n",
      "finish in 0.283s, 50 probes kept.\n",
      "-- check region:316 7:45579999-45584999_strand_+_gene_Chr7-6Mb_reg_317, 9724 candidate probes\n",
      "--- 3358 probes passed check_dic selection.\n",
      "finish in 0.270s, 53 probes kept.\n",
      "-- check region:317 7:45584999-45589999_strand_+_gene_Chr7-6Mb_reg_318, 9862 candidate probes\n",
      "--- 7334 probes passed check_dic selection.\n",
      "finish in 0.413s, 76 probes kept.\n",
      "-- check region:318 7:45589999-45594999_strand_+_gene_Chr7-6Mb_reg_319, 9750 candidate probes\n",
      "--- 3326 probes passed check_dic selection.\n",
      "finish in 0.272s, 48 probes kept.\n",
      "-- check region:319 7:45594999-45599999_strand_+_gene_Chr7-6Mb_reg_320, 9806 candidate probes\n",
      "--- 5308 probes passed check_dic selection.\n",
      "finish in 0.340s, 74 probes kept.\n",
      "-- check region:320 7:45599999-45604999_strand_+_gene_Chr7-6Mb_reg_321, 9456 candidate probes\n",
      "--- 2316 probes passed check_dic selection.\n",
      "finish in 0.233s, 40 probes kept.\n",
      "-- check region:321 7:45604999-45609999_strand_+_gene_Chr7-6Mb_reg_322, 9858 candidate probes\n",
      "--- 4806 probes passed check_dic selection.\n",
      "finish in 0.324s, 60 probes kept.\n",
      "-- check region:322 7:45609999-45614999_strand_+_gene_Chr7-6Mb_reg_323, 9652 candidate probes\n",
      "--- 4170 probes passed check_dic selection.\n",
      "finish in 0.300s, 54 probes kept.\n",
      "-- check region:323 7:45614999-45619999_strand_+_gene_Chr7-6Mb_reg_324, 9700 candidate probes\n",
      "--- 8152 probes passed check_dic selection.\n",
      "finish in 0.437s, 87 probes kept.\n",
      "-- check region:324 7:45619999-45624999_strand_+_gene_Chr7-6Mb_reg_325, 9910 candidate probes\n",
      "--- 6778 probes passed check_dic selection.\n",
      "finish in 0.393s, 78 probes kept.\n",
      "-- check region:325 7:45624999-45629999_strand_+_gene_Chr7-6Mb_reg_326, 9770 candidate probes\n",
      "--- 7608 probes passed check_dic selection.\n",
      "finish in 0.419s, 83 probes kept.\n",
      "-- check region:326 7:45629999-45634999_strand_+_gene_Chr7-6Mb_reg_327, 9844 candidate probes\n",
      "--- 7114 probes passed check_dic selection.\n",
      "finish in 0.401s, 78 probes kept.\n",
      "-- check region:327 7:45634999-45639999_strand_+_gene_Chr7-6Mb_reg_328, 9918 candidate probes\n",
      "--- 7530 probes passed check_dic selection.\n",
      "finish in 0.418s, 80 probes kept.\n",
      "-- check region:328 7:45639999-45644999_strand_+_gene_Chr7-6Mb_reg_329, 9918 candidate probes\n",
      "--- 8452 probes passed check_dic selection.\n",
      "finish in 0.451s, 86 probes kept.\n",
      "-- check region:329 7:45644999-45649999_strand_+_gene_Chr7-6Mb_reg_330, 9918 candidate probes\n",
      "--- 6184 probes passed check_dic selection.\n",
      "finish in 0.372s, 80 probes kept.\n",
      "-- check region:330 7:45649999-45654999_strand_+_gene_Chr7-6Mb_reg_331, 9564 candidate probes\n",
      "--- 5956 probes passed check_dic selection.\n",
      "finish in 0.361s, 74 probes kept.\n",
      "-- check region:331 7:45654999-45659999_strand_+_gene_Chr7-6Mb_reg_332, 9710 candidate probes\n",
      "--- 4270 probes passed check_dic selection.\n",
      "finish in 0.303s, 59 probes kept.\n",
      "-- check region:332 7:45659999-45664999_strand_+_gene_Chr7-6Mb_reg_333, 9648 candidate probes\n",
      "--- 4874 probes passed check_dic selection.\n",
      "finish in 0.323s, 64 probes kept.\n",
      "-- check region:333 7:45664999-45669999_strand_+_gene_Chr7-6Mb_reg_334, 9820 candidate probes\n",
      "--- 5344 probes passed check_dic selection.\n",
      "finish in 0.341s, 69 probes kept.\n",
      "-- check region:334 7:45669999-45674999_strand_+_gene_Chr7-6Mb_reg_335, 9850 candidate probes\n",
      "--- 4830 probes passed check_dic selection.\n",
      "finish in 0.324s, 62 probes kept.\n",
      "-- check region:335 7:45674999-45679999_strand_+_gene_Chr7-6Mb_reg_336, 9918 candidate probes\n",
      "--- 7444 probes passed check_dic selection.\n",
      "finish in 0.418s, 85 probes kept.\n",
      "-- check region:336 7:45679999-45684999_strand_+_gene_Chr7-6Mb_reg_337, 9738 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 7490 probes passed check_dic selection.\n",
      "finish in 0.414s, 80 probes kept.\n",
      "-- check region:337 7:45684999-45689999_strand_+_gene_Chr7-6Mb_reg_338, 9854 candidate probes\n",
      "--- 6154 probes passed check_dic selection.\n",
      "finish in 0.370s, 66 probes kept.\n",
      "-- check region:338 7:45689999-45694999_strand_+_gene_Chr7-6Mb_reg_339, 9918 candidate probes\n",
      "--- 7976 probes passed check_dic selection.\n",
      "finish in 0.438s, 91 probes kept.\n",
      "-- check region:339 7:45694999-45699999_strand_+_gene_Chr7-6Mb_reg_340, 9312 candidate probes\n",
      "--- 4176 probes passed check_dic selection.\n",
      "finish in 0.295s, 55 probes kept.\n",
      "-- check region:340 7:45699999-45704999_strand_+_gene_Chr7-6Mb_reg_341, 9832 candidate probes\n",
      "--- 7976 probes passed check_dic selection.\n",
      "finish in 0.431s, 88 probes kept.\n",
      "-- check region:341 7:45704999-45709999_strand_+_gene_Chr7-6Mb_reg_342, 9918 candidate probes\n",
      "--- 8648 probes passed check_dic selection.\n",
      "finish in 0.458s, 89 probes kept.\n",
      "-- check region:342 7:45709999-45714999_strand_+_gene_Chr7-6Mb_reg_343, 9918 candidate probes\n",
      "--- 8218 probes passed check_dic selection.\n",
      "finish in 0.444s, 79 probes kept.\n",
      "-- check region:343 7:45714999-45719999_strand_+_gene_Chr7-6Mb_reg_344, 9918 candidate probes\n",
      "--- 7694 probes passed check_dic selection.\n",
      "finish in 0.425s, 81 probes kept.\n",
      "-- check region:344 7:45719999-45724999_strand_+_gene_Chr7-6Mb_reg_345, 9918 candidate probes\n",
      "--- 8424 probes passed check_dic selection.\n",
      "finish in 0.451s, 84 probes kept.\n",
      "-- check region:345 7:45724999-45729999_strand_+_gene_Chr7-6Mb_reg_346, 9820 candidate probes\n",
      "--- 7538 probes passed check_dic selection.\n",
      "finish in 0.418s, 78 probes kept.\n",
      "-- check region:346 7:45729999-45734999_strand_+_gene_Chr7-6Mb_reg_347, 9875 candidate probes\n",
      "--- 6308 probes passed check_dic selection.\n",
      "finish in 0.377s, 72 probes kept.\n",
      "-- check region:347 7:45734999-45739999_strand_+_gene_Chr7-6Mb_reg_348, 9532 candidate probes\n",
      "--- 2972 probes passed check_dic selection.\n",
      "finish in 0.255s, 46 probes kept.\n",
      "-- check region:348 7:45739999-45744999_strand_+_gene_Chr7-6Mb_reg_349, 9564 candidate probes\n",
      "--- 5402 probes passed check_dic selection.\n",
      "finish in 0.340s, 68 probes kept.\n",
      "-- check region:349 7:45744999-45749999_strand_+_gene_Chr7-6Mb_reg_350, 9830 candidate probes\n",
      "--- 6134 probes passed check_dic selection.\n",
      "finish in 0.368s, 75 probes kept.\n",
      "-- check region:350 7:45749999-45754999_strand_+_gene_Chr7-6Mb_reg_351, 9712 candidate probes\n",
      "--- 3982 probes passed check_dic selection.\n",
      "finish in 0.293s, 55 probes kept.\n",
      "-- check region:351 7:45754999-45759999_strand_+_gene_Chr7-6Mb_reg_352, 9238 candidate probes\n",
      "--- 6516 probes passed check_dic selection.\n",
      "finish in 0.372s, 78 probes kept.\n",
      "-- check region:352 7:45759999-45764999_strand_+_gene_Chr7-6Mb_reg_353, 9362 candidate probes\n",
      "--- 3316 probes passed check_dic selection.\n",
      "finish in 0.265s, 51 probes kept.\n",
      "-- check region:353 7:45764999-45769999_strand_+_gene_Chr7-6Mb_reg_354, 9366 candidate probes\n",
      "--- 4142 probes passed check_dic selection.\n",
      "finish in 0.293s, 53 probes kept.\n",
      "-- check region:354 7:45769999-45774999_strand_+_gene_Chr7-6Mb_reg_355, 9608 candidate probes\n",
      "--- 5226 probes passed check_dic selection.\n",
      "finish in 0.335s, 62 probes kept.\n",
      "-- check region:355 7:45774999-45779999_strand_+_gene_Chr7-6Mb_reg_356, 9744 candidate probes\n",
      "--- 6180 probes passed check_dic selection.\n",
      "finish in 0.367s, 72 probes kept.\n",
      "-- check region:356 7:45779999-45784999_strand_+_gene_Chr7-6Mb_reg_357, 9552 candidate probes\n",
      "--- 5854 probes passed check_dic selection.\n",
      "finish in 0.355s, 74 probes kept.\n",
      "-- check region:357 7:45784999-45789999_strand_+_gene_Chr7-6Mb_reg_358, 9890 candidate probes\n",
      "--- 7724 probes passed check_dic selection.\n",
      "finish in 0.424s, 85 probes kept.\n",
      "-- check region:358 7:45789999-45794999_strand_+_gene_Chr7-6Mb_reg_359, 9918 candidate probes\n",
      "--- 7118 probes passed check_dic selection.\n",
      "finish in 0.403s, 79 probes kept.\n",
      "-- check region:359 7:45794999-45799999_strand_+_gene_Chr7-6Mb_reg_360, 9800 candidate probes\n",
      "--- 6656 probes passed check_dic selection.\n",
      "finish in 0.384s, 73 probes kept.\n",
      "-- check region:360 7:45799999-45804999_strand_+_gene_Chr7-6Mb_reg_361, 9894 candidate probes\n",
      "--- 6940 probes passed check_dic selection.\n",
      "finish in 0.401s, 80 probes kept.\n",
      "-- check region:361 7:45804999-45809999_strand_+_gene_Chr7-6Mb_reg_362, 9918 candidate probes\n",
      "--- 7994 probes passed check_dic selection.\n",
      "finish in 0.439s, 86 probes kept.\n",
      "-- check region:362 7:45809999-45814999_strand_+_gene_Chr7-6Mb_reg_363, 9918 candidate probes\n",
      "--- 8580 probes passed check_dic selection.\n",
      "finish in 0.450s, 89 probes kept.\n",
      "-- check region:363 7:45814999-45819999_strand_+_gene_Chr7-6Mb_reg_364, 9918 candidate probes\n",
      "--- 7292 probes passed check_dic selection.\n",
      "finish in 0.411s, 86 probes kept.\n",
      "-- check region:364 7:45819999-45824999_strand_+_gene_Chr7-6Mb_reg_365, 9594 candidate probes\n",
      "--- 6330 probes passed check_dic selection.\n",
      "finish in 0.372s, 73 probes kept.\n",
      "-- check region:365 7:45824999-45829999_strand_+_gene_Chr7-6Mb_reg_366, 9918 candidate probes\n",
      "--- 8092 probes passed check_dic selection.\n",
      "finish in 0.439s, 82 probes kept.\n",
      "-- check region:366 7:45829999-45834999_strand_+_gene_Chr7-6Mb_reg_367, 9918 candidate probes\n",
      "--- 8508 probes passed check_dic selection.\n",
      "finish in 0.455s, 90 probes kept.\n",
      "-- check region:367 7:45834999-45839999_strand_+_gene_Chr7-6Mb_reg_368, 9542 candidate probes\n",
      "--- 3830 probes passed check_dic selection.\n",
      "finish in 0.284s, 56 probes kept.\n",
      "-- check region:368 7:45839999-45844999_strand_+_gene_Chr7-6Mb_reg_369, 9881 candidate probes\n",
      "--- 6906 probes passed check_dic selection.\n",
      "finish in 0.394s, 79 probes kept.\n",
      "-- check region:369 7:45844999-45849999_strand_+_gene_Chr7-6Mb_reg_370, 9280 candidate probes\n",
      "--- 3708 probes passed check_dic selection.\n",
      "finish in 0.274s, 55 probes kept.\n",
      "-- check region:370 7:45849999-45854999_strand_+_gene_Chr7-6Mb_reg_371, 9918 candidate probes\n",
      "--- 7424 probes passed check_dic selection.\n",
      "finish in 0.414s, 81 probes kept.\n",
      "-- check region:371 7:45854999-45859999_strand_+_gene_Chr7-6Mb_reg_372, 9918 candidate probes\n",
      "--- 6298 probes passed check_dic selection.\n",
      "finish in 0.376s, 70 probes kept.\n",
      "-- check region:372 7:45859999-45864999_strand_+_gene_Chr7-6Mb_reg_373, 9918 candidate probes\n",
      "--- 7812 probes passed check_dic selection.\n",
      "finish in 0.435s, 86 probes kept.\n",
      "-- check region:373 7:45864999-45869999_strand_+_gene_Chr7-6Mb_reg_374, 9886 candidate probes\n",
      "--- 7230 probes passed check_dic selection.\n",
      "finish in 0.407s, 83 probes kept.\n",
      "-- check region:374 7:45869999-45874999_strand_+_gene_Chr7-6Mb_reg_375, 9904 candidate probes\n",
      "--- 8338 probes passed check_dic selection.\n",
      "finish in 0.447s, 83 probes kept.\n",
      "-- check region:375 7:45874999-45879999_strand_+_gene_Chr7-6Mb_reg_376, 9826 candidate probes\n",
      "--- 3386 probes passed check_dic selection.\n",
      "finish in 0.274s, 46 probes kept.\n",
      "-- check region:376 7:45879999-45884999_strand_+_gene_Chr7-6Mb_reg_377, 9882 candidate probes\n",
      "--- 5460 probes passed check_dic selection.\n",
      "finish in 0.345s, 64 probes kept.\n",
      "-- check region:377 7:45884999-45889999_strand_+_gene_Chr7-6Mb_reg_378, 9772 candidate probes\n",
      "--- 6046 probes passed check_dic selection.\n",
      "finish in 0.365s, 68 probes kept.\n",
      "-- check region:378 7:45889999-45894999_strand_+_gene_Chr7-6Mb_reg_379, 9854 candidate probes\n",
      "--- 5770 probes passed check_dic selection.\n",
      "finish in 0.355s, 73 probes kept.\n",
      "-- check region:379 7:45894999-45899999_strand_+_gene_Chr7-6Mb_reg_380, 9862 candidate probes\n",
      "--- 6954 probes passed check_dic selection.\n",
      "finish in 0.403s, 75 probes kept.\n",
      "-- check region:380 7:45899999-45904999_strand_+_gene_Chr7-6Mb_reg_381, 9330 candidate probes\n",
      "--- 4604 probes passed check_dic selection.\n",
      "finish in 0.305s, 55 probes kept.\n",
      "-- check region:381 7:45904999-45909999_strand_+_gene_Chr7-6Mb_reg_382, 9822 candidate probes\n",
      "--- 4876 probes passed check_dic selection.\n",
      "finish in 0.327s, 60 probes kept.\n",
      "-- check region:382 7:45909999-45914999_strand_+_gene_Chr7-6Mb_reg_383, 9748 candidate probes\n",
      "--- 3818 probes passed check_dic selection.\n",
      "finish in 0.291s, 44 probes kept.\n",
      "-- check region:383 7:45914999-45919999_strand_+_gene_Chr7-6Mb_reg_384, 9716 candidate probes\n",
      "--- 7436 probes passed check_dic selection.\n",
      "finish in 0.410s, 79 probes kept.\n",
      "-- check region:384 7:45919999-45924999_strand_+_gene_Chr7-6Mb_reg_385, 9918 candidate probes\n",
      "--- 8254 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.441s, 86 probes kept.\n",
      "-- check region:385 7:45924999-45929999_strand_+_gene_Chr7-6Mb_reg_386, 9874 candidate probes\n",
      "--- 2740 probes passed check_dic selection.\n",
      "finish in 0.255s, 42 probes kept.\n",
      "-- check region:386 7:45929999-45934999_strand_+_gene_Chr7-6Mb_reg_387, 9284 candidate probes\n",
      "--- 2518 probes passed check_dic selection.\n",
      "finish in 0.237s, 42 probes kept.\n",
      "-- check region:387 7:45934999-45939999_strand_+_gene_Chr7-6Mb_reg_388, 9890 candidate probes\n",
      "--- 6494 probes passed check_dic selection.\n",
      "finish in 0.384s, 75 probes kept.\n",
      "-- check region:388 7:45939999-45944999_strand_+_gene_Chr7-6Mb_reg_389, 9768 candidate probes\n",
      "--- 5578 probes passed check_dic selection.\n",
      "finish in 0.348s, 64 probes kept.\n",
      "-- check region:389 7:45944999-45949999_strand_+_gene_Chr7-6Mb_reg_390, 9918 candidate probes\n",
      "--- 6662 probes passed check_dic selection.\n",
      "finish in 0.388s, 76 probes kept.\n",
      "-- check region:390 7:45949999-45954999_strand_+_gene_Chr7-6Mb_reg_391, 9378 candidate probes\n",
      "--- 3118 probes passed check_dic selection.\n",
      "finish in 0.259s, 40 probes kept.\n",
      "-- check region:391 7:45954999-45959999_strand_+_gene_Chr7-6Mb_reg_392, 9918 candidate probes\n",
      "--- 5502 probes passed check_dic selection.\n",
      "finish in 0.350s, 71 probes kept.\n",
      "-- check region:392 7:45959999-45964999_strand_+_gene_Chr7-6Mb_reg_393, 8858 candidate probes\n",
      "--- 2858 probes passed check_dic selection.\n",
      "finish in 0.244s, 46 probes kept.\n",
      "-- check region:393 7:45964999-45969999_strand_+_gene_Chr7-6Mb_reg_394, 9918 candidate probes\n",
      "--- 7608 probes passed check_dic selection.\n",
      "finish in 0.419s, 84 probes kept.\n",
      "-- check region:394 7:45969999-45974999_strand_+_gene_Chr7-6Mb_reg_395, 9918 candidate probes\n",
      "--- 6942 probes passed check_dic selection.\n",
      "finish in 0.399s, 76 probes kept.\n",
      "-- check region:395 7:45974999-45979999_strand_+_gene_Chr7-6Mb_reg_396, 9910 candidate probes\n",
      "--- 6254 probes passed check_dic selection.\n",
      "finish in 0.376s, 74 probes kept.\n",
      "-- check region:396 7:45979999-45984999_strand_+_gene_Chr7-6Mb_reg_397, 9824 candidate probes\n",
      "--- 7056 probes passed check_dic selection.\n",
      "finish in 0.400s, 71 probes kept.\n",
      "-- check region:397 7:45984999-45989999_strand_+_gene_Chr7-6Mb_reg_398, 9896 candidate probes\n",
      "--- 6618 probes passed check_dic selection.\n",
      "finish in 0.387s, 80 probes kept.\n",
      "-- check region:398 7:45989999-45994999_strand_+_gene_Chr7-6Mb_reg_399, 9718 candidate probes\n",
      "--- 6076 probes passed check_dic selection.\n",
      "finish in 0.364s, 74 probes kept.\n",
      "-- check region:399 7:45994999-45999999_strand_+_gene_Chr7-6Mb_reg_400, 9918 candidate probes\n",
      "--- 6718 probes passed check_dic selection.\n",
      "finish in 0.391s, 79 probes kept.\n",
      "-- check region:400 7:45999999-46004999_strand_+_gene_Chr7-6Mb_reg_401, 9858 candidate probes\n",
      "--- 5150 probes passed check_dic selection.\n",
      "finish in 0.337s, 62 probes kept.\n",
      "-- check region:401 7:46004999-46009999_strand_+_gene_Chr7-6Mb_reg_402, 9712 candidate probes\n",
      "--- 6030 probes passed check_dic selection.\n",
      "finish in 0.369s, 83 probes kept.\n",
      "-- check region:402 7:46009999-46014999_strand_+_gene_Chr7-6Mb_reg_403, 9570 candidate probes\n",
      "--- 5790 probes passed check_dic selection.\n",
      "finish in 0.353s, 65 probes kept.\n",
      "-- check region:403 7:46014999-46019999_strand_+_gene_Chr7-6Mb_reg_404, 9738 candidate probes\n",
      "--- 7040 probes passed check_dic selection.\n",
      "finish in 0.400s, 81 probes kept.\n",
      "-- check region:404 7:46019999-46024999_strand_+_gene_Chr7-6Mb_reg_405, 9736 candidate probes\n",
      "--- 5112 probes passed check_dic selection.\n",
      "finish in 0.333s, 62 probes kept.\n",
      "-- check region:405 7:46024999-46029999_strand_+_gene_Chr7-6Mb_reg_406, 9758 candidate probes\n",
      "--- 5022 probes passed check_dic selection.\n",
      "finish in 0.329s, 64 probes kept.\n",
      "-- check region:406 7:46029999-46034999_strand_+_gene_Chr7-6Mb_reg_407, 9886 candidate probes\n",
      "--- 5964 probes passed check_dic selection.\n",
      "finish in 0.361s, 75 probes kept.\n",
      "-- check region:407 7:46034999-46039999_strand_+_gene_Chr7-6Mb_reg_408, 9634 candidate probes\n",
      "--- 5694 probes passed check_dic selection.\n",
      "finish in 0.351s, 72 probes kept.\n",
      "-- check region:408 7:46039999-46044999_strand_+_gene_Chr7-6Mb_reg_409, 9808 candidate probes\n",
      "--- 6736 probes passed check_dic selection.\n",
      "finish in 0.388s, 77 probes kept.\n",
      "-- check region:409 7:46044999-46049999_strand_+_gene_Chr7-6Mb_reg_410, 8908 candidate probes\n",
      "--- 6616 probes passed check_dic selection.\n",
      "finish in 0.370s, 71 probes kept.\n",
      "-- check region:410 7:46049999-46054999_strand_+_gene_Chr7-6Mb_reg_411, 9178 candidate probes\n",
      "--- 5198 probes passed check_dic selection.\n",
      "finish in 0.327s, 63 probes kept.\n",
      "-- check region:411 7:46054999-46059999_strand_+_gene_Chr7-6Mb_reg_412, 9818 candidate probes\n",
      "--- 6786 probes passed check_dic selection.\n",
      "finish in 0.392s, 72 probes kept.\n",
      "-- check region:412 7:46059999-46064999_strand_+_gene_Chr7-6Mb_reg_413, 9872 candidate probes\n",
      "--- 7108 probes passed check_dic selection.\n",
      "finish in 0.401s, 79 probes kept.\n",
      "-- check region:413 7:46064999-46069999_strand_+_gene_Chr7-6Mb_reg_414, 9866 candidate probes\n",
      "--- 5660 probes passed check_dic selection.\n",
      "finish in 0.356s, 72 probes kept.\n",
      "-- check region:414 7:46069999-46074999_strand_+_gene_Chr7-6Mb_reg_415, 9822 candidate probes\n",
      "--- 6026 probes passed check_dic selection.\n",
      "finish in 0.365s, 71 probes kept.\n",
      "-- check region:415 7:46074999-46079999_strand_+_gene_Chr7-6Mb_reg_416, 9908 candidate probes\n",
      "--- 6936 probes passed check_dic selection.\n",
      "finish in 0.396s, 75 probes kept.\n",
      "-- check region:416 7:46079999-46084999_strand_+_gene_Chr7-6Mb_reg_417, 9918 candidate probes\n",
      "--- 7916 probes passed check_dic selection.\n",
      "finish in 0.432s, 80 probes kept.\n",
      "-- check region:417 7:46084999-46089999_strand_+_gene_Chr7-6Mb_reg_418, 9842 candidate probes\n",
      "--- 4868 probes passed check_dic selection.\n",
      "finish in 0.326s, 70 probes kept.\n",
      "-- check region:418 7:46089999-46094999_strand_+_gene_Chr7-6Mb_reg_419, 9466 candidate probes\n",
      "--- 4898 probes passed check_dic selection.\n",
      "finish in 0.321s, 65 probes kept.\n",
      "-- check region:419 7:46094999-46099999_strand_+_gene_Chr7-6Mb_reg_420, 9918 candidate probes\n",
      "--- 8926 probes passed check_dic selection.\n",
      "finish in 0.469s, 89 probes kept.\n",
      "-- check region:420 7:46099999-46104999_strand_+_gene_Chr7-6Mb_reg_421, 9790 candidate probes\n",
      "--- 7438 probes passed check_dic selection.\n",
      "finish in 0.414s, 77 probes kept.\n",
      "-- check region:421 7:46104999-46109999_strand_+_gene_Chr7-6Mb_reg_422, 9918 candidate probes\n",
      "--- 8974 probes passed check_dic selection.\n",
      "finish in 0.471s, 91 probes kept.\n",
      "-- check region:422 7:46109999-46114999_strand_+_gene_Chr7-6Mb_reg_423, 9918 candidate probes\n",
      "--- 8102 probes passed check_dic selection.\n",
      "finish in 0.439s, 84 probes kept.\n",
      "-- check region:423 7:46114999-46119999_strand_+_gene_Chr7-6Mb_reg_424, 9858 candidate probes\n",
      "--- 8344 probes passed check_dic selection.\n",
      "finish in 0.448s, 92 probes kept.\n",
      "-- check region:424 7:46119999-46124999_strand_+_gene_Chr7-6Mb_reg_425, 9872 candidate probes\n",
      "--- 8334 probes passed check_dic selection.\n",
      "finish in 0.452s, 87 probes kept.\n",
      "-- check region:425 7:46124999-46129999_strand_+_gene_Chr7-6Mb_reg_426, 9862 candidate probes\n",
      "--- 6516 probes passed check_dic selection.\n",
      "finish in 0.381s, 82 probes kept.\n",
      "-- check region:426 7:46129999-46134999_strand_+_gene_Chr7-6Mb_reg_427, 9878 candidate probes\n",
      "--- 7338 probes passed check_dic selection.\n",
      "finish in 0.416s, 81 probes kept.\n",
      "-- check region:427 7:46134999-46139999_strand_+_gene_Chr7-6Mb_reg_428, 9862 candidate probes\n",
      "--- 7058 probes passed check_dic selection.\n",
      "finish in 0.401s, 80 probes kept.\n",
      "-- check region:428 7:46139999-46144999_strand_+_gene_Chr7-6Mb_reg_429, 9382 candidate probes\n",
      "--- 6090 probes passed check_dic selection.\n",
      "finish in 0.358s, 65 probes kept.\n",
      "-- check region:429 7:46144999-46149999_strand_+_gene_Chr7-6Mb_reg_430, 9866 candidate probes\n",
      "--- 6578 probes passed check_dic selection.\n",
      "finish in 0.384s, 74 probes kept.\n",
      "-- check region:430 7:46149999-46154999_strand_+_gene_Chr7-6Mb_reg_431, 9726 candidate probes\n",
      "--- 6664 probes passed check_dic selection.\n",
      "finish in 0.386s, 79 probes kept.\n",
      "-- check region:431 7:46154999-46159999_strand_+_gene_Chr7-6Mb_reg_432, 9897 candidate probes\n",
      "--- 7258 probes passed check_dic selection.\n",
      "finish in 0.408s, 80 probes kept.\n",
      "-- check region:432 7:46159999-46164999_strand_+_gene_Chr7-6Mb_reg_433, 9868 candidate probes\n",
      "--- 6332 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.377s, 73 probes kept.\n",
      "-- check region:433 7:46164999-46169999_strand_+_gene_Chr7-6Mb_reg_434, 9892 candidate probes\n",
      "--- 6288 probes passed check_dic selection.\n",
      "finish in 0.378s, 73 probes kept.\n",
      "-- check region:434 7:46169999-46174999_strand_+_gene_Chr7-6Mb_reg_435, 9882 candidate probes\n",
      "--- 6632 probes passed check_dic selection.\n",
      "finish in 0.386s, 72 probes kept.\n",
      "-- check region:435 7:46174999-46179999_strand_+_gene_Chr7-6Mb_reg_436, 9918 candidate probes\n",
      "--- 8168 probes passed check_dic selection.\n",
      "finish in 0.440s, 86 probes kept.\n",
      "-- check region:436 7:46179999-46184999_strand_+_gene_Chr7-6Mb_reg_437, 9712 candidate probes\n",
      "--- 6162 probes passed check_dic selection.\n",
      "finish in 0.367s, 76 probes kept.\n",
      "-- check region:437 7:46184999-46189999_strand_+_gene_Chr7-6Mb_reg_438, 9780 candidate probes\n",
      "--- 5716 probes passed check_dic selection.\n",
      "finish in 0.353s, 66 probes kept.\n",
      "-- check region:438 7:46189999-46194999_strand_+_gene_Chr7-6Mb_reg_439, 9867 candidate probes\n",
      "--- 5742 probes passed check_dic selection.\n",
      "finish in 0.355s, 69 probes kept.\n",
      "-- check region:439 7:46194999-46199999_strand_+_gene_Chr7-6Mb_reg_440, 9918 candidate probes\n",
      "--- 8878 probes passed check_dic selection.\n",
      "finish in 0.471s, 85 probes kept.\n",
      "-- check region:440 7:46199999-46204999_strand_+_gene_Chr7-6Mb_reg_441, 9862 candidate probes\n",
      "--- 8024 probes passed check_dic selection.\n",
      "finish in 0.433s, 84 probes kept.\n",
      "-- check region:441 7:46204999-46209999_strand_+_gene_Chr7-6Mb_reg_442, 9918 candidate probes\n",
      "--- 8336 probes passed check_dic selection.\n",
      "finish in 0.445s, 88 probes kept.\n",
      "-- check region:442 7:46209999-46214999_strand_+_gene_Chr7-6Mb_reg_443, 9918 candidate probes\n",
      "--- 7750 probes passed check_dic selection.\n",
      "finish in 0.424s, 84 probes kept.\n",
      "-- check region:443 7:46214999-46219999_strand_+_gene_Chr7-6Mb_reg_444, 9876 candidate probes\n",
      "--- 8238 probes passed check_dic selection.\n",
      "finish in 0.443s, 88 probes kept.\n",
      "-- check region:444 7:46219999-46224999_strand_+_gene_Chr7-6Mb_reg_445, 9892 candidate probes\n",
      "--- 7230 probes passed check_dic selection.\n",
      "finish in 0.407s, 78 probes kept.\n",
      "-- check region:445 7:46224999-46229999_strand_+_gene_Chr7-6Mb_reg_446, 9918 candidate probes\n",
      "--- 7972 probes passed check_dic selection.\n",
      "finish in 0.436s, 85 probes kept.\n",
      "-- check region:446 7:46229999-46234999_strand_+_gene_Chr7-6Mb_reg_447, 9858 candidate probes\n",
      "--- 6134 probes passed check_dic selection.\n",
      "finish in 0.370s, 73 probes kept.\n",
      "-- check region:447 7:46234999-46239999_strand_+_gene_Chr7-6Mb_reg_448, 9908 candidate probes\n",
      "--- 7074 probes passed check_dic selection.\n",
      "finish in 0.404s, 76 probes kept.\n",
      "-- check region:448 7:46239999-46244999_strand_+_gene_Chr7-6Mb_reg_449, 9894 candidate probes\n",
      "--- 6496 probes passed check_dic selection.\n",
      "finish in 0.383s, 75 probes kept.\n",
      "-- check region:449 7:46244999-46249999_strand_+_gene_Chr7-6Mb_reg_450, 9870 candidate probes\n",
      "--- 7730 probes passed check_dic selection.\n",
      "finish in 0.424s, 83 probes kept.\n",
      "-- check region:450 7:46249999-46254999_strand_+_gene_Chr7-6Mb_reg_451, 9918 candidate probes\n",
      "--- 7658 probes passed check_dic selection.\n",
      "finish in 0.423s, 77 probes kept.\n",
      "-- check region:451 7:46254999-46259999_strand_+_gene_Chr7-6Mb_reg_452, 9842 candidate probes\n",
      "--- 6154 probes passed check_dic selection.\n",
      "finish in 0.367s, 75 probes kept.\n",
      "-- check region:452 7:46259999-46264999_strand_+_gene_Chr7-6Mb_reg_453, 9910 candidate probes\n",
      "--- 7490 probes passed check_dic selection.\n",
      "finish in 0.416s, 87 probes kept.\n",
      "-- check region:453 7:46264999-46269999_strand_+_gene_Chr7-6Mb_reg_454, 9918 candidate probes\n",
      "--- 7702 probes passed check_dic selection.\n",
      "finish in 0.429s, 75 probes kept.\n",
      "-- check region:454 7:46269999-46274999_strand_+_gene_Chr7-6Mb_reg_455, 9808 candidate probes\n",
      "--- 7094 probes passed check_dic selection.\n",
      "finish in 0.402s, 81 probes kept.\n",
      "-- check region:455 7:46274999-46279999_strand_+_gene_Chr7-6Mb_reg_456, 9918 candidate probes\n",
      "--- 6518 probes passed check_dic selection.\n",
      "finish in 0.383s, 75 probes kept.\n",
      "-- check region:456 7:46279999-46284999_strand_+_gene_Chr7-6Mb_reg_457, 9884 candidate probes\n",
      "--- 7070 probes passed check_dic selection.\n",
      "finish in 0.405s, 77 probes kept.\n",
      "-- check region:457 7:46284999-46289999_strand_+_gene_Chr7-6Mb_reg_458, 9918 candidate probes\n",
      "--- 8460 probes passed check_dic selection.\n",
      "finish in 0.452s, 91 probes kept.\n",
      "-- check region:458 7:46289999-46294999_strand_+_gene_Chr7-6Mb_reg_459, 9876 candidate probes\n",
      "--- 5756 probes passed check_dic selection.\n",
      "finish in 0.355s, 72 probes kept.\n",
      "-- check region:459 7:46294999-46299999_strand_+_gene_Chr7-6Mb_reg_460, 9902 candidate probes\n",
      "--- 7346 probes passed check_dic selection.\n",
      "finish in 0.413s, 83 probes kept.\n",
      "-- check region:460 7:46299999-46304999_strand_+_gene_Chr7-6Mb_reg_461, 9918 candidate probes\n",
      "--- 6824 probes passed check_dic selection.\n",
      "finish in 0.393s, 75 probes kept.\n",
      "-- check region:461 7:46304999-46309999_strand_+_gene_Chr7-6Mb_reg_462, 9526 candidate probes\n",
      "--- 7492 probes passed check_dic selection.\n",
      "finish in 0.411s, 77 probes kept.\n",
      "-- check region:462 7:46309999-46314999_strand_+_gene_Chr7-6Mb_reg_463, 9898 candidate probes\n",
      "--- 7102 probes passed check_dic selection.\n",
      "finish in 0.405s, 86 probes kept.\n",
      "-- check region:463 7:46314999-46319999_strand_+_gene_Chr7-6Mb_reg_464, 9918 candidate probes\n",
      "--- 5920 probes passed check_dic selection.\n",
      "finish in 0.360s, 73 probes kept.\n",
      "-- check region:464 7:46319999-46324999_strand_+_gene_Chr7-6Mb_reg_465, 9830 candidate probes\n",
      "--- 5266 probes passed check_dic selection.\n",
      "finish in 0.337s, 67 probes kept.\n",
      "-- check region:465 7:46324999-46329999_strand_+_gene_Chr7-6Mb_reg_466, 9918 candidate probes\n",
      "--- 6664 probes passed check_dic selection.\n",
      "finish in 0.386s, 77 probes kept.\n",
      "-- check region:466 7:46329999-46334999_strand_+_gene_Chr7-6Mb_reg_467, 9794 candidate probes\n",
      "--- 4176 probes passed check_dic selection.\n",
      "finish in 0.299s, 55 probes kept.\n",
      "-- check region:467 7:46334999-46339999_strand_+_gene_Chr7-6Mb_reg_468, 9838 candidate probes\n",
      "--- 5886 probes passed check_dic selection.\n",
      "finish in 0.360s, 75 probes kept.\n",
      "-- check region:468 7:46339999-46344999_strand_+_gene_Chr7-6Mb_reg_469, 9918 candidate probes\n",
      "--- 6878 probes passed check_dic selection.\n",
      "finish in 0.401s, 80 probes kept.\n",
      "-- check region:469 7:46344999-46349999_strand_+_gene_Chr7-6Mb_reg_470, 9768 candidate probes\n",
      "--- 6698 probes passed check_dic selection.\n",
      "finish in 0.387s, 80 probes kept.\n",
      "-- check region:470 7:46349999-46354999_strand_+_gene_Chr7-6Mb_reg_471, 9670 candidate probes\n",
      "--- 7022 probes passed check_dic selection.\n",
      "finish in 0.393s, 79 probes kept.\n",
      "-- check region:471 7:46354999-46359999_strand_+_gene_Chr7-6Mb_reg_472, 9892 candidate probes\n",
      "--- 5936 probes passed check_dic selection.\n",
      "finish in 0.361s, 72 probes kept.\n",
      "-- check region:472 7:46359999-46364999_strand_+_gene_Chr7-6Mb_reg_473, 9896 candidate probes\n",
      "--- 6236 probes passed check_dic selection.\n",
      "finish in 0.373s, 74 probes kept.\n",
      "-- check region:473 7:46364999-46369999_strand_+_gene_Chr7-6Mb_reg_474, 9776 candidate probes\n",
      "--- 7490 probes passed check_dic selection.\n",
      "finish in 0.418s, 85 probes kept.\n",
      "-- check region:474 7:46369999-46374999_strand_+_gene_Chr7-6Mb_reg_475, 9796 candidate probes\n",
      "--- 6820 probes passed check_dic selection.\n",
      "finish in 0.391s, 74 probes kept.\n",
      "-- check region:475 7:46374999-46379999_strand_+_gene_Chr7-6Mb_reg_476, 9918 candidate probes\n",
      "--- 8204 probes passed check_dic selection.\n",
      "finish in 0.447s, 90 probes kept.\n",
      "-- check region:476 7:46379999-46384999_strand_+_gene_Chr7-6Mb_reg_477, 9884 candidate probes\n",
      "--- 5522 probes passed check_dic selection.\n",
      "finish in 0.355s, 68 probes kept.\n",
      "-- check region:477 7:46384999-46389999_strand_+_gene_Chr7-6Mb_reg_478, 9836 candidate probes\n",
      "--- 3066 probes passed check_dic selection.\n",
      "finish in 0.262s, 51 probes kept.\n",
      "-- check region:478 7:46389999-46394999_strand_+_gene_Chr7-6Mb_reg_479, 9886 candidate probes\n",
      "--- 6618 probes passed check_dic selection.\n",
      "finish in 0.386s, 73 probes kept.\n",
      "-- check region:479 7:46394999-46399999_strand_+_gene_Chr7-6Mb_reg_480, 9890 candidate probes\n",
      "--- 8612 probes passed check_dic selection.\n",
      "finish in 0.451s, 88 probes kept.\n",
      "-- check region:480 7:46399999-46404999_strand_+_gene_Chr7-6Mb_reg_481, 9784 candidate probes\n",
      "--- 7450 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.415s, 82 probes kept.\n",
      "-- check region:481 7:46404999-46409999_strand_+_gene_Chr7-6Mb_reg_482, 9918 candidate probes\n",
      "--- 7846 probes passed check_dic selection.\n",
      "finish in 0.432s, 86 probes kept.\n",
      "-- check region:482 7:46409999-46414999_strand_+_gene_Chr7-6Mb_reg_483, 9918 candidate probes\n",
      "--- 7602 probes passed check_dic selection.\n",
      "finish in 0.431s, 86 probes kept.\n",
      "-- check region:483 7:46414999-46419999_strand_+_gene_Chr7-6Mb_reg_484, 9878 candidate probes\n",
      "--- 8056 probes passed check_dic selection.\n",
      "finish in 0.435s, 83 probes kept.\n",
      "-- check region:484 7:46419999-46424999_strand_+_gene_Chr7-6Mb_reg_485, 9868 candidate probes\n",
      "--- 7138 probes passed check_dic selection.\n",
      "finish in 0.400s, 84 probes kept.\n",
      "-- check region:485 7:46424999-46429999_strand_+_gene_Chr7-6Mb_reg_486, 9918 candidate probes\n",
      "--- 8044 probes passed check_dic selection.\n",
      "finish in 0.438s, 85 probes kept.\n",
      "-- check region:486 7:46429999-46434999_strand_+_gene_Chr7-6Mb_reg_487, 9834 candidate probes\n",
      "--- 7936 probes passed check_dic selection.\n",
      "finish in 0.427s, 86 probes kept.\n",
      "-- check region:487 7:46434999-46439999_strand_+_gene_Chr7-6Mb_reg_488, 9894 candidate probes\n",
      "--- 8336 probes passed check_dic selection.\n",
      "finish in 0.446s, 87 probes kept.\n",
      "-- check region:488 7:46439999-46444999_strand_+_gene_Chr7-6Mb_reg_489, 9910 candidate probes\n",
      "--- 7640 probes passed check_dic selection.\n",
      "finish in 0.424s, 82 probes kept.\n",
      "-- check region:489 7:46444999-46449999_strand_+_gene_Chr7-6Mb_reg_490, 9812 candidate probes\n",
      "--- 6244 probes passed check_dic selection.\n",
      "finish in 0.372s, 70 probes kept.\n",
      "-- check region:490 7:46449999-46454999_strand_+_gene_Chr7-6Mb_reg_491, 8792 candidate probes\n",
      "--- 5182 probes passed check_dic selection.\n",
      "finish in 0.319s, 59 probes kept.\n",
      "-- check region:491 7:46454999-46459999_strand_+_gene_Chr7-6Mb_reg_492, 9918 candidate probes\n",
      "--- 6764 probes passed check_dic selection.\n",
      "finish in 0.395s, 77 probes kept.\n",
      "-- check region:492 7:46459999-46464999_strand_+_gene_Chr7-6Mb_reg_493, 9916 candidate probes\n",
      "--- 7408 probes passed check_dic selection.\n",
      "finish in 0.413s, 79 probes kept.\n",
      "-- check region:493 7:46464999-46469999_strand_+_gene_Chr7-6Mb_reg_494, 9820 candidate probes\n",
      "--- 7334 probes passed check_dic selection.\n",
      "finish in 0.405s, 81 probes kept.\n",
      "-- check region:494 7:46469999-46474999_strand_+_gene_Chr7-6Mb_reg_495, 9918 candidate probes\n",
      "--- 6452 probes passed check_dic selection.\n",
      "finish in 0.379s, 75 probes kept.\n",
      "-- check region:495 7:46474999-46479999_strand_+_gene_Chr7-6Mb_reg_496, 9830 candidate probes\n",
      "--- 6574 probes passed check_dic selection.\n",
      "finish in 0.389s, 80 probes kept.\n",
      "-- check region:496 7:46479999-46484999_strand_+_gene_Chr7-6Mb_reg_497, 9736 candidate probes\n",
      "--- 6688 probes passed check_dic selection.\n",
      "finish in 0.387s, 74 probes kept.\n",
      "-- check region:497 7:46484999-46489999_strand_+_gene_Chr7-6Mb_reg_498, 9522 candidate probes\n",
      "--- 5210 probes passed check_dic selection.\n",
      "finish in 0.335s, 59 probes kept.\n",
      "-- check region:498 7:46489999-46494999_strand_+_gene_Chr7-6Mb_reg_499, 9904 candidate probes\n",
      "--- 6740 probes passed check_dic selection.\n",
      "finish in 0.393s, 74 probes kept.\n",
      "-- check region:499 7:46494999-46499999_strand_+_gene_Chr7-6Mb_reg_500, 9918 candidate probes\n",
      "--- 6142 probes passed check_dic selection.\n",
      "finish in 0.370s, 68 probes kept.\n",
      "-- check region:500 7:46499999-46504999_strand_+_gene_Chr7-6Mb_reg_501, 9918 candidate probes\n",
      "--- 6930 probes passed check_dic selection.\n",
      "finish in 0.396s, 83 probes kept.\n",
      "-- check region:501 7:46504999-46509999_strand_+_gene_Chr7-6Mb_reg_502, 9352 candidate probes\n",
      "--- 4770 probes passed check_dic selection.\n",
      "finish in 0.311s, 67 probes kept.\n",
      "-- check region:502 7:46509999-46514999_strand_+_gene_Chr7-6Mb_reg_503, 9818 candidate probes\n",
      "--- 5630 probes passed check_dic selection.\n",
      "finish in 0.350s, 63 probes kept.\n",
      "-- check region:503 7:46514999-46519999_strand_+_gene_Chr7-6Mb_reg_504, 9792 candidate probes\n",
      "--- 6032 probes passed check_dic selection.\n",
      "finish in 0.363s, 73 probes kept.\n",
      "-- check region:504 7:46519999-46524999_strand_+_gene_Chr7-6Mb_reg_505, 9914 candidate probes\n",
      "--- 6562 probes passed check_dic selection.\n",
      "finish in 0.385s, 70 probes kept.\n",
      "-- check region:505 7:46524999-46529999_strand_+_gene_Chr7-6Mb_reg_506, 9918 candidate probes\n",
      "--- 6586 probes passed check_dic selection.\n",
      "finish in 0.384s, 73 probes kept.\n",
      "-- check region:506 7:46529999-46534999_strand_+_gene_Chr7-6Mb_reg_507, 9894 candidate probes\n",
      "--- 7922 probes passed check_dic selection.\n",
      "finish in 0.433s, 84 probes kept.\n",
      "-- check region:507 7:46534999-46539999_strand_+_gene_Chr7-6Mb_reg_508, 9818 candidate probes\n",
      "--- 6696 probes passed check_dic selection.\n",
      "finish in 0.389s, 76 probes kept.\n",
      "-- check region:508 7:46539999-46544999_strand_+_gene_Chr7-6Mb_reg_509, 9888 candidate probes\n",
      "--- 4316 probes passed check_dic selection.\n",
      "finish in 0.307s, 62 probes kept.\n",
      "-- check region:509 7:46544999-46549999_strand_+_gene_Chr7-6Mb_reg_510, 9824 candidate probes\n",
      "--- 6246 probes passed check_dic selection.\n",
      "finish in 0.376s, 76 probes kept.\n",
      "-- check region:510 7:46549999-46554999_strand_+_gene_Chr7-6Mb_reg_511, 9918 candidate probes\n",
      "--- 7602 probes passed check_dic selection.\n",
      "finish in 0.416s, 80 probes kept.\n",
      "-- check region:511 7:46554999-46559999_strand_+_gene_Chr7-6Mb_reg_512, 9720 candidate probes\n",
      "--- 7558 probes passed check_dic selection.\n",
      "finish in 0.415s, 81 probes kept.\n",
      "-- check region:512 7:46559999-46564999_strand_+_gene_Chr7-6Mb_reg_513, 9916 candidate probes\n",
      "--- 6704 probes passed check_dic selection.\n",
      "finish in 0.394s, 76 probes kept.\n",
      "-- check region:513 7:46564999-46569999_strand_+_gene_Chr7-6Mb_reg_514, 9806 candidate probes\n",
      "--- 5276 probes passed check_dic selection.\n",
      "finish in 0.343s, 68 probes kept.\n",
      "-- check region:514 7:46569999-46574999_strand_+_gene_Chr7-6Mb_reg_515, 9906 candidate probes\n",
      "--- 5604 probes passed check_dic selection.\n",
      "finish in 0.352s, 70 probes kept.\n",
      "-- check region:515 7:46574999-46579999_strand_+_gene_Chr7-6Mb_reg_516, 9908 candidate probes\n",
      "--- 5916 probes passed check_dic selection.\n",
      "finish in 0.361s, 71 probes kept.\n",
      "-- check region:516 7:46579999-46584999_strand_+_gene_Chr7-6Mb_reg_517, 9918 candidate probes\n",
      "--- 7042 probes passed check_dic selection.\n",
      "finish in 0.401s, 77 probes kept.\n",
      "-- check region:517 7:46584999-46589999_strand_+_gene_Chr7-6Mb_reg_518, 9918 candidate probes\n",
      "--- 7376 probes passed check_dic selection.\n",
      "finish in 0.408s, 84 probes kept.\n",
      "-- check region:518 7:46589999-46594999_strand_+_gene_Chr7-6Mb_reg_519, 9866 candidate probes\n",
      "--- 6288 probes passed check_dic selection.\n",
      "finish in 0.374s, 69 probes kept.\n",
      "-- check region:519 7:46594999-46599999_strand_+_gene_Chr7-6Mb_reg_520, 9734 candidate probes\n",
      "--- 5692 probes passed check_dic selection.\n",
      "finish in 0.352s, 67 probes kept.\n",
      "-- check region:520 7:46599999-46604999_strand_+_gene_Chr7-6Mb_reg_521, 9918 candidate probes\n",
      "--- 7442 probes passed check_dic selection.\n",
      "finish in 0.412s, 84 probes kept.\n",
      "-- check region:521 7:46604999-46609999_strand_+_gene_Chr7-6Mb_reg_522, 9918 candidate probes\n",
      "--- 8108 probes passed check_dic selection.\n",
      "finish in 0.439s, 86 probes kept.\n",
      "-- check region:522 7:46609999-46614999_strand_+_gene_Chr7-6Mb_reg_523, 9918 candidate probes\n",
      "--- 7906 probes passed check_dic selection.\n",
      "finish in 0.426s, 84 probes kept.\n",
      "-- check region:523 7:46614999-46619999_strand_+_gene_Chr7-6Mb_reg_524, 9786 candidate probes\n",
      "--- 6252 probes passed check_dic selection.\n",
      "finish in 0.373s, 72 probes kept.\n",
      "-- check region:524 7:46619999-46624999_strand_+_gene_Chr7-6Mb_reg_525, 9848 candidate probes\n",
      "--- 5630 probes passed check_dic selection.\n",
      "finish in 0.351s, 63 probes kept.\n",
      "-- check region:525 7:46624999-46629999_strand_+_gene_Chr7-6Mb_reg_526, 8516 candidate probes\n",
      "--- 4932 probes passed check_dic selection.\n",
      "finish in 0.307s, 59 probes kept.\n",
      "-- check region:526 7:46629999-46634999_strand_+_gene_Chr7-6Mb_reg_527, 9856 candidate probes\n",
      "--- 6000 probes passed check_dic selection.\n",
      "finish in 0.364s, 70 probes kept.\n",
      "-- check region:527 7:46634999-46639999_strand_+_gene_Chr7-6Mb_reg_528, 9918 candidate probes\n",
      "--- 7146 probes passed check_dic selection.\n",
      "finish in 0.405s, 82 probes kept.\n",
      "-- check region:528 7:46639999-46644999_strand_+_gene_Chr7-6Mb_reg_529, 9918 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 5496 probes passed check_dic selection.\n",
      "finish in 0.349s, 68 probes kept.\n",
      "-- check region:529 7:46644999-46649999_strand_+_gene_Chr7-6Mb_reg_530, 9340 candidate probes\n",
      "--- 5354 probes passed check_dic selection.\n",
      "finish in 0.334s, 68 probes kept.\n",
      "-- check region:530 7:46649999-46654999_strand_+_gene_Chr7-6Mb_reg_531, 9714 candidate probes\n",
      "--- 5278 probes passed check_dic selection.\n",
      "finish in 0.335s, 65 probes kept.\n",
      "-- check region:531 7:46654999-46659999_strand_+_gene_Chr7-6Mb_reg_532, 9918 candidate probes\n",
      "--- 5956 probes passed check_dic selection.\n",
      "finish in 0.363s, 73 probes kept.\n",
      "-- check region:532 7:46659999-46664999_strand_+_gene_Chr7-6Mb_reg_533, 9634 candidate probes\n",
      "--- 5192 probes passed check_dic selection.\n",
      "finish in 0.331s, 74 probes kept.\n",
      "-- check region:533 7:46664999-46669999_strand_+_gene_Chr7-6Mb_reg_534, 9820 candidate probes\n",
      "--- 6836 probes passed check_dic selection.\n",
      "finish in 0.392s, 79 probes kept.\n",
      "-- check region:534 7:46669999-46674999_strand_+_gene_Chr7-6Mb_reg_535, 9902 candidate probes\n",
      "--- 5706 probes passed check_dic selection.\n",
      "finish in 0.354s, 75 probes kept.\n",
      "-- check region:535 7:46674999-46679999_strand_+_gene_Chr7-6Mb_reg_536, 9862 candidate probes\n",
      "--- 5230 probes passed check_dic selection.\n",
      "finish in 0.336s, 62 probes kept.\n",
      "-- check region:536 7:46679999-46684999_strand_+_gene_Chr7-6Mb_reg_537, 9894 candidate probes\n",
      "--- 6240 probes passed check_dic selection.\n",
      "finish in 0.375s, 70 probes kept.\n",
      "-- check region:537 7:46684999-46689999_strand_+_gene_Chr7-6Mb_reg_538, 9918 candidate probes\n",
      "--- 7360 probes passed check_dic selection.\n",
      "finish in 0.413s, 81 probes kept.\n",
      "-- check region:538 7:46689999-46694999_strand_+_gene_Chr7-6Mb_reg_539, 9154 candidate probes\n",
      "--- 2186 probes passed check_dic selection.\n",
      "finish in 0.221s, 33 probes kept.\n",
      "-- check region:539 7:46694999-46699999_strand_+_gene_Chr7-6Mb_reg_540, 9686 candidate probes\n",
      "--- 3790 probes passed check_dic selection.\n",
      "finish in 0.286s, 49 probes kept.\n",
      "-- check region:540 7:46699999-46704999_strand_+_gene_Chr7-6Mb_reg_541, 9918 candidate probes\n",
      "--- 7394 probes passed check_dic selection.\n",
      "finish in 0.420s, 83 probes kept.\n",
      "-- check region:541 7:46704999-46709999_strand_+_gene_Chr7-6Mb_reg_542, 9706 candidate probes\n",
      "--- 4888 probes passed check_dic selection.\n",
      "finish in 0.322s, 67 probes kept.\n",
      "-- check region:542 7:46709999-46714999_strand_+_gene_Chr7-6Mb_reg_543, 9688 candidate probes\n",
      "--- 6412 probes passed check_dic selection.\n",
      "finish in 0.375s, 72 probes kept.\n",
      "-- check region:543 7:46714999-46719999_strand_+_gene_Chr7-6Mb_reg_544, 9918 candidate probes\n",
      "--- 6008 probes passed check_dic selection.\n",
      "finish in 0.369s, 77 probes kept.\n",
      "-- check region:544 7:46719999-46724999_strand_+_gene_Chr7-6Mb_reg_545, 9722 candidate probes\n",
      "--- 5040 probes passed check_dic selection.\n",
      "finish in 0.333s, 63 probes kept.\n",
      "-- check region:545 7:46724999-46729999_strand_+_gene_Chr7-6Mb_reg_546, 9820 candidate probes\n",
      "--- 4792 probes passed check_dic selection.\n",
      "finish in 0.322s, 66 probes kept.\n",
      "-- check region:546 7:46729999-46734999_strand_+_gene_Chr7-6Mb_reg_547, 9788 candidate probes\n",
      "--- 6100 probes passed check_dic selection.\n",
      "finish in 0.365s, 75 probes kept.\n",
      "-- check region:547 7:46734999-46739999_strand_+_gene_Chr7-6Mb_reg_548, 9614 candidate probes\n",
      "--- 6904 probes passed check_dic selection.\n",
      "finish in 0.393s, 83 probes kept.\n",
      "-- check region:548 7:46739999-46744999_strand_+_gene_Chr7-6Mb_reg_549, 7654 candidate probes\n",
      "--- 4250 probes passed check_dic selection.\n",
      "finish in 0.272s, 62 probes kept.\n",
      "-- check region:549 7:46744999-46749999_strand_+_gene_Chr7-6Mb_reg_550, 9790 candidate probes\n",
      "--- 6568 probes passed check_dic selection.\n",
      "finish in 0.387s, 79 probes kept.\n",
      "-- check region:550 7:46749999-46754999_strand_+_gene_Chr7-6Mb_reg_551, 9918 candidate probes\n",
      "--- 6154 probes passed check_dic selection.\n",
      "finish in 0.375s, 81 probes kept.\n",
      "-- check region:551 7:46754999-46759999_strand_+_gene_Chr7-6Mb_reg_552, 9918 candidate probes\n",
      "--- 6794 probes passed check_dic selection.\n",
      "finish in 0.400s, 80 probes kept.\n",
      "-- check region:552 7:46759999-46764999_strand_+_gene_Chr7-6Mb_reg_553, 9850 candidate probes\n",
      "--- 6038 probes passed check_dic selection.\n",
      "finish in 0.368s, 72 probes kept.\n",
      "-- check region:553 7:46764999-46769999_strand_+_gene_Chr7-6Mb_reg_554, 9918 candidate probes\n",
      "--- 4964 probes passed check_dic selection.\n",
      "finish in 0.328s, 63 probes kept.\n",
      "-- check region:554 7:46769999-46774999_strand_+_gene_Chr7-6Mb_reg_555, 9842 candidate probes\n",
      "--- 5030 probes passed check_dic selection.\n",
      "finish in 0.327s, 68 probes kept.\n",
      "-- check region:555 7:46774999-46779999_strand_+_gene_Chr7-6Mb_reg_556, 9842 candidate probes\n",
      "--- 5438 probes passed check_dic selection.\n",
      "finish in 0.342s, 73 probes kept.\n",
      "-- check region:556 7:46779999-46784999_strand_+_gene_Chr7-6Mb_reg_557, 9892 candidate probes\n",
      "--- 4922 probes passed check_dic selection.\n",
      "finish in 0.326s, 62 probes kept.\n",
      "-- check region:557 7:46784999-46789999_strand_+_gene_Chr7-6Mb_reg_558, 9850 candidate probes\n",
      "--- 4894 probes passed check_dic selection.\n",
      "finish in 0.324s, 65 probes kept.\n",
      "-- check region:558 7:46789999-46794999_strand_+_gene_Chr7-6Mb_reg_559, 9792 candidate probes\n",
      "--- 4188 probes passed check_dic selection.\n",
      "finish in 0.300s, 59 probes kept.\n",
      "-- check region:559 7:46794999-46799999_strand_+_gene_Chr7-6Mb_reg_560, 9914 candidate probes\n",
      "--- 4790 probes passed check_dic selection.\n",
      "finish in 0.325s, 64 probes kept.\n",
      "-- check region:560 7:46799999-46804999_strand_+_gene_Chr7-6Mb_reg_561, 9624 candidate probes\n",
      "--- 3628 probes passed check_dic selection.\n",
      "finish in 0.275s, 56 probes kept.\n",
      "-- check region:561 7:46804999-46809999_strand_+_gene_Chr7-6Mb_reg_562, 9786 candidate probes\n",
      "--- 4314 probes passed check_dic selection.\n",
      "finish in 0.301s, 64 probes kept.\n",
      "-- check region:562 7:46809999-46814999_strand_+_gene_Chr7-6Mb_reg_563, 9802 candidate probes\n",
      "--- 3180 probes passed check_dic selection.\n",
      "finish in 0.264s, 50 probes kept.\n",
      "-- check region:563 7:46814999-46819999_strand_+_gene_Chr7-6Mb_reg_564, 9864 candidate probes\n",
      "--- 4100 probes passed check_dic selection.\n",
      "finish in 0.298s, 57 probes kept.\n",
      "-- check region:564 7:46819999-46824999_strand_+_gene_Chr7-6Mb_reg_565, 9918 candidate probes\n",
      "--- 4952 probes passed check_dic selection.\n",
      "finish in 0.325s, 67 probes kept.\n",
      "-- check region:565 7:46824999-46829999_strand_+_gene_Chr7-6Mb_reg_566, 9460 candidate probes\n",
      "--- 4462 probes passed check_dic selection.\n",
      "finish in 0.303s, 61 probes kept.\n",
      "-- check region:566 7:46829999-46834999_strand_+_gene_Chr7-6Mb_reg_567, 9640 candidate probes\n",
      "--- 5930 probes passed check_dic selection.\n",
      "finish in 0.360s, 75 probes kept.\n",
      "-- check region:567 7:46834999-46839999_strand_+_gene_Chr7-6Mb_reg_568, 9912 candidate probes\n",
      "--- 6364 probes passed check_dic selection.\n",
      "finish in 0.377s, 72 probes kept.\n",
      "-- check region:568 7:46839999-46844999_strand_+_gene_Chr7-6Mb_reg_569, 9916 candidate probes\n",
      "--- 6634 probes passed check_dic selection.\n",
      "finish in 0.386s, 75 probes kept.\n",
      "-- check region:569 7:46844999-46849999_strand_+_gene_Chr7-6Mb_reg_570, 9818 candidate probes\n",
      "--- 6904 probes passed check_dic selection.\n",
      "finish in 0.397s, 74 probes kept.\n",
      "-- check region:570 7:46849999-46854999_strand_+_gene_Chr7-6Mb_reg_571, 9734 candidate probes\n",
      "--- 4482 probes passed check_dic selection.\n",
      "finish in 0.311s, 65 probes kept.\n",
      "-- check region:571 7:46854999-46859999_strand_+_gene_Chr7-6Mb_reg_572, 9606 candidate probes\n",
      "--- 3768 probes passed check_dic selection.\n",
      "finish in 0.285s, 59 probes kept.\n",
      "-- check region:572 7:46859999-46864999_strand_+_gene_Chr7-6Mb_reg_573, 9782 candidate probes\n",
      "--- 4272 probes passed check_dic selection.\n",
      "finish in 0.304s, 54 probes kept.\n",
      "-- check region:573 7:46864999-46869999_strand_+_gene_Chr7-6Mb_reg_574, 9788 candidate probes\n",
      "--- 3036 probes passed check_dic selection.\n",
      "finish in 0.261s, 48 probes kept.\n",
      "-- check region:574 7:46869999-46874999_strand_+_gene_Chr7-6Mb_reg_575, 9536 candidate probes\n",
      "--- 4292 probes passed check_dic selection.\n",
      "finish in 0.300s, 57 probes kept.\n",
      "-- check region:575 7:46874999-46879999_strand_+_gene_Chr7-6Mb_reg_576, 9818 candidate probes\n",
      "--- 5330 probes passed check_dic selection.\n",
      "finish in 0.341s, 64 probes kept.\n",
      "-- check region:576 7:46879999-46884999_strand_+_gene_Chr7-6Mb_reg_577, 9288 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 3612 probes passed check_dic selection.\n",
      "finish in 0.272s, 52 probes kept.\n",
      "-- check region:577 7:46884999-46889999_strand_+_gene_Chr7-6Mb_reg_578, 9704 candidate probes\n",
      "--- 3082 probes passed check_dic selection.\n",
      "finish in 0.260s, 48 probes kept.\n",
      "-- check region:578 7:46889999-46894999_strand_+_gene_Chr7-6Mb_reg_579, 9660 candidate probes\n",
      "--- 4906 probes passed check_dic selection.\n",
      "finish in 0.321s, 65 probes kept.\n",
      "-- check region:579 7:46894999-46899999_strand_+_gene_Chr7-6Mb_reg_580, 9902 candidate probes\n",
      "--- 3934 probes passed check_dic selection.\n",
      "finish in 0.291s, 61 probes kept.\n",
      "-- check region:580 7:46899999-46904999_strand_+_gene_Chr7-6Mb_reg_581, 9878 candidate probes\n",
      "--- 4038 probes passed check_dic selection.\n",
      "finish in 0.296s, 57 probes kept.\n",
      "-- check region:581 7:46904999-46909999_strand_+_gene_Chr7-6Mb_reg_582, 9918 candidate probes\n",
      "--- 5054 probes passed check_dic selection.\n",
      "finish in 0.331s, 67 probes kept.\n",
      "-- check region:582 7:46909999-46914999_strand_+_gene_Chr7-6Mb_reg_583, 9822 candidate probes\n",
      "--- 4450 probes passed check_dic selection.\n",
      "finish in 0.308s, 61 probes kept.\n",
      "-- check region:583 7:46914999-46919999_strand_+_gene_Chr7-6Mb_reg_584, 9918 candidate probes\n",
      "--- 4758 probes passed check_dic selection.\n",
      "finish in 0.320s, 69 probes kept.\n",
      "-- check region:584 7:46919999-46924999_strand_+_gene_Chr7-6Mb_reg_585, 9644 candidate probes\n",
      "--- 3990 probes passed check_dic selection.\n",
      "finish in 0.290s, 56 probes kept.\n",
      "-- check region:585 7:46924999-46929999_strand_+_gene_Chr7-6Mb_reg_586, 9686 candidate probes\n",
      "--- 4004 probes passed check_dic selection.\n",
      "finish in 0.290s, 61 probes kept.\n",
      "-- check region:586 7:46929999-46934999_strand_+_gene_Chr7-6Mb_reg_587, 9536 candidate probes\n",
      "--- 4042 probes passed check_dic selection.\n",
      "finish in 0.291s, 52 probes kept.\n",
      "-- check region:587 7:46934999-46939999_strand_+_gene_Chr7-6Mb_reg_588, 9778 candidate probes\n",
      "--- 4500 probes passed check_dic selection.\n",
      "finish in 0.310s, 62 probes kept.\n",
      "-- check region:588 7:46939999-46944999_strand_+_gene_Chr7-6Mb_reg_589, 9288 candidate probes\n",
      "--- 4472 probes passed check_dic selection.\n",
      "finish in 0.301s, 62 probes kept.\n",
      "-- check region:589 7:46944999-46949999_strand_+_gene_Chr7-6Mb_reg_590, 9738 candidate probes\n",
      "--- 3142 probes passed check_dic selection.\n",
      "finish in 0.260s, 49 probes kept.\n",
      "-- check region:590 7:46949999-46954999_strand_+_gene_Chr7-6Mb_reg_591, 9784 candidate probes\n",
      "--- 6308 probes passed check_dic selection.\n",
      "finish in 0.377s, 81 probes kept.\n",
      "-- check region:591 7:46954999-46959999_strand_+_gene_Chr7-6Mb_reg_592, 9910 candidate probes\n",
      "--- 5578 probes passed check_dic selection.\n",
      "finish in 0.348s, 72 probes kept.\n",
      "-- check region:592 7:46959999-46964999_strand_+_gene_Chr7-6Mb_reg_593, 9886 candidate probes\n",
      "--- 4434 probes passed check_dic selection.\n",
      "finish in 0.312s, 61 probes kept.\n",
      "-- check region:593 7:46964999-46969999_strand_+_gene_Chr7-6Mb_reg_594, 9810 candidate probes\n",
      "--- 5154 probes passed check_dic selection.\n",
      "finish in 0.334s, 66 probes kept.\n",
      "-- check region:594 7:46969999-46974999_strand_+_gene_Chr7-6Mb_reg_595, 9178 candidate probes\n",
      "--- 2004 probes passed check_dic selection.\n",
      "finish in 0.212s, 39 probes kept.\n",
      "-- check region:595 7:46974999-46979999_strand_+_gene_Chr7-6Mb_reg_596, 9826 candidate probes\n",
      "--- 3218 probes passed check_dic selection.\n",
      "finish in 0.266s, 51 probes kept.\n",
      "-- check region:596 7:46979999-46984999_strand_+_gene_Chr7-6Mb_reg_597, 9816 candidate probes\n",
      "--- 3608 probes passed check_dic selection.\n",
      "finish in 0.280s, 55 probes kept.\n",
      "-- check region:597 7:46984999-46989999_strand_+_gene_Chr7-6Mb_reg_598, 9760 candidate probes\n",
      "--- 5762 probes passed check_dic selection.\n",
      "finish in 0.352s, 71 probes kept.\n",
      "-- check region:598 7:46989999-46994999_strand_+_gene_Chr7-6Mb_reg_599, 9918 candidate probes\n",
      "--- 4690 probes passed check_dic selection.\n",
      "finish in 0.316s, 66 probes kept.\n",
      "-- check region:599 7:46994999-46999999_strand_+_gene_Chr7-6Mb_reg_600, 9892 candidate probes\n",
      "--- 6828 probes passed check_dic selection.\n",
      "finish in 0.391s, 79 probes kept.\n",
      "-- check region:600 7:46999999-47004999_strand_+_gene_Chr7-6Mb_reg_601, 9868 candidate probes\n",
      "--- 5820 probes passed check_dic selection.\n",
      "finish in 0.355s, 74 probes kept.\n",
      "-- check region:601 7:47004999-47009999_strand_+_gene_Chr7-6Mb_reg_602, 9868 candidate probes\n",
      "--- 5790 probes passed check_dic selection.\n",
      "finish in 0.356s, 71 probes kept.\n",
      "-- check region:602 7:47009999-47014999_strand_+_gene_Chr7-6Mb_reg_603, 9800 candidate probes\n",
      "--- 3536 probes passed check_dic selection.\n",
      "finish in 0.279s, 53 probes kept.\n",
      "-- check region:603 7:47014999-47019999_strand_+_gene_Chr7-6Mb_reg_604, 9520 candidate probes\n",
      "--- 3826 probes passed check_dic selection.\n",
      "finish in 0.285s, 57 probes kept.\n",
      "-- check region:604 7:47019999-47024999_strand_+_gene_Chr7-6Mb_reg_605, 9794 candidate probes\n",
      "--- 5428 probes passed check_dic selection.\n",
      "finish in 0.342s, 70 probes kept.\n",
      "-- check region:605 7:47024999-47029999_strand_+_gene_Chr7-6Mb_reg_606, 9220 candidate probes\n",
      "--- 3780 probes passed check_dic selection.\n",
      "finish in 0.279s, 55 probes kept.\n",
      "-- check region:606 7:47029999-47034999_strand_+_gene_Chr7-6Mb_reg_607, 9336 candidate probes\n",
      "--- 3558 probes passed check_dic selection.\n",
      "finish in 0.269s, 53 probes kept.\n",
      "-- check region:607 7:47034999-47039999_strand_+_gene_Chr7-6Mb_reg_608, 8964 candidate probes\n",
      "--- 2710 probes passed check_dic selection.\n",
      "finish in 0.236s, 39 probes kept.\n",
      "-- check region:608 7:47039999-47044999_strand_+_gene_Chr7-6Mb_reg_609, 9542 candidate probes\n",
      "--- 5802 probes passed check_dic selection.\n",
      "finish in 0.354s, 73 probes kept.\n",
      "-- check region:609 7:47044999-47049999_strand_+_gene_Chr7-6Mb_reg_610, 9710 candidate probes\n",
      "--- 6106 probes passed check_dic selection.\n",
      "finish in 0.366s, 71 probes kept.\n",
      "-- check region:610 7:47049999-47054999_strand_+_gene_Chr7-6Mb_reg_611, 9914 candidate probes\n",
      "--- 8812 probes passed check_dic selection.\n",
      "finish in 0.464s, 88 probes kept.\n",
      "-- check region:611 7:47054999-47059999_strand_+_gene_Chr7-6Mb_reg_612, 9896 candidate probes\n",
      "--- 7702 probes passed check_dic selection.\n",
      "finish in 0.424s, 84 probes kept.\n",
      "-- check region:612 7:47059999-47064999_strand_+_gene_Chr7-6Mb_reg_613, 9918 candidate probes\n",
      "--- 7880 probes passed check_dic selection.\n",
      "finish in 0.430s, 86 probes kept.\n",
      "-- check region:613 7:47064999-47069999_strand_+_gene_Chr7-6Mb_reg_614, 9792 candidate probes\n",
      "--- 6606 probes passed check_dic selection.\n",
      "finish in 0.384s, 72 probes kept.\n",
      "-- check region:614 7:47069999-47074999_strand_+_gene_Chr7-6Mb_reg_615, 9914 candidate probes\n",
      "--- 7512 probes passed check_dic selection.\n",
      "finish in 0.422s, 86 probes kept.\n",
      "-- check region:615 7:47074999-47079999_strand_+_gene_Chr7-6Mb_reg_616, 9526 candidate probes\n",
      "--- 7218 probes passed check_dic selection.\n",
      "finish in 0.405s, 80 probes kept.\n",
      "-- check region:616 7:47079999-47084999_strand_+_gene_Chr7-6Mb_reg_617, 9918 candidate probes\n",
      "--- 7662 probes passed check_dic selection.\n",
      "finish in 0.423s, 81 probes kept.\n",
      "-- check region:617 7:47084999-47089999_strand_+_gene_Chr7-6Mb_reg_618, 9918 candidate probes\n",
      "--- 7878 probes passed check_dic selection.\n",
      "finish in 0.436s, 83 probes kept.\n",
      "-- check region:618 7:47089999-47094999_strand_+_gene_Chr7-6Mb_reg_619, 9912 candidate probes\n",
      "--- 6406 probes passed check_dic selection.\n",
      "finish in 0.383s, 70 probes kept.\n",
      "-- check region:619 7:47094999-47099999_strand_+_gene_Chr7-6Mb_reg_620, 9918 candidate probes\n",
      "--- 6478 probes passed check_dic selection.\n",
      "finish in 0.381s, 78 probes kept.\n",
      "-- check region:620 7:47099999-47104999_strand_+_gene_Chr7-6Mb_reg_621, 9918 candidate probes\n",
      "--- 6800 probes passed check_dic selection.\n",
      "finish in 0.395s, 76 probes kept.\n",
      "-- check region:621 7:47104999-47109999_strand_+_gene_Chr7-6Mb_reg_622, 9918 candidate probes\n",
      "--- 7284 probes passed check_dic selection.\n",
      "finish in 0.405s, 80 probes kept.\n",
      "-- check region:622 7:47109999-47114999_strand_+_gene_Chr7-6Mb_reg_623, 9868 candidate probes\n",
      "--- 7656 probes passed check_dic selection.\n",
      "finish in 0.428s, 83 probes kept.\n",
      "-- check region:623 7:47114999-47119999_strand_+_gene_Chr7-6Mb_reg_624, 9918 candidate probes\n",
      "--- 7820 probes passed check_dic selection.\n",
      "finish in 0.432s, 86 probes kept.\n",
      "-- check region:624 7:47119999-47124999_strand_+_gene_Chr7-6Mb_reg_625, 9702 candidate probes\n",
      "--- 5500 probes passed check_dic selection.\n",
      "finish in 0.347s, 75 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:625 7:47124999-47129999_strand_+_gene_Chr7-6Mb_reg_626, 9794 candidate probes\n",
      "--- 6142 probes passed check_dic selection.\n",
      "finish in 0.366s, 74 probes kept.\n",
      "-- check region:626 7:47129999-47134999_strand_+_gene_Chr7-6Mb_reg_627, 9918 candidate probes\n",
      "--- 8432 probes passed check_dic selection.\n",
      "finish in 0.448s, 85 probes kept.\n",
      "-- check region:627 7:47134999-47139999_strand_+_gene_Chr7-6Mb_reg_628, 9792 candidate probes\n",
      "--- 4248 probes passed check_dic selection.\n",
      "finish in 0.302s, 54 probes kept.\n",
      "-- check region:628 7:47139999-47144999_strand_+_gene_Chr7-6Mb_reg_629, 9728 candidate probes\n",
      "--- 6312 probes passed check_dic selection.\n",
      "finish in 0.372s, 74 probes kept.\n",
      "-- check region:629 7:47144999-47149999_strand_+_gene_Chr7-6Mb_reg_630, 9700 candidate probes\n",
      "--- 5562 probes passed check_dic selection.\n",
      "finish in 0.347s, 68 probes kept.\n",
      "-- check region:630 7:47149999-47154999_strand_+_gene_Chr7-6Mb_reg_631, 9782 candidate probes\n",
      "--- 5842 probes passed check_dic selection.\n",
      "finish in 0.356s, 74 probes kept.\n",
      "-- check region:631 7:47154999-47159999_strand_+_gene_Chr7-6Mb_reg_632, 9732 candidate probes\n",
      "--- 4162 probes passed check_dic selection.\n",
      "finish in 0.298s, 57 probes kept.\n",
      "-- check region:632 7:47159999-47164999_strand_+_gene_Chr7-6Mb_reg_633, 9918 candidate probes\n",
      "--- 6862 probes passed check_dic selection.\n",
      "finish in 0.392s, 80 probes kept.\n",
      "-- check region:633 7:47164999-47169999_strand_+_gene_Chr7-6Mb_reg_634, 9918 candidate probes\n",
      "--- 6648 probes passed check_dic selection.\n",
      "finish in 0.386s, 79 probes kept.\n",
      "-- check region:634 7:47169999-47174999_strand_+_gene_Chr7-6Mb_reg_635, 9192 candidate probes\n",
      "--- 3238 probes passed check_dic selection.\n",
      "finish in 0.257s, 57 probes kept.\n",
      "-- check region:635 7:47174999-47179999_strand_+_gene_Chr7-6Mb_reg_636, 3758 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.061s, 0 probes kept.\n",
      "-- check region:636 7:47179999-47184999_strand_+_gene_Chr7-6Mb_reg_637, 8334 candidate probes\n",
      "--- 406 probes passed check_dic selection.\n",
      "finish in 0.149s, 14 probes kept.\n",
      "-- check region:637 7:47184999-47189999_strand_+_gene_Chr7-6Mb_reg_638, 4790 candidate probes\n",
      "--- 1352 probes passed check_dic selection.\n",
      "finish in 0.124s, 27 probes kept.\n",
      "-- check region:638 7:47189999-47194999_strand_+_gene_Chr7-6Mb_reg_639, 9446 candidate probes\n",
      "--- 3026 probes passed check_dic selection.\n",
      "finish in 0.254s, 51 probes kept.\n",
      "-- check region:639 7:47194999-47199999_strand_+_gene_Chr7-6Mb_reg_640, 8176 candidate probes\n",
      "--- 1574 probes passed check_dic selection.\n",
      "finish in 0.186s, 24 probes kept.\n",
      "-- check region:640 7:47199999-47204999_strand_+_gene_Chr7-6Mb_reg_641, 8706 candidate probes\n",
      "--- 1492 probes passed check_dic selection.\n",
      "finish in 0.189s, 37 probes kept.\n",
      "-- check region:641 7:47204999-47209999_strand_+_gene_Chr7-6Mb_reg_642, 9300 candidate probes\n",
      "--- 3348 probes passed check_dic selection.\n",
      "finish in 0.265s, 54 probes kept.\n",
      "-- check region:642 7:47209999-47214999_strand_+_gene_Chr7-6Mb_reg_643, 9156 candidate probes\n",
      "--- 3924 probes passed check_dic selection.\n",
      "finish in 0.276s, 59 probes kept.\n",
      "-- check region:643 7:47214999-47219999_strand_+_gene_Chr7-6Mb_reg_644, 8998 candidate probes\n",
      "--- 2754 probes passed check_dic selection.\n",
      "finish in 0.237s, 52 probes kept.\n",
      "-- check region:644 7:47219999-47224999_strand_+_gene_Chr7-6Mb_reg_645, 5932 candidate probes\n",
      "--- 596 probes passed check_dic selection.\n",
      "finish in 0.116s, 11 probes kept.\n",
      "-- check region:645 7:47224999-47229999_strand_+_gene_Chr7-6Mb_reg_646, 6696 candidate probes\n",
      "--- 738 probes passed check_dic selection.\n",
      "finish in 0.132s, 15 probes kept.\n",
      "-- check region:646 7:47229999-47234999_strand_+_gene_Chr7-6Mb_reg_647, 9576 candidate probes\n",
      "--- 1048 probes passed check_dic selection.\n",
      "finish in 0.187s, 27 probes kept.\n",
      "-- check region:647 7:47234999-47239999_strand_+_gene_Chr7-6Mb_reg_648, 9450 candidate probes\n",
      "--- 2696 probes passed check_dic selection.\n",
      "finish in 0.242s, 50 probes kept.\n",
      "-- check region:648 7:47239999-47244999_strand_+_gene_Chr7-6Mb_reg_649, 7616 candidate probes\n",
      "--- 1142 probes passed check_dic selection.\n",
      "finish in 0.162s, 26 probes kept.\n",
      "-- check region:649 7:47244999-47249999_strand_+_gene_Chr7-6Mb_reg_650, 9122 candidate probes\n",
      "--- 2348 probes passed check_dic selection.\n",
      "finish in 0.225s, 40 probes kept.\n",
      "-- check region:650 7:47249999-47254999_strand_+_gene_Chr7-6Mb_reg_651, 9228 candidate probes\n",
      "--- 2624 probes passed check_dic selection.\n",
      "finish in 0.238s, 57 probes kept.\n",
      "-- check region:651 7:47254999-47259999_strand_+_gene_Chr7-6Mb_reg_652, 8326 candidate probes\n",
      "--- 894 probes passed check_dic selection.\n",
      "finish in 0.165s, 17 probes kept.\n",
      "-- check region:652 7:47259999-47264999_strand_+_gene_Chr7-6Mb_reg_653, 472 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.008s, 0 probes kept.\n",
      "-- check region:653 7:47264999-47269999_strand_+_gene_Chr7-6Mb_reg_654, 7780 candidate probes\n",
      "--- 698 probes passed check_dic selection.\n",
      "finish in 0.149s, 17 probes kept.\n",
      "-- check region:654 7:47269999-47274999_strand_+_gene_Chr7-6Mb_reg_655, 8510 candidate probes\n",
      "--- 1894 probes passed check_dic selection.\n",
      "finish in 0.201s, 35 probes kept.\n",
      "-- check region:655 7:47274999-47279999_strand_+_gene_Chr7-6Mb_reg_656, 5336 candidate probes\n",
      "--- 148 probes passed check_dic selection.\n",
      "finish in 0.092s, 3 probes kept.\n",
      "-- check region:656 7:47279999-47284999_strand_+_gene_Chr7-6Mb_reg_657, 6708 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.109s, 0 probes kept.\n",
      "-- check region:657 7:47284999-47289999_strand_+_gene_Chr7-6Mb_reg_658, 9034 candidate probes\n",
      "--- 138 probes passed check_dic selection.\n",
      "finish in 0.152s, 8 probes kept.\n",
      "-- check region:658 7:47289999-47294999_strand_+_gene_Chr7-6Mb_reg_659, 9848 candidate probes\n",
      "--- 448 probes passed check_dic selection.\n",
      "finish in 0.176s, 17 probes kept.\n",
      "-- check region:659 7:47294999-47299999_strand_+_gene_Chr7-6Mb_reg_660, 9144 candidate probes\n",
      "--- 1618 probes passed check_dic selection.\n",
      "finish in 0.201s, 35 probes kept.\n",
      "-- check region:660 7:47299999-47304999_strand_+_gene_Chr7-6Mb_reg_661, 9600 candidate probes\n",
      "--- 2118 probes passed check_dic selection.\n",
      "finish in 0.222s, 42 probes kept.\n",
      "-- check region:661 7:47304999-47309999_strand_+_gene_Chr7-6Mb_reg_662, 9484 candidate probes\n",
      "--- 2764 probes passed check_dic selection.\n",
      "finish in 0.245s, 58 probes kept.\n",
      "-- check region:662 7:47309999-47314999_strand_+_gene_Chr7-6Mb_reg_663, 9484 candidate probes\n",
      "--- 2676 probes passed check_dic selection.\n",
      "finish in 0.245s, 50 probes kept.\n",
      "-- check region:663 7:47314999-47319999_strand_+_gene_Chr7-6Mb_reg_664, 9374 candidate probes\n",
      "--- 2354 probes passed check_dic selection.\n",
      "finish in 0.230s, 41 probes kept.\n",
      "-- check region:664 7:47319999-47324999_strand_+_gene_Chr7-6Mb_reg_665, 9612 candidate probes\n",
      "--- 1502 probes passed check_dic selection.\n",
      "finish in 0.209s, 30 probes kept.\n",
      "-- check region:665 7:47324999-47329999_strand_+_gene_Chr7-6Mb_reg_666, 8204 candidate probes\n",
      "--- 460 probes passed check_dic selection.\n",
      "finish in 0.148s, 9 probes kept.\n",
      "-- check region:666 7:47329999-47334999_strand_+_gene_Chr7-6Mb_reg_667, 9730 candidate probes\n",
      "--- 442 probes passed check_dic selection.\n",
      "finish in 0.172s, 15 probes kept.\n",
      "-- check region:667 7:47334999-47339999_strand_+_gene_Chr7-6Mb_reg_668, 9572 candidate probes\n",
      "--- 2916 probes passed check_dic selection.\n",
      "finish in 0.254s, 58 probes kept.\n",
      "-- check region:668 7:47339999-47344999_strand_+_gene_Chr7-6Mb_reg_669, 9268 candidate probes\n",
      "--- 3342 probes passed check_dic selection.\n",
      "finish in 0.262s, 56 probes kept.\n",
      "-- check region:669 7:47344999-47349999_strand_+_gene_Chr7-6Mb_reg_670, 9792 candidate probes\n",
      "--- 5002 probes passed check_dic selection.\n",
      "finish in 0.328s, 68 probes kept.\n",
      "-- check region:670 7:47349999-47354999_strand_+_gene_Chr7-6Mb_reg_671, 9350 candidate probes\n",
      "--- 2836 probes passed check_dic selection.\n",
      "finish in 0.246s, 54 probes kept.\n",
      "-- check region:671 7:47354999-47359999_strand_+_gene_Chr7-6Mb_reg_672, 9300 candidate probes\n",
      "--- 3710 probes passed check_dic selection.\n",
      "finish in 0.272s, 51 probes kept.\n",
      "-- check region:672 7:47359999-47364999_strand_+_gene_Chr7-6Mb_reg_673, 8406 candidate probes\n",
      "--- 130 probes passed check_dic selection.\n",
      "finish in 0.139s, 3 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:673 7:47364999-47369999_strand_+_gene_Chr7-6Mb_reg_674, 9492 candidate probes\n",
      "--- 1262 probes passed check_dic selection.\n",
      "finish in 0.199s, 32 probes kept.\n",
      "-- check region:674 7:47369999-47374999_strand_+_gene_Chr7-6Mb_reg_675, 9880 candidate probes\n",
      "--- 3354 probes passed check_dic selection.\n",
      "finish in 0.269s, 48 probes kept.\n",
      "-- check region:675 7:47374999-47379999_strand_+_gene_Chr7-6Mb_reg_676, 9234 candidate probes\n",
      "--- 1882 probes passed check_dic selection.\n",
      "finish in 0.212s, 38 probes kept.\n",
      "-- check region:676 7:47379999-47384999_strand_+_gene_Chr7-6Mb_reg_677, 8786 candidate probes\n",
      "--- 2152 probes passed check_dic selection.\n",
      "finish in 0.213s, 36 probes kept.\n",
      "-- check region:677 7:47384999-47389999_strand_+_gene_Chr7-6Mb_reg_678, 6966 candidate probes\n",
      "--- 124 probes passed check_dic selection.\n",
      "finish in 0.116s, 4 probes kept.\n",
      "-- check region:678 7:47389999-47394999_strand_+_gene_Chr7-6Mb_reg_679, 9080 candidate probes\n",
      "--- 3082 probes passed check_dic selection.\n",
      "finish in 0.249s, 51 probes kept.\n",
      "-- check region:679 7:47394999-47399999_strand_+_gene_Chr7-6Mb_reg_680, 9812 candidate probes\n",
      "--- 2040 probes passed check_dic selection.\n",
      "finish in 0.226s, 37 probes kept.\n",
      "-- check region:680 7:47399999-47404999_strand_+_gene_Chr7-6Mb_reg_681, 8666 candidate probes\n",
      "--- 1726 probes passed check_dic selection.\n",
      "finish in 0.197s, 33 probes kept.\n",
      "-- check region:681 7:47404999-47409999_strand_+_gene_Chr7-6Mb_reg_682, 8742 candidate probes\n",
      "--- 2276 probes passed check_dic selection.\n",
      "finish in 0.217s, 48 probes kept.\n",
      "-- check region:682 7:47409999-47414999_strand_+_gene_Chr7-6Mb_reg_683, 6638 candidate probes\n",
      "--- 1294 probes passed check_dic selection.\n",
      "finish in 0.153s, 27 probes kept.\n",
      "-- check region:683 7:47414999-47419999_strand_+_gene_Chr7-6Mb_reg_684, 5226 candidate probes\n",
      "--- 224 probes passed check_dic selection.\n",
      "finish in 0.092s, 7 probes kept.\n",
      "-- check region:684 7:47419999-47424999_strand_+_gene_Chr7-6Mb_reg_685, 1624 candidate probes\n",
      "--- 220 probes passed check_dic selection.\n",
      "finish in 0.035s, 7 probes kept.\n",
      "-- check region:685 7:47424999-47429999_strand_+_gene_Chr7-6Mb_reg_686, 1184 candidate probes\n",
      "--- 354 probes passed check_dic selection.\n",
      "finish in 0.032s, 7 probes kept.\n",
      "-- check region:686 7:47429999-47434999_strand_+_gene_Chr7-6Mb_reg_687, 1824 candidate probes\n",
      "--- 26 probes passed check_dic selection.\n",
      "finish in 0.031s, 1 probes kept.\n",
      "-- check region:687 7:47434999-47439999_strand_+_gene_Chr7-6Mb_reg_688, 2170 candidate probes\n",
      "--- 432 probes passed check_dic selection.\n",
      "finish in 0.050s, 9 probes kept.\n",
      "-- check region:688 7:47439999-47444999_strand_+_gene_Chr7-6Mb_reg_689, 2730 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.044s, 0 probes kept.\n",
      "-- check region:689 7:47444999-47449999_strand_+_gene_Chr7-6Mb_reg_690, 1164 candidate probes\n",
      "--- 132 probes passed check_dic selection.\n",
      "finish in 0.024s, 5 probes kept.\n",
      "-- check region:690 7:47449999-47454999_strand_+_gene_Chr7-6Mb_reg_691, 3382 candidate probes\n",
      "--- 578 probes passed check_dic selection.\n",
      "finish in 0.074s, 16 probes kept.\n",
      "-- check region:691 7:47454999-47459999_strand_+_gene_Chr7-6Mb_reg_692, 9626 candidate probes\n",
      "--- 1368 probes passed check_dic selection.\n",
      "finish in 0.197s, 32 probes kept.\n",
      "-- check region:692 7:47459999-47464999_strand_+_gene_Chr7-6Mb_reg_693, 9876 candidate probes\n",
      "--- 1568 probes passed check_dic selection.\n",
      "finish in 0.211s, 39 probes kept.\n",
      "-- check region:693 7:47464999-47469999_strand_+_gene_Chr7-6Mb_reg_694, 9254 candidate probes\n",
      "--- 468 probes passed check_dic selection.\n",
      "finish in 0.167s, 16 probes kept.\n",
      "-- check region:694 7:47469999-47474999_strand_+_gene_Chr7-6Mb_reg_695, 9490 candidate probes\n",
      "--- 672 probes passed check_dic selection.\n",
      "finish in 0.176s, 19 probes kept.\n",
      "-- check region:695 7:47474999-47479999_strand_+_gene_Chr7-6Mb_reg_696, 6922 candidate probes\n",
      "--- 642 probes passed check_dic selection.\n",
      "finish in 0.133s, 15 probes kept.\n",
      "-- check region:696 7:47479999-47484999_strand_+_gene_Chr7-6Mb_reg_697, 7884 candidate probes\n",
      "--- 548 probes passed check_dic selection.\n",
      "finish in 0.144s, 17 probes kept.\n",
      "-- check region:697 7:47484999-47489999_strand_+_gene_Chr7-6Mb_reg_698, 9446 candidate probes\n",
      "--- 998 probes passed check_dic selection.\n",
      "finish in 0.185s, 32 probes kept.\n",
      "-- check region:698 7:47489999-47494999_strand_+_gene_Chr7-6Mb_reg_699, 9728 candidate probes\n",
      "--- 1210 probes passed check_dic selection.\n",
      "finish in 0.196s, 28 probes kept.\n",
      "-- check region:699 7:47494999-47499999_strand_+_gene_Chr7-6Mb_reg_700, 9566 candidate probes\n",
      "--- 2062 probes passed check_dic selection.\n",
      "finish in 0.219s, 37 probes kept.\n",
      "-- check region:700 7:47499999-47504999_strand_+_gene_Chr7-6Mb_reg_701, 9846 candidate probes\n",
      "--- 1380 probes passed check_dic selection.\n",
      "finish in 0.204s, 41 probes kept.\n",
      "-- check region:701 7:47504999-47509999_strand_+_gene_Chr7-6Mb_reg_702, 4434 candidate probes\n",
      "--- 566 probes passed check_dic selection.\n",
      "finish in 0.092s, 9 probes kept.\n",
      "-- check region:702 7:47509999-47514999_strand_+_gene_Chr7-6Mb_reg_703, 7232 candidate probes\n",
      "--- 1452 probes passed check_dic selection.\n",
      "finish in 0.165s, 25 probes kept.\n",
      "-- check region:703 7:47514999-47519999_strand_+_gene_Chr7-6Mb_reg_704, 8640 candidate probes\n",
      "--- 2742 probes passed check_dic selection.\n",
      "finish in 0.232s, 41 probes kept.\n",
      "-- check region:704 7:47519999-47524999_strand_+_gene_Chr7-6Mb_reg_705, 9204 candidate probes\n",
      "--- 1612 probes passed check_dic selection.\n",
      "finish in 0.203s, 31 probes kept.\n",
      "-- check region:705 7:47524999-47529999_strand_+_gene_Chr7-6Mb_reg_706, 9654 candidate probes\n",
      "--- 2500 probes passed check_dic selection.\n",
      "finish in 0.239s, 46 probes kept.\n",
      "-- check region:706 7:47529999-47534999_strand_+_gene_Chr7-6Mb_reg_707, 9266 candidate probes\n",
      "--- 3008 probes passed check_dic selection.\n",
      "finish in 0.250s, 55 probes kept.\n",
      "-- check region:707 7:47534999-47539999_strand_+_gene_Chr7-6Mb_reg_708, 9056 candidate probes\n",
      "--- 1526 probes passed check_dic selection.\n",
      "finish in 0.195s, 31 probes kept.\n",
      "-- check region:708 7:47539999-47544999_strand_+_gene_Chr7-6Mb_reg_709, 9282 candidate probes\n",
      "--- 294 probes passed check_dic selection.\n",
      "finish in 0.158s, 5 probes kept.\n",
      "-- check region:709 7:47544999-47549999_strand_+_gene_Chr7-6Mb_reg_710, 9394 candidate probes\n",
      "--- 1150 probes passed check_dic selection.\n",
      "finish in 0.189s, 29 probes kept.\n",
      "-- check region:710 7:47549999-47554999_strand_+_gene_Chr7-6Mb_reg_711, 9182 candidate probes\n",
      "--- 704 probes passed check_dic selection.\n",
      "finish in 0.170s, 24 probes kept.\n",
      "-- check region:711 7:47554999-47559999_strand_+_gene_Chr7-6Mb_reg_712, 4432 candidate probes\n",
      "--- 1348 probes passed check_dic selection.\n",
      "finish in 0.117s, 27 probes kept.\n",
      "-- check region:712 7:47559999-47564999_strand_+_gene_Chr7-6Mb_reg_713, 7020 candidate probes\n",
      "--- 2392 probes passed check_dic selection.\n",
      "finish in 0.194s, 46 probes kept.\n",
      "-- check region:713 7:47564999-47569999_strand_+_gene_Chr7-6Mb_reg_714, 3386 candidate probes\n",
      "--- 818 probes passed check_dic selection.\n",
      "finish in 0.084s, 18 probes kept.\n",
      "-- check region:714 7:47569999-47574999_strand_+_gene_Chr7-6Mb_reg_715, 516 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.009s, 0 probes kept.\n",
      "-- check region:715 7:47574999-47579999_strand_+_gene_Chr7-6Mb_reg_716, 726 candidate probes\n",
      "--- 86 probes passed check_dic selection.\n",
      "finish in 0.016s, 3 probes kept.\n",
      "-- check region:716 7:47579999-47584999_strand_+_gene_Chr7-6Mb_reg_717, 5184 candidate probes\n",
      "--- 1056 probes passed check_dic selection.\n",
      "finish in 0.117s, 23 probes kept.\n",
      "-- check region:717 7:47584999-47589999_strand_+_gene_Chr7-6Mb_reg_718, 9244 candidate probes\n",
      "--- 920 probes passed check_dic selection.\n",
      "finish in 0.178s, 24 probes kept.\n",
      "-- check region:718 7:47589999-47594999_strand_+_gene_Chr7-6Mb_reg_719, 8972 candidate probes\n",
      "--- 1016 probes passed check_dic selection.\n",
      "finish in 0.179s, 29 probes kept.\n",
      "-- check region:719 7:47594999-47599999_strand_+_gene_Chr7-6Mb_reg_720, 8696 candidate probes\n",
      "--- 1534 probes passed check_dic selection.\n",
      "finish in 0.189s, 41 probes kept.\n",
      "-- check region:720 7:47599999-47604999_strand_+_gene_Chr7-6Mb_reg_721, 9156 candidate probes\n",
      "--- 3442 probes passed check_dic selection.\n",
      "finish in 0.265s, 55 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:721 7:47604999-47609999_strand_+_gene_Chr7-6Mb_reg_722, 9658 candidate probes\n",
      "--- 2846 probes passed check_dic selection.\n",
      "finish in 0.252s, 52 probes kept.\n",
      "-- check region:722 7:47609999-47614999_strand_+_gene_Chr7-6Mb_reg_723, 9350 candidate probes\n",
      "--- 1242 probes passed check_dic selection.\n",
      "finish in 0.192s, 32 probes kept.\n",
      "-- check region:723 7:47614999-47619999_strand_+_gene_Chr7-6Mb_reg_724, 9020 candidate probes\n",
      "--- 86 probes passed check_dic selection.\n",
      "finish in 0.151s, 6 probes kept.\n",
      "-- check region:724 7:47619999-47624999_strand_+_gene_Chr7-6Mb_reg_725, 8982 candidate probes\n",
      "--- 528 probes passed check_dic selection.\n",
      "finish in 0.166s, 13 probes kept.\n",
      "-- check region:725 7:47624999-47629999_strand_+_gene_Chr7-6Mb_reg_726, 8900 candidate probes\n",
      "--- 1526 probes passed check_dic selection.\n",
      "finish in 0.192s, 39 probes kept.\n",
      "-- check region:726 7:47629999-47634999_strand_+_gene_Chr7-6Mb_reg_727, 8714 candidate probes\n",
      "--- 1662 probes passed check_dic selection.\n",
      "finish in 0.195s, 40 probes kept.\n",
      "-- check region:727 7:47634999-47639999_strand_+_gene_Chr7-6Mb_reg_728, 6650 candidate probes\n",
      "--- 1046 probes passed check_dic selection.\n",
      "finish in 0.142s, 22 probes kept.\n",
      "-- check region:728 7:47639999-47644999_strand_+_gene_Chr7-6Mb_reg_729, 4518 candidate probes\n",
      "--- 314 probes passed check_dic selection.\n",
      "finish in 0.083s, 13 probes kept.\n",
      "-- check region:729 7:47644999-47649999_strand_+_gene_Chr7-6Mb_reg_730, 4320 candidate probes\n",
      "--- 496 probes passed check_dic selection.\n",
      "finish in 0.084s, 14 probes kept.\n",
      "-- check region:730 7:47649999-47654999_strand_+_gene_Chr7-6Mb_reg_731, 9344 candidate probes\n",
      "--- 2080 probes passed check_dic selection.\n",
      "finish in 0.221s, 51 probes kept.\n",
      "-- check region:731 7:47654999-47659999_strand_+_gene_Chr7-6Mb_reg_732, 9410 candidate probes\n",
      "--- 978 probes passed check_dic selection.\n",
      "finish in 0.181s, 23 probes kept.\n",
      "-- check region:732 7:47659999-47664999_strand_+_gene_Chr7-6Mb_reg_733, 9520 candidate probes\n",
      "--- 2928 probes passed check_dic selection.\n",
      "finish in 0.252s, 57 probes kept.\n",
      "-- check region:733 7:47664999-47669999_strand_+_gene_Chr7-6Mb_reg_734, 9616 candidate probes\n",
      "--- 996 probes passed check_dic selection.\n",
      "finish in 0.186s, 25 probes kept.\n",
      "-- check region:734 7:47669999-47674999_strand_+_gene_Chr7-6Mb_reg_735, 9236 candidate probes\n",
      "--- 2380 probes passed check_dic selection.\n",
      "finish in 0.227s, 46 probes kept.\n",
      "-- check region:735 7:47674999-47679999_strand_+_gene_Chr7-6Mb_reg_736, 8974 candidate probes\n",
      "--- 478 probes passed check_dic selection.\n",
      "finish in 0.159s, 10 probes kept.\n",
      "-- check region:736 7:47679999-47684999_strand_+_gene_Chr7-6Mb_reg_737, 8426 candidate probes\n",
      "--- 40 probes passed check_dic selection.\n",
      "finish in 0.140s, 2 probes kept.\n",
      "-- check region:737 7:47684999-47689999_strand_+_gene_Chr7-6Mb_reg_738, 9678 candidate probes\n",
      "--- 1938 probes passed check_dic selection.\n",
      "finish in 0.223s, 38 probes kept.\n",
      "-- check region:738 7:47689999-47694999_strand_+_gene_Chr7-6Mb_reg_739, 9522 candidate probes\n",
      "--- 1872 probes passed check_dic selection.\n",
      "finish in 0.214s, 41 probes kept.\n",
      "-- check region:739 7:47694999-47699999_strand_+_gene_Chr7-6Mb_reg_740, 9680 candidate probes\n",
      "--- 3902 probes passed check_dic selection.\n",
      "finish in 0.288s, 61 probes kept.\n",
      "-- check region:740 7:47699999-47704999_strand_+_gene_Chr7-6Mb_reg_741, 9862 candidate probes\n",
      "--- 2984 probes passed check_dic selection.\n",
      "finish in 0.259s, 55 probes kept.\n",
      "-- check region:741 7:47704999-47709999_strand_+_gene_Chr7-6Mb_reg_742, 9764 candidate probes\n",
      "--- 2030 probes passed check_dic selection.\n",
      "finish in 0.223s, 37 probes kept.\n",
      "-- check region:742 7:47709999-47714999_strand_+_gene_Chr7-6Mb_reg_743, 9712 candidate probes\n",
      "--- 3490 probes passed check_dic selection.\n",
      "finish in 0.277s, 60 probes kept.\n",
      "-- check region:743 7:47714999-47719999_strand_+_gene_Chr7-6Mb_reg_744, 9560 candidate probes\n",
      "--- 2144 probes passed check_dic selection.\n",
      "finish in 0.225s, 44 probes kept.\n",
      "-- check region:744 7:47719999-47724999_strand_+_gene_Chr7-6Mb_reg_745, 9386 candidate probes\n",
      "--- 1364 probes passed check_dic selection.\n",
      "finish in 0.195s, 32 probes kept.\n",
      "-- check region:745 7:47724999-47729999_strand_+_gene_Chr7-6Mb_reg_746, 9360 candidate probes\n",
      "--- 1622 probes passed check_dic selection.\n",
      "finish in 0.201s, 35 probes kept.\n",
      "-- check region:746 7:47729999-47734999_strand_+_gene_Chr7-6Mb_reg_747, 9006 candidate probes\n",
      "--- 1066 probes passed check_dic selection.\n",
      "finish in 0.179s, 28 probes kept.\n",
      "-- check region:747 7:47734999-47739999_strand_+_gene_Chr7-6Mb_reg_748, 9272 candidate probes\n",
      "--- 2662 probes passed check_dic selection.\n",
      "finish in 0.240s, 54 probes kept.\n",
      "-- check region:748 7:47739999-47744999_strand_+_gene_Chr7-6Mb_reg_749, 9516 candidate probes\n",
      "--- 3050 probes passed check_dic selection.\n",
      "finish in 0.254s, 52 probes kept.\n",
      "-- check region:749 7:47744999-47749999_strand_+_gene_Chr7-6Mb_reg_750, 9538 candidate probes\n",
      "--- 3070 probes passed check_dic selection.\n",
      "finish in 0.254s, 52 probes kept.\n",
      "-- check region:750 7:47749999-47754999_strand_+_gene_Chr7-6Mb_reg_751, 9874 candidate probes\n",
      "--- 2036 probes passed check_dic selection.\n",
      "finish in 0.228s, 45 probes kept.\n",
      "-- check region:751 7:47754999-47759999_strand_+_gene_Chr7-6Mb_reg_752, 9846 candidate probes\n",
      "--- 3892 probes passed check_dic selection.\n",
      "finish in 0.288s, 60 probes kept.\n",
      "-- check region:752 7:47759999-47764999_strand_+_gene_Chr7-6Mb_reg_753, 9400 candidate probes\n",
      "--- 2860 probes passed check_dic selection.\n",
      "finish in 0.247s, 48 probes kept.\n",
      "-- check region:753 7:47764999-47769999_strand_+_gene_Chr7-6Mb_reg_754, 9904 candidate probes\n",
      "--- 752 probes passed check_dic selection.\n",
      "finish in 0.188s, 25 probes kept.\n",
      "-- check region:754 7:47769999-47774999_strand_+_gene_Chr7-6Mb_reg_755, 9624 candidate probes\n",
      "--- 2728 probes passed check_dic selection.\n",
      "finish in 0.243s, 48 probes kept.\n",
      "-- check region:755 7:47774999-47779999_strand_+_gene_Chr7-6Mb_reg_756, 9904 candidate probes\n",
      "--- 2716 probes passed check_dic selection.\n",
      "finish in 0.253s, 52 probes kept.\n",
      "-- check region:756 7:47779999-47784999_strand_+_gene_Chr7-6Mb_reg_757, 9838 candidate probes\n",
      "--- 3364 probes passed check_dic selection.\n",
      "finish in 0.272s, 50 probes kept.\n",
      "-- check region:757 7:47784999-47789999_strand_+_gene_Chr7-6Mb_reg_758, 9700 candidate probes\n",
      "--- 2650 probes passed check_dic selection.\n",
      "finish in 0.245s, 56 probes kept.\n",
      "-- check region:758 7:47789999-47794999_strand_+_gene_Chr7-6Mb_reg_759, 8798 candidate probes\n",
      "--- 286 probes passed check_dic selection.\n",
      "finish in 0.154s, 13 probes kept.\n",
      "-- check region:759 7:47794999-47799999_strand_+_gene_Chr7-6Mb_reg_760, 9102 candidate probes\n",
      "--- 1294 probes passed check_dic selection.\n",
      "finish in 0.188s, 27 probes kept.\n",
      "-- check region:760 7:47799999-47804999_strand_+_gene_Chr7-6Mb_reg_761, 8516 candidate probes\n",
      "--- 174 probes passed check_dic selection.\n",
      "finish in 0.143s, 5 probes kept.\n",
      "-- check region:761 7:47804999-47809999_strand_+_gene_Chr7-6Mb_reg_762, 9356 candidate probes\n",
      "--- 1246 probes passed check_dic selection.\n",
      "finish in 0.194s, 23 probes kept.\n",
      "-- check region:762 7:47809999-47814999_strand_+_gene_Chr7-6Mb_reg_763, 9204 candidate probes\n",
      "--- 2022 probes passed check_dic selection.\n",
      "finish in 0.215s, 40 probes kept.\n",
      "-- check region:763 7:47814999-47819999_strand_+_gene_Chr7-6Mb_reg_764, 9654 candidate probes\n",
      "--- 1564 probes passed check_dic selection.\n",
      "finish in 0.206s, 42 probes kept.\n",
      "-- check region:764 7:47819999-47824999_strand_+_gene_Chr7-6Mb_reg_765, 9522 candidate probes\n",
      "--- 1192 probes passed check_dic selection.\n",
      "finish in 0.192s, 24 probes kept.\n",
      "-- check region:765 7:47824999-47829999_strand_+_gene_Chr7-6Mb_reg_766, 9596 candidate probes\n",
      "--- 2714 probes passed check_dic selection.\n",
      "finish in 0.246s, 50 probes kept.\n",
      "-- check region:766 7:47829999-47834999_strand_+_gene_Chr7-6Mb_reg_767, 9440 candidate probes\n",
      "--- 2840 probes passed check_dic selection.\n",
      "finish in 0.245s, 49 probes kept.\n",
      "-- check region:767 7:47834999-47839999_strand_+_gene_Chr7-6Mb_reg_768, 9728 candidate probes\n",
      "--- 1958 probes passed check_dic selection.\n",
      "finish in 0.219s, 42 probes kept.\n",
      "-- check region:768 7:47839999-47844999_strand_+_gene_Chr7-6Mb_reg_769, 9794 candidate probes\n",
      "--- 1070 probes passed check_dic selection.\n",
      "finish in 0.193s, 24 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:769 7:47844999-47849999_strand_+_gene_Chr7-6Mb_reg_770, 9882 candidate probes\n",
      "--- 794 probes passed check_dic selection.\n",
      "finish in 0.184s, 21 probes kept.\n",
      "-- check region:770 7:47849999-47854999_strand_+_gene_Chr7-6Mb_reg_771, 9918 candidate probes\n",
      "--- 866 probes passed check_dic selection.\n",
      "finish in 0.182s, 28 probes kept.\n",
      "-- check region:771 7:47854999-47859999_strand_+_gene_Chr7-6Mb_reg_772, 9842 candidate probes\n",
      "--- 2008 probes passed check_dic selection.\n",
      "finish in 0.223s, 47 probes kept.\n",
      "-- check region:772 7:47859999-47864999_strand_+_gene_Chr7-6Mb_reg_773, 9700 candidate probes\n",
      "--- 1310 probes passed check_dic selection.\n",
      "finish in 0.199s, 29 probes kept.\n",
      "-- check region:773 7:47864999-47869999_strand_+_gene_Chr7-6Mb_reg_774, 9016 candidate probes\n",
      "--- 146 probes passed check_dic selection.\n",
      "finish in 0.149s, 6 probes kept.\n",
      "-- check region:774 7:47869999-47874999_strand_+_gene_Chr7-6Mb_reg_775, 9340 candidate probes\n",
      "--- 1676 probes passed check_dic selection.\n",
      "finish in 0.210s, 36 probes kept.\n",
      "-- check region:775 7:47874999-47879999_strand_+_gene_Chr7-6Mb_reg_776, 9884 candidate probes\n",
      "--- 3634 probes passed check_dic selection.\n",
      "finish in 0.281s, 67 probes kept.\n",
      "-- check region:776 7:47879999-47884999_strand_+_gene_Chr7-6Mb_reg_777, 9608 candidate probes\n",
      "--- 1718 probes passed check_dic selection.\n",
      "finish in 0.213s, 39 probes kept.\n",
      "-- check region:777 7:47884999-47889999_strand_+_gene_Chr7-6Mb_reg_778, 9474 candidate probes\n",
      "--- 2168 probes passed check_dic selection.\n",
      "finish in 0.226s, 41 probes kept.\n",
      "-- check region:778 7:47889999-47894999_strand_+_gene_Chr7-6Mb_reg_779, 8770 candidate probes\n",
      "--- 386 probes passed check_dic selection.\n",
      "finish in 0.153s, 6 probes kept.\n",
      "-- check region:779 7:47894999-47899999_strand_+_gene_Chr7-6Mb_reg_780, 9140 candidate probes\n",
      "--- 1068 probes passed check_dic selection.\n",
      "finish in 0.182s, 26 probes kept.\n",
      "-- check region:780 7:47899999-47904999_strand_+_gene_Chr7-6Mb_reg_781, 9468 candidate probes\n",
      "--- 1702 probes passed check_dic selection.\n",
      "finish in 0.207s, 38 probes kept.\n",
      "-- check region:781 7:47904999-47909999_strand_+_gene_Chr7-6Mb_reg_782, 8086 candidate probes\n",
      "--- 40 probes passed check_dic selection.\n",
      "finish in 0.133s, 1 probes kept.\n",
      "-- check region:782 7:47909999-47914999_strand_+_gene_Chr7-6Mb_reg_783, 9414 candidate probes\n",
      "--- 128 probes passed check_dic selection.\n",
      "finish in 0.155s, 8 probes kept.\n",
      "-- check region:783 7:47914999-47919999_strand_+_gene_Chr7-6Mb_reg_784, 9586 candidate probes\n",
      "--- 2294 probes passed check_dic selection.\n",
      "finish in 0.232s, 48 probes kept.\n",
      "-- check region:784 7:47919999-47924999_strand_+_gene_Chr7-6Mb_reg_785, 9704 candidate probes\n",
      "--- 3204 probes passed check_dic selection.\n",
      "finish in 0.265s, 54 probes kept.\n",
      "-- check region:785 7:47924999-47929999_strand_+_gene_Chr7-6Mb_reg_786, 9816 candidate probes\n",
      "--- 1812 probes passed check_dic selection.\n",
      "finish in 0.215s, 40 probes kept.\n",
      "-- check region:786 7:47929999-47934999_strand_+_gene_Chr7-6Mb_reg_787, 9676 candidate probes\n",
      "--- 2794 probes passed check_dic selection.\n",
      "finish in 0.250s, 48 probes kept.\n",
      "-- check region:787 7:47934999-47939999_strand_+_gene_Chr7-6Mb_reg_788, 9568 candidate probes\n",
      "--- 2940 probes passed check_dic selection.\n",
      "finish in 0.252s, 49 probes kept.\n",
      "-- check region:788 7:47939999-47944999_strand_+_gene_Chr7-6Mb_reg_789, 9768 candidate probes\n",
      "--- 658 probes passed check_dic selection.\n",
      "finish in 0.178s, 17 probes kept.\n",
      "-- check region:789 7:47944999-47949999_strand_+_gene_Chr7-6Mb_reg_790, 9382 candidate probes\n",
      "--- 1676 probes passed check_dic selection.\n",
      "finish in 0.209s, 39 probes kept.\n",
      "-- check region:790 7:47949999-47954999_strand_+_gene_Chr7-6Mb_reg_791, 9840 candidate probes\n",
      "--- 1258 probes passed check_dic selection.\n",
      "finish in 0.199s, 25 probes kept.\n",
      "-- check region:791 7:47954999-47959999_strand_+_gene_Chr7-6Mb_reg_792, 9868 candidate probes\n",
      "--- 2116 probes passed check_dic selection.\n",
      "finish in 0.232s, 43 probes kept.\n",
      "-- check region:792 7:47959999-47964999_strand_+_gene_Chr7-6Mb_reg_793, 9806 candidate probes\n",
      "--- 1150 probes passed check_dic selection.\n",
      "finish in 0.197s, 29 probes kept.\n",
      "-- check region:793 7:47964999-47969999_strand_+_gene_Chr7-6Mb_reg_794, 9574 candidate probes\n",
      "--- 1574 probes passed check_dic selection.\n",
      "finish in 0.210s, 37 probes kept.\n",
      "-- check region:794 7:47969999-47974999_strand_+_gene_Chr7-6Mb_reg_795, 9446 candidate probes\n",
      "--- 1106 probes passed check_dic selection.\n",
      "finish in 0.188s, 24 probes kept.\n",
      "-- check region:795 7:47974999-47979999_strand_+_gene_Chr7-6Mb_reg_796, 9826 candidate probes\n",
      "--- 1120 probes passed check_dic selection.\n",
      "finish in 0.192s, 24 probes kept.\n",
      "-- check region:796 7:47979999-47984999_strand_+_gene_Chr7-6Mb_reg_797, 9918 candidate probes\n",
      "--- 1984 probes passed check_dic selection.\n",
      "finish in 0.229s, 44 probes kept.\n",
      "-- check region:797 7:47984999-47989999_strand_+_gene_Chr7-6Mb_reg_798, 9844 candidate probes\n",
      "--- 2614 probes passed check_dic selection.\n",
      "finish in 0.266s, 52 probes kept.\n",
      "-- check region:798 7:47989999-47994999_strand_+_gene_Chr7-6Mb_reg_799, 9910 candidate probes\n",
      "--- 2010 probes passed check_dic selection.\n",
      "finish in 0.225s, 45 probes kept.\n",
      "-- check region:799 7:47994999-47999999_strand_+_gene_Chr7-6Mb_reg_800, 8110 candidate probes\n",
      "--- 1564 probes passed check_dic selection.\n",
      "finish in 0.183s, 27 probes kept.\n",
      "-- check region:800 7:47999999-48004999_strand_+_gene_Chr7-6Mb_reg_801, 7158 candidate probes\n",
      "--- 1446 probes passed check_dic selection.\n",
      "finish in 0.165s, 23 probes kept.\n",
      "-- check region:801 7:48004999-48009999_strand_+_gene_Chr7-6Mb_reg_802, 9854 candidate probes\n",
      "--- 2818 probes passed check_dic selection.\n",
      "finish in 0.254s, 51 probes kept.\n",
      "-- check region:802 7:48009999-48014999_strand_+_gene_Chr7-6Mb_reg_803, 9216 candidate probes\n",
      "--- 24 probes passed check_dic selection.\n",
      "finish in 0.148s, 4 probes kept.\n",
      "-- check region:803 7:48014999-48019999_strand_+_gene_Chr7-6Mb_reg_804, 9670 candidate probes\n",
      "--- 2510 probes passed check_dic selection.\n",
      "finish in 0.239s, 43 probes kept.\n",
      "-- check region:804 7:48019999-48024999_strand_+_gene_Chr7-6Mb_reg_805, 9846 candidate probes\n",
      "--- 2362 probes passed check_dic selection.\n",
      "finish in 0.243s, 48 probes kept.\n",
      "-- check region:805 7:48024999-48029999_strand_+_gene_Chr7-6Mb_reg_806, 9340 candidate probes\n",
      "--- 3022 probes passed check_dic selection.\n",
      "finish in 0.253s, 51 probes kept.\n",
      "-- check region:806 7:48029999-48034999_strand_+_gene_Chr7-6Mb_reg_807, 9918 candidate probes\n",
      "--- 3280 probes passed check_dic selection.\n",
      "finish in 0.272s, 53 probes kept.\n",
      "-- check region:807 7:48034999-48039999_strand_+_gene_Chr7-6Mb_reg_808, 9918 candidate probes\n",
      "--- 4158 probes passed check_dic selection.\n",
      "finish in 0.299s, 64 probes kept.\n",
      "-- check region:808 7:48039999-48044999_strand_+_gene_Chr7-6Mb_reg_809, 9694 candidate probes\n",
      "--- 4498 probes passed check_dic selection.\n",
      "finish in 0.307s, 66 probes kept.\n",
      "-- check region:809 7:48044999-48049999_strand_+_gene_Chr7-6Mb_reg_810, 8790 candidate probes\n",
      "--- 1460 probes passed check_dic selection.\n",
      "finish in 0.188s, 27 probes kept.\n",
      "-- check region:810 7:48049999-48054999_strand_+_gene_Chr7-6Mb_reg_811, 9520 candidate probes\n",
      "--- 602 probes passed check_dic selection.\n",
      "finish in 0.175s, 18 probes kept.\n",
      "-- check region:811 7:48054999-48059999_strand_+_gene_Chr7-6Mb_reg_812, 9576 candidate probes\n",
      "--- 772 probes passed check_dic selection.\n",
      "finish in 0.176s, 22 probes kept.\n",
      "-- check region:812 7:48059999-48064999_strand_+_gene_Chr7-6Mb_reg_813, 9534 candidate probes\n",
      "--- 3420 probes passed check_dic selection.\n",
      "finish in 0.265s, 52 probes kept.\n",
      "-- check region:813 7:48064999-48069999_strand_+_gene_Chr7-6Mb_reg_814, 9650 candidate probes\n",
      "--- 1520 probes passed check_dic selection.\n",
      "finish in 0.204s, 23 probes kept.\n",
      "-- check region:814 7:48069999-48074999_strand_+_gene_Chr7-6Mb_reg_815, 9748 candidate probes\n",
      "--- 194 probes passed check_dic selection.\n",
      "finish in 0.166s, 8 probes kept.\n",
      "-- check region:815 7:48074999-48079999_strand_+_gene_Chr7-6Mb_reg_816, 9856 candidate probes\n",
      "--- 4238 probes passed check_dic selection.\n",
      "finish in 0.330s, 63 probes kept.\n",
      "-- check region:816 7:48079999-48084999_strand_+_gene_Chr7-6Mb_reg_817, 9916 candidate probes\n",
      "--- 5250 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.335s, 67 probes kept.\n",
      "-- check region:817 7:48084999-48089999_strand_+_gene_Chr7-6Mb_reg_818, 9862 candidate probes\n",
      "--- 2156 probes passed check_dic selection.\n",
      "finish in 0.232s, 46 probes kept.\n",
      "-- check region:818 7:48089999-48094999_strand_+_gene_Chr7-6Mb_reg_819, 9800 candidate probes\n",
      "--- 4328 probes passed check_dic selection.\n",
      "finish in 0.302s, 68 probes kept.\n",
      "-- check region:819 7:48094999-48099999_strand_+_gene_Chr7-6Mb_reg_820, 9810 candidate probes\n",
      "--- 4268 probes passed check_dic selection.\n",
      "finish in 0.301s, 62 probes kept.\n",
      "-- check region:820 7:48099999-48104999_strand_+_gene_Chr7-6Mb_reg_821, 8932 candidate probes\n",
      "--- 2072 probes passed check_dic selection.\n",
      "finish in 0.213s, 30 probes kept.\n",
      "-- check region:821 7:48104999-48109999_strand_+_gene_Chr7-6Mb_reg_822, 8720 candidate probes\n",
      "--- 3558 probes passed check_dic selection.\n",
      "finish in 0.259s, 49 probes kept.\n",
      "-- check region:822 7:48109999-48114999_strand_+_gene_Chr7-6Mb_reg_823, 9434 candidate probes\n",
      "--- 2506 probes passed check_dic selection.\n",
      "finish in 0.236s, 37 probes kept.\n",
      "-- check region:823 7:48114999-48119999_strand_+_gene_Chr7-6Mb_reg_824, 9902 candidate probes\n",
      "--- 4584 probes passed check_dic selection.\n",
      "finish in 0.313s, 68 probes kept.\n",
      "-- check region:824 7:48119999-48124999_strand_+_gene_Chr7-6Mb_reg_825, 9640 candidate probes\n",
      "--- 1574 probes passed check_dic selection.\n",
      "finish in 0.210s, 30 probes kept.\n",
      "-- check region:825 7:48124999-48129999_strand_+_gene_Chr7-6Mb_reg_826, 9812 candidate probes\n",
      "--- 36 probes passed check_dic selection.\n",
      "finish in 0.160s, 1 probes kept.\n",
      "-- check region:826 7:48129999-48134999_strand_+_gene_Chr7-6Mb_reg_827, 9446 candidate probes\n",
      "--- 698 probes passed check_dic selection.\n",
      "finish in 0.176s, 13 probes kept.\n",
      "-- check region:827 7:48134999-48139999_strand_+_gene_Chr7-6Mb_reg_828, 7742 candidate probes\n",
      "--- 1874 probes passed check_dic selection.\n",
      "finish in 0.189s, 27 probes kept.\n",
      "-- check region:828 7:48139999-48144999_strand_+_gene_Chr7-6Mb_reg_829, 9918 candidate probes\n",
      "--- 1766 probes passed check_dic selection.\n",
      "finish in 0.219s, 39 probes kept.\n",
      "-- check region:829 7:48144999-48149999_strand_+_gene_Chr7-6Mb_reg_830, 6408 candidate probes\n",
      "--- 356 probes passed check_dic selection.\n",
      "finish in 0.114s, 9 probes kept.\n",
      "-- check region:830 7:48149999-48154999_strand_+_gene_Chr7-6Mb_reg_831, 9290 candidate probes\n",
      "--- 1286 probes passed check_dic selection.\n",
      "finish in 0.192s, 25 probes kept.\n",
      "-- check region:831 7:48154999-48159999_strand_+_gene_Chr7-6Mb_reg_832, 9896 candidate probes\n",
      "--- 1512 probes passed check_dic selection.\n",
      "finish in 0.207s, 29 probes kept.\n",
      "-- check region:832 7:48159999-48164999_strand_+_gene_Chr7-6Mb_reg_833, 9918 candidate probes\n",
      "--- 1096 probes passed check_dic selection.\n",
      "finish in 0.192s, 32 probes kept.\n",
      "-- check region:833 7:48164999-48169999_strand_+_gene_Chr7-6Mb_reg_834, 9840 candidate probes\n",
      "--- 3566 probes passed check_dic selection.\n",
      "finish in 0.274s, 56 probes kept.\n",
      "-- check region:834 7:48169999-48174999_strand_+_gene_Chr7-6Mb_reg_835, 9876 candidate probes\n",
      "--- 4334 probes passed check_dic selection.\n",
      "finish in 0.303s, 64 probes kept.\n",
      "-- check region:835 7:48174999-48179999_strand_+_gene_Chr7-6Mb_reg_836, 9888 candidate probes\n",
      "--- 4626 probes passed check_dic selection.\n",
      "finish in 0.314s, 68 probes kept.\n",
      "-- check region:836 7:48179999-48184999_strand_+_gene_Chr7-6Mb_reg_837, 9884 candidate probes\n",
      "--- 4730 probes passed check_dic selection.\n",
      "finish in 0.316s, 67 probes kept.\n",
      "-- check region:837 7:48184999-48189999_strand_+_gene_Chr7-6Mb_reg_838, 9784 candidate probes\n",
      "--- 858 probes passed check_dic selection.\n",
      "finish in 0.184s, 16 probes kept.\n",
      "-- check region:838 7:48189999-48194999_strand_+_gene_Chr7-6Mb_reg_839, 9312 candidate probes\n",
      "--- 770 probes passed check_dic selection.\n",
      "finish in 0.174s, 16 probes kept.\n",
      "-- check region:839 7:48194999-48199999_strand_+_gene_Chr7-6Mb_reg_840, 9918 candidate probes\n",
      "--- 4106 probes passed check_dic selection.\n",
      "finish in 0.293s, 66 probes kept.\n",
      "-- check region:840 7:48199999-48204999_strand_+_gene_Chr7-6Mb_reg_841, 9850 candidate probes\n",
      "--- 4818 probes passed check_dic selection.\n",
      "finish in 0.318s, 70 probes kept.\n",
      "-- check region:841 7:48204999-48209999_strand_+_gene_Chr7-6Mb_reg_842, 9918 candidate probes\n",
      "--- 3624 probes passed check_dic selection.\n",
      "finish in 0.285s, 53 probes kept.\n",
      "-- check region:842 7:48209999-48214999_strand_+_gene_Chr7-6Mb_reg_843, 9812 candidate probes\n",
      "--- 1042 probes passed check_dic selection.\n",
      "finish in 0.189s, 18 probes kept.\n",
      "-- check region:843 7:48214999-48219999_strand_+_gene_Chr7-6Mb_reg_844, 9382 candidate probes\n",
      "--- 392 probes passed check_dic selection.\n",
      "finish in 0.169s, 11 probes kept.\n",
      "-- check region:844 7:48219999-48224999_strand_+_gene_Chr7-6Mb_reg_845, 9918 candidate probes\n",
      "--- 3050 probes passed check_dic selection.\n",
      "finish in 0.259s, 52 probes kept.\n",
      "-- check region:845 7:48224999-48229999_strand_+_gene_Chr7-6Mb_reg_846, 9846 candidate probes\n",
      "--- 4182 probes passed check_dic selection.\n",
      "finish in 0.298s, 65 probes kept.\n",
      "-- check region:846 7:48229999-48234999_strand_+_gene_Chr7-6Mb_reg_847, 9818 candidate probes\n",
      "--- 550 probes passed check_dic selection.\n",
      "finish in 0.175s, 22 probes kept.\n",
      "-- check region:847 7:48234999-48239999_strand_+_gene_Chr7-6Mb_reg_848, 9918 candidate probes\n",
      "--- 1500 probes passed check_dic selection.\n",
      "finish in 0.208s, 31 probes kept.\n",
      "-- check region:848 7:48239999-48244999_strand_+_gene_Chr7-6Mb_reg_849, 9868 candidate probes\n",
      "--- 3406 probes passed check_dic selection.\n",
      "finish in 0.270s, 59 probes kept.\n",
      "-- check region:849 7:48244999-48249999_strand_+_gene_Chr7-6Mb_reg_850, 9620 candidate probes\n",
      "--- 3258 probes passed check_dic selection.\n",
      "finish in 0.261s, 56 probes kept.\n",
      "-- check region:850 7:48249999-48254999_strand_+_gene_Chr7-6Mb_reg_851, 5984 candidate probes\n",
      "--- 376 probes passed check_dic selection.\n",
      "finish in 0.110s, 11 probes kept.\n",
      "-- check region:851 7:48254999-48259999_strand_+_gene_Chr7-6Mb_reg_852, 7706 candidate probes\n",
      "--- 1272 probes passed check_dic selection.\n",
      "finish in 0.168s, 27 probes kept.\n",
      "-- check region:852 7:48259999-48264999_strand_+_gene_Chr7-6Mb_reg_853, 9898 candidate probes\n",
      "--- 2728 probes passed check_dic selection.\n",
      "finish in 0.247s, 43 probes kept.\n",
      "-- check region:853 7:48264999-48269999_strand_+_gene_Chr7-6Mb_reg_854, 9896 candidate probes\n",
      "--- 2000 probes passed check_dic selection.\n",
      "finish in 0.223s, 33 probes kept.\n",
      "-- check region:854 7:48269999-48274999_strand_+_gene_Chr7-6Mb_reg_855, 9548 candidate probes\n",
      "--- 1432 probes passed check_dic selection.\n",
      "finish in 0.202s, 38 probes kept.\n",
      "-- check region:855 7:48274999-48279999_strand_+_gene_Chr7-6Mb_reg_856, 9498 candidate probes\n",
      "--- 3146 probes passed check_dic selection.\n",
      "finish in 0.257s, 51 probes kept.\n",
      "-- check region:856 7:48279999-48284999_strand_+_gene_Chr7-6Mb_reg_857, 9908 candidate probes\n",
      "--- 2234 probes passed check_dic selection.\n",
      "finish in 0.233s, 42 probes kept.\n",
      "-- check region:857 7:48284999-48289999_strand_+_gene_Chr7-6Mb_reg_858, 9676 candidate probes\n",
      "--- 2184 probes passed check_dic selection.\n",
      "finish in 0.231s, 51 probes kept.\n",
      "-- check region:858 7:48289999-48294999_strand_+_gene_Chr7-6Mb_reg_859, 7700 candidate probes\n",
      "--- 1434 probes passed check_dic selection.\n",
      "finish in 0.172s, 28 probes kept.\n",
      "-- check region:859 7:48294999-48299999_strand_+_gene_Chr7-6Mb_reg_860, 6812 candidate probes\n",
      "--- 1306 probes passed check_dic selection.\n",
      "finish in 0.152s, 30 probes kept.\n",
      "-- check region:860 7:48299999-48304999_strand_+_gene_Chr7-6Mb_reg_861, 4032 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.065s, 0 probes kept.\n",
      "-- check region:861 7:48304999-48309999_strand_+_gene_Chr7-6Mb_reg_862, 9108 candidate probes\n",
      "--- 2822 probes passed check_dic selection.\n",
      "finish in 0.240s, 44 probes kept.\n",
      "-- check region:862 7:48309999-48314999_strand_+_gene_Chr7-6Mb_reg_863, 9704 candidate probes\n",
      "--- 4064 probes passed check_dic selection.\n",
      "finish in 0.292s, 65 probes kept.\n",
      "-- check region:863 7:48314999-48319999_strand_+_gene_Chr7-6Mb_reg_864, 9822 candidate probes\n",
      "--- 1848 probes passed check_dic selection.\n",
      "finish in 0.220s, 38 probes kept.\n",
      "-- check region:864 7:48319999-48324999_strand_+_gene_Chr7-6Mb_reg_865, 9462 candidate probes\n",
      "--- 1358 probes passed check_dic selection.\n",
      "finish in 0.195s, 33 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:865 7:48324999-48329999_strand_+_gene_Chr7-6Mb_reg_866, 9110 candidate probes\n",
      "--- 2110 probes passed check_dic selection.\n",
      "finish in 0.217s, 35 probes kept.\n",
      "-- check region:866 7:48329999-48334999_strand_+_gene_Chr7-6Mb_reg_867, 8798 candidate probes\n",
      "--- 306 probes passed check_dic selection.\n",
      "finish in 0.151s, 11 probes kept.\n",
      "-- check region:867 7:48334999-48339999_strand_+_gene_Chr7-6Mb_reg_868, 9408 candidate probes\n",
      "--- 2100 probes passed check_dic selection.\n",
      "finish in 0.218s, 43 probes kept.\n",
      "-- check region:868 7:48339999-48344999_strand_+_gene_Chr7-6Mb_reg_869, 9860 candidate probes\n",
      "--- 1460 probes passed check_dic selection.\n",
      "finish in 0.208s, 37 probes kept.\n",
      "-- check region:869 7:48344999-48349999_strand_+_gene_Chr7-6Mb_reg_870, 9884 candidate probes\n",
      "--- 3786 probes passed check_dic selection.\n",
      "finish in 0.289s, 62 probes kept.\n",
      "-- check region:870 7:48349999-48354999_strand_+_gene_Chr7-6Mb_reg_871, 8872 candidate probes\n",
      "--- 296 probes passed check_dic selection.\n",
      "finish in 0.154s, 8 probes kept.\n",
      "-- check region:871 7:48354999-48359999_strand_+_gene_Chr7-6Mb_reg_872, 7752 candidate probes\n",
      "--- 408 probes passed check_dic selection.\n",
      "finish in 0.137s, 8 probes kept.\n",
      "-- check region:872 7:48359999-48364999_strand_+_gene_Chr7-6Mb_reg_873, 9892 candidate probes\n",
      "--- 1816 probes passed check_dic selection.\n",
      "finish in 0.218s, 39 probes kept.\n",
      "-- check region:873 7:48364999-48369999_strand_+_gene_Chr7-6Mb_reg_874, 5038 candidate probes\n",
      "--- 558 probes passed check_dic selection.\n",
      "finish in 0.099s, 11 probes kept.\n",
      "-- check region:874 7:48369999-48374999_strand_+_gene_Chr7-6Mb_reg_875, 9482 candidate probes\n",
      "--- 910 probes passed check_dic selection.\n",
      "finish in 0.187s, 20 probes kept.\n",
      "-- check region:875 7:48374999-48379999_strand_+_gene_Chr7-6Mb_reg_876, 9918 candidate probes\n",
      "--- 2750 probes passed check_dic selection.\n",
      "finish in 0.249s, 51 probes kept.\n",
      "-- check region:876 7:48379999-48384999_strand_+_gene_Chr7-6Mb_reg_877, 8674 candidate probes\n",
      "--- 278 probes passed check_dic selection.\n",
      "finish in 0.147s, 7 probes kept.\n",
      "-- check region:877 7:48384999-48389999_strand_+_gene_Chr7-6Mb_reg_878, 9784 candidate probes\n",
      "--- 3232 probes passed check_dic selection.\n",
      "finish in 0.266s, 44 probes kept.\n",
      "-- check region:878 7:48389999-48394999_strand_+_gene_Chr7-6Mb_reg_879, 9914 candidate probes\n",
      "--- 2576 probes passed check_dic selection.\n",
      "finish in 0.246s, 47 probes kept.\n",
      "-- check region:879 7:48394999-48399999_strand_+_gene_Chr7-6Mb_reg_880, 9918 candidate probes\n",
      "--- 1922 probes passed check_dic selection.\n",
      "finish in 0.223s, 40 probes kept.\n",
      "-- check region:880 7:48399999-48404999_strand_+_gene_Chr7-6Mb_reg_881, 5700 candidate probes\n",
      "--- 2522 probes passed check_dic selection.\n",
      "finish in 0.176s, 34 probes kept.\n",
      "-- check region:881 7:48404999-48409999_strand_+_gene_Chr7-6Mb_reg_882, 8348 candidate probes\n",
      "--- 670 probes passed check_dic selection.\n",
      "finish in 0.158s, 13 probes kept.\n",
      "-- check region:882 7:48409999-48414999_strand_+_gene_Chr7-6Mb_reg_883, 7706 candidate probes\n",
      "--- 1716 probes passed check_dic selection.\n",
      "finish in 0.181s, 26 probes kept.\n",
      "-- check region:883 7:48414999-48419999_strand_+_gene_Chr7-6Mb_reg_884, 5968 candidate probes\n",
      "--- 732 probes passed check_dic selection.\n",
      "finish in 0.119s, 12 probes kept.\n",
      "-- check region:884 7:48419999-48424999_strand_+_gene_Chr7-6Mb_reg_885, 9210 candidate probes\n",
      "--- 2012 probes passed check_dic selection.\n",
      "finish in 0.213s, 33 probes kept.\n",
      "-- check region:885 7:48424999-48429999_strand_+_gene_Chr7-6Mb_reg_886, 9678 candidate probes\n",
      "--- 132 probes passed check_dic selection.\n",
      "finish in 0.156s, 7 probes kept.\n",
      "-- check region:886 7:48429999-48434999_strand_+_gene_Chr7-6Mb_reg_887, 9574 candidate probes\n",
      "--- 298 probes passed check_dic selection.\n",
      "finish in 0.161s, 14 probes kept.\n",
      "-- check region:887 7:48434999-48439999_strand_+_gene_Chr7-6Mb_reg_888, 9888 candidate probes\n",
      "--- 628 probes passed check_dic selection.\n",
      "finish in 0.180s, 27 probes kept.\n",
      "-- check region:888 7:48439999-48444999_strand_+_gene_Chr7-6Mb_reg_889, 9867 candidate probes\n",
      "--- 2970 probes passed check_dic selection.\n",
      "finish in 0.255s, 58 probes kept.\n",
      "-- check region:889 7:48444999-48449999_strand_+_gene_Chr7-6Mb_reg_890, 9752 candidate probes\n",
      "--- 3584 probes passed check_dic selection.\n",
      "finish in 0.275s, 55 probes kept.\n",
      "-- check region:890 7:48449999-48454999_strand_+_gene_Chr7-6Mb_reg_891, 9918 candidate probes\n",
      "--- 1352 probes passed check_dic selection.\n",
      "finish in 0.198s, 40 probes kept.\n",
      "-- check region:891 7:48454999-48459999_strand_+_gene_Chr7-6Mb_reg_892, 9918 candidate probes\n",
      "--- 2358 probes passed check_dic selection.\n",
      "finish in 0.235s, 40 probes kept.\n",
      "-- check region:892 7:48459999-48464999_strand_+_gene_Chr7-6Mb_reg_893, 9632 candidate probes\n",
      "--- 3040 probes passed check_dic selection.\n",
      "finish in 0.254s, 55 probes kept.\n",
      "-- check region:893 7:48464999-48469999_strand_+_gene_Chr7-6Mb_reg_894, 9854 candidate probes\n",
      "--- 1872 probes passed check_dic selection.\n",
      "finish in 0.218s, 43 probes kept.\n",
      "-- check region:894 7:48469999-48474999_strand_+_gene_Chr7-6Mb_reg_895, 9906 candidate probes\n",
      "--- 2430 probes passed check_dic selection.\n",
      "finish in 0.237s, 46 probes kept.\n",
      "-- check region:895 7:48474999-48479999_strand_+_gene_Chr7-6Mb_reg_896, 9820 candidate probes\n",
      "--- 3680 probes passed check_dic selection.\n",
      "finish in 0.278s, 60 probes kept.\n",
      "-- check region:896 7:48479999-48484999_strand_+_gene_Chr7-6Mb_reg_897, 9554 candidate probes\n",
      "--- 3956 probes passed check_dic selection.\n",
      "finish in 0.282s, 61 probes kept.\n",
      "-- check region:897 7:48484999-48489999_strand_+_gene_Chr7-6Mb_reg_898, 9144 candidate probes\n",
      "--- 844 probes passed check_dic selection.\n",
      "finish in 0.178s, 15 probes kept.\n",
      "-- check region:898 7:48489999-48494999_strand_+_gene_Chr7-6Mb_reg_899, 9788 candidate probes\n",
      "--- 492 probes passed check_dic selection.\n",
      "finish in 0.174s, 15 probes kept.\n",
      "-- check region:899 7:48494999-48499999_strand_+_gene_Chr7-6Mb_reg_900, 9918 candidate probes\n",
      "--- 3256 probes passed check_dic selection.\n",
      "finish in 0.268s, 58 probes kept.\n",
      "-- check region:900 7:48499999-48504999_strand_+_gene_Chr7-6Mb_reg_901, 9918 candidate probes\n",
      "--- 4224 probes passed check_dic selection.\n",
      "finish in 0.299s, 68 probes kept.\n",
      "-- check region:901 7:48504999-48509999_strand_+_gene_Chr7-6Mb_reg_902, 9890 candidate probes\n",
      "--- 3224 probes passed check_dic selection.\n",
      "finish in 0.265s, 48 probes kept.\n",
      "-- check region:902 7:48509999-48514999_strand_+_gene_Chr7-6Mb_reg_903, 9862 candidate probes\n",
      "--- 366 probes passed check_dic selection.\n",
      "finish in 0.167s, 14 probes kept.\n",
      "-- check region:903 7:48514999-48519999_strand_+_gene_Chr7-6Mb_reg_904, 9808 candidate probes\n",
      "--- 1662 probes passed check_dic selection.\n",
      "finish in 0.214s, 36 probes kept.\n",
      "-- check region:904 7:48519999-48524999_strand_+_gene_Chr7-6Mb_reg_905, 9526 candidate probes\n",
      "--- 150 probes passed check_dic selection.\n",
      "finish in 0.161s, 9 probes kept.\n",
      "-- check region:905 7:48524999-48529999_strand_+_gene_Chr7-6Mb_reg_906, 9874 candidate probes\n",
      "--- 732 probes passed check_dic selection.\n",
      "finish in 0.186s, 23 probes kept.\n",
      "-- check region:906 7:48529999-48534999_strand_+_gene_Chr7-6Mb_reg_907, 9794 candidate probes\n",
      "--- 1750 probes passed check_dic selection.\n",
      "finish in 0.214s, 42 probes kept.\n",
      "-- check region:907 7:48534999-48539999_strand_+_gene_Chr7-6Mb_reg_908, 7702 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.123s, 0 probes kept.\n",
      "-- check region:908 7:48539999-48544999_strand_+_gene_Chr7-6Mb_reg_909, 8594 candidate probes\n",
      "--- 26 probes passed check_dic selection.\n",
      "finish in 0.143s, 3 probes kept.\n",
      "-- check region:909 7:48544999-48549999_strand_+_gene_Chr7-6Mb_reg_910, 9886 candidate probes\n",
      "--- 174 probes passed check_dic selection.\n",
      "finish in 0.167s, 4 probes kept.\n",
      "-- check region:910 7:48549999-48554999_strand_+_gene_Chr7-6Mb_reg_911, 9918 candidate probes\n",
      "--- 3502 probes passed check_dic selection.\n",
      "finish in 0.274s, 50 probes kept.\n",
      "-- check region:911 7:48554999-48559999_strand_+_gene_Chr7-6Mb_reg_912, 9804 candidate probes\n",
      "--- 3680 probes passed check_dic selection.\n",
      "finish in 0.278s, 62 probes kept.\n",
      "-- check region:912 7:48559999-48564999_strand_+_gene_Chr7-6Mb_reg_913, 9918 candidate probes\n",
      "--- 966 probes passed check_dic selection.\n",
      "finish in 0.188s, 22 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:913 7:48564999-48569999_strand_+_gene_Chr7-6Mb_reg_914, 8600 candidate probes\n",
      "--- 196 probes passed check_dic selection.\n",
      "finish in 0.144s, 7 probes kept.\n",
      "-- check region:914 7:48569999-48574999_strand_+_gene_Chr7-6Mb_reg_915, 8342 candidate probes\n",
      "--- 32 probes passed check_dic selection.\n",
      "finish in 0.136s, 1 probes kept.\n",
      "-- check region:915 7:48574999-48579999_strand_+_gene_Chr7-6Mb_reg_916, 9864 candidate probes\n",
      "--- 1712 probes passed check_dic selection.\n",
      "finish in 0.210s, 32 probes kept.\n",
      "-- check region:916 7:48579999-48584999_strand_+_gene_Chr7-6Mb_reg_917, 8944 candidate probes\n",
      "--- 3170 probes passed check_dic selection.\n",
      "finish in 0.247s, 48 probes kept.\n",
      "-- check region:917 7:48584999-48589999_strand_+_gene_Chr7-6Mb_reg_918, 9914 candidate probes\n",
      "--- 1746 probes passed check_dic selection.\n",
      "finish in 0.215s, 36 probes kept.\n",
      "-- check region:918 7:48589999-48594999_strand_+_gene_Chr7-6Mb_reg_919, 9908 candidate probes\n",
      "--- 2314 probes passed check_dic selection.\n",
      "finish in 0.235s, 34 probes kept.\n",
      "-- check region:919 7:48594999-48599999_strand_+_gene_Chr7-6Mb_reg_920, 9582 candidate probes\n",
      "--- 720 probes passed check_dic selection.\n",
      "finish in 0.178s, 17 probes kept.\n",
      "-- check region:920 7:48599999-48604999_strand_+_gene_Chr7-6Mb_reg_921, 8840 candidate probes\n",
      "--- 74 probes passed check_dic selection.\n",
      "finish in 0.144s, 3 probes kept.\n",
      "-- check region:921 7:48604999-48609999_strand_+_gene_Chr7-6Mb_reg_922, 9878 candidate probes\n",
      "--- 3082 probes passed check_dic selection.\n",
      "finish in 0.262s, 49 probes kept.\n",
      "-- check region:922 7:48609999-48614999_strand_+_gene_Chr7-6Mb_reg_923, 9912 candidate probes\n",
      "--- 2296 probes passed check_dic selection.\n",
      "finish in 0.235s, 41 probes kept.\n",
      "-- check region:923 7:48614999-48619999_strand_+_gene_Chr7-6Mb_reg_924, 9752 candidate probes\n",
      "--- 1230 probes passed check_dic selection.\n",
      "finish in 0.198s, 33 probes kept.\n",
      "-- check region:924 7:48619999-48624999_strand_+_gene_Chr7-6Mb_reg_925, 8454 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.138s, 0 probes kept.\n",
      "-- check region:925 7:48624999-48629999_strand_+_gene_Chr7-6Mb_reg_926, 9820 candidate probes\n",
      "--- 1828 probes passed check_dic selection.\n",
      "finish in 0.222s, 38 probes kept.\n",
      "-- check region:926 7:48629999-48634999_strand_+_gene_Chr7-6Mb_reg_927, 7434 candidate probes\n",
      "--- 980 probes passed check_dic selection.\n",
      "finish in 0.153s, 16 probes kept.\n",
      "-- check region:927 7:48634999-48639999_strand_+_gene_Chr7-6Mb_reg_928, 6844 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.112s, 0 probes kept.\n",
      "-- check region:928 7:48639999-48644999_strand_+_gene_Chr7-6Mb_reg_929, 9800 candidate probes\n",
      "--- 2648 probes passed check_dic selection.\n",
      "finish in 0.249s, 40 probes kept.\n",
      "-- check region:929 7:48644999-48649999_strand_+_gene_Chr7-6Mb_reg_930, 9430 candidate probes\n",
      "--- 80 probes passed check_dic selection.\n",
      "finish in 0.155s, 4 probes kept.\n",
      "-- check region:930 7:48649999-48654999_strand_+_gene_Chr7-6Mb_reg_931, 9348 candidate probes\n",
      "--- 1046 probes passed check_dic selection.\n",
      "finish in 0.183s, 21 probes kept.\n",
      "-- check region:931 7:48654999-48659999_strand_+_gene_Chr7-6Mb_reg_932, 9876 candidate probes\n",
      "--- 3334 probes passed check_dic selection.\n",
      "finish in 0.268s, 48 probes kept.\n",
      "-- check region:932 7:48659999-48664999_strand_+_gene_Chr7-6Mb_reg_933, 9804 candidate probes\n",
      "--- 2276 probes passed check_dic selection.\n",
      "finish in 0.231s, 36 probes kept.\n",
      "-- check region:933 7:48664999-48669999_strand_+_gene_Chr7-6Mb_reg_934, 9918 candidate probes\n",
      "--- 3844 probes passed check_dic selection.\n",
      "finish in 0.286s, 60 probes kept.\n",
      "-- check region:934 7:48669999-48674999_strand_+_gene_Chr7-6Mb_reg_935, 9734 candidate probes\n",
      "--- 1212 probes passed check_dic selection.\n",
      "finish in 0.196s, 28 probes kept.\n",
      "-- check region:935 7:48674999-48679999_strand_+_gene_Chr7-6Mb_reg_936, 9670 candidate probes\n",
      "--- 24 probes passed check_dic selection.\n",
      "finish in 0.157s, 4 probes kept.\n",
      "-- check region:936 7:48679999-48684999_strand_+_gene_Chr7-6Mb_reg_937, 9330 candidate probes\n",
      "--- 1462 probes passed check_dic selection.\n",
      "finish in 0.199s, 25 probes kept.\n",
      "-- check region:937 7:48684999-48689999_strand_+_gene_Chr7-6Mb_reg_938, 9850 candidate probes\n",
      "--- 3572 probes passed check_dic selection.\n",
      "finish in 0.278s, 55 probes kept.\n",
      "-- check region:938 7:48689999-48694999_strand_+_gene_Chr7-6Mb_reg_939, 9874 candidate probes\n",
      "--- 2348 probes passed check_dic selection.\n",
      "finish in 0.234s, 34 probes kept.\n",
      "-- check region:939 7:48694999-48699999_strand_+_gene_Chr7-6Mb_reg_940, 9688 candidate probes\n",
      "--- 18 probes passed check_dic selection.\n",
      "finish in 0.155s, 1 probes kept.\n",
      "-- check region:940 7:48699999-48704999_strand_+_gene_Chr7-6Mb_reg_941, 8790 candidate probes\n",
      "--- 486 probes passed check_dic selection.\n",
      "finish in 0.159s, 8 probes kept.\n",
      "-- check region:941 7:48704999-48709999_strand_+_gene_Chr7-6Mb_reg_942, 9834 candidate probes\n",
      "--- 2912 probes passed check_dic selection.\n",
      "finish in 0.256s, 62 probes kept.\n",
      "-- check region:942 7:48709999-48714999_strand_+_gene_Chr7-6Mb_reg_943, 9570 candidate probes\n",
      "--- 20 probes passed check_dic selection.\n",
      "finish in 0.154s, 2 probes kept.\n",
      "-- check region:943 7:48714999-48719999_strand_+_gene_Chr7-6Mb_reg_944, 9482 candidate probes\n",
      "--- 2936 probes passed check_dic selection.\n",
      "finish in 0.253s, 42 probes kept.\n",
      "-- check region:944 7:48719999-48724999_strand_+_gene_Chr7-6Mb_reg_945, 9915 candidate probes\n",
      "--- 5064 probes passed check_dic selection.\n",
      "finish in 0.328s, 67 probes kept.\n",
      "-- check region:945 7:48724999-48729999_strand_+_gene_Chr7-6Mb_reg_946, 9148 candidate probes\n",
      "--- 2734 probes passed check_dic selection.\n",
      "finish in 0.239s, 44 probes kept.\n",
      "-- check region:946 7:48729999-48734999_strand_+_gene_Chr7-6Mb_reg_947, 9276 candidate probes\n",
      "--- 756 probes passed check_dic selection.\n",
      "finish in 0.172s, 21 probes kept.\n",
      "-- check region:947 7:48734999-48739999_strand_+_gene_Chr7-6Mb_reg_948, 9164 candidate probes\n",
      "--- 136 probes passed check_dic selection.\n",
      "finish in 0.152s, 7 probes kept.\n",
      "-- check region:948 7:48739999-48744999_strand_+_gene_Chr7-6Mb_reg_949, 9918 candidate probes\n",
      "--- 20 probes passed check_dic selection.\n",
      "finish in 0.162s, 2 probes kept.\n",
      "-- check region:949 7:48744999-48749999_strand_+_gene_Chr7-6Mb_reg_950, 9652 candidate probes\n",
      "--- 1914 probes passed check_dic selection.\n",
      "finish in 0.221s, 31 probes kept.\n",
      "-- check region:950 7:48749999-48754999_strand_+_gene_Chr7-6Mb_reg_951, 9918 candidate probes\n",
      "--- 5092 probes passed check_dic selection.\n",
      "finish in 0.335s, 61 probes kept.\n",
      "-- check region:951 7:48754999-48759999_strand_+_gene_Chr7-6Mb_reg_952, 9918 candidate probes\n",
      "--- 7476 probes passed check_dic selection.\n",
      "finish in 0.415s, 81 probes kept.\n",
      "-- check region:952 7:48759999-48764999_strand_+_gene_Chr7-6Mb_reg_953, 9918 candidate probes\n",
      "--- 6308 probes passed check_dic selection.\n",
      "finish in 0.374s, 81 probes kept.\n",
      "-- check region:953 7:48764999-48769999_strand_+_gene_Chr7-6Mb_reg_954, 9918 candidate probes\n",
      "--- 5934 probes passed check_dic selection.\n",
      "finish in 0.362s, 71 probes kept.\n",
      "-- check region:954 7:48769999-48774999_strand_+_gene_Chr7-6Mb_reg_955, 9900 candidate probes\n",
      "--- 5118 probes passed check_dic selection.\n",
      "finish in 0.333s, 70 probes kept.\n",
      "-- check region:955 7:48774999-48779999_strand_+_gene_Chr7-6Mb_reg_956, 9842 candidate probes\n",
      "--- 5516 probes passed check_dic selection.\n",
      "finish in 0.345s, 75 probes kept.\n",
      "-- check region:956 7:48779999-48784999_strand_+_gene_Chr7-6Mb_reg_957, 9912 candidate probes\n",
      "--- 5926 probes passed check_dic selection.\n",
      "finish in 0.360s, 79 probes kept.\n",
      "-- check region:957 7:48784999-48789999_strand_+_gene_Chr7-6Mb_reg_958, 9888 candidate probes\n",
      "--- 7244 probes passed check_dic selection.\n",
      "finish in 0.407s, 87 probes kept.\n",
      "-- check region:958 7:48789999-48794999_strand_+_gene_Chr7-6Mb_reg_959, 9918 candidate probes\n",
      "--- 4380 probes passed check_dic selection.\n",
      "finish in 0.307s, 63 probes kept.\n",
      "-- check region:959 7:48794999-48799999_strand_+_gene_Chr7-6Mb_reg_960, 9918 candidate probes\n",
      "--- 4582 probes passed check_dic selection.\n",
      "finish in 0.311s, 69 probes kept.\n",
      "-- check region:960 7:48799999-48804999_strand_+_gene_Chr7-6Mb_reg_961, 9866 candidate probes\n",
      "--- 4244 probes passed check_dic selection.\n",
      "finish in 0.297s, 63 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:961 7:48804999-48809999_strand_+_gene_Chr7-6Mb_reg_962, 9908 candidate probes\n",
      "--- 5208 probes passed check_dic selection.\n",
      "finish in 0.333s, 69 probes kept.\n",
      "-- check region:962 7:48809999-48814999_strand_+_gene_Chr7-6Mb_reg_963, 9908 candidate probes\n",
      "--- 3546 probes passed check_dic selection.\n",
      "finish in 0.277s, 56 probes kept.\n",
      "-- check region:963 7:48814999-48819999_strand_+_gene_Chr7-6Mb_reg_964, 9918 candidate probes\n",
      "--- 6230 probes passed check_dic selection.\n",
      "finish in 0.371s, 74 probes kept.\n",
      "-- check region:964 7:48819999-48824999_strand_+_gene_Chr7-6Mb_reg_965, 9876 candidate probes\n",
      "--- 6064 probes passed check_dic selection.\n",
      "finish in 0.362s, 73 probes kept.\n",
      "-- check region:965 7:48824999-48829999_strand_+_gene_Chr7-6Mb_reg_966, 9918 candidate probes\n",
      "--- 6180 probes passed check_dic selection.\n",
      "finish in 0.371s, 73 probes kept.\n",
      "-- check region:966 7:48829999-48834999_strand_+_gene_Chr7-6Mb_reg_967, 9896 candidate probes\n",
      "--- 6992 probes passed check_dic selection.\n",
      "finish in 0.399s, 82 probes kept.\n",
      "-- check region:967 7:48834999-48839999_strand_+_gene_Chr7-6Mb_reg_968, 9878 candidate probes\n",
      "--- 7034 probes passed check_dic selection.\n",
      "finish in 0.396s, 80 probes kept.\n",
      "-- check region:968 7:48839999-48844999_strand_+_gene_Chr7-6Mb_reg_969, 9804 candidate probes\n",
      "--- 6536 probes passed check_dic selection.\n",
      "finish in 0.385s, 77 probes kept.\n",
      "-- check region:969 7:48844999-48849999_strand_+_gene_Chr7-6Mb_reg_970, 9828 candidate probes\n",
      "--- 6502 probes passed check_dic selection.\n",
      "finish in 0.379s, 75 probes kept.\n",
      "-- check region:970 7:48849999-48854999_strand_+_gene_Chr7-6Mb_reg_971, 9540 candidate probes\n",
      "--- 5002 probes passed check_dic selection.\n",
      "finish in 0.322s, 68 probes kept.\n",
      "-- check region:971 7:48854999-48859999_strand_+_gene_Chr7-6Mb_reg_972, 9918 candidate probes\n",
      "--- 5658 probes passed check_dic selection.\n",
      "finish in 0.351s, 68 probes kept.\n",
      "-- check region:972 7:48859999-48864999_strand_+_gene_Chr7-6Mb_reg_973, 9916 candidate probes\n",
      "--- 6978 probes passed check_dic selection.\n",
      "finish in 0.398s, 77 probes kept.\n",
      "-- check region:973 7:48864999-48869999_strand_+_gene_Chr7-6Mb_reg_974, 9918 candidate probes\n",
      "--- 7320 probes passed check_dic selection.\n",
      "finish in 0.410s, 78 probes kept.\n",
      "-- check region:974 7:48869999-48874999_strand_+_gene_Chr7-6Mb_reg_975, 9918 candidate probes\n",
      "--- 8290 probes passed check_dic selection.\n",
      "finish in 0.444s, 88 probes kept.\n",
      "-- check region:975 7:48874999-48879999_strand_+_gene_Chr7-6Mb_reg_976, 9886 candidate probes\n",
      "--- 7070 probes passed check_dic selection.\n",
      "finish in 0.402s, 82 probes kept.\n",
      "-- check region:976 7:48879999-48884999_strand_+_gene_Chr7-6Mb_reg_977, 9918 candidate probes\n",
      "--- 7458 probes passed check_dic selection.\n",
      "finish in 0.410s, 84 probes kept.\n",
      "-- check region:977 7:48884999-48889999_strand_+_gene_Chr7-6Mb_reg_978, 9844 candidate probes\n",
      "--- 6632 probes passed check_dic selection.\n",
      "finish in 0.386s, 79 probes kept.\n",
      "-- check region:978 7:48889999-48894999_strand_+_gene_Chr7-6Mb_reg_979, 9834 candidate probes\n",
      "--- 6784 probes passed check_dic selection.\n",
      "finish in 0.391s, 78 probes kept.\n",
      "-- check region:979 7:48894999-48899999_strand_+_gene_Chr7-6Mb_reg_980, 9868 candidate probes\n",
      "--- 7400 probes passed check_dic selection.\n",
      "finish in 0.412s, 80 probes kept.\n",
      "-- check region:980 7:48899999-48904999_strand_+_gene_Chr7-6Mb_reg_981, 9748 candidate probes\n",
      "--- 5858 probes passed check_dic selection.\n",
      "finish in 0.357s, 72 probes kept.\n",
      "-- check region:981 7:48904999-48909999_strand_+_gene_Chr7-6Mb_reg_982, 9918 candidate probes\n",
      "--- 7546 probes passed check_dic selection.\n",
      "finish in 0.425s, 80 probes kept.\n",
      "-- check region:982 7:48909999-48914999_strand_+_gene_Chr7-6Mb_reg_983, 9840 candidate probes\n",
      "--- 6000 probes passed check_dic selection.\n",
      "finish in 0.362s, 73 probes kept.\n",
      "-- check region:983 7:48914999-48919999_strand_+_gene_Chr7-6Mb_reg_984, 9804 candidate probes\n",
      "--- 7172 probes passed check_dic selection.\n",
      "finish in 0.404s, 82 probes kept.\n",
      "-- check region:984 7:48919999-48924999_strand_+_gene_Chr7-6Mb_reg_985, 9904 candidate probes\n",
      "--- 5626 probes passed check_dic selection.\n",
      "finish in 0.351s, 69 probes kept.\n",
      "-- check region:985 7:48924999-48929999_strand_+_gene_Chr7-6Mb_reg_986, 9882 candidate probes\n",
      "--- 6360 probes passed check_dic selection.\n",
      "finish in 0.377s, 77 probes kept.\n",
      "-- check region:986 7:48929999-48934999_strand_+_gene_Chr7-6Mb_reg_987, 9904 candidate probes\n",
      "--- 5810 probes passed check_dic selection.\n",
      "finish in 0.360s, 76 probes kept.\n",
      "-- check region:987 7:48934999-48939999_strand_+_gene_Chr7-6Mb_reg_988, 9690 candidate probes\n",
      "--- 6206 probes passed check_dic selection.\n",
      "finish in 0.368s, 73 probes kept.\n",
      "-- check region:988 7:48939999-48944999_strand_+_gene_Chr7-6Mb_reg_989, 9918 candidate probes\n",
      "--- 5914 probes passed check_dic selection.\n",
      "finish in 0.362s, 73 probes kept.\n",
      "-- check region:989 7:48944999-48949999_strand_+_gene_Chr7-6Mb_reg_990, 9166 candidate probes\n",
      "--- 5498 probes passed check_dic selection.\n",
      "finish in 0.338s, 66 probes kept.\n",
      "-- check region:990 7:48949999-48954999_strand_+_gene_Chr7-6Mb_reg_991, 9918 candidate probes\n",
      "--- 7030 probes passed check_dic selection.\n",
      "finish in 0.402s, 81 probes kept.\n",
      "-- check region:991 7:48954999-48959999_strand_+_gene_Chr7-6Mb_reg_992, 9918 candidate probes\n",
      "--- 7284 probes passed check_dic selection.\n",
      "finish in 0.408s, 78 probes kept.\n",
      "-- check region:992 7:48959999-48964999_strand_+_gene_Chr7-6Mb_reg_993, 9890 candidate probes\n",
      "--- 8018 probes passed check_dic selection.\n",
      "finish in 0.437s, 86 probes kept.\n",
      "-- check region:993 7:48964999-48969999_strand_+_gene_Chr7-6Mb_reg_994, 9914 candidate probes\n",
      "--- 7104 probes passed check_dic selection.\n",
      "finish in 0.405s, 82 probes kept.\n",
      "-- check region:994 7:48969999-48974999_strand_+_gene_Chr7-6Mb_reg_995, 9916 candidate probes\n",
      "--- 7904 probes passed check_dic selection.\n",
      "finish in 0.430s, 83 probes kept.\n",
      "-- check region:995 7:48974999-48979999_strand_+_gene_Chr7-6Mb_reg_996, 9918 candidate probes\n",
      "--- 7346 probes passed check_dic selection.\n",
      "finish in 0.416s, 81 probes kept.\n",
      "-- check region:996 7:48979999-48984999_strand_+_gene_Chr7-6Mb_reg_997, 9918 candidate probes\n",
      "--- 8092 probes passed check_dic selection.\n",
      "finish in 0.438s, 85 probes kept.\n",
      "-- check region:997 7:48984999-48989999_strand_+_gene_Chr7-6Mb_reg_998, 9730 candidate probes\n",
      "--- 5792 probes passed check_dic selection.\n",
      "finish in 0.355s, 68 probes kept.\n",
      "-- check region:998 7:48989999-48994999_strand_+_gene_Chr7-6Mb_reg_999, 9708 candidate probes\n",
      "--- 6916 probes passed check_dic selection.\n",
      "finish in 0.392s, 76 probes kept.\n",
      "-- check region:999 7:48994999-48999999_strand_+_gene_Chr7-6Mb_reg_1000, 9758 candidate probes\n",
      "--- 6832 probes passed check_dic selection.\n",
      "finish in 0.388s, 77 probes kept.\n",
      "-- check region:1000 7:48999999-49004999_strand_+_gene_Chr7-6Mb_reg_1001, 9916 candidate probes\n",
      "--- 7008 probes passed check_dic selection.\n",
      "finish in 0.402s, 76 probes kept.\n",
      "-- check region:1001 7:49004999-49009999_strand_+_gene_Chr7-6Mb_reg_1002, 9906 candidate probes\n",
      "--- 6046 probes passed check_dic selection.\n",
      "finish in 0.364s, 66 probes kept.\n",
      "-- check region:1002 7:49009999-49014999_strand_+_gene_Chr7-6Mb_reg_1003, 9880 candidate probes\n",
      "--- 7448 probes passed check_dic selection.\n",
      "finish in 0.413s, 82 probes kept.\n",
      "-- check region:1003 7:49014999-49019999_strand_+_gene_Chr7-6Mb_reg_1004, 9894 candidate probes\n",
      "--- 7512 probes passed check_dic selection.\n",
      "finish in 0.418s, 83 probes kept.\n",
      "-- check region:1004 7:49019999-49024999_strand_+_gene_Chr7-6Mb_reg_1005, 9872 candidate probes\n",
      "--- 7414 probes passed check_dic selection.\n",
      "finish in 0.415s, 79 probes kept.\n",
      "-- check region:1005 7:49024999-49029999_strand_+_gene_Chr7-6Mb_reg_1006, 9918 candidate probes\n",
      "--- 7396 probes passed check_dic selection.\n",
      "finish in 0.415s, 88 probes kept.\n",
      "-- check region:1006 7:49029999-49034999_strand_+_gene_Chr7-6Mb_reg_1007, 9918 candidate probes\n",
      "--- 7614 probes passed check_dic selection.\n",
      "finish in 0.421s, 86 probes kept.\n",
      "-- check region:1007 7:49034999-49039999_strand_+_gene_Chr7-6Mb_reg_1008, 9918 candidate probes\n",
      "--- 6960 probes passed check_dic selection.\n",
      "finish in 0.397s, 76 probes kept.\n",
      "-- check region:1008 7:49039999-49044999_strand_+_gene_Chr7-6Mb_reg_1009, 9918 candidate probes\n",
      "--- 7736 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.426s, 81 probes kept.\n",
      "-- check region:1009 7:49044999-49049999_strand_+_gene_Chr7-6Mb_reg_1010, 9918 candidate probes\n",
      "--- 8192 probes passed check_dic selection.\n",
      "finish in 0.439s, 90 probes kept.\n",
      "-- check region:1010 7:49049999-49054999_strand_+_gene_Chr7-6Mb_reg_1011, 9852 candidate probes\n",
      "--- 6780 probes passed check_dic selection.\n",
      "finish in 0.389s, 80 probes kept.\n",
      "-- check region:1011 7:49054999-49059999_strand_+_gene_Chr7-6Mb_reg_1012, 9918 candidate probes\n",
      "--- 6632 probes passed check_dic selection.\n",
      "finish in 0.386s, 79 probes kept.\n",
      "-- check region:1012 7:49059999-49064999_strand_+_gene_Chr7-6Mb_reg_1013, 9882 candidate probes\n",
      "--- 6878 probes passed check_dic selection.\n",
      "finish in 0.396s, 78 probes kept.\n",
      "-- check region:1013 7:49064999-49069999_strand_+_gene_Chr7-6Mb_reg_1014, 9888 candidate probes\n",
      "--- 7392 probes passed check_dic selection.\n",
      "finish in 0.413s, 84 probes kept.\n",
      "-- check region:1014 7:49069999-49074999_strand_+_gene_Chr7-6Mb_reg_1015, 9918 candidate probes\n",
      "--- 7736 probes passed check_dic selection.\n",
      "finish in 0.428s, 85 probes kept.\n",
      "-- check region:1015 7:49074999-49079999_strand_+_gene_Chr7-6Mb_reg_1016, 9868 candidate probes\n",
      "--- 6696 probes passed check_dic selection.\n",
      "finish in 0.393s, 77 probes kept.\n",
      "-- check region:1016 7:49079999-49084999_strand_+_gene_Chr7-6Mb_reg_1017, 9836 candidate probes\n",
      "--- 7744 probes passed check_dic selection.\n",
      "finish in 0.421s, 83 probes kept.\n",
      "-- check region:1017 7:49084999-49089999_strand_+_gene_Chr7-6Mb_reg_1018, 9918 candidate probes\n",
      "--- 8492 probes passed check_dic selection.\n",
      "finish in 0.448s, 87 probes kept.\n",
      "-- check region:1018 7:49089999-49094999_strand_+_gene_Chr7-6Mb_reg_1019, 9918 candidate probes\n",
      "--- 8262 probes passed check_dic selection.\n",
      "finish in 0.446s, 91 probes kept.\n",
      "-- check region:1019 7:49094999-49099999_strand_+_gene_Chr7-6Mb_reg_1020, 9410 candidate probes\n",
      "--- 6014 probes passed check_dic selection.\n",
      "finish in 0.357s, 71 probes kept.\n",
      "-- check region:1020 7:49099999-49104999_strand_+_gene_Chr7-6Mb_reg_1021, 9914 candidate probes\n",
      "--- 6888 probes passed check_dic selection.\n",
      "finish in 0.395s, 80 probes kept.\n",
      "-- check region:1021 7:49104999-49109999_strand_+_gene_Chr7-6Mb_reg_1022, 9816 candidate probes\n",
      "--- 5708 probes passed check_dic selection.\n",
      "finish in 0.354s, 70 probes kept.\n",
      "-- check region:1022 7:49109999-49114999_strand_+_gene_Chr7-6Mb_reg_1023, 9910 candidate probes\n",
      "--- 3514 probes passed check_dic selection.\n",
      "finish in 0.279s, 67 probes kept.\n",
      "-- check region:1023 7:49114999-49119999_strand_+_gene_Chr7-6Mb_reg_1024, 9726 candidate probes\n",
      "--- 6480 probes passed check_dic selection.\n",
      "finish in 0.380s, 73 probes kept.\n",
      "-- check region:1024 7:49119999-49124999_strand_+_gene_Chr7-6Mb_reg_1025, 9898 candidate probes\n",
      "--- 6252 probes passed check_dic selection.\n",
      "finish in 0.371s, 76 probes kept.\n",
      "-- check region:1025 7:49124999-49129999_strand_+_gene_Chr7-6Mb_reg_1026, 9894 candidate probes\n",
      "--- 7994 probes passed check_dic selection.\n",
      "finish in 0.437s, 82 probes kept.\n",
      "-- check region:1026 7:49129999-49134999_strand_+_gene_Chr7-6Mb_reg_1027, 9874 candidate probes\n",
      "--- 8146 probes passed check_dic selection.\n",
      "finish in 0.448s, 81 probes kept.\n",
      "-- check region:1027 7:49134999-49139999_strand_+_gene_Chr7-6Mb_reg_1028, 9770 candidate probes\n",
      "--- 6318 probes passed check_dic selection.\n",
      "finish in 0.377s, 76 probes kept.\n",
      "-- check region:1028 7:49139999-49144999_strand_+_gene_Chr7-6Mb_reg_1029, 9862 candidate probes\n",
      "--- 6106 probes passed check_dic selection.\n",
      "finish in 0.368s, 75 probes kept.\n",
      "-- check region:1029 7:49144999-49149999_strand_+_gene_Chr7-6Mb_reg_1030, 9904 candidate probes\n",
      "--- 6054 probes passed check_dic selection.\n",
      "finish in 0.364s, 75 probes kept.\n",
      "-- check region:1030 7:49149999-49154999_strand_+_gene_Chr7-6Mb_reg_1031, 9918 candidate probes\n",
      "--- 7432 probes passed check_dic selection.\n",
      "finish in 0.413s, 83 probes kept.\n",
      "-- check region:1031 7:49154999-49159999_strand_+_gene_Chr7-6Mb_reg_1032, 9742 candidate probes\n",
      "--- 6182 probes passed check_dic selection.\n",
      "finish in 0.368s, 69 probes kept.\n",
      "-- check region:1032 7:49159999-49164999_strand_+_gene_Chr7-6Mb_reg_1033, 9918 candidate probes\n",
      "--- 7828 probes passed check_dic selection.\n",
      "finish in 0.427s, 85 probes kept.\n",
      "-- check region:1033 7:49164999-49169999_strand_+_gene_Chr7-6Mb_reg_1034, 9738 candidate probes\n",
      "--- 6536 probes passed check_dic selection.\n",
      "finish in 0.381s, 77 probes kept.\n",
      "-- check region:1034 7:49169999-49174999_strand_+_gene_Chr7-6Mb_reg_1035, 9918 candidate probes\n",
      "--- 6612 probes passed check_dic selection.\n",
      "finish in 0.388s, 78 probes kept.\n",
      "-- check region:1035 7:49174999-49179999_strand_+_gene_Chr7-6Mb_reg_1036, 9918 candidate probes\n",
      "--- 7026 probes passed check_dic selection.\n",
      "finish in 0.402s, 80 probes kept.\n",
      "-- check region:1036 7:49179999-49184999_strand_+_gene_Chr7-6Mb_reg_1037, 9844 candidate probes\n",
      "--- 7470 probes passed check_dic selection.\n",
      "finish in 0.418s, 82 probes kept.\n",
      "-- check region:1037 7:49184999-49189999_strand_+_gene_Chr7-6Mb_reg_1038, 9918 candidate probes\n",
      "--- 7962 probes passed check_dic selection.\n",
      "finish in 0.432s, 83 probes kept.\n",
      "-- check region:1038 7:49189999-49194999_strand_+_gene_Chr7-6Mb_reg_1039, 9902 candidate probes\n",
      "--- 7382 probes passed check_dic selection.\n",
      "finish in 0.414s, 83 probes kept.\n",
      "-- check region:1039 7:49194999-49199999_strand_+_gene_Chr7-6Mb_reg_1040, 9918 candidate probes\n",
      "--- 7510 probes passed check_dic selection.\n",
      "finish in 0.417s, 85 probes kept.\n",
      "-- check region:1040 7:49199999-49204999_strand_+_gene_Chr7-6Mb_reg_1041, 9918 candidate probes\n",
      "--- 8526 probes passed check_dic selection.\n",
      "finish in 0.454s, 89 probes kept.\n",
      "-- check region:1041 7:49204999-49209999_strand_+_gene_Chr7-6Mb_reg_1042, 9862 candidate probes\n",
      "--- 7286 probes passed check_dic selection.\n",
      "finish in 0.408s, 77 probes kept.\n",
      "-- check region:1042 7:49209999-49214999_strand_+_gene_Chr7-6Mb_reg_1043, 9814 candidate probes\n",
      "--- 7376 probes passed check_dic selection.\n",
      "finish in 0.410s, 81 probes kept.\n",
      "-- check region:1043 7:49214999-49219999_strand_+_gene_Chr7-6Mb_reg_1044, 9918 candidate probes\n",
      "--- 7248 probes passed check_dic selection.\n",
      "finish in 0.404s, 75 probes kept.\n",
      "-- check region:1044 7:49219999-49224999_strand_+_gene_Chr7-6Mb_reg_1045, 9918 candidate probes\n",
      "--- 7426 probes passed check_dic selection.\n",
      "finish in 0.420s, 79 probes kept.\n",
      "-- check region:1045 7:49224999-49229999_strand_+_gene_Chr7-6Mb_reg_1046, 9896 candidate probes\n",
      "--- 4320 probes passed check_dic selection.\n",
      "finish in 0.312s, 57 probes kept.\n",
      "-- check region:1046 7:49229999-49234999_strand_+_gene_Chr7-6Mb_reg_1047, 9918 candidate probes\n",
      "--- 6232 probes passed check_dic selection.\n",
      "finish in 0.372s, 75 probes kept.\n",
      "-- check region:1047 7:49234999-49239999_strand_+_gene_Chr7-6Mb_reg_1048, 9872 candidate probes\n",
      "--- 5184 probes passed check_dic selection.\n",
      "finish in 0.336s, 61 probes kept.\n",
      "-- check region:1048 7:49239999-49244999_strand_+_gene_Chr7-6Mb_reg_1049, 9918 candidate probes\n",
      "--- 7208 probes passed check_dic selection.\n",
      "finish in 0.405s, 85 probes kept.\n",
      "-- check region:1049 7:49244999-49249999_strand_+_gene_Chr7-6Mb_reg_1050, 9918 candidate probes\n",
      "--- 8592 probes passed check_dic selection.\n",
      "finish in 0.454s, 86 probes kept.\n",
      "-- check region:1050 7:49249999-49254999_strand_+_gene_Chr7-6Mb_reg_1051, 9918 candidate probes\n",
      "--- 7244 probes passed check_dic selection.\n",
      "finish in 0.409s, 82 probes kept.\n",
      "-- check region:1051 7:49254999-49259999_strand_+_gene_Chr7-6Mb_reg_1052, 9918 candidate probes\n",
      "--- 7866 probes passed check_dic selection.\n",
      "finish in 0.432s, 82 probes kept.\n",
      "-- check region:1052 7:49259999-49264999_strand_+_gene_Chr7-6Mb_reg_1053, 9918 candidate probes\n",
      "--- 8150 probes passed check_dic selection.\n",
      "finish in 0.437s, 86 probes kept.\n",
      "-- check region:1053 7:49264999-49269999_strand_+_gene_Chr7-6Mb_reg_1054, 9918 candidate probes\n",
      "--- 7102 probes passed check_dic selection.\n",
      "finish in 0.403s, 81 probes kept.\n",
      "-- check region:1054 7:49269999-49274999_strand_+_gene_Chr7-6Mb_reg_1055, 9770 candidate probes\n",
      "--- 6874 probes passed check_dic selection.\n",
      "finish in 0.393s, 80 probes kept.\n",
      "-- check region:1055 7:49274999-49279999_strand_+_gene_Chr7-6Mb_reg_1056, 9918 candidate probes\n",
      "--- 6850 probes passed check_dic selection.\n",
      "finish in 0.393s, 78 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1056 7:49279999-49284999_strand_+_gene_Chr7-6Mb_reg_1057, 9804 candidate probes\n",
      "--- 7228 probes passed check_dic selection.\n",
      "finish in 0.405s, 79 probes kept.\n",
      "-- check region:1057 7:49284999-49289999_strand_+_gene_Chr7-6Mb_reg_1058, 9896 candidate probes\n",
      "--- 6420 probes passed check_dic selection.\n",
      "finish in 0.378s, 82 probes kept.\n",
      "-- check region:1058 7:49289999-49294999_strand_+_gene_Chr7-6Mb_reg_1059, 9872 candidate probes\n",
      "--- 7268 probes passed check_dic selection.\n",
      "finish in 0.407s, 79 probes kept.\n",
      "-- check region:1059 7:49294999-49299999_strand_+_gene_Chr7-6Mb_reg_1060, 9870 candidate probes\n",
      "--- 7194 probes passed check_dic selection.\n",
      "finish in 0.405s, 80 probes kept.\n",
      "-- check region:1060 7:49299999-49304999_strand_+_gene_Chr7-6Mb_reg_1061, 9850 candidate probes\n",
      "--- 7596 probes passed check_dic selection.\n",
      "finish in 0.420s, 83 probes kept.\n",
      "-- check region:1061 7:49304999-49309999_strand_+_gene_Chr7-6Mb_reg_1062, 9490 candidate probes\n",
      "--- 6320 probes passed check_dic selection.\n",
      "finish in 0.371s, 73 probes kept.\n",
      "-- check region:1062 7:49309999-49314999_strand_+_gene_Chr7-6Mb_reg_1063, 9878 candidate probes\n",
      "--- 6290 probes passed check_dic selection.\n",
      "finish in 0.372s, 74 probes kept.\n",
      "-- check region:1063 7:49314999-49319999_strand_+_gene_Chr7-6Mb_reg_1064, 9810 candidate probes\n",
      "--- 6174 probes passed check_dic selection.\n",
      "finish in 0.368s, 71 probes kept.\n",
      "-- check region:1064 7:49319999-49324999_strand_+_gene_Chr7-6Mb_reg_1065, 9888 candidate probes\n",
      "--- 6948 probes passed check_dic selection.\n",
      "finish in 0.399s, 80 probes kept.\n",
      "-- check region:1065 7:49324999-49329999_strand_+_gene_Chr7-6Mb_reg_1066, 9918 candidate probes\n",
      "--- 8166 probes passed check_dic selection.\n",
      "finish in 0.443s, 80 probes kept.\n",
      "-- check region:1066 7:49329999-49334999_strand_+_gene_Chr7-6Mb_reg_1067, 9918 candidate probes\n",
      "--- 7728 probes passed check_dic selection.\n",
      "finish in 0.426s, 81 probes kept.\n",
      "-- check region:1067 7:49334999-49339999_strand_+_gene_Chr7-6Mb_reg_1068, 9842 candidate probes\n",
      "--- 5232 probes passed check_dic selection.\n",
      "finish in 0.338s, 67 probes kept.\n",
      "-- check region:1068 7:49339999-49344999_strand_+_gene_Chr7-6Mb_reg_1069, 9906 candidate probes\n",
      "--- 6922 probes passed check_dic selection.\n",
      "finish in 0.395s, 78 probes kept.\n",
      "-- check region:1069 7:49344999-49349999_strand_+_gene_Chr7-6Mb_reg_1070, 9716 candidate probes\n",
      "--- 6246 probes passed check_dic selection.\n",
      "finish in 0.368s, 74 probes kept.\n",
      "-- check region:1070 7:49349999-49354999_strand_+_gene_Chr7-6Mb_reg_1071, 9750 candidate probes\n",
      "--- 6392 probes passed check_dic selection.\n",
      "finish in 0.377s, 69 probes kept.\n",
      "-- check region:1071 7:49354999-49359999_strand_+_gene_Chr7-6Mb_reg_1072, 9910 candidate probes\n",
      "--- 5190 probes passed check_dic selection.\n",
      "finish in 0.340s, 61 probes kept.\n",
      "-- check region:1072 7:49359999-49364999_strand_+_gene_Chr7-6Mb_reg_1073, 9918 candidate probes\n",
      "--- 7034 probes passed check_dic selection.\n",
      "finish in 0.400s, 79 probes kept.\n",
      "-- check region:1073 7:49364999-49369999_strand_+_gene_Chr7-6Mb_reg_1074, 9918 candidate probes\n",
      "--- 6648 probes passed check_dic selection.\n",
      "finish in 0.386s, 79 probes kept.\n",
      "-- check region:1074 7:49369999-49374999_strand_+_gene_Chr7-6Mb_reg_1075, 9918 candidate probes\n",
      "--- 7548 probes passed check_dic selection.\n",
      "finish in 0.420s, 81 probes kept.\n",
      "-- check region:1075 7:49374999-49379999_strand_+_gene_Chr7-6Mb_reg_1076, 9912 candidate probes\n",
      "--- 6326 probes passed check_dic selection.\n",
      "finish in 0.377s, 76 probes kept.\n",
      "-- check region:1076 7:49379999-49384999_strand_+_gene_Chr7-6Mb_reg_1077, 9918 candidate probes\n",
      "--- 6750 probes passed check_dic selection.\n",
      "finish in 0.388s, 80 probes kept.\n",
      "-- check region:1077 7:49384999-49389999_strand_+_gene_Chr7-6Mb_reg_1078, 9788 candidate probes\n",
      "--- 4894 probes passed check_dic selection.\n",
      "finish in 0.324s, 59 probes kept.\n",
      "-- check region:1078 7:49389999-49394999_strand_+_gene_Chr7-6Mb_reg_1079, 9918 candidate probes\n",
      "--- 6624 probes passed check_dic selection.\n",
      "finish in 0.388s, 75 probes kept.\n",
      "-- check region:1079 7:49394999-49399999_strand_+_gene_Chr7-6Mb_reg_1080, 9918 candidate probes\n",
      "--- 6128 probes passed check_dic selection.\n",
      "finish in 0.371s, 70 probes kept.\n",
      "-- check region:1080 7:49399999-49404999_strand_+_gene_Chr7-6Mb_reg_1081, 9872 candidate probes\n",
      "--- 5960 probes passed check_dic selection.\n",
      "finish in 0.362s, 72 probes kept.\n",
      "-- check region:1081 7:49404999-49409999_strand_+_gene_Chr7-6Mb_reg_1082, 9482 candidate probes\n",
      "--- 5706 probes passed check_dic selection.\n",
      "finish in 0.348s, 63 probes kept.\n",
      "-- check region:1082 7:49409999-49414999_strand_+_gene_Chr7-6Mb_reg_1083, 9918 candidate probes\n",
      "--- 7182 probes passed check_dic selection.\n",
      "finish in 0.408s, 88 probes kept.\n",
      "-- check region:1083 7:49414999-49419999_strand_+_gene_Chr7-6Mb_reg_1084, 9738 candidate probes\n",
      "--- 6916 probes passed check_dic selection.\n",
      "finish in 0.392s, 76 probes kept.\n",
      "-- check region:1084 7:49419999-49424999_strand_+_gene_Chr7-6Mb_reg_1085, 9918 candidate probes\n",
      "--- 6612 probes passed check_dic selection.\n",
      "finish in 0.385s, 76 probes kept.\n",
      "-- check region:1085 7:49424999-49429999_strand_+_gene_Chr7-6Mb_reg_1086, 9900 candidate probes\n",
      "--- 7280 probes passed check_dic selection.\n",
      "finish in 0.409s, 77 probes kept.\n",
      "-- check region:1086 7:49429999-49434999_strand_+_gene_Chr7-6Mb_reg_1087, 9508 candidate probes\n",
      "--- 5198 probes passed check_dic selection.\n",
      "finish in 0.330s, 62 probes kept.\n",
      "-- check region:1087 7:49434999-49439999_strand_+_gene_Chr7-6Mb_reg_1088, 9886 candidate probes\n",
      "--- 6696 probes passed check_dic selection.\n",
      "finish in 0.388s, 78 probes kept.\n",
      "-- check region:1088 7:49439999-49444999_strand_+_gene_Chr7-6Mb_reg_1089, 9918 candidate probes\n",
      "--- 6828 probes passed check_dic selection.\n",
      "finish in 0.396s, 77 probes kept.\n",
      "-- check region:1089 7:49444999-49449999_strand_+_gene_Chr7-6Mb_reg_1090, 9814 candidate probes\n",
      "--- 5368 probes passed check_dic selection.\n",
      "finish in 0.341s, 67 probes kept.\n",
      "-- check region:1090 7:49449999-49454999_strand_+_gene_Chr7-6Mb_reg_1091, 9866 candidate probes\n",
      "--- 6866 probes passed check_dic selection.\n",
      "finish in 0.393s, 79 probes kept.\n",
      "-- check region:1091 7:49454999-49459999_strand_+_gene_Chr7-6Mb_reg_1092, 9758 candidate probes\n",
      "--- 4888 probes passed check_dic selection.\n",
      "finish in 0.323s, 60 probes kept.\n",
      "-- check region:1092 7:49459999-49464999_strand_+_gene_Chr7-6Mb_reg_1093, 9752 candidate probes\n",
      "--- 5532 probes passed check_dic selection.\n",
      "finish in 0.347s, 66 probes kept.\n",
      "-- check region:1093 7:49464999-49469999_strand_+_gene_Chr7-6Mb_reg_1094, 9916 candidate probes\n",
      "--- 5964 probes passed check_dic selection.\n",
      "finish in 0.362s, 70 probes kept.\n",
      "-- check region:1094 7:49469999-49474999_strand_+_gene_Chr7-6Mb_reg_1095, 9918 candidate probes\n",
      "--- 8360 probes passed check_dic selection.\n",
      "finish in 0.451s, 88 probes kept.\n",
      "-- check region:1095 7:49474999-49479999_strand_+_gene_Chr7-6Mb_reg_1096, 9856 candidate probes\n",
      "--- 7184 probes passed check_dic selection.\n",
      "finish in 0.405s, 81 probes kept.\n",
      "-- check region:1096 7:49479999-49484999_strand_+_gene_Chr7-6Mb_reg_1097, 9886 candidate probes\n",
      "--- 4982 probes passed check_dic selection.\n",
      "finish in 0.328s, 68 probes kept.\n",
      "-- check region:1097 7:49484999-49489999_strand_+_gene_Chr7-6Mb_reg_1098, 9792 candidate probes\n",
      "--- 5574 probes passed check_dic selection.\n",
      "finish in 0.347s, 63 probes kept.\n",
      "-- check region:1098 7:49489999-49494999_strand_+_gene_Chr7-6Mb_reg_1099, 9832 candidate probes\n",
      "--- 6824 probes passed check_dic selection.\n",
      "finish in 0.393s, 74 probes kept.\n",
      "-- check region:1099 7:49494999-49499999_strand_+_gene_Chr7-6Mb_reg_1100, 9814 candidate probes\n",
      "--- 6336 probes passed check_dic selection.\n",
      "finish in 0.378s, 72 probes kept.\n",
      "-- check region:1100 7:49499999-49504999_strand_+_gene_Chr7-6Mb_reg_1101, 9918 candidate probes\n",
      "--- 7228 probes passed check_dic selection.\n",
      "finish in 0.408s, 79 probes kept.\n",
      "-- check region:1101 7:49504999-49509999_strand_+_gene_Chr7-6Mb_reg_1102, 9914 candidate probes\n",
      "--- 6706 probes passed check_dic selection.\n",
      "finish in 0.391s, 76 probes kept.\n",
      "-- check region:1102 7:49509999-49514999_strand_+_gene_Chr7-6Mb_reg_1103, 9896 candidate probes\n",
      "--- 5688 probes passed check_dic selection.\n",
      "finish in 0.357s, 76 probes kept.\n",
      "-- check region:1103 7:49514999-49519999_strand_+_gene_Chr7-6Mb_reg_1104, 9914 candidate probes\n",
      "--- 6316 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.377s, 80 probes kept.\n",
      "-- check region:1104 7:49519999-49524999_strand_+_gene_Chr7-6Mb_reg_1105, 9848 candidate probes\n",
      "--- 6560 probes passed check_dic selection.\n",
      "finish in 0.383s, 73 probes kept.\n",
      "-- check region:1105 7:49524999-49529999_strand_+_gene_Chr7-6Mb_reg_1106, 9904 candidate probes\n",
      "--- 6344 probes passed check_dic selection.\n",
      "finish in 0.377s, 75 probes kept.\n",
      "-- check region:1106 7:49529999-49534999_strand_+_gene_Chr7-6Mb_reg_1107, 9824 candidate probes\n",
      "--- 4814 probes passed check_dic selection.\n",
      "finish in 0.325s, 63 probes kept.\n",
      "-- check region:1107 7:49534999-49539999_strand_+_gene_Chr7-6Mb_reg_1108, 9832 candidate probes\n",
      "--- 6388 probes passed check_dic selection.\n",
      "finish in 0.378s, 73 probes kept.\n",
      "-- check region:1108 7:49539999-49544999_strand_+_gene_Chr7-6Mb_reg_1109, 9910 candidate probes\n",
      "--- 7206 probes passed check_dic selection.\n",
      "finish in 0.405s, 78 probes kept.\n",
      "-- check region:1109 7:49544999-49549999_strand_+_gene_Chr7-6Mb_reg_1110, 9754 candidate probes\n",
      "--- 5896 probes passed check_dic selection.\n",
      "finish in 0.360s, 71 probes kept.\n",
      "-- check region:1110 7:49549999-49554999_strand_+_gene_Chr7-6Mb_reg_1111, 9918 candidate probes\n",
      "--- 8000 probes passed check_dic selection.\n",
      "finish in 0.435s, 87 probes kept.\n",
      "-- check region:1111 7:49554999-49559999_strand_+_gene_Chr7-6Mb_reg_1112, 9918 candidate probes\n",
      "--- 7312 probes passed check_dic selection.\n",
      "finish in 0.410s, 81 probes kept.\n",
      "-- check region:1112 7:49559999-49564999_strand_+_gene_Chr7-6Mb_reg_1113, 9720 candidate probes\n",
      "--- 6346 probes passed check_dic selection.\n",
      "finish in 0.374s, 75 probes kept.\n",
      "-- check region:1113 7:49564999-49569999_strand_+_gene_Chr7-6Mb_reg_1114, 9898 candidate probes\n",
      "--- 7106 probes passed check_dic selection.\n",
      "finish in 0.406s, 79 probes kept.\n",
      "-- check region:1114 7:49569999-49574999_strand_+_gene_Chr7-6Mb_reg_1115, 9886 candidate probes\n",
      "--- 7182 probes passed check_dic selection.\n",
      "finish in 0.404s, 78 probes kept.\n",
      "-- check region:1115 7:49574999-49579999_strand_+_gene_Chr7-6Mb_reg_1116, 9906 candidate probes\n",
      "--- 7416 probes passed check_dic selection.\n",
      "finish in 0.414s, 80 probes kept.\n",
      "-- check region:1116 7:49579999-49584999_strand_+_gene_Chr7-6Mb_reg_1117, 9918 candidate probes\n",
      "--- 7204 probes passed check_dic selection.\n",
      "finish in 0.406s, 82 probes kept.\n",
      "-- check region:1117 7:49584999-49589999_strand_+_gene_Chr7-6Mb_reg_1118, 9826 candidate probes\n",
      "--- 6796 probes passed check_dic selection.\n",
      "finish in 0.391s, 75 probes kept.\n",
      "-- check region:1118 7:49589999-49594999_strand_+_gene_Chr7-6Mb_reg_1119, 9894 candidate probes\n",
      "--- 5714 probes passed check_dic selection.\n",
      "finish in 0.358s, 67 probes kept.\n",
      "-- check region:1119 7:49594999-49599999_strand_+_gene_Chr7-6Mb_reg_1120, 9918 candidate probes\n",
      "--- 7824 probes passed check_dic selection.\n",
      "finish in 0.431s, 82 probes kept.\n",
      "-- check region:1120 7:49599999-49604999_strand_+_gene_Chr7-6Mb_reg_1121, 9848 candidate probes\n",
      "--- 7734 probes passed check_dic selection.\n",
      "finish in 0.425s, 82 probes kept.\n",
      "-- check region:1121 7:49604999-49609999_strand_+_gene_Chr7-6Mb_reg_1122, 9866 candidate probes\n",
      "--- 7276 probes passed check_dic selection.\n",
      "finish in 0.407s, 78 probes kept.\n",
      "-- check region:1122 7:49609999-49614999_strand_+_gene_Chr7-6Mb_reg_1123, 9918 candidate probes\n",
      "--- 4808 probes passed check_dic selection.\n",
      "finish in 0.322s, 62 probes kept.\n",
      "-- check region:1123 7:49614999-49619999_strand_+_gene_Chr7-6Mb_reg_1124, 9782 candidate probes\n",
      "--- 6698 probes passed check_dic selection.\n",
      "finish in 0.389s, 76 probes kept.\n",
      "-- check region:1124 7:49619999-49624999_strand_+_gene_Chr7-6Mb_reg_1125, 9860 candidate probes\n",
      "--- 4606 probes passed check_dic selection.\n",
      "finish in 0.316s, 59 probes kept.\n",
      "-- check region:1125 7:49624999-49629999_strand_+_gene_Chr7-6Mb_reg_1126, 9906 candidate probes\n",
      "--- 4428 probes passed check_dic selection.\n",
      "finish in 0.310s, 56 probes kept.\n",
      "-- check region:1126 7:49629999-49634999_strand_+_gene_Chr7-6Mb_reg_1127, 9856 candidate probes\n",
      "--- 6710 probes passed check_dic selection.\n",
      "finish in 0.387s, 75 probes kept.\n",
      "-- check region:1127 7:49634999-49639999_strand_+_gene_Chr7-6Mb_reg_1128, 9886 candidate probes\n",
      "--- 7348 probes passed check_dic selection.\n",
      "finish in 0.411s, 78 probes kept.\n",
      "-- check region:1128 7:49639999-49644999_strand_+_gene_Chr7-6Mb_reg_1129, 9862 candidate probes\n",
      "--- 5950 probes passed check_dic selection.\n",
      "finish in 0.363s, 75 probes kept.\n",
      "-- check region:1129 7:49644999-49649999_strand_+_gene_Chr7-6Mb_reg_1130, 9800 candidate probes\n",
      "--- 4846 probes passed check_dic selection.\n",
      "finish in 0.323s, 65 probes kept.\n",
      "-- check region:1130 7:49649999-49654999_strand_+_gene_Chr7-6Mb_reg_1131, 9876 candidate probes\n",
      "--- 4122 probes passed check_dic selection.\n",
      "finish in 0.299s, 52 probes kept.\n",
      "-- check region:1131 7:49654999-49659999_strand_+_gene_Chr7-6Mb_reg_1132, 9904 candidate probes\n",
      "--- 6026 probes passed check_dic selection.\n",
      "finish in 0.367s, 78 probes kept.\n",
      "-- check region:1132 7:49659999-49664999_strand_+_gene_Chr7-6Mb_reg_1133, 9904 candidate probes\n",
      "--- 6094 probes passed check_dic selection.\n",
      "finish in 0.370s, 67 probes kept.\n",
      "-- check region:1133 7:49664999-49669999_strand_+_gene_Chr7-6Mb_reg_1134, 9828 candidate probes\n",
      "--- 5400 probes passed check_dic selection.\n",
      "finish in 0.343s, 71 probes kept.\n",
      "-- check region:1134 7:49669999-49674999_strand_+_gene_Chr7-6Mb_reg_1135, 9918 candidate probes\n",
      "--- 3856 probes passed check_dic selection.\n",
      "finish in 0.289s, 54 probes kept.\n",
      "-- check region:1135 7:49674999-49679999_strand_+_gene_Chr7-6Mb_reg_1136, 9632 candidate probes\n",
      "--- 6114 probes passed check_dic selection.\n",
      "finish in 0.360s, 72 probes kept.\n",
      "-- check region:1136 7:49679999-49684999_strand_+_gene_Chr7-6Mb_reg_1137, 9918 candidate probes\n",
      "--- 6502 probes passed check_dic selection.\n",
      "finish in 0.382s, 76 probes kept.\n",
      "-- check region:1137 7:49684999-49689999_strand_+_gene_Chr7-6Mb_reg_1138, 9918 candidate probes\n",
      "--- 6014 probes passed check_dic selection.\n",
      "finish in 0.363s, 74 probes kept.\n",
      "-- check region:1138 7:49689999-49694999_strand_+_gene_Chr7-6Mb_reg_1139, 9338 candidate probes\n",
      "--- 5204 probes passed check_dic selection.\n",
      "finish in 0.327s, 64 probes kept.\n",
      "-- check region:1139 7:49694999-49699999_strand_+_gene_Chr7-6Mb_reg_1140, 9916 candidate probes\n",
      "--- 6068 probes passed check_dic selection.\n",
      "finish in 0.367s, 71 probes kept.\n",
      "-- check region:1140 7:49699999-49704999_strand_+_gene_Chr7-6Mb_reg_1141, 9918 candidate probes\n",
      "--- 7144 probes passed check_dic selection.\n",
      "finish in 0.403s, 81 probes kept.\n",
      "-- check region:1141 7:49704999-49709999_strand_+_gene_Chr7-6Mb_reg_1142, 9900 candidate probes\n",
      "--- 5478 probes passed check_dic selection.\n",
      "finish in 0.344s, 67 probes kept.\n",
      "-- check region:1142 7:49709999-49714999_strand_+_gene_Chr7-6Mb_reg_1143, 9910 candidate probes\n",
      "--- 3582 probes passed check_dic selection.\n",
      "finish in 0.280s, 58 probes kept.\n",
      "-- check region:1143 7:49714999-49719999_strand_+_gene_Chr7-6Mb_reg_1144, 9786 candidate probes\n",
      "--- 4220 probes passed check_dic selection.\n",
      "finish in 0.303s, 48 probes kept.\n",
      "-- check region:1144 7:49719999-49724999_strand_+_gene_Chr7-6Mb_reg_1145, 9842 candidate probes\n",
      "--- 5596 probes passed check_dic selection.\n",
      "finish in 0.347s, 68 probes kept.\n",
      "-- check region:1145 7:49724999-49729999_strand_+_gene_Chr7-6Mb_reg_1146, 9906 candidate probes\n",
      "--- 3648 probes passed check_dic selection.\n",
      "finish in 0.283s, 49 probes kept.\n",
      "-- check region:1146 7:49729999-49734999_strand_+_gene_Chr7-6Mb_reg_1147, 9782 candidate probes\n",
      "--- 3890 probes passed check_dic selection.\n",
      "finish in 0.293s, 61 probes kept.\n",
      "-- check region:1147 7:49734999-49739999_strand_+_gene_Chr7-6Mb_reg_1148, 9888 candidate probes\n",
      "--- 2534 probes passed check_dic selection.\n",
      "finish in 0.247s, 42 probes kept.\n",
      "-- check region:1148 7:49739999-49744999_strand_+_gene_Chr7-6Mb_reg_1149, 9910 candidate probes\n",
      "--- 5918 probes passed check_dic selection.\n",
      "finish in 0.361s, 71 probes kept.\n",
      "-- check region:1149 7:49744999-49749999_strand_+_gene_Chr7-6Mb_reg_1150, 9874 candidate probes\n",
      "--- 5278 probes passed check_dic selection.\n",
      "finish in 0.338s, 59 probes kept.\n",
      "-- check region:1150 7:49749999-49754999_strand_+_gene_Chr7-6Mb_reg_1151, 9918 candidate probes\n",
      "--- 5738 probes passed check_dic selection.\n",
      "finish in 0.354s, 65 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1151 7:49754999-49759999_strand_+_gene_Chr7-6Mb_reg_1152, 9908 candidate probes\n",
      "--- 6934 probes passed check_dic selection.\n",
      "finish in 0.396s, 77 probes kept.\n",
      "-- check region:1152 7:49759999-49764999_strand_+_gene_Chr7-6Mb_reg_1153, 9918 candidate probes\n",
      "--- 6228 probes passed check_dic selection.\n",
      "finish in 0.375s, 76 probes kept.\n",
      "-- check region:1153 7:49764999-49769999_strand_+_gene_Chr7-6Mb_reg_1154, 9918 candidate probes\n",
      "--- 5974 probes passed check_dic selection.\n",
      "finish in 0.363s, 77 probes kept.\n",
      "-- check region:1154 7:49769999-49774999_strand_+_gene_Chr7-6Mb_reg_1155, 9918 candidate probes\n",
      "--- 6852 probes passed check_dic selection.\n",
      "finish in 0.399s, 81 probes kept.\n",
      "-- check region:1155 7:49774999-49779999_strand_+_gene_Chr7-6Mb_reg_1156, 9918 candidate probes\n",
      "--- 6466 probes passed check_dic selection.\n",
      "finish in 0.382s, 77 probes kept.\n",
      "-- check region:1156 7:49779999-49784999_strand_+_gene_Chr7-6Mb_reg_1157, 9916 candidate probes\n",
      "--- 5160 probes passed check_dic selection.\n",
      "finish in 0.332s, 69 probes kept.\n",
      "-- check region:1157 7:49784999-49789999_strand_+_gene_Chr7-6Mb_reg_1158, 9918 candidate probes\n",
      "--- 5874 probes passed check_dic selection.\n",
      "finish in 0.364s, 68 probes kept.\n",
      "-- check region:1158 7:49789999-49794999_strand_+_gene_Chr7-6Mb_reg_1159, 9918 candidate probes\n",
      "--- 5038 probes passed check_dic selection.\n",
      "finish in 0.330s, 64 probes kept.\n",
      "-- check region:1159 7:49794999-49799999_strand_+_gene_Chr7-6Mb_reg_1160, 9852 candidate probes\n",
      "--- 5182 probes passed check_dic selection.\n",
      "finish in 0.333s, 69 probes kept.\n",
      "-- check region:1160 7:49799999-49804999_strand_+_gene_Chr7-6Mb_reg_1161, 9918 candidate probes\n",
      "--- 5682 probes passed check_dic selection.\n",
      "finish in 0.352s, 72 probes kept.\n",
      "-- check region:1161 7:49804999-49809999_strand_+_gene_Chr7-6Mb_reg_1162, 9918 candidate probes\n",
      "--- 5786 probes passed check_dic selection.\n",
      "finish in 0.355s, 75 probes kept.\n",
      "-- check region:1162 7:49809999-49814999_strand_+_gene_Chr7-6Mb_reg_1163, 9918 candidate probes\n",
      "--- 5176 probes passed check_dic selection.\n",
      "finish in 0.332s, 70 probes kept.\n",
      "-- check region:1163 7:49814999-49819999_strand_+_gene_Chr7-6Mb_reg_1164, 9918 candidate probes\n",
      "--- 5202 probes passed check_dic selection.\n",
      "finish in 0.333s, 65 probes kept.\n",
      "-- check region:1164 7:49819999-49824999_strand_+_gene_Chr7-6Mb_reg_1165, 9918 candidate probes\n",
      "--- 6156 probes passed check_dic selection.\n",
      "finish in 0.368s, 82 probes kept.\n",
      "-- check region:1165 7:49824999-49829999_strand_+_gene_Chr7-6Mb_reg_1166, 9918 candidate probes\n",
      "--- 6170 probes passed check_dic selection.\n",
      "finish in 0.368s, 80 probes kept.\n",
      "-- check region:1166 7:49829999-49834999_strand_+_gene_Chr7-6Mb_reg_1167, 9918 candidate probes\n",
      "--- 5838 probes passed check_dic selection.\n",
      "finish in 0.356s, 67 probes kept.\n",
      "-- check region:1167 7:49834999-49839999_strand_+_gene_Chr7-6Mb_reg_1168, 9906 candidate probes\n",
      "--- 5794 probes passed check_dic selection.\n",
      "finish in 0.357s, 74 probes kept.\n",
      "-- check region:1168 7:49839999-49844999_strand_+_gene_Chr7-6Mb_reg_1169, 9918 candidate probes\n",
      "--- 5620 probes passed check_dic selection.\n",
      "finish in 0.351s, 70 probes kept.\n",
      "-- check region:1169 7:49844999-49849999_strand_+_gene_Chr7-6Mb_reg_1170, 9918 candidate probes\n",
      "--- 4440 probes passed check_dic selection.\n",
      "finish in 0.306s, 66 probes kept.\n",
      "-- check region:1170 7:49849999-49854999_strand_+_gene_Chr7-6Mb_reg_1171, 9908 candidate probes\n",
      "--- 4402 probes passed check_dic selection.\n",
      "finish in 0.306s, 67 probes kept.\n",
      "-- check region:1171 7:49854999-49859999_strand_+_gene_Chr7-6Mb_reg_1172, 9918 candidate probes\n",
      "--- 5832 probes passed check_dic selection.\n",
      "finish in 0.357s, 72 probes kept.\n",
      "-- check region:1172 7:49859999-49864999_strand_+_gene_Chr7-6Mb_reg_1173, 9904 candidate probes\n",
      "--- 5164 probes passed check_dic selection.\n",
      "finish in 0.335s, 73 probes kept.\n",
      "-- check region:1173 7:49864999-49869999_strand_+_gene_Chr7-6Mb_reg_1174, 9808 candidate probes\n",
      "--- 6562 probes passed check_dic selection.\n",
      "finish in 0.385s, 71 probes kept.\n",
      "-- check region:1174 7:49869999-49874999_strand_+_gene_Chr7-6Mb_reg_1175, 9918 candidate probes\n",
      "--- 5464 probes passed check_dic selection.\n",
      "finish in 0.347s, 72 probes kept.\n",
      "-- check region:1175 7:49874999-49879999_strand_+_gene_Chr7-6Mb_reg_1176, 9884 candidate probes\n",
      "--- 6608 probes passed check_dic selection.\n",
      "finish in 0.388s, 68 probes kept.\n",
      "-- check region:1176 7:49879999-49884999_strand_+_gene_Chr7-6Mb_reg_1177, 9904 candidate probes\n",
      "--- 4000 probes passed check_dic selection.\n",
      "finish in 0.296s, 55 probes kept.\n",
      "-- check region:1177 7:49884999-49889999_strand_+_gene_Chr7-6Mb_reg_1178, 9918 candidate probes\n",
      "--- 6328 probes passed check_dic selection.\n",
      "finish in 0.375s, 76 probes kept.\n",
      "-- check region:1178 7:49889999-49894999_strand_+_gene_Chr7-6Mb_reg_1179, 9902 candidate probes\n",
      "--- 5006 probes passed check_dic selection.\n",
      "finish in 0.330s, 64 probes kept.\n",
      "-- check region:1179 7:49894999-49899999_strand_+_gene_Chr7-6Mb_reg_1180, 9910 candidate probes\n",
      "--- 4834 probes passed check_dic selection.\n",
      "finish in 0.326s, 68 probes kept.\n",
      "-- check region:1180 7:49899999-49904999_strand_+_gene_Chr7-6Mb_reg_1181, 9904 candidate probes\n",
      "--- 6474 probes passed check_dic selection.\n",
      "finish in 0.382s, 77 probes kept.\n",
      "-- check region:1181 7:49904999-49909999_strand_+_gene_Chr7-6Mb_reg_1182, 9918 candidate probes\n",
      "--- 7172 probes passed check_dic selection.\n",
      "finish in 0.407s, 77 probes kept.\n",
      "-- check region:1182 7:49909999-49914999_strand_+_gene_Chr7-6Mb_reg_1183, 9918 candidate probes\n",
      "--- 7780 probes passed check_dic selection.\n",
      "finish in 0.427s, 87 probes kept.\n",
      "-- check region:1183 7:49914999-49919999_strand_+_gene_Chr7-6Mb_reg_1184, 9912 candidate probes\n",
      "--- 6832 probes passed check_dic selection.\n",
      "finish in 0.395s, 79 probes kept.\n",
      "-- check region:1184 7:49919999-49924999_strand_+_gene_Chr7-6Mb_reg_1185, 9918 candidate probes\n",
      "--- 7352 probes passed check_dic selection.\n",
      "finish in 0.409s, 81 probes kept.\n",
      "-- check region:1185 7:49924999-49929999_strand_+_gene_Chr7-6Mb_reg_1186, 9838 candidate probes\n",
      "--- 5516 probes passed check_dic selection.\n",
      "finish in 0.350s, 68 probes kept.\n",
      "-- check region:1186 7:49929999-49934999_strand_+_gene_Chr7-6Mb_reg_1187, 9918 candidate probes\n",
      "--- 6046 probes passed check_dic selection.\n",
      "finish in 0.365s, 65 probes kept.\n",
      "-- check region:1187 7:49934999-49939999_strand_+_gene_Chr7-6Mb_reg_1188, 9918 candidate probes\n",
      "--- 7070 probes passed check_dic selection.\n",
      "finish in 0.401s, 81 probes kept.\n",
      "-- check region:1188 7:49939999-49944999_strand_+_gene_Chr7-6Mb_reg_1189, 9890 candidate probes\n",
      "--- 6210 probes passed check_dic selection.\n",
      "finish in 0.372s, 77 probes kept.\n",
      "-- check region:1189 7:49944999-49949999_strand_+_gene_Chr7-6Mb_reg_1190, 9896 candidate probes\n",
      "--- 6694 probes passed check_dic selection.\n",
      "finish in 0.390s, 75 probes kept.\n",
      "-- check region:1190 7:49949999-49954999_strand_+_gene_Chr7-6Mb_reg_1191, 9882 candidate probes\n",
      "--- 5600 probes passed check_dic selection.\n",
      "finish in 0.352s, 65 probes kept.\n",
      "-- check region:1191 7:49954999-49959999_strand_+_gene_Chr7-6Mb_reg_1192, 9918 candidate probes\n",
      "--- 7026 probes passed check_dic selection.\n",
      "finish in 0.399s, 77 probes kept.\n",
      "-- check region:1192 7:49959999-49964999_strand_+_gene_Chr7-6Mb_reg_1193, 9866 candidate probes\n",
      "--- 6674 probes passed check_dic selection.\n",
      "finish in 0.390s, 75 probes kept.\n",
      "-- check region:1193 7:49964999-49969999_strand_+_gene_Chr7-6Mb_reg_1194, 9910 candidate probes\n",
      "--- 6622 probes passed check_dic selection.\n",
      "finish in 0.384s, 75 probes kept.\n",
      "-- check region:1194 7:49969999-49974999_strand_+_gene_Chr7-6Mb_reg_1195, 9890 candidate probes\n",
      "--- 6596 probes passed check_dic selection.\n",
      "finish in 0.383s, 73 probes kept.\n",
      "-- check region:1195 7:49974999-49979999_strand_+_gene_Chr7-6Mb_reg_1196, 9906 candidate probes\n",
      "--- 7762 probes passed check_dic selection.\n",
      "finish in 0.426s, 83 probes kept.\n",
      "-- check region:1196 7:49979999-49984999_strand_+_gene_Chr7-6Mb_reg_1197, 9918 candidate probes\n",
      "--- 7606 probes passed check_dic selection.\n",
      "finish in 0.420s, 81 probes kept.\n",
      "-- check region:1197 7:49984999-49989999_strand_+_gene_Chr7-6Mb_reg_1198, 9918 candidate probes\n",
      "--- 5344 probes passed check_dic selection.\n",
      "finish in 0.341s, 74 probes kept.\n",
      "-- check region:1198 7:49989999-49994999_strand_+_gene_Chr7-6Mb_reg_1199, 9918 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 3370 probes passed check_dic selection.\n",
      "finish in 0.274s, 51 probes kept.\n",
      "-- check region:1199 7:49994999-49999999_strand_+_gene_Chr7-6Mb_reg_1200, 9918 candidate probes\n",
      "--- 4212 probes passed check_dic selection.\n",
      "finish in 0.301s, 63 probes kept.\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ae777bea",
   "metadata": {},
   "source": [
    "### save probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "3c0947ea",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\reports\\mouse_5Mb_5kb.pbr\n",
      "-- number of probes kept: 72574\n"
     ]
    }
   ],
   "source": [
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "056451ee",
   "metadata": {},
   "outputs": [],
   "source": [
    "del(pb_designer)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e728e379",
   "metadata": {},
   "source": [
    "## Reload from here:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "cc1d880e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pb_designer already exists in RAM, skip.\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "if 'probe_savefile' not in locals():\n",
    "    probe_savefile = os.path.join(report_folder, f'mouse_5Mb_5kb.pbr')\n",
    "    print(probe_savefile)\n",
    "\n",
    "# Parse segemented list\n",
    "if 'pb_designer' not in locals():\n",
    "    pb_designer = library_tools.design.pb_reports_class()\n",
    "    pb_designer.load_from_file(probe_savefile)\n",
    "else:\n",
    "    print(\"pb_designer already exists in RAM, skip.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 147,
   "id": "e335fac7",
   "metadata": {},
   "outputs": [],
   "source": [
    "for _seq, _info in pb_designer.kept_probes.items():\n",
    "    # replace bad name if applicable:\n",
    "    if 'Chr7-6Mb_reg_' in _info['name']:\n",
    "        _info['name'] = _info['name'].replace('Chr7-6Mb_reg_', 'Chr7-6Mb-seg-')\n",
    "        _info['reg_name'] = _info['reg_name'].replace('Chr7-6Mb_reg_', 'Chr7-6Mb-seg-')\n",
    "    #print(_info)\n",
    "    #break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 290,
   "id": "7cdd3a9e",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading pb_dict from file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\reports\\pb_dict.pkl\n"
     ]
    }
   ],
   "source": [
    "overwrite_pb_dict = False\n",
    "verbose = True\n",
    "pb_dict_filename = os.path.join(report_folder, 'pb_dict.pkl')\n",
    "\n",
    "cand_prob_nums = []\n",
    "pb_dict = {}\n",
    "\n",
    "if not os.path.exists(pb_dict_filename) or overwrite_pb_dict:\n",
    "    # get genes\n",
    "    print(\"Sort genes\")\n",
    "    gene_names = np.unique([_v['reg_name'].split('gene_')[1] for _s, _v in pb_designer.kept_probes.items()])\n",
    "    #region_inds = np.unique()\n",
    "    print(f\"- {len(gene_names)} genes\")\n",
    "    for _gene_name in sorted(gene_names, key=lambda v:int(v.split('seg-')[1])):\n",
    "        _sel_pbs = {_s:_v for _s, _v in pb_designer.kept_probes.items() if _v['reg_name'].split('gene_')[1]==_gene_name}\n",
    "\n",
    "        if len(_sel_pbs) > 0:\n",
    "            cand_prob_nums.append(len(_sel_pbs))\n",
    "            if verbose:\n",
    "                print(f\"-- appending region:{_gene_name} with {len(_sel_pbs)} probes\")\n",
    "            _splitted_pbde = library_tools.design.pb_reports_class()\n",
    "            _splitted_pbde.kept_probes = _sel_pbs\n",
    "            pb_dict[_gene_name] = _splitted_pbde\n",
    "    # sazve\n",
    "    print(f\"Saving pb_dict into file: {pb_dict_filename}\")\n",
    "    pickle.dump(pb_dict, open(pb_dict_filename, 'wb'))\n",
    "else:\n",
    "    print(f\"Loading pb_dict from file: {pb_dict_filename}\")\n",
    "    pb_dict = pickle.load(open(pb_dict_filename, 'rb'))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 165,
   "id": "ee8f9a76",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7sfQnYXtO1/8qXyDzLIJMkhtxUkCCCoqi6pb00VaUXlyq9RbVabfkbaqq26OW2ty1622o1hhqSEFoU1aBkkoFI3RCZEyESQiYy/J/fqf0533n3PnvtffZ533Ped53n8eA7e1jrt9Y+7zrr/PbarbZv376d5BIEBAFBQBAQBAQBQUAQEAQaBIFWEgA3iKVFTUFAEBAEBAFBQBAQBASBCAEJgMURBAFBQBAQBAQBQUAQEAQaCgEJgBvK3KKsICAICAKCgCAgCAgCgoAEwOIDgoAgIAgIAoKAICAICAINhYAEwA1lblFWEBAEBAFBQBAQBAQBQUACYPEBQUAQEAQEAUFAEBAEBIGGQkAC4IYytygrCAgCgoAgIAgIAoKAICABsPiAICAICAKCgCAgCAgCgkBDISABcEOZW5QVBAQBQUAQEAQEAUFAEJAAWHxAEBAEBAFBQBAQBAQBQaChEJAAuKHMLcoKAoKAICAICAKCgCAgCEgALD4gCAgCgoAgIAgIAoKAINBQCEgA3FDmFmUFAUFAEBAEBAFBQBAQBCQAFh8QBAQBQUAQEAQEAUFAEGgoBCQAbihzi7KCgCAgCAgCgoAgIAgIAhIAiw8IAoKAICAICAKCgCAgCDQUAhIAN5S5RVlBQBAQBAQBQUAQEAQEAQmAxQcEAUFAEBAEBAFBQBAQBBoKAQmAG8rcoqwgIAgIAoKAICAICAKCgATA4gOCgCAgCAgCgoAgIAgIAg2FgATADWVuUVYQEAQEAUFAEBAEBAFBQAJg8QFBQBAQBAQBQUAQEAQEgYZCQALghjK3KCsICAKCgCAgCAgCgoAgIAGw+IAgIAgIAoKAICAICAKCQEMhIAFwQ5lblBUEBAFBQBAQBAQBQUAQkABYfEAQEAQEAUFAEBAEBAFBoKEQkAC4ocwtygoCgoAgIAgIAoKAICAISAAsPiAICAKCgCAgCAgCgoAg0FAISADcUOYWZQUBQUAQEAQEAUFAEBAEJAAWHxAEBAFBQBAQBAQBQUAQaCgEJABuKHOLsoKAICAICAKCgCAgCAgCEgCLDwgCgoAgIAgIAoKAICAINBQCEgA3lLlFWUFAEBAEBAFBQBAQBAQBCYDFBwQBQUAQEAQEAUFAEBAEGgoBCYAbytyirCAgCAgCgoAgIAgIAoKABMDiA4KAICAICAKCgCAgCAgCDYWABMANZW5RVhAQBAQBQUAQEAQEAUFAAmDxAUFAEBAEBAFBQBAQBASBhkJAAuCGMrcoKwgIAoKAICAICAKCgCAgAbD4gCBQJwh8+ctfpttuu62FNq1ataKOHTvS7rvvTqeddhqdf/751NTUFETjRYsW0dChQ+miiy6ia6+9NsiYZRjk8MMPp5dffplef/31mogLm5500kn0xz/+Mdj8v/jFLyIbrl69msaOHRt0bB8ht2/fTt///vfpV7/6Fb333nv0jW98g954443Ivzdu3Ejt27cn5e/q/zHPtm3baOHChbTrrrv6TKvtM2TIENppp51oypQpwcaUgQQBQaD2CEgAXHsbiASCQBAEVEDw3//939SrV69oTAQSCCAmTpxIjz32WBRI/M///E+Q+SQAro8AeO7cubTXXnvRPvvsQ+ecc04UPH7yk58M4iO+gzz00EN07LHH0pFHHklf+tKXaOTIkbRlyxZasGABnXzyydFLXDIAXrduHX3qU5+KZA/5Qnb//fdThw4d6NOf/rSvOtJPEBAECoiABMAFNIqIJAj4IKACAmTAkLWKX1u3bqWPf/zjNGPGDFq2bBn169fPZ4oWfSQAro8A+M4776RTTjmF7r77bjrxxBMz+0WIAX70ox/RpZdeSlOnTqUxY8Zoh0wGwI3qjyHwljEEgUZEQALgRrS66FyXCKQFwFD4xhtvpO985zs0YcIE+vznP58Zg0YNOOqNAvH73/+ezjjjDHr44Yfp6KOPzuwXIQa48sor6aqrrqJ//OMfNHz4cAmAQ4AqYwgCgkALBCQAFocQBOoEAVsA/NOf/pS+/e1vN2f60P6ZZ56h//f//l/E48Un5ptuuinKBr799tt0+eWXR9SJVatW0cCBAyPeKXiZ4BTjUgEwguoddtiBbr311ohuceihh0afoEeNGtUCWQRYP/zhD2nWrFnUunVrOuSQQ+iaa66hfffdN9UCSs57772Xvvvd70ZcTHBAP/e5z9F//dd/Uc+ePaP+pqDplltuiT7tP/nkk4TgVckNefFJHf+GvgcccEDEOW3bti1985vfpL/+9a/UvXv3iDv9gx/8IJIZlwqAwUf93ve+R//3f/9Hu+yyS0QvOffcc1vosnz5crrsssvoz3/+czQHuNiQ5etf/3pzOxWAjh8/PrIDMvRf+cpX6Je//KUWF8UB3n///Qk2BW8XGELGJHUBWAGXZ599lvAVYPTo0ZFdQS1QukyePLnFPOoLAuSAvZXs0BGBMuytsPjb3/5GRxxxBP3ud7+LqDXz5s2jo446ih588MFozHHjxkUy4u/wG8z74x//OJWji68XixcvbiETqDzJjG/8/6En5IhfSo/NmzdHfodM99KlS6lv3770xS9+McKlS5cuLXwZL4mQ/e9//zvtscce9Pzzz0e2jXOAYf82bdrQxRdfHGWp58yZE/nJv//7vxMy1/BNdc2fPz+yKTAGPxn86uOPPz7yXeWPdfL4ETUEgdIhIAFw6UwmAgsCegRsAfBnP/vZKJh55ZVXaLfddosCinvuuYc6deoUBcEI0M466yzq1q0bHXzwwVFg99WvfpX23nvvKOj8wx/+EP0dgSGCRBVIoj/6IABEEI2AB0HHtGnT6GMf+1gkLII8BHUIek844YQoUP7Nb35DK1eupCeeeCKiZ5guyPnAAw8QAj8EEAceeGAUuCO4QiADHXwCYAT1PXr0oK997WtRYIRg+l/+5V9o/fr1UTB10EEH0R133EFPPfUU/e///m+EhQoan3vuuUie008/PeLPQoann346CkIR8OJasWIFIUj94IMPoqAXgdejjz5KkyZNigLsn/3sZ83YILBEMAZZBgwYEMlxzDHHGANgYI4gDJsa0Q/BMgLWv/zlL82BIDjf//Zv/xZlUBHEq4D0xRdfjF6CYAe0ue+++yL9EMzD1vg68NZbb0UvBODVIqhHUIq2sEMccxUAQx5wdRGId+3alU499dQog4sgE1xeZJaxiQ0vWAjEQW2AD+oucG7ha3j5uv766yO6DsZLC4Dfeecduuuuu6IXPOiMlzXogWAUcyMAhW9DP+gPfcF5hs3atWvXwpcRwKMP7HbeeedFuicDYAS27777buTTe+65J0FmrC1gCJlxLVmyhPbbb79oTSg7/frXv6Y333yT1qxZIwGwPMgFgRojIAFwjQ0g0wsCoRBQAcLMmTNp0KBB0bAINhDcIct588030xe+8IUo4MGl2uMeAjB1XXLJJVGW7vbbb4+yweq64YYbogwsAlz8oMcDYHyqVnMiwED2FwEI5kIQhXvInCGAUheCFgSPvXv3jjJtpkvJiawysmnqQpCKQBjjILvomgHu06cPvfrqq81ZQAR2kDe+URBjox2CKmRocUEPBFTA44ILLoj+hiDnE5/4BAF7ZH133HHHCF8EccgQxjnZwA7ZUvwdAZnKACNITVbx0GGCwBv/4AUDGV1cyNIjuzxs2LCI541sI/4fAT6CdWToceHFBBl6+ASyrHiR0VEgsNEMASUCRLy0qAvYoGIEAj5kMVUADN3jmeTXXnstmh8BpAr0MQaCQgSMCDIVnjoddbZMC4AR6OooOUq3JO0HdkEmFnaATqovqprAJ+KVUnQBMHRFFQ4E2rgU3ps2bYrsjwvBMew5ffr05q8c8Cfoj5cVyQCHevLJOIKAHwISAPvhJr0EgcIhoCuDpoREthBBDYIX9dlXtUewgh9+deHTL37IQQ9AoKWu999/PwpWkTlD4KOChv/8z/+MqAPx61//9V+jz+74wUewhGwjgoHPfOYzLdrhczwCcwQEyHzqLlNmG8EnKl4gqEKA7RoAI6uILLK6kLnFp/JkYIKx8Q/0wYUAGIEuqAcIINWlNpMhcMRmMlAzkEWOz4G2eFlAwIi58LKhgrTki4jJwWAT4ItscvyCHZBhRFYdQRiCY2Skzz777BbtkHnH53t85kfmPRkA46UJn/TRH1jELwTayIYi841+KgCGHZHxVZd6WcLXArzkxC+8bCHrjxcj+KXuChUAI/uMYBU+nrzwsgAdkTVXvqx7CdEFwPAFfClQLxYYGwE1svt4GQJlA5VYkBFH5jx+XXfdddEXFwmATR4ufxcEqoOABMDVwVlmEQRyR0AFisjc4nM7LmSyQE/AZ3B8po5fqj3oCPF7KPmEIA+c3eSF4Hft2rVRwKCCBvAm8ek5fuGzOQJbBGMIfPGDn3YhoECwqLuUnAg4FP8Y7TAmggnF9XQNgMFlBe1BXao/+KqKuoF7ugAIGCCDG7+QkQVtANlzBHkIFNMu9eKgAlCU/gJNxXYhAI5TKFR7fHpHhhwZX2R3QUlIuxCwg7eaDIBVkIvAGTZMXsgqoywZgl8VAOPFKs5rBuUD3Ou0CxQRUzWSUAEw7IiazaYLQTCoPsqX8YXjJz/5SYvmOvvDR0DpiF/AG9QSBL+gkCAAxjpIcrnxFQRUHgmAbZ4u9wWBfBGQADhffGV0QaBqCNg4wElBdAcJoA0+J4NeoAuAEfjg4AFwIFXQoCgR8fERPCErDL4jspLIdCIQQMChu8CVRGClu0xycgNgBGf4zJ3cBJc8wMMUQOsCIGS2sZkvfoEnjSAeARSoI/3796fjjjsumlt34T6y7a5VGBAAf+tb34qy3/ELFBFkdvHJHTxvZPyxiQ3ZZt01YsSIKABNzo8DPvB3UwCMFyoE+sicqgAYgXI80wwuM3i2oJSgve4CtSK+YSzeJlQADC41aB/IeusuvOyB155W0URnf91BKPEAWGGIF0O8IMYvZO7BMZYAuGqPRplIENAiIAGwOIYgUCcIhAqAkTUDBQKfjZMUCGS1sLELn7BV0ABOK4Lg+IUAevbs2dFmH3AlFac0mZVE0IhP4QjSTMEQNwDG5358ise8CNTVhSAQ1SZCBsDgOSMDqKohYC5kuiEr+Kb49I7AD3zbRx55pAU2eCnAxjp8fh88eLBXAAz+Lagl8QtZZ8gAagaymqA3IBudzL5DdtBGEIBCRhMFAi8lCHDjFzL6CNzBb/3tb39rDIAxL156kI3GpsX4Bd+BX8FH4v6VRwCMwyvwdQEZ+yTdAhsXwVPGV43QATA4waCRwK+R2Y9fyReyOnn8iBqCQOkQkAC4dCYTgQUBPQKhAmCVWU1uggNdALvckXlEBlIFDfjUD14rfvBxIQOJDCH4lAiu8DkYHFpkO8E7xa57XAiOsQkMn4zxyd7EB+UGwMg0g1aAklzogwu8ZfAwX3rppaABMHil4PqqgB7zINDDBirwmVEJATxgbPRCkI+XBnUpesif/vSniBPtkwHGywI+wyvuNgJa0FxwaASCVlQwQOYSFBic9KaysMiGQk5kMCEnNuvp5lfZzOQmOGxqQyYfm8jwGd+UAQbe2OyFDDjaqk1loKvg78g+gzJiunwywNjYt/POO7eoxICqE6BmwHdBeVEXgl9sYFNVG0IHwJgHPGlQIkCVQSYaF14ssTZeeOEFyQDLg1wQqDECEgDX2AAyvSAQCoFQATDKoSGQQgYYAWW8DBqylsnSUQiiwDkG7xPZR2SDEeSisgNKjeHCbntkijHWf/zHf0TBLjiioFIgkFS76XVYcANgFWhjbgQ2CPoQDIPjjIxoyAwwghpsdkJAiM17eFlAaS/QPoAZLuCHYAeUEQS9KPuFTWEIisD1xYYpBIY+ATAwx+d7vIhADuALWgaqYqj6ywg8UdkCLx8o4QY8UF4MgSe40xdeeGEkp25+BITwAfCuITsCbVAewF9F9hljI3trCoAxLgJOfP5HFhybIDds2BBtwoSPYGMY/h4yAAbO2OCJYBPUA1Q8Abf9sMMOi15CsOkRWW/QQxDE48UNNkOVjzwCYLyU4OULL3jgbKtsO3wesgI7yCaXICAI1AYBCYBrg7vMKggERyBUAAzBEKRcccUV0Wd2/Dcya9gwBY4pAi9cKmhAdg0ZXASByDyixBX+hgME4heCJvwdFAVUT0B1AIxnqner+nIDYLQHtQAZbFRpAKcYWWhUTPjUpz4VNABGUI1DECA/spoIulBFInmUMO6BloHgEQEqKA/gByNAVzj6BMCogIGAChlOUEhAdwD3GJ/z4xcy1aB/INBDIIYsMV5EEAyqyzQ/bAqdQOFAzVsE8KA+oH/yIIwkB1iNjaw87uELQefOnSPaB/wqSYtILgafDDDGAAbgQiPARF1ebOZEEA8MkPVF1hsva/BRzKFK9+URAEMeZOmxsQ4vjcAMLwLwFbx86OghwR8KMqAgIAgYEZAAWJxDEBAEBAFBQBAIjAA44igbmOQ5qzJoKDOYfEkMLIIMJwgIAikISAAs7iEICAKCgCAgCARGANln8JJBeVAZc9RYRqUQZJxRKSJ+4Ebg6WU4QUAQsCAgAbC4iCAgCAgCgoAgEBgBHGxy5plnRtUuQH1AZQiUhQMthXvoSWCRZDhBQBCIISABsLiDICAICAKCgCCQAwLY8IiqKai6gQAY5fnA/0Z1DLkEAUGgtgjUfQCMzQ+PP/54i3qW2ISDjRwzZsyIygBhhy42KqgLDyoc64ni6agfiZ3D2GyCTSByCQKCgCAgCAgCgoAgIAiUG4G6DoBRjgm7pVGMXBV0R6kk7IRGDUvcQ3kclPlBgItC8rgQ/OL/UUIJJY6wYxcljVDbErvX5RIEBAFBQBAQBAQBQUAQKC8CdRkAL1++nM4666yo9AzK3KDsjQqAcUIRakBiE4IqvI8Ti3B6Ez5ToaA9Tru6/vrrm4/2RF1UnH4E3lbyJKvyml4kFwQEAUFAEBAEBAFBoDERqMsA+MEHH4yK64P+cPXVV0fBrgqAcfISTqy68847my0OigTqQq5atSpqi+L1KJw/bNiw5jagQaCIPzLDcgkCgoAgIAgIAoKAICAIlBeBugyA4+ZAEf14AIwgFoX3UYtRXShWjqM5cYQrTu/BCUI4tUgVqkc7FLjH35LnusfnQkF6/KMuHHuJMjg4Rcl0zGt5XUckFwQEAUFAEBAEBIEiI4CTIt98883o4CEcoS7XRwg0XACMjWwnn3xylBlWF/i9u+66a0SZQLCMo1pRrzFeoxEnSq1YsSLaUGe61OlF4mCCgCAgCAgCgoAgIAgUBQEcgb7//vsXRZxCyNFwATDegkCD0GWAcXwqgmF1bn0yA7x582Z64IEHjIZLZoCR/cURpXC8fv36FcLgIoQgIAgIAoKAICAINAYCK1eupDFjxkRHtg8ZMqQxlGZq2XABMIJflD4bN25cM0Q6DvCrr74aZYXVBQ4wajhiAx33wrnz2ISHQHjgwIHcbtJOEBAEBAFBQBAQBASBzAhIHGKGsOEC4GuvvZZuueUWwjns6njKiy++mCZOnBhVgUCWt0+fPnTjjTdGp/jg8q0CIY6Xee3KAIKAICAICAKCgCDgiYDEIRIAN1eBeOONN6I6wDiJB/V9QU8455xzoqD49NNPj5C69NJL6Ve/+lVU9gyfDHByD7jBL774olMdYHE8zxUr3QQBQUAQEAQEAUEgMwISh0gA3OIkOFR7wOlvs2bNiri53/nOd+i8885rRgkb4FAbGAdhbNy4MTpIA9QHV/6MOF7mtSsDCAKCgCAgCAgCgoAnAhKHNHAA7OkzQbqJ4wWBUQYRBAQBQUAQEAQEAQ8EJA6RANjDbbJ3EcfLjqGMIAgIAoKAICAICAJ+CEgcIgGwn+dk7CWOlxFA6S4ICAKCgCAgCAgC3ghIHCIBsLfzZOkojpcFPekrCAgCgoAgIAgIAlkQkDhEAuAs/uPdVxzPGzrpKAgIAoKAICAICAIZEZA4RALgjC7k110czw836SUICAKCgCAgCAgC2RGQOEQC4Oxe5DGCOJ4HaNJFEBAEBAFBQBAQBIIgIHGIBMBBHMl1EHE8V8SkvSAgCAgCgoAgIAiEQkDiEAmAQ/mS0zjieE5wSWNBQBAQBAQBQUAQCIiAxCESAAd0J/5Q4nh8rKSlICAICAKCgCAgCIRFQOIQCYDDehRzNHE8JlDSTBAQBAQBQUAQEASCIyBxiATAwZ2KM6A4HgclaSMICAKCgCAgCAgCeSAgcYgEwHn4lXVMcTwrRNJAEBAEBAFBQBAQBHJCQOIQCYBzcq30YcXxagK7TCoICAKCgCAgCAgCRCRxiATANVkI4ng1gV0mFQQEAUFAEBAEBAEJgFN9oNX27du3i5fkg4AEwPngKqMKAoKAICAICAKCgB0BiUMkA2z3khxaiOPlAKoMKQgIAoKAICAICAIsBCQOkQCY5SihG4njhUZUxhMEBAFBQBAQBAQBLgISh0gAzPWVoO3E8YLCKYMJAoKAICAICAKCgAMCEodIAOzgLuGaiuOFw1JGEgQEAUFAEBAEBAE3BCQOkQDYzWMCtRbHCwSkDCMICAKCgCAgCAgCzghIHCIBsLPThOggjhcCRRlDEBAEBAFBQBAQBHwQkDhEAmAfv8ncRxwvM4QygCAgCAgCgoAgIAh4IiBxiATAnq6TrZs4Xjb8pLcgIAgIAoKAICAI+CMgcYgEwP7ek6GnOF4G8KRr3SPw/pZtNH7mMlqyZgMN7tmRvrDfQNqhdVPd6y0KCgKCgCBQLQQkDpEAuFq+1mIecbyawC6TlgABBL+n/XYqTVm4plnaA3fZkcadOUaC4BLYT0QUBASBciAgcYgEwDXxVHG8msAuk5YAgbumLaGLJ7xYIem1x+9FXxqzcwk0EBEFAUFAECg+AhKHSABcEy8Vx6sJ7DJpCRC47pGX6ea/LaiQ9JzDd6WLjh5eAg1EREFAEBAEio+AxCESANfES8XxagK7TJqCQFF4t+OeW0zff2CuVwa4KDqIowkCgsBHCMi6LKY3SBwiAXBNPFMcryawy6QGBIrCu4Ucp/5mCk1btLaFpAcM7Um3n3VAKge4KDqIkwkCgkDL4Fc4/cX0CIlDJACuiWeK49UEdpnUgEBReLcmOX44dk865cDBqfYrig7iZIKAIPARArIui+sNEodIAFwT7xTHqwnsMqkBgaLwbrPIkaWvOIYgIAjkg4Csy3xwDTGqxCESAIfwI+cxxPGcIQvWQfholVC6ZmnywtBVjrgmnL55yR3MOWUgQaBOEFBr7ZG5r9Pk+W9WaCVVXWpvaIlDJACuiReK49UEdhKeqB53F1xc2rpaOcvYtr62+66ySntBQBDgP0/iLaWudzE8R+IQCYBr4onieDWBnThZwtpIVvtZP9i6jcY/v4wWW05fyxtDrhw6xNL65i137S0oEggCxUDAtNYOH9abjt5zJznZsRhmIolDJACuiSuK49UEdhI+Wnbcy4phWeXObjEZQRCoLgKy1qqLt+9sEodIAOzrO5n6ieNlgs+7s2QB06HjcGSz1Om1GY4zv20M032xvS9y0k8QcEPAd63luf7dNGiM1hKHSABcE08Xx6sJ7MIBToGdw5HNUqfXZnHO/LYx0u7nPX4W2aSvIFBPCPisNZ8+9YRZLXSROEQC4Fr4nXBvaoL6PyfNwjGtodi5T83J2mSp02tTgDO/bQzbfbG9DSG5LwiEQcB1rVVj/YfRrH5GkQBYAuCaeLM4Xk1gl0lTEODw9jhtfEHOc2xfmaSfICAIVAcBWf/VwTk+i8QhEgBX3+uI6iIDnORrHTuyP02as4KWWKoI1ATwxKTCNau0AicDw2nja1+Xsd/btIW+/8BcWvTWehq6Yye65vN7Use2bVhT17Pt61k3lnGlUVAEfPzJpw+Edln/aUrq5t++nWj8zGWl+G0KakDLYBIASwBcTX9rnqvsjqfja3Vp34be3bSlWcei1noUrpne5RFUHnTtEy1s2LV9G5pyyZHNwWWe2HHH5shpWtTcOWryUMg4aT3rlhEa6e6BgI8/+fRRomXpmzbGmKE9o9vTFq4p/G+Th5kydSl7HJJJeUvnVtu3471JrjwQKLvjmd7Wk1gV8bSfUJmGPPyilmNycXHl9rnoxBn723fPpomzllcMe/w+A+jGk0alTsfV0UXmorStZ92KgnEjyeHjTz594phy1n+aDbi/SxijiL9N1favsscheeIlAXCO6Jbd8Ux8rSRk5xy+K1109PAckXQfWrhmeszKgsvnb/o7zVrydoUS++zcnSaee3CqQ5RFR3evJqlx7QOa9DEi4LNWfPqENAH3dwlzFvG3KSQWnLHKHodwdPRtIwGwL3KMfmV3PO6bNvct28Tp9OWT6UyAse6esYTGPbuY5r/xXkUTk6whZWC4RosmoedOGy9r9sZVN1t7k6ymDPA+g7rTgbvuSIN7djSeNGXSEdnjvt3ap/a1yVvr+0WzX63xkPl5CJjWmY8/mfpcM3YEtW5qyp2Dy/1dAjLqeR/6GctDnaKSnLXmJZc9DuFi7dNOAmAf1Jh9yu54ITnAJk7nUxceQWePe56mBOBumerXKnOZ+MoheGlMl6hoFnpu23i2+756+PRLk2XzB9squMqtWxFtjRG2XOxZFu66Dcci2c8mq9wvBgJpPgMC5Gm/ner0/NWNV00Oruv8PjqGsFxR1mrZ45AQtjCNIQFwjujWg+Ml+VrHjepPk2avoMWOVSDSMnqzllZ+6uZmlePmS8sMIPt33Ql70w6tmyos7pMFCeU2oefmjJeVg1ct3Te8v4UumziXFr61nmg7kYufxHVc9c4mmqDhE/v4WCjds4xTFPtl0UH6Vg8B2zPBx5+SfbZs206X3T+3Qqm81phOZkw+/vllFb9NNv3zskSt5k3qUw9xSF42kgA4L2TrpAxaKHhMnM6+XdvRqnWbK6bx4W6lccPSxqslpy1F+so6AAAgAElEQVT03KHHC2V/3Tgusrq0Tc6VpW+e+svYgkA1EKiG/1djDl+saiVbreaVAJjvKRIA87Fybtmob1463tOF972g3dUPTqdLZi/NCGkZ4DR+Wi3f1E1zH7d3P3pn0xZqRURH7dGXTtx/kDZ7ncTDNN7hw3rT0XvuZOTNxsdRPOrH573RPP/YfQZU1H/OWnfThavr0lbpovS47dnF9KoDHzyJRa05fDqfT9roiOG9I/7l8rc3lprj7PyQlQ5WBLI+33TP8+Ta980Aq7EXrl5Pq9/dTL27tKOhvTqxnlNWxT9skFV/7jzcZ3FeWXGTnI0ah3DsJgEwByXPNo3oeCbe0/98aRQd9OMnWnA4wemceumRdN4ds5w4aCZzgGd84I8fp/c2b23RZPTgHtTU1MpYI7KWXC0bb1kpcsDQnnT7WQdYg2CdLnEwbHWbTfIkObQhOH8uHHMdjy+N12vDlYNnLf0i7ZFj0w19bXb2fKRJtxIikMWPuXxbn+dB2rMqpP9m0T+LuWs1r2SA+VaTAJiPlXPLRgyA0zJ1Jh7mF/YbqOVuuQJumnvsqP50/+wVFcPF38R9eHCu8pna3z5lsZY/l2zPzRwoXR6e+zpNnv9mqt7crAVXV66MajwXrq5LW9tO8R+O3ZNOOXBwqlq1yhzZsLbppvq72sI2r9wvLwK+zzeurwEZrKnWTa3Y+0NsY4f0X1/9s1q8VvPG5W7EOIRrNwmAuUh5tGtExzPxnlC/VVfX1YfrazJFLef2cI/mLty6lq5Y+XDQuLKY9HWVMT6Oi7y2tjY9OHLa5shi8yx9bbqpsTk6ZpFD+tY/AlxfAxKu/mYb23W8+reGn4aNGIdwkZIAmIuUR7tGdDzTW/2wPp2d6vJ6wG08Z/5zo/rRA7NXVgwZKsOg48jpqk2YdLJlQnwzemnZeFNFjHHPLabvP1C5m5trjyyYmuQFTxyfWOPcQFst0kcM2W8XLMueAU6rfMK1p7RrbAS4zyagZKqzHX8+DujenohaRVx1U3UWlzXa2Nbhad+IcQgPGSIJgLlIebRrRMezcVDjMIbkeWFcE19ty7ZtNHNxy1Jr4I9OveRI6ti2jYdlP+oSgufFwWzMkJ50x1ftHOC4Mq4cuzRuaR4c4CTwHByUz+g4wToeos64XNuHsG0m5zJ05nCAVdfQaywPfWTM4iLA3RNg4uPr1mlc22Rtb/Hb8L7QiHEIF0UJgLlIebRrVMezcVBdKhK4ws6tT4lxs2QrlVyhsoQ2zDicVR1WGPei+15g1cFN41D/6Pi9Kuo/Yz5d3U1Xm8Xbp8mbzAy52DopE9f2ReDwmex6z/Sl9Jd5q6JKHZ8c3odmLllr5bpnsY30bUwETHsUVMY3rc42zq25eMKLTsDJlwsnuKyNGzUOsQKDbxHbt+MdTa48EGh0xysCh9K3NjDXH0LrGHo86MEdk9uOi41vO19uoK1fXJ565BcWxX6+dpd+xUTA5ldp96HRzX9b4KRYPa5NJwACN270OCQNTgmAAztbfLiyOZ6Ny4oyY+CHLnprPQ3dsRNd8/k9UykEnOyobc4086T1VffSuKDcLGCaDBwdXVws9HiYm1tD977nl2mzNVycXG1pam/jHZrksfWL2yE5RlKWY0f2j+oeo37wzMVrcRAd7dLL7vMutvZt64ob136+8ki/8iOQtnZtzyTTvgH4nU8GWPw1rD+VLQ4Jq336aBIA54h2mRzPxndE8HvQtU/Qu5u2NCPWtX0bmpLCo7WNabtvC35NZ9jbeGcYl1MHluMaWXTQjR96PMyhG1PH2fvt6aPpzN9P96rJ7Cp3WvvNH2zT1nO22U035ughPej/Xn831W85+Chb2Xye4zNZ2qThpvN74QBnQbsx+trWrs3nTv3NFJq2aG0LsNTz1fYsTqvl3Rjo569lmeKQ/NFoOYMEwDkiXibHs73lf/vu2dqT3MDXuvGkUUYU0ziUtjl9M6+mrMOwvp2pf7cOTiercdwjNE809HjQgVtD17cms6st09qnZY1sXGguLzieZXLJHANLm89zfMa3jQ3nPHzHV1bpVw4EbD6VfH4M7tmx+aQ2U9/4Oo375MDuHSJQln14YuFxo/pX7C1wqaBTDoRrK2WZ4pBqIyUBcI6Il8nxbDyvz9/0d20dX9T3nXjuwV4o2uZMG9SHdybcsn8imgV3k01cx/SxH+Z2tSFHLhfuMGTI4vNeCyXWiaNP1jmkf2MhkMWnsvRtLJRrp22Z4pBqoyQBcI6Il8nx0rIAx+87kE781XM0e2nLUmJp2TAOX9g0J6dKhE8G0cYtc+WwQn+fPjm6XPPQWTh9Ovkw3t0zltDj896Iqg4ctUdfOnH/Qc1HM7va0sd+kEv5huLoLlmzgeIZqaTsafxE+PX4mcvIVjM4OWY8AxzHOV7jNE2mLPY34faJ3XtRr87tqHeXdi3qJWeZS/o2BgK2DLCJH4+1Z6oAoSo5gAKBNYa2WB9btm2nJ19+U/sMUWgX9ZkayhuqrV+Z4pBQGHPHkQCYi5RHuzI5nonnpeOFKihMfEguX1g3ZxzmNP6ijZdm4gebPq/ZeHCmoNB1Hg83cu5i08V2Pzkh2qfx/ICpqy1t9tPNF5eLwx1Mk/vWL+9fwXdOG1/n8646OxtS08E2p+oi3N8QaDfGGLa1mHzGJdde8v8Vapy63Mm9GK7PprJZqBb6lSkOqbY9JQDOEfGyOZ6OP2iqDIDTuXAwg+4gCRe+sK3+bVrWNo3v6MqFtGVBdG7i0ydHd2semiOXCz5pHNm4fVxtabOfqnO78u2N2lMEk1gmfSWNn9jU1Epb8UJlmBU3EVUgnjdUgeBwh21fHXz8wYazGjOPuX3klT7FR8C0Fjk+Du3wezBL84WQozmHj18vvsx5NnMwc2lTtjjERbesbSUAzopgSv96cDwfjpcPX9hnnpCm85nfp09ImU1jhZbLtZZyNeePY5DkB/vwjF04xhzusMt4rr5hmz/PuV1llfblRMDmY0or8OJnLamkyHG0jvtp6GcHZ/5qtqmFfvUQh+RlIwmA80IWO12XLaNBgwbR0qVLaeDAgTnOlN/QPm+sLhlgJbnPPCG19pnfp09ImU1jhZaLmwHOy5bcLBQ3A5xWn9Ql08SRy2U8V9+wzZ/n3K6ySvtyImDzMaUVOL8TZi33UlIywGFOJTWBXw9xiJdjMTpJAMwAybdJPTiejR+mNjjEN/3oOMDt2zTRqQcNpt16d6b4Bia1cQibJB6bt4oWrl7fDLcrjzHL5gKdnjt2bkuDenQ0HoBQCz4XxxdDyaXwhE3+8tLrtOitDS2mN9VSdpmfYzPdeBwOMPzw2F8808Kn+ndvT18/fDcau88Ap5rHOjltNU5d/Zdj23ibND5w3nO7yirti4VA2ubN5PP5gdkraHqszm+Xdm3o3c0f1YPHc0DHqR+yY0favGUbrXxnk1F59L3l1P3o6ofmRQcsDerWgV5Y8U6LZ01IX1abef/y0qpoA99O3drTp0fs1GJDr4ulOM8v25oNqZ9O9nqIQ1xs4tJWAmAXtBzb1ovj6fhhtqL7G97fQpdNnEsLVr9Hy9dupNXvvd+MnmnTBBrgtC1UGBjaq1NzrUkO7C5Bl2k8pecrb7xHd0xZTJu2bGtuatrw58Kl5egRqk1WuXR4Dt2xIw3q2ZGaWrWy1lLmzO9is+R4tvqhts1i+NG59cujWTVI0+SEvcY/v4wWr9lAyRqnqKmcd01Thctrq9fT6nc3SxWIUAuojsexrY3k87lzu9b03uatzYgk/x+b3e4464DoPtYCfDGZzOjXrR21a9NEi97a2DzOkF4d6d6vHUSfvGFyi4Nq4tDj9+Chbx6SeuIo11SmTbHo73MwksvzKy4j59nI1YnTrl7iEI6urm0kAHZFzKF9PTse9zM79xNaHFafT7dceTjm86FwcMYtU5uQeJr0znMOjt9x/SxPOcvkEyJrfSDAWRuumnJoDLoxOZvnuOvUJrNNb9d5yvJcqOc4xGZz230JgG0IZbhfz47HJfNzN1HEYfbZvMOVh2NOn018nHHL1CYknia985yD43dcP8tTzjL5hMhaHwhw1oarppyNbLox+3ZtR6vWbU6djrtObTLb9HadpyzPhXqOQ2w2t92XANiGUIb7ZXQ8Lqcp7YCBL43ZuRk121u3Dl7bcbe6PqZ5VEF226fouN5TFrylLemjDkBIYsQ9lCGDKwXvarNzVjwhsO8caZkY25gKKI7f6XxDN76pFOA1Y0dQ66amqMi/4sDHC//7HobB1ZGDcXDHkQFLg4DiuyYPrzH5cxbF4muJs/bUXIN6dKClaz+iRehk4B5+k7ZucO+i8S/QxJSNevHnjmms+N9Nh4Akn186O2APwqQ5K1o8O2y/Ub72KWMc4qura7+GDYA/+OADuvLKK2ncuHG0du1aGjVqFF133XX08Y9/PMJw9uzZdP7559OMGTNoxx13pG9+85v03e9+1wnfsjkel9Nk4lLpeFQ2vpkO0DFDekY1hl0eCFk2BOn6tm5FtHX7R9IpDnCbpiayFYbPe1ODkxNqGnPsnAVPFZjZDgnhHpiiVODIndZWh1vcVqbxdYfB6Ir86/7m6gtZdXSdL6svSf9iIpD2jPY9BMamqfI9vATqDrIx7f1oIqKPdluYZ0nb+Jq2bmybVTFj/LfL5Tlg24xrsoOtnw1rl/tli0NcdMvatmED4CuuuIL+93//l2677TbaZZddouD3nnvuoX/84x/Url07Gj58OI0dO5YuuOACmjJlCp177rl000030RlnnMHGvGyOx+U0pR0wcMqBgyvw4Rbuj3d05WOhL+a56L4XtOV40sYz6QN+Gs7sHLpjJ7rm83tGGzG42Q0f+dmOlbEh186+eEI8zhycNnFVXdvHN5tgg9r0RWvo/tkrKtBTtkobHxva1GY3ZHZxpOtl989lWcLFF1x0dGnLElQa1Q0CttKFcX9Obt5UG0wfnvs6TZ7/phMmytdvn7JYuz44nF+XCTlrF3mMiye8WDFs3y7t6F926lJRBSLt65eu1Buy3327tdceyc79vYBwLs8JF4zKFoe46Ja1bcMGwMj4HnnkkXTDDTdEGK5bt466detG9913H82fP59++ctf0qJFi6hNmzbR/UsuuYQmTJhAL7/8Mhvzsjkel9PEbZcEysbBird35WOpvj6yufTh6uArP9u5MjQMoa9NP84cnDZxNV3bc/1P6eIyPtcPIIMNK18dXeTN4C7StYQIuB5eo1PRxcdVf9taynJghk5G23y4j+vmvy2o6G5alya9TbKnrW8XDF2eEy4uWbY4xEW3rG0bNgBG8AvHePTRR6PDKv77v/+bLrvsMnrppZfoG9/4BnXv3p3uvPPOZnwff/xxOuqoo2jVqlXUp08fFu5lczzT2+rYkf1p7cYPkAyNyl/hjVqX/bJxIk2cKR2Yvm/DaW/vrm/pOi4y943+6uNG0GkfH2L0ExvP03af44BJ7tkRw3tHnFWUKNJldhTmPjw3JY/q+4ghexT3EZM/jB3Vn37yxZEVFJi0jOe/7d2fvv/A3KiWaDxjH8fJ9uWCy2vHmFw/QFsXX3bJ6prkRUbqB2P3rBrHkOOL0qa6CNgywMfvO5CSNdxBFbh7xhJSnOHuHXfQfjFJ0+Sq4/agtm1ak2n9D+vTmXWsORct9YxOe+6/tf597fPOtD/ENQOs+91T9L28nhNcfNCubHGIi25Z2zZsADx37lw66aSTaN68edS6dWtqamqKsr/HHXcc7b333nTMMcdEtAh1od2IESNo+vTpNHr0aBbuZXM8HfcpWfMRiu8/pAe1atWKpi1c04wDlxNpqzGJAX1qMsYDMFeOblZOc6e2TbT+/ZYsNug59ZIjtfUrbTxP232O85l0MvWN8/dc8UvDPj6fzkd0/mXyARMuN5+yL33iJ0+2qCWqq9ts40We8btpNC1W8D/NF3WyjB7Sg/7v9XetcqTZj2t7m32ryTHk+KO0qS4CNl8/8/fTaUri+b1t+3aakfB/E28Xa9mlfZr2unXDRUv9Vuh4vmn15tX4Os68CwfYxvsXDjDXkrVp17AB8L333ks/+9nP6Hvf+x4NGDCAfv3rX9Pdd99NTz31FB1//PF08skn09VXX91slddee4123XVXevrpp+mQQw7RWgs0CvyjrpUrV9KYMWNKdRRynDeZlrHFm3frplbRAQCunMg4Zwp8s4kzl9Nf5q1qzjCfuP8gpw1wSWNwdEhm5UycNV32LlnI/O8LVtODc1ZW+ISqGpG8Ycvy2e5zHhXczIPaYa0ObfDJoCt5TH3VHC68WYzJwR5yX3iffne3Dn+TnU1HuaZVJEn6gUk/lwww9OYUyufaN+4rrnJw/EzaFBcB+NE905dWPFtdq0BgbfTq0q7ioJW7py9l8eBNWV/f50ISceXXnOe+zlrc5wyyuj5rXmeHz+87gHUITwjvKlsiLoTO3DEaMgBevHgx7b777vTEE0/QoYce2owV/rtXr1706quv0mc+8xltBnjmzJm0zz77aPFFVYmrrrqq4t7SpUtp4MCBXJsUpp0Lj6wIXCcdcFyeJLedbg7XusG2uWz3OQ7CtUeSd5ZlbltfrkxKPy4nzgX/kPy+pB1s+nPsxm3jiiXG5eLJlUHalRMBV99x5comUbFxZ13lSY6vk89lzCzroppr3tfbJAA2I9eQATCyvyeeeCJt3LiR2rdv34wOssF//vOfafDgwVHpM5RIUxeHA1wPGWAO/xOYJN+aXTJStkxUCP6rstvv/76QrnxwXsUK4Mqfxu9Scv7umYVaXhsyJ9d+Ye8Krp0pA8PZ0Ywayxx8uPZI8uBMvNJkplgng0kvlUU1jW16PNn8RPUzndwHLvFPv9TyZTWNN6vb4a2TwYR/iMx9EgvXudJ+JH3qa/v+6Eq/YiHAfa7rpDatAVtdXTWW6euK+hLosjdEJ5/uGf3HaUujPQGci/uc0Y1lW/OcZzVHxixtJACWALgFAihrdtBBB9HUqVMjioK6jj766CggPvDAA+mWW26hBQsWRPxgXBdffDFNnDixrqtAcDnAOo4uaroe+OPHW5wZD9y6tGtD727e0oyxrU4plwPJeSCYZNLVGdbVpO3SrjUN79eVpsd4cWlc2bhM4KA+deERdPa451tw7dBfV1eWU4923JljSMd1M/HYdLU4dbjZ6nfG+6TJD71MPFrUHtXdA5986/btNHPx2y1ES+NQJ3Xg2tmVF+nCD4RtNn+wjQ669olMHOC4bmlrwVbbVMetzsKt56w3aVNMBHR+xNmLAW1ca7sn66ebnhfc+W2Imvae6PjJeawJ1zVq+/2z6etzXwJgCYBbILBt2zY6/PDD6c0334xq+4Ke8Ic//IF+/OMf0zPPPBPVBUYdYGyIu/DCC2natGl0zjnnREHx6aefzvbBsjmesQrEqP60dsNHVSB0HN20jGNancQkmLY3ajb4ROSSGeRmTDE/Mgam2pI79+hAo4f0jOoGPzB7hbb+JPon68oqDq7Sz8QDdcEnyT375PA+NHPJWmMtXJNOScxNGR3oZeLBpnFsTVVFXDIztz27kK6YlJ7pt1WByMK9TfMJFz3iWNtsnaxzjL7L3t4YcfI3b9lqxcNlLUnb8iJg4/WHqsaiENI97335uRizY9vWtOH9rRUGQE1hPDe59bh1FgzxVSTEszpP7ypbHJInFsmxG5ICARBw+hvKnj300EPRf++11170ox/9iA477LAII1R7wOlvs2bNon79+tF3vvMdOu+885xsUzbHy8JncuELp4GYRYbkuCG4oTpZubUlQ+qi5Mg6Zlp/zKGrl5nEII3TZxrDp48LN4+DC6eNbYH74OeiR3z+LPJm6WvDQO6XCwGbL9juJ7W18Wtt/m7rn5yvb9d2tGrd5soAeOfuhIwq55llsphN1iyWdsU1y1xpfcsWh+SFg27chg2AqwFy2RzPlnFKwywtg5rkj/qMw82iKc7VwtXr6eEXV2rPmddVBwiVAY7LmQVPE0bcMZM1gFG/GZl7E08X2RR8lnzqldXWpZGWATZlkX36AEtdvVLdEdkcXDhtbMqnjWHSneu7ybm58rrwsX1lseEi94uLgM2PTLx4ZEe/OHoQew+DQkCXVc3CQR7Uo4PxOb7Pzj3YXF+dhfDb9Kk9+kRfrp58+c1glYgwlw33anlM2eKQauGCeSQAzhHtsjleFv6trS4pYObwn7LKkKxhmzSvrj4s2rjWa+RwcbPoYnJLzpiuXNe0JaCrJ5vGYTbhErpPMgjm4pL0D45PxvGpJucvi042nnmOjz0ZumAI2HzWtFdAV+/dxOmNq5zkDXM4yKgF/PLKddY9JGoetcfiP/8wo6J2t05uV5OE4Mtz1q+rXD7tyxaH+Ojo20cCYF/kGP3K6HgcHqRJ9TjndOXbG7WVETgZKF8ZbFlcZDnv+OoB2sMpoFNajVjdKXIcOTltGK7UooltzDQcFP/4ovteIF3VAzWRytqjTvOk2Sua6z0rrnKaDKZ7rn1s1TKSuNlwQXtOG5s9XPXQZaxtc6j7NnnTskw2njlXBmlXfgRceappGsf3MPzpxZX0tOarEedLWJwr7FIjXD3HTXsskIHG167xzy+jhw0nUnIsyvmtso1jW7+2/iHulzEOCaE3ZwwJgDkoebZpZMerBf/Jxi2z8b1qIbOna6V24/Cxs2KVh9zJMevFHnliJRjliW79j217DugQiD9HOf6XpU3a/KHHTc5l+70oi3c0chxis5EEwDaEMtxvBMdz5ZqGeKs2mcSWAbbNHZKzxa3/iBJeqFe56K31NHTHTlH1iI5t22TwOjP3DIPaag2riW1YuQiowwJUifEzl9GSD08STFbBwPjGqiQj+9PooT2jvqvf3Uw9O7WlNevfp95d2tHQXp2ineG6rCvXJpg7r7YuuHHahvRZznzSpn4QgI9/777Z9MDsylMs07SM7+mw1f5OW8eccXRyqGdTGnf5lAMHR11tvwlpeiafgcDrjmmL6K4pS2nTB1tp38E96EfH72V8Xsef7YN7dKS9BnWjp+avDsoz5nhjI8QhHBx0bSQA9kWO0a/eHc+Va+rKt2RA3KKJjnOlGnDmDsXZ4o6jqz1s4ii7YJFml9vPOiAKDrNixZVHN4+udie35i5nXu5YJp/g2k8Fyll5xRydTG1cZM0yj/StLwQ4ezZsGis+sKn2t3rW6J5z8bFNvOK05wReoE3c5Th/N+05l6afjsd88q+foxnMeuU2nTF3CJ6xzUa4X+9xCAcDUxsJgLOgZ+lb747H4ZqCh7U4JcsXGn7FuXpt9fqKs+s5XMwQnC1uVs5Up1hXpcIVJ93588n6zVmx4sjkkoHRZZ0ho42vrJODe9Kfbk6u/TCvS1sOXj5tQvisz7zSp7wIpK3L3ft0plfeeI+lXFrtb+7XJkxkqo2Oe7rfENtzJb6udc+5nXt2jPRD3eyB3TvQ1m3b6YmX3zBmZ9Pm0z2vTc/2JKghv7SZDFbvcQjLUQ2NJADOgl6DB8AcrmmO8BZ2aA43DcK71CkurLIWwVw4hibOncsYSpzkWFyboH9ebctqQ5G7/hBIW1Ommt06FDg10Tnr15VvaxvTdTybhW14TTz34BZDmJ7tyXlCy6nTQwJgs3UlALZ5fob79ex4+LSUdha865ttknN57Mj+NGnOilSOaAbTtOjqMrdqizrD4J/qeKecrCDGOfFXz9HspS2PAIZgvhlgF95qKOwwThomJo6gbn7lM0k9wLe78sHKU97SdHDNACsu+19eWkX/9/q79Ma7lYX3s2aLQ2IeH6tWds9LHxk3fwRsGc20CjFx6cZ+eEro5PlvVgg9dmR/WrvxAzJVBIp34PxexP18xdsbUrnLVx73MZqzdF3q3grTutH9Pe05pnten3/XLHpgzgqrITl6WwexNKjnOCQrNhIAZ0UwpX+9Op6NV+XKbdKNp6s/O+7MMdrNTVlM6DK3rsatmjvOJ7XxMtPw8+UA2+bMglFaXxuXWFeP1sbtS3JqO7drXVEfNE0mVw5wGp9QZ99k8Ckc4Ly8S8bNC4G0fQK3nLoffeInT9K7m7Y0T9+5bROODaD3NEcS62R0WbO++zNM2KCmMF5i4/Inn6um56WpfvbNp+xLh1z3BK1/f1uLafE7NfWSI1tshMPYOr5wUl7X30lfX6jXOMQXj3g/CYBDoGgYo14dz5Q9GNa3M51+0JCoBiOHb6tgs/G5VLs83pZd5jad9KWTL42XaZrTVqc4zVU5Wec8XN2GX2hun9IBmac+Xds7VYHwrYWKDM91J+xt9OlacnBrZfc8fEnGrC4Cpn0CLl9tkhKjsgOqspgyyLiPUylxgX87uGdHY9WW+Ni254xqi7W6dft2Qo3g5BXP1JrGM51Yafr71ceNoNM+PqTFVKaxj927H63btEWqQFTXzVNnkwA4R2PUawDswpHkwGvjc6kx8uBLucwNOdLOnefKFxo/yJXHmCFsx8VEzeVij4uOHs4R0drGNqerDtYJAzaold0DqiBDFQwB23pIEzeNE4x+vmuJKxPGn/LaWzRrSSW1DNxmxdU1jWfiP5v+rtOnaGuyXuOQEMtGAuAQKBrGqBfHS3KiTKf26DK0qu+rb7xHMxevJWRRd+nVst4t9+1ed8a8gp7Dg4y3GdC9ffRJ79GXXteeZJQ0KXRzyQAn+yuO6ePz3jBy4rJkuH0zgRzcdLqoGr42Lt41Y0dQ66amZi63jdvN9YUeHXegQ3frRfsN6UHL397kVPGD689Kb1090GQNY05d47gPtCKKMmGuX0uStkjLZKVlrXN87MnQBUPAheuKL3fcNahT0/acVBlT17XAlQnzT124hibOWl4h3siB3eiLowdFz6LpC9fQLM3+i1GDutHspe9U9E3LDCdPCTXJimfhB9u2sesIh3KjeolDQuERH0cC4DxQ/XDMenA8HVcqRB3XOCdLN4eOQ2biTHH4r2cjShwAACAASURBVDbectwNTPxjLgdYFzCaalaqthweXJqrcjDQyeXKX3XBUecnNm431xfSsEjD0uTP27ZvpxmL1lYMq6sHmsSMsx7SOJeqXqrPo8jGwc6DN+8jp/SpDQKuXFf4i+45Bx83rZHkMyyNU4/1//SFR9B//mEGTUustzROrGn9xFFV/Td/sI0OuvaJFhxgF/R1zygdN9jldwL4bdm2jWYy6wi7yGtrWw9xiE1H3/sSAPsix+hXD45neptFNrZ1U6vUGr+2t/Y4JyvJo9y8ZStdMaly57/vTnybLHFzIkvRtk2TVjef2rlpc8dPQ3LhTevcz5WL6pM15uD4id170Wf26kemLwVJ2ZM25fpC2hI0ZdPT/BnjPfLS67Rq3SbaqWt7+vSInSoytBz9lVxxGdL6Zcn8Y660WslZx2Y85qRJgRFw5boqf9E9S6DmPdOX0l/mrYp4rJ8c3if6DdBxedH/i7c8q82mYq+DLvuK8dP89fYpi+my++dWoK3be7Lh/S10yq+nGuexmQy/TcnMbhyTVe9s0vKcTfilPQt9q/7YdFD36yEO4erq2k4CYFfEHNrXg+Nl4TPZeFtxTlYSVpd5OW1tssTn9+WpmVyjqPWSObhx7aLDj4u5DW/uOBwb+ugcH9dFlrheeftAVr0cHmvStEQIuHJdbWvRRXVTLdy+XdvRqnWVZQYxdtr8rj7uslaTetlwCClL2u+gC96mtvUQh4TAQTeGBMB5IVvSIwi5/Mhk5lLHM7PtJk5783XJTnLa+mbuTO6Rxp2NnwE/dMdONGJAV/rBQ//QDpXkNbtycpPtkxzbNM6tCZO0rDQHR2SAe3VuR6vf20xPvbLausKQNTl+34GU5NWqjDhnzuQkO/foQKOH9KRrPr9nixJFprF0u7l1grvIEioDzPGJPCqLWA0nDQqPgMkvhvXpTPM1p72F/GJgOg2NkwHW7dd4bN4q0tUbNnH0H5n7urY9x2i6DHC8nyv3Pu25ATzuOfsgp8pJHB1UGwmAzWhJAOziSY5ty+Z4XL5vHAbTOe6mv6u+tnq3LpxWTlsud9UmF+RPm0/HP0MFzZbVIz9C0HZuvSufNclLS+Pc2jDh1tN1WRZcfl2yrrKNQ22SQVf/U1ejU1fPUzcmXm6S/MIu7VrT8H5daXqM05jEDlj7zMvxbcipk4u71lzsJ23LhYBtjSef5SE54zqfxHp8ysIBTttrkURft86SHH1Xi9n2KZh+A9Q8pudmWm3grHtA0nQsWxziaq8s7SUAzoKepW/ZHM/G933Y8EZt2iEbrwGLKhDPG6pAmGB04bRy2sbb4Pz36YvW0P2aepG2LEhaxtm0AznNVdR8nEw2Jwthc+m4fgoTk211WCRxxHzgBeqyM0oWlVU+blR/mjR7RQt+telLQVJO8A//PHclvbrqvYijjU+HyPIuW7sxqgLx4vJ36BVNViv5peH8P86y1gk1YWhbI4vXbDDWNjVxGNP8jesTtsx03jxDm8/J/dohYFvjIfchJLUEF/eyiXNp4VvrCV/D1BcZUw1ibhUKk8xpX7VU/WGsUXWK5849O9LWbdvpiZffiHjN3TvuwP5NcOXeo/1dU5fQL558lX3SZAivKVscEkJn7hgSAHOR8mhXNsez8ZpqySfzgN/axaavaYC0fqYalGnCKL6Zqzy+HLc8alfaZAnJ7TNhaeIcJjl23Ha6eVxtFB/Dpy+3jw3/vHmG1sUmDWqOANeXai2ozZchn+l5klVH1/6u7SG7T58sNilbHJJFV9e+EgC7IubQvmyOZ8s2+e4odoDM2pTDh7QO8mEDm76mccY9t5i+/0DlbmTweWcsXqutQZkmky0DrE4iS9aa5VZZSM7tUkkDJ671696hOatpqndry0D6ZDnjNUNN/OC4P0xZ8JZ213eSY2fiJvbt0o6+eeTuqbV5Tbbn1ArmZLqTtuLytG34SwaY+1So33Yc/rtaTwtXr6c33tlEaza8T02tWjnVrM76jDatsbhlXKu8cE8pdf1N4K7PuOzcZ0goTyxbHBJKb844EgBzUPJsUzbHs/ENTfdxTnry7HgOl9YVVpt8ruOZOGpTEme7x8dNq0cJPu8tp+5XgUXrVkRbcYqG5kpyXU38NV2tWU6dXQ6fDWJx6mym1bv1rZGs5jbxY1Ez9Oxxz9OUhWua0VOY6eY0YR3HOY0vi0nS6k3ruMicWsFpXPk03qWNw5mGhQIsj7Xoutakfe0RsPHQ2zQ1URp/Nq1Or9Iu6zOa8xxy3RcRR96mg6v83PWpNvSa9LPJlcV7yhaHZNHVta8EwK6IObQvo+PZuLS6+z6ZLQcYm5u6vp3b5vAZz5ZpQ2YCPNc47+2K4/agh198PeK9gnuMS1c7E39P45Xp9EnWY05ybHWcW1O9YRNH1YZjsvbla6vXO53MhvFNvFzTjvG0E6cG9ehAS9durBA7njWy1Ql1yZInK3mk+dUX9htI459fllo/Oym4jcOZxN+Hb2+zsdyvDwTS+O/7D+1JF094MVXRLPsjvjRmZyuIprWT/BKVVjPd9gy16WD7DfRdn+iXtofglAMHW/HxaVDGOMRHT58+EgD7oMbs0yiOVy1OU+h5fMaz8dNs9SM5rmObIz5GiPnUeC7zhpbBtWYo9MZ1898WVEAKvuusJW9X/D2JlWttXq6/cNtxfCHeJq9xXeWQ9uVFII3/jsyqbj25rPWsPpq1P+dZFvKZ6bo+Q+nn4oGNEoe4YKLaSgDsgxqzTz05XhqvyyeTaoIwra6t6fQdZAmxhTe+y5gzvu00H90YtgxwMhvIdJXmZtD/ovEvsHnEKpvhw7tL9tn0wVa68sHK0/dsOqRlVLicQhMvd9SgbtrTpK4ZO4KeX/y2FidujdM0W8Z1UjqY6opCltZNTbTkwwoQJhyT7ZAN1mWyTLZ0rT1qs5vcbzwETOsMHPEiZ4B161Gtt+Q6sj1DOc8r09hxj+E+Pzn1wG1Z6SyeWk9xSBYcdH0lAA6NaGy8enE8Gy/Kdp8LsW6cJIc1jU+LedL4jpzxbfUYbZyvLFwu29hJHNP4nz56dG7Xmt7bvNVorjQOsCmQ43IKwcs98MePV8w/enAPampqRdNiHGCdHDYfM9XmtHF6bTbRyaLDcf8hPahVq5Z6cOssc3i+Nnvb8JH7jYFA2r6HbduoosZ1HBXOsy3rb4Gtv899rg62sZPBb/LZ1qltE61/v2XF92R9cZc5QnlkvcQhofCIjyMBcB6ofjhmvTgeJ8Prypvyya5yTWXa8Z6WQUue+542l9L1Ty+upKc1J535vs3bsstxmVRliLS6mT6VF5J6J7l3uM/lsHL0ict427ML6YpJlRnoJM/ZpfqFrcZpWj1S6Grb5c2VBThyak7b1ppr7VHumpF2jYOAqTavyfe4FRQUgll/C9L629ZHFh1sY8c9hPNsU+2Tz+Gs+Lh6ar3EIa56c9pLAMxBybNNvThetXhLvhzUpHlMNU9D61Gt8XTuF+ex+cjBxToLX44zh48enHEVZlnkxxg2bLmyZOUl++Dk+diSbg2KgM3XiwCLTUbb/TQdXPpy1z3my/oMyop7vcQhWXHQ9ZcAOA9UPxyzTI4XguPrw0P1fatOMxsyFl85eCgluWG2bJ5qj8+EqPP72ur3qIla0X5DetBuvTuzx0MVgh07t6Ndev3z5COUF7p7xhJ6fN4btH37durZqS317dqeBvVERYhWEXcUJxOtfm8zPaXJKOt0rVYGmJvNjnN91SlLb767mSbMWp66wnz4cVmyL5zlHvdjG0+cK0vaaYnx3fHGDFafztS/e4eoHisySDquNtdWHAykTX0g4PJMdsmAmtBxmS8NYVcePKeWuu0rn6k+L9buD8buSZPmrGjm+XO//EDHWq/LMsUh1V51EgDniHhZHM/GS7LdB4ScNjaobXxL9HfhBLueEx/tgtbUNFZyu44XyduuNQ3bqUu0YcvnSuPlhuYAc2sGJ/VIs1ua/L78ON18rvzktB/wJLcvDRcur/y3p4+mM38/XVvPOM6f5qwBHaYcfqaP/0mf8iLg+kx2bc95Dvhw09Pk0NX+ttVS5zzXMKduP4DSMTmG7nnD4QDXwpvKEofUAhsJgHNEvSyOx3nzt/GWOGNwoE7jOOJNHBnVSbNXNNdRPWavneiqSfNo+uI1tHRNeu1XzG+rqWqqO6tk1/G5LrrvBWumk6N7vA24q8gUczOoNvvo5k/2cakZHB+PmwXVyeDLj9Ppi/G5/GSTPXx44lwcuTay+ahO9qzVR1z9U9oXHwGfZzLXR3Xa+8znM45Nxvh92xccNb/PMyy5N2Hzlq3aPQySAS7uWpEAOEfblCUAduE+meAKMYYa22cs1z6m9n27tqNV6zYbvULH53Lhg3HdLa3ObXyMWvPLIEsW/YsgfxxPVz/i2tOnnQuuRcPRR1/pExaBavtyqPlCjZP2bHKpCW6yCneMWq/NssQhYb2fN5oEwDycvFqVxfFMb7879+hAo4f0bOaxjp+5rJkDxeXXJt9+ORwxTiYhOY6Jk2WqH5lWU3jWUjNdIV5VAJ/jgImpPqyX03zYCdkFlP+yncz0id170Wf26lfBT84yd7KvzWY+2RM1R5zLHFJm3VgmPTic32QN32NH9m/BCTTV9M2ikwuutmoXWeSQvv9EwLYOioZTtU8dCzWf7fnPWceDe3aMnoncU0pd1pqyc/yZMKB7e5q+aC09MHtFhRtIBrhoK+MjeSQAztE2ZQmAbbxD8FiH9+saLXB1cfiwPm3UD02Sh2njedk4oFyuZhoHOO4qPrVoXVwNnM5bv7w/nfG7aTQthrtpDB+uHUceDi8wzX84OOUle1w/kx46bm6y1rTO/zm8Qg6+aW1svERd32pgmVWvMvbnrIOi6WXyn7z44qHm8+EAmzj2obj3HA5wUddjWeKQWqwfCYBzRL1Mjqd4U7c+s5Dmv/EeCxVX/qbtzT4+qU8tyCQnK56Vc+F2qjqZqALRilpRp3at6ZlX32JhcshuO9L6zVujig5L11Zyknfv04lG9O8WVYHYuWfHaMxHXnrdWE+41ruNuTZT9npt9fqookXvLu1oaK9OURYGF7i5uDd94RrSZdjzzpKk2d/Gs2YZPqfd3qpO8W3PLvJel1z5pZ0ZAe46KBqGt09ZTJfdP7dCrLzWW6j5TM9/13UMPfEM4uwNiM85sDsq9BAte3sjIZuc3B/BeS5X8+tWmt+VKQ6p9vqRADhHxMvoeKaz4nUwuXKbQnG7fMbx6aN09uFiusyX1hYy3Py3BSwvdbUHZ1AXPWoxHmdOtDHpYarPyx033i4P/LP4oI8O0kePQOh1UC2cqy133vO5ruO81iTnNyGvuV19p4xxiKuOvu0lAPZFjtGvjI5nOitepy74p706t2uR7UNJJ58ajsfvOzDi06Imbt+ubWnWknei/x66Y2Ut3ZVvb9Rmw3xOPuNwJ134YXjr79WlXWqmU+m68MNsqan+LzhmKJ820VJLV9km1Bn3cVuHznxxxsuDa+maOWIs76BZNZvOLj6YV3bPB5N66cPx2yLqaqwr3bcznX7QEDpx/0GUPMbc5otperp8aePixeHoc+tsc+dMawd5vnffbHpg9srU4YqyDssYh4SwE2cMCYA5KHm2KaPj6c6K13EgdZCAf5jGuTLVcNT1iY/PqaVr4z7aeM5p/bl1Z5M8sSRGJnx0WHK4s/F+rvLb8FJjh+Y+2saz3fdcisY61Trfs9kRMoTkAHN05vog166+ODZqP46NioiN7bmX5ANn1ZO712LcmWMqAm8dftzxuFzfrDbi8vKLtA7LGIdktRO3vwTAXKQ82pXV8XRnxSNLAB7Vw3Nfp8nz3zSiYXsT13G7TDt1OZBzMrhqHFt91bQ3dlvdWVNVCTW34oPZdFX6cDhmybGTmRx1P2v2ylZ3k2OneBsffneIbIppXl0d38smztXWYFb28a2ZrMOKax+bD6qd7yY/cLWTtG+JQOh1UC1802qrQwaf0xjTZPepw2sazyWjXA37pH2JGTuqP/Xr3iHiDOdRFcbXX8oah/jq69JPAmAXtBzb1qPj2bhPJk5lGh/KNmYa7D48qzx4ajYdlJyh2sUxsWGQh76OS4HdvCiyVlOOas7FNoQ0rCsE0p478edHaF/MOl7W/qGNyMUx9LxZxqvHOCQLHvG+hQmAt23bRnfddRc988wz9P7779N2fC+PXbfeemsonas2TpkdD1SI7z8wlxa9tb6Zh9uxbRsynZeuQLVlgNEuyTFzyXYmjeeTGeRm3EyOouPI2TK7qmakrWaw7Ux7nUw2DNL0jXOvOZmLLPxAzsLLahudf5myMSZd8PeLxr+g5V7jpEBqRS3WhE4vF5xC6ByXwWXupOxZ+nLsW9Q2Sm/Fy49XMVH1vrEngbNGfHXME/u0zOWwPp2pf/cOdNQefQm/uqaqEa7PCuCQ1bdd+4fG0OW3Sj2H0efuGUvo8Xlv4FFBRwzvTa2bmmj5hxUlqp0dLnMc4ruWuP0KEwBfcMEF9LOf/YxGjhxJ3bp1q5D/ySef5OpUmHZldTwdD7hr+zb01IVH0H/+YYaxLq2NA6w2yCVr/Nr4riYOsG89yyw8N1PfNB6zTT/lsFlrHacF7Lq6yq68uSy4cRdl1jm4/X3smKwPjDUx5ZIjCS+GyQA0rY61Luh0aZ+GJVd/U9AeSg6uvYvQLo0na6svHkr+LHbjyMDlru4/pAe1atWKpi1c0zws57nu+uzJwgE28WtDY6gbb/SQHvTyynX03uatLVQeM6Qn3fHVAwgvS6f+Zkpq7fZq84PLGodw/Dprm8IEwL1796arrrqKzj333Kw6FaZ/WR3PVAkC2S9dDdfkaWQ+HM94Dd9+XdvTzCVraXGsCgSC53umL6W/zFsVvVUjW6Hbwcw1vi9fLC0joepNJmvhmjLcpioaSoeQnE8X7rUpo+yajeHaItnO1zYuGSfXqhCDenTQ1nXGF48bTxrVQgUfnLLoHJ/cZ27VP0tfX1sXoZ9LhQ0lr+2ri6te1cBe1ZTGM9RUSQdy6+qpc09U0+md1be5/UNj6OIXwOyUAwcbM95JXEL7T5q/lTUOcV1DPu0LEwB36dKFZs+eTbvuuquPHoXsU1bHM9UC7tu1Ha1at7kCaxsHNd6haJwuV8fxkd+nj6tcPu1d5XJt7yNT1j5cGV3riZp8H5z3iece3EJsrgxZddX1zzJ3lr556FKtMW28fJ0cLs88jh7Vxt6Vy1pt+TiYJduEltHFL7h7PJTMof1HAmAfjyEqTAB8wgkn0BFHHEFf//rX/TQpYK+yBsCuGWCXt9nQb+nVNruP/D59qqGXq1yu7auhQ3IOroyuGWDT149QGeBQWHH1182XpW8o+WsxjkumT8nn8szj6FRt7NN01ulWbfk4mPmufe7YLn7huncjtP9IAMy1ast2hQmAr7/+erryyivp6KOPpo997GPUrl27FpJefvnlfhrWsFfZAmBF+H/1jffojimLadOWbc3oKQ7w2eOepykxflj/7u3p64fvxqYjuPK00jY1cDc82NrZ7gOE+CaZx+atImyWUZeN06XTOY6bzyYbjsw21/exRdE4okkcjh3Zn878/fQWPqqzj073XXp1ognnfpx0Pn7mIUPpp4+/Qu9u2lKxJlAaML5BSlfvGlzS40b2126ECWFLJZSrTeM+kqWvzdfyuh8CO+x5OPYXz7RY00pe2G3b9u00Y9HaZhXwtzvOOoBVx5ard0jsOZiYOMGmfRUu8nHmd8FFHZBk2oDo+mw2yZf8+6dH7ETH3/xsC7/QccLV2sYz4I13NtGUhW/R65ovpdzfCy423HZli0O4eoVoV5gAeOjQoUZ9QMp/7bXXQuhb1THK5Hi6B1yvzm1pYI+OhMDgms/vGW32AR/r7ulL6KYnF9CKdzaxg8A48FxOV9pD13SoRnJzhe3Bbbuvgt9k4AdMwEMe2qsTq+ajCTefTTYcmbmOzrWFGs+1PVcOn3YmHG798miaNHtFxCFP27W/fvMW+reftwx8ECyj/4SZyyp8HBtgBnbv0MxNv/zYPSqCZRVsQx/UzYYMA7q3p0lzVlZsLIKvcv3YBZ8sNsrS10XGEG1DrAPTc2/sPgNot96dCS9UZ/x+egvb+W6+tekcAnsXTOKcYM6+Co58LvPb8OCMZXqRNT2bTWNyDsTBM/+hbx4SvfikrW2lV7cObWjkwO70qY/1pdZNrWiZVIGwmbzq9wsTAFdd8ypMWKYA2OUTl0vbLDCnzYNyPRdPeLFi+OSnJZustvuYgNOGo6fPJzXduKHk4chc5DZZccjqX9z5s85TZBvUUjYu/mky2saw3a+l/kV8NoTEizMWp00cJ1fqUxJj7u+L6ldNqoPJF8sUh1R7PRUuAH700UfphRdeoB122IFGjBhBn/zkJ6l169bVxiXIfGVyPJcNBC5tswCZNg/GvflvCyqGT24usMlqu48JOG04evpsqtCNG0oejsxFbpMVh6z+xZ0/6zxFtkEtZePinyajbQzb/VrqX8RnQ0i8OGNx2sRxct38msSY+/ui+lVzs5sEwO6rsTAB8Ntvv03/+q//SjNmzKDu3bsTDsZYt24d7bfffvTYY49FfyvbVasA2IeD5fImnfYWfd0Je1dw47jyoF28gHiXDm3owTkrK8yOkjNNTa1yzwCrwu+mwytc3+5rnQHm2qHW64wrp4vPumbLOF8YuOvAJwOsDk7RHcwgRx3/05q+9o/7l+kIc9umJlX2qtZrJTm/LyY6PXTr0LZfIeT8nLFMbUz2kQzwwKK5bE3lKUwAfNZZZ9HUqVOj0+D23HPPCJQ5c+bQqaeeSocccgjdfPPNNQXKZ/JaBMAc3pTpYcfd4KSbQ42Z3HDElce0KUMnKzh4t355f+/NTrYDJ0yF3+Oy2Da+cTGuFgeYawcfPw/Zx0VOl7Zceyi7cri53HXgymVPOzjFx+9C2qdIY/nYX9enS/s2LTY3Jp8PuoMN8uIBZ8XXBxPu2uA8q0LND5k4Y5l+N1w383E4wKbNtKaDL4riI7WIQ7L6cbX6FyYAxkEY48ePp0984hMtdJ88eTKddNJJ9Prrr1cLk2Dz1MLxOG/NJgU5mxxUX7S96L4XaMKs5RXDxTOjXHlcsqOYEHOogydsm51serkcEnH4sN509J47sTa+6XAOdbiFTac8M0PBFohmIK6/xP1QbUjxOaY2DUcOxtx14DKP7Whw1y8Pedqr1mNzbBSXMS0D2Ldbe+2mydunLDYeD/ylMTvXGoKK+V0x0Sng8jxO+mOI+V3Wt6t9TPIl/37cqP6szbSm+YvylaAWcUjhFoVBoMIEwKA4TJkyhYYPH95C1Jdffpn23Xdf2rBhQ1kwbZazFo7nyonKAipnLk4byODCj0X7vLlVXLmz4FfNvmXRpyxyxm0XWmbbWsjb96vpl9Wey8dWPn2qrVfo+Ww+GJ+v1v5Ya/vUen6b7WsRh9hkKsr9wgTARx55ZER9+NnPftYCm2984xv0/PPP07PPPlsUzNhy1MLxXDNobGU8s3VceVwyDhAl77drrtxZ8Ev25XJffeastj5puiTvodTUpDkrolq6Nk6mj+4h++j0Mh0Tq3i8yRrB8bqmcd1VBts0ntJDMsD+FvVZB6Y+uq9Bea5hf63de7o8j2vtjz42dX32muyKv180/gWamPIlVPWtFZ+/FnGIu8fVpkdhAuDnnnsuOglu5MiRdPDBBxNq/z799NPR8cgPP/wwIUAu21ULx+PwpkLhyJmL0wbyuHCA0T5vfhVX7mpimWWuaurjynlNcjDTOJlZMMja16SXjj9oKpgPGabFDpLR6aobT8kuHOBsVvRZB2lcb0jjwhvPJn31eut05nCAqyfhRzP52DQup62/y7pPrlPdXoJqr+VaxCG18AOfOQsTAEc/DNOm0Q033EBz586l7du301577UXf/va36cADD/TRreZ9auV4ITlYNhA5c3HaYJ5kYfZPDu8TFRB/5KXX6elXVleIknfmgSu3DSPO/RBZDNs81dLHp+pBUnYcMWziZNr0zOt+ml5JPrqNx5smY5zf/trq9bT63c3Uu0s79qEreelfL+P6rAPVB6f+TZ7/pvZZxKkcUiYMQ+1XqIbOPjZVctmeva6VI/DsUtWQbJn0vH/DoGOt4pBq2D3rHIUKgLMqU7T+4nhhLFJ0jlUILetJR5+6t0kMa80r1NnUxUYuHMoy6B7Cx+thDB/fLqIv14MtQulgW9eutYPj9rY9B6rhGxKHmD2lpgHwV77ylYjz26VLF8J/p1233nprKH+v2jhlcDwfzloah5O7E58zr2rz5xdXajPAOo5lmWqkxjHIwn21YWm777IgdGMla4Oasp/IjOw7uId2R31ShmpkRlz0Rltbpig+ni3zY8sAF7G6gCteRW3vsh6SbU2+DX+ttwxwUe0XWi4bx9tk830GdadZS9+uEAe+YKshrzpV4zlXhjgktE2549U0AAbnd+LEidEhF/jvtOvJJ5/k6lSYdkV3PBv3SQekro8rZ5Mzr413t/+QHhFPPM6nLBM/MgSOsI8NS9t9l8XC5QWm1bHV3XP1HxeZQ7Z1wdKV067kLJMPh8S2WmO52jBZGz2NB8upHV0tPWUePgK23xrOMyu+ftM4/HGpqrXWix6H8C0VvmVNA+Dw6hRrxKI7nktGSyHLzWylvdly5rXNM3ZUf7p/9grt23cZsmc+9Uh13m3D0nbfZcXYbBIfC1U6nl+8VlsnGvfA7Vb1m7n1Nl1kzautC9fQVB/UJFucO5iX/I0+rst6MLVN+i/43+rLk4t/NLotiqS/jeMdt7npa51av6YqLp/YvRf16tyu6nz+oschtfSDwgTATz31lBYHZPnatm1LAwcOpAEDBtQSK+e5i+54Nu6TTmEbp0n1SeM2cea1zbPPzt1p1pLKz0/V4FQ5O4KmAwcDzjy2cWz3OXOoNjabxMeCHXDdCEAdnQAAIABJREFU/LcFFVOUxUYu2GRZK5w1k1UW6f9PBFzWg0tbwbc+EODY3NbGdr/aSBU9Dqk2HvH5ChMA77DDDrRt27ZINlSAwIXgN34dfvjh0WlxoEyU4Sq647lkQxTe3Cxg3hngNP5VlgywCz/Q1QdDcX7j89psaLvvogPX9hgzjRNZpEwnx96ubQZ0b4+nFz02b5W2YoAJ82rwAV3sXY9tbXzPeDY35NqpNZbw4btnLKHH571B+FU9ao++dOL+g5oz17WWz3V+05rMuj+FY3NbG9t9pSvnueKKi6590eOQEDr6jlGYAPjOO++kSy65hH7xi1/QoYceGumDk+HOO+88Ouecc6JSaBdccAGNGjWKbrnlFl99q9qv6I7nwoeLL9okL651K6Kt/3xnia6u7dvQlEuOpI5t22jx5sybxstKckbVJFk5VRy5fB0oFOc3Ob9NZtt9F324HOC0uqihbOUit6ktBxvfNmnylZ2/HgL7Wo1h43vGnyEc29dKD5d5TXz0vGupu8jo0tZkFx331nV/Acfmtja2+9CV08YFk7S2RY9DQunpM05hAuDddtuNfvnLX9KnP/3pFno88cQTdPbZZ9Mrr7xCOCzj+OOPp5UrV/roWvU+ZXA8H85avI9v9QLOvKpNvBbqm+9u1vJKQ2QVuW/uPo4UivOrm9uGpe2+iz6utUHR/qL7XtDarNYZT469s7SJ4zqsb2fq361Dc+YN98Y/v6yZBx3PPLrYQ9q6I2Dje8b9MuTacZc0TI+0Lze1XoM+GrrW5U3OYdOZY3NbG9t9znPFBxtdnzLEIaF0dR2nMAFwp06doiOPhw8f3kKHl19+mfbZZx/auHEjLVmyJLq/YcMGVz1r0r4RHK/afKc85yvr2DVxbodJ88TVQYyKphy5srSJT9govOcs9qh2X45tqy1THvOlcffL6JeudXmTmBZB52r6XiPEIb7rpjAB8GGHHUa77LIL/eY3v6HWrVtH+mzdupXOOuss+sc//hHRIcaNG0fXXnstvfTSS776VrVf2R2Pw1Gq5pssjJdnJjUPXRSGj6ScIOXLWebYRzm8ra3tvuvCiXMOV769kea/8V7FELZMjOucru05fFDTju54DWrTV5C4PJwvFPXG03S1R7Xbpz1L1EleWWQKsaZcxjC1bZQMsGlfSNKG6rnjgm0WP9D1zeO3xiRj2eOQ0NjHxytMAIzs75FHHkk9evSg/fbbL9oQh7+988479Mgjj0T/j01wP//5z+lrX/tanpgEG7vMjsflKHHbhQI1Ly4t5Autiwvf0BUfF1ltbW33fWQ79TdTaNqitcauWfnarjLp2nPso+MV6uqCdm7Xmt7bvDVVrDSd642nGcI+eY+RZv+s/hliTbmMkdYWe8p167GeOMDKV5KcXxMHuNY1m11sm3UdlDkOyaq7rX9hAmAICm4vNrjNmjWL2rRpQyNHjqRzzz2XevfuHWWBly9fTp/61KdsOhXmfpkdz+UN1cZ3Cm2QEBxkk0whdeFkGH1PrnOxj62t7b6r/dIyTocP601H77kTFYXzyuGDQtY4X9d0MhQHJ1PWu96ydBwsitAmL456iDXlMoatLfS8Z/pS+su8VXVRBSLNbvja0rdbe8KppKYa4za8quGbIX9r0uQtcxyStx0KFQArZTdv3hzV/k2WQcsbjNDjl9nxqslRyoJ7keXMUzaXsW1tbfdd7VNGzqELBi71kJPYmfiHZcTM1S+K2t7F9lwdQozpMoZLW64ORW+XRecsfYuOS1K+MscheWNdqAAY2d/rr78+2uw2f/58+slPfkL9+vWjyy+/PG8cchm/zI4X4g3ZxrGK31e1U5e/vTF6c9dlCHXtTbVWcXLPKQcOzmzXLLzMEBgqBZJydOnQhh6cU1kNRZdhtMlhu+8KYhmzmS4YpOlnwyqZAVd2HffsYi1PGuNx+MO2eYt83/acyFt2F9tzZXEdU4eBiX/ussbj2dCifHWJP9PGz1xGS9ZsMD7z0daV28zZW+Bin1r7J9fnTO3KHIdk1d3WvzABMOoAg+7wrW99KwqC586dSw899BBddNFFdNVVV9GFF15o08X5/h/+8IdoU91rr71Gu+66K1155ZX0xS9+MRpn9uzZdP7559OMGTNoxx13pG9+85v03e9+12mOMjteVo6SrT+HfznuzDHNhdpt7ZOGCcFvy8rLtGHAdSaTHFydbXLY7nPljP+wlY1z6IKBrq2OF2yqVw2cwDEFv/iM301L5UorTLNyUl1tWK32LrjnJVMeMriMaWqr45+b/EA3hmsN3Lzw1Y3LxcfGbU7WpOeukxDz+9LXqokz5ipzHJI3VoUJgPfdd98o4Dz99NOpS5cuNGfOnKgqxG233UbXXHNNVAc45HX77bfTl7/8Zbrhhhvos5/9LN11111RAPzMM8/QsGHDonJrY8eOjQ7fQAUKBOc33XQTnXHGGWwxyu54WThKtjdsThYt/ibPaZ80DCcTkGbMEJnMLBgq2bi6p2W9bXLY7rOd/sOGZeQcumDAqYes+IcPGyqAIDs3YdZyNrRZ/Zk9URUb2p4T1RLFxfZcmbhjpmGQ5J+nZXHz3BvB1Znbjmt3Wzsuxjq5OH1t83P1rWW7sscheWJXmAAYdYCR9R06dGiLAHjhwoW0xx57RHWAQ104ahnB9QknnBDRLNSFQzhQaQIXDuVYtGhRtBkPF06pmzBhAqEuMfdqZMezcaw4PMo4X5LTPmmXrPUei8LL5OqeVV+uX0s7NwSy1i1Vs9WjfW3PCTeky9k6DwzyGDMkulz5uO1CyhYfq9bzh9CrkeMQG36FCYBBQcAxyMccc0yLABiZ2iuuuIIWLFhg04V9H0Hsxz72MZo5c2Z0yEby+sxnPkPdu3cn0DLU9fjjj9NRRx1Fq1atoj59+rDmytvxXDi0Oh4TSsEoDhaHg8tS+sNGtjdnTlYz7wywjdsVIgPsglmyrZLPVEM42V7xS48d2Z8mzVlh5daFkM3G38syR730NfmRZIDNdb2vOm4Patumda4+XBT/GvfcYvr+A3MrxOFk/PPgx1YDF9vvg5KB2872LPfViTu/7/jV6Jd3HFINHfKaozABMHi/N998c0RJOO200+jee++NMrCXXXZZREO49NJLg2EwadIk+tznPkdPP/00/eAHP4jKriHzjLmOPfZY2nvvvaNA/Lrrrmuec968eTRixAiaPn06jR49miVLno5n48TazrTXcRbjSnG5VCYgbBwrF/kxh629K9/NJp+a8+RfP0czFr/dQk3MNfWSI6lj239+HcjjsumbNqcrFq7yc7BzHbOe26dxPIUDvE1bozZvHy6Kv2XZZ5AXP7Ya2HCfIZx2nDa+OuU5tq9Mrv3yjENcZSla+8IEwAAGNIOf/vSntGnTpggn0A/OPvvs6G9NTU3BsENW+T/+4z+ioBfZZdQbHj9+PP3whz+kxx57LDpo4+STT6arr766eU61UQ5B8yGHHKKVZd26dYR/1IW6xmPGjKGlS5fSwIEDg8mPgVwyqJy2OuE4GYg0pWwcq/j9gd07REMtS6kCkdbeVO/RJB/3zf72KYvpsvv9sjNZDG6Sb1jfztS/Wwf65PA+1LqpVVTXc/L8N61TZbVlfAIudlahGqiBaS3EudKgZvXs1Jb6dm1PO/fsaF0P9QKfaY0l9Qvpw0XBzrSWOFVsbOvQ9vytNQZc+WztbDhk1dM2f9bx8+4vAbAZ4cIEwJMnT6aPf/zj9MEHHxCyrTj5DRvRunbtGtw/7rnnHjrppJOiTW3nnHNO8/jYDIcLZdhAg9BlgE20CfTDJjpUrEheeQTAHF6o4gxy2upArkfOodKTy+3itgvtpNx5ubYNaUuubKExkfHqE4Fa+HBRkMyylrL0LYr+IeQQHNJRlAC4BAEwTnt79NFHCdUg8r6effZZOvjggyMKRDybi1JrKL02ZMiQqPTZuHHjmkXhcIAlA8yzHD4r3T1jCT0+7w3nU4lcuF5pbblZA247nub8Vtx5TfzB5Ewh64FyZeNrW52WLpz5pEQufqf6mvr4jJUVoVrMyZWZ+4WqSBngUHhmWUtZ+nJtU7R2Otxd6iX7rutQ9q4FnhIAlyAABr8Wm+COOOKI3H0EFSX69u0bHbARr+0L3u8OO+wQZaJxKAc23rVu3TqS5+KLL6aJEycWpgqEjSNaaw6wyYh5cd6SNRlt3C3b/XgQ41trMosjc+QzYZnkT4bmU3Jky6J7Hn1d1ovuR9LVB9J4v2f+fjpNWbimeZqsfHsbXkW3l06+0D5rw8jlfkg8s4yVpa+LvkVpG3pNcfHjtisKTkk5JAAuQQCMjW4IOkE92H333alDh39yQtUV+jQ41BbGxrtf/epXEU/3j3/8YxQQP/HEE1HZNdAvjjvuuOgAjmnTpkVUCciHOsXcK2/Hc+HQ2uqWcji4XL3T2mWprOCS8eC05XK7uO1C4BMfwzavScerjxtBbds00eI1G2jVO5u0tWazZtNssoXGIut4nCyjCROOLyXlc638kNUePmsuzzld7ZX0J1dOv+t8Wdr7+EPafFnWUpa+WTCoRd803F3qJSvZuXbktqsFJpw5845DODIUtU1hOMDYkGa6WrVqFZ3WFvq68cYbo6zz8uXLo7Jo4O+iOgQuVHvA6W+oEIHjmL/zne/Qeeed5ySCOF4lXFlq67pwvVzaOhm1QI05OnLaFEil3ETh8ExNPGkfDF1r/4bkaCdB9JE/N0PUwcCCZ22MGBp37njcdrVBxT6rxCFmjAoTANvNWL4WtXI8H76Srk+8TvDgnh0pxFnyRcoAh/YoH9yzyMDJTHDaZJGhLH0lA/xihalUBjgLN7os9g8pZ4g1Va3nbUi9az1WCNzjOtjGUzYy1WEv0heUNNvUKg6ptb9w5pcAmIOSZ5taOJ4PX0nXR1cnOARXsSgcYE+TGrv54J5VBs6cnDZZ5ShDf+EAT9XyjvGSm+Q3x+0ZYs2XwT9cZMy6pqr5vHXRq+hts+Ke1C9tvHpaF7WIQ4ruS0o+CYBztFQtHM/2VqtTl5MdU/1CvPXGa5+2IqKj9uhLJ+4/iJIb2XSyunDeXNpmdQMf3LPOif4cHTltQshS9DFcOPNJXXwwTKv9O/75ZRFHO9SXFRv2Jlk4az/EmrfJV7b7Pv6gdORgHvJ5WzZs0+TNgrvL74nJRuq0zRBfQ6tll1rEIdXSLes8EgBnRTClfy0cz4evxOFHKjXz5CrmaIrch/bBPXehZAJBwIIAZ+3Lmg/rRhzM5XkbFnPX0erpeV6LOMQV71q1L0wAjFOQsNmtnq5aOJ5PJrIMGYkQ/Fo1xsLV62n1u5upd5d2NLRXJydus0kOH9zryddFFzMCNr7ngO7tiagVLU85BdHV/9HeVGs7PpapSkhcmzwywK761JN/leF5Ww28i+wD9fQ8r0UcUg3/CTFHYQLg0aNH06233kp77713CL0KMUYtHM+HJ1V0TpqPTkkHSOOAcnmOrpwx7riFcFYRIhcEuGsrPnnSb1z9P41nf+uX96dkHeJkzd00WUKA5KpPiDmLNAbXJ+r5+VF0Hyi6fC7+XIs4xEW+WrYtTADcq1cvmjp1Ku266661xCPo3LVyPB+elK1OcLW4ijoDhHgbt2VdOFkumxw+uAd1OBmscAjY/M4kcNwfbX6XHCNtTpwIOGHW8opp1UmB1agH7qpP4YwaQKAiP28DqGcdogw+UC/P81rFIVYnKECDwgTAP/nJT+hPf/oTfe9736Pddtut4iCMnXfeuQBwuYkgjueGl6l1CD6WjXfH4TmGkCMMIjJKWRCw+Z1Jj7g/uvpd2pz77NydZi15u2Jajv+HwtxVn1DzyjjFQUB8oHq2kDjEjHVhAuCmpqZmKeNcYMUN3rp1a/U8JtBMZXC8IvOwlBlCZAtsmbgQGWCu24TEPORYOvnzHp+LWVnbjXtuMX3/gbnO4v9w7J50yoGDo36u/u+TAdbVBM7rq4+LPrX2v7T5ay2bs1MRUVFkdvEBHz25fYqCB1den3ZliEN89ArRpzAB8OTJk1P1Oeyww0LoW9Uxiu54ZeE5hZDTxIuEQxwwtCfdftYB1jJsoeRI1l315fqFkCdtQeQ9flUXYw0mM/nc/kN6RBt+py1cY5Qq7pOudnDlACv/09U+9fXNEH7lqndoE9cb57/WeMbtUwRZiiBDaJ/VjVf0OKQaGJjmKEwAHBdw8+bN1K5du1riEmTuojteUd7COWCH4GPdPmUxXXZ/ZTYunm2zyZJVjpCYhxxLp3fe49uwLvt9E37wN9S9VrWAV769ke6fvaJC3fhXCVe/S6u17VoTmPN1xNVWHH1q7X9p828noosnmE/Xc8WjGu1rjWdSR44P5IlL0fDIS9eixyF56c0Zt1AB8C233ELXXXcdLV26lObPn0/gBffr148uv/xyji6Fa1N0x2s0HlYR9A0pQ8ixdIsn7/ELt2ADC8TFj9susHgVwxVFDiVYreVJmx8y3vy3BRUYVpNL7eoPtcbTVd682zcKHkWPQ/K2c9r4hQmA77zzTjr33HPpW9/6Fl1//fU0d+5ceuihh+iiiy6iq666ii688MJa4uQ1d9Edr1HegJXxfPT15YiZ+vnIYHI+n7F0cuHT9/iZy2hJ4mQy0/hXHbcHtW3TuqK91yIJ0MnXRgGmTh2Cax9Tu2vGjqDWTU3NOB87sj9NmrMiN9y58pqUNtnBZp9qrBUfW+edAYbeplrNrvLaMMZ4We3rKlPR2+eFB8cW1cSm6HFINbFIzlWYAHjfffel888/n04//XTq0qULzZkzh3bZZRe67bbb6JprrqFXXnmlljh5zV10x2sUDpQynqu+ru0584TkWbrKp2s/ZmjPSOw4HzXOCT31N1No2qK1Lfw/WTc2D54od8G5YsAdN0Q7rmxcu+SNO1deHTamvr89fXRF3eG4vxSZZ5unbGk8bc5+hLgNuHbjtgvh+2UYIw888hgzK5ZFj0Oy6pelf2EC4E6dOkVZ36FDh7YIgBcuXEh77LEHbdy4MYueNelbBserNQ+r2oZx0dc3Q2Dr5yKDDR+XsWyVMOJzKd6niTedlCsPnqhN9zJktbj2Sbbbsm27lq+eN+5ceZNymHzLVHdY+Us11wrHn5Jt0vDwxSrNb3HPdS3ZMIzrlEVmH/yK3ic0Hi62qBY2ZYhDqoVFcp7CBMA4AOMXv/gFHXPMMS0C4Ntvv52uuOIKWrCgkm9VK9C484rjcZEqZjtfjphvv7xRcKlJq7iM3D614j4WFeustiw67kn9TPLa6g7Xq/1s9k+zr+taalQMbRjX4n4RbSFxiNkTChMAg/d788030w033ECnnXYa3XvvvbRo0SK67LLL6IILLqBLL720Fv6caU5xvEzw5daZy70zvc0P69uZTj9oSLSTf4fWH9WvVgJXKwvgyrn0yQBz+7hmrUIZt1pYh5IX43A4gr64u/qETi+OfHlngA8f1puO3nMn+sJ+A63lCZUsPnKHtKvLWGn2dV1LZVwDLlhVs21WHyqiLSQOKUEADBEvueQS+ulPf0qbNm2KJG7Tpg2dffbZ0d/iB2VUc0FkmUscLwt6+fR14d7p+FxxqUz1g6vBA/PhXOr4x2kcYAT3unny5qK6WL4aWLvIY2vLldcHdx+fSL7AceVL6uk7t22NcfnlvnLb7JXXfZfnkE2Gsulu06dW90PgGGKM0PpLHFKSABhibtiwgebNm0fbtm2j4cOHU9euXUP7Q9XGE8erGtTsiVwzL+CIXXTfCzRh1nLtHKZsTWhuWaiMm04ujK1q0upO/0r2OW5Uf5o0ewUtTlSNYBshcMO8sQ4prkuGyBV3Xx5uXD8X+ZK4mOxgs4+6//Dc12ny/Dcr4OZkRLPIHdK+LmOl1Wp2GQdtbRi7jteI7UP5UNFsIXFISQLgd999l/74xz/Siy++SK1bt6b99tuPTjjhBGrfvn0p16M4XvHM5sO98+mTt+a+nMu85ZLx0xHIkyMYwifylM/mG1nmztLXJpfcbwwE6tWHJA4pQQD88ssv0xFHHEEIgocNGxZlgFH6bMCAAfTXv/6VBg4cWLpVKI6Xj8my8LRcM8DQwKdPPpp/NKpJpn0GdadZS992zqJlwTRvXetpfN8sE4e3XusMcFY7+WKTtkZ12WNfX/ftlxUX6V8dBLL4X3Uk9JtF4pASBMBHHXUU7bDDDnTHHXdQjx49IolXr15NJ598MnXu3JkmTJjgZ/0a9hLHCw9+Vo6VD/fOp094zVuOyOGIqh42HmVWTPPWtZ7G98Ga63++PNw4vj7yhbJPlrm5fbntkjr59guFjYyTPwL1amOJQ0oQACPInTp1Ko0YMaKFtDgQ49BDD6V169blvwICzyCOFxjQQKcZ+XDvfPqE177liHGu2ap3Nml5yqjDet0Je6fupK/XzEfe+PuO78oRdPkC4cvDjeviKp8vDrp+Webm9PX1dd9+IbGRsfJHgOND+UsRdgaJQ0oQAIP28POf/5w+/elPt5D26aefplNOOYWWLFkS1iuqMJo4XniQ65WnlRWpLLhk6ZtVbulvR6CIHHS71MVs4evrvv2KiYJI1UgISBxSggD4wQcfpG9961t044030mGHHRbRIaZPn07nnnsunXXWWdFmOHXtvPPOpfBfcbzwZpJMjB7TLLhk6RvewjJiEgGXDLCgl46Ar6/79hN7CAK1RkDikBIEwPE6v61atWqWeDuKlxIR/ob/xr+3bt1aa59izZ+343E2xrAEDdgo740i9crTymqCLLhk6ZtV7kbun1wrx47sT5PmrKAlifJyXA5wGpZ5r8tQc8efaXje9+zUlvp2bU9De3VyOhQjLk9c9wHd29OkOStp2sI1zU36d29PXz98N+PBNmioWyOcfo3s32XSvZbrI2+c8o5D8pY/z/ELcxLc5MmT2XoiQ1yGK0/HC/GjGBrDagVS9cjTCmGLLLhk6RtC9kYbg7OJMb55MQsHvVrrUmdDl7lNzzQ1rm0zJ3d+HP7y2b12ol9Nfo1WvPPPQ5dw2caHDe6evoRuenKBU79G8+2y6evio2XTDfLmGYeUEY+4zIUJgMsOpE7+PB2viJ9F5TNhPXqx6JQHAr5HHfvIUst16TI3BxPOoRhxjEKUhuOM5yqXjx2lTz4IuPhoPhLkO2qecUi+kuc/ugTAOWKcp+MVcWOMbBTJ0Zlk6LpCIG39xhU95/Bd6aKjh2fSvZbr0mVuDiaueIQ4HCQOvos+mYwmnauGQL3bNM84pGpGymkiCYBzAjbvTw/cDPB7m7bQ9x+YS4veWk9Dd+xE13x+T+rYtk2kdWjeU72/SYfGK0fXcx663nQruj6cbCeMiDJ2fbu1J90R1f+/vTMBk6JK8njQ3UADAi0IiJwtxzAKCIog4niM4qCOLIeDN+jgqrAqDgoseKLggI7XrhwqKAqjotyy46iw3tqc3SMtq4zchwMohyA02tL7RWqV1VV5vJf1XubLqn9933470u/Fi/hFZHZ0VGSkiJOZw6i5n9J8m1d5O1Utk3tmiarQ9n2HfengVoFNtkuEiWyl1Ulmw9rVadeBIykIveQ7yetzygm09/APxE+v9DypUaV+YtNjUSSOMnlNGL+3gowJJMDO0YsEWOOVrTPwRHqAOfntPmEpHSgrj1tZJz+PisacT3k5OTRw+jIqSngYxKsHzgtVJvdSwTYv75vz8yj4SqQHuHZ+XqVrV/b6tDsj5iUnWW57eK8KHZzs4uedr5lWRMs37bUNJtmzY3/kJ9/nnCJVRL4dn2Oq59LBI5UfzO5WWI9m3dCN2CbV91lzrrTM0CTo+0XQ5+nMQ6IeAUiANXpQd+B5PRjzp9kltpUfriqdXliPRs9bI10B8cKVqQ9ThVEl8GKt6ueZZltU7Em+Vnp3OoEWleygzXsOkdOLTbwqlIkx4VZ9dXo5io4qrMgLW2J2Jd7TVE2BYJmj5nxq+6IY5nVu2wbUq/3xwlMmROxhuWwTzzDScZ9Vde1Dzk8Egvy9FfT9SXceEuUYMioBfumll+jss8+mpk2b0rhx4+iVV16hHj160JNPPkn5+fmR4xx24PWd/BEVb9mXwq1z8wKrkjPl3fUpP5PtsYucU3wqnMl9YplmWybYo8IGPzJ09OEmXnJ+dPJ5yVbapuuZCS+5rATusyo8mDkygr4Gws5DTPacMQkwJ7zjx4+npUuXWrN+OfHlF2C8++67dPHFF9MTTzxhMkdb3cIOPD8VYJFX50bOEQoUDvqvdlmV0+kpM902WRam2ePHNypsEO2/5a/p567eZs0fdqo8J/pApgqd7DtZu5LZOc1K9ooR0WcmvOSI2qOiAuwnbmT1x/rgCcheA+lqGHYekq7+OvcbkwCfeOKJNHHiRPrDH/5AI0aMoI8++og+/vhj4lchX3HFFbR9+3adHLTIDjvwZHuAYxBEeuG0ADNYaNB9WzIo0tUt3f0yugax1iR7/Orid18iX5E+Y56Jy5/EF0Mk9+gmykz33iBjl4j+ovqIPDPhJza95PrtAZbh5Edv7AmPQNC+DTsPCY+098nGJMDc4vDll19a7Q+dOnWivn370n333UebN2+mX//613To0CFvawxbYULgHfq+nO6eX0obbaZAuPXGpVPlMcwNytQJsk9MRmkVFQVTbZPhkLjWFHvS8Y0KG0T7VZM5x6ZPNC2oYf1om88pEHb+E7VLpB+Z5Yveq7yemUgn1l5dsZXeWrvTdgqEqL2J56cTN37twL7gCPiJCb/amZCH+NVd9z5jEuDWrVvTM888Qy1atKA2bdpYFeDu3bvTrFmz6MEHH6QvvvhCNwvl8qMQeEH3IymHDIEEH5obBCb5RqS/N0bShGcBRPU1QVfVEWhS3Ki2DfKCJRCFPCRYIr+cZkwC/Je//IUeeughqlGjBjVo0IBKSkpo8uTJVjvE2LFj6c477wyLke9zgwg8tz4xkR4yXZUGkbN9g8XGSgR0+RCY3QmIxLg0zQ+/AAAgAElEQVSsb0Rk+vWLaEVVpqrqVxeRfaL6JleA/TL0u0/EFtk1TraP63My5ebkWP3afmdDy+qSzetNigm/fggiD/GrW9j7jEmAGcTrr79OGzZsoGuuuYbq169Pf/3rX+nw4cPWw3BR/OgOPLdeItHeM7c+4dgLM2TZB93jJKtfpq0H7+A9KspcdB1bILPWj8V28u16gEX7av3oILPHTt/cKkQ/8myxnz+xueaJL/fxM3dXN3sZu51iwWRfydoXhfWmxYRfZrrzEL96mbDPqAQ4BuTIkSNUvXp1E/ikpYPuwHOrLonOn5StUIkA0SFT5NxsXhNkT1k2c47ZLhPjor6RkenXB3a6sKy5q7ZZ84dNqyqK9DAnVoD9MvS7z68fRPYl+6r8aAXdvaA0ZatoD7TImVjzCwETY8KPf3TnIX50MmWPUQnw1KlTrUkQW7dupXXr1tEjjzxCjRs3pnvvvdcUXlJ66A48tz4xVlRk/qSOXjMdMqXAYzEIaCagI8Z1yNSMIVDxInxE1tgp7XdfkACioGOQPHSflSm8dechuv2gU74xCTC/BGPo0KF0++2308MPP0ylpaW0ePFiGjVqlNUDPHLkSJ0ctMjWHXi6K8D9Tm0anw8qUxnKlL+ctQSFi1D+ym32yi20ZO0u26fJY1ud+tIyoV8taOZ+z0s3xu18zd/amFLhS4ylJgX8EqIqtF3hJAgv7n748J4Rc0poYclXjlVSp2skXX962SP6c7dre9TcT23f7Cn7JjtRXbJ93cxPNtM9C6Nfcdedh0Q5ToxJgE899VQaNmwYDRo0iGrXrk3/+Mc/iGcDv/DCC9Zb4f75z39GjrPuwFPRA+wkY/qgLjR4xgoq2rgnzl20NzBTeqeCDDhmds20Ilq+aW+lY7sV1qNZN3Sjqrk51r/r8FeQdmbKWenEuJOvT295rPUSoMSZvKLXnEqudrYlytetkx8+/MyD3fXDeseuIbfnIkSfmVDJOVmWzLVtp4duv+i03TTZovdj0/S200d3HhIFBk46GpMA16pVy6r6FhYWVkqAN27cSCeddJL1MFzUPkEEnlt/oWjvod26Oau2pfUOe9Gzo+ZTXfqKvqnK7e1e84pTXxaD/kBdHiPyG+Nuvh7fpz3l5lQJtR9XZPqCzrjyw8drz9VntCCvKq9ff6qKMNlr2+5cnX5RZWcU5Dj5gq9PjqUofYLIQ6LEI1FXYxLgVq1a0VNPPUUXXXRRpQSY5wDzCzHWr18fOcZRDrxM6X+KStC4zTxNnHPqtK5z8wIq3rIvxdxMnJEaFZ866Snq67DsFJm/qzOu/PAR2WP6PU322raLD51+CSsewzjX9FiRYRLlPETGTj9rjUmAue93ypQp9Oijj9LAgQPptddeo02bNtHdd99Nw4cPp7vuusuPfaHuMSnwZPtDvaoloYLNwMNRAc5ApzqYJOrrsIiYXAGOVTiT72dOExKYYWxPmPc0kfsvKsBhRXzquWHGimoKJuUhqm1LV54xCTAbMmbMGHriiSeorKzMsisvL49uvvlm699ycn7qgYzSx5TA89Ov6GdPlHxjmq6iPWcyfYLoCTTNyz/pI+rrsLQ3tQfYrZeXZ+QeraiglS499GHd00TPlbm2MRNY79Uh6jO9WqiRbkoeosYatVKMSoDZtEOHDtHatWvp6NGj1K5dO6pTp45aiwOUZkrg+f1rNuyeuABdZcRRzPvVFVvprbU7XadAOPkF/jLCjUJKiPpaSJiGRYmx1LSghnXCtgCnQLjxcevPZD3drp8wrhGZ+6/Mtc22mjq/WUNIBi4yjFjRYaQpeYgO29KVaVQCzJVfHofGD8PxizA6dOhAAwYMsCrBUfyYEniZ1M8UxTiAziAAAuoIRO1+FjV91XkKkkwgYEoeYgKLZB2MSYB52sNZZ51F3377LbVt25Z+/PFH62UYLVq0oDfeeINatmxpIj9XnUwJvEyZZxi1ABDp+/OyyW4e6oDTm8XHonntd/u5rH5u62VlpaM39ppJIDEGRGYH+40ZmYqqCCm/eojI5jVe+uo+X1RPrMtMAqbkISbSNSYBvvDCCyk/P59mzpxJdevWtVjt3r2brrjiCqpZsya9/vrrJvIzPgE2vd8wck4VVFhFD5lO38nq57behBmqgm7BMk0EZPuGZeMvUe109iabr1KWE1pcO5qCDmKFCCABdsZkTALMSe7KlSutmb+Jn5KSEurRowd99913Qs42aZEJgZdJ8wxN8q2XLl5VH6/9bpUj/lm68z5l9VPx1kERm7EmmgRkJ0fIxl8yFVX9menqIeotJ32DOl9UT6zLPAIm5CGmUjUmAea2h8mTJ9MFF1xQidU777xD1113HW3evNlUho56mRB46D8LJ2xUcBeZberXOln93NazDlPeTZ3TjZmkfr0TvX2ys4Nl408XkbD1CPt8XVwh1xwCJuQh5tCorEmoCfCWLVvi2ixatIj+8pe/WC/D4F7g3NxcWrVqFd144410zz330LXXXmsqQ6MTYFQYwgkbFdx1zouV1Q8V4HDiKCqnqqwA9zu1Kc1dvY227DlELerVpP6nNU2r592tx1b2OlDtj7DPV20P5JlHAAmws09CTYB5tm+VKlXi2lVwMyFRyr/xGn4oLmofEwIviB63qPklCH1VcEcPcBCewhkqCKjqAZ4+qAsNnrGCijbuiauVzjxrr+vQ6+cq2LjJCPt83fZBfvgETMhDwqdgr0GoCfB7770nzOWcc84RXmvKQlMCT1W/nClco6KHCu4658XK6ue2XlZWVHwIPcUJyM4OtouZOau20eh5a1IO9dvzLlJhDTt2wz5f3MNYGUUCpuQhJrILNQF2AsLTH6pWrUoFBQUmMhPWCYEnjAoLQQAEQIBU98SqlgcXgUDUCCAPcfaYUQnwpEmTaPz48bRz505L4+OPP55GjBhBt99+e9RiztI36MDDPMlIhkkklUasRdJtxiodi6e/l/6L3lu3W1kFOMgZ6KbMyca1aWyYh6JY0HlIKEb6PNSYBHjatGl0yy23WP939tlnWz2/77//Pk2dOpU4Mf7jH//o08TwtgUZeOglC8/P2XYyYi3bPK7XXtn+YVFtdPbQJ+tgyqxfXJui0ZE964LMQ6JG1ZgEuF27djRs2DAaMmRIJYac/HISvGZNal+Y6bCDDDyRXjfTeUG/aBBArEXDT1HR0imezm3bgHq1P973FIggZ6CbMiUF12ZUoj44PYPMQ4KzSs1JxiTANWrUoNLSUmrVqlUly9avX0/t27enw4cPq7E4QClBBh563QJ0bJYfhVjL8gBQbL6ueNIl1858U+ZkB2mz4jCAOE0EgsxDNJmgTawxCTBXgMeOHUuXX355JWNffvllGj16NG3atEkbBF2Cgww8/OWvy4uQm0wAsYaYUElARzxxK8CouZ/S/OLtKaqmW1lOFOjVu8xn1atVjebZ6BGbbKGyZ1cHS5W+hqzgCQSZhwRvXXonGpMAP/7449YDcA8++KD16mOe/fvBBx/QvffeS7feeivdd9996Vkawu4gAw+9XyE4OEuPRKxlqeM1ma06nrx6imNmpDNfmGWInsNra+fn0YGy8jjB2Nk8+n7g9GWBzT3W5EKINZhAkHmIwRhsVTMmAT569CgNHz6cpkyZQuXlP90o+G1w/Ca4//qv/yJ+aUbUPkEHHuZJRi1CoqsvYi26vjNRc5XxJPJWuhgDv/OFeb/TOW0bHkPrdh1MwdyvcxNqVDe/0tvtdFRsVbI0MVagkxyBoPMQOe3CXW1MAswvxTjzzDOprKyMPv/8c+K3wnFbRJ06dcIllMbpCLw04GErCIAACPgg4NQHaydqyLmtaFSvdj5OIceZxZ2bF1Dxln0pMu3OQs+uL/TYJEEAeYgzLGMS4AYNGtCbb75Jp556qoRrzV5qQuCp7C8zmza0C5oAx9bslVtoydpdxC8073lSIxpwejOqmhu9b2uCZmfSeVG5R4jGW9gV4M7NCqh4a2oCbFdtdtJ1XJ+TKTcnh7bsOVSpYuwVN1HxpZcdJv086kxNyENM8meiLsYkwCeffDI99dRTdN5555nKSlqvsANPdW+dNABsyFgCQc5YzViIBhgWlXuETLzZ2dS1sJ5Fe/nGPXHqOnuAnXp+k/84VKlrVHxpQNgLq5AJTMPOQ4Rhh7DQmASY+3953u/FF19Mbdq0IR6Llvjhh+Gi9gk78HT0l0XNB9BXDwG3Kls6fZV6tIVUJwJRuUfIxptdHywzmLtqG22WrKq6RQ+fM2rOp7ZTHux6fu1kJetafrSC7l5QmrLU67qKii+jdDVmAtOw8xCT/W1MAlxYWOjIiSdCbNiwwWSOtrqFHXjoL4tcyERGYbc+y3T6KiMDIEMUjco9wuR4U83Qrzy/+zIklLWYkQlMw85DtDhGkVBjEmBF9hglJujAS+5V8ltJMAoilDGSgGxFzkgjoJTjJIPxfdrT1We0ECLk1pvr1T/p9fOYAm7x5rdfVsg4gUWqq4Ru8vqd2pTmrt5m2xusWg8B0zN+SSYwDToPiVJQGJUA8yi0pUuXWq895hFop512Gp111llR4llJ1yADT2UvWWSBQ/HACMj0ZAamFA6SJpCuH932P3fd6TR4xgrHGbcy/ZVO55ze8lhrZrzK3l5ZiDJ2iMh2kjd9UBdlPEX0wBr7Wc/p9o4HzTXIPCRo29I9z5gE+KuvvqLf/e531uuQCwoKrDFo+/fvt16KsXjxYqpbt266tjruX7dunTV9gh/Cu+6666x1JSUlNGzYMFq5ciXVr1+fbrvtNrrzzjuldAgy8Nzee5+bU0Vp35sUBCzOWALcu/jqiq301tqdmAIRYS/PKtrsq+eUTXarzHIPrNsb0GSra3bxVkHkW3eVLlM9e9dO3pxV22j0vDUpaif2BqvWQyWjqMqKOtMg85Co+diYBLhfv360detWmjVrFv3qV7+yOH722Wc0cOBA6tixIz3//PNa2P7www/W/GFOdPkMToC/+eYbawZxnz59rJdzFBUV0dChQ2ny5Ml0/fXXC+sRZOBlQq+SMFgsBAEQUEYgnXuHW2+u1zzcdM6NGa9ChjKQmgVlk62aUWaV+CDzkKiBNSYB5hdevPPOO1bbQ+Jn2bJldNFFF9GePb+Mr1EJecyYMVaCy2fHEuA///nPNGnSJNq0aRPl5eVZx/G6efPmWS/pEP0EGXhubyU6oaAGndeugTVXcvu+w1JzJUVtxTozCIj2VKrUNowzVeqfbbJUPivgpwJ8btsG1Kv98STzjIJTjDmdHzuj/2lNjZlLnWzDpaecQPOKtwnP0XaylSvtEy/raIyd2XY9mW5vkHmI6SyS9TMmAW7evDnNnj2bunfvXknHVatWWaPRdu7cqZzt+++/T3379rXaHfj8WALM53EbxksvvRQ/c8mSJdSzZ09Lj4YNGwrpEmTgHSwrp+4TllZ637ybklHrYxICnuWLVPciiuAM40wRvbDGnoDqZwVke4ATtRKdzesWYxUVRAOnL6vUZ5x4hin3OTsbjqmeSweP/FjJUd0K69GsG7rZJrN2MmKbTbET1515BILMQ8yz3l0jYxLgGTNm0KOPPmq1QJxyyimW1hs3bqRrr72WrrrqKqsFQeVn37591jlc7WX5/CBFLAHmlguuOk+cODF+5Nq1a4lf1rFixQrq0qWLrSrffvst8f/FPtzX3LVrV6u1o2nTpirVT5El8/aj2GavuZJaFYZw5QRkeypVKBDGmSr0zlYZOp4VcOsFj/VPvlH6L3pv3e4U7DxtwusZBa8Y8zrDhPuczP3ZTV+3ucMm2Jmt15XJdiMBdvaOMQlws2bNaNeuXVReXk61a9emqlWrWm0P/DAcJ6eJnx9/rPxXs5/gu/LKKy3Zr7zyirU9MQFu3bq1lRQ/8MADcdE8h7hVq1b0wQcfOE6muP/++2ns2LEp6gSRALv14jnxwbxWP5Fj7p4wegTDONNcD5ivWVj+Sudc0b2i68Lwksz92eu+bLKdYbDFme4EkABHIAF+4YUXhON40KBBwmvtFs6cOZNGjx5tjVs79thjUxLgDh06WG0XdhXg1atXU+fOnW3Pj1oFuM8pJ9Dewz+E9gQ/ekfTCuOUzV6VMrWn/SQtjDN12GGaTKdrI91rRpW/ZPVI51zRubg795c5vpFNtkdW1j6n+InJ+btDBdxun1clNx2WpsU59NFPAAlwBBJg/WHwywnnnXceffjhh1S9evX4P3733XfWf3OVt0WLFtboM06UYx/Te4Dd+sPs2Mr2n6n2D3pHVRMNZ2Yl/BicH73mwIpoosJffmTYPaNQJz+PisacTzWr/fSgsVsSmdznyz2vdjxq5+fZPgch0yPrxz473b3uyX7vwar0E4kXrIk+ASTASIArEdi+fTsdPny40r+1adPGqvhefvnl9PLLL9PUqVNp/fr11gs5+MMV4/nz5xs7BYJ1TJxX2LSgBv14tIKWfr7LqvD+tl1Dq9du277D5FQpYRle1QdVtwNUMVSRrCwnjJmVYZyph54ZUt2e9nebqyuqfbr+8nPt+tmTaI/MXNzOzQqoeOu+FByi97Z0dY0d7DWhonenE2j+6u2+5min60PRWMG66BNAAowE2DOKE3uAuReZ5wD37t2bRo4cScuXL6chQ4ZYSbFM+4WpgefWj+bVf+YJUnAB+tgEQWFZ1hFwuja85uoGBcrPtetnj5c9ujip0lWVHC8O+DkIuBEwNQ8xwWvGPAQXNozEBJh14WkP/Pa34uJiaty4Md1xxx10yy23SKlpauC5PZEsWiWRAmGzWFWVJV09sB8ETCOguwIsaq/s7F23e4eTTeP6nGzNJ9+y55D0fHInmUFXgBM5NSnI50eqrXnrTt+0pXuPVdWfLBoHWBdtAqbmISZQRQKs0QumBp7b7E6nGZSqMaGPTTVRyMsUAk7XhooeYFFGsrN3vXpsVc8fZjvcemyTe4G99EvkInNv8urzTUcPO1/J6Cbqa6zLbAKm5iEmUDcqAeZZu6WlpVRWVpbChl+JHLWPyYHnNrszKM7oYwuKNM6JGgGnayOoa8brGxo/eiTvkXkTnJP/3Obi8hvSGtXNl64s81mi9onM901Hj2S7vfwStTiHvvoJmJyH6Lfe/QRjEuAJEyZYrxu2+3B7gorZv0HDRuAFTRzngQAIqCAQRP+qqjNUyfHDTWS+r8rnKsK01Q8f7AmfAPIQZx8YkwAff/zxdPPNN1vTFhLHk4UfPv41QOD5Z4edIAACagj46Rl160NOnKnrR3bMKtEzvCio6i/2Y4toBVh2DrGTzagAe0UDfp5MAHlIBBLgevXqWdMW+C1smfJB4GWKJ2EHCESTgN+eUbfe1lg/bUUFkd183pmDu1LV3BxPYCJn+JXTtbCedf7yjXvierj1AevglAhApgfZDZxfPT2dgQUZSwB5SAQS4Ntvv92q/Ca+fS3qEYnAi7oHoT8IRJtAOhVDt/5anmRQwfPR561JASQz5cDrjCu6NhdyQLr9xelymrtqG23ec4h4/vqKTXtoQcmOtLi4GS3anywEDosyngDykAgkwDt27KCOHTtSrVq1qLCwkHJyKlcQ/vd//zdygYrAi5zLoDAIZBSBdHtG3fYzqCnvrk/hJdvzmq6Odg6TlSm73i1IVMrKqGCEMaEQQB4SgQT44osvpmXLltEFF1xANWvWTNH4+eefDyV40jkUgZcOPewFARBIl0A6lU0+222/igqw1xmiFeBkTrJ2y65384tKWen6H/tBAHlIBBJgTnqXLl1K3bt3z5iIReBljCtDN8TPAzqhKw0FQieQTs8o7529cgtNeWc97dj/y2hKVT3AMTh+dXS7JmRlyq53c2y6suzs4n7ruau3pbwwBPeF0C8x4xVAHhKBBPjEE0+kRYsWUfv27Y0PKFEFEXiipLBO5y9U0M1uAn56Ru2SuBMK8uk/zm1NA05vFn/IzY9sO2/IyhFJMmVlyq53iyq/smReGBLki1Gy+wqKtvXIQyKQAL/66qs0depUmjRpErVt25Zyc3OjHXVEhMCLvAuNMABfqRrhhqxSwvSYM10/v8EiMlYtJptfsDGveHvKUTIPIfrVE/uiQwB5SAQS4DZt2tCWLVuovLzcVlu8CCM6Fxw0VUsAD9Wo5Qlp3gRMjznT9fMmbL9C5MUasZ2dmxdQ8ZZ9KYJkH0L0qyv2RYMAEuAIJMAvvPCCazQNGjQoGtGWoCUCL3IuM1LhTK12GQk7i5SK9fguWbuLqhBRz5Maxdsbwog5mX7WMPQTDQ0ZO5JlogIsShnrRAkgD4lAAizqzCitQ+BFyVvm6irS72iu9tDMRAIcU9dMK6Llm/ZWUq9bYT2adUM3SvclF7I2y8a47HpZffyuT1cv9AD7JY99TgSQh0QgAX7ggQdcI/jee++NXIQj8CLnMmMV9vtQjbEGQbFQCbhVGmM9pEHGnJ+KbpD6iTrLjx3Jsu3s4jWxl220qFeT+p/W1HoQ0UQGoqywLhgCyEMikADzyy8SPz/88APt2rWLqlWrRmeeeSa99dZbwUSLwlMQeAphQhQIgIAyAm69pmH0kGZKT2+m2KEs0CAodALIQyKQANup+O233xL3/p533nl02223hR5Isgog8GSJYT0IgEAQBEQqwEHoETtDReU0SH2dznKzo9+pTW1n+Yrq7TX32G5OsKhsrMtcAshDIpoAs9qlpaV0ySWX0ObNmyMXoQi8yLkMCoNAVhDw6gHmr9eD/KTbOxukrm5nOdmR7sxeNz5B92ubwhp6iBFAHhLhBLi4uJh69OhBhw4dEvO2QasQeAY5A6qAAAhUIsD9o6+u2Epvrd2ZMgUiDFSZ0s9qZ8ecVdto9Lw1KVhFZ/YG8UrqMHyOM/UTQB4SgQT4xRdfrKRlRUUF7d+/n55++mlq3LgxLVmyRH+kKD4BgacYKMSBAAiAQAQJpNsb7LafcUx5d30KlTB6uSPomoxXGXlIBBLgnJzUr9yqVq1qVX/5DXH8driofRB4UfMY9AUBcQLpzHsVPyX8ldlipx/SyWwuPeUEWvSPHbRlzyFKnNbgVMHlt7lNvKxj/NXSTjq47T+1xbF094JS39VlP3ZjT3QIIA+JQAIcnXAS1xSBJ84KK0EgSgQypWfVi3m22OnFwe7ndmxq5+fRgbJf3mZ6xon1aebgrrZzlWMyY2vc+q7tzort71pYz/qfyzfuiaspItOPzdgTPQLIQ5AAhxK1CLxQsONQENBOIFOmFniByhY7vTjY/Vz0rW2Jc5VHzfmU5hVv91Wt5d5ip/3j+7Sn3JwqtDmp8uzHLuzJLALIQwxNgH/7298KRVqVKlVo6dKlQmtNWoTAM8kb0AUE1BFIt6dTnSZ6JWWLnX4ous1STpSX2IubLs909/uxE3uiTQB5iKEJ8PXXX+8aWR988AFt2LCB6tSpQ/v27YtcFCLwIucyKAwCQgQyvTIa6239e+m/6L11u1OYcO9qo7r51KQgn4iq0PZ9hyv1vCZvEOkjFlkj5BxFi7z0ka0As1p+4iZRj537y3xXkBVhgZiIEUAeYmgC7KTWgQMHaPjw4TR9+nS64IILaNq0adS8efOIhR0RAi9yLoPCICBEIJN7Y936TRlOcp9rIjC73lMRViJrhByjaJGIPjI9wLH+XhG5iSb4OUMRAojJEALIQyKUAL/99tv07//+71bF9+GHH6Ybb7wxsmGIwIus66A4CHgSyJS5tcmGOlUpz23bgOrVqmZbgUyUkTzbVqTqKbLG0yEKF4jqkxwDvTudQItKdrj24srEjdv0B67AJ06aUGg+RGUQAeQhEUiADx48aFV9udp7/vnnW9XfKFZ9E1Ej8DLoLgJTQCBLCPiZOZuIJnn+rEjfqsiaIPGboo8pegTJHmepJYA8xPAEOFb13bt3r1X1vemmm9RGQEjSTA08r962kHDhWBAAAQMI+HnrWKLabRseQycU1KCeJzWiAac3I5G3oIlWXIPC46VPUPdQLz3ceASlY1A+wTn+CJiah/izRu2uKhX8yrWQPlz1veOOOypVfZs1axaSNuqPNTHwZHvQ1FOBRBAAAZMJuN0j+LfFwOnLqChh5qybLd0K69Fz151Og2esqLQnuVfYtPuSLANdc3f9cvG7z+S4hG7+CJiYh/izRP2uUBPgli1b0tatW+nEE0+ka665hnjcmdPn3nvvVW+9ZokmBl46FQXNuCAeBEDAEAJufaqJP2taUMPS+K21O22nRfDPuCe4/2lNae6qbcp6Y4PA5MQg6HuoTM9wjEvQOgbhD5zhj4CJeYg/S9TvCj0Bdkt6Y+byGh6HFrWPiYGHnrKoRRH0BQHzCbjNxE3uCTbfGncNo3APjYKOUY+DqOhvYh5iCrtQE2BTIOjSI6jAk+n1QmVAl7chFwSiS0DmHmJnpdtM3OSpECZSitm/8evv6OsDR6hB7epUeFwtq3Kd/IpiJ1vH9TmZcnNyaIsBb2PDfd7EKAtHp6DykHCsS+9UJMDp8XPdHUTgyfZ6ya7XiAeiQQAEDCCg4p7AMq6ZVkTLN+2tZBH3AM+6oVtKEmmA2XEV3OYei8417lpYz5K3PKE3WldfsAg7FT4VOQdrzCcQRB5iPgV7DZEAa/RcEIHn5y99Pz1lGjFBNAiAQIgE/NxD7NTl+8qrK7Za/cD8NEdsCkRyBTVEU22P9nqjm10FO/keWn60gu5eUJoiP8zqN+7zpkVaOPoEkYeEY1n6pyIBTp+ho4QgAg+9XhodCNEgkAUEsv0e4ta/zO4X6WHOdoZZcJlE1sQg8pCowkECrNFzQQSequqNRgzaRafbv6hdQRwAAgYTUH0PSbwemxTkE1EV2r7vsNK3lolc8yJr2C1eFWB+A16v9sfb9gPH3KqaocHhAtUiRiCIPCRiSOLqIgHW6LkgAi/be72y3X6N4QvRWUJA5TXk1k/LOFX0xYroK7Im5l4vnWPr3HSXOS9LwgpmGkIgiDzEEFOl1UACLFbpwo8AACAASURBVI1MfENQgZfNvV6ovIjHI1aCgBMBVfcQr2oqn59uX6zINS+yJpFFzP4NP0+B+PrgEXr/n1+n4HLTXRVDRCkIqCQQVB6iUuegZCEB1kgagacR7s+i0XunnzFOAAFRAl79tCxHpKfW7TyRa15kTbpniDLBOhAIkwDyEGf6SIA1RiYCTyPcn0XLVnr0a4QTQCB7CCT32TpNQ0gk4lZFFenbFbnmRda4eUl2v4jeOqIirHN12AKZegggD0ECrCeyPKQi8PRjR++dfsY4AQTsCNhde3bzcBP3quijPVhWTt0nLKUDZeVx0XXy86hozPlUs1qe9W/p3hdk9susVRlJYZ2r0gbI0k8AeQgSYP1RZnMCAi8Y7Oi9C4YzTgGBRAJOVdLxfdpTbk4V2rznEDUtqGFt2SYwBUK06iq6Lt37guh+UX1UR09Y56q2A/L0EkAeggRYb4Q5SEfghYIdh4IACARAIN0+22QVReWJrgsAgXVEWPqEdW5QXHGOGgLIQ5AAq4kkSSkIPElgWA4CIKCUQHKP6KWnnECL/rGDtuw5JDWX167XdM6qbTR63poUfcf1OZlyc3Kkz3CqaCbL8/PWNRkOon21sXV/L/0XvbdudwqHdHudvQIBFWAvQvg5E0AeggQ4lCsBgRcKdhwKAiDg0AdbOz+vUu+syFxep17T6YO60OAZK6ho4544b7seYJEzWIBoT7HsGXZynThUVBANnL6skk12+nvNDlbR6+wVxOgB9iKEnyMBdo8BTIHQeI0gAdYIF6JBAARcCYjM5GUBXnN53SqN/U9rSnNXbbP6fVvUq0l+qrOJRiT33TrJS+wz5nNZj6q5ObY8ZDhUENlWtZMZOckM+q1xon3KuFSylwDyEFSAQ4l+BF4o2HEoCICAS29qMhyvubwyvaYya0WcpEKeyGxi1oU58GfKu+tTVEtmlI5e6ewVYYY1IJBIAHkIEuBQrogwAk+0fy0UIDgUBEAgMAIylc8rujZ31Eum11RmrQgIEXle9zwZDulWgL2q6azrqLmf0vzi7Snme+0V4YU1IJBMIIw8JCpeQAuERk8FHXjoCdPoTIgGgYgRsLsf5FYh+pGzvJ8/yfNz7UyUua/IrBXB6SXP6+d8RlA9wF69zm59w157RVhhDQjYEQg6D4mSF5AAa/RW0IEnUi3RaC5EgwAIGEYgsUd05/4ymuez8ijTayqzVgSXmzzRe16yjN6dTqBFJTvivcuJPcSi+ouui9nopGu/zk1o4mUdHXuYRRhhDQg4EQg6D4mSJ5AAa/RW0IGH3jKNzoRoEIg4gUy8P0TJpijpGvFQh/oJBILOQ6IEHwmwRm8FHXii1RCNJkM0CICAoQR03h+8+nCTkYisF1kjOjvYbUqEk7tEzpdxtS7+qvWUsQlrzScQdB5iPpFfNEQCrNFbQQeeSD+cRnMhGgRAwGACuu4PsnJF1ousYdSis4Nle2xFz5dxd1RkytiEteYTCDoPMZ8IEuBAfBRG4Mn2pQUCAoeAAAgYQUDH/UG2simyXmRNDKjo7GCZKQsy58s4VjV/XXrK2IS1ZhMIIw8xmwgS4ED8g8ALBDMOAQEQCJGAbG+ryHqRNU4mp7M3JlOFjCBcEhU9g2CBM+wJIA9xjgy0QGi8ahB4GuFCNAiAgBAB3T2ibm9Fu+CkhkRUhbbvO2y9KY57cees2ub5tjW3iQnH1a5OXx84Qg1qV6fC42qlvAXOaW/nZgX013/vRjWr5Xlyi0plNSp6egLHAm0EkIcgAdYWXG6CEXihYMehIAACPxPQ0XeaDNdtvm3yWu7FnT6oCw2esYKKNu6J/zi5R1dkdm9ss8je2FqRuce8NghuKoI0KnqqsBUy/BFAHoIE2F/kpLkLgZcmQGwHARBIi0BQFcJYb+sbpf+i99btdtWZe3G5Ejx31TbbObyxzSIzjGNrk/t7ee+AqZ9Q8dZ9Kbrw3N3HLu/kyVV1v67ngT4XREVPn+ZhW5oEkIcgAU4zhPxtR+D544ZdIAACaggE3SPqdF6iNUPObUWjerWTMtBLrp3MvpM/ouItqQlw5+YFNH9oD6nzsRgEokoAeQgS4FBiF4EXCnYcCgIg8DOBoCrAMeBO5yU6ZFyfkyk3J4e27DkU7wuumpvj6rOZn2ymexaWOq6xm/Dwp9klNN/mzXfcC/zqzd2tN6/p7o/2G4im6uXXHuwLjwDyECTAoUQfAi8U7DgUBEDgZwJB94h69QOf3vJYqlKlCi136f9Ndh7LvGZaES3ftNfWr04zfg+WlVP3CUvpQFl5yj7RXuQwAilon4VhI84MjgDyECTAwUVbwkkIvFCw41AQAIEEAkH3iPJ5o+Z8SvNsqq99Op1AC0p2pPjHbUavU1X57DbH0cUdGqdMgUgUfuj7crr62WWOvcB2OsrMC9YRaEFX7XXYAJnmEEAeggQ4lGhE4IWCHYeCAAiETMCpZ5f7b+36ct36gtPtY1apSxBY07U3CB1xRnQIIA9BAhxKtCLwQsGOQ0EABEIm4FTF/LdOjWlhyVdKKsCilVqVurDiuvtz3WYgT7yso9W7jA8IiBJAHoIEWDRWlK5D4CnFCWEgAAIRIWDXx9q1sB4draiglUm9vN0K69GsG7o5Jnbp9sSapIuI+9z6qJ36nUXkYk12EkAeggQ4lMhH4IWCHYeCAAgYQCC597j8aAXdvSB1ksP4Pu3p6jNauGqcbh+zKl2C6s9166MWrXwbEAJQwQACyEOQAIcShgi8ULDjUBAAAQMJmNTb6lcXv/v8uCPIs/zohz3RIIA8BAlwKJGKwAsFOw4FARAwkEBQ1VMR0/3q4nefiE7Ja4I8y49+2BMNAshDkACHEqkIvFCwG3Oo7odljDEUioCAAIF0e3ljR6i4rvzq4nefAJ6UJV5nOXFQwcePvthjJgHkIUiAUwjs2bOHxowZQ4sXL6Zvv/2WOnbsSBMmTKCzzjrLWltSUkLDhg2jlStXUv369em2226jO++8UyrCEXhSuDJqsdcvr4wyFsaAgCCBdHt5VV5XfnXxu08QUaVlTmc5cZg+qAsNnrGCiiReNOJHL+yJDgHkIUiAUwhceOGFtGvXLnrqqaeoQYMGNGnSJJo2bRqtXr3a+u927dpRnz59aPjw4VRUVERDhw6lyZMn0/XXXy8c+Qg8YVQZtxBfX2acS2GQAQRwXf3kBLdRaSa+3MOA0MlaFZCHIAGuRODLL7+kNm3a0EcffURnnnmm9bOKigpq27YtXXnllVSjRg0rId60aRPl5eVZP+dq8bx58+jzzz8XvpAQeMKoMm4hHmDJOJfCIAMI4Lr6yQlRe7mHAaGTtSogD0ECXInAvn376OOPP6bzzz+fqlevHv9Z69atqVevXrRhwwYqKCigl156Kf6zJUuWUM+ePWnnzp3UsGFDoYspjMBz6/9Cb5iQ25QsQqVKCUYIAYFKBKJ4Xem476ICjAtDlEAYeYiobmGvq1LBpU986LXXXqMBAwbQokWL6K677qKLLrqIJk6cGCezdu1aOvnkk2nFihXUpUsXIWJBB55bfxx7eeD0ZegNE/Jc+otU9iqmrw0kgEBmEIjadaVLX/QAZ0Y8B2FF0HlIEDapOgMJMJHVCsGVX64IL1iwgLgSfNVVV9EDDzwQ58xV4VatWtEHH3wQf1Au2Qn8MB3/X+zz1VdfUdeuXWnr1q3UtGlTVT5zlONWHeG/ckbPW5OyF0PV9bklyIdl9FkBySBgFoEoXVc6K9ZOHKLEx6zIykxtkAA7+zXrE+CFCxdaye4ZZ5xhTYTg/t8OHTrQxRdfbFsB5ofkOnfubEv0/vvvp7Fjx6b8LKgE2K0/jpWa8u76FN2GnNuKRvVql5lXPqwCARAAgRAJoGc5RPg42iKABBgJsC0BngDBo8769etHs2bNivcDc/LLo89mzpwZ3yfSAxxWBTjWY/b30n/Re+t221Z5UQHG3RAEQEAVAR19rap0M0mOzgqwSXZCF3MJIAFGApxCYMqUKdZoM57v+/jjj1NOTk58Dc8Dnjp1Kq1fv55yc3Otfx89ejTNnz/fuCkQdr1gicaecWJ9mjm4K6EH2NwbFDQDgSgR0NXXGiUGorqClSgprNNFAAkwEuBKBNatW0ft27en3//+99Zs38QPt0AcOXLEmgPcu3dvGjlyJC1fvpyGDBliJcWDBg0SjtMgAs+pwnBu2wbUq/3x1P+0plQ196fkHr1hwq7DQhAAAQcCqGrKhQbuu3K8sFotgSDyELUaByctK3uAH3roIWvSg92HE9wZM2ZY0x64OlxcXEyNGzemO+64g2655RYpzwQReOgxk3IJFoMACKRJAPecNAFiOwgESCCIPCRAc5QelZUJsFKCLsKCCLyZn2ymexaWpmiB6Q5BeRnngEB2ERCpAJveI+xXP7/7sitCYK1JBILIQ0yyV0YXJMAytCTX6g48vhlfM62Ilm/aW0mzboX1aNYN3eKtD5JqYzkIgAAIOBLw6mv1+nnYaP3q53df2Pbi/OwmoDsPiTJdJMAavac78JwqMeP7tKerz2ih0TKIBgEQyGYCbn2tIhXiMNn51c/vvjBtxdkgoDsPiTJhJMAavac78NCLp9F5EA0CIOCLgOn3Jb/6+d3nCyI2gYAiArrzEEVqhiIGCbBG7LoDDxUJjc6DaBDIYAIHy8qtZwc2fH2QcqgKndbyWGrd4JhKU2O8zHfqhw37vuTVp+tXP7/7vDiK/tzLLlE5WJddBHTnIVGmiQRYo/d0Bx560jQ6D6JBIEMJcPLbfcJSOlBWnmJhbG54bHSiEwK3e0+YM8dF7okia+zs9rtPRRiFebYK/SEjPAK685DwLEv/ZCTA6TN0lBBE4GHGpEYHQjQIZCCBP80uofnF2x0tE5kg41UNDeu+5KVXzGi/+vndl24YidqV7jnYn3kEgshDokoNCbBGzyHwNMKFaBAAAV8E+k7+iIq37HPcO+TcVjSqVztX2ab2w5qqly9HJWzKVLvS5YL93gSQhzgzQgLsHT++VyDwfKPDRhAAAU0E0q0A89fxI+aU0MKSr1I0FKkeazLLEqurUhrrmd70zXdUWL8WjevbnmpWy9NpSiXZTnb169yEJl7WESMvA/NE9A5CHoIEOJSoReCFgh2HggAIuBBIpwfYafY4H2fC/HEdvbJ2vOrk51HRmPMDS4Lt7Iq5WLRvGxdFdhJAHoIEOJTIR+CFgh2HggAIeBA49H053T3/pykQVSSmQDhVIvk4U+aPq+7TdaqYc/X1scs7BRZrbNeoOZ/SPJv+7bAr74FBwEHSBJCHIAGWDhoVGxB4KihCBgiAgCkEnHpRWT+R3mFT7JDRw6lnunPzApo/tIeMqLTXohc4bYRZJwB5CBLgUILe1MDDPMlQwgGHgkCoBLyue6+fs/JRqACrhhxEBZjZz165hZas3UVViKjnSY1owOnNUnp7dfU4q2YGeeYQMDUPMYEQHoLT6AUTA09Hj5xGhBANAiCggIDXde/185gKpvcAK0CVIkJ3D7ATU7uealE/6eAAmdEkYGIeYgpJJMAaPWFi4KGCoNHhEA0ChhLwuu69fp5oFveijnjtH7SgZEeKtZnaixrrmd6oYQqEW1XdjqfqHmdDQxZqKSJgYh6iyLS0xSABThuhswATAw89ZBodDtEgYCgBr+ve6+fJZsmuNxSLEWplY1+1EeCzRAkT8xBT0CMB1uiJsALPrZdPptKjEQ1EgwAIaCaQeB/Yub/MdXqA7H1Bdn2iqSK9xprRhC5exDespCmTNUIHBgV8EwgrD/GtcIAbkQBrhB1G4Hn1iHn9XCMOiAYBEAiIgN11Xjs/jw6Ulcc1SJwfK3tfkF0fO9TvvoCwBXKMHYNjqufSwSM/ppxvwmzlQKDgEG0EwshDtBmjWDASYMVAE8WFEXgilRn0kGl0OkSDgAEE3N4c1qhuPrWoV5P6n9a00pQB2fuC7HrGInJ/MgCfVhWcGHRqVpdKtu5POTtT+6q1QobwOIEw8pCo4EcCrNFTYQQeevM0OhSiQSAiBEy9D5iqV5BudWLAc4WLt+xLUSVT5ysHyTybzwojD4kKbyTAGj0VRuA5VRfQS6bR0RANAoYRMK3SGut5/duar+iDf34tVeUMumdY53kse9TcT2m+zdvc+M1yeMubYRdSBqgTRh4SFWxIgDV6KozAk5kpqdF0iAYBEAiRgEm9tna6JKJx63MN2g6d57lx4H7s6YO60OAZK6ho4544nsQ+7RDDCUdHmEAYeUhUcCEB1uipsAJvVtFmuntBqVSVRSMGiAYBEAiBgJ8eXR1qus255fPcvp0KupKt8zy3vuyJl3W0+rFN8ZmOOIDMcAiElYeEY63cqUiA5XhJrQ4r8NBnJ+UmLAYBENBIwG3OLR/r1uMa9L1M53k6ZWt0H0RHnEBYeUgUsCEB1uilsAJPZxVDIy6IBgEQyEACXhVgtykHQd/LdJ6nU3YGhg1MUkQgrDxEkfpaxSAB1og3rMDT2cemERdEgwAIZCABr97XmYO7VhrHlogg6HuZzvMOlpVT9wlLK81irpOfR0Vjzqea1fIy0PMwyQQCYeUhJtjupQMSYC9Cafw8zMBDL1kajsNWEAABpQRi96MNX39HXx84Qg1qV6fC42qlzCK2OzToe5mu81ABVhpSECZIIMw8RFDF0JYhAdaIHoGnES5EgwAIgECECKAHOELOyiBVkYc4OxMJsMZANznwdM661IjUVnQm2RI0O5yXPQRwnaTv60SGTQryiagKbd93uNKb9ZzW7Nxfhjm/6bsAEiQJmJyHSJqifDkSYOVIfxFoauDp7HPTiNMx+R04fRlmZwYNHudFikAmXfNhgfeaZ+w0yzdR39r5eZV6gDHnNyxvZs+5puYhJngACbBGL5gaeJnUi5ZJtmgMRYjOcgK4TtIPAK9pFnyC09vcEk/nNY3q5leqGqevHSSAgD0BU/MQE/yFBFijF0wNvEzqRcskWzSGIkRnOQFcJ+kHgNc8Yz6hc/MCKt6yz/Uwt7nH6WsJCSBQmYCpeYgJfkICrNELpgZeJlWDMskWjaEI0VlOwMTrJGo9yaoqwG5zj7M8TGG+BgKm5iEaTJUWiQRYGpn4BlMDL5P6ATPJFvHIwkoQkCNg2nVimj4iNFX0AKPnV4Q01qgkYGoeotJGv7KQAPslJ7DP5MDTNetSAIvyJZlki3I4EAgCPxMw6ToxsSItEiiJDJsW1LC2bEuaAiGyRuQsrAEBFQRMzkNU2JeODCTA6dDz2IvA0wgXokEABCJLAD3JkXUdFI8YAeQhzg5DAqwxmBF4GuFCNAiAgFICIj25dmsqKojmrt5GW/YcEp5sYHoFWISFUvgJwsI8W5dNkBseAeQhSIBDiT4EXijYcSgIgIAkAZGeXLs1XQvrWSct37gnfqJIn6vIeZImKFsepm5hnq0MIAQZRQB5CBLgUAISgRcKdhwKAiAgSUCkIisyBSF2rMikA5N6khNxibCQxCu8PMyzhZXEwkgRQB6CBDiUgEXghYIdh4IACEgSEOnJFZmDGzs2yrNuRVhI4hVeHubZwkpiYaQIIA9BAhxKwCLwQsGOQ0EABAQIJPaa7txfRvOKt6fsSqzkqq4AC6gotcRP76zdnjmrttHoeWtcWYgodrCsnO5ZWEqbvvmOCuvXonF921PNanmuW9OpAPuxX8QOrIk2AeQhSIBDiWAEXijYcSgIgIAHAbte09r5eXSgrDy+M7mXV2UPsGoH+emdddozfVAXGjxjBRVJ9jUn2sTJb/cJSyvxrJOfR0VjzndNgv3Ywef63afaD5BnHgHkIUiAQ4lKBF4o2HEoCICABwGnSmO/zk2oUd18x2kOdn27fNTcVdtos8QUCNUO8lM5ddvT/7Smadn0p9klNN+mos58H7u8k6v5fnqj/div2geQZyYB5CFIgEOJTAReKNhxKAiAgAeBTOs19WOPnz2igdV38kdUvGVfyvLOzQto/tAeomKE1+m0RVgJLDSSAPIQJMChBCYCLxTsOBQEQMBnBdhpegN/xf7X5Zvo5aKtVPbDj3Rqi2PpoX4dPHtag3KEnwqonz2i9jhVgDs3K6BZN3SjRf/YITQ3WbSv18mWc9s2oF7tjyeuaFfNzRFVH+syiADyECTAoYQzAi8U7DgUBEDAg4BMzyivverZT2jl5soVTe4ZXubR0xqUI2TsienkZ4+oPXY9wLG9Xr3WfvSzsyVRV5HZzKK2YV20CCAPQQIcSsQi8ELBjkNBAAQECIj2mrpNfxDpaRVQRckSUXsSD/OzR1TZQ9+X09XPLqPiramtEMky7CrvshXqmC1vlP6L3lu3O0VNkdnMorZhXXQIIA9BAhxKtCLwQsGOQ0EABBQScJv/q6unVaH6oYoSnZ1sNzfZb1+v332hgsLh2gggD0ECrC243AQj8ELBjkNBAAQUEohKBVihycpEic5OVlEBjiktWzlWZiwEGUkAeQgS4FACM8jAE31YgkHIrA0FHA4FARAwhoCKHmAd95yYzI1ff0dfHzhCDWpXp8Ljahn1wJefecsxx/vtUfa7z5iAgyJKCQSZhyhVPABhVSoqKioCOCcrjwgq8GRueDJrs9JpMBoEQCCFAPeXvrxsC81atpnKvpebAqHjnuP20JdpD3wl9xn37nQCLSrZITQ32W+Pst99CP3MIxBUHhJFckiANXotqMCT+cpLZq1GNBANAiCQJQR03HO8WgvwwFeWBBfM9CQQVB7iqYiBC5AAa3RKUIEn89CDzFqNaCAaBEAgSwjouOd4PVxm91BZluCGmSBQiUBQeUgUsSMB1ui1oAJPpsIis1YjGogGARCIIAGvXl67n89ZtY1Gz1uTYq1olVZGZuwQUdkRdAFUBgEpAkHlIVJKGbIYCbBGRwQVeDI9djJrNaKBaBAAgYgR8Lp3OP18+qAuNHjGCirauCdusWifrozMmHBR2RHDD3VBwBeBoPIQX8qFvAkJsEYHBBl4Mg89yKzViAeiQQAEIkTA69sjt5/zq3jnrtom9OBXIhIRmRsMngIRIfdC1QwlEGQeEjWESIA1egyBpxEuRIMACARKwKuX1+vnfpTVIdOPHtgDAlElgDzE2XNIgDVGNQJPI1yIBgEQ0E4gsf925/4ymle8PeXMWL+tV4XYj7IqZXr1L/vRT+ceN30Tf9akIJ+IqtD2fYepRb2aRs1B1skHssUIIA9BAiwWKYpXIfAUA4U4EACBwAjIvsTBq0fYj+KqZKqS48cGP3vc9OXJ/QOnL6vUU514Bnqg/RDP3D3IQ5AAhxLdCLxQsONQEAABBQScqq/9OjehRnXzbauNOp4vUCFTZSVZAVpPEW768pur7KZqJArFFAxPxFmzAHkIEuBQgh2BFwp2HAoCIKCAQCb130bNFjd92bVT3l3v6mHMQVZwAWSICOQhSIBDCWVTAi9qvW+hOAuHgkCWEfDqMR0191Oa79Lz64XLpPtO1CrAMz/ZTPcsLE1BzNX3b777nt5bt9sVPyrAXtGZPT83JQ8xkTgegtPoFRMCL2q9bxrdAdEgAAI/E/DbYyraX2rafcc0fdwCkXW9ZloRLd+0t9Ky2vl5dKCs3DOGRX3kKQgLMoKACXmIqSCRAGv0jAmBF7XKh0Z3QDQIgMDPBPz0mHL1ceJlHalqbo4nRxPvOyp6iT0NV7DAiZ2d6HPbNqCeJzWyfrQNUyAU0M88ESbkIaZSRQKs0TMmBF7Uet80ugOiQQAEfibgp8dUpq8U9x3/oebEzk6ijE/8a4SdUSZgQh5iKj8kwBo9Y0LgOVUTmh9bg7q0rEfj+ranmtXyNFKAaBAAAdMI+KkAy/SVilSATeoRNsk/MhXgZJ/EmG7E2/FMcmmoupiQh4QKwOVwJMAaPWNC4Nn1viWaXCc/j4rGnI8kWGMcQDQImEZAtgdYtq/Uq+fW6+em8QpSHzs2XQvrWSos37gnrkqyT9zu9bL+C9JenKWXgAl5iF4L/UtHAuyfnedOUwIv1vv23Icbad2ugyl6c2/fY5d38rQHC0AABDKHgFtPrIp+WTcZIhXizCEtb4kdO5Yyd9U22rznkO0MZq/KsUwFX15j7DCVgCl5iIl8kABr9Ippgdd38kdUvGVfisWdmxfQ/KE9NJKAaBAAARD4hQB6hNVHg1fvMPqF1TOPgkTT8hCTmCEB1ugN0wJv2MvFtPAfO1AB1uhziAaBTCWgsmdXdQVYpW4qZamIBRF9eI3T3OaYDqgAq/BG9GSYloeYRBAJsIM3jh49SmPHjqVp06bR3r176ayzzqLJkydT69athf1nUuDxDfKqZz+hlZsrV4B5tuQy9AAL+xQLQSAbCaju2VUpz1RZKuJExDav5zxYD/QAq/BGNGWYlIeYRhAJsINHOPnlhPf555+nJk2a0MiRI2nDhg302WefUbVq1YT8aFLgOVVcHuh9Mg08s6WQPVgEAiCQnQRUV2yZooo+Y5ajUjeVslREiog+Tms6Nysgfniu8Lha1P+0pkLzm1XoDBlmETApDzGLDBESYBuPfP/993TcccfRww8/TDfffLO1Yt++fXTCCSfQc889R1dccYWQH00KPPTcCbkMi0AABGwImHz/UKmbSlkqAklEH5E1KnSBjGgSMCkPMY0gEmAbjyxfvpy6detGX3zxBbVt2za+gtsgOnbsaFWGRT4mBZ5IJUHEJqwBARDIPgIm3z9U6qZSloooEdFHZI0KXSAjmgRMykNMI4gE2MYj8+bNo/79+9OhQ4eoRo0a8RUDBgyw/m3x4sVCfjQp8ER6yYSMwiIQAIGsI2Dy/UOlbiplqQgSEX1E1qjQBTKiScCkPMQ0gkiAbTwya9Ysuvbaa+nHH3+knJxf3ns/cOBA2rFjBy1ZssTWj99++y3x/8U+W7dupTPPPJO4oty4cePQfc89d3/79Cvavu8wNSmoQRd3bIy+sNC9AgVAIBoETL5/qNRNpSwVnhXRR2SNCl0gI3oEvvrqvQDW8AAAEg9JREFUK+ratStt3LiRWrbE8z6JHkQCbBPPc+fOpcsuu8y2AnzkyBFauHCh7VVw//33W5Mj8AEBEAABEAABEAABUwhwIe700083RR0j9EACbOOGWA/wl19+Sa1atYqv4B7gU045hSZNmiRUAS4rKyOuAhcWFlJeXp4Wh8f+ujOlyqzFyICEgqVa0OCpjidYgqU6AmolITbV8dTBsry8nHbv3k0dOnSg/Px8dcpmgCQkwDZO5Cpvw4YN6bHHHqPBgwdbK/xMgQgiPtDfo44yWKpjyZLAUx1PsARLdQTUSkJsquMJlupYikhCAuxA6a677qKnn37aGnvGfTMjRoygTZs20Zo1a4TnAIs4IN01uGDSJfjLfrBUxxIJMFiqJaBOGq5zdSxxnYOlWgLBSkMC7MCbH4AbM2aM9SKMw4cP09lnn221PpjWRI6buboLBizVscQvRrBUS0CdNFzn6ljiOgdLtQSClYYEOFjeyk/jqRPcqjF8+HCqU6eOcvnZJBAs1XobPNXxBEuwVEdArSTEpjqeYKmOpYgkJMAilLAGBEAABEAABEAABEAgYwggAc4YV8IQEAABEAABEAABEAABEQJIgEUoYQ0IgAAIgAAIgAAIgEDGEEACnDGuhCEgAAIgAAIgAAIgAAIiBJAAi1DCGhAAARAAARAAARAAgYwhgAQ4oq48evSo9drladOm0d69e4nfUjd58mRq3bp1RC3Sp/aePXuskXaLFy8mfsq2Y8eONGHCBIsZf0pKSmjYsGG0cuVKql+/Pt1222105513xhUCa3vfrFu3jk499VR66qmn6LrrrgNLnyH84osvWvG4YcMG682T/Er1P/zhD+ApyfOHH36w2M2cOdO6J3bq1IkmTpxIZ555JlhKsBw3bhwtWbKE3n333fguFfdILxkSKkZqqR3P119/nR544AH6v//7PzruuOOs653/u0aNGpZtIr9zspWnSucjAVZJM0BZnPxywstzips0aUIjR460foF+9tlnRr2oI0AkjkddeOGFtGvXLitRa9CggTXPmf9wWL16tfXf7dq1oz59+lij5IqKimjo0KEW2+uvv96SCdapaDnZ4MSC/2jgGOQE+JtvvgFLyYCfNWuWxe7RRx+lSy65hF5++WUrifvwww+pbdu24CnB87777qNnnnmGXnjhBTrxxBOt5PfVV1+1kozq1auDpQDLJ554wroP8tz7WAKs4roWkSGgXuSW2PH84IMP6LzzziNOjC+77DL68ssv6cYbb6Tzzz/fupeK/M7JVp6qAwAJsGqiAcj7/vvvrb8aH374Ybr55putE019VXMAOFyP4JtLmzZt6KOPPopXgioqKqzk4sorr7T+4uaEmN/yl5eXZ8niavG8efPo888/J7C2x8uM+I+Fd955J54A//nPfwZLiYDnOOREjX8JPvLII/Gdv/vd7+jcc8+1/huxKQ6UK76cRPAfE/zhb3vq1q1Lc+bMIf62AiydWW7fvp1uuOEG4uSsWbNm1KhRo3gCrOK69pIh7uVorHTjec0111gFmbfeeituDP8hzAWXgwcPUpUqVTx/v2cbT11eRwKsi6xGucuXL6du3brRF198YSVysQ9/pc9f73P1Ep+fCPAfBh9//LH1i5GrQLEPt4r06tXLqpoXFBTQSy+9FP8Zf/3Xs2dP2rlzp5UYg3XlaHr//fepb9++VutI8+bN4wnwxRdfDJYSFx7/gfXrX//a+iaic+fOKTvBUwImkXWN81ve3nzzTSuJe/zxx+nuu++2vhW79dZbEZsuOPkref72gauS/FU83/diFWAVceglo2HDhnLONny1G8/i4mLKycmhU045pVICfO2119Lu3but30lev3OyjacudyMB1kVWo1yuTvbv358OHToU7xni4wYMGGD9G/e64uNM4LXXXrNYLVq0iO666y666KKLrK9LY5+1a9fSySefTCtWrKAtW7aAdQJK/oOCb9xcgbjqqqusakWsBYL/+AJL8SuP4+/f/u3frKrbgw8+SPyLsbCw0EraLr30UuuPWfAU51laWkqXX3458fWbm5trJRlc/e3duzdYimO0WnISE2AVceglo0uXLhIaRmtpMs9k7flbRk54OV5XrVplffvo9fs9m3mq9D4SYJU0A5LFX5fwX4s//vijddHEPgMHDqQdO3ZYDzDgY0+AWyG48svVogULFlgPDXIix1WP2Cf2MBInJvyLAKx/YcltI/zV/SuvvGL9Y2ICDJZyV13sOuakl/tX+Q+LuXPn0vjx4+ntt9+mm266CbEpgZT/sH3yySdpxIgR1nMRzz77LM2ePZv4G4t+/fqBpSDL5IRNxXXtJSP2QLKgipFa5pYAl5eX09VXX239LuI45URY5Pd7NvNU6XwkwCppBiSLf0ly36BdBfjIkSO0cOHCgDSJ1jHMhZPdM844w6qSc/9vhw4diL9OsqsA81fTnAyD9U9+5qfrR48eTWvWrKFjjz02JQEGS7nrgR/Q4ooltywNGTIkvpkfhuMPf/uA2BRjunnzZqvXf+nSpfSb3/wmvon/Nz8vwc8CgKUYy+SETcV17SXDrgVITFvzVzklwAcOHLC+ieTnKPiPN/7Whz8iv9+zmadKjyMBVkkzIFmxHmC+qfPYpNiH/4rmKhI/7IFPZQI8AYJHnXEliP/CjvUD8y9FHn3GyV3sY9cDDNZkPbnM0wkSe6m/++476785Dlu0aAGWEhce96b36NHDaoFIrIDxRBf+A61ly5bgKcgz1tZ0+PBhys/Pj+/iavDf/vY3xKYgR16WnLCpuEd6yci0HuBE3HYJ8FdffWW1N23cuNGq/vK9NfYR+f2ezTwlQtlzKRJgT0TmLeAqL98wHnvsMRo8eLClIKZAOPtpypQp1mgznu/LD8Ykto3w/NWpU6fS+vXrrb5B/nCVc/78+dYUCLD+hSs/2cwJRuKHq25cPedKJj9EA5bi9wtmyU/b33vvvZXmTvMvxqpVq1pTS8BTjCdPJOnevTstW7aMunbtGt/E7U6cEPO3PmApxjI5YVNxj/SSIaZZNFcl8+QZ1Xxt85SSv//979a3kIkfkd852cxTZRQgAVZJM0BZ/PDW008/Tc8995xVKeJKB/er8tfT1apVC1ATs4/i8Uft27en3//+9ynTMbgFgm82PAeYH5Thyhv/9c1fR/Mvy0GDBlnGgbWzjxN7gHm0D1jKXQ/81D2PM+RrmRM37q3mhJi/yj/ppJPAUxAnvziAR8fxU/TcUtK0aVPiF4zww5r8rQWPm0NsisFMTthUXNciMsS0i96qZJ7831ws4OSXp8AkfnguPRdivH7nZDNPlRGABFglzQBl8QNwPIuVn8DnShIPLufWB06G8fmFwEMPPWTdTOw+nODOmDHDmvbA1WF+Cr9x48Z0xx130C233BLfAtZiCTCvAkv5q4+/yeEWHa6w8y9EfvEKT4cATzmWXFnjCRrcPsL/mytrfP2fc845YCmB0u4rexXXtZcMCRUjtTSRJ/+hVqtWLSorK7O1gVsi+He4yO+cbOWp0vlIgFXShCwQAAEQAAEQAAEQAAHjCSABNt5FUBAEQAAEQAAEQAAEQEAlASTAKmlCFgiAAAiAAAiAAAiAgPEEkAAb7yIoCAIgAAIgAAIgAAIgoJIAEmCVNCELBEAABEAABEAABEDAeAJIgI13ERQEARAAARAAARAAARBQSQAJsEqakAUCIAACIAACIAACIGA8ASTAxrsICoIACIAACIAACIAACKgkgARYJU3IAgEQAAEQAAEQAAEQMJ4AEmDjXQQFQQAEwiLAr9d97733Kh3PrxrnNwb26dPHetUuv1Jb1YffAsVvjrr//vtViUyRk3wGv8468VO9enXrVcKXX3659RbFmjVratMFgkEABEAgLAJIgMMij3NBAASMJ8AJcKNGjejJJ5+M63rw4EF666236Pbbb6cbb7zReo2xqs/u3buthPqYY45RJVIoAX7iiSeshJc/bN/y5cvpzjvvtF7N/Oabb1JeXp42fSAYBEAABMIggAQ4DOo4EwRAIBIEOAHmiumMGTNS9P3jH/9ICxcupG+++SYStsSUtKsAP//881blOfGzatUqOv3002natGnEtuIDAiAAAplEAAlwJnkTtoAACCgl4JYADx06lObMmUO7du2yzvz+++/pnnvuoVmzZtH+/fupffv29MADD9CFF14Y14krx6NHj6a1a9fSiSeeSHfccQcNHjyYNm7caCXaycnp//zP/9CDDz5IpaWlVKdOHbryyitp/PjxlJ+fb8nk9oVnnnmGXn31Vfrwww+pXr16dMstt1hnOH1EE2Def/bZZxO3fCxZskQpVwgDARAAgbAJIAEO2wM4HwRAwFgCdglweXm51RZwxRVXWC0Qjz76qKX/VVddRZ999hlxOwH30L7++uv0n//5nzR//ny65JJLqKSkxKqo/ulPf7KSXv7vIUOG0N69e20T4AULFlD//v2tfuABAwbQP//5T2t9165dae7cufEEuG7dulYbBv/7zJkzady4cfT+++/Tb37zG1uuMgnwf/zHf9Ds2bPp66+/NtZHUAwEQAAE/BBAAuyHGvaAAAhkBQFOgD/66CPiB8Nin8OHD1OLFi2sloExY8ZY/bFffvkltWnThlasWEFdunSJrx00aBBt3ryZ3n33XeL/vW7dOvrkk0/iP//v//5vuu2222wT4G7dulGzZs2sKnPsw0l17969rQoy9+dyBXjYsGFW0h37cBV4xIgRjlVgmQSYH4J7+OGH6YcffsgKf8NIEACB7CGABDh7fA1LQQAEJAlwAly/fn2aOHEiHT16lJYtW2ZVcPv160eTJ0+OPxz22muvWVXaWrVqVTqBE8eCggLauXMndejQgXr27EmPPfZYfA1XgTt37mybAPP0hYceesh62C724dYKlvfKK69YD61xAjx16lS66aab4ms4wb3++uvpvvvus7VWJgHmdgo+CxVgycDBchAAAeMJIAE23kVQEARAICwCdi0Q77zzDl1wwQVW0slJMH+4TYBbIlavXk21a9eupG5ubi4VFhZaie4555xTqVrrlgDzNIgJEyZYFd7YZ8+ePVZCzi0QnIRzApz8AJvXKDWZBLh79+6WPdy7jA8IgAAIZBIBJMCZ5E3YAgIgoJSA00NwI0eOpEceeYTeeOMN6tWrl9X7yw+9LV682Or3jX24hSAnJ8d6kI1bJj7//HMqKiqK//zpp5+mm2++2bYCzD29zZs3r9QCwVMneP5wYguErgSYp0BwOwdPwOD2DXxAAARAIJMIIAHOJG/CFhAAAaUEnBLgsrIy6tixIx05csRKfnlu76WXXmo92DZp0iQrGZ43bx5xojx9+nSrJYEnOXTq1Mmar8tjxTiJ5YfoePbvpk2brL7ixOost1Vwm8PYsWOt/8/9w/wQHMvgXmD+qKoA280B5gf4WrduTW+//TZxFRsfEAABEMgkAkiAM8mbsAUEQEApAbcxaPyGuPPOO49uvfVW60UZhw4dst6cxu0Q3KrAY86GDx9ON9xwQ1wnTlx5RBlPdPjVr34Vn/KwY8cO6+1yye0JL7/8svW2uS+++IIaNGhgTZrghDj29jlVCXAiNB63xi0bfBY/oBcbuaYULISBAAiAQMgEkACH7AAcDwIgkB0EeEIET4zgXuDY56WXXrKqwfz2NbxtLTviAFaCAAiYQQAJsBl+gBYgAAIZTuDZZ5+1xpO9+OKLVhsDj07jB+l69Ohh+6a5DMcB80AABEAgVAJIgEPFj8NBAASyhUBFRYX1MBw/VLZ9+3Zq2LCh9Wa3xJaGbGEBO0EABEAgbAJIgMP2AM4HARAAARAAARAAARAIlAAS4EBx4zAQAAEQAAEQAAEQAIGwCSABDtsDOB8EQAAEQAAEQAAEQCBQAkiAA8WNw0AABEAABEAABEAABMImgAQ4bA/gfBAAARAAARAAARAAgUAJIAEOFDcOAwEQAAEQAAEQAAEQCJsAEuCwPYDzQQAEQAAEQAAEQAAEAiWABDhQ3DgMBEAABEAABEAABEAgbAJIgMP2AM4HARAAARAAARAAARAIlAAS4EBx4zAQAAEQAAEQAAEQAIGwCSABDtsDOB8EQAAEQAAEQAAEQCBQAkiAA8WNw0AABEAABEAABEAABMImgAQ4bA/gfBAAARAAARAAARAAgUAJ/D/N+BTyXvog4wAAAABJRU5ErkJggg==\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# number of probes histogram\n",
    "fig, ax = plt.subplots()\n",
    "ax.plot(cand_prob_nums, '.')\n",
    "ax.set_ylabel(\"Num probes per region\")\n",
    "ax.set_xlabel(\"Region ID\")\n",
    "ax.set_title(\"Probe number before filtering\")\n",
    "ax.set_ylim([0,100])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 162,
   "id": "f854344b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:Chr7-6Mb-seg-1, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-2, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-3, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-4, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-5, directly append 18 probes\n",
      "-- filtering region:Chr7-6Mb-seg-6, directly append 19 probes\n",
      "-- filtering region:Chr7-6Mb-seg-7, directly append 30 probes\n",
      "-- filtering region:Chr7-6Mb-seg-8, directly append 6 probes\n",
      "-- filtering region:Chr7-6Mb-seg-9, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-10, directly append 41 probes\n",
      "-- filtering region:Chr7-6Mb-seg-11, directly append 44 probes\n",
      "-- filtering region:Chr7-6Mb-seg-12, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-13, directly append 20 probes\n",
      "-- filtering region:Chr7-6Mb-seg-14, directly append 26 probes\n",
      "-- filtering region:Chr7-6Mb-seg-15, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-16, directly append 58 probes\n",
      "-- filtering region:Chr7-6Mb-seg-17, directly append 14 probes\n",
      "-- filtering region:Chr7-6Mb-seg-18, directly append 13 probes\n",
      "-- filtering region:Chr7-6Mb-seg-19, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-20, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-21, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-22, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-23, directly append 42 probes\n",
      "-- filtering region:Chr7-6Mb-seg-24, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-25, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-26, directly append 3 probes\n",
      "-- filtering region:Chr7-6Mb-seg-27, directly append 47 probes\n",
      "-- filtering region:Chr7-6Mb-seg-28, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-29, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-30, directly append 1 probes\n",
      "-- filtering region:Chr7-6Mb-seg-31, directly append 20 probes\n",
      "-- filtering region:Chr7-6Mb-seg-32, directly append 2 probes\n",
      "-- filtering region:Chr7-6Mb-seg-33, directly append 2 probes\n",
      "-- filtering region:Chr7-6Mb-seg-34, directly append 15 probes\n",
      "-- filtering region:Chr7-6Mb-seg-35, directly append 6 probes\n",
      "-- filtering region:Chr7-6Mb-seg-36, directly append 31 probes\n",
      "-- filtering region:Chr7-6Mb-seg-37, directly append 11 probes\n",
      "-- filtering region:Chr7-6Mb-seg-38, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-39, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-40, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-41, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-42, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-43, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-44, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-45, directly append 50 probes\n",
      "-- filtering region:Chr7-6Mb-seg-46, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-47, directly append 58 probes\n",
      "-- filtering region:Chr7-6Mb-seg-48, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-49, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-50, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-51, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-52, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-53, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-54, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-55, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-56, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-57, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-58, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-59, directly append 58 probes\n",
      "-- filtering region:Chr7-6Mb-seg-60, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-61, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-62, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-63, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-64, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-65, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-66, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-67, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-68, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-69, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-70, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-71, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-72, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-73, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-74, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-75, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-76, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-77, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-78, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-79, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-80, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-81, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-82, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-83, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-84, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-85, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-86, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-87, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-88, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-89, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-90, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-91, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-92, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-93, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-94, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-95, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-96, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-97, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-98, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-99, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-100, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-101, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-102, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-103, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-104, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-105, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-106, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-107, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-108, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-109, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-110, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-111, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-112, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-113, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-114, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-115, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-116, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-117, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-118, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-119, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-120, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-121, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-122, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-123, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-124, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-125, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-126, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-127, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-128, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-129, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-130, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-131, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-132, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-133, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-134, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-135, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-136, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-137, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-138, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-139, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-140, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-141, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-142, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-143, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-144, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-145, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-146, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-147, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-148, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-149, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-150, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-151, directly append 53 probes\n",
      "-- filtering region:Chr7-6Mb-seg-152, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-153, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-154, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-155, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-156, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-157, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-158, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-159, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-160, directly append 11 probes\n",
      "-- filtering region:Chr7-6Mb-seg-161, directly append 29 probes\n",
      "-- filtering region:Chr7-6Mb-seg-162, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-163, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-164, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-165, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-166, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-167, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-168, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-169, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-170, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-171, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-172, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-173, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-174, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-175, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-176, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-177, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-178, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-179, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-180, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-181, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-182, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-183, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-184, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-185, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-186, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-187, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-188, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-189, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-190, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-191, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-192, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-193, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-194, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-195, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-196, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-197, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-198, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-199, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-200, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-201, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-202, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-203, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-204, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-205, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-206, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-207, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-208, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-209, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-210, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-211, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-212, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-213, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-214, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-215, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-216, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-217, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-218, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-219, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-220, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-221, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-222, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-223, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-224, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-225, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-226, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-227, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-228, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-229, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-230, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-231, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-232, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-233, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-234, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-235, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-236, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-237, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-238, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-239, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-240, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-241, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-242, directly append 58 probes\n",
      "-- filtering region:Chr7-6Mb-seg-243, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-244, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-245, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-246, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-247, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-248, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-249, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-250, directly append 21 probes\n",
      "-- filtering region:Chr7-6Mb-seg-251, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-252, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-253, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-254, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-255, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-256, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-257, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-258, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-259, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-260, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-261, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-262, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-263, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-264, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-265, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-266, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-267, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-268, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-269, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-270, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-271, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-272, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-273, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-274, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-275, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-276, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-277, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-278, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-279, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-280, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-281, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-282, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-283, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-284, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-285, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-286, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-287, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-288, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-289, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-290, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-291, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-292, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-293, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-294, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-295, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-296, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-297, directly append 36 probes\n",
      "-- filtering region:Chr7-6Mb-seg-298, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-299, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-300, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-301, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-302, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-303, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-304, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-305, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-306, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-307, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-308, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-309, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-310, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-311, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-312, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-313, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-314, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-315, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-316, directly append 50 probes\n",
      "-- filtering region:Chr7-6Mb-seg-317, directly append 53 probes\n",
      "-- filtering region:Chr7-6Mb-seg-318, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-319, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-320, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-321, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-322, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-323, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-324, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-325, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-326, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-327, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-328, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-329, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-330, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-331, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-332, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-333, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-334, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-335, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-336, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-337, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-338, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-339, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-340, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-341, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-342, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-343, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-344, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-345, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-346, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-347, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-348, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-349, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-350, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-351, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-352, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-353, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-354, directly append 53 probes\n",
      "-- filtering region:Chr7-6Mb-seg-355, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-356, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-357, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-358, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-359, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-360, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-361, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-362, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-363, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-364, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-365, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-366, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-367, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-368, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-369, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-370, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-371, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-372, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-373, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-374, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-375, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-376, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-377, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-378, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-379, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-380, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-381, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-382, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-383, directly append 44 probes\n",
      "-- filtering region:Chr7-6Mb-seg-384, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-385, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-386, directly append 42 probes\n",
      "-- filtering region:Chr7-6Mb-seg-387, directly append 42 probes\n",
      "-- filtering region:Chr7-6Mb-seg-388, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-389, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-390, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-391, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-392, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-393, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-394, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-395, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-396, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-397, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-398, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-399, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-400, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-401, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-402, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-403, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-404, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-405, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-406, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-407, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-408, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-409, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-410, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-411, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-412, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-413, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-414, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-415, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-416, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-417, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-418, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-419, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-420, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-421, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-422, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-423, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-424, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-425, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-426, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-427, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-428, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-429, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-430, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-431, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-432, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-433, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-434, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-435, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-436, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-437, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-438, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-439, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-440, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-441, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-442, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-443, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-444, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-445, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-446, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-447, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-448, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-449, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-450, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-451, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-452, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-453, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-454, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-455, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-456, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-457, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-458, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-459, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-460, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-461, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-462, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-463, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-464, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-465, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-466, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-467, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-468, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-469, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-470, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-471, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-472, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-473, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-474, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-475, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-476, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-477, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-478, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-479, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-480, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-481, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-482, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-483, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-484, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-485, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-486, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-487, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-488, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-489, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-490, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-491, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-492, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-493, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-494, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-495, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-496, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-497, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-498, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-499, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-500, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-501, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-502, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-503, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-504, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-505, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-506, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-507, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-508, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-509, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-510, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-511, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-512, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-513, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-514, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-515, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-516, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-517, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-518, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-519, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-520, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-521, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-522, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-523, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-524, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-525, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-526, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-527, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-528, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-529, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-530, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-531, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-532, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-533, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-534, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-535, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-536, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-537, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-538, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-539, directly append 33 probes\n",
      "-- filtering region:Chr7-6Mb-seg-540, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-541, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-542, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-543, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-544, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-545, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-546, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-547, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-548, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-549, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-550, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-551, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-552, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-553, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-554, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-555, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-556, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-557, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-558, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-559, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-560, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-561, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-562, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-563, directly append 50 probes\n",
      "-- filtering region:Chr7-6Mb-seg-564, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-565, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-566, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-567, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-568, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-569, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-570, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-571, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-572, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-573, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-574, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-575, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-576, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-577, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-578, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-579, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-580, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-581, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-582, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-583, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-584, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-585, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-586, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-587, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-588, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-589, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-590, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-591, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-592, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-593, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-594, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-595, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-596, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-597, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-598, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-599, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-600, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-601, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-602, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-603, directly append 53 probes\n",
      "-- filtering region:Chr7-6Mb-seg-604, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-605, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-606, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-607, directly append 53 probes\n",
      "-- filtering region:Chr7-6Mb-seg-608, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-609, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-610, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-611, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-612, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-613, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-614, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-615, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-616, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-617, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-618, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-619, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-620, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-621, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-622, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-623, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-624, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-625, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-626, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-627, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-628, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-629, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-630, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-631, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-632, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-633, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-634, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-635, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-637, directly append 14 probes\n",
      "-- filtering region:Chr7-6Mb-seg-638, directly append 27 probes\n",
      "-- filtering region:Chr7-6Mb-seg-639, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-640, directly append 24 probes\n",
      "-- filtering region:Chr7-6Mb-seg-641, directly append 37 probes\n",
      "-- filtering region:Chr7-6Mb-seg-642, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-643, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-644, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-645, directly append 11 probes\n",
      "-- filtering region:Chr7-6Mb-seg-646, directly append 15 probes\n",
      "-- filtering region:Chr7-6Mb-seg-647, directly append 27 probes\n",
      "-- filtering region:Chr7-6Mb-seg-648, directly append 50 probes\n",
      "-- filtering region:Chr7-6Mb-seg-649, directly append 26 probes\n",
      "-- filtering region:Chr7-6Mb-seg-650, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-651, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-652, directly append 17 probes\n",
      "-- filtering region:Chr7-6Mb-seg-654, directly append 17 probes\n",
      "-- filtering region:Chr7-6Mb-seg-655, directly append 35 probes\n",
      "-- filtering region:Chr7-6Mb-seg-656, directly append 3 probes\n",
      "-- filtering region:Chr7-6Mb-seg-658, directly append 8 probes\n",
      "-- filtering region:Chr7-6Mb-seg-659, directly append 17 probes\n",
      "-- filtering region:Chr7-6Mb-seg-660, directly append 35 probes\n",
      "-- filtering region:Chr7-6Mb-seg-661, directly append 42 probes\n",
      "-- filtering region:Chr7-6Mb-seg-662, directly append 58 probes\n",
      "-- filtering region:Chr7-6Mb-seg-663, directly append 50 probes\n",
      "-- filtering region:Chr7-6Mb-seg-664, directly append 41 probes\n",
      "-- filtering region:Chr7-6Mb-seg-665, directly append 30 probes\n",
      "-- filtering region:Chr7-6Mb-seg-666, directly append 9 probes\n",
      "-- filtering region:Chr7-6Mb-seg-667, directly append 15 probes\n",
      "-- filtering region:Chr7-6Mb-seg-668, directly append 58 probes\n",
      "-- filtering region:Chr7-6Mb-seg-669, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-670, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-671, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-672, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-673, directly append 3 probes\n",
      "-- filtering region:Chr7-6Mb-seg-674, directly append 32 probes\n",
      "-- filtering region:Chr7-6Mb-seg-675, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-676, directly append 38 probes\n",
      "-- filtering region:Chr7-6Mb-seg-677, directly append 36 probes\n",
      "-- filtering region:Chr7-6Mb-seg-678, directly append 4 probes\n",
      "-- filtering region:Chr7-6Mb-seg-679, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-680, directly append 37 probes\n",
      "-- filtering region:Chr7-6Mb-seg-681, directly append 33 probes\n",
      "-- filtering region:Chr7-6Mb-seg-682, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-683, directly append 27 probes\n",
      "-- filtering region:Chr7-6Mb-seg-684, directly append 7 probes\n",
      "-- filtering region:Chr7-6Mb-seg-685, directly append 7 probes\n",
      "-- filtering region:Chr7-6Mb-seg-686, directly append 7 probes\n",
      "-- filtering region:Chr7-6Mb-seg-687, directly append 1 probes\n",
      "-- filtering region:Chr7-6Mb-seg-688, directly append 9 probes\n",
      "-- filtering region:Chr7-6Mb-seg-690, directly append 5 probes\n",
      "-- filtering region:Chr7-6Mb-seg-691, directly append 16 probes\n",
      "-- filtering region:Chr7-6Mb-seg-692, directly append 32 probes\n",
      "-- filtering region:Chr7-6Mb-seg-693, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-694, directly append 16 probes\n",
      "-- filtering region:Chr7-6Mb-seg-695, directly append 19 probes\n",
      "-- filtering region:Chr7-6Mb-seg-696, directly append 15 probes\n",
      "-- filtering region:Chr7-6Mb-seg-697, directly append 17 probes\n",
      "-- filtering region:Chr7-6Mb-seg-698, directly append 32 probes\n",
      "-- filtering region:Chr7-6Mb-seg-699, directly append 28 probes\n",
      "-- filtering region:Chr7-6Mb-seg-700, directly append 37 probes\n",
      "-- filtering region:Chr7-6Mb-seg-701, directly append 41 probes\n",
      "-- filtering region:Chr7-6Mb-seg-702, directly append 9 probes\n",
      "-- filtering region:Chr7-6Mb-seg-703, directly append 25 probes\n",
      "-- filtering region:Chr7-6Mb-seg-704, directly append 41 probes\n",
      "-- filtering region:Chr7-6Mb-seg-705, directly append 31 probes\n",
      "-- filtering region:Chr7-6Mb-seg-706, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-707, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-708, directly append 31 probes\n",
      "-- filtering region:Chr7-6Mb-seg-709, directly append 5 probes\n",
      "-- filtering region:Chr7-6Mb-seg-710, directly append 29 probes\n",
      "-- filtering region:Chr7-6Mb-seg-711, directly append 24 probes\n",
      "-- filtering region:Chr7-6Mb-seg-712, directly append 27 probes\n",
      "-- filtering region:Chr7-6Mb-seg-713, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-714, directly append 18 probes\n",
      "-- filtering region:Chr7-6Mb-seg-716, directly append 3 probes\n",
      "-- filtering region:Chr7-6Mb-seg-717, directly append 23 probes\n",
      "-- filtering region:Chr7-6Mb-seg-718, directly append 24 probes\n",
      "-- filtering region:Chr7-6Mb-seg-719, directly append 29 probes\n",
      "-- filtering region:Chr7-6Mb-seg-720, directly append 41 probes\n",
      "-- filtering region:Chr7-6Mb-seg-721, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-722, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-723, directly append 32 probes\n",
      "-- filtering region:Chr7-6Mb-seg-724, directly append 6 probes\n",
      "-- filtering region:Chr7-6Mb-seg-725, directly append 13 probes\n",
      "-- filtering region:Chr7-6Mb-seg-726, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-727, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-728, directly append 22 probes\n",
      "-- filtering region:Chr7-6Mb-seg-729, directly append 13 probes\n",
      "-- filtering region:Chr7-6Mb-seg-730, directly append 14 probes\n",
      "-- filtering region:Chr7-6Mb-seg-731, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-732, directly append 23 probes\n",
      "-- filtering region:Chr7-6Mb-seg-733, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-734, directly append 25 probes\n",
      "-- filtering region:Chr7-6Mb-seg-735, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-736, directly append 10 probes\n",
      "-- filtering region:Chr7-6Mb-seg-737, directly append 2 probes\n",
      "-- filtering region:Chr7-6Mb-seg-738, directly append 38 probes\n",
      "-- filtering region:Chr7-6Mb-seg-739, directly append 41 probes\n",
      "-- filtering region:Chr7-6Mb-seg-740, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-741, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-742, directly append 37 probes\n",
      "-- filtering region:Chr7-6Mb-seg-743, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-744, directly append 44 probes\n",
      "-- filtering region:Chr7-6Mb-seg-745, directly append 32 probes\n",
      "-- filtering region:Chr7-6Mb-seg-746, directly append 35 probes\n",
      "-- filtering region:Chr7-6Mb-seg-747, directly append 28 probes\n",
      "-- filtering region:Chr7-6Mb-seg-748, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-749, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-750, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-751, directly append 45 probes\n",
      "-- filtering region:Chr7-6Mb-seg-752, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-753, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-754, directly append 25 probes\n",
      "-- filtering region:Chr7-6Mb-seg-755, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-756, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-757, directly append 50 probes\n",
      "-- filtering region:Chr7-6Mb-seg-758, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-759, directly append 13 probes\n",
      "-- filtering region:Chr7-6Mb-seg-760, directly append 27 probes\n",
      "-- filtering region:Chr7-6Mb-seg-761, directly append 5 probes\n",
      "-- filtering region:Chr7-6Mb-seg-762, directly append 23 probes\n",
      "-- filtering region:Chr7-6Mb-seg-763, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-764, directly append 42 probes\n",
      "-- filtering region:Chr7-6Mb-seg-765, directly append 24 probes\n",
      "-- filtering region:Chr7-6Mb-seg-766, directly append 50 probes\n",
      "-- filtering region:Chr7-6Mb-seg-767, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-768, directly append 42 probes\n",
      "-- filtering region:Chr7-6Mb-seg-769, directly append 24 probes\n",
      "-- filtering region:Chr7-6Mb-seg-770, directly append 21 probes\n",
      "-- filtering region:Chr7-6Mb-seg-771, directly append 28 probes\n",
      "-- filtering region:Chr7-6Mb-seg-772, directly append 47 probes\n",
      "-- filtering region:Chr7-6Mb-seg-773, directly append 29 probes\n",
      "-- filtering region:Chr7-6Mb-seg-774, directly append 6 probes\n",
      "-- filtering region:Chr7-6Mb-seg-775, directly append 36 probes\n",
      "-- filtering region:Chr7-6Mb-seg-776, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-777, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-778, directly append 41 probes\n",
      "-- filtering region:Chr7-6Mb-seg-779, directly append 6 probes\n",
      "-- filtering region:Chr7-6Mb-seg-780, directly append 26 probes\n",
      "-- filtering region:Chr7-6Mb-seg-781, directly append 38 probes\n",
      "-- filtering region:Chr7-6Mb-seg-782, directly append 1 probes\n",
      "-- filtering region:Chr7-6Mb-seg-783, directly append 8 probes\n",
      "-- filtering region:Chr7-6Mb-seg-784, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-785, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-786, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-787, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-788, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-789, directly append 17 probes\n",
      "-- filtering region:Chr7-6Mb-seg-790, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-791, directly append 25 probes\n",
      "-- filtering region:Chr7-6Mb-seg-792, directly append 43 probes\n",
      "-- filtering region:Chr7-6Mb-seg-793, directly append 29 probes\n",
      "-- filtering region:Chr7-6Mb-seg-794, directly append 37 probes\n",
      "-- filtering region:Chr7-6Mb-seg-795, directly append 24 probes\n",
      "-- filtering region:Chr7-6Mb-seg-796, directly append 24 probes\n",
      "-- filtering region:Chr7-6Mb-seg-797, directly append 44 probes\n",
      "-- filtering region:Chr7-6Mb-seg-798, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-799, directly append 45 probes\n",
      "-- filtering region:Chr7-6Mb-seg-800, directly append 27 probes\n",
      "-- filtering region:Chr7-6Mb-seg-801, directly append 23 probes\n",
      "-- filtering region:Chr7-6Mb-seg-802, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-803, directly append 4 probes\n",
      "-- filtering region:Chr7-6Mb-seg-804, directly append 43 probes\n",
      "-- filtering region:Chr7-6Mb-seg-805, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-806, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-807, directly append 53 probes\n",
      "-- filtering region:Chr7-6Mb-seg-808, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-809, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-810, directly append 27 probes\n",
      "-- filtering region:Chr7-6Mb-seg-811, directly append 18 probes\n",
      "-- filtering region:Chr7-6Mb-seg-812, directly append 22 probes\n",
      "-- filtering region:Chr7-6Mb-seg-813, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-814, directly append 23 probes\n",
      "-- filtering region:Chr7-6Mb-seg-815, directly append 8 probes\n",
      "-- filtering region:Chr7-6Mb-seg-816, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-817, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-818, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-819, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-820, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-821, directly append 30 probes\n",
      "-- filtering region:Chr7-6Mb-seg-822, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-823, directly append 37 probes\n",
      "-- filtering region:Chr7-6Mb-seg-824, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-825, directly append 30 probes\n",
      "-- filtering region:Chr7-6Mb-seg-826, directly append 1 probes\n",
      "-- filtering region:Chr7-6Mb-seg-827, directly append 13 probes\n",
      "-- filtering region:Chr7-6Mb-seg-828, directly append 27 probes\n",
      "-- filtering region:Chr7-6Mb-seg-829, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-830, directly append 9 probes\n",
      "-- filtering region:Chr7-6Mb-seg-831, directly append 25 probes\n",
      "-- filtering region:Chr7-6Mb-seg-832, directly append 29 probes\n",
      "-- filtering region:Chr7-6Mb-seg-833, directly append 32 probes\n",
      "-- filtering region:Chr7-6Mb-seg-834, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-835, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-836, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-837, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-838, directly append 16 probes\n",
      "-- filtering region:Chr7-6Mb-seg-839, directly append 16 probes\n",
      "-- filtering region:Chr7-6Mb-seg-840, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-841, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-842, directly append 53 probes\n",
      "-- filtering region:Chr7-6Mb-seg-843, directly append 18 probes\n",
      "-- filtering region:Chr7-6Mb-seg-844, directly append 11 probes\n",
      "-- filtering region:Chr7-6Mb-seg-845, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-846, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-847, directly append 22 probes\n",
      "-- filtering region:Chr7-6Mb-seg-848, directly append 31 probes\n",
      "-- filtering region:Chr7-6Mb-seg-849, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-850, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-851, directly append 11 probes\n",
      "-- filtering region:Chr7-6Mb-seg-852, directly append 27 probes\n",
      "-- filtering region:Chr7-6Mb-seg-853, directly append 43 probes\n",
      "-- filtering region:Chr7-6Mb-seg-854, directly append 33 probes\n",
      "-- filtering region:Chr7-6Mb-seg-855, directly append 38 probes\n",
      "-- filtering region:Chr7-6Mb-seg-856, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-857, directly append 42 probes\n",
      "-- filtering region:Chr7-6Mb-seg-858, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-859, directly append 28 probes\n",
      "-- filtering region:Chr7-6Mb-seg-860, directly append 30 probes\n",
      "-- filtering region:Chr7-6Mb-seg-862, directly append 44 probes\n",
      "-- filtering region:Chr7-6Mb-seg-863, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-864, directly append 38 probes\n",
      "-- filtering region:Chr7-6Mb-seg-865, directly append 33 probes\n",
      "-- filtering region:Chr7-6Mb-seg-866, directly append 35 probes\n",
      "-- filtering region:Chr7-6Mb-seg-867, directly append 11 probes\n",
      "-- filtering region:Chr7-6Mb-seg-868, directly append 43 probes\n",
      "-- filtering region:Chr7-6Mb-seg-869, directly append 37 probes\n",
      "-- filtering region:Chr7-6Mb-seg-870, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-871, directly append 8 probes\n",
      "-- filtering region:Chr7-6Mb-seg-872, directly append 8 probes\n",
      "-- filtering region:Chr7-6Mb-seg-873, directly append 39 probes\n",
      "-- filtering region:Chr7-6Mb-seg-874, directly append 11 probes\n",
      "-- filtering region:Chr7-6Mb-seg-875, directly append 20 probes\n",
      "-- filtering region:Chr7-6Mb-seg-876, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-877, directly append 7 probes\n",
      "-- filtering region:Chr7-6Mb-seg-878, directly append 44 probes\n",
      "-- filtering region:Chr7-6Mb-seg-879, directly append 47 probes\n",
      "-- filtering region:Chr7-6Mb-seg-880, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-881, directly append 34 probes\n",
      "-- filtering region:Chr7-6Mb-seg-882, directly append 13 probes\n",
      "-- filtering region:Chr7-6Mb-seg-883, directly append 26 probes\n",
      "-- filtering region:Chr7-6Mb-seg-884, directly append 12 probes\n",
      "-- filtering region:Chr7-6Mb-seg-885, directly append 33 probes\n",
      "-- filtering region:Chr7-6Mb-seg-886, directly append 7 probes\n",
      "-- filtering region:Chr7-6Mb-seg-887, directly append 14 probes\n",
      "-- filtering region:Chr7-6Mb-seg-888, directly append 27 probes\n",
      "-- filtering region:Chr7-6Mb-seg-889, directly append 58 probes\n",
      "-- filtering region:Chr7-6Mb-seg-890, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-891, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-892, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-893, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-894, directly append 43 probes\n",
      "-- filtering region:Chr7-6Mb-seg-895, directly append 46 probes\n",
      "-- filtering region:Chr7-6Mb-seg-896, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-897, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-898, directly append 15 probes\n",
      "-- filtering region:Chr7-6Mb-seg-899, directly append 15 probes\n",
      "-- filtering region:Chr7-6Mb-seg-900, directly append 58 probes\n",
      "-- filtering region:Chr7-6Mb-seg-901, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-902, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-903, directly append 14 probes\n",
      "-- filtering region:Chr7-6Mb-seg-904, directly append 36 probes\n",
      "-- filtering region:Chr7-6Mb-seg-905, directly append 9 probes\n",
      "-- filtering region:Chr7-6Mb-seg-906, directly append 23 probes\n",
      "-- filtering region:Chr7-6Mb-seg-907, directly append 42 probes\n",
      "-- filtering region:Chr7-6Mb-seg-909, directly append 3 probes\n",
      "-- filtering region:Chr7-6Mb-seg-910, directly append 4 probes\n",
      "-- filtering region:Chr7-6Mb-seg-911, directly append 50 probes\n",
      "-- filtering region:Chr7-6Mb-seg-912, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-913, directly append 22 probes\n",
      "-- filtering region:Chr7-6Mb-seg-914, directly append 7 probes\n",
      "-- filtering region:Chr7-6Mb-seg-915, directly append 1 probes\n",
      "-- filtering region:Chr7-6Mb-seg-916, directly append 32 probes\n",
      "-- filtering region:Chr7-6Mb-seg-917, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-918, directly append 36 probes\n",
      "-- filtering region:Chr7-6Mb-seg-919, directly append 34 probes\n",
      "-- filtering region:Chr7-6Mb-seg-920, directly append 17 probes\n",
      "-- filtering region:Chr7-6Mb-seg-921, directly append 3 probes\n",
      "-- filtering region:Chr7-6Mb-seg-922, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-923, directly append 41 probes\n",
      "-- filtering region:Chr7-6Mb-seg-924, directly append 33 probes\n",
      "-- filtering region:Chr7-6Mb-seg-926, directly append 38 probes\n",
      "-- filtering region:Chr7-6Mb-seg-927, directly append 16 probes\n",
      "-- filtering region:Chr7-6Mb-seg-929, directly append 40 probes\n",
      "-- filtering region:Chr7-6Mb-seg-930, directly append 4 probes\n",
      "-- filtering region:Chr7-6Mb-seg-931, directly append 21 probes\n",
      "-- filtering region:Chr7-6Mb-seg-932, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-933, directly append 36 probes\n",
      "-- filtering region:Chr7-6Mb-seg-934, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-935, directly append 28 probes\n",
      "-- filtering region:Chr7-6Mb-seg-936, directly append 4 probes\n",
      "-- filtering region:Chr7-6Mb-seg-937, directly append 25 probes\n",
      "-- filtering region:Chr7-6Mb-seg-938, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-939, directly append 34 probes\n",
      "-- filtering region:Chr7-6Mb-seg-940, directly append 1 probes\n",
      "-- filtering region:Chr7-6Mb-seg-941, directly append 8 probes\n",
      "-- filtering region:Chr7-6Mb-seg-942, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-943, directly append 2 probes\n",
      "-- filtering region:Chr7-6Mb-seg-944, directly append 42 probes\n",
      "-- filtering region:Chr7-6Mb-seg-945, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-946, directly append 44 probes\n",
      "-- filtering region:Chr7-6Mb-seg-947, directly append 21 probes\n",
      "-- filtering region:Chr7-6Mb-seg-948, directly append 7 probes\n",
      "-- filtering region:Chr7-6Mb-seg-949, directly append 2 probes\n",
      "-- filtering region:Chr7-6Mb-seg-950, directly append 31 probes\n",
      "-- filtering region:Chr7-6Mb-seg-951, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-952, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-953, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-954, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-955, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-956, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-957, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-958, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-959, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-960, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-961, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-962, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-963, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-964, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-965, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-966, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-967, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-968, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-969, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-970, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-971, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-972, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-973, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-974, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-975, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-976, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-977, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-978, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-979, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-980, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-981, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-982, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-983, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-984, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-985, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-986, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-987, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-988, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-989, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-990, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-991, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-992, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-993, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-994, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-995, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-996, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-997, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-998, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-999, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1000, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1001, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1002, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1003, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1004, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1005, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1006, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1007, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1008, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1009, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1010, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1011, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1012, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1013, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1014, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1015, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1016, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1017, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1018, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1019, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1020, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1021, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1022, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1023, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1024, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1025, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1026, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1027, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1028, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1029, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1030, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1031, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1032, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1033, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1034, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1035, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1036, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1037, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1038, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1039, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1040, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1041, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1042, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1043, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1044, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1045, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1046, directly append 57 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1047, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1048, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1049, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1050, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1051, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1052, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1053, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1054, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1055, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1056, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1057, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1058, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1059, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1060, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1061, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1062, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1063, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1064, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1065, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1066, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1067, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1068, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1069, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1070, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1071, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1072, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1073, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1074, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1075, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1076, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1077, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1078, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1079, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1080, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1081, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1082, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1083, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1084, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1085, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1086, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1087, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1088, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1089, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1090, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1091, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1092, directly append 60 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1093, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1094, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1095, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1096, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1097, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1098, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1099, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1100, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1101, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1102, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1103, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1104, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1105, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1106, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1107, directly append 63 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1108, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1109, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1110, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1111, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1112, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1113, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1114, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1115, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1116, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1117, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1118, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1119, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1120, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1121, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1122, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1123, directly append 62 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1124, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1125, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1126, directly append 56 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1127, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1128, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1129, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1130, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1131, directly append 52 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1132, directly append 78 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1133, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1134, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1135, directly append 54 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1136, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1137, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1138, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1139, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1140, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1141, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1142, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1143, directly append 58 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1144, directly append 48 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1145, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1146, directly append 49 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1147, directly append 61 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1148, directly append 42 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1149, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1150, directly append 59 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1151, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1152, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1153, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1154, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1155, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1156, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1157, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1158, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1159, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1160, directly append 69 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1161, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1162, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1163, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1164, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1165, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1166, directly append 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1167, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1168, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1169, directly append 70 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1170, directly append 66 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1171, directly append 67 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1172, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1173, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1174, directly append 71 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1175, directly append 72 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1176, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1177, directly append 55 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1178, directly append 76 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1179, directly append 64 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1180, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1181, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1182, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1183, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1184, directly append 79 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1185, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1186, directly append 68 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1187, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1188, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1189, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1190, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1191, directly append 65 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1192, directly append 77 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1193, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1194, directly append 75 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1195, directly append 73 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1196, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1197, screen genome to keep 80 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1198, directly append 74 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1199, directly append 51 probes\n",
      "-- filtering region:Chr7-6Mb-seg-1200, directly append 63 probes\n",
      "region: Chr7-6Mb-seg-5 removed with 18 probes\n",
      "region: Chr7-6Mb-seg-6 removed with 19 probes\n",
      "region: Chr7-6Mb-seg-8 removed with 6 probes\n",
      "region: Chr7-6Mb-seg-13 removed with 20 probes\n",
      "region: Chr7-6Mb-seg-17 removed with 14 probes\n",
      "region: Chr7-6Mb-seg-18 removed with 13 probes\n",
      "region: Chr7-6Mb-seg-26 removed with 3 probes\n",
      "region: Chr7-6Mb-seg-30 removed with 1 probes\n",
      "region: Chr7-6Mb-seg-31 removed with 20 probes\n",
      "region: Chr7-6Mb-seg-32 removed with 2 probes\n",
      "region: Chr7-6Mb-seg-33 removed with 2 probes\n",
      "region: Chr7-6Mb-seg-34 removed with 15 probes\n",
      "region: Chr7-6Mb-seg-35 removed with 6 probes\n",
      "region: Chr7-6Mb-seg-37 removed with 11 probes\n",
      "region: Chr7-6Mb-seg-160 removed with 11 probes\n",
      "region: Chr7-6Mb-seg-637 removed with 14 probes\n",
      "region: Chr7-6Mb-seg-645 removed with 11 probes\n",
      "region: Chr7-6Mb-seg-646 removed with 15 probes\n",
      "region: Chr7-6Mb-seg-652 removed with 17 probes\n",
      "region: Chr7-6Mb-seg-654 removed with 17 probes\n",
      "region: Chr7-6Mb-seg-656 removed with 3 probes\n",
      "region: Chr7-6Mb-seg-658 removed with 8 probes\n",
      "region: Chr7-6Mb-seg-659 removed with 17 probes\n",
      "region: Chr7-6Mb-seg-666 removed with 9 probes\n",
      "region: Chr7-6Mb-seg-667 removed with 15 probes\n",
      "region: Chr7-6Mb-seg-673 removed with 3 probes\n",
      "region: Chr7-6Mb-seg-678 removed with 4 probes\n",
      "region: Chr7-6Mb-seg-684 removed with 7 probes\n",
      "region: Chr7-6Mb-seg-685 removed with 7 probes\n",
      "region: Chr7-6Mb-seg-686 removed with 7 probes\n",
      "region: Chr7-6Mb-seg-687 removed with 1 probes\n",
      "region: Chr7-6Mb-seg-688 removed with 9 probes\n",
      "region: Chr7-6Mb-seg-690 removed with 5 probes\n",
      "region: Chr7-6Mb-seg-691 removed with 16 probes\n",
      "region: Chr7-6Mb-seg-694 removed with 16 probes\n",
      "region: Chr7-6Mb-seg-695 removed with 19 probes\n",
      "region: Chr7-6Mb-seg-696 removed with 15 probes\n",
      "region: Chr7-6Mb-seg-697 removed with 17 probes\n",
      "region: Chr7-6Mb-seg-702 removed with 9 probes\n",
      "region: Chr7-6Mb-seg-709 removed with 5 probes\n",
      "region: Chr7-6Mb-seg-714 removed with 18 probes\n",
      "region: Chr7-6Mb-seg-716 removed with 3 probes\n",
      "region: Chr7-6Mb-seg-724 removed with 6 probes\n",
      "region: Chr7-6Mb-seg-725 removed with 13 probes\n",
      "region: Chr7-6Mb-seg-729 removed with 13 probes\n",
      "region: Chr7-6Mb-seg-730 removed with 14 probes\n",
      "region: Chr7-6Mb-seg-736 removed with 10 probes\n",
      "region: Chr7-6Mb-seg-737 removed with 2 probes\n",
      "region: Chr7-6Mb-seg-759 removed with 13 probes\n",
      "region: Chr7-6Mb-seg-761 removed with 5 probes\n",
      "region: Chr7-6Mb-seg-774 removed with 6 probes\n",
      "region: Chr7-6Mb-seg-779 removed with 6 probes\n",
      "region: Chr7-6Mb-seg-782 removed with 1 probes\n",
      "region: Chr7-6Mb-seg-783 removed with 8 probes\n",
      "region: Chr7-6Mb-seg-789 removed with 17 probes\n",
      "region: Chr7-6Mb-seg-803 removed with 4 probes\n",
      "region: Chr7-6Mb-seg-811 removed with 18 probes\n",
      "region: Chr7-6Mb-seg-815 removed with 8 probes\n",
      "region: Chr7-6Mb-seg-826 removed with 1 probes\n",
      "region: Chr7-6Mb-seg-827 removed with 13 probes\n",
      "region: Chr7-6Mb-seg-830 removed with 9 probes\n",
      "region: Chr7-6Mb-seg-838 removed with 16 probes\n",
      "region: Chr7-6Mb-seg-839 removed with 16 probes\n",
      "region: Chr7-6Mb-seg-843 removed with 18 probes\n",
      "region: Chr7-6Mb-seg-844 removed with 11 probes\n",
      "region: Chr7-6Mb-seg-851 removed with 11 probes\n",
      "region: Chr7-6Mb-seg-867 removed with 11 probes\n",
      "region: Chr7-6Mb-seg-871 removed with 8 probes\n",
      "region: Chr7-6Mb-seg-872 removed with 8 probes\n",
      "region: Chr7-6Mb-seg-874 removed with 11 probes\n",
      "region: Chr7-6Mb-seg-875 removed with 20 probes\n",
      "region: Chr7-6Mb-seg-877 removed with 7 probes\n",
      "region: Chr7-6Mb-seg-882 removed with 13 probes\n",
      "region: Chr7-6Mb-seg-884 removed with 12 probes\n",
      "region: Chr7-6Mb-seg-886 removed with 7 probes\n",
      "region: Chr7-6Mb-seg-887 removed with 14 probes\n",
      "region: Chr7-6Mb-seg-898 removed with 15 probes\n",
      "region: Chr7-6Mb-seg-899 removed with 15 probes\n",
      "region: Chr7-6Mb-seg-903 removed with 14 probes\n",
      "region: Chr7-6Mb-seg-905 removed with 9 probes\n",
      "region: Chr7-6Mb-seg-909 removed with 3 probes\n",
      "region: Chr7-6Mb-seg-910 removed with 4 probes\n",
      "region: Chr7-6Mb-seg-914 removed with 7 probes\n",
      "region: Chr7-6Mb-seg-915 removed with 1 probes\n",
      "region: Chr7-6Mb-seg-920 removed with 17 probes\n",
      "region: Chr7-6Mb-seg-921 removed with 3 probes\n",
      "region: Chr7-6Mb-seg-927 removed with 16 probes\n",
      "region: Chr7-6Mb-seg-930 removed with 4 probes\n",
      "region: Chr7-6Mb-seg-936 removed with 4 probes\n",
      "region: Chr7-6Mb-seg-940 removed with 1 probes\n",
      "region: Chr7-6Mb-seg-941 removed with 8 probes\n",
      "region: Chr7-6Mb-seg-943 removed with 2 probes\n",
      "region: Chr7-6Mb-seg-948 removed with 7 probes\n",
      "region: Chr7-6Mb-seg-949 removed with 2 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts\n",
    "reload(library_tools.quality_check)\n",
    "\n",
    "num_probes_kept = 80 # max num of probes kept for each region\n",
    "filtered_pb_dict = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dict, num_probes_kept,\n",
    "                                                                  hit_type='genome')\n",
    "\n",
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 21\n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dict.items():\n",
    "    if len(_pb_class.kept_probes) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.kept_probes)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dict[_rid])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 163,
   "id": "17a59658",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(1191, 1097)"
      ]
     },
     "execution_count": 163,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(pb_dict), len(filtered_pb_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 166,
   "id": "a0248433",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7snQe4HVXV99e9N8m96Z2QnlDyQgIkhCT0JiJFwYAIvIAEhPcVIgLS8lGkCCgBQXhFQKWIAaQlgYgCAtIhjRQSAgbSewjpvd3vWQP7Onfu7Nlrz+yZM+ee/zyPj5qzy1q/vfbcdfb5z5qy6urqasIFAiAAAiAAAiAAAiAAAiVCoAwJcImsNNwEARAAARAAARAAARDwCCABRiCAAAiAAAiAAAiAAAiUFAEkwCW13HAWBEAABEAABEAABEAACTBiAARAAARAAARAAARAoKQIIAEuqeWGsyAAAiAAAiAAAiAAAkiAEQMgAAIgAAIgAAIgAAIlRQAJcEktN5wFARAAARAAARAAARBAAowYAAEQAAEQAAEQAAEQKCkCSIBLarnhLAiAAAiAAAiAAAiAABJgxAAIgAAIgAAIgAAIgEBJEUACXFLLDWdBAARAAARAAARAAASQACMGQAAEQAAEQAAEQAAESooAEuCSWm44CwIgAAIgAAIgAAIggAQYMQACIAACIAACIAACIFBSBJAAl9Ryw1kQAAEQAAEQAAEQAAEkwIgBEAABEAABEAABEACBkiKABLiklhvOggAIgAAIgAAIgAAIIAFGDIAACIAACIAACIAACJQUASTAJbXccBYEQAAEQAAEQAAEQAAJMGIABEAABEAABEAABECgpAggAS6p5YazIAACIAACIAACIAACSIARAyAAAiAAAiAAAiAAAiVFAAlwSS03nAUBEAABEAABEAABEEACjBgAARAAARAAARAAARAoKQJIgEtqueEsCIAACIAACIAACIAAEmDEAAiAAAiAAAiAAAiAQEkRQAJcUssNZ0EABEAABEAABEAABJAAIwZAAARAAARAAARAAARKigAS4JJabjgLAiAAAiAAAiAAAiCABBgxAAIgAAIgAAIgAAIgUFIEkACX1HLDWRAAARAAARAAARAAASTAiAEQAAEQAAEQAAEQAIGSIoAEuKSWG86CAAiAAAiAAAiAAAggAUYMgAAIgAAIgAAIgAAIlBQBJMAltdxwFgRAAARAAARAAARAAAkwYgAEQAAEQAAEQAAEQKCkCCABLqnlhrMgAAIgAAIgAAIgAAJIgBEDIAACIAACIAACIAACJUUACXBJLTecBQEQAAEQAAEQAAEQQAKMGACBEiBw3nnn0eOPP17L07KyMmrSpAntueeedO6559Jll11G5eXlTmjMnTuXevbsScOGDaM77rjDyZjFMMhRRx1Fn332GS1dujTX5m7dupV+9rOf0TPPPEPbt2+nX//6197/X758OVVVVVGLFi1StX/hwoXEMfnBBx9487322ms0YMAAOuOMM+jpp5/25ub49P9//jfX9r311lt09NFH04MPPkgXXXRRqj5jcBAAgXwRQAKcr/WANSCQCgGVAP/2t7+ldu3aeXNUV1fT+vXrafTo0V4CwgnQ//3f/zmZHwlwvhPg+++/31vv008/nb7zne/QoYceSnPmzKGzzjqLPvzwQ9prr72cxIFukNNOO41GjhxJ119/PXXr1s2b94UXXqAePXrQYYcdFpoAv/zyy87tW7ZsmRf7Bx54oPdFEBcIgEDpEEACXDprDU9LmIBKgDnJ4STDf+3YsYMOOeQQmjhxIvHJXMeOHROTQgKc7wT4f//3f+lPf/oTcQK4yy67eOt988030y233EKffvpp6glwr169qEGDBjRjxgxtrAVPgLO0L/EGwAAgAAK5J4AEOPdLBANBIDmBqASYR7/nnnvoyiuvpFGjRtEpp5ySeEIkwPlOgFU8bNq0yZMgZJ0A85ewXXfdlcaOHYsEOPFuwwAgAAJxCCABjkMNfUCgyAiYEuB7772Xfv7zn3uaUP5ZnNu/99579P/+3//zdLysE33ggQfo7LPPptWrV9ONN97oSSf4BLFLly6eVvMXv/iFpynmSyXAnFQ3bNiQHn30UU9ucfjhh3ua4H79+tUiyD9v33777TR58mSqqKjwfga/7bbbqH///pGklZ3PPfccXXXVVV5CxQnd97//ffrNb35Dbdq0iUzuHnroIbr44ovpzTffJNbvKrvZ3lmzZnl2s7/8E/kf/vAHatSoEV166aX0r3/9i1q1auVpp2+99VbPZr6UBpj11ldffTX9+9//pt12282TGwwdOrSWL4sWLaIbbriB/vGPf3hz8E/wbMtPf/rTmnZ//vOf6fzzz/fkArwOfEL/4x//mH7/+9+HcmHGw4cP977IzJ4929N082kr67uZFV98suq/unfv7tnt14gfeeSRxPpYvljTzHayz5ww9+nTh6655hovTtTFp7O8Xixj+MlPfuL5w/HA8eO/lObW/29Dhgwh9jN44uv//0ENuyv7DjrooFoaYLX+jz32GM2bN89bf47x//qv/6KbbrqJTj311Fr+PPXUU3TnnXd669y5c2cvBvmXlNdff92LJVwgAAL5JYAEOL9rA8tAwBkBUwL83e9+10vEPv/8c9pjjz28ZOnZZ5+lpk2bekkMJzQXXnghtWzZ0tOL8h/8//mf/6H99tvPSzr/8pe/eP/OSRIniSqR4P7chxNATqI50d6yZQuNHz+e9t57b88/Tn44qeOkl7WhnMQ9/PDDtGTJEnrjjTc8eYbuYjtffPFFL3kaPHgwcULDifuIESPohz/8oecDX7qfz3UJMCf1rVu39pK5BQsWeMk0J0EbNmzwEqaDDz6YnnzySXrnnXfoj3/8o8dCJcCsoWV7OLHbd999PRveffddL1HmRJKvxYsX08CBA2nbtm1e0tuhQwd69dVXacyYMV6Cfd9999Ww4QS4efPmni2cZLEdJ5xwQigStm3cuHFeEs06Xn4Yj+2bP3++l9By4vjEE094/8Y2cYLHfvL8d911l/elhhM6/oJy7LHH0rRp07x14S8SLJvg9eRknP3mXw34S5Pi+8tf/tIbi/+NvxB8+9vf9nz0X0pzy204Lnhddt99d49nVALMTNOwT62neghOxS1/KWCJBj8Yx76wr/yF5eOPP6Z99tnHc4n78JcajjnWMHNf1lY3a9bMWy8kwM5uXxgIBFIhgAQ4FawYFATyRUAlwJMmTaKuXbt6xrH2l5M7ToL4j/kPfvADev75573PVHv+jBMwdV133XVexQBOovg0WF133323d/rFCS6fNvoTYNaUqjk5oeLkimUWPNfatWu9z/gEkhNZda1Zs8ZLHtu3b08fffSRFqayk0+V+YRUXZwIciLM4/CptG0CzLrYL774wktk+OJkmu31PyjIY3O7733ve15SyBf78fbbbxPzuOKKK7x/48T/iCOOIGbPSVTbtm09vpxsTp06tZYmm9nxg4j87/zlQp0A80lzsIpHEMqECRNo0KBBxA86Xn755TUf81jM3G+7VALB/vBJMid+fOLN186dO71YeeWVV7zEmtdI8eVfBlhHbLrCJBBRCTCPF7aGSe0LVoFQcctfCPhLHifpfPGa8lwc//xLBX9JU19G3n//fe9XDr745J3ZcAKNBNgUBfgcBApLAAlwYfljdhDIhEBYGTQ1MZ908QkWn16phE+15+SHy5mpq3fv3rR582ZPHuD/KZ3LanEitP/++3snjSqR4FNDlg74L646wOWvOIHkn8z51JeTuxNPPLFWO06mODHnn/052Qi7dCfbnHxyIsgJGifYtgnwOeec450iq4tPbjnxUVIJ9e88Nv+H/eGLkyROdFesWOGdhKuLfyrnLwx//etfPekAn6jyqad/Dm7LXxY4Wea5ONlSCXDwi4guaJgpS0AqKyu9Jlzpg79A8Eksf5HhcfiSJMBfffWVVzGE15Dt8V98Ws2M+BScY0fx5V8A+MuH6XKRALuwT5cAK1mG8oO/qHEyzL9UPPLII96XNf7FIfhFkNvzLxssFUECbIoCfA4ChSWABLiw/DE7CGRCQCU8/AebT7f4Yn0o/1Hnn8r5p23/pdrzSZf/s8aNG3tJHmt2gxcnv6tWrfL+8KsE2P8zuWrPPxtzYssSB058gzrR4LicXHKyGHYpO/mnbKU/5nY8JmthVdUL2wSYtcsse1CX6s9VC5R0gz8LJnLMhhnwqav/YskH64j59JwTUX4ALOpSXxxUAvzSSy8Ry1RM18aNG4n1q5zYsZyFT7GZDV/+pE6SALOUgn/ej7p+9atf0bXXXluTAE+fPt3TCJsuFwmwC/t0CTDrt1kKoi7+0sexrxiqXzx4XYMyD/51g7XsSIBNUYDPQaCwBJAAF5Y/ZgeBTAiYNMBBI8ISJG7Dp4t8wheWAPft29c7+Zo5c2ZNAqwkEf7xWVfJp8JffvmlV4qLTzr5oS5+WCvsOuCAAzxtadils1OaAKt6uMGH4IIv8NAl0GEJMJ/CcgLkv1gnzUk861j5JLhTp0508skne7KEsIs/59N2lQAz7+OPPz4yVlinzXpdTnyPOeYY4vVg6QMnaKyztU2AWXfL+mtOxlkCEnbxA378H9sSZS4SYBf26RLg4PoHE2CW3HDir6Qqfjb//d//7dVSRgKcya0Nk4BAbAJIgGOjQ0cQKB4CrhJgPv3kZIClEUEJBP9czskWP7imToBZ08pJsP/iBHrKlCm0cuVK761f/BM6SwPOPPPMOkkj//TMkgBVqkuaqAcTYH4AjSUVPC8nhuriSgVcvcBlAsw6Z35jmaoMwXPxSTevAWtETzrpJO/knStisI7Wf/GXAn7AjN+KxjpSmwSYH0LjSgV///vfa8lJeK3iJMB8Qs+JOD98xw8L+i8+WWddMJ8Q8y8KhUiAXdgXNwFmPTh/KeDqIyzh8V+8diyBQQJcPPdHWFqaBJAAl+a6w+sSI+AqAVaJZVD7yHIB/tlYPYClEmD+qZ91reoBKn5Qi6UA/FAXJ3es42QNLZ928sNESrvKyTE/BMYaVi5HxTrlsEt6AswnzXySyfIAVQ6MdctcZu2TTz5xmgDzA1P+hJ7n4USR5QisZ+bXDLMOmB+c45Nh/0/oSh6iklibBFg9QBd82QmfMvNJN58887rxFcZNfUlgHrwefPFDdSz74H/jhJwvXhN+8I+rhijJQxYJcBr2xU2A+YsZ69J5XflLjPqywzHMp/B4CK7EbrBwtygJIAEuymWD0SBgR8BVAsw/s3NSxKeKnFD6y6DxyReX1uIkViXAXPGATwi51BefivFpMH/OD2ZxqTG+uOoBJ2881o9+9CMv2eUTR5ZScCLJNYZ1lzQBVok2z82JOp/AcjLMGmd+2t/lCTD/LM6VHy655BIvSeKkk/WqLPtgZnwxP/4iwJIRTnq59Bw/QMZ1mFnry+XQWKNtkwD/85//pOOOO8474VZl2fjEmZM8TtC4dJqqtBHGjUuj8WnvBRdc4Mkz+D9sN5/Ysw6c7eS3BHL1Cp6L15RrQ/OVRQKchn1xE2D2mWU7vMZc/o9jlEvbcSxzlQz+4sdfRHCBAAjklwAS4PyuDSwDAWcEXCXAbBAnsvxTO1dw4P/drVs3Yt0jayL5QSG+VALMJ8N8gstJINe85dqy/G+sG/VfnFTxv7NEgasncAk0Hk9X71b1lSbA3J6lBXyCzVUaWFPMp9BckYLr1bpMgDmpvv766z37OQniur1cRcL/4gi2hz9jWQYnk6wb5lNDPqXlBF1xtEmAeUxuzzpjTrDZR65Zy3Pzv/HpJNcFZr5h3NgGtpFPsLnyB5/c88UJPSe4/O8sf2E5BSfYXGtYnXxmkQCnYV+SBJjZcFUNfiCOT/d5H7CenRN1fhCSXyCCCwRAIL8EkADnd21gGQiAAAiAQA4J8MtcuOJG2MOZrJPnkoD8hQsXCIBAfgkgAc7v2sAyEAABEACBHBJQv3AEX/zBp+VcDpDrUPvL6OXQBZgEAiVPAAlwyYcAAIAACIAACNgS4OokrJFmLTRLdvgBR65vzRdLeUy1nm3nQ3sQAAG3BJAAu+WJ0UAABEAABEqAAD8Qyi8C4QcNOfnlt/uxppyrVahXf5cABrgIAkVLoN4nwFzj8/XXX/eehFYXfzvnp84nTpxI/JT6pZdeSldddVXN5/wUL7/P/uGHH/YeZuCyNvy0Mz+pjQsEQAAEQAAEQAAEQKC4CdTrBJhLLrEWi3+qUgkwl0PiV7/ye9z5M67DyeV9OMHlV5Tyxckv/38uk8RljK655hrvqWquhclPUOMCARAAARAAARAAARAoXgL1MgFetGgRXXjhhV5NUv4piuuQqgT417/+tVe/kR9iUMX1uXQN/4zFZWu4aD2/0YrfA8+vbOWLf+riNyJxyZvg26qKd+lhOQiAAAiAAAiAAAiUJoF6mQD/7W9/8wros/yBXw/Kya5KgE888UTvrVRPPfVUzYqzRILrky5btsxrywXquTh+r169atqwDIIL9avC76UZLvAaBEAABEAABEAABIqfQL1MgP3LwgXf/QkwJ7FcXH/48OE1zfhVn3369CF+Tev8+fPpBz/4gVfjURWj54ZcIJ7/7aWXXtKuOr8ek/+jLi4av2DBAq+ovO5VrsUfQvAABEAABEAABEAgjwT4rZRffvmlV6mkqqoqjyYWzKaSS4D5QbazzjrLOxlWF+t7+e1GLJngZJlfx7pjxw7vVaTq4rdG8asu+bRYd6m3IRVsNTExCIAACIAACIAACAQIjB8/ngYOHAguPgIllwDztyCWQYSdAPMrUjkZPu2000JPgPntPy+++KL4BJhPfw855BDiwOvYsSMCDwRAAARAAARAAAQyI7BkyRIaNGiQ9+r1Hj16ZDZvMUxUcgkwJ79c+mzEiBE16xOmAeZ3u/OpsLpYA9y3b1/vATrpxbUh+SE8ToS7dOki7YZ2IAACIAACIAACIJCYAPIQPcKSS4DvuOMOeuihh2jWrFlUUVHhkbn22mtp9OjRXhUIPuXdZZdd6J577qELLrjA+zxuFQgEXuK9iwFAAARAAARAAARiEkAeggS4pgrE8uXLvTrAJ598slffl+UJ/CpLToqHDBnikbr++uvpD3/4g1f2jH8yuPrqqz1t8LRp06zqACPwYu5YdAMBEAABEAABEEhMAHkIEuBab4Ljag/89rfJkyd72twrr7ySLrnkkhpK/AAc1wbmF2Fs2rTJe5EGSx9s9TMIvMR7FwOAAAiAAAiAAAjEJIA8pIQT4Jgx46QbAs8JRgwCAiAAAiAAAiAQgwDyECTAMcImeRcEXnKGGAEEQAAEQAAEQCAeAeQhSIDjRU7CXgi8hADRHQRAAARAAARAIDYB5CFIgGMHT5KOCLwk9NAXBEAABEAABEAgCQHkIUiAk8RP7L4IvNjo0BEEQAAEQAAEQCAhAeQhSIAThlC87gi8eNzQCwRAAARAAARAIDkB5CFIgJNHUYwREHgxoKELCIAACIAACICAEwLIQ5AAOwkk20EQeLbE0B4EQAAEQAAEQMAVAeQhSIBdxZLVOAg8K1xoDAIgAAIgAAIg4JAA8hAkwA7DST4UAk/OCi1BAARAAARAAATcEkAeggTYbUQJR0PgCUGhGQiAAAiAAAiAgHMCyEOQADsPKsmACDwJJbQBARAAARAAARBIgwDyECTAacSVcUwEnhERGoAACIAACIAACKREAHkIEuCUQit6WAReQbBjUhAAARAAARAAASJCHoIEuCAbAYFXEOyYFARAAARAAARAAAlwZAyUVVdXVyNK0iGABDgdrhgVBEAABEAABEDATAB5CE6AzVGSQgsEXgpQMSQIgAAIgAAIgICIAPIQJMCiQHHdCIHnmijGAwEQAAEQAAEQkBJAHoIEWBorTtsh8JzixGAgAAIgAAIgAAIWBJCHIAG2CBd3TRF47lhiJBAAARAAARAAATsCyEOQANtFjKPWCDxHIDEMCIAACIAACICANQHkIUiArYPGRQcEnguKGAMEQAAEQAAEQCAOAeQhSIDjxE3iPgi8xAgxAAiAAAiAAAiAQEwCyEOQAMcMnWTdEHjJ+KE3CIAACIAACIBAfALIQ5AAx4+eBD0ReAngoSsIgAAIgAAIgEAiAshDkAAnCqC4nRF4ccmhHwiAAAiAAAiAQFICyEOQACeNoVj9EXixsKETCIAACIAACICAAwLIQ5AAOwgj+yEQePbM0AMEQAAEQAAEQMANAeQhSIDdRJLlKAg8S2BoDgIgAAIgAAIg4IwA8hAkwM6CyWYgBJ4NLbQFARAAARAAARBwSQB5CBJgl/EkHguBJ0aFhiAAAiAAAiAAAo4JIA9BAuw4pGTDIfBknNAKBEAABEAABEDAPQHkIUiA3UeVYEQEngASmoAACIAACIAACKRCAHkIEuBUAss0KALPRAifgwAIgAAIgAAIpEUAeQgS4LRiK3JcBF5BsGNSEAABEAABEAABIkIeggS4IBsBgVcQ7JgUBEAABEAABEAACXBkDJRVV1dXI0rSIYAEOB2uGBUEQAAEQAAEQMBMAHkIToDNUZJCCwReClAxJAiAAAiAAAiAgIgA8hAkwKJAcd0IgeeaKMYDARAAARAAARCQEkAeggRYGitO2yHwnOLEYCAAAiAAAiAAAhYEkIcgAbYIF3dNEXjuWGIkEAABEAABEAABOwLIQ5AA20WMo9YIPEcgMQwIgAAIgAAIgIA1AeQhSICtg8ZFBwSeC4oYAwRAAARAAARAIA4B5CFIgOPETeI+CLzECDEACIAACIAACIBATALIQ5AAxwydZN0QeMn4oTcIgAAIgAAIgEB8AshDkADHj54EPRF4CeChKwiAAAiAAAiAQCICyEOQACcKoLidEXhxyaEfCIAACIAACIBAUgLIQ5AAJ42hWP0ReLGwoRMIgAAIgAAIgIADAshDkAA7CCP7IRB49szQAwRAAARAAARAwA0B5CFIgN1EkuUoCDxLYGgOAiAAAiAAAiDgjADyECTAzoLJZiAEng0ttAUBEAABEAABEHBJAHkIEmCX8SQeC4EnRoWGIAACIAACIAACjgkgD0EC7DikZMMh8GSc0AoEQAAEQAAEQMA9AeQhSIDdR5VgRASeABKagAAIgAAIgAAIpEIAeQgS4FQCyzQoAs9ECJ+DAAiAAAiAAAikRQB5CBLgtGIrclwEXkGwY1IQAAEQAAEQAAEiQh6CBLggGwGBVxDsmBQEQAAEQAAEQAAJcGQMlFVXV1cjStIhgAQ4Ha4YFQRAAARAAARAwEwAeQhOgM1RkkILBF4KUDEkCIAACIAACICAiADyECTAokBx3QiB55ooxgMBEAABEAABEJASQB6CBFgaK07bIfCc4sRgIAACIAACIAACFgSQhyABtggXd00ReO5YYiQQAAEQAAEQAAE7AshDkADbRYyj1gg8RyAxDAiAAAiAAAiAgDUB5CFIgK2DxkUHBJ4LihgDBEAABEAABEAgDgHkIUiA48RN4j4IvMQIMQAIgAAIgAAIgEBMAshDkADHDJ1k3RB4yfihNwiAAAiAAAiAQHwCyEOQAMePngQ9EXgJ4KErCIAACIAACIBAIgLIQ5AAJwqguJ0ReHHJoR8IgAAIgAAIgEBSAshDkAAnjaFY/RF4sbChEwiAAAiAAAiAgAMCyEOQADsII/shEHj2zNADBEAABEAABEDADQHkIUiA6xDYtm0b3XzzzTRixAhatWoV9evXj4YPH06HHHKI13bKlCl02WWX0cSJE6lt27Z06aWX0lVXXWUVkQg8K1xoDAIgAAIgAAIg4JAA8hAkwHUI3HTTTfTHP/6RHn/8cdptt9285PfZZ5+lTz/9lCorK2mvvfaiwYMH0xVXXEFjx46loUOH0gMPPEDnn3++ODQReGJUaAgCIAACIAACIOCYAPIQJMB1CPCJ7zHHHEN3332399natWupZcuW9Pzzz9PMmTPp97//Pc2dO5caNGjgfX7dddfRqFGj6LPPPhOHJwJPjAoNQQAEQAAEQAAEHBNAHoIEuA4BTn45MF599VXq2rUr/fa3v6UbbriBPvnkE/rZz35GrVq1oqeeeqqm3+uvv07HHnssLVu2jHbZZRdRiOY98NZv3k6/eHE6zf1qA/Vs25RuO2Uf2rmT6NqRH9N7s1bQzp07qVlVQyqjMmpSWUFnDupK5WVl9OZnX1IZER3buwMN3r8zjZm6mOas2EBLV22iT5eto01bthGVcYsyqmpYRpUNK2jRqk20dUc1dW7VmP77wK60s5ro6bHzaPHaLdSoooy6t2tKA3u0oT3aN6MfHNCFGlaURzLeun0njZy0kOav3Ejd2zShk/p2olGTF9I/pi2haQvW0JbtO6i8vJzaNm1IbZo1ok1bdtCqjduobbNK+tHB3em/B3WrNQeP9+T4ufTUh/Ppqw1bvXanD+xC0xet9eZQfJo0+voLkX/+zq2qaPvOanptxjKauXSd97/baeYJcypsLMX46L3a07ad1fTMuP/YFWa/KSB5jmcmzqdXpi+tZSP7+PGCNTRp/kpas3E7VVSUUWWDCvqvXZvT8X12pdMHdvU4KT5PfjCPFq3Z7E3XrLIBtWjckKi6mtZt2UGVDcrpgO6t6Ven7ksNysvrrI+Kk+VrNtOKDVto+dottGvLKjouMA+vK8fT4q820ti5X9HKDdu8cOL5erRvSv27taY1G7dRy8YNafzcr2j2lxto245qat2kIR24Wxtq36yKpixYTTuqq704bVpZQSvWbfXmOvK/2tPHC9d4MU/VRM2rvl7P1Zu20oKvNtL6zTv4n6m8vMwbb2CP1rR+yw6vTZumjahDiyrq2qaxF9ts46R5q7z2Pdo0oX27tqR/fbacPl+23mOxf9dWdECP1rR07ZZaMfr6jOW19s+zH82nv45dQJu37aD+3/BTcWZaV//nwT0Rto/867hw9SZvHzZpVEGd2zSmzb49wnu0UUUFLVq9ybNdsidtbEXb5ATU/Xv2ivVUTmVerPH9U90LVZwd3qsdTZm/miYvWEVbtu309izfxzu2rKJj9t7Fi99XP1nq3Tf5XsP76oLDenj3hfdnrfDismFFBTWvqvDilvfD1h07qXlVQ9qzQzNnEJIyAAAgAElEQVRq26QRrdy4lZbxfaGsjNo2a+jtpS/Xb6Xq6mpq1aQR9dq1ObWuakifLl3rxRTHXeOG5TV/X/jvRONGDWj1xm3UuFHdvzV8H6woL68Vj9XV5N3TlJ/BNnxf+eVLM+iLL9fR6g3bqEXjBrR203ZvH+/evpn39y64z3R7KOzfmWXw76d/PHXP/ecnyzw26l534r4dPbv8f3fj7HdpBOU9D5H6kUa7smqO0BK8pk+fTmeccQbNmDGDKioqvGSJT39PPvlk2m+//eiEE07wZBHq4nZ9+vShCRMm0IABA0TE8hx4fPM8+I43aN3m7TW+NGtUzrkMbdi2U+QfN+IEwj+GuGNEw4N2a0sjLhikTYL5xnLuI+No7JyV/7G9sqImUZHYMKhHG3ryfw6sSe7O+tOHNHHe6siuLaoa0NjrjvGSu+D8uo7+eXTJr3Qsf3/TuMHE6JyHx9L4uaskaGq1ObBnG3r0vIE05NFxRj6qY7PKCtq7Ywua4JvPFCdqngv+PKHWulobnOMOzEUl08rMpo3KacPW2vuNWY277pg6f5yjXAvbE8F9xG0kcR42j2lP5hh7vTQt7P7t33/BOKtvEAb1bEM7q6tpYsQ9raKMaEdEdqPu5/5DjeC9mOP+kSEDKHhfGtCjNf176bpaf/v84/Fe091zg3YF7XC9VnnOQ1z7ajteySbAzz33HN1333109dVXU+fOnelPf/oTPfPMM/TOO+/QqaeeSmeddRb98pe/rOE5e/Zs2n333endd9+lww47LJQzyyj4P+pasmQJDRo0iBYsWEBdunSxXZtU2//8mSk0evKiVOdIMvgdp+5LZw7qFjrEX8fPp2tHTUsyvNdXzWEz3qn7d6aBPdtYzZ+WL1Hj+uHY+BcGlX0elUGsZDVP4sDJYABmcc8Z/cQz6dbYHyNJ40Aab2Kj0TA2gbzfv2M7lnFH/z7T7Q+b+5Iaz3av2e53G0xIgPW0SjIBnjdvHu255570xhtv0OGHH15Dh/93u3bt6IsvvqATTzwx9AR40qRJtP/++4cS5aoSt9xyS53P8pgAn/LA+zR5fvSJp80mc9324qN2p2HH7xU67PBXPqMH35qVeEo1h814+3drRXwqYDN/Wr5EjeuHY+NfGFT2OYtYyWqexIGTwQDMYvTQQ8Uz6dbYHyNJ40Aab2Kj0TA2gbzfv2M7lnFH/z7T7Q+b+5Iaz3av2e53G0xIgJEA1yLAp7+nn346bdq0iaqqqmo+49Pgf/zjH9S9e3ev9BmXSFOXRAOME2CbbRnd9qhe7en4fXYN1R7afrvWzRTnBJh1nayHvOGF6WJneZ5T+3eppYlVmsokvij7TdrPJHOwkzYnIGIoIQ2Z7eQF+f1SlsQ32762J0JRp1cdWlZ5Ol7WptvEbdBmnADbrmJ67XEC7IYtToDdcCzWUUryBJjLmh188ME0btw4T6KgruOPP95LiA866CB66KGHaNasWZ4+mK9rr72WRo8eXW+qQLCG7KBfv15Hk8gPOeRJFB6mPUxDA3zmHz+gSfPXiPYx68/4Gu/TIOs6slb3sfMH1tGQKb9Yc51EAxzWP0z7GVcDzPb/4UcH0GHD36ijVdX5zI8vBlXkJg1w88oGtG7Lf/ToooWoB43C9psrDXCQOcft9p07aZJB6x6GNW2dYj1YykxdiNIAh+2/TI3LyWQmDmlrgHV6e2iAcxIg/ChzKT4Ex9UNjjrqKPryyy+92r6sz/3LX/5Cv/71r+m9997z6gJzHWB+IO6aa66h8ePH08UXX+wlxUOGDBGvXt5/enj8gzl005gZdfzp27kFzV+1yasCwU/68pO9/KQ4V06oKCujNz5bTktWb6KZy9drWbRq3MB7wreqwdd956/8TxWIsw7s5mXZT46bS4vXfF0Foke7pt5T/u998VWdMcNOnrbt2EkjP1pI876pAnFyv040etIiemnaYu9p5k3bdmgfgPh+3070m9P71nrI7omx80JPx7q2bkwLVm2qY9Ptg/ehivIyenn6Unp75pd1Pt+leSVd8q09PGbPf7QwVDOs/PL70qVVY9qxs9pjzMnRt/baxfv/T42fR1+t/7o6hb8KhET7ycbxHM9OWED/mL6EPl+6znvamytVnDGwK328cDW9M/NLWr2pbgLKfnJFBKnmWsfrlyf3oUYNyunv05bQu5+vEO2h/To1p8VrNntVIPhLWdgXs04tGtGqTdtrqkAc9E0VCD5J5odkmGGTqga0Yt0W2rVFFbWoakgvTVtSZ/7OLStp/ZbtNVUguCM/qR68jtiznVe1gi+uAvGRqgLRtgn17dqKXv90GX2xbL3nK/+sydeYqXXn69WhGfXu2IJemLK4zhzM6txDeogY+Rv544ifOg/TbfN6cjz95cM5tJD3+DdVILq1aex9wVm1YSs1qCin5eu2iPahtZHo4IzAxq3b6ew/jQv91WRwv05e1RuOf76vhsX8nrs0ox8d1N2z5+VPvq6eo6pAHNC9Fb08fVmkrfyldb+uLalt00a0csNWWraWq8PUrgLBsRb1IFpwAv5bsXHr11VXgpfyKex+G9b+e/t2DPWb77H8ZTCsCkTw74r6pS7s3/nfbhg9neb4qij5qzno/qbc9L3eNG3RGm0/ZwHyzUB5z0Nc+2szXkkmwAyI3/7GZc9eeukl73/vu+++9Ktf/YqOPPJIjx9Xe+C3v02ePJk6duxIV155JV1yySU2bL0ya1xiLY8aYHZEohvUOWzSOMXRCyaxJ2hnlH1httnqv0z6YYn2Mg4jqZ+2Y0ex5zmlmmedXs7EKyzO0mAojTFpO9MNIQ5X27ULsyGJ/Un6mnjgc7cEJGslaSO9r/jbSeLU9HciOG+U3pbnc3kvcrsSdUeLwz0Nm/Keh6Ths3TMkk2ApYCStMt74ElPD8MYmHSlURpeHdMk9vCYSgvLJ3MT5qzU6knDTpR1c+t0qSb9sOTpexeaShtmqi6lvw6tqvOrG8f21EWnFzbxCosJKUOdvtomboNrYcM16h4RNQ4fMIedrGcZF2H6cdMvFknuiejrloAkTkd8OM+rVxu8+NeAsw/qXqumuar5rIsB/xhhcRqMJ1vd+ff7daQXp9T9xYTn5fm43neYL2FUdfduvke1a17p/SrENYH59Lp980rq2a6p03rXkrVxGw3ho+U9D8mCgW4OJMAp0s974Elqh+rwhPUNa2tTPzQLe7je7BMXfl3/139F+RPUUvp9ktgsaRM3DKVj6+pSKh6sJbbRCYfpoHU1M028dGP5a0Hr/Ayr0RkVcza8wmqCRtWnDlvDqPkk+u0048IV07g2ol9yAqZ4jqpHq6u9rdvHfmvDNOFhtoTtba6h+9mStXWePxnQvTUrKEJr+ypbz39sfJ165vyymrKyslrPZOieOTDVBrb5e2VaPdPamPq7+jzveYgrP+OMgwQ4DjVhn2IIPJ3mSeKi6jt7xQbv2/SK9VvonRB9p82JVlx7TCfSrLkccnCPmjebhfnHcw97/uNQ7SSfGqin6YNvxZLYLGkjYa6z26+HDntrVxQftT46zZp/Tv/JPv972LwmX8M+143lnzusX5zTSpN9ak5pO9O6RY3jao44cRF1QsUxZIopk9/4PBsCUTFkui9G/WKjYkD3nIP0VxP1vIR6XkN3Ksy/NoVp4vnf7/phX+2zFDw+/5Kl4lWnf5euhs3fK9OYae5v09zq82LIQ6S+uG6HBNg1Ud94pRZ4hdQ8mbRmEr0aL10hfUgxFLV+8Zw2+lwpxzR98Y9dX9crC35glwXlws5hui+aNPs290RpPKX5vEWUvdKVyNs9Tmq3rl2p5SE2vJAA29CybFsfAy+OjtTlN2rdEphOOqQ2uNZtmWr0+v3h0kZR75a3DL9azSUnwCaGPKCUo42tYYxYHjBy0kKa/02Vj7BTbZ5DZzNXa+AqF6zt69qmsVcD983PvvSeij+2d4fIXwJ4XJt1y6K9DU9pW9exLp0X7dInoOL3H4aqK712aRZazcf/S4/uVxY+PR5+2n41cjJdPAWfB7F945q655i0zIqq5D4WtQLBe5z6m/fPT5YRny7v2rLKqwSjnp8IGyv4d/Lovdp7VWxs7kGuoqQ+5iGu2CABdkUyZJz6FngSHakL3WScJYnS8NroulzqtmzGCqvr6bL2qmntWBNt0nXbcJSuoVQ3qJvbZLPODp0WXCWzNnFss85xxpeytG1na7ft+GhfGAJx90SYtSY9sH9fRtUm5rGjap+bnh+IekYhuJdtnucI+mxTQz3qeRLJ8xRR9yCXkVPf8hCXbJAAu6QZGKu+BZ7kFLGQmqegJjnuk72ufLA5YdO92cn2jWBR4axqAf9zxjLtSWiwJjGPt3D1Ju9NYrpT2CRbyOa0Rnf6rGzWaRV19unGs1k3Hjvt9kn4mvq6inXTPPg8OwKmPcX1fz8PqeGuOw3mfcJ7X/d8hE2Fl7Da5/57iy4eTT4F97LubwHXix8zZTGp51ZMVSBs5426H4RFQBq/qAXnqW95iMudhATYJc16ngDb1tZNEW1RDC3VxLEzpzzwPk2eX/c1wGm+Iz4PEE0aRb+NJm2ezVg8rm48m3XjcdJun4d1gg3FQ8C0D3S6X5Me2BTnpnmj9pyJrmls073BNL7u8zjzmvrY3NPi2u3vhwRYTxEJsIsI04xRnwKPf1YaNvJjGj15Uai3cb/JBrWWJ/XtRGOmLjZqP10vm40dqi3XG+bqF7qTZsnJoBrrsffmhGrxkpwA2+pYXTPl8UysTCcsfptUjOk0w1HxaXP6Ylo3v76vurqa1m7eRlMW1H2NtqsT5jTWxT9mHuIkbR9LaXzTntJVftD9O1dh6Niqsad/DXu74OC+nWjVpm3Gt4PyGtj8nfDH5eLVG7X1gXnc2wb38TS2YTXOJbGu2wMmlmH+6LTKNvcgl/Fan/IQl1x4LCTAron6xqsvgWfSlMXVMoWNG1VzN62lsrEjrHarsitMPxalJTVxTaIBzoO+U6LLDuMZVRdY2t4UK3E1wNJ6yS5qEZt8cPF5HuLEhR8Y4z8ETPvuwbP70xF3vUnrNv/n1ed8r3njyiPp6N+8Vac+r59t8P7crLIisr2/r80zBKZ7o39cvl/wa88nzl1VKwwk2mCJ9visP31IE+fV/XVuUI829OT/1K4pr3vWIiw+4/7dtI31+pKH2PotaY8EWEIpZpv6Eni6b8GS2rpR6EzfrlVfm1ODOEtlY4fu7V06W+PU6OzWujEN6BH+rnqpf6ZTTOk4SdqZuEbpAXnesDq0pjH99vKboPiPY7c2TWjHzmp647Pl4ioQcbWIPH/w6fgwhnnR3uYhTpLEGPqGE4h6HiKqqkPYCW9wBlUTPareLld+4GorfMV5hkC6z9mW/t1b0w0v1H3THc8teZuk6e2Vuvro6k16fj46u0/u25EGdG9jdQ9yFdv1JQ9xxcM/DhLgNKh+M2Z9CTyT9isuQqlWKi19l7Lbxg7u8+Bbs7Qu29iaFlc2Ls2xpett4mrDynatuH2c8U2+mXxKa16TXXE/z0OcxLUd/eIRsK3DG5xFUjc86d6T7DO116LuyX47bP02+RnmYx73U33JQ+JFe3QvJMBpUC3yBDioh9K9uSfqZFbpJKNqJ0q/5Yd90/Yvm1TD6G/XuVUVK4A8rfGEOStp8oK6P3EFQ4P9tT0BDo7h18Tq5nVx4p30ZE/KVOcfczXp9tjPU/t3qVXv16QBl8YM27VL80riXym4HnCHFlVePWBe80UWVS2ke8HPIWz9dLplU61jv95YWsPY5pam42naczZzoG16BKL2qa2uVXcSGnYfPHNQN231E26v4sf0DICOjHSfm+7JSr/M9/sJc1fRi1MW15mS7xPL122p8++m6hZhbwfV2c0a5Yrycvpi+XqaNG+V93dkt3ZN6bZT9qEmjRqkFyB8Ar9wIXXt2pUWLFhAXbp0SXWuYhscCXCKK1aMgRemvYrSZHL92LAkSFcH0a97CpsrTFMWV6/pt81GU8b+6LTINhrgMC5BTXCwjY1OLip0k2g74/a1YazT3pk04Lr4DNMAmra2iXWcucLGjLundHpCl9rBLOYwrQM+j0cgap+G3ackdXjPf2w8jQ9oaf3W+eOb54/Sxz52/kC64M8TaOyclXUclOw9Uy1dtQ+kunwbykE/g/dtm78PYX8/lS1JnvOQ+lOMeYjUt6TtkAAnJRjRvxgDL+pEqKK8jNT73KNqwpq+vftPyIJayC3bd9BNY2Zov40HP5CedJps8o/L3+z5mznXiwzzN269YZMNEu2oTbjG1ZlKmUrXwt+O39B24r4dvbqiOi1icFxdjU//2jwzYYFWBxjFLOq0PWov8JhcT5mrQHAtUT5h7tmuaWitZNO6++2TaBa5vYtfCdS8Oo2jyzls4hZtZQSi9qnulypJHd5nJyzwYpt/cfjWXrsQ3/d1Ol5d7LAHphNlU3zpxg579oTvdVc/N5VeCDnhldH8ulXwrXWqr/9eqtM+69jqfkFVYyep9CPxrRjzEIlfLtogAXZBUTNGMQaeCw2TSb8VpQ+znV/a3mSTfwmT6td0IWWyIa15bUNcyjQ4rsk/bi/R5AXHlXCRzB3GwWUsxl13XexF+SRhIl33uOstHR/t0iEQtW48Y9izCi7jhueIilFdXWFFw2SLbVzGvQfY3vtd25V2rfdizEPS2TF1R0UCnCLpYgg8qcYxqHfin510+kXTiVecU7ekNVVNNvnDwEb/qNPZhWk3+RTh5r/VPd1Wc4fNa6vFDbYP09RGrR3bomOlOx1R9ksYq5qhfLrUqklD0YlNmFY4+AuEZO6wrc5vv+rUqrH3xPrpA7uSXzbjSh8bty5olE/SGJXEj24efmPYeYf0qMMlxVsmhrYgoFs3vld/tWErvT3zyzqjmU5dLab3mkbFqPQEOOzZDNbpm05ala2q/yvTl4b6bONTmK432D+K+/DT9qt1DzEx4s+5Ws2zFx1cp5+N3VFtiyEPceWr7ThIgG2JWbTPe+BJtYlBvZNJE6zTFTI6k37RVn8qbW+jTzXZ6L/xhtX5fWTIAArT0plqZkrrVo64YFDozTLMR9u1Y99MrHT6PVO/MP+D9oVp65hnUEsYVnPZpBk0bd0g//Wbt9NBv369Tp3TsPqfurF1e2Fgj9ZUVlZG4336SBufJDFqszei2EnmMrHF5+4JSPa7f1aT7jaOhVEa8kfPM2uAo56pYHviPBMg8UNyL4pzn5M+AxC0MY21UXPkPQ+RrFdabZAAp0W2CJ6+lOh9o2o9BtEFtb2sJXvlk6W0bO1m2rVFFR3XZ1fRaZKtdlXa3t+uSyuuCECefe9+vqJOFEhOSqJOAiT1NMNCT6IBtT0Nl4S4Tmf7suZURWdDGGPWD0bFkf/U5eR+nepor3Va4TCbVcwtXbPJq/jQoUVlTRUIrgfMF+sbw07H+DMJ/2C7KL5Re4xPnMNqHfvHi9I2mmLURssdpeW08VcSa2jjjoBEm2r65SapNWyDXzfs/zXF9LyE5JebqFPZqF+r/HWI1f1eaZn5PjN60qIarbPu16io+9yw5z8OfTNeWB/FgatAvPrJUlqwiu9PtS/Tfo67TkiA9eSQAMeNKkG/vAeeRMtko6syaboEyDJvImGgM8q2rqTEOYlGVsfZZq2CttiOabvWSfSsSdYojLnUFmm7qHV1YXvcMWz6mWLHdr0lsY42bgnYrLfbmeOPZoo7HrmQOv005s56nfKeh8SPnuQ9kQAnZ6gdIe+BJzkhknxDVwCkusS4yCV6RtuxdfpM07dxtmXYyI9p9ORFdabk2pNxn0aWnEAGa2xyzd3ubZqQ6WnjKDa2p8qqvibPq/S4UesTFUcm1rq+rJ3jV5FyHU1TbPg/jzqN9sdwlM1R2mE/Z5v40vkg2adha2uj5Tbtc9Ma2e47tHdPQKpZV3GWtCatac9JPJTo4+M8MyLdn8rGOHvMZn/FvSdIGJra5D0PMdmf5udIgFOkm/fAk2gEdTphybvXXaKV2Go7X5R+7YkLa7/j3T+2SevK+k4uQxR8N71E7+rX99rq68K02XE0wEFfTdpaXU3fYC3NsHEk+tIo3lxH851rjqaLRnxUq96oqY6nTo8drFNt8p1Z6Xywia+o+I6q6RpWh1utnylOJesT5Z/tfkP7dAlINOumvTT2umNEL2ZwcT+OelZEkTJpY00xLo3fOP6Y5tbpgePeB+NGT97zkLh+ueiHBNgFRc0YxRB4Ev1sWBtd3dW0TorifEM3La30xCQ4jum0jNvzSSJfqp4m69FO6d+5lr41TO8aTGh02kzdE9Y8r79ec9gcbJdJe+r32aQP5bY6e8J04X4mwcoLujXjGDz9oQ9D39jHJ8Fhb/JTc+vWq1/XljRlwZo6U+psXrJ6E81cvj7UxLC4t4kvU3xL9mmYYaqfRMuttJxxdPumvYbP0ycg+ZXFdO+S1qQ1xavEW90YYb8uRY1ninHuK/m7FGePmeYOzmtzT5AwlLQphjxE4kcabZAAp0H1mzHrc+BlrWNKY764YybVrdmEnK3OOA2tpsRfXc1Pl/ac8sD7NHl+3VdW84Nuy9bWfZWpmtsVQ1tNsE182bS1iR/VNu3x49iEPm4JSOLTtJelNWldxJOLMfwEJf67Jf6f0aS+SNu5tLM+5yFJOSEBTkowon8xB55J3+XiBMCPLmw+f71anW7TX1c2rJZrcHkkWlDTaYHpFIXndKGHjtIZs8Yt7DRS2W5av6iwD/bdvG1HZP1iHktyAhw2p5przooNtHzNZlq5cSuVl5WF1uX9+TNTQjXXuhPg2wb3oYryctLVBzUxDNorOWHjPson3bzKLqXdZg310+MX0C9enF4HkaQGsuoUR4Pt+u2DKd5OMbSBgCQ+TZrbPJwAB++/kntZ1L2SsZnu6ZI5ovZZ1P49c1C3mpVz/XdTsimKOQ+R+JekDRLgJPQMfYs18CR6KEkbKVqdzpj7++ukVpQR7WBx7TdXORHtDEwSpSkNm8dUZ1KXtAXr/wbbSbStpiTUNEewv9KcxdWMquQtOK+pfrFEAxyHY5DhqvVbacDtr9WKAY6JcdcfQ5c8ObmWBjhMDy2JxyjNoUTTa9IFhtnF/6bT1IfVUpXWGjXpoBUPk85Swg1tCk/AFJ98X4jStLOePmsNcFgd9eBzEHHa+FfDdC+2+Vum+1u1fedOmjSv9q9T/LdlXEBTbTOXq4gq1jzElf9R4yABTpFysQae9FtqHM1UGG7JiarNMtlWNJC8/Sc4v9931oaGVX0wnTpE+WTDJFjnU7p+SdYiTKdnGw8SHyVVMbgNn6L6dc26ihi6U19prdSomqfM0/RkuE2lDv4Voby8jK4dNa3OUkm1hUE9s03tUps9h7b5IBAVn7rY7Nq6MQ3s0YZuO2Uf0QNwylPb/R5GyDSG5F6ma9OrQzMacrD5TYaSOZTtknuW38+omsDzvqneE3yrpetIKtY8xDWHsPGQAKdIuVgDL2udkkmXZrtEade0DdqTBi8bJkF/k9gjndeFtlcyV9y6yK50v7axZ2Iv8VnNyb7z9eBbs+qYEXfNTfbZ+ov2xUOgGNdeYrOkTdQq2fS32b88p4v7ZNIIK9Y8JKnfkv5IgCWUYrbJc+DF0QuGfZu10U7pMNp+qzYth07XaDqd89ezfWbifHp9xnKqrq6mNk0bUYcWVdSzXdOamrd+G6JOVto2q6Td2jX1TlcalJdT2Lhd2/Bb6cpo0epN1LlVlfe/X4t4W1nQ/0KcANucbvt1vivWbaH2zSs9lpLTUOkJsF9nx3x0axJXqxwVcza6cptYZ99Z8ZPkBPiIPdt5b8JTzHV6bpv1NO0/fJ4NAdt7r81JpzTe/XXA43qd9G9R1F7v0LLKq5Eedcqq00XzveLWwfvQmKmLKW6t9TzsqzznIXFjxlU/JMCuSIaMk9fAM+mQTJ8rV6XtTIhNuknVP6jXjdKmSnWSftuUlvX8x8bT+LmrQs2OMy4P1Lyygnrt2pw+CujETGz8nw/o0Zr+vXQdrdu8PdI21xrgODrpqBhRn+m0r/719mvobOJN1/aRIQPogj9P0NYMtlkPbmurK5fq3W313NI9FLZnTBpJWyZonz4Bm73g8n4dZ15TMh2l8ZXMZ7sH/fbodNO6vzk6Db9UA5x+ZNSdIa95SCFYBOdEApziKuQ18CQnASZtFmOTjCPFa6qnyN/G+RR1zJTFpLRT6n3uj38wN7Iigt8GyTyjQt7u5h9Dp+vS6SulDHTt1AmvzYmpZP108wX7SuoV68YynXia3poXZG3jl66tzRimtYtz+hQ2P8+jq80stdcU2zpfXFQsMXHC524JxL33SmPJdj/HPemU+CGx2d9GVzHIplZ31GoFa63z6XLWdfFtoimveYiND2m1RQKcFlkiymvg2WieovC4Gsc/R5wxXfbR1bP122irMU4aYqZ6thLbktoQt79JM2finQcNXdZ7IC5r1c/EPDh+3hkn5VEf+8e557ng4Hpe1+OxjzZj2u4VHj9sv9jM6WIdbMbIax5i40NabZEAp0U2xwmw5KlZfw1enYZK8u1d4ZXq1SRjBsfSnYz6v/FL++g0ov4w8WtuJbWKk4aYOqEznabyPKz5PHHfjpGat6T2+Pub1tVks4l3IerURvkkjaOwWr+SPeVibUzMg3PEqYLiws5iHMMU71n5pFvjtE/zXc9rut/b7EWl8zWN6V8j273CfYO1ufm5jQlzV9GLUxbXWf64J+Mu4wgJsJ4mEmCXkRYYK6+BZ9ILDuzRmsrKymrV4JVqX5O0Y3wmzZethpJfLWzThzWiURpg/xKH6cGCelnVPqxmsTT0lEbTVMfTP14WtV1Na6VbT2Wnrn5wkEsWvvi/qOk0iWHa6jBdNuu99+rYwvujqK6wWEnLL9P+9vNNou+Wxm99aSeJ96x81WlX09Zzu543imnUswxxPwu+at60V8L2R9hzBGHrntb+to2xvDiZDVkAACAASURBVOYhtn6k0R4JcBpUvxkzz4HHmilbzWrcmoY238gZXZTmK+oEoqK8rEYf7H/q17aPv5amqgKxcsNWeufzFaJo4RM1rhzx0bxV3lP8ZUQ0eUHdV/j269qSDuzZlrq1aeKNu3D1JjLVFH5i7Dy64YW6bwwLMyzt0wfpuqr1nL1iA/mrQKg18n8+Yc7KUFZp+6L4Rfmkq8ggCgpNo7T8Ukz/Pm0JvRsSt/wrBsdomN49LZuScMpDX2m8Z2Wr7l6Q9vq5nld3v4+zF5XvEt2wWid/2y6tuCLP1/di/uUz7PmH5z9aGFqZxb/uhfjlShd3ec5DstorunmQAKe4AnkPPFv9U1ytoEt9VJyx4vQJhoUNq7g1WnlOk61J7HAd6iZb48yXxpg2dkTNz+OE1eS1GT/YNu6eks4Zx5+0bZLanrd2hY5N6T0p7fXLikNeY1dyD057DWz2Rt7zEBtfXLdFAuyaqG+8vAeerf4pWFPUf8oapdUynZyovl8sX0+Tvjk1VbVzmzRq4EkYVP1cPiGduXx9nVWLOvWI86R+cAIbVkr/OeebE88V67eEnh77daKm+r+q7SvTl9LbM78URa3pJCipntG0riIjA42kYya1XWdbnFOnOH6qPqY1Mo1t4hDHn6Q2mWwu1s+lsZmVf5JnOYI/+SvbTHET5YNu3jDdu25+3fh+u6KqOUjrY7tcC2XbPzS/qvjnytMeynse4nKNbMdCAmxLzKJ93gNPp+cK0wCHuS2tVWqr81Jz8bvp37nmaPrfv0zU1uXltiatVZI6kf4/GEFtqK4mJPcZP2dlZKSE9dV1sGmrxojDxNQnaF8amkjJmJI2Ftu0VlPbWNXVMg5qB9PQAEs42PpjGwNxORdjPwnvLP0y6Vd1euCkftg8UzHigkEkTYJt7tNJap3HWSMTa/+YedtDec9D4qyHqz5IgF2RDBmnGAJP9+54dofrkr5sOHGUvq3KVuelcO7ftVWoJpQ/D74BLWop49aJ9I8pqd9qqtVrU9PX1LbXLs1CT8Ml+jNXp1k2WjvpVjON6cp2nT1R84d9pqsBGqyuoPaUqmMd9XYqCSspB1t/pAmLxMb61sYUm1n7a3qWw6b2rc2pZZCDpBKPiY3tL3VZroXpF0CuZ96xVWPjW+dMDNL4vBjykDT8loyJBFhCKWab+hB4Jr0Ta534CtNGSnRQpvE7tKikZWu3hK6AZPywjmlq2Ez+xKnpqxtTV0NXwiVNBjG3i7hb3mwvlD2Fmle8UGiYCYGoe05WNWtdxKKLMdICLr2vpzV/knHrQx6SxP+ovrlJgHfu3El//etf6b333qOtW7cSP33vvx599NG0GKQ2brEEnl9jyxULju3dgU4f2NX76Ur3nnQFTXoCzO3DdGemJ2qjToBtTiz8iyw9OTMFRhx/VA3JYSM/ptHCN85FnYzEfYrfxCCORjBOHxPjsM9NtkvHtLFX15b/XbeWN5+8N01dsJbmfrWBerZt6r3JkDXtusvGHh7DFYegPbZ22LaXrk8h2vl9Ubr8Rd9UBDipbycaM3UxzV+50dlJnwt2UaeTg/t2olWbtnmVaNR9XXfPTaLhdRGLccZwwS8szoLjmn7ZU3+LVD/1/AdXWuEKQu2bV1LPdk0zq8/u96lY8pBC7PfcJMBXXHEF3XfffdS3b19q2bJlHRZvvvlmIfgkmrMYAk+nA2b92KPnDYysiSvVAKvkN6y+alRNxSgNcJJ6l0k1cHH9kda95fH9OjKdvWHspPoz19pQF0ylm83FXDZj2PBXPoTVB+Z4HnvdMaFJsI09ao44fUyMbce0bW+av5Cfm3Sermsmu2Knu4c3q6yg9Vt21EKq7usX/HkCjfU9p5BUo+7CF9sxbNtLYytsXJ3Wn8f012kP/o0Lzim9P0ttlbQrhjxE4kcabXKTALdv355uueUWGjp0aBp+FmTMYgi8qNMD3Vu6wt42ZtJjRX27Zy0k6425CoSqneuvAqHTKSfRKprsNQWMxJ+wure60xfWkA3s0aam/mRQH6qzN4kftrrsOJU24p7Sm/gn8dv29NT2BJ73zY7q6tA3Q/Fn95zRr457cU6/eJCkHIKG2Nph2960roX83KTzDLMtSXy7ZBe8R7Zq0pBeCHkzGfvANqt7rtKju9DwuohFmzFc8vOvrW5cvkeHMbV5U6fif+agbpmFejHkIZnBCEyUmwS4efPmNGXKFNp99681pfXhKobAi9I2JdGYBtcvz/quOLEW15+4/eLYGLdPHBvj9Ilrn4t+NvbG0WCPnf0VTZ5f9+UnvKdGDz20jgs29rjwXzeGrR227dO0PenYJp1n2PgSvb0r1jb+5UEXbGNvnLZpxV6c/T7s+L20NdyDviWJmTiciiEPieOXiz65SYBPO+00Ovroo+mnP/2pC79yMUYxBF6cE+A4px5pfVsv1ELH9Sduvyz9jGNjnD5Z+hScy8Ze2xNg3h/j5qwM1Xi7PgF2zdCGC89t2961vS7HK+YTYGl8c7u0KkO4XAvJWGnFXpz9zie60viJ8/dTwkPXphjykCT+JembmwT4zjvvpJtvvpmOP/542nvvvamysrKWXzfeeGMSPwvSN8+B5xfr//OTpTT3q421GOm0Yp1aVdFPj9qj5iE5KVhbvZbp4QbT5367TG1Nn4eNxQ85vDZjGfF/q0ui7wrj4GfKz36OnLTQ+kEbGx9Ma2a7VjxenD4mO1x+HuTDDzQFdZC69QvzjSU6o4YeQheN+KiWlpIfdPlO7w7UsVUl3f3Pz2nd5u01bigNcIPy8jprHFbXlHWHJ/ftROohrKAsxuWaKyNt19G2vcs1DRsrCZOvtbTjaPzc8BreQU0tr8+TFx4ornMbtDcOO6l/Ol2wy9rAUlukay4dz/+3S3oP1o0d9u9btu2kk+5/r869PeyZC7VH+cHI5Ws207i5K2nJms1alyV/I6S8pO3ynIdIfUirXW4S4J49e2p9LCsro9mzZ6fFILVx8xp4YTfenm2bUNc2Tai8rKxWFQjWZD0zYT498OYsWuzb2HE2slTfZfrDYPo8mLCGPXynCrQnHYsTIX662uYJXx3TuA+i2PggDXbpWvnHi9NHak+Sdjo+j543gMZMWUySurwbtmyn7/2u7h9FNQbrvYN/jPlBuC6tGnvjqyoQnPzq4pF9ZC08t+cKBGOmLqn1QhXJg5E2Lx7QMbVdR9v2SdYyqm/SfRCWNPZo14SO670rdW3T2NN0T5i7qsaEJA/iqkFs2Nn6Z/vsRJq2mNZc6pvuy2jUPVg3ti6hZVv9LzLie/xLlx7mPbzqZxS2R5WfHVtW0oE921K7ZpWoAmFa/AJ+npsEuIAMUps6rwmw7U9Htu2TAjXNZ/rcP7+prelzm7Fs/Jb+XMZjmn4ys/HBxsb60tYFH9MYps8VS1ftpOPUlzWU+JGUSVT/Qrx6N+hzUv8kDKVtXNsiHU/aTnLf1j3kHcbARjai+pvu21LWSdvlNQ9J6peL/rlLgF999VX6+OOPqWHDhtSnTx/61re+RRUVFS58zXyMvAae7cMDtu2TgjbNZ/rcP7+prelzm7Fs/LZ54Mb00ISNDzY21pe2LviYxjB9rli6aicdp76socSPpEyi+vP8cV/2I7Fd0iapf5I5pG1c2yIdT9pOct/WPeQdxsDmhSKqv+m+LWWdtF1e85Ckfrnon5sEePXq1fSd73yHJk6cSK1atSJ+McbatWvpgAMOoNdee837t2K7sg48qYbK9lu0rr3pVcRSe7jdMxPn0+szlnsF23UlfNQrfnWlxGy+pau2JhZ+H5at2UxxXzwRjN08nQBL1ylP+8/GZtMaS/wyjWH6XM0R9YDN8NP2q9GTmsbTfe5/mUHwRQ5JX70s4VTINiZmOttULL2iee171Mt+VAmsLPyO61+UbWH7SPIcgmtbpOPp2kWtQ9SeC7ufh/Gy+dui+uMEOItdkWyO3CTAF154IY0bN857G9w+++zjeTV16lQ655xz6LDDDqMHH3wwmacF6J1lAizVUDEGm7a69n6cYXpg6Ry6hzUqyoh21H4ZoDel7mUSNg8wSbWUYQ8muSqGH8anEBpg6ToVYPtop7S12bZ92MSmMUyfqzHD2qnPpHHJ9a+l8WPap3la16S2SNfAP0/Ueqj7Deuq+V5wzsNjabxPA8yfu9ABS/2O458p+Q3q0aX3oCxs0f1dsV2HpBrgqL8tYbZkHRem+MkyDzHZkrfPc5MA84swRo4cSUcccUQtRm+//TadccYZtHTp0ryxM9qTZeBJv0Ero20eeOA+qv3LEack/uLeUntsTkKV7WGF3KNOt0y+2r4Qgk+iO7SsSvw61LB52Uf1IFT3Nk3Er840+agLVuk6GYM9wwZxbI7Lx++WaQzT5/69N+z5j42/JpjGC35uel0rz5+XU6m0wsXELDivza9bT4ydRze8ML2O6VkytfUvirPNvTfMR5e2+P/GmB5KjbMOOluT3oN1tmT5y4BpL2WZh5hsydvnuUmAWeIwduxY2muvvWox+uyzz6h///60cWPtMl15AxlmT5aBF0cbFYehdJ6k7aJsy0pbJfUhDse89ClGH4vR5uB6p+GDRFee1d7JS3yb7LBZB5u2pnnz8LkkXpSdeYqbPK1DnmzRxVSWeUge4trGhtwkwMccc4wnfbjvvvtq2f+zn/2MPvroI/rggw9s/MpF2ywDL86pWBxI0nmStouyLatv11If4nA09bHRuJrGinMKlPapVpR/Jl2iSy12EnaSvjo/JRpe/y8AwXG4jvGYqYtr1YvWaeP9dqa9rhImeWpjs8cl+u2s9q0LhklPgF3YEGcMmzWLGl+6Vrp2/O/DRn4c+sIb/z5T/blm/Ip1W6h980qrsplxGPn7ZJmHJLU16/65SYA//PBD701wffv2pUMPPZS49u+7777rvR755ZdfJk6Qi+3KMvBca7J0rKXz2LQL01EFtbZ+e7LS3Ul9cB2XWc6b5VyKU9ScuhdCcF9/bU5XWmzXa+cfz4WfOs17mP9hdU399sSp3Z0mnzyMbRP/Jv12GP88M5fqyPPmg82apfV3LGqvmbT8yqasuGaZh+RhT9vYkJsE2PsDN3483X333TR9+nSqrq6mfffdl37+85/TQQcdZONTbtpmHXiuNVk6kNJ5bNo9O2EB/XPGMq8KBBc1P6V/Z+8lBX+ftoTe/XxFHVOyOsmS+uAy6FydcEhtytrHKP909VbDfHGlxZZysm1nWkephldarzSojeeXcPC1cPWmxHp1W9+Lqb1N/HNbnX5bt05Z3aviME+qgY0zp4s+NmsWNp9pb6o+thUkVKUiflCVL9MpexaxkXUe4mJ9sxojVwlwVk5nNQ8CLznpYtBYJfey9gj13ec49VbDGOdJlxhmn+066tpL65XmnYfrfVKo8WzXCetSqJXSzyvdm0nX2qSzziI2kIfo46CgCfCPf/xjT/PbvHlz4v8ddT366KP520UGi/IWeFLNU5hbEg2i+tYbhUVqA7fj2sAjPphHM5evrzMkf3M+tX8XGjlpYS0dpMSGPAaSn4sLjauJs+lzW0ZROrngGum0qqztLi8vo2tHTRNNn8XpicgQTSPpKZPptKlf15Y0ZcEaoyl552F0oAANpPtAsj+L8QS4AMhzMWXUya6/wo/uXtVrl2aiv0u6e7mCkMWezVsekosA+MaIgibArPkdPXq095IL/t9R15tvvpknbiJb8hR4SXRTYX3jaDClNnA7XX1FBm9bC1i0WAVs5IqvcsHE2fS5LQrdeDpNJP/7+Y+ND62p+uh5A+mCP0+gsXNW1pghrU1qa3fa7W05R8V9lCZe7QmuV1usXwDTXgvdl/pgHVxpTXOpBjsrnWch+BXznNJ7rklX72cg1eurPlnFRp7ykLzFTEET4LzBcG1PngLP9jTKz8KkY5J+m5XaEDVfnLfBuV5X1+NJTyOkyY2Js+lzW/9sdXJ86qGrWaur8cw2xamPbOuL6/a2WkVdXVG2a/+urWjygtV1TAzqDl37UF/Hk+4Dm/1pu971lW0x+OVfq6hf3bjGPN97dDXw/W9E1Z0Y8x5t17wSVSByFhi5SYDfeeedUDRcDaJRo0bUpUsX6ty5c87wRZuTpwRYqnkK88ikY1J9THomqQ1R86k5pGMVQ8C49sU0nulzW2ZxdHI8x4NvzaozlSmGbG0rtvZRsa/TApc6s7hrLN0H0nZx7UC/whOQrLGrNll7m6c8JGvfTfPlJgFu2LAh7dy507OXK0Dwxcmv/zrqqKO8t8WxZKIYrjwFnvS0I4xrnk6AbxvchyrKy+kV4RvpbONEqgm0HdffPjhH1Gmo/+160jlNa236XDqPahfnBFhX7cF/miI98ba1N257aWxI2uk09bq4ZpuhMY27cuH9dHEbjEHX+8WtF1+Ppp6ZeH3G8ppKOqcP7FovJDGm/eTi+RTJGrtqo9bf5JerOMlTHuLKJ1fj5CYBfuqpp+i6666j+++/nw4//HDPP34z3CWXXEIXX3yxVwrtiiuuoH79+tFDDz3kyv9Ux8lT4NnqEYMJW1ArV1FGtOPr7yne1aKqAY297hhq0qiBlqnUBp0WcmCP1t6XIn89WP9kSTVVUvuSBE3YHK41riY/TJ/b+merAWatali9X5draeuDqb2UmaSdRH8YtIdrX4fpo5PGvMnv+vx52DqExWBYrOaJu+5+mVW99DRjxLSfJHtJslamedSXDJNmXDKOdCxXXPOUh7jyydU4uUmA99hjD/r9739Pxx13XC3f3njjDbrooovo888/J35ZxqmnnkpLlixx5X+q4+Qt8JLo06R6KdOJpdQGbhesDcz59g0vTK+zZq5ODSXf8JMGjG4OroBQUV5G81ZudFKz1cTZ9Lmtn7rxouZRn+m0dVk8IS31UxobknbSX1R6dWhGnVo29upiq9M81+sm9b++tpPGYJ65R8VTnvZQnBgy7SfpXpJwkKyxqzYmv+Kw0vXJWx7i0rekY+UmAW7atKn3yuO99tqrlk+fffYZ7b///rRp0yaaP3++9/nGjRuT+p1J//oaeBItVBqA05437fGZSRZzpME+zTGLgYnURkk7V5r6NNek1MaWrFtemUiemcir7Sa7TOtSrHvJ5JeJi83n9TUPsWGga5ubBPjII4+k3XbbjR5++GGqqKjw7N2xYwddeOGF9Omnn3pyiBEjRtAdd9xBn3zyiQvfUx+jmALPRo+U5bdX/yLZPI0dZ3HT9osZS94dH8d27mOzhtL2tmNKbFdjzlmxwXsqesX6LfROAd/2J7E5qV5Uadfnr9xIptqgyh6b6g7rN2+nX7w4neZ+tYF6tm1Kt52yT6QcSeJzKbWJurcMP22/WFpa271jaq/7vBRPgNWvfrrnJ4Kxq06ATYyzivm0/9b4/SimPCQr/mqe3CTAfPp7zDHHUOvWremAAw7wHojjf1uzZg298sor3v/nh+B+97vf0U9+8pOsOcWar1gCT6pbUhBs28eCF9LJld5LZ0+afkXpDSUaNRNDW9sl7SVtTHYFPzfpLlV7F0xsbYtqb7Jb2RumFw3TeJvq+tpw4OT34DveoHWbt9e4INHku+RT7GO53p+2e8fUPupzjrmwuun1VQPsjzXJ3oram4W6z5jW2+V+KpY8xKXP0rFykwCzwazt5QfcJk+eTA0aNKC+ffvS0KFDqX379t4p8KJFi+jb3/621LeCtyuWwIvzbVSihUpjAVxpkXW2peVXGidMfh9s11DSXtLGdo1Nmj1Xem5buyTt4+pFdadUurq+QVtM+sWfPzOFRk9eVMcFPkG+54x+EtfQhoh4fYc9/zGNCmFpWoMgQNu9Y2pv+jzsmYn6UgXCtO+Cz0+c3K8TjZmyuM7zFCaGWW+CtP7WBP0oljwka/48X64SYAVgy5YtXu3fYBm0QgBKMmexBF6WeqQkPIN9i8nutG21HV/SXtLGdj1Nmr1iqGlry8W2TnKQqYnJKQ+8T5Pn131BBtcNHj30UNslKun2tmurg2U7jqm96fNSWLSkDJL2L1bGxZKHFIJvrhJgPv298847vYfdZs6cSXfddRd17NiRbrzxxkKwSTxnsQSey2/GEo2Vv03nVlX8PYwWrd4UWf0grM9rM5bR2zO/rLNOfCJw9kHdE68fDxDUq7ZvXkk92zUlfjuQTZ1al4zZrqDms0/nFnTrS5/W8Vl3ciWxR9LGFrLpBNj2pM12fhftbblEnf6HnTYGbeST3A4tq2rtD47LZybOJ677+umStbR83ZY6rvEJ87MXHWwVpy74JB1Dcg9JOoeuv+3aJhnH72fUm8i4so5Ug54Wl6Tj2qyprdZZes+wXVsbm5PySbN/seQhaTLQjZ2bBJjrALPc4fLLL/eS4OnTp9NLL71Ew4YNo1tuuYWuueYa53z+8pe/eA/VzZ49m3bffXe6+eab6Yc//KE3z5QpU+iyyy6jiRMnUtu2benSSy+lq666ysqGYgk8V3okyThSLaU/uTT1CS6KK+2bS12ghI00uMI0n+VE9PVrZP5zRXGQ2CNpI7VZtXPJ1HZuV+1tuejaPzJkAF3w5wk0ds7KGtOC2uDg/2fNIvc7/7HxNH7uKqNLhdI4Gg3TNLBlG3ceXT9X85vGCfs8bK25ZjbfC033wDyvs4mFfy1MWmdTHd6oeHBlh83Bh+v4jDNeseQhcXxL2ic3CXD//v29hHPIkCHUvHlzmjp1qlcV4vHHH6fbbrvNqwPs8nriiSfovPPOo7vvvpu++93v0l//+lcvAX7vvfeoV69eXrm1wYMHey/f4AoUnJw/8MADdP7554vNKKbAc6FHknzDNp0AMtzgN3pJn+CiSE8FohbTNK/tHC4Ys706zWfQF9NJuMQeSRvxhvimoRpz9jdVIOKeqtvO67K9LRdpnWS/flF3Kqh7I5zOP9s4dcnJdizJPcR2TNv2tmurGz9qnDgVbUxa2Lyus82amtomXRtpf5MdtjFVyPbFlIdkzSk3CTDXAeZT3549e9ZKgOfMmUO9e/f26gC7uvhVy5xcn3baaZ7MQl38Eg6uNMEXv5Rj7ty53sN4fPFb6kaNGkVcl1h6lVrgSTRWJg0osw1qHiV9gmti0k1K1tA0r4s5JHYE2+g0n2kwiGMf+rghkFQ7rKwoVJzGoSC5h8QZN299kviZpG8hONjYa9M2TV/yYocLH0stD7FhlpsEmCUI/BrkE044oVYCzCe1N910E82aNcvGr8i2nMTuvffeNGnSJO8lG8HrxBNPpFatWhHLMtT1+uuv07HHHkvLli2jXXbZRWRL2oEn0aeGaWe5FinXX3V98ib51mw6VWWwWZ0AmzReJluzPnFR9j723hyauXy9MQZVRYWT+naiMVMXE6979zZNrPXLxol8DUxMbcYq9bZJtcOKX9ZxmmTdony+dfA+mcVxEh8kfSX3SjVOcE/pqorkdZ1tfJW2Tfs+I7VDstaFbpN2HlJo/5LMn5sEmHW/Dz74oCdJOPfcc+m5557zTmBvuOEGT4Zw/fXXJ/GzVt8xY8bQ97//fXr33Xfp1ltv9cqu8ckzz3XSSSfRfvvt5yXiw4cPr+k3Y8YM6tOnD02YMIEGDBggsiXNwJNoKcNqkoYZ7ko/JtFYxdGymfpEaed0C5XU1qzrrJoYVJQR7eB3RYdccfiIAjzQSMI0zril2idKO1xKGmC1/lnFcRbxJq3dHBYDYbVvXd3D0/Dd5r4gaStpk9SPLOZIaqO0f5p5iNSGvLbLTQLMgFhmcO+999LmzZs9Xiw/uOiii7x/Ky/nx3zcXHyq/KMf/chLevl0mesNjxw5km6//XZ67bXXvBdtnHXWWfTLX/6yZkL1oBwnzYcddlioIWvXriX+j7q4rvGgQYNowYIF1KVLFzfGfzOK5HSS86FrR00Tzevq9ECisfK36dKqsWffQkMViKg+urqPUY5Lv+G7rA0qWghNI5293Vo3pgE92tBNJ/eml6ctpZenLw2tjBEc1tV6+8eVMk3CodT6RmmHn52wgP45YxmxpKtN00bUoUUVdWvTRLSf8swxas9lEcdZsJHuFV27YO1b26o0Wfjon0Pyd0G1N7WVskvqo8mOpONn1R8JsJ50bhLgt99+mw455BDatm0b8Wkrv/mNH0Rr0aKF8zh59tln6YwzzvAearv44otrxueH4fjiMmwsgwg7AdbJJrgfP0THFSuCVxoJsESfynY8+JZMOlJMOkEXAWGj8bJp68K2sDGkNpjiQo2dxnpLbUyLEcatPwQKGcdZUJTuFWm7LGzOyxxgYrcSSICLIAHmt729+uqrxNUg0r4++OADOvTQQz0JhP80l0utcem1Hj16eKXPRowYUWOKRAOME+DoleOflVTt0jIiOrZ3B7J5W5Gt7iuqvc0pgk3btGJXaoPplwFl322D+1BFeblTXbDUxrQYuRw3TDtvqlXtn982Vk19Wc40ctJCp+tlwyuJPzbzqLbSOE7jlwypvUmYSPeKtJ3U5mJrF8b4+Y8Whv6yKY0Fm3WzaZtXtkiAiyABZn0tPwR39NFHpx5HXFGiQ4cO3gs2/LV9WffbsGFD7ySaX8rBD95VVFR49lx77bU0evTo3FSByKMGOGrh2N4k76u31WSZ2ps+DyYkSepPughoqb1h2sKgPjgtDaHURhc80hzDpLc26S2TcMij5jOJP3HXKWzOPGmAkzKR9pe2i8s5z/2iNPDB+tmmPan8tOFp0zbPHJEAF0ECzA+6cdLJ0oM999yTGjf+WhuqLtdvg+Pawvzg3R/+8AdPp/v00097CfEbb7zhlV1j+cXJJ5/svYBj/PjxnlSC7eM6xdIr7cCT1FMN087OS6kKRBSXqBMdyTd325MQSXsbjZdNW2l82LaT2CCpL5rmU+QSG239zrq95PQxKmYlsafzSTK36ivZNy7YJfEnyfzBWIqj9U8yf5z7mc2aSPeKtF1avhZq3Ki4Y83zyI8WEv8ts6lsYxPLNm0Lb0aGWAAAIABJREFUxUgyb9p5iMSGvLbJjQaYH0jTXWVlZd7b2lxf99xzj3fqvGjRIq8sGut3uToEX1ztgd/+xhUi+HXMV155JV1yySVWJiDw/oMrStMn0aPa6r5s21stbI4bS/yWtMmxi6mbJtGfRsVsEr6SuRUAyb5xASuJPy7mz+MYYJL+qqTB2GZMm7bp04g/A/IQPbvcJMDxlze/PQsReBLNkq6NpG9c2nk8AY7ri6lfmhxNc0tOLSRtTPPU588lp7A4ASbiN9K1a16ZSk3xvMdX3D0UvDdkWaM770yD9sVlHOWnZEy1Rq9oKurYnPLngXkh8pA8+C2xAQmwhFLMNlkHnkSzlIauSoInbxpgic1x2kjWIM640j6S+SVtpPPVx3bQANdeVYkeV/WQajGLPW7i7CEJx1LhJ1n/OIxN45rGTLr3TfMX4vOs85BC+Bh3TiTAcckJ+mUdeJJvt1Ea0VGTF9XxyuW3XdayqdqlcapA2GrhbNsLltTYRLIGxkESNpD4LWmT0Iyi7h6nVrXf4SR8w/ry2HE0j64WwW/TsjWbKexeoeZyec9wZX8a49iuseSXBbazVPhJ1sSWcdIxdWuk3qqZ93rLYf5nnYdI1iAvbZAAp7gSWQeeRLOka7N/t1Y0ef7qOjSy0hmmuAyZDi1Zg0wNwmQg4JiASaeMe0Y4cBM31Qv8HAesxXD18f6ddR5igbvgTXOTAPPbjPhht/p0ZR14ktPHQp4Ax1lbl3paNdacFRsS6RajbJKsQRwO6FN6BMLiTNUDtonhOHuI+4TV7Ob5h438mEaH/FqU9AQ4jp3FFBWlegJcTOtaH+/fWechxbQnc5MADxgwgB599FHab7/9iolfpK1ZB55J38TGFkoDHGdRJf5Ix43Sdtno7kw2mT6X2ot2pU0gLI7C6jcrSroYjhOPOr3+wB6tvUOK8XNWahfHZi/5B4ljZ7FFSClqgIttXYvNXskeyDoPkdiUlza5SYDbtWtH48aNo9133z0vbBLbUYjAk2imdG0kfRNDsRjA5bdx0+mLVHcnsSlvHC2Qo2lOCJjiNczMsBiWxGtwLNu59+/aijg579muKcXVSMaxMydLZWVGnmsbWzkibFyM61rf7t+FyEOE4VHwZrlJgO+66y76+9//TldffTXtsccedV6E0a1bt4LDsjUAgWdLrHZ7l3osk/5OqrtzaVMyOuhdnwmY4jXM97AYjhOvtnNL907UesWxsz6vf33xDeta+JVEHqJfg9wkwOXl5TVW+rXAShu8Y8eOwkeSpQV5C7xi0mIxapenB6ZTLZcnwDZh4npNXI+n8yWreWxY1pe2zNaksw3zlevyDj9tP2pY8Z97aZw9ZNorwbl53g4tq6zeyCU9dQ7bl4WIvSg99nzLt5GlGaeFYBPlT5z4S5MPj503Rmn7m7c8JG1/bcbPTQL89ttvR9p95JFH2viVi7Z5Crxi1Da5tFmna+RAObBnG3riwgNrJQ5Rid+5j4yjsT4dZF50jy55RW2grObJxSbO2IgorXqUBliZGYzFOGtlowFuXtWA1m3eXkMp7b0Qx5+kSyjVY8f1Pal9qn8h2Jhsz5tNebPHxM/F53nKQ1z443KM3CTAfqe2bNlClZWVLv0syFh5Crw8fhOXLIpLPdYTY+fRDS9MrzPt7YP3obMP6i4xx2vjyibXa+J6PB2QrOYRL0g9ahhVpYVPd/niesCzV2ygCXNW0uQFdUsXBk9N48Srrma3mn/eyo2kqwcs/TUluGwSOwsRezYn4nF9dxHChWAjsVuyrpJxXLTJKyMXvunGyFMekqafccbOVQL80EMP0fDhw2nBggU0c+ZMYl1wx44d6cYbb4zjW8H75CnwoMUiyhsD1/a4Hk+3gbKap+AbuAAG2LC1aZuGK4WYP09zhjF1oYeOu1aFYBPX1kL1K0VGecpDCrXuunlzkwA/9dRTNHToULr88svpzjvvpOnTp9NLL71Ew4YNo1tuuYWuueaavLEz2pOnwCvFb77BBYrLIIlmLKpvXHtcn8za6ht1dvNJ+g8HdKWRkxZSnnSRileSdTRudkcNbGIiqu2p/bvUWoeT+naiMVMXO10XG1sZT5B/lE26tbKd08WyuDwBZr/C6iv7ddthNktitxBsXPDNcoy0GUnWKUt/ea485SFZ+26aLzcJcP/+/emyyy6jIUOGUPPmzWnq1Km022670eOPP0633XYbff755yZfcvd5ngKvFLVPwYCIwyBOH3/CFaUXTjK27o+krT45jr7RRiNaaF2kdC3ycvOwiQld20eGDKAL/jyhlk7dlVbXzymprTqb+GUbujiO+syURMZd4zh7RLc/z3l4LI2fu6rWx6ZnEKScpe3icqgP/dJklObYSdjnKQ9J4kcafXOTADdt2tQ79e3Zs2etBHjOnDnUu3dv2rRpUxr+pzpm3gIvT1qsVMFHDG7LIMmJgaSvrT0mbrbjxT3d0umpw+wrpC5S2SNZCxPbrD63WcOwts9/tJCuHTXNaK6LdZHaKo0ztqmaKNR+Za90TiMAiwZhc3J31mOzHrp7mybGGshRDKLWwiZ2C8HGAmMumqbFyGadsgSRtzwkS99Nc+UmAeYXYNx///10wgkn1EqAn3jiCbrpppto1qxZJl9y9zkCL3dLYm1QEs1Ykr7WhsbsYFPz1a9vjNsvppmJuxXDWiR28psBpGuTpV7VxiZ248G36t7vs7TX1Vr4x4liEOVbKcVuGtyzGjOv64Q8RB8BuUmAWff74IMP0t13303nnnsuPffcczR37ly64YYb6IorrqDrr78+qzh2Ng8CzxlKpwPZ6PB03+p7dWhGQw7uQacP7Kotn5b1iUCU/sxWUxkG3H9KJT3R43FcnDQmDYCs1yKpvWH9pfpC6dpIa+yy7CBK1y2xy8Ym3QnwUb3a0/H77Go8aVXsJHalsU66MbM4Ac7Sn/oyl6s4yes9BnlIESTAbOJ1111H9957L23evNmzuEGDBnTRRRd5/+Z/UUaxbDwEXv5WSqdf1enwwnRdfq+i9HtZasKi5rLVVIbVm5XUl5X0K1REZLkWafhoY39YW4kGOI7WVWqXjU1h8epnKtGVS+1KY610Y9ree/yJvK22P0u/inkul3HiciyXTJGHFEkCzGZu3LiRZsyYQTt37qS99tqLWrRo4TIWMh0LgZcpbtFkcU5hWDM27PmPadTkRaFzRJ1wpqU3CxoSdfoQR1PJ45v0jS50kaJFc9Qoq7VwZG6tYWxPl4K+ntyvE42ZsjhSryo9pWXDVMzb2GVjk2r78vSl9PbML+sgNf2qYGNXGuulG1NXX9n0AF8xx26WfG3nch0neVwn5CFFkgCvW7eOnn76aZo2bRpVVFTQAQccQKeddhpVVVXZxnUu2iPwcrEMtYxwrcPjwfOgTYzSn7GN9VFTmb/oSs+iLPSFUp2uP+bTtivu+HH7pbeCGDmPBEohTpCHFEEC/Nlnn9HRRx9NnAT36tXLOwHm0medO3emf/3rX9SlS5c87p9ImxB47pcsqV4rzgkwexG3n3sC4SPq7Dt1/8701YatsU7ReKakvLPyv77PE/ekSq3fnBUbaMW6LdS+eSX1bNc0VEeb9glwnDWK67e0X5z4jtMnju/okz4BaZykb0l6MyAPKYIE+Nhjj6WGDRvSk08+Sa1bt/YsXrFiBZ111lnUrFkzGjVqVHoRktLICDy3YF1orJLo8OLU8HRLQD+aRGPp712sOsqseOZtnjixH6VfD1v/NDXAcXnG8Vt9cTPpZuOMHadPXN/RL30CpbCeyEOKIAHmJHfcuHHUp0+fWtbyCzEOP/xwWrt2bfq7wfEMCDy3QF19W0+iw3t2wgL654xlVEZEx/buEFkFwq335tH8+rNlazaHapZtnqR3xdtsOVpICNjqC00numE62ji6blu7JL7628Qd39QvTnzH6WPrL9pnS8AUJ9la43425CFFkACz7OF3v/sdHXfccbWsfffdd+nss8+m+fPnu4+MlEdE4LkFXAp6LVfEXLByMYYrfzCOPQGTpjcP2nV7r9z1iBPfcfq4sxgjgYA9AeQhRZAA/+1vf6PLL7+c7rnnHjryyCM9OcSECRNo6NChdOGFF3oPw6mrW7du9lFQgB4IPLfQcfoi5+mClYsx5BajpWsCcU6AXduQ5/HixHecPnlmANvqPwHkIUWQAPvr/JaV8Q/MX1/VXBSSiPjf+H/zf+/YsaMoojbtwGP90jMT59PrM5YX9Cf5rB4KKQW9lqvAdsHKxRiu/ME4/yEQ3G8n9e1EY6YupvmBV/LaaoCjGKe1xyXj+u9z/DegTdNG1KFFlfZhPqkfnVtV0ZipS2j8nJU1XTq1qqKfHrWHVtoU9gwB179+8sIDtS/EQezml4Ak/vJrvcyytPMQmRX5bJWbN8G9/fbbYkJ8QlwMV5qBF/dhLtfcsk6S6rtey+X6uGDlYgyXPpX6WJKHHf0PuKn1my2oAqFjm9Yel4yru88pWyUPc6q2uof8vrvvrvSHt2fT4jVfv4CJL924nj2PjKuVNEe9DKfU4zXP/kviL8/2S21LMw+R2pDXdrlJgPMKKIldaQZeXspy4SfBJBGCviBgR8Aka1CjmV4UYTNrWntcMq7EX6mvUaUCw15yEzauxGYbtmhbOAKlspZp5iGFWz03MyMBdsMxdJQ0Ay/uCx1cu4uHQlwTxXggoCdgerBN9XT5gFtae1wyrsRfqa+6sfbv1oomz19dB3rYuBKbEb/FQaBU1jLNPKQ4VlpvJRLgFFcwzcAznQB/b79O9IsXp9PcrzZQz7ZN6caTehO/VjSoE0zqfql8i1acSkEzVkq+Ftt6Sk5Eef34BSgdWlZR9zZNQl96Id33zGfYyI9pdMhrwP0npIqj5IUbau6oE9nhp+3naWol/iY9Ae7aujEtWLWpDhKbE+DbBvchflol+DwGP8IyctJC5/dd6fqhnZ5AIf92ZXnfSTMPKfb4QgKc4gqmGXhRGuCHzjmAjrjrTVq3eXuNdxVlRDu+fp7Qu2y0c1GISkVHxQzga1saccGgevOwTzGup0QD3LyqQa29H3evSx+ik7YL3kck/TiBDHsBTZz7WNR8QduiNMDBF2zwQ3A7q6tp4txVtYYZ2KO199C2/yG7uGuR4p+pkh26UPs/63nTzEOKPXiQAKe4gmkHnu6FDtc8H35iE3RVenJiQlQqD0oV8sTAtAauPy8FX4vVx+B+O7lfJxozZTHNW7mRdC9AibPXJSe0HHemU9qoudmXYc9/HPrSFtXPf59LWgUiaj61h/j0XJ1Ah+2rIP/tO6vphhemi7dgnLUQD46GVgQK8bcr6/tO2nmIFfCcNc5VAvzUU0/REUccQV26dKHbbruNnn76aTr00EPpvvvuo6qqqpyhM5tTqMA75YH3QzVtQYul2jmzp6XRolQ0Y7yapeBrffTRpU/SsUw6XdN9RjqPq7tMUnuDdpjGw33X1crVj3GyjvdC5SHFsFq5SYA54b399tvpjTfe8H424sSXX4Dx1ltv0Yknnkj33ntvMfCsZWOhAu/nz0wJ1ewFAZpOOooOeMoGZ/3NPak7SXRmxeZrHFZ59jHu2rn0STeWep22qj/8yvSl9PbML7VLYDrxtLXZz4Zr+RKV0aLVm8R65yQn1mFOmsYL9gnjEXe948Q9+hSWgG28J7W2UHlIUruz6J+bBHi33Xaj4cOH0w9/+EO6+uqr6f3336cPPviA+FXIZ555Ji1atCgLHk7nKFTgrd+8nQ6+441IDbByFJo0+ZJnrd2SW1a3ZVJbk/ZPYntWffPqYxK7kvQNcjdpZoNa47B1k9xfbGw22RR3viT3Q93zGFINsI3/We0NzJMegazXu1B5SHoE3Y2cmwSYJQ5ffPGFJ3/o168fnXLKKXTTTTfRvHnzaO+996aNGze68zqjkQoZeBu3bqcbRk+nOd9Ugbjp5N50y5gZkVq7jLAU9TSF0IzFAebilKFYfI3DR/XJo49J186lT2osriATdcqreB6xZztq16yS2jevtHpTm9RmyWmr6cSZbXXxghB/3Omex+A2Iz9a6OmzdRU5kq53kvhH38IQkMa7C+sKmYe4sD/NMXKTAO+xxx70xz/+kbp370577rmndwJ88MEH0xNPPEG33nor/fvf/06TQypj5y3wstYepQIVg4oIYK1FmHLZKI9rJ9W5mvS+SYFL7EjbhqQ+BPvncb1d+4jxCkcgb3lI4UjUnTk3CfBvfvMb+tWvfkWNGzem9u3b05QpU+iBBx7w5BC33HILXXXVVXniJrIli8AL047pak+mfdIAHZsoLDJplPZaZ+JEPZxEskfirp1k7LhIJSevPLbk9DWuDdxPYsftg/ehsw/q7k0Tl0ncfnF8i6q24aKWcxybSr1PluufNuss8pC0fUhr/NwkwOzg3/72N5o9ezadc8451LZtW3ryySdp06ZN3sNwxXilHXhhWiKuSclXWO3JLdt21tEGt6hqQGOvO4aaNGqQCHHWuqZExpZAZ6xH/hZZuibSdn4P4/SxIRQ2vqt6w0ntCPY/sGcbeuLCA4kPAoI1e+NqhCX9bPwwrV0h2Ma1v771S3svZc0r7Twka39czperBFg5tmXLFqqsrHTpZ0HGSjvwJKchynE+meH3YFw7alodFi5ObeKeWhVkYUpk0ix1ZiWCNJGbNnvEdu1sxo7rRFT94aRvnLOxyW/HktWb6IUpi0PvaXHvd1mwDBrs98llLWcbrmj7NYFCrH+a7NPOQ9K0Pe2xc5UAP/TQQ14liAULFtDMmTPprrvuoo4dO9KNN96YNodUxk878CR6OOUY6+L4evCtWXV8daGZg44tlRDCoPWIQJp7JM2x87wEUX7Hvd8VmmWh58/zemdhW33jn3YeksWapDVHbhJgfgnG0KFD6fLLL6c777yTpk+fTi+99BINGzbM0wBfc801aTFIbdy0A8/1CXAS3VN9+9acWlAIB+a1eGbifHp9xnIqI6Jje3eg0wd2DX0NcdS6JVlToaloJiSQdI+otZyzYgOtWLelVqWF5z9amNqvO0L3avS2YfY1rCiXDqNtF7Yn+JQ37C1stw3uQx/NWx1aD1394qXbG0nXydZRvx1c13jC3FX0YsipNuq225KN137Eh/PoFy/WfbOfi19K41mUrFfaeUgy6wrbOzcJcP/+/emyyy6jIUOGUPPmzWnq1KnEtYEff/xx761wn3/+eWFJxZg97cCz1QBHaeLi6uUUlvqmm4qx3M666OqKKm2jP5mI4p50TZ05hIE8Akn2SFT9W9anPjJkAF3w5wk0ds7KGtpp6laDS2qyb8QFg0K/vElDQ7cnwmrthj0HoeZRzzw0KC/X6oOz3DdR3MLYZLmm0rWpT+1s7r3F4nfaeUixcAizMzcJcNOmTb1T3549e9ZKgOfMmUO9e/f2HoYrtiuLwAvTCjInXe1JnbbQxamHrW6x2NYzK3ujTvaDpxBR6xZXA5mVn6U4T9w9Yvq1h+PiBwd0MdacTYu5xL4zB3WLPX3U+Fz1oaK8rKbW7vad1aGnwmpyyfMQcdfJ1kETt7DxivUk0pZNIdrr1sNfWaQQdiWZM4s8JIl9heybmwR49913p/vvv59OOOGEWgkw1wHmF2LMmlVXu1pIcJK5iynw6pvuSbI+eW0Tpe0O6rXT0EDmlUsp22XS+7vQ8Sfhm7Z9LvaE8i/t5yFsOJq4hY1V6LW28a/Y2tbHv4PFlIdkHS+5SYBZ9/vggw/S3XffTeeeey4999xzNHfuXLrhhhvoiiuuoOuvvz5rNonnK3Tg2eg/XZwAJwaGATwCOAFGIAQJmE4KC30qmLZ9kj2h7nevGN5aJzkBdhmBUfdhEzecALtcCfNY9fHvYKHzEDP1wrXITQLMCK677jq69957afPmzR6RBg0a0EUXXeT9W3l58ocossZcyMCz1Rvats+aZSnNZ6NDgwa4NCIjbY1tUopp22faE2G63TCflIY2K52v6b5q0gCjHnDSyLTrb1ovu9Hy0bqQeUg+COit+P/tnQm0FdWVvzcPkEEFBAFBBp8CsSODKIKA0dCKcYg0osEhUSQaFRvBgGLjDA4BbTXpFsQEIwp/Z0CQbqNCG0UTBgWiSCutjIIBFBBlUuD916nk3tx3X91bp6pOVZ2676u1siLcc/bZ5zv7Fr+3365dVglg5eauXbtkxYoVcuDAATn22GOlQYMGtjMs6F+SgRfkJ9m46t5Se6AxOq7O4vnF6+W1FZs8u0AUOzfONMZDi3ipzFmucukCYaLLQlj3o/av2Hei0P3u1PaHy+GH1KnUMSPDKo7vhs59ONePVo3qOcfw2fbdonor9zu+pcxetjFb36zqvG0467CxYvP8OOIizv0nqUPi3GeQtawSwCrzq9qhqYfh1IswOnXqJAMHDnQywWm8kgy8UqxlSmMM4DMEIBA9AVvvd7b6Ff2JsIItBJLUIbYwKOSHNQJYdXs45ZRTZMeOHdKhQwfZv3+/8zKMtm3byiuvvCJHHXWU7Syr+Jdk4OlkHlIHNKUO+6nFLrbFYn1gTWaFgvhLL+KUBmdMbuf3uhWpIRv+nuXMzWoGiT21BVP3u6DrF8JYyC/Vp7hmWZms27rLyfSS2Y0pEKvhMknqENtxWyOAzzzzTKlbt65MnTpVGjZs6HDbsmWLXHzxxVK/fn15+eWXbWdplQD+Zs8+6Tlunny9Z1/Wr0wPzPoHpTOjnroACNn7NXe/UddYZtYKUgNHHXIaIzM+n73qXE3U5QaJ23wCJmzo2HTrU0x/3/jisbqthAAufOLWCGAlct99912n52/utWzZMundu7fs3LkzdXGbZOCZyoikDrplDps6B6+nxU11AQjiL72ILQs6y9zxil3lronODGFrN4PEvg7qfL8K9Sk29R3W8Ykx1YdAkjrEdsrWCGBV9jBx4kQ544wzKjF744035IorrpC1a9faztKqDDC1Z3aEi6lz8OoXaqo3aBB/6UVsR6zZ6oVX7Cq/bejNGyT2gzCPa50gvjGn9AgggC3NAK9bty7r2ezZs+Xf//3fnZdhqFrgmjVrynvvvSdXX3213H777XLZZZelLjKTDLyoshmpO4SEHTZ1Dl5ZNFPZoyD+kgFOOMgsX94rdnUywANOaCXTl3xmpGa2UJ1vkNgPgj6udYL4xpzSI5CkDrGdZqIZYNXbt0aNGllGFao5o0iVv1Nj1ENxabuSDLwo6tnSxt8Gf02dAzXANpwmPgQhELYG+PFB3eTKKYtlweqt2eWD1szaUK9u6p4Q5CyYU/0IJKlDbKedqAB+8803tfmcdtpp2mNtGZh04IWtibOFY9r9MHUOUfdZzXAO4i+9iNMepdH6X6zXbW4HBLc4evG9z2T0jA+qOBjktx5e2dcgsR+EXFzrBPGNOaVFIGkdYjPNRAVwITCq+0Pt2rWlUaNGNrPz9I3A80TEAAhAAAJFCZismTVpi2ODQBoIoEMKn5JVAnjChAly7733yqZNmxyPjzjiCLnpppvkhhtuSEOcVfExzsAz3b8ylcBx2goCxKIVx5B6JzJx9Iflf5U3V24xkgGe+ue1cvus5UZsuQHOj/3zurSU2X/ZaKR2ObMe36/Uh3asG4hTh8S6MQOLWSOAJ0+eLEOHDnX+d+qppzo1v2+99ZZMmjRJlDD++c9/bmC78ZqIK/CoKYv3XFmtMAFikegwQUC3btjPC2CUzZ9NXiCL1myr5GKP8sYy7aoeoV8x7ObzoXVrVerFHrR2OVf8Xv74QiP10CbOCRv2E4hLh9hPoqqH1gjgY489VoYPHy5Dhgyp5KUSv0oEf/BB1Row24HHFXhedW22c8K/0iFALJbOWSa5k0Jx9MMOTeWsjkcEenNaIZv39u8oPz25bejt6nS7UIsEqV3OOMf3K/QxVTsDcemQNIK1RgDXq1dPli9fLsccc0wljp9++ql07NhRdu/enTq+cQUedW2pC42SdZhYLNmjjXVjUcRRFDZzoej0O1bjw/TsjnoPsR4yi8VCIC4dEstmDC9ijQBWGeAxY8bIRRddVGmLzzzzjIwePVrWrFljeOvRm4sr8MgKRH+WrKBHgFjU48So4gRMx5EqT7h5+vsyc+mGKguHySrn1uNu+mqPzHCxn7+gygAH7WtsmgtxWPoE4tIhaSRpjQB++OGHnQfg7r77bufVx6r37/z58+WOO+6Q66+/Xu68887U8Y0r8Ki7TF1olKzDxGLJHm2sGzMZR171xJmN+a3P1an5zYem1gjT19gkl1gPlMUSIxCXDklsgyEWtkYAHzhwQEaMGCGPPvqo7Nu3z9mSehucehPcf/zHf4h6aUbarjgDj76SaYuO0vWXWCzds41zZ6biSLc2V+3NT31uIbtdWzeSpeu3V0E1oOuRMv7CzhK2r7EpLnGeJWslRyBOHZLcLoOtbI0AVi/F6NWrl+zZs0c++ugjUW+FU2URDRo0CLYzC2YReBYcAi5AAALVmoBuba6C5Kc+t5Ddrm0aydJ1VQVwxjZ1vNU6HGPfPDqkMHJrBHDTpk3l1VdflRNOOCH2AIlqwSQDj16RUZ0qdnUIqPh77t11MnfFZlEvO+/7/eYy8KTWoVtN6azNmOgJ2HB/yfiw+oud8sXXe6XpoXWk/PCDq3SIsCUDnMkuF/JHZYibN6wrbRvXL9jlwgbu0UeXHSuUCuskdYgdJ5kCAXzcccfJI488In369LGdmbZ/SQUedWLaR8TACAhE3W81Apcx6YOADfeXYnW9+bW8bmO7lzd2drxo9dbszk3UAGeMFev/q1M77OaLDdx9hEmqh5YS66R0SBoCwJoMsKr/Vf1+zznnHGnfvr2otmi5l3oYLm1XUoHHk8Jpi5TS8rdYxs1PjWVpUSmd3dhwf/HK6ubHmVvdrDqR6e99Jmu37iqadS12csruzS++79r9oVhGN9efQt0j8vdgA/fSieLiOykl1knpkDTEijUCuLy8vCDKkEWWAAAgAElEQVQv1RFi1apVaeBZycekAo8as9SFSkk5XKzm0k+NZUlBKaHN2HB/8arrjTPOwvLQna87roRCLbGtlBLrpHRIYofnY2FrBLAPn1MzNM7A0+lHSfYtNaGTakfJAKf6+Dyd9/tGtW/27JPbZy2XNV/ulPImB8s953eU+gfVkkI1ljq1l14ZYJ16Ws+Nag4Imy3UrQkO2z1CczsME5GwZ2oTxDh1iE371vHFKgGsWqHNmzfPee2xaoF24oknyimnnKKzDyvHxBV4QWvKrISGU6knQA1w6o+w6Ab8nK8Svz3HzZOv9/yttaW6GtStJW+N6iPXTn1PFuTV4Or2yC1WA1ys/jaKkwlbL6p7/9ZlE8Ueq5vNsGdqE6+4dIhNe9b1xRoB/Pnnn8uPfvQj53XIjRo1ctqgffXVV85LMebMmSMNGzbU3ZPvcStXrnS6T6iH8K644gpn/rJly2T48OHy7rvvSpMmTWTYsGFy4403+rIdV+DpZhBq10xfL2VfwBlsDQFV4/j84vXy2opNdIGw5lTMOTJtwVq57aXlVQzm/5bpl88tc337WrFeuW5vU3P77VWmjnZVTheILV/vda3Hjfq3X2F78+rWBF9wYqvQdcvmoqC0LYU9U1voxKVDbNmvHz+sEcADBgyQ9evXy7Rp0+R73/ues4cPP/xQLr/8cuncubM88cQTfvalPfa7775z+g8roavWUAL4yy+/dHoQ9+/f33k5x4IFC+S6666TiRMnyuDBg7VtxxV4pVSvpA2XgRCAQGIEdO855098x7UnbvMGdWTTjr1V/Pfqoeu1YV2/vOwk+Xkp7CFJfqxdmUBcOiSN3K0RwOqFF2+88YZT9pB7LVy4UM4++2zZuvUf7WpMgr7lllscgavWzgjgX/3qVzJhwgRZs2aN1KpVy1lOjZsxY4bzkg7dK67AK5QB7tD8EPlpjzZSs6xMNmzfHfhJZ939Ms5OAjo1lVF6nvT6Ue6tutnOnOUflv9V3ly5pcr2w2aAC2WG3TK4bnFVqE428xa2JH4LluvnkY3qyr4DFfLGR1sK/mbEb411dYtB9uuPQFw6xJ9Xdoy2RgC3adNGnnvuOenZs2clMu+9957TGm3Tpk3Gib311lty/vnnO+UOav2MAFbrqTKMp59+Orvm3LlzpW/fvo4fzZo10/IlrsBzq7Mr5KDfXpdaG2WQtQSSrmVLen1rDyaFjhWru1Xbcbu3+KkBziDRqeEtFFdudbIZu0nc+7yYKd96lDeWaVf1yL4kxk+NdQrDCJdjJhCXDol5W0aWs0YAT5kyRR588EGnBKJLly7O5lavXi2XXXaZXHrppU4Jgslr+/btzjoq26vsq1ZrGQGsSi5U1nn8+PHZJVesWCHqZR2LFy+Wbt26ubqyY8cOUf/LXKquuXv37k5pR6tWrUy6X8mW1xPR+QtHXQ8X2UYx7JtA0k8zJ72+b2BMKEig0Fn+sENTOavjEQXfXrbr231y28zlsjqvC0TQHrrKwWJxpepkC/Xmjfvep3tvzvdLt8aacIWAFwEEcGFC1gjg1q1by+bNm2Xfvn1y6KGHSu3atZ2yB/UwnBKnudf+/fu9ztzz80suucSx/eyzzzpjcwVwu3btHFE8duzYrB3Vh/iYY46R+fPnF+xMcdddd8mYMWOqrB21APbqiZnvUJw9Mj0PggGREki6njDp9SOFW82MR3GWQW16zfP6PK6j070359+TbfE/Lk6sEx0BBHAKBPCTTz6pHQGDBg3SHus2cOrUqTJ69Gin3dphhx1WRQB36tTJKbtwywAvWbJEunbt6rp+WjLA/Y9vKQ/8pEv2V26hYAaYTE1oAGgBpySdgU16/YDYUj0tTH/dYhv3e5Y633O/NjP+FZs34IRWcvP09127T/itBdbZQyFmam4hP/Ln8Na3VH/lrHYeAZwCARxnBPXp00fefvttqVOnTnbZnTt3On9WWd62bds6rc+UUM5cNtcA69SZ5fPNrzuLiz81oXGR/ts6SfNOev14aSe/mp/aWL81sX7OUndsoRrhBbec7rwso5i4vPzxhVp9hPNt6O5bdw9uPvq5J7vdi8OsnXwU4oFNBBDACOBKBDZs2CC7d++u9Hft27d3Mr4XXXSRPPPMMzJp0iT59NNPnRdyqEtljGfOnGllFwjlX25PzM079sjWnd9KWY0a0qBeLZn9l89dIyDuejjlRNCMj003lLT5knQ/y6TXT9t5hfG3WE9w3f66xdbXPUvd77nuODef3Hwp1AUif77OvS+Mb4Xmqt++ndDmMJn30WbP/ti6rMPEC3NLnwACGAHsGeW5NcCqFln1Ae7Xr5+MGjVKFi1aJEOGDHFEsZ/yCxsCr1gNWhK1wNS2eYYiAyAQmECh71fY/rp+HdL9nuuO010/aM2tm/0wvoWZq7tXxkFAh4ANOkTHzyTGWPMQXBKbz10zVwCrv1fdHtTb35YuXSotWrSQkSNHytChQ325aUPgFXsKWScL4mvDGoPDZFU0zDMEAtWaQNQZ4GJwc+tlN321R+uNbEE7SxTyI2jXBTd7uveq/Drh87q0dN6SN3Pphipm/dxzw9QfV+svAZuvRMAGHWLrkSCAIzwZGwLPtp6S1LZFGHCYrvYEoqwB9hK/+TW5Qfv55q6jW6+bmVPofhfEps69ym1M/r4za/vZi87a1T7YAaBFwAYdouVoAoOsEsCq1+7y5ctlz549VVCoVyKn7bIl8FQt2fOL18trKzZ51p3FwZjatjgos0Z1JVDo+xXl965Y5rl5w7pF30KZ8esVzbfLeZ1roR66Xv2K3ex6MdPNOPvtPqGbffZiwecQsEWH2HgS1gjgcePGOa8bdrtUeYKJ3r9xHwCBFzdx1oMABJIgYKLm1YQNtXdTdnQ4mqw5zl0vzj3o7JMx6SWADil8dtYI4COOOEKuvfZap9tCbnuy9IadCIGX5tPDdwhAQBHQqUUt2PWgS0vpVt5YNmzfXTQLrNYplkUef2Fn7b7lhezc0/84qVlWJuu27vL0RWfPxXzOjxwywHyXkiKADkmBAG7cuLHTbUG9ha1ULgKvVE6SfUCgehLQrUXV7XtbrA62mI2w9bPdyxs7B7ho9dbsQRayqbvnzA8HXrXPmQXD7sHP/OoZrezajQA6JAUC+IYbbnAyv7lvX0t7OBN4aT9B/IdA9SbgpxZV1cve/OL7rt0fcikW64RQzIafDgr5tbv7DlQ4nRnyLzebfvas7OWv1e/4lnLbzOVaXTCKRZdX/XH1jkx2r0sAHZICAbxx40bp3LmzHHzwwVJeXi5lZWWVvP6f//kf3fO2ZhyBZ81R4AgEIBCAgN9aVJ2aWK/+437X1NmWH5t+xhZa24QNnX0xBgJeBNAhKRDA55xzjixcuFDOOOMMqV+/fhWPn3jiCa9ztu5zAs+6I8EhCEDABwG/2VCdrghemVy/a+psx49NP2MLrW3Chs6+GAMBLwLokBQIYCV6582bJz179vQ6z9R8TuCl5qhS56juQzqp2xgOW0XATz2scvybPfvkvEfeltVf7HTdh04dq981cxcq9L3wY9PP2EKH5ceG24s0Zv9lY/ZhPfVijdw/X3BiK+0HAq0KJpxJhAA6JAUC+Oijj5bZs2dLx44dEwmSKBYl8KKgik0//7hCCwJhCejWorrFZXmT+jL4lHL5/Ks9np0Xcv3UXTNf/OY/kJYruP3Y9DO2EF8dGzov0tB5oUjYM2Z+6RJAh6RAAD///PMyadIkmTBhgnTo0EFq1qyZ+ogk8FJ/hFZugF+vWnks1d6ppOMy6fWDBIBOyYibXa8ykiC+MKc0CaBDUiCA27dvL+vWrZN9+/a5esuLMErzy8mu/BPgARv/zJgRPYGk4zLp9YMQ1nlo0M2u14OEQXxhTmkSQACnQAA/+eSTRaNv0KBBqYtOAi91R5YKh9OY6UoFWJzUIqDqfG+ftVzWfLlTypscLPec31HqH1Sr4IsswmQr/dS62/S90PWbDLBWyDEoBAF0SAoEcIjztXYqgWft0aTaMWqAU318qXZeid+e4+bJ13v+8Zu6BnVryYJbTpdaZWVSrAbX78b9xrnf8X790R3vxw9qgHWpMi4oAXRICgTw2LFji57vHXfcEfT8E5tH4CWGvuQX1nnApuQhsMHYCfzyuWUyc+mGKuuqV/0+dNHxVV4KEaZjQZCMrg3fC79+u71IY/ayjbL2769sVi/WyP1zGKaxBwwLJk4AHZICAaxefpF7fffdd7J582Y56KCDpFevXvLaa68lHkh+HSDw/BJjPAQgYDOB8ye+I0vXba/iYtc2jWTmdb2Nup7Gml4FIK1+Gz08jFlDAB2SAgHs5uKOHTtE1f726dNHhg0bZk1A6TpC4OmSYhwEIJAGAl4ZYJN78JtJNbl2GFuF/FZZ8uYN63q2gyvWy3j6ks+y/YHJBIc5peozFx2SUgGs3F6+fLmce+65snbt2tRFLIGXuiPDYQhAoAiBYjXA6kE4k5efWlqT64a1pVPXW+iFIIX2/PigbnLllMWyYPXWrHs6LxUJuxfmp58AOiTFAnjp0qXSu3dv2bVrV+oikcBL3ZHhMAQg4EFg17f75LaZy2V1XheIKMDZUNMbZF+5fm/6ao/McKmbduuOUSx7rGsjiL/MKV0C6JAUCOCnnnqqkpcVFRXy1VdfyWOPPSYtWrSQuXPnpi5CCbzUHRkOQwACEDBKwE9NcKGxqsbarfaafsBGj6okjaFDUiCAy8rKqnhZu3ZtJ/ur3hCn3g6XtovAS9uJ4S8EoiOg2xs2Og/isVxd9pmhmb/f87q0lNl/2Zit1d13oEJue2l5Ffj39u8oPz25baW/L5QB7n98S3lp2cYqNsL0WI4nGlglaQLokBQI4KSDJIr1CbwoqGITAukjkNZ6Vr+kq8s+c8Vvfu/jQ+vWqtQnuXt5YzlQUSHvrtlWCWeP8sYy7aoeUrvmP5I/it/PJi+QRXljTzrqMKlRo4YsogbYb0hW+/HoEARwIl8CAi8R7CwKAesIpLWjgV+Q1WWfGS66b3Lzk8GdtmBtwYxxzbIa2f7AdIHwG53Vczw6xFIB/M///M9aEal+8p03b57WWJsGEXg2nQa+QCA5An7qQJPzMvzK1WWfGVKF9ptP0k8Nb3VjGD7qsFCMADrEUgE8ePDgopE7f/58WbVqlTRo0EC2b6/afN32sCfwbD8h/INAPARKPTOaqYP9w/K/ypsrt1SBqjKgD/ykS6Vf96tBXvXCXp/HcXrFfNDNAKsewDpdHNRaN09/3/Vte9T7xnHapbcGOsRSAVzIra+//lpGjBghjz/+uJxxxhkyefJkadOmTeoik8BL3ZHhMAQiIVDKtbFue3ODmF/z6sXE6/NIDirPqJcPuj1/dfr4FuNIz984Trs010CHpEgAv/766/KLX/zCyfjef//9cvXVV6c2Kgm81B4djkPAOIG09rT1AqGbBVV2crOYXllxr8+9/DLxuY4P+efa7/iWMnvZxiq1ul7nX6wH8PgLO1fJnpvYHzZKnwA6JAUC+JtvvnGyvirbe/rppzvZ3zRmfXNRE3ilf3NhhxCo7gR062AVp9y+tV61rl6fx8E9Th/iXCsOdqxhBwF0iOUCOJP13bZtm5P1veaaa+yInJBe2BJ4NtTRhUTJdAhAwFICfjLAHZodIi0b1ZO+328uFSKu3Q4yWWKd7GvUSAr5cE//46RmWVm216+Jjgx+9ss9PeqTLx37tugQG4nWqFCvXEvoUlnfkSNHVsr6tm7dOiFvzC9rQ+B51bCZ3zUWIQCB6kTA7R5zSJ2a8s3e/UUxePW2teHe5eaD6uurLtM9eXX3qzuuOsUgey1MwAYdYuv5JCqAjzrqKFm/fr0cffTR8rOf/cxp9F3ouuOOO2xlWNAvGwLPT1YhdYBxGAIQsIKAWx3szCUb5LUVm+Tz7btl5eZvXP1Ub0Mr1tvWq242js3n+1DozW4mujTo7Jd7ehynXjpr2KBDbKWZuAAuJnoz0NQY1Q4tbZcNgUddWdqiBn8hUFoEitUI59YEp2XXSd9Tk14/LeeEn38jYIMOsfUsEhXAtkIx5VdcgRekT6WJbIUpTtiBAATSSUCnFrVYjbCN96HMnlZ/sVO++HqvND20jpQffrBk6nzjrAt2iwoywOn8riTldVw6JKn9hVkXARyGnsfcOALPqx7M6/MIt49pCECghAno3lvUuJ9NXiCL1myrRCO/L7ANqHR68aqnZi5/fKEsWL0163JUdcFuTHS528ATH5InEIcOSX6XwTxAAAfjpjUrjsDTyQbo1JVpbYhBEIAABP5OQOfek4Gl7kHPL17v1ASrJz1UF4iBJ7W2rretV0eLTMY6zrpgt4Djns7XUJdAHDpE1xfbxiGAIzyROAKPerAIDxDTEIBAQQKleO/x6mlcqGa5FFkQ+qVBIA4dklZSCOAITy6OwPOThYlwq4mZ1qlBTMw5FoZACRMIcu/xqq8Ng6vYvUD3PuGVAR7Q9UhxeytbEBZh9spcCOgSiEOH6Ppi2zgEcIQnEkfgVed6sOq89wjDFtMQ0CLg9/unU19bu2aZ1tr5g4r54laze/LRTWTqld2rlGAU8zGzpttcvywCbZJJEAhAIA4dEsAtK6YggCM8hrgCr7rWg5F1iTB4MQ0BDQJ+7j1e2dUwHSGK3QvUm55Gz/igym4KrZfZ06ovdsri1Vtl6frtWnP9sNBAyxAIGCEQlw4x4mzMRhDAEQIn8CKEKyLU3UXLF+sQMEkgaH2tjg/F7gVq/qN//LSKGZ0exNxjdOgzxmYC6JDCp4MAjjByCbwI4YoIGeBo+WIdAmEJ5Nbebvpqj8xYuqGgSbeMrG7t7tQ/r5XbZy13zdT6zQDnGvFzj9H1NQjTKG0H8Yc56SGADkEAJxKtBF602Km7i5Yv1iEQhoDb9/PQurXk6z37qpgNU1fr1WfYTw1wvmO69xjdcUF4Rmk7iD/MSRcBdAgCOJGIJfCix07dXfSMWQECQQgUyp6qTgqHH1rH9S1rQbKvhda5t39H+enJbR2TYe4TOnP9ZIr9sozStl9fGJ8+AugQBHAiUUvgJYKdRSEAAQsIhK2f1Z2vOy5KJFH6EKXtKJlg2w4C6BAEcCKRSOAlgp1FIQABAwTy607P69JSZv9lo6zbukvaNq4vF5zYquCb3NTcm6e/LzNdan7v6X+c1Cwr87RTLPM54IRWMn3JZ46NQrXFhbo8+NmXV+1t5vM/LP+rvLlySxXquT542Sp0ZGSADQRzNTaBDkEAJxL+BF4i2FkUAhAISUCnfjdIL93u5Y0dzxat3pr10I8dNfbxQd3kyimLZUGOjfzaYj82C831qh326hmc60OYOt4wc0OGAdNLgAA6BAGcSBgTeIlgZ1EIQCAkAa+evRnzblnWYrW/J7Q9TG57yb1bw8Xd21Tx2q3+9sX3PnPt66tqi5s3rFs0O+1nX17dIwrZ+mGHpnJWxyMqZcjDZnF16pBDHjnTS5QAOgQBnEhoE3iJYGdRCEAgJAGvnr0Z8269dKPqyZtZM0xNrJ99qfWK9Q/244efsSGPjukQqEQAHYIATuQrEXfgBa0xSwQOi0IAAtYS8JMpzc/cmnwrmxsgnWxqoXuhn30FzQC7ZcWL9Sl2y3xbGxg4ljoCceuQNAHiRRgRnlacgUedWIQHiWkIVDMCbveTmjVE9itV+PerQd1asuCW06X+QbUq0Sl2L/Kqq9XB7HWv87u+yRrgQv2MfzZ5gSxas63S9nqUN5ZpV/Uo+CChDgvGQMCLQJw6xMsX2z5HAEd4InEGnk5WJMKtYhoCECgxArl1p347LRSrWTVRz1rMhte9MH9uv+NbyuxlG2WtS3cLL1+9PlchodOnuMRCh+1YRCBOHWLRtrVcQQBrYQo2KM7Ao8Ys2BkxCwIQ8CaQpvuLbb7a5o/3aTOilAjEqUPSxg0BHOGJxRl4XlmPCLeJaQhAoMQJhLm/6D6boDNOZ4yf7gw6x6azZjE7Ydhl7Ib1QWefjClNAnHqkLQRRABHeGJxBp5XXVyE28Q0BCBQ4gSC3l905+mM0xmjjsFPf16vY9Nds5idsDbCzvfaI5+XNoE4dUjaSCKAIzyxuANPpx4twu1iGgIQKGECQe4vutlPnXE6YzL4M76+ovGGtqizt8p+EHYZv/zsu4TDj60FJBC3DgnoZiLTEMARYifwIoSLaQhAwHoCuvWvOuN0xuQDCTIn10bY+SYOyAYfTOwDG8kQQIcU5o4AjjAmCbwI4WIaAhCInEDY2tNC2ctT2x8uhx9SR5oeWkfKDz9Y9h2o8HxDXCFb/bu0lG27v5MaItL3+81l4Emts63FCs3p2rqR/L9f9KjSwi0fqA3ZVxt8iDzQWCAyAugQBHBkwVXMMIGXCHYWhQAEDBAwUXvqVY+bcbN7eWPnPxet3pr1PL+nrputQ+rUlG/27q+029z+usXWL9THONeYCQZhj8IGH8LugfnJEUCHIIATiT4CLxHsLAoBCBggYCrz6FWPm3H13v4dpWZZDdd+vJkxOr2J1djct7GpOQMn/VmWrt9ehcqArkfKQxcdX5RWmPpdA8fgmLDBB1N7wU68BNAhCOB4I+7vqxF4iWBnUQhAwAAB07WnhexlXB3yw2Pk5rOO1fa8mL18W+dPfEeWrqsqgLu2aSQzr+utvSYDIZA2AugQBHAiMUvgJYKdRSEAAQMETGWAM64Uspf5/J7+x0nNsjJZ5/JGtvztqLKAm6e/LzOXbnDdaW4GWA345XPLXMd2aH6I/Lx3uVxwYitRr2mevuQzrfUN4M2aCFtnbdIXbJUeAXQIAjiRqCbwEsHOohCAgAECpmtPi9XjHlqnphzbooEsXrMt63l+DXDmA6+64twa4Mycb/bsk57j5snXe/a5ktGpQTaAtIoJ04yj8BGb6SaADkEAJxLBBF4i2FkUAhAwRMB07amyd/OL78uMApnbfLfzM7nq80KZZJXNHdTzqEpdIHLt7fp2n9w2c7ksXrtV1m/drUXIbX2tiZqDTGfZNZdlWDUigA5BACcS7gReIthZFAIQsJiAVy1wrutudcFha5PDrm8Sbdi9mPQFW6VJAB2CAE4ksgm8RLCzKAQgYDEBr1rgXNf9ZIB1s7Vh11f+marbLeSL6k4x/sLO2X7GFh8nrllOAB2CAE4kRAm8RLCzKAQgYDGBIP18c7cTtm426fW99pL5vFANtMVHi2sWEkCHIIATCUsCLxHsLAoBCFhOQKefr+oL/NOT27ruJGxtcpj1TdftFquL1s1qW37cuJcgAXQIAjiR8CPwEsHOohCAQIoIJF0H63d9v+N1jiIKmzrrMqb0CaBDEMCJRDmBlwh2FoUABFJEwHRG1e/W/a7vd7yOP1HY1FmXMaVPAB2CAE4kygm8RLBbt6ipB2as2xgOQcAAgbA1vcqFMN8xv+v7Ha+DqJDNxwd1k9l/2Vjp5RxJvbBDZx+MsY8AOgQBXIXA1q1b5ZZbbpE5c+bIjh07pHPnzjJu3Dg55ZRTnLHLli2T4cOHy7vvvitNmjSRYcOGyY033ugrugk8X7hKcnAU/1iWJCg2Va0JhKnpNfEd87u+3/E6h5tv87wuLeXKKYtlweqt2elJvbBDx3/G2EkAHYIArkLgzDPPlM2bN8sjjzwiTZs2lQkTJsjkyZNlyZIlzp+PPfZY6d+/v4wYMUIWLFgg1113nUycOFEGDx6sHeUEnjaqkh3IrzZL9mjZmCUESvU7ZqJdmyVHhBsJEkCHIIArEfjkk0+kffv28s4770ivXr2czyoqKqRDhw5yySWXSL169RxBvGbNGqlVq5bzucoWz5gxQz766CPtUCbwtFGV7EAebinZo2VjlhAo1e+YTS/ssOSocSMAAXQIArgSge3bt8uf/vQnOf3006VOnTrZz9q1aydnnXWWrFq1Sho1aiRPP/109rO5c+dK3759ZdOmTdKsWTOtMCTwtDCV9KBSzU6V9KGxuVQRKNXvGBngVIWhtc6iQxDAnsH5wgsvyMCBA2X27Nly6623ytlnny3jx4/PzluxYoUcd9xxsnjxYunWrZunPTWAwNPCVNKDTNQnljQgNgeBkARK9Tvmti9qgEMGSzWcjg5BABcNe1UKoTK/KiP80ksvicoEX3rppTJ27NjsPJUVPuaYY2T+/PnZB+XyjaqH6dT/Mtfnn38u3bt3l/Xr10urVq2q4VePLSsCUTwwA1kIQOAfBEr1O+a2L7Xr6e99Jmu37pK2jevLBSe24pXJfBkKEkAAI4ALEpg1a5Yjdk8++WSnI4Sq/+3UqZOcc845rhlg9ZBc165dXe3dddddMmbMmCqfIYC5O0EAAhCAAAQgEDcBBDAC2JWA6gChWp0NGDBApk2blq0HVuJXtT6bOnVqdp5ODTAZ4Li/2qwHAQhAAAIQgEAhAghgBHAVAo8++qjT2kz193344YelrKwsO0b1A540aZJ8+umnUrNmTefvR48eLTNnzqQLBPcZCEAAAhCAAARSQQABjACuRGDlypXSsWNH+fGPf+z09s29VAnE3r17nT7A/fr1k1GjRsmiRYtkyJAhjigeNGiQdtATeNqoGAgBCEAAAhCAgGEC6BAEcCUC9913n9Ppwe1SAnfKlClOtweVHV66dKm0aNFCRo4cKUOHDvUVmgSeL1wMhgAEIAABCEDAIAF0CALYYDjpmyLw9FkxEgIQgAAEIAABswTQIQhgsxGlaY3A0wTFMAhAAAIQgAAEjBNAhyCAjQeVjkECT4cSYyAAAQhAAAIQiIIAOgQBHEVcedok8DwRMQACEIAABCAAgYgIoEMQwBGFVnGzBF4i2FkUAhCAAAQgAAERQYcggBP5IhB4iWBnUQhAAAIQgAAEEMBFY6BGRUVFBVESDQEEcDRcsQoBCEAAAhCAgDcBdAgZYO8oiWAEgRcBVExCAAIQgAAEIKBFAB2CANYKFNODCDzTRLEHAQhAAAIQgIAuAYVwTsEAABhDSURBVHQIAlg3VoyOI/CM4sQYBCAAAQhAAAI+CKBDEMA+wsXcUALPHEssQQACEIAABCDgjwA6BAHsL2IMjSbwDIHEDAQgAAEIQAACvgmgQxDAvoPGxAQCzwRFbEAAAhCAAAQgEIQAOgQBHCRuQs8h8EIjxAAEIAABCEAAAgEJoEMQwAFDJ9w0Ai8cP2ZDAAIQgAAEIBCcADoEARw8ekLMJPBCwGMqBCAAAQhAAAKhCKBDEMChAijoZAIvKDnmQQACEIAABCAQlgA6BAEcNoYCzSfwAmFjEgQgAAEIQAACBgigQxDABsLIvwkCzz8zZkAAAhCAAAQgYIYAOgQBbCaSfFoh8HwCYzgEIAABCEAAAsYIoEMQwMaCyY8hAs8PLcZCAAIQgAAEIGCSADoEAWwynrRtEXjaqBgIAQhAAAIQgIBhAugQBLDhkNIzR+DpcWIUBCAAAQhAAALmCaBDEMDmo0rDIoGnAYkhEIAABCAAAQhEQgAdggCOJLC8jBJ4XoT4HAIQgAAEIACBqAigQxDAUcVWUbsEXiLYWRQCEIAABCAAARFBhyCAE/kiEHiJYGdRCEAAAhCAAAQQwEVjoEZFRUUFURINAQRwNFyxCgEIQAACEICANwF0CBlg7yiJYASBFwFUTEIAAhCAAAQgoEUAHYIA1goU04MIPNNEsQcBCEAAAhCAgC4BdAgCWDdWjI4j8IzixBgEIAABCEAAAj4IoEMQwD7CxdxQAs8cSyxBAAIQgAAEIOCPADoEAewvYgyNJvAMgcQMBCAAAQhAAAK+CaBDEMC+g8bEBALPBEVsQAACEIAABCAQhAA6BAEcJG5CzyHwQiPEAAQgAAEIQAACAQmgQxDAAUMn3DQCLxw/ZkMAAhCAAAQgEJwAOgQBHDx6Qswk8ELAYyoEIAABCEAAAqEIoEMQwKECKOhkAi8oOeZBAAIQgAAEIBCWADoEARw2hgLNJ/ACYWMSBCAAAQhAAAIGCKBDEMAGwsi/CQLPPzNmQAACEIAABCBghgA6BAFsJpJ8WiHwfAJjOAQgAAEIQAACxgigQxDAxoLJjyECzw8txkIAAhCAAAQgYJIAOgQBbDKetG0ReNqoGAgBCEAAAhCAgGEC6BAEsOGQ0jNH4OlxYhQEIAABCEAAAuYJoEMQwOajSsMigacBiSEQgAAEIAABCERCAB2CAI4ksLyMEnhehPgcAhCAAAQgAIGoCKBDEMBRxVZRuwReIthZFAIQgAAEIAABEUGHIIAT+SIQeIlgZ1EIQAACEIAABBDARWOgRkVFRQVREg0BBHA0XLEKAQhAAAIQgIA3AXQIGWDvKIlgBIEXAVRMQgACEIAABCCgRQAdggDWChTTgwg800SxBwEIQAACEICALgF0CAJYN1aMjiPwjOLEGAQgAAEIQAACPgigQxDAPsLF3FACzxxLLEEAAhCAAAQg4I8AOgQB7C9iDI0m8AyBxAwEIAABCEAAAr4JoEMQwL6DxsQEAs8ERWxAAAIQgAAEIBCEADoEARwkbkLPIfBCI8QABCAAAQhAAAIBCaBDEMABQyfcNAIvHD9mQwACEIAABCAQnAA6BAEcPHpCzCTwQsBjKgQgAAEIQAACoQigQxDAoQIo6GQCLyg55kEAAhCAAAQgEJYAOgQBHDaGAs0n8AJhYxIEIAABCEAAAgYIoEMQwAbCyL8JAs8/M2ZAAAIQgAAEIGCGADoEAWwmknxaIfB8AmM4BCAAAQhAAALGCKBDEMDGgsmPIQLPDy3GQgACEIAABCBgkgA6BAFsMp60bRF42qgYCAEIQAACEICAYQLoEASw4ZDSM0fg6XFiFAQgAAEIQAAC5gmgQxDA5qNKwyKBpwGJIRCAAAQgAAEIREIAHYIAjiSwvIwSeF6E+BwCEIAABCAAgagIoEMQwFHFVlG7BF4i2FkUAhCAAAQgAAERQYcggBP5IhB4iWBnUQhAAAIQgAAEEMBFY6BGRUVFBVESDQEEcDRcsQoBCEAAAhCAgDcBdAgZYO8oyRtx4MABGTNmjEyePFm2bdsmp5xyikycOFHatWunbYvA00bFQAhAAAIQgAAEDBNAhyCAfYeUEr9K8D7xxBNy5JFHyqhRo2TVqlXy4YcfykEHHaRlj8DTwsQgCEAAAhCAAAQiIIAOQQD7Cqtvv/1WDj/8cLn//vvl2muvdeZu375dWrZsKb///e/l4osv1rJH4GlhYhAEIAABCEAAAhEQQIcggH2F1aJFi6RHjx7y8ccfS4cOHbJzVRlE586dncywzkXg6VBiDAQgAAEIQAACURBAhyCAfcXVjBkz5IILLpBdu3ZJvXr1snMHDhzo/N2cOXO07BF4WpgYBAEIQAACEIBABATQIQhgX2E1bdo0ueyyy2T//v1SVlaWnXv55ZfLxo0bZe7cua72duzYIep/mWv9+vXSq1cvURnlFi1a+PKBwRCAAAQgAAEIQCAMgc8//1y6d+8uq1evlqOOOiqMqZKbSxs0lyOdPn26XHjhha4Z4L1798qsWbNcA+Guu+5yOkdwQQACEIAABCAAAVsIqETcSSedZIs7VviBAHY5hkwN8CeffCLHHHNMdoSqAe7SpYtMmDBBKwO8Z88eUVng8vJyqVWrViQHnvnpjixzJHizRuEcLd9c67COhzWc4+GsVoF1PKzhXJXzvn37ZMuWLdKpUyepW7duPAeRklUQwC4HpbK8zZo1k4ceekiuvPJKZ0SQLhBxxAD1PXFQ5nWS8VD+2yrEdDy04RwPZ2IazvERYCU/BBDABWjdeuut8thjjzltz1TdzE033SRr1qyRDz74QLsPsJ+DCDqWf8SCkvM3D87+eIUZDesw9PTnwlmfVdiRsA5LUG8+nPU4MepvBBDABSJBPQB3yy23OC/C2L17t5x66qlO6YNtReR84eP5KsM5Hs5ky+AcH4H4VuL+EQ9rOMfDuVRWQQCn/CRV1wlVqjFixAhp0KBByndjr/twju9sYB0PazjHw1mtAut4WMM5Hs6lsgoCuFROkn1AAAIQgAAEIAABCGgRQABrYWIQBCAAAQhAAAIQgECpEEAAl8pJsg8IQAACEIAABCAAAS0CCGAtTAyCAAQgAAEIQAACECgVAgjgUjlJ9gEBCEAAAhCAAAQgoEUAAayFyb5BBw4ccF67PHnyZNm2bZuot9RNnDhR2rVrZ5+zFnu0detWp93dnDlznCe1O3fuLOPGjXN4qmvZsmUyfPhweffdd6VJkyYybNgwufHGG7M74hz8H+7KlSvlhBNOkEceeUSuuOIKOPtH6DnjqaeecuJ41apVztss1Wvaf/KTn8Dak5y/Ad99953DdurUqc59+Pjjj5fx48dLr169YO0PZcHR99xzj8ydO1f++Mc/ZseYuC972TDkPmYsJoAAtvhwirmmxK8SvKpP8ZFHHimjRo1y/rH78MMPrXpRh+14zzzzTNm8ebMjxpo2ber0elY/VCxZssT587HHHiv9+/d32swtWLBArrvuOof74MGDna1xDv5OWAkGJQ7UDxQqdpUA/vLLL+HsD2PR0dOmTXO4Pvjgg3LuuefKM88844i0t99+Wzp06ABrg6zvvPNO+e1vfytPPvmkHH300Y74ff755+V///d/pU6dOrAOyfrXv/61c+9VffgzAtjE/ULHRkjXmZ4CAgjgFBxSvovffvutHH744XL//ffLtdde63xs66uabcb7ySefSPv27eWdd97JZmwqKiockXDJJZdIvXr1HEGs3gBYq1YtZysqWzxjxgz56KOPhHPwf7qKn/pB4o033sgK4F/96ldw9o/SdYaKXyXELrzwQnnggQeyY370ox/JD3/4Q+fPxLQh2CJOxvf00093fthQl/otUsOGDeXFF18U9ZsOWAdjvWHDBrnqqqtk/vz50rp1a2nevHlWAJu4X3jZCOY1s9JGAAGcthMTkUWLFkmPHj3k448/dsRa5lK/tle/wlcZSi5vAuqHhj/96U/OP2AqW5O5VBnJWWed5WTUGzVqJE8//XT2M/WruL59+8qmTZscYcw5eHPOjHjrrbfk/PPPd8pK2rRpkxXA55xzDpz1MRYdqX4w+6d/+ifnNxhdu3atMhbWhkD/3Yy6d6i3j7366quOUHv44Yfltttuc34Td/311xPXAXG//PLLzm8uVPnD2LFjnXttJgNsIoa9bDRr1iyg50xLEwEEcJpO6+++qgzkBRdcILt27XKylJlr4MCBzt+pelauYAReeOEFURxnz54tt956q5x99tnOrzUz14oVK+S4446TxYsXy7p16zgHTczqh40uXbqIyrxceumlUqNGjawAVj+0wVkTpMcwFbf/8i//4mTO7r77blm6dKmUl5c7ouy8885zfkCGtRnWysry5cvloosuEnVfqFmzppSVlTnZ3379+sHaEGZVzpMrgE3EsJeNbt26GfIeMzYTQADbfDoFfFM1fpdddpns37/fueFmrssvv1w2btzoPDDA5Z+AKoVQmV+V1XnppZecBwqVWFMZiMyVeahICQx1U+Yc9DirkhL16/lnn33WmZArgOGsx1BnVObeoESvqk9VP3RMnz5d7r33Xnn99dflmmuuIaZ1QGqOUT8w/+Y3v5GbbrrJeRbjd7/7nTz33HOiftsxYMAAWGtyLDYsXwCbuF942cg8BG3AfUxYTAABbPHhFHJN/YOmavzcMsB79+6VWbNmpXBXybqsmCmxe/LJJzsZdJVZ79Spk6hflbllgNWvmJUY5hy8z009IT969Gj54IMP5LDDDqsigOHszVB3hHoAS2UkVRnUkCFDstPUw3DqUr+1IKZ1aRYft3btWucZgnnz5skPfvCD7GD13+oZDfWMAazDs84XwCbuF1423MqHwu8EC7YRQADbdiIa/mRqgNUNVrU4ylzqp1aV8VEPXnDpE1AdIFSrM5WxURm0TD2w+sdLtT5TAi5zudUAcw7FWffp08fpQJBbZ71z507nzyp+27ZtC2f9cC06UtW09+7d2ymByM1iqS4x6ge7o446CtaGWGfKpXbv3i1169bNWlXZ4P/+7/8mrg1xzhfAJu7LXjaoATZ0eJabQQBbfkBu7qksr/qCPvTQQ3LllVc6Q+gCEewgH330Uae1mervqx5gyS0pUX1UJ02aJJ9++qlT36culcmcOXOm0wWCc9Bjrp7oViIh91KZM5VZV9lK9bALnPVYeo1SnNUT83fccUelftWq7rd27dpOtxNYe1HU+1x1M+nZs6csXLhQunfvnp2kyqiUIFa/TYK1Hstio/IFsIn7speN8F5jIQ0EEMBpOCUXH9UDWo899pj8/ve/d7I6KuugalLVr5kPOuiglO4qXrdVm6KOHTvKj3/84yqdM1QJhBK4qg+weqBFZdBU5l39Wln9ozZo0CDHWc4h2Jnl1gCrPsxwDsbRbZZ6cl61SFT3ByXMVN21EsTqV/Xf//73YW0ItXoJjmott2XLFuf+0apVK1EvIFEPeqrfeKh2dMR1eNj5AtjE/ULHRnjPsWA7AQSw7SdUwD/1AJzqqapeJqCyPqpRuCp9UGKYS4/Afffd5whYt0sJ3ClTpjjdHlR2WD1N36JFCxk5cqQMHTo0O4Vz0GOdPypXAKvP4ByMY6FZ6rdDqrRHZd9VWzT1whbVHQLWZjmrt7+pDhuqvET9t6otVfeV0047DdaGUOcLYFMx7HXPMeQ+ZiwmgAC2+HBwDQIQgAAEIAABCEDAPAEEsHmmWIQABCAAAQhAAAIQsJgAAtjiw8E1CEAAAhCAAAQgAAHzBBDA5pliEQIQgAAEIAABCEDAYgIIYIsPB9cgAAEIQAACEIAABMwTQACbZ4pFCEAAAhCAAAQgAAGLCSCALT4cXIMABCAAAQhAAAIQME8AAWyeKRYhAAEIQAACEIAABCwmgAC2+HBwDQIQgAAEIAABCEDAPAEEsHmmWIQABEqEgHrV7ZtvvllpN+pV4+qtgP3793dee6tem23qUm9yVG++uuuuu0yZrGInfw31Vr7cq06dOs5rfS+66CLnTYn169ePzBcMQwACEEiKAAI4KfKsCwEIWE9ACeDmzZvLb37zm6yv33zzjbz22mtyww03yNVXX+28ctjUtWXLFkdQH3LIIaZMagngX//6147gVZfa36JFi+TGG290XqP86quvSq1atSLzB8MQgAAEkiCAAE6COmtCAAKpIKAEsMqYTpkypYq/P//5z2XWrFny5ZdfpmIvGSfdMsBPPPGEk3nOvd577z056aSTZPLkyaL2ygUBCECglAgggEvpNNkLBCBglEAxAXzdddfJiy++KJs3b3bW/Pbbb+X222+XadOmyVdffSUdO3aUsWPHyplnnpn1SWWOR48eLStWrJCjjz5aRo4cKVdeeaWsXr3aEdr54vS//uu/5O6775bly5dLgwYN5JJLLpF7771X6tat69hU5Qu//e1v5fnnn5e3335bGjduLEOHDnXWKHTpCmA1/9RTTxVV8jF37lyjXDEGAQhAIGkCCOCkT4D1IQABawm4CeB9+/Y5ZQEXX3yxUwLx4IMPOv5feuml8uGHH4oqJ1A1tC+//LL827/9m8ycOVPOPfdcWbZsmZNR/eUvf+mIXvXnIUOGyLZt21wF8EsvvSQXXHCBUw88cOBA+b//+z9nfPfu3WX69OlZAdywYUOnDEP9/dSpU+Wee+6Rt956S37wgx+4cvUjgP/1X/9VnnvuOfniiy+sPSMcgwAEIBCEAAI4CDXmQAAC1YKAEsDvvPOOqAfDMtfu3bulbdu2TsnALbfc4tTHfvLJJ9K+fXtZvHixdOvWLTt20KBBsnbtWvnjH/8o6r9Xrlwpf/7zn7Of/+d//qcMGzbMVQD36NFDWrdu7WSZM5cS1f369XMyyKo+V2WAhw8f7ojuzKWywDfddFPBLLAfAawegrv//vvlu+++qxbnzSYhAIHqQwABXH3Omp1CAAI+CSgB3KRJExk/frwcOHBAFi5c6GRwBwwYIBMnTsw+HPbCCy84WdqDDz640gpKODZq1Eg2bdoknTp1kr59+8pDDz2UHaOywF27dnUVwKr7wn333ec8bJe5VGmFsvfss886D60pATxp0iS55pprsmOUwB08eLDceeedrrv1I4BVOYVaiwywz8BhOAQgYD0BBLD1R4SDEIBAUgTcSiDeeOMNOeOMMxzRqUSwulSZgCqJWLJkiRx66KGV3K1Zs6aUl5c7Qve0006rlK0tJoBVN4hx48Y5Gd7MtXXrVkeQqxIIJcKVAM5/gM2rlZofAdyzZ09nP6p2mQsCEIBAKRFAAJfSabIXCEDAKIFCD8GNGjVKHnjgAXnllVfkrLPOcmp/1UNvc+bMcep9M5cqISgrK3MeZFMlEx999JEsWLAg+/ljjz0m1157rWsGWNX0tmnTplIJhOo6ofoP55ZARCWAVRcIVc6hOmCo8g0uCEAAAqVEAAFcSqfJXiAAAaMECgngPXv2SOfOnWXv3r2O+FV9e8877zznwbYJEyY4YnjGjBmihPLjjz/ulCSoTg7HH3+8019XtRVTIlY9RKd6/65Zs8apK87NzqqyClXmMGbMGOf/Vf2weghO2VC1wOoylQF26wOsHuBr166dvP7666Ky2FwQgAAESokAAriUTpO9QAACRgkUa4Om3hDXp08fuf76650XZezatct5c5oqh1ClCqrN2YgRI+Sqq67K+qSEq2pRpjo6fO9738t2edi4caPzdrn88oRnnnnGedvcxx9/LE2bNnU6TShBnHn7nCkBnAtNtVtTJRtqLfWAXqblmlGwGIMABCCQMAEEcMIHwPIQgED1IKA6RKiOEaoWOHM9/fTTTjZYvX2Nt61VjzhglxCAgB0EEMB2nANeQAACJU7gd7/7ndOe7KmnnnLKGFTrNPUgXe/evV3fNFfiONgeBCAAgUQJIIATxc/iEIBAdSFQUVHhPAynHirbsGGDNGvWzHmzW25JQ3VhwT4hAAEIJE0AAZz0CbA+BCAAAQhAAAIQgECsBBDAseJmMQhAAAIQgAAEIACBpAkggJM+AdaHAAQgAAEIQAACEIiVAAI4VtwsBgEIQAACEIAABCCQNAEEcNInwPoQgAAEIAABCEAAArESQADHipvFIAABCEAAAhCAAASSJoAATvoEWB8CEIAABCAAAQhAIFYCCOBYcbMYBCAAAQhAAAIQgEDSBBDASZ8A60MAAhCAAAQgAAEIxEoAARwrbhaDAAQgAAEIQAACEEiaAAI46RNgfQhAAAIQgAAEIACBWAkggGPFzWIQgAAEIAABCEAAAkkTQAAnfQKsDwEIQAACEIAABCAQK4H/D3CICLjKFWEuAAAAAElFTkSuQmCC\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pb_num_dict = {_k:len(_v.kept_probes) for _k,_v in filtered_pb_dict.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "# number of probes histogram\n",
    "fig, ax = plt.subplots()\n",
    "ax.plot(pb_nums, '.')\n",
    "ax.set_ylabel(\"Num probes per region\")\n",
    "ax.set_xlabel(\"Region ID\")\n",
    "ax.set_title(\"Probe number after filtering\")\n",
    "ax.set_ylim([0,100])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 167,
   "id": "619614a2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\chr_2_final_region_num.pkl\n"
     ]
    }
   ],
   "source": [
    "chr_2_region_savefile = os.path.join(library_folder, 'chr_2_final_region_num.pkl')\n",
    "overwrite = True\n",
    "\n",
    "filtered_reg_names = {}\n",
    "for _reg, _pbde in filtered_pb_dict.items():\n",
    "    if len(_pbde.kept_probes) > 0:\n",
    "        filtered_reg_names[_reg] = list(_pbde.kept_probes.values())[0]['reg_name']\n",
    "\n",
    "if not os.path.exists(chr_2_region_savefile) or overwrite:\n",
    "    chr_2_region_num = {}\n",
    "    for _reg, _info in filtered_reg_names.items():\n",
    "        _chr = _info.split('_')[0].split(':')[0]\n",
    "        if _chr not in chr_2_region_num:\n",
    "            chr_2_region_num[_chr] = 1\n",
    "        else:\n",
    "            chr_2_region_num[_chr] += 1\n",
    "    print(f\"Saving into file: {chr_2_region_savefile}\")\n",
    "\n",
    "    pickle.dump(chr_2_region_num, open(chr_2_region_savefile, 'wb'))\n",
    "else:\n",
    "    print(f\"loading from file: {chr_2_region_savefile}\")\n",
    "    chr_2_region_num = pickle.load(open(chr_2_region_savefile, 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 168,
   "id": "1b0275cd",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'7': 1097}"
      ]
     },
     "execution_count": 168,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chr_2_region_num"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 170,
   "id": "8c304389",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving region names into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\chr_2_region_names.pkl\n"
     ]
    }
   ],
   "source": [
    "# \n",
    "reg_2_start = {}\n",
    "\n",
    "for _name, _info in filtered_reg_names.items():\n",
    "    _locus = _info.split('_')[0]\n",
    "    _start,_end = _locus.split(':')[1].split('-')\n",
    "    reg_2_start[_name] = int(_start)\n",
    "#    \n",
    "chr_2_region_names = {}\n",
    "\n",
    "for _reg, _info in filtered_reg_names.items():\n",
    "    _chr = _info.split('_')[0].split(':')[0]\n",
    "    if _chr not in chr_2_region_names:\n",
    "        chr_2_region_names[_chr] = [_reg]\n",
    "    else:\n",
    "        chr_2_region_names[_chr].append(_reg)\n",
    "# sort\n",
    "for _chr, _regs in chr_2_region_names.items():\n",
    "    _sorted_regs = sorted(_regs, key=lambda _r: reg_2_start[_r])\n",
    "    chr_2_region_names[_chr] = _sorted_regs\n",
    "    \n",
    "overwrite_region_names = True\n",
    "chr_2_region_name_savefile = os.path.join(library_folder, 'chr_2_region_names.pkl')\n",
    "\n",
    "if not os.path.exists(chr_2_region_name_savefile) or overwrite_region_names:\n",
    "    print(f\"Saving region names into file: {chr_2_region_name_savefile}\")\n",
    "    pickle.dump(chr_2_region_names, open(chr_2_region_name_savefile, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2c3d9549",
   "metadata": {},
   "source": [
    "<a id='3'></a>\n",
    "# 3. Load and asssemble probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d373567a",
   "metadata": {},
   "source": [
    "<a id='3.1'></a>\n",
    "## 3.1 design readout mapping dict (region_to_readout)\n",
    "\n",
    "load directly from http://localhost:8889/notebooks/Library_design/CTP-14-human_brain_genome/Design-codebook_long_sequential.ipynb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 180,
   "id": "136b756f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1097"
      ]
     },
     "execution_count": 180,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# summarize total readout usage\n",
    "region_2_readout_dict = pickle.load(open(os.path.join(library_folder, 'region_2_readout.pkl'), 'rb'))\n",
    "region_2_readout_dict = {str(_r):_bs for _r,_bs in region_2_readout_dict.items()}\n",
    "len(region_2_readout_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 181,
   "id": "23097f52",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Chr7-6Mb-seg-1': ['c11', 'c13', 'c16'],\n",
       " 'Chr7-6Mb-seg-2': ['c30', 'c32', 'c35'],\n",
       " 'Chr7-6Mb-seg-3': ['c49', 'c51', 'c54'],\n",
       " 'Chr7-6Mb-seg-4': ['c68', 'c70', 'c73'],\n",
       " 'Chr7-6Mb-seg-7': ['c87', 'c89', 'c92'],\n",
       " 'Chr7-6Mb-seg-9': ['c106', 'c108', 'c111'],\n",
       " 'Chr7-6Mb-seg-10': ['c125', 'c127', 'c130'],\n",
       " 'Chr7-6Mb-seg-11': ['c144', 'c146', 'c149'],\n",
       " 'Chr7-6Mb-seg-12': ['c163', 'c165', 'c168'],\n",
       " 'Chr7-6Mb-seg-14': ['c179', 'c185', 'c186'],\n",
       " 'Chr7-6Mb-seg-15': ['c0', 'c2', 'c18'],\n",
       " 'Chr7-6Mb-seg-16': ['c19', 'c21', 'c37'],\n",
       " 'Chr7-6Mb-seg-19': ['c38', 'c40', 'c56'],\n",
       " 'Chr7-6Mb-seg-20': ['c57', 'c59', 'c75'],\n",
       " 'Chr7-6Mb-seg-21': ['c76', 'c78', 'c94'],\n",
       " 'Chr7-6Mb-seg-22': ['c95', 'c97', 'c113'],\n",
       " 'Chr7-6Mb-seg-23': ['c114', 'c116', 'c132'],\n",
       " 'Chr7-6Mb-seg-24': ['c133', 'c135', 'c151'],\n",
       " 'Chr7-6Mb-seg-25': ['c152', 'c154', 'c170'],\n",
       " 'Chr7-6Mb-seg-27': ['c174', 'c175', 'c187'],\n",
       " 'Chr7-6Mb-seg-28': ['c3', 'c5', 'c6'],\n",
       " 'Chr7-6Mb-seg-29': ['c22', 'c24', 'c25'],\n",
       " 'Chr7-6Mb-seg-36': ['c41', 'c43', 'c44'],\n",
       " 'Chr7-6Mb-seg-38': ['c60', 'c62', 'c63'],\n",
       " 'Chr7-6Mb-seg-39': ['c79', 'c81', 'c82'],\n",
       " 'Chr7-6Mb-seg-40': ['c98', 'c100', 'c101'],\n",
       " 'Chr7-6Mb-seg-41': ['c117', 'c119', 'c120'],\n",
       " 'Chr7-6Mb-seg-42': ['c136', 'c138', 'c139'],\n",
       " 'Chr7-6Mb-seg-43': ['c155', 'c157', 'c158'],\n",
       " 'Chr7-6Mb-seg-44': ['c177', 'c178', 'c183'],\n",
       " 'Chr7-6Mb-seg-45': ['c9', 'c12', 'c17'],\n",
       " 'Chr7-6Mb-seg-46': ['c28', 'c31', 'c36'],\n",
       " 'Chr7-6Mb-seg-47': ['c47', 'c50', 'c55'],\n",
       " 'Chr7-6Mb-seg-48': ['c66', 'c69', 'c74'],\n",
       " 'Chr7-6Mb-seg-49': ['c85', 'c88', 'c93'],\n",
       " 'Chr7-6Mb-seg-50': ['c104', 'c107', 'c112'],\n",
       " 'Chr7-6Mb-seg-51': ['c123', 'c126', 'c131'],\n",
       " 'Chr7-6Mb-seg-52': ['c142', 'c145', 'c150'],\n",
       " 'Chr7-6Mb-seg-53': ['c161', 'c164', 'c169'],\n",
       " 'Chr7-6Mb-seg-54': ['c173', 'c176', 'c180'],\n",
       " 'Chr7-6Mb-seg-55': ['c4', 'c7', 'c8'],\n",
       " 'Chr7-6Mb-seg-56': ['c23', 'c26', 'c27'],\n",
       " 'Chr7-6Mb-seg-57': ['c42', 'c45', 'c46'],\n",
       " 'Chr7-6Mb-seg-58': ['c61', 'c64', 'c65'],\n",
       " 'Chr7-6Mb-seg-59': ['c80', 'c83', 'c84'],\n",
       " 'Chr7-6Mb-seg-60': ['c99', 'c102', 'c103'],\n",
       " 'Chr7-6Mb-seg-61': ['c118', 'c121', 'c122'],\n",
       " 'Chr7-6Mb-seg-62': ['c137', 'c140', 'c141'],\n",
       " 'Chr7-6Mb-seg-63': ['c156', 'c159', 'c160'],\n",
       " 'Chr7-6Mb-seg-64': ['c171', 'c181', 'c182'],\n",
       " 'Chr7-6Mb-seg-65': ['c13', 'c14', 'c18'],\n",
       " 'Chr7-6Mb-seg-66': ['c32', 'c33', 'c37'],\n",
       " 'Chr7-6Mb-seg-67': ['c51', 'c52', 'c56'],\n",
       " 'Chr7-6Mb-seg-68': ['c70', 'c71', 'c75'],\n",
       " 'Chr7-6Mb-seg-69': ['c89', 'c90', 'c94'],\n",
       " 'Chr7-6Mb-seg-70': ['c108', 'c109', 'c113'],\n",
       " 'Chr7-6Mb-seg-71': ['c127', 'c128', 'c132'],\n",
       " 'Chr7-6Mb-seg-72': ['c146', 'c147', 'c151'],\n",
       " 'Chr7-6Mb-seg-73': ['c165', 'c166', 'c170'],\n",
       " 'Chr7-6Mb-seg-74': ['c175', 'c178', 'c179'],\n",
       " 'Chr7-6Mb-seg-75': ['c0', 'c1', 'c3'],\n",
       " 'Chr7-6Mb-seg-76': ['c19', 'c20', 'c22'],\n",
       " 'Chr7-6Mb-seg-77': ['c38', 'c39', 'c41'],\n",
       " 'Chr7-6Mb-seg-78': ['c57', 'c58', 'c60'],\n",
       " 'Chr7-6Mb-seg-79': ['c76', 'c77', 'c79'],\n",
       " 'Chr7-6Mb-seg-80': ['c95', 'c96', 'c98'],\n",
       " 'Chr7-6Mb-seg-81': ['c114', 'c115', 'c117'],\n",
       " 'Chr7-6Mb-seg-82': ['c133', 'c134', 'c136'],\n",
       " 'Chr7-6Mb-seg-83': ['c152', 'c153', 'c155'],\n",
       " 'Chr7-6Mb-seg-84': ['c172', 'c184', 'c188'],\n",
       " 'Chr7-6Mb-seg-85': ['c2', 'c6', 'c11'],\n",
       " 'Chr7-6Mb-seg-86': ['c21', 'c25', 'c30'],\n",
       " 'Chr7-6Mb-seg-87': ['c40', 'c44', 'c49'],\n",
       " 'Chr7-6Mb-seg-88': ['c59', 'c63', 'c68'],\n",
       " 'Chr7-6Mb-seg-89': ['c78', 'c82', 'c87'],\n",
       " 'Chr7-6Mb-seg-90': ['c97', 'c101', 'c106'],\n",
       " 'Chr7-6Mb-seg-91': ['c116', 'c120', 'c125'],\n",
       " 'Chr7-6Mb-seg-92': ['c135', 'c139', 'c144'],\n",
       " 'Chr7-6Mb-seg-93': ['c154', 'c158', 'c163'],\n",
       " 'Chr7-6Mb-seg-94': ['c173', 'c186', 'c187'],\n",
       " 'Chr7-6Mb-seg-95': ['c5', 'c10', 'c15'],\n",
       " 'Chr7-6Mb-seg-96': ['c24', 'c29', 'c34'],\n",
       " 'Chr7-6Mb-seg-97': ['c43', 'c48', 'c53'],\n",
       " 'Chr7-6Mb-seg-98': ['c62', 'c67', 'c72'],\n",
       " 'Chr7-6Mb-seg-99': ['c81', 'c86', 'c91'],\n",
       " 'Chr7-6Mb-seg-100': ['c100', 'c105', 'c110'],\n",
       " 'Chr7-6Mb-seg-101': ['c119', 'c124', 'c129'],\n",
       " 'Chr7-6Mb-seg-102': ['c138', 'c143', 'c148'],\n",
       " 'Chr7-6Mb-seg-103': ['c157', 'c162', 'c167'],\n",
       " 'Chr7-6Mb-seg-104': ['c174', 'c178', 'c189'],\n",
       " 'Chr7-6Mb-seg-105': ['c4', 'c9', 'c18'],\n",
       " 'Chr7-6Mb-seg-106': ['c23', 'c28', 'c37'],\n",
       " 'Chr7-6Mb-seg-107': ['c42', 'c47', 'c56'],\n",
       " 'Chr7-6Mb-seg-108': ['c61', 'c66', 'c75'],\n",
       " 'Chr7-6Mb-seg-109': ['c80', 'c85', 'c94'],\n",
       " 'Chr7-6Mb-seg-110': ['c99', 'c104', 'c113'],\n",
       " 'Chr7-6Mb-seg-111': ['c118', 'c123', 'c132'],\n",
       " 'Chr7-6Mb-seg-112': ['c137', 'c142', 'c151'],\n",
       " 'Chr7-6Mb-seg-113': ['c156', 'c161', 'c170'],\n",
       " 'Chr7-6Mb-seg-114': ['c176', 'c179', 'c188'],\n",
       " 'Chr7-6Mb-seg-115': ['c1', 'c8', 'c16'],\n",
       " 'Chr7-6Mb-seg-116': ['c20', 'c27', 'c35'],\n",
       " 'Chr7-6Mb-seg-117': ['c39', 'c46', 'c54'],\n",
       " 'Chr7-6Mb-seg-118': ['c58', 'c65', 'c73'],\n",
       " 'Chr7-6Mb-seg-119': ['c77', 'c84', 'c92'],\n",
       " 'Chr7-6Mb-seg-120': ['c96', 'c103', 'c111'],\n",
       " 'Chr7-6Mb-seg-121': ['c115', 'c122', 'c130'],\n",
       " 'Chr7-6Mb-seg-122': ['c134', 'c141', 'c149'],\n",
       " 'Chr7-6Mb-seg-123': ['c153', 'c160', 'c168'],\n",
       " 'Chr7-6Mb-seg-124': ['c180', 'c182', 'c186'],\n",
       " 'Chr7-6Mb-seg-125': ['c0', 'c14', 'c17'],\n",
       " 'Chr7-6Mb-seg-126': ['c19', 'c33', 'c36'],\n",
       " 'Chr7-6Mb-seg-127': ['c38', 'c52', 'c55'],\n",
       " 'Chr7-6Mb-seg-128': ['c57', 'c71', 'c74'],\n",
       " 'Chr7-6Mb-seg-129': ['c76', 'c90', 'c93'],\n",
       " 'Chr7-6Mb-seg-130': ['c95', 'c109', 'c112'],\n",
       " 'Chr7-6Mb-seg-131': ['c114', 'c128', 'c131'],\n",
       " 'Chr7-6Mb-seg-132': ['c133', 'c147', 'c150'],\n",
       " 'Chr7-6Mb-seg-133': ['c152', 'c166', 'c169'],\n",
       " 'Chr7-6Mb-seg-134': ['c175', 'c181', 'c184'],\n",
       " 'Chr7-6Mb-seg-135': ['c4', 'c10', 'c13'],\n",
       " 'Chr7-6Mb-seg-136': ['c23', 'c29', 'c32'],\n",
       " 'Chr7-6Mb-seg-137': ['c42', 'c48', 'c51'],\n",
       " 'Chr7-6Mb-seg-138': ['c61', 'c67', 'c70'],\n",
       " 'Chr7-6Mb-seg-139': ['c80', 'c86', 'c89'],\n",
       " 'Chr7-6Mb-seg-140': ['c99', 'c105', 'c108'],\n",
       " 'Chr7-6Mb-seg-141': ['c118', 'c124', 'c127'],\n",
       " 'Chr7-6Mb-seg-142': ['c137', 'c143', 'c146'],\n",
       " 'Chr7-6Mb-seg-143': ['c156', 'c162', 'c165'],\n",
       " 'Chr7-6Mb-seg-144': ['c172', 'c177', 'c185'],\n",
       " 'Chr7-6Mb-seg-145': ['c6', 'c7', 'c12'],\n",
       " 'Chr7-6Mb-seg-146': ['c25', 'c26', 'c31'],\n",
       " 'Chr7-6Mb-seg-147': ['c44', 'c45', 'c50'],\n",
       " 'Chr7-6Mb-seg-148': ['c63', 'c64', 'c69'],\n",
       " 'Chr7-6Mb-seg-149': ['c82', 'c83', 'c88'],\n",
       " 'Chr7-6Mb-seg-150': ['c101', 'c102', 'c107'],\n",
       " 'Chr7-6Mb-seg-151': ['c120', 'c121', 'c126'],\n",
       " 'Chr7-6Mb-seg-152': ['c139', 'c140', 'c145'],\n",
       " 'Chr7-6Mb-seg-153': ['c158', 'c159', 'c164'],\n",
       " 'Chr7-6Mb-seg-154': ['c173', 'c174', 'c188'],\n",
       " 'Chr7-6Mb-seg-155': ['c2', 'c5', 'c9'],\n",
       " 'Chr7-6Mb-seg-156': ['c21', 'c24', 'c28'],\n",
       " 'Chr7-6Mb-seg-157': ['c40', 'c43', 'c47'],\n",
       " 'Chr7-6Mb-seg-158': ['c59', 'c62', 'c66'],\n",
       " 'Chr7-6Mb-seg-159': ['c78', 'c81', 'c85'],\n",
       " 'Chr7-6Mb-seg-161': ['c97', 'c100', 'c104'],\n",
       " 'Chr7-6Mb-seg-162': ['c116', 'c119', 'c123'],\n",
       " 'Chr7-6Mb-seg-163': ['c135', 'c138', 'c142'],\n",
       " 'Chr7-6Mb-seg-164': ['c154', 'c157', 'c161'],\n",
       " 'Chr7-6Mb-seg-165': ['c176', 'c182', 'c189'],\n",
       " 'Chr7-6Mb-seg-166': ['c0', 'c10', 'c11'],\n",
       " 'Chr7-6Mb-seg-167': ['c19', 'c29', 'c30'],\n",
       " 'Chr7-6Mb-seg-168': ['c38', 'c48', 'c49'],\n",
       " 'Chr7-6Mb-seg-169': ['c57', 'c67', 'c68'],\n",
       " 'Chr7-6Mb-seg-170': ['c76', 'c86', 'c87'],\n",
       " 'Chr7-6Mb-seg-171': ['c95', 'c105', 'c106'],\n",
       " 'Chr7-6Mb-seg-172': ['c114', 'c124', 'c125'],\n",
       " 'Chr7-6Mb-seg-173': ['c133', 'c143', 'c144'],\n",
       " 'Chr7-6Mb-seg-174': ['c152', 'c162', 'c163'],\n",
       " 'Chr7-6Mb-seg-175': ['c171', 'c175', 'c177'],\n",
       " 'Chr7-6Mb-seg-176': ['c1', 'c13', 'c17'],\n",
       " 'Chr7-6Mb-seg-177': ['c20', 'c32', 'c36'],\n",
       " 'Chr7-6Mb-seg-178': ['c39', 'c51', 'c55'],\n",
       " 'Chr7-6Mb-seg-179': ['c58', 'c70', 'c74'],\n",
       " 'Chr7-6Mb-seg-180': ['c77', 'c89', 'c93'],\n",
       " 'Chr7-6Mb-seg-181': ['c96', 'c108', 'c112'],\n",
       " 'Chr7-6Mb-seg-182': ['c115', 'c127', 'c131'],\n",
       " 'Chr7-6Mb-seg-183': ['c134', 'c146', 'c150'],\n",
       " 'Chr7-6Mb-seg-184': ['c153', 'c165', 'c169'],\n",
       " 'Chr7-6Mb-seg-185': ['c179', 'c183', 'c184'],\n",
       " 'Chr7-6Mb-seg-186': ['c3', 'c4', 'c16'],\n",
       " 'Chr7-6Mb-seg-187': ['c22', 'c23', 'c35'],\n",
       " 'Chr7-6Mb-seg-188': ['c41', 'c42', 'c54'],\n",
       " 'Chr7-6Mb-seg-189': ['c60', 'c61', 'c73'],\n",
       " 'Chr7-6Mb-seg-190': ['c79', 'c80', 'c92'],\n",
       " 'Chr7-6Mb-seg-191': ['c98', 'c99', 'c111'],\n",
       " 'Chr7-6Mb-seg-192': ['c117', 'c118', 'c130'],\n",
       " 'Chr7-6Mb-seg-193': ['c136', 'c137', 'c149'],\n",
       " 'Chr7-6Mb-seg-194': ['c155', 'c156', 'c168'],\n",
       " 'Chr7-6Mb-seg-195': ['c178', 'c181', 'c187'],\n",
       " 'Chr7-6Mb-seg-196': ['c8', 'c14', 'c15'],\n",
       " 'Chr7-6Mb-seg-197': ['c27', 'c33', 'c34'],\n",
       " 'Chr7-6Mb-seg-198': ['c46', 'c52', 'c53'],\n",
       " 'Chr7-6Mb-seg-199': ['c65', 'c71', 'c72'],\n",
       " 'Chr7-6Mb-seg-200': ['c84', 'c90', 'c91'],\n",
       " 'Chr7-6Mb-seg-201': ['c103', 'c109', 'c110'],\n",
       " 'Chr7-6Mb-seg-202': ['c122', 'c128', 'c129'],\n",
       " 'Chr7-6Mb-seg-203': ['c141', 'c147', 'c148'],\n",
       " 'Chr7-6Mb-seg-204': ['c160', 'c166', 'c167'],\n",
       " 'Chr7-6Mb-seg-205': ['c175', 'c186', 'c188'],\n",
       " 'Chr7-6Mb-seg-206': ['c5', 'c11', 'c18'],\n",
       " 'Chr7-6Mb-seg-207': ['c24', 'c30', 'c37'],\n",
       " 'Chr7-6Mb-seg-208': ['c43', 'c49', 'c56'],\n",
       " 'Chr7-6Mb-seg-209': ['c62', 'c68', 'c75'],\n",
       " 'Chr7-6Mb-seg-210': ['c81', 'c87', 'c94'],\n",
       " 'Chr7-6Mb-seg-211': ['c100', 'c106', 'c113'],\n",
       " 'Chr7-6Mb-seg-212': ['c119', 'c125', 'c132'],\n",
       " 'Chr7-6Mb-seg-213': ['c138', 'c144', 'c151'],\n",
       " 'Chr7-6Mb-seg-214': ['c157', 'c163', 'c170'],\n",
       " 'Chr7-6Mb-seg-215': ['c171', 'c172', 'c174'],\n",
       " 'Chr7-6Mb-seg-216': ['c2', 'c3', 'c17'],\n",
       " 'Chr7-6Mb-seg-217': ['c21', 'c22', 'c36'],\n",
       " 'Chr7-6Mb-seg-218': ['c40', 'c41', 'c55'],\n",
       " 'Chr7-6Mb-seg-219': ['c59', 'c60', 'c74'],\n",
       " 'Chr7-6Mb-seg-220': ['c78', 'c79', 'c93'],\n",
       " 'Chr7-6Mb-seg-221': ['c97', 'c98', 'c112'],\n",
       " 'Chr7-6Mb-seg-222': ['c116', 'c117', 'c131'],\n",
       " 'Chr7-6Mb-seg-223': ['c135', 'c136', 'c150'],\n",
       " 'Chr7-6Mb-seg-224': ['c154', 'c155', 'c169'],\n",
       " 'Chr7-6Mb-seg-225': ['c184', 'c185', 'c189'],\n",
       " 'Chr7-6Mb-seg-226': ['c1', 'c9', 'c10'],\n",
       " 'Chr7-6Mb-seg-227': ['c20', 'c28', 'c29'],\n",
       " 'Chr7-6Mb-seg-228': ['c39', 'c47', 'c48'],\n",
       " 'Chr7-6Mb-seg-229': ['c58', 'c66', 'c67'],\n",
       " 'Chr7-6Mb-seg-230': ['c77', 'c85', 'c86'],\n",
       " 'Chr7-6Mb-seg-231': ['c96', 'c104', 'c105'],\n",
       " 'Chr7-6Mb-seg-232': ['c115', 'c123', 'c124'],\n",
       " 'Chr7-6Mb-seg-233': ['c134', 'c142', 'c143'],\n",
       " 'Chr7-6Mb-seg-234': ['c153', 'c161', 'c162'],\n",
       " 'Chr7-6Mb-seg-235': ['c180', 'c183', 'c188'],\n",
       " 'Chr7-6Mb-seg-236': ['c4', 'c5', 'c12'],\n",
       " 'Chr7-6Mb-seg-237': ['c23', 'c24', 'c31'],\n",
       " 'Chr7-6Mb-seg-238': ['c42', 'c43', 'c50'],\n",
       " 'Chr7-6Mb-seg-239': ['c61', 'c62', 'c69'],\n",
       " 'Chr7-6Mb-seg-240': ['c80', 'c81', 'c88'],\n",
       " 'Chr7-6Mb-seg-241': ['c99', 'c100', 'c107'],\n",
       " 'Chr7-6Mb-seg-242': ['c118', 'c119', 'c126'],\n",
       " 'Chr7-6Mb-seg-243': ['c137', 'c138', 'c145'],\n",
       " 'Chr7-6Mb-seg-244': ['c156', 'c157', 'c164'],\n",
       " 'Chr7-6Mb-seg-245': ['c176', 'c181', 'c186'],\n",
       " 'Chr7-6Mb-seg-246': ['c3', 'c7', 'c18'],\n",
       " 'Chr7-6Mb-seg-247': ['c22', 'c26', 'c37'],\n",
       " 'Chr7-6Mb-seg-248': ['c41', 'c45', 'c56'],\n",
       " 'Chr7-6Mb-seg-249': ['c60', 'c64', 'c75'],\n",
       " 'Chr7-6Mb-seg-250': ['c79', 'c83', 'c94'],\n",
       " 'Chr7-6Mb-seg-251': ['c98', 'c102', 'c113'],\n",
       " 'Chr7-6Mb-seg-252': ['c117', 'c121', 'c132'],\n",
       " 'Chr7-6Mb-seg-253': ['c136', 'c140', 'c151'],\n",
       " 'Chr7-6Mb-seg-254': ['c155', 'c159', 'c170'],\n",
       " 'Chr7-6Mb-seg-255': ['c173', 'c177', 'c182'],\n",
       " 'Chr7-6Mb-seg-256': ['c2', 'c15', 'c16'],\n",
       " 'Chr7-6Mb-seg-257': ['c21', 'c34', 'c35'],\n",
       " 'Chr7-6Mb-seg-258': ['c40', 'c53', 'c54'],\n",
       " 'Chr7-6Mb-seg-259': ['c59', 'c72', 'c73'],\n",
       " 'Chr7-6Mb-seg-260': ['c78', 'c91', 'c92'],\n",
       " 'Chr7-6Mb-seg-261': ['c97', 'c110', 'c111'],\n",
       " 'Chr7-6Mb-seg-262': ['c116', 'c129', 'c130'],\n",
       " 'Chr7-6Mb-seg-263': ['c135', 'c148', 'c149'],\n",
       " 'Chr7-6Mb-seg-264': ['c154', 'c167', 'c168'],\n",
       " 'Chr7-6Mb-seg-265': ['c178', 'c180', 'c185'],\n",
       " 'Chr7-6Mb-seg-266': ['c8', 'c12', 'c13'],\n",
       " 'Chr7-6Mb-seg-267': ['c27', 'c31', 'c32'],\n",
       " 'Chr7-6Mb-seg-268': ['c46', 'c50', 'c51'],\n",
       " 'Chr7-6Mb-seg-269': ['c65', 'c69', 'c70'],\n",
       " 'Chr7-6Mb-seg-270': ['c84', 'c88', 'c89'],\n",
       " 'Chr7-6Mb-seg-271': ['c103', 'c107', 'c108'],\n",
       " 'Chr7-6Mb-seg-272': ['c122', 'c126', 'c127'],\n",
       " 'Chr7-6Mb-seg-273': ['c141', 'c145', 'c146'],\n",
       " 'Chr7-6Mb-seg-274': ['c160', 'c164', 'c165'],\n",
       " 'Chr7-6Mb-seg-275': ['c172', 'c179', 'c187'],\n",
       " 'Chr7-6Mb-seg-276': ['c0', 'c4', 'c6'],\n",
       " 'Chr7-6Mb-seg-277': ['c19', 'c23', 'c25'],\n",
       " 'Chr7-6Mb-seg-278': ['c38', 'c42', 'c44'],\n",
       " 'Chr7-6Mb-seg-279': ['c57', 'c61', 'c63'],\n",
       " 'Chr7-6Mb-seg-280': ['c76', 'c80', 'c82'],\n",
       " 'Chr7-6Mb-seg-281': ['c95', 'c99', 'c101'],\n",
       " 'Chr7-6Mb-seg-282': ['c114', 'c118', 'c120'],\n",
       " 'Chr7-6Mb-seg-283': ['c133', 'c137', 'c139'],\n",
       " 'Chr7-6Mb-seg-284': ['c152', 'c156', 'c158'],\n",
       " 'Chr7-6Mb-seg-285': ['c174', 'c183', 'c186'],\n",
       " 'Chr7-6Mb-seg-286': ['c7', 'c11', 'c17'],\n",
       " 'Chr7-6Mb-seg-287': ['c26', 'c30', 'c36'],\n",
       " 'Chr7-6Mb-seg-288': ['c45', 'c49', 'c55'],\n",
       " 'Chr7-6Mb-seg-289': ['c64', 'c68', 'c74'],\n",
       " 'Chr7-6Mb-seg-290': ['c83', 'c87', 'c93'],\n",
       " 'Chr7-6Mb-seg-291': ['c102', 'c106', 'c112'],\n",
       " 'Chr7-6Mb-seg-292': ['c121', 'c125', 'c131'],\n",
       " 'Chr7-6Mb-seg-293': ['c140', 'c144', 'c150'],\n",
       " 'Chr7-6Mb-seg-294': ['c159', 'c163', 'c169'],\n",
       " 'Chr7-6Mb-seg-295': ['c177', 'c181', 'c188'],\n",
       " 'Chr7-6Mb-seg-296': ['c3', 'c12', 'c15'],\n",
       " 'Chr7-6Mb-seg-297': ['c22', 'c31', 'c34'],\n",
       " 'Chr7-6Mb-seg-298': ['c41', 'c50', 'c53'],\n",
       " 'Chr7-6Mb-seg-299': ['c60', 'c69', 'c72'],\n",
       " 'Chr7-6Mb-seg-300': ['c79', 'c88', 'c91'],\n",
       " 'Chr7-6Mb-seg-301': ['c98', 'c107', 'c110'],\n",
       " 'Chr7-6Mb-seg-302': ['c117', 'c126', 'c129'],\n",
       " 'Chr7-6Mb-seg-303': ['c136', 'c145', 'c148'],\n",
       " 'Chr7-6Mb-seg-304': ['c155', 'c164', 'c167'],\n",
       " 'Chr7-6Mb-seg-305': ['c171', 'c176', 'c184'],\n",
       " 'Chr7-6Mb-seg-306': ['c5', 'c14', 'c16'],\n",
       " 'Chr7-6Mb-seg-307': ['c24', 'c33', 'c35'],\n",
       " 'Chr7-6Mb-seg-308': ['c43', 'c52', 'c54'],\n",
       " 'Chr7-6Mb-seg-309': ['c62', 'c71', 'c73'],\n",
       " 'Chr7-6Mb-seg-310': ['c81', 'c90', 'c92'],\n",
       " 'Chr7-6Mb-seg-311': ['c100', 'c109', 'c111'],\n",
       " 'Chr7-6Mb-seg-312': ['c119', 'c128', 'c130'],\n",
       " 'Chr7-6Mb-seg-313': ['c138', 'c147', 'c149'],\n",
       " 'Chr7-6Mb-seg-314': ['c157', 'c166', 'c168'],\n",
       " 'Chr7-6Mb-seg-315': ['c175', 'c180', 'c189'],\n",
       " 'Chr7-6Mb-seg-316': ['c2', 'c7', 'c13'],\n",
       " 'Chr7-6Mb-seg-317': ['c21', 'c26', 'c32'],\n",
       " 'Chr7-6Mb-seg-318': ['c40', 'c45', 'c51'],\n",
       " 'Chr7-6Mb-seg-319': ['c59', 'c64', 'c70'],\n",
       " 'Chr7-6Mb-seg-320': ['c78', 'c83', 'c89'],\n",
       " 'Chr7-6Mb-seg-321': ['c97', 'c102', 'c108'],\n",
       " 'Chr7-6Mb-seg-322': ['c116', 'c121', 'c127'],\n",
       " 'Chr7-6Mb-seg-323': ['c135', 'c140', 'c146'],\n",
       " 'Chr7-6Mb-seg-324': ['c154', 'c159', 'c165'],\n",
       " 'Chr7-6Mb-seg-325': ['c173', 'c183', 'c185'],\n",
       " 'Chr7-6Mb-seg-326': ['c0', 'c8', 'c9'],\n",
       " 'Chr7-6Mb-seg-327': ['c19', 'c27', 'c28'],\n",
       " 'Chr7-6Mb-seg-328': ['c38', 'c46', 'c47'],\n",
       " 'Chr7-6Mb-seg-329': ['c57', 'c65', 'c66'],\n",
       " 'Chr7-6Mb-seg-330': ['c76', 'c84', 'c85'],\n",
       " 'Chr7-6Mb-seg-331': ['c95', 'c103', 'c104'],\n",
       " 'Chr7-6Mb-seg-332': ['c114', 'c122', 'c123'],\n",
       " 'Chr7-6Mb-seg-333': ['c133', 'c141', 'c142'],\n",
       " 'Chr7-6Mb-seg-334': ['c152', 'c160', 'c161'],\n",
       " 'Chr7-6Mb-seg-335': ['c178', 'c182', 'c188'],\n",
       " 'Chr7-6Mb-seg-336': ['c6', 'c10', 'c17'],\n",
       " 'Chr7-6Mb-seg-337': ['c25', 'c29', 'c36'],\n",
       " 'Chr7-6Mb-seg-338': ['c44', 'c48', 'c55'],\n",
       " 'Chr7-6Mb-seg-339': ['c63', 'c67', 'c74'],\n",
       " 'Chr7-6Mb-seg-340': ['c82', 'c86', 'c93'],\n",
       " 'Chr7-6Mb-seg-341': ['c101', 'c105', 'c112'],\n",
       " 'Chr7-6Mb-seg-342': ['c120', 'c124', 'c131'],\n",
       " 'Chr7-6Mb-seg-343': ['c139', 'c143', 'c150'],\n",
       " 'Chr7-6Mb-seg-344': ['c158', 'c162', 'c169'],\n",
       " 'Chr7-6Mb-seg-345': ['c177', 'c180', 'c187'],\n",
       " 'Chr7-6Mb-seg-346': ['c4', 'c11', 'c14'],\n",
       " 'Chr7-6Mb-seg-347': ['c23', 'c30', 'c33'],\n",
       " 'Chr7-6Mb-seg-348': ['c42', 'c49', 'c52'],\n",
       " 'Chr7-6Mb-seg-349': ['c61', 'c68', 'c71'],\n",
       " 'Chr7-6Mb-seg-350': ['c80', 'c87', 'c90'],\n",
       " 'Chr7-6Mb-seg-351': ['c99', 'c106', 'c109'],\n",
       " 'Chr7-6Mb-seg-352': ['c118', 'c125', 'c128'],\n",
       " 'Chr7-6Mb-seg-353': ['c137', 'c144', 'c147'],\n",
       " 'Chr7-6Mb-seg-354': ['c156', 'c163', 'c166'],\n",
       " 'Chr7-6Mb-seg-355': ['c173', 'c179', 'c181'],\n",
       " 'Chr7-6Mb-seg-356': ['c1', 'c15', 'c18'],\n",
       " 'Chr7-6Mb-seg-357': ['c20', 'c34', 'c37'],\n",
       " 'Chr7-6Mb-seg-358': ['c39', 'c53', 'c56'],\n",
       " 'Chr7-6Mb-seg-359': ['c58', 'c72', 'c75'],\n",
       " 'Chr7-6Mb-seg-360': ['c77', 'c91', 'c94'],\n",
       " 'Chr7-6Mb-seg-361': ['c96', 'c110', 'c113'],\n",
       " 'Chr7-6Mb-seg-362': ['c115', 'c129', 'c132'],\n",
       " 'Chr7-6Mb-seg-363': ['c134', 'c148', 'c151'],\n",
       " 'Chr7-6Mb-seg-364': ['c153', 'c167', 'c170'],\n",
       " 'Chr7-6Mb-seg-365': ['c172', 'c186', 'c189'],\n",
       " 'Chr7-6Mb-seg-366': ['c3', 'c9', 'c13'],\n",
       " 'Chr7-6Mb-seg-367': ['c22', 'c28', 'c32'],\n",
       " 'Chr7-6Mb-seg-368': ['c41', 'c47', 'c51'],\n",
       " 'Chr7-6Mb-seg-369': ['c60', 'c66', 'c70'],\n",
       " 'Chr7-6Mb-seg-370': ['c79', 'c85', 'c89'],\n",
       " 'Chr7-6Mb-seg-371': ['c98', 'c104', 'c108'],\n",
       " 'Chr7-6Mb-seg-372': ['c117', 'c123', 'c127'],\n",
       " 'Chr7-6Mb-seg-373': ['c136', 'c142', 'c146'],\n",
       " 'Chr7-6Mb-seg-374': ['c155', 'c161', 'c165'],\n",
       " 'Chr7-6Mb-seg-375': ['c171', 'c183', 'c187'],\n",
       " 'Chr7-6Mb-seg-376': ['c2', 'c12', 'c14'],\n",
       " 'Chr7-6Mb-seg-377': ['c21', 'c31', 'c33'],\n",
       " 'Chr7-6Mb-seg-378': ['c40', 'c50', 'c52'],\n",
       " 'Chr7-6Mb-seg-379': ['c59', 'c69', 'c71'],\n",
       " 'Chr7-6Mb-seg-380': ['c78', 'c88', 'c90'],\n",
       " 'Chr7-6Mb-seg-381': ['c97', 'c107', 'c109'],\n",
       " 'Chr7-6Mb-seg-382': ['c116', 'c126', 'c128'],\n",
       " 'Chr7-6Mb-seg-383': ['c135', 'c145', 'c147'],\n",
       " 'Chr7-6Mb-seg-384': ['c154', 'c164', 'c166'],\n",
       " 'Chr7-6Mb-seg-385': ['c175', 'c182', 'c185'],\n",
       " 'Chr7-6Mb-seg-386': ['c7', 'c10', 'c16'],\n",
       " 'Chr7-6Mb-seg-387': ['c26', 'c29', 'c35'],\n",
       " 'Chr7-6Mb-seg-388': ['c45', 'c48', 'c54'],\n",
       " 'Chr7-6Mb-seg-389': ['c64', 'c67', 'c73'],\n",
       " 'Chr7-6Mb-seg-390': ['c83', 'c86', 'c92'],\n",
       " 'Chr7-6Mb-seg-391': ['c102', 'c105', 'c111'],\n",
       " 'Chr7-6Mb-seg-392': ['c121', 'c124', 'c130'],\n",
       " 'Chr7-6Mb-seg-393': ['c140', 'c143', 'c149'],\n",
       " 'Chr7-6Mb-seg-394': ['c159', 'c162', 'c168'],\n",
       " 'Chr7-6Mb-seg-395': ['c174', 'c176', 'c177'],\n",
       " 'Chr7-6Mb-seg-396': ['c5', 'c8', 'c17'],\n",
       " 'Chr7-6Mb-seg-397': ['c24', 'c27', 'c36'],\n",
       " 'Chr7-6Mb-seg-398': ['c43', 'c46', 'c55'],\n",
       " 'Chr7-6Mb-seg-399': ['c62', 'c65', 'c74'],\n",
       " 'Chr7-6Mb-seg-400': ['c81', 'c84', 'c93'],\n",
       " 'Chr7-6Mb-seg-401': ['c100', 'c103', 'c112'],\n",
       " 'Chr7-6Mb-seg-402': ['c119', 'c122', 'c131'],\n",
       " 'Chr7-6Mb-seg-403': ['c138', 'c141', 'c150'],\n",
       " 'Chr7-6Mb-seg-404': ['c157', 'c160', 'c169'],\n",
       " 'Chr7-6Mb-seg-405': ['c171', 'c179', 'c180'],\n",
       " 'Chr7-6Mb-seg-406': ['c1', 'c2', 'c4'],\n",
       " 'Chr7-6Mb-seg-407': ['c20', 'c21', 'c23'],\n",
       " 'Chr7-6Mb-seg-408': ['c39', 'c40', 'c42'],\n",
       " 'Chr7-6Mb-seg-409': ['c58', 'c59', 'c61'],\n",
       " 'Chr7-6Mb-seg-410': ['c77', 'c78', 'c80'],\n",
       " 'Chr7-6Mb-seg-411': ['c96', 'c97', 'c99'],\n",
       " 'Chr7-6Mb-seg-412': ['c115', 'c116', 'c118'],\n",
       " 'Chr7-6Mb-seg-413': ['c134', 'c135', 'c137'],\n",
       " 'Chr7-6Mb-seg-414': ['c153', 'c154', 'c156'],\n",
       " 'Chr7-6Mb-seg-415': ['c173', 'c178', 'c184'],\n",
       " 'Chr7-6Mb-seg-416': ['c9', 'c11', 'c15'],\n",
       " 'Chr7-6Mb-seg-417': ['c28', 'c30', 'c34'],\n",
       " 'Chr7-6Mb-seg-418': ['c47', 'c49', 'c53'],\n",
       " 'Chr7-6Mb-seg-419': ['c66', 'c68', 'c72'],\n",
       " 'Chr7-6Mb-seg-420': ['c85', 'c87', 'c91'],\n",
       " 'Chr7-6Mb-seg-421': ['c104', 'c106', 'c110'],\n",
       " 'Chr7-6Mb-seg-422': ['c123', 'c125', 'c129'],\n",
       " 'Chr7-6Mb-seg-423': ['c142', 'c144', 'c148'],\n",
       " 'Chr7-6Mb-seg-424': ['c161', 'c163', 'c167'],\n",
       " 'Chr7-6Mb-seg-425': ['c187', 'c188', 'c189'],\n",
       " 'Chr7-6Mb-seg-426': ['c0', 'c12', 'c16'],\n",
       " 'Chr7-6Mb-seg-427': ['c19', 'c31', 'c35'],\n",
       " 'Chr7-6Mb-seg-428': ['c38', 'c50', 'c54'],\n",
       " 'Chr7-6Mb-seg-429': ['c57', 'c69', 'c73'],\n",
       " 'Chr7-6Mb-seg-430': ['c76', 'c88', 'c92'],\n",
       " 'Chr7-6Mb-seg-431': ['c95', 'c107', 'c111'],\n",
       " 'Chr7-6Mb-seg-432': ['c114', 'c126', 'c130'],\n",
       " 'Chr7-6Mb-seg-433': ['c133', 'c145', 'c149'],\n",
       " 'Chr7-6Mb-seg-434': ['c152', 'c164', 'c168'],\n",
       " 'Chr7-6Mb-seg-435': ['c174', 'c181', 'c185'],\n",
       " 'Chr7-6Mb-seg-436': ['c1', 'c5', 'c7'],\n",
       " 'Chr7-6Mb-seg-437': ['c20', 'c24', 'c26'],\n",
       " 'Chr7-6Mb-seg-438': ['c39', 'c43', 'c45'],\n",
       " 'Chr7-6Mb-seg-439': ['c58', 'c62', 'c64'],\n",
       " 'Chr7-6Mb-seg-440': ['c77', 'c81', 'c83'],\n",
       " 'Chr7-6Mb-seg-441': ['c96', 'c100', 'c102'],\n",
       " 'Chr7-6Mb-seg-442': ['c115', 'c119', 'c121'],\n",
       " 'Chr7-6Mb-seg-443': ['c134', 'c138', 'c140'],\n",
       " 'Chr7-6Mb-seg-444': ['c153', 'c157', 'c159'],\n",
       " 'Chr7-6Mb-seg-445': ['c172', 'c182', 'c183'],\n",
       " 'Chr7-6Mb-seg-446': ['c3', 'c10', 'c14'],\n",
       " 'Chr7-6Mb-seg-447': ['c22', 'c29', 'c33'],\n",
       " 'Chr7-6Mb-seg-448': ['c41', 'c48', 'c52'],\n",
       " 'Chr7-6Mb-seg-449': ['c60', 'c67', 'c71'],\n",
       " 'Chr7-6Mb-seg-450': ['c79', 'c86', 'c90'],\n",
       " 'Chr7-6Mb-seg-451': ['c98', 'c105', 'c109'],\n",
       " 'Chr7-6Mb-seg-452': ['c117', 'c124', 'c128'],\n",
       " 'Chr7-6Mb-seg-453': ['c136', 'c143', 'c147'],\n",
       " 'Chr7-6Mb-seg-454': ['c155', 'c162', 'c166'],\n",
       " 'Chr7-6Mb-seg-455': ['c171', 'c178', 'c186'],\n",
       " 'Chr7-6Mb-seg-456': ['c6', 'c13', 'c15'],\n",
       " 'Chr7-6Mb-seg-457': ['c25', 'c32', 'c34'],\n",
       " 'Chr7-6Mb-seg-458': ['c44', 'c51', 'c53'],\n",
       " 'Chr7-6Mb-seg-459': ['c63', 'c70', 'c72'],\n",
       " 'Chr7-6Mb-seg-460': ['c82', 'c89', 'c91'],\n",
       " 'Chr7-6Mb-seg-461': ['c101', 'c108', 'c110'],\n",
       " 'Chr7-6Mb-seg-462': ['c120', 'c127', 'c129'],\n",
       " 'Chr7-6Mb-seg-463': ['c139', 'c146', 'c148'],\n",
       " 'Chr7-6Mb-seg-464': ['c158', 'c165', 'c167'],\n",
       " 'Chr7-6Mb-seg-465': ['c177', 'c179', 'c189'],\n",
       " 'Chr7-6Mb-seg-466': ['c16', 'c17', 'c18'],\n",
       " 'Chr7-6Mb-seg-467': ['c35', 'c36', 'c37'],\n",
       " 'Chr7-6Mb-seg-468': ['c54', 'c55', 'c56'],\n",
       " 'Chr7-6Mb-seg-469': ['c73', 'c74', 'c75'],\n",
       " 'Chr7-6Mb-seg-470': ['c92', 'c93', 'c94'],\n",
       " 'Chr7-6Mb-seg-471': ['c111', 'c112', 'c113'],\n",
       " 'Chr7-6Mb-seg-472': ['c130', 'c131', 'c132'],\n",
       " 'Chr7-6Mb-seg-473': ['c149', 'c150', 'c151'],\n",
       " 'Chr7-6Mb-seg-474': ['c168', 'c169', 'c170'],\n",
       " 'Chr7-6Mb-seg-475': ['c174', 'c180', 'c184'],\n",
       " 'Chr7-6Mb-seg-476': ['c2', 'c8', 'c10'],\n",
       " 'Chr7-6Mb-seg-477': ['c21', 'c27', 'c29'],\n",
       " 'Chr7-6Mb-seg-478': ['c40', 'c46', 'c48'],\n",
       " 'Chr7-6Mb-seg-479': ['c59', 'c65', 'c67'],\n",
       " 'Chr7-6Mb-seg-480': ['c78', 'c84', 'c86'],\n",
       " 'Chr7-6Mb-seg-481': ['c97', 'c103', 'c105'],\n",
       " 'Chr7-6Mb-seg-482': ['c116', 'c122', 'c124'],\n",
       " 'Chr7-6Mb-seg-483': ['c135', 'c141', 'c143'],\n",
       " 'Chr7-6Mb-seg-484': ['c154', 'c160', 'c162'],\n",
       " 'Chr7-6Mb-seg-485': ['c176', 'c185', 'c187'],\n",
       " 'Chr7-6Mb-seg-486': ['c7', 'c9', 'c14'],\n",
       " 'Chr7-6Mb-seg-487': ['c26', 'c28', 'c33'],\n",
       " 'Chr7-6Mb-seg-488': ['c45', 'c47', 'c52'],\n",
       " 'Chr7-6Mb-seg-489': ['c64', 'c66', 'c71'],\n",
       " 'Chr7-6Mb-seg-490': ['c83', 'c85', 'c90'],\n",
       " 'Chr7-6Mb-seg-491': ['c102', 'c104', 'c109'],\n",
       " 'Chr7-6Mb-seg-492': ['c121', 'c123', 'c128'],\n",
       " 'Chr7-6Mb-seg-493': ['c140', 'c142', 'c147'],\n",
       " 'Chr7-6Mb-seg-494': ['c159', 'c161', 'c166'],\n",
       " 'Chr7-6Mb-seg-495': ['c172', 'c173', 'c175'],\n",
       " 'Chr7-6Mb-seg-496': ['c1', 'c11', 'c12'],\n",
       " 'Chr7-6Mb-seg-497': ['c20', 'c30', 'c31'],\n",
       " 'Chr7-6Mb-seg-498': ['c39', 'c49', 'c50'],\n",
       " 'Chr7-6Mb-seg-499': ['c58', 'c68', 'c69'],\n",
       " 'Chr7-6Mb-seg-500': ['c77', 'c87', 'c88'],\n",
       " 'Chr7-6Mb-seg-501': ['c96', 'c106', 'c107'],\n",
       " 'Chr7-6Mb-seg-502': ['c115', 'c125', 'c126'],\n",
       " 'Chr7-6Mb-seg-503': ['c134', 'c144', 'c145'],\n",
       " 'Chr7-6Mb-seg-504': ['c153', 'c163', 'c164'],\n",
       " 'Chr7-6Mb-seg-505': ['c181', 'c183', 'c189'],\n",
       " 'Chr7-6Mb-seg-506': ['c6', 'c8', 'c18'],\n",
       " 'Chr7-6Mb-seg-507': ['c25', 'c27', 'c37'],\n",
       " 'Chr7-6Mb-seg-508': ['c44', 'c46', 'c56'],\n",
       " 'Chr7-6Mb-seg-509': ['c63', 'c65', 'c75'],\n",
       " 'Chr7-6Mb-seg-510': ['c82', 'c84', 'c94'],\n",
       " 'Chr7-6Mb-seg-511': ['c101', 'c103', 'c113'],\n",
       " 'Chr7-6Mb-seg-512': ['c120', 'c122', 'c132'],\n",
       " 'Chr7-6Mb-seg-513': ['c139', 'c141', 'c151'],\n",
       " 'Chr7-6Mb-seg-514': ['c158', 'c160', 'c170'],\n",
       " 'Chr7-6Mb-seg-515': ['c177', 'c184', 'c186'],\n",
       " 'Chr7-6Mb-seg-516': ['c4', 'c15', 'c17'],\n",
       " 'Chr7-6Mb-seg-517': ['c23', 'c34', 'c36'],\n",
       " 'Chr7-6Mb-seg-518': ['c42', 'c53', 'c55'],\n",
       " 'Chr7-6Mb-seg-519': ['c61', 'c72', 'c74'],\n",
       " 'Chr7-6Mb-seg-520': ['c80', 'c91', 'c93'],\n",
       " 'Chr7-6Mb-seg-521': ['c99', 'c110', 'c112'],\n",
       " 'Chr7-6Mb-seg-522': ['c118', 'c129', 'c131'],\n",
       " 'Chr7-6Mb-seg-523': ['c137', 'c148', 'c150'],\n",
       " 'Chr7-6Mb-seg-524': ['c156', 'c167', 'c169'],\n",
       " 'Chr7-6Mb-seg-525': ['c174', 'c179', 'c182'],\n",
       " 'Chr7-6Mb-seg-526': ['c0', 'c5', 'c13'],\n",
       " 'Chr7-6Mb-seg-527': ['c19', 'c24', 'c32'],\n",
       " 'Chr7-6Mb-seg-528': ['c38', 'c43', 'c51'],\n",
       " 'Chr7-6Mb-seg-529': ['c57', 'c62', 'c70'],\n",
       " 'Chr7-6Mb-seg-530': ['c76', 'c81', 'c89'],\n",
       " 'Chr7-6Mb-seg-531': ['c95', 'c100', 'c108'],\n",
       " 'Chr7-6Mb-seg-532': ['c114', 'c119', 'c127'],\n",
       " 'Chr7-6Mb-seg-533': ['c133', 'c138', 'c146'],\n",
       " 'Chr7-6Mb-seg-534': ['c152', 'c157', 'c165'],\n",
       " 'Chr7-6Mb-seg-535': ['c171', 'c185', 'c188'],\n",
       " 'Chr7-6Mb-seg-536': ['c1', 'c6', 'c14'],\n",
       " 'Chr7-6Mb-seg-537': ['c20', 'c25', 'c33'],\n",
       " 'Chr7-6Mb-seg-538': ['c39', 'c44', 'c52'],\n",
       " 'Chr7-6Mb-seg-539': ['c58', 'c63', 'c71'],\n",
       " 'Chr7-6Mb-seg-540': ['c77', 'c82', 'c90'],\n",
       " 'Chr7-6Mb-seg-541': ['c96', 'c101', 'c109'],\n",
       " 'Chr7-6Mb-seg-542': ['c115', 'c120', 'c128'],\n",
       " 'Chr7-6Mb-seg-543': ['c134', 'c139', 'c147'],\n",
       " 'Chr7-6Mb-seg-544': ['c153', 'c158', 'c166'],\n",
       " 'Chr7-6Mb-seg-545': ['c175', 'c176', 'c183'],\n",
       " 'Chr7-6Mb-seg-546': ['c3', 'c8', 'c11'],\n",
       " 'Chr7-6Mb-seg-547': ['c22', 'c27', 'c30'],\n",
       " 'Chr7-6Mb-seg-548': ['c41', 'c46', 'c49'],\n",
       " 'Chr7-6Mb-seg-549': ['c60', 'c65', 'c68'],\n",
       " 'Chr7-6Mb-seg-550': ['c79', 'c84', 'c87'],\n",
       " 'Chr7-6Mb-seg-551': ['c98', 'c103', 'c106'],\n",
       " 'Chr7-6Mb-seg-552': ['c117', 'c122', 'c125'],\n",
       " 'Chr7-6Mb-seg-553': ['c136', 'c141', 'c144'],\n",
       " 'Chr7-6Mb-seg-554': ['c155', 'c160', 'c163'],\n",
       " 'Chr7-6Mb-seg-555': ['c172', 'c180', 'c181'],\n",
       " 'Chr7-6Mb-seg-556': ['c10', 'c12', 'c18'],\n",
       " 'Chr7-6Mb-seg-557': ['c29', 'c31', 'c37'],\n",
       " 'Chr7-6Mb-seg-558': ['c48', 'c50', 'c56'],\n",
       " 'Chr7-6Mb-seg-559': ['c67', 'c69', 'c75'],\n",
       " 'Chr7-6Mb-seg-560': ['c86', 'c88', 'c94'],\n",
       " 'Chr7-6Mb-seg-561': ['c105', 'c107', 'c113'],\n",
       " 'Chr7-6Mb-seg-562': ['c124', 'c126', 'c132'],\n",
       " 'Chr7-6Mb-seg-563': ['c143', 'c145', 'c151'],\n",
       " 'Chr7-6Mb-seg-564': ['c162', 'c164', 'c170'],\n",
       " 'Chr7-6Mb-seg-565': ['c201', 'c203', 'c206'],\n",
       " 'Chr7-6Mb-seg-566': ['c220', 'c222', 'c225'],\n",
       " 'Chr7-6Mb-seg-567': ['c239', 'c241', 'c244'],\n",
       " 'Chr7-6Mb-seg-568': ['c258', 'c260', 'c263'],\n",
       " 'Chr7-6Mb-seg-569': ['c277', 'c279', 'c282'],\n",
       " 'Chr7-6Mb-seg-570': ['c296', 'c298', 'c301'],\n",
       " 'Chr7-6Mb-seg-571': ['c315', 'c317', 'c320'],\n",
       " 'Chr7-6Mb-seg-572': ['c334', 'c336', 'c339'],\n",
       " 'Chr7-6Mb-seg-573': ['c350', 'c356', 'c357'],\n",
       " 'Chr7-6Mb-seg-574': ['c369', 'c375', 'c376'],\n",
       " 'Chr7-6Mb-seg-575': ['c190', 'c192', 'c208'],\n",
       " 'Chr7-6Mb-seg-576': ['c209', 'c211', 'c227'],\n",
       " 'Chr7-6Mb-seg-577': ['c228', 'c230', 'c246'],\n",
       " 'Chr7-6Mb-seg-578': ['c247', 'c249', 'c265'],\n",
       " 'Chr7-6Mb-seg-579': ['c266', 'c268', 'c284'],\n",
       " 'Chr7-6Mb-seg-580': ['c285', 'c287', 'c303'],\n",
       " 'Chr7-6Mb-seg-581': ['c304', 'c306', 'c322'],\n",
       " 'Chr7-6Mb-seg-582': ['c323', 'c325', 'c341'],\n",
       " 'Chr7-6Mb-seg-583': ['c345', 'c346', 'c358'],\n",
       " 'Chr7-6Mb-seg-584': ['c364', 'c365', 'c377'],\n",
       " 'Chr7-6Mb-seg-585': ['c193', 'c195', 'c196'],\n",
       " 'Chr7-6Mb-seg-586': ['c212', 'c214', 'c215'],\n",
       " 'Chr7-6Mb-seg-587': ['c231', 'c233', 'c234'],\n",
       " 'Chr7-6Mb-seg-588': ['c250', 'c252', 'c253'],\n",
       " 'Chr7-6Mb-seg-589': ['c269', 'c271', 'c272'],\n",
       " 'Chr7-6Mb-seg-590': ['c288', 'c290', 'c291'],\n",
       " 'Chr7-6Mb-seg-591': ['c307', 'c309', 'c310'],\n",
       " 'Chr7-6Mb-seg-592': ['c326', 'c328', 'c329'],\n",
       " 'Chr7-6Mb-seg-593': ['c348', 'c349', 'c354'],\n",
       " 'Chr7-6Mb-seg-594': ['c367', 'c368', 'c373'],\n",
       " 'Chr7-6Mb-seg-595': ['c199', 'c202', 'c207'],\n",
       " 'Chr7-6Mb-seg-596': ['c218', 'c221', 'c226'],\n",
       " 'Chr7-6Mb-seg-597': ['c237', 'c240', 'c245'],\n",
       " 'Chr7-6Mb-seg-598': ['c256', 'c259', 'c264'],\n",
       " 'Chr7-6Mb-seg-599': ['c275', 'c278', 'c283'],\n",
       " 'Chr7-6Mb-seg-600': ['c294', 'c297', 'c302'],\n",
       " 'Chr7-6Mb-seg-601': ['c313', 'c316', 'c321'],\n",
       " 'Chr7-6Mb-seg-602': ['c332', 'c335', 'c340'],\n",
       " 'Chr7-6Mb-seg-603': ['c344', 'c347', 'c351'],\n",
       " 'Chr7-6Mb-seg-604': ['c363', 'c366', 'c370'],\n",
       " 'Chr7-6Mb-seg-605': ['c194', 'c197', 'c198'],\n",
       " 'Chr7-6Mb-seg-606': ['c213', 'c216', 'c217'],\n",
       " 'Chr7-6Mb-seg-607': ['c232', 'c235', 'c236'],\n",
       " 'Chr7-6Mb-seg-608': ['c251', 'c254', 'c255'],\n",
       " 'Chr7-6Mb-seg-609': ['c270', 'c273', 'c274'],\n",
       " 'Chr7-6Mb-seg-610': ['c289', 'c292', 'c293'],\n",
       " 'Chr7-6Mb-seg-611': ['c308', 'c311', 'c312'],\n",
       " 'Chr7-6Mb-seg-612': ['c327', 'c330', 'c331'],\n",
       " 'Chr7-6Mb-seg-613': ['c342', 'c352', 'c353'],\n",
       " 'Chr7-6Mb-seg-614': ['c361', 'c371', 'c372'],\n",
       " 'Chr7-6Mb-seg-615': ['c203', 'c204', 'c208'],\n",
       " 'Chr7-6Mb-seg-616': ['c222', 'c223', 'c227'],\n",
       " 'Chr7-6Mb-seg-617': ['c241', 'c242', 'c246'],\n",
       " 'Chr7-6Mb-seg-618': ['c260', 'c261', 'c265'],\n",
       " 'Chr7-6Mb-seg-619': ['c279', 'c280', 'c284'],\n",
       " 'Chr7-6Mb-seg-620': ['c298', 'c299', 'c303'],\n",
       " 'Chr7-6Mb-seg-621': ['c317', 'c318', 'c322'],\n",
       " 'Chr7-6Mb-seg-622': ['c336', 'c337', 'c341'],\n",
       " 'Chr7-6Mb-seg-623': ['c346', 'c349', 'c350'],\n",
       " 'Chr7-6Mb-seg-624': ['c365', 'c368', 'c369'],\n",
       " 'Chr7-6Mb-seg-625': ['c190', 'c191', 'c193'],\n",
       " 'Chr7-6Mb-seg-626': ['c209', 'c210', 'c212'],\n",
       " 'Chr7-6Mb-seg-627': ['c228', 'c229', 'c231'],\n",
       " 'Chr7-6Mb-seg-628': ['c247', 'c248', 'c250'],\n",
       " 'Chr7-6Mb-seg-629': ['c266', 'c267', 'c269'],\n",
       " 'Chr7-6Mb-seg-630': ['c285', 'c286', 'c288'],\n",
       " 'Chr7-6Mb-seg-631': ['c304', 'c305', 'c307'],\n",
       " 'Chr7-6Mb-seg-632': ['c323', 'c324', 'c326'],\n",
       " 'Chr7-6Mb-seg-633': ['c343', 'c355', 'c359'],\n",
       " 'Chr7-6Mb-seg-634': ['c362', 'c374', 'c378'],\n",
       " 'Chr7-6Mb-seg-635': ['c192', 'c196', 'c201'],\n",
       " 'Chr7-6Mb-seg-638': ['c211', 'c215', 'c220'],\n",
       " 'Chr7-6Mb-seg-639': ['c230', 'c234', 'c239'],\n",
       " 'Chr7-6Mb-seg-640': ['c249', 'c253', 'c258'],\n",
       " 'Chr7-6Mb-seg-641': ['c268', 'c272', 'c277'],\n",
       " 'Chr7-6Mb-seg-642': ['c287', 'c291', 'c296'],\n",
       " 'Chr7-6Mb-seg-643': ['c306', 'c310', 'c315'],\n",
       " 'Chr7-6Mb-seg-644': ['c325', 'c329', 'c334'],\n",
       " 'Chr7-6Mb-seg-647': ['c344', 'c357', 'c358'],\n",
       " 'Chr7-6Mb-seg-648': ['c363', 'c376', 'c377'],\n",
       " 'Chr7-6Mb-seg-649': ['c195', 'c200', 'c205'],\n",
       " 'Chr7-6Mb-seg-650': ['c214', 'c219', 'c224'],\n",
       " 'Chr7-6Mb-seg-651': ['c233', 'c238', 'c243'],\n",
       " 'Chr7-6Mb-seg-655': ['c252', 'c257', 'c262'],\n",
       " 'Chr7-6Mb-seg-660': ['c271', 'c276', 'c281'],\n",
       " 'Chr7-6Mb-seg-661': ['c290', 'c295', 'c300'],\n",
       " 'Chr7-6Mb-seg-662': ['c309', 'c314', 'c319'],\n",
       " 'Chr7-6Mb-seg-663': ['c328', 'c333', 'c338'],\n",
       " 'Chr7-6Mb-seg-664': ['c345', 'c349', 'c360'],\n",
       " 'Chr7-6Mb-seg-665': ['c364', 'c368', 'c379'],\n",
       " 'Chr7-6Mb-seg-668': ['c194', 'c199', 'c208'],\n",
       " 'Chr7-6Mb-seg-669': ['c213', 'c218', 'c227'],\n",
       " 'Chr7-6Mb-seg-670': ['c232', 'c237', 'c246'],\n",
       " 'Chr7-6Mb-seg-671': ['c251', 'c256', 'c265'],\n",
       " 'Chr7-6Mb-seg-672': ['c270', 'c275', 'c284'],\n",
       " 'Chr7-6Mb-seg-674': ['c289', 'c294', 'c303'],\n",
       " 'Chr7-6Mb-seg-675': ['c308', 'c313', 'c322'],\n",
       " 'Chr7-6Mb-seg-676': ['c327', 'c332', 'c341'],\n",
       " 'Chr7-6Mb-seg-677': ['c347', 'c350', 'c359'],\n",
       " 'Chr7-6Mb-seg-679': ['c366', 'c369', 'c378'],\n",
       " 'Chr7-6Mb-seg-680': ['c191', 'c198', 'c206'],\n",
       " 'Chr7-6Mb-seg-681': ['c210', 'c217', 'c225'],\n",
       " 'Chr7-6Mb-seg-682': ['c229', 'c236', 'c244'],\n",
       " 'Chr7-6Mb-seg-683': ['c248', 'c255', 'c263'],\n",
       " 'Chr7-6Mb-seg-692': ['c267', 'c274', 'c282'],\n",
       " 'Chr7-6Mb-seg-693': ['c286', 'c293', 'c301'],\n",
       " 'Chr7-6Mb-seg-698': ['c305', 'c312', 'c320'],\n",
       " 'Chr7-6Mb-seg-699': ['c324', 'c331', 'c339'],\n",
       " 'Chr7-6Mb-seg-700': ['c351', 'c353', 'c357'],\n",
       " 'Chr7-6Mb-seg-701': ['c370', 'c372', 'c376'],\n",
       " 'Chr7-6Mb-seg-703': ['c190', 'c204', 'c207'],\n",
       " 'Chr7-6Mb-seg-704': ['c209', 'c223', 'c226'],\n",
       " 'Chr7-6Mb-seg-705': ['c228', 'c242', 'c245'],\n",
       " 'Chr7-6Mb-seg-706': ['c247', 'c261', 'c264'],\n",
       " 'Chr7-6Mb-seg-707': ['c266', 'c280', 'c283'],\n",
       " 'Chr7-6Mb-seg-708': ['c285', 'c299', 'c302'],\n",
       " 'Chr7-6Mb-seg-710': ['c304', 'c318', 'c321'],\n",
       " 'Chr7-6Mb-seg-711': ['c323', 'c337', 'c340'],\n",
       " 'Chr7-6Mb-seg-712': ['c346', 'c352', 'c355'],\n",
       " 'Chr7-6Mb-seg-713': ['c365', 'c371', 'c374'],\n",
       " 'Chr7-6Mb-seg-717': ['c194', 'c200', 'c203'],\n",
       " 'Chr7-6Mb-seg-718': ['c213', 'c219', 'c222'],\n",
       " 'Chr7-6Mb-seg-719': ['c232', 'c238', 'c241'],\n",
       " 'Chr7-6Mb-seg-720': ['c251', 'c257', 'c260'],\n",
       " 'Chr7-6Mb-seg-721': ['c270', 'c276', 'c279'],\n",
       " 'Chr7-6Mb-seg-722': ['c289', 'c295', 'c298'],\n",
       " 'Chr7-6Mb-seg-723': ['c308', 'c314', 'c317'],\n",
       " 'Chr7-6Mb-seg-726': ['c327', 'c333', 'c336'],\n",
       " 'Chr7-6Mb-seg-727': ['c343', 'c348', 'c356'],\n",
       " 'Chr7-6Mb-seg-728': ['c362', 'c367', 'c375'],\n",
       " 'Chr7-6Mb-seg-731': ['c196', 'c197', 'c202'],\n",
       " 'Chr7-6Mb-seg-732': ['c215', 'c216', 'c221'],\n",
       " 'Chr7-6Mb-seg-733': ['c234', 'c235', 'c240'],\n",
       " 'Chr7-6Mb-seg-734': ['c253', 'c254', 'c259'],\n",
       " 'Chr7-6Mb-seg-735': ['c272', 'c273', 'c278'],\n",
       " 'Chr7-6Mb-seg-738': ['c291', 'c292', 'c297'],\n",
       " 'Chr7-6Mb-seg-739': ['c310', 'c311', 'c316'],\n",
       " 'Chr7-6Mb-seg-740': ['c329', 'c330', 'c335'],\n",
       " 'Chr7-6Mb-seg-741': ['c344', 'c345', 'c359'],\n",
       " 'Chr7-6Mb-seg-742': ['c363', 'c364', 'c378'],\n",
       " 'Chr7-6Mb-seg-743': ['c192', 'c195', 'c199'],\n",
       " 'Chr7-6Mb-seg-744': ['c211', 'c214', 'c218'],\n",
       " 'Chr7-6Mb-seg-745': ['c230', 'c233', 'c237'],\n",
       " 'Chr7-6Mb-seg-746': ['c249', 'c252', 'c256'],\n",
       " 'Chr7-6Mb-seg-747': ['c268', 'c271', 'c275'],\n",
       " 'Chr7-6Mb-seg-748': ['c287', 'c290', 'c294'],\n",
       " 'Chr7-6Mb-seg-749': ['c306', 'c309', 'c313'],\n",
       " 'Chr7-6Mb-seg-750': ['c325', 'c328', 'c332'],\n",
       " 'Chr7-6Mb-seg-751': ['c347', 'c353', 'c360'],\n",
       " 'Chr7-6Mb-seg-752': ['c366', 'c372', 'c379'],\n",
       " 'Chr7-6Mb-seg-753': ['c190', 'c200', 'c201'],\n",
       " 'Chr7-6Mb-seg-754': ['c209', 'c219', 'c220'],\n",
       " 'Chr7-6Mb-seg-755': ['c228', 'c238', 'c239'],\n",
       " 'Chr7-6Mb-seg-756': ['c247', 'c257', 'c258'],\n",
       " 'Chr7-6Mb-seg-757': ['c266', 'c276', 'c277'],\n",
       " 'Chr7-6Mb-seg-758': ['c285', 'c295', 'c296'],\n",
       " 'Chr7-6Mb-seg-760': ['c304', 'c314', 'c315'],\n",
       " 'Chr7-6Mb-seg-762': ['c323', 'c333', 'c334'],\n",
       " 'Chr7-6Mb-seg-763': ['c342', 'c346', 'c348'],\n",
       " 'Chr7-6Mb-seg-764': ['c361', 'c365', 'c367'],\n",
       " 'Chr7-6Mb-seg-765': ['c191', 'c203', 'c207'],\n",
       " 'Chr7-6Mb-seg-766': ['c210', 'c222', 'c226'],\n",
       " 'Chr7-6Mb-seg-767': ['c229', 'c241', 'c245'],\n",
       " 'Chr7-6Mb-seg-768': ['c248', 'c260', 'c264'],\n",
       " 'Chr7-6Mb-seg-769': ['c267', 'c279', 'c283'],\n",
       " 'Chr7-6Mb-seg-770': ['c286', 'c298', 'c302'],\n",
       " 'Chr7-6Mb-seg-771': ['c305', 'c317', 'c321'],\n",
       " 'Chr7-6Mb-seg-772': ['c324', 'c336', 'c340'],\n",
       " 'Chr7-6Mb-seg-773': ['c350', 'c354', 'c355'],\n",
       " 'Chr7-6Mb-seg-775': ['c369', 'c373', 'c374'],\n",
       " 'Chr7-6Mb-seg-776': ['c193', 'c194', 'c206'],\n",
       " 'Chr7-6Mb-seg-777': ['c212', 'c213', 'c225'],\n",
       " 'Chr7-6Mb-seg-778': ['c231', 'c232', 'c244'],\n",
       " 'Chr7-6Mb-seg-780': ['c250', 'c251', 'c263'],\n",
       " 'Chr7-6Mb-seg-781': ['c269', 'c270', 'c282'],\n",
       " 'Chr7-6Mb-seg-784': ['c288', 'c289', 'c301'],\n",
       " 'Chr7-6Mb-seg-785': ['c307', 'c308', 'c320'],\n",
       " 'Chr7-6Mb-seg-786': ['c326', 'c327', 'c339'],\n",
       " 'Chr7-6Mb-seg-787': ['c349', 'c352', 'c358'],\n",
       " 'Chr7-6Mb-seg-788': ['c368', 'c371', 'c377'],\n",
       " 'Chr7-6Mb-seg-790': ['c198', 'c204', 'c205'],\n",
       " 'Chr7-6Mb-seg-791': ['c217', 'c223', 'c224'],\n",
       " 'Chr7-6Mb-seg-792': ['c236', 'c242', 'c243'],\n",
       " 'Chr7-6Mb-seg-793': ['c255', 'c261', 'c262'],\n",
       " 'Chr7-6Mb-seg-794': ['c274', 'c280', 'c281'],\n",
       " 'Chr7-6Mb-seg-795': ['c293', 'c299', 'c300'],\n",
       " 'Chr7-6Mb-seg-796': ['c312', 'c318', 'c319'],\n",
       " 'Chr7-6Mb-seg-797': ['c331', 'c337', 'c338'],\n",
       " 'Chr7-6Mb-seg-798': ['c346', 'c357', 'c359'],\n",
       " 'Chr7-6Mb-seg-799': ['c365', 'c376', 'c378'],\n",
       " 'Chr7-6Mb-seg-800': ['c195', 'c201', 'c208'],\n",
       " 'Chr7-6Mb-seg-801': ['c214', 'c220', 'c227'],\n",
       " 'Chr7-6Mb-seg-802': ['c233', 'c239', 'c246'],\n",
       " 'Chr7-6Mb-seg-804': ['c252', 'c258', 'c265'],\n",
       " 'Chr7-6Mb-seg-805': ['c271', 'c277', 'c284'],\n",
       " 'Chr7-6Mb-seg-806': ['c290', 'c296', 'c303'],\n",
       " 'Chr7-6Mb-seg-807': ['c309', 'c315', 'c322'],\n",
       " 'Chr7-6Mb-seg-808': ['c328', 'c334', 'c341'],\n",
       " 'Chr7-6Mb-seg-809': ['c342', 'c343', 'c345'],\n",
       " 'Chr7-6Mb-seg-810': ['c361', 'c362', 'c364'],\n",
       " 'Chr7-6Mb-seg-812': ['c192', 'c193', 'c207'],\n",
       " 'Chr7-6Mb-seg-813': ['c211', 'c212', 'c226'],\n",
       " 'Chr7-6Mb-seg-814': ['c230', 'c231', 'c245'],\n",
       " 'Chr7-6Mb-seg-816': ['c249', 'c250', 'c264'],\n",
       " 'Chr7-6Mb-seg-817': ['c268', 'c269', 'c283'],\n",
       " 'Chr7-6Mb-seg-818': ['c287', 'c288', 'c302'],\n",
       " 'Chr7-6Mb-seg-819': ['c306', 'c307', 'c321'],\n",
       " 'Chr7-6Mb-seg-820': ['c325', 'c326', 'c340'],\n",
       " 'Chr7-6Mb-seg-821': ['c355', 'c356', 'c360'],\n",
       " 'Chr7-6Mb-seg-822': ['c374', 'c375', 'c379'],\n",
       " 'Chr7-6Mb-seg-823': ['c191', 'c199', 'c200'],\n",
       " 'Chr7-6Mb-seg-824': ['c210', 'c218', 'c219'],\n",
       " 'Chr7-6Mb-seg-825': ['c229', 'c237', 'c238'],\n",
       " 'Chr7-6Mb-seg-828': ['c248', 'c256', 'c257'],\n",
       " 'Chr7-6Mb-seg-829': ['c267', 'c275', 'c276'],\n",
       " 'Chr7-6Mb-seg-831': ['c286', 'c294', 'c295'],\n",
       " 'Chr7-6Mb-seg-832': ['c305', 'c313', 'c314'],\n",
       " 'Chr7-6Mb-seg-833': ['c324', 'c332', 'c333'],\n",
       " 'Chr7-6Mb-seg-834': ['c351', 'c354', 'c359'],\n",
       " 'Chr7-6Mb-seg-835': ['c370', 'c373', 'c378'],\n",
       " 'Chr7-6Mb-seg-836': ['c194', 'c195', 'c202'],\n",
       " 'Chr7-6Mb-seg-837': ['c213', 'c214', 'c221'],\n",
       " 'Chr7-6Mb-seg-840': ['c232', 'c233', 'c240'],\n",
       " 'Chr7-6Mb-seg-841': ['c251', 'c252', 'c259'],\n",
       " 'Chr7-6Mb-seg-842': ['c270', 'c271', 'c278'],\n",
       " 'Chr7-6Mb-seg-845': ['c289', 'c290', 'c297'],\n",
       " 'Chr7-6Mb-seg-846': ['c308', 'c309', 'c316'],\n",
       " 'Chr7-6Mb-seg-847': ['c327', 'c328', 'c335'],\n",
       " 'Chr7-6Mb-seg-848': ['c347', 'c352', 'c357'],\n",
       " 'Chr7-6Mb-seg-849': ['c366', 'c371', 'c376'],\n",
       " 'Chr7-6Mb-seg-850': ['c193', 'c197', 'c208'],\n",
       " 'Chr7-6Mb-seg-852': ['c212', 'c216', 'c227'],\n",
       " 'Chr7-6Mb-seg-853': ['c231', 'c235', 'c246'],\n",
       " 'Chr7-6Mb-seg-854': ['c250', 'c254', 'c265'],\n",
       " 'Chr7-6Mb-seg-855': ['c269', 'c273', 'c284'],\n",
       " 'Chr7-6Mb-seg-856': ['c288', 'c292', 'c303'],\n",
       " 'Chr7-6Mb-seg-857': ['c307', 'c311', 'c322'],\n",
       " 'Chr7-6Mb-seg-858': ['c326', 'c330', 'c341'],\n",
       " 'Chr7-6Mb-seg-859': ['c344', 'c348', 'c353'],\n",
       " 'Chr7-6Mb-seg-860': ['c363', 'c367', 'c372'],\n",
       " 'Chr7-6Mb-seg-862': ['c192', 'c205', 'c206'],\n",
       " 'Chr7-6Mb-seg-863': ['c211', 'c224', 'c225'],\n",
       " 'Chr7-6Mb-seg-864': ['c230', 'c243', 'c244'],\n",
       " 'Chr7-6Mb-seg-865': ['c249', 'c262', 'c263'],\n",
       " 'Chr7-6Mb-seg-866': ['c268', 'c281', 'c282'],\n",
       " 'Chr7-6Mb-seg-868': ['c287', 'c300', 'c301'],\n",
       " 'Chr7-6Mb-seg-869': ['c306', 'c319', 'c320'],\n",
       " 'Chr7-6Mb-seg-870': ['c325', 'c338', 'c339'],\n",
       " 'Chr7-6Mb-seg-873': ['c349', 'c351', 'c356'],\n",
       " 'Chr7-6Mb-seg-876': ['c368', 'c370', 'c375'],\n",
       " 'Chr7-6Mb-seg-878': ['c198', 'c202', 'c203'],\n",
       " 'Chr7-6Mb-seg-879': ['c217', 'c221', 'c222'],\n",
       " 'Chr7-6Mb-seg-880': ['c236', 'c240', 'c241'],\n",
       " 'Chr7-6Mb-seg-881': ['c255', 'c259', 'c260'],\n",
       " 'Chr7-6Mb-seg-883': ['c274', 'c278', 'c279'],\n",
       " 'Chr7-6Mb-seg-885': ['c293', 'c297', 'c298'],\n",
       " 'Chr7-6Mb-seg-888': ['c312', 'c316', 'c317'],\n",
       " 'Chr7-6Mb-seg-889': ['c331', 'c335', 'c336'],\n",
       " 'Chr7-6Mb-seg-890': ['c343', 'c350', 'c358'],\n",
       " 'Chr7-6Mb-seg-891': ['c362', 'c369', 'c377'],\n",
       " 'Chr7-6Mb-seg-892': ['c190', 'c194', 'c196'],\n",
       " 'Chr7-6Mb-seg-893': ['c209', 'c213', 'c215'],\n",
       " 'Chr7-6Mb-seg-894': ['c228', 'c232', 'c234'],\n",
       " 'Chr7-6Mb-seg-895': ['c247', 'c251', 'c253'],\n",
       " 'Chr7-6Mb-seg-896': ['c266', 'c270', 'c272'],\n",
       " 'Chr7-6Mb-seg-897': ['c285', 'c289', 'c291'],\n",
       " 'Chr7-6Mb-seg-900': ['c304', 'c308', 'c310'],\n",
       " 'Chr7-6Mb-seg-901': ['c323', 'c327', 'c329'],\n",
       " 'Chr7-6Mb-seg-902': ['c345', 'c354', 'c357'],\n",
       " 'Chr7-6Mb-seg-904': ['c364', 'c373', 'c376'],\n",
       " 'Chr7-6Mb-seg-906': ['c197', 'c201', 'c207'],\n",
       " 'Chr7-6Mb-seg-907': ['c216', 'c220', 'c226'],\n",
       " 'Chr7-6Mb-seg-911': ['c235', 'c239', 'c245'],\n",
       " 'Chr7-6Mb-seg-912': ['c254', 'c258', 'c264'],\n",
       " 'Chr7-6Mb-seg-913': ['c273', 'c277', 'c283'],\n",
       " 'Chr7-6Mb-seg-916': ['c292', 'c296', 'c302'],\n",
       " 'Chr7-6Mb-seg-917': ['c311', 'c315', 'c321'],\n",
       " 'Chr7-6Mb-seg-918': ['c330', 'c334', 'c340'],\n",
       " 'Chr7-6Mb-seg-919': ['c348', 'c352', 'c359'],\n",
       " 'Chr7-6Mb-seg-922': ['c367', 'c371', 'c378'],\n",
       " 'Chr7-6Mb-seg-923': ['c193', 'c202', 'c205'],\n",
       " 'Chr7-6Mb-seg-924': ['c212', 'c221', 'c224'],\n",
       " 'Chr7-6Mb-seg-926': ['c231', 'c240', 'c243'],\n",
       " 'Chr7-6Mb-seg-929': ['c250', 'c259', 'c262'],\n",
       " 'Chr7-6Mb-seg-931': ['c269', 'c278', 'c281'],\n",
       " 'Chr7-6Mb-seg-932': ['c288', 'c297', 'c300'],\n",
       " 'Chr7-6Mb-seg-933': ['c307', 'c316', 'c319'],\n",
       " 'Chr7-6Mb-seg-934': ['c326', 'c335', 'c338'],\n",
       " 'Chr7-6Mb-seg-935': ['c342', 'c347', 'c355'],\n",
       " 'Chr7-6Mb-seg-937': ['c361', 'c366', 'c374'],\n",
       " 'Chr7-6Mb-seg-938': ['c195', 'c204', 'c206'],\n",
       " 'Chr7-6Mb-seg-939': ['c214', 'c223', 'c225'],\n",
       " 'Chr7-6Mb-seg-942': ['c233', 'c242', 'c244'],\n",
       " 'Chr7-6Mb-seg-944': ['c252', 'c261', 'c263'],\n",
       " 'Chr7-6Mb-seg-945': ['c271', 'c280', 'c282'],\n",
       " 'Chr7-6Mb-seg-946': ['c290', 'c299', 'c301'],\n",
       " 'Chr7-6Mb-seg-947': ['c309', 'c318', 'c320'],\n",
       " 'Chr7-6Mb-seg-950': ['c328', 'c337', 'c339'],\n",
       " 'Chr7-6Mb-seg-951': ['c346', 'c351', 'c360'],\n",
       " 'Chr7-6Mb-seg-952': ['c365', 'c370', 'c379'],\n",
       " 'Chr7-6Mb-seg-953': ['c192', 'c197', 'c203'],\n",
       " 'Chr7-6Mb-seg-954': ['c211', 'c216', 'c222'],\n",
       " 'Chr7-6Mb-seg-955': ['c230', 'c235', 'c241'],\n",
       " 'Chr7-6Mb-seg-956': ['c249', 'c254', 'c260'],\n",
       " 'Chr7-6Mb-seg-957': ['c268', 'c273', 'c279'],\n",
       " 'Chr7-6Mb-seg-958': ['c287', 'c292', 'c298'],\n",
       " 'Chr7-6Mb-seg-959': ['c306', 'c311', 'c317'],\n",
       " 'Chr7-6Mb-seg-960': ['c325', 'c330', 'c336'],\n",
       " 'Chr7-6Mb-seg-961': ['c344', 'c354', 'c356'],\n",
       " 'Chr7-6Mb-seg-962': ['c363', 'c373', 'c375'],\n",
       " 'Chr7-6Mb-seg-963': ['c190', 'c198', 'c199'],\n",
       " 'Chr7-6Mb-seg-964': ['c209', 'c217', 'c218'],\n",
       " 'Chr7-6Mb-seg-965': ['c228', 'c236', 'c237'],\n",
       " 'Chr7-6Mb-seg-966': ['c247', 'c255', 'c256'],\n",
       " 'Chr7-6Mb-seg-967': ['c266', 'c274', 'c275'],\n",
       " 'Chr7-6Mb-seg-968': ['c285', 'c293', 'c294'],\n",
       " 'Chr7-6Mb-seg-969': ['c304', 'c312', 'c313'],\n",
       " 'Chr7-6Mb-seg-970': ['c323', 'c331', 'c332'],\n",
       " 'Chr7-6Mb-seg-971': ['c349', 'c353', 'c359'],\n",
       " 'Chr7-6Mb-seg-972': ['c368', 'c372', 'c378'],\n",
       " 'Chr7-6Mb-seg-973': ['c196', 'c200', 'c207'],\n",
       " 'Chr7-6Mb-seg-974': ['c215', 'c219', 'c226'],\n",
       " 'Chr7-6Mb-seg-975': ['c234', 'c238', 'c245'],\n",
       " 'Chr7-6Mb-seg-976': ['c253', 'c257', 'c264'],\n",
       " 'Chr7-6Mb-seg-977': ['c272', 'c276', 'c283'],\n",
       " 'Chr7-6Mb-seg-978': ['c291', 'c295', 'c302'],\n",
       " 'Chr7-6Mb-seg-979': ['c310', 'c314', 'c321'],\n",
       " 'Chr7-6Mb-seg-980': ['c329', 'c333', 'c340'],\n",
       " 'Chr7-6Mb-seg-981': ['c348', 'c351', 'c358'],\n",
       " 'Chr7-6Mb-seg-982': ['c367', 'c370', 'c377'],\n",
       " 'Chr7-6Mb-seg-983': ['c194', 'c201', 'c204'],\n",
       " 'Chr7-6Mb-seg-984': ['c213', 'c220', 'c223'],\n",
       " 'Chr7-6Mb-seg-985': ['c232', 'c239', 'c242'],\n",
       " 'Chr7-6Mb-seg-986': ['c251', 'c258', 'c261'],\n",
       " 'Chr7-6Mb-seg-987': ['c270', 'c277', 'c280'],\n",
       " 'Chr7-6Mb-seg-988': ['c289', 'c296', 'c299'],\n",
       " 'Chr7-6Mb-seg-989': ['c308', 'c315', 'c318'],\n",
       " 'Chr7-6Mb-seg-990': ['c327', 'c334', 'c337'],\n",
       " 'Chr7-6Mb-seg-991': ['c344', 'c350', 'c352'],\n",
       " 'Chr7-6Mb-seg-992': ['c363', 'c369', 'c371'],\n",
       " 'Chr7-6Mb-seg-993': ['c191', 'c205', 'c208'],\n",
       " 'Chr7-6Mb-seg-994': ['c210', 'c224', 'c227'],\n",
       " 'Chr7-6Mb-seg-995': ['c229', 'c243', 'c246'],\n",
       " 'Chr7-6Mb-seg-996': ['c248', 'c262', 'c265'],\n",
       " 'Chr7-6Mb-seg-997': ['c267', 'c281', 'c284'],\n",
       " 'Chr7-6Mb-seg-998': ['c286', 'c300', 'c303'],\n",
       " 'Chr7-6Mb-seg-999': ['c305', 'c319', 'c322'],\n",
       " 'Chr7-6Mb-seg-1000': ['c324', 'c338', 'c341'],\n",
       " 'Chr7-6Mb-seg-1001': ['c343', 'c357', 'c360'],\n",
       " 'Chr7-6Mb-seg-1002': ['c362', 'c376', 'c379'],\n",
       " 'Chr7-6Mb-seg-1003': ['c193', 'c199', 'c203'],\n",
       " 'Chr7-6Mb-seg-1004': ['c212', 'c218', 'c222'],\n",
       " 'Chr7-6Mb-seg-1005': ['c231', 'c237', 'c241'],\n",
       " 'Chr7-6Mb-seg-1006': ['c250', 'c256', 'c260'],\n",
       " 'Chr7-6Mb-seg-1007': ['c269', 'c275', 'c279'],\n",
       " 'Chr7-6Mb-seg-1008': ['c288', 'c294', 'c298'],\n",
       " 'Chr7-6Mb-seg-1009': ['c307', 'c313', 'c317'],\n",
       " 'Chr7-6Mb-seg-1010': ['c326', 'c332', 'c336'],\n",
       " 'Chr7-6Mb-seg-1011': ['c342', 'c354', 'c358'],\n",
       " 'Chr7-6Mb-seg-1012': ['c361', 'c373', 'c377'],\n",
       " 'Chr7-6Mb-seg-1013': ['c192', 'c202', 'c204'],\n",
       " 'Chr7-6Mb-seg-1014': ['c211', 'c221', 'c223'],\n",
       " 'Chr7-6Mb-seg-1015': ['c230', 'c240', 'c242'],\n",
       " 'Chr7-6Mb-seg-1016': ['c249', 'c259', 'c261'],\n",
       " 'Chr7-6Mb-seg-1017': ['c268', 'c278', 'c280'],\n",
       " 'Chr7-6Mb-seg-1018': ['c287', 'c297', 'c299'],\n",
       " 'Chr7-6Mb-seg-1019': ['c306', 'c316', 'c318'],\n",
       " 'Chr7-6Mb-seg-1020': ['c325', 'c335', 'c337'],\n",
       " 'Chr7-6Mb-seg-1021': ['c346', 'c353', 'c356'],\n",
       " 'Chr7-6Mb-seg-1022': ['c365', 'c372', 'c375'],\n",
       " 'Chr7-6Mb-seg-1023': ['c197', 'c200', 'c206'],\n",
       " 'Chr7-6Mb-seg-1024': ['c216', 'c219', 'c225'],\n",
       " 'Chr7-6Mb-seg-1025': ['c235', 'c238', 'c244'],\n",
       " 'Chr7-6Mb-seg-1026': ['c254', 'c257', 'c263'],\n",
       " 'Chr7-6Mb-seg-1027': ['c273', 'c276', 'c282'],\n",
       " 'Chr7-6Mb-seg-1028': ['c292', 'c295', 'c301'],\n",
       " 'Chr7-6Mb-seg-1029': ['c311', 'c314', 'c320'],\n",
       " 'Chr7-6Mb-seg-1030': ['c330', 'c333', 'c339'],\n",
       " 'Chr7-6Mb-seg-1031': ['c345', 'c347', 'c348'],\n",
       " 'Chr7-6Mb-seg-1032': ['c364', 'c366', 'c367'],\n",
       " 'Chr7-6Mb-seg-1033': ['c195', 'c198', 'c207'],\n",
       " 'Chr7-6Mb-seg-1034': ['c214', 'c217', 'c226'],\n",
       " 'Chr7-6Mb-seg-1035': ['c233', 'c236', 'c245'],\n",
       " 'Chr7-6Mb-seg-1036': ['c252', 'c255', 'c264'],\n",
       " 'Chr7-6Mb-seg-1037': ['c271', 'c274', 'c283'],\n",
       " 'Chr7-6Mb-seg-1038': ['c290', 'c293', 'c302'],\n",
       " 'Chr7-6Mb-seg-1039': ['c309', 'c312', 'c321'],\n",
       " 'Chr7-6Mb-seg-1040': ['c328', 'c331', 'c340'],\n",
       " 'Chr7-6Mb-seg-1041': ['c342', 'c350', 'c351'],\n",
       " 'Chr7-6Mb-seg-1042': ['c361', 'c369', 'c370'],\n",
       " 'Chr7-6Mb-seg-1043': ['c191', 'c192', 'c194'],\n",
       " 'Chr7-6Mb-seg-1044': ['c210', 'c211', 'c213'],\n",
       " 'Chr7-6Mb-seg-1045': ['c229', 'c230', 'c232'],\n",
       " 'Chr7-6Mb-seg-1046': ['c248', 'c249', 'c251'],\n",
       " 'Chr7-6Mb-seg-1047': ['c267', 'c268', 'c270'],\n",
       " 'Chr7-6Mb-seg-1048': ['c286', 'c287', 'c289'],\n",
       " 'Chr7-6Mb-seg-1049': ['c305', 'c306', 'c308'],\n",
       " 'Chr7-6Mb-seg-1050': ['c324', 'c325', 'c327'],\n",
       " 'Chr7-6Mb-seg-1051': ['c344', 'c349', 'c355'],\n",
       " 'Chr7-6Mb-seg-1052': ['c363', 'c368', 'c374'],\n",
       " 'Chr7-6Mb-seg-1053': ['c199', 'c201', 'c205'],\n",
       " 'Chr7-6Mb-seg-1054': ['c218', 'c220', 'c224'],\n",
       " 'Chr7-6Mb-seg-1055': ['c237', 'c239', 'c243'],\n",
       " 'Chr7-6Mb-seg-1056': ['c256', 'c258', 'c262'],\n",
       " 'Chr7-6Mb-seg-1057': ['c275', 'c277', 'c281'],\n",
       " 'Chr7-6Mb-seg-1058': ['c294', 'c296', 'c300'],\n",
       " 'Chr7-6Mb-seg-1059': ['c313', 'c315', 'c319'],\n",
       " 'Chr7-6Mb-seg-1060': ['c332', 'c334', 'c338'],\n",
       " 'Chr7-6Mb-seg-1061': ['c358', 'c359', 'c360'],\n",
       " 'Chr7-6Mb-seg-1062': ['c377', 'c378', 'c379'],\n",
       " 'Chr7-6Mb-seg-1063': ['c190', 'c202', 'c206'],\n",
       " 'Chr7-6Mb-seg-1064': ['c209', 'c221', 'c225'],\n",
       " 'Chr7-6Mb-seg-1065': ['c228', 'c240', 'c244'],\n",
       " 'Chr7-6Mb-seg-1066': ['c247', 'c259', 'c263'],\n",
       " 'Chr7-6Mb-seg-1067': ['c266', 'c278', 'c282'],\n",
       " 'Chr7-6Mb-seg-1068': ['c285', 'c297', 'c301'],\n",
       " 'Chr7-6Mb-seg-1069': ['c304', 'c316', 'c320'],\n",
       " 'Chr7-6Mb-seg-1070': ['c323', 'c335', 'c339'],\n",
       " 'Chr7-6Mb-seg-1071': ['c345', 'c352', 'c356'],\n",
       " 'Chr7-6Mb-seg-1072': ['c364', 'c371', 'c375'],\n",
       " 'Chr7-6Mb-seg-1073': ['c191', 'c195', 'c197'],\n",
       " 'Chr7-6Mb-seg-1074': ['c210', 'c214', 'c216'],\n",
       " 'Chr7-6Mb-seg-1075': ['c229', 'c233', 'c235'],\n",
       " 'Chr7-6Mb-seg-1076': ['c248', 'c252', 'c254'],\n",
       " 'Chr7-6Mb-seg-1077': ['c267', 'c271', 'c273'],\n",
       " 'Chr7-6Mb-seg-1078': ['c286', 'c290', 'c292'],\n",
       " 'Chr7-6Mb-seg-1079': ['c305', 'c309', 'c311'],\n",
       " 'Chr7-6Mb-seg-1080': ['c324', 'c328', 'c330'],\n",
       " 'Chr7-6Mb-seg-1081': ['c343', 'c353', 'c354'],\n",
       " 'Chr7-6Mb-seg-1082': ['c362', 'c372', 'c373'],\n",
       " 'Chr7-6Mb-seg-1083': ['c193', 'c200', 'c204'],\n",
       " 'Chr7-6Mb-seg-1084': ['c212', 'c219', 'c223'],\n",
       " 'Chr7-6Mb-seg-1085': ['c231', 'c238', 'c242'],\n",
       " 'Chr7-6Mb-seg-1086': ['c250', 'c257', 'c261'],\n",
       " 'Chr7-6Mb-seg-1087': ['c269', 'c276', 'c280'],\n",
       " 'Chr7-6Mb-seg-1088': ['c288', 'c295', 'c299'],\n",
       " 'Chr7-6Mb-seg-1089': ['c307', 'c314', 'c318'],\n",
       " 'Chr7-6Mb-seg-1090': ['c326', 'c333', 'c337'],\n",
       " 'Chr7-6Mb-seg-1091': ['c342', 'c349', 'c357'],\n",
       " 'Chr7-6Mb-seg-1092': ['c361', 'c368', 'c376'],\n",
       " 'Chr7-6Mb-seg-1093': ['c196', 'c203', 'c205'],\n",
       " 'Chr7-6Mb-seg-1094': ['c215', 'c222', 'c224'],\n",
       " 'Chr7-6Mb-seg-1095': ['c234', 'c241', 'c243'],\n",
       " 'Chr7-6Mb-seg-1096': ['c253', 'c260', 'c262'],\n",
       " 'Chr7-6Mb-seg-1097': ['c272', 'c279', 'c281'],\n",
       " 'Chr7-6Mb-seg-1098': ['c291', 'c298', 'c300'],\n",
       " 'Chr7-6Mb-seg-1099': ['c310', 'c317', 'c319'],\n",
       " 'Chr7-6Mb-seg-1100': ['c329', 'c336', 'c338'],\n",
       " 'Chr7-6Mb-seg-1101': ['c348', 'c350', 'c360'],\n",
       " 'Chr7-6Mb-seg-1102': ['c367', 'c369', 'c379'],\n",
       " 'Chr7-6Mb-seg-1103': ['c206', 'c207', 'c208'],\n",
       " ...}"
      ]
     },
     "execution_count": 181,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_2_readout_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a4dd2783",
   "metadata": {},
   "source": [
    "<a id='3.2'></a>\n",
    "## 3.2 loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8e899029",
   "metadata": {},
   "source": [
    "### primers\n",
    "pick primer set 20,21"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 172,
   "id": "39032f2b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1B09_primer_20\n",
      "Name: W1B09_primer_20\n",
      "Description: W1B09_primer_20\n",
      "Number of features: 0\n",
      "Seq('TAGGCGTGTCGGCCAACCAG')\n",
      "- Picked primer: ID: W1B10_primer_21\n",
      "Name: W1B10_primer_21\n",
      "Description: W1B10_primer_21\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCGGGTCGATCATCGGCTTTG')\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Primers'\n",
    "primers = library_tools.assemble.load_primers([20, 21], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3dfa7962",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 173,
   "id": "fd693b58",
   "metadata": {},
   "outputs": [],
   "source": [
    "# collect types from region_2_readout_dict\n",
    "readout_usage_by_type = {}\n",
    "for _region, _infos in region_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in readout_usage_by_type:\n",
    "            readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in readout_usage_by_type[_info[0]]:\n",
    "            readout_usage_by_type[_info[0]].append(int(_info[1:]))\n",
    "\n",
    "readout_usage_dict = {_type:len(_v) for _type,_v in readout_usage_by_type.items()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 174,
   "id": "7bfc790e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_750.fasta\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_647.fasta\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_561.fasta\n",
      "[127, 127, 127]\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3.classes import _allowed_kwds\n",
    "reload(ia.library_tools.assemble)\n",
    "readout_dict ={}\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets'\n",
    "\n",
    "# combo\n",
    "readout_type = 'combo'\n",
    "# \n",
    "_type_char = _allowed_kwds[readout_type]\n",
    "_readout_start = 0\n",
    "_num_color = 3\n",
    "\n",
    "readout_dict[_type_char] = library_tools.assemble.load_readouts(readout_usage_dict[_type_char], \n",
    "                                                       _type='highest_snr', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=_num_color, # 750,647,561\n",
    "                                                       _start_id=_readout_start,\n",
    "                                                       )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 179,
   "id": "07899710",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save readout usage dict\n"
     ]
    }
   ],
   "source": [
    "# save\n",
    "readout_usage_file = os.path.join(library_folder, 'readout_usage.pkl')\n",
    "if not os.path.isfile(readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save readout usage dict')\n",
    "    pickle.dump(readout_dict, open(readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3d1eeb3b",
   "metadata": {},
   "source": [
    "<a id='3.3'></a>\n",
    "## 3.3 assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 188,
   "id": "c05a37c3",
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "# params\n",
    "assemble_params = {\n",
    "    'primer_len':20,\n",
    "    'num_readout_per_probe':4, \n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 189,
   "id": "c190aff0",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['c']\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-1\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-2\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-3\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-4\n",
      "--- assemblying 30 probes in region: Chr7-6Mb-seg-7\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-9\n",
      "--- assemblying 41 probes in region: Chr7-6Mb-seg-10\n",
      "--- assemblying 44 probes in region: Chr7-6Mb-seg-11\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-12\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-14\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-15\n",
      "--- assemblying 58 probes in region: Chr7-6Mb-seg-16\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-19\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-20\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-21\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-22\n",
      "--- assemblying 42 probes in region: Chr7-6Mb-seg-23\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-24\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-25\n",
      "--- assemblying 47 probes in region: Chr7-6Mb-seg-27\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-28\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-29\n",
      "--- assemblying 31 probes in region: Chr7-6Mb-seg-36\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-38\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-39\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-40\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-41\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-42\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-43\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-44\n",
      "--- assemblying 50 probes in region: Chr7-6Mb-seg-45\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-46\n",
      "--- assemblying 58 probes in region: Chr7-6Mb-seg-47\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-48\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-49\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-50\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-51\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-52\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-53\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-54\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-55\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-56\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-57\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-58\n",
      "--- assemblying 58 probes in region: Chr7-6Mb-seg-59\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-60\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-61\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-62\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-63\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-64\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-65\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-66\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-67\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-68\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-69\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-70\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-71\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-72\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-73\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-74\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-75\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-76\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-77\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-78\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-79\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-80\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-81\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-82\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-83\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-84\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-85\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-86\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-87\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-88\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-89\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-90\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-91\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-92\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-93\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-94\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-95\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-96\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-97\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-98\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-99\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-100\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-101\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-102\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-103\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-104\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-105\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-106\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-107\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-108\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-109\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-110\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-111\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-112\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-113\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-114\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-115\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-116\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-117\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-118\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-119\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-120\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-121\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-122\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-123\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-124\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-125\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-126\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-127\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-128\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-129\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-130\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-131\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-132\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-133\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-134\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-135\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-136\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-137\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-138\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-139\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-140\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-141\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-142\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-143\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-144\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-145\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-146\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-147\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-148\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-149\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-150\n",
      "--- assemblying 53 probes in region: Chr7-6Mb-seg-151\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-152\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-153\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-154\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-155\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-156\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-157\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-158\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-159\n",
      "--- assemblying 29 probes in region: Chr7-6Mb-seg-161\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-162\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-163\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-164\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-165\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-166\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-167\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-168\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-169\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-170\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-171\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-172\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-173\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-174\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-175\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-176\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-177\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-178\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-179\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-180\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-181\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-182\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-183\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-184\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-185\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-186\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-187\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-188\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-189\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-190\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-191\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-192\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-193\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-194\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-195\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-196\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-197\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-198\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-199\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-200\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-201\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-202\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-203\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-204\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-205\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-206\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-207\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-208\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-209\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-210\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-211\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-212\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-213\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-214\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-215\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-216\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-217\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-218\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-219\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-220\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-221\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-222\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-223\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-224\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-225\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-226\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-227\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-228\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-229\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-230\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-231\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-232\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-233\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-234\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-235\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-236\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-237\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-238\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-239\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-240\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-241\n",
      "--- assemblying 58 probes in region: Chr7-6Mb-seg-242\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-243\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-244\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-245\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-246\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-247\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-248\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-249\n",
      "--- assemblying 21 probes in region: Chr7-6Mb-seg-250\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-251\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-252\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-253\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-254\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-255\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-256\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-257\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-258\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-259\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-260\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-261\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-262\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-263\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-264\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-265\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-266\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-267\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-268\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-269\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-270\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-271\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-272\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-273\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-274\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-275\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-276\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-277\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-278\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-279\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-280\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-281\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-282\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-283\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-284\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-285\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-286\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-287\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-288\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-289\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-290\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-291\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-292\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-293\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-294\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-295\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-296\n",
      "--- assemblying 36 probes in region: Chr7-6Mb-seg-297\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-298\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-299\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-300\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-301\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-302\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-303\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-304\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-305\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-306\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-307\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-308\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-309\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-310\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-311\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-312\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-313\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-314\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-315\n",
      "--- assemblying 50 probes in region: Chr7-6Mb-seg-316\n",
      "--- assemblying 53 probes in region: Chr7-6Mb-seg-317\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-318\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-319\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-320\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-321\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-322\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-323\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-324\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-325\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-326\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-327\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-328\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-329\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-330\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-331\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-332\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-333\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-334\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-335\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-336\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-337\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-338\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-339\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-340\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-341\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-342\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-343\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-344\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-345\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-346\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-347\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-348\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-349\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-350\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-351\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-352\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-353\n",
      "--- assemblying 53 probes in region: Chr7-6Mb-seg-354\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-355\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-356\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-357\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-358\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-359\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-360\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-361\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-362\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-363\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-364\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-365\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-366\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-367\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-368\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-369\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-370\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-371\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-372\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-373\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-374\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-375\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-376\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-377\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-378\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-379\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-380\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-381\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-382\n",
      "--- assemblying 44 probes in region: Chr7-6Mb-seg-383\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-384\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-385\n",
      "--- assemblying 42 probes in region: Chr7-6Mb-seg-386\n",
      "--- assemblying 42 probes in region: Chr7-6Mb-seg-387\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-388\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-389\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-390\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-391\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-392\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-393\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-394\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-395\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-396\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-397\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-398\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-399\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-400\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-401\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-402\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-403\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-404\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-405\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-406\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-407\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-408\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-409\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-410\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-411\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-412\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-413\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-414\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-415\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-416\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-417\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-418\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-419\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-420\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-421\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-422\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-423\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-424\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-425\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-426\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-427\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-428\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-429\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-430\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-431\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-432\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-433\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-434\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-435\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-436\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-437\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-438\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-439\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-440\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-441\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-442\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-443\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-444\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-445\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-446\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-447\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-448\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-449\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-450\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-451\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-452\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-453\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-454\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-455\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-456\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-457\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-458\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-459\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-460\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-461\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-462\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-463\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-464\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-465\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-466\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-467\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-468\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-469\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-470\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-471\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-472\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-473\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-474\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-475\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-476\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-477\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-478\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-479\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-480\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-481\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-482\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-483\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-484\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-485\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-486\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-487\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-488\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-489\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-490\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-491\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-492\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-493\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-494\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-495\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-496\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-497\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-498\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-499\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-500\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-501\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-502\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-503\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-504\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-505\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-506\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-507\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-508\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-509\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-510\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-511\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-512\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-513\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-514\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-515\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-516\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-517\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-518\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-519\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-520\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-521\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-522\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-523\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-524\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-525\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-526\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-527\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-528\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-529\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-530\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-531\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-532\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-533\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-534\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-535\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-536\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-537\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-538\n",
      "--- assemblying 33 probes in region: Chr7-6Mb-seg-539\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-540\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-541\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-542\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-543\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-544\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-545\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-546\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-547\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-548\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-549\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-550\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-551\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-552\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-553\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-554\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-555\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-556\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-557\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-558\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-559\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-560\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-561\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-562\n",
      "--- assemblying 50 probes in region: Chr7-6Mb-seg-563\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-564\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-565\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-566\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-567\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-568\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-569\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-570\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-571\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-572\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-573\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-574\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-575\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-576\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-577\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-578\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-579\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-580\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-581\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-582\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-583\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-584\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-585\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-586\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-587\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-588\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-589\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-590\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-591\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-592\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-593\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-594\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-595\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-596\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-597\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-598\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-599\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-600\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-601\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-602\n",
      "--- assemblying 53 probes in region: Chr7-6Mb-seg-603\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-604\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-605\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-606\n",
      "--- assemblying 53 probes in region: Chr7-6Mb-seg-607\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-608\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-609\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-610\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-611\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-612\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-613\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-614\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-615\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-616\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-617\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-618\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-619\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-620\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-621\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-622\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-623\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-624\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-625\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-626\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-627\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-628\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-629\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-630\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-631\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-632\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-633\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-634\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-635\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-638\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-639\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-640\n",
      "--- assemblying 37 probes in region: Chr7-6Mb-seg-641\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-642\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-643\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-644\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-647\n",
      "--- assemblying 50 probes in region: Chr7-6Mb-seg-648\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-649\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-650\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-651\n",
      "--- assemblying 35 probes in region: Chr7-6Mb-seg-655\n",
      "--- assemblying 35 probes in region: Chr7-6Mb-seg-660\n",
      "--- assemblying 42 probes in region: Chr7-6Mb-seg-661\n",
      "--- assemblying 58 probes in region: Chr7-6Mb-seg-662\n",
      "--- assemblying 50 probes in region: Chr7-6Mb-seg-663\n",
      "--- assemblying 41 probes in region: Chr7-6Mb-seg-664\n",
      "--- assemblying 30 probes in region: Chr7-6Mb-seg-665\n",
      "--- assemblying 58 probes in region: Chr7-6Mb-seg-668\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-669\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-670\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-671\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-672\n",
      "--- assemblying 32 probes in region: Chr7-6Mb-seg-674\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-675\n",
      "--- assemblying 38 probes in region: Chr7-6Mb-seg-676\n",
      "--- assemblying 36 probes in region: Chr7-6Mb-seg-677\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-679\n",
      "--- assemblying 37 probes in region: Chr7-6Mb-seg-680\n",
      "--- assemblying 33 probes in region: Chr7-6Mb-seg-681\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-682\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-683\n",
      "--- assemblying 32 probes in region: Chr7-6Mb-seg-692\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-693\n",
      "--- assemblying 32 probes in region: Chr7-6Mb-seg-698\n",
      "--- assemblying 28 probes in region: Chr7-6Mb-seg-699\n",
      "--- assemblying 37 probes in region: Chr7-6Mb-seg-700\n",
      "--- assemblying 41 probes in region: Chr7-6Mb-seg-701\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-703\n",
      "--- assemblying 41 probes in region: Chr7-6Mb-seg-704\n",
      "--- assemblying 31 probes in region: Chr7-6Mb-seg-705\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-706\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-707\n",
      "--- assemblying 31 probes in region: Chr7-6Mb-seg-708\n",
      "--- assemblying 29 probes in region: Chr7-6Mb-seg-710\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-711\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-712\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-713\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-717\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-718\n",
      "--- assemblying 29 probes in region: Chr7-6Mb-seg-719\n",
      "--- assemblying 41 probes in region: Chr7-6Mb-seg-720\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-721\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-722\n",
      "--- assemblying 32 probes in region: Chr7-6Mb-seg-723\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-726\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-727\n",
      "--- assemblying 22 probes in region: Chr7-6Mb-seg-728\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-731\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-732\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-733\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-734\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-735\n",
      "--- assemblying 38 probes in region: Chr7-6Mb-seg-738\n",
      "--- assemblying 41 probes in region: Chr7-6Mb-seg-739\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-740\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-741\n",
      "--- assemblying 37 probes in region: Chr7-6Mb-seg-742\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-743\n",
      "--- assemblying 44 probes in region: Chr7-6Mb-seg-744\n",
      "--- assemblying 32 probes in region: Chr7-6Mb-seg-745\n",
      "--- assemblying 35 probes in region: Chr7-6Mb-seg-746\n",
      "--- assemblying 28 probes in region: Chr7-6Mb-seg-747\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-748\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-749\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-750\n",
      "--- assemblying 45 probes in region: Chr7-6Mb-seg-751\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-752\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-753\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-754\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-755\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-756\n",
      "--- assemblying 50 probes in region: Chr7-6Mb-seg-757\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-758\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-760\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-762\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-763\n",
      "--- assemblying 42 probes in region: Chr7-6Mb-seg-764\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-765\n",
      "--- assemblying 50 probes in region: Chr7-6Mb-seg-766\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-767\n",
      "--- assemblying 42 probes in region: Chr7-6Mb-seg-768\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-769\n",
      "--- assemblying 21 probes in region: Chr7-6Mb-seg-770\n",
      "--- assemblying 28 probes in region: Chr7-6Mb-seg-771\n",
      "--- assemblying 47 probes in region: Chr7-6Mb-seg-772\n",
      "--- assemblying 29 probes in region: Chr7-6Mb-seg-773\n",
      "--- assemblying 36 probes in region: Chr7-6Mb-seg-775\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-776\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-777\n",
      "--- assemblying 41 probes in region: Chr7-6Mb-seg-778\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-780\n",
      "--- assemblying 38 probes in region: Chr7-6Mb-seg-781\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-784\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-785\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-786\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-787\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-788\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-790\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-791\n",
      "--- assemblying 43 probes in region: Chr7-6Mb-seg-792\n",
      "--- assemblying 29 probes in region: Chr7-6Mb-seg-793\n",
      "--- assemblying 37 probes in region: Chr7-6Mb-seg-794\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-795\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-796\n",
      "--- assemblying 44 probes in region: Chr7-6Mb-seg-797\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-798\n",
      "--- assemblying 45 probes in region: Chr7-6Mb-seg-799\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-800\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-801\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-802\n",
      "--- assemblying 43 probes in region: Chr7-6Mb-seg-804\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-805\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-806\n",
      "--- assemblying 53 probes in region: Chr7-6Mb-seg-807\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-808\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-809\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-810\n",
      "--- assemblying 22 probes in region: Chr7-6Mb-seg-812\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-813\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-814\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-816\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-817\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-818\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-819\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-820\n",
      "--- assemblying 30 probes in region: Chr7-6Mb-seg-821\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-822\n",
      "--- assemblying 37 probes in region: Chr7-6Mb-seg-823\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-824\n",
      "--- assemblying 30 probes in region: Chr7-6Mb-seg-825\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-828\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-829\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-831\n",
      "--- assemblying 29 probes in region: Chr7-6Mb-seg-832\n",
      "--- assemblying 32 probes in region: Chr7-6Mb-seg-833\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-834\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-835\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-836\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-837\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-840\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-841\n",
      "--- assemblying 53 probes in region: Chr7-6Mb-seg-842\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-845\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-846\n",
      "--- assemblying 22 probes in region: Chr7-6Mb-seg-847\n",
      "--- assemblying 31 probes in region: Chr7-6Mb-seg-848\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-849\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-850\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-852\n",
      "--- assemblying 43 probes in region: Chr7-6Mb-seg-853\n",
      "--- assemblying 33 probes in region: Chr7-6Mb-seg-854\n",
      "--- assemblying 38 probes in region: Chr7-6Mb-seg-855\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-856\n",
      "--- assemblying 42 probes in region: Chr7-6Mb-seg-857\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-858\n",
      "--- assemblying 28 probes in region: Chr7-6Mb-seg-859\n",
      "--- assemblying 30 probes in region: Chr7-6Mb-seg-860\n",
      "--- assemblying 44 probes in region: Chr7-6Mb-seg-862\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-863\n",
      "--- assemblying 38 probes in region: Chr7-6Mb-seg-864\n",
      "--- assemblying 33 probes in region: Chr7-6Mb-seg-865\n",
      "--- assemblying 35 probes in region: Chr7-6Mb-seg-866\n",
      "--- assemblying 43 probes in region: Chr7-6Mb-seg-868\n",
      "--- assemblying 37 probes in region: Chr7-6Mb-seg-869\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-870\n",
      "--- assemblying 39 probes in region: Chr7-6Mb-seg-873\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-876\n",
      "--- assemblying 44 probes in region: Chr7-6Mb-seg-878\n",
      "--- assemblying 47 probes in region: Chr7-6Mb-seg-879\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-880\n",
      "--- assemblying 34 probes in region: Chr7-6Mb-seg-881\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-883\n",
      "--- assemblying 33 probes in region: Chr7-6Mb-seg-885\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-888\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 58 probes in region: Chr7-6Mb-seg-889\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-890\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-891\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-892\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-893\n",
      "--- assemblying 43 probes in region: Chr7-6Mb-seg-894\n",
      "--- assemblying 46 probes in region: Chr7-6Mb-seg-895\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-896\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-897\n",
      "--- assemblying 58 probes in region: Chr7-6Mb-seg-900\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-901\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-902\n",
      "--- assemblying 36 probes in region: Chr7-6Mb-seg-904\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-906\n",
      "--- assemblying 42 probes in region: Chr7-6Mb-seg-907\n",
      "--- assemblying 50 probes in region: Chr7-6Mb-seg-911\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-912\n",
      "--- assemblying 22 probes in region: Chr7-6Mb-seg-913\n",
      "--- assemblying 32 probes in region: Chr7-6Mb-seg-916\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-917\n",
      "--- assemblying 36 probes in region: Chr7-6Mb-seg-918\n",
      "--- assemblying 34 probes in region: Chr7-6Mb-seg-919\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-922\n",
      "--- assemblying 41 probes in region: Chr7-6Mb-seg-923\n",
      "--- assemblying 33 probes in region: Chr7-6Mb-seg-924\n",
      "--- assemblying 38 probes in region: Chr7-6Mb-seg-926\n",
      "--- assemblying 40 probes in region: Chr7-6Mb-seg-929\n",
      "--- assemblying 21 probes in region: Chr7-6Mb-seg-931\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-932\n",
      "--- assemblying 36 probes in region: Chr7-6Mb-seg-933\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-934\n",
      "--- assemblying 28 probes in region: Chr7-6Mb-seg-935\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-937\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-938\n",
      "--- assemblying 34 probes in region: Chr7-6Mb-seg-939\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-942\n",
      "--- assemblying 42 probes in region: Chr7-6Mb-seg-944\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-945\n",
      "--- assemblying 44 probes in region: Chr7-6Mb-seg-946\n",
      "--- assemblying 21 probes in region: Chr7-6Mb-seg-947\n",
      "--- assemblying 31 probes in region: Chr7-6Mb-seg-950\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-951\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-952\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-953\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-954\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-955\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-956\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-957\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-958\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-959\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-960\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-961\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-962\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-963\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-964\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-965\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-966\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-967\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-968\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-969\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-970\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-971\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-972\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-973\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-974\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-975\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-976\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-977\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-978\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-979\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-980\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-981\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-982\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-983\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-984\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-985\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-986\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-987\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-988\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-989\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-990\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-991\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-992\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-993\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-994\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-995\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-996\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-997\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-998\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-999\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1000\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1001\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-1002\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1003\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1004\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1005\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1006\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1007\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1008\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1009\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1010\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1011\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1012\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-1013\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1014\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1015\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1016\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1017\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1018\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1019\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-1020\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1021\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-1022\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-1023\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-1024\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1025\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1026\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1027\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1028\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1029\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1030\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1031\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-1032\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1033\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1034\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-1035\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1036\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1037\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1038\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1039\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1040\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1041\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1042\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1043\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1044\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1045\n",
      "--- assemblying 57 probes in region: Chr7-6Mb-seg-1046\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1047\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-1048\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1049\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1050\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1051\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1052\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1053\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1054\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1055\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-1056\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1057\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1058\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1059\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1060\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1061\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-1062\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-1063\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-1064\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1065\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1066\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1067\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-1068\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-1069\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-1070\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-1071\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-1072\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1073\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1074\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1075\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1076\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1077\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-1078\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1079\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-1080\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-1081\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-1082\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1083\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1084\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1085\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1086\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-1087\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-1088\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1089\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-1090\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1091\n",
      "--- assemblying 60 probes in region: Chr7-6Mb-seg-1092\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-1093\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-1094\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1095\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1096\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-1097\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-1098\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-1099\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-1100\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1101\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1102\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1103\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1104\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-1105\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1106\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-1107\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-1108\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-1109\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-1110\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1111\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1112\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1113\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1114\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-1115\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1116\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1117\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1118\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-1119\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1120\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1121\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-1122\n",
      "--- assemblying 62 probes in region: Chr7-6Mb-seg-1123\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1124\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-1125\n",
      "--- assemblying 56 probes in region: Chr7-6Mb-seg-1126\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1127\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-1128\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1129\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-1130\n",
      "--- assemblying 52 probes in region: Chr7-6Mb-seg-1131\n",
      "--- assemblying 78 probes in region: Chr7-6Mb-seg-1132\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-1133\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-1134\n",
      "--- assemblying 54 probes in region: Chr7-6Mb-seg-1135\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-1136\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1137\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-1138\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-1139\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-1140\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1141\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-1142\n",
      "--- assemblying 58 probes in region: Chr7-6Mb-seg-1143\n",
      "--- assemblying 48 probes in region: Chr7-6Mb-seg-1144\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-1145\n",
      "--- assemblying 49 probes in region: Chr7-6Mb-seg-1146\n",
      "--- assemblying 61 probes in region: Chr7-6Mb-seg-1147\n",
      "--- assemblying 42 probes in region: Chr7-6Mb-seg-1148\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-1149\n",
      "--- assemblying 59 probes in region: Chr7-6Mb-seg-1150\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-1151\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1152\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1153\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1154\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1155\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1156\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-1157\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-1158\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-1159\n",
      "--- assemblying 69 probes in region: Chr7-6Mb-seg-1160\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-1161\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1162\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-1163\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-1164\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1165\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1166\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-1167\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-1168\n",
      "--- assemblying 70 probes in region: Chr7-6Mb-seg-1169\n",
      "--- assemblying 66 probes in region: Chr7-6Mb-seg-1170\n",
      "--- assemblying 67 probes in region: Chr7-6Mb-seg-1171\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-1172\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-1173\n",
      "--- assemblying 71 probes in region: Chr7-6Mb-seg-1174\n",
      "--- assemblying 72 probes in region: Chr7-6Mb-seg-1175\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-1176\n",
      "--- assemblying 55 probes in region: Chr7-6Mb-seg-1177\n",
      "--- assemblying 76 probes in region: Chr7-6Mb-seg-1178\n",
      "--- assemblying 64 probes in region: Chr7-6Mb-seg-1179\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-1180\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1181\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1182\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1183\n",
      "--- assemblying 79 probes in region: Chr7-6Mb-seg-1184\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1185\n",
      "--- assemblying 68 probes in region: Chr7-6Mb-seg-1186\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-1187\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1188\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1189\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1190\n",
      "--- assemblying 65 probes in region: Chr7-6Mb-seg-1191\n",
      "--- assemblying 77 probes in region: Chr7-6Mb-seg-1192\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1193\n",
      "--- assemblying 75 probes in region: Chr7-6Mb-seg-1194\n",
      "--- assemblying 73 probes in region: Chr7-6Mb-seg-1195\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1196\n",
      "--- assemblying 80 probes in region: Chr7-6Mb-seg-1197\n",
      "--- assemblying 74 probes in region: Chr7-6Mb-seg-1198\n",
      "--- assemblying 51 probes in region: Chr7-6Mb-seg-1199\n",
      "--- assemblying 63 probes in region: Chr7-6Mb-seg-1200\n",
      "-- 70833 probes assembled in total.\n",
      "-- saving 70833 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "# run assembly\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dict, \n",
    "                                               region_2_readout_dict, readout_dict, primers, \n",
    "                                               primer_len=assemble_params['primer_len'], \n",
    "                                               num_readout_per_probe=assemble_params['num_readout_per_probe'], \n",
    "                                               unique_readout_per_probe=True,\n",
    "                                               rc_targets=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9265b92d",
   "metadata": {},
   "source": [
    "# 4.Check quality"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8dca4823",
   "metadata": {},
   "source": [
    "\n",
    "## 4.1 Load probes\n",
    "\n",
    "Checking procedures could restart from here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 190,
   "id": "a651e7eb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 70833\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "\n",
    "# candidate full-length probe filename\n",
    "candidate_basename = 'candidate_probes.fasta' # old version\n",
    "# load full probes\n",
    "full_records = []\n",
    "with open(os.path.join(library_folder, candidate_basename), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fe532250",
   "metadata": {},
   "source": [
    "### Manually input the following info to double check!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 185,
   "id": "cf1d4bf8",
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 4\n",
    "primer_len=20\n",
    "target_len=42\n",
    "min_num_probes = 21\n",
    "# merfish related\n",
    "num_on_bits = 3"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2da410b1",
   "metadata": {},
   "source": [
    "### 4.1.1 check primer usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 186,
   "id": "3cba3f0b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 70833\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n",
    "print(primer_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "30ce1025",
   "metadata": {},
   "source": [
    "### 4.1.2 check region probe number"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 194,
   "id": "761720bd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: Chr7-6Mb-seg-1 -> 52\n",
      "gene: Chr7-6Mb-seg-2 -> 60\n",
      "gene: Chr7-6Mb-seg-3 -> 49\n",
      "gene: Chr7-6Mb-seg-4 -> 52\n",
      "gene: Chr7-6Mb-seg-7 -> 30\n",
      "gene: Chr7-6Mb-seg-9 -> 60\n",
      "gene: Chr7-6Mb-seg-10 -> 41\n",
      "gene: Chr7-6Mb-seg-11 -> 44\n",
      "gene: Chr7-6Mb-seg-12 -> 40\n",
      "gene: Chr7-6Mb-seg-14 -> 26\n",
      "gene: Chr7-6Mb-seg-15 -> 66\n",
      "gene: Chr7-6Mb-seg-16 -> 58\n",
      "gene: Chr7-6Mb-seg-19 -> 40\n",
      "gene: Chr7-6Mb-seg-20 -> 49\n",
      "gene: Chr7-6Mb-seg-21 -> 40\n",
      "gene: Chr7-6Mb-seg-22 -> 51\n",
      "gene: Chr7-6Mb-seg-23 -> 42\n",
      "gene: Chr7-6Mb-seg-24 -> 51\n",
      "gene: Chr7-6Mb-seg-25 -> 46\n",
      "gene: Chr7-6Mb-seg-27 -> 47\n",
      "gene: Chr7-6Mb-seg-28 -> 60\n",
      "gene: Chr7-6Mb-seg-29 -> 40\n",
      "gene: Chr7-6Mb-seg-36 -> 31\n",
      "gene: Chr7-6Mb-seg-38 -> 39\n",
      "gene: Chr7-6Mb-seg-39 -> 39\n",
      "gene: Chr7-6Mb-seg-40 -> 51\n",
      "gene: Chr7-6Mb-seg-41 -> 60\n",
      "gene: Chr7-6Mb-seg-42 -> 56\n",
      "gene: Chr7-6Mb-seg-43 -> 64\n",
      "gene: Chr7-6Mb-seg-44 -> 49\n",
      "gene: Chr7-6Mb-seg-45 -> 50\n",
      "gene: Chr7-6Mb-seg-46 -> 60\n",
      "gene: Chr7-6Mb-seg-47 -> 58\n",
      "gene: Chr7-6Mb-seg-48 -> 71\n",
      "gene: Chr7-6Mb-seg-49 -> 67\n",
      "gene: Chr7-6Mb-seg-50 -> 80\n",
      "gene: Chr7-6Mb-seg-51 -> 80\n",
      "gene: Chr7-6Mb-seg-52 -> 79\n",
      "gene: Chr7-6Mb-seg-53 -> 74\n",
      "gene: Chr7-6Mb-seg-54 -> 70\n",
      "gene: Chr7-6Mb-seg-55 -> 70\n",
      "gene: Chr7-6Mb-seg-56 -> 68\n",
      "gene: Chr7-6Mb-seg-57 -> 68\n",
      "gene: Chr7-6Mb-seg-58 -> 62\n",
      "gene: Chr7-6Mb-seg-59 -> 58\n",
      "gene: Chr7-6Mb-seg-60 -> 48\n",
      "gene: Chr7-6Mb-seg-61 -> 80\n",
      "gene: Chr7-6Mb-seg-62 -> 80\n",
      "gene: Chr7-6Mb-seg-63 -> 80\n",
      "gene: Chr7-6Mb-seg-64 -> 75\n",
      "gene: Chr7-6Mb-seg-65 -> 80\n",
      "gene: Chr7-6Mb-seg-66 -> 70\n",
      "gene: Chr7-6Mb-seg-67 -> 72\n",
      "gene: Chr7-6Mb-seg-68 -> 80\n",
      "gene: Chr7-6Mb-seg-69 -> 80\n",
      "gene: Chr7-6Mb-seg-70 -> 68\n",
      "gene: Chr7-6Mb-seg-71 -> 80\n",
      "gene: Chr7-6Mb-seg-72 -> 74\n",
      "gene: Chr7-6Mb-seg-73 -> 69\n",
      "gene: Chr7-6Mb-seg-74 -> 64\n",
      "gene: Chr7-6Mb-seg-75 -> 76\n",
      "gene: Chr7-6Mb-seg-76 -> 67\n",
      "gene: Chr7-6Mb-seg-77 -> 80\n",
      "gene: Chr7-6Mb-seg-78 -> 69\n",
      "gene: Chr7-6Mb-seg-79 -> 80\n",
      "gene: Chr7-6Mb-seg-80 -> 77\n",
      "gene: Chr7-6Mb-seg-81 -> 66\n",
      "gene: Chr7-6Mb-seg-82 -> 77\n",
      "gene: Chr7-6Mb-seg-83 -> 64\n",
      "gene: Chr7-6Mb-seg-84 -> 63\n",
      "gene: Chr7-6Mb-seg-85 -> 74\n",
      "gene: Chr7-6Mb-seg-86 -> 73\n",
      "gene: Chr7-6Mb-seg-87 -> 79\n",
      "gene: Chr7-6Mb-seg-88 -> 71\n",
      "gene: Chr7-6Mb-seg-89 -> 80\n",
      "gene: Chr7-6Mb-seg-90 -> 79\n",
      "gene: Chr7-6Mb-seg-91 -> 52\n",
      "gene: Chr7-6Mb-seg-92 -> 69\n",
      "gene: Chr7-6Mb-seg-93 -> 80\n",
      "gene: Chr7-6Mb-seg-94 -> 80\n",
      "gene: Chr7-6Mb-seg-95 -> 80\n",
      "gene: Chr7-6Mb-seg-96 -> 49\n",
      "gene: Chr7-6Mb-seg-97 -> 61\n",
      "gene: Chr7-6Mb-seg-98 -> 78\n",
      "gene: Chr7-6Mb-seg-99 -> 80\n",
      "gene: Chr7-6Mb-seg-100 -> 80\n",
      "gene: Chr7-6Mb-seg-101 -> 75\n",
      "gene: Chr7-6Mb-seg-102 -> 80\n",
      "gene: Chr7-6Mb-seg-103 -> 66\n",
      "gene: Chr7-6Mb-seg-104 -> 75\n",
      "gene: Chr7-6Mb-seg-105 -> 75\n",
      "gene: Chr7-6Mb-seg-106 -> 80\n",
      "gene: Chr7-6Mb-seg-107 -> 80\n",
      "gene: Chr7-6Mb-seg-108 -> 80\n",
      "gene: Chr7-6Mb-seg-109 -> 80\n",
      "gene: Chr7-6Mb-seg-110 -> 70\n",
      "gene: Chr7-6Mb-seg-111 -> 80\n",
      "gene: Chr7-6Mb-seg-112 -> 59\n",
      "gene: Chr7-6Mb-seg-113 -> 68\n",
      "gene: Chr7-6Mb-seg-114 -> 59\n",
      "gene: Chr7-6Mb-seg-115 -> 64\n",
      "gene: Chr7-6Mb-seg-116 -> 39\n",
      "gene: Chr7-6Mb-seg-117 -> 61\n",
      "gene: Chr7-6Mb-seg-118 -> 80\n",
      "gene: Chr7-6Mb-seg-119 -> 80\n",
      "gene: Chr7-6Mb-seg-120 -> 80\n",
      "gene: Chr7-6Mb-seg-121 -> 80\n",
      "gene: Chr7-6Mb-seg-122 -> 80\n",
      "gene: Chr7-6Mb-seg-123 -> 76\n",
      "gene: Chr7-6Mb-seg-124 -> 61\n",
      "gene: Chr7-6Mb-seg-125 -> 69\n",
      "gene: Chr7-6Mb-seg-126 -> 66\n",
      "gene: Chr7-6Mb-seg-127 -> 80\n",
      "gene: Chr7-6Mb-seg-128 -> 72\n",
      "gene: Chr7-6Mb-seg-129 -> 66\n",
      "gene: Chr7-6Mb-seg-130 -> 69\n",
      "gene: Chr7-6Mb-seg-131 -> 69\n",
      "gene: Chr7-6Mb-seg-132 -> 79\n",
      "gene: Chr7-6Mb-seg-133 -> 76\n",
      "gene: Chr7-6Mb-seg-134 -> 71\n",
      "gene: Chr7-6Mb-seg-135 -> 75\n",
      "gene: Chr7-6Mb-seg-136 -> 64\n",
      "gene: Chr7-6Mb-seg-137 -> 66\n",
      "gene: Chr7-6Mb-seg-138 -> 67\n",
      "gene: Chr7-6Mb-seg-139 -> 72\n",
      "gene: Chr7-6Mb-seg-140 -> 64\n",
      "gene: Chr7-6Mb-seg-141 -> 66\n",
      "gene: Chr7-6Mb-seg-142 -> 57\n",
      "gene: Chr7-6Mb-seg-143 -> 65\n",
      "gene: Chr7-6Mb-seg-144 -> 69\n",
      "gene: Chr7-6Mb-seg-145 -> 69\n",
      "gene: Chr7-6Mb-seg-146 -> 61\n",
      "gene: Chr7-6Mb-seg-147 -> 73\n",
      "gene: Chr7-6Mb-seg-148 -> 72\n",
      "gene: Chr7-6Mb-seg-149 -> 59\n",
      "gene: Chr7-6Mb-seg-150 -> 63\n",
      "gene: Chr7-6Mb-seg-151 -> 53\n",
      "gene: Chr7-6Mb-seg-152 -> 61\n",
      "gene: Chr7-6Mb-seg-153 -> 70\n",
      "gene: Chr7-6Mb-seg-154 -> 80\n",
      "gene: Chr7-6Mb-seg-155 -> 67\n",
      "gene: Chr7-6Mb-seg-156 -> 64\n",
      "gene: Chr7-6Mb-seg-157 -> 72\n",
      "gene: Chr7-6Mb-seg-158 -> 72\n",
      "gene: Chr7-6Mb-seg-159 -> 62\n",
      "gene: Chr7-6Mb-seg-161 -> 29\n",
      "gene: Chr7-6Mb-seg-162 -> 61\n",
      "gene: Chr7-6Mb-seg-163 -> 75\n",
      "gene: Chr7-6Mb-seg-164 -> 80\n",
      "gene: Chr7-6Mb-seg-165 -> 71\n",
      "gene: Chr7-6Mb-seg-166 -> 80\n",
      "gene: Chr7-6Mb-seg-167 -> 78\n",
      "gene: Chr7-6Mb-seg-168 -> 80\n",
      "gene: Chr7-6Mb-seg-169 -> 80\n",
      "gene: Chr7-6Mb-seg-170 -> 80\n",
      "gene: Chr7-6Mb-seg-171 -> 80\n",
      "gene: Chr7-6Mb-seg-172 -> 78\n",
      "gene: Chr7-6Mb-seg-173 -> 80\n",
      "gene: Chr7-6Mb-seg-174 -> 80\n",
      "gene: Chr7-6Mb-seg-175 -> 59\n",
      "gene: Chr7-6Mb-seg-176 -> 79\n",
      "gene: Chr7-6Mb-seg-177 -> 79\n",
      "gene: Chr7-6Mb-seg-178 -> 80\n",
      "gene: Chr7-6Mb-seg-179 -> 67\n",
      "gene: Chr7-6Mb-seg-180 -> 80\n",
      "gene: Chr7-6Mb-seg-181 -> 80\n",
      "gene: Chr7-6Mb-seg-182 -> 80\n",
      "gene: Chr7-6Mb-seg-183 -> 70\n",
      "gene: Chr7-6Mb-seg-184 -> 75\n",
      "gene: Chr7-6Mb-seg-185 -> 57\n",
      "gene: Chr7-6Mb-seg-186 -> 72\n",
      "gene: Chr7-6Mb-seg-187 -> 56\n",
      "gene: Chr7-6Mb-seg-188 -> 54\n",
      "gene: Chr7-6Mb-seg-189 -> 57\n",
      "gene: Chr7-6Mb-seg-190 -> 54\n",
      "gene: Chr7-6Mb-seg-191 -> 69\n",
      "gene: Chr7-6Mb-seg-192 -> 75\n",
      "gene: Chr7-6Mb-seg-193 -> 71\n",
      "gene: Chr7-6Mb-seg-194 -> 60\n",
      "gene: Chr7-6Mb-seg-195 -> 75\n",
      "gene: Chr7-6Mb-seg-196 -> 73\n",
      "gene: Chr7-6Mb-seg-197 -> 74\n",
      "gene: Chr7-6Mb-seg-198 -> 67\n",
      "gene: Chr7-6Mb-seg-199 -> 76\n",
      "gene: Chr7-6Mb-seg-200 -> 80\n",
      "gene: Chr7-6Mb-seg-201 -> 80\n",
      "gene: Chr7-6Mb-seg-202 -> 80\n",
      "gene: Chr7-6Mb-seg-203 -> 79\n",
      "gene: Chr7-6Mb-seg-204 -> 80\n",
      "gene: Chr7-6Mb-seg-205 -> 63\n",
      "gene: Chr7-6Mb-seg-206 -> 78\n",
      "gene: Chr7-6Mb-seg-207 -> 71\n",
      "gene: Chr7-6Mb-seg-208 -> 76\n",
      "gene: Chr7-6Mb-seg-209 -> 71\n",
      "gene: Chr7-6Mb-seg-210 -> 80\n",
      "gene: Chr7-6Mb-seg-211 -> 80\n",
      "gene: Chr7-6Mb-seg-212 -> 71\n",
      "gene: Chr7-6Mb-seg-213 -> 66\n",
      "gene: Chr7-6Mb-seg-214 -> 61\n",
      "gene: Chr7-6Mb-seg-215 -> 55\n",
      "gene: Chr7-6Mb-seg-216 -> 79\n",
      "gene: Chr7-6Mb-seg-217 -> 61\n",
      "gene: Chr7-6Mb-seg-218 -> 60\n",
      "gene: Chr7-6Mb-seg-219 -> 70\n",
      "gene: Chr7-6Mb-seg-220 -> 54\n",
      "gene: Chr7-6Mb-seg-221 -> 67\n",
      "gene: Chr7-6Mb-seg-222 -> 51\n",
      "gene: Chr7-6Mb-seg-223 -> 54\n",
      "gene: Chr7-6Mb-seg-224 -> 57\n",
      "gene: Chr7-6Mb-seg-225 -> 76\n",
      "gene: Chr7-6Mb-seg-226 -> 80\n",
      "gene: Chr7-6Mb-seg-227 -> 73\n",
      "gene: Chr7-6Mb-seg-228 -> 68\n",
      "gene: Chr7-6Mb-seg-229 -> 76\n",
      "gene: Chr7-6Mb-seg-230 -> 74\n",
      "gene: Chr7-6Mb-seg-231 -> 70\n",
      "gene: Chr7-6Mb-seg-232 -> 79\n",
      "gene: Chr7-6Mb-seg-233 -> 73\n",
      "gene: Chr7-6Mb-seg-234 -> 80\n",
      "gene: Chr7-6Mb-seg-235 -> 72\n",
      "gene: Chr7-6Mb-seg-236 -> 80\n",
      "gene: Chr7-6Mb-seg-237 -> 80\n",
      "gene: Chr7-6Mb-seg-238 -> 80\n",
      "gene: Chr7-6Mb-seg-239 -> 71\n",
      "gene: Chr7-6Mb-seg-240 -> 80\n",
      "gene: Chr7-6Mb-seg-241 -> 69\n",
      "gene: Chr7-6Mb-seg-242 -> 58\n",
      "gene: Chr7-6Mb-seg-243 -> 71\n",
      "gene: Chr7-6Mb-seg-244 -> 71\n",
      "gene: Chr7-6Mb-seg-245 -> 74\n",
      "gene: Chr7-6Mb-seg-246 -> 69\n",
      "gene: Chr7-6Mb-seg-247 -> 74\n",
      "gene: Chr7-6Mb-seg-248 -> 77\n",
      "gene: Chr7-6Mb-seg-249 -> 55\n",
      "gene: Chr7-6Mb-seg-250 -> 21\n",
      "gene: Chr7-6Mb-seg-251 -> 46\n",
      "gene: Chr7-6Mb-seg-252 -> 70\n",
      "gene: Chr7-6Mb-seg-253 -> 79\n",
      "gene: Chr7-6Mb-seg-254 -> 62\n",
      "gene: Chr7-6Mb-seg-255 -> 74\n",
      "gene: Chr7-6Mb-seg-256 -> 71\n",
      "gene: Chr7-6Mb-seg-257 -> 71\n",
      "gene: Chr7-6Mb-seg-258 -> 75\n",
      "gene: Chr7-6Mb-seg-259 -> 68\n",
      "gene: Chr7-6Mb-seg-260 -> 63\n",
      "gene: Chr7-6Mb-seg-261 -> 75\n",
      "gene: Chr7-6Mb-seg-262 -> 71\n",
      "gene: Chr7-6Mb-seg-263 -> 74\n",
      "gene: Chr7-6Mb-seg-264 -> 67\n",
      "gene: Chr7-6Mb-seg-265 -> 69\n",
      "gene: Chr7-6Mb-seg-266 -> 56\n",
      "gene: Chr7-6Mb-seg-267 -> 76\n",
      "gene: Chr7-6Mb-seg-268 -> 80\n",
      "gene: Chr7-6Mb-seg-269 -> 71\n",
      "gene: Chr7-6Mb-seg-270 -> 64\n",
      "gene: Chr7-6Mb-seg-271 -> 69\n",
      "gene: Chr7-6Mb-seg-272 -> 77\n",
      "gene: Chr7-6Mb-seg-273 -> 77\n",
      "gene: Chr7-6Mb-seg-274 -> 80\n",
      "gene: Chr7-6Mb-seg-275 -> 71\n",
      "gene: Chr7-6Mb-seg-276 -> 67\n",
      "gene: Chr7-6Mb-seg-277 -> 80\n",
      "gene: Chr7-6Mb-seg-278 -> 70\n",
      "gene: Chr7-6Mb-seg-279 -> 65\n",
      "gene: Chr7-6Mb-seg-280 -> 48\n",
      "gene: Chr7-6Mb-seg-281 -> 48\n",
      "gene: Chr7-6Mb-seg-282 -> 72\n",
      "gene: Chr7-6Mb-seg-283 -> 74\n",
      "gene: Chr7-6Mb-seg-284 -> 80\n",
      "gene: Chr7-6Mb-seg-285 -> 80\n",
      "gene: Chr7-6Mb-seg-286 -> 73\n",
      "gene: Chr7-6Mb-seg-287 -> 80\n",
      "gene: Chr7-6Mb-seg-288 -> 80\n",
      "gene: Chr7-6Mb-seg-289 -> 69\n",
      "gene: Chr7-6Mb-seg-290 -> 60\n",
      "gene: Chr7-6Mb-seg-291 -> 46\n",
      "gene: Chr7-6Mb-seg-292 -> 71\n",
      "gene: Chr7-6Mb-seg-293 -> 72\n",
      "gene: Chr7-6Mb-seg-294 -> 70\n",
      "gene: Chr7-6Mb-seg-295 -> 46\n",
      "gene: Chr7-6Mb-seg-296 -> 61\n",
      "gene: Chr7-6Mb-seg-297 -> 36\n",
      "gene: Chr7-6Mb-seg-298 -> 61\n",
      "gene: Chr7-6Mb-seg-299 -> 73\n",
      "gene: Chr7-6Mb-seg-300 -> 71\n",
      "gene: Chr7-6Mb-seg-301 -> 69\n",
      "gene: Chr7-6Mb-seg-302 -> 70\n",
      "gene: Chr7-6Mb-seg-303 -> 48\n",
      "gene: Chr7-6Mb-seg-304 -> 79\n",
      "gene: Chr7-6Mb-seg-305 -> 67\n",
      "gene: Chr7-6Mb-seg-306 -> 61\n",
      "gene: Chr7-6Mb-seg-307 -> 70\n",
      "gene: Chr7-6Mb-seg-308 -> 62\n",
      "gene: Chr7-6Mb-seg-309 -> 54\n",
      "gene: Chr7-6Mb-seg-310 -> 56\n",
      "gene: Chr7-6Mb-seg-311 -> 74\n",
      "gene: Chr7-6Mb-seg-312 -> 59\n",
      "gene: Chr7-6Mb-seg-313 -> 60\n",
      "gene: Chr7-6Mb-seg-314 -> 64\n",
      "gene: Chr7-6Mb-seg-315 -> 72\n",
      "gene: Chr7-6Mb-seg-316 -> 50\n",
      "gene: Chr7-6Mb-seg-317 -> 53\n",
      "gene: Chr7-6Mb-seg-318 -> 76\n",
      "gene: Chr7-6Mb-seg-319 -> 48\n",
      "gene: Chr7-6Mb-seg-320 -> 74\n",
      "gene: Chr7-6Mb-seg-321 -> 40\n",
      "gene: Chr7-6Mb-seg-322 -> 60\n",
      "gene: Chr7-6Mb-seg-323 -> 54\n",
      "gene: Chr7-6Mb-seg-324 -> 80\n",
      "gene: Chr7-6Mb-seg-325 -> 78\n",
      "gene: Chr7-6Mb-seg-326 -> 80\n",
      "gene: Chr7-6Mb-seg-327 -> 78\n",
      "gene: Chr7-6Mb-seg-328 -> 80\n",
      "gene: Chr7-6Mb-seg-329 -> 80\n",
      "gene: Chr7-6Mb-seg-330 -> 80\n",
      "gene: Chr7-6Mb-seg-331 -> 74\n",
      "gene: Chr7-6Mb-seg-332 -> 59\n",
      "gene: Chr7-6Mb-seg-333 -> 64\n",
      "gene: Chr7-6Mb-seg-334 -> 69\n",
      "gene: Chr7-6Mb-seg-335 -> 62\n",
      "gene: Chr7-6Mb-seg-336 -> 80\n",
      "gene: Chr7-6Mb-seg-337 -> 80\n",
      "gene: Chr7-6Mb-seg-338 -> 66\n",
      "gene: Chr7-6Mb-seg-339 -> 80\n",
      "gene: Chr7-6Mb-seg-340 -> 55\n",
      "gene: Chr7-6Mb-seg-341 -> 80\n",
      "gene: Chr7-6Mb-seg-342 -> 80\n",
      "gene: Chr7-6Mb-seg-343 -> 79\n",
      "gene: Chr7-6Mb-seg-344 -> 80\n",
      "gene: Chr7-6Mb-seg-345 -> 80\n",
      "gene: Chr7-6Mb-seg-346 -> 78\n",
      "gene: Chr7-6Mb-seg-347 -> 72\n",
      "gene: Chr7-6Mb-seg-348 -> 46\n",
      "gene: Chr7-6Mb-seg-349 -> 68\n",
      "gene: Chr7-6Mb-seg-350 -> 75\n",
      "gene: Chr7-6Mb-seg-351 -> 55\n",
      "gene: Chr7-6Mb-seg-352 -> 78\n",
      "gene: Chr7-6Mb-seg-353 -> 51\n",
      "gene: Chr7-6Mb-seg-354 -> 53\n",
      "gene: Chr7-6Mb-seg-355 -> 62\n",
      "gene: Chr7-6Mb-seg-356 -> 72\n",
      "gene: Chr7-6Mb-seg-357 -> 74\n",
      "gene: Chr7-6Mb-seg-358 -> 80\n",
      "gene: Chr7-6Mb-seg-359 -> 79\n",
      "gene: Chr7-6Mb-seg-360 -> 73\n",
      "gene: Chr7-6Mb-seg-361 -> 80\n",
      "gene: Chr7-6Mb-seg-362 -> 80\n",
      "gene: Chr7-6Mb-seg-363 -> 80\n",
      "gene: Chr7-6Mb-seg-364 -> 80\n",
      "gene: Chr7-6Mb-seg-365 -> 73\n",
      "gene: Chr7-6Mb-seg-366 -> 80\n",
      "gene: Chr7-6Mb-seg-367 -> 80\n",
      "gene: Chr7-6Mb-seg-368 -> 56\n",
      "gene: Chr7-6Mb-seg-369 -> 79\n",
      "gene: Chr7-6Mb-seg-370 -> 55\n",
      "gene: Chr7-6Mb-seg-371 -> 80\n",
      "gene: Chr7-6Mb-seg-372 -> 70\n",
      "gene: Chr7-6Mb-seg-373 -> 80\n",
      "gene: Chr7-6Mb-seg-374 -> 80\n",
      "gene: Chr7-6Mb-seg-375 -> 80\n",
      "gene: Chr7-6Mb-seg-376 -> 46\n",
      "gene: Chr7-6Mb-seg-377 -> 64\n",
      "gene: Chr7-6Mb-seg-378 -> 68\n",
      "gene: Chr7-6Mb-seg-379 -> 73\n",
      "gene: Chr7-6Mb-seg-380 -> 75\n",
      "gene: Chr7-6Mb-seg-381 -> 55\n",
      "gene: Chr7-6Mb-seg-382 -> 60\n",
      "gene: Chr7-6Mb-seg-383 -> 44\n",
      "gene: Chr7-6Mb-seg-384 -> 79\n",
      "gene: Chr7-6Mb-seg-385 -> 80\n",
      "gene: Chr7-6Mb-seg-386 -> 42\n",
      "gene: Chr7-6Mb-seg-387 -> 42\n",
      "gene: Chr7-6Mb-seg-388 -> 75\n",
      "gene: Chr7-6Mb-seg-389 -> 64\n",
      "gene: Chr7-6Mb-seg-390 -> 76\n",
      "gene: Chr7-6Mb-seg-391 -> 40\n",
      "gene: Chr7-6Mb-seg-392 -> 71\n",
      "gene: Chr7-6Mb-seg-393 -> 46\n",
      "gene: Chr7-6Mb-seg-394 -> 80\n",
      "gene: Chr7-6Mb-seg-395 -> 76\n",
      "gene: Chr7-6Mb-seg-396 -> 74\n",
      "gene: Chr7-6Mb-seg-397 -> 71\n",
      "gene: Chr7-6Mb-seg-398 -> 80\n",
      "gene: Chr7-6Mb-seg-399 -> 74\n",
      "gene: Chr7-6Mb-seg-400 -> 79\n",
      "gene: Chr7-6Mb-seg-401 -> 62\n",
      "gene: Chr7-6Mb-seg-402 -> 80\n",
      "gene: Chr7-6Mb-seg-403 -> 65\n",
      "gene: Chr7-6Mb-seg-404 -> 80\n",
      "gene: Chr7-6Mb-seg-405 -> 62\n",
      "gene: Chr7-6Mb-seg-406 -> 64\n",
      "gene: Chr7-6Mb-seg-407 -> 75\n",
      "gene: Chr7-6Mb-seg-408 -> 72\n",
      "gene: Chr7-6Mb-seg-409 -> 77\n",
      "gene: Chr7-6Mb-seg-410 -> 71\n",
      "gene: Chr7-6Mb-seg-411 -> 63\n",
      "gene: Chr7-6Mb-seg-412 -> 72\n",
      "gene: Chr7-6Mb-seg-413 -> 79\n",
      "gene: Chr7-6Mb-seg-414 -> 72\n",
      "gene: Chr7-6Mb-seg-415 -> 71\n",
      "gene: Chr7-6Mb-seg-416 -> 75\n",
      "gene: Chr7-6Mb-seg-417 -> 80\n",
      "gene: Chr7-6Mb-seg-418 -> 70\n",
      "gene: Chr7-6Mb-seg-419 -> 65\n",
      "gene: Chr7-6Mb-seg-420 -> 80\n",
      "gene: Chr7-6Mb-seg-421 -> 77\n",
      "gene: Chr7-6Mb-seg-422 -> 80\n",
      "gene: Chr7-6Mb-seg-423 -> 80\n",
      "gene: Chr7-6Mb-seg-424 -> 80\n",
      "gene: Chr7-6Mb-seg-425 -> 80\n",
      "gene: Chr7-6Mb-seg-426 -> 80\n",
      "gene: Chr7-6Mb-seg-427 -> 80\n",
      "gene: Chr7-6Mb-seg-428 -> 80\n",
      "gene: Chr7-6Mb-seg-429 -> 65\n",
      "gene: Chr7-6Mb-seg-430 -> 74\n",
      "gene: Chr7-6Mb-seg-431 -> 79\n",
      "gene: Chr7-6Mb-seg-432 -> 80\n",
      "gene: Chr7-6Mb-seg-433 -> 73\n",
      "gene: Chr7-6Mb-seg-434 -> 73\n",
      "gene: Chr7-6Mb-seg-435 -> 72\n",
      "gene: Chr7-6Mb-seg-436 -> 80\n",
      "gene: Chr7-6Mb-seg-437 -> 76\n",
      "gene: Chr7-6Mb-seg-438 -> 66\n",
      "gene: Chr7-6Mb-seg-439 -> 69\n",
      "gene: Chr7-6Mb-seg-440 -> 80\n",
      "gene: Chr7-6Mb-seg-441 -> 80\n",
      "gene: Chr7-6Mb-seg-442 -> 80\n",
      "gene: Chr7-6Mb-seg-443 -> 80\n",
      "gene: Chr7-6Mb-seg-444 -> 80\n",
      "gene: Chr7-6Mb-seg-445 -> 78\n",
      "gene: Chr7-6Mb-seg-446 -> 80\n",
      "gene: Chr7-6Mb-seg-447 -> 73\n",
      "gene: Chr7-6Mb-seg-448 -> 76\n",
      "gene: Chr7-6Mb-seg-449 -> 75\n",
      "gene: Chr7-6Mb-seg-450 -> 80\n",
      "gene: Chr7-6Mb-seg-451 -> 77\n",
      "gene: Chr7-6Mb-seg-452 -> 75\n",
      "gene: Chr7-6Mb-seg-453 -> 80\n",
      "gene: Chr7-6Mb-seg-454 -> 75\n",
      "gene: Chr7-6Mb-seg-455 -> 80\n",
      "gene: Chr7-6Mb-seg-456 -> 75\n",
      "gene: Chr7-6Mb-seg-457 -> 77\n",
      "gene: Chr7-6Mb-seg-458 -> 80\n",
      "gene: Chr7-6Mb-seg-459 -> 72\n",
      "gene: Chr7-6Mb-seg-460 -> 80\n",
      "gene: Chr7-6Mb-seg-461 -> 75\n",
      "gene: Chr7-6Mb-seg-462 -> 77\n",
      "gene: Chr7-6Mb-seg-463 -> 80\n",
      "gene: Chr7-6Mb-seg-464 -> 73\n",
      "gene: Chr7-6Mb-seg-465 -> 67\n",
      "gene: Chr7-6Mb-seg-466 -> 77\n",
      "gene: Chr7-6Mb-seg-467 -> 55\n",
      "gene: Chr7-6Mb-seg-468 -> 75\n",
      "gene: Chr7-6Mb-seg-469 -> 80\n",
      "gene: Chr7-6Mb-seg-470 -> 80\n",
      "gene: Chr7-6Mb-seg-471 -> 79\n",
      "gene: Chr7-6Mb-seg-472 -> 72\n",
      "gene: Chr7-6Mb-seg-473 -> 74\n",
      "gene: Chr7-6Mb-seg-474 -> 80\n",
      "gene: Chr7-6Mb-seg-475 -> 74\n",
      "gene: Chr7-6Mb-seg-476 -> 80\n",
      "gene: Chr7-6Mb-seg-477 -> 68\n",
      "gene: Chr7-6Mb-seg-478 -> 51\n",
      "gene: Chr7-6Mb-seg-479 -> 73\n",
      "gene: Chr7-6Mb-seg-480 -> 80\n",
      "gene: Chr7-6Mb-seg-481 -> 80\n",
      "gene: Chr7-6Mb-seg-482 -> 80\n",
      "gene: Chr7-6Mb-seg-483 -> 80\n",
      "gene: Chr7-6Mb-seg-484 -> 80\n",
      "gene: Chr7-6Mb-seg-485 -> 80\n",
      "gene: Chr7-6Mb-seg-486 -> 80\n",
      "gene: Chr7-6Mb-seg-487 -> 80\n",
      "gene: Chr7-6Mb-seg-488 -> 80\n",
      "gene: Chr7-6Mb-seg-489 -> 80\n",
      "gene: Chr7-6Mb-seg-490 -> 70\n",
      "gene: Chr7-6Mb-seg-491 -> 59\n",
      "gene: Chr7-6Mb-seg-492 -> 77\n",
      "gene: Chr7-6Mb-seg-493 -> 79\n",
      "gene: Chr7-6Mb-seg-494 -> 80\n",
      "gene: Chr7-6Mb-seg-495 -> 75\n",
      "gene: Chr7-6Mb-seg-496 -> 80\n",
      "gene: Chr7-6Mb-seg-497 -> 74\n",
      "gene: Chr7-6Mb-seg-498 -> 59\n",
      "gene: Chr7-6Mb-seg-499 -> 74\n",
      "gene: Chr7-6Mb-seg-500 -> 68\n",
      "gene: Chr7-6Mb-seg-501 -> 80\n",
      "gene: Chr7-6Mb-seg-502 -> 67\n",
      "gene: Chr7-6Mb-seg-503 -> 63\n",
      "gene: Chr7-6Mb-seg-504 -> 73\n",
      "gene: Chr7-6Mb-seg-505 -> 70\n",
      "gene: Chr7-6Mb-seg-506 -> 73\n",
      "gene: Chr7-6Mb-seg-507 -> 80\n",
      "gene: Chr7-6Mb-seg-508 -> 76\n",
      "gene: Chr7-6Mb-seg-509 -> 62\n",
      "gene: Chr7-6Mb-seg-510 -> 76\n",
      "gene: Chr7-6Mb-seg-511 -> 80\n",
      "gene: Chr7-6Mb-seg-512 -> 80\n",
      "gene: Chr7-6Mb-seg-513 -> 76\n",
      "gene: Chr7-6Mb-seg-514 -> 68\n",
      "gene: Chr7-6Mb-seg-515 -> 70\n",
      "gene: Chr7-6Mb-seg-516 -> 71\n",
      "gene: Chr7-6Mb-seg-517 -> 77\n",
      "gene: Chr7-6Mb-seg-518 -> 80\n",
      "gene: Chr7-6Mb-seg-519 -> 69\n",
      "gene: Chr7-6Mb-seg-520 -> 67\n",
      "gene: Chr7-6Mb-seg-521 -> 80\n",
      "gene: Chr7-6Mb-seg-522 -> 80\n",
      "gene: Chr7-6Mb-seg-523 -> 80\n",
      "gene: Chr7-6Mb-seg-524 -> 72\n",
      "gene: Chr7-6Mb-seg-525 -> 63\n",
      "gene: Chr7-6Mb-seg-526 -> 59\n",
      "gene: Chr7-6Mb-seg-527 -> 70\n",
      "gene: Chr7-6Mb-seg-528 -> 80\n",
      "gene: Chr7-6Mb-seg-529 -> 68\n",
      "gene: Chr7-6Mb-seg-530 -> 68\n",
      "gene: Chr7-6Mb-seg-531 -> 65\n",
      "gene: Chr7-6Mb-seg-532 -> 73\n",
      "gene: Chr7-6Mb-seg-533 -> 74\n",
      "gene: Chr7-6Mb-seg-534 -> 79\n",
      "gene: Chr7-6Mb-seg-535 -> 75\n",
      "gene: Chr7-6Mb-seg-536 -> 62\n",
      "gene: Chr7-6Mb-seg-537 -> 70\n",
      "gene: Chr7-6Mb-seg-538 -> 80\n",
      "gene: Chr7-6Mb-seg-539 -> 33\n",
      "gene: Chr7-6Mb-seg-540 -> 49\n",
      "gene: Chr7-6Mb-seg-541 -> 80\n",
      "gene: Chr7-6Mb-seg-542 -> 67\n",
      "gene: Chr7-6Mb-seg-543 -> 72\n",
      "gene: Chr7-6Mb-seg-544 -> 77\n",
      "gene: Chr7-6Mb-seg-545 -> 63\n",
      "gene: Chr7-6Mb-seg-546 -> 66\n",
      "gene: Chr7-6Mb-seg-547 -> 75\n",
      "gene: Chr7-6Mb-seg-548 -> 80\n",
      "gene: Chr7-6Mb-seg-549 -> 62\n",
      "gene: Chr7-6Mb-seg-550 -> 79\n",
      "gene: Chr7-6Mb-seg-551 -> 80\n",
      "gene: Chr7-6Mb-seg-552 -> 80\n",
      "gene: Chr7-6Mb-seg-553 -> 72\n",
      "gene: Chr7-6Mb-seg-554 -> 63\n",
      "gene: Chr7-6Mb-seg-555 -> 68\n",
      "gene: Chr7-6Mb-seg-556 -> 73\n",
      "gene: Chr7-6Mb-seg-557 -> 62\n",
      "gene: Chr7-6Mb-seg-558 -> 65\n",
      "gene: Chr7-6Mb-seg-559 -> 59\n",
      "gene: Chr7-6Mb-seg-560 -> 64\n",
      "gene: Chr7-6Mb-seg-561 -> 56\n",
      "gene: Chr7-6Mb-seg-562 -> 64\n",
      "gene: Chr7-6Mb-seg-563 -> 50\n",
      "gene: Chr7-6Mb-seg-564 -> 57\n",
      "gene: Chr7-6Mb-seg-565 -> 67\n",
      "gene: Chr7-6Mb-seg-566 -> 61\n",
      "gene: Chr7-6Mb-seg-567 -> 75\n",
      "gene: Chr7-6Mb-seg-568 -> 72\n",
      "gene: Chr7-6Mb-seg-569 -> 75\n",
      "gene: Chr7-6Mb-seg-570 -> 74\n",
      "gene: Chr7-6Mb-seg-571 -> 65\n",
      "gene: Chr7-6Mb-seg-572 -> 59\n",
      "gene: Chr7-6Mb-seg-573 -> 54\n",
      "gene: Chr7-6Mb-seg-574 -> 48\n",
      "gene: Chr7-6Mb-seg-575 -> 57\n",
      "gene: Chr7-6Mb-seg-576 -> 64\n",
      "gene: Chr7-6Mb-seg-577 -> 52\n",
      "gene: Chr7-6Mb-seg-578 -> 48\n",
      "gene: Chr7-6Mb-seg-579 -> 65\n",
      "gene: Chr7-6Mb-seg-580 -> 61\n",
      "gene: Chr7-6Mb-seg-581 -> 57\n",
      "gene: Chr7-6Mb-seg-582 -> 67\n",
      "gene: Chr7-6Mb-seg-583 -> 61\n",
      "gene: Chr7-6Mb-seg-584 -> 69\n",
      "gene: Chr7-6Mb-seg-585 -> 56\n",
      "gene: Chr7-6Mb-seg-586 -> 61\n",
      "gene: Chr7-6Mb-seg-587 -> 52\n",
      "gene: Chr7-6Mb-seg-588 -> 62\n",
      "gene: Chr7-6Mb-seg-589 -> 62\n",
      "gene: Chr7-6Mb-seg-590 -> 49\n",
      "gene: Chr7-6Mb-seg-591 -> 80\n",
      "gene: Chr7-6Mb-seg-592 -> 72\n",
      "gene: Chr7-6Mb-seg-593 -> 61\n",
      "gene: Chr7-6Mb-seg-594 -> 66\n",
      "gene: Chr7-6Mb-seg-595 -> 39\n",
      "gene: Chr7-6Mb-seg-596 -> 51\n",
      "gene: Chr7-6Mb-seg-597 -> 55\n",
      "gene: Chr7-6Mb-seg-598 -> 71\n",
      "gene: Chr7-6Mb-seg-599 -> 66\n",
      "gene: Chr7-6Mb-seg-600 -> 79\n",
      "gene: Chr7-6Mb-seg-601 -> 74\n",
      "gene: Chr7-6Mb-seg-602 -> 71\n",
      "gene: Chr7-6Mb-seg-603 -> 53\n",
      "gene: Chr7-6Mb-seg-604 -> 57\n",
      "gene: Chr7-6Mb-seg-605 -> 70\n",
      "gene: Chr7-6Mb-seg-606 -> 55\n",
      "gene: Chr7-6Mb-seg-607 -> 53\n",
      "gene: Chr7-6Mb-seg-608 -> 39\n",
      "gene: Chr7-6Mb-seg-609 -> 73\n",
      "gene: Chr7-6Mb-seg-610 -> 71\n",
      "gene: Chr7-6Mb-seg-611 -> 80\n",
      "gene: Chr7-6Mb-seg-612 -> 80\n",
      "gene: Chr7-6Mb-seg-613 -> 80\n",
      "gene: Chr7-6Mb-seg-614 -> 72\n",
      "gene: Chr7-6Mb-seg-615 -> 80\n",
      "gene: Chr7-6Mb-seg-616 -> 80\n",
      "gene: Chr7-6Mb-seg-617 -> 80\n",
      "gene: Chr7-6Mb-seg-618 -> 80\n",
      "gene: Chr7-6Mb-seg-619 -> 70\n",
      "gene: Chr7-6Mb-seg-620 -> 78\n",
      "gene: Chr7-6Mb-seg-621 -> 76\n",
      "gene: Chr7-6Mb-seg-622 -> 80\n",
      "gene: Chr7-6Mb-seg-623 -> 80\n",
      "gene: Chr7-6Mb-seg-624 -> 80\n",
      "gene: Chr7-6Mb-seg-625 -> 75\n",
      "gene: Chr7-6Mb-seg-626 -> 74\n",
      "gene: Chr7-6Mb-seg-627 -> 80\n",
      "gene: Chr7-6Mb-seg-628 -> 54\n",
      "gene: Chr7-6Mb-seg-629 -> 74\n",
      "gene: Chr7-6Mb-seg-630 -> 68\n",
      "gene: Chr7-6Mb-seg-631 -> 74\n",
      "gene: Chr7-6Mb-seg-632 -> 57\n",
      "gene: Chr7-6Mb-seg-633 -> 80\n",
      "gene: Chr7-6Mb-seg-634 -> 79\n",
      "gene: Chr7-6Mb-seg-635 -> 57\n",
      "gene: Chr7-6Mb-seg-638 -> 27\n",
      "gene: Chr7-6Mb-seg-639 -> 51\n",
      "gene: Chr7-6Mb-seg-640 -> 24\n",
      "gene: Chr7-6Mb-seg-641 -> 37\n",
      "gene: Chr7-6Mb-seg-642 -> 54\n",
      "gene: Chr7-6Mb-seg-643 -> 59\n",
      "gene: Chr7-6Mb-seg-644 -> 52\n",
      "gene: Chr7-6Mb-seg-647 -> 27\n",
      "gene: Chr7-6Mb-seg-648 -> 50\n",
      "gene: Chr7-6Mb-seg-649 -> 26\n",
      "gene: Chr7-6Mb-seg-650 -> 40\n",
      "gene: Chr7-6Mb-seg-651 -> 57\n",
      "gene: Chr7-6Mb-seg-655 -> 35\n",
      "gene: Chr7-6Mb-seg-660 -> 35\n",
      "gene: Chr7-6Mb-seg-661 -> 42\n",
      "gene: Chr7-6Mb-seg-662 -> 58\n",
      "gene: Chr7-6Mb-seg-663 -> 50\n",
      "gene: Chr7-6Mb-seg-664 -> 41\n",
      "gene: Chr7-6Mb-seg-665 -> 30\n",
      "gene: Chr7-6Mb-seg-668 -> 58\n",
      "gene: Chr7-6Mb-seg-669 -> 56\n",
      "gene: Chr7-6Mb-seg-670 -> 68\n",
      "gene: Chr7-6Mb-seg-671 -> 54\n",
      "gene: Chr7-6Mb-seg-672 -> 51\n",
      "gene: Chr7-6Mb-seg-674 -> 32\n",
      "gene: Chr7-6Mb-seg-675 -> 48\n",
      "gene: Chr7-6Mb-seg-676 -> 38\n",
      "gene: Chr7-6Mb-seg-677 -> 36\n",
      "gene: Chr7-6Mb-seg-679 -> 51\n",
      "gene: Chr7-6Mb-seg-680 -> 37\n",
      "gene: Chr7-6Mb-seg-681 -> 33\n",
      "gene: Chr7-6Mb-seg-682 -> 48\n",
      "gene: Chr7-6Mb-seg-683 -> 27\n",
      "gene: Chr7-6Mb-seg-692 -> 32\n",
      "gene: Chr7-6Mb-seg-693 -> 39\n",
      "gene: Chr7-6Mb-seg-698 -> 32\n",
      "gene: Chr7-6Mb-seg-699 -> 28\n",
      "gene: Chr7-6Mb-seg-700 -> 37\n",
      "gene: Chr7-6Mb-seg-701 -> 41\n",
      "gene: Chr7-6Mb-seg-703 -> 25\n",
      "gene: Chr7-6Mb-seg-704 -> 41\n",
      "gene: Chr7-6Mb-seg-705 -> 31\n",
      "gene: Chr7-6Mb-seg-706 -> 46\n",
      "gene: Chr7-6Mb-seg-707 -> 55\n",
      "gene: Chr7-6Mb-seg-708 -> 31\n",
      "gene: Chr7-6Mb-seg-710 -> 29\n",
      "gene: Chr7-6Mb-seg-711 -> 24\n",
      "gene: Chr7-6Mb-seg-712 -> 27\n",
      "gene: Chr7-6Mb-seg-713 -> 46\n",
      "gene: Chr7-6Mb-seg-717 -> 23\n",
      "gene: Chr7-6Mb-seg-718 -> 24\n",
      "gene: Chr7-6Mb-seg-719 -> 29\n",
      "gene: Chr7-6Mb-seg-720 -> 41\n",
      "gene: Chr7-6Mb-seg-721 -> 55\n",
      "gene: Chr7-6Mb-seg-722 -> 52\n",
      "gene: Chr7-6Mb-seg-723 -> 32\n",
      "gene: Chr7-6Mb-seg-726 -> 39\n",
      "gene: Chr7-6Mb-seg-727 -> 40\n",
      "gene: Chr7-6Mb-seg-728 -> 22\n",
      "gene: Chr7-6Mb-seg-731 -> 51\n",
      "gene: Chr7-6Mb-seg-732 -> 23\n",
      "gene: Chr7-6Mb-seg-733 -> 57\n",
      "gene: Chr7-6Mb-seg-734 -> 25\n",
      "gene: Chr7-6Mb-seg-735 -> 46\n",
      "gene: Chr7-6Mb-seg-738 -> 38\n",
      "gene: Chr7-6Mb-seg-739 -> 41\n",
      "gene: Chr7-6Mb-seg-740 -> 61\n",
      "gene: Chr7-6Mb-seg-741 -> 55\n",
      "gene: Chr7-6Mb-seg-742 -> 37\n",
      "gene: Chr7-6Mb-seg-743 -> 60\n",
      "gene: Chr7-6Mb-seg-744 -> 44\n",
      "gene: Chr7-6Mb-seg-745 -> 32\n",
      "gene: Chr7-6Mb-seg-746 -> 35\n",
      "gene: Chr7-6Mb-seg-747 -> 28\n",
      "gene: Chr7-6Mb-seg-748 -> 54\n",
      "gene: Chr7-6Mb-seg-749 -> 52\n",
      "gene: Chr7-6Mb-seg-750 -> 52\n",
      "gene: Chr7-6Mb-seg-751 -> 45\n",
      "gene: Chr7-6Mb-seg-752 -> 60\n",
      "gene: Chr7-6Mb-seg-753 -> 48\n",
      "gene: Chr7-6Mb-seg-754 -> 25\n",
      "gene: Chr7-6Mb-seg-755 -> 48\n",
      "gene: Chr7-6Mb-seg-756 -> 52\n",
      "gene: Chr7-6Mb-seg-757 -> 50\n",
      "gene: Chr7-6Mb-seg-758 -> 56\n",
      "gene: Chr7-6Mb-seg-760 -> 27\n",
      "gene: Chr7-6Mb-seg-762 -> 23\n",
      "gene: Chr7-6Mb-seg-763 -> 40\n",
      "gene: Chr7-6Mb-seg-764 -> 42\n",
      "gene: Chr7-6Mb-seg-765 -> 24\n",
      "gene: Chr7-6Mb-seg-766 -> 50\n",
      "gene: Chr7-6Mb-seg-767 -> 49\n",
      "gene: Chr7-6Mb-seg-768 -> 42\n",
      "gene: Chr7-6Mb-seg-769 -> 24\n",
      "gene: Chr7-6Mb-seg-770 -> 21\n",
      "gene: Chr7-6Mb-seg-771 -> 28\n",
      "gene: Chr7-6Mb-seg-772 -> 47\n",
      "gene: Chr7-6Mb-seg-773 -> 29\n",
      "gene: Chr7-6Mb-seg-775 -> 36\n",
      "gene: Chr7-6Mb-seg-776 -> 67\n",
      "gene: Chr7-6Mb-seg-777 -> 39\n",
      "gene: Chr7-6Mb-seg-778 -> 41\n",
      "gene: Chr7-6Mb-seg-780 -> 26\n",
      "gene: Chr7-6Mb-seg-781 -> 38\n",
      "gene: Chr7-6Mb-seg-784 -> 48\n",
      "gene: Chr7-6Mb-seg-785 -> 54\n",
      "gene: Chr7-6Mb-seg-786 -> 40\n",
      "gene: Chr7-6Mb-seg-787 -> 48\n",
      "gene: Chr7-6Mb-seg-788 -> 49\n",
      "gene: Chr7-6Mb-seg-790 -> 39\n",
      "gene: Chr7-6Mb-seg-791 -> 25\n",
      "gene: Chr7-6Mb-seg-792 -> 43\n",
      "gene: Chr7-6Mb-seg-793 -> 29\n",
      "gene: Chr7-6Mb-seg-794 -> 37\n",
      "gene: Chr7-6Mb-seg-795 -> 24\n",
      "gene: Chr7-6Mb-seg-796 -> 24\n",
      "gene: Chr7-6Mb-seg-797 -> 44\n",
      "gene: Chr7-6Mb-seg-798 -> 52\n",
      "gene: Chr7-6Mb-seg-799 -> 45\n",
      "gene: Chr7-6Mb-seg-800 -> 27\n",
      "gene: Chr7-6Mb-seg-801 -> 23\n",
      "gene: Chr7-6Mb-seg-802 -> 51\n",
      "gene: Chr7-6Mb-seg-804 -> 43\n",
      "gene: Chr7-6Mb-seg-805 -> 48\n",
      "gene: Chr7-6Mb-seg-806 -> 51\n",
      "gene: Chr7-6Mb-seg-807 -> 53\n",
      "gene: Chr7-6Mb-seg-808 -> 64\n",
      "gene: Chr7-6Mb-seg-809 -> 66\n",
      "gene: Chr7-6Mb-seg-810 -> 27\n",
      "gene: Chr7-6Mb-seg-812 -> 22\n",
      "gene: Chr7-6Mb-seg-813 -> 52\n",
      "gene: Chr7-6Mb-seg-814 -> 23\n",
      "gene: Chr7-6Mb-seg-816 -> 63\n",
      "gene: Chr7-6Mb-seg-817 -> 67\n",
      "gene: Chr7-6Mb-seg-818 -> 46\n",
      "gene: Chr7-6Mb-seg-819 -> 68\n",
      "gene: Chr7-6Mb-seg-820 -> 62\n",
      "gene: Chr7-6Mb-seg-821 -> 30\n",
      "gene: Chr7-6Mb-seg-822 -> 49\n",
      "gene: Chr7-6Mb-seg-823 -> 37\n",
      "gene: Chr7-6Mb-seg-824 -> 68\n",
      "gene: Chr7-6Mb-seg-825 -> 30\n",
      "gene: Chr7-6Mb-seg-828 -> 27\n",
      "gene: Chr7-6Mb-seg-829 -> 39\n",
      "gene: Chr7-6Mb-seg-831 -> 25\n",
      "gene: Chr7-6Mb-seg-832 -> 29\n",
      "gene: Chr7-6Mb-seg-833 -> 32\n",
      "gene: Chr7-6Mb-seg-834 -> 56\n",
      "gene: Chr7-6Mb-seg-835 -> 64\n",
      "gene: Chr7-6Mb-seg-836 -> 68\n",
      "gene: Chr7-6Mb-seg-837 -> 67\n",
      "gene: Chr7-6Mb-seg-840 -> 66\n",
      "gene: Chr7-6Mb-seg-841 -> 70\n",
      "gene: Chr7-6Mb-seg-842 -> 53\n",
      "gene: Chr7-6Mb-seg-845 -> 52\n",
      "gene: Chr7-6Mb-seg-846 -> 65\n",
      "gene: Chr7-6Mb-seg-847 -> 22\n",
      "gene: Chr7-6Mb-seg-848 -> 31\n",
      "gene: Chr7-6Mb-seg-849 -> 59\n",
      "gene: Chr7-6Mb-seg-850 -> 56\n",
      "gene: Chr7-6Mb-seg-852 -> 27\n",
      "gene: Chr7-6Mb-seg-853 -> 43\n",
      "gene: Chr7-6Mb-seg-854 -> 33\n",
      "gene: Chr7-6Mb-seg-855 -> 38\n",
      "gene: Chr7-6Mb-seg-856 -> 51\n",
      "gene: Chr7-6Mb-seg-857 -> 42\n",
      "gene: Chr7-6Mb-seg-858 -> 51\n",
      "gene: Chr7-6Mb-seg-859 -> 28\n",
      "gene: Chr7-6Mb-seg-860 -> 30\n",
      "gene: Chr7-6Mb-seg-862 -> 44\n",
      "gene: Chr7-6Mb-seg-863 -> 65\n",
      "gene: Chr7-6Mb-seg-864 -> 38\n",
      "gene: Chr7-6Mb-seg-865 -> 33\n",
      "gene: Chr7-6Mb-seg-866 -> 35\n",
      "gene: Chr7-6Mb-seg-868 -> 43\n",
      "gene: Chr7-6Mb-seg-869 -> 37\n",
      "gene: Chr7-6Mb-seg-870 -> 62\n",
      "gene: Chr7-6Mb-seg-873 -> 39\n",
      "gene: Chr7-6Mb-seg-876 -> 51\n",
      "gene: Chr7-6Mb-seg-878 -> 44\n",
      "gene: Chr7-6Mb-seg-879 -> 47\n",
      "gene: Chr7-6Mb-seg-880 -> 40\n",
      "gene: Chr7-6Mb-seg-881 -> 34\n",
      "gene: Chr7-6Mb-seg-883 -> 26\n",
      "gene: Chr7-6Mb-seg-885 -> 33\n",
      "gene: Chr7-6Mb-seg-888 -> 27\n",
      "gene: Chr7-6Mb-seg-889 -> 58\n",
      "gene: Chr7-6Mb-seg-890 -> 55\n",
      "gene: Chr7-6Mb-seg-891 -> 40\n",
      "gene: Chr7-6Mb-seg-892 -> 40\n",
      "gene: Chr7-6Mb-seg-893 -> 55\n",
      "gene: Chr7-6Mb-seg-894 -> 43\n",
      "gene: Chr7-6Mb-seg-895 -> 46\n",
      "gene: Chr7-6Mb-seg-896 -> 60\n",
      "gene: Chr7-6Mb-seg-897 -> 61\n",
      "gene: Chr7-6Mb-seg-900 -> 58\n",
      "gene: Chr7-6Mb-seg-901 -> 68\n",
      "gene: Chr7-6Mb-seg-902 -> 48\n",
      "gene: Chr7-6Mb-seg-904 -> 36\n",
      "gene: Chr7-6Mb-seg-906 -> 23\n",
      "gene: Chr7-6Mb-seg-907 -> 42\n",
      "gene: Chr7-6Mb-seg-911 -> 50\n",
      "gene: Chr7-6Mb-seg-912 -> 62\n",
      "gene: Chr7-6Mb-seg-913 -> 22\n",
      "gene: Chr7-6Mb-seg-916 -> 32\n",
      "gene: Chr7-6Mb-seg-917 -> 48\n",
      "gene: Chr7-6Mb-seg-918 -> 36\n",
      "gene: Chr7-6Mb-seg-919 -> 34\n",
      "gene: Chr7-6Mb-seg-922 -> 49\n",
      "gene: Chr7-6Mb-seg-923 -> 41\n",
      "gene: Chr7-6Mb-seg-924 -> 33\n",
      "gene: Chr7-6Mb-seg-926 -> 38\n",
      "gene: Chr7-6Mb-seg-929 -> 40\n",
      "gene: Chr7-6Mb-seg-931 -> 21\n",
      "gene: Chr7-6Mb-seg-932 -> 48\n",
      "gene: Chr7-6Mb-seg-933 -> 36\n",
      "gene: Chr7-6Mb-seg-934 -> 60\n",
      "gene: Chr7-6Mb-seg-935 -> 28\n",
      "gene: Chr7-6Mb-seg-937 -> 25\n",
      "gene: Chr7-6Mb-seg-938 -> 55\n",
      "gene: Chr7-6Mb-seg-939 -> 34\n",
      "gene: Chr7-6Mb-seg-942 -> 62\n",
      "gene: Chr7-6Mb-seg-944 -> 42\n",
      "gene: Chr7-6Mb-seg-945 -> 67\n",
      "gene: Chr7-6Mb-seg-946 -> 44\n",
      "gene: Chr7-6Mb-seg-947 -> 21\n",
      "gene: Chr7-6Mb-seg-950 -> 31\n",
      "gene: Chr7-6Mb-seg-951 -> 61\n",
      "gene: Chr7-6Mb-seg-952 -> 80\n",
      "gene: Chr7-6Mb-seg-953 -> 80\n",
      "gene: Chr7-6Mb-seg-954 -> 71\n",
      "gene: Chr7-6Mb-seg-955 -> 70\n",
      "gene: Chr7-6Mb-seg-956 -> 75\n",
      "gene: Chr7-6Mb-seg-957 -> 79\n",
      "gene: Chr7-6Mb-seg-958 -> 80\n",
      "gene: Chr7-6Mb-seg-959 -> 63\n",
      "gene: Chr7-6Mb-seg-960 -> 69\n",
      "gene: Chr7-6Mb-seg-961 -> 63\n",
      "gene: Chr7-6Mb-seg-962 -> 69\n",
      "gene: Chr7-6Mb-seg-963 -> 56\n",
      "gene: Chr7-6Mb-seg-964 -> 74\n",
      "gene: Chr7-6Mb-seg-965 -> 73\n",
      "gene: Chr7-6Mb-seg-966 -> 73\n",
      "gene: Chr7-6Mb-seg-967 -> 80\n",
      "gene: Chr7-6Mb-seg-968 -> 80\n",
      "gene: Chr7-6Mb-seg-969 -> 77\n",
      "gene: Chr7-6Mb-seg-970 -> 75\n",
      "gene: Chr7-6Mb-seg-971 -> 68\n",
      "gene: Chr7-6Mb-seg-972 -> 68\n",
      "gene: Chr7-6Mb-seg-973 -> 77\n",
      "gene: Chr7-6Mb-seg-974 -> 78\n",
      "gene: Chr7-6Mb-seg-975 -> 80\n",
      "gene: Chr7-6Mb-seg-976 -> 80\n",
      "gene: Chr7-6Mb-seg-977 -> 80\n",
      "gene: Chr7-6Mb-seg-978 -> 79\n",
      "gene: Chr7-6Mb-seg-979 -> 78\n",
      "gene: Chr7-6Mb-seg-980 -> 80\n",
      "gene: Chr7-6Mb-seg-981 -> 72\n",
      "gene: Chr7-6Mb-seg-982 -> 80\n",
      "gene: Chr7-6Mb-seg-983 -> 73\n",
      "gene: Chr7-6Mb-seg-984 -> 80\n",
      "gene: Chr7-6Mb-seg-985 -> 69\n",
      "gene: Chr7-6Mb-seg-986 -> 77\n",
      "gene: Chr7-6Mb-seg-987 -> 76\n",
      "gene: Chr7-6Mb-seg-988 -> 73\n",
      "gene: Chr7-6Mb-seg-989 -> 73\n",
      "gene: Chr7-6Mb-seg-990 -> 66\n",
      "gene: Chr7-6Mb-seg-991 -> 80\n",
      "gene: Chr7-6Mb-seg-992 -> 78\n",
      "gene: Chr7-6Mb-seg-993 -> 80\n",
      "gene: Chr7-6Mb-seg-994 -> 80\n",
      "gene: Chr7-6Mb-seg-995 -> 80\n",
      "gene: Chr7-6Mb-seg-996 -> 80\n",
      "gene: Chr7-6Mb-seg-997 -> 80\n",
      "gene: Chr7-6Mb-seg-998 -> 68\n",
      "gene: Chr7-6Mb-seg-999 -> 76\n",
      "gene: Chr7-6Mb-seg-1000 -> 77\n",
      "gene: Chr7-6Mb-seg-1001 -> 76\n",
      "gene: Chr7-6Mb-seg-1002 -> 66\n",
      "gene: Chr7-6Mb-seg-1003 -> 80\n",
      "gene: Chr7-6Mb-seg-1004 -> 80\n",
      "gene: Chr7-6Mb-seg-1005 -> 79\n",
      "gene: Chr7-6Mb-seg-1006 -> 80\n",
      "gene: Chr7-6Mb-seg-1007 -> 80\n",
      "gene: Chr7-6Mb-seg-1008 -> 76\n",
      "gene: Chr7-6Mb-seg-1009 -> 80\n",
      "gene: Chr7-6Mb-seg-1010 -> 80\n",
      "gene: Chr7-6Mb-seg-1011 -> 80\n",
      "gene: Chr7-6Mb-seg-1012 -> 79\n",
      "gene: Chr7-6Mb-seg-1013 -> 78\n",
      "gene: Chr7-6Mb-seg-1014 -> 80\n",
      "gene: Chr7-6Mb-seg-1015 -> 80\n",
      "gene: Chr7-6Mb-seg-1016 -> 77\n",
      "gene: Chr7-6Mb-seg-1017 -> 80\n",
      "gene: Chr7-6Mb-seg-1018 -> 80\n",
      "gene: Chr7-6Mb-seg-1019 -> 80\n",
      "gene: Chr7-6Mb-seg-1020 -> 71\n",
      "gene: Chr7-6Mb-seg-1021 -> 80\n",
      "gene: Chr7-6Mb-seg-1022 -> 70\n",
      "gene: Chr7-6Mb-seg-1023 -> 67\n",
      "gene: Chr7-6Mb-seg-1024 -> 73\n",
      "gene: Chr7-6Mb-seg-1025 -> 76\n",
      "gene: Chr7-6Mb-seg-1026 -> 80\n",
      "gene: Chr7-6Mb-seg-1027 -> 80\n",
      "gene: Chr7-6Mb-seg-1028 -> 76\n",
      "gene: Chr7-6Mb-seg-1029 -> 75\n",
      "gene: Chr7-6Mb-seg-1030 -> 75\n",
      "gene: Chr7-6Mb-seg-1031 -> 80\n",
      "gene: Chr7-6Mb-seg-1032 -> 69\n",
      "gene: Chr7-6Mb-seg-1033 -> 80\n",
      "gene: Chr7-6Mb-seg-1034 -> 77\n",
      "gene: Chr7-6Mb-seg-1035 -> 78\n",
      "gene: Chr7-6Mb-seg-1036 -> 80\n",
      "gene: Chr7-6Mb-seg-1037 -> 80\n",
      "gene: Chr7-6Mb-seg-1038 -> 80\n",
      "gene: Chr7-6Mb-seg-1039 -> 80\n",
      "gene: Chr7-6Mb-seg-1040 -> 80\n",
      "gene: Chr7-6Mb-seg-1041 -> 80\n",
      "gene: Chr7-6Mb-seg-1042 -> 77\n",
      "gene: Chr7-6Mb-seg-1043 -> 80\n",
      "gene: Chr7-6Mb-seg-1044 -> 75\n",
      "gene: Chr7-6Mb-seg-1045 -> 79\n",
      "gene: Chr7-6Mb-seg-1046 -> 57\n",
      "gene: Chr7-6Mb-seg-1047 -> 75\n",
      "gene: Chr7-6Mb-seg-1048 -> 61\n",
      "gene: Chr7-6Mb-seg-1049 -> 80\n",
      "gene: Chr7-6Mb-seg-1050 -> 80\n",
      "gene: Chr7-6Mb-seg-1051 -> 80\n",
      "gene: Chr7-6Mb-seg-1052 -> 80\n",
      "gene: Chr7-6Mb-seg-1053 -> 80\n",
      "gene: Chr7-6Mb-seg-1054 -> 80\n",
      "gene: Chr7-6Mb-seg-1055 -> 80\n",
      "gene: Chr7-6Mb-seg-1056 -> 78\n",
      "gene: Chr7-6Mb-seg-1057 -> 79\n",
      "gene: Chr7-6Mb-seg-1058 -> 80\n",
      "gene: Chr7-6Mb-seg-1059 -> 79\n",
      "gene: Chr7-6Mb-seg-1060 -> 80\n",
      "gene: Chr7-6Mb-seg-1061 -> 80\n",
      "gene: Chr7-6Mb-seg-1062 -> 73\n",
      "gene: Chr7-6Mb-seg-1063 -> 74\n",
      "gene: Chr7-6Mb-seg-1064 -> 71\n",
      "gene: Chr7-6Mb-seg-1065 -> 80\n",
      "gene: Chr7-6Mb-seg-1066 -> 80\n",
      "gene: Chr7-6Mb-seg-1067 -> 80\n",
      "gene: Chr7-6Mb-seg-1068 -> 67\n",
      "gene: Chr7-6Mb-seg-1069 -> 78\n",
      "gene: Chr7-6Mb-seg-1070 -> 74\n",
      "gene: Chr7-6Mb-seg-1071 -> 69\n",
      "gene: Chr7-6Mb-seg-1072 -> 61\n",
      "gene: Chr7-6Mb-seg-1073 -> 79\n",
      "gene: Chr7-6Mb-seg-1074 -> 79\n",
      "gene: Chr7-6Mb-seg-1075 -> 80\n",
      "gene: Chr7-6Mb-seg-1076 -> 76\n",
      "gene: Chr7-6Mb-seg-1077 -> 80\n",
      "gene: Chr7-6Mb-seg-1078 -> 59\n",
      "gene: Chr7-6Mb-seg-1079 -> 75\n",
      "gene: Chr7-6Mb-seg-1080 -> 70\n",
      "gene: Chr7-6Mb-seg-1081 -> 72\n",
      "gene: Chr7-6Mb-seg-1082 -> 63\n",
      "gene: Chr7-6Mb-seg-1083 -> 80\n",
      "gene: Chr7-6Mb-seg-1084 -> 76\n",
      "gene: Chr7-6Mb-seg-1085 -> 76\n",
      "gene: Chr7-6Mb-seg-1086 -> 77\n",
      "gene: Chr7-6Mb-seg-1087 -> 62\n",
      "gene: Chr7-6Mb-seg-1088 -> 78\n",
      "gene: Chr7-6Mb-seg-1089 -> 77\n",
      "gene: Chr7-6Mb-seg-1090 -> 67\n",
      "gene: Chr7-6Mb-seg-1091 -> 79\n",
      "gene: Chr7-6Mb-seg-1092 -> 60\n",
      "gene: Chr7-6Mb-seg-1093 -> 66\n",
      "gene: Chr7-6Mb-seg-1094 -> 70\n",
      "gene: Chr7-6Mb-seg-1095 -> 80\n",
      "gene: Chr7-6Mb-seg-1096 -> 80\n",
      "gene: Chr7-6Mb-seg-1097 -> 68\n",
      "gene: Chr7-6Mb-seg-1098 -> 63\n",
      "gene: Chr7-6Mb-seg-1099 -> 74\n",
      "gene: Chr7-6Mb-seg-1100 -> 72\n",
      "gene: Chr7-6Mb-seg-1101 -> 79\n",
      "gene: Chr7-6Mb-seg-1102 -> 76\n",
      "gene: Chr7-6Mb-seg-1103 -> 76\n",
      "gene: Chr7-6Mb-seg-1104 -> 80\n",
      "gene: Chr7-6Mb-seg-1105 -> 73\n",
      "gene: Chr7-6Mb-seg-1106 -> 75\n",
      "gene: Chr7-6Mb-seg-1107 -> 63\n",
      "gene: Chr7-6Mb-seg-1108 -> 73\n",
      "gene: Chr7-6Mb-seg-1109 -> 78\n",
      "gene: Chr7-6Mb-seg-1110 -> 71\n",
      "gene: Chr7-6Mb-seg-1111 -> 80\n",
      "gene: Chr7-6Mb-seg-1112 -> 80\n",
      "gene: Chr7-6Mb-seg-1113 -> 75\n",
      "gene: Chr7-6Mb-seg-1114 -> 79\n",
      "gene: Chr7-6Mb-seg-1115 -> 78\n",
      "gene: Chr7-6Mb-seg-1116 -> 80\n",
      "gene: Chr7-6Mb-seg-1117 -> 80\n",
      "gene: Chr7-6Mb-seg-1118 -> 75\n",
      "gene: Chr7-6Mb-seg-1119 -> 67\n",
      "gene: Chr7-6Mb-seg-1120 -> 80\n",
      "gene: Chr7-6Mb-seg-1121 -> 80\n",
      "gene: Chr7-6Mb-seg-1122 -> 78\n",
      "gene: Chr7-6Mb-seg-1123 -> 62\n",
      "gene: Chr7-6Mb-seg-1124 -> 76\n",
      "gene: Chr7-6Mb-seg-1125 -> 59\n",
      "gene: Chr7-6Mb-seg-1126 -> 56\n",
      "gene: Chr7-6Mb-seg-1127 -> 75\n",
      "gene: Chr7-6Mb-seg-1128 -> 78\n",
      "gene: Chr7-6Mb-seg-1129 -> 75\n",
      "gene: Chr7-6Mb-seg-1130 -> 65\n",
      "gene: Chr7-6Mb-seg-1131 -> 52\n",
      "gene: Chr7-6Mb-seg-1132 -> 78\n",
      "gene: Chr7-6Mb-seg-1133 -> 67\n",
      "gene: Chr7-6Mb-seg-1134 -> 71\n",
      "gene: Chr7-6Mb-seg-1135 -> 54\n",
      "gene: Chr7-6Mb-seg-1136 -> 72\n",
      "gene: Chr7-6Mb-seg-1137 -> 76\n",
      "gene: Chr7-6Mb-seg-1138 -> 74\n",
      "gene: Chr7-6Mb-seg-1139 -> 64\n",
      "gene: Chr7-6Mb-seg-1140 -> 71\n",
      "gene: Chr7-6Mb-seg-1141 -> 80\n",
      "gene: Chr7-6Mb-seg-1142 -> 67\n",
      "gene: Chr7-6Mb-seg-1143 -> 58\n",
      "gene: Chr7-6Mb-seg-1144 -> 48\n",
      "gene: Chr7-6Mb-seg-1145 -> 68\n",
      "gene: Chr7-6Mb-seg-1146 -> 49\n",
      "gene: Chr7-6Mb-seg-1147 -> 61\n",
      "gene: Chr7-6Mb-seg-1148 -> 42\n",
      "gene: Chr7-6Mb-seg-1149 -> 71\n",
      "gene: Chr7-6Mb-seg-1150 -> 59\n",
      "gene: Chr7-6Mb-seg-1151 -> 65\n",
      "gene: Chr7-6Mb-seg-1152 -> 77\n",
      "gene: Chr7-6Mb-seg-1153 -> 76\n",
      "gene: Chr7-6Mb-seg-1154 -> 77\n",
      "gene: Chr7-6Mb-seg-1155 -> 80\n",
      "gene: Chr7-6Mb-seg-1156 -> 77\n",
      "gene: Chr7-6Mb-seg-1157 -> 69\n",
      "gene: Chr7-6Mb-seg-1158 -> 68\n",
      "gene: Chr7-6Mb-seg-1159 -> 64\n",
      "gene: Chr7-6Mb-seg-1160 -> 69\n",
      "gene: Chr7-6Mb-seg-1161 -> 72\n",
      "gene: Chr7-6Mb-seg-1162 -> 75\n",
      "gene: Chr7-6Mb-seg-1163 -> 70\n",
      "gene: Chr7-6Mb-seg-1164 -> 65\n",
      "gene: Chr7-6Mb-seg-1165 -> 80\n",
      "gene: Chr7-6Mb-seg-1166 -> 80\n",
      "gene: Chr7-6Mb-seg-1167 -> 67\n",
      "gene: Chr7-6Mb-seg-1168 -> 74\n",
      "gene: Chr7-6Mb-seg-1169 -> 70\n",
      "gene: Chr7-6Mb-seg-1170 -> 66\n",
      "gene: Chr7-6Mb-seg-1171 -> 67\n",
      "gene: Chr7-6Mb-seg-1172 -> 72\n",
      "gene: Chr7-6Mb-seg-1173 -> 73\n",
      "gene: Chr7-6Mb-seg-1174 -> 71\n",
      "gene: Chr7-6Mb-seg-1175 -> 72\n",
      "gene: Chr7-6Mb-seg-1176 -> 68\n",
      "gene: Chr7-6Mb-seg-1177 -> 55\n",
      "gene: Chr7-6Mb-seg-1178 -> 76\n",
      "gene: Chr7-6Mb-seg-1179 -> 64\n",
      "gene: Chr7-6Mb-seg-1180 -> 68\n",
      "gene: Chr7-6Mb-seg-1181 -> 77\n",
      "gene: Chr7-6Mb-seg-1182 -> 77\n",
      "gene: Chr7-6Mb-seg-1183 -> 80\n",
      "gene: Chr7-6Mb-seg-1184 -> 79\n",
      "gene: Chr7-6Mb-seg-1185 -> 80\n",
      "gene: Chr7-6Mb-seg-1186 -> 68\n",
      "gene: Chr7-6Mb-seg-1187 -> 65\n",
      "gene: Chr7-6Mb-seg-1188 -> 80\n",
      "gene: Chr7-6Mb-seg-1189 -> 77\n",
      "gene: Chr7-6Mb-seg-1190 -> 75\n",
      "gene: Chr7-6Mb-seg-1191 -> 65\n",
      "gene: Chr7-6Mb-seg-1192 -> 77\n",
      "gene: Chr7-6Mb-seg-1193 -> 75\n",
      "gene: Chr7-6Mb-seg-1194 -> 75\n",
      "gene: Chr7-6Mb-seg-1195 -> 73\n",
      "gene: Chr7-6Mb-seg-1196 -> 80\n",
      "gene: Chr7-6Mb-seg-1197 -> 80\n",
      "gene: Chr7-6Mb-seg-1198 -> 74\n",
      "gene: Chr7-6Mb-seg-1199 -> 51\n",
      "gene: Chr7-6Mb-seg-1200 -> 63\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   species_marker = 'gene_',\n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 224,
   "id": "95f60f31",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7t3Q3QlWWdP/AfbhTmDralaCul1vaKUGICJiLaiNC/tq2dnN0wl3VYMwxWaVsb7M9LvCZSqxlG0Ow29kKjErEGtbJDlCYgsUkGlb1orAGywyo1ChPEf66b/3MGBPQ51/U8z3mecz5nxmnC+3fOuT/X94Evt9e5T6+DBw8eDA8CBAgQIECAAAECLSLQSwFukZV2mgQIECBAgAABApWAAiwIBAgQIECAAAECLSWgALfUcjtZAgQIECBAgAABBVgGCBAgQIAAAQIEWkpAAW6p5XayBAgQIECAAAECCrAMECBAgAABAgQItJSAAtxSy+1kCRAgQIAAAQIEFGAZIECAAAECBAgQaCkBBbilltvJEiBAgAABAgQIKMAyQIAAAQIECBAg0FICCnBLLbeTJUCAAAECBAgQUIBlgAABAgQIECBAoKUEFOCWWm4nS4AAAQIECBAgoADLAAECBAgQIECAQEsJKMAttdxOlgABAgQIECBAQAGWAQIECBAgQIAAgZYSUIBbarmdLAECBAgQIECAgAIsAwQIECBAgAABAi0loAC31HI7WQIECBAgQIAAAQVYBggQIECAAAECBFpKQAFuqeV2sgQIECBAgAABAgqwDBAgQIAAAQIECLSUgALcUsvtZAkQIECAAAECBBRgGSBAgAABAgQIEGgpAQW4pZbbyRIgQIAAAQIECCjAMkCAAAECBAgQINBSAgpwSy23kyVAgAABAgQIEFCAZYAAAQIECBAgQKClBBTgllpuJ0uAAAECBAgQIKAAywABAgQIECBAgEBLCSjALbXcTpYAAQIECBAgQEABlgECBAgQIECAAIGWElCAW2q5nSwBAgQIECBAgIACLAMECBAgQIAAAQItJaAAt9RyO1kCBAgQIECAAAEFWAYIECBAgAABAgRaSkABbqnldrIECBAgQIAAAQIKsAwQIECAAAECBAi0lIAC3FLL7WQJECBAgAABAgQUYBkgQIAAAQIECBBoKQEFuKWW28kSIECAAAECBAgowDJAgAABAgQIECDQUgIKcEstt5MlQIAAAQIECBBQgGWAAAECBAgQIECgpQQU4JZabidLgAABAgQIECCgAMsAAQIECBAgQIBASwkowC213E6WAAECBAgQIEBAAZYBAgQIECBAgACBlhJQgFtquZ0sAQIECBAgQICAAiwDBAgQIECAAAECLSWgALfUcjtZAgQIECBAgAABBVgGCBAgQIAAAQIEWkpAAW6p5XayBAgQIECAAAECCrAMECBAgAABAgQItJSAAtxSy+1kCRAgQIAAAQIEFGAZIECAAAECBAgQaCkBBbilltvJEiBAgAABAgQIKMAyQIAAAQIECBAg0FICCnBLLbeTJUCAAAECBAgQUIBlgAABAgQIECBAoKUEFOCWWm4nS4AAAQIECBAgoAB3Ygb27t0bP/7xj+PUU0+NF73oRZ34Sp6aAAECBAgQIHCkwP79+2PXrl0xcODA6NOnD57DBBTgTozDQw89FEOGDOnEV/DUBAgQIECAAIHnF9iwYUOcf/75mBTgrsnAY489FmeffXak4L3yla/smhf1KgQIECBAgACBiNi+fXt1Ie7Xv/51nHXWWUwU4K7JwH//93/Hq171qti2bVv079+/a17UqxAgQIAAAQIEIkIPOX4MbIHoxB8RwetEXE9NgAABAgQIPK+AHqIAN+RHRPAawu5FCRAgQIAAAVeAnzcDrgB34o+IAtyJuJ6aAAECBAgQcAU4MwMKcCZce8YU4PYoOYYAAQIECBDoDAE95PiqCnBnJO7/P6fgdSKupyZAgAABAgRcAc7MgAKcCdeeMQW4PUqOIUCAAAECBDpDQA9xBbgzcvWCzyl4L0jkAAIECBAgQKCTBPQQBbiTovX8Tyt4DWH3ogQIECBAgIC7QDxvBmyB6MQfEQW4E3E9NQECBAgQIPC8AnqIK8AN+RERvIawe1ECBAgQIEDAFeDmvwI8cuTIWLt27TFP9Gtf+1r8zd/8TfXvVq5cGTfddFNs3bq1+mriyZMnx4QJE46au+WWW+L222+PHTt2xMCBA2P+/PmRXqPehwJcr5jjCRAgQIAAgY4S0EOa/Arwli1bYs+ePUec5b/8y7/EPffcE9u3b49TTjklHnzwwRgxYkRcddVVceWVV8YDDzwQ06ZNi0WLFsX48eNrs6n8TpkyJebMmRODBw+OxYsXx/Lly2PDhg1VGa7nIXj1aDmWAAECBAgQ6EgBPaTJC/CxTu81r3lNvOlNb4pvfetb1b8eM2ZM7N69O9avX187/Jprrol77703UkBOOOGE2LdvX5x22mmRfv3mm2+ujjtw4EBVfAcNGhRLly6tK5eCVxeXgwkQIECAAIEOFNBDWqwA/+AHP4gLL7wwvvKVr8QHPvCBqtj27ds35s2bFzfccENNI22bSFsbNm7cGOedd16sWbMmLr300ti0aVOce+65teNmzJgRCxYsiKeffjp69erV7mgKXrupHEiAAAECBAh0sIAe0mIF+CMf+Uj867/+azz55JNx0kknRdoiMWDAgFi1alWMHj26prFr167o169f3HnnndW2iIULF8Z1110XzzzzTJx44om14+6666644oorYtu2bdXe4fY+BK+9Uo4jQIAAAQIEOlpAD2mhArx///4444wz4h3veEd89atfrc487fcdPnx4tQ942LBhNY10bO/evePWW2+NSZMmxezZs2PmzJmxd+/eI8RWr14dl112WTz88MPVVojjPdI+5MP3Iqf9x0OGDKm7OHf0D4DnI0CAAAECBFpPQAFuoQKcrvK+853vjH//93+Pd73rXUcU4HXr1sXQoUOPKsC33XZbTJw4sSrAs2bNimefffYIsfvuuy9GjRoVmzdvft4Pwk2fPj3SdonnPuq9ctx6P6LOmAABAgQIEOhoAQW4hQrwBz/4wWqrQ7r6mq7upke9WyBSAe7Tp09Nrb1bIFwB7ugfXc9HgACBiLM+fujDzJ39eGze/+nsl/D8BLpUQAFukQKcimu6i8PYsWPjjjvuqJ21D8F16c+bFyNAgECHCijAHcrpyVpIQAFukQL89a9/vfrSi+9973tx0UUXHXHW6TZoTz31VLUPuO1x7bXXxooVK466DVr69XTHiPRIt0FL+37TrdDcBq2FftdwqgQIdBsBBbjbLIU30sMEFOAWKcDvec974kc/+lE89thjR92urO2LMMaNG1ddIU4fjJs6depxvwhj7ty51RdhLFmyJJYtW+aLMHrYD723S4BA8wgowM2zls6kawUU4BYowP/7v/8bp59+elx//fXxqU996phnnL4KOX3L2+FfhZxue3b44+DBg9H2Vcg7d+6srvymL8W45JJL6k6t4NVNZoAAAQJHCSjAQkEgT0APaYECnBeNzp0SvM719ewECLSGgALcGuvsLDteQA9RgDs+Ve14RsFrB5JDCBAg8AICCrCIEMgT0EMU4LzkFE4JXiGgcQIECITboAkBgVwBPUQBzs1O0ZzgFfEZJkCAQCXgCrAgEMgT0EMU4LzkFE4JXiGgcQIECCjAMkAgW0APUYCzw1MyKHglemYJECBwSMAVYEkgkCeghyjAeckpnBK8QkDjBAgQUIBlgEC2gB6iAGeHp2RQ8Er0zBIgQMAVYBkgUCKghyjAJfnJnhW8bDqDBAgQqAnYAiEMBPIE9BAFOC85hVOCVwhonAABArZAyACBbAE9RAHODk/JoOCV6JklQIDAIQFXgCWBQJ6AHqIA5yWncErwCgGNEyBAQAGWAQLZAnqIApwdnpJBwSvRM0uAAAFXgGWAQImAHqIAl+Qne1bwsukMEiBAoCZgC4QwEMgT0EMU4LzkFE4JXiGgcQIECNgCIQMEsgX0EAU4Ozwlg4JXomeWAAEChwRcAZYEAnkCeogCnJecwinBKwQ0ToAAAQVYBghkC+ghCnB2eEoGBa9EzywBAgRcAZYBAiUCeogCXJKf7FnBy6YzSIAAgZqALRDCQCBPQA9RgPOSUzgleIWAxgkQIGALhAwQyBbQQxTg7PCUDApeiZ5ZAgQIHBJwBVgSCOQJ6CEKcF5yCqcErxDQOAECBBRgGSCQLaCHKMDZ4SkZFLwSPbMECBBwBVgGCJQI6CEKcEl+smcFL5vOIAECBGoCtkAIA4E8AT1EAc5LTuGU4BUCGidAgIAtEDJAIFtAD1GAs8NTMih4JXpmCRAgcEjAFWBJIJAnoIcowHnJKZwSvEJA4wQIEFCAZYBAtoAeogBnh6dkUPBK9MwSIEDAFWAZIFAioIcowCX5yZ4VvGw6gwQIEKgJ2AIhDATyBPQQBTgvOYVTglcIaJwAAQK2QMgAgWwBPUQBzg5PyaDgleiZJUCAwCEBV4AlgUCegB6iAOclp3BK8AoBjRMgQEABlgEC2QJ6iAKcHZ6SQcEr0TNLgAABV4BlgECJgB6iAJfkJ3tW8LLpDBIgQKAmYAuEMBDIE9BDFOC85BROCV4hoHECBAjYAiEDBLIF9BAFODs8JYOCV6JnlgABAocEXAGWBAJ5AnqIApyXnMIpwSsENE6AAAEFWAYIZAvoIQpwdnhKBgWvRM8sAQIEXAGWAQIlAnqIAlySn+xZwcumM0iAAIGagC0QwkAgT0APUYDzklM4JXiFgMYJECBgC4QMEMgW0EMU4OzwlAwKXomeWQIECBwScAVYEgjkCeghCnBecgqnBK8Q0DgBAgQUYBkgkC2ghyjA2eEpGRS8Ej2zBAgQcAVYBgiUCOghLVKAv/jFL8Ztt90WP/vZz6Jv374xbNiwWLFiRe3sV65cGTfddFNs3bo1+vfvH5MnT44JEyYcpXPLLbfE7bffHjt27IiBAwfG/PnzY+TIkXVnUPDqJjNAgACBowRsgRAKAnkCekgLFODp06fHZz7zmargDh06NHbv3h3f/va3Y9GiRdXZP/jggzFixIi46qqr4sorr4wHHnggpk2bVv378ePH14RS+Z0yZUrMmTMnBg8eHIsXL47ly5fHhg0bqjJcz0Pw6tFyLAECBI4toABLBoE8AT2kyQtwuqKbymm6wjtq1Khjnu2YMWOqUrx+/frav7/mmmvi3nvvjRSQE044Ifbt2xennXZapF+/+eabq+MOHDhQPfegQYNi6dKldSVQ8OricjABAgSOKaAACwaBPAE9pMkL8I033hjf+MY34uc///kxzzQV27QlYt68eXHDDTfUjlm7dm21tWHjxo1x3nnnxZo1a+LSSy+NTZs2xbnnnls7bsaMGbFgwYJ4+umno1evXu1OoeC1m8qBBAgQOK6AAiwcBPIE9JAmL8AXX3xxnHLKKfHWt741PvvZz8ZTTz0VF1xwQdx6663Vr23ZsiUGDBgQq1atitGjR9c0du3aFf369Ys777yz2haxcOHCuO666+KZZ56JE088sXbcXXfdFVdccUVs27at2jvc3ofgtVfKcQQIEDi+gAIsHQTyBPSQJi/Ab3jDG+K3v/1tnHHGGdXe3Re/+MWRrto+9thj8eijj8ZPfvKTGD58eLUPOH0wru2xf//+6N27d1WUJ02aFLNnz46ZM2fG3r17jxBbvXp1XHbZZfHwww9XWyGO99izZ0+kf9oe27dvjyFDhtRdnPNibooAAQLNKaAAN+e6OqvOF1CAm7wAv+51r4tf/OIX8cgjj1RXetMjlc+zzz47PvnJT8aFF15YFeB169ZVH5B7bgFOd46YOHFiVYBnzZoVzz777BFi9913X7W3ePPmzc/7Qbj0QbxUvJ/7qPfKcef/SHgFAgQI9BwBBbjnrJV32r0EFOAmL8Cp1D7++OPVbcsOf6TtD295y1si7RGuZwtEKsB9+vSpPVV7t0C4Aty9fvC9GwIEmkNAAW6OdXQWXS+gADd5AR43blx1y7PnFuBUftOtzD7/+c/7EFzX/9x5RQIECHSIgALcIYyepAUFFOAmL8B33313vP/9748f//jHcc4551Rn+8QTT8RrXvOa+NSnPhXXX399pNugpQ/HpX3AbY9rr722+qKM594GLf16umNEeqTboKV9v+lWaG6D1oK/ezhlAgQaLqAAN3wJvIEeKqAAN3kBTiU1fdjsd7/7XbWHN30ILu39ffLJJ6tvhTvppJNqX4SRrhaPHTu2+iKMqVOnHveLMObOnVtdPV6yZEksW7bMF2H00B9+b5sAgZ4voAD3/DV0Bo0RUICbvACn00tlN93j91vf+lb84Q9/iHRrtPTNcOkOEW2P9EUZ6VveDv8q5HTbs8MfBw8ejLavQt65c2d15Td9KcYll1xSd3oFr24yAwQIEDhKQAEWCgJ5AnpICxTgvGh07pTgda6vZydAoDUEFODWWGdn2fECeogC3PGpasczCl47kBxCgACBFxBQgEWEQJ6AHqIA5yWncErwCgGNEyBAICIUYDEgkCeghyjAeckpnBK8QkDjBAgQUIBlgEC2gB6iAGeHp2RQ8Er0zBIgQOCQgCvAkkAgT0APUYDzklM4JXiFgMYJECCgAMsAgWwBPUQBzg5PyaDgleiZJUCAgCvAMkCgREAPUYBL8pM9K3jZdAYJECBQE7AFQhgI5AnoIQpwXnIKpwSvENA4AQIEbIGQAQLZAnqIApwdnpJBwSvRM0uAAIFDAq4ASwKBPAE9RAHOS07hlOAVAhonQICAAiwDBLIF9BAFODs8JYOCV6JnlgABAq4AywCBEgE9RAEuyU/2rOBl0xkkQIBATcAWCGEgkCeghyjAeckpnBK8QkDjBAgQsAVCBghkC+ghCnB2eEoGBa9EzywBAgQOCbgCLAkE8gT0EAU4LzmFU4JXCGicAAECCrAMEMgW0EMU4OzwlAwKXomeWQIECLgCLAMESgT0EAW4JD/Zs4KXTWeQAAECNQFbIISBQJ6AHqIA5yWncErwCgGNEyBAwBYIGSCQLaCHKMDZ4SkZFLwSPbMECBA4JOAKsCQQyBPQQxTgvOQUTgleIaBxAgQIKMAyQCBbQA9RgLPDUzIoeCV6ZgkQIOAKsAwQKBHQQxTgkvxkzwpeNp1BAgQI1ARsgRAGAnkCeogCnJecwinBKwQ0ToAAAVsgZIBAtoAeogBnh6dkUPBK9MwSIEDgkIArwJJAIE9AD1GA85JTOCV4hYDGCRAgoADLAIFsAT1EAc4OT8mg4JXomSVAgIArwDJAoERAD1GAS/KTPSt42XQGCRAgUBOwBUIYCOQJ6CEKcF5yCqcErxDQOAECBGyBkAEC2QJ6iAKcHZ6SQcEr0TNLgACBQwKuAEsCgTwBPUQBzktO4ZTgFQIaJ0CAgAIsAwSyBfQQBTg7PCWDgleiZ5YAAQKuAMsAgRIBPUQBLslP9qzgZdMZJECAQE3AFghhIJAnoIcowHnJKZwSvEJA4wQIELAFQgYIZAvoIQpwdnhKBgWvRM8sAQIEDgm4AiwJBPIE9BAFOC85hVOCVwhonAABAgqwDBDIFtBDFODs8JQMCl6JnlkCBAi4AiwDBEoE9BAFuCQ/2bOCl01nkAABAjUBWyCEgUCegB6iAOclp3BK8AoBjRMgQMAWCBkgkC2ghyjA2eEpGRS8Ej2zBAgQOCTgCrAkEMgT0EMU4LzkFE4JXiGgcQIECCjAMkAgW0APUYCzw1MyKHglemYJECDgCrAMECgR0EMU4JL8ZM8KXjadQQIECNQEbIEQBgJ5AnqIApyXnMIpwSsENE6AAAFbIGSAQLaAHtLkBfjf/u3f4u///u+POssbb7wx5s2bV/v1lStXxk033RRbt26N/v37x+TJk2PChAlHzd1yyy1x++23x44dO2LgwIExf/78GDlyZN0BFLy6yQwQIEDgKAFXgIWCQJ6AHtIiBfjb3/52nHzyybWzPeOMM+JVr3pV9f8ffPDBGDFiRFx11VVx5ZVXxgMPPBDTpk2LRYsWxfjx42szqfxOmTIl5syZE4MHD47FixfH8uXLY8OGDVUZruchePVoOZYAAQLHFlCAJYNAnoAe0iIFeNeuXXHKKacc82zHjBkTu3fvjvXr19f+/TXXXBP33ntvpICccMIJsW/fvjjttNMi/frNN99cHXfgwIGq+A4aNCiWLl1aVwIFry4uBxMgQOCYAgqwYBDIE9BDWrwAp2Lbt2/fajvEDTfcUNNYu3ZttbVh48aNcd5558WaNWvi0ksvjU2bNsW5555bO27GjBmxYMGCePrpp6NXr17tTqHgtZvKgQQIEDiugAIsHATyBPSQFinA/fr1i//5n/+JM888M/7hH/4h/vmf/zn+5E/+JLZs2RIDBgyIVatWxejRo2sa6YpxmrnzzjurbRELFy6M6667Lp555pk48cQTa8fdddddccUVV8S2bduqvcPtfQhee6UcR4AAgeMLKMDSQSBPQA9p8gL8ne98p9raMHTo0OoK7YoVK+KOO+6ID3/4w9WH2dJ+3+HDh1f7gIcNG1bT2L9/f/Tu3TtuvfXWmDRpUsyePTtmzpwZe/fuPUJs9erVcdlll8XDDz9cbYU43mPPnj2R/ml7bN++PYYMGVJ3cc6LuSkCBAg0p4AC3Jzr6qw6X0ABbvICfKzT+9jHPhaf+cxnqvL5q1/9qirA69atq0py26OtAN92220xceLEqgDPmjUrnn322SOe8r777otRo0bF5s2bn/eDcNOnT4+0XeK5j3qvHHf+j4RXIECAQM8RUIB7zlp5p91LQAFuwQL80EMPVVdf063P0paIerZApALcp0+fmlp7t0C4Aty9fvC9GwIEmkNAAW6OdXQWXS+gALdgAU63LUtXe1MBTh9s8yG4rv/B84oECBDoCAEFuCMUPUcrCijALViAP/rRj1Z7e9Pin3766ZFug/bUU09V+4DbHtdee221X/i5t0FLv972BRrpNmhp32+6FZrboLXibx/OmQCBRgsowI1eAa/fUwUU4CYvwJdffnm84x3viHPOOac601Rqv/CFL8Q//uM/VvuA06PtizDGjRsXY8eOrT4YN3Xq1ON+EcbcuXOrL8JYsmRJLFu2zBdh9NSffu+bAIEeL6AA9/gldAINElCAm7wAp6KbbnGWFvqPf/xjvP71r6++3S19sO3w+/am7RDpW94O/yrkdNuzwx8HDx6Mtq9C3rlzZ3XlN30pxiWXXFJ3fAWvbjIDBAgQOEpAARYKAnkCekiTF+C8WHT+lOB1vrFXIECg+QUU4OZfY2fYOQJ6iALcOcl6gWcVvIawe1ECBJpMQAFusgV1Ol0moIcowF0WtsNfSPAawu5FCRBoMgEFuMkW1Ol0mYAeogB3WdgU4IZQe1ECBJpYQAFu4sV1ap0qoAArwJ0asOM9ueA1hN2LEiDQZAIKcJMtqNPpMgE9RAHusrC5AtwQai9KgEATCyjATby4Tq1TBRRgBbhTA+YKcEN4vSgBAi0ioAC3yEI7zQ4XUIAV4A4PVXueUPDao+QYAgQIPL+AAiwhBPIE9BAFOC85hVOCVwhonAABAhGhAIsBgTwBPUQBzktO4ZTgFQIaJ0CAgAIsAwSyBfQQBTg7PCWDgleiZ5YAAQKHBFwBlgQCeQJ6iAKcl5zCKcErBDROgAABBVgGCGQL6CEKcHZ4SgYFr0TPLAECBFwBlgECJQJ6iAJckp/sWcHLpjNIgACBmoAtEMJAIE9AD1GA85JTOCV4hYDGCRAgYAuEDBDIFtBDFODs8JQMCl6JnlkCBAgcEnAFWBII5AnoIQpwXnIKpwSvENA4AQIEFGAZIJAtoIcowNnhKRkUvBI9swQIEHAFWAYIlAjoIQpwSX6yZwUvm84gAQIEagK2QAgDgTwBPUQBzktO4ZTgFQIaJ0CAgC0QMkAgW0APUYCzw1MyKHglemYJECBwSMAVYEkgkCeghyjAeckpnBK8QkDjBAgQUIBlgEC2gB6iAGeHp2RQ8Er0zBIgQMAVYBkgUCKghyjAJfnJnhW8bDqDBAgQqAnYAiEMBPIE9BAFOC85hVOCVwhonAABArZAyACBbAE9RAHODk/JoOCV6JklQIDAIQFXgCWBQJ6AHqIA5yWncErwCgGNEyBAQAGWAQLZAnqIApwdnpJBwSvRM0uAAAFXgGWAQImAHqIAl+Qne1bwsukMEiBAoCZgC4QwEMgT0EMU4LzkFE4JXiGgcQIECNgCIQMEsgX0EAU4Ozwlg4JXomeWAAEChwRcAZYEAnkCeogCnJecwinBKwQ0ToAAAQVYBghkC+ghCnB2eEoGBa9EzywBAgRcAZYBAiUCeogCXJKf7FnBy6YzSIAAgZqALRDCQCBPQA9RgPOSUzgleIWAxgkQIGALhAwQyBbQQxTg7PCUDApeiZ5ZAgQIHBJwBVgSCOQJ6CEKcF5yCqcErxDQOAECBBRgGSCQLaCHKMDZ4SkZFLwSPbMECBBwBVgGCJQI6CEKcEl+smcFL5vOIAECBGoCtkAIA4E8AT1EAc5LTuGU4BUCGidAgIAtEDJAIFtAD1GAs8NTMih4JXpmCRAgcEjAFWBJIJAnoIcowHnJKZwSvEJA4wQIEFCAZYBAtoAeogBnh6dkUPBK9MwSIEDAFWAZIFAioIe0WAH+/e9/H2984xvjiSeeiIceeije9ra31QRWrlwZN910U2zdujX69+8fkydPjgkTJhwldMstt8Ttt98eO3bsiIEDB8b8+fNj5MiRdeVQ8OricjABAgSOKWALhGAQyBPQQ1qsAN94443xpS99KXbu3HlEAX7wwQdjxIgRcdVVV8WVV14ZDzzwQEybNi0WLVoU48ePryml8jtlypSYM2dODB48OBYvXhzLly+PDRs2VGW4vQ/Ba6+U4wgQIHB8AQVYOgjkCeghLVSAf/rTn1ZXfBcsWBDXXnvtEQV4zJgxsXv37li/fn1N5Jprrol77703UkhOOOGE2LdvX5x22mmRfv3mm2+ujjtw4EBVfAcNGhRLly5tdwoFr91UDiRAgMBxBRRg4SCQJ6CHtFABHjVqVFVU3/Wud8Ull1xSK8Cp2Pbt2zfmzZtxQp5aAAAgAElEQVQXN9xwQ01k7dq11daGjRs3xnnnnRdr1qyJSy+9NDZt2hTnnntu7bgZM2ZUpfrpp5+OXr16tSuJgtcuJgcRIEDgeQUUYAEhkCegh7RIAb777rvjuuuui0cffbQqsIcX4C1btsSAAQNi1apVMXr06JrIrl27ol+/fnHnnXdW2yIWLlxYPcczzzwTJ554Yu24u+66K6644orYtm1btXe4PQ/Ba4+SYwgQIPD8AgqwhBDIE9BDWqAAp8KaPvg2ffr0uPrqq+O73/3uEQU47fcdPnx4pH3Aw4YNq4ns378/evfuHbfeemtMmjQpZs+eHTNnzoy9e/ceobZ69eq47LLL4uGHH66uMB/rsWfPnkj/tD22b98eQ4YMqas050XcFAECBJpXQAFu3rV1Zp0roAC3QAFOH1q77777qv29aS/v8QrwunXrYujQoUcV4Ntuuy0mTpxYFeBZs2bFs88+e4Raeu60vWLz5s3H/SBcKt9pq8RzH/VcNe7cHwXPToAAgZ4noAD3vDXzjruHgALc5AX48ccfj9e//vXxjW98I97+9rdXZ3v//ffHu9/97mpPb/pQ3G9+85u6tkCkAtynT5+aXHu2QLgC3D1+4L0LAgSaS0ABbq71dDZdJ6AAN3kBbrvae7zTTFd804fdfAiu637ovBIBAgQ6SkAB7ihJz9NqAgpwkxfgp556Kn70ox8dcZbp/6e7PXz+85+P888/v7qfb7oNWjo27QNue6Rbpa1YseKo26ClX093jEiPdBu0tO833QrNbdBa7bcP50uAQKMFFOBGr4DX76kCCnCTF+Bjnd5z9wCnY9q+CGPcuHExduzY6oswpk6detwvwpg7d25VnJcsWRLLli3zRRg99XcA75sAgR4toAD36OXz5hsooAArwDWB9FXI6QNzh38Vcrrt2eGPgwcPRttXIadvk0tXftOXYqTbqtXzELx6tBxLgACBYwsowJJBIE9AD2nBApwXlY6dEryO9fRsBAi0poAC3Jrr7qzLBfQQBbg8RRnPIHgZaEYIECDwHAEFWCQI5AnoIQpwXnIKpwSvENA4AQIEIkIBFgMCeQJ6iAKcl5zCKcErBDROgAABBVgGCGQL6CEKcHZ4SgYFr0TPLAECBA4JuAIsCQTyBPQQBTgvOYVTglcIaJwAAQIKsAwQyBbQQxTg7PCUDApeiZ5ZAgQIuAIsAwRKBPQQBbgkP9mzgpdNZ5AAAQI1AVsghIFAnoAeogDnJadwSvAKAY0TIEDAFggZIJAtoIcowNnhKRkUvBI9swQIEDgk4AqwJBDIE9BDFOC85BROCV4hoHECBAgowDJAIFtAD1GAs8NTMih4JXpmCRAg4AqwDBAoEdBDFOCS/GTPCl42nUECBAjUBGyBEAYCeQJ6iAKcl5zCKcErBDROgAABWyBkgEC2gB6iAGeHp2RQ8Er0zBIgQOCQgCvAkkAgT0APUYDzklM4JXiFgMYJECCgAMsAgWwBPUQBzg5PyaDgleiZJUCAgCvAMkCgREAPUYBL8pM9K3jZdAYJECBQE7AFQhgI5AnoIQpwXnIKpwSvENA4AQIEbIGQAQLZAnqIApwdnpJBwSvRM0uAAIFDAq4ASwKBPAE9RAHOS07hlOAVAhonQICAAiwDBLIF9BAFODs8JYOCV6JnlgABAq4AywCBEgE9RAEuyU/2rOBl0xkkQIBATcAWCGEgkCeghyjAeckpnBK8QkDjBAgQsAVCBghkC+ghCnB2eEoGBa9EzywBAgQOCbgCLAkE8gT0EAU4LzmFU4JXCGicAAECCrAMEMgW0EMU4OzwlAwKXomeWQIECLgCLAMESgT0EAW4JD/Zs4KXTWeQAAECNQFbIISBQJ6AHqIA5yWncErwCgGNEyBAwBYIGSCQLaCHKMDZ4SkZFLwSPbMECBA4JOAKsCQQyBPQQxTgvOQUTgleIaBxAgQIKMAyQCBbQA9RgLPDUzIoeCV6ZgkQIOAKsAwQKBHQQxTgkvxkzwpeNp1BAgQI1ARsgRAGAnkCeogCnJecwinBKwQ0ToAAAVsgZIBAtoAeogBnh6dkUPBK9MwSIEDgkIArwJJAIE9AD1GA85JTOCV4hYDGCRAgoADLAIFsAT1EAc4OT8mg4JXomSVAgIArwDJAoERAD1GAS/KTPSt42XQGCRAgUBOwBUIYCOQJ6CEKcF5yCqcErxDQOAECBGyBkAEC2QJ6iAKcHZ6SQcEr0TNLgACBQwKuAEsCgTwBPUQBzktO4ZTgFQIaJ0CAgAIsAwSyBfQQBTg7PCWDgleiZ5YAAQKuAMsAgRIBPUQBLslP9qzgZdMZJECAQE3AFghhIJAnoIcowHnJKZwSvEJA4wQIELAFQgYIZAvoIU1egL/zne/EnDlzYsuWLbFnz54444wz4q/+6q9i2rRpcfLJJ9fOfuXKlXHTTTfF1q1bo3///jF58uSYMGHCUTq33HJL3H777bFjx44YOHBgzJ8/P0aOHFl3AAWvbjIDBAgQOErAFWChIJAnoIc0eQH+2te+Fps3b44hQ4bEn/3Zn8UjjzwS06dPj8GDB8d//Md/VGf/4IMPxogRI+Kqq66KK6+8Mh544IGqIC9atCjGjx9fE0rld8qUKVWhTvOLFy+O5cuXx4YNG6oyXM9D8OrRciwBAgSOLaAASwaBPAE9pMkL8LFOLxXXa665Jp544on48z//8xgzZkzs3r071q9fXzs8/ft77703UkBOOOGE2LdvX5x22mnV3M0331wdd+DAgar4Dho0KJYuXVpXAgWvLi4HEyBA4JgCCrBgEMgT0ENasAAvW7Ys/vqv/zoee+yxOP3006Nv374xb968uOGGG2oaa9eurbY2bNy4Mc4777xYs2ZNXHrppbFp06Y499xza8fNmDEjFixYEE8//XT06tWr3SkUvHZTOZAAAQLHFVCAhYNAnoAe0iIFOF2t/cMf/lDtBb766qvj1a9+daxYsaL6/wMGDIhVq1bF6NGjaxq7du2Kfv36xZ133llti1i4cGFcd9118cwzz8SJJ55YO+6uu+6KK664IrZt21btHW7vQ/DaK+U4AgQIHF9AAZYOAnkCekiLFOBUTtOWh/RIRffuu++Ok046qdrvO3z48Gof8LBhw2oa+/fvj969e8ett94akyZNitmzZ8fMmTNj7969R4itXr06Lrvssnj44YerrRDHe6QP4KV/2h7bt2+v9iXXW5zzYm6KAAECzSmgADfnujqrzhdQgFukAKcPwv3+97+Pn/zkJ1WR/Yu/+Iu47777Yt26dVUBTv87dOjQowrwbbfdFhMnTqwK8KxZs+LZZ589Qiw9x6hRo6oP2j3fB+HSB+/SdonnPhTgzv8h9woECDSvgALcvGvrzDpXQAFukQJ8+Gn+8Ic/jLe97W2Rti+8+c1vrmsLRCrAffr0qT1de7dAuALcuT/Inp0AgdYUUIBbc92ddbmAAtyCBTjtB37JS15SXdFNH3zzIbjyHyTPQIAAgUYIKMCNUPeazSCgALdgAb7//vvjoosuiq9//evVB9jSbdCeeuqpah9w2+Paa6+tPiT33NugpV9Pd4xIj1Sk077ftPXBbdCa4bcD50CAQE8TUIB72op5v91FQAFu8gL8vve9r9rukIpquntD+rBauo9vuqfvQw89FC9+8YtrX4Qxbty4GDt2bPXBuKlTpx73izDmzp1bfRHGkiVLIt1SzRdhdJcfZ++DAIFWE1CAW23FnW9HCSjATV6A09XadKX3l7/8Zfzxj3+Ms846K1Ip/qd/+qdq60PbI30VcvqWt8O/Cjnd9uzwx8GDB6Ptq5B37txZXflNZfqSSy6pO4+CVzeZAQIECBwloAALBYE8AT2kyQtwXiw6f0rwOt/YKxAg0PwCCnDzr7Ez7BwBPUQB7pxkvcCzCl5D2L0oAQJNJqAAN9mCOp0uE9BDFOAuC9vhLyR4DWH3ogQINJmAAtxkC+p0ukxAD1GAuyxsCnBDqL0oAQJNLKAAN/HiOrVOFVCAFeBODdjxnlzwGsLuRQkQaDIBBbjJFtTpdJmAHqIAd1nYXAFuCLUXJUCgiQUU4CZeXKfWqQIKsALcqQFzBbghvF6UAIEWEVCAW2ShnWaHCyjACnCHh6o9Tyh47VFyDAECBJ5fQAGWEAJ5AnqIApyXnMIpwSsENE6AAIGIUIDFgECegB6iAOclp3BK8AoBjRMgQEABlgEC2QJ6iAKcHZ6SQcEr0TNLgACBQwKuAEsCgTwBPUQBzktO4ZTgFQIaJ0CAgAIsAwSyBfQQBTg7PCWDgleiZ5YAAQKuAMsAgRIBPUQBLslP9qzgZdMZJECAQE3AFghhIJAnoIcowHnJKZwSvEJA4wQIELAFQgYIZAvoIQpwdnhKBgWvRM8sAQIEDgm4AiwJBPIE9BAFOC85hVOCVwhonAABAgqwDBDIFtBDFODs8JQMCl6JnlkCBAi4AiwDBEoE9BAFuCQ/2bOCl01nkAABAjUBWyCEgUCegB6iAOclp3BK8AoBjRMgQMAWCBkgkC2ghyjA2eEpGRS8Ej2zBAgQOCTgCrAkEMgT0EMU4LzkFE4JXiGgcQIECCjAMkAgW0APUYCzw1MyKHglemYJECDgCrAMECgR0EMU4JL8ZM8KXjadQQIECNQEbIEQBgJ5AnqIApyXnMIpwSsENE6AAAFbIGSAQLaAHqIAZ4enZFDwSvTMEiBA4JCAK8CSQCBPQA9RgPOSUzgleIWAxgkQIKAAywCBbAE9RAHODk/JoOCV6JklQICAK8AyQKBEQA9RgEvykz0reNl0BgkQIFATsAVCGAjkCeghCnBecgqnBK8Q0DgBAgRsgZABAtkCeogCnB2ekkHBK9EzS4AAgUMCrgBLAoE8AT1EAc5LTuGU4BUCGidAgIACLAMEsgX0EAU4Ozwlg4JXomeWAAECrgDLAIESAT1EAS7JT/as4GXTGSRAgEBNwBYIYSCQJ6CHKMB5ySmcErxCQOMECBCwBUIGCGQL6CEKcHZ4SgYFr0TPLAECBA4JuAIsCQTyBPQQBTgvOYVTglcIaJwAAQIKsAwQyBbQQxTg7PCUDApeiZ5ZAgQIuAIsAwRKBPQQBbgkP9mzgpdNZ5AAAQI1AVsghIFAnoAeogDnJadwSvAKAY0TIEDAFggZIJAtoIcowNnhKRkUvBI9swQIEDgk4AqwJBDIE9BDFOC85BROCV4hoHECBAgowDJAIFtAD1GAs8NTMih4JXpmCRAg4AqwDBAoEdBDmrwA33XXXfGVr3wlfvjDH8bu3bvjta99bXz4wx+OD33oQ3HCCSfUzn7lypVx0003xdatW6N///4xefLkmDBhwlE6t9xyS9x+++2xY8eOGDhwYMyfPz9GjhxZdwYFr24yAwQIEDhKwBYIoSCQJ6CHNHkBHjZsWJx55pnx3ve+N0477bRYs2ZNzJ07N66//vqqvKbHgw8+GCNGjIirrroqrrzyynjggQdi2rRpsWjRohg/fnxNKJXfKVOmxJw5c2Lw4MGxePHiWL58eWzYsKEqw/U8BK8eLccSIEDg2AIKsGQQyBPQQ5q8AO/atStOPfXUI84yXd2944474qmnnoqXvOQlMWbMmOrq8Pr162vHXXPNNXHvvfdGCki6Urxv376qQKdfv/nmm6vjDhw4UBXfQYMGxdKlS+tKoODVxeVgAgQIHFNAARYMAnkCekiTF+Bjnd6dd95ZXe397W9/Gy9/+cujb9++MW/evLjhhhtqh69du7ba2rBx48Y477zzqivHl156aWzatCnOPffc2nEzZsyIBQsWxNNPPx29evVqdwoFr91UDiRAgMBxBRRg4SCQJ6CHtGABTldx77nnnnjyySfjZz/7WQwYMCBWrVoVo0ePrmmkK8f9+vWLVJbTtoiFCxfGddddF88880yceOKJtePSHuMrrrgitm3bVu0dbu9D8Nor5TgCBAgcX0ABlg4CeQJ6SIsV4HRF94ILLqj2+H7iE5+o9vsOHz682gec9gu3Pfbv3x+9e/eOW2+9NSZNmhSzZ8+OmTNnxt69e48QW716dVx22WXx8MMPV1shjvfYs2dPpH/aHtu3b48hQ4bUXZzzYm6KAAECzSmgADfnujqrzhdQgFuoAKc7NwwdOrS6Uvvd7363KrhtBXjdunXVv3tuAb7tttti4sSJVQGeNWtWPPvss0eI3XfffTFq1KjYvHnz834Qbvr06ZG2Szz3Ue+V487/kfAKBAgQ6DkCCnDPWSvvtHsJKMAtUoDTHt20pzddwb3//vvjFa94RXXmW7ZsqWsLRCrAffr0qam1dwuEK8Dd6wffuyFAoDkEFODmWEdn0fUCCnALFOBUei+//PJ49NFHq60O6bZobY90dwcfguv6HzyvSIAAgY4QUIA7QtFztKKAAtzkBTjt5X3f+94X3/ve96p/jrVPN90GLd0SLZXjtse1114bK1asOOo2aOnX0x0j0iPdBi09X7oVmtugteJvH86ZAIFGCyjAjV4Br99TBRTgJi/A6RvfvvCFL1T37r3ooouOONs3v/nN1dXfti/CGDduXIwdO7baFzx16tTjfhFG+iKN9EUYS5YsiWXLlvkijJ760+99EyDQ4wUU4B6/hE6gQQIKcJMX4LPOOisef/zxY55lurdv29cYp69CTt/ydvhXIafbnh3+OHjwYLR9FfLOnTurK7+pWF9yySV1x1fw6iYzQIAAgaMEFGChIJAnoIc0eQHOi0XnTwle5xt7BQIEml9AAW7+NXaGnSOghyjAnZOsF3hWwWsIuxclQKDJBBTgJltQp9NlAnqIAtxlYTv8hQSvIexelACBJhNQgJtsQZ1OlwnoIQpwl4VNAW4ItRclQKCJBRTgJl5cp9apAgqwAtypATvekwteQ9i9KAECTSagADfZgjqdLhPQQxTgLgubK8ANofaiBAg0sYAC3MSL69Q6VUABVoA7NWCuADeE14sSINAiAgpwiyy00+xwAQVYAe7wULXnCQWvPUqOIUCAwPMLKMASQiBPQA9RgPOSUzgleIWAxgkQIBARCrAYEMgT0EMU4LzkFE4JXiGgcQIECCjAMkAgW0APUYCzw1MyKHglemYJECBwSMAVYEkgkCeghyjAeckpnBK8QkDjBAgQUIBlgEC2gB6iAGeHp2RQ8Er0zBIgQMAVYBkgUCKghyjAJfnJnhW8bDqDBAgQqAnYAiEMBPIE9BAFOC85hVOCVwhonAABArZAyACBbAE9RAHODk/JoOCV6JklQIDAIQFXgCWBQJ6AHqIA5yWncErwCgGNEyBAQAGWAQLZAnqIApwdnpJBwSvRM0uAAAFXgGWAQImAHqIAl+Qne1bwsukMEiBAoCZgC4QwEMgT0EMU4LzkFE4JXiGgcQIECNgCIQMEsgX0EAU4Ozwlg4JXomeWAAEChwRcAZYEAnkCeogCnJecwinBKwQ0ToAAAQVYBghkC+ghCnB2eEoGBa9EzywBAgRcAZYBAiUCeogCXJKf7FnBy6YzSIAAgZqALRDCQCBPQA9RgPOSUzgleIWAxgkQIGALhAwQyBbQQxTg7PCUDApeiZ5ZAgQIHBJwBVgSCOQJ6CEKcF5yCqcErxDQOAECBBRgGSCQLaCHKMDZ4SkZFLwSPbMECBBwBVgGCJQI6CEKcEl+smcFL5vOIAECBGoCtkAIA4E8AT1EAc5LTuGU4BUCGidAgIAtEDJAIFtAD1GAs8NTMih4JXpmCRAgcEjAFWBJIJAnoIcowHnJKZwSvEJA4wQIEFCAZYBAtoAeogBnh6dkUPBK9MwSIEDAFWAZIFAioIcowCX5yZ4VvGw6gwQIEKgJ2AIhDATyBPQQBTgvOYVTglcIaJwAAQK2QMgAgWwBPUQBzg5PyaDgleiZJUCAwCEBV4AlgUCegB6iAOclp3BK8AoBjRMgQEABlgEC2QJ6iAKcHZ6SQcEr0TNLgAABV4BlgECJgB6iAJfkJ3tW8LLpDBIgQKAmYAuEMBDIE9BDFOC85BROCV4hoHECBAjYAiEDBLIF9BAFODs8JYOCV6JnlgABAocEXAGWBAJ5AnqIApyXnMIpwSsENE6AAAEFWAYIZAvoIQpwdnhKBgWvRM8sAQIEXAGWAQIlAnpICxTgX/ziF3HLLbfEunXr4pFHHok3vvGN1f8+97Fy5cq46aabYuvWrdG/f/+YPHlyTJgw4ajj0nPdfvvtsWPHjhg4cGDMnz8/Ro4cWVcOBa8uLgcTIEDgmAK2QAgGgTwBPaQFCvA3v/nN+MhHPhJDhw6Nn//85/HHP/7xqAL84IMPxogRI+Kqq66KK6+8Mh544IGYNm1aLFq0KMaPH19TSuV3ypQpMWfOnBg8eHAsXrw4li9fHhs2bKjKcHsfgtdeKccRIEDg+AIKsHQQyBPQQ1qgAKfCe8IJJ1RnOm7cuNi4ceNRBXjMmDGxe/fuWL9+fU3kmmuuiXvvvTdSSNL8vn374rTTTov06zfffHN13IEDB6riO2jQoFi6dGm7Uyh47aZyIAECBI4roAALB4E8AT2kBQrw4ad4rAKcim3fvn1j3rx5ccMNN9QOX7t2bbW1IRXm8847L9asWROXXnppbNq0Kc4999zacTNmzIgFCxbE008/Hb169WpXEgWvXUwOIkCAwPMKKMACQiBPQA9RgGPLli0xYMCAWLVqVYwePbomsmvXrujXr1/ceeed1baIhQsXxnXXXRfPPPNMnHjiibXj7rrrrrjiiiti27Zt1d7h9jwErz1KjiFAgMDzCyjAEkIgT0APUYCr/b7Dhw+PtA942LBhNZH9+/dH796949Zbb41JkybF7NmzY+bMmbF3794j1FavXh2XXXZZPPzww9VWiGM99uzZE+mftsf27dtjyJAhdZXmvIibIkCAQPMKKMDNu7bOrHMFFGAFuFaA010i0gfl2h5tBfi2226LiRMnVgV41qxZ8eyzzx6hdt9998WoUaNi8+bNx/0g3PTp0yNtlXjuo56rxp37o+DZCRAg0PMEFOCet2becfcQUIAV4Lq3QKQC3KdPn5pce7ZAuALcPX7gvQsCBJpLQAFurvV0Nl0noAArwNXdHXwIrut+6LwSAQIEOkpAAe4oSc/TagIKsAJcCaTboD311FPVPuC2x7XXXhsrVqw46jZo6dfTHSPSI90GLe37TbdCcxu0Vvvtw/kSINBoAQW40Svg9XuqgALcAgU43bUhfctbenzuc5+LX/7yl/HpT3+6+v8XX3xxnHrqqVXxTV+EkW6TNnbs2Gpf8NSpU4/7RRhz586tvghjyZIlsWzZMl+E0VN/B/C+CRDo0QIKcI9ePm++gQIKcAsU4MceeyzOPvvsY55purdv29cYp5KcvuXt8K9CTrc9O/xx8ODB6muV01ch79y5s7rym74U45JLLqkrxoJXF5eDCRAgcEwBBVgwCOQJ6CEtUIDzotG5U4LXub6enQCB1hBQgFtjnZ1lxwvoIQpwx6eqHc8oeO1AcggBAgReQEABFhECeQJ6iAKcl5zCKcErBDROgACBiFCAxYBAnoAeogDnJadwSvAKAY0TIEBAAZYBAtkCeogCnB2ekkHBK9EzS4AAgUMCrgBLAoE8AT1EAc5LTuGU4BUCGidAgIACLAMEsgX0EAU4Ozwlg4JXomeWAAECrgDLAIESAT1EAS7JT/as4GXTGSRAgEBNwBYIYSCQJ6CHKMB5ySmcErxCQOMECHRrga4qpl2F8Ni8/9NVL+V1CHSJgB6iAHdJ0J77IoLXEHYvSoBAFwkowF0E7WUIZAroIQpwZnTKxgSvzM80AQLdW0AB7t7r490R0EMU4Ib8FAheQ9i9KAECXSSgAHcRtJchkCmghyjAmdEpGxO8Mj/TBAh0bwEFuHuvj3dHQA9RgBvyUyB4DWH3ogQIdJGAAtxF0F6GQKaAHqIAZ0anbEzwyvxMEyCQJ9BsxTRPof4pd4Go38xE9xbQQxTghiRU8BrC7kUJtLyAApwXAQU4z81U9xXQQxTghqRT8BrC7kUJtLyAApwXAQU4z81U9xXQQxTghqRT8BrC7kUJdFsBxbTbLk31xhTg7r0+3l39AnqIAlx/ajpgQvA6ANFTEGgiAQW4ey+mAty918e7q19AD1GA609NB0wIXgcgegoCTSSgAHfvxVSAu/f6eHf1C+ghCnD9qemACcHrAERPQaCJBBTg7r2YCnD3Xh/vrn4BPUQBrj81HTAheB2A6CkINJGAAty9F1MB7t7r493VL6CHKMD1p6YDJgSvAxA9BYEmElCAu/diKsDde328u/oF9BAFuP7UdMCE4HUAoqcg0EQCCnD3XkwFuHuvj3dXv4AeogDXn5oOmBC8DkD0FASaSEAB7t6LqQB37/Xx7uoX0EMU4PpT0wETgtcBiJ6CQBcIKKZdgNwDXkIB7gGL5C3WJaCHKMB1BaajDha8jpL0PAQ6V0AB7lzfnvLsCnBPWSnvs70CeogC3N6sdOhxgtehnJ6MQKcJKMCdRtujnlgB7lHL5c22Q0APUYDbEZOOP0TwOt7UMxLoDAEFuDNUe95zKsA9b8284+cX0EMU4Ib8jAheQ9i9aCcLdGVZ7KpC0pXn1MnL4+kLBLoqbwVv0SiBugT0EAW4rsB01MGC11GSnqc7CXRlWeyqQtKV59Sd1tJ7OVKgq/LGnUBXCeghCnBXZe2I1xG8hrB70U4W6Mqy2FWFpCvPqZOXx9MXCHRV3greolECdQnoIQpwXYHpqIMFr6MkPU93EujKsthVhaQrz6k7raX34gqwDDS3gB6iADck4YLXEPZu96JdVa6UxW639N5QDxPoqp+hHsbi7fZgAT1EAW5IfAWvIezd7kW7qgB3uxP3hgj0MAEFuIctmLf7ggJ6iAL8giHpjAMErzNUe95zKsA9b82849YUUIBbc92b+az1EAW4IfkWvIawd7sXVYC73ZJ4QwSOKaAAC0azCeghCnBDMi14DWHvdi+qAHe7JfGGCCjAMtASAnqIAv86+eMAAA55SURBVNyQoAteQ9i73YsqwN1uSbwhAgqwDLSEgB6iADck6ILXEPZu96IKcLdbEm+IgAIsAy0hoIcowA0JuuA1hL3bvagC3O2WxBsioADLQEsI6CEKcEOCLngNYe92L6oAd7sl8YYIKMAy0BICeogC3JCgC15D2Nv9ooppu6kcSKAlBNwFoiWWuaVOUg9RgBsSeMFrCHu7X1QBbjeVAwm0hIAC3BLL3FInqYcowHUH/uc//3lMmjQpvv/978dJJ50Uf/u3fxvz5s2LE088sd3PJXjtpmrIgQpwQ9i9KIFuK6AAd9ul8cYyBfQQBbiu6Dz11FNxzjnnxJlnnhn/9//+33jyySdj8uTJMXr06Pjyl7/c7ucSvHZTNeRABbgh7F6UQLcVUIC77dJ4Y5kCeogCXFd0PvWpT8UnP/nJePzxx+OUU06pZr/61a/G2LFjY8uWLfGmN72pXc8neO1iOuogxTTPzRQBAmUCCnCZn+nuJ6CHKMB1pfLiiy+Ol73sZfHNb36zNrdv3744+eSTY/bs2fHRj360Xc8neO1iUoDzmEwRINDBAgpwB4N6uoYL6CEKcF0h7NevX1x99dXVnt/DHwMGDIgLLrgglixZ0q7na7bguTLbrmV3EAECPVRAAe6hC+dtH1eg2XpIRy51r4MHDx7syCdshufq3bt3zJw5Mz7+8Y8fcTrDhw+PVI6XLVt2zNPcs2dPpH/aHtu2bYu3v/3tsWHDhnjlK1/Z42mGzfnPHn8OToAAAQLHE1g35R1wCDSVwPbt22PIkCHx61//Os4666ymOrfSk1GAjyGYCvCsWbPixhtvPOLfXnjhhXH66afHPffcc0z36dOnx4wZM0rXxDwBAgQIECBAoMME0oW4888/v8OerxmeSAE+xirmboF47hXgvXv3RroKfPbZZ8eLXvSiTslL29/umuUqc6cgNfBJrU8D8dv50taonVANOsz6NAi+nS9rfdoJ1aDD9u/fH7t27YqBAwdGnz59GvQuuufLKsDHWJeO+hBcVyy5/T1doZz/GtYn366rJq1RV0nnvY71yXPrqinr01XSXqejBRTgY4im26ClPcDpNmiveMUrqiOWLl1afRlGPbdB6+jFOtbz+c2nK5TzX8P65Nt11aQ16irpvNexPnluXTVlfbpK2ut0tIACfAzRti/CSBvGD/8ijMsvv7yuL8Lo6MVSgLtCtGNfwx8OHevZGc9mjTpDteOe0/p0nGVnPJP16QxVz9kVAgrwcZTTVyFPnDgx7r///njpS19aXf1NV4br+SrkrljAtO/405/+dPVNdX379u2Kl/QadQhYnzqwGnSoNWoQfDtf1vq0E6pBh1mfBsF72WIBBbiY0BMQIECAAAECBAj0JAEFuCetlvdKgAABAgQIECBQLKAAFxN6AgIECBAgQIAAgZ4koAD3pNXyXgkQIECAAAECBIoFFOBiQk9AgAABAgQIECDQkwQU4J60Woe913SXikmTJsX3v//9OOmkk6q7VMybN6/b3aWih/LW9bbvuuuu+MpXvhI//OEPY/fu3fHa1742PvzhD8eHPvShOOGEE2rPtXLlyrjpppti69at0b9//+rOHRMmTKjrtRxcLvD73/8+3vjGN8YTTzwRDz30ULztbW+zRuWsxc/wxS9+MW677bb42c9+Vt3RZtiwYbFixQprUyzbMU+wfPnymDt3bvX7V7ob0oUXXlj9/ze84Q1HvIDf5zrG27N0voAC3PnGHf4KbfcpPvPMM4+4T/Ho0aO73X2KO/zku+ETpj+o01q8973vjdNOOy3WrFlT/cFw/fXXx/z586t3/OCDD8aIESPiqquuiiuvvDIeeOCBmDZtWixatCjGjx/fDc+qed/SjTfeGF/60pdi586dRxRga9S4NZ8+fXp85jOfqf6COHTo0Oovkt/+9rernw8/P41bl7ZXXr16dYwaNar6veuDH/xgpD+D0pqlW6D95Cc/qd2C089Q49fKO2i/gALcfqtuc2S6H/EnP/nJ6pvqTjnllOp9ffWrX42xY8d2u2+q6zZonfhG0vesn3rqqUe8Qrq6e8cdd1R/ULzkJS+JMWPGVH+or1+/vnbcNddcE/fee2+kG8kffqW4E99qyz/1T3/60+qK74IFC+Laa689ogBbo8bEI11RHDhwYKQrh6lkHethbRqzNm2vmv6S/p//+Z/xq1/9Knr16lX98oYNG6q/rKR1S+uTHtapsevk1esTUIDr8+oWR1988cXxspe9LL75zW/W3s++ffvi5JNPjtmzZ8dHP/rRbvE+W/lN3HnnndXV3t/+9rfx8pe/vLpCkrao3HDDDTWWtWvXxsiRI2Pjxo1x3nnntTJXl517KliDBg2Kd73rXXHJJZfUCnD6+bFGXbYMR7xQuiL/jW98I9K2rmM9rE1j1uXwV/27v/u7+K//+q/YvHlz7ZfTeqXtD9/61rfine98Z1inxq+Td1CfgAJcn1e3OLpfv35x9dVXV4Xq8MeAAQPiggsuiCVLlnSL99nKbyJd3b3nnnviySefrPY0prVZtWpVpG0qbY905TitZSrL6T8tenSuwN133x3XXXddPProo7Fp06YjCvCWLVusUefyH/fZ01/o03/Jeutb3xqf/exnq/9qkn4fu/XWW6tfszYNWpjDXvZ73/tevOMd76i+dbRtC0T6DMovfvGLqhin/8plnRq/Tt5BfQIKcH1e3eLo3r17x8yZM+PjH//4Ee9n+PDhVaFatmxZt3ifrfom0hXd9Ad42uP7iU98otrvm9Ym7Y9L+4XbHvv374+0lukP+vSHiUfnCTzzzDPVB9/SvsX0l8fvfve7RxRga9R59i/0zOkqYvovJWeccUbMmTMnXvziF8eMGTPiscceq/6ykvaY+vl5IcXO//dpu9YHPvCB+N3vfle92Jvf/Ob4zne+U32gNz38DHX+GniFjhVQgDvWs0ueLZWmWbNmRfpPh4c/0qdyTz/99OrKo0djBHbs2FHti0t/KKSSldaq7Q+GdevWVf/uuQU4ffJ94sSJjXnDLfKqU6ZMifvuu6/ag532Wx+vAFujrg/E6173uupK4iOPPFJdhU+P7du3x9lnn1191iH9vpYKsLXp+rVpe8Uf/OAH1TaHcePGxV/+5V/G008/Xf1lZe/evdXvb2n7kN/nGrc+XjlPQAHOc2volC0QDeU/7ounPxTSnt70h8L9998fr3jFK6pj/afBxq5X+rDo61//+mqf6dvf/vbqzaT1efe7313dsSN9KO43v/mNLRANWqb0l8K0Rukvj4c/0vaHt7zlLdVf9G0hatDi/P+XTT8jr371q4/4r4tpC1f6i36640360K/f5xq7Rl69fgEFuH6zhk/4EFzDl+CoN5BK7+WXX179J9u01SHdFq3t4cMhjV2vtqu9x3sXqYClDyT6EFxj1ildVUy3PHtuAU7ld/DgwfH5z3/e2jRmaWqv+tKXvrTacjd16tQj3km6Sp+uDH/uc5/zIbgGr5GXr19AAa7frOET6TZoaQ9wumrSdpVx6dKl1ZdhpL+Fv+lNb2r4e2ylN5D28r7vfe+L9EGR9E+6y8BzH+n2QOnDPakctz3SbbjSjf7dBq1z05Lcf/SjHx3xIun/pztypHJ1/vnnV0XLGnXuOhzv2dOHE9///vfHj3/84zjnnHOqw9KXlLzmNa+J9Htdup+2tWnM2rS9avozJf1XlMPvPJT+wpKuCqc7D33sYx+rDrVOjV0nr16fgAJcn1e3OLrtizDOOuusI74II12B/PKXv9wt3mMrvYn0jW9f+MIX4uabb46LLrroiFNPHxRJVxbbbhCfrnal+zWn/XLpaoovwmhMUp67Bzi9C2vUmLU4cOBADBkypPpwVfpsQ/oQXNr723YHlfRNl9amMWvT9qq333579TmFdBeV97znPdVf5tMe4HQRJn1I8ZWvfGV1qHVq7Dp59foEFOD6vLrN0ekejOk3pLSXMf3nqXT1N10tSV9R6dG1AukvIukPgmM90h7TtC84PdIN49OHsQ7/KuT0B4pH1wscqwBbo65fh7ZXTGU3XZFP95T9wx/+EGmbV/pmuMO/ZtfPT+PW5+DBg7F48eJYuHBh9YHFP/3TP63+0pKu/qYvMTn8YZ0at05euT4BBbg+L0cTIECAAAECBAj0cAEFuIcvoLdPgAABAgQIECBQn4ACXJ+XowkQIECAAAECBHq4gALcwxfQ2ydAgAABAgQIEKhPQAGuz8vRBAgQIECAAAECPVxAAe7hC+jtEyBAgAABAgQI1CegANfn5WgCBAgQIECAAIEeLqAA9/AF9PYJECBAgAABAgTqE1CA6/NyNAECBAgQIECAQA8XUIB7+AJ6+wQIECBAgAABAvUJKMD1eTmaAAECBAgQIECghwsowD18Ab19AgQIECBAgACB+gQU4Pq8HE2AAAECBAgQINDDBRTgHr6A3j4BAgQIECBAgEB9AgpwfV6OJkCAAAECBAgQ6OECCnAPX0BvnwABAgQIECBAoD4BBbg+L0cTIECAAAECBAj0cAEFuIcvoLdPgAABAgQIECBQn4ACXJ+XowkQIECAAAECBHq4gALcwxfQ2ydAgAABAgQIEKhPQAGuz8vRBAgQIECAAAECPVxAAe7hC+jtEyBAgAABAgQI1CegANfn5WgCBAgQIECAAIEeLqAA9/AF9PYJECBAgAABAgTqE1CA6/NyNAECBAgQIECAQA8XUIB7+AJ6+wQIECBAgAABAvUJKMD1eTmaAAECBAgQIECghwsowD18Ab19AgQIECBAgACB+gQU4Pq8HE2AAAECBAgQINDDBRTgHr6A3j4BAgQIECBAgEB9AgpwfV6OJkCAAAECBAgQ6OECCnAPX0BvnwABAgQIECBAoD4BBbg+L0cTIECAAAECBAj0cAEFuIcvoLdPgAABAgQIECBQn4ACXJ+XowkQIECAAAECBHq4wP8DeILtkvXcC/wAAAAASUVORK5CYII=\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.figure()\n",
    "plt.hist(list(reg_size_dic.values()), bins=np.arange(0,100,5))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2db97b4d",
   "metadata": {},
   "source": [
    "### 4.1.3 check readout usage for each gene"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 198,
   "id": "8d0a6c3d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Chr7-6Mb-seg-1 -> ['NDB_801', 'NDB_725', 'NDB_710']\n",
      "Chr7-6Mb-seg-2 -> ['NDB_400', 'NDB_750', 'NDB_825']\n",
      "Chr7-6Mb-seg-3 -> ['NDB_458', 'NDB_466', 'NDB_511']\n",
      "Chr7-6Mb-seg-4 -> ['NDB_786', 'NDB_860', 'NDB_680']\n",
      "Chr7-6Mb-seg-7 -> ['NDB_754', 'NDB_1020', 'NDB_681']\n",
      "Chr7-6Mb-seg-9 -> ['NDB_461', 'NDB_631', 'NDB_532']\n",
      "Chr7-6Mb-seg-10 -> ['NDB_921', 'NDB_887', 'NDB_506']\n",
      "Chr7-6Mb-seg-11 -> ['NDB_835', 'NDB_846', 'NDB_864']\n",
      "Chr7-6Mb-seg-12 -> ['NDB_941', 'NDB_385', 'NDB_838']\n",
      "Chr7-6Mb-seg-14 -> ['NDB_396', 'NDB_660', 'NDB_706']\n",
      "Chr7-6Mb-seg-15 -> ['NDB_784', 'NDB_759', 'NDB_718']\n",
      "Chr7-6Mb-seg-16 -> ['NDB_971', 'NDB_847', 'NDB_431']\n",
      "Chr7-6Mb-seg-19 -> ['NDB_768', 'NDB_554', 'NDB_621']\n",
      "Chr7-6Mb-seg-20 -> ['NDB_901', 'NDB_375', 'NDB_529']\n",
      "Chr7-6Mb-seg-21 -> ['NDB_443', 'NDB_508', 'NDB_878']\n",
      "Chr7-6Mb-seg-22 -> ['NDB_438', 'NDB_332', 'NDB_741']\n",
      "Chr7-6Mb-seg-23 -> ['NDB_616', 'NDB_444', 'NDB_805']\n",
      "Chr7-6Mb-seg-24 -> ['NDB_347', 'NDB_730', 'NDB_896']\n",
      "Chr7-6Mb-seg-25 -> ['NDB_813', 'NDB_989', 'NDB_687']\n",
      "Chr7-6Mb-seg-27 -> ['NDB_514', 'NDB_356', 'NDB_788']\n",
      "Chr7-6Mb-seg-28 -> ['NDB_826', 'NDB_795', 'NDB_376']\n",
      "Chr7-6Mb-seg-29 -> ['NDB_743', 'NDB_643', 'NDB_683']\n",
      "Chr7-6Mb-seg-36 -> ['NDB_852', 'NDB_917', 'NDB_762']\n",
      "Chr7-6Mb-seg-38 -> ['NDB_487', 'NDB_516', 'NDB_409']\n",
      "Chr7-6Mb-seg-39 -> ['NDB_455', 'NDB_403', 'NDB_470']\n",
      "Chr7-6Mb-seg-40 -> ['NDB_405', 'NDB_758', 'NDB_1035']\n",
      "Chr7-6Mb-seg-41 -> ['NDB_1075', 'NDB_789', 'NDB_388']\n",
      "Chr7-6Mb-seg-42 -> ['NDB_752', 'NDB_880', 'NDB_482']\n",
      "Chr7-6Mb-seg-43 -> ['NDB_630', 'NDB_641', 'NDB_453']\n",
      "Chr7-6Mb-seg-44 -> ['NDB_1066', 'NDB_665', 'NDB_445']\n",
      "Chr7-6Mb-seg-45 -> ['NDB_865', 'NDB_817', 'NDB_690']\n",
      "Chr7-6Mb-seg-46 -> ['NDB_815', 'NDB_965', 'NDB_778']\n",
      "Chr7-6Mb-seg-47 -> ['NDB_804', 'NDB_915', 'NDB_746']\n",
      "Chr7-6Mb-seg-48 -> ['NDB_571', 'NDB_448', 'NDB_957']\n",
      "Chr7-6Mb-seg-49 -> ['NDB_737', 'NDB_698', 'NDB_391']\n",
      "Chr7-6Mb-seg-50 -> ['NDB_663', 'NDB_954', 'NDB_398']\n",
      "Chr7-6Mb-seg-51 -> ['NDB_1060', 'NDB_427', 'NDB_780']\n",
      "Chr7-6Mb-seg-52 -> ['NDB_446', 'NDB_473', 'NDB_397']\n",
      "Chr7-6Mb-seg-53 -> ['NDB_816', 'NDB_1044', 'NDB_647']\n",
      "Chr7-6Mb-seg-54 -> ['NDB_498', 'NDB_1023', 'NDB_637']\n",
      "Chr7-6Mb-seg-55 -> ['NDB_578', 'NDB_713', 'NDB_777']\n",
      "Chr7-6Mb-seg-56 -> ['NDB_936', 'NDB_456', 'NDB_760']\n",
      "Chr7-6Mb-seg-57 -> ['NDB_961', 'NDB_661', 'NDB_728']\n",
      "Chr7-6Mb-seg-58 -> ['NDB_773', 'NDB_812', 'NDB_432']\n",
      "Chr7-6Mb-seg-59 -> ['NDB_489', 'NDB_417', 'NDB_1027']\n",
      "Chr7-6Mb-seg-60 -> ['NDB_715', 'NDB_883', 'NDB_536']\n",
      "Chr7-6Mb-seg-61 -> ['NDB_467', 'NDB_335', 'NDB_390']\n",
      "Chr7-6Mb-seg-62 -> ['NDB_507', 'NDB_747', 'NDB_832']\n",
      "Chr7-6Mb-seg-63 -> ['NDB_721', 'NDB_853', 'NDB_899']\n",
      "Chr7-6Mb-seg-64 -> ['NDB_994', 'NDB_734', 'NDB_828']\n",
      "Chr7-6Mb-seg-65 -> ['NDB_725', 'NDB_807', 'NDB_718']\n",
      "Chr7-6Mb-seg-66 -> ['NDB_750', 'NDB_790', 'NDB_431']\n",
      "Chr7-6Mb-seg-67 -> ['NDB_466', 'NDB_974', 'NDB_621']\n",
      "Chr7-6Mb-seg-68 -> ['NDB_860', 'NDB_858', 'NDB_529']\n",
      "Chr7-6Mb-seg-69 -> ['NDB_1020', 'NDB_856', 'NDB_878']\n",
      "Chr7-6Mb-seg-70 -> ['NDB_631', 'NDB_653', 'NDB_741']\n",
      "Chr7-6Mb-seg-71 -> ['NDB_887', 'NDB_501', 'NDB_805']\n",
      "Chr7-6Mb-seg-72 -> ['NDB_846', 'NDB_433', 'NDB_896']\n",
      "Chr7-6Mb-seg-73 -> ['NDB_385', 'NDB_452', 'NDB_687']\n",
      "Chr7-6Mb-seg-74 -> ['NDB_356', 'NDB_665', 'NDB_396']\n",
      "Chr7-6Mb-seg-75 -> ['NDB_784', 'NDB_755', 'NDB_826']\n",
      "Chr7-6Mb-seg-76 -> ['NDB_971', 'NDB_699', 'NDB_743']\n",
      "Chr7-6Mb-seg-77 -> ['NDB_768', 'NDB_844', 'NDB_852']\n",
      "Chr7-6Mb-seg-78 -> ['NDB_901', 'NDB_875', 'NDB_487']\n",
      "Chr7-6Mb-seg-79 -> ['NDB_443', 'NDB_465', 'NDB_455']\n",
      "Chr7-6Mb-seg-80 -> ['NDB_438', 'NDB_634', 'NDB_405']\n",
      "Chr7-6Mb-seg-81 -> ['NDB_616', 'NDB_905', 'NDB_1075']\n",
      "Chr7-6Mb-seg-82 -> ['NDB_347', 'NDB_615', 'NDB_752']\n",
      "Chr7-6Mb-seg-83 -> ['NDB_813', 'NDB_946', 'NDB_630']\n",
      "Chr7-6Mb-seg-84 -> ['NDB_881', 'NDB_935', 'NDB_711']\n",
      "Chr7-6Mb-seg-85 -> ['NDB_759', 'NDB_376', 'NDB_801']\n",
      "Chr7-6Mb-seg-86 -> ['NDB_847', 'NDB_683', 'NDB_400']\n",
      "Chr7-6Mb-seg-87 -> ['NDB_554', 'NDB_762', 'NDB_458']\n",
      "Chr7-6Mb-seg-88 -> ['NDB_375', 'NDB_409', 'NDB_786']\n",
      "Chr7-6Mb-seg-89 -> ['NDB_508', 'NDB_470', 'NDB_754']\n",
      "Chr7-6Mb-seg-90 -> ['NDB_332', 'NDB_1035', 'NDB_461']\n",
      "Chr7-6Mb-seg-91 -> ['NDB_444', 'NDB_388', 'NDB_921']\n",
      "Chr7-6Mb-seg-92 -> ['NDB_730', 'NDB_482', 'NDB_835']\n",
      "Chr7-6Mb-seg-93 -> ['NDB_989', 'NDB_453', 'NDB_941']\n",
      "Chr7-6Mb-seg-94 -> ['NDB_498', 'NDB_706', 'NDB_788']\n",
      "Chr7-6Mb-seg-95 -> ['NDB_795', 'NDB_533', 'NDB_652']\n",
      "Chr7-6Mb-seg-96 -> ['NDB_643', 'NDB_738', 'NDB_464']\n",
      "Chr7-6Mb-seg-97 -> ['NDB_917', 'NDB_568', 'NDB_387']\n",
      "Chr7-6Mb-seg-98 -> ['NDB_516', 'NDB_716', 'NDB_868']\n",
      "Chr7-6Mb-seg-99 -> ['NDB_403', 'NDB_657', 'NDB_863']\n",
      "Chr7-6Mb-seg-100 -> ['NDB_758', 'NDB_1033', 'NDB_771']\n",
      "Chr7-6Mb-seg-101 -> ['NDB_789', 'NDB_866', 'NDB_619']\n",
      "Chr7-6Mb-seg-102 -> ['NDB_880', 'NDB_1047', 'NDB_539']\n",
      "Chr7-6Mb-seg-103 -> ['NDB_641', 'NDB_505', 'NDB_618']\n",
      "Chr7-6Mb-seg-104 -> ['NDB_514', 'NDB_665', 'NDB_889']\n",
      "Chr7-6Mb-seg-105 -> ['NDB_578', 'NDB_865', 'NDB_718']\n",
      "Chr7-6Mb-seg-106 -> ['NDB_936', 'NDB_815', 'NDB_431']\n",
      "Chr7-6Mb-seg-107 -> ['NDB_961', 'NDB_804', 'NDB_621']\n",
      "Chr7-6Mb-seg-108 -> ['NDB_773', 'NDB_571', 'NDB_529']\n",
      "Chr7-6Mb-seg-109 -> ['NDB_489', 'NDB_737', 'NDB_878']\n",
      "Chr7-6Mb-seg-110 -> ['NDB_715', 'NDB_663', 'NDB_741']\n",
      "Chr7-6Mb-seg-111 -> ['NDB_467', 'NDB_1060', 'NDB_805']\n",
      "Chr7-6Mb-seg-112 -> ['NDB_507', 'NDB_446', 'NDB_896']\n",
      "Chr7-6Mb-seg-113 -> ['NDB_721', 'NDB_816', 'NDB_687']\n",
      "Chr7-6Mb-seg-114 -> ['NDB_1023', 'NDB_396', 'NDB_711']\n",
      "Chr7-6Mb-seg-115 -> ['NDB_755', 'NDB_777', 'NDB_710']\n",
      "Chr7-6Mb-seg-116 -> ['NDB_699', 'NDB_760', 'NDB_825']\n",
      "Chr7-6Mb-seg-117 -> ['NDB_844', 'NDB_728', 'NDB_511']\n",
      "Chr7-6Mb-seg-118 -> ['NDB_875', 'NDB_432', 'NDB_680']\n",
      "Chr7-6Mb-seg-119 -> ['NDB_465', 'NDB_1027', 'NDB_681']\n",
      "Chr7-6Mb-seg-120 -> ['NDB_634', 'NDB_536', 'NDB_532']\n",
      "Chr7-6Mb-seg-121 -> ['NDB_905', 'NDB_390', 'NDB_506']\n",
      "Chr7-6Mb-seg-122 -> ['NDB_615', 'NDB_832', 'NDB_864']\n",
      "Chr7-6Mb-seg-123 -> ['NDB_946', 'NDB_899', 'NDB_838']\n",
      "Chr7-6Mb-seg-124 -> ['NDB_637', 'NDB_828', 'NDB_706']\n",
      "Chr7-6Mb-seg-125 -> ['NDB_784', 'NDB_807', 'NDB_690']\n",
      "Chr7-6Mb-seg-126 -> ['NDB_971', 'NDB_790', 'NDB_778']\n",
      "Chr7-6Mb-seg-127 -> ['NDB_768', 'NDB_974', 'NDB_746']\n",
      "Chr7-6Mb-seg-128 -> ['NDB_901', 'NDB_858', 'NDB_957']\n",
      "Chr7-6Mb-seg-129 -> ['NDB_443', 'NDB_856', 'NDB_391']\n",
      "Chr7-6Mb-seg-130 -> ['NDB_438', 'NDB_653', 'NDB_398']\n",
      "Chr7-6Mb-seg-131 -> ['NDB_616', 'NDB_501', 'NDB_780']\n",
      "Chr7-6Mb-seg-132 -> ['NDB_347', 'NDB_433', 'NDB_397']\n",
      "Chr7-6Mb-seg-133 -> ['NDB_813', 'NDB_452', 'NDB_647']\n",
      "Chr7-6Mb-seg-134 -> ['NDB_356', 'NDB_734', 'NDB_935']\n",
      "Chr7-6Mb-seg-135 -> ['NDB_578', 'NDB_533', 'NDB_725']\n",
      "Chr7-6Mb-seg-136 -> ['NDB_936', 'NDB_738', 'NDB_750']\n",
      "Chr7-6Mb-seg-137 -> ['NDB_961', 'NDB_568', 'NDB_466']\n",
      "Chr7-6Mb-seg-138 -> ['NDB_773', 'NDB_716', 'NDB_860']\n",
      "Chr7-6Mb-seg-139 -> ['NDB_489', 'NDB_657', 'NDB_1020']\n",
      "Chr7-6Mb-seg-140 -> ['NDB_715', 'NDB_1033', 'NDB_631']\n",
      "Chr7-6Mb-seg-141 -> ['NDB_467', 'NDB_866', 'NDB_887']\n",
      "Chr7-6Mb-seg-142 -> ['NDB_507', 'NDB_1047', 'NDB_846']\n",
      "Chr7-6Mb-seg-143 -> ['NDB_721', 'NDB_505', 'NDB_385']\n",
      "Chr7-6Mb-seg-144 -> ['NDB_881', 'NDB_1066', 'NDB_660']\n",
      "Chr7-6Mb-seg-145 -> ['NDB_376', 'NDB_713', 'NDB_817']\n",
      "Chr7-6Mb-seg-146 -> ['NDB_683', 'NDB_456', 'NDB_965']\n",
      "Chr7-6Mb-seg-147 -> ['NDB_762', 'NDB_661', 'NDB_915']\n",
      "Chr7-6Mb-seg-148 -> ['NDB_409', 'NDB_812', 'NDB_448']\n",
      "Chr7-6Mb-seg-149 -> ['NDB_470', 'NDB_417', 'NDB_698']\n",
      "Chr7-6Mb-seg-150 -> ['NDB_1035', 'NDB_883', 'NDB_954']\n",
      "Chr7-6Mb-seg-151 -> ['NDB_388', 'NDB_335', 'NDB_427']\n",
      "Chr7-6Mb-seg-152 -> ['NDB_482', 'NDB_747', 'NDB_473']\n",
      "Chr7-6Mb-seg-153 -> ['NDB_453', 'NDB_853', 'NDB_1044']\n",
      "Chr7-6Mb-seg-154 -> ['NDB_498', 'NDB_514', 'NDB_711']\n",
      "Chr7-6Mb-seg-155 -> ['NDB_759', 'NDB_795', 'NDB_865']\n",
      "Chr7-6Mb-seg-156 -> ['NDB_847', 'NDB_643', 'NDB_815']\n",
      "Chr7-6Mb-seg-157 -> ['NDB_554', 'NDB_917', 'NDB_804']\n",
      "Chr7-6Mb-seg-158 -> ['NDB_375', 'NDB_516', 'NDB_571']\n",
      "Chr7-6Mb-seg-159 -> ['NDB_508', 'NDB_403', 'NDB_737']\n",
      "Chr7-6Mb-seg-161 -> ['NDB_332', 'NDB_758', 'NDB_663']\n",
      "Chr7-6Mb-seg-162 -> ['NDB_444', 'NDB_789', 'NDB_1060']\n",
      "Chr7-6Mb-seg-163 -> ['NDB_730', 'NDB_880', 'NDB_446']\n",
      "Chr7-6Mb-seg-164 -> ['NDB_989', 'NDB_641', 'NDB_816']\n",
      "Chr7-6Mb-seg-165 -> ['NDB_1023', 'NDB_828', 'NDB_889']\n",
      "Chr7-6Mb-seg-166 -> ['NDB_784', 'NDB_533', 'NDB_801']\n",
      "Chr7-6Mb-seg-167 -> ['NDB_971', 'NDB_738', 'NDB_400']\n",
      "Chr7-6Mb-seg-168 -> ['NDB_768', 'NDB_568', 'NDB_458']\n",
      "Chr7-6Mb-seg-169 -> ['NDB_901', 'NDB_716', 'NDB_786']\n",
      "Chr7-6Mb-seg-170 -> ['NDB_443', 'NDB_657', 'NDB_754']\n",
      "Chr7-6Mb-seg-171 -> ['NDB_438', 'NDB_1033', 'NDB_461']\n",
      "Chr7-6Mb-seg-172 -> ['NDB_616', 'NDB_866', 'NDB_921']\n",
      "Chr7-6Mb-seg-173 -> ['NDB_347', 'NDB_1047', 'NDB_835']\n",
      "Chr7-6Mb-seg-174 -> ['NDB_813', 'NDB_505', 'NDB_941']\n",
      "Chr7-6Mb-seg-175 -> ['NDB_994', 'NDB_356', 'NDB_1066']\n",
      "Chr7-6Mb-seg-176 -> ['NDB_755', 'NDB_725', 'NDB_690']\n",
      "Chr7-6Mb-seg-177 -> ['NDB_699', 'NDB_750', 'NDB_778']\n",
      "Chr7-6Mb-seg-178 -> ['NDB_844', 'NDB_466', 'NDB_746']\n",
      "Chr7-6Mb-seg-179 -> ['NDB_875', 'NDB_860', 'NDB_957']\n",
      "Chr7-6Mb-seg-180 -> ['NDB_465', 'NDB_1020', 'NDB_391']\n",
      "Chr7-6Mb-seg-181 -> ['NDB_634', 'NDB_631', 'NDB_398']\n",
      "Chr7-6Mb-seg-182 -> ['NDB_905', 'NDB_887', 'NDB_780']\n",
      "Chr7-6Mb-seg-183 -> ['NDB_615', 'NDB_846', 'NDB_397']\n",
      "Chr7-6Mb-seg-184 -> ['NDB_946', 'NDB_385', 'NDB_647']\n",
      "Chr7-6Mb-seg-185 -> ['NDB_396', 'NDB_445', 'NDB_935']\n",
      "Chr7-6Mb-seg-186 -> ['NDB_826', 'NDB_578', 'NDB_710']\n",
      "Chr7-6Mb-seg-187 -> ['NDB_743', 'NDB_936', 'NDB_825']\n",
      "Chr7-6Mb-seg-188 -> ['NDB_852', 'NDB_961', 'NDB_511']\n",
      "Chr7-6Mb-seg-189 -> ['NDB_487', 'NDB_773', 'NDB_680']\n",
      "Chr7-6Mb-seg-190 -> ['NDB_455', 'NDB_489', 'NDB_681']\n",
      "Chr7-6Mb-seg-191 -> ['NDB_405', 'NDB_715', 'NDB_532']\n",
      "Chr7-6Mb-seg-192 -> ['NDB_1075', 'NDB_467', 'NDB_506']\n",
      "Chr7-6Mb-seg-193 -> ['NDB_752', 'NDB_507', 'NDB_864']\n",
      "Chr7-6Mb-seg-194 -> ['NDB_630', 'NDB_721', 'NDB_838']\n",
      "Chr7-6Mb-seg-195 -> ['NDB_665', 'NDB_734', 'NDB_788']\n",
      "Chr7-6Mb-seg-196 -> ['NDB_777', 'NDB_807', 'NDB_652']\n",
      "Chr7-6Mb-seg-197 -> ['NDB_760', 'NDB_790', 'NDB_464']\n",
      "Chr7-6Mb-seg-198 -> ['NDB_728', 'NDB_974', 'NDB_387']\n",
      "Chr7-6Mb-seg-199 -> ['NDB_432', 'NDB_858', 'NDB_868']\n",
      "Chr7-6Mb-seg-200 -> ['NDB_1027', 'NDB_856', 'NDB_863']\n",
      "Chr7-6Mb-seg-201 -> ['NDB_536', 'NDB_653', 'NDB_771']\n",
      "Chr7-6Mb-seg-202 -> ['NDB_390', 'NDB_501', 'NDB_619']\n",
      "Chr7-6Mb-seg-203 -> ['NDB_832', 'NDB_433', 'NDB_539']\n",
      "Chr7-6Mb-seg-204 -> ['NDB_899', 'NDB_452', 'NDB_618']\n",
      "Chr7-6Mb-seg-205 -> ['NDB_356', 'NDB_706', 'NDB_711']\n",
      "Chr7-6Mb-seg-206 -> ['NDB_795', 'NDB_801', 'NDB_718']\n",
      "Chr7-6Mb-seg-207 -> ['NDB_643', 'NDB_400', 'NDB_431']\n",
      "Chr7-6Mb-seg-208 -> ['NDB_917', 'NDB_458', 'NDB_621']\n",
      "Chr7-6Mb-seg-209 -> ['NDB_516', 'NDB_786', 'NDB_529']\n",
      "Chr7-6Mb-seg-210 -> ['NDB_403', 'NDB_754', 'NDB_878']\n",
      "Chr7-6Mb-seg-211 -> ['NDB_758', 'NDB_461', 'NDB_741']\n",
      "Chr7-6Mb-seg-212 -> ['NDB_789', 'NDB_921', 'NDB_805']\n",
      "Chr7-6Mb-seg-213 -> ['NDB_880', 'NDB_835', 'NDB_896']\n",
      "Chr7-6Mb-seg-214 -> ['NDB_641', 'NDB_941', 'NDB_687']\n",
      "Chr7-6Mb-seg-215 -> ['NDB_994', 'NDB_881', 'NDB_514']\n",
      "Chr7-6Mb-seg-216 -> ['NDB_759', 'NDB_826', 'NDB_690']\n",
      "Chr7-6Mb-seg-217 -> ['NDB_847', 'NDB_743', 'NDB_778']\n",
      "Chr7-6Mb-seg-218 -> ['NDB_554', 'NDB_852', 'NDB_746']\n",
      "Chr7-6Mb-seg-219 -> ['NDB_375', 'NDB_487', 'NDB_957']\n",
      "Chr7-6Mb-seg-220 -> ['NDB_508', 'NDB_455', 'NDB_391']\n",
      "Chr7-6Mb-seg-221 -> ['NDB_332', 'NDB_405', 'NDB_398']\n",
      "Chr7-6Mb-seg-222 -> ['NDB_444', 'NDB_1075', 'NDB_780']\n",
      "Chr7-6Mb-seg-223 -> ['NDB_730', 'NDB_752', 'NDB_397']\n",
      "Chr7-6Mb-seg-224 -> ['NDB_989', 'NDB_630', 'NDB_647']\n",
      "Chr7-6Mb-seg-225 -> ['NDB_935', 'NDB_660', 'NDB_889']\n",
      "Chr7-6Mb-seg-226 -> ['NDB_755', 'NDB_865', 'NDB_533']\n",
      "Chr7-6Mb-seg-227 -> ['NDB_699', 'NDB_815', 'NDB_738']\n",
      "Chr7-6Mb-seg-228 -> ['NDB_844', 'NDB_804', 'NDB_568']\n",
      "Chr7-6Mb-seg-229 -> ['NDB_875', 'NDB_571', 'NDB_716']\n",
      "Chr7-6Mb-seg-230 -> ['NDB_465', 'NDB_737', 'NDB_657']\n",
      "Chr7-6Mb-seg-231 -> ['NDB_634', 'NDB_663', 'NDB_1033']\n",
      "Chr7-6Mb-seg-232 -> ['NDB_905', 'NDB_1060', 'NDB_866']\n",
      "Chr7-6Mb-seg-233 -> ['NDB_615', 'NDB_446', 'NDB_1047']\n",
      "Chr7-6Mb-seg-234 -> ['NDB_946', 'NDB_816', 'NDB_505']\n",
      "Chr7-6Mb-seg-235 -> ['NDB_637', 'NDB_445', 'NDB_711']\n",
      "Chr7-6Mb-seg-236 -> ['NDB_578', 'NDB_795', 'NDB_817']\n",
      "Chr7-6Mb-seg-237 -> ['NDB_936', 'NDB_643', 'NDB_965']\n",
      "Chr7-6Mb-seg-238 -> ['NDB_961', 'NDB_917', 'NDB_915']\n",
      "Chr7-6Mb-seg-239 -> ['NDB_773', 'NDB_516', 'NDB_448']\n",
      "Chr7-6Mb-seg-240 -> ['NDB_489', 'NDB_403', 'NDB_698']\n",
      "Chr7-6Mb-seg-241 -> ['NDB_715', 'NDB_758', 'NDB_954']\n",
      "Chr7-6Mb-seg-242 -> ['NDB_467', 'NDB_789', 'NDB_427']\n",
      "Chr7-6Mb-seg-243 -> ['NDB_507', 'NDB_880', 'NDB_473']\n",
      "Chr7-6Mb-seg-244 -> ['NDB_721', 'NDB_641', 'NDB_1044']\n",
      "Chr7-6Mb-seg-245 -> ['NDB_1023', 'NDB_734', 'NDB_706']\n",
      "Chr7-6Mb-seg-246 -> ['NDB_826', 'NDB_713', 'NDB_718']\n",
      "Chr7-6Mb-seg-247 -> ['NDB_743', 'NDB_456', 'NDB_431']\n",
      "Chr7-6Mb-seg-248 -> ['NDB_852', 'NDB_661', 'NDB_621']\n",
      "Chr7-6Mb-seg-249 -> ['NDB_487', 'NDB_812', 'NDB_529']\n",
      "Chr7-6Mb-seg-250 -> ['NDB_455', 'NDB_417', 'NDB_878']\n",
      "Chr7-6Mb-seg-251 -> ['NDB_405', 'NDB_883', 'NDB_741']\n",
      "Chr7-6Mb-seg-252 -> ['NDB_1075', 'NDB_335', 'NDB_805']\n",
      "Chr7-6Mb-seg-253 -> ['NDB_752', 'NDB_747', 'NDB_896']\n",
      "Chr7-6Mb-seg-254 -> ['NDB_630', 'NDB_853', 'NDB_687']\n",
      "Chr7-6Mb-seg-255 -> ['NDB_498', 'NDB_1066', 'NDB_828']\n",
      "Chr7-6Mb-seg-256 -> ['NDB_759', 'NDB_652', 'NDB_710']\n",
      "Chr7-6Mb-seg-257 -> ['NDB_847', 'NDB_464', 'NDB_825']\n",
      "Chr7-6Mb-seg-258 -> ['NDB_554', 'NDB_387', 'NDB_511']\n",
      "Chr7-6Mb-seg-259 -> ['NDB_375', 'NDB_868', 'NDB_680']\n",
      "Chr7-6Mb-seg-260 -> ['NDB_508', 'NDB_863', 'NDB_681']\n",
      "Chr7-6Mb-seg-261 -> ['NDB_332', 'NDB_771', 'NDB_532']\n",
      "Chr7-6Mb-seg-262 -> ['NDB_444', 'NDB_619', 'NDB_506']\n",
      "Chr7-6Mb-seg-263 -> ['NDB_730', 'NDB_539', 'NDB_864']\n",
      "Chr7-6Mb-seg-264 -> ['NDB_989', 'NDB_618', 'NDB_838']\n",
      "Chr7-6Mb-seg-265 -> ['NDB_665', 'NDB_637', 'NDB_660']\n",
      "Chr7-6Mb-seg-266 -> ['NDB_777', 'NDB_817', 'NDB_725']\n",
      "Chr7-6Mb-seg-267 -> ['NDB_760', 'NDB_965', 'NDB_750']\n",
      "Chr7-6Mb-seg-268 -> ['NDB_728', 'NDB_915', 'NDB_466']\n",
      "Chr7-6Mb-seg-269 -> ['NDB_432', 'NDB_448', 'NDB_860']\n",
      "Chr7-6Mb-seg-270 -> ['NDB_1027', 'NDB_698', 'NDB_1020']\n",
      "Chr7-6Mb-seg-271 -> ['NDB_536', 'NDB_954', 'NDB_631']\n",
      "Chr7-6Mb-seg-272 -> ['NDB_390', 'NDB_427', 'NDB_887']\n",
      "Chr7-6Mb-seg-273 -> ['NDB_832', 'NDB_473', 'NDB_846']\n",
      "Chr7-6Mb-seg-274 -> ['NDB_899', 'NDB_1044', 'NDB_385']\n",
      "Chr7-6Mb-seg-275 -> ['NDB_881', 'NDB_396', 'NDB_788']\n",
      "Chr7-6Mb-seg-276 -> ['NDB_784', 'NDB_578', 'NDB_376']\n",
      "Chr7-6Mb-seg-277 -> ['NDB_971', 'NDB_936', 'NDB_683']\n",
      "Chr7-6Mb-seg-278 -> ['NDB_768', 'NDB_961', 'NDB_762']\n",
      "Chr7-6Mb-seg-279 -> ['NDB_901', 'NDB_773', 'NDB_409']\n",
      "Chr7-6Mb-seg-280 -> ['NDB_443', 'NDB_489', 'NDB_470']\n",
      "Chr7-6Mb-seg-281 -> ['NDB_438', 'NDB_715', 'NDB_1035']\n",
      "Chr7-6Mb-seg-282 -> ['NDB_616', 'NDB_467', 'NDB_388']\n",
      "Chr7-6Mb-seg-283 -> ['NDB_347', 'NDB_507', 'NDB_482']\n",
      "Chr7-6Mb-seg-284 -> ['NDB_813', 'NDB_721', 'NDB_453']\n",
      "Chr7-6Mb-seg-285 -> ['NDB_514', 'NDB_445', 'NDB_706']\n",
      "Chr7-6Mb-seg-286 -> ['NDB_713', 'NDB_801', 'NDB_690']\n",
      "Chr7-6Mb-seg-287 -> ['NDB_456', 'NDB_400', 'NDB_778']\n",
      "Chr7-6Mb-seg-288 -> ['NDB_661', 'NDB_458', 'NDB_746']\n",
      "Chr7-6Mb-seg-289 -> ['NDB_812', 'NDB_786', 'NDB_957']\n",
      "Chr7-6Mb-seg-290 -> ['NDB_417', 'NDB_754', 'NDB_391']\n",
      "Chr7-6Mb-seg-291 -> ['NDB_883', 'NDB_461', 'NDB_398']\n",
      "Chr7-6Mb-seg-292 -> ['NDB_335', 'NDB_921', 'NDB_780']\n",
      "Chr7-6Mb-seg-293 -> ['NDB_747', 'NDB_835', 'NDB_397']\n",
      "Chr7-6Mb-seg-294 -> ['NDB_853', 'NDB_941', 'NDB_647']\n",
      "Chr7-6Mb-seg-295 -> ['NDB_1066', 'NDB_734', 'NDB_711']\n",
      "Chr7-6Mb-seg-296 -> ['NDB_826', 'NDB_817', 'NDB_652']\n",
      "Chr7-6Mb-seg-297 -> ['NDB_743', 'NDB_965', 'NDB_464']\n",
      "Chr7-6Mb-seg-298 -> ['NDB_852', 'NDB_915', 'NDB_387']\n",
      "Chr7-6Mb-seg-299 -> ['NDB_487', 'NDB_448', 'NDB_868']\n",
      "Chr7-6Mb-seg-300 -> ['NDB_455', 'NDB_698', 'NDB_863']\n",
      "Chr7-6Mb-seg-301 -> ['NDB_405', 'NDB_954', 'NDB_771']\n",
      "Chr7-6Mb-seg-302 -> ['NDB_1075', 'NDB_427', 'NDB_619']\n",
      "Chr7-6Mb-seg-303 -> ['NDB_752', 'NDB_473', 'NDB_539']\n",
      "Chr7-6Mb-seg-304 -> ['NDB_630', 'NDB_1044', 'NDB_618']\n",
      "Chr7-6Mb-seg-305 -> ['NDB_994', 'NDB_1023', 'NDB_935']\n",
      "Chr7-6Mb-seg-306 -> ['NDB_795', 'NDB_807', 'NDB_710']\n",
      "Chr7-6Mb-seg-307 -> ['NDB_643', 'NDB_790', 'NDB_825']\n",
      "Chr7-6Mb-seg-308 -> ['NDB_917', 'NDB_974', 'NDB_511']\n",
      "Chr7-6Mb-seg-309 -> ['NDB_516', 'NDB_858', 'NDB_680']\n",
      "Chr7-6Mb-seg-310 -> ['NDB_403', 'NDB_856', 'NDB_681']\n",
      "Chr7-6Mb-seg-311 -> ['NDB_758', 'NDB_653', 'NDB_532']\n",
      "Chr7-6Mb-seg-312 -> ['NDB_789', 'NDB_501', 'NDB_506']\n",
      "Chr7-6Mb-seg-313 -> ['NDB_880', 'NDB_433', 'NDB_864']\n",
      "Chr7-6Mb-seg-314 -> ['NDB_641', 'NDB_452', 'NDB_838']\n",
      "Chr7-6Mb-seg-315 -> ['NDB_356', 'NDB_637', 'NDB_889']\n",
      "Chr7-6Mb-seg-316 -> ['NDB_759', 'NDB_713', 'NDB_725']\n",
      "Chr7-6Mb-seg-317 -> ['NDB_847', 'NDB_456', 'NDB_750']\n",
      "Chr7-6Mb-seg-318 -> ['NDB_554', 'NDB_661', 'NDB_466']\n",
      "Chr7-6Mb-seg-319 -> ['NDB_375', 'NDB_812', 'NDB_860']\n",
      "Chr7-6Mb-seg-320 -> ['NDB_508', 'NDB_417', 'NDB_1020']\n",
      "Chr7-6Mb-seg-321 -> ['NDB_332', 'NDB_883', 'NDB_631']\n",
      "Chr7-6Mb-seg-322 -> ['NDB_444', 'NDB_335', 'NDB_887']\n",
      "Chr7-6Mb-seg-323 -> ['NDB_730', 'NDB_747', 'NDB_846']\n",
      "Chr7-6Mb-seg-324 -> ['NDB_989', 'NDB_853', 'NDB_385']\n",
      "Chr7-6Mb-seg-325 -> ['NDB_498', 'NDB_445', 'NDB_660']\n",
      "Chr7-6Mb-seg-326 -> ['NDB_784', 'NDB_777', 'NDB_865']\n",
      "Chr7-6Mb-seg-327 -> ['NDB_971', 'NDB_760', 'NDB_815']\n",
      "Chr7-6Mb-seg-328 -> ['NDB_768', 'NDB_728', 'NDB_804']\n",
      "Chr7-6Mb-seg-329 -> ['NDB_901', 'NDB_432', 'NDB_571']\n",
      "Chr7-6Mb-seg-330 -> ['NDB_443', 'NDB_1027', 'NDB_737']\n",
      "Chr7-6Mb-seg-331 -> ['NDB_438', 'NDB_536', 'NDB_663']\n",
      "Chr7-6Mb-seg-332 -> ['NDB_616', 'NDB_390', 'NDB_1060']\n",
      "Chr7-6Mb-seg-333 -> ['NDB_347', 'NDB_832', 'NDB_446']\n",
      "Chr7-6Mb-seg-334 -> ['NDB_813', 'NDB_899', 'NDB_816']\n",
      "Chr7-6Mb-seg-335 -> ['NDB_665', 'NDB_828', 'NDB_711']\n",
      "Chr7-6Mb-seg-336 -> ['NDB_376', 'NDB_533', 'NDB_690']\n",
      "Chr7-6Mb-seg-337 -> ['NDB_683', 'NDB_738', 'NDB_778']\n",
      "Chr7-6Mb-seg-338 -> ['NDB_762', 'NDB_568', 'NDB_746']\n",
      "Chr7-6Mb-seg-339 -> ['NDB_409', 'NDB_716', 'NDB_957']\n",
      "Chr7-6Mb-seg-340 -> ['NDB_470', 'NDB_657', 'NDB_391']\n",
      "Chr7-6Mb-seg-341 -> ['NDB_1035', 'NDB_1033', 'NDB_398']\n",
      "Chr7-6Mb-seg-342 -> ['NDB_388', 'NDB_866', 'NDB_780']\n",
      "Chr7-6Mb-seg-343 -> ['NDB_482', 'NDB_1047', 'NDB_397']\n",
      "Chr7-6Mb-seg-344 -> ['NDB_453', 'NDB_505', 'NDB_647']\n",
      "Chr7-6Mb-seg-345 -> ['NDB_1066', 'NDB_637', 'NDB_788']\n",
      "Chr7-6Mb-seg-346 -> ['NDB_578', 'NDB_801', 'NDB_807']\n",
      "Chr7-6Mb-seg-347 -> ['NDB_936', 'NDB_400', 'NDB_790']\n",
      "Chr7-6Mb-seg-348 -> ['NDB_961', 'NDB_458', 'NDB_974']\n",
      "Chr7-6Mb-seg-349 -> ['NDB_773', 'NDB_786', 'NDB_858']\n",
      "Chr7-6Mb-seg-350 -> ['NDB_489', 'NDB_754', 'NDB_856']\n",
      "Chr7-6Mb-seg-351 -> ['NDB_715', 'NDB_461', 'NDB_653']\n",
      "Chr7-6Mb-seg-352 -> ['NDB_467', 'NDB_921', 'NDB_501']\n",
      "Chr7-6Mb-seg-353 -> ['NDB_507', 'NDB_835', 'NDB_433']\n",
      "Chr7-6Mb-seg-354 -> ['NDB_721', 'NDB_941', 'NDB_452']\n",
      "Chr7-6Mb-seg-355 -> ['NDB_498', 'NDB_396', 'NDB_734']\n",
      "Chr7-6Mb-seg-356 -> ['NDB_755', 'NDB_652', 'NDB_718']\n",
      "Chr7-6Mb-seg-357 -> ['NDB_699', 'NDB_464', 'NDB_431']\n",
      "Chr7-6Mb-seg-358 -> ['NDB_844', 'NDB_387', 'NDB_621']\n",
      "Chr7-6Mb-seg-359 -> ['NDB_875', 'NDB_868', 'NDB_529']\n",
      "Chr7-6Mb-seg-360 -> ['NDB_465', 'NDB_863', 'NDB_878']\n",
      "Chr7-6Mb-seg-361 -> ['NDB_634', 'NDB_771', 'NDB_741']\n",
      "Chr7-6Mb-seg-362 -> ['NDB_905', 'NDB_619', 'NDB_805']\n",
      "Chr7-6Mb-seg-363 -> ['NDB_615', 'NDB_539', 'NDB_896']\n",
      "Chr7-6Mb-seg-364 -> ['NDB_946', 'NDB_618', 'NDB_687']\n",
      "Chr7-6Mb-seg-365 -> ['NDB_881', 'NDB_706', 'NDB_889']\n",
      "Chr7-6Mb-seg-366 -> ['NDB_826', 'NDB_865', 'NDB_725']\n",
      "Chr7-6Mb-seg-367 -> ['NDB_743', 'NDB_815', 'NDB_750']\n",
      "Chr7-6Mb-seg-368 -> ['NDB_852', 'NDB_804', 'NDB_466']\n",
      "Chr7-6Mb-seg-369 -> ['NDB_487', 'NDB_571', 'NDB_860']\n",
      "Chr7-6Mb-seg-370 -> ['NDB_455', 'NDB_737', 'NDB_1020']\n",
      "Chr7-6Mb-seg-371 -> ['NDB_405', 'NDB_663', 'NDB_631']\n",
      "Chr7-6Mb-seg-372 -> ['NDB_1075', 'NDB_1060', 'NDB_887']\n",
      "Chr7-6Mb-seg-373 -> ['NDB_752', 'NDB_446', 'NDB_846']\n",
      "Chr7-6Mb-seg-374 -> ['NDB_630', 'NDB_816', 'NDB_385']\n",
      "Chr7-6Mb-seg-375 -> ['NDB_994', 'NDB_445', 'NDB_788']\n",
      "Chr7-6Mb-seg-376 -> ['NDB_759', 'NDB_817', 'NDB_807']\n",
      "Chr7-6Mb-seg-377 -> ['NDB_847', 'NDB_965', 'NDB_790']\n",
      "Chr7-6Mb-seg-378 -> ['NDB_554', 'NDB_915', 'NDB_974']\n",
      "Chr7-6Mb-seg-379 -> ['NDB_375', 'NDB_448', 'NDB_858']\n",
      "Chr7-6Mb-seg-380 -> ['NDB_508', 'NDB_698', 'NDB_856']\n",
      "Chr7-6Mb-seg-381 -> ['NDB_332', 'NDB_954', 'NDB_653']\n",
      "Chr7-6Mb-seg-382 -> ['NDB_444', 'NDB_427', 'NDB_501']\n",
      "Chr7-6Mb-seg-383 -> ['NDB_730', 'NDB_473', 'NDB_433']\n",
      "Chr7-6Mb-seg-384 -> ['NDB_989', 'NDB_1044', 'NDB_452']\n",
      "Chr7-6Mb-seg-385 -> ['NDB_356', 'NDB_828', 'NDB_660']\n",
      "Chr7-6Mb-seg-386 -> ['NDB_713', 'NDB_533', 'NDB_710']\n",
      "Chr7-6Mb-seg-387 -> ['NDB_456', 'NDB_738', 'NDB_825']\n",
      "Chr7-6Mb-seg-388 -> ['NDB_661', 'NDB_568', 'NDB_511']\n",
      "Chr7-6Mb-seg-389 -> ['NDB_812', 'NDB_716', 'NDB_680']\n",
      "Chr7-6Mb-seg-390 -> ['NDB_417', 'NDB_657', 'NDB_681']\n",
      "Chr7-6Mb-seg-391 -> ['NDB_883', 'NDB_1033', 'NDB_532']\n",
      "Chr7-6Mb-seg-392 -> ['NDB_335', 'NDB_866', 'NDB_506']\n",
      "Chr7-6Mb-seg-393 -> ['NDB_747', 'NDB_1047', 'NDB_864']\n",
      "Chr7-6Mb-seg-394 -> ['NDB_853', 'NDB_505', 'NDB_838']\n",
      "Chr7-6Mb-seg-395 -> ['NDB_514', 'NDB_1023', 'NDB_1066']\n",
      "Chr7-6Mb-seg-396 -> ['NDB_795', 'NDB_777', 'NDB_690']\n",
      "Chr7-6Mb-seg-397 -> ['NDB_643', 'NDB_760', 'NDB_778']\n",
      "Chr7-6Mb-seg-398 -> ['NDB_917', 'NDB_728', 'NDB_746']\n",
      "Chr7-6Mb-seg-399 -> ['NDB_516', 'NDB_432', 'NDB_957']\n",
      "Chr7-6Mb-seg-400 -> ['NDB_403', 'NDB_1027', 'NDB_391']\n",
      "Chr7-6Mb-seg-401 -> ['NDB_758', 'NDB_536', 'NDB_398']\n",
      "Chr7-6Mb-seg-402 -> ['NDB_789', 'NDB_390', 'NDB_780']\n",
      "Chr7-6Mb-seg-403 -> ['NDB_880', 'NDB_832', 'NDB_397']\n",
      "Chr7-6Mb-seg-404 -> ['NDB_641', 'NDB_899', 'NDB_647']\n",
      "Chr7-6Mb-seg-405 -> ['NDB_994', 'NDB_396', 'NDB_637']\n",
      "Chr7-6Mb-seg-406 -> ['NDB_755', 'NDB_759', 'NDB_578']\n",
      "Chr7-6Mb-seg-407 -> ['NDB_699', 'NDB_847', 'NDB_936']\n",
      "Chr7-6Mb-seg-408 -> ['NDB_844', 'NDB_554', 'NDB_961']\n",
      "Chr7-6Mb-seg-409 -> ['NDB_875', 'NDB_375', 'NDB_773']\n",
      "Chr7-6Mb-seg-410 -> ['NDB_465', 'NDB_508', 'NDB_489']\n",
      "Chr7-6Mb-seg-411 -> ['NDB_634', 'NDB_332', 'NDB_715']\n",
      "Chr7-6Mb-seg-412 -> ['NDB_905', 'NDB_444', 'NDB_467']\n",
      "Chr7-6Mb-seg-413 -> ['NDB_615', 'NDB_730', 'NDB_507']\n",
      "Chr7-6Mb-seg-414 -> ['NDB_946', 'NDB_989', 'NDB_721']\n",
      "Chr7-6Mb-seg-415 -> ['NDB_498', 'NDB_665', 'NDB_935']\n",
      "Chr7-6Mb-seg-416 -> ['NDB_865', 'NDB_801', 'NDB_652']\n",
      "Chr7-6Mb-seg-417 -> ['NDB_815', 'NDB_400', 'NDB_464']\n",
      "Chr7-6Mb-seg-418 -> ['NDB_804', 'NDB_458', 'NDB_387']\n",
      "Chr7-6Mb-seg-419 -> ['NDB_571', 'NDB_786', 'NDB_868']\n",
      "Chr7-6Mb-seg-420 -> ['NDB_737', 'NDB_754', 'NDB_863']\n",
      "Chr7-6Mb-seg-421 -> ['NDB_663', 'NDB_461', 'NDB_771']\n",
      "Chr7-6Mb-seg-422 -> ['NDB_1060', 'NDB_921', 'NDB_619']\n",
      "Chr7-6Mb-seg-423 -> ['NDB_446', 'NDB_835', 'NDB_539']\n",
      "Chr7-6Mb-seg-424 -> ['NDB_816', 'NDB_941', 'NDB_618']\n",
      "Chr7-6Mb-seg-425 -> ['NDB_788', 'NDB_711', 'NDB_889']\n",
      "Chr7-6Mb-seg-426 -> ['NDB_784', 'NDB_817', 'NDB_710']\n",
      "Chr7-6Mb-seg-427 -> ['NDB_971', 'NDB_965', 'NDB_825']\n",
      "Chr7-6Mb-seg-428 -> ['NDB_768', 'NDB_915', 'NDB_511']\n",
      "Chr7-6Mb-seg-429 -> ['NDB_901', 'NDB_448', 'NDB_680']\n",
      "Chr7-6Mb-seg-430 -> ['NDB_443', 'NDB_698', 'NDB_681']\n",
      "Chr7-6Mb-seg-431 -> ['NDB_438', 'NDB_954', 'NDB_532']\n",
      "Chr7-6Mb-seg-432 -> ['NDB_616', 'NDB_427', 'NDB_506']\n",
      "Chr7-6Mb-seg-433 -> ['NDB_347', 'NDB_473', 'NDB_864']\n",
      "Chr7-6Mb-seg-434 -> ['NDB_813', 'NDB_1044', 'NDB_838']\n",
      "Chr7-6Mb-seg-435 -> ['NDB_514', 'NDB_734', 'NDB_660']\n",
      "Chr7-6Mb-seg-436 -> ['NDB_755', 'NDB_795', 'NDB_713']\n",
      "Chr7-6Mb-seg-437 -> ['NDB_699', 'NDB_643', 'NDB_456']\n",
      "Chr7-6Mb-seg-438 -> ['NDB_844', 'NDB_917', 'NDB_661']\n",
      "Chr7-6Mb-seg-439 -> ['NDB_875', 'NDB_516', 'NDB_812']\n",
      "Chr7-6Mb-seg-440 -> ['NDB_465', 'NDB_403', 'NDB_417']\n",
      "Chr7-6Mb-seg-441 -> ['NDB_634', 'NDB_758', 'NDB_883']\n",
      "Chr7-6Mb-seg-442 -> ['NDB_905', 'NDB_789', 'NDB_335']\n",
      "Chr7-6Mb-seg-443 -> ['NDB_615', 'NDB_880', 'NDB_747']\n",
      "Chr7-6Mb-seg-444 -> ['NDB_946', 'NDB_641', 'NDB_853']\n",
      "Chr7-6Mb-seg-445 -> ['NDB_881', 'NDB_828', 'NDB_445']\n",
      "Chr7-6Mb-seg-446 -> ['NDB_826', 'NDB_533', 'NDB_807']\n",
      "Chr7-6Mb-seg-447 -> ['NDB_743', 'NDB_738', 'NDB_790']\n",
      "Chr7-6Mb-seg-448 -> ['NDB_852', 'NDB_568', 'NDB_974']\n",
      "Chr7-6Mb-seg-449 -> ['NDB_487', 'NDB_716', 'NDB_858']\n",
      "Chr7-6Mb-seg-450 -> ['NDB_455', 'NDB_657', 'NDB_856']\n",
      "Chr7-6Mb-seg-451 -> ['NDB_405', 'NDB_1033', 'NDB_653']\n",
      "Chr7-6Mb-seg-452 -> ['NDB_1075', 'NDB_866', 'NDB_501']\n",
      "Chr7-6Mb-seg-453 -> ['NDB_752', 'NDB_1047', 'NDB_433']\n",
      "Chr7-6Mb-seg-454 -> ['NDB_630', 'NDB_505', 'NDB_452']\n",
      "Chr7-6Mb-seg-455 -> ['NDB_994', 'NDB_665', 'NDB_706']\n",
      "Chr7-6Mb-seg-456 -> ['NDB_376', 'NDB_725', 'NDB_652']\n",
      "Chr7-6Mb-seg-457 -> ['NDB_683', 'NDB_750', 'NDB_464']\n",
      "Chr7-6Mb-seg-458 -> ['NDB_762', 'NDB_466', 'NDB_387']\n",
      "Chr7-6Mb-seg-459 -> ['NDB_409', 'NDB_860', 'NDB_868']\n",
      "Chr7-6Mb-seg-460 -> ['NDB_470', 'NDB_1020', 'NDB_863']\n",
      "Chr7-6Mb-seg-461 -> ['NDB_1035', 'NDB_631', 'NDB_771']\n",
      "Chr7-6Mb-seg-462 -> ['NDB_388', 'NDB_887', 'NDB_619']\n",
      "Chr7-6Mb-seg-463 -> ['NDB_482', 'NDB_846', 'NDB_539']\n",
      "Chr7-6Mb-seg-464 -> ['NDB_453', 'NDB_385', 'NDB_618']\n",
      "Chr7-6Mb-seg-465 -> ['NDB_1066', 'NDB_396', 'NDB_889']\n",
      "Chr7-6Mb-seg-466 -> ['NDB_710', 'NDB_690', 'NDB_718']\n",
      "Chr7-6Mb-seg-467 -> ['NDB_825', 'NDB_778', 'NDB_431']\n",
      "Chr7-6Mb-seg-468 -> ['NDB_511', 'NDB_746', 'NDB_621']\n",
      "Chr7-6Mb-seg-469 -> ['NDB_680', 'NDB_957', 'NDB_529']\n",
      "Chr7-6Mb-seg-470 -> ['NDB_681', 'NDB_391', 'NDB_878']\n",
      "Chr7-6Mb-seg-471 -> ['NDB_532', 'NDB_398', 'NDB_741']\n",
      "Chr7-6Mb-seg-472 -> ['NDB_506', 'NDB_780', 'NDB_805']\n",
      "Chr7-6Mb-seg-473 -> ['NDB_864', 'NDB_397', 'NDB_896']\n",
      "Chr7-6Mb-seg-474 -> ['NDB_838', 'NDB_647', 'NDB_687']\n",
      "Chr7-6Mb-seg-475 -> ['NDB_514', 'NDB_637', 'NDB_935']\n",
      "Chr7-6Mb-seg-476 -> ['NDB_759', 'NDB_777', 'NDB_533']\n",
      "Chr7-6Mb-seg-477 -> ['NDB_847', 'NDB_760', 'NDB_738']\n",
      "Chr7-6Mb-seg-478 -> ['NDB_554', 'NDB_728', 'NDB_568']\n",
      "Chr7-6Mb-seg-479 -> ['NDB_375', 'NDB_432', 'NDB_716']\n",
      "Chr7-6Mb-seg-480 -> ['NDB_508', 'NDB_1027', 'NDB_657']\n",
      "Chr7-6Mb-seg-481 -> ['NDB_332', 'NDB_536', 'NDB_1033']\n",
      "Chr7-6Mb-seg-482 -> ['NDB_444', 'NDB_390', 'NDB_866']\n",
      "Chr7-6Mb-seg-483 -> ['NDB_730', 'NDB_832', 'NDB_1047']\n",
      "Chr7-6Mb-seg-484 -> ['NDB_989', 'NDB_899', 'NDB_505']\n",
      "Chr7-6Mb-seg-485 -> ['NDB_1023', 'NDB_660', 'NDB_788']\n",
      "Chr7-6Mb-seg-486 -> ['NDB_713', 'NDB_865', 'NDB_807']\n",
      "Chr7-6Mb-seg-487 -> ['NDB_456', 'NDB_815', 'NDB_790']\n",
      "Chr7-6Mb-seg-488 -> ['NDB_661', 'NDB_804', 'NDB_974']\n",
      "Chr7-6Mb-seg-489 -> ['NDB_812', 'NDB_571', 'NDB_858']\n",
      "Chr7-6Mb-seg-490 -> ['NDB_417', 'NDB_737', 'NDB_856']\n",
      "Chr7-6Mb-seg-491 -> ['NDB_883', 'NDB_663', 'NDB_653']\n",
      "Chr7-6Mb-seg-492 -> ['NDB_335', 'NDB_1060', 'NDB_501']\n",
      "Chr7-6Mb-seg-493 -> ['NDB_747', 'NDB_446', 'NDB_433']\n",
      "Chr7-6Mb-seg-494 -> ['NDB_853', 'NDB_816', 'NDB_452']\n",
      "Chr7-6Mb-seg-495 -> ['NDB_881', 'NDB_498', 'NDB_356']\n",
      "Chr7-6Mb-seg-496 -> ['NDB_755', 'NDB_801', 'NDB_817']\n",
      "Chr7-6Mb-seg-497 -> ['NDB_699', 'NDB_400', 'NDB_965']\n",
      "Chr7-6Mb-seg-498 -> ['NDB_844', 'NDB_458', 'NDB_915']\n",
      "Chr7-6Mb-seg-499 -> ['NDB_875', 'NDB_786', 'NDB_448']\n",
      "Chr7-6Mb-seg-500 -> ['NDB_465', 'NDB_754', 'NDB_698']\n",
      "Chr7-6Mb-seg-501 -> ['NDB_634', 'NDB_461', 'NDB_954']\n",
      "Chr7-6Mb-seg-502 -> ['NDB_905', 'NDB_921', 'NDB_427']\n",
      "Chr7-6Mb-seg-503 -> ['NDB_615', 'NDB_835', 'NDB_473']\n",
      "Chr7-6Mb-seg-504 -> ['NDB_946', 'NDB_941', 'NDB_1044']\n",
      "Chr7-6Mb-seg-505 -> ['NDB_734', 'NDB_445', 'NDB_889']\n",
      "Chr7-6Mb-seg-506 -> ['NDB_376', 'NDB_777', 'NDB_718']\n",
      "Chr7-6Mb-seg-507 -> ['NDB_683', 'NDB_760', 'NDB_431']\n",
      "Chr7-6Mb-seg-508 -> ['NDB_762', 'NDB_728', 'NDB_621']\n",
      "Chr7-6Mb-seg-509 -> ['NDB_409', 'NDB_432', 'NDB_529']\n",
      "Chr7-6Mb-seg-510 -> ['NDB_470', 'NDB_1027', 'NDB_878']\n",
      "Chr7-6Mb-seg-511 -> ['NDB_1035', 'NDB_536', 'NDB_741']\n",
      "Chr7-6Mb-seg-512 -> ['NDB_388', 'NDB_390', 'NDB_805']\n",
      "Chr7-6Mb-seg-513 -> ['NDB_482', 'NDB_832', 'NDB_896']\n",
      "Chr7-6Mb-seg-514 -> ['NDB_453', 'NDB_899', 'NDB_687']\n",
      "Chr7-6Mb-seg-515 -> ['NDB_1066', 'NDB_935', 'NDB_706']\n",
      "Chr7-6Mb-seg-516 -> ['NDB_578', 'NDB_652', 'NDB_690']\n",
      "Chr7-6Mb-seg-517 -> ['NDB_936', 'NDB_464', 'NDB_778']\n",
      "Chr7-6Mb-seg-518 -> ['NDB_961', 'NDB_387', 'NDB_746']\n",
      "Chr7-6Mb-seg-519 -> ['NDB_773', 'NDB_868', 'NDB_957']\n",
      "Chr7-6Mb-seg-520 -> ['NDB_489', 'NDB_863', 'NDB_391']\n",
      "Chr7-6Mb-seg-521 -> ['NDB_715', 'NDB_771', 'NDB_398']\n",
      "Chr7-6Mb-seg-522 -> ['NDB_467', 'NDB_619', 'NDB_780']\n",
      "Chr7-6Mb-seg-523 -> ['NDB_507', 'NDB_539', 'NDB_397']\n",
      "Chr7-6Mb-seg-524 -> ['NDB_721', 'NDB_618', 'NDB_647']\n",
      "Chr7-6Mb-seg-525 -> ['NDB_514', 'NDB_396', 'NDB_828']\n",
      "Chr7-6Mb-seg-526 -> ['NDB_784', 'NDB_795', 'NDB_725']\n",
      "Chr7-6Mb-seg-527 -> ['NDB_971', 'NDB_643', 'NDB_750']\n",
      "Chr7-6Mb-seg-528 -> ['NDB_768', 'NDB_917', 'NDB_466']\n",
      "Chr7-6Mb-seg-529 -> ['NDB_901', 'NDB_516', 'NDB_860']\n",
      "Chr7-6Mb-seg-530 -> ['NDB_443', 'NDB_403', 'NDB_1020']\n",
      "Chr7-6Mb-seg-531 -> ['NDB_438', 'NDB_758', 'NDB_631']\n",
      "Chr7-6Mb-seg-532 -> ['NDB_616', 'NDB_789', 'NDB_887']\n",
      "Chr7-6Mb-seg-533 -> ['NDB_347', 'NDB_880', 'NDB_846']\n",
      "Chr7-6Mb-seg-534 -> ['NDB_813', 'NDB_641', 'NDB_385']\n",
      "Chr7-6Mb-seg-535 -> ['NDB_994', 'NDB_660', 'NDB_711']\n",
      "Chr7-6Mb-seg-536 -> ['NDB_755', 'NDB_376', 'NDB_807']\n",
      "Chr7-6Mb-seg-537 -> ['NDB_699', 'NDB_683', 'NDB_790']\n",
      "Chr7-6Mb-seg-538 -> ['NDB_844', 'NDB_762', 'NDB_974']\n",
      "Chr7-6Mb-seg-539 -> ['NDB_875', 'NDB_409', 'NDB_858']\n",
      "Chr7-6Mb-seg-540 -> ['NDB_465', 'NDB_470', 'NDB_856']\n",
      "Chr7-6Mb-seg-541 -> ['NDB_634', 'NDB_1035', 'NDB_653']\n",
      "Chr7-6Mb-seg-542 -> ['NDB_905', 'NDB_388', 'NDB_501']\n",
      "Chr7-6Mb-seg-543 -> ['NDB_615', 'NDB_482', 'NDB_433']\n",
      "Chr7-6Mb-seg-544 -> ['NDB_946', 'NDB_453', 'NDB_452']\n",
      "Chr7-6Mb-seg-545 -> ['NDB_356', 'NDB_1023', 'NDB_445']\n",
      "Chr7-6Mb-seg-546 -> ['NDB_826', 'NDB_777', 'NDB_801']\n",
      "Chr7-6Mb-seg-547 -> ['NDB_743', 'NDB_760', 'NDB_400']\n",
      "Chr7-6Mb-seg-548 -> ['NDB_852', 'NDB_728', 'NDB_458']\n",
      "Chr7-6Mb-seg-549 -> ['NDB_487', 'NDB_432', 'NDB_786']\n",
      "Chr7-6Mb-seg-550 -> ['NDB_455', 'NDB_1027', 'NDB_754']\n",
      "Chr7-6Mb-seg-551 -> ['NDB_405', 'NDB_536', 'NDB_461']\n",
      "Chr7-6Mb-seg-552 -> ['NDB_1075', 'NDB_390', 'NDB_921']\n",
      "Chr7-6Mb-seg-553 -> ['NDB_752', 'NDB_832', 'NDB_835']\n",
      "Chr7-6Mb-seg-554 -> ['NDB_630', 'NDB_899', 'NDB_941']\n",
      "Chr7-6Mb-seg-555 -> ['NDB_881', 'NDB_637', 'NDB_734']\n",
      "Chr7-6Mb-seg-556 -> ['NDB_533', 'NDB_817', 'NDB_718']\n",
      "Chr7-6Mb-seg-557 -> ['NDB_738', 'NDB_965', 'NDB_431']\n",
      "Chr7-6Mb-seg-558 -> ['NDB_568', 'NDB_915', 'NDB_621']\n",
      "Chr7-6Mb-seg-559 -> ['NDB_716', 'NDB_448', 'NDB_529']\n",
      "Chr7-6Mb-seg-560 -> ['NDB_657', 'NDB_698', 'NDB_878']\n",
      "Chr7-6Mb-seg-561 -> ['NDB_1033', 'NDB_954', 'NDB_741']\n",
      "Chr7-6Mb-seg-562 -> ['NDB_866', 'NDB_427', 'NDB_805']\n",
      "Chr7-6Mb-seg-563 -> ['NDB_1047', 'NDB_473', 'NDB_896']\n",
      "Chr7-6Mb-seg-564 -> ['NDB_505', 'NDB_1044', 'NDB_687']\n",
      "Chr7-6Mb-seg-565 -> ['NDB_694', 'NDB_879', 'NDB_705']\n",
      "Chr7-6Mb-seg-566 -> ['NDB_929', 'NDB_472', 'NDB_622']\n",
      "Chr7-6Mb-seg-567 -> ['NDB_513', 'NDB_359', 'NDB_581']\n",
      "Chr7-6Mb-seg-568 -> ['NDB_910', 'NDB_606', 'NDB_702']\n",
      "Chr7-6Mb-seg-569 -> ['NDB_926', 'NDB_538', 'NDB_1036']\n",
      "Chr7-6Mb-seg-570 -> ['NDB_654', 'NDB_791', 'NDB_776']\n",
      "Chr7-6Mb-seg-571 -> ['NDB_928', 'NDB_645', 'NDB_732']\n",
      "Chr7-6Mb-seg-572 -> ['NDB_806', 'NDB_1051', 'NDB_922']\n",
      "Chr7-6Mb-seg-573 -> ['NDB_480', 'NDB_708', 'NDB_700']\n",
      "Chr7-6Mb-seg-574 -> ['NDB_604', 'NDB_931', 'NDB_908']\n",
      "Chr7-6Mb-seg-575 -> ['NDB_587', 'NDB_862', 'NDB_977']\n",
      "Chr7-6Mb-seg-576 -> ['NDB_594', 'NDB_611', 'NDB_840']\n",
      "Chr7-6Mb-seg-577 -> ['NDB_517', 'NDB_519', 'NDB_577']\n",
      "Chr7-6Mb-seg-578 -> ['NDB_350', 'NDB_556', 'NDB_494']\n",
      "Chr7-6Mb-seg-579 -> ['NDB_591', 'NDB_479', 'NDB_1056']\n",
      "Chr7-6Mb-seg-580 -> ['NDB_589', 'NDB_486', 'NDB_904']\n",
      "Chr7-6Mb-seg-581 -> ['NDB_629', 'NDB_601', 'NDB_488']\n",
      "Chr7-6Mb-seg-582 -> ['NDB_960', 'NDB_416', 'NDB_429']\n",
      "Chr7-6Mb-seg-583 -> ['NDB_460', 'NDB_635', 'NDB_500']\n",
      "Chr7-6Mb-seg-584 -> ['NDB_593', 'NDB_474', 'NDB_471']\n",
      "Chr7-6Mb-seg-585 -> ['NDB_551', 'NDB_628', 'NDB_557']\n",
      "Chr7-6Mb-seg-586 -> ['NDB_1053', 'NDB_341', 'NDB_372']\n",
      "Chr7-6Mb-seg-587 -> ['NDB_544', 'NDB_903', 'NDB_925']\n",
      "Chr7-6Mb-seg-588 -> ['NDB_371', 'NDB_934', 'NDB_476']\n",
      "Chr7-6Mb-seg-589 -> ['NDB_483', 'NDB_608', 'NDB_495']\n",
      "Chr7-6Mb-seg-590 -> ['NDB_703', 'NDB_492', 'NDB_1063']\n",
      "Chr7-6Mb-seg-591 -> ['NDB_785', 'NDB_1048', 'NDB_920']\n",
      "Chr7-6Mb-seg-592 -> ['NDB_561', 'NDB_503', 'NDB_876']\n",
      "Chr7-6Mb-seg-593 -> ['NDB_592', 'NDB_1013', 'NDB_895']\n",
      "Chr7-6Mb-seg-594 -> ['NDB_845', 'NDB_537', 'NDB_449']\n",
      "Chr7-6Mb-seg-595 -> ['NDB_491', 'NDB_779', 'NDB_379']\n",
      "Chr7-6Mb-seg-596 -> ['NDB_627', 'NDB_933', 'NDB_386']\n",
      "Chr7-6Mb-seg-597 -> ['NDB_712', 'NDB_625', 'NDB_441']\n",
      "Chr7-6Mb-seg-598 -> ['NDB_428', 'NDB_818', 'NDB_439']\n",
      "Chr7-6Mb-seg-599 -> ['NDB_894', 'NDB_510', 'NDB_809']\n",
      "Chr7-6Mb-seg-600 -> ['NDB_955', 'NDB_1042', 'NDB_666']\n",
      "Chr7-6Mb-seg-601 -> ['NDB_362', 'NDB_902', 'NDB_976']\n",
      "Chr7-6Mb-seg-602 -> ['NDB_423', 'NDB_393', 'NDB_383']\n",
      "Chr7-6Mb-seg-603 -> ['NDB_783', 'NDB_573', 'NDB_871']\n",
      "Chr7-6Mb-seg-604 -> ['NDB_829', 'NDB_436', 'NDB_689']\n",
      "Chr7-6Mb-seg-605 -> ['NDB_1014', 'NDB_459', 'NDB_421']\n",
      "Chr7-6Mb-seg-606 -> ['NDB_958', 'NDB_418', 'NDB_836']\n",
      "Chr7-6Mb-seg-607 -> ['NDB_392', 'NDB_614', 'NDB_450']\n",
      "Chr7-6Mb-seg-608 -> ['NDB_939', 'NDB_798', 'NDB_481']\n",
      "Chr7-6Mb-seg-609 -> ['NDB_547', 'NDB_736', 'NDB_872']\n",
      "Chr7-6Mb-seg-610 -> ['NDB_644', 'NDB_368', 'NDB_693']\n",
      "Chr7-6Mb-seg-611 -> ['NDB_522', 'NDB_729', 'NDB_649']\n",
      "Chr7-6Mb-seg-612 -> ['NDB_1039', 'NDB_562', 'NDB_413']\n",
      "Chr7-6Mb-seg-613 -> ['NDB_772', 'NDB_995', 'NDB_774']\n",
      "Chr7-6Mb-seg-614 -> ['NDB_869', 'NDB_381', 'NDB_496']\n",
      "Chr7-6Mb-seg-615 -> ['NDB_879', 'NDB_751', 'NDB_977']\n",
      "Chr7-6Mb-seg-616 -> ['NDB_472', 'NDB_1028', 'NDB_840']\n",
      "Chr7-6Mb-seg-617 -> ['NDB_359', 'NDB_384', 'NDB_577']\n",
      "Chr7-6Mb-seg-618 -> ['NDB_606', 'NDB_775', 'NDB_494']\n",
      "Chr7-6Mb-seg-619 -> ['NDB_538', 'NDB_638', 'NDB_1056']\n",
      "Chr7-6Mb-seg-620 -> ['NDB_791', 'NDB_948', 'NDB_904']\n",
      "Chr7-6Mb-seg-621 -> ['NDB_645', 'NDB_691', 'NDB_488']\n",
      "Chr7-6Mb-seg-622 -> ['NDB_1051', 'NDB_650', 'NDB_429']\n",
      "Chr7-6Mb-seg-623 -> ['NDB_635', 'NDB_1013', 'NDB_480']\n",
      "Chr7-6Mb-seg-624 -> ['NDB_474', 'NDB_537', 'NDB_604']\n",
      "Chr7-6Mb-seg-625 -> ['NDB_587', 'NDB_819', 'NDB_551']\n",
      "Chr7-6Mb-seg-626 -> ['NDB_594', 'NDB_742', 'NDB_1053']\n",
      "Chr7-6Mb-seg-627 -> ['NDB_517', 'NDB_824', 'NDB_544']\n",
      "Chr7-6Mb-seg-628 -> ['NDB_350', 'NDB_402', 'NDB_371']\n",
      "Chr7-6Mb-seg-629 -> ['NDB_591', 'NDB_493', 'NDB_483']\n",
      "Chr7-6Mb-seg-630 -> ['NDB_589', 'NDB_563', 'NDB_703']\n",
      "Chr7-6Mb-seg-631 -> ['NDB_629', 'NDB_891', 'NDB_785']\n",
      "Chr7-6Mb-seg-632 -> ['NDB_960', 'NDB_1072', 'NDB_561']\n",
      "Chr7-6Mb-seg-633 -> ['NDB_407', 'NDB_914', 'NDB_765']\n",
      "Chr7-6Mb-seg-634 -> ['NDB_648', 'NDB_867', 'NDB_739']\n",
      "Chr7-6Mb-seg-635 -> ['NDB_862', 'NDB_557', 'NDB_694']\n",
      "Chr7-6Mb-seg-638 -> ['NDB_611', 'NDB_372', 'NDB_929']\n",
      "Chr7-6Mb-seg-639 -> ['NDB_519', 'NDB_925', 'NDB_513']\n",
      "Chr7-6Mb-seg-640 -> ['NDB_556', 'NDB_476', 'NDB_910']\n",
      "Chr7-6Mb-seg-641 -> ['NDB_479', 'NDB_495', 'NDB_926']\n",
      "Chr7-6Mb-seg-642 -> ['NDB_486', 'NDB_1063', 'NDB_654']\n",
      "Chr7-6Mb-seg-643 -> ['NDB_601', 'NDB_920', 'NDB_928']\n",
      "Chr7-6Mb-seg-644 -> ['NDB_416', 'NDB_876', 'NDB_806']\n",
      "Chr7-6Mb-seg-647 -> ['NDB_783', 'NDB_700', 'NDB_500']\n",
      "Chr7-6Mb-seg-648 -> ['NDB_829', 'NDB_908', 'NDB_471']\n",
      "Chr7-6Mb-seg-649 -> ['NDB_628', 'NDB_951', 'NDB_401']\n",
      "Chr7-6Mb-seg-650 -> ['NDB_341', 'NDB_526', 'NDB_735']\n",
      "Chr7-6Mb-seg-651 -> ['NDB_903', 'NDB_395', 'NDB_940']\n",
      "Chr7-6Mb-seg-655 -> ['NDB_934', 'NDB_585', 'NDB_365']\n",
      "Chr7-6Mb-seg-660 -> ['NDB_608', 'NDB_709', 'NDB_600']\n",
      "Chr7-6Mb-seg-661 -> ['NDB_492', 'NDB_707', 'NDB_727']\n",
      "Chr7-6Mb-seg-662 -> ['NDB_1048', 'NDB_873', 'NDB_329']\n",
      "Chr7-6Mb-seg-663 -> ['NDB_503', 'NDB_796', 'NDB_882']\n",
      "Chr7-6Mb-seg-664 -> ['NDB_460', 'NDB_1013', 'NDB_802']\n",
      "Chr7-6Mb-seg-665 -> ['NDB_593', 'NDB_537', 'NDB_740']\n",
      "Chr7-6Mb-seg-668 -> ['NDB_1014', 'NDB_491', 'NDB_977']\n",
      "Chr7-6Mb-seg-669 -> ['NDB_958', 'NDB_627', 'NDB_840']\n",
      "Chr7-6Mb-seg-670 -> ['NDB_392', 'NDB_712', 'NDB_577']\n",
      "Chr7-6Mb-seg-671 -> ['NDB_939', 'NDB_428', 'NDB_494']\n",
      "Chr7-6Mb-seg-672 -> ['NDB_547', 'NDB_894', 'NDB_1056']\n",
      "Chr7-6Mb-seg-674 -> ['NDB_644', 'NDB_955', 'NDB_904']\n",
      "Chr7-6Mb-seg-675 -> ['NDB_522', 'NDB_362', 'NDB_488']\n",
      "Chr7-6Mb-seg-676 -> ['NDB_1039', 'NDB_423', 'NDB_429']\n",
      "Chr7-6Mb-seg-677 -> ['NDB_573', 'NDB_480', 'NDB_765']\n",
      "Chr7-6Mb-seg-679 -> ['NDB_436', 'NDB_604', 'NDB_739']\n",
      "Chr7-6Mb-seg-680 -> ['NDB_819', 'NDB_421', 'NDB_705']\n",
      "Chr7-6Mb-seg-681 -> ['NDB_742', 'NDB_836', 'NDB_622']\n",
      "Chr7-6Mb-seg-682 -> ['NDB_824', 'NDB_450', 'NDB_581']\n",
      "Chr7-6Mb-seg-683 -> ['NDB_402', 'NDB_481', 'NDB_702']\n",
      "Chr7-6Mb-seg-692 -> ['NDB_493', 'NDB_872', 'NDB_1036']\n",
      "Chr7-6Mb-seg-693 -> ['NDB_563', 'NDB_693', 'NDB_776']\n",
      "Chr7-6Mb-seg-698 -> ['NDB_891', 'NDB_649', 'NDB_732']\n",
      "Chr7-6Mb-seg-699 -> ['NDB_1072', 'NDB_413', 'NDB_922']\n",
      "Chr7-6Mb-seg-700 -> ['NDB_871', 'NDB_774', 'NDB_700']\n",
      "Chr7-6Mb-seg-701 -> ['NDB_689', 'NDB_496', 'NDB_908']\n",
      "Chr7-6Mb-seg-703 -> ['NDB_587', 'NDB_751', 'NDB_379']\n",
      "Chr7-6Mb-seg-704 -> ['NDB_594', 'NDB_1028', 'NDB_386']\n",
      "Chr7-6Mb-seg-705 -> ['NDB_517', 'NDB_384', 'NDB_441']\n",
      "Chr7-6Mb-seg-706 -> ['NDB_350', 'NDB_775', 'NDB_439']\n",
      "Chr7-6Mb-seg-707 -> ['NDB_591', 'NDB_638', 'NDB_809']\n",
      "Chr7-6Mb-seg-708 -> ['NDB_589', 'NDB_948', 'NDB_666']\n",
      "Chr7-6Mb-seg-710 -> ['NDB_629', 'NDB_691', 'NDB_976']\n",
      "Chr7-6Mb-seg-711 -> ['NDB_960', 'NDB_650', 'NDB_383']\n",
      "Chr7-6Mb-seg-712 -> ['NDB_635', 'NDB_995', 'NDB_914']\n",
      "Chr7-6Mb-seg-713 -> ['NDB_474', 'NDB_381', 'NDB_867']\n",
      "Chr7-6Mb-seg-717 -> ['NDB_1014', 'NDB_951', 'NDB_879']\n",
      "Chr7-6Mb-seg-718 -> ['NDB_958', 'NDB_526', 'NDB_472']\n",
      "Chr7-6Mb-seg-719 -> ['NDB_392', 'NDB_395', 'NDB_359']\n",
      "Chr7-6Mb-seg-720 -> ['NDB_939', 'NDB_585', 'NDB_606']\n",
      "Chr7-6Mb-seg-721 -> ['NDB_547', 'NDB_709', 'NDB_538']\n",
      "Chr7-6Mb-seg-722 -> ['NDB_644', 'NDB_707', 'NDB_791']\n",
      "Chr7-6Mb-seg-723 -> ['NDB_522', 'NDB_873', 'NDB_645']\n",
      "Chr7-6Mb-seg-726 -> ['NDB_1039', 'NDB_796', 'NDB_1051']\n",
      "Chr7-6Mb-seg-727 -> ['NDB_407', 'NDB_592', 'NDB_708']\n",
      "Chr7-6Mb-seg-728 -> ['NDB_648', 'NDB_845', 'NDB_931']\n",
      "Chr7-6Mb-seg-731 -> ['NDB_557', 'NDB_459', 'NDB_779']\n",
      "Chr7-6Mb-seg-732 -> ['NDB_372', 'NDB_418', 'NDB_933']\n",
      "Chr7-6Mb-seg-733 -> ['NDB_925', 'NDB_614', 'NDB_625']\n",
      "Chr7-6Mb-seg-734 -> ['NDB_476', 'NDB_798', 'NDB_818']\n",
      "Chr7-6Mb-seg-735 -> ['NDB_495', 'NDB_736', 'NDB_510']\n",
      "Chr7-6Mb-seg-738 -> ['NDB_1063', 'NDB_368', 'NDB_1042']\n",
      "Chr7-6Mb-seg-739 -> ['NDB_920', 'NDB_729', 'NDB_902']\n",
      "Chr7-6Mb-seg-740 -> ['NDB_876', 'NDB_562', 'NDB_393']\n",
      "Chr7-6Mb-seg-741 -> ['NDB_783', 'NDB_460', 'NDB_765']\n",
      "Chr7-6Mb-seg-742 -> ['NDB_829', 'NDB_593', 'NDB_739']\n",
      "Chr7-6Mb-seg-743 -> ['NDB_862', 'NDB_628', 'NDB_491']\n",
      "Chr7-6Mb-seg-744 -> ['NDB_611', 'NDB_341', 'NDB_627']\n",
      "Chr7-6Mb-seg-745 -> ['NDB_519', 'NDB_903', 'NDB_712']\n",
      "Chr7-6Mb-seg-746 -> ['NDB_556', 'NDB_934', 'NDB_428']\n",
      "Chr7-6Mb-seg-747 -> ['NDB_479', 'NDB_608', 'NDB_894']\n",
      "Chr7-6Mb-seg-748 -> ['NDB_486', 'NDB_492', 'NDB_955']\n",
      "Chr7-6Mb-seg-749 -> ['NDB_601', 'NDB_1048', 'NDB_362']\n",
      "Chr7-6Mb-seg-750 -> ['NDB_416', 'NDB_503', 'NDB_423']\n",
      "Chr7-6Mb-seg-751 -> ['NDB_573', 'NDB_774', 'NDB_802']\n",
      "Chr7-6Mb-seg-752 -> ['NDB_436', 'NDB_496', 'NDB_740']\n",
      "Chr7-6Mb-seg-753 -> ['NDB_587', 'NDB_951', 'NDB_694']\n",
      "Chr7-6Mb-seg-754 -> ['NDB_594', 'NDB_526', 'NDB_929']\n",
      "Chr7-6Mb-seg-755 -> ['NDB_517', 'NDB_395', 'NDB_513']\n",
      "Chr7-6Mb-seg-756 -> ['NDB_350', 'NDB_585', 'NDB_910']\n",
      "Chr7-6Mb-seg-757 -> ['NDB_591', 'NDB_709', 'NDB_926']\n",
      "Chr7-6Mb-seg-758 -> ['NDB_589', 'NDB_707', 'NDB_654']\n",
      "Chr7-6Mb-seg-760 -> ['NDB_629', 'NDB_873', 'NDB_928']\n",
      "Chr7-6Mb-seg-762 -> ['NDB_960', 'NDB_796', 'NDB_806']\n",
      "Chr7-6Mb-seg-763 -> ['NDB_772', 'NDB_635', 'NDB_592']\n",
      "Chr7-6Mb-seg-764 -> ['NDB_869', 'NDB_474', 'NDB_845']\n",
      "Chr7-6Mb-seg-765 -> ['NDB_819', 'NDB_879', 'NDB_379']\n",
      "Chr7-6Mb-seg-766 -> ['NDB_742', 'NDB_472', 'NDB_386']\n",
      "Chr7-6Mb-seg-767 -> ['NDB_824', 'NDB_359', 'NDB_441']\n",
      "Chr7-6Mb-seg-768 -> ['NDB_402', 'NDB_606', 'NDB_439']\n",
      "Chr7-6Mb-seg-769 -> ['NDB_493', 'NDB_538', 'NDB_809']\n",
      "Chr7-6Mb-seg-770 -> ['NDB_563', 'NDB_791', 'NDB_666']\n",
      "Chr7-6Mb-seg-771 -> ['NDB_891', 'NDB_645', 'NDB_976']\n",
      "Chr7-6Mb-seg-772 -> ['NDB_1072', 'NDB_1051', 'NDB_383']\n",
      "Chr7-6Mb-seg-773 -> ['NDB_480', 'NDB_895', 'NDB_914']\n",
      "Chr7-6Mb-seg-775 -> ['NDB_604', 'NDB_449', 'NDB_867']\n",
      "Chr7-6Mb-seg-776 -> ['NDB_551', 'NDB_1014', 'NDB_705']\n",
      "Chr7-6Mb-seg-777 -> ['NDB_1053', 'NDB_958', 'NDB_622']\n",
      "Chr7-6Mb-seg-778 -> ['NDB_544', 'NDB_392', 'NDB_581']\n",
      "Chr7-6Mb-seg-780 -> ['NDB_371', 'NDB_939', 'NDB_702']\n",
      "Chr7-6Mb-seg-781 -> ['NDB_483', 'NDB_547', 'NDB_1036']\n",
      "Chr7-6Mb-seg-784 -> ['NDB_703', 'NDB_644', 'NDB_776']\n",
      "Chr7-6Mb-seg-785 -> ['NDB_785', 'NDB_522', 'NDB_732']\n",
      "Chr7-6Mb-seg-786 -> ['NDB_561', 'NDB_1039', 'NDB_922']\n",
      "Chr7-6Mb-seg-787 -> ['NDB_1013', 'NDB_995', 'NDB_500']\n",
      "Chr7-6Mb-seg-788 -> ['NDB_537', 'NDB_381', 'NDB_471']\n",
      "Chr7-6Mb-seg-790 -> ['NDB_421', 'NDB_751', 'NDB_401']\n",
      "Chr7-6Mb-seg-791 -> ['NDB_836', 'NDB_1028', 'NDB_735']\n",
      "Chr7-6Mb-seg-792 -> ['NDB_450', 'NDB_384', 'NDB_940']\n",
      "Chr7-6Mb-seg-793 -> ['NDB_481', 'NDB_775', 'NDB_365']\n",
      "Chr7-6Mb-seg-794 -> ['NDB_872', 'NDB_638', 'NDB_600']\n",
      "Chr7-6Mb-seg-795 -> ['NDB_693', 'NDB_948', 'NDB_727']\n",
      "Chr7-6Mb-seg-796 -> ['NDB_649', 'NDB_691', 'NDB_329']\n",
      "Chr7-6Mb-seg-797 -> ['NDB_413', 'NDB_650', 'NDB_882']\n",
      "Chr7-6Mb-seg-798 -> ['NDB_635', 'NDB_700', 'NDB_765']\n",
      "Chr7-6Mb-seg-799 -> ['NDB_474', 'NDB_908', 'NDB_739']\n",
      "Chr7-6Mb-seg-800 -> ['NDB_628', 'NDB_694', 'NDB_977']\n",
      "Chr7-6Mb-seg-801 -> ['NDB_341', 'NDB_929', 'NDB_840']\n",
      "Chr7-6Mb-seg-802 -> ['NDB_903', 'NDB_513', 'NDB_577']\n",
      "Chr7-6Mb-seg-804 -> ['NDB_934', 'NDB_910', 'NDB_494']\n",
      "Chr7-6Mb-seg-805 -> ['NDB_608', 'NDB_926', 'NDB_1056']\n",
      "Chr7-6Mb-seg-806 -> ['NDB_492', 'NDB_654', 'NDB_904']\n",
      "Chr7-6Mb-seg-807 -> ['NDB_1048', 'NDB_928', 'NDB_488']\n",
      "Chr7-6Mb-seg-808 -> ['NDB_503', 'NDB_806', 'NDB_429']\n",
      "Chr7-6Mb-seg-809 -> ['NDB_772', 'NDB_407', 'NDB_460']\n",
      "Chr7-6Mb-seg-810 -> ['NDB_869', 'NDB_648', 'NDB_593']\n",
      "Chr7-6Mb-seg-812 -> ['NDB_862', 'NDB_551', 'NDB_379']\n",
      "Chr7-6Mb-seg-813 -> ['NDB_611', 'NDB_1053', 'NDB_386']\n",
      "Chr7-6Mb-seg-814 -> ['NDB_519', 'NDB_544', 'NDB_441']\n",
      "Chr7-6Mb-seg-816 -> ['NDB_556', 'NDB_371', 'NDB_439']\n",
      "Chr7-6Mb-seg-817 -> ['NDB_479', 'NDB_483', 'NDB_809']\n",
      "Chr7-6Mb-seg-818 -> ['NDB_486', 'NDB_703', 'NDB_666']\n",
      "Chr7-6Mb-seg-819 -> ['NDB_601', 'NDB_785', 'NDB_976']\n",
      "Chr7-6Mb-seg-820 -> ['NDB_416', 'NDB_561', 'NDB_383']\n",
      "Chr7-6Mb-seg-821 -> ['NDB_914', 'NDB_708', 'NDB_802']\n",
      "Chr7-6Mb-seg-822 -> ['NDB_867', 'NDB_931', 'NDB_740']\n",
      "Chr7-6Mb-seg-823 -> ['NDB_819', 'NDB_491', 'NDB_951']\n",
      "Chr7-6Mb-seg-824 -> ['NDB_742', 'NDB_627', 'NDB_526']\n",
      "Chr7-6Mb-seg-825 -> ['NDB_824', 'NDB_712', 'NDB_395']\n",
      "Chr7-6Mb-seg-828 -> ['NDB_402', 'NDB_428', 'NDB_585']\n",
      "Chr7-6Mb-seg-829 -> ['NDB_493', 'NDB_894', 'NDB_709']\n",
      "Chr7-6Mb-seg-831 -> ['NDB_563', 'NDB_955', 'NDB_707']\n",
      "Chr7-6Mb-seg-832 -> ['NDB_891', 'NDB_362', 'NDB_873']\n",
      "Chr7-6Mb-seg-833 -> ['NDB_1072', 'NDB_423', 'NDB_796']\n",
      "Chr7-6Mb-seg-834 -> ['NDB_871', 'NDB_895', 'NDB_765']\n",
      "Chr7-6Mb-seg-835 -> ['NDB_689', 'NDB_449', 'NDB_739']\n",
      "Chr7-6Mb-seg-836 -> ['NDB_1014', 'NDB_628', 'NDB_779']\n",
      "Chr7-6Mb-seg-837 -> ['NDB_958', 'NDB_341', 'NDB_933']\n",
      "Chr7-6Mb-seg-840 -> ['NDB_392', 'NDB_903', 'NDB_625']\n",
      "Chr7-6Mb-seg-841 -> ['NDB_939', 'NDB_934', 'NDB_818']\n",
      "Chr7-6Mb-seg-842 -> ['NDB_547', 'NDB_608', 'NDB_510']\n",
      "Chr7-6Mb-seg-845 -> ['NDB_644', 'NDB_492', 'NDB_1042']\n",
      "Chr7-6Mb-seg-846 -> ['NDB_522', 'NDB_1048', 'NDB_902']\n",
      "Chr7-6Mb-seg-847 -> ['NDB_1039', 'NDB_503', 'NDB_393']\n",
      "Chr7-6Mb-seg-848 -> ['NDB_573', 'NDB_995', 'NDB_700']\n",
      "Chr7-6Mb-seg-849 -> ['NDB_436', 'NDB_381', 'NDB_908']\n",
      "Chr7-6Mb-seg-850 -> ['NDB_551', 'NDB_459', 'NDB_977']\n",
      "Chr7-6Mb-seg-852 -> ['NDB_1053', 'NDB_418', 'NDB_840']\n",
      "Chr7-6Mb-seg-853 -> ['NDB_544', 'NDB_614', 'NDB_577']\n",
      "Chr7-6Mb-seg-854 -> ['NDB_371', 'NDB_798', 'NDB_494']\n",
      "Chr7-6Mb-seg-855 -> ['NDB_483', 'NDB_736', 'NDB_1056']\n",
      "Chr7-6Mb-seg-856 -> ['NDB_703', 'NDB_368', 'NDB_904']\n",
      "Chr7-6Mb-seg-857 -> ['NDB_785', 'NDB_729', 'NDB_488']\n",
      "Chr7-6Mb-seg-858 -> ['NDB_561', 'NDB_562', 'NDB_429']\n",
      "Chr7-6Mb-seg-859 -> ['NDB_783', 'NDB_592', 'NDB_774']\n",
      "Chr7-6Mb-seg-860 -> ['NDB_829', 'NDB_845', 'NDB_496']\n",
      "Chr7-6Mb-seg-862 -> ['NDB_862', 'NDB_401', 'NDB_705']\n",
      "Chr7-6Mb-seg-863 -> ['NDB_611', 'NDB_735', 'NDB_622']\n",
      "Chr7-6Mb-seg-864 -> ['NDB_519', 'NDB_940', 'NDB_581']\n",
      "Chr7-6Mb-seg-865 -> ['NDB_556', 'NDB_365', 'NDB_702']\n",
      "Chr7-6Mb-seg-866 -> ['NDB_479', 'NDB_600', 'NDB_1036']\n",
      "Chr7-6Mb-seg-868 -> ['NDB_486', 'NDB_727', 'NDB_776']\n",
      "Chr7-6Mb-seg-869 -> ['NDB_601', 'NDB_329', 'NDB_732']\n",
      "Chr7-6Mb-seg-870 -> ['NDB_416', 'NDB_882', 'NDB_922']\n",
      "Chr7-6Mb-seg-873 -> ['NDB_1013', 'NDB_871', 'NDB_708']\n",
      "Chr7-6Mb-seg-876 -> ['NDB_537', 'NDB_689', 'NDB_931']\n",
      "Chr7-6Mb-seg-878 -> ['NDB_421', 'NDB_779', 'NDB_879']\n",
      "Chr7-6Mb-seg-879 -> ['NDB_836', 'NDB_933', 'NDB_472']\n",
      "Chr7-6Mb-seg-880 -> ['NDB_450', 'NDB_625', 'NDB_359']\n",
      "Chr7-6Mb-seg-881 -> ['NDB_481', 'NDB_818', 'NDB_606']\n",
      "Chr7-6Mb-seg-883 -> ['NDB_872', 'NDB_510', 'NDB_538']\n",
      "Chr7-6Mb-seg-885 -> ['NDB_693', 'NDB_1042', 'NDB_791']\n",
      "Chr7-6Mb-seg-888 -> ['NDB_649', 'NDB_902', 'NDB_645']\n",
      "Chr7-6Mb-seg-889 -> ['NDB_413', 'NDB_393', 'NDB_1051']\n",
      "Chr7-6Mb-seg-890 -> ['NDB_407', 'NDB_480', 'NDB_500']\n",
      "Chr7-6Mb-seg-891 -> ['NDB_648', 'NDB_604', 'NDB_471']\n",
      "Chr7-6Mb-seg-892 -> ['NDB_587', 'NDB_1014', 'NDB_557']\n",
      "Chr7-6Mb-seg-893 -> ['NDB_594', 'NDB_958', 'NDB_372']\n",
      "Chr7-6Mb-seg-894 -> ['NDB_517', 'NDB_392', 'NDB_925']\n",
      "Chr7-6Mb-seg-895 -> ['NDB_350', 'NDB_939', 'NDB_476']\n",
      "Chr7-6Mb-seg-896 -> ['NDB_591', 'NDB_547', 'NDB_495']\n",
      "Chr7-6Mb-seg-897 -> ['NDB_589', 'NDB_644', 'NDB_1063']\n",
      "Chr7-6Mb-seg-900 -> ['NDB_629', 'NDB_522', 'NDB_920']\n",
      "Chr7-6Mb-seg-901 -> ['NDB_960', 'NDB_1039', 'NDB_876']\n",
      "Chr7-6Mb-seg-902 -> ['NDB_460', 'NDB_895', 'NDB_700']\n",
      "Chr7-6Mb-seg-904 -> ['NDB_593', 'NDB_449', 'NDB_908']\n",
      "Chr7-6Mb-seg-906 -> ['NDB_459', 'NDB_694', 'NDB_379']\n",
      "Chr7-6Mb-seg-907 -> ['NDB_418', 'NDB_929', 'NDB_386']\n",
      "Chr7-6Mb-seg-911 -> ['NDB_614', 'NDB_513', 'NDB_441']\n",
      "Chr7-6Mb-seg-912 -> ['NDB_798', 'NDB_910', 'NDB_439']\n",
      "Chr7-6Mb-seg-913 -> ['NDB_736', 'NDB_926', 'NDB_809']\n",
      "Chr7-6Mb-seg-916 -> ['NDB_368', 'NDB_654', 'NDB_666']\n",
      "Chr7-6Mb-seg-917 -> ['NDB_729', 'NDB_928', 'NDB_976']\n",
      "Chr7-6Mb-seg-918 -> ['NDB_562', 'NDB_806', 'NDB_383']\n",
      "Chr7-6Mb-seg-919 -> ['NDB_592', 'NDB_995', 'NDB_765']\n",
      "Chr7-6Mb-seg-922 -> ['NDB_845', 'NDB_381', 'NDB_739']\n",
      "Chr7-6Mb-seg-923 -> ['NDB_551', 'NDB_779', 'NDB_401']\n",
      "Chr7-6Mb-seg-924 -> ['NDB_1053', 'NDB_933', 'NDB_735']\n",
      "Chr7-6Mb-seg-926 -> ['NDB_544', 'NDB_625', 'NDB_940']\n",
      "Chr7-6Mb-seg-929 -> ['NDB_371', 'NDB_818', 'NDB_365']\n",
      "Chr7-6Mb-seg-931 -> ['NDB_483', 'NDB_510', 'NDB_600']\n",
      "Chr7-6Mb-seg-932 -> ['NDB_703', 'NDB_1042', 'NDB_727']\n",
      "Chr7-6Mb-seg-933 -> ['NDB_785', 'NDB_902', 'NDB_329']\n",
      "Chr7-6Mb-seg-934 -> ['NDB_561', 'NDB_393', 'NDB_882']\n",
      "Chr7-6Mb-seg-935 -> ['NDB_772', 'NDB_573', 'NDB_914']\n",
      "Chr7-6Mb-seg-937 -> ['NDB_869', 'NDB_436', 'NDB_867']\n",
      "Chr7-6Mb-seg-938 -> ['NDB_628', 'NDB_751', 'NDB_705']\n",
      "Chr7-6Mb-seg-939 -> ['NDB_341', 'NDB_1028', 'NDB_622']\n",
      "Chr7-6Mb-seg-942 -> ['NDB_903', 'NDB_384', 'NDB_581']\n",
      "Chr7-6Mb-seg-944 -> ['NDB_934', 'NDB_775', 'NDB_702']\n",
      "Chr7-6Mb-seg-945 -> ['NDB_608', 'NDB_638', 'NDB_1036']\n",
      "Chr7-6Mb-seg-946 -> ['NDB_492', 'NDB_948', 'NDB_776']\n",
      "Chr7-6Mb-seg-947 -> ['NDB_1048', 'NDB_691', 'NDB_732']\n",
      "Chr7-6Mb-seg-950 -> ['NDB_503', 'NDB_650', 'NDB_922']\n",
      "Chr7-6Mb-seg-951 -> ['NDB_635', 'NDB_871', 'NDB_802']\n",
      "Chr7-6Mb-seg-952 -> ['NDB_474', 'NDB_689', 'NDB_740']\n",
      "Chr7-6Mb-seg-953 -> ['NDB_862', 'NDB_459', 'NDB_879']\n",
      "Chr7-6Mb-seg-954 -> ['NDB_611', 'NDB_418', 'NDB_472']\n",
      "Chr7-6Mb-seg-955 -> ['NDB_519', 'NDB_614', 'NDB_359']\n",
      "Chr7-6Mb-seg-956 -> ['NDB_556', 'NDB_798', 'NDB_606']\n",
      "Chr7-6Mb-seg-957 -> ['NDB_479', 'NDB_736', 'NDB_538']\n",
      "Chr7-6Mb-seg-958 -> ['NDB_486', 'NDB_368', 'NDB_791']\n",
      "Chr7-6Mb-seg-959 -> ['NDB_601', 'NDB_729', 'NDB_645']\n",
      "Chr7-6Mb-seg-960 -> ['NDB_416', 'NDB_562', 'NDB_1051']\n",
      "Chr7-6Mb-seg-961 -> ['NDB_783', 'NDB_895', 'NDB_708']\n",
      "Chr7-6Mb-seg-962 -> ['NDB_829', 'NDB_449', 'NDB_931']\n",
      "Chr7-6Mb-seg-963 -> ['NDB_587', 'NDB_421', 'NDB_491']\n",
      "Chr7-6Mb-seg-964 -> ['NDB_594', 'NDB_836', 'NDB_627']\n",
      "Chr7-6Mb-seg-965 -> ['NDB_517', 'NDB_450', 'NDB_712']\n",
      "Chr7-6Mb-seg-966 -> ['NDB_350', 'NDB_481', 'NDB_428']\n",
      "Chr7-6Mb-seg-967 -> ['NDB_591', 'NDB_872', 'NDB_894']\n",
      "Chr7-6Mb-seg-968 -> ['NDB_589', 'NDB_693', 'NDB_955']\n",
      "Chr7-6Mb-seg-969 -> ['NDB_629', 'NDB_649', 'NDB_362']\n",
      "Chr7-6Mb-seg-970 -> ['NDB_960', 'NDB_413', 'NDB_423']\n",
      "Chr7-6Mb-seg-971 -> ['NDB_1013', 'NDB_774', 'NDB_765']\n",
      "Chr7-6Mb-seg-972 -> ['NDB_537', 'NDB_496', 'NDB_739']\n",
      "Chr7-6Mb-seg-973 -> ['NDB_557', 'NDB_951', 'NDB_379']\n",
      "Chr7-6Mb-seg-974 -> ['NDB_372', 'NDB_526', 'NDB_386']\n",
      "Chr7-6Mb-seg-975 -> ['NDB_925', 'NDB_395', 'NDB_441']\n",
      "Chr7-6Mb-seg-976 -> ['NDB_476', 'NDB_585', 'NDB_439']\n",
      "Chr7-6Mb-seg-977 -> ['NDB_495', 'NDB_709', 'NDB_809']\n",
      "Chr7-6Mb-seg-978 -> ['NDB_1063', 'NDB_707', 'NDB_666']\n",
      "Chr7-6Mb-seg-979 -> ['NDB_920', 'NDB_873', 'NDB_976']\n",
      "Chr7-6Mb-seg-980 -> ['NDB_876', 'NDB_796', 'NDB_383']\n",
      "Chr7-6Mb-seg-981 -> ['NDB_592', 'NDB_871', 'NDB_500']\n",
      "Chr7-6Mb-seg-982 -> ['NDB_845', 'NDB_689', 'NDB_471']\n",
      "Chr7-6Mb-seg-983 -> ['NDB_1014', 'NDB_694', 'NDB_751']\n",
      "Chr7-6Mb-seg-984 -> ['NDB_958', 'NDB_929', 'NDB_1028']\n",
      "Chr7-6Mb-seg-985 -> ['NDB_392', 'NDB_513', 'NDB_384']\n",
      "Chr7-6Mb-seg-986 -> ['NDB_939', 'NDB_910', 'NDB_775']\n",
      "Chr7-6Mb-seg-987 -> ['NDB_547', 'NDB_926', 'NDB_638']\n",
      "Chr7-6Mb-seg-988 -> ['NDB_644', 'NDB_654', 'NDB_948']\n",
      "Chr7-6Mb-seg-989 -> ['NDB_522', 'NDB_928', 'NDB_691']\n",
      "Chr7-6Mb-seg-990 -> ['NDB_1039', 'NDB_806', 'NDB_650']\n",
      "Chr7-6Mb-seg-991 -> ['NDB_783', 'NDB_480', 'NDB_995']\n",
      "Chr7-6Mb-seg-992 -> ['NDB_829', 'NDB_604', 'NDB_381']\n",
      "Chr7-6Mb-seg-993 -> ['NDB_819', 'NDB_401', 'NDB_977']\n",
      "Chr7-6Mb-seg-994 -> ['NDB_742', 'NDB_735', 'NDB_840']\n",
      "Chr7-6Mb-seg-995 -> ['NDB_824', 'NDB_940', 'NDB_577']\n",
      "Chr7-6Mb-seg-996 -> ['NDB_402', 'NDB_365', 'NDB_494']\n",
      "Chr7-6Mb-seg-997 -> ['NDB_493', 'NDB_600', 'NDB_1056']\n",
      "Chr7-6Mb-seg-998 -> ['NDB_563', 'NDB_727', 'NDB_904']\n",
      "Chr7-6Mb-seg-999 -> ['NDB_891', 'NDB_329', 'NDB_488']\n",
      "Chr7-6Mb-seg-1000 -> ['NDB_1072', 'NDB_882', 'NDB_429']\n",
      "Chr7-6Mb-seg-1001 -> ['NDB_407', 'NDB_700', 'NDB_802']\n",
      "Chr7-6Mb-seg-1002 -> ['NDB_648', 'NDB_908', 'NDB_740']\n",
      "Chr7-6Mb-seg-1003 -> ['NDB_551', 'NDB_491', 'NDB_879']\n",
      "Chr7-6Mb-seg-1004 -> ['NDB_1053', 'NDB_627', 'NDB_472']\n",
      "Chr7-6Mb-seg-1005 -> ['NDB_544', 'NDB_712', 'NDB_359']\n",
      "Chr7-6Mb-seg-1006 -> ['NDB_371', 'NDB_428', 'NDB_606']\n",
      "Chr7-6Mb-seg-1007 -> ['NDB_483', 'NDB_894', 'NDB_538']\n",
      "Chr7-6Mb-seg-1008 -> ['NDB_703', 'NDB_955', 'NDB_791']\n",
      "Chr7-6Mb-seg-1009 -> ['NDB_785', 'NDB_362', 'NDB_645']\n",
      "Chr7-6Mb-seg-1010 -> ['NDB_561', 'NDB_423', 'NDB_1051']\n",
      "Chr7-6Mb-seg-1011 -> ['NDB_772', 'NDB_895', 'NDB_500']\n",
      "Chr7-6Mb-seg-1012 -> ['NDB_869', 'NDB_449', 'NDB_471']\n",
      "Chr7-6Mb-seg-1013 -> ['NDB_862', 'NDB_779', 'NDB_751']\n",
      "Chr7-6Mb-seg-1014 -> ['NDB_611', 'NDB_933', 'NDB_1028']\n",
      "Chr7-6Mb-seg-1015 -> ['NDB_519', 'NDB_625', 'NDB_384']\n",
      "Chr7-6Mb-seg-1016 -> ['NDB_556', 'NDB_818', 'NDB_775']\n",
      "Chr7-6Mb-seg-1017 -> ['NDB_479', 'NDB_510', 'NDB_638']\n",
      "Chr7-6Mb-seg-1018 -> ['NDB_486', 'NDB_1042', 'NDB_948']\n",
      "Chr7-6Mb-seg-1019 -> ['NDB_601', 'NDB_902', 'NDB_691']\n",
      "Chr7-6Mb-seg-1020 -> ['NDB_416', 'NDB_393', 'NDB_650']\n",
      "Chr7-6Mb-seg-1021 -> ['NDB_635', 'NDB_774', 'NDB_708']\n",
      "Chr7-6Mb-seg-1022 -> ['NDB_474', 'NDB_496', 'NDB_931']\n",
      "Chr7-6Mb-seg-1023 -> ['NDB_459', 'NDB_951', 'NDB_705']\n",
      "Chr7-6Mb-seg-1024 -> ['NDB_418', 'NDB_526', 'NDB_622']\n",
      "Chr7-6Mb-seg-1025 -> ['NDB_614', 'NDB_395', 'NDB_581']\n",
      "Chr7-6Mb-seg-1026 -> ['NDB_798', 'NDB_585', 'NDB_702']\n",
      "Chr7-6Mb-seg-1027 -> ['NDB_736', 'NDB_709', 'NDB_1036']\n",
      "Chr7-6Mb-seg-1028 -> ['NDB_368', 'NDB_707', 'NDB_776']\n",
      "Chr7-6Mb-seg-1029 -> ['NDB_729', 'NDB_873', 'NDB_732']\n",
      "Chr7-6Mb-seg-1030 -> ['NDB_562', 'NDB_796', 'NDB_922']\n",
      "Chr7-6Mb-seg-1031 -> ['NDB_460', 'NDB_573', 'NDB_592']\n",
      "Chr7-6Mb-seg-1032 -> ['NDB_593', 'NDB_436', 'NDB_845']\n",
      "Chr7-6Mb-seg-1033 -> ['NDB_628', 'NDB_421', 'NDB_379']\n",
      "Chr7-6Mb-seg-1034 -> ['NDB_341', 'NDB_836', 'NDB_386']\n",
      "Chr7-6Mb-seg-1035 -> ['NDB_903', 'NDB_450', 'NDB_441']\n",
      "Chr7-6Mb-seg-1036 -> ['NDB_934', 'NDB_481', 'NDB_439']\n",
      "Chr7-6Mb-seg-1037 -> ['NDB_608', 'NDB_872', 'NDB_809']\n",
      "Chr7-6Mb-seg-1038 -> ['NDB_492', 'NDB_693', 'NDB_666']\n",
      "Chr7-6Mb-seg-1039 -> ['NDB_1048', 'NDB_649', 'NDB_976']\n",
      "Chr7-6Mb-seg-1040 -> ['NDB_503', 'NDB_413', 'NDB_383']\n",
      "Chr7-6Mb-seg-1041 -> ['NDB_772', 'NDB_480', 'NDB_871']\n",
      "Chr7-6Mb-seg-1042 -> ['NDB_869', 'NDB_604', 'NDB_689']\n",
      "Chr7-6Mb-seg-1043 -> ['NDB_819', 'NDB_862', 'NDB_1014']\n",
      "Chr7-6Mb-seg-1044 -> ['NDB_742', 'NDB_611', 'NDB_958']\n",
      "Chr7-6Mb-seg-1045 -> ['NDB_824', 'NDB_519', 'NDB_392']\n",
      "Chr7-6Mb-seg-1046 -> ['NDB_402', 'NDB_556', 'NDB_939']\n",
      "Chr7-6Mb-seg-1047 -> ['NDB_493', 'NDB_479', 'NDB_547']\n",
      "Chr7-6Mb-seg-1048 -> ['NDB_563', 'NDB_486', 'NDB_644']\n",
      "Chr7-6Mb-seg-1049 -> ['NDB_891', 'NDB_601', 'NDB_522']\n",
      "Chr7-6Mb-seg-1050 -> ['NDB_1072', 'NDB_416', 'NDB_1039']\n",
      "Chr7-6Mb-seg-1051 -> ['NDB_783', 'NDB_1013', 'NDB_914']\n",
      "Chr7-6Mb-seg-1052 -> ['NDB_829', 'NDB_537', 'NDB_867']\n",
      "Chr7-6Mb-seg-1053 -> ['NDB_491', 'NDB_694', 'NDB_401']\n",
      "Chr7-6Mb-seg-1054 -> ['NDB_627', 'NDB_929', 'NDB_735']\n",
      "Chr7-6Mb-seg-1055 -> ['NDB_712', 'NDB_513', 'NDB_940']\n",
      "Chr7-6Mb-seg-1056 -> ['NDB_428', 'NDB_910', 'NDB_365']\n",
      "Chr7-6Mb-seg-1057 -> ['NDB_894', 'NDB_926', 'NDB_600']\n",
      "Chr7-6Mb-seg-1058 -> ['NDB_955', 'NDB_654', 'NDB_727']\n",
      "Chr7-6Mb-seg-1059 -> ['NDB_362', 'NDB_928', 'NDB_329']\n",
      "Chr7-6Mb-seg-1060 -> ['NDB_423', 'NDB_806', 'NDB_882']\n",
      "Chr7-6Mb-seg-1061 -> ['NDB_500', 'NDB_765', 'NDB_802']\n",
      "Chr7-6Mb-seg-1062 -> ['NDB_471', 'NDB_739', 'NDB_740']\n",
      "Chr7-6Mb-seg-1063 -> ['NDB_587', 'NDB_779', 'NDB_705']\n",
      "Chr7-6Mb-seg-1064 -> ['NDB_594', 'NDB_933', 'NDB_622']\n",
      "Chr7-6Mb-seg-1065 -> ['NDB_517', 'NDB_625', 'NDB_581']\n",
      "Chr7-6Mb-seg-1066 -> ['NDB_350', 'NDB_818', 'NDB_702']\n",
      "Chr7-6Mb-seg-1067 -> ['NDB_591', 'NDB_510', 'NDB_1036']\n",
      "Chr7-6Mb-seg-1068 -> ['NDB_589', 'NDB_1042', 'NDB_776']\n",
      "Chr7-6Mb-seg-1069 -> ['NDB_629', 'NDB_902', 'NDB_732']\n",
      "Chr7-6Mb-seg-1070 -> ['NDB_960', 'NDB_393', 'NDB_922']\n",
      "Chr7-6Mb-seg-1071 -> ['NDB_460', 'NDB_995', 'NDB_708']\n",
      "Chr7-6Mb-seg-1072 -> ['NDB_593', 'NDB_381', 'NDB_931']\n",
      "Chr7-6Mb-seg-1073 -> ['NDB_819', 'NDB_628', 'NDB_459']\n",
      "Chr7-6Mb-seg-1074 -> ['NDB_742', 'NDB_341', 'NDB_418']\n",
      "Chr7-6Mb-seg-1075 -> ['NDB_824', 'NDB_903', 'NDB_614']\n",
      "Chr7-6Mb-seg-1076 -> ['NDB_402', 'NDB_934', 'NDB_798']\n",
      "Chr7-6Mb-seg-1077 -> ['NDB_493', 'NDB_608', 'NDB_736']\n",
      "Chr7-6Mb-seg-1078 -> ['NDB_563', 'NDB_492', 'NDB_368']\n",
      "Chr7-6Mb-seg-1079 -> ['NDB_891', 'NDB_1048', 'NDB_729']\n",
      "Chr7-6Mb-seg-1080 -> ['NDB_1072', 'NDB_503', 'NDB_562']\n",
      "Chr7-6Mb-seg-1081 -> ['NDB_407', 'NDB_774', 'NDB_895']\n",
      "Chr7-6Mb-seg-1082 -> ['NDB_648', 'NDB_496', 'NDB_449']\n",
      "Chr7-6Mb-seg-1083 -> ['NDB_551', 'NDB_951', 'NDB_751']\n",
      "Chr7-6Mb-seg-1084 -> ['NDB_1053', 'NDB_526', 'NDB_1028']\n",
      "Chr7-6Mb-seg-1085 -> ['NDB_544', 'NDB_395', 'NDB_384']\n",
      "Chr7-6Mb-seg-1086 -> ['NDB_371', 'NDB_585', 'NDB_775']\n",
      "Chr7-6Mb-seg-1087 -> ['NDB_483', 'NDB_709', 'NDB_638']\n",
      "Chr7-6Mb-seg-1088 -> ['NDB_703', 'NDB_707', 'NDB_948']\n",
      "Chr7-6Mb-seg-1089 -> ['NDB_785', 'NDB_873', 'NDB_691']\n",
      "Chr7-6Mb-seg-1090 -> ['NDB_561', 'NDB_796', 'NDB_650']\n",
      "Chr7-6Mb-seg-1091 -> ['NDB_772', 'NDB_1013', 'NDB_700']\n",
      "Chr7-6Mb-seg-1092 -> ['NDB_869', 'NDB_537', 'NDB_908']\n",
      "Chr7-6Mb-seg-1093 -> ['NDB_557', 'NDB_879', 'NDB_401']\n",
      "Chr7-6Mb-seg-1094 -> ['NDB_372', 'NDB_472', 'NDB_735']\n",
      "Chr7-6Mb-seg-1095 -> ['NDB_925', 'NDB_359', 'NDB_940']\n",
      "Chr7-6Mb-seg-1096 -> ['NDB_476', 'NDB_606', 'NDB_365']\n",
      "Chr7-6Mb-seg-1097 -> ['NDB_495', 'NDB_538', 'NDB_600']\n",
      "Chr7-6Mb-seg-1098 -> ['NDB_1063', 'NDB_791', 'NDB_727']\n",
      "Chr7-6Mb-seg-1099 -> ['NDB_920', 'NDB_645', 'NDB_329']\n",
      "Chr7-6Mb-seg-1100 -> ['NDB_876', 'NDB_1051', 'NDB_882']\n",
      "Chr7-6Mb-seg-1101 -> ['NDB_592', 'NDB_480', 'NDB_802']\n",
      "Chr7-6Mb-seg-1102 -> ['NDB_845', 'NDB_604', 'NDB_740']\n",
      "Chr7-6Mb-seg-1103 -> ['NDB_705', 'NDB_379', 'NDB_977']\n",
      "Chr7-6Mb-seg-1104 -> ['NDB_622', 'NDB_386', 'NDB_840']\n",
      "Chr7-6Mb-seg-1105 -> ['NDB_581', 'NDB_441', 'NDB_577']\n",
      "Chr7-6Mb-seg-1106 -> ['NDB_702', 'NDB_439', 'NDB_494']\n",
      "Chr7-6Mb-seg-1107 -> ['NDB_1036', 'NDB_809', 'NDB_1056']\n",
      "Chr7-6Mb-seg-1108 -> ['NDB_776', 'NDB_666', 'NDB_904']\n",
      "Chr7-6Mb-seg-1109 -> ['NDB_732', 'NDB_976', 'NDB_488']\n",
      "Chr7-6Mb-seg-1110 -> ['NDB_922', 'NDB_383', 'NDB_429']\n",
      "Chr7-6Mb-seg-1111 -> ['NDB_460', 'NDB_871', 'NDB_914']\n",
      "Chr7-6Mb-seg-1112 -> ['NDB_593', 'NDB_689', 'NDB_867']\n",
      "Chr7-6Mb-seg-1113 -> ['NDB_862', 'NDB_421', 'NDB_951']\n",
      "Chr7-6Mb-seg-1114 -> ['NDB_611', 'NDB_836', 'NDB_526']\n",
      "Chr7-6Mb-seg-1115 -> ['NDB_519', 'NDB_450', 'NDB_395']\n",
      "Chr7-6Mb-seg-1116 -> ['NDB_556', 'NDB_481', 'NDB_585']\n",
      "Chr7-6Mb-seg-1117 -> ['NDB_479', 'NDB_872', 'NDB_709']\n",
      "Chr7-6Mb-seg-1118 -> ['NDB_486', 'NDB_693', 'NDB_707']\n",
      "Chr7-6Mb-seg-1119 -> ['NDB_601', 'NDB_649', 'NDB_873']\n",
      "Chr7-6Mb-seg-1120 -> ['NDB_416', 'NDB_413', 'NDB_796']\n",
      "Chr7-6Mb-seg-1121 -> ['NDB_573', 'NDB_708', 'NDB_500']\n",
      "Chr7-6Mb-seg-1122 -> ['NDB_436', 'NDB_931', 'NDB_471']\n",
      "Chr7-6Mb-seg-1123 -> ['NDB_459', 'NDB_491', 'NDB_751']\n",
      "Chr7-6Mb-seg-1124 -> ['NDB_418', 'NDB_627', 'NDB_1028']\n",
      "Chr7-6Mb-seg-1125 -> ['NDB_614', 'NDB_712', 'NDB_384']\n",
      "Chr7-6Mb-seg-1126 -> ['NDB_798', 'NDB_428', 'NDB_775']\n",
      "Chr7-6Mb-seg-1127 -> ['NDB_736', 'NDB_894', 'NDB_638']\n",
      "Chr7-6Mb-seg-1128 -> ['NDB_368', 'NDB_955', 'NDB_948']\n",
      "Chr7-6Mb-seg-1129 -> ['NDB_729', 'NDB_362', 'NDB_691']\n",
      "Chr7-6Mb-seg-1130 -> ['NDB_562', 'NDB_423', 'NDB_650']\n",
      "Chr7-6Mb-seg-1131 -> ['NDB_407', 'NDB_783', 'NDB_635']\n",
      "Chr7-6Mb-seg-1132 -> ['NDB_648', 'NDB_829', 'NDB_474']\n",
      "Chr7-6Mb-seg-1133 -> ['NDB_819', 'NDB_694', 'NDB_779']\n",
      "Chr7-6Mb-seg-1134 -> ['NDB_742', 'NDB_929', 'NDB_933']\n",
      "Chr7-6Mb-seg-1135 -> ['NDB_824', 'NDB_513', 'NDB_625']\n",
      "Chr7-6Mb-seg-1136 -> ['NDB_402', 'NDB_910', 'NDB_818']\n",
      "Chr7-6Mb-seg-1137 -> ['NDB_493', 'NDB_926', 'NDB_510']\n",
      "Chr7-6Mb-seg-1138 -> ['NDB_563', 'NDB_654', 'NDB_1042']\n",
      "Chr7-6Mb-seg-1139 -> ['NDB_891', 'NDB_928', 'NDB_902']\n",
      "Chr7-6Mb-seg-1140 -> ['NDB_1072', 'NDB_806', 'NDB_393']\n",
      "Chr7-6Mb-seg-1141 -> ['NDB_995', 'NDB_895', 'NDB_802']\n",
      "Chr7-6Mb-seg-1142 -> ['NDB_381', 'NDB_449', 'NDB_740']\n",
      "Chr7-6Mb-seg-1143 -> ['NDB_557', 'NDB_421', 'NDB_977']\n",
      "Chr7-6Mb-seg-1144 -> ['NDB_372', 'NDB_836', 'NDB_840']\n",
      "Chr7-6Mb-seg-1145 -> ['NDB_925', 'NDB_450', 'NDB_577']\n",
      "Chr7-6Mb-seg-1146 -> ['NDB_476', 'NDB_481', 'NDB_494']\n",
      "Chr7-6Mb-seg-1147 -> ['NDB_495', 'NDB_872', 'NDB_1056']\n",
      "Chr7-6Mb-seg-1148 -> ['NDB_1063', 'NDB_693', 'NDB_904']\n",
      "Chr7-6Mb-seg-1149 -> ['NDB_920', 'NDB_649', 'NDB_488']\n",
      "Chr7-6Mb-seg-1150 -> ['NDB_876', 'NDB_413', 'NDB_429']\n",
      "Chr7-6Mb-seg-1151 -> ['NDB_592', 'NDB_914', 'NDB_700']\n",
      "Chr7-6Mb-seg-1152 -> ['NDB_845', 'NDB_867', 'NDB_908']\n",
      "Chr7-6Mb-seg-1153 -> ['NDB_1014', 'NDB_401', 'NDB_379']\n",
      "Chr7-6Mb-seg-1154 -> ['NDB_958', 'NDB_735', 'NDB_386']\n",
      "Chr7-6Mb-seg-1155 -> ['NDB_392', 'NDB_940', 'NDB_441']\n",
      "Chr7-6Mb-seg-1156 -> ['NDB_939', 'NDB_365', 'NDB_439']\n",
      "Chr7-6Mb-seg-1157 -> ['NDB_547', 'NDB_600', 'NDB_809']\n",
      "Chr7-6Mb-seg-1158 -> ['NDB_644', 'NDB_727', 'NDB_666']\n",
      "Chr7-6Mb-seg-1159 -> ['NDB_522', 'NDB_329', 'NDB_976']\n",
      "Chr7-6Mb-seg-1160 -> ['NDB_1039', 'NDB_882', 'NDB_383']\n",
      "Chr7-6Mb-seg-1161 -> ['NDB_460', 'NDB_480', 'NDB_774']\n",
      "Chr7-6Mb-seg-1162 -> ['NDB_593', 'NDB_604', 'NDB_496']\n",
      "Chr7-6Mb-seg-1163 -> ['NDB_587', 'NDB_628', 'NDB_879']\n",
      "Chr7-6Mb-seg-1164 -> ['NDB_594', 'NDB_341', 'NDB_472']\n",
      "Chr7-6Mb-seg-1165 -> ['NDB_517', 'NDB_903', 'NDB_359']\n",
      "Chr7-6Mb-seg-1166 -> ['NDB_350', 'NDB_934', 'NDB_606']\n",
      "Chr7-6Mb-seg-1167 -> ['NDB_591', 'NDB_608', 'NDB_538']\n",
      "Chr7-6Mb-seg-1168 -> ['NDB_589', 'NDB_492', 'NDB_791']\n",
      "Chr7-6Mb-seg-1169 -> ['NDB_629', 'NDB_1048', 'NDB_645']\n",
      "Chr7-6Mb-seg-1170 -> ['NDB_960', 'NDB_503', 'NDB_1051']\n",
      "Chr7-6Mb-seg-1171 -> ['NDB_772', 'NDB_708', 'NDB_765']\n",
      "Chr7-6Mb-seg-1172 -> ['NDB_869', 'NDB_931', 'NDB_739']\n",
      "Chr7-6Mb-seg-1173 -> ['NDB_819', 'NDB_557', 'NDB_751']\n",
      "Chr7-6Mb-seg-1174 -> ['NDB_742', 'NDB_372', 'NDB_1028']\n",
      "Chr7-6Mb-seg-1175 -> ['NDB_824', 'NDB_925', 'NDB_384']\n",
      "Chr7-6Mb-seg-1176 -> ['NDB_402', 'NDB_476', 'NDB_775']\n",
      "Chr7-6Mb-seg-1177 -> ['NDB_493', 'NDB_495', 'NDB_638']\n",
      "Chr7-6Mb-seg-1178 -> ['NDB_563', 'NDB_1063', 'NDB_948']\n",
      "Chr7-6Mb-seg-1179 -> ['NDB_891', 'NDB_920', 'NDB_691']\n",
      "Chr7-6Mb-seg-1180 -> ['NDB_1072', 'NDB_876', 'NDB_650']\n",
      "Chr7-6Mb-seg-1181 -> ['NDB_635', 'NDB_573', 'NDB_895']\n",
      "Chr7-6Mb-seg-1182 -> ['NDB_474', 'NDB_436', 'NDB_449']\n",
      "Chr7-6Mb-seg-1183 -> ['NDB_551', 'NDB_421', 'NDB_694']\n",
      "Chr7-6Mb-seg-1184 -> ['NDB_1053', 'NDB_836', 'NDB_929']\n",
      "Chr7-6Mb-seg-1185 -> ['NDB_544', 'NDB_450', 'NDB_513']\n",
      "Chr7-6Mb-seg-1186 -> ['NDB_371', 'NDB_481', 'NDB_910']\n",
      "Chr7-6Mb-seg-1187 -> ['NDB_483', 'NDB_872', 'NDB_926']\n",
      "Chr7-6Mb-seg-1188 -> ['NDB_703', 'NDB_693', 'NDB_654']\n",
      "Chr7-6Mb-seg-1189 -> ['NDB_785', 'NDB_649', 'NDB_928']\n",
      "Chr7-6Mb-seg-1190 -> ['NDB_561', 'NDB_413', 'NDB_806']\n",
      "Chr7-6Mb-seg-1191 -> ['NDB_407', 'NDB_871', 'NDB_995']\n",
      "Chr7-6Mb-seg-1192 -> ['NDB_648', 'NDB_689', 'NDB_381']\n",
      "Chr7-6Mb-seg-1193 -> ['NDB_951', 'NDB_779', 'NDB_977']\n",
      "Chr7-6Mb-seg-1194 -> ['NDB_526', 'NDB_933', 'NDB_840']\n",
      "Chr7-6Mb-seg-1195 -> ['NDB_395', 'NDB_625', 'NDB_577']\n",
      "Chr7-6Mb-seg-1196 -> ['NDB_585', 'NDB_818', 'NDB_494']\n",
      "Chr7-6Mb-seg-1197 -> ['NDB_709', 'NDB_510', 'NDB_1056']\n",
      "Chr7-6Mb-seg-1198 -> ['NDB_707', 'NDB_1042', 'NDB_904']\n",
      "Chr7-6Mb-seg-1199 -> ['NDB_873', 'NDB_902', 'NDB_488']\n",
      "Chr7-6Mb-seg-1200 -> ['NDB_796', 'NDB_393', 'NDB_429']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "check_readout_dict = pickle.load(open(os.path.join(library_folder, 'readout_usage.pkl'), 'rb'))\n",
    "\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(full_records, check_readout_dict, \n",
    "                                                                     total_readout_num=num_on_bits)\n",
    "print(reg2readout_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3e64afd0",
   "metadata": {},
   "source": [
    "### 4.1.4 check readout usage in the library"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 199,
   "id": "1647e94f",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- extracting probes from >region:Chr7-6Mb-seg-1, region:Chr7-6Mb-seg-2, region:Chr7-6Mb-seg-3, region:Chr7-6Mb-seg-4, region:Chr7-6Mb-seg-7, region:Chr7-6Mb-seg-9, region:Chr7-6Mb-seg-10, region:Chr7-6Mb-seg-11, region:Chr7-6Mb-seg-12, region:Chr7-6Mb-seg-14, region:Chr7-6Mb-seg-15, region:Chr7-6Mb-seg-16, region:Chr7-6Mb-seg-19, region:Chr7-6Mb-seg-20, region:Chr7-6Mb-seg-21, region:Chr7-6Mb-seg-22, region:Chr7-6Mb-seg-23, region:Chr7-6Mb-seg-24, region:Chr7-6Mb-seg-25, region:Chr7-6Mb-seg-27, region:Chr7-6Mb-seg-28, region:Chr7-6Mb-seg-29, region:Chr7-6Mb-seg-36, region:Chr7-6Mb-seg-38, region:Chr7-6Mb-seg-39, region:Chr7-6Mb-seg-40, region:Chr7-6Mb-seg-41, region:Chr7-6Mb-seg-42, region:Chr7-6Mb-seg-43, region:Chr7-6Mb-seg-44, region:Chr7-6Mb-seg-45, region:Chr7-6Mb-seg-46, region:Chr7-6Mb-seg-47, region:Chr7-6Mb-seg-48, region:Chr7-6Mb-seg-49, region:Chr7-6Mb-seg-50, region:Chr7-6Mb-seg-51, region:Chr7-6Mb-seg-52, region:Chr7-6Mb-seg-53, region:Chr7-6Mb-seg-54, region:Chr7-6Mb-seg-55, region:Chr7-6Mb-seg-56, region:Chr7-6Mb-seg-57, region:Chr7-6Mb-seg-58, region:Chr7-6Mb-seg-59, region:Chr7-6Mb-seg-60, region:Chr7-6Mb-seg-61, region:Chr7-6Mb-seg-62, region:Chr7-6Mb-seg-63, region:Chr7-6Mb-seg-64, region:Chr7-6Mb-seg-65, region:Chr7-6Mb-seg-66, region:Chr7-6Mb-seg-67, region:Chr7-6Mb-seg-68, region:Chr7-6Mb-seg-69, region:Chr7-6Mb-seg-70, region:Chr7-6Mb-seg-71, region:Chr7-6Mb-seg-72, region:Chr7-6Mb-seg-73, region:Chr7-6Mb-seg-74, region:Chr7-6Mb-seg-75, region:Chr7-6Mb-seg-76, region:Chr7-6Mb-seg-77, region:Chr7-6Mb-seg-78, region:Chr7-6Mb-seg-79, region:Chr7-6Mb-seg-80, region:Chr7-6Mb-seg-81, region:Chr7-6Mb-seg-82, region:Chr7-6Mb-seg-83, region:Chr7-6Mb-seg-84, region:Chr7-6Mb-seg-85, region:Chr7-6Mb-seg-86, region:Chr7-6Mb-seg-87, region:Chr7-6Mb-seg-88, region:Chr7-6Mb-seg-89, region:Chr7-6Mb-seg-90, region:Chr7-6Mb-seg-91, region:Chr7-6Mb-seg-92, region:Chr7-6Mb-seg-93, region:Chr7-6Mb-seg-94, region:Chr7-6Mb-seg-95, region:Chr7-6Mb-seg-96, region:Chr7-6Mb-seg-97, region:Chr7-6Mb-seg-98, region:Chr7-6Mb-seg-99, region:Chr7-6Mb-seg-100, region:Chr7-6Mb-seg-101, region:Chr7-6Mb-seg-102, region:Chr7-6Mb-seg-103, region:Chr7-6Mb-seg-104, region:Chr7-6Mb-seg-105, region:Chr7-6Mb-seg-106, region:Chr7-6Mb-seg-107, region:Chr7-6Mb-seg-108, region:Chr7-6Mb-seg-109, region:Chr7-6Mb-seg-110, region:Chr7-6Mb-seg-111, region:Chr7-6Mb-seg-112, region:Chr7-6Mb-seg-113, region:Chr7-6Mb-seg-114, region:Chr7-6Mb-seg-115, region:Chr7-6Mb-seg-116, region:Chr7-6Mb-seg-117, region:Chr7-6Mb-seg-118, region:Chr7-6Mb-seg-119, region:Chr7-6Mb-seg-120, region:Chr7-6Mb-seg-121, region:Chr7-6Mb-seg-122, region:Chr7-6Mb-seg-123, region:Chr7-6Mb-seg-124, region:Chr7-6Mb-seg-125, region:Chr7-6Mb-seg-126, region:Chr7-6Mb-seg-127, region:Chr7-6Mb-seg-128, region:Chr7-6Mb-seg-129, region:Chr7-6Mb-seg-130, region:Chr7-6Mb-seg-131, region:Chr7-6Mb-seg-132, region:Chr7-6Mb-seg-133, region:Chr7-6Mb-seg-134, region:Chr7-6Mb-seg-135, region:Chr7-6Mb-seg-136, region:Chr7-6Mb-seg-137, region:Chr7-6Mb-seg-138, region:Chr7-6Mb-seg-139, region:Chr7-6Mb-seg-140, region:Chr7-6Mb-seg-141, region:Chr7-6Mb-seg-142, region:Chr7-6Mb-seg-143, region:Chr7-6Mb-seg-144, region:Chr7-6Mb-seg-145, region:Chr7-6Mb-seg-146, region:Chr7-6Mb-seg-147, region:Chr7-6Mb-seg-148, region:Chr7-6Mb-seg-149, region:Chr7-6Mb-seg-150, region:Chr7-6Mb-seg-151, region:Chr7-6Mb-seg-152, region:Chr7-6Mb-seg-153, region:Chr7-6Mb-seg-154, region:Chr7-6Mb-seg-155, region:Chr7-6Mb-seg-156, region:Chr7-6Mb-seg-157, region:Chr7-6Mb-seg-158, region:Chr7-6Mb-seg-159, region:Chr7-6Mb-seg-161, region:Chr7-6Mb-seg-162, region:Chr7-6Mb-seg-163, region:Chr7-6Mb-seg-164, region:Chr7-6Mb-seg-165, region:Chr7-6Mb-seg-166, region:Chr7-6Mb-seg-167, region:Chr7-6Mb-seg-168, region:Chr7-6Mb-seg-169, region:Chr7-6Mb-seg-170, region:Chr7-6Mb-seg-171, region:Chr7-6Mb-seg-172, region:Chr7-6Mb-seg-173, region:Chr7-6Mb-seg-174, region:Chr7-6Mb-seg-175, region:Chr7-6Mb-seg-176, region:Chr7-6Mb-seg-177, region:Chr7-6Mb-seg-178, region:Chr7-6Mb-seg-179, region:Chr7-6Mb-seg-180, region:Chr7-6Mb-seg-181, region:Chr7-6Mb-seg-182, region:Chr7-6Mb-seg-183, region:Chr7-6Mb-seg-184, region:Chr7-6Mb-seg-185, region:Chr7-6Mb-seg-186, region:Chr7-6Mb-seg-187, region:Chr7-6Mb-seg-188, region:Chr7-6Mb-seg-189, region:Chr7-6Mb-seg-190, region:Chr7-6Mb-seg-191, region:Chr7-6Mb-seg-192, region:Chr7-6Mb-seg-193, region:Chr7-6Mb-seg-194, region:Chr7-6Mb-seg-195, region:Chr7-6Mb-seg-196, region:Chr7-6Mb-seg-197, region:Chr7-6Mb-seg-198, region:Chr7-6Mb-seg-199, region:Chr7-6Mb-seg-200, region:Chr7-6Mb-seg-201, region:Chr7-6Mb-seg-202, region:Chr7-6Mb-seg-203, region:Chr7-6Mb-seg-204, region:Chr7-6Mb-seg-205, region:Chr7-6Mb-seg-206, region:Chr7-6Mb-seg-207, region:Chr7-6Mb-seg-208, region:Chr7-6Mb-seg-209, region:Chr7-6Mb-seg-210, region:Chr7-6Mb-seg-211, region:Chr7-6Mb-seg-212, region:Chr7-6Mb-seg-213, region:Chr7-6Mb-seg-214, region:Chr7-6Mb-seg-215, region:Chr7-6Mb-seg-216, region:Chr7-6Mb-seg-217, region:Chr7-6Mb-seg-218, region:Chr7-6Mb-seg-219, region:Chr7-6Mb-seg-220, region:Chr7-6Mb-seg-221, region:Chr7-6Mb-seg-222, region:Chr7-6Mb-seg-223, region:Chr7-6Mb-seg-224, region:Chr7-6Mb-seg-225, region:Chr7-6Mb-seg-226, region:Chr7-6Mb-seg-227, region:Chr7-6Mb-seg-228, region:Chr7-6Mb-seg-229, region:Chr7-6Mb-seg-230, region:Chr7-6Mb-seg-231, region:Chr7-6Mb-seg-232, region:Chr7-6Mb-seg-233, region:Chr7-6Mb-seg-234, region:Chr7-6Mb-seg-235, region:Chr7-6Mb-seg-236, region:Chr7-6Mb-seg-237, region:Chr7-6Mb-seg-238, region:Chr7-6Mb-seg-239, region:Chr7-6Mb-seg-240, region:Chr7-6Mb-seg-241, region:Chr7-6Mb-seg-242, region:Chr7-6Mb-seg-243, region:Chr7-6Mb-seg-244, region:Chr7-6Mb-seg-245, region:Chr7-6Mb-seg-246, region:Chr7-6Mb-seg-247, region:Chr7-6Mb-seg-248, region:Chr7-6Mb-seg-249, region:Chr7-6Mb-seg-250, region:Chr7-6Mb-seg-251, region:Chr7-6Mb-seg-252, region:Chr7-6Mb-seg-253, region:Chr7-6Mb-seg-254, region:Chr7-6Mb-seg-255, region:Chr7-6Mb-seg-256, region:Chr7-6Mb-seg-257, region:Chr7-6Mb-seg-258, region:Chr7-6Mb-seg-259, region:Chr7-6Mb-seg-260, region:Chr7-6Mb-seg-261, region:Chr7-6Mb-seg-262, region:Chr7-6Mb-seg-263, region:Chr7-6Mb-seg-264, region:Chr7-6Mb-seg-265, region:Chr7-6Mb-seg-266, region:Chr7-6Mb-seg-267, region:Chr7-6Mb-seg-268, region:Chr7-6Mb-seg-269, region:Chr7-6Mb-seg-270, region:Chr7-6Mb-seg-271, region:Chr7-6Mb-seg-272, region:Chr7-6Mb-seg-273, region:Chr7-6Mb-seg-274, region:Chr7-6Mb-seg-275, region:Chr7-6Mb-seg-276, region:Chr7-6Mb-seg-277, region:Chr7-6Mb-seg-278, region:Chr7-6Mb-seg-279, region:Chr7-6Mb-seg-280, region:Chr7-6Mb-seg-281, region:Chr7-6Mb-seg-282, region:Chr7-6Mb-seg-283, region:Chr7-6Mb-seg-284, region:Chr7-6Mb-seg-285, region:Chr7-6Mb-seg-286, region:Chr7-6Mb-seg-287, region:Chr7-6Mb-seg-288, region:Chr7-6Mb-seg-289, region:Chr7-6Mb-seg-290, region:Chr7-6Mb-seg-291, region:Chr7-6Mb-seg-292, region:Chr7-6Mb-seg-293, region:Chr7-6Mb-seg-294, region:Chr7-6Mb-seg-295, region:Chr7-6Mb-seg-296, region:Chr7-6Mb-seg-297, region:Chr7-6Mb-seg-298, region:Chr7-6Mb-seg-299, region:Chr7-6Mb-seg-300, region:Chr7-6Mb-seg-301, region:Chr7-6Mb-seg-302, region:Chr7-6Mb-seg-303, region:Chr7-6Mb-seg-304, region:Chr7-6Mb-seg-305, region:Chr7-6Mb-seg-306, region:Chr7-6Mb-seg-307, region:Chr7-6Mb-seg-308, region:Chr7-6Mb-seg-309, region:Chr7-6Mb-seg-310, region:Chr7-6Mb-seg-311, region:Chr7-6Mb-seg-312, region:Chr7-6Mb-seg-313, region:Chr7-6Mb-seg-314, region:Chr7-6Mb-seg-315, region:Chr7-6Mb-seg-316, region:Chr7-6Mb-seg-317, region:Chr7-6Mb-seg-318, region:Chr7-6Mb-seg-319, region:Chr7-6Mb-seg-320, region:Chr7-6Mb-seg-321, region:Chr7-6Mb-seg-322, region:Chr7-6Mb-seg-323, region:Chr7-6Mb-seg-324, region:Chr7-6Mb-seg-325, region:Chr7-6Mb-seg-326, region:Chr7-6Mb-seg-327, region:Chr7-6Mb-seg-328, region:Chr7-6Mb-seg-329, region:Chr7-6Mb-seg-330, region:Chr7-6Mb-seg-331, region:Chr7-6Mb-seg-332, region:Chr7-6Mb-seg-333, region:Chr7-6Mb-seg-334, region:Chr7-6Mb-seg-335, region:Chr7-6Mb-seg-336, region:Chr7-6Mb-seg-337, region:Chr7-6Mb-seg-338, region:Chr7-6Mb-seg-339, region:Chr7-6Mb-seg-340, region:Chr7-6Mb-seg-341, region:Chr7-6Mb-seg-342, region:Chr7-6Mb-seg-343, region:Chr7-6Mb-seg-344, region:Chr7-6Mb-seg-345, region:Chr7-6Mb-seg-346, region:Chr7-6Mb-seg-347, region:Chr7-6Mb-seg-348, region:Chr7-6Mb-seg-349, region:Chr7-6Mb-seg-350, region:Chr7-6Mb-seg-351, region:Chr7-6Mb-seg-352, region:Chr7-6Mb-seg-353, region:Chr7-6Mb-seg-354, region:Chr7-6Mb-seg-355, region:Chr7-6Mb-seg-356, region:Chr7-6Mb-seg-357, region:Chr7-6Mb-seg-358, region:Chr7-6Mb-seg-359, region:Chr7-6Mb-seg-360, region:Chr7-6Mb-seg-361, region:Chr7-6Mb-seg-362, region:Chr7-6Mb-seg-363, region:Chr7-6Mb-seg-364, region:Chr7-6Mb-seg-365, region:Chr7-6Mb-seg-366, region:Chr7-6Mb-seg-367, region:Chr7-6Mb-seg-368, region:Chr7-6Mb-seg-369, region:Chr7-6Mb-seg-370, region:Chr7-6Mb-seg-371, region:Chr7-6Mb-seg-372, region:Chr7-6Mb-seg-373, region:Chr7-6Mb-seg-374, region:Chr7-6Mb-seg-375, region:Chr7-6Mb-seg-376, region:Chr7-6Mb-seg-377, region:Chr7-6Mb-seg-378, region:Chr7-6Mb-seg-379, region:Chr7-6Mb-seg-380, region:Chr7-6Mb-seg-381, region:Chr7-6Mb-seg-382, region:Chr7-6Mb-seg-383, "
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "region:Chr7-6Mb-seg-384, region:Chr7-6Mb-seg-385, region:Chr7-6Mb-seg-386, region:Chr7-6Mb-seg-387, region:Chr7-6Mb-seg-388, region:Chr7-6Mb-seg-389, region:Chr7-6Mb-seg-390, region:Chr7-6Mb-seg-391, region:Chr7-6Mb-seg-392, region:Chr7-6Mb-seg-393, region:Chr7-6Mb-seg-394, region:Chr7-6Mb-seg-395, region:Chr7-6Mb-seg-396, region:Chr7-6Mb-seg-397, region:Chr7-6Mb-seg-398, region:Chr7-6Mb-seg-399, region:Chr7-6Mb-seg-400, region:Chr7-6Mb-seg-401, region:Chr7-6Mb-seg-402, region:Chr7-6Mb-seg-403, region:Chr7-6Mb-seg-404, region:Chr7-6Mb-seg-405, region:Chr7-6Mb-seg-406, region:Chr7-6Mb-seg-407, region:Chr7-6Mb-seg-408, region:Chr7-6Mb-seg-409, region:Chr7-6Mb-seg-410, region:Chr7-6Mb-seg-411, region:Chr7-6Mb-seg-412, region:Chr7-6Mb-seg-413, region:Chr7-6Mb-seg-414, region:Chr7-6Mb-seg-415, region:Chr7-6Mb-seg-416, region:Chr7-6Mb-seg-417, region:Chr7-6Mb-seg-418, region:Chr7-6Mb-seg-419, region:Chr7-6Mb-seg-420, region:Chr7-6Mb-seg-421, region:Chr7-6Mb-seg-422, region:Chr7-6Mb-seg-423, region:Chr7-6Mb-seg-424, region:Chr7-6Mb-seg-425, region:Chr7-6Mb-seg-426, region:Chr7-6Mb-seg-427, region:Chr7-6Mb-seg-428, region:Chr7-6Mb-seg-429, region:Chr7-6Mb-seg-430, region:Chr7-6Mb-seg-431, region:Chr7-6Mb-seg-432, region:Chr7-6Mb-seg-433, region:Chr7-6Mb-seg-434, region:Chr7-6Mb-seg-435, region:Chr7-6Mb-seg-436, region:Chr7-6Mb-seg-437, region:Chr7-6Mb-seg-438, region:Chr7-6Mb-seg-439, region:Chr7-6Mb-seg-440, region:Chr7-6Mb-seg-441, region:Chr7-6Mb-seg-442, region:Chr7-6Mb-seg-443, region:Chr7-6Mb-seg-444, region:Chr7-6Mb-seg-445, region:Chr7-6Mb-seg-446, region:Chr7-6Mb-seg-447, region:Chr7-6Mb-seg-448, region:Chr7-6Mb-seg-449, region:Chr7-6Mb-seg-450, region:Chr7-6Mb-seg-451, region:Chr7-6Mb-seg-452, region:Chr7-6Mb-seg-453, region:Chr7-6Mb-seg-454, region:Chr7-6Mb-seg-455, region:Chr7-6Mb-seg-456, region:Chr7-6Mb-seg-457, region:Chr7-6Mb-seg-458, region:Chr7-6Mb-seg-459, region:Chr7-6Mb-seg-460, region:Chr7-6Mb-seg-461, region:Chr7-6Mb-seg-462, region:Chr7-6Mb-seg-463, region:Chr7-6Mb-seg-464, region:Chr7-6Mb-seg-465, region:Chr7-6Mb-seg-466, region:Chr7-6Mb-seg-467, region:Chr7-6Mb-seg-468, region:Chr7-6Mb-seg-469, region:Chr7-6Mb-seg-470, region:Chr7-6Mb-seg-471, region:Chr7-6Mb-seg-472, region:Chr7-6Mb-seg-473, region:Chr7-6Mb-seg-474, region:Chr7-6Mb-seg-475, region:Chr7-6Mb-seg-476, region:Chr7-6Mb-seg-477, region:Chr7-6Mb-seg-478, region:Chr7-6Mb-seg-479, region:Chr7-6Mb-seg-480, region:Chr7-6Mb-seg-481, region:Chr7-6Mb-seg-482, region:Chr7-6Mb-seg-483, region:Chr7-6Mb-seg-484, region:Chr7-6Mb-seg-485, region:Chr7-6Mb-seg-486, region:Chr7-6Mb-seg-487, region:Chr7-6Mb-seg-488, region:Chr7-6Mb-seg-489, region:Chr7-6Mb-seg-490, region:Chr7-6Mb-seg-491, region:Chr7-6Mb-seg-492, region:Chr7-6Mb-seg-493, region:Chr7-6Mb-seg-494, region:Chr7-6Mb-seg-495, region:Chr7-6Mb-seg-496, region:Chr7-6Mb-seg-497, region:Chr7-6Mb-seg-498, region:Chr7-6Mb-seg-499, region:Chr7-6Mb-seg-500, region:Chr7-6Mb-seg-501, region:Chr7-6Mb-seg-502, region:Chr7-6Mb-seg-503, region:Chr7-6Mb-seg-504, region:Chr7-6Mb-seg-505, region:Chr7-6Mb-seg-506, region:Chr7-6Mb-seg-507, region:Chr7-6Mb-seg-508, region:Chr7-6Mb-seg-509, region:Chr7-6Mb-seg-510, region:Chr7-6Mb-seg-511, region:Chr7-6Mb-seg-512, region:Chr7-6Mb-seg-513, region:Chr7-6Mb-seg-514, region:Chr7-6Mb-seg-515, region:Chr7-6Mb-seg-516, region:Chr7-6Mb-seg-517, region:Chr7-6Mb-seg-518, region:Chr7-6Mb-seg-519, region:Chr7-6Mb-seg-520, region:Chr7-6Mb-seg-521, region:Chr7-6Mb-seg-522, region:Chr7-6Mb-seg-523, region:Chr7-6Mb-seg-524, region:Chr7-6Mb-seg-525, region:Chr7-6Mb-seg-526, region:Chr7-6Mb-seg-527, region:Chr7-6Mb-seg-528, region:Chr7-6Mb-seg-529, region:Chr7-6Mb-seg-530, region:Chr7-6Mb-seg-531, region:Chr7-6Mb-seg-532, region:Chr7-6Mb-seg-533, region:Chr7-6Mb-seg-534, region:Chr7-6Mb-seg-535, region:Chr7-6Mb-seg-536, region:Chr7-6Mb-seg-537, region:Chr7-6Mb-seg-538, region:Chr7-6Mb-seg-539, region:Chr7-6Mb-seg-540, region:Chr7-6Mb-seg-541, region:Chr7-6Mb-seg-542, region:Chr7-6Mb-seg-543, region:Chr7-6Mb-seg-544, region:Chr7-6Mb-seg-545, region:Chr7-6Mb-seg-546, region:Chr7-6Mb-seg-547, region:Chr7-6Mb-seg-548, region:Chr7-6Mb-seg-549, region:Chr7-6Mb-seg-550, region:Chr7-6Mb-seg-551, region:Chr7-6Mb-seg-552, region:Chr7-6Mb-seg-553, region:Chr7-6Mb-seg-554, region:Chr7-6Mb-seg-555, region:Chr7-6Mb-seg-556, region:Chr7-6Mb-seg-557, region:Chr7-6Mb-seg-558, region:Chr7-6Mb-seg-559, region:Chr7-6Mb-seg-560, region:Chr7-6Mb-seg-561, region:Chr7-6Mb-seg-562, region:Chr7-6Mb-seg-563, region:Chr7-6Mb-seg-564, region:Chr7-6Mb-seg-565, region:Chr7-6Mb-seg-566, region:Chr7-6Mb-seg-567, region:Chr7-6Mb-seg-568, region:Chr7-6Mb-seg-569, region:Chr7-6Mb-seg-570, region:Chr7-6Mb-seg-571, region:Chr7-6Mb-seg-572, region:Chr7-6Mb-seg-573, region:Chr7-6Mb-seg-574, region:Chr7-6Mb-seg-575, region:Chr7-6Mb-seg-576, region:Chr7-6Mb-seg-577, region:Chr7-6Mb-seg-578, region:Chr7-6Mb-seg-579, region:Chr7-6Mb-seg-580, region:Chr7-6Mb-seg-581, region:Chr7-6Mb-seg-582, region:Chr7-6Mb-seg-583, region:Chr7-6Mb-seg-584, region:Chr7-6Mb-seg-585, region:Chr7-6Mb-seg-586, region:Chr7-6Mb-seg-587, region:Chr7-6Mb-seg-588, region:Chr7-6Mb-seg-589, region:Chr7-6Mb-seg-590, region:Chr7-6Mb-seg-591, region:Chr7-6Mb-seg-592, region:Chr7-6Mb-seg-593, region:Chr7-6Mb-seg-594, region:Chr7-6Mb-seg-595, region:Chr7-6Mb-seg-596, region:Chr7-6Mb-seg-597, region:Chr7-6Mb-seg-598, region:Chr7-6Mb-seg-599, region:Chr7-6Mb-seg-600, region:Chr7-6Mb-seg-601, region:Chr7-6Mb-seg-602, region:Chr7-6Mb-seg-603, region:Chr7-6Mb-seg-604, region:Chr7-6Mb-seg-605, region:Chr7-6Mb-seg-606, region:Chr7-6Mb-seg-607, region:Chr7-6Mb-seg-608, region:Chr7-6Mb-seg-609, region:Chr7-6Mb-seg-610, region:Chr7-6Mb-seg-611, region:Chr7-6Mb-seg-612, region:Chr7-6Mb-seg-613, region:Chr7-6Mb-seg-614, region:Chr7-6Mb-seg-615, region:Chr7-6Mb-seg-616, region:Chr7-6Mb-seg-617, region:Chr7-6Mb-seg-618, region:Chr7-6Mb-seg-619, region:Chr7-6Mb-seg-620, region:Chr7-6Mb-seg-621, region:Chr7-6Mb-seg-622, region:Chr7-6Mb-seg-623, region:Chr7-6Mb-seg-624, region:Chr7-6Mb-seg-625, region:Chr7-6Mb-seg-626, region:Chr7-6Mb-seg-627, region:Chr7-6Mb-seg-628, region:Chr7-6Mb-seg-629, region:Chr7-6Mb-seg-630, region:Chr7-6Mb-seg-631, region:Chr7-6Mb-seg-632, region:Chr7-6Mb-seg-633, region:Chr7-6Mb-seg-634, region:Chr7-6Mb-seg-635, region:Chr7-6Mb-seg-638, region:Chr7-6Mb-seg-639, region:Chr7-6Mb-seg-640, region:Chr7-6Mb-seg-641, region:Chr7-6Mb-seg-642, region:Chr7-6Mb-seg-643, region:Chr7-6Mb-seg-644, region:Chr7-6Mb-seg-647, region:Chr7-6Mb-seg-648, region:Chr7-6Mb-seg-649, region:Chr7-6Mb-seg-650, region:Chr7-6Mb-seg-651, region:Chr7-6Mb-seg-655, region:Chr7-6Mb-seg-660, region:Chr7-6Mb-seg-661, region:Chr7-6Mb-seg-662, region:Chr7-6Mb-seg-663, region:Chr7-6Mb-seg-664, region:Chr7-6Mb-seg-665, region:Chr7-6Mb-seg-668, region:Chr7-6Mb-seg-669, region:Chr7-6Mb-seg-670, region:Chr7-6Mb-seg-671, region:Chr7-6Mb-seg-672, region:Chr7-6Mb-seg-674, region:Chr7-6Mb-seg-675, region:Chr7-6Mb-seg-676, region:Chr7-6Mb-seg-677, region:Chr7-6Mb-seg-679, region:Chr7-6Mb-seg-680, region:Chr7-6Mb-seg-681, region:Chr7-6Mb-seg-682, region:Chr7-6Mb-seg-683, region:Chr7-6Mb-seg-692, region:Chr7-6Mb-seg-693, region:Chr7-6Mb-seg-698, region:Chr7-6Mb-seg-699, region:Chr7-6Mb-seg-700, region:Chr7-6Mb-seg-701, region:Chr7-6Mb-seg-703, region:Chr7-6Mb-seg-704, region:Chr7-6Mb-seg-705, region:Chr7-6Mb-seg-706, region:Chr7-6Mb-seg-707, region:Chr7-6Mb-seg-708, region:Chr7-6Mb-seg-710, region:Chr7-6Mb-seg-711, region:Chr7-6Mb-seg-712, region:Chr7-6Mb-seg-713, region:Chr7-6Mb-seg-717, region:Chr7-6Mb-seg-718, region:Chr7-6Mb-seg-719, region:Chr7-6Mb-seg-720, region:Chr7-6Mb-seg-721, region:Chr7-6Mb-seg-722, region:Chr7-6Mb-seg-723, region:Chr7-6Mb-seg-726, region:Chr7-6Mb-seg-727, region:Chr7-6Mb-seg-728, region:Chr7-6Mb-seg-731, region:Chr7-6Mb-seg-732, region:Chr7-6Mb-seg-733, region:Chr7-6Mb-seg-734, region:Chr7-6Mb-seg-735, region:Chr7-6Mb-seg-738, region:Chr7-6Mb-seg-739, region:Chr7-6Mb-seg-740, region:Chr7-6Mb-seg-741, region:Chr7-6Mb-seg-742, region:Chr7-6Mb-seg-743, region:Chr7-6Mb-seg-744, region:Chr7-6Mb-seg-745, region:Chr7-6Mb-seg-746, region:Chr7-6Mb-seg-747, region:Chr7-6Mb-seg-748, region:Chr7-6Mb-seg-749, region:Chr7-6Mb-seg-750, region:Chr7-6Mb-seg-751, region:Chr7-6Mb-seg-752, region:Chr7-6Mb-seg-753, region:Chr7-6Mb-seg-754, region:Chr7-6Mb-seg-755, region:Chr7-6Mb-seg-756, region:Chr7-6Mb-seg-757, region:Chr7-6Mb-seg-758, region:Chr7-6Mb-seg-760, region:Chr7-6Mb-seg-762, region:Chr7-6Mb-seg-763, region:Chr7-6Mb-seg-764, region:Chr7-6Mb-seg-765, region:Chr7-6Mb-seg-766, region:Chr7-6Mb-seg-767, region:Chr7-6Mb-seg-768, region:Chr7-6Mb-seg-769, region:Chr7-6Mb-seg-770, region:Chr7-6Mb-seg-771, region:Chr7-6Mb-seg-772, region:Chr7-6Mb-seg-773, region:Chr7-6Mb-seg-775, region:Chr7-6Mb-seg-776, region:Chr7-6Mb-seg-777, region:Chr7-6Mb-seg-778, region:Chr7-6Mb-seg-780, region:Chr7-6Mb-seg-781, region:Chr7-6Mb-seg-784, region:Chr7-6Mb-seg-785, region:Chr7-6Mb-seg-786, region:Chr7-6Mb-seg-787, region:Chr7-6Mb-seg-788, region:Chr7-6Mb-seg-790, region:Chr7-6Mb-seg-791, region:Chr7-6Mb-seg-792, region:Chr7-6Mb-seg-793, region:Chr7-6Mb-seg-794, region:Chr7-6Mb-seg-795, region:Chr7-6Mb-seg-796, region:Chr7-6Mb-seg-797, region:Chr7-6Mb-seg-798, region:Chr7-6Mb-seg-799, region:Chr7-6Mb-seg-800, region:Chr7-6Mb-seg-801, region:Chr7-6Mb-seg-802, region:Chr7-6Mb-seg-804, region:Chr7-6Mb-seg-805, region:Chr7-6Mb-seg-806, region:Chr7-6Mb-seg-807, region:Chr7-6Mb-seg-808, region:Chr7-6Mb-seg-809, region:Chr7-6Mb-seg-810, region:Chr7-6Mb-seg-812, region:Chr7-6Mb-seg-813, region:Chr7-6Mb-seg-814, region:Chr7-6Mb-seg-816, region:Chr7-6Mb-seg-817, region:Chr7-6Mb-seg-818, region:Chr7-6Mb-seg-819, region:Chr7-6Mb-seg-820, region:Chr7-6Mb-seg-821, region:Chr7-6Mb-seg-822, region:Chr7-6Mb-seg-823, region:Chr7-6Mb-seg-824, region:Chr7-6Mb-seg-825, region:Chr7-6Mb-seg-828, region:Chr7-6Mb-seg-829, region:Chr7-6Mb-seg-831, region:Chr7-6Mb-seg-832, region:Chr7-6Mb-seg-833, region:Chr7-6Mb-seg-834, region:Chr7-6Mb-seg-835, region:Chr7-6Mb-seg-836, region:Chr7-6Mb-seg-837, region:Chr7-6Mb-seg-840, region:Chr7-6Mb-seg-841, region:Chr7-6Mb-seg-842, region:Chr7-6Mb-seg-845, region:Chr7-6Mb-seg-846, region:Chr7-6Mb-seg-847, region:Chr7-6Mb-seg-848, region:Chr7-6Mb-seg-849, region:Chr7-6Mb-seg-850, region:Chr7-6Mb-seg-852, region:Chr7-6Mb-seg-853, region:Chr7-6Mb-seg-854, region:Chr7-6Mb-seg-855, region:Chr7-6Mb-seg-856, region:Chr7-6Mb-seg-857, region:Chr7-6Mb-seg-858, region:Chr7-6Mb-seg-859, region:Chr7-6Mb-seg-860, region:Chr7-6Mb-seg-862, region:Chr7-6Mb-seg-863, region:Chr7-6Mb-seg-864, region:Chr7-6Mb-seg-865, region:Chr7-6Mb-seg-866, region:Chr7-6Mb-seg-868, region:Chr7-6Mb-seg-869, region:Chr7-6Mb-seg-870, region:Chr7-6Mb-seg-873, region:Chr7-6Mb-seg-876, region:Chr7-6Mb-seg-878, region:Chr7-6Mb-seg-879, region:Chr7-6Mb-seg-880, region:Chr7-6Mb-seg-881, region:Chr7-6Mb-seg-883, region:Chr7-6Mb-seg-885, region:Chr7-6Mb-seg-888, region:Chr7-6Mb-seg-889, region:Chr7-6Mb-seg-890, region:Chr7-6Mb-seg-891, region:Chr7-6Mb-seg-892, region:Chr7-6Mb-seg-893, region:Chr7-6Mb-seg-894, region:Chr7-6Mb-seg-895, region:Chr7-6Mb-seg-896, region:Chr7-6Mb-seg-897, region:Chr7-6Mb-seg-900, region:Chr7-6Mb-seg-901, region:Chr7-6Mb-seg-902, region:Chr7-6Mb-seg-904, "
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "region:Chr7-6Mb-seg-906, region:Chr7-6Mb-seg-907, region:Chr7-6Mb-seg-911, region:Chr7-6Mb-seg-912, region:Chr7-6Mb-seg-913, region:Chr7-6Mb-seg-916, region:Chr7-6Mb-seg-917, region:Chr7-6Mb-seg-918, region:Chr7-6Mb-seg-919, region:Chr7-6Mb-seg-922, region:Chr7-6Mb-seg-923, region:Chr7-6Mb-seg-924, region:Chr7-6Mb-seg-926, region:Chr7-6Mb-seg-929, region:Chr7-6Mb-seg-931, region:Chr7-6Mb-seg-932, region:Chr7-6Mb-seg-933, region:Chr7-6Mb-seg-934, region:Chr7-6Mb-seg-935, region:Chr7-6Mb-seg-937, region:Chr7-6Mb-seg-938, region:Chr7-6Mb-seg-939, region:Chr7-6Mb-seg-942, region:Chr7-6Mb-seg-944, region:Chr7-6Mb-seg-945, region:Chr7-6Mb-seg-946, region:Chr7-6Mb-seg-947, region:Chr7-6Mb-seg-950, region:Chr7-6Mb-seg-951, region:Chr7-6Mb-seg-952, region:Chr7-6Mb-seg-953, region:Chr7-6Mb-seg-954, region:Chr7-6Mb-seg-955, region:Chr7-6Mb-seg-956, region:Chr7-6Mb-seg-957, region:Chr7-6Mb-seg-958, region:Chr7-6Mb-seg-959, region:Chr7-6Mb-seg-960, region:Chr7-6Mb-seg-961, region:Chr7-6Mb-seg-962, region:Chr7-6Mb-seg-963, region:Chr7-6Mb-seg-964, region:Chr7-6Mb-seg-965, region:Chr7-6Mb-seg-966, region:Chr7-6Mb-seg-967, region:Chr7-6Mb-seg-968, region:Chr7-6Mb-seg-969, region:Chr7-6Mb-seg-970, region:Chr7-6Mb-seg-971, region:Chr7-6Mb-seg-972, region:Chr7-6Mb-seg-973, region:Chr7-6Mb-seg-974, region:Chr7-6Mb-seg-975, region:Chr7-6Mb-seg-976, region:Chr7-6Mb-seg-977, region:Chr7-6Mb-seg-978, region:Chr7-6Mb-seg-979, region:Chr7-6Mb-seg-980, region:Chr7-6Mb-seg-981, region:Chr7-6Mb-seg-982, region:Chr7-6Mb-seg-983, region:Chr7-6Mb-seg-984, region:Chr7-6Mb-seg-985, region:Chr7-6Mb-seg-986, region:Chr7-6Mb-seg-987, region:Chr7-6Mb-seg-988, region:Chr7-6Mb-seg-989, region:Chr7-6Mb-seg-990, region:Chr7-6Mb-seg-991, region:Chr7-6Mb-seg-992, region:Chr7-6Mb-seg-993, region:Chr7-6Mb-seg-994, region:Chr7-6Mb-seg-995, region:Chr7-6Mb-seg-996, region:Chr7-6Mb-seg-997, region:Chr7-6Mb-seg-998, region:Chr7-6Mb-seg-999, region:Chr7-6Mb-seg-1000, region:Chr7-6Mb-seg-1001, region:Chr7-6Mb-seg-1002, region:Chr7-6Mb-seg-1003, region:Chr7-6Mb-seg-1004, region:Chr7-6Mb-seg-1005, region:Chr7-6Mb-seg-1006, region:Chr7-6Mb-seg-1007, region:Chr7-6Mb-seg-1008, region:Chr7-6Mb-seg-1009, region:Chr7-6Mb-seg-1010, region:Chr7-6Mb-seg-1011, region:Chr7-6Mb-seg-1012, region:Chr7-6Mb-seg-1013, region:Chr7-6Mb-seg-1014, region:Chr7-6Mb-seg-1015, region:Chr7-6Mb-seg-1016, region:Chr7-6Mb-seg-1017, region:Chr7-6Mb-seg-1018, region:Chr7-6Mb-seg-1019, region:Chr7-6Mb-seg-1020, region:Chr7-6Mb-seg-1021, region:Chr7-6Mb-seg-1022, region:Chr7-6Mb-seg-1023, region:Chr7-6Mb-seg-1024, region:Chr7-6Mb-seg-1025, region:Chr7-6Mb-seg-1026, region:Chr7-6Mb-seg-1027, region:Chr7-6Mb-seg-1028, region:Chr7-6Mb-seg-1029, region:Chr7-6Mb-seg-1030, region:Chr7-6Mb-seg-1031, region:Chr7-6Mb-seg-1032, region:Chr7-6Mb-seg-1033, region:Chr7-6Mb-seg-1034, region:Chr7-6Mb-seg-1035, region:Chr7-6Mb-seg-1036, region:Chr7-6Mb-seg-1037, region:Chr7-6Mb-seg-1038, region:Chr7-6Mb-seg-1039, region:Chr7-6Mb-seg-1040, region:Chr7-6Mb-seg-1041, region:Chr7-6Mb-seg-1042, region:Chr7-6Mb-seg-1043, region:Chr7-6Mb-seg-1044, region:Chr7-6Mb-seg-1045, region:Chr7-6Mb-seg-1046, region:Chr7-6Mb-seg-1047, region:Chr7-6Mb-seg-1048, region:Chr7-6Mb-seg-1049, region:Chr7-6Mb-seg-1050, region:Chr7-6Mb-seg-1051, region:Chr7-6Mb-seg-1052, region:Chr7-6Mb-seg-1053, region:Chr7-6Mb-seg-1054, region:Chr7-6Mb-seg-1055, region:Chr7-6Mb-seg-1056, region:Chr7-6Mb-seg-1057, region:Chr7-6Mb-seg-1058, region:Chr7-6Mb-seg-1059, region:Chr7-6Mb-seg-1060, region:Chr7-6Mb-seg-1061, region:Chr7-6Mb-seg-1062, region:Chr7-6Mb-seg-1063, region:Chr7-6Mb-seg-1064, region:Chr7-6Mb-seg-1065, region:Chr7-6Mb-seg-1066, region:Chr7-6Mb-seg-1067, region:Chr7-6Mb-seg-1068, region:Chr7-6Mb-seg-1069, region:Chr7-6Mb-seg-1070, region:Chr7-6Mb-seg-1071, region:Chr7-6Mb-seg-1072, region:Chr7-6Mb-seg-1073, region:Chr7-6Mb-seg-1074, region:Chr7-6Mb-seg-1075, region:Chr7-6Mb-seg-1076, region:Chr7-6Mb-seg-1077, region:Chr7-6Mb-seg-1078, region:Chr7-6Mb-seg-1079, region:Chr7-6Mb-seg-1080, region:Chr7-6Mb-seg-1081, region:Chr7-6Mb-seg-1082, region:Chr7-6Mb-seg-1083, region:Chr7-6Mb-seg-1084, region:Chr7-6Mb-seg-1085, region:Chr7-6Mb-seg-1086, region:Chr7-6Mb-seg-1087, region:Chr7-6Mb-seg-1088, region:Chr7-6Mb-seg-1089, region:Chr7-6Mb-seg-1090, region:Chr7-6Mb-seg-1091, region:Chr7-6Mb-seg-1092, region:Chr7-6Mb-seg-1093, region:Chr7-6Mb-seg-1094, region:Chr7-6Mb-seg-1095, region:Chr7-6Mb-seg-1096, region:Chr7-6Mb-seg-1097, region:Chr7-6Mb-seg-1098, region:Chr7-6Mb-seg-1099, region:Chr7-6Mb-seg-1100, region:Chr7-6Mb-seg-1101, region:Chr7-6Mb-seg-1102, region:Chr7-6Mb-seg-1103, region:Chr7-6Mb-seg-1104, region:Chr7-6Mb-seg-1105, region:Chr7-6Mb-seg-1106, region:Chr7-6Mb-seg-1107, region:Chr7-6Mb-seg-1108, region:Chr7-6Mb-seg-1109, region:Chr7-6Mb-seg-1110, region:Chr7-6Mb-seg-1111, region:Chr7-6Mb-seg-1112, region:Chr7-6Mb-seg-1113, region:Chr7-6Mb-seg-1114, region:Chr7-6Mb-seg-1115, region:Chr7-6Mb-seg-1116, region:Chr7-6Mb-seg-1117, region:Chr7-6Mb-seg-1118, region:Chr7-6Mb-seg-1119, region:Chr7-6Mb-seg-1120, region:Chr7-6Mb-seg-1121, region:Chr7-6Mb-seg-1122, region:Chr7-6Mb-seg-1123, region:Chr7-6Mb-seg-1124, region:Chr7-6Mb-seg-1125, region:Chr7-6Mb-seg-1126, region:Chr7-6Mb-seg-1127, region:Chr7-6Mb-seg-1128, region:Chr7-6Mb-seg-1129, region:Chr7-6Mb-seg-1130, region:Chr7-6Mb-seg-1131, region:Chr7-6Mb-seg-1132, region:Chr7-6Mb-seg-1133, region:Chr7-6Mb-seg-1134, region:Chr7-6Mb-seg-1135, region:Chr7-6Mb-seg-1136, region:Chr7-6Mb-seg-1137, region:Chr7-6Mb-seg-1138, region:Chr7-6Mb-seg-1139, region:Chr7-6Mb-seg-1140, region:Chr7-6Mb-seg-1141, region:Chr7-6Mb-seg-1142, region:Chr7-6Mb-seg-1143, region:Chr7-6Mb-seg-1144, region:Chr7-6Mb-seg-1145, region:Chr7-6Mb-seg-1146, region:Chr7-6Mb-seg-1147, region:Chr7-6Mb-seg-1148, region:Chr7-6Mb-seg-1149, region:Chr7-6Mb-seg-1150, region:Chr7-6Mb-seg-1151, region:Chr7-6Mb-seg-1152, region:Chr7-6Mb-seg-1153, region:Chr7-6Mb-seg-1154, region:Chr7-6Mb-seg-1155, region:Chr7-6Mb-seg-1156, region:Chr7-6Mb-seg-1157, region:Chr7-6Mb-seg-1158, region:Chr7-6Mb-seg-1159, region:Chr7-6Mb-seg-1160, region:Chr7-6Mb-seg-1161, region:Chr7-6Mb-seg-1162, region:Chr7-6Mb-seg-1163, region:Chr7-6Mb-seg-1164, region:Chr7-6Mb-seg-1165, region:Chr7-6Mb-seg-1166, region:Chr7-6Mb-seg-1167, region:Chr7-6Mb-seg-1168, region:Chr7-6Mb-seg-1169, region:Chr7-6Mb-seg-1170, region:Chr7-6Mb-seg-1171, region:Chr7-6Mb-seg-1172, region:Chr7-6Mb-seg-1173, region:Chr7-6Mb-seg-1174, region:Chr7-6Mb-seg-1175, region:Chr7-6Mb-seg-1176, region:Chr7-6Mb-seg-1177, region:Chr7-6Mb-seg-1178, region:Chr7-6Mb-seg-1179, region:Chr7-6Mb-seg-1180, region:Chr7-6Mb-seg-1181, region:Chr7-6Mb-seg-1182, region:Chr7-6Mb-seg-1183, region:Chr7-6Mb-seg-1184, region:Chr7-6Mb-seg-1185, region:Chr7-6Mb-seg-1186, region:Chr7-6Mb-seg-1187, region:Chr7-6Mb-seg-1188, region:Chr7-6Mb-seg-1189, region:Chr7-6Mb-seg-1190, region:Chr7-6Mb-seg-1191, region:Chr7-6Mb-seg-1192, region:Chr7-6Mb-seg-1193, region:Chr7-6Mb-seg-1194, region:Chr7-6Mb-seg-1195, region:Chr7-6Mb-seg-1196, region:Chr7-6Mb-seg-1197, region:Chr7-6Mb-seg-1198, region:Chr7-6Mb-seg-1199, region:Chr7-6Mb-seg-1200, {'NDB_801': ['Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-546'], 'NDB_725': ['Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-526'], 'NDB_710': ['Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-466'], 'NDB_400': ['Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-547'], 'NDB_750': ['Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-527'], 'NDB_825': ['Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-467'], 'NDB_458': ['Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-548'], 'NDB_466': ['Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-528'], 'NDB_511': ['Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-468'], 'NDB_786': ['Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-549'], 'NDB_860': ['Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-529'], 'NDB_680': ['Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-469'], 'NDB_754': ['Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-550'], 'NDB_1020': ['Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-530'], 'NDB_681': ['Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-470'], 'NDB_461': ['Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-551'], 'NDB_631': ['Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-531'], 'NDB_532': ['Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-471'], 'NDB_921': ['Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-552'], 'NDB_887': ['Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-532'], 'NDB_506': ['Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-472'], 'NDB_835': ['Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-553'], 'NDB_846': ['Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-533'], 'NDB_864': ['Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-473'], 'NDB_941': ['Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-554'], 'NDB_385': ['Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-534'], 'NDB_838': ['Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-474'], 'NDB_396': ['Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-525'], 'NDB_660': ['Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-535'], 'NDB_706': ['Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-515'], 'NDB_784': ['Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-526'], 'NDB_759': ['Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-476'], 'NDB_718': ['Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-556'], 'NDB_971': ['Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-527'], 'NDB_847': ['Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-477'], 'NDB_431': ['Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-557'], 'NDB_768': ['Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-528'], 'NDB_554': ['Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-478'], 'NDB_621': ['Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-558'], 'NDB_901': ['Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-529'], 'NDB_375': ['Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-479'], 'NDB_529': ['Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-559'], 'NDB_443': ['Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-530'], 'NDB_508': ['Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-480'], 'NDB_878': ['Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-560'], 'NDB_438': ['Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-531'], 'NDB_332': ['Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-481'], 'NDB_741': ['Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-561', 'Chr7-6Mb-seg-561', 'Chr7-6Mb-seg-561', 'Chr7-6Mb-seg-561'], 'NDB_616': ['Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-532'], 'NDB_444': ['Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-482'], 'NDB_805': ['Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-562', 'Chr7-6Mb-seg-562', 'Chr7-6Mb-seg-562', 'Chr7-6Mb-seg-562'], 'NDB_347': ['Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-533'], 'NDB_730': ['Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-483'], 'NDB_896': ['Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-563', 'Chr7-6Mb-seg-563', 'Chr7-6Mb-seg-563', 'Chr7-6Mb-seg-563'], 'NDB_813': ['Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-534'], 'NDB_989': ['Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-484'], 'NDB_687': ['Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-564', 'Chr7-6Mb-seg-564', 'Chr7-6Mb-seg-564', 'Chr7-6Mb-seg-564'], 'NDB_514': ['Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-525'], 'NDB_356': ['Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-545', 'Chr7-6Mb-seg-545', 'Chr7-6Mb-seg-545', 'Chr7-6Mb-seg-545'], 'NDB_788': ['Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-485'], 'NDB_826': ['Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-546'], 'NDB_795': ['Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-526'], 'NDB_376': ['Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-85', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-536'], 'NDB_743': ['Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-547'], 'NDB_643': ['Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-527'], 'NDB_683': ['Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-86', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-537'], 'NDB_852': ['Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-548'], 'NDB_917': ['Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-528'], 'NDB_762': ['Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-87', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-538'], 'NDB_487': ['Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-549'], 'NDB_516': ['Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-529'], 'NDB_409': ['Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-88', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-539'], 'NDB_455': ['Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-550'], 'NDB_403': ['Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-530'], 'NDB_470': ['Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-89', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-540'], 'NDB_405': ['Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-551'], 'NDB_758': ['Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-531'], 'NDB_1035': ['Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-90', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-541'], 'NDB_1075': ['Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-552'], 'NDB_789': ['Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-532'], 'NDB_388': ['Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-91', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-542'], 'NDB_752': ['Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-553'], 'NDB_880': ['Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-533'], 'NDB_482': ['Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-92', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-543'], 'NDB_630': ['Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-554'], 'NDB_641': ['Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-534'], 'NDB_453': ['Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-93', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-544'], 'NDB_1066': ['Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-515'], 'NDB_665': ['Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-74', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-455'], 'NDB_445': ['Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-545', 'Chr7-6Mb-seg-545', 'Chr7-6Mb-seg-545', 'Chr7-6Mb-seg-545'], 'NDB_865': ['Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-486'], 'NDB_817': ['Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-556'], 'NDB_690': ['Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-516'], 'NDB_815': ['Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-487'], 'NDB_965': ['Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-557'], 'NDB_778': ['Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-517'], 'NDB_804': ['Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-488'], 'NDB_915': ['Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-558'], 'NDB_746': ['Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-518'], 'NDB_571': ['Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-489'], 'NDB_448': ['Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-559'], 'NDB_957': ['Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-519'], 'NDB_737': ['Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-490'], 'NDB_698': ['Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-560'], 'NDB_391': ['Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-520'], 'NDB_663': ['Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-491'], 'NDB_954': ['Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-561', 'Chr7-6Mb-seg-561', 'Chr7-6Mb-seg-561', 'Chr7-6Mb-seg-561'], 'NDB_398': ['Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-521', 'Chr7-6Mb-seg-521', 'Chr7-6Mb-seg-521', 'Chr7-6Mb-seg-521'], 'NDB_1060': ['Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-492'], 'NDB_427': ['Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-562', 'Chr7-6Mb-seg-562', 'Chr7-6Mb-seg-562', 'Chr7-6Mb-seg-562'], 'NDB_780': ['Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-522', 'Chr7-6Mb-seg-522', 'Chr7-6Mb-seg-522', 'Chr7-6Mb-seg-522'], 'NDB_446': ['Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-493'], 'NDB_473': ['Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-563', 'Chr7-6Mb-seg-563', 'Chr7-6Mb-seg-563', 'Chr7-6Mb-seg-563'], 'NDB_397': ['Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-523', 'Chr7-6Mb-seg-523', 'Chr7-6Mb-seg-523', 'Chr7-6Mb-seg-523'], 'NDB_816': ['Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-494'], 'NDB_1044': ['Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-564', 'Chr7-6Mb-seg-564', 'Chr7-6Mb-seg-564', 'Chr7-6Mb-seg-564'], 'NDB_647': ['Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-524'], 'NDB_498': ['Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-94', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-495'], 'NDB_1023': ['Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-545', 'Chr7-6Mb-seg-545', 'Chr7-6Mb-seg-545', 'Chr7-6Mb-seg-545'], 'NDB_637': ['Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-555'], 'NDB_578': ['Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-516'], 'NDB_713': ['Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-486'], 'NDB_777': ['Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-55', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-546'], 'NDB_936': ['Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-517'], 'NDB_456': ['Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-487'], 'NDB_760': ['Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-56', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-547'], 'NDB_961': ['Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-518'], 'NDB_661': ['Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-488'], 'NDB_728': ['Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-57', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-548'], 'NDB_773': ['Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-519'], 'NDB_812': ['Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-489'], 'NDB_432': ['Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-58', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-549'], 'NDB_489': ['Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-520'], 'NDB_417': ['Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-490'], 'NDB_1027': ['Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-59', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-550'], 'NDB_715': ['Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-521', 'Chr7-6Mb-seg-521', 'Chr7-6Mb-seg-521', 'Chr7-6Mb-seg-521'], 'NDB_883': ['Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-491'], 'NDB_536': ['Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-60', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-551'], 'NDB_467': ['Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-522', 'Chr7-6Mb-seg-522', 'Chr7-6Mb-seg-522', 'Chr7-6Mb-seg-522'], 'NDB_335': ['Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-492'], 'NDB_390': ['Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-61', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-552'], 'NDB_507': ['Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-523', 'Chr7-6Mb-seg-523', 'Chr7-6Mb-seg-523', 'Chr7-6Mb-seg-523'], 'NDB_747': ['Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-493'], 'NDB_832': ['Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-62', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-553'], 'NDB_721': ['Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-524'], 'NDB_853': ['Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-494'], 'NDB_899': ['Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-63', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-554'], 'NDB_994': ['Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-535'], 'NDB_734': ['Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-555'], 'NDB_828': ['Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-64', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-525'], 'NDB_807': ['Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-65', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-536'], 'NDB_790': ['Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-66', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-537'], 'NDB_974': ['Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-67', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-538'], 'NDB_858': ['Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-68', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-539'], 'NDB_856': ['Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-540'], 'NDB_653': ['Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-541'], 'NDB_501': ['Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-71', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-542'], 'NDB_433': ['Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-72', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-543'], 'NDB_452': ['Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-73', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-544'], 'NDB_755': ['Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-75', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-536'], 'NDB_699': ['Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-76', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-537'], 'NDB_844': ['Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-77', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-538'], 'NDB_875': ['Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-78', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-539'], 'NDB_465': ['Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-79', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-540'], 'NDB_634': ['Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-80', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-541'], 'NDB_905': ['Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-81', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-542'], 'NDB_615': ['Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-82', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-543'], 'NDB_946': ['Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-83', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-544'], 'NDB_881': ['Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-555'], 'NDB_935': ['Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-515'], 'NDB_711': ['Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-84', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-535'], 'NDB_533': ['Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-556'], 'NDB_652': ['Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-95', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-516'], 'NDB_738': ['Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-557'], 'NDB_464': ['Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-96', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-517'], 'NDB_568': ['Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-558'], 'NDB_387': ['Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-97', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-518'], 'NDB_716': ['Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-559'], 'NDB_868': ['Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-98', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-519'], 'NDB_657': ['Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-560'], 'NDB_863': ['Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-99', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-520'], 'NDB_1033': ['Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-561', 'Chr7-6Mb-seg-561', 'Chr7-6Mb-seg-561', 'Chr7-6Mb-seg-561'], 'NDB_771': ['Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-521', 'Chr7-6Mb-seg-521', 'Chr7-6Mb-seg-521', 'Chr7-6Mb-seg-521'], 'NDB_866': ['Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-562', 'Chr7-6Mb-seg-562', 'Chr7-6Mb-seg-562', 'Chr7-6Mb-seg-562'], 'NDB_619': ['Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-522', 'Chr7-6Mb-seg-522', 'Chr7-6Mb-seg-522', 'Chr7-6Mb-seg-522'], 'NDB_1047': ['Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-563', 'Chr7-6Mb-seg-563', 'Chr7-6Mb-seg-563', 'Chr7-6Mb-seg-563'], 'NDB_539': ['Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-523', 'Chr7-6Mb-seg-523', 'Chr7-6Mb-seg-523', 'Chr7-6Mb-seg-523'], 'NDB_505': ['Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-564', 'Chr7-6Mb-seg-564', 'Chr7-6Mb-seg-564', 'Chr7-6Mb-seg-564'], 'NDB_618': ['Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-524'], 'NDB_889': ['Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-505'], 'NDB_694': ['Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-1183'], 'NDB_879': ['Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-1163'], 'NDB_705': ['Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1103'], 'NDB_929': ['Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-1184'], 'NDB_472': ['Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-1164'], 'NDB_622': ['Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1104'], 'NDB_513': ['Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-1185'], 'NDB_359': ['Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-1165'], 'NDB_581': ['Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1105'], 'NDB_910': ['Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-1186'], 'NDB_606': ['Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-1166'], 'NDB_702': ['Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1106'], 'NDB_926': ['Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-1187'], 'NDB_538': ['Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-1167'], 'NDB_1036': ['Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1107'], 'NDB_654': ['Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-1188'], 'NDB_791': ['Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-1168'], 'NDB_776': ['Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1108'], 'NDB_928': ['Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-1189'], 'NDB_645': ['Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-1169'], 'NDB_732': ['Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1109'], 'NDB_806': ['Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-1190'], 'NDB_1051': ['Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-1170'], 'NDB_922': ['Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1110'], 'NDB_480': ['Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-1161'], 'NDB_708': ['Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-1171'], 'NDB_700': ['Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-1151'], 'NDB_604': ['Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-1162'], 'NDB_931': ['Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-1172'], 'NDB_908': ['Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-1152'], 'NDB_587': ['Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-1163'], 'NDB_862': ['Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1113'], 'NDB_977': ['Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-1193'], 'NDB_594': ['Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-1164'], 'NDB_611': ['Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1114'], 'NDB_840': ['Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-1194'], 'NDB_517': ['Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-1165'], 'NDB_519': ['Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1115'], 'NDB_577': ['Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-1195'], 'NDB_350': ['Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-1166'], 'NDB_556': ['Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1116'], 'NDB_494': ['Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-1196'], 'NDB_591': ['Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-1167'], 'NDB_479': ['Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1117'], 'NDB_1056': ['Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-1197'], 'NDB_589': ['Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-1168'], 'NDB_486': ['Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1118'], 'NDB_904': ['Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-1198'], 'NDB_629': ['Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-1169'], 'NDB_601': ['Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1119'], 'NDB_488': ['Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-1199'], 'NDB_960': ['Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-1170'], 'NDB_416': ['Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1120'], 'NDB_429': ['Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-1200'], 'NDB_460': ['Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-1161'], 'NDB_635': ['Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-1181'], 'NDB_500': ['Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1121'], 'NDB_593': ['Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-1162'], 'NDB_474': ['Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-1182'], 'NDB_471': ['Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1122'], 'NDB_551': ['Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-1183'], 'NDB_628': ['Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-1163'], 'NDB_557': ['Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-1173'], 'NDB_1053': ['Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-1184'], 'NDB_341': ['Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-1164'], 'NDB_372': ['Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-1174'], 'NDB_544': ['Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-1185'], 'NDB_903': ['Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-1165'], 'NDB_925': ['Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-1175'], 'NDB_371': ['Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-1186'], 'NDB_934': ['Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-1166'], 'NDB_476': ['Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-1176'], 'NDB_483': ['Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-1187'], 'NDB_608': ['Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-1167'], 'NDB_495': ['Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-1177'], 'NDB_703': ['Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-1188'], 'NDB_492': ['Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-1168'], 'NDB_1063': ['Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-1178'], 'NDB_785': ['Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-1189'], 'NDB_1048': ['Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-1169'], 'NDB_920': ['Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-1179'], 'NDB_561': ['Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-1190'], 'NDB_503': ['Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-1170'], 'NDB_876': ['Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-1180'], 'NDB_592': ['Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-1151'], 'NDB_1013': ['Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1091'], 'NDB_895': ['Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-1181'], 'NDB_845': ['Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-1152'], 'NDB_537': ['Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1092'], 'NDB_449': ['Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-1182'], 'NDB_491': ['Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1123'], 'NDB_779': ['Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-1193'], 'NDB_379': ['Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-1153'], 'NDB_627': ['Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1124'], 'NDB_933': ['Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-1194'], 'NDB_386': ['Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-1154'], 'NDB_712': ['Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1125'], 'NDB_625': ['Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-1195'], 'NDB_441': ['Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-1155'], 'NDB_428': ['Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1126'], 'NDB_818': ['Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-1196'], 'NDB_439': ['Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-1156'], 'NDB_894': ['Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1127'], 'NDB_510': ['Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-1197'], 'NDB_809': ['Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-1157'], 'NDB_955': ['Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1128'], 'NDB_1042': ['Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-1198'], 'NDB_666': ['Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-1158'], 'NDB_362': ['Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1129'], 'NDB_902': ['Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-1199'], 'NDB_976': ['Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-1159'], 'NDB_423': ['Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1130'], 'NDB_393': ['Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-1200'], 'NDB_383': ['Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-1160'], 'NDB_783': ['Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1131'], 'NDB_573': ['Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-1181'], 'NDB_871': ['Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-981', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-1191'], 'NDB_829': ['Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1132'], 'NDB_436': ['Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-1182'], 'NDB_689': ['Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-982', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-1192'], 'NDB_1014': ['Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-1153'], 'NDB_459': ['Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-953', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1123'], 'NDB_421': ['Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-963', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-1183'], 'NDB_958': ['Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-1154'], 'NDB_418': ['Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-954', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1124'], 'NDB_836': ['Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-964', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-1184'], 'NDB_392': ['Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-1155'], 'NDB_614': ['Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-955', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1125'], 'NDB_450': ['Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-965', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-1185'], 'NDB_939': ['Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-1156'], 'NDB_798': ['Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-956', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1126'], 'NDB_481': ['Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-966', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-1186'], 'NDB_547': ['Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-1157'], 'NDB_736': ['Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-957', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1127'], 'NDB_872': ['Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-967', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-1187'], 'NDB_644': ['Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-1158'], 'NDB_368': ['Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-958', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1128'], 'NDB_693': ['Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-968', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-1188'], 'NDB_522': ['Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-1159'], 'NDB_729': ['Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-959', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1129'], 'NDB_649': ['Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-969', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-1189'], 'NDB_1039': ['Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-1160'], 'NDB_562': ['Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-960', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1130'], 'NDB_413': ['Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-970', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-1190'], 'NDB_772': ['Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-1171'], 'NDB_995': ['Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-991', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-1191'], 'NDB_774': ['Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-1161'], 'NDB_869': ['Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-1172'], 'NDB_381': ['Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-992', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-1192'], 'NDB_496': ['Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-1162'], 'NDB_751': ['Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-983', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-1173'], 'NDB_1028': ['Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-984', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-1174'], 'NDB_384': ['Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-985', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-1175'], 'NDB_775': ['Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-986', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-1176'], 'NDB_638': ['Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-987', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-1177'], 'NDB_948': ['Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-988', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-1178'], 'NDB_691': ['Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-989', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-1179'], 'NDB_650': ['Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-990', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-1180'], 'NDB_819': ['Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-1173'], 'NDB_742': ['Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-1174'], 'NDB_824': ['Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-1175'], 'NDB_402': ['Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-1176'], 'NDB_493': ['Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-1177'], 'NDB_563': ['Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-1178'], 'NDB_891': ['Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-1179'], 'NDB_1072': ['Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-1180'], 'NDB_407': ['Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-1191'], 'NDB_914': ['Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-935', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-1151'], 'NDB_765': ['Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-971', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-1171'], 'NDB_648': ['Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-1192'], 'NDB_867': ['Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-937', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-1152'], 'NDB_739': ['Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-972', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-1172'], 'NDB_951': ['Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-973', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-1193'], 'NDB_401': ['Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-993', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-1153'], 'NDB_526': ['Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-974', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-1194'], 'NDB_735': ['Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-994', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-1154'], 'NDB_395': ['Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-975', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-1195'], 'NDB_940': ['Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-995', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-1155'], 'NDB_585': ['Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-976', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-1196'], 'NDB_365': ['Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-996', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-1156'], 'NDB_709': ['Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-977', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-1197'], 'NDB_600': ['Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-997', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-1157'], 'NDB_707': ['Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-978', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-1198'], 'NDB_727': ['Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-998', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-1158'], 'NDB_873': ['Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-979', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-1199'], 'NDB_329': ['Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-999', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-1159'], 'NDB_796': ['Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-980', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-1200'], 'NDB_882': ['Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-934', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-1160'], 'NDB_802': ['Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1141'], 'NDB_740': ['Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1142']}\n",
      " > Done.\n",
      "-- inverting region_to_readout dict\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NDB_1013 -> ['Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-971']\n",
      "NDB_1014 -> ['Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-983']\n",
      "NDB_1020 -> ['Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-69', 'Chr7-6Mb-seg-7']\n",
      "NDB_1023 -> ['Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-545']\n",
      "NDB_1027 -> ['Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-59']\n",
      "NDB_1028 -> ['Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-939', 'Chr7-6Mb-seg-984']\n",
      "NDB_1033 -> ['Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-561']\n",
      "NDB_1035 -> ['Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-90']\n",
      "NDB_1036 -> ['Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-945']\n",
      "NDB_1039 -> ['Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-990']\n",
      "NDB_1042 -> ['Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-932']\n",
      "NDB_1044 -> ['Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-53', 'Chr7-6Mb-seg-564']\n",
      "NDB_1047 -> ['Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-563']\n",
      "NDB_1048 -> ['Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-947']\n",
      "NDB_1051 -> ['Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-960']\n",
      "NDB_1053 -> ['Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-924']\n",
      "NDB_1056 -> ['Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-997']\n",
      "NDB_1060 -> ['Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-51']\n",
      "NDB_1063 -> ['Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-978']\n",
      "NDB_1066 -> ['Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-515']\n",
      "NDB_1072 -> ['Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-833']\n",
      "NDB_1075 -> ['Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-81']\n",
      "NDB_329 -> ['Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-933', 'Chr7-6Mb-seg-999']\n",
      "NDB_332 -> ['Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-90']\n",
      "NDB_335 -> ['Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-61']\n",
      "NDB_341 -> ['Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-939']\n",
      "NDB_347 -> ['Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-82']\n",
      "NDB_350 -> ['Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-966']\n",
      "NDB_356 -> ['Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-545', 'Chr7-6Mb-seg-74']\n",
      "NDB_359 -> ['Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-955']\n",
      "NDB_362 -> ['Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-969']\n",
      "NDB_365 -> ['Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-929', 'Chr7-6Mb-seg-996']\n",
      "NDB_368 -> ['Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-738', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-958']\n",
      "NDB_371 -> ['Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-929']\n",
      "NDB_372 -> ['Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-586', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-974']\n",
      "NDB_375 -> ['Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-88']\n",
      "NDB_376 -> ['Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-85']\n",
      "NDB_379 -> ['Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-973']\n",
      "NDB_381 -> ['Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-992']\n",
      "NDB_383 -> ['Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-772', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-980']\n",
      "NDB_384 -> ['Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-942', 'Chr7-6Mb-seg-985']\n",
      "NDB_385 -> ['Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-73']\n",
      "NDB_386 -> ['Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-974']\n",
      "NDB_387 -> ['Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-97']\n",
      "NDB_388 -> ['Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-91']\n",
      "NDB_390 -> ['Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-61']\n",
      "NDB_391 -> ['Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-520']\n",
      "NDB_392 -> ['Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-985']\n",
      "NDB_393 -> ['Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-934']\n",
      "NDB_395 -> ['Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-719', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-975']\n",
      "NDB_396 -> ['Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-74']\n",
      "NDB_397 -> ['Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-523']\n",
      "NDB_398 -> ['Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-341', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-521']\n",
      "NDB_400 -> ['Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-86']\n",
      "NDB_401 -> ['Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1153', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-923', 'Chr7-6Mb-seg-993']\n",
      "NDB_402 -> ['Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-628', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-996']\n",
      "NDB_403 -> ['Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-400', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-99']\n",
      "NDB_405 -> ['Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-221', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-80']\n",
      "NDB_407 -> ['Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-890']\n",
      "NDB_409 -> ['Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-88']\n",
      "NDB_413 -> ['Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-889', 'Chr7-6Mb-seg-970']\n",
      "NDB_416 -> ['Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1050', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-960']\n",
      "NDB_417 -> ['Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-59']\n",
      "NDB_418 -> ['Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-954']\n",
      "NDB_421 -> ['Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-963']\n",
      "NDB_423 -> ['Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-602', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-970']\n",
      "NDB_427 -> ['Chr7-6Mb-seg-151', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-562']\n",
      "NDB_428 -> ['Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-966']\n",
      "NDB_429 -> ['Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-676', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-858']\n",
      "NDB_431 -> ['Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-66']\n",
      "NDB_432 -> ['Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-58']\n",
      "NDB_433 -> ['Chr7-6Mb-seg-132', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-72']\n",
      "NDB_436 -> ['Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-937']\n",
      "NDB_438 -> ['Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-80']\n",
      "NDB_439 -> ['Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-976']\n",
      "NDB_441 -> ['Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-975']\n",
      "NDB_443 -> ['Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-530', 'Chr7-6Mb-seg-79']\n",
      "NDB_444 -> ['Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-91']\n",
      "NDB_445 -> ['Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-545']\n",
      "NDB_446 -> ['Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-52']\n",
      "NDB_448 -> ['Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-559']\n",
      "NDB_449 -> ['Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-904', 'Chr7-6Mb-seg-962']\n",
      "NDB_450 -> ['Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-965']\n",
      "NDB_452 -> ['Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-73']\n",
      "NDB_453 -> ['Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-93']\n",
      "NDB_455 -> ['Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-79']\n",
      "NDB_456 -> ['Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-56']\n",
      "NDB_458 -> ['Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-87']\n",
      "NDB_459 -> ['Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-605', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-953']\n",
      "NDB_460 -> ['Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-902']\n",
      "NDB_461 -> ['Chr7-6Mb-seg-171', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-9', 'Chr7-6Mb-seg-90']\n",
      "NDB_464 -> ['Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-96']\n",
      "NDB_465 -> ['Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-180', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-440', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-79']\n",
      "NDB_466 -> ['Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-67']\n",
      "NDB_467 -> ['Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-522', 'Chr7-6Mb-seg-61']\n",
      "NDB_470 -> ['Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-39', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-89']\n",
      "NDB_471 -> ['Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-982']\n",
      "NDB_472 -> ['Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-954']\n",
      "NDB_473 -> ['Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-52', 'Chr7-6Mb-seg-563']\n",
      "NDB_474 -> ['Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1182', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-952']\n",
      "NDB_476 -> ['Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-976']\n",
      "NDB_479 -> ['Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-957']\n",
      "NDB_480 -> ['Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-991']\n",
      "NDB_481 -> ['Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-966']\n",
      "NDB_482 -> ['Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-343', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-92']\n",
      "NDB_483 -> ['Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-931']\n",
      "NDB_486 -> ['Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-958']\n",
      "NDB_487 -> ['Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-78']\n",
      "NDB_488 -> ['Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-999']\n",
      "NDB_489 -> ['Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-280', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-59']\n",
      "NDB_491 -> ['Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-963']\n",
      "NDB_492 -> ['Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-946']\n",
      "NDB_493 -> ['Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-997']\n",
      "NDB_494 -> ['Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-1146', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-996']\n",
      "NDB_495 -> ['Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-977']\n",
      "NDB_496 -> ['Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-972']\n",
      "NDB_498 -> ['Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-94']\n",
      "NDB_500 -> ['Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-890', 'Chr7-6Mb-seg-981']\n",
      "NDB_501 -> ['Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-382', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-492', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-71']\n",
      "NDB_503 -> ['Chr7-6Mb-seg-1040', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-750', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-847', 'Chr7-6Mb-seg-950']\n",
      "NDB_505 -> ['Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-564']\n",
      "NDB_506 -> ['Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-192', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-472']\n",
      "NDB_507 -> ['Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-283', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-523', 'Chr7-6Mb-seg-62']\n",
      "NDB_508 -> ['Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-220', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-320', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-410', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-89']\n",
      "NDB_510 -> ['Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-931']\n",
      "NDB_511 -> ['Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-3', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-468']\n",
      "NDB_513 -> ['Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-985']\n",
      "NDB_514 -> ['Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-395', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-525']\n",
      "NDB_516 -> ['Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-38', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-98']\n",
      "NDB_517 -> ['Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-705', 'Chr7-6Mb-seg-755', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-965']\n",
      "NDB_519 -> ['Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1115', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-955']\n",
      "NDB_522 -> ['Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-675', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-989']\n",
      "NDB_526 -> ['Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1084', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-974']\n",
      "NDB_529 -> ['Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-509', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-68']\n",
      "NDB_532 -> ['Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-9']\n",
      "NDB_533 -> ['Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-95']\n",
      "NDB_536 -> ['Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-481', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-551', 'Chr7-6Mb-seg-60']\n",
      "NDB_537 -> ['Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-788', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-972']\n",
      "NDB_538 -> ['Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-957']\n",
      "NDB_539 -> ['Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-523']\n",
      "NDB_544 -> ['Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1085', 'Chr7-6Mb-seg-1185', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-814', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-926']\n",
      "NDB_547 -> ['Chr7-6Mb-seg-1047', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-781', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-987']\n",
      "NDB_551 -> ['Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-923']\n",
      "NDB_554 -> ['Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-258', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-87']\n",
      "NDB_556 -> ['Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-578', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-816', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-956']\n",
      "NDB_557 -> ['Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-973']\n",
      "NDB_561 -> ['Chr7-6Mb-seg-1010', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-820', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-934']\n",
      "NDB_562 -> ['Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1080', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-612', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-858', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-960']\n",
      "NDB_563 -> ['Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1078', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-998']\n",
      "NDB_568 -> ['Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-97']\n",
      "NDB_571 -> ['Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-158', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-489']\n",
      "NDB_573 -> ['Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-935']\n",
      "NDB_577 -> ['Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-577', 'Chr7-6Mb-seg-617', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-995']\n",
      "NDB_578 -> ['Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-55']\n",
      "NDB_581 -> ['Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1105', 'Chr7-6Mb-seg-567', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-778', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-942']\n",
      "NDB_585 -> ['Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1116', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-828', 'Chr7-6Mb-seg-976']\n",
      "NDB_587 -> ['Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-892', 'Chr7-6Mb-seg-963']\n",
      "NDB_589 -> ['Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-968']\n",
      "NDB_591 -> ['Chr7-6Mb-seg-1067', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-579', 'Chr7-6Mb-seg-629', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-896', 'Chr7-6Mb-seg-967']\n",
      "NDB_592 -> ['Chr7-6Mb-seg-1031', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-981']\n",
      "NDB_593 -> ['Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-584', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-904']\n",
      "NDB_594 -> ['Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1164', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-704', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-964']\n",
      "NDB_600 -> ['Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1097', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-866', 'Chr7-6Mb-seg-931', 'Chr7-6Mb-seg-997']\n",
      "NDB_601 -> ['Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-749', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-959']\n",
      "NDB_604 -> ['Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1162', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-624', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-891', 'Chr7-6Mb-seg-992']\n",
      "NDB_606 -> ['Chr7-6Mb-seg-1006', 'Chr7-6Mb-seg-1096', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-768', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-956']\n",
      "NDB_608 -> ['Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1167', 'Chr7-6Mb-seg-589', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-842', 'Chr7-6Mb-seg-945']\n",
      "NDB_611 -> ['Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-813', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-954']\n",
      "NDB_614 -> ['Chr7-6Mb-seg-1025', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-607', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-853', 'Chr7-6Mb-seg-911', 'Chr7-6Mb-seg-955']\n",
      "NDB_615 -> ['Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-233', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-543', 'Chr7-6Mb-seg-82']\n",
      "NDB_616 -> ['Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-282', 'Chr7-6Mb-seg-332', 'Chr7-6Mb-seg-432', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-81']\n",
      "NDB_618 -> ['Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-464', 'Chr7-6Mb-seg-524']\n",
      "NDB_619 -> ['Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-202', 'Chr7-6Mb-seg-262', 'Chr7-6Mb-seg-302', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-522']\n",
      "NDB_621 -> ['Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-558', 'Chr7-6Mb-seg-67']\n",
      "NDB_622 -> ['Chr7-6Mb-seg-1024', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-939']\n",
      "NDB_625 -> ['Chr7-6Mb-seg-1015', 'Chr7-6Mb-seg-1065', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1195', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-880', 'Chr7-6Mb-seg-926']\n",
      "NDB_627 -> ['Chr7-6Mb-seg-1004', 'Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1124', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-744', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-964']\n",
      "NDB_628 -> ['Chr7-6Mb-seg-1033', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-585', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-938']\n",
      "NDB_629 -> ['Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-581', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-969']\n",
      "NDB_630 -> ['Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-304', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-454', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-83']\n",
      "NDB_631 -> ['Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-531', 'Chr7-6Mb-seg-70', 'Chr7-6Mb-seg-9']\n",
      "NDB_634 -> ['Chr7-6Mb-seg-120', 'Chr7-6Mb-seg-181', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-80']\n",
      "NDB_635 -> ['Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-583', 'Chr7-6Mb-seg-623', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-951']\n",
      "NDB_637 -> ['Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-54', 'Chr7-6Mb-seg-555']\n",
      "NDB_638 -> ['Chr7-6Mb-seg-1017', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-1177', 'Chr7-6Mb-seg-619', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-945', 'Chr7-6Mb-seg-987']\n",
      "NDB_641 -> ['Chr7-6Mb-seg-103', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-43', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-534']\n",
      "NDB_643 -> ['Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-207', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-96']\n",
      "NDB_644 -> ['Chr7-6Mb-seg-1048', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-845', 'Chr7-6Mb-seg-897', 'Chr7-6Mb-seg-988']\n",
      "NDB_645 -> ['Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1169', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-959']\n",
      "NDB_647 -> ['Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-344', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-53']\n",
      "NDB_648 -> ['Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1082', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-891']\n",
      "NDB_649 -> ['Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-969']\n",
      "NDB_650 -> ['Chr7-6Mb-seg-1020', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1130', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-622', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-950', 'Chr7-6Mb-seg-990']\n",
      "NDB_652 -> ['Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-516', 'Chr7-6Mb-seg-95']\n",
      "NDB_653 -> ['Chr7-6Mb-seg-130', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-451', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-541', 'Chr7-6Mb-seg-70']\n",
      "NDB_654 -> ['Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1138', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-642', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-988']\n",
      "NDB_657 -> ['Chr7-6Mb-seg-139', 'Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-340', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-480', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-99']\n",
      "NDB_660 -> ['Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-325', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-535']\n",
      "NDB_661 -> ['Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-318', 'Chr7-6Mb-seg-388', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-57']\n",
      "NDB_663 -> ['Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-231', 'Chr7-6Mb-seg-331', 'Chr7-6Mb-seg-371', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-50']\n",
      "NDB_665 -> ['Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-265', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-44', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-74']\n",
      "NDB_666 -> ['Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-916', 'Chr7-6Mb-seg-978']\n",
      "NDB_680 -> ['Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-469']\n",
      "NDB_681 -> ['Chr7-6Mb-seg-119', 'Chr7-6Mb-seg-190', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-390', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-7']\n",
      "NDB_683 -> ['Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-86']\n",
      "NDB_687 -> ['Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-474', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-564', 'Chr7-6Mb-seg-73']\n",
      "NDB_689 -> ['Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1192', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-952', 'Chr7-6Mb-seg-982']\n",
      "NDB_690 -> ['Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-336', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-516']\n",
      "NDB_691 -> ['Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-621', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-796', 'Chr7-6Mb-seg-947', 'Chr7-6Mb-seg-989']\n",
      "NDB_693 -> ['Chr7-6Mb-seg-1038', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-610', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-968']\n",
      "NDB_694 -> ['Chr7-6Mb-seg-1053', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1183', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-906', 'Chr7-6Mb-seg-983']\n",
      "NDB_698 -> ['Chr7-6Mb-seg-149', 'Chr7-6Mb-seg-240', 'Chr7-6Mb-seg-270', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-430', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-560']\n",
      "NDB_699 -> ['Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-357', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-437', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-76']\n",
      "NDB_700 -> ['Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-902']\n",
      "NDB_702 -> ['Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1106', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-683', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-865', 'Chr7-6Mb-seg-944']\n",
      "NDB_703 -> ['Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1188', 'Chr7-6Mb-seg-590', 'Chr7-6Mb-seg-630', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-818', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-932']\n",
      "NDB_705 -> ['Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-776', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-938']\n",
      "NDB_706 -> ['Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-14', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-285', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-94']\n",
      "NDB_707 -> ['Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1118', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-758', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-978']\n",
      "NDB_708 -> ['Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1121', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-573', 'Chr7-6Mb-seg-727', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-961']\n",
      "NDB_709 -> ['Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1087', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1197', 'Chr7-6Mb-seg-660', 'Chr7-6Mb-seg-721', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-977']\n",
      "NDB_710 -> ['Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-466']\n",
      "NDB_711 -> ['Chr7-6Mb-seg-114', 'Chr7-6Mb-seg-154', 'Chr7-6Mb-seg-205', 'Chr7-6Mb-seg-235', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-84']\n",
      "NDB_712 -> ['Chr7-6Mb-seg-1005', 'Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1125', 'Chr7-6Mb-seg-597', 'Chr7-6Mb-seg-670', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-965']\n",
      "NDB_713 -> ['Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-386', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-55']\n",
      "NDB_715 -> ['Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-140', 'Chr7-6Mb-seg-191', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-281', 'Chr7-6Mb-seg-351', 'Chr7-6Mb-seg-411', 'Chr7-6Mb-seg-521', 'Chr7-6Mb-seg-60']\n",
      "NDB_716 -> ['Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-479', 'Chr7-6Mb-seg-559', 'Chr7-6Mb-seg-98']\n",
      "NDB_718 -> ['Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-466', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-556', 'Chr7-6Mb-seg-65']\n",
      "NDB_721 -> ['Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-143', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-244', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-524', 'Chr7-6Mb-seg-63']\n",
      "NDB_725 -> ['Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-135', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-456', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-65']\n",
      "NDB_727 -> ['Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1158', 'Chr7-6Mb-seg-661', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-932', 'Chr7-6Mb-seg-998']\n",
      "NDB_728 -> ['Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-478', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-57']\n",
      "NDB_729 -> ['Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1129', 'Chr7-6Mb-seg-611', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-959']\n",
      "NDB_730 -> ['Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-383', 'Chr7-6Mb-seg-413', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-92']\n",
      "NDB_732 -> ['Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-869', 'Chr7-6Mb-seg-947']\n",
      "NDB_734 -> ['Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-245', 'Chr7-6Mb-seg-295', 'Chr7-6Mb-seg-355', 'Chr7-6Mb-seg-435', 'Chr7-6Mb-seg-505', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-64']\n",
      "NDB_735 -> ['Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1094', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-650', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-863', 'Chr7-6Mb-seg-924', 'Chr7-6Mb-seg-994']\n",
      "NDB_736 -> ['Chr7-6Mb-seg-1027', 'Chr7-6Mb-seg-1077', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-735', 'Chr7-6Mb-seg-855', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-957']\n",
      "NDB_737 -> ['Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-159', 'Chr7-6Mb-seg-230', 'Chr7-6Mb-seg-330', 'Chr7-6Mb-seg-370', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-49', 'Chr7-6Mb-seg-490']\n",
      "NDB_738 -> ['Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-557', 'Chr7-6Mb-seg-96']\n",
      "NDB_739 -> ['Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-679', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-835', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-972']\n",
      "NDB_740 -> ['Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1062', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1142', 'Chr7-6Mb-seg-665', 'Chr7-6Mb-seg-752', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-952']\n",
      "NDB_741 -> ['Chr7-6Mb-seg-110', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-22', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-471', 'Chr7-6Mb-seg-511', 'Chr7-6Mb-seg-561', 'Chr7-6Mb-seg-70']\n",
      "NDB_742 -> ['Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1074', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1174', 'Chr7-6Mb-seg-626', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-766', 'Chr7-6Mb-seg-824', 'Chr7-6Mb-seg-994']\n",
      "NDB_743 -> ['Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-247', 'Chr7-6Mb-seg-29', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-76']\n",
      "NDB_746 -> ['Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-288', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-468', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-518']\n",
      "NDB_747 -> ['Chr7-6Mb-seg-152', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-493', 'Chr7-6Mb-seg-62']\n",
      "NDB_750 -> ['Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-457', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-66']\n",
      "NDB_751 -> ['Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1123', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-703', 'Chr7-6Mb-seg-790', 'Chr7-6Mb-seg-938', 'Chr7-6Mb-seg-983']\n",
      "NDB_752 -> ['Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-223', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-303', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-453', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-82']\n",
      "NDB_754 -> ['Chr7-6Mb-seg-170', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-290', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-500', 'Chr7-6Mb-seg-550', 'Chr7-6Mb-seg-7', 'Chr7-6Mb-seg-89']\n",
      "NDB_755 -> ['Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-176', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-356', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-75']\n",
      "NDB_758 -> ['Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-161', 'Chr7-6Mb-seg-211', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-311', 'Chr7-6Mb-seg-40', 'Chr7-6Mb-seg-401', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-531']\n",
      "NDB_759 -> ['Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-256', 'Chr7-6Mb-seg-316', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-406', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-85']\n",
      "NDB_760 -> ['Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-507', 'Chr7-6Mb-seg-547', 'Chr7-6Mb-seg-56']\n",
      "NDB_762 -> ['Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-338', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-458', 'Chr7-6Mb-seg-508', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-87']\n",
      "NDB_765 -> ['Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-677', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-798', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-971']\n",
      "NDB_768 -> ['Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-168', 'Chr7-6Mb-seg-19', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-77']\n",
      "NDB_771 -> ['Chr7-6Mb-seg-100', 'Chr7-6Mb-seg-201', 'Chr7-6Mb-seg-261', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-361', 'Chr7-6Mb-seg-421', 'Chr7-6Mb-seg-461', 'Chr7-6Mb-seg-521']\n",
      "NDB_772 -> ['Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1091', 'Chr7-6Mb-seg-1171', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-763', 'Chr7-6Mb-seg-809', 'Chr7-6Mb-seg-935']\n",
      "NDB_773 -> ['Chr7-6Mb-seg-108', 'Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-189', 'Chr7-6Mb-seg-239', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-58']\n",
      "NDB_774 -> ['Chr7-6Mb-seg-1021', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1161', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-971']\n",
      "NDB_775 -> ['Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1086', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-1176', 'Chr7-6Mb-seg-618', 'Chr7-6Mb-seg-706', 'Chr7-6Mb-seg-793', 'Chr7-6Mb-seg-944', 'Chr7-6Mb-seg-986']\n",
      "NDB_776 -> ['Chr7-6Mb-seg-1028', 'Chr7-6Mb-seg-1068', 'Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-693', 'Chr7-6Mb-seg-784', 'Chr7-6Mb-seg-868', 'Chr7-6Mb-seg-946']\n",
      "NDB_777 -> ['Chr7-6Mb-seg-115', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-476', 'Chr7-6Mb-seg-506', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-55']\n",
      "NDB_778 -> ['Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-177', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-287', 'Chr7-6Mb-seg-337', 'Chr7-6Mb-seg-397', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-467', 'Chr7-6Mb-seg-517']\n",
      "NDB_779 -> ['Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1063', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-595', 'Chr7-6Mb-seg-731', 'Chr7-6Mb-seg-836', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-923']\n",
      "NDB_780 -> ['Chr7-6Mb-seg-131', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-222', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-51', 'Chr7-6Mb-seg-522']\n",
      "NDB_783 -> ['Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1131', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-647', 'Chr7-6Mb-seg-741', 'Chr7-6Mb-seg-859', 'Chr7-6Mb-seg-961', 'Chr7-6Mb-seg-991']\n",
      "NDB_784 -> ['Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-15', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-276', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-75']\n",
      "NDB_785 -> ['Chr7-6Mb-seg-1009', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-785', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-857', 'Chr7-6Mb-seg-933']\n",
      "NDB_786 -> ['Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-209', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-549', 'Chr7-6Mb-seg-88']\n",
      "NDB_788 -> ['Chr7-6Mb-seg-195', 'Chr7-6Mb-seg-27', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-345', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-485', 'Chr7-6Mb-seg-94']\n",
      "NDB_789 -> ['Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-162', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-242', 'Chr7-6Mb-seg-312', 'Chr7-6Mb-seg-402', 'Chr7-6Mb-seg-41', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-532']\n",
      "NDB_790 -> ['Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-197', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-447', 'Chr7-6Mb-seg-487', 'Chr7-6Mb-seg-537', 'Chr7-6Mb-seg-66']\n",
      "NDB_791 -> ['Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1098', 'Chr7-6Mb-seg-1168', 'Chr7-6Mb-seg-570', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-722', 'Chr7-6Mb-seg-770', 'Chr7-6Mb-seg-885', 'Chr7-6Mb-seg-958']\n",
      "NDB_795 -> ['Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-396', 'Chr7-6Mb-seg-436', 'Chr7-6Mb-seg-526', 'Chr7-6Mb-seg-95']\n",
      "NDB_796 -> ['Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1090', 'Chr7-6Mb-seg-1120', 'Chr7-6Mb-seg-1200', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-726', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-833', 'Chr7-6Mb-seg-980']\n",
      "NDB_798 -> ['Chr7-6Mb-seg-1026', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1126', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-854', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-956']\n",
      "NDB_801 -> ['Chr7-6Mb-seg-1', 'Chr7-6Mb-seg-166', 'Chr7-6Mb-seg-206', 'Chr7-6Mb-seg-286', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-85']\n",
      "NDB_802 -> ['Chr7-6Mb-seg-1001', 'Chr7-6Mb-seg-1061', 'Chr7-6Mb-seg-1101', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-664', 'Chr7-6Mb-seg-751', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-951']\n",
      "NDB_804 -> ['Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-328', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-418', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-488']\n",
      "NDB_805 -> ['Chr7-6Mb-seg-111', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-23', 'Chr7-6Mb-seg-252', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-472', 'Chr7-6Mb-seg-512', 'Chr7-6Mb-seg-562', 'Chr7-6Mb-seg-71']\n",
      "NDB_806 -> ['Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1140', 'Chr7-6Mb-seg-1190', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-808', 'Chr7-6Mb-seg-918', 'Chr7-6Mb-seg-990']\n",
      "NDB_807 -> ['Chr7-6Mb-seg-125', 'Chr7-6Mb-seg-196', 'Chr7-6Mb-seg-306', 'Chr7-6Mb-seg-346', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-486', 'Chr7-6Mb-seg-536', 'Chr7-6Mb-seg-65']\n",
      "NDB_809 -> ['Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1107', 'Chr7-6Mb-seg-1157', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-707', 'Chr7-6Mb-seg-769', 'Chr7-6Mb-seg-817', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-977']\n",
      "NDB_812 -> ['Chr7-6Mb-seg-148', 'Chr7-6Mb-seg-249', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-389', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-58']\n",
      "NDB_813 -> ['Chr7-6Mb-seg-133', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-284', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-534', 'Chr7-6Mb-seg-83']\n",
      "NDB_815 -> ['Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-227', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-367', 'Chr7-6Mb-seg-417', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-487']\n",
      "NDB_816 -> ['Chr7-6Mb-seg-113', 'Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-374', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-53']\n",
      "NDB_817 -> ['Chr7-6Mb-seg-145', 'Chr7-6Mb-seg-236', 'Chr7-6Mb-seg-266', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-376', 'Chr7-6Mb-seg-426', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-496', 'Chr7-6Mb-seg-556']\n",
      "NDB_818 -> ['Chr7-6Mb-seg-1016', 'Chr7-6Mb-seg-1066', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1196', 'Chr7-6Mb-seg-598', 'Chr7-6Mb-seg-734', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-881', 'Chr7-6Mb-seg-929']\n",
      "NDB_819 -> ['Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1073', 'Chr7-6Mb-seg-1133', 'Chr7-6Mb-seg-1173', 'Chr7-6Mb-seg-625', 'Chr7-6Mb-seg-680', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-993']\n",
      "NDB_824 -> ['Chr7-6Mb-seg-1045', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1135', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-627', 'Chr7-6Mb-seg-682', 'Chr7-6Mb-seg-767', 'Chr7-6Mb-seg-825', 'Chr7-6Mb-seg-995']\n",
      "NDB_825 -> ['Chr7-6Mb-seg-116', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-2', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-307', 'Chr7-6Mb-seg-387', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-467']\n",
      "NDB_826 -> ['Chr7-6Mb-seg-186', 'Chr7-6Mb-seg-216', 'Chr7-6Mb-seg-246', 'Chr7-6Mb-seg-28', 'Chr7-6Mb-seg-296', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-446', 'Chr7-6Mb-seg-546', 'Chr7-6Mb-seg-75']\n",
      "NDB_828 -> ['Chr7-6Mb-seg-124', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-255', 'Chr7-6Mb-seg-335', 'Chr7-6Mb-seg-385', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-525', 'Chr7-6Mb-seg-64']\n",
      "NDB_829 -> ['Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1132', 'Chr7-6Mb-seg-604', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-742', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-962', 'Chr7-6Mb-seg-992']\n",
      "NDB_832 -> ['Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-203', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-333', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-483', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-62']\n",
      "NDB_835 -> ['Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-173', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-293', 'Chr7-6Mb-seg-353', 'Chr7-6Mb-seg-423', 'Chr7-6Mb-seg-503', 'Chr7-6Mb-seg-553', 'Chr7-6Mb-seg-92']\n",
      "NDB_836 -> ['Chr7-6Mb-seg-1034', 'Chr7-6Mb-seg-1114', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-681', 'Chr7-6Mb-seg-791', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-964']\n",
      "NDB_838 -> ['Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-194', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-314', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-434', 'Chr7-6Mb-seg-474']\n",
      "NDB_840 -> ['Chr7-6Mb-seg-1104', 'Chr7-6Mb-seg-1144', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-576', 'Chr7-6Mb-seg-616', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-852', 'Chr7-6Mb-seg-994']\n",
      "NDB_844 -> ['Chr7-6Mb-seg-117', 'Chr7-6Mb-seg-178', 'Chr7-6Mb-seg-228', 'Chr7-6Mb-seg-358', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-77']\n",
      "NDB_845 -> ['Chr7-6Mb-seg-1032', 'Chr7-6Mb-seg-1102', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-594', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-860', 'Chr7-6Mb-seg-922', 'Chr7-6Mb-seg-982']\n",
      "NDB_846 -> ['Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-142', 'Chr7-6Mb-seg-183', 'Chr7-6Mb-seg-273', 'Chr7-6Mb-seg-323', 'Chr7-6Mb-seg-373', 'Chr7-6Mb-seg-463', 'Chr7-6Mb-seg-533', 'Chr7-6Mb-seg-72']\n",
      "NDB_847 -> ['Chr7-6Mb-seg-156', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-217', 'Chr7-6Mb-seg-257', 'Chr7-6Mb-seg-317', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-477', 'Chr7-6Mb-seg-86']\n",
      "NDB_852 -> ['Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-218', 'Chr7-6Mb-seg-248', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-368', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-548', 'Chr7-6Mb-seg-77']\n",
      "NDB_853 -> ['Chr7-6Mb-seg-153', 'Chr7-6Mb-seg-254', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-394', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-494', 'Chr7-6Mb-seg-63']\n",
      "NDB_856 -> ['Chr7-6Mb-seg-129', 'Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-310', 'Chr7-6Mb-seg-350', 'Chr7-6Mb-seg-380', 'Chr7-6Mb-seg-450', 'Chr7-6Mb-seg-490', 'Chr7-6Mb-seg-540', 'Chr7-6Mb-seg-69']\n",
      "NDB_858 -> ['Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-309', 'Chr7-6Mb-seg-349', 'Chr7-6Mb-seg-379', 'Chr7-6Mb-seg-449', 'Chr7-6Mb-seg-489', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-68']\n",
      "NDB_860 -> ['Chr7-6Mb-seg-138', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-269', 'Chr7-6Mb-seg-319', 'Chr7-6Mb-seg-369', 'Chr7-6Mb-seg-4', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-68']\n",
      "NDB_862 -> ['Chr7-6Mb-seg-1013', 'Chr7-6Mb-seg-1043', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-635', 'Chr7-6Mb-seg-743', 'Chr7-6Mb-seg-812', 'Chr7-6Mb-seg-862', 'Chr7-6Mb-seg-953']\n",
      "NDB_863 -> ['Chr7-6Mb-seg-200', 'Chr7-6Mb-seg-260', 'Chr7-6Mb-seg-300', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-420', 'Chr7-6Mb-seg-460', 'Chr7-6Mb-seg-520', 'Chr7-6Mb-seg-99']\n",
      "NDB_864 -> ['Chr7-6Mb-seg-11', 'Chr7-6Mb-seg-122', 'Chr7-6Mb-seg-193', 'Chr7-6Mb-seg-263', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-393', 'Chr7-6Mb-seg-433', 'Chr7-6Mb-seg-473']\n",
      "NDB_865 -> ['Chr7-6Mb-seg-105', 'Chr7-6Mb-seg-155', 'Chr7-6Mb-seg-226', 'Chr7-6Mb-seg-326', 'Chr7-6Mb-seg-366', 'Chr7-6Mb-seg-416', 'Chr7-6Mb-seg-45', 'Chr7-6Mb-seg-486']\n",
      "NDB_866 -> ['Chr7-6Mb-seg-101', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-342', 'Chr7-6Mb-seg-392', 'Chr7-6Mb-seg-452', 'Chr7-6Mb-seg-482', 'Chr7-6Mb-seg-562']\n",
      "NDB_867 -> ['Chr7-6Mb-seg-1052', 'Chr7-6Mb-seg-1112', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-634', 'Chr7-6Mb-seg-713', 'Chr7-6Mb-seg-775', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-937']\n",
      "NDB_868 -> ['Chr7-6Mb-seg-199', 'Chr7-6Mb-seg-259', 'Chr7-6Mb-seg-299', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-419', 'Chr7-6Mb-seg-459', 'Chr7-6Mb-seg-519', 'Chr7-6Mb-seg-98']\n",
      "NDB_869 -> ['Chr7-6Mb-seg-1012', 'Chr7-6Mb-seg-1042', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-614', 'Chr7-6Mb-seg-764', 'Chr7-6Mb-seg-810', 'Chr7-6Mb-seg-937']\n",
      "NDB_871 -> ['Chr7-6Mb-seg-1041', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-603', 'Chr7-6Mb-seg-700', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-873', 'Chr7-6Mb-seg-951', 'Chr7-6Mb-seg-981']\n",
      "NDB_872 -> ['Chr7-6Mb-seg-1037', 'Chr7-6Mb-seg-1117', 'Chr7-6Mb-seg-1147', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-609', 'Chr7-6Mb-seg-692', 'Chr7-6Mb-seg-794', 'Chr7-6Mb-seg-883', 'Chr7-6Mb-seg-967']\n",
      "NDB_873 -> ['Chr7-6Mb-seg-1029', 'Chr7-6Mb-seg-1089', 'Chr7-6Mb-seg-1119', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-662', 'Chr7-6Mb-seg-723', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-979']\n",
      "NDB_875 -> ['Chr7-6Mb-seg-118', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-229', 'Chr7-6Mb-seg-359', 'Chr7-6Mb-seg-409', 'Chr7-6Mb-seg-439', 'Chr7-6Mb-seg-499', 'Chr7-6Mb-seg-539', 'Chr7-6Mb-seg-78']\n",
      "NDB_876 -> ['Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1150', 'Chr7-6Mb-seg-1180', 'Chr7-6Mb-seg-592', 'Chr7-6Mb-seg-644', 'Chr7-6Mb-seg-740', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-980']\n",
      "NDB_878 -> ['Chr7-6Mb-seg-109', 'Chr7-6Mb-seg-21', 'Chr7-6Mb-seg-210', 'Chr7-6Mb-seg-250', 'Chr7-6Mb-seg-360', 'Chr7-6Mb-seg-470', 'Chr7-6Mb-seg-510', 'Chr7-6Mb-seg-560', 'Chr7-6Mb-seg-69']\n",
      "NDB_879 -> ['Chr7-6Mb-seg-1003', 'Chr7-6Mb-seg-1093', 'Chr7-6Mb-seg-1163', 'Chr7-6Mb-seg-565', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-765', 'Chr7-6Mb-seg-878', 'Chr7-6Mb-seg-953']\n",
      "NDB_880 -> ['Chr7-6Mb-seg-102', 'Chr7-6Mb-seg-163', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-243', 'Chr7-6Mb-seg-313', 'Chr7-6Mb-seg-403', 'Chr7-6Mb-seg-42', 'Chr7-6Mb-seg-443', 'Chr7-6Mb-seg-533']\n",
      "NDB_881 -> ['Chr7-6Mb-seg-144', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-275', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-445', 'Chr7-6Mb-seg-495', 'Chr7-6Mb-seg-555', 'Chr7-6Mb-seg-84']\n",
      "NDB_882 -> ['Chr7-6Mb-seg-1000', 'Chr7-6Mb-seg-1060', 'Chr7-6Mb-seg-1100', 'Chr7-6Mb-seg-1160', 'Chr7-6Mb-seg-663', 'Chr7-6Mb-seg-797', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-934']\n",
      "NDB_883 -> ['Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-251', 'Chr7-6Mb-seg-291', 'Chr7-6Mb-seg-321', 'Chr7-6Mb-seg-391', 'Chr7-6Mb-seg-441', 'Chr7-6Mb-seg-491', 'Chr7-6Mb-seg-60']\n",
      "NDB_887 -> ['Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-141', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-272', 'Chr7-6Mb-seg-322', 'Chr7-6Mb-seg-372', 'Chr7-6Mb-seg-462', 'Chr7-6Mb-seg-532', 'Chr7-6Mb-seg-71']\n",
      "NDB_889 -> ['Chr7-6Mb-seg-104', 'Chr7-6Mb-seg-165', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-315', 'Chr7-6Mb-seg-365', 'Chr7-6Mb-seg-425', 'Chr7-6Mb-seg-465', 'Chr7-6Mb-seg-505']\n",
      "NDB_891 -> ['Chr7-6Mb-seg-1049', 'Chr7-6Mb-seg-1079', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-631', 'Chr7-6Mb-seg-698', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-832', 'Chr7-6Mb-seg-999']\n",
      "NDB_894 -> ['Chr7-6Mb-seg-1007', 'Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1127', 'Chr7-6Mb-seg-599', 'Chr7-6Mb-seg-672', 'Chr7-6Mb-seg-747', 'Chr7-6Mb-seg-829', 'Chr7-6Mb-seg-967']\n",
      "NDB_895 -> ['Chr7-6Mb-seg-1011', 'Chr7-6Mb-seg-1081', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1181', 'Chr7-6Mb-seg-593', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-834', 'Chr7-6Mb-seg-902', 'Chr7-6Mb-seg-961']\n",
      "NDB_896 -> ['Chr7-6Mb-seg-112', 'Chr7-6Mb-seg-213', 'Chr7-6Mb-seg-24', 'Chr7-6Mb-seg-253', 'Chr7-6Mb-seg-363', 'Chr7-6Mb-seg-473', 'Chr7-6Mb-seg-513', 'Chr7-6Mb-seg-563', 'Chr7-6Mb-seg-72']\n",
      "NDB_899 -> ['Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-204', 'Chr7-6Mb-seg-274', 'Chr7-6Mb-seg-334', 'Chr7-6Mb-seg-404', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-514', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-63']\n",
      "NDB_901 -> ['Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-169', 'Chr7-6Mb-seg-20', 'Chr7-6Mb-seg-279', 'Chr7-6Mb-seg-329', 'Chr7-6Mb-seg-429', 'Chr7-6Mb-seg-529', 'Chr7-6Mb-seg-78']\n",
      "NDB_902 -> ['Chr7-6Mb-seg-1019', 'Chr7-6Mb-seg-1069', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1199', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-846', 'Chr7-6Mb-seg-888', 'Chr7-6Mb-seg-933']\n",
      "NDB_903 -> ['Chr7-6Mb-seg-1035', 'Chr7-6Mb-seg-1075', 'Chr7-6Mb-seg-1165', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-745', 'Chr7-6Mb-seg-802', 'Chr7-6Mb-seg-840', 'Chr7-6Mb-seg-942']\n",
      "NDB_904 -> ['Chr7-6Mb-seg-1108', 'Chr7-6Mb-seg-1148', 'Chr7-6Mb-seg-1198', 'Chr7-6Mb-seg-580', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-806', 'Chr7-6Mb-seg-856', 'Chr7-6Mb-seg-998']\n",
      "NDB_905 -> ['Chr7-6Mb-seg-121', 'Chr7-6Mb-seg-182', 'Chr7-6Mb-seg-232', 'Chr7-6Mb-seg-362', 'Chr7-6Mb-seg-412', 'Chr7-6Mb-seg-442', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-542', 'Chr7-6Mb-seg-81']\n",
      "NDB_908 -> ['Chr7-6Mb-seg-1002', 'Chr7-6Mb-seg-1092', 'Chr7-6Mb-seg-1152', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-648', 'Chr7-6Mb-seg-701', 'Chr7-6Mb-seg-799', 'Chr7-6Mb-seg-849', 'Chr7-6Mb-seg-904']\n",
      "NDB_910 -> ['Chr7-6Mb-seg-1056', 'Chr7-6Mb-seg-1136', 'Chr7-6Mb-seg-1186', 'Chr7-6Mb-seg-568', 'Chr7-6Mb-seg-640', 'Chr7-6Mb-seg-756', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-912', 'Chr7-6Mb-seg-986']\n",
      "NDB_914 -> ['Chr7-6Mb-seg-1051', 'Chr7-6Mb-seg-1111', 'Chr7-6Mb-seg-1151', 'Chr7-6Mb-seg-633', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-773', 'Chr7-6Mb-seg-821', 'Chr7-6Mb-seg-935']\n",
      "NDB_915 -> ['Chr7-6Mb-seg-147', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-268', 'Chr7-6Mb-seg-298', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-428', 'Chr7-6Mb-seg-47', 'Chr7-6Mb-seg-498', 'Chr7-6Mb-seg-558']\n",
      "NDB_917 -> ['Chr7-6Mb-seg-157', 'Chr7-6Mb-seg-208', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-36', 'Chr7-6Mb-seg-398', 'Chr7-6Mb-seg-438', 'Chr7-6Mb-seg-528', 'Chr7-6Mb-seg-97']\n",
      "NDB_920 -> ['Chr7-6Mb-seg-1099', 'Chr7-6Mb-seg-1149', 'Chr7-6Mb-seg-1179', 'Chr7-6Mb-seg-591', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-739', 'Chr7-6Mb-seg-900', 'Chr7-6Mb-seg-979']\n",
      "NDB_921 -> ['Chr7-6Mb-seg-10', 'Chr7-6Mb-seg-172', 'Chr7-6Mb-seg-212', 'Chr7-6Mb-seg-292', 'Chr7-6Mb-seg-352', 'Chr7-6Mb-seg-422', 'Chr7-6Mb-seg-502', 'Chr7-6Mb-seg-552', 'Chr7-6Mb-seg-91']\n",
      "NDB_922 -> ['Chr7-6Mb-seg-1030', 'Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1110', 'Chr7-6Mb-seg-572', 'Chr7-6Mb-seg-699', 'Chr7-6Mb-seg-786', 'Chr7-6Mb-seg-870', 'Chr7-6Mb-seg-950']\n",
      "NDB_925 -> ['Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1145', 'Chr7-6Mb-seg-1175', 'Chr7-6Mb-seg-587', 'Chr7-6Mb-seg-639', 'Chr7-6Mb-seg-733', 'Chr7-6Mb-seg-894', 'Chr7-6Mb-seg-975']\n",
      "NDB_926 -> ['Chr7-6Mb-seg-1057', 'Chr7-6Mb-seg-1137', 'Chr7-6Mb-seg-1187', 'Chr7-6Mb-seg-569', 'Chr7-6Mb-seg-641', 'Chr7-6Mb-seg-757', 'Chr7-6Mb-seg-805', 'Chr7-6Mb-seg-913', 'Chr7-6Mb-seg-987']\n",
      "NDB_928 -> ['Chr7-6Mb-seg-1059', 'Chr7-6Mb-seg-1139', 'Chr7-6Mb-seg-1189', 'Chr7-6Mb-seg-571', 'Chr7-6Mb-seg-643', 'Chr7-6Mb-seg-760', 'Chr7-6Mb-seg-807', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-989']\n",
      "NDB_929 -> ['Chr7-6Mb-seg-1054', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1184', 'Chr7-6Mb-seg-566', 'Chr7-6Mb-seg-638', 'Chr7-6Mb-seg-754', 'Chr7-6Mb-seg-801', 'Chr7-6Mb-seg-907', 'Chr7-6Mb-seg-984']\n",
      "NDB_931 -> ['Chr7-6Mb-seg-1022', 'Chr7-6Mb-seg-1072', 'Chr7-6Mb-seg-1122', 'Chr7-6Mb-seg-1172', 'Chr7-6Mb-seg-574', 'Chr7-6Mb-seg-728', 'Chr7-6Mb-seg-822', 'Chr7-6Mb-seg-876', 'Chr7-6Mb-seg-962']\n",
      "NDB_933 -> ['Chr7-6Mb-seg-1014', 'Chr7-6Mb-seg-1064', 'Chr7-6Mb-seg-1134', 'Chr7-6Mb-seg-1194', 'Chr7-6Mb-seg-596', 'Chr7-6Mb-seg-732', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-879', 'Chr7-6Mb-seg-924']\n",
      "NDB_934 -> ['Chr7-6Mb-seg-1036', 'Chr7-6Mb-seg-1076', 'Chr7-6Mb-seg-1166', 'Chr7-6Mb-seg-588', 'Chr7-6Mb-seg-655', 'Chr7-6Mb-seg-746', 'Chr7-6Mb-seg-804', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-944']\n",
      "NDB_935 -> ['Chr7-6Mb-seg-134', 'Chr7-6Mb-seg-185', 'Chr7-6Mb-seg-225', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-415', 'Chr7-6Mb-seg-475', 'Chr7-6Mb-seg-515', 'Chr7-6Mb-seg-84']\n",
      "NDB_936 -> ['Chr7-6Mb-seg-106', 'Chr7-6Mb-seg-136', 'Chr7-6Mb-seg-187', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-347', 'Chr7-6Mb-seg-407', 'Chr7-6Mb-seg-517', 'Chr7-6Mb-seg-56']\n",
      "NDB_939 -> ['Chr7-6Mb-seg-1046', 'Chr7-6Mb-seg-1156', 'Chr7-6Mb-seg-608', 'Chr7-6Mb-seg-671', 'Chr7-6Mb-seg-720', 'Chr7-6Mb-seg-780', 'Chr7-6Mb-seg-841', 'Chr7-6Mb-seg-895', 'Chr7-6Mb-seg-986']\n",
      "NDB_940 -> ['Chr7-6Mb-seg-1055', 'Chr7-6Mb-seg-1095', 'Chr7-6Mb-seg-1155', 'Chr7-6Mb-seg-651', 'Chr7-6Mb-seg-792', 'Chr7-6Mb-seg-864', 'Chr7-6Mb-seg-926', 'Chr7-6Mb-seg-995']\n",
      "NDB_941 -> ['Chr7-6Mb-seg-12', 'Chr7-6Mb-seg-174', 'Chr7-6Mb-seg-214', 'Chr7-6Mb-seg-294', 'Chr7-6Mb-seg-354', 'Chr7-6Mb-seg-424', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-554', 'Chr7-6Mb-seg-93']\n",
      "NDB_946 -> ['Chr7-6Mb-seg-123', 'Chr7-6Mb-seg-184', 'Chr7-6Mb-seg-234', 'Chr7-6Mb-seg-364', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-444', 'Chr7-6Mb-seg-504', 'Chr7-6Mb-seg-544', 'Chr7-6Mb-seg-83']\n",
      "NDB_948 -> ['Chr7-6Mb-seg-1018', 'Chr7-6Mb-seg-1088', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-1178', 'Chr7-6Mb-seg-620', 'Chr7-6Mb-seg-708', 'Chr7-6Mb-seg-795', 'Chr7-6Mb-seg-946', 'Chr7-6Mb-seg-988']\n",
      "NDB_951 -> ['Chr7-6Mb-seg-1023', 'Chr7-6Mb-seg-1083', 'Chr7-6Mb-seg-1113', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-649', 'Chr7-6Mb-seg-717', 'Chr7-6Mb-seg-753', 'Chr7-6Mb-seg-823', 'Chr7-6Mb-seg-973']\n",
      "NDB_954 -> ['Chr7-6Mb-seg-150', 'Chr7-6Mb-seg-241', 'Chr7-6Mb-seg-271', 'Chr7-6Mb-seg-301', 'Chr7-6Mb-seg-381', 'Chr7-6Mb-seg-431', 'Chr7-6Mb-seg-50', 'Chr7-6Mb-seg-501', 'Chr7-6Mb-seg-561']\n",
      "NDB_955 -> ['Chr7-6Mb-seg-1008', 'Chr7-6Mb-seg-1058', 'Chr7-6Mb-seg-1128', 'Chr7-6Mb-seg-600', 'Chr7-6Mb-seg-674', 'Chr7-6Mb-seg-748', 'Chr7-6Mb-seg-831', 'Chr7-6Mb-seg-968']\n",
      "NDB_957 -> ['Chr7-6Mb-seg-128', 'Chr7-6Mb-seg-179', 'Chr7-6Mb-seg-219', 'Chr7-6Mb-seg-289', 'Chr7-6Mb-seg-339', 'Chr7-6Mb-seg-399', 'Chr7-6Mb-seg-469', 'Chr7-6Mb-seg-48', 'Chr7-6Mb-seg-519']\n",
      "NDB_958 -> ['Chr7-6Mb-seg-1044', 'Chr7-6Mb-seg-1154', 'Chr7-6Mb-seg-606', 'Chr7-6Mb-seg-669', 'Chr7-6Mb-seg-718', 'Chr7-6Mb-seg-777', 'Chr7-6Mb-seg-837', 'Chr7-6Mb-seg-893', 'Chr7-6Mb-seg-984']\n",
      "NDB_960 -> ['Chr7-6Mb-seg-1070', 'Chr7-6Mb-seg-1170', 'Chr7-6Mb-seg-582', 'Chr7-6Mb-seg-632', 'Chr7-6Mb-seg-711', 'Chr7-6Mb-seg-762', 'Chr7-6Mb-seg-901', 'Chr7-6Mb-seg-970']\n",
      "NDB_961 -> ['Chr7-6Mb-seg-107', 'Chr7-6Mb-seg-137', 'Chr7-6Mb-seg-188', 'Chr7-6Mb-seg-238', 'Chr7-6Mb-seg-278', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-408', 'Chr7-6Mb-seg-518', 'Chr7-6Mb-seg-57']\n",
      "NDB_965 -> ['Chr7-6Mb-seg-146', 'Chr7-6Mb-seg-237', 'Chr7-6Mb-seg-267', 'Chr7-6Mb-seg-297', 'Chr7-6Mb-seg-377', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-46', 'Chr7-6Mb-seg-497', 'Chr7-6Mb-seg-557']\n",
      "NDB_971 -> ['Chr7-6Mb-seg-126', 'Chr7-6Mb-seg-16', 'Chr7-6Mb-seg-167', 'Chr7-6Mb-seg-277', 'Chr7-6Mb-seg-327', 'Chr7-6Mb-seg-427', 'Chr7-6Mb-seg-527', 'Chr7-6Mb-seg-76']\n",
      "NDB_974 -> ['Chr7-6Mb-seg-127', 'Chr7-6Mb-seg-198', 'Chr7-6Mb-seg-308', 'Chr7-6Mb-seg-348', 'Chr7-6Mb-seg-378', 'Chr7-6Mb-seg-448', 'Chr7-6Mb-seg-488', 'Chr7-6Mb-seg-538', 'Chr7-6Mb-seg-67']\n",
      "NDB_976 -> ['Chr7-6Mb-seg-1039', 'Chr7-6Mb-seg-1109', 'Chr7-6Mb-seg-1159', 'Chr7-6Mb-seg-601', 'Chr7-6Mb-seg-710', 'Chr7-6Mb-seg-771', 'Chr7-6Mb-seg-819', 'Chr7-6Mb-seg-917', 'Chr7-6Mb-seg-979']\n",
      "NDB_977 -> ['Chr7-6Mb-seg-1103', 'Chr7-6Mb-seg-1143', 'Chr7-6Mb-seg-1193', 'Chr7-6Mb-seg-575', 'Chr7-6Mb-seg-615', 'Chr7-6Mb-seg-668', 'Chr7-6Mb-seg-800', 'Chr7-6Mb-seg-850', 'Chr7-6Mb-seg-993']\n",
      "NDB_989 -> ['Chr7-6Mb-seg-164', 'Chr7-6Mb-seg-224', 'Chr7-6Mb-seg-25', 'Chr7-6Mb-seg-264', 'Chr7-6Mb-seg-324', 'Chr7-6Mb-seg-384', 'Chr7-6Mb-seg-414', 'Chr7-6Mb-seg-484', 'Chr7-6Mb-seg-93']\n",
      "NDB_994 -> ['Chr7-6Mb-seg-175', 'Chr7-6Mb-seg-215', 'Chr7-6Mb-seg-305', 'Chr7-6Mb-seg-375', 'Chr7-6Mb-seg-405', 'Chr7-6Mb-seg-455', 'Chr7-6Mb-seg-535', 'Chr7-6Mb-seg-64']\n",
      "NDB_995 -> ['Chr7-6Mb-seg-1071', 'Chr7-6Mb-seg-1141', 'Chr7-6Mb-seg-1191', 'Chr7-6Mb-seg-613', 'Chr7-6Mb-seg-712', 'Chr7-6Mb-seg-787', 'Chr7-6Mb-seg-848', 'Chr7-6Mb-seg-919', 'Chr7-6Mb-seg-991']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(reg_readout_dic, full_records, \n",
    "                                                                    check_readout_dict, primer_len=primer_len, \n",
    "                                                                    target_len=target_len,\n",
    "                                                                    probe_readout_num=num_readout_per_probe)\n",
    "print(readout2reg_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7445b10e",
   "metadata": {},
   "source": [
    "### 4.1.5 check bewteen probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 203,
   "id": "3a6a50cc",
   "metadata": {},
   "outputs": [],
   "source": [
    "# construct internal map\n",
    "reload(check)\n",
    "int_map = check._construct_internal_map(full_records, library_folder, save=False) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 204,
   "id": "11132a68",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n"
     ]
    }
   ],
   "source": [
    "# internal check probes\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, int_map, readout_dict)\n",
    "print(readout_count_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 205,
   "id": "056864be",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_7:44014999-44019999_gene_Chr7-6Mb-seg-4_pb_5_pos_438_strand_+_readouts_[NDB_680_c,NDB_680_c,NDB_680_c,NDB_680_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:53, dumped\n",
      "--- Sequence:loc_7:44104999-44109999_gene_Chr7-6Mb-seg-22_pb_0_pos_6_strand_+_readouts_[NDB_438_c,NDB_438_c,NDB_438_c,NDB_438_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:57, dumped\n",
      "--- Sequence:loc_7:44174999-44179999_gene_Chr7-6Mb-seg-36_pb_29_pos_4234_strand_+_readouts_[NDB_762_c,NDB_762_c,NDB_762_c,NDB_762_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:197, dumped\n",
      "--- Sequence:loc_7:44189999-44194999_gene_Chr7-6Mb-seg-39_pb_30_pos_3533_strand_+_readouts_[NDB_455_c,NDB_455_c,NDB_455_c,NDB_455_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:51, dumped\n",
      "--- Sequence:loc_7:44209999-44214999_gene_Chr7-6Mb-seg-43_pb_1_pos_252_strand_+_readouts_[NDB_641_c,NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:55, dumped\n",
      "--- Sequence:loc_7:44239999-44244999_gene_Chr7-6Mb-seg-49_pb_6_pos_376_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:58, dumped\n",
      "--- Sequence:loc_7:44239999-44244999_gene_Chr7-6Mb-seg-49_pb_11_pos_699_strand_+_readouts_[NDB_391_c,NDB_391_c,NDB_391_c,NDB_391_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:80, dumped\n",
      "--- Sequence:loc_7:44239999-44244999_gene_Chr7-6Mb-seg-49_pb_12_pos_741_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:76, dumped\n",
      "--- Sequence:loc_7:44239999-44244999_gene_Chr7-6Mb-seg-49_pb_13_pos_815_strand_+_readouts_[NDB_698_c,NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:82, dumped\n",
      "--- Sequence:loc_7:44239999-44244999_gene_Chr7-6Mb-seg-49_pb_17_pos_996_strand_-_readouts_[NDB_391_c,NDB_391_c,NDB_391_c,NDB_391_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:58, dumped\n",
      "--- Sequence:loc_7:44394999-44399999_gene_Chr7-6Mb-seg-80_pb_63_pos_4270_strand_+_readouts_[NDB_438_c,NDB_438_c,NDB_438_c,NDB_438_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:53, dumped\n",
      "--- Sequence:loc_7:44394999-44399999_gene_Chr7-6Mb-seg-80_pb_65_pos_4354_strand_+_readouts_[NDB_405_c,NDB_405_c,NDB_405_c,NDB_405_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:61, dumped\n",
      "--- Sequence:loc_7:44414999-44419999_gene_Chr7-6Mb-seg-84_pb_54_pos_4320_strand_+_readouts_[NDB_881_c,NDB_881_c,NDB_881_c,NDB_881_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:73, dumped\n",
      "--- Sequence:loc_7:44414999-44419999_gene_Chr7-6Mb-seg-84_pb_55_pos_4363_strand_-_readouts_[NDB_935_c,NDB_935_c,NDB_935_c,NDB_935_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:60, dumped\n",
      "--- Sequence:loc_7:44414999-44419999_gene_Chr7-6Mb-seg-84_pb_56_pos_4463_strand_+_readouts_[NDB_711_c,NDB_711_c,NDB_711_c,NDB_711_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:79, dumped\n",
      "--- Sequence:loc_7:44414999-44419999_gene_Chr7-6Mb-seg-84_pb_58_pos_4631_strand_-_readouts_[NDB_935_c,NDB_935_c,NDB_935_c,NDB_935_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:98, dumped\n",
      "--- Sequence:loc_7:44414999-44419999_gene_Chr7-6Mb-seg-84_pb_59_pos_4678_strand_+_readouts_[NDB_711_c,NDB_711_c,NDB_711_c,NDB_711_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:92, dumped\n",
      "--- Sequence:loc_7:44414999-44419999_gene_Chr7-6Mb-seg-84_pb_60_pos_4803_strand_+_readouts_[NDB_881_c,NDB_881_c,NDB_881_c,NDB_881_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:99, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_40_pos_3737_strand_+_readouts_[NDB_568_c,NDB_568_c,NDB_568_c,NDB_568_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:96, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_41_pos_3786_strand_+_readouts_[NDB_387_c,NDB_387_c,NDB_387_c,NDB_387_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:74, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_42_pos_3867_strand_+_readouts_[NDB_917_c,NDB_917_c,NDB_917_c,NDB_917_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:122, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_43_pos_3947_strand_-_readouts_[NDB_568_c,NDB_568_c,NDB_568_c,NDB_568_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:98, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_44_pos_4012_strand_+_readouts_[NDB_387_c,NDB_387_c,NDB_387_c,NDB_387_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:119, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_45_pos_4069_strand_-_readouts_[NDB_917_c,NDB_917_c,NDB_917_c,NDB_917_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:72, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_46_pos_4139_strand_+_readouts_[NDB_568_c,NDB_568_c,NDB_568_c,NDB_568_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:62, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_47_pos_4181_strand_+_readouts_[NDB_387_c,NDB_387_c,NDB_387_c,NDB_387_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:116, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_49_pos_4329_strand_+_readouts_[NDB_568_c,NDB_568_c,NDB_568_c,NDB_568_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:65, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_51_pos_4414_strand_+_readouts_[NDB_917_c,NDB_917_c,NDB_917_c,NDB_917_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:81, dumped\n",
      "--- Sequence:loc_7:44479999-44484999_gene_Chr7-6Mb-seg-97_pb_52_pos_4459_strand_+_readouts_[NDB_568_c,NDB_568_c,NDB_568_c,NDB_568_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:54, dumped\n",
      "--- Sequence:loc_7:44619999-44624999_gene_Chr7-6Mb-seg-125_pb_7_pos_857_strand_+_readouts_[NDB_807_c,NDB_807_c,NDB_807_c,NDB_807_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:106, dumped\n",
      "--- Sequence:loc_7:44619999-44624999_gene_Chr7-6Mb-seg-125_pb_8_pos_1055_strand_+_readouts_[NDB_690_c,NDB_690_c,NDB_690_c,NDB_690_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:102, dumped\n",
      "--- Sequence:loc_7:44619999-44624999_gene_Chr7-6Mb-seg-125_pb_9_pos_1106_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:108, dumped\n",
      "--- Sequence:loc_7:44619999-44624999_gene_Chr7-6Mb-seg-125_pb_10_pos_1199_strand_-_readouts_[NDB_807_c,NDB_807_c,NDB_807_c,NDB_807_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:127, dumped\n",
      "--- Sequence:loc_7:44619999-44624999_gene_Chr7-6Mb-seg-125_pb_11_pos_1242_strand_+_readouts_[NDB_690_c,NDB_690_c,NDB_690_c,NDB_690_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:63, dumped\n",
      "--- Sequence:loc_7:44619999-44624999_gene_Chr7-6Mb-seg-125_pb_12_pos_1354_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:128, dumped\n",
      "--- Sequence:loc_7:44619999-44624999_gene_Chr7-6Mb-seg-125_pb_14_pos_1494_strand_+_readouts_[NDB_690_c,NDB_690_c,NDB_690_c,NDB_690_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:101, dumped\n",
      "--- Sequence:loc_7:44619999-44624999_gene_Chr7-6Mb-seg-125_pb_15_pos_1540_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:100, dumped\n",
      "--- Sequence:loc_7:44699999-44704999_gene_Chr7-6Mb-seg-141_pb_3_pos_153_strand_+_readouts_[NDB_467_c,NDB_467_c,NDB_467_c,NDB_467_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:51, dumped\n",
      "--- Sequence:loc_7:44699999-44704999_gene_Chr7-6Mb-seg-141_pb_4_pos_278_strand_-_readouts_[NDB_866_c,NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:51, dumped\n",
      "--- Sequence:loc_7:44719999-44724999_gene_Chr7-6Mb-seg-145_pb_45_pos_3435_strand_+_readouts_[NDB_376_c,NDB_376_c,NDB_376_c,NDB_376_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:66, dumped\n",
      "--- Sequence:loc_7:44719999-44724999_gene_Chr7-6Mb-seg-145_pb_47_pos_3573_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:68, dumped\n",
      "--- Sequence:loc_7:44719999-44724999_gene_Chr7-6Mb-seg-145_pb_48_pos_3615_strand_+_readouts_[NDB_376_c,NDB_376_c,NDB_376_c,NDB_376_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:68, dumped\n",
      "--- Sequence:loc_7:44734999-44739999_gene_Chr7-6Mb-seg-148_pb_51_pos_3516_strand_-_readouts_[NDB_409_c,NDB_409_c,NDB_409_c,NDB_409_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:120, dumped\n",
      "--- Sequence:loc_7:44734999-44739999_gene_Chr7-6Mb-seg-148_pb_52_pos_3566_strand_+_readouts_[NDB_812_c,NDB_812_c,NDB_812_c,NDB_812_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:82, dumped\n",
      "--- Sequence:loc_7:44734999-44739999_gene_Chr7-6Mb-seg-148_pb_53_pos_3633_strand_+_readouts_[NDB_448_c,NDB_448_c,NDB_448_c,NDB_448_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:66, dumped\n",
      "--- Sequence:loc_7:44734999-44739999_gene_Chr7-6Mb-seg-148_pb_54_pos_3796_strand_+_readouts_[NDB_409_c,NDB_409_c,NDB_409_c,NDB_409_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:88, dumped\n",
      "--- Sequence:loc_7:44734999-44739999_gene_Chr7-6Mb-seg-148_pb_55_pos_3862_strand_+_readouts_[NDB_812_c,NDB_812_c,NDB_812_c,NDB_812_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:116, dumped\n",
      "--- Sequence:loc_7:44734999-44739999_gene_Chr7-6Mb-seg-148_pb_56_pos_3913_strand_+_readouts_[NDB_448_c,NDB_448_c,NDB_448_c,NDB_448_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:102, dumped\n",
      "--- Sequence:loc_7:44734999-44739999_gene_Chr7-6Mb-seg-148_pb_57_pos_4006_strand_+_readouts_[NDB_409_c,NDB_409_c,NDB_409_c,NDB_409_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:68, dumped\n",
      "--- Sequence:loc_7:44734999-44739999_gene_Chr7-6Mb-seg-148_pb_58_pos_4120_strand_+_readouts_[NDB_812_c,NDB_812_c,NDB_812_c,NDB_812_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:60, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_7:44859999-44864999_gene_Chr7-6Mb-seg-173_pb_23_pos_1836_strand_+_readouts_[NDB_835_c,NDB_835_c,NDB_835_c,NDB_835_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:85, dumped\n",
      "--- Sequence:loc_7:44859999-44864999_gene_Chr7-6Mb-seg-173_pb_24_pos_1878_strand_+_readouts_[NDB_347_c,NDB_347_c,NDB_347_c,NDB_347_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:65, dumped\n",
      "--- Sequence:loc_7:44859999-44864999_gene_Chr7-6Mb-seg-173_pb_26_pos_1993_strand_-_readouts_[NDB_835_c,NDB_835_c,NDB_835_c,NDB_835_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:109, dumped\n",
      "--- Sequence:loc_7:44859999-44864999_gene_Chr7-6Mb-seg-173_pb_27_pos_2054_strand_+_readouts_[NDB_347_c,NDB_347_c,NDB_347_c,NDB_347_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:83, dumped\n",
      "--- Sequence:loc_7:44859999-44864999_gene_Chr7-6Mb-seg-173_pb_28_pos_2096_strand_+_readouts_[NDB_1047_c,NDB_1047_c,NDB_1047_c,NDB_1047_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:113, dumped\n",
      "--- Sequence:loc_7:45004999-45009999_gene_Chr7-6Mb-seg-202_pb_77_pos_4795_strand_-_readouts_[NDB_619_c,NDB_619_c,NDB_619_c,NDB_619_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:52, dumped\n",
      "--- Sequence:loc_7:45004999-45009999_gene_Chr7-6Mb-seg-202_pb_78_pos_4850_strand_+_readouts_[NDB_390_c,NDB_390_c,NDB_390_c,NDB_390_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:52, dumped\n",
      "--- Sequence:loc_7:45254999-45259999_gene_Chr7-6Mb-seg-252_pb_9_pos_656_strand_+_readouts_[NDB_1075_c,NDB_1075_c,NDB_1075_c,NDB_1075_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:69, dumped\n",
      "--- Sequence:loc_7:45254999-45259999_gene_Chr7-6Mb-seg-252_pb_10_pos_809_strand_+_readouts_[NDB_335_c,NDB_335_c,NDB_335_c,NDB_335_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:64, dumped\n",
      "--- Sequence:loc_7:45254999-45259999_gene_Chr7-6Mb-seg-252_pb_11_pos_906_strand_+_readouts_[NDB_805_c,NDB_805_c,NDB_805_c,NDB_805_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:68, dumped\n",
      "--- Sequence:loc_7:45619999-45624999_gene_Chr7-6Mb-seg-325_pb_4_pos_216_strand_+_readouts_[NDB_445_c,NDB_445_c,NDB_445_c,NDB_445_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:67, dumped\n",
      "--- Sequence:loc_7:45619999-45624999_gene_Chr7-6Mb-seg-325_pb_6_pos_305_strand_+_readouts_[NDB_498_c,NDB_498_c,NDB_498_c,NDB_498_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:53, dumped\n",
      "--- Sequence:loc_7:45819999-45824999_gene_Chr7-6Mb-seg-365_pb_59_pos_4054_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:54, dumped\n",
      "--- Sequence:loc_7:45844999-45849999_gene_Chr7-6Mb-seg-370_pb_44_pos_4264_strand_-_readouts_[NDB_1020_c,NDB_1020_c,NDB_1020_c,NDB_1020_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:53, dumped\n",
      "--- Sequence:loc_7:45844999-45849999_gene_Chr7-6Mb-seg-370_pb_45_pos_4330_strand_+_readouts_[NDB_455_c,NDB_455_c,NDB_455_c,NDB_455_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:59, dumped\n",
      "--- Sequence:loc_7:45949999-45954999_gene_Chr7-6Mb-seg-391_pb_18_pos_1527_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c,NDB_883_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:61, dumped\n",
      "--- Sequence:loc_7:45949999-45954999_gene_Chr7-6Mb-seg-391_pb_19_pos_1580_strand_+_readouts_[NDB_1033_c,NDB_1033_c,NDB_1033_c,NDB_1033_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:64, dumped\n",
      "--- Sequence:loc_7:45949999-45954999_gene_Chr7-6Mb-seg-391_pb_20_pos_1668_strand_+_readouts_[NDB_532_c,NDB_532_c,NDB_532_c,NDB_532_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:85, dumped\n",
      "--- Sequence:loc_7:45949999-45954999_gene_Chr7-6Mb-seg-391_pb_22_pos_1769_strand_+_readouts_[NDB_1033_c,NDB_1033_c,NDB_1033_c,NDB_1033_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:63, dumped\n",
      "--- Sequence:loc_7:45949999-45954999_gene_Chr7-6Mb-seg-391_pb_24_pos_1931_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c,NDB_883_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:94, dumped\n",
      "--- Sequence:loc_7:46004999-46009999_gene_Chr7-6Mb-seg-402_pb_28_pos_1796_strand_-_readouts_[NDB_390_c,NDB_390_c,NDB_390_c,NDB_390_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:84, dumped\n",
      "--- Sequence:loc_7:46004999-46009999_gene_Chr7-6Mb-seg-402_pb_34_pos_2107_strand_+_readouts_[NDB_390_c,NDB_390_c,NDB_390_c,NDB_390_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:59, dumped\n",
      "--- Sequence:loc_7:46004999-46009999_gene_Chr7-6Mb-seg-402_pb_35_pos_2168_strand_+_readouts_[NDB_780_c,NDB_780_c,NDB_780_c,NDB_780_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:60, dumped\n",
      "--- Sequence:loc_7:46004999-46009999_gene_Chr7-6Mb-seg-402_pb_37_pos_2302_strand_+_readouts_[NDB_390_c,NDB_390_c,NDB_390_c,NDB_390_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:52, dumped\n",
      "--- Sequence:loc_7:46004999-46009999_gene_Chr7-6Mb-seg-402_pb_39_pos_2434_strand_+_readouts_[NDB_789_c,NDB_789_c,NDB_789_c,NDB_789_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:83, dumped\n",
      "--- Sequence:loc_7:46009999-46014999_gene_Chr7-6Mb-seg-403_pb_12_pos_1203_strand_+_readouts_[NDB_880_c,NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:109, dumped\n",
      "--- Sequence:loc_7:46009999-46014999_gene_Chr7-6Mb-seg-403_pb_13_pos_1266_strand_+_readouts_[NDB_832_c,NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:62, dumped\n",
      "--- Sequence:loc_7:46009999-46014999_gene_Chr7-6Mb-seg-403_pb_14_pos_1313_strand_+_readouts_[NDB_397_c,NDB_397_c,NDB_397_c,NDB_397_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:77, dumped\n",
      "--- Sequence:loc_7:46009999-46014999_gene_Chr7-6Mb-seg-403_pb_15_pos_1391_strand_+_readouts_[NDB_880_c,NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:96, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_12_pos_726_strand_-_readouts_[NDB_641_c,NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:73, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_13_pos_773_strand_-_readouts_[NDB_899_c,NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:140, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_14_pos_820_strand_+_readouts_[NDB_647_c,NDB_647_c,NDB_647_c,NDB_647_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:70, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_15_pos_891_strand_+_readouts_[NDB_641_c,NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:130, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_17_pos_975_strand_+_readouts_[NDB_647_c,NDB_647_c,NDB_647_c,NDB_647_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:111, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_19_pos_1077_strand_+_readouts_[NDB_899_c,NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:80, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_21_pos_1180_strand_+_readouts_[NDB_641_c,NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:110, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_22_pos_1228_strand_+_readouts_[NDB_899_c,NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:118, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_23_pos_1270_strand_-_readouts_[NDB_647_c,NDB_647_c,NDB_647_c,NDB_647_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:117, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_46_pos_2566_strand_-_readouts_[NDB_899_c,NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:53, dumped\n",
      "--- Sequence:loc_7:46014999-46019999_gene_Chr7-6Mb-seg-404_pb_47_pos_2608_strand_+_readouts_[NDB_647_c,NDB_647_c,NDB_647_c,NDB_647_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:51, dumped\n",
      "--- Sequence:loc_7:46039999-46044999_gene_Chr7-6Mb-seg-409_pb_25_pos_1754_strand_+_readouts_[NDB_375_c,NDB_375_c,NDB_375_c,NDB_375_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:59, dumped\n",
      "--- Sequence:loc_7:46039999-46044999_gene_Chr7-6Mb-seg-409_pb_26_pos_1797_strand_+_readouts_[NDB_773_c,NDB_773_c,NDB_773_c,NDB_773_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:60, dumped\n",
      "--- Sequence:loc_7:46039999-46044999_gene_Chr7-6Mb-seg-409_pb_27_pos_1839_strand_-_readouts_[NDB_875_c,NDB_875_c,NDB_875_c,NDB_875_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:66, dumped\n",
      "--- Sequence:loc_7:46054999-46059999_gene_Chr7-6Mb-seg-412_pb_54_pos_3664_strand_+_readouts_[NDB_905_c,NDB_905_c,NDB_905_c,NDB_905_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:66, dumped\n",
      "--- Sequence:loc_7:46054999-46059999_gene_Chr7-6Mb-seg-412_pb_55_pos_3706_strand_+_readouts_[NDB_444_c,NDB_444_c,NDB_444_c,NDB_444_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:60, dumped\n",
      "--- Sequence:loc_7:46069999-46074999_gene_Chr7-6Mb-seg-415_pb_16_pos_1005_strand_+_readouts_[NDB_665_c,NDB_665_c,NDB_665_c,NDB_665_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:53, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_7:46179999-46184999_gene_Chr7-6Mb-seg-437_pb_17_pos_1188_strand_+_readouts_[NDB_456_c,NDB_456_c,NDB_456_c,NDB_456_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:55, dumped\n",
      "--- Sequence:loc_7:46319999-46324999_gene_Chr7-6Mb-seg-465_pb_21_pos_1492_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:54, dumped\n",
      "--- Sequence:loc_7:47269999-47274999_gene_Chr7-6Mb-seg-655_pb_14_pos_1888_strand_+_readouts_[NDB_365_c,NDB_365_c,NDB_365_c,NDB_365_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:66, dumped\n",
      "--- Sequence:loc_7:47319999-47324999_gene_Chr7-6Mb-seg-665_pb_29_pos_4721_strand_+_readouts_[NDB_740_c,NDB_740_c,NDB_740_c,NDB_740_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:60, dumped\n",
      "--- Sequence:loc_7:47374999-47379999_gene_Chr7-6Mb-seg-676_pb_13_pos_1306_strand_+_readouts_[NDB_423_c,NDB_423_c,NDB_423_c,NDB_423_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:53, dumped\n",
      "--- Sequence:loc_7:47709999-47714999_gene_Chr7-6Mb-seg-743_pb_27_pos_2543_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:66, dumped\n",
      "--- Sequence:loc_7:47729999-47734999_gene_Chr7-6Mb-seg-747_pb_18_pos_3767_strand_+_readouts_[NDB_479_c,NDB_479_c,NDB_479_c,NDB_479_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:52, dumped\n",
      "--- Sequence:loc_7:47784999-47789999_gene_Chr7-6Mb-seg-758_pb_3_pos_188_strand_+_readouts_[NDB_589_c,NDB_589_c,NDB_589_c,NDB_589_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:59, dumped\n",
      "--- Sequence:loc_7:47839999-47844999_gene_Chr7-6Mb-seg-769_pb_13_pos_1245_strand_+_readouts_[NDB_538_c,NDB_538_c,NDB_538_c,NDB_538_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:52, dumped\n",
      "--- Sequence:loc_7:47924999-47929999_gene_Chr7-6Mb-seg-786_pb_29_pos_4263_strand_+_readouts_[NDB_922_c,NDB_922_c,NDB_922_c,NDB_922_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:54, dumped\n",
      "--- Sequence:loc_7:47954999-47959999_gene_Chr7-6Mb-seg-792_pb_11_pos_1045_strand_+_readouts_[NDB_940_c,NDB_940_c,NDB_940_c,NDB_940_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:56, dumped\n",
      "--- Sequence:loc_7:48289999-48294999_gene_Chr7-6Mb-seg-859_pb_12_pos_2730_strand_+_readouts_[NDB_783_c,NDB_783_c,NDB_783_c,NDB_783_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:51, dumped\n",
      "--- Sequence:loc_7:48359999-48364999_gene_Chr7-6Mb-seg-873_pb_17_pos_3459_strand_+_readouts_[NDB_708_c,NDB_708_c,NDB_708_c,NDB_708_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:51, dumped\n",
      "--- Sequence:loc_7:48989999-48994999_gene_Chr7-6Mb-seg-999_pb_52_pos_3573_strand_+_readouts_[NDB_329_c,NDB_329_c,NDB_329_c,NDB_329_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:61, dumped\n",
      "--- Sequence:loc_7:48989999-48994999_gene_Chr7-6Mb-seg-999_pb_55_pos_3737_strand_+_readouts_[NDB_329_c,NDB_329_c,NDB_329_c,NDB_329_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:97, dumped\n",
      "--- Sequence:loc_7:48989999-48994999_gene_Chr7-6Mb-seg-999_pb_56_pos_3860_strand_+_readouts_[NDB_488_c,NDB_488_c,NDB_488_c,NDB_488_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:55, dumped\n",
      "--- Sequence:loc_7:48989999-48994999_gene_Chr7-6Mb-seg-999_pb_57_pos_3902_strand_+_readouts_[NDB_891_c,NDB_891_c,NDB_891_c,NDB_891_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:99, dumped\n",
      "--- Sequence:loc_7:48989999-48994999_gene_Chr7-6Mb-seg-999_pb_58_pos_3983_strand_+_readouts_[NDB_329_c,NDB_329_c,NDB_329_c,NDB_329_c]_primers_[W1B09_primer_20,W1B10_primer_21] got hits:93, dumped\n",
      "-- total probes removed by internal screening: 114\n"
     ]
    }
   ],
   "source": [
    "# check between probes\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map, \n",
    "                                                          primer_len=primer_len, \n",
    "                                                          target_len=target_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 206,
   "id": "495f8a7d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# clear memory\n",
    "del(int_map)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ef0c0c1c",
   "metadata": {},
   "source": [
    "### 4.1.6 finish checking, save"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 207,
   "id": "0b11845a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "70719 70833\n"
     ]
    }
   ],
   "source": [
    "print(len(kept_records), len(cand_probes))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 208,
   "id": "56768f43",
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4c40e93f",
   "metadata": {},
   "source": [
    "## 4.2 BLAST screening of probes\n",
    "\n",
    "create the local blast database by:\n",
    "```bash\n",
    "makeblastdb -in C:\\Users\\puzheng\\Documents\\Tools\\lowercase_masked\\Mus_musculus.GRCm38.dna_sm.merged.fa -dbtype nucl -parse_seqids -out GRCm38_ensembl -title \"Mouse chromosomes soft masked\" -blastdb_version 4![image.png](attachment:image.png)\n",
    "```\n",
    "\n",
    "The outputs are:\n",
    "```bash\n",
    "Building a new DB, current time: 03/30/2021 21:49:18\n",
    "New DB name:   C:\\Users\\puzheng\\Documents\\Tools\\GRCm38_ensembl\n",
    "New DB title:  Mouse chromosomes soft masked\n",
    "Sequence type: Nucleotide\n",
    "Keep MBits: T\n",
    "Maximum file size: 1000000000B\n",
    "Adding sequences from FASTA; added 66 sequences in 42.2668 seconds.\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 209,
   "id": "5045ff8f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<module 'ImageAnalysis3.library_tools.quality_check' from 'E:\\\\Users\\\\puzheng_new\\\\Documents\\\\Chromatin_Analysis_Scripts\\\\Library_design\\\\CTP-14-human_brain_genome\\\\..\\\\..\\\\..\\\\..\\\\Documents\\\\ImageAnalysis3\\\\library_tools\\\\quality_check.py'>"
      ]
     },
     "execution_count": 209,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 210,
   "id": "bef107d6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "70719\n"
     ]
    }
   ],
   "source": [
    "if 'kept_records' not in locals() or len(kept_records) == 0:\n",
    "    kept_records = []\n",
    "    with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'r') as handle:\n",
    "        for record in SeqIO.parse(handle, \"fasta\"):\n",
    "            kept_records.append(record)\n",
    "print(len(kept_records))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 212,
   "id": "75e15498",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\blast_database\\GRCm38_ensembl\n",
      "-- number of probes loaded: 70719\n",
      "-- processing region: Chr7-6Mb-seg-1 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1\n",
      "--- total time for blast Chr7-6Mb-seg-1: 5.539s.\n",
      "-- processing region: Chr7-6Mb-seg-10 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-10.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-10\n",
      "--- total time for blast Chr7-6Mb-seg-10: 2.867s.\n",
      "-- processing region: Chr7-6Mb-seg-100 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-100.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-100\n",
      "--- total time for blast Chr7-6Mb-seg-100: 5.308s.\n",
      "-- processing region: Chr7-6Mb-seg-1000 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1000.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1000\n",
      "--- total time for blast Chr7-6Mb-seg-1000: 4.848s.\n",
      "-- processing region: Chr7-6Mb-seg-1001 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1001.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1001\n",
      "--- total time for blast Chr7-6Mb-seg-1001: 3.762s.\n",
      "-- processing region: Chr7-6Mb-seg-1002 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1002.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1002\n",
      "--- total time for blast Chr7-6Mb-seg-1002: 3.766s.\n",
      "-- processing region: Chr7-6Mb-seg-1003 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1003.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1003\n",
      "--- total time for blast Chr7-6Mb-seg-1003: 9.753s.\n",
      "-- processing region: Chr7-6Mb-seg-1004 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1004.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1004\n",
      "--- total time for blast Chr7-6Mb-seg-1004: 4.061s.\n",
      "-- processing region: Chr7-6Mb-seg-1005 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1005.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1005\n",
      "--- total time for blast Chr7-6Mb-seg-1005: 4.178s.\n",
      "-- processing region: Chr7-6Mb-seg-1006 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1006.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1006\n",
      "--- total time for blast Chr7-6Mb-seg-1006: 4.498s.\n",
      "-- processing region: Chr7-6Mb-seg-1007 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1007.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1007\n",
      "--- total time for blast Chr7-6Mb-seg-1007: 4.172s.\n",
      "-- processing region: Chr7-6Mb-seg-1008 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1008.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1008\n",
      "--- total time for blast Chr7-6Mb-seg-1008: 4.080s.\n",
      "-- processing region: Chr7-6Mb-seg-1009 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1009.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1009\n",
      "--- total time for blast Chr7-6Mb-seg-1009: 3.692s.\n",
      "-- processing region: Chr7-6Mb-seg-101 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-101.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-101\n",
      "--- total time for blast Chr7-6Mb-seg-101: 4.400s.\n",
      "-- processing region: Chr7-6Mb-seg-1010 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1010.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1010\n",
      "--- total time for blast Chr7-6Mb-seg-1010: 3.673s.\n",
      "-- processing region: Chr7-6Mb-seg-1011 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1011.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1011\n",
      "--- total time for blast Chr7-6Mb-seg-1011: 4.556s.\n",
      "-- processing region: Chr7-6Mb-seg-1012 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1012.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1012\n",
      "--- total time for blast Chr7-6Mb-seg-1012: 4.021s.\n",
      "-- processing region: Chr7-6Mb-seg-1013 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1013.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1013\n",
      "--- total time for blast Chr7-6Mb-seg-1013: 4.418s.\n",
      "-- processing region: Chr7-6Mb-seg-1014 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1014.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1014\n",
      "--- total time for blast Chr7-6Mb-seg-1014: 4.676s.\n",
      "-- processing region: Chr7-6Mb-seg-1015 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1015.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1015\n",
      "--- total time for blast Chr7-6Mb-seg-1015: 4.551s.\n",
      "-- processing region: Chr7-6Mb-seg-1016 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1016.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1016\n",
      "--- total time for blast Chr7-6Mb-seg-1016: 3.732s.\n",
      "-- processing region: Chr7-6Mb-seg-1017 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1017.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1017\n",
      "--- total time for blast Chr7-6Mb-seg-1017: 3.685s.\n",
      "-- processing region: Chr7-6Mb-seg-1018 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1018.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1018\n",
      "--- total time for blast Chr7-6Mb-seg-1018: 4.042s.\n",
      "-- processing region: Chr7-6Mb-seg-1019 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1019.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1019\n",
      "--- total time for blast Chr7-6Mb-seg-1019: 4.408s.\n",
      "-- processing region: Chr7-6Mb-seg-102 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-102.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-102\n",
      "--- total time for blast Chr7-6Mb-seg-102: 4.585s.\n",
      "-- processing region: Chr7-6Mb-seg-1020 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1020.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1020\n",
      "--- total time for blast Chr7-6Mb-seg-1020: 3.876s.\n",
      "-- processing region: Chr7-6Mb-seg-1021 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1021.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1021\n",
      "--- total time for blast Chr7-6Mb-seg-1021: 3.862s.\n",
      "-- processing region: Chr7-6Mb-seg-1022 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1022.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1022\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-1022: 5.469s.\n",
      "-- processing region: Chr7-6Mb-seg-1023 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1023.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1023\n",
      "--- total time for blast Chr7-6Mb-seg-1023: 6.708s.\n",
      "-- processing region: Chr7-6Mb-seg-1024 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1024.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1024\n",
      "--- total time for blast Chr7-6Mb-seg-1024: 3.872s.\n",
      "-- processing region: Chr7-6Mb-seg-1025 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1025.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1025\n",
      "--- total time for blast Chr7-6Mb-seg-1025: 3.520s.\n",
      "-- processing region: Chr7-6Mb-seg-1026 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1026.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1026\n",
      "--- total time for blast Chr7-6Mb-seg-1026: 4.305s.\n",
      "-- processing region: Chr7-6Mb-seg-1027 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1027.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1027\n",
      "--- total time for blast Chr7-6Mb-seg-1027: 3.896s.\n",
      "-- processing region: Chr7-6Mb-seg-1028 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1028.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1028\n",
      "--- total time for blast Chr7-6Mb-seg-1028: 4.078s.\n",
      "-- processing region: Chr7-6Mb-seg-1029 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1029.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1029\n",
      "--- total time for blast Chr7-6Mb-seg-1029: 4.558s.\n",
      "-- processing region: Chr7-6Mb-seg-103 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-103.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-103\n",
      "--- total time for blast Chr7-6Mb-seg-103: 3.381s.\n",
      "-- processing region: Chr7-6Mb-seg-1030 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1030.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1030\n",
      "--- total time for blast Chr7-6Mb-seg-1030: 8.209s.\n",
      "-- processing region: Chr7-6Mb-seg-1031 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1031.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1031\n",
      "--- total time for blast Chr7-6Mb-seg-1031: 4.399s.\n",
      "-- processing region: Chr7-6Mb-seg-1032 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1032.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1032\n",
      "--- total time for blast Chr7-6Mb-seg-1032: 4.238s.\n",
      "-- processing region: Chr7-6Mb-seg-1033 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1033.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1033\n",
      "--- total time for blast Chr7-6Mb-seg-1033: 4.271s.\n",
      "-- processing region: Chr7-6Mb-seg-1034 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1034.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1034\n",
      "--- total time for blast Chr7-6Mb-seg-1034: 5.480s.\n",
      "-- processing region: Chr7-6Mb-seg-1035 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1035.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1035\n",
      "--- total time for blast Chr7-6Mb-seg-1035: 4.455s.\n",
      "-- processing region: Chr7-6Mb-seg-1036 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1036.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1036\n",
      "--- total time for blast Chr7-6Mb-seg-1036: 4.298s.\n",
      "-- processing region: Chr7-6Mb-seg-1037 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1037.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1037\n",
      "--- total time for blast Chr7-6Mb-seg-1037: 4.620s.\n",
      "-- processing region: Chr7-6Mb-seg-1038 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1038.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1038\n",
      "--- total time for blast Chr7-6Mb-seg-1038: 4.580s.\n",
      "-- processing region: Chr7-6Mb-seg-1039 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1039.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1039\n",
      "--- total time for blast Chr7-6Mb-seg-1039: 5.004s.\n",
      "-- processing region: Chr7-6Mb-seg-104 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-104.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-104\n",
      "--- total time for blast Chr7-6Mb-seg-104: 4.287s.\n",
      "-- processing region: Chr7-6Mb-seg-1040 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1040.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1040\n",
      "--- total time for blast Chr7-6Mb-seg-1040: 4.841s.\n",
      "-- processing region: Chr7-6Mb-seg-1041 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1041.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1041\n",
      "--- total time for blast Chr7-6Mb-seg-1041: 3.847s.\n",
      "-- processing region: Chr7-6Mb-seg-1042 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1042.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1042\n",
      "--- total time for blast Chr7-6Mb-seg-1042: 3.538s.\n",
      "-- processing region: Chr7-6Mb-seg-1043 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1043.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1043\n",
      "--- total time for blast Chr7-6Mb-seg-1043: 3.891s.\n",
      "-- processing region: Chr7-6Mb-seg-1044 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1044.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1044\n",
      "--- total time for blast Chr7-6Mb-seg-1044: 3.577s.\n",
      "-- processing region: Chr7-6Mb-seg-1045 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1045.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1045\n",
      "--- total time for blast Chr7-6Mb-seg-1045: 4.205s.\n",
      "-- processing region: Chr7-6Mb-seg-1046 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1046.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1046\n",
      "--- total time for blast Chr7-6Mb-seg-1046: 6.287s.\n",
      "-- processing region: Chr7-6Mb-seg-1047 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1047.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1047\n",
      "--- total time for blast Chr7-6Mb-seg-1047: 4.450s.\n",
      "-- processing region: Chr7-6Mb-seg-1048 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1048.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1048\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-1048: 3.315s.\n",
      "-- processing region: Chr7-6Mb-seg-1049 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1049.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1049\n",
      "--- total time for blast Chr7-6Mb-seg-1049: 4.195s.\n",
      "-- processing region: Chr7-6Mb-seg-105 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-105.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-105\n",
      "--- total time for blast Chr7-6Mb-seg-105: 3.838s.\n",
      "-- processing region: Chr7-6Mb-seg-1050 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1050.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1050\n",
      "--- total time for blast Chr7-6Mb-seg-1050: 3.612s.\n",
      "-- processing region: Chr7-6Mb-seg-1051 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1051.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1051\n",
      "--- total time for blast Chr7-6Mb-seg-1051: 3.870s.\n",
      "-- processing region: Chr7-6Mb-seg-1052 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1052.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1052\n",
      "--- total time for blast Chr7-6Mb-seg-1052: 4.073s.\n",
      "-- processing region: Chr7-6Mb-seg-1053 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1053.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1053\n",
      "--- total time for blast Chr7-6Mb-seg-1053: 4.110s.\n",
      "-- processing region: Chr7-6Mb-seg-1054 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1054.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1054\n",
      "--- total time for blast Chr7-6Mb-seg-1054: 3.824s.\n",
      "-- processing region: Chr7-6Mb-seg-1055 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1055.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1055\n",
      "--- total time for blast Chr7-6Mb-seg-1055: 3.958s.\n",
      "-- processing region: Chr7-6Mb-seg-1056 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1056.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1056\n",
      "--- total time for blast Chr7-6Mb-seg-1056: 4.227s.\n",
      "-- processing region: Chr7-6Mb-seg-1057 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1057.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1057\n",
      "--- total time for blast Chr7-6Mb-seg-1057: 4.054s.\n",
      "-- processing region: Chr7-6Mb-seg-1058 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1058.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1058\n",
      "--- total time for blast Chr7-6Mb-seg-1058: 4.063s.\n",
      "-- processing region: Chr7-6Mb-seg-1059 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1059.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1059\n",
      "--- total time for blast Chr7-6Mb-seg-1059: 4.258s.\n",
      "-- processing region: Chr7-6Mb-seg-106 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-106.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-106\n",
      "--- total time for blast Chr7-6Mb-seg-106: 4.808s.\n",
      "-- processing region: Chr7-6Mb-seg-1060 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1060.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1060\n",
      "--- total time for blast Chr7-6Mb-seg-1060: 4.563s.\n",
      "-- processing region: Chr7-6Mb-seg-1061 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1061.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1061\n",
      "--- total time for blast Chr7-6Mb-seg-1061: 3.991s.\n",
      "-- processing region: Chr7-6Mb-seg-1062 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1062.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1062\n",
      "--- total time for blast Chr7-6Mb-seg-1062: 3.792s.\n",
      "-- processing region: Chr7-6Mb-seg-1063 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1063.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1063\n",
      "--- total time for blast Chr7-6Mb-seg-1063: 4.471s.\n",
      "-- processing region: Chr7-6Mb-seg-1064 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1064.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1064\n",
      "--- total time for blast Chr7-6Mb-seg-1064: 4.968s.\n",
      "-- processing region: Chr7-6Mb-seg-1065 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1065.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1065\n",
      "--- total time for blast Chr7-6Mb-seg-1065: 4.327s.\n",
      "-- processing region: Chr7-6Mb-seg-1066 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1066.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1066\n",
      "--- total time for blast Chr7-6Mb-seg-1066: 3.710s.\n",
      "-- processing region: Chr7-6Mb-seg-1067 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1067.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1067\n",
      "--- total time for blast Chr7-6Mb-seg-1067: 3.466s.\n",
      "-- processing region: Chr7-6Mb-seg-1068 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1068.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1068\n",
      "--- total time for blast Chr7-6Mb-seg-1068: 3.989s.\n",
      "-- processing region: Chr7-6Mb-seg-1069 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1069.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1069\n",
      "--- total time for blast Chr7-6Mb-seg-1069: 5.069s.\n",
      "-- processing region: Chr7-6Mb-seg-107 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-107.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-107\n",
      "--- total time for blast Chr7-6Mb-seg-107: 3.685s.\n",
      "-- processing region: Chr7-6Mb-seg-1070 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1070.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1070\n",
      "--- total time for blast Chr7-6Mb-seg-1070: 9.157s.\n",
      "-- processing region: Chr7-6Mb-seg-1071 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1071.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1071\n",
      "--- total time for blast Chr7-6Mb-seg-1071: 3.725s.\n",
      "-- processing region: Chr7-6Mb-seg-1072 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1072.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1072\n",
      "--- total time for blast Chr7-6Mb-seg-1072: 25.057s.\n",
      "-- processing region: Chr7-6Mb-seg-1073 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1073.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1073\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-1073: 5.767s.\n",
      "-- processing region: Chr7-6Mb-seg-1074 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1074.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1074\n",
      "--- total time for blast Chr7-6Mb-seg-1074: 4.024s.\n",
      "-- processing region: Chr7-6Mb-seg-1075 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1075.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1075\n",
      "--- total time for blast Chr7-6Mb-seg-1075: 4.143s.\n",
      "-- processing region: Chr7-6Mb-seg-1076 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1076.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1076\n",
      "--- total time for blast Chr7-6Mb-seg-1076: 4.154s.\n",
      "-- processing region: Chr7-6Mb-seg-1077 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1077.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1077\n",
      "--- total time for blast Chr7-6Mb-seg-1077: 4.837s.\n",
      "-- processing region: Chr7-6Mb-seg-1078 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1078.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1078\n",
      "--- total time for blast Chr7-6Mb-seg-1078: 3.392s.\n",
      "-- processing region: Chr7-6Mb-seg-1079 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1079.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1079\n",
      "--- total time for blast Chr7-6Mb-seg-1079: 4.592s.\n",
      "-- processing region: Chr7-6Mb-seg-108 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-108.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-108\n",
      "--- total time for blast Chr7-6Mb-seg-108: 3.440s.\n",
      "-- processing region: Chr7-6Mb-seg-1080 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1080.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1080\n",
      "--- total time for blast Chr7-6Mb-seg-1080: 3.693s.\n",
      "-- processing region: Chr7-6Mb-seg-1081 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1081.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1081\n",
      "--- total time for blast Chr7-6Mb-seg-1081: 4.268s.\n",
      "-- processing region: Chr7-6Mb-seg-1082 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1082.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1082\n",
      "--- total time for blast Chr7-6Mb-seg-1082: 3.901s.\n",
      "-- processing region: Chr7-6Mb-seg-1083 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1083.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1083\n",
      "--- total time for blast Chr7-6Mb-seg-1083: 10.151s.\n",
      "-- processing region: Chr7-6Mb-seg-1084 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1084.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1084\n",
      "--- total time for blast Chr7-6Mb-seg-1084: 4.897s.\n",
      "-- processing region: Chr7-6Mb-seg-1085 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1085.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1085\n",
      "--- total time for blast Chr7-6Mb-seg-1085: 3.629s.\n",
      "-- processing region: Chr7-6Mb-seg-1086 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1086.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1086\n",
      "--- total time for blast Chr7-6Mb-seg-1086: 5.741s.\n",
      "-- processing region: Chr7-6Mb-seg-1087 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1087.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1087\n",
      "--- total time for blast Chr7-6Mb-seg-1087: 3.458s.\n",
      "-- processing region: Chr7-6Mb-seg-1088 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1088.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1088\n",
      "--- total time for blast Chr7-6Mb-seg-1088: 3.881s.\n",
      "-- processing region: Chr7-6Mb-seg-1089 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1089.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1089\n",
      "--- total time for blast Chr7-6Mb-seg-1089: 4.786s.\n",
      "-- processing region: Chr7-6Mb-seg-109 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-109.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-109\n",
      "--- total time for blast Chr7-6Mb-seg-109: 3.732s.\n",
      "-- processing region: Chr7-6Mb-seg-1090 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1090.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1090\n",
      "--- total time for blast Chr7-6Mb-seg-1090: 3.740s.\n",
      "-- processing region: Chr7-6Mb-seg-1091 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1091.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1091\n",
      "--- total time for blast Chr7-6Mb-seg-1091: 3.696s.\n",
      "-- processing region: Chr7-6Mb-seg-1092 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1092.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1092\n",
      "--- total time for blast Chr7-6Mb-seg-1092: 3.519s.\n",
      "-- processing region: Chr7-6Mb-seg-1093 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1093.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1093\n",
      "--- total time for blast Chr7-6Mb-seg-1093: 3.774s.\n",
      "-- processing region: Chr7-6Mb-seg-1094 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1094.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1094\n",
      "--- total time for blast Chr7-6Mb-seg-1094: 4.213s.\n",
      "-- processing region: Chr7-6Mb-seg-1095 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1095.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1095\n",
      "--- total time for blast Chr7-6Mb-seg-1095: 3.841s.\n",
      "-- processing region: Chr7-6Mb-seg-1096 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1096.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1096\n",
      "--- total time for blast Chr7-6Mb-seg-1096: 4.391s.\n",
      "-- processing region: Chr7-6Mb-seg-1097 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1097.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1097\n",
      "--- total time for blast Chr7-6Mb-seg-1097: 5.304s.\n",
      "-- processing region: Chr7-6Mb-seg-1098 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1098.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1098\n",
      "--- total time for blast Chr7-6Mb-seg-1098: 3.469s.\n",
      "-- processing region: Chr7-6Mb-seg-1099 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1099.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1099\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-1099: 4.738s.\n",
      "-- processing region: Chr7-6Mb-seg-11 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-11.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-11\n",
      "--- total time for blast Chr7-6Mb-seg-11: 3.227s.\n",
      "-- processing region: Chr7-6Mb-seg-110 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-110.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-110\n",
      "--- total time for blast Chr7-6Mb-seg-110: 3.714s.\n",
      "-- processing region: Chr7-6Mb-seg-1100 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1100.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1100\n",
      "--- total time for blast Chr7-6Mb-seg-1100: 6.899s.\n",
      "-- processing region: Chr7-6Mb-seg-1101 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1101.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1101\n",
      "--- total time for blast Chr7-6Mb-seg-1101: 4.769s.\n",
      "-- processing region: Chr7-6Mb-seg-1102 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1102.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1102\n",
      "--- total time for blast Chr7-6Mb-seg-1102: 5.896s.\n",
      "-- processing region: Chr7-6Mb-seg-1103 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1103.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1103\n",
      "--- total time for blast Chr7-6Mb-seg-1103: 4.676s.\n",
      "-- processing region: Chr7-6Mb-seg-1104 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1104.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1104\n",
      "--- total time for blast Chr7-6Mb-seg-1104: 4.695s.\n",
      "-- processing region: Chr7-6Mb-seg-1105 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1105.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1105\n",
      "--- total time for blast Chr7-6Mb-seg-1105: 4.250s.\n",
      "-- processing region: Chr7-6Mb-seg-1106 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1106.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1106\n",
      "--- total time for blast Chr7-6Mb-seg-1106: 4.073s.\n",
      "-- processing region: Chr7-6Mb-seg-1107 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1107.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1107\n",
      "--- total time for blast Chr7-6Mb-seg-1107: 3.658s.\n",
      "-- processing region: Chr7-6Mb-seg-1108 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1108.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1108\n",
      "--- total time for blast Chr7-6Mb-seg-1108: 4.143s.\n",
      "-- processing region: Chr7-6Mb-seg-1109 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1109.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1109\n",
      "--- total time for blast Chr7-6Mb-seg-1109: 4.281s.\n",
      "-- processing region: Chr7-6Mb-seg-111 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-111.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-111\n",
      "--- total time for blast Chr7-6Mb-seg-111: 3.484s.\n",
      "-- processing region: Chr7-6Mb-seg-1110 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1110.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1110\n",
      "--- total time for blast Chr7-6Mb-seg-1110: 3.768s.\n",
      "-- processing region: Chr7-6Mb-seg-1111 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1111.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1111\n",
      "--- total time for blast Chr7-6Mb-seg-1111: 3.969s.\n",
      "-- processing region: Chr7-6Mb-seg-1112 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1112.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1112\n",
      "--- total time for blast Chr7-6Mb-seg-1112: 3.750s.\n",
      "-- processing region: Chr7-6Mb-seg-1113 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1113.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1113\n",
      "--- total time for blast Chr7-6Mb-seg-1113: 5.105s.\n",
      "-- processing region: Chr7-6Mb-seg-1114 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1114.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1114\n",
      "--- total time for blast Chr7-6Mb-seg-1114: 5.730s.\n",
      "-- processing region: Chr7-6Mb-seg-1115 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1115.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1115\n",
      "--- total time for blast Chr7-6Mb-seg-1115: 4.034s.\n",
      "-- processing region: Chr7-6Mb-seg-1116 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1116.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1116\n",
      "--- total time for blast Chr7-6Mb-seg-1116: 4.862s.\n",
      "-- processing region: Chr7-6Mb-seg-1117 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1117.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1117\n",
      "--- total time for blast Chr7-6Mb-seg-1117: 4.331s.\n",
      "-- processing region: Chr7-6Mb-seg-1118 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1118.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1118\n",
      "--- total time for blast Chr7-6Mb-seg-1118: 3.962s.\n",
      "-- processing region: Chr7-6Mb-seg-1119 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1119.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1119\n",
      "--- total time for blast Chr7-6Mb-seg-1119: 4.054s.\n",
      "-- processing region: Chr7-6Mb-seg-112 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-112.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-112\n",
      "--- total time for blast Chr7-6Mb-seg-112: 4.786s.\n",
      "-- processing region: Chr7-6Mb-seg-1120 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1120.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1120\n",
      "--- total time for blast Chr7-6Mb-seg-1120: 3.697s.\n",
      "-- processing region: Chr7-6Mb-seg-1121 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1121.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1121\n",
      "--- total time for blast Chr7-6Mb-seg-1121: 3.410s.\n",
      "-- processing region: Chr7-6Mb-seg-1122 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1122.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1122\n",
      "--- total time for blast Chr7-6Mb-seg-1122: 6.721s.\n",
      "-- processing region: Chr7-6Mb-seg-1123 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1123.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1123\n",
      "--- total time for blast Chr7-6Mb-seg-1123: 8.104s.\n",
      "-- processing region: Chr7-6Mb-seg-1124 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1124.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1124\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-1124: 3.789s.\n",
      "-- processing region: Chr7-6Mb-seg-1125 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1125.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1125\n",
      "--- total time for blast Chr7-6Mb-seg-1125: 4.471s.\n",
      "-- processing region: Chr7-6Mb-seg-1126 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1126.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1126\n",
      "--- total time for blast Chr7-6Mb-seg-1126: 3.380s.\n",
      "-- processing region: Chr7-6Mb-seg-1127 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1127.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1127\n",
      "--- total time for blast Chr7-6Mb-seg-1127: 5.272s.\n",
      "-- processing region: Chr7-6Mb-seg-1128 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1128.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1128\n",
      "--- total time for blast Chr7-6Mb-seg-1128: 3.492s.\n",
      "-- processing region: Chr7-6Mb-seg-1129 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1129.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1129\n",
      "--- total time for blast Chr7-6Mb-seg-1129: 5.187s.\n",
      "-- processing region: Chr7-6Mb-seg-113 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-113.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-113\n",
      "--- total time for blast Chr7-6Mb-seg-113: 3.769s.\n",
      "-- processing region: Chr7-6Mb-seg-1130 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1130.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1130\n",
      "--- total time for blast Chr7-6Mb-seg-1130: 3.424s.\n",
      "-- processing region: Chr7-6Mb-seg-1131 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1131.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1131\n",
      "--- total time for blast Chr7-6Mb-seg-1131: 4.246s.\n",
      "-- processing region: Chr7-6Mb-seg-1132 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1132.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1132\n",
      "--- total time for blast Chr7-6Mb-seg-1132: 3.673s.\n",
      "-- processing region: Chr7-6Mb-seg-1133 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1133.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1133\n",
      "--- total time for blast Chr7-6Mb-seg-1133: 3.699s.\n",
      "-- processing region: Chr7-6Mb-seg-1134 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1134.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1134\n",
      "--- total time for blast Chr7-6Mb-seg-1134: 4.132s.\n",
      "-- processing region: Chr7-6Mb-seg-1135 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1135.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1135\n",
      "--- total time for blast Chr7-6Mb-seg-1135: 4.719s.\n",
      "-- processing region: Chr7-6Mb-seg-1136 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1136.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1136\n",
      "--- total time for blast Chr7-6Mb-seg-1136: 3.907s.\n",
      "-- processing region: Chr7-6Mb-seg-1137 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1137.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1137\n",
      "--- total time for blast Chr7-6Mb-seg-1137: 4.153s.\n",
      "-- processing region: Chr7-6Mb-seg-1138 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1138.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1138\n",
      "--- total time for blast Chr7-6Mb-seg-1138: 3.433s.\n",
      "-- processing region: Chr7-6Mb-seg-1139 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1139.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1139\n",
      "--- total time for blast Chr7-6Mb-seg-1139: 3.671s.\n",
      "-- processing region: Chr7-6Mb-seg-114 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-114.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-114\n",
      "--- total time for blast Chr7-6Mb-seg-114: 5.783s.\n",
      "-- processing region: Chr7-6Mb-seg-1140 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1140.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1140\n",
      "--- total time for blast Chr7-6Mb-seg-1140: 4.101s.\n",
      "-- processing region: Chr7-6Mb-seg-1141 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1141.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1141\n",
      "--- total time for blast Chr7-6Mb-seg-1141: 4.000s.\n",
      "-- processing region: Chr7-6Mb-seg-1142 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1142.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1142\n",
      "--- total time for blast Chr7-6Mb-seg-1142: 4.337s.\n",
      "-- processing region: Chr7-6Mb-seg-1143 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1143.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1143\n",
      "--- total time for blast Chr7-6Mb-seg-1143: 4.694s.\n",
      "-- processing region: Chr7-6Mb-seg-1144 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1144.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1144\n",
      "--- total time for blast Chr7-6Mb-seg-1144: 3.353s.\n",
      "-- processing region: Chr7-6Mb-seg-1145 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1145.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1145\n",
      "--- total time for blast Chr7-6Mb-seg-1145: 3.381s.\n",
      "-- processing region: Chr7-6Mb-seg-1146 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1146.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1146\n",
      "--- total time for blast Chr7-6Mb-seg-1146: 3.354s.\n",
      "-- processing region: Chr7-6Mb-seg-1147 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1147.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1147\n",
      "--- total time for blast Chr7-6Mb-seg-1147: 6.596s.\n",
      "-- processing region: Chr7-6Mb-seg-1148 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1148.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1148\n",
      "--- total time for blast Chr7-6Mb-seg-1148: 5.126s.\n",
      "-- processing region: Chr7-6Mb-seg-1149 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1149.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1149\n",
      "--- total time for blast Chr7-6Mb-seg-1149: 4.139s.\n",
      "-- processing region: Chr7-6Mb-seg-115 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-115.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-115\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-115: 5.476s.\n",
      "-- processing region: Chr7-6Mb-seg-1150 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1150.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1150\n",
      "--- total time for blast Chr7-6Mb-seg-1150: 2.938s.\n",
      "-- processing region: Chr7-6Mb-seg-1151 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1151.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1151\n",
      "--- total time for blast Chr7-6Mb-seg-1151: 3.260s.\n",
      "-- processing region: Chr7-6Mb-seg-1152 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1152.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1152\n",
      "--- total time for blast Chr7-6Mb-seg-1152: 5.456s.\n",
      "-- processing region: Chr7-6Mb-seg-1153 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1153.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1153\n",
      "--- total time for blast Chr7-6Mb-seg-1153: 4.201s.\n",
      "-- processing region: Chr7-6Mb-seg-1154 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1154.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1154\n",
      "--- total time for blast Chr7-6Mb-seg-1154: 4.425s.\n",
      "-- processing region: Chr7-6Mb-seg-1155 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1155.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1155\n",
      "--- total time for blast Chr7-6Mb-seg-1155: 4.946s.\n",
      "-- processing region: Chr7-6Mb-seg-1156 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1156.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1156\n",
      "--- total time for blast Chr7-6Mb-seg-1156: 3.974s.\n",
      "-- processing region: Chr7-6Mb-seg-1157 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1157.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1157\n",
      "--- total time for blast Chr7-6Mb-seg-1157: 4.621s.\n",
      "-- processing region: Chr7-6Mb-seg-1158 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1158.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1158\n",
      "--- total time for blast Chr7-6Mb-seg-1158: 3.284s.\n",
      "-- processing region: Chr7-6Mb-seg-1159 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1159.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1159\n",
      "--- total time for blast Chr7-6Mb-seg-1159: 3.171s.\n",
      "-- processing region: Chr7-6Mb-seg-116 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-116.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-116\n",
      "--- total time for blast Chr7-6Mb-seg-116: 4.994s.\n",
      "-- processing region: Chr7-6Mb-seg-1160 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1160.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1160\n",
      "--- total time for blast Chr7-6Mb-seg-1160: 4.855s.\n",
      "-- processing region: Chr7-6Mb-seg-1161 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1161.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1161\n",
      "--- total time for blast Chr7-6Mb-seg-1161: 4.206s.\n",
      "-- processing region: Chr7-6Mb-seg-1162 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1162.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1162\n",
      "--- total time for blast Chr7-6Mb-seg-1162: 4.006s.\n",
      "-- processing region: Chr7-6Mb-seg-1163 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1163.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1163\n",
      "--- total time for blast Chr7-6Mb-seg-1163: 4.212s.\n",
      "-- processing region: Chr7-6Mb-seg-1164 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1164.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1164\n",
      "--- total time for blast Chr7-6Mb-seg-1164: 4.154s.\n",
      "-- processing region: Chr7-6Mb-seg-1165 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1165.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1165\n",
      "--- total time for blast Chr7-6Mb-seg-1165: 5.009s.\n",
      "-- processing region: Chr7-6Mb-seg-1166 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1166.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1166\n",
      "--- total time for blast Chr7-6Mb-seg-1166: 3.870s.\n",
      "-- processing region: Chr7-6Mb-seg-1167 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1167.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1167\n",
      "--- total time for blast Chr7-6Mb-seg-1167: 3.389s.\n",
      "-- processing region: Chr7-6Mb-seg-1168 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1168.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1168\n",
      "--- total time for blast Chr7-6Mb-seg-1168: 4.354s.\n",
      "-- processing region: Chr7-6Mb-seg-1169 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1169.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1169\n",
      "--- total time for blast Chr7-6Mb-seg-1169: 4.477s.\n",
      "-- processing region: Chr7-6Mb-seg-117 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-117.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-117\n",
      "--- total time for blast Chr7-6Mb-seg-117: 3.956s.\n",
      "-- processing region: Chr7-6Mb-seg-1170 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1170.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1170\n",
      "--- total time for blast Chr7-6Mb-seg-1170: 3.744s.\n",
      "-- processing region: Chr7-6Mb-seg-1171 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1171.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1171\n",
      "--- total time for blast Chr7-6Mb-seg-1171: 4.494s.\n",
      "-- processing region: Chr7-6Mb-seg-1172 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1172.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1172\n",
      "--- total time for blast Chr7-6Mb-seg-1172: 3.913s.\n",
      "-- processing region: Chr7-6Mb-seg-1173 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1173.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1173\n",
      "--- total time for blast Chr7-6Mb-seg-1173: 4.878s.\n",
      "-- processing region: Chr7-6Mb-seg-1174 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1174.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1174\n",
      "--- total time for blast Chr7-6Mb-seg-1174: 5.422s.\n",
      "-- processing region: Chr7-6Mb-seg-1175 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1175.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1175\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-1175: 5.155s.\n",
      "-- processing region: Chr7-6Mb-seg-1176 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1176.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1176\n",
      "--- total time for blast Chr7-6Mb-seg-1176: 3.392s.\n",
      "-- processing region: Chr7-6Mb-seg-1177 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1177.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1177\n",
      "--- total time for blast Chr7-6Mb-seg-1177: 3.949s.\n",
      "-- processing region: Chr7-6Mb-seg-1178 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1178.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1178\n",
      "--- total time for blast Chr7-6Mb-seg-1178: 3.970s.\n",
      "-- processing region: Chr7-6Mb-seg-1179 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1179.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1179\n",
      "--- total time for blast Chr7-6Mb-seg-1179: 4.024s.\n",
      "-- processing region: Chr7-6Mb-seg-118 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-118.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-118\n",
      "--- total time for blast Chr7-6Mb-seg-118: 4.210s.\n",
      "-- processing region: Chr7-6Mb-seg-1180 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1180.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1180\n",
      "--- total time for blast Chr7-6Mb-seg-1180: 3.746s.\n",
      "-- processing region: Chr7-6Mb-seg-1181 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1181.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1181\n",
      "--- total time for blast Chr7-6Mb-seg-1181: 4.277s.\n",
      "-- processing region: Chr7-6Mb-seg-1182 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1182.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1182\n",
      "--- total time for blast Chr7-6Mb-seg-1182: 3.502s.\n",
      "-- processing region: Chr7-6Mb-seg-1183 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1183.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1183\n",
      "--- total time for blast Chr7-6Mb-seg-1183: 9.725s.\n",
      "-- processing region: Chr7-6Mb-seg-1184 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1184.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1184\n",
      "--- total time for blast Chr7-6Mb-seg-1184: 5.268s.\n",
      "-- processing region: Chr7-6Mb-seg-1185 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1185.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1185\n",
      "--- total time for blast Chr7-6Mb-seg-1185: 3.717s.\n",
      "-- processing region: Chr7-6Mb-seg-1186 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1186.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1186\n",
      "--- total time for blast Chr7-6Mb-seg-1186: 4.211s.\n",
      "-- processing region: Chr7-6Mb-seg-1187 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1187.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1187\n",
      "--- total time for blast Chr7-6Mb-seg-1187: 4.006s.\n",
      "-- processing region: Chr7-6Mb-seg-1188 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1188.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1188\n",
      "--- total time for blast Chr7-6Mb-seg-1188: 4.195s.\n",
      "-- processing region: Chr7-6Mb-seg-1189 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1189.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1189\n",
      "--- total time for blast Chr7-6Mb-seg-1189: 5.746s.\n",
      "-- processing region: Chr7-6Mb-seg-119 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-119.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-119\n",
      "--- total time for blast Chr7-6Mb-seg-119: 3.300s.\n",
      "-- processing region: Chr7-6Mb-seg-1190 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1190.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1190\n",
      "--- total time for blast Chr7-6Mb-seg-1190: 4.081s.\n",
      "-- processing region: Chr7-6Mb-seg-1191 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1191.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1191\n",
      "--- total time for blast Chr7-6Mb-seg-1191: 3.508s.\n",
      "-- processing region: Chr7-6Mb-seg-1192 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1192.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1192\n",
      "--- total time for blast Chr7-6Mb-seg-1192: 3.937s.\n",
      "-- processing region: Chr7-6Mb-seg-1193 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1193.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1193\n",
      "--- total time for blast Chr7-6Mb-seg-1193: 4.166s.\n",
      "-- processing region: Chr7-6Mb-seg-1194 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1194.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1194\n",
      "--- total time for blast Chr7-6Mb-seg-1194: 4.966s.\n",
      "-- processing region: Chr7-6Mb-seg-1195 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1195.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1195\n",
      "--- total time for blast Chr7-6Mb-seg-1195: 4.654s.\n",
      "-- processing region: Chr7-6Mb-seg-1196 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1196.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1196\n",
      "--- total time for blast Chr7-6Mb-seg-1196: 4.370s.\n",
      "-- processing region: Chr7-6Mb-seg-1197 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1197.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1197\n",
      "--- total time for blast Chr7-6Mb-seg-1197: 3.949s.\n",
      "-- processing region: Chr7-6Mb-seg-1198 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1198.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1198\n",
      "--- total time for blast Chr7-6Mb-seg-1198: 6.039s.\n",
      "-- processing region: Chr7-6Mb-seg-1199 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1199.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1199\n",
      "--- total time for blast Chr7-6Mb-seg-1199: 3.940s.\n",
      "-- processing region: Chr7-6Mb-seg-12 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-12.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-12\n",
      "--- total time for blast Chr7-6Mb-seg-12: 3.352s.\n",
      "-- processing region: Chr7-6Mb-seg-120 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-120.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-120\n",
      "--- total time for blast Chr7-6Mb-seg-120: 4.371s.\n",
      "-- processing region: Chr7-6Mb-seg-1200 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-1200.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-1200\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-1200: 4.511s.\n",
      "-- processing region: Chr7-6Mb-seg-121 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-121.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-121\n",
      "--- total time for blast Chr7-6Mb-seg-121: 3.522s.\n",
      "-- processing region: Chr7-6Mb-seg-122 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-122.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-122\n",
      "--- total time for blast Chr7-6Mb-seg-122: 4.196s.\n",
      "-- processing region: Chr7-6Mb-seg-123 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-123.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-123\n",
      "--- total time for blast Chr7-6Mb-seg-123: 3.951s.\n",
      "-- processing region: Chr7-6Mb-seg-124 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-124.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-124\n",
      "--- total time for blast Chr7-6Mb-seg-124: 3.317s.\n",
      "-- processing region: Chr7-6Mb-seg-125 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-125.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-125\n",
      "--- total time for blast Chr7-6Mb-seg-125: 3.586s.\n",
      "-- processing region: Chr7-6Mb-seg-126 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-126.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-126\n",
      "--- total time for blast Chr7-6Mb-seg-126: 5.508s.\n",
      "-- processing region: Chr7-6Mb-seg-127 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-127.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-127\n",
      "--- total time for blast Chr7-6Mb-seg-127: 3.857s.\n",
      "-- processing region: Chr7-6Mb-seg-128 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-128.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-128\n",
      "--- total time for blast Chr7-6Mb-seg-128: 4.017s.\n",
      "-- processing region: Chr7-6Mb-seg-129 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-129.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-129\n",
      "--- total time for blast Chr7-6Mb-seg-129: 3.842s.\n",
      "-- processing region: Chr7-6Mb-seg-130 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-130.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-130\n",
      "--- total time for blast Chr7-6Mb-seg-130: 3.839s.\n",
      "-- processing region: Chr7-6Mb-seg-131 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-131.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-131\n",
      "--- total time for blast Chr7-6Mb-seg-131: 3.957s.\n",
      "-- processing region: Chr7-6Mb-seg-132 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-132.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-132\n",
      "--- total time for blast Chr7-6Mb-seg-132: 4.069s.\n",
      "-- processing region: Chr7-6Mb-seg-133 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-133.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-133\n",
      "--- total time for blast Chr7-6Mb-seg-133: 3.740s.\n",
      "-- processing region: Chr7-6Mb-seg-134 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-134.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-134\n",
      "--- total time for blast Chr7-6Mb-seg-134: 3.846s.\n",
      "-- processing region: Chr7-6Mb-seg-135 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-135.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-135\n",
      "--- total time for blast Chr7-6Mb-seg-135: 6.204s.\n",
      "-- processing region: Chr7-6Mb-seg-136 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-136.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-136\n",
      "--- total time for blast Chr7-6Mb-seg-136: 6.916s.\n",
      "-- processing region: Chr7-6Mb-seg-137 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-137.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-137\n",
      "--- total time for blast Chr7-6Mb-seg-137: 3.677s.\n",
      "-- processing region: Chr7-6Mb-seg-138 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-138.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-138\n",
      "--- total time for blast Chr7-6Mb-seg-138: 4.201s.\n",
      "-- processing region: Chr7-6Mb-seg-139 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-139.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-139\n",
      "--- total time for blast Chr7-6Mb-seg-139: 5.166s.\n",
      "-- processing region: Chr7-6Mb-seg-14 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-14.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-14\n",
      "--- total time for blast Chr7-6Mb-seg-14: 2.285s.\n",
      "-- processing region: Chr7-6Mb-seg-140 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-140.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-140\n",
      "--- total time for blast Chr7-6Mb-seg-140: 4.740s.\n",
      "-- processing region: Chr7-6Mb-seg-141 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-141.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-141\n",
      "--- total time for blast Chr7-6Mb-seg-141: 5.848s.\n",
      "-- processing region: Chr7-6Mb-seg-142 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-142.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-142\n",
      "--- total time for blast Chr7-6Mb-seg-142: 6.928s.\n",
      "-- processing region: Chr7-6Mb-seg-143 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-143.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-143\n",
      "--- total time for blast Chr7-6Mb-seg-143: 4.401s.\n",
      "-- processing region: Chr7-6Mb-seg-144 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-144.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-144\n",
      "--- total time for blast Chr7-6Mb-seg-144: 15.915s.\n",
      "-- processing region: Chr7-6Mb-seg-145 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-145.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-145\n",
      "--- total time for blast Chr7-6Mb-seg-145: 4.867s.\n",
      "-- processing region: Chr7-6Mb-seg-146 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-146.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-146\n",
      "--- total time for blast Chr7-6Mb-seg-146: 3.794s.\n",
      "-- processing region: Chr7-6Mb-seg-147 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-147.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-147\n",
      "--- total time for blast Chr7-6Mb-seg-147: 3.648s.\n",
      "-- processing region: Chr7-6Mb-seg-148 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-148.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-148\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-148: 4.026s.\n",
      "-- processing region: Chr7-6Mb-seg-149 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-149.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-149\n",
      "--- total time for blast Chr7-6Mb-seg-149: 4.688s.\n",
      "-- processing region: Chr7-6Mb-seg-15 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-15.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-15\n",
      "--- total time for blast Chr7-6Mb-seg-15: 4.344s.\n",
      "-- processing region: Chr7-6Mb-seg-150 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-150.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-150\n",
      "--- total time for blast Chr7-6Mb-seg-150: 3.975s.\n",
      "-- processing region: Chr7-6Mb-seg-151 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-151.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-151\n",
      "--- total time for blast Chr7-6Mb-seg-151: 3.352s.\n",
      "-- processing region: Chr7-6Mb-seg-152 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-152.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-152\n",
      "--- total time for blast Chr7-6Mb-seg-152: 4.292s.\n",
      "-- processing region: Chr7-6Mb-seg-153 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-153.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-153\n",
      "--- total time for blast Chr7-6Mb-seg-153: 4.188s.\n",
      "-- processing region: Chr7-6Mb-seg-154 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-154.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-154\n",
      "--- total time for blast Chr7-6Mb-seg-154: 4.614s.\n",
      "-- processing region: Chr7-6Mb-seg-155 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-155.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-155\n",
      "--- total time for blast Chr7-6Mb-seg-155: 4.137s.\n",
      "-- processing region: Chr7-6Mb-seg-156 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-156.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-156\n",
      "--- total time for blast Chr7-6Mb-seg-156: 4.272s.\n",
      "-- processing region: Chr7-6Mb-seg-157 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-157.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-157\n",
      "--- total time for blast Chr7-6Mb-seg-157: 3.876s.\n",
      "-- processing region: Chr7-6Mb-seg-158 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-158.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-158\n",
      "--- total time for blast Chr7-6Mb-seg-158: 4.715s.\n",
      "-- processing region: Chr7-6Mb-seg-159 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-159.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-159\n",
      "--- total time for blast Chr7-6Mb-seg-159: 3.909s.\n",
      "-- processing region: Chr7-6Mb-seg-16 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-16.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-16\n",
      "--- total time for blast Chr7-6Mb-seg-16: 5.761s.\n",
      "-- processing region: Chr7-6Mb-seg-161 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-161.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-161\n",
      "--- total time for blast Chr7-6Mb-seg-161: 2.193s.\n",
      "-- processing region: Chr7-6Mb-seg-162 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-162.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-162\n",
      "--- total time for blast Chr7-6Mb-seg-162: 5.851s.\n",
      "-- processing region: Chr7-6Mb-seg-163 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-163.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-163\n",
      "--- total time for blast Chr7-6Mb-seg-163: 5.364s.\n",
      "-- processing region: Chr7-6Mb-seg-164 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-164.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-164\n",
      "--- total time for blast Chr7-6Mb-seg-164: 3.712s.\n",
      "-- processing region: Chr7-6Mb-seg-165 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-165.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-165\n",
      "--- total time for blast Chr7-6Mb-seg-165: 3.464s.\n",
      "-- processing region: Chr7-6Mb-seg-166 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-166.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-166\n",
      "--- total time for blast Chr7-6Mb-seg-166: 4.228s.\n",
      "-- processing region: Chr7-6Mb-seg-167 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-167.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-167\n",
      "--- total time for blast Chr7-6Mb-seg-167: 4.387s.\n",
      "-- processing region: Chr7-6Mb-seg-168 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-168.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-168\n",
      "--- total time for blast Chr7-6Mb-seg-168: 3.721s.\n",
      "-- processing region: Chr7-6Mb-seg-169 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-169.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-169\n",
      "--- total time for blast Chr7-6Mb-seg-169: 3.805s.\n",
      "-- processing region: Chr7-6Mb-seg-170 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-170.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-170\n",
      "--- total time for blast Chr7-6Mb-seg-170: 3.701s.\n",
      "-- processing region: Chr7-6Mb-seg-171 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-171.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-171\n",
      "--- total time for blast Chr7-6Mb-seg-171: 3.620s.\n",
      "-- processing region: Chr7-6Mb-seg-172 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-172.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-172\n",
      "--- total time for blast Chr7-6Mb-seg-172: 3.988s.\n",
      "-- processing region: Chr7-6Mb-seg-173 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-173.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-173\n",
      "--- total time for blast Chr7-6Mb-seg-173: 3.165s.\n",
      "-- processing region: Chr7-6Mb-seg-174 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-174.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-174\n",
      "--- total time for blast Chr7-6Mb-seg-174: 3.147s.\n",
      "-- processing region: Chr7-6Mb-seg-175 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-175.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-175\n",
      "--- total time for blast Chr7-6Mb-seg-175: 4.445s.\n",
      "-- processing region: Chr7-6Mb-seg-176 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-176.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-176\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-176: 3.981s.\n",
      "-- processing region: Chr7-6Mb-seg-177 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-177.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-177\n",
      "--- total time for blast Chr7-6Mb-seg-177: 4.030s.\n",
      "-- processing region: Chr7-6Mb-seg-178 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-178.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-178\n",
      "--- total time for blast Chr7-6Mb-seg-178: 3.753s.\n",
      "-- processing region: Chr7-6Mb-seg-179 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-179.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-179\n",
      "--- total time for blast Chr7-6Mb-seg-179: 3.596s.\n",
      "-- processing region: Chr7-6Mb-seg-180 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-180.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-180\n",
      "--- total time for blast Chr7-6Mb-seg-180: 3.224s.\n",
      "-- processing region: Chr7-6Mb-seg-181 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-181.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-181\n",
      "--- total time for blast Chr7-6Mb-seg-181: 3.888s.\n",
      "-- processing region: Chr7-6Mb-seg-182 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-182.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-182\n",
      "--- total time for blast Chr7-6Mb-seg-182: 3.181s.\n",
      "-- processing region: Chr7-6Mb-seg-183 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-183.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-183\n",
      "--- total time for blast Chr7-6Mb-seg-183: 4.099s.\n",
      "-- processing region: Chr7-6Mb-seg-184 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-184.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-184\n",
      "--- total time for blast Chr7-6Mb-seg-184: 4.385s.\n",
      "-- processing region: Chr7-6Mb-seg-185 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-185.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-185\n",
      "--- total time for blast Chr7-6Mb-seg-185: 4.955s.\n",
      "-- processing region: Chr7-6Mb-seg-186 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-186.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-186\n",
      "--- total time for blast Chr7-6Mb-seg-186: 4.570s.\n",
      "-- processing region: Chr7-6Mb-seg-187 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-187.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-187\n",
      "--- total time for blast Chr7-6Mb-seg-187: 3.748s.\n",
      "-- processing region: Chr7-6Mb-seg-188 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-188.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-188\n",
      "--- total time for blast Chr7-6Mb-seg-188: 4.460s.\n",
      "-- processing region: Chr7-6Mb-seg-189 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-189.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-189\n",
      "--- total time for blast Chr7-6Mb-seg-189: 4.804s.\n",
      "-- processing region: Chr7-6Mb-seg-19 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-19.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-19\n",
      "--- total time for blast Chr7-6Mb-seg-19: 4.721s.\n",
      "-- processing region: Chr7-6Mb-seg-190 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-190.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-190\n",
      "--- total time for blast Chr7-6Mb-seg-190: 14.787s.\n",
      "-- processing region: Chr7-6Mb-seg-191 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-191.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-191\n",
      "--- total time for blast Chr7-6Mb-seg-191: 4.806s.\n",
      "-- processing region: Chr7-6Mb-seg-192 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-192.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-192\n",
      "--- total time for blast Chr7-6Mb-seg-192: 5.775s.\n",
      "-- processing region: Chr7-6Mb-seg-193 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-193.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-193\n",
      "--- total time for blast Chr7-6Mb-seg-193: 3.636s.\n",
      "-- processing region: Chr7-6Mb-seg-194 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-194.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-194\n",
      "--- total time for blast Chr7-6Mb-seg-194: 3.279s.\n",
      "-- processing region: Chr7-6Mb-seg-195 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-195.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-195\n",
      "--- total time for blast Chr7-6Mb-seg-195: 4.480s.\n",
      "-- processing region: Chr7-6Mb-seg-196 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-196.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-196\n",
      "--- total time for blast Chr7-6Mb-seg-196: 3.350s.\n",
      "-- processing region: Chr7-6Mb-seg-197 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-197.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-197\n",
      "--- total time for blast Chr7-6Mb-seg-197: 4.317s.\n",
      "-- processing region: Chr7-6Mb-seg-198 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-198.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-198\n",
      "--- total time for blast Chr7-6Mb-seg-198: 4.350s.\n",
      "-- processing region: Chr7-6Mb-seg-199 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-199.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-199\n",
      "--- total time for blast Chr7-6Mb-seg-199: 7.816s.\n",
      "-- processing region: Chr7-6Mb-seg-2 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-2.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-2\n",
      "--- total time for blast Chr7-6Mb-seg-2: 3.723s.\n",
      "-- processing region: Chr7-6Mb-seg-20 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-20.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-20\n",
      "--- total time for blast Chr7-6Mb-seg-20: 2.847s.\n",
      "-- processing region: Chr7-6Mb-seg-200 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-200.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-200\n",
      "--- total time for blast Chr7-6Mb-seg-200: 4.071s.\n",
      "-- processing region: Chr7-6Mb-seg-201 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-201.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-201\n",
      "--- total time for blast Chr7-6Mb-seg-201: 3.603s.\n",
      "-- processing region: Chr7-6Mb-seg-202 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-202.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-202\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-202: 3.466s.\n",
      "-- processing region: Chr7-6Mb-seg-203 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-203.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-203\n",
      "--- total time for blast Chr7-6Mb-seg-203: 3.669s.\n",
      "-- processing region: Chr7-6Mb-seg-204 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-204.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-204\n",
      "--- total time for blast Chr7-6Mb-seg-204: 3.355s.\n",
      "-- processing region: Chr7-6Mb-seg-205 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-205.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-205\n",
      "--- total time for blast Chr7-6Mb-seg-205: 5.000s.\n",
      "-- processing region: Chr7-6Mb-seg-206 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-206.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-206\n",
      "--- total time for blast Chr7-6Mb-seg-206: 4.086s.\n",
      "-- processing region: Chr7-6Mb-seg-207 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-207.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-207\n",
      "--- total time for blast Chr7-6Mb-seg-207: 3.804s.\n",
      "-- processing region: Chr7-6Mb-seg-208 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-208.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-208\n",
      "--- total time for blast Chr7-6Mb-seg-208: 4.161s.\n",
      "-- processing region: Chr7-6Mb-seg-209 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-209.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-209\n",
      "--- total time for blast Chr7-6Mb-seg-209: 4.549s.\n",
      "-- processing region: Chr7-6Mb-seg-21 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-21.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-21\n",
      "--- total time for blast Chr7-6Mb-seg-21: 2.674s.\n",
      "-- processing region: Chr7-6Mb-seg-210 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-210.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-210\n",
      "--- total time for blast Chr7-6Mb-seg-210: 3.139s.\n",
      "-- processing region: Chr7-6Mb-seg-211 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-211.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-211\n",
      "--- total time for blast Chr7-6Mb-seg-211: 3.613s.\n",
      "-- processing region: Chr7-6Mb-seg-212 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-212.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-212\n",
      "--- total time for blast Chr7-6Mb-seg-212: 4.345s.\n",
      "-- processing region: Chr7-6Mb-seg-213 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-213.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-213\n",
      "--- total time for blast Chr7-6Mb-seg-213: 4.203s.\n",
      "-- processing region: Chr7-6Mb-seg-214 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-214.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-214\n",
      "--- total time for blast Chr7-6Mb-seg-214: 17.902s.\n",
      "-- processing region: Chr7-6Mb-seg-215 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-215.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-215\n",
      "--- total time for blast Chr7-6Mb-seg-215: 4.795s.\n",
      "-- processing region: Chr7-6Mb-seg-216 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-216.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-216\n",
      "--- total time for blast Chr7-6Mb-seg-216: 3.883s.\n",
      "-- processing region: Chr7-6Mb-seg-217 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-217.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-217\n",
      "--- total time for blast Chr7-6Mb-seg-217: 6.594s.\n",
      "-- processing region: Chr7-6Mb-seg-218 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-218.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-218\n",
      "--- total time for blast Chr7-6Mb-seg-218: 4.281s.\n",
      "-- processing region: Chr7-6Mb-seg-219 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-219.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-219\n",
      "--- total time for blast Chr7-6Mb-seg-219: 3.985s.\n",
      "-- processing region: Chr7-6Mb-seg-22 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-22.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-22\n",
      "--- total time for blast Chr7-6Mb-seg-22: 3.676s.\n",
      "-- processing region: Chr7-6Mb-seg-220 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-220.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-220\n",
      "--- total time for blast Chr7-6Mb-seg-220: 3.728s.\n",
      "-- processing region: Chr7-6Mb-seg-221 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-221.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-221\n",
      "--- total time for blast Chr7-6Mb-seg-221: 4.816s.\n",
      "-- processing region: Chr7-6Mb-seg-222 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-222.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-222\n",
      "--- total time for blast Chr7-6Mb-seg-222: 4.486s.\n",
      "-- processing region: Chr7-6Mb-seg-223 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-223.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-223\n",
      "--- total time for blast Chr7-6Mb-seg-223: 4.878s.\n",
      "-- processing region: Chr7-6Mb-seg-224 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-224.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-224\n",
      "--- total time for blast Chr7-6Mb-seg-224: 5.626s.\n",
      "-- processing region: Chr7-6Mb-seg-225 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-225.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-225\n",
      "--- total time for blast Chr7-6Mb-seg-225: 3.358s.\n",
      "-- processing region: Chr7-6Mb-seg-226 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-226.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-226\n",
      "--- total time for blast Chr7-6Mb-seg-226: 3.372s.\n",
      "-- processing region: Chr7-6Mb-seg-227 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-227.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-227\n",
      "--- total time for blast Chr7-6Mb-seg-227: 5.513s.\n",
      "-- processing region: Chr7-6Mb-seg-228 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-228.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-228\n",
      "--- total time for blast Chr7-6Mb-seg-228: 3.836s.\n",
      "-- processing region: Chr7-6Mb-seg-229 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-229.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-229\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-229: 3.654s.\n",
      "-- processing region: Chr7-6Mb-seg-23 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-23.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-23\n",
      "--- total time for blast Chr7-6Mb-seg-23: 2.584s.\n",
      "-- processing region: Chr7-6Mb-seg-230 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-230.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-230\n",
      "--- total time for blast Chr7-6Mb-seg-230: 3.602s.\n",
      "-- processing region: Chr7-6Mb-seg-231 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-231.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-231\n",
      "--- total time for blast Chr7-6Mb-seg-231: 5.829s.\n",
      "-- processing region: Chr7-6Mb-seg-232 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-232.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-232\n",
      "--- total time for blast Chr7-6Mb-seg-232: 4.279s.\n",
      "-- processing region: Chr7-6Mb-seg-233 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-233.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-233\n",
      "--- total time for blast Chr7-6Mb-seg-233: 5.036s.\n",
      "-- processing region: Chr7-6Mb-seg-234 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-234.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-234\n",
      "--- total time for blast Chr7-6Mb-seg-234: 4.213s.\n",
      "-- processing region: Chr7-6Mb-seg-235 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-235.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-235\n",
      "--- total time for blast Chr7-6Mb-seg-235: 3.139s.\n",
      "-- processing region: Chr7-6Mb-seg-236 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-236.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-236\n",
      "--- total time for blast Chr7-6Mb-seg-236: 3.732s.\n",
      "-- processing region: Chr7-6Mb-seg-237 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-237.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-237\n",
      "--- total time for blast Chr7-6Mb-seg-237: 3.191s.\n",
      "-- processing region: Chr7-6Mb-seg-238 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-238.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-238\n",
      "--- total time for blast Chr7-6Mb-seg-238: 3.541s.\n",
      "-- processing region: Chr7-6Mb-seg-239 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-239.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-239\n",
      "--- total time for blast Chr7-6Mb-seg-239: 3.939s.\n",
      "-- processing region: Chr7-6Mb-seg-24 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-24.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-24\n",
      "--- total time for blast Chr7-6Mb-seg-24: 2.944s.\n",
      "-- processing region: Chr7-6Mb-seg-240 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-240.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-240\n",
      "--- total time for blast Chr7-6Mb-seg-240: 4.592s.\n",
      "-- processing region: Chr7-6Mb-seg-241 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-241.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-241\n",
      "--- total time for blast Chr7-6Mb-seg-241: 4.658s.\n",
      "-- processing region: Chr7-6Mb-seg-242 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-242.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-242\n",
      "--- total time for blast Chr7-6Mb-seg-242: 4.978s.\n",
      "-- processing region: Chr7-6Mb-seg-243 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-243.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-243\n",
      "--- total time for blast Chr7-6Mb-seg-243: 6.855s.\n",
      "-- processing region: Chr7-6Mb-seg-244 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-244.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-244\n",
      "--- total time for blast Chr7-6Mb-seg-244: 3.824s.\n",
      "-- processing region: Chr7-6Mb-seg-245 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-245.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-245\n",
      "--- total time for blast Chr7-6Mb-seg-245: 4.277s.\n",
      "-- processing region: Chr7-6Mb-seg-246 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-246.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-246\n",
      "--- total time for blast Chr7-6Mb-seg-246: 5.194s.\n",
      "-- processing region: Chr7-6Mb-seg-247 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-247.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-247\n",
      "--- total time for blast Chr7-6Mb-seg-247: 3.580s.\n",
      "-- processing region: Chr7-6Mb-seg-248 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-248.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-248\n",
      "--- total time for blast Chr7-6Mb-seg-248: 4.467s.\n",
      "-- processing region: Chr7-6Mb-seg-249 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-249.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-249\n",
      "--- total time for blast Chr7-6Mb-seg-249: 3.829s.\n",
      "-- processing region: Chr7-6Mb-seg-25 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-25.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-25\n",
      "--- total time for blast Chr7-6Mb-seg-25: 3.130s.\n",
      "-- processing region: Chr7-6Mb-seg-250 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-250.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-250\n",
      "--- total time for blast Chr7-6Mb-seg-250: 6.058s.\n",
      "-- processing region: Chr7-6Mb-seg-251 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-251.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-251\n",
      "--- total time for blast Chr7-6Mb-seg-251: 3.935s.\n",
      "-- processing region: Chr7-6Mb-seg-252 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-252.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-252\n",
      "--- total time for blast Chr7-6Mb-seg-252: 7.560s.\n",
      "-- processing region: Chr7-6Mb-seg-253 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-253.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-253\n",
      "--- total time for blast Chr7-6Mb-seg-253: 4.948s.\n",
      "-- processing region: Chr7-6Mb-seg-254 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-254.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-254\n",
      "--- total time for blast Chr7-6Mb-seg-254: 3.230s.\n",
      "-- processing region: Chr7-6Mb-seg-255 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-255.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-255\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-255: 4.100s.\n",
      "-- processing region: Chr7-6Mb-seg-256 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-256.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-256\n",
      "--- total time for blast Chr7-6Mb-seg-256: 4.997s.\n",
      "-- processing region: Chr7-6Mb-seg-257 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-257.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-257\n",
      "--- total time for blast Chr7-6Mb-seg-257: 8.895s.\n",
      "-- processing region: Chr7-6Mb-seg-258 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-258.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-258\n",
      "--- total time for blast Chr7-6Mb-seg-258: 4.800s.\n",
      "-- processing region: Chr7-6Mb-seg-259 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-259.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-259\n",
      "--- total time for blast Chr7-6Mb-seg-259: 4.388s.\n",
      "-- processing region: Chr7-6Mb-seg-260 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-260.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-260\n",
      "--- total time for blast Chr7-6Mb-seg-260: 5.215s.\n",
      "-- processing region: Chr7-6Mb-seg-261 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-261.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-261\n",
      "--- total time for blast Chr7-6Mb-seg-261: 3.699s.\n",
      "-- processing region: Chr7-6Mb-seg-262 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-262.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-262\n",
      "--- total time for blast Chr7-6Mb-seg-262: 4.170s.\n",
      "-- processing region: Chr7-6Mb-seg-263 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-263.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-263\n",
      "--- total time for blast Chr7-6Mb-seg-263: 4.481s.\n",
      "-- processing region: Chr7-6Mb-seg-264 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-264.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-264\n",
      "--- total time for blast Chr7-6Mb-seg-264: 3.672s.\n",
      "-- processing region: Chr7-6Mb-seg-265 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-265.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-265\n",
      "--- total time for blast Chr7-6Mb-seg-265: 4.952s.\n",
      "-- processing region: Chr7-6Mb-seg-266 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-266.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-266\n",
      "--- total time for blast Chr7-6Mb-seg-266: 3.453s.\n",
      "-- processing region: Chr7-6Mb-seg-267 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-267.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-267\n",
      "--- total time for blast Chr7-6Mb-seg-267: 4.824s.\n",
      "-- processing region: Chr7-6Mb-seg-268 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-268.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-268\n",
      "--- total time for blast Chr7-6Mb-seg-268: 3.473s.\n",
      "-- processing region: Chr7-6Mb-seg-269 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-269.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-269\n",
      "--- total time for blast Chr7-6Mb-seg-269: 3.129s.\n",
      "-- processing region: Chr7-6Mb-seg-27 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-27.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-27\n",
      "--- total time for blast Chr7-6Mb-seg-27: 3.583s.\n",
      "-- processing region: Chr7-6Mb-seg-270 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-270.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-270\n",
      "--- total time for blast Chr7-6Mb-seg-270: 3.575s.\n",
      "-- processing region: Chr7-6Mb-seg-271 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-271.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-271\n",
      "--- total time for blast Chr7-6Mb-seg-271: 4.448s.\n",
      "-- processing region: Chr7-6Mb-seg-272 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-272.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-272\n",
      "--- total time for blast Chr7-6Mb-seg-272: 3.598s.\n",
      "-- processing region: Chr7-6Mb-seg-273 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-273.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-273\n",
      "--- total time for blast Chr7-6Mb-seg-273: 11.819s.\n",
      "-- processing region: Chr7-6Mb-seg-274 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-274.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-274\n",
      "--- total time for blast Chr7-6Mb-seg-274: 4.043s.\n",
      "-- processing region: Chr7-6Mb-seg-275 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-275.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-275\n",
      "--- total time for blast Chr7-6Mb-seg-275: 4.633s.\n",
      "-- processing region: Chr7-6Mb-seg-276 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-276.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-276\n",
      "--- total time for blast Chr7-6Mb-seg-276: 4.071s.\n",
      "-- processing region: Chr7-6Mb-seg-277 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-277.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-277\n",
      "--- total time for blast Chr7-6Mb-seg-277: 3.690s.\n",
      "-- processing region: Chr7-6Mb-seg-278 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-278.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-278\n",
      "--- total time for blast Chr7-6Mb-seg-278: 4.554s.\n",
      "-- processing region: Chr7-6Mb-seg-279 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-279.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-279\n",
      "--- total time for blast Chr7-6Mb-seg-279: 4.095s.\n",
      "-- processing region: Chr7-6Mb-seg-28 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-28.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-28\n",
      "--- total time for blast Chr7-6Mb-seg-28: 4.541s.\n",
      "-- processing region: Chr7-6Mb-seg-280 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-280.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-280\n",
      "--- total time for blast Chr7-6Mb-seg-280: 4.926s.\n",
      "-- processing region: Chr7-6Mb-seg-281 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-281.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-281\n",
      "--- total time for blast Chr7-6Mb-seg-281: 3.538s.\n",
      "-- processing region: Chr7-6Mb-seg-282 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-282.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-282\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-282: 3.322s.\n",
      "-- processing region: Chr7-6Mb-seg-283 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-283.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-283\n",
      "--- total time for blast Chr7-6Mb-seg-283: 3.433s.\n",
      "-- processing region: Chr7-6Mb-seg-284 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-284.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-284\n",
      "--- total time for blast Chr7-6Mb-seg-284: 4.095s.\n",
      "-- processing region: Chr7-6Mb-seg-285 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-285.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-285\n",
      "--- total time for blast Chr7-6Mb-seg-285: 3.270s.\n",
      "-- processing region: Chr7-6Mb-seg-286 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-286.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-286\n",
      "--- total time for blast Chr7-6Mb-seg-286: 3.996s.\n",
      "-- processing region: Chr7-6Mb-seg-287 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-287.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-287\n",
      "--- total time for blast Chr7-6Mb-seg-287: 4.217s.\n",
      "-- processing region: Chr7-6Mb-seg-288 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-288.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-288\n",
      "--- total time for blast Chr7-6Mb-seg-288: 5.765s.\n",
      "-- processing region: Chr7-6Mb-seg-289 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-289.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-289\n",
      "--- total time for blast Chr7-6Mb-seg-289: 3.986s.\n",
      "-- processing region: Chr7-6Mb-seg-29 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-29.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-29\n",
      "--- total time for blast Chr7-6Mb-seg-29: 2.539s.\n",
      "-- processing region: Chr7-6Mb-seg-290 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-290.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-290\n",
      "--- total time for blast Chr7-6Mb-seg-290: 4.818s.\n",
      "-- processing region: Chr7-6Mb-seg-291 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-291.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-291\n",
      "--- total time for blast Chr7-6Mb-seg-291: 5.291s.\n",
      "-- processing region: Chr7-6Mb-seg-292 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-292.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-292\n",
      "--- total time for blast Chr7-6Mb-seg-292: 6.555s.\n",
      "-- processing region: Chr7-6Mb-seg-293 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-293.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-293\n",
      "--- total time for blast Chr7-6Mb-seg-293: 4.110s.\n",
      "-- processing region: Chr7-6Mb-seg-294 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-294.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-294\n",
      "--- total time for blast Chr7-6Mb-seg-294: 4.335s.\n",
      "-- processing region: Chr7-6Mb-seg-295 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-295.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-295\n",
      "--- total time for blast Chr7-6Mb-seg-295: 3.269s.\n",
      "-- processing region: Chr7-6Mb-seg-296 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-296.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-296\n",
      "--- total time for blast Chr7-6Mb-seg-296: 4.220s.\n",
      "-- processing region: Chr7-6Mb-seg-297 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-297.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-297\n",
      "--- total time for blast Chr7-6Mb-seg-297: 2.580s.\n",
      "-- processing region: Chr7-6Mb-seg-298 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-298.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-298\n",
      "--- total time for blast Chr7-6Mb-seg-298: 3.543s.\n",
      "-- processing region: Chr7-6Mb-seg-299 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-299.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-299\n",
      "--- total time for blast Chr7-6Mb-seg-299: 3.814s.\n",
      "-- processing region: Chr7-6Mb-seg-3 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-3.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-3\n",
      "--- total time for blast Chr7-6Mb-seg-3: 5.972s.\n",
      "-- processing region: Chr7-6Mb-seg-300 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-300.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-300\n",
      "--- total time for blast Chr7-6Mb-seg-300: 3.832s.\n",
      "-- processing region: Chr7-6Mb-seg-301 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-301.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-301\n",
      "--- total time for blast Chr7-6Mb-seg-301: 3.770s.\n",
      "-- processing region: Chr7-6Mb-seg-302 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-302.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-302\n",
      "--- total time for blast Chr7-6Mb-seg-302: 8.983s.\n",
      "-- processing region: Chr7-6Mb-seg-303 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-303.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-303\n",
      "--- total time for blast Chr7-6Mb-seg-303: 2.792s.\n",
      "-- processing region: Chr7-6Mb-seg-304 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-304.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-304\n",
      "--- total time for blast Chr7-6Mb-seg-304: 3.782s.\n",
      "-- processing region: Chr7-6Mb-seg-305 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-305.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-305\n",
      "--- total time for blast Chr7-6Mb-seg-305: 3.756s.\n",
      "-- processing region: Chr7-6Mb-seg-306 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-306.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-306\n",
      "--- total time for blast Chr7-6Mb-seg-306: 3.261s.\n",
      "-- processing region: Chr7-6Mb-seg-307 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-307.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-307\n",
      "--- total time for blast Chr7-6Mb-seg-307: 6.684s.\n",
      "-- processing region: Chr7-6Mb-seg-308 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-308.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-308\n",
      "--- total time for blast Chr7-6Mb-seg-308: 4.206s.\n",
      "-- processing region: Chr7-6Mb-seg-309 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-309.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-309\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-309: 3.607s.\n",
      "-- processing region: Chr7-6Mb-seg-310 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-310.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-310\n",
      "--- total time for blast Chr7-6Mb-seg-310: 3.975s.\n",
      "-- processing region: Chr7-6Mb-seg-311 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-311.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-311\n",
      "--- total time for blast Chr7-6Mb-seg-311: 4.190s.\n",
      "-- processing region: Chr7-6Mb-seg-312 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-312.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-312\n",
      "--- total time for blast Chr7-6Mb-seg-312: 4.746s.\n",
      "-- processing region: Chr7-6Mb-seg-313 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-313.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-313\n",
      "--- total time for blast Chr7-6Mb-seg-313: 4.534s.\n",
      "-- processing region: Chr7-6Mb-seg-314 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-314.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-314\n",
      "--- total time for blast Chr7-6Mb-seg-314: 4.236s.\n",
      "-- processing region: Chr7-6Mb-seg-315 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-315.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-315\n",
      "--- total time for blast Chr7-6Mb-seg-315: 3.892s.\n",
      "-- processing region: Chr7-6Mb-seg-316 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-316.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-316\n",
      "--- total time for blast Chr7-6Mb-seg-316: 4.291s.\n",
      "-- processing region: Chr7-6Mb-seg-317 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-317.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-317\n",
      "--- total time for blast Chr7-6Mb-seg-317: 3.836s.\n",
      "-- processing region: Chr7-6Mb-seg-318 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-318.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-318\n",
      "--- total time for blast Chr7-6Mb-seg-318: 11.375s.\n",
      "-- processing region: Chr7-6Mb-seg-319 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-319.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-319\n",
      "--- total time for blast Chr7-6Mb-seg-319: 6.275s.\n",
      "-- processing region: Chr7-6Mb-seg-320 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-320.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-320\n",
      "--- total time for blast Chr7-6Mb-seg-320: 4.909s.\n",
      "-- processing region: Chr7-6Mb-seg-321 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-321.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-321\n",
      "--- total time for blast Chr7-6Mb-seg-321: 4.358s.\n",
      "-- processing region: Chr7-6Mb-seg-322 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-322.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-322\n",
      "--- total time for blast Chr7-6Mb-seg-322: 3.621s.\n",
      "-- processing region: Chr7-6Mb-seg-323 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-323.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-323\n",
      "--- total time for blast Chr7-6Mb-seg-323: 3.573s.\n",
      "-- processing region: Chr7-6Mb-seg-324 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-324.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-324\n",
      "--- total time for blast Chr7-6Mb-seg-324: 4.048s.\n",
      "-- processing region: Chr7-6Mb-seg-325 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-325.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-325\n",
      "--- total time for blast Chr7-6Mb-seg-325: 8.947s.\n",
      "-- processing region: Chr7-6Mb-seg-326 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-326.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-326\n",
      "--- total time for blast Chr7-6Mb-seg-326: 4.695s.\n",
      "-- processing region: Chr7-6Mb-seg-327 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-327.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-327\n",
      "--- total time for blast Chr7-6Mb-seg-327: 3.727s.\n",
      "-- processing region: Chr7-6Mb-seg-328 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-328.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-328\n",
      "--- total time for blast Chr7-6Mb-seg-328: 3.563s.\n",
      "-- processing region: Chr7-6Mb-seg-329 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-329.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-329\n",
      "--- total time for blast Chr7-6Mb-seg-329: 3.270s.\n",
      "-- processing region: Chr7-6Mb-seg-330 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-330.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-330\n",
      "--- total time for blast Chr7-6Mb-seg-330: 4.926s.\n",
      "-- processing region: Chr7-6Mb-seg-331 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-331.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-331\n",
      "--- total time for blast Chr7-6Mb-seg-331: 4.645s.\n",
      "-- processing region: Chr7-6Mb-seg-332 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-332.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-332\n",
      "--- total time for blast Chr7-6Mb-seg-332: 3.473s.\n",
      "-- processing region: Chr7-6Mb-seg-333 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-333.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-333\n",
      "--- total time for blast Chr7-6Mb-seg-333: 3.572s.\n",
      "-- processing region: Chr7-6Mb-seg-334 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-334.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-334\n",
      "--- total time for blast Chr7-6Mb-seg-334: 4.820s.\n",
      "-- processing region: Chr7-6Mb-seg-335 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-335.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-335\n",
      "--- total time for blast Chr7-6Mb-seg-335: 4.964s.\n",
      "-- processing region: Chr7-6Mb-seg-336 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-336.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-336\n",
      "--- total time for blast Chr7-6Mb-seg-336: 5.257s.\n",
      "-- processing region: Chr7-6Mb-seg-337 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-337.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-337\n",
      "--- total time for blast Chr7-6Mb-seg-337: 4.411s.\n",
      "-- processing region: Chr7-6Mb-seg-338 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-338.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-338\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-338: 4.550s.\n",
      "-- processing region: Chr7-6Mb-seg-339 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-339.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-339\n",
      "--- total time for blast Chr7-6Mb-seg-339: 4.935s.\n",
      "-- processing region: Chr7-6Mb-seg-340 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-340.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-340\n",
      "--- total time for blast Chr7-6Mb-seg-340: 4.103s.\n",
      "-- processing region: Chr7-6Mb-seg-341 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-341.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-341\n",
      "--- total time for blast Chr7-6Mb-seg-341: 4.434s.\n",
      "-- processing region: Chr7-6Mb-seg-342 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-342.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-342\n",
      "--- total time for blast Chr7-6Mb-seg-342: 3.792s.\n",
      "-- processing region: Chr7-6Mb-seg-343 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-343.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-343\n",
      "--- total time for blast Chr7-6Mb-seg-343: 3.058s.\n",
      "-- processing region: Chr7-6Mb-seg-344 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-344.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-344\n",
      "--- total time for blast Chr7-6Mb-seg-344: 3.904s.\n",
      "-- processing region: Chr7-6Mb-seg-345 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-345.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-345\n",
      "--- total time for blast Chr7-6Mb-seg-345: 3.697s.\n",
      "-- processing region: Chr7-6Mb-seg-346 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-346.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-346\n",
      "--- total time for blast Chr7-6Mb-seg-346: 3.532s.\n",
      "-- processing region: Chr7-6Mb-seg-347 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-347.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-347\n",
      "--- total time for blast Chr7-6Mb-seg-347: 8.352s.\n",
      "-- processing region: Chr7-6Mb-seg-348 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-348.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-348\n",
      "--- total time for blast Chr7-6Mb-seg-348: 5.181s.\n",
      "-- processing region: Chr7-6Mb-seg-349 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-349.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-349\n",
      "--- total time for blast Chr7-6Mb-seg-349: 7.858s.\n",
      "-- processing region: Chr7-6Mb-seg-350 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-350.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-350\n",
      "--- total time for blast Chr7-6Mb-seg-350: 10.726s.\n",
      "-- processing region: Chr7-6Mb-seg-351 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-351.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-351\n",
      "--- total time for blast Chr7-6Mb-seg-351: 3.409s.\n",
      "-- processing region: Chr7-6Mb-seg-352 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-352.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-352\n",
      "--- total time for blast Chr7-6Mb-seg-352: 6.030s.\n",
      "-- processing region: Chr7-6Mb-seg-353 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-353.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-353\n",
      "--- total time for blast Chr7-6Mb-seg-353: 4.115s.\n",
      "-- processing region: Chr7-6Mb-seg-354 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-354.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-354\n",
      "--- total time for blast Chr7-6Mb-seg-354: 4.271s.\n",
      "-- processing region: Chr7-6Mb-seg-355 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-355.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-355\n",
      "--- total time for blast Chr7-6Mb-seg-355: 3.710s.\n",
      "-- processing region: Chr7-6Mb-seg-356 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-356.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-356\n",
      "--- total time for blast Chr7-6Mb-seg-356: 8.044s.\n",
      "-- processing region: Chr7-6Mb-seg-357 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-357.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-357\n",
      "--- total time for blast Chr7-6Mb-seg-357: 4.335s.\n",
      "-- processing region: Chr7-6Mb-seg-358 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-358.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-358\n",
      "--- total time for blast Chr7-6Mb-seg-358: 4.204s.\n",
      "-- processing region: Chr7-6Mb-seg-359 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-359.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-359\n",
      "--- total time for blast Chr7-6Mb-seg-359: 3.913s.\n",
      "-- processing region: Chr7-6Mb-seg-36 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-36.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-36\n",
      "--- total time for blast Chr7-6Mb-seg-36: 4.234s.\n",
      "-- processing region: Chr7-6Mb-seg-360 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-360.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-360\n",
      "--- total time for blast Chr7-6Mb-seg-360: 3.584s.\n",
      "-- processing region: Chr7-6Mb-seg-361 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-361.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-361\n",
      "--- total time for blast Chr7-6Mb-seg-361: 3.798s.\n",
      "-- processing region: Chr7-6Mb-seg-362 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-362.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-362\n",
      "--- total time for blast Chr7-6Mb-seg-362: 3.746s.\n",
      "-- processing region: Chr7-6Mb-seg-363 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-363.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-363\n",
      "--- total time for blast Chr7-6Mb-seg-363: 3.291s.\n",
      "-- processing region: Chr7-6Mb-seg-364 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-364.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-364\n",
      "--- total time for blast Chr7-6Mb-seg-364: 3.685s.\n",
      "-- processing region: Chr7-6Mb-seg-365 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-365.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-365\n",
      "--- total time for blast Chr7-6Mb-seg-365: 3.940s.\n",
      "-- processing region: Chr7-6Mb-seg-366 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-366.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-366\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-366: 4.507s.\n",
      "-- processing region: Chr7-6Mb-seg-367 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-367.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-367\n",
      "--- total time for blast Chr7-6Mb-seg-367: 3.088s.\n",
      "-- processing region: Chr7-6Mb-seg-368 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-368.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-368\n",
      "--- total time for blast Chr7-6Mb-seg-368: 3.217s.\n",
      "-- processing region: Chr7-6Mb-seg-369 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-369.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-369\n",
      "--- total time for blast Chr7-6Mb-seg-369: 4.224s.\n",
      "-- processing region: Chr7-6Mb-seg-370 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-370.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-370\n",
      "--- total time for blast Chr7-6Mb-seg-370: 3.099s.\n",
      "-- processing region: Chr7-6Mb-seg-371 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-371.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-371\n",
      "--- total time for blast Chr7-6Mb-seg-371: 4.900s.\n",
      "-- processing region: Chr7-6Mb-seg-372 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-372.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-372\n",
      "--- total time for blast Chr7-6Mb-seg-372: 3.410s.\n",
      "-- processing region: Chr7-6Mb-seg-373 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-373.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-373\n",
      "--- total time for blast Chr7-6Mb-seg-373: 3.861s.\n",
      "-- processing region: Chr7-6Mb-seg-374 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-374.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-374\n",
      "--- total time for blast Chr7-6Mb-seg-374: 4.232s.\n",
      "-- processing region: Chr7-6Mb-seg-375 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-375.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-375\n",
      "--- total time for blast Chr7-6Mb-seg-375: 3.971s.\n",
      "-- processing region: Chr7-6Mb-seg-376 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-376.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-376\n",
      "--- total time for blast Chr7-6Mb-seg-376: 3.894s.\n",
      "-- processing region: Chr7-6Mb-seg-377 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-377.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-377\n",
      "--- total time for blast Chr7-6Mb-seg-377: 4.700s.\n",
      "-- processing region: Chr7-6Mb-seg-378 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-378.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-378\n",
      "--- total time for blast Chr7-6Mb-seg-378: 3.563s.\n",
      "-- processing region: Chr7-6Mb-seg-379 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-379.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-379\n",
      "--- total time for blast Chr7-6Mb-seg-379: 5.060s.\n",
      "-- processing region: Chr7-6Mb-seg-38 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-38.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-38\n",
      "--- total time for blast Chr7-6Mb-seg-38: 3.160s.\n",
      "-- processing region: Chr7-6Mb-seg-380 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-380.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-380\n",
      "--- total time for blast Chr7-6Mb-seg-380: 5.196s.\n",
      "-- processing region: Chr7-6Mb-seg-381 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-381.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-381\n",
      "--- total time for blast Chr7-6Mb-seg-381: 6.211s.\n",
      "-- processing region: Chr7-6Mb-seg-382 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-382.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-382\n",
      "--- total time for blast Chr7-6Mb-seg-382: 3.775s.\n",
      "-- processing region: Chr7-6Mb-seg-383 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-383.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-383\n",
      "--- total time for blast Chr7-6Mb-seg-383: 4.146s.\n",
      "-- processing region: Chr7-6Mb-seg-384 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-384.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-384\n",
      "--- total time for blast Chr7-6Mb-seg-384: 4.213s.\n",
      "-- processing region: Chr7-6Mb-seg-385 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-385.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-385\n",
      "--- total time for blast Chr7-6Mb-seg-385: 4.208s.\n",
      "-- processing region: Chr7-6Mb-seg-386 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-386.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-386\n",
      "--- total time for blast Chr7-6Mb-seg-386: 4.980s.\n",
      "-- processing region: Chr7-6Mb-seg-387 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-387.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-387\n",
      "--- total time for blast Chr7-6Mb-seg-387: 8.882s.\n",
      "-- processing region: Chr7-6Mb-seg-388 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-388.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-388\n",
      "--- total time for blast Chr7-6Mb-seg-388: 4.129s.\n",
      "-- processing region: Chr7-6Mb-seg-389 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-389.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-389\n",
      "--- total time for blast Chr7-6Mb-seg-389: 3.157s.\n",
      "-- processing region: Chr7-6Mb-seg-39 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-39.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-39\n",
      "--- total time for blast Chr7-6Mb-seg-39: 2.270s.\n",
      "-- processing region: Chr7-6Mb-seg-390 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-390.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-390\n",
      "--- total time for blast Chr7-6Mb-seg-390: 6.492s.\n",
      "-- processing region: Chr7-6Mb-seg-391 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-391.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-391\n",
      "--- total time for blast Chr7-6Mb-seg-391: 2.944s.\n",
      "-- processing region: Chr7-6Mb-seg-392 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-392.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-392\n",
      "--- total time for blast Chr7-6Mb-seg-392: 4.966s.\n",
      "-- processing region: Chr7-6Mb-seg-393 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-393.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-393\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-393: 4.877s.\n",
      "-- processing region: Chr7-6Mb-seg-394 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-394.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-394\n",
      "--- total time for blast Chr7-6Mb-seg-394: 7.334s.\n",
      "-- processing region: Chr7-6Mb-seg-395 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-395.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-395\n",
      "--- total time for blast Chr7-6Mb-seg-395: 4.106s.\n",
      "-- processing region: Chr7-6Mb-seg-396 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-396.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-396\n",
      "--- total time for blast Chr7-6Mb-seg-396: 4.142s.\n",
      "-- processing region: Chr7-6Mb-seg-397 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-397.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-397\n",
      "--- total time for blast Chr7-6Mb-seg-397: 3.838s.\n",
      "-- processing region: Chr7-6Mb-seg-398 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-398.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-398\n",
      "--- total time for blast Chr7-6Mb-seg-398: 4.774s.\n",
      "-- processing region: Chr7-6Mb-seg-399 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-399.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-399\n",
      "--- total time for blast Chr7-6Mb-seg-399: 4.581s.\n",
      "-- processing region: Chr7-6Mb-seg-4 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-4.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-4\n",
      "--- total time for blast Chr7-6Mb-seg-4: 2.822s.\n",
      "-- processing region: Chr7-6Mb-seg-40 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-40.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-40\n",
      "--- total time for blast Chr7-6Mb-seg-40: 3.907s.\n",
      "-- processing region: Chr7-6Mb-seg-400 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-400.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-400\n",
      "--- total time for blast Chr7-6Mb-seg-400: 4.303s.\n",
      "-- processing region: Chr7-6Mb-seg-401 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-401.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-401\n",
      "--- total time for blast Chr7-6Mb-seg-401: 7.652s.\n",
      "-- processing region: Chr7-6Mb-seg-402 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-402.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-402\n",
      "--- total time for blast Chr7-6Mb-seg-402: 4.301s.\n",
      "-- processing region: Chr7-6Mb-seg-403 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-403.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-403\n",
      "--- total time for blast Chr7-6Mb-seg-403: 3.610s.\n",
      "-- processing region: Chr7-6Mb-seg-404 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-404.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-404\n",
      "--- total time for blast Chr7-6Mb-seg-404: 3.560s.\n",
      "-- processing region: Chr7-6Mb-seg-405 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-405.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-405\n",
      "--- total time for blast Chr7-6Mb-seg-405: 4.189s.\n",
      "-- processing region: Chr7-6Mb-seg-406 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-406.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-406\n",
      "--- total time for blast Chr7-6Mb-seg-406: 4.471s.\n",
      "-- processing region: Chr7-6Mb-seg-407 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-407.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-407\n",
      "--- total time for blast Chr7-6Mb-seg-407: 3.652s.\n",
      "-- processing region: Chr7-6Mb-seg-408 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-408.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-408\n",
      "--- total time for blast Chr7-6Mb-seg-408: 3.713s.\n",
      "-- processing region: Chr7-6Mb-seg-409 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-409.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-409\n",
      "--- total time for blast Chr7-6Mb-seg-409: 3.527s.\n",
      "-- processing region: Chr7-6Mb-seg-41 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-41.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-41\n",
      "--- total time for blast Chr7-6Mb-seg-41: 3.823s.\n",
      "-- processing region: Chr7-6Mb-seg-410 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-410.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-410\n",
      "--- total time for blast Chr7-6Mb-seg-410: 3.739s.\n",
      "-- processing region: Chr7-6Mb-seg-411 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-411.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-411\n",
      "--- total time for blast Chr7-6Mb-seg-411: 3.030s.\n",
      "-- processing region: Chr7-6Mb-seg-412 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-412.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-412\n",
      "--- total time for blast Chr7-6Mb-seg-412: 3.644s.\n",
      "-- processing region: Chr7-6Mb-seg-413 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-413.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-413\n",
      "--- total time for blast Chr7-6Mb-seg-413: 3.834s.\n",
      "-- processing region: Chr7-6Mb-seg-414 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-414.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-414\n",
      "--- total time for blast Chr7-6Mb-seg-414: 6.612s.\n",
      "-- processing region: Chr7-6Mb-seg-415 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-415.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-415\n",
      "--- total time for blast Chr7-6Mb-seg-415: 4.279s.\n",
      "-- processing region: Chr7-6Mb-seg-416 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-416.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-416\n",
      "--- total time for blast Chr7-6Mb-seg-416: 3.812s.\n",
      "-- processing region: Chr7-6Mb-seg-417 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-417.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-417\n",
      "--- total time for blast Chr7-6Mb-seg-417: 4.077s.\n",
      "-- processing region: Chr7-6Mb-seg-418 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-418.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-418\n",
      "--- total time for blast Chr7-6Mb-seg-418: 6.433s.\n",
      "-- processing region: Chr7-6Mb-seg-419 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-419.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-419\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-419: 4.343s.\n",
      "-- processing region: Chr7-6Mb-seg-42 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-42.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-42\n",
      "--- total time for blast Chr7-6Mb-seg-42: 3.680s.\n",
      "-- processing region: Chr7-6Mb-seg-420 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-420.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-420\n",
      "--- total time for blast Chr7-6Mb-seg-420: 3.423s.\n",
      "-- processing region: Chr7-6Mb-seg-421 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-421.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-421\n",
      "--- total time for blast Chr7-6Mb-seg-421: 3.448s.\n",
      "-- processing region: Chr7-6Mb-seg-422 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-422.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-422\n",
      "--- total time for blast Chr7-6Mb-seg-422: 3.518s.\n",
      "-- processing region: Chr7-6Mb-seg-423 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-423.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-423\n",
      "--- total time for blast Chr7-6Mb-seg-423: 3.350s.\n",
      "-- processing region: Chr7-6Mb-seg-424 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-424.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-424\n",
      "--- total time for blast Chr7-6Mb-seg-424: 3.759s.\n",
      "-- processing region: Chr7-6Mb-seg-425 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-425.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-425\n",
      "--- total time for blast Chr7-6Mb-seg-425: 4.053s.\n",
      "-- processing region: Chr7-6Mb-seg-426 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-426.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-426\n",
      "--- total time for blast Chr7-6Mb-seg-426: 4.627s.\n",
      "-- processing region: Chr7-6Mb-seg-427 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-427.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-427\n",
      "--- total time for blast Chr7-6Mb-seg-427: 3.692s.\n",
      "-- processing region: Chr7-6Mb-seg-428 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-428.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-428\n",
      "--- total time for blast Chr7-6Mb-seg-428: 3.466s.\n",
      "-- processing region: Chr7-6Mb-seg-429 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-429.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-429\n",
      "--- total time for blast Chr7-6Mb-seg-429: 4.670s.\n",
      "-- processing region: Chr7-6Mb-seg-43 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-43.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-43\n",
      "--- total time for blast Chr7-6Mb-seg-43: 3.303s.\n",
      "-- processing region: Chr7-6Mb-seg-430 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-430.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-430\n",
      "--- total time for blast Chr7-6Mb-seg-430: 3.993s.\n",
      "-- processing region: Chr7-6Mb-seg-431 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-431.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-431\n",
      "--- total time for blast Chr7-6Mb-seg-431: 4.396s.\n",
      "-- processing region: Chr7-6Mb-seg-432 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-432.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-432\n",
      "--- total time for blast Chr7-6Mb-seg-432: 3.805s.\n",
      "-- processing region: Chr7-6Mb-seg-433 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-433.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-433\n",
      "--- total time for blast Chr7-6Mb-seg-433: 4.696s.\n",
      "-- processing region: Chr7-6Mb-seg-434 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-434.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-434\n",
      "--- total time for blast Chr7-6Mb-seg-434: 3.698s.\n",
      "-- processing region: Chr7-6Mb-seg-435 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-435.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-435\n",
      "--- total time for blast Chr7-6Mb-seg-435: 4.257s.\n",
      "-- processing region: Chr7-6Mb-seg-436 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-436.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-436\n",
      "--- total time for blast Chr7-6Mb-seg-436: 3.909s.\n",
      "-- processing region: Chr7-6Mb-seg-437 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-437.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-437\n",
      "--- total time for blast Chr7-6Mb-seg-437: 4.442s.\n",
      "-- processing region: Chr7-6Mb-seg-438 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-438.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-438\n",
      "--- total time for blast Chr7-6Mb-seg-438: 5.587s.\n",
      "-- processing region: Chr7-6Mb-seg-439 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-439.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-439\n",
      "--- total time for blast Chr7-6Mb-seg-439: 4.020s.\n",
      "-- processing region: Chr7-6Mb-seg-44 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-44.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-44\n",
      "--- total time for blast Chr7-6Mb-seg-44: 3.185s.\n",
      "-- processing region: Chr7-6Mb-seg-440 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-440.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-440\n",
      "--- total time for blast Chr7-6Mb-seg-440: 3.409s.\n",
      "-- processing region: Chr7-6Mb-seg-441 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-441.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-441\n",
      "--- total time for blast Chr7-6Mb-seg-441: 4.126s.\n",
      "-- processing region: Chr7-6Mb-seg-442 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-442.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-442\n",
      "--- total time for blast Chr7-6Mb-seg-442: 3.419s.\n",
      "-- processing region: Chr7-6Mb-seg-443 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-443.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-443\n",
      "--- total time for blast Chr7-6Mb-seg-443: 4.733s.\n",
      "-- processing region: Chr7-6Mb-seg-444 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-444.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-444\n",
      "--- total time for blast Chr7-6Mb-seg-444: 4.298s.\n",
      "-- processing region: Chr7-6Mb-seg-445 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-445.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-445\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-445: 3.796s.\n",
      "-- processing region: Chr7-6Mb-seg-446 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-446.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-446\n",
      "--- total time for blast Chr7-6Mb-seg-446: 3.674s.\n",
      "-- processing region: Chr7-6Mb-seg-447 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-447.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-447\n",
      "--- total time for blast Chr7-6Mb-seg-447: 4.207s.\n",
      "-- processing region: Chr7-6Mb-seg-448 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-448.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-448\n",
      "--- total time for blast Chr7-6Mb-seg-448: 4.185s.\n",
      "-- processing region: Chr7-6Mb-seg-449 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-449.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-449\n",
      "--- total time for blast Chr7-6Mb-seg-449: 4.958s.\n",
      "-- processing region: Chr7-6Mb-seg-45 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-45.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-45\n",
      "--- total time for blast Chr7-6Mb-seg-45: 3.755s.\n",
      "-- processing region: Chr7-6Mb-seg-450 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-450.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-450\n",
      "--- total time for blast Chr7-6Mb-seg-450: 3.971s.\n",
      "-- processing region: Chr7-6Mb-seg-451 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-451.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-451\n",
      "--- total time for blast Chr7-6Mb-seg-451: 4.315s.\n",
      "-- processing region: Chr7-6Mb-seg-452 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-452.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-452\n",
      "--- total time for blast Chr7-6Mb-seg-452: 4.623s.\n",
      "-- processing region: Chr7-6Mb-seg-453 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-453.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-453\n",
      "--- total time for blast Chr7-6Mb-seg-453: 4.048s.\n",
      "-- processing region: Chr7-6Mb-seg-454 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-454.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-454\n",
      "--- total time for blast Chr7-6Mb-seg-454: 3.710s.\n",
      "-- processing region: Chr7-6Mb-seg-455 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-455.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-455\n",
      "--- total time for blast Chr7-6Mb-seg-455: 3.961s.\n",
      "-- processing region: Chr7-6Mb-seg-456 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-456.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-456\n",
      "--- total time for blast Chr7-6Mb-seg-456: 6.032s.\n",
      "-- processing region: Chr7-6Mb-seg-457 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-457.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-457\n",
      "--- total time for blast Chr7-6Mb-seg-457: 4.341s.\n",
      "-- processing region: Chr7-6Mb-seg-458 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-458.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-458\n",
      "--- total time for blast Chr7-6Mb-seg-458: 3.692s.\n",
      "-- processing region: Chr7-6Mb-seg-459 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-459.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-459\n",
      "--- total time for blast Chr7-6Mb-seg-459: 4.056s.\n",
      "-- processing region: Chr7-6Mb-seg-46 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-46.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-46\n",
      "--- total time for blast Chr7-6Mb-seg-46: 3.492s.\n",
      "-- processing region: Chr7-6Mb-seg-460 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-460.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-460\n",
      "--- total time for blast Chr7-6Mb-seg-460: 4.224s.\n",
      "-- processing region: Chr7-6Mb-seg-461 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-461.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-461\n",
      "--- total time for blast Chr7-6Mb-seg-461: 4.696s.\n",
      "-- processing region: Chr7-6Mb-seg-462 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-462.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-462\n",
      "--- total time for blast Chr7-6Mb-seg-462: 3.766s.\n",
      "-- processing region: Chr7-6Mb-seg-463 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-463.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-463\n",
      "--- total time for blast Chr7-6Mb-seg-463: 4.835s.\n",
      "-- processing region: Chr7-6Mb-seg-464 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-464.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-464\n",
      "--- total time for blast Chr7-6Mb-seg-464: 4.141s.\n",
      "-- processing region: Chr7-6Mb-seg-465 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-465.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-465\n",
      "--- total time for blast Chr7-6Mb-seg-465: 4.330s.\n",
      "-- processing region: Chr7-6Mb-seg-466 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-466.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-466\n",
      "--- total time for blast Chr7-6Mb-seg-466: 4.379s.\n",
      "-- processing region: Chr7-6Mb-seg-467 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-467.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-467\n",
      "--- total time for blast Chr7-6Mb-seg-467: 5.954s.\n",
      "-- processing region: Chr7-6Mb-seg-468 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-468.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-468\n",
      "--- total time for blast Chr7-6Mb-seg-468: 3.954s.\n",
      "-- processing region: Chr7-6Mb-seg-469 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-469.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-469\n",
      "--- total time for blast Chr7-6Mb-seg-469: 4.132s.\n",
      "-- processing region: Chr7-6Mb-seg-47 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-47.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-47\n",
      "--- total time for blast Chr7-6Mb-seg-47: 5.179s.\n",
      "-- processing region: Chr7-6Mb-seg-470 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-470.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-470\n",
      "--- total time for blast Chr7-6Mb-seg-470: 3.934s.\n",
      "-- processing region: Chr7-6Mb-seg-471 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-471.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-471\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-471: 4.959s.\n",
      "-- processing region: Chr7-6Mb-seg-472 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-472.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-472\n",
      "--- total time for blast Chr7-6Mb-seg-472: 4.798s.\n",
      "-- processing region: Chr7-6Mb-seg-473 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-473.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-473\n",
      "--- total time for blast Chr7-6Mb-seg-473: 3.586s.\n",
      "-- processing region: Chr7-6Mb-seg-474 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-474.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-474\n",
      "--- total time for blast Chr7-6Mb-seg-474: 3.935s.\n",
      "-- processing region: Chr7-6Mb-seg-475 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-475.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-475\n",
      "--- total time for blast Chr7-6Mb-seg-475: 4.287s.\n",
      "-- processing region: Chr7-6Mb-seg-476 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-476.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-476\n",
      "--- total time for blast Chr7-6Mb-seg-476: 3.732s.\n",
      "-- processing region: Chr7-6Mb-seg-477 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-477.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-477\n",
      "--- total time for blast Chr7-6Mb-seg-477: 5.674s.\n",
      "-- processing region: Chr7-6Mb-seg-478 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-478.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-478\n",
      "--- total time for blast Chr7-6Mb-seg-478: 3.833s.\n",
      "-- processing region: Chr7-6Mb-seg-479 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-479.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-479\n",
      "--- total time for blast Chr7-6Mb-seg-479: 3.717s.\n",
      "-- processing region: Chr7-6Mb-seg-48 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-48.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-48\n",
      "--- total time for blast Chr7-6Mb-seg-48: 4.303s.\n",
      "-- processing region: Chr7-6Mb-seg-480 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-480.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-480\n",
      "--- total time for blast Chr7-6Mb-seg-480: 3.895s.\n",
      "-- processing region: Chr7-6Mb-seg-481 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-481.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-481\n",
      "--- total time for blast Chr7-6Mb-seg-481: 4.207s.\n",
      "-- processing region: Chr7-6Mb-seg-482 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-482.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-482\n",
      "--- total time for blast Chr7-6Mb-seg-482: 3.958s.\n",
      "-- processing region: Chr7-6Mb-seg-483 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-483.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-483\n",
      "--- total time for blast Chr7-6Mb-seg-483: 4.061s.\n",
      "-- processing region: Chr7-6Mb-seg-484 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-484.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-484\n",
      "--- total time for blast Chr7-6Mb-seg-484: 4.299s.\n",
      "-- processing region: Chr7-6Mb-seg-485 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-485.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-485\n",
      "--- total time for blast Chr7-6Mb-seg-485: 4.418s.\n",
      "-- processing region: Chr7-6Mb-seg-486 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-486.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-486\n",
      "--- total time for blast Chr7-6Mb-seg-486: 3.681s.\n",
      "-- processing region: Chr7-6Mb-seg-487 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-487.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-487\n",
      "--- total time for blast Chr7-6Mb-seg-487: 4.158s.\n",
      "-- processing region: Chr7-6Mb-seg-488 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-488.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-488\n",
      "--- total time for blast Chr7-6Mb-seg-488: 3.606s.\n",
      "-- processing region: Chr7-6Mb-seg-489 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-489.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-489\n",
      "--- total time for blast Chr7-6Mb-seg-489: 4.006s.\n",
      "-- processing region: Chr7-6Mb-seg-49 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-49.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-49\n",
      "--- total time for blast Chr7-6Mb-seg-49: 5.715s.\n",
      "-- processing region: Chr7-6Mb-seg-490 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-490.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-490\n",
      "--- total time for blast Chr7-6Mb-seg-490: 5.442s.\n",
      "-- processing region: Chr7-6Mb-seg-491 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-491.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-491\n",
      "--- total time for blast Chr7-6Mb-seg-491: 3.279s.\n",
      "-- processing region: Chr7-6Mb-seg-492 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-492.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-492\n",
      "--- total time for blast Chr7-6Mb-seg-492: 4.247s.\n",
      "-- processing region: Chr7-6Mb-seg-493 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-493.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-493\n",
      "--- total time for blast Chr7-6Mb-seg-493: 3.606s.\n",
      "-- processing region: Chr7-6Mb-seg-494 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-494.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-494\n",
      "--- total time for blast Chr7-6Mb-seg-494: 3.826s.\n",
      "-- processing region: Chr7-6Mb-seg-495 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-495.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-495\n",
      "--- total time for blast Chr7-6Mb-seg-495: 4.058s.\n",
      "-- processing region: Chr7-6Mb-seg-496 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-496.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-496\n",
      "--- total time for blast Chr7-6Mb-seg-496: 5.033s.\n",
      "-- processing region: Chr7-6Mb-seg-497 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-497.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-497\n",
      "--- total time for blast Chr7-6Mb-seg-497: 3.933s.\n",
      "-- processing region: Chr7-6Mb-seg-498 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-498.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-498\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-498: 3.361s.\n",
      "-- processing region: Chr7-6Mb-seg-499 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-499.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-499\n",
      "--- total time for blast Chr7-6Mb-seg-499: 3.591s.\n",
      "-- processing region: Chr7-6Mb-seg-50 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-50.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-50\n",
      "--- total time for blast Chr7-6Mb-seg-50: 4.756s.\n",
      "-- processing region: Chr7-6Mb-seg-500 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-500.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-500\n",
      "--- total time for blast Chr7-6Mb-seg-500: 3.752s.\n",
      "-- processing region: Chr7-6Mb-seg-501 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-501.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-501\n",
      "--- total time for blast Chr7-6Mb-seg-501: 3.978s.\n",
      "-- processing region: Chr7-6Mb-seg-502 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-502.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-502\n",
      "--- total time for blast Chr7-6Mb-seg-502: 3.926s.\n",
      "-- processing region: Chr7-6Mb-seg-503 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-503.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-503\n",
      "--- total time for blast Chr7-6Mb-seg-503: 3.554s.\n",
      "-- processing region: Chr7-6Mb-seg-504 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-504.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-504\n",
      "--- total time for blast Chr7-6Mb-seg-504: 3.621s.\n",
      "-- processing region: Chr7-6Mb-seg-505 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-505.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-505\n",
      "--- total time for blast Chr7-6Mb-seg-505: 3.320s.\n",
      "-- processing region: Chr7-6Mb-seg-506 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-506.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-506\n",
      "--- total time for blast Chr7-6Mb-seg-506: 4.622s.\n",
      "-- processing region: Chr7-6Mb-seg-507 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-507.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-507\n",
      "--- total time for blast Chr7-6Mb-seg-507: 4.178s.\n",
      "-- processing region: Chr7-6Mb-seg-508 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-508.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-508\n",
      "--- total time for blast Chr7-6Mb-seg-508: 3.774s.\n",
      "-- processing region: Chr7-6Mb-seg-509 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-509.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-509\n",
      "--- total time for blast Chr7-6Mb-seg-509: 3.948s.\n",
      "-- processing region: Chr7-6Mb-seg-51 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-51.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-51\n",
      "--- total time for blast Chr7-6Mb-seg-51: 3.943s.\n",
      "-- processing region: Chr7-6Mb-seg-510 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-510.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-510\n",
      "--- total time for blast Chr7-6Mb-seg-510: 3.811s.\n",
      "-- processing region: Chr7-6Mb-seg-511 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-511.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-511\n",
      "--- total time for blast Chr7-6Mb-seg-511: 3.998s.\n",
      "-- processing region: Chr7-6Mb-seg-512 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-512.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-512\n",
      "--- total time for blast Chr7-6Mb-seg-512: 3.729s.\n",
      "-- processing region: Chr7-6Mb-seg-513 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-513.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-513\n",
      "--- total time for blast Chr7-6Mb-seg-513: 3.823s.\n",
      "-- processing region: Chr7-6Mb-seg-514 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-514.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-514\n",
      "--- total time for blast Chr7-6Mb-seg-514: 4.642s.\n",
      "-- processing region: Chr7-6Mb-seg-515 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-515.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-515\n",
      "--- total time for blast Chr7-6Mb-seg-515: 3.617s.\n",
      "-- processing region: Chr7-6Mb-seg-516 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-516.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-516\n",
      "--- total time for blast Chr7-6Mb-seg-516: 5.281s.\n",
      "-- processing region: Chr7-6Mb-seg-517 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-517.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-517\n",
      "--- total time for blast Chr7-6Mb-seg-517: 4.323s.\n",
      "-- processing region: Chr7-6Mb-seg-518 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-518.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-518\n",
      "--- total time for blast Chr7-6Mb-seg-518: 4.416s.\n",
      "-- processing region: Chr7-6Mb-seg-519 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-519.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-519\n",
      "--- total time for blast Chr7-6Mb-seg-519: 4.586s.\n",
      "-- processing region: Chr7-6Mb-seg-52 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-52.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-52\n",
      "--- total time for blast Chr7-6Mb-seg-52: 4.713s.\n",
      "-- processing region: Chr7-6Mb-seg-520 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-520.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-520\n",
      "--- total time for blast Chr7-6Mb-seg-520: 3.761s.\n",
      "-- processing region: Chr7-6Mb-seg-521 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-521.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-521\n",
      "--- total time for blast Chr7-6Mb-seg-521: 3.623s.\n",
      "-- processing region: Chr7-6Mb-seg-522 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-522.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-522\n",
      "--- total time for blast Chr7-6Mb-seg-522: 4.439s.\n",
      "-- processing region: Chr7-6Mb-seg-523 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-523.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-523\n",
      "--- total time for blast Chr7-6Mb-seg-523: 3.778s.\n",
      "-- processing region: Chr7-6Mb-seg-524 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-524.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-524\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-524: 4.035s.\n",
      "-- processing region: Chr7-6Mb-seg-525 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-525.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-525\n",
      "--- total time for blast Chr7-6Mb-seg-525: 3.725s.\n",
      "-- processing region: Chr7-6Mb-seg-526 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-526.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-526\n",
      "--- total time for blast Chr7-6Mb-seg-526: 3.575s.\n",
      "-- processing region: Chr7-6Mb-seg-527 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-527.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-527\n",
      "--- total time for blast Chr7-6Mb-seg-527: 4.200s.\n",
      "-- processing region: Chr7-6Mb-seg-528 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-528.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-528\n",
      "--- total time for blast Chr7-6Mb-seg-528: 4.500s.\n",
      "-- processing region: Chr7-6Mb-seg-529 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-529.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-529\n",
      "--- total time for blast Chr7-6Mb-seg-529: 4.130s.\n",
      "-- processing region: Chr7-6Mb-seg-53 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-53.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-53\n",
      "--- total time for blast Chr7-6Mb-seg-53: 5.287s.\n",
      "-- processing region: Chr7-6Mb-seg-530 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-530.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-530\n",
      "--- total time for blast Chr7-6Mb-seg-530: 4.071s.\n",
      "-- processing region: Chr7-6Mb-seg-531 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-531.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-531\n",
      "--- total time for blast Chr7-6Mb-seg-531: 4.494s.\n",
      "-- processing region: Chr7-6Mb-seg-532 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-532.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-532\n",
      "--- total time for blast Chr7-6Mb-seg-532: 5.998s.\n",
      "-- processing region: Chr7-6Mb-seg-533 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-533.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-533\n",
      "--- total time for blast Chr7-6Mb-seg-533: 4.907s.\n",
      "-- processing region: Chr7-6Mb-seg-534 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-534.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-534\n",
      "--- total time for blast Chr7-6Mb-seg-534: 3.772s.\n",
      "-- processing region: Chr7-6Mb-seg-535 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-535.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-535\n",
      "--- total time for blast Chr7-6Mb-seg-535: 4.303s.\n",
      "-- processing region: Chr7-6Mb-seg-536 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-536.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-536\n",
      "--- total time for blast Chr7-6Mb-seg-536: 5.748s.\n",
      "-- processing region: Chr7-6Mb-seg-537 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-537.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-537\n",
      "--- total time for blast Chr7-6Mb-seg-537: 4.622s.\n",
      "-- processing region: Chr7-6Mb-seg-538 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-538.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-538\n",
      "--- total time for blast Chr7-6Mb-seg-538: 5.808s.\n",
      "-- processing region: Chr7-6Mb-seg-539 with 33 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-539.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-539\n",
      "--- total time for blast Chr7-6Mb-seg-539: 8.675s.\n",
      "-- processing region: Chr7-6Mb-seg-54 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-54.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-54\n",
      "--- total time for blast Chr7-6Mb-seg-54: 4.044s.\n",
      "-- processing region: Chr7-6Mb-seg-540 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-540.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-540\n",
      "--- total time for blast Chr7-6Mb-seg-540: 4.037s.\n",
      "-- processing region: Chr7-6Mb-seg-541 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-541.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-541\n",
      "--- total time for blast Chr7-6Mb-seg-541: 3.950s.\n",
      "-- processing region: Chr7-6Mb-seg-542 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-542.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-542\n",
      "--- total time for blast Chr7-6Mb-seg-542: 3.722s.\n",
      "-- processing region: Chr7-6Mb-seg-543 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-543.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-543\n",
      "--- total time for blast Chr7-6Mb-seg-543: 3.330s.\n",
      "-- processing region: Chr7-6Mb-seg-544 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-544.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-544\n",
      "--- total time for blast Chr7-6Mb-seg-544: 4.494s.\n",
      "-- processing region: Chr7-6Mb-seg-545 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-545.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-545\n",
      "--- total time for blast Chr7-6Mb-seg-545: 4.221s.\n",
      "-- processing region: Chr7-6Mb-seg-546 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-546.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-546\n",
      "--- total time for blast Chr7-6Mb-seg-546: 5.203s.\n",
      "-- processing region: Chr7-6Mb-seg-547 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-547.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-547\n",
      "--- total time for blast Chr7-6Mb-seg-547: 7.359s.\n",
      "-- processing region: Chr7-6Mb-seg-548 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-548.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-548\n",
      "--- total time for blast Chr7-6Mb-seg-548: 4.887s.\n",
      "-- processing region: Chr7-6Mb-seg-549 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-549.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-549\n",
      "--- total time for blast Chr7-6Mb-seg-549: 20.294s.\n",
      "-- processing region: Chr7-6Mb-seg-55 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-55.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-55\n",
      "--- total time for blast Chr7-6Mb-seg-55: 4.764s.\n",
      "-- processing region: Chr7-6Mb-seg-550 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-550.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-550\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-550: 3.425s.\n",
      "-- processing region: Chr7-6Mb-seg-551 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-551.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-551\n",
      "--- total time for blast Chr7-6Mb-seg-551: 9.538s.\n",
      "-- processing region: Chr7-6Mb-seg-552 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-552.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-552\n",
      "--- total time for blast Chr7-6Mb-seg-552: 5.312s.\n",
      "-- processing region: Chr7-6Mb-seg-553 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-553.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-553\n",
      "--- total time for blast Chr7-6Mb-seg-553: 3.970s.\n",
      "-- processing region: Chr7-6Mb-seg-554 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-554.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-554\n",
      "--- total time for blast Chr7-6Mb-seg-554: 3.654s.\n",
      "-- processing region: Chr7-6Mb-seg-555 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-555.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-555\n",
      "--- total time for blast Chr7-6Mb-seg-555: 4.391s.\n",
      "-- processing region: Chr7-6Mb-seg-556 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-556.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-556\n",
      "--- total time for blast Chr7-6Mb-seg-556: 5.066s.\n",
      "-- processing region: Chr7-6Mb-seg-557 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-557.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-557\n",
      "--- total time for blast Chr7-6Mb-seg-557: 4.679s.\n",
      "-- processing region: Chr7-6Mb-seg-558 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-558.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-558\n",
      "--- total time for blast Chr7-6Mb-seg-558: 3.469s.\n",
      "-- processing region: Chr7-6Mb-seg-559 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-559.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-559\n",
      "--- total time for blast Chr7-6Mb-seg-559: 4.206s.\n",
      "-- processing region: Chr7-6Mb-seg-56 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-56.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-56\n",
      "--- total time for blast Chr7-6Mb-seg-56: 4.894s.\n",
      "-- processing region: Chr7-6Mb-seg-560 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-560.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-560\n",
      "--- total time for blast Chr7-6Mb-seg-560: 3.634s.\n",
      "-- processing region: Chr7-6Mb-seg-561 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-561.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-561\n",
      "--- total time for blast Chr7-6Mb-seg-561: 4.318s.\n",
      "-- processing region: Chr7-6Mb-seg-562 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-562.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-562\n",
      "--- total time for blast Chr7-6Mb-seg-562: 4.747s.\n",
      "-- processing region: Chr7-6Mb-seg-563 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-563.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-563\n",
      "--- total time for blast Chr7-6Mb-seg-563: 3.815s.\n",
      "-- processing region: Chr7-6Mb-seg-564 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-564.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-564\n",
      "--- total time for blast Chr7-6Mb-seg-564: 3.787s.\n",
      "-- processing region: Chr7-6Mb-seg-565 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-565.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-565\n",
      "--- total time for blast Chr7-6Mb-seg-565: 4.070s.\n",
      "-- processing region: Chr7-6Mb-seg-566 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-566.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-566\n",
      "--- total time for blast Chr7-6Mb-seg-566: 4.332s.\n",
      "-- processing region: Chr7-6Mb-seg-567 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-567.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-567\n",
      "--- total time for blast Chr7-6Mb-seg-567: 4.442s.\n",
      "-- processing region: Chr7-6Mb-seg-568 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-568.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-568\n",
      "--- total time for blast Chr7-6Mb-seg-568: 4.030s.\n",
      "-- processing region: Chr7-6Mb-seg-569 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-569.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-569\n",
      "--- total time for blast Chr7-6Mb-seg-569: 3.496s.\n",
      "-- processing region: Chr7-6Mb-seg-57 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-57.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-57\n",
      "--- total time for blast Chr7-6Mb-seg-57: 3.997s.\n",
      "-- processing region: Chr7-6Mb-seg-570 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-570.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-570\n",
      "--- total time for blast Chr7-6Mb-seg-570: 3.458s.\n",
      "-- processing region: Chr7-6Mb-seg-571 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-571.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-571\n",
      "--- total time for blast Chr7-6Mb-seg-571: 4.197s.\n",
      "-- processing region: Chr7-6Mb-seg-572 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-572.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-572\n",
      "--- total time for blast Chr7-6Mb-seg-572: 4.407s.\n",
      "-- processing region: Chr7-6Mb-seg-573 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-573.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-573\n",
      "--- total time for blast Chr7-6Mb-seg-573: 3.551s.\n",
      "-- processing region: Chr7-6Mb-seg-574 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-574.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-574\n",
      "--- total time for blast Chr7-6Mb-seg-574: 3.959s.\n",
      "-- processing region: Chr7-6Mb-seg-575 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-575.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-575\n",
      "--- total time for blast Chr7-6Mb-seg-575: 3.806s.\n",
      "-- processing region: Chr7-6Mb-seg-576 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-576.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-576\n",
      "--- total time for blast Chr7-6Mb-seg-576: 3.909s.\n",
      "-- processing region: Chr7-6Mb-seg-577 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-577.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-577\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-577: 4.788s.\n",
      "-- processing region: Chr7-6Mb-seg-578 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-578.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-578\n",
      "--- total time for blast Chr7-6Mb-seg-578: 5.767s.\n",
      "-- processing region: Chr7-6Mb-seg-579 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-579.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-579\n",
      "--- total time for blast Chr7-6Mb-seg-579: 3.969s.\n",
      "-- processing region: Chr7-6Mb-seg-58 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-58.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-58\n",
      "--- total time for blast Chr7-6Mb-seg-58: 4.001s.\n",
      "-- processing region: Chr7-6Mb-seg-580 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-580.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-580\n",
      "--- total time for blast Chr7-6Mb-seg-580: 7.079s.\n",
      "-- processing region: Chr7-6Mb-seg-581 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-581.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-581\n",
      "--- total time for blast Chr7-6Mb-seg-581: 4.569s.\n",
      "-- processing region: Chr7-6Mb-seg-582 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-582.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-582\n",
      "--- total time for blast Chr7-6Mb-seg-582: 3.627s.\n",
      "-- processing region: Chr7-6Mb-seg-583 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-583.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-583\n",
      "--- total time for blast Chr7-6Mb-seg-583: 4.263s.\n",
      "-- processing region: Chr7-6Mb-seg-584 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-584.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-584\n",
      "--- total time for blast Chr7-6Mb-seg-584: 4.196s.\n",
      "-- processing region: Chr7-6Mb-seg-585 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-585.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-585\n",
      "--- total time for blast Chr7-6Mb-seg-585: 8.344s.\n",
      "-- processing region: Chr7-6Mb-seg-586 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-586.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-586\n",
      "--- total time for blast Chr7-6Mb-seg-586: 4.121s.\n",
      "-- processing region: Chr7-6Mb-seg-587 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-587.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-587\n",
      "--- total time for blast Chr7-6Mb-seg-587: 3.613s.\n",
      "-- processing region: Chr7-6Mb-seg-588 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-588.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-588\n",
      "--- total time for blast Chr7-6Mb-seg-588: 3.767s.\n",
      "-- processing region: Chr7-6Mb-seg-589 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-589.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-589\n",
      "--- total time for blast Chr7-6Mb-seg-589: 3.448s.\n",
      "-- processing region: Chr7-6Mb-seg-59 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-59.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-59\n",
      "--- total time for blast Chr7-6Mb-seg-59: 4.204s.\n",
      "-- processing region: Chr7-6Mb-seg-590 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-590.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-590\n",
      "--- total time for blast Chr7-6Mb-seg-590: 6.018s.\n",
      "-- processing region: Chr7-6Mb-seg-591 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-591.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-591\n",
      "--- total time for blast Chr7-6Mb-seg-591: 8.818s.\n",
      "-- processing region: Chr7-6Mb-seg-592 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-592.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-592\n",
      "--- total time for blast Chr7-6Mb-seg-592: 3.415s.\n",
      "-- processing region: Chr7-6Mb-seg-593 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-593.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-593\n",
      "--- total time for blast Chr7-6Mb-seg-593: 4.141s.\n",
      "-- processing region: Chr7-6Mb-seg-594 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-594.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-594\n",
      "--- total time for blast Chr7-6Mb-seg-594: 4.431s.\n",
      "-- processing region: Chr7-6Mb-seg-595 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-595.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-595\n",
      "--- total time for blast Chr7-6Mb-seg-595: 5.364s.\n",
      "-- processing region: Chr7-6Mb-seg-596 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-596.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-596\n",
      "--- total time for blast Chr7-6Mb-seg-596: 8.699s.\n",
      "-- processing region: Chr7-6Mb-seg-597 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-597.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-597\n",
      "--- total time for blast Chr7-6Mb-seg-597: 5.275s.\n",
      "-- processing region: Chr7-6Mb-seg-598 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-598.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-598\n",
      "--- total time for blast Chr7-6Mb-seg-598: 4.491s.\n",
      "-- processing region: Chr7-6Mb-seg-599 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-599.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-599\n",
      "--- total time for blast Chr7-6Mb-seg-599: 4.418s.\n",
      "-- processing region: Chr7-6Mb-seg-60 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-60.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-60\n",
      "--- total time for blast Chr7-6Mb-seg-60: 4.255s.\n",
      "-- processing region: Chr7-6Mb-seg-600 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-600.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-600\n",
      "--- total time for blast Chr7-6Mb-seg-600: 4.444s.\n",
      "-- processing region: Chr7-6Mb-seg-601 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-601.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-601\n",
      "--- total time for blast Chr7-6Mb-seg-601: 3.239s.\n",
      "-- processing region: Chr7-6Mb-seg-602 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-602.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-602\n",
      "--- total time for blast Chr7-6Mb-seg-602: 4.117s.\n",
      "-- processing region: Chr7-6Mb-seg-603 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-603.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-603\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-603: 10.477s.\n",
      "-- processing region: Chr7-6Mb-seg-604 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-604.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-604\n",
      "--- total time for blast Chr7-6Mb-seg-604: 4.642s.\n",
      "-- processing region: Chr7-6Mb-seg-605 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-605.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-605\n",
      "--- total time for blast Chr7-6Mb-seg-605: 4.527s.\n",
      "-- processing region: Chr7-6Mb-seg-606 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-606.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-606\n",
      "--- total time for blast Chr7-6Mb-seg-606: 3.782s.\n",
      "-- processing region: Chr7-6Mb-seg-607 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-607.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-607\n",
      "--- total time for blast Chr7-6Mb-seg-607: 3.672s.\n",
      "-- processing region: Chr7-6Mb-seg-608 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-608.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-608\n",
      "--- total time for blast Chr7-6Mb-seg-608: 2.826s.\n",
      "-- processing region: Chr7-6Mb-seg-609 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-609.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-609\n",
      "--- total time for blast Chr7-6Mb-seg-609: 4.890s.\n",
      "-- processing region: Chr7-6Mb-seg-61 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-61.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-61\n",
      "--- total time for blast Chr7-6Mb-seg-61: 3.669s.\n",
      "-- processing region: Chr7-6Mb-seg-610 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-610.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-610\n",
      "--- total time for blast Chr7-6Mb-seg-610: 3.879s.\n",
      "-- processing region: Chr7-6Mb-seg-611 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-611.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-611\n",
      "--- total time for blast Chr7-6Mb-seg-611: 3.979s.\n",
      "-- processing region: Chr7-6Mb-seg-612 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-612.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-612\n",
      "--- total time for blast Chr7-6Mb-seg-612: 3.770s.\n",
      "-- processing region: Chr7-6Mb-seg-613 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-613.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-613\n",
      "--- total time for blast Chr7-6Mb-seg-613: 4.157s.\n",
      "-- processing region: Chr7-6Mb-seg-614 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-614.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-614\n",
      "--- total time for blast Chr7-6Mb-seg-614: 3.509s.\n",
      "-- processing region: Chr7-6Mb-seg-615 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-615.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-615\n",
      "--- total time for blast Chr7-6Mb-seg-615: 4.134s.\n",
      "-- processing region: Chr7-6Mb-seg-616 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-616.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-616\n",
      "--- total time for blast Chr7-6Mb-seg-616: 4.363s.\n",
      "-- processing region: Chr7-6Mb-seg-617 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-617.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-617\n",
      "--- total time for blast Chr7-6Mb-seg-617: 4.633s.\n",
      "-- processing region: Chr7-6Mb-seg-618 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-618.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-618\n",
      "--- total time for blast Chr7-6Mb-seg-618: 3.641s.\n",
      "-- processing region: Chr7-6Mb-seg-619 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-619.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-619\n",
      "--- total time for blast Chr7-6Mb-seg-619: 3.349s.\n",
      "-- processing region: Chr7-6Mb-seg-62 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-62.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-62\n",
      "--- total time for blast Chr7-6Mb-seg-62: 3.863s.\n",
      "-- processing region: Chr7-6Mb-seg-620 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-620.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-620\n",
      "--- total time for blast Chr7-6Mb-seg-620: 4.447s.\n",
      "-- processing region: Chr7-6Mb-seg-621 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-621.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-621\n",
      "--- total time for blast Chr7-6Mb-seg-621: 3.937s.\n",
      "-- processing region: Chr7-6Mb-seg-622 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-622.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-622\n",
      "--- total time for blast Chr7-6Mb-seg-622: 3.893s.\n",
      "-- processing region: Chr7-6Mb-seg-623 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-623.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-623\n",
      "--- total time for blast Chr7-6Mb-seg-623: 3.885s.\n",
      "-- processing region: Chr7-6Mb-seg-624 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-624.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-624\n",
      "--- total time for blast Chr7-6Mb-seg-624: 4.766s.\n",
      "-- processing region: Chr7-6Mb-seg-625 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-625.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-625\n",
      "--- total time for blast Chr7-6Mb-seg-625: 10.636s.\n",
      "-- processing region: Chr7-6Mb-seg-626 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-626.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-626\n",
      "--- total time for blast Chr7-6Mb-seg-626: 4.272s.\n",
      "-- processing region: Chr7-6Mb-seg-627 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-627.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-627\n",
      "--- total time for blast Chr7-6Mb-seg-627: 3.299s.\n",
      "-- processing region: Chr7-6Mb-seg-628 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-628.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-628\n",
      "--- total time for blast Chr7-6Mb-seg-628: 5.000s.\n",
      "-- processing region: Chr7-6Mb-seg-629 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-629.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-629\n",
      "--- total time for blast Chr7-6Mb-seg-629: 4.160s.\n",
      "-- processing region: Chr7-6Mb-seg-63 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-63.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-63\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-63: 3.498s.\n",
      "-- processing region: Chr7-6Mb-seg-630 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-630.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-630\n",
      "--- total time for blast Chr7-6Mb-seg-630: 4.055s.\n",
      "-- processing region: Chr7-6Mb-seg-631 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-631.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-631\n",
      "--- total time for blast Chr7-6Mb-seg-631: 4.964s.\n",
      "-- processing region: Chr7-6Mb-seg-632 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-632.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-632\n",
      "--- total time for blast Chr7-6Mb-seg-632: 3.523s.\n",
      "-- processing region: Chr7-6Mb-seg-633 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-633.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-633\n",
      "--- total time for blast Chr7-6Mb-seg-633: 9.090s.\n",
      "-- processing region: Chr7-6Mb-seg-634 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-634.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-634\n",
      "--- total time for blast Chr7-6Mb-seg-634: 4.179s.\n",
      "-- processing region: Chr7-6Mb-seg-635 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-635.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-635\n",
      "--- total time for blast Chr7-6Mb-seg-635: 3.678s.\n",
      "-- processing region: Chr7-6Mb-seg-638 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-638.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-638\n",
      "--- total time for blast Chr7-6Mb-seg-638: 2.080s.\n",
      "-- processing region: Chr7-6Mb-seg-639 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-639.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-639\n",
      "--- total time for blast Chr7-6Mb-seg-639: 4.350s.\n",
      "-- processing region: Chr7-6Mb-seg-64 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-64.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-64\n",
      "--- total time for blast Chr7-6Mb-seg-64: 6.450s.\n",
      "-- processing region: Chr7-6Mb-seg-640 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-640.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-640\n",
      "--- total time for blast Chr7-6Mb-seg-640: 1.709s.\n",
      "-- processing region: Chr7-6Mb-seg-641 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-641.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-641\n",
      "--- total time for blast Chr7-6Mb-seg-641: 3.120s.\n",
      "-- processing region: Chr7-6Mb-seg-642 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-642.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-642\n",
      "--- total time for blast Chr7-6Mb-seg-642: 4.600s.\n",
      "-- processing region: Chr7-6Mb-seg-643 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-643.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-643\n",
      "--- total time for blast Chr7-6Mb-seg-643: 3.944s.\n",
      "-- processing region: Chr7-6Mb-seg-644 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-644.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-644\n",
      "--- total time for blast Chr7-6Mb-seg-644: 3.281s.\n",
      "-- processing region: Chr7-6Mb-seg-647 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-647.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-647\n",
      "--- total time for blast Chr7-6Mb-seg-647: 4.743s.\n",
      "-- processing region: Chr7-6Mb-seg-648 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-648.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-648\n",
      "--- total time for blast Chr7-6Mb-seg-648: 6.191s.\n",
      "-- processing region: Chr7-6Mb-seg-649 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-649.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-649\n",
      "--- total time for blast Chr7-6Mb-seg-649: 8.205s.\n",
      "-- processing region: Chr7-6Mb-seg-65 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-65.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-65\n",
      "--- total time for blast Chr7-6Mb-seg-65: 5.143s.\n",
      "-- processing region: Chr7-6Mb-seg-650 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-650.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-650\n",
      "--- total time for blast Chr7-6Mb-seg-650: 3.490s.\n",
      "-- processing region: Chr7-6Mb-seg-651 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-651.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-651\n",
      "--- total time for blast Chr7-6Mb-seg-651: 3.233s.\n",
      "-- processing region: Chr7-6Mb-seg-655 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-655.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-655\n",
      "--- total time for blast Chr7-6Mb-seg-655: 2.908s.\n",
      "-- processing region: Chr7-6Mb-seg-66 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-66.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-66\n",
      "--- total time for blast Chr7-6Mb-seg-66: 4.450s.\n",
      "-- processing region: Chr7-6Mb-seg-660 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-660.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-660\n",
      "--- total time for blast Chr7-6Mb-seg-660: 4.260s.\n",
      "-- processing region: Chr7-6Mb-seg-661 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-661.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-661\n",
      "--- total time for blast Chr7-6Mb-seg-661: 3.049s.\n",
      "-- processing region: Chr7-6Mb-seg-662 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-662.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-662\n",
      "--- total time for blast Chr7-6Mb-seg-662: 6.773s.\n",
      "-- processing region: Chr7-6Mb-seg-663 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-663.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-663\n",
      "--- total time for blast Chr7-6Mb-seg-663: 13.034s.\n",
      "-- processing region: Chr7-6Mb-seg-664 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-664.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-664\n",
      "--- total time for blast Chr7-6Mb-seg-664: 3.048s.\n",
      "-- processing region: Chr7-6Mb-seg-665 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-665.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-665\n",
      "--- total time for blast Chr7-6Mb-seg-665: 2.342s.\n",
      "-- processing region: Chr7-6Mb-seg-668 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-668.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-668\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-668: 4.802s.\n",
      "-- processing region: Chr7-6Mb-seg-669 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-669.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-669\n",
      "--- total time for blast Chr7-6Mb-seg-669: 5.374s.\n",
      "-- processing region: Chr7-6Mb-seg-67 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-67.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-67\n",
      "--- total time for blast Chr7-6Mb-seg-67: 4.290s.\n",
      "-- processing region: Chr7-6Mb-seg-670 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-670.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-670\n",
      "--- total time for blast Chr7-6Mb-seg-670: 4.369s.\n",
      "-- processing region: Chr7-6Mb-seg-671 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-671.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-671\n",
      "--- total time for blast Chr7-6Mb-seg-671: 6.734s.\n",
      "-- processing region: Chr7-6Mb-seg-672 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-672.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-672\n",
      "--- total time for blast Chr7-6Mb-seg-672: 3.188s.\n",
      "-- processing region: Chr7-6Mb-seg-674 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-674.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-674\n",
      "--- total time for blast Chr7-6Mb-seg-674: 3.132s.\n",
      "-- processing region: Chr7-6Mb-seg-675 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-675.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-675\n",
      "--- total time for blast Chr7-6Mb-seg-675: 4.021s.\n",
      "-- processing region: Chr7-6Mb-seg-676 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-676.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-676\n",
      "--- total time for blast Chr7-6Mb-seg-676: 7.869s.\n",
      "-- processing region: Chr7-6Mb-seg-677 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-677.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-677\n",
      "--- total time for blast Chr7-6Mb-seg-677: 3.613s.\n",
      "-- processing region: Chr7-6Mb-seg-679 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-679.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-679\n",
      "--- total time for blast Chr7-6Mb-seg-679: 3.362s.\n",
      "-- processing region: Chr7-6Mb-seg-68 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-68.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-68\n",
      "--- total time for blast Chr7-6Mb-seg-68: 5.424s.\n",
      "-- processing region: Chr7-6Mb-seg-680 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-680.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-680\n",
      "--- total time for blast Chr7-6Mb-seg-680: 8.595s.\n",
      "-- processing region: Chr7-6Mb-seg-681 with 33 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-681.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-681\n",
      "--- total time for blast Chr7-6Mb-seg-681: 4.569s.\n",
      "-- processing region: Chr7-6Mb-seg-682 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-682.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-682\n",
      "--- total time for blast Chr7-6Mb-seg-682: 5.844s.\n",
      "-- processing region: Chr7-6Mb-seg-683 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-683.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-683\n",
      "--- total time for blast Chr7-6Mb-seg-683: 3.115s.\n",
      "-- processing region: Chr7-6Mb-seg-69 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-69.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-69\n",
      "--- total time for blast Chr7-6Mb-seg-69: 4.202s.\n",
      "-- processing region: Chr7-6Mb-seg-692 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-692.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-692\n",
      "--- total time for blast Chr7-6Mb-seg-692: 5.252s.\n",
      "-- processing region: Chr7-6Mb-seg-693 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-693.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-693\n",
      "--- total time for blast Chr7-6Mb-seg-693: 7.934s.\n",
      "-- processing region: Chr7-6Mb-seg-698 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-698.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-698\n",
      "--- total time for blast Chr7-6Mb-seg-698: 3.870s.\n",
      "-- processing region: Chr7-6Mb-seg-699 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-699.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-699\n",
      "--- total time for blast Chr7-6Mb-seg-699: 3.708s.\n",
      "-- processing region: Chr7-6Mb-seg-7 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-7.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-7\n",
      "--- total time for blast Chr7-6Mb-seg-7: 3.958s.\n",
      "-- processing region: Chr7-6Mb-seg-70 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-70.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-70\n",
      "--- total time for blast Chr7-6Mb-seg-70: 4.117s.\n",
      "-- processing region: Chr7-6Mb-seg-700 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-700.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-700\n",
      "--- total time for blast Chr7-6Mb-seg-700: 5.561s.\n",
      "-- processing region: Chr7-6Mb-seg-701 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-701.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-701\n",
      "--- total time for blast Chr7-6Mb-seg-701: 7.052s.\n",
      "-- processing region: Chr7-6Mb-seg-703 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-703.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-703\n",
      "--- total time for blast Chr7-6Mb-seg-703: 2.744s.\n",
      "-- processing region: Chr7-6Mb-seg-704 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-704.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-704\n",
      "--- total time for blast Chr7-6Mb-seg-704: 3.624s.\n",
      "-- processing region: Chr7-6Mb-seg-705 with 31 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-705.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-705\n",
      "--- total time for blast Chr7-6Mb-seg-705: 3.449s.\n",
      "-- processing region: Chr7-6Mb-seg-706 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-706.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-706\n",
      "--- total time for blast Chr7-6Mb-seg-706: 4.449s.\n",
      "-- processing region: Chr7-6Mb-seg-707 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-707.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-707\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-707: 4.227s.\n",
      "-- processing region: Chr7-6Mb-seg-708 with 31 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-708.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-708\n",
      "--- total time for blast Chr7-6Mb-seg-708: 6.094s.\n",
      "-- processing region: Chr7-6Mb-seg-71 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-71.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-71\n",
      "--- total time for blast Chr7-6Mb-seg-71: 4.907s.\n",
      "-- processing region: Chr7-6Mb-seg-710 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-710.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-710\n",
      "--- total time for blast Chr7-6Mb-seg-710: 10.050s.\n",
      "-- processing region: Chr7-6Mb-seg-711 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-711.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-711\n",
      "--- total time for blast Chr7-6Mb-seg-711: 5.523s.\n",
      "-- processing region: Chr7-6Mb-seg-712 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-712.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-712\n",
      "--- total time for blast Chr7-6Mb-seg-712: 2.121s.\n",
      "-- processing region: Chr7-6Mb-seg-713 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-713.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-713\n",
      "--- total time for blast Chr7-6Mb-seg-713: 2.810s.\n",
      "-- processing region: Chr7-6Mb-seg-717 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-717.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-717\n",
      "--- total time for blast Chr7-6Mb-seg-717: 1.781s.\n",
      "-- processing region: Chr7-6Mb-seg-718 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-718.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-718\n",
      "--- total time for blast Chr7-6Mb-seg-718: 9.646s.\n",
      "-- processing region: Chr7-6Mb-seg-719 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-719.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-719\n",
      "--- total time for blast Chr7-6Mb-seg-719: 6.188s.\n",
      "-- processing region: Chr7-6Mb-seg-72 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-72.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-72\n",
      "--- total time for blast Chr7-6Mb-seg-72: 3.667s.\n",
      "-- processing region: Chr7-6Mb-seg-720 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-720.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-720\n",
      "--- total time for blast Chr7-6Mb-seg-720: 3.651s.\n",
      "-- processing region: Chr7-6Mb-seg-721 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-721.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-721\n",
      "--- total time for blast Chr7-6Mb-seg-721: 3.035s.\n",
      "-- processing region: Chr7-6Mb-seg-722 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-722.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-722\n",
      "--- total time for blast Chr7-6Mb-seg-722: 10.510s.\n",
      "-- processing region: Chr7-6Mb-seg-723 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-723.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-723\n",
      "--- total time for blast Chr7-6Mb-seg-723: 6.121s.\n",
      "-- processing region: Chr7-6Mb-seg-726 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-726.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-726\n",
      "--- total time for blast Chr7-6Mb-seg-726: 2.840s.\n",
      "-- processing region: Chr7-6Mb-seg-727 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-727.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-727\n",
      "--- total time for blast Chr7-6Mb-seg-727: 2.660s.\n",
      "-- processing region: Chr7-6Mb-seg-728 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-728.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-728\n",
      "--- total time for blast Chr7-6Mb-seg-728: 2.904s.\n",
      "-- processing region: Chr7-6Mb-seg-73 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-73.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-73\n",
      "--- total time for blast Chr7-6Mb-seg-73: 3.121s.\n",
      "-- processing region: Chr7-6Mb-seg-731 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-731.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-731\n",
      "--- total time for blast Chr7-6Mb-seg-731: 5.448s.\n",
      "-- processing region: Chr7-6Mb-seg-732 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-732.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-732\n",
      "--- total time for blast Chr7-6Mb-seg-732: 10.734s.\n",
      "-- processing region: Chr7-6Mb-seg-733 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-733.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-733\n",
      "--- total time for blast Chr7-6Mb-seg-733: 8.209s.\n",
      "-- processing region: Chr7-6Mb-seg-734 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-734.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-734\n",
      "--- total time for blast Chr7-6Mb-seg-734: 8.871s.\n",
      "-- processing region: Chr7-6Mb-seg-735 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-735.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-735\n",
      "--- total time for blast Chr7-6Mb-seg-735: 10.617s.\n",
      "-- processing region: Chr7-6Mb-seg-738 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-738.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-738\n",
      "--- total time for blast Chr7-6Mb-seg-738: 4.051s.\n",
      "-- processing region: Chr7-6Mb-seg-739 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-739.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-739\n",
      "--- total time for blast Chr7-6Mb-seg-739: 3.061s.\n",
      "-- processing region: Chr7-6Mb-seg-74 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-74.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-74\n",
      "--- total time for blast Chr7-6Mb-seg-74: 4.931s.\n",
      "-- processing region: Chr7-6Mb-seg-740 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-740.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-740\n",
      "--- total time for blast Chr7-6Mb-seg-740: 16.100s.\n",
      "-- processing region: Chr7-6Mb-seg-741 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-741.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-741\n",
      "--- total time for blast Chr7-6Mb-seg-741: 12.527s.\n",
      "-- processing region: Chr7-6Mb-seg-742 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-742.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-742\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-742: 4.939s.\n",
      "-- processing region: Chr7-6Mb-seg-743 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-743.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-743\n",
      "--- total time for blast Chr7-6Mb-seg-743: 4.230s.\n",
      "-- processing region: Chr7-6Mb-seg-744 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-744.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-744\n",
      "--- total time for blast Chr7-6Mb-seg-744: 3.487s.\n",
      "-- processing region: Chr7-6Mb-seg-745 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-745.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-745\n",
      "--- total time for blast Chr7-6Mb-seg-745: 7.855s.\n",
      "-- processing region: Chr7-6Mb-seg-746 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-746.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-746\n",
      "--- total time for blast Chr7-6Mb-seg-746: 4.615s.\n",
      "-- processing region: Chr7-6Mb-seg-747 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-747.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-747\n",
      "--- total time for blast Chr7-6Mb-seg-747: 5.506s.\n",
      "-- processing region: Chr7-6Mb-seg-748 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-748.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-748\n",
      "--- total time for blast Chr7-6Mb-seg-748: 4.866s.\n",
      "-- processing region: Chr7-6Mb-seg-749 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-749.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-749\n",
      "--- total time for blast Chr7-6Mb-seg-749: 3.880s.\n",
      "-- processing region: Chr7-6Mb-seg-75 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-75.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-75\n",
      "--- total time for blast Chr7-6Mb-seg-75: 5.546s.\n",
      "-- processing region: Chr7-6Mb-seg-750 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-750.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-750\n",
      "--- total time for blast Chr7-6Mb-seg-750: 3.716s.\n",
      "-- processing region: Chr7-6Mb-seg-751 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-751.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-751\n",
      "--- total time for blast Chr7-6Mb-seg-751: 9.429s.\n",
      "-- processing region: Chr7-6Mb-seg-752 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-752.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-752\n",
      "--- total time for blast Chr7-6Mb-seg-752: 3.850s.\n",
      "-- processing region: Chr7-6Mb-seg-753 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-753.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-753\n",
      "--- total time for blast Chr7-6Mb-seg-753: 4.314s.\n",
      "-- processing region: Chr7-6Mb-seg-754 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-754.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-754\n",
      "--- total time for blast Chr7-6Mb-seg-754: 3.046s.\n",
      "-- processing region: Chr7-6Mb-seg-755 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-755.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-755\n",
      "--- total time for blast Chr7-6Mb-seg-755: 3.197s.\n",
      "-- processing region: Chr7-6Mb-seg-756 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-756.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-756\n",
      "--- total time for blast Chr7-6Mb-seg-756: 8.075s.\n",
      "-- processing region: Chr7-6Mb-seg-757 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-757.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-757\n",
      "--- total time for blast Chr7-6Mb-seg-757: 2.668s.\n",
      "-- processing region: Chr7-6Mb-seg-758 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-758.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-758\n",
      "--- total time for blast Chr7-6Mb-seg-758: 3.901s.\n",
      "-- processing region: Chr7-6Mb-seg-76 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-76.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-76\n",
      "--- total time for blast Chr7-6Mb-seg-76: 4.184s.\n",
      "-- processing region: Chr7-6Mb-seg-760 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-760.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-760\n",
      "--- total time for blast Chr7-6Mb-seg-760: 3.155s.\n",
      "-- processing region: Chr7-6Mb-seg-762 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-762.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-762\n",
      "--- total time for blast Chr7-6Mb-seg-762: 16.461s.\n",
      "-- processing region: Chr7-6Mb-seg-763 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-763.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-763\n",
      "--- total time for blast Chr7-6Mb-seg-763: 9.532s.\n",
      "-- processing region: Chr7-6Mb-seg-764 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-764.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-764\n",
      "--- total time for blast Chr7-6Mb-seg-764: 4.098s.\n",
      "-- processing region: Chr7-6Mb-seg-765 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-765.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-765\n",
      "--- total time for blast Chr7-6Mb-seg-765: 8.882s.\n",
      "-- processing region: Chr7-6Mb-seg-766 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-766.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-766\n",
      "--- total time for blast Chr7-6Mb-seg-766: 6.208s.\n",
      "-- processing region: Chr7-6Mb-seg-767 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-767.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-767\n",
      "--- total time for blast Chr7-6Mb-seg-767: 3.508s.\n",
      "-- processing region: Chr7-6Mb-seg-768 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-768.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-768\n",
      "--- total time for blast Chr7-6Mb-seg-768: 3.325s.\n",
      "-- processing region: Chr7-6Mb-seg-769 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-769.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-769\n",
      "--- total time for blast Chr7-6Mb-seg-769: 3.663s.\n",
      "-- processing region: Chr7-6Mb-seg-77 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-77.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-77\n",
      "--- total time for blast Chr7-6Mb-seg-77: 4.648s.\n",
      "-- processing region: Chr7-6Mb-seg-770 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-770.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-770\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-770: 4.009s.\n",
      "-- processing region: Chr7-6Mb-seg-771 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-771.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-771\n",
      "--- total time for blast Chr7-6Mb-seg-771: 3.675s.\n",
      "-- processing region: Chr7-6Mb-seg-772 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-772.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-772\n",
      "--- total time for blast Chr7-6Mb-seg-772: 9.362s.\n",
      "-- processing region: Chr7-6Mb-seg-773 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-773.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-773\n",
      "--- total time for blast Chr7-6Mb-seg-773: 2.344s.\n",
      "-- processing region: Chr7-6Mb-seg-775 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-775.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-775\n",
      "--- total time for blast Chr7-6Mb-seg-775: 4.559s.\n",
      "-- processing region: Chr7-6Mb-seg-776 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-776.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-776\n",
      "--- total time for blast Chr7-6Mb-seg-776: 10.806s.\n",
      "-- processing region: Chr7-6Mb-seg-777 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-777.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-777\n",
      "--- total time for blast Chr7-6Mb-seg-777: 5.117s.\n",
      "-- processing region: Chr7-6Mb-seg-778 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-778.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-778\n",
      "--- total time for blast Chr7-6Mb-seg-778: 11.373s.\n",
      "-- processing region: Chr7-6Mb-seg-78 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-78.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-78\n",
      "--- total time for blast Chr7-6Mb-seg-78: 3.881s.\n",
      "-- processing region: Chr7-6Mb-seg-780 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-780.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-780\n",
      "--- total time for blast Chr7-6Mb-seg-780: 1.815s.\n",
      "-- processing region: Chr7-6Mb-seg-781 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-781.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-781\n",
      "--- total time for blast Chr7-6Mb-seg-781: 10.596s.\n",
      "-- processing region: Chr7-6Mb-seg-784 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-784.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-784\n",
      "--- total time for blast Chr7-6Mb-seg-784: 4.994s.\n",
      "-- processing region: Chr7-6Mb-seg-785 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-785.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-785\n",
      "--- total time for blast Chr7-6Mb-seg-785: 15.644s.\n",
      "-- processing region: Chr7-6Mb-seg-786 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-786.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-786\n",
      "--- total time for blast Chr7-6Mb-seg-786: 3.098s.\n",
      "-- processing region: Chr7-6Mb-seg-787 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-787.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-787\n",
      "--- total time for blast Chr7-6Mb-seg-787: 3.204s.\n",
      "-- processing region: Chr7-6Mb-seg-788 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-788.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-788\n",
      "--- total time for blast Chr7-6Mb-seg-788: 3.359s.\n",
      "-- processing region: Chr7-6Mb-seg-79 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-79.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-79\n",
      "--- total time for blast Chr7-6Mb-seg-79: 3.664s.\n",
      "-- processing region: Chr7-6Mb-seg-790 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-790.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-790\n",
      "--- total time for blast Chr7-6Mb-seg-790: 11.477s.\n",
      "-- processing region: Chr7-6Mb-seg-791 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-791.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-791\n",
      "--- total time for blast Chr7-6Mb-seg-791: 3.886s.\n",
      "-- processing region: Chr7-6Mb-seg-792 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-792.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-792\n",
      "--- total time for blast Chr7-6Mb-seg-792: 5.087s.\n",
      "-- processing region: Chr7-6Mb-seg-793 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-793.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-793\n",
      "--- total time for blast Chr7-6Mb-seg-793: 3.764s.\n",
      "-- processing region: Chr7-6Mb-seg-794 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-794.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-794\n",
      "--- total time for blast Chr7-6Mb-seg-794: 3.523s.\n",
      "-- processing region: Chr7-6Mb-seg-795 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-795.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-795\n",
      "--- total time for blast Chr7-6Mb-seg-795: 9.280s.\n",
      "-- processing region: Chr7-6Mb-seg-796 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-796.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-796\n",
      "--- total time for blast Chr7-6Mb-seg-796: 10.627s.\n",
      "-- processing region: Chr7-6Mb-seg-797 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-797.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-797\n",
      "--- total time for blast Chr7-6Mb-seg-797: 4.848s.\n",
      "-- processing region: Chr7-6Mb-seg-798 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-798.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-798\n",
      "--- total time for blast Chr7-6Mb-seg-798: 5.437s.\n",
      "-- processing region: Chr7-6Mb-seg-799 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-799.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-799\n",
      "--- total time for blast Chr7-6Mb-seg-799: 3.673s.\n",
      "-- processing region: Chr7-6Mb-seg-80 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-80.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-80\n",
      "--- total time for blast Chr7-6Mb-seg-80: 3.978s.\n",
      "-- processing region: Chr7-6Mb-seg-800 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-800.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-800\n",
      "--- total time for blast Chr7-6Mb-seg-800: 2.199s.\n",
      "-- processing region: Chr7-6Mb-seg-801 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-801.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-801\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-801: 1.898s.\n",
      "-- processing region: Chr7-6Mb-seg-802 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-802.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-802\n",
      "--- total time for blast Chr7-6Mb-seg-802: 9.390s.\n",
      "-- processing region: Chr7-6Mb-seg-804 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-804.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-804\n",
      "--- total time for blast Chr7-6Mb-seg-804: 4.348s.\n",
      "-- processing region: Chr7-6Mb-seg-805 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-805.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-805\n",
      "--- total time for blast Chr7-6Mb-seg-805: 3.595s.\n",
      "-- processing region: Chr7-6Mb-seg-806 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-806.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-806\n",
      "--- total time for blast Chr7-6Mb-seg-806: 3.461s.\n",
      "-- processing region: Chr7-6Mb-seg-807 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-807.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-807\n",
      "--- total time for blast Chr7-6Mb-seg-807: 6.426s.\n",
      "-- processing region: Chr7-6Mb-seg-808 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-808.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-808\n",
      "--- total time for blast Chr7-6Mb-seg-808: 3.636s.\n",
      "-- processing region: Chr7-6Mb-seg-809 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-809.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-809\n",
      "--- total time for blast Chr7-6Mb-seg-809: 4.025s.\n",
      "-- processing region: Chr7-6Mb-seg-81 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-81.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-81\n",
      "--- total time for blast Chr7-6Mb-seg-81: 4.281s.\n",
      "-- processing region: Chr7-6Mb-seg-810 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-810.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-810\n",
      "--- total time for blast Chr7-6Mb-seg-810: 4.134s.\n",
      "-- processing region: Chr7-6Mb-seg-812 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-812.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-812\n",
      "--- total time for blast Chr7-6Mb-seg-812: 9.315s.\n",
      "-- processing region: Chr7-6Mb-seg-813 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-813.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-813\n",
      "--- total time for blast Chr7-6Mb-seg-813: 3.430s.\n",
      "-- processing region: Chr7-6Mb-seg-814 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-814.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-814\n",
      "--- total time for blast Chr7-6Mb-seg-814: 10.301s.\n",
      "-- processing region: Chr7-6Mb-seg-816 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-816.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-816\n",
      "--- total time for blast Chr7-6Mb-seg-816: 3.809s.\n",
      "-- processing region: Chr7-6Mb-seg-817 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-817.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-817\n",
      "--- total time for blast Chr7-6Mb-seg-817: 4.178s.\n",
      "-- processing region: Chr7-6Mb-seg-818 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-818.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-818\n",
      "--- total time for blast Chr7-6Mb-seg-818: 6.947s.\n",
      "-- processing region: Chr7-6Mb-seg-819 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-819.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-819\n",
      "--- total time for blast Chr7-6Mb-seg-819: 3.871s.\n",
      "-- processing region: Chr7-6Mb-seg-82 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-82.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-82\n",
      "--- total time for blast Chr7-6Mb-seg-82: 4.260s.\n",
      "-- processing region: Chr7-6Mb-seg-820 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-820.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-820\n",
      "--- total time for blast Chr7-6Mb-seg-820: 3.466s.\n",
      "-- processing region: Chr7-6Mb-seg-821 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-821.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-821\n",
      "--- total time for blast Chr7-6Mb-seg-821: 2.066s.\n",
      "-- processing region: Chr7-6Mb-seg-822 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-822.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-822\n",
      "--- total time for blast Chr7-6Mb-seg-822: 2.831s.\n",
      "-- processing region: Chr7-6Mb-seg-823 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-823.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-823\n",
      "--- total time for blast Chr7-6Mb-seg-823: 2.838s.\n",
      "-- processing region: Chr7-6Mb-seg-824 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-824.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-824\n",
      "--- total time for blast Chr7-6Mb-seg-824: 4.593s.\n",
      "-- processing region: Chr7-6Mb-seg-825 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-825.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-825\n",
      "--- total time for blast Chr7-6Mb-seg-825: 6.551s.\n",
      "-- processing region: Chr7-6Mb-seg-828 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-828.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-828\n",
      "--- total time for blast Chr7-6Mb-seg-828: 2.718s.\n",
      "-- processing region: Chr7-6Mb-seg-829 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-829.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-829\n",
      "--- total time for blast Chr7-6Mb-seg-829: 7.598s.\n",
      "-- processing region: Chr7-6Mb-seg-83 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-83.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-83\n",
      "--- total time for blast Chr7-6Mb-seg-83: 3.895s.\n",
      "-- processing region: Chr7-6Mb-seg-831 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-831.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-831\n",
      "--- total time for blast Chr7-6Mb-seg-831: 5.809s.\n",
      "-- processing region: Chr7-6Mb-seg-832 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-832.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-832\n",
      "--- total time for blast Chr7-6Mb-seg-832: 2.934s.\n",
      "-- processing region: Chr7-6Mb-seg-833 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-833.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-833\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-833: 3.758s.\n",
      "-- processing region: Chr7-6Mb-seg-834 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-834.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-834\n",
      "--- total time for blast Chr7-6Mb-seg-834: 4.764s.\n",
      "-- processing region: Chr7-6Mb-seg-835 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-835.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-835\n",
      "--- total time for blast Chr7-6Mb-seg-835: 3.436s.\n",
      "-- processing region: Chr7-6Mb-seg-836 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-836.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-836\n",
      "--- total time for blast Chr7-6Mb-seg-836: 5.611s.\n",
      "-- processing region: Chr7-6Mb-seg-837 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-837.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-837\n",
      "--- total time for blast Chr7-6Mb-seg-837: 4.641s.\n",
      "-- processing region: Chr7-6Mb-seg-84 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-84.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-84\n",
      "--- total time for blast Chr7-6Mb-seg-84: 3.543s.\n",
      "-- processing region: Chr7-6Mb-seg-840 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-840.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-840\n",
      "--- total time for blast Chr7-6Mb-seg-840: 4.359s.\n",
      "-- processing region: Chr7-6Mb-seg-841 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-841.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-841\n",
      "--- total time for blast Chr7-6Mb-seg-841: 4.014s.\n",
      "-- processing region: Chr7-6Mb-seg-842 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-842.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-842\n",
      "--- total time for blast Chr7-6Mb-seg-842: 9.300s.\n",
      "-- processing region: Chr7-6Mb-seg-845 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-845.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-845\n",
      "--- total time for blast Chr7-6Mb-seg-845: 3.571s.\n",
      "-- processing region: Chr7-6Mb-seg-846 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-846.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-846\n",
      "--- total time for blast Chr7-6Mb-seg-846: 4.424s.\n",
      "-- processing region: Chr7-6Mb-seg-847 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-847.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-847\n",
      "--- total time for blast Chr7-6Mb-seg-847: 9.809s.\n",
      "-- processing region: Chr7-6Mb-seg-848 with 31 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-848.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-848\n",
      "--- total time for blast Chr7-6Mb-seg-848: 4.699s.\n",
      "-- processing region: Chr7-6Mb-seg-849 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-849.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-849\n",
      "--- total time for blast Chr7-6Mb-seg-849: 5.921s.\n",
      "-- processing region: Chr7-6Mb-seg-85 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-85.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-85\n",
      "--- total time for blast Chr7-6Mb-seg-85: 7.126s.\n",
      "-- processing region: Chr7-6Mb-seg-850 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-850.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-850\n",
      "--- total time for blast Chr7-6Mb-seg-850: 7.619s.\n",
      "-- processing region: Chr7-6Mb-seg-852 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-852.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-852\n",
      "--- total time for blast Chr7-6Mb-seg-852: 9.469s.\n",
      "-- processing region: Chr7-6Mb-seg-853 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-853.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-853\n",
      "--- total time for blast Chr7-6Mb-seg-853: 4.768s.\n",
      "-- processing region: Chr7-6Mb-seg-854 with 33 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-854.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-854\n",
      "--- total time for blast Chr7-6Mb-seg-854: 2.426s.\n",
      "-- processing region: Chr7-6Mb-seg-855 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-855.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-855\n",
      "--- total time for blast Chr7-6Mb-seg-855: 8.172s.\n",
      "-- processing region: Chr7-6Mb-seg-856 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-856.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-856\n",
      "--- total time for blast Chr7-6Mb-seg-856: 4.372s.\n",
      "-- processing region: Chr7-6Mb-seg-857 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-857.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-857\n",
      "--- total time for blast Chr7-6Mb-seg-857: 11.826s.\n",
      "-- processing region: Chr7-6Mb-seg-858 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-858.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-858\n",
      "--- total time for blast Chr7-6Mb-seg-858: 7.494s.\n",
      "-- processing region: Chr7-6Mb-seg-859 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-859.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-859\n",
      "--- total time for blast Chr7-6Mb-seg-859: 2.869s.\n",
      "-- processing region: Chr7-6Mb-seg-86 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-86.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-86\n",
      "--- total time for blast Chr7-6Mb-seg-86: 4.040s.\n",
      "-- processing region: Chr7-6Mb-seg-860 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-860.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-860\n",
      "--- total time for blast Chr7-6Mb-seg-860: 8.315s.\n",
      "-- processing region: Chr7-6Mb-seg-862 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-862.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-862\n",
      "--- total time for blast Chr7-6Mb-seg-862: 8.051s.\n",
      "-- processing region: Chr7-6Mb-seg-863 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-863.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-863\n",
      "--- total time for blast Chr7-6Mb-seg-863: 4.131s.\n",
      "-- processing region: Chr7-6Mb-seg-864 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-864.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-864\n",
      "--- total time for blast Chr7-6Mb-seg-864: 12.861s.\n",
      "-- processing region: Chr7-6Mb-seg-865 with 33 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-865.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-865\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-865: 4.986s.\n",
      "-- processing region: Chr7-6Mb-seg-866 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-866.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-866\n",
      "--- total time for blast Chr7-6Mb-seg-866: 13.463s.\n",
      "-- processing region: Chr7-6Mb-seg-868 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-868.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-868\n",
      "--- total time for blast Chr7-6Mb-seg-868: 10.320s.\n",
      "-- processing region: Chr7-6Mb-seg-869 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-869.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-869\n",
      "--- total time for blast Chr7-6Mb-seg-869: 8.109s.\n",
      "-- processing region: Chr7-6Mb-seg-87 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-87.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-87\n",
      "--- total time for blast Chr7-6Mb-seg-87: 4.789s.\n",
      "-- processing region: Chr7-6Mb-seg-870 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-870.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-870\n",
      "--- total time for blast Chr7-6Mb-seg-870: 6.415s.\n",
      "-- processing region: Chr7-6Mb-seg-873 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-873.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-873\n",
      "--- total time for blast Chr7-6Mb-seg-873: 3.992s.\n",
      "-- processing region: Chr7-6Mb-seg-876 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-876.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-876\n",
      "--- total time for blast Chr7-6Mb-seg-876: 9.651s.\n",
      "-- processing region: Chr7-6Mb-seg-878 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-878.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-878\n",
      "--- total time for blast Chr7-6Mb-seg-878: 3.824s.\n",
      "-- processing region: Chr7-6Mb-seg-879 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-879.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-879\n",
      "--- total time for blast Chr7-6Mb-seg-879: 12.249s.\n",
      "-- processing region: Chr7-6Mb-seg-88 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-88.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-88\n",
      "--- total time for blast Chr7-6Mb-seg-88: 4.357s.\n",
      "-- processing region: Chr7-6Mb-seg-880 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-880.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-880\n",
      "--- total time for blast Chr7-6Mb-seg-880: 7.019s.\n",
      "-- processing region: Chr7-6Mb-seg-881 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-881.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-881\n",
      "--- total time for blast Chr7-6Mb-seg-881: 2.775s.\n",
      "-- processing region: Chr7-6Mb-seg-883 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-883.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-883\n",
      "--- total time for blast Chr7-6Mb-seg-883: 4.690s.\n",
      "-- processing region: Chr7-6Mb-seg-885 with 33 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-885.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-885\n",
      "--- total time for blast Chr7-6Mb-seg-885: 3.503s.\n",
      "-- processing region: Chr7-6Mb-seg-888 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-888.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-888\n",
      "--- total time for blast Chr7-6Mb-seg-888: 7.955s.\n",
      "-- processing region: Chr7-6Mb-seg-889 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-889.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-889\n",
      "--- total time for blast Chr7-6Mb-seg-889: 7.679s.\n",
      "-- processing region: Chr7-6Mb-seg-89 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-89.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-89\n",
      "--- total time for blast Chr7-6Mb-seg-89: 4.988s.\n",
      "-- processing region: Chr7-6Mb-seg-890 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-890.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-890\n",
      "--- total time for blast Chr7-6Mb-seg-890: 3.917s.\n",
      "-- processing region: Chr7-6Mb-seg-891 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-891.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-891\n",
      "--- total time for blast Chr7-6Mb-seg-891: 10.362s.\n",
      "-- processing region: Chr7-6Mb-seg-892 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-892.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-892\n",
      "--- total time for blast Chr7-6Mb-seg-892: 4.211s.\n",
      "-- processing region: Chr7-6Mb-seg-893 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-893.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-893\n",
      "--- total time for blast Chr7-6Mb-seg-893: 4.581s.\n",
      "-- processing region: Chr7-6Mb-seg-894 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-894.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-894\n",
      "--- total time for blast Chr7-6Mb-seg-894: 4.998s.\n",
      "-- processing region: Chr7-6Mb-seg-895 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-895.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-895\n",
      "--- total time for blast Chr7-6Mb-seg-895: 6.461s.\n",
      "-- processing region: Chr7-6Mb-seg-896 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-896.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-896\n",
      "--- total time for blast Chr7-6Mb-seg-896: 6.838s.\n",
      "-- processing region: Chr7-6Mb-seg-897 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-897.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-897\n",
      "--- total time for blast Chr7-6Mb-seg-897: 3.501s.\n",
      "-- processing region: Chr7-6Mb-seg-9 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-9.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-9\n",
      "--- total time for blast Chr7-6Mb-seg-9: 4.767s.\n",
      "-- processing region: Chr7-6Mb-seg-90 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-90.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-90\n",
      "--- total time for blast Chr7-6Mb-seg-90: 4.677s.\n",
      "-- processing region: Chr7-6Mb-seg-900 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-900.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-900\n",
      "--- total time for blast Chr7-6Mb-seg-900: 8.818s.\n",
      "-- processing region: Chr7-6Mb-seg-901 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-901.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-901\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-901: 6.501s.\n",
      "-- processing region: Chr7-6Mb-seg-902 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-902.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-902\n",
      "--- total time for blast Chr7-6Mb-seg-902: 4.115s.\n",
      "-- processing region: Chr7-6Mb-seg-904 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-904.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-904\n",
      "--- total time for blast Chr7-6Mb-seg-904: 12.593s.\n",
      "-- processing region: Chr7-6Mb-seg-906 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-906.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-906\n",
      "--- total time for blast Chr7-6Mb-seg-906: 8.302s.\n",
      "-- processing region: Chr7-6Mb-seg-907 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-907.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-907\n",
      "--- total time for blast Chr7-6Mb-seg-907: 15.670s.\n",
      "-- processing region: Chr7-6Mb-seg-91 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-91.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-91\n",
      "--- total time for blast Chr7-6Mb-seg-91: 6.074s.\n",
      "-- processing region: Chr7-6Mb-seg-911 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-911.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-911\n",
      "--- total time for blast Chr7-6Mb-seg-911: 6.072s.\n",
      "-- processing region: Chr7-6Mb-seg-912 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-912.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-912\n",
      "--- total time for blast Chr7-6Mb-seg-912: 5.416s.\n",
      "-- processing region: Chr7-6Mb-seg-913 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-913.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-913\n",
      "--- total time for blast Chr7-6Mb-seg-913: 12.341s.\n",
      "-- processing region: Chr7-6Mb-seg-916 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-916.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-916\n",
      "--- total time for blast Chr7-6Mb-seg-916: 3.180s.\n",
      "-- processing region: Chr7-6Mb-seg-917 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-917.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-917\n",
      "--- total time for blast Chr7-6Mb-seg-917: 4.670s.\n",
      "-- processing region: Chr7-6Mb-seg-918 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-918.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-918\n",
      "--- total time for blast Chr7-6Mb-seg-918: 5.711s.\n",
      "-- processing region: Chr7-6Mb-seg-919 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-919.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-919\n",
      "--- total time for blast Chr7-6Mb-seg-919: 2.636s.\n",
      "-- processing region: Chr7-6Mb-seg-92 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-92.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-92\n",
      "--- total time for blast Chr7-6Mb-seg-92: 4.111s.\n",
      "-- processing region: Chr7-6Mb-seg-922 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-922.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-922\n",
      "--- total time for blast Chr7-6Mb-seg-922: 4.107s.\n",
      "-- processing region: Chr7-6Mb-seg-923 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-923.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-923\n",
      "--- total time for blast Chr7-6Mb-seg-923: 9.935s.\n",
      "-- processing region: Chr7-6Mb-seg-924 with 33 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-924.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-924\n",
      "--- total time for blast Chr7-6Mb-seg-924: 4.687s.\n",
      "-- processing region: Chr7-6Mb-seg-926 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-926.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-926\n",
      "--- total time for blast Chr7-6Mb-seg-926: 4.186s.\n",
      "-- processing region: Chr7-6Mb-seg-929 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-929.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-929\n",
      "--- total time for blast Chr7-6Mb-seg-929: 4.333s.\n",
      "-- processing region: Chr7-6Mb-seg-93 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-93.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-93\n",
      "--- total time for blast Chr7-6Mb-seg-93: 3.761s.\n",
      "-- processing region: Chr7-6Mb-seg-931 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-931.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-931\n",
      "--- total time for blast Chr7-6Mb-seg-931: 6.993s.\n",
      "-- processing region: Chr7-6Mb-seg-932 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-932.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-932\n",
      "--- total time for blast Chr7-6Mb-seg-932: 4.597s.\n",
      "-- processing region: Chr7-6Mb-seg-933 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-933.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-933\n",
      "--- total time for blast Chr7-6Mb-seg-933: 4.126s.\n",
      "-- processing region: Chr7-6Mb-seg-934 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-934.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-934\n",
      "--- total time for blast Chr7-6Mb-seg-934: 3.521s.\n",
      "-- processing region: Chr7-6Mb-seg-935 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-935.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-935\n",
      "--- total time for blast Chr7-6Mb-seg-935: 4.663s.\n",
      "-- processing region: Chr7-6Mb-seg-937 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-937.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-937\n",
      "--- total time for blast Chr7-6Mb-seg-937: 2.112s.\n",
      "-- processing region: Chr7-6Mb-seg-938 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-938.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-938\n",
      "--- total time for blast Chr7-6Mb-seg-938: 4.165s.\n",
      "-- processing region: Chr7-6Mb-seg-939 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-939.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-939\n",
      "--- total time for blast Chr7-6Mb-seg-939: 2.933s.\n",
      "-- processing region: Chr7-6Mb-seg-94 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-94.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-94\n",
      "--- total time for blast Chr7-6Mb-seg-94: 3.657s.\n",
      "-- processing region: Chr7-6Mb-seg-942 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-942.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-942\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-942: 8.437s.\n",
      "-- processing region: Chr7-6Mb-seg-944 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-944.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-944\n",
      "--- total time for blast Chr7-6Mb-seg-944: 5.028s.\n",
      "-- processing region: Chr7-6Mb-seg-945 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-945.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-945\n",
      "--- total time for blast Chr7-6Mb-seg-945: 3.232s.\n",
      "-- processing region: Chr7-6Mb-seg-946 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-946.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-946\n",
      "--- total time for blast Chr7-6Mb-seg-946: 3.522s.\n",
      "-- processing region: Chr7-6Mb-seg-947 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-947.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-947\n",
      "--- total time for blast Chr7-6Mb-seg-947: 6.036s.\n",
      "-- processing region: Chr7-6Mb-seg-95 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-95.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-95\n",
      "--- total time for blast Chr7-6Mb-seg-95: 4.118s.\n",
      "-- processing region: Chr7-6Mb-seg-950 with 31 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-950.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-950\n",
      "--- total time for blast Chr7-6Mb-seg-950: 3.971s.\n",
      "-- processing region: Chr7-6Mb-seg-951 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-951.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-951\n",
      "--- total time for blast Chr7-6Mb-seg-951: 4.473s.\n",
      "-- processing region: Chr7-6Mb-seg-952 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-952.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-952\n",
      "--- total time for blast Chr7-6Mb-seg-952: 4.151s.\n",
      "-- processing region: Chr7-6Mb-seg-953 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-953.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-953\n",
      "--- total time for blast Chr7-6Mb-seg-953: 4.523s.\n",
      "-- processing region: Chr7-6Mb-seg-954 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-954.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-954\n",
      "--- total time for blast Chr7-6Mb-seg-954: 3.720s.\n",
      "-- processing region: Chr7-6Mb-seg-955 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-955.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-955\n",
      "--- total time for blast Chr7-6Mb-seg-955: 4.794s.\n",
      "-- processing region: Chr7-6Mb-seg-956 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-956.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-956\n",
      "--- total time for blast Chr7-6Mb-seg-956: 4.240s.\n",
      "-- processing region: Chr7-6Mb-seg-957 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-957.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-957\n",
      "--- total time for blast Chr7-6Mb-seg-957: 5.383s.\n",
      "-- processing region: Chr7-6Mb-seg-958 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-958.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-958\n",
      "--- total time for blast Chr7-6Mb-seg-958: 4.409s.\n",
      "-- processing region: Chr7-6Mb-seg-959 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-959.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-959\n",
      "--- total time for blast Chr7-6Mb-seg-959: 3.857s.\n",
      "-- processing region: Chr7-6Mb-seg-96 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-96.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-96\n",
      "--- total time for blast Chr7-6Mb-seg-96: 6.304s.\n",
      "-- processing region: Chr7-6Mb-seg-960 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-960.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-960\n",
      "--- total time for blast Chr7-6Mb-seg-960: 4.711s.\n",
      "-- processing region: Chr7-6Mb-seg-961 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-961.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-961\n",
      "--- total time for blast Chr7-6Mb-seg-961: 3.668s.\n",
      "-- processing region: Chr7-6Mb-seg-962 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-962.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-962\n",
      "--- total time for blast Chr7-6Mb-seg-962: 3.939s.\n",
      "-- processing region: Chr7-6Mb-seg-963 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-963.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-963\n",
      "--- total time for blast Chr7-6Mb-seg-963: 3.922s.\n",
      "-- processing region: Chr7-6Mb-seg-964 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-964.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-964\n",
      "--- total time for blast Chr7-6Mb-seg-964: 4.760s.\n",
      "-- processing region: Chr7-6Mb-seg-965 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-965.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-965\n",
      "--- total time for blast Chr7-6Mb-seg-965: 3.819s.\n",
      "-- processing region: Chr7-6Mb-seg-966 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-966.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-966\n",
      "--- total time for blast Chr7-6Mb-seg-966: 3.827s.\n",
      "-- processing region: Chr7-6Mb-seg-967 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-967.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-967\n",
      "--- total time for blast Chr7-6Mb-seg-967: 4.769s.\n",
      "-- processing region: Chr7-6Mb-seg-968 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-968.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-968\n",
      "--- total time for blast Chr7-6Mb-seg-968: 4.232s.\n",
      "-- processing region: Chr7-6Mb-seg-969 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-969.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-969\n",
      "--- total time for blast Chr7-6Mb-seg-969: 4.331s.\n",
      "-- processing region: Chr7-6Mb-seg-97 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-97.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-97\n",
      "--- total time for blast Chr7-6Mb-seg-97: 3.928s.\n",
      "-- processing region: Chr7-6Mb-seg-970 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-970.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-970\n",
      "--- total time for blast Chr7-6Mb-seg-970: 4.277s.\n",
      "-- processing region: Chr7-6Mb-seg-971 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-971.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-971\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-971: 4.344s.\n",
      "-- processing region: Chr7-6Mb-seg-972 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-972.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-972\n",
      "--- total time for blast Chr7-6Mb-seg-972: 3.910s.\n",
      "-- processing region: Chr7-6Mb-seg-973 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-973.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-973\n",
      "--- total time for blast Chr7-6Mb-seg-973: 5.034s.\n",
      "-- processing region: Chr7-6Mb-seg-974 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-974.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-974\n",
      "--- total time for blast Chr7-6Mb-seg-974: 4.796s.\n",
      "-- processing region: Chr7-6Mb-seg-975 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-975.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-975\n",
      "--- total time for blast Chr7-6Mb-seg-975: 3.806s.\n",
      "-- processing region: Chr7-6Mb-seg-976 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-976.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-976\n",
      "--- total time for blast Chr7-6Mb-seg-976: 3.813s.\n",
      "-- processing region: Chr7-6Mb-seg-977 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-977.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-977\n",
      "--- total time for blast Chr7-6Mb-seg-977: 3.517s.\n",
      "-- processing region: Chr7-6Mb-seg-978 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-978.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-978\n",
      "--- total time for blast Chr7-6Mb-seg-978: 3.963s.\n",
      "-- processing region: Chr7-6Mb-seg-979 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-979.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-979\n",
      "--- total time for blast Chr7-6Mb-seg-979: 3.859s.\n",
      "-- processing region: Chr7-6Mb-seg-98 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-98.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-98\n",
      "--- total time for blast Chr7-6Mb-seg-98: 4.547s.\n",
      "-- processing region: Chr7-6Mb-seg-980 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-980.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-980\n",
      "--- total time for blast Chr7-6Mb-seg-980: 4.135s.\n",
      "-- processing region: Chr7-6Mb-seg-981 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-981.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-981\n",
      "--- total time for blast Chr7-6Mb-seg-981: 3.643s.\n",
      "-- processing region: Chr7-6Mb-seg-982 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-982.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-982\n",
      "--- total time for blast Chr7-6Mb-seg-982: 4.565s.\n",
      "-- processing region: Chr7-6Mb-seg-983 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-983.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-983\n",
      "--- total time for blast Chr7-6Mb-seg-983: 4.040s.\n",
      "-- processing region: Chr7-6Mb-seg-984 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-984.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-984\n",
      "--- total time for blast Chr7-6Mb-seg-984: 4.205s.\n",
      "-- processing region: Chr7-6Mb-seg-985 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-985.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-985\n",
      "--- total time for blast Chr7-6Mb-seg-985: 4.559s.\n",
      "-- processing region: Chr7-6Mb-seg-986 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-986.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-986\n",
      "--- total time for blast Chr7-6Mb-seg-986: 5.450s.\n",
      "-- processing region: Chr7-6Mb-seg-987 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-987.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-987\n",
      "--- total time for blast Chr7-6Mb-seg-987: 4.110s.\n",
      "-- processing region: Chr7-6Mb-seg-988 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-988.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-988\n",
      "--- total time for blast Chr7-6Mb-seg-988: 3.986s.\n",
      "-- processing region: Chr7-6Mb-seg-989 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-989.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-989\n",
      "--- total time for blast Chr7-6Mb-seg-989: 3.708s.\n",
      "-- processing region: Chr7-6Mb-seg-99 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-99.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-99\n",
      "--- total time for blast Chr7-6Mb-seg-99: 4.946s.\n",
      "-- processing region: Chr7-6Mb-seg-990 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-990.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-990\n",
      "--- total time for blast Chr7-6Mb-seg-990: 4.045s.\n",
      "-- processing region: Chr7-6Mb-seg-991 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-991.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-991\n",
      "--- total time for blast Chr7-6Mb-seg-991: 3.877s.\n",
      "-- processing region: Chr7-6Mb-seg-992 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-992.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-992\n",
      "--- total time for blast Chr7-6Mb-seg-992: 3.598s.\n",
      "-- processing region: Chr7-6Mb-seg-993 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-993.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-993\n",
      "--- total time for blast Chr7-6Mb-seg-993: 3.725s.\n",
      "-- processing region: Chr7-6Mb-seg-994 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-994.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-994\n",
      "--- total time for blast Chr7-6Mb-seg-994: 3.756s.\n",
      "-- processing region: Chr7-6Mb-seg-995 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-995.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-995\n",
      "--- total time for blast Chr7-6Mb-seg-995: 3.607s.\n",
      "-- processing region: Chr7-6Mb-seg-996 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-996.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-996\n",
      "--- total time for blast Chr7-6Mb-seg-996: 4.462s.\n",
      "-- processing region: Chr7-6Mb-seg-997 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-997.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-997\n",
      "--- total time for blast Chr7-6Mb-seg-997: 3.962s.\n",
      "-- processing region: Chr7-6Mb-seg-998 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-998.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-998\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-seg-998: 3.306s.\n",
      "-- processing region: Chr7-6Mb-seg-999 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast\\probe_gene_Chr7-6Mb-seg-999.fasta\n",
      "--- *blasting region: Chr7-6Mb-seg-999\n",
      "--- total time for blast Chr7-6Mb-seg-999: 4.393s.\n"
     ]
    }
   ],
   "source": [
    "blast_database = os.path.join(reference_folder, 'blast_database', 'GRCm38_ensembl')\n",
    "\n",
    "check.Blast_probes(kept_records, library_folder, num_threads=44, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7c6a9506",
   "metadata": {},
   "source": [
    "### 4.2.1 screen probe by blast results\n",
    "\n",
    "This step has 2 different cutoffs:\n",
    "\n",
    "hard_cutoff: whether a probe has any ~40mer hit somewhere else, if so just dump the probe\n",
    "\n",
    "soft_cutoff: whether a probe has a lot of ~20mer off-target hits, if exceed certain amount then dump the probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 239,
   "id": "6f6749b3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 70719\n",
      "- Number of regions in this library: 1097\n",
      "-- checking probes in region: Chr7-6Mb-seg-1\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-2\n",
      "--- number of probes: 60 , kept by blast: 60 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-3\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-4\n",
      "--- number of probes: 51 , kept by blast: 51 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: Chr7-6Mb-seg-7\n",
      "--- gene=Chr7-6Mb-seg-7, id=0 removed by soft count = 679\n",
      "--- gene=Chr7-6Mb-seg-7, id=1 removed by soft count = 337\n",
      "--- gene=Chr7-6Mb-seg-7, id=8 removed by soft count = 96\n",
      "--- gene=Chr7-6Mb-seg-7, id=9 removed by soft count = 193\n",
      "--- gene=Chr7-6Mb-seg-7, id=11 removed by soft count = 223\n",
      "--- gene=Chr7-6Mb-seg-7, id=13 removed by soft count = 127\n",
      "--- gene=Chr7-6Mb-seg-7, id=15 removed by soft count = 431\n",
      "--- gene=Chr7-6Mb-seg-7, id=16 removed by soft count = 136\n",
      "--- gene=Chr7-6Mb-seg-7, id=19 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-7, id=21 removed by soft count = 90\n",
      "--- gene=Chr7-6Mb-seg-7, id=25 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-7, id=27 removed by soft count = 683\n",
      "--- gene=Chr7-6Mb-seg-7, id=28 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-7, id=29 removed by soft count = 212\n",
      "--- number of probes: 30 , kept by blast: 16 , if remove dups: 16\n",
      "-- number of probes kept for this region: 16\n",
      "-- checking probes in region: Chr7-6Mb-seg-9\n",
      "--- gene=Chr7-6Mb-seg-9, id=0 removed by soft count = 301\n",
      "--- gene=Chr7-6Mb-seg-9, id=1 removed by soft count = 1942\n",
      "--- number of probes: 60 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-10\n",
      "--- gene=Chr7-6Mb-seg-10, id=11 removed by hard count = 3\n",
      "--- number of probes: 41 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: Chr7-6Mb-seg-11\n",
      "--- number of probes: 44 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-12\n",
      "--- gene=Chr7-6Mb-seg-12, id=30 removed by soft count = 89\n",
      "--- gene=Chr7-6Mb-seg-12, id=33 removed by soft count = 872\n",
      "--- gene=Chr7-6Mb-seg-12, id=34 removed by soft count = 486\n",
      "--- gene=Chr7-6Mb-seg-12, id=35 removed by soft count = 294\n",
      "--- gene=Chr7-6Mb-seg-12, id=36 removed by soft count = 168\n",
      "--- gene=Chr7-6Mb-seg-12, id=37 removed by soft count = 156\n",
      "--- number of probes: 40 , kept by blast: 34 , if remove dups: 32\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-14\n",
      "--- gene=Chr7-6Mb-seg-14, id=0 removed by soft count = 97\n",
      "--- gene=Chr7-6Mb-seg-14, id=3 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-14, id=11 removed by soft count = 81\n",
      "--- gene=Chr7-6Mb-seg-14, id=13 removed by soft count = 170\n",
      "--- gene=Chr7-6Mb-seg-14, id=14 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-14, id=15 removed by soft count = 287\n",
      "--- number of probes: 26 , kept by blast: 20 , if remove dups: 19\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-seg-15\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-16\n",
      "--- gene=Chr7-6Mb-seg-16, id=37 removed by soft count = 253\n",
      "--- gene=Chr7-6Mb-seg-16, id=55 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-16, id=57 removed by soft count = 535\n",
      "--- number of probes: 58 , kept by blast: 55 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-19\n",
      "--- gene=Chr7-6Mb-seg-19, id=14 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-19, id=15 removed by soft count = 113\n",
      "--- gene=Chr7-6Mb-seg-19, id=16 removed by soft count = 233\n",
      "--- gene=Chr7-6Mb-seg-19, id=17 removed by soft count = 544\n",
      "--- gene=Chr7-6Mb-seg-19, id=19 removed by soft count = 4628\n",
      "--- number of probes: 40 , kept by blast: 35 , if remove dups: 32\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: Chr7-6Mb-seg-20\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-21\n",
      "--- number of probes: 40 , kept by blast: 40 , if remove dups: 39\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: Chr7-6Mb-seg-22\n",
      "--- gene=Chr7-6Mb-seg-22, id=13 removed by hard count = 3\n",
      "--- gene=Chr7-6Mb-seg-22, id=32 removed by soft count = 95\n",
      "--- number of probes: 50 , kept by blast: 48 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-23\n",
      "--- number of probes: 42 , kept by blast: 42 , if remove dups: 41\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: Chr7-6Mb-seg-24\n",
      "--- number of probes: 51 , kept by blast: 51 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: Chr7-6Mb-seg-25\n",
      "--- gene=Chr7-6Mb-seg-25, id=0 removed by soft count = 113\n",
      "--- gene=Chr7-6Mb-seg-25, id=1 removed by soft count = 90\n",
      "--- gene=Chr7-6Mb-seg-25, id=43 removed by soft count = 332\n",
      "--- gene=Chr7-6Mb-seg-25, id=44 removed by soft count = 789\n",
      "--- gene=Chr7-6Mb-seg-25, id=45 removed by soft count = 147\n",
      "--- number of probes: 46 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: Chr7-6Mb-seg-27\n",
      "--- gene=Chr7-6Mb-seg-27, id=1 removed by soft count = 159\n",
      "--- gene=Chr7-6Mb-seg-27, id=3 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-27, id=4 removed by soft count = 183\n",
      "--- gene=Chr7-6Mb-seg-27, id=5 removed by soft count = 230\n",
      "--- gene=Chr7-6Mb-seg-27, id=6 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-27, id=7 removed by soft count = 81\n",
      "--- number of probes: 47 , kept by blast: 41 , if remove dups: 40\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: Chr7-6Mb-seg-28\n",
      "--- gene=Chr7-6Mb-seg-28, id=19 removed by soft count = 154\n",
      "--- number of probes: 60 , kept by blast: 59 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Chr7-6Mb-seg-29\n",
      "--- gene=Chr7-6Mb-seg-29, id=7 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-29, id=30 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-29, id=31 removed by soft count = 85\n",
      "--- gene=Chr7-6Mb-seg-29, id=36 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-29, id=39 removed by soft count = 53\n",
      "--- number of probes: 40 , kept by blast: 35 , if remove dups: 34\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: Chr7-6Mb-seg-36\n",
      "--- gene=Chr7-6Mb-seg-36, id=0 removed by soft count = 2256\n",
      "--- number of probes: 30 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: Chr7-6Mb-seg-38\n",
      "--- gene=Chr7-6Mb-seg-38, id=0 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-38, id=1 removed by soft count = 185\n",
      "--- gene=Chr7-6Mb-seg-38, id=2 removed by soft count = 251\n",
      "--- gene=Chr7-6Mb-seg-38, id=3 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-38, id=4 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-38, id=6 removed by soft count = 238\n",
      "--- gene=Chr7-6Mb-seg-38, id=9 removed by soft count = 215\n",
      "--- gene=Chr7-6Mb-seg-38, id=10 removed by soft count = 279\n",
      "--- number of probes: 39 , kept by blast: 31 , if remove dups: 28\n",
      "-- number of probes kept for this region: 31\n",
      "-- checking probes in region: Chr7-6Mb-seg-39\n",
      "--- gene=Chr7-6Mb-seg-39, id=35 removed by hard count = 3\n",
      "--- number of probes: 38 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: Chr7-6Mb-seg-40\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-40, id=8 removed by soft count = 1185\n",
      "--- gene=Chr7-6Mb-seg-40, id=9 removed by soft count = 213\n",
      "--- gene=Chr7-6Mb-seg-40, id=41 removed by soft count = 62\n",
      "--- number of probes: 51 , kept by blast: 48 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-41\n",
      "--- gene=Chr7-6Mb-seg-41, id=50 removed by soft count = 124\n",
      "--- gene=Chr7-6Mb-seg-41, id=53 removed by soft count = 185\n",
      "--- gene=Chr7-6Mb-seg-41, id=59 removed by soft count = 82\n",
      "--- number of probes: 60 , kept by blast: 57 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-42\n",
      "--- number of probes: 56 , kept by blast: 56 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-43\n",
      "--- number of probes: 63 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-44\n",
      "--- gene=Chr7-6Mb-seg-44, id=1 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-44, id=4 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-44, id=7 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-44, id=10 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-44, id=13 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-44, id=16 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-44, id=22 removed by soft count = 155\n",
      "--- gene=Chr7-6Mb-seg-44, id=25 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-44, id=28 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-44, id=34 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-44, id=37 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-44, id=40 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-44, id=43 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-44, id=46 removed by soft count = 58\n",
      "--- number of probes: 49 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: Chr7-6Mb-seg-45\n",
      "--- gene=Chr7-6Mb-seg-45, id=48 removed by soft count = 241\n",
      "--- number of probes: 50 , kept by blast: 49 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-46\n",
      "--- number of probes: 60 , kept by blast: 60 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-47\n",
      "--- gene=Chr7-6Mb-seg-47, id=52 removed by soft count = 761\n",
      "--- gene=Chr7-6Mb-seg-47, id=53 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-47, id=56 removed by soft count = 289\n",
      "--- number of probes: 58 , kept by blast: 55 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-48\n",
      "--- gene=Chr7-6Mb-seg-48, id=1 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-48, id=47 removed by soft count = 81\n",
      "--- number of probes: 71 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-49\n",
      "--- gene=Chr7-6Mb-seg-49, id=30 removed by soft count = 849\n",
      "--- gene=Chr7-6Mb-seg-49, id=48 removed by soft count = 85\n",
      "--- gene=Chr7-6Mb-seg-49, id=56 removed by soft count = 608\n",
      "--- gene=Chr7-6Mb-seg-49, id=58 removed by soft count = 119\n",
      "--- number of probes: 62 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-50\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-51\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-52\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-53\n",
      "--- gene=Chr7-6Mb-seg-53, id=63 removed by soft count = 900\n",
      "--- gene=Chr7-6Mb-seg-53, id=71 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-53, id=72 removed by soft count = 51\n",
      "--- number of probes: 74 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-54\n",
      "--- gene=Chr7-6Mb-seg-54, id=67 removed by soft count = 69\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-55\n",
      "--- gene=Chr7-6Mb-seg-55, id=0 removed by soft count = 238\n",
      "--- gene=Chr7-6Mb-seg-55, id=8 removed by soft count = 99\n",
      "--- gene=Chr7-6Mb-seg-55, id=42 removed by soft count = 91\n",
      "--- gene=Chr7-6Mb-seg-55, id=46 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-55, id=50 removed by soft count = 90\n",
      "--- number of probes: 70 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-56\n",
      "--- gene=Chr7-6Mb-seg-56, id=1 removed by soft count = 169\n",
      "--- gene=Chr7-6Mb-seg-56, id=2 removed by soft count = 142\n",
      "--- gene=Chr7-6Mb-seg-56, id=3 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-56, id=38 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-seg-56, id=58 removed by soft count = 416\n",
      "--- number of probes: 68 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-57\n",
      "--- gene=Chr7-6Mb-seg-57, id=8 removed by soft count = 202\n",
      "--- number of probes: 68 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-58\n",
      "--- gene=Chr7-6Mb-seg-58, id=17 removed by soft count = 196\n",
      "--- gene=Chr7-6Mb-seg-58, id=53 removed by soft count = 118\n",
      "--- gene=Chr7-6Mb-seg-58, id=56 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-58, id=60 removed by soft count = 70\n",
      "--- number of probes: 62 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-59\n",
      "--- gene=Chr7-6Mb-seg-59, id=7 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-59, id=11 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-59, id=17 removed by soft count = 86\n",
      "--- gene=Chr7-6Mb-seg-59, id=18 removed by soft count = 989\n",
      "--- gene=Chr7-6Mb-seg-59, id=19 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-59, id=43 removed by soft count = 188\n",
      "--- gene=Chr7-6Mb-seg-59, id=44 removed by soft count = 71\n",
      "--- number of probes: 58 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-60\n",
      "--- gene=Chr7-6Mb-seg-60, id=4 removed by soft count = 809\n",
      "--- gene=Chr7-6Mb-seg-60, id=19 removed by soft count = 442\n",
      "--- gene=Chr7-6Mb-seg-60, id=20 removed by soft count = 127\n",
      "--- gene=Chr7-6Mb-seg-60, id=22 removed by soft count = 179\n",
      "--- gene=Chr7-6Mb-seg-60, id=26 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-60, id=42 removed by soft count = 204\n",
      "--- number of probes: 48 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: Chr7-6Mb-seg-61\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-62\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-63\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-64\n",
      "--- gene=Chr7-6Mb-seg-64, id=4 removed by soft count = 3271\n",
      "--- gene=Chr7-6Mb-seg-64, id=6 removed by soft count = 153\n",
      "--- gene=Chr7-6Mb-seg-64, id=74 removed by soft count = 154\n",
      "--- number of probes: 75 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-65\n",
      "--- gene=Chr7-6Mb-seg-65, id=2 removed by soft count = 318\n",
      "--- gene=Chr7-6Mb-seg-65, id=31 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-65, id=36 removed by soft count = 77\n",
      "--- number of probes: 80 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-66\n",
      "--- gene=Chr7-6Mb-seg-66, id=35 removed by hard count = 0\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-67\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-68\n",
      "--- gene=Chr7-6Mb-seg-68, id=62 removed by soft count = 2567\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-69\n",
      "--- gene=Chr7-6Mb-seg-69, id=16 removed by soft count = 262\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-70\n",
      "--- gene=Chr7-6Mb-seg-70, id=23 removed by soft count = 94\n",
      "--- number of probes: 68 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-71\n",
      "--- gene=Chr7-6Mb-seg-71, id=0 removed by soft count = 127\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-72\n",
      "--- gene=Chr7-6Mb-seg-72, id=11 removed by soft count = 2497\n",
      "--- gene=Chr7-6Mb-seg-72, id=71 removed by hard count = 0\n",
      "--- number of probes: 74 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-73\n",
      "--- gene=Chr7-6Mb-seg-73, id=67 removed by soft count = 138\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-74\n",
      "--- gene=Chr7-6Mb-seg-74, id=4 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-74, id=10 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-74, id=12 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-74, id=19 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-74, id=34 removed by soft count = 172\n",
      "--- gene=Chr7-6Mb-seg-74, id=47 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-74, id=49 removed by soft count = 150\n",
      "--- gene=Chr7-6Mb-seg-74, id=50 removed by soft count = 129\n",
      "--- gene=Chr7-6Mb-seg-74, id=52 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-74, id=55 removed by soft count = 88\n",
      "--- gene=Chr7-6Mb-seg-74, id=58 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-74, id=59 removed by soft count = 52\n",
      "--- number of probes: 64 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-75\n",
      "--- gene=Chr7-6Mb-seg-75, id=2 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-75, id=5 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-75, id=12 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-75, id=13 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-75, id=15 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-75, id=16 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-75, id=17 removed by soft count = 253\n",
      "--- gene=Chr7-6Mb-seg-75, id=20 removed by soft count = 86\n",
      "--- number of probes: 76 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-76\n",
      "--- gene=Chr7-6Mb-seg-76, id=32 removed by soft count = 134\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-77\n",
      "--- gene=Chr7-6Mb-seg-77, id=7 removed by soft count = 98\n",
      "--- gene=Chr7-6Mb-seg-77, id=8 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-77, id=10 removed by soft count = 66\n",
      "--- number of probes: 80 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-78\n",
      "--- gene=Chr7-6Mb-seg-78, id=37 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-78, id=38 removed by soft count = 103\n",
      "--- number of probes: 69 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-79\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-80\n",
      "--- gene=Chr7-6Mb-seg-80, id=42 removed by soft count = 57\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-81\n",
      "--- gene=Chr7-6Mb-seg-81, id=22 removed by soft count = 164\n",
      "--- gene=Chr7-6Mb-seg-81, id=55 removed by soft count = 92\n",
      "--- gene=Chr7-6Mb-seg-81, id=57 removed by soft count = 216\n",
      "--- number of probes: 66 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-82\n",
      "--- gene=Chr7-6Mb-seg-82, id=71 removed by soft count = 224\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-83\n",
      "--- gene=Chr7-6Mb-seg-83, id=47 removed by soft count = 97\n",
      "--- gene=Chr7-6Mb-seg-83, id=50 removed by soft count = 266\n",
      "--- gene=Chr7-6Mb-seg-83, id=51 removed by soft count = 388\n",
      "--- number of probes: 64 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-84\n",
      "--- gene=Chr7-6Mb-seg-84, id=16 removed by soft count = 130\n",
      "--- gene=Chr7-6Mb-seg-84, id=47 removed by soft count = 146\n",
      "--- gene=Chr7-6Mb-seg-84, id=48 removed by soft count = 230\n",
      "--- gene=Chr7-6Mb-seg-84, id=49 removed by soft count = 93\n",
      "--- gene=Chr7-6Mb-seg-84, id=54 removed by hard count = 7\n",
      "--- number of probes: 57 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-85\n",
      "--- gene=Chr7-6Mb-seg-85, id=55 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-85, id=73 removed by soft count = 89\n",
      "--- number of probes: 74 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-86\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-87\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-87, id=56 removed by soft count = 1903\n",
      "--- gene=Chr7-6Mb-seg-87, id=60 removed by soft count = 156\n",
      "--- gene=Chr7-6Mb-seg-87, id=61 removed by soft count = 145\n",
      "--- number of probes: 79 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-88\n",
      "--- gene=Chr7-6Mb-seg-88, id=4 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-88, id=19 removed by soft count = 138\n",
      "--- number of probes: 71 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-89\n",
      "--- gene=Chr7-6Mb-seg-89, id=17 removed by soft count = 131\n",
      "--- gene=Chr7-6Mb-seg-89, id=48 removed by soft count = 132\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-90\n",
      "--- gene=Chr7-6Mb-seg-90, id=11 removed by soft count = 125\n",
      "--- gene=Chr7-6Mb-seg-90, id=44 removed by soft count = 575\n",
      "--- number of probes: 79 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-91\n",
      "--- gene=Chr7-6Mb-seg-91, id=6 removed by soft count = 130\n",
      "--- gene=Chr7-6Mb-seg-91, id=19 removed by hard count = 0\n",
      "--- gene=Chr7-6Mb-seg-91, id=40 removed by soft count = 310\n",
      "--- gene=Chr7-6Mb-seg-91, id=41 removed by soft count = 82\n",
      "--- number of probes: 52 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-92\n",
      "--- gene=Chr7-6Mb-seg-92, id=0 removed by soft count = 220\n",
      "--- gene=Chr7-6Mb-seg-92, id=2 removed by soft count = 394\n",
      "--- gene=Chr7-6Mb-seg-92, id=15 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-92, id=51 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-92, id=65 removed by soft count = 66\n",
      "--- number of probes: 69 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-93\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-94\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-95\n",
      "--- gene=Chr7-6Mb-seg-95, id=63 removed by hard count = 0\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-96\n",
      "--- gene=Chr7-6Mb-seg-96, id=15 removed by soft count = 336\n",
      "--- gene=Chr7-6Mb-seg-96, id=27 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-96, id=41 removed by soft count = 60\n",
      "--- number of probes: 49 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-97\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-98\n",
      "--- gene=Chr7-6Mb-seg-98, id=72 removed by soft count = 80\n",
      "--- number of probes: 78 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-99\n",
      "--- gene=Chr7-6Mb-seg-99, id=65 removed by soft count = 54\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-100\n",
      "--- gene=Chr7-6Mb-seg-100, id=17 removed by soft count = 350\n",
      "--- gene=Chr7-6Mb-seg-100, id=39 removed by soft count = 421\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-101\n",
      "--- gene=Chr7-6Mb-seg-101, id=21 removed by soft count = 248\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-102\n",
      "--- gene=Chr7-6Mb-seg-102, id=24 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-102, id=59 removed by soft count = 3108\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-103\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-104\n",
      "--- gene=Chr7-6Mb-seg-104, id=1 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-104, id=10 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-104, id=37 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-104, id=52 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-104, id=55 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-104, id=64 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-104, id=70 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-104, id=71 removed by soft count = 78\n",
      "--- number of probes: 75 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-105\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-106\n",
      "--- gene=Chr7-6Mb-seg-106, id=24 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-106, id=32 removed by soft count = 122\n",
      "--- gene=Chr7-6Mb-seg-106, id=53 removed by soft count = 2434\n",
      "--- number of probes: 80 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-107\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-108\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-109\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-110\n",
      "--- gene=Chr7-6Mb-seg-110, id=8 removed by soft count = 205\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-111\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-112\n",
      "--- gene=Chr7-6Mb-seg-112, id=19 removed by soft count = 128\n",
      "--- gene=Chr7-6Mb-seg-112, id=46 removed by soft count = 501\n",
      "--- number of probes: 59 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Chr7-6Mb-seg-113\n",
      "--- gene=Chr7-6Mb-seg-113, id=53 removed by soft count = 96\n",
      "--- gene=Chr7-6Mb-seg-113, id=67 removed by soft count = 99\n",
      "--- number of probes: 68 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-114\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-114, id=4 removed by soft count = 284\n",
      "--- gene=Chr7-6Mb-seg-114, id=17 removed by soft count = 281\n",
      "--- gene=Chr7-6Mb-seg-114, id=20 removed by soft count = 135\n",
      "--- gene=Chr7-6Mb-seg-114, id=22 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-114, id=25 removed by soft count = 110\n",
      "--- gene=Chr7-6Mb-seg-114, id=30 removed by soft count = 147\n",
      "--- gene=Chr7-6Mb-seg-114, id=31 removed by soft count = 108\n",
      "--- gene=Chr7-6Mb-seg-114, id=33 removed by soft count = 113\n",
      "--- gene=Chr7-6Mb-seg-114, id=37 removed by soft count = 319\n",
      "--- gene=Chr7-6Mb-seg-114, id=42 removed by soft count = 272\n",
      "--- gene=Chr7-6Mb-seg-114, id=43 removed by soft count = 241\n",
      "--- gene=Chr7-6Mb-seg-114, id=44 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-114, id=48 removed by soft count = 67\n",
      "--- gene=Chr7-6Mb-seg-114, id=57 removed by soft count = 438\n",
      "--- gene=Chr7-6Mb-seg-114, id=58 removed by soft count = 525\n",
      "--- number of probes: 59 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-115\n",
      "--- gene=Chr7-6Mb-seg-115, id=0 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-115, id=1 removed by soft count = 161\n",
      "--- gene=Chr7-6Mb-seg-115, id=8 removed by soft count = 745\n",
      "--- gene=Chr7-6Mb-seg-115, id=9 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-115, id=13 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-115, id=14 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-115, id=23 removed by soft count = 1014\n",
      "--- gene=Chr7-6Mb-seg-115, id=46 removed by soft count = 77\n",
      "--- number of probes: 64 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-116\n",
      "--- gene=Chr7-6Mb-seg-116, id=15 removed by soft count = 634\n",
      "--- gene=Chr7-6Mb-seg-116, id=25 removed by soft count = 1335\n",
      "--- number of probes: 39 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: Chr7-6Mb-seg-117\n",
      "--- gene=Chr7-6Mb-seg-117, id=3 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-117, id=54 removed by soft count = 68\n",
      "--- gene=Chr7-6Mb-seg-117, id=56 removed by soft count = 402\n",
      "--- gene=Chr7-6Mb-seg-117, id=57 removed by soft count = 215\n",
      "--- gene=Chr7-6Mb-seg-117, id=58 removed by soft count = 119\n",
      "--- number of probes: 61 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-118\n",
      "--- gene=Chr7-6Mb-seg-118, id=17 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-118, id=43 removed by soft count = 204\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-119\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-120\n",
      "--- gene=Chr7-6Mb-seg-120, id=28 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-120, id=46 removed by soft count = 66\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-121\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-122\n",
      "--- gene=Chr7-6Mb-seg-122, id=35 removed by soft count = 61\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-123\n",
      "--- gene=Chr7-6Mb-seg-123, id=1 removed by soft count = 155\n",
      "--- number of probes: 76 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-124\n",
      "--- gene=Chr7-6Mb-seg-124, id=3 removed by hard count = 0\n",
      "--- gene=Chr7-6Mb-seg-124, id=45 removed by soft count = 107\n",
      "--- number of probes: 61 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-125\n",
      "--- gene=Chr7-6Mb-seg-125, id=0 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-125, id=3 removed by soft count = 169\n",
      "--- gene=Chr7-6Mb-seg-125, id=4 removed by soft count = 282\n",
      "--- gene=Chr7-6Mb-seg-125, id=7 removed by hard count = 4\n",
      "--- number of probes: 61 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Chr7-6Mb-seg-126\n",
      "--- gene=Chr7-6Mb-seg-126, id=31 removed by soft count = 243\n",
      "--- number of probes: 66 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-127\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-128\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-129\n",
      "--- gene=Chr7-6Mb-seg-129, id=3 removed by soft count = 61\n",
      "--- number of probes: 66 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-130\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-131\n",
      "--- gene=Chr7-6Mb-seg-131, id=22 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-131, id=60 removed by soft count = 67\n",
      "--- number of probes: 69 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-132\n",
      "--- gene=Chr7-6Mb-seg-132, id=24 removed by soft count = 56\n",
      "--- number of probes: 79 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-133\n",
      "--- number of probes: 76 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-134\n",
      "--- gene=Chr7-6Mb-seg-134, id=18 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-134, id=50 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-134, id=51 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-134, id=55 removed by soft count = 81\n",
      "--- gene=Chr7-6Mb-seg-134, id=56 removed by soft count = 81\n",
      "--- number of probes: 71 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-135\n",
      "--- gene=Chr7-6Mb-seg-135, id=26 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-135, id=59 removed by soft count = 749\n",
      "--- gene=Chr7-6Mb-seg-135, id=72 removed by soft count = 234\n",
      "--- number of probes: 75 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-136\n",
      "--- gene=Chr7-6Mb-seg-136, id=4 removed by soft count = 118\n",
      "--- gene=Chr7-6Mb-seg-136, id=43 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-136, id=48 removed by soft count = 360\n",
      "--- gene=Chr7-6Mb-seg-136, id=56 removed by soft count = 68\n",
      "--- number of probes: 64 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-137\n",
      "--- gene=Chr7-6Mb-seg-137, id=0 removed by soft count = 51\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-137, id=31 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-137, id=59 removed by soft count = 781\n",
      "--- number of probes: 66 , kept by blast: 63 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-138\n",
      "--- gene=Chr7-6Mb-seg-138, id=45 removed by soft count = 137\n",
      "--- gene=Chr7-6Mb-seg-138, id=52 removed by soft count = 81\n",
      "--- gene=Chr7-6Mb-seg-138, id=66 removed by soft count = 150\n",
      "--- number of probes: 67 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-139\n",
      "--- gene=Chr7-6Mb-seg-139, id=0 removed by soft count = 136\n",
      "--- gene=Chr7-6Mb-seg-139, id=2 removed by soft count = 265\n",
      "--- gene=Chr7-6Mb-seg-139, id=16 removed by soft count = 255\n",
      "--- gene=Chr7-6Mb-seg-139, id=48 removed by soft count = 131\n",
      "--- number of probes: 72 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-140\n",
      "--- gene=Chr7-6Mb-seg-140, id=16 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-140, id=24 removed by soft count = 108\n",
      "--- gene=Chr7-6Mb-seg-140, id=41 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-140, id=52 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-140, id=53 removed by soft count = 148\n",
      "--- gene=Chr7-6Mb-seg-140, id=55 removed by soft count = 69\n",
      "--- number of probes: 64 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-141\n",
      "--- gene=Chr7-6Mb-seg-141, id=8 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-141, id=37 removed by soft count = 129\n",
      "--- gene=Chr7-6Mb-seg-141, id=43 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-141, id=46 removed by soft count = 510\n",
      "--- gene=Chr7-6Mb-seg-141, id=63 removed by soft count = 973\n",
      "--- number of probes: 64 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-142\n",
      "--- gene=Chr7-6Mb-seg-142, id=29 removed by soft count = 817\n",
      "--- gene=Chr7-6Mb-seg-142, id=30 removed by soft count = 528\n",
      "--- number of probes: 57 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-143\n",
      "--- gene=Chr7-6Mb-seg-143, id=0 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-143, id=16 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-143, id=19 removed by soft count = 89\n",
      "--- number of probes: 65 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-144\n",
      "--- gene=Chr7-6Mb-seg-144, id=11 removed by soft count = 189\n",
      "--- gene=Chr7-6Mb-seg-144, id=14 removed by soft count = 27170\n",
      "--- gene=Chr7-6Mb-seg-144, id=29 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-144, id=41 removed by soft count = 98\n",
      "--- number of probes: 69 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-145\n",
      "--- gene=Chr7-6Mb-seg-145, id=14 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-145, id=30 removed by soft count = 51\n",
      "--- number of probes: 66 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-146\n",
      "--- gene=Chr7-6Mb-seg-146, id=44 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-146, id=47 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-146, id=49 removed by soft count = 218\n",
      "--- gene=Chr7-6Mb-seg-146, id=50 removed by soft count = 158\n",
      "--- number of probes: 61 , kept by blast: 57 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-147\n",
      "--- gene=Chr7-6Mb-seg-147, id=72 removed by soft count = 207\n",
      "--- number of probes: 73 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-148\n",
      "--- gene=Chr7-6Mb-seg-148, id=0 removed by soft count = 226\n",
      "--- gene=Chr7-6Mb-seg-148, id=1 removed by soft count = 1641\n",
      "--- gene=Chr7-6Mb-seg-148, id=2 removed by soft count = 380\n",
      "--- gene=Chr7-6Mb-seg-148, id=3 removed by soft count = 101\n",
      "--- number of probes: 64 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-149\n",
      "--- gene=Chr7-6Mb-seg-149, id=0 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-149, id=11 removed by soft count = 324\n",
      "--- gene=Chr7-6Mb-seg-149, id=25 removed by soft count = 261\n",
      "--- gene=Chr7-6Mb-seg-149, id=33 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-149, id=38 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-149, id=48 removed by soft count = 627\n",
      "--- number of probes: 59 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-150\n",
      "--- gene=Chr7-6Mb-seg-150, id=15 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-150, id=17 removed by soft count = 81\n",
      "--- gene=Chr7-6Mb-seg-150, id=18 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-150, id=19 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-150, id=20 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-150, id=22 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-150, id=23 removed by soft count = 131\n",
      "--- gene=Chr7-6Mb-seg-150, id=26 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-150, id=27 removed by soft count = 147\n",
      "--- gene=Chr7-6Mb-seg-150, id=61 removed by soft count = 106\n",
      "--- number of probes: 63 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-151\n",
      "--- gene=Chr7-6Mb-seg-151, id=46 removed by soft count = 186\n",
      "--- number of probes: 53 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-152\n",
      "--- gene=Chr7-6Mb-seg-152, id=4 removed by soft count = 133\n",
      "--- gene=Chr7-6Mb-seg-152, id=6 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-152, id=10 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-152, id=17 removed by soft count = 355\n",
      "--- gene=Chr7-6Mb-seg-152, id=18 removed by soft count = 122\n",
      "--- gene=Chr7-6Mb-seg-152, id=19 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-152, id=20 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-152, id=21 removed by soft count = 99\n",
      "--- gene=Chr7-6Mb-seg-152, id=35 removed by soft count = 103\n",
      "--- number of probes: 61 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-153\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-154\n",
      "--- gene=Chr7-6Mb-seg-154, id=10 removed by soft count = 148\n",
      "--- gene=Chr7-6Mb-seg-154, id=15 removed by soft count = 593\n",
      "--- gene=Chr7-6Mb-seg-154, id=36 removed by soft count = 78\n",
      "--- number of probes: 80 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-155\n",
      "--- gene=Chr7-6Mb-seg-155, id=39 removed by soft count = 53\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-156\n",
      "--- gene=Chr7-6Mb-seg-156, id=41 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-156, id=42 removed by soft count = 140\n",
      "--- gene=Chr7-6Mb-seg-156, id=44 removed by soft count = 220\n",
      "--- gene=Chr7-6Mb-seg-156, id=45 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-156, id=46 removed by soft count = 240\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 64 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-157\n",
      "--- gene=Chr7-6Mb-seg-157, id=68 removed by soft count = 95\n",
      "--- number of probes: 72 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-158\n",
      "--- gene=Chr7-6Mb-seg-158, id=0 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-158, id=2 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-158, id=6 removed by soft count = 604\n",
      "--- gene=Chr7-6Mb-seg-158, id=9 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-158, id=10 removed by soft count = 506\n",
      "--- gene=Chr7-6Mb-seg-158, id=39 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-158, id=40 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-158, id=41 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-158, id=42 removed by soft count = 180\n",
      "--- gene=Chr7-6Mb-seg-158, id=43 removed by soft count = 169\n",
      "--- gene=Chr7-6Mb-seg-158, id=46 removed by soft count = 158\n",
      "--- number of probes: 72 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-159\n",
      "--- gene=Chr7-6Mb-seg-159, id=11 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-159, id=45 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-159, id=46 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-159, id=49 removed by soft count = 51\n",
      "--- number of probes: 62 , kept by blast: 58 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-161\n",
      "--- gene=Chr7-6Mb-seg-161, id=0 removed by soft count = 204\n",
      "--- gene=Chr7-6Mb-seg-161, id=1 removed by soft count = 128\n",
      "--- gene=Chr7-6Mb-seg-161, id=2 removed by soft count = 102\n",
      "--- number of probes: 29 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: Chr7-6Mb-seg-162\n",
      "--- gene=Chr7-6Mb-seg-162, id=20 removed by soft count = 276\n",
      "--- gene=Chr7-6Mb-seg-162, id=21 removed by soft count = 467\n",
      "--- gene=Chr7-6Mb-seg-162, id=26 removed by soft count = 99\n",
      "--- gene=Chr7-6Mb-seg-162, id=40 removed by soft count = 420\n",
      "--- gene=Chr7-6Mb-seg-162, id=53 removed by soft count = 679\n",
      "--- gene=Chr7-6Mb-seg-162, id=54 removed by soft count = 2061\n",
      "--- gene=Chr7-6Mb-seg-162, id=55 removed by soft count = 58\n",
      "--- number of probes: 61 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-163\n",
      "--- gene=Chr7-6Mb-seg-163, id=0 removed by soft count = 127\n",
      "--- gene=Chr7-6Mb-seg-163, id=4 removed by soft count = 582\n",
      "--- number of probes: 75 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-164\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-165\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-166\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-167\n",
      "--- gene=Chr7-6Mb-seg-167, id=21 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-167, id=57 removed by hard count = 5\n",
      "--- number of probes: 78 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-168\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-169\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-170\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-171\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-172\n",
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-173\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-174\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-175\n",
      "--- gene=Chr7-6Mb-seg-175, id=26 removed by soft count = 430\n",
      "--- gene=Chr7-6Mb-seg-175, id=27 removed by soft count = 128\n",
      "--- gene=Chr7-6Mb-seg-175, id=30 removed by soft count = 217\n",
      "--- gene=Chr7-6Mb-seg-175, id=31 removed by soft count = 55\n",
      "--- number of probes: 59 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-176\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-177\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-178\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-179\n",
      "--- gene=Chr7-6Mb-seg-179, id=19 removed by soft count = 98\n",
      "--- gene=Chr7-6Mb-seg-179, id=60 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-179, id=61 removed by soft count = 52\n",
      "--- number of probes: 67 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-180\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-181\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-182\n",
      "--- gene=Chr7-6Mb-seg-182, id=14 removed by soft count = 53\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-183\n",
      "--- gene=Chr7-6Mb-seg-183, id=8 removed by soft count = 79\n",
      "--- gene=Chr7-6Mb-seg-183, id=23 removed by soft count = 51\n",
      "--- number of probes: 70 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-184\n",
      "--- gene=Chr7-6Mb-seg-184, id=58 removed by hard count = 5\n",
      "--- gene=Chr7-6Mb-seg-184, id=74 removed by soft count = 101\n",
      "--- number of probes: 75 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-185\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-185, id=3 removed by soft count = 1564\n",
      "--- gene=Chr7-6Mb-seg-185, id=18 removed by soft count = 80\n",
      "--- number of probes: 57 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-186\n",
      "--- gene=Chr7-6Mb-seg-186, id=2 removed by soft count = 236\n",
      "--- gene=Chr7-6Mb-seg-186, id=14 removed by soft count = 131\n",
      "--- number of probes: 72 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-187\n",
      "--- gene=Chr7-6Mb-seg-187, id=13 removed by soft count = 513\n",
      "--- gene=Chr7-6Mb-seg-187, id=28 removed by soft count = 152\n",
      "--- gene=Chr7-6Mb-seg-187, id=37 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-187, id=42 removed by soft count = 99\n",
      "--- gene=Chr7-6Mb-seg-187, id=50 removed by soft count = 68\n",
      "--- gene=Chr7-6Mb-seg-187, id=51 removed by soft count = 70\n",
      "--- number of probes: 56 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: Chr7-6Mb-seg-188\n",
      "--- gene=Chr7-6Mb-seg-188, id=14 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-188, id=19 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-188, id=27 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-188, id=52 removed by soft count = 57\n",
      "--- number of probes: 54 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: Chr7-6Mb-seg-189\n",
      "--- gene=Chr7-6Mb-seg-189, id=0 removed by soft count = 224\n",
      "--- gene=Chr7-6Mb-seg-189, id=2 removed by soft count = 156\n",
      "--- gene=Chr7-6Mb-seg-189, id=6 removed by soft count = 168\n",
      "--- gene=Chr7-6Mb-seg-189, id=17 removed by soft count = 154\n",
      "--- gene=Chr7-6Mb-seg-189, id=49 removed by soft count = 114\n",
      "--- number of probes: 57 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-190\n",
      "--- gene=Chr7-6Mb-seg-190, id=0 removed by soft count = 8906\n",
      "--- gene=Chr7-6Mb-seg-190, id=15 removed by soft count = 440\n",
      "--- gene=Chr7-6Mb-seg-190, id=16 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-190, id=48 removed by soft count = 2193\n",
      "--- number of probes: 54 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: Chr7-6Mb-seg-191\n",
      "--- gene=Chr7-6Mb-seg-191, id=4 removed by soft count = 221\n",
      "--- gene=Chr7-6Mb-seg-191, id=7 removed by soft count = 3284\n",
      "--- gene=Chr7-6Mb-seg-191, id=48 removed by soft count = 92\n",
      "--- number of probes: 69 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-192\n",
      "--- gene=Chr7-6Mb-seg-192, id=0 removed by soft count = 894\n",
      "--- gene=Chr7-6Mb-seg-192, id=5 removed by soft count = 588\n",
      "--- gene=Chr7-6Mb-seg-192, id=13 removed by soft count = 269\n",
      "--- number of probes: 75 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-193\n",
      "--- gene=Chr7-6Mb-seg-193, id=50 removed by soft count = 73\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-194\n",
      "--- gene=Chr7-6Mb-seg-194, id=18 removed by soft count = 138\n",
      "--- gene=Chr7-6Mb-seg-194, id=41 removed by soft count = 108\n",
      "--- number of probes: 60 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-195\n",
      "--- gene=Chr7-6Mb-seg-195, id=15 removed by soft count = 75\n",
      "--- gene=Chr7-6Mb-seg-195, id=18 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-195, id=33 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-195, id=54 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-195, id=60 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-195, id=63 removed by soft count = 51\n",
      "--- number of probes: 75 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-196\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-197\n",
      "--- number of probes: 74 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-198\n",
      "--- gene=Chr7-6Mb-seg-198, id=48 removed by soft count = 1240\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-199\n",
      "--- gene=Chr7-6Mb-seg-199, id=9 removed by soft count = 431\n",
      "--- gene=Chr7-6Mb-seg-199, id=12 removed by soft count = 447\n",
      "--- gene=Chr7-6Mb-seg-199, id=52 removed by soft count = 377\n",
      "--- number of probes: 76 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-200\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-201\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-202\n",
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-203\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-204\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-205\n",
      "--- gene=Chr7-6Mb-seg-205, id=14 removed by soft count = 240\n",
      "--- gene=Chr7-6Mb-seg-205, id=36 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-205, id=39 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-205, id=44 removed by soft count = 319\n",
      "--- gene=Chr7-6Mb-seg-205, id=59 removed by soft count = 350\n",
      "--- gene=Chr7-6Mb-seg-205, id=61 removed by soft count = 59\n",
      "--- number of probes: 63 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Chr7-6Mb-seg-206\n",
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-207\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-208\n",
      "--- number of probes: 76 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-209\n",
      "--- gene=Chr7-6Mb-seg-209, id=55 removed by soft count = 651\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-210\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-211\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-211, id=79 removed by soft count = 172\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-212\n",
      "--- gene=Chr7-6Mb-seg-212, id=3 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-212, id=46 removed by soft count = 74\n",
      "--- number of probes: 71 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-213\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-214\n",
      "--- gene=Chr7-6Mb-seg-214, id=10 removed by soft count = 248\n",
      "--- gene=Chr7-6Mb-seg-214, id=12 removed by soft count = 75\n",
      "--- gene=Chr7-6Mb-seg-214, id=15 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-214, id=40 removed by soft count = 37067\n",
      "--- gene=Chr7-6Mb-seg-214, id=57 removed by soft count = 1515\n",
      "--- number of probes: 61 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-215\n",
      "--- gene=Chr7-6Mb-seg-215, id=14 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-215, id=15 removed by soft count = 136\n",
      "--- number of probes: 55 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-216\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-217\n",
      "--- gene=Chr7-6Mb-seg-217, id=4 removed by soft count = 1122\n",
      "--- gene=Chr7-6Mb-seg-217, id=5 removed by soft count = 1248\n",
      "--- gene=Chr7-6Mb-seg-217, id=10 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-217, id=25 removed by soft count = 314\n",
      "--- number of probes: 61 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Chr7-6Mb-seg-218\n",
      "--- gene=Chr7-6Mb-seg-218, id=56 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-218, id=57 removed by soft count = 205\n",
      "--- gene=Chr7-6Mb-seg-218, id=59 removed by soft count = 109\n",
      "--- number of probes: 60 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Chr7-6Mb-seg-219\n",
      "--- gene=Chr7-6Mb-seg-219, id=61 removed by soft count = 149\n",
      "--- gene=Chr7-6Mb-seg-219, id=64 removed by soft count = 59\n",
      "--- number of probes: 70 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-220\n",
      "--- gene=Chr7-6Mb-seg-220, id=11 removed by soft count = 144\n",
      "--- number of probes: 54 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-221\n",
      "--- gene=Chr7-6Mb-seg-221, id=1 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-221, id=6 removed by soft count = 162\n",
      "--- gene=Chr7-6Mb-seg-221, id=12 removed by soft count = 70\n",
      "--- number of probes: 67 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-222\n",
      "--- gene=Chr7-6Mb-seg-222, id=24 removed by soft count = 1001\n",
      "--- gene=Chr7-6Mb-seg-222, id=48 removed by soft count = 179\n",
      "--- number of probes: 51 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-223\n",
      "--- gene=Chr7-6Mb-seg-223, id=0 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-223, id=2 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-223, id=4 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-223, id=20 removed by soft count = 489\n",
      "--- number of probes: 54 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: Chr7-6Mb-seg-224\n",
      "--- gene=Chr7-6Mb-seg-224, id=15 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-224, id=18 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-224, id=44 removed by soft count = 459\n",
      "--- number of probes: 57 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-225\n",
      "--- number of probes: 76 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-226\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-227\n",
      "--- gene=Chr7-6Mb-seg-227, id=4 removed by soft count = 1224\n",
      "--- gene=Chr7-6Mb-seg-227, id=7 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-227, id=41 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-227, id=50 removed by soft count = 210\n",
      "--- number of probes: 73 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-228\n",
      "--- gene=Chr7-6Mb-seg-228, id=65 removed by soft count = 173\n",
      "--- number of probes: 68 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-229\n",
      "--- number of probes: 76 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-230\n",
      "--- gene=Chr7-6Mb-seg-230, id=65 removed by soft count = 51\n",
      "--- number of probes: 74 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-231\n",
      "--- gene=Chr7-6Mb-seg-231, id=69 removed by soft count = 1322\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-232\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-233\n",
      "--- gene=Chr7-6Mb-seg-233, id=20 removed by soft count = 788\n",
      "--- gene=Chr7-6Mb-seg-233, id=47 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-233, id=49 removed by soft count = 242\n",
      "--- number of probes: 73 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-234\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-235\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-236\n",
      "--- gene=Chr7-6Mb-seg-236, id=36 removed by soft count = 66\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-237\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-238\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-239\n",
      "--- gene=Chr7-6Mb-seg-239, id=21 removed by soft count = 55\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-240\n",
      "--- gene=Chr7-6Mb-seg-240, id=41 removed by hard count = 0\n",
      "--- gene=Chr7-6Mb-seg-240, id=76 removed by soft count = 131\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-241\n",
      "--- gene=Chr7-6Mb-seg-241, id=26 removed by soft count = 191\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-242\n",
      "--- gene=Chr7-6Mb-seg-242, id=8 removed by soft count = 444\n",
      "--- gene=Chr7-6Mb-seg-242, id=9 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-242, id=16 removed by soft count = 91\n",
      "--- gene=Chr7-6Mb-seg-242, id=19 removed by soft count = 97\n",
      "--- gene=Chr7-6Mb-seg-242, id=24 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-242, id=43 removed by soft count = 176\n",
      "--- number of probes: 58 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-243\n",
      "--- gene=Chr7-6Mb-seg-243, id=15 removed by soft count = 488\n",
      "--- gene=Chr7-6Mb-seg-243, id=48 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-243, id=56 removed by soft count = 1015\n",
      "--- number of probes: 71 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-244\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-245\n",
      "--- gene=Chr7-6Mb-seg-245, id=49 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-245, id=50 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-245, id=51 removed by soft count = 155\n",
      "--- gene=Chr7-6Mb-seg-245, id=65 removed by soft count = 57\n",
      "--- number of probes: 74 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-246\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-247\n",
      "--- number of probes: 74 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-248\n",
      "--- gene=Chr7-6Mb-seg-248, id=44 removed by soft count = 64\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-249\n",
      "--- gene=Chr7-6Mb-seg-249, id=30 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-249, id=36 removed by soft count = 147\n",
      "--- gene=Chr7-6Mb-seg-249, id=46 removed by soft count = 180\n",
      "--- gene=Chr7-6Mb-seg-249, id=47 removed by soft count = 91\n",
      "--- gene=Chr7-6Mb-seg-249, id=50 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-249, id=53 removed by soft count = 71\n",
      "--- number of probes: 55 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-250\n",
      "--- gene=Chr7-6Mb-seg-250, id=0 removed by soft count = 1735\n",
      "--- gene=Chr7-6Mb-seg-250, id=1 removed by soft count = 296\n",
      "--- gene=Chr7-6Mb-seg-250, id=2 removed by soft count = 127\n",
      "--- gene=Chr7-6Mb-seg-250, id=4 removed by soft count = 1659\n",
      "--- gene=Chr7-6Mb-seg-250, id=8 removed by soft count = 142\n",
      "--- gene=Chr7-6Mb-seg-250, id=9 removed by soft count = 484\n",
      "--- gene=Chr7-6Mb-seg-250, id=10 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-250, id=12 removed by soft count = 231\n",
      "--- gene=Chr7-6Mb-seg-250, id=14 removed by soft count = 130\n",
      "--- number of probes: 21 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-seg-251\n",
      "--- gene=Chr7-6Mb-seg-251, id=7 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-251, id=8 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-251, id=10 removed by soft count = 68\n",
      "--- gene=Chr7-6Mb-seg-251, id=14 removed by soft count = 132\n",
      "--- gene=Chr7-6Mb-seg-251, id=20 removed by soft count = 259\n",
      "--- gene=Chr7-6Mb-seg-251, id=39 removed by soft count = 325\n",
      "--- gene=Chr7-6Mb-seg-251, id=44 removed by soft count = 389\n",
      "--- number of probes: 46 , kept by blast: 39 , if remove dups: 36\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: Chr7-6Mb-seg-252\n",
      "--- gene=Chr7-6Mb-seg-252, id=34 removed by soft count = 220\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-253\n",
      "--- gene=Chr7-6Mb-seg-253, id=35 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-253, id=78 removed by soft count = 715\n",
      "--- number of probes: 79 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-254\n",
      "--- number of probes: 62 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-255\n",
      "--- gene=Chr7-6Mb-seg-255, id=11 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-255, id=14 removed by soft count = 253\n",
      "--- gene=Chr7-6Mb-seg-255, id=17 removed by soft count = 145\n",
      "--- number of probes: 74 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-256\n",
      "--- gene=Chr7-6Mb-seg-256, id=7 removed by soft count = 2016\n",
      "--- gene=Chr7-6Mb-seg-256, id=39 removed by soft count = 553\n",
      "--- gene=Chr7-6Mb-seg-256, id=57 removed by soft count = 114\n",
      "--- number of probes: 71 , kept by blast: 68 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-257\n",
      "--- gene=Chr7-6Mb-seg-257, id=57 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-257, id=58 removed by soft count = 10200\n",
      "--- number of probes: 71 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-258\n",
      "--- gene=Chr7-6Mb-seg-258, id=2 removed by soft count = 212\n",
      "--- gene=Chr7-6Mb-seg-258, id=7 removed by soft count = 168\n",
      "--- number of probes: 75 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-259\n",
      "--- gene=Chr7-6Mb-seg-259, id=5 removed by soft count = 128\n",
      "--- gene=Chr7-6Mb-seg-259, id=13 removed by soft count = 211\n",
      "--- gene=Chr7-6Mb-seg-259, id=29 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-259, id=66 removed by soft count = 54\n",
      "--- number of probes: 68 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-260\n",
      "--- gene=Chr7-6Mb-seg-260, id=21 removed by soft count = 85\n",
      "--- gene=Chr7-6Mb-seg-260, id=26 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-260, id=29 removed by soft count = 82\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-260, id=30 removed by soft count = 649\n",
      "--- gene=Chr7-6Mb-seg-260, id=32 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-260, id=35 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-260, id=53 removed by soft count = 65\n",
      "--- number of probes: 63 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-261\n",
      "--- gene=Chr7-6Mb-seg-261, id=59 removed by soft count = 203\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-262\n",
      "--- gene=Chr7-6Mb-seg-262, id=14 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-262, id=21 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-262, id=31 removed by soft count = 52\n",
      "--- number of probes: 71 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-263\n",
      "--- gene=Chr7-6Mb-seg-263, id=15 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-263, id=33 removed by soft count = 107\n",
      "--- number of probes: 74 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-264\n",
      "--- gene=Chr7-6Mb-seg-264, id=11 removed by soft count = 190\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-265\n",
      "--- gene=Chr7-6Mb-seg-265, id=12 removed by soft count = 96\n",
      "--- gene=Chr7-6Mb-seg-265, id=13 removed by soft count = 266\n",
      "--- gene=Chr7-6Mb-seg-265, id=36 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-265, id=48 removed by soft count = 90\n",
      "--- gene=Chr7-6Mb-seg-265, id=51 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-265, id=54 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-265, id=57 removed by soft count = 70\n",
      "--- number of probes: 69 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-266\n",
      "--- gene=Chr7-6Mb-seg-266, id=41 removed by soft count = 219\n",
      "--- number of probes: 56 , kept by blast: 55 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: Chr7-6Mb-seg-267\n",
      "--- gene=Chr7-6Mb-seg-267, id=0 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-267, id=4 removed by soft count = 736\n",
      "--- number of probes: 76 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-268\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-269\n",
      "--- gene=Chr7-6Mb-seg-269, id=43 removed by soft count = 168\n",
      "--- gene=Chr7-6Mb-seg-269, id=67 removed by soft count = 53\n",
      "--- number of probes: 71 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-270\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-271\n",
      "--- gene=Chr7-6Mb-seg-271, id=60 removed by soft count = 317\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-272\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-273\n",
      "--- gene=Chr7-6Mb-seg-273, id=8 removed by soft count = 297\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-274\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-275\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-276\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-277\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-278\n",
      "--- gene=Chr7-6Mb-seg-278, id=5 removed by soft count = 533\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-279\n",
      "--- gene=Chr7-6Mb-seg-279, id=17 removed by soft count = 197\n",
      "--- gene=Chr7-6Mb-seg-279, id=57 removed by soft count = 81\n",
      "--- gene=Chr7-6Mb-seg-279, id=58 removed by soft count = 51\n",
      "--- number of probes: 65 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-280\n",
      "--- gene=Chr7-6Mb-seg-280, id=29 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-280, id=39 removed by soft count = 1203\n",
      "--- gene=Chr7-6Mb-seg-280, id=42 removed by soft count = 140\n",
      "--- gene=Chr7-6Mb-seg-280, id=44 removed by soft count = 312\n",
      "--- gene=Chr7-6Mb-seg-280, id=45 removed by soft count = 184\n",
      "--- gene=Chr7-6Mb-seg-280, id=47 removed by soft count = 114\n",
      "--- number of probes: 48 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: Chr7-6Mb-seg-281\n",
      "--- gene=Chr7-6Mb-seg-281, id=21 removed by soft count = 59\n",
      "--- number of probes: 48 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: Chr7-6Mb-seg-282\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-283\n",
      "--- gene=Chr7-6Mb-seg-283, id=47 removed by soft count = 79\n",
      "--- number of probes: 74 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-284\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-285\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-286\n",
      "--- gene=Chr7-6Mb-seg-286, id=18 removed by soft count = 177\n",
      "--- number of probes: 73 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-287\n",
      "--- gene=Chr7-6Mb-seg-287, id=34 removed by soft count = 56\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-288\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-288, id=40 removed by soft count = 137\n",
      "--- gene=Chr7-6Mb-seg-288, id=42 removed by soft count = 335\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-289\n",
      "--- gene=Chr7-6Mb-seg-289, id=20 removed by soft count = 176\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-290\n",
      "--- gene=Chr7-6Mb-seg-290, id=13 removed by soft count = 978\n",
      "--- number of probes: 60 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-291\n",
      "--- gene=Chr7-6Mb-seg-291, id=1 removed by soft count = 296\n",
      "--- gene=Chr7-6Mb-seg-291, id=29 removed by soft count = 141\n",
      "--- number of probes: 46 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-292\n",
      "--- gene=Chr7-6Mb-seg-292, id=16 removed by soft count = 1198\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-293\n",
      "--- gene=Chr7-6Mb-seg-293, id=50 removed by soft count = 162\n",
      "--- gene=Chr7-6Mb-seg-293, id=51 removed by soft count = 100\n",
      "--- number of probes: 72 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-294\n",
      "--- gene=Chr7-6Mb-seg-294, id=44 removed by soft count = 183\n",
      "--- gene=Chr7-6Mb-seg-294, id=48 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-294, id=62 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-294, id=65 removed by soft count = 175\n",
      "--- number of probes: 70 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-295\n",
      "--- gene=Chr7-6Mb-seg-295, id=33 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-295, id=40 removed by soft count = 115\n",
      "--- gene=Chr7-6Mb-seg-295, id=41 removed by soft count = 54\n",
      "--- number of probes: 46 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: Chr7-6Mb-seg-296\n",
      "--- gene=Chr7-6Mb-seg-296, id=39 removed by soft count = 420\n",
      "--- gene=Chr7-6Mb-seg-296, id=40 removed by soft count = 729\n",
      "--- number of probes: 61 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-297\n",
      "--- gene=Chr7-6Mb-seg-297, id=27 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-297, id=28 removed by soft count = 94\n",
      "--- gene=Chr7-6Mb-seg-297, id=29 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-297, id=30 removed by soft count = 129\n",
      "--- number of probes: 36 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: Chr7-6Mb-seg-298\n",
      "--- gene=Chr7-6Mb-seg-298, id=23 removed by soft count = 342\n",
      "--- gene=Chr7-6Mb-seg-298, id=26 removed by soft count = 75\n",
      "--- number of probes: 61 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-299\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-300\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-301\n",
      "--- gene=Chr7-6Mb-seg-301, id=15 removed by soft count = 123\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-302\n",
      "--- gene=Chr7-6Mb-seg-302, id=2 removed by soft count = 227\n",
      "--- gene=Chr7-6Mb-seg-302, id=22 removed by soft count = 1915\n",
      "--- gene=Chr7-6Mb-seg-302, id=33 removed by soft count = 5539\n",
      "--- number of probes: 70 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-303\n",
      "--- gene=Chr7-6Mb-seg-303, id=15 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-303, id=19 removed by soft count = 57\n",
      "--- number of probes: 48 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-304\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-305\n",
      "--- gene=Chr7-6Mb-seg-305, id=4 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-305, id=54 removed by hard count = 5\n",
      "--- gene=Chr7-6Mb-seg-305, id=58 removed by hard count = 3\n",
      "--- gene=Chr7-6Mb-seg-305, id=59 removed by hard count = 4\n",
      "--- number of probes: 67 , kept by blast: 63 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-306\n",
      "--- gene=Chr7-6Mb-seg-306, id=35 removed by soft count = 109\n",
      "--- number of probes: 61 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-307\n",
      "--- gene=Chr7-6Mb-seg-307, id=33 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-307, id=34 removed by soft count = 211\n",
      "--- gene=Chr7-6Mb-seg-307, id=42 removed by soft count = 3321\n",
      "--- gene=Chr7-6Mb-seg-307, id=43 removed by soft count = 754\n",
      "--- number of probes: 70 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-308\n",
      "--- gene=Chr7-6Mb-seg-308, id=7 removed by soft count = 341\n",
      "--- gene=Chr7-6Mb-seg-308, id=10 removed by soft count = 519\n",
      "--- gene=Chr7-6Mb-seg-308, id=11 removed by soft count = 165\n",
      "--- gene=Chr7-6Mb-seg-308, id=12 removed by soft count = 307\n",
      "--- number of probes: 62 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-309\n",
      "--- number of probes: 54 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-310\n",
      "--- gene=Chr7-6Mb-seg-310, id=31 removed by soft count = 198\n",
      "--- number of probes: 56 , kept by blast: 55 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-311\n",
      "--- gene=Chr7-6Mb-seg-311, id=20 removed by hard count = 0\n",
      "--- gene=Chr7-6Mb-seg-311, id=23 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-311, id=24 removed by soft count = 72\n",
      "--- number of probes: 74 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-312\n",
      "--- gene=Chr7-6Mb-seg-312, id=37 removed by soft count = 232\n",
      "--- gene=Chr7-6Mb-seg-312, id=38 removed by soft count = 886\n",
      "--- gene=Chr7-6Mb-seg-312, id=57 removed by soft count = 59\n",
      "--- number of probes: 59 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-313\n",
      "--- gene=Chr7-6Mb-seg-313, id=58 removed by soft count = 1256\n",
      "--- gene=Chr7-6Mb-seg-313, id=59 removed by soft count = 339\n",
      "--- number of probes: 60 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-314\n",
      "--- gene=Chr7-6Mb-seg-314, id=0 removed by soft count = 166\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-314, id=2 removed by soft count = 321\n",
      "--- gene=Chr7-6Mb-seg-314, id=24 removed by soft count = 137\n",
      "--- gene=Chr7-6Mb-seg-314, id=60 removed by soft count = 180\n",
      "--- gene=Chr7-6Mb-seg-314, id=61 removed by soft count = 198\n",
      "--- number of probes: 64 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-315\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-316\n",
      "--- gene=Chr7-6Mb-seg-316, id=34 removed by soft count = 448\n",
      "--- gene=Chr7-6Mb-seg-316, id=35 removed by soft count = 148\n",
      "--- gene=Chr7-6Mb-seg-316, id=36 removed by soft count = 1681\n",
      "--- gene=Chr7-6Mb-seg-316, id=37 removed by soft count = 717\n",
      "--- gene=Chr7-6Mb-seg-316, id=39 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-316, id=40 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-316, id=43 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-316, id=44 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-316, id=46 removed by soft count = 76\n",
      "--- number of probes: 50 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: Chr7-6Mb-seg-317\n",
      "--- gene=Chr7-6Mb-seg-317, id=29 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-317, id=43 removed by soft count = 65\n",
      "--- number of probes: 53 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-318\n",
      "--- gene=Chr7-6Mb-seg-318, id=33 removed by soft count = 1168\n",
      "--- number of probes: 76 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-319\n",
      "--- gene=Chr7-6Mb-seg-319, id=11 removed by soft count = 226\n",
      "--- gene=Chr7-6Mb-seg-319, id=16 removed by soft count = 1699\n",
      "--- gene=Chr7-6Mb-seg-319, id=41 removed by soft count = 51\n",
      "--- number of probes: 48 , kept by blast: 45 , if remove dups: 43\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: Chr7-6Mb-seg-320\n",
      "--- gene=Chr7-6Mb-seg-320, id=2 removed by soft count = 152\n",
      "--- gene=Chr7-6Mb-seg-320, id=27 removed by soft count = 68\n",
      "--- gene=Chr7-6Mb-seg-320, id=48 removed by soft count = 211\n",
      "--- number of probes: 74 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-321\n",
      "--- gene=Chr7-6Mb-seg-321, id=16 removed by soft count = 148\n",
      "--- gene=Chr7-6Mb-seg-321, id=18 removed by soft count = 320\n",
      "--- gene=Chr7-6Mb-seg-321, id=21 removed by soft count = 1032\n",
      "--- gene=Chr7-6Mb-seg-321, id=22 removed by soft count = 341\n",
      "--- gene=Chr7-6Mb-seg-321, id=25 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-321, id=27 removed by soft count = 231\n",
      "--- gene=Chr7-6Mb-seg-321, id=28 removed by soft count = 309\n",
      "--- gene=Chr7-6Mb-seg-321, id=29 removed by soft count = 431\n",
      "--- gene=Chr7-6Mb-seg-321, id=34 removed by soft count = 243\n",
      "--- number of probes: 40 , kept by blast: 31 , if remove dups: 31\n",
      "-- number of probes kept for this region: 31\n",
      "-- checking probes in region: Chr7-6Mb-seg-322\n",
      "--- gene=Chr7-6Mb-seg-322, id=28 removed by soft count = 68\n",
      "--- number of probes: 60 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-323\n",
      "--- gene=Chr7-6Mb-seg-323, id=26 removed by soft count = 159\n",
      "--- gene=Chr7-6Mb-seg-323, id=44 removed by soft count = 73\n",
      "--- number of probes: 54 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-324\n",
      "--- gene=Chr7-6Mb-seg-324, id=14 removed by soft count = 145\n",
      "--- gene=Chr7-6Mb-seg-324, id=17 removed by soft count = 179\n",
      "--- gene=Chr7-6Mb-seg-324, id=20 removed by soft count = 113\n",
      "--- number of probes: 80 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-325\n",
      "--- gene=Chr7-6Mb-seg-325, id=18 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-325, id=48 removed by soft count = 2197\n",
      "--- gene=Chr7-6Mb-seg-325, id=69 removed by soft count = 251\n",
      "--- number of probes: 76 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-326\n",
      "--- gene=Chr7-6Mb-seg-326, id=74 removed by soft count = 401\n",
      "--- gene=Chr7-6Mb-seg-326, id=75 removed by soft count = 105\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-327\n",
      "--- gene=Chr7-6Mb-seg-327, id=5 removed by soft count = 349\n",
      "--- number of probes: 78 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-328\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-329\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-330\n",
      "--- gene=Chr7-6Mb-seg-330, id=54 removed by soft count = 238\n",
      "--- gene=Chr7-6Mb-seg-330, id=62 removed by soft count = 170\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-331\n",
      "--- gene=Chr7-6Mb-seg-331, id=55 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-331, id=63 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-331, id=65 removed by soft count = 99\n",
      "--- number of probes: 74 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-332\n",
      "--- gene=Chr7-6Mb-seg-332, id=34 removed by soft count = 110\n",
      "--- gene=Chr7-6Mb-seg-332, id=35 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-332, id=36 removed by soft count = 342\n",
      "--- gene=Chr7-6Mb-seg-332, id=37 removed by soft count = 73\n",
      "--- number of probes: 59 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-333\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-334\n",
      "--- gene=Chr7-6Mb-seg-334, id=39 removed by soft count = 93\n",
      "--- gene=Chr7-6Mb-seg-334, id=45 removed by soft count = 596\n",
      "--- gene=Chr7-6Mb-seg-334, id=56 removed by hard count = 3\n",
      "--- number of probes: 69 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-335\n",
      "--- gene=Chr7-6Mb-seg-335, id=5 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-335, id=6 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-335, id=9 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-335, id=10 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-335, id=11 removed by soft count = 488\n",
      "--- gene=Chr7-6Mb-seg-335, id=12 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-335, id=15 removed by soft count = 133\n",
      "--- gene=Chr7-6Mb-seg-335, id=30 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-335, id=31 removed by soft count = 149\n",
      "--- gene=Chr7-6Mb-seg-335, id=33 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-335, id=36 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-335, id=39 removed by soft count = 136\n",
      "--- gene=Chr7-6Mb-seg-335, id=40 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-335, id=42 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-335, id=48 removed by soft count = 52\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-335, id=49 removed by soft count = 209\n",
      "--- number of probes: 62 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-336\n",
      "--- gene=Chr7-6Mb-seg-336, id=47 removed by soft count = 808\n",
      "--- gene=Chr7-6Mb-seg-336, id=48 removed by soft count = 241\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-337\n",
      "--- gene=Chr7-6Mb-seg-337, id=32 removed by soft count = 243\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-338\n",
      "--- gene=Chr7-6Mb-seg-338, id=45 removed by soft count = 1193\n",
      "--- gene=Chr7-6Mb-seg-338, id=46 removed by soft count = 748\n",
      "--- gene=Chr7-6Mb-seg-338, id=47 removed by soft count = 786\n",
      "--- gene=Chr7-6Mb-seg-338, id=48 removed by soft count = 756\n",
      "--- number of probes: 66 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-339\n",
      "--- gene=Chr7-6Mb-seg-339, id=29 removed by soft count = 197\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-340\n",
      "--- gene=Chr7-6Mb-seg-340, id=53 removed by soft count = 51\n",
      "--- number of probes: 55 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-341\n",
      "--- gene=Chr7-6Mb-seg-341, id=52 removed by soft count = 788\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-342\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-343\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-344\n",
      "--- gene=Chr7-6Mb-seg-344, id=23 removed by soft count = 234\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-345\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-346\n",
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-347\n",
      "--- gene=Chr7-6Mb-seg-347, id=67 removed by soft count = 91\n",
      "--- gene=Chr7-6Mb-seg-347, id=69 removed by soft count = 2244\n",
      "--- number of probes: 72 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-348\n",
      "--- gene=Chr7-6Mb-seg-348, id=14 removed by soft count = 390\n",
      "--- gene=Chr7-6Mb-seg-348, id=15 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-348, id=28 removed by soft count = 128\n",
      "--- gene=Chr7-6Mb-seg-348, id=35 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-348, id=39 removed by soft count = 1455\n",
      "--- number of probes: 46 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: Chr7-6Mb-seg-349\n",
      "--- gene=Chr7-6Mb-seg-349, id=48 removed by soft count = 12013\n",
      "--- gene=Chr7-6Mb-seg-349, id=49 removed by soft count = 59\n",
      "--- number of probes: 68 , kept by blast: 66 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-350\n",
      "--- gene=Chr7-6Mb-seg-350, id=28 removed by soft count = 163\n",
      "--- gene=Chr7-6Mb-seg-350, id=29 removed by soft count = 14048\n",
      "--- number of probes: 75 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-351\n",
      "--- gene=Chr7-6Mb-seg-351, id=7 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-351, id=16 removed by soft count = 298\n",
      "--- number of probes: 55 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-352\n",
      "--- gene=Chr7-6Mb-seg-352, id=15 removed by soft count = 2773\n",
      "--- gene=Chr7-6Mb-seg-352, id=16 removed by soft count = 1243\n",
      "--- gene=Chr7-6Mb-seg-352, id=22 removed by soft count = 451\n",
      "--- number of probes: 78 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-353\n",
      "--- gene=Chr7-6Mb-seg-353, id=24 removed by soft count = 145\n",
      "--- gene=Chr7-6Mb-seg-353, id=26 removed by soft count = 159\n",
      "--- gene=Chr7-6Mb-seg-353, id=37 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-353, id=46 removed by soft count = 144\n",
      "--- gene=Chr7-6Mb-seg-353, id=47 removed by soft count = 84\n",
      "--- number of probes: 51 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-354\n",
      "--- gene=Chr7-6Mb-seg-354, id=37 removed by soft count = 534\n",
      "--- number of probes: 53 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-355\n",
      "--- number of probes: 62 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-356\n",
      "--- gene=Chr7-6Mb-seg-356, id=27 removed by soft count = 5108\n",
      "--- gene=Chr7-6Mb-seg-356, id=33 removed by soft count = 120\n",
      "--- number of probes: 72 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-357\n",
      "--- gene=Chr7-6Mb-seg-357, id=4 removed by soft count = 121\n",
      "--- gene=Chr7-6Mb-seg-357, id=6 removed by soft count = 204\n",
      "--- gene=Chr7-6Mb-seg-357, id=7 removed by soft count = 183\n",
      "--- gene=Chr7-6Mb-seg-357, id=8 removed by soft count = 104\n",
      "--- number of probes: 74 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-358\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-359\n",
      "--- gene=Chr7-6Mb-seg-359, id=24 removed by soft count = 158\n",
      "--- number of probes: 79 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-360\n",
      "--- gene=Chr7-6Mb-seg-360, id=22 removed by soft count = 79\n",
      "--- number of probes: 73 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-361\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-362\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-363\n",
      "--- gene=Chr7-6Mb-seg-363, id=19 removed by soft count = 56\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-363, id=54 removed by soft count = 105\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-364\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 56\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-365\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-366\n",
      "--- gene=Chr7-6Mb-seg-366, id=59 removed by soft count = 582\n",
      "--- gene=Chr7-6Mb-seg-366, id=60 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-366, id=61 removed by soft count = 66\n",
      "--- number of probes: 80 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-367\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-368\n",
      "--- gene=Chr7-6Mb-seg-368, id=15 removed by soft count = 140\n",
      "--- gene=Chr7-6Mb-seg-368, id=16 removed by soft count = 161\n",
      "--- number of probes: 56 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-369\n",
      "--- gene=Chr7-6Mb-seg-369, id=0 removed by soft count = 247\n",
      "--- gene=Chr7-6Mb-seg-369, id=1 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-369, id=3 removed by soft count = 61\n",
      "--- number of probes: 79 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-370\n",
      "--- number of probes: 53 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-371\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-372\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-373\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-374\n",
      "--- gene=Chr7-6Mb-seg-374, id=28 removed by soft count = 161\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-375\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-376\n",
      "--- gene=Chr7-6Mb-seg-376, id=37 removed by soft count = 242\n",
      "--- gene=Chr7-6Mb-seg-376, id=38 removed by soft count = 393\n",
      "--- gene=Chr7-6Mb-seg-376, id=39 removed by soft count = 188\n",
      "--- gene=Chr7-6Mb-seg-376, id=44 removed by soft count = 196\n",
      "--- number of probes: 46 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: Chr7-6Mb-seg-377\n",
      "--- gene=Chr7-6Mb-seg-377, id=1 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-377, id=2 removed by soft count = 431\n",
      "--- gene=Chr7-6Mb-seg-377, id=3 removed by soft count = 2244\n",
      "--- number of probes: 64 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-378\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-379\n",
      "--- gene=Chr7-6Mb-seg-379, id=14 removed by soft count = 100\n",
      "--- gene=Chr7-6Mb-seg-379, id=61 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-379, id=62 removed by soft count = 606\n",
      "--- gene=Chr7-6Mb-seg-379, id=63 removed by soft count = 149\n",
      "--- gene=Chr7-6Mb-seg-379, id=65 removed by soft count = 250\n",
      "--- number of probes: 73 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-380\n",
      "--- gene=Chr7-6Mb-seg-380, id=60 removed by soft count = 264\n",
      "--- gene=Chr7-6Mb-seg-380, id=66 removed by soft count = 1111\n",
      "--- number of probes: 75 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-381\n",
      "--- gene=Chr7-6Mb-seg-381, id=47 removed by soft count = 1765\n",
      "--- number of probes: 55 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-382\n",
      "--- gene=Chr7-6Mb-seg-382, id=6 removed by soft count = 165\n",
      "--- gene=Chr7-6Mb-seg-382, id=7 removed by soft count = 201\n",
      "--- number of probes: 60 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-383\n",
      "--- gene=Chr7-6Mb-seg-383, id=4 removed by soft count = 634\n",
      "--- gene=Chr7-6Mb-seg-383, id=6 removed by soft count = 129\n",
      "--- gene=Chr7-6Mb-seg-383, id=7 removed by soft count = 125\n",
      "--- gene=Chr7-6Mb-seg-383, id=8 removed by soft count = 600\n",
      "--- gene=Chr7-6Mb-seg-383, id=9 removed by soft count = 989\n",
      "--- gene=Chr7-6Mb-seg-383, id=10 removed by soft count = 426\n",
      "--- gene=Chr7-6Mb-seg-383, id=11 removed by soft count = 101\n",
      "--- number of probes: 44 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: Chr7-6Mb-seg-384\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-385\n",
      "--- gene=Chr7-6Mb-seg-385, id=62 removed by soft count = 147\n",
      "--- gene=Chr7-6Mb-seg-385, id=63 removed by soft count = 191\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-386\n",
      "--- gene=Chr7-6Mb-seg-386, id=3 removed by soft count = 576\n",
      "--- gene=Chr7-6Mb-seg-386, id=4 removed by soft count = 1038\n",
      "--- gene=Chr7-6Mb-seg-386, id=5 removed by soft count = 141\n",
      "--- gene=Chr7-6Mb-seg-386, id=6 removed by soft count = 124\n",
      "--- gene=Chr7-6Mb-seg-386, id=8 removed by soft count = 142\n",
      "--- gene=Chr7-6Mb-seg-386, id=10 removed by soft count = 131\n",
      "--- gene=Chr7-6Mb-seg-386, id=11 removed by soft count = 669\n",
      "--- gene=Chr7-6Mb-seg-386, id=12 removed by soft count = 432\n",
      "--- gene=Chr7-6Mb-seg-386, id=14 removed by soft count = 1098\n",
      "--- number of probes: 42 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: Chr7-6Mb-seg-387\n",
      "--- gene=Chr7-6Mb-seg-387, id=10 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-387, id=11 removed by soft count = 601\n",
      "--- gene=Chr7-6Mb-seg-387, id=13 removed by soft count = 121\n",
      "--- gene=Chr7-6Mb-seg-387, id=16 removed by soft count = 1967\n",
      "--- gene=Chr7-6Mb-seg-387, id=17 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-387, id=19 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-387, id=24 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-387, id=26 removed by soft count = 17833\n",
      "--- number of probes: 42 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-388\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-389\n",
      "--- gene=Chr7-6Mb-seg-389, id=61 removed by soft count = 135\n",
      "--- number of probes: 64 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-390\n",
      "--- gene=Chr7-6Mb-seg-390, id=1 removed by soft count = 302\n",
      "--- gene=Chr7-6Mb-seg-390, id=3 removed by soft count = 1621\n",
      "--- gene=Chr7-6Mb-seg-390, id=65 removed by soft count = 303\n",
      "--- gene=Chr7-6Mb-seg-390, id=67 removed by soft count = 151\n",
      "--- gene=Chr7-6Mb-seg-390, id=68 removed by soft count = 174\n",
      "--- gene=Chr7-6Mb-seg-390, id=69 removed by soft count = 108\n",
      "--- gene=Chr7-6Mb-seg-390, id=70 removed by soft count = 67\n",
      "--- gene=Chr7-6Mb-seg-390, id=71 removed by soft count = 70\n",
      "--- number of probes: 76 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-391\n",
      "--- gene=Chr7-6Mb-seg-391, id=24 removed by soft count = 90\n",
      "--- number of probes: 35 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-392\n",
      "--- gene=Chr7-6Mb-seg-392, id=36 removed by soft count = 482\n",
      "--- gene=Chr7-6Mb-seg-392, id=37 removed by soft count = 214\n",
      "--- gene=Chr7-6Mb-seg-392, id=38 removed by soft count = 599\n",
      "--- gene=Chr7-6Mb-seg-392, id=39 removed by soft count = 735\n",
      "--- gene=Chr7-6Mb-seg-392, id=40 removed by soft count = 113\n",
      "--- gene=Chr7-6Mb-seg-392, id=41 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-392, id=43 removed by soft count = 397\n",
      "--- gene=Chr7-6Mb-seg-392, id=44 removed by soft count = 916\n",
      "--- number of probes: 71 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-393\n",
      "--- gene=Chr7-6Mb-seg-393, id=17 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-393, id=27 removed by soft count = 385\n",
      "--- gene=Chr7-6Mb-seg-393, id=29 removed by soft count = 933\n",
      "--- gene=Chr7-6Mb-seg-393, id=41 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-393, id=42 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-393, id=45 removed by soft count = 135\n",
      "--- number of probes: 46 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: Chr7-6Mb-seg-394\n",
      "--- gene=Chr7-6Mb-seg-394, id=9 removed by soft count = 3852\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-395\n",
      "--- gene=Chr7-6Mb-seg-395, id=32 removed by soft count = 216\n",
      "--- gene=Chr7-6Mb-seg-395, id=34 removed by soft count = 225\n",
      "--- number of probes: 76 , kept by blast: 74 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-396\n",
      "--- gene=Chr7-6Mb-seg-396, id=19 removed by soft count = 81\n",
      "--- gene=Chr7-6Mb-seg-396, id=53 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-396, id=63 removed by soft count = 145\n",
      "--- number of probes: 74 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-397\n",
      "--- gene=Chr7-6Mb-seg-397, id=18 removed by soft count = 76\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-398\n",
      "--- gene=Chr7-6Mb-seg-398, id=17 removed by soft count = 359\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-399\n",
      "--- gene=Chr7-6Mb-seg-399, id=12 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-399, id=51 removed by soft count = 269\n",
      "--- number of probes: 74 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-400\n",
      "--- gene=Chr7-6Mb-seg-400, id=16 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-seg-400, id=76 removed by soft count = 144\n",
      "--- number of probes: 79 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-401\n",
      "--- gene=Chr7-6Mb-seg-401, id=23 removed by soft count = 201\n",
      "--- gene=Chr7-6Mb-seg-401, id=52 removed by soft count = 70\n",
      "--- number of probes: 62 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-402\n",
      "--- gene=Chr7-6Mb-seg-402, id=30 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-402, id=31 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-402, id=33 removed by soft count = 68\n",
      "--- gene=Chr7-6Mb-seg-402, id=35 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-402, id=40 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-402, id=44 removed by soft count = 97\n",
      "--- number of probes: 75 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-403\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-404\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-405\n",
      "--- gene=Chr7-6Mb-seg-405, id=4 removed by soft count = 113\n",
      "--- gene=Chr7-6Mb-seg-405, id=37 removed by hard count = 0\n",
      "--- number of probes: 62 , kept by blast: 60 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-406\n",
      "--- gene=Chr7-6Mb-seg-406, id=0 removed by soft count = 406\n",
      "--- gene=Chr7-6Mb-seg-406, id=2 removed by soft count = 171\n",
      "--- gene=Chr7-6Mb-seg-406, id=3 removed by soft count = 127\n",
      "--- gene=Chr7-6Mb-seg-406, id=20 removed by soft count = 269\n",
      "--- gene=Chr7-6Mb-seg-406, id=26 removed by soft count = 51\n",
      "--- number of probes: 64 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-407\n",
      "--- gene=Chr7-6Mb-seg-407, id=35 removed by soft count = 89\n",
      "--- gene=Chr7-6Mb-seg-407, id=37 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-407, id=40 removed by soft count = 97\n",
      "--- number of probes: 75 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-408\n",
      "--- gene=Chr7-6Mb-seg-408, id=69 removed by soft count = 52\n",
      "--- number of probes: 72 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-409\n",
      "--- gene=Chr7-6Mb-seg-409, id=0 removed by soft count = 124\n",
      "--- number of probes: 74 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-410\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-411\n",
      "--- gene=Chr7-6Mb-seg-411, id=42 removed by soft count = 80\n",
      "--- number of probes: 63 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-412\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-412, id=19 removed by soft count = 78\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-413\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-414\n",
      "--- gene=Chr7-6Mb-seg-414, id=10 removed by soft count = 762\n",
      "--- gene=Chr7-6Mb-seg-414, id=22 removed by soft count = 142\n",
      "--- gene=Chr7-6Mb-seg-414, id=23 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-414, id=27 removed by soft count = 937\n",
      "--- gene=Chr7-6Mb-seg-414, id=29 removed by soft count = 547\n",
      "--- gene=Chr7-6Mb-seg-414, id=30 removed by soft count = 262\n",
      "--- number of probes: 72 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-415\n",
      "--- gene=Chr7-6Mb-seg-415, id=1 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-415, id=18 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-415, id=36 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-415, id=39 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-415, id=42 removed by soft count = 128\n",
      "--- gene=Chr7-6Mb-seg-415, id=43 removed by soft count = 306\n",
      "--- gene=Chr7-6Mb-seg-415, id=48 removed by soft count = 153\n",
      "--- gene=Chr7-6Mb-seg-415, id=57 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-415, id=60 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-415, id=63 removed by soft count = 51\n",
      "--- number of probes: 70 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-416\n",
      "--- gene=Chr7-6Mb-seg-416, id=22 removed by soft count = 120\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-417\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-418\n",
      "--- gene=Chr7-6Mb-seg-418, id=2 removed by soft count = 250\n",
      "--- gene=Chr7-6Mb-seg-418, id=3 removed by soft count = 421\n",
      "--- gene=Chr7-6Mb-seg-418, id=4 removed by soft count = 1728\n",
      "--- gene=Chr7-6Mb-seg-418, id=5 removed by soft count = 866\n",
      "--- gene=Chr7-6Mb-seg-418, id=6 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-418, id=21 removed by soft count = 130\n",
      "--- gene=Chr7-6Mb-seg-418, id=22 removed by soft count = 240\n",
      "--- gene=Chr7-6Mb-seg-418, id=24 removed by soft count = 279\n",
      "--- gene=Chr7-6Mb-seg-418, id=25 removed by soft count = 113\n",
      "--- gene=Chr7-6Mb-seg-418, id=26 removed by soft count = 418\n",
      "--- gene=Chr7-6Mb-seg-418, id=28 removed by soft count = 359\n",
      "--- gene=Chr7-6Mb-seg-418, id=29 removed by soft count = 228\n",
      "--- number of probes: 70 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-419\n",
      "--- gene=Chr7-6Mb-seg-419, id=51 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-419, id=55 removed by soft count = 253\n",
      "--- gene=Chr7-6Mb-seg-419, id=59 removed by soft count = 487\n",
      "--- gene=Chr7-6Mb-seg-419, id=61 removed by soft count = 238\n",
      "--- number of probes: 65 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-420\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-421\n",
      "--- gene=Chr7-6Mb-seg-421, id=40 removed by soft count = 303\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-422\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-423\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-424\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-425\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-426\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-427\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-428\n",
      "--- gene=Chr7-6Mb-seg-428, id=37 removed by soft count = 69\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-429\n",
      "--- gene=Chr7-6Mb-seg-429, id=59 removed by soft count = 492\n",
      "--- number of probes: 65 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-430\n",
      "--- number of probes: 74 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-431\n",
      "--- gene=Chr7-6Mb-seg-431, id=9 removed by soft count = 88\n",
      "--- number of probes: 79 , kept by blast: 78 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-432\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-433\n",
      "--- gene=Chr7-6Mb-seg-433, id=55 removed by soft count = 93\n",
      "--- number of probes: 73 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-434\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-435\n",
      "--- gene=Chr7-6Mb-seg-435, id=35 removed by soft count = 63\n",
      "--- number of probes: 72 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-436\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-437\n",
      "--- gene=Chr7-6Mb-seg-437, id=12 removed by soft count = 120\n",
      "--- gene=Chr7-6Mb-seg-437, id=59 removed by soft count = 74\n",
      "--- number of probes: 75 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-438\n",
      "--- gene=Chr7-6Mb-seg-438, id=21 removed by soft count = 323\n",
      "--- gene=Chr7-6Mb-seg-438, id=38 removed by soft count = 2000\n",
      "--- gene=Chr7-6Mb-seg-438, id=40 removed by soft count = 509\n",
      "--- gene=Chr7-6Mb-seg-438, id=41 removed by soft count = 52\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 66 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-439\n",
      "--- gene=Chr7-6Mb-seg-439, id=29 removed by soft count = 177\n",
      "--- gene=Chr7-6Mb-seg-439, id=30 removed by soft count = 422\n",
      "--- gene=Chr7-6Mb-seg-439, id=31 removed by soft count = 436\n",
      "--- gene=Chr7-6Mb-seg-439, id=32 removed by soft count = 136\n",
      "--- gene=Chr7-6Mb-seg-439, id=33 removed by soft count = 199\n",
      "--- number of probes: 69 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-440\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-441\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-442\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-443\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-444\n",
      "--- gene=Chr7-6Mb-seg-444, id=41 removed by soft count = 55\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-445\n",
      "--- gene=Chr7-6Mb-seg-445, id=74 removed by soft count = 853\n",
      "--- number of probes: 78 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-446\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-447\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-448\n",
      "--- gene=Chr7-6Mb-seg-448, id=72 removed by soft count = 124\n",
      "--- number of probes: 76 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-449\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-450\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-451\n",
      "--- gene=Chr7-6Mb-seg-451, id=31 removed by soft count = 233\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-452\n",
      "--- gene=Chr7-6Mb-seg-452, id=17 removed by soft count = 193\n",
      "--- gene=Chr7-6Mb-seg-452, id=18 removed by soft count = 133\n",
      "--- gene=Chr7-6Mb-seg-452, id=20 removed by soft count = 854\n",
      "--- gene=Chr7-6Mb-seg-452, id=21 removed by soft count = 402\n",
      "--- number of probes: 75 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-453\n",
      "--- gene=Chr7-6Mb-seg-453, id=74 removed by soft count = 65\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-454\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-455\n",
      "--- gene=Chr7-6Mb-seg-455, id=13 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-455, id=28 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-455, id=31 removed by soft count = 98\n",
      "--- gene=Chr7-6Mb-seg-455, id=42 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-455, id=43 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-455, id=49 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-455, id=55 removed by soft count = 54\n",
      "--- number of probes: 80 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-456\n",
      "--- gene=Chr7-6Mb-seg-456, id=17 removed by soft count = 166\n",
      "--- gene=Chr7-6Mb-seg-456, id=72 removed by soft count = 52\n",
      "--- number of probes: 75 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-457\n",
      "--- gene=Chr7-6Mb-seg-457, id=6 removed by soft count = 99\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-458\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-459\n",
      "--- gene=Chr7-6Mb-seg-459, id=60 removed by soft count = 74\n",
      "--- number of probes: 72 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-460\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-461\n",
      "--- gene=Chr7-6Mb-seg-461, id=29 removed by soft count = 136\n",
      "--- gene=Chr7-6Mb-seg-461, id=30 removed by soft count = 404\n",
      "--- gene=Chr7-6Mb-seg-461, id=31 removed by soft count = 363\n",
      "--- gene=Chr7-6Mb-seg-461, id=32 removed by soft count = 185\n",
      "--- gene=Chr7-6Mb-seg-461, id=34 removed by soft count = 107\n",
      "--- number of probes: 75 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-462\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-463\n",
      "--- gene=Chr7-6Mb-seg-463, id=39 removed by soft count = 60\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-464\n",
      "--- gene=Chr7-6Mb-seg-464, id=54 removed by soft count = 171\n",
      "--- number of probes: 73 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-465\n",
      "--- gene=Chr7-6Mb-seg-465, id=11 removed by soft count = 203\n",
      "--- gene=Chr7-6Mb-seg-465, id=32 removed by soft count = 188\n",
      "--- gene=Chr7-6Mb-seg-465, id=33 removed by soft count = 100\n",
      "--- number of probes: 66 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-466\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-467\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-467, id=13 removed by soft count = 1863\n",
      "--- gene=Chr7-6Mb-seg-467, id=17 removed by soft count = 241\n",
      "--- gene=Chr7-6Mb-seg-467, id=20 removed by soft count = 203\n",
      "--- gene=Chr7-6Mb-seg-467, id=21 removed by soft count = 1248\n",
      "--- gene=Chr7-6Mb-seg-467, id=22 removed by soft count = 910\n",
      "--- gene=Chr7-6Mb-seg-467, id=23 removed by soft count = 296\n",
      "--- gene=Chr7-6Mb-seg-467, id=24 removed by soft count = 610\n",
      "--- gene=Chr7-6Mb-seg-467, id=31 removed by soft count = 109\n",
      "--- number of probes: 55 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: Chr7-6Mb-seg-468\n",
      "--- gene=Chr7-6Mb-seg-468, id=10 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-468, id=11 removed by soft count = 542\n",
      "--- gene=Chr7-6Mb-seg-468, id=12 removed by soft count = 121\n",
      "--- number of probes: 75 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-469\n",
      "--- gene=Chr7-6Mb-seg-469, id=60 removed by soft count = 418\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-470\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-471\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-472\n",
      "--- gene=Chr7-6Mb-seg-472, id=9 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-472, id=13 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-472, id=14 removed by soft count = 95\n",
      "--- gene=Chr7-6Mb-seg-472, id=15 removed by soft count = 304\n",
      "--- gene=Chr7-6Mb-seg-472, id=20 removed by soft count = 66\n",
      "--- number of probes: 72 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-473\n",
      "--- number of probes: 74 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-474\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-475\n",
      "--- gene=Chr7-6Mb-seg-475, id=7 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-475, id=8 removed by soft count = 364\n",
      "--- gene=Chr7-6Mb-seg-475, id=49 removed by soft count = 54\n",
      "--- number of probes: 74 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-476\n",
      "--- gene=Chr7-6Mb-seg-476, id=72 removed by soft count = 56\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-477\n",
      "--- gene=Chr7-6Mb-seg-477, id=26 removed by soft count = 1159\n",
      "--- gene=Chr7-6Mb-seg-477, id=28 removed by soft count = 1895\n",
      "--- number of probes: 68 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-478\n",
      "--- gene=Chr7-6Mb-seg-478, id=1 removed by soft count = 125\n",
      "--- gene=Chr7-6Mb-seg-478, id=3 removed by soft count = 87\n",
      "--- gene=Chr7-6Mb-seg-478, id=5 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-478, id=26 removed by soft count = 134\n",
      "--- gene=Chr7-6Mb-seg-478, id=27 removed by soft count = 51\n",
      "--- number of probes: 51 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-479\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-480\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-481\n",
      "--- gene=Chr7-6Mb-seg-481, id=12 removed by soft count = 53\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-482\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-483\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-484\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-485\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-486\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-487\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-488\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-489\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-490\n",
      "--- gene=Chr7-6Mb-seg-490, id=37 removed by soft count = 250\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-491\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-492\n",
      "--- gene=Chr7-6Mb-seg-492, id=1 removed by soft count = 391\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-493\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-494\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-495\n",
      "--- gene=Chr7-6Mb-seg-495, id=22 removed by soft count = 98\n",
      "--- gene=Chr7-6Mb-seg-495, id=24 removed by soft count = 162\n",
      "--- number of probes: 75 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-496\n",
      "--- gene=Chr7-6Mb-seg-496, id=18 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-496, id=19 removed by soft count = 301\n",
      "--- gene=Chr7-6Mb-seg-496, id=20 removed by soft count = 140\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-496, id=40 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-496, id=41 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-496, id=42 removed by soft count = 242\n",
      "--- number of probes: 80 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-497\n",
      "--- gene=Chr7-6Mb-seg-497, id=13 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-497, id=61 removed by soft count = 70\n",
      "--- number of probes: 74 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-498\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-499\n",
      "--- gene=Chr7-6Mb-seg-499, id=58 removed by soft count = 111\n",
      "--- number of probes: 74 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-500\n",
      "--- gene=Chr7-6Mb-seg-500, id=19 removed by soft count = 162\n",
      "--- number of probes: 68 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-501\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-502\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-503\n",
      "--- gene=Chr7-6Mb-seg-503, id=25 removed by soft count = 114\n",
      "--- number of probes: 63 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-504\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-505\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-506\n",
      "--- gene=Chr7-6Mb-seg-506, id=9 removed by soft count = 114\n",
      "--- number of probes: 73 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-507\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-508\n",
      "--- number of probes: 76 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-509\n",
      "--- gene=Chr7-6Mb-seg-509, id=16 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-509, id=33 removed by soft count = 170\n",
      "--- gene=Chr7-6Mb-seg-509, id=34 removed by soft count = 269\n",
      "--- gene=Chr7-6Mb-seg-509, id=35 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-509, id=36 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-509, id=41 removed by soft count = 122\n",
      "--- number of probes: 62 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-510\n",
      "--- gene=Chr7-6Mb-seg-510, id=52 removed by soft count = 95\n",
      "--- number of probes: 76 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-511\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-512\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-513\n",
      "--- gene=Chr7-6Mb-seg-513, id=32 removed by soft count = 104\n",
      "--- number of probes: 76 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-514\n",
      "--- gene=Chr7-6Mb-seg-514, id=8 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-514, id=34 removed by soft count = 184\n",
      "--- gene=Chr7-6Mb-seg-514, id=67 removed by soft count = 162\n",
      "--- number of probes: 68 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-515\n",
      "--- gene=Chr7-6Mb-seg-515, id=49 removed by soft count = 55\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-516\n",
      "--- gene=Chr7-6Mb-seg-516, id=1 removed by soft count = 155\n",
      "--- gene=Chr7-6Mb-seg-516, id=2 removed by soft count = 2599\n",
      "--- number of probes: 71 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-517\n",
      "--- gene=Chr7-6Mb-seg-517, id=4 removed by soft count = 56\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-518\n",
      "--- gene=Chr7-6Mb-seg-518, id=69 removed by soft count = 233\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-519\n",
      "--- gene=Chr7-6Mb-seg-519, id=34 removed by soft count = 92\n",
      "--- gene=Chr7-6Mb-seg-519, id=36 removed by soft count = 364\n",
      "--- number of probes: 69 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-520\n",
      "--- gene=Chr7-6Mb-seg-520, id=27 removed by soft count = 88\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-521\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-522\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-523\n",
      "--- gene=Chr7-6Mb-seg-523, id=4 removed by soft count = 226\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-524\n",
      "--- gene=Chr7-6Mb-seg-524, id=3 removed by soft count = 61\n",
      "--- number of probes: 72 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-525\n",
      "--- number of probes: 63 , kept by blast: 63 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-526\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-527\n",
      "--- gene=Chr7-6Mb-seg-527, id=11 removed by soft count = 426\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-528\n",
      "--- gene=Chr7-6Mb-seg-528, id=14 removed by soft count = 205\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-529\n",
      "--- gene=Chr7-6Mb-seg-529, id=19 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-529, id=42 removed by soft count = 579\n",
      "--- gene=Chr7-6Mb-seg-529, id=43 removed by soft count = 171\n",
      "--- gene=Chr7-6Mb-seg-529, id=44 removed by soft count = 113\n",
      "--- gene=Chr7-6Mb-seg-529, id=45 removed by soft count = 79\n",
      "--- gene=Chr7-6Mb-seg-529, id=48 removed by soft count = 112\n",
      "--- number of probes: 68 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-530\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-531\n",
      "--- gene=Chr7-6Mb-seg-531, id=21 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-531, id=22 removed by soft count = 153\n",
      "--- gene=Chr7-6Mb-seg-531, id=38 removed by hard count = 0\n",
      "--- number of probes: 65 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-532\n",
      "--- gene=Chr7-6Mb-seg-532, id=18 removed by soft count = 1709\n",
      "--- gene=Chr7-6Mb-seg-532, id=23 removed by soft count = 190\n",
      "--- number of probes: 73 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-533\n",
      "--- gene=Chr7-6Mb-seg-533, id=62 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-533, id=72 removed by soft count = 97\n",
      "--- gene=Chr7-6Mb-seg-533, id=73 removed by soft count = 152\n",
      "--- number of probes: 74 , kept by blast: 71 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-534\n",
      "--- gene=Chr7-6Mb-seg-534, id=72 removed by soft count = 72\n",
      "--- number of probes: 79 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-535\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-536\n",
      "--- gene=Chr7-6Mb-seg-536, id=1 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-536, id=19 removed by soft count = 462\n",
      "--- gene=Chr7-6Mb-seg-536, id=20 removed by soft count = 75\n",
      "--- gene=Chr7-6Mb-seg-536, id=37 removed by soft count = 1642\n",
      "--- gene=Chr7-6Mb-seg-536, id=46 removed by soft count = 193\n",
      "--- number of probes: 62 , kept by blast: 57 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: Chr7-6Mb-seg-537\n",
      "--- gene=Chr7-6Mb-seg-537, id=48 removed by soft count = 318\n",
      "--- gene=Chr7-6Mb-seg-537, id=49 removed by soft count = 132\n",
      "--- number of probes: 70 , kept by blast: 68 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-538\n",
      "--- gene=Chr7-6Mb-seg-538, id=4 removed by soft count = 300\n",
      "--- gene=Chr7-6Mb-seg-538, id=64 removed by soft count = 351\n",
      "--- gene=Chr7-6Mb-seg-538, id=65 removed by soft count = 193\n",
      "--- number of probes: 80 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-539\n",
      "--- gene=Chr7-6Mb-seg-539, id=11 removed by soft count = 1109\n",
      "--- gene=Chr7-6Mb-seg-539, id=21 removed by soft count = 565\n",
      "--- gene=Chr7-6Mb-seg-539, id=31 removed by soft count = 682\n",
      "--- gene=Chr7-6Mb-seg-539, id=32 removed by soft count = 186\n",
      "--- number of probes: 33 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: Chr7-6Mb-seg-540\n",
      "--- gene=Chr7-6Mb-seg-540, id=1 removed by soft count = 172\n",
      "--- gene=Chr7-6Mb-seg-540, id=18 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-540, id=21 removed by soft count = 98\n",
      "--- gene=Chr7-6Mb-seg-540, id=22 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-540, id=48 removed by soft count = 372\n",
      "--- number of probes: 49 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-541\n",
      "--- gene=Chr7-6Mb-seg-541, id=43 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-541, id=44 removed by soft count = 53\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-542\n",
      "--- gene=Chr7-6Mb-seg-542, id=49 removed by soft count = 350\n",
      "--- gene=Chr7-6Mb-seg-542, id=54 removed by soft count = 323\n",
      "--- gene=Chr7-6Mb-seg-542, id=55 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-542, id=56 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-542, id=57 removed by soft count = 128\n",
      "--- number of probes: 67 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-543\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-544\n",
      "--- gene=Chr7-6Mb-seg-544, id=49 removed by soft count = 214\n",
      "--- gene=Chr7-6Mb-seg-544, id=57 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-544, id=59 removed by soft count = 185\n",
      "--- gene=Chr7-6Mb-seg-544, id=64 removed by soft count = 53\n",
      "--- number of probes: 77 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-545\n",
      "--- gene=Chr7-6Mb-seg-545, id=32 removed by soft count = 686\n",
      "--- gene=Chr7-6Mb-seg-545, id=39 removed by soft count = 115\n",
      "--- gene=Chr7-6Mb-seg-545, id=41 removed by soft count = 74\n",
      "--- number of probes: 63 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-546\n",
      "--- gene=Chr7-6Mb-seg-546, id=3 removed by soft count = 297\n",
      "--- gene=Chr7-6Mb-seg-546, id=6 removed by soft count = 255\n",
      "--- gene=Chr7-6Mb-seg-546, id=16 removed by soft count = 106\n",
      "--- number of probes: 66 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-547\n",
      "--- gene=Chr7-6Mb-seg-547, id=29 removed by soft count = 1892\n",
      "--- gene=Chr7-6Mb-seg-547, id=48 removed by soft count = 55\n",
      "--- number of probes: 75 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-548\n",
      "--- gene=Chr7-6Mb-seg-548, id=62 removed by soft count = 83\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-549\n",
      "--- gene=Chr7-6Mb-seg-549, id=44 removed by soft count = 4651\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-549, id=58 removed by soft count = 865\n",
      "--- gene=Chr7-6Mb-seg-549, id=59 removed by soft count = 183\n",
      "--- gene=Chr7-6Mb-seg-549, id=60 removed by soft count = 49973\n",
      "--- number of probes: 62 , kept by blast: 58 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-550\n",
      "--- gene=Chr7-6Mb-seg-550, id=62 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-550, id=66 removed by soft count = 55\n",
      "--- number of probes: 79 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-551\n",
      "--- gene=Chr7-6Mb-seg-551, id=15 removed by soft count = 2526\n",
      "--- gene=Chr7-6Mb-seg-551, id=78 removed by soft count = 127\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-552\n",
      "--- gene=Chr7-6Mb-seg-552, id=12 removed by soft count = 598\n",
      "--- gene=Chr7-6Mb-seg-552, id=59 removed by soft count = 73\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-553\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-554\n",
      "--- gene=Chr7-6Mb-seg-554, id=1 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-554, id=6 removed by soft count = 89\n",
      "--- gene=Chr7-6Mb-seg-554, id=8 removed by soft count = 145\n",
      "--- gene=Chr7-6Mb-seg-554, id=10 removed by soft count = 68\n",
      "--- gene=Chr7-6Mb-seg-554, id=11 removed by soft count = 262\n",
      "--- gene=Chr7-6Mb-seg-554, id=23 removed by soft count = 91\n",
      "--- number of probes: 63 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Chr7-6Mb-seg-555\n",
      "--- gene=Chr7-6Mb-seg-555, id=56 removed by soft count = 244\n",
      "--- number of probes: 68 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-556\n",
      "--- gene=Chr7-6Mb-seg-556, id=38 removed by soft count = 90\n",
      "--- number of probes: 73 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-557\n",
      "--- gene=Chr7-6Mb-seg-557, id=10 removed by soft count = 313\n",
      "--- gene=Chr7-6Mb-seg-557, id=28 removed by soft count = 319\n",
      "--- gene=Chr7-6Mb-seg-557, id=51 removed by soft count = 86\n",
      "--- number of probes: 62 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-558\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-559\n",
      "--- gene=Chr7-6Mb-seg-559, id=3 removed by soft count = 659\n",
      "--- number of probes: 59 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-560\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-561\n",
      "--- gene=Chr7-6Mb-seg-561, id=32 removed by soft count = 119\n",
      "--- number of probes: 56 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-562\n",
      "--- gene=Chr7-6Mb-seg-562, id=16 removed by soft count = 94\n",
      "--- gene=Chr7-6Mb-seg-562, id=35 removed by soft count = 169\n",
      "--- gene=Chr7-6Mb-seg-562, id=36 removed by soft count = 81\n",
      "--- number of probes: 64 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-563\n",
      "--- gene=Chr7-6Mb-seg-563, id=11 removed by soft count = 2138\n",
      "--- number of probes: 50 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-564\n",
      "--- gene=Chr7-6Mb-seg-564, id=50 removed by soft count = 107\n",
      "--- number of probes: 57 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-565\n",
      "--- gene=Chr7-6Mb-seg-565, id=65 removed by soft count = 280\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-566\n",
      "--- gene=Chr7-6Mb-seg-566, id=19 removed by soft count = 209\n",
      "--- gene=Chr7-6Mb-seg-566, id=23 removed by soft count = 71\n",
      "--- number of probes: 61 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-567\n",
      "--- gene=Chr7-6Mb-seg-567, id=68 removed by soft count = 55\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-568\n",
      "--- gene=Chr7-6Mb-seg-568, id=25 removed by soft count = 65\n",
      "--- number of probes: 72 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-569\n",
      "--- gene=Chr7-6Mb-seg-569, id=33 removed by soft count = 67\n",
      "--- gene=Chr7-6Mb-seg-569, id=70 removed by soft count = 78\n",
      "--- number of probes: 75 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-570\n",
      "--- number of probes: 74 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-571\n",
      "--- gene=Chr7-6Mb-seg-571, id=13 removed by soft count = 62\n",
      "--- number of probes: 65 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-572\n",
      "--- gene=Chr7-6Mb-seg-572, id=18 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-572, id=38 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-572, id=43 removed by soft count = 764\n",
      "--- gene=Chr7-6Mb-seg-572, id=57 removed by soft count = 768\n",
      "--- number of probes: 59 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-573\n",
      "--- gene=Chr7-6Mb-seg-573, id=4 removed by soft count = 437\n",
      "--- number of probes: 54 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-574\n",
      "--- gene=Chr7-6Mb-seg-574, id=24 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-574, id=29 removed by soft count = 89\n",
      "--- gene=Chr7-6Mb-seg-574, id=30 removed by soft count = 111\n",
      "--- number of probes: 48 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: Chr7-6Mb-seg-575\n",
      "--- gene=Chr7-6Mb-seg-575, id=9 removed by soft count = 217\n",
      "--- number of probes: 57 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-576\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-577\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-577, id=6 removed by soft count = 288\n",
      "--- gene=Chr7-6Mb-seg-577, id=9 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-577, id=41 removed by soft count = 67\n",
      "--- number of probes: 52 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-578\n",
      "--- gene=Chr7-6Mb-seg-578, id=8 removed by soft count = 91\n",
      "--- gene=Chr7-6Mb-seg-578, id=11 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-578, id=26 removed by soft count = 1474\n",
      "--- number of probes: 48 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: Chr7-6Mb-seg-579\n",
      "--- gene=Chr7-6Mb-seg-579, id=49 removed by soft count = 181\n",
      "--- number of probes: 65 , kept by blast: 64 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-580\n",
      "--- gene=Chr7-6Mb-seg-580, id=1 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-580, id=45 removed by soft count = 398\n",
      "--- number of probes: 61 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-581\n",
      "--- gene=Chr7-6Mb-seg-581, id=22 removed by soft count = 389\n",
      "--- gene=Chr7-6Mb-seg-581, id=25 removed by soft count = 52\n",
      "--- number of probes: 57 , kept by blast: 55 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-582\n",
      "--- gene=Chr7-6Mb-seg-582, id=66 removed by soft count = 76\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-583\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-584\n",
      "--- gene=Chr7-6Mb-seg-584, id=35 removed by soft count = 161\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-585\n",
      "--- gene=Chr7-6Mb-seg-585, id=0 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-585, id=3 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-585, id=6 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-585, id=9 removed by soft count = 127\n",
      "--- gene=Chr7-6Mb-seg-585, id=12 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-585, id=15 removed by soft count = 720\n",
      "--- gene=Chr7-6Mb-seg-585, id=18 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-585, id=21 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-585, id=24 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-585, id=27 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-585, id=30 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-585, id=33 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-585, id=36 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-585, id=39 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-585, id=42 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-585, id=45 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-585, id=48 removed by soft count = 153\n",
      "--- gene=Chr7-6Mb-seg-585, id=51 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-585, id=54 removed by soft count = 106\n",
      "--- number of probes: 56 , kept by blast: 37 , if remove dups: 29\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: Chr7-6Mb-seg-586\n",
      "--- gene=Chr7-6Mb-seg-586, id=47 removed by soft count = 109\n",
      "--- number of probes: 61 , kept by blast: 60 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: Chr7-6Mb-seg-587\n",
      "--- gene=Chr7-6Mb-seg-587, id=41 removed by soft count = 73\n",
      "--- number of probes: 52 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-588\n",
      "--- gene=Chr7-6Mb-seg-588, id=8 removed by soft count = 60\n",
      "--- number of probes: 62 , kept by blast: 61 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-589\n",
      "--- gene=Chr7-6Mb-seg-589, id=50 removed by soft count = 157\n",
      "--- number of probes: 62 , kept by blast: 61 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-590\n",
      "--- gene=Chr7-6Mb-seg-590, id=13 removed by soft count = 3162\n",
      "--- gene=Chr7-6Mb-seg-590, id=16 removed by soft count = 1742\n",
      "--- gene=Chr7-6Mb-seg-590, id=17 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-590, id=19 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-590, id=47 removed by soft count = 248\n",
      "--- number of probes: 49 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-591\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-592\n",
      "--- gene=Chr7-6Mb-seg-592, id=8 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-592, id=22 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-592, id=71 removed by soft count = 120\n",
      "--- number of probes: 72 , kept by blast: 69 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-593\n",
      "--- gene=Chr7-6Mb-seg-593, id=17 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-593, id=30 removed by soft count = 247\n",
      "--- gene=Chr7-6Mb-seg-593, id=47 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-593, id=52 removed by soft count = 243\n",
      "--- number of probes: 61 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Chr7-6Mb-seg-594\n",
      "--- gene=Chr7-6Mb-seg-594, id=7 removed by soft count = 319\n",
      "--- gene=Chr7-6Mb-seg-594, id=9 removed by soft count = 55\n",
      "--- number of probes: 66 , kept by blast: 64 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-595\n",
      "--- gene=Chr7-6Mb-seg-595, id=2 removed by soft count = 1171\n",
      "--- gene=Chr7-6Mb-seg-595, id=6 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-seg-595, id=7 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-595, id=8 removed by soft count = 328\n",
      "--- gene=Chr7-6Mb-seg-595, id=11 removed by soft count = 81\n",
      "--- gene=Chr7-6Mb-seg-595, id=14 removed by soft count = 114\n",
      "--- gene=Chr7-6Mb-seg-595, id=15 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-595, id=17 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-595, id=29 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-595, id=38 removed by soft count = 403\n",
      "--- number of probes: 39 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: Chr7-6Mb-seg-596\n",
      "--- gene=Chr7-6Mb-seg-596, id=30 removed by soft count = 154\n",
      "--- gene=Chr7-6Mb-seg-596, id=36 removed by soft count = 238\n",
      "--- gene=Chr7-6Mb-seg-596, id=38 removed by soft count = 235\n",
      "--- gene=Chr7-6Mb-seg-596, id=47 removed by soft count = 85\n",
      "--- gene=Chr7-6Mb-seg-596, id=49 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-596, id=50 removed by soft count = 105\n",
      "--- number of probes: 51 , kept by blast: 45 , if remove dups: 43\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: Chr7-6Mb-seg-597\n",
      "--- gene=Chr7-6Mb-seg-597, id=0 removed by soft count = 667\n",
      "--- gene=Chr7-6Mb-seg-597, id=1 removed by soft count = 97\n",
      "--- gene=Chr7-6Mb-seg-597, id=3 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-597, id=7 removed by soft count = 114\n",
      "--- gene=Chr7-6Mb-seg-597, id=10 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-597, id=12 removed by soft count = 284\n",
      "--- number of probes: 55 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-598\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-598, id=48 removed by soft count = 440\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-599\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-600\n",
      "--- gene=Chr7-6Mb-seg-600, id=11 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-600, id=12 removed by soft count = 52\n",
      "--- number of probes: 79 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-601\n",
      "--- number of probes: 74 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-602\n",
      "--- gene=Chr7-6Mb-seg-602, id=2 removed by soft count = 103\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-603\n",
      "--- gene=Chr7-6Mb-seg-603, id=7 removed by soft count = 266\n",
      "--- gene=Chr7-6Mb-seg-603, id=8 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-603, id=9 removed by soft count = 165\n",
      "--- gene=Chr7-6Mb-seg-603, id=10 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-603, id=13 removed by soft count = 233\n",
      "--- gene=Chr7-6Mb-seg-603, id=44 removed by soft count = 20444\n",
      "--- number of probes: 53 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: Chr7-6Mb-seg-604\n",
      "--- gene=Chr7-6Mb-seg-604, id=28 removed by soft count = 206\n",
      "--- gene=Chr7-6Mb-seg-604, id=40 removed by soft count = 857\n",
      "--- gene=Chr7-6Mb-seg-604, id=44 removed by soft count = 361\n",
      "--- number of probes: 57 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-605\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-606\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-607\n",
      "--- gene=Chr7-6Mb-seg-607, id=10 removed by soft count = 83\n",
      "--- number of probes: 53 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-608\n",
      "--- gene=Chr7-6Mb-seg-608, id=16 removed by soft count = 654\n",
      "--- number of probes: 39 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: Chr7-6Mb-seg-609\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-610\n",
      "--- gene=Chr7-6Mb-seg-610, id=45 removed by soft count = 75\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-611\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-612\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-613\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-614\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-615\n",
      "--- gene=Chr7-6Mb-seg-615, id=49 removed by soft count = 280\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-616\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-617\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-618\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-619\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-620\n",
      "--- gene=Chr7-6Mb-seg-620, id=26 removed by soft count = 115\n",
      "--- gene=Chr7-6Mb-seg-620, id=65 removed by soft count = 73\n",
      "--- number of probes: 78 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-621\n",
      "--- gene=Chr7-6Mb-seg-621, id=30 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-621, id=68 removed by soft count = 52\n",
      "--- number of probes: 76 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-622\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-623\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-624\n",
      "--- gene=Chr7-6Mb-seg-624, id=71 removed by soft count = 975\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-625\n",
      "--- gene=Chr7-6Mb-seg-625, id=2 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-625, id=5 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-625, id=8 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-625, id=11 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-625, id=14 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-625, id=17 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-625, id=20 removed by soft count = 122\n",
      "--- gene=Chr7-6Mb-seg-625, id=23 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-625, id=26 removed by soft count = 117\n",
      "--- gene=Chr7-6Mb-seg-625, id=29 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-625, id=32 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-625, id=35 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-625, id=38 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-625, id=41 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-625, id=44 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-625, id=46 removed by soft count = 68\n",
      "--- gene=Chr7-6Mb-seg-625, id=47 removed by soft count = 108\n",
      "--- gene=Chr7-6Mb-seg-625, id=50 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-625, id=53 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-625, id=56 removed by soft count = 102\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-625, id=59 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-625, id=62 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-625, id=65 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-625, id=68 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-625, id=71 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-625, id=74 removed by soft count = 116\n",
      "--- number of probes: 75 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-626\n",
      "--- gene=Chr7-6Mb-seg-626, id=1 removed by soft count = 80\n",
      "--- number of probes: 74 , kept by blast: 73 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-627\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-628\n",
      "--- gene=Chr7-6Mb-seg-628, id=24 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-628, id=26 removed by soft count = 314\n",
      "--- gene=Chr7-6Mb-seg-628, id=49 removed by soft count = 3029\n",
      "--- number of probes: 54 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-629\n",
      "--- gene=Chr7-6Mb-seg-629, id=0 removed by soft count = 113\n",
      "--- number of probes: 74 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-630\n",
      "--- gene=Chr7-6Mb-seg-630, id=4 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-630, id=13 removed by soft count = 158\n",
      "--- number of probes: 68 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-631\n",
      "--- gene=Chr7-6Mb-seg-631, id=6 removed by soft count = 152\n",
      "--- gene=Chr7-6Mb-seg-631, id=16 removed by soft count = 134\n",
      "--- number of probes: 74 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-632\n",
      "--- gene=Chr7-6Mb-seg-632, id=6 removed by soft count = 140\n",
      "--- gene=Chr7-6Mb-seg-632, id=21 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-632, id=34 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-632, id=35 removed by soft count = 98\n",
      "--- number of probes: 57 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-633\n",
      "--- gene=Chr7-6Mb-seg-633, id=0 removed by soft count = 174\n",
      "--- gene=Chr7-6Mb-seg-633, id=2 removed by soft count = 273\n",
      "--- gene=Chr7-6Mb-seg-633, id=69 removed by soft count = 803\n",
      "--- gene=Chr7-6Mb-seg-633, id=70 removed by soft count = 6471\n",
      "--- number of probes: 80 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-634\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-635\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Chr7-6Mb-seg-638\n",
      "--- gene=Chr7-6Mb-seg-638, id=3 removed by soft count = 95\n",
      "--- gene=Chr7-6Mb-seg-638, id=4 removed by soft count = 142\n",
      "--- gene=Chr7-6Mb-seg-638, id=26 removed by soft count = 53\n",
      "--- number of probes: 27 , kept by blast: 24 , if remove dups: 24\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-seg-639\n",
      "--- gene=Chr7-6Mb-seg-639, id=45 removed by soft count = 946\n",
      "--- gene=Chr7-6Mb-seg-639, id=46 removed by soft count = 111\n",
      "--- number of probes: 51 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-640\n",
      "--- number of probes: 24 , kept by blast: 24 , if remove dups: 24\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-seg-641\n",
      "--- gene=Chr7-6Mb-seg-641, id=2 removed by soft count = 354\n",
      "--- number of probes: 37 , kept by blast: 36 , if remove dups: 35\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: Chr7-6Mb-seg-642\n",
      "--- gene=Chr7-6Mb-seg-642, id=22 removed by soft count = 136\n",
      "--- gene=Chr7-6Mb-seg-642, id=37 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-642, id=39 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-642, id=40 removed by soft count = 1134\n",
      "--- gene=Chr7-6Mb-seg-642, id=43 removed by soft count = 194\n",
      "--- gene=Chr7-6Mb-seg-642, id=44 removed by soft count = 284\n",
      "--- number of probes: 54 , kept by blast: 48 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: Chr7-6Mb-seg-643\n",
      "--- gene=Chr7-6Mb-seg-643, id=58 removed by soft count = 111\n",
      "--- number of probes: 59 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-644\n",
      "--- gene=Chr7-6Mb-seg-644, id=0 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-644, id=1 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-644, id=2 removed by soft count = 60\n",
      "--- number of probes: 52 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-647\n",
      "--- gene=Chr7-6Mb-seg-647, id=10 removed by soft count = 848\n",
      "--- gene=Chr7-6Mb-seg-647, id=11 removed by soft count = 485\n",
      "--- gene=Chr7-6Mb-seg-647, id=13 removed by soft count = 538\n",
      "--- gene=Chr7-6Mb-seg-647, id=14 removed by soft count = 795\n",
      "--- gene=Chr7-6Mb-seg-647, id=15 removed by soft count = 1118\n",
      "--- gene=Chr7-6Mb-seg-647, id=16 removed by soft count = 1843\n",
      "--- gene=Chr7-6Mb-seg-647, id=18 removed by soft count = 148\n",
      "--- gene=Chr7-6Mb-seg-647, id=19 removed by soft count = 94\n",
      "--- gene=Chr7-6Mb-seg-647, id=20 removed by soft count = 269\n",
      "--- gene=Chr7-6Mb-seg-647, id=26 removed by hard count = 3\n",
      "--- number of probes: 27 , kept by blast: 17 , if remove dups: 17\n",
      "-- number of probes kept for this region: 17\n",
      "-- checking probes in region: Chr7-6Mb-seg-648\n",
      "--- gene=Chr7-6Mb-seg-648, id=10 removed by soft count = 377\n",
      "--- gene=Chr7-6Mb-seg-648, id=11 removed by soft count = 1052\n",
      "--- gene=Chr7-6Mb-seg-648, id=12 removed by soft count = 479\n",
      "--- gene=Chr7-6Mb-seg-648, id=14 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-648, id=15 removed by soft count = 58\n",
      "--- number of probes: 50 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: Chr7-6Mb-seg-649\n",
      "--- gene=Chr7-6Mb-seg-649, id=8 removed by soft count = 1831\n",
      "--- gene=Chr7-6Mb-seg-649, id=9 removed by soft count = 6260\n",
      "--- gene=Chr7-6Mb-seg-649, id=10 removed by soft count = 7577\n",
      "--- gene=Chr7-6Mb-seg-649, id=12 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-649, id=13 removed by soft count = 261\n",
      "--- number of probes: 26 , kept by blast: 21 , if remove dups: 21\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: Chr7-6Mb-seg-650\n",
      "--- gene=Chr7-6Mb-seg-650, id=33 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-650, id=38 removed by soft count = 335\n",
      "--- gene=Chr7-6Mb-seg-650, id=39 removed by soft count = 448\n",
      "--- number of probes: 40 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: Chr7-6Mb-seg-651\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Chr7-6Mb-seg-655\n",
      "--- gene=Chr7-6Mb-seg-655, id=0 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-655, id=6 removed by hard count = 3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 34 , kept by blast: 32 , if remove dups: 31\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: Chr7-6Mb-seg-660\n",
      "--- gene=Chr7-6Mb-seg-660, id=0 removed by soft count = 419\n",
      "--- gene=Chr7-6Mb-seg-660, id=2 removed by soft count = 1605\n",
      "--- gene=Chr7-6Mb-seg-660, id=5 removed by soft count = 443\n",
      "--- gene=Chr7-6Mb-seg-660, id=21 removed by soft count = 66\n",
      "--- number of probes: 35 , kept by blast: 31 , if remove dups: 31\n",
      "-- number of probes kept for this region: 31\n",
      "-- checking probes in region: Chr7-6Mb-seg-661\n",
      "--- gene=Chr7-6Mb-seg-661, id=16 removed by soft count = 574\n",
      "--- number of probes: 42 , kept by blast: 41 , if remove dups: 31\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: Chr7-6Mb-seg-662\n",
      "--- gene=Chr7-6Mb-seg-662, id=10 removed by soft count = 234\n",
      "--- gene=Chr7-6Mb-seg-662, id=11 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-662, id=12 removed by soft count = 100\n",
      "--- gene=Chr7-6Mb-seg-662, id=14 removed by soft count = 249\n",
      "--- gene=Chr7-6Mb-seg-662, id=15 removed by soft count = 199\n",
      "--- gene=Chr7-6Mb-seg-662, id=16 removed by soft count = 125\n",
      "--- gene=Chr7-6Mb-seg-662, id=17 removed by soft count = 75\n",
      "--- gene=Chr7-6Mb-seg-662, id=18 removed by soft count = 652\n",
      "--- gene=Chr7-6Mb-seg-662, id=20 removed by soft count = 968\n",
      "--- gene=Chr7-6Mb-seg-662, id=23 removed by soft count = 357\n",
      "--- gene=Chr7-6Mb-seg-662, id=27 removed by soft count = 262\n",
      "--- gene=Chr7-6Mb-seg-662, id=28 removed by soft count = 218\n",
      "--- number of probes: 58 , kept by blast: 46 , if remove dups: 42\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: Chr7-6Mb-seg-663\n",
      "--- gene=Chr7-6Mb-seg-663, id=6 removed by soft count = 673\n",
      "--- gene=Chr7-6Mb-seg-663, id=8 removed by soft count = 2625\n",
      "--- gene=Chr7-6Mb-seg-663, id=9 removed by soft count = 11581\n",
      "--- gene=Chr7-6Mb-seg-663, id=11 removed by soft count = 55\n",
      "--- number of probes: 50 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-664\n",
      "--- gene=Chr7-6Mb-seg-664, id=12 removed by soft count = 166\n",
      "--- number of probes: 41 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: Chr7-6Mb-seg-665\n",
      "--- gene=Chr7-6Mb-seg-665, id=7 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-665, id=23 removed by soft count = 98\n",
      "--- gene=Chr7-6Mb-seg-665, id=24 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-665, id=25 removed by soft count = 171\n",
      "--- gene=Chr7-6Mb-seg-665, id=26 removed by soft count = 94\n",
      "--- gene=Chr7-6Mb-seg-665, id=27 removed by soft count = 287\n",
      "--- gene=Chr7-6Mb-seg-665, id=28 removed by soft count = 102\n",
      "--- number of probes: 29 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-seg-668\n",
      "--- gene=Chr7-6Mb-seg-668, id=13 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-668, id=14 removed by soft count = 205\n",
      "--- gene=Chr7-6Mb-seg-668, id=15 removed by soft count = 467\n",
      "--- gene=Chr7-6Mb-seg-668, id=17 removed by soft count = 450\n",
      "--- gene=Chr7-6Mb-seg-668, id=18 removed by soft count = 380\n",
      "--- gene=Chr7-6Mb-seg-668, id=20 removed by soft count = 60\n",
      "--- number of probes: 58 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-669\n",
      "--- gene=Chr7-6Mb-seg-669, id=52 removed by soft count = 352\n",
      "--- gene=Chr7-6Mb-seg-669, id=55 removed by soft count = 1736\n",
      "--- number of probes: 56 , kept by blast: 54 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-670\n",
      "--- gene=Chr7-6Mb-seg-670, id=33 removed by soft count = 64\n",
      "--- number of probes: 68 , kept by blast: 67 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-671\n",
      "--- gene=Chr7-6Mb-seg-671, id=5 removed by soft count = 1255\n",
      "--- gene=Chr7-6Mb-seg-671, id=37 removed by soft count = 63\n",
      "--- number of probes: 54 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-672\n",
      "--- gene=Chr7-6Mb-seg-672, id=2 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-672, id=47 removed by soft count = 98\n",
      "--- number of probes: 51 , kept by blast: 49 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-674\n",
      "--- gene=Chr7-6Mb-seg-674, id=7 removed by soft count = 301\n",
      "--- gene=Chr7-6Mb-seg-674, id=8 removed by soft count = 279\n",
      "--- gene=Chr7-6Mb-seg-674, id=9 removed by soft count = 110\n",
      "--- gene=Chr7-6Mb-seg-674, id=10 removed by soft count = 383\n",
      "--- gene=Chr7-6Mb-seg-674, id=12 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-674, id=13 removed by soft count = 186\n",
      "--- gene=Chr7-6Mb-seg-674, id=14 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-674, id=16 removed by soft count = 198\n",
      "--- gene=Chr7-6Mb-seg-674, id=18 removed by soft count = 375\n",
      "--- gene=Chr7-6Mb-seg-674, id=19 removed by soft count = 241\n",
      "--- gene=Chr7-6Mb-seg-674, id=20 removed by soft count = 185\n",
      "--- gene=Chr7-6Mb-seg-674, id=22 removed by soft count = 223\n",
      "--- gene=Chr7-6Mb-seg-674, id=24 removed by soft count = 218\n",
      "--- gene=Chr7-6Mb-seg-674, id=27 removed by soft count = 180\n",
      "--- gene=Chr7-6Mb-seg-674, id=28 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-674, id=29 removed by soft count = 81\n",
      "--- gene=Chr7-6Mb-seg-674, id=30 removed by soft count = 89\n",
      "--- number of probes: 32 , kept by blast: 15 , if remove dups: 15\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: Chr7-6Mb-seg-675\n",
      "--- gene=Chr7-6Mb-seg-675, id=7 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-675, id=9 removed by soft count = 252\n",
      "--- gene=Chr7-6Mb-seg-675, id=10 removed by soft count = 143\n",
      "--- gene=Chr7-6Mb-seg-675, id=12 removed by soft count = 594\n",
      "--- number of probes: 48 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-676\n",
      "--- gene=Chr7-6Mb-seg-676, id=19 removed by soft count = 2583\n",
      "--- gene=Chr7-6Mb-seg-676, id=20 removed by soft count = 184\n",
      "--- gene=Chr7-6Mb-seg-676, id=21 removed by soft count = 1251\n",
      "--- gene=Chr7-6Mb-seg-676, id=22 removed by soft count = 1841\n",
      "--- gene=Chr7-6Mb-seg-676, id=23 removed by soft count = 11101\n",
      "--- gene=Chr7-6Mb-seg-676, id=30 removed by soft count = 100\n",
      "--- gene=Chr7-6Mb-seg-676, id=31 removed by soft count = 93\n",
      "--- number of probes: 37 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: Chr7-6Mb-seg-677\n",
      "--- gene=Chr7-6Mb-seg-677, id=28 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-677, id=30 removed by soft count = 846\n",
      "--- number of probes: 36 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-679\n",
      "--- gene=Chr7-6Mb-seg-679, id=0 removed by soft count = 185\n",
      "--- number of probes: 51 , kept by blast: 50 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-680\n",
      "--- gene=Chr7-6Mb-seg-680, id=0 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-680, id=1 removed by soft count = 3007\n",
      "--- gene=Chr7-6Mb-seg-680, id=2 removed by soft count = 688\n",
      "--- gene=Chr7-6Mb-seg-680, id=3 removed by soft count = 219\n",
      "--- gene=Chr7-6Mb-seg-680, id=4 removed by soft count = 431\n",
      "--- gene=Chr7-6Mb-seg-680, id=5 removed by soft count = 850\n",
      "--- gene=Chr7-6Mb-seg-680, id=6 removed by soft count = 410\n",
      "--- gene=Chr7-6Mb-seg-680, id=7 removed by soft count = 7288\n",
      "--- number of probes: 37 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: Chr7-6Mb-seg-681\n",
      "--- gene=Chr7-6Mb-seg-681, id=18 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-seg-681, id=22 removed by soft count = 2137\n",
      "--- number of probes: 33 , kept by blast: 31 , if remove dups: 30\n",
      "-- number of probes kept for this region: 31\n",
      "-- checking probes in region: Chr7-6Mb-seg-682\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-682, id=0 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-seg-682, id=1 removed by soft count = 1655\n",
      "--- gene=Chr7-6Mb-seg-682, id=2 removed by soft count = 713\n",
      "--- number of probes: 48 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: Chr7-6Mb-seg-683\n",
      "--- gene=Chr7-6Mb-seg-683, id=5 removed by soft count = 987\n",
      "--- gene=Chr7-6Mb-seg-683, id=6 removed by soft count = 160\n",
      "--- gene=Chr7-6Mb-seg-683, id=8 removed by soft count = 174\n",
      "--- gene=Chr7-6Mb-seg-683, id=20 removed by soft count = 53\n",
      "--- number of probes: 27 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-seg-692\n",
      "--- gene=Chr7-6Mb-seg-692, id=7 removed by soft count = 207\n",
      "--- gene=Chr7-6Mb-seg-692, id=31 removed by soft count = 3559\n",
      "--- number of probes: 32 , kept by blast: 30 , if remove dups: 17\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: Chr7-6Mb-seg-693\n",
      "--- gene=Chr7-6Mb-seg-693, id=0 removed by soft count = 10667\n",
      "--- gene=Chr7-6Mb-seg-693, id=1 removed by soft count = 143\n",
      "--- gene=Chr7-6Mb-seg-693, id=2 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-693, id=4 removed by soft count = 391\n",
      "--- gene=Chr7-6Mb-seg-693, id=5 removed by soft count = 53\n",
      "--- number of probes: 39 , kept by blast: 34 , if remove dups: 6\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-698\n",
      "--- gene=Chr7-6Mb-seg-698, id=1 removed by soft count = 1255\n",
      "--- number of probes: 32 , kept by blast: 31 , if remove dups: 4\n",
      "-- number of probes kept for this region: 31\n",
      "-- checking probes in region: Chr7-6Mb-seg-699\n",
      "--- gene=Chr7-6Mb-seg-699, id=15 removed by soft count = 88\n",
      "--- gene=Chr7-6Mb-seg-699, id=20 removed by soft count = 126\n",
      "--- gene=Chr7-6Mb-seg-699, id=25 removed by soft count = 702\n",
      "--- number of probes: 28 , kept by blast: 25 , if remove dups: 18\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-seg-700\n",
      "--- gene=Chr7-6Mb-seg-700, id=0 removed by soft count = 3076\n",
      "--- gene=Chr7-6Mb-seg-700, id=33 removed by soft count = 219\n",
      "--- gene=Chr7-6Mb-seg-700, id=34 removed by soft count = 234\n",
      "--- gene=Chr7-6Mb-seg-700, id=36 removed by soft count = 2038\n",
      "--- number of probes: 37 , kept by blast: 33 , if remove dups: 32\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: Chr7-6Mb-seg-701\n",
      "--- gene=Chr7-6Mb-seg-701, id=0 removed by soft count = 793\n",
      "--- gene=Chr7-6Mb-seg-701, id=1 removed by soft count = 1045\n",
      "--- gene=Chr7-6Mb-seg-701, id=2 removed by soft count = 165\n",
      "--- gene=Chr7-6Mb-seg-701, id=3 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-701, id=4 removed by soft count = 205\n",
      "--- gene=Chr7-6Mb-seg-701, id=6 removed by soft count = 383\n",
      "--- gene=Chr7-6Mb-seg-701, id=7 removed by soft count = 525\n",
      "--- gene=Chr7-6Mb-seg-701, id=8 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-701, id=28 removed by soft count = 688\n",
      "--- gene=Chr7-6Mb-seg-701, id=30 removed by soft count = 165\n",
      "--- gene=Chr7-6Mb-seg-701, id=31 removed by soft count = 114\n",
      "--- gene=Chr7-6Mb-seg-701, id=33 removed by soft count = 1688\n",
      "--- gene=Chr7-6Mb-seg-701, id=36 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-701, id=37 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-701, id=38 removed by soft count = 57\n",
      "--- number of probes: 41 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: Chr7-6Mb-seg-703\n",
      "--- gene=Chr7-6Mb-seg-703, id=17 removed by soft count = 98\n",
      "--- gene=Chr7-6Mb-seg-703, id=18 removed by soft count = 417\n",
      "--- number of probes: 25 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-seg-704\n",
      "--- gene=Chr7-6Mb-seg-704, id=28 removed by soft count = 129\n",
      "--- gene=Chr7-6Mb-seg-704, id=31 removed by soft count = 52\n",
      "--- number of probes: 41 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: Chr7-6Mb-seg-705\n",
      "--- gene=Chr7-6Mb-seg-705, id=6 removed by soft count = 837\n",
      "--- gene=Chr7-6Mb-seg-705, id=7 removed by soft count = 291\n",
      "--- gene=Chr7-6Mb-seg-705, id=24 removed by soft count = 1017\n",
      "--- gene=Chr7-6Mb-seg-705, id=25 removed by soft count = 212\n",
      "--- number of probes: 31 , kept by blast: 27 , if remove dups: 27\n",
      "-- number of probes kept for this region: 27\n",
      "-- checking probes in region: Chr7-6Mb-seg-706\n",
      "--- gene=Chr7-6Mb-seg-706, id=15 removed by soft count = 1368\n",
      "--- gene=Chr7-6Mb-seg-706, id=19 removed by soft count = 2425\n",
      "--- number of probes: 46 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-707\n",
      "--- gene=Chr7-6Mb-seg-707, id=19 removed by soft count = 397\n",
      "--- gene=Chr7-6Mb-seg-707, id=38 removed by soft count = 207\n",
      "--- number of probes: 55 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-708\n",
      "--- gene=Chr7-6Mb-seg-708, id=13 removed by soft count = 2032\n",
      "--- gene=Chr7-6Mb-seg-708, id=16 removed by soft count = 5806\n",
      "--- number of probes: 31 , kept by blast: 29 , if remove dups: 28\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: Chr7-6Mb-seg-710\n",
      "--- gene=Chr7-6Mb-seg-710, id=0 removed by soft count = 36433\n",
      "--- number of probes: 29 , kept by blast: 28 , if remove dups: 28\n",
      "-- number of probes kept for this region: 28\n",
      "-- checking probes in region: Chr7-6Mb-seg-711\n",
      "--- gene=Chr7-6Mb-seg-711, id=9 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-711, id=10 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-711, id=11 removed by soft count = 177\n",
      "--- gene=Chr7-6Mb-seg-711, id=14 removed by soft count = 1736\n",
      "--- gene=Chr7-6Mb-seg-711, id=15 removed by soft count = 1149\n",
      "--- gene=Chr7-6Mb-seg-711, id=16 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-711, id=18 removed by soft count = 3077\n",
      "--- gene=Chr7-6Mb-seg-711, id=19 removed by soft count = 516\n",
      "--- gene=Chr7-6Mb-seg-711, id=20 removed by soft count = 67\n",
      "--- gene=Chr7-6Mb-seg-711, id=21 removed by soft count = 171\n",
      "--- gene=Chr7-6Mb-seg-711, id=22 removed by soft count = 153\n",
      "--- number of probes: 24 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-seg-712\n",
      "--- gene=Chr7-6Mb-seg-712, id=14 removed by soft count = 95\n",
      "--- gene=Chr7-6Mb-seg-712, id=22 removed by soft count = 92\n",
      "--- number of probes: 27 , kept by blast: 25 , if remove dups: 25\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-seg-713\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-717\n",
      "--- number of probes: 23 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-seg-718\n",
      "--- gene=Chr7-6Mb-seg-718, id=3 removed by soft count = 496\n",
      "--- gene=Chr7-6Mb-seg-718, id=4 removed by soft count = 613\n",
      "--- gene=Chr7-6Mb-seg-718, id=5 removed by soft count = 4170\n",
      "--- gene=Chr7-6Mb-seg-718, id=6 removed by soft count = 18209\n",
      "--- gene=Chr7-6Mb-seg-718, id=7 removed by soft count = 93\n",
      "--- number of probes: 24 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-seg-719\n",
      "--- gene=Chr7-6Mb-seg-719, id=3 removed by soft count = 1921\n",
      "--- gene=Chr7-6Mb-seg-719, id=4 removed by soft count = 158\n",
      "--- gene=Chr7-6Mb-seg-719, id=6 removed by soft count = 304\n",
      "--- gene=Chr7-6Mb-seg-719, id=7 removed by soft count = 1078\n",
      "--- gene=Chr7-6Mb-seg-719, id=8 removed by soft count = 199\n",
      "--- number of probes: 29 , kept by blast: 24 , if remove dups: 8\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-seg-720\n",
      "--- gene=Chr7-6Mb-seg-720, id=0 removed by soft count = 412\n",
      "--- gene=Chr7-6Mb-seg-720, id=38 removed by soft count = 243\n",
      "--- gene=Chr7-6Mb-seg-720, id=40 removed by soft count = 67\n",
      "--- number of probes: 41 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: Chr7-6Mb-seg-721\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-722\n",
      "--- gene=Chr7-6Mb-seg-722, id=8 removed by soft count = 205\n",
      "--- gene=Chr7-6Mb-seg-722, id=10 removed by soft count = 293\n",
      "--- gene=Chr7-6Mb-seg-722, id=11 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-722, id=17 removed by soft count = 124\n",
      "--- gene=Chr7-6Mb-seg-722, id=48 removed by soft count = 6212\n",
      "--- gene=Chr7-6Mb-seg-722, id=49 removed by soft count = 768\n",
      "--- gene=Chr7-6Mb-seg-722, id=50 removed by soft count = 315\n",
      "--- gene=Chr7-6Mb-seg-722, id=51 removed by soft count = 475\n",
      "--- number of probes: 52 , kept by blast: 44 , if remove dups: 43\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-723\n",
      "--- gene=Chr7-6Mb-seg-723, id=0 removed by soft count = 2243\n",
      "--- gene=Chr7-6Mb-seg-723, id=1 removed by soft count = 572\n",
      "--- gene=Chr7-6Mb-seg-723, id=3 removed by soft count = 2034\n",
      "--- gene=Chr7-6Mb-seg-723, id=4 removed by soft count = 274\n",
      "--- gene=Chr7-6Mb-seg-723, id=5 removed by soft count = 90\n",
      "--- gene=Chr7-6Mb-seg-723, id=6 removed by soft count = 145\n",
      "--- gene=Chr7-6Mb-seg-723, id=7 removed by soft count = 125\n",
      "--- number of probes: 32 , kept by blast: 25 , if remove dups: 15\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-seg-726\n",
      "--- gene=Chr7-6Mb-seg-726, id=38 removed by soft count = 86\n",
      "--- number of probes: 39 , kept by blast: 38 , if remove dups: 32\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: Chr7-6Mb-seg-727\n",
      "--- gene=Chr7-6Mb-seg-727, id=2 removed by soft count = 93\n",
      "--- number of probes: 40 , kept by blast: 39 , if remove dups: 38\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: Chr7-6Mb-seg-728\n",
      "--- gene=Chr7-6Mb-seg-728, id=2 removed by soft count = 476\n",
      "--- gene=Chr7-6Mb-seg-728, id=21 removed by soft count = 395\n",
      "--- number of probes: 22 , kept by blast: 20 , if remove dups: 20\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-seg-731\n",
      "--- gene=Chr7-6Mb-seg-731, id=0 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-731, id=13 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-731, id=22 removed by soft count = 146\n",
      "--- gene=Chr7-6Mb-seg-731, id=23 removed by soft count = 623\n",
      "--- gene=Chr7-6Mb-seg-731, id=25 removed by soft count = 127\n",
      "--- gene=Chr7-6Mb-seg-731, id=38 removed by soft count = 170\n",
      "--- gene=Chr7-6Mb-seg-731, id=39 removed by soft count = 438\n",
      "--- gene=Chr7-6Mb-seg-731, id=40 removed by soft count = 132\n",
      "--- gene=Chr7-6Mb-seg-731, id=42 removed by soft count = 145\n",
      "--- gene=Chr7-6Mb-seg-731, id=43 removed by soft count = 138\n",
      "--- gene=Chr7-6Mb-seg-731, id=45 removed by soft count = 187\n",
      "--- gene=Chr7-6Mb-seg-731, id=47 removed by soft count = 125\n",
      "--- gene=Chr7-6Mb-seg-731, id=48 removed by soft count = 131\n",
      "--- gene=Chr7-6Mb-seg-731, id=49 removed by soft count = 264\n",
      "--- gene=Chr7-6Mb-seg-731, id=50 removed by soft count = 75\n",
      "--- number of probes: 51 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: Chr7-6Mb-seg-732\n",
      "--- gene=Chr7-6Mb-seg-732, id=0 removed by soft count = 189\n",
      "--- gene=Chr7-6Mb-seg-732, id=1 removed by soft count = 252\n",
      "--- gene=Chr7-6Mb-seg-732, id=2 removed by soft count = 156\n",
      "--- gene=Chr7-6Mb-seg-732, id=3 removed by soft count = 617\n",
      "--- gene=Chr7-6Mb-seg-732, id=4 removed by soft count = 20763\n",
      "--- gene=Chr7-6Mb-seg-732, id=5 removed by soft count = 1814\n",
      "--- gene=Chr7-6Mb-seg-732, id=7 removed by soft count = 1730\n",
      "--- gene=Chr7-6Mb-seg-732, id=9 removed by soft count = 784\n",
      "--- gene=Chr7-6Mb-seg-732, id=10 removed by soft count = 1373\n",
      "--- gene=Chr7-6Mb-seg-732, id=22 removed by soft count = 205\n",
      "--- number of probes: 23 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-seg-733\n",
      "--- gene=Chr7-6Mb-seg-733, id=12 removed by soft count = 5181\n",
      "--- gene=Chr7-6Mb-seg-733, id=13 removed by soft count = 1805\n",
      "--- gene=Chr7-6Mb-seg-733, id=35 removed by soft count = 4578\n",
      "--- number of probes: 57 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-734\n",
      "--- gene=Chr7-6Mb-seg-734, id=0 removed by soft count = 1057\n",
      "--- gene=Chr7-6Mb-seg-734, id=1 removed by soft count = 447\n",
      "--- gene=Chr7-6Mb-seg-734, id=2 removed by soft count = 1681\n",
      "--- gene=Chr7-6Mb-seg-734, id=3 removed by soft count = 1397\n",
      "--- gene=Chr7-6Mb-seg-734, id=4 removed by soft count = 238\n",
      "--- gene=Chr7-6Mb-seg-734, id=5 removed by soft count = 15392\n",
      "--- gene=Chr7-6Mb-seg-734, id=8 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-734, id=24 removed by soft count = 291\n",
      "--- number of probes: 25 , kept by blast: 17 , if remove dups: 17\n",
      "-- number of probes kept for this region: 17\n",
      "-- checking probes in region: Chr7-6Mb-seg-735\n",
      "--- gene=Chr7-6Mb-seg-735, id=1 removed by soft count = 4249\n",
      "--- gene=Chr7-6Mb-seg-735, id=27 removed by soft count = 97\n",
      "--- gene=Chr7-6Mb-seg-735, id=44 removed by soft count = 244\n",
      "--- gene=Chr7-6Mb-seg-735, id=45 removed by soft count = 7788\n",
      "--- number of probes: 46 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: Chr7-6Mb-seg-738\n",
      "--- gene=Chr7-6Mb-seg-738, id=0 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-seg-738, id=1 removed by soft count = 2183\n",
      "--- gene=Chr7-6Mb-seg-738, id=3 removed by soft count = 761\n",
      "--- gene=Chr7-6Mb-seg-738, id=34 removed by soft count = 317\n",
      "--- number of probes: 38 , kept by blast: 34 , if remove dups: 33\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-739\n",
      "--- gene=Chr7-6Mb-seg-739, id=37 removed by soft count = 223\n",
      "--- number of probes: 41 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: Chr7-6Mb-seg-740\n",
      "--- gene=Chr7-6Mb-seg-740, id=44 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-740, id=52 removed by soft count = 46619\n",
      "--- gene=Chr7-6Mb-seg-740, id=54 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-740, id=55 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-740, id=58 removed by soft count = 1256\n",
      "--- number of probes: 61 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-741\n",
      "--- gene=Chr7-6Mb-seg-741, id=6 removed by soft count = 16684\n",
      "--- gene=Chr7-6Mb-seg-741, id=41 removed by soft count = 122\n",
      "--- gene=Chr7-6Mb-seg-741, id=43 removed by soft count = 151\n",
      "--- gene=Chr7-6Mb-seg-741, id=45 removed by soft count = 142\n",
      "--- gene=Chr7-6Mb-seg-741, id=49 removed by soft count = 206\n",
      "--- gene=Chr7-6Mb-seg-741, id=51 removed by soft count = 126\n",
      "--- gene=Chr7-6Mb-seg-741, id=52 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-741, id=53 removed by soft count = 2336\n",
      "--- gene=Chr7-6Mb-seg-741, id=54 removed by soft count = 270\n",
      "--- number of probes: 55 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-742\n",
      "--- gene=Chr7-6Mb-seg-742, id=17 removed by soft count = 1633\n",
      "--- gene=Chr7-6Mb-seg-742, id=19 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-742, id=20 removed by soft count = 74\n",
      "--- number of probes: 37 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-743\n",
      "--- gene=Chr7-6Mb-seg-743, id=9 removed by soft count = 126\n",
      "--- gene=Chr7-6Mb-seg-743, id=10 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-743, id=39 removed by soft count = 202\n",
      "--- gene=Chr7-6Mb-seg-743, id=49 removed by soft count = 89\n",
      "--- number of probes: 59 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-744\n",
      "--- gene=Chr7-6Mb-seg-744, id=17 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-744, id=19 removed by soft count = 121\n",
      "--- gene=Chr7-6Mb-seg-744, id=22 removed by soft count = 70\n",
      "--- number of probes: 44 , kept by blast: 41 , if remove dups: 39\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: Chr7-6Mb-seg-745\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-745, id=13 removed by soft count = 5214\n",
      "--- number of probes: 32 , kept by blast: 31 , if remove dups: 29\n",
      "-- number of probes kept for this region: 31\n",
      "-- checking probes in region: Chr7-6Mb-seg-746\n",
      "--- gene=Chr7-6Mb-seg-746, id=33 removed by soft count = 341\n",
      "--- gene=Chr7-6Mb-seg-746, id=34 removed by soft count = 3100\n",
      "--- number of probes: 35 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: Chr7-6Mb-seg-747\n",
      "--- gene=Chr7-6Mb-seg-747, id=0 removed by soft count = 440\n",
      "--- gene=Chr7-6Mb-seg-747, id=2 removed by soft count = 502\n",
      "--- gene=Chr7-6Mb-seg-747, id=3 removed by soft count = 1611\n",
      "--- gene=Chr7-6Mb-seg-747, id=21 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-747, id=22 removed by soft count = 707\n",
      "--- gene=Chr7-6Mb-seg-747, id=23 removed by soft count = 532\n",
      "--- gene=Chr7-6Mb-seg-747, id=25 removed by soft count = 999\n",
      "--- gene=Chr7-6Mb-seg-747, id=26 removed by soft count = 168\n",
      "--- number of probes: 27 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-seg-748\n",
      "--- gene=Chr7-6Mb-seg-748, id=0 removed by soft count = 245\n",
      "--- gene=Chr7-6Mb-seg-748, id=46 removed by soft count = 433\n",
      "--- number of probes: 54 , kept by blast: 52 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-749\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-750\n",
      "--- gene=Chr7-6Mb-seg-750, id=7 removed by soft count = 545\n",
      "--- gene=Chr7-6Mb-seg-750, id=46 removed by soft count = 51\n",
      "--- number of probes: 52 , kept by blast: 50 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-751\n",
      "--- gene=Chr7-6Mb-seg-751, id=10 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-751, id=12 removed by soft count = 187\n",
      "--- gene=Chr7-6Mb-seg-751, id=13 removed by soft count = 295\n",
      "--- gene=Chr7-6Mb-seg-751, id=14 removed by soft count = 317\n",
      "--- gene=Chr7-6Mb-seg-751, id=15 removed by soft count = 204\n",
      "--- gene=Chr7-6Mb-seg-751, id=18 removed by soft count = 285\n",
      "--- gene=Chr7-6Mb-seg-751, id=19 removed by soft count = 412\n",
      "--- gene=Chr7-6Mb-seg-751, id=20 removed by soft count = 554\n",
      "--- gene=Chr7-6Mb-seg-751, id=21 removed by soft count = 487\n",
      "--- gene=Chr7-6Mb-seg-751, id=22 removed by soft count = 4017\n",
      "--- gene=Chr7-6Mb-seg-751, id=27 removed by soft count = 266\n",
      "--- gene=Chr7-6Mb-seg-751, id=28 removed by soft count = 178\n",
      "--- gene=Chr7-6Mb-seg-751, id=35 removed by soft count = 71\n",
      "--- number of probes: 45 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: Chr7-6Mb-seg-752\n",
      "--- number of probes: 60 , kept by blast: 60 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-753\n",
      "--- gene=Chr7-6Mb-seg-753, id=13 removed by soft count = 546\n",
      "--- gene=Chr7-6Mb-seg-753, id=14 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-753, id=18 removed by soft count = 100\n",
      "--- number of probes: 48 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: Chr7-6Mb-seg-754\n",
      "--- gene=Chr7-6Mb-seg-754, id=1 removed by soft count = 99\n",
      "--- gene=Chr7-6Mb-seg-754, id=3 removed by soft count = 211\n",
      "--- gene=Chr7-6Mb-seg-754, id=4 removed by soft count = 705\n",
      "--- gene=Chr7-6Mb-seg-754, id=5 removed by soft count = 200\n",
      "--- gene=Chr7-6Mb-seg-754, id=6 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-754, id=8 removed by soft count = 122\n",
      "--- gene=Chr7-6Mb-seg-754, id=9 removed by soft count = 133\n",
      "--- gene=Chr7-6Mb-seg-754, id=10 removed by soft count = 117\n",
      "--- gene=Chr7-6Mb-seg-754, id=12 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-754, id=13 removed by soft count = 150\n",
      "--- gene=Chr7-6Mb-seg-754, id=14 removed by soft count = 128\n",
      "--- gene=Chr7-6Mb-seg-754, id=15 removed by soft count = 108\n",
      "--- gene=Chr7-6Mb-seg-754, id=16 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-754, id=19 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-754, id=20 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-754, id=21 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-754, id=22 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-754, id=23 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-754, id=24 removed by soft count = 60\n",
      "--- number of probes: 25 , kept by blast: 6 , if remove dups: 6\n",
      "-- number of probes kept for this region: 6\n",
      "-- checking probes in region: Chr7-6Mb-seg-755\n",
      "--- gene=Chr7-6Mb-seg-755, id=1 removed by soft count = 115\n",
      "--- gene=Chr7-6Mb-seg-755, id=2 removed by soft count = 67\n",
      "--- gene=Chr7-6Mb-seg-755, id=3 removed by soft count = 138\n",
      "--- number of probes: 48 , kept by blast: 45 , if remove dups: 42\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: Chr7-6Mb-seg-756\n",
      "--- gene=Chr7-6Mb-seg-756, id=0 removed by soft count = 162\n",
      "--- gene=Chr7-6Mb-seg-756, id=1 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-756, id=2 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-756, id=4 removed by soft count = 519\n",
      "--- gene=Chr7-6Mb-seg-756, id=14 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-756, id=15 removed by soft count = 91\n",
      "--- gene=Chr7-6Mb-seg-756, id=16 removed by soft count = 3121\n",
      "--- gene=Chr7-6Mb-seg-756, id=17 removed by soft count = 3979\n",
      "--- gene=Chr7-6Mb-seg-756, id=18 removed by soft count = 226\n",
      "--- number of probes: 52 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: Chr7-6Mb-seg-757\n",
      "--- gene=Chr7-6Mb-seg-757, id=4 removed by hard count = 0\n",
      "--- number of probes: 50 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-758\n",
      "--- gene=Chr7-6Mb-seg-758, id=0 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-758, id=33 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-758, id=34 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-758, id=35 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-758, id=37 removed by soft count = 190\n",
      "--- gene=Chr7-6Mb-seg-758, id=49 removed by soft count = 345\n",
      "--- gene=Chr7-6Mb-seg-758, id=50 removed by soft count = 139\n",
      "--- gene=Chr7-6Mb-seg-758, id=51 removed by soft count = 88\n",
      "--- gene=Chr7-6Mb-seg-758, id=53 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-758, id=54 removed by soft count = 195\n",
      "--- number of probes: 55 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: Chr7-6Mb-seg-760\n",
      "--- gene=Chr7-6Mb-seg-760, id=17 removed by soft count = 164\n",
      "--- gene=Chr7-6Mb-seg-760, id=18 removed by soft count = 118\n",
      "--- gene=Chr7-6Mb-seg-760, id=19 removed by soft count = 268\n",
      "--- gene=Chr7-6Mb-seg-760, id=25 removed by soft count = 443\n",
      "--- gene=Chr7-6Mb-seg-760, id=26 removed by soft count = 661\n",
      "--- number of probes: 27 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-seg-762\n",
      "--- gene=Chr7-6Mb-seg-762, id=0 removed by soft count = 513\n",
      "--- gene=Chr7-6Mb-seg-762, id=1 removed by soft count = 52996\n",
      "--- gene=Chr7-6Mb-seg-762, id=2 removed by soft count = 3453\n",
      "--- number of probes: 23 , kept by blast: 20 , if remove dups: 20\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-seg-763\n",
      "--- gene=Chr7-6Mb-seg-763, id=15 removed by soft count = 23761\n",
      "--- gene=Chr7-6Mb-seg-763, id=16 removed by soft count = 248\n",
      "--- gene=Chr7-6Mb-seg-763, id=17 removed by soft count = 277\n",
      "--- number of probes: 40 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: Chr7-6Mb-seg-764\n",
      "--- gene=Chr7-6Mb-seg-764, id=40 removed by soft count = 231\n",
      "--- gene=Chr7-6Mb-seg-764, id=41 removed by soft count = 709\n",
      "--- number of probes: 42 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: Chr7-6Mb-seg-765\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-765, id=0 removed by soft count = 3386\n",
      "--- gene=Chr7-6Mb-seg-765, id=2 removed by soft count = 808\n",
      "--- gene=Chr7-6Mb-seg-765, id=4 removed by soft count = 263\n",
      "--- gene=Chr7-6Mb-seg-765, id=5 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-765, id=6 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-765, id=22 removed by soft count = 54\n",
      "--- number of probes: 24 , kept by blast: 18 , if remove dups: 18\n",
      "-- number of probes kept for this region: 18\n",
      "-- checking probes in region: Chr7-6Mb-seg-766\n",
      "--- gene=Chr7-6Mb-seg-766, id=0 removed by soft count = 760\n",
      "--- gene=Chr7-6Mb-seg-766, id=2 removed by soft count = 75\n",
      "--- gene=Chr7-6Mb-seg-766, id=24 removed by soft count = 121\n",
      "--- gene=Chr7-6Mb-seg-766, id=43 removed by soft count = 199\n",
      "--- gene=Chr7-6Mb-seg-766, id=44 removed by soft count = 2289\n",
      "--- number of probes: 50 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: Chr7-6Mb-seg-767\n",
      "--- gene=Chr7-6Mb-seg-767, id=31 removed by soft count = 182\n",
      "--- number of probes: 49 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-768\n",
      "--- gene=Chr7-6Mb-seg-768, id=14 removed by soft count = 143\n",
      "--- gene=Chr7-6Mb-seg-768, id=17 removed by soft count = 90\n",
      "--- gene=Chr7-6Mb-seg-768, id=21 removed by soft count = 57\n",
      "--- number of probes: 42 , kept by blast: 39 , if remove dups: 38\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: Chr7-6Mb-seg-769\n",
      "--- gene=Chr7-6Mb-seg-769, id=3 removed by soft count = 1139\n",
      "--- gene=Chr7-6Mb-seg-769, id=21 removed by soft count = 399\n",
      "--- gene=Chr7-6Mb-seg-769, id=22 removed by soft count = 654\n",
      "--- number of probes: 23 , kept by blast: 20 , if remove dups: 15\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-seg-770\n",
      "--- gene=Chr7-6Mb-seg-770, id=0 removed by soft count = 502\n",
      "--- gene=Chr7-6Mb-seg-770, id=1 removed by soft count = 1728\n",
      "--- gene=Chr7-6Mb-seg-770, id=2 removed by soft count = 230\n",
      "--- gene=Chr7-6Mb-seg-770, id=16 removed by soft count = 99\n",
      "--- gene=Chr7-6Mb-seg-770, id=17 removed by soft count = 611\n",
      "--- gene=Chr7-6Mb-seg-770, id=18 removed by soft count = 179\n",
      "--- number of probes: 21 , kept by blast: 15 , if remove dups: 8\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: Chr7-6Mb-seg-771\n",
      "--- gene=Chr7-6Mb-seg-771, id=6 removed by soft count = 838\n",
      "--- gene=Chr7-6Mb-seg-771, id=9 removed by soft count = 561\n",
      "--- gene=Chr7-6Mb-seg-771, id=12 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-771, id=15 removed by soft count = 603\n",
      "--- gene=Chr7-6Mb-seg-771, id=18 removed by soft count = 134\n",
      "--- gene=Chr7-6Mb-seg-771, id=21 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-771, id=26 removed by soft count = 67\n",
      "--- number of probes: 28 , kept by blast: 21 , if remove dups: 14\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: Chr7-6Mb-seg-772\n",
      "--- gene=Chr7-6Mb-seg-772, id=16 removed by soft count = 136\n",
      "--- gene=Chr7-6Mb-seg-772, id=17 removed by soft count = 548\n",
      "--- gene=Chr7-6Mb-seg-772, id=19 removed by soft count = 526\n",
      "--- gene=Chr7-6Mb-seg-772, id=23 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-772, id=25 removed by soft count = 99\n",
      "--- gene=Chr7-6Mb-seg-772, id=26 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-772, id=28 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-772, id=29 removed by soft count = 6045\n",
      "--- gene=Chr7-6Mb-seg-772, id=30 removed by soft count = 706\n",
      "--- gene=Chr7-6Mb-seg-772, id=32 removed by soft count = 103\n",
      "--- number of probes: 47 , kept by blast: 37 , if remove dups: 36\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: Chr7-6Mb-seg-773\n",
      "--- gene=Chr7-6Mb-seg-773, id=0 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-seg-773, id=27 removed by soft count = 68\n",
      "--- number of probes: 29 , kept by blast: 27 , if remove dups: 27\n",
      "-- number of probes kept for this region: 27\n",
      "-- checking probes in region: Chr7-6Mb-seg-775\n",
      "--- gene=Chr7-6Mb-seg-775, id=1 removed by soft count = 371\n",
      "--- gene=Chr7-6Mb-seg-775, id=2 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-775, id=3 removed by soft count = 479\n",
      "--- gene=Chr7-6Mb-seg-775, id=4 removed by soft count = 100\n",
      "--- gene=Chr7-6Mb-seg-775, id=25 removed by soft count = 305\n",
      "--- gene=Chr7-6Mb-seg-775, id=26 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-775, id=28 removed by soft count = 1459\n",
      "--- gene=Chr7-6Mb-seg-775, id=32 removed by soft count = 157\n",
      "--- gene=Chr7-6Mb-seg-775, id=34 removed by soft count = 248\n",
      "--- gene=Chr7-6Mb-seg-775, id=35 removed by soft count = 101\n",
      "--- number of probes: 36 , kept by blast: 26 , if remove dups: 25\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: Chr7-6Mb-seg-776\n",
      "--- gene=Chr7-6Mb-seg-776, id=0 removed by soft count = 118\n",
      "--- gene=Chr7-6Mb-seg-776, id=3 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-776, id=6 removed by soft count = 113\n",
      "--- gene=Chr7-6Mb-seg-776, id=9 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-776, id=12 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-776, id=15 removed by soft count = 108\n",
      "--- gene=Chr7-6Mb-seg-776, id=18 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-776, id=21 removed by soft count = 110\n",
      "--- gene=Chr7-6Mb-seg-776, id=24 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-776, id=27 removed by soft count = 114\n",
      "--- gene=Chr7-6Mb-seg-776, id=30 removed by soft count = 117\n",
      "--- gene=Chr7-6Mb-seg-776, id=33 removed by soft count = 115\n",
      "--- gene=Chr7-6Mb-seg-776, id=36 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-776, id=39 removed by soft count = 2193\n",
      "--- gene=Chr7-6Mb-seg-776, id=42 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-776, id=45 removed by soft count = 108\n",
      "--- gene=Chr7-6Mb-seg-776, id=48 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-776, id=51 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-776, id=54 removed by soft count = 117\n",
      "--- gene=Chr7-6Mb-seg-776, id=57 removed by soft count = 145\n",
      "--- gene=Chr7-6Mb-seg-776, id=58 removed by soft count = 291\n",
      "--- gene=Chr7-6Mb-seg-776, id=59 removed by soft count = 85\n",
      "--- gene=Chr7-6Mb-seg-776, id=60 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-776, id=63 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-776, id=65 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-776, id=66 removed by soft count = 109\n",
      "--- number of probes: 67 , kept by blast: 41 , if remove dups: 40\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: Chr7-6Mb-seg-777\n",
      "--- gene=Chr7-6Mb-seg-777, id=1 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-777, id=28 removed by soft count = 1125\n",
      "--- gene=Chr7-6Mb-seg-777, id=31 removed by soft count = 219\n",
      "--- gene=Chr7-6Mb-seg-777, id=32 removed by soft count = 533\n",
      "--- gene=Chr7-6Mb-seg-777, id=33 removed by soft count = 227\n",
      "--- gene=Chr7-6Mb-seg-777, id=34 removed by soft count = 252\n",
      "--- gene=Chr7-6Mb-seg-777, id=36 removed by soft count = 177\n",
      "--- gene=Chr7-6Mb-seg-777, id=37 removed by soft count = 571\n",
      "--- gene=Chr7-6Mb-seg-777, id=38 removed by soft count = 921\n",
      "--- number of probes: 39 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: Chr7-6Mb-seg-778\n",
      "--- gene=Chr7-6Mb-seg-778, id=0 removed by soft count = 810\n",
      "--- gene=Chr7-6Mb-seg-778, id=40 removed by soft count = 6891\n",
      "--- number of probes: 41 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: Chr7-6Mb-seg-780\n",
      "--- number of probes: 26 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: Chr7-6Mb-seg-781\n",
      "--- gene=Chr7-6Mb-seg-781, id=13 removed by soft count = 278\n",
      "--- gene=Chr7-6Mb-seg-781, id=31 removed by soft count = 210\n",
      "--- gene=Chr7-6Mb-seg-781, id=33 removed by soft count = 379\n",
      "--- gene=Chr7-6Mb-seg-781, id=37 removed by soft count = 8674\n",
      "--- number of probes: 38 , kept by blast: 34 , if remove dups: 33\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-784\n",
      "--- gene=Chr7-6Mb-seg-784, id=19 removed by soft count = 589\n",
      "--- gene=Chr7-6Mb-seg-784, id=21 removed by soft count = 2934\n",
      "--- number of probes: 48 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-785\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-785, id=46 removed by soft count = 28825\n",
      "--- gene=Chr7-6Mb-seg-785, id=52 removed by soft count = 463\n",
      "--- number of probes: 54 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-786\n",
      "--- gene=Chr7-6Mb-seg-786, id=26 removed by hard count = 3\n",
      "--- gene=Chr7-6Mb-seg-786, id=27 removed by soft count = 129\n",
      "--- number of probes: 39 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: Chr7-6Mb-seg-787\n",
      "--- number of probes: 48 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-788\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-790\n",
      "--- gene=Chr7-6Mb-seg-790, id=9 removed by soft count = 544\n",
      "--- gene=Chr7-6Mb-seg-790, id=11 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-790, id=12 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-790, id=24 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-790, id=25 removed by soft count = 120\n",
      "--- gene=Chr7-6Mb-seg-790, id=26 removed by soft count = 139\n",
      "--- gene=Chr7-6Mb-seg-790, id=27 removed by soft count = 132\n",
      "--- gene=Chr7-6Mb-seg-790, id=28 removed by soft count = 68\n",
      "--- gene=Chr7-6Mb-seg-790, id=29 removed by hard count = 3\n",
      "--- gene=Chr7-6Mb-seg-790, id=30 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-790, id=31 removed by soft count = 292\n",
      "--- gene=Chr7-6Mb-seg-790, id=34 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-790, id=35 removed by soft count = 175\n",
      "--- gene=Chr7-6Mb-seg-790, id=36 removed by soft count = 75\n",
      "--- gene=Chr7-6Mb-seg-790, id=37 removed by soft count = 17175\n",
      "--- gene=Chr7-6Mb-seg-790, id=38 removed by soft count = 15572\n",
      "--- number of probes: 39 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-seg-791\n",
      "--- gene=Chr7-6Mb-seg-791, id=2 removed by soft count = 767\n",
      "--- gene=Chr7-6Mb-seg-791, id=14 removed by soft count = 328\n",
      "--- number of probes: 25 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-seg-792\n",
      "--- gene=Chr7-6Mb-seg-792, id=2 removed by soft count = 424\n",
      "--- gene=Chr7-6Mb-seg-792, id=19 removed by soft count = 266\n",
      "--- gene=Chr7-6Mb-seg-792, id=20 removed by soft count = 1786\n",
      "--- gene=Chr7-6Mb-seg-792, id=21 removed by soft count = 417\n",
      "--- gene=Chr7-6Mb-seg-792, id=22 removed by soft count = 1648\n",
      "--- gene=Chr7-6Mb-seg-792, id=23 removed by soft count = 2629\n",
      "--- gene=Chr7-6Mb-seg-792, id=24 removed by soft count = 215\n",
      "--- gene=Chr7-6Mb-seg-792, id=30 removed by soft count = 171\n",
      "--- number of probes: 42 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-793\n",
      "--- gene=Chr7-6Mb-seg-793, id=7 removed by soft count = 93\n",
      "--- gene=Chr7-6Mb-seg-793, id=8 removed by soft count = 769\n",
      "--- gene=Chr7-6Mb-seg-793, id=9 removed by soft count = 553\n",
      "--- gene=Chr7-6Mb-seg-793, id=10 removed by soft count = 469\n",
      "--- gene=Chr7-6Mb-seg-793, id=11 removed by soft count = 750\n",
      "--- gene=Chr7-6Mb-seg-793, id=22 removed by soft count = 418\n",
      "--- gene=Chr7-6Mb-seg-793, id=23 removed by soft count = 353\n",
      "--- gene=Chr7-6Mb-seg-793, id=24 removed by soft count = 86\n",
      "--- number of probes: 29 , kept by blast: 21 , if remove dups: 20\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: Chr7-6Mb-seg-794\n",
      "--- gene=Chr7-6Mb-seg-794, id=1 removed by soft count = 79\n",
      "--- gene=Chr7-6Mb-seg-794, id=2 removed by soft count = 450\n",
      "--- number of probes: 37 , kept by blast: 35 , if remove dups: 34\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: Chr7-6Mb-seg-795\n",
      "--- gene=Chr7-6Mb-seg-795, id=0 removed by soft count = 267\n",
      "--- gene=Chr7-6Mb-seg-795, id=1 removed by soft count = 24990\n",
      "--- gene=Chr7-6Mb-seg-795, id=20 removed by soft count = 52\n",
      "--- number of probes: 24 , kept by blast: 21 , if remove dups: 21\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: Chr7-6Mb-seg-796\n",
      "--- gene=Chr7-6Mb-seg-796, id=11 removed by soft count = 145\n",
      "--- gene=Chr7-6Mb-seg-796, id=12 removed by soft count = 191\n",
      "--- gene=Chr7-6Mb-seg-796, id=13 removed by soft count = 2663\n",
      "--- gene=Chr7-6Mb-seg-796, id=14 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-796, id=15 removed by soft count = 2097\n",
      "--- gene=Chr7-6Mb-seg-796, id=16 removed by soft count = 839\n",
      "--- gene=Chr7-6Mb-seg-796, id=17 removed by soft count = 377\n",
      "--- gene=Chr7-6Mb-seg-796, id=18 removed by soft count = 259\n",
      "--- gene=Chr7-6Mb-seg-796, id=20 removed by soft count = 835\n",
      "--- gene=Chr7-6Mb-seg-796, id=21 removed by soft count = 3754\n",
      "--- gene=Chr7-6Mb-seg-796, id=22 removed by soft count = 115\n",
      "--- number of probes: 24 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-seg-797\n",
      "--- gene=Chr7-6Mb-seg-797, id=14 removed by soft count = 219\n",
      "--- gene=Chr7-6Mb-seg-797, id=16 removed by soft count = 2396\n",
      "--- gene=Chr7-6Mb-seg-797, id=35 removed by soft count = 114\n",
      "--- number of probes: 44 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: Chr7-6Mb-seg-798\n",
      "--- gene=Chr7-6Mb-seg-798, id=8 removed by soft count = 124\n",
      "--- gene=Chr7-6Mb-seg-798, id=9 removed by soft count = 2900\n",
      "--- gene=Chr7-6Mb-seg-798, id=11 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-798, id=47 removed by soft count = 104\n",
      "--- number of probes: 52 , kept by blast: 48 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: Chr7-6Mb-seg-799\n",
      "--- gene=Chr7-6Mb-seg-799, id=7 removed by soft count = 754\n",
      "--- gene=Chr7-6Mb-seg-799, id=37 removed by soft count = 89\n",
      "--- number of probes: 45 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: Chr7-6Mb-seg-800\n",
      "--- number of probes: 27 , kept by blast: 27 , if remove dups: 26\n",
      "-- number of probes kept for this region: 27\n",
      "-- checking probes in region: Chr7-6Mb-seg-801\n",
      "--- gene=Chr7-6Mb-seg-801, id=3 removed by soft count = 53\n",
      "--- number of probes: 23 , kept by blast: 22 , if remove dups: 20\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-seg-802\n",
      "--- gene=Chr7-6Mb-seg-802, id=11 removed by soft count = 125\n",
      "--- gene=Chr7-6Mb-seg-802, id=12 removed by soft count = 489\n",
      "--- gene=Chr7-6Mb-seg-802, id=13 removed by soft count = 88\n",
      "--- gene=Chr7-6Mb-seg-802, id=14 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-802, id=15 removed by soft count = 771\n",
      "--- gene=Chr7-6Mb-seg-802, id=16 removed by soft count = 767\n",
      "--- gene=Chr7-6Mb-seg-802, id=18 removed by soft count = 407\n",
      "--- gene=Chr7-6Mb-seg-802, id=50 removed by soft count = 16427\n",
      "--- number of probes: 51 , kept by blast: 43 , if remove dups: 39\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: Chr7-6Mb-seg-804\n",
      "--- gene=Chr7-6Mb-seg-804, id=23 removed by soft count = 364\n",
      "--- gene=Chr7-6Mb-seg-804, id=34 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-804, id=37 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-804, id=38 removed by soft count = 667\n",
      "--- gene=Chr7-6Mb-seg-804, id=40 removed by soft count = 100\n",
      "--- number of probes: 43 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: Chr7-6Mb-seg-805\n",
      "--- gene=Chr7-6Mb-seg-805, id=29 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-805, id=35 removed by soft count = 1414\n",
      "--- gene=Chr7-6Mb-seg-805, id=36 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-805, id=43 removed by soft count = 52\n",
      "--- number of probes: 48 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-806\n",
      "--- number of probes: 51 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-807\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-807, id=6 removed by soft count = 3486\n",
      "--- gene=Chr7-6Mb-seg-807, id=7 removed by soft count = 280\n",
      "--- gene=Chr7-6Mb-seg-807, id=8 removed by soft count = 217\n",
      "--- gene=Chr7-6Mb-seg-807, id=12 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-807, id=36 removed by soft count = 110\n",
      "--- gene=Chr7-6Mb-seg-807, id=37 removed by soft count = 159\n",
      "--- number of probes: 53 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: Chr7-6Mb-seg-808\n",
      "--- gene=Chr7-6Mb-seg-808, id=7 removed by soft count = 64\n",
      "--- number of probes: 64 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-809\n",
      "--- gene=Chr7-6Mb-seg-809, id=33 removed by soft count = 124\n",
      "--- number of probes: 66 , kept by blast: 65 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-810\n",
      "--- gene=Chr7-6Mb-seg-810, id=23 removed by soft count = 244\n",
      "--- gene=Chr7-6Mb-seg-810, id=24 removed by soft count = 4485\n",
      "--- gene=Chr7-6Mb-seg-810, id=25 removed by soft count = 232\n",
      "--- number of probes: 27 , kept by blast: 24 , if remove dups: 24\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-seg-812\n",
      "--- gene=Chr7-6Mb-seg-812, id=1 removed by soft count = 217\n",
      "--- gene=Chr7-6Mb-seg-812, id=3 removed by soft count = 250\n",
      "--- gene=Chr7-6Mb-seg-812, id=4 removed by soft count = 1050\n",
      "--- gene=Chr7-6Mb-seg-812, id=5 removed by soft count = 1466\n",
      "--- gene=Chr7-6Mb-seg-812, id=6 removed by soft count = 2113\n",
      "--- gene=Chr7-6Mb-seg-812, id=7 removed by soft count = 143\n",
      "--- gene=Chr7-6Mb-seg-812, id=10 removed by soft count = 201\n",
      "--- gene=Chr7-6Mb-seg-812, id=13 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-812, id=16 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-812, id=19 removed by soft count = 109\n",
      "--- number of probes: 22 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-seg-813\n",
      "--- gene=Chr7-6Mb-seg-813, id=25 removed by hard count = 0\n",
      "--- number of probes: 52 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-814\n",
      "--- gene=Chr7-6Mb-seg-814, id=15 removed by soft count = 403\n",
      "--- gene=Chr7-6Mb-seg-814, id=16 removed by soft count = 411\n",
      "--- gene=Chr7-6Mb-seg-814, id=17 removed by soft count = 21375\n",
      "--- gene=Chr7-6Mb-seg-814, id=18 removed by soft count = 325\n",
      "--- gene=Chr7-6Mb-seg-814, id=20 removed by soft count = 90\n",
      "--- gene=Chr7-6Mb-seg-814, id=22 removed by soft count = 3707\n",
      "--- number of probes: 23 , kept by blast: 17 , if remove dups: 17\n",
      "-- number of probes kept for this region: 17\n",
      "-- checking probes in region: Chr7-6Mb-seg-816\n",
      "--- gene=Chr7-6Mb-seg-816, id=0 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-816, id=1 removed by soft count = 67\n",
      "--- number of probes: 63 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-817\n",
      "--- gene=Chr7-6Mb-seg-817, id=25 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-817, id=43 removed by soft count = 59\n",
      "--- number of probes: 67 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-818\n",
      "--- gene=Chr7-6Mb-seg-818, id=3 removed by soft count = 494\n",
      "--- gene=Chr7-6Mb-seg-818, id=11 removed by soft count = 339\n",
      "--- gene=Chr7-6Mb-seg-818, id=14 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-818, id=15 removed by soft count = 178\n",
      "--- gene=Chr7-6Mb-seg-818, id=18 removed by soft count = 195\n",
      "--- gene=Chr7-6Mb-seg-818, id=19 removed by soft count = 79\n",
      "--- gene=Chr7-6Mb-seg-818, id=20 removed by soft count = 1571\n",
      "--- gene=Chr7-6Mb-seg-818, id=22 removed by soft count = 180\n",
      "--- gene=Chr7-6Mb-seg-818, id=23 removed by soft count = 143\n",
      "--- gene=Chr7-6Mb-seg-818, id=25 removed by soft count = 256\n",
      "--- gene=Chr7-6Mb-seg-818, id=31 removed by soft count = 151\n",
      "--- gene=Chr7-6Mb-seg-818, id=38 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-818, id=39 removed by soft count = 164\n",
      "--- number of probes: 46 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: Chr7-6Mb-seg-819\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-820\n",
      "--- number of probes: 62 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-821\n",
      "--- number of probes: 30 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: Chr7-6Mb-seg-822\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-823\n",
      "--- gene=Chr7-6Mb-seg-823, id=7 removed by soft count = 195\n",
      "--- gene=Chr7-6Mb-seg-823, id=25 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-823, id=29 removed by soft count = 79\n",
      "--- number of probes: 37 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-824\n",
      "--- gene=Chr7-6Mb-seg-824, id=25 removed by soft count = 248\n",
      "--- gene=Chr7-6Mb-seg-824, id=27 removed by soft count = 187\n",
      "--- gene=Chr7-6Mb-seg-824, id=44 removed by soft count = 156\n",
      "--- gene=Chr7-6Mb-seg-824, id=65 removed by soft count = 103\n",
      "--- number of probes: 68 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-825\n",
      "--- gene=Chr7-6Mb-seg-825, id=3 removed by soft count = 1145\n",
      "--- gene=Chr7-6Mb-seg-825, id=8 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-825, id=9 removed by soft count = 93\n",
      "--- gene=Chr7-6Mb-seg-825, id=10 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-825, id=11 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-825, id=16 removed by soft count = 153\n",
      "--- gene=Chr7-6Mb-seg-825, id=17 removed by soft count = 4533\n",
      "--- gene=Chr7-6Mb-seg-825, id=18 removed by soft count = 129\n",
      "--- gene=Chr7-6Mb-seg-825, id=19 removed by soft count = 489\n",
      "--- gene=Chr7-6Mb-seg-825, id=20 removed by soft count = 1117\n",
      "--- gene=Chr7-6Mb-seg-825, id=22 removed by soft count = 51\n",
      "--- number of probes: 30 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-seg-828\n",
      "--- gene=Chr7-6Mb-seg-828, id=19 removed by soft count = 306\n",
      "--- gene=Chr7-6Mb-seg-828, id=20 removed by soft count = 492\n",
      "--- gene=Chr7-6Mb-seg-828, id=22 removed by soft count = 95\n",
      "--- gene=Chr7-6Mb-seg-828, id=25 removed by soft count = 502\n",
      "--- gene=Chr7-6Mb-seg-828, id=26 removed by soft count = 101\n",
      "--- number of probes: 27 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-seg-829\n",
      "--- gene=Chr7-6Mb-seg-829, id=4 removed by soft count = 147\n",
      "--- gene=Chr7-6Mb-seg-829, id=5 removed by soft count = 785\n",
      "--- gene=Chr7-6Mb-seg-829, id=11 removed by soft count = 89\n",
      "--- gene=Chr7-6Mb-seg-829, id=12 removed by soft count = 314\n",
      "--- gene=Chr7-6Mb-seg-829, id=15 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-829, id=19 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-829, id=20 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-829, id=22 removed by soft count = 119\n",
      "--- gene=Chr7-6Mb-seg-829, id=23 removed by soft count = 159\n",
      "--- gene=Chr7-6Mb-seg-829, id=24 removed by soft count = 85\n",
      "--- gene=Chr7-6Mb-seg-829, id=25 removed by soft count = 656\n",
      "--- gene=Chr7-6Mb-seg-829, id=26 removed by soft count = 198\n",
      "--- gene=Chr7-6Mb-seg-829, id=28 removed by soft count = 243\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-829, id=29 removed by soft count = 88\n",
      "--- gene=Chr7-6Mb-seg-829, id=30 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-829, id=34 removed by soft count = 630\n",
      "--- gene=Chr7-6Mb-seg-829, id=35 removed by soft count = 418\n",
      "--- gene=Chr7-6Mb-seg-829, id=38 removed by soft count = 1500\n",
      "--- number of probes: 39 , kept by blast: 21 , if remove dups: 21\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: Chr7-6Mb-seg-831\n",
      "--- gene=Chr7-6Mb-seg-831, id=1 removed by soft count = 333\n",
      "--- gene=Chr7-6Mb-seg-831, id=2 removed by soft count = 3302\n",
      "--- gene=Chr7-6Mb-seg-831, id=4 removed by soft count = 257\n",
      "--- gene=Chr7-6Mb-seg-831, id=5 removed by soft count = 139\n",
      "--- gene=Chr7-6Mb-seg-831, id=13 removed by soft count = 178\n",
      "--- gene=Chr7-6Mb-seg-831, id=15 removed by soft count = 2104\n",
      "--- number of probes: 25 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-seg-832\n",
      "--- gene=Chr7-6Mb-seg-832, id=2 removed by soft count = 321\n",
      "--- gene=Chr7-6Mb-seg-832, id=17 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-832, id=23 removed by soft count = 145\n",
      "--- gene=Chr7-6Mb-seg-832, id=24 removed by soft count = 306\n",
      "--- gene=Chr7-6Mb-seg-832, id=26 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-832, id=27 removed by soft count = 545\n",
      "--- gene=Chr7-6Mb-seg-832, id=28 removed by soft count = 220\n",
      "--- number of probes: 29 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-seg-833\n",
      "--- gene=Chr7-6Mb-seg-833, id=0 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-seg-833, id=1 removed by soft count = 273\n",
      "--- gene=Chr7-6Mb-seg-833, id=2 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-833, id=3 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-833, id=4 removed by soft count = 146\n",
      "--- gene=Chr7-6Mb-seg-833, id=6 removed by soft count = 725\n",
      "--- gene=Chr7-6Mb-seg-833, id=7 removed by soft count = 204\n",
      "--- gene=Chr7-6Mb-seg-833, id=13 removed by soft count = 67\n",
      "--- gene=Chr7-6Mb-seg-833, id=15 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-833, id=16 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-833, id=21 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-833, id=22 removed by soft count = 87\n",
      "--- gene=Chr7-6Mb-seg-833, id=23 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-833, id=24 removed by soft count = 114\n",
      "--- gene=Chr7-6Mb-seg-833, id=25 removed by soft count = 99\n",
      "--- gene=Chr7-6Mb-seg-833, id=26 removed by soft count = 186\n",
      "--- gene=Chr7-6Mb-seg-833, id=31 removed by soft count = 2292\n",
      "--- number of probes: 32 , kept by blast: 15 , if remove dups: 15\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: Chr7-6Mb-seg-834\n",
      "--- gene=Chr7-6Mb-seg-834, id=0 removed by soft count = 164\n",
      "--- gene=Chr7-6Mb-seg-834, id=1 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-834, id=2 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-834, id=3 removed by soft count = 85\n",
      "--- gene=Chr7-6Mb-seg-834, id=4 removed by soft count = 251\n",
      "--- gene=Chr7-6Mb-seg-834, id=6 removed by soft count = 79\n",
      "--- gene=Chr7-6Mb-seg-834, id=11 removed by soft count = 118\n",
      "--- gene=Chr7-6Mb-seg-834, id=18 removed by soft count = 72\n",
      "--- number of probes: 56 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-835\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-836\n",
      "--- gene=Chr7-6Mb-seg-836, id=33 removed by soft count = 3328\n",
      "--- gene=Chr7-6Mb-seg-836, id=40 removed by soft count = 113\n",
      "--- number of probes: 68 , kept by blast: 66 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-837\n",
      "--- gene=Chr7-6Mb-seg-837, id=36 removed by soft count = 1164\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-840\n",
      "--- gene=Chr7-6Mb-seg-840, id=15 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-840, id=22 removed by soft count = 122\n",
      "--- number of probes: 66 , kept by blast: 64 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-841\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-842\n",
      "--- gene=Chr7-6Mb-seg-842, id=31 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-842, id=37 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-842, id=46 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-842, id=50 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-842, id=51 removed by soft count = 1324\n",
      "--- gene=Chr7-6Mb-seg-842, id=52 removed by soft count = 5691\n",
      "--- number of probes: 53 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: Chr7-6Mb-seg-845\n",
      "--- gene=Chr7-6Mb-seg-845, id=4 removed by soft count = 180\n",
      "--- gene=Chr7-6Mb-seg-845, id=47 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-845, id=49 removed by soft count = 129\n",
      "--- number of probes: 52 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-846\n",
      "--- gene=Chr7-6Mb-seg-846, id=9 removed by soft count = 193\n",
      "--- gene=Chr7-6Mb-seg-846, id=10 removed by soft count = 131\n",
      "--- gene=Chr7-6Mb-seg-846, id=59 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-846, id=62 removed by soft count = 194\n",
      "--- gene=Chr7-6Mb-seg-846, id=63 removed by soft count = 337\n",
      "--- gene=Chr7-6Mb-seg-846, id=64 removed by soft count = 64\n",
      "--- number of probes: 65 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-847\n",
      "--- gene=Chr7-6Mb-seg-847, id=3 removed by soft count = 199\n",
      "--- gene=Chr7-6Mb-seg-847, id=5 removed by soft count = 133\n",
      "--- gene=Chr7-6Mb-seg-847, id=6 removed by soft count = 393\n",
      "--- gene=Chr7-6Mb-seg-847, id=7 removed by soft count = 126\n",
      "--- gene=Chr7-6Mb-seg-847, id=10 removed by soft count = 174\n",
      "--- gene=Chr7-6Mb-seg-847, id=11 removed by soft count = 1066\n",
      "--- gene=Chr7-6Mb-seg-847, id=12 removed by soft count = 594\n",
      "--- gene=Chr7-6Mb-seg-847, id=14 removed by soft count = 186\n",
      "--- gene=Chr7-6Mb-seg-847, id=15 removed by soft count = 118\n",
      "--- gene=Chr7-6Mb-seg-847, id=16 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-847, id=19 removed by soft count = 2550\n",
      "--- gene=Chr7-6Mb-seg-847, id=20 removed by soft count = 1371\n",
      "--- gene=Chr7-6Mb-seg-847, id=21 removed by soft count = 15513\n",
      "--- number of probes: 22 , kept by blast: 9 , if remove dups: 9\n",
      "-- number of probes kept for this region: 9\n",
      "-- checking probes in region: Chr7-6Mb-seg-848\n",
      "--- gene=Chr7-6Mb-seg-848, id=0 removed by soft count = 283\n",
      "--- gene=Chr7-6Mb-seg-848, id=6 removed by soft count = 156\n",
      "--- gene=Chr7-6Mb-seg-848, id=8 removed by soft count = 204\n",
      "--- gene=Chr7-6Mb-seg-848, id=13 removed by soft count = 322\n",
      "--- gene=Chr7-6Mb-seg-848, id=14 removed by soft count = 198\n",
      "--- number of probes: 31 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: Chr7-6Mb-seg-849\n",
      "--- gene=Chr7-6Mb-seg-849, id=12 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-849, id=43 removed by soft count = 723\n",
      "--- gene=Chr7-6Mb-seg-849, id=45 removed by soft count = 447\n",
      "--- number of probes: 59 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-850\n",
      "--- gene=Chr7-6Mb-seg-850, id=0 removed by soft count = 119\n",
      "--- gene=Chr7-6Mb-seg-850, id=3 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-850, id=6 removed by soft count = 110\n",
      "--- gene=Chr7-6Mb-seg-850, id=9 removed by soft count = 112\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-850, id=12 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-850, id=15 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-850, id=18 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-850, id=21 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-850, id=24 removed by soft count = 115\n",
      "--- gene=Chr7-6Mb-seg-850, id=27 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-850, id=30 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-850, id=33 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-850, id=36 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-850, id=39 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-850, id=42 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-850, id=45 removed by soft count = 120\n",
      "--- gene=Chr7-6Mb-seg-850, id=47 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-850, id=48 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-850, id=51 removed by soft count = 124\n",
      "--- gene=Chr7-6Mb-seg-850, id=54 removed by soft count = 106\n",
      "--- number of probes: 56 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: Chr7-6Mb-seg-852\n",
      "--- gene=Chr7-6Mb-seg-852, id=21 removed by soft count = 7016\n",
      "--- gene=Chr7-6Mb-seg-852, id=22 removed by soft count = 371\n",
      "--- gene=Chr7-6Mb-seg-852, id=23 removed by soft count = 1343\n",
      "--- gene=Chr7-6Mb-seg-852, id=24 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-852, id=25 removed by soft count = 1151\n",
      "--- number of probes: 27 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-seg-853\n",
      "--- gene=Chr7-6Mb-seg-853, id=0 removed by soft count = 1071\n",
      "--- gene=Chr7-6Mb-seg-853, id=1 removed by soft count = 196\n",
      "--- gene=Chr7-6Mb-seg-853, id=8 removed by soft count = 399\n",
      "--- gene=Chr7-6Mb-seg-853, id=10 removed by soft count = 59\n",
      "--- number of probes: 43 , kept by blast: 39 , if remove dups: 38\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: Chr7-6Mb-seg-854\n",
      "--- gene=Chr7-6Mb-seg-854, id=25 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-854, id=29 removed by soft count = 175\n",
      "--- number of probes: 33 , kept by blast: 31 , if remove dups: 31\n",
      "-- number of probes kept for this region: 31\n",
      "-- checking probes in region: Chr7-6Mb-seg-855\n",
      "--- gene=Chr7-6Mb-seg-855, id=0 removed by soft count = 1935\n",
      "--- gene=Chr7-6Mb-seg-855, id=1 removed by soft count = 554\n",
      "--- gene=Chr7-6Mb-seg-855, id=2 removed by soft count = 79\n",
      "--- gene=Chr7-6Mb-seg-855, id=5 removed by soft count = 154\n",
      "--- gene=Chr7-6Mb-seg-855, id=7 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-855, id=9 removed by soft count = 982\n",
      "--- gene=Chr7-6Mb-seg-855, id=10 removed by soft count = 2429\n",
      "--- gene=Chr7-6Mb-seg-855, id=11 removed by soft count = 2241\n",
      "--- gene=Chr7-6Mb-seg-855, id=12 removed by soft count = 147\n",
      "--- gene=Chr7-6Mb-seg-855, id=14 removed by soft count = 317\n",
      "--- gene=Chr7-6Mb-seg-855, id=18 removed by soft count = 309\n",
      "--- gene=Chr7-6Mb-seg-855, id=19 removed by soft count = 68\n",
      "--- gene=Chr7-6Mb-seg-855, id=20 removed by soft count = 197\n",
      "--- gene=Chr7-6Mb-seg-855, id=28 removed by soft count = 66\n",
      "--- number of probes: 38 , kept by blast: 24 , if remove dups: 24\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-seg-856\n",
      "--- gene=Chr7-6Mb-seg-856, id=9 removed by soft count = 96\n",
      "--- gene=Chr7-6Mb-seg-856, id=13 removed by soft count = 201\n",
      "--- gene=Chr7-6Mb-seg-856, id=17 removed by soft count = 79\n",
      "--- gene=Chr7-6Mb-seg-856, id=28 removed by soft count = 129\n",
      "--- number of probes: 51 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: Chr7-6Mb-seg-857\n",
      "--- gene=Chr7-6Mb-seg-857, id=4 removed by soft count = 118\n",
      "--- gene=Chr7-6Mb-seg-857, id=7 removed by soft count = 8939\n",
      "--- gene=Chr7-6Mb-seg-857, id=8 removed by soft count = 5940\n",
      "--- gene=Chr7-6Mb-seg-857, id=10 removed by soft count = 79\n",
      "--- gene=Chr7-6Mb-seg-857, id=17 removed by soft count = 359\n",
      "--- gene=Chr7-6Mb-seg-857, id=19 removed by soft count = 2130\n",
      "--- gene=Chr7-6Mb-seg-857, id=32 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-857, id=39 removed by soft count = 560\n",
      "--- gene=Chr7-6Mb-seg-857, id=41 removed by soft count = 347\n",
      "--- number of probes: 42 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: Chr7-6Mb-seg-858\n",
      "--- gene=Chr7-6Mb-seg-858, id=0 removed by soft count = 642\n",
      "--- gene=Chr7-6Mb-seg-858, id=2 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-858, id=4 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-858, id=8 removed by soft count = 693\n",
      "--- gene=Chr7-6Mb-seg-858, id=9 removed by soft count = 1636\n",
      "--- gene=Chr7-6Mb-seg-858, id=10 removed by soft count = 99\n",
      "--- gene=Chr7-6Mb-seg-858, id=17 removed by soft count = 191\n",
      "--- gene=Chr7-6Mb-seg-858, id=18 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-858, id=21 removed by soft count = 652\n",
      "--- gene=Chr7-6Mb-seg-858, id=24 removed by soft count = 330\n",
      "--- gene=Chr7-6Mb-seg-858, id=26 removed by soft count = 441\n",
      "--- gene=Chr7-6Mb-seg-858, id=28 removed by soft count = 127\n",
      "--- gene=Chr7-6Mb-seg-858, id=29 removed by soft count = 140\n",
      "--- gene=Chr7-6Mb-seg-858, id=33 removed by soft count = 680\n",
      "--- gene=Chr7-6Mb-seg-858, id=37 removed by soft count = 453\n",
      "--- gene=Chr7-6Mb-seg-858, id=38 removed by soft count = 327\n",
      "--- gene=Chr7-6Mb-seg-858, id=41 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-858, id=43 removed by soft count = 263\n",
      "--- gene=Chr7-6Mb-seg-858, id=46 removed by soft count = 145\n",
      "--- gene=Chr7-6Mb-seg-858, id=48 removed by soft count = 159\n",
      "--- gene=Chr7-6Mb-seg-858, id=49 removed by soft count = 97\n",
      "--- gene=Chr7-6Mb-seg-858, id=50 removed by soft count = 139\n",
      "--- number of probes: 51 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: Chr7-6Mb-seg-859\n",
      "--- gene=Chr7-6Mb-seg-859, id=0 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-859, id=1 removed by soft count = 265\n",
      "--- gene=Chr7-6Mb-seg-859, id=4 removed by soft count = 274\n",
      "--- gene=Chr7-6Mb-seg-859, id=7 removed by soft count = 55\n",
      "--- number of probes: 27 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-seg-860\n",
      "--- gene=Chr7-6Mb-seg-860, id=16 removed by soft count = 208\n",
      "--- gene=Chr7-6Mb-seg-860, id=17 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-860, id=27 removed by soft count = 178\n",
      "--- gene=Chr7-6Mb-seg-860, id=28 removed by soft count = 210\n",
      "--- gene=Chr7-6Mb-seg-860, id=29 removed by soft count = 1937\n",
      "--- number of probes: 30 , kept by blast: 25 , if remove dups: 24\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-seg-862\n",
      "--- gene=Chr7-6Mb-seg-862, id=2 removed by soft count = 2084\n",
      "--- gene=Chr7-6Mb-seg-862, id=3 removed by soft count = 382\n",
      "--- gene=Chr7-6Mb-seg-862, id=32 removed by soft count = 212\n",
      "--- gene=Chr7-6Mb-seg-862, id=33 removed by soft count = 199\n",
      "--- number of probes: 44 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: Chr7-6Mb-seg-863\n",
      "--- gene=Chr7-6Mb-seg-863, id=5 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-863, id=54 removed by soft count = 116\n",
      "--- gene=Chr7-6Mb-seg-863, id=60 removed by soft count = 129\n",
      "--- gene=Chr7-6Mb-seg-863, id=61 removed by soft count = 154\n",
      "--- gene=Chr7-6Mb-seg-863, id=63 removed by soft count = 187\n",
      "--- number of probes: 65 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-864\n",
      "--- gene=Chr7-6Mb-seg-864, id=1 removed by soft count = 1297\n",
      "--- gene=Chr7-6Mb-seg-864, id=2 removed by soft count = 675\n",
      "--- gene=Chr7-6Mb-seg-864, id=3 removed by soft count = 3865\n",
      "--- gene=Chr7-6Mb-seg-864, id=8 removed by soft count = 652\n",
      "--- gene=Chr7-6Mb-seg-864, id=15 removed by soft count = 385\n",
      "--- gene=Chr7-6Mb-seg-864, id=17 removed by soft count = 612\n",
      "--- gene=Chr7-6Mb-seg-864, id=18 removed by soft count = 154\n",
      "--- gene=Chr7-6Mb-seg-864, id=20 removed by soft count = 286\n",
      "--- gene=Chr7-6Mb-seg-864, id=22 removed by soft count = 73\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-864, id=24 removed by soft count = 2418\n",
      "--- gene=Chr7-6Mb-seg-864, id=34 removed by soft count = 271\n",
      "--- gene=Chr7-6Mb-seg-864, id=35 removed by soft count = 202\n",
      "--- gene=Chr7-6Mb-seg-864, id=37 removed by soft count = 62\n",
      "--- number of probes: 38 , kept by blast: 25 , if remove dups: 25\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-seg-865\n",
      "--- gene=Chr7-6Mb-seg-865, id=0 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-865, id=1 removed by soft count = 242\n",
      "--- gene=Chr7-6Mb-seg-865, id=3 removed by soft count = 297\n",
      "--- gene=Chr7-6Mb-seg-865, id=4 removed by soft count = 99\n",
      "--- gene=Chr7-6Mb-seg-865, id=6 removed by soft count = 663\n",
      "--- gene=Chr7-6Mb-seg-865, id=7 removed by soft count = 750\n",
      "--- gene=Chr7-6Mb-seg-865, id=8 removed by soft count = 554\n",
      "--- gene=Chr7-6Mb-seg-865, id=9 removed by soft count = 352\n",
      "--- gene=Chr7-6Mb-seg-865, id=14 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-865, id=17 removed by soft count = 221\n",
      "--- gene=Chr7-6Mb-seg-865, id=18 removed by soft count = 94\n",
      "--- gene=Chr7-6Mb-seg-865, id=19 removed by soft count = 1656\n",
      "--- gene=Chr7-6Mb-seg-865, id=20 removed by soft count = 160\n",
      "--- gene=Chr7-6Mb-seg-865, id=21 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-865, id=25 removed by soft count = 646\n",
      "--- number of probes: 33 , kept by blast: 18 , if remove dups: 17\n",
      "-- number of probes kept for this region: 18\n",
      "-- checking probes in region: Chr7-6Mb-seg-866\n",
      "--- gene=Chr7-6Mb-seg-866, id=0 removed by soft count = 113\n",
      "--- gene=Chr7-6Mb-seg-866, id=4 removed by soft count = 379\n",
      "--- gene=Chr7-6Mb-seg-866, id=7 removed by soft count = 449\n",
      "--- gene=Chr7-6Mb-seg-866, id=8 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-866, id=9 removed by soft count = 290\n",
      "--- gene=Chr7-6Mb-seg-866, id=10 removed by soft count = 347\n",
      "--- gene=Chr7-6Mb-seg-866, id=34 removed by soft count = 35062\n",
      "--- number of probes: 35 , kept by blast: 28 , if remove dups: 26\n",
      "-- number of probes kept for this region: 28\n",
      "-- checking probes in region: Chr7-6Mb-seg-868\n",
      "--- gene=Chr7-6Mb-seg-868, id=15 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-868, id=16 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-seg-868, id=18 removed by soft count = 16840\n",
      "--- gene=Chr7-6Mb-seg-868, id=19 removed by soft count = 190\n",
      "--- gene=Chr7-6Mb-seg-868, id=21 removed by soft count = 131\n",
      "--- gene=Chr7-6Mb-seg-868, id=23 removed by soft count = 162\n",
      "--- gene=Chr7-6Mb-seg-868, id=25 removed by soft count = 288\n",
      "--- gene=Chr7-6Mb-seg-868, id=26 removed by soft count = 135\n",
      "--- gene=Chr7-6Mb-seg-868, id=28 removed by soft count = 291\n",
      "--- gene=Chr7-6Mb-seg-868, id=32 removed by soft count = 141\n",
      "--- gene=Chr7-6Mb-seg-868, id=33 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-868, id=34 removed by soft count = 260\n",
      "--- gene=Chr7-6Mb-seg-868, id=36 removed by soft count = 79\n",
      "--- gene=Chr7-6Mb-seg-868, id=37 removed by soft count = 566\n",
      "--- number of probes: 43 , kept by blast: 29 , if remove dups: 28\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: Chr7-6Mb-seg-869\n",
      "--- gene=Chr7-6Mb-seg-869, id=17 removed by soft count = 155\n",
      "--- gene=Chr7-6Mb-seg-869, id=18 removed by soft count = 366\n",
      "--- gene=Chr7-6Mb-seg-869, id=19 removed by soft count = 269\n",
      "--- gene=Chr7-6Mb-seg-869, id=20 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-869, id=21 removed by soft count = 1368\n",
      "--- gene=Chr7-6Mb-seg-869, id=22 removed by soft count = 264\n",
      "--- gene=Chr7-6Mb-seg-869, id=26 removed by soft count = 117\n",
      "--- gene=Chr7-6Mb-seg-869, id=29 removed by soft count = 454\n",
      "--- gene=Chr7-6Mb-seg-869, id=30 removed by soft count = 2627\n",
      "--- gene=Chr7-6Mb-seg-869, id=31 removed by soft count = 1400\n",
      "--- gene=Chr7-6Mb-seg-869, id=33 removed by soft count = 2365\n",
      "--- gene=Chr7-6Mb-seg-869, id=34 removed by soft count = 241\n",
      "--- number of probes: 37 , kept by blast: 25 , if remove dups: 25\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-seg-870\n",
      "--- gene=Chr7-6Mb-seg-870, id=0 removed by soft count = 119\n",
      "--- gene=Chr7-6Mb-seg-870, id=1 removed by soft count = 122\n",
      "--- gene=Chr7-6Mb-seg-870, id=3 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-870, id=25 removed by soft count = 639\n",
      "--- gene=Chr7-6Mb-seg-870, id=28 removed by soft count = 230\n",
      "--- gene=Chr7-6Mb-seg-870, id=34 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-870, id=48 removed by soft count = 350\n",
      "--- gene=Chr7-6Mb-seg-870, id=61 removed by soft count = 785\n",
      "--- number of probes: 62 , kept by blast: 54 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-873\n",
      "--- gene=Chr7-6Mb-seg-873, id=2 removed by soft count = 1196\n",
      "--- gene=Chr7-6Mb-seg-873, id=5 removed by soft count = 181\n",
      "--- number of probes: 38 , kept by blast: 36 , if remove dups: 32\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: Chr7-6Mb-seg-876\n",
      "--- gene=Chr7-6Mb-seg-876, id=0 removed by soft count = 1294\n",
      "--- gene=Chr7-6Mb-seg-876, id=1 removed by soft count = 519\n",
      "--- gene=Chr7-6Mb-seg-876, id=2 removed by soft count = 88\n",
      "--- gene=Chr7-6Mb-seg-876, id=3 removed by soft count = 859\n",
      "--- gene=Chr7-6Mb-seg-876, id=4 removed by soft count = 751\n",
      "--- gene=Chr7-6Mb-seg-876, id=5 removed by soft count = 631\n",
      "--- gene=Chr7-6Mb-seg-876, id=6 removed by soft count = 292\n",
      "--- gene=Chr7-6Mb-seg-876, id=50 removed by soft count = 237\n",
      "--- number of probes: 51 , kept by blast: 43 , if remove dups: 42\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: Chr7-6Mb-seg-878\n",
      "--- gene=Chr7-6Mb-seg-878, id=0 removed by soft count = 1491\n",
      "--- gene=Chr7-6Mb-seg-878, id=1 removed by soft count = 786\n",
      "--- gene=Chr7-6Mb-seg-878, id=9 removed by soft count = 52\n",
      "--- number of probes: 44 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: Chr7-6Mb-seg-879\n",
      "--- gene=Chr7-6Mb-seg-879, id=18 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-seg-879, id=23 removed by soft count = 249\n",
      "--- gene=Chr7-6Mb-seg-879, id=24 removed by soft count = 110\n",
      "--- gene=Chr7-6Mb-seg-879, id=25 removed by soft count = 751\n",
      "--- gene=Chr7-6Mb-seg-879, id=27 removed by soft count = 32177\n",
      "--- gene=Chr7-6Mb-seg-879, id=33 removed by soft count = 216\n",
      "--- gene=Chr7-6Mb-seg-879, id=35 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-879, id=36 removed by soft count = 484\n",
      "--- gene=Chr7-6Mb-seg-879, id=39 removed by soft count = 155\n",
      "--- gene=Chr7-6Mb-seg-879, id=43 removed by soft count = 299\n",
      "--- gene=Chr7-6Mb-seg-879, id=44 removed by soft count = 687\n",
      "--- gene=Chr7-6Mb-seg-879, id=45 removed by soft count = 155\n",
      "--- number of probes: 47 , kept by blast: 35 , if remove dups: 34\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: Chr7-6Mb-seg-880\n",
      "--- gene=Chr7-6Mb-seg-880, id=0 removed by soft count = 1931\n",
      "--- gene=Chr7-6Mb-seg-880, id=2 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-880, id=3 removed by soft count = 194\n",
      "--- gene=Chr7-6Mb-seg-880, id=4 removed by soft count = 166\n",
      "--- gene=Chr7-6Mb-seg-880, id=8 removed by soft count = 98\n",
      "--- gene=Chr7-6Mb-seg-880, id=9 removed by soft count = 97\n",
      "--- gene=Chr7-6Mb-seg-880, id=11 removed by soft count = 377\n",
      "--- gene=Chr7-6Mb-seg-880, id=12 removed by soft count = 448\n",
      "--- gene=Chr7-6Mb-seg-880, id=13 removed by soft count = 170\n",
      "--- gene=Chr7-6Mb-seg-880, id=14 removed by soft count = 747\n",
      "--- gene=Chr7-6Mb-seg-880, id=15 removed by soft count = 175\n",
      "--- gene=Chr7-6Mb-seg-880, id=17 removed by soft count = 1113\n",
      "--- number of probes: 40 , kept by blast: 28 , if remove dups: 28\n",
      "-- number of probes kept for this region: 28\n",
      "-- checking probes in region: Chr7-6Mb-seg-881\n",
      "--- gene=Chr7-6Mb-seg-881, id=14 removed by soft count = 79\n",
      "--- gene=Chr7-6Mb-seg-881, id=15 removed by soft count = 232\n",
      "--- gene=Chr7-6Mb-seg-881, id=22 removed by soft count = 67\n",
      "--- gene=Chr7-6Mb-seg-881, id=25 removed by soft count = 98\n",
      "--- gene=Chr7-6Mb-seg-881, id=32 removed by soft count = 193\n",
      "--- gene=Chr7-6Mb-seg-881, id=33 removed by soft count = 83\n",
      "--- number of probes: 34 , kept by blast: 28 , if remove dups: 26\n",
      "-- number of probes kept for this region: 28\n",
      "-- checking probes in region: Chr7-6Mb-seg-883\n",
      "--- gene=Chr7-6Mb-seg-883, id=17 removed by soft count = 4568\n",
      "--- number of probes: 26 , kept by blast: 25 , if remove dups: 24\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-seg-885\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-885, id=0 removed by soft count = 176\n",
      "--- gene=Chr7-6Mb-seg-885, id=31 removed by soft count = 1420\n",
      "--- gene=Chr7-6Mb-seg-885, id=32 removed by soft count = 486\n",
      "--- number of probes: 33 , kept by blast: 30 , if remove dups: 29\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: Chr7-6Mb-seg-888\n",
      "--- gene=Chr7-6Mb-seg-888, id=0 removed by soft count = 201\n",
      "--- gene=Chr7-6Mb-seg-888, id=1 removed by soft count = 1526\n",
      "--- gene=Chr7-6Mb-seg-888, id=2 removed by soft count = 353\n",
      "--- gene=Chr7-6Mb-seg-888, id=3 removed by soft count = 936\n",
      "--- gene=Chr7-6Mb-seg-888, id=5 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-888, id=6 removed by soft count = 686\n",
      "--- gene=Chr7-6Mb-seg-888, id=8 removed by soft count = 550\n",
      "--- gene=Chr7-6Mb-seg-888, id=10 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-888, id=11 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-888, id=12 removed by soft count = 202\n",
      "--- gene=Chr7-6Mb-seg-888, id=13 removed by soft count = 741\n",
      "--- gene=Chr7-6Mb-seg-888, id=14 removed by soft count = 532\n",
      "--- gene=Chr7-6Mb-seg-888, id=16 removed by soft count = 665\n",
      "--- gene=Chr7-6Mb-seg-888, id=20 removed by soft count = 703\n",
      "--- gene=Chr7-6Mb-seg-888, id=21 removed by soft count = 8667\n",
      "--- gene=Chr7-6Mb-seg-888, id=22 removed by soft count = 529\n",
      "--- gene=Chr7-6Mb-seg-888, id=24 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-888, id=26 removed by soft count = 177\n",
      "--- number of probes: 27 , kept by blast: 9 , if remove dups: 9\n",
      "-- number of probes kept for this region: 9\n",
      "-- checking probes in region: Chr7-6Mb-seg-889\n",
      "--- gene=Chr7-6Mb-seg-889, id=0 removed by soft count = 1576\n",
      "--- gene=Chr7-6Mb-seg-889, id=1 removed by soft count = 3662\n",
      "--- gene=Chr7-6Mb-seg-889, id=2 removed by soft count = 194\n",
      "--- gene=Chr7-6Mb-seg-889, id=3 removed by soft count = 314\n",
      "--- gene=Chr7-6Mb-seg-889, id=6 removed by soft count = 843\n",
      "--- gene=Chr7-6Mb-seg-889, id=7 removed by soft count = 712\n",
      "--- gene=Chr7-6Mb-seg-889, id=8 removed by soft count = 299\n",
      "--- gene=Chr7-6Mb-seg-889, id=9 removed by soft count = 923\n",
      "--- gene=Chr7-6Mb-seg-889, id=12 removed by soft count = 253\n",
      "--- gene=Chr7-6Mb-seg-889, id=13 removed by soft count = 170\n",
      "--- gene=Chr7-6Mb-seg-889, id=14 removed by soft count = 160\n",
      "--- gene=Chr7-6Mb-seg-889, id=15 removed by soft count = 149\n",
      "--- gene=Chr7-6Mb-seg-889, id=17 removed by soft count = 70\n",
      "--- number of probes: 58 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: Chr7-6Mb-seg-890\n",
      "--- gene=Chr7-6Mb-seg-890, id=4 removed by soft count = 514\n",
      "--- number of probes: 55 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-891\n",
      "--- gene=Chr7-6Mb-seg-891, id=1 removed by soft count = 129\n",
      "--- gene=Chr7-6Mb-seg-891, id=2 removed by soft count = 366\n",
      "--- gene=Chr7-6Mb-seg-891, id=3 removed by soft count = 163\n",
      "--- gene=Chr7-6Mb-seg-891, id=5 removed by soft count = 489\n",
      "--- gene=Chr7-6Mb-seg-891, id=6 removed by soft count = 130\n",
      "--- gene=Chr7-6Mb-seg-891, id=7 removed by soft count = 97\n",
      "--- gene=Chr7-6Mb-seg-891, id=13 removed by soft count = 5594\n",
      "--- gene=Chr7-6Mb-seg-891, id=15 removed by soft count = 152\n",
      "--- gene=Chr7-6Mb-seg-891, id=16 removed by soft count = 126\n",
      "--- gene=Chr7-6Mb-seg-891, id=19 removed by soft count = 1087\n",
      "--- gene=Chr7-6Mb-seg-891, id=20 removed by soft count = 197\n",
      "--- gene=Chr7-6Mb-seg-891, id=22 removed by soft count = 563\n",
      "--- gene=Chr7-6Mb-seg-891, id=24 removed by soft count = 132\n",
      "--- gene=Chr7-6Mb-seg-891, id=25 removed by soft count = 329\n",
      "--- gene=Chr7-6Mb-seg-891, id=26 removed by soft count = 274\n",
      "--- gene=Chr7-6Mb-seg-891, id=27 removed by soft count = 739\n",
      "--- gene=Chr7-6Mb-seg-891, id=33 removed by soft count = 162\n",
      "--- gene=Chr7-6Mb-seg-891, id=36 removed by soft count = 52\n",
      "--- number of probes: 40 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-seg-892\n",
      "--- gene=Chr7-6Mb-seg-892, id=27 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-892, id=28 removed by soft count = 197\n",
      "--- number of probes: 40 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: Chr7-6Mb-seg-893\n",
      "--- gene=Chr7-6Mb-seg-893, id=10 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-893, id=11 removed by soft count = 196\n",
      "--- gene=Chr7-6Mb-seg-893, id=43 removed by soft count = 238\n",
      "--- gene=Chr7-6Mb-seg-893, id=46 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-893, id=52 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-893, id=54 removed by soft count = 52\n",
      "--- number of probes: 55 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-894\n",
      "--- gene=Chr7-6Mb-seg-894, id=3 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-894, id=5 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-894, id=6 removed by soft count = 400\n",
      "--- gene=Chr7-6Mb-seg-894, id=7 removed by soft count = 88\n",
      "--- gene=Chr7-6Mb-seg-894, id=9 removed by soft count = 307\n",
      "--- gene=Chr7-6Mb-seg-894, id=13 removed by soft count = 440\n",
      "--- gene=Chr7-6Mb-seg-894, id=15 removed by soft count = 345\n",
      "--- gene=Chr7-6Mb-seg-894, id=37 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-894, id=38 removed by soft count = 136\n",
      "--- number of probes: 43 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: Chr7-6Mb-seg-895\n",
      "--- gene=Chr7-6Mb-seg-895, id=2 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-895, id=3 removed by soft count = 227\n",
      "--- gene=Chr7-6Mb-seg-895, id=5 removed by soft count = 100\n",
      "--- gene=Chr7-6Mb-seg-895, id=6 removed by soft count = 129\n",
      "--- gene=Chr7-6Mb-seg-895, id=7 removed by soft count = 526\n",
      "--- gene=Chr7-6Mb-seg-895, id=10 removed by soft count = 194\n",
      "--- gene=Chr7-6Mb-seg-895, id=11 removed by soft count = 533\n",
      "--- gene=Chr7-6Mb-seg-895, id=12 removed by soft count = 132\n",
      "--- gene=Chr7-6Mb-seg-895, id=13 removed by soft count = 230\n",
      "--- gene=Chr7-6Mb-seg-895, id=43 removed by soft count = 283\n",
      "--- number of probes: 46 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: Chr7-6Mb-seg-896\n",
      "--- gene=Chr7-6Mb-seg-896, id=18 removed by soft count = 3975\n",
      "--- gene=Chr7-6Mb-seg-896, id=35 removed by soft count = 82\n",
      "--- number of probes: 60 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-897\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-900\n",
      "--- gene=Chr7-6Mb-seg-900, id=0 removed by soft count = 265\n",
      "--- gene=Chr7-6Mb-seg-900, id=1 removed by soft count = 135\n",
      "--- gene=Chr7-6Mb-seg-900, id=2 removed by soft count = 209\n",
      "--- gene=Chr7-6Mb-seg-900, id=4 removed by soft count = 1480\n",
      "--- gene=Chr7-6Mb-seg-900, id=5 removed by soft count = 594\n",
      "--- gene=Chr7-6Mb-seg-900, id=6 removed by soft count = 1555\n",
      "--- gene=Chr7-6Mb-seg-900, id=7 removed by soft count = 68\n",
      "--- gene=Chr7-6Mb-seg-900, id=8 removed by soft count = 772\n",
      "--- gene=Chr7-6Mb-seg-900, id=9 removed by soft count = 656\n",
      "--- gene=Chr7-6Mb-seg-900, id=10 removed by soft count = 185\n",
      "--- gene=Chr7-6Mb-seg-900, id=11 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-900, id=13 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-900, id=20 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-900, id=23 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-seg-900, id=56 removed by soft count = 236\n",
      "--- number of probes: 58 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: Chr7-6Mb-seg-901\n",
      "--- gene=Chr7-6Mb-seg-901, id=0 removed by soft count = 301\n",
      "--- gene=Chr7-6Mb-seg-901, id=1 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-901, id=4 removed by soft count = 245\n",
      "--- gene=Chr7-6Mb-seg-901, id=5 removed by soft count = 186\n",
      "--- gene=Chr7-6Mb-seg-901, id=6 removed by soft count = 417\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-901, id=8 removed by soft count = 164\n",
      "--- gene=Chr7-6Mb-seg-901, id=10 removed by soft count = 51\n",
      "--- number of probes: 68 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-902\n",
      "--- gene=Chr7-6Mb-seg-902, id=41 removed by soft count = 76\n",
      "--- gene=Chr7-6Mb-seg-902, id=44 removed by soft count = 444\n",
      "--- gene=Chr7-6Mb-seg-902, id=46 removed by soft count = 280\n",
      "--- number of probes: 48 , kept by blast: 45 , if remove dups: 44\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-904\n",
      "--- gene=Chr7-6Mb-seg-904, id=0 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-904, id=1 removed by soft count = 234\n",
      "--- gene=Chr7-6Mb-seg-904, id=2 removed by soft count = 528\n",
      "--- gene=Chr7-6Mb-seg-904, id=3 removed by soft count = 10951\n",
      "--- gene=Chr7-6Mb-seg-904, id=28 removed by soft count = 1345\n",
      "--- number of probes: 36 , kept by blast: 31 , if remove dups: 31\n",
      "-- number of probes kept for this region: 31\n",
      "-- checking probes in region: Chr7-6Mb-seg-906\n",
      "--- gene=Chr7-6Mb-seg-906, id=0 removed by soft count = 904\n",
      "--- gene=Chr7-6Mb-seg-906, id=2 removed by soft count = 1429\n",
      "--- gene=Chr7-6Mb-seg-906, id=3 removed by soft count = 886\n",
      "--- gene=Chr7-6Mb-seg-906, id=4 removed by soft count = 285\n",
      "--- gene=Chr7-6Mb-seg-906, id=5 removed by soft count = 303\n",
      "--- gene=Chr7-6Mb-seg-906, id=6 removed by soft count = 2997\n",
      "--- gene=Chr7-6Mb-seg-906, id=7 removed by soft count = 2518\n",
      "--- gene=Chr7-6Mb-seg-906, id=8 removed by soft count = 202\n",
      "--- gene=Chr7-6Mb-seg-906, id=12 removed by soft count = 86\n",
      "--- gene=Chr7-6Mb-seg-906, id=15 removed by soft count = 160\n",
      "--- gene=Chr7-6Mb-seg-906, id=16 removed by soft count = 95\n",
      "--- gene=Chr7-6Mb-seg-906, id=18 removed by soft count = 109\n",
      "--- gene=Chr7-6Mb-seg-906, id=19 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-906, id=21 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-906, id=22 removed by soft count = 166\n",
      "--- number of probes: 23 , kept by blast: 8 , if remove dups: 8\n",
      "-- number of probes kept for this region: 8\n",
      "-- checking probes in region: Chr7-6Mb-seg-907\n",
      "--- gene=Chr7-6Mb-seg-907, id=0 removed by soft count = 4334\n",
      "--- gene=Chr7-6Mb-seg-907, id=29 removed by soft count = 395\n",
      "--- gene=Chr7-6Mb-seg-907, id=30 removed by soft count = 141\n",
      "--- gene=Chr7-6Mb-seg-907, id=32 removed by soft count = 158\n",
      "--- gene=Chr7-6Mb-seg-907, id=33 removed by soft count = 23714\n",
      "--- gene=Chr7-6Mb-seg-907, id=34 removed by soft count = 91\n",
      "--- gene=Chr7-6Mb-seg-907, id=35 removed by soft count = 3003\n",
      "--- gene=Chr7-6Mb-seg-907, id=36 removed by soft count = 17031\n",
      "--- gene=Chr7-6Mb-seg-907, id=37 removed by soft count = 728\n",
      "--- gene=Chr7-6Mb-seg-907, id=39 removed by soft count = 324\n",
      "--- gene=Chr7-6Mb-seg-907, id=40 removed by soft count = 919\n",
      "--- gene=Chr7-6Mb-seg-907, id=41 removed by soft count = 5112\n",
      "--- number of probes: 42 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: Chr7-6Mb-seg-911\n",
      "--- gene=Chr7-6Mb-seg-911, id=0 removed by soft count = 198\n",
      "--- gene=Chr7-6Mb-seg-911, id=1 removed by soft count = 290\n",
      "--- gene=Chr7-6Mb-seg-911, id=2 removed by soft count = 3623\n",
      "--- gene=Chr7-6Mb-seg-911, id=3 removed by soft count = 282\n",
      "--- number of probes: 50 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-912\n",
      "--- gene=Chr7-6Mb-seg-912, id=6 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-912, id=10 removed by soft count = 1298\n",
      "--- gene=Chr7-6Mb-seg-912, id=60 removed by soft count = 207\n",
      "--- number of probes: 62 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-913\n",
      "--- gene=Chr7-6Mb-seg-913, id=9 removed by soft count = 530\n",
      "--- gene=Chr7-6Mb-seg-913, id=14 removed by soft count = 387\n",
      "--- gene=Chr7-6Mb-seg-913, id=15 removed by soft count = 2085\n",
      "--- gene=Chr7-6Mb-seg-913, id=16 removed by soft count = 2138\n",
      "--- gene=Chr7-6Mb-seg-913, id=17 removed by soft count = 14376\n",
      "--- gene=Chr7-6Mb-seg-913, id=18 removed by soft count = 1781\n",
      "--- gene=Chr7-6Mb-seg-913, id=19 removed by soft count = 284\n",
      "--- gene=Chr7-6Mb-seg-913, id=20 removed by soft count = 1392\n",
      "--- number of probes: 22 , kept by blast: 14 , if remove dups: 14\n",
      "-- number of probes kept for this region: 14\n",
      "-- checking probes in region: Chr7-6Mb-seg-916\n",
      "--- gene=Chr7-6Mb-seg-916, id=1 removed by soft count = 660\n",
      "--- gene=Chr7-6Mb-seg-916, id=2 removed by soft count = 87\n",
      "--- gene=Chr7-6Mb-seg-916, id=18 removed by soft count = 138\n",
      "--- number of probes: 32 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: Chr7-6Mb-seg-917\n",
      "--- gene=Chr7-6Mb-seg-917, id=25 removed by soft count = 243\n",
      "--- gene=Chr7-6Mb-seg-917, id=29 removed by hard count = 0\n",
      "--- gene=Chr7-6Mb-seg-917, id=46 removed by soft count = 956\n",
      "--- gene=Chr7-6Mb-seg-917, id=47 removed by soft count = 1527\n",
      "--- number of probes: 48 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-918\n",
      "--- gene=Chr7-6Mb-seg-918, id=7 removed by soft count = 191\n",
      "--- gene=Chr7-6Mb-seg-918, id=27 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-918, id=28 removed by soft count = 87\n",
      "--- gene=Chr7-6Mb-seg-918, id=30 removed by soft count = 2441\n",
      "--- number of probes: 36 , kept by blast: 32 , if remove dups: 31\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: Chr7-6Mb-seg-919\n",
      "--- gene=Chr7-6Mb-seg-919, id=33 removed by soft count = 248\n",
      "--- number of probes: 34 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: Chr7-6Mb-seg-922\n",
      "--- gene=Chr7-6Mb-seg-922, id=8 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-922, id=26 removed by soft count = 124\n",
      "--- gene=Chr7-6Mb-seg-922, id=30 removed by soft count = 88\n",
      "--- number of probes: 49 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-923\n",
      "--- gene=Chr7-6Mb-seg-923, id=0 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-923, id=3 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-923, id=6 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-923, id=9 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-923, id=12 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-923, id=15 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-923, id=18 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-923, id=21 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-923, id=24 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-923, id=27 removed by soft count = 232\n",
      "--- gene=Chr7-6Mb-seg-923, id=30 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-923, id=33 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-923, id=34 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-923, id=36 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-923, id=37 removed by soft count = 522\n",
      "--- gene=Chr7-6Mb-seg-923, id=38 removed by soft count = 2861\n",
      "--- gene=Chr7-6Mb-seg-923, id=39 removed by soft count = 2442\n",
      "--- gene=Chr7-6Mb-seg-923, id=40 removed by soft count = 662\n",
      "--- number of probes: 41 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-seg-924\n",
      "--- gene=Chr7-6Mb-seg-924, id=14 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-924, id=16 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-924, id=19 removed by soft count = 93\n",
      "--- gene=Chr7-6Mb-seg-924, id=25 removed by soft count = 625\n",
      "--- gene=Chr7-6Mb-seg-924, id=26 removed by soft count = 93\n",
      "--- gene=Chr7-6Mb-seg-924, id=28 removed by soft count = 304\n",
      "--- gene=Chr7-6Mb-seg-924, id=29 removed by soft count = 176\n",
      "--- gene=Chr7-6Mb-seg-924, id=32 removed by soft count = 481\n",
      "--- number of probes: 33 , kept by blast: 25 , if remove dups: 25\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-seg-926\n",
      "--- gene=Chr7-6Mb-seg-926, id=10 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-926, id=11 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-926, id=13 removed by soft count = 548\n",
      "--- gene=Chr7-6Mb-seg-926, id=14 removed by soft count = 168\n",
      "--- gene=Chr7-6Mb-seg-926, id=15 removed by soft count = 535\n",
      "--- gene=Chr7-6Mb-seg-926, id=16 removed by soft count = 154\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-926, id=17 removed by soft count = 245\n",
      "--- gene=Chr7-6Mb-seg-926, id=20 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-926, id=21 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-926, id=31 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-926, id=33 removed by soft count = 294\n",
      "--- gene=Chr7-6Mb-seg-926, id=34 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-seg-926, id=35 removed by soft count = 361\n",
      "--- gene=Chr7-6Mb-seg-926, id=37 removed by soft count = 127\n",
      "--- number of probes: 38 , kept by blast: 24 , if remove dups: 24\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-seg-929\n",
      "--- gene=Chr7-6Mb-seg-929, id=1 removed by soft count = 1782\n",
      "--- gene=Chr7-6Mb-seg-929, id=2 removed by soft count = 288\n",
      "--- gene=Chr7-6Mb-seg-929, id=13 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-929, id=14 removed by soft count = 100\n",
      "--- number of probes: 40 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: Chr7-6Mb-seg-931\n",
      "--- gene=Chr7-6Mb-seg-931, id=0 removed by soft count = 429\n",
      "--- gene=Chr7-6Mb-seg-931, id=1 removed by soft count = 14674\n",
      "--- gene=Chr7-6Mb-seg-931, id=3 removed by soft count = 94\n",
      "--- gene=Chr7-6Mb-seg-931, id=5 removed by soft count = 171\n",
      "--- gene=Chr7-6Mb-seg-931, id=6 removed by soft count = 358\n",
      "--- gene=Chr7-6Mb-seg-931, id=7 removed by soft count = 82\n",
      "--- number of probes: 21 , kept by blast: 15 , if remove dups: 15\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: Chr7-6Mb-seg-932\n",
      "--- gene=Chr7-6Mb-seg-932, id=1 removed by soft count = 262\n",
      "--- gene=Chr7-6Mb-seg-932, id=41 removed by soft count = 1531\n",
      "--- gene=Chr7-6Mb-seg-932, id=47 removed by soft count = 141\n",
      "--- number of probes: 48 , kept by blast: 45 , if remove dups: 44\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: Chr7-6Mb-seg-933\n",
      "--- gene=Chr7-6Mb-seg-933, id=0 removed by soft count = 2458\n",
      "--- gene=Chr7-6Mb-seg-933, id=1 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-933, id=2 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-seg-933, id=4 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-933, id=5 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-933, id=6 removed by soft count = 167\n",
      "--- gene=Chr7-6Mb-seg-933, id=22 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-933, id=24 removed by soft count = 145\n",
      "--- number of probes: 36 , kept by blast: 28 , if remove dups: 28\n",
      "-- number of probes kept for this region: 28\n",
      "-- checking probes in region: Chr7-6Mb-seg-934\n",
      "--- gene=Chr7-6Mb-seg-934, id=17 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-934, id=27 removed by soft count = 54\n",
      "--- number of probes: 60 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-935\n",
      "--- gene=Chr7-6Mb-seg-935, id=1 removed by soft count = 374\n",
      "--- gene=Chr7-6Mb-seg-935, id=5 removed by soft count = 534\n",
      "--- gene=Chr7-6Mb-seg-935, id=9 removed by soft count = 119\n",
      "--- gene=Chr7-6Mb-seg-935, id=12 removed by soft count = 165\n",
      "--- gene=Chr7-6Mb-seg-935, id=21 removed by soft count = 511\n",
      "--- gene=Chr7-6Mb-seg-935, id=24 removed by soft count = 119\n",
      "--- gene=Chr7-6Mb-seg-935, id=25 removed by soft count = 169\n",
      "--- gene=Chr7-6Mb-seg-935, id=26 removed by soft count = 415\n",
      "--- gene=Chr7-6Mb-seg-935, id=27 removed by soft count = 197\n",
      "--- number of probes: 28 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-seg-937\n",
      "--- gene=Chr7-6Mb-seg-937, id=1 removed by soft count = 85\n",
      "--- gene=Chr7-6Mb-seg-937, id=2 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-937, id=3 removed by soft count = 57\n",
      "--- number of probes: 25 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-seg-938\n",
      "--- gene=Chr7-6Mb-seg-938, id=40 removed by soft count = 136\n",
      "--- gene=Chr7-6Mb-seg-938, id=41 removed by soft count = 75\n",
      "--- gene=Chr7-6Mb-seg-938, id=45 removed by soft count = 54\n",
      "--- number of probes: 55 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-939\n",
      "--- gene=Chr7-6Mb-seg-939, id=28 removed by soft count = 317\n",
      "--- gene=Chr7-6Mb-seg-939, id=30 removed by soft count = 58\n",
      "--- gene=Chr7-6Mb-seg-939, id=31 removed by soft count = 237\n",
      "--- gene=Chr7-6Mb-seg-939, id=32 removed by soft count = 54\n",
      "--- number of probes: 34 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: Chr7-6Mb-seg-942\n",
      "--- gene=Chr7-6Mb-seg-942, id=0 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-942, id=6 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-942, id=10 removed by soft count = 233\n",
      "--- gene=Chr7-6Mb-seg-942, id=11 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-942, id=12 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-942, id=15 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-942, id=17 removed by soft count = 530\n",
      "--- gene=Chr7-6Mb-seg-942, id=18 removed by soft count = 95\n",
      "--- gene=Chr7-6Mb-seg-942, id=20 removed by soft count = 198\n",
      "--- gene=Chr7-6Mb-seg-942, id=27 removed by soft count = 177\n",
      "--- gene=Chr7-6Mb-seg-942, id=30 removed by soft count = 583\n",
      "--- gene=Chr7-6Mb-seg-942, id=31 removed by soft count = 416\n",
      "--- gene=Chr7-6Mb-seg-942, id=32 removed by soft count = 187\n",
      "--- gene=Chr7-6Mb-seg-942, id=35 removed by soft count = 75\n",
      "--- gene=Chr7-6Mb-seg-942, id=36 removed by soft count = 240\n",
      "--- gene=Chr7-6Mb-seg-942, id=57 removed by soft count = 1772\n",
      "--- number of probes: 62 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Chr7-6Mb-seg-944\n",
      "--- gene=Chr7-6Mb-seg-944, id=0 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-944, id=27 removed by soft count = 176\n",
      "--- number of probes: 42 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: Chr7-6Mb-seg-945\n",
      "--- gene=Chr7-6Mb-seg-945, id=9 removed by soft count = 55\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-946\n",
      "--- gene=Chr7-6Mb-seg-946, id=11 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-946, id=42 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-946, id=43 removed by soft count = 62\n",
      "--- number of probes: 44 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: Chr7-6Mb-seg-947\n",
      "--- gene=Chr7-6Mb-seg-947, id=9 removed by soft count = 454\n",
      "--- gene=Chr7-6Mb-seg-947, id=11 removed by soft count = 658\n",
      "--- gene=Chr7-6Mb-seg-947, id=12 removed by soft count = 373\n",
      "--- gene=Chr7-6Mb-seg-947, id=13 removed by soft count = 2776\n",
      "--- gene=Chr7-6Mb-seg-947, id=14 removed by soft count = 653\n",
      "--- gene=Chr7-6Mb-seg-947, id=16 removed by soft count = 152\n",
      "--- gene=Chr7-6Mb-seg-947, id=20 removed by soft count = 368\n",
      "--- number of probes: 21 , kept by blast: 14 , if remove dups: 13\n",
      "-- number of probes kept for this region: 14\n",
      "-- checking probes in region: Chr7-6Mb-seg-950\n",
      "--- gene=Chr7-6Mb-seg-950, id=1 removed by soft count = 637\n",
      "--- gene=Chr7-6Mb-seg-950, id=2 removed by soft count = 92\n",
      "--- gene=Chr7-6Mb-seg-950, id=3 removed by soft count = 228\n",
      "--- gene=Chr7-6Mb-seg-950, id=5 removed by soft count = 723\n",
      "--- gene=Chr7-6Mb-seg-950, id=8 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-950, id=10 removed by soft count = 444\n",
      "--- gene=Chr7-6Mb-seg-950, id=11 removed by soft count = 268\n",
      "--- number of probes: 31 , kept by blast: 24 , if remove dups: 24\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-seg-951\n",
      "--- gene=Chr7-6Mb-seg-951, id=21 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-951, id=23 removed by soft count = 169\n",
      "--- gene=Chr7-6Mb-seg-951, id=25 removed by soft count = 619\n",
      "--- gene=Chr7-6Mb-seg-951, id=27 removed by soft count = 771\n",
      "--- gene=Chr7-6Mb-seg-951, id=29 removed by soft count = 259\n",
      "--- number of probes: 61 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-952\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-953\n",
      "--- gene=Chr7-6Mb-seg-953, id=71 removed by soft count = 104\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-954\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-955\n",
      "--- gene=Chr7-6Mb-seg-955, id=20 removed by soft count = 100\n",
      "--- gene=Chr7-6Mb-seg-955, id=21 removed by soft count = 131\n",
      "--- gene=Chr7-6Mb-seg-955, id=22 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-955, id=24 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-955, id=26 removed by soft count = 115\n",
      "--- number of probes: 70 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-956\n",
      "--- gene=Chr7-6Mb-seg-956, id=28 removed by soft count = 640\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-957\n",
      "--- gene=Chr7-6Mb-seg-957, id=32 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-957, id=33 removed by soft count = 283\n",
      "--- gene=Chr7-6Mb-seg-957, id=74 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-957, id=75 removed by soft count = 178\n",
      "--- gene=Chr7-6Mb-seg-957, id=76 removed by soft count = 93\n",
      "--- gene=Chr7-6Mb-seg-957, id=78 removed by soft count = 159\n",
      "--- number of probes: 79 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-958\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-959\n",
      "--- gene=Chr7-6Mb-seg-959, id=52 removed by soft count = 127\n",
      "--- number of probes: 63 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-960\n",
      "--- gene=Chr7-6Mb-seg-960, id=34 removed by soft count = 1733\n",
      "--- gene=Chr7-6Mb-seg-960, id=39 removed by soft count = 75\n",
      "--- gene=Chr7-6Mb-seg-960, id=40 removed by soft count = 322\n",
      "--- number of probes: 69 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-961\n",
      "--- number of probes: 63 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-962\n",
      "--- gene=Chr7-6Mb-seg-962, id=28 removed by soft count = 122\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-963\n",
      "--- gene=Chr7-6Mb-seg-963, id=54 removed by soft count = 85\n",
      "--- number of probes: 56 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-964\n",
      "--- number of probes: 74 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-965\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-966\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-967\n",
      "--- gene=Chr7-6Mb-seg-967, id=18 removed by soft count = 225\n",
      "--- gene=Chr7-6Mb-seg-967, id=19 removed by soft count = 66\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-968\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-969\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-970\n",
      "--- gene=Chr7-6Mb-seg-970, id=74 removed by soft count = 528\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-971\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-972\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-973\n",
      "--- gene=Chr7-6Mb-seg-973, id=68 removed by soft count = 93\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-974\n",
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-975\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-976\n",
      "--- gene=Chr7-6Mb-seg-976, id=22 removed by soft count = 62\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-977\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-978\n",
      "--- gene=Chr7-6Mb-seg-978, id=29 removed by soft count = 160\n",
      "--- number of probes: 79 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-979\n",
      "--- gene=Chr7-6Mb-seg-979, id=66 removed by soft count = 69\n",
      "--- number of probes: 78 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-980\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-981\n",
      "--- gene=Chr7-6Mb-seg-981, id=18 removed by soft count = 70\n",
      "--- number of probes: 72 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-982\n",
      "--- gene=Chr7-6Mb-seg-982, id=78 removed by soft count = 88\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-983\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-984\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-985\n",
      "--- gene=Chr7-6Mb-seg-985, id=6 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-985, id=68 removed by soft count = 281\n",
      "--- number of probes: 69 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-986\n",
      "--- gene=Chr7-6Mb-seg-986, id=27 removed by soft count = 137\n",
      "--- gene=Chr7-6Mb-seg-986, id=34 removed by soft count = 573\n",
      "--- gene=Chr7-6Mb-seg-986, id=36 removed by soft count = 824\n",
      "--- number of probes: 77 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-987\n",
      "--- gene=Chr7-6Mb-seg-987, id=15 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-987, id=16 removed by soft count = 63\n",
      "--- gene=Chr7-6Mb-seg-987, id=17 removed by soft count = 83\n",
      "--- number of probes: 76 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-988\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-989\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-990\n",
      "--- gene=Chr7-6Mb-seg-990, id=8 removed by soft count = 152\n",
      "--- gene=Chr7-6Mb-seg-990, id=10 removed by soft count = 71\n",
      "--- number of probes: 66 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-991\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-992\n",
      "--- gene=Chr7-6Mb-seg-992, id=30 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-992, id=32 removed by soft count = 101\n",
      "--- number of probes: 78 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-993\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-994\n",
      "--- gene=Chr7-6Mb-seg-994, id=49 removed by soft count = 74\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-995\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-996\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-997\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-998\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-999\n",
      "--- gene=Chr7-6Mb-seg-999, id=3 removed by soft count = 122\n",
      "--- gene=Chr7-6Mb-seg-999, id=4 removed by soft count = 210\n",
      "--- number of probes: 71 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1000\n",
      "--- gene=Chr7-6Mb-seg-1000, id=74 removed by soft count = 1147\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1001\n",
      "--- number of probes: 76 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1002\n",
      "--- gene=Chr7-6Mb-seg-1002, id=42 removed by soft count = 59\n",
      "--- gene=Chr7-6Mb-seg-1002, id=43 removed by soft count = 53\n",
      "--- gene=Chr7-6Mb-seg-1002, id=44 removed by soft count = 52\n",
      "--- number of probes: 66 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1003\n",
      "--- gene=Chr7-6Mb-seg-1003, id=0 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1003, id=3 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1003, id=6 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1003, id=9 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1003, id=12 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1003, id=15 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1003, id=18 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1003, id=21 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1003, id=24 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-1003, id=26 removed by soft count = 112\n",
      "--- gene=Chr7-6Mb-seg-1003, id=27 removed by soft count = 113\n",
      "--- gene=Chr7-6Mb-seg-1003, id=30 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1003, id=33 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1003, id=36 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1003, id=39 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1003, id=42 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-1003, id=45 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1003, id=48 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1003, id=51 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1003, id=54 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1003, id=57 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1003, id=60 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1003, id=63 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1003, id=66 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-1003, id=69 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1003, id=72 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1003, id=75 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-1003, id=78 removed by soft count = 102\n",
      "--- number of probes: 80 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-1004\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1005\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1006\n",
      "--- gene=Chr7-6Mb-seg-1006, id=9 removed by soft count = 317\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1007\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1008\n",
      "--- gene=Chr7-6Mb-seg-1008, id=11 removed by soft count = 70\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-1008, id=50 removed by soft count = 229\n",
      "--- number of probes: 76 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1009\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1010\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1011\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1012\n",
      "--- gene=Chr7-6Mb-seg-1012, id=46 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-1012, id=66 removed by soft count = 57\n",
      "--- number of probes: 79 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1013\n",
      "--- gene=Chr7-6Mb-seg-1013, id=47 removed by soft count = 187\n",
      "--- number of probes: 78 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1014\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1015\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1016\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1017\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1018\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1019\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1020\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1021\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1022\n",
      "--- gene=Chr7-6Mb-seg-1022, id=54 removed by soft count = 498\n",
      "--- gene=Chr7-6Mb-seg-1022, id=56 removed by soft count = 183\n",
      "--- gene=Chr7-6Mb-seg-1022, id=59 removed by soft count = 973\n",
      "--- gene=Chr7-6Mb-seg-1022, id=60 removed by soft count = 796\n",
      "--- gene=Chr7-6Mb-seg-1022, id=61 removed by soft count = 736\n",
      "--- gene=Chr7-6Mb-seg-1022, id=62 removed by soft count = 351\n",
      "--- gene=Chr7-6Mb-seg-1022, id=64 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-1022, id=67 removed by soft count = 950\n",
      "--- gene=Chr7-6Mb-seg-1022, id=68 removed by soft count = 833\n",
      "--- number of probes: 70 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1023\n",
      "--- gene=Chr7-6Mb-seg-1023, id=1 removed by soft count = 829\n",
      "--- gene=Chr7-6Mb-seg-1023, id=2 removed by soft count = 745\n",
      "--- gene=Chr7-6Mb-seg-1023, id=3 removed by soft count = 827\n",
      "--- gene=Chr7-6Mb-seg-1023, id=5 removed by soft count = 797\n",
      "--- gene=Chr7-6Mb-seg-1023, id=6 removed by soft count = 724\n",
      "--- gene=Chr7-6Mb-seg-1023, id=8 removed by soft count = 822\n",
      "--- gene=Chr7-6Mb-seg-1023, id=9 removed by soft count = 680\n",
      "--- gene=Chr7-6Mb-seg-1023, id=10 removed by soft count = 878\n",
      "--- gene=Chr7-6Mb-seg-1023, id=12 removed by soft count = 433\n",
      "--- gene=Chr7-6Mb-seg-1023, id=13 removed by soft count = 806\n",
      "--- gene=Chr7-6Mb-seg-1023, id=14 removed by soft count = 779\n",
      "--- gene=Chr7-6Mb-seg-1023, id=19 removed by soft count = 763\n",
      "--- gene=Chr7-6Mb-seg-1023, id=20 removed by soft count = 856\n",
      "--- gene=Chr7-6Mb-seg-1023, id=22 removed by soft count = 789\n",
      "--- gene=Chr7-6Mb-seg-1023, id=24 removed by soft count = 847\n",
      "--- gene=Chr7-6Mb-seg-1023, id=25 removed by soft count = 690\n",
      "--- gene=Chr7-6Mb-seg-1023, id=26 removed by soft count = 833\n",
      "--- gene=Chr7-6Mb-seg-1023, id=27 removed by soft count = 730\n",
      "--- gene=Chr7-6Mb-seg-1023, id=28 removed by soft count = 774\n",
      "--- gene=Chr7-6Mb-seg-1023, id=30 removed by soft count = 780\n",
      "--- gene=Chr7-6Mb-seg-1023, id=31 removed by soft count = 305\n",
      "--- gene=Chr7-6Mb-seg-1023, id=33 removed by soft count = 740\n",
      "--- gene=Chr7-6Mb-seg-1023, id=34 removed by soft count = 370\n",
      "--- gene=Chr7-6Mb-seg-1023, id=37 removed by soft count = 117\n",
      "--- gene=Chr7-6Mb-seg-1023, id=40 removed by soft count = 882\n",
      "--- gene=Chr7-6Mb-seg-1023, id=47 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-1023, id=49 removed by soft count = 743\n",
      "--- gene=Chr7-6Mb-seg-1023, id=50 removed by soft count = 82\n",
      "--- gene=Chr7-6Mb-seg-1023, id=51 removed by soft count = 97\n",
      "--- gene=Chr7-6Mb-seg-1023, id=53 removed by soft count = 280\n",
      "--- number of probes: 67 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: Chr7-6Mb-seg-1024\n",
      "--- gene=Chr7-6Mb-seg-1024, id=4 removed by soft count = 199\n",
      "--- number of probes: 73 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1025\n",
      "--- number of probes: 76 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1026\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1027\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1028\n",
      "--- number of probes: 76 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1029\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1030\n",
      "--- gene=Chr7-6Mb-seg-1030, id=32 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-1030, id=35 removed by soft count = 161\n",
      "--- gene=Chr7-6Mb-seg-1030, id=66 removed by soft count = 10819\n",
      "--- gene=Chr7-6Mb-seg-1030, id=70 removed by soft count = 72\n",
      "--- number of probes: 75 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1031\n",
      "--- gene=Chr7-6Mb-seg-1031, id=24 removed by soft count = 58\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1032\n",
      "--- gene=Chr7-6Mb-seg-1032, id=14 removed by soft count = 156\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1033\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1034\n",
      "--- gene=Chr7-6Mb-seg-1034, id=29 removed by soft count = 55\n",
      "--- gene=Chr7-6Mb-seg-1034, id=63 removed by soft count = 266\n",
      "--- number of probes: 77 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1035\n",
      "--- gene=Chr7-6Mb-seg-1035, id=39 removed by soft count = 205\n",
      "--- number of probes: 78 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1036\n",
      "--- gene=Chr7-6Mb-seg-1036, id=0 removed by soft count = 143\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1037\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1038\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1039\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1040\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1041\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1042\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1043\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1044\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1045\n",
      "--- gene=Chr7-6Mb-seg-1045, id=24 removed by soft count = 90\n",
      "--- number of probes: 79 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1046\n",
      "--- gene=Chr7-6Mb-seg-1046, id=27 removed by soft count = 114\n",
      "--- gene=Chr7-6Mb-seg-1046, id=28 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-1046, id=30 removed by soft count = 178\n",
      "--- gene=Chr7-6Mb-seg-1046, id=36 removed by soft count = 618\n",
      "--- gene=Chr7-6Mb-seg-1046, id=39 removed by soft count = 2930\n",
      "--- gene=Chr7-6Mb-seg-1046, id=40 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-1046, id=41 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1046, id=42 removed by soft count = 54\n",
      "--- number of probes: 57 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: Chr7-6Mb-seg-1047\n",
      "--- gene=Chr7-6Mb-seg-1047, id=22 removed by soft count = 120\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1048\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1049\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1050\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1051\n",
      "--- gene=Chr7-6Mb-seg-1051, id=75 removed by soft count = 69\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1052\n",
      "--- gene=Chr7-6Mb-seg-1052, id=2 removed by soft count = 84\n",
      "--- gene=Chr7-6Mb-seg-1052, id=4 removed by soft count = 62\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1053\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1054\n",
      "--- gene=Chr7-6Mb-seg-1054, id=5 removed by soft count = 119\n",
      "--- gene=Chr7-6Mb-seg-1054, id=36 removed by soft count = 140\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1055\n",
      "--- gene=Chr7-6Mb-seg-1055, id=72 removed by soft count = 78\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1056\n",
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1057\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1058\n",
      "--- gene=Chr7-6Mb-seg-1058, id=60 removed by soft count = 86\n",
      "--- gene=Chr7-6Mb-seg-1058, id=77 removed by soft count = 79\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1059\n",
      "--- gene=Chr7-6Mb-seg-1059, id=71 removed by soft count = 287\n",
      "--- number of probes: 79 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1060\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1061\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1062\n",
      "--- gene=Chr7-6Mb-seg-1062, id=39 removed by hard count = 5\n",
      "--- number of probes: 73 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1063\n",
      "--- gene=Chr7-6Mb-seg-1063, id=10 removed by soft count = 193\n",
      "--- gene=Chr7-6Mb-seg-1063, id=32 removed by soft count = 80\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-1063, id=36 removed by soft count = 63\n",
      "--- number of probes: 74 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1064\n",
      "--- gene=Chr7-6Mb-seg-1064, id=7 removed by soft count = 355\n",
      "--- gene=Chr7-6Mb-seg-1064, id=51 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-1064, id=58 removed by soft count = 60\n",
      "--- number of probes: 71 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1065\n",
      "--- gene=Chr7-6Mb-seg-1065, id=76 removed by soft count = 86\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1066\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1067\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1068\n",
      "--- gene=Chr7-6Mb-seg-1068, id=13 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-1068, id=61 removed by soft count = 57\n",
      "--- number of probes: 67 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1069\n",
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1070\n",
      "--- gene=Chr7-6Mb-seg-1070, id=50 removed by soft count = 287\n",
      "--- number of probes: 74 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1071\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1072\n",
      "--- gene=Chr7-6Mb-seg-1072, id=14 removed by soft count = 181\n",
      "--- gene=Chr7-6Mb-seg-1072, id=39 removed by soft count = 92\n",
      "--- gene=Chr7-6Mb-seg-1072, id=55 removed by soft count = 117319\n",
      "--- number of probes: 61 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: Chr7-6Mb-seg-1073\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1074\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1075\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1076\n",
      "--- gene=Chr7-6Mb-seg-1076, id=22 removed by soft count = 158\n",
      "--- gene=Chr7-6Mb-seg-1076, id=24 removed by soft count = 56\n",
      "--- number of probes: 76 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1077\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1078\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-1079\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1080\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1081\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1082\n",
      "--- gene=Chr7-6Mb-seg-1082, id=9 removed by soft count = 74\n",
      "--- number of probes: 63 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1083\n",
      "--- gene=Chr7-6Mb-seg-1083, id=0 removed by soft count = 108\n",
      "--- gene=Chr7-6Mb-seg-1083, id=3 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1083, id=6 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1083, id=9 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1083, id=12 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1083, id=15 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1083, id=18 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1083, id=21 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1083, id=22 removed by soft count = 144\n",
      "--- gene=Chr7-6Mb-seg-1083, id=24 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1083, id=27 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1083, id=30 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1083, id=33 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-1083, id=36 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1083, id=39 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1083, id=42 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1083, id=45 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1083, id=48 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1083, id=51 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1083, id=54 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1083, id=57 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1083, id=60 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1083, id=63 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1083, id=66 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1083, id=69 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1083, id=72 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1083, id=75 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1083, id=78 removed by soft count = 103\n",
      "--- number of probes: 80 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Chr7-6Mb-seg-1084\n",
      "--- gene=Chr7-6Mb-seg-1084, id=25 removed by soft count = 234\n",
      "--- gene=Chr7-6Mb-seg-1084, id=34 removed by soft count = 150\n",
      "--- number of probes: 76 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1085\n",
      "--- gene=Chr7-6Mb-seg-1085, id=59 removed by soft count = 109\n",
      "--- number of probes: 76 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1086\n",
      "--- gene=Chr7-6Mb-seg-1086, id=73 removed by soft count = 796\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1087\n",
      "--- gene=Chr7-6Mb-seg-1087, id=32 removed by hard count = 7\n",
      "--- number of probes: 62 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1088\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1089\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1090\n",
      "--- gene=Chr7-6Mb-seg-1090, id=55 removed by soft count = 121\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1091\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1092\n",
      "--- gene=Chr7-6Mb-seg-1092, id=11 removed by soft count = 62\n",
      "--- number of probes: 60 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-1093\n",
      "--- gene=Chr7-6Mb-seg-1093, id=5 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-1093, id=6 removed by soft count = 64\n",
      "--- gene=Chr7-6Mb-seg-1093, id=43 removed by soft count = 66\n",
      "--- number of probes: 66 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1094\n",
      "--- gene=Chr7-6Mb-seg-1094, id=27 removed by soft count = 134\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1095\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1096\n",
      "--- gene=Chr7-6Mb-seg-1096, id=20 removed by soft count = 62\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1097\n",
      "--- gene=Chr7-6Mb-seg-1097, id=26 removed by soft count = 2185\n",
      "--- gene=Chr7-6Mb-seg-1097, id=46 removed by soft count = 89\n",
      "--- number of probes: 68 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1098\n",
      "--- gene=Chr7-6Mb-seg-1098, id=62 removed by soft count = 119\n",
      "--- number of probes: 63 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1099\n",
      "--- gene=Chr7-6Mb-seg-1099, id=39 removed by soft count = 193\n",
      "--- gene=Chr7-6Mb-seg-1099, id=40 removed by soft count = 103\n",
      "--- number of probes: 74 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1100\n",
      "--- gene=Chr7-6Mb-seg-1100, id=58 removed by soft count = 552\n",
      "--- number of probes: 72 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1101\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1102\n",
      "--- gene=Chr7-6Mb-seg-1102, id=41 removed by soft count = 85\n",
      "--- gene=Chr7-6Mb-seg-1102, id=43 removed by soft count = 1120\n",
      "--- gene=Chr7-6Mb-seg-1102, id=70 removed by soft count = 148\n",
      "--- number of probes: 76 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1103\n",
      "--- gene=Chr7-6Mb-seg-1103, id=21 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-1103, id=24 removed by soft count = 200\n",
      "--- gene=Chr7-6Mb-seg-1103, id=25 removed by soft count = 198\n",
      "--- number of probes: 76 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1104\n",
      "--- gene=Chr7-6Mb-seg-1104, id=75 removed by soft count = 176\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1105\n",
      "--- gene=Chr7-6Mb-seg-1105, id=3 removed by soft count = 110\n",
      "--- number of probes: 73 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1106\n",
      "--- gene=Chr7-6Mb-seg-1106, id=59 removed by soft count = 133\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1107\n",
      "--- gene=Chr7-6Mb-seg-1107, id=11 removed by soft count = 76\n",
      "--- number of probes: 63 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1108\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1109\n",
      "--- gene=Chr7-6Mb-seg-1109, id=50 removed by soft count = 451\n",
      "--- number of probes: 78 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1110\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1111\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1112\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1113\n",
      "--- gene=Chr7-6Mb-seg-1113, id=23 removed by soft count = 74\n",
      "--- gene=Chr7-6Mb-seg-1113, id=70 removed by soft count = 124\n",
      "--- gene=Chr7-6Mb-seg-1113, id=74 removed by soft count = 67\n",
      "--- number of probes: 75 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1114\n",
      "--- gene=Chr7-6Mb-seg-1114, id=67 removed by soft count = 92\n",
      "--- number of probes: 79 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1115\n",
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1116\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1117\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1118\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1119\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1120\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1121\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1122\n",
      "--- gene=Chr7-6Mb-seg-1122, id=14 removed by soft count = 7513\n",
      "--- number of probes: 78 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1123\n",
      "--- gene=Chr7-6Mb-seg-1123, id=14 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-1123, id=19 removed by soft count = 279\n",
      "--- gene=Chr7-6Mb-seg-1123, id=20 removed by soft count = 300\n",
      "--- gene=Chr7-6Mb-seg-1123, id=39 removed by soft count = 283\n",
      "--- gene=Chr7-6Mb-seg-1123, id=45 removed by soft count = 493\n",
      "--- gene=Chr7-6Mb-seg-1123, id=56 removed by soft count = 4635\n",
      "--- number of probes: 62 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: Chr7-6Mb-seg-1124\n",
      "--- gene=Chr7-6Mb-seg-1124, id=55 removed by soft count = 75\n",
      "--- number of probes: 76 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1125\n",
      "--- gene=Chr7-6Mb-seg-1125, id=50 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-seg-1125, id=51 removed by soft count = 115\n",
      "--- gene=Chr7-6Mb-seg-1125, id=52 removed by soft count = 254\n",
      "--- gene=Chr7-6Mb-seg-1125, id=55 removed by soft count = 175\n",
      "--- gene=Chr7-6Mb-seg-1125, id=56 removed by soft count = 201\n",
      "--- gene=Chr7-6Mb-seg-1125, id=58 removed by soft count = 198\n",
      "--- number of probes: 59 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-1126\n",
      "--- gene=Chr7-6Mb-seg-1126, id=35 removed by soft count = 122\n",
      "--- number of probes: 56 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Chr7-6Mb-seg-1127\n",
      "--- gene=Chr7-6Mb-seg-1127, id=17 removed by soft count = 2351\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1128\n",
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1129\n",
      "--- gene=Chr7-6Mb-seg-1129, id=43 removed by soft count = 604\n",
      "--- gene=Chr7-6Mb-seg-1129, id=52 removed by soft count = 358\n",
      "--- gene=Chr7-6Mb-seg-1129, id=57 removed by soft count = 93\n",
      "--- gene=Chr7-6Mb-seg-1129, id=58 removed by soft count = 79\n",
      "--- number of probes: 75 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1130\n",
      "--- gene=Chr7-6Mb-seg-1130, id=12 removed by soft count = 72\n",
      "--- gene=Chr7-6Mb-seg-1130, id=13 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-1130, id=17 removed by soft count = 190\n",
      "--- number of probes: 65 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1131\n",
      "--- gene=Chr7-6Mb-seg-1131, id=5 removed by soft count = 552\n",
      "--- number of probes: 52 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-1132\n",
      "--- gene=Chr7-6Mb-seg-1132, id=67 removed by soft count = 62\n",
      "--- number of probes: 78 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1133\n",
      "--- gene=Chr7-6Mb-seg-1133, id=5 removed by soft count = 52\n",
      "--- gene=Chr7-6Mb-seg-1133, id=13 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-1133, id=47 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-1133, id=48 removed by soft count = 193\n",
      "--- number of probes: 67 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1134\n",
      "--- gene=Chr7-6Mb-seg-1134, id=2 removed by soft count = 85\n",
      "--- gene=Chr7-6Mb-seg-1134, id=62 removed by soft count = 61\n",
      "--- number of probes: 71 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1135\n",
      "--- gene=Chr7-6Mb-seg-1135, id=9 removed by soft count = 91\n",
      "--- gene=Chr7-6Mb-seg-1135, id=22 removed by soft count = 808\n",
      "--- gene=Chr7-6Mb-seg-1135, id=35 removed by soft count = 194\n",
      "--- number of probes: 54 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-1136\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1137\n",
      "--- gene=Chr7-6Mb-seg-1137, id=45 removed by soft count = 85\n",
      "--- number of probes: 76 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1138\n",
      "--- number of probes: 74 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1139\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1140\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1141\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1142\n",
      "--- gene=Chr7-6Mb-seg-1142, id=18 removed by soft count = 284\n",
      "--- gene=Chr7-6Mb-seg-1142, id=19 removed by soft count = 201\n",
      "--- number of probes: 67 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1143\n",
      "--- gene=Chr7-6Mb-seg-1143, id=23 removed by soft count = 87\n",
      "--- gene=Chr7-6Mb-seg-1143, id=26 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-1143, id=28 removed by soft count = 511\n",
      "--- gene=Chr7-6Mb-seg-1143, id=29 removed by soft count = 467\n",
      "--- number of probes: 58 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: Chr7-6Mb-seg-1144\n",
      "--- number of probes: 48 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-1145\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1146\n",
      "--- gene=Chr7-6Mb-seg-1146, id=1 removed by soft count = 55\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 49 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: Chr7-6Mb-seg-1147\n",
      "--- gene=Chr7-6Mb-seg-1147, id=12 removed by soft count = 229\n",
      "--- gene=Chr7-6Mb-seg-1147, id=13 removed by soft count = 57\n",
      "--- gene=Chr7-6Mb-seg-1147, id=15 removed by soft count = 200\n",
      "--- gene=Chr7-6Mb-seg-1147, id=25 removed by soft count = 110\n",
      "--- gene=Chr7-6Mb-seg-1147, id=27 removed by soft count = 56\n",
      "--- gene=Chr7-6Mb-seg-1147, id=34 removed by soft count = 228\n",
      "--- gene=Chr7-6Mb-seg-1147, id=48 removed by soft count = 73\n",
      "--- gene=Chr7-6Mb-seg-1147, id=51 removed by soft count = 84\n",
      "--- number of probes: 61 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-1148\n",
      "--- gene=Chr7-6Mb-seg-1148, id=1 removed by soft count = 773\n",
      "--- gene=Chr7-6Mb-seg-1148, id=3 removed by soft count = 1314\n",
      "--- gene=Chr7-6Mb-seg-1148, id=4 removed by soft count = 309\n",
      "--- gene=Chr7-6Mb-seg-1148, id=5 removed by soft count = 161\n",
      "--- gene=Chr7-6Mb-seg-1148, id=6 removed by soft count = 448\n",
      "--- gene=Chr7-6Mb-seg-1148, id=8 removed by soft count = 158\n",
      "--- gene=Chr7-6Mb-seg-1148, id=9 removed by soft count = 125\n",
      "--- gene=Chr7-6Mb-seg-1148, id=10 removed by soft count = 276\n",
      "--- gene=Chr7-6Mb-seg-1148, id=11 removed by soft count = 129\n",
      "--- gene=Chr7-6Mb-seg-1148, id=12 removed by soft count = 214\n",
      "--- gene=Chr7-6Mb-seg-1148, id=15 removed by soft count = 710\n",
      "--- gene=Chr7-6Mb-seg-1148, id=17 removed by soft count = 335\n",
      "--- number of probes: 42 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: Chr7-6Mb-seg-1149\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1150\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: Chr7-6Mb-seg-1151\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1152\n",
      "--- gene=Chr7-6Mb-seg-1152, id=32 removed by soft count = 221\n",
      "--- gene=Chr7-6Mb-seg-1152, id=33 removed by soft count = 60\n",
      "--- number of probes: 77 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1153\n",
      "--- gene=Chr7-6Mb-seg-1153, id=56 removed by soft count = 62\n",
      "--- number of probes: 76 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1154\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1155\n",
      "--- gene=Chr7-6Mb-seg-1155, id=36 removed by soft count = 78\n",
      "--- gene=Chr7-6Mb-seg-1155, id=76 removed by soft count = 126\n",
      "--- number of probes: 80 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1156\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1157\n",
      "--- gene=Chr7-6Mb-seg-1157, id=18 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-1157, id=45 removed by soft count = 55\n",
      "--- number of probes: 69 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1158\n",
      "--- gene=Chr7-6Mb-seg-1158, id=0 removed by soft count = 72\n",
      "--- number of probes: 68 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1159\n",
      "--- gene=Chr7-6Mb-seg-1159, id=21 removed by soft count = 236\n",
      "--- number of probes: 64 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1160\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1161\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1162\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1163\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1164\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1165\n",
      "--- gene=Chr7-6Mb-seg-1165, id=47 removed by soft count = 133\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1166\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1167\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1168\n",
      "--- gene=Chr7-6Mb-seg-1168, id=0 removed by soft count = 123\n",
      "--- gene=Chr7-6Mb-seg-1168, id=2 removed by soft count = 71\n",
      "--- number of probes: 74 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1169\n",
      "--- gene=Chr7-6Mb-seg-1169, id=13 removed by soft count = 217\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1170\n",
      "--- gene=Chr7-6Mb-seg-1170, id=31 removed by soft count = 70\n",
      "--- gene=Chr7-6Mb-seg-1170, id=60 removed by soft count = 395\n",
      "--- number of probes: 66 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1171\n",
      "--- gene=Chr7-6Mb-seg-1171, id=0 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-1171, id=38 removed by soft count = 181\n",
      "--- number of probes: 67 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1172\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1173\n",
      "--- gene=Chr7-6Mb-seg-1173, id=9 removed by soft count = 118\n",
      "--- gene=Chr7-6Mb-seg-1173, id=71 removed by soft count = 265\n",
      "--- number of probes: 73 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1174\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Chr7-6Mb-seg-1174, id=13 removed by soft count = 71\n",
      "--- gene=Chr7-6Mb-seg-1174, id=14 removed by soft count = 142\n",
      "--- gene=Chr7-6Mb-seg-1174, id=16 removed by soft count = 459\n",
      "--- number of probes: 71 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1175\n",
      "--- gene=Chr7-6Mb-seg-1175, id=53 removed by soft count = 51\n",
      "--- gene=Chr7-6Mb-seg-1175, id=64 removed by soft count = 1700\n",
      "--- number of probes: 72 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1176\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1177\n",
      "--- gene=Chr7-6Mb-seg-1177, id=48 removed by soft count = 333\n",
      "--- gene=Chr7-6Mb-seg-1177, id=49 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-1177, id=50 removed by soft count = 91\n",
      "--- gene=Chr7-6Mb-seg-1177, id=51 removed by soft count = 391\n",
      "--- number of probes: 55 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: Chr7-6Mb-seg-1178\n",
      "--- number of probes: 76 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1179\n",
      "--- gene=Chr7-6Mb-seg-1179, id=6 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-seg-1179, id=14 removed by soft count = 317\n",
      "--- number of probes: 64 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1180\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1181\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1182\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1183\n",
      "--- gene=Chr7-6Mb-seg-1183, id=0 removed by soft count = 105\n",
      "--- gene=Chr7-6Mb-seg-1183, id=3 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1183, id=6 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1183, id=9 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1183, id=12 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1183, id=15 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-1183, id=18 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1183, id=21 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1183, id=24 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1183, id=27 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1183, id=30 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1183, id=33 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1183, id=36 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1183, id=39 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1183, id=42 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1183, id=45 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1183, id=48 removed by soft count = 107\n",
      "--- gene=Chr7-6Mb-seg-1183, id=51 removed by soft count = 111\n",
      "--- gene=Chr7-6Mb-seg-1183, id=54 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1183, id=57 removed by soft count = 104\n",
      "--- gene=Chr7-6Mb-seg-1183, id=60 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1183, id=63 removed by soft count = 106\n",
      "--- gene=Chr7-6Mb-seg-1183, id=66 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1183, id=69 removed by soft count = 103\n",
      "--- gene=Chr7-6Mb-seg-1183, id=72 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1183, id=75 removed by soft count = 102\n",
      "--- gene=Chr7-6Mb-seg-1183, id=78 removed by soft count = 103\n",
      "--- number of probes: 80 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: Chr7-6Mb-seg-1184\n",
      "--- gene=Chr7-6Mb-seg-1184, id=74 removed by soft count = 215\n",
      "--- number of probes: 79 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1185\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1186\n",
      "--- gene=Chr7-6Mb-seg-1186, id=11 removed by soft count = 506\n",
      "--- gene=Chr7-6Mb-seg-1186, id=31 removed by soft count = 83\n",
      "--- number of probes: 68 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1187\n",
      "--- gene=Chr7-6Mb-seg-1187, id=53 removed by soft count = 755\n",
      "--- gene=Chr7-6Mb-seg-1187, id=54 removed by soft count = 196\n",
      "--- number of probes: 65 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1188\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1189\n",
      "--- gene=Chr7-6Mb-seg-1189, id=3 removed by soft count = 924\n",
      "--- gene=Chr7-6Mb-seg-1189, id=4 removed by soft count = 762\n",
      "--- gene=Chr7-6Mb-seg-1189, id=57 removed by soft count = 62\n",
      "--- gene=Chr7-6Mb-seg-1189, id=59 removed by soft count = 335\n",
      "--- number of probes: 77 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1190\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1191\n",
      "--- gene=Chr7-6Mb-seg-1191, id=6 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-seg-1191, id=31 removed by soft count = 78\n",
      "--- number of probes: 65 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1192\n",
      "--- gene=Chr7-6Mb-seg-1192, id=34 removed by soft count = 101\n",
      "--- gene=Chr7-6Mb-seg-1192, id=36 removed by soft count = 240\n",
      "--- gene=Chr7-6Mb-seg-1192, id=38 removed by soft count = 55\n",
      "--- number of probes: 77 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1193\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1194\n",
      "--- gene=Chr7-6Mb-seg-1194, id=0 removed by soft count = 72\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1195\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1196\n",
      "--- gene=Chr7-6Mb-seg-1196, id=6 removed by soft count = 58\n",
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1197\n",
      "--- gene=Chr7-6Mb-seg-1197, id=0 removed by soft count = 299\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 80 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1198\n",
      "--- gene=Chr7-6Mb-seg-1198, id=65 removed by soft count = 98\n",
      "--- gene=Chr7-6Mb-seg-1198, id=67 removed by soft count = 80\n",
      "--- number of probes: 74 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Chr7-6Mb-seg-1199\n",
      "--- gene=Chr7-6Mb-seg-1199, id=2 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-seg-1199, id=38 removed by soft count = 158\n",
      "--- gene=Chr7-6Mb-seg-1199, id=40 removed by soft count = 153\n",
      "--- gene=Chr7-6Mb-seg-1199, id=41 removed by soft count = 77\n",
      "--- gene=Chr7-6Mb-seg-1199, id=42 removed by soft count = 170\n",
      "--- gene=Chr7-6Mb-seg-1199, id=43 removed by soft count = 590\n",
      "--- number of probes: 51 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: Chr7-6Mb-seg-1200\n",
      "--- gene=Chr7-6Mb-seg-1200, id=49 removed by soft count = 729\n",
      "--- number of probes: 63 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "- summarize\n",
      "-- region: Chr7-6Mb-seg-1 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-2 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-3 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-4 , number of probes: 50\n",
      "-- region: Chr7-6Mb-seg-7 , number of probes: 16\n",
      "-- region: Chr7-6Mb-seg-9 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-10 , number of probes: 40\n",
      "-- region: Chr7-6Mb-seg-11 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-12 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-14 , number of probes: 20\n",
      "-- region: Chr7-6Mb-seg-15 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-16 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-19 , number of probes: 35\n",
      "-- region: Chr7-6Mb-seg-20 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-21 , number of probes: 40\n",
      "-- region: Chr7-6Mb-seg-22 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-23 , number of probes: 42\n",
      "-- region: Chr7-6Mb-seg-24 , number of probes: 50\n",
      "-- region: Chr7-6Mb-seg-25 , number of probes: 41\n",
      "-- region: Chr7-6Mb-seg-27 , number of probes: 41\n",
      "-- region: Chr7-6Mb-seg-28 , number of probes: 57\n",
      "-- region: Chr7-6Mb-seg-29 , number of probes: 35\n",
      "-- region: Chr7-6Mb-seg-36 , number of probes: 29\n",
      "-- region: Chr7-6Mb-seg-38 , number of probes: 31\n",
      "-- region: Chr7-6Mb-seg-39 , number of probes: 37\n",
      "-- region: Chr7-6Mb-seg-40 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-41 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-42 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-43 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-44 , number of probes: 35\n",
      "-- region: Chr7-6Mb-seg-45 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-46 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-47 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-48 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-49 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-50 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-51 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-52 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-53 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-54 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-55 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-56 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-57 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-58 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-59 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-60 , number of probes: 42\n",
      "-- region: Chr7-6Mb-seg-61 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-62 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-63 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-64 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-65 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-66 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-67 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-68 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-69 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-70 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-71 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-72 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-73 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-74 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-75 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-76 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-77 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-78 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-79 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-80 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-81 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-82 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-83 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-84 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-85 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-86 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-87 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-88 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-89 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-90 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-91 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-92 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-93 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-94 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-95 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-96 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-97 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-98 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-99 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-100 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-101 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-102 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-103 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-104 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-105 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-106 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-107 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-108 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-109 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-110 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-111 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-112 , number of probes: 57\n",
      "-- region: Chr7-6Mb-seg-113 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-114 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-115 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-116 , number of probes: 37\n",
      "-- region: Chr7-6Mb-seg-117 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-118 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-119 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-120 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-121 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-122 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-123 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-124 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-125 , number of probes: 57\n",
      "-- region: Chr7-6Mb-seg-126 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-127 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-128 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-129 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-130 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-131 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-132 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-133 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-134 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-135 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-136 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-137 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-138 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-139 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-140 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-141 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-142 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-143 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-144 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-145 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-146 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-147 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-148 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-149 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-150 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-151 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-152 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-153 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-154 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-155 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-156 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-157 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-158 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-159 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-161 , number of probes: 26\n",
      "-- region: Chr7-6Mb-seg-162 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-163 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-164 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-165 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-166 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-167 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-168 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-169 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-170 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-171 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-172 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-173 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-174 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-175 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-176 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-177 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-178 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-179 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-180 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-181 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-182 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-183 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-184 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-185 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-186 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-187 , number of probes: 50\n",
      "-- region: Chr7-6Mb-seg-188 , number of probes: 50\n",
      "-- region: Chr7-6Mb-seg-189 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-190 , number of probes: 50\n",
      "-- region: Chr7-6Mb-seg-191 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-192 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-193 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-194 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-195 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-196 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-197 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-198 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-199 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-200 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-201 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-202 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-203 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-204 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-205 , number of probes: 57\n",
      "-- region: Chr7-6Mb-seg-206 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-207 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-208 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-209 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-210 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-211 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-212 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-213 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-214 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-215 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-216 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-217 , number of probes: 57\n",
      "-- region: Chr7-6Mb-seg-218 , number of probes: 57\n",
      "-- region: Chr7-6Mb-seg-219 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-220 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-221 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-222 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-223 , number of probes: 50\n",
      "-- region: Chr7-6Mb-seg-224 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-225 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-226 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-227 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-228 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-229 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-230 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-231 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-232 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-233 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-234 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-235 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-236 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-237 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-238 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-239 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-240 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-241 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-242 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-243 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-244 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-245 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-246 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-247 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-248 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-249 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-250 , number of probes: 12\n",
      "-- region: Chr7-6Mb-seg-251 , number of probes: 39\n",
      "-- region: Chr7-6Mb-seg-252 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-253 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-254 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-255 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-256 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-257 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-258 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-259 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-260 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-261 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-262 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-263 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-264 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-265 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-266 , number of probes: 50\n",
      "-- region: Chr7-6Mb-seg-267 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-268 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-269 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-270 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-271 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-272 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-273 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-274 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-275 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-276 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-277 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-278 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-279 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-280 , number of probes: 42\n",
      "-- region: Chr7-6Mb-seg-281 , number of probes: 47\n",
      "-- region: Chr7-6Mb-seg-282 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-283 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-284 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-285 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-286 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-287 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-288 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-289 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-290 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-291 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-292 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-293 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-294 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-295 , number of probes: 43\n",
      "-- region: Chr7-6Mb-seg-296 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-297 , number of probes: 32\n",
      "-- region: Chr7-6Mb-seg-298 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-299 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-300 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-301 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-302 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-303 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-304 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-305 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-306 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-307 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-308 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-309 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-310 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-311 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-312 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-313 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-314 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-315 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-316 , number of probes: 41\n",
      "-- region: Chr7-6Mb-seg-317 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-318 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-319 , number of probes: 43\n",
      "-- region: Chr7-6Mb-seg-320 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-321 , number of probes: 31\n",
      "-- region: Chr7-6Mb-seg-322 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-323 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-324 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-325 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-326 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-327 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-328 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-329 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-330 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-331 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-332 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-333 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-334 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-335 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-336 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-337 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-338 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-339 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-340 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-341 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-342 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-343 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-344 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-345 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-346 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-347 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-348 , number of probes: 41\n",
      "-- region: Chr7-6Mb-seg-349 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-350 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-351 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-352 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-353 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-354 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-355 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-356 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-357 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-358 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-359 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-360 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-361 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-362 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-363 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-364 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-365 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-366 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-367 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-368 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-369 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-370 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-371 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-372 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-373 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-374 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-375 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-376 , number of probes: 42\n",
      "-- region: Chr7-6Mb-seg-377 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-378 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-379 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-380 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-381 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-382 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-383 , number of probes: 37\n",
      "-- region: Chr7-6Mb-seg-384 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-385 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-386 , number of probes: 33\n",
      "-- region: Chr7-6Mb-seg-387 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-388 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-389 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-390 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-391 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-392 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-393 , number of probes: 40\n",
      "-- region: Chr7-6Mb-seg-394 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-395 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-396 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-397 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-398 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-399 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-400 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-401 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-402 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-403 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-404 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-405 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-406 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-407 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-408 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-409 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-410 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-411 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-412 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-413 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-414 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-415 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-416 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-417 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-418 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-419 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-420 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-421 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-422 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-423 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-424 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-425 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-426 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-427 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-428 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-429 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-430 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-431 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-432 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-433 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-434 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-435 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-436 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-437 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-438 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-439 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-440 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-441 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-442 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-443 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-444 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-445 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-446 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-447 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-448 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-449 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-450 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-451 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-452 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-453 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-454 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-455 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-456 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-457 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-458 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-459 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-460 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-461 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-462 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-463 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-464 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-465 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-466 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-467 , number of probes: 47\n",
      "-- region: Chr7-6Mb-seg-468 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-469 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-470 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-471 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-472 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-473 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-474 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-475 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-476 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-477 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-478 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-479 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-480 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-481 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-482 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-483 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-484 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-485 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-486 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-487 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-488 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-489 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-490 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-491 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-492 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-493 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-494 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-495 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-496 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-497 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-498 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-499 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-500 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-501 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-502 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-503 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-504 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-505 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-506 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-507 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-508 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-509 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-510 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-511 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-512 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-513 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-514 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-515 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-516 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-517 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-518 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-519 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-520 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-521 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-522 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-523 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-524 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-525 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-526 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-527 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-528 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-529 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-530 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-531 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-532 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-533 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-534 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-535 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-536 , number of probes: 45\n",
      "-- region: Chr7-6Mb-seg-537 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-538 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-539 , number of probes: 29\n",
      "-- region: Chr7-6Mb-seg-540 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-541 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-542 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-543 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-544 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-545 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-546 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-547 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-548 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-549 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-550 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-551 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-552 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-553 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-554 , number of probes: 57\n",
      "-- region: Chr7-6Mb-seg-555 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-556 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-557 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-558 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-559 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-560 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-561 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-562 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-563 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-564 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-565 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-566 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-567 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-568 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-569 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-570 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-571 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-572 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-573 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-574 , number of probes: 45\n",
      "-- region: Chr7-6Mb-seg-575 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-576 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-577 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-578 , number of probes: 45\n",
      "-- region: Chr7-6Mb-seg-579 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-580 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-581 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-582 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-583 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-584 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-585 , number of probes: 37\n",
      "-- region: Chr7-6Mb-seg-586 , number of probes: 47\n",
      "-- region: Chr7-6Mb-seg-587 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-588 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-589 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-590 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-591 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-592 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-593 , number of probes: 57\n",
      "-- region: Chr7-6Mb-seg-594 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-595 , number of probes: 29\n",
      "-- region: Chr7-6Mb-seg-596 , number of probes: 43\n",
      "-- region: Chr7-6Mb-seg-597 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-598 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-599 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-600 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-601 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-602 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-603 , number of probes: 47\n",
      "-- region: Chr7-6Mb-seg-604 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-605 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-606 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-607 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-608 , number of probes: 38\n",
      "-- region: Chr7-6Mb-seg-609 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-610 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-611 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-612 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-613 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-614 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-615 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-616 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-617 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-618 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-619 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-620 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-621 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-622 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-623 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-624 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-625 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-626 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-627 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-628 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-629 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-630 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-631 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-632 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-633 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-634 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-635 , number of probes: 57\n",
      "-- region: Chr7-6Mb-seg-638 , number of probes: 24\n",
      "-- region: Chr7-6Mb-seg-639 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-640 , number of probes: 24\n",
      "-- region: Chr7-6Mb-seg-641 , number of probes: 36\n",
      "-- region: Chr7-6Mb-seg-642 , number of probes: 47\n",
      "-- region: Chr7-6Mb-seg-643 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-644 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-647 , number of probes: 17\n",
      "-- region: Chr7-6Mb-seg-648 , number of probes: 45\n",
      "-- region: Chr7-6Mb-seg-649 , number of probes: 21\n",
      "-- region: Chr7-6Mb-seg-650 , number of probes: 37\n",
      "-- region: Chr7-6Mb-seg-651 , number of probes: 57\n",
      "-- region: Chr7-6Mb-seg-655 , number of probes: 32\n",
      "-- region: Chr7-6Mb-seg-660 , number of probes: 31\n",
      "-- region: Chr7-6Mb-seg-661 , number of probes: 41\n",
      "-- region: Chr7-6Mb-seg-662 , number of probes: 42\n",
      "-- region: Chr7-6Mb-seg-663 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-664 , number of probes: 40\n",
      "-- region: Chr7-6Mb-seg-665 , number of probes: 22\n",
      "-- region: Chr7-6Mb-seg-668 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-669 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-670 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-671 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-672 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-674 , number of probes: 15\n",
      "-- region: Chr7-6Mb-seg-675 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-676 , number of probes: 30\n",
      "-- region: Chr7-6Mb-seg-677 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-679 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-680 , number of probes: 29\n",
      "-- region: Chr7-6Mb-seg-681 , number of probes: 31\n",
      "-- region: Chr7-6Mb-seg-682 , number of probes: 45\n",
      "-- region: Chr7-6Mb-seg-683 , number of probes: 23\n",
      "-- region: Chr7-6Mb-seg-692 , number of probes: 30\n",
      "-- region: Chr7-6Mb-seg-693 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-698 , number of probes: 31\n",
      "-- region: Chr7-6Mb-seg-699 , number of probes: 25\n",
      "-- region: Chr7-6Mb-seg-700 , number of probes: 33\n",
      "-- region: Chr7-6Mb-seg-701 , number of probes: 26\n",
      "-- region: Chr7-6Mb-seg-703 , number of probes: 23\n",
      "-- region: Chr7-6Mb-seg-704 , number of probes: 39\n",
      "-- region: Chr7-6Mb-seg-705 , number of probes: 27\n",
      "-- region: Chr7-6Mb-seg-706 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-707 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-708 , number of probes: 29\n",
      "-- region: Chr7-6Mb-seg-710 , number of probes: 28\n",
      "-- region: Chr7-6Mb-seg-711 , number of probes: 13\n",
      "-- region: Chr7-6Mb-seg-712 , number of probes: 25\n",
      "-- region: Chr7-6Mb-seg-713 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-717 , number of probes: 23\n",
      "-- region: Chr7-6Mb-seg-718 , number of probes: 19\n",
      "-- region: Chr7-6Mb-seg-719 , number of probes: 24\n",
      "-- region: Chr7-6Mb-seg-720 , number of probes: 38\n",
      "-- region: Chr7-6Mb-seg-721 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-722 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-723 , number of probes: 25\n",
      "-- region: Chr7-6Mb-seg-726 , number of probes: 38\n",
      "-- region: Chr7-6Mb-seg-727 , number of probes: 39\n",
      "-- region: Chr7-6Mb-seg-728 , number of probes: 20\n",
      "-- region: Chr7-6Mb-seg-731 , number of probes: 36\n",
      "-- region: Chr7-6Mb-seg-732 , number of probes: 13\n",
      "-- region: Chr7-6Mb-seg-733 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-734 , number of probes: 17\n",
      "-- region: Chr7-6Mb-seg-735 , number of probes: 42\n",
      "-- region: Chr7-6Mb-seg-738 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-739 , number of probes: 40\n",
      "-- region: Chr7-6Mb-seg-740 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-741 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-742 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-743 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-744 , number of probes: 41\n",
      "-- region: Chr7-6Mb-seg-745 , number of probes: 31\n",
      "-- region: Chr7-6Mb-seg-746 , number of probes: 33\n",
      "-- region: Chr7-6Mb-seg-747 , number of probes: 19\n",
      "-- region: Chr7-6Mb-seg-748 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-749 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-750 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-751 , number of probes: 32\n",
      "-- region: Chr7-6Mb-seg-752 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-753 , number of probes: 45\n",
      "-- region: Chr7-6Mb-seg-754 , number of probes: 6\n",
      "-- region: Chr7-6Mb-seg-755 , number of probes: 42\n",
      "-- region: Chr7-6Mb-seg-756 , number of probes: 43\n",
      "-- region: Chr7-6Mb-seg-757 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-758 , number of probes: 45\n",
      "-- region: Chr7-6Mb-seg-760 , number of probes: 22\n",
      "-- region: Chr7-6Mb-seg-762 , number of probes: 20\n",
      "-- region: Chr7-6Mb-seg-763 , number of probes: 37\n",
      "-- region: Chr7-6Mb-seg-764 , number of probes: 40\n",
      "-- region: Chr7-6Mb-seg-765 , number of probes: 18\n",
      "-- region: Chr7-6Mb-seg-766 , number of probes: 45\n",
      "-- region: Chr7-6Mb-seg-767 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-768 , number of probes: 39\n",
      "-- region: Chr7-6Mb-seg-769 , number of probes: 20\n",
      "-- region: Chr7-6Mb-seg-770 , number of probes: 15\n",
      "-- region: Chr7-6Mb-seg-771 , number of probes: 21\n",
      "-- region: Chr7-6Mb-seg-772 , number of probes: 37\n",
      "-- region: Chr7-6Mb-seg-773 , number of probes: 27\n",
      "-- region: Chr7-6Mb-seg-775 , number of probes: 26\n",
      "-- region: Chr7-6Mb-seg-776 , number of probes: 41\n",
      "-- region: Chr7-6Mb-seg-777 , number of probes: 30\n",
      "-- region: Chr7-6Mb-seg-778 , number of probes: 39\n",
      "-- region: Chr7-6Mb-seg-780 , number of probes: 26\n",
      "-- region: Chr7-6Mb-seg-781 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-784 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-785 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-786 , number of probes: 37\n",
      "-- region: Chr7-6Mb-seg-787 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-788 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-790 , number of probes: 23\n",
      "-- region: Chr7-6Mb-seg-791 , number of probes: 23\n",
      "-- region: Chr7-6Mb-seg-792 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-793 , number of probes: 21\n",
      "-- region: Chr7-6Mb-seg-794 , number of probes: 35\n",
      "-- region: Chr7-6Mb-seg-795 , number of probes: 21\n",
      "-- region: Chr7-6Mb-seg-796 , number of probes: 13\n",
      "-- region: Chr7-6Mb-seg-797 , number of probes: 41\n",
      "-- region: Chr7-6Mb-seg-798 , number of probes: 47\n",
      "-- region: Chr7-6Mb-seg-799 , number of probes: 43\n",
      "-- region: Chr7-6Mb-seg-800 , number of probes: 27\n",
      "-- region: Chr7-6Mb-seg-801 , number of probes: 22\n",
      "-- region: Chr7-6Mb-seg-802 , number of probes: 43\n",
      "-- region: Chr7-6Mb-seg-804 , number of probes: 38\n",
      "-- region: Chr7-6Mb-seg-805 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-806 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-807 , number of probes: 47\n",
      "-- region: Chr7-6Mb-seg-808 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-809 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-810 , number of probes: 24\n",
      "-- region: Chr7-6Mb-seg-812 , number of probes: 12\n",
      "-- region: Chr7-6Mb-seg-813 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-814 , number of probes: 17\n",
      "-- region: Chr7-6Mb-seg-816 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-817 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-818 , number of probes: 33\n",
      "-- region: Chr7-6Mb-seg-819 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-820 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-821 , number of probes: 30\n",
      "-- region: Chr7-6Mb-seg-822 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-823 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-824 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-825 , number of probes: 19\n",
      "-- region: Chr7-6Mb-seg-828 , number of probes: 22\n",
      "-- region: Chr7-6Mb-seg-829 , number of probes: 21\n",
      "-- region: Chr7-6Mb-seg-831 , number of probes: 19\n",
      "-- region: Chr7-6Mb-seg-832 , number of probes: 22\n",
      "-- region: Chr7-6Mb-seg-833 , number of probes: 15\n",
      "-- region: Chr7-6Mb-seg-834 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-835 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-836 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-837 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-840 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-841 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-842 , number of probes: 47\n",
      "-- region: Chr7-6Mb-seg-845 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-846 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-847 , number of probes: 9\n",
      "-- region: Chr7-6Mb-seg-848 , number of probes: 26\n",
      "-- region: Chr7-6Mb-seg-849 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-850 , number of probes: 36\n",
      "-- region: Chr7-6Mb-seg-852 , number of probes: 22\n",
      "-- region: Chr7-6Mb-seg-853 , number of probes: 39\n",
      "-- region: Chr7-6Mb-seg-854 , number of probes: 31\n",
      "-- region: Chr7-6Mb-seg-855 , number of probes: 24\n",
      "-- region: Chr7-6Mb-seg-856 , number of probes: 47\n",
      "-- region: Chr7-6Mb-seg-857 , number of probes: 33\n",
      "-- region: Chr7-6Mb-seg-858 , number of probes: 29\n",
      "-- region: Chr7-6Mb-seg-859 , number of probes: 23\n",
      "-- region: Chr7-6Mb-seg-860 , number of probes: 25\n",
      "-- region: Chr7-6Mb-seg-862 , number of probes: 40\n",
      "-- region: Chr7-6Mb-seg-863 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-864 , number of probes: 25\n",
      "-- region: Chr7-6Mb-seg-865 , number of probes: 18\n",
      "-- region: Chr7-6Mb-seg-866 , number of probes: 28\n",
      "-- region: Chr7-6Mb-seg-868 , number of probes: 29\n",
      "-- region: Chr7-6Mb-seg-869 , number of probes: 25\n",
      "-- region: Chr7-6Mb-seg-870 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-873 , number of probes: 36\n",
      "-- region: Chr7-6Mb-seg-876 , number of probes: 43\n",
      "-- region: Chr7-6Mb-seg-878 , number of probes: 41\n",
      "-- region: Chr7-6Mb-seg-879 , number of probes: 35\n",
      "-- region: Chr7-6Mb-seg-880 , number of probes: 28\n",
      "-- region: Chr7-6Mb-seg-881 , number of probes: 28\n",
      "-- region: Chr7-6Mb-seg-883 , number of probes: 25\n",
      "-- region: Chr7-6Mb-seg-885 , number of probes: 30\n",
      "-- region: Chr7-6Mb-seg-888 , number of probes: 9\n",
      "-- region: Chr7-6Mb-seg-889 , number of probes: 45\n",
      "-- region: Chr7-6Mb-seg-890 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-891 , number of probes: 22\n",
      "-- region: Chr7-6Mb-seg-892 , number of probes: 38\n",
      "-- region: Chr7-6Mb-seg-893 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-894 , number of probes: 34\n",
      "-- region: Chr7-6Mb-seg-895 , number of probes: 36\n",
      "-- region: Chr7-6Mb-seg-896 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-897 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-900 , number of probes: 43\n",
      "-- region: Chr7-6Mb-seg-901 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-902 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-904 , number of probes: 31\n",
      "-- region: Chr7-6Mb-seg-906 , number of probes: 8\n",
      "-- region: Chr7-6Mb-seg-907 , number of probes: 30\n",
      "-- region: Chr7-6Mb-seg-911 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-912 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-913 , number of probes: 14\n",
      "-- region: Chr7-6Mb-seg-916 , number of probes: 29\n",
      "-- region: Chr7-6Mb-seg-917 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-918 , number of probes: 32\n",
      "-- region: Chr7-6Mb-seg-919 , number of probes: 33\n",
      "-- region: Chr7-6Mb-seg-922 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-923 , number of probes: 23\n",
      "-- region: Chr7-6Mb-seg-924 , number of probes: 25\n",
      "-- region: Chr7-6Mb-seg-926 , number of probes: 24\n",
      "-- region: Chr7-6Mb-seg-929 , number of probes: 36\n",
      "-- region: Chr7-6Mb-seg-931 , number of probes: 15\n",
      "-- region: Chr7-6Mb-seg-932 , number of probes: 44\n",
      "-- region: Chr7-6Mb-seg-933 , number of probes: 28\n",
      "-- region: Chr7-6Mb-seg-934 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-935 , number of probes: 19\n",
      "-- region: Chr7-6Mb-seg-937 , number of probes: 22\n",
      "-- region: Chr7-6Mb-seg-938 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-939 , number of probes: 30\n",
      "-- region: Chr7-6Mb-seg-942 , number of probes: 46\n",
      "-- region: Chr7-6Mb-seg-944 , number of probes: 40\n",
      "-- region: Chr7-6Mb-seg-945 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-946 , number of probes: 41\n",
      "-- region: Chr7-6Mb-seg-947 , number of probes: 14\n",
      "-- region: Chr7-6Mb-seg-950 , number of probes: 24\n",
      "-- region: Chr7-6Mb-seg-951 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-952 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-953 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-954 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-955 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-956 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-957 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-958 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-959 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-960 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-961 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-962 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-963 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-964 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-965 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-966 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-967 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-968 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-969 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-970 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-971 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-972 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-973 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-974 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-975 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-976 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-977 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-978 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-979 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-980 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-981 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-982 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-983 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-984 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-985 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-986 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-987 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-988 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-989 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-990 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-991 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-992 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-993 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-994 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-995 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-996 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-997 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-998 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-999 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1000 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1001 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1002 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1003 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-1004 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1005 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1006 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1007 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1008 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1009 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1010 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1011 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1012 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1013 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1014 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1015 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1016 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1017 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1018 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1019 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1020 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1021 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1022 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1023 , number of probes: 37\n",
      "-- region: Chr7-6Mb-seg-1024 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1025 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1026 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1027 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1028 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1029 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1030 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1031 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1032 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1033 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1034 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1035 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1036 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1037 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1038 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1039 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1040 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1041 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1042 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1043 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1044 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1045 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1046 , number of probes: 49\n",
      "-- region: Chr7-6Mb-seg-1047 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1048 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1049 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1050 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1051 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1052 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1053 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1054 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1055 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1056 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1057 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1058 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1059 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1060 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1061 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1062 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1063 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1064 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1065 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1066 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1067 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1068 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1069 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1070 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1071 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1072 , number of probes: 58\n",
      "-- region: Chr7-6Mb-seg-1073 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1074 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1075 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1076 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1077 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1078 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-1079 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1080 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1081 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1082 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1083 , number of probes: 52\n",
      "-- region: Chr7-6Mb-seg-1084 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1085 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1086 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1087 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1088 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1089 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1090 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1091 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1092 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-1093 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1094 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1095 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1096 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1097 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1098 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1099 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1100 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1101 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1102 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1103 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1104 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1105 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1106 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1107 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1108 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1109 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1110 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1111 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1112 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1113 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1114 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1115 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1116 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1117 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1118 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1119 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1120 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1121 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1122 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1123 , number of probes: 56\n",
      "-- region: Chr7-6Mb-seg-1124 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1125 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-1126 , number of probes: 55\n",
      "-- region: Chr7-6Mb-seg-1127 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1128 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1129 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1130 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1131 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-1132 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1133 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1134 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1135 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-1136 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1137 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1138 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1139 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1140 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1141 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1142 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1143 , number of probes: 54\n",
      "-- region: Chr7-6Mb-seg-1144 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-1145 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1146 , number of probes: 48\n",
      "-- region: Chr7-6Mb-seg-1147 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-1148 , number of probes: 30\n",
      "-- region: Chr7-6Mb-seg-1149 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1150 , number of probes: 59\n",
      "-- region: Chr7-6Mb-seg-1151 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1152 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1153 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1154 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1155 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1156 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1157 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1158 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1159 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1160 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1161 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1162 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1163 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1164 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1165 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1166 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1167 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1168 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1169 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1170 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1171 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1172 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1173 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1174 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1175 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1176 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1177 , number of probes: 51\n",
      "-- region: Chr7-6Mb-seg-1178 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1179 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1180 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1181 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1182 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1183 , number of probes: 53\n",
      "-- region: Chr7-6Mb-seg-1184 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1185 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1186 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1187 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1188 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1189 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1190 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1191 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1192 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1193 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1194 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1195 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1196 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1197 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1198 , number of probes: 60\n",
      "-- region: Chr7-6Mb-seg-1199 , number of probes: 45\n",
      "-- region: Chr7-6Mb-seg-1200 , number of probes: 60\n",
      "- Number of probes kept: 58492\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\.\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "num_probes_per_region = 60\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(library_folder, \n",
    "                                                                                             num_probes_per_region,\n",
    "                                                                                             keep_mode='front',\n",
    "                                                                                             hard_thres=target_len,\n",
    "                                                                                             soft_count_th=50,\n",
    "                                                                                             )"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d06655b7",
   "metadata": {},
   "source": [
    "# 5. Summary Table and plots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 240,
   "id": "b4ee64e3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "58492\n"
     ]
    }
   ],
   "source": [
    "blasted_records = []\n",
    "with open(os.path.join(library_folder, 'blast_full_probes.fasta'), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        blasted_records.append(record)\n",
    "print(len(blasted_records))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 241,
   "id": "89b0500e",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "\n",
    "region_info_dict = {\n",
    "    'gene_name':[],\n",
    "    'locus_info':[],\n",
    "    'chr':[],\n",
    "    #'chr_order':[],\n",
    "    'num_probes':[],\n",
    "    'readouts': [],\n",
    "    #'type':[],\n",
    "}\n",
    "\n",
    "probe_name_re_string = r'loc_(?P<loc>.+)_gene_(?P<gene>.+)_pb_(?P<pb>[0-9]+)_pos_(?P<pos>[0-9]+)(.+)_readouts_\\[(?P<readouts>.+?)\\]_primers_\\[(?P<fwd_primers>.+?),(?P<rev_primers>.+?)\\]'\n",
    "\n",
    "for _pb in blasted_records:\n",
    "    \n",
    "    _match = re.match(probe_name_re_string, _pb.id, )\n",
    "    # append all if not exist\n",
    "    if _match.groupdict()['gene'] not in region_info_dict['gene_name']:\n",
    "        # append new\n",
    "        region_info_dict['gene_name'].append(_match.groupdict()['gene'])\n",
    "        region_info_dict['locus_info'].append(_match.groupdict()['loc'])\n",
    "        region_info_dict['chr'].append(_match.groupdict()['loc'].split(':')[0])\n",
    "        region_info_dict['num_probes'].append(1)\n",
    "        region_info_dict['readouts'].append(list(np.unique(_match.groupdict()['readouts'].split(','))))\n",
    "    else:\n",
    "        _ind = region_info_dict['gene_name'].index(_match.groupdict()['gene'])\n",
    "        region_info_dict['num_probes'][_ind] += 1\n",
    "        for _rd in np.unique(_match.groupdict()['readouts'].split(',')):\n",
    "            if _rd not in region_info_dict['readouts'][_ind]:\n",
    "                region_info_dict['readouts'][_ind].append(_rd)\n",
    "        \n",
    "    #break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 242,
   "id": "99d35478",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene_name</th>\n",
       "      <th>locus_info</th>\n",
       "      <th>chr</th>\n",
       "      <th>num_probes</th>\n",
       "      <th>readouts</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Chr7-6Mb-seg-1</td>\n",
       "      <td>7:43999999-44004999</td>\n",
       "      <td>7</td>\n",
       "      <td>52</td>\n",
       "      <td>[NDB_801_c, NDB_725_c, NDB_710_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Chr7-6Mb-seg-2</td>\n",
       "      <td>7:44004999-44009999</td>\n",
       "      <td>7</td>\n",
       "      <td>59</td>\n",
       "      <td>[NDB_400_c, NDB_750_c, NDB_825_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Chr7-6Mb-seg-3</td>\n",
       "      <td>7:44009999-44014999</td>\n",
       "      <td>7</td>\n",
       "      <td>48</td>\n",
       "      <td>[NDB_458_c, NDB_466_c, NDB_511_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Chr7-6Mb-seg-4</td>\n",
       "      <td>7:44014999-44019999</td>\n",
       "      <td>7</td>\n",
       "      <td>50</td>\n",
       "      <td>[NDB_786_c, NDB_860_c, NDB_680_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Chr7-6Mb-seg-7</td>\n",
       "      <td>7:44029999-44034999</td>\n",
       "      <td>7</td>\n",
       "      <td>16</td>\n",
       "      <td>[NDB_681_c, NDB_754_c, NDB_1020_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1092</th>\n",
       "      <td>Chr7-6Mb-seg-1196</td>\n",
       "      <td>7:49974999-49979999</td>\n",
       "      <td>7</td>\n",
       "      <td>60</td>\n",
       "      <td>[NDB_585_c, NDB_818_c, NDB_494_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1093</th>\n",
       "      <td>Chr7-6Mb-seg-1197</td>\n",
       "      <td>7:49979999-49984999</td>\n",
       "      <td>7</td>\n",
       "      <td>60</td>\n",
       "      <td>[NDB_510_c, NDB_1056_c, NDB_709_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1094</th>\n",
       "      <td>Chr7-6Mb-seg-1198</td>\n",
       "      <td>7:49984999-49989999</td>\n",
       "      <td>7</td>\n",
       "      <td>60</td>\n",
       "      <td>[NDB_707_c, NDB_1042_c, NDB_904_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1095</th>\n",
       "      <td>Chr7-6Mb-seg-1199</td>\n",
       "      <td>7:49989999-49994999</td>\n",
       "      <td>7</td>\n",
       "      <td>45</td>\n",
       "      <td>[NDB_873_c, NDB_902_c, NDB_488_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1096</th>\n",
       "      <td>Chr7-6Mb-seg-1200</td>\n",
       "      <td>7:49994999-49999999</td>\n",
       "      <td>7</td>\n",
       "      <td>60</td>\n",
       "      <td>[NDB_796_c, NDB_393_c, NDB_429_c]</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1097 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "              gene_name           locus_info chr  num_probes  \\\n",
       "0        Chr7-6Mb-seg-1  7:43999999-44004999   7          52   \n",
       "1        Chr7-6Mb-seg-2  7:44004999-44009999   7          59   \n",
       "2        Chr7-6Mb-seg-3  7:44009999-44014999   7          48   \n",
       "3        Chr7-6Mb-seg-4  7:44014999-44019999   7          50   \n",
       "4        Chr7-6Mb-seg-7  7:44029999-44034999   7          16   \n",
       "...                 ...                  ...  ..         ...   \n",
       "1092  Chr7-6Mb-seg-1196  7:49974999-49979999   7          60   \n",
       "1093  Chr7-6Mb-seg-1197  7:49979999-49984999   7          60   \n",
       "1094  Chr7-6Mb-seg-1198  7:49984999-49989999   7          60   \n",
       "1095  Chr7-6Mb-seg-1199  7:49989999-49994999   7          45   \n",
       "1096  Chr7-6Mb-seg-1200  7:49994999-49999999   7          60   \n",
       "\n",
       "                                readouts  \n",
       "0      [NDB_801_c, NDB_725_c, NDB_710_c]  \n",
       "1      [NDB_400_c, NDB_750_c, NDB_825_c]  \n",
       "2      [NDB_458_c, NDB_466_c, NDB_511_c]  \n",
       "3      [NDB_786_c, NDB_860_c, NDB_680_c]  \n",
       "4     [NDB_681_c, NDB_754_c, NDB_1020_c]  \n",
       "...                                  ...  \n",
       "1092   [NDB_585_c, NDB_818_c, NDB_494_c]  \n",
       "1093  [NDB_510_c, NDB_1056_c, NDB_709_c]  \n",
       "1094  [NDB_707_c, NDB_1042_c, NDB_904_c]  \n",
       "1095   [NDB_873_c, NDB_902_c, NDB_488_c]  \n",
       "1096   [NDB_796_c, NDB_393_c, NDB_429_c]  \n",
       "\n",
       "[1097 rows x 5 columns]"
      ]
     },
     "execution_count": 242,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "region_info_df = pd.DataFrame(region_info_dict)\n",
    "region_info_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 243,
   "id": "1ba832a8",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Add additional information\n",
    "_types = []\n",
    "for _g in region_info_df['gene_name']:\n",
    "    if 'hSE-' in _g:\n",
    "        _types.append('SE')\n",
    "    elif 'enhancer-' in _g:\n",
    "        _types.append('enhancer')\n",
    "    else:\n",
    "        _types.append('genome')\n",
    "# add to this df\n",
    "region_info_df['type'] = _types\n",
    "\n",
    "region_info_df['chr_order'] = -1\n",
    "for _chr in np.unique(region_info_df['chr']):\n",
    "    _reg_starts = [int(_info.split(':')[1].split('-')[0])\n",
    "                   for _info in region_info_df.loc[region_info_df['chr']==_chr,\n",
    "                                                   'locus_info']\n",
    "                  ]\n",
    "    _reg_orders = np.array(_reg_starts).argsort().argsort().astype(np.int32)\n",
    "    region_info_df.loc[region_info_df['chr']==_chr,'chr_order'] = _reg_orders"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 244,
   "id": "1076083e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene_name</th>\n",
       "      <th>locus_info</th>\n",
       "      <th>chr</th>\n",
       "      <th>num_probes</th>\n",
       "      <th>readouts</th>\n",
       "      <th>type</th>\n",
       "      <th>chr_order</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Chr7-6Mb-seg-1</td>\n",
       "      <td>7:43999999-44004999</td>\n",
       "      <td>7</td>\n",
       "      <td>52</td>\n",
       "      <td>[NDB_801_c, NDB_725_c, NDB_710_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Chr7-6Mb-seg-2</td>\n",
       "      <td>7:44004999-44009999</td>\n",
       "      <td>7</td>\n",
       "      <td>59</td>\n",
       "      <td>[NDB_400_c, NDB_750_c, NDB_825_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Chr7-6Mb-seg-3</td>\n",
       "      <td>7:44009999-44014999</td>\n",
       "      <td>7</td>\n",
       "      <td>48</td>\n",
       "      <td>[NDB_458_c, NDB_466_c, NDB_511_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Chr7-6Mb-seg-4</td>\n",
       "      <td>7:44014999-44019999</td>\n",
       "      <td>7</td>\n",
       "      <td>50</td>\n",
       "      <td>[NDB_786_c, NDB_860_c, NDB_680_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Chr7-6Mb-seg-7</td>\n",
       "      <td>7:44029999-44034999</td>\n",
       "      <td>7</td>\n",
       "      <td>16</td>\n",
       "      <td>[NDB_681_c, NDB_754_c, NDB_1020_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1092</th>\n",
       "      <td>Chr7-6Mb-seg-1196</td>\n",
       "      <td>7:49974999-49979999</td>\n",
       "      <td>7</td>\n",
       "      <td>60</td>\n",
       "      <td>[NDB_585_c, NDB_818_c, NDB_494_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>1092</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1093</th>\n",
       "      <td>Chr7-6Mb-seg-1197</td>\n",
       "      <td>7:49979999-49984999</td>\n",
       "      <td>7</td>\n",
       "      <td>60</td>\n",
       "      <td>[NDB_510_c, NDB_1056_c, NDB_709_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>1093</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1094</th>\n",
       "      <td>Chr7-6Mb-seg-1198</td>\n",
       "      <td>7:49984999-49989999</td>\n",
       "      <td>7</td>\n",
       "      <td>60</td>\n",
       "      <td>[NDB_707_c, NDB_1042_c, NDB_904_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>1094</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1095</th>\n",
       "      <td>Chr7-6Mb-seg-1199</td>\n",
       "      <td>7:49989999-49994999</td>\n",
       "      <td>7</td>\n",
       "      <td>45</td>\n",
       "      <td>[NDB_873_c, NDB_902_c, NDB_488_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>1095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1096</th>\n",
       "      <td>Chr7-6Mb-seg-1200</td>\n",
       "      <td>7:49994999-49999999</td>\n",
       "      <td>7</td>\n",
       "      <td>60</td>\n",
       "      <td>[NDB_796_c, NDB_393_c, NDB_429_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>1096</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1097 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "              gene_name           locus_info chr  num_probes  \\\n",
       "0        Chr7-6Mb-seg-1  7:43999999-44004999   7          52   \n",
       "1        Chr7-6Mb-seg-2  7:44004999-44009999   7          59   \n",
       "2        Chr7-6Mb-seg-3  7:44009999-44014999   7          48   \n",
       "3        Chr7-6Mb-seg-4  7:44014999-44019999   7          50   \n",
       "4        Chr7-6Mb-seg-7  7:44029999-44034999   7          16   \n",
       "...                 ...                  ...  ..         ...   \n",
       "1092  Chr7-6Mb-seg-1196  7:49974999-49979999   7          60   \n",
       "1093  Chr7-6Mb-seg-1197  7:49979999-49984999   7          60   \n",
       "1094  Chr7-6Mb-seg-1198  7:49984999-49989999   7          60   \n",
       "1095  Chr7-6Mb-seg-1199  7:49989999-49994999   7          45   \n",
       "1096  Chr7-6Mb-seg-1200  7:49994999-49999999   7          60   \n",
       "\n",
       "                                readouts    type  chr_order  \n",
       "0      [NDB_801_c, NDB_725_c, NDB_710_c]  genome          0  \n",
       "1      [NDB_400_c, NDB_750_c, NDB_825_c]  genome          1  \n",
       "2      [NDB_458_c, NDB_466_c, NDB_511_c]  genome          2  \n",
       "3      [NDB_786_c, NDB_860_c, NDB_680_c]  genome          3  \n",
       "4     [NDB_681_c, NDB_754_c, NDB_1020_c]  genome          4  \n",
       "...                                  ...     ...        ...  \n",
       "1092   [NDB_585_c, NDB_818_c, NDB_494_c]  genome       1092  \n",
       "1093  [NDB_510_c, NDB_1056_c, NDB_709_c]  genome       1093  \n",
       "1094  [NDB_707_c, NDB_1042_c, NDB_904_c]  genome       1094  \n",
       "1095   [NDB_873_c, NDB_902_c, NDB_488_c]  genome       1095  \n",
       "1096   [NDB_796_c, NDB_393_c, NDB_429_c]  genome       1096  \n",
       "\n",
       "[1097 rows x 7 columns]"
      ]
     },
     "execution_count": 244,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_info_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 245,
   "id": "c3d2f5a2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "saving region info to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\summary_region_info.csv\n"
     ]
    }
   ],
   "source": [
    "overwrite_region_info = True\n",
    "region_info_filename = os.path.join(library_folder, r'summary_region_info.csv')\n",
    "\n",
    "if not os.path.exists(region_info_filename) or overwrite_region_info:\n",
    "    print(f\"saving region info to file: {region_info_filename}\")\n",
    "    region_info_df.to_csv(region_info_filename, index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0d1a3125",
   "metadata": {},
   "source": [
    "## Generate summary plots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 246,
   "id": "f0cc3784",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: Chr7-6Mb-seg-1 -> 52\n",
      "gene: Chr7-6Mb-seg-2 -> 59\n",
      "gene: Chr7-6Mb-seg-3 -> 48\n",
      "gene: Chr7-6Mb-seg-4 -> 50\n",
      "gene: Chr7-6Mb-seg-7 -> 16\n",
      "gene: Chr7-6Mb-seg-9 -> 58\n",
      "gene: Chr7-6Mb-seg-10 -> 40\n",
      "gene: Chr7-6Mb-seg-11 -> 44\n",
      "gene: Chr7-6Mb-seg-12 -> 34\n",
      "gene: Chr7-6Mb-seg-14 -> 20\n",
      "gene: Chr7-6Mb-seg-15 -> 60\n",
      "gene: Chr7-6Mb-seg-16 -> 54\n",
      "gene: Chr7-6Mb-seg-19 -> 35\n",
      "gene: Chr7-6Mb-seg-20 -> 48\n",
      "gene: Chr7-6Mb-seg-21 -> 40\n",
      "gene: Chr7-6Mb-seg-22 -> 46\n",
      "gene: Chr7-6Mb-seg-23 -> 42\n",
      "gene: Chr7-6Mb-seg-24 -> 50\n",
      "gene: Chr7-6Mb-seg-25 -> 41\n",
      "gene: Chr7-6Mb-seg-27 -> 41\n",
      "gene: Chr7-6Mb-seg-28 -> 57\n",
      "gene: Chr7-6Mb-seg-29 -> 35\n",
      "gene: Chr7-6Mb-seg-36 -> 29\n",
      "gene: Chr7-6Mb-seg-38 -> 31\n",
      "gene: Chr7-6Mb-seg-39 -> 37\n",
      "gene: Chr7-6Mb-seg-40 -> 46\n",
      "gene: Chr7-6Mb-seg-41 -> 56\n",
      "gene: Chr7-6Mb-seg-42 -> 53\n",
      "gene: Chr7-6Mb-seg-43 -> 60\n",
      "gene: Chr7-6Mb-seg-44 -> 35\n",
      "gene: Chr7-6Mb-seg-45 -> 48\n",
      "gene: Chr7-6Mb-seg-46 -> 58\n",
      "gene: Chr7-6Mb-seg-47 -> 54\n",
      "gene: Chr7-6Mb-seg-48 -> 60\n",
      "gene: Chr7-6Mb-seg-49 -> 58\n",
      "gene: Chr7-6Mb-seg-50 -> 60\n",
      "gene: Chr7-6Mb-seg-51 -> 60\n",
      "gene: Chr7-6Mb-seg-52 -> 60\n",
      "gene: Chr7-6Mb-seg-53 -> 60\n",
      "gene: Chr7-6Mb-seg-54 -> 60\n",
      "gene: Chr7-6Mb-seg-55 -> 60\n",
      "gene: Chr7-6Mb-seg-56 -> 60\n",
      "gene: Chr7-6Mb-seg-57 -> 60\n",
      "gene: Chr7-6Mb-seg-58 -> 58\n",
      "gene: Chr7-6Mb-seg-59 -> 51\n",
      "gene: Chr7-6Mb-seg-60 -> 42\n",
      "gene: Chr7-6Mb-seg-61 -> 60\n",
      "gene: Chr7-6Mb-seg-62 -> 60\n",
      "gene: Chr7-6Mb-seg-63 -> 60\n",
      "gene: Chr7-6Mb-seg-64 -> 60\n",
      "gene: Chr7-6Mb-seg-65 -> 60\n",
      "gene: Chr7-6Mb-seg-66 -> 60\n",
      "gene: Chr7-6Mb-seg-67 -> 60\n",
      "gene: Chr7-6Mb-seg-68 -> 60\n",
      "gene: Chr7-6Mb-seg-69 -> 60\n",
      "gene: Chr7-6Mb-seg-70 -> 60\n",
      "gene: Chr7-6Mb-seg-71 -> 60\n",
      "gene: Chr7-6Mb-seg-72 -> 60\n",
      "gene: Chr7-6Mb-seg-73 -> 60\n",
      "gene: Chr7-6Mb-seg-74 -> 52\n",
      "gene: Chr7-6Mb-seg-75 -> 60\n",
      "gene: Chr7-6Mb-seg-76 -> 60\n",
      "gene: Chr7-6Mb-seg-77 -> 60\n",
      "gene: Chr7-6Mb-seg-78 -> 60\n",
      "gene: Chr7-6Mb-seg-79 -> 60\n",
      "gene: Chr7-6Mb-seg-80 -> 60\n",
      "gene: Chr7-6Mb-seg-81 -> 60\n",
      "gene: Chr7-6Mb-seg-82 -> 60\n",
      "gene: Chr7-6Mb-seg-83 -> 60\n",
      "gene: Chr7-6Mb-seg-84 -> 52\n",
      "gene: Chr7-6Mb-seg-85 -> 60\n",
      "gene: Chr7-6Mb-seg-86 -> 60\n",
      "gene: Chr7-6Mb-seg-87 -> 60\n",
      "gene: Chr7-6Mb-seg-88 -> 60\n",
      "gene: Chr7-6Mb-seg-89 -> 60\n",
      "gene: Chr7-6Mb-seg-90 -> 60\n",
      "gene: Chr7-6Mb-seg-91 -> 48\n",
      "gene: Chr7-6Mb-seg-92 -> 60\n",
      "gene: Chr7-6Mb-seg-93 -> 60\n",
      "gene: Chr7-6Mb-seg-94 -> 60\n",
      "gene: Chr7-6Mb-seg-95 -> 60\n",
      "gene: Chr7-6Mb-seg-96 -> 46\n",
      "gene: Chr7-6Mb-seg-97 -> 49\n",
      "gene: Chr7-6Mb-seg-98 -> 60\n",
      "gene: Chr7-6Mb-seg-99 -> 60\n",
      "gene: Chr7-6Mb-seg-100 -> 60\n",
      "gene: Chr7-6Mb-seg-101 -> 60\n",
      "gene: Chr7-6Mb-seg-102 -> 60\n",
      "gene: Chr7-6Mb-seg-103 -> 60\n",
      "gene: Chr7-6Mb-seg-104 -> 60\n",
      "gene: Chr7-6Mb-seg-105 -> 60\n",
      "gene: Chr7-6Mb-seg-106 -> 60\n",
      "gene: Chr7-6Mb-seg-107 -> 60\n",
      "gene: Chr7-6Mb-seg-108 -> 60\n",
      "gene: Chr7-6Mb-seg-109 -> 60\n",
      "gene: Chr7-6Mb-seg-110 -> 60\n",
      "gene: Chr7-6Mb-seg-111 -> 60\n",
      "gene: Chr7-6Mb-seg-112 -> 57\n",
      "gene: Chr7-6Mb-seg-113 -> 60\n",
      "gene: Chr7-6Mb-seg-114 -> 44\n",
      "gene: Chr7-6Mb-seg-115 -> 56\n",
      "gene: Chr7-6Mb-seg-116 -> 37\n",
      "gene: Chr7-6Mb-seg-117 -> 56\n",
      "gene: Chr7-6Mb-seg-118 -> 60\n",
      "gene: Chr7-6Mb-seg-119 -> 60\n",
      "gene: Chr7-6Mb-seg-120 -> 60\n",
      "gene: Chr7-6Mb-seg-121 -> 60\n",
      "gene: Chr7-6Mb-seg-122 -> 60\n",
      "gene: Chr7-6Mb-seg-123 -> 60\n",
      "gene: Chr7-6Mb-seg-124 -> 59\n",
      "gene: Chr7-6Mb-seg-125 -> 57\n",
      "gene: Chr7-6Mb-seg-126 -> 60\n",
      "gene: Chr7-6Mb-seg-127 -> 60\n",
      "gene: Chr7-6Mb-seg-128 -> 60\n",
      "gene: Chr7-6Mb-seg-129 -> 60\n",
      "gene: Chr7-6Mb-seg-130 -> 60\n",
      "gene: Chr7-6Mb-seg-131 -> 60\n",
      "gene: Chr7-6Mb-seg-132 -> 60\n",
      "gene: Chr7-6Mb-seg-133 -> 60\n",
      "gene: Chr7-6Mb-seg-134 -> 60\n",
      "gene: Chr7-6Mb-seg-135 -> 60\n",
      "gene: Chr7-6Mb-seg-136 -> 60\n",
      "gene: Chr7-6Mb-seg-137 -> 60\n",
      "gene: Chr7-6Mb-seg-138 -> 60\n",
      "gene: Chr7-6Mb-seg-139 -> 60\n",
      "gene: Chr7-6Mb-seg-140 -> 58\n",
      "gene: Chr7-6Mb-seg-141 -> 59\n",
      "gene: Chr7-6Mb-seg-142 -> 55\n",
      "gene: Chr7-6Mb-seg-143 -> 60\n",
      "gene: Chr7-6Mb-seg-144 -> 60\n",
      "gene: Chr7-6Mb-seg-145 -> 60\n",
      "gene: Chr7-6Mb-seg-146 -> 55\n",
      "gene: Chr7-6Mb-seg-147 -> 60\n",
      "gene: Chr7-6Mb-seg-148 -> 60\n",
      "gene: Chr7-6Mb-seg-149 -> 53\n",
      "gene: Chr7-6Mb-seg-150 -> 53\n",
      "gene: Chr7-6Mb-seg-151 -> 52\n",
      "gene: Chr7-6Mb-seg-152 -> 52\n",
      "gene: Chr7-6Mb-seg-153 -> 60\n",
      "gene: Chr7-6Mb-seg-154 -> 60\n",
      "gene: Chr7-6Mb-seg-155 -> 60\n",
      "gene: Chr7-6Mb-seg-156 -> 59\n",
      "gene: Chr7-6Mb-seg-157 -> 60\n",
      "gene: Chr7-6Mb-seg-158 -> 60\n",
      "gene: Chr7-6Mb-seg-159 -> 52\n",
      "gene: Chr7-6Mb-seg-161 -> 26\n",
      "gene: Chr7-6Mb-seg-162 -> 54\n",
      "gene: Chr7-6Mb-seg-163 -> 60\n",
      "gene: Chr7-6Mb-seg-164 -> 60\n",
      "gene: Chr7-6Mb-seg-165 -> 60\n",
      "gene: Chr7-6Mb-seg-166 -> 60\n",
      "gene: Chr7-6Mb-seg-167 -> 60\n",
      "gene: Chr7-6Mb-seg-168 -> 60\n",
      "gene: Chr7-6Mb-seg-169 -> 60\n",
      "gene: Chr7-6Mb-seg-170 -> 60\n",
      "gene: Chr7-6Mb-seg-171 -> 60\n",
      "gene: Chr7-6Mb-seg-172 -> 60\n",
      "gene: Chr7-6Mb-seg-173 -> 60\n",
      "gene: Chr7-6Mb-seg-174 -> 60\n",
      "gene: Chr7-6Mb-seg-175 -> 55\n",
      "gene: Chr7-6Mb-seg-176 -> 60\n",
      "gene: Chr7-6Mb-seg-177 -> 60\n",
      "gene: Chr7-6Mb-seg-178 -> 60\n",
      "gene: Chr7-6Mb-seg-179 -> 60\n",
      "gene: Chr7-6Mb-seg-180 -> 60\n",
      "gene: Chr7-6Mb-seg-181 -> 60\n",
      "gene: Chr7-6Mb-seg-182 -> 60\n",
      "gene: Chr7-6Mb-seg-183 -> 60\n",
      "gene: Chr7-6Mb-seg-184 -> 60\n",
      "gene: Chr7-6Mb-seg-185 -> 55\n",
      "gene: Chr7-6Mb-seg-186 -> 60\n",
      "gene: Chr7-6Mb-seg-187 -> 50\n",
      "gene: Chr7-6Mb-seg-188 -> 50\n",
      "gene: Chr7-6Mb-seg-189 -> 52\n",
      "gene: Chr7-6Mb-seg-190 -> 50\n",
      "gene: Chr7-6Mb-seg-191 -> 60\n",
      "gene: Chr7-6Mb-seg-192 -> 60\n",
      "gene: Chr7-6Mb-seg-193 -> 60\n",
      "gene: Chr7-6Mb-seg-194 -> 58\n",
      "gene: Chr7-6Mb-seg-195 -> 60\n",
      "gene: Chr7-6Mb-seg-196 -> 60\n",
      "gene: Chr7-6Mb-seg-197 -> 60\n",
      "gene: Chr7-6Mb-seg-198 -> 60\n",
      "gene: Chr7-6Mb-seg-199 -> 60\n",
      "gene: Chr7-6Mb-seg-200 -> 60\n",
      "gene: Chr7-6Mb-seg-201 -> 60\n",
      "gene: Chr7-6Mb-seg-202 -> 60\n",
      "gene: Chr7-6Mb-seg-203 -> 60\n",
      "gene: Chr7-6Mb-seg-204 -> 60\n",
      "gene: Chr7-6Mb-seg-205 -> 57\n",
      "gene: Chr7-6Mb-seg-206 -> 60\n",
      "gene: Chr7-6Mb-seg-207 -> 60\n",
      "gene: Chr7-6Mb-seg-208 -> 60\n",
      "gene: Chr7-6Mb-seg-209 -> 60\n",
      "gene: Chr7-6Mb-seg-210 -> 60\n",
      "gene: Chr7-6Mb-seg-211 -> 60\n",
      "gene: Chr7-6Mb-seg-212 -> 60\n",
      "gene: Chr7-6Mb-seg-213 -> 60\n",
      "gene: Chr7-6Mb-seg-214 -> 56\n",
      "gene: Chr7-6Mb-seg-215 -> 53\n",
      "gene: Chr7-6Mb-seg-216 -> 60\n",
      "gene: Chr7-6Mb-seg-217 -> 57\n",
      "gene: Chr7-6Mb-seg-218 -> 57\n",
      "gene: Chr7-6Mb-seg-219 -> 60\n",
      "gene: Chr7-6Mb-seg-220 -> 53\n",
      "gene: Chr7-6Mb-seg-221 -> 60\n",
      "gene: Chr7-6Mb-seg-222 -> 49\n",
      "gene: Chr7-6Mb-seg-223 -> 50\n",
      "gene: Chr7-6Mb-seg-224 -> 54\n",
      "gene: Chr7-6Mb-seg-225 -> 60\n",
      "gene: Chr7-6Mb-seg-226 -> 60\n",
      "gene: Chr7-6Mb-seg-227 -> 60\n",
      "gene: Chr7-6Mb-seg-228 -> 60\n",
      "gene: Chr7-6Mb-seg-229 -> 60\n",
      "gene: Chr7-6Mb-seg-230 -> 60\n",
      "gene: Chr7-6Mb-seg-231 -> 60\n",
      "gene: Chr7-6Mb-seg-232 -> 60\n",
      "gene: Chr7-6Mb-seg-233 -> 60\n",
      "gene: Chr7-6Mb-seg-234 -> 60\n",
      "gene: Chr7-6Mb-seg-235 -> 60\n",
      "gene: Chr7-6Mb-seg-236 -> 60\n",
      "gene: Chr7-6Mb-seg-237 -> 60\n",
      "gene: Chr7-6Mb-seg-238 -> 60\n",
      "gene: Chr7-6Mb-seg-239 -> 60\n",
      "gene: Chr7-6Mb-seg-240 -> 60\n",
      "gene: Chr7-6Mb-seg-241 -> 60\n",
      "gene: Chr7-6Mb-seg-242 -> 52\n",
      "gene: Chr7-6Mb-seg-243 -> 60\n",
      "gene: Chr7-6Mb-seg-244 -> 60\n",
      "gene: Chr7-6Mb-seg-245 -> 60\n",
      "gene: Chr7-6Mb-seg-246 -> 60\n",
      "gene: Chr7-6Mb-seg-247 -> 60\n",
      "gene: Chr7-6Mb-seg-248 -> 60\n",
      "gene: Chr7-6Mb-seg-249 -> 49\n",
      "gene: Chr7-6Mb-seg-250 -> 12\n",
      "gene: Chr7-6Mb-seg-251 -> 39\n",
      "gene: Chr7-6Mb-seg-252 -> 60\n",
      "gene: Chr7-6Mb-seg-253 -> 60\n",
      "gene: Chr7-6Mb-seg-254 -> 60\n",
      "gene: Chr7-6Mb-seg-255 -> 60\n",
      "gene: Chr7-6Mb-seg-256 -> 60\n",
      "gene: Chr7-6Mb-seg-257 -> 60\n",
      "gene: Chr7-6Mb-seg-258 -> 60\n",
      "gene: Chr7-6Mb-seg-259 -> 60\n",
      "gene: Chr7-6Mb-seg-260 -> 56\n",
      "gene: Chr7-6Mb-seg-261 -> 60\n",
      "gene: Chr7-6Mb-seg-262 -> 60\n",
      "gene: Chr7-6Mb-seg-263 -> 60\n",
      "gene: Chr7-6Mb-seg-264 -> 60\n",
      "gene: Chr7-6Mb-seg-265 -> 60\n",
      "gene: Chr7-6Mb-seg-266 -> 50\n",
      "gene: Chr7-6Mb-seg-267 -> 60\n",
      "gene: Chr7-6Mb-seg-268 -> 60\n",
      "gene: Chr7-6Mb-seg-269 -> 60\n",
      "gene: Chr7-6Mb-seg-270 -> 60\n",
      "gene: Chr7-6Mb-seg-271 -> 60\n",
      "gene: Chr7-6Mb-seg-272 -> 60\n",
      "gene: Chr7-6Mb-seg-273 -> 60\n",
      "gene: Chr7-6Mb-seg-274 -> 60\n",
      "gene: Chr7-6Mb-seg-275 -> 60\n",
      "gene: Chr7-6Mb-seg-276 -> 60\n",
      "gene: Chr7-6Mb-seg-277 -> 60\n",
      "gene: Chr7-6Mb-seg-278 -> 60\n",
      "gene: Chr7-6Mb-seg-279 -> 60\n",
      "gene: Chr7-6Mb-seg-280 -> 42\n",
      "gene: Chr7-6Mb-seg-281 -> 47\n",
      "gene: Chr7-6Mb-seg-282 -> 60\n",
      "gene: Chr7-6Mb-seg-283 -> 60\n",
      "gene: Chr7-6Mb-seg-284 -> 60\n",
      "gene: Chr7-6Mb-seg-285 -> 60\n",
      "gene: Chr7-6Mb-seg-286 -> 60\n",
      "gene: Chr7-6Mb-seg-287 -> 60\n",
      "gene: Chr7-6Mb-seg-288 -> 60\n",
      "gene: Chr7-6Mb-seg-289 -> 60\n",
      "gene: Chr7-6Mb-seg-290 -> 59\n",
      "gene: Chr7-6Mb-seg-291 -> 44\n",
      "gene: Chr7-6Mb-seg-292 -> 60\n",
      "gene: Chr7-6Mb-seg-293 -> 60\n",
      "gene: Chr7-6Mb-seg-294 -> 60\n",
      "gene: Chr7-6Mb-seg-295 -> 43\n",
      "gene: Chr7-6Mb-seg-296 -> 59\n",
      "gene: Chr7-6Mb-seg-297 -> 32\n",
      "gene: Chr7-6Mb-seg-298 -> 59\n",
      "gene: Chr7-6Mb-seg-299 -> 60\n",
      "gene: Chr7-6Mb-seg-300 -> 60\n",
      "gene: Chr7-6Mb-seg-301 -> 60\n",
      "gene: Chr7-6Mb-seg-302 -> 60\n",
      "gene: Chr7-6Mb-seg-303 -> 46\n",
      "gene: Chr7-6Mb-seg-304 -> 60\n",
      "gene: Chr7-6Mb-seg-305 -> 60\n",
      "gene: Chr7-6Mb-seg-306 -> 60\n",
      "gene: Chr7-6Mb-seg-307 -> 60\n",
      "gene: Chr7-6Mb-seg-308 -> 58\n",
      "gene: Chr7-6Mb-seg-309 -> 54\n",
      "gene: Chr7-6Mb-seg-310 -> 54\n",
      "gene: Chr7-6Mb-seg-311 -> 60\n",
      "gene: Chr7-6Mb-seg-312 -> 56\n",
      "gene: Chr7-6Mb-seg-313 -> 58\n",
      "gene: Chr7-6Mb-seg-314 -> 59\n",
      "gene: Chr7-6Mb-seg-315 -> 60\n",
      "gene: Chr7-6Mb-seg-316 -> 41\n",
      "gene: Chr7-6Mb-seg-317 -> 51\n",
      "gene: Chr7-6Mb-seg-318 -> 60\n",
      "gene: Chr7-6Mb-seg-319 -> 43\n",
      "gene: Chr7-6Mb-seg-320 -> 60\n",
      "gene: Chr7-6Mb-seg-321 -> 31\n",
      "gene: Chr7-6Mb-seg-322 -> 59\n",
      "gene: Chr7-6Mb-seg-323 -> 52\n",
      "gene: Chr7-6Mb-seg-324 -> 60\n",
      "gene: Chr7-6Mb-seg-325 -> 60\n",
      "gene: Chr7-6Mb-seg-326 -> 60\n",
      "gene: Chr7-6Mb-seg-327 -> 60\n",
      "gene: Chr7-6Mb-seg-328 -> 60\n",
      "gene: Chr7-6Mb-seg-329 -> 60\n",
      "gene: Chr7-6Mb-seg-330 -> 60\n",
      "gene: Chr7-6Mb-seg-331 -> 60\n",
      "gene: Chr7-6Mb-seg-332 -> 55\n",
      "gene: Chr7-6Mb-seg-333 -> 60\n",
      "gene: Chr7-6Mb-seg-334 -> 60\n",
      "gene: Chr7-6Mb-seg-335 -> 46\n",
      "gene: Chr7-6Mb-seg-336 -> 60\n",
      "gene: Chr7-6Mb-seg-337 -> 60\n",
      "gene: Chr7-6Mb-seg-338 -> 60\n",
      "gene: Chr7-6Mb-seg-339 -> 60\n",
      "gene: Chr7-6Mb-seg-340 -> 54\n",
      "gene: Chr7-6Mb-seg-341 -> 60\n",
      "gene: Chr7-6Mb-seg-342 -> 60\n",
      "gene: Chr7-6Mb-seg-343 -> 60\n",
      "gene: Chr7-6Mb-seg-344 -> 60\n",
      "gene: Chr7-6Mb-seg-345 -> 60\n",
      "gene: Chr7-6Mb-seg-346 -> 60\n",
      "gene: Chr7-6Mb-seg-347 -> 60\n",
      "gene: Chr7-6Mb-seg-348 -> 41\n",
      "gene: Chr7-6Mb-seg-349 -> 60\n",
      "gene: Chr7-6Mb-seg-350 -> 60\n",
      "gene: Chr7-6Mb-seg-351 -> 53\n",
      "gene: Chr7-6Mb-seg-352 -> 60\n",
      "gene: Chr7-6Mb-seg-353 -> 46\n",
      "gene: Chr7-6Mb-seg-354 -> 52\n",
      "gene: Chr7-6Mb-seg-355 -> 60\n",
      "gene: Chr7-6Mb-seg-356 -> 60\n",
      "gene: Chr7-6Mb-seg-357 -> 60\n",
      "gene: Chr7-6Mb-seg-358 -> 60\n",
      "gene: Chr7-6Mb-seg-359 -> 60\n",
      "gene: Chr7-6Mb-seg-360 -> 60\n",
      "gene: Chr7-6Mb-seg-361 -> 60\n",
      "gene: Chr7-6Mb-seg-362 -> 60\n",
      "gene: Chr7-6Mb-seg-363 -> 60\n",
      "gene: Chr7-6Mb-seg-364 -> 60\n",
      "gene: Chr7-6Mb-seg-365 -> 60\n",
      "gene: Chr7-6Mb-seg-366 -> 60\n",
      "gene: Chr7-6Mb-seg-367 -> 60\n",
      "gene: Chr7-6Mb-seg-368 -> 54\n",
      "gene: Chr7-6Mb-seg-369 -> 60\n",
      "gene: Chr7-6Mb-seg-370 -> 53\n",
      "gene: Chr7-6Mb-seg-371 -> 60\n",
      "gene: Chr7-6Mb-seg-372 -> 60\n",
      "gene: Chr7-6Mb-seg-373 -> 60\n",
      "gene: Chr7-6Mb-seg-374 -> 60\n",
      "gene: Chr7-6Mb-seg-375 -> 60\n",
      "gene: Chr7-6Mb-seg-376 -> 42\n",
      "gene: Chr7-6Mb-seg-377 -> 60\n",
      "gene: Chr7-6Mb-seg-378 -> 60\n",
      "gene: Chr7-6Mb-seg-379 -> 60\n",
      "gene: Chr7-6Mb-seg-380 -> 60\n",
      "gene: Chr7-6Mb-seg-381 -> 54\n",
      "gene: Chr7-6Mb-seg-382 -> 58\n",
      "gene: Chr7-6Mb-seg-383 -> 37\n",
      "gene: Chr7-6Mb-seg-384 -> 60\n",
      "gene: Chr7-6Mb-seg-385 -> 60\n",
      "gene: Chr7-6Mb-seg-386 -> 33\n",
      "gene: Chr7-6Mb-seg-387 -> 34\n",
      "gene: Chr7-6Mb-seg-388 -> 60\n",
      "gene: Chr7-6Mb-seg-389 -> 60\n",
      "gene: Chr7-6Mb-seg-390 -> 60\n",
      "gene: Chr7-6Mb-seg-391 -> 34\n",
      "gene: Chr7-6Mb-seg-392 -> 60\n",
      "gene: Chr7-6Mb-seg-393 -> 40\n",
      "gene: Chr7-6Mb-seg-394 -> 60\n",
      "gene: Chr7-6Mb-seg-395 -> 60\n",
      "gene: Chr7-6Mb-seg-396 -> 60\n",
      "gene: Chr7-6Mb-seg-397 -> 60\n",
      "gene: Chr7-6Mb-seg-398 -> 60\n",
      "gene: Chr7-6Mb-seg-399 -> 60\n",
      "gene: Chr7-6Mb-seg-400 -> 60\n",
      "gene: Chr7-6Mb-seg-401 -> 60\n",
      "gene: Chr7-6Mb-seg-402 -> 60\n",
      "gene: Chr7-6Mb-seg-403 -> 60\n",
      "gene: Chr7-6Mb-seg-404 -> 60\n",
      "gene: Chr7-6Mb-seg-405 -> 59\n",
      "gene: Chr7-6Mb-seg-406 -> 59\n",
      "gene: Chr7-6Mb-seg-407 -> 60\n",
      "gene: Chr7-6Mb-seg-408 -> 60\n",
      "gene: Chr7-6Mb-seg-409 -> 60\n",
      "gene: Chr7-6Mb-seg-410 -> 60\n",
      "gene: Chr7-6Mb-seg-411 -> 60\n",
      "gene: Chr7-6Mb-seg-412 -> 60\n",
      "gene: Chr7-6Mb-seg-413 -> 60\n",
      "gene: Chr7-6Mb-seg-414 -> 60\n",
      "gene: Chr7-6Mb-seg-415 -> 60\n",
      "gene: Chr7-6Mb-seg-416 -> 60\n",
      "gene: Chr7-6Mb-seg-417 -> 60\n",
      "gene: Chr7-6Mb-seg-418 -> 58\n",
      "gene: Chr7-6Mb-seg-419 -> 60\n",
      "gene: Chr7-6Mb-seg-420 -> 60\n",
      "gene: Chr7-6Mb-seg-421 -> 60\n",
      "gene: Chr7-6Mb-seg-422 -> 60\n",
      "gene: Chr7-6Mb-seg-423 -> 60\n",
      "gene: Chr7-6Mb-seg-424 -> 60\n",
      "gene: Chr7-6Mb-seg-425 -> 60\n",
      "gene: Chr7-6Mb-seg-426 -> 60\n",
      "gene: Chr7-6Mb-seg-427 -> 60\n",
      "gene: Chr7-6Mb-seg-428 -> 60\n",
      "gene: Chr7-6Mb-seg-429 -> 60\n",
      "gene: Chr7-6Mb-seg-430 -> 60\n",
      "gene: Chr7-6Mb-seg-431 -> 60\n",
      "gene: Chr7-6Mb-seg-432 -> 60\n",
      "gene: Chr7-6Mb-seg-433 -> 60\n",
      "gene: Chr7-6Mb-seg-434 -> 60\n",
      "gene: Chr7-6Mb-seg-435 -> 60\n",
      "gene: Chr7-6Mb-seg-436 -> 60\n",
      "gene: Chr7-6Mb-seg-437 -> 60\n",
      "gene: Chr7-6Mb-seg-438 -> 60\n",
      "gene: Chr7-6Mb-seg-439 -> 60\n",
      "gene: Chr7-6Mb-seg-440 -> 60\n",
      "gene: Chr7-6Mb-seg-441 -> 60\n",
      "gene: Chr7-6Mb-seg-442 -> 60\n",
      "gene: Chr7-6Mb-seg-443 -> 60\n",
      "gene: Chr7-6Mb-seg-444 -> 60\n",
      "gene: Chr7-6Mb-seg-445 -> 60\n",
      "gene: Chr7-6Mb-seg-446 -> 60\n",
      "gene: Chr7-6Mb-seg-447 -> 60\n",
      "gene: Chr7-6Mb-seg-448 -> 60\n",
      "gene: Chr7-6Mb-seg-449 -> 60\n",
      "gene: Chr7-6Mb-seg-450 -> 60\n",
      "gene: Chr7-6Mb-seg-451 -> 60\n",
      "gene: Chr7-6Mb-seg-452 -> 60\n",
      "gene: Chr7-6Mb-seg-453 -> 60\n",
      "gene: Chr7-6Mb-seg-454 -> 60\n",
      "gene: Chr7-6Mb-seg-455 -> 60\n",
      "gene: Chr7-6Mb-seg-456 -> 60\n",
      "gene: Chr7-6Mb-seg-457 -> 60\n",
      "gene: Chr7-6Mb-seg-458 -> 60\n",
      "gene: Chr7-6Mb-seg-459 -> 60\n",
      "gene: Chr7-6Mb-seg-460 -> 60\n",
      "gene: Chr7-6Mb-seg-461 -> 60\n",
      "gene: Chr7-6Mb-seg-462 -> 60\n",
      "gene: Chr7-6Mb-seg-463 -> 60\n",
      "gene: Chr7-6Mb-seg-464 -> 60\n",
      "gene: Chr7-6Mb-seg-465 -> 60\n",
      "gene: Chr7-6Mb-seg-466 -> 60\n",
      "gene: Chr7-6Mb-seg-467 -> 47\n",
      "gene: Chr7-6Mb-seg-468 -> 60\n",
      "gene: Chr7-6Mb-seg-469 -> 60\n",
      "gene: Chr7-6Mb-seg-470 -> 60\n",
      "gene: Chr7-6Mb-seg-471 -> 60\n",
      "gene: Chr7-6Mb-seg-472 -> 60\n",
      "gene: Chr7-6Mb-seg-473 -> 60\n",
      "gene: Chr7-6Mb-seg-474 -> 60\n",
      "gene: Chr7-6Mb-seg-475 -> 60\n",
      "gene: Chr7-6Mb-seg-476 -> 60\n",
      "gene: Chr7-6Mb-seg-477 -> 60\n",
      "gene: Chr7-6Mb-seg-478 -> 46\n",
      "gene: Chr7-6Mb-seg-479 -> 60\n",
      "gene: Chr7-6Mb-seg-480 -> 60\n",
      "gene: Chr7-6Mb-seg-481 -> 60\n",
      "gene: Chr7-6Mb-seg-482 -> 60\n",
      "gene: Chr7-6Mb-seg-483 -> 60\n",
      "gene: Chr7-6Mb-seg-484 -> 60\n",
      "gene: Chr7-6Mb-seg-485 -> 60\n",
      "gene: Chr7-6Mb-seg-486 -> 60\n",
      "gene: Chr7-6Mb-seg-487 -> 60\n",
      "gene: Chr7-6Mb-seg-488 -> 60\n",
      "gene: Chr7-6Mb-seg-489 -> 60\n",
      "gene: Chr7-6Mb-seg-490 -> 60\n",
      "gene: Chr7-6Mb-seg-491 -> 59\n",
      "gene: Chr7-6Mb-seg-492 -> 60\n",
      "gene: Chr7-6Mb-seg-493 -> 60\n",
      "gene: Chr7-6Mb-seg-494 -> 60\n",
      "gene: Chr7-6Mb-seg-495 -> 60\n",
      "gene: Chr7-6Mb-seg-496 -> 60\n",
      "gene: Chr7-6Mb-seg-497 -> 60\n",
      "gene: Chr7-6Mb-seg-498 -> 59\n",
      "gene: Chr7-6Mb-seg-499 -> 60\n",
      "gene: Chr7-6Mb-seg-500 -> 60\n",
      "gene: Chr7-6Mb-seg-501 -> 60\n",
      "gene: Chr7-6Mb-seg-502 -> 60\n",
      "gene: Chr7-6Mb-seg-503 -> 60\n",
      "gene: Chr7-6Mb-seg-504 -> 60\n",
      "gene: Chr7-6Mb-seg-505 -> 60\n",
      "gene: Chr7-6Mb-seg-506 -> 60\n",
      "gene: Chr7-6Mb-seg-507 -> 60\n",
      "gene: Chr7-6Mb-seg-508 -> 60\n",
      "gene: Chr7-6Mb-seg-509 -> 56\n",
      "gene: Chr7-6Mb-seg-510 -> 60\n",
      "gene: Chr7-6Mb-seg-511 -> 60\n",
      "gene: Chr7-6Mb-seg-512 -> 60\n",
      "gene: Chr7-6Mb-seg-513 -> 60\n",
      "gene: Chr7-6Mb-seg-514 -> 60\n",
      "gene: Chr7-6Mb-seg-515 -> 60\n",
      "gene: Chr7-6Mb-seg-516 -> 60\n",
      "gene: Chr7-6Mb-seg-517 -> 60\n",
      "gene: Chr7-6Mb-seg-518 -> 60\n",
      "gene: Chr7-6Mb-seg-519 -> 60\n",
      "gene: Chr7-6Mb-seg-520 -> 60\n",
      "gene: Chr7-6Mb-seg-521 -> 60\n",
      "gene: Chr7-6Mb-seg-522 -> 60\n",
      "gene: Chr7-6Mb-seg-523 -> 60\n",
      "gene: Chr7-6Mb-seg-524 -> 60\n",
      "gene: Chr7-6Mb-seg-525 -> 60\n",
      "gene: Chr7-6Mb-seg-526 -> 59\n",
      "gene: Chr7-6Mb-seg-527 -> 60\n",
      "gene: Chr7-6Mb-seg-528 -> 60\n",
      "gene: Chr7-6Mb-seg-529 -> 60\n",
      "gene: Chr7-6Mb-seg-530 -> 60\n",
      "gene: Chr7-6Mb-seg-531 -> 60\n",
      "gene: Chr7-6Mb-seg-532 -> 60\n",
      "gene: Chr7-6Mb-seg-533 -> 60\n",
      "gene: Chr7-6Mb-seg-534 -> 60\n",
      "gene: Chr7-6Mb-seg-535 -> 60\n",
      "gene: Chr7-6Mb-seg-536 -> 45\n",
      "gene: Chr7-6Mb-seg-537 -> 60\n",
      "gene: Chr7-6Mb-seg-538 -> 60\n",
      "gene: Chr7-6Mb-seg-539 -> 29\n",
      "gene: Chr7-6Mb-seg-540 -> 44\n",
      "gene: Chr7-6Mb-seg-541 -> 60\n",
      "gene: Chr7-6Mb-seg-542 -> 60\n",
      "gene: Chr7-6Mb-seg-543 -> 60\n",
      "gene: Chr7-6Mb-seg-544 -> 60\n",
      "gene: Chr7-6Mb-seg-545 -> 60\n",
      "gene: Chr7-6Mb-seg-546 -> 60\n",
      "gene: Chr7-6Mb-seg-547 -> 60\n",
      "gene: Chr7-6Mb-seg-548 -> 60\n",
      "gene: Chr7-6Mb-seg-549 -> 56\n",
      "gene: Chr7-6Mb-seg-550 -> 60\n",
      "gene: Chr7-6Mb-seg-551 -> 60\n",
      "gene: Chr7-6Mb-seg-552 -> 60\n",
      "gene: Chr7-6Mb-seg-553 -> 60\n",
      "gene: Chr7-6Mb-seg-554 -> 57\n",
      "gene: Chr7-6Mb-seg-555 -> 60\n",
      "gene: Chr7-6Mb-seg-556 -> 60\n",
      "gene: Chr7-6Mb-seg-557 -> 59\n",
      "gene: Chr7-6Mb-seg-558 -> 60\n",
      "gene: Chr7-6Mb-seg-559 -> 58\n",
      "gene: Chr7-6Mb-seg-560 -> 60\n",
      "gene: Chr7-6Mb-seg-561 -> 55\n",
      "gene: Chr7-6Mb-seg-562 -> 60\n",
      "gene: Chr7-6Mb-seg-563 -> 49\n",
      "gene: Chr7-6Mb-seg-564 -> 56\n",
      "gene: Chr7-6Mb-seg-565 -> 60\n",
      "gene: Chr7-6Mb-seg-566 -> 59\n",
      "gene: Chr7-6Mb-seg-567 -> 60\n",
      "gene: Chr7-6Mb-seg-568 -> 60\n",
      "gene: Chr7-6Mb-seg-569 -> 60\n",
      "gene: Chr7-6Mb-seg-570 -> 60\n",
      "gene: Chr7-6Mb-seg-571 -> 60\n",
      "gene: Chr7-6Mb-seg-572 -> 55\n",
      "gene: Chr7-6Mb-seg-573 -> 53\n",
      "gene: Chr7-6Mb-seg-574 -> 45\n",
      "gene: Chr7-6Mb-seg-575 -> 56\n",
      "gene: Chr7-6Mb-seg-576 -> 60\n",
      "gene: Chr7-6Mb-seg-577 -> 49\n",
      "gene: Chr7-6Mb-seg-578 -> 45\n",
      "gene: Chr7-6Mb-seg-579 -> 60\n",
      "gene: Chr7-6Mb-seg-580 -> 59\n",
      "gene: Chr7-6Mb-seg-581 -> 54\n",
      "gene: Chr7-6Mb-seg-582 -> 60\n",
      "gene: Chr7-6Mb-seg-583 -> 60\n",
      "gene: Chr7-6Mb-seg-584 -> 60\n",
      "gene: Chr7-6Mb-seg-585 -> 37\n",
      "gene: Chr7-6Mb-seg-586 -> 47\n",
      "gene: Chr7-6Mb-seg-587 -> 51\n",
      "gene: Chr7-6Mb-seg-588 -> 60\n",
      "gene: Chr7-6Mb-seg-589 -> 59\n",
      "gene: Chr7-6Mb-seg-590 -> 44\n",
      "gene: Chr7-6Mb-seg-591 -> 60\n",
      "gene: Chr7-6Mb-seg-592 -> 60\n",
      "gene: Chr7-6Mb-seg-593 -> 57\n",
      "gene: Chr7-6Mb-seg-594 -> 55\n",
      "gene: Chr7-6Mb-seg-595 -> 29\n",
      "gene: Chr7-6Mb-seg-596 -> 43\n",
      "gene: Chr7-6Mb-seg-597 -> 49\n",
      "gene: Chr7-6Mb-seg-598 -> 60\n",
      "gene: Chr7-6Mb-seg-599 -> 60\n",
      "gene: Chr7-6Mb-seg-600 -> 60\n",
      "gene: Chr7-6Mb-seg-601 -> 60\n",
      "gene: Chr7-6Mb-seg-602 -> 60\n",
      "gene: Chr7-6Mb-seg-603 -> 47\n",
      "gene: Chr7-6Mb-seg-604 -> 54\n",
      "gene: Chr7-6Mb-seg-605 -> 60\n",
      "gene: Chr7-6Mb-seg-606 -> 55\n",
      "gene: Chr7-6Mb-seg-607 -> 52\n",
      "gene: Chr7-6Mb-seg-608 -> 38\n",
      "gene: Chr7-6Mb-seg-609 -> 60\n",
      "gene: Chr7-6Mb-seg-610 -> 60\n",
      "gene: Chr7-6Mb-seg-611 -> 60\n",
      "gene: Chr7-6Mb-seg-612 -> 60\n",
      "gene: Chr7-6Mb-seg-613 -> 60\n",
      "gene: Chr7-6Mb-seg-614 -> 60\n",
      "gene: Chr7-6Mb-seg-615 -> 60\n",
      "gene: Chr7-6Mb-seg-616 -> 60\n",
      "gene: Chr7-6Mb-seg-617 -> 60\n",
      "gene: Chr7-6Mb-seg-618 -> 60\n",
      "gene: Chr7-6Mb-seg-619 -> 60\n",
      "gene: Chr7-6Mb-seg-620 -> 60\n",
      "gene: Chr7-6Mb-seg-621 -> 60\n",
      "gene: Chr7-6Mb-seg-622 -> 60\n",
      "gene: Chr7-6Mb-seg-623 -> 60\n",
      "gene: Chr7-6Mb-seg-624 -> 60\n",
      "gene: Chr7-6Mb-seg-625 -> 49\n",
      "gene: Chr7-6Mb-seg-626 -> 60\n",
      "gene: Chr7-6Mb-seg-627 -> 60\n",
      "gene: Chr7-6Mb-seg-628 -> 51\n",
      "gene: Chr7-6Mb-seg-629 -> 60\n",
      "gene: Chr7-6Mb-seg-630 -> 60\n",
      "gene: Chr7-6Mb-seg-631 -> 60\n",
      "gene: Chr7-6Mb-seg-632 -> 53\n",
      "gene: Chr7-6Mb-seg-633 -> 60\n",
      "gene: Chr7-6Mb-seg-634 -> 60\n",
      "gene: Chr7-6Mb-seg-635 -> 57\n",
      "gene: Chr7-6Mb-seg-638 -> 24\n",
      "gene: Chr7-6Mb-seg-639 -> 49\n",
      "gene: Chr7-6Mb-seg-640 -> 24\n",
      "gene: Chr7-6Mb-seg-641 -> 36\n",
      "gene: Chr7-6Mb-seg-642 -> 47\n",
      "gene: Chr7-6Mb-seg-643 -> 58\n",
      "gene: Chr7-6Mb-seg-644 -> 49\n",
      "gene: Chr7-6Mb-seg-647 -> 17\n",
      "gene: Chr7-6Mb-seg-648 -> 45\n",
      "gene: Chr7-6Mb-seg-649 -> 21\n",
      "gene: Chr7-6Mb-seg-650 -> 37\n",
      "gene: Chr7-6Mb-seg-651 -> 57\n",
      "gene: Chr7-6Mb-seg-655 -> 32\n",
      "gene: Chr7-6Mb-seg-660 -> 31\n",
      "gene: Chr7-6Mb-seg-661 -> 41\n",
      "gene: Chr7-6Mb-seg-662 -> 42\n",
      "gene: Chr7-6Mb-seg-663 -> 46\n",
      "gene: Chr7-6Mb-seg-664 -> 40\n",
      "gene: Chr7-6Mb-seg-665 -> 22\n",
      "gene: Chr7-6Mb-seg-668 -> 52\n",
      "gene: Chr7-6Mb-seg-669 -> 51\n",
      "gene: Chr7-6Mb-seg-670 -> 60\n",
      "gene: Chr7-6Mb-seg-671 -> 52\n",
      "gene: Chr7-6Mb-seg-672 -> 48\n",
      "gene: Chr7-6Mb-seg-674 -> 15\n",
      "gene: Chr7-6Mb-seg-675 -> 44\n",
      "gene: Chr7-6Mb-seg-676 -> 30\n",
      "gene: Chr7-6Mb-seg-677 -> 34\n",
      "gene: Chr7-6Mb-seg-679 -> 49\n",
      "gene: Chr7-6Mb-seg-680 -> 29\n",
      "gene: Chr7-6Mb-seg-681 -> 31\n",
      "gene: Chr7-6Mb-seg-682 -> 45\n",
      "gene: Chr7-6Mb-seg-683 -> 23\n",
      "gene: Chr7-6Mb-seg-692 -> 30\n",
      "gene: Chr7-6Mb-seg-693 -> 34\n",
      "gene: Chr7-6Mb-seg-698 -> 31\n",
      "gene: Chr7-6Mb-seg-699 -> 25\n",
      "gene: Chr7-6Mb-seg-700 -> 33\n",
      "gene: Chr7-6Mb-seg-701 -> 26\n",
      "gene: Chr7-6Mb-seg-703 -> 23\n",
      "gene: Chr7-6Mb-seg-704 -> 39\n",
      "gene: Chr7-6Mb-seg-705 -> 27\n",
      "gene: Chr7-6Mb-seg-706 -> 44\n",
      "gene: Chr7-6Mb-seg-707 -> 53\n",
      "gene: Chr7-6Mb-seg-708 -> 29\n",
      "gene: Chr7-6Mb-seg-710 -> 28\n",
      "gene: Chr7-6Mb-seg-711 -> 13\n",
      "gene: Chr7-6Mb-seg-712 -> 25\n",
      "gene: Chr7-6Mb-seg-713 -> 46\n",
      "gene: Chr7-6Mb-seg-717 -> 23\n",
      "gene: Chr7-6Mb-seg-718 -> 19\n",
      "gene: Chr7-6Mb-seg-719 -> 24\n",
      "gene: Chr7-6Mb-seg-720 -> 38\n",
      "gene: Chr7-6Mb-seg-721 -> 55\n",
      "gene: Chr7-6Mb-seg-722 -> 44\n",
      "gene: Chr7-6Mb-seg-723 -> 25\n",
      "gene: Chr7-6Mb-seg-726 -> 38\n",
      "gene: Chr7-6Mb-seg-727 -> 39\n",
      "gene: Chr7-6Mb-seg-728 -> 20\n",
      "gene: Chr7-6Mb-seg-731 -> 36\n",
      "gene: Chr7-6Mb-seg-732 -> 13\n",
      "gene: Chr7-6Mb-seg-733 -> 54\n",
      "gene: Chr7-6Mb-seg-734 -> 17\n",
      "gene: Chr7-6Mb-seg-735 -> 42\n",
      "gene: Chr7-6Mb-seg-738 -> 34\n",
      "gene: Chr7-6Mb-seg-739 -> 40\n",
      "gene: Chr7-6Mb-seg-740 -> 56\n",
      "gene: Chr7-6Mb-seg-741 -> 46\n",
      "gene: Chr7-6Mb-seg-742 -> 34\n",
      "gene: Chr7-6Mb-seg-743 -> 55\n",
      "gene: Chr7-6Mb-seg-744 -> 41\n",
      "gene: Chr7-6Mb-seg-745 -> 31\n",
      "gene: Chr7-6Mb-seg-746 -> 33\n",
      "gene: Chr7-6Mb-seg-747 -> 19\n",
      "gene: Chr7-6Mb-seg-748 -> 51\n",
      "gene: Chr7-6Mb-seg-749 -> 52\n",
      "gene: Chr7-6Mb-seg-750 -> 48\n",
      "gene: Chr7-6Mb-seg-751 -> 32\n",
      "gene: Chr7-6Mb-seg-752 -> 59\n",
      "gene: Chr7-6Mb-seg-753 -> 45\n",
      "gene: Chr7-6Mb-seg-754 -> 6\n",
      "gene: Chr7-6Mb-seg-755 -> 42\n",
      "gene: Chr7-6Mb-seg-756 -> 43\n",
      "gene: Chr7-6Mb-seg-757 -> 49\n",
      "gene: Chr7-6Mb-seg-758 -> 45\n",
      "gene: Chr7-6Mb-seg-760 -> 22\n",
      "gene: Chr7-6Mb-seg-762 -> 20\n",
      "gene: Chr7-6Mb-seg-763 -> 37\n",
      "gene: Chr7-6Mb-seg-764 -> 40\n",
      "gene: Chr7-6Mb-seg-765 -> 18\n",
      "gene: Chr7-6Mb-seg-766 -> 45\n",
      "gene: Chr7-6Mb-seg-767 -> 48\n",
      "gene: Chr7-6Mb-seg-768 -> 39\n",
      "gene: Chr7-6Mb-seg-769 -> 20\n",
      "gene: Chr7-6Mb-seg-770 -> 15\n",
      "gene: Chr7-6Mb-seg-771 -> 21\n",
      "gene: Chr7-6Mb-seg-772 -> 37\n",
      "gene: Chr7-6Mb-seg-773 -> 27\n",
      "gene: Chr7-6Mb-seg-775 -> 26\n",
      "gene: Chr7-6Mb-seg-776 -> 41\n",
      "gene: Chr7-6Mb-seg-777 -> 30\n",
      "gene: Chr7-6Mb-seg-778 -> 39\n",
      "gene: Chr7-6Mb-seg-780 -> 26\n",
      "gene: Chr7-6Mb-seg-781 -> 34\n",
      "gene: Chr7-6Mb-seg-784 -> 46\n",
      "gene: Chr7-6Mb-seg-785 -> 52\n",
      "gene: Chr7-6Mb-seg-786 -> 37\n",
      "gene: Chr7-6Mb-seg-787 -> 48\n",
      "gene: Chr7-6Mb-seg-788 -> 48\n",
      "gene: Chr7-6Mb-seg-790 -> 23\n",
      "gene: Chr7-6Mb-seg-791 -> 23\n",
      "gene: Chr7-6Mb-seg-792 -> 34\n",
      "gene: Chr7-6Mb-seg-793 -> 21\n",
      "gene: Chr7-6Mb-seg-794 -> 35\n",
      "gene: Chr7-6Mb-seg-795 -> 21\n",
      "gene: Chr7-6Mb-seg-796 -> 13\n",
      "gene: Chr7-6Mb-seg-797 -> 41\n",
      "gene: Chr7-6Mb-seg-798 -> 47\n",
      "gene: Chr7-6Mb-seg-799 -> 43\n",
      "gene: Chr7-6Mb-seg-800 -> 27\n",
      "gene: Chr7-6Mb-seg-801 -> 22\n",
      "gene: Chr7-6Mb-seg-802 -> 43\n",
      "gene: Chr7-6Mb-seg-804 -> 38\n",
      "gene: Chr7-6Mb-seg-805 -> 44\n",
      "gene: Chr7-6Mb-seg-806 -> 51\n",
      "gene: Chr7-6Mb-seg-807 -> 47\n",
      "gene: Chr7-6Mb-seg-808 -> 60\n",
      "gene: Chr7-6Mb-seg-809 -> 60\n",
      "gene: Chr7-6Mb-seg-810 -> 24\n",
      "gene: Chr7-6Mb-seg-812 -> 12\n",
      "gene: Chr7-6Mb-seg-813 -> 51\n",
      "gene: Chr7-6Mb-seg-814 -> 17\n",
      "gene: Chr7-6Mb-seg-816 -> 60\n",
      "gene: Chr7-6Mb-seg-817 -> 60\n",
      "gene: Chr7-6Mb-seg-818 -> 33\n",
      "gene: Chr7-6Mb-seg-819 -> 60\n",
      "gene: Chr7-6Mb-seg-820 -> 60\n",
      "gene: Chr7-6Mb-seg-821 -> 30\n",
      "gene: Chr7-6Mb-seg-822 -> 49\n",
      "gene: Chr7-6Mb-seg-823 -> 34\n",
      "gene: Chr7-6Mb-seg-824 -> 60\n",
      "gene: Chr7-6Mb-seg-825 -> 19\n",
      "gene: Chr7-6Mb-seg-828 -> 22\n",
      "gene: Chr7-6Mb-seg-829 -> 21\n",
      "gene: Chr7-6Mb-seg-831 -> 19\n",
      "gene: Chr7-6Mb-seg-832 -> 22\n",
      "gene: Chr7-6Mb-seg-833 -> 15\n",
      "gene: Chr7-6Mb-seg-834 -> 48\n",
      "gene: Chr7-6Mb-seg-835 -> 60\n",
      "gene: Chr7-6Mb-seg-836 -> 60\n",
      "gene: Chr7-6Mb-seg-837 -> 60\n",
      "gene: Chr7-6Mb-seg-840 -> 60\n",
      "gene: Chr7-6Mb-seg-841 -> 60\n",
      "gene: Chr7-6Mb-seg-842 -> 47\n",
      "gene: Chr7-6Mb-seg-845 -> 49\n",
      "gene: Chr7-6Mb-seg-846 -> 59\n",
      "gene: Chr7-6Mb-seg-847 -> 9\n",
      "gene: Chr7-6Mb-seg-848 -> 26\n",
      "gene: Chr7-6Mb-seg-849 -> 56\n",
      "gene: Chr7-6Mb-seg-850 -> 36\n",
      "gene: Chr7-6Mb-seg-852 -> 22\n",
      "gene: Chr7-6Mb-seg-853 -> 39\n",
      "gene: Chr7-6Mb-seg-854 -> 31\n",
      "gene: Chr7-6Mb-seg-855 -> 24\n",
      "gene: Chr7-6Mb-seg-856 -> 47\n",
      "gene: Chr7-6Mb-seg-857 -> 33\n",
      "gene: Chr7-6Mb-seg-858 -> 29\n",
      "gene: Chr7-6Mb-seg-859 -> 23\n",
      "gene: Chr7-6Mb-seg-860 -> 25\n",
      "gene: Chr7-6Mb-seg-862 -> 40\n",
      "gene: Chr7-6Mb-seg-863 -> 60\n",
      "gene: Chr7-6Mb-seg-864 -> 25\n",
      "gene: Chr7-6Mb-seg-865 -> 18\n",
      "gene: Chr7-6Mb-seg-866 -> 28\n",
      "gene: Chr7-6Mb-seg-868 -> 29\n",
      "gene: Chr7-6Mb-seg-869 -> 25\n",
      "gene: Chr7-6Mb-seg-870 -> 51\n",
      "gene: Chr7-6Mb-seg-873 -> 36\n",
      "gene: Chr7-6Mb-seg-876 -> 43\n",
      "gene: Chr7-6Mb-seg-878 -> 41\n",
      "gene: Chr7-6Mb-seg-879 -> 35\n",
      "gene: Chr7-6Mb-seg-880 -> 28\n",
      "gene: Chr7-6Mb-seg-881 -> 28\n",
      "gene: Chr7-6Mb-seg-883 -> 25\n",
      "gene: Chr7-6Mb-seg-885 -> 30\n",
      "gene: Chr7-6Mb-seg-888 -> 9\n",
      "gene: Chr7-6Mb-seg-889 -> 45\n",
      "gene: Chr7-6Mb-seg-890 -> 54\n",
      "gene: Chr7-6Mb-seg-891 -> 22\n",
      "gene: Chr7-6Mb-seg-892 -> 38\n",
      "gene: Chr7-6Mb-seg-893 -> 49\n",
      "gene: Chr7-6Mb-seg-894 -> 34\n",
      "gene: Chr7-6Mb-seg-895 -> 36\n",
      "gene: Chr7-6Mb-seg-896 -> 58\n",
      "gene: Chr7-6Mb-seg-897 -> 60\n",
      "gene: Chr7-6Mb-seg-900 -> 43\n",
      "gene: Chr7-6Mb-seg-901 -> 60\n",
      "gene: Chr7-6Mb-seg-902 -> 44\n",
      "gene: Chr7-6Mb-seg-904 -> 31\n",
      "gene: Chr7-6Mb-seg-906 -> 8\n",
      "gene: Chr7-6Mb-seg-907 -> 30\n",
      "gene: Chr7-6Mb-seg-911 -> 46\n",
      "gene: Chr7-6Mb-seg-912 -> 59\n",
      "gene: Chr7-6Mb-seg-913 -> 14\n",
      "gene: Chr7-6Mb-seg-916 -> 29\n",
      "gene: Chr7-6Mb-seg-917 -> 44\n",
      "gene: Chr7-6Mb-seg-918 -> 32\n",
      "gene: Chr7-6Mb-seg-919 -> 33\n",
      "gene: Chr7-6Mb-seg-922 -> 46\n",
      "gene: Chr7-6Mb-seg-923 -> 23\n",
      "gene: Chr7-6Mb-seg-924 -> 25\n",
      "gene: Chr7-6Mb-seg-926 -> 24\n",
      "gene: Chr7-6Mb-seg-929 -> 36\n",
      "gene: Chr7-6Mb-seg-931 -> 15\n",
      "gene: Chr7-6Mb-seg-932 -> 44\n",
      "gene: Chr7-6Mb-seg-933 -> 28\n",
      "gene: Chr7-6Mb-seg-934 -> 58\n",
      "gene: Chr7-6Mb-seg-935 -> 19\n",
      "gene: Chr7-6Mb-seg-937 -> 22\n",
      "gene: Chr7-6Mb-seg-938 -> 52\n",
      "gene: Chr7-6Mb-seg-939 -> 30\n",
      "gene: Chr7-6Mb-seg-942 -> 46\n",
      "gene: Chr7-6Mb-seg-944 -> 40\n",
      "gene: Chr7-6Mb-seg-945 -> 60\n",
      "gene: Chr7-6Mb-seg-946 -> 41\n",
      "gene: Chr7-6Mb-seg-947 -> 14\n",
      "gene: Chr7-6Mb-seg-950 -> 24\n",
      "gene: Chr7-6Mb-seg-951 -> 56\n",
      "gene: Chr7-6Mb-seg-952 -> 60\n",
      "gene: Chr7-6Mb-seg-953 -> 60\n",
      "gene: Chr7-6Mb-seg-954 -> 60\n",
      "gene: Chr7-6Mb-seg-955 -> 60\n",
      "gene: Chr7-6Mb-seg-956 -> 60\n",
      "gene: Chr7-6Mb-seg-957 -> 60\n",
      "gene: Chr7-6Mb-seg-958 -> 60\n",
      "gene: Chr7-6Mb-seg-959 -> 60\n",
      "gene: Chr7-6Mb-seg-960 -> 60\n",
      "gene: Chr7-6Mb-seg-961 -> 60\n",
      "gene: Chr7-6Mb-seg-962 -> 60\n",
      "gene: Chr7-6Mb-seg-963 -> 55\n",
      "gene: Chr7-6Mb-seg-964 -> 60\n",
      "gene: Chr7-6Mb-seg-965 -> 60\n",
      "gene: Chr7-6Mb-seg-966 -> 60\n",
      "gene: Chr7-6Mb-seg-967 -> 60\n",
      "gene: Chr7-6Mb-seg-968 -> 60\n",
      "gene: Chr7-6Mb-seg-969 -> 60\n",
      "gene: Chr7-6Mb-seg-970 -> 60\n",
      "gene: Chr7-6Mb-seg-971 -> 60\n",
      "gene: Chr7-6Mb-seg-972 -> 60\n",
      "gene: Chr7-6Mb-seg-973 -> 60\n",
      "gene: Chr7-6Mb-seg-974 -> 60\n",
      "gene: Chr7-6Mb-seg-975 -> 60\n",
      "gene: Chr7-6Mb-seg-976 -> 60\n",
      "gene: Chr7-6Mb-seg-977 -> 60\n",
      "gene: Chr7-6Mb-seg-978 -> 60\n",
      "gene: Chr7-6Mb-seg-979 -> 60\n",
      "gene: Chr7-6Mb-seg-980 -> 60\n",
      "gene: Chr7-6Mb-seg-981 -> 60\n",
      "gene: Chr7-6Mb-seg-982 -> 60\n",
      "gene: Chr7-6Mb-seg-983 -> 60\n",
      "gene: Chr7-6Mb-seg-984 -> 60\n",
      "gene: Chr7-6Mb-seg-985 -> 60\n",
      "gene: Chr7-6Mb-seg-986 -> 60\n",
      "gene: Chr7-6Mb-seg-987 -> 60\n",
      "gene: Chr7-6Mb-seg-988 -> 60\n",
      "gene: Chr7-6Mb-seg-989 -> 60\n",
      "gene: Chr7-6Mb-seg-990 -> 60\n",
      "gene: Chr7-6Mb-seg-991 -> 60\n",
      "gene: Chr7-6Mb-seg-992 -> 60\n",
      "gene: Chr7-6Mb-seg-993 -> 60\n",
      "gene: Chr7-6Mb-seg-994 -> 60\n",
      "gene: Chr7-6Mb-seg-995 -> 60\n",
      "gene: Chr7-6Mb-seg-996 -> 60\n",
      "gene: Chr7-6Mb-seg-997 -> 60\n",
      "gene: Chr7-6Mb-seg-998 -> 60\n",
      "gene: Chr7-6Mb-seg-999 -> 60\n",
      "gene: Chr7-6Mb-seg-1000 -> 60\n",
      "gene: Chr7-6Mb-seg-1001 -> 60\n",
      "gene: Chr7-6Mb-seg-1002 -> 60\n",
      "gene: Chr7-6Mb-seg-1003 -> 52\n",
      "gene: Chr7-6Mb-seg-1004 -> 60\n",
      "gene: Chr7-6Mb-seg-1005 -> 60\n",
      "gene: Chr7-6Mb-seg-1006 -> 60\n",
      "gene: Chr7-6Mb-seg-1007 -> 60\n",
      "gene: Chr7-6Mb-seg-1008 -> 60\n",
      "gene: Chr7-6Mb-seg-1009 -> 60\n",
      "gene: Chr7-6Mb-seg-1010 -> 60\n",
      "gene: Chr7-6Mb-seg-1011 -> 60\n",
      "gene: Chr7-6Mb-seg-1012 -> 60\n",
      "gene: Chr7-6Mb-seg-1013 -> 60\n",
      "gene: Chr7-6Mb-seg-1014 -> 60\n",
      "gene: Chr7-6Mb-seg-1015 -> 60\n",
      "gene: Chr7-6Mb-seg-1016 -> 60\n",
      "gene: Chr7-6Mb-seg-1017 -> 60\n",
      "gene: Chr7-6Mb-seg-1018 -> 60\n",
      "gene: Chr7-6Mb-seg-1019 -> 60\n",
      "gene: Chr7-6Mb-seg-1020 -> 60\n",
      "gene: Chr7-6Mb-seg-1021 -> 60\n",
      "gene: Chr7-6Mb-seg-1022 -> 60\n",
      "gene: Chr7-6Mb-seg-1023 -> 37\n",
      "gene: Chr7-6Mb-seg-1024 -> 60\n",
      "gene: Chr7-6Mb-seg-1025 -> 60\n",
      "gene: Chr7-6Mb-seg-1026 -> 60\n",
      "gene: Chr7-6Mb-seg-1027 -> 60\n",
      "gene: Chr7-6Mb-seg-1028 -> 60\n",
      "gene: Chr7-6Mb-seg-1029 -> 60\n",
      "gene: Chr7-6Mb-seg-1030 -> 60\n",
      "gene: Chr7-6Mb-seg-1031 -> 60\n",
      "gene: Chr7-6Mb-seg-1032 -> 60\n",
      "gene: Chr7-6Mb-seg-1033 -> 60\n",
      "gene: Chr7-6Mb-seg-1034 -> 60\n",
      "gene: Chr7-6Mb-seg-1035 -> 60\n",
      "gene: Chr7-6Mb-seg-1036 -> 60\n",
      "gene: Chr7-6Mb-seg-1037 -> 60\n",
      "gene: Chr7-6Mb-seg-1038 -> 60\n",
      "gene: Chr7-6Mb-seg-1039 -> 60\n",
      "gene: Chr7-6Mb-seg-1040 -> 60\n",
      "gene: Chr7-6Mb-seg-1041 -> 60\n",
      "gene: Chr7-6Mb-seg-1042 -> 60\n",
      "gene: Chr7-6Mb-seg-1043 -> 60\n",
      "gene: Chr7-6Mb-seg-1044 -> 60\n",
      "gene: Chr7-6Mb-seg-1045 -> 60\n",
      "gene: Chr7-6Mb-seg-1046 -> 49\n",
      "gene: Chr7-6Mb-seg-1047 -> 60\n",
      "gene: Chr7-6Mb-seg-1048 -> 60\n",
      "gene: Chr7-6Mb-seg-1049 -> 60\n",
      "gene: Chr7-6Mb-seg-1050 -> 60\n",
      "gene: Chr7-6Mb-seg-1051 -> 60\n",
      "gene: Chr7-6Mb-seg-1052 -> 60\n",
      "gene: Chr7-6Mb-seg-1053 -> 60\n",
      "gene: Chr7-6Mb-seg-1054 -> 60\n",
      "gene: Chr7-6Mb-seg-1055 -> 60\n",
      "gene: Chr7-6Mb-seg-1056 -> 60\n",
      "gene: Chr7-6Mb-seg-1057 -> 60\n",
      "gene: Chr7-6Mb-seg-1058 -> 60\n",
      "gene: Chr7-6Mb-seg-1059 -> 60\n",
      "gene: Chr7-6Mb-seg-1060 -> 60\n",
      "gene: Chr7-6Mb-seg-1061 -> 60\n",
      "gene: Chr7-6Mb-seg-1062 -> 60\n",
      "gene: Chr7-6Mb-seg-1063 -> 60\n",
      "gene: Chr7-6Mb-seg-1064 -> 60\n",
      "gene: Chr7-6Mb-seg-1065 -> 60\n",
      "gene: Chr7-6Mb-seg-1066 -> 60\n",
      "gene: Chr7-6Mb-seg-1067 -> 60\n",
      "gene: Chr7-6Mb-seg-1068 -> 60\n",
      "gene: Chr7-6Mb-seg-1069 -> 60\n",
      "gene: Chr7-6Mb-seg-1070 -> 60\n",
      "gene: Chr7-6Mb-seg-1071 -> 60\n",
      "gene: Chr7-6Mb-seg-1072 -> 58\n",
      "gene: Chr7-6Mb-seg-1073 -> 60\n",
      "gene: Chr7-6Mb-seg-1074 -> 60\n",
      "gene: Chr7-6Mb-seg-1075 -> 60\n",
      "gene: Chr7-6Mb-seg-1076 -> 60\n",
      "gene: Chr7-6Mb-seg-1077 -> 60\n",
      "gene: Chr7-6Mb-seg-1078 -> 59\n",
      "gene: Chr7-6Mb-seg-1079 -> 60\n",
      "gene: Chr7-6Mb-seg-1080 -> 60\n",
      "gene: Chr7-6Mb-seg-1081 -> 60\n",
      "gene: Chr7-6Mb-seg-1082 -> 60\n",
      "gene: Chr7-6Mb-seg-1083 -> 52\n",
      "gene: Chr7-6Mb-seg-1084 -> 60\n",
      "gene: Chr7-6Mb-seg-1085 -> 60\n",
      "gene: Chr7-6Mb-seg-1086 -> 60\n",
      "gene: Chr7-6Mb-seg-1087 -> 60\n",
      "gene: Chr7-6Mb-seg-1088 -> 60\n",
      "gene: Chr7-6Mb-seg-1089 -> 60\n",
      "gene: Chr7-6Mb-seg-1090 -> 60\n",
      "gene: Chr7-6Mb-seg-1091 -> 60\n",
      "gene: Chr7-6Mb-seg-1092 -> 59\n",
      "gene: Chr7-6Mb-seg-1093 -> 60\n",
      "gene: Chr7-6Mb-seg-1094 -> 60\n",
      "gene: Chr7-6Mb-seg-1095 -> 60\n",
      "gene: Chr7-6Mb-seg-1096 -> 60\n",
      "gene: Chr7-6Mb-seg-1097 -> 60\n",
      "gene: Chr7-6Mb-seg-1098 -> 60\n",
      "gene: Chr7-6Mb-seg-1099 -> 60\n",
      "gene: Chr7-6Mb-seg-1100 -> 60\n",
      "gene: Chr7-6Mb-seg-1101 -> 60\n",
      "gene: Chr7-6Mb-seg-1102 -> 60\n",
      "gene: Chr7-6Mb-seg-1103 -> 60\n",
      "gene: Chr7-6Mb-seg-1104 -> 60\n",
      "gene: Chr7-6Mb-seg-1105 -> 60\n",
      "gene: Chr7-6Mb-seg-1106 -> 60\n",
      "gene: Chr7-6Mb-seg-1107 -> 60\n",
      "gene: Chr7-6Mb-seg-1108 -> 60\n",
      "gene: Chr7-6Mb-seg-1109 -> 60\n",
      "gene: Chr7-6Mb-seg-1110 -> 60\n",
      "gene: Chr7-6Mb-seg-1111 -> 60\n",
      "gene: Chr7-6Mb-seg-1112 -> 60\n",
      "gene: Chr7-6Mb-seg-1113 -> 60\n",
      "gene: Chr7-6Mb-seg-1114 -> 60\n",
      "gene: Chr7-6Mb-seg-1115 -> 60\n",
      "gene: Chr7-6Mb-seg-1116 -> 60\n",
      "gene: Chr7-6Mb-seg-1117 -> 60\n",
      "gene: Chr7-6Mb-seg-1118 -> 60\n",
      "gene: Chr7-6Mb-seg-1119 -> 60\n",
      "gene: Chr7-6Mb-seg-1120 -> 60\n",
      "gene: Chr7-6Mb-seg-1121 -> 60\n",
      "gene: Chr7-6Mb-seg-1122 -> 60\n",
      "gene: Chr7-6Mb-seg-1123 -> 56\n",
      "gene: Chr7-6Mb-seg-1124 -> 60\n",
      "gene: Chr7-6Mb-seg-1125 -> 53\n",
      "gene: Chr7-6Mb-seg-1126 -> 55\n",
      "gene: Chr7-6Mb-seg-1127 -> 60\n",
      "gene: Chr7-6Mb-seg-1128 -> 60\n",
      "gene: Chr7-6Mb-seg-1129 -> 60\n",
      "gene: Chr7-6Mb-seg-1130 -> 60\n",
      "gene: Chr7-6Mb-seg-1131 -> 51\n",
      "gene: Chr7-6Mb-seg-1132 -> 60\n",
      "gene: Chr7-6Mb-seg-1133 -> 60\n",
      "gene: Chr7-6Mb-seg-1134 -> 60\n",
      "gene: Chr7-6Mb-seg-1135 -> 51\n",
      "gene: Chr7-6Mb-seg-1136 -> 60\n",
      "gene: Chr7-6Mb-seg-1137 -> 60\n",
      "gene: Chr7-6Mb-seg-1138 -> 60\n",
      "gene: Chr7-6Mb-seg-1139 -> 60\n",
      "gene: Chr7-6Mb-seg-1140 -> 60\n",
      "gene: Chr7-6Mb-seg-1141 -> 60\n",
      "gene: Chr7-6Mb-seg-1142 -> 60\n",
      "gene: Chr7-6Mb-seg-1143 -> 54\n",
      "gene: Chr7-6Mb-seg-1144 -> 48\n",
      "gene: Chr7-6Mb-seg-1145 -> 60\n",
      "gene: Chr7-6Mb-seg-1146 -> 48\n",
      "gene: Chr7-6Mb-seg-1147 -> 53\n",
      "gene: Chr7-6Mb-seg-1148 -> 30\n",
      "gene: Chr7-6Mb-seg-1149 -> 60\n",
      "gene: Chr7-6Mb-seg-1150 -> 59\n",
      "gene: Chr7-6Mb-seg-1151 -> 60\n",
      "gene: Chr7-6Mb-seg-1152 -> 60\n",
      "gene: Chr7-6Mb-seg-1153 -> 60\n",
      "gene: Chr7-6Mb-seg-1154 -> 60\n",
      "gene: Chr7-6Mb-seg-1155 -> 60\n",
      "gene: Chr7-6Mb-seg-1156 -> 60\n",
      "gene: Chr7-6Mb-seg-1157 -> 60\n",
      "gene: Chr7-6Mb-seg-1158 -> 60\n",
      "gene: Chr7-6Mb-seg-1159 -> 60\n",
      "gene: Chr7-6Mb-seg-1160 -> 60\n",
      "gene: Chr7-6Mb-seg-1161 -> 60\n",
      "gene: Chr7-6Mb-seg-1162 -> 60\n",
      "gene: Chr7-6Mb-seg-1163 -> 60\n",
      "gene: Chr7-6Mb-seg-1164 -> 60\n",
      "gene: Chr7-6Mb-seg-1165 -> 60\n",
      "gene: Chr7-6Mb-seg-1166 -> 60\n",
      "gene: Chr7-6Mb-seg-1167 -> 60\n",
      "gene: Chr7-6Mb-seg-1168 -> 60\n",
      "gene: Chr7-6Mb-seg-1169 -> 60\n",
      "gene: Chr7-6Mb-seg-1170 -> 60\n",
      "gene: Chr7-6Mb-seg-1171 -> 60\n",
      "gene: Chr7-6Mb-seg-1172 -> 60\n",
      "gene: Chr7-6Mb-seg-1173 -> 60\n",
      "gene: Chr7-6Mb-seg-1174 -> 60\n",
      "gene: Chr7-6Mb-seg-1175 -> 60\n",
      "gene: Chr7-6Mb-seg-1176 -> 60\n",
      "gene: Chr7-6Mb-seg-1177 -> 51\n",
      "gene: Chr7-6Mb-seg-1178 -> 60\n",
      "gene: Chr7-6Mb-seg-1179 -> 60\n",
      "gene: Chr7-6Mb-seg-1180 -> 60\n",
      "gene: Chr7-6Mb-seg-1181 -> 60\n",
      "gene: Chr7-6Mb-seg-1182 -> 60\n",
      "gene: Chr7-6Mb-seg-1183 -> 53\n",
      "gene: Chr7-6Mb-seg-1184 -> 60\n",
      "gene: Chr7-6Mb-seg-1185 -> 60\n",
      "gene: Chr7-6Mb-seg-1186 -> 60\n",
      "gene: Chr7-6Mb-seg-1187 -> 60\n",
      "gene: Chr7-6Mb-seg-1188 -> 60\n",
      "gene: Chr7-6Mb-seg-1189 -> 60\n",
      "gene: Chr7-6Mb-seg-1190 -> 60\n",
      "gene: Chr7-6Mb-seg-1191 -> 60\n",
      "gene: Chr7-6Mb-seg-1192 -> 60\n",
      "gene: Chr7-6Mb-seg-1193 -> 60\n",
      "gene: Chr7-6Mb-seg-1194 -> 60\n",
      "gene: Chr7-6Mb-seg-1195 -> 60\n",
      "gene: Chr7-6Mb-seg-1196 -> 60\n",
      "gene: Chr7-6Mb-seg-1197 -> 60\n",
      "gene: Chr7-6Mb-seg-1198 -> 60\n",
      "gene: Chr7-6Mb-seg-1199 -> 45\n",
      "gene: Chr7-6Mb-seg-1200 -> 60\n",
      "False\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "# get region size dict\n",
    "min_num_probes = 21\n",
    "reg_size_dic, len_check = check._check_region_size(blasted_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 247,
   "id": "2efb4394",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAApQAAAFKCAYAAACjE8/+AAAAAXNSR0IArs4c6QAAIABJREFUeF7sXQXYFUUXPnR3l3R3dze22CCKIoIN1k9IKWBhYGIrFhYGDZLS3d0g3Q0f8D/v3Dv7nTvfbN3AC848j48fd2dmZ96d3X33zDnvSXb58uXLZIpBwCBgEDAIGAQMAgYBg4BBIEwEkhlCGSZypplBwCBgEDAIGAQMAgYBg4BAwBBKsxAMAgYBg4BBwCBgEDAIGAQiQsAQyojgM40NAgYBg4BBwCBgEDAIGAQMoTRrwCBgEDAIGAQMAgYBg4BBICIEDKGMCD7T2CBgEDAIGAQMAgYBg4BBwBBKswYMAgYBg4BBwCBgEDAIGAQiQsAQyojgM40NAgYBg4BBwCBgEDAIGAQMoTRrwCBgEDAIGAQMAgYBg4BBICIEDKGMCD7T2CBgEDAIGAQMAgYBg4BBwBBKswYMAgYBg4BBwCBgEDAIGAQiQsAQyojgM40NAgYBg4BBwCBgEDAIGAQMoTRrwCBgEDAIGAQMAgYBg4BBICIEDKGMCD7T2CBgEDAIGAQMAgYBg4BBwBBKswYMAgYBg4BBwCBgEDAIGAQiQsAQyojgM40NAgYBg4BBwCBgEDAIGAQMoTRrwCBgEDAIGAQMAgYBg4BBICIEDKGMCD7T2CBgEDAIGAQMAgYBg4BBwBBKswYMAgYBg4BBwCBgEDAIGAQiQsAQyojgM40NAgYBg4BBwCBgEDAIGAQMoTRrwCBgEDAIGAQMAgYBg4BBICIEDKGMCD7T2CBgEDAIGAQMAgYBg4BBwBBKswYMAgYBg4BBwCBgEDAIGAQiQsAQyojgM40NAgYBg4BBwCBgEDAIGAQMoTRrwCBgEDAIGAQMAgYBg4BBICIEDKGMCD7T2CBgEDAIGAQMAgYBg4BBwBBKswYMAgYBg4BBwCBgEDAIGAQiQsAQyojgM40NAgYBg4BBwCBgEDAIGAQMoTRrwCBgEDAIGAQMAgYBg4BBICIEDKGMCD7T2CBgEDAIGAQMAgYBg4BBwBBKswYMAgYBg4BBwCBgEDAIGAQiQsAQyojgM40NAgYBg4BBwCBgEDAIGAQMoTRrwCBgEDAIGAQMAgYBg4BBICIEDKGMCD7T2CBgEDAIGAQMAgYBg4BBwBBKswYMAgYBg4BBwCBgEDAIGAQiQsAQyojgM40NAgYBg4BBwCBgEDAIGAQMoTRrwCBgEDAIGAQMAgYBg4BBICIEDKGMCD7T2CBgEDAIGAQMAgYBg4BBwBBKswYMAgYBg4BBwCBgEDAIGAQiQsAQyojgS2x89uxZWrlyJeXKlYtSpkwZpV5NNwYBg4BBwCBgEDAIxCsCCQkJdODAAapYsSKlTZs2Xod5RcZlCGWUYF64cCHVqlUrSr2ZbgwCBgGDgEHAIGAQuFoQWLBgAdWsWfNqGW5MxmkIZZRg3bZtGxUtWpSwqPLlyxelXk03BgGDgEHAIGAQMAjEKwJ79uwRxqStW7dSkSJF4nWYV2RchlBGCeZdu3ZRoUKFaOfOnVSwYMEo9Wq6MQgYBAwCBgGDgEEgXhEw7/7EK2MIZZRWqVlUUQLSdGMQMAgYBAwCBoGrBAHz7jeEMupL1SyqqENqOjQIGAQMAgYBg0BcI2De/YZQRn2BmkUVdUhNhwYBg4BBwCBgEIhrBMy73xDKqC9Qs6iiDqnp0CBgEDAIGAQMAnGNgHn3G0IZ9QVqFlXUITUdGgQMAgYBg4BBIK4RMO9+QyijvkDNooo6pKZDg4BBwCBgEDAIxDUC5t1vCGXUF6hZVFGH1HRoEDAIGAQMAgaBuEbAvPsNoYz6AjWLKuqQmg4NAgYBg4BBwCAQ1wiYd78hlFFfoGZRRR1S06FBwCBgEDAIGATiGgHz7jeEMuoL1CyqqENqOjQIGAQMAgYBg0BcI2De/YZQRn2BmkUVdUhNhwYBg4BBwCBgEIhrBMy73xDKqC9Qs6iiDqnp0CBgEDAIGAQMAnGNgHn3G0IZ9QVqFlXUITUdGgQMAgYBg4BBIK4RMO9+QyijvkDNooo6pKZDg4BBwCBgEDAIxDUC5t3/HyGUn332GQ0fPpzWr19PmTNnpjp16tAff/xhzX7cuHHUp08fWrt2LRUsWJB69uxJjz76aFiL1yyqsGAzjQwCBgGDgEHAIHDVImDe/f8BQjlgwAB66623BGGsXbs2HT58mCZMmEAjRowQs587dy41atSIOnXqRB07dqTZs2dT//79xfEuXbr4XtxmUfmGzDQwCBgEDAIGAYPAVY2Aefdf44QSFseKFSsSLJCtWrXSLta2bdsKkjl//nzreNeuXWnMmDGEBZI8eXJfi9wsKl9wmcoGAYOAQcAgYBC46hEw7/5rnFC+8MILNHr0aNqwYYN2sZ47d05sgb/yyivUo0cPq86MGTOoSZMmtGjRIqpevbqvhW4WlS+4TGWDgEHAIGAQMAhc9QiYd/81TigbN25MOXPmpCpVqtC7775LR48epbp169I777wjfluzZg2VL1+exo8fT23atLHQOHDgAOXOnZtGjhwptsH9lCu5qBZuO0zJkxFVL5w9yRDPXrhIk9fso5pFstO8LYdo84GTlCVdKlEvTaoUVLlgFlq07Qhdunw5pO2xMxcofeqUlCpFMvH76fMXCVUypEkh/savGdOmpGzpU9PBk+eSnPfE2QTKnzUtNS2Tmyat3kcYR5qUySlNyhR0/OyFJPXRz3U50tPSHUcI565cMCuVL5CFJq3eS3uPnaXiuTNS8VwZxXFezl+8RGfPX6TMwTnh2OFT5+nGyvnp1LkEWrT9CJ04e0GMU5aT5xIoRbJklC51CvFTxjQpKXXK5JRw8TLtOnqGWpbNQ8t2HaVzFy5qLzn6z5o+FSVPloyuy56eUqVMTpv3nxR1Ey5dppNnE8Tx42cTKG2q5JQ6Rah1G+cD2hifuA7AJVUK8e/W5fPSil3HaNeR02Ie2TOkpgJZ01GGNCkpfeoUtPvoGTqXcIl2HTlDLcrmpuW7jlnjxPlwXZIlQ58p6Mz5BAuXQ6fOi3HkyJiaUiZPTqfPB84tC/oE7pnTpqS0qVIQ6ufIkJoKZU8vsNl//CxdvES09/hZcS1xLAUWXbBgrQTWSWK/Fy9dFn1iDtEqGOeFi5fENZNFXavyOuB3jPPCxctiDfBxYA2kTJ5MzFUtOAeuvVxTWPdHTp+nIjky0IVLl+jM+cC6wBoHNpgn/sO/AUn+rOnE/w+fDuCJ61fluqw0Z9Mh0Q7X/viZC+Ke3LDvBCVcukS5M6Wl+iVy0riVe8T9hOuN61UqTyZasuOImLNunOeDeKA+/sb9VqVQVlu40Q+eB5ULZRXjuhrKzsOnafr6/dS8bB5aufsYLd95lApkS0fl82ehAyfOifvg7IVLNGbFP2K95c6cltpVyEsplfsOc5296SBlTpuKKhbM4jj11f8co7Er9lCr8nkd8ZSdHDp5juZsPkR1iuWgv9buI6wvWXTPKL+4q2tS154/l+z6l2vZ6z158OR5ypkxtZhPMkpGKVMkE+ssU9rA/YdnTqrkic9SL/PC8wP3TdZ0qcW7AOPGuwTXEPcLStb0qUWdAlnT0uQ1+8UYvBT5bsKzXT7D1Hbr9p4Q9yvuv3trX+elW191ruS739fA/oXKyS5fVpjFvzCIaJ+ydOnS9M8//1CBAgVoyJAhlDp1aho4cCBt27aNNm7cSKtXr6YGDRoIP0oE6siSkJBAqVKlEsTzySefdBzW8ePHCf/JsmfPHqpVqxbt3LlTBPjEqoB4NHh1muh+bq9mlC9L6EtiwB+r6cs522J1+pj2ixcqHsb/pQIyeO3dgf+lKxj+XB9qUJQ++3tr+B0EW64e2FoQUl35cPpmenXCOnFo2yvXR3yuK9FBizdn0KbgB5vufO/eU5X+OXqGho4PzAvlvXur0g2V8odUX7X7GN3w7t/it1UDW4d8lPCKIDxlXpxg/bT0xZaUzeWjqOkb02nrwVNXAg5zjighgI/NxS+2jFJvid0YQpmIxTVJKEuWLEmbNm2iVatWCUskCghf0aJFadCgQVS/fn1BKOfNmycCdmSRhBKR4U888YTjwkPQD0iqWmJNKPE1/NBXi8Rpv+hck5qWzh0yhCL/G5tkTLB04evv6OlESyG+APHljgIrHb5CUXJlSiP+xpe/U4GlDhYflP0nzoV8oavtQJqK5shg/fzPsTPi69RryZclLaVLlYJOnEsQFgoUWPnyZE5LOw6ftu2mWM4MdODkOWFJQsFXOr685VzVhvhazpkxTcjPWxxeGnkzpxUWPLWgD/nlDcuiJMmwXsKyhy/0SAr6FlYQDYbABdYNtQA/4Ihy9EzASuC3YL5YS/uOn6VTQatdhtQpxHXAOWFNlQXYR1pgJTwSXLPyPBzP3JnSCJKgnpufF+Pg6xNrICuzbmM94DgKrDC4J/g88LuwMF++bM050nnJ9rAsLtt5VNtdtvSpQqzsuB9hgdGVWc83FZZlXan/ylRrPlcLodQ9w/jc2pTPS3mzpA35cH7xhnIEgs7LJzO30OBxa8VPM55rQoXZM4jXg0W04WuBj3SUPx9v4GrRVMeINYL1yJ9RuN/zKx/8XtYOrHiwFKLY9cGfS/mzpNVa3vGMwP3i5Z68ePkybT9k/yzF/QcLsHwv8Gec05zcnjWYn927wO0ZAss+LKZq4e+mMxcu0p5jgWc07u+VA1p7uQS+6hhCmQjXNUkoQRK3b99Oe/fuDVkY2O6uXLkywccy0i3vf8tCia2gB75YKOb1aaca1KJcnpA56h7G2A4unScjvTEp0ad08K0VqEPtwqLt7R/OEVvFKB92qCa2mT6YvtnxplrYp4Ugnyj/+2UF/bBwZ0h9kB55s2M7eFm/xOCoLl8toilr93m+aX/uVpdqFMlO41fuoe7fLhHt6hXPQbBUVH95irafUnky0qQejYlbbPteX1Zsxy/Ydljb5pHGxahX27KueMoK33apTR0+TQzqkr8Pu6Myta8esFJfP3wWrf4nYMmuXyIH3VylAD3/8wrPc9dV7NqoGK3fe4JmbDiQ5HDtotlp/tak88M24af31xT1+YvWz0C+61Kb6pXISX1Gr6Rv5+8QTbGFNOTWimIrt9VbM8VveNks6tvCT9fauh/N2EyvBK1QN1XOT8PvqUo3vfe3cBFA+ahjdWpTIa8guLWH/CV+4xbfigWy0J9PNKAXfl5BoxYF1me/G8rRg4x4zNl8kO79JHANH6xflNpXL0DXDw9YtWS5vXpBsWU2eunuiOfEO8Aa3bAv4Dqhludal6bHmpawfv5y9lYa8Ocabd0pPRtRidyZtMeuRULZtkJeQd6+mrvdmjPu7S4Ni4Vg8PHMzTRkXMCKOfO5psLFRlfW7T1Obd6eZR0Kh1C2r1aQht1ZWWzDP/7dUtFXw5I5aeRDiQYLr4vnx0U7rWdEk9K56MvOtZI0Ld57nFiTKKMfrUdVr8uWpM62g6eoyRvTrd+dPihUK63aGd4hZfNlotcmrBeHPulUg1oq7x7d/D6dtYVeHhsg9boCjOCaBTcVtbh9AA36cw19PjuphX/Jiy0td5dF2w7T7R/NFV0bQul1BYZf75oklA888ICQCFIJJchktWrV6KOPPrqqgnJmbTxAa/ccp4caFKO5mw9Rx88CL8AR91UXPni86AjlLVXyC78g/kIaeltFuqdWwJ/kzhFzaUGQhKBPbBW9O3WT46pa3LcF5Qha83Tb7HiZg5iiwNqylBHKHqOW+Xo5//ZYfTH+KWv2UZevA9bZxqVy0bv3VqVKAyZpxykJJQgJiAkKSPSEVXtp1saD2jaPNy1Bz7Yu7YqnrACiKx9WvNHbd1WhW6oWED/d+O7fFg54eIIM3vfZgvDvWCLq1ri42BLUkfIGJXLS35uSzq9VuTz0caca4rxfzN5KA23IidPAvn+4DtUtnoPem7rR+jh5pmUpeqJ5SbEFWe+VqaJ5tAglJwRYw2/fXZVufn+28KlD+fi+6sLnDS9WvGBRuNVYEsoXf1tFI+cFyMfLt1SgjnUCH1Io2LbE9iUKCOUdNQpS23cSyQV+v6N6QeE/K0lpRBePNYZPo2oNlYdVQvn13G3U7/fV2lM7EaBYE0pYbr+as40KZkuf5OM2XJzcLJQglFhj8priPLCgjX2yIRVhlvERMzZb2+JOVtzF249Q+w/nWMPFhwpI5h01Ctn6napjxLMUz9SJq/fSIyMXi76alclNnz8Q+IjzU35ZvIue+Wm5aNK8TG76TNNHsV5jKcgnST4f1XPsOHSaGr2eaHl1Imi4jkV7Be4hXcH9Vy5/ZougeyHd6Gfk3G30os26xXEQ5vlbDhMsiWpxI5SDx66hT2YlJZTL+7ey4gb4tTWE0s8qDK/uNUkof/75Z7rjjjto5cqVVKFCBYHM7t27qVixYvTqq6/S008/TZANQrAO/Chl6datmxA+jzfZIPnwgiWoaM4MdM8n88SQP+hQjdpVzBdy5e0IZYOSuejZ4EMKDV65rSLdHSSU93w8j+ZuCQQQfHZ/DUGA3p6y0XFFLevXUjhSi74YaZONrq+UTzi5oyCYY/OQdlZ//AXvZdmOeaIBVSiQRTjqS+ts09K56IMO1alsv0TfJ96XJJRvT9lgzQWWw/Gr9tCUtfu1p326RUl6ukUpVzxlBTxUb3wv1JqFY8KSVjngz8Utao1K5SJYUqQlz8vcdXVAfBFsNX5VqAUedUG0dZbLdhXzCrxQ8CLGNfBbfuhaRwQh/L5sNz31wzLRXJJnbNNJch8tQsmtG7dVLUBv3lWFbv1gNi3dESCU3EIv133HOtfRN/MC1tNHGhWjXu3K0ktj1li+iq/dXonurFHImjq3zGB9lC+QOcRahYp31SgkXEY4gfGLna4+LPfcDYXXeaFNGerepLj10zfztlNfm2v2U7e6IuBAV2JNKBFE1zVIoNa91Ea79eoXKy+EEj6O3wWt5LJ/uGOsGZQYZMkt3E6EEh/suo+8QtnT0aznm2mHr47xgXpFaMBN5Wnqun304JeBj15Y8GDJ81v4/cU/BHk/nFD+/lh9EXSlFu5vj2NuBM0Jd1hgr6+U15rbgj7NRWCZWxm1cAe98MtK22ogzNgxkm5JvKLbeIeOX0sjZmxJ0jf3KUaA220fBD4WDKF0u1qRH78mCeXFixdFgMyJEyfo5ZdfFkE58J3cv3+/yJqTIUMGS9gc1swOHToIYfN+/frFpbC5vNHxUr2n9nV0R9CE/87dVcQWKi92hLJNhXzU7ZvAlzPKa+0r0Z01Ay/WDp/Oo9nBiFT4Za7adYyGTdZLLsn2Kwa0snww35q8gd75K5SA4gHLg4P4w8EvoZzwdEMqkzczzdl0kO4NbjGDUOJhXaLPeO1dIAklf6m8f281EVU7dmWA6Krl2Val6PFmJV3xlBUm9WikJYc8QIBb1PA1PuDG8iHbUOHcwk82LykIpSTsvA87QnlDpXz03r3VRNXvF+ygXr/aP+TtxjSqax2qXSyHiPoeEvRN692urCAR3EoYLUKJgBWQQRS5pQhLEqwOKJ8/UIOalQm4fExbv5+mrt0vLMwLtx6mmRsPEEgZglUQlILgFBTdPYMtRkTtP9+mjMBVJfz31CokfF+9BNCUzJ2RNjoElHBsnYLQ/te2jLBEy+J0zUY+VIsalsylvWwNXp1q+dG5vaDDWYvDJq23djP4R2Y4fck2boQSH0dQrvh+QaibDdrzOXollBNW7aFu3wRcadRih5k6Ruw84F7g5NSODLph8+fyf+iJ7wPb5rDGftgxqYQdJ5R21sI9x85Q3aGBXQMVG90Y+JxgIf958S4r8OjOGgXp1faV6M3JG4S7wX11i7hNQxwfvXQX9RgVsLbqCjCCaon0lZZ1Bt5Unu6v53yO1yeuo/enJXXN4h828FG+5f3ZoltDKD1dsogqXZOEEoiAPEJjcuzYsXThwgWClBAy5yACXBYIn/fu3Tsk9eJjjz0WFqCxcsy9dOkyFQtu591WrYDYrpNfXG/eWZluqxYaUa57GN9cJb+wynB/v9dvryS2dFA4ocTLCRZK6StjBwaPmnxnykZ6a0ooAcVWKCel/MHcc9Qy+tWHP9qUno2pRO6MYlse2/MoIJTYTrLbpsGLfXLPxiHbuyAgY5bvsT23+hLHeZxebtOfbaIlh/BDbRu0HHOLGrbAQCj5NlQ4i61ny1K0ft8JX4RSbhnjfD8t2knPheHH6WQJ41hFi1DyrXlsO79+R+UQf19dUJoOT/7Bw6+Nru6m/SeoxZsBX1BZ4CeKYB3pOmF3zWCV337oFK3anaj+EM71RZtebcvQI4xQOll65Na/7lyxJpRvTlpPw4PuMV6io73g4YVQZkqTSuuCEA6h5FvM6vi8EkrpLgOXJLmDFC6h5L7i/EOQj437UModHHXs3LcYx9w+KDjuqPvjwp30/C8Bf2+5pe/l+vE6+Oh97Ds9WUc9EGb478tgS9n2x0fqUq2iequ7rMPXHj/npsFtLQmpFbuO0k3vGULp97qFW/+aJZThAhJuu1gRSujIlWRWOLxYf1q8SwyTk0I5bjtCCf8wWMtkeQMv52DgCCeUCLwAoeSSHDpM1g5qY+k66gjlSzeXD/Gd4Q+zh79eJLTx7Aq2yKXDOerICE1oUt4a3L4Aofyicy1bwicJJfy7+v8R8D3D3P5c8Y/WsoHjOsd+u5cbgj9mv9DM8hvkc+G+rbd9MJuWBLdoW5TNQ/1vLBcSURrOeoP1YM2e474IJT5G3ryzijjdb0t309OjAlvWfooMjrJrI7FCsBaCtiIt3G/w7pqF6JX2lUII5VcP1hJb/G7l/Wmb6PWJgWACuHRA39CuwELZfNiMkMPYRgehdAtUww4Com/tIrfdxsmP925Xhro2SrRQOn0EwOq6fOcxoQOrBqbEnFBO3kDDg7sTPBjCz1zVul4IJSzGsKCphT9nuGTS3y80FX6euuLkn8r7w/YpnifYIVDXCD7y8Pvi7Yep/YeBj95wCSX3w4Qx4J27qyYZthdCCZJWc3Bi0KJfQgl8pZsU7oGXb6no+7LiOY/nvV3BR9jS7Ufon2Aktqzn9vGKetydife/dWg7SoYHNOSimHSUsVD6vny+GxhC6RsyfYNYEUqn6DseWCNHpXsYw5JZqWBWgr6bLNy6yQkltjVBKGVknp1OIt9W0BFKbKnLr1uckz/M4PvUe7T9liskbriTtnwZ8IeDG6Hs0aIUPdWipHgBSEL566P16I9l/9jqdA64sRw9UD9UeoTjCZUk6QiPvxf0aUE1NFHm3LePR9DjBdP/pvIEv7ZICiypuEZ+tryxZfXa7ZXFaXkkqp9x/NK9HlUvnDSaVF170nfRT9+6utzXU1pIOJ5OW728P04s3NrwIB3ZR6e6hYV0kBuhBOndcuCUrYqAHzzUjxsnKxpcKaavD0T8I7qeS1/FmlBy6y8P1PMzV7WuF0IJN4tflySNuufPmQ+mb7J2Wmb/r5ltgA1fH+pYeH/8g0m1qMndDW4RC5dQcmk46TusjotveY99soEQfVcLxNe5CoYboZT+tvD1/vrBWiEfntJH1O91hT/3/Z/bByHC13zpziO083CivBHO4fbxijrv/rVR65rF5wnBeqnaYAil36vnv74hlP4x07aIFaGETl6F/hO153zplgp0H4tYRSXdw3jLkHa088hpavx6ooTEW3dVplurBrbLOaHEjQyyIqOAEYyg023cOLgtpQpmptARSgRrcCsYv8kTLl6y9X3EeOAfxXUw5/duLrQOIZXT+u3AdqQToYQFr0vDoiJ7DCeU455sKAJKRsxM6siNPtUIYBVPrvEIK+rSfi21UeZfPFBTZDBBueOjObRwW8DnD/p5/W8qF+LXFM7y69OurMjs44dQwg9w6G2VxOmcfMacxgNCXk0jTyLb4PogMwnIn8xKFM78ZJtv52+nPqMDwUMdal9Hg2+tGGKhhGwTss24FU4spB+oXRtsWfP7BPXur1tY+GKqhFLV0APxhIVT+iO7jcvpuEoonazKPFpclcdp+NpU62XtRijCGS+3EnEpsXD6km3cCOX1FfOJICmdjBOfI7dMOxFKXk8dt45Q6uYm5ajW/HOc2g0PqASESyjhD9w5KA0nfYfVc3JCiecaIrDVcvT0eaoyaLL1s9v1h+IAttvhRgW91j+W/0NPBn05scPV78Zyvi8rdwHQNb61agGh2qDq/f7Sva42Exzvw+668XlCHUWqNhhC6fvy+W5gCKVvyPQNYkUo1YcCP7vOcVl9GEMcduqzTURqP5lhB33w4AROKKFpBkuglHqAZYanFZPnR9S2TMWnI5SIQH80qBmJNurDzOmlgS1TbgGQlo8tB05Ss+B2pBOh5NZTTign92hEcHiXPl/qlXy1fUW6q2Zoai4+zkxpUgrhYhSIusOPlGfYkP192bkmNQkKznNJJgQT9LuhPNUZGtBMDLfg5bV4xxFfhJJvWXH5JT9jsNO789OHn7o8EAUfTviA4hbK7x6uTfWK+yOUbnNQRa4xXlhnEEGsEkoQhknMdaNLg6IiIEcXZe9n3qirCnXzyF+1LyQpkELYqoB3rAklv/e9Rv66YeFKKCvlE6lUQXjUYkco5/yvmUiRqStcBsupP6dxyY9R7oMbLqHkgT1eCKUMWlTHLlLaDkyUVXMjlGp7BDDKZ7gMOnK7dupxVZJJPQ5CCSuiqsfqthuCfnjQFe+Xz5MbIQyh9Hv1/Nc3hNI/ZtoWsSKUyAag21bFIKQVA3pr8FX5oGM1qjU4lKwUy5WBpj7TROTH5kTGjlD+8Xh9YaGUliE7aRPup6IjlNj2lZqRGKsfQqnq8y3v14qypE9F/GXvRCg3vNxWWDBQVEIJAiD96dQLyf1K5TH+EkHqLpmtBDnP17/U1gqY4n3xbdW7Rsy1hMbhL9T/hnJUKyjCHe7Sw4cEpDb8WChhZRt4c0BCi8sv+RmDnTyJnz781OWBKHL8nFBKGSO3PrmF0m57UPahfnjhd1hnYI1UCSU2m/gRAAAgAElEQVQsZVwx4NEmxcWL0Y9ov93Y4WvbmblfOLkp4EUpZVfgtoGANxBSSAk1em2alU2K37NumHk9Dv9JRP6iLOjdXOTVjrR4IZTwkBsTlCWzIxPcguVEKO22TtXnltO4pBwVt3CHSyi5mgX3febz5D6UE59uRKXzJhW2V3e3/BJKaPZKZRAoDmBb32/hbkq6tiCUIH3wCefFbTcEde0SNPB5btx3gloGEy4YQun36vmvbwilf8y0LWJFKNVIPX5yGQkqH3TIZgALHC+SUKoO2sgyg/oo3EKJFy4eAlI7TLUWoj78KrcOTcwLrHtowAcHLxtE8HWuX4T63xhIgSmL08MZVlW+BSJ1xTgpdiKUPMoPAuDSdxQWxR8W7LDN3OAmw8RFs0FYQVx185AZZTBXrvEJvGFd5I7y4Sw/WEOgG+qHUPItK2xL6zL8uI0FHxvwxb1ShQeiyDXECaWXSFCMlRNKp6wyqMsF2uU8YXnE9VYJJT7oeBYQ6JgiY9C4lUn1Qf1ipvrz8shftS+7dJt4sXJCye8Lv+Oxq8/JmHRNibRvN0KJyOdLly9rceZkglse5/ZqRvls0iByUqyO3euWt3x2YNtY+kiHSyjnbzlEd30c0Bq2I5R8yxvyZaXyJCWUp88nULl+ie5Sfgkl38nQJX3wcp15Bi1dffiIbjpw0sp+Jeu47SSgHpcV433zeXKLsSGUXq5YZHUMoYwMP6t1rAglf0CpQ5XZNOQDuFaR7EkCAuSW95FT56nqS4n+NNBkhMUMhRNKfO3CQimj+3TZPLDdu4kJlaOPsi9OCAmkga9amXyZRdQj/Nzgz8iL00ujTN5MtG7vCav6+pfbiPbcydyJUKqWGESHY+u+ZJ5MjpkbOCby5HychbKns/zR8BJfb0MoueWMYwvpHliO7NJFel2KCMZCNhw/hPLhhkWpz/UBHyikOrs7+MLyek7Us5Mn8dOHn7o8EAV5moEdJ5ReHPdxPk4onVLwoa6q3YffsN2HNS8JJXxk8aGFAB6pF4h6z7cpTev2nNBuxfqZN+qq7izcWqT25UQoG78+zcrRLO8jv2Nxqs9J27xezUWO7UiKW8YW9A1CmXDxMk1YnZS4h0ModTsscg5eCaWUo9p/4qy1SxQuoeSR4l4IJVx58GxTixrQ6ZdQcl/OJ5qVoGdahWYR83Kd1fSPahsQym2HTllKGPK4XfYf3p7vPvHf+Ty5agNcllYONLm8vVy3cOsYQhkuckq7WBFKNX0WP61Me+dEKIvkSE/Tn2tKPJOJeMmyLDuc9MCCA0IpxWjRftuh0yGzhSDzhsFtQ36777P5ISkN3b4wnQhl5YJZaHkwXzNOIv01+Rxkjlu1H9V6ql5erq2mHpO5ofnvvH9YexHFi4It0HUv6S2UXPKi46fzrVSIeHiCFHFiH87yw/Ya/PT8EEq+ZcVfWH7Of6UJJRdFloSYE0ovflZirbNoXyf5GNTV7Qggah3+wpJQSnkcHrSAtri2CMr4ZUlSORs/OKPuoJvLUycmHu0kv2IXOIcXKyeUawa1FgLtkRTo4iaHxEGweN1W9npOLpBv1waEEoGC3H9V1uVkgltPnba8dYkZdP05PbOkHBX3eQ83Uw6XR/NCKKVOr4oXMCrVNzHxg19COXPDAeoUjNAOl1A6vb8wXjwTdx09Y6X+lXPw4l5jl/GLz5OrNhhC6fUuDL+eIZThYxfSMlaEkgeiqEOF7hn0z+SDLl+WtLRH0fOCVQ3pw9Ttj486ViNkz0HhhHLas00EoZTRfbrMH5JM8fHgwYMHkCxuvmpOD+eaRbJZkdGcIJ45f9FKtWhHKHXWUz5Op+AGnUYhHye3nELaaO1LbbRb3jxCkRNtONhjy7vyIH3+ca9LESkCp67f74tQPta0OD3XOuADhahKrknq9bxu19RrP17r8WslpYg4oXT7aJHn8UMouYVJtgcZB4eShFLmClYJJYKG1vxzzFbn1Ou8UU9VcOBSMmo/XgnlygGtKFPaVH6GEVIX7gBYN01K5RIi8yicULqRdS8nVnV3dW3gOnLmfII2hSpkaJD6FIUTSqcob/iASi1N9XxeLZTSb5r7LYZLKFfuOmaldPXiQ/nXM42peK6MSaBSyblfQsl9OWNJKPceP0tzNgdS/8rixb3GTn6Oz5P7tBpC6eUOjKyOIZSR4We1jhWh5E7F6lAlSXAiZ9iyxsP0XMJFKt03Me81t8ZxQoktQRBKmd2gXL7MSRymM6ROQatZzlyM64EvFlhaePi3naO4nIPTmOuXyGFJr3BrKJcbstvylr6NdpfVSTJHJ5TNx8mxkHmDJcHhEeCc6HCiDVH6F28sp5Ua8rMM4a8F64wfC6X8+MB53Bzl7cYy/qmGVDZfUnkSP2P3U5cTNmlh5YTSixUD55Pb2Fi3y/q3suSudGPRBcEh2AZSSA1fm0YIUlvSt6Ww0vEUeejrldsq0up/jkcl57cqYcW3H9Vx26VwVC2UkaZGfPy7JVYgjHxpc0LplC/b63VXn1N2hPLUuQSaum6/tls5Nu4b6UR27TKuoHOvhFIqDvDxh7vlzbUTvVgopz7TmIppCKXqPuCXUHLXmHAJpdsHAiyUB0+dDzFGAHe7dJL8gttlj+Lz5IGcxofS610Yfj1DKMPHLqRlrAgl19FShypfsk7kDFbLub2ah+RaRj88XRsnlCCf+EKW0X3q9jPa6r70HvxyYcgD3s6vxwuhBFmcFhRqlsQN7fgD0s5CKS2Hdpd12rr91PnLhdrDPJhGN84KBTJbafUkqca2ErZJX5mwziJ4/OuaE+27ahQShNJOV9TrUkRAFXzqdDnJ7XJ5S6F3nINLaXg9J+rZyZP46cNPXR7ZDFKHXNs8l7eXl4483/7jZylt6hRW/nm7cRw+dZ6qMV9j1JMBCbjO0KOEPy6KaqGESgC2vD+fvdXPNLV1B99agTrULmwdcxKIdiKUTV6fZrmsRCo83uWrhZZVUL60ufVXlSwKBwSnRA6yP1goT5y9EPIBy88lx8Z9I50I5RsT19N70zZph+uVUH7/cB2qWzwH8VS54Voo+f3phVBiV6lozgza8fN3g19CuWjbYbr9o0DWn3AJJdraBc/gGAgl5I3+Uj4OvLjX2GWP4vPkqg2GUIZzR/prYwilP7xsa8eKUDpZkxB92veGco75pvNkTkPze7cIIWOYxCedahAeeCicUMKxHhZKmS6rRuFsIlKbFwiPY9uPF/6ywe92fj2yjRMJbl0+D01cHUjNqD4EZDs7QqmznvJxztl8kO79ZL72OupEr/k4OblWSTUsutJiyB+Gnb9YYJFjWLkQGVzeRqje61JEAAAkU/wQShnAhXPo0gt6ObddNKmXtuHU4ZHN8oXGCaWXl47f8+p0X59sVoJ6agISVEIJMf+1e4/TiBl64Xw/Yxlya0VCDnFZ/t54kDp+pl+3TlvenFD6kfXBdumR0+dDsu7we1xu+/MsM07ExuvcVdccXTtsa2NsszYe1HarI5RO1tPXJ66j96dtduwLB52eWTwQT9YLl1DyyGQvhHL6s02oSAwIJVJN3hZMdRsJofx1yS7q+eNyLb4glHATUP1hvdzbdv1yQslVGwyh9HoXhl/PEMrwsQtpGStCibzAt7Ac3PykMh2W04MOadiQjg2lRO9xlBDMHcj9BTmhRLYLkFhpxatTLDvN23I4ZK7Z0qeipf2cCaWdX4/sqGivsXT5sh58Ln+EjA0IglCJqNxOUufu9tBwEtrVBXnw/qtel5WWBvNyq+fh24F8a/ihLxdaX9/I9tL3+nKWH2i4Sw+5wmGh1GUKsbNQvtCmDHVvEsgN7eYobzcuN6tzuPOxa8cjm+WWPSeUdhlCIhmHKgaNvp5qXpJ6tCyVpFsu/IyDsBxDJuXdqXprl59xYfv87lqJhNLpQ8iJUDZ9Y7qIRkdxks5RxybJI7fa4yMTwUEo8CnFs+KnxbvolfHrxG9u97yX+cPyWHGAs48xCCUsyVA60BVJKHgWHydC+dqEdbZpNb1aKPnHqHxmQKf0/Q7VvEw7pA4PJPFCKJ2UCyKxUPI0kpEQSie/dRDKcwmXknwce/HXtuuXXzOu2uD2bvB9oYINYvXuD3c8/2Y7QyijhH6sFpVTRK7MHuJEKCHGvThIyBDxJ9Mo8vSAnFCCvMFCKfOvcn9GCRUycyzqm0jy8LtqoXSzVsDi8uBXC7VpHRG8IiNlc2dKI3JmywJLIIJ/sN2Jr3I1/RZ83JYpZJdfYieLr84RnGOLPNYgpCiZ06akFQMSJSi4hZJb8rp8tcgSusb16nN9WW12HT/LEB8DkGTS5QS3I5RI1/hwo2LiNE5SVE7jcLM6+5mDl7qTVu+lriMXi6rQeHy6RamQLe9YbMHrCI08tzpmdXsWlmPop9oJ53uZs6yjZm1yknpy2vLmhNKPj6Nc94VzpKcZzzUVw+r69aIQSxKi2uHXPDRIKN00Pr3MX0fo1XYglPB1VQM5ZD0doXQiXa9OWEewtDqRUxxzes5yTdRHv10srKf44CmUPb2XaYfU4R98XoJynK5rJISSPysjIZTYuZE++SoYIJQXL1+m35eF6id7+Vi0E/vnhJKrNhhC6Xsp+m5gCKVvyPQNYkUoucitemZsoUKT0OlBx62JXCvyi841qWkwPSAnlNjKwoNECl83LJkzydYSNPhgneBFJZRO2zCyHcgtrCt4AHMxaMwLafdQ8mdJS3N6NbdOBT/KCxcvW5lw1C0yTqB1V4pvJ6nHdV/FHFuu86lu+3NCyV+s3KqDbC/QguRSHuEsP3nt1KAQ9GVHKHkqP53WopdxRMMC5eU8sg4XVoaaAayU3EIZiy14BHuoLgny3LqxvzRmjfARQ4FfMiJW+/2+2s80tXVfa1+J7qxZyDq2cNthuiPoz6Y28Eoo/fg4ynV/Xfb0NPN5PaGE+wYExoeMC1goo3E9nFLNynmDUIIozN8aunMij0sdWi4H5DT3oePX2ropeLVQcqkw9RnldzHwDz4vFspYEcp1e49Tm7cDeckjIZQTV++lR4IfhioWIJTJkiVLIrXlJQDQTuyfXzOu2mCivP2uRP/1DaH0j5m2RawIJZduUE98Z42C9NrtlR0JJSc+5ftNoFPnL4pueEQzJ5TIJoOgnHs+CWRqaFQqV5IIPNVqiHp8axf/dhOQ5nN55sflIQ8UbOV/OWebqMJfaDrgVSuRjuzydrpczfK47oXICWXtotmtl5hqCeWEkkddPjJykeUPinnhJVyiT6I2XDjLT147bsGT/dgRSi6Urabh9DoGu2hSr+391pu6bh89+OUi0UxqrnJCGQ2LmDomnQ/fs61K0ePNSmqHj+1e5BRG+fyBGnT09AVbfzE/83/99kp0R41EQum0U4E0oPjIUgterNxC6eeDgBNKfBwidzwIGrcKYi3DbWXwuLXi1NGwGOuCotR53VwlP+05ejZJEgdZT+rWuhFK3AewdOLDbMRMvd+rV0LpVWTfyxrg96cXQukUcBSJhZIrjERCKJ0kr0AoU6VITqMW7QyBxstasiOq/Jpx1QZDKL2svsjqGEIZGX5W61gRSi4uqw5VPmycLJSl8mSkST0ai6YV+0+kE+cSxN9IjQiyiMIJ5bqX2hD8NmUmFR5xLc+vI5RqlLef7bUXfl4R8kBBZpKPgw94menH7jKpshRIjzivd6JFU22HaF+7XNq6Fy7Htl7xHNYLVfXt5ISSW0O6jVxsZfRA+kO8hIv1HhfRqpOEklvwZId4QCOXs1q4rqGahtPrYLxYnb325aUel8qRpI4TSj8Eycv5UEcXZcwDmtR+uHUL1+XchYvWNr3Xc+rqqXnludi1Wt+JUDZ7Y7qVxtSPD6xc99Cx7VyvKA0asybJMLGWUWT6SS/blG6Y6GSb1DbIOLXryJkkwYKynkwxyfUl1bXLo7F1fuKyL6+EkmvPus3R7Ti/P70QSifR9kgIJddAjoRQOkle4XmVLnUK+nZ+YEdKFjfZOdTTfVDjd37N+AeKIZRuKy/y44ZQRo6h6CFWhNJJ5gY345t3VXG0UHKrW+WBk4REA8o3D9WmBiVzir85oUR+avgJSgtl8zK5k0g6yMhxDh2PZsbvfkSOe49eSRCplQUyLVLGo0TujCJi3K6o4r3qFrna7tjpC7bC4joSzB/IDUrktAIB1K11Tih5P3w7H9leercrS0V7ORNKZAJZtO0IVS6UxbJu8nlIQskteBCAL5s3M335YE0r9RtvA9cIuBLI4vQRYoe1ny3TaNxW/GNKkjpOKN38dMMZg04HkQc0qX0OGbfW+vj5tkttkefeTkXAz3ggXt++ekGriZMYvVdC6WUbUV0fBbOlI9wz8kOUzwF+uZivJJReInPdMNAJy6ttQCi3Hz5tBcipx/EMgysN15dU14oXeSKVnDjdM9EklJwEefGhdAq2ioRQclHwSAil45Z3tQJCykvuSMlrGS1CyV0ojA+l290X+XFDKCPHUPQQK0LplHLtVhDKOyvbEhSVXEFfDw8rFLz8kGMbhRPKLUPaiW1dSShblM1jBZVIqLwQSqfMFCrk/X5fRV/P3W79DJ81WBdQkKlnsgOhVMV78QL8+4VmtlfV6UWik1XhD2S+/a8GJnFCyS0Gj327xIpglNle3Mic/MK2ywTxZeea1KR0buJf/tyKputf9cmzS1vmdDv4sTpH47biUjk6C2UsCK5OiPl/bcsQNF91ZfDYNfTJrIAPJaRjMqROaWU5kfWxJmFR81NwX99WLZFQOgWTORLKYdOtdKF+CJ9cQ0iMcPyMPaGEwDv8SFH86ILaYeHFHQPPPQQ/gWTrisxZzgml6q7hRZ7IH6GsRwjai0bhgUleLJROOdQjIZRcwzESQukY5V2tAGVLn9ryQ5b4efHH9WKh5FgaQhmN1enchyGUUcI4VoTSKbMLvtSH3VmFittsoarkqsbLU4TPEIrM7IC/eb5pkBkeUQp5HlUjTLetrFoo/UiUDPxzNX0xO+AzidKrbRkrcpRv2dtdqmK9xlJQDYl4VKquPt/qUo/rMonwBzLf/udyTOiHE0r+gOdyQl6E6PlL7IcFO+h/v65MMg1JKLnYtRuhVC1eVwOh5P7DOkIZC4KryyXdu10Z6tpITyhfHrOGPg0G5cCPLkfGNMJvkResye2HTvt60kDT8paqBaw2EExvNzwQIKEWpBuVcmD8GO7lZoxQOmUWgiwRPraalQlo03ohlMAlZfLk1na418xFTkBw3UC7eiCUm/afFGoUugK3nbSpUtCwSestCSfVPcKLPBG/FzkmunP++mg9qnZddAglDwzzQijn925OeTKn1WIRCaHk5D4SQmn3HMOAscuWK3OaJEFRXtwzvPhQ8utsCKWvR1BYlQ2hDAu2pI1iRSjtpBEwAjinw9eqpE2QhxrQUmvwFNp/IkAouRCvE6FsWyEvjV+1N2TCMvsO/1FNvej01ayix7cNcaz/jeVo4J8Bq0fpPJloYo9GjlepZJ9xVlCCm8+l04thzaDWlD51IAuKLPyBzLf/1eAfTigX9GlOuTMFHvDIiQ4BbBSZ7cWrhdIutZiM8p618QDd99kC0bcboVQJitM2lB3YftwYonFbzd18yLKU6whlLMajWrwxD/gKdmkYkFxSy6A/11iZcZBys0C2dElcDrAmYVHzU5Be8+YqiYSSR9yq/aRInkxkwlILCGXzYdNp84HAuXU6q/id++zJrUYvhBIffthalveq19zqTjhwq5hdPRBK6H0izaWurB3URvjlORFKL/JE6NurD2U05i7nwndRvBBK/rxR8YiEUHL3g0gIJdeTVccHQpk/a7okmYq8BNx5IZQ8t7ohlH6eQOHVNYQyPNyStIoVoXTaLoB8BgilnQwN0nHBd0iWukP/oj3Hzop/ciFeJ0IJcV41I4uOUEK3EhYzWfxk5eC5a9Eeaef6jF4luiqTNxNNeNqZUJbuO16I46K4+Vyiztdzt9GCrYeFNZNnyJDO/Pzi8gcyMl9IYWedPqbMlMNT3D31w1JLY02m8PNKKH9ctJOe/3lFkrUmCeXsTQcteSc3QgnRbQjGywJLbZ/fVlGujKlpuEchbifn/yjdRiHdcMksHaGM1XjU68Mll9R5cus6tnwRxFJl0OSQaliTsKj5KSqh5BG3XvtRCSWXtuF9cLF/udUuMYDbzImzCVofShDKNCmT04Dgx58dYfU6XtRzUmGQ/YCErN17gpCWVlfkhyFPqahqqB45dZ6qKik2dX15JZS/PVafqhTK6meqtnW524UXH0pIuOEDV1ciIZSHTp6j6i9PEd1GQijxrIGfPKyo7/y1MWSYuJbX5UhPb08J/d2L5i0nlFiLkNbCLlCNItmtc3DXBhOUE5Xl6diJIZRRwjhWhPKXxbvomZ/0aatAECAvUubFCdpZFMuVgaY+k0goIYQNjTMU/nJxIpQIEEGaP150gS9JCCWz0nmBmD/4MKfngkTKC6Es128CnQ7KIXmxaMrxcKdz/KbLdcvHxVNCqtv+3ELJt857jFpmZbSRKfy8Ekq7XLVSlJ5nT3EjlB90qEbtKuZzfek4XSs/bgxerrlbHa69qCOUfqzgbufix9XrA4t55/pFtV0M+GO1FVAQELJOlyTTC9bwur0n/AxBZN3hHwAgpC3enBHSh07Si1dQCaXMN60OhOdslpZsL4QSvqXpUqWg/n8EdDejIZ2j3pM60EBCYJ1cv0+PKaTPkG+dp1RULV5eosnVZ4LTfRuN7X45V+6W48VCiUxocMHRlUgIJSfdkRBKp3sL1xLvqTcmBXzmZfGi4MB9KEc+VIsalgyolvDCrb2GUPp6BIVV2RDKsGBL2ihWhPLHhTvp+V+SWqkwArxwEGxRtp+eUKrWugavTrWCA/jD34lQwgoqt2zlrHWEstPnC0L0Kp0ecm4PPlhnnvphmahWNl9mQnSqU6nQf6LIB4vih1Cq1hA3QtmuYl5LgF210nJCuWJAKxG5iNJz1DJLxkdmXPFKKO0+JqB3CF837uvqRiiRrrF1+byuLx0nnJ18taJ0G4V0wy1nOkLpxwruZ3zq9eEanmo//X9fRV8FA8oQSAC/5XL9JoZUK5cvM62xsabZjeu9e6vSDZUSLco8HZ9sY6c5Ko9Dj5H7V/NAPH5eWOvvHDFX/CQtoxIDrPOTNhbKLg2K0jfzt9PZC4HdAZ4txm5ekCQbv2oPdWtUnLJlSJ2kGpeqsesDJAT+kxttrL4rB7SiTGlThRBK1SfPST6Mn9erhVKXZcvPmlPrSvy9EEq+I2LXD37XPd+cxnhFCGW1AmJX6bUJ60OG4kXz1guh5KoNhlBGsiK9tTWE0htOrrViRSjtIn0xIFgPX21fKUlmDzlYlVw1fn2aFRzAt6fembKR3poS+ELEQ2fbwVPUJBhYwCV8ZL+I/EQUNy/3fTY/JKOO00NOByaXg/moY3Xq9k0g5R5exuNcCCXX1/Tz4FRTEOoeuNw39PpK+Uhua6ukmhNKaSHBWLhou8y44pVQ/rpkl1YkW4r+ciLgRig/7VSDWpQLBFw4vXScFnqsCJzdObn2oo5QOm31ud6wDhXU6zPo5vLUqW4RbYsXf1tFCHBCgVUFhLJ039APvIoFstgGkNgN4/17qxHWmyw6y12T0rlo+vpENxO1r+H3VBU+vLLIYC61Hv8wQRt8REoMYIk/dT5BbHu7Fe6XbVdX9gtr/4j7aiSpprPEqpVAshDhLX1D1ePI9oWEDjxHtxo17DVblFdCGY0Idz4PiZPd2uOBiEiXC11ct3vbL6E8c/6iZaxQ16PbWnC7/vI4riXeUzJ9p/zdiySYF0LJ3QcMoQz3qnlvZwild6wca8aKUHI/OXUAeOGAUMJCpyvqdjHPmsGjErEtgJzY8D2B1QMFqQ8h0gyHaZlPWZ7DC6Fc+mJLrQXCDkT4iCGFHbYt4JfV5etAhpTy+TPT2CedLZQ8KAdtvD44VYkSXTv4EX08awvBl/TTWVsta62KASeUMigAY3n2p+X08+JdYi6SFHkllL8t3U1PjwpYanmR4+RblW6E0o5MoF+38chzOzn/R+k2CumGay/qCKXfjxavY1Tx4KLwah99f1tJ38wLaKhCxgjrQs2EVLlQVluJG7sxqS4KOt9CN0KJ7ELDgvJbOM8nnWoQ/IDVwoOfpGVUYgCJMLiTeCGUXDnCbl6yXwTzQC9SLV58RUFClu44SrDa6sryfq0oS3pnQukl+Ed9ljjdJ34kmbysQ5AlbOtjqzlliuRJmnBCqVOnkA0i2fJGH8gitOPwaRFQiBSJkRYVQ1xL6OfKbEuyfy9JFLwQSq7aYAhlpFfPvf01SSi//PJL6ty5c5LZv/DCC/TKK69Yv48bN4769OlDa9eupYIFC1LPnj3p0UcfdUdNUyNWhBKn6vr1oiTSPfgdJGdo+4pUacAk7ZhVMsYjPr06keu0vrwQSqeHnBvAPFVXhQKZacwTzoRSfUh5JZSqiLJbO+4P6UQopQ4e5vn8z8vpx0UBQilJnxuBk+PQBWTdVaMQvXp7JdEf3xJ2I5RcyF7F3208sr5fNwa36+x2HClAb3zvb1FNRyj9frS4nU/3EsZvCBLrULuwtnmf0SutLB+IOse6UIXrq16X1VaE225MH3WsRm0qJFoodXI6uixWvD/ui4zf7axMXJ5JElm5JhB8duaCN0Jpt6XOxyT7TZsqOa17KSmhXL/3BLV+e6bjpQIJwdq3k2KSz55XJ6yjD6cH0mKqQtk7Dp2mRq9Pc10SXi2U0SaUbgODK4OM7JcEWtcmUkLpNg6/x3WEErtQUhxf9udFY9YLoeSqDSbK2+/V8l//miaUEyZMoCxZslioFChQgAoVCuTHnTt3LjVq1Ig6depEHTt2pNmzZ1P//v1pxIgR1KVLF99IxpJQ8vR9fGAglENurWib+QVbbX8+0cBq0vLNGZbfkVefn3AJpdx28g0kEfEMMOoc3B6aOO5GDKgXCiIAACAASURBVGUfqmO+W7uePy6jX5cE0hqqGp/cQsmjxf/3ywr6YWEgT63MuOJG4OQ44LvKtyzRx901C9Er7QOEkm8JuxFKp+1It/FIvGJlEbRbI1zMW0conV6k4aw72UbFQ80yxPvu9etKYc1HkUFCavuaRbLRwm1HfA0Jbh9tKiT6vOoEv5uVyU1T1+237RcWyYeDln5UeuuuynRr1USxdNmQC8hLIhsOobQLjOADlP1mSJ2CVg9qk2Tsn87akoRcqJXaVytIC7Ydop2H9WLxcguY51nHRwGyjj3dohRVKJBFWDdVvVAdkPFKKLmF0ulZezUQSjzjpfSUvAYzn2sqor+dihdCifYSA0MofT2Cwqp8TRPKAwcOUM6cgWwwamnbti0dPnyY5s+fbx3q2rUrjRkzRmS9SZ486TaDE8KxJJQ82wofA4JEBt9S0Vb+AlttiD6UpfVbM63ISK8+PzpCqctGwwN7cD4emOJ3ZfJ0k7EklKp0iBuhfO6n5fRTcPta1fjkhHLr0HbW9lCvX1fQ9wsChBLSFo80Lu66xSzHge2mJ5gPHPrghJJvCbsRSqfUcF4JZawsgnbrg4t56whlJGvMaU2qeLzaviLdVTMxbSVvy6+v9OlU29cuml1kn/JT1CAq1ZoOP0cIYP/lQCg/vq96iLuK3Ty4QD7atCqf11qjkKOBS4yXLW+ZEtQLtnYvdy9r8fbqBQnb9PCB1mlwyg8fTijlmFAfwUpefDXRxiuhHPtkAyqfP9F44edah1OXE0qn+yDeCGWbt2eGKB7AxeLQyfOWUoDEwkvSAu5X7JSZzRDKcFZYeG3+k4Ty3LlzlDlzZrH93aNHDwu5GTNmUJMmTWjRokVUvXp1X4jGklDybCt8UBAdf/mWCpZWmDpgbLWNfjSRUPKb2esWjVdCydM3Yhw8MMUXkEQhKQVjSSjVvN5uhPKFn1fQqEUBcqhm5OGEkvfD85RLgWy3l6ZsjwAg9MsLJ5R8S9iNUDrJmriNR54/EjcGv2sA9fn2p45QRrLGvJAeWUdNW8nb8jUhCbeKZ73iOWjO5kO+IFD9Hbk1Hb5gc3s3p6d/WEpT1tpbKEFKHxkZCG5DsQvw4Ck8ZfCWnAPkaBAp64VQSn1UL9hmTZ+KlvVrlaSql7UIQgnfcmjqwt9aatDKziSxHzp+bZIMLJIketlal3Vlv05jg2RUufyZfV3jSCpzQul0H8QboYRPOjKv4YMcaxofRt/M204v/h6QnpLFC6FEXXwMJU9GWskg9boZH8pIVpy3ttc0ocydOzcdPHiQChcuTA8//DA9//zzlCJFClqzZg2VL1+exo8fT23aJG67wKKJNiNHjhTb4H5KLAklF8fmY2pTPi+9fGsFQkpFXalROBv93L2edajdO7Ms+RKvX9Q6Qlkoezqa9XxolLdKKFcPbE0Z0oRmnfGKJ7eYxNKHUk2/5kYo+fZmkRzpafpzTa0p2RFK7mMnBbLdXppyHONW7qFHv7UnlHxL2I1QOn1AuI1HTjJWW8x264IHaOgIpS6zkdc15oX0yDrwRbyjRsBVRi3cR1ZailQ8G5bMGaKA4GWMalQ+t6ZDrqfvDeWoy1eLaMrafbbdcbUEVLLL+MN3BD67vwY1L5vHslAiZz0ImxdCKeWsvGCLqGRsTavFy1oEoZy54YDI+gW9SSkZJvuSwWNOhNIplSUfk1cLJaTNIHF2pQr3oXR61sYbodTh8+387VYiC3k8mlmwLAtlmpS0cmDrqF+iWL77oz7YGHd4TRLKiRMniq3s2rVri63HP/74gz788EPq3r07vffee8JfskGDBsKPsk6dOhbECQkJlCpVKnrnnXfoySefdIT++PHjhP9k2bNnD9WqVYt27twpAnyiWXgwCO+3Rdk8tOPwKdqwT5+Fo1aR7PRjt7pWk+uHz7LSlXl9AOoIpbrdixPc+8m8ECsMj3T2iwVeFtC1RIkloeQ5c/GVu2Xo9Y5D5eRQzUJkRyi5rIwUyHZ7acqXmC6PO7dQ8peiG6FUgxL4RN3GI+vGaovZDnS+LakjlDJns9/15VZfxUPNg87b8yh+SXDV9m7R2LrxSGInj3FruiSU8I+UmZt0fcAfsts3iR8kfI3w+jwITormyznkyJCazl/0RiidpKnk+bjlE0FeavGyFu+oXpCmrT8gLFywdB49fSGkG6mXOnTcWhoxc0uSc+D+4h9jTuvBK6GUUl5uaytax7mF0unD6moglPBBxsc6L05b2H4xNITSL2Lh178mCaUOjueee47eeustQfi2bNkiCOW8efME6ZRFEsrhw4fTE0884YjqgAEDaODAgUnqxIJQcv1IP5cavlujHkkklDe++7elh+f1AeiVUN7z8TyauyVxWy+Slz3PUe2FUHINTJDsT+9Pqm+nw41nUUifOgWt0QQJ8Hb9fl9FXwdFrNWc4Vzeib+EeBspkO320kwklHstPU45Dk4oeX5nN0LplHnCbTzy3FIw2s8ajKQuF7nWEUoeTR/JedxIjV0wC9pxnVE5HhVPngPe6zhVax+3podLKJ9qXpJ6tCyVZAggpTJ4B/JSkA6Tkep+CKX0v3Sao8RGzTSlEk6nPkAoEYx06NR5ggX14MnzIdVlcNSQcWvpYxtCyf2Po0EonT7YvF5zP/U4oXT6eL8aCOWohTvohV9CCWU006oaQulnZUVW9z9DKBcuXCgsiJAKwhZ4pFveV9JCWWnARDruQVhYXQrw3fru4UQL7E3v/U0rdh0T1VShX7tlpCOUqv8g2qqEMpKXPY869UIoj525QGiTMkUy8TJMmyqFp7vifMIlKw+6anHUdcDT7BXPlYH+YmktUX/FrqOUL0u6kLy6vM1LN5en++oW8RyUo8OeE8oN+05Qq7cCEituhNIpatIroYzEjcHTBVEqcYF9HaHcOLgtpdJo9IVzLt5GxUPNq83r8sh/BHsg6ENtz3PAex2b6o/IcxJLQmknJybP8WGHatSduUx0b1JcKA2ohedE/vrBWlS/RE4rww62piEO7WXLW41M181VYqOTHkN9L2sRhHLy2n3CMgmdzH3Hz4WcSqYIHTx2DX0ya2uSYeCDbcmOI3TbB3NcL4dXC6XX56nrCT1W4ITS6eP9aiCUuhSz0Uzzagilx0UVhWr/GUK5YMECYY0EoWzWrNlVFZRTqs94se3ktzQokZO+6ZJogb35/dmWwLKa2zZSQnn3x3Np3pbESFaIFkO8OJzCrX1eCGU450CbhIuXLBHqWkWzi9RxTmXQn2vo89mBF5Sa1tKu3cA/V9MXs7eJwwig6linsOtLU77EuOVI9n9PrUI09LaAbNCm/SeoxZveCKWXKEg3HGPls2h3Xq4VqCOUksC5jdvvcZXUyOwxun64O4qM7lfbw9d5wuq9voahCtHzrCWSUD4ychFNXG3vQwlNSe6D+1CDooJQqvcld62AXmntYtmpZJ/xYrx+CCUidjFXnRC3nLzERqcUgTpeCOWdNQrShFV7xUc2iCmivXmRa92JUPKkAE4XxiuhVFM7+rrYYVTmhNLp4/1qIJS6FLPSyhwGNEmaGEIZDRS99fGfIZTPPPOM8I2EA23evHkJskFHjx4VfpSydOvWTfhbxptskJeHrO5yNyqVi2BxkOWW92cTcumiTOnZWJAit6KzkqkBKehDJZRci9HtHOpxLrQcS0LJRW+RF/3de6o6DpW/oErmzkiTezZ2nRonoVLP0O16ypcY922TJ+KEcvOBk9R82AxxyM1C6ZQ20W088tyRuDG4AqWpwLOZ6Agll2cKp3+7NioeWBdYH7qCSOvflv0jDsnrpraHXuzYlXt8DVGV4OHuGZJQ2unTyhNByJyrBEBAesvBkyKNZO92Za3x8OCv77rUpupFslnpI7OlT0UJly57slCiQ1j64U6TJmXSXYJLly5Tsd7jxHl1uxz43ctaBKEcv3IvnTiXQIWyp0uiRykDOpwI5fwth+iuj+e5XpOrgVA6fbxfDYRy9NJd1GPU8pBrERNCmTYlrRxggnJcF30EFa5JQtm6dWtq3rw5VahQQUADkvjxxx/TU089JfwoUaSw+QMPPEAdOnQQgTr9+vWLS2FzLw9Z3RpAMMCXnRMJ5a0fzLYydkx9pjEVyxUeoSyWKwNNVbZ77xoxN0RrLxLr0ZzNB+neTwL6oLEklOgffqVr9xwXBLtIzgyOtxL3ySqVJyNN6uFOKF8es4Y+/Ttg1ZQ6gG7XU77EuMC7HBgnlFyc2Y1Qes336wTAlSaUPDuMjlC6ReWH+1xUr49THmNJ6nlWKrU9yCg0Rf0UfAjig1AWfPw0GzZDCHJLwtT9m8U0fpW95RMWw8e/S8zlzc/PsePyVCCU1QpnozIvBvKRI+gFGVm8bHnL/u1SPHIXE9UHWbZ1uzdQD4RyzIo9IiWkzkIpJWf4vafOnT9jnK6LV0LpdcfHzxpwqsstlE6uH/KD9rXbK9GdNkoF0RpTuP3oMoLJwKpw++TtjIUyGih66+OaJJQgjpAEgqXx0qVLVKpUKZH9BoE2PB8ptr979+4dknrxscce84acUiuW0gFeHrK6QSMY4LMHalqH2n84R6QsQ5n2bBNhTXArOgsl5DEQJc7LnSPm0gIm3hyJ9YjnFo41oYTG3ulzFz3lHedRo6XzZKKJPRq5wUfcSiIf6m7XU77EuD6gPBEnlFzY141Qes2m4TShSPxiXYHSVNhz7AzVHTpVHPk3CSV8EdtWTEyDqA718KnzBKFu6c+pXt9bquS3rJhecdClysRaxdZ31vSpRTePfruYxq20J5SwrKrC+PL8nCjxjEw4b/XC2ahsvwChzJIuFYHM+vHhtsOL+4HqfJBxPrd7A3WQfvT35bvp7IVLhKAhBOfwIv2FXxqzhj4Lfszx45g799N2uibeCaW3HR+v19+tHieUTrtBuHYHTp6j3JnSunX5rx3XJXBw2lHxO1BDKP0iFn79a5JQhg9H+C3jkVCqEc93fDTHSgHnJVcq0NARSjUDD+qphDIS69G8LYfo7uB2VKwJpZ8rznXtyuTNRBOedieUnIS+cUdlgoae20tTYjd9/X564IuFIUPkhHLn4dPU8LVAPmI3QulVWsQJj1gFwdidk6cb/DcJpZdgEz4H9foi97RM2el1vXnJi22XQUueA76faupOHaHkFiKkT6x2XTYq33+iqJo5bUq6TOTLQgnLaNsK+Qjrk1v9j5+9QJUGTLIg0O1iuN0bklCOXrabYPFMlyqFyDXOi3y2cXcTlVDq7i3dtfFKKJ1UFLxecz/1OKHcMqQdJYfu2VVadAkcpJZoNKZkCGU0UPTWhyGU3nByrRWPhLJ1+Tw04r5ECR1O+rzkSsWkueixBEHVt4w2oeT+TfFEKHkqN52VVrdIOAl9887KdFs174SSC7zLvjmh5D6GboTSq5+V00KPxC/W9QbSVIgXQulFDseJUCL39C9LdvmCAFvP9Uro08bKjtwIJaLTn/phmfa8nChxHzZstSPDVsUg8QuHUILIgrCBREtlAwwCltxqL022xnNv7etoyK0VQ8bnlVACT/h26orcfXEilDp3kkgIpVcXIl+LwKEyFzaPZDcoWuOJpJ8pa/ZRF5ZzHn3JbEeR9CvbGkIZDRS99WEIpTecXGvFklD+umQX9fwx1GnZdUBEhNSMH3ZMTCHJ/Ry9praCZMjN780WL4O9x8+K0yLzx8iHEqPHBaH8aC4t2JYY5R2JhRJb5yC/KPFKKBHgME7Z9tddk1cnrKMPp28Wh96+qwrdUrWAZwsl1+OUfXNCiejW+q8EtoTdCKWTFcPLSxznuNKWkHghlHY+gXb3oIrnrVUL0Oilu21vWRiXVG70/cN1qG7xHI63ORfT11X0Sih5lC2CgaoUykqVBwYsidjKh+3Lz5a3el75LNh//CzVGvJXyFDV54SXtQjprJ8W7xK+nboiyZ0TodQpKOj68mqh9OpC5OW57aUOt1BG8qz1cq5Y18E7psGrU0PknyB6j7Sf0SiGUEYDRW99GELpDSfXWrEklPtPnKVag0MfxK4DIqLrK+UjBBRYZISJj/vJRAA/HPgpyRSPqm8m+ufb6fh3JA85LukRT4SSk0MehOF0LV6fuI7enxYglFJ+xu2lKbHT+XlxQsmDVtwIpdP1cBuPnN+VtoTEC6FUs9a43XsqnshXDD9Fu5IqRTK6cDGUHP3QtQ7VKeZMKB//bokITrEr+IB5epS7hZLrAEKuCISyyqCAJRH5j5Ml80coYYnnH8By7fH1KsccDqHEPTBq4c4kJFz2KRUsuGQXxwjnhOxQt28S85zbYeiVUE5/tolrUJ/buvFz/FoilJg3V9zAvxf3bUE5DKH0syTiou5VRyjPnz9PqVMHnNLjqcSSUB44cY5qDtbn63bCQJXC4ekR/WYi4LmE21XMSx90SLR8Ygy3fziHFgUDfiIllIu3H6b2H8afhfK1Cevog6C10SvRfWPienpv2iZxmeBbdkOl/J4tlFw+SV7nf5NQRvKREM69Gi+EUqYj9DoHv7JBaVImF/myeRnVtQ7VdiGUCLhxih5Hhh9VjkVH5H5ctJOe/3mFOARB9SoFs1LV4NY0cmXDgurHQglfYaSkVM/FdUX5MWQBWrjtMNUrntOKLnfCGvfADwt30mW9gZJ+6V6Xjp9JoL/W7aNv5u1I0hXWMZdKcjqXV0Lp1Sfd6xpyq3etEUrMl983TqoUbtiox42F0i9i4dePW0I5cuRIoRMpUyCuWrWKbr31Vtq6datIm/jjjz9S7ty5w595lFvGI6G8uUp+eufuRG3FDp/Oo9mbAukR/ep88VzCiFp9m/UbfUJ5hBCRjuLVEhjly6ntjlsbKxbIQn8+0cD1tG9OWk/DpwYIpZSfcbMIypeYTtrkv0QoD508R9VfDnxIvXRLBbqvTmGxLqRSQawIrnp9VJFxt4uutm9VLg9NWmMvQA61BcgB8QKRfYjtOxUE3DhZPlVLIe+LY8dT3yHlY+WCWS3cwyGUUDOQBBXnlOfiuqmcUErfbmxlgyi6FdwD3y9wr2fXD8ajiyzW1TeE0u1qRO84v2+WvtjSk/KGl7MbQukFpejUiVtCWalSJXrkkUdIyvg0bdqU9u/fT927d6d3332XGjZsSJ9++ml0UIhCL7EklAdPnrO2m/0M9baqBejNu6pYTXjOa786XzxCE7Idr94eyNYiC3/R85eIn/HKujwtWjwRSm5trFQwC/3xuAdCOXkDDf9ro5jaiPuqU+vyeT1bKHm0u8Tmv0QoMWekrtxy8BR92qmGyPDybxBKVRPSbU2rhNItlzeiyD//e2uID/LP3epSjSLOhPKpH5bS70FRdd2Yht1RmZ5hlkI7Qvnd/B3Ue3QglzK29ysVzGrtiGRInUKkk/RjoXzltor0v18D/cG6uWXo9eJvnipUjgWEze0DS53bPbWuo+8XBCyPA28qT/3/WO12SUKO45w67UNdJ14JpdcgR18Ddah8rVsol/VracljRYqZIZSRIui9fdwSysyZM9Pvv/9OIJIHDx4U2W3GjBlDbdq0oVGjRtGzzz5LO3eG/5XqHSJvNWNJKLmlxttoArUQXTrszspWE04o/coynDyXQBWCUiKwFMFixMttH8ymJTsCWXhQIrEeLd1xhG4N5tmNV0Kpk07SXZs3GaGUW6duL1CJnS6bx3+NUKqY/huEUqcJ6XQfqtcXCQamrz9g2+SX7vWE9iPXTcS2bfXCzoSSZ+nRda5uPfM6/P4cOW87vfjbKnEYxB0fSzJ4Jn3qFJTSJ6EcdHN56vd7gORhO3/9y23F36v/OUbXD/87ZKjhEcpECyUsqqt3H6dhkzd4fjTinF4DHb0SSq9Bjp4H6VLxWiSURXuNtdwYlvdrRVnSp4oKXIZQRgVGT53ELaHMmjWrII7IevPTTz9Rp06dxBZ4mjRpaObMmeL3M2dCc7h6mnGMKsWSUKpyG16nAM1DvFRkuf/zBQQpGhS/sgynziVY2nTICfziDeVChhFNQon0kEgTiRJPhHLYpPX0bnD7GoELvz1W3/VS8G1ySUy8Ekr4ld3xUWJqUJzMEMorv+XtRROSLwT1+kIVYdbGg7Zr5ffH6hM+UHhmF0kynRaYG6GEVuq6vSe0XXCi9OXsrTTgzzWi3iONi1HalCnonaBVHTqPsAwfO3PBda3LCn2vL0svj10r/omgnpUDA+nulu88SjcH72tZN1JCiY80ENU3JvkjlNxv1GlisNieOn+REFjldN8aQul5edhW5CTZKRGD3zMZQukXsfDrxy2hbNy4MYFUDhkyhJBjGxbLsWPHipl+88031LdvX9q2bVv4M49yy1gSSh4Q42fYSFH22u2JhLLzFwtoWtBS4jeKjucSljmp+ViiSSj5iyeeCCX3h4RW3+hH3Qkl16GUkbteCSWPdpdYc0LJ1wWkWm6uUkBU0/UfjSjvSKzOftatU91+v6+ir+duF1ViNR4Vv+8eri0CRrwWtX3dYjlo7paA77KujH2yAZXPnyUkq9LoR+tR1euyOZ6yx6hljnJETo05dsgmA+uorqRNlZxSp0jua8u7R4tS9NaUAMFDJpvFL7YUf8P3VfpGy3NFTCg716TVu/0Tyh8W7LC25b1c1z8fb0A3vhdqXeXtZDpML31Fo861aKHkc1oxoBVlTmsslNFYK1eyj7gllMitfcMNN9Dx48cpU6ZMNGXKFKpRIyDS3b59e0qRIoUIzImXEktCefT0eUvGw898VV/Hh75cSH+t2y+6CMfp+as52wiO9f1vLC/8qnjhecLxeyQv+xW7jtJN78WfhZJvX1e7Liv96oFQ8tSL0uqkI3xPNitB2w+fpocbFqMKBbLYvoDhPwZCLwvIADKS9LuhnJUtg/cPK3WLsrmpTQX71IFuBJe//P2sv1jURUQw9AWR5xoqBrEoKh5eJHz4ONT2SATANVrVMU/q0YhK5ckUQihh/YYV3Kn0HLWMfnXQt3Rqy+/PT2ZuocHjAhZFtYRDKB+sX5Q+nx3IX583c1qa17u5+FtncQ+HUPLgHUSlr/nnOL0+cb3npYBzfjt/O/UZHdjm91Jev70SPReMhNfV9yPD5uV8bnWudUK5ckArymQIpdsyiLvjcUsogdSJEydow4YNVLx4cWGtlAU5uEuUKCFydMdLiUdCya1ZwKnLVwtpytoAoYym0zP6wxY1tqpliYRQrtx1zLIGxJWFkvlDwucNBNGtIKjkyzkBSzqsHBULZtFaED/oUI3aKfmieXCSPI9KKHXn54TGy3W4mgilG97ROK7i4SXimp9XbY+1IiPTdeOTQtxDxq2lj2duEVXkNrjTfHr+uMx3Skfd/fnRjM2ELFC6Ah9I/OcnKAcfMT8vDmQGKpQ9Hc16vpn4W+cTHCmhRAT+6jAI5ci52+jFoJ+nlzXj5I+K9n5l2Lyc06nOtU4oVw1sTVAYiEYxW97RQNFbH3FNKL1NIT5qxZJQcskeP7NVyUeXrxbRlLUB+ZJo+qigv2gSylW7j9EN7wa2l+KJUL41eYPlW1azSDb6qZs7oUSwA4IeUMY/1ZCQslFH4D7sUI3aKoSS+5LK624IpZ87ILy66vXxEnHNz6S2r1wwCy3fdcx2MDJCmOd9/+Px+iLa2qk88+Ny3ykdZX8DbixHD9QvKv75/rRNthY+kMm0qVL48qHkMknFc2Wgv55pIs6jk8EKh1Bi52XUokBAJjL74Hnh10LJ/Ua9rBKniHm0n9urGeXLks5LV1Gpcy0SSp5OcvXA1pTBEMqorJUr2UlcE8rVq1fTSy+9RAsXLiQQtrlz51K1atWoT58+QouybdtA9GA8lJgSyjMXrFRofuaq5srt+vUiSw8vmj4qGBOc7eH7KIsXy5jdXOKVUL49ZQO9PSUgAaTLZ66bT69fV1oSJzKDh45QftSxWpJtaV0QgyGUfu6A8Oqq18dLxDU/E9JhIi2mLA83LEqfzApsAeuKtG5xQjnmiQaW64NdO4iHS0tgODOV5333r422UdIIyEnr00LJfUZL58lEE3s0EsObvekgdfh0fshQwyGU8A3/cVHAAgpJp5VhEEonv1Edlm6E0q+ubzjXi7e51gnlmkGtKX3q6Foo1SDVSK+BbB/Ld3+0xnil+olbQjl58mS6/vrrBYFs3rw5DR06lBYtWiT+PWDAAFq8eDH9+eefVwon1/PEclFxDUjXgbAKHWpfR4NvTfS3e2TkIpq4OmChjOaWAvq7+b2/Q6wwkRBK+ES1Gz5LjDOeLJTvTNloBRtAdBpboW7l+Z+XWy8/aYnSE8rq1KZC3pDu+Na/PGAIpRvikR9fv/cEtX57ptXRr4/Wo2ouATL8rLuOnBZEr2C29ATJr1bl81LTN6bbDkxKePEAritBKKXFnH8oqYNEWkhEevvZ8sa9sWDrYdFVqTwZaVKPxuJvXW76SAnlyIcChPK1Cf58KD+dtcWKRPeyYpxE4tHeEEovKDrX4RbKtYPaULrUKSLvFO+63cdo+vr9dH+9IlHzywy533ftokKFCgkZw4IFC0ZlzFdrJ3FLKBGAU65cOfr6668pISFBpFuUhBL6lBA8B4mLlxJLQolAhIoDJvmeasc619HLtyQSyu7fLKbxq/aKfqK5pRBtQrl2z3Fq+058E8raRbPTKA+Ekvu5SYuQjlBK0XN+kbml1hBK38s/ogY3vDuLVu0+LvrwEiDjdDIETTV8bZptFZlmDn6M8GdEkZHfTv1GaqGUhJKrF+gIJSxFfmSDahTOZqVhLZE7I8Eyj4KX+gNfLAw5RaSEElJcIJSvTtD7gOrwwzn9EkqnNJY4h99EEREtTiK61i2U0SSUkWLt1j6W7363c8fb8bgllGnTphUWyJYtW9LFixcpVapUFqGcMWOG0KE8e/Zs3OAZy0UVLqHsVLcwDbo5UYD8sW+X0NiVewRm0b5hb3rvb1rB/MQisVCu23uc2rwdf4QSGW8Q6Y1Sp1h2+qGru4WSZzORwNHKSgAAIABJREFU2p86QvnxfdWFJYsXnRC0sVBemVueE0ov/oxOo4LFssGr9oRSup+AFH0YzBUvyV4sCeWEpxtSmbyZiWulqueDqDl82fwQSiggyCQHxXJloKlBH8pp6/ZT5y+jSyihEYrnTqwJZbfGxS2yr7smC3o3p9yZ016ZxXmNEsoSvcdRwqVAgvZ1L7URvrtXQ4nlu/9qmD8fY9wSyvz584tt7vvvvz8Jofzkk0+EPiXyesdLieWi4llq/Mz3/rqFaSAnlN8tobErAoQy2jcsz7aB/iMhlHzLMZ62vLmvGfzEvu9ax/VyPMYwl9kfWrw5gzbtPxnS9pNONahluTwhv/Gtf3nAEEpXyKNS4frhs0T0MIqX7Wenk8KfEn6VdkV+3HFCKcleLAnlxKcbUem8mUSEt7SMqueDPBiibf0QSsgdScUH5Cmf9mwgKOevtfvooa8WhZwiHAvlHdUL0k/BKHIQSlgo7aLUdfjhnE5SSeEsIL+Zx8I5B29zLVooDaGMdFX8++3jllA++uijBHmgiRMnCokgWCjhN5kvXz6C6PmNN95Ib7zxxr+PYHAEsSSUPEuNnwk/UK8IDbipvNXkie+X0p/L/xH/Xv9yG0qTMnpfgBcvXSb4wMgSCaHkOX/L5ctM455q6GfaMav73tSNVkaO+iVy0Ldd3Allt5GLacLqUDcDZD6atHpviLAyUt61UAglt9QaQhmzy6rtuN07s2jNngCh9LL97DS6f46eoXpBQgldxv43lqPu3y6xmmwc3JZSpUhOr01YRx8ELZSS7MWSUEr9Sx4MpJ4vHAslj2ovnCM9zXiuqegWa77ryMVRJZTfwUIZBqH8eOZmGjLO+za52+rzm3nMrT+349c6oYz2+8kNz0iOx/LdH8m4/o22cUsojx07Ri1atKAVK1ZQxYoVacmSJVS5cmXasmULlS5dmqZOnUoZM2b8NzDTnjOWi+r0+QQq12+i77l2rl9EiJDL8uT3S+mPIKGULzHfnTo08Kt/aNfVxn0nqOVbgaCIeCWUDUrkpG+61HaFj0s1bXi5rUhjh6JanZHirXnZUAulIZSu8MasAnx44cuLMu7JhlQuf+awz7X32FmqM/Qv0V7KTfF7ZevQdpQsWbKQrWdJ9q4EoeT6l9EglBUKZLb8T7kO5YRVe6nbN5ETSq5ziSxGCF4baqOjqcMPH7sjZmz21cbt4i/q24JyZkzjVi1qx691QsmflVEDLUYdxfLdH6Mhx6zbuCWUmPGFCxdEmkVEfB88eJCyZ88uSCbyeiNIJ55KLBfVmfMXqWy/Cb6nq+bc5rl/Nw1uSylTBMhNtEq0COWm/SeoxZvxRyi5Xh/yM498yJ1Q8nSXkjjoCOXnD9SgZmXMlne01mKk/bR5e6aVB9vL9rPT+fYdP0u1hwQIpVQH0N0rb0xcT+9N2yTqTenZiErkzuQ4jUiDcib3aEQl82QKySEeDUKJj0Bp3S2QNR0hiwzK+JV7Qiyz+C2cLe/21Qpa+pvfP1yHkFnLD6HEBwIizv20cVtPflPZuvXndvxaJ5SxMHi4YRru8Vi++8Md07/VLq4J5b8FSjjnjeWi4nm0/YytS4Oi1PeGclYTnvt3y5B2Vqo+P3061Y0eoTxJ8DNEiScLZTiE8r7P5tOsjQfFXLgbgOoi8MUDNalpmdwh8JqgnGitTP/9cELpxVrodIb9x89SrSChlOoA7oSyMSFC2qlEImzOSStSWcpUier5IBuEoJyjpy94BrFM3kwWGc+XJS3N7RVIvThmxT/0+HdLrX6SJSPCc6hor0RXGS8n4YQSaTFBKP1sX2MbHr7Ifvwu3cYlI/Xd6kXr+LVIKEv2GUcXLgaCcmJh8IgW9mo/sXz3x2rMseo37gklUi8uWLCA9uzZQwjUgZwQtrzjrcRyUYVLKCGo3Of6RELJJWy4tSxaWEaLUCJfePNh8UcoP5i+ydK7Qy5piCq7lXs/mUdzNh9KQijxwxezt9LAP9eIY8hJ3LR0KKE0skFu6MbueOu3ZtL6fSfECbxYCx0J5YmzVGtwwEIp1QF098qwSevp3akBC+VfzzSm4rmuDKEc+Odq+mJ2ID2oWlKnSC70AP0E5UB7csO+QNBZnsxpaH7vFuJvuNvA7UYWBPwgOLBkn/G+LuRt1QpYKSdHCUJ5zDYXua5j+IU+06q0r8hwtwEufbElZctw5XbNrkVCWarPeDp/8ZKAevOQdoT1cTWUWL77r4b58zHGLaE8efIkde3alX788Ue6dOmSCMrBFnjy5MnpjjvuIER6/1d8KMMllF0bFaPe7cpa15tbNCIJmrFb5NEilFsOnKRmcU4oG5fKJdK+uZU7R8y1RJ5VzH9bupueHrVMdIGcxE0UQmmEzd3Qjd3xVm/NsEiRF3LnNJKDJ89RjZeniCr1iueg7x6uE5J+U64Lrgcp83s79RtJLm/0KzM38Xzz0SCUSLe4+cAp0VWuTGkIASsofL3j3yB2Kwe09u3Owwklkgsgo9TgcWt9LYbn25T2JYbu1rkhlG4IuR/nhDIWO2juIwivhiGUibjFLaHs3Lkz/fLLL/T222/TbbfdRlmzZqWjR4+K33r06EHt27enL774IrwVEINWsVxU5xIuUum+/n0oH2lUjHoxQvncT8stuY14JpRbD56yMovE05Y3NAKl3l3T0rnoi87uhPKOj+bQwm1HxIpzIpQgpyCpvJjUizG4UT122fLNGbQxKO3khdw5dYtsOdWDhFIGc+k+vjihnP5sEyqSM4PjaCMllJIo9/t9FX09N5BvXkco06dJ4WvLu1jODLTlYIBQIlAFASsovy7ZRT1/XG6dAtbPxS+28J20QSWU2PJ+eaw/Qvlc69K+8n+7LZtl/VpS1vTGQumGk9NxTihjsYMWydic2sby3R+rMceq37gllJkzZ6ZXX32VunfvnmTuH3zwAfXq1YsQCR4vJZaL6nzCJSrV19+2EHCBGO//2paxIIJ8CXwTKxbI4inLi19so2Wh3H7oFDV+PZCqLp4IJbT6pN9VszK56fMHarpCBC3JWz6YTS3L5qH3O1QLqc8tNkgh17BkKKGElt8t788OaRMLHcqv5myjAX+upsG3VKTeo1fazikWHyGuAP5LFbhWqBdy5zRMyERVe2myqCKDubSEcvIGgng+yoznmlDhHFeGUL742yqCjqyugPT5JZRFcqSnbYdOi+5yZEhNi19sKf5GOkoEEskCxYP5vZpT1SA2Xi/1bVUL0K9Ld4vqP3ULWCj/bUIpNWa9ziHSetfklnff8YR3HcrV9KyJ5bs/0nVypdvHLaHMnTu3SLvYpk2bJJhMmDCBOnbsKCK/46XEclFduHjJt58RcOnepDi90CaRUOI3aFoiN2/yGPinRItQ7jh0mhq9HsgsEk+EkkuNNC+Tmz7zQCgxB0gEZUidQkjD8MIJJVLINSiZM+T4kh1H6LYP5oT8FgtCiRMgG1OmtKlCtmLVe+tqeshH+lxoPmy6tW0rc7CH2+eRU+ct0iRdJXT3yluTN9A7QULp5ZyRWiilb2if0Svp2/k7tNMD6Uuf2p+F8rrs6WnH4QChzJY+FS3t10r8/ePCnfT8Lyus86RJmZxmvdDU8i/1iu+tVQvQ6CCh/LlbXSGi/q8Tyv6tKEu6VF6nEHE9QygjhjBqHcTy3R+1QV6hjuKWUL7wwgu0bt06+u2330JexJcvX6abb75ZBOa8/vrrnmCCP2aZMmVo9+7dtHDhQhHYIwvE0/v06UNr164Vid179uxJEFX3W2K5qBIuXqISPh3XMf7Hmhan51qHEkq/8/JTP1qEkuc+jldC2aJsbvr0fncLpRN+nFAi40f9EqGEcvH2I9T+w1BCeW/t62jIrYn52XX9R3IddGkh5Tn+S4Sy2bDptCXoBzjr+aZUKHt6P7dCSN2jp89TlUEBC2WT0rnoy861tD6Ub0/ZQG9PCVgovZwzUkIpZYN6/bqSvl8QPUJZMFs62nXkjJhH5rQpacWA1gQpsGnrDoT4OuLDFtvuUvTdK8CcUP7SvS4t3eGfUD7bqpSVpMDreZ3qyfSZ0ejLSx+GUHpB6crUieW7/8rMIHpniStC+eabb1ozS0hIoOHDhxNyeiMrDiyW+/fvF/m9z507R08++SQ999xznpAAOf3qq69o3759IYRy7ty51KhRI6FrCYvn7NmzqX///jRixAjq0qWLp75lpVguqnAJ5eNNS9Czra9cRHwkRIaDHa+EkmfXaFE2D316f+KHia/FEqzMCSUyftRTCKXOh9ILoSzaayxdDqhv+N46MoQygBsnlNBRhJ5iuAUR0pUHThLNpe+t7l7hhPLvF5pSwWzOJLbnqGXW1m84Y5NySL1+XUHfL9ip7QJWRER5+5ENyp8lLf1z7KzoL1OalPTJ/TXo7o/nJekflk9kBGr4mn2ec92gQgllPWGhfGlMQC3Ba3mmZSkaNnmD1+qu9VYOaCUs/FeqGEJ5pZB2P08s3/3uZ4+vGnFFKBHB7bVg+/DixYuu1WHlhEVy2LBh1K1btxBC2bZtWzp8+DDNnz/f6geR5WPGjCEsEj/jieWiUjULXScdrPBEsxJCHuNKlWgRyl1HTlODV+Nvy5vn/21VLg993CmKhPLh2lSveKiFEtb4+z5bIESi4YeH4oVQLtx2WMizPN6sBHWoXdjX5TeEMgBXszemW4Elc3s1o3xZwieUx89eoEoDAoRSukro7pV3pmykt6YESI4XEhspoZSC7S/8vIJGLYoeoUR6yb3HA4QSrh51i+ekKWv3JVmHyBGOPOlN3gj4S3stt1TJT78tC6SQ/fXResJC6ZdQ9mxZit6MIqFcNbC1yHl+pcq1SChL9x1P54wP5ZVaQjE5T1wRyljMsFWrVlSpUiW64YYbqGnTphahhJUTgT+vvPKKiBqXZcaMGdSkSRNatGgRVa9e3fOQYkkoL126TMVYnmx1UCAZ32l8oJ5sVoJ6XoWEcvfRM1Q/mPu4bL7MND5Ocnl/OmuL5avVunweGnFf9AglMn7ULZ5Du954kJIXQul50WoqGkIZAKXpG9MJagMo83o1p7xZ0oYNK/xTKwYJpbRs6wglAnIkyZnzv2aU38UqGimhxH2F+4urP6iTDMdCmTtTGtp/4pzoCtvacOXQEUpYL/94ooGl6OAVYE4oRwcJ5f/bOwvwqI4tjv8LBHd39+CEJEBwp9AihQpSrDgUaXEJToFSpFgLLS2UlmKFBwQPENzd3YI7Ifj7zmzu5q5l7+7eu3s3OfN9fO81O3fu3P/M3fntmTnnjHTQQqk2UJ4cUUcEgHdXYaB0l9L276Pl2m//7vqqEauBcunSpejWrRvOnz8vcoHLgfLUqVPw9fVFSEiIiePPvXv3xPb6ggULxDa40qLlpCJLVUzZJFoE5LR6qP7bGgXQu1ZBpY/gcj21LJTkjS6dq9IrUNb1zYzZrZT/4LAmrnzLmzJ+BOa1DpRyJyUGSpenqaIGqk4MNXoq7xtUAxlTOg+U8rzttYpmwq+t/ayeoZQDpRKIlWe+UvRQZpWkHOUxpXB0Bijlt6HrKQnAxlOWFko6X/lft4rGmLNKn8EcKGnLW0oQoLSN3jULGq3BSq+JqR4DpesqsoXSdQ093YKugfLBgweYMWMGwsLCxNY05fKmM4/kNJMunfXFVxI0IiJCOOIEBwejXbt22Lp1qwlQ0nnJoKAg0DnKwMBA4zjQ2U0Koj516lRxTtNWefr0KeifVCiTj7+/P65fvy6ce9QuMVmOWgbmxMI9lofqe9UsgF41vQ8ow5+8RPlxW4SEegXKesUyY1ZL9YCSMn4E2ABK+ZlSBkq13yzr7VWZGIqrUaFvKDA3Beh2tkS8fouiw9aLyyXLtrUfXz9vOW90FNk7qAYy2YFYV4FyTc8g+GZNhZicexL7xENiH8fOUMp1orBD5Ii0wQpQpk7qg2VdKhizYinV99NSWbEyasubgPTwtUcOAyV9N0oOUErvG1O9UyPrIGlCtlC6oiUDpSvq6eNa3QLlxYsXBTxSaKAKFSogc+bMuH37Nnbt2oX06dNj+/btyJcvn00VBw0ahI0bN4rzkXQW0hZQ7tmzBwEBARZASQ5BPXr0sNk+geqIESMsPnc3UPasUQAUlsRaHDn6Ff5tzQJum2lqWShvP4lE4DhDqjo9AeW8HZeNZ7XqF8+MmS3UA0rK+OGfJ63VsdIDUA6sVxidqth+39w2ydx0o0oTtuD6Q4On8sEhNZEuufNA+fL1O2M2GMmybe1dkeeKd4eFks4vFsuWCjGBqatASenzKCLC+pOWFkoKKbSkcwURG9eR8knJrCKNI5WVUUAZHJXCVGk7tHsjhWhSek1M9U6PrCucl9xVeMvbXUrbv4+Wu5P2766vGroFykaNGoHyeFPMyZw5cxpVI2Cj2JQFCxbEihUrrKp59epV4+cEo1R27NghvMVDQ0OFk861a9dc2vLWi4WS4tXN23EJf1jJdEGejD1qeB9Q3nkaiYCx+gPK33ZchnRW6+PiWSwClTv6asu3vClAc7nc1oFS7qTkKQslZTuhrCdxpciB0tW0evLUqdIPET0A5f+6B6F49lTo9c9ho5OL+fjSGchEPvEc8vKWt0GhV+sUzYx1J29bTJ20yRLi306BqDl5u0PTSg6Uq7pXxKGrj+BpoKSc5GTJdVeJjUBZeGgIIt9wYHN3zSEt7qNboEyVKhXmzZuHzz77zOK5lyxZIsL62MqUI1kjbQlGFklyvvEWpxx6Dltb3hSvjixn83ddsXhcirXWvbr3AeXdp5Hw1yFQ/r7zsnFrrUGJLPj5K9PMN46+oHKgpADNfjaAUu6k5CmgdNVK56g2nq4f9MMWYyxFV9PqyYHy4xJZMOOrMlbPUMotlEo8y2MCQSX6EYwVypzCqbSuStqX6pBV1hpQUhYdOjtc6yfHgLJhyaz4X5SFkqCYEgAMX3XSkS6BdnakrEQOXWijMgOl6yoyULquoadb0C1QpkiRQjjGkKXSvJBlkmJHPnv2zKp+lPP7yJEjJp/Rf5M39+zZs1GuXDmUKVMGFDaI6tI5SqlQaKFVq1bpKmxQTEBJ4UUonI01oKR8td2q5XfbHFNry/vus0hj9gw9bXnLgZIWtelflnZJW6VAKXdS8hRQHhpaC2RRiiuFogwQyFM56mIWFHnqVOmHiLV3ZebWC5iw7qy4pxIvb1eBks4fbj93T9XwOdbmB503Djlh3UJJQFnbQaAkDVcfCxe3om17SgDgKFB2rZoPM7deVG06nx1dF4kSsIXSFUEZKF1RTx/X6hYoaVubPK7pHCQ540jl0aNHqFWrFjJkyCA8tJUW8zOUdJ0U2LxNmzZo0aKFCGw+bNgw3QU2jwkoyZLx6/bL+G3nZQspvBUo7z9/Bb/Rm8Tz6Ako5++8bNxao223aW4CSrmTkqeA0tVtX6XvqV7qyYHS1aDV8tSp0ryxBpSztl7ED+vOCAmUxKF0FSgp5M64tWew78pDTWW3BZR0hvKfjuVRZ4pjFkqy8q6RASVZKIetdMxC2bFyXvyy/ZJqz31udD1Qmkp3ldi45V1k6Dq8fGOILe1NWbn4DGX0rNctUJ44cQKVKlUCeV1Xr15dOOVQppvNmzcjYcKEwimHwv4oLdaAkq6l1IvkwCNPvUihhhwtWk8qW1ve5A1KX4y07W1eKI835fN2V1HLQvng+SuU1SFQ/rHritESQp6mU79wj4VS7qTkKaB0ddvXXXNQrftUGLfZmO3F1ZAw8kxXFPJmyhelrW55z952EeND3AeUFBR8vBuAks6Nrj1uaaEkL2+yUNadEubQsNH55TXHoy2U5OU91EGgbFsxN37faXlMyKGOyCqfH1MPPvEZKJ3Vj64rOmwdIl4zULqioaev1S1QkjAEaZSOkRxqyDJJlkoK9UNb11qE5nFlMDwFlBTSZM62i5hrBSgH1CuMzm70zFULKCkrTJlRhtzHerJQ/rn7itESIoGBK3NG6Za33EnJU0B5dFhtpErqvtRyruiqxrVyoHTVg1ee6YrSBv70eSm7QKkk9eK3/xw2hs9x5pkpZM8PIdpbKG0BZaokBqCsN9UxoJRbPCn0ETnlOAqUrQJzWY2M4YyOdM2FMfWQgIHSWfnEdQyULsmni4t1CZSRkZEiT3erVq1EbEdvKJ4CSnKWIMvGr2GWFspB9QujY2Xvs1BSGKTSOgdKCQxcmZtKgVLupOQxoHTxHKErOnni2vLjNiM8Kh+1qw4X8kxXTUpnw2QbQLnn0gNjzmsl2+yuAiU5gk1Yfxb7Lmu75S23KMrHkgKb05Z3/WmOASXF8pTCEFFw9oNkofzvhEPT5Ev/HDbzlzvUUFRlBkpnVDO9hoHSdQ093YIugZJESZ48OdasWYMqVap4WiNF9/cUUNLZNgLKOVbOAw2uXwTfVM6rqP9qVFLLQvk44jVKjdSfhXLB7itGS0iTMtkwuXkpl2RTDJQyJyVPAeWx4NpImTjuWCgDx2425qN21eFCnumqaZns+LF5SasWSppMyw/dEOGZKLuMveIqUFKoqknrz2Kvh4AyhQDKQHw8bYfJo9rLYlO7aCZjoHRKH3mALJQOAuVnZbNj6cEb9iRW/PnFsfVBMTfdVWLjGUoGSnfNHu3uo1ugpBzcNWvWRL9+/bR7ehVb9hRQ0lbkzG0XMGeb5QHzIR8XQYdK3geUTyLeoOTIDWJ09LTlTcHjpYVLAgNXppBSoLz37BXKjTE4KXkKKE+MqIPkbsxV7IqualwbMHYT7jw15KNW43yc9GOrWdnsmNjMNlA60veefx82Bvh25DqpLgXT/3GD54CS5hMBZYPppkD5Y7OS6LvkqM1HonzoUm7wdb0q4cCVRxjiIFDKs+04o535NZfG1kc8BkqXpPQdtg4v+AylSxp6+mLdAiVlsKFc2pRmkcL7UH7tjyhKrqzIvb89LaSngJIsRzNDLworpXkZ2qAo2gflcZs0alkon7x8g5IjDEBZOHMKrOtV2W3PENONFu65aly4yMIxqVlJl/qlFCjlXu+eAkpXHVNcEsoDFxPAE8hTUcP6JL0bn/vlwA+flbBpoXTkUV0FSoK5KZvOYc8lbbe8bT1TsoTxsbhTeQugnNy8JPr8axsoaxTOiM1n7opm1/eqjP1XHjoMlLa24R3RX1738rj6FuuTs20puS42WigZKJWMvL7r6BYoKV2iVMxBUvr7u3cGjzA9FE8BJVmOZoZesBpTzd1AOeJ/J4XnpKtWxaeRb1AiWN9AKVmaXJl7SoFS7vWuNVB++cse7L70wOKx3J2r2BVd1biWwlYRyFNRAxYkoFzYPgBBBdLrAij//iYQUzd7DiiTElB2LI+GP5taKKd8Xgq9FpvGEZaPafXCGbElCig39DYA5eAVjp2hlG+bqzFf3B3m5qeN50TqyKypEmPXwBpqPILH22Cg9PgQuNwB3QLl/Pnz7f7i+/rrr10WQK0GPAWUtNBTho0ZoZYWyuENi6JtRfdZKCmA884L91EmVxqQB6ez5VnkGxTXIVD+tfeqceFq7pcdEz5zj4VS7vWuNVA+f/UWbX7bJ86lyYurns7OzgVPXScHSjVggfKxU6D0wLzpxCOpYc131UK56JsATN98weoPCHfoTmkdF3cKxCc/7zS5HcV3pWezVaoVyoDQs/fEx84CpRxK1XhWNeaII/2gUFRhF+6jVPbUSBNLEg4wUDoyA/RZV7dAqU+5bPfKU0BJHqg/b7mAn0MvWHQuuGFRtHEjUKo1ZgQ1xYavF83pact70d5rGLTiuOjXF+VyYHzTEi49slILpdzrXWugpAeSpwCUHtBVT2eXhPLAxX6jN+L+89fizlrAgi6AskOA+N7YddHSIu0OyRP7xBMWyk9nmALlz1+VRvdFtoGySsEM2HbOAJQbhYXykfG9VNrvSgXSI+z8faXV7dbTYo7YvWksq8BA6f0DqnugpHzdx48fR3h4OLJkyYLixYuD8nzrrXgKKClDw/Qt5zF9iyVQjvzUF63L59abVHb78+LVW/jqECj/3ncNA5cbgJLCjoxr4h6glHu9uwMo5SkA4ypQlh21EQ9e6Bsoe/x92JjT2u5LZaUCbb/TWHsKKBMliCfOUDYyA8qZLcqg61+HbD6SHAY39TEApfReKtWhfN50qlpmGSiVKm+7HhkRyJig1Y8413tovQWt136t+q1Fu7oFyvfv32PIkCGYPn06Xrx4YXz2ZMmSoXv37hg9ejTix3df7lR74ms9qWxlyqH4Z9O2XMC0zectujjqU1+08kKgfPn6HYoMWyeeR08WSlOgzIlxTYrbmxYxfq7UQin3encHUMpTAEoP4GroHJeE8sDFFFifjhpotbipYaF0FSgXtPcXznw7L3jGQkmpChd3DETjmbtMRnh2yzLovFApUFYRZygdBcpyudMIEJUXCvtDQeidKQyUzqhmeg0DpesaeroF3QJl3759BUwOGDAATZo0EV7ed+/exbJlyzBhwgQBlT/++KOn9TPe31NASeEq6HA2/TMvoxsVQ8vAXLrRSGlHIt+8Q+Gh+gPKf/Zdw4AoC6UaYKcYKGVe72rc1944yFMASnXdnavYXh+1/rz0yA14FPFG10DZfdEhrI7Kae2MHn+28xdpW3dcUG/r15F+JIwfD/90CkQTM6Cc06osOi04aLOpivnTGSF4c98q2H/5ofG9VHr/UjlS48j1xybVKbe4NOZK25HqMVA6qphlfQZK1zX0dAu6Bcp06dKJGJT9+/e30GjcuHGYNGkSHjzwzC9ra4PmKaAkD1SCySmbLIFyTONiaBHAQKnWS7Z4/zX0X2bY8m4RkBNjGrvHQin3encHUFIqz3FROaUl7dSIxajWOLijnVIjN+BxLAfK+W3LYW7YZY8BpU/8jzCzRVl88+cBkyH9tbWfxd/kFeTb1Vv6VhGZfqQfekrnRrFsKXHi5lOT6rnSJcXVBxFKmzCpx0DplGwmFzFQuq6hp1uePmmbAAAgAElEQVTQLVCmTp0aS5YsQa1atSw02rBhA5o3b47Hj01/YXpSTE8BJX2R0Xb35I3nLB5/fJPi+MI/pydlcerer96+Q6Eh+rNQ/rv/OvotOyaeqWVgToxupB5QLutSHmVzpbWql9zr3R1A+cv2ixi79oxJX9ydWs6piaPiRXEBKH9vWw6/7bisqnOKkiGgcMIfYthZ/q2NH9rNN4VMebuBedMaY2eGflcV+y4/MP7QU3J/qlMoUwqcvfPMFGisQKbS9hgolSplu17x4evxjM9Qui6kB1vQLVB26NABlLJs3rx5FvK0a9dO/O23337zoHSmt/YEUFL4jN/b+pt45VJ+3BxpkwrryqY+VZAkoX7OmSodLAo/VHBIiKjuakxLpfdUUk8OlK3L58LIT4spucxmHfmWd0xAKfd6dwdQzg27hNFrTpv0W43g3i6J5eaLKbA+BdinogUsqHGG0tUt79/blMNvO90PlAnifYS3MZxVpH61nb/f5ogH5ElrTBfpLFDmzZAMl+5Fn82nm1XIl85pByUt5oibp7zHb8dA6fEhcLkDugXKP//8E4MHD0b+/PnRqFEj4xnKFStW4OLFixgzZozI9y0VOmfpyeIJoJQCLsvPvFEe4H2DauDdhw/wiR8dHN6T2jh67zfv3qPAYB0C5YHr6LfUYKH8unwujFARKJd2Lg+/3NYtlHoASjWCezs6DzxZv0TwejyN1M7jVA9ASZZASkSgZvgcJWNGzjj0o9FWoa34Nr/bBkr/3Gmx74ohu89WYaF8aNw5UHJ/qpMzbVJce2i6vV3XNzPWnbyttAmTegyUTslmclHx4PV4puE753oPrbeg9dqvVb+1aFe3QCnPlGPvwSmTjqez5mg9qax5eUtfYnKLUuaUibFnkHdnTqCgvfl1CJRLDlzH91FA2aZCbgR/4mtvasb4uVKnHHkYJXdYKOftuIxRq0/F6QVTa2uJHoCye7X8OHrjsduBkjLkRETlbLb2gpD3eat5+2y+O3IP7W3fV8XeS44DJWWYufUk0uQelP1qycEbTr3TDJROycZA6bpsumpBt0B59epVh4TKlcuzzieeBMo/dl3B8FUnhV7Z0yTBjv7VHdJOb5UpdEe+QWtFt/S05b304A18t8SQY9idQBnx+i2KDjMEemegdM9s1dpBQA2g7LboENa44OXtHiUt75IicQKjJcpaH/7qEIAWc/fa7F7ZXGlwMCqT0/bvq2HP5QfGnQOlz5QhRSJjrnbpGnqn5++6orSJOP2DyymR7Fyk9a6AFn2mNrVe+7Xqtxbt6hYotXhYLdvUelLFZKFcuOcqhvxnyGWbJ30y0Lkiby7v339A3iig1FMcSjlQtq2YG8MbusdCKY/L6Q6gnLzhrIhtKhU1tve9bT5qnbVDFaD86xDWHA/3NmlhLzwPpYT86lfbQFk6Z2ocvmZwyAzrVw17Lj0w7hwoFcO8DzTHE8SPB7LOO1PYQumMaqbXMFC6rqGnW2CgVGkEPAmU8nA2BTImx8Y+VVR6Ks80Q85YeQYaLJR6AsplB2+gb5SFsl3FPBjWsKhLAind8pbH5VQjXJG9TvdbehT/HjBs/e0cUB3ZUiexd0ms+5yB0vqQtgrMBf88aUFB1Z0t1qyD8rb+6RiIL37ZY7N5eQxJZ4EyeaIExqwsdCNyOhsfchq/hjFQOjuurl7HQOmqgp6/noFSpTHwJFDKQUdPAOaKtJIFR0/Ps/zQDfT517Dl3SEoD4Y0iJ1A2fq3fdgelSs5rsWflOZs0WHrjOf8tLA+eauFkoCyfL50MaZGtPfeZ0mVGOFm5xfl1/zbqTyaz9lts5mS2VPh6I0n4vMd/clC+dB4FMXevaXPKY945JtoxyByOhsfcgZztl9S2oRJPS3miFMd8eKLGCi9ePCius5AqdIYehIoVx65iW//OSKepHi2VPhfjyCVnspzzegRKFccvoHeiw1A+U2lPBj8sXuAUh6X0x0WSkpjR2kmqcTVhbLI0HV4+eadZhp4K1BSuKwK+dKj80LbmWzsfWvkSJsE1x++tFmNQmg1nWUbKEtkT4VjLgKleegimufjQk5jzjYGSnvjp9XnDJRaKeu+dhkoVdLak0BJB/PpgD4VOl+0omtFlZ7Kc83oHSg7Vs6LQfWLuCSQ0i1veVxOdwDl/eevMPS/E6jtmwmNS2d36Rm99eLCQ0OMFiwtoNqbgbJSgQwxZrKxN+Z0zvvyfdMYkPJrlnetYJGOUf65PMsNHcnYc/GB8SiKvXvb+pzG+Id1Z0B57J0pWswRZ/rhzddoHftVK220Xvu16rcW7eoKKNOmTYtNmzahTJkyoODlQ4cORZ48ebR4btXb1HpSxeSUs/7kbWPuWwqpsaRzBdWfz90N6hEo5QDYqXJeDHQTUMrjcroDKN091nq8X6EhIXgVFStRC1jwZqCsWihDjJls7I1n/ozJceHuc5vV/utWEY1m7LT5uW/WlDh5y5A2cdeA6titElBOXH8GM0IZKO2Nn1afM1Bqpaz72tUVUCZOnBhbtmxBhQoVQHEo9+zZA39/f/ep4cKdPAmUm0/fQfs/DKnKKM/t3x0DXXgSfVyqR6CUHy3oVCUvBtZzj4VSHpeTgdI985MyNUnBtxkoozUnb+iqhTOibQyBx+2NkLW0h/JrVnWviE9+tg2UFErsdLgBKHcPNACldLbZ3r1tfU5jPGn9WfwcGh3dwJG2tJgjjtw/NtRloPT+UdQVUJYsWRI5c+ZE06ZNhYVy2LBhyJs3r02VW7durZsR8CRQbjt3D1//ZggETOnDFn3DQKnFxJADZZeq+dC/bmGXbnP42iM0nrlLtEHx9HKmS2q1PXlcTgZKlyRXfHHBwSF4/c7gtKEFLKhhoez610GsPe5cZhfFQphVpFiNVQpmiDE1or225UBore7qHkFoMH2HzWbIUe/MbUMe7j0Da2D3pfvGs8327h0TUJqHy3KkLS3miCP3jw11GSi9fxR1BZQbN27E119/jdu3b4Oy31D4GFtFD9lx5H3zJFDuunAfX0UFAmag1O6llANl16r50M9FoKSe/r7zMihzyOflctrsuDwuJwOlduMrb7nA4LV4887w/aMFLKgBlF0WHkTICfcDZbncaY1ntp0ZDfmWtbXr1/QMwsfTbAOl3MK5d1AN7LqoElBuPIdpm88780iazBGnOuLFF5UauQGPI95o9s5pJY3Wa79W/daiXV0BpfSAjx8/Bp2nXL9+vThPaaukS5fO6kd03dixY3Hq1Ck8ffoU2bJlE/nAhw8fjlSpUhmvWbt2rcgXfvr0aWTPnh19+vRB165dndJZ60kV0xnKvZce4POouG0MlE4Nn6KLVh29hZ5R8fe6VcuH7+u4ZqFUdFNA/LCS4nIyUCpVzbV6+Qetxdv3DJTmKpKFkoBQSkFKn1Oc0puPbXttm7ch99K2Nkoh31ZCvalhNgeQYu2ejzqDuU8A5QP0WmyIcuFsoR8NUzadw5RNDJTOaujqdQyUriro+et1CZQkyx9//IEGDRrAFjTGJN3ff/+NY8eOifOXadKkwYkTJxAcHCzgdMOGDeLS3bt3o3LlyqBt85YtW2Lnzp0COOfMmYMOHTo4PDKeBEpKQ9Z0lmHrlIHS4aFTfMH/jt4yBnSmPMjf1Smk+FpXK0o/KBgoXVVS2fWU+pOOGlBhC2W0ZgSUeTMkw7CVhlSvVCjd641HyoGyZI7UOHrdkOnGWlnfqzLqTNlu83O5U4+aQEnWyckbzymbIGa1tJgjTnXEiy9ioPTiwYvqum6BUpL2ypUrAvYePnworJZBQUFwJm/3r7/+io4dO+LmzZvImjUr6tWrJ9rcuzc6xRd9vnr1apGbk5yCHCmeBMpjNx4bD7FXzJ8Of3XgM5SOjJ3SuquP3UL3RYYMIT2q50ff2u4HyjmtyqKOb2alXeZ6TiqQd+AaRPEkA6VMQwLKxqWz4VOZF7ajQClPnWhteDb2roxaP9kGynwZkuHiPUPYoX2DawinHCkOr5PDLcZ4+ubz+JGB0lkJXb6OgdJlCT3egG6B8t27d+jcuTN+//13vH8fndGAQI8cdmbPnu0Q9C1fvlw4+xCgZs6cGSlTpsT48ePRu3dv4yBs27YNVatWxYEDB1C2bFmHBseTQHny1hPjmaMahTNiXptyDvVdj5X16OUtj/fZs3p+9HEjUB6/8QSXH7xAwxJZxPliLtoq4O1AOaFpCfRbdkx1kQgogz/xhfwIjr1A5ead8MuVBgeuPrLZt819q6DGj9tsfp43fTJciopjuX9wTXGGUg2gnBF6ARPXn3VKM7ZQOiWbyUWlR27AIz5D6bqQHmxBt0BJMSgnTJiAESNG4MsvvxQQSM46//zzj9ia7t+/v/gspkJQ+ubNG3GWkiCUPMhXrVol/tvX1xchISGoW7eusYl79+4hY8aMWLBggdgGd6R4EijP3XmG2lG/6D8ungUzWtg+d+rIM3myrt6B8tsaBdC7VkFPSsT31lCBPAPXQPIJ1AIWtHbKoT5bO3ftqmQSUMpTU+ZMmxTXHkYobppyge+7/NBm/dDvqqLapK02P8+dLimuPDDc78CQmth5QR2gnLn1AiasY6BUPJAqVywzaiMevngtWtXinVO5u8bmtF77teq3Fu3qFihpW7tLly4YMGCAxXOTZXHWrFm4evVqjJqQow1tcVMhcFy6dCmSJUsmttBp65zOUQYGRm8Pv337Fj4+Ppg6dSp69uwZY9vk7EP/pBIeHi7ObF6/fl04+KhdYnLKuXTvOapH/aJvWiY7fmxeUu3bu709PQLl2uPhxhzGvWoWQK+aDJRunxhuuqEawBdTV9VoPyYvb62Asm3F3Bje0Be+w9bhxWtDaspc6ZLiahTgKRmewLxpRf5tW2Xb91VRZaJtoJTf7+CQmtjhAFAmjB/PGA5Kfn/Si7LkULYcZ4o3AZAzz+eOaxgo3aGytvfQLVBSkPP//e9/qFWrloUCFF6oYcOGiIyMjFEdcsx5/vw5Tp48iVGjRiF//vygaylgOgEl/W9AQICxDQkop02bhh49esTYNjn5WLOQegIorz2IQOWJoaK/LQNzYnSj4trOGje0rkegDDkeji5/GVJc9q5ZEN/WLOAGJfgWnlBADeDTGig7LziIdSethw3SCijbVcyDYQ2Lotjw9Xj+6q14RLnFUMlYkeMgeWbbKmH9qqHSBMP3mbUi32I/NLSWAEop+oK9+1OIrogoEJbXJb3mbLuIcSEMlPY01OrzP3dfEc5eZPHe3q+aVrdRvV22UEZLqlugLFq0KGrWrAmCO/NC1kNK0Uhb10rLwYMH4efnhyVLloDadnXLW08WyluPX6LC+C1Cig5BeTCkQVGlsui2nh6Bct2JcHReaADKPrUKomcNBkrdTiAXO+YNQNlpwQGsP3nH6pNqDZTFh6/HsyigtJeb27yDlQqkR9j5+zZHaEf/agj6wRQoyfOb8svvu/LQJEzR4Sig7BEVzsvesKdK4oMnLw2xDs2B8pftFzF2rTKgND83yhZKe8rb/5zi7R669ggFM6dAysQ+9i/QSQ0GSi8Ayl9++UU45Xz++edo3ry5OEN5584d/Pvvv1i8eLFwyvnmm28UTyk6T5koUSKMHj1aOOLEJqecu08j4T92s9DC3eFsFA+AgxX1DpR9axVEDwZKB0fVe6ozUFofK8lCWTx4PZ5FGiyUjgJl5YIZsP3cPZuTgfJzSz+QpUoEbL0XH8GKwzddAsr0yRPi/nPDOT1zoJwbdgmj15xWNEnrF89skqWIgVKRbLGyEgOlFwAldXH69Oliq/r+/fvGzDkZMmQQKRm7devm0OTcsWMHKlWqJGCUAJXCBlEAdTpHKRUCWHLa8bawQeFPXqL8OIOF8vs6hdCtWn6HtNFjZX0C5W10XnhQyPVd7YLoXp0tlHqcO2r0iYEyZqCUp8mTe10r0b5aoQwIPWsbKCmdYuA4ww9ka0CZNVVi3HpiOO50ZJhhy1sK52Xv/llSJUZ41LXm7c/bcRmjVivb9fq4RBZQ1Ad5/+zdmz+PnQowUHoJUFI3KWTQmTNn8OjRIxGHslChQnbDBTVp0kRsb5coUQJJkiTB0aNHhcd4pkyZsH//fiRMmNAY2LxNmzZo0aKFcNQhUPXGwOYUgLnm5G0gsAzrVx0ZUiTy+jdXj0C5/uRtdFpgAMrYAu5eP1E0egDJAeuzstkxqZn6Tm7Bq05i/q4rom26hzPFE1ve7YPyYGiDopDHDKRA55ei4kIqeQ4Kbbb5zF2bVW0BZZ/FR7D88E1kTpkYt58agPLosNoIu3BPMVDaCnFEFsbfdlzGSIVA2bBkVlCiAwZKJSMeu+swUHoRUDozFckLnCyRFy9eFECaO3duEGR+9913YqtbKpR6cdCgQSapFx21fEptaT2pYvLypj5EvnmHV2/fg84IxYaiR6DccPI2OkYBZb+6hdC1qvdbgmPDXNHqGe49ewXaItUi7iel07z3/BUypkjsdPedAcopn5dyKU2hBJRyj1x5oHElD1OraCZsPGX97CddT/m5A6KO8MiBrc+/R7D80E1kTJEId5+9MgDl8NrYcf6+4tzitqypBJTzd15G8P+UWSg/LZUVK48wUCoZ79heR+u135v0061TjjeJSH3VelLZA0pv08tef/UOlP3rFkaXqvnsPQZ/zgpopoAzQDm7ZRmjY5kzHZOAsuyojXgQFTNQngpRSZt1fDPZdCai6ymdonQm3BpQ0g4MwT6VY8G1EXZOOVAWypQCZ+88s+gmAaXkZazkGShbEJ3nlPdPyXVcJ/YpoPXa702KMVCqNFpaTyoGSpUGyoVmyKryzZ8HRAsD6hVG5yoMlC7IyZe6qEDHPw9ggw1Lny0v79ktyxrPATtzeymKhN/ojUbnFkeB0tyhxbwflE7Rf4zlGcq+/x7FskM3hNVYcqw5TkB5/r4xPqy9Z/LNmhInb0XHD5YD4YLdVzBUlqM8praalMkmrKUMlPYUj/2fa732e5OCDJQqjZbWk4qBUqWBcqGZTafuoEMUUA6sVxidGChdUJMvdVUBZ4CScsFL54Cduf83lfJg8MdF4Td6E+4/N1gJC2ZKjnN3niturkGJLFgtc2gxv5DSKZYbs8nkzwTI3y05iqUHbyBtsoTGjConRtRB2Ll7xviw9jpB51WpDfNC7S/ccxVD/jthrwnxuXk77OWtSLZYWUnrtd+bRGOgVGm0tJ5UDJQqDZQLzWw+fQft/zBYKAfXL4JvKud1oTW+lBVwTQGylts6i2jLQvlLq7LGc8DO3F0CSgI+advZ1jayrfY/KZkVq2QOLUqB8vslR7Hk4A2kSepjzPl8ckQdEYJISjhg75lsPT/ptWjvNQxacdxeE+Lz5n7Z8e+BaDBloFQkW6yspPXa702iMVCqNFpaT6q4CpS0WK3vXVmlUXKtGTlQDvm4CDpUYqB0TVG+2hUFnAHKX1v7GY9tOHNvCSj9x2wyOsYUzpwCZ25bnku01X6jUlnxn8yhxbwe5ecmC6i8ELBJQJk6qQ8eRxiCkxNQhp2/p/hc6ML2AWg5b69F16j9v/ddw8DlyoDyi3I58M/+68Z2GCidmU2x4xqt135vUknXQEmpESmzDaUzNE+zSJ6XK1eu1I3WWk8qBkrPD/WWM3fQbr7BQslA6fnxiOs9cAYo57b2Mx7bcEY/CSgDxm7CnaeGLW9HgbJJ6Wwi/I+tQvm5y1oByn5LjwqrYMrECfA0Kqj6qZEGC6WUwcreMy3tXB6fzY6OPSzVJyBcvP8a+i9TBpRf+ucUACq/3t69+fPYqYDWa783qaZboJw6darIaJMxY0bky5dPxI40L6GhtvO9unsQtJ5UDJTuHlHL+4WeuYu28/czUHp+KLgHgLA0OrrlPe9rP+OxDWdElIAycOxmYyxIR4HS1jlGqT+Un5vCEskLAV//pcew+MB1pEicwJil5/TIutgmgNIQH9ZeWdalAprO2mVRjdr/d/919Ft2zF4T4vOvAnKKLXIGSkVyxepKWq/93iSeboEyR44caNy4MaZMmWI3kLkeBNd6UjFQen6UQ8/eRdvfDUBJwZ0phAoXVsBTCjgDlL+18TNa2eX9VupY07FyXgyqXwQVxm02ZqspkiUlTodbek7b0uVzvxwCDG0Ve0CZPFECPI/KI35mVF1sPascKP/tVB7N51i3UC45cB3fL1UGlC0Dc2LhHgZKT819Pd1X67VfT89qry+6Bcp06dKJvN01atSw9wy6+FzrScVA6flhZqD0/BhwD6IV6PDHAWw6bT1AuC2nnN/blDNa2V0Byorjt+Dm45eiCUeB0ny72HxMDw+thdJWLJQDlh0T5xaTJoyPiNfvxGUElGShVOq5/leHALSYa/0M5bKDN9B3yVFFU6x1+Vz4c/dVY10+Q6lItlhZSeu135tE0y1QdunSBcmSJcOkSZO8Qk+tJ1VcBUqK9za5eSldzIGtZ++iDVsodTEW3AnAKaBsW85oZZdrqDSWZKfKeTGwfhHIgbJolpQ45YCFktKWTlx/1uYQUn7uUiMtt7wHLj+Gv/ddRxKf+Hj5xgCUZ0cbLJRKgfJ3G89PQLj80A30+VcZULapkFukzpQKA2XcfSO1Xvu9SVndAuWrV6/Qvn17kTqxZs2aSJ06tYWulE5RL0XrSRXXgHLdidvC+kLOL6mTWp6f9cS4kyXk69/2iVsPa1AU7XjL2xPDwPeMUsAZoJzftpzxR5FcyDzpk+Hy/Rd2tZWAMuiHLbjxyGChtBUs3Lyxr8vnQoL48UAWypqTt9m8F+XnLjlyg8nnBGwSUCZKEE+kmaVybnQ90A89KSWqvQeIKWzQf4dvKk5L2bZibvy+k4HSnt5x4XOt135v0lC3QHn48GFxhvLatehzKnJhycv73TvDr1Q9FK0nVVwDSj2MqXkfyJu0dRRQDm9YFG0r8hlKPY5TXOlThz/2Y9Ppu1Yf19aW9x/t/I0/iuQXZkudxLiFHZN+narkxcB6RVBpwhZcf2gAymLZUuLEzZjPUP7YrCSals0u6l+69xzVf4wBKIfXRskR1oDyuPCsTpggHl7LgJJ+6EkZrOyN/YyvyljN+016rTxyE9/+c8ReE+Jzyhg0d8dlY122UCqSLVZW0nrt9ybRdAuUZcqUwdu3bzFu3DgULFjQqpd3rly5dKO11pOKgdLzQ81A6fkx4B5EK6AmUGZMkcgYV1IJUFaeEIprDyMUA+VPn5dE49IGoCRLaLVJW23e5qgNoKSg4+RZ7RP/I7x590Fcf34MWSiVAyX1o/diy21tAkIKtt7z78OKphg5J/2y/RIDpSK1Ynclrdd+b1JPt0CZNGlSLF++HHXr1vUKPbWeVAyUnp8GFEC51TzDljdbKD0/HnG9B84A5Z/t/I1Wdrl+8uwzSoCyysRQXH2gHCinfF4KjUpnE01fuf8CVWMAymPBtVEi2NJCKQFlgngf4e17A1BeiAJKKSWqvTnxX7eKaDRjp0U1AsrVx26h+yJlQEmW2jnbGCjt6R0XPtd67fcmDXULlIGBgejatStat27tFXpqPakYKD0/DXacv2/MssFA6fnxiOs9cAYoF7T3N/4okuuXIlECPIsKxSP9PXPKxMZYk9LfpC3vqhNDccUBoJz6RSl8WsoAlNceRKDyRNsxhI8H10ZxK0A5eMVx/LX3GuJ9BETxJC6OrQ+KD6sEKCl25NjGxbHuRLhFIHQCyjXHwq1uh1ubZ12q5sOsrRfZQhnXX0IAWq/93iSxboHy0KFDwilnwoQJqFatGhIkSKBrXbWeVAyUnh/+nRfuG0OOBDcsijZ8htLzgxKHe9B+/n5sPuPYGUpbqQflji6SpObBu+nvnavkw4B6hVF90lZcinLiUXKGcvqXpdGwZFbR9PWHEag0wTZQnhhRB8WGrzcZWQK+If8dN4n9SBUuEVCevasoWPvqHkEoli0Vzt5+hjpTtlu0H3I8XHFO8G7V8mFGKANlHH79jI+u9drvTRrrFihTpEiBN2/eiH/x4sVDkiRJTHQlp5wnT57oRmutJxUDpeeHWg6UIz7xxdcVcnu+U9yDOKuAmkApt/pJgpoH76a/k2Wuf93CqP7jVly6Z/AKVwKU5AzzcYksqgPl5XH1seWMMqAM+baSiJl57s4z1P7JEigpsoTSjDs9qufH9C0X2EIZZ9++6AfXeu33Jol1C5TBwcEgaIypDB8+XDdaaz2pGCg9P9S7LtzHV1FBkRkoPT8ecb0HMYW5seXlbctCaU1L8+DdcqCs8eNWXHQAKGe2KIP6xQ1AeeNRBIJ+sG2hpPzcRYdZWiiH/ncCC/ZEBxOntug5t5y5YzX7j/kzbehdGQUzpcD5O89QywpQrj95W3E8y541CmDa5vMMlHH9JeQtb1ND34cPHwynm7m4pAADpUvyecXFuy7ex1e/GrJsMFB6xZDF6k6+f/8BISdu4/L951h+6KZxC1oCLWs/Qh0BSkotOk8WGkcOlBRH8sLd50JfJXEoZ7csg7rFDEBJGXYoMLqtQtlvCg9dZ/IxgeOwlSdMstM4CpSb+lSBFMDdPA4mtU950ZWGH+pVswCmbGKgjNUvmMKH03rtV9gNXVTTrYVSF+o40AmtJxVbKB0YDI2q7r74AF/+uke0PvJTX7Quz1veGknNzTqowOdzdmPv5YfGq2xZKG2lHrR2OzovOXtb9DlBqtO1aj70q1sYtSZvw3kHgHJOq7Ko45tZ3ObW45eoEANQUvabQkOUAyU55bSdv9/kEVIkToBnkW9N/ralbxXkzZAcF+89Rw2zOJik1+bTdxSdxaRG+9QqiMkbz5no7eCQcfVYooDWa783yaRboKxevbpdHbdssf0r1+7FKlfQelIxUKo8YE40t+fSA3zxCwOlE9LxJRoroAVQmp8TlANl7Z+24dwd5RbKX1v7oVbRTEKF8CcvUX6c7e9uCgWUf3CIiWIEfMNXnsAfsvzZdCLq8riPhWFnZaQAACAASURBVFNO26iUqNJFCePHw+t3hmw6Utn2fVXkSpfMamB1R7bOqb3vahfEpA0MlBpPa69oXuu13ytEiOqkboGyQYMGFmcoHz58iGPHjok0jKVLl8aqVat0o7XWk4qB0vNDLQfKUZ/6ohVbKD0/KNwDoYAjQEnhbnZcuC/SmpIjyoGrj9Czen5MkzmZUJvmVjg5UNb5aTvO3nkm7q3EKWduaz/UjALKyDfv4D9mE56aWRCloSTP7byD1loAZfCqkyb5s8mR6JINoBzXpDgGLj9u0kZYv2rIkTap1TiYBJTWwDRdsoR48OK1xSwzz0fOmXLi7ouo9drvTcrqFihtiXj37l188skn6Nu3L5o1a6YbrbWeVAyUnh/qvZce4PMoCyUDpefHg3sQrYAjQOmXO404/1g0S0qRE5u8tdMk87GwGpI39w/rzpjILG15152yHWduG4BSyRnK39r4oXphg4WSyt1nkfAfs9nqEFrbrqe/mQNl/HgfiTiUlMu7jZmFkry/T4U/FV7bUorIXQOqI2vqJLj64AWqTDTN1EPtW2uHnInIy6DbokMmfTXXhoEy7r6NWq/93qSs1wElibtixQoMGDAAZ8+e1Y3WWk8qBkrPD/W+yw/RfM5u0REGSs+PB/fAOaCsmD+9hXR3n0bCf6wp4JEFc/Sa0yZ1Kf7i93UKw1Gg/L1tOVQrlNGkLWvfaVRBKVBSxpwLNoBSAjz5Wc+9g2ogU8rEVgOrU315alWpo7RVXyhTCotA7APrFca4kGjYZqCMu2+j1mu/NynrlUC5bNkytG3bFk+fPtWN1lpPKgZKzw/1/isP0Wx2FFA2KoZWgfrJJe95dbgHnlSAfujQDx6pxOSUYw0o7z17hXJjNpk8AkUyGL7qpMnfulfLj+/qFEL9qWHCAkhFiYXyj3b+qFIwg0tAacvLe9u5e/j6N0NKVPnz0/+Xn/XcN7gGMqZIbDWwOuklz4QltTPvaz8Rasg8EPug+oUxdi0DpSfnvF7urfXar5fnVNIP3QIlZcoxL69fv8bp06cxYsQI5M+fH5s2mX4BKnlgrepoPakYKLUaOeXtMlAq14prulcBa0A5I/QCJm04i3rFMmPt8duiQ+TlbQ0o7z9/Bb/Rpt+nlKaQ8mfLiwSUH08Lw8lbyoGScohXtgOUPvE/QuPS2TDhs5Iw/74j4Gs1by/Czt+3AEdzoKSA7KMbFRf15Gc9Dw6piXTJE1nEwexXtxC6Vs0PeeIC6Sa0VU9AaR43k1KvjvjfKQuAde+o8930oIDWa78enlFpH3QLlJQdxzywuRQyMyAgAIsWLUKePHmUPqfm9bSeVAyUmg+h3RscuPIQn7GF0q5OXMH9ClgDSurFs8g3ApQ6LzT8QF/UIQAVrGx5P3zxGmVGbTTp+ITPSqDf0mNWgbLB9DCcuKkcKCn+ZVAB06128+80SrmYLGF88b1vDShb/7ZPbEvLi/lWNQFv39oFjWuHfGv+yLBaSJ00oUkcTDpHuvbbSqJJeZxZ6R60VU9AaR4384emxdF/WTRs85a3++e8Xu6o9dqvl+dU0g/dAuW2bdss+p84cWJkz54d2bJli/HZlixZgr/++gsHDx4EeYbny5cPXbp0QadOnUQaR6msXbsWgwcPFlZPardPnz7o2rWrEt0s6mg9qRgonRoWVS86ePUhms4ybHmPblQMLXnLW1V9uTHnFbAFlNTiuhPhdoHyccRrlBppCpRTPi+FXouPmHSKQgn1rV0IDafvwPGbhtS3Sra8rYGsNWiUbmbtM3PHI6pLIBd2/h5azTNseVOecYqfKZV6U8NwOmpr/lhwbaRM7GMStqhc7jRY0rmCqC6P4iBdT1v1BTImt4ibOfWLUvj2n2htGCidn7vefqXWa7836aNboHRFxMDAQOTKlQuNGzdGpkyZEBoainHjxqFXr16YOHGiaHr37t2oXLkyWrdujZYtW2Lnzp2gVI5z5sxBhw4dHL691pOKgdLhIVH9goNXH6HprF2i3TGNi6FFAJ+hVF1kbtApBVwFyicv36DkiA0m96b82+bezRJQfvLzDhy74QBQfhOACvlitlDKocwaUH46YyeOXn9s0kdzoCRnmU4yoJSf9TxJFtBECXD7SSQCxxkckORAKY/iIN2EturJQinVl/5OzjpSVh0pHqZTA8cXeb0CWq/93iRQrATKe/fuIUMG0wPgZH2cNWsWHj9+jESJEqFevXrCerl3ryGVHpWOHTti9erVoAkit2QqGVCtJxUDpZJR0LYOA6W2+nLrzisgB8pRZg5jSiyUtDVePNgUKCm7TacFB006JQHlpz/vwFEHgPLfTuXhnyetKbCGXsDE9dGROuwBpXz7WmrI3JmGnGU6Vo62UMrPep4eWRdJEsbHnaeRCIjyaPfLlQZLuxgslPIz0lL7tFVfIFNyY33p7wva+xutolI8TOdHj6/0ZgW0Xvu9SRtdAWWKFCkszk3aEpPO2Tx5YviFrKQsWLBAWCNv3bqFtGnTImXKlBg/fjx69+5tvJy22atWrYoDBw6gbNmySpo11tF6UjFQOjQcmlQ+dO0RmsxkC6Um4nKjLikgB0rz7VclQPni1Vv4Dl9v0gdySGk3/4B1oJRZC5VseS/vWgFlcqaxeEb595o9oKw6MRRXHkSYtGEOlIPrF8E3lfMa68jPelJKx0QJ4pvEwJQDpfyMtNQAOTERUJrHzFzaubzxPLUUvsilAeSLvVYBrdd+bxJGV0AZHBxsFyh37NiBzZs3i3rv3r1TrDVZHyncEAVGp/iVvr6+CAkJQd26dY1tkGUzY8aMIPikbXBHitaTioHSkdHQpu7ha4/QmIFSG3G5VZcUiAkoQ46Ho8tfMTvlvHz9DkWGmebPJocU85SGlFGnT+1CaDRjJ45EbT8rAcr/dQ9C8eypXALKwLGbcftppAVQyr2zKXZmh0rRQCk/60kpHRPEjwd5iCQ5UMp3IKSb0NnPAplSWIRUWt0jCA2m7xDVKM3juTH1XBo/vth7FdB67fcmZXQFlDEJR2ccCTgJJkuUKIFhw4ahSZMmirQmi2P58uXFGckhQ4aI85JBQUHiHCWdt5TK27dv4ePjg6lTp6Jnz54xtk0xMOVxMMPDw+Hv74/r168LBx+1CwOl2oo63h4toLSQUqGQKl8F5HS8Eb6CFdBAAVeBktIhFh5qBpRtyqHt/P0mve1Zo4BIydhk5k4cumY4z6gEKNf2rISiWVO6BJSlR27Ao4g3MQLl0AZF0T4oOvqH/KwnZc8hQ4Q8RFLZXGmwLGrLW/6DUbrJ398ECguleUilTX2qoOZkg+NoogTxcHY0A6UG09ormmSgjB4m3QNlWFiYiDtJjjUlS5YUINmoUSPFE+327dugMEMEeVu3bhXAKAHlnj17xGfmQDlt2jT06NEjxnsQ3FK/zIs7gDIwb1rhyVjVLPOEYlG4olMKyIGScgV/6c9A6ZSQfJHqCjSfvRv7rhgCm5tveZtYKK04x9A1r96+Q6EhpkBJQb3fvv9gco5SAkpyTiOLHhUKvSMFOTePzyg96MbelYWlz7w4suVdZOg6vHxjuitFz7rrwn18NddwFn5Yg6JoJwNKuSOPpIs8RFKZnKmxvGtFca38/Zb6+U/HQOGUYx5SaUf/asbYlEl84uP0qOidLtUHlxvUtQIMlF4AlNu3bxcWSYLAMmXKCJCkHN6OFDpjSWciIyMjQVvl6dKlE5efOnXK5S1vT1ooOUSFI7NAvbrkYUoLFBUGSvV05ZZcV8BVoHzz7j0KDA4x6cjc1n6oWigD8sv+LgHlZ7N24YAVoCRPavOzmNRo6HdVkSd9MpeAMs/ANSKvtryYAyUBbduK0RZK+da89L356MVrlI6KuVk6Z2qsiALKYzce45OfDe+3VBZHAaVUX/r7/sE1jdvgFDvz5EgGStdnsXe2wECpY6AkgCTLHznIlCtXToDkxx9/7PBMI4isU6cOzp8/L7a2KYyQVF69euXVTjkMlA5PB1UukC84DJSqSMqNqKSAq0D57v0H5Bu01qQ3FBqneuGMJn//tkYB9K5VEM1m78L+KwYLpXzL+9TIOig6zNS5h+qE9auGHGmTugSUto79yAOSBzcsijYyoGw8cycOR23NS9+b8pibcqA8cfOJ8Vyk1FHyTi+YKblFjM7jwbWNXvHJEyUABWXnEjcVYKDUKVCSNZG2uGkbmkBS7jDjyFSls5B0vpKsnPSPzlyaFwobRCGECDal0rlzZ6xatUr3YYMYKB2ZDerVPX7jCRr+bDiIP75JcXzBW97qicstuaSAq0BJWcjyDDQFSnKkKZYtpcnfJaAcu/Y0ftl+SfRZvuVNoXnMnXuozu6B1ZElVRKXgNI81iY1Rt+Fuy8+wJe/7hFtU/7xryvkNt5HftZT+t6Ux9y0B5RLOhNQprCI0Xl+TD2jRTdF4gQ4HsxA6dIE9uKLGSh1CpRS7MdkyZLZ9faOKWwQZcT55ZdfMGHCBFSqZEirJZWiRYsK66QU2LxNmzZo0aKFOFdJEOsNgc0ZKD3z7cNA6Rnd+a72FXAVKOkOcgtg/7qF0aWqIZ6j/O8SUEa8fitiSFKw8p82njOeoTwzqq6Fcw+1QVvEGVIkcgkoKSD5tC3nsWjvNWM75kA58lNftC4fDZTys57S9+bTyDcoERVzs1SO1Pivm+EM5albT1F/WphJH5d1KS/Ofkr16cMfm5VE07LZjbqkSuKDo8Nr2x8krhErFWCg1ClQWnNyiWkGkte2tZI7d25cvXrV6mfk3EOWUCqUenHQoEEmqRe7devm1KTXelLZOrzuVGf5IqcUkAMl5fL9vBw75TglJF+kugIxAeXa4+HoKoUNsuGUIwdHcycTa0ApfwB5NhpbQCnl0TZ/cEeccqRr8w9aK5yFqBAkylMmjvrUF61kQCk/6ykBpTyIe8kcqbEyCijP3H6KulPMgbKC2PKWgr5/6Z8D45oYdrykvqdJ6oPDwxgoVZ/UXtKg1mu/l8gguql7L29vEVPrScVA6fmZwEDp+THgHlhXQClQUhic8vkMzom24C5pwvg4JXMykX/39KpZAL1qFjS5VJ6NxhZQ0pnDFIl9bN5TgkOpQkx5vgsOCcHrt++tA6VZliBrusiDuMuB8uztZ6gzZbtJHykge6FMKYyORtaAMl2yhDg4tBZPzTiqgNZrvzfJykCp0mhpPakYKFUaKBeakTvlsIXSBSH5UtUVUBMozb2W5d89vWsWxLc1C5j0X26hpGw0reftw97LhhBGUiHQTOwTXxWglIcPIqujPAf3mMbF0CIg2gHz1+2XMGbtaSN80v+h7XrJcahk9lRY2T1IfH7+zjPU+skUKGk7nCyUUv0vyuXA+KamFsr0yRPhwJCaqo8pN+gdCmi99nuHCoZeMlCqNFpaTyoGSpUGyoVm5EA5oWkJNC+Xw4XW+FJWQD0F9AKU50bXw/NXb7H80A2MXmMAOSpSlhrzJ3Zmy7vY8PXiHlTMgdI84QCFQ/pn3zWQJbJE9tTiGnlWIDlQXrj7DDUnmwIlbYeTU47kaGQNKOlsKJ0R5RI3FdB67fcmVRkoVRotrScVA6VKA+VCMwyULojHl2qqgDyMj7nTnvwMpZItb1cslOT97BM/nnjWgoND8PqdYWtaylKjBlCWCF6Pp5HRQLnv8kOQBzgVJRms5FmBSmRPhVVRFsqL956jxo+G7DdSWdXdAJRSFiFrQJkpZSLsHcRAqekE13HjWq/9On50i64xUKo0WlpPKgZKlQbKhWbkgc3ZQumCkHyp6gqoCZTmcRUd2fKWWyLNzzpae2hnLJTyFIwEz/uvPESz2QagVBLOS54VqHi2VPhfD8OW9+X7L1Bt0laTblLopIKZkxuzCH3ulwM/fGa65Z0lVWLsHlhD9THlBr1DAa3Xfu9QwdBLBkqVRkvrScVAqdJAudCMPDXbhM9KoLkfb3m7ICdfqqICKw7fQO/FR0WLrlooYwLK1T0oNmUqk57Lz1BqAZSdKufFwPpFjPf0G70R95+/Nj6rHCiVnG0mhx6CXSpyoLxy/wWqmgElPS9ZKKX61oAyW+ok2DmguoqjyU15kwJar/3epAUDpUqjpfWkYqBUaaBcaEYOlBM/K4FmDJQuqMmXqqkABSbfeeEB8mdMjsypEps0veZYOLotOiT+pmTL2xZQfvQRbV1bZi2TA+XFsfURP95H4l6FhoTglcwb29rzKrFQyrfRqY2AsZtw5+krI1AeuPIQn0VZKJXsHLx9996YTlIOlNceRKDyxFBT7XoGCS9vKf2kNaDMniYJdvRnoFRzPntTW1qv/d6kBQOlSqOl9aRioFRpoFxohoHSBfH4Uo8pIAfKfzoGIjBvzGGDUiRKgOOyVILSd49frjRY2qWCxXPIgfLS2PqIFwWUhYeGIPJNdHgfZ4HS3OJaYdxm3HoSaR0oFewcyNNMUiag1T0MyS+uP4xApQmmQLm2ZyUUypzCmH6yuV92TPispKgv6ZIzbVJs71fNY+PLN/asAlqv/Z59OsfuzkDpmF42a2s9qRgoVRooF5phoHRBPL7UYwqoBZRlc6XBMjtAKXe+MQ/voxZQBv2wBTcevTQC5cGrD9F0luEMpZKdg/fvPyBvVN5yykO+pqcBKG88ikDQD6ZAGfJtJWGhlOpbA8pc6ZJi2/cMlB6b4B6+sdZrv4cfz6HbM1A6JJftylpPqkErjouUY9/XKYRu1fKr1GtuxhEFGCgdUYvr6kUBh4HSLDe19GO2TM7UWN7VkKZQXuQWSjlQFh22DhGv3xnBTy2grDIxFFcfRFgFyknNSuKzstljlF6et1wOlLcev0SF8VtMrl3fq7KIQynlOW9WNjsmNjO1UOZJnwyh3xmyr3GJewpovfZ7k6IMlCqNltaTin5VX3sYAfo1THnMubhfgcPXHqHxzF3ixkoWLvf3kO/IClgq4ChQpkycAMeC6xgbcgQo5dvTvsPW4YUGQFl90lZcuv/CKlBObl4STcrEDJR0ofRMRbOkxNpvDRbK8CcvUX6cKVBu6E1AmcJY3xpQ5s2QDFv6MlDG1XdP67Xfm3RloFRptHhSqSSkjps5dO0RmkQB5Y/NSqKpHUuIjh+FuxaHFPAUUJoHILcmuRKnHPMzlDUnb8OFu8+tAuVPn5dE49LOAeWdp5EIGLvZpJsbe1dGATtAmS9DMmxmoIxDb5Tpo/LaH60HA6VKrwFPKpWE1HEzDJQ6Hhzumk0FQo6Ho8tfBi/vxR0DEWDHKceWhbJ0ztRYYWXLu97UMJwOf2oEPKkjfqM34f7zaG9stYCy4fQdOH7zifF+ci9vR4GySJaUoHOSVO4+i4T/GFOg3NSnMvJnjLZQWgtsXiBjcmzsU4VnYBxVgNd+BkrVpz5PKtUl1V2DDJS6GxLukAIFKJB37Z+2I1GCeCCv5QRRmWzML5WshamS+ODo8NrGj6W/OwqU5CzTYu5etC6fG4NkcSTl93XGQnnm9lM0nbkLn5TKinFNSpgENp/6RSl8WiqbXVWk+xbOnALrelUW9e89e4VyYzaZXLupTxURiqn9/P04eO0R1n1b2RiWSWqDnHbW9za0wSXuKcBrPwOl6rOeJ5XqkuquwYNXH6HpLMMZSqVntXT3ENyhOKkAxV6ks9dSjEhrItgDylI5UuO/bjE75ZhvT1MQ8YQJDKkYY7onfSa/1l5UC3m78tSLrgDlg+evUHa0KVBu6VsFeTMkBznyvHn3weRZrEFpnJxccfyhee1noFT9FeBJpbqkumtQHp6EgVJ3w8MdclEBe0BZMkdqrHQQKO11yRkLpXmbey89wOe/7BF/nvZlaXxSMqu92xqdbOQWykcvXqP0qI0m15L3NnlxxwTD8m1zuzfmCrFOAV77GShVn9Q8qVSXVHcNMlDqbki4QyoqIMFd6qQ+ODLMcsvbFlDaOkOppGtqAOXuiw/w5a8GoJz+ZWk0dBIoH0e8RqmRpkC5o381ZE+TNEagrFwwA/5s56/kcblOLFSA134GStWnNU8q1SXVXYNyoFR6+F93D8EdYgVsKOCtQLnr4n189ete8VQ/f1UaDUoot1DKzz8+efkGJUdsMKrTuHQ2/PR5KZvz5ect57H6WDjmfu1nEzp5ssV+BXjtZ6BUfZbzpFJdUt016Iw3qe4egjvECtgByjRJfXDYmoUyeyqs7B5kcbU8sLn5GUp7Yqthodx14T6+mmsAyhlflcHHJbLYu61xy5uClm/obfDQfhr5BiWCDUBZtVAGzG/LVke7QnIF8NrPQKn6a8CTSnVJddegHCinfF4KjUrb9ybV3UNwh1gBO0CZNllCHBpay1hLgr4S2VNhlRWg9PSW984L94U3OZVZLcqgXnHngPL5q7eg2JkMlPyKOKIAr/0MlI7MF0V1eVIpksmrKzFQevXwceftKCCBY1D+9FjYIcACKJuUyYbJzS23gF0BSv8xm3D3mWWsSnte3vJH2XH+PlrOMwDl7JZlULeYc0AZ8fotig4zAGWVghnwB5+L5HdGgQK89jNQKpgmjlXhSeWYXt5Ye/+Vh2g2e7foOlsovXEEuc8xKUAB0P87chMjPilmjLVI9ZcdvIHNZ+5gTKPiSJMsoUUTrgDlhbvPMD7kLNpWzI2K+dNbQCz9wd42etj5e2g1b59TQCkPSv7y9TsUGbaOgZJfE4cU4LWfgdKhCaOkMk8qJSp5dx1n4t159xNz71kB+wq4ApS2WnfEQrn93D20/s0AlHNalUUd38x2Oy21T0HLKXg5lcg371B4KAOlXfG4gokCvPYzUKr+SvCkUl1S3TXIQKm7IeEO6UABTwPltnP38HUUUP7SqixqOwmUFCy94JAQoSiHAtLBxPKSLvDaz0Cp+lTlSaW6pLpr8OqDF6gycavoF2UMocwhXFiBuK6AFkBZcfwW3Hz8EkkTxsepkXVjlHjr2bto8/t+UefX1n6oVTST3SGRLJT5MiTD5r5VRf03796jwGAGSrvicQW2UNqYAx99oJxSXFxWgIHSZQm9ooF/9l0TW2NtKubxiv5yJ1kBrRXQAiivP4zAgj1X8UW5HCL1YUwl9MxdtJ1vAMq5rf1Q00mgfPf+A/INWivaYQul1rMm9rTPa3/0WMZaoLxw4QImTZqEPXv24MSJEyhcuLD4X/Oydu1aDB48GKdPn0b27NnRp08fdO3a1eHZzpPKYcn4AlaAFYgFCmgBlI7IsuXMHbSbf0BcMu9rP9QootxCmTdDMmyJslC+f/8BeaOAslKB9FjQPtrT3ZH+cN24pQCv/XEAKFeuXInu3bsjICAA586dw/v37y2Acvfu3ahcuTJat26Nli1bYufOnRg+fDjmzJmDDh06OPRW8KRySC6uzAqwArFEAU8D5ebTd9D+DwNQ/tbGD9ULOwCU6ZNhy3eGLW/arMsz0GChZKCMJZPTDY/Ba38cAEoCyHjx4oknbdOmDQ4cOGABlPXq1cPDhw+xd68hhhmVjh07YvXq1SL6vXS9kjnJk0qJSlyHFWAFYpsCngbKTafuoMOfBqD8vU05VCuc0a7E0hnKvDKgpIukvzNQ2pWQK0QpwGt/HABK+Wy3BpSvXr1CypQpMX78ePTu3dtYfdu2bahataoA0LJlyyp+aXhSKZaKK7ICrEAsUsDTQLnx1B18IwFl23KoVoiBMhZNL90/Cq/9DJQ4deoUfH19ERISgrp1o70I7927h4wZM2LBggViG1xp4UmlVCmuxwqwArFJgbpTtuPM7WfikewFIdfiuTecvI2OCw6Kpue3LYeqDgBlnvTJEBq15c0WSi1GJ/a3yWs/A6U4LxkUFAQ6RxkYGGhU5O3bt/Dx8cHUqVPRs2dPm2/D06dPQf+kEh4eDn9/f1y/fl0493BhBVgBViAuKOBpoFx/8jY6RQHln+38hYe2vSJtbedOlxRbv69mrG4r/aS99vjzuKsAAyUDpREoyQucHHekIgHltGnT0KNHD5tvSXBwMEaMGGHxOQNl3P1i4SdnBeKiAp7e8l534jY6LzRYKBe090elAgyUcXEeeuqZGSgZKF3e8mYLpadeX74vK8AK6EkBzwNlODovPCQkWdg+AEEFonOC29JJskTmSpcU29hCqafp5HV9YaBkoAQ75Xjde8sdZgVYAR0q4Okt77XHw9H1LwNQ/tUhABXzuw6UFfOnw18doo9C6VB27pJOFGCgZKAUClDYoMePH4tzlFLp3LkzVq1axWGDdPKycjdYAVZA3wp4GijXHAtHt0UGoFzUIQAVHADKnGmTYns/yzOUDJT6nnN66h0DZRwAyoiICFAWHCozZszAxYsXMXnyZPHfVapUQYYMGQRIUmBzCivUokULca5y2LBhHNhcT28r94UVYAV0rYCnt7yfv3oLv9Eb4RMvHvYPqYnEPvHt6tXhj/3YdPquhUVT2gpnoLQrIVeIUoCBMg4A5ZUrV5Anj/V8y6GhoSLWJBWCzkGDBpmkXuzWrZvDLwtPKocl4wtYAVYgFijgaaAkCQkq430EJE2YQJGilGbxYcRrpE+eyKQ+A6Ui+biSTAFe++MAULp7xvOkcrfifD9WgBXQgwKe3vJWUwMJKCvkS4dF3/AZSjW1ja1t8drPQKn63OZJpbqk3CArwAp4gQKxESirF86I39qU8wL1uYueVoDXfgZK1ecgTyrVJeUGWQFWwAsUiE1AOWjFcWw4eQfLu1RAznRJvUB97qKnFeC1n4FS9TnIk0p1SblBVoAV8AIFYhNQktx0vjIeHcjkwgooUIDXfgZKBdPEsSo8qRzTi2uzAqxA7FAgtgFl7BgVfgp3KcBrPwOl6nONJ5XqknKDrAAr4AUKMFB6wSBxFzVTgNd+BkrVJxdPKtUl5QZZAVbACxRgoPSCQeIuaqYAr/0MlKpPLp5UqkvKDbICrIAXKMBA6QWDxF3UTAFe+xkoVZ9cPKlUl5QbZAVYAS9QgIHSCwaJu6iZArz2M1CqPrl4UqkuKTfICrACnTV+SQAAEfhJREFUXqDAd0uOYunBG6KnV8Z/7AU95i6yAuopwGs/A6V6symqJZ5UqkvKDbICrIAXKPAk4g1GrTmFGoUzol7xLF7QY+4iK6CeArz2M1CqN5sYKFXXkhtkBVgBVoAVYAW8QQEGSgZK1ecpTyrVJeUGWQFWgBVgBVgBXSvAaz8DpeoTlCeV6pJyg6wAK8AKsAKsgK4V4LWfgVL1CcqTSnVJuUFWgBVgBVgBVkDXCvDaz0Cp+gTlSaW6pNwgK8AKsAKsACugawV47WegVH2C8qRSXVJukBVgBVgBVoAV0LUCvPYzUKo+QXlSqS4pN8gKsAKsACvACuhaAV77GShVn6A8qVSXlBtkBVgBVoAVYAV0rQCv/QyUqk9QnlSqS8oNsgKsACvACrACulaA134GStUnKE8q1SXlBlkBVoAVYAVYAV0rwGs/A6XqE5QnleqScoOsACvACrACrICuFeC1n4FS9QnKk0p1SblBVoAVYAVYAVZA1wrw2s9AqfoE5UmluqTcICvACrACrAAroGsFeO1noFR9gvKkUl1SbpAVYAVYAVaAFdC1Arz2M1CqPkF5UqkuKTfICrACrAArwAroWgFe+xkoVZ+gPKlUl5QbZAVYAVaAFWAFdK0Ar/0MlEYFzp07h549eyIsLAzJkiXDl19+ifHjxyNJkiQOTWKeVA7JxZVZAVaAFWAFWAGvV4DXfgZKocDjx49RrFgx5MqVC0OHDsXdu3fRp08f1K1bFwsXLnRoovOkckgurswKsAKsACvACni9Arz2M1AKBX744QeMHDkSV69eRfr06cXfFi1ahBYtWuDUqVMoUqSI4snOk0qxVFyRFWAFWAFWgBWIFQrw2s9AKRSoUqUKUqdOjZUrVxoVefXqFVKlSoUxY8agb9++iic8TyrFUnFFVoAVYAVYAVYgVijAaz8DpVAgY8aMaNeunTgzKS++vr4oX7485s6dq3jC86RSLBVXZAVYAVaAFWAFYoUCvPYzUAoFfHx8MGrUKAwYMMBkYgcFBQnYXL58uc0J//TpU9A/qVy/fh0VKlTAvn37kCVLlljxovBDsAKsACvACrACrIBtBcLDw+Hv74/Lly8jd+7ccVqqjz58+PAhripAQDl69Gj079/fRIKKFSsic+bMWLZsmU1pgoODMWLEiLgqHT83K8AKsAKsACvACkQpQMakcuXKxWk94jRQurLlbW6hjIyMBFkp8+TJgwQJEqg6qaRfQGz9VE9W1lQ9Lakl1lNdPVlT9fVkTVlTLRR4+/Yt7t27h+LFiyNx4sRa3MJr2ozTQKmmU46WI85nNNRXlzVVV1PWU109qTXWlDVVXwH1W+R5qr6m3tpinAZKChtEZygpbFC6dOnEGP7zzz8iuLmjYYO0nAD8wqqvLmuqrqasp7p6MlCqrydryppqowC3KikQp4FSCmxOB2nlgc3r1KnjcGBzLacUL9bqq8uaqqsp66mungw/6uvJmrKm2ijArTJQRilAqRd79OiBHTt2IGnSpMI6SZZLR1Mvajml6Lzm5MmTRRaflClTanmrONM2a6ruULOe6upJrbGmrKn6CqjfIs9T9TX11hbjtIXSWweN+80KsAKsACvACrACrICeFGCg1NNocF9YAVaAFWAFWAFWgBXwQgUYKL1w0LjLrAArwAqwAqwAK8AK6EkBBko9jQb3hRVgBVgBVoAVYAVYAS9UgIFSx4NGDkM9e/ZEWFgYkiVLJhyGKO+4nhyG9CDfkiVL8Ndff+HgwYN4+PAh8uXLhy5duqBTp06IFy+esYtr167F4MGDcfr0aWTPnl04OXXt2tXiESZNmoSff/4Zt2/fFsFqJ06ciKpVq+rhUT3Wh+fPn6Nw4cK4efMm9u/fDz8/P9bVydGYN28epk2bhrNnzwonu8DAQKxatYr1dFLP//77D+PGjRPvNX03UqYz+u9ChQqZtMjvv6XAFy5cAH3f7dmzBydOnBDvOP2veVFTu2fPnuG7777D0qVL8erVK1SvXh3Tp09Hrly5nJwBfJleFGCg1MtImPVDCmlEL5k8pFHdunV1FdJID/LRgkw6NW7cGJkyZUJoaKhYUHr16iVgkMru3btRuXJltG7dGi1btsTOnTsxfPhwzJkzBx06dDA+Bn25Dho0CGPHjkWZMmXw66+/ghYsylJEcBlXC6Un/eOPP3Dnzh0ToGRdHZsRlLL1p59+Ej9sAgICxA+gdevWiXnI89QxLan2pk2bULt2bfFOt2rVCvS9SRqT5/HJkyeNUTF4nlrXduXKlejevbuYi2TAeP/+vQVQqq1dgwYNcOjQIfz4449ifIYNGybG69ixY2wscfwV0NUVDJS6Go7ozlDoopEjR4qg6+nTpxcfLFq0CC1atNBV0HU9yEdprzJkyGDSFbI+zpo1SywwiRIlQr169cTivXfvXmO9jh07YvXq1SIjCVky6dcyASn9fcKECaLeu3fvBEiWKFFCBL2Pi+XMmTPCIkkLQOfOnU2AknVVPiPIgkZziaw9BEHWCuupXE+qST8GN2/ejEuXLuGjjz4SF9OPPwIk0pn0pMK6WteVAFLaxWnTpg0OHDhgAZRqakffv2QAWLNmDerXry86de3aNbGrRFZK+n7h4r0KMFDqdOy8JS2kTuXDggULhDXy1q1bSJs2rfglTMcFevfubezytm3bxFY2fYmWLVtWWDZp+4V+PZcuXdpYb8SIEQKmnjx5Yly09PrcWvSL4IeAmiwL1apVMwIlATjrqlxxsvKuWLFCWIKsFdZTuZZSza+//hqHDx8W1i2pkL603S1BC+uqTFdrQKm2drQrRMc96Me99AOAekffK8mTJ8f//vc/ZZ3lWrpUgIFSl8MCZMyYEe3atRMQJC++vr4oX7485s6dq9Oe66NbZGVctmwZ7t69K86qkW4hISGgIwNSIcsm6UzwSVtmM2fORLdu3RAREWGy9UJnNJs3b47r16+Ls5dxqdA5J9Lk/PnzArTlQEnpSVlX5bOBfiTSbkOpUqWENYas5/QuT506VfyN9VSupVRz+/btqFGjhkj8IG1507lzOhtIoEm7E6yrMl2tAaXa2tH3KFkk6cymvNB3zPr168W4cfFeBRgodTp2Pj4+Is/4gAEDTHoYFBQkIGj58uU67bnnu0UWR1qo6dfwkCFDxHlJ0o3OAtF2i1Tevn0L0pkWdFqExowZIzSPjIw0eQg6p1WrVi0cPXpUWOriSiGwpkP6dCaNftxs3brVBChZV8dmAlnNyGKeLVs2cUY3YcKEIOv3lStXBLDTmT+ep45pSrXp2MpXX30FcvagUrRoUQEn0o8/nqfKNLUGlGprR9+j8ePHF+eG5YW+p+kHPVkuuXivAgyUOh07Ap3Ro0eDtsnkhTwYM2fOLKxvXCwVIM9sOj9FiwkBEOkofSnSr2L6zBwoaQuG0m8SUJLmL1++NGl448aN4swbbavFJcccck6iZ6dzT3TOyhZQsq7K3sQCBQoICwx50ZJll0p4eDjy5MkjzkvTu01AyXoq05Nq7dq1S5zFIxj65JNPxLEUgnX6UUjvPR3J4PdfmZ4xAaVac5KAMkGCBGK3SF7ISW327Nl48OCBss5yLV0qwECpy2HhLW9nhoUWEzoTSYsJ5WZPly6daMbRbRsCysSJExu7EBe3vMkZrGDBguLMX4UKFYQWpGnDhg3FWVNy0qGtK0e2vOO6rvRjhnSlHz3yQtvdJUuWFD8eWU/H3nyahzlz5jTZsaGjLPSDkiI9kHMev//KNFVjy9veO85b3srGwltrMVDqdOTYKcexgSGIrFOnjtg6pK1teUwztQ+WO9Yz76wtWSNt9Z7giJya2ClH+fjSgk1bfeZASTBJIarIQsN6KteTaiZNmlQcC6LQM/JCVl+yXM6YMUNEb2Bd7evKTjn2NeIaMSvAQKnTGUJhg+g8H1k0JEsbha2h4Ob0i7tIkSI67bn7u0VnIZs0aQI6oE//rJ1zpNAX5ARBsCkVClFBAaXNwwbR3yVnKAobRO3RVndcChtEWh05csRkMOm/yUuewKdcuXICglhX5fOdHJyaNWuG48ePo1ixYuJCChSfN29e0PtOcVNZT+V6Uk36HiRLOsVTlAoBO1kt6QjL999/L/7MutrXNaawQWp9d0phg+QOkuTsSO8Ahw2yP0Z6r8FAqdMRkgKb586d2ySwOVnhFi5cqNNee6ZblBHnl19+EbEjK1WqZNIJOqBP1gkpOC99aVIsTzpXRVYNW4HNabuMgIm86ckBKq4HNidRzc9Q0t9YV+Vznn6c+Pv7C+cROqtLTjl0dlKKREDZsFhP5XpSTcpoReefyUv4008/FT8a6Qwl/RAnJ6csWbKIBllX67qS4x3F66RC1tyLFy8Kj3kqtEtG8X3V1o7Cj5EHvjywOR1X4sDmjs19PdZmoNTjqET1ieKp0ZclnV2jrR2yTpIlg1Mvmg4aQTctINYKnfeT0ibSFyc5mshTL9JCJC8fPnwQqchooaKsMGSZJFClcDlxvVgDStKEdVU+MwgeycpLMRLfvHkjFm3KnCNPE8h6KteT3lfKZkUewuTwRLEMCdrJOmnuQMe6WupKEQboeIA7vzspK46UevH169ecelH5dNd9TQZK3Q8Rd5AVYAVYAVaAFWAFWAF9K8BAqe/x4d6xAqwAK8AKsAKsACugewUYKHU/RNxBVoAVYAVYAVaAFWAF9K0AA6W+x4d7xwqwAqwAK8AKsAKsgO4VYKDU/RBxB1kBVoAVYAVYAVaAFdC3AgyU+h4f7h0rwArYUIByjFMubKlQvFbKPU6e/BTUWotiy9Ndi3tJbX700UeYOHGi8Iy1VSh/PcUGlUc10LJP3DYrwAqwAuYKMFDynGAFWAGvVICAkkI6bdmyRfT/1q1bIiD9wYMHERYWZkwZqebDUcgTSixAIWkobqQ7CuVRpsxPUkxFa/dkoHTHSPA9WAFWICYFGCh5frACrIBXKkBASTFDnz9/bux/eHg4smXLhm+++UYErY8rhYEyrow0PycroF8FGCj1OzbcM1aAFYhBAWtASdUzZcokshxRejepULaPwYMHg1K/JUiQAB9//DGmTJmCjBkzGutQCk5Ku7l582akTZtWpEKkgPmrV68GBYCmYm3Lm/LIU9uUmvP+/fsiFeDAgQPx1VdfGduW0tpRwHwKbE5JC3x9fTFr1iyULVs2xnG2tuVNmXaoLYLp2rVro0OHDuKZeMubXxlWgBXwlAIMlJ5Snu/LCrACLilgDSgJsNKkSQNKx0nARYVgkrIl0bnK9u3b48WLFxgyZAjozCVtJ1OhjCt+fn4iOxKl3UyVKpXYTiegjB8/foxA2bRpU5EtiCCPcnQTWM6fPx9//vknWrVqJdonoCQwzZo1KwYMGCDSgdL/0hY6pbvz8fGxqYU5UErpBulMZc2aNbFhwwYsXrxY5AVnoHRpSvHFrAAr4IICDJQuiMeXsgKsgOcUkICS8jdToe1ugrR169YJUCxQoID4O6U3fPv2rUhhSnBGhfI80zlIgjwCTQJCsvBt377dmA+eYC979uzCWmnLQkn5h0uWLCnyIHft2tUoRp06dXD27FnjdQSUBJjHjx8XlkkqmzZtQq1atcR5z6CgIEVASfnAc+bMiRo1aoj2pELW0L///puB0nPTke/MCsR5BRgo4/wUYAFYAe9UwNzLm56CrImrVq0yenlHREQIayCdtezevbvJg5KjS8eOHTF8+HDhLU5b4I8ePTKp8+mnn+Lo0aM2gZJAktqlrW6yeEqFLJRt27bFtWvXkCNHDmGhJICkbXWpkBMRnfckEPziiy8UASVZTCl3/fLly9G4cWPjNfTfZCllC6V3zmXuNSsQGxRgoIwNo8jPwArEQQUkL2+yKr5//x7nz58XFkqCyBMnTgivaNoGJiujrUJb4HPnzhVnJ8lbnM42ygs592zcuNEmUNI2N8HomzdvTK4jK2m9evVw5MgRYcGUzlBSv6RCEJohQwb8/vvv4nNbRb7lTZbX8uXLC2trxYoVjZfs3LlTWDkZKOPgi8CPzAroRAEGSp0MBHeDFWAFHFPA2hlKChnk7+8vLI/k8ELnJVOkSCFiUzZq1MjiBunTpxcWP2ctlDNnzkS3bt3w4MEDsTUuFWsWSvLEdhUo2ULp2Bzh2qwAK+A+Bf4PPmujlEULj1sAAAAASUVORK5CYII=\" width=\"599.9999869953505\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.style.use('default')\n",
    "#plt.style.use('dark_background')\n",
    "\n",
    "# Plot number of probes per region\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(list(reg_size_dic.values()))\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"Num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"Region id\", labelpad=1)\n",
    "plt.ylim([0, np.max(list(reg_size_dic.values()))*1.1])\n",
    "plt.savefig(os.path.join(report_folder, 'afterblast_nProbes_per_region.png'),\n",
    "            transparent=True, dpi=300)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a9429fce",
   "metadata": {},
   "source": [
    "# Create count table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 278,
   "id": "b427acb4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "- Start multi-processing comsume 58492 sequences 44 threads, finish in 57.421s\n",
      "- Total sequences loaded: 8539832\n",
      "Time to compute unique and clip: 2.3927886486053467\n",
      "Time to update matrix: 6.848090410232544\n",
      "- start saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\ctp14-mouse-5kb_library_17w.npy\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "from tqdm import tqdm\n",
    "overwrite_table = False\n",
    "# library\n",
    "library_table_file = os.path.join(reference_folder, 'ctp14-mouse-5kb_library_17w.npy')\n",
    "\n",
    "if not os.path.exists(library_table_file) or overwrite_table:\n",
    "    # library\n",
    "    _library_filenames = [os.path.join(library_folder, 'blast_full_probes.fasta')]\n",
    "    print(len(_library_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=library_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_library_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=44) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {library_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e8c6acaa",
   "metadata": {},
   "source": [
    "# Design corresponding sequential as validation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 344,
   "id": "b24c0c32",
   "metadata": {},
   "outputs": [],
   "source": [
    "## load DNA-MERFISH count-table to select probes\n",
    "library_table_file = os.path.join(reference_folder, 'ctp14-mouse-5kb_library_17w.npy')\n",
    "lib_table = library_tools.design.countTable(save_file=library_table_file, word=17)\n",
    "lib_table.load()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 346,
   "id": "1ebbe549",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 346,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "984fdb10",
   "metadata": {},
   "source": [
    "## select based on groups:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 248,
   "id": "3e94e314",
   "metadata": {},
   "outputs": [],
   "source": [
    "region_2_group_id = pickle.load(open(os.path.join(library_folder, 'region_2_group_id.pkl'), 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 350,
   "id": "e3ad9ec1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0 31 55\n",
      "7 30 55\n",
      "8 31 55\n",
      "92\n"
     ]
    }
   ],
   "source": [
    "# split blasted probes into genes\n",
    "gene_2_probes = check.split_probe_by_gene(blasted_records,)\n",
    "\n",
    "sel_sequential_pb_dict = {}\n",
    "kept_group_th = 0.54 # percentage of regions that could be sequentially labeled\n",
    "_library_hit_th = 14\n",
    "_sequential_probe_num_th = 12\n",
    "\n",
    "# for each group, check pb_dict\n",
    "for _g, _region_names in group_2_regions.items():\n",
    "    _sel_regions = []\n",
    "    for _rname in _region_names:\n",
    "        if _rname not in pb_dict:\n",
    "            print(f\"{_rname} doesn't exist.\")\n",
    "            continue\n",
    "        # final probe num\n",
    "        _final_probe_num = len(gene_2_probes[_rname])\n",
    "        _design_probe_num = len(pb_dict[_rname].kept_probes)\n",
    "        if _design_probe_num - _final_probe_num > 12:\n",
    "            #print(_rname, _design_probe_num, _final_probe_num)\n",
    "            _sel_regions.append(_rname)\n",
    "    # check if all regions in this group have extra probes:\n",
    "    if len(_sel_regions) >= len(_region_names) * kept_group_th:\n",
    "        print(_g, len(_sel_regions), len(_region_names))\n",
    "        # for probes in these regions, add to sel_sequential_pb_dict\n",
    "        \n",
    "        for _rname in _sel_regions:\n",
    "            # collect used\n",
    "            _used_pb_pos = [int(_r.id.split('pos_')[1].split('_')[0]) for _r in gene_2_probes[_rname]]\n",
    "            # re-assemble kept_pbs for each region\n",
    "            _kept_pbs = {}\n",
    "            for _seq, _info in pb_dict[_rname].kept_probes.items():\n",
    "                # map with multiplexed library\n",
    "                _library_hits = lib_table.get(_seq)\n",
    "                if _library_hits > _library_hit_th:\n",
    "                    #print(_library_hits, '*')\n",
    "                    continue\n",
    "                if _info['pb_index'] not in _used_pb_pos:\n",
    "                    _new_info = {_k:_v for _k,_v in _info.items()}\n",
    "                    # add sequential tag\n",
    "                    _new_info['reg_name'] = _new_info['reg_name'].replace('Chr7-6Mb-seg', 'Chr7-6Mb-sequential-seg')\n",
    "                    # store group info_\n",
    "                    _new_info['reg_name'] += f'_group_{_g}'\n",
    "                    _new_info['group_id'] = _g\n",
    "                    # map library\n",
    "                    _new_info['map_library'] = _library_hits\n",
    "                    # append\n",
    "                    _kept_pbs[_seq] = _new_info\n",
    "            if len(_kept_pbs) < _sequential_probe_num_th:\n",
    "                continue\n",
    "            #print(len(_kept_pbs))\n",
    "            _splitted_pbde = library_tools.design.pb_reports_class()\n",
    "            _splitted_pbde.kept_probes = _kept_pbs\n",
    "            \n",
    "            sel_sequential_pb_dict[_rname] = _splitted_pbde\n",
    "            \n",
    "print(len(sel_sequential_pb_dict))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "da8411d1",
   "metadata": {},
   "source": [
    "## add sequential tag"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 351,
   "id": "18fea07a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "4"
      ]
     },
     "execution_count": 351,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_readout_per_probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 352,
   "id": "729f1389",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Chr7-6Mb-seg-65': ['u0', 'u0', 'u0', 'u0'], 'Chr7-6Mb-seg-75': ['u1', 'u1', 'u1', 'u1'], 'Chr7-6Mb-seg-85': ['u2', 'u2', 'u2', 'u2'], 'Chr7-6Mb-seg-95': ['u3', 'u3', 'u3', 'u3'], 'Chr7-6Mb-seg-105': ['u4', 'u4', 'u4', 'u4'], 'Chr7-6Mb-seg-135': ['u5', 'u5', 'u5', 'u5'], 'Chr7-6Mb-seg-166': ['u6', 'u6', 'u6', 'u6'], 'Chr7-6Mb-seg-176': ['u7', 'u7', 'u7', 'u7'], 'Chr7-6Mb-seg-196': ['u8', 'u8', 'u8', 'u8'], 'Chr7-6Mb-seg-206': ['u9', 'u9', 'u9', 'u9'], 'Chr7-6Mb-seg-216': ['u10', 'u10', 'u10', 'u10'], 'Chr7-6Mb-seg-226': ['u11', 'u11', 'u11', 'u11'], 'Chr7-6Mb-seg-236': ['u12', 'u12', 'u12', 'u12'], 'Chr7-6Mb-seg-286': ['u13', 'u13', 'u13', 'u13'], 'Chr7-6Mb-seg-326': ['u14', 'u14', 'u14', 'u14'], 'Chr7-6Mb-seg-336': ['u15', 'u15', 'u15', 'u15'], 'Chr7-6Mb-seg-346': ['u16', 'u16', 'u16', 'u16'], 'Chr7-6Mb-seg-366': ['u17', 'u17', 'u17', 'u17'], 'Chr7-6Mb-seg-396': ['u18', 'u18', 'u18', 'u18'], 'Chr7-6Mb-seg-416': ['u19', 'u19', 'u19', 'u19'], 'Chr7-6Mb-seg-426': ['u20', 'u20', 'u20', 'u20'], 'Chr7-6Mb-seg-436': ['u21', 'u21', 'u21', 'u21'], 'Chr7-6Mb-seg-446': ['u22', 'u22', 'u22', 'u22'], 'Chr7-6Mb-seg-456': ['u23', 'u23', 'u23', 'u23'], 'Chr7-6Mb-seg-466': ['u24', 'u24', 'u24', 'u24'], 'Chr7-6Mb-seg-476': ['u25', 'u25', 'u25', 'u25'], 'Chr7-6Mb-seg-486': ['u26', 'u26', 'u26', 'u26'], 'Chr7-6Mb-seg-496': ['u27', 'u27', 'u27', 'u27'], 'Chr7-6Mb-seg-506': ['u28', 'u28', 'u28', 'u28'], 'Chr7-6Mb-seg-536': ['u29', 'u29', 'u29', 'u29'], 'Chr7-6Mb-seg-556': ['u30', 'u30', 'u30', 'u30'], 'Chr7-6Mb-seg-52': ['u31', 'u31', 'u31', 'u31'], 'Chr7-6Mb-seg-62': ['u32', 'u32', 'u32', 'u32'], 'Chr7-6Mb-seg-72': ['u33', 'u33', 'u33', 'u33'], 'Chr7-6Mb-seg-82': ['u34', 'u34', 'u34', 'u34'], 'Chr7-6Mb-seg-102': ['u35', 'u35', 'u35', 'u35'], 'Chr7-6Mb-seg-122': ['u36', 'u36', 'u36', 'u36'], 'Chr7-6Mb-seg-132': ['u37', 'u37', 'u37', 'u37'], 'Chr7-6Mb-seg-163': ['u38', 'u38', 'u38', 'u38'], 'Chr7-6Mb-seg-173': ['u39', 'u39', 'u39', 'u39'], 'Chr7-6Mb-seg-203': ['u40', 'u40', 'u40', 'u40'], 'Chr7-6Mb-seg-233': ['u41', 'u41', 'u41', 'u41'], 'Chr7-6Mb-seg-253': ['u42', 'u42', 'u42', 'u42'], 'Chr7-6Mb-seg-263': ['u43', 'u43', 'u43', 'u43'], 'Chr7-6Mb-seg-273': ['u44', 'u44', 'u44', 'u44'], 'Chr7-6Mb-seg-283': ['u45', 'u45', 'u45', 'u45'], 'Chr7-6Mb-seg-343': ['u46', 'u46', 'u46', 'u46'], 'Chr7-6Mb-seg-363': ['u47', 'u47', 'u47', 'u47'], 'Chr7-6Mb-seg-373': ['u48', 'u48', 'u48', 'u48'], 'Chr7-6Mb-seg-413': ['u49', 'u49', 'u49', 'u49'], 'Chr7-6Mb-seg-423': ['u50', 'u50', 'u50', 'u50'], 'Chr7-6Mb-seg-433': ['u51', 'u51', 'u51', 'u51'], 'Chr7-6Mb-seg-443': ['u52', 'u52', 'u52', 'u52'], 'Chr7-6Mb-seg-453': ['u53', 'u53', 'u53', 'u53'], 'Chr7-6Mb-seg-463': ['u54', 'u54', 'u54', 'u54'], 'Chr7-6Mb-seg-473': ['u55', 'u55', 'u55', 'u55'], 'Chr7-6Mb-seg-483': ['u56', 'u56', 'u56', 'u56'], 'Chr7-6Mb-seg-493': ['u57', 'u57', 'u57', 'u57'], 'Chr7-6Mb-seg-513': ['u58', 'u58', 'u58', 'u58'], 'Chr7-6Mb-seg-523': ['u59', 'u59', 'u59', 'u59'], 'Chr7-6Mb-seg-533': ['u60', 'u60', 'u60', 'u60'], 'Chr7-6Mb-seg-53': ['u61', 'u61', 'u61', 'u61'], 'Chr7-6Mb-seg-63': ['u62', 'u62', 'u62', 'u62'], 'Chr7-6Mb-seg-93': ['u63', 'u63', 'u63', 'u63'], 'Chr7-6Mb-seg-123': ['u64', 'u64', 'u64', 'u64'], 'Chr7-6Mb-seg-133': ['u65', 'u65', 'u65', 'u65'], 'Chr7-6Mb-seg-164': ['u66', 'u66', 'u66', 'u66'], 'Chr7-6Mb-seg-174': ['u67', 'u67', 'u67', 'u67'], 'Chr7-6Mb-seg-184': ['u68', 'u68', 'u68', 'u68'], 'Chr7-6Mb-seg-204': ['u69', 'u69', 'u69', 'u69'], 'Chr7-6Mb-seg-234': ['u70', 'u70', 'u70', 'u70'], 'Chr7-6Mb-seg-274': ['u71', 'u71', 'u71', 'u71'], 'Chr7-6Mb-seg-284': ['u72', 'u72', 'u72', 'u72'], 'Chr7-6Mb-seg-304': ['u73', 'u73', 'u73', 'u73'], 'Chr7-6Mb-seg-324': ['u74', 'u74', 'u74', 'u74'], 'Chr7-6Mb-seg-344': ['u75', 'u75', 'u75', 'u75'], 'Chr7-6Mb-seg-364': ['u76', 'u76', 'u76', 'u76'], 'Chr7-6Mb-seg-374': ['u77', 'u77', 'u77', 'u77'], 'Chr7-6Mb-seg-384': ['u78', 'u78', 'u78', 'u78'], 'Chr7-6Mb-seg-394': ['u79', 'u79', 'u79', 'u79'], 'Chr7-6Mb-seg-404': ['u80', 'u80', 'u80', 'u80'], 'Chr7-6Mb-seg-424': ['u81', 'u81', 'u81', 'u81'], 'Chr7-6Mb-seg-434': ['u82', 'u82', 'u82', 'u82'], 'Chr7-6Mb-seg-444': ['u83', 'u83', 'u83', 'u83'], 'Chr7-6Mb-seg-454': ['u84', 'u84', 'u84', 'u84'], 'Chr7-6Mb-seg-464': ['u85', 'u85', 'u85', 'u85'], 'Chr7-6Mb-seg-474': ['u86', 'u86', 'u86', 'u86'], 'Chr7-6Mb-seg-484': ['u87', 'u87', 'u87', 'u87'], 'Chr7-6Mb-seg-494': ['u88', 'u88', 'u88', 'u88'], 'Chr7-6Mb-seg-504': ['u89', 'u89', 'u89', 'u89'], 'Chr7-6Mb-seg-534': ['u90', 'u90', 'u90', 'u90'], 'Chr7-6Mb-seg-544': ['u91', 'u91', 'u91', 'u91']}\n"
     ]
    }
   ],
   "source": [
    "sequential_region_2_readout_dict = {}\n",
    "for _i, _reg in enumerate(sel_sequential_pb_dict):\n",
    "    sequential_region_2_readout_dict[_reg] = [f\"u{_i}\"] * num_readout_per_probe\n",
    "print(sequential_region_2_readout_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2ffdde78",
   "metadata": {},
   "source": [
    "## loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f2820aad",
   "metadata": {},
   "source": [
    "### primers doesn't change"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6f89badf",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 353,
   "id": "15e880ee",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'u': 92}"
      ]
     },
     "execution_count": 353,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# collect types from region_2_readout_dict\n",
    "sequential_readout_usage_by_type = {}\n",
    "for _region, _infos in sequential_region_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in sequential_readout_usage_by_type:\n",
    "            sequential_readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in sequential_readout_usage_by_type[_info[0]]:\n",
    "            sequential_readout_usage_by_type[_info[0]].append(int(_info[1:]))\n",
    "\n",
    "readout_usage_dict = {_type:len(_v) for _type,_v in sequential_readout_usage_by_type.items()}\n",
    "\n",
    "# collect types from region_2_readout_dict\n",
    "sequential_readout_usage_dict = {}\n",
    "for _region, _infos in sequential_region_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in sequential_readout_usage_by_type:\n",
    "            sequential_readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in sequential_readout_usage_by_type[_info[0]]:\n",
    "            sequential_readout_usage_by_type[_info[0]].append(int(_info[1:]))\n",
    "\n",
    "sequential_readout_usage_dict = {_type:len(_v) for _type,_v in sequential_readout_usage_by_type.items()}\n",
    "sequential_readout_usage_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 354,
   "id": "b20c45c3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "127\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_w_stv_750.fasta 175 127 31\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_w_stv_647.fasta 174 127 31\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_w_stv_561.fasta 180 127 31\n",
      "[31, 31, 31]\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3.classes import _allowed_kwds\n",
    "reload(ia.library_tools.assemble)\n",
    "readout_dict ={}\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets'\n",
    "\n",
    "# combo\n",
    "readout_type = 'unique'\n",
    "# \n",
    "_type_char = _allowed_kwds[readout_type]\n",
    "_readout_start = 127\n",
    "print(_readout_start)\n",
    "_num_color = 3\n",
    "\n",
    "readout_dict[_type_char] = library_tools.assemble.load_readouts(\n",
    "    sequential_readout_usage_dict[_type_char], \n",
    "    _type='highest_snr_w_stv', \n",
    "    _readout_folder=readout_folder, \n",
    "    _num_colors=_num_color, # 750,647,561\n",
    "    _start_id=_readout_start,\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 355,
   "id": "9e6808eb",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'u': [SeqRecord(seq=Seq('CACTATACTTATCTGGTCCGGCGAGGTGTG'), id='NDB_823', name='NDB_823', description='NDB_823', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTATGTAGCGTCGGGTCAGCTTAACTCCG'), id='NDB_944', name='NDB_944', description='NDB_944', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCGCTATCGTCCTAGTGGCGTGATGGAGTT'), id='NDB_462', name='NDB_462', description='NDB_462', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCGATTATAAGCTAGGTGCGTCAGGCTCT'), id='NDB_1105', name='NDB_1105', description='NDB_1105', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGAGGCATATTCCAATGCGAACGCTGCA'), id='NDB_419', name='NDB_419', description='NDB_419', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATCATTTCCGAGAAGCCTTCGCAATTTCG'), id='NDB_996', name='NDB_996', description='NDB_996', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCGATGCACGTTATGTCGTAAACTTTGGC'), id='NDB_814', name='NDB_814', description='NDB_814', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAGTTGCCTATTCCGAAGCCTGTAATTAA'), id='NDB_911', name='NDB_911', description='NDB_911', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTCTCGACAAACGTCTTCGAGGCATGAAT'), id='NDB_420', name='NDB_420', description='NDB_420', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGTAGATACTCTACACGTGCAGCGGCGAA'), id='NDB_988', name='NDB_988', description='NDB_988', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACCTTTAGGCGATGTTCGTGCCAATCTTT'), id='NDB_374', name='NDB_374', description='NDB_374', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCGTGGGTGATTGTTCGGATTACCGTCCC'), id='NDB_924', name='NDB_924', description='NDB_924', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCGTCGTGTAAGCCTACGTGACGTCAACG'), id='NDB_949', name='NDB_949', description='NDB_949', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGACATGCTGTATTCCGCATTAGGGTCTCT'), id='NDB_749', name='NDB_749', description='NDB_749', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATACTGCATCCGAGATGTCGTCAACTACAA'), id='NDB_1089', name='NDB_1089', description='NDB_1089', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCACTAGGTGTTACGTGACCGCGGGAAT'), id='NDB_553', name='NDB_553', description='NDB_553', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCGTCTTGTACATTAGCCTGTCGGAATGA'), id='NDB_410', name='NDB_410', description='NDB_410', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTTATACATGACTACCGGCCATGTCATTT'), id='NDB_969', name='NDB_969', description='NDB_969', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTCAATGGTATTCTAGCGGTCGCGCAACAG'), id='NDB_799', name='NDB_799', description='NDB_799', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATAAGTCTCTTATCGCGAGCTGATCACTT'), id='NDB_422', name='NDB_422', description='NDB_422', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCGTAATGTATCGGTCCCATACGATGATC'), id='NDB_1011', name='NDB_1011', description='NDB_1011', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCGCTGTGAGCTAACGCGCTTTGAAGCACT'), id='NDB_586', name='NDB_586', description='NDB_586', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTGTAGACTGCAACTGGTCCACGCTTATA'), id='NDB_983', name='NDB_983', description='NDB_983', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTAATTGCGAACAGTGACGTACAGCCTCC'), id='NDB_1041', name='NDB_1041', description='NDB_1041', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACAACCTAGTAGTATACCGGCACCGGATG'), id='NDB_1018', name='NDB_1018', description='NDB_1018', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAACTGGTTTATCACGTAGCGCTCATTACG'), id='NDB_521', name='NDB_521', description='NDB_521', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATCTCAACACGTGGGTCGCGGCTATACTA'), id='NDB_678', name='NDB_678', description='NDB_678', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGCTACATCTAGTGCACGTTATTGTTCGA'), id='NDB_670', name='NDB_670', description='NDB_670', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCGATCTCGTTTGGACTTGTGACGCTGTC'), id='NDB_842', name='NDB_842', description='NDB_842', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTATTGAGCTGATAGCGCCGTTATGTTCC'), id='NDB_579', name='NDB_579', description='NDB_579', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCGTTTAGGTCTACGCCGTCTCGTGGTAAG'), id='NDB_748', name='NDB_748', description='NDB_748', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCTGTCGGTCTAGAGCGAAGGTTACGCG'), id='NDB_668', name='NDB_668', description='NDB_668', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTTAGCTATACGGAACGTTAGAGACCCGT'), id='NDB_843', name='NDB_843', description='NDB_843', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCTTCGTGAATGCGATCTGCGTGCTTAT'), id='NDB_907', name='NDB_907', description='NDB_907', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATTTCGTTGCACCACGGTCGGTCTTTAAA'), id='NDB_761', name='NDB_761', description='NDB_761', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCTAGTATGGTGTTCTCCGCTCGGTTGTT'), id='NDB_525', name='NDB_525', description='NDB_525', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTAAAGTTGCGATCTAGGGTCCCTAACTAA'), id='NDB_595', name='NDB_595', description='NDB_595', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGCAGTGCGCTGTTGGTTAGCTACGTTCA'), id='NDB_821', name='NDB_821', description='NDB_821', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGCACGAATAGACGTTTGCTTACACATGC'), id='NDB_753', name='NDB_753', description='NDB_753', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGATGCCGCGTCCCTTATTAACGAAAGCG'), id='NDB_484', name='NDB_484', description='NDB_484', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCATAATGTCGCGGTCAGTCTAGTTAGCC'), id='NDB_1016', name='NDB_1016', description='NDB_1016', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAGATACTCGTCTTGGTAATTCGTACGTCC'), id='NDB_1095', name='NDB_1095', description='NDB_1095', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGGTACTTTGGTGCGGACCTTCTCATCCG'), id='NDB_973', name='NDB_973', description='NDB_973', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTTTGAGCTTTGCGGCGATCGGTCGAATA'), id='NDB_389', name='NDB_389', description='NDB_389', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCATGTTATCTATGTCCGATCGTACGAAG'), id='NDB_633', name='NDB_633', description='NDB_633', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGATAAGATCAACTCACTCGCGGGACGACT'), id='NDB_442', name='NDB_442', description='NDB_442', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGCATAAGGCATAGATCGCGGTCATCTTA'), id='NDB_353', name='NDB_353', description='NDB_353', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACGATATTGTGCATCAGGCCGTCGAAGCC'), id='NDB_609', name='NDB_609', description='NDB_609', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTGTAGGATGCGAAATTCCGCCAATTTGC'), id='NDB_1111', name='NDB_1111', description='NDB_1111', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGCACCGTAGTCGTTGCCGTACTTTATTT'), id='NDB_377', name='NDB_377', description='NDB_377', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGAGTACGCCAGCTAAAGAACACGTACATG'), id='NDB_534', name='NDB_534', description='NDB_534', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGCTAGTTCTTCAGTTCGTTGTCGCGAGA'), id='NDB_793', name='NDB_793', description='NDB_793', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACCATGGATCTCGTTTCGATGCCAAGCGA'), id='NDB_767', name='NDB_767', description='NDB_767', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTATCCGGCTAGTTAACACGCACATATCT'), id='NDB_684', name='NDB_684', description='NDB_684', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCCAATCGAGCGGATTATCCTGTCTAGCA'), id='NDB_859', name='NDB_859', description='NDB_859', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGTATATTAGACCGCCCGCAGGATGAGGT'), id='NDB_497', name='NDB_497', description='NDB_497', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGGTGCGCTTATGTCTACGATCGTGGCTG'), id='NDB_651', name='NDB_651', description='NDB_651', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACTAGTTAGGGAGTCGCTCGCCGAGAATC'), id='NDB_475', name='NDB_475', description='NDB_475', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGCTATAACCTTTACGCATCCGTTGGATT'), id='NDB_986', name='NDB_986', description='NDB_986', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACCATTTGTTCGGCGAGACGTGGCCAATT'), id='NDB_588', name='NDB_588', description='NDB_588', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAACTCGATGAGGTGCATCAGCCCGCTAAT'), id='NDB_841', name='NDB_841', description='NDB_841', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTATGTTGTCGTGGATGCCGCACGAGATA'), id='NDB_659', name='NDB_659', description='NDB_659', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATATGGCGTAGGTCTCGCGCCTCTAAATT'), id='NDB_720', name='NDB_720', description='NDB_720', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGCACGAAGTTTACGCTTATCAGATCGTT'), id='NDB_607', name='NDB_607', description='NDB_607', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCTAATAGTCGGGTGTCGGCTACGTTTAC'), id='NDB_548', name='NDB_548', description='NDB_548', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTAATGAATCTTCCCGTCCGAATGTTACTC'), id='NDB_726', name='NDB_726', description='NDB_726', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGATTTGGTATACCCGGACCCGTATATCT'), id='NDB_610', name='NDB_610', description='NDB_610', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGACGACGCACGTTCGTACCGCGTACTTCG'), id='Stv_32', name='Stv_32', description='Stv_32 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTTTGTCACAAATGCGCGTATATCGTCTC'), id='NDB_717', name='NDB_717', description='NDB_717', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACAAATCCGACCAGATCGGACGATCATGGG'), id='Stv_3', name='Stv_3', description='Stv_3 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTGCTCGCAAGTGCGCACGAGTTGAACTG'), id='Stv_33', name='Stv_33', description='Stv_33 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGGATTCTAGACCTACCGGGTGCAGAACC'), id='NDB_1071', name='NDB_1071', description='NDB_1071', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAGTATGCAGCGCGATTGACCGTCTCGTT'), id='Stv_4', name='Stv_4', description='Stv_4 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTTTCCATACGCGGCGTGAATATCGCGGC'), id='Stv_35', name='Stv_35', description='Stv_35 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTATACAATTGCGTTTCCCGTCATCGGCTG'), id='NDB_576', name='NDB_576', description='NDB_576', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGTCGTACGCCGATGCGCAGCAATTCACT'), id='Stv_5', name='Stv_5', description='Stv_5 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTACAGCGTTACGGGCCCAGACGTTTCAT'), id='Stv_36', name='Stv_36', description='Stv_36 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCAAATCGCTACGACCGCCGAGATATTAAG'), id='NDB_870', name='NDB_870', description='NDB_870', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGAAACATCGGCCACGGTCCCGTTGAACTT'), id='Stv_6', name='Stv_6', description='Stv_6 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTATCGTGCGTGTCCGCCAGTACCGTGAG'), id='Stv_37', name='Stv_37', description='Stv_37 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCATCAGGTTCGTATGGCTATGAGCTCG'), id='NDB_468', name='NDB_468', description='NDB_468', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGAATCCACCGTCCAGCGCGTCAAACAGA'), id='Stv_7', name='Stv_7', description='Stv_7 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTGCGGGAACCGTATCCCTGGCGCGGACT'), id='Stv_39', name='Stv_39', description='Stv_39 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCATAGTCACAATGCATCGCCTTTACTAT'), id='NDB_978', name='NDB_978', description='NDB_978', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCGAAATCCCCGTAACGAGCGTCCCTTGC'), id='Stv_8', name='Stv_8', description='Stv_8 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATCCGGCGTACGTAAGGCAGCTTGCGTTA'), id='Stv_40', name='Stv_40', description='Stv_40 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCCGCCAATGACGTCGGTTTGGACGAGA'), id='Stv_91', name='Stv_91', description='Stv_91 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCATGAGTTGCCTGGCGTTGCGACGACTAA'), id='Stv_9', name='Stv_9', description='Stv_9 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGGGATTCGTACGAGCCCTCTTGGACGGG'), id='Stv_42', name='Stv_42', description='Stv_42 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGTTGATGTCCCTTGTGAGCGCCCGACAT'), id='Stv_92', name='Stv_92', description='Stv_92 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCGTCGTCTCCGGTCCACCGTTGCGCTTAC'), id='Stv_10', name='Stv_10', description='Stv_10 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTTGTTCGGCTCGGAGCGTTACTGCGGGC'), id='Stv_44', name='Stv_44', description='Stv_44 old_barcode', dbxrefs=[])]}"
      ]
     },
     "execution_count": 355,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "readout_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 356,
   "id": "549a8b81",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save sequential readout usage dict\n"
     ]
    }
   ],
   "source": [
    "# save\n",
    "sequential_readout_usage_file = os.path.join(library_folder, 'sequential_readout_usage.pkl')\n",
    "if not os.path.isfile(sequential_readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save sequential readout usage dict')\n",
    "    pickle.dump(readout_dict, open(sequential_readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9c5b4c20",
   "metadata": {},
   "source": [
    "## rerun assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 361,
   "id": "a5a1ca99",
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "# params\n",
    "assemble_params = {\n",
    "    'primer_len':20,\n",
    "    'num_readout_per_probe':4, \n",
    "}\n",
    "sequential_assemble_folder = os.path.join(library_folder, 'Sequential')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 362,
   "id": "e5903806",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['u']\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-65\n",
      "--- assemblying 16 probes in region: Chr7-6Mb-seg-75\n",
      "--- assemblying 14 probes in region: Chr7-6Mb-seg-85\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-95\n",
      "--- assemblying 15 probes in region: Chr7-6Mb-seg-105\n",
      "--- assemblying 15 probes in region: Chr7-6Mb-seg-135\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-166\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-176\n",
      "--- assemblying 13 probes in region: Chr7-6Mb-seg-196\n",
      "--- assemblying 18 probes in region: Chr7-6Mb-seg-206\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-216\n",
      "--- assemblying 21 probes in region: Chr7-6Mb-seg-226\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-236\n",
      "--- assemblying 13 probes in region: Chr7-6Mb-seg-286\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-326\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-336\n",
      "--- assemblying 18 probes in region: Chr7-6Mb-seg-346\n",
      "--- assemblying 22 probes in region: Chr7-6Mb-seg-366\n",
      "--- assemblying 14 probes in region: Chr7-6Mb-seg-396\n",
      "--- assemblying 15 probes in region: Chr7-6Mb-seg-416\n",
      "--- assemblying 22 probes in region: Chr7-6Mb-seg-426\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-436\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-446\n",
      "--- assemblying 15 probes in region: Chr7-6Mb-seg-456\n",
      "--- assemblying 17 probes in region: Chr7-6Mb-seg-466\n",
      "--- assemblying 30 probes in region: Chr7-6Mb-seg-476\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-486\n",
      "--- assemblying 20 probes in region: Chr7-6Mb-seg-496\n",
      "--- assemblying 13 probes in region: Chr7-6Mb-seg-506\n",
      "--- assemblying 17 probes in region: Chr7-6Mb-seg-536\n",
      "--- assemblying 13 probes in region: Chr7-6Mb-seg-556\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-52\n",
      "--- assemblying 28 probes in region: Chr7-6Mb-seg-62\n",
      "--- assemblying 14 probes in region: Chr7-6Mb-seg-72\n",
      "--- assemblying 17 probes in region: Chr7-6Mb-seg-82\n",
      "--- assemblying 22 probes in region: Chr7-6Mb-seg-102\n",
      "--- assemblying 21 probes in region: Chr7-6Mb-seg-122\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-132\n",
      "--- assemblying 15 probes in region: Chr7-6Mb-seg-163\n",
      "--- assemblying 30 probes in region: Chr7-6Mb-seg-173\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-203\n",
      "--- assemblying 13 probes in region: Chr7-6Mb-seg-233\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-253\n",
      "--- assemblying 14 probes in region: Chr7-6Mb-seg-263\n",
      "--- assemblying 17 probes in region: Chr7-6Mb-seg-273\n",
      "--- assemblying 14 probes in region: Chr7-6Mb-seg-283\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-343\n",
      "--- assemblying 29 probes in region: Chr7-6Mb-seg-363\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-373\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-413\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-423\n",
      "--- assemblying 13 probes in region: Chr7-6Mb-seg-433\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-443\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-453\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-463\n",
      "--- assemblying 14 probes in region: Chr7-6Mb-seg-473\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-483\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-493\n",
      "--- assemblying 16 probes in region: Chr7-6Mb-seg-513\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-523\n",
      "--- assemblying 14 probes in region: Chr7-6Mb-seg-533\n",
      "--- assemblying 14 probes in region: Chr7-6Mb-seg-53\n",
      "--- assemblying 20 probes in region: Chr7-6Mb-seg-63\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-93\n",
      "--- assemblying 16 probes in region: Chr7-6Mb-seg-123\n",
      "--- assemblying 16 probes in region: Chr7-6Mb-seg-133\n",
      "--- assemblying 22 probes in region: Chr7-6Mb-seg-164\n",
      "--- assemblying 28 probes in region: Chr7-6Mb-seg-174\n",
      "--- assemblying 15 probes in region: Chr7-6Mb-seg-184\n",
      "--- assemblying 20 probes in region: Chr7-6Mb-seg-204\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-234\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-274\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-284\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-304\n",
      "--- assemblying 27 probes in region: Chr7-6Mb-seg-324\n",
      "--- assemblying 21 probes in region: Chr7-6Mb-seg-344\n",
      "--- assemblying 26 probes in region: Chr7-6Mb-seg-364\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-374\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-384\n",
      "--- assemblying 24 probes in region: Chr7-6Mb-seg-394\n",
      "--- assemblying 21 probes in region: Chr7-6Mb-seg-404\n",
      "--- assemblying 32 probes in region: Chr7-6Mb-seg-424\n",
      "--- assemblying 13 probes in region: Chr7-6Mb-seg-434\n",
      "--- assemblying 28 probes in region: Chr7-6Mb-seg-444\n",
      "--- assemblying 15 probes in region: Chr7-6Mb-seg-454\n",
      "--- assemblying 13 probes in region: Chr7-6Mb-seg-464\n",
      "--- assemblying 25 probes in region: Chr7-6Mb-seg-474\n",
      "--- assemblying 23 probes in region: Chr7-6Mb-seg-484\n",
      "--- assemblying 21 probes in region: Chr7-6Mb-seg-494\n",
      "--- assemblying 13 probes in region: Chr7-6Mb-seg-504\n",
      "--- assemblying 19 probes in region: Chr7-6Mb-seg-534\n",
      "--- assemblying 17 probes in region: Chr7-6Mb-seg-544\n",
      "-- 1852 probes assembled in total.\n",
      "-- saving 1852 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\Sequential\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\Sequential\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "# run assembly\n",
    "\n",
    "cand_sequential_probes, readout_summary = Assemble_probes(\n",
    "    library_folder, sel_sequential_pb_dict, \n",
    "    sequential_region_2_readout_dict, readout_dict, primers, \n",
    "    primer_len=assemble_params['primer_len'], \n",
    "    num_readout_per_probe=assemble_params['num_readout_per_probe'], \n",
    "    unique_readout_per_probe=True,\n",
    "    save_folder=sequential_assemble_folder,\n",
    "    rc_targets=False, overwrite=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 369,
   "id": "5906d149",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7t3QuUXWV9N/5fIjcBbSuEi4ZLEGkEgy8qiVg0sS7AVlDBhdYFiwZpQWko4B28kEgFLxiLlNKqfW3+oqWgQETjhf4bEJVIYqGCIpCU0BTCpSJEiBFi5l3PwYnJ5MycfXn27BnO56zFUpjnefben/3dJ9/ZOZcJAwMDA+FBgAABAgQIECBAoE8EJijAfXKmHSYBAgQIECBAgEBHQAEWBAIECBAgQIAAgb4SUID76nQ7WAIECBAgQIAAAQVYBggQIECAAAECBPpKQAHuq9PtYAkQIECAAAECBBRgGSBAgAABAgQIEOgrAQW4r063gyVAgAABAgQIEFCAZYAAAQIECBAgQKCvBBTgvjrdDpYAAQIECBAgQEABlgECBAgQIECAAIG+ElCA++p0O1gCBAgQIECAAAEFWAYIECBAgAABAgT6SkAB7qvT7WAJECBAgAABAgQUYBkgQIAAAQIECBDoKwEFuK9Ot4MlQIAAAQIECBBQgGWAAAECBAgQIECgrwQU4L463Q6WAAECBAgQIEBAAZYBAgQIECBAgACBvhJQgPvqdDtYAgQIECBAgAABBVgGCBAgQIAAAQIE+kpAAe6r0+1gCRAgQIAAAQIEFGAZIECAAAECBAgQ6CsBBbivTreDJUCAAAECBAgQUIBlgAABAgQIECBAoK8EFOC+Ot0OlgABAgQIECBAQAGWAQIECBAgQIAAgb4SUID76nQ7WAIECBAgQIAAAQVYBggQIECAAAECBPpKQAHuq9PtYAkQIECAAAECBBRgGSBAgAABAgQIEOgrAQW4r063gyVAgAABAgQIEFCAZYAAAQIECBAgQKCvBBTgvjrdDpYAAQIECBAgQEABlgECBAgQIECAAIG+ElCA++p0O1gCBAgQIECAAAEFWAYIECBAgAABAgT6SkAB7qvT7WAJECBAgAABAgQUYBkgQIAAAQIECBDoKwEFuK9Ot4MlQIAAAQIECBBQgGWAAAECBAgQIECgrwQU4L463Q6WAAECBAgQIEBAAZYBAgQIECBAgACBvhJQgPvqdDtYAgQIECBAgAABBVgGCBAgQIAAAQIE+kpAAe6r0+1gCRAgQIAAAQIEFGAZIECAAAECBAgQ6CsBBbivTreDJUCAAAECBAgQUIBlgAABAgQIECBAoK8EFOC+Ot0OlgABAgQIECBAQAGWAQIECBAgQIAAgb4SUID76nQ7WAIECBAgQIAAAQVYBggQIECAAAECBPpKQAHuq9PtYAkQIECAAAECBBRgGSBAgAABAgQIEOgrAQW4r063gyVAgAABAgQIEFCAZYAAAQIECBAgQKCvBBTgvjrdDpYAAQIECBAgQEABlgECBAgQIECAAIG+ElCA++p0O1gCBAgQIECAAAEFWAYIECBAgAABAgT6SkAB7qvT7WAJECBAgAABAgQU4AYzsG7durj11ltj0qRJsdVWWzW4JUsTIECAAAECBDYXWL9+fTz00EMxbdq02G677fBsIqAANxiHpUuXxvTp0xvcgqUJECBAgAABAiML3HTTTXHwwQdjUoBHJwMrV66MKVOmRAre7rvvPjobtRUCBAgQIECAQESsXr26cyPu7rvvjr333puJAjw6Gfif//mf2GOPPWLVqlUxefLk0dmorRAgQIAAAQIEIkIPGT4GXgLR4CUieA3iWpoAAQIECBAYUUAPUYBbuUQErxV2GyVAgAABAgTcAR4xA+4AN3iJKMAN4lqaAAECBAgQcAe4YgYU4IpwRaYpwEWUjCFAgAABAgSaENBDhldVgJtI3G/XFLwGcS1NgAABAgQIuANcMQMKcEW4ItMU4CJKxhAgQIAAAQJNCOgh7gA3kaueawpeTyIDCBAgQIAAgYYE9BAFuKFojbys4LXCbqMECBAgQICAT4EYMQNeAtHgJaIAN4hraQIECBAgQGBEAT3EHeBWLhHBa4XdRgkQIECAAAF3gPvzDvAVV1wRX/rSl+JHP/pRPPzww/H85z8/3vGOd8Qpp5wSEydOHBFlwYIFcf7558fKlStj3333jXPOOSeOPfbY0heTAlyazAQCBAgQIEAgk4Ae0od3gF/+8pfHXnvtFUcffXTsuuuusXjx4k6pPeOMM+KTn/zksCJf+cpXOmX3/e9/fxx++OFx9dVXx0UXXRTf+ta3Ov9e5iF4ZbSMJUCAQJ8KPPF4xI8vf+qfxx+K2GFSxIFvfuqfbXYYHZRu+7D/GyImTIj4ydXd92u4/Z56ZMTPvr758XRba6T101FvavLM50T8wV4Rj9wTsfbhp4yGzu82Jrdjr3PV6+fpuIqMyXTW9ZA+LMAPPfRQTJo0abMjf+c73xmXXHJJPPLII7Htttt2VXnhC18Y06ZNi8svv3zjz4844oh49NFHY8mSJaUiKXiluAwmQIBA/wn8710R/98bItbcGzFhYsTAht/977OfF3HCwoidX9CsS7d9iAkRMfDb7f72/w/uX9qvoy6MuOb07vs99Di6rTXS+jvs8lTxfuyB31lsIdBl/4aO2XR/czj2OlcjmQyey7SPo3i+9ZA+LMDdDvmLX/xinHDCCXHffffF7rvvvsWQu+++O/bZZ5+48sorO3eOBx/pJREnnnhiPPjgg7HzzjsXfiISvMJUBhIgQKD/BNKdwL87OOKXq58qvt0K3LN2j5iztLk7wb32oetZmfBUQU2Pbvs91s5kKsJ1HXs69TBJ+7DjrhGp+D92/6idbz1EAe4InHzyyfHVr361U2Sf8YxnbKGyaNGieN3rXhe33357TJ06dePPly5dGtOnT48bbrghDj300MKXtuAVpjKQAAEC/Sew7AsRXz+j93GnO4svnd17XJURRfehytpjbU4dx9F0qrOfQ8z1EAU4li1bFoccckjnDW0f/OAHu4qkN80df/zxsXr16thtt902jlm+fHm84AUviIULF8brX//6YTXXrFkT6Z/BR1onFedVq1bF5MmTx9pTgf0hQIAAgTYF/u+fRKxaMvJd1HTncM9DIk5c1MyeFtmHZrY8uqvWdRwtp7r7qQAXzlVffA7w/fffHzNmzOiU0Ouuuy623nrrEQtwGp/eODf4uOuuu2K//faLr33ta3HUUUcNizt37tyYN2/eFj9XgAvn0UACBAj0j8BFL4v4+V29j3enF0Sctqz3uCojiu5DlbXH2pw6jqPpVGc/FeDCqXvaF+D05rVZs2bFunXr4nvf+17stNNOw+LUfQmEO8CFc2cgAQIECBS5q5j5juAW6EX24elwpuo6jpZT3f1UgAun9WldgFPpTZ/gkO7g3njjjZ2PRRvp4U1whXNjIAECBAjUFSj6utKMrwndYpeL7kPdYx0L8+s4jqZTnf1UgAsn7WlbgNevXx/HHHNMfPe73+38c+CBBxZCSR+D9uIXvzguu+yyjeNf+9rXdj46zcegFSI0iAABAgSKCPT6ZIEcn17Qaz967UPX+T4FYksWnwLRK2pj7edP2wKcvvHts5/9bHziE5+IV77ylZu577///vHsZz87TjrppEgfcZbK8uAjfYPcW97yljjrrLPisMMO67zx7cILL/RFGGMtufaHAAECTweBXp8tm+Pza3s5+RzgXkJP/bzXufI5wMUcx8iop20B3nvvveOee+7pypy+FS69Lnj27NmdAjwwMPhh308NT//tvPPO2/hVyOnNbb4KeYwk1m4QIEDg6SaQ7sLeesVT33z22IMRO+7y1LfATTu2uc//HWrYbR/SN62lz6396dXd92u4/f7D10Xc8Y3Nj6fbWiOtn/ZvU5Ptd3rqm+B+kb4J7udPGQ2d321Mbsde56rXz9NxFRmTKeM+Bm14yKdtAc6UnVrLCF4tPpMJECBAgACBGgJ6iAJcIz7VpwpedTszCRAgQIAAgXoCeogCXC9BFWcLXkU40wgQIECAAIHaAnqIAlw7RFUWELwqauYQIECAAAECOQT0EAU4R45KryF4pclMIECAAAECBDIJ6CEKcKYolVtG8Mp5GU2AAAECBAjkE9BDFOB8aSqxkuCVwDKUAAECBAgQyCqghyjAWQNVdDHBKyplHAECBAgQIJBbQA9RgHNnqtB6gleIySACBAgQIECgAQE9RAFuIFa9lxS83kZGECBAgAABAs0I6CEKcDPJ6rGq4LXCbqMECBAgQIBAROghCnArF4LgtcJuowQIECBAgIACPGIGJgwMDAxISTMCCnAzrlYlQIAAAQIEegvoIe4A905JAyMErwFUSxIgQIAAAQKFBPQQBbhQUHIPErzcotYjQIAAAQIEigroIQpw0axkHSd4WTktRoAAAQIECJQQ0EMU4BJxyTdU8PJZWokAAQIECBAoJ6CHKMDlEpNptOBlgrQMAQIECBAgUFpAD1GAS4cmxwTBy6FoDQIECBAgQKCKgB6iAFfJTe05gleb0AIECBAgQIBARQE9RAGuGJ160wSvnp/ZBAgQIECAQHUBPUQBrp6eGjMFrwaeqQQIECBAgEAtAT1EAa4VoKqTBa+qnHkECBAgQIBAXQE9RAGum6FK8wWvEptJBAgQIECAQAYBPUQBzhCj8ksIXnkzMwgQIECAAIE8AnqIApwnSSVXEbySYIYTIECAAAEC2QT0EAU4W5jKLCR4ZbSMJUCAAAECBHIK6CEKcM48FV5L8ApTGUiAAAECBAhkFtBDFODMkSq2nOAVczKKAAECBAgQyC+ghyjA+VNVYEXBK4BkCAECBAgQINCIgB6iADcSrF6LCl4vIT8nQIAAAQIEmhLQQxTgprI14rqC1wq7jRIgQIAAAQIRoYcowK1cCILXCruNEiBAgAABAgrwiBmYMDAwMCAlzQgowM24WpUAAQIECBDoLaCHuAPcOyUNjBC8BlAtSYAAAQIECBQS0EMU4EJByT1I8HKLWo8AAQIECBAoKqCHKMBFs5J1nOBl5bQYAQIECBAgUEJAD1GAS8Ql31DBy2dpJQIECBAgQKCcgB6iAJdLTKbRgpcJ0jIECBAgQIBAaQE9RAEuHZocEwQvh6I1CBAgQIAAgSoCeogCXCU3tecIXm1CCxAgQIAAAQIVBfQQBbhidOpNE7x6fmYTIECAAAEC1QX0EAW4enpqzBS8GnimEiBAgAABArUE9BAFuFaAqk4WvKpy5hEgQIAAAQJ1BfQQBbhuhirNF7xKbCYRIECAAAECGQT0EAU4Q4zKLyF45c3MIECAAAECBPII6CEKcJ4klVxF8EqCGU6AAAECBAhkE9BDFOBsYSqzkOCV0TKWAAECBAgQyCmghyjAOfNUeC3BK0xlIAECBAgQIJBZQA9RgDNHqthyglfMySgCBAgQIEAgv4AeogDnT1WBFQWvAJIhBAgQIECAQCMCeogC3Eiwei0qeL2E/JwAAQIECBBoSkAPUYCbytaI6wpeK+w2SoAAAQIECESEHqIAt3IhCF4r7DZKgAABAgQIKMAjZmDCwMDAgJQ0I6AAN+NqVQIECBAgQKC3gB7iDnDvlDQwQvAaQLUkAQIECBAgUEhAD1GACwUl9yDByy1qPQIECBAgQKCogB6iABfNStZxgpeV02IECBAgQIBACQE9RAEuEZd8QwUvn6WVCBAgQIAAgXICeogCXC4xmUYLXiZIyxAgQIAAAQKlBfQQBbh0aHJMELwcitYgQIAAAQIEqgjoIQpwldzUniN4tQktQIAAAQIECFQU0EP6tAAvX748LrjggliyZEncdtttMXXq1M7/9no8/vjjce6558YVV1wRq1evjuc973lx3HHHxVlnnRXbbrttr+kbfy54hakMJECAAAECBDIL6CF9WoAXLlwYc+bMiRkzZsSdd94ZGzZsKFSATzjhhLj66qvjox/9aLzoRS+Km266KT70oQ/F29/+9vjMZz5TOJ6CV5jKQAIECBAgQCCzgB7SpwU4Fd6JEyd2jn727NmxbNmyngV4/fr18axnPSve+973xrx58zbKnXrqqfHVr341HnjggcLxFLzCVAYSIECAAAECmQX0kD4twJsedtEC/OSTT8b2228fH//4x+Od73znxiXOPvvs+PznPx8PPvhg4XgKXmEqAwkQIECAAIHMAnqIAlz4DnCiOvnkk+Paa6+Nyy67LA444IBYunRpvPnNb47TTjstPvzhDxeOp+AVpjKQAAECBAgQyCyghyjApQrwb37zm87rfdMd38FHKr+9Xv+7Zs2aSP8MPtIb6KZPnx6rVq2KyZMnZ4615QgQIECAAAECwwsowApwqQL8nve8Jy699NL4yEc+En/4h38YP/rRj+Kcc86JM888c7PXBQ9lnTt3btefK8CenggQIECAAIHRFlCAFeDCBTh9TNq0adMifYLE61//+o1yF154Ybz73e+Oe++9N3bZZZeuou4Aj/albXsECBAgQIDAcAIKsAJcuABffvnl8Za3vCXuueee2HPPPTfKXX/99TFr1qzOR6IdfPDBha42wSvEZBABAgQIECDQgIAeogAXLsA//OEP4+Uvf3lceeWVcfTRR2+U+9SnPtW5A5w+BWLSpEmFYip4hZgMIkCAAAECBBoQ0EP6tACvXbs2Fi1a1Dn6iy++OFasWBHz58/v/PvMmTM7Rfakk06KBQsWRPr83/RIb4B7xSteEStXruy8nje9Bjh9CkR6PfCRRx7Z+WSIog/BKyplHAECBAgQIJBbQA/p0wKcSuyUKVO6Hv3ixYs7L2lInw+cCvDAwMDGcekub/rmt+985ztx//33xx577BFvetOb4gMf+EDsuOOOhfMpeIWpDCRAgAABAgQyC+ghfVqAM+eo9HKCV5rMBAIECBAgQCCTgB6iAGeKUrllBK+cl9EECBAgQIBAPgE9RAHOl6YSKwleCSxDCRAgQIAAgawCeogCnDVQRRcTvKJSxhEgQIAAAQK5BfQQBTh3pgqtJ3iFmAwiQIAAAQIEGhDQQxTgBmLVe0nB621kBAECBAgQINCMgB6iADeTrB6rCl4r7DZKgAABAgQIRIQeogC3ciEIXivsNkqAAAECBAgowCNmYMLApt8AIS5ZBRTgrJwWI0CAAAECBEoI6CHuAJeIS76hgpfP0koECBAgQIBAOQE9RAEul5hMowUvE6RlCBAgQIAAgdICeogCXDo0OSYIXg5FaxAgQIAAAQJVBPQQBbhKbmrPEbzahBYgQIAAAQIEKgroIQpwxejUmyZ49fzMJkCAAAECBKoL6CEKcPX01JgpeDXwTCVAgAABAgRqCeghCnCtAFWdLHhV5cwjQIAAAQIE6groIQpw3QxVmi94ldhMIkCAAAECBDII6CEKcIYYlV9C8MqbmUGAAAECBAjkEdBDFOA8SSq5iuCVBDOcAAECBAgQyCaghyjA2cJUZiHBK6NlLAECBAgQIJBTQA9RgHPmqfBagleYykACBAgQIEAgs4AeogBnjlSx5QSvmJNRBAgQIECAQH4BPUQBzp+qAisKXgEkQwgQIECAAIFGBPQQBbiRYPVaVPB6Cfk5AQIECBAg0JSAHqIAN5WtEdcVvFbYbZQAAQIECBCICD1EAW7lQhC8VthtlAABAgQIEFCAR8zAhIGBgQEpaUZAAW7G1aoECBAgQIBAbwE9xB3g3ilpYITgNYBqSQIECBAgQKCQgB6iABcKSu5Bgpdb1HoECBAgQIBAUQE9RAEumpWs4wQvK6fFCBAgQIAAgRICeogCXCIu+YYKXj5LKxEgQIAAAQLlBPQQBbhcYjKNFrxMkJYhQIAAAQIESgvoIQpw6dDkmCB4ORStQYAAAQIECFQR0EMU4Cq5qT1H8GoTWoAAAQIECBCoKKCHKMAVo1NvmuDV8zObAAECBAgQqC6ghyjA1dNTY6bg1cAzlQABAgQIEKgloIcowLUCVHWy4FWVM48AAQIECBCoK6CHKMB1M1RpvuBVYjOJAAECBAgQyCCghyjAGWJUfgnBK29mBgECBAgQIJBHQA9RgPMkqeQqglcSzHACBAgQIEAgm4AeogBnC1OZhQSvjJaxBAgQIECAQE4BPUQBzpmnwmsJXmEqAwkQIECAAIHMAnqIApw5UsWWE7xiTkYRIECAAAEC+QX0EAU4f6oKrCh4BZAMIUCAAAECBBoR0EMU4EaC1WtRwesl5OcECBAgQIBAUwJ6iALcVLZGXFfwWmG3UQIECBAgQCAi9BAFuJULQfBaYbdRAgQIECBAQAEeMQMTBgYGBqSkGQEFuBlXqxIgQIAAAQK9BfQQd4B7p6SBEYLXAKolCRAgQIAAgUICeogCXCgouQcJXm5R6xEgQIAAAQJFBfQQBbhoVrKOE7ysnBYjQIAAAQIESgjoIQpwibjkGyp4+SytRIAAAQIECJQT0EMU4HKJyTRa8DJBWoYAAQIECBAoLaCHKMClQ5NjguDlULQGAQIECBAgUEVAD1GAq+Sm9hzBq01oAQIECBAgQKCigB6iAFeMTr1pglfPz2wCBAgQIECguoAeogBXT0+NmYJXA89UAgQIECBAoJaAHqIA1wpQ1cmCV1XOPAIECBAgQKCugB6iANfNUKX5gleJzSQCBAgQIEAgg4AeogBniFH5JQSvvJkZBAgQIECAQB4BPUQBzpOkkqsIXkkwwwkQIECAAIFsAnqIApwtTGUWErwyWsYSIECAAAECOQX0kD4twMuXL48LLrgglixZErfddltMnTq1879FHg8//HB88IMfjKuuuip+8YtfxJ577hnvete74pRTTikyvTNG8ApTGUiAAAECBAhkFtBD+rQAL1y4MObMmRMzZsyIO++8MzZs2FCoAD/22GNxyCGHxDOf+cx497vfHbvsskvcdddd8eSTT8app55aOJ6CV5jKQAIECBAgQCCzgB7SpwU4Fd6JEyd2jn727NmxbNmyQgX47LPPjssvvzxuvfXWTgmu+hC8qnLmESBAgAABAnUF9JA+LcCbHnaZArzbbrvF6aefHmeddVat7AleLT6TCRAgQIAAgRoCeogCXPgO8N133x377LNPXHLJJfH1r389rr322thxxx3jz/7szzqvJy5zR1jwaly1phIgQIAAAQK1BPQQBbhwAb7xxhvjFa94Raf0HnvssXHcccfFT3/6087d4Le+9a3xuc99bljNNWvWRPpn8LF69eqYPn16rFq1KiZPnlwrxCYTIECAAAECBMoIKMAKcOEC/P3vfz8OPfTQOPjgg+Omm27aKDd//vx4z3veE/fee2+kl0h0e8ydOzfmzZu3xY8U4DKXq7EECBAgQIBADgEFWAEuXIBvv/322H///eN973tffOxjH9sod8stt8RBBx0U//7v/x6vfvWru4q6A5zjcrUGAQIECBAgkENAAVaACxfgJ554Ip71rGfFmWeeuVkBvvnmm+MlL3lJXHfddTFz5sxCuRS8QkwGESBAgAABAg0I6CEKcOECnKiOPPLIeOCBB2Lp0qUb5dIb4NLrgO+7776YNGlSoZgKXiEmgwgQIECAAIEGBPSQPi3Aa9eujUWLFnWO/uKLL44VK1ZEei1veqS7uKnInnTSSbFgwYJYv379RqX02t/0OuD0yQ/HH398501wH/jAB+Lkk0+OT3/604UjKniFqQwkQIAAAQIEMgvoIX1agFeuXBlTpkzpevSLFy+OWbNmde4MpwI8MDCw2bj08Wfpjm/6MoyddtopTjjhhDj33HNj6623LhxPwStMZSABAgQIECCQWUAP6dMCnDlHpZcTvNJkJhAgQIAAAQKZBPQQBThTlMotI3jlvIwmQIAAAQIE8gnoIQpwvjSVWEnwSmAZSoAAAQIECGQV0EMU4KyBKrqY4BWVMo4AAQIECBDILaCHKMC5M1VoPcErxGQQAQIECBAg0ICAHqIANxCr3ksKXm8jIwgQIECAAIFmBPQQBbiZZPVYVfBaYbdRAgQIECBAICL0EAW4lQtB8Fpht1ECBAgQIEBAAR4xAxMGhn4DhMhkE1CAs1FaiAABAgQIECgpoIe4A1wyMnmGC14eR6sQIECAAAEC5QX0EAW4fGoyzBC8DIiWIECAAAECBCoJ6CEKcKXg1J0keHUFzSdAgAABAgSqCughCnDV7NSaJ3i1+EwmQIAAAQIEagjoIQpwjfhUnyp41e3MJECAAAECBOoJ6CEKcL0EVZwteBXhTCNAgAABAgRqC+ghCnDtEFVZQPCqqJlDgAABAgQI5BDQQxTgHDkqvYbglSYzgQABAgQIEMgkoIcowJmiVG4ZwSvnZTQBAgQIECCQT0APUYDzpanESoJXAstQAgQIECBAIKuAHqIAZw1U0cUEr6iUcQQIECBAgEBuAT1EAc6dqULrCV4hJoMIECBAgACBBgT0EAW4gVj1XlLwehsZQYAAAQIECDQjoIcowM0kq8eqgtcKu40SIECAAAECEaGHKMCtXAiC1wq7jRIgQIAAAQIK8IgZmDAwMDAgJc0IKMDNuFqVAAECBAgQ6C2gh7gD3DslDYwQvAZQLUmAAAECBAgUEtBDFOBCQck9SPByi1qPAAECBAgQKCqghyjARbOSdZzgZeW0GAECBAgQIFBCQA9RgEvEJd9QwctnaSUCBAgQIECgnIAeogCXS0ym0YKXCdIyBAgQIECAQGkBPUQBLh2aHBMEL4eiNQgQIECAAIEqAnqIAlwlN7XnCF5tQgsQIECAAAECFQX0EAW4YnTqTRO8en5mEyBAgAABAtUF9BAFuHp6aswUvBp4phIgQIAAAQK1BPQQBbhWgKpOFryqcuYRIECAAAECdQX0EAW4boYqzRe8SmwmESBAgAABAhkE9BAFOEOMyi8heOXNzCBAgAABAgTyCOghCnCeJJVcRfBKghlOgAABAgQIZBPQQxTgbGEqs5DgldEylgABAgQIEMgpoIcowDnzVHgtwStMZSABAgQIECCQWUAPUYAzR6rYcoJXzMkoAgQIECBAIL+AHqIA509VgRUFrwCSIQQIECBAgEAjAnqIAtxIsHotKni9hPycAAECBAgQaEpAD1GAm8rWiOsKXivsNkqAAAECBAisXfwJAAAgAElEQVREhB6iALdyIQheK+w2SoAAAQIECCjAI2ZgwsDAwICUNCOgADfjalUCBAgQIECgt4Ae4g5w75Q0MELwGkC1JAECBAgQIFBIQA9RgAsFJfcgwcstaj0CBAgQIECgqIAeogAXzUrWcYKXldNiBAgQIECAQAkBPUQBLhGXfEMFL5+llQgQIECAAIFyAnqIAlwuMZlGC14mSMsQIECAAAECpQX0EAW4dGhyTBC8HIrWIECAAAECBKoI6CEKcJXc1J4jeLUJLUCAAAECBAhUFNBDFOCK0ak3TfDq+ZlNgAABAgQIVBfQQxTg6umpMVPwauCZSoAAAQIECNQS0EMU4FoBqjpZ8KrKmUeAAAECBAjUFdBDFOC6Gao0X/AqsZlEgAABAgQIZBDQQxTgDDEqv4TglTczgwABAgQIEMgjoIcowHmSVHIVwSsJZjgBAgQIECCQTUAPUYCzhanMQoJXRstYAgQIECBAIKeAHqIA58xT4bUErzCVgQQIECBAgEBmAT1EAc4cqWLLCV4xJ6MIECBAgACB/AJ6SJ8W4OXLl8cFF1wQS5Ysidtuuy2mTp3a+d8yj6uuuiqOOeaYOOCAA0rPFbwy0sYSIECAAAECOQX0kD4twAsXLow5c+bEjBkz4s4774wNGzaUKrG/+tWvYv/994/0vzvvvHOpuYlc8HJextYiQIAAAQIEygjoIX1agFPhnThxYufoZ8+eHcuWLStVYj/84Q/H9ddfH1OmTCk9VwEuc4kaS4AAAQIECOQWUID7tABvethlC/CKFSviwAMPjB/84Afx6U9/WgHOfVVajwABAgQIEGhUQAFWgEvfAT7yyCNjjz32iEsuuaT03EFuwWv0urY4AQIECBAgMIKAHqIAlyqx11xzTfz5n/9553XD6bW/Re8er1mzJtI/g4/Vq1fH9OnTY9WqVTF58mQXKQECBAgQIEBg1AQUYAW4cIldt25d5xMfzjjjjDjttNM6ckUL8Ny5c2PevHlbaCvAo3at2xABAgQIECDwWwEFWAEuXGI/9rGPxT/90z/FjTfeGFtttVVH7tRTT41bbrml83rg7bffPrbZZpuuou4Ae84hQIAAAQIExoqAAqwAFy7A6W7vggULhhVLrwl++9vfXijbgleIySACBAgQIECgAQE9RAEuXIB/9rOfxf3337+ZWLorfMcdd8QXvvCF2G+//eK5z31uoZgKXiEmgwgQIECAAIEGBPSQPi3Aa9eujUWLFnWO/uKLL4700Wbz58/v/PvMmTNj0qRJcdJJJ3Xu+K5fv35YpaKvAR66gOA1cDVbkgABAgQIECgkoIf0aQFeuXJl50ssuj0WL14cs2bN6twZTgV4YGBAAS50ORlEgAABAgQIjAcBBbhPC3Db4RS8ts+A7RMgQIAAgf4V0EMU4FbSL3itsNsoAQIECBAgEBF6iALcyoUgeK2w2ygBAgQIECCgAI+YgQkDI734VXxqCSjAtfhMJkCAAAECBGoI6CHuANeIT/WpglfdzkwCBAgQIECgnoAeogDXS1DF2YJXEc40AgQIECBAoLaAHqIA1w5RlQUEr4qaOQQIECBAgEAOAT1EAc6Ro9JrCF5pMhMIECBAgACBTAJ6iAKcKUrllhG8cl5GEyBAgAABAvkE9BAFOF+aSqwkeCWwDCVAgAABAgSyCughCnDWQBVdTPCKShlHgAABAgQI5BbQQxTg3JkqtJ7gFWIyiAABAgQIEGhAQA9RgBuIVe8lBa+3kREECBAgQIBAMwJ6iALcTLJ6rCp4rbDbKAECBAgQIOCrkEfMgK9CbvASUYAbxLU0AQIECBAgMKKAHuIOcCuXiOC1wm6jBAgQIECAgDvA7gC3dRUowG3J2y4BAgQIECCgh7gD3MpVIHitsNsoAQIECBAg4A6wO8BtXQUKcFvytkuAAAECBAjoIe4At3IVCF4r7DZKgAABAgQIuAPsDnBbV4EC3Ja87RIgQIAAAQJ6iDvArVwFgtcKu40SIECAAAEC7gC7A9zWVaAAtyVvuwQIECBAgIAe4g5wK1eB4LXCbqMECBAgQICAO8DuALd1FSjAbcnbLgECBAgQIKCHuAPcylUgeK2w2ygBAgQIECDgDrA7wG1dBQpwW/K2S4AAAQIECOgh7gC3chUIXivsNkqAAAECBAi4A+wOcFtXgQLclrztEiBAgAABAnqIO8CtXAWC1wq7jRIgQIAAAQLuALsD3NZVoAC3JW+7BAgQIECAgB7iDnArV4HgtcJuowQIECBAgIA7wO4At3UVKMBtydsuAQIECBAgoIe4A9zKVSB4rbDbKAECBAgQIOAOsDvAbV0FCnBb8rZLgAABAgQI6CHuALdyFQheK+w2SoAAAQIECLgD7A5wW1eBAtyWvO0SIECAAAECeog7wK1cBYLXCruNEiBAgAABAu4AuwPc1lWgALclb7sECBAgQICAHuIOcCtXgeC1wm6jBAgQIECAgDvA7gC3dRUowG3J2y4BAgQIECCgh7gD3MpVIHitsNsoAQIECBAg4A6wO8BtXQUKcFvytkuAAAECBAjoIe4At3IVCF4r7DZKgAABAgQIuAPsDnBbV4EC3Ja87RIgQIAAAQJ6iDvArVwFgtcKu40SIECAAAEC7gC7A9zWVaAAtyVvuwQIECBAgIAe4g5wK1eB4LXCbqMECBAgQICAO8DuALd1FSjAbcnbLgECBAgQIKCHuAPcylUgeK2w2ygBAgQIECDgDrA7wG1dBQpwW/K2S4AAAQIECOgh7gC3chUIXivsNkqAAAECBAi4A+wOcFtXgQLclrztEiBAgAABAnqIO8CtXAWC1wq7jRIgQIAAAQLuALsD3NZVoAC3JW+7BAgQIECAgB7iDnArV4HgtcJuowQIECBAgIA7wO4At3UVKMBtydsuAQIECBAgoIe4A9zKVSB4rbDbKAECBAgQIOAOsDvAbV0FCnBb8rZLgAABAgQI6CHuALdyFQheK+w2SoAAAQIECLgD7A5wW1eBAtyWvO0SIECAAAECeog7wK1cBYLXCruNEiBAgAABAu4A9+8d4OXLl8cFF1wQS5Ysidtuuy2mTp3a+d+RHmvWrIn58+fHN7/5zbjjjjti6623jpe+9KVx3nnnxUte8pJSF5QCXIrLYAIECBAgQCCjgB7Sp3eAFy5cGHPmzIkZM2bEnXfeGRs2bOhZgFNBPuyww+Jtb3tbvOpVr4onn3wyLrzwwrjhhhviBz/4QakSLHgZr2JLESBAgAABAqUE9JA+LcCp8E6cOLFz9LNnz45ly5b1LMCPP/54TJgwIbbffvuNauvWrYt99tknjjjiiPjCF75QOHyCV5jKQAIECBAgQCCzgB7SpwV408MuWoCHo3rNa14TW221VXz7298uHE/BK0xlIAECBAgQIJBZQA9RgAvfAe5Gle4K77HHHnHCCSfE3/7t3xaOp+AVpjKQAAECBAgQyCyghyjAtQrwmWeeGZdccknn5RP77rvvsJrpDXTpn8HH6tWrY/r06bFq1aqYPHly5lhbjgABAgQIECAwvIACrABXLsBf/vKX47jjjouLL744Tj311BGvs7lz58a8efO2GKMAe3oiQIAAAQIERltAAVaAKxXga6+9No488sg4/fTT4xOf+ETP3LoD3JPIAAIECBAgQGCUBBRgBbh0Ab7pppsivfHtDW94Q3zxi1/sfDJE2YfglRUzngABAgQIEMgloIcowKUK8O233x6vfOUr42Uve1lcc801nS/DqPIQvCpq5hAgQIAAAQI5BPSQPi3Aa9eujUWLFnWOPr2Gd8WKFZ1veUuPmTNnxqRJk+Kkk06KBQsWxPr16zv//cEHH+wU3/QFGJdeemnssMMOG/W23XbbOOiggwpnUvAKUxlIgAABAgQIZBbQQ/q0AK9cuTKmTJnS9egXL14cs2bN6twZTgV4YGCgM+66666LV7/61V3n7LXXXpHWLPoQvKJSxhEgQIAAAQK5BfSQPi3AuYNUdj3BKytmPAECBAgQIJBLQA9RgHNlqdQ6gleKy2ACBAgQIEAgo4AeogBnjFPxpQSvuJWRBAgQIECAQF4BPUQBzpuogqsJXkEowwgQIECAAIHsAnqIApw9VEUWFLwiSsYQIECAAAECTQjoIQpwE7nquabg9SQygAABAgQIEGhIQA9RgBuK1sjLCl4r7DZKgAABAgQIRIQeogC3ciEIXivsNkqAAAECBAgowCNmYMLA4DdAiEp2AQU4O6kFCRAgQIAAgYICeog7wAWjkneY4OX1tBoBAgQIECBQXEAPUYCLpyXjSMHLiGkpAgQIECBAoJSAHqIAlwpMrsGCl0vSOgQIECBAgEBZAT1EAS6bmSzjBS8Lo0UIECBAgACBCgJ6iAJcITb1pwhefUMrECBAgAABAtUE9BAFuFpyas4SvJqAphMgQIAAAQKVBfQQBbhyeOpMFLw6euYSIECAAAECdQT0EAW4Tn4qzxW8ynQmEiBAgAABAjUF9BAFuGaEqk0XvGpuZhEgQIAAAQL1BfQQBbh+iiqsIHgV0EwhQIAAAQIEsgjoIQpwliCVXUTwyooZT4AAAQIECOQS0EMU4FxZKrWO4JXiMpgAAQIECBDIKKCHKMAZ41R8KcErbmUkAQIECBAgkFdAD1GA8yaq4GqCVxDKMAIECBAgQCC7gB6iAGcPVZEFBa+IkjEECBAgQIBAEwJ6iALcRK56ril4PYkMIECAAAECBBoS0EMU4IaiNfKygtcKu40SIECAAAECEaGHKMCtXAiC1wq7jRIgQIAAAQIK8IgZmDAwMDAgJc0IKMDNuFqVAAECBAgQ6C2gh7gD3DslDYwQvAZQLUmAAAECBAgUEtBDFOBCQck9SPByi1qPAAECBAgQKCqghyjARbOSdZzgZeW0GAECBAgQIFBCQA9RgEvEJd9QwctnaSUCBAgQIECgnIAeogCXS0ym0YKXCdIyBAgQIECAQGkBPUQBLh2aHBMEL4eiNQgQIECAAIEqAnqIAlwlN7XnCF5tQgsQIECAAAECFQX0EAW4YnTqTRO8en5mEyBAgAABAtUF9BAFuHp6aswUvBp4phIgQIAAAQK1BPQQBbhWgKpOFryqcuYRIECAAAECdQX0EAW4boYqzRe8SmwmESBAgAABAhkE9BAFOEOMyi8heOXNzCBAgAABAgTyCOghCnCeJJVcRfBKghlOgAABAgQIZBPQQxTgbGEqs5DgldEylgABAgQIEMgpoIcowDnzVHgtwStMZSABAgQIECCQWUAPUYAzR6rYcoJXzMkoAgQIECBAIL+AHqIA509VgRUFrwCSIQQIECBAgEAjAnqIAtxIsHotKni9hPycAAECBAgQaEpAD1GAm8rWiOsKXivsNkqAAAECBAhEhB6iALdyIQheK+w2SoAAAQIECCjAI2ZgwsDAwICUNCOgADfjalUCBAgQIECgt4Ae4g5w75Q0MELwGkC1JAECBAgQIFBIQA9RgAsFJfcgwcstaj0CBAgQIECgqIAeogAXzUrWcYKXldNiBAgQIECAQAkBPUQBLhGXfEMFL5+llQgQIECAAIFyAnqIAlwuMZlGC14mSMsQIECAAAECpQX0EAW4dGhyTBC8HIrWIECAAAECBKoI6CEKcJXc1J4jeLUJLUCAAAECBAhUFNBDFOCK0ak3TfDq+ZlNgAABAgQIVBfQQxTg6umpMVPwauCZSoAAAQIECNQS0EMU4FoBqjpZ8KrKmUeAAAECBAjUFdBDFOC6Gao0X/AqsZlEgAABAgQIZBDQQxTgDDEqv4TglTczgwABAgQIEMgjoIf0aQFevnx5XHDBBbFkyZK47bbbYurUqZ3/LfJYsGBBnH/++bFy5crYd99945xzzoljjz22yNSNYwSvFJfBBAgQIECAQEYBPaRPC/DChQtjzpw5MWPGjLjzzjtjw4YNhQrwV77ylU7Zff/73x+HH354XH311XHRRRfFt771rc6/F300Gby1T6yPq2++L666+X/i548/ETvtsE28btpzIyYMxDd+vHrE/3b0QZPjjQc9N7bfZqvotk6vnw/dzu8/c+vY8zk7xKpfPB6/WPtkZ1/SGocfsEt85ycPbtzHoeOKzhvu2Mru5+B+FTn2dI6r+JY5piJGRfa523nvtnY3x+H2dySjquc/1/Y3Pe/pPA3NcNGc9crB0PxWuca65Whw/0f6Wbo2m3r0uuYHt1t0XK/9LLtO0ee2kRyLZLSJ58tueRx6DXfzGsloaE66XbNFjiXHtV7k+ajoc0+v5/Uqx93rGHtltcjPe+W518+rZqTIvnUb02QPqbpPY2XehIGBgYGxsjO59yMV3okTJ3aWnT17dixbtqxQAX7hC18Y06ZNi8svv3zjLh1xxBHx6KOPdu4mF300FbwVDz0Wx3/+h7H60XUxYULE0DM4ISKGntTB/zY4fvff2y7OP2ZanHXlrZut0+vnmx57t+2knw+uMXFCxIaB3/17L7ci84oeR7f97HVsgz+ftOM2nZ1+6Je/Lu079BiHO6Yix7qp5XDna6TjHMm/17krs72ix1wmO0XyeelfzOgsOdK10C1zRXNQ1m9oNrvlaKSMbbpf6dieP2nHXpdM6Z93e+7ott2i43rtQNl1ij631blWt8hrpufL4fLY67yOZNTruajMsQx3nZa51tu8dkbKWpHnsxzXVK889/oztWpGel1nI/28qR5SZ5/GytyndQHeFLloAb777rtjn332iSuvvDKOPvrojUukl0SceOKJ8eCDD8bOO+9c6Pw1Ebz02+VrPnV9PLBmXadcVn2kJ4z0ZJYe3dbp9fOq2809r8p+VpmTe7/Lrjfa+zza2+v2B/lI+UzldNKzto20nw/+8teVroW2j3G4DKRj2/XZ28X//66Znb+lyfXo9dwxuN1rTvujOOqi7w/7HFN0/4pub/A4e43P5VBlnV5ZKZLHbm5j4Zh7HVvXXyJ7/PlRxbjJOUUzO9I+9DpXvRyrZqSuSxM9pO4+jZX5CvCQM7Fo0aJ43eteF7fffnvnNcODj6VLl8b06dPjhhtuiEMPPbTQ+WsieF/+4X/H2VfdWmj7BhEgMH4F0t2kt07fM9sBFH3uOPqg58VVN9/bc7u99q/o9gbXKTq+546N8QGbuvXLMY+VU9IrsyPt52ieqzr7OfQYmughY+V81t0PBXiI4Je+9KU4/vjjY/Xq1bHbbrtt/Gl6Q90LXvCCSK8rfv3rX9/Vfc2aNZH+GXykNVJpXrVqVUyePLnuuerMP/YffhDL7vnFFi97yLK4RQgQGBMC6e739L2fE/96yiHZ9qfIc0fa7o7bbBWPPbF+xOeYIvtXdHuDx1lkfDaMlhYa6tYPx9wS9RabLZLZkfZ1tM5V3f1UgIsnTgEepgDff//9seuuu2786V133RX77bdffO1rX4ujjjqqq/DcuXNj3rx5W/wsZwH+409dF//10OPFz7CRBAiMS4F9Ju0Q//6uWdn2vehzx9bPmBBP/qb366t67V/R7Q2uU3R8NpCWFtrUrV+OuSXqLTbbK7Mj7edonqs6+6kAF0+bAjzEqs5LINwBLh48IwkQGF4g912gtKUid7DcAW42le4AN+s70up1r6ki10+Oo6u7nwpw8bOgAA+xGutvghvN1yEVj5GRBAjkFsj5OsC0b0WfO7wGOPeZ3Hw9rwFu1nek1etcU0WvnxxHV2c/FeDiZ0AB7mKVPgbtxS9+cVx22WUbf/ra1742HnnkkdY/Bq3XO1GLnvpe71jt9fOi22l6XJX9rDKn6ePotf5o7/Nob2/o8ffafpF3VI810177M/jzHO9Y77atXs8dPgVi+DOUI48+BaLoFZB/XI5rqtf101RG6mp4E9wI1/XT+XOA165dG+klDelx8cUXx4oVK2L+/Pmdf585c2ZMmjQpTjrppEgfcbZ+/fqNSldccUW85S1vibPOOisOO+ywzhvfLrzwwjHzRRg9Pyuz22cD//a/Ff0cx0KfDdllOwmx6GfcblF6frveiJ+/WvA4Nl17cH+KHnuvz94cXKfbNooeU1GjXvs80nEWcRxuf5s+/90MN83OmP8c4ALXmM8Bjuj1ualDP5u16HNb4c8BHuY5qte1W/Y5I+W16me8Zv0c4BFymeNa7/V8VOR5baTnz6LndbhjGekYfQ5wvjfj1y3lY2X+0/oOcPoa4ylTpnS1Xrx4ccyaNavzBRmpAA/9PpD0384777yNX4Wc3uA2lr4KOf02uvCW++Lqm++Nhx77dUzacdv402m7dz7SfdGt94/439540PPiDf/nd98EN3SdXj8fup3nbL9N7PGc7WPVw2vj4bVPdPYlrXHY/rvEtT99cOM+Dh1XdN5wx1Z2Pwf3q8ixp9BU8S1zTEWMiuxzt/Pebe1ujsPt70hGVc9/ru1vet7TeRp6LRTNWa8cDM1vlWusW44G93+kn+X8/N+hT4DdnjuGmnZzHZrFon+IFd3e4HpFn9tGciyS0SaeL6u6jWQ0NCfdrtkix5LjWi/yfFT0uafX83qV4+51jEUzO9K4Xnnu9fOqGam67+4ADy/3tC7AVQOTa57g5ZK0DgECBAgQIFBWQA9RgMtmJst4wcvCaBECBAgQIECggoAeogBXiE39KYJX39AKBAgQIECAQDUBPUQBrpacmrMEryag6QQIECBAgEBlAT1EAa4cnjoTBa+OnrkECBAgQIBAHQE9RAGuk5/KcwWvMp2JBAgQIECAQE0BPUQBrhmhatMFr5qbWQQIECBAgEB9AT1EAa6fogorCF4FNFMIECBAgACBLAJ6iAKcJUhlFxG8smLGEyBAgAABArkE9BAFOFeWSq0jeKW4DCZAgAABAgQyCughCnDGOBVfSvCKWxlJgAABAgQI5BXQQxTgvIkquJrgFYQyjAABAgQIEMguoIcowNlDVWTBlStXxpQpU+Kmm26K3XffvcgUYwgQIECAAAECWQRWr14d06dPj7vvvjv23nvvLGs+XRaZMDAwMPB0OZixdhxLly7tBM+DAAECBAgQINCWQLoRd/DBB7e1+TG5XQW4wdOybt26uPXWW2PSpEmx1VZbNbKlwd/u3GVuhLfyos5LZbrGJzo3jRNX2oDzUoltVCY5N6PCXGkjvc7N+vXr46GHHopp06bFdtttV2kbT9dJCvA4P7Ne3zM2T6DzMjbPS9or52ZsnhvnZWyeF9fM2D0vzk29c6MA1/NrfbY/NFo/BV13wHkZm+fFHxjOy9gVGLt75vnMuRm7AtX3TAGubjcmZnpiGhOnYYudcF7G5nlRgJ2XsSswdvfM85lzM3YFqu+ZAlzdbkzMXLNmTcyfPz/e+c53xrOf/ewxsU92IsJ5GbspcG7G5rlxXsbmeUl75dw4N2NXoPqeKcDV7cwkQIAAAQIECBAYhwIK8Dg8aXaZAAECBAgQIECguoACXN3OTAIECBAgQIAAgXEooACPw5NmlwkQIECAAAECBKoLKMDV7cwkQIAAAQIECBAYhwIK8Dg8aWmX77zzzvjrv/7ruOGGG2KHHXaIt771rfGxj30snvnMZ47TIxpbu718+fK44IILYsmSJXHbbbfF1KlTO/879LFo0aL4wAc+ELfffntMnjy582kcp5566hbj0lp/93d/F/fff3/nG3k++clPxqxZszYb98tf/jLe/e53x1e+8pX49a9/HX/8x38cF110Uey1115jC6fFvbniiiviS1/6UvzoRz+Khx9+OJ7//OfHO97xjjjllFNi4sSJG/fMeRndk/Ttb387zjvvvPjpT3/a+cSA5z3vefHGN74xzjnnnPi93/s952V0T8ewW3vsscc6z2X33ntvLF26NF72spc5Ny2dm3/+53+OE088cYutv+997+v8WT748FzW3AlSgJuzbWzlRx55JF70ohd1itGHPvShePDBBzvF67WvfW1ceumljW23nxZeuHBhzJkzJ2bMmNH5ZWPDhg1bFOAbb7wxXvWqV8UJJ5wQxx9/fHz/+9/v/IH/j//4j/EXf/EXG7lS+T377LM7BeElL3lJfO5zn4urr7460tdXpzI8+DjyyCPjP/7jP+JTn/pU5yPtPvzhD3fKxI9//GO/2PwW6eUvf3kn90cffXTsuuuusXjx4jj//PPjjDPO6PxSkR7Oy+hfqf/yL//Syen06dPjD/7gDzrXyty5czt5/853vuO8jP4p6brFVK4WLFgQDzzwwGYF2DUz+idosAB/61vf2uyXxPTL4x577OGaGYVTogCPAnLuTXz84x+Pj3zkI3HPPffEzjvv3Fn+y1/+chx33HGdOzAvfOELc2+y79ZLhXfwjuLs2bNj2bJlWxTgP/mTP+nchfzhD3+40efkk0+Or3/9652v203z053cVNTSf//EJz7RGfeb3/ymU3wPPPDAuOyyyzr/La2Ryt03vvGN+NM//dPOf/vv//7vzh3OdBf47W9/e9+dg24HnL7TftKkSZv9KP3yd8kll0T6xXDbbbcN52VsRCX9opdyn+42Pve5z3VeWj4tP/vZzzp3fNMv2On5ZNM7wK6Z0T85gwU4PacN/jk+dC+cl2bPiwLcrG8jq8+cOTN+//d/P9JdysFHKlrprxo/+tGPxrve9a5Gttuvi3YrwMk73aVNf1V15plnbqS5/vrrOy9tSIX5pS99aecOZXopQ7qze9BBB20cN2/evM4fRI8++mhMmDChc+f4M5/5TKdQp38ffLz61a+OHXfcMa655pp+5e953F/84hc7d+Hvu+++eM5znuO89BQbnQFXXnllvOlNb4qVK1fGbrvt5ryMDvuwWzn88MM7v3Snv2lKzyuDBdhzWTsnplcBdl6aPy8KcPPG2bewyy67xNve9rbNXieUNnLAAQfEIYccEp///Oezb7OfF+xWgNOd9uT9zW9+s/PSk8FH+m0+nZ9UytLLIv7+7/8+/uqv/irWrl272csY0mtZ3/zmN8eqVas6rx1O/z/d8U2vOd70keam11em1+mBO3UAAAd7SURBVCR7dBdIdxm/+tWvdl4KdMcddzgvLQYl/e3Gk08+2fmbqPQcteeee8bXvva1zr+7Xto7Mel9Bem55K677ur8Mr5pAXZu2jkvgwU4/Xnxv//7v52Xdv3lX/5lvPe9741nPOMZrplROC0K8Cgg597E1ltvHeeee268//3v32zpQw89tFO+0p0Xj3wC3Qpwer1v8k6vnUsvXRh8rF+/PtL5ufDCCztvUkx35NO5Wrdu3WY79G//9m9x2GGHxX/+53927sqk/5+e9NLrwTZ9fPCDH+yU6HRn2GNLgXSnPf3Sl+6gJyvnpd2UpF/m0kse0iP9YpiKV3qTrvPS3nlJv3ynN76l12SnX0quu+66zQqwc9POuUk3NtJL39L7TNLf+qVfFNNLudKbetMbpp2X5s+LAty8cfYtpIL1N3/zN5He0LDp44/+6I86f9WY7oZ55BMYqQCnO7bpCWxoAU4vZzjttNM6BTidq1/96leb7dC1114b6a8k0xuH0uuBUwHeaqutOneUN32kT5j4h3/4h/j5z3+e74CeJiulT9RI9ql0pT/U03Ux+IeG89LOSU55Tp808JOf/KTzi9++++4bKevpfKRfGJ2X0T8v6Q246RykspXelzBcAXZuRv/cDN3ie97znvj0pz/d+ZvB//qv/3LNNHxKFOCGgZtY3ksgmlAdfs0cL4FIBXi77bbbuBEvgah3DtNrp9NrrdOd9e9973ux0047dRYs+9e5zku98zDS7PRRdelNVynr+++/f6mXQDgvec5LeqP0fvvtF1dddVW84hWv6Cyarpejjjqq8/6EdH7SS6/KvDzFuclzbrqtkl6XnT5JJX30WXpJhPPSnHVaWQFu1reR1b0JrhHWYRf1JrjR9e61tVR6jzjiiM7rGdNLUDb9nGRvHOmlN3o/T68HTp/Kkf4GJL1R1JtGR89+cEuDd3uH23L6G5T0xl3nZvTPTbctpo/GTOckFeD05mnnpdnzogA369vI6ulj0NJfL6bf7gfvfKWP00pfhuFj0PKTj/QxaOmjt1IJG3ykjxdKr+Ua+jFo6b8Pfrh5Kgbpdb/ppQ9DPwZt0zfVpb8G22effXwM2ianNL3G+phjjonvfve7nX+S49BH+ugg5yX/dVB2xXSn8ZWvfGX867/+a+dNns5LWcH649N1cMstt2y2UPr39AtJemnVwQcf3PmsZuemvnWOFdInOKX3j6Q/P9LLGZ2XHKrDr6EAN+vbyOqDX4Sx9957b/ZFGOmumC/CyEOe3jiSfgtPj4svvjhWrFgR8+fP7/x7ugOfPot28MPjU0FOn8GcXn+avrxiuC/CSF/YkP6wSZ/Skd6o2O2LMG6++ebNvggj/VW/L8L43TlN3/j22c9+tvOZyqlcbfpIf82e7pg4L3mugTKrpF9K0l+np19I0rdRpjd3pnOUPgM7/bXuNtts47yUAW1w7NDXAKdNuWYaBB9m6fTn9Wte85rOl1qlR7pxkp7bTj/99M7rgJ2X5s+JAty8cSNbSN9Olt5kle6ybL/99p27v+nOsK9CzsOdPrt0ypQpXRdLr50b/BrjVJLTm0w2/Srk9HFDmz4GBgY6X6uc3tmbvoEp3flN5SB9FNGmj/Stb4NfhfzEE0/4KuQu+umXvvQ3H90ezkue7FdZJf3tRrrTm35RTF8ik85TKsUpz+mXksGH66WKbt453Qpw2oJzk9e512qp6Ka/8Ut3e9M1k16rnb5BNP25vulnwTsvvSSr/1wBrm5nJgECBAgQIECAwDgUUIDH4UmzywQIECBAgAABAtUFFODqdmYSIECAAAECBAiMQwEFeByeNLtMgAABAgQIECBQXUABrm5nJgECBAgQIECAwDgUUIDH4UmzywQIECBAgAABAtUFFODqdmYSIECAAAECBAiMQwEFeByeNLtMgAABAgQIECBQXUABrm5nJgECBAgQIECAwDgUUIDH4UmzywQIECBAgAABAtUFFODqdmYSIECAAAECBAiMQwEFeByeNLtMgAABAgQIECBQXUABrm5nJgECBAgQIECAwDgUUIDH4UmzywQIECBAgAABAtUFFODqdmYSIECAAAECBAiMQwEFeByeNLtMgAABAgQIECBQXUABrm5nJgECBAgQIECAwDgUUIDH4UmzywQIECBAgAABAtUFFODqdmYSIECAAAECBAiMQwEFeByeNLtMgAABAgQIECBQXUABrm5nJgECBAgQIECAwDgUUIDH4UmzywQIECBAgAABAtUFFODqdmYSIECAAAECBAiMQwEFeByeNLtMgAABAgQIECBQXUABrm5nJgECBAgQIECAwDgUUIDH4UmzywQIECBAgAABAtUFFODqdmYSIECAAAECBAiMQwEFeByeNLtMgAABAgQIECBQXUABrm5nJgECBAgQIECAwDgUUIDH4UmzywQIECBAgAABAtUFFODqdmYSIECAAAECBAiMQwEFeByeNLtMgAABAgQIECBQXUABrm5nJgECBAgQIECAwDgUUIDH4UmzywQIECBAgAABAtUFFODqdmYSIECAAAECBAiMQ4H/B8XcZgrE00fVAAAAAElFTkSuQmCC\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<matplotlib.collections.PathCollection at 0x2192f449100>"
      ]
     },
     "execution_count": 369,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sel_region = 'Chr7-6Mb-seg-85'\n",
    "plt.figure()\n",
    "plt.scatter([int(_info['pb_index']) \n",
    "             for _info in pb_dict[sel_region].kept_probes.values()],\n",
    "             np.ones(len(pb_dict[sel_region].kept_probes)))\n",
    "plt.scatter([int(_info['pb_index']) \n",
    "             for _info in sel_sequential_pb_dict[sel_region].kept_probes.values()],\n",
    "             2*np.ones(len(sel_sequential_pb_dict[sel_region].kept_probes)))\n",
    "\n",
    "#plt.scatter([int(_p) for _p in genome_name_2_pb_pos['5']], \n",
    "#            2*np.ones(len(genome_name_2_pb_pos['5'])) )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 379,
   "id": "65865761",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: Chr7-6Mb-sequential-seg-65 -> 23\n",
      "gene: Chr7-6Mb-sequential-seg-75 -> 16\n",
      "gene: Chr7-6Mb-sequential-seg-85 -> 14\n",
      "gene: Chr7-6Mb-sequential-seg-95 -> 23\n",
      "gene: Chr7-6Mb-sequential-seg-105 -> 15\n",
      "gene: Chr7-6Mb-sequential-seg-135 -> 15\n",
      "gene: Chr7-6Mb-sequential-seg-166 -> 26\n",
      "gene: Chr7-6Mb-sequential-seg-176 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-196 -> 13\n",
      "gene: Chr7-6Mb-sequential-seg-206 -> 18\n",
      "gene: Chr7-6Mb-sequential-seg-216 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-226 -> 21\n",
      "gene: Chr7-6Mb-sequential-seg-236 -> 25\n",
      "gene: Chr7-6Mb-sequential-seg-286 -> 13\n",
      "gene: Chr7-6Mb-sequential-seg-326 -> 23\n",
      "gene: Chr7-6Mb-sequential-seg-336 -> 25\n",
      "gene: Chr7-6Mb-sequential-seg-346 -> 18\n",
      "gene: Chr7-6Mb-sequential-seg-366 -> 22\n",
      "gene: Chr7-6Mb-sequential-seg-396 -> 14\n",
      "gene: Chr7-6Mb-sequential-seg-416 -> 15\n",
      "gene: Chr7-6Mb-sequential-seg-426 -> 22\n",
      "gene: Chr7-6Mb-sequential-seg-436 -> 26\n",
      "gene: Chr7-6Mb-sequential-seg-446 -> 25\n",
      "gene: Chr7-6Mb-sequential-seg-456 -> 15\n",
      "gene: Chr7-6Mb-sequential-seg-466 -> 17\n",
      "gene: Chr7-6Mb-sequential-seg-476 -> 30\n",
      "gene: Chr7-6Mb-sequential-seg-486 -> 25\n",
      "gene: Chr7-6Mb-sequential-seg-496 -> 20\n",
      "gene: Chr7-6Mb-sequential-seg-506 -> 13\n",
      "gene: Chr7-6Mb-sequential-seg-536 -> 17\n",
      "gene: Chr7-6Mb-sequential-seg-556 -> 13\n",
      "gene: Chr7-6Mb-sequential-seg-52 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-62 -> 28\n",
      "gene: Chr7-6Mb-sequential-seg-72 -> 14\n",
      "gene: Chr7-6Mb-sequential-seg-82 -> 17\n",
      "gene: Chr7-6Mb-sequential-seg-102 -> 22\n",
      "gene: Chr7-6Mb-sequential-seg-122 -> 21\n",
      "gene: Chr7-6Mb-sequential-seg-132 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-163 -> 15\n",
      "gene: Chr7-6Mb-sequential-seg-173 -> 30\n",
      "gene: Chr7-6Mb-sequential-seg-203 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-233 -> 13\n",
      "gene: Chr7-6Mb-sequential-seg-253 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-263 -> 14\n",
      "gene: Chr7-6Mb-sequential-seg-273 -> 17\n",
      "gene: Chr7-6Mb-sequential-seg-283 -> 14\n",
      "gene: Chr7-6Mb-sequential-seg-343 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-363 -> 29\n",
      "gene: Chr7-6Mb-sequential-seg-373 -> 26\n",
      "gene: Chr7-6Mb-sequential-seg-413 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-423 -> 24\n",
      "gene: Chr7-6Mb-sequential-seg-433 -> 13\n",
      "gene: Chr7-6Mb-sequential-seg-443 -> 24\n",
      "gene: Chr7-6Mb-sequential-seg-453 -> 27\n",
      "gene: Chr7-6Mb-sequential-seg-463 -> 26\n",
      "gene: Chr7-6Mb-sequential-seg-473 -> 14\n",
      "gene: Chr7-6Mb-sequential-seg-483 -> 26\n",
      "gene: Chr7-6Mb-sequential-seg-493 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-513 -> 16\n",
      "gene: Chr7-6Mb-sequential-seg-523 -> 24\n",
      "gene: Chr7-6Mb-sequential-seg-533 -> 14\n",
      "gene: Chr7-6Mb-sequential-seg-53 -> 14\n",
      "gene: Chr7-6Mb-sequential-seg-63 -> 20\n",
      "gene: Chr7-6Mb-sequential-seg-93 -> 26\n",
      "gene: Chr7-6Mb-sequential-seg-123 -> 16\n",
      "gene: Chr7-6Mb-sequential-seg-133 -> 16\n",
      "gene: Chr7-6Mb-sequential-seg-164 -> 22\n",
      "gene: Chr7-6Mb-sequential-seg-174 -> 28\n",
      "gene: Chr7-6Mb-sequential-seg-184 -> 15\n",
      "gene: Chr7-6Mb-sequential-seg-204 -> 20\n",
      "gene: Chr7-6Mb-sequential-seg-234 -> 23\n",
      "gene: Chr7-6Mb-sequential-seg-274 -> 27\n",
      "gene: Chr7-6Mb-sequential-seg-284 -> 25\n",
      "gene: Chr7-6Mb-sequential-seg-304 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-324 -> 27\n",
      "gene: Chr7-6Mb-sequential-seg-344 -> 21\n",
      "gene: Chr7-6Mb-sequential-seg-364 -> 26\n",
      "gene: Chr7-6Mb-sequential-seg-374 -> 23\n",
      "gene: Chr7-6Mb-sequential-seg-384 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-394 -> 24\n",
      "gene: Chr7-6Mb-sequential-seg-404 -> 21\n",
      "gene: Chr7-6Mb-sequential-seg-424 -> 32\n",
      "gene: Chr7-6Mb-sequential-seg-434 -> 13\n",
      "gene: Chr7-6Mb-sequential-seg-444 -> 28\n",
      "gene: Chr7-6Mb-sequential-seg-454 -> 15\n",
      "gene: Chr7-6Mb-sequential-seg-464 -> 13\n",
      "gene: Chr7-6Mb-sequential-seg-474 -> 25\n",
      "gene: Chr7-6Mb-sequential-seg-484 -> 23\n",
      "gene: Chr7-6Mb-sequential-seg-494 -> 21\n",
      "gene: Chr7-6Mb-sequential-seg-504 -> 13\n",
      "gene: Chr7-6Mb-sequential-seg-534 -> 19\n",
      "gene: Chr7-6Mb-sequential-seg-544 -> 17\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(cand_sequential_probes, \n",
    "                                                   species_marker = 'gene_',\n",
    "                                                   min_size=12)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 376,
   "id": "f298b4ea",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- total probes removed by internal screening: 0\n"
     ]
    }
   ],
   "source": [
    "# check these probes with int_map\n",
    "\n",
    "kept_probes = library_tools.quality_check._check_between_probes(cand_sequential_probes, lib_table,\n",
    "                                                  primer_len=primer_len, \n",
    "                                                  target_len=target_len)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d876ef7d",
   "metadata": {},
   "source": [
    "## Blast cand_sequential_probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 381,
   "id": "6093d472",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\blast_database\\GRCm38_ensembl\n",
      "-- number of probes loaded: 1852\n",
      "-- processing region: Chr7-6Mb-sequential-seg-102 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-102.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-102\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-102: 6.651s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-105 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-105.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-105\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-105: 1.490s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-122 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-122.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-122\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-122: 1.839s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-123 with 16 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-123.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-123\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-123: 1.312s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-132 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-132.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-132\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-132: 1.751s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-133 with 16 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-133.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-133\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-133: 1.534s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-135 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-135.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-135\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-135: 5.244s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-163 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-163.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-163\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-163: 4.677s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-164 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-164.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-164\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-164: 1.546s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-166 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-166.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-166\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-166: 1.805s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-173 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-173.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-173\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-173: 1.592s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-174 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-174.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-174\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-174: 1.198s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-176 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-176.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-176\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-176: 1.533s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-184 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-184.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-184\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-184: 1.826s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-196 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-196.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-196\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-196: 1.599s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-203 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-203.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-203\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-203: 1.339s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-204 with 20 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-204.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-204\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-204: 1.831s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-206 with 18 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-206.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-206\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-206: 1.375s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-216 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-216.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-216\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-216: 1.751s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-226 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-226.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-226\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-226: 1.308s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-233 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-233.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-233\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-233: 3.578s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-234 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-234.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-234\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-234: 1.674s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-236 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-236.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-236\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-236: 1.591s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-253 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-253.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-253\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-sequential-seg-253: 2.249s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-263 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-263.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-263\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-263: 1.640s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-273 with 17 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-273.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-273\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-273: 1.553s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-274 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-274.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-274\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-274: 1.574s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-283 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-283.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-283\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-283: 1.460s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-284 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-284.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-284\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-284: 1.327s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-286 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-286.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-286\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-286: 1.771s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-304 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-304.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-304\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-304: 1.404s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-324 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-324.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-324\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-324: 1.608s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-326 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-326.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-326\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-326: 2.039s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-336 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-336.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-336\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-336: 2.864s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-343 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-343.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-343\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-343: 1.585s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-344 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-344.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-344\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-344: 1.623s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-346 with 18 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-346.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-346\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-346: 1.500s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-363 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-363.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-363\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-363: 1.696s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-364 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-364.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-364\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-364: 1.612s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-366 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-366.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-366\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-366: 2.482s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-373 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-373.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-373\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-373: 1.493s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-374 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-374.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-374\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-374: 4.540s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-384 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-384.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-384\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-384: 1.529s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-394 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-394.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-394\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-394: 5.162s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-396 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-396.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-396\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-396: 1.485s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-404 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-404.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-404\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-404: 1.563s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-413 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-413.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-413\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-413: 1.691s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-416 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-416.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-416\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-sequential-seg-416: 1.620s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-423 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-423.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-423\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-423: 2.009s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-424 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-424.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-424\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-424: 4.982s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-426 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-426.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-426\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-426: 1.593s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-433 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-433.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-433\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-433: 1.544s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-434 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-434.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-434\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-434: 1.100s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-436 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-436.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-436\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-436: 1.868s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-443 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-443.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-443\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-443: 1.413s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-444 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-444.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-444\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-444: 2.141s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-446 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-446.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-446\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-446: 1.895s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-453 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-453.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-453\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-453: 2.065s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-454 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-454.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-454\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-454: 1.296s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-456 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-456.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-456\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-456: 1.836s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-463 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-463.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-463\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-463: 2.326s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-464 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-464.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-464\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-464: 1.564s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-466 with 17 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-466.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-466\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-466: 1.411s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-473 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-473.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-473\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-473: 1.289s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-474 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-474.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-474\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-474: 1.457s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-476 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-476.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-476\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-476: 1.610s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-483 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-483.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-483\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-483: 1.875s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-484 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-484.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-484\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-484: 1.655s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-486 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-486.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-486\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-486: 1.728s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-493 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-493.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-493\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-493: 1.410s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-494 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-494.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-494\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-494: 1.483s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-496 with 20 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-496.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-496\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Chr7-6Mb-sequential-seg-496: 2.064s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-504 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-504.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-504\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-504: 1.176s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-506 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-506.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-506\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-506: 1.678s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-513 with 16 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-513.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-513\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-513: 1.697s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-52 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-52.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-52\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-52: 1.579s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-523 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-523.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-523\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-523: 1.829s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-53 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-53.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-53\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-53: 2.206s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-533 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-533.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-533\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-533: 1.747s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-534 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-534.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-534\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-534: 1.448s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-536 with 17 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-536.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-536\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-536: 4.219s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-544 with 17 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-544.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-544\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-544: 1.635s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-556 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-556.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-556\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-556: 1.556s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-62 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-62.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-62\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-62: 1.668s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-63 with 20 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-63.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-63\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-63: 1.613s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-65 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-65.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-65\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-65: 2.618s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-72 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-72.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-72\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-72: 1.815s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-75 with 16 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-75.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-75\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-75: 5.222s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-82 with 17 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-82.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-82\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-82: 2.073s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-85 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-85.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-85\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-85: 2.914s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-93 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-93.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-93\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-93: 1.762s.\n",
      "-- processing region: Chr7-6Mb-sequential-seg-95 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\blast_sequential\\probe_gene_Chr7-6Mb-sequential-seg-95.fasta\n",
      "--- *blasting region: Chr7-6Mb-sequential-seg-95\n",
      "--- total time for blast Chr7-6Mb-sequential-seg-95: 6.063s.\n"
     ]
    }
   ],
   "source": [
    "blast_database = os.path.join(reference_folder, 'blast_database', 'GRCm38_ensembl')\n",
    "\n",
    "check.Blast_probes(cand_sequential_probes, library_folder, \n",
    "                   blast_subfolder='blast_sequential',\n",
    "                   num_threads=44, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 386,
   "id": "abff2c8b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 1852\n",
      "- Number of regions in this library: 92\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-65\n",
      "--- gene=Chr7-6Mb-sequential-seg-65, id=0 removed by soft count = 404\n",
      "--- gene=Chr7-6Mb-sequential-seg-65, id=3 removed by soft count = 195\n",
      "--- gene=Chr7-6Mb-sequential-seg-65, id=4 removed by soft count = 82\n",
      "--- number of probes: 23 , kept by blast: 20 , if remove dups: 20\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-75\n",
      "--- gene=Chr7-6Mb-sequential-seg-75, id=0 removed by soft count = 91\n",
      "--- gene=Chr7-6Mb-sequential-seg-75, id=2 removed by soft count = 245\n",
      "--- gene=Chr7-6Mb-sequential-seg-75, id=3 removed by soft count = 108\n",
      "--- gene=Chr7-6Mb-sequential-seg-75, id=4 removed by soft count = 66\n",
      "--- gene=Chr7-6Mb-sequential-seg-75, id=5 removed by soft count = 95\n",
      "--- gene=Chr7-6Mb-sequential-seg-75, id=6 removed by soft count = 157\n",
      "--- number of probes: 16 , kept by blast: 10 , if remove dups: 10\n",
      "-- number of probes kept for this region: 10\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-85\n",
      "--- gene=Chr7-6Mb-sequential-seg-85, id=13 removed by soft count = 150\n",
      "--- number of probes: 14 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-95\n",
      "--- gene=Chr7-6Mb-sequential-seg-95, id=5 removed by hard count = 0\n",
      "--- number of probes: 23 , kept by blast: 22 , if remove dups: 22\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-105\n",
      "--- number of probes: 15 , kept by blast: 15 , if remove dups: 15\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-135\n",
      "--- gene=Chr7-6Mb-sequential-seg-135, id=0 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-sequential-seg-135, id=1 removed by soft count = 1319\n",
      "--- gene=Chr7-6Mb-sequential-seg-135, id=12 removed by soft count = 237\n",
      "--- number of probes: 15 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-166\n",
      "--- number of probes: 26 , kept by blast: 26 , if remove dups: 26\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-176\n",
      "--- number of probes: 19 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-196\n",
      "--- number of probes: 13 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-206\n",
      "--- number of probes: 18 , kept by blast: 18 , if remove dups: 18\n",
      "-- number of probes kept for this region: 18\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-216\n",
      "--- number of probes: 19 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-226\n",
      "--- number of probes: 21 , kept by blast: 21 , if remove dups: 20\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-236\n",
      "--- gene=Chr7-6Mb-sequential-seg-236, id=4 removed by soft count = 85\n",
      "--- number of probes: 25 , kept by blast: 24 , if remove dups: 24\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-286\n",
      "--- gene=Chr7-6Mb-sequential-seg-286, id=0 removed by soft count = 233\n",
      "--- number of probes: 13 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-326\n",
      "--- gene=Chr7-6Mb-sequential-seg-326, id=16 removed by soft count = 400\n",
      "--- gene=Chr7-6Mb-sequential-seg-326, id=17 removed by soft count = 58\n",
      "--- number of probes: 23 , kept by blast: 21 , if remove dups: 21\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-336\n",
      "--- gene=Chr7-6Mb-sequential-seg-336, id=4 removed by soft count = 811\n",
      "--- gene=Chr7-6Mb-sequential-seg-336, id=5 removed by soft count = 305\n",
      "--- number of probes: 25 , kept by blast: 23 , if remove dups: 20\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-346\n",
      "--- number of probes: 18 , kept by blast: 18 , if remove dups: 18\n",
      "-- number of probes kept for this region: 18\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-366\n",
      "--- gene=Chr7-6Mb-sequential-seg-366, id=1 removed by soft count = 609\n",
      "--- number of probes: 22 , kept by blast: 21 , if remove dups: 21\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-396\n",
      "--- gene=Chr7-6Mb-sequential-seg-396, id=0 removed by soft count = 108\n",
      "--- gene=Chr7-6Mb-sequential-seg-396, id=3 removed by soft count = 135\n",
      "--- number of probes: 14 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-416\n",
      "--- gene=Chr7-6Mb-sequential-seg-416, id=0 removed by soft count = 126\n",
      "--- number of probes: 15 , kept by blast: 14 , if remove dups: 14\n",
      "-- number of probes kept for this region: 14\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-426\n",
      "--- number of probes: 22 , kept by blast: 22 , if remove dups: 22\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-436\n",
      "--- number of probes: 26 , kept by blast: 26 , if remove dups: 26\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-446\n",
      "--- number of probes: 25 , kept by blast: 25 , if remove dups: 25\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-456\n",
      "--- gene=Chr7-6Mb-sequential-seg-456, id=0 removed by soft count = 166\n",
      "--- gene=Chr7-6Mb-sequential-seg-456, id=12 removed by soft count = 51\n",
      "--- number of probes: 15 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-466\n",
      "--- number of probes: 17 , kept by blast: 17 , if remove dups: 17\n",
      "-- number of probes kept for this region: 17\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-476\n",
      "--- gene=Chr7-6Mb-sequential-seg-476, id=21 removed by soft count = 53\n",
      "--- number of probes: 30 , kept by blast: 29 , if remove dups: 29\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-486\n",
      "--- number of probes: 25 , kept by blast: 25 , if remove dups: 25\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-496\n",
      "--- gene=Chr7-6Mb-sequential-seg-496, id=1 removed by soft count = 198\n",
      "--- gene=Chr7-6Mb-sequential-seg-496, id=2 removed by soft count = 178\n",
      "--- gene=Chr7-6Mb-sequential-seg-496, id=3 removed by soft count = 65\n",
      "--- gene=Chr7-6Mb-sequential-seg-496, id=5 removed by soft count = 354\n",
      "--- number of probes: 20 , kept by blast: 16 , if remove dups: 16\n",
      "-- number of probes kept for this region: 16\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-506\n",
      "--- gene=Chr7-6Mb-sequential-seg-506, id=0 removed by soft count = 90\n",
      "--- number of probes: 13 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-536\n",
      "--- gene=Chr7-6Mb-sequential-seg-536, id=0 removed by soft count = 54\n",
      "--- gene=Chr7-6Mb-sequential-seg-536, id=1 removed by soft count = 212\n",
      "--- gene=Chr7-6Mb-sequential-seg-536, id=2 removed by soft count = 69\n",
      "--- gene=Chr7-6Mb-sequential-seg-536, id=4 removed by soft count = 1604\n",
      "--- number of probes: 17 , kept by blast: 13 , if remove dups: 1\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-556\n",
      "--- gene=Chr7-6Mb-sequential-seg-556, id=0 removed by soft count = 89\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 13 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-52\n",
      "--- number of probes: 19 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-62\n",
      "--- number of probes: 28 , kept by blast: 28 , if remove dups: 28\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-72\n",
      "--- gene=Chr7-6Mb-sequential-seg-72, id=0 removed by soft count = 2502\n",
      "--- number of probes: 14 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-82\n",
      "--- gene=Chr7-6Mb-sequential-seg-82, id=11 removed by soft count = 220\n",
      "--- number of probes: 17 , kept by blast: 16 , if remove dups: 16\n",
      "-- number of probes kept for this region: 16\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-102\n",
      "--- gene=Chr7-6Mb-sequential-seg-102, id=1 removed by soft count = 83\n",
      "--- gene=Chr7-6Mb-sequential-seg-102, id=2 removed by soft count = 3060\n",
      "--- number of probes: 22 , kept by blast: 20 , if remove dups: 20\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-122\n",
      "--- gene=Chr7-6Mb-sequential-seg-122, id=0 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-sequential-seg-122, id=16 removed by soft count = 53\n",
      "--- number of probes: 21 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-132\n",
      "--- gene=Chr7-6Mb-sequential-seg-132, id=0 removed by soft count = 62\n",
      "--- number of probes: 19 , kept by blast: 18 , if remove dups: 18\n",
      "-- number of probes kept for this region: 18\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-163\n",
      "--- gene=Chr7-6Mb-sequential-seg-163, id=0 removed by soft count = 130\n",
      "--- gene=Chr7-6Mb-sequential-seg-163, id=1 removed by soft count = 2290\n",
      "--- number of probes: 15 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-173\n",
      "--- gene=Chr7-6Mb-sequential-seg-173, id=13 removed by hard count = 3\n",
      "--- number of probes: 30 , kept by blast: 29 , if remove dups: 27\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-203\n",
      "--- number of probes: 19 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-233\n",
      "--- gene=Chr7-6Mb-sequential-seg-233, id=0 removed by soft count = 901\n",
      "--- gene=Chr7-6Mb-sequential-seg-233, id=1 removed by soft count = 166\n",
      "--- gene=Chr7-6Mb-sequential-seg-233, id=2 removed by soft count = 225\n",
      "--- number of probes: 13 , kept by blast: 10 , if remove dups: 10\n",
      "-- number of probes kept for this region: 10\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-253\n",
      "--- gene=Chr7-6Mb-sequential-seg-253, id=18 removed by soft count = 361\n",
      "--- number of probes: 19 , kept by blast: 18 , if remove dups: 18\n",
      "-- number of probes kept for this region: 18\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-263\n",
      "--- gene=Chr7-6Mb-sequential-seg-263, id=0 removed by soft count = 72\n",
      "--- number of probes: 14 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-273\n",
      "--- number of probes: 17 , kept by blast: 17 , if remove dups: 17\n",
      "-- number of probes kept for this region: 17\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-283\n",
      "--- gene=Chr7-6Mb-sequential-seg-283, id=0 removed by soft count = 79\n",
      "--- number of probes: 14 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-343\n",
      "--- number of probes: 19 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-363\n",
      "--- number of probes: 29 , kept by blast: 29 , if remove dups: 29\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-373\n",
      "--- number of probes: 26 , kept by blast: 26 , if remove dups: 26\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-413\n",
      "--- number of probes: 19 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-423\n",
      "--- number of probes: 24 , kept by blast: 24 , if remove dups: 24\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-433\n",
      "--- gene=Chr7-6Mb-sequential-seg-433, id=0 removed by soft count = 69\n",
      "--- number of probes: 13 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-443\n",
      "--- number of probes: 24 , kept by blast: 24 , if remove dups: 24\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-453\n",
      "--- gene=Chr7-6Mb-sequential-seg-453, id=20 removed by soft count = 65\n",
      "--- number of probes: 27 , kept by blast: 26 , if remove dups: 26\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-463\n",
      "--- number of probes: 26 , kept by blast: 26 , if remove dups: 26\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-473\n",
      "--- number of probes: 14 , kept by blast: 14 , if remove dups: 14\n",
      "-- number of probes kept for this region: 14\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-483\n",
      "--- number of probes: 26 , kept by blast: 26 , if remove dups: 26\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-493\n",
      "--- number of probes: 19 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-513\n",
      "--- gene=Chr7-6Mb-sequential-seg-513, id=0 removed by soft count = 72\n",
      "--- number of probes: 16 , kept by blast: 15 , if remove dups: 15\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-523\n",
      "--- gene=Chr7-6Mb-sequential-seg-523, id=0 removed by soft count = 276\n",
      "--- number of probes: 24 , kept by blast: 23 , if remove dups: 23\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-533\n",
      "--- gene=Chr7-6Mb-sequential-seg-533, id=2 removed by soft count = 80\n",
      "--- gene=Chr7-6Mb-sequential-seg-533, id=13 removed by soft count = 161\n",
      "--- number of probes: 14 , kept by blast: 12 , if remove dups: 10\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-53\n",
      "--- gene=Chr7-6Mb-sequential-seg-53, id=3 removed by soft count = 958\n",
      "--- number of probes: 14 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-63\n",
      "--- number of probes: 20 , kept by blast: 20 , if remove dups: 20\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-93\n",
      "--- number of probes: 26 , kept by blast: 26 , if remove dups: 26\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-123\n",
      "--- gene=Chr7-6Mb-sequential-seg-123, id=0 removed by soft count = 163\n",
      "--- number of probes: 16 , kept by blast: 15 , if remove dups: 15\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-133\n",
      "--- number of probes: 16 , kept by blast: 16 , if remove dups: 16\n",
      "-- number of probes kept for this region: 16\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-164\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 22 , kept by blast: 22 , if remove dups: 22\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-174\n",
      "--- number of probes: 28 , kept by blast: 28 , if remove dups: 28\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-184\n",
      "--- gene=Chr7-6Mb-sequential-seg-184, id=0 removed by hard count = 5\n",
      "--- gene=Chr7-6Mb-sequential-seg-184, id=12 removed by soft count = 61\n",
      "--- gene=Chr7-6Mb-sequential-seg-184, id=14 removed by soft count = 105\n",
      "--- number of probes: 15 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-204\n",
      "--- number of probes: 20 , kept by blast: 20 , if remove dups: 20\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-234\n",
      "--- number of probes: 23 , kept by blast: 23 , if remove dups: 23\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-274\n",
      "--- number of probes: 27 , kept by blast: 27 , if remove dups: 27\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-284\n",
      "--- number of probes: 25 , kept by blast: 25 , if remove dups: 25\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-304\n",
      "--- number of probes: 19 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-324\n",
      "--- gene=Chr7-6Mb-sequential-seg-324, id=0 removed by soft count = 117\n",
      "--- gene=Chr7-6Mb-sequential-seg-324, id=1 removed by soft count = 143\n",
      "--- gene=Chr7-6Mb-sequential-seg-324, id=2 removed by soft count = 92\n",
      "--- number of probes: 27 , kept by blast: 24 , if remove dups: 24\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-344\n",
      "--- gene=Chr7-6Mb-sequential-seg-344, id=0 removed by soft count = 216\n",
      "--- number of probes: 21 , kept by blast: 20 , if remove dups: 20\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-364\n",
      "--- number of probes: 26 , kept by blast: 26 , if remove dups: 16\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-374\n",
      "--- gene=Chr7-6Mb-sequential-seg-374, id=2 removed by soft count = 3408\n",
      "--- number of probes: 23 , kept by blast: 22 , if remove dups: 22\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-384\n",
      "--- number of probes: 19 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-394\n",
      "--- gene=Chr7-6Mb-sequential-seg-394, id=0 removed by soft count = 3961\n",
      "--- number of probes: 24 , kept by blast: 23 , if remove dups: 23\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-404\n",
      "--- gene=Chr7-6Mb-sequential-seg-404, id=3 removed by hard count = 3\n",
      "--- number of probes: 21 , kept by blast: 20 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-424\n",
      "--- number of probes: 32 , kept by blast: 32 , if remove dups: 31\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-434\n",
      "--- number of probes: 13 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-444\n",
      "--- number of probes: 28 , kept by blast: 28 , if remove dups: 28\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-454\n",
      "--- number of probes: 15 , kept by blast: 15 , if remove dups: 15\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-464\n",
      "--- gene=Chr7-6Mb-sequential-seg-464, id=0 removed by soft count = 162\n",
      "--- number of probes: 13 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-474\n",
      "--- number of probes: 25 , kept by blast: 25 , if remove dups: 23\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-484\n",
      "--- number of probes: 23 , kept by blast: 23 , if remove dups: 23\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-494\n",
      "--- number of probes: 21 , kept by blast: 21 , if remove dups: 21\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-504\n",
      "--- number of probes: 13 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-534\n",
      "--- gene=Chr7-6Mb-sequential-seg-534, id=12 removed by soft count = 86\n",
      "--- number of probes: 19 , kept by blast: 18 , if remove dups: 18\n",
      "-- number of probes kept for this region: 18\n",
      "-- checking probes in region: Chr7-6Mb-sequential-seg-544\n",
      "--- gene=Chr7-6Mb-sequential-seg-544, id=0 removed by soft count = 209\n",
      "--- gene=Chr7-6Mb-sequential-seg-544, id=1 removed by soft count = 60\n",
      "--- gene=Chr7-6Mb-sequential-seg-544, id=2 removed by soft count = 172\n",
      "--- number of probes: 17 , kept by blast: 14 , if remove dups: 14\n",
      "-- number of probes kept for this region: 14\n",
      "- summarize\n",
      "-- region: Chr7-6Mb-sequential-seg-65 , number of probes: 20\n",
      "-- region: Chr7-6Mb-sequential-seg-75 , number of probes: 10\n",
      "-- region: Chr7-6Mb-sequential-seg-85 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-95 , number of probes: 22\n",
      "-- region: Chr7-6Mb-sequential-seg-105 , number of probes: 15\n",
      "-- region: Chr7-6Mb-sequential-seg-135 , number of probes: 12\n",
      "-- region: Chr7-6Mb-sequential-seg-166 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-176 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-196 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-206 , number of probes: 18\n",
      "-- region: Chr7-6Mb-sequential-seg-216 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-226 , number of probes: 20\n",
      "-- region: Chr7-6Mb-sequential-seg-236 , number of probes: 24\n",
      "-- region: Chr7-6Mb-sequential-seg-286 , number of probes: 12\n",
      "-- region: Chr7-6Mb-sequential-seg-326 , number of probes: 21\n",
      "-- region: Chr7-6Mb-sequential-seg-336 , number of probes: 20\n",
      "-- region: Chr7-6Mb-sequential-seg-346 , number of probes: 18\n",
      "-- region: Chr7-6Mb-sequential-seg-366 , number of probes: 21\n",
      "-- region: Chr7-6Mb-sequential-seg-396 , number of probes: 12\n",
      "-- region: Chr7-6Mb-sequential-seg-416 , number of probes: 14\n",
      "-- region: Chr7-6Mb-sequential-seg-426 , number of probes: 22\n",
      "-- region: Chr7-6Mb-sequential-seg-436 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-446 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-456 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-466 , number of probes: 17\n",
      "-- region: Chr7-6Mb-sequential-seg-476 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-486 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-496 , number of probes: 16\n",
      "-- region: Chr7-6Mb-sequential-seg-506 , number of probes: 12\n",
      "-- region: Chr7-6Mb-sequential-seg-536 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-556 , number of probes: 12\n",
      "-- region: Chr7-6Mb-sequential-seg-52 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-62 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-72 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-82 , number of probes: 16\n",
      "-- region: Chr7-6Mb-sequential-seg-102 , number of probes: 20\n",
      "-- region: Chr7-6Mb-sequential-seg-122 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-132 , number of probes: 18\n",
      "-- region: Chr7-6Mb-sequential-seg-163 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-173 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-203 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-233 , number of probes: 10\n",
      "-- region: Chr7-6Mb-sequential-seg-253 , number of probes: 18\n",
      "-- region: Chr7-6Mb-sequential-seg-263 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-273 , number of probes: 17\n",
      "-- region: Chr7-6Mb-sequential-seg-283 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-343 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-363 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-373 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-413 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-423 , number of probes: 24\n",
      "-- region: Chr7-6Mb-sequential-seg-433 , number of probes: 12\n",
      "-- region: Chr7-6Mb-sequential-seg-443 , number of probes: 24\n",
      "-- region: Chr7-6Mb-sequential-seg-453 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-463 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-473 , number of probes: 14\n",
      "-- region: Chr7-6Mb-sequential-seg-483 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-493 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-513 , number of probes: 15\n",
      "-- region: Chr7-6Mb-sequential-seg-523 , number of probes: 23\n",
      "-- region: Chr7-6Mb-sequential-seg-533 , number of probes: 12\n",
      "-- region: Chr7-6Mb-sequential-seg-53 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-63 , number of probes: 20\n",
      "-- region: Chr7-6Mb-sequential-seg-93 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-123 , number of probes: 15\n",
      "-- region: Chr7-6Mb-sequential-seg-133 , number of probes: 16\n",
      "-- region: Chr7-6Mb-sequential-seg-164 , number of probes: 22\n",
      "-- region: Chr7-6Mb-sequential-seg-174 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-184 , number of probes: 12\n",
      "-- region: Chr7-6Mb-sequential-seg-204 , number of probes: 20\n",
      "-- region: Chr7-6Mb-sequential-seg-234 , number of probes: 23\n",
      "-- region: Chr7-6Mb-sequential-seg-274 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-284 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-304 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-324 , number of probes: 24\n",
      "-- region: Chr7-6Mb-sequential-seg-344 , number of probes: 20\n",
      "-- region: Chr7-6Mb-sequential-seg-364 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-374 , number of probes: 22\n",
      "-- region: Chr7-6Mb-sequential-seg-384 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-394 , number of probes: 23\n",
      "-- region: Chr7-6Mb-sequential-seg-404 , number of probes: 19\n",
      "-- region: Chr7-6Mb-sequential-seg-424 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-434 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-444 , number of probes: 25\n",
      "-- region: Chr7-6Mb-sequential-seg-454 , number of probes: 15\n",
      "-- region: Chr7-6Mb-sequential-seg-464 , number of probes: 12\n",
      "-- region: Chr7-6Mb-sequential-seg-474 , number of probes: 23\n",
      "-- region: Chr7-6Mb-sequential-seg-484 , number of probes: 23\n",
      "-- region: Chr7-6Mb-sequential-seg-494 , number of probes: 21\n",
      "-- region: Chr7-6Mb-sequential-seg-504 , number of probes: 13\n",
      "-- region: Chr7-6Mb-sequential-seg-534 , number of probes: 18\n",
      "-- region: Chr7-6Mb-sequential-seg-544 , number of probes: 14\n",
      "- Number of probes kept: 1731\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_long_5kb\\Sequential\\.\\blast_sequential_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "import ImageAnalysis3.library_tools.quality_check as qc\n",
    "reload(check)\n",
    "num_probes_per_region = 25\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = qc.Screening_Probes_by_Blast(\n",
    "    sequential_assemble_folder, \n",
    "    num_probes_per_region,\n",
    "    probe_filename='candidate_probes.fasta',\n",
    "    blast_subfolder='blast_sequential',\n",
    "    keep_mode='end',\n",
    "    hard_thres=target_len,\n",
    "    soft_count_th=50,\n",
    "    save_filename='blast_sequential_full_probes.fasta',\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 391,
   "id": "99c465e8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7sXQWUFMcWvYHF3d3dZbHF3SIECJIQCCQEiOMJ7pBAIIEQI/xAQgxNiOBui+ziu7C4u7vzz6vZGmqb7pnukZ1h5tU5//ywU3qrpuf1q/fufe7x48ePwYURYAQYAUaAEWAEGAFGgBEIEgSeYwM4SHaal8kIMAKMACPACDACjAAjIBBgA5gPAiPACDACjAAjwAgwAoxAUCHABnBQbTcvlhFgBBgBRoARYAQYAUaADWA+A4wAI8AIMAKMACPACDACQYUAG8BBtd28WEaAEWAEGAFGgBFgBBgBNoD5DDACjAAjwAgwAowAI8AIBBUCbAAH1XbzYhkBRoARYAQYAUaAEWAE2ADmM8AIMAKMACPACDACjAAjEFQIsAEcVNvNi2UEGAFGgBFgBBgBRoARYAOYzwAjwAgwAowAI8AIMAKMQFAhwAZwUG03L5YRYAQYAUaAEWAEGAFGgA1gPgOMACPACDACjAAjwAgwAkGFABvAQbXdvFhGgBFgBBgBRoARYAQYATaA+QwwAowAI8AIMAKMACPACAQVAmwAB9V282IZAUaAEWAEGAFGgBFgBNgA5jPACDACjAAjwAgwAowAIxBUCLABHFTbzYtlBBgBRoARYAQYAUaAEWADmM8AI8AIMAKMACPACDACjEBQIcAGcFBtNy+WEWAEGAFGgBFgBBgBRoANYD4DjAAjwAgwAowAI8AIMAJBhQAbwEG13bxYRoARYAQYAUaAEWAEGAE2gPkMMAKMACPACDACjAAjwAgEFQJsAAfVdvNiGQFGgBFgBBgBRoARYATYAOYzwAgwAowAI8AIMAKMACMQVAiwARxU282LZQQYAUaAEWAEGAFGgBFgA5jPACPACDACjAAjwAgwAoxAUCHABnBQbTcvlhFgBBgBRoARYAQYAUaADWA+A4wAI8AIMAKMACPACDACQYUAG8BBtd28WEaAEWAEGAFGgBFgBBgBNoD5DDACjAAjwAgwAowAI8AIBBUCbAAH1XbzYhkBRoARYAQYAUaAEWAE2ADmM8AIMAKMACPACDACjAAjEFQIsAEcVNvNi2UEGAFGgBFgBBgBRoARYAOYzwAjwAgwAowAI8AIMAKMQFAhwAZwUG03L5YRYAQYAUaAEWAEGAFGgA1gPgOMACPACDACjAAjwAgwAkGFABvAQbXdvFhGgBFgBBgBRoARYAQYATaA+QwwAowAI8AIMAKMACPACAQVAmwAB9V282IZAUaAEWAEGAFGgBFgBNgA5jPACDACjAAjwAgwAowAIxBUCLABHFTbzYtlBBgBRoARYAQYAUaAEWADmM8AI8AIMAKMACPACDACjEBQIcAGcFBtNy+WEWAEGAFGgBFgBBgBRoANYD4DjAAjwAgwAowAI8AIMAJBhQAbwEG13bxYRoARYAQYAUaAEWAEGAE2gPkMMAKMACPACDACjAAjwAgEFQJsAAfVdvNiGQFGgBFgBBgBRoARYATYAOYzwAgwAowAI8AIMAKMACMQVAiwARxU282LZQQYAUaAEWAEGAFGgBFgA5jPACPACDACjAAjwAgwAoxAUCHABnBQbTcvlhFgBBgBRoARYAQYAUaADWA+A4wAI8AIMAKMACPACDACQYUAG8BBtd28WEaAEWAEGAFGgBFgBBgBNoD5DDACjAAjwAgwAowAI8AIBBUCbAAH1XbzYhkBRoARYAQYAUaAEWAE2ADmM8AIMAKMACPACDACjAAjEFQIBJUBvHjxYowePRrR0dG4du0acuTIgZdffhlDhgxBmjRp7Bu/YMECDBgwAHv27EHOnDnRs2dPvPvuu0F1MHixjAAjwAgwAowAI8AIBCoCQWUA//7779i5cycqVaqEdOnSYffu3Rg6dCjKly+PJUuWiD0ODw9HzZo10aFDB7z++utYv369MJC///57dO7cOVDPAa+LEWAEGAFGgBFgBBiBoEEgqAxgvV394Ycf0KVLF5w8eRLZs2dHkyZNcOnSJWzatMlenT7/999/ceLECSRIkCBoDgcvlBFgBBgBRoARYAQYgUBEIOgN4Hnz5qFly5Y4cuQIsmbNitSpU+PTTz9Fjx497Pu9evVq1K5dGxEREQgNDQ3Ec8BrYgQYAUaAEWAEGAFGIGgQCEoD+OHDh7h//76IBX7zzTeRO3du/P333+LfJUqUwMKFC9G4cWP7ITh//jwyZ86MGTNmiLAILowAI8AIMAKMACPACDACzy4CQWkAU2IbhTxQIUN3zpw5SJEihYj3rV69uogDrlKlin1XHzx4gESJEmHixIn48MMPDXebEuvof7LcuXMHx48fR758+RASEvLsnhKeOSPACDACjAAjwAg8cwiQ/UJOvFKlSiFp0qTP3Py9OeGgNIApEe7GjRuIiorCiBEjULBgQSxduhQbN24UBjD9f+XKlZ8ygCdNmoQPPvjAcD8ooW7YsGHe3C/umxFgBBgBRoARYAQYAUsIbN68GRUrVrTUJtArB6UBrG5qZGQkKlSogNmzZ6N48eJuhUBoPcDk/a1atSro4GXLli3QzxKvjxFgBBgBRoAR8BgCi6NOY8j8aNFf6ZxpMKVDBY/1HSwdnT59WjBfHT58GHnz5g2WZZtaZ9AbwBQPnCRJEowcOVIkvnkyCY5YI3LlyiXCICjsggsjwAgwAowAI8AImENgVsRx9J2zU1QukzMN5r9f3VxDrmVHgO0Q48MQ9AbwunXrUKNGDcycOROtW7cWNGhXrlwRccCydOvWTSTJWaVB44PHTyFGgBFgBBgBRsA1BGaEH8Gg+VGicbFsqbHwoxqudRTErdgOYQNYINCiRQsR7lC6dGkkS5YMO3bswNixY5ElSxZs2bIFiRMntgthdOzYEe3atROJcYMHD3ZJCIMPXhA/dXjpjAAjwAgwAm4hMHXtIYz8b4/oo1DmlFjas5Zb/QVjY7ZD2AAWCBC/L3l6Dx48iEePHol4GDKKe/fuLUIfZCEp5P79+8eRQn7vvfcsf3f44FmGjBswAowAI8AIMAICgckr9uPzJfvEf+fNkByr+tRhZCwiwHYIG8AWj4xnqvPB8wyO3AsjwAgwAoxA8CHw+eIYTF55QCw8R9pkWP9J3eADwc0Vsx3CBrCbR8i15nzwXMONWzECjAAjwAgwAqP+i8YPaw8LIDKlSoItA+ozKBYRYDuEDWCLR8Yz1fngeQZH7oURYAQYAUYg+BAY9NduzNh4VCw8XfJE2Da4YfCB4OaK2Q5hA9jNI+Racz54ruHGrRgBRoARYAQYgb5zdmBWxAkBRMokIdg9rBGDYhEBtkPYALZ4ZDxTnQ+eZ3DkXhgBRoARYASCD4EPf9+Gv3ecEgtPHJIA+0Y2CT4Q3Fwx2yFsALt5hFxrzgfPNdy4FSPACDACjAAj0HVGBBZHnRVAJHgOODTmeQbFIgJsh7ABbPHIeKY6HzzP4Mi9uI7AtTv3ce/BI2RMmcT1Tril3yJw8cZdhCRIgDTJE/ntHHlijICrCLzx42as3nfe3vzg6KZISJYwF9MIsB3CBrDpw+LJinzwPIkm92UVgfsPH6HGZytx9fZ9rP24DhvBVgH08/r0clNtzAokCkmAdR/XQfLEIX4+Y54eI2ANgTbfh2PT4Uv2RntHNEbSRAmtdRLktdkOYQPYJ18BPng+gZ0HjUXgzNU7qDJmufjXz29WQs3CmRibAEJg98mreOGrdWJFf71XDWVzpQ2g1fFSGAHg5a/XY/vxK3Yodg5tiNRJ+bbDytlgO4QNYCvnxWN1+eB5DEruyAUEjly4idqfrxItP29VBq+E5nShF27irwhsOXIJrb4LF9Ob9Go5vFQmu79OlefFCLiEQOMv12Dvmev2tlsHNUD6FIld6itYG7EdwgawT84+HzyfwM6DxiIQc+Y6Gn25RvyrT6MieK9OQcYmgBCg2EiKkaTSu2FhvF+3UACtjpfCCAB1P1+FQxdu2qHY1L8esqROytBYQIDtEDaALRwXz1Xlg+c5LLkn6wjsOH4Fzb5eLxq+EZYHw5qVtN4Jt/BbBBbtPoNuv0SK+bWukBNjXynjt3PliTECriBQdcxynLp6x950bd86yJU+uStdBW0btkPYAPbJ4eeD5xPYedBYBDYfvoTW39uuyBuXyIrv2ocyNgGEwF/bTqL7zO1iRVXyp8cfXcICaHW8FEYAKD9iKS7dvGeHYmXv2siXMQVDYwEBtkPYALZwXDxXlQ+e57DknqwjsGbfeXSIvSIvnzst5r1bzXon3MJvEfht0zH0/3OXmF/2NEmxoV89v50rT4wRcAWBEoMX4ea9h/amS3rUROEsqVzpKmjbsB3CBrBPDj8fPJ/AzoPGIrAk6gy6zLBdkedImwzrP6nL2AQQAv9bdxgj/o0WK3ruOYAoopKEMEVUAG1x0C+lQP8FePjosR2Hfz+ojpI50gQ9LlYAYDuEDWAr58VjdfngeQxK7sgFBP7ZcQof/L5NtEyU8DkhI/ocWUpcAgKBr1cewLjFMfa1rOhVC/kzpQyItfEiGIEHDx+h4ICFcYD4892qKJc7HYNjAQG2Q9gAtnBcPFeVD57nsOSerCMwO+I4+szZaW/IFELWMfTnFuMW78XXKw/apzitU0XUKZLZn6fMc2METCNw4+4DlByyOE792d3CUDFvetN9cEWA7RA2gH3yPeCD5xPYedBYBH7ZeBQD/9ptx2NR9xoomjU14xMgCAz/Jxo/rj9sX83wZiXQISxvgKyOlxHsCJDMd+jIZXFg+K1zZVQtmDHYobG0frZD2AC2dGA8VZkPnqeQ5H5cQWDq2kMY+d8ee9Of3qyEWqwG5wqUftmm37xd+H3zMfvcOlfPh4EvFPfLufKkGAGrCJy8chvVPl0Rp9n0ThVRm285LEHJdggbwJYOjKcq88HzFJLcjysIaGNEx75SGq0r5HKlK27jhwj0mLkdf247aZ9Zw+JZMKVDBT+cKU+JEbCOwMHzN1Bv/Oo4Dad2qID6xbNY7yyIW7AdwgawT44/HzyfwM6DxiIwYUkMJq04YMfDn9TCiKGCeIp7NyqCpImYucCVQ9t1RgQWR521Ny2aNRUWda/pSlfcJh4RmLLmIBImSIC3queLx1GfvaGiT11D00lr40z823bl0aRUNsPFLNp9GpFHL4vnCjOi2GBiO4QNYJ98+/ng+QR2HjQWgdEL9mDKmkN2PDqE5cFwP1GDqzRqGc5dv4tv2pVHUwc/aLyZxggQxzNxPadOGoJrdx4geeKEiBrWiJk+/PjQHL14E7XGrRIzZFlfxxu19dhltPhmQ5xKE9uWRbOyOQwbho5Yios37+GHDhXQgD3FbAA7eRY89/jx4ycke3784HgWp8YG8LO4a4Ez5yHzd+On8KP2BTUqkQXft/ePK/JCAxbg/sPH+LRFKbStlDtwQI/HlbT+Lhybj1xCpbzpxf9TiRhYHxlTJonHWfBQVhDYcuQSWn1nU2dc1rMmCmZmUQcj/MIPXsSrP2wUHycJSYC7Dx7h81Zl8EpoTkPIC/ZfgAePHjutZ2XPnvW6bIcY7yAbwF483XzwvAgud+0UgY/n7MTMiOP2emVzpcVf7/leDe7eg0coPNDG78nMBU630bDCC1+txe6T1/B6ldz4ZaMtGW7uO1URmod5Ul1H1bstl0WfReefI8QgCz6sgeLZmZXFCPGVMefQadoW8XHGlIlx4cY9hy/Mjx49Rv7+C0T9Uc1Lol3lPN7dzGekd7ZD2AD2yVHlg+cT2HnQWAS6/7ENf20/ZcfDX+Ryr966jzLDl4h5DXy+GDrXyM975gIC9cavwsHzN9GvSVF8uWw/bt9/iC/blMXL5YyviF0Yhpt4EIG5kSfQa/YO0eP896qhTK60Huw9sLpatPsMuv0SKUR8MqdKCmKFGNGsBNobUP3duf8QRQctEiAMfqE43uQYa4EF2yFsAPvkycAHzyew86CxCHSbEYlFUWeQKkkIrt99IH5IYkY0QYIEvlWDO3XlNqrG0hv1aVQE79UpyHvmAgJEESWNAvIAx5y9jp4NCuPDeoVc6I2bxAcCP647jOGx8tUs6uAY8fnbT+KjP7aL51eGlIlx5OItDHqhuGHy4NXb91FmmO3F+uPGRfFO7QLxsaV+PwbbIWwA++SQ8sHzCew8aCwCHadtxqqY8yiRPTWiTl0Tf40cWB8ZfBwjeuDcDdSfYKM3+qheIfRoUJj3zAUEyo9Yiks374l4R/KWLdtzFi3L58T41mVc6I2bxAcCE5buw6Tl+8VQLOrgGPGZW47h47m7RPhDuuSJsf/cDXzSpCi61dI3bM9fv4uKo2zCGd3rF0L3+vxcISzYDmEDOD6ebU+NwQfPJ7DzoLEIvDplI8IPXUSTklmxcPcZ8Vd/iDvcdeIqXpy8TsyHvDTkreFiHYGigxbizv1H+Pq18oL6iVThKCFuVrcw651xi3hBYOjfUZi+4YgYi6WrHUP+c/gRDJ4fhRxpkyFNskSIPn0NjqgcVeEMfq48wZbtEDaA4+Xhph2ED55PYOdBYxFo/s16bDt2BW/XyIcf1tokc/1BSWnjoYtoO8WW3c3qZa4dVzXh58eOFXD80m0M+TsKWVMnxcb+9VzrlFt5HQFVvGRK+1A0LJHV62M+qwMQX/LoBXuRP1MKEQax48RVEd5DYT565ciFm6j9uY1i7s1q+TD4RVZFJCzYDmED2CfPAD54PoGdB41FoOnEtcJrQj8Y5HWi6/KxLUujdUXfqsGt3HsOnabbsrv9iZv4WTo4t+89RLHBtoSf39+uAkoAkpjuHdGYxUX8dDPfnL4FK/aeE7Ob/Fo5vFA6u5/O1PfT+mr5foxfug/Fs6UWHNcRRy87vDHad/Y6Gn6xRkycmFFGvlzK94vwgxmwHcIGsE+OIR88n8DOg8YiUHf8Khw6fxP9mxbFvK0nsffMdfRqUBgf+DhJ6t+dp/D+b9vELNtWzIVPW5bmPbOIwMUbdxE60hbvSNR2KZOE2OOqmV/WIpjxWL3ltxtEuAqVL9qUQfNyxpy28Tgtvxxq3OK9+HrlQZTLnRZJQxKKcC66zRrwvL5nd/fJq3jhK1toVavQnBjXimPhCQu2Q9gA9skXnA+eT2DnQWMRkCwBxLW7bM85oRrmD56RWRHH0XfOTjHLFuVyYEKbsrxnFhE4cfkWqn+2UrRa3L0m8mRILjzCJGtEIRF1i2ax2CNXjw8EKPmTkkCpfNayFNpUZBEYI9xH/BuN/607jLD8GZAoJIF4fnWsmhdDXyqh24ReLOgFg8pLZbJj0qvl4mNL/X4MtkPYAPbJIeWD5xPYedBYBCqMXCrI4+mHNuLIZcyOPIGGxbNgSgffqsFNX38YQ/+JFrN8vnQ2kcTFxRoCB85dR/0JtuveNX3qIHeG5Agbsxynr97B0BeLo2O1fNY65NrxgkCFkctw4cZdMdaIl0uifRUWazACfuBfu4TAS50imZDgueewfO85vFY5N0Y31w9tUJXj/En1Ml4OloNB2A5hA9gnZ5APnk9g50FjESg5ZDFu3H2AiW3LYv/ZG5i88oAg3icCfl+Wb1YdwNhFMWIKDYpnwQ8+Nsi1WNy69wDJE4f4EiKnY+88cQUvTV4v6m0ZUB+ZUiVB6+/DsfnwJXSqlhdDXtT3kjntmCt4DYHHjx+jyMBFuPfwkRjD02INN+/SuU2I557zLc+3pwDsPXsH5kSeQOPYREHiNHcU2rB633m88eNmMXztIpkwvVMlT03lme6H7RA2gH1ygPng+QR2HjQWgUIDFuD+w8f47vXyOHf9rqAUypYmKcL7+ZYl4PPFMcIYp1KrcCb89Kb//FCNXbQX3685hKkdKqBO0cx+e5Y2HbqINrFMGruHNRIxwH1m7xBe/vrFMmPqGxX9du7BOjE1cZEwcMRpaxWj/WevC2rBKvkzBIzh98Hv2/DPjlN4uWx2PHwM8d/Ny+XAFwYhU0ujz+LtWJlpCpv4vUsVqzAGZH22Q9gA9snB5oPnE9h5UAAPHz1Ggf4LBBbEN3r3/iMhKxqS4DnsG+lbNbjh/0QLzloqVQtkwG9v+88P1UuT12HniavoUjM/+jct5rdnaVXMOXScZmPSODi6KRImeA4ya75wlpRY0qOW3849WCd25uodVBmz3L58Tyakzgg/gkHzo0TfO4Y0FLy5z3ohY5aM2jYVcuH+w0eYt+2kw5Cp/3aexnu/bRXLLp87Lea969ubLn/Bn+0QNoB9chb54PkEdh4UAF3jFx+8WGDx29uVkSxRQjT/xpYgEjGwPjL6UA2u37yd+H3zcTGXCnnSYc47Vf1mzxp/uUawZTiKNfSHyS7afRrdftmKxCEJxAsNFSkdmzRRAuwZ3jhgrsL9AW9PzCHmzHU0+tIWt03lg7oF0athEU90DfVWJVAU5jr8uFkkvr0RlkcIvsyMOO4wh+GvbSfRfeZ2gSepX/73YQ2PYPusd8J2CBvAPjnDfPB8AjsPCgjOX5LKpTLv3apCIKHqpyvEv//7sDpKZE/jM5w+/H0b/t5xSoxfOmca/P1+dZ/NRTtw3c9X4dCFm2hWNjsmtvXfLPJ5W0+g56wdwtNHHj8q245dtr/kbO5fD5lTJ/UbXHkiEPHZFKctS9da+dGviWduGWT4C/Xdr0lRdDWQC36W9kHGtHetmR837z0QCXF1i2bGjx31w3tmbTmOvnNt7DIFM6fEsp58C0JYsB3CBrBPvvd88HwCOw8K4NSV23aDl+SP6Qeh8MCFAptpHSv6NL61808RWLbnrJhL0aypsKh7Tb/ZM0kd5+9xtL9uOooBf+6Oo/ymcgPP6RaGCnnT+w2uPBFgSdQZdJkRaYfCk8mKlPxFSWBUAoVZpdnkdTb1t7oFcf3uA0xbfwQ1CmXEjLcq6x6nGRuPYtBfu8VnudInw9q+dfnYsQHs8Aw895hSU7l4BQE2gL0CK3dqAoHDF26iTqws6PJetVAgU0qEjliKizfv4dMWpdC2ku/4R1/7YSM2HLwoVpE/Ywqs6F3bxIrip4qkqfL3JJqpaw9h5H974uBHj3Ji/rh57yEmtC6DFuVZZCF+To25UWZHHEefWP5rauFJTm4ZukP9Eif06j51zE3Kj2vJNfVpVARXb9/HlDWHUDlfeszsGqY7a+IMJu5gKplTJcHmAfX9eHXxNzW2Q4yxZgPYi+eQD54XweWuHSKw5/Q1NJm4VtRZ/0ld5EibTPyb/k7SyB/6UA3u5a/XY/vxK2JuNC+an7+U0kMX49qdB34XmqHFR5WJXfDRk1hHuccf1SuEHg0K+wusPA8A8qVFgkHJXZ+94hkVRAp3orAnWXYMbog0yZ/tRLja41biyMVbGPh8MVy+dU+owjlKbvt21UF8tmivgEANDQr2w8d2CBvAPvkO8MHzCew8KCAMTDI0qcikt47TNmNVzHm0q5wbowzI5OMDvIZfrMa+szY1LOKvJR5bfylFBi7E3QePkD9TCqzo5T+eaS0+RNf2zaqDCM2TDnOVJMKuMyKwOOosK+z5y4FS5jFhSQwmrbDR/1FxROllZfr3HjyyhzfJdr92roxqBTNa6cbv6kphl5EvlxTiIV8u2+/wxXTisv34Ytk+sQ5KBN07wpYcGuyF7RA2gH3yHeCD5xPYeVAAGw9dRFsNT2zfOTswK4J4YrNg6hu+U4Or/tkKnLh8W+xT2uSJsH2wLYnL14VCCPL3XyDkhLOkToJN/f3HMNdiM+yfKBETWb1gRvzS+UlM5Kj/ovHD2sN+x67h6731h/GHzN+Nn8KP2qfiqVjdk1dug2LXqSR4Dnj0GPi4cVG8U7uAPyzb5TmUG74El2/dx7hXSgse83GLYxzmDIxbvFd4iSUORA8YKKIgLoPIMcAOoeMQCHdOlpO2bAB7EVzu2iECqirS/lFNkChhAoxfEoOvVhzw+fW+el1LylXRwxv7xW4S12ihAbZEQRKWIIEJfy2SSk6rpCcTgfzNs+6vOMbnvLr/sQ1/bbexn1DxlAri1mOX0SKW4rBi3nTYcuQyni+VDV+3e7YlxosPXoRb9x7iq1fL4fTV2xi9YC8KZEqB5QY3M6MX7BFxwrLI51587rE/jsV2iPGusAHsxRPLB8+L4HLXDhFYHHUGXWdECoEE8oRQkcYRUaJt7O87NbiigxYKXk8qJMxxIHZ+vt5Sko2mJDIqpCZ7yI89SB/9sQ3zt5/CS2WyY9KrT+ja1Bcf4gJOljihr2Hl8WMR6DRtM1bG2JgaqHhKrldyQtN3qXv9Qvh8yb5nngWBbmNIyIe82SSVfuLyLQz7J9phgt/Qv6MwfcMRO75SITHYDyDbIWwA++Q7wAfPJ7DzoIooQorECREV62GVNExkFJN4Av1/fBdVoU6OTYZmAh/MRbt2lTuZPosa1ggpkoTEN0SmxuvycwSWxKpkqYlUKvvHkh41UThLKlP9cSXvI9D8m/XYdsyW/EnFUyqIP4cfscucf9ayNEhAgsr2wQ2QNnli7y/MCyOocc0/v1kJxy7dwsC/diN7mqTYYCDl3m/eLvy++Zh9NpED6yODDwV/vACLS12yHcIGsEsHx91GfPDcRZDbu4rArIjj6DtnJ9KnSIytgxrYfhCVxLjNA+ohc6r4F0pQvaxybXtHNEbSRL73VNI1a9gYWywlFV9hZGbP2/9vE9buv4COVfNi6Esl7E3IcCAPu/Sc0TU7F/9AoO74VTh0/qZ48aQXQQpXmN3NfRVEGftaJldaTO9YEeViBXB+easyqhd6NhPhrt+5j1JDl4iNm9U1DIcv3MDHc3cJBUtK6tUrvWbtwNytJ+wfbfikLrKnTeYfm+/DWbAdwgawT45fMBw8MhrmRp4QnKP8sHH9mO06cRURRy+hfZU8CEmYwPWOYlvOCD+CQfOj4nhMVAPv3w+qo2SO+FeDO3f9DiqNWh5nfaRkRrRFvi5HL95ErXGr7NNY2bs28mVM4etp6Y7/yrcbEHH0skh0ooQntUgxj0EvFMdb1fN5df437z4QcZdEU6Ubm9fDAAAgAElEQVQtxD3dISyP00Sk/3aexp37D9Ey1L94i8MPXkT06WviJcMTtyUVRi7FhRv3RILl2Wt3QQbr/Pequb0/vWfvwJzIE3aZYJlk2rdxEbxbu6Db/et1sGDXadDet6qQyyv9n79+FxVHLRN9E0YHz994SvlQO/AHv2/DP7EKk/SZP39/vQKaQafBYIe4iifHALuKnIl2wXDwBv61S0hUvlopN8a0KGUCFa6ih0D9Catx4NwNj2VvS85RMuDoh4AKJXmRGhyxHPzYsQLqFo1/7+CRCzdRO1agQ+JANGiUtOXrsv/sdTT4Yo19Gr56STCDw/OT1iLqlD6nc9sp4dh46BI8qTRmNKfvVx/EmIU27lW98ue7VVEudzrDzy/fvIcKo5YJj+javnWQK31yM8uPlzrSYKUr+JqFM7k1JsW00nfv/sPHKJkjNXafvIZi2VJjocLh7OoA8jaAXp5HvFwS7/wSiYW7z6Bpqaz4pl2oq92a2rMVvWohf6aUHh+DYn6rf7ZS9Lu4e03sO3sdZOCqIV3aQSUFoPz7ou41UDRrao/P7VnrMBjsEFf3hA1gV5Ez0S4YDp582NYpkgnTOlUygQpX0SLw6JHtx/HBo8fInT45VvWu7XZM7OQV+0UyjFZqWP6o08sKvbTEd4k+dQ1NJ9kEOmSRQh3xPRfteLtPXsULX62z//mPLlVQJX8GX09Ld3x5nU4iAZ1r5I9TR34nW5bPifGty3h1/tLooNjMQkq8sZTlpQQmR2EYh87fQN3xq8UcZ3cLQ0U/kW++++AhigxcJOblCVW9W/ceoPhgW4IlJb8RHzfJky/rWcvt/ZGKab0aFMYH9Qrhm1UHMHZRDHKmS4Z1H3teZCbmzHU0+tL2ougtWXVyBpBTgMrqPrWFgE+3X7YiccIE2DdKn99Xm2RInmPysgd7CQY7xNU9ZgPYVeRMtAuGg/fW9C1Yvvecx+LZTMAacFWI5J0keGXxhMfp88UxmLzyAMrmSou/lGvWphPXimtdyhbvXj/+lcIijlzCK9+Fx9lDMvjz+kGoQeTRy2j57Qb73P73RgXUKxb/XnIzB1wVCXi9Sp44TSTfc8PiWTClg3f5nquOWY5TV++gX5Oi6FrrCe+sSmH1YpnshktSX4i8ZUyZwVNbRw3VGd28FF6r7N7Lohp+9EpoThGyQC+7a/q6L1ks+XLHtiyN1hVzYe3+82j/P1si3LZBDZAuhWcT4TYfvoTW39u+w5+1LIU2Fd3DRm9/1JfRjf3qIerUVbz1U4SoeniMPr9vu6kbsf6ATWKdCsUOV8qX3pXtD6g2wWCHuLphbAC7ipyJdsFw8F6fugnrDlxwSFBuAqqgrkIP9+cnPfE8Ni6RFd+1d+/qUgoiVM6XHjO7htnxlV4S+kGnH/b4Lmv2nbdnqcux/YWtgGI+X/1hox2SiW3LolnZHPENkanxyg5fgiu37mN8qzJPxc6O+Dca/1t3GGH5M+D3LlVM9edKJfXF7bfOlVFVUR6TXM+ftyoDMviMyrZjl9E8lsPWn/BWw2E8EUutSpNT3DbJ9mZLkxThBowGZvdD9VRP71QRtYtkxpVb91B2+FLRhSdeprVzkWwy9HdvyaqrL6PEZrHzxFX7c8OI37f1d+HYfOSSfboz3qqEGoXcC10xuw/+XC8Y7BBX8WcD2FXkTLQLhoMnHzo50iYDXWVzsY7Ayr3n0Gn6FntDSrgJ/6QuMqd2naVh0F+7Be9vrcKZ8NObT0JTPpm7E39sOY76xTJj6hsVrU/WzRaSs1Ttxl9ibVfFnEPHaU/2wROePzfhMmwuJZu/aVceTUtli1Pvi6X7MHH5fpTKkQb/fFDdW1PAyphz6BSLlzaR0ZGHWp2Q+tJBkrdab7bXJu+kY/Wmok+jInivjnvJZKoyIxnU9JKSIUViRMYytLi6zuOXbqHGWFus7IIPa6B4dlvMa42xK3D80m14Yu7auc2OOI4+c3aKP3tLVn3DgQt4beomMQbxWRODjXw5NeK3bvb1euw4/oRmbmqHCqjPLCgIBjvE1e8PG8CuImeiXTAcPPnQSZ00BDuH+q9ylont8lmVPzYfwyfzdiFVkhA8BkBUYb0bFsb7dQu5PCeja/AJS2IwacUBrxtHRhOft/WEyOYmoQlKxqMy952qCM1jnCjlMggWG6qeLWo6oGkxvF0zbnytxS69Up1ixkmymcq0ThVRp0jmOOPIBMi8GZJjVR/3r9iNFvHV8v0Yv3Qf9Map8/kqECexM++p+tLxSZOi6KaEUXgFPJOdLt9z1n7l/mHdgujZsIjJlvrVVGEakval70CqpCHY5eYzU/WUqry37/26Ff/tOg1P3CZpVyTPF/3dU2p22jFUpwDxhJPanQydMmKNaTJxrYgVluXr18qD5KaDvQSDHeLqHrMB7CpyJtoFw8GTCRikY8Da6yYOhU6VScv3Y8JSW8IaGYK/bjoG8qhTfKCr9Esf/r4Nf+94Winsl41HBaF85lRJsHmAPp+ma6sw10qOT7RnV2/fF41+f7sKwgr4Ptns352n8P5v2+wL8VWctDMkiX6qRKxinV6i3qwtx9F37k6PeBgdzeXtnyOwNPosKMaX5GrVIp8LzjyQi3afQbdfIkXT9+oUQJ9GcSndnGHhrc/lixr1/3aNfBjwfHG3hlJ5uUc0K4n3ftuKJCEJEDNSP6HL7GALd53GO79uRaKEzyFmRBN78iyFWHy2aK94jnj6Zk6+RNMcy+RMg/nve/6WQa5LJr2RZ5ecLVSIB5j4gLWl3vhVOHj+pv3PX7Qpg+bl/Itaz+y+erJeMNghruLFBrCryJloFwwHT2ajExwsPWniUOhUGfDnLmH0UrgCcXfKeGA9757ZEWR2fqvQnBjX6gkTABksZLjQC8v+UU1dNrDNzkNbb8qagxi9YK+QND168Zb4mEI0aO2+LsRn3Wv2Dvs0PGH4eGNNauytXqa73XgISSAU/7xVZJhD/6ZF0aXmkwQ4Gk/eDDnzns7ffhIf/bFdTJE4g4c3K+mt6Vrqd9r6w0J6l4qkF7PUgaaySktICYNdZkSK7+ChMc+70y2mrz+Mof9EP2Xorj9wAe1iQwhICIcEcTxVhszfjZ/Cj4ruvCWr/te2k+g+c7vdS64mS1JSXNY0T4eHybAPuc5PW5RCWx8w3XgKZ0/1Ewx2iKtYBZUBPHv2bPz666+IjIzEpUuXUKBAAbzzzjvo2rUrEiSwiQ907NgRP/3001N4Lly4EI0bN7aEczAcPEm6TsAYPZgsgRaElTv/FIFle85CGqvSeHDnevGNHzeDqKi0P947T1zBS5NtnpTN/eu5FWfsylZ9uWwfvlxmi0/ddfKq6MIZVZYr47jShmRUSU5VFn/ltlbjPpf2qBmHfozmrrIAxIxsjCQhnlfZU4UK9Dz4xBJAbAFda+ZHv6bFDLdDequpQotyOTChTVlXts7jbeQ5pY61L5GuDDZ+SQy+WnFA0HL1qF/IHmtOt2au3vLQPMjLS95eLdvL1Vv3UWa4TUnN0y+Y3f/Yhr+2nxJ9e0tWXYaFET848YSrSYlGfNGVRy8TAiOyDHupBN6omteV7QqoNsFgh7i6YUFlAFepUgV58uRB8+bNkSVLFqxcuRJjxoxB9+7dMW7cOIEhGcBr164VhrJaihUrhjRprClnBcPBI7Ue+jGkovdj7OrBDKZ2zSavw44TV/F+nYLo3agIpFFAPy7rP66r6+1whk+b78Ox6fClp65vz1y9gypjbEps/7xfHaVyWjvTzsZ19vnoBXuEchixU2w7fgUk3auXyOWsH298/tOGIxjyd5S965fKZMckzdW+N8a12ieJAjSMFezQMwZUyWuj62KrY2rrqzGaO4c2ROqkcZX8Ovy4GcT48UZYHgxz4NX9OfwIBs+3YV6/WBZMfcO7tG1m1z307yhM33BEVH+hdDZMfq282aa69QbP342fw48KQY1uNfPHSfBKltj1FxQp/9uoRBZ83z4udrXGrRS3LO7mE2gXpOXb9YaQjfRsSy5jVUBnea9aIJVBbZF0cPLvejcTbm3iM9o4GOwQV7cmqAzg8+fPI1OmuFetPXv2xLfffosrV64gSZIkwgCOiIjA7t27XcXU3i4YDl6ZYUvssZz+kszk9sbFcwdVRi/HmWt3MKJZCbQPywsiza88ajmu333gMs3Qy1+vF5nT0qiWS3rw8BEKxarB+SJLWioHknBKxJHLYo1ftimLl8v5nm5MhmdIrOoVzYz/dYx/pgxnx89ZPCTJxtaLFZfwlhzsxGX78cWyfUIqWioNqvPu8nMElkSfRduKufBpy9KGS/phzSGMWrBHfK6l7HOGgzc/7zlzO+ZtO+kxw1zG5FO8NIV6tIrlwnZXBlyqwOmFj1CcMclM6xnH7mDX/Jv12HbsCduCN1hcpMKgFAs5eeU2SOKbipHCW8khi0UCsSxSGMSdtQZC22CwQ1zdp6AygPVAmjFjBjp06IBTp04hW7ZsbABbPEnFBi3C7fsPRSvJQ2mxi6CuThKwpAJH//99+1A0KpFV4CE9RqSwtfbjupavSWUSkp73R3rtfUHzJQ2L50tlA1FDXbx5D5LA39cHQarnyXn4k0GmYqNSakUNa4QUSULiQHfu2h1UGu1dL78M2zHykpNs7T87TuHlstnxZdu4CXLqZCWTBP2teLbUWOABaWBPnCMp8EN9VS+YEb90ruxWtzIk6fUqudEqNJc9octd72nDL1Zj39kbunRn360+iE8X7gU9Qza4yTesLl7N+6C/e0NWXb5glcieGv99WAOqMImRwV14wELce/jIPlXty79bG/gMN2YD2Hjzgt4A7tKlC+bOnYtz584hYcKEwgCeOXOm8AbfunULpUqVwqBBg/Dyyy9b/goEw8HL3+8/PIqls5r8Wjm8UNpY9ckygEHQQH2w//luVZTLbaMD23vmGhp/aZMMduUHpu7nq3Dowk1dKq8XvlqL3Sev4aN6hdCjQfyqwXWbEYlFUWdEXCUJqJy+egcjXi4pYpV9XdTsdppLyRyp8e8HNXw9rafGV/l3iSIqAWVTKeX2vYcoNtgm46sVqPDUYuSthRFVXJ/ZOzA78oRTGi6pWEjz8pQymifW+Mq3GxBx9LLoqkKedJjzTlW3ulVvZIi3WcqBb/ikLrKnTeZy3/IGbuwrpdG6Qq44/ahcuipFmsuDxTaUcuqyH2/Iqo9dtBffrDooWHHoZvHyzXsoN8Im7jHv3aooH/uclHN4/Pgx8vWzUQPK4q9JrO7ib7V9MNghVjGR9YPaAKZQh7CwMAwZMgQDBw4UmEycOBEhISEoUaKECIug8IglS5aAEuheeeUVhzhfu3YN9D9ZTp8+jUqVKuH48ePImTPw6FjoOr3ggIX29XrjQejqwX5W2qmSn9ofwxbfrMfWY1dc4tqUErUyrELF483pW7Bi7zm8WikXxrQwvp72BobyypZiQ1ftOy9iFJ1xxXpjHnp9jlmwB9+vOWT/KH/GFFjRu3Z8DW96nAW7TuPdX41ptMgYKDRgIR48eozvXg9F45K2WwVPFfWlTY+GjcZRQ12mdXoixKKdw8h/ozF13WHx53TJE2Hb4IaemqZb/TSYsBr7z90QfXjiRUh9Ia1TNDPqT1gt+nZHBvzO/YcoOsj2oqOX6EY0g2QgU3GHUUYFks4W3Vjdfxjr9QDQo35hfFTfdc5yvY0a/k80flx/GNUKZsCvnauI0AYKcaAys0sVVM4flzZRVcQLSfCcOPv+xCri1mF0szEbwMYABq0BfObMGVSuXFkYpqtWrUKiRHGTOCRkjx49QtWqVYVhGx1to8UxKkOHDsWwYcOe+jhQDWCVj5QWzUkH1p9UK/aexZvTbRr3WolPGR8pE0Gs9B46YqktvEDHM9R79g7MiTzh8dhAM/OTnjWSg10WfVYYGR83Lgr6t6+LmvhEc/EVV7IzHCRdW9rkibDdwGCUCUHOpIidjaX3uXpmdw1tiFSaBDhqIw1bZ3LM0lCmNmS40HfgOVJJ8XGpNGoZzsUm9xbIlALLe7n3ImT/PrYsjSr5M6DmOJt6mzuJwyobiFFcrDa/wF1YKT+h+GCbIZo0UQLcuf8I3pBVl9SQdYtmxo8dK0I1cH/tXBnVFNltmsv1O/dRaqjN2KfvBcmEt6mQC5+9Er8v+O7i6432bAAboxqUBvDVq1dRu3Zt3LlzB+vWrUOGDI5J+Ikhom/fviIkIlky4+uqYPMAq9dSdMSccX5648v9rPcpqbcypkyMiIEN4ixHSwVkZa0lBi/CzXsPMbFtWTQrGzfBTPJ41iiUETPeci+20cqcqK5Ua6IEFQqFiDp1zSseJKvzovpEgUb7IUuKxAkRNdwa9aEr41ptI8VEHMV2Sk7UIS8WR6dq+awO4bC+pAjLnykFVhgYhuMW78XXKw+ifO60mPduNcP+ZKiErLB3RGMkTeQ6K4KnFkpeTmIooeKumITWI0+UZZKJ5b8Pq6NEdteYWFS55m2DGiCdDtevo1wAV7A6ffU2wsbYktEklaE3ZNUlu0XTUlnxTbtQOFM/vHjjLkJHLhPzypU+mZCBdhZ/7sr6n8U2bACzAWxHgIzeRo0aYf/+/QgPDxe0aM7K2LFj8fHHHzs1gLX9BPrBO3vtDirHJtvQ2jtWzYuhL5VwBid/riAgjYli2VJjoSYBSEsGbwW4Av0XPJVYJ9tTYgwlyHgittHKnKhu7XErcSQ27IGSpPSYKqz26an68keXFLruxho/ejG2nhrP1X7kzYAjA7TpxLWIPn3NZRYRR3Pr/NMWLNtzDs3KZsdEgwQ3mdzmLLFNJsvJ8TYPqIfMqZ4WOXAVK1faqaEF1F7v5dRKv+r1PXEmF86S0m6sqXH/VvqkusTwQEwPpJZGfM96nnPJx+ypeFiSGqaXWCrE20xMGaVzpsHfHlaDkwwWzcvlwBex3NDymTalfSgaxiYLS8xUw5zOHJ39JiWz4tvXQ63CGnD1A90OcWfDgsoD/ODBA7Ro0QJr1qwR/ytd2vn1CIVAEH8weX+tUqMF+sE7dvGW/SqPDmHL8jkxvvUT1TF3DmawtO3/5y78tukYahfJhOmaWElXFb3U2Gw9Zg6zxok39kBeLVO8+J/bTgqxhC4186O/A7EEb8xDr09pjJFn9dTVO6KKP6obSulsmSGvtxYjHmhPYCn3cODzxdC5Rn7dLs0Y6dRQ0qXJTow4Xj0xb7N9aF/s3b0JUCm8FnxYQ3go5XX9rK5hqJQvvdmpxakn1eoceailXHXrCjkx9hX3n80qAwmFLpEQR5bUSbCpv2dl1SXLiJqnUHTQQhFy8fVr5fF86WxxsDh68SZqjVsl/kbsLcSBLsMnXAI3gBoFuh3izlYFlQFMim9TpkwBeXRr1Iib3V28eHFcvnxZsEC8+uqrQiWO/k1JcCSYQUwRJKBhpQT6wTtw7jrqT1hjh6Rh8SyY0sE/iOyt7JMv60pvml68mio2YMUTqXqc9JKUpCxr3gzJsapPnXhdvuTqpNAMikNeu/+C39wcSGNMepAImE396yFLat96JLUbJNW/KuZNh9nd9NkJ5LnydKKjahzqJSPJuUqBC2fhA1IwQ7b7671qQtXMl0UVGqF5kCANKba5WlQZX0p0JVlimbz2y1uVUb1QRpe6ljc5jsJMZIiJp7iAF0edQdcZkQITErCh//aGrLpMllVvFUsNXYzrdx7ohnWpv0UkqELKmjKBziVwA6hRoNsh7mxVUBnAefPmxdGjNg1zbSEjlzzCnTp1ElLJJJqROHFiVKhQAZ988okIm7BaAv3gRZ26iucnrbPDUrVABvz2dhWrMAV1/Re/WickgfXip1UaIyuxkWo8nN4Vq4w7ju8kL4qFpGtMos0j+WOKcV4u2ChygzzCvi4dp23GqpjzgveVKNqo+INHUouLTNZzFMPdY+Z24WEnTxl5zDxVKHGx888RoDy1XUMbIaWGg1iOI9UMnYUPyCt62c4dg9BTa9xy5JJdqEL2qU1QtTJW+MGLePWHjaIJ8TYnS5QQ+fvbKLumdawIYoVwpUhObUdX/TIZ0VOc1rMijqPvnJ2CsYPYPYjejYqnZdVbfxeOzUcuoWut/OjXxCalXX7EUly6eQ/jXimNVhrKN/W3SIZmOHpBdAXvZ7VNoNsh7uxLUBnA7gDlSttAP3hbj11Gi2822KGhpIh/PqjuClRB20ZeJ+tx4UYevYyW39rw3T64AdImT2wKJ/XKleKKKb5YLfO3n8RHf2wXxgtd8cdXUWMrKZObkrkW7j7jN6Ezr/2wERsOXgSJO/y945SA5e/3q6F0Tt96JLX78/GcnZgZcRyOblxU6d2f3zSmIbO6918s3YeJy/fDGTOCPGOpkoRgl4Mz1uzr9SBlO1m+bVceTUrFvd62Okd360sjX+1HT+7Z7DiLdp9Bt18i47BcFOy/wG2aunZTN2L9gYsO5aZluFPRrKmwqHtNs1M2rCdvj0gB8Le3K9sT4jytBvfS5HXYeeJqHK7yyqOX4ey1u9AT8Nl27DKax/4WEf0ZyU57IzbZbQB90EGg2yHuQMoGsDvoOWkb6AdPjQcjKHxxpe7F7fN616T+VmiAzSOql9ihcgRbuYp3JoW7NPosKDaQri7paje+aKfIe0NeHCrkmZ6+4Qjmbz8Fkof96lVjtTCvb0TsAPSyQS8ddO1Kc6NCSUthBRyzxMTX/OQ4UlbXURKaWRYGq3OXHNJqcpJeH9LoS5SQqM2MwwckS4Hswx9UASk0h6gC1eJOcp70hmdIkRiRg2xML1JBk849nX9XiuQq7tOoCN6rU1C3CxmK4ik1uPFLYvDVigMokyst5nQLE5zAjx8D/3ujAuoVy+LKMnTbSIW7vo2L4N3atrVV/2wFTly+jeHNSqBDWN447TYduog2U2xe9vfqFBAMJEWypMLiHu4b/R5blI86CnQ7xB1Y2QB2Bz0nbQP94K3Zdx4UwyeLs+tOL0L9THatStbOf6+a+FFRy4FzN+yE+av71EaeDClMrVMbc6hVmlp/4ALaTd0k+rISWmFqcAeVVN7Sxd1rgrxJpBbmqfhEd+cnw1FIIW/Siv3ih31qhwqoX9xzP+zuzpHay8QmR/G93646KBKUCmVOiaU9a3liWFAIC0ksn79+16l4iapWRy9ZFDOqV+p8vgqHL9y0f+QPoij/W3cYI/6Ny/m+pk8d5M6Q3CUcp6w5iNEL9kIVVpEKbuNblUHLUNdEkkoPXYxrdx7AEdez9MS7m8gnFz7or92YsfEoahbOBLpZqDByGS7c0PfKugRWbKNa41YKkZzBLxTHm9VtNH7yrOglX67dfx7t/2f7LZLJeeyQsYEZ6HaIO+eMDWB30AtyA1h6EiUMiUMSYN/IJl5ENLC63nXiKl6cbIuh3tivHrKmiZtsdeLyLVT/zEaYTwZjkaypTAGghqZsHdRAJN2oRb0u1Pvc1CAuVFKTi9b2rYNvVx8UDBh1imQS8YS+Lo2+WIOYs9dBXqdvVh4U6lN6PMq+nufrUzeJGOVO1fJiyIv6tIOSKzhr6qTY2L+eR6Z85uodO3/t7G5hqJjXmL1AjXuNHt4IyROH6M4hbMxyIYctS/f6hdC9fvzKc2snNmHpPhDTBsW5Xr51X3y8pEdNFM5i7vun7U964ym5j5L8qEjD8dMWpdC2Um7L+6PKXc94qxJqFMqk24f6IuJOHLPsXN4+yFsbSbfnaVl1Ge4wqnlJtKtsoyqVXmE94Zzle87irZ9sgkIUTkaGerY0SRHezzNn3/IG+VEDNoCNN4MNYC8e1EA/eJKHUoWQ+CiThPieyN6L2+qxrtWEov0jmyAkYYI4fZOnreIoG7m7lVhUbdJNCk2iUsyZ62j0pY29gwzRXOld82xZBUI1vCMH1hdXqRRqQElnv3SOX0EOvblLjmLyMP2w9pCIN1R/gK2u11v1ZajGu7ULoG/jorrDeNrzR4PIF15KgNs9tBG050qdiNmXLJnYJNu+VT2f8C77ssgkQ4qb3XvmuuXvn3buUu2uVuFMQrKYSrVPV4Bi9fWkys2sXaX9cvRyrP3OZUiZxEz3hnXe+HEzVu87j9er5MbIl0uh07TNWBlz3uOJrGWHLxFqbqqH/PlJa4VwTs8GhfFhvbjSy3bKyIQJxHe2z5yd4sWfXvCDvQS6HeLO/rIB7A56TtoG+sH7c9sJ9JgZN1aODBt3H7Je3BK/6pq8n8QDnDFlEkQMfJpHU5X3tMIXqnp9Dox62rBWQxHc8WxZBVPLakEJVd+vOYRKedNjVrcwq915vH7VMcsF/y95kIhj9dD5m34p7y29br0bFsb7deMaAhIUVa7YUQiCFRAnLInBpBUHTIVVqGE44f3qIlsafQXN4oMX4da9h/ZpeIqv1sq6tHUlgwZxcxMrCBVHlG/OxpL80mrMtvqyZcSl7KhflanCUYLsofM3UHf8atHVil61kD9TSmfTdfg5sT6QeA3F2fZpVBSfzN2JP7YcR72imfG/jhXd6lttrMf5K8fWY8xRE3tHtygF8lR7KuzDY4vyUUeBboe4AysbwO6gF+QGsJTqVWFY1bs28mY0F6vqKejpap28ms+XyoYEBrGGnhrLk/3IjHojQQOSYqUkEyqOrjm1c1Kzzg/o8JeqNGnxybuqeryJ15iummVCDcVA+7pUGLkUF27cAyVi/bLpqMhCJ08TeZz8qTiKhZTzVA2kHUMaIk2yRLpLoJchqkt0ac5ubqS3j2imJsSqcxnhohpeK3vXBrEGaItKi5cqaYjgeJXSt77EWyb6ETc3sW1QIc8teXBdKZLrmNgJhjcrKbqQCWx61/lmxvh35ym8/9s2UNhZzAh9FTjqR/2uz3u3KsrnTmfYPYVknbxyC41LGrNw1P18FQ5duIkBTYvh7Zr5xXeYwkVK5kiNfz+Iy61vZh16dehcEE2cNga/1XcbsOXIZXSrVQCfNIl78zE74rjd60uUisRPHJLgOeg9/6ch7F0AACAASURBVFyd17Pajg1g451jA9iLpzrQD57MMKZM7/sPHwsk/3m/OkrldE3b3tWtqDF2hdB+t2IkujqWJ9v1m7cTv28+bqhYpOXNbWAyGcsZzZkaP/hb58qoWtA1In6rWGiv5SVFk54MtNW+PVG/1JDFuH73Ab5sUxZ/bDmGjYcuwR+u5LVrqzJ6Oc5cu+MwPGPvmWto/KVNsnbdx3WQM51+mMurUzYi/NBFfNayFNpUdByLKsc1k6imUvEt6l4DRbPGpeKjed1/+AiFBthe8ChBjAwrR9zGnthjM320+GY9th67gndqFwAp2j149Bjftw9FI438rpm+qI6kevugbkH0alhENJPX+T3qF8ZH9fW9+I76l4l6pCq3tm9dw6oqxtM6VUSdIvqcw48ePUboyKUi5tmRPHPoiKW4eNP2kti6Yi5BZTjwr93wJKf43QcPUWTgIrEmlRda0hTqfSd/3XQUA/7cLVTpPmtZGh2nbRHt9W7AzO5boNQLdDvEnX1iA9gd9Jy0DfSDJ+VOKdGGfpCpxKdBReOp3LJjXymN1hqCdC9ur9tdS09T24q58GlLfVluV+iSZm45ho/nUmhFYkQMfDoGzsjD4vaCnHQgbwwypUqCLQPq4/vVBzFm4V6nnLLenpfsn7zt5HUnLtq5W09g2R4S6ciFMS2cS6bH1xxpHMkg8EWbMmheTp9BQE2gNDJAqS+ZbGQmianE4EW4ee+hqcRAM7cMaogPyQGTLDYxofj6NqDe+FU4eP6m8DJOXnHA7WRIPY+9vM5/v05B9G5kM4qtlDEL9ojwodA86TD3HX01QNmfmX1TKQr1hCaoL3pu0AsLvRB893ooGpfMao8Lp4s3SoDW5jFYWZOse/X2fXHGqajJljL++I2wPBgW60mXbaQsNL0QjG1Zxi484igB05W5PYttAt0OcWdP2AB2B70gN4C/XnkA4xbHgJJF9p+7AeK1dcdT4spWHLt4CzXH2ZgSXE0ocWVcT7R54au12H3ymsNr9nLDlwivjCOqI+1czMjQmvlR9MQa1T6k1ypPhuRY3aeOiLMd9k80nHmxPD0Pvf7oxz1fP5s6F3GakhCGP3EUq3MuPGAh7j18hO9eL294XW1kRKj90JrJ00Z9qYpbRniTip/Z77gzOW4ag+iziA2BCoVgUFKtM4GN+DgLkqGBrtLHL9kn5mnGQ240N/kdVl/QpQJel5r50b+pTenMSpFxymZCRmRsux5/rhxTpVw04hVW91TyY+88cQUvTbapwVnhKne01nPX76DSqOWiinqj+ETe+2nlSPVlelyrMnaBpm2DGiCdhgXHCs6BUJcNYONdZAPYiyc80A+ejP8qkzMNjl66JbJ2rRhqnoA+4sglvPJduOhKjx/SE2N4qw9ieCCmB0dMA5ImauTLJfF6FRsdkLNi5x3NlAIretXWrS7HdpWGydkc9D6fvGI/Pl+yTyjTkUKdem25qf/TSYCujOFqG/XalUJpSKGOkhTrFs2MHz2Y3OPq/GS7Bw8foWBs2MD0ThVR2+BKmwxVMlip/NixAuoWfZrLWPXAkvjH0Jf0KdWoD7Pj6s3TKH5W9VJL8RFPXqW7grX6UvBNu/IYvWCPEF8Y9lIJvFE1rviCmf6pP9ov7YuDpLJzhrvRGDIcwEz7JhPXYs/pa+jVoDA+0LAnyP7J+05GORU9Dyv9XQ1rWfBhDRTPnhpnr91B5dE2Y9UKU40j7NQk3aU9aqJQLP3cO79Eiu/lK6E5xe+MWtRwqs9blcbzk4zpJc3sWyDVCXQ7xJ29YgPYHfSctA30gzdm4R58v/oQSHOdQiAoDnfIi8XRqZqNuDw+ikrF5igrPj7mYmUMMigKxaooORJbkNniZuIu5fhmYmsl0byVfq2sT68uCTOQQIO8tp0VcRx9/YSuSMu4sWzPWUwhhop86UEMHP5SVC+cM2aCkkMWO7y+V29PnIV6mPHoajGScr9Gt0KqYiF9d+nlKFmihNgzorHP4NbGxw/+OwrkHaVwCEq+slrUc6Xulwx/alc5N0Y1L2W1W8gwDTNJdG2nhIt49s7V82GgAcXcot2n0e2XrWIejUtkxXftQ5+ak8rssf6TusiRNpl4MaLQIVKz9JRozIFz11F/go2mURUgcaSAqCrUEXVa/Qk25gsrAkKWN+EZaRDodog728AGsDvoBbkBPOyfKExbfwTVCmbApZv3nXoZvAH1j+sOY3isapOr8XTemJezPs16TqRUrCO5U+1Ykni/XO60+PNdfXYF2W98vjRIflWZ6OQsWc8Zhp78XL2OJ2aM1THn8cWyfTBi6PDk2Fb6UrmhnSWcyqQ1o9sDlSPWmbSxFh8SdXBWZJjNpFfL4SUdud+oU1ftnjq6ifhk3i7RpScEG5zNzehzVezj3w+qo9+8Xdh18ipcFegwisXuOiMCi6POolVoTtCVvdVSauhiwZphRklOjqXnOZXjSkpG+rfRc0PlF989rBFSxvKLm7nJsrI+VQJ+c/96yJzaJhDUa9YOEZtPbD9ftysfp0sZE02UiuNbl0GNsbawONWDbGUOgVSXDWDj3WQD2IsnPdAP3oA/d+HX2Gti8hDRNZqrMW2uboP0QlP7t2vkw4DnfUuib3YdZmPnmk1ehx0W6bhG/huNqesOIyx/BvzepYrulKSYAmW6kxcpPkqf2TviSB+r5PX7RvlWQfDUlduo+ukKAQOFZ5Bc9Mj/9sDf5FRVr+2ynjVRMLOxOpkzqi1VPctZLKl6/U34UBiLs2JnDDBITo08ehl0DqmQ97DzzzYlL0e8ts7GdPdzlT2DRGJ6ztpuSL1lZizVyFf5kCU38Mtls+PLtuXMdGWvc+veAxQfvFj8W2VJMOqk75wdmBVxAsQi80OHCrrVZHgSfUieXfLwaotKr0gvKc+RIgoAM7kMVhaohrXtGNwQaZLbKPwk57DeOqQzhkR1JrQuIyS7qdBLTMkc8ctKZGWt8VE30O0QdzBkA9gd9Jy0DfSD13v2DsyJPCGuzB48euSTrPmeM7dj3raTYifaV8kjRAyehSJVtZxlT8tkma4186OfyWQZqTxFRP7TDSSG2/9vE9buvwAzMYSewvO937aKRCfJI6saYIfHNLX/oHpqPCv9HLlwE7U/XyWaLO9VS7zMkffPSKTESt+erKuq+DmiN6MxJZ2XkWKc5E6lus5indVwBSNeX+06VWER+m5qy4aDF/DaD5vEn4n5gejCqMSnOqF2TpsOXUSbKRvFn3cObYj3ft3q1vdEXaPKSECG9bytJ13iPVbPqhkP56j/ovHD2sMOw3mG/xONH9cfFusmWktidJAGrsRo1pbj6Dt3JzKkSIxIRWHtrelbsHzvOThis7HyHaCXz3ZTbedi74jGSJrIpixK8sYzNh6F3nNNdcZ80bosygy3sUjMfScMoXmMJbutzOtZrRvodog7+8IGsDvoOWkb6AdPxmTR9SYZcn9tPyWyub9+Le71lBchRrupG7H+wEUxhD+oSJldq1n+TEmib5SYojee+mKiF8tHbeS1aHxipo17XLv/PNr/b7NYgq8ltLXy0NuOXxFqUr6OSdXuL6lwEYUWFZJ5JblXo6InwKDWlZnz9DcKY/q1s/5tAX2uxn9u+KQusqfVV3ZT+5eiCUbJqSv3nkOn6Ta+VjLmq39mu7b+78PqKJHdN167JVFn0GVGpHieHRjVFF1/iRRUX64ad/KWQ2tUSm9m/WJZMPUNfa+s0b6qRrojkRPZXnp3ia1nUfeaut12/2ObeH7Lone27Mm1GVNgRe8nybX0ovj7Zs8ljBq9GEsjXe+sqrdLE9uWQ9FBNh7h+KblNPv8j896gW6HuIMlG8DuoOekbaAfvG4zIrEo6oyIY6O3dHo7V/XuvQitvWtKdqAkFSpkiFO84bNQzCoovf1zhOUfYD3pVS0mkkYpPl9Y2nwfjk2HL9lDVdQf8l1DGyJVUn21svjYT1LBenGyLXN884B6oDjEN6fbruQ9JSXsiXWoHsU9wxsjWWKbd0yvSI+7UXyvGj7kjE9267HLdmopZ4a3nIuzOHOZeJU4YQJEDqqPUkNtXjtnyX2ewNGoD+kVT5s8EbYPbggz3yVH85Hc11pO7sHzd+Pn8KOoWTgTfn6zkqUl/bPjlJhXkpAEwkOq9dRqO5sRfgSD5kchW5qkCO9XT3cseSMkP9QLc5G5BRT/TXHysjhTtLS0OAALdp3Gu79uFeuLGfkkNEqeV73E1I/+2GanLZzYpqxQkqMyrWNF1CmqL/5hdV7Pav1At0Pc2Rc2gN1Bz0nbQD94UhqVMplJavWbVQdRPndazDNIvPIG1DIZhPpuVCILvm9vzZvijTmZ6VN6gOoVzYz/OaDZciVWUBrNJOX62Sv6Ig7qlWF80Xy9NHmdkBeWogtqElbkwPrIkDKJGei8Uify6CW0/NZGA0VxhxQLKq/CfW2cqwtesfes3TAnOWlH0t9SabB+scyY+kbFp3CTXjP6wJmUrZoAFTWsEVLEJkA52gwp9qAqoKn1/9p2Et1nbgfJIBPmBQY8LX/rlc120OnUtYfixH5r49atzkd62fNrKAlH/BsN4sV2FKdvNJacY+70ybGmbx2nUyJOa7rNSJ44IaKH6zNsyDhe2ZkexZ4MrdI6OWQCnafCheZtPYGes3aI3xTycMvy+eIYTF55QDdJT1KktSyfUyTBmeHKdgpcgFQIdDvEnW1iA9gd9ILcAJbhB52q5QWpe41dFINCmVNiac9aXkT1SdcqZRH9Nb69z+4sUr48vFrpaVJ3tV8ZztCkZFZ8+/rT1ER6c3B29U1tZFxg5XzpMTOeaL4kdVO/JkXRtVYB4WV94Sv/4OvccOACXouNOyTPKsW8+svc1D2WtH9mQjOIw5ao3Iz2WMZuUv8FM6fEMgff25Ux59ApVl7WrEdc0m8ZJac+8Y4mQcTA+pBS1I4U7tz5zplpO2FJDCatOADiNp//fnVIT62rz5axi/bqOgYkJWCFPOkwx4mSm3becl+JfnJ2N8cqcNR29b7zIBU1KhTbmzgkwVNQyHht+YGeqqZ8GdfetMmQBcqJ2+8BNTgKp6CwCi0n9MRl+wUzi97LmjzL8nkqz9LEtmXRrGwOM1sfsHXYADbeWjaAvXjsA/3gtfpugz1DOke6ZCJJwdE1m6ehPnrxJmqNsyUuUYlPY87dtTSduBbRp685pVeSXpc6RTJhmkFCm3YuZlSm5LVl6Zxp8Pf71d1djqn22qSo/Wevo8EXT/N9murMw5W0Bh6R8cukODIMyUD0h0JJp/RSRLG/FIrgqExavh8UalM8W2os+KjGU1Wbf7Me245dEX/PmS4Z1n38dOa/bCTDFSiWdf+opqagkNK1HcLyYLhGupY6+GnDEQz5O8o+tv18NCuB9mHWRSdMTcpJJWnwSqo+aWy6ygctb1q0318ZAuXK909e95sNX1LjxkmCnJwV2lJ00ELcuf/I/mc9ekSjF2tPv8hKhUith/ubVQeEk0Uvllmb1Fth5FJcuHEPY1uWRuuKuTxxNJ7ZPgLdDnFnY9gAdgc9J20D/eDJK+0P6xUSEqYf/bFdcEMSR2R8FFW9iMbTxqbFxxxcHUPKrY5uXgqvVc5t2I0rV6V26rS6BdGzYRHdvlXp0OUGanGurs2oXZlhS0ASvZK7VH2BcUbp5em5aPtbHHUGXWdEIiTBczgwuqlQ6CN+UyrEUFDGBO+tt+dI/ct4TiOqKnUOzqSmpRgKtSGjiIwjo+IKZ7OzREvtGWz0xRrEnL0OK5zXNN+5kSfww9pDQh3MXcoraVy+UDobJr9WXrxA0IuE9Ahb3eP3f9uKf3eehpbuTJWRN0pMMxrr1SkbEX7oIujmbciLxup9sv3hCzdRJ5bhRO9lTqVVI+/wvQePdBl1iKVjx/Er0Ia0nLt2x047Zva7QmEOJKJEnmbtd4vEcshDrr1N/GHNIYxasAfacBJap/alv9qnK4Ry3QiLL1OUl9Bnzk6QgIm2UJz8d6+HIiTh0x50q+ciPusHuh3iDpZsALuDXpAbwKpIA72Vv/VT/CYN/bvzFN7/bZt9FxxlOXtxmy13fT9WPenxY+B/b1RAvWJPy9TKTmXiiZXYajPiGdKQyp4mKTYYJMZYXpiTBtq4vNNXbyNsjI1715eZ/zS+Nk5SqwhWtWBGT8PhUn8yE59eOJ29uEhvsUzo0g6oxs9THO6uocYvrjO3HMPHc4kWLjEiBjr2PMtxVJYYveRU6aGWYiOSm5oU10h5zWyRdG9W2+n133HaZqyKOQ+p0Ca9jkWypMLiHvoMCo7mKT2TWhYXM3LlRv3KFxcZSuQMp0s376H8iKWi2tx3qgolRrWo0sMUXrD75DXdfAqpSqll9SCZ50IDFgg1uCntQ9GwRFZnUxKUd2RM69EwfrlsH75cth+lcqTBPx88uZ2avv4whv4TjVzpk2Ft37i3FfI2Q4ohOWMgMZqgzM0w+lwPP6eL9XEFNoCNN4ANYC8ezkA/eORVIO8CPRBL50xr15In/szU8ZDRT0kk5CGVJb+GnseLW+tW16rh54yoXcoaG11j603E6IdKrUtes16zn040cWthDhqT0V9owEJR46c3K4l47Ys37iJ0pM3L+ue7VVEud9wfZm/NRa9faSymS54I2wY3xOPHj1Ggv7Uf9fiYr4yD1BoHemMbCRdQXfLykYStLOT5o/hQo/Jz+BEMnh9lKJKg104KMBglp8r4WMlAoTU+zeIpBT+sUAUa9S0NKcmdLJUmzSacafu135JpbmOceeeN5kfSw0Tx9eDRY3zbrjyalMrmFCZqUzD2u6fHiqCK8hCjz+zIE7q3aeWGL8HlW/eF17Z1hbhhBZVHL8PZa3dhpDqonWTYmOU4ffUO9HIbPl24F9+tPghtjPOvm45iwJ+7kTV1UmzsH5fN4vlJaxF16hp61C+Mj+oXghkngB5w8qWN1PDaVX7CXT3yv2hcuXUfw14qgTeq+iY8x+lGG1QIdDvEVVyoHRvA7qDnpG2gHzx5zTS8WQlUzJseTSauFYhInXgvQiu6lvKXcpz49Ga6szY1Jo8otzKnskl96hUrHj/ZXv64kCiIngAB1YtvFTYKfaAQCCqzu4WJ80LXjP5AfUVzkpnsWVInwab+tlAAmUjzZZuyeLmcfyTS2Kmg8qbHrG5hDo+hNrFPpUxTpbhlJ45YJVzxWDpLIJPhPVULZMBvb1dxmXJMxg57gtO67vhVOHT+JqR31Sghy+z3X3prtV5Tacyp581Mn6q31sqtiaOzrMa/0zxJAVH7LKUXQjKiydv7fftQNNJ4eY0Mfb01PSKP8UBbX3o3W6qq2y+dK9u7mBVxHH3n7NS9hdCqHkoP84cOwsD05tbl5wgsiT4LLYOOjH92JCdtZv98USfQ7RB3MGUD2B30gtwAlnGsn7UshWoFM9qJ7Bd1r4GiWZ1LpboLvZa83UxikLtjeqK9SrZPCUUJiXXfoLjieZOemnGvlEYrjadGDqNmhpOsaSIvx7WduXoHVcbY5EnlD/fdBw9RZKCNsH7GW5VQo1AmT8DrUh96Hrkqo5fjzLU7pr1aLg1ssdGQ+bvxk0n+2Djcxv3rIXPqJy9ae05fs7+wyimoqlvaablyEyGZRqrkT48/ujxtrGup+FwVVCg9dDGu3XngER5wmTxFz7Q2FXPjz20n0GPmDqROGoKdDkJEjLax7PAlwnNI8clkPMkijTmrzyxVJc3KTZt0Vuh5MCXtGK3xs5al8c6vW0UsPN0ISJo99WX1jy5VUCV/hjhL7vzTFqEE6oh6UTZQ4+v1YtnlOdDS9znaC/miMeiF4nirej57THDXWvnRr0kx098yo5AVScHmaiiM6Ql4oSIbwMagsgHshQMnuwz0gydjCIm2qG6RLHb5Senh8yK0ouvXftiIDQcvCo8AZfymSJwQUQY8l96ei5X+STCEGDPMeH+exF7aqKLMlOKDF+HWvYf46tVyeLFMdt0mW45cQqvvYnlvhzQUnJveLHpSuuRVytfPRlj/Y8cKqFvUOBbam3OjvmVClkoHpqVt8/YczPQvwwpIftxI5U/2o5V3LpDpCZOFakjJ+tsHN0Da5PrKcjIWna6G/zTJ8z1+SQy+WnHAMDm116wdmLv1hP0aXN7omKX3onmrnsmGxbNgSgfXecCpLwrTofCC714vj8Yls7l1U0KezoKxsbE/dKiABsWfnG9XkgppvdIjLUN1zJwZqiNZZ2SIgNpO5T4e37osKBabisrNfeLyLYcOjv5/7hK3KGbYalTWCBJBIRVIVcxDykQ/Xyobvm73RFVU5nzoUQBqb71clXpv/V04Nh+5BK30vAwnIl9F1DDHAjRm9yS+6gW6HeIOjmwAu4Oek7aBfvCKDFyIuw8eCenjxiWziphJKvGlviMNFLpG23rsivCkEkepvxdpGJihQJI/lKmShGCXCXYNs7GrUaeu4vlJ8cfBq/7obVa8kZQ8c//hE4PDV3unRxlmhk0jvucrWQWM1N3U+Vy4cRd0S0OFlLuIJUUWqSam1t/Yrx6yptEPx3GFjcSZBK92Lc7q62F95/5Du+ytpC5zdU9u3n2AEkMWi+a/vV0ZVQtkhCv8x3L8a3fuo3Ssut2srmEgKjVZpNqZs9hr7VokfzAxJxDjgtkimSPerJYPg18sHqeZ7JOeo5SsKCWpF3xYA8Wz227y1OdFeL+6yJYmrhS2jE03k6ugSmBT39sGNUA6RdL7vV+34r9dp9GifA5MaF3WPlfJ1KJHxRc6Yiku3nxCeyY90s541rX4vfjVOuw6+USsR35+6sptVP3UlrA7950whOZ5spdm98BX9QLdDnEHVzaA3UHPSdtAPnhkaJHcJDEZTO1QAfWLZ0GJwYtw895DxBf5uIxrI4ohqWN/YFQTv6epkV68+sWyYOobjj1WVvlX1eQmklglqVW9olIjLe9VC6p30BtfCSPZ4/g+M0Zrk1ecKpWevGHQMxq8gZGZPq38sDsKMZEcvOqYq/vURp4MKXSnIfmoaxfJhOkm+ailZ9EoObXzTxFYtucs2lbMhU9blrbzApuheJOTVBMpK5mIi3aEsWrkyDAdVxTw5BhqvO7i7jVRJGsq+/DLos+i888RIPEIir12JmcsG0rjkG526IbHbJGy9VIpTW0nmQ/oefR1u3L2sKRpnSqiThGbjLAqwR09vBGSJw6JM7SR5LPe/GRd+Zk2ZE4raiHrqSqIh8fExUwb42xkRDvDSxtLLOvT7x29TJKRPfTF4uhYLZ+zrvzm80C2Q9wFmQ1gdxF00D6QD56eoWU1E9gd6FXuSkp0IPUmKsRBTFzE/lysZLtb9UCZTSxTk6CcMVF4AkujdZiJV/bE+M760FNNMyMp7axfT38u1RfNGuXE9EDf1W/alUdThTFACqHQVTpl9lPRGmnq3KUioZnQC9nOGdWe9ppaMpNYibc9dvEWao5bKYY0c6PiaD/UuOh1H9dBznTJ4Y5ctyOBCFdj8KWH8r06BdCnkXmquI/n7MTMiOPQe+mW51wmEcrvpIyDJsxk0ix5Xyk2WGuwq8aps5wC6S2WeyFZYeS/X5+6CesOXHiK53jt/vNo/z99RTt5GynPec+Z2zFv20mYFQuRY9cYuwLHL93WNXLlc1vrmfb0d9zT/QWyHeIuVmwAu4tgkBrAN+4+QMnY60KZFFF/wmocOHdDcHgSJ6c3ixrfSMleRF5ORY1b8+b47vRNbBn0Y6sXj6ftV/VA6XletPXNCjiYNZTdWafaVkr4JgmhmL8ndFvypWlU85JxaIc8Na7ZfmRymXqNLpMsrf6Imh3TlXqS81bynTrrQ3s1LOtTDDrFotOVNSkSUvn7/WqCzlCvSBncZmWzY2Jbc55HmeiVIUViROqo1tnjLWMTldTkUAplMuMVjT51DU0n2dhn3JVh33joItpO2Sj62jW0IVIlTYS9Z66h8ZeusduocdYkr62ycFj9Xss9kWIyVhXOZHy1npf8lW83IOLoZfHMpme3pBDr1aAwPqhXSAztzMOrGvt6IRLqmZLxwvJvWlo1OZ93ahfAx42fGPnq/kQNa4QUsY4O9TZSxlr3m7cTv2/WN/gdfWdkYvenLUqhbaW4AkUydK1wlpRY0qOWs6+e33zOBrDxVrAB7MVjGsgHTyVXlxyukkPTqnfClS1Qr9Qpvo6UgKjEFwWbK3OWbaRRoveQ1fa79dhltPjGlpSijZXTm4OzZBXZhiiI7DHbylWnO+ty1HZ2xHHxkqLNepcelyEvFkcnH14r6v1gSpYCK9f+3sJP9muV39SIE1peEVOmPWXvU9HGqaprccUbLsVFjJJT7THW9QqhZ4PCcCXcQE3m1BNIsLIfMsaUcgkolIoMcPVF26oktj3OVyfRK/LoJbT81vbMcpR8qM7/6q379kTj39+ugrACcZkYHK1VKs/psRhI0Yj+TYuiS80CkJRfr1fJjZEvlxLdyiRRPRU2+lx98dbGm2vnJUNf5N+1yn/Sy929fiF0r1/Y3jzy6GV7gt6OwQ2RJrktcVePY3zo31GYvuEIrMaFO6KLU1/Q6KZRGwZi5azFZ91AtkPcxZENYHcRdNA+kA+enoqXvNLsEJYHw5uV9CKyT5S7kiZKAErWqDt+tRgvPuJZ3VmY+rA2kyyoericeVZoXuSBJ088lVW9ayNvRv2YTvpcTWIkL6c3i4w51cZ3+gvTgl7muSOvmTexctS3NGgl3ZOzeRgl9cikKOKJJk8wFUcx40b0UI7G18pLa+tK6eO+jYvg3doF4ShkwGgcNbTGmZyzM6xmbTmOvnPjvqSp9H1WQ4UkY4PevFTxCWdc4HLeKq2d1Rf9XzYexUAD5hnpVZZUbTLchVgryKNKRYqWGLGAqNy+ejzBKvaSM1j+Tft7IeNwtTeJKmYk2024UlGTF+WLgf27my+9eLEzW54k5YaKxG61qL95c7qFoULeZyMRLpDtELP7alSPDWB3EXTQzQXm+wAAIABJREFUPpAP3tGLN1Fr3Cqx+mU9a6Jg5lSQXiUzGeruwi4TbPJkSA566MkMXTVz2d0xvNFeL9HG0Th69GGO6lsxIuIz/vbbVQdB2ebaa2oZDqJet3oDd2d9vvfbVlCYhnp29ZghnPXj7c9lyMiYFqVAGe7OikzkI25UMpplkcZn74aFMX7pPpHMqqXqUvumG5bNhy+hS8386N/UHK+qGueql5yq5W5VnylLe9REoSxPksaM1qnKoZtlSjHq64c1hzBqwR7ky5gCK3vXFtVUr6vV7H9SMyNVM5VaT46thlbIeGNneynDiIg6bM+Ixg75w7V9SdYPchjsHfEkBElPJU5S3pXJmQbz37dJEcvbEEc0Z5I3e0SzEmgfZqyWJs8wJQDSudMqBRrdCqkx2hs+qYvsaW1MFJdv3kM5jdTzhCUxIi/ECluGisX0ThVROzYBUGJJoRYVRy0HsasMfqE43qz+bCTCBbId4uw74+xzNoCdIeTG58/CwaMrc5Kw1OrDO1v2/rPX0eCLNaLa2r51kCt9cqjZxM7YDZz17+xzmbREMW3Ehyq17ue9WxXlfSip62zealKN6sUwanfyym0QiT0VMwIj6jWhs5AJVcmvg4MfLGdrMvO50Q+Sv1CNySt+yUhAa5JS2/SStbpPHTPL9Hodyb1tVp1OKltpVdIqjlomrq1HNy8l5MRv33fMG+3KPqkxm3rJqdJgkvHfaliV2e+x5Mkm4PXosaxsiGQCUY0mlWbtl7cqo3qhjKa7lPRiUupZbWj1xZbaypdIVyTf1+w7L0IbqBDvbpKQhOK/9UIXpPhOtjRJEd7PJjksKeuIcedLgxhweUY+qFsQvRoW0cWJQq/Iy/rosS1me/+5G0/xRFcatQznrt+F9iVPvd1a06cOcmdILsbQS+iVIR9Fs6bCou41Te2ZXl6LtmGnaZuxMuY8WpTLgQltnlC0mRrAR5WeBTvER9CwFLI3gff3g0dvtMT5SEaWVc+p6mnc1L8esqROCqn8VDlfesy0cO3kyh589Mc2zN9+Ci+UzibUiyR/p9XYOFfGdqeNGme4X1FaMurTEZerXhtH8rfa+kaUP+6sz6itEY9sq+82YMuRy7Cq2OTpOb7x42aQx1K9jn0iQpIYEQMbeHpIl/pzdEWr16FWbILqxBV8CAXFPxMThFatTO3PauwxtVUlv/WSU6VK2vhWZdAyNKdgqyDWCipaZgAjsORLivzcHRpEmRhI1IEUDiKx0tI9mt04R8p2KkXakh41UdiEt1vGqdcqnEngY6XsOH4FJA9MRQ252Hf2OhpqHBlS9IFioeUzykwIjHyJbBWaE+NaldGd3rnrd1BplE0RkoxIYmrQhkVJZT8SWGpe7ol6Xtxbx1rCs05FD0tnFHx6k1OftcSxTC9C2jJh6T7QzZCeV9/KfsRnXX+3Q+ITC+1Y7AH2Ivr+fvBU74Yj1TA9iPQ8jVbI0N2Fve2UcGw8dAlEBzXg+WLxmtDlztxlMphZJSfVKzGzSxVU1kiQaueikswTv6iUMtWbM/0g0g8j0cj1NPDYuLNWta2RvKmk9epULS+GvFjCU8NZ7keep87V82FgbKiAvF7XXhtb7txDDdT4cUfxuupwMhmoesGM+KVzZfGReq1P8ZEf/r7NqeSzUTKdo6WpV9Z6MavFBi0SnufJr5XDC6VtioUyLl39m6MxpESzrOMODSLhQIl7Wo5dvXma2VJHIWGu0BDK74orORZ6IWu0BjXxUGKnJt7KWyoZt+voWSHPWlj+DPi9SxVdiFTHCYURDP83WnjuY0Y8kV3WUprJjtTwsYUf1UCxbDaRDr28B0nBZ4VT2hFvs5zD0uizIEOfwjd2D33CRGHmPPiqjr/bIb7ChcZlA9iL6Pv7wVNjp+gq9LXKzmMKJVx6xOg/rjssHmi50yfHmr7evTKWmcv9mhRF11oFxLWaTVHs6eQFL26x5a6nrz+Mof9EI2e6ZFj3cV2n7dW4NDNeMcnXSXGC+0Y9ifXTG0gmQmnjQ51OyoUK0mOvNS7klWK7yrkxqrkt49wXRdKLqQwm6suEO55FT61Hpa4zKzcuqZvUa31VBIWYDUhc48jFWyJGmM6CXtFKzZpZkzrOil61kF+RYtYT0qE+JQ2V2RjnMQv34PvVh+zTIbnwjCltyVFWi7wFUNkPqA9XY+Uln23Hqnkx9KW4L3fqs9dsuIeMjR34fDF0rpHf0vKu3LqHssOXijZqLLM9rpjoCUfYJInVsCuZ+CfjtR2NLZ//jozO5XvO4q2fbAbkH29XQZtY2jl5Q+BIHl0N11Ap+/Rig//P3lVAW1Vt7SktSKdIeenulG6wUTEQBUVEVJ5g0kjIe6ioiAHqD4pYiC3d3d1d0o006D/mPneeO8+6K/cJrnjXGG88uWftVXuffeaa6wsiNOa4yf70ZtvhM9B4aADWpzKF4RsXnWqK082JcuWkHodEefra5pMD4CiuflJ/8LhkFgWStssxa/NhaDdqqVedggPX7KZtX7J6qEGM2VHCQtK/Y+VC53fsfrBpKFeG2LmRbStD09KhzGRxHD+t/ANe+HYV2BCCVI5Lfuemu06GscX6T49ZBpPXHwLdsWk0xiO2ecf7c2HdH6HazEj6Inm91X2bQuYbA7JL16ocPn0Bqr0ROD62VSQYOWc7vDFhE3D5Ki7BtaJ3E8CN0OZDZ0CUo+Lz9BMEcta8CLFSudTRxrZny5LwVF1zkEcOdTRW4iP4uUd0IiJqLNcaPB32n7oAA+4pA6iaYVtUChx4vQ3elPeD2f8SvSd57wGTyoJsfFz28P/aVYGGJXJ71VABBKEfeTKlg0U9AnhfDkUhpRqCq6Dm+gNV8kuXgLvbYUYXbZ7FQsoYuElB+cw6QwImJvR88FPJsR2qw21FEjDXoYTEWkHeCod3UMY6aCGfLhWs7dfM6pap7NrFiwk/b6vEYtV5FCsl9TgkilM3Np0cABuXyH+FpP7gcSKbrbA+rQZpIiJODEXrsZBtbyrEjsXraPpfPfWV/Mfjq6eqQ63COaDKwKlw9M9LwN2LotF3uG3qiDGqtskueNjDFeGu8oGjYlVJEKxPC5gN0xU/5gZ+56+COsRyDLqxy/DQ6/efgtuHzfMus5Gg87s2ttepMJC662VSXHSMm+IG3Ly2BNTvXr3vlBYKQzAAF6iUjtR2+sJlKNdvijd0ns22OWrn8+367Sr4ceUfwT+RIo3tmvJ6Dd6aBZi1FoNvCspdM691h8yEPcfPSRUDeJA55slqUKeo3LKcxsfvvQ0ZVjZ/IlAObV0eWlUKYGtVsDWuVd66Sn4oEk9c0ymF8N8TVNFANQ2xvDttC7w7batnwPLjs7WCtsukuqBT3eByZ2S+hO1zLWjaqAZPwlKl8JzrbIqsHdl1lDjQEQJt+otVnaQeh8RqHWT9JAfAUVz9pP7g8Z2z7JhOtzQkq3Nj6pSeJA8W7ny0aUBzSJc6wDSOdNlx5M+g7i+J08dS0SCc+fT5eR18sXA3cKKNqT1UuMBgQnRMkl3nArEge1Su92kai9/PySRF3GjJSFp++wjnOjri5fJGKtxkOP2Ecy2XzuIyULo2CcfMv6dE7iOHtqAjW9046C6ROMNjaTyFQOa+zSkEjYfblYvkVE6G4sfZtFGyfR/RyQL1aZsZl61Z8Ht2XzloXTUhy+lXqo/0dYnkx/vkR/2fPV4FGpUMZGRVhdsAcxc0l+ep9v9mwL4T54GbzqgMI4j0iG6V7WsXCm5WdMf+5y9dhZJ9JnlDksmI4d+JC0ByanSyQM52Omy0atMg+93h1sw7B9u5Cuqslvk6k4144ZwZYPqLAbm8pFySehxyLdcuOQCO4uon9QeP43hd/c3HL98HL45b7R0L464bix9xdz/Lz4kbZFna8O1ZsOPIWSA3Iz/txuIaMlxoWTYPfNimslWXLkew5Nhk83J+/df1MGr+LritSHYY20FOWrEaoEUl0p0Vj9lV5DiLJiNahSS5Bt5TBh6NP+Z2VeCI6IAkjbm6AmITHKqEpzKpU6aAD2dtgyGTNgeZ7CaGPw88bMl32LfObVClgtBpzHKYtP4g3F85n6dKYSqkc0z1bLHRYrsYkBbpOdEbs8gjkOHDTeNCY4jCPSd4OrefPlYFGpdKHOAm8BYqQfMyeiMaMrJwwbSKY7x92FxYv/80cIc1OoHBkyU8YaJCeGjkhTxTr3AQqjD5hbpQPI9an5kkzFRawJQ9JblB0XZ5z7FzUPfNACxCzOar8MEy4q+fZIytFTfHMSO84qZ4S2bTM3GtPk/qcci1WhfsNzkAjuLqJ/UHj77IuASuWUA6Ws2VMS0s6Rk4aueZ2Wg6shG+K32alIDZECRu+M3SRPH2S5smzKvtDzw24nIES0eapfNmgt+71NFOjwTvVe5OkVwblbh935/XwecLdztblkZybNiWLMuuwyNGun+b9rjEne0Ji0ytheQKq8W7ZKnw2TQmTr6zUSLhc1GRUznhiGu6vjxuNYxbvi+ROYJqfUh7lj63gRPI2uKwKjFbTUE2Ks70uTPBTER3z/hRviooJ2iTDW+BnM3C+a7SPLjiiirj/sr3q+G7ZfugccncXsB8x/sBKNCi7o0gT+Z0yqmTrOFTdVCdJ/FaES66S7z9NW2+EFuNGGuZLBvvrEiPCXBFwEGTbBtC77bFw/GW7ToO938csJq2xe+TdTf+rmzoHzjVlBWOxXf9Pth8zyNdJ6nHIZGer0t7yQGwy2o51k3qDx594XFaOuka2bRlR+0ufvCOSxlSXebYREfsnMUfTh/RupZe+LZHvDgOFw1WsiytVCAL/ND5Nu00/BDy/K4LYbTpqJPaiaV2tG7sMhIlzwr6IR75XSvVdTzzhBJ3uPEzFZlhDZ1CNC+dxzORMZkc8O+1Sh9VNQ4VOVWmI45t0KlErcLZ4aunzKcSZKVN/eswqrq14soHImHvidFLYcamw55KDqrl2BSeyVTp/BKxTKe/TH098+VymLjuIISDO+08djlMWHsQ+GkfJQ66NSkGGJRSIfWQcvkyw6vNS0CbTxd7H23s3xxuTKOGttGz1bRUbhgZb6PM14syxHTSQrbLVJ/bPcssomWSdLLA1dSO7B5+t2wvvPL9GiBokO4+k5udKy7c5tmJdJ2kHodEer4u7SUHwC6r5Vg3qT943EWpzC2Z4Lfn9RlDPn0Zuzwc1ySXpR342wb4dN5OoAwWXhtLSS+XsYp1/QTq93ww3zMV0DksUT/9f90A/zd/J9gEEKPm74TXf90A+bPdCHNfMUuyhTNvFYkqlllo3fhVx9Ekys+JQ+GsQzjXEu7elKHifXAlht+71IbSeTMDSc9RQEdBSIsyeeCjRxPDcrhajOkIXJyfauPDlShW92kKmdMHFDbIaKDsLZnh1+cDNry6QvJsVMeGKCprjxMeRXw16fm6wMR4AEZGQWK/Kscz2fhavjcXNhw4HZZmNxlpNC6ZCz59vKrXDQVyHPqDfycd3dyZ0nr63J3HrgCUVkQXOd3Gi0huMgc2lHREoxPEktNGJWi7nD8L4OaKE9HW9GsKmdKFKq/ITDIIjpclfWpY1ScAxzNlkmVrTA54NtrBKB04beNhuLtCXnhP4YxnenZj9XlSj0NitQ6yfpID4CiuflJ/8Ei3EZfA1e51+Iyt8NaULZ4YOYqSUynWcyJcuvoXfNSmErQoq8e1+V16mWB9u1FLYNbmIyBqePrtI1rXNX1nNmw59Ce80rw4dK5fxKqbB0cshMU7j4PqWJE30vPHtTB28R5oWCIX/F+7wI+cqsTK6QzxkOimhYVLMOG/XSAbVovlo5JufEH927tLQ9so20Wbhu6SoaK2ZHJbomUtPTNETBLHITMaMI2VPleRU1UYTXLwQgUBVBIwFVI2oHo2RFFZm5wPIZLM/OD25209Co9+FsiaquAqtDav31UaHq9VSDlVPIlAxYwzF6+ATobMtFakmVylYFb4/planiNg8V6TpO9rcqxEpRCEJvT8cR3kzJgWUGZMV35YsQ+6fbcaMqRJCWiswYNlTnAj4qNou8zXjVs2U5/BDdX95QDVKbDIlE78KKbIkjqqudJ7i8sLmtb/Wn2e1OOQa7Uu2G9yABzF1U/qDx4dgeMS2Bz78KWSCezj5yomdSSXmQJC7trlSp6J5Hhc2vKjViGz6VX16aKqIFPycJmLbV0uXyTiKz+atR1QGq5orptgard6tk1GtJ4O69t46GzPaeq1FiWgU73CEe3XtTEKFmxNVLB9ruBAWTdSAyDFC5VNNY2PwxVMGFBxTipyqoqlnyDjZzYw4HOjflXkK9Na6yQce/y4Fr6y3FRSP2QwkRYNJhQyXLbYfm6aEY75ApEf6bvGsd1cVgznwG2s0RwFLadt7H95Zl80JZFlxfm6o1wZQk06xDutyWA+RFYddG8ZaFM9oMn8+YJd0PeX9SGWygdPXYAagwOa2XTyYXoG0OIYTyBs+BP8+SUitqn9a/V5Uo9DrtW6JAfAUV75pP7g0fETLoONcxhfLiJlVCuUDb7rVDP4EclJRVMkXKbXqXIai/Itdm5eJ42kaqzjF8tgyoZD0LpKPhhyv54Z/+xXKwB/fO+teAu882AF7fg4CdJkm+w8UXaBysEJq+APKwZgricQ4YxHvPbU+cuA9wWLSFgicwQb+EkkxyRrixQ+XDcL4rGxSL6i94AKNy4j0tnOlY7uX2xSDJ5nGNMJaw8EjtUFnVbuTGbSb+WyWzQevyowojQcnx9tEGxgRXTd2MW7vawpJwmLa6ZSRhHr8WDUdQPC2xLHxLOkU7vWhaK5E9QdOHSmTtEcMHfrUc94YvwztbS3nsvbiQ533Chj68AWkCplCuDKJhgwL9pxDJ77aiVw2T7eIZFp+91ZCtrdFnAtJE5IXI4MMCP+1MCP056LRjufp7h5sP1uxKpeUo9DYrUOsn6SM8BRXP2k/uCRBiQtgS2zHOur9CPJUYtL7UR6iUv1mQTnLl0Fzp621bR9a/Jm2H38HCAZS0fmUI0ZJdgwk/JKsxJQNl9mp6lhxqpoz4kei9nFspkgHzZ4sw6fL4NpGw8ByQzpBsjl5Pxqi9osgE5Pl5yo8mZOBwu6B5yoYl10AbrKwCPWY8T+SH8UiUm/PGfGx9IY+akDHhujoxgWkjSjzBeaE0xgcCa6nqtPmEhQ4rqoMO8/rtwHXb9dDZnSpYI1zKlrzpYj8Nj/LfGakR2B8/b5faO/i2Qu2/sUPP5mQRRd6wenLmZbZeOwfVcSyQs3C5v6N4cUiEvwUUgTmrLSK/ecgHs/XOC1JGZr0XkO8boo44YucQdPX7CCVeE7rlSfyXD+cuj7GfugAJxDKTj5EDO1mw6c8eQ1OZ6XT5VOFLhZiYzMq9OgVi2dqywkZaNtXQt93LKIXJLU45CITNJnI8kBsM+Fs7ksqT94FDTSXMhG0mZupN/aqEQu+IxhTR8auRAW7TgOeGyGWeBIF5VcEclp6Qwm+EvR1daU5kFwhIer5YfBrco5Tc8vSZAkiZqVzg0j2lbR9umiMiGzEHWakGXljQdOezJ1WOa/1tA7qqQSKxyybqj8R1h02XLJvlsuh+9qwVOXePky24a4iggSuWoOnuFdSqYRFPyptKNlOqu2fRM5lcOV8FqZjCL+nQdly3s1huw3pVV2tevoWaj/1izvcwyYTp67DM/UL+ypFrgWCnIr5M8CPz0bqp6i4jvo+hDxtrK6pC/cuX5heEUzZgrwbCAIujGJuGTEYT/5+TJAMRF0BERXT14Ib0t/szlVwrqU2RY3I0Rw5IRr0XZ5/6nzXuacWzPzMVHbnENB7fKNIRLuUNcZi8qUQ1wrV01y4p34+S1wfT7DqZ/U45Bw5hbutckBcLgrqLk+qT94JH9EU1DZV8qmqMKakqaozXG9n6XffuRPaPT2bO/SGS/Wg7icN3n//caEjTByzo4QZQixfX5sheS9CV1qW0lJ8XbqvzkTdh07B7eXuxk+eKSS0xSO/XkRKg+c5l0jHg/qGur90zrATGn94jlhdPtq2j5Nrl78Yq7FOvvl+lAwe2LrUqcJKipzXOCqPk0gS/o0wZqUCcyYLhWgqPy1KFy/mj9TOJZu366CH1b+AbeXvRk+aON2vyM9F3IRtHkOeN+kzYoY5jvK3RzUdCW1AxmGkl8ftJVNmQK2DLKzlaXrVeRUklEUFUj4MznrpfpQSGKnS21zbDIG79uPnAWuceuy/kQErFcsJ3z+ROh3jIh5/Ijd1DYpLogJAn6dLbmVNsA2xFbduEKkwXo0gllbjniyX1nTp4aV8eoJ/HqCrwTvZa1C0O+u0qapA51CiVrnqjUh3gja2J+9eBX6ayBRlDVHh7r/NA7Itv134ib4ePZ2IHIfDZA0g23dC8lW2/bdTrA7/E4Nd/wtMC5iBCsk9TgkglN1bio5AHZeMvsLkvqDR648NCNuSWqapUo71A9j2tQX/5yztZFlTC48Q6dshmEztkH5fJnhZ8XxMD+KxzZ/7FwLKhbIat09ukQV7xWAMPj5MVK5X5kGQFq5NeKywTcdE/DWsuvuGj4P1uw75Wl6YgZGV3jmE5U8cFMQjcItRsVjbRuyUDTGxNvkFsNihpo2H7LAKNrjEts3yZWpxkOyZ22qF4DmZfJA288CEAOCPJmy8OFsUlTkVAxYMHAR8cw6K1xxfkt2HofWIwJmB9VvzeYppTxcrQAMbmWn1cvbo/eZDGZEMJ2bM6eDhZYwHZnmrjh+29MaOlVz0Q6XPQtcmxgxv0g4GzxxE6iUDGjzQm3ZvFOwLkkxcplK/DvpKYsZUzqhwPdVqpQ3eC6FxXNnhMld6yaahsxSXSX9SFC59x+uCHeWz2v8upLW8n2V8sHbrc0uhESOVKmnGDuMUYWkHofEaBmk3SQHwFFc/aT+4NGLlZZgbIfqcFuRHFYrojoajraz108r/4AXvl3lyeysZ249NqYOG/afhpbDAkfxWB6onA/etLBbpfo8gMUf3G+f1gej4kLqoAC6RUfc8vCZ20B2PCte5yKzdvLcJajQf6rXBJJbkOQSjUKSSujUhHa8XBpp6oZDgKcGeAxra+4Q6TFyC28RBmRzlB3p8ajaI4Jjq4q3wFADwZG3QRhytLttVDIX/Oeb0O8Pfacypk0Fa19PnIVPUGZI62FFXcoL36yEn1bt9wIQDESokF6sqD/OYUpfPVUdahVWv484NAOtxdHkwfaYXpwDBaOP1SwI/e8uE/Lx98v3AW4+VJlS2XrYYMcpAWEK2gnDTaodLuvP64rudEhKGzFnB1QtlBXGdUpMbhMhcrb904mCCGMgK+b/NCoKXdnmnGBluEFDyAti0lU4d9zw4Mbn6Xpx0L1FSW96qk1qxf5T4MS5y56lNmajTYU2irZSmsFTR4EIbuon1p8n9Tgk1uvB+0sOgKO4+kn9wRNtRF2IWaqjTQrWyscLm0d6eVVkFTqmLJQ9Pcx6uYG0W34UjxXSpU4Bi3s0hsw3hoqtq8bMyUCuRCRsM8Sek4n/m9bIBYPoosJx8cpVTwcUi18LWdPY8XNdBnH2liOAP4BYMDhOnTKFTZMRrcPF90Xb1Penb4W3p24BmbB/RAdh0RgFTBgoDLJ0JMNmuc5vnaI5vSNmLqVGUlSpU+IGpWWikchcHy2G61VRkVOJcS8eW7u473EZPzy2xkBVZeZhGq+ojczrE3kM3xebBthBQCjY05GBKeOos0VHjGzx3gEy2qePVYHGpXKbpqL8HPWuC/ecEGwLN6ZoO61ybaNTNWrw7QfKw30WgSTfmHBidZWB0+Donxc9Nz00YaFC9tdou1w4VwYYMXsHiOpCVJc2FtyWWvWMySTTdItHCSERr666JilomNs8DEk9DrGZQ7TqJAfA0VpZAEjqDx5pnNISuIisq5zXSKpJRagJd7lJkkiEA9gcU3KGOf7YX776N5hE6Pl4KROGf/Mzv1mbD0O7UUu9JrcNCsgA2RSZzI/qOnJ24jqZqrouwYbNOFV1iP0tI7bESolCN36VKQNeQ2YxBbKlhzmvyDdW4ayNy7WPfLIIFmw/BrY/0NQ2YSQx01cjLju8L0CFZm4+DO3jn8vtbyQmQ4XznVaRU4lxX7tIDviyQ/WQZSCpQFPmjmsGI7Tjy0V7wO9xNGH7Zda2JN+Fg9w52M6CmrSW+95ZCtrHy3WJ95pn5tHBTlY4Pl1lqezyDJEkHgazKEU3fdNhUJG4vly0G3r9tC7YvG0Azo1TpnWr5+kHIymtaLyqxGePV4FGJRMC+WDSJF9mD5I2esEuQOm1MU+GPhc4EEq8tK1R0DPowKLC7lIywDZz7eK4if3aJF1c7k206ib1OCRa87ZpNzkAtlkln3WS+oNHR2s0PdsXBda/76MFgPqgIuvaRv/S53J6lxFWD49z+Y/GuGV74WUNoQOvnbTuIHT6cjlg8NuizM2A8kKINUPmv87ek8ZLWSv8tx/ZLr94V7IltbHorNB/iseGNwUPNCdy0nrnwfJwb0XzMaGfe6cL4LnGrEiQ89OXn2t0KgffLd0Lr4xf42wU42ccpmtcf6CpPQ4PqlQwq2fqwIl0HFe/oX8zSJ8mVchQwsl0qciphJ+UkcQoeORar7K1oQAENaSblMzt2aPb4ORlbdFxucxJTrdBUt0z+l7pLLRtMN180+wqQScbW90hM2HP8XOA73p8/6HGsEqFYsr6g9BxzPJgM6JGtmruqHZTss8kL9NMzo/cmOLX52qHSEjS+w03yA1K5ISvl+wFzAZ/+nhixRsiWXOZx6BVtQANIjgYqoLg75SpcLWUZxuYXTrDgQaZxhLJz5N6HBLJubq2lRwAu66YQ/2k/uDRS5+m5KKheef782DtH6dAPOIjzUqVkLnD8kmrksqBaAtMx6Hp06SEDQwbzBvh2qMjH6sCD41c5H08/pmaULlgNuPQCIOJFVU6lbpG/AZTLha4JXpPhAuX/4Lhj1SEO8qZiR8uGWMhJ2IKAAAgAElEQVTjAikqUAAl4j2xOmfyL+nRCHJlSue3G9/X6SAAfjctvgejudD1B5qaIgc53EAhdGfiuoOAcmhDWweMUrgZwcreTSBrhgSVDvx8yKRN8OGs7VZGCOLwVeRUIsvK1DVQMg/x8qJ5htg2PVeoX4zYZsxs2+DkxXYQGlCk5wT462+AEW0rQ7PSeUKq6FRMZLcLybKF462/xWwnr6+SkuR1KDjUGWq4PGv03kYc7k+r/oDdx86BLOuNbXKZRPy3SwaaLMRpE8PbEr/nxBFAGTZUVPh51X7v/2XKCkQu5EQ1lfa56jdKtV66UwDZNbFy0nS5v7K6ST0OCXd+4Vz/rwqAx40bB2PHjoXly5fD8ePHoXDhwvDMM8/A008/DSlSJBxHT5gwAXr27AkbN26EfPnyQbdu3aBz587O65zUHzwUOkeMGRUxqNRNWEW24sepLsf8toureklRtgKlLPEYV5bRpSM9ZHOjBBRCQFA6iQcDunGQwgLWcXXOw2v8Hqf/vOqPRMQl2TgR0hDXIwHjZ4MXJDvWaIq5q1wDcQ6bD56BZu/O8aYz95UGkD9bettHIWL1aH1RUQSVRXjxC1uJ2OBYQ3SkqztWl/XLN34lbs7kkYj4d339/lNw+7B53qULuzeEmzMn6DTj31Qse5s5qsipqqwdtkmb3I5146BHywDRSVY4Cale8Zzw5uTNvrDapy9chnL9Ak6A33asAdXjsod052oFzcml33eqCVUKyTfXNoThgb9t8DLbIlbaZu1ldR79dDHM23YUUFFi/PJ9cObiFVCd/vCsLba1uEcjyG25QSWyGmF1ieyK72fEmXPNYa79jDCdpbtOgEqJgaTH+AkgGqd48DaBwHj/RwtgmeSUUrV2rphhbocsgw75vUeRvi6pxyGRnq9Le/+qALhGjRpQsGBBuPfeeyF37twwc+ZMGDx4MLzwwgvw5ptvBn4AFi6EunXrwmOPPQaPPvoozJ8/H/r27QsjRoyADh06uKxtksYAc6FwDOYuXf3Lyj2MFkAViLpmS1wWlLsMIfwBX4JUOL4XLVTRNUkswaP4nBlgxov1gxgudEZagmS49HoyHGETqV1X0pZfQhVXUdj2RmKSEo3HD6mNyDpcV9PlntjU1UmJ7Tx6FtDaGsv0F+tB4XhdZ5t2I1WHMuzZMqSBFb2bhDTrl7gYqbHxdqoOmgbofvbfVmXhoWoJJCJTXzz4uDVHQC+XGwlwbW2ZFjjBFfxI/6lwkh0+XwrTNiL+NLFsma06ApehQvWagb9vBB0JVrVOXN1FNELBa/j6mLSJsb7O+ZCPwUbekNR2XJU/VHOljQeSBvF0A4vKKILjdrGei1NoArEtF3z6eFWg5EPuTGk94jEv+0+eh1r/DZiz4OeHTl8EFdFTBhtRkddkhDndd8XVpn7xjmPwYPwp4pp+TSFTOjsyten7GunPkwNg9Yr+qwLgI0eOQM6cOUNWA7O7H330EZw8eRLSpk0LLVq08LLDixcvDtbr2LEj/Pbbb15AyzPFpgc1KT94POuBjPB9J847if3XGjwd9p+64BERkJBAJZoZPT5m0X+dv4zW9msKGSUvI5JeKntLZvj1+dqAmZpqb0z3suAm/DOXEKK5iooBpudBJdhuus42C3nq/GXAlzgWW7yei3GGaZyqz3Xa0LHSItaNnX6cZSQ9v9J1ftdKd13ZvpO9jB23ALfpZ9GOY0G4DyoZIEQGTQcerBoIovedOAe1/zfT+29ZAKgyvbHpm5+6cA1dnokUzRVIOs1kMMDr1SycXesgphsrz/DKMuCuz2jIcX/PRpAroxzWQ9CSSgWywA+dQ93naLwEe4mUtTzBLhCOtO6P0143IiaXrxUpN5B9ss09xzq02S+W+yaY0rUeEBSG3r28HbRdRot4XlROoglubQkYYeKjiFhm0h22VU1BjfeLV/7yDI5wg2AqpufGdH2sPk/KcUis1kDVz78qAJYtwpgxY7xs7/79+yFbtmyQKVMm+O9//wtdu3YNVp89ezbUr18fli1bBpUrV7a+Z0n5weOC86j/imQknY2wOOnKA6bCsbOXYMh95aB11fzBj/luHr3dS+fNbL1eporcJUrMVNnY+sqsZIlBjExlFIdXkeH4y47Guah7I8iT2R6z6tdUgQcw619vBhnShpKUaDzc6c7W1IQ0ODmr2nQfXD/XCcxjRhMzm1h+fvY2QPm8WBeCpiCZarYgocfNA1wwkNGYg6uzFY2BQxzob9wdC6WpMNDBgjbAiKPlhbDvfjR2VeRU7k73WotQ6+JeP631FB1M5iMc+1m1UDZ4cdxqX9h8Lm8oIwFyB0cb8xx+GiUav/B1FTfk4jODJ16l+06Gc5eugo5M5/Ks0SYcddTPXrrqXTrv1QaQL6scekQnRK4YZIIV4YYLyXuvjV8L3y5DclsgIywW0XZZRcwjN0T+bHBcM9cXtpGZo3EgDhzhY1iIuGdaV27FTWoXpmuuxedJOQ65FuvB+/zXB8CY3R0/fjwcPnwYNm/eDKVLl4aJEydC8+bNg+uEmeNcuXIBBssIi7At0X7wMHjYeugMXP37b0B9T5fC5XVIRL5igSzwoyITIbZN2ah3H6wA91S8JfjxmQuXoawGT+cyRrEu/6ESA0GeeVa90OmHlTPguQbsd0/X9KyUZYXIUPwz1yN7v7a6HCO3vFdjTyxeVvw4zdGPhC0O2s/9U2H0sC0/WWu8Dn+wdhw9C3ikz/GEfsb30aztgAofoisZtuUa/Pjp3+YaPKVAzD4WV81m/lxQX5z4+efFK1Cm72TvIxkGVkUyshm3ipyqIyiR2oouM4p9cynGSgWyAgbqrplKbIesnlEdBuFT4iaYm3N8/VQNwGyzrtCcTbrBhI9WuZ7xjYkOS2xzH6gOOfDxa3TqEpRFlX03dP3ydxaS3l4dvwZmbj7i6f+iDrBYiPhIf1cRskkGs1bh7PDVUzW86s3emQObD52Bl5sVB67eoDJhkY37/KWAcgWWrzpUh1oWhlBJYfNuc++jHYfYjCGp1vlXB8CY0a1Zs6aH8e3Vq5eH961du7aHA0a8MJUrV65A6tSp4b333oMuXboo7+Xp06cB/0flwIEDUK1aNdi7d69Hpot0+WHFPuj23WrPyAGP410Kz2giIQK1FzELijtZm0IEuo/aVIIWZROOi0SxdRsilk1/WIdUFGRkJZvduIx5jlmWpu/Mga2H//TcglA+TFYoSMK+MWDA8tvztaHMLfYZbpV7nmn+/BgeyXt5s4SSlOh6niGf/XJ9KJg9g6lpCOd429h4fAXK9nH3JroWJZNK9A788Lg4EaKUF2JAI5G51qlU8PF9+WR1qF3UzinRdm1s6/GNgmswdOLsJag4IOD4R4WfoPAj6M+fqOZlXnmxteyVzUVFTtVJVH04a5tnh0vH56o1Ctp+NywCFQpkgSdGL/OqoqNgCmRbWZavl+wBPFrPcVMaWNYrFAOOTfDs4Kh2VaFBiVzalnV4V34h4aPjcmSAGS/VT9QmV+eIlEIKzZU6M6n1dP9hjSdL5krC4xtHfF77/LweNhw4DSqugWi73L1FCXi6XmLpMq5pTe51yCFALoGoZkEGGSqjD77g/Dti64rpJ2i2fCQjWi05AFYv5782AD548CBUr17dC0xnzZrlBbgUAC9atMj7jAoFwMOGDYPnn39euZr9+vWD119/PdHn0QqAKXOBhC/MXLgUZIIjUxcL6iRi1kWGgZS1iUHjrd0Dx0UymR/KDkdaW5Zwg7cVyQ5jOyRsUHAcnLGsCkw7jVkOk9YfTBToUgCk+iHC9umHgOAi+Dddxli2bjb2qLLrOFFsxov1IE5BFOObGlvGtl9YhsuzRseosqwOl4wa1b4qNCiuDy6oXzIRwKwdQlFUWXGbcb45eRN8MHM7yDKO+KwjPvHKX3+Di1OiTb8udThkyRVaJMNYrhacCFG2C+8Fh0bQ+II4cWY/azt2FTmV9GhlihY2pjbYf8O3Z8GOI2cBgyXElj7yaYC34aqXq3KX5HNUbfhl60CZXVMAT5bBKn1vrjLgGtSr7g+aX3QeuyL4sUlbHO9fl29WAn53H6tZyPa2A35v8FQBYRYI30DFjqN/XoLBrcp6xEexiLbLKoOit6dsTiR3R3r2/e8uHTJGgkvwEz/VBA6cOg81BweIeBO61IFSeTMZ54pzxO8NyufJvjfGBmJUITkAVi/0vzIAPnXqlIfpvXDhAsybNw+yZw8caW3YsCEsCESsM8AmByfd94sTq/AFhZlkxIWtV2jo8rZMagNBgtzdpaGtw0tTN16e4X3/4YpwJ1OAwOtsdvB0FP94zYLw+t0BFyEsfB1VbF4KXvEID7OPWFwCNqx/9wfzPW3N5xsWgRebFrd+/XFcte7l7EexYPDEjZ71KMoPUUbFemCWFVUZGrqcgi+ZBquqC2J+4+eqbJHl8ICkplQmCq7scNt+Xerx51+3CVK1WbL3JDh/OYD5RMgIShTyo/5SfSZ5WFNRXQXrBzOtjYp6gZBL4RttTk6tNmgaHD6T2BYX2/5p5R/wwrerQHbSw/smDeuB95TxApZWHy7wPpZpGevGTEFV+XyZ4efnakurkoOaDRZXJ/vHG6dTDBW+ljTVddrmLvcC63JTD/y3jJQmtomBno1RkHgdwRqebVDY22BiUeFr6R5QGyqlk2HTt8LQqVugdN5M8HuXOl51IuqJ19D3umZcdvi6Y2jCRBwrhwS6fL+ilexxva+6+skBsHp1/nUBMAa9zZo1g61bt3pQB5RFo3Lx4sV/FAmOKx+gfin+YNgWnj3++NFKweNDGz1DrsYgy4LS8aaIybIdm6weBWrZM6QBZJOLMmc2x1Eq4g3H2qkwfnWGzIC9x897x2xDJm/2lCM+bFMJWjL4h2l+ZD2NpJ9OkuM91fXHz16CSvFH2DoSjh/HqnBcvkzzpc9NZhsUnMk2Nqo+SL8YP0fpK5S1czn25u3KiDX8c1WGyXb+kai3Yf9paDlsrteUTKnA1AdJqGG9HDelhWW9QqWo8PnC50zmhKbS/Db1iZ+ryKm0qZAFlNM3HoInPzfDGTDDiHAk5CEUy53R9/r0+2W9BwGT2TLTHHUBu7gOlM1UuZlR/e+X7wOU9VKZ6iRAMxLfL5u1l9URybwuxGfXPp8eswwmrz8EGIAu3HHMu1x1OkdZf+pD5JbQ3wmKxrPrqs2J7mRHnIvf7xdpB4tqSK5rFc36yQGwenX/VQEwQhlatWoFc+bM8f5Xrly5RCuDMmgoiYbBMZVOnTrBL7/8kuRk0NbsOwl3DZ/vDXNpz8aQM6OcHCW7/fTyzZo+NaAr2gMfB+ZrI+1lYo2TLA0GeSLD288XHYNNdBZC1QkZjhTbtMHptXxvrodDkzlMUda6R8sS0LFuKPYMj5ARp4pHxJilREIH2g2/eX85eKBKggKGaW7kjjTAMTN+9uIVjw2ORUfC4YHDzsFyMxBxjIRDRDIZ4kKjUVSESerLT4a1dJ9JQRY7tuOCHxbn+Nr4NfDN0r3QpFRu+OSxxPar4QSAkVrPcC2jCS6A45GRrlSyhlifDDh631EKUJ7KpajIqTrJKZ41Vp3IcNMXvGeFc2aAhm/P9obmksHD+oSDb146D3zcVq7yQ2tgkkvE9ghqpTJzoPUzGdyoNJRd1l+sKxIi/Sh72PZPOsckvaf7nRJtlzEp07xMYiky2ftK5X4pyxarxm5yQ1Rd1+jtWZ6utmtSw3YNI1EvOQBWr+K/KgBGx7eRI0fCkCFDoE6dwPEJlVKlSnnZXzLCaNeuHbRp08bDBffp0ydJGmGgAkSTdwIuWnNebgAFstu7aJHFJmLA0HMdj6uw6CRxaK1MR/JEaLDVXzR9yYlVjfV0QvTFek70DD1EYh61rzuKJ4IaQiswE8kLF7ZHndQnRy8D1AYVMWemebgco/K2uGmJSrQe6xO+zwUTPnbxbk8/VSZQb5qPzecYqBTpOTG4eRBtZrENyk6q8IFiP1y1AOeKGySZpa7N+LAOSeGh9idqgIrlng/mw6q9JwGPcl9uFirZZdtHuPV4dl8nraXqh+A3+LkM6qH7bpgy+Lq5ycipJg4Bz8bNf60h4DtKLFyZ4aunqntqIK4YTmqTvvsPVM4HbypIsCqlAdncSZ2CXNBU60MW3CpXSZJJ48f94T5H/PQO2zKNMZz+xKwuQm+2DmwhPakRbZdVZEPCTaN2/bxXG3rDU0GoRszeDoMnbrIid3OFIRcM+d3D58HqfaecYW3hrKvrtckBsHrF/lUBcKFChWD37t3S1UBXOMQFe4HEhAnQo0ePECvkZ5991vW5i7oTHBewn/xCXSieJ6P1GEkOB5UfMKgiIfyJ/6kDJW/WEwBM7l3Pf70SMGi9u0JeeO+h0GDSeoCsIv2g6I4osXrZfpPhzIXAkSiXZqOm6BhTFmgNn7EV3pqyReokNXfrEWj72RKvGZRfw2Bi2+E/PfLgM/UTM5Vlc7QJBHVrQxqwOiIWqYJkSpcK1vQLtfRVtU14y4xpU8FawQbYz70Sr7FRUSCIgYr4IrbJ8XrtbysEo+bvglQpbvCgMS6nINRu0Ja30i0wtHWFRNPWmTZEYo1s2uBubiqrb107pOSAdWSbBTJckMGWKvSf4p14vP1AebivspuajYycalL+MDmz4RxEzev8WdMHlS5+6FwLUBbNttgErBTodGlYBLoZ8Pt00mRyVzSd2FAGFaUZEWoWiYInZUV6BohbWCIJUxPHN3vLEUCdcSo6kjUnoWF93NTUKpxYcYVw07Rh15FoR8/fCf1+3QD5s90Ic18JBMuq4pdw+Mgni2DB9mNR3UiEe9+TA2D1Cv6rAuBwHyTX66P94HGpGZmAvW68SCTAIyIkfnzxZPWgg5iNsoHJ7Y3cevxYp+qCHRPmVkWGoDZ1TlqcFCjCQEjWiGSSXH4MqW/Tj77p2SK8o84FjDCDGAQiJMamkM2yjBhlc72pjg05kfC8PVuWhKfqxpmahIXbj8HDnyzy6qECBB7vI4HLZUPCOyGd24er5YfBrRLDogjLqMsQGgcdZgUiRNkSVcXuSO8Z/y6TjtMRNFVHzDZTkt1/k6Qbd11USb6Jm/C8mW901nGl8ZMmcRcNye/BEQth8c7j0LFuHPRoWVI7ddrQ9buzFLS7TQ0Z4RtrmX17OBbUugHShgbr2J662NxrsQ6/R/iZjmSIMDNU2vg7PjBXSZGRsQrZluu4H/Q+tDHxIJ13Vx3pp75YBrg5fbBKfvjf/YnfHX7WLdLXRDsOifR4Y9lecgAcxdWO9oPnKtDOp8qZ7ygphsdIWGxccNbuOwV3Dp/n1V8isfokFnQklAUoC4LEHST/pE6ZQnnHav9vhmfpLPvhMUnW8M2EKIRO8yGZLFIgUNl1ygZowk2bHkNyShKd9/h1o+bvhNctMx50Hf8R3jSgOaRLndI0FKfPTTa72Jgu+yjrjLCTxI4nDG+BbOk9iIwrGY6yo6I6CPWts3J2WowwKpMGtozAZtPsK9+vhu+W7fOqygI9CvCeqnMr9Ly9VLBJDlf49LEq4KrrLQtQDp++4FmQY5GRokKyegrdXU7mwk0QBjmuTl40yfpvzoRdx84l0pHl6+rimGjCvFO73OFRRmL+zzcr4edV+z3FGxGWZXPPVXUIz4yfR1PaD6FJuHmibLMKY0/jpAQG/lsl9Rc8sUqXCtb2awa6zZKLTv745fs8J0FXTX0TfCqc+xSpa6Mdh0RqnNeineQAOIqrHu0Hz499I02XsguNSuSCz9pVBSIV6TKMdK1JbovgBCXyZIRJL9T1vcKYNUXi2Ilzl0FljckbJ0KCTBbLJgNLmRvxesqe3VMhL7z7UEV4cvRSmL7psKdniRkUm8JxxNO61YUiuezhKti+jRKBztFMNUZOrnKVj7KZtw1OnWS2/tOoKHArU1X7n8zZAYMmbPRwn0jc42RQV5c07MOU3SOViGgy5k1rSdhHm+NcWVvkoIWfyaAmKrc+k+Shadwyciq3l0b78aK5E38XSJZN9T7iwSPJqxGxznRSJI6ZbN3/d19ZeLBqYo1arG97CsDx+ibTDBPxqsPnS2HaRnzPyE8mTGuv+py+b/j5uE41AW2ko1XovYXtmzghpBeOdVUum5SpJZc9nf27WFc3R+JC2GrhU1syd9ForaXfdqMdh/gdV1K4LjkAjuJdiMWDR8eTSN5BEo9tIZvIO8rdDMMfqQQuci4mwgBhr0wi66axUqbvhhsCJL/82fQkP3qByrB3XEpMhRGkHzmRDEWYPsL/PffVCvhtzQGggNg0D/x8/f5TcPuwQNbcj4wVsfh1MAEizZS5JRP89nwoyVM1Ru4ypyIc2cxPVYeTW1B6CzOYYiHVEMRTI4zBVOj0ovqt2eDbeGzkHe/PhXV/nIYWZfLAR4/Kmfyqdonk9lyDIvBSs8T6zGTNiyYoeDR7LYpM/sllHO9M3QLvTd/qXSLT+lW5FPq1quZjE8mpfFM09xX599pEvJNJpflRE8FxksmFLnC2tdXl7xmTo5jJuIZOmjrUvhV63ZGQlXe577K6HA+Ozp/IA4lWIXw1tq+yN6a+249a4tklY1GRsUktAiFbiIXXnTBN23AIOnyxDPD3A41EdFrGfhU3InnaGa17EIs4JFpjj3a7yQFwFFc4Fg9exf5TvAwpWviila9toR88wi41GTrbswO2wVGaDDjoOMmFjCUbNzrVoSSSbeaNAilZttiGMEgOTgWzp4fZLzfwhoRHwOX6TYEzF68E15iOk01HenxOXNrJRmpOXA8SlZdJuFFdsgl1sS21ycbZPlOyegu2H4VHPgk4dG3o3wzSp0msVU0/kraQEiJZ8qNhIscgGW5B94aQK2M662HTBkf1A03PRbgnGtYDklQMYvbzZ4Gfn73NuSn6gccLZZJx5KwnEld1GTbbQRA5lZwhQyBUPRpBrkyJ75VJM1uEweBYaBM/6N4y0KZ6gr67bpw8w/3FE9W8d42sEMzGpO3Lca+m4NLEpbDBJtveA17v2a9WAGZHsUTj1If3xR3edBl2vIbWGP9btVnmvz0Y1O46dlYpf2fCWPNx+v2Ok9QaEseRQJ4USyzikKQ4b5sxJQfANqvks04sHjy/rmuUBWhXqxD0u6s03PvhfFi556QV1GDSuoPQ6cvlHvN+2xstE60O7dJT3ADeLt2Pi9C2w2eg8dCAxJstTk3HyLXJOnHWMlnFchIPEQRthPPFRZm56TC0H73U+zO6cKXSYJlljxvdH1WWEq95/df1niKCzCpa9QgfOXPRkyHDgoFV+fxZfD7t8stsTA1c8JXYC22MeGYMpdGqD5rmaQO7Mtsp2FI5yhG2mksvRXSRLBpDK9mRc3ZIJcwsLgfCEGNdmdLLy+NWw7jl+6BZ6dwwom2CFjJXZJjSta5nOOFaRHJqCISqb1MPdykWk/QcbXg44dMGyyv2w7H5OpMZ+s7XKZoDxjxZXbkEKJeHY8di0mY3WZwTOVSmTe56D3j9nj+uhbGL9wC+n7cNaumMmXfpmwJLvMbknEmbPKzLXQN5f/O2HoVHPwtsqFEOENew+bsBCU/xNEHlQigbv8xi2Waen83bCQgvQv7BnFcCSZOkVmIRhyS1OduOJzkAtl0pH/Vi8eAR7tX1Jdnqw/mwggW8hAGUMcTFqZssOnnWD2XDMjg41FFf/X/dAP83f6dHbsGjeR35ja55YvRSmLHpsBRrxn+YlvdqDNklR/H8+JKyZPw6JNvkyZwOhkzaBB/O2g4VC2SBHzvbZeN+W7MfnvtqJRB2zfVxoiyp7jiU1DcI123Th6inKpMesmlHVcf0rOB1rkxq0qwV4SCEa8dAFSEztmQ4cvnre2cpaC9h7X+3bC+88v0aQNOYlX2ahrMcvq8lrGGD4jlhVPtqzu2QRjReuLhHI8gtZF17/7QOULe1fvGcMJq1zzeirlrjNEiRnCoGMWlTJSZe0gb9sZoFoT+zLac2ZYYIrmRKbMs2Y0unKyZiL99Ey5Qd+I0zEURJttElo23zYND7C101l/duYnOJ7zpcw11FbKPGSW0H/711UAvpO190P9155GwCIVs4TbCBX1HfJjt01QJ8u3QPvDp+LcRiLf3ehFjEIX7Hdq2vSw6Ao3gHYvHgEfbRpDkpTlP8saBjMRtcK3eRkwUEHNvmB++KhLXqb0wHxB8+37AIvGjQ3aS5EVlN5sDEg3Kd0DmRNggKQgEcyuPgdRhU+TkuoxclSam5PnZkLvJojQIw8B458Y7UClxMIThJ6bPHq0Cjkrldh6at/82SPfDaD2tBN2969mxcqRCSgq54KHsmEqT4c6czDBEHbMKbBg1GUqaALYNaRHR9bBvzc29520S4QuWMVX2aJrISVwUAJpyqzfhFcqoNNpOeCZWWuAzvTlljl3eGDRwD52iLr6f3xY2pU8LGAc21y8PhJb8+VxvK5sscUt+FmGxzH8RAUydL5tKeri7BPPC0EGENWdKnUVYnsxc8EVjVp4n05FAkDu44+ifc91HAxRSv4e1zfsOC1xpCXomhCg3GL5kt3MRGpNZZ104s4pBYzCMafSQHwNFY1fg2Y/HgPfDxAli664TSIlg1vbpDZsKe4+eArD27/7AGvl6yFxqXzAWfPl5Vuyom9zB+bOpq0IEdk3wNkhfwWCtfVjuHO50kjc2PLvZNQTQFkRTsIlEEMX1Y/m/eTuj/m53AOi0kHZVxfLHLo0f2qjot2qChQ8VbYOiDiQ0dVP2V7D0Jzl++KiVHuYxRVtfGarnbt6vgh5V/eCROmRMbb/fMhctQtt8U708yW2jSaG5aKrdn8W1TTDh6rhHtB75iMwZTnXAVAXDjgBlePKqtXzxXou7emrwZhs/cBhXyZwHUFKdiUnwxjRs/F8mpNux8gvPUKpwdvnqqRqJuZCYRfkhjnNCrwqhj57auYuSweXPmdJ4xi66cPHcJKvSf6lUZ/0xNqFwwQY0BN6/w/P4AACAASURBVKaFe07wdHH9yM/p+kVr9S8W7gaEc5S5JTTotrmfrnVw858pXWpoUVZP0sZndPSCXVA0V0aoXTSxCQb2K0pwoiGRimPADXNQLQZVY1TlpXGrARM7riRajkm+Vu8G0/2IRRxiGkNS/Tw5AI7inYnFg2fSMFVNj3B5RExwcR0y6c1y5vi3HWtA9bjsTqt8/0cLYNnuE+B63KsjqtgaCVDAS3JTRHjjph5+srlElih1cyaY4IMsYaMJ6keeDW8MaQybSCpONzG+MuH6yuXLDL88V1vaBN03G1Ih/uAhZhfLjBfrQVzOUAY73RtkiWPWRzzqlw2AJLdk6ghYn2MJ1/Rr6v2Yx7rQRrdTvcLwWguzUobr+Ei6UCTzcAtmvzrRIjnVRnOVVC8K58wA018MOHTyIjPbIRUB3SmJ2A7xGUxGMCRDZ1K2cZGA1Om4Y5CKJx1YZBs91/t7vdTfdPB0EPOL0DjkdrQbJedWoF09nuhhQQv7EnnUDqcup1B8LZfuOg4PfBzIQPshN8fivsQiDonFPKLRR3IAHI1VjW8zFg8eSXe1rpIPhtxf3no24o++C5uV2yhTVpR3zO02R7atDE1L57Ee15ZDZ6DpOwHy2yePVQEMimyLjqiScBSf1juKUxXOHMYjNST7LdpxHIgsiNfZ4FrF9kkup1qhbPBdJ3dbU2JT6zKbfi17CQNLpwG2621Tz5TJwzZIZ7desZzw+RN6fCuHssjMAzCoqD5ouqfaYZJdovGTzfSItpWhmeRZjQQMwGatdHVcVFr89EXaynE5MsCMlxICTpNdr01fIjlVtLOVtUGnQCqLblKt4EogdHqDajioimNTCM6VJT0eu6vx3babXpdEArqfFe050RsmPvf4/FPhZiEyeITN3K7HOtuP/AmN3g5sgNH0Bn8vOo5Z7hH6dgy+PWTKnOBoIvgmnLDYa7tjZxv2n4aWwwIkPBPM4lrdj1jEIddqbuH2mxwAh7uCmutj8eDZ6lPyYfLjNcJ9mrK6/HqbbCbZbb55fzl4oEp+61WmIBYFyVEL0kUtYfDEjTBi9g6QBZkEWzBBELjqA5oqvDZ+LWAmgQeHNsoG4oSJee2a1aZ2bMwYKHPumiX0Qx6yvaF0tCiqC/DryaShZlx2+Lpj4uNuXpecoG5KmwowAJYVInTlzZwO5r7aEDC7pyohrmPtq0IDCTyAH6XKss62axFOPSJEvXFvWXikutysIZz26ehezHAS/tnVIpaPhcipOG4cv813kWeeZdCEhNOOBJMIP65ctgofJLume+5wzi4SiTqXvaTwzIXzPEXrWg6vQxOVzYfOKMnFHC5FCj6qcbX5dBHM33YM2t9WCPreWdp6+NzgSGXqYt1YlCrGIg6J0tCj3mxyABzFJY7Fg+eC3aWpypj/5LFuyoRgGyq8IF9KstvsdXtJ6FAnzmqV0Ta12hvT4MyFK2DrCsYb1h23uxxNUkb0hcZFPfMAEYdno20rTpg2KjY4V9likQxWtVuzAb7MZYV0Q13XjiTWnm1QGF5uFtnjdcIu67JyLkYThMUUM5V8PTj5xWTtLbPqFdf20OkLHikTi8y61+rhDrMSuZy5Gt7YdksyaSKbnTKx4Wh6i+RUG1MPLluImb5CAn6T3Pu4KgrBImx4DLQutidfk9cfhKfHLPc2UyjtqCp0ImebhVaZcHCsq0y1w/a+Xm/1Dpw6DzUHB2ANE7rUAYREdPtuNcieT7RixvXFYnKIdDXjoXW1ldG7lvchFnHItZxfOH0nB8DhrJ7h2lg8eCQX5qL9KjsassXC4ZTpmI87cYlLQXab5J5ms8wUhGPCbt6retaurL0PZ22DIZM2Q7HcN8GUrgHCGhWS/rFx8+o8djlMWHsQ0Phg08EzXhNcA9VF65P67/D5Mpi28RCQ8YjNevA6Q6dshmEztnk6vSojBL/H5H6hEzZzoMwKh5CI15FLWdlbMsOvz8txwnQNPe814rLBNx3VUBIK6k3B0Klzl6F8/wCpTiQiUZ+nL1z2zFCwmDJJNmviWofbeH/5ZHUlQci1XV5fleH8eskewMAS5QiX9FRDh3R9i5lZut86XLiJRyDb7BHcpnaRHPBlB7VWLx+rLWSBy5upJLqwXSLi2Zq6lOk7GVDDWlQ0Wbj9GDz8ySJvqDKoTzj3+p98rfjbhQEwypDJVGYwwx7Xw45ISERNW9gUrWEsvpvh3q9YxCHhjvFaXZ8cAEdx5WPx4L05eRN8MHM7VCqQBX6w1KQVj5GK5s4IJntjvkx9f14Hn8eziFWi8H7IeRS0uOjY8nHplBZsheyxPcpQ8ba5dBrPTtlqo+pMOmweQRvpNZOeraofVx1em/FSHZuNEM2teO6MMLlrXW3zZEOtkseii203UyYpKmzvytW/oEg8VtNFXs1lnXR1Y4EHVZnbuECjVHMQyakEVdJp6mLwUrLPJLhw+S+pOgmZRHAtaJeTBBqr7QmarakCwYl0jo18nSoNmAqoPy5CxUi1Buui45mtpnWknrmk2g7fGH3fqSbgaU/vn9cDwp0WSFQ3SvSe6D1DppMTspp31dPH5xTfDQilsjVsivXaxiIOifWcItVfcgAcqZWUtBOLB8/laJ+GyJm05Lluq4eJbST8aOSGTx+XS02ZdDzF5eLH1n71aHXybITN02FRaUxcqB//Jma/uIC9zFVL9kiRPJdLRpy3YyMn5hcnSvCMO8rdDMMfqRTRb4QsUBE7sJkbXdP644WwZNdxeKrOrdDz9lLKsdrCaWwl+1RH1RFdLEVjfjZcruNSSb3RZrBorptgarwMoGvb4ubTdjOqg1HJnnWCM5TOmwl+72JnS2urf75m30m4a3jA4W1Jz0ZKq21y5ux/d2l4rGYh41KpNKgpI58hTUpY31+vJ2zs5DqqwCFLqI6xfv8pGPj7RiiUPT3Mirev59Mt//oUT09+aOvy0KpSPuVKkP677X3jDZXrNxlOX7hi7ONa3YZYxCHXam7h9pscAIe7gprrY/HgUdZT9QKQDU8UE8+aIY21IxK2F9RtLXszfNBGHjAR6Ut0llItlwtxSdWGjtFNmUMbswV+LI59VSmYFb5/plawW06UG/9MLUBYhamIZgCm+uLnqOGKa6TTF/X7IvbjIGc7ftEGV3bdFwt3QZ+f14NJYgqvdcGW2xAqRVa5iDWl8RKpE9UFEN8ZyxILqaVFO47BQyMDR+7cvdHWAEK3HmLG1zbrStJvHevGQY+WJUO6kJlEjJyzHd6YsAnicmaAGRLpNNkYbR0wuTqNaLnL23U1r6BTm353loJ2zIWQlDLCgZ7E8hmNVV9cOeOLJ6rBuv2nlLA3HBNtlAa3KgsPV1OTRysPmArHzl6CIfeXg9YOpG3sgzY9A+4uDW0tNj2xWivqJxZxSKznFKn+kgPgSK2kpJ1YPHg2kkLi0HiGk7Q9XcD8NpqJLnbBXLrK1dGOz03nyiMy0U23nYxCsF6rSrfA0NYJxhIc92UiV1A/Nd6YDgdPX4CB95SBR2sUNHWf6HM60s+WIQ2sUNiX+iVKkQqDCVfrPGgAoDENf6Qi3FEur7QJkqjLmTEtLNXgTE3H4mLjNpJ6tm5R4WSI/Kwbv4abuGwfFJ3j8JV7TsC9Hy7wuuVW4WQBLG4CXeYkklNtjU9Up0jcvZCbRLhspGj8oiW8al4y2JhYlwdnoxSKIuI1qqN3lSydy7pfj3W5cgaeFK794xS8O20rlLklE/z2fOKsv2qDIa6NSQtct5Z+uRexuj+xiENiNZdI95McAEd6RVl7sXjwfly5D7p+uxoypksFa/vJZaHEKU5Zf9DTTuTi7xevXIXivSZ5VU1BHWFGH6qaH/57XznpCgbZ+hbZGGKg43jmv9oQ8mRO5+uu6NzeiJxiOjqnjunHF/+NahAvNC4WHBPHfam0Y8UJlO072dOmfffBCnBPxVuc52fSHhZ/GFwsjYMEu3yZ4WeFWYXzgAGAs7BFnVPeno0xAtbnZDRbgxWTqQonNKI+dI6b0kqniuYbaMKBJhQoMxfLYrs+4YxJtRHwQywTxyGSU4lkKrMs59eqCI9IGkPyGBZuEvHdsr3wyvdrQFSysAleXmleHDrXL6KsyrHivzx3G5TLlyVR3WN/XoTKA6d5f/+hcy2oVMB8MqSSIHQhhoZz3/+J1xbrOREuXf0LPn60EqzZdwo+nKXmwND3tnuLEvC04nuL787CPSbAX38DuOrW4/rd/cF8WL33JDzXoAi81Kx4klvSWMQhSW7SlgNKDoAtF8pPtVg8eJPWHYBOX66A1ClvgK2D1PI8fPykpSqKzNOL5cM2laClxraSjg0fq1kQ+t9dRro0CcYTaWBZryba5aMXiI0TmK4hbmKxeWBzSJsqZbB6kIzVqKhnkGAqZPaB9WT4MZeAlr9gXc09aJy0aUGFDJRhugF9olnhWemxHarDbUXkVqKyeRPOk9s9m9bH5nMk9yDJxxQQ/Lp6Pzz/9Uq4MXVK2DhAjXfcdvgMNB4aMEkxWZvS+Ey22rbkpiCG2+L5QcwhyjKJ98hmzWR1SDcXbYznvNLAbzPa61S6s5GAx4jkVNLwJV1g1cBUkndclo6bRJg2ibJ+6GRmwD1loK3mZMZGCYTDaaa/WA8KCy6Fsv5V76WBv22AT+fthGicykTlAYpho5Stff/hil7gqVunlu/NhQ0HToOOlOgilyabpo3SDf4G4HshS/o0MVypQFexiENiPqkIdZgcAEdoIWXNxOLBs5Xn4eNTwSYIB2WyxKVsKtfgFOc/ce0BeGZsIDDfMrCFMhjYfPAMNHs3ENTYHhuqbpkOK2mTCeDtclUMmTyWirwiG1uI7nKH6lDLITil9uZsOQK48cAik2HiuGVkR1cplM36yfZzdGzTuK1IvK3GKjdH4DhV3VhwY4AavvjjI2by8Trdpom3a/PMY/3lu08AatTiBhKtlSNRYpEN3H/yPNSKt41FfdVSeQO2sTZ4f9McRXIqBQxP3HYr9LlTTWSk0y3RfIIHmtyYxLRJlI3TFrOrc22jdjm3QneawMeh0p8lnLRfRRzTPfknf86JbXiC84VGkYiUhXTZWa4sMa5TTajq8O7Edez4xTKYsuEQPFA5H7ypcCBE/saXi3cDwjYalrB3N43EfYpFHBKJcV6LNpID4CiueiwePNsMFp+minVf/82ZsOvYOTCZVxBuTmecwANIXbBC2T90mtrQv7nWtct0q3RMbVeiArKNMWjGnfu0F+tB+jSpQrpv8NYsjzhoWiu8iB+fmiw5VXNcvOMYPBhPUlrbrylkTJc6pGo4Zg2EL86aPjWs1NjBmtZf/JxbCC/q3kgJbZm5+TC0H7XUuxyz2yrnNpM9rmp8ZJsrMyGxtfq1zVoSEUumS+q6flSfyHwu+rauffFs/Y+da0HF+OP7Z8eugN/XHoBWFW+BoQ8m4OBd2hfJqbamA/wdwrVwVYo1fDODm+40qVJoh8ldAE2GKdgQHpPjNSrYE3+OdVrBfFAPj1wEC3ccAzGZgCci+G68q3zeiG2kXO5ZUq5LxFpM1GAA/PWSvaDS+7aBvnGZQT9GN7RJbFk2D3zYprJ06QjrrUsaRWvNYxGHRGvs0W43OQCO4grH4sFzkS+jqaqsjG2dxEg0XEdYQ3ma24fN87rUeaR/vmAX9P1FrePocnt0TG1i8b/9QHm4z5LFj0djf8PfIVAKGg8drdmQ9nYdPQv135rlXTqtWz1AqIFrwaM+hIpgkWWX9hw7B3XfnOl97mrJSXa3GDBg4BCpwsX8dZsgWw1qgqUUzpkBpluy/HEuhCWVWS3TSYVp7qQick+FvPDuQ+rMLhG+MIjfOrBFRPRb/Vj8ut7DsxevQGkJrjbBcrgAIJPeTxHJqXe8PxfW/XEaTN8dxFzjJhQLz/SqnitXtQye+bMxOFGZVtCa+JEuU8HJiLSLygV+193PvfonXEPJDCQUY9b9hxV/gCr4fPz/lgCekurgevzdOa1bXSiSK6PTMpCCUb1iOQG5DrJCvz93ls8LCN2IZYlFHBLL+USyr+QAOJKrKbQViweP/0jMfrk+FMyewTgjYnaLQvSUjTAdTdqwXm21cilgULF4jZNhFXQvMsI3f9SmErTQ4Jtt+yOJpqfrxUH3FqESTWIbtplQXd8y7WZe31amSdYHz1xtG9QCUqXUZ85s18j2SHrZruNw/8cLvWZX920KmW8MzW5Tf0TIkgWyujHpjDZUeHixPdKRbloqN4x8TK59jdeQogb+98reTQAlBsMtrgomfvpTmX2QS2D72wpB3ztL+2kauKnDzsEtock7c6wIhWcuXIay8Q5833SsATXisnv9q0wi+PdsSY9GkCuTnkxr+46iSVcZOBWO/nkJVBAxghKpTBlki0cOkQ9Xyw+DWyUQisnq2Za06+vG/EMvIoWevneW8iBHv605AKqNKcETdA6cHIank7hTLZdJKYV/t3TuqdG6HbGIQ6I19mi3mxwAR3GFY/Hg/XHyPKBEE5ZJL9SBEnkC2D1d6fPzOg83VbdYTkAtRSr0smhdJR8Mub+8sgl6AfW5oxQ8UftWaT3Vj5dYmXbPdYrmAJWrnGk+9LkKBsBfQDo1Att+sJ6tfijWNcEXbPo1ZZH9nARQv3x8a/o1hUwCvMJmfLI6pF6AhLA1GoUSDl1BGTSUQ5MVOo43ZWHFawnzLpNZIwUSk3KALQyBXM9wDLZEKNP62kIGTO2YPi/SYwJcERytSEUDlS9QAcNPEXHWmNXde/w8iNq3YtsIPyrVZzKcv3w1xCpYlWnlJEkbh0bTplIcD0nhvX5XaXi8VmKTi/enb4W3p27xLNQnvaB3NKS2SRFDlFqkLLkMt+7nHlxP15CmOrq2Ldt1wsPfqn6zCEqie2eYTtdMa2cyo+ISo7fmyOAReGNZYhGHxHI+kewrOQCO5GoKbcXiwbNl2vOhvTRuNSAur0WZPPDRowmYpRe/Ww3jVyT+u7hENpq2tlJhfoMa2W1TEcE4g9sPyUHW1zNfLoeJ6w6CScoJr52x6RA8MXqZ14xfW9MDp85DzcGBjc7vXWpD6byZQ4blevzLL+aZs8U9GkFuQ+bM9itD8BaTwQUPROa/1tAzxJAVCsaerhsH3QVjBN2YuM0vYjO5OgMZjJiydqRrbbIcJ6gEjidSz1qsJNhkR/wEiwonEBOfTZzPkTMX4b+tysJDGnMCXEPiJXDLYyLViSYRtq5+9KyEjKtPU8icXn7yQPVNRFrK/rtk+ciFUTwad+EY2H4fr5d6XDoO7+GszUfg0RoFYOA9iSE6qt86vhbcBIZjzW3Xy2QXzk/n0qOz3+vNIqYQYzPGWMQhNuNIinWSA+Ao3pVYPHjcGvIrS4UBVdbBNstF/vUm15yK/afAiXOXte46BLsI54iVbqFKCuzgqQtQY/B0r5ofkoPsEen23SoPeyZuImR1/cgzie2Y3Oe4uYkoAWd6xFWsetN1ps9NmRG6nvevkzejkwcb4iEfGyeKihALIoSanBQJRmHK7rUftQRmbj7idW+rEW1ax6qDpnkBo8nNytSO6fOgG9Z95aB11fxedRu4k6ldkZza6O3ZcObCFXjnwfJwb0W9q17rEQsB7x8nDwUd33JkgBksm8YzbTZkU74x1ZEvaX6mrCwFWyaYDF+vl8ethnHL94Fo0R6re266d0nxc74pw2djwfZjoILtkSNpwxK54P/aVZVOR2UDbjt3OkVSmRTxABvblJGYbfvyUy8WcYifcSWFa5ID4CjehVg8eH4MENqNWuLtmlH3EvUvqbw9ZTO8P2MblDcYIthKB9moSjR7Zw5sPnQGXmpaDJ5rWDSsu6FaC65xaqsfaxqIDfGB2vh6yR5APVU0WUACm58SstF5qjrUKhyq86szATH1F40NAvY5eMJGGDFnB4hYc3E8Npk7vLclek+Ci1f+8kgkmDGzLRwnL95/MmkonjsjTO6qPra21eJt/fFCWLLruDe0SASsOO/ivSd5piImfW7b9VDVk7ndkZMW4i3bM6tel75EfHqjobO9+djg8WVqCMQbKHtLZvj1+drBoXCDDBujFFfSmgkOQgZBJggZXztVgEZat+89VAHuruBunONyf/5pdbkK0eIdx2HZ7hOg4mIQAfa2ItlhbIca0qnSCZGLlj5v6Pc1BwCNk1REWiIZ0zWRgkbZ3rdYxCG2Y0lq9ZID4CjekVg9eCV7T/JwcraBAf1Ii0fJwcyKwb2taM8JcPnq354TT/MyNytX0MYhhzIdb9xbFlAYP9xSrNfERMFCNI74KbirVigbfNeppnbYtllGXSPc/nVUu6rQoESukOo6G2jTmrqy4U3t0edkoqDLvmBdLkPEjQ14PxzeYsPY59dymND4Z2pB5YIJDl3vTtviWamKwZQ4x2+X7oFXx5s3MaSQgte/3Kw4PNtA7S5ms4584+NqcGLTPq9DUk0cblBt0DQ4fOYihPP95HrQU7rWhabvBHS/baTHZLAClXW3isinWocvF+2GXj+tgzyZ0sGiHo2My9X2s8Uwd+tRaFerEPS7KzEhkLLVLsQ12akb/66jbqyLq6NxEtdBBSIIdqwb5/ErVu87BV0aFoFuTRO7sP1v0iZAox+dlbctEVa1dDyDLJO/o+eMrv9KksCI5m2JVRwSzTlEq+3kADhaKxtDBxZbSAJNVXWUR5lKGVmIruUvZ9MPmIkohtmtoj0nxhNv9MG07W0q128ynL5wJcS9zY9Wsqm/96ZthXembYHSeTPB710Se9Dz6ynIsqmr65dbgIobj3DscrnI/+j2VaF+8dDg2rQWqs8pe3d3hbzwnkY67OS5S1Chf8AxTmY6gn/nWURbtRP+zBbpGbA6FZ346AcSg2IMjlXFFsZCpx7YzpO1b4Xed6iNHmzWNVrZeVnfLd6bC2iJzF2zZN8nm3HzOpyciiYtpPhhA9miTTknD5FJhEz7NWFzXhmal8mjHaqrAyIpNqgs4Amb6nKa9caEjTByzg6odms2wI0dFlsCset9uF7q840Iwgs2HVSfINpscBNcS/2d0HEVm1V9miRyeyPZUVr/dx+sAPdUjF1WPzkAVj/5yQFwFN8KsXrwZEeXumk1fGsW7Dh6FnimB+uTKUW61Clg0wC5HqyL5a5JzD2crJ5qfpSx4sfPtkYLLo/CJ3N2wKAJGyFOwCHK2pD9yLn0RXV19sui25Zr+5GWicP+CWrTpnoBGHSvWkOWO+V9/VQNqFk4IHfFC8c4b+zfHG5Mk2BzbTNXlcshZRNrFc4OXz0lPyLF9skwAx2otw9qqdT3JZF+vObeirfAOz7NI2hO4Uo02awN1ZG5ZhXvNdGDnYQDv+Dfc8xoPvl5gBAqZuNlYyWYApKH0CgHi25j5WJRTsTGigWywI+dbzMulUkLuubg6XDg1AUYcHdpaFszsUqErIO3Jm+G4TO3AR9DOKY2xklcBxUIZ4/EN8T/7jhyFlARomPdwolmR5ucYrlvgild60lnP3r+Tuj36wbIn+1GmPtKQ+cVwk0jbh6xyEi8lOWnhru3KAFP10s8VueOLS+IVRxiOZwkVS05AI7i7YjVg2diJ4tTVKk4mI5ysB0X28heP62FLxftAZVAODqpIdsZix8Bctmtk2EWCYOFbnObI2T0QMdauTOlhcU99LjeHj+uBZTiMkEBTI8iBVcy9rwtRlXVBwm1v/VAebjf0ijENF6S7zJJaNnI1FGGO2O6VLBWI6mmGhN9R15tXgKeqZ/w40NY7vrFc8Lo9nIRe2xzwfaj8Mgni73mN/RvlsgZkPolOBL+WyeMb1o7+pyfXkRSok7Wv2j37IdfIGuXb5qHti4P3b5b7VWzIaRy0wti6OtMInTfEXFsKjlI1b1REdbEe++C25WdJNmSQm2foeutHtf2nb/9KOw7oZbUo/diwezpYfbLDaRLQUFy0Vw3wdRu8iBZt4acw4AQn2K5Q400yImS2ojEyZDLPY1VHOIypqRSNzkAjuKdiNWDZ+vgRlOlY02RhY2i4hi0YFGJ+CMbHXG7WH557jYoly+LcgXfnLwJPpi5HSrkzwI/PZs4w7J893G476OAAcKK3k0AWbThFhlrPRpWvz+u3Addv10NNgHZf75ZCT+v2u8Rt8JxAar9vxney16mQ0pkLl2mQ7e2rlbRNveJCI42WNi47r9LIQrUD80PXfTQTc+1qPCZtgYXNlqh3FoXx2fCFdvMYeqGQ4DkqhQ3AGzTZJ5t2jLVEcmxLqc9urZ5II1kutd/3eBVt3FF5MEgkYd0WFvaAJs0hrH/oMNe2ZvhgzaVTMsDOuIrkvqQf4DFBUYky1DqLN2Ng/wXVEDCGRLP0J573rajWow6JSpuzpwOFnaX47zfmboF3ptu5gGoltYkRUpmMnT9HeVuhuGPmJ+3SN3KWMUhkRpvLNtJDoCjuNqxevDoB8FWH1UmeI/LwHGWKiF57p5kMt5QyRXRkvMf962DWgLax4ZbZPhmcmgy6dG69E3MYRwzuqdxbVmxnQ6fL4VpGw+D6Pbk0h/WJQF42REaMePL5csMvzyXwIy37YPaRrMDzNhGorgE1XTU/sEjleD2comJlXSMqGNz68bcacxymLQ+sW6zSodVbGvroTOegxkW1XeD601jPdPzhj+cePyKzoQlb5Yb2KBeN8prZUmfGlb1aRqJ26Js4+kxy2Dy+kPwQOV88OYD5UGlq+1nEERO/U+jol6wgcXGdYsrOxB5iIiGMm1iF9k2+l6qML3iPAf9vgE+mbsTZDq/PDGAm33c9NsUGUHW9rTBpv3rsQ7/zqLJyslzl0F1ckXJD5VEGa7P4IkbYcTsHWBDaJatJ98ojnmyGtQpmjOkWsv35sKGA6cB4VN//w2++/F7L2MVh/gd37W8LjkAjuLqx+rBs/E7p2nyTIX4ZTWZLWAbXFJsxov1IC7nTcoVNOkjErNe93JyvT0yqaKPZ28HtKv0e8QlG4PobpU2lRqTKh4tu86J6tMPW3dYawAAIABJREFUf7cmxaBLo1DJOPpx9vsSv2v4PFiDbOpGRQHbj0QhPKaN3mvZfpO12rCkXe0XV0swlAbFc8IoBnWwNTSx2fjx7w+u342pU8LGAQHcqqwQOUaHP6YASXeEG4l7hW2IJxWRxKLSqRMqKIxesMsbss71j89JlF3UySu6nIa5qjYMnbIZhilkInVSe7r7M2bhLuj98/qQzRIlBjBgQuMc3eY6Uvf+n9QOQVGal84Dc7YegXOXrsKwhyvCXRJpRCKvZkADingMuTjXvj+vg88X7ga/bqQmMjdBDlFrfNexcxCL7zKfY6zikH/SM0RjTQ6Ao3jXYvXgUXYLsZu4E9YVHeOeZ1u+6VgDasQlJiPZunbhGExZ0nCPtWXzpOMmbqyh++Hye/s5fEPG/OXt0o9yuMEl6V92rl8YXmkeaktLx7kty+aBD9skuPvZzo+khbjhgO21snqoFlK45wQv4/HpY1Wgcanc2uaqDJwKR/+8BP+7ryw8WDWxHB7hiVV6n6axEtlI1Lh+cvRSmL4Js/MFPN1eVTEZkeB1PAiidnSEPXJezJf1Rpj3qpx8Q8+u38y+aV34569+vwa+XbYXyMhhz7FzUPfNmV6VcDH6RE7FY+sfVv7htWmLaSbSLhGdCOcr01kOSjzWi4PuLUpqp0+ZOdmGUnYhmaHINKM5fMwFziVTICBZrpvSpgLEPSeX0BWgzWyjErlg9pYjWhWhyesPwtNjlkMqPKl7o6V0KcXn3s96l399isePETPRno53r0lw6epfgAE7nkLhxhh5BLHa2MQqDvGzbtf6muQAOIp3IFYPXhDLVu5mwCNkXdl/8jzU+m/AUndClzpQKm/C0St+WeN6BIIWUS6K2uRYSFMGR0Zg4WMLN2spm6csoBn42wb4dN5OqBmXHb7uqGb6uzwKJuYvb0v8AXfph9d95JNFStcj+kylUWrqk4hFqMWMmq/hFi7lZGNKYIJLEP65zx2l4InatzoPj8gwItPbpO1KHfFjzi+frA61i4YakWA9/t2g63TWztR3mpRIzmwu/UEMNzvlslAiKUw0sMifLb1LcyF1CZuLQQtuOLBsGdjCMw8wFdqcEXmIiIayrB+t6eM1C8LrdyeY/Mj6oDHZPlM6QtXMTYeh/eilXjcIiUqV0jwvrCuTL3Qh2JrW7nr8nOBQmLFFXWYsKklOG5c3OvnADDI+U34KKTGJ/AyeVELIDmqOY7Hd/PkZi3hNrOKQSIw11m0kB8BRXPFYPXh8R/yZwu6Rprnt8BloPDSAZZTpqdJRNLK1W1VKbFOK3usPfBwgrom2suJSbth/GloOU8vDkJ0w7ow/buuetZTdumfHroDf1x6AVpVugaGtK3hVyJABf3xN62P7OHBxf1N2jLJfg+4tA22qF7TtIlE9kv+RyYoRhldUObDtjIgl91TIC+9qNHtt2+Mbrd+71IbSeTNrL6VjbdTNxUCHF55FGf5IRbijnL0LHLWjcv1SmcKIg8UxFOk5EZDoNrJtZWhaOrHG7IJtR+GRTwNKEVRUxh74edN3ZsOWQ396VVWkU8I7xoI4Q3J9hHFdu+8U3Dl8nje+JT0bQa6M6Wxvf6J6hM1FQwJ07kK4P9oP22TBiEWPa4B60oV7TPDalwU9XCHgf/eX046XlE/evL8cPFAlYP2sK6jkgu/aXBnTwpKeocovRIp1zdqS9CSXeRsxezsMnrgJ4gyGRKbxXq+fU0KjUoEssGLPSW+aqk0pT8Ko1FvomXFx8BPXlr7LIuGXn6J8/Ghl6PTlcu9S029GJO9drOKQSI45Vm0lB8BRXOlYPXikZWqT4TQxjE2awlyPddOA5pAutRr7+sfJ84DtYRGzzfg3CuhMx88ut6jbt6u8I9bbGbPblujk0g8nvfz87G1QXkN6KdN3MmAmwEUeSTYWHV6V8LZvP1Ae7vMhY2arhmC7Rq76tfQDIgvgOWxnXKeaULVQNtthBOvN2XIE0JgFC39uya3w+YZF4EWJkxTvyKQxO2X9Qeg4ZnmQ7ILX6hQB6NgU66kIpbpNj/MiGC4IWqHnzwL4TLtsdk1jIXIq4vC3Hv7TiI/m7QVPim7NBqgjXLbfFO9j2ckCBcsm85VQ3KbZNAP7+2HFPk/CLVO6VLBGkOIjLVkT8VFcJ4KJ8SP6cAmtpnvxT/+czGsQirL50BlvOip3yBV7TkCrD/XKRmTYZHNqoFo70tB+tkFheLlZAjxt1d6TcM8H873LkDxLkKJouzryccYqDvknPlfJAXAU71qsHjzCN6rkxvgU0TnnoZGLvD+tf70ZZEibKmQFyM1IJV01Y9MheGJ0QMgeCRopNMoN/PhHZnBw9/B5no3lcw2KwEvNEttY+rk1smwvykghseTBKvnBlBWy7fPsxStQuu9kr7rKvAE/47ASGyysrn+CuojZQG4kocqEmObFjxXHPFndVN34uQtGGhujAKlr42Lwn8ahBD8eTKsUGEwD4nbYC7s3hJsz3+hdQu5nNu5d1d+YBodOq22Bg9J4+J26AR29roCKAMghFTiOL56oBnWLhbLH8e863Ldpzq6fE8a1RJ6MMOmFuuBC9DT1ReRUzJ6itXLW9KlhpaWqBVdK+KZjTagxeLrXnUxH2HYj58dieuLaA/DM2BUebAPhG7yQni+qeUz8j94Zkl/HoRP0PnVJaJjW/Xr8nDYIiJ1HWUgsKuWN9ftPwe3DAqcYi7o3gjyZE59i2J4C6dZSBaXiv5cIjanYfyqcuah+L0TjfsUqDonG2KPdZnIAHMUVjtWDJ/5w6aZkCmDph0pFNpq07gB0+nIFpE55A6B0ma6Y2LGuGDybW0WBXO0iOeDLDoFATkaMs2lLV4dbQmNWqlFJOcnLNlC2Gc9r49fAN0v3QuOSueHTx6sEL9l19CzUjzcUmdq1LhQVhNht2qasCh4r/mDhimVqk7vv2WAiKdATMyjYjyp7axoD/5xDMnjg1PDtWVonKd4GmrageUuv20tChzpxibofs2i3pxWbJ1M6SJs6Bew+dk5Zl4vnY0OqY3id9J3L/G3qUqBJtsPTNhyCDl8s8zLa4aoRtPl0EczfdswLHlGJBtdoUQ+5Lqs4VoKvIHno1+drA5qaYJn5Un3AsfJCmGncTOCmQlUOn74A1d4IBNImPXNqgwerCN/gso39f90A/zffnWcgO1FTfc9t7uG/oc7707fC21O3eJuoE+cue1OWnTDi3zkxVWWhHgkFHBURnWQMM9+Y2oMM0vsmknKTpnseqzjENI6k+HlyABzFuxKrB8/G7Yam+dua/fDcVyuVR5CmI1f6MbLFuqksaHE8JG8USW90lDtD2TPEGn7/TC1v2qrjqXBvPZFx0NwCTS5khf/Q6vCgNmNRZWkX7zgGD8Zn9U24bFU/w2dshbembAHK/tmMR1eH5IdMUmDUBhGdnqpzK/S8vVRI0/QjIjt6th2nSqtTZy4itq2TocO6JLdXOGcGyHRjali55yTIAnqsu2zXcbg/HkuP/1aduJA6hsz9z3butvUogCfTAHpX6KzRbdsmcirVR0moWQpnLrFNfmqFR8dt4nHWMlzy4AkbYcQcs6arH7c1nT4vKXq48hm409/afk0hY7rUYLJctl3z67Uefc+QPIrqClhUpiomlza8Vge/sl1D4rO0KJMHPno0gc8iauE/PHIRLNxxDLhKkW0ffuvFKg7xO75reV1yABzF1Y/Vg/f1kj0e0StnxrSetqaukDZv9gxpYHnvJomqEoZOxYj9btleeOX7NZ5rG8r9mIpKAYEHJKrjX1Pbss/fnbbFY9pyFy4TrMNPP3iNLrinNl10k03jUP24U7CJVs+Ib7UhFol9fTZvJ+DRa4Fs6WHOK3LLUNP4+OdjF++Gnj+ukxKGZO3o2Pt0whGujrOoJ4vjIIKiTFJLHOcDHy+ApbtOgOp0JIihzZcZctyUNl5eLT8MbpWYjIVOVkg8pPJYzYLQX1AtwBMUNJC4fPVv+PjRStC8TGKDEJd7YqormgaQQkEkTDiInEpjkEmJqcbHLdMH3F3a083FIiM12eJnOTZzWa/G3v0ylZV7TsC98XhSUeqsw+fLYNpGd5gVb3N5r8aQ/aa0QW6EjOxqGuO/4XN6V/G5qqBRh89cgGqDApl+VQKCTiLRpbD9be4KM9g2KaiIWsJkslG5YFYY/0wtID4Kmv2YFJsidS9jFYdEaryxbCc5AI7iasfqwaOsbMa0qWCtQTeSyBoq7dGeP66FsYv3gGgYQMtEEj22R5hIAMAfGzETxk0DZFg+v7dF5uted8hM2HP8HITzgpONx+bFyZn0i3s0gtyZ/DPpVT/udHQdTvAq0yP1ew/wOmKyYzZ0+ov1jU3p3PLoWJvDWowNSipQtpc/B0REsyEPEllGFqxid3QMjsYWSIYat3xfUFNXHM6o+TuDlsD4mSxzyLHd5ILmZ96219BGihQJaBOTO1NaWNxDv7E29UHkVKon6jHrrucwIjLSUKlIyKyFZW1zfLOJzEvX66QPaXPUsW4c9Gip1x/m45FhVAmT6tqW6R5cL5+TeQifjwrfi9q8+B3H8n2nmlBFQqCtOmgaIKnZZhOsWkMVhIww6U1K5fakRSmJUbVQVhjXKXBCGe0Sqzgk2vOIRvvJAXA0VjW+zVg9eDZi3zRNnZg71qEdq+oLSrtvWzcbcql7tEYBGHhPgr4sJyUteK0h5M0SICWFWyiw4HqvJpMFv302e2eOx0JWHV9juyYtZJe+VfeO5KvCeanSJorLMbmMTawbNJ6IVxQwtUVOb1y+jq4hfJ3sM1O7/HMiXXLFB4Kx2MirmVzjSFAff+xQwgrtVVX3hKA6NL6KBbLAjwL22saZ0WX+prqkYoHYVsS4ukCrTG0TOZXqVbs1m8fcty2kpILYXsSEqzb7su+/rI8Jaw9AZyS0pUwBWwaFEtpUY+KZ6Okv1oPCzAVTJYNlmh+32CZraDLokBFCTe39Gz6nE08+V5X5CD9pVCkvkPRnOFA81btZtNuORLLC9R7HKg5xHVdSqJ8cAEfxLsTqweMkIZO4vEkxwkSok2VYdUtIkArEyCJWloqfDIzNrZJpdZbqM8mzy9RhdW3aFuvYYIuJSITXmlQzTGNQbT5IyD2cYzVOeNo+SK/uYRonfu5q4KCTqiOCXKd6hQHJI35Lu1FLYNbmI8CPllFTVqfty/sinJ/KbY9rKaMagE7LVcyIyuSzuOvivFcbQL6s/o0obNYMXbVww4pl66AW8MncHTBk0mYolvsmmNK1nk0TyjqEX6cKrrazRB7Cje3e4+eVJDpbOJgJCiabyMFTF5QKFKQQMuCeMtC2hr3WNyewEo5VZ/Uc1k24Ti4mTgCfDjrmIS9FLJysPKpdVWhQIleiOkV7TggbZqSSwaN31zP1CwNKPJLuczhwNdfbGKs4xHVcSaF+cgAcxbsQqwfPRa/z9V/Xw6j5u+C2ItlhbIfErmhfLNwFfQRver5EJPdT5pZM8NvzZrkf0V2K2op0xpHaFbU6bdUa/DwGxGzXERpUBgx++qMjaRF+8tDIhbBox/GwiBXcxEElGO8yZlOwKLZFR4XNSueGEW0TFC6wHmlThwthEYkqV67+5ZlbYNHp9dJYUeEBiWL1iuWEzyUKAzzARknCl79fo5T7omcH4QUorYaqKrh55fhtTm4kgpTLPXCtK/b3ydydMGz6VoiEDbOY8aYjYdsxEnkIoQ9//Q1KkwiyEc6YLhWsFbR6eV+UibM9ycJrdTbyJXpPhAuX/3LeZHN1EpRPw40TWT3Hgvhou/5JqR69U/mYdImfYj0nemQ5GY4eN79krGLzDlCtA+HnRXk/cTPDSY9+Ccuu9yJWcYjruJJC/eQAOIp3IVYPHocTmHCmJokdCiBVPyBDJm2CD2dtB1u5LE4M+vm52sHVpuNVFRbZ722ho03cYW8e2AI4jlKn1+unPxt9YZ17lGuflPkQSUmRkNbxQwrSjZ/WxtZdqddPa+HLRYmx59wF7sM2laBlWf9EMHKQouN312fDpDAQJMnVjQPs48nPAxJi2waFSmbhupEzWuOSuWDaxoA1sHiMS9AmhCSglJwfcqPLM8atnJEYhhlghHFUK5QNvutkD1eQ9fnO1C3w3vSADSwWV2c7OuWg61VBua1MI43HdiOP/aqO08Mh9MoMdWzUZVzu6/VWl97xfF47B6tdBcnARmZExPHl33SsATXisvtarqBGtACpIXgFcQy4g6hfyUrXAcYqDnEdV1KonxwAR/EuxOrB45I+s16qD4UEbUw+RZNTEhpGYPCi0v6kIKJGXDZAUXpT4SL2XPaIAuNIZJf4GKZvPOQFHljwpXj0z0uAJAcsJsc201zEz+lHWYR38HqfzNkBgyZshLgcGWDGS2YymG4MKlkqwkeqTBds5sXVKiJh00lZabQ1RntjU1GdTHAWN7KokU3tt3w4a5t3pE9qEqEZPXPbptMPMtXo1qQY4BE/KQbIVAbK9ZsMpy9cgRcaF/VUS7BQBpDmR4orLqYRftcGrxMhF5/O3QmjF+zy5hKuOQpBp2h891fOB289UN56uIRzpwtUrpdcf1rU6uWd+TGbwM3Yrd0DNszc1IaCbvy7q1GLSNKqWCBrMCOpOrK3XrTrtCL9RtH0ZMYkfOrEARlyXzloXTXU8vronxe9jDsWlZmGzTJy+BBloy9f/QuKxp8wjWpfFRoUzwXcgMWvaZHNeHidWMUhruNKCvWTA+Ao3oVYPXj8GE0lCE7T1LHtsQ7X3JThqgjSoDoGFpdT9SPe48e1gNnR+sVzwuj2asF619szf9vRoE4osrsPn74YtJ+MRGDHx9P9hzXw9ZK90KhELvisXVXpUCnTxGXZXOdE9WWbE57B+KpDdahVJIev5rlrXzjtUOfk7IYB3guNixnHpCJfcr1cW7kqVWffLt0Dr45fCyQBeOj0Bageb4Zgo0Ri2syQ2ggG/E1K5g4+d2Kmh2cM0USFNmziEaxoTGFcxDArcDwqkrywf3y+ReMVP92IqheuEl+0FtS3akwmp0u63tYxTpwrQR04aZIk/FyJfdg2D4jwe1cmX2YoF2/1rLL39bP+19M1szYfhnajlganZNKkJwgVSui1rVkoZCn2nTgHtf830/vb5BfqQvE8GX0tFXe+XNm7CWTNkAa4BjxPvohZYV8dOlwUqzjEYUhJpmpyABzFWxGrB88lk/XIJ4tgwfZj8MRtt0KfOxNn5rgsD7eMpWXiTHeUdTEV1TEuMerDZfaL/fOAaXWfprD/1HnP7hbL/NcaevJUkSo2WSTXjLlubDJrWs5MV4nB286XdHLDySRTX/XenOk5oWEwiFlgUxk6ZTMMm7ENRHksguRkSJMScEMWDgyAVA4QR4qwhD9Onoc6QwI/fjbHkSoMNs2N60IjVKNsfCAjHq3uOXYuGBxP6VoXWo9YCCfPXYYh95eD1lUSMlRB+JClkoZpjU2fi6oTmAH+ceUfEA65kvokKBD92/ZkgOoTeYj+fU+FvPDuQwmkWvq7LZTH7/unYv8pnvsYZq8xi83vpeyI3bTmIgYVA7Cag2d4l/3epTaUzpvZ1MS/7nPOV8DJmzTpSYte5uBo4xRns8D89ITUPGR/w7bQyRD7RVIckuOiXWIVh0R7HtFoPzkAjsaqxrcZqwfPRuqFpnn3B/MBsX5cCoovAX+hy4KCrt+ucvpR5BkZTuTBH30kBHSofSv0sjgit71NIh4aveLv+2iBdzntzG3bMtWzCVBsssSmfuhzTqBY068pZEqXOiRjT3+zbU+sRz8UkbDprDRgKhw/ewlkx46y8amc6MjYJBIOdWKW5tjZi9B46BxvOCqbVD7WH1fug67frgYVPp5nB28vezMU7zXJI9981KYStGDYZU5aXdWnCTw4YpEnp/dS02LwXMOiwS5VBFK/99d03Ymzl6DigKletR861/IywBPWHoT7KuWDt1vbwxVk/dBGhj5TOeSpxsgJelhHlUHefPAMNHs3cE91yhlkj466wv3uKm1amuDnNQdPhwOnLgCpPZD+K8JU0No5baqU1m1Rxbjuv3vEvpFtK3vkPnomTXA2546ukwv4exCnZNKk1xkhufBndMvHHecmvVAHSuTJBCpiMSWhXJ89v7cvVnGI3/Fdy+uSA+Aorn6sHjzEpsX1mAB//w2e2DYyrFWF9CpfaV4cOtcvkqga/xGUYS51eq2yPlVSTkQCUo3D723hupqIx0MDjEc/W+w1t3lgc18/UKqx2EjCPf/1Sk/6RuWs5zLPNftOwl3D53uXkA0sMaLRchjVG8LJkBJuV3U6YDtW7mAmBn+qNsgyVDTOILmwpqVyw0iLEwfdGENxzvXg4pWrcPuwed4lstMOsa1J6w5Cpy+XQ6oUN3gyYXytuaIE4f1IGmvgPWXgUSaNRVhuxC5uHtAc0GBj7tajnnwWBlZUbDDmtvfEpl7IcfxT1QEzwDM2HYZHqheAN+5N0PC2aUusIzrfIU66S6OEYN/UJodnYN2n68ZBd4nhBCcZ6SBPpkSAajwN3poFeOqC2cTHahaCWv+d7vEMZBbepjnR58V7TYSLV/7ynMFuyXojoHkQFnT1RHfP5BK6Atw9Dz8xGQDRve7SsAh0a1o8pLGQE8O+TSHzjal9LXfo72ZNqFwwG9D3XJQ8oySSSk7R1wA0F8UqDon0uGPRXnIAHMVVjuWDR1q3wx6u6AVbqkJ4qNfvKg2P1wrFQ+E1HLgvk4UxYYjFflWOb3RcHGmpH9H7HX+snh6zHKLBpP98wS7o+8t6D1aB8ApZeXL00nhL3AKe01A4ZcuhM9D0nUB2i47ZCJdaKHt64CRDP/1EQk8Y++UnErZED9LR5AYm2Nb9Hy2AZbtPROSk4NS5y1C+f8AValynml4gq7K2la2fDIJC9TiZibCbnBTHgz3Sc6a5vjRuNaDChxjkq0xk/Nxbm2tEzVRUgdDBpWzapDqcnIp/69GyBHSsa3/8yxU78HpVAG2L6yblFNdx0D3FbP2tOW4K2lnPeLEexDFjDJe14UYMuTKmhUc+DWzYN/ZvDjemcc8ou/T9T6zLs7Y4/iK5bgKEf6kKnTbKNk2677TL2ly68pdnW47liyeq/X97ZwEvVbX98UUjKJ1yaaRD6tIlStnYgoGiIqEif0SlkVBREMQAfT6VeCaKIiWIlDRIdyOXlhJBkPv/rHNZw76HMzOnZ+bOb38+ft7jzs7vXjNnnb1XECdskZCi1+fMSr+90jzQnT49spVx7NT1Uw+xM79ItoEC7CF9PwUvlKerusTA1bTO3lCtY+ToIZ+Lw8dj9YrTwDuvnFYFw5gqnetlJy1+0JbpPS1w7deiUiHXdkGf+50jZLzw5e9Br62dDBws9qPa5wNjF9PSXccdnRBJf0anW0N+2kgcr9WOA45+7dKXk4xy3Kca2smsZ7XYiOrT7iYOmU2HT58nIwcWq3vHJ9Mc95ftLj9sX5M4nNyD45Zo3QQLpK+OoZpQsOlCrmyZAx+rjqhiuymxfvVXnfIArFU8N33zbH2S0IIcO5h5SQmWRtzquq3UV2Omslyt3PMndW5aml5qZT8BCY+/cNvRwE0M/zvYC3iouYqiyHWM7Dn572ZS33I9SX/LJ9t8wm22qMlv2N540fZjFCwihdk+5TCAzYVYJp8ev5KCpXo222darqeaufA6KxbOQdOeDx6TXp5ZRiYH4aIeWeEo3x259RIH6ErX56CfnrsyPwkBqn/ZtzKWlbp+6iFW5hUNdaEAe7gLfgpewzd+IbZ3HXB7RXq8QXCnI/W6jZ1bjEqo3OjyNm02T73RdbjRdZFb26A+APmUj09Ne3+3nvSKlRvjybWuxBw26tNqNIRQ8zI63ZKwdlbjqhqN41aaTtXUQJ8yNtj6JMaxGvJLvZJ3EqReHVOSDPBpPJ/cs/kBl3AZFLnOpqRTQR0qVdMbsScOZv6ivwKVmwT9SZHYZPduU4GealzKDZEN24fETOW0sHwCvOHAKXIjJa961cyTeOOeKvRAbfOKJ7dp/vavtOPIX9oagt0csVkL215zGf9kIjW6Ib/hmu3G2pWEHGr8ZqcZJsWumE1lsmfJENLOPOwGxkEF9feFl6t/cdQj6PjZCpq96RA9lFiUhrWtmupjca50IwW8OEgOv7cq3VerKIkNvz6MoN4EyonZmpnt9lMPMTOfaKoTVwrw9u3b6a233qIlS5bQ+vXrqXz58tr/quXxxx+nzz777Ko9mj59OrVq1crS3vkpeGJTG8qzVPU4DhVjMtT1oJxKdW1Whv6vZWp7qmBw5HRaHlqq562Tq0Oj8dQHIF+/sw3y4J/cicOrH89MzFHVZrBjI2dKjFEmKjlhtupVb8TuhzUHiBXqrJnSa9evdn+YjWyVw31xjDLmqSYfbslJy5HzNYezni3LUbmC11HHEDGv9XMOZV+q2iVKuDZJ/6t/AOqdYCSOLJtksCKeno//iEg9GdTHLw3H0+7n8oLACiqfAPN39ZXW5emZJubNFYzG1l9b24mYINy4/2BKpxl/iHBmXqHYPfHpcs0umuOks88Fh9Rb/EpzYntuu0XC5/W7rSJlypieOONg4ZxZtX5Rriagmrnxp+Fuv7pMXEU/rUuittWL0IgHbkzVoYTpDBdJwsw+iHmhHEKpqdHViCV6J1j1JsnMOFbr+KmHWJ1bpOvHlQI8ZcoU6tq1K9WpU4e2bt1Kly5dMlSAFyxYQBMnTky1NxUqVKCcOa2FpPFT8O4cs5DW7D+pOZawfZxRUWO9hsp6E8pBpM2oBbQx6VRQGzyjcUWhlgdpqtTN/VpQzmz2HA+MxlIfgBysnk+wRs7eSvprKDe+eGZS1YY6Tbc6B6NoH6JgW7VlNBpbjdjhJE2nGovZrB2jUQav2RsPBRTULa+1dqRkyHrVBB01iuXWbDhDneCrnFTTjh+6NqCqCbkCH6tX/Bx/OmumDCSJM/RXtHKSKS+rq/b+SW3fT4m/0kTtAAAgAElEQVRUIsozyzEH0b942VyjVWX3zIRCyZ3qI8AnwGZulczIsXpCzvWN0tKG60dOzrleqBd48YcIpiQ7uYESJ2CZ6zNNStErrSuEm3rIzyUsFv8+JvPp9vTNYe1aHQ0Y443VmzBeSsMy+WhCxzpBVyWOtEbh/MYv3k19p4T24zCLS3257tKsDMkLW4cGJaj/7VcijaiRlpzEHjY7Lz/1ELNzipZ6caUAs8KbPn3Kmzqf9K5YscJQATb6u50N81Pw5CQwlGmCGpj7x64NqUqCsUIfKkSQPLytnAq1fX8Rrdp7ImBLKN707Ji2TTnxssPYqI1qw7zuj5OupXPVj7Vu/0m6fUxKFIFgKajlYWznxEs/npqJ6pPHa9FN5QuSxO51o381prCZuLjB9ks90dRHSwjW5pfNh+iJT1My+O0c2kY7BRVnsVBOhlZlRk6D7q5ehBqXzaddN+fImpHWDmgZtis16Yg+rbZRvOvxS/Zop3n6ME1iyzri/mrUtkYCqcH4JZFNqLHCTtRBBfX7zSfAnCnLDUdV/amdRMqwMlWxneY2oZJEhPNz0MdhLlvQfPIDUaZk3m6EKhPHuh63lNXC5r3L8bB9iv1shX+01D125jzVvJy9jed0U/kC9EmQRET8uYSiNEqeEkhukz87/dLDWaZOec5xbF9+uZXwa/rwhupBRigzHbd4+6mHuDVnv/qJKwVYhZrWFODH/7uMft1yhNrXLUaD7zKONqCGEgqVNEGCxBvF/2z05i+07/jf1P/2itQhhK2xyrrDf5fR3C1HAuGUxOEp37VZtBMvt8uNg2ZpiQU4/zo7qrAiYjZznZW5qCmo5/5fUyqpS0GtetVzxq/mFYKHpzM7Lnsas8fx++1qaJ7GnAaZy6Sn6lD90vaywMnYqtI1sWMdamAzq5ydFL7qCaqEqxMTArNpt80w7PP9OpqwZK/G7tYqhbTMcGblUN1PeQGRMSXOrapMi424hDtjkxLVKVQiZKhmO6IYcpIOPo3lEi67o5l1m61z6+gF2q0J3yLxCfDpcxeJ7YHvql7EbBeG9dSXb7vyKi9E3D5Ukoj6w+bQAY7Va5D5i9uGS/YTaqGSwZLrhDt5NAtMDdN1+vxF+u+i3VS/dF6a9FRds13EVb1T5y4EsuXxwltVKkQfPlIzKINgpkjcYPScbTTi561hHenMABZnu0frFadBd1YOOFoOubsytatTPFUXkgpdEqqY6d9uHSjAwclBATawAf7yyy8pS5YsdPbsWapSpQr17duX7rrrLsvy56fgydVcqKD1Zn/4Q6UJldimVrynX/hiNX3/+4FARqn35m6n4TO3aDaYM7s3tsw1XAOZI//wrNpzgr5dtZ9aVy5EH7QP/iMZrk+jz/WZs/RZm06fuxA0G5id8biN/HDy6SGfEjV/e57WlVlns3DjihOUnE6Gq2/0uXg5h4vPqbZVg9tLwhSxt3ygVlF6497Uzit25sVt+GHHDz1OTX1frQTqZ/H6M5jzlFylqo5si3cco4c+Sh1lQn0JVU/Z5dRSnMM2HjhFbUZ7k8EwFDv1tobjABsl8rDDXnVO5facaINNUKwUcR7iNqESl4jJVTDnwXDp3kPNadCPG+mTRbu0KvwSyhn/nJb7PvyNlu/+kzo1KU1snvHlin1aLHczmTadjh2L7dUTVJ5/uBjrw6ZtorHzdxraCksElprFcxPHvXdS1OyCfPhyxYSpBrWqnFpOxGeHfRHYXMLL4qce4uU6vOgbCrBOAR41ahRlzJiRKlWqRCdOnKAPPviAZs2aRV9//TXde++9Iffg1KlTxP9JSUpKosTERNq3bx8lJCR4sX+BPl/86neavOoP4gxU77WrYTiWPk1wMNtbSfFbp2Qe+vKZeqn6ktNVK2+u/aesp88W7yFxBpKHiJsne+okJQ0vO5Ws2HPctWxWeqj6iBO1S+RJVcVsTFIrgqHaFBfPm40e/iglZqiZMF5mxhHlwUmazmC2r6HGV1PYruxzM+W9NkvA69/Nh4Qau5lt86w6SAZzTJOkKGpMUqO4zarduGpnLdelEt/WSHk2s39O66h2i3wSySWUva3Z8fRKi500vxySzUxWx3B+Cmroqx1DUsxtzBZ5geJbA06ekimDfec3GVNNT3/o9DnimwM20Rmpc9gyO8e0Xk9NOsNr5ZTU/DwKVmTPKhfJQVO7pQ6XJs+iBmXy0sSOzk7cJZ43n0jzC3u1gSkxx798ui7VKZU31fTEzNBsOFEnewoFODg9KMA6BViPiu2G69evrym2GzduDCmHAwYMoIEDB15Vxw8FuPd362ji0r3UrFx++m+HRMN5ztt6hDi4PpdQYZ8kHFZC7mtoYa/UCR7MJtxQJzBi1hYa/ct2qpqQk37o2pAC6ZRDKOtOvvBqtrulO48Tr1ufZctJ/9JW9Sb/7IlEzcxCLeFMJOzMQTVBYc/l57/4nbJnzkAbBlmLUBJsbAnz5CRN5+CpG+njhbvI6AUq2LjqiScrFgWvy0rl+83QzD3CJXexwlEiXXDmvK43ldFuIqykWQ5mAvTWzC00Zm5q202jeMgSdkkfaYPDsc3fesWEKVTWOSvrtVpXzJX4JolvTrhMuhy/22pfan3VOZX/bufGgk1Q+kxZT5ws4oWbjR19uW+9z4F+3uFSWodaJ0cC4Zem+2omkFvxy8V8jX+j9v15Nqwpm5N9SAtt9bIULlPh/5btpVcmr9Midqzse0sqBPx3/rx5+QL0nxB2xGa4qaYWr91ZmZq+9avWzCgjoRxYeXEzqZ8rFODguwcFOIwCzOiGDx9OL730kmYScc011wSlGckTYFE6Qp2qGnnaGy1m1oaDgWDsWwa3TnXKUebVaZpn+thHalJLkwksRKHmE8t5PZtpsVfVh72ZHxcrde4Ys5DWXo6IsWTHMVq2+zi54a1tNAd98HO1jp1wYOHWKR7jL7curwXLHzpts2Z7zDbIbpSAuYqDl5OXv11LXyzfR0ZOJ8HmuP3wabp5REqWO77ezpIxA9UdNkf795QuDTRzDzeKGqGCHUbHzd8ZeDEz07/e01vayMNPPUlST6rEZjhYrOWeX6+hr1fuD1x9f7V8H7307VrDh7aZedqtI9e4vHccO5WLUUp0O/2Lcyq35cyJ7NzoRZEEJMFSekuGLjedK52s46nPVxCfSrOpz86jZzRzCK9+r5zMM5raii8Ezyncy7r6ndfflIWKEGF1vWJOUb1YLupza8XAbcWqvrcQH1aohSN9fDhvB9Uolosmd76S/MbqmGbqQwEOTgkKsAkF+M0336RevXqFVYD1mP0UPDllDeU9/O3K/dTj6/Be72qmHdXLOVUc4Q61qVm5Ama+f1qaV74e4ixHv/drQZIcIlTINlMdB6kkNnX8EGHnKreC+RsNx9dcbArBNl/31Ext5vLbjqMBE4WNg1pStswZnSxLa6sm1mAHJXYMsnLSGm4CYi8nWcrC1Tf6XI20YPYaV/XMZ9vY43/9Qw9cztK2uu8tlFv3ALEzL26jnjTfXu164hNZK5nv1ExgPVteyY4mCqw+nbHIh5gMDZ22SVO69WMGTpATctKUrg0p4J2eLzv94tLLjRlmcjuTWCKP9uLIZWq3hlS5iLUQkKG+K/yZhHszMyerdcKlH/faB8HqfCVeLMep3XTwtJZwhSNCdGt+g9Wu4qY+O/9yWE8u4ZIyqRFI9A6lZnxnzEIVuSpb8Fp6qWX5QAjH7UPaEEc8Ust/F+2igT9u1F4C+WXQy+KnHuLlOrzoGwqwCROIunXrasqvPmlGuA3xU/De/3U7vTljC/GXb1Z347zoEpYpXJB11VNd8przWtXMXFauRcXmjn8D+Meg0Ztzib3c7aRDDcecP1dTX/JJ886jfwVNnWqmv1B1Qnmcu5lmU+Zw7we/0Yo9fxKH2uEf9qlrk8I6gVhZo5xQOknTqfeGNjP+wZPnAie+rHCxEtDzm7V0XRYOUdbCdlIO/diqXTYroXzaZsX+T04X9bE95UGqD7Yv2dwkTvPzX6ymKYpDqMxPTiUlZNrwmZvpvbk7wma5MsPWSh05veffka2HzmhNQ0WMsdK3OKdyG7ds1o3GVxVKfeIDrq9PRW1lDV7UDZiEVS1MHFpx7/GzxP4LTzQMntHTi3nEUp/ii8JzDpeUiW9iyvedYRhTW0x+2tUpRkPuNo6eZJaL6l/wfPMbtBucYAk2pq1Los4TV1HmDOmJo97YTTpkZm5+6iFm5hNNdeJKAWYldtq0aRr/9957j3bs2EEjRozQ/t2kSRNNyeXwaA899BCVLl2a/vzzT80Jbu7cufTtt9/S3XffbWnv/BQ8eaMMpbiMm79DuzIvZSLmoWSE4vSc7eumhHAxykRmBojq4c+OP/wgPHfBXdtOdR5q6kvO2nTo1HkyCkVjZu7h6oSKi/z96j/ohS9/p2uzZNQe+G4UVQHjk21m+1SjktT71opudK+diHIKXzV0l9WOJVtgl2alST0lDdUPn/hyJAQu33WuT3M3H9bsxt1OYMI2xXx9yoW/KxzSL5TdvH7Ocl39YO2i9Po9VyJTsG29ka05O22x8xZ7+LPZithY66/n9bGx+/2wXgvX5kX4vlD7IKYcnDqcvzdcFvZqRgm5s1kVg6vqS8Yz/oDjQ7vhQGY0qR5frQkZ+cWMv4TjxVrooNc3a7XID3x7wLJy7K9/6M17qpJf2f8sTDVqqoozME9IHEdDTU4co/UJg+T72LFhSepzm7PfULnpzHlNJu37/saMzVQ6f3aaYxBfWHVId/OGy4iBn3pI1AiIyYnElQK8e/duKlnS+K2aldyqVatShw4daOXKlXTkyBHKnDkz1apVi15++WVq2dK6AuOn4H25fG/YmKYjf95Koy6HgPqxW8OQIiIPbvV6yWwiDX3HqknFrO6NqcXIFFtPJ7FmQ01eToDYk5rtGN2KZWo05u3vLiROtmFkzjFhyR7q8/16YmVi6avuxDtWr3cX7zhKu4+dpWDhnkz+BqSqluplxWaWPtVOmR8EZoqapZC9pict26udlHrhJCKh3vj0hcN8hYsjqs5fTnDZfIIzjUmRk3lRdOXvojDfXyuB3ry3GkmUDVaGVTZqKuXlvW+mQVM3ai8j4UI8mWFrpY6YwLCTHr+kcuH55L8ui5VuDOuKcypfB+8Y2sZxf8E6CKfgcrpvdob0m22w+Ups6qbl8hNH/zh/8RKNebg63Vb1es8YxXrHkrGQ16H/LhmtTW6l9HHy5WU93CmyGV5qAiC+IeJEMmxK9FWn1JGUuC/VLGPGC42ofKEcZoawVcdPPcTWBCPYKK4UYL85+yl4U37/Q4sIEOq0UewPw+VOZ05yLacqB+qX1koKR/V6m53nnhm/UtsKrwL8qydAszYeIrZdtuK0Z0VOJAOf0Sns2Hk7aNh0cyfuZsdUr9qnrz9If1/4l9zIAifjs5d7k+Ep3sv8smIlS5b0IVfd6u1BuPWpJ7OcHYlDF63ee4KeaVyKXmnjLNWsfmw5DZK/W1GExGv85goF6OPHage6ljBmetvNKw6BKfXFdnHkA9Xo7upXbMYPnPib6l9OfMEmIG/O3KI5ikpQ/XD83PpcQkap/UlcZqdjyMtitswZaKNLUUuM5iQOwfVK5aX/PX11aCt9Yh6n63LafuCPG7TkF2zLv3RXit21nUx5TucRS+3V77AZcxFJYMIJcNisT0qwbG12WCzYdoQe+U9KlCXxLwj2cq2GBTSKIGRn/GBt/NRD3Jy3H31BAfaQsp+Cp0Zu4NMVI5uicCcjKgp5EFYsnIOmPZ8SO3H74TPEp3tcrKQAVW2H+W2dPWC5BEsf7HRL5MeOMzUt3H5U686rlJOhbMiEoYR/c7oubi9ey/xDzgoSly+erkt1dXEm7Y6l7pVdZnbSP6tpnjlrXq9v19LRM/94Yroipz7CKFwcUZWlxMjWK1cSHk3/MOZrUI4RzN7e45+sQ5Ukc58utJj6AsARI0bN2U5r9p2gbjeVoR4tytndTsvtxJFHbRgqZKKVAcQ5NZhdpJW+QtUVh8Ibi+ai77tc7WGvOsm+0trdlys7axCbZA7Ht/ngaa2LrzvVI31ccTt9p9U2csvE6zPzoi2HESXyZqNfezYLYGn21q/EKeD73FqBOjYq5QjXqr1/Utv3f9P6YNnj2OYPJRajYW2NbYvFjvnNe6vS/bWKOho7VGM/9RDPFuFRx1CAPQLL3fopeOrbp6SS1S8tEPLFRIgriRjBJ8p8AsQKtepB/9vLN9H1JsMYsXJTrm9KTFe2neQQWVzcerDq1ynBzVl535iUkpjErVBO+rFUcwt9xINgypITkZO89hz6jH+4uRilYXYyRpUBMzWzESvJTmS8VJFCHq9NzcqbixTC7SW0EY/LUUO42FXCQ61fzEikjhUHmLdnbaF3f0kd75f7kUxuettNcSrkBy/H5+YHLhcjxzJJsvF62ypa5iq3HsxWZEFNN8zt2HE12Au1lX65rlxDq9nyrPZhpv6YX7bRW7O2Bo3vLKd+biZYMTOvYHVEplS7a69ux5zMM5rath61QHOU5WJGgZy+LomenbiKMqZPRxzaU6Iy1B06hw6eOmdKiQ63fjXxTb5rM2sv8KFMK8QkyGs59FMPCcco2j6HAuzhjvgpeCv3HKd7PlisrWZNENvNZ8avoJkbDoXNnMN9LN99nO77MKU/iWOoZuuyGsZInBYkvBJ7969zyTFMv4USj5GjXSSdPKd9PP35RlShsPt2VhL+qmWlgjT2kVqppmInHm44cRQnpUwZ0tGFf5O16hsGtqTsWZyHWJOx5XTlpVblqHNTa2k6T569QNUGpWRAsnqKJeYBbBvLMTK5hEp5G45VsM8lY5N8ro/oEKpfyfh2Q4Fr6ecXr0RbEeX9vYdraCm/paihB8c9WosevBzajSNb5MiaKdVQ8lDvfnNZ+vS3XfTn2QumHu52ORi1m7h0D/X+bn3gIzfNFTp+tpxmbzrsatxqozXIS0ew+NhiP+pVFBqr+/HunG309s9btYgAbJPOZX7PZlQsr3PHQ6tziZX6d45ZSGv2n9Sma8YEbP0fJ+m2dxdq9VWnTjmFNQpjaZXF/j/PUsM35qZq1ve2ivRkkGgedqLlWJ0T1/dTD7Ezv0i2gQLsIX0/BW/DgZN06+iULzhn0iqc8+og8/KFM5N+UQ0XxdeIfKWjpnG1ahcoShWnED165jxJUgwv8Esq3iwZ02sOJV4+UNTsP3zFrZauk1ZpYcruuvF6eufBKw5TTtYsAdSlDzcjTEifEmkiXIB5o3U4ce6QU1RJwsCnNJtfa+V6tABx9JL5W0k6oIY6kvidakbATzvUpqZKfOy5Ww5Th/8u14bihyzH4eYsdBwXWm+mJJEk+Nr0qxX7PLVdDyaD4skun+fOlolW92vhRGQDbeW2xErmPTsDi/OphJTT9yE3HCPur0Zta3ibot7M/PllT8zCpL6kAzfTPh7riJM2r/39djWoTZUrL51GPE6du0BVB6S8mKshPCU5i/7F1Q5TNUqStH/ngRvprupFDLsTXxWjwxM74wdr46ce4ua8/egLCrCHlP0UvJ1HztBNb6fY5/7SowmVyn/tVSvTh2QKtXQ2W+DYiaxAyhu2GTOLYH2qP1hcJ5h9nhvbIfZeal9uebLr5yc2njWL59bMLNTiZoxJ6Vcieci/zYS0s8pUHCDtRGBQzWSsZvuS60gx73ASizjUmvXyYSUhy9cr9mnxiVXFUD31/qZTPapVIk9geDUboGSe09shSuWXvllDX63YT5zNccnOFGcoN+27zcjB1LUHqOuk1YGq4WKGm+lT6kj611DJeqz0F6yuPvGOWk9No/vRo7W0zHuRLp8s3KVF/VALv/hlzZQh0lOL2vEfHLc48B35+NFadLOJfaw+aJZ2q8ImRg8mFiO930HzCs5kQX0RFnBqHH09TLmp9PJZyGP6qYdErcAEmRgUYA93zE/BUyMt/PRcQ6p0/dWZm+SK1WyWoVtGzKNth8/Q/7UoS11vuoHmbDpET362QiO2a5ixo10wnE98upw4Jq8UN3KvBxvr00W7aMCPqR8obmVi048Zyt7w/g8Xa9m0wmUqsiKCcgUvbYJ5ulvpU19XnHLspOnUx3zmmJhmi8SJZbvTS8lkKUGF2TG4Hp+uvvTN2kATKzZ4P61NIj7J5NsFtiXkwkld+Fqdi97URj0Rb3RDPlqw7SgFi8IitqB5s2fWYsEa9WdlnXbqSvIWaRtMWbfTtzinupm50Ggeskccym3zayl7JEUNt+f3y0UwZpKgSD7nm4/tQ1p7mhzBzv5FUxu5zeQ5hVIy1TmL2QQnEerVqjypkRisJHYKxUFN0cz1QmVRlOeU19ng/NRDoklGzMwFCrAZSjbr+Cl4oU6hZPpNh8/V4saa9XgVZyGJYRrIXpMxvebAZqXIqaK0ua9mAg2/r5qVLkzX/WLZXnp58rpU9XcObUPpdekoTXcYoqI4DRmZdIitoZk4lWbnoj8tuvPG62mUS+YVMgcZIyH3NbSwl7U0nepLEjtP6VOAhlqn6tnN9UJ5UJvlZVRPnSN/bvb7wHVVkwZZn+r8suClZlQ0zxXbTTWropj/3Fa1MI15uMZVU9MrQlwhmDmTk/WHaqve8nA9N80V5LvycJ1iNNRh1q1Qawj1op508m+qNyzlZcWtFM9O9+Kr5fu0rGFScmTl7IfW4847nUcstVcdWc2+yHCCH46tzTb6bPKQOrFTfeJbPKdFzLikn1DO4uKYx/4cW15r7cnziefhpx7ilJ/f7aEAe0jcT8E7f/FfKtdnhraaYJ7z4ojGYVlYuQhX1PiUXz5TjySzmR0HNrGVlTG9iO8qfX+3ej91/zIligAXNx159Mz+t2wvcWxYThTAZhZS1MxmE56sQw1vyBcOt6nPJy3dS3ySJsXN02XpU67B7aTpdJL9rs2oBYGoHTwXPqXh0xq3ixquiPsedGclerReCVPDGNnBq/3pbTdVUyIZgJ1i2DlGX2ZuOBiIkS2feXVzEWyx6gk+16mWkJOmdA2dNMcUOCLilLSr952gKkVyenq9v2j7UWr38VJtWnpTgm2HTtMtlxPxRIujmf73yusoGWb3K5rriUM3z3Fy5/pUo1h45VXSi0tYSvVlKNitqVUGEg5R2oUyZeHxF20/RgWuy0INyuSzdFhgZV5+6iFW5hUNdaEAe7gLfgoeP2jL9J6uOc6Me6QmtahU6KqViZe9Ga9ZbizpleUHWU4qOMTLij63WCKnt119pXV5esZkljBLA/E19OWQN9LOznzNjikJSLJnzkAblOD+nBaXnZq4/N7vFsqVLbPZLkPWk6gCUsnK6aXZCagRQKym6ZRTTDu2o/r4vG44phitWU32wZ+LTaAZPqo3+bJXm1OBHFlJPTU1euDVHzaHDlyORsJj6NOxyrhqlBX+G58M8U2LUUxvM3O1U0e1Web2ZpLm2BnHyzacTph9Drjo5TfUZ17OKVTfertrfYSRSM0rmscVB2Oeo9mTfMmWymZZa/q30MIMSljCYH4zVhlIiD1u5+XBi5V5+amHWJlXNNSFAuzhLvgteKEUXDvOH3KVmC5dykkKK8B9p2wgOycU+qt7M7Eb7W7N3M2HqcOnKZ73XLxypuK+xWaSGbGZhSgrklCgWJ5sNP+lK4HX7a5J2ol9o/yb0/Fy1iE3y95jZ6nx8JRwPlYy/nH993/dTm/O2EJ2HuL3j11MfAIpxeyDzeraVY9wbqvPyhaqvx1HzlDzy86mEn9ZTYG6zcB289bRC2jDgZSYpVyCvYCqJ1Jcz8sXt2BrVM05uI4+c5ZV1pGoHyoizq9bDtPjl6Ny8F5lypA+ElNMNaYkMZI/eu0UFfEFuzABiWmv/QZ3b0w3FLwubK+cZvqhj5Zo9ThU6IGTfxP7xXCxEtc+1ECSEp3reG3bG3bBlyv4rYeYnVc01IMC7OEu+C14tQbP1kKMGZ1oqQb/EzvW0a5cwpXth0/TzSPma9Xm9GhCrFgO/mkT2XGM0Z9ccrYvp163web/2/aj9PDlK1Cu46Ydo35MdaxNg1rRNZlTPLfliu5WE0lHwu2D+rnefvXLp+tSHZeywMk4qqyYdTCRtmrms8mdr87CFWqt7T9eGsjcpz2k+rcgK050ZjnyyyA7q0gcZSsnzUbXphJ1QE6W9PNQHXb4s0lP1aH6pa/+/rEXOc8rOSW8M3kR4SMcI/3pOEdJ4GgJsVTUiDj6JDFsA8q2oEYOcpFao2pXznPgDJYTOqYOqRipuUXruBIxhednNiup6qzKL9ccc1kyt0mse6frffSTZYEMnW5mAHUyL7/1ECdz9bstFGAPifsteGJ/1P/2itShQclUKzt25jzVHDxb+5tZmylVEeLc9BziavjMLVSu4HU0s3tjS+RmbzxEHT9PiSDB5bvO9am6CbstS4Ncrqxec/KfqhfLRd9ZVMbMjrt675909+X0l6r9pxcOcDynhduOUvv/pNg3WvnxN7seqVd1wEw6de4iDb+3Kt1nIU1n3+/XE5tBNCmbnzjHvZWiRgoJpkxa6S9U3TpDZ9OhU+e1KlbCYZ38+wJVG5g60YdRbGB17Be+WE3f/34g8Cd+mSxtEKaQK9Qa/LOWQcpruQ3GRo3/zXX4doFvGWKpHDjxN9W/HJVDn1EtmM1+JNenf2H3Oi5sJNfq1ti9v1tHE5fu1boze3rL5oEc95dffPmlN3f2TPTwRym/perhhZM5dp64kqatO6h10bRcfvq0g7XfQCdjB2vrtx7ixRq86hMKsFdkI+B92XLkfNpy6DQZZfBSwzFZudYWRYEdhY6d+YdGzdmmObH82M2aY4xqV8rIvcjwJVup2mny37w8UTGKAKC+bJg9bTcrhit2H6d7L2fo4zZeOUlJCDwrIcJ4PqLsiae12XVxvU7jV9KMDSkPD69PT1RbPSun3GqsT1bwWdEXs4+yBa+lWd2vZIeTtYszabsH6m0AACAASURBVEA+B7YkTmBiVFRHwGbl8mvpk/0s+mD+99ZM0FJix1JRHVD1L/vj5u+godM2R+R0PRhD/W9j2xpFaMT9N8YSct/nqjpVW8lKetNbv9LOo39pDrblCl1LT3yacijjVpQgyQzKfbatXoRGPBD5fYQCHFw8oQB7+NX1W/DufG8Rrdl3grrdVIZ6tCiXamVbDp6mlu+kmDPoQzWFQiCxbNlzPXPG9MRxaI2SPoTDqHpfc12rmeTC9a9+vv3wGeKQWlK8vMY1erFQ7QyDpaW2sh617rr9J+n2MSkZ/67LmpHWeRQuScwRzGQNVOcn4YkeSixKw9pWtbTM5/63mn5Yk3JSakeBtjKYZLvjNmbDKEn/ZXtP165PP2hXg1pXKUziXR7spkFiRXN7vbOkfs6SPIX/7mYGQbNs1FsfbtO+bjEafFcVs82jot7f//xLFfqlRMTRm5tIrGU3o1s4XTT/ZvNvt5RH6han1+6q7LTbNN1+6LRNNG7+Tm2NRmnFgy1ezbbIcbk7T0wd09spNFUx79iwJPUxiPbidAyr7f3WQ6zOL5L1oQB7SN9vwZPsOEZfvGBX9eGWL+kaWYksmjsbfbJoF9UvnZcmPVU3XNNUnx8+dY4Sh87R/ua1d7s+J7uXioTRaZMoPF6ke1ZfJErnz05zejS1tA9mK0vWrlaVCtGHj9Q024ycJP/4v6/XENvTcpFg9aYHtlhR9SI3axIkQ7AJBJtCcGrje2omUKh02Nxm4tI91Pu79VpzznLHdqnBSq9v1tKXK/ZpH1t9+bCIwLC6mh2LK0TLQ9zK2i5dSqZSr07TmnzyeC26qfyVDF+yV17eClmZK9dVsyfyvzs1KU0cOxwlOAHJosY1rGTNExOtBmXy0j01EujFr9Zofgbsb+BGeWvmFhozd7vWldXbMzfGN+rDbz3Eq3V40S8UYC+oXu7Tb8ETG8p2dYrREF2g+WDOWuGWP2r2Nho5e6vmSFarRG6asGSvLdsm9WSpYI4stPTVKzFzw83B6udHTp8njnksxauECty/kXPh05+voFkbD1GwhAdW16PWVyM02HkRMTv269M304fzdli2n5Zsgy/eUpY4xbCVIpnCuI2V0GRWxpC6/aesp88W79H+aTUGqIQ141M6Pq0TxT2Y7aZEieCxwmVBGzFrC43+JeUB+txNZehF3U2OnbVabaNms+rSrDT1bBl7ypis4f12NahNlcIBBPJCbyfNt1WOZuvrb6wk86bZ9vFYTw2racV84aP5O2nItE1aZCB+0eAXUzefR2qmTq9/w8zuu996iNl5RUM9KMAe7oLfgtdl4ir6aV2Soe1RsHBd4ZYviQ04piFHNPh65X5qUbEgjbPhGV6uz3Q6f/GSp1EZeD2nz12gKgNSHJW4BEs8EG7tZj43Ci8nCpIXsY7Vk/S7qxehkR7ZmEkMaKuhfBq+8Qvt//NvMnLEDMdTvT50KzVpsDHlxY4/n/1iYypTIHwYJemr+du/0o4jf5Hs77MTVtL09QcpmO2mmlzijmrX0+gQTmUTluyhPt+nnBZ7EeM53B7w5+IAyf/fbNp0M/36WUdO6UfcX43a1kgIDC0vp15morS6TvWlltva+e5YHTPW60uYSb5N3DakjenlSLIZzlDZq1U5zR7cTlSjYAOOX7xbCxXKJVg8ftOTdami33qIS9P2pRsowB5i9lvwQp1uBEvYEG75akQFvjbizDV2PcMTh8ymw6fPE/czsaM1E4pw81Q//+diSjgpKV6fpKnxlzm8HIej4xIs3JWVtejrqlEInmlSil5pXcFJd0HbStprq+Yqoniw4xQ7UFkpql2fFTt1K2NIXfVBZTUj2B1jFtLa/Sfp+eY3UPdbypKEOXu0XnEadOfVtptqOMGnGpWk3rdenQVO5iUvqvxvqxE47HAwaiMZI/mzYEk73BrLq37EeXfI3ZWpXZ3igWEeGreEFu88Rk80KEn9bg++D17Ny6hfffxnL2Ok+7kuL8caO28HDZu+2XKyiU1JpwKxfx+oVVQzN3IzTKYa7vObTvWoVok8XmIw1bffeoipSUVJJSjAHm6E34In9k1G4Vck/E++a7MQe82aLao5AV8Vcegotp16+37rnuESWcCuAm12znwqW/rVaXTpcjxVr1LqynzUFNOFcmalDpcD7VtxzjC7NtXkgtPp8um2F2XlnuN0zweLta6NYmRyNIo82TNTKSWcl5qNcOwjNamlQTbCUHMV+7mM6dNpdn0ZPUxSoCYUWfpqcyqYI6tpjA+MXUxLdx0nUWbbvr+IVu09EdRuWbUT792mAj3VuFTQsVSHqEidIKnpXK2kiTYN0IeKTYbPpT3Hzmopp9XvyO3vLqR1f1x5efFhKmGHUKPGcGW92UbYDuKwgiRWypUtE/3ez7z97pnzF4kPLLhwwhHOvlitaC6a0sVazPJgyGesP0idJqzUPnYru5zT7fVbD3E6Xz/bQwH2kLbfgicnaEZ2hh8v2KklsbDqmMVKTaX+M+nsP/8GSNm1qRUHqcfrl6ABd1TykDxRhb4z6O8LKXP2+iHedPhc2n3srHZlzZxG/Lw1rLOT3cWrSuaYh6vTbVXdzQIn81KjW0x/vhFVKJwjMGXJqMQPHz6pvS5rJu2zs/9cpIr9Uh4udkwY5FrTzSvJYJyX7DxGD467khUqZ7aUNZgpelt7CakWzOmF44/yjQT/b7g05Goc3q871aPaEThB4ggqbJfK5Y17qtADtYuZwRJVdYz2hJ3j6r0+R3uJj5R5iREkfWZCCa8XVUCjbDLiWGon5brE2uZQhKwQ1y2Vh754up4rK1R9bdyOAGR3gn7rIXbnGYl2UIA9pO634LHiNXrONsMYqoN+3KhFcEgsmYe+esbal12Nmcq47CqwfOrGcTg5rFKVhJwekieqPmgW/Xn2gjaG11fJ4vjF9pJr/zippUf28pSbw26xacrHj9UOGk/WKVz1pPnTDrWpabkCgS7VeL2D76pM7eumXDGrytuPXRta3mNOM8wOZQ/VLkb31y7qdAkh27OZDGfrK5L7GsthvrpMWkUsy2KDLXbPA26vSI/rEtDIJDj8FtsCf/RYLcpx+YUh2ATZGTDpxN+anX0kUvXe9u4CWv9HSurmcAq7p5vkoHOjkJCLth+ldpczRHqZiMfqtPWh5759th7VLB75q3Or6/Cz/sGT54gjuXAYQqu3YHe/v4hW7z0RmK6bCSvOX/yXnvp8JZXKl93zQx6zvP3WQ8zOKxrqQQH2cBf8FjzxQL2hwLX084upA/J3/GwFzd50SLPLtBrY/qnPV2hKnZRnGpeiV9p4Y3vq1nbUGzaHkk6e07rz+kpR8r+zVzE7DR48dS5mbSdV/jcOmkUnzl4g1SaRnfDqvf6LdprJpWLhHFoUhXTp0pFq62o2Palb++1nPxLsXqI+yMuW1y9afq1R5JnH+7B9TWpVuZBfQ7s2jt5MhTsWJ2G+zZh2WWZdG9BBR2rYNu5Gf+PioGs0NSDw/BeraYqSmdFqqMdYg+q3HhJLfKAAe7hbfguepGRNyH0NLex1U6qVtRg5j7YeOmPLq3vw1I308cJdgf68dipzY0vELIH74jTOzZQTTDf6V/sQJyiOksFROLj876m6VK90XreH8rU/ySyohmWSGMfp01HAxprt59iOTo01bSU7k6+LcmEwfdxffWIMF4aIaBdqkhCvvzteLfTx/y6jX7ccCSTyYF8Gfim+eClZSzLB4euiqZR5dZo2Ny5eO4BG07ojMRc11CCP72U0nUisTz+m33pINKzZ7BygAJslZaOe34L31fJ99NK3aylv9sy0su8tgRmz3ShnRjp34ZKtK83PF++mfpdDu3CnsRCnUjXbYJMPNv3wqohJADsYHj1zXhvGCwc4r+YfrF9R7CUzFZ/6Nn5zLv1x4m9iO/DfdhzVHI3Ym/qNe6vS/K1H6NFPlmndbRncirJkzOD3lH0Z740ZmwMZEflFRyKOpBXbTcnmxzBj9UVOvpNy4yW3YxzOkZ0exW7dF4ExMYjqs2DkdGqiC1QxSeDrFfuo5zdrA7XtZK00OVRUVPNbD4mKRZucBBRgk6DsVPNb8DiNLKeT5R/5jYNaBaZ8+PQ5ShySkoXNatYrbjN3y+FAZAP+dyyERrpzzEJas/+ktuap3RpS5SLe2Ry/+OXvNHn1HwHebP/1S4hsX3ZkKRJtJKyeXPWrcsBMF24/Spww45pMGWhp7+a0YOtRYvtYTpm9dXDrSEzZlzHlFJyv0tnZr/prP2vjphXbzc4TV9K0dQdt/174sglhBun+5e/03eo/tGQ0ox+sTk3f+pX2Hj9LD9YuSq/fYy1Ftx/rUWMvp+WXRz9Yhhtj6c5j9MBlB1iua9enJdw40fK533pItKzbzDygAJuhZLOO34I3e+Mh6vj5CkqXjoiz47BdJhcOWXXvhykhrVb2uZnyXpvF0orYOan52/MCbQbeUYkeq1/CUh9+V75/7GLN6YjLnB5NqLQSrsvtufT+bh1NXLo30G24ZAduj+9Vf5JulMMFfd+lAYktuIQN4tNuvla+8G8yvXZnJU3x7fXtOsp3bWZa0efKDYRX84tUv/9ZuItem7pRi6gy4ck61OjNudpUZrzQiMoXuhItI1Lzczqu+kI37blGVPH62FvTK5PX0v+W7aObKxTQfqse+U/KzcQPXRtQ1YRcThG53p5jh/P3yWpiB9cnEgcdsgNd3WEpB0JcvE67HmmkfushkV6vlfGhAFuhZbGu34Knejmr+dEnr9qv5TzPnjkDrR/YMqAYm10Oe7aW7zuDki/H1R3Wtop2BR7Nha/i+Uqey5JXmhPH5/WqqAkceIxoCrHkZM1iU359zqw0uXMDavBGivPbm/dUDURpYE/sqWuTtGDyfN3Mofb8CGPmZF1O236xbC+9PHkd5b8uC41/MpFavbNA6zKt2G6+MnkdcdxwLtESy9TqnomddsMy+ei6rBm1TH2Vi+Sgqd0aWe3Kl/qSPdJqXFtfJpfGBmGnw/L9ZhBHguHS/eay9PzN1tK2xxISv/WQWGIDBdjD3fJb8NSsbav73kK5s2fWVvfO7K30zuxtjjLeyA809/f2fdXoHotZvjzEbNi1GrliTf8WlPMa83Ferc5VzUvPbb94ui7VLRXbDnC8junrkujZiauIE1N0vamMJkPXZcmomTtky5xRw8R2wA9/tFT7/43L5tdeOqoUyUk/dmtoFWPM1FezKn7+ZB2654PftLmr37mYWYzBRNWU1Itevok4HXasFbHT5pcxTs3NDmZD765CD9eJzhd3cdq1mno81vYlWuYr6cx5PrFg0ueEm996iJO5+t0WCrCHxP0WvI0HTlGb0SmnUb+9fBNdf/nBJVeaYstpZ8kSVojbepmAwc7cjNp0+99q+nHNAe2jbUNaexpPlWMbc055Lmx1srZ/i6hzsrHDVX2h4pMpDommT/fLDpZsHrPz6F+BIeqXzkuTnvIu1bWdtbjZRjU1+rRDIj122fGP7Z7ZDCTWC9t1fzhvh7YMOyZT0bD+UbO30cjZWwNT4duvpb1v9ixuttM1S5bMcgWvo5ndGzvtDu3DEJBkNlzN60RJkd4Mv/WQSK/XyvhQgK3QsljXb8HbffQvzdmDi2r3KnE9JXWrxWVo1V/6Zg19tWK/9v8jlaLVyrw5ocI3K/dT5gzpaesQbx2yxi/ZQ5yGmkup/Nnplx5NrUw1auvu//MsNXwjxb5VipGd60fzd9KQaZsCdVpULKglcUirRc32NPKBatT9yzW+yJlfPNUbDTaZ4oxZsVbGzttBw6anvJRy4ZNfPgGO1nLr6AW04cApql4sF33X2Z20vNG61miYl3rLoZp0RcPc3J6D33qI2/P3sj8owB7S9Vvw1ExcauSDxCGz6fDp85qj0iP17Dmviec749JnBvMQoe2u+3y/jiYs2Ut+2NSJjTVP9q4br6d3Hqxue97R1JBtv8v1mRGYUo1iuTRbYH05/tc/VHfoHPrn3xSbuntqJNDb91eLpqW4Ohc13vErrctripYfcubqIkJ0JiHDuMr2Ia0pY4bYO9XWh270OhKM0725671F9Pu+E9Tohnw0/sk6TrtD+zAExJGVq41+qDqx43JaLX7rIbHEEQqwh7vlt+Cd/PsCVRs4S1vR153qUe0Seejvf/7VYgBzcRKnVOweuZ9YiA3KXvr8I8cOXL+90tzDXSaasT6JOk1YpY2RVhzgBJiaUpozCLKjm1FRsyul9bBCWw+dphYj52sYOjYsqSWJSUu2m58s3EWDpm6kDOnT0Y6hbTz97njVucRE5/6rJeSkKV2j2yZdotak9axkXu231X45syn7iXCJhRtNq+tT6/uthziZq99toQB7SNxvwbvw7yW6ofd0bUWfP5GoOSWpD+u5/9eUSubLbmvFfDrBpxRcvn22PtUsnttWP341Gj5zM703dweVKXAtzdalhXZ7DmoCCK+Tbrg993D9SUKRHFkz0rLeN1PWTMbJLdTYmt1uKkM9WpQL13XMfr7v+NlA6LM7b7xeS6ualmw3Jy3dS69+t06LGrNBiSceSxumvrC/cU8VeqB2dDq/CdP2Hy/V4mqn9duTaJEh9bkoz8pomZvb8/BbD3F7/l72BwXYQ7qREDxJqflh+5rUqnIhUh12trxm30nn1LkLVGfIHDp38V9a1OuKg52H+Bx1zWGcOJxTs3L56b8dEh31Fa6xxElmW0nOMpU9Bm0mg61RHCifblyKXm1TISgKdoZrPWoBbT54mt68tyrdX6toOGwx+zmbfNS4nPyiXqm8tHjnMQpmHhKLi5QXurIFr6VZ3ZvE4hICsc/ZNIUdgiVqSbQuRnwsXrylLD3XPO2G5IoW/ucu/Eu1h8ym0+cueh4nPtJrjoQeEuk1mx0fCrBZUjbqRULwqvSfSafPX6R3HriR7qpehOQ6040r2jX7TtBf5y9S/TL5bNDwtwnbr87eeJhqlchNBXN4FwNYVsVKQ57smT3NOOcvwZTRWNlbtusY3VyhYFhbUE6RzPaxLSsV8jTqRiQ4qGPyw5PjYnPhG4bth8+kKdtNjpP686ZD2ql2CZs3RpHeI34hm7PpMBXLm43KFrwu0tMJO/6ff/2jnQA3r1Ag6pX1sIuJkQrr/zip/b7xTWlaLpHQQ2KFJxRgD3cqEoInDm+SrEK8Xfmk6n9Pp93QVB5uI7oGgVQEWLkq/eo0upRMWnxptr1vXbkQfdC+JkiBAAiAQFQRiIQeElUAQkwGCrCHOxUJwWsyfC7tOXaW+t5WkZ5sWJIk3uEDtYrSG/dW9XC16BoE4odA5f4z6cz5i4EFs3MgOwmigAAIgEA0EYiEHhJN6w81FyjAHu5UJARPnJZ6tixHXZqVoZtHzNOuaOXfHi4XXYNA3BCQmxZZcFqPfBE3G4uFgkAaIxAJPSRWEEIB9nCnIiF4Ek+ya7MyxA4VkvP83Yeq0+1pONahh9uIrkHgKgKSulY+6Ny0NL3UqjxIgQAIgEBUEYiEHhJVAEJMBgqwhzsVCcF7+KMl9NuOY5r5w1ONSlHdYXO0FU7p0oCqFc3l4WrRNQjEDwGOeLEp6VRgwbhhiZ+9x0pBIJYIREIPiRU+UIA93KlICN6Tny6nOZsP00OJxeju6kWIA6xzWd33FsqdPbOHq0XXIBA/BCS9uKx44B2V6LH69rIsxg81rBQEQMBvApHQQ/xeo93xoADbJWeiXSQEr8ukVfTT2iRN+a1fOi/1/GYtXZc1I63t34LSpUtnYtaoAgIgEI7AI/9ZSgu2HQ1UC5UlL1xf+BwEQAAEvCIQCT3Eq7W43S8UYLeJKv1FQvB6fr2Gvl65nzilJgeyH/3Ldqp0fQ766blGHq4UXYNAfBHoNH4lzdhwMLDoD9vXoFaVC8cXBKwWBEAg6glEQg+JeiiXJwgF2MOdioTg9Zuynj5fvEcL7p0nWyb6/vcDiFHq4R6j6/gkIBnyZPXjn0ykRjek7YD68bnTWDUIxDaBSOghsUIMCrCHOxUJwRs2bRONnb+TEkvkoYuXLtGqvSfomSal6JXWwdPYeogAXYNAmiTQ5/t1NGHJ3sDaJneuTzWK5U6Ta8WiQAAEYpdAJPSQWKEFBdjDnYqE4I38eSuNmrONqhTJSUkn/6ajZ/6hIXdXpnZ1inu4UnQNAvFFYOi0TTRu/s7Aomd1bxwTKXfja5ewWhAAgUjoIbFCHQqwhzsVCcEbO28HDZu+mQrnzEpJJ89pq8P1rIebjK7jksA7s7fSO7O3Bda+sFczSsidLS5ZYNEgAALRSyASekj00kg9MyjAHu5UJATv88W7qd+UDalWNb9nMyqWFw9nD7caXccZgXHzd9DQaZsDq/693y2UKxvCDMaZGGC5IBD1BCKhh0Q9lMsThALs4U5FQvC+WrGPXvpmbWBVGdKno82vtaJMGdJ7uFJ0DQLxRWD8kj3U9/v1gUVvG9Ia37H4EgGsFgRigkAk9JCYAENEUIA93KlICN7UtQeo66TVgVUVzXMNLXjpJg9Xia5BIP4ITF61n178ao228MwZ09PWwa3jDwJWDAIgEPUEIqGHRD0UnAB7v0WRELw5mw7Rk5+tCCyuQZm8NLFjXe8XixFAII4IzFifRJ0mrNJWnCd7ZlrV95Y4Wj2WCgIgECsEIqGHxAobnAB7uFORELzfth+lhz9eGlgVp0Qe1raKh6tE1yAQfwTmbz1Cj36yTFs4blnib/+xYhCIFQKR0ENihQ0UYA93KhKCt3rvn3T3+78FVtWrVXl6tmlpD1eJrkEg/gis2H2c7v1wsbbw8oWuoxkvNI4/CFgxCIBA1BOIhB4S9VAuTxAKsIc7FQnB23zwFLV6Z0FgVe89XINurYoUrR5uM7qOQwIbD5yiNqNTvme1iuemb56tH4cUsGQQAIFoJxAJPSTamcj84koB3r59O7311lu0ZMkSWr9+PZUvX177X32ZNm0a9e7dmzZt2kQJCQn04osvUufOnS3vaSQEb8+xv6jJ8F8Dc53arSFVLpLT8tzRAARAIDiB3Uf/oqZvpXzPOO34508kAhcIgAAIRB2BSOghUQchyITiSgGeMmUKde3alerUqUNbt26lS5cuXaUAL168mBo3bkyPPvootW/fnhYtWkT9+/ensWPHUseOHS3tayQE7/Cpc5Q4dE5gnmv6t6Cc12SyNG9UBgEQCE3g8OlzlDgk5XvWpkoher9dTSADARAAgagjEAk9JOogQAEmTeFNnz4lHu7jjz9OK1asuEoBbt26NR0/fpyWLr3iSPb000/T1KlTiQVJ2pvZ4EgI3ulzF6jKgFna9FjxZQUYBQRAwF0CZ85fpMr9Z2qd3lczgYbfV83dAdAbCIAACLhAIBJ6iAvT9qWLuDoBVokaKcDnz5+nHDly0Ouvv07du3cPVJ83bx41bdpUU5hr1jR/0hMJwbvw7yW6ofd0be5VE3LSD10b+iJIGAQE4onAv5eSqfSr01JepuuXoAF3VIqn5WOtIAACMUIgEnpIjKCJ30QYRgrwxo0bqVKlSjR9+nRq1apVYA+PHDlCBQoUoPHjx2tmEWZLpATvht7T6MK/yZrzGzvBoYAACLhPoHzf6XTuwiXq2qwM/V/Lcu4PgB5BAARAwCGBSOkhDqftS3OcACtOcGzv27BhQ2I74Lp1rySPuHjxImXKlIlGjRpFzz33XNCNOXXqFPF/UpKSkigxMZH27dunOdP5VaoOmEmnzl3Uwp9xGDQUEAAB9wnUeO1nOv7XP9p3DKEG3eeLHkEABJwTgAIcnCEUYAMFmKNEsKOcFFGAR48eTd26dQtKc8CAATRw4MCrPvdbAb55xDzafvgMjXrwRrrzxiLOv0HoAQRA4CoCbUYtoI1Jp+idB26ku6rjewYRAQEQiD4CUIChAF9FwAsTiGg5AV63/ySt2HOc2tctTpkypDj9oYAACLhLgGMBL9t1jB6qU4yyZMzgbufoDQRAAARcIAAFGAqwKQU4LTjBufB9QRcgAAIgAAIgAAJpgAAUYCjAphRgrsRh0E6cOKHZAUvp1KkT/fDDDzERBi0NfF+xBBAAARAAARAAARcIQAGGAqwROHv2LHGWNy7vvfce7dixg0aMGKH9u0mTJpQ/f35N8eVEGGwi0a5dOy0RRr9+/WImEYYL3xd0AQIgAAIgAAIgkAYIQAGGAqwR2L17N5UsWdKQxty5c7VYv1xYSX711VdTpULu0qWL5a8CBM8yMjQAARAAARAAARBwiQD0ECjALomStW4geNZ4oTYIgAAIgAAIgIB7BKCHQAF2T5os9ATBswALVUEABEAABEAABFwlAD0ECrCrAmW2MwieWVKoBwIgAAIgAAIg4DYB6CFQgN2WKVP9QfBMYUIlEAABEAABEAABDwhAD4EC7IFYhe8SgheeEWqAAAiAAAiAAAh4QwB6CBRgbyQrTK8QvIhgx6AgAAIgAAIgAAJEWv6CokWL0r59+yghIQFMFALpkpOTk0HEGwIQPG+4olcQAAEQAAEQAIHwBKCH4AQ4vJR4UAOC5wFUdAkCIAACIAACIGCKAPQQKMCmBMXtShA8t4miPxAAARAAARAAAbMEoIdAATYrK67Wg+C5ihOdgQAIgAAIgAAIWCAAPQQKsAVxca8qBM89lugJBEAABEAABEDAGgHoIVCArUmMS7UheC6BRDcgAAIgAAIgAAKWCUAPgQJsWWjcaADBc4Mi+gABEAABEAABELBDAHoIFGA7cuO4DQTPMUJ0AAIgAAIgAAIgYJMA9BAowDZFx1kzCJ4zfmgNAiAAAiAAAiBgnwD0ECjA9qXHQUsIngN4aAoCIAACIAACIOCIAPQQKMCOBMhuYwieXXJoBwIgAAIgAAIg4JQA9BAowE5lyFZ7CJ4tbGgEAiAAAiAAAiDgAgHoIVCAXRAj611A8KwzQwsQAAEQAAEQAAF3CEAPgQLsjiRZ7AWCZxEYqoMACIAACIAACLhGAHoIK87gQQAAGKpJREFUFGDXhMlKRxA8K7RQFwRAAARAAARAwE0C0EOgALspT6b7guCZRoWKIAACIAACIAACLhOAHgIF2GWRMtcdBM8cJ9QCARAAARAAARBwnwD0ECjA7kuViR4heCYgoQoIgAAIgAAIgIAnBKCHQAH2RLDCdQrBC0cIn4MACIAACIAACHhFAHoIFGCvZCtkvxC8iGDHoCAAAiAAAiAAAkQEPQQKcES+CBC8iGDHoCAAAiAAAiAAAlCAQ8pAuuTk5GRIiTcEoAB7wxW9ggAIgAAIgAAIhCcAPQQnwOGlxIMaEDwPoKJLEAABEAABEAABUwSgh0ABNiUobleC4LlNFP2BAAiAAAiAAAiYJQA9BAqwWVlxtR4Ez1Wc6AwEQAAEQAAEQMACAeghUIAtiIt7VSF47rFETyAAAiAAAiAAAtYIQA+BAmxNYlyqDcFzCSS6AQEQAAEQAAEQsEwAeggUYMtC40YDCJ4bFNEHCIAACIAACICAHQLQQ6AA25Ebx20geI4RogMQAAEQAAEQAAGbBKCHQAG2KTrOmkHwnPFDaxAAARAAARAAAfsEoIdAAbYvPQ5aQvAcwENTEAABEAABEAABRwSgh0ABdiRAdhtD8OySQzsQAAEQAAEQAAGnBKCHQAF2KkO22kPwbGFDIxAAARAAARAAARcIQA+BAuyCGFnvAoJnnRlagAAIgAAIgAAIuEMAeggUYHckyWIvEDyLwFAdBEAABEAABEDANQLQQ6AAuyZMVjqC4FmhhbogAAIgAAIgAAJuEoAeAgXYTXky3RcEzzQqVAQBEAABEAABEHCZAPQQKMAui5S57iB45jihFgiAAAiAAAiAgPsEoIdAAXZfqkz0CMEzAQlVQAAEQAAEQAAEPCEAPQQKsCeCFa5TCF44QvgcBEAABEAABEDAKwLQQ6AAeyVbIfuF4EUEOwYFARAAARAAARAgIughUIAj8kWA4EUEOwYFARAAARAAARCAAhxSBtIlJycnQ0q8IQAF2Buu6BUEQAAEQAAEQCA8AeghOAEOLyUe1IDgeQAVXYIACIAACIAACJgiAD0ECrApQXG7EgTPbaLoDwRAAARAAARAwCwB6CFQgM3Kiqv1IHiu4kRnIAACIAACIAACFghAD4ECbEFc3KsKwXOPJXoCARAAARAAARCwRgB6CBRgaxLjUm0Inksg0Q0IgAAIgAAIgIBlAtBDoABbFho3GkDw3KCIPkAABEAABEAABOwQgB4CBdiO3DhuA8FzjBAdgAAIgAAIgAAI2CQAPQQKsCXR+fTTT6lDhw5XtenVqxe9/vrrpvuC4JlGhYogAAIgAAIgAAIuE4AeAgXYkkiJAjxjxgzKmTNnoG2RIkWoaNGipvuC4JlGhYogAAIgAAIgAAIuE4AeAgXYkkiJAnzkyBHKly+fpbZqZQiebXRoCAIgAAIgAAIg4JAA9BAowJZECAqwJVyoDAIgAAIgAAIgEIUEoABDAbYklqIAFyhQgI4ePUrFixenp556il566SXKkCGD6b4geKZRoSIIgAAIgAAIgIDLBKCHQAG2JFIzZ86kpUuXUp06dShdunT0ww8/0AcffEDPPvssjRkzJmhfp06dIv5PSlJSEiUmJtK+ffsoISHB0hxQGQRAAARAAARAAAScEIACDAXYifxobXv27EkjR47UlNnChQsb9jdgwAAaOHDgVZ9BAXaMHx2AAAiAAAiAAAhYJAAFGAqwRZG5uvry5cu109xp06ZR69atDfvDCbBjzOgABEAABEAABEDAJQJQgKEAOxalZcuWaSYRoRRg/SAQPMfY0QEIgAAIgAAIgIBNAtBDoADbFJ0rzXr06EGjRo0iFqZChQqZ6g+CZwoTKoEACIAACIAACHhAAHoIFGBLYtWyZUtq3rw5Va5cWWvHTnDjxo2j559/XrMDNlsgeGZJoR4IgAAIgAAIgIDbBKCHQAG2JFOs6E6fPl077b106RKVLVuWOnbsSN26ddOiQpgtEDyzpFAPBEAABEAABEDAbQLQQ6AAuy1TpvqD4JnChEogAAIgAAIgAAIeEIAeAgXYA7EK3yUELzwj1AABEAABEAABEPCGAPQQKMDeSFaYXiF4EcGOQUEABEAABEAABIg0U86iRYsiIZeBNKRLTk5OhpR4QwCC5w1X9AoCIAACIAACIBCeAPQQnACHlxIPakDwPICKLkEABEAABEAABEwRgB4CBdiUoLhdCYLnNlH0BwIgAAIgAAIgYJYA9BAowGZlxdV6EDxXcaIzEAABEAABEAABCwSgh0ABtiAu7lWF4LnHEj2BAAiAAAiAAAhYIwA9BAqwNYlxqTYEzyWQ6AYEQAAEQAAEQMAyAeghUIAtC40bDSB4blBEHyAAAiAAAiAAAnYIQA+BAmxHbhy3geA5RogOQAAEQAAEQAAEbBKAHgIF2KboOGsGwXPGD61BAARAAARAAATsE4AeAgXYvvQ4aAnBcwAPTUEABEAABEAABBwRgB4CBdiRANltDMGzSw7tQAAEQAAEQAAEnBKAHgIF2KkM2WoPwbOFDY1AAARAAARAAARcIAA9BAqwC2JkvQsInnVmaAECIAACIAACIOAOAeghUIDdkSSLvUDwLAJDdRAAARAAARAAAdcIQA+BAuyaMFnpCIJnhRbqggAIgAAIgAAIuEkAeggUYDflyXRfEDzTqFARBEAABEAABEDAZQLQQ6AAuyxS5rqD4JnjhFogAAIgAAIgAALuE4AeAgXYfaky0SMEzwQkVAEBEAABEAABEPCEAPQQKMCeCFa4TiF44QjhcxAAARAAARAAAa8IQA+BAuyVbIXsF4IXEewYFARAAARAAARAgIigh0ABjsgXAYIXEewYFARAAARAAARAAApwSBlIl5ycnAwp8YYAFGBvuKJXEAABEAABEACB8ASgh+AEOLyUeFADgucBVHQJAiAAAiAAAiBgigD0ECjApgTF7UoQPLeJoj8QAAEQAAEQAAGzBKCHQAE2Kyuu1oPguYoTnYEACIAACIAACFggAD0ECrAFcXGvKgTPPZboCQRAAARAAARAwBoB6CFQgK1JjEu1IXgugUQ3IAACIAACIAAClglAD4ECbFlo3GgAwXODIvoAARAAARAAARCwQwB6CBRgO3LjuA0EzzFCdAACIAACIAACIGCTAPQQKMA2RcdZMwieM35oDQIgAAIgAAIgYJ8A9BAowPalx0FLCJ4DeGgKAiAAAiAAAiDgiAD0ECjAjgTIbmMInl1yaAcCIAACIAACIOCUAPQQKMBOZchWewieLWxoBAIgAAIgAAIg4AIB6CFQgF0QI+tdQPCsM0MLEAABEAABEAABdwhAD4EC7I4kWewFgmcRGKqDAAiAAAiAAAi4RgB6CBRg14TJSkcQPCu0UBcEQAAEQAAEQMBNAtBDoAC7KU+m+4LgmUaFiiAAAiAAAiAAAi4TgB4CBdhlkTLXHQTPHCfUAgEQAAEQAAEQcJ8A9BAowO5LlYkeIXgmIKEKCIAACIAACICAJwSgh0AB9kSwwnUKwQtHCJ+DAAiAAAiAAAh4RQB6CBRgr2QrZL8QvIhgx6AgAAIgAAIgAAJEBD0ECnBEvggQvIhgx6AgAAIgAAIgAAJQgEPKQLrk5ORkSIk3BKAAe8MVvYIACIAACIAACIQnAD0EJ8DhpcSDGhA8D6CiSxAAARAAARAAAVMEoIdAATYlKG5XguC5TRT9gQAIgAAIgAAImCUAPQQKsFlZcbUeBM9VnOgMBEAABEAABEDAAgHoIVCALYiLe1UheO6xRE8gAAIgAAIgAALWCEAPgQJsTWJcqg3BcwkkugEBEAABEAABELBMAHoIFGDLQuNGAwieGxTRBwiAAAiAAAiAgB0C0EOgANuRG8dtIHiOEaIDEAABEAABEAABmwSgh0ABtik6zppB8JzxQ2sQAAEQAAEQAAH7BKCHQAG2Lz0OWkLwHMBDUxAAARAAARAAAUcEoIdAAXYkQHYbQ/DskkM7EAABEAABEAABpwSgh0ABtixDW7dupeeee44WLFhA2bNnp4ceeohef/11uuaaa0z3BcEzjQoVQQAEQAAEQAAEXCYAPQQKsCWROnHiBFWuXJmKFy9Offv2pcOHD9OLL75IrVq1ogkTJpjuC4JnGhUqggAIgAAIgAAIuEwAeggUYEsi9cYbb9CgQYNoz549lC9fPq3tpEmTqF27drRx40aqUKGCqf4geKYwoRIIgAAIgAAIgIAHBKCHQAG2JFZNmjShXLly0ZQpUwLtzp8/Tzlz5qQhQ4ZQjx49TPUHwTOFCZVAAARAAARAAAQ8IAA9BAqwJbEqUKAAPfHEE5rNr1oqVapE9erVo48//thUfxA8U5hQCQRAAARAAARAwAMC0EOgAFsSq0yZMtFrr71GL7/8cqp2DRs2JFaOJ0+ebNjfqVOniP+Tsm/fPqpfvz4tW7aMChcubGkOqAwCIAACIAACIAACTggkJSVRYmIi7dq1i0qUKOGkqzTXNl1ycnJymluVwwWxAjx48GDq1atXqp4aNGhAhQoVom+//dZwhAEDBtDAgQMdjo7mIAACIAACIAACIOAeAT6Iq127tnsdpoGeoAAbbKJdEwj9CfC5c+eIT4FLlixJGTNm9ERc5O0Op8ye4PWkU+yZJ1g97RR75ileTzrHnnmC1dNOsWfu47148SIdOXKEqlSpQlmzZnV/gBjuEQqwwea55QTnh1zAvscPyu6OgT1zl6cfvWHP/KDs7hjYM3d5+tEb9swPyhhDCEABNpAFDoPGNsAcBi1v3rxajS+++EJLhmElDJofYoYfDD8ouzsG9sxdnn70hj3zg7K7Y2DP3OXpR2/YMz8oYwwowCFkQBJhsMG4mgijZcuWlhJh+CFm+MHwg7K7Y2DP3OXpR2/YMz8ouzsG9sxdnn70hj3zgzLGgAIcRgY4FXK3bt1o4cKFlC1bNu30l0+GraRC9kPM2O54xIgRWqa6HDly+DEkxnBIAHvmEGAEmmPPIgDd4ZDYM4cAI9AcexYB6HE8JEwg4njzsXQQAAEQAAEQAAEQiEcCUIDjcdexZhAAARAAARAAARCIYwJQgON487F0EAABEAABEAABEIhHAlCA43HXsWYQAAEQAAEQAAEQiGMCUIDjePOxdBAAARAAARAAARCIRwJQgGN01zlKxXPPPUcLFiyg7Nmza1EqXn/99aiLUhGjeB1N++uvv6aJEyfSypUr6fjx41S6dGl69tln6ZlnnqH06dMH+p42bRr17t2bNm3aRAkJCVokj86dOzsaG43dIXDmzBkqX748/fHHH7R8+XKqVasW9s0dtK738p///IdGjx5NW7Zs0SLh1K1bl3744Qfsl+uk3enw+++/p2HDhmm/exxVqUGDBtq/y5Url2oA/D66wxu9BCcABTgGpUPiFBcvXjxVnOJWrVpFXZziGMTreMr8AOa9ufvuu6lgwYI0d+5c7Qf+hRdeoOHDh2v9L168mBo3bkyPPvootW/fnhYtWkT9+/ensWPHUseOHR3PAR04I9CrVy/67LPP6NChQ6kUYOybM65utx4wYACNHDlSe5GsU6eO9sI5Y8YM7XuE75nbtJ33N3v2bGrRooX2m/fII48QP8t4Dzn82YYNGwKhPPE9c84aPYQnAAU4PKOoq8HxiAcNGqRlqsuXL582v0mTJlG7du2iLlNd1MHzYUKcdz1//vypRuLT3Q8++ED7wc+SJQu1bt1ae1gvXbo0UO/pp5+mqVOnEgeDV0+KfZgyhlAIbN68WTvxffvtt6lTp06pFGDsW/SICp8gVqlShfikkJUqo4L9ip794pnwy/2cOXNo586dlC5dOm1yy5Yt015eeB95v7hg36Jr39LqbKAAx+DONmnShHLlykVTpkwJzP78+fOUM2dOGjJkCPXo0SMGV5W2pzx+/HjttPfAgQOUJ08e7aSDTVa6d+8eWPi8efOoadOmtGLFCqpZs2baBhLFq2NlqmrVqnTbbbdRs2bNAgowf8ewb9GzcXxK/9133xGbgxkV7Ff07JXM5LHHHqPVq1fT2rVrA5Pj/WPzh59++onatGlD2Lfo27e0OiMowDG4swUKFKAnnnhCU6DUUqlSJapXrx59/PHHMbiqtD1lPt399ttv6fDhw5qtIu/V9OnTic1WpPDJMe8tK8t8RYjiP4FvvvmGunTpQtu2baNVq1alUoA3btyIffN/S4KOyAcBfAN244030rvvvqvdrvDv36hRo7S/Yb+iaLMuT2X+/PnUvHlzLXupmECwL8v27ds1xZhvx7Bv0bdvaXVGUIBjcGczZcpEr732Gr388supZt+wYUNNgZo8eXIMrirtTplPdPnBzDa+ffr00ex9ea/Yzo3thaVcvHiReG/5Ac4PBRR/CZw9e1ZzfGObRH7B/PXXX1MpwNg3f/cj3Gh8asg3KkWKFKGhQ4dS5syZaeDAgbR7927tBYZtSvE9C0fR/8/ZzOvhhx+m06dPa4NXrFiRZs6cqTkCc8H3zP89idcRoQDH4M6zkjR48GDiK0C1sDdtoUKFtJNGlOggcPDgQc2+jX/cWaHivZMf+CVLlmif6RVg9mjv1q1bdCwgjmbx6quv0s8//6zZZbMNdjAFGPsWHUJxww03aCeH69ev107muSQlJVHJkiU1Hwn+PWQFGPsVHfvFs/jtt980M4fHH3+c7rjjDjp58qT28nLu3Dntd5FNjPD7GD37ldZnAgU4BncYJhCxsWn84842vfzjvnDhQsqbN682cVzxRd/+sUNp2bJlNZvS+vXraxPkPbv99tu1KB7sFLd3716YQETR1vHLI+8bv2Sqhc0fqlWrph0QwNQoijaMSPseFStWLNUtJZt+8QEBR8phZ2H8PkbXnqXl2UABjsHdhRNc9G8aK70tW7bUrmLZ1IHDokmBk0f07Z+c9gabGStb7KQIJ7jo2Ts+ReSQZ3oFmJXfGjVq0Icffoj9ip7t0maSLVs2zXSvX79+qWbGp/Z8Mvzee+/BCS7K9iwtTwcKcAzuLodBYxtgPv2QU8UvvvhCS4bBb88VKlSIwVWlnSmzLW/btm2JHT74P44ooC8c5oeddlg5lsIhtziAP8Kg+S8LvBe///57qoH53xylgxWp2rVra0oV9s3/vQk2Ijss3nfffbRu3TqqXLmyVo0Tl5QqVYr4N5LjbmO/ome/eCb8bOKbFjWCEb/A8KkwRzDq2bOnNmHsW3TtW1qdDRTgGNxZSYRRokSJVIkw+MRxwoQJMbiitDVlzvg2btw4evPNN6lRo0apFscOH3yKKIHe+RSL4zez3RufiiARRvTIgt4GmGeGfYue/fn3338pMTFRc6Zinwh2gmPbX4m0whkysV/Rs188kzFjxmj+DRxp5c4779QOAdgGmA9z2GmxcOHC2oSxb9G1b2l1NlCAY3RnOXYi/5CwnSJfK/HpL596cGpJlMgS4BcT/kE3KmxPynbBXDjwOzteqamQ+cGAEh0EjBRg7Ft07I3MgpVdPqXnGLIXLlwgNg/jzHBqWl18z6Jnz5KTk+mjjz6i999/X3NgvPbaa7WXGD795aQmasG+Rc++pdWZQAFOqzuLdYEACIAACIAACIAACBgSgAIMwQABEAABEAABEAABEIgrAlCA42q7sVgQAAEQAAEQAAEQAAEowJABEAABEAABEAABEACBuCIABTiuthuLBQEQAAEQAAEQAAEQgAIMGQABEAABEAABEAABEIgrAlCA42q7sVgQAAEQAAEQAAEQAAEowJABEAABEAABEAABEACBuCIABTiuthuLBQEQAAEQAAEQAAEQgAIMGQABEAABEAABEAABEIgrAlCA42q7sVgQAAEQAAEQAAEQAAEowJABEAABEAABEAABEACBuCIABTiuthuLBQEQAAEQAAEQAAEQgAIMGQABEAABEAABEAABEIgrAlCA42q7sVgQAAEQAAEQAAEQAAEowJABEAABEAABEAABEACBuCIABTiuthuLBQEQAAEQAAEQAAEQgAIMGQABEAABEAABEAABEIgrAlCA42q7sVgQAAEQAAEQAAEQAAEowJABEAABEAABEAABEACBuCIABTiuthuLBQEQAAEQAAEQAAEQgAIMGQABEAABEAABEAABEIgrAlCA42q7sVgQAAEQAAEQAAEQAAEowJABEAABEAABEAABEACBuCIABTiuthuLBQEQAAEQAAEQAAEQgAIMGQABEAABEAABEAABEIgrAlCA42q7sVgQAAEQAAEQAAEQAAEowJABEAABEAABEAABEACBuCIABTiuthuLBQEQAAEQAAEQAAEQgAIMGQABEAABEAABEAABEIgrAlCA42q7sVgQAAEQAAEQAAEQAAEowJABEAABEAABEAABEACBuCIABTiuthuLBQEQAAEQAAEQAAEQgAIMGQABEAABEAABEAABEIgrAlCA42q7sVgQAAEQAAEQAAEQAIH/B+viY01911qQAAAAAElFTkSuQmCC\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "fig, ax = plt.subplots()\n",
    "ax.plot([np.sum(_flg) for _flg in blast_keep_dic.values()])\n",
    "ax.set_ylim([0,30])\n",
    "fig.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c19b578f",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "postanalysis",
   "language": "python",
   "name": "postanalysis"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
